<!DOCTYPE html>
<html lang="en">
<head>
<title>Notes on Fundamentals, Concepts, and Techniques in Molecular Biology</title>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8">
<link href="./stdcss/std.css" rel="stylesheet">
<link href="./stdcss/biblio.css" rel="stylesheet">
</head>
<body>
<p id="title">
Notes on Fundamentals, Concepts, and Techniques in Molecular Biology

<div class="container">
  <!-- ================================================================ -->
<p class="authors">
   Williams WV, Rosenbaum H, Weiner DB
<p>
<i>PCR Methods and Applications</i>, Cold Springs Harbor,
  <span class="pages">
   86-88.
  </span>
<p class="reftitl">
Effect of RNA concentration on cDNA synthesis for DNA amplification
<p class="1stP">
What effect does the abundance of message have on the amplification
techniques?  Primers used for checking for low abundance message
were for human c-myc oncogene:

<table class="center seqtbl">
<tr> <td>c-myc  5':  <td> 1763 <td> GCTTCTCAGAGGTTGG    <td> 1779
<tr> <td>c-myc  3':  <td> 2205 <td> CGTCTAAGCAGCTGCAAG  <td> 2188
</table>
<p>
Primers used for highly expressed message are derived from
human and murine g-actin, aligned for maximal homology utilizing
Wordsearch and Segments.  Jurkat (human T) cells are grown
in RPMI-1640 with pen/strep, L-Gln, and 10% FCS present;
passed twice weekly.  Cells centrifuged (1200 rpm/10 min)
and lysed in 4 M guanidine isothiocyanate, 0.5% N-lauroyl sarcosyl,
25 mM Na citrate, and 0.1 M 2-mercaptoethanol.  Lysate
extracted with phenol, then phenol/CHCl<sub>3</sub>/isoamyl alcohol,
and RNA is precipitated in 50% EtOH.  Pellet resuspended in
50 &mu;l diethylpyrocarbonate-treated water, run on gels,
measured for amount at 260 nm.   Varying amounts used for
cDNA synthesis using either 0.5 or 1.0 mM dNTPs:  200-400 units
Moloney murine leukemia virus reverse transcriptase in a
DTT-containing buffer with 4 units RNase inhibitor (GIBCO/BRL
supplies buffer, RTase, inhibitor), and random priming hexamers
(0.6 to 1.2 &mu;g), with dNTPs present;  reaction preincubated
10 min at 25&deg;, then done 60 min at 42&deg;, then 5 min at 95&deg;,
stored &minus;20&deg;.
<p>
The cDNA products are then serially diluted and reacted
for 30 cycles with c-myc primers accordingly:  final concentrations
are dNTPs at 0.2 mM, primers at 0.8 &mu;M, and a 1:20 dilution of
cDNA if not previously purified of synthesis products (MgCl<sub>2</sub>
to be added later).   Denaturation is to 95&deg; for 5 min, then
MgCl<sub>2</sub> added to 2.5 mM final concentration, then 1 unit
<i>Taq</i> polymerase.  Samples are covered with mineral oil,
amplified using a program of 3 min at 94&deg;, then 15 cycles of 94&deg;
for 1 min (melt), 60&deg; for 1 min, (anneal), 72&deg; for 1 min (elongate).
Another 15 cycles are started with 94&deg; for 1 min, 60&deg; for 1.5 min,
then 72&deg; for 2 min.  Final elongation at 72&deg; for 7 min,
cooling to 4&deg; is performed.  Reaction is analyzed using 3% agarose
gels stained with ethidium bromide and UV illuminated.
<p>
Results with the low-abundance message c-myc show that UV
intensity varied with the RNA used in the cDNA synthesis
reaction, the variation increasing with increasing RNA
but leveling off at 10 &mu;g RNA.  Densitometric scanning
verified the visually apparent results.  Altering the
amounts of added dNTP and RTase did not appear to cause
changes in the amplified DNA levels.
<p>
As for high abundance message (g-actin), cDNA dilutions were
varied and amplified products measured as before.  Different
RNA and RTase affected product amounts considerably.
10 &mu;g RNA provided the best reaction and multiple dilution
of cDNA worked well (1:1000 worked well).  45 &mu;g RNA was not
as effective as 10 &mu;g.  These studies indicate that 10-15 &mu;g RNA
for both low- and high-abundance messages are the best input for
PCR amplification of the product.
</div>

</body>
</html>
