<!DOCTYPE html>
<html lang="en">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8">
<title>Notes on the Literature in Proteomics (Vol 1: 19xx-2006)</title>
<link href=".../stdcss/std.css" rel="stylesheet">
<link href=".../../stdcss/biblio.css" rel="stylesheet">
</head>
<body>
<p id="title">
Notes on the Literature in Proteomics
<p id="subtitle">
Vol 1:  19xx-2006
<!-- ============================================================= -->
<hr style="margin-top:4em;">
<p>
This is the first in a series of documents containing notes (more than
abstracts but less than the full published reports) on the data and results
in papers relevant to research on methods and methodologies in proteomics,
or in which proteomics is used as a tool.  The (HTML) titles for these
documents are:

<ul>
<li>Notes on the Literature in Proteomics (Vol 1: 19xx-2006)
 <br>[this document:  <tt>ProteomicsLit(19xx-2006).html</tt>]
<li>Notes on the Literature in Proteomics (Vol 2: 2007-)
 <br>[document:
 <tt><a href="ProteomicsLit(2007-).html">ProteomicsLit(2007-).html</a></tt>]
</ul>
<!-- ============================================================= -->
<p class="tags">
mucins, oligosaccharides
<p class="ref">
 <span class="authors">
Carlson DM, Blackwell C
 </span>
 <span class="year">
(1968)
 </span>
 <span class="jname">
J. Biol. Chem.
 </span>
 <span class="jvol">
243,
 </span>
 <span class="pages">
515-526.
 </span>
<p class="reftitl">
Structures and Immunochemical Properties of Oligosaccharides Isolated
from Pig Submaxillary Mucins
<p class="Para1">
Work on pig gastric mucosa has shown the following blood group
substances A, H, and AH.  Pig type A is thought to have identical
glycan structure to human type A.  Extracts of porcine submaxillary
glands show serologically similar to gastric secretions, being A, H,
AH, and I (the absence of A or H).  In this study sialic acid
is determined by resorcinol; after sialidase treatment, thiobarbituric
acid method is used.  The Dische and Shettles method is used to determine
fucose, galactose by orcinol-sulfuric acid.
<p>
Oligosaccharides were first reduced with sodium borohydride in alkaline
conditions:  1 g mucin is placed in 50 mM KOH with 1.0 M NaBH<sub>4</sub>
for 15 h at 45&deg; in 400 ml volume.  Borohydride is eliminated with
slow addition of 4 M acetic acid to pH 5.  This is then passed  through
4 &times; 40 cm Dowex 50-X8 (H<sup>+</sup>, 200-400 mesh).  The effluent
evapd on rotovap and B(OH)<sub>3</sub> removed as methyl borate.  A
clear syrup is dissolved in 250 ml water and sialic acid-containing
oligosaccharides adsorbed to 2 &times; 30 cm Dowex 1-X2 (Cl<sup>&minus;</sup>,
50-100 mesh).  Neutral sugars are washed off column unbound until eluate
is negative to anthrone reagent;  these are then fractionated on
3 &times; 4 cm charcoal-Celite.  N-acetylgalactosaminitol and oligosaccharide
V are batch-eluted with steps of 100 ml 5% EtOH then 100 ml 15% EtOH.
Sialic acid oligosaccharides are eluted from Dowex 1 with a 1.8 L total
volume 0 to 100 mM NaCl gradient in water, with 1 ml/min flow and fractionated
5 ml per.  Resorcinol and anthrone were used to test fractions.  Fracs
46-139 had sialic acid and hexose (pooled 46-105, and 106-140), 141-230
had sialic acid.  Pools were evapd, redissolved in water, desalted on Biogel
P-2.  Neutral sugars can be separated using charcoal-Celite, but
paper chromatography was required for sialic acid sugars.  Chromatography
on Schleicher &amp; Schuell No. 589 Green Ribbon was done with
6:4:3 n-butanol:pyridine:water (solvent A).  Periodate-benzidine was
used to show spots, which are eluted with water and cleaned up on Dowex 50-X8
(H<sup>+</sup> 200-400 mesh).
<p>
<b>Results</b>. Mucins which are immunochemically distinct A<sup>+</sup>-PSM and
A<sup>&minus;</sup>-PSM were obtained from pig submaxillary gland extracts
which were treated at pH 4.5 and precipitated as highly charged polyanionic
mucins with cetylpyridinium chloride.  The aggregate clot was redissolved
in physiological saline and then fractionated using EtOH.  In the
ultracentrifuge the mucins were monodisperse but not pure/homogeneous.
Carbohydrate compositions were determined for the different glands.
Per 100 g mucin, there are 23.4 g GalNAc, 8.5 g fucose, 11.5 g galactose,
and 14.6 g N-glycolylneurmaninic acid (N-GN).  There was less GalNAc,
fucose, galactose and more N-GN in H and I glands.
<p>
Carbohydrate structure of glycoproteins is facilitated by release
of sugar chains with minimum of degradation.  Previous research used
alkaline conditions involving 5-7 d incubations at room temperature but
yields were low with significant degradation.  Released of reduced sugars
by hydroxide worked sufficiently well.  there are high recoveries of
sialic acid and fucose and total loss of hexosamine.  Hydrolysis of
A<sup>+</sup>-PSM showed recoveries of GalNAc of 30-40%.  Galactose
is recovered at 70-85%.  Chromatography on charcoal-Celite or Biogel P-2
gives better than 90% yield.
<p>
Saccharide fractions show homogeneity in at least 5 chromatographic solvent
systems tried and in borate buffered electrophoresis.  Oligosaccharides
II, III, V, and VI are obtained from A<sup>-</sup>-PSM, with no I and IV
oligosaccharides.  A<sup>+</sup>-PSM has all six.  Oligosaccharide I
is a pentasaccharide with equimolar amounts of GalNAc, galactose, fucose,
N-GN, and N-acetylgalactosaminitol.  All other oligosaccharides lack
one or more of these, but all contain the alcohol of GalNAc.  The fucose
is isolated as a syrup and has a rotation of &minus;73&deg; with reported
rotation &minus;76&deg;;  the phenylhydrazone melts at 170-172&deg; with
169-172&deg; observed;  moreover a mix of the phenylhydrazone of pure
fucose and the phenylhydrazone of the isolated syrup was 169-171&deg;.
The fractions of Gal and GalNAc compared against the standards in
Gal oxidase reaction system.
<p>
Fraction VI passed through Dowex-1 chloride and is eluted from charcoal-Celite
with 5% EtOH.  In three paper chromatographic solvents and in borate
electrophoresis it moves with N-acetylgalactosaminitol.  Material crystallized
from ethanol melts at 174-175&deg;, while pure GalNAc reduced with
sodium borohydride to produce N-acetylgalactosaminitol produced a crystal
melting at 174-176&deg;.  A mixture of the Frac VI sugar and the reduced
standard also had the same melting point.  Both Frac VI and the standard
are treated with periodate, then borohydride, and these products have
<i>R</i><sub>f</sub> values identical with <i>N</i>-acetylserinol in three
solvent systems.
<p>
Frac V oligosaccharide is mildly acid hydrolyzed and liberates 89% of the
fucose, monitored by increases in reducing substances and no detectable
release of galactose or <i>N</i>-acetylgalactosaminitol.  Charcoal-Celite
separated fucose from galactosyl&rarr;N-acetylgalactosaminitol.  The latter
has a rotation of &minus;48&deg;, and is destroyed by &beta;-galactosidase.
The trisaccharide with a high negative rotation (&minus;115&deg;) suggests
fucose is &alpha;-linked.  The trisaccharide is active to galactose oxidase
and methylation gives 3,4,6-trimethylgalactose, suggesting that the C2
of galactose is substituted (methylations are done with 5-20 &micro;mol
oligosaccharide dried in a PTFE-lined screw cap to which 0.5 ml DMF,
0.5 ml DMSO, 250 mg BaO, 100 mg Ba(OH)<sub>2</sub>, and 0.25 ml
CH<sub>3</sub>I is added along with glass beads and the contents shaken
vigorously 24 h at RT.  BaO and CH<sub>3</sub>I are added again and shaking
done another 24 h.  Methylated sugars are extracted to chloroform, and
the organic phase is dried.  Biogel P-2 chromatography further purifies,
and the anthrone is used to detect derivatives.  These are hydrolyzed
to methylated monosaccharides with 2 N H<sub>2</sub>SO<sub>4</sub> for
3 h, the hydrolysates passed over Dowex 1 carbonate form (to neutralize),
and amino sugars removed with Dowex 50-H<sup>+</sup>
<!-- ============================================================= -->
<p class="tags">
phosphoproteins
<p class="ref">
 <span class="authors">
Cutting JA, Roth TF
 </span>
 <span class="year">
(1973)
 </span>
 <span class="jname">
Anal. Biochem.
 </span>
 <span class="jvol">
54,
 </span>
 <span class="pages">
386-394.
 </span>
<p class="reftitl">
Staining of Phospho-Proteins on Acrylamide Gel Electropherograms
<p class="Para1">
Many phosphorylated proteins are found abundant in nature: caseins in mammalian
milk, phosvitins in eggs.  A specific stain is necessary.
<p>
0.6 &times; 10 cm tube 7% gels after Davis were prepared and cast.
Electrophoresis is done to 1 cm from the end with the tracer dye.
Chicken yolk phosvitin is purified using the Mecham and Olcott method using
freshly laid infertile eggs from White Leghorn hens.
<p>
The Fiske-SubbaRow reagent was first tried using dilute NaOH solutions;
use of 1 N NaOH, extended hydrolysis at 80&deg;, and as much as 200 &micro;g
phosvitin would normally show the presence of phosphate in solution.
The basic dye method for sensitivity enhancement of the ammonium molybdate
reagent was tried, using methyl green in place of malachite green.  Methyl
green was incompatible with TCA, so 10% sulfosalicylic acid is used in the fix.
(For details on the ammonium molybdate method, see Akamatsu, J. Biochem. 39: 203,
1952; Itawa and Ui, Clin Chim Acta 14: 361, 1966;  the methyl green recommendations
comes from Van Belle, Anal. Biochem. 33: 132, 1970.)
<p>
Gels with 54 &micro;g phosvitin were treated with 1 N NaOH at 80&deg; for 1 h
then treated with a mixture of 3 vol 0.2% methyl green in water and 1 vol
4.2% ammonium molybdate in 4.5 N HCl (a final concentration of 0.15% methyl
green, ~1% ammomium molybdate, and ~ 1.1 N HCl).  After 15 min, gels are
transferred to 10% SSA and show a broad green band where phosvitin should be.
If 50 mM CaCl<sub>2</sub> is included in 10% SSA prior to hydrolysis, a
calcium phosphate precipitate forms to prevent phosphate ion diffusion.
<p>
Because only staining is happening at the surface rather than throughout the
band.  To compensate, a nitrophospho-molybdo complex is produced by adding
ammonium molybdate in nitric acid.  Because methyl green is insoluble in all
mineral acids like HNO<sub>3</sub>, the dye is dissolved in 7% acetic acid
to complex it, and this produces a sharp well defined band.
<p>
The following procedure was developed.
<ol>
<li>Fix gels in 10% sulfosalicylic acid (SSA).  The time of fixation should be
sufficient to allow any free phosphate and low MW phospho compounds to diffuse
out of the gel.  In serum samples, this could take as long as 12 h with several
changes of fixative; the step can be omitted when no phosphate is present.
<li>Gels are transferred to fresh 10% SSA containing 0.5 M calcium chloride
	for 1 h or longer.  (Note several concentrations of calcium chloride were tested
	and 0.5 M was optimal.)
<li>Gels are rapidly rinsed in deionized water to remove surface presence of
	calcium chloride, which could cause irregular staining.
<li>The gel is placed in 0.5 N NaOH at 60&deg; for 30 min.  Note the gel shrinks
as hydrolysis ensues, a function of time and temperature of incubation and
of hydroxide concentration.  Different concentrations (0.1, 0.5, 1.0, and
2.0) of hydroxide, times (30, 60, 120 min), and temperatures (37, 60, 80&deg;)
were all tried.
<li>Rinse twice for 10 min each with 1% ammonium molybdate, letting the ions
penetrate gel before decreasing pH, which forms the phospho-molybdate complex.
If acidification occurs prior to molybdate complexation, the calcium phosphate
solubilization occurs with the result of broader bands and missing fine bands.
The second molybdate rinse is done to reduce the level of NaOH.
<li>Gels are transferred to 1% ammonium molybdate in 1 N nitric acid for 30 min.
<li>Transfer to a 0.5% methyl green in 7% acetic acid solution for 30 min.
<li>Destain with several changes of 10% SSA.
<li>Gels are best kept in 7% acetic acid.
</ol>
<p>
As little as 0.3 &micro;g phosvitin was detectable, and because it has 10% phosphate,
this means as little as 1 nmol phosphate was detected.
<!-- ============================================================= -->
<p class="tags">
protein folding
<p class="ref">
 <span class="authors">
Go N, Taketomi H
 </span>
 <span class="year">
(1978)
 </span>
 <span class="jname">
Proc. Natl. Acad. Sci. USA
 </span>
 <span class="jvol">
75,
 </span>
 <span class="pages">
559-563.
 </span>
<p class="reftitl">
Respective roles of short- and long-range interactions in protein folding
<p class="Para1">
Long-range interaction in protein folding  are important because large
protein fragments do not conserve the same conformation that they possess
when made part of a native structure of a larger or intact protein.  Thus
interfragment (long-range) interactions are important.  It is also observed
that denaturing transitions in globular proteins occur in an all-or-none
manner, whereas if short-range transitions were dominant, denaturation
would be seen as a gradient of states.  Consider a 2D square lattice in 
a computer.  The <q>protein molecule</q> is a self-avoiding chain polyer
of <i>N</i> units connected linearly by bonds with length same as lattice
constant of square lattice.  Conformations taken depend on bond angle,
and two forces govern long- and short-range interaction.
<p>
In the long-range interaction, system energy decreases by &epsilon; when
a preassigned pair of units occupies nearest-neighbor lattice points.
Assignment of pairs of interactable units defines specificity of long-range
interactions.  Consider polymer chain of 49 units linked linearly. Three
different specificities are assumed:  specificity A (strong limit), specificity
B (intermediate), and specificity C (weak limit).  For specificity A, there
are only 36 pairs of units occurring in nearest-neighbor lattice points
in conformation shown in Fig 1.  For example, unit 1 has nearest neighbor
units 4, 6, and 8, so interactions 1-4, 1-6, 1-8 are specificity A (strong).
With all units determined, there are 36 strong interactions, so the long-range
interaction energy is &minus;36&epsilon;.  Other conformations grossly 
different will have interaction energy much closer to zero.  With specificity
C all 552 pairs of units that can be geometrically occupy nearest-neighbor
lattice points are assigned.  Thus all conformations compactly folded into
a 7&times;7 square have same lowest energy, &minus;36&epsilon;.  And no
particular conformation will be chosen as a native structure.  With 
specificity B, randomly selected 148 pairs as well as 36 pairs that are 
assigned with specificity A, are assigned; as a result 184 pairs are assigned.
As an example, unit 1 is interactable with units 4, 6, 8, 14, 22, and 28.
184 is one-third of 552 pairs of specificity C, so most of the conformations
are compactly folded into a 7&times;7 square are expected to have interaction
energy of about &minus;12&epsilon; (1/3 of &minus;36&epsilon;).  Because 
pairs of specificity A are retained for specificity B, the native structure 
at low temperature is expected for specificity B as well as A.
<p>
For specificity A, reversible all-orn-none type transition between native
and denatured are observed.  With specificity C, compact (globular) but
not specific conformations are stable at low temperatures.  The globule-coil
transition is reversible but of the diffuse type, as in helix-coil transition.
Long-range interaction specificity is essential for all-or-none character
of the transition.  Specificity B is regarded as the most realistic of the
three, stable at low temp and a transition to denatured state occurs quickly, 
again being all-or-none.  In the cited report, transition was not reversible
within time range of computer experiment, so the present paper assumes specificity
B is a long-range interaction.
<p>
For short-range interaction the energy term is function of a <q>bond angle</q>
of each unit.  The bond angle is teh angle between two bond vectors, one
pointing from (<i>i</i> &minus; 1)th unit to the <i>i</i>th unit, and the other
from <i>i</i>th unit to the (<i>i</i> + 1)th unit.  Bond angle can thus be
90&deg;, 180&deg;, and 270&deg; in the square conformation.  The bond energy
of the <i>i</i>th unit is lower by &epsilon;&prime; than two other possible
cases of the value of the angle (<i>i</i> = 2,3,...,48).  The Monte Carlo
method is used.  Four different ratios of values &epsilon; and &epsilon;&prime;:
(&epsilon;, &epsilon;&prime;) = 
(0.75&epsilon;<sub>0</sub>,0.25&epsilon;<sub>0</sub>), 
(0.5&epsilon;<sub>0</sub>,0.5&epsilon;<sub>0</sub>),
(0.25&epsilon;<sub>0</sub>,0.75&epsilon;<sub>0</sub>), and (0,0.25&epsilon;<sub>0</sub>).
&epsilon;<sub>0</sub> is the interaction energy, which will be used to defined the 
dimensionless temperature <i>T</i>* = <i>kT/&epsilon;</i><sub>0</sub>.
The case of (&epsilon;,&epsilon;&prime;) = (&epsilon;<sub>0</sub>,0) is same
as case of specificity B in previous paper.
<p>
For each ratio of &epsilon; and &epsilon;&prime;, computer analysis of
conformational changes were done for different <i>T*</i> values to
learn the transition temperature <i>T*</i><sub>m</sub>.  Two different
initial conformations are chosen at each temperature, the native and
one that is 49 units of a straight line (representing denatured).
After reaching <i>T*</i><sub>m</sub>, more computer analysis is done at
<i>T*</i> = <i>T*</i><sub>m</sub>.  Four figures show the plotted results
of the analysis, for the four (&epsilon;,&epsilon;&prime;).  On the abscissa
is a trial number, which is approximately proportional to physical time.
The ordinate value reports the degree of the order of the system, each unit
(&minus;&epsilon;<sub>0</sub>) representing the interaction energy.  A dashed
line is drawn in the plot to indicate expected count  <i>m</i> of short-range
interaction energies at the high temp limit.  At any finite temperatures 
average values of the count of short-range interaction energies are higher
than these values.  At low temperature <i>T*</i> = 0.3 for (&epsilon;, &epsilon;&prime;) = 
(0.75&epsilon;<sub>0</sub>,0.25&epsilon;<sub>0</sub>), the folding into
native conformation is not found within 150,000 trials; the lattice polymer
is trapped in a local minimum with about <i>m</i> = 25 and cannot get out
of it.  For (&epsilon;, &epsilon;&prime;) = 
(0.5&epsilon;<sub>0</sub>,0.5&epsilon;<sub>0</sub>), the folding into native
conformation occurs at trial 60,000.  For (&epsilon;, &epsilon;&prime;) = 
(0.25&epsilon;<sub>0</sub>,0.75&epsilon;<sub>0</sub>), folding into native
occurs quickly within 20,000 trials.  And for For (&epsilon;, &epsilon;&prime;) = 
(0,&epsilon;<sub>0</sub>), it is even more rapid.
<p>
At temps close to <i>T*</i><sub>m</sub> both folding and unfolding occur
as can be seen in one plot.  The transitions are rare, indicating an all-or-none
character in transition.  With (&epsilon;, &epsilon;&prime;) = 
(0.5&epsilon;<sub>0</sub>,0.5&epsilon;<sub>0</sub>), transitions occur more
frequently and with (&epsilon;, &epsilon;&prime;) = 
(0.25&epsilon;<sub>0</sub>,0.75&epsilon;<sub>0</sub>), they are so frequent
that populations at intermediate states are as significant as those of completely
folded or unfolded conformations.  With (&epsilon;,&epsilon;&prime;) = 
(0,&epsilon;<sub>0</sub>) there are no long-range interactions except for
self-avoidance of the chain, and lattice value fluctuates around the average
value of <i>m</i> = 34.
<p>
These results all shows that folding to native form is possible but could
not happen when only long-range interactions with specificity B are assumed.
Short-range interactions are required to accelerate the process.  Giving
greater weight to short-range accelerated folding.  Another effect of the
short range is to make the transition <q>diffuse</q>:  in fact when only
short-range interactions are involved, the transition is not one where there
is clear line between native and denatured.  Thus long-range interactions are
essential for the all-or-none type transitions, which occur as fact.
<!-- ============================================================= -->
<p class="tags">
mass spectrometry detectors
<p class="ref">
 <span class="authors">
Wiza JL
 </span>
 <span class="year">
(1979)
 </span>
 <span class="jname">
Nuclear Instr. Meth.
 </span>
 <span class="jvol">
162,
 </span>
 <span class="pages">
587-601.
 </span>
<p class="reftitl">
Microchannel Plate Detectors
<p class="Para1">
An MCP is an array of about 104-107 miniature electron multipliers oriented
parallel to one another.  Channel diam are 10-100 &micro;m and the
length-to-diameter ratio (&alpha;) is about 40-100.  Axes of the channle
are normal or slightly biased (8&deg; to surface).  The skeleton is lead
glass treated to optimize secondardy emission.  Each channel is a continuous
dynode acting as its own dynode resistor chain.  A nichrome or inconel
metalling coating on the front and back surfaces is connected to input
and output electrodes.  Total resistance is about 10<sup>9</sup> &Omega;
between electrodes.  Gain factors of 10<sup>4</sup>-10<sup>7</sup>
are achieved in ultra time resolution of &lt; 100 ps.  Spatial resolution
is based on channel dimensions (diam. typically 12 &micro;m) and
interchannel spacings (15 &micro;m center-to-center typically).
<p>
Detection of charged particles and photons is possible (X-ray and EUV for
photons).  Farnsworth (1930) provided the idea for this system but
it was not until the 1960s until the first prototypes emerged.  This followed
with the development of glass fiber drawing technology with which to make
microchannels.  Multiple fibers are bonded together to construct the plate
array.
<p>
Electron multiplier theory is well developed.  A consideration is given
to the angle of the channel to the normal.  Consider a straight channel
(exactly normal).  The incident electron produced &delta; secondary electrons,
and each of these produces &delta; secondaries.  Thus the gain is
G = &delta;  The gain can be related as follows:
<table class="eqnserif" style="text-align:center;">
<tr><td rowspan="2"><span style="font-size:200%;">(</span>
 <td class="numer"><i>AV</i>
 <td rowspan="2"><span style="font-size:200%;">)</span><sup
    style="vertical-align:2em;">4<i>V</i>&alpha;<sub>2</sub>/<i>V</i></sup>
<tr><td>2<i>&alpha;V</i><sub>0</sub><i>l</i> / 2
</table>
<p>
where <i>V</i> is total channel voltage, <i>V</i><sub>0</sub> is initial
energy of emitted secondary electron (~1 eV), &alpha; is the
length-to-diameter ratio, and <i>A</i> is a proportionality constant
such that &delta; = <i>AV<sub>c</sub></i><sup>&frac12;</sup>.
<i>V<sub>c</sub></i> is electron collision energy in eV and
<i>A</i> ~ 0.2.  With increases in <i>V</i>, the secondary electron
yield &delta; increases, since each collision occurs with higher
<i>V<sub>c</sub></i>.  Also the count of collisions must decrease so that
there is an extremum in the <i>G</i> vs. <i>V</i> characteristic.
There is also an extremum in &alpha;: for <i>V</i> = 1000 V,
<i>G</i><sub>M</sub> = 1635 and &alpha;<sub>M</sub> = 60.  According
to Guest &alpha;<sub>M</sub> = <i>V</i> / 22 = 45.  In any case, a range
of 40-60 is suitable for image intensification.  The charge height
pulse distribution shows that the statistics of the channel multiplication
process approximate a negative exponential (plot log counts vs. channel
number for a straight line).  As for energies of emergent electrons
with &alpha;=60 and <i>V</i>=1000 V, median energy is 32.5 eV with many
having in excess of 100 eV.  Increases in the gain are likely to cause
positive ions in the high charge density region, typical when residual
gas molecules are present or when desorbed from channel wall (ion feedback).
<p>
Suppression of ion feedback allows higher gains and the charge pulse
height distribution changes from a negative exponential to quasi-Gaussian
with full width half maximum of 10<sup>8</sup> or more (compared to
10<sup>3</sup>-10<sup>5</sup> for the straight channel with no ion suppression).
This shows space charge saturation at channel output.  A state of
dynamic equilibrium is reached as yields are not multipiled (&delta; ~ 1)
when kinetic energy of electrons is reduced by wall interaction.
Channel gain is thus limited by geometry and overall voltage:  maximum
gain from space-charge saturated channel for fixed <i>V</i> and &alpha;
is proportional to channel diameter.  While a 1 mm channel saturates
at 10<sup>8</sup>, and 10 &micro;m one saturates at 10<sup>6</sup>.
That MCPs with typical 25 &micro;m pores have gains slightly above
10<sup>6</sup> verifies theory here.  Pulse height distributions that
have a negative exponential do not permit the use of a channel for
counting since they are sensitive to shifts in gain.
<p>
Ion feedback suppression can be achieved simply by curving the channel.
The chevron is typically used, formed from the use of two juxtaposed
plates.  The bias angle is either 8&deg;/8&deg; or 0&deg;/15&deg;.,
large enough to inhibit positive ion formation at end of the channel.
The plates themselves are separated by 50-150 &micro;m and each has
its own voltage for gains of 10<sup>4</sup>.  Gains of 10<sup>7</sup>
or better are achieved with the chevron because the number of excited
channels increases between plates 1 and 2.
<p>
Other attempts to reduce ion feedback include use of angled electrostatic
fields created by channel walls with insulating strips.  These strips
become charged however with a time-dependent reduction in gain.
<!-- ============================================================= -->
<p class="tags">
colorimmetric protein assay
<p class="ref">
 <span class="authors">
Hazra AK, Chock SP, Albers RW
 </span>
 <span class="year">
(1984)
 </span>
 <span class="jname">
Anal. Biochem.
 </span>
 <span class="jvol">
137,
 </span>
 <span class="pages">
437-443.
 </span>
<p class="reftitl">
Protein Determination with Trinitrobenzene Sulfoate, A Method Relatively
Independent of Amino Acid Composition
<p class="Para1">
Fluorescence and spectrophotometric techniques can be susceptible to variability
in fluorescence yields and stability of amino acid derivatives.  Although
a precision ninhydrin continuous flow method has been reported, ninhydrin
is sensitive to oxygen and can produce the same molar absorbance with ammonia
and an amino acid.  2,4,6-trinitrobenzene sulfonate (TNBS) is well known for
primary determination of primary amines and is used in protein determination
here.  Unlike other methods, the one used in protein hydrosylates here is
independent of amino acid composition and uses a method with short hydrolysis
time.
<p>
For this method the Pierce Chemical amino acids standard kit 22 and
Sequanal-grade constant boiling HCl.  Fatty acid-free BSA and other proteins were
used here.  Grade I and II TNBS from Sigma is used;  grade II is yellowish in
color becomes white with crystallization.  Recrystallization is done with
1-25 g TNBS dissolved in one part water (w/w) by heating.  HCl (1.18 spec grav
1.18) is added to about 2 M.  Crystalline solid emerges with cooling and
is washed on filter with cold 1 M HCl and dried in dessicator.  The following
reagents are prepared:  (1) 0.25 M borate buffer, made with 9.54 g
Na<sub>2</sub>B<sub>4</sub>O<sub>7</sub> &bull; 10 H<sub>2</sub>O in 100 ml
(crystals might fall out of solution over time and are re-dissolved by warming
and stirring); (2) 2.5 M NaOH, 10 g NaOH in 100 ml; (3) 0.2 M KH<sub>2</sub>PO<sub>4</sub>,
made as 2.72 g per 100 ml. (4) TNBS in borate buffer:  35 mg recrystalled
TNBS is added to 1 ml of 0.25 M borate buffer just before use.  (5) Reagent A:
a mix of 20 vols 0.25 M borate to 1 vol 2.5 M NaOH;  (6) Reagent B: To 100 ml
of 0.2 potassium phosphate add 62.5 mg K<sub>2</sub>SO<sub>3</sub>, just
before use.  The following steps are carried out in the <b>standard procedure</b>:
<ol>
<li>Samples of protein from 25-100 &micro;g are placed in 6&times;50 mm
borosilicate glass and dried in centrifugal evaporator.
<li>Add 100 &micro;l constant boiling HCl
<li>Flush tube and seal with nitrogen
<li>Heat for 1 h at 145&deg; in aluminum heating block.
<li>Centrifuge for 1 min at 1000 rpm to settle the HCl from the side of the tube.
<li>Transfer a 3-5 &micro;l aliquot (equivalent to 3-5 &micro;g) from hydrolysis
tube to 10 &times; 75 mm plastic or borosilicate tube and dry down in centrifugal
vacuum evaporator.
<li>To each tube, add 100 &micro;l of Reagent A.
<li>Follow this quickly with 5 &micro;l of fresh TNBS reagent.
<li>After 10 min, add 900 &micro;l of Reagent B
<li>Read absorbance at 416 nm.  The yellow color is stable for several hours
at room temperature.
</ol>
<p>For the <b>micro</b> method, it similar to the standard method with the
following notes and points:
<ol>
<li>Using 1-10 &micro;g, add only 10 &micro;l HCl to the container.
<li>Do not transfer the hydrolysate to a separate tube:  rather dry the
hydrolysate down entirely to evaporate the HCl and other volatiles.
<li>Reagent A and TNBS reagent are added as above.
<li>Transfer the volume after yellow color development to 10 &times; 75 mm
tube containing 800 &micro;l reagent B.  Rinse hydrolysis tube with 100 &micro;l
Reagent B to get the balance and transfer to get the 1 ml volume.
</ol>
<p>In addition a fluorescamine method by Stowel et al was used.  Also the
Markwell et al modification of the Lowry protein method was used.
<!-- ============================================================= -->
<p class="tags">
proline-rich phosphoproteins
<p class="ref">
 <span class="authors">
Saitoh E, Isemura S, Sanada K
 </span>
 <span class="year">
(1985)
 </span>
 <span class="jname">
Arch. Oral Biol.
 </span>
 <span class="jvol">
30,
 </span>
 <span class="pages">
641-643.
 </span>
<p class="reftitl">
Inhibition of Calcium-Carbonate Precipitation by Human Salivary Proline-Rich
Phosphoproteins
<p class="Para1">
Human saliva contains proline-rich proteins (PRP) I, II, III, and IV and
also staherin.  These proteins inhibit calcium phosphate precipitation or
crystal growth, keeping it super-saturated.  In the early 70s, lyophilized
parotid saliva was dissolved in 50 mM TrisHCl pH 8.0 with 50 mM NaCl and
purified over DEAE-Sephadex A-25 and then purity checked on anionic polyacrylamide
gel electrophoresis.  To measure precipitation rates, 5 ml of 20 mM CaCl<sub>2</sub>
is mixed with 5 ml 20 mM sodium bicarbonate pH 8.7 and 50 &micro;l of PRP or
water control is added and absorbance at 570 nm measured (pH can also be
monitored).  Another test system mimicking saliva conditions can be used to
measure precipitation.  In PRP absence, absorbance is increased and pH
decreased rapidly.  With PRP IV added, absorbance increases are markedly
reduced.  The absorbance-vs-time curve was shifted to the right.  PRP III also
prevented nucleation and precipitation was in progress.
<!-- ============================================================= -->
<p class="tags">
codon usage
<p class="ref">
 <span class="authors">
Sharp PM, Li W-H
 </span>
 <span class="year">
(1987)
 </span>
 <span class="jname">
Nucl. Acid Res.
 </span>
 <span class="jvol">
15,
 </span>
 <span class="pages">
1281-1295.
 </span>
<p class="reftitl">
The codon adaptation index - a measure of directional synonymous
codon usage bias, and its potential applications
<p class="Para1">
There is a clear bias in the use of codons where multiple codons can
specify a single amino acid, indicating no randomness.  McLachlan
had quantified this with a <i>codon preference bias</i>.  Sharp et al.
proposed using the chi square value as a measure of the deviation from
randomness and scaling that to gene length (total number of codons)
to permit comparisons between genes.
<p>
It is clear that any nonrandom pattern of codon preference in any
particular gene is determined by natural selection and mutation.
Selection pressure comes from translational efficiency.  Where expression
of genes is at a high level, there appears to be the greatest synonymous
use of codons.  Differences in codon usage that are detected in highly
expressed genes are likely attributable to differences in translational
efficiency.  We construct a table of relative synonymous codon usage
(RSCU) values specific to highly expressed genes:  an RSCU value for
a codon is the observed frequency of that codon divided by the frequency
expected given equal usage:
<table class="eqnserif" style="text-align:center;">
<tr><td rowspan="2" style="vertical-align:top;padding:1em 1em 0 0;"><i>RSCU<sub>ij</sub></i> =
 <td class="numer"><i>x<sub>ij</sub></i>
<tr><td>
 <table class="eqnserif" style="margin:0;font-size:90%;">
 <tr><td class="numer">1
  <td rowspan="2" style="font-size:130%;padding-left:0.5em;">&Sigma;<span
    style="vertical-align:1.5em;font-size:60%;margin-left:-0.75em"><i>n<sub>i</sub></i></span>
 <span style="font-size:60%;vertical-align:-1em;margin-left:-1.25em"><i>j</i>=1</span>
  <i>x<sub>ij</sub></i>
  <tr><td><i>n<sub>i</sub></i>
 </table>
</table>
<p>
where <i>x<sub>ij</sub></i> is the count of occurrences of the
<i>j</i><sup>th</sup> codon for the <i>i</i><sup>th</sup> amino acid,
and <i>n<sub>i</sub></i> is the count (from 1 to 6) of the alternative
codons of the <i>i</i><sup>th</sup> amino acid.  The <i><b>relative
adaptivness of a codon</b></i>, <i>w<sub>ij</sub></i>, is the
frequency of use of a codon compared to the frequency of the optimal
codon for an amino acid:
<p style="text-align:center;" class="eqnserif">
 <i>w<sub>ij</sub> = RSCU<sub>ij</sub></i> / <i>RSCU</i><sub>imax</sub>
  &minus; <i>x<sub>ij</sub></i> / <i>x</i><sub>imax</sub>
<p>
where <i>RSCU</i><sub>imax</sub> and <i>x</i><sub>imax</sub> are the
<i>RSCU</i> and <i>x</i> are values for the most frequently used codons
for the <i>i</i><sup>th</sup> amino acid.
<p>
27 genes from <i>E. coli</i> were identified as highly expressed,
including 17 ribosomal proteins, four outer membranes, and four elongation
factors.  24 genes in yeast identified as highly expressed, including
16 ribosomal, one EF, and seven loci of very abundant enzymes.
<p>
The Codon Adaptation Index (CAI) for a gene is then determined to be
the geometric mean of the RSCU values for each codon divided by the maximum
possible CAI for a gene of the same amino acid composition:
<p style="text-align:center;" class="eqnserif">
  CAI = CAI<sub>obs</sub> / CAI<sub>max</sub>
<p>
where
<p style="text-align:center;" class="eqnserif">
  CAI<sub>obs</sub> =
 <span style="font-size:200%;">(</span><span style="font-size:125%;">&Pi;</span><span
    style="vertical-align:2em;font-size:60%;margin-left:-1em"><i>L</i></span>
 <span style="font-size:60%;vertical-align:-1.5em;margin-left:-1.5em"><i>k</i>=1</span>
  <i>RSCU<sub>k</sub></i><span
  style="font-size:200%;">)</span><sup style="vertical-align:1em;">1/<i>L</i></sup>
<p style="text-align:center;" class="eqnserif">
  CAI<sub>max</sub> =
 <span style="font-size:200%;">(</span><span style="font-size:125%;">&Pi;</span><span
    style="vertical-align:2em;font-size:60%;margin-left:-1em"><i>L</i></span>
 <span style="font-size:60%;vertical-align:-1.5em;margin-left:-1.5em"><i>k</i>=1</span>
  <i>RSCU<sub>k<sub>max</sub></sub></i><span
  style="font-size:200%;">)</span><sup style="vertical-align:1em;">1/<i>L</i></sup>
<p>
where <i>RSCU<sub>k</sub></i> is the RSCU value for the <i>k<sup>th</sup></i>
codon for the gen, <i>RSCU<sub>k<sub>max</sub></sub></i> is the maxium
RSCU value for the amino acid encoded by the <i>k<sup>th</sup></i> codon
in the gene, and <i>L</i> is the number of codons in the gene.
The CAI for a codon never used in a reference set becomes zero if it
appears in any other gene, and so <i>x<sub>ij</sub></i> is set to 0.5
to overcome this problem.  The count of AUG and UGG codons are subtracted
from <i>L</i> as RSCU values for these codons are fixed at 1.0 and do
not factor into CAI.
<p>
An example of calculation can be seen with the <i>rpsU</i> from <i>E. coli</i>.
There are 70 codons excluding the initiation codon, and sequence starts:
<p style="text-align:center;font:normal 100% 'Courier New',Courier,monospace;">
.CCG.GTA.ATT.AAA.GTA. . . . .
<p>
From a table of reference values, the following equations apply:
<p style="text-align:center;font:normal 100% 'Courier New',Courier,monospace;">
CAI<sub>obs</sub> = (3.288 &times; 1.111 &times; 0.466 &times; 1.596
  &times; 1.111 &times; .....)<sup>1/70</sup>
<br>
CAI<sub>max</sub> = (3.288 &times; 2.244 &times; 2.525 &times; 1.596
  &times; 2.244 &times; .....)<sup>1/70</sup>
<p>
Note that the equation for CAI = CAI<sub>obs</sub>/CAI<sub>max</sub> is
equivalent to
<p style="text-align:center;" class="eqnserif">
  CAI =
 <span style="font-size:200%;">(</span><span style="font-size:125%;">&Pi;</span><span
    style="vertical-align:2em;font-size:60%;margin-left:-1em"><i>L</i></span>
 <span style="font-size:60%;vertical-align:-1.5em;margin-left:-1.5em"><i>k</i>=1</span>
  <i>w<sub>k</sub></i><span
  style="font-size:200%;">)</span><sup style="vertical-align:1em;">1/<i>L</i></sup>
<p>
where <i>w<sub>k</sub></i> is the <i>w</i> value for the <i>k<sup>th</sup></i>
codon in the gene.  Hence:
<p style="text-align:center;font:normal 100% 'Courier New',Courier,monospace;">
CAI = (1.00 &times; 0.495 &times; 0.185 &times; 1.000 &times;
     0.495 &times; .....)<sup>1/70</sup>
<p>
A more computable value (to avoid real number underflow) is
<p style="text-align:center;font:normal 100% 'Courier New',Courier,monospace;">
CAI = exp [ (1/<i>L</i>) <span style="font-size:150%;"> &Sigma;<span
    style="vertical-align:1.5em;font-size:60%;margin-left:-0.75em"><i>L</i></span>
 <span style="font-size:60%;vertical-align:-1em;margin-left:-2em"><i>j</i>=1</span>
 </span> ln <i>w<sub>k</sub></i> ]
<p>
There are parallels in CAI values and expression levels.  Ribosomal genes
are highly expressed and have high CAI values.  The yeast ribosomal gene
S33 is the only one with CAI &lt; 0.6, but it is a very short gene
(<i>L</i> = 65).  Operons are a good example:  expression levels for
these operons are <i>rpsU</i> &gt;&gt; <i>rpoD</i> &gt;&gt; <i>dnaG</i>
and the CAI values for genes are 0.726, 0.582, and 0.271, respectively.
This is especially seen for the genes of the <i>unc</i> operon, in which
8 of the 9 genes code for subunits of the F<sub>0</sub> and F<sub>1</sub>
assemblies of the H<sup>+</sup>-ATPase complex, and the CAI values rank
in the same order as the levels of expression.  One complex gene has a
very low CAI, but it is a very short gene.
<p>
Experimental evidence exists that codon usage affects expression level.
AGG codon use has been shown to control translation rate in <i>E coli</i>.
For a heterologous gene to achieve maximal expression in any host,
codon usage in that host is important.  CAI is only an approximate indicator
of codon suitability in synthetic selection.
<p>
Viral genes have no difficulty expressing themselves as foreign genes in
hosts.
<!-- ============================================================= -->
<p class="tags">
laser desorption/ionization multiphoton ionization, mass spectrometry
<p class="ref">
 <span class="authors">
Grotemeyer J, Schlag EW
 </span>
 <span class="year">
(1988)
 </span>
 <span class="jname">
Organic Mass Spectrom.
 </span>
 <span class="jvol">
23,
 </span>
 <span class="pages">
388-396.
 </span>
<p class="reftitl">
Peptides Investigated by Laser Desorption-Multiphoton Ionization Mass
Spectrometry
<p class="Para1">
Multiphoton ionization (MUPI) is a means of producing ionized molecules
for the MS, useful in chemical analysis and ion chemistry.  Resonance-enhanced
multiphoton ionization (REMPI) is but a form of this useful in soft ionization.
Laser desorption is used as the means of producing gas phase molecules using
infrared (IR) photons.  A pulsed IR laser produces rapid heating before
decomposition can occur.  Because more neutrals than ions are formed in the
process (by 10,000:1), a better ionization technique is used.  Peptides are
molecules of interest in this case.  MUPI combines the possibility of soft
ionization and tuning the fragmentation intensity.  Gas phase neutrals
absorb two photons with the energy rather small (wavelength-dependent),
optimized to produce an ion with mininal fragmentation.  Other adjustments
include laser intensity; power density increases clearly affect fragmentation.
<p>
The mass spectrometer is a laser desorption, laser ionization,  and reflection
ToF MS.  Samples are dissolved in solvent (typically MeOH) and deposited
on a probe tip.  The probe tip is mounted in vacuum after solvent evaporation
close to pulsed valve nozzle.  An LP30 CO<sub>2</sub> laser provides IR photons,
and desorbed neutrals are injected into supersonic beam of noble gas emitted
from nozzle of pulsed valve.  The beam passes through a Lambda Physics
FL 2001 pulsed, frequency-doubled dye laser pumped by a Lambda Physics EMG 102.
The right wavelength is selected to excite appropriate to intermediate level.
Ionization efficiency is 0.1-10%, where impact ionization methods would have
a lower efficiency of 0.01-0.5%.  The wavelength used was 2681 &Aring; close to
0-0 transition of &pi;-electrons of toluene.
<p>
Mass spectra were obtained for pure dipeptides Leu-Phe and Gln-Phe in 1:1
mixture.  Most intense signals aer <i>m/z</i> 278 and 293 for the parent ions.
Ionization is formed by absorption of two photons, and fragmentation induced
by third photon, something that can be optimized.  With fragmentation there
is a [M-H<sub>2</sub>O]<sup>+</sup> at <i>m/z</i> 240 formed from Leu-Phe,
with additional signals at m/z 87 and 86.
<p>
[more to be read on this paper]
<!-- ============================================================= -->
<p class="tags">
capillary zone electrophoresis (CZE)
<p class="ref">
 <span class="authors">
Huang X, Gordon MJ, Zare RN
 </span>
 <span class="year">
(1988)
 </span>
 <span class="jname">
Anal. Chem.
 </span>
 <span class="jvol">
60,
 </span>
 <span class="pages">
1837-1838.
 </span>
<p class="reftitl">
Current-Monitoring Method for Measuring the Electroosmotic Flow Rate
in Capillary Zone Electrophoresis
<p class="Para1">
One method for measuring EOF is by the use of uncharged (electrically
neutral) marker solute that has no interaction with the silica capillary
wall and which can be detected with the current systems.  Another is to
measure the mass of electrolyte moved from inlet to outlet over a timed
interval, but this requires a digital balance and means to prevent evaporation.
Another means is to record the time history of current in the CZE.
A circuit diagram below shows how to set up the measuring system:
<pre style="line-height:0.85em;">
                ___________
               |           |
    +----------| HV Supply |-----------------------------+
    |          |___________|                             |
    |                                                    |
    |                                                    |
    | Pt                                                 |
    |wire                                                |
    |        ________________________  Pt                |
    |       | ______________________ | wire   10k ohm    |
    |______ ||     capillary        || ______/\/\/\/\____+
           |||                      |||     |resistor|   |
         ! |||!                    !|||!    |        | -----
         ! |""!                    !""|!    |        |  ---
         !____!                    !___!   _|________|_  - ground
                                          |signal/chart|
           R1                        R2   |  recorder  |
                                          |____________|
</pre>
<p>
Two reservoirs R1 and R2 hold the capillary.  Polarity must be chosen
so that the EOF is from R1 to R2:  this usually means that the R1
electrode is positive because uncoated caps cause flow toward the
cathode.  The capillary is filled with a buffer at concentration <i>C</i>
and this is placed in R2.  R1 is filled with the same buffer but at
concentration <i>C&prime;</i>.  During CZE, the tube filled with <i>C</i>
will change to <i>C&prime;</i>, and so will the electrical resistance
in the tube.  The current <i>I</i> can be followed.  A resistor of
10 k&Omega; (2 W) is inserted between the R2 electrode and the ground:
the use of the 10 k&Omega; resistor means a 1 &micro;A current change
produces a 10 mV potential drop across the resistor.  A signal or
chart recorder is connected across the resistor.  A difference between
<i>C</i> and <i>C&prime;</i> need not be great (as little as 5%).
<i>C</i> is 20 mM sodium phosphate pH 7 and <i>C&prime;</i> is 0.95<i>C</i>.
The power supply can deliver 19-21 kV and the capillary is 75 &micro;m
&times; 63 cm, which creates a current of 40 &micro;A.
<p>
The results show that the current falls with time as the electrolyte
<i>C</i> is displaced with <i>C&prime;</i>.  If EOF is reversed, then
there is no current change.  The time interval &Delta;<i>t</i> is the
time required for EOF from R1 to fill the capillary.  The EOF,
<i>v</i><sub>eo</sub> is essentially <i>L</i> / &Delta;<i>t</i>, the
capillary length divided by the time.
<p>
Is EOF a function of electrolyte concentration (or the difference between
the reservoirs?  In a first experiment R1 was given <i>C</i> and R2 and
the capillary <i>C&prime;</i>.  The current trace was not different than
before the concentrations were reversed.  EOF could also be assessed by
evaluating conductivity changes as a function of time:  there was no
difference between conductivity changes and current changes.  Pyridine
was also placed as a neutral marker and the UV followed.  With a field
strength of 300 V/cm and the R1, R2 and capillary filled with identical
buffer the appearance of the UV peak indicated an EOF of 0.15 cm/s.
The same result was found for different concentrations placed in R1 and R2
and in the capillary as indicated above when the neutral marker was used
but was 0.015 cm/s for the current-monitoring method.
<!-- ============================================================= -->
<p class="tags">
laser desorption/ionization mass spectrometry
<p class="ref">
 <span class="authors">
Karas M, Hillenkamp F
 </span>
 <span class="year">
(1988)
 </span>
 <span class="jname">
Anal. Chem.
 </span>
 <span class="jvol">
60,
 </span>
 <span class="pages">
2299-2301.
 </span>
<p class="reftitl">
Laser Desorption Ionization of Proteins with Molecular Masses Exceeding
10,000 Daltons
<p class="Para1">
<b>Correspondence</b>.  For bioorganic molecules greater than 10,000 Da
plasma desorption mass spectrometry (PDMS) has been the usual.  Tanaka
et al showed laser desorption of proteins up to a mass of 34,000 Da
of lysozyme.  A pulsed nitrogen laser with metal powder matrix was used,
finely dispersed in glycerol.
<p>
In this process reflector-type ToF MS with Q-switched quadrupole Ng-Yag
laser (10 ns pulse, 266 nm wavelength, with 10-50 &micro;m spot diameter.
Samples are dissolved in water at 10 &micro;M.  1 &micro;l (10 pmol) of
samples is mixed with 1 &micro;l 1 mM nictonic acid.  The sample covered
about 5 mm<sup>2</sup> on a metallic substrate.  Laser shots are aimed at
one spot 20-50 times (collected over 2-5 min, at 10 scans/min or one per
6 seconds, with 1 Hz firing rate).  Data collection was also limited
by the PC data acquisition rate.  A LAMMA 1000 was used with microprobe
for inorganic and organic in the low mass range.  Ion kinetic energy is
3 keV and postacceleration potential is 9000 V.  For lysozyme (MW 14306)
the velocity at the conversion electrode is 11000 m/s and lower for ions
of higher mass;  a minimum 17,000 m/s in PDMS work had been found necessary
to detect insulin.  A Biomation transient recorder with acquisition memory
of 2048 channels (very low time resolution in high mass range) was used,
which limits mass calibration accuracy.  Calibration was with sodium and
matrix signals in the low mass range.
<p>
Spectra of hen egg white lysoszyme, bovine milk &beta;-lactoglobulin A,
and bovine albumin were obtained.  The effective mass range was controlled
by an analog-to-digital time resolution of 200 ns/channel from <i>m/z</i>
0 to 16,000.  For higher mass molecules, a time lag is imposed before
detection is turned on, or the detector is adjusted to reduce time resolution.
In addition to the molecular ion (M<sup>+</sup>), a dimer and a doubly
charged ion, as well as a 3M<sup>2+</sup> signal in the lysozyme spectrum,
and also found in the lactoglobulin spectrum.  Peak centroids are marked
on the <i>m/z</i> scale of the raw spectra.  For porcine trypsin a
500 ns/channel time resolution was used.
<p>
Results show that laser desorption will produce ions in a mass range
inaccessible to PDMS.  Multiply charged ions are observed in all spectra,
something not seen in LDI.  These are common in PDMS for high mass proteins,
indicating the stability of charge centers rather than anything in the
excitation mechanism.  The FWHM of ion peaks is larger than expected
for the mass resolution of 600 in the low mass range.  A resolution of
the molecular ion signal expected to show both protonated and cationized
species is not possible, but due partly to low time resolution of the
recorder (one channel of 200 ns is 15 Da in a 14000 Da protein).
<!-- ============================================================= -->
<p class="tags">
peptide, nuclear magnetic resonance (NMR), Fourier-transform infrared 
spectroscopy (FTIR), circular dichroism (CD)
<p class="ref">
 <span class="authors">
Fry DC, Byler DM, Susi H, Brown EM, Kuby SA, Mildvan AS
 </span>
 <span class="year">
(1988)
 </span>
 <span class="jname">
Biochemistry
 </span>
 <span class="jvol">
27,
 </span>
 <span class="pages">
3588-3598.
 </span>
<p class="reftitl">
Solution Structure of the 45-Residue MgATP-Binding Peptide of Adenylate
Kinase As Examined by 2-D NMR, FTIR, and CD Spectroscopy
<p class="Para1">
Peptide 1-45 is the synthetic N-terminal sequence of the rabbit muscle
adenylate kinase (catalyzes MgATP + AMP &hArr; MgADP + ADP) which binds the
MgATP substrate.  Proton NMR was used and the binding of MgATP was confirmed
using time-dependent intramolecular NOEs.  Cr(III)-ATP was used as a
paramagnetic T1 probe as well as intermolecular NOEs were used together to
show the interaction sites.  This information was used to place MgATP within
the solved crystal structure of the porcine enzyme.  Recent development of
2D-NMR methods allows solution structure determination of up to 100 amino
acid long polypeptides.  Peptide at 4-5 mM in 150 mM NaCl and 40 mM 2-mercaptoEtOH
in 90:10 water:D2O at pH 5.6 was made into a 400 &micro;l volume for NMR
work:  the peptide was made 1:1 with MgATP when it was present.  COSY and
NOESY were done at 500 MHz:  a 90&deg; pulse with sweep widths of 4386 Hz
in both dimensions, collecting 512 FIDs each with 2000 data points for COSY
or 256 FIDs with 1000 data points for NOESY.  For each FID there were 128
(COSY) or 208 (NOESY) transients plus two dummy transients with 230 ms (COSY)
and 120 ms (NOESY) per transient.  A 900 ms relaxation delay precedes each
transient.  H2O resonance was suppressed by direct irradiation at 10 dB
below 200 mW durin the relaxation delay, and at 50 dB below 200 mW during
mixing time in NOESY.  Mixing time of 400 ms in NOESY allows NOE development
and varied randomly &plusmn;5% to avoid COSY-type cross peaks.  The 2D data
were zero-filled to final size of 2000 &times; 2000 points for COSY and
1000 &times; 1000 for NOESY, the Fourier-transformed, using multiplication
by unshifted sine-bell functions in both dimensions.
<p>
Peptide was made up at 8.4 mM in the same NaCl-reducing buffer in a 50 &micro;l
volume.  Samples were made up in D<sub>2</sub>O using lyophilization twice.  Spectra
were collected at 29.5&plusmn;1&deg; using variable path cell with CaF<sub>2</sub>
windows with path of 75 &micro;m.  Interferometer:  Nicolet 7199 with water-cooled
Globar source and wide range MCT (Hg/Cd/Te) detector.  4000 interferograms
were co-added, phase-corrected, zero-filled one time, apodized with Happ-Genzel
function, and Fourier-transformed.  Nominal resolution was 2 cm<sup>-1</sup>.
CD spectra were acquired at 25 &plusmn; 1&deg; C using Jasco 41C spectropolarimeter
calibrated with <i>d</i>-10-camphorsulfonic acid.  0.5 mg/ml peptide in 0.15 M
NaCl was placed in 20 mm path cylindrical cell.  16 scans from 260-185 nm with
sensitivity of 0.005 deg/cm were obtained with 1 s time constant, 0.2 nm spectral
step, and 2 nm bandwidth.
<p>
<b>Results</b>.  Fig 1 shows 1 D NMR of the 1-45 peptide.  Fig 2 is the
COSY spectrum zoomed in on the aliphatic region.  Peak assignments were aided
by 1D spectrum particularly through a full pH range titration, and also by
1D and 2D spectra of peptide 31-45 reported by us earlier.  Table 1 shows
all the chemical shift assigments for every proton in the peptide.
A secondary structure map is shown from the results of NMR with and without
the MgATP.  NOESY spectra pfovide the information about distant bond interactions.
<p>
From FTIR, the amide I band shows 6-9 components, some detected only by curve
fitting.
<!-- ============================================================= -->
<p class="tags">
deglycosylation
<p class="ref">
 <span class="authors">
E. Horvath, A. M. Edwards, J. C. Bell, and P. E. Braun
 </span>
 <span class="year">
(1989)
 </span>
 <span class="jname">
J. Neurosci. Res.
 </span>
 <span class="jvol">
24,
 </span>
 <span class="pages">
398-401.
 </span>
<p class="reftitl">
Chemical Deglycosylation on a Micro-Scale of Membrane Glycoproteins With
Retention of Phosphoryl-Protein Linkages
<p class="Para1">
TFMS was obtained in 10 ml sealed ampules and cooled in dry ice before
opening.  Anisole was stored over molecular sieve at &minus;70&deg;.
Samples of protein (5-50 &micro;g) are precipitated with 15% TCA, then
acetone washed, then thoroughly dried in dessicator.  5 &micro;l anisole
and 45 &micro;l TFMS are added, samples nitrogen flushed, and tightly capped.
After vortexing they are left on ice 2 h.  Ice cold diethyl ether (800&micro;l)
is added, samples left on dry ice 2 min, then pyridine added dropwise (100 &micro;l).
<p>
<!-- ============================================================= -->
<p class="tags">
mass spectrometry
<p class="ref">
 <span class="authors">
Beavis RC, Chait BT
 </span>
 <span class="year">
(1990)
 </span>
 <span class="jname">
Proc. Natl. Acad. Sci. USA
 </span>
 <span class="jvol">
87,
 </span>
 <span class="pages">
6873-6877.
 </span>
<p class="reftitl">
Rapid, Sensitive Analysis of Protein Mixtures by Mass Spectrometry
<p class="Para1">
A mixture of commercially purified proteins was analyzed by a user-built
mass spectrometry system.  The laser is a frequency tripled output of
a Q-switched Lumonics HY400 Nd/Yt aluminum garnet laser (355 nm, 10 ns
pulse).  Product ions are accelerated in a 30 kV source into a 2 m tube
in a 30 &micro;Pa vacuum.
<!-- ============================================================= -->
<p class="tags">
small angle neutron scattering
<p class="ref">
 <span class="authors">
Haris PI, Chapman D, Harrison RA, Smith KF, Perkins SJ
 </span>
 <span class="year">
(1990)
 </span>
 <span class="jname">
Biochemistry
 </span>
 <span class="jvol">
29,
 </span>
 <span class="pages">
1377-1380.
 </span>
<p class="reftitl">
Conformational Transition Between Native and Reactive Center Cleaved Forms of
&alpha;<sub>1</sub>-Antitrypsin by Fourier Transform Infrared Spectroscropy and
Small-Angle Neutron Scattering
<p class="Para1">
Serpin family supermembers include many plasma proteinase inhibitors and present
reactive site as ideal substrate to target proteinases.  The reaction produces a
1:1 complex of proteinase and inhibitor with reactive site cleaved (split).
Crystal structures of the split form or &alpha;<sub>1</sub>-antitrypsin (a1-AT)
show significant conformational change with Met-358 and Ser-359 at the split
being 70 &Aring; apart.  FTIR provides information on secondary structure and
has advantage over CD in that it provides a baseline, the spectrum is clearly
resolved, and amide proton exchange rates can be monitored to show H-bonding
identifications.  Neutron scattering is more beneficial than X-ray scattering as
sample transmission in pure <sup>2</sup>H<sub>2</sub>O (2 mm path) is reduced
only 3% in 1.8 M phosphate relative to 50 mM phosphate., while for X-rays (1 mm
path) it is reduced 81% in 1.2 M phosphate.  Combined results offer sufficient
information to propose mechanism as to how split is stabilized.
<p>
a1-AT protein was dialyzed over 36 h in sodium phosphate ranging from 50 mM to
1.80 M in 100% <sup>2</sup>H<sub>2</sub>O at 6&deg; with four buffer changes. A
Perkin-Elmer 1750 FTIR with TGS detector was used to see spectra along with P-E
Model 7300 data station.  23-25 mg/ml a1-AT based on A<sub>280</sub> = 5.4
(relative to 1% solution in 1 cm path) was scanned 400 times (1 h accumulation)
at spectral rez = 4/cm with sample pathlength of 50 &micro;m (teflon spacer).
The Perkin-Elmer ENHANCE (Haris et al 1986)  was used for spectral deconvolution
and analogous to Kauppinen et al (1981) method using deconv. parameters &sigma;
= 8 and k = 2.25.  Second derivative calculated over 13/cm range.  <b>Neutron
scattering</b> is done at 20&deg; on small-angle scattering instrument D17 at
Institut Laue Langevin with &lambda; = 1.105 nm (wavelength spread
&Delta;&lambda;/&lambda; of 10%) and sample-detector distance of 3.396 m with
<i>Q</i> range of 0.07-0.60/nm (<i>Q</i> = 4&pi; sin <i>&theta;</i> / &lambda;;
2<i>&theta;</i> = scattering angle.  2 mm-thick Hellma cells are used for data
collection.  Control data for scattering of buffer dialysate, cadmium,
H<sub>2</sub>O, and empty Hellma quartz cell with measurement of sample and
buffer transmissions. Radius of gyration <i>R</i><sub>G</sub> and forward
scattering at zero scattering angle <i>I</i>(0) were determined from plot of
Gunnier equation: ln <i>I</i>(<i>Q</i>) = ln <i>I</i>(0) &minus;
<i>R</i><sub>G</sub><sup>2</sup> <i>Q</i><sup>2</sup>/3 in domain of <i>Q</i>
from 0.15-0.58 per nm except for native a1-AT above 1.2 M phosphate, where it
was 0.15-0.41 per nm.
<p>
<b>Results</b>.  FTIR spectra show the amide I band maxima for native and split
a1-AT at 1636 cm<sup>&minus;1</sup>.  Maximum at 1568-1570 cm<sup>&minus;1</sup>
is -COO<sup>&minus;1</sup> side chain feature, while 1516 cm<sup>&minus;1</sup>
generally is a Tyr side chain absorption.  Both samples show an amide I band
consistent with &beta;-sheet.  The split form contains additional features at
1650 and 1693 cm<sup>&minus;1</sup> seen in both high and low phosphate
concentrations (0.05, 0.50, 0.80, 1.25, 1.60, and 1.80 M);  these are revealed
in second derivative spectra.  Main &beta;-sheet is seen in 1633-1634
cm<sup>&minus;1</sup> seen in native and split.  Split form show more intense
&beta;-sheet and &alpha;-helix bands at 1694 and 1652 cm<sup>&minus;1</sup>,
with broader 1633 cm<sup>&minus;1</sup> band. Bands at 1650 and 1690
cm<sup>&minus;1</sup> have been attributed to &alpha;-helix and anti-parallel
&beta;-sheet, respectively.  For native and split a1-AT weaker features are
observed near 1646, 1662, and 1674-1682 cm<sup>&minus;1</sup>.  The feature at
1646 cm<sup>&minus;1</sup> is likely represenative of disordered structure.
Turn structures are likely at 1662 and 1674-1682 cm<sup>&minus;1</sup>, but
overlap from &beta;-sheet and 3<sub>10</sub> helices can occur.  Deconvolution
for resolution enhancement led to similar results.  These FTIR data show
significantly enhanced &alpha;-helical and &beta;-sheet content with reactive
site cleavage.  Curve fitting of the amide I band absorption changes is not
sufficiently accurate to do quantitative assessment of content differences.
Crystal structure of split a1-AT reveals 32% &alpha;-helix and 40% &beta;-sheet
and suggests that strand A4 in native form is withdrawn from &beta;-sheet. If
20-30 more H-bonds form in the split in a &beta;-sheet structure, this would
account for a greater strength at 1694 cm<sup>&minus;1</sup>.  Increased
&alpha;-helix would not be expected.  The crystal structure shows that helices A
and B flank strand B6, helices D, E, and F alternate with strands A2, A1 and A3,
and helices H and I flank strand A6. The infrared of the native form says either
helices are not formed or they are distorted along length while at same time
&beta;-sheet is incompletely formed.  But with splitting, there is a drive to
tighter bonding of both helices and sheets.
<p>
The amide II band at 1552 cm<sup>&minus;1</sup> is a check on the
<sup>1</sup>H-<sup>2</sup>H exchange of backbone NH protons.  Identical and
simultaenous dialysis in <sup>2</sup>H<sub>2</sub>O shows that the native is 15%
lower than split form.  Deconvoluted spectra show at 1550 cm<sup>&minus;1</sup>
visible for split a1-AT but not native, showing exchange was more complete for
native.  This would be so if the split form represents a more closed, stable
structure.  400 MHz <sup>1</sup>H-NMR confirms this on the same samples:
integration of spectra downfield of 195 aromatic proton signals show 11% of 377
NH protons  are not exchanged in native a1-AT and 19% of protons were not
exchanged in split form. Other NMR spectral differences include observation of
narrower line widths in split form and rearranged patterns in ring current
shifted methyl proton signals at high field end, both consistent with structural
and dynamic differences.
<p>
For small-angle neutron scattering, plots of Gunnier <i>R</i><sub>G</sub> and
<i>I</i>(0)/<i>cT</i><sub>s</sub> against phosphate concentration from 0.05 to
1.8 M were made, where <i>c</i> is also the variation in a1-AT from 3-12 mg/ml
and <i>T</i><sub>s</sub> is sample transmission.  Extrapolations to zero
<i>c</i> are made.  Up to 0.80 M phosphate, no differences can be seen between
native and split forms.  For hemoglobin, aspartate transcarbamylase, and
hexokinase, conformational changes after ligation are seen from Gunnier
analyses.  Folded tertiary structures of native and split a1-AT are essentially
similar, and an upper limit on structural changes in polypeptide backbone are
seen in FTIR, CD, and NMR.
<p>
High phosphate concentrations are used in the crystallization of a1-AT, and so
FTIR spectra are obtained at 1.25, 1.60, and 1.80 M phosphate.  No significant
diference at high concentrations and 0.05 and 0.50 M phosphate are seen.  But
neutron scattering data show that above 1 M phosphate the <i>I</i>(0)/<i>c</i>
increase to double the low phosphate concentration values (see plots) for native
form while unchanged in the split.  This shows that native a1-AT will dimerize
in 1.8 M phosphate.  The <i>R</i><sub>G</sub> rise in parallel for the native.
Hence the presence of many amino acids on the surface of the native not seen in
the split form is demonstrated.
<!-- ============================================================= -->
<p class="tags">
FAB-MS, HLPC
<p class="ref">
 <span class="authors">
van Breemen RB, Canjura FL, Schwartz SJ
 </span>
 <span class="year">
(1991)
 </span>
 <span class="jname">
J. Chromatog.
 </span>
 <span class="jvol">
542,
 </span>
 <span class="pages">
373-383.
 </span>
<p class="reftitl">
High Performance Liquid Chromatography Continuous Flow Fast Atom
Bombardment Mass Spectrometry of Chlorophyll Derivatives
<p class="Para1">
Mass spectra have been acquired on chlorophylls with a variety of
ionization techniques.  Laser desorption, field desorption, plasma
desorption, fast atom bombardment, and <q>in-beam</q> (desorption)
electron impact.  FAB is often preferred for MS/MS studies because
of a continuous stream of ions produced, and it is the only one
coupled to HPLC separations.  In FAB, a solution of low volatility
(high glycerol component) is bombarded with Xe or Ar atoms accelerated
to 3-10 keV (in a variation, the fast atoms are subsituted for fast
ions, called liquid secondary-ion MS).  When coupled to HPLC,
the elution from the HPLC column is mixed with a matrix suitable to
promote FAB and then the mix is passed to the ion source.  The matrix
usually contains glycerol and the post-column mixing is preferred without
the need to affect chromatographic methods.  Effluent from HPLC is
passed to fused silica capillary, through a stainless steel frit in the
ion source for frit-FAB.  The gun is aimed on the outflow frit end.
Solvent from the frit rapidly evaporates and is pumped away from the system.
<p>
Chlorophyll from spinach leaves was extracted and measured.  Incubation
of leaves at 65&deg; for 30 min activates the enzyme chlorophyllase,
converting them to chlorophyllides <i>a</i> and <i>b</i>.  These are
extracted in acetone, and pheophytins extracted using Et-O-Et from the
acidified chlorophyll extract.  Pyropheophytins were obtained by
extraction of spinach puree treated at 145&deg; for 7 min.
<p>
HPLC was an ABi system using Vydac 2.1 &times; 150 mm C18 with 5 &micro;m
particles having a 300 &Aring; pore.  Flow was 70 &micro;l/min.
Positive-ion FAB spectra were obtained from JEOL JMS-HX110HF double-focusing
MS employing an HPLC-frit-FAB-MS interface with 6 keV Xe atoms.
Accelerating voltage is 10 keV with a resolution of 1000.  Scanning of
the <i>m/z</i> 300-1000 took 7 seconds;  for pyropheophytins the scan
time was 15 s.  Profiling was recorded as continuum spectra, but
HPLC-frit-FAB mode was centroided.  Eluate from the HPLC was split
so that a 5 &micro;l/min flow entered the interface (14:1 split).
Ion source temp was 40&deg; to prevent solvent freezing on the frit.
<p>
Different matrices have been tried in the past for chlorophylls.
3-nitrobenzyl alcohol produces good M<sup>+</sup> ions. [M+H]<sup>+</sup>
and [M-H]<sup>+</sup> ions (protonated and deprotonated) are seen at
lower relative abundance.  DTT and DTE or thioglycerol are more acidic
and produce effective protonation.  Glycerol was used because of its
proven compatibility and greater volatility than 3NBA.  It is not useful
for hydrophobic compounds nor in the standard probe used for FAB-MS
of chlorophylls.  Used with ethyl acetate and MeOH from the mobile phase
the glycerol provided a suitable condition for ionization of a wide
variety of chlorophyll degradation products.
<p>
[There is more to this paper that was not read largely because of absence
of an FAB-MS system.]
<!-- ============================================================= -->
<p class="tags">
chlorophyll, mass spectrometry
<p class="ref">
 <span class="authors">
van Breemen RB, Canjura FL, Schwartz SJ
 </span>
 <span class="year">
(1991)
 </span>
 <span class="jname">
J. Agric. Food Chem.
 </span>
 <span class="jvol">
39,
 </span>
 <span class="pages">
1452-1456.
 </span>
<p class="reftitl">
Identification of Chlorophyll Derivatives by Mass Spectrometry
<p class="Para1">
<!-- ============================================================= -->
<p class="tags">
salivary proteomics
<p class="ref">
 <span class="authors">
Isemura S, Saitoh E, Sanada K, Minakata K
 </span>
 <span class="year">
(1991)
 </span>
 <span class="jname">
J. Biochem.
 </span>
 <span class="jvol">
110,
 </span>
 <span class="pages">
648-654.
 </span>
<p class="reftitl">
Identification of Full-Sized Forms of Salivary (S-Type) Cystatins (Cystatin
SN, Cystatin SA, Cystatin S, and Two Phosphorylated Forms of Cystatin S) in
Human Whole Saliva and Determination of Phosphorylation Sites of Cystatin S
<p class="Para1">
Saliva was collected in an ice-cold medium with lots of protease inhibitors
A total of 10 ml per person was collected and pooled.  Precipitation was
with 4 vols cold MeOH.  MeOH evaporated and the protein was lyophilized.
Proteins were loaded to DE32 and gradient-eluted with a 100 &times; 100 mol
A and B system where B contains 3% NaCl (~0.5 M NaCl).  From this we
ran polyacrylamide gels and transblotted to nitrocellulose.  In some
cases, we pre-treated with alkaline phosphatase before this work-up.
Blots were then worked up with anti-cystatin S.  Several bands were
illuminated.  In some cases bands were missing that had been treated
with alkaline phosphatase, indicating a net loss of a negative charge
(generally the phosphate).
<p>
It is well known that degradative enzymes are present in saliva that
rapidly breakdown proteins.  The N-terminal portions of cystatins are
broken down rapidly in particular.  Use of rapid precipitation techniques
(MeOH) help to eliminate proteolytic activity.
<p>
Certain mammary gland casein kinases (including casein kinase II)
have been proposed to phosphorylated sequences such as SX[E/S(p)]
and SXX[E/S(p)], where the first serine gets phosphorylated, and
the C-side residue is an anionic residue like glutamic acid or
a phosphorylated serine.
<p>
Cystatin superfamily is composed of three familes (designated 1-3), distinguished
by structural homology.  Family 2 cystatin are secreted cysteine protease
inhibitors which have two disulfide bonds and molecular weights of 15 kDa.
Cystatins S, SA, and SN belong to family 2 and obtained from whole human
saliva; these have about 90% sequence homology.  Saliva also contains cystatin
C as family 2 member with 50% homology to S type.  Family 2 is composed of
a multigene family:  <i>CST1</i> coding cystatin SN, <i>CST2</i> coding
cystatin SA, <i>CST3</i> coding cystatin C, <i>CST4</i> coding cystatin S and
two pseudogenes, <i>CSTP1</i> and <i>CST5 (CSTP2)</i>.  A 7th gene has been
found but not its expression product.

<!-- ============================================================= -->
<p class="tags">
silver staining
<p class="ref">
 <span class="authors">
Rabilloud T
 </span>
 <span class="year">
(1992)
 </span>
 <span class="jname">
Electrophoresis
 </span>
 <span class="jvol">
13,
 </span>
 <span class="pages">
429-439.
 </span>
<p class="reftitl">
A Comparison Between Low Background Silver Diammine and Silver Nitrate
Protein Stains
<p class="Para1">
SDS-PAGE was in 19 &times; 16 cm gels with 1.5 mm depth.  A pH 8.5 330 mM TrisHCl
resolving gel is used (compared to 375 mM pH 8.8 Laemmli).  Stacking is a
250 mM BisTris pH 6.5, and electrode is 50 mM Tris, 385 mM glycine, 0.1% SDS.
Stacking is 4.5% T (2.6% C), and resolving is 10% T (2.6% C).  For each 60 ml
gel a 0.5 ml 10% APS (0.83%) and 25 &micro;l TEMED (0.04%) is used.  Gels stained
with diammine are prepared with PDA crosslinker using thiosulfate initiator
(0.5 ml 10% thiosulfate pentahydrate), 0.5 mL 10% APS, and 50 &micro;l TEMED.
<p>
Gel staining was 250 ml per gel except for the silver diammine (100 ml for gel),
with plastic trays used.
<p>
Acidic silver methods tested are divided into <b>fast</b> and <b>long</b> (&gt; 5 h) methods.
Fast methods are those of Merrill et al. (in <i>Two-Dimensional Gel
Electrophoresis of Proteins</i>, Celis and Bravo, eds., Academic Press:
Orlando FL, pp 93-109, 1984), Hempelmann et al (<i>Electrophoresis '84</i>,
Neuhoff, ed., Verlag Chemie: Weinheim, pp 328-330, 1984), Johansson and
Skoog (<i>J. Biochem. Biophys. Methods</i> <b>14</b> Suppl.: 33, 1987),
Morissey (<i>Anal. Biochem.</i> <b>117</b>: 307-310, 1981), Doucet and Trifaro
(<i>Anal. Biochem.</i> <b>168</b>: 265-271, 1988), Blum et al.
(<i>Electrophoresis</i> <b>8</b>: 93-99, 1987), and Rabilloud et al
(<i>Electrophoresis</i> <b>9</b>: 288-290, 1988).  Long methods, usually
requiring overnight fixation and extensive washing (to remove sensitizing
agent typically), are those in the family of Heukeshoven and Dernick
(<i>Electrophoresis Forum '86</i>, Radola, ed. Technische Universit&auml;t:
M&uuml;nchen, pp 22-27, 1986).  Typical modifications use of double the
amount of silver nitrate (0.2% instead of 0.1%) with doubled incubation times,
and potassium carbonate instead of sodium carbonate with hypo included in
the developer, and the use of Tris-acetate as a stop instead of disodium
EDTA for extended times.  The method of Sammons et al (<i>Electrophoresis</i>
<b>2</b>: 135-141, 1981) gave a consistently low level yellow background
and was not tested further.
<p>
Silver diammine methods are based on that of Oakley et al (<i>Anal.
Biochem.</i> <b>105</b>: 361-363, 1980) and have been <q>dramatically
improved</q> (see Hochstrasser, Patchornik, and Merril, <i>Anal. Biochem.</i>
<b>173</b>: 412-423, 1989; and Hochstrasser and Merril, <i>Appl. Theor.
Electrophoresis</i> <b>1</b>: 35-40, 1989).  As a basis, Hochstrasser
and Merril's method was used as a basis.  Additional tested methods are
Marshall and Latner (<i>Electrophoresis</i> <b>2</b>: 228-235, 1981),
Wray et al. (<i>Anal. Biochem.</i> <b>118</b>: 197-203, 1981), and
Mold et al. (<i>Anal. Biochem.</i> <b>135</b>: 44-47, 1983).  Fixation,
the presence or absence of sensitizing steps (and the use of different agents
such as glutaraldehyde), the use of methylamine or ammonium hydroxide as
the amine complexing agent, and large variations on rinsing after fixation
are the variations of these methods.  Strictly consistent features of these
methods are use of HCHO and citric acid in the developer and a high (0.8%)
silver nitrate concentration and NaOH as the alkalinizer in the silvering step.
The stop is usually 5% acetic acid in 20% EtOH, although one uses tris-acetate.
<p>
Note of the seven fast method protocols, the first three protocols listed were
plagued by low sensitivity or high backgrounds or both (even with thiosulfate
addition as a means to prevent high backgrounds), and so more extensive
testing was limited to the latter four protocols.
<p>
In the initial stages, the sensitivity  of silver staining methods were tested.
The detection limiits of typical marker proteins were determined.
<table>
<caption>Detection limits bv silver staining for marker proteins&dagger;</caption>
<tr><th><th>Myosin<th>&beta;-Galactosidase<th>Phosphorylase
<th>Bovine Serum Albumin<th>Ovalbumin<th>Carbonic Anhydrase
<tr><td colspan="8" class="left" style="font-size:75%;"><i>All values in nanograms</i>
<tr><td class="left" style="width:12em;">Doucet and Trifaro
	<td>20 <td> 20 <td> 10 <td> 5-10* <td> 20 <td> 10
<tr><td class="left">Blum et ul.
	<td> 5 <td> 5  <td> 10 <td> 10 <td> 10 <td> 10
<tr><td class="left"> Rabilloud et al.
	<td> 2-5* <td> 2-5* <td> 5 <td> 5 <td> 10 <td> 10
<tr><td class="left"> Morrissey
	<td> 20 <td> 20 <td> 20 <td> 10 <td> 20 <td> 10
<tr><td class="left"> Heukeshoven and Dernick
	<td>5 <td> 5 <td> 10 <td> 5 <td> 5 <td> 10
<tr> <td class="left">Tetrathionate method (H&nbsp;&amp;&nbsp;D&nbsp;variant)
	<td> 2 <td> 2 <td> 2 <td> 2 <td> 5 <td> 10
<tr><td class="left"> Wray et a1.
	<td> 10 <td> 5 <td> 10 <td> 5 <td> 10 <td> 10
<tr><td class="left"> Modification of Mold et al.
	<td> 2 <td> 1 <td> 1 <td> 2 <td> 2 <td> 50
<tr><td class="left">Hochstrasser et al.
	<td> 2 <td> 1-2* <td> 1-2* <td> 2 <td> 2 <td> 5
<tr><td colspan="8" style="font-size:83%;" class="left">
&dagger; The sensitivities ares expressed in ng detected in this gel system.
The bands are ca. 8 &times; 2 mm; the sensitivities given here should therefore
be divided by 16 to obtain the ng/mm2, so that 10 ng correspond to 625 pg/mm<sup>2</sup>.
As the gels are 1.5 mm thick, a 10 ng detection corresponds to a 57 pg/mm<sup>2</sup>
(57 ng/mL).<br>
* indicates when the results are highly variable from gel
to gel.  This variability seems greater in the high molecular weight region
owing to a more intense background development. This background seems to be
linked to several factors, including the cleanliness of the combs, whichs
seems difficult to keep at a sufficient level owing to adsorption to the
plastic. The detection limits for soybean trypsin inhibitor are not given
since the protein migrates too close from the dye front to allow correct
separation and estimation.
</table>
<p>
The least successful methods are those using DTT as a sensitizing agent.  Yellow
to red backgrounds appear (Doucet and Trifaro and the Morrissey methods).  Adding
thiosulfate to developer only partially relieves problem.  Blum et al.
and Rabilloud et al methods were best, although latter gives blacker bands.
<p>
Long silver nitrate methods give superior results although are prone to surface
staining and formation of dark precipitates from silver carbonate.  Hollow spots
form which can be corrected by increasing the silver nitrate, decreasing the rinse
between silver nitrate and developer, and adding 10 mg/L thiosulfate to developer.
<p>
The silver diammine methods gave the best results although original methods gave
point streaks (rainy) gels.  This is prevented by using alkylating the thiol reagent
present in sample buffer (iodoacetamide is superior to iodoacetic acid).
When sensitizers are not used, poorer sensitivity is seen.  Use of
naphthalene-2,7-disulfonicc acid instead of aldehyde works well (but one marker protein
had its sensitivity reduced considerably, which means certain proteins may not be
detected).
<p>
When small and/or basic peptides must be visualized formaldehyde, and not
alcohol-acid, fixation is required, and a silver diammine method should be used.
<p>
2D electrophoresis of acidic, neutral, and basic proteins was also tested.  Even
the fast method performed as well as the longer.  All did well for acidic and neutral
proteins, with apparent higher sensitivity of silver diammine methods.  Basic proteins
were much better detected with silver diammine staining, giving darker spots and less
<q>hollow</q> staining.
<p>
The <q>hollowness</q> of staining appears to be dependent on silver concentration, being
high for 0.1% silver nitrate, moderate for 0.2% levels, and very low when diammine
methods which typically use 0.4-0.8%, given a long development period (hollowness appears
at first, then the spots darken as development continues).
<!-- ============================================================= -->
<p class="tags">
protein identification, 2D gel electrophoresis, sequence databases
<p class="ref">
 <span class="authors">
Henzel WJ et al.
 </span>
 <span class="year">
(1993)
 </span>
 <span class="jname">
Proc. Natl. Acad. Sci. USA
 </span>
 <span class="jvol">
90,
 </span>
 <span class="pages">
5011-5015.
 </span>
<p class="reftitl">
Identifying Proteins From Two-Dimensional Gels by Molecular Mass Searching
of Peptide Fragments in Protein Sequence Databases
<p class="Para1">
<!-- ============================================================= -->
<p class="tags">
protein identification
<p class="ref">
 <span class="authors">
Mann M, Hojrup P, Roepstorff P
 </span>
 <span class="year">
(1993)
 </span>
 <span class="jname">
Biol. Mass Spectrom.
 </span>
 <span class="jvol">
22,
 </span>
 <span class="pages">
338-345.
 </span>
<p class="reftitl">
Use of Mass Spectrometric Molecular Weight Information To Identify Proteins
in Sequence Databases
<p class="Para1">
<!-- ============================================================= -->
<p class="tags">
protein identification
<p class="ref">
 <span class="authors">
Pappin DJC, Hojrup P, Bleasby AJ
 </span>
 <span class="year">
(1993)
 </span>
 <span class="jname">
Curr. Biol.
 </span>
 <span class="jvol">
3,
 </span>
 <span class="pages">
327-332.
 </span>
<p class="reftitl">
Rapid Identification of Proteins by Peptide-Mass Fingerprinting
<p class="Para1">
<!-- ============================================================= -->
<p class="tags">
mass fingerprinting
<p class="ref">
 <span class="authors">
James P, Quadroni M, Carafoli E, Gonnet G
 </span>
 <span class="year">
(1993)
 </span>
 <span class="jname">
Biochem. Biophys. Res. Commun.
 </span>
 <span class="jvol">
195,
 </span>
 <span class="pages">
58-64.
 </span>
<p class="reftitl">
Protein Identification by Mass Profile Fingerprinting
<p class="Para1">
<!-- ============================================================= -->
<p class="tags">
peptide mass maps, protein identification
<p class="ref">
 <span class="authors">
Yates JR III, Speicher S, Griffin PR, Hunkapiller T
 </span>
 <span class="year">
(1993)
 </span>
 <span class="jname">
Anal. Biochem.
 </span>
 <span class="jvol">
214,
 </span>
 <span class="pages">
397-408.
 </span>
<p class="reftitl">
Peptide Mass Maps:  A Highly Informative Approach to Protein Identification
<p class="Para1">
<!-- ============================================================= -->
<p class="tags">
MALDI analysis, mass accuracy
<p class="ref">
 <span class="authors">
Vorm O, Mann M
 </span>
 <span class="year">
(1994)
 </span>
 <span class="jname">
J. Am. Soc. Mass Spectrom.
 </span>
 <span class="jvol">
5,
 </span>
 <span class="pages">
955-958.
 </span>
<p class="reftitl">
Improved Mass Accuracy in Matrix-Assisted Laser Desorption/Ionization
Time-of-Flight Mass Spectrometry of Peptides
<p class="abstract">
<b>ABSTRACT</b>.
One problem of matrix-assisted laser desorption ionization coupled
to time-of-flight mass spectrometry is the moderate mass accuracy
that typically can be obtained in routine applications. Here we
report improved mass accuracy for peptides, even when low amounts
and complex peptide mixtures are used. A new procedure for preparing
matrix surfaces is used, and there is no need to mix the matrix
with the sample or to add internal standards. Examples are shown
with a mass accuracy better than 50 ppm in a peptide mixture.
Peptide mapping as well as sequencing by creating "ragged ends" or
"ladder sequencing" should benefit especially from the improved
mass accuracy.
<!-- ============================================================= -->
<p class="tags">
zinc-peptide interaction, fragmentation, FD
<p class="ref">
 <span class="authors">
Loo JA, Hu P-F, Smith RD
 </span>
 <span class="year">
(1994)
 </span>
 <span class="jname">
J. Am. Soc. Mass Spectrom.
 </span>
 <span class="jvol">
5,
 </span>
 <span class="pages">
959-965.
 </span>
<p class="reftitl">
Interaction of Angiotensin Peptides and Zinc Metal Ions Probed by
Electrospray Ionization Mass Spectrometry
<p class="Para1">
1-10 mM ZnCl<sub>2</sub> was added to a 50-100 pmol/&micro;l (&micro;M)
solution of angiotensin II (FW = 1046, DRVYIHPF) at pH 6.  The peptide
bound one Zn<sup>2+</sup> from the appearance of mass spectra.  In some
cases two Zn ions were bound.  Angiotensin I (FW=1296, DRVYIHPFHL) has
two His residues;  it also has one Zn residue binding with some proportion
having two bound Zn ions.  Complex stoichiometry follows the formula
[M + <i>a</i>Zn<sup>2+</sup> + (<i>c</i> &minus; 2<i>a</i>)H<sup>+</sup>]<sup><i>c</i>+</sup>
where <i>c</i> is the charge.  At pH 6-7 the metal complex formation is
at its highest, but falls off at the extremes (pH 3 or 10).
Association with Zn can actually be identified since Zn has many isotopes
with known natural abundance, and this can be used as a fingerprint.
<p>
Protonated and metal-bound peptides were subjected to collisionally
activated dissociation.  CAD of [M + 2H]<sup>2+</sup> angiotensin II
shows a series of <i>a<sub>n</sub>/b<sub>n</sub></i> singly charged
products in both a low energy collision (triple quad) and higher
energy collision (Finnigan MAT 900 double focusing forward geometryMS)
CAD of the [M + Zn]<sup>2+</sup> ion produces primarily [<i>b</i><sub>6</sub>
+ Zn]<sup>2+</sup>/[<i>a</i><sub>6</sub> + Zn]<sup>2+</sup> pair also
with ions formed by water loss.
<p>
Zn also influences the fragmentation.  Protonated +2 and +3 angiotensin I
produce +1 and +2 fragments along the entire backbone.  The CAD spectra
of [M + Zn]<sup>2+</sup> at <i>m/z</i> 681 produces an ion at <i>m/z</i>
615 which is [<i>b</i><sub>9</sub> + Zn]<sup>2+</sup>, the most abundant
product;  ion [<i>b</i><sub>6</sub> + Zn]<sup>2+</sup> is also observed.
The fragmentation of His<sup>9</sup>-Leu<sup>10</sup> is favored over
that of His<sup>6</sup>-Pro<sup>7</sup>.  This is probably related to
a preferential binding of Zn to the His whose fragmentation is induced by
the binding.  When [Ala<sup>6</sup>]-angiotensin II is used instead,
the Ala<sup>6</sup>-Pro bond is only one fragmentation site in the presence
or absence of Zn ion.  In contrast to [Ala<sup>6</sup>-angio II + Zn]<sup>2+</sup>,
the acidified ion [Ala<sup>6</sup>-angio II + 2H]<sup>2+</sup> produces
abundant <i>b</i><sub>6</sub><sup>+</sup> and <i>b</i><sub>6</sub><sup>2+</sup>
ions.  Substance P, lacking His (RPKPQQFFGLM-nh<sub>2</sub>), shows the
same fragmentation pattern whether associated with Zn or protonated form
for the 2+ ion.  Protonated species produce singly and doubly charged
<i>a</i><sub>7</sub>, <i>a</i><sub>8</sub>, <i>b</i><sub>9</sub>,
<i>a</i><sub>10</sub>, and <i>b</i><sub>10</sub>.  Zn-assocation produces
[<i>b</i><sub>10</sub> + Zn &minus; H]<sup>+</sup> ions of same type.
<p>
Some have said that the amide X&mdash;Pro bond is susceptible to fragmentation
in multiply charged ions under CAD conditions.   The process may not be
favored if X is a positively charged residue such as Arg, Lys, His.
Pro may not play a significant role in the [<i>b</i><sub>6</sub> + Zn]<sup>2+</sup>
ion formation:  substitution of Pro<sup>7</sup> for Ala<sup>7</sup>
produces a fragmentation pattern similar to the unsubstituted with Zn present.
<p>
One ion is a water-loss product, accounting for 30% of ions in the
Zn-angio II complex.  Water loss product is also significant in
CAD spectra of angiotensin I and of [Ala<sup>6</sup>]-angiotensin II
in Zn presence.  Water loss comes from the N-terminal Asp residue.  This does not happen
with Substance P, which has no N-terminal Asp.  Without Zn, a
[M &minus; H<sub>2</sub>O + Zn]<sup>2+</sup> ion is not observed.
The acidified/protonated form alone shows no water loss product.  Note
that with the Ala<sup>6</sup> subsitution, Zn probably moves to another
more preferable binding site (N-terminal Asp), and water loss ensues with
metal presence.  Tests of other N-terminal formations (acetylated-Asparate)
did not yield water loss.
<p>
Consider the nature of peptides and proteins and the metals they bind.
Histidine is a component of many metal-binding proteins, coordinating
imidazole nitrogens around the metal.  Consider the angiotensin structure.
One of the imidazole nitrogens coordinates with Zn, and carbonyl oxygens
from proline and from the preceding amino acid also coordinate with their
nonbonding pairs on the oxygens.  This folds the peptide in a way that
entropy favors fragmentation to a <i>b</i>-type ion.  A bidentate formation
of the two His to Zn could occurs with angiotensin I (suggested by NMR).
The pair <i>a</i><sub>6</sub><sup>2+</sup>/<i>b</i><sub>6</sub><sup>2+</sup>
forms very weakly compared to <i>a</i><sub>9</sub><sup>2+</sup>/<i>b</i><sub>9</sub><sup>2+</sup>
from angiontensin I [M + Zn]<sup>2+</sup> resulting from the bidentate
coordination, promoting formation of latter pair and inhibiting formation
of former.
<!-- ============================================================= -->
<p class="tags">
mobile proton theory, CID, peptide fragmentation, FD
<p class="ref">
 <span class="authors">
Downward KM, Biemann K
 </span>
 <span class="year">
(1994)
 </span>
 <span class="jname">
J. Am. Soc. Mass Spectrom.
 </span>
 <span class="jvol">
5,
 </span>
 <span class="pages">
966-975.
 </span>
<p class="reftitl">
The Effect of Charge State and The Localization of Charge on The
Collision-Induced Dissociation of Peptide Ions
<p class="abstract">
<b>ABSTRACT</b>.
The effect that charge state has on the collision-induced
dissociation (CID) of peptide ions is examined in detail for
several representative peptides under high-energy collision
conditions. The CID spectra of singly and doubly charged precursor
ions (generated by fast-atom bombardment and electrospray ionization,
respectively) are compared for several peptides with similar
primary structure. It is shown that for peptides that contain
highly basic amino acids, the dissociation of doubly charged ions
is strongly influenced by the position of these residues within
the peptide and the general observations reported concerning the
dissociation of singly charged ions can be extended to precursors
with higher charge states. Based on the dissociation behavior of
the doubly charged ions of these peptides, it is demonstrated
that two charges can reside in close proximity in the precursor ions,
overcoming possible repulsion effects, when favored by a high
concentration of basic sites. In addition, this work illustrates
that in the case of doubly charged ions, the charge state of some
fragment ions can be determined directly from the mass-to-charge
ratio assignments of the CID spectrum.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Eng JK, McCormack AL, Yates JR 3rd
 </span>
 <span class="year">
(1994)
 </span>
 <span class="jname">
J. Am. Soc. Mass Spectrom.
 </span>
 <span class="jvol">
5,
 </span>
 <span class="pages">
976-989.
 </span>
<p class="reftitl">
An Approach To Correlate Tandem Mass Spectral Data of Peptides
With Amino Acid Sequences in A Protein Database
<p class="abstract">
<b>ABSTRACT</b>.
A method to correlate the uninterpreted tandem mass spectra of
peptides produced under low energy (1050 eV) collision conditions
with amino acid sequences in the Genpept database has been developed.
In this method the protein database is searched to identify
linear amino acid sequences within a mass tolerance of  1 u of
the precursor ion molecular weight. A cross-correlation function
is then used to provide a measurement of similarity between the
mass-to-charge ratios for the fragment ions predicted from
amino acid sequences obtained from the database and the fragment
ions observed in the tandem mass spectrum. In general, a difference
greater than 0.1 between the normalized cross-correlation functions
of the first- and second-ranked search results indicates a successful
match between sequence and spectrum. Searches of species-specific
protein databases with tandem mass spectra acquired from peptides
obtained from the enzymatically digested total proteins of E. coli
and S. cerevisiae cells allowed matching of the spectra to amino
acid sequences within proteins of these organisms. The approach
described in this manuscript provides a convenient method to
interpret tandem mass spectra with known sequences in a protein database.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Zhou HX, Lyu PC, Wemmer DE, Kallenbach NR
 </span>
 <span class="year">
(1994)
 </span>
 <span class="jname">
J. Am. Chem. Soc.
 </span>
 <span class="jvol">
116,
 </span>
 <span class="pages">
1139-1140.
 </span>
<p class="reftitl">
Structure of A C-Terminal &alpha;-Helix Cap in A Synthetic Peptide
<p class="Para1">
The end regions of an &alpha;-helix are critical since the amino acids are
necessary to stabilize the helix.  Amino acids within the center of the
helix length would form H-bonds with NH and CO groups four residues away.
It is therefore the case that the first and last four residues of a helix
do not form the H-bonds required for helix formation.  In globular proteins
the mean length of alpha helices is 12 residues, meaning the majority of residues
actually are end groups.  To promote the helix stabilization, very flexible
glycine residues interact with NH and CO groups and polar side chains.
<p>
This use of certain residues is known as helix capping.  The capping stabilization
has been demonstrated by sequence analysis and site-directed mutation of native
proteins, and structual analysis of synthetic peptides.  Perhaps the first
evidence of capping was the observation of an unusual H-bond on the myoglobin
C-terminal H-helix.  Another report shows that one-third of helices end
 with residue in left-handed (&alpha;<sub>L</sub>) conformation. having
&phi; and &psi; angles &gt; 0, promoted by Gly residues.  Changing the C-cap
from Gly to Ala destablizes the helix by 3.1 kcal/mol.
<p>
This report shows how the NH of the side chain of an Asn residue forms H-bond
with helix main chain of CO four residues away.  The 20-amino acid long
synthetic peptide ac-YMSEDELKAAEAAFKRHxPT-nh<sub>2</sub>, where 'x' is a site
for testing substitution of several amino acids, represents the consensus
sequence of residues that make up the &alpha;-helix shown by Richardson
and Richardson (<i>Science</i> <b>240</b>: 1648, 1988).  A local 3<sub>10</sub>
helix at the C-terminus and the side chain H bonds  in a loop.  While it strengthens
the helix, the formation of the loop also blocks helical continuation and
more strongly than the Pro19 residue (shown by CD spectroscopy at 222 nm).
<p>
<sup>1</sup>H-NMR was used to examine how N18 significantly enhanced helicity
compared to other substituted amino acids (Q, A, S, T, G).  2D TOCSY and NOESY
experiments identify spin systems and sequential assignments.  Each amide from
residue 4 (Glu) links to the side chain NH of N18, and longer range bonding between
C<sub>&alpha;</sub>H and amide NH from residue 5 to 14.
<p>
Three regions of the NOESY spectrum provide structure detail about the C-terminus.
NOE cross peaks show C-cap structures involving N18 side chain.  Two &beta;
protons of N18 have distinct chemical shifts, with one &beta;&prime; proton
closer to the peptide NH than the other &beta; proton.  The side chain amide
proton is close to the C<sub>&alpha;</sub>H of F14.  A ROESY experiment
shows that weak peaks are not because of spin diffusion..  These NMR studies show
H-bonding between CO of Lys 15 and NH of Asn 18, with possible bifurcation to
CO of Phe 14 as well.  Only Asn 18 side chain of NH  bonds to CO of Phe 14.
A weak NOE connect N18 backbone amide to &beta; proton of His 17.
<p>
From the CD spectroscopy, order of helix stabilization for the X 18 site is
N &gt; Q &gt; (A, S) &gt; (T, G).  Multiple interpretations are possible:
Q H-bonding is less entropically favorable than N because of the extra methylene;
Ala is still stabilizing at helix end, but not maximally; Ser and Thr can contribute
a &gamma;-proton to H-bond with CO of Phe 14, but there is no evidence; and Gly should
be better than Ala at capping stabilization, although not shown for this peptide
(it is possible that the presence of Pro affects the ability of Gly).
The common feature in N- and C-caps seen so far is that H-bonds
<!-- ============================================================= -->
<p class="tags">
tau protein, paired helical fragments (PHF), Alzheimer's disease
<p class="ref">
 <span class="authors">
Schweers O, Sch&ouml;nbrunn-Hanebeck E, Marx A, Mandelkow E
 </span>
 <span class="year">
(1994)
 </span>
 <span class="jname">
J. Biol. Chem.
 </span>
 <span class="jvol">
269,
 </span>
 <span class="pages">
24290-24297.
 </span>
<p class="reftitl">
Structural Studies of Tau Protein and Alzheimer Paired Helical Filaments Show No
Evidence for &beta;-Structure
<p class="Para1">
Amyloid plaques and neurofibrillary tangles are features of the age-related
dementia Alzheimer's disease.  Both plaques and tangles arise from abnormally
aggregated protein. Plaques contain A&beta; peptide aggregated into fibers
(A&beta; peptide is membrane protein breakdown product) and tangles contain tau
protein aggregated into PHFs. Abnormal protein formation and fibers are seen in
other disease states as well, such as prion protein in scrapie or
Creuztfeld-Jakob-Disease or fibers of systemic amyloidosis consisting of
transthyretin.  Aggregation might occur as a result of intermolecular
&beta;-sheet assembly.  &beta;-Amyloid and PHFs are very stable in interactions
between interlocking &beta;-strands.  Moreover aggregation in vitro occurs where
proteins adopt &beta;-sheets in solution, as seen in CD and FTIR.  &beta;-Sheets
also are seen in abnormal fibers by X-ray diffraction or FTIR.  Alzheimer
&beta;-amyloid hve a cross &beta;-structure where strands meander across fiber
width such that a 4.7 &Aring; reflection occurs on meridian of diffraction
pattern corresponding to separation of strands.  But not in all cases should it
be assumed that fibrous aggregation is governed by similar mechanisms of
formation. For example while transthyretin is composed mainly of &beta;-sheet
shown by x-ray crystallography, the interaction between subunits is mediated by
thiols on the protein surface forming disulfides.  Tau has only a negligible
fraction of &alpha;- or &beta; conformation, if any, similar to other
microtubule-associated proteins (MAP2), a finding seen in tau isolated from
brain or bacterial recombinant, and whether endogenously phosphorylated or free
of it.  Tau may acquire its &beta;-stranding as a result of aggregation, which
provides an interesting mechanism for PHF assembly.  Amyloid fibers and PHFs
co-exist in senile plaques, and so the cross-&beta; pattern of tangle
preparation could arise from the &beta;-amyloid component, indicating that
x-rays can pick out periodic structures even present only as contaminants. <p>
<b>Results</b>.  The domain structure of bovine brain tau is reviewed.  CD shows
little secondary structure, a finding also seen in E coli recombinant human tau.
Boiled and dissolved in dilute acid, tau retains it microtubule
assembly-promoting activity. All tau isoforms have large number of hydrophilic
resides and prolines which are not usual for proteins compactly folded.
Secondary structure prediction reports do not show significant structural
features.  Gel filtration analysis reveals that tau aggregates into dimers and
higher order assemblies.  Dimer association is anti-parallel, possible precursor
to PHF formation.  Electron microscopy is not informative on tau assemblies
alone because of low contrast, but glycerol-spray rotary shadow technique shows
elongated and flexible rod 35 nm long. <p> Small-angle X-ray scattering can
determine overall molecular shape and internal structure. Scattering data were
obtained for bovine tau isoform 4 (largest of isoforms) and a control lysozyme
protein and plotted according to Gunnier (log <i>I</i> vs. <i>s</i><sup>2</sup>
[nm<sup>&minus;2</sup>]) and Kratky (<i>I&times;s</i><sup>2</sup> vs. <i>s</i>
[nm<sup>&minus;1</sup>]. The Gunnier plot shows for globular proteins like
lysozyme a straight line from which a radius of gyration (<i>R</i><sub>G</sub>)
of 1.5 &plusmn; 0.05 nm is obtained.  With tau there is a continuous curvature
from which no <i>R</i><sub>G</sub> value can be determined. This is also
determined for tau domains (designated K0, K1, K2, and K12).  The chain is this
<q>worm-like</q> or described as <q>Gaussian coil</q> and not compacted to
globular form.  This approximates a denatured state.  A Kratky plot of the
scattering data shows three segments:  a rise, a roughly horizontal region, and
another rise.  The segment 2-3 transition is a measure of persistence length of
chain (1.97 nm). <p> Domain K12 contains two tyrosine residues (positions 310
and 394) and no tryptophan. Absorption of the phenol group depends on solution
environment:  changing the concentration of urea from 0 to 8 M shows no change
in extinction coefficient, and spectrum between 240 and 310 is the same,
indicating full exposure of Tyr residues to nonhydrophobic conditions in all
cases. <p> Human tau isoform 23 and the K12 domains were observed in CD
spectroscopy and the data show &alpha;-helical content of 0 and 10%,
respectively, consistent with a previous report and of values so low as to be
unreliable.  Increasing temperature and ionic strength had no discernible effect
on spectra.  Increasing TFE, which is known to promote helix formation, to 50%
increased K12 domain helix to 30% content.  Spectra of monomers and cross-linked
dimers
<!-- ============================================================= -->
<p class="tags">
MALDI, analytical improvement techniques
<p class="ref">
 <span class="authors">
Kallweit UK, B&ouml;nsen KO, Kresbach GM, Widmer HM
 </span>
 <span class="year">
(1996)
 </span>
 <span class="jname">
Rapid Commun. Mass Spectrom.
 </span>
 <span class="jvol">
10,
 </span>
 <span class="pages">
845-849.
 </span>
<p class="reftitl">
Matrix Compatible Buffers for Analysis of Protein With Matrix-Assisted
Laser Desorption/Ionization Mass Spectrometry
<p class="Para1">
The Linear Scientific (Hewlett-Packard) LDI-1700 mass spectrometer is used.
Laser power is 2-3 &micro;J/pulse for the 337 nm laser, with 10 shots per
spectrum in positive mode. DAHC is diammonium hydrogen citrate
[NH<sub>4</sub>OOC-CH<sub>2</sub>-C(COOH)-CH<sub>2</sub>-COONH<sub>4</sub>],
HIC is 1-hydroxyisoquinoline, DHAP is 2,6-dihydroxyacetophenone.
<p>
Formation of fine crystals with analyte embedded is ideal. 8 buffer with 5
different matrices were tested. Crystallization was characterized as excellent,
good, difficult, or none using 5 different matrices (pure chemicals and
combinations: 2,5-DHB, DHB/HIC, DHAP/DAHC, SA, SA/DAHC) in 50 mM of buffer salt
(ACES, BES, HEPES, imidazole, MOPS, TES, Tris, phosphate). Phosphate made
crystallization difficult with all matrices, and the DHAP/DAHC crystallized <q>
good</q> only with MOPS, TES, and Tris. Sinapinic acid, DHB/HIC were <q>good</q>
 with all buffer salts not phosphate, and DHAC/SA was good with every one
(except phosphate), and <q>excellent</q> with ACES, BES, and MOPS. 2,5-DHB was
<q>good</q> or <q>excellent</q> with ACES, BES, imidazole and Tris, but not
with MOPS or TES, and not at all with HEPES. The phosphate combination formed
<q>wet crystals</q>. DAHC helps probably because it complex alkali ions or has
a favorable lattice. Phosphate inclusion always resulted in bad signals.
<p>
BSA was dissolved to 10 &micro;M in the buffers and 2 &micro;l mixed with
2 &micro;l of matrix, and 1 &micro;l applied to probe tip and dried in
vacuum for fast crystallization.  For dealing with protein, sample is
desalted by drop dialysis (5-20 &micro;l drop on 25 nm pore Millipore MF
for 30-60 min then removed and mix with matrix immediately). Or the following
is done:  Immobilon CD  4 mm dia. is soaked with MeOH and then water
to make surface hydrophobic for aqueous substances; membrane immersed in
50 &micro;l sample, shaken softly for 30-60 min; membrane rinsed 2 min with
ultrapure water and then mixed with 30 &micro;l sinapinic acid/DAHC solution.
<p>
With HEPES and MOPS, BSA gives good signal, even enhanced from pure water.
BES, Tris, and TES give less homogeneous signals and broader peaks.
Imidazole and ACES give noisy baselines and even broader, almost undetectable
peaks.
<p>
Concentration dependence was checked with buffers at 50, 100, 250, and 500 mM
using sinapinic/DAHC: in general buffer concentration should be &lt; 100 mM so
as not to affect crystallization. The best spectra always are found when
crystallization is fast and homogenous. Excessive salt concentration causes a
heterogeneous crystallization, and especially when buffer salts exceed matrix
mole for mole. Low peak intensities and broad peaks are to be expected. At 500
mM several salts form wet films (Tris, MOPS, HEPES) while others form dry
crystals (ACES, BES, TES, imidazole). Close look at MOPS shows that at 50 mM
there is no signal interference, but at 100 mM the baseline is more noisy, with
ion suppression. At 250 mM, the trimeric form of BSA [3M+H]<sup>+</sup> is not
seen and the matrix/analyte crystal is a heterogeneous layer. BSA in 500 mM
MOPS is undetectable entirely since it is not a crystal but a wet film.
Desalting using Immobilon membrane or drop dialysis restored the spectrum
signal (best results with Immobilon CD membrane; note that Immobilon-PSQ, -N,
and -S membranes gave smaller intensities). The best results with this
desalting technique occur when the protein interacts with the membrane; the
protein can be desorbed using the matrix solution, which disrupts the
protein-membrane interaction. With drop dialysis, it helps to add DAHC to the
reservoir solution.
<!-- ============================================================= -->
<p class="tags">
ToF PD-MS
<p class="ref">
 <span class="authors">
Zubarev RA, H&aring;kannson P, Sundqvist B
 </span>
 <span class="year">
(1996)
 </span>
 <span class="jname">
Rapid Commun. Mass Spectrom.
 </span>
 <span class="jvol">
10,
 </span>
 <span class="pages">
1386-1392.
 </span>
<p class="reftitl">
Accurate Monoisotopic Mass Measurements of Peptides:  Possibilities
and Limitations of High Resolution Time-of-Flight Particle Desorption
Mass Spectrometry
<p class="Para1">
With a &Delta;<i>M / M</i> = &plusmn;1 ppm, no more than 1% of all possible
peptide combinations will fall within the interval <i>M</i> &plusmn; &Delta;<i>
M</i> (99% confidence). Confidence is affected the tolerance set. A 90%
confidence will allow a mass accuracy of 10 ppm. Average isotopic mass can not
be used in this case since the natural distribution of isotopic abundances is
wider than this tolerance. In the low and medium mass range, monoisotopic mass
is better for comparison.
<p>
The DIPLOMA mass spectrometer was built for plasma desorption mass spectrometry
(PDMS) and is also capable of SIMS mode. Its resolution is 5000 (with one
reflectron) for organic ions. Clustered ions (CsBr) can be resoluted to 10000
FWHM. The TDC has a 0.5 ns time resolution. X- and Y-deflection plates and an
Einzel lens are used in secondary ion beam focusing toward the reflectron. A
beam of 72.3 MeV <sup>127</sup>I<sup>13+</sup> ions from a tandem accelerator
were aimed in PDMS mode. A SIMS gun generates a 25 keV Ga<sup>+</sup> ion beam
with pulse of 5 ns width. Targets are prepared on 1 &times; 1 cm silicone slice
(Biolon as makes commericial PDMS targets) with a 12 mm brass disc featuring a
2 &micro;m thick alumninized mylar film. Matrix is 200 &micro;g/cm<sup>2</sup>
layer of nitrocellulose deposited by electrospraying from acetone solution.
Peptides are incubated in 1 M HCl at 100&deg; at times from 2-10 min and dried
on bare silicon slices. Bradykinins were partially acid hydrolyzed in this way
for the mass accuracy test. TOF spectra had peak positions determined, mass
scale calibration with low mass ion exact masses.
<p>
For peak position determination consider the low mass isobaric ions C<sub>
6</sub><sup>+</sup> and C<sub>4</sub>H<sub>8</sub>O, with only a mass
difference of 0.058 u. First a peak location is determined: this can be a one
peak, or with a better analysis, two more more overlapping peaks. Peak
multiplicity <i>m</i>, or the number of peaks, is evaluated. Then <i>m</i>
Gaussian curves are fit to the curves in the location. The width of the curves
is constrained to 0.5-2.0 of the expected peak width, with 2.0 representing a
value derived from the known resolution of the instrument. If broad metastable
peaks are there, it can lead to too many Gaussians.
<p>
Of course, with ToF, the scale is <i>t</i> = <i>A</i> &radic;<i>m/z</i> + <i>
B</i> where <i>A</i> is the slope dependent on pathlength and voltages, and <i>
B</i> represents the time delay in the electronics. This is fitted with a
series of <i>n</i> equations as above solved for <i>n</i> known peaks.
<p>
194 peptides had their masses determined and compared to the theoretical.
Deviations revealed both random (&plusmn;22 ppm, or &plusmn;0.015 mDa, std dev)
and systematic error (+16 ppm, or +0.010 mDa, std dev.). The systematic error
reflects that peaks of petpide ions appear heavier than their theoretical
masses. Why? One reason could be peak shape, with many peaks being asymmetric.
The MH<sup>+</sup> and MNa<sup>+</sup> of Leu-enkephalin show tailing on the
high mass side. Gaussian fits are symmetrical, but will shift the peak toward a
skew when real peaks are not symmetrical.
<p>
Other sources of error are <i>metastable decomposition</i>: there is an
appearance of a lower mass tailing that can affect mass accuracy. The Einzel
lens, used to focus secondary ion beam and increase sensitivity has two modes
of operation that depend on polarity, with one mode being same polarity as
secondary ions. At certain value no more ions pass through (cut-off voltage).
This mode delays the time of flight of ions. In another mode of operationthe
polarity is reversed and causes acceleration with no cut-off effect, with a
shorter <i>t</i>. Use of a higher potential is required to get good focusing.
There is however better resolution of overlapping peaks, despite an equal
amount of tailing.
<p>
Sample thickness also affects resolution, with thicker samples producing
predictably broader peaks. Tailing is quite pronounced with thick (&gt;3 &micro;
m) targets. If inhomogeneity is &plusmn;1 &micro;m in a field strength of 3
kV/mm, then the potential difference on the target surface varies &plusmn;3 V,
which should be easily compensated in the reflectron. So peak broadening may be
the result of another mechanism.
<p>
MCP detectors can limit resolution as well. The microchannels have a 7&deg;
angle to the normal of the plate surface. <q>Illumination</q> of ions
penetrating the channel (bombardment to start a cascade) is a function of the
penetration depth: it drops by 50% at a depth of <i>d</i> = 0.4 <i>D</i>/&beta;,
 where <i>D</i> is the channel diameter and &beta; is the angle to the normal.
Where <i>D</i> is typically 10 &micro;m, <i>d</i> = 40 &micro;m, which is 17
ppm if the total flight path is 2.3 m as in the DIPLOMA. Since peak broadening
by the MCP affects all ions similarly, the tailing effect cannot be explained
by the MCP.
<p>
Reflectron imperfections (grids, fringing fields) could cause non-ideal initial
energy compensation. Peak shapes were compared on machines with the same ToF
components (reflectors, detectors, data acquisition systems) but with different
ionization techniques. With PDMS and SIMS, both ion desorption techniques, peak
tailing was clearly observed. But with ESI-MS there is no tailing. Thus
ionization and not instrument system factors affect this.

<p>
Molecules in PDMS can acquire charge prior to desorption (preformed ions),
or at the moment of desorption (desorption/ionization), or in the gas
phase.  In the latter case, one mechanism of gas phase ionization is
a reaction between unionized and ionized molecules with a transfer of
charge.  In another mechanism large non-covalently bound aggregates
of molecules desorb (<q>chunk ejection</q>) with chunks internally
excited and undergoing metastable decomposition.  A time delay occurs
in these events and there is less kinetic energy than with desorption
of pre-formed or co-formed ions.  The kinetic energy is lower than the
acceleration potential, and thus there is an energy deficit.  For certain
ions (H<sup>+</sup>, C<sup>+</sup>, CH<sup>+</sup>, even
C<sub>60</sub><sup>+</sup>) there is no real observed deficit.  However
about half of all molecular ions, specific fragments including immonium
ions, there is a deficit in the tens of eV.  This depended on the target
surface:  there was a greater deficit for <q>fluffy</q> samples
typically produced by an electrospray than for smooth or crystalline
target.  Also protonated molecules showed the deficit more than
sodiated (MNa<sup>+</sup>) molecules.  With SIMS and PDMS, the ionization
in the gas phase may be dominant, and so tailing is likely.
<p>
Consider that the flow of ions having the same <i>m/z</i>
is expressed as a current <i>I</i> with an exponential time decay of
formation:  <i>I</i>(<i>t</i>) = <i>I</i><sub>0</sub> exp (&minus;<i>t</i>/
&alpha;).  In the absence of an energy deficit is a Gaussian with a
dispersion &sigma;<sub>0</sub>.  The shape of the <q>real</q> ion
peak is described by <i>F</i>(<i>z</i>) in the spectrum in the presence
of a deficit is a convolution of the Gaussian with the exponent:
<p style="text-align:center;" class="eqnserif" >
<i>F</i>(<i>z</i>) =
<span style="font-size:150%;vertical-align:middle;">&int;</span><sub>0</sub><sup
style="vertical-align:1em;margin-left:-0.5em;">&infin;</sup>
e<sup>&minus;1/&alpha;</sup> e<sup>&minus;(<i>t &minus; z</i>)<sup>2</sup>/
2&sigma;</sup> <i>dt</i>  [1]
<p>
The factor &alpha; represents the time of ion formation and the average
energy deficit.  &alpha;=0 when the ions are promptly ejected from the
target surface.  When &alpha;=&sigma;<sub>0</sub>, the centroid
<i>C</i><sub>0</sub> and the position <i>Z</i><sub>0</sub> of the peak
is shifted toward later times.  The centroid is actually <i>C</i><sub>0</sub> =
&alpha; and the peak dispersion will be:
<p style="text-align:center;" class="eqnserif">
&sigma;<sub>&alpha;</sub><sup>2</sup> = &sigma;<sub>0</sub><sup>2</sup> +
 2 &alpha;<sup>2</sup> [2]
<p>
Since tailing occurs, <i>C</i><sub>0</sub> &gt; <i>Z</i><sub>0</sub> for
any &alpha;.   If <i>C</i><sub>0</sub> = &alpha; = 1, then <i>Z</i><sub>0</sub> =
0.7 (units of &sigma;<sub>0</sub>).  In this event it is better to select
the peak height rather than centroid in fixing the time (<i>m/z</i>), but
the peak height also moves with &alpha;.
<p>
When peak position shift can be determined, mass accuracy is improved.
Consider the algorithm:
<ol>
<li>First determine the peak centroid on a TOF scale (<i>C</i><sub>TOF</sub>)
and the peak dispersion &sigma;<sub>&alpha;</sub><sup>2</sup>.
<li>If &sigma;<sub>0</sub><sup>2</sup> is known, find &alpha; using eqn [1] above.
<li>Calculate <i>C</i><sub>0</sub> from <i>C</i><sub>0</sub> = &alpha;,
and then subtract it from <i>C</i><sub>TOF</sub>.  This will give the
<q>true</q> peak position on the TOF scale.
</ol>
<p>
A problem occurs with overlapping peaks, finding centroids and dispersions.
In simulating a number of peak shapes for different &alpha; (and therefore
<i>Z</i><sub>0</sub>), the FWHM was compared to an ideal peak with
&alpha;=<i>Z</i><sub>0</sub>=0.  The width of the ideal peak is
FWHM<sub>0</sub>=2.35&sigma;<sub>0</sub>.  In the region 0&lt;<i>Z</i><sub>0</sub>&lt;1.5,
<p style="text-align:center;" class="eqnserif">
FWHM = FWHM<sub>0</sub> (1 + 0.6 <i>Z</i><sub>0</sub><sup>2</sup>)  [3]
<p>
If the FWHM &lt; FWHM<sub>0</sub> the correction value <i>Z</i><sub>0</sub>
is subtracted from the TOF of the peak to get the true position.
FWHM<sub>0</sub> and &sigma;<sub>0</sub> are determined from known resolving
power of instrument.
<p>
An linear approximation to the
<!-- ============================================================= -->
<p class="tags">
proline effect, CID, peptide fragmentation, FD
<p class="ref">
 <span class="authors">
Vaisar T, Urban J
 </span>
 <span class="year">
(1996)
 </span>
 <span class="jname">
J. Mass Spectrom.
 </span>
 <span class="jvol">
31,
 </span>
 <span class="pages">
1185-1187.
 </span>
<p class="reftitl">
Probing the Proline Effect in CID of Protonated Peptides
<p class="Para1">
<b>LETTER</b>.  Protonated peptides with Pro show interesting CID spectra.
There are abundant y-type ions arising from cleavage of amide bond
on the N-terminal side of proline.  This has been reported in high energy
collisions on sector instruments, low-energy collisions, and on hybrids.
Seen in multiply charged protein ions.  There is a high basicity of Pro
relative other ions.
<p>
Using a peptide with structure AVxLG where x=N-alkylated amino acid,
we looked at CID spectra.  All peptides were synthetic and RP-HPLC purified.
Spectra were obtained on triple quad (Fisons VG Quattro) with ESI.
The argon pressure was set so there is 20% ion transmittance at 10 eV frame
collision energy.  With peptide [AVALG+H]<sup>+</sup>, a b<sub>4</sub> ion is prominent,
along with complements b<sub>2</sub>/y<sub>3</sub> and b<sub>3</sub>/y<sub>2</sub>
in lower abundance.  With [AVPLG+H]<sup>+</sup>, the y<sub>3</sub> ion is dominant,
reflecting V-P bond cleavage and the greater proton affinity for proline.
For peptide [AV(NMe-Ala)LG+H]<sup>+</sup>, there is a dominant b<sub>3</sub> ion (93% of TIC of fragments,
similar to the Pro-containing peptide.  [AV(Pip)LG+H]<sup>+</sup> (peptide
with pipecolinic acid, a homologue of Pro with 6-member ring) shows an
abundant b<sub>3</sub> ion similar to NMe-Ala peptide.
[AV(Sar)LG+H]<sup>+</sup> is the sarcosine-containing peptide, showing
y<sub>3</sub> (63% of TIC fragment current) and b<sub>3</sub> (22%),
with minor b<sub>2</sub> and b<sub>4</sub> ions.  Alkylation of the amide
N atom has a clear effect.  It clearly produces b3 and y3 ions (as 3rd
residue), while an alkyl side chain (off the &alpha;-carbon) promotes
b3 ion formation.  The sarcosine peptide b3/y3 ratio is 0.4, while
NMe-Ala shows a 26.5 value (b3 dominates), Pip is 5.2, and Pro it is 0.03.
When merely Ala, the ratio is 0.9.   The dominance of a <i>b</i> or <i>y</i> ion will
depend largely on the proton affinity of the N-terminal residue on the
C-terminal fragment.
<p>
The formation of b ions will be a key to explaining the <q>proline effect</q>.
It is proposed they form an oxazolone protonated ring rather than an linear
acylium.  Formation of a <i>b</i> ion with a ring product on C-terminal side
of Pro would be highly strained, so the absence of such ions is not unexpected.
It would be bicyclic.  A b3 ion in the peptides used here would be unlike
for Pro as 3rd residue.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Lui M, Tempst P, Erdjument-Bromage H
 </span>
 <span class="year">
(1996)
 </span>
 <span class="jname">
Anal. Biochem.
 </span>
 <span class="jvol">
241,
 </span>
 <span class="pages">
156-166.
 </span>
<p class="reftitl">
Methodical Analysis of Protein-Nitrocellulose Interactions to Design
A Refined Digestion Protocol
<p class="Para1">
It is often the case that electroblotted proteins from 1-D and 2-D gels
contain N-terminal blocking which must be overcome by the use of <q>internal
sequencing</q> techniques that involve in situ micropreparative digestion.
Unfortunately peptide recovery is a problem (70% for trypsin, 50% for
endoproteinase Lys-C on average).  Aebersold's group has reported that
alkaline pH and organic solvents promote elution.  Detergents were tried by
others.
<p>
In the first experiments elution of intact proteins was tried with a variety
of organic solvents and detergents.  Standard marker proteins were
electroblotted to NC membranes:  lysozyme 14 kDa, Hb 16 kDa, ovalbumin 45 kDa,
G6PD 55 kDa, &beta;-galactoside 116 kDa.  Proteins were radioiodinated
and a few nanograms of labeled protein mixed with 1 &micro;g unlabeled protein,
run on gels, blotted to NC, then bands excised and counted, stored at
&minus;70&deg;.
<p>
Several different organic solvents were tried (11, with up to 40% concentration).
None of the proteins dissociated.  pH extremes (20 mM KOH or 20% TFA
in water) also did not help.  In a 100 mM ammonium bicarbonate (pH 8.5)
buffer, MeCN did not elute large proteins until reaching 40% concentration.
Even more protein was released if the pH was increased to 10.5 and 12.0
(2- and 3-fold more, respectively, for a 40% and 60% release on average).
EtOH, isopropanol, and DMSO do not promote elution from NC under mild
alkaline conditions.  With TFA, at 10% concentration at 37&deg; and
40% MeCN provided a 25% elution.  Use of higher temperature and higher
TFA concentrations gave a false impression of improved elution of
radioactivity, which actually was the result of protein degradation.
<p>
Several organic bases were then tried (since acid promotes degradation).
Triethanolamine, triisopropanolamine, pyridine (PYRI),  pyrimidine, morpholine,
<i>N</i>-ethylmorpholine, piperazine, cyclen, pyrrolidine carbonitrile,
and piperidine (PIP).  PYRI at 40% (pH 8.5) and PIP at 20% (pH 13.10)
were excellent at 37&deg; and 65&deg; in elution proteins.  In combination
with 40% MeCN and 1% PIP alone, near quantitative recovery was obtained
(90%);  even with 0.1% PIP/40% MeCN, a 65-70% recovery was better than
any other combination.  pH alone did not account for the effect since
a 10 mM KOH with MeCN was less effective, despite the higher pH.
<p>
Different detergents were tested at three different alkaline pH (8.5 with
200 mM TrisHCl, 10.0 with 200 mM CAPS, and 11.5 with 10 mM KOH) at 37&deg;
and 65&deg;.  At pH 8.5, only 3% CTAB among ionic
detergents (SDS, sarkosyl, sodium deoxycholate, all 3%) yielded the
best recovery from NC;  the zwitterionics (Zwit 3-16 at 0.3 and 3%,
CHAPS at 3% did not do well, LDAO at 3% and 0.3%, Empigen at 0.3 and 3%) gave respectable
recoveries.  Among nonionics, only Tween and Triton were sufficiently good
to warrant a further look (Genapol allowed 40-50% recovery, followed
by Pluronic at 20-30%, while MEGA-10 and the glucoside detergents were poor).
pH did not really promote greater elution except in the case of Triton
and Zwittergent;  with Triton it helped, but with Zwittergent elution
was reduced.
<p>
Protein size may have been a factor.  Hb, lysozyme, and ovalbumin eluted
better with a lower 0.3% detergent than the larger &beta;-galactosidase
and G6PD proteins.
<p>
In general combinations of detergents and organic solvents promote dissociation
of bound material, but in the case of NC membranes, recoveries were
reduced by the combined use (as much as 15-30%).  For NC membranes, it
recommended that digestions be done in the absence of MeCN.
<p>
Zwittergent 3-16 is a sulfobetaine (N-hexadecyl-N,N-dimethyl-3-ammonia-1-propanesulfonate)
composed of zwitterionic head and unbranched 16-carbon tail, the latter
of which may be important.  The N-dodecyl form resulted in 10% less recovery,
and the N-octyl (Zwittergent 3-08) did not promote extraction at all.  Use
of the detergent is compatible with RP HPLC: it does not absorb at 214 nm
and does not affect peak height or resolution.  It elutes very late into
a gradient.
<p>
For in situ digestion, the membrane area should be at a minimum.  Prior to
trypsin addition, membranes should be blocked with nonprotein material
(0.5% PVP-40).  But even without the blocking agent use of Zwittergent
3-16 helps to recovery considerably.
<p>
The large proteins at 55 and 116 kDa were used on electroblotted membranes
and digested without prior blocking using different detergents.  80-90%
of radioactivity was recovered after 2 h of 0.2 &micro;g trypsin in
100 mM ammonium bicarbonate with 1% Zwittergent 3-16;  lower recoveries
(50-70%) were seen with Triton X or Tween80 at 1%.  Endoproteinase Lys-C
produces larger fragments, but the recovery with Zwittergent 3-16 was the same
but much lower with nonionics.
<!-- ============================================================= -->
<p class="tags">
CID, FD
<p class="ref">
 <span class="authors">
Gonzalez J, Besada V, Garay H, Reyes O, Padront G, Tambara Y, Takaot T,
Shimonishi Y
 </span>
 <span class="year">
(1996)
 </span>
 <span class="jname">
J. Mass Spectrom.
 </span>
 <span class="jvol">
31,
 </span>
 <span class="pages">
150-158.
 </span>
<p class="reftitl">
Effect of the Position of a Basic Amino Acid on <i>C</i>-Terminal Rearrangement
of Protonated Peptides Upon Collision-induced Dissociation
<p class="Para1">
Peptides are fragmented into ions that are related to their sequence. N-terminal
ions are classified as <i>a, b, c, d,</i> and C-terminal ions as <i>x, y, z, v,
w</i>.  The ions <i>d</i>, <i>v</i>, and <i>w</i> series specify side chain
fragmentation seen with high energy CID.  Interesting is that N-terminal ions
are prominent when there is an N-terminal basic amino acid, and if at the
C-terminus, C-terminus ions are more prominent.  When basic residues in middle
of peptide, the CID spectrum is more complex.  In metal-cationized and
protonated peptides, the C-terminal amino acid is lost by transfer of the
C-terminal &alpha;-hydroxy moiety to the carbonyl carbon of the penultimate
amino acid residue.  The rearrangement is [<i>b</i><sub><i>n</i>-1</sub> +
H<sub>2</sub>O].  Labeling peptides by acid-catalyzed isotope exchange with
<sup>18</sup>O, the mechanism has been studied and is promoted by a basic amino
acid (Arg) and depends on gas phase conformation.
<p>
A JEOL JMS-HX/HX110A four-sector tandem MS with FAB source, collision cell,
and photodiode array detector was used.  0.5 nmol peptides in 50% MeCN
are put on stainless steel probe tip, dried, mixed with glycerol and acidified
with 0.1 M HCl.  Positive ions are generated with Xe atoms at 4000 V,
accelerated in 10,000 V field.  Ions might be filtered by MS1 and accelerated
at 8 kV into the collision cell before MS2.  He atoms form the collision gas
and are adjusted to reduce parent ion abundance in spectra by half.  CID
product ions are scanned by both <i>E</i> (electric) and <i>B</i> (magnetic) fields such
that the <i>B/E</i> ratio is constant (<q><i>B/E</i>-linked scan mode</q>).
<p>
Peptides were labeled with <sup>18</sup>O.  Product ion spectra of a partially
<sup>18</sup>O-labeled peptide Val<sup>148</sup>-Ala-Ile-Val-Glu-Arg-Tyr-Lys<sup>155</sup>
was obtained during API digest of outer membrane of <i>N. meningitidis</i>.
Three groups of ions are seen:
<ol>
<li>C-terminal ion show doublet peaks with similar abundances to that of
parent [<i>z</i><sub>3</sub>, <i>y</i><sub>3</sub>, and <i>wa</i><sub>4</sub>]
<li>N-terminal ions show peaks with natural isotopic distributions from loss
of <sup>18</sup>O at C-terminal COOH [<i>a</i><sub>7</sub> and <i>b</i><sub>7</sub>]
<li>ion with isotopic enrichment to C-terminal ions and containing <sup>18</sup>O, but
with lower content, which is rearrangement ion [<i>b</i><sub>7</sub> + H<sub>2</sub>O]
</ol>
<p>A mechanism to explain the product spectra has been proposed.  Precursor
ions are [M+1]<sup>+</sup> and [M+3]<sup>+</sup> (<sup>16</sup>O and <sup>18</sup>O isotopic forms,
respectively).  In acid-catalyzed labeling, the <sup>18</sup>O isotope can become either
the hydroxyl oxygen or carbonyl oxygen of the COOH moiety on the end terminal
residue.  In all cases, the hydroxyl oxygen creates an electrophilic bond with
the carbonyl carbon of the previous residue to make 5-centered ring.  This
ring fragments (C-O and NH=CH-R are formed, the last an immonium) to create
a [<i>b</i><sub>n-1</sub> + H<sub>2</sub>O] ion.  Two rearrangement ions are
seen, differing by 2 Da, the difference between the oxygen isotopes.  Note
that only one of three routes in the mechanism puts the <sup>18</sup>O atom on the product
ion:  the population of <sup>18</sup>O-labeled product ions should be lower in C-terminal
and parent ions, but higher than for N-terminal derived ions.
<table>
<tr><th>Peptide
   <th> [<i>b</i><sub><i>n</i> &minus; 1</sub> + H<sub>2</sub>O]/<i>b<sub>n</sub></i> &minus; 1
<tr><td>QGIEF <td>0.0<sup>a</sup>
<tr><td>HGIEF <td>0.4
<tr><td>KGIEF <td>0.9
<tr><td>RGIEF <td>5.6
<tr><td>RASVF <td>6.0
<tr><td>ARSVF <td>6.0
<tr><td>ASRVF <td>5.0
<tr><td>ASVRF <td>9.0
<tr><td>ASVFR <td>&mdash;<sup>a</sup>
<tr><td>ASVKF <td>2.1
<tr><td>SVRF <td>9.8
<tr><td>EPQRF <td>6.8
<tr><td>VAIVERY <td>9.5
<tr><td>DGVVSRL <td>12.2
<tr><td colspan="2"><sup>a</sup> These not detected
</table>
<p>
This cyclization and loss of the C-terminal residue happens significantly when
Arg is the residue. When Lys is substituted, the rearrangement ion is present
at less abundance; it is less basic than Arg. Four synthetic peptides with
N-terminal residue being Gln, His, Lys, and Arg were analyzed for product ion
spectra in the <i>m/z</i> 350-500 range. The abundance ratio [<i>b</i><sub>
4</sub> + H<sub>2</sub>O]/<i>b</i><sub>4</sub> increases when basicity is
increased of the N-terminal amino acid (see table above). This ratio was
significantly larger for Lys-peptide than Gln-peptide. Note that residual
masses are identical since Lys and Gln are isobaric.
<p>
The effect of position was then assessed.  All peptides had the same composition,
but the position of Arg was altered.  All peptides produce the rearrangement ion
[<i>b</i><sub>4</sub> + H<sub>2</sub>O] except when  Arg is the C-terminal residue,
a reasonable finding since it usually produces C-terminal and not N-terminal ions.
The peptide with Arg at <i>n</i>&minus;1 was the most likely to show rearrangement.
Intensity ratios [<i>b</i><sub>n-1</sub>+H<sub>2</sub>O]/<i>b</i><sub>n-1</sub>
for several chymotryptic peptides from recombinant proteins al with Arg at
penultimate.  A table of values shows clearly that basicity affects the ability
to induce rearrangement.
<p>
It had been reported that metal-cationized peptides (but not protonated) that lack
a basic amino acids show rearrangement products, suggesting that alkali metals
interact with the COO anion at the C-terminus.  Its presence facilitates nucleophilic
attack of the -OH group on C-terminal -COOH to the carbonyl of the <i>n</i>&minus;1
residue.  But with Arg, the protonated guanidino might form a salt bridge with
C-terminal -COO anion in vacuo letting it interact with the penultimate carbonyl
as well.  A mechanism is shown similar to the metal-cationized mechanism by
which Arg side chain facilitates interaction between C-terminal -COO and the
carbonyl carbon, with loss of the CO and immonium ions making up the C-terminal
amino acid.
<p>
A previous paper suggested that low-energy collision produced these ions and that
it resulted from metastable decomposition.  To further verify the mechanism,
two types of peptides were measured by B/E-linked scan with and w/o collision
gase on MS1 and MS/MS.  Spectrum without gas represents metastable ions and show
rearrangement ions [<i>b</i><sub>6</sub>+H<sub>2</sub>O] in highest abundance.  When gas is added, rearrangement
abundance is lower, and product ions <i>da</i><sub>4</sub> and <i>db</i><sub>4</sub>
appear in spectrum.  The latter occur from loss of larger and smaller substituents
attached to &beta;-arbon of 4th amino acid (Ile), resp., and characteristic of
high energy collision.  Rearrangement ion was nearly suppressed at 2 kV collision
energy.  High energy collision with a cell floated at several kilovolts excludes
ions in front of and behind cell, because ions don't pass through analyzer.  Thus
rearrangement forms from metastable decomposition.  In contrast, the rearrangement
ion produced from a synthetic peptide of ASVRF is observed in the MS/MS while
most abundant in B/E-linked scan spectrum.  This is consistent with metastable
decomposition in field-free region and with CID, but in sequence-dependent manner.
<!-- ============================================================= -->
<p class="tags">
FD
<p class="ref">
 <span class="authors">
Haller I, Mirza UA, Chait BT
 </span>
 <span class="year">
(1996)
 </span>
 <span class="jname">
J. Am. Soc. Mass Spectrom.
 </span>
 <span class="jvol">
7,
 </span>
 <span class="pages">
677-681.
 </span>
<p class="reftitl">
Collision Induced Decomposition of Peptides:  Choice of Collision Parameters
<p class="Para1">
In a triple quad MS with typical conditions (Ar collision gas, &lt; 100 V ion
acceleration) peptide bond scission usually dominates fragmentation pathways. 2+
peptides consist mostly of identifiable <i>b</i> and <i>y</i>-series ions.
Complicating the interpretation of spectra are other possible fragmentation sites
and neutral losses of water and ammonia.
<table>
<caption>The amino acid sequence, mass, and the obtained fitting parameters
for the peptides used</caption>
<tr><th rowspan="2">Amino acid sequence
  <th rowspan="2">Molecular mass
 <th colspan="3">V<sub>opt</sub> (V) at (mtorr)
 <th colspan="3">Slope <i>b</i> (V<sup>&minus;1</sup>) at (mtorr)
  	<tr><th>2.7<th>1.5<th>0.88<th>2.7<th>1.5<th>0.88
<tr><td>TSK<td>334.4<td>6.8<td>7.23<td>11.3<td>0.83<td>0.71<td>0.30
<tr><td>PFGK<td>447.5<td>9.82<td>10.3<td>13.9<td>0.74<td>0.69<td>0.48
<tr><td>VESSK<td>548.6<td>10.0<td>10.7<td>15.2<td>0.76<td>0.71<td>0.31
<tr><td>SIGSLAK<td>674.8<td>8.9<td>9.4<td>13.6<td>0.64<td>0.66<td>0.52
<tr><td>VHLTP VEK<td>922.1<td>17.8<td>19.4<td>30.6<td>0.50<td>0.50<td>0.13
<tr><td>KRQHP GKR<td>1006.2<td>27.3<td>30.5<td>45.7<td>0.40<td>0.49<td>0.23
<tr><td>PHPFH FFVYK<td>1318.5<td>21.1<td>23.6<td>33.9<td>0.47<td>0.42<td>0.34
<tr><td>QATVG DVNTD RPGLL DLK<td>1912.1<td>38.3<td>44.6<td>71.2<td>0.47<td>0.43<td>0.12
<tr><td>GGR<td>288.3<td>5.17<td>5.98<td>8.0<td>0.55<td>0.75<td>0.42
<tr><td>GGYR<td>451.5<td>6.90<td>7.91<td>11.1<td>0.44<td>0.51<td>0.34
<tr><td>YIGSR<td>594.7<td>8.27<td>8.98<td>13.2<td>0.73<td>0.65<td>0.41
<tr><td>HLGLA R<td>665.8<td>13.7<td>14.7<td>19.8<td>0.59<td>0.69<td>0.42
<tr><td>ASHLG LAR<td>824.0<td>15.4<td>16.6<td>23.7<td>0.73<td>0.65<td>0.41
<tr><td>PPGFS PFR<td>904.1<td>18.4<td>19.4<td>27.2<td>0.59<td>0.59<td>0.25
<tr><td>RPPGH SPFR<td>1060.3<td>26.2<td>29.4<td>46.1<td>0.40<td>0.38<td>0.14
<tr><td>HSDAV FTDNY TR<td>1425.5<td>27.6<td>31.5<td>47.5<td>0.56<td>0.51<td>0.13
<tr><td>EGVND NEEGH FSAR<td>1570.6<td>27.9<td>30.4<td>53.5<td>0.30<td>0.39<td>0.11
<tr><td>ADSGE GDFLA EGGGV R<td>1536.6<td>23.8<td>27.7<td>41.9<td>0.50<td>0.51<td>0.24
</table>
<p>
The peptide ASHLGLAR was set up in the 2+ state and tested at three different
collision energies with a gas pressure of 1.5 mtorr.  At 12 V collision cell
offset, the parent ion is seen with highest relative intensity at <i>m/z</i>
412, but so is a <i>y</i><sub>6</sub><sup>2+</sup> at <i>m/z</i> 318 and a
<i>y</i><sub>5</sub> at <i>m/z</i> 514.  Note that <i>E</i><sub>lab</sub>
(energy at laboratory frame) is actually 24 eV, since the precursor ion is a +2
state.  At <i>V</i><sub>c</sub> = 17.0 V (<i>E</i><sub>lab</sub> = 34 eV), seven
<i>y</i> ions (1, 4, 5, 6, 7, and the 2+ ions of 6 and 7) are identified, along
with a <i>b</i><sub>3</sub> and <i>a</i><sub>3</sub> ion. The precursor ion
still has the highest relative intensity.  At <i>V</i><sub>c</sub> = 30.0 V
(<i>E</i><sub>lab</sub> = 60 eV), the precursor had been fully fragmented, but
only three <i>y</i> ions could be seen and a low intensity <i>b</i><sub>3</sub>
ion.  The rest of the ions were low <i>m/z</i> peaks in the 100-300 range with
no assignment, usually the result of secondary fragmentations of <i>b</i> and
<i>y</i> ions. Of the three collision cell V offsets, <i>V</i><sub>c</sub> =
17.0 V provided the most (sequence) information, with 8 amino acids identified.
<p>

Plots of the abundance of product ions versus collision cell offset voltage can
be done to assess optimal collision cell offsets.  This was done for four
product ions, <i>m/z</i> 109, <i>y</i><sub>4</sub>, <i>y</i><sub>5</sub>, and
<i>y</i><sub>4</sub><sup>2+</sup>, at collision offsets of <i>V</i><sub>c</sub>
= 2, 6, 8, 10, 12, 13, 16, 17, 18, 19, 20, 21, 23, 25, and 30 V.  The
[M+2H]<sup>2+</sup> abundance of the precursor was also plotted in one of the plots
to track the loss of parent.  In the three y ion plots, a maximum is seen,
and the maxima generally occur at the same voltage.  The m/z 109 is representative
of the histidine immonium ion and the result of multiple fragmentations.  Its
plot shows no maximum but rather a steadily increasing abundance curve up to
30 V.
<p>

To find the optimum collision offset, it is observed that the maxima in product 
ion peaks occur at a voltage where the sum of intensities of three strongest 
fragment ions is 1.5 times the precursor ion intensity.  The conversion ratio 
<i>R</i> = sum of the abundances of the three strongest fragment ions in the CID 
spectrum divided by remaining precursor ion abundance. <i>V</i><sub>opt</sub> is 
the offset at which R = 1.5.  Plot log <i>R</i> vs. <i>V</i><sub>c</sub> and a 
straight line is approximated.  From <i>R</i> = 1.5 on the plot, find 
<i>V</i><sub>opt</sub> = <i>V</i><sub>c</sub>.  Other plots make use of absolute 
intensity data, but the conversion ratio diminishes the effects that arise from 
fluctuations in the electrospray ionization yield.  The observed exponential
dependence indicates decomposition reaction is driven by a build-up (via
multiple collisions) of internal energy that is approximately proportional to 
ion kinetic energy.  Other factors contributing are collision cross section and
time available for decomposition.  Present experiments results do not allow 
evaluation of relative contributions of these other factors. <p><b>Peptide Mass 
Effects</b>.  Peptides from 3-18 residues listed in the table above had their 
CID spectra checked for 2+ state precursors with <i>V</i><sub>c</sub> = 
<i>V</i><sub>opt</sub> (or at least close to it). These peptides are 
representative of typical tryptic peptide sequences, having K and R residues on 
COOH terminus (see first eight entries of table for K and the rest for R 
residues).  Spectra were obtained for three different <i>V</i><sub>c</sub> that 
bounded <i>V</i><sub>opt</sub>, and at three different pressures (0.88, 1.5, and 
2.7 mtorr); thus 9 spectra for each peptide.  In all cases, the conversion ratio 
<i>R</i> varied exponentially with <i>V</i><sub>c</sub> and the <i>R</i> = 1.5
criterion was in agreement with the goal of balancing peaks arising from single 
vs. multiple cleavage reactions.  A plot of log conversion ratio <i>R</i> from 
0.4 to 5.0 vs. <i>V</i><sub>c</sub> at the three different pressures is shown 
for one of the 18 peptides (HLGLAR) and shows how conversion ratio is also 
dependent upon gas pressure.  Data for each pressure were fitted to <i>R</i> = 
1.5 &times; exp[<i>b</i>(<i>V</i><sub>c</sub> &minus; <i>V</i><sub>opt</sub>)]. 
The fitting parameters <i>V</i><sub>opt</sub> and <i>b</i> are shown in the 
table above.
<p>
V<sub>opt</sub> shows a strong linear correlation with the peptide size, and this is indepedent
of pressure:  that is, Vope increases linearly with molecular mass for each of the
pressures, and slope is inversely dependent upon pressure.  Correlation coefficients
of 0.87, 0.89, and 0.83 were found for the pressures of 2.7, 1.5, and 0.88 mtorr,
respectively.  Thus higher collision voltage offsets are required as the gas pressure
goes down.
<p>
No dependence was found for the COOH terminal residue.  Deviations that arise due
to pressure differences are not random errors.  Peptides that contain more than
one R and/or K residue do require more energy for fragmentation than those with
excess acidic residues, which is consistent with prior observations.  Factors
affecting fragmentation include center-of-mass energy available for excitation,
excitation cross-section, and degrees of freedom in the excited peptide.  Thus
a observation of a simple linear dependence of V<sub>opt</sub> on molecular mass was unexpected.
These observations should be tested in triple quads of different design.
<p>
One implication for LC-MS/MS-based protein sequencing is the use of multiple
collision energy profiles based on mass and charge of peptide ion.  Once precursor
mass and charge are assessed in MS mode, the switch to MS/MS mode will set the
proper profile.  The following three steps are proposed:
<ol>
<li>use one or two scans in MS mode (Ar is in collision cell, and offset voltage
set to 0) to get precursor mass and charge at onset of HPLC peak
<li>software monitoring sets collision voltage to V<sub>opt</sub> after computation of the
value
<li>CID spectra are averaged until the end of peak elution
</ol>
<p>
The first two steps must be completed within seconds to get good MS/MS spectra.
<!-- ============================================================= -->
<p class="tags">
selenocysteine
<p class="ref">
 <span class="authors">
Stadtman TC
 </span>
 <span class="year">
(1996)
 </span>
 <span class="jname">
Annu. Rev. Biochemistry
 </span>
 <span class="jvol">
65,
 </span>
 <span class="pages">
83-100.
 </span>
<p class="reftitl">
Selenocysteine
<p class="Para1">
Selenocysteine is not recognized as the 21st amino acid.  August B&ouml;ck and 
colleagues have found that <i>E coli</i> mutants were defective in ability to
synthesize Se-dependent formate dehydrogenases and these revealed four gene products
that involved generation and insertion of SeCys into proteins.  One product is of a
unique tRNA and two others are new enzymes making SeCys- and Ser-tRNA.  Another
is a elongation factor recognizing SeCys-tRNA.  Selenium-containing enzymes play
an important role in survival.  
<ul>
<li>Glutathione peroxidases scavenge deleterious peroxides.
<li>Deiodinase enzymes synthesize and maintain optimum levels of thyroid hormones.
</ul>
<p>
Genes in <i>E coli</i> <i>sel</i>A, <i>sel</i>B, <i>sel</i>C, and <i>sel</i>D
have been identified.  Seleno-Cys is actually formed on a tRNA:  
tRNA<sup>Sec</sup>, product of <i>sel</i>C, is esterfied with amino acid Ser
in the normal fashion.  The <i>sel</i>A product is a PLP-dependent enzyme
that converts the Ser-tRNA<sup>Sec</sup> to produce aminoacrylyl-tRNA<sup>Sec</sup>.
The selenocysteyl-tRNA<sup>Sec</sup> forms when selenophosphate reacts with 
the double bond in the aminoacrylyl derivative.  Selenophosphate is produced
by the reaction of hydrogen selenide (H<sub>2</sub>Se) with ATP, mediated
by the <i>sel</i>D product.  It must be noted that the selenocysteine is
built on the tRNA:  while it may exist freely in the protoplasm (it does),
it cannot be ligated to tRNA directly.  The L-serine must be ligated to tRNA
from which it is synthesized.
<p>
For eukaryotes, the mechanism has not been fully established.  There is a 
tRNA<sup>Sec</sup> analogous to that in <i>E coli</i>, with esterification of
Ser, and conversion to the selenocysteyl form.  However, an 
<i>O</i>-phosphoseryl-tRNA<sup>Sec</sup> is also present, although there is no
evidence of it having an intermediate role.  A homolog for <i>sel</i>D has been
discovered, so the synthesis of selenophosphate is more likely.
<p>
It has been observed that in <i>sel</i>D mutants, free selenocysteine is
incorporated into proteins, but only because it is ligated to tRNA<sup>Cys</sup>
and not tRNA<sup>Sec</sup>, its special anticodon.
<p>
The function of the remaining gene <i>sel</i>B was determined when its mutants 
were found to accumulate selenocysteyl-tRNA<sup>Sec</sup>, indicating that they 
were not incorporated into protein.  It is likely a translation factor required 
to transport the charged tRNA to the ribosome, and thus a functional homolog of 
EF-Tu.  Both RNA protection studies and direct binding assays confirm this 
special affinity.  SELB has little to no binding for Ser-tRNA<sup>Sec</sup>. 
SELB mediates in part the deacylation of the SeC-tRNA<sup>Sec</sup>, which 
decreases markedly under alkaline conditions; deacylation of 
Ser-tRNA<sup>SeC</sup> is not affected.
<!-- ============================================================= -->
<p class="tags">
salt clusters, ESI-MS, FD
<p class="ref">
 <span class="authors">
Zhou S, Hamburger M
 </span>
 <span class="year">
(1996)
 </span>
 <span class="jname">
Rapid Comm. Mass Spectrom.
 </span>
 <span class="jvol">
10,
 </span>
 <span class="pages">
797-800.
 </span>
<p class="reftitl">
Formation of Sodium Cluster Ions in Electrospray Mass Spectrometry
<p class="Para1">
Two different models have been proposed for how solvent evaporation and 
formation of charged droplets which grow smaller and have higher charge to mass 
ratios.  Iribarne and Thomson have proposed the direct ion evaporation 
(emission) theory, while Dole et al with support from R&ouml;llgen have proosed 
the charged residue model.  Kebarle and coworkers have proposed a modified 
charge residue model called the single ion in droplet theory (SIDT).  The SIDT 
can be subjected to experimental testing.  When a nonvolatile ionic salt in the 
solvent is sufficiently concentrated, ion separation from counterion in the 
evaporative process is incomplete. Unseparated paired electrolyte ions would 
remain in the final droplet together with one excess ion, and this leads to a 
formation of a series of ion clusters. Sodium ion clusters from sodium salt 
solutions with concentrations higher than 0.25 mM under electrospray conditions 
occurs.
<p>
At low sodium ion salt concentrations (50 &micro;M), the dominant ions species 
are clusters of Na<sup>+</sup>(CH<sub>3</sub>CN)<sub><i>n</i></sub>.  At a cone 
voltage of 30-35 V, there are no naked Na<sup>+</sup> ions.  At cone voltages 
above 55 V the Na<sup>+</sup>(CH<sub>3</sub>CN)<sub><i>n</i></sub> clusters 
dissociate. As [sodium acetate] &gt; 250 &micro;M, ion clusters with the 
composition 
Na<sup>+</sup>(CH<sub>3</sub>COONa)<sub><i>n</i>1</sub>(HCOONa)<sub><i>n</i>2</sub> 
appear.  When sodium formate alone, the dominant cluster is 
Na<sup>+</sup>(HCOONa)<sub><i>n</i></sub>, where <i>n</i> ranges from 1 to 20. 
At low sodium salt concentration (&lt; 250 &micro;M), Na<sup>+</sup> would be 
separated from their counter ions, and no or few unseparated ion pairs would be 
in the droplets.  There is a critical concentration at which cluster ions 
appear, generally at around 100 &micro;M.  Kebarle et al have reported a plateau 
for total droplet current at electrolyte concentrations &gt; 100 &micro;M. 
Another group showed formation of ion-pairs between analyte and counterion 
responsible for changes in charge state distribution at analyte concentrations 
&gt; 100 &micro;M.  Electrophoretic migration leads to ion separation at the ESI 
interface and  ion migration to droplet surface.  An ion monolayer at the 
surface might be complete at about 100 &micro;M where sodium salts are dominant 
electrolyte.  This halts or decelerates further ion separation either by 
shielding electric field or by repulsion via electrostatic interaction.  In the 
droplet fission process unseparated ion pair are retained in the droplet 
together with excess ions, and this can explain formation of ion clusters.
<p>
An issue of the SIDT model considered by Kebarle and coworkers in selectivity of
gas-phase ion formation among different ion species.  Population of the different
ions at droplet surface should be determined by electrophoretic mobility and 
their concentration in bulk solution according to 
<p class="eqnserif">
<i>N<sub>i</sub></i> = (<i>V<sub>s</sub>/V<sub>i</sub></i>)<i>K<sub>i</sub>C<sub>i</sub>/</i>(1 
+ &sum;<i>K<sub>i</sub>C<sub>i</sub></i>)
<p>
where <i>N<sub>i</sub></i> is ion population at droplet surface, <i>V<sub>i</sub></i> 
is specific volume of ion particle, <i>V<sub>s</sub></i> is saturation volume of
all ions at droplet surface, <i>K<sub>i</sub></i> is ion equilibrium constant
or adsorption coefficient, and <i>C<sub>i</sub></i> is concentration of ions in 
bulk solution.
<p>
Fission of droplets is uneven and the offspring droplets are smaller than their
parent ions, and carrying a much higher charge density.  More uneven fission would 
produce droplets containing only a single ion, while ion pairs exist in original 
parent droplets.  Non-volatile sodium salts show this:  flow injection of NaCl
into 80% MeCN produce Na<sup>+</sup>(NaCl)<sub><i>n</i></sub> where <i>n</i> &gt;
4 and Na<sup>+</sup>(CH<sub>3</sub>CN)<sub><i>n</i></sub> clusters were formed.
The isotopic distribution of chlorine made these findings easily identifiable.
<p>
Stability of Na<sup>+</sup>(CH<sub>3</sub>CN)<sub><i>n</i></sub> and 
Na<sup>+</sup>(CH<sub>3</sub>CN)<sub><i>n</i></sub> were assessed by increasing the 
OV settings:  at OV = 55 V, the Na<sup>+</sup>(CH<sub>3</sub>CN)<sub><i>n</i></sub>
clusters were dissociating, while with 
Na<sup>+</sup>(CH<sub>3</sub>CN)<sub><i>n</i></sub>
clusters no dissociation was occurring significantly at OV = 135 V.
<!-- ============================================================= -->
<p class="tags">
FD
<p class="ref">
 <span class="authors">
Wang J, Ke F, Siu KW, Guevremont R
 </span>
 <span class="year">
(1996)
 </span>
 <span class="jname">
J. Mass Spectrom.
 </span>
 <span class="jvol">
31,
 </span>
 <span class="pages">
159-168.
 </span>
<p class="reftitl">
Electrospray Tandem Mass Spectrometry of Alkali Metal-containing Anionic 
Complexes
<p class="Para1">
Alkali metal adducts readily observable in FAB-MS of peptides.  Fragmentation
pathways are distinct from protonated peptides.  Interaction of especially
sodium and potassium with peptides may play important role.  Gas-phase MS/MS
might indicate how alkali metal ions associate with peptides intrinsically
without effects of solvent molecules.  Previous work with FAB-MS used matrix
of glycerol/thioglycerol saturated with X-OH or X-Cl where X is alkali metal.
ESI is better than FAB with respect to compatibility.  Study of tripeptide
in water/MeOH at pH 12 with X-OH produces several cationic X-peptide adducts.
These were [M-H+2X]<sup>+</sup> &gt; [M-2H+3X]<sup>+</sup> &gt; [M-3H+4X]<sup>+</sup>
in order of decreasing abundance and M is tripeptide and X = Li, Na, or K.
FAB-MS cannot even produce the [M-2H+3X]<sup>+</sup> and [M-3H+4X]<sup>+</sup>
ions and the first ion is low in count.  Metal ions usually coordinate with
carbonyl O atom of peptide backbone.  Also interactions with C-terminal 
carboxylate anion (-COO<sup>&minus;</sup>) and peptide N atoms.
<p><b>Experimental</b>. Peptides were available from Sigma and dissolved to 1 mM 
in 1:1 water/MeOH adjusted to pH 12 in presence of XOH (alkali hydroxides). For 
deuterium labeling tripeptides aer dissolved in 1:1 
CH<sub>3</sub>OD:D<sub>2</sub>O adjusted to pH 12 (only a few minutes for 
equilibration).  Acylation of N-terminus done with 1% acetic anhydride in 1:1 
MeOH:water for 1 h, then adjust to pH 12 with XOH. A TAGA 6000E SciEx triple 
quad (QqQ) is used with electrospray fashioned using 3 cm 33-guage (Hamilton 100 
&micro;m i.d.) stainless steel tube attached to other tube with epoxy glue and 
electropolished before use.  Adjustments to argon gas pressure are done to 
determine the number of collisions a molecule undergoes.  Collision energy is 
about 40 eV.
<p>
All tripeptides used in study have neutral side chains. Negative ion ESI-MS of 
LGG and GGV with KOH (at pH 12) shows ions [M-H]<sup>&minus;</sup>, 
[M-2H+K]<sup>&minus;</sup>, and [M-3H+K]<sup>&minus;</sup> in order of 
decreasing abundance, indicating successive deprotonations of the peptide. Other 
tripeptides were similiar in these spectra, except that tripeptides APG and GPA 
where the [M-3H+2K]<sup>&minus;</sup> was not detected.  (The authors do not 
mention, but the spectra show other significant ions that are probably 
contaminants.)
<p>
CID spectra were obtained on the ion [M-2H+Na]<sup>&minus;</sup> for the 
peptides AGG, GGA, LGG, GGL, VGG, and GGV.  Conditions were adjusted for
just one collision, and consequently this creates low product ion counts.
Four major ions are evident:
[<i>x</i><sub>1</sub>&nbsp;&minus;&nbsp;2H]<sup>&minus;</sup>,
[<i>b</i><sub>2</sub>&nbsp;&minus;&nbsp;2H]<sup>&minus;</sup>,&nbsp;
[<i>y</i><sub>2</sub>&nbsp;&minus;&nbsp;H&nbsp;+&nbsp;Na]<sup>&minus;</sup>,
[<i>y</i><sub>2</sub>&nbsp;&minus;&nbsp;H&nbsp;+&nbsp;Na&nbsp;+&nbsp;CO<sub>2</sub>]<sup>&minus;</sup>.
Of these ions, the first three are found in all six tripeptides, and the last
ion is found only in tripeptides of the sequence _GG.  These same product
ion were found in product ion spectra for [M-2H+X]<sup>&minus;</sup> of GPA,
APG, and PGG, and of product ions for this parent ion in other alkali metals.
<p>
CID spectra of selected ion [M-2H+X]<sup>&minus;</sup> for M=GGL and X=Li, Na, 
K, and Cs shows greater fragmentation from Li to Cs (Li &gt; Na ~ K &gt; Cs) 
but same major product ions present for all metal ions.
<p>
Acylation of N-terminus alters fragmentation of [M-2H+X]<sup>&minus;</sup> in 
multiple ways.  For Ac-VGG and Ac-GGV, major product ions are 
[<i>x</i><sub>1</sub>&nbsp;&minus;&nbsp;2H]<sup>&minus;</sup>,
[<i>b</i><sub>1</sub>&nbsp;&minus;&nbsp;2H]<sup>&minus;</sup>,
[<i>x</i><sub>2</sub>&nbsp;&minus;&nbsp;3H&nbsp;+&nbsp;Na]<sup>&minus;</sup>,
[<i>x</i><sub>2</sub>&nbsp;&minus;&nbsp;HCOOH]<sup>&minus;</sup>, and
[<i>c</i><sub>1</sub>&nbsp;&minus;&nbsp;H&nbsp;+&nbsp;Na]<sup>&minus;</sup>
for both.  In addition Ac-VGG shows
[<i>y</i><sub>2</sub> &minus; H + Na]<sup>&minus;</sup>.
<p>
The parent ion [M - 3H + 2X]<sup>&minus;</sup> was next fragmented for peptides
AGG, LGG, VGG, and GGA, GGL, and GGV.  There is only one major product:
[<i>y</i><sub>2</sub> &minus; 2H + 2Na]<sup>&minus;</sup>.  It is also a prominent
product ion for PGG [M - 3H + 2X]<sup>&minus;</sup>.  With GPA and APG,
this ion has much lower intensity.
<p>
These fragmentation patterns strongly indicate that the metal coordinates with 
the tripeptide wrapped around it.  Major bonding is with carboxylate oxygen and 
to the amide N of the C-terminal amino acid, which is missing a proton.  Other 
coordinations are likely with nonbonding electron pairs of the amide N atoms of 
the other residues.  The same pattern occurs with the molecule missing 3 protons 
and having 2 metal atoms: one metal atom coordinates the background and the 
other forms a salt linkage to the C-terminal carboxylate O atom.  Others have 
already proposed similar structures.  Bonding to deprotonated carboxylate anion 
is required by MS/MS results and <i>ab initio</i> calculations.  Yet another 
structure that is plausible is for the N atom on the 2nd residue instead of the 
C-terminal residue to be deprotonated.  Fragmentation scheme to 
[<i>y</i><sub>2</sub> &minus; H + Na]<sup>&minus;</sup> is fairly predicable, 
with formation of R<sub>1</sub>CH=NH and loss of CO.  Metal is coordinated to 
R<sub>3</sub> N atom having no proton, and to R<sub>3</sub> carboxylate O atom, 
with coordination to R<sub>2</sub> N atom too.  Formation of 
[<i>y</i><sub>2</sub> &minus; H + Na &minus; CO<sub>2</sub>]<sup>&minus;</sup> 
involves either one or two steps, depending on starting structure.  Another 
scheme is shown as to how [<i>x</i><sub>1</sub> &minus; 2H]<sup>&minus;</sup> is 
formed.  Yet another scheme shows how [<i>b</i><sub>1</sub> &minus; 
2H]<sup>&minus;</sup>.  Formation of <i>y</i> ions requires transfer of proton 
from the neutral to ionic product.  The &beta;- (-CH-) and &gamma;- (-NH-) 
hydrogen of preceding residue have been identified as the proton transferred. 
Using deuterated <i>d</i><sub>4</sub>-[M - H + 2X]<sup>+</sup> it was shown that 
[<i>y</i><sub>1</sub> + 2X]<sup>+</sup> product is <i>d</i><sub>1</sub>, and a 
mechanism involving the -CH- and not -NH- hydrogen from previous amino acid 
residue.  In this case the [M - 2H + X]<sup>&minus;</sup> anion, a 
<i>d</i><sub>3</sub> ion forms a <i>d</i><sub>2</sub> product ion and so the 
hydrogen must come from the N-terminus.  When proline is part of the tripeptide, 
there are only 4 replaceable hydrogen atoms.  Thus a fully deuterated 
[M-2H+X]<sup>&minus;</sup> is <i>d</i><sub>2</sub>, and fragmentation of M=GPA 
or APG produces a monodeuterated [<i>y</i><sub>2</sub> &minus; H + 
X]<sup>&minus;</sup>, but a di-deuterated ion when M = PGG.  Deuterium labeling
also proves the mechanism for parent ion where M contains or does not contain
a proline, in which non-deuterated [<i>x</i><sub>1</sub> - 2H]<sup>&minus;</sup>
would be formed.
<p>
With N-acylated forms of ion [AcM-2H+X]<sup>&minus;</sup>, there is a much 
richer fragmentation chemistry.  Major product ions are [<i>x</i><sub>1</sub> - 
2H]&minus; and [<i>y</i><sub>2</sub> - H + X]<sup>&minus;</sup>. The 
<i>d</i><sub>2</sub>-[AcM-2H+X]<sup>&minus;</sup> di-deuterated form shows a 
formation of <i>d</i><sub>0</sub>-[<i>b</i><sub>1</sub> - 2H]<sup>&minus;</sup>. 
Schemes are outlined in the paper for formation of other ions from this parent: 
[<i>x</i><sub>2</sub> - 3H + X]<sup>&minus;</sup>, [<i>x</i><sub>2</sub> - 
HCOOH]<sup>&minus;</sup>, and [<i>c</i><sub>1</sub> - H + X]<sup>&minus;</sup>. 
To form [<i>x</i><sub>2</sub> - 3H + X]<sup>&minus;</sup>, a proton movement 
from ionic to neutral product is required.  If a di-deuterated parent produces a 
non-deuterated product, a hydrogen from -NH- and not -CH- must occur in the 
scheme.  One sheme shows transfer of -NH- of R<sub>2</sub> amino acid and 
elimination of R<sub>1</sub>CH<sub>2</sub>NHCOCH<sub>3</sub>.  GPA and APG lack 
the -NH- on R<sub>2</sub> so no product ion of this type is formed. With PGG 
however, this type is observed since the critical R<sub>2</sub> -NH- is present. 
Product ion [<i>x</i><sub>2</sub> - HCOOH]<sup>&minus;</sup> is formed from 
nonproline peptides and from PGG.  Transfer of -NH- hydrogen on R<sub>1</sub>
to &beta;-C on R<sub>1</sub> with loss of R<sub>1</sub>CH=NCOCH<sub>3</sub>
will lead to [<i>x</i><sub>2</sub> - H + X]<sup>&minus;</sup>, an intermediate,
and this will lead to HCOOX elimination to form the [<i>y</i><sub>2</sub> - HCOOH]<sup>&minus;</sup>
product.  For di-deuterated [AcM-2H+X]<sup>&minus;</sup> parent, this
should lead to <i>d</i><sub>1</sub>-[<i>x</i><sub>2</sub> - HCOOH]<sup>&minus;</sup>
and it does.  This product ion does not form from Ac-GPA or Ac-PGA, as expected.
With PGG, the mechanism suggests a -CH- hydrogen from proline ring helps to
form [<i>x</i><sub>2</sub> - H + X]<sup>&minus;</sup> intermediate to 
&beta;-C of R<sub>1</sub> with loss of proline ring and this leads to formation
of [<i>x</i><sub>2</sub> - HCOOH]<sup>&minus;</sup> with no original exchangeable
hydrogen.  For formation of [<i>c</i><sub>1</sub> - H + X]<sup>&minus;</sup>
a cleavage of N-C bond on R<sub>2</sub>.  This is not possible for APG and
PGA peptides and so this product ion is not observed.  CID of 
<i>d</i><sub>2</sub>-[Ac-M &minus; 2H + X]<sup>&minus;</sup> of nonproline
containing tripeptides produce <i>d</i><sub>1</sub>-[<i>c</i><sub>1</sub> - H 
+ X]<sup>&minus;</sup> as expected.
<p>
The scheme to produce [<i>y</i><sub>2</sub> - 2H + 2Na]<sup>&minus;</sup>
from [M - 3H + 2Na]<sup>&minus;</sup> is also given.  One precursor involves
a transfer of H from N-terminus to ionic fragment, while another involves
transfer of centrally coordinated Na to the fragment.  Deuterium labeling
confirms the scheme.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Dikler S, Kelly JW, Russell DH
 </span>
 <span class="year">
(1997)
 </span>
 <span class="jname">
J. Mass Spectrom.
 </span>
 <span class="jvol">
32,
 </span>
 <span class="pages">
1337-1349.
 </span>
<p class="reftitl">
Improving Mass Spectrometric Sequencing of Arginine-containing Peptides by
Derivatization with Acetylacetone
<p class="abstract">
<b>ABSTRACT</b>. Modifcation of arginine residues in bradykinin,
[1-5]-bradykinin, splenopentin and two synthetic pentapeptides with
acetylacetone (pentane-2,4-dione) signficantly increases the relative abundance
of sequence-specic fragment ions produced by matrix-assisted laser
desorption/ionization (MALDI). The fragmentation efficiency as measured by
post-source decay in a reflectron time-of-flight mass spectrometer increases by a
factor of 2-3.5. Peptide bonds adjacent to modified residues are more
susceptible to cleavage than in the non-derivatized peptide ions. The increased
lability of these bonds gives rise to more complete sequence information. In
addition, the relative abundances of sequence-specific fragment ions are
enhanced. This strategy makes it possible to obtain valuable structural
information from arginine-containing peptides that otherwise do not fragment
well.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
B. K&uuml;ster B, Wheeler SF, Hunter AP, Dwek RA, Harvey DJ
 </span>
 <span class="year">
(1997)
 </span>
 <span class="jname">
Anal. Biochem.
 </span>
 <span class="jvol">
250,
 </span>
 <span class="pages">
82-101.
 </span>
<p class="reftitl">
Sequencing of N-Linked Oligosaccharides Directly from Protein Gels:
In-Gel Deglycosylation Followed by Matrix-Assisted Laser Desorption/Ionization
Mass Spectrometry and Normal-Phase High-Performance Liquid Chromatography
<p class="Para1">
Glycoproteins are purified by SDS-PAGE and detected either with Coomassie
or negative stains.  Bands of interest are excised and deglycosylated
in-gel with PNGase F.  Several <q>standard</q> glycoproteins were
looked at.  The use of gels permits concentration of protein into a very
small volume.  12.5 and 17.5% gel which are 0.75 mm thick permit analysis
in the 10-200 kDa range.  Bisacrylamide was reduced to 0.07% (as opposed to
being 0.8%).  The lower crosslink helps for in-gel deglycosylation.
<p>
Staining is by Coomassie or copper ions (Bio-Rad makes a kit for this).
1-15 &micro;g (50-100 pmol) of protein is loaded to the gel.  Note that
the stain should not have a high amount of acetic acid so that sialic
acid moieties are not affected.  For the copper stain, the background and
not the protein is stained, and this often lacks sufficient contrast; however
the stain is rapid.   Selected bands are excised and then proteins are
reduced and alkylated.
<p>
A completely denatured rather than native protein is a better substrate
for PNGase F.  Water purity is especially critical.  PEG and glucose
molecules in starched gloves can ruin MALDI results.  An in-gel buffer
of 20 mM NaHCO<sub>3</sub> pH 7.0 was used.  Phosphate and citrate buffers
are not compatible with anionic sugars.  Ammonium buffers and cause
formation of glycosamines, making reductive amination for labeling purposes
impossible.  PNGase F and sialidase were dialyzed against buffer prior
to use.  pH 7 was used when both enzymes were used together since it
was between the pH optima of these enzymes.  PNGase F was used at 100 U/ml;
others have used in a range of 0.6-60 U/ml, but more was used because it
was thought some loss would occur for an in-gel digest.  Note that the
total protein of enzyme was estimated to be 100 ng protein, not enough
to interfere with

<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
March RE
 </span>
 <span class="year">
(1997)
 </span>
 <span class="jname">
J. Mass Spectrom.
 </span>
 <span class="jvol">
32,
 </span>
 <span class="pages">
351-369.
 </span>
<p class="reftitl">
An Introduction To Quadrupole Ion Trap Mass Spectrometry
<p class="Para1">
<b>REVIEW</b>.
The quadrupole ion trap can store gaseous ions and provide measurement
of a wide range at high resolution using a pressure of 1 mTorr helium gas.
The ion trap is essentially an electric field test tube.  Since
mass/charge ratios can be measured, the instrument thus becomes a mass
spectrometer.  The potential well (or <q>basin</q>) of the ion
trap is tipped in a particular <q>direction</q> and ions tumble
out of the well, leaving in order of ascending mass/charge ratio.
Alternatively the perspective can be taken that ions are made to leave
the basin when one side of the basin is lowered progressively.
As ions are emitted, they strike a detector and are counted.  Ejection
is actually achieved by ramping up in linear fashion the amplitude of
a radiofrequency potential applied to one of the ion trap electrodes.
The initial amplitude and ramping rate are known and so the <i>m/z</i>
is known.  This method is known as the <q>mass-selective axial
instability mode.</q>
<p>
The quadrupole consists of three differently shaped electrodes.  Two
are identical and have hyperboloidal geometry (small inverted saucers)
and are called <i>end-cap electrodes</i>.  One end-cap electrode has
a single small central aperture through which electrons/ions are gated
periodically and the other has several small apertures arranged centrally
as well in which ions pass to the detector.  Only a single hold is in
end-cap electrodes having external ion sources.  The 3rd electrode
also has hyperboloidal geometry having two sheets rather than one and
is called the <i>ring electrode</i>.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Corthals GL, Molloy MP, Herbert BR, Williams KL, Gooley AA
 </span>
 <span class="year">
(1997)
 </span>
 <span class="jname">
Electrophoresis
 </span>
 <span class="jvol">
18,
 </span>
 <span class="pages">
317-323.
 </span>
<p class="reftitl">
Prefractionation of Protein Samples Prior to Two-Dimensional Electrophoresis
<p class="abstract">
<b>ABSTRACT</b>.  Thousands of proteins may be visualised on a two-dimensional (2-D) gel,
but only hundreds are present at levels sufficient for chemical analysis.
Therefore, prefractionation of protein samples prior to 2-D polyacrylamide
gel electrophoresis (PAGE) will be important for the investigation of
proteins that are present at sub-picogram levels in physiological samples.
We describe an approach to prefractionate protein samples prior to
2-D PAGE using the Gradiflow, which is a new (preparative)
electrokinetic membrane apparatus designed to fractionate
proteins in a number of different ways. We have fractionated
human serum under nonreducing conditions using the 'reflux' mode,
in which proteins are fractionated according to their relative
mobility under controlled electrophoretic conditions, where
the current is periodically reversed. We describe how fractionation
occurs and present examples of enrichment of specific proteins.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Vidal P, Morris MC, Chaloin L, Mery J, Heitz F, Divita G
 </span>
 <span class="year">
(1997)
 </span>
 <span class="jname">
Lett. Peptide Sci.
 </span>
 <span class="jvol">
4,
 </span>
 <span class="pages">
227-230.
 </span>
<p class="reftitl">
Conformations of a synthetic peptide which facilitates the cellular
delivery of nucleic acids
<p class="Para1">
The peptide Ac-GALFLGFLGAAGSTMGAWSQPKSKRKV-Cya was synthesized.  The first
17 N-terminal residues come from a hydrophobic region of HIV gp41 fragment
and the final seven from a highly basic segment of a nuclear localization
sequence (NLS) of SV40.  The Trp residue provides intrinsic fluorescence,
while a proline residue helps to break conformation.  Synthesis was done using
AEDI-Expansin resin on a Fmoc/tBu system and double couplings for Arg, Lys,
Ser, and Thr.  The peptide was purified on RP-HPLC with MeCN+0.1% TFA
gradient and, with theoretical mass 2867.5, shows up on ESI-MS in positive
mode at [M+H]<sup>+</sup> <i>m/z</i> 2864.  In addition an amino acid analysis
was done.
<p>
The peptide was evaluated by circular dichroism, and the spectrum of peptide
in pure water shows a single minimum at 198 nm, indicating unordered structure.
When placed in 30% trifluoroethanol (TFE), the spectrum now shows two minima
at 221 and 207 nm and a maximum at 193 nm, strongly indicating organization
into &alpha;-helix.  Increasing to 50% does not alter the spectrum from a
plot of &minus;Ellpticity vs. percent TFE in water at &lambda;= 222 nm.
The water-TFE titration is characterized by isodichroic point at 202 nm
suggesting equilibrium between two conformational states.  One state can be
assigned to the &alpha;-helix, but the nature of the state in pure water is
not known although it is not random coil.  In a 20 mM potassium phosphate
pH 7.2 buffer, the spectrum shows a single minimum centered at 212 nm, seen
also at higher (100 mM) phosphate concentration.  There is no peptide
concentration dependence on spectrum.  The minimum position is at a rather
low wavelength compare to that typical for pure &beta;-sheet (217 nm), and
there is a suggestion of some &beta;-sheet conformation.  That there is a mix
of structures is seen also in low ellipticity intensity and that no crossover
occurs.  Changes in peptide concentration also show changes in CD spectrum:
increases in concentration appear to induce structural arrangements which
appear to be &beta;-sheet.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Moniatte M, Lesieur C, Vecsey-Semjen B, Buckley JT, Pattus F,
van der Goot FG, Van Dorsselaer A
 </span>
 <span class="year">
(1997)
 </span>
 <span class="jname">
Int. J. Mass Spectrom. Ion Proc.
 </span>
 <span class="jvol">
169/170
 </span>
 <span class="pages">
179-199.
 </span>
<p class="reftitl">
Matrix-Assisted Laser Desorption/Ionization Time-of-Flight Mass Spectrometry
in The Subunit Stoichiometry Study of High-Mass Non-Covalent Complexes
<p class="Para1">
It is generally believed that multimeric proteins associated noncovalently
cannot withstand sample preparation conditions for MALDI and that nonetheless
ionization of the multimer cannot occur under MALDI conditions.
Karas and Hillenkamp however reported in some case noncovalent complexes
of catalase and strepavidin but that results were not always reproducible
and that monomers were more often than not observed.
<p>
Proaerolysin was purified from pathogen <i>Aeromonas hydrophila</i>.
It is a 335 kDa heptamer with UV concentration of 2.5 AU at 280 nm where
1 mg/ml is present.  4 mg/ml was dialyzed against 5 mM ammonium acetate
pH 7.6 extensively at 4&deg;.  20 &micro;l of this is treated with 50 mg/ml
trypsin (1:30 enzyme:substrate) which converts proaerolysin dimer to
aerolysin.  At different times 1 &micro;l is removed and added to
4&micro;l trypsin-chymotrypsin inhibitor (100 &micro;g/ml) and immediately
frozen in liq. nitrogen.  1 &micro;l is used for MALDI-TOF and the balance
4 &micro;l is used in SDS-PAGE.  Alpha-hemolysin was purified from Wood 46
strain and 2 mg/ml dialyzed against 20 mM NaCl, 20 mM HEPES pH 7.0 at 4&deg;
overnight., then incubated 37&deg; for 2 h with some vortexing.  Oligomer
monomer separation of the hemolysin was done on Sephadex G-75 1.5 &times;
70 cm column in a 5 mM ammonium acetate pH 6.7 buffer, concentrated to
1 mg/ml on vacuum centrifuge.  Catalase from bovine liver, pyruvate kinase
from rabbit muscle, yeast ADH, and concanavalin A were all from Sigma and
prepared as fresh 20 pmol/&micro;l stock in 10 to 50 mM ammonium acetate
pH 7.0.  Dilutions of stocks were done only just before MALDI spotting since
dilutions over time may result in dissociation of multimers.  Controls
of stocks included preparation in 50% MeCN with 1% formic acid.
Matrices used includ sinapinic acid, ferulic acid, 6-aza-2-thiothymine (ATT),
and 2,6-dihydroxyacetophenone (2,6-DHAP).
<p>
Sample well spotting is two-step: 1 &micro;l of sinapinic acid saturated
in acetone is spotted to the well surface to form microcrystalline base
coat.  Then 0.5 &micro;l sample dissolved in matrix at 13 mg/ml in
pure water or 70:30 water:MeCN is applied.  Drying is in air (takes 1 min)
or vacuum (within 10 s).  A finer crystalline surface is seen as opposed
to larger needles.  The base coat is believed to serve as nuclei for
orienting crystallization.  Other matrices are in classical preparatins:
sinapinic as 13 mg in 70:30 water:MeCN; ferulic acid in neat tetrahydrofuran
as 25 and 40 mg/ml solution;  ATT and 2,6-DHAP as 2 and 10 mg/ml in
20 mM diammoniumhydrogen citrate (DAHC).  1 &micro;l sample is mixed
with 1 &micro;l matrix and then 0.5 &micro;l is applied and dried under
vacuum.
<p>
The Bruker BIFLEX MALDI Tof with single stage source SCOUT with X-Y multisample
probe including imaging and gridless reflector.  A 337 nm 3 Hz pulsing
nitrogen laser is used.  Linear positive ion mode with 30 kV acceleration.
A HIMAS detector with Venetian blind conversion dynode, MCP, and scintillator
with multiplier was equipped;  it can be operated in post-acceleration mode
or not, but PAD is required for improving sensitivity of high mass ions.
Ions are postaccelerated and then detected as secondary ions in a single
MCP.  PAD is set to 8-8.5 kV.  Mass calibration involved use of ACTH 18-39,
insulin (MW 5733.6), and cytochrome c equine (MW 12360).  A Micromass
VG Bio-Q triple quadrupole with Quattro (Q1) upgrade with <i>m/z</i> range
to 4000 and Q2 with range to <i>m/z</i> 8000 was used so that detection
of low charge states was possible.  Mass analysis was set on Q1 when
charge states with <i>m/z</i> 4000 or less were studied.  Q1 and hexapole
collision are set to high pass filters in rf only mode to transmit to
Q2 for higher mass charge states.  Solution is 25 mM ammonium acetate pH 7
using 10 pmol/&micro;l sample.  Sample cone is 20-120 V and source temp
is 40 &deg; to minimize denaturation.
<p>
Proaerolysin is a 52 kDa dimer activated by 40 amino acid cleaved from
C-terminus, this determined by ESI-MS.  The 48 kDa dimer activated toxin
binds to receptor on a target cell with oligomerization of the toxin
to create transmembrane channel leading to cell death.  Under SDS-PAGE
this complex is stable, but in 70% formic acid for 30 min at RT it will
dissociate;  it has been shown to associate as a heptamer of 335 kDa
using MALDI ToF, but the heptamer is not observed in ESI-MS.  The time-dependent
formation of the heptamer was studied with trypsin in a MALDI spotting-compatible
buffer (5 mM ammonium acetate)  with MALDI-ToF and SDS-PAGE monitoring.
<p>
Using standard dried droplet technique produced spectra with heptamer
seen, but peaks were very broad and not reproducible.  So a base coat
to create crystallization nuclei upon which 1 &micro;l of sample/matrix
with 10 pmol sample in matrix in plain water or MeCN/water mix were tried.
The better microcrystallization produced reproducible and narrower peaks.
The dried droplet technique can be managed if the 10 pmol/&micro;l sample
is diluted 4 times; spectra will quickly decay however.
<p>
Spectra in range <i>m/z</i>50,000 to 400,000 are viewed over time course
of <i>t</i> = 0, 3, 6, and 12 h.  At time zero, a 2M<sup>+</sup> peak
at <i>m/z</i> 104,000 is barely seen above the more dominant <i>m/z</i>
52,000 monomer form.  This continues to be barely detected at 3 and 6 h.
At 12 h, several detectable peaks corresponding to heptamer in multiple
charge states (7M<sup>+</sup>, 7M<sup>2+</sup>, 7M<sup>3+</sup>, and the
4+ form also).  In addition the 48 kDa areolysin form is seen.
<p>
<i>S aureus</i> alpha-hemolysin is a 33.2 kDa hydrophilic monomer that
also binds to cell membranes, whose monomers assemble to form a 232.4 kDa
oligomer transmembrane pore, established also to be a heptamer.
With sinapinic acid dried droplet, we could see only a monomer of aHL
using MALDI-TOF.  In ferulic acid both the monomer at <i>m/z</i> 33,800
and a broad peak centered at <i>m/z</i> 241000 were seen.
In ATT in 20 mM ammonium citrate two broad peaks at 241 and 120.5 kDa
are seen, being the +1 and +2 states of the heptamer. Peaks at 33.4
and 66.5 kDa were seen (+1 states of monomer and dimer).  Some trimer,
but no pentamers or hexamers were seen.  Small peptide adducts caused
peak broadening.  Greater dilution improved signal to noise with improved
resolution to eliminate pentamer, hexamer and octamer signals, and only
heptamer at 234.9 kDa.  Sample at 500 fmol/&micro;l with low laser energy
was best.
<p>
Four well known tetramers were characterized.  They easily dissociate
in SDS-PAGE.  All were dialyzed in 20 mM ammonium acetate pH 7 and then
mixed with matrix.  The two-step sinapinic acid was tried as well as
ferulic acid in THF and 2,6-DHAP in 20 mM aqueous DAHC.  Bovine catalase
is an assembly of a 60 kDa homotetramer.  The sinapinic acid and
2,6-DHAP preps gave poor spectra, but ferulic acid showed tetramer and
monomer as two series of broad peaks.  Peak broadness is caused by
matrix and salt adduct associations and possibly by loss of heme.
Spectra with oligomer demonstrations decay quickly.  The ESI-MS spectra
showed a broad peak of tiny peaks with no multiple charge states, but
desolvation is a problem because of the absence of organic components.
Adjustments to cone voltage and gas flows were of little help.
<p>
With pyruvate kinase (57.2 kDa subunit forming homotetramer) ferulic acid
spectra were not good, nor was 2,6-DHAP in DHAC.  Sinapinic dried droplet
worked if the sample-matrix second application was rapidly dried under
vacuum, although spectra are observed with air drying.  The rapid drying
probably means that the tetramer is in contact with denaturing conditions
for only a brief period and so most of the tetramer remains intact.
In ESI-MS, multiply charged ions of the tetramer were seen in <i>m/z</i>
range 6000-6700.
<p>
Yeast ADH could be observed only with 2,6-DAPH in DHAC.  Monomer and dimer
are seen with no trimer or pentamer.  Rapid drying was necessary
for maintaining tetramer.  pH may play a role in dissociation since
sinapinic acid has a pH 2.5 while the 2,6-DHAP has a pH of about 5.  ESI-MS
also revealed the tetramer quite easily.
<p>
Concanavalin A is predominantly a dimer at pH 5.8 and tetramer at pH 7.5
and above.  Matrix and sample will not cocrystallize at pH 8.5 and so
an ammonium buffer at pH 7.0 was used.  Intense dimer and weaker tetramer
peaks are seen in this case.  With ferulic acid and sinapinic two-step,
only monomer and dimer peaks are seen and fast evaporation was of no use.
ESI-MS spectra showed presence of monomer, dimer and tetamer in
ammonium acetate pH 7.0, as shown by others.
<!-- ============================================================= -->
<p class="tags">
FD
<p class="ref">
 <span class="authors">
Li H, Siu KWM, Guevremont R, Le Blanc JCY
 </span>
 <span class="year">
(1997)
 </span>
 <span class="jname">
J. Am. Soc. Mass Spectrom.
 </span>
 <span class="jvol">
8, 
 </span>
 <span class="pages">
781-792.
 </span>
<p class="reftitl">
Complexes of Silver(I) With Peptides and Proteins as Produced in Electrospray 
Mass Spectrometry
<p class="Para1">
Silver(I) cation is considered a <q>soft</q> cation in Pearson's classification
preferring to bind to soft ligands that are polarizable.  The <q>soft</q> 
functional groups in proteins include sulfur on Cys and Met.  Metal binding
to these groups can offer much information analytically.  Fragmentation
pathways of metal-bound peptides can be distinct from protonated peptides.
The binding of transition metals diffes from other metals (alkali, alkaline
earth).  Peptides were cationized with protons or metals, deposited on silver,
then bombarded with Cs ions to obtain spectra. Results suggested that Ag<sup>+</sup>
was likely chelated to N-termnal N atom and carbonyl O atom of 2nd residue
whereas Na<sup>+</sup> was chelated only to carbonyl O atom.  Oligopeptides
have been studied with Co(II), Mn(II), Ni(II), and Fe(II) and it was suggested
that chelation occurs to N-terminal amino group, deprotonated amide N atoms,
and C-terminal COO<sup>&minus;</sup> anion.  Interaction of metal with aromatic
rings when present is seen.  Cys-containing peptides interact with Fe(II).
The His and carbonyl O atoms of angiotensin was found to interact with Zn(II).
Cu(II) binds via N- and C-terminus.
<p><b>Experimental</b>.  A SCIEX TAGA 6000E triple quad with home-made
electrospray probe, and PE-SCIEX API 300 ESI-MS are used.  Positive ion
mode with scan speeds of 10-50 ms per <i>m/z</i> unit with ten summed scans
is used to obtain spectra.  Argon or nitrogen is used in collision cell 
at pressures designed to produce only a single collision, and collision
energies of 40-80 eV are used.  59 oligopeptides having lengths of 2 to 11 
residues, and a few proteins.  About half had one or more Met residues, and
one-third had residues with only neutral side chains.  All are obtained
from Sigma.  Peptide concentrations were from 10 &micro;M to 2 mM, and solutions
of AgNO<sub>3</sub> ranged from 1-10 mM in 50 mM acetic acid in 1:1 water:MeOH.
In some cases acetic acid was not present.  Flow rates were 1 &micro;l/min.
The semiempirical program ZINDO was used for molecular modeling.  Geometry
of complex formation is optimized by minimizing complex total energy.
<p>
Bovine insulin was mixed with Ag<sup>+</sup> in 50 mM acetic acid and the 
spectrum showed four clusters of peaks with cationic charges ranging from 5+ to 
8+ in which the ions had a [M + <i>m</i>H + <i>n</i>Ag]<sup>(<i>m + n</i>)+</sup>
pattern where, for example the 6+ cluster,  <i>m</i> ranged from 1 to 6 and
<i>n</i> also ranged from 1 to 6.  Bovine insulin has two N- and C-termini,
four basic residues, and six Cys residues all connected in three disulfide 
bonds.  In just acid, it is fully protonated to 6+ for the possible protonations.
In silver, charges higher than 6+ are possible indicating that silver does
not merely displace the position held by protons.
<p>
Because of the presence of <q>soft</q> atoms like sulfur which could bind to 
silver, the peptide YGGFM (Met-enkephalin) and YGGFL (Leu-enkephalin) were 
tested as a sulfur- and nonsulfur-containing nearly identical molecule. The 
spectra in silver were quite different.  Met-enkepalin had a 
[M+H+Ag]<sup>2+</sup> form while Leu-enk was [M+H]<sup>+</sup>.  While 
[M+Ag]<sup>+</sup> and [M - H + 2Ag]<sup>+</sup> were present in both spectra, 
they were less abundant relative to [M + H + Ag]<sup>2+</sup> for Met-enk and to 
[M+H]<sup>+</sup> for Leu-enk.  Most important is that silver ion binds despite 
the absence of sulfur.  Spectra for pH 7 solutions were quite similar to pH 3 
solutions: increasing pH from 3 to 7 created a small increase of ratio of 
abundance of Ag-containing to non-Ag-containing peptide (from 12.1 to 14.6 
for Met-enk and 0.33 to 0.49 for Leu-enk).
<p>
A longer oligopeptide, substance P (RPKPQQFFGLM-nh<sub>2</sub>) and its norleucine
analogue (like Met, but a methylene -CH<sub>2</sub>- in place of the S atom), 
were tried.  Both are C-terminal amides lacking a carboxylate, which could 
complex with silver.  The ESI-MS spectra for pH 3 solution shows a prominent
ion for [M+2H+Ag]<sup>3+</sup> but the Nle<sup>11</sup>-Substance P major 
ion is [M + 2H]<sup>2+</sup>; while the [M+2H+Ag]<sup>3+</sup> ion is present,
it is far less of a presence in the spectrum.  In pH 7 the major ions are the same
as for pH 3.
<p>
A set of 24 di- and tripeptides with and without Met were tested in pH 3.
For peptides with one Met, the [M+Ag]<sup>+</sup>:[M+H]<sup>+</sup> ratio
was 0.39; for peptides with no Met, the ratio is 0.05, and for Met-Met, the
ratio is 5.9.  For Met-enk, the ratio is 12.1, Leu-enk 0.33, substance P
13.4, and [Nle<sup>11</sup>]substance P 0.12.  Thus the presence of sulfur
strongly affects silver affinity.
<p>
In another special ESI-MS setup, the peptide solution (YGGFM) is infused into
one spray system and silver nitrate infused from another side, the needle both
charged at 3000 V.  Both sprays traverse a copper tube charged at 1000 V,
with compressed air blown in.  The gas phase ions then mix at a T.  Ag does not
bind to Met-enk in this case, which means adducts do not form at 1 atm or in 
the lens region of the MS; they can only form in solution.
<p>
In the next phase, CID fragmentation is done on di- and tripeptides.  A
product ion spectrum of GlyAlaAla [M + <sup>107</sup>Ag]<sup>+</sup> shows that
most product ions are Ag-associated.  These product ions in general are the 
following species:
<ol>
<li>[<i>a<sub>n</sub></i> - H + Ag]<sup>+</sup>
<li>[<i>b<sub>n</sub></i> - H + Ag]<sup>+</sup>
<li>[<i>y<sub>n</sub></i> + H + Ag]<sup>+</sup>
<li>[<i>b<sub>n</sub></i> + OH + Ag]<sup>+</sup>
</ol>
<p>
The first three species are seen in low energy fragmentation of singly substituted
silver bound peptides.  The last ion species is seen with fragmentation of 
alkali metal-containing peptides.  In the product ion spectrum of [M+Ag]<sup>+</sup>
of GlyMet.  If Ag binds to the Met S atom, then an N-terminal fragment ion shorter
than and including <i>b</i><sub>1</sub> would be cleaved at amide bond and not
contain sulfur.  The [<i>b</i><sub>1</sub> + OH + Ag]<sup>+</sup> is at the 
<i>m/z</i> 182 ion, which means that Ag<sup>+</sup> must be bound to functional
groups on N-terminal side of point of cleavage.  ZINDO was used to determine 
the three lowest energy structures of the [M+Ag]<sup>+</sup> ion.  It is not 
so much important that they are really the lowest energy structures, but they all
show Ag<sup>+</sup> bound to the Met S atoms and to functional groups such as
carbonyl O and amide N atoms, and to N-terminal amine N.  Thus a chelation is
shown.  These models also are true when sulfur is not present.  Peptides with
three or more residues also show this chelation.  The [M+Ag]<sup>+</sup> ion
of Met-enk shows a product ion spectrum populated with N-terminal fragment 
ions ([<i>a</i><sub>4</sub>-H+Ag]<sup>+</sup>, <i>b</i><sub>3</sub>-H+Ag]<sup>+</sup>,
<i>b</i><sub>3</sub>+OH+Ag]<sup>+</sup>) which do not have Met side chain but 
they are complexed to silver.   The C-terminal side silver-containing ions 
([<i>y</i><sub>3</sub>+H+Ag]<sup>+</sup>, <i>y</i><sub>4</sub>+H+Ag]<sup>+</sup>)
contain the sulfur but have low abundance.  If Met is replaced by Leu, then 
there are no <i>y</i> ions.  These findings had been reported by Tang et al
with Cs<sup>+</sup> bombardment.
<p>
For the [M+H+Ag]<sup>2+</sup> precursor instead of [M+Ag]<sup>+</sup>, the
doubly charged will form a product with minimized coulombic repulsion, a short
peptide with no basic residues and proton on the N-terminal amine N and silver
ion on carboxylate anion.  The more abundant <i>y</i> ions and less abundant
<i>b</i> ions containing Ag support this expectation.  For the triply charged
precursor [M+2H+Ag]<sup>3+</sup> of substance P and Nle analogue, there are
strong intensities for <i>b</i> ions ([<i>b</i><sub>10</sub> + Ag]<sup>2+</sup>
for Met-containing and [<i>b</i><sub>9</sub> + Ag]<sup>2+</sup> for Nle-containing).
Additionally substance P produces [<i>y</i><sub>n</sub> + H + Ag]<sup>+</sup>
where <i>n</i> is 5 and 6, but the Nle form does not produce these ions
All product ion spectra were recorded several times to ensure reproducibility.
<p>
An interesting finding is what happens to as the peptide grows longer and whether
it contains sulfur atoms.  The hypothesis is that while silver is <q>chelated</q>
by several groups it is anchored by the S atom of Met.  The Ag-S bond is special
in that it has strong &sigma; bonds with readily polarizable S atom, but also
&pi; bond from electrons in metal <i>d</i>&pi; to ligand <i>p</i>&pi; orbitals.
Sulfur has low electronegativity and high polarizability.  Ag(I), Au(I) and
Hg(II) readily bond to S atoms which is why they are used as heavy atom 
derivatives in crystallography.  In acidic conditions, the carboxyl group is
unionized and thus does not compete.  Amide forms of peptides reveal the effect
of carboxylate anion.  In neutral and acidic solutions, the metal competes for
protons to bind the amino terminal.  There is evidence to suggest that solution
and gas-phase structures are distinct and that rearrangement in the gas phase
occurs within a few milliseconds.  In solution silver ion is bound to S atom,
but rearrangement suggests less of a role in interact with silver.  Ion movements
and structural rearrangements are more likely since the absence of solvation
increases coulombic effects.  In particular the loss of water may require the
peptide to <q>solvate</q> the silver ion and wrap around it.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Rouse JC, Strang A-M, Yu W, Vath JE
 </span>
 <span class="year">
(1998)
 </span>
 <span class="jname">
Anal. Biochem.
 </span>
 <span class="jvol">
256,
 </span>
 <span class="pages">
33-46.
 </span>
<p class="reftitl">
Isomeric Differentiation of Asparagine-Linked Oligosaccharides by
Matrix-Assisted Laser Desorption-Ionization Postsource Decay
Time-of-Flight Mass Spectrometry
<p class="abstract">
<b>ABSTRACT</b>.
Matrix-assisted laser desorption-ionization (MALDI) postsource decay (PSD) was
used to differentiate glycoprotein- released N-linked oligosaccharide isomers
directly from aliquots of glycosidase digests and peak fractions collected from
high-pH anion exchange chromatography (HPAEC) with minimal sample handling and
material. With the implementation of instrumental tuning and acquisition
controls, MALDIPSD of NMR-characterized high-mannose, hybrid, and complex
standards resulted in spectra with reproducible fragment ion peak intensity
ratios which correlated well to the respective oligosaccharide branching
patterns. A knowledge-based strategy was utilized to characterize unknown
isomeric N-glycan structures in which specific fragment ion types and their
distributions in the unknown PSD spectrum were compared to those in PSD spectra
of standards possessing similar structural features. This PSD knowledge-based
isomeric differentiation strategy was applied to distinguishing recombinant
glycoprotein-derived Man7 D1 versus D2/D3 isomers directly from a PNGaseF
digest aliquot of high-mannose N-glycans based on branching differences. A
precursor ion selection device was employed to isolate the component of
interest from the mass profile without additional chromatographic isolation
steps. MALDIMS signal-tobackground was maximized for direct PSD with
on-theprobe sample clean-up methods. The asialo complex Nglycan PSD knowledge
base was used to differentiate HPAEC peak fractions containing the tri- and
tri*-antennary branching isomers and two tetraantennary isomers with antennal
versus core fucose locations. Although the particular asialo complex N-glycan
isomers here were well separated by HPAEC, MALDIMS alerted us to their presence
using m/z-derived monosaccharide compositions and PSD fragmentation allowed us
to differentiate these structures using the HPAEC elution positions as guides.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Jai-nhuknan J, Cassady CJ
 </span>
 <span class="year">
(1998)
 </span>
 <span class="jname">
J. Am Soc. Mass Spectrom.
 </span>
 <span class="jvol">
9,
 </span>
 <span class="pages">
540-544.
 </span>
<p class="reftitl">
Negative Ion Matrix-Assisted Laser Desorption/Ionization Time-of-Flight
Post-Source Decay Calibration By Using Fibrinopeptide B
<p class="abstract">
<b>ABSTRACT</b>.
Fibrinopeptide B (Mr 1552.58) was employed as a calibration compound for
matrix-assisted laser desorption/ionization time-of-flight (MALDI-TOF)
post-source decay (PSD) fragment ion analysis in the negative mode. Experiments
were performed by using both continuous and delayed extraction, with the
maximum reflectron voltages being 30 and 21 kV, respectively. For comparison, a
common positive ion PSD calibrant, ACTH(1839) (Mr 2466.7), was also employed
with positive ion calibration constants being applied to negative ion spectra.
Using fibrinopeptide B as the calibrant, the negative ion PSD results for
angiotensin II (Mr 1046.2), renin substrate tetradecapeptide (horse) (Mr 1759.0),
and the custom-synthesized peptide (K2G4)2 (Mr 987.1) showed a factor of 1.52
improvement in absolute mass accuracy. Typical absolute mass-to-charge ratio
accuracies were within 61 Thomson and were achieved even when the peptide being
analyzed was more massive than fibrinopeptide B. In addition, both calibrants
showed increased accuracy when experiments were conducted in the delayed
extraction mode. Other advantages of using fibrinopeptide B are its moderate
cost and the ability to perform calibration and sample analysis for negative
ion PSD under the same instrumental conditions.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Wannet WJB, Op den Camp HJM, Wisselink, van der Drift C,
Van Griensven LJLD, Vogels GD
 </span>
 <span class="year">
(1998)
 </span>
 <span class="jname">
Biochim. Biophys. Acta
 </span>
 <span class="jvol">
1425,
 </span>
 <span class="pages">
177-188.
 </span>
<p class="reftitl">
Purification and Characterization of Trehalose Phosphorylase from the
Commercial Mushroom <i>Agaricus bisporus</i>
<p class="Para1">
Fruit bodies of <i>A. bisporus</i> strain Horst U1 were obtained from
mycelium cultivated on commercially prepared compost at growth stage 3.
Mycelium was grown in static 200 ml liq. cultures in Fernbach flask
on Dijkstra medium with 50 mM glucose, and harvested 21 days at 24&deg;.
Fruit bodies are frozen in liq. nitrogen and ground with glass beads
(100 &micro;m size) in mortar.  Powdered frozen material is suspended
in equal amount of KP buffer (50 mM potassium phosphate pH 7.0, 2 mM DTT,
0.5 mM PMSF, 20% glycerol).  This is centrifuged at
40,000 <i>g</i>/30 min/4&deg;.  The supernatant (800 ml) was concentrated
5-10 fold in Minitan with MWCO 30 kDa.  Alternatively fruit bodies were
processed in Braun food-processing machine to yield similar activities.
Solid ammonium sulfate (ground) was added to 30% saturation.  This was
then spun at 30,000 <i>g</i> for 30 min, 4&deg;, and a 50% cut was then
taken.    The precipitate was resuspended in 50 mM KP buffer, dialyzed
in the Minitan.  Dialyzed enzyme was run in a Damon/IEC-B60 ultracentrifuge
at 200,000 <i>g</i> for 1 h, 4&deg; in a Beckman SB 283 swing out rotor
to remove membrane fragments.
<p>
Supernatant was divided into several portions, each loaded on to Q-Sepharose
Fast Flow (1.5 &times; 30 cm) equilibrated on 0.3 M KP buffer.  After washing,
elution was with 0.3-0.7 M KP linear gradient.  Fractions pooled were
concentrated on YM30 Diaflo (Amicon).
<p>
[continued:  but the only thing interesting here was the tissue processing
for the mushroom]
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Cottrell JC, Green BN
 </span>
 <span class="year">
(1998)
 </span>
 <span class="jname">
Micromass Application Note 212
 </span>
<p class="reftitl">
MaxEnt:  An Essential Maximum Entropy Based Tool for Interpreting
Multiply-Charged Electrospray Data
<p class="Para1">
A 20 kDa protein may generate a series of 10-20 peaks in an electrospray
because of multiply charged states. In analyzing these peaks, each is 
transformed to a single peak on the scale to provide a <q>zero-charge</q> 
spectrum. An early method produced artefacts and a baseline increasing with 
mass. The Micromass Transform method produces artefact-free zero-charge 
spectrum. Ferrige et al. (<i>Rapid Commun. Mass Spectrom.</i> <b>6</b>: 707, 
1992) described a method based on maximum entropy. The analysis is 
computationally intensive and processing over a wide range takes 5 minutes with 
a coarse mass scale of 3-5 Da per channel. This does not yield accurate masses, 
so a second analysis is repeated over the approximate masses in a narrower 0.
5-1 Da per channel scale.
<p>
Methane monooxygenase A is a non-heme iron protein composed of three subunits
with masses of 20, 45, and 60 kDa (&gamma;, &beta;, and &alpha; subnits,
respectively). Each chain is multiply charged in ESI, and these charge peaks
overlap. Peaks are evaluated for the zero-charge form.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Vestal M, Juhasz P
 </span>
 <span class="year">
(1998)
 </span>
 <span class="jname">
J. Am Soc. Mass Spectrom.
 </span>
 <span class="jvol">
9,
 </span>
 <span class="pages">
892-911.
 </span>
<p class="reftitl">
Resolution and Mass Accuracy in Matrix-Assisted Laser Desorption Ionization
Time-of-Flight
<p class="Para1">
Early work with MALDI-ToF involved poor resolution and limited mass
accuracy.  The major limitations on resolution and accuracy is the
broad distribution of intial velocities.  Delayed extraction minimizes
the effects on initial velocity.
<p>
Although basic physics theory describes time of flight ion movement,
for any combination of uniform electric fields the exact equation for
TOF can be approximated by expression:
<p class="eqnserif leftindent">
<i>t</i> = (<i>Df</i><sub>0</sub>/<i>v<sub>n</sub></i>)[1 +
<i>f</i><sub>1</sub><i>p</i>/<i>f</i><sub>0</sub> +
<i>f</i><sub>2</sub><i>p</i><sup>2</sup>/<i>f</i><sub>0</sub> +
<i>f</i><sub>3</sub><i>p</i><sup>3</sup>/<i>f</i><sub>0</sub> + ...
&minus; (2<i>d<sub>a</sub>y</i>/<i>f</i><sub>0</sub><i>D</i>)
(<i>v</i><sub>0</sub>/<i>v</i><sub>n</sub>)]&nbsp;&nbsp;&nbsp;[1]
<p>
where <i>D</i> is the pathlength of the field-free region,
<i>d<sub>a</sub></i> is length of first acceleration region,
<i>v</i><sub>0</sub> is initial ion velocity, <i>v<sub>n</sub></i> is
ion velocity of particular <i>m/z</i> after acceleration from rest
through total accelerating potential <i>V</i>, and <i>y</i> is ratio
of <i>V</i> to the portion applied across first accelerating region.
The analyzer geometry and relative operating voltages are contained in
dimensionless coefficients <i>f</i><sub>0</sub> to <i>f</i><sub>3</sub>;
there are also expressions for common configurations include one- and two-field
ion sources and mirrors. <i>p</i> is a dimensionless perturbation is
the energy deficit of an ion produced at position <i>x</i>, with
initial velocity <i>v</i><sub>0</sub> divided by normal kinetic energy
after first stage of acceleration.
<p>
The <i>m/z</i> of ions is within the expressions for <i>p</i> and
<i>v<sub>n</sub></i>:
<p class="eqnserif leftindent">
<i>p</i> = <i>x/d<sub>a</sub></i> &minus; <i>yT</i><sub>0</sub>/<i>zV</i> =
  <i>x/d<sub>a</sub></i> &minus; <i>y</i>(<i>v</i><sub>0</sub>/<i>v<sub>n</sub></i>)<sup>2</sup>
  &nbsp;&nbsp;&nbsp;[2]
<br>
<i>v<sub>n</sub></i> = (2<i>zV</i>/<i>m</i>)<sup>&frac12;</sup>&nbsp;&nbsp;&nbsp;[3]
<br>
<i>T</i><sub>0</sub> = <i>mv</i><sub>0</sub><sup>2</sup>/2&nbsp;&nbsp;&nbsp;[4]
<p>
Eq [1] accuracy is limited only by truncation of power series in <i>p</i>.
If power series continues to infinity, eq [1] is exactly equivalent to
equation from which it was derived.  The last term of eq [1] which depends
on first power of initial velocity is exact in all cases.  <i>p</i> is
on order of 1% or less for most applications considered, and error obtained
by truncating after cubic term is &lt; 1 ppm.  For higher accuracy or if
<i>p</i> is larger, more terms must be added to expansion.
<p>
Rewriting eq [1]:
<p class="eqnserif leftindent">
<i>t</i> = <i>t<sub>n</sub></i>[(1 + <i>&epsilon;</i>(<i>x, v</i><sub>0</sub>)]
&nbsp;&nbsp;&nbsp;[5]
<p>
where
<p class="eqnserif leftindent">
<i>t<sub>n</sub></i> = <i>Df</i><sub>0</sub>/<i>v<sub>n</sub></i>
&nbsp;&nbsp;&nbsp;[6]
<br><i>&epsilon;</i>(<i>x, v</i><sub>0</sub>) =
 <i>f</i><sub>1</sub><i>p</i><i>f</i><sub>0</sub> +
 <i>f</i><sub>2</sub><i>p</i><sup>2</sup><i>f</i><sub>0</sub> +
 <i>f</i><sub>3</sub><i>p</i><sup>3</sup><i>f</i><sub>0</sub> + ... &minus;
 (2<i>d<sub>a</sub>y/f</i><sub>0</sub><i>D</i>)(<i>v</i><sub>0</sub>/<i>v<sub>n</sub></i>)
&nbsp;&nbsp;&nbsp;[7]
<p>
Eq [5] rearranges to:
<p class="eqnserif leftindent">
<i>&epsilon;</i>(<i>x, v</i><sub>0</sub>) = (<i>t</i> &minus;
<i>t<sub>n</sub></i>) / <i>t<sub>n</sub></i>&nbsp;&nbsp;&nbsp;[8]
<p>
Note that nominal flight time <i>t<sub>n</sub></i> is given by effective
flight distance <i>Df</i><sub>0</sub> divided by nominal velocity
<i>v<sub>n</sub></i>.  The distribution of flight times relative to
nominal is expressed in <i>&epsilon;</i>(<i>x, v</i><sub>0</sub>).
<p>
Focusing in TOF MS means minimizing the dependence of flight time on
initial conditions.  The general condition for focusing to a particular
order is that the derivative of the quantity in question with respect
to variable in question (initial position or velocity) disappears at the
mean of the distribution of the variable.  First order focusing means
the first derivative must be zero, and second order focusing means first
and second order derivative vanish, and so on.  The first derivative
of flight time with respect velocity and position cannot vanish simultaneously,
so if space focusing might be used when velocity distribution is narrow
while velocity focusing might be used when spatial distribution is narrow.
In cases where orthogonal injection or gas phase ionization in TOF ion
source occurs, the ratio <i>v</i><sub>0</sub>/<i>v<sub>n</sub></i> is
small and independent of <i>m/z</i>; the challenge is to reduce dependence
on initial position <i>x</i>.  While called space focusing, in reality it
is energy focusing.
<p>
For space focusing, assume negligible initial velocity:
<p class="eqnserif leftindent">
<i>&epsilon;</i>(<i>x, v</i><sub>0</sub> = 0) =
 <i>f</i><sub>1</sub>(<i>x/d<sub>a</sub></i>)/<i>f</i><sub>0</sub> +
 <i>f</i><sub>2</sub>(<i>x/d<sub>a</sub></i>)<sup>2</sup><i>f</i><sub>0</sub> +
 <i>f</i><sub>3</sub>(<i>x/d<sub>a</sub></i>)<sup>3</sup><i>f</i><sub>0</sub> + ...
&nbsp;&nbsp;&nbsp;[9]
<p>
The particular coefficient in the power series in <i>x/d<sub>a</sub></i>
should vanish for space focusing to a particular order.  For first order
space focusing, <i>f</i><sub>1</sub> should be zero, for second order
<i>f</i><sub>1</sub> and <i>f</i><sub>2</sub> should be zero, and so on.
Space focusing will not affect  last term of Eq [1] or [7], propotional
to first power of initial velocity.
<p>
Velocity focusing requires minimizing
flight time dependence on initial velocity, including first power term.
This can be obtained by delaying the accelerating field application.
Ions are produced in a field-free region, and then the first acceleration
field is delayed for some time &Delta;<i>t</i>.  Ion position is given by
<p class="eqnserif leftindent">
<i>x = v</i><sub>0</sub>&Delta;<i>t</i>
&nbsp;&nbsp;&nbsp;[10]
<p>

<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Molloy MP, Herbert BR, Walsh BJ, Tyler MI, Traini M, Sanchez JC,
Hochstrasser DF, Williams KL, Gooley AA
 </span>
 <span class="year">
(1998)
 </span>
 <span class="jname">
Electrophoresis
 </span>
 <span class="jvol">
18,
 </span>
 <span class="pages">
837-844.
 </span>
<p class="reftitl">
Extraction of Membrane Proteins by Differential Solubilization
for separation using two-dimensional gel electrophoresis
<p class="abstract">
<b>ABSTRACT</b>.
We describe the extraction and enrichment of membrane proteins
for separation by two-dimensional polyacrylamide gel electrophoresis
(2-D PAGE) after differential solubilization of an Escherichia coli
cell lysate. In a simple three-step sequential solubilization
protocol applicable for whole cell lysates, membrane proteins
are partitioned from other cellular proteins by their
insolubility in solutions conventionally used for
isoelectric focusing (IEF). As the first step, Tris-base was
used to solubilize many cytosolic proteins. The resultant pellet
was then subjected to conventional solubilizing solutions
(urea, 3-[(3-cholamidopropyl)dimethylammonio]-1-propanesulfonate,
dithiothreitol, Tris, carrier ampholytes). Following the completion
of this step, 89% of the initial E. coli sample mass was solubilized.
Finally, the membrane protein rich pellet was partially solubilized
using a combination of urea, thiourea, tributyl phosphine and
multiple zwitterionic surfactants. Using N-terminal sequence
tagging and peptide mass fingerprinting we have identified
11 membrane proteins from this pellet. Two of these outer
membrane proteins (Omp), OmpW and OmpX, have previously been
known only as an open reading frame in E. coli, while OmpC,
OmpT and OmpTOLC have not previously been identified on a
2-D gel. The prefractionation of an entire cell lysate into
multiple fractions, based on solubility, results in simplified
protein patterns following 2-D PAGE using broad-range pH 3.5-10
immobilized pH gradients (IPGs). Additional advantages of
sample prefractionation are that protein identification
and gel matching, for database construction, is a more
manageable task, the procedure requires no specialized
apparatus, and the sequential extraction is conducted in
a single centrifuge tube, minimizing protein loss.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Herbert BR, Molloy MP, Gooley AA, Walsh BJ, Bryson WG, Williams KL
 </span>
 <span class="year">(1998)
 </span>
 <span class="jname">
Electrophoresis
 </span>
 <span class="jvol">
19,
 </span>
 <span class="pages">
845-851.
 </span>
<p class="reftitl">
Improved Protein Solubility in Two-Dimensional Electrophoresis
Using Tributyl Phosphine As Reducing Agent
<p class="abstract">
<b>ABSTRACT</b>. In this study, dithiothreitol was replaced by
tributyl phosphine as the reducing agent in both the sample
solution for the first-dimensional isoelectric focusing and
during the immobilised pH gradient (IPG) equilibration procedure.
Tributyl phosphine improves protein solubility during isoelectric
focusing, which results in shorter run times and increased
resolution. Tributyl phosphine is nonionic and thus does not
migrate in the IPG, therefore maintaining reducing conditions
during the course of the first-dimensional separation. The
increased solubility provided by the maintenance of reducing
conditions gives improved focusing and decreased horizontal
streaking on the subsequent second-dimension gel. The use of
tributyl phosphine in the equilibration step allows the
procedure to be simplified, incorporating reduction and
alkylation in a single step. This is possible because,
in direct contrast to dithiothreitol (DTT), tributyl phosphine
does not contain a free thiol and therefore does not react with
thiol-specific alkylating reagents.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Papac DI, Briggs JB, Chin ET, Jones AJS
 </span>
 <span class="year">
(1998)
 </span>
 <span class="jname">
Glycobiol.
 </span>
 <span class="jvol">
8,
 </span>
 <span class="pages">
445-454.
 </span>
<p class="reftitl">
A High-Throughput Microscale Method to Release N-Linked Oligosaccharides
from Glycoproteins for Matrix-Assisted Laser Desportion/Ionization
Time-of-Flight Mass Spectrometric Analysis
<p class="Para1">
The MultiScreen-IP is a 96-well plate from Millipore (Cat #S2EM004M99,
10 plates) composed of PVDF membrane, 0.45 &micro;m pore size.  It was
used to develop a high-throughput microscale method to remove sugars.
The plate comes in a wide variety of membranes although Immobilon P
(PVDF) is used here.  The system has the ability to concentrate bound
material from dilute solutions.  An underside vacuum is created gently
from a manifold; a rubber gasket with small aperture moves effluent from
each well down.   The vacuum was not used however:  rather the bound
material is recovered by pipet and transferred to conical tubes.
<p>
Plate wells are wetted with MeOH, washed repeatedly with water, then
protein is added and reductively alkylated.  Membrane is then blocked
with PVP360 to prevent glycosidase adsorption.  After washing away block,
the glycosidase is added.  Peptide <i>N</i>-glycosidase F (PNGaseF) is
added in a pH 8.3  10 mM Tris acetate buffer (recommended buffer is
20 mM sodium phosphate pH 7.5 with 50 mM EDTA, not used because of lactone
formation on SCX resin and decrease in S/N).  Incubation time and concentration
of PNGaseF must be determined for each substrate.  This is done by comparing
the mass spectral intensity during the release of the N-linked oligosaccharide.
20 &micro;g of recombinant t-PA is added to each well and variable amounts
of glycosidase are added from 3.1 to 50 U/ml in 50 &micro;l volume.
Best conditions were 18 h when 6.3 U/ml PNGaseF added.  With 12.5 U/ml
there was no changed in spectral intensity after 2, 4, or 24 h digestion time.
Thus if a 25 U/ml treatment for 3 h, this ought to ensure thorough deglycosylation.
<p>
Reversed-phase HPLC @ 214 nm was used to monitor trypsin digestion of
rt-PA either before or after PNGaseF treatment.  50 &micro;g of rt-PA
is applied to wells and then bound material subjected to digestion.
13 peptides could be seen.  Two glycopeptides (of three expected) are
seen as the first to elute.  The T11 peptide when glycosylated comes off
at 55 min, and when deglycosylated at 58.5 min;  this is a high-mannose
type.  Unfortunately none of these peptides were ionizable in the MALDI-TOF.
<p>
Sialyated oligosaccharides are analyzed in negative ion mode.  Neutral
oligosaccharides of the high-mannose type are seen as sodium adducts
[M + Na]<sup>+</sup>, acquired in positive ion mode.  With citrate,
negative ion mode can be used [M + 191]<sup>&minus;</sup>.  Only 1% of
material was recovered, corresponding to ~850 fmol of rt-PA.  The
signal-to-noise was 10:1 even with a minor species, and the mass accuracy
was better than 0.05%.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Campagna S, Vitoux B, Humbert G, Girardet JM, Linden G, Haertle T, Gaillard JL
 </span>
 <span class="year">
(1998)
 </span>
 <span class="jname">
J. Dairy Sci.
 </span>
 <span class="jvol">
81,
 </span>
 <span class="pages">
3139-3148.
 </span>
<p class="reftitl">
Conformational Studies of a Synthetic Peptide from the Putative
Lipid-binding Domain of Bovine Milk Component PP3
<p class="Para1">
Component PP3 is a bovine milk glycosylated phosphoprotein from the
proteose-peptone fraction with 28 kDa mass and 135 amino acids.  It has
54% homology with camel whey protein.  The gene itself has 56% homology with
murine and rat glycosylation-dependent cell adhesion molecule (GlyCAM-1),
indicating an immune defense function.  The phosphorylated N-terminal domain
has two Fe(III) binding sites. and there is a suggestion of PSer residues
binding calcium ions too.  The C-terminus is proposed to have an amphipathic
&alpha;-helix and so it is a surfactant too, possibly binding lipids in
function.  PP3 has a strong affinity for an oil-water interface and it
inhibits lipoprotein lipase prresent in milk.  Nonetheless the PP3 function
is not really known.  PP3 119-135 was prepared on Perkin-Elmer Synergy 432
using 9-Fmoc chemistry.  Secondary structure prediction was done by seven
different published algorithms, and assignment to helix or sheet was made
if four of the seven programs agreed; moreover, a stable &beta;-sheet or
&alpha;-helix required at least five consecutive residues.  Hydrophobic cluster
analysis (HCA) [Garboraud et al, <i>FEBS Lett</i> <b>224</b>: 149, 1987) was
also used, with HCA alignments done by a plot program from Doriane SA (Paris, FR).
<p>
In examining the secondary structure of the peptide, comparisons were made
to camel whey protein and mouse/rat GlyCAM-1.  The C-terminal region is
residue 97-135 and predicted to have &alpha;-helix.  Several methods were used
to evaluate this and together they provide the highest confidence, while
alone, the methods are less accurate than 65% for a single sequence input.
Use of evolutionary information provided by multiple sequence alignments have
been shown to improve prediction confidence to 70%.  Other PP3 predicted
helical regions are 8-13, 19-28, 43-48 and 68-79, all reduced length.
Serines at 29, 34, 38, 40, and 46 are phosphorylated and Thr-17 and Asn-77
are glycosylated, and probably affect accuracy of prediction since models
do not assume PTMs.
<p>
An HCA plot was developed to show the relationship between the four homologous
proteins.  In the 2D pattern nearly identical features are seen in the C-terminal
region.  Superimposable hydrophobic clusters display shapes typical of amphipathic
&alpha;-helix.  The segment 33-52 in PP3 show cluster shapes readily recognized
within homologous parts of related proteins, and these two clusters may be critical
for function and maintenance of the stability of the protein.  Similarity between
C-terminal domains of artiodactyl and rodent proteins is not observable in primary
structure alignment of bovine PP3 and homologues.  Use of HCA can however aid in
detecting folding similarities despite nonconservation of fragments as detection
is not sensitive to insertions or gaps in other sequence alignment methods.
Sequence alignment and secondary structure prediction as combined analyses are
more appropriate for identifying consserved domains of protein families s matrix
choice in alignment gives less bias.  Amphipathy in residues 99-135 can be
assayed quantitatively with the Eisenberg et al helical hydrophobic moment.
A helical hydrophobic moment reaches its maximum when each residue is offset
100 degrees from the preceding (usual for &alpha;-helix).  The mean hydrophobic
moment per residue is 0.49.  Both isolation and synthesis of the peptide proved
difficult because of its extremely high aggregation potential, so the peptide
119-135 was studied isntead.
<p>
CD spectra were recorded from 10-500 &micro;M 119-135 peptide and spectral
appearance was independent of changes in concentration; thus the peptide is
a monomer in this concentration range.  Spectra show random coiling.
In 50% and 90% TFE, spectra show two bands with negative ellipticity centered
at 222 and 208 nm, where <i>n</i>&rarr;<i>&pi;</i> and
<i>&pi;</i>&rarr;<i>&pi;</i>* transitions occur, respectively.  A positive band
at 190 nm indicates another <i>&pi;</i>&rarr;<i>&pi;</i>* transition, usually
indicative of &alpha;-helix.  In CD spectra of known 3<sub>10</sub> helices
&minus;[&theta;]<sub>208</sub> &gt; &minus;[&theta;]<sub>222</sub>, although
this is controversial since one report showed completely contrasting results
and 3<sub>10</sub> and &alpha;-helices may not be distinguishable by CD.
Ellipticity for a 17-residue peptide helix is [&theta;]<sub>222</sub> = &minus;32,000
deg-cm<sup>2</sup>dmol<sup>&minus;1</sup>.  Used as a measure of 100% helicity,
the peptide show 23% helicity in 50% TFE and 29% helicity in 90% TFE.  How TFE
induces helicity is not completely understood, although Jasanoff and Fersht have
proposed how it might work.  In SDS-containing buffer, CD spectra were changed
in and SDS concentration- and SDS-to-peptide-dependent way.  10 mM SDS is sufficient
to form micelles and the CD spectrum shows 46% &alpha;-helical structure.
Increasing SDS to 50 mM does not move the helicity at all (47%).  At lower
concentrations of SDS (1.75 mM), there are two clear minima at 208 and 222,
strongly indicating &alpha;-helix.  This probably discounts the idea that formation
of micelles induces helix since 1.75 mM is below the critical micellar concentration.
It may be the case that peptide molecules accumulate SDS molecules around themselves
and create a microenvironment concentration that becomes micellar.  At [SDS] &lt;
1.75 mM CD spectrum show single minimum at 216 nm, similar to &beta;-sheet and
non-micellar SDS promotes &beta;-structures while keeping molecule soluble.  At
very low [SDS] (&lt; 1 mM), peptide 119-135 precipitates likely because &beta;-sheet
promotes aggregation.
<p>Thus amphipathic &alpha;-helix is promoted in TFE and in
SDS &gt; 1.75 mM.  The helix affinity for a lipid-aqueous interface is consistent
with amphipathic structure.  Dimyristoyl phosphatidylglycerol (DMPG) liposomes
were prepared as lipid:peptide 10:1 and CD spectra show &alpha;-helix, and
below the ratio it is &beta;-sheet (a single negative minimum at 217 nm).
When lipid:peptide exceeds 10:1 helicity decreases.  At 90:1 molar ellipticity
minima at 222 and 208 nm are nearly the same intensity, and
[&theta;]<sub>222</sub>:[&theta;]<sub>208</sub> is 1.01, thought to correspond
to coiled-coil &alpha;-helices.  Normally the ratio is lower than 1 in plane
single-stranded &alpha;-helix.
<p>In adequate concentrations of charged lipid peptide 119-135 folds into an
&alpha;-helix.  Anionic lipids (SDS) promote helical conformations and their
interaction with the positively charged C-terminal might be important.  Peptide
was dissolved L-&alpha;-lysophosphatidylcholine (LPC) which are electrically
neutral zwitterions.  At 20:1 peptide:lipid CD spectra show peptide 119-135
with double minima characterstic of &alpha;-helix.  At 203 nm is an isodichroic
point with various peptide:LPC ratios and so there is a two-state &alpha;-helix
and random coil equilibrium.  For transition from random coil to helix the
hydrophobicity.  Anionic lipid heads do not have an absolutely required
attraction to positively charged Lys residues.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Das C, Raghothama S, Balaram P
 </span>
 <span class="year">
(1998)
 </span>
 <span class="jname">
J. Am. Chem. Soc.
 </span>
 <span class="jvol">
120,
 </span>
 <span class="pages">
5812-5813
 </span>
<p class="reftitl">
A Designed Three Stranded &beta;-Sheet Peptide as a Multiple &beta;-Hairpin Model
<p class="Para1">
&beta;-Hairpins likely nucleate &beta;-sheets in protein folding.  Multiple
hairpin structures are successive strands connected by short loops of 2-5
residues and found in transmembrane &beta;-barrels which are typical of
bacterial porins.  Isolated &beta;-hairpins are stable in aqueous and
organic solvents.  Central nucleating segment can for either type I&prime;
or type II&prime; &beta;-turns, features widely seen in proteins.  Peptides
which can model/mimic will from first principles show rigorous test of
understanding of factors affecting polypeptide stereochemistry.
Purposeful design of peptides has mostly been on helical structures although
the first one targeted &beta;-sheets.  &beta;-sheets are complicated by
limited solubility and tendency of strands to aggregate in solution.
Strong residue pair correlations are not sen in &beta;-hairpins.  Many
amino acids with &beta;-branched side chains (Val, Ile, Thr) are seen in
&beta;-sheet structures and so designed peptides make use of these residues.
Type II&prime; &beta;-turn conformation is adopted by a central Pro-Xxx which
stablizes the &beta;-hairpin formation.  The pyrrolidine ring restricts
the &phi; angle in Pro to ~60&deg;, and the key determinants in &beta;-hairpin
are &beta;-turn nucleation and H-bonding between strands.
<p>
The peptide Boc-Leu-Phe-Val-dPro-<b>Gly</b>-Leu-Val-Leu-Ala-<b>dPro</b>-Gly-Phe-Val-Leu-OMe
is a 14-residue (bold residues every fifth) with two D-Pro-Gly
residues that provide the &beta;-turn, giving three strands of a 3-4-3 residue
hairpinning.  It is highly soluble in organic solvents, including chloroform.
500 MHz proton NMR in CDCl<sub>3</sub> showed sharp spectrum.  As there was
no concentration dependence seen with NH chemical shifts over range of 7.1 to
0.2 mM, this indicates that there was no aggregation effects.  When there is
a wide dispersion of NH and C<sup>&alpha;</sup>H shifts, this facilitates
complete assignment and indicative of well-defined conformational species in
solution.  A combination of TOCSY and ROESY was used to assign all backbone
proton resonance.  A table presents all chemical shifts and coupling constants
and their assignments.  Vicinal <i>J</i><sub>HNC<sup>&alpha;</sup>H</sub>
values for potential strand residues are &gt; 8.5 Hz, several &gt; 9 Hz
which indicate &phi;~120&deg; throughout the strand segments.  When the free
radical 2,2,6,6-tetramethylpiperidine-1-oxyl (TEMPO) is added there are only
two resonances, Gly5 and Gly11, which broaden significantly.  Adding
deuterated DMSO [(CD)<sub>3</sub>SO] to CDCl<sub>3</sub> would perturb only
exposed NH groups that create solute-solvent H-bonding.  From the table of
chemical shifts (&Delta;&delta; values), only four NH groups on Phe2, Gly5,
Gly11, and Val13 show large downfield shifts indicating their exposure.
Eight remaining NH groups seem solvent-inaccessible and are probably involved
in intramolecular H-bonding.  The Gly5-Leu6 and Gly11-Phe12 show two
critical N<sub><i>i</i></sub>H &harr; N<sub><i>i</i>+1</sub>H NOEs establish
the backbone dihedral anlge for residues 11 and 12 (&phi;~&minus;60&deg; &plusmn;
30&deg;; &psi;~&minus;40&deg;&plusmn;30&deg;) consistent with residues in the
<i>i</i> + 2 position of type II&prime; &beta;-turns.  There are an absence
of interresidue N<sub><i>i</i></sub>H &harr; N<sub><i>i</i>+1</sub>H NOEs in
the strand segments and there are intense interresidue
C<sup>&alpha;</sup><sub><i>i</i></sub>H &harr; N<sub><i>i</i>+1</sub>H NOEs
Multiple hairpin conformation evidence is seen be observation of cross-strand
NOEs between NH in Leu1/Leu8, Val3/Leu6, Val7/Leu14, and Ala9/Phe12.
Interstrand C<sup>&alpha;</sup><sub><i>i</i></sub>H &harr;
C<sup>&alpha;</sup><sub><i>i</i></sub>H between Phe2/Val7 establishes
the N-terminal hairpin between residues 1 and 8.  A ROESY spectrum is shown
after addition of 16% (CD<sub>3</sub>)<sub>2</sub>SO to CDCl<sub>3</sub>.
Thus Phe2/Val7 and Leu8/Val13 C<sup>&alpha;</sup><sub><i>i</i></sub>H &harr;
C<sup>&alpha;</sup><sub><i>i</i></sub>H NOEs are observed, demonstrating
presence of both the hairpin segments in the molecule.  A long-range NOE is
also seen between methyl group of C-terminal est and the Leu6 C<sup>&alpha;</sup>H,
showing that antiparallel strands remain tight even at cap ends.
<p>
With eight solvent-shielded NH groups and four solvent-exposed NH groups and
the high <i>J</i><sub>HNC<sup>&alpha;</sup>H</sub> for strand residues and
the finding of as many as seven critical interstrand long-range NOEs together
provides evidence of 3-stranded &beta;-sheet.  With 35 NOE constraints and
8 interstrand H-bonding constraints we obtain a restrained molecular dynamic
silution converged to family of 3-D structures with mean RMSD = 0.66&Aring;
for backbone atoms (see figure).
<p>
It is noted that aggregation did not occur even in a poorly solvating solvent
like chloroform.  &beta;-Hairpin formation has been reported by us with
octapeptide Boc-Leu-Val-Val-(d)Pro-Gly-Leu-Val-Val-OMe in both organic solvents
and crystals.  If only residues 1-8 are considered in one hairpin, two NH
groups on the hairpin are exposed and four are internally H-bonded.  With
a two-hairpin three-stranded structure as many as 8 internally H-bonded NH groups
while on NH groups on residues 2 and 13 are exposed; Gly NH groups in the turns
do not contribute to extended 2-D sheet formation.  With only one free NH group
on the two outer strands, aggregation is limited, as aggregation is a process
facilitated by cooperative formation of several H-bonds in sheets.  In this
peptide, any enthalpic gain by formation of limited intermolecular H bonds is
offset by entropic losses due to desolvation and restriction of side chain
flexibility and molecular motion.  Scaling this peptide design to larger
intramolecular &beta;-sheets seems possible, and a demonstration of stable
multiple hairpin formation in organic solvents is of relevance in mimicking
porin-like structures.  Note that &beta;-sheets in proteins is largely buried
in hydrophobic environments, which is consistent with importance of cross-strand
H-bonding in maintaining structural integrity.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Yuan M, Namikoshi M, Otsuki A, Watanabe MF, Rinehart KL
 </span>
 <span class="year">
(1998)
 </span>
 <span class="jname">
J. Am Soc. Mass Spectrom.
 </span>
 <span class="jvol">
10,
 </span>
 <span class="pages">
1138-1151.
 </span>
<p class="reftitl">
Electrospray Ionization Mass Spectrometric Analysis of Microcystins,
Cyclic Heptapeptide Hepatotoxins:  Modulation of Charge States and
[M + H]<sup>+</sup> and [M + Na]<sup>+</sup> Ratio
<p class="Para1">
Microcystins are cyclic 7-peptide toxins made by cyanobacteria, with
up to 60 different structural variations.  Structure is generally
of the form cyclo(<span class="smallcap">D-</span>Ala-X-<span
class="smallcap">D-</span>-MeAsp-Z-Adda-<span
class="smallcap">&nbsp;</span>.  Identification for toxological analysis by
ESI-MS is envisioned.  Microcystins form doubly-protonated ions
[M + 2H]<sup>2+</sup> in a high ratio compared to single ions
[M + H]<sup>+</sup>.  Mechanism of ionization is debated:  Wang et al
found that pH, solvent composition, and sample concentration are factors,
while Le Blanc et al found no pH dependence but that proton affinity
depended on nitrogen bases added.  Use of ammonium acetate is recommended
especially with very acidic cyclic peptides since it improves ionization.
The doubly charged forms are more likely to appear on peptides with
Arg residues.  Adduct forms ([M + Na]<sup>+</sup> and [M + K]<sup>+</sup>)
are typically found but fragmentation spectra are of no help with
adducts (thus not helping with structural info).
<p>
Microcystins-RR, -LR, -YR, and LA were injected by flow method with a
47.5:47.5:5.0 MeOH:water:acetic acid mobile phase and the <i>m/z</i>
values are reported for both +1 and +2 ions.  For the RR form the base
peak is the [M + 2H]<sup>2+</sup> ion while for the LR and YR forms
it is the singly charged.  The LA form has no Arg residue and the
[M + Na]<sup>+</sup> is the base peak.  In general presence of basic
residues like Arg results in ionization appearing in the spectrum.
<p>
Protonation sites in cyclic peptides include amide nitrogens with evidence
seen in MS/MS spectra.  Arg and Adda (a complex phenyldecadienoic acid)
can also be protonated to provide ions with very favorable free energies
of formation (calculated by Winmopac 97).
<p>
Protonation of Arg probably means a electrostatic interaction with a
Glu or MeAsp residue.  In an acidic mobile phase, any carboxylate accepts
either a proton or Na<sup>+</sup> to form [M + H]<sup>+</sup> or
[M + Na]<sup>+</sup>.  Generally positive ESI-MS gives greater sensitivity
than does negative ion ESI-MS.  Microcystin-RR in 1:1 MeOH:water shows no
ions in negative mode, but those missing an Arg residue have very high
sensitivity in negative ESI-MS.  Isopropanol is found to be a good
solvent, and increases in the pH clearly improve neg mode ESI-MS sensitivity.
An isopropanol/water mobile phase (pH 9-11) was suitable for microcystin LR.
<p>
Ion current in ESI-MS is related to analyte concentration of species
with singly charged nitrogen bases and multiply charged proteins.
The LR form was studied by direct infusion at 10 and 100 &micro;g/ml.
The base peak is the <i>m/z</i> 995 [M + H]<sup>+</sup> form at both
concentrations.  A [M + Na + H]<sup>2+</sup> and [M + K + H]<sup>2+</sup>
are present at <i>m/z</i> 509 and 517, respectively, at 25% and 10%
abundance at the higher 100 &micro;g/ml concentration.
In the lower concentration, the <i>m/z</i> 517, 509 are present at
40% adn 33% (thus the K form predominates).  The [M + 2H]<sup>2+</sup>
at <i>m/z</i> 498 is seen at 25% abundance, and this is not
present at all in the higher concentration.  Plots of detected ion
sum the singly and doubly protonated forms against the infused LR
concentration were studied using SIM mode; flow rate 200 &micro;l/min
and 5 &micro;l injection.  A very linear correlation was found with the
<i>m/z</i> 995 ion between concentration and summed ion current in the
0.1 to 1 &micro;M range.  The 2+/1+ ion ratios were examined at 0.1, 1
and 10 &micro;M.  For the various ions these were 2H/1H = 0.12 and
1Na+1H/1H = 0.32 for 10 &micro;M;  2H/1H = 0.48 and 1Na+1H/1H = 0.58
at 1 &micro;M, and 2H/1H = 0.48 and 1Na+1H/1H = 3.27 at 0.1 &micro;M.
The trend is that doubly charged ions of any kind increase inversely with
concentration.  With higher concentrations the lower abundance of
multiple charge state ions is reduced because of coulombic repulsion
which destabilizes their formation.  Mobile phase pH was also checked
from pH 3-12 using acetic acid (3-6), NaOH (9-11) and NH<sub>4</sub>OH
(11-12).  The 2H/1H ratio for LR did not change significnantly between
pH 3-11, and the 1H intensity of the RR form increased signficantly between
pH 3-6.  These results can be explained by noting that Arg-containing
microcystins form an ionic bond with COO<sup>&minus;</sup> in Glu or MeAsp
and so the response was connected with buffer pK constant.

<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Figeys D, Gygi SP, Zhang Y, Watts J, Gu M, Aebersold R
 </span>
 <span class="year">
(1998)
 </span>
 <span class="jname">
Electrophoresis
 </span>
 <span class="jvol">
19,
 </span>
 <span class="pages">
1811-1818.
 </span>
<p class="reftitl">
Electrophoresis Combined With Novel Mass Spectrometry Techniques:
Powerful Tools for The Analysis of Proteins and Proteomes
<p class="abstract">
<b>ABSTRACT</b>.
Analytical and preparative electrophoresis separation techniques
have been essential tools in protein biochemistry and the biological
sciences in general. The combination of high resolution electrophoresis
techniques with high performance analytical procedures has dramatically
enhanced analytical protein biochemistry. In this report we describe
the combination of electrophoretic separation techniques with
electrospray ionization (ESI) tandem mass spectrometry (MS/MS).
A series of different techniques, consisting of automated high
performance liquid chromatography (HPLC)-MS/MS, capillary-HPLC-MS/MS,
and solid phase extraction (SPE)-capillary zone electrophoresis
(CZE)-MS/MS, are described in the context of the identification
of high pmol to the low fmol amounts of proteins. Application of
these powerful new tools for the analysis of proteins on a large
proteome-wide scale is presented. Furthermore, the combination
of orthogonal separation techniques, such as immobilized metal
affinity chromatography (IMAC) with SPE-CZE-MS/MS, and IMAC
followed by HPLC, and by SPE-CZE-MS/MS, are presented for the
detailed investigation of post-translational modifications of
specific proteins.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Haynes PA, Gygi SP, Figeys D, Aebersold R
 </span>
 <span class="year">
(1998)
 </span>
 <span class="jname">
Electrophoresis
 </span>
 <span class="jvol">
19,
 </span>
 <span class="pages">
1862-1871.
 </span>
<p class="reftitl">
Proteome Analysis: Biological Assay or Data Archive?
<p class="abstract">
<b>REVIEW</b>
<p>
<b>ABSTRACT</b>.
In this review we examine the current state of proteome analysis.
There are three main issues discussed: why it is necessary to study
proteomes; how proteomes can be analyzed with current technology;
and how proteome analysis can be used to enhance biological research.
We conclude that proteome analysis is an essential tool in the
understanding of regulated biological systems. Current technology,
while still mostly limited to the more abundant proteins, enables
the use of proteome analysis both to establish databases of proteins
present, and to perform biological assays involving measurement of
multiple variables. We believe that the utility of proteome analysis
in future biological research will continue to be enhanced by further
improvements in analytical technology.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Chevallet M, Santoni V, Poinas A, Rouquie D, Fuchs A, Kieffer S,
Rossignol M, Lunardi J, Garin J, Rabilloud T.
 </span>
 <span class="year">
(1998)
 </span>
 <span class="jname">
Electrophoresis
 </span>
 <span class="jvol">
19,
 </span>
 <span class="pages">
1901-1909.
 </span>
<p class="reftitl">
New Zwitterionic Detergents Improve The Analysis of Membrane Proteins
by Two-Dimensional Electrophoresis
<p class="abstract">
<b>ABSTRACT</b>.
Severe quantitative loss of protein is often observed in
high-resolution two-dimensional electrophoresis of membrane proteins,
while the resolution is usually not affected. To improve
the solubility of proteins in this technique, we tested
denaturing cocktails containing various detergents and chaotropes.
Best results were obtained with a denaturing solution containing urea,
thiourea, and zwitterionic detergents, synthesized for this purpose.
Among the dozen detergents synthesized and tested, amidosulfobetaines
with an alkyl tail containing 14-16 carbons proved most efficient,
solubilizing previously undetected membrane proteins.
<!-- ============================================================= -->
<p class="tags">
FD
<p class="ref">
 <span class="authors">
Wenschuh H, Halada P, Lamer S, Jungblut P, Krause E
 </span>
 <span class="year">
(1998)
 </span>
 <span class="jname">
Rapid Comm. Mass Spectrom.
 </span>
 <span class="jvol">
12,
 </span>
 <span class="pages">
115-119.
 </span>
<p class="reftitl">
The Ease Of Peptide Detection By Matrix-Assisted Laser Desorption/Ionization
Mass Spectrometry: The Effect Of Secondary Structure On Signal Intensity
<p class="Para1">
Since Karas and Hillenkamp introduced MALDI-MS, it has proven quite useful for
peptide mass fingerprinting and able to tolerate contaminants like salts.  There
are linear relationships between ion signal intensities and petpide/protein
substrate concentration.  Other factors include use of different matrices and
altering sample prep method.
<p>
This study makes use of model peptides shown the table below:
<table style="width:120%;margin:0 -10%">
<caption>Synthetic model peptides used in this study</caption>
<tr><th>Peptide <th>Sequence  <th>Mass (av) <th>Secondary structure
  <th>Charge  <th>Hydrophobicity<sup><i>a</i></sup>
<tr><td>KLAL
  <td>Ac-KLALKLALKALKLALKLA-amide
  <td>1960.6
  <td>large &alpha;-helical propensity
  <td>+5
  <td>64.3
<tr><td>D-K, A-KLAL
  <td>Ac-KLALKLALkaLKLALKLA-amide
  <td>1960.6
  <td>diminished &alpha;-helical propensity
  <td>+5
  <td>64.3
<tr><td>VT5
  <td>DPKGDPKGVTVTVTVTVTGKGDPKPD-amide
  <td>2607.9
  <td>concentration dependent &beta;-sheet formation
  <td>+5, &minus;4
  <td>20.7
<tr><td>D-V, T-VT5
  <td>DPKGDPKGVTVTvtVTVTGKGDPKPD-amide
  <td>2607.9
  <td>no secondary structure
  <td> +5, &minus;4
  <td>20.7
<tr><td>VT6
  <td>DPKGDPKGVTVTVTVTVTVTGKGDPKPD-amide
  <td>2808.2
  <td>&beta;-sheet
  <td> +5, &minus;4
  <td>26.3
<tr><td>D-T, V-VT6
  <td>DPKGDPKGVTVTVtvTVTVTGKGDPKPD-amide
  <td>2808.2
  <td>diminished &beta;-sheet propensity
  <td>+5, &minus;4
  <td> 26.3
<tr><td>Reference peptide
  <td>Ac-Ile-His-MeAla-Lys-Ile-His-MeAla-His-Ile-amide
  <td>1108.3
  <td>no secondary structure
  <td>+4
  <td>17.8
<tr><td colspan="6">
<sup><i>a</i></sup> R<sub>c</sub> denotes retention coefficients calculated by
summing the Guo values of the amino acids.
</table>
<p>The peptide KLAL is 80% &alpha;-helix in TFE/water while VT5 and VT6 are
&beta;-sheet.  Analog peptides were constructed in the D-amino acid form
to disrupt secondary structure (they are show as lowercase letters).
MALDI analysis of 0.5 pmol of VT6 peptide gave no signal intenssity, the KLAL
and VT5 were weak.  With 1.0 and 1.5 pmolpeaks were sufficiently high enough
to detect peptide.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Marshall AG, Hendrickson CL, Jackson GS
 </span>
 <span class="year">
(1998)
 </span>
 <span class="jname">
Mass Spectrom. Rev.
 </span>
 <span class="jvol">
17,
 </span>
 <span class="pages">
1-35.
 </span>
<p class="reftitl">
Fourier Transform Ion Cyclotron Resonance Mass Spectrometry: A Primer
<p class="Para1">
FT-ICR can be understood from idealized models:
<ol>
<li>ion cyclotron frequency, radius, velocity, and energy as a function of
ion mass, ion charge, and magnetic field strength follow directly from the
motion of an ion in a spatially uniform static magnetic field
<li>ion cyclotron motion may be rendered spatially coherent (and thus observable)
by the application of a spatially uniform rf electric field (excitation)
at the same frequency as (i.e., <q>resonant</q> with) the ion cycltron 
frequency; the ICR (time domain) signal result from induction (detection)
of an oscillating image charge on two conductive infinitely extended opposed
parallel electrodes; a frequence domain is obtained by Fourier transformaion
of digitized time domain ICR signal and this converts to mass domain spectrum
<li>ions are confined by a 3D axial quadrupolar dc electric field and this
shifts ion cyclotron frequency whereas excitation and detection remain
linear
<li>collisions broaden the ICR signal in a simple way:  make it possible to
cool and compress ion packet for improved detection and even multiple
remeasurement
<li>although FT-ICR MS has been coupled to nearly every ion source, most
ion sources work best outside the magnet and so several methods are developed
to guide externally generated ions into the trap
<li>all above features may be combined in various experimental <q>event
sequences</q> (much like pulse sequences in FT-NMR spectroscopy) to perform
tandem-in-time mass spectrometery (MS/MS or MS<sup>n</sup>)
</ol>
<p>
A spatially uniform magnetic field is established <i><b>B</b></i> =
&minus;<i>B</i><sub>0</sub><b>k</b> and the <i>z</i>-axis proceeds in
direction opposite <i>B</i>.  An ion moves in the presence and experiences
the force:
<p class="eqnserif">
<i>F = ma = m</i> (<i>dv / dt</i>) = <i>qv &times; B</i>&nbsp;&nbsp;&nbsp;[1]
<p>
and <i>m, q</i> and <i>v</i> are the mass, charge, and velocity of the ion.
Note the cross product of the velocity and magnetic field vectors (charge
is scalar).  This describes the ion cyclotron motion.  If the ion is 
negative, it will move in a clockwise motion if <i><b>B</b></i> is directed
perpendicular in the direction of the reader's view (not direction of the 
reader).  If the ion is positively charged, the movement is counterclockwise
in the field.  Alternatively, if the thumb points in the direction of 
<b><i>B</i></b>, for positive ions, the curl of the left hand describes their
motion while for negative ions, it is the curl of the right hand.
<p>
Because velocity is circular and not linear, it is described by an <i>x</i>
and <i>y</i> (planar or 2D) components.  The angular acceleration will be
|<i>d<b>v</b>/dt</i>| = <i>v<sub>xy</sub></i><sup>2</sup> / <i>r</i>, and
so eqn [1] is substituted as follows:
<p class="eqnserif">
<i>mv<sub>xy</sub></i><sup>2</sup> / <i>r</i> = <i>qv<sub>xy</sub> &times; 
B</i>&nbsp;&nbsp;&nbsp;[2]
<p>
The angular velocity &omega; (rad/s) about the <i>z</i> axis is 
&omega; = <i>v<sub>xy</sub> / r</i> and 
<p class="eqnserif">
<i>m&omega;<sup>2</sup>r = q&omega;rB</i><sub>0</sub><br>
<i>&omega; = qB</i><sub>0</sub>/<i>m</i>&nbsp;&nbsp;&nbsp;[3]
<p>
Eqn [3] is the simplification.  Dividing <i>&omega;</i> (rad/s) by 
2<i>&pi;</i> rad(ians) gives a value in s<sup>&minus;</sup>, which is the
frequency of motion <i>&nu;<sub>c</sub></i>.  Moreover charge <i>q =
ze</i>, where <i>z</i> is an integer multiple of the universal unit of
charge <i>e</i> = 1.602 &times; 10<sup>&minus;19</sup> C, true for all 
protons and electrons that confer charge.  With all the substitutions,
it can be shown that:
<p class="eqnserif">
<i>&nu;<sub>c</sub> = &omega;</i>/2<i>&pi; = k</i> 
{<i>B</i><sub>0</sub> / (<i>m/z</i>)}&nbsp;&nbsp;&nbsp;[4]
<p>
where <i>k</i> = (1.602 &times; 10<sup>&minus;19</sup> C) / {(2<i>&pi;</i>) 
1.660 &times; 10<sup>&minus;27</sup> kg/u)} = 1.535611 &times; 10<sup>7</sup> Hz 
&bull; amu / T, if <i>B</i> is in tesla (T, 1 T = 1 kg / C s), and <i>m</i> is 
in mass units (u).   Eqn [4] is the cyclotron equation.  All ions with the same 
<i>m/z</i> have the same frequency and this is independent of their velocity: 
focusing on translational energy is not essential to determine precisely 
<i>m/z</i>.  At a typical static magnetic field of 7.0 T, ICR frequencies should 
range from a few kHz to a few MHz for the usual range of molecules (plot log 
<i>&nu;<sub>c</sub></i> from 1.0 to 10<sup>9</sup> Hz against log <i>m/z</i> from 
1 to 10<sup>7</sup> at different magnetic field strengths from 1.0 to 9.4 T to 
see straight lines with negative slopes).
<p>
Rearranging Eq [2]:
<p class="eqnserif">
<i>r = mv<sub>xy</sub> / qB</i><sub>0</sub> = <i>mv<sub>xy</sub> / zeB</i><sub>0</sub>&nbsp;&nbsp;&nbsp;[5]<br>
<i>r</i> = (1.036427 &times; 10<sup>&minus;8</sup> T s / m amu)
(<i>m/z</i>)(<i>v<sub>xy</sub> / B</i><sub>0</sub>)
<p>
if <i>r</i> is in meters, <i>B</i><sub>0</sub> is in tesla, <i>m</i> is in u, 
and <i>v<sub>xy</sub></i> is in m/s.  This is the ion cyclotron radius for ion 
with velocity <i>v<sub>xy</sub></i> (plot <i>r</i> in mm from 0 to 2.5 against 
<i>m/z</i> from 0 to 10,000 at different <i>B</i><sub>0</sub> from 1.0 to 9.4 T 
to see hyperbolic lines).  The average <i>x-y</i> translational energy of an ion 
in equilibrium with surroundings at temperature <i>T</i> is 
<p class="eqnserif">
&frac12; <i>m</i>&lang;<i>v<sub>xy</sub></i><sup>2</sup>&rang;  &#8776; <i>kT</i>&nbsp;&nbsp;&nbsp;[6]
<p>
where <i>k</i> is the Boltzmann constant.  Rearranging Eqn [6] for <i>v<sub>xy</sub></i>
and then substituting for Eqn [5]:
<p class="eqnserif">
 <i>r</i> = (2<i>mkT</i>)<sup>&frac12;</sup> / <i>zeB</i><sub>0</sub><br>
 <i>r</i> = 1.336510 &times; 10<sup>&minus;6</sup> (<i>mT</i>)<sup>&frac12;</sup> 
/ <i>zB</i><sub>0</sub> &nbsp;&nbsp;&nbsp;[7]
<p>
if all variables are in the units previously mentioned.  At room temperature
(<i>T</i> = 298 K) a <i>z</i> = 1 ion of mass <i>m</i> = 100 u in a magnetic
field of <i>B</i><sub>0</sub>= 3 T (30,000 Gausss) will have an orbital
radius ~ 0.08 mm.  If <i>m</i> = 10,000, the radius is 10-fold greater.  If Eqn
[5] is rearranged:
<p class="eqnserif">
<i>v<sub>xy</sub></i> = <i>qrB</i><sub>0</sub>/<i>m</i> = <i>zerB</i><sub>0</sub>/<i>m</i><br>
<i>v<sub>xy</sub></i> = (9.64853 &times; 10<sup>7</sup> u/T s)(<i>rB</i><sub>0</sub>)/(<i>m/z</i>)&nbsp;&nbsp;&nbsp;[8]
<p>
A 100 u z=1 ion with an orbital radius of 1 cm in a 3.0 T field will have a
translational velocity of <i>v<sub>xy</sub></i> = 2.97 &times; 10<sup>4</sup>
and its kinetic energy is:
<p class="eqnserif">
Kinetic energy = &frac12; <i>mv<sub>xy</sub></i><sup>2</sup> = 
(<i>zerB</i><sub>0</sub>)<sup>2</sup> / 2<i>m</i> <br>
K.E. = (4.824265 &times; 10<sup>7</sup> eV u / T<sup>2</sup> m<sup>2</sup>) 
(<i>zrB</i><sub>0</sub>)<sup>2</sup> / <i>m</i>&nbsp;&nbsp;&nbsp;[9]
<p>
computes to 434 eV.  Plots of <i>E</i> ranging from 0 to 10 keV vs <i>r</i>
ranging from 0 to 1.5 cm under different <i>B</i><sub>0</sub> ranging from 1.0
to 9.4 T show plots of a parabola.  It should be noted that the ion will travel
nearly 30 km in one second and this offers higher mass resolution compared
to a several meter long magnetic sector beam instrument.
<p>
<b>Azimuthal Dipolar Frequency Excitation</b>.
The cyclotron motion is not itself useful.  What is useful is the excitation 
of the ion using a spatially uniform electric field that oscillates (or rotates)
at or near the frequency of the cyclotron frequency of ion of a particular 
<i>m/z</i>.  The excitation occurs in three ways:
<ol>
<li>accelerating ions coherently to a larger detectable orbital radius
<li>increasing ion kinetic energy above threshold for ion dissocation or
ion-molecule reaction
<li>accelerating ions to cyclotron radius for greater than the trap radius so
they are ejected
</ol>
<p>
Applying an <em>azimuthal</em> (perpendicular to plane of <i><b>B</b></i>)
spatially uniform electric field which oscillates sinusoidally with time along 
the &plusmn; <i>y</i> axis
<p class="eqnserif">
<i><b>E</b></i>(<i>t</i>) =  <i>E</i><sub>0</sub> 
cos <i>&omega;<sub>c</sub>t</i><b>j</b>&nbsp;&nbsp;&nbsp;[10]
<p>
Note that <b>i</b> and <b>j</b> are unit vectors along the <i>x-</i> and 
<i>y</i>-axes; the <i>y</i> axis passes through the two plates perpendicularly, 
whereas the <i>xz</i> plane is parallel to the plates. <i>E</i><sub>0</sub> is 
generated by two opposed infinitely extended flat parallel conductive flat 
plates oscillating at +<i>V</i><sub>0</sub> and &minus;<i>V</i><sub>0</sub> 
separated by <i>d</i> meters, so that 
<p class="eqnserif">
<i>E</i><sub>0</sub> = 2 <i>V</i><sub>0</sub>/<i>d</i> = 
<i>V<sub>p-p</sub></i>/<i>d</i>&nbsp;&nbsp;&nbsp;[11]
<p>
The linearly polarized electric field may be analyzed into two counter-rotating 
components <i><b>E</b><sub>L</sub></i>(<i>t</i>) and 
<i><b>E</b><sub>R</sub></i>(<i>t</i>)
<p class="eqnserif">
<i><b>E</b></i>(<i>t</i>) = <i><b>E</b><sub>L</sub></i>(<i>t</i>) +
<i><b>E</b><sub>R</sub></i>(<i>t</i>)&nbsp;&nbsp;&nbsp;[11a]
<p>
where
<p class="eqnserif">
<i><b>E</b><sub>R</sub></i>(<i>t</i>) = <i>E</i><sub>0</sub>/2 
  (cos <i>&omega;t</i><b>j</b> + sin <i>&omega;t</i><b>i</b>)&nbsp;&nbsp;&nbsp;[11b]<br>
<i><b>E</b><sub>L</sub></i>(<i>t</i>) = <i>E</i><sub>0</sub>/2 
  (cos <i>&omega;t</i><b>j</b> &minus; sin <i>&omega;t</i><b>i</b>)&nbsp;&nbsp;&nbsp;[11c]
<p>
The rf electric field component, <i><b>E</b><sub>R</sub></i>(<i>t</i>), rotates
in the same sense (viz. counterclockwise in <i>xy</i> plane) and at same 
frequency (i.e. in resonance with) the ion of interest pushes that ion continuously
forward in its orbit.  The electric field component, <i><b>E</b><sub>L</sub></i>(<i>t</i>),
which rotates in the opposite sense of the ion, is off-resonance by 2<i>&omega;</i>
and has virtually no significant net effect after several cycles of the excitation
frequency.
<p>
Ions absorb power according to the dot product of the electric field force
and the velocity vector
<p class="eqnserif">
<i>A</i>(<i>t</i>) = Force &bull; velocity = <i>qE</i>(<i>t</i>) &bull; <i>v<sub>xy</sub></i>
&nbsp;&nbsp;&nbsp;[12]
<p>
For positive or negative ions at rest initially and subjected to oscillating 
resonant excitation (Eqs 10 and 11) for a period <i>T</i><sub>excite</sub>
from Eq 12 it can be shown that the post-excitation ion cyclotron radius <i>r</i> is
<p class="eqnserif">
<i>r = E</i><sub>0</sub><i>T</i><sub>excite</sub>/2<i>B</i><sub>0</sub>&nbsp;&nbsp;&nbsp;[13a]<br>
<i>r = V<sub>p-p</sub>T</i><sub>excite</sub>/2<i>dB</i><sub>0</sub>&nbsp;&nbsp;&nbsp;[13b]
<p>
The excitation radius is independent of <i>m/z</i>.  All ions of given <i>m/z</i>
range can be excited to same orbital radius with rf electric field whose 
magnitude is constant with frequency.  Ion kinetic energy following azimuthal
dipolar resonant excitation for period <i>T</i><sub>excite</sub> is related
to <i>&omega;<sub>c</sub> = qB</i><sub>0</sub>/<i>m</i>:
<p class="eqnserif">
KE<sub>post-excitation</sub> = <i>m&omega;</i><sup>2</sup><i>r</i><sup>2</sup> / 2
= (<i>qE</i><sub>0</sub><i>T</i><sub>excite</sub>)<sup>2</sup>/8<i>m</i>
= (<i>qV<sub>p-p</sub>T</i><sub>excite</sub>)<sup>2</sup>/8<i>md</i><sup>2</sup>
&nbsp;&nbsp;&nbsp;[14a]<br>
KE<sub>post-excitation</sub> = (1.20607 &times; 10<sup>7</sup> eV u m<sup>2</sup>/
V<sup>2</sup> s<sup>2</sup>)(<i>zV<sub>p-p</sub>T</i><sub>excite</sub>)<sup>2</sup>/<i>md</i><sup>2</sup>
&nbsp;&nbsp;&nbsp;[14b]
<p>
A 100 u <i>z</i>=1 ion excited by &plusmn; 1 V applied for 1 ms to infinitely 
extended parallel flat electrodes 2 cm apart should have post-excitation energy 
of 1.2 keV. High collision energies are easy to generate.  Post-excitation 
energy is not dependent on magnetic field strength <i>B</i><sub>0</sub>, but the 
radius does vary inversely with <i>B</i><sub>0</sub>, meaning that with a given 
post-excitation radius, and the energy increases with square of the magnetic 
field. Plots of log energy <i>E</i> (in eV, ranging from 10<sup>&minus;3</sup> 
to 10<sup>7</sup>) vs. log <i>m/z</i> (from 10<sup>&minus;1</sup> to 10<sup>6</sup>) 
produce several straight plots with negative slopes when <i>B</i><sub>0</sub> is 
varied from 1.0 to 9.4 T, all lines with a constant <i>r</i> = 1 cm.  In another
log <i>E</i> vs. log <i>m/z</i> plot, <i>V<sub>p-p</sub></i> = 1 V and <i>d</i> = 2 cm
for the plate separation and applied peak-to-peak voltage, and <i>T</i><sub>excite</sub>
is varied from 100 &micro;s to 1 s logarithmically.  For a 1000 u z=1 ion, 
a 1 s excitation can impart as much as 10<sup>7</sup> eV.  Thus the FT-ICR might
be equipped with ability to change energy in the ion by varying the magnetic
field, or by altering <i>V<sub>p-p</sub></i> or <i>T</i><sub>excite</sub>. 
<p>
<b>Azimuthal Dipolar Single-Frequency Detection</b>.   ICR motion does not 
generate an observable signal, namely the net difference between charge induced
in two opposed parallel electrodes.  When ions are injected into the trap and
the cyclotron motion forms, orbital motion phase is <em>random</em>.  For an 
ion ensemble, charge induced in either of two opposed detector plates is balanced
on average by equal charge induced by an ion with 180&deg; out of phase, meaning
on far side of orbit, and net difference is zero between two plates.  Even if 
all ions with same <i>m/z</i> were in phase in the orbit, the radius is too small
to induce detection signals without the excitation.  But with resonant excitation,
a <em>spatial coherence</em> occurs and detection enabled; orbits are actually
centered on the <i>z</i>-axis, and with the increase in orbital radius the ion
packet is moved significantly off axis.
<p>
The difference in <q>image</q> charge, &Delta;<i>Q</i> induced by an ion of charge
<i>q</i> and the time variation of that signal are shown:
<p class="eqnserif">
&Delta;<i>Q</i> = &minus;2<i>qy/d</i>&nbsp;&nbsp;&nbsp;[15]<br>
<i>d</i>&Delta;<i>Q/dt</i> = &minus;2<i>q</i>(<i>dy/dt</i>)<i>/d</i>
<p>
There is no dependency on magnetic field strength.  As the <i>y</i>-axis is
perpendicular to the two detection plates (passes through them), induction is
clearly a function of the position of the ions in along <i>y</i> in the orbit,
which many factors affect.  From Eq 5, increases in the <i>y</i> component of 
velocity (<i>dy/dt</i> &gt; 0) should cause linear increases in orbital radius.
In addition detected signal in more sensitive to multiply charged (<i>z</i> &gt; 1)
ions (eg. from ESI).  Linearity is important because (i) ICR signal varies with
ion cyclotron post-excitation radius and this depends linearly on excitation
voltage amplitude-duration product, and the ICR response at any frequency is
proportional to excitation spectral magnitude at that frequency; (ii) a
Fourier transform of time-domain ICR response produces same absorption spectrum
that would have been obtained by measuring power absorption while sweeping
across the <i>m/z</i> range.  The <q>superposition</q> principle implies that
signals from any number of ions of arbitrary <i>m/z</i> values just add at the
detector, and ions of a wide <i>m/z</i> range can be detected simultaneously.
Thus a multichannel advantage exists in pulsed excitation followed by Fourier
transformation yields a spectrum of <i>N</i> data points in 1/<i>N</i> the
time it takes to scan the spectrum one channel at a time.
<p>
<b>Broadband excitation</b>. Having looked at response of an ion in a static
homogeneous magnetic field <i><b>B</b> = &minus;B</i><sub>0</sub><i><b>k</b></i>
to a oscillating electric field with single frequency resonant 
(<i><b>E</b></i>(<i>t</i>) = <i>E</i><sub>0</sub>cos <i>&omega;<sub>c</sub>t</i><b>j</b>
it can be seen (in a Fourier transform of <i>f</i>(<i>t</i>)&nbsp;vs.&nbsp; <i>t</i> &rarr;
<i>M</i>(<i>&omega;</i>)&nbsp;vs.&nbsp;<i>&omega;</i>) that other frequencies are
excited as well, the peak width being <i>m/</i>&Delta;<i>m</i><sub>50%</sub>
The frequency domain spectrum, <i>E</i>(<i>&nu;</i>) of a single frequency
excitation at <i>&omega;<sub>c</sub></i> = 2<i>&pi;&nu;<sub>c</sub></i>
when applied steadily for <i>T</i><sub>excite</sub> seconds is given by 
<p class="eqnserif">
<i>E</i>(<i>&nu;</i>) = [<i>E</i><sub>0</sub> sin(2<i>&pi;&nu;<sub>c</sub>T</i><sub>excite</sub>) 
] / 2<i>&pi;&nu;<sub>c</sub></i>&nbsp;&nbsp;&nbsp;[16]
<p>
While the applied excitation may be precisely centered at frequency <i>&omega;</i>
which differs from <i>&omega;<sub>c</sub></i>, turning excitation voltage
on at time zero and off at <i>T</i><sub>excite</sub> broadens the spectral
range of excitation to bandwidth of ~1/<i>T</i><sub>excite</sub> rad/s.
Thus the longer the time domain signal duration, the narrower the frequency
domain spectral width.  If an off-resonsance excitation at frequency <i>&nu;</i>
is sufficiently brief such that (1/<i>T</i><sub>excite</sub> &ge; 
2<i>&pi;</i> |<i>&nu;&minus;&nu;<sub>c</sub></i>|, it will still excite ions
at cyclotron frequency <i>&omega;<sub>c</sub></i>.  This defines the precision
by which ions can be mass-selected.  Ion cyclotron frequencies are so high that
as little as once second can be very mass-selective: ~10 ppm for <i>m/z</i>
1000  at 7.0 T.  Eq 16 shows single freq exictation of duration <i>T</i><sub>excite</sub>
will excite ions with frequencies whose range span ~0.1/<i>T</i><sub>excite</sub>
<p>
If a single frequency resonant excitation voltage of &plusmn; 1 V is applied to
electrodes 2 cm apart to excite to a radius of 1 cm in 1.4 ms, excitation of 
ions anywhere within bandwidth of ~1 MHz will require excitation of ~0.1 &micro;s
corresponding to a &plusmn; 10,000 V rf excitation voltage, inconviently large.
Thus a broadband ICR excitation is done by a frequency-sweep (<q>chirp</q>)
excitation, achieving a relatively flat magnitude of excitation over broad
frequency rang using relatively low excitation voltage.  Disadvantages of
the chirp excitation: a nonuniform excitation amplitude leading to variations
in ion relative abundances, and limited mass selectivity at the start and
end of the sweep.  To solve this problem, one instead defines the excitation
in the mass profile which is then converted to the frequency-domain excitation
spectrum, followed by an inverse Fourier transform to get the time-domain
excitation waveform.  One waveform may just contain excitation that keep the
ions in the trap; another may select which ions get ejected to enrich for
low abundance ions.  Mass spectral <q>windows</q> within which none or all
ions are excited or ejected provide for mass selectivity for MS/MS or 
enhanced dynamic range by selective ejection of highly abundant ions.  The
use of the <b>stored waveform inverse Fourier transform</b> (SWIFT) (described
above) achieves the flattest and most frequency-selected excitation magnitude
spectrum theoretically possible for a given time-domain excitation period.
See Guan and Marshall for how SWIFT excitation is used for the Penning ICR
and Paul quadrupole ion traps.
<p>
<b>Broadband Detection and Limits</b>.  Coherently orbiting ion packets induce
differential currents between two opposed detection plates and modeled as
a current source.  The receiver plates and wiring connecting the ion trap to
the detection preamplifier have an inherent resistance and capacitance in
parallel.  At typical ICR frequencies (&gt; 10 kHz) the signal-to-noise 
is independent of cyclotron frequency.  But at lower ICR frequencies (&lt;
10 kHz), signal varies directly with frequency.  Detected S/N ratio reflects
the relative current differential induced on detection plates throughout
most of frequency range excited by broadband chirp or SWIFT waveform.
Detection limit is calculated from:
<p class="eqnserif">
<i>N</i> = [<i>CV</i><sub><i>d</i>(<i>p-p</i>)</sub>]/[<i>qA</i><sub>1</sub>(<i>r</i>)]
&nbsp;&nbsp;&nbsp;[17]
<p>
where <i>C</i> is capacitance in detection circuit, 
<i>V</i><sub><i>d</i>(<i>p-p</i>)</sub> is peak-to-peak amplitude of detected 
voltage (calibrated for given spectrometer) and <i>A</i><sub>1</sub>(<i>r</i>) 
is coefficient approx. proportional to <i>r</i> and is determined graphically. 
<i>N</i> is the minimum number of ions which can be detected from an undamped 
signal in a single 1 s acquisition period to produce S/N=3:1.  For a typical 
operating circuit, <i>C</i> = 50 pF, <i>V</i><sub><i>d</i>(<i>p-p</i>)</sub> = 3 
&times; 10<sup>&minus;7</sup> V, and <i>A</i><sub>1</sub>(<i>r</i>) = 0.5 
(meaning the ion is excited to about half of its maximal cyclotron radius), the 
S/N=3:1 means that 187 ions is the detection limit.
<p>
<b>Ion-Neutral Collisions</b>.   Eq 16 describe what happens to ions with
a single value <i>m/z</i> in the absence of collision, obtained by Fourier
transformat of a digitized time-domain signal to give a <q>sinc</q> function
spectrum looking like Eq. 9, except <i>T</i><sub>excitation</sub> is
<i>T</i><sub>acquisition</sub>.  Ion-neutral collisions are model as an
effect of a frictional damping force:
<p class="eqnserif">
Force = <i>m</i> (<i>dv/dt</i>) = <i>q<b>E</b> + qv &times; <b>B</b> 
&minus; f</i><b>v</b>&nbsp;&nbsp;&nbsp;[18]
<p>
The model leads to an exponential damping of the time-domain ICR signal
for <i>N<sub>i</sub></i> ions of ICR frequency <i>&nu;<sub>i</sub></i>:
<p class="eqnserif">
<i>s<sub>i</sub></i>(<i>t</i>) &prop; <i>N<sub>i</sub></i> 
exp(&minus;<i>t/&tau;<sub>i</sub></i>) cos(2<i>&pi;&nu;<sub>i</sub>t</i>)&nbsp;&nbsp;&nbsp;[19]
<p>
where the Fourier transform produces a Lorentzian frequency-domain magnitude-mode
spectrum:
<p class="eqnserif">
<i>M</i>(<i>&omega;</i>) &prop; <i>&tau;/</i>[1 + (<i>&omega;</i> &minus; 
2<i>&pi;&nu;<sub>i</sub></i>]<sup>&frac12;</sup>
<p>
FT-ICR spectral line shape approaches the <q>sinc</q> shape in the limit
of zero pressure (<i>&tau; &gt;&gt; T</i><sub>acq'n</sub>, with essentially
no ion-neutral collisions during data acquisition).
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Blom N, Gammeltoft S, Brunak S
 </span>
 <span class="year">
(1999)
 </span>
 <span class="jname">
J. Mol. Biol.
 </span>
 <span class="jvol">
294,
 </span>
 <span class="pages">
1351-1362.
 </span>
<p class="reftitl">
Sequence and Structure-Based Prediction of Eukaryotic Protein Phosphorylation
Sites
<p class="Para1">

Protein phosphorylation at serine, threonine or tyrosine residues affects a 
multitude of cellular signaling processes. How is specicity in substrate 
recognition and phosphorylation by protein kinases achieved? Here, we present an 
articial neural network method that predicts phosphorylation sites in 
independent sequences with a sensitivity in the range from 69% to 96 %. As an 
example, we predict novel phosphorylation sites in the p300/CBP protein that may 
regulate interaction with transcription factors and histone acetyltransferase 
activity. In addition, serine and threonine residues in p300/CBP that can be 
modified by O-linked glycosylation with N-acetylglucosamine are identied. 
Glycosylation may prevent phosphorylation at these sites, a mechanism named 
yin-yang regulation. The prediction server is available on the Internet at <a 
href="http://www.cbs.dtu.dk/services/NetPhos/">http://www.cbs.dtu.dk/services/NetPhos/</a>
or via e-mail to <a href="mailto:NetPhos@cbs.dtu.dk">NetPhos@cbs.dtu.dk</a>.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Gygi SP, Han DK, Gingras AC, Sonenberg N, Aebersold R
 </span>
 <span class="year">
(1999)
 </span>
 <span class="jname">
Electrophoresis
 </span>
 <span class="jvol">
20,
 </span>
 <span class="pages">
310-319.
 </span>
<p class="reftitl">
Protein Analysis by Mass Spectrometry and Sequence Database Searching:
Tools for Cancer Research in The Post-Genomic Era
<p class="abstract">
<b>ABSTRACT</b>.
The post-genomic era is characterized by the deposition of sequence information 
for entire genomes in databases. Currently, besides the protein sequences for 
known human proteins, there are partial sequences from thousands more human 
proteins for which no biological function has been assigned. A powerful new tool 
for the unambiguous identification and characterization of gel-separated 
proteins is accomplished by the combination of mass spectrometry and sequence 
database searching. This combination provides the cancer biologist with the 
ability to (i) identify the potential protein:protein associations and (ii) 
fully characterize function-critical post-translational modifications, both 
directly from silver-stained polyacrylamide gels. In this report we describe the 
application of tandem mass spectrometry and database searching to two problems 
which are prototypical for cancer research and indeed for biomedical research in 
general. The first is the identification of gel-separated, low abundance 
proteins based on amino acid sequence composition following 
coimmunoprecipitation with the human apoptosis inhibitor protein BclX(L). The 
second is the determination of the precise sites of phosphorylation of the human 
regulatory protein 4E-BP1, which controls mRNA translation.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Molloy MP, Herbert BR, Williams KL, Gooley AA
 </span>
 <span class="year">
(1999)
 </span>
 <span class="jname">
Electrophoresis
 </span>
 <span class="jvol">
20,
 </span>
 <span class="pages">
701-704.
 </span>
<p class="reftitl">
Extraction of Escherichia coli proteins with organic
solvents prior to two-dimensional electrophoresis
<p class="abstract">
<b>ABSTRACT</b>.
Compared to soluble proteins, hydrophobic proteins, in particular
membrane proteins, are an underrepresented protein species on
two-dimensional (2-D) gels. One possibility
is that many hydrophobic proteins are simply not extracted from the sample prior to
2-D gel separation. We attempted to isolate hydrophobic proteins from Escherichia coli
by extracting with organic solvents, then reconstituting the extracted proteins in highly
solubilising sample solution amenable to 2-D electrophoresis using immobilized pH
gradients (IPGs). This was conducted by an extraction with a mixture of chloroform
and methanol, followed by solubilisation using a combination of urea, thiourea, sulfobetaine
detergents and tributyl phosphine. Peptide mass fingerprinting assisted in the
identification of 13 proteins, 8 of which have not previously been reported on 2-D gels.
Five of these new proteins possess a positive hydropathy plot. These results suggest
that organic solvent extractions may be useful for selectively isolating some proteins
that have previously been missing from proteome maps.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Santoni V, Rabilloud T, Doumas P, Rouquie D, Mansion M, Kieffer S,
Garin J, Rossignol M
 </span>
 <span class="year">
(1999)
 </span>
 <span class="jname">
Electrophoresis
 </span>
 <span class="jvol">
20,
 </span>
 <span class="pages">
705-711.
 </span>
<p class="reftitl">
Towards the recovery of hydrophobic proteins on two-dimensional
electrophoresis gels
<p class="abstract">
<b>ABSTRACT</b>.
An extensive proteomic approach relies on the possibility to visualize
and analyze various types of proteins, including hydrophobic proteins
which are rarely detectable on two-dimensional electrophoresis (2-DE)
gels. In this study, two methods were employed for the purification
of hydrophobic proteins from Arabidopsis thaliana leaf plasma membrane
(PM) model plants, prior to analysis on 2-DE immobilized pH gradient
(IPG) gels. Solubilization efficiency of two detergents,
(3-[(3-cholomidopropyl)-1-propanesulfonic acid (CHAPS) and C8phi,
were tested for the recovery of hydrophobic proteins. An
immunological approach was used to determine the efficiency of
the above methods. Fractionation of proteins by Triton X-114
combined with solubilization with CHAPS resulted in the inability
to detect hydrophobic proteins on 2-DE gels. The use of C8phi for
protein solubilization did not improve this result. On the contrary,
after treatment of membranes with alkaline buffer, the solubilization
of PM proteins with detergent C8phi permitted the recovery of such
proteins on 2-DE gels. The combination of membrane washing and the
use of zwitterionic detergent resulted in the resolution of several
integral proteins and the disappearance of peripheral proteins.
In the resolution of expressed genome proteins, both large pH gradients
in the first dimension and various acrylamide concentrations
in the second dimension must be used. Notwithstanding, it is important
to combine various sample treatments and different detergents
in order to resolve soluble and hydrophobic proteins.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
G&ouml;rg A, Obermaier C, Boguth G, Weiss W
 </span>
 <span class="year">
(1999)
 </span>
 <span class="jname">
Electrophoresis
 </span>
 <span class="jvol">
20,
 </span>
 <span class="pages">
712-717.
 </span>
<p class="reftitl">
Recent developments in two-dimensional gel
electrophoresis with immobilized pH gradients:
Wide pH gradients up to pH 12, longer separation
distances and simplified procedures
<p class="abstract">
<b>ABSTRACT</b>.
Wide-range immobilized pH 312 and 612 gradients were generated. Depending on
the extraction method of sample preparation, proteins with pIs up to pH 11.7 were
resolved. Highly reproducible protein patterns, focused to the steady-state with
round-shaped spots up to the basic end were obtained. Moreover, because a strong
water transport from cathode to anode (reverse electroendosmotic flow) inherent to
narrow immobilized pH gradients (IPGs) exceeding pH 11, such as IPG 1012, was
negligible, the wide-range IPGs 312 and 612 could be run under standard conditions
as originally described by Grg et al. (Electrophoresis 1988, 9, 531546). The
wide-range immobilized pH gradient 312 proved to be perfectly suited for an overview
separation of total cell extracts. Resolution could be increased by extending the
separation distance from 18 to 24 cm. Furthermore, two-dimensional gel electrophoresis
with IPGs (IPG-Dalt) was simplified by the use of an integrated system
(IPGphor) where sample application by in-gel rehydration and isoelectric focusing
(IEF) are performed automatically in a one-step procedure, overnight, without human
assistance.

<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Feny&ouml; D
 </span>
 <span class="year">
(1999)
 </span>
 <span class="jname">
Bioinformatics
 </span>
 <span class="jvol">
15,
 </span>
 <span class="pages">
339-340.
 </span>
<p class="reftitl">
The Biopolymer Markup Language
<p class="abstract">
<b>ABSTRACT</b>.
SUMMARY: An XML derived from a data model designed to be a hierarchical
representation of an organism has been specified and a browser to use
this language has been developed. AVAILABILITY: The language definition
is available in HTML form at
<a href="http://www.proteometrics.com/BIOML/">Proteometrics</a>.
The BioML browser is available on request from the author.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Peleg L, Ries L, Getslev V, Lusky A, Chaki R, Lipitz S, Barkai G
 </span>
 <span class="year">
(1999)
 </span>
 <span class="jname">
Prenat. Diag.
 </span>
 <span class="jvol">
19,
 </span>
 <span class="pages">
224-228.
 </span>
<p class="reftitl">
Heat Stable and Urea Resistant Alkaline Phosphatase in Maternal Neutrophils
from Normal and Down Syndrome Pregnancies
<p class="Para1">
Human alkaline phosphatase (ALP) has four isoenzymes.  The placental and 
germ-line enzymes (placental ALP = pALP) can be used to assess intra-uterine 
growth retardation and low birth weight.  pALP increases in maternal serum and 
it was thought it might be a biochemical indicator along with the triple test.
<p>
Blood was taken from mothers in 1st and 2nd trimester pregnancies who also went 
other screening (amniocentesis, chorionic villus sampling), and also from 3rd 
trimester followups.  Fetuses with Downs were determined by prenatal karyotyping 
and blood from aborted fetuses.  Sera were taken from 7 ml blood and stored deep 
freeze.  PMN cells were from 7 ml heparin venous blood with neutrophil yield of 
91%.  Cells were sonicated (130 W) 30 s on-30 s off three times.  Extracts were 
spun 16900 <i>g</i> 40 min at 4&deg;;  pellets were resonicated in Krebs-Ringers 
with 0.1% TX-100 for 30 s.  ALP is extracted in cold butanol 30-45 min 4&deg;, 
and supernatant dialyzed against 100 mM Tris pH 8 with 1 mM ZnCl<sub>2</sub> and 
MgCl<sub>2</sub>.  pALP activity is assayed with 4-methylumbelliferyl phosphate 
(according to Grozdea et al, 1984) and with p-nitrophenyl phosphate (according 
to Gainer and Stinton 1982); protein assay by Lowry.  Half of samples are kept 
on ice and other are treated in 56&deg; or 60&deg; water bath for 10, 20, 40, 
and 60 min.  Another part of enzyme is treated to 3.7 or 5 M urea for 5, 15, 25, 
40 and 60 min at 37&deg;.
<p>
All pregnancies showed significant elevations in pALP.  A significant 2.2% and 
5% increase in heat-stable and urea-resistant activity, respectively, was seen 
in 1st trimester.  This became 64.5% and 49.3% respectively in 3rd trimester. 
Specific activity showed increases as well: 200 nmol/hr/mg protein from 1st 
trimester compared to nonpregnant controls having 95 nmol/hr/mg protein (p &le; 
0.5).  Between trimesters howevever there is no significant difference in 
specific activity or in resistant neonatal ALP.
<p>
The 3.7 M urea incubation inactivation was studied.  By 15 min, about 35% of 
activity is lost, and by 40 min, 60% is lost.  After 1 hour, only 30% of 
activity is remaining, indicating a half-life of about 30 minutes under these 
conditions.  There were no differences between samples of mothers carrying Downs 
and normal fetuses.
<p>
The ratio of pALP activities resistant to urea (3.7 M at 40 and 60 min) and 
heat-stabile (56&deg; 20 min) from Downs and normal fetuses were plotted, with a 
range from 0.5 to 1.6 generally (Downs/normal).  13 Downs cases were studied, 
and normals matched for gestational age were done. No signficant differences for 
urea at 40 min were seen, but at 60 min a small different of 18% was seen.  The 
difference was not significant by a <i>t</i> test but was significant using 
Wilcoxon sign ranks test of ratios (<i>p</i> &lt; 0.05).  19 Downs with matched 
controls were test for heat stability  but no signficant difference was seen. 
Other experiments included use of p-nitrophenyl phosphate, inactivation at 
53&deg; and 60&deg;, addition of 0.01% BSA, or heat-treated male serum to 
neutrophil extracts to increase carrier proteins, 5 M urea, but in no case could 
significant differences be seen.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Oda Y, Huang K, Cross FR, Cowburn D, Chait BT
 </span>
 <span class="year">
(1999)
 </span>
 <span class="jname">
Proc. Natl. Acad. Sci. USA
 </span>
 <span class="jvol">
96,
 </span>
 <span class="pages">
6591-6596.
 </span>
<p class="reftitl">
Accurate Quantitation of Protein Expression and Site-Specific Phosphorylation
<p class="abstract">
<b>ABSTRACT</b>.
A mass spectrometry-based method is described
for simultaneous identification and quantitation of
individual proteins and for determining changes in the levels
of modifications at specific sites on individual proteins.
Accurate quantitation is achieved through the use of wholecell
stable isotope labeling. This approach was applied to the
detection of abundance differences of proteins present in
wild-type versus mutant cell populations and to the identification
of in vivo phosphorylation sites in the PAK-related yeast
Ste20 protein kinase that depend specifically on the G1 cyclin
Cln2. The present method is general and affords a quantitative
description of cellular differences at the level of protein
expression and modification, thus providing information that
is critical to the understanding of complex biological phenomena.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Keough T, Youngquist RS, Lacey MP
 </span>
 <span class="year">
(1999)
 </span>
 <span class="jname">
Proc. Natl. Acad. Sci USA
 </span>
 <span class="jvol">
96,
 </span>
 <span class="pages">
7131-7136.
 </span>
<p class="reftitl">
A Method for High-Sensitivity Peptide Sequencing Using Postsource Decay
Matrix-Assisted Laser Desorption Ionization Mass Spectrometry
<p class="abstract">
<b>ABSTRACT</b>.
A method has been developed for de novo
peptide sequencing using matrix-assisted laser desorption
ionization mass spectrometry. This method will facilitate
biological studies that require rapid determination of peptide
or protein sequences, e.g., determination of posttranslational
modifications, identification of active compounds isolated
from combinatorial peptide libraries, and the selective identification
of proteins as part of proteome studies. The method
involves fast, one-step addition of a sulfonic acid group to the
N terminus of tryptic peptides followed by acquisition of
postsource decay (PSD) fragment ion spectra. The derivatives
are designed to promote efficient charge site-initiated fragmentation
of the backbone amide bonds and to selectively
enhance the detection of a single fragment ion series that
contains the C terminus of the molecule (y-ions). The overall
method has been applied to pmol quantities of peptides. The
resulting PSD fragment ion spectra often exhibit uninterrupted
sequences of 20 or more amino acid residues. However,
fragmentation efficiency decreases considerably at amide
bonds on the C-terminal side of Pro. The spectra are simple
enough that de novo sequence tagging is routine. The technique
has been successfully applied to peptide mixtures, to
high-mass peptides (up to 3,600 Da) and to the unambiguous
identification of proteins isolated from two-dimensional gel
electrophoresis. The PSD spectra of these derivatized peptides
often allow far more selective protein sequence database
searches than those obtained from the spectra of native peptides.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Osinga R, Tramper J, Wijffels RH
 </span>
 <span class="year">
(1999)
 </span>
 <span class="jname">
Mar. Biotechnol.
 </span>
 <span class="jvol">
1,
 </span>
 <span class="pages">
509-532.
 </span>
<p class="reftitl">
Cultivation of Marine Sponges
<p class="Para1">
<b>REVIEW</b>.  The porifera are primitive, multicellular, sessile, 
filter-feeding animals that ingest organic particles between 0.1 to 50 &micro;m, 
which include phytoplankton, heterotrophic bacteria, heterotrophic eukaryotes, 
and detritus.  Their ability to inhabit nutrient-poor environments is likely due 
to their capacity to filter large amounts of water.  About 10,000 species, 
mostly marine, are described.
<p>
Sponge anatomy shows that water is usually brought in through smaller pores, the 
<i>ostia</i>, on the surface of the outer body and ejected through a larger 
opening, the <i>osculum</i>, often at the top of the sponge in the filtration 
scheme.  Special epithelial cells called <i>pinacocytes</i> line the outer 
surface and internal opening, channels and chambers. <i>Choanocytes</i> are 
flagellated cells that form the walls of the chambers and rhythmically beat to 
produce water currents directing flow toward the cavities and channels. Most of 
the mass of the sponge is a gelatinous matrix with free-floating 
undifferentiated cells called the <i>mesohyl</i>; this matrix also contains a 
skeleton of spicules, needle-like structures composed of silicon or calcium 
carbonate, and spongin, a collagen-like fiber.  <i>Archaeocytes</i> are ameboid 
cells in the mesohyl moving through the matrix, phagocytizing larger particles. 
Choanocytes feature a collar around the flagellum that is composed of microvilli 
which trap very tiny particles (bacteria, prochlorophytes).  Choanocytes digest 
some particles and transfer others to archaeocytes.
<p>
Secondary metabolites of sponges include cytotoxins, antifouling agents, 
anitbiotics, anti-inflammatory, and anti-viral agents.  Some metabolites may not 
be produced by sponges but by endosymbiotic microorganisms. Halichondrin B is an 
anticancer drug isolated from <i>Halichondria okadai</i> and 
<i>Lissodendorix</i> sp. which successfully passed a preclinical test but is 
produced in such little amounts that the supply problem has not been overcome.
<p>
Sponge aquaculture has focused on natural bath sponge production. 13,000 tons 
were produced in the peak year 1938.  Moore described in 1910 that cultivation 
could be done by cutting into pieces and attaching to aluminum wire embedded in 
concrete disks which were placed in the sea. Other fix sponge cuttings on nylon 
ropes;  razor-sharp knives should be used in the cut and one side must contain 
skin cells (pinacoderm), and smaller cuttings should have proportionately higher 
pinacoderm.  Moreover sponge cuttings must be done completely submerged, since 
introducing air into channels blocks water flow.  Sponge farming should be 
removed from fresh water streams feeding the sea, as salinity changes are 
critical. Sponges that do not grow a great deal of biomass (grow the least well) 
produce higher concentrations of metabolites.
<p>
Short-term cultivation has been used mainly to study sponge biology than to 
harvest commercial mass.  Studies by Wilson a century ago showed that cells of 
<i>Microcionia prolifera</i> could be dissociated using a thin mesh, which can 
reaggregate and differentiate into small sponges with functioning aquiferous 
system in a week.  Cell dissociation can be achieved by mechanical or chemical 
means;  cell adhesion is significantly dependent upon Ca<sup>2+</sup> and 
Mg<sup>2+</sup>, which can be removed from seawater.  Cell types within the 
sponge body can also be separated using standard cell type (usually density 
gradients) techniques, particularly to obtain stem cells.
<p>
The <b>sandwich method</b> refers to a technique from Ankel and Eigenbrodt in 
which young sponges obtained from gemmules (egg-like reduction bodies, usually a 
characteristic of freshwater sponges) a grown between glass slides or plates and 
a coverslip.  Multicellular single cell layer forms (<q>two-dimensional</q>) are 
used for direct microscopic examination. Instead of gemmules, Langenbruch took 
approx. 1 mm<sup>3</sup> explants of the marine <i>H. panica</i> into the 
sandwich but could still make morphogenetic studies (on a dissecting scope?). 
This method has also been used to study the effects of light on tropical sponges 
(<i>Desmapsamma anchorata</i> and <i>Aplysina fistularis insularis</i>).
<p>
Long-term in vitro cultivation methods have been reported by public and private 
aquaria.  The consensus is that unfiltered seawater is the best.  Some factors 
have been defined.  <b>Food</b> is a limited resource in most closed systems, 
since they are usually trying to filter and keep water clean, and the natural 
environment of sponges is rarely that way;  suitable organic particles must be 
supplied in any farming operation.
<p>
<i>Desmospongia</i> is the largest of classes in the phylum Porifera and their 
skeletons and that of the glass sponges (class Hexactinellida) are composed of 
many siliceous spicules.  <i>Sclerocytes</i> secrete the structures which are 
formed from <b>silicon</b> they obtain via silicic acid and crystallizing it 
around a protein filament.  Spicular silica can compose up to 62% of the dry 
weight of spp. in the order Haplosclerida, and 75% of of spp. in order 
Petrosiida.  The calcareous sponges (class Calcarea) form spicules of 
CaCO<sub>3</sub>, usually present in seawater and not likely to be 
growth-limiting as silicic acid might.
<p>
<b>Oxygen</b> is another growth-limiting factor.  Sponge respiration rates vary 
between 0.2 and 25 &micro;mol O<sub>2</sub> per hour per cm<sup>3</sup> sponge 
volume.  Such rates are comparable to other invertebrates such as crustaceans 
and molluscs, cultivatable at high densities.  Certain sponges are sensitive to 
hypoxia.
<p>
Cyanobacteria are photosynthetic endosymbionts of tropical sponges and thus 
require light, similar to zooxanthellae in coral.  They usually serve as food 
sources for the sponges.  Indeed rather than supplementing culture systems 
directly with food, it might be preferable to provide light as an energy source.
<p>
Some factors inhibit growth when present, aside from those above that limit it 
when absent or insufficient.  Clay particles and silt can reduce the pumping 
rate of the aquiferous channels; <i>Spongia officinalis</i> had a reduced growth 
rate.  Air (bubbles) can also damage the flow system in sponges; air in the 
choanocyte chambers stops the pump, and sponges die after even only a brief 
exposure.  Some sponges in intertidal areas adapt to air presence however:  this 
may be due to chemicals on the surface that maintain wetness, or rapid closing 
of ostia.  Light may also be growth inhibitory in certain spp. possibly due to 
sensitivity to UV.
<p>
The average seawater salinity is 35 g per kilogram.  Species living in shallow 
or estuarine waters probably adapt to rapid changes, but marine species require 
a strict osmoregulation.  Tropical bath sponges are intolerant to sudden 
downshocks.  The wool sponge <i>Hippospongia lachne</i> could withstand salinity 
up to 46&permil; but below 26&permil; was lethal. As for temperature, sponges 
generally are not subjected to rapid changes in temperature except the seasonal 
variation;  generally they can tolerate sudden decreases rather than increases.
<p>
Seawater is usually alkaline (between pH 7.8 and 8.4) and sudden drops in pH 
would indicate changes in salinity (low) or hypoxia leading to metabolic 
acidosis.  The presence of ammonia in any form (NH<sub>3</sub> or 
NH<sub>4</sub><sup>+</sup>) can lead to growth inhibition;  LC<sub>50</sub> can 
be 60-100 &micro;M.  The ionic form is less toxic in that ammonia can permeate 
cell membranes and alter metabolism intracellularly. A role is also played by 
dissolved oxygen, which at lower concentrations enhances ammonia toxicity.  A 
study of the effects of ammonia in high density cultures would be worthwhile.
<p>
In a bioreactor/aquarium farm for sponges, both natural and artificial (from the 
structure of the system) toxins, contaminants and poisons may accumulate and 
kill or affect growth.  Usually, the natural toxins are used to stop predation 
or prevent fouling or settlement by competitive species;  there may be inter- 
and intraspecific interactions that affect desired growth in a closed system 
however.  Another problem is the death and decay of sponge species.  Some 
release poisons (acetylene) while bacteria can cause ammonia accumulation during 
decay.
<p>
Sponges are also susceptible to host-specific pathogens (disease) that are 
extremely contagious.  This is more so with high density populations. Generally 
higher temperatures appear to increase disease susceptibility. Many sponge 
die-offs recently have been in warmer waters, possibly a result of global 
warming.
<p>
Sponges have few predators.  This is largely because they have spiculated 
skeletons, and they have endosymbionts that produce toxins.  Nudibranches do eat 
sponges, and also use their toxins to prevent predation against themselves. 
Care must be taken not to introduce adult nudibranches, sea urchins, and 
parrotfish to cultures.
<p>
<b>In vitro culture design</b>.  Adult individuals are harvested from the 
natural population and explants (cuttings) are set rather than the parents as 
the cuttings grow faster than intacts.  Parents used should appear healthy and 
free of injury/lesions.  Sponge or explants that start to decay or change to an 
unhealthy appearance should be removed from the culture immediately.
<p>
Alternatively, and probably essential for some species, is to raise sponges from 
early developmental stages such as eggs/larvae/reduction bodies. Such sponges 
may survive better in the long-term and adapt to growth in vitro. Sponges 
usually produce reproductive elements as the water warms, although continual 
production of these forms may occur too.  Larval formation and release is 
triggered by temperature and light in some cases.  In other cases, reduction 
bodies such as gemmules are obtained by adverse environments, such as 
diminishing food or energy sources.  These can be harvested and used to trigger 
rapid growth under ideal conditions.
<p>
Transport of substrate energy to the sponge is controlled by both diffusion and 
the active pumping on the aquiferous system.  Effectiveness of transport is 
determined by a factor &eta;, which is defined as the conversion rate of 
substrate divided by the rate if there is no limitation in transport of 
substrate.  &eta; = 1 if there is no limitation regarding transport.  &eta; will 
be lower if transport is only by diffusion.
<p>
For some sponges, a diet may be restricted to the types of bacteria or other 
organisms.  In some cases, sponges can obtain small molecules (amino acids, 
polyunsaturated fatty acids) directly from the seawater.
<p>
<b>Sponge cell culture</b>.  No continuous cell lines have been developed
to date.  A methodology has been described by which to approach development.
<ol>
<li>Cell dissociation of whole bodies or explants is done mechanically,
chemically, or enzymatically, or a combination.
<li>Cell types are selected by unit gravity or density gradient centrifugation.
<li>Antibiotics are used to produce axenic cultures.
</ol>
<p>
Cultures grown in calcium- and magnesium-free artificial seawater, pH 7 
phosphate, and 5% BSA are ready.  Mitogenic agents are needed to stimulate 
growth as well, and sometimes antibiotics are useless.  One problem is that 
cells, when dissociated, rapidly lose their telomerase activity and thus their 
totipotency.  Cell-cell contact was essential to maintain elevated telomerase.

<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Miranker AD
 </span>
 <span class="year">
(1999)
 </span>
 <span class="jname">
Proc. Natl. Acad. Sci USA
 </span>
 <span class="jvol">
97,
 </span>
 <span class="pages">
14025-14027.
 </span>
<p class="reftitl">
Mass Spectrometry of Proteins of Known Mass
<p class="abstract">
<b>COMMENTARY</b>.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Wilkins MR, Gasteiger E, Gooley AA, Herbert BR, Molloy MP, Binz P-A,
Ou K, Sanchez J-C, Bairoch A, Williams KL, Hochstrasser DF
 </span>
 <span class="year">
(1999)
 </span>
 <span class="jname">
J. Mol. Biol.
 </span>
 <span class="jvol">
289,
 </span>
 <span class="pages">
645-657.
 </span>
<p class="reftitl">
High-Throughput Mass Spectrometric Discovery of Protein Post-Translational
Modifications
<p class="Para1">
Software tools such as MOWSE, MS-Fit, and PepIdent are available through the 
internet.  Some of them can search a few common post-translational or 
artifactual modifications (phosphorylation in one, Met oxidation in the other) 
to account for unknown masses.  To do this on a large scale rapidly to predict 
sequences that are modified is rather difficult. FindMod is added to these 
tools.  The user generates the peptide mass fingerprint as usual to identify the 
protein.  FindMod then searches for post-translation mods by comparing 
experimental peptide masses that did not match with the protein against those 
calculated from the assigned sequence, and looking for mass differences that 
might be due to modification.
<p>
For elucidating rules that FindMod might follow, 5153 incidences of
modifications were analyzed.  Modifications were all those known in the
literature.  <i>N</i>-linked glycosylation was not considered since
the mass difference is difficult to predict.  The program can predict
22 of the more common modifications listed in the table below.
<table id="postranstable" style="text-align:center;margin:0 -20%;">
<col style="width:10em;"><col span="2" style="width:6em;">
<col style="width:10em;">
<col>
<col style="width:10em;">
<col style="width:25em;">
<tr><th rowspan="2">Modification
    <th colspan="2">&Delta; mass
	 <th rowspan="2">Kingdom
 <th rowspan="2">Amino acids <th rowspan="2">Position in protein
 <th rowspan="2">Rule
<tr><th>Monoisotopic <th>Average
<tr><td>Acetylation <td>42.0106 <td>42.0373 <td>All
  <td>K<p>All but F, H, K, N, R, W, Y
 <td>Anywhere<td>&mdash;
<tr><td>Amidation <td>&minus;0.9840 <td>&minus;0.9847
 <td>Eukaryotes <td>All <td> C terminus after terminal cleavage event
 <td>G must have been C-terminal to site of cleavage
<tr><td>Biotin <td>226.0776<td>226.2934 <td>All <td>K <td>Anywhere
  <td>PROSITE pattern PDOC00167<sup>a</sup>
<tr><td>*Carbamylation <td>43.00581 <td>43.02504 <td>All <td>All<p>K
 <td>N terminus <p>Anywhere <td>&mdash;<p>&mdash;
<tr><td>C-Mannosylation <td>162.0538 <td>162.1424 <td>Eukaryotes <td>W
  <td>Anywhere <td>&mdash;
<tr><td>*Cys-propionamide <td>71.0371 <td>71.0788 <td>All <td>C
  <td>Anywhere <td>&mdash;
<tr><td>Deamidation <td>0.9840 <td>0.9847 <td>All <td>N<p>Q
  <td>Anywhere<p>Anywhere <td>Must be followed by a G<p>&mdash;
<tr><td>Flavin-adenine dinucleotide <td>783.1415 <td>783.542
 <td>All <td>C, H <td>Anywhere  <td>&mdash;
<tr><td>Farnesylation <td>204.1878 <td>204.3556 <td>Eukaryotes, Viruses
 <td>C <td>Anywhere <td>&mdash;
<tr><td>Formylation <td>27.9949  <td>28.0104 <td>All <td>All <td>N terminus
 <td>&mdash;
<tr><td>*Glycation (glucosylation) <td>162.0528 <td>162.1424
 <td>All <td>All <td>N terminus  <td>&mdash;
<tr><td>Geranyl-geranylation <td>272.2504 <td>272.4741
  <td>Eukaryotes, Viruses <td>C <td>Anywhere <td>&mdash;
<tr><td>Gammacarboxyglutamic acid <td>43.9898 <td>44.0098
 <td>Eukaryotes <td>E <td>Anywhere <td>&mdash;
<tr><td>Hydroxylation <td>15.9949 <td>15.9994 <td>Eukaryotes
 <td>D, N, K, P <td>Anywhere <td>&mdash;
<tr><td>Lipoyl <td>188.033 <td>188.3027 <td>All <td>K <td>Anywhere
 <td>PROSITE pattern PDC00168<sup>c</sup>
<tr><td>Methylation <td>14.0157 <td>15.0269
 <td>All<p>Prokaryotes<p>Eukaryotes<p>Prokaryotes
 <td>C, D, E, H, K, N, R, Q<p>A N<p>A, P<p>F, I, L, M, Y
 <td>Anywhere<p>N terminus<p>N terminus
   <p>N terminus after cleavage of signal peptide
 <td>&mdash;<p>&mdash;<p>&mdash;<p>PROSITE pattern PS00409<sup>d</sup>
<tr><td>Myristoylation <td>210.1984<td>210.3598
  <td>Eukaryotes<p>Eukaryotes, Viruses
 <td>K <p>G <td>Anywhere<p>N terminus <td>&mdash;<p>&mdash;
<tr><td>N-acyl diglyceride (tripalmitate)
 <td>788.7258 <td>789.3202 <td>Prokaryotes, Archaebacteria, Phage
 <td>C <td>N terminus after cleavage of signal peptide
 <td>PROSITE pattern PDOC00013<sup>b</sup>
<tr><td>O-GlcNac <td>203.0794 <td> 203.1950
 <td>Eukaryotes <td>S, T <td>Anywhere <td>&mdash;
<tr><td>Palmitoylation <td>238.2297<td>238.4136
 <td>Eukaryotes <td>C, S, T <td>Anywhere <td>&mdash;
<tr><td>Phosphorylation <td>79.9663 <td>79.9799
 <td>Prokaryotes<p>Eukaryotes, Viruses
 <td>C, D, H, S, T <p>H, D, S, T, Y
 <td>Anywhere<p>Anywhere <td>&mdash;<p>&mdash;
<tr><td>Pyridoxal phosphate <td>229.014 <td>229.129
 <td>All <td>K <td> Anywhere <td>D, P not allowed &minus;1, E, P<p>not allowed +1
<tr><td>Phosphopantetheine <td>339.078 <td>339.3234
 <td>All <td>S <td>Anywhere <td>PROSITE pattern PDC0012<sup>e</sup>
<tr><td>Pyrrolidone carboxylic acid <td>&minus;17.0266 <td>&minus;17.0306
 <td>All <td>Q <td>N terminus <td>&mdash;
<tr><td>Sulfation <td>79.9568 <td>80.0642 <td>Eukaryotes <td>Y
 <td>Anywhere <td>PROSITE pattern PDOC0003<sup>f</sup>
<tr><td>*Sulfoxide formation <td>15.9949 <td>15.9994
 <td>All <td>M <td>Anywhere <td>&mdash;
<tr><td colspan="7">
Modications preceded by an asterisk (*) are generally artifactual and
can also be searched for by FindMod.<p>
<sup>a</sup> The regular expression
[GN]-[DEQTR]-x-[LIVMFY]-x(2)-[LIVM]-x-[AIV]-M-K-[LMA]-x(3)-[LIVM]-x-[SAV]
where K is the biotin attachment site.<p>
<sup>b</sup> The regular expression
{DERK}(6)-[LIVMFWSTAG](2)-[LIVMFYSTAGCQ]-[AGS]-C where C is the
lipid attachment site. Additional rules are that:
(1) The cysteine must be between positions 15 and 35 of
the sequence in consideration
(2) There must be at least one Lys or one Arg in the rst seven
positions of the sequence.<p>
<sup>c</sup> The regular expression
[GN]-x(2)-[LIVF]-x(5)-[LIVFC]-x(2)-[LIVFA]-x(3)-K-[STAIV]-[STAVQDN]-x(2)-[LIVMFS]-x(5)-[GCN]-x-[LIVMFY]
where K is the lipoyl-binding site.<p>
<sup>d</sup> The regular expression
[KRHEQSTAG]-G-[FYLIVM]-[ST]-[LT]-[LIVP]-E-[LIVMFWSTAG].<p>
<sup>e</sup> The regular expression
[DEQGSTALMKRH]-[LIVMFYSTAC]-[GNQ]-[LIVMFYAG]-[DNEKHS]-S-[LIVMST]-{PCFY}-[STAGCPQLIVMF]-[LIVMATN]-[DENQGTAKRHLM]-[LIVMWSTA]-[LIVGSTACR]-x(2)-[LIVMFA]
where S is the pantetheine attachment site.<p>
<sup>f</sup> See http://www.expasy.ch/sprot/prosite.html.
</table>
<p>
All modifications cause a mass change.  Rules are simple.  As an example 
formylation can happen to the N-terminal position in any species. Distinctions 
are made for some modifications, such as methylation, depending on cell classes 
(prokaryotes, eukaryotes).  Other artifacts account for variable modifications 
(carbamylation, sulfoxide, cys-propionamide formation) occurring during sample 
preparation. The following four cases illustrate the use of FindMod in the 
discovery of modifications of peptides.  PMF was done using a MALDI-ToF MS and 
fragmentation/sequencing spectra were obtained by MALDI-ToF MS/MS PSD.
<p>
<b>Case 1:  Lysine dimethylation in <i>E. coli</i> Elongation Factor Tu</b>.
The protein EFTu (P02990).  One pepdtide of mass 1631.80 was not
a theoretical match and after processing through FindMod, was reported
to be AFDQIDNAPEEKAR with a &Delta;<i>m</i> of +28.03 Da.  This modification
was reported to occur in one of three ways: (1) dimethylation of an
amino acid (possibly K),
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Harvey DJ
 </span>
 <span class="year">
(1999)
 </span>
 <span class="jname">
Mass Spectrom. Rev.
 </span>
 <span class="jvol">
18,
 </span>
 <span class="pages">
349-451.
 </span>
<p class="reftitl">
Matrix-Assisted Laser Desorption/Ionization Mass Spectrometry of Carbohydrates
<p class="Para1">
<b>REVIEW</b>.
Carbohydrates (CHOs) are the most abundant and diverse biomolecules.
They form branched structures unlike the linear polymers of proteins
and nucleic acids: for a hexasaccharide, there are more than 10<sup>12</sup>
possible isomers.  Only a very few isomers are encountered since their
biosynthesis is dependent on a limited number of glycosyltransferases.
<p>
Most monosaccharides have the formula C<sub>n</sub>H<sub>2n</sub>O<sub>n</sub>, 
where <i>n</i> = 3 (trioses) to 9 (nonoses), and <i>n</i> = 6 are the most 
abundant.  Reducing sugars contain <i>n</i> &minus; 1 hydroxy (OH) groups and 
either an aldehyde (CHO) or keto (-CHOH-C(=O)-CH<sub>2</sub>OH) group. 
Deoxysugars and those oxidized to form carboxylic (uronic) acid groups also 
occur.  Some are reduced to alcohols, or there is an amination or acylamination. 
Some hydroxyls get methylated.
<p>
Sugars form rings typically.  Ribose (ribofuranose) has a five-membered ring 
with fifth carbon out of the ring.  Arabinose (arabinopyranose) is a 6-membered 
ring of a 5 carbon sugar.  Xylose is an arabinose epimer at carbon 4; and xylose 
carbons 2-4 have the same stereo configuration as glucose.  Mannose is a carbon 
2 epimer of glucose, and galactose an carbon 4 epimer.  Fucose is a 6-carbon 
6-membered right with the 6th carbon fully methylated.  N-acetylglucosamine is 
essentially the amination of carbon 2 of glucose, followed by acylation with 
acetate.  Glucuronic acid is the fully oxidized 6th carbon carboxylate form of 
glucose. Rhamnose is an epimer of fucose at carbon 4.  N-acetylneuraminic acid 
is also called <b>sialic acid</b> in which a 6-membered ring joins carbon 2 to 6 
through the hydroxyl of carbon 6, and carbon 1 is in carboxyl form, carbon 4 is 
acylaminated, and carbon 6 is joined to a 3-carbon glycerol.  Oligosaccharides 
are composed of 2-10 monosaccharides, and polysaccharides are considered to have 
&gt; 10 units.
<p>
Proteins become glycosylated either as N- or O-linked glycans.   N-linkage
is achieved by glycosyltransferases that recognize the NX(S/T) (Asn-X-Ser/Thr)
motif, where X is any amino acid but Pro.  O-linked connect to Ser and Thr
and also to glycosylphosphatidylinositol lipid anchors.  N-linked glycans
form a biantennary (sometimes more)  trimannosyl chitobiose core.
<p>
[section here on glycolipids]
<p>
<b>MALDI MS</b>.  Techniques for ionization of sugars are as numerous as the 
sugars themselves.  Electron impact ionizaiton (EI) works only with smaller 
molecules which usually must be derivatized.  MALDI can operate without sample 
conversion/derivatization.  MALDI of sugars was investigated in the late 1980s 
mostly by Karas and Hillenkamp, while looking at matrices for ionizing proteins. 
Stachyose was one of the first looked at;  this phytochemical tetrasaccharide 
will ionize without matrix, but when nicotinic acid or tryptophan were added for 
the Nd-YAG (266 nm) laser, signal improved;  the major MALDI ion was a sodium 
form.  3-nitrobenzyl alcohol and 3-nitrophenyl octyl ether are liquid matrices 
and they produced instead a radical ion or [M-17]<sup>+</sup>, respectively. 
MALDI is 10-100 times more sensitive than FAB and <sup>252</sup>Cf plasma 
desorption MS.  It also worked for tryptic glycopeptides where PD failed. 
Ionization of MNa<sup>+</sup> ions is not as sensitive as MH<sup>+</sup>, where 
the former is CHOs and latter are peptides.  Generally high laser power is 
required and this reduces the number of spectra that can be obtained;  laser aim 
must be in continuous movement to keep passable spectra collected in series.
<p>
Lasers are typically nitrogen gas (337 nm).  Infrared lasers such as Er-YAG 
(2.94 &micro;m, 80-90 ns pulse) and Er-YSGG (2.79 &micro;m, 80 ns) can be used 
for large glycoproteins, using glycerol or succinic acid matrices.  Much more 
sample is consumed (by two orders) than for UV lasers. The first commercial 
MALDI-TOF was made in 1990 but had poor resolution as a linear type (200-300). 
Resolution of 2000 FWHM in magnetic sector instruments were used in CHO 
analysis, requiring array type detectors since they were pulsed.  Eventually 
reflectron tubes were available, then delayed extraction or time-lag focusing 
ion sources were developed. Resolution of 10,000 or more with mass accuracies in 
low ppm range are available today.
<p>
Interesting is the use of magnetic sector MS with conventional point detector. 
The instrument is set to scan 0-2400 u over 1 min, and many spectra are recorded 
and averaged.  This requires use of a matrix that can maintain a signal for 
several minutes.  A liquid matrix of &alpha;-cyano-4-hydroxycinnamic acid 
(4-HCCA) and 3-aminoquinoline proved to be satisfactory.  Resolution of 8000 
FWHM and mass accuracy to 2 ppm were possible.  MS/MS of maltodecaose also could 
be made with He gas in collision.
<p>
In another approach, 6-10 laser pulses are fired across each mass peak and
the resulting spectra are smoothed for the peak profile.  A single scan
is sufficient to produce a high resolution (~20,000) spectrum.  Ion trap
and FT-ICR MS have been used successfully.
<p>
<b>Matrices</b>.  These are optimized not just for the laser but the class of 
CHO.  Substituted benzoic acids were used to look at free neutral CHO molecules. 
3-amino-4-hydroxbenzoic acid was the first matrix used, producing 
MNa<sup>+</sup> ions having good signals with as little as 1 pmol dextran 
hydrosylate and N-linked glycans from glycoproteins. 2,5-DHB was used 
(originally used for proteins) and is the most popular for CHO MALDI-MS; it also 
produces MNa<sup>+</sup> ions.  Inclusion of potassium and lithium also produces 
ions of this type.  Alkali ion affinity appears to be Cs &gt; K &gt; Na; &gt; Li 
&gt; H.  Cs will not ionize smaller CHOs as it will glycans.
<p>
2,5-DHB will crystallize from MeCN or EtOH/water as long needle-shaped crystals, 
orientied radially;  re-dissolution in absolute EtOH will produce a more even 
film, increasing sensitivity roughly 10-fold.  It has been understood that 
isomers of 2,5-DHB are not effective unless there is an <i>ortho</i>-hydroxy 
group.  The reason is that an intramolecular proton transfer occurs between the 
phenol and carboxyl members, as shown by all <i>ortho</i> isomers being 
effective matrices and the rest not.  2,5-DHB was the most effective, although 
it is thought that photochemical decomposition occurs in which UV light induces 
elimination of CO<sub>2</sub>.  Additional oxidation produces 
<i>p</i>-benzoquinone, relatively stable.  Working from this assumption it has 
been possible to discover other matrices as 1,4-dihydroxy-2-naphthoic acid as it 
produces stable species of this type.  2,5-DHB matrix mixes also improve 
sensitivity and resolution.  Mixes disorder crystal lattice and therefore 
provide a <q>softer</q> desorption.  10% 2-hydroxy-5-methoxybenzoic acid with 
2,5-DHB produces something called <q>super 2,5-DHB</q>.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Pfeifer T, Drewello M, Schierhorn A
 </span>
 <span class="year">
(1999)
 </span>
 <span class="jname">
J. Mass Spectrom.
 </span>
 <span class="jvol">
34,
 </span>
 <span class="pages">
644-650.
 </span>
<p class="reftitl">
Using a Matrix-Assisted Laser Desorption/Ionization Time-of-Flight
Mass Spectrometer for Combined In-Source Decay/Post-Source Decay Experiments
<p class="abstract">
<b>ABSTRACT</b>.
Mass spectrometric experiments with fragment ions have not yet been possible with a matrix-assisted laser
desorption/ionization time-of-flight (MALDI-TOF) mass spectrometer because intense signals of fragment ions
were rarely observed during continuous extraction and the mass resolution of in-source formed fragment ions
has been low. This paper describes a combination of MALDI in-source decay and post-source decay experiments
on a MALDI-TOF mass spectrometer equipped with delayed extraction. Fragment ions initially formed in the
ion source were selected by the precursor ion gate and investigated by post-source decay. The in-source formed
fragment ions were sufficiently excited to undergo further metastable decay. The new method was applied
to linear peptides, the cyclic peptide gramicidin S and a pentasaccharide, leading to unambiguous structural
information.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Link AJ, Eng J, Schieltz DM, Carmack E, Mize GJ, Morris DR, Garvik BM, Yates JR 3rd
 </span>
 <span class="year">
(1999)
 </span>
 <span class="jname">
Nat. Biotechnol.
 </span>
 <span class="jvol">
17,
 </span>
 <span class="pages">
676-682.
 </span>
<p class="reftitl">
Direct Analysis of Protein Complexes Using Mass Spectrometry
<p class="Para1">
Many important complexes of proteins exist in nature.  RNA polymerase II
transcription complex contains at least 50 different proteins, and eukaryotic
ribosome consists of 78-80 unique proteins.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Vogel KG, Meyers AB
 </span>
 <span class="year">
(1999)
 </span>
 <span class="jname">
Clin. Ortho. Rel. Res.
 </span>
 <span class="jvol">
367S,
 </span>
 <span class="pages">
S344-S355.
 </span>
<p class="reftitl">
Proteins in the Tensile Region of Adult Bovine Deep Flexor Tendon
<p class="Para1">
Type I collagen is the major protein of tendon.  This study is undertaken to 
find ECM molecules in tendon also known to be present in cartilage, using gel 
electrophoresis and Westerns.  Deep flexor tendon was obtained from font feet of 
1-2 year old cows, and the tensile region (proximal to point of tendon 
bifurcation) was cut into small pieces with razor blade, powdered in liquid 
nitrogen with stainless steel ball or grinding mill. Storage was at 
&minus;20&deg;.  200 mg wet weight frozen tissue powder is put in screw cap tube 
with 4 ml extraction buffer and rotated vigorously end over end 24 h in cold 
room or at RT.  This is centrifuged 30 min at 10 krpm (Beckman JA-20) at 5&deg;. 
Supernatant is recovered and pellet extracted in series.  Extraction is in 
series as follows:  (1) 24 h with 1&deg; PBS (50 mM pH 7.4 sodium phosphate + 
150 mM NaCl) containing 5 mM benzamidine HCl, 5 mM N-ethylmaleimide, 2 mM EDTA, 
1 mM PMSF for protease inhibition; (2) 4 M guanidine HCl in 50 mM sodium acetate 
pH 6.0 repeated 3 times at 1&deg;; (3) 4 M GuanHCl + acetate but at 18&deg; and 
repeated 3 times; and (4) Guan buffer containing 10 mM DTT at 18&deg;, also 
repeated 3 times.  Protein assays are by BCA (Pierce) checking for Guan 
interference.
<p>
The use of DTT in the later extraction helps to remove proteins that might be 
attached by disulfide bonds.  A pellet that is quite gel-like (<q>gooey</q>) 
remains after all extractions; this is composed mostly of cross-linked type I 
collagen.  11% of the total protein is removed in the first 24 h PBS extraction. 
68% of the protein is extracted in 4 M guan either at cold or room temp, and an 
additional 21% is solubilized in the guan + DTT extraction.  Experiments with 
N-ethylmaleimide demonstrated that denaturation of protein by 4 M guan HCl was 
not leading to disulfide bond formation of proteins with other parts of the 
matrix.
<p>
SDS-PAGE showed that most protein extracted in PBS had masses &lt; 70 kDa.
Higher MW proteins were in the first 4 M guan HCl extraction, but these
were observed only in samples that had been reduced.  Rinsing in 4 M guan
HCl extracts Type I collagen &alpha; chains.
<p>
Western analysis was then done.  Proteins in the supernatant samples were 
analyzed before and after digestion with chondroitinase ABC (this removes 
chondroitin sulfate and dermatan sulfate glycosaminoglycan chains) and with and 
without reduction by thiols.  Some decorin was solubilized in PBS whereas all of 
it was removed in the 4 M guanidine.  After chondroitinase ABC digestion, core 
protein of the proteoglycan was solubilized. A 45 kDa band is found in 
undigested core protein lacking a glycosaminoglycan chain.  Fibromodulin is 
extracted in both PBS and guanidine, seen as a broad immunoreactivity band at 60 
kDa; a higher MW band reactive to anti-fibromodulin was not altered under 
reducing conditions or by digestion with chondroitinase ABC and could be 
fibromodulin carrying variable amounts of keratan sulfate or other type of 
glycosylation.  Aggregcan was seen on the immunoblot only with chondroitinase 
ABC-digested guanidine extracts: intact aggregcan monomer would not have exited 
from the stacking gel and this protein does not carry keratan sulfate chains. 
Type IV collagen chains were found in the guanidine extract
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Chaurand P, Leutzenkirchen F, Spengler B
 </span>
 <span class="year">
(1999)
 </span>
 <span class="jname">
J. Am Soc. Mass Spectrom.
 </span>
 <span class="jvol">
10,
 </span>
 <span class="pages">
91-103.
 </span>
<p class="reftitl">
Peptide and Protein Identification by Matrix-Assisted Laser Desorption
Ionization (MALDI) and MALDI-Post-Source Decay Time-of-Flight Mass
Spectrometry
<p class="abstract">
<b>ABSTRACT</b>.
The potential of matrix-assisted laser desorption ionization (MALDI)
and MALDI-post-source decay (PSD) time-of-flight mass spectrometry
for the characterization of peptides and proteins
is discussed. Recent instrumental developments provide for levels of
sensitivity and accuracy that make these techniques major analytical
tools for proteome analysis. New software
developments employing protein database searches have greatly enhanced
the fields of application of MALDI-PSD. Peptides and proteins can
be easily identified even if only a partial
sequence information is determined. Derivatization procedures have
been optimized for MALDI-PSD to increase the structural information
and to obtain a complete peptide sequence even in critical cases.
They are fast, simple and can be performed on target. MALDI-PSD is also
a very powerful tool to characterize or elucidate post-translational
or chemically induced modifications. In association with database
searches, proteins issued from electrophoretic gels
can be identified after specific enzymatic cleavages and peptide mapping.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Gygi SP, Rist B, Gerber SA, Turecek F, Gelb MH, Aebersold R
 </span>
 <span class="year">
(1999)
 </span>
 <span class="jname">
Nat. Biotechnol.
 </span>
 <span class="jvol">
17,
 </span>
 <span class="pages">
994-999.
 </span>
<p class="reftitl">
Quantitative Analysis of Complex Protein Mixtures Using Isotope-Coded
Affinity Tags
<p class="abstract">
<b>ABSTRACT</b>.
We describe an approach for the accurate quantification and
concurrent sequence identification of the individual proteins
within complex mixtures. The method is based on a class of new
chemical reagents termed isotope-coded affinity tags (ICATs)
and tandem mass spectrometry. Using this strategy, we compared
protein expression in the yeast Saccharomyces cerevisiae, using
either ethanol or galactose as a carbon source. The measured
differences in protein expression correlated with known yeast
metabolic function under glucose-repressed conditions. The method
is redundant if multiple cysteinyl residues are present, and the
relative quantification is highly accurate because it is based
on stable isotope dilution techniques. The ICAT approach should
provide a widely applicable means to compare quantitatively
global protein expression in cells and tissues.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Easterling ML, Mize TH, Amster IJ
 </span>
 <span class="year">
(1999)
 </span>
 <span class="jname">
Anal. Chem.
 </span>
 <span class="jvol">
71,
 </span>
 <span class="pages">
624-632.
 </span>
<p class="reftitl">
Routine Parts-Per-Million Mass Accuracy for High-Mass Ions:  Space-Charge
Effects in MALDI FT-ICR
<p class="Para1">
The elemental composition of low MW samples can often be determined by
an accurate mass measurement.  The count of peptides needed to identify
a protein is inversely proportional to mass accuracy of data.  FT-ICR
has a very high precision but there is no guarantee of accuracy.  ICR
signals arise from measurement of image charge created by coherent motion
of ions between a set of opposed electrodes in a strong magnetic field.
Ions in a magnetic field show a frequency of motion that is
<i>&omega; = zB/m</i> where &omega; is cyclotron frequency and is
proportional to applied magnetic field <i>B</i> and inversely proportional
to <i>m/z</i>.  In order to get a narrow spectral line width, a static
electric trapping potential is applied to confine ions within the magnetic
field.  Electric field influences ion motion and observed cyclotron frequency.
Perturbation to trapping electric field are caused by collective electric
potential of ions within trapping volume.  Effect is called <b>space
charge</b> and affects cyclotron frequency measurement.  In describing
the omegatron, the first ICR MS, this space charge effect was addressed.
<p>
The scanning ICR works by ejecting ions one mass at a time from the trapped
ion cloud.  In FT-ICR ions are excited into coherent cyclotron motion
with substantial radius of gyration then detected simultaneously.
To account for perturbations, an observed cyclotron frequency included
new factors:
<table class="eqnserif">
<tr><td rowspan="2">&omega;<sub>obs</sub> =
 <td style="border-bottom:1px solid black;"><i>zB</i>
 <td rowspan="2">&minus;
 <td style="border-bottom:1px solid black;">2<i>aV</i>
 <td rowspan="2">&minus;
 <td style="border-bottom:1px solid black;"><i>z&rho;G<sub>i</sub></i>
<tr><td><i>m</i> <td><i>a<sup>2</sup>B</i> <td><i>&epsilon;</i><sub>0</sub><i>B</i>
</table>
<p>
The first term is the unperturbed cyclotron frequency, the second term describes
the magnetron frequency for an ion in perfectly quadrupolar static trapping
field.  The last term is the space-charge component of the mass shift,
where <i>z</i> is elementary charge magnitude, <i>&rho;</i> is ion density,
<i>G<sub>i</sub></i>
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Shen T-L, Huang Z-H, Laivenieks M, Zeikus JG, Gage DA, Allison J
 </span>
 <span class="year">
(1999)
 </span>
 <span class="jname">
J. Mass Spectrom.
 </span>
 <span class="jvol">
34,
 </span>
 <span class="pages">
1154-1165.
 </span>
<p class="reftitl">
Evaluation of Charge Derivatization of A Proteolytic Protein Digest for
Improved Mass Spectrometric Analysis:  <i>de Novo</i> Sequencing By
Matrix-Assisted Laser Desorption/Ionization Post-Source Decay Mass Spectrometry
<p class="abstract">
<b>ABSTRACT</b>.
A simple mass spectrometric method to sequence a recombinant phosphoenolpyruvate carboxykinase of known
structure and a novel variant of unknown structure isolated from Anaerobiospirillum succiniciproducens and
Actinobacillus succinogenes 130Z, respectively, was evaluated. The proteolytic digests of the proteins were each
chemically derivatized at the N-terminus by addition of a tris(trimethoxyphenyl)phosphoniumacetyl (TMPPYAc)
group to produce peptides with a fixed positive charge. The derivatized digests were then partially separated
by reversed-phase high-performance liquid chromatography. The fractions collected were subjected to matrixassisted
laser desorption/ionization post-source decay (MALDI/PSD) mass spectrometric analysis. The resulting
spectra are sufficiently simple to allow the sequence to be read directly without extensive interpretation. This
is in contrast to spectra of underivatized peptides obtained by MALDI/PSD or conventional tandem mass
spectrometry, where full sequence interpretation can be challenging. Aided with a set of very simple established
rules, it was shown that the sequence of TMPPY-Ac derivatives can be derived strictly from predictable fragment
ion series. In most cases, this is sufficient to determine extensive, unambiguous, peptide sequences de novo. The
partial sequence (35%) of the unknown phosphoenolpyruvate carboxykinase from Actinobacillus succinogenes
130Z was obtained entirely by the mass spectrometric method evaluated here, which provided the basis for
evaluating homology and for the design of oligonucleotide probes for cloning the corresponding gene.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Hoffmann R, Metzger S, Spengler B, Otvos L Jr
 </span>
 <span class="year">
(1999)
 </span>
 <span class="jname">
J. Mass Spectrom.
 </span>
 <span class="jvol">
34,
 </span>
 <span class="pages">
1195-1204.
 </span>
<p class="reftitl">
Sequencing of Peptides Phosphorylated on
Serines and Threonines by Post-source Decay
in Matrix-assisted Laser Desorption/Ionization
Time-of-flight Mass Spectrometry
<p class="abstract">
<b>ABSTRACT</b>.
In the era of complete genome sequences, biochemical and medical research will focus more on the dynamic
proteome of a cell. Regulation of proteins by post-translational modifications, which are not determined by
the gene sequence, are already intensively studied. One example is phosphorylation of serines and threonines,
probably the single most common cellular regulatory mechanism. In this paper we describe the sequencing of
mono- and bisphosphorylated peptides, including identification of the phosphorylation sites, by post-source
decay (PSD) in matrix-assisted laser desorption/ionization time-of-flight mass spectrometry. In addition to
dephosphorylation of the parent ions, we studied the influence of the phosphate group on the fragmentation
of peptides. Generally, peptides phosphorylated on serine and threonine residues displayed no difference in
their fragmentation patterns. The intensities of the resulting fragment ion signals depend only on the peptide
sequence and not on either the phosphorylated amino acid or its position in the peptide chain. Phosphorylation
increased the bond cleavage C-terminal to the phosphorylation site more than 10-fold, resulting in abundant
signals, which typically dominated the PSD spectra. The produced C-terminally phosphorylated b-type fragment
ions showed characteristic dephosphorylated fragment ions bn  H3PO4 (98 Da) and bn  HPO3 (80 Da) of
higher abundances than the phosphorylated fragment ion. As a second layer to identify the phosphorylation site,
all internally phosphorylated fragment ions were accompanied by minor, but always detectable, signals of the
dephosphorylated fragment ions. Interpretation of PSD spectra of phosphopeptides was not more complicated
than for unphosphorylated peptides, despite the increased number of obtained fragment ion signals.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Cotter RJ, Fancher C, Cornish TJ
 </span>
 <span class="year">
(1999)
 </span>
 <span class="jname">
J. Mass Spectrom.
 </span>
 <span class="jvol">
34,
 </span>
 <span class="pages">
1368-1372.
 </span>
<p class="reftitl">
Miniaturized Time-of-flight Mass Spectrometer for Peptide and Oligonucleotide
Analysis
<p class="Para1">
As part of the process of miniaturizing mass spectrometers in which they would 
have multiple applications, particularly in the field, we recently introduced an 
endcap reflectron to provide nearly quadratic reflectron potential to obtain 
high mass resolution:  these provide kinetic energy focusing to infinite order 
(flight times are independent of kinetic energy). The endcap ToF MS assembly is 
composed of a piston-like cylinder with an adjustable drift region and 
reflectron detph.  A rod-like probe assembly is inserted on one end and 
concentrically surrounding is a detector anode (so the source and detector are 
within the same region.  On the other end is a reflectron assembly in which the 
reflectron electron mounted on an adjustable plug whose depth is controlled by 
an adjustment rod, and the plug inserts into a grounded cylinder that forms the 
flight tube.  Concentric to the reflectron assembly at the center is an opening 
from which the laser light enters opposite the source to illuminate it directly 
and begin the reflected flight.  The entire drift region length is 2.00 inches 
usually, and the diameter of the assembly is 2.20 inches.  The detector is dual 
channelplate.  A 3.3 mm stainless steel probe moves through the vacuum wall via 
a ball valve in the center hole and behind the grid located at the front of the 
detector.  Probe tip is electrically insulated from probe body and shielded from 
the detector using a 4.0 mm dia grounded sleeve.  The source region is 0.5 mm to 
reduce effects of source length on mass resolution, and a 4500 V potential can 
be applied.  A fine nickel mesh grid terminates the high source field, either 
333 lines per inch for 70% transmission or 1000 lines per inch for 50% 
transmission.  Arcing from the probe tip is reduced by polishing it with 
12,000-grit abrasive.  Reflectron is ground 1.75 inch (44.5 mm) i.d. stainless 
stell cylinder 2.125 in (54 mm) long, and capped with 1.6 mm thick stainless 
stell electrode plate mounted on movable polycarbonate plug, and terminated on 
other end with grounded nickel mesh (333 lines per inch).  Four ceramic rods 
extending from detector through holes running through the reflectron plug 
provide alignment.  Reflectron depth can be adjusted from 0.63 to 1.34 in or the 
drift region from 0.16 to 1.18 in.  The entire assembly is mounted within a 
vacuum chamber. The laser is a pulsed Laser Photonic Model LN-1000 nitrogen 
laser, focused through fused silica window and through a CVI Laser 250 mm focal 
length lens.  The probe is kept at +4.4 kV, ions reflected back to detector 
which is biased at -3.3 kV and using reflectron voltages of +6.0 kV.  Detector 
is Galileo MCP, modified with addition of shielded and offset conical anode, 
coupled to 2 Gsample per second adaptor on a Lecroy Model 9374M 1 GHz digital 
oscilloscope, and data acquisition and processing was done using TOFWARE from 
ILYS software.  Peptides at 1 &micro;M are put on the probe at 0.5 &micro;l 
followed by 0.5 &micro;l of satd CHCA in 50% EtOH. In some cases, 0.5 &micro;l 
sample, then 0.5 &micro;l 5% ammonium citrate, then 0.5 &micro;l of matrix.  For 
oligonucleotides, 0.5 &micro;l of 1 &micro;M solution then 0.5 &micro;l matrix, 
which is satd 6-aza-2-thiothymine (AAT) in 50% MeCN, from which 90 micro;l 
supernatant is mixed with 10 &micro;l 50 mM ammonium citrate.  Spectra are the 
average of 50 scans.
<p>
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Pleissner K-P, Hoffman F, Kriegel K, Wenk C, Wegner S, Sahlstr&ouml;m A,
Oswald H, Alt H, Fleck E
 </span>
 <span class="year">
(1999)
 </span>
 <span class="jname">
Electrophoresis
 </span>
 <span class="jvol">
20,
 </span>
 <span class="pages">
755-765.
 </span>
<p class="reftitl">
New Algorithmic Approaches to Protein Spot Detection and Pattern Matching in
Two-Dimensional Electrophoresis Gel Databases
<p class="Para1">
Before pattern matching can occur, spots must be detected for which there are
several algorithms.  Current spot detection mthods fall into two classes:
Gaussian fitting and Laplacian of Gaussian (LoG) spot detectors (current
commercial systems are Kepler, Gellab-II, PDQuest, Melanie, Phoretix.
A new <b>hierarchical watershed transformation</b> (WST) combined with
feature extraction was used.  Gel images are downloaded from internet,
followed by calculating a gradient image, performing the WST, localizing
spot regions, merging partial spot regions, then calculating spot list.
<p>
Watersheds correspond to regions above a topographic or contoured plane
and which fall below a <q>waterline</q>.  Each watershed forms from a computation
of local/global maxima and minima in the image signal, creating image
segmentation.  Regions are obtained using the WST on the gradient image
(first derivative).  A mosaic image is achieved.  Well-located closed
contours are a result of WST, although strong over-segmentation can also
result from image noise amplifed in the gradient image calculation.
Prior to WST, a Gaussian smoothing prefilter can alleviate noise effects.
<p>
Ideally all regions should correspond to a spot in the mosaic image, but
oversegmentation a spot may be divided into several subregions (partial
spot regions).  Algorithms must be used that merge several regions together
to form a single spot.  A preprocessing step which is <b>gray-value
thresholding</b>:  a number of spot regions is reduced using a simple
signal threshold on the assumption that spots have a significantly lower
gray value than the background where black = 0 and white = 255 (in an 8-bit
set of levels).  After WST to obtain regions, the average gray value is
determined.  Then the gray value distribution for the gel image is constructed,
and applied thresholds evaluated.  Spots and partial spot regin in the mosaic
will border on background regions.  Regions with a neighboring region having
a significantly higher average could be spot or partial spot regions.  Suppose
there are <b><i>R</i></b> set of regions and <i><b>N</b></i>= |<b><i>R</i></b>|
total count of regions in the mosaic image.  Neighbor (<i>r</i>) is the
set of neighboring regins of <i>r</i> &isin; <b><i>R</i></b>.
<p>
<b>Criterion 1</b>.  Neighbor difference <i>D</i>(<i>r</i>) of a region
is given by:  <i>D</i>(<i>r</i>) =
inf{|&mu;(<i>r</i>) - &mu;(<i>r</i>&prime;)| |<i>r</i>&prime; &isin;
Neighbor(<i>r</i>)}
<p>
A candidate for a spot or partial spot region is a region <i>r</i> such
that <i>D</i>(<i>r</i>) &gt; <i>t</i><sub>1</sub> where <i>t</i><sub>1</sub>
is a threshold given by 40% of the gray-value variation within the image.
The assumption is not true for all partial spot regions, and especially
partial spot regions in middle of spots not bordering on a background region.
These regions will have low gray values and can be found using a second
threshold value <i>t</i><sub>2</sub>.
<p>
<b>Criterion 2</b>.  A spot or partial spot region candidate region
<i>r</i> &isin; <i>R</i> is such that &mu;(<i>r</i>) &lt; <i>t</i><sub>2</sub>
where <i>t</i><sub>2</sub> is a threshold give by 20% of the total gray
variation within the gel image.  The 40% and 20% threshold values were
determined empirically.
<p>
The preprocessing step uses two specific gray-value thresholds to
reduce the count of candidate regions significantly, but it is not
specific to find exactly the correct spot candidates  or remove
all nonspecific regions.  Convex curvatures are obvious features found
on a topographic surface representing the gel image signal;  it is also
true for partial spot regions.  WST is performed on the gradient image
(1st derivative) and thus the watersheds separate regions of convex and
concave curvature.  Only regions with convexities are to be found as
spot candidates.  The curvature <i>C</i>(<i>r</i>) of a region <i>r</i>
&isin; <i>R</i> is defined as:
<p class="eqn">
<i>C</i>(<i>r</i>) = &sum;<i>f</i> &prime;&prime;(<i>p</i>)
&nbsp;&nbsp;&nbsp;<i>p</i>&isin;<i>r</i>
<p>
where the sum is taken over all pixels (<i>p</i>) in region <i>r</i>
and <i>f</i> &prime;&prime; is the second derivative of the image.  A
candidate for a spot or partial spot region is a region <i>r</i> that
satisfied <i>C</i>(<i>r</i>).  With the two gray-value criteria and the
curvature thresholding, remaining regions correspond to regions of spots
or partial spots.
<p>
Spots have two characteristics:  approximately elliptical shapes; and for 
neighboring partial spot regions within a spot, the local curvature 
<i>C</i><sub>L</sub> is convex in a small neighborhood along their common border 
(watershed line).  These are used in the merging of partial spot regions but the 
two properties are not sufficient to characterize the regions, and so a 
combination is used.  To examine similarity of a region <i>r</i> to an ellipse, 
and ellipse <i>E</i><sub>r</sub> is made within the circumscribed rectangle of a 
region.  A &chi;<sup>2</sup> test is used to look at the difference or 
similarity between the constructed ellipse <i>E</i><sub>r</sub> and the region 
<i>r</i>.  A region is merged with neighboring regions if the similarity of the 
merged region  to an ellipse is higher than the similarity of each separate 
region to an ellipse.  The local curvature <i>C</i><sub>L</sub> for the 
neighboring regions <i>r</i> and <i>r</i>&prime; is found through the 2nd 
derivative in a 3 &times; 3 pixel neighborhood along the common border.  The 2nd 
derivative is computed in the direction of the line connecting the centers of 
area/gravity of <i>r</i> and <i>r</i>&prime;.  Partial spot regions, together 
representing one spot, are expected to have a positive curvature along their 
watershed line, and in this case, regions can be merged.  It the local curvature 
is negative (concave), then regions should not be merged.  Using both criteria 
the algorithm would not lead to merging two or more neighboring circular spots, 
even if they overlap.  Spots consisting of separate spot regions resulting from 
a segmentation process should nonetheless be merged.  At the end, each identifed 
spot <i>s</i> is assocated with coordinate <i>x</i>(<i>s</i>), 
<i>y</i>(<i>s</i>) of its center of gravity with intensity <i>i</i>(<i>s</i>) 
being the sum of all gray-values in the region.  Vectors <i>x</i>(<i>s</i>), 
<i>y</i>(<i>s</i>), <i>i</i>(<i>s</i>) of all spots are assembled into a list.
<p>
With spot detection complete, between-gel spot matching is the next phase with 
several algorithms described.  Matching proficiency can be can be improved using 
pre-assigned corresponding spot pairs (landmarks), and some algorithms only work 
with landmarking.  The Melanie II gel analysis system will work without manual 
landmarking.  This paper reports a geometric method of spot patterns and 
associated intensities and is independent of geometric and densitometric 
(gray-value) image resolution.  It is robust in the presence of noise 
(additional/missing spots).  Matching makes use of the spot detection data (2D 
coordinates + intensity) only;  the intensity  is used only to obtain an 
intensity order relationship while the actual intensity value <i>i</i>(<i>s</i>) 
itself is not important.
<p>
For geometric point pattern matching, let <i>P</i> denote a point pattern which 
in this implementation is a set of intensive points from a small window in a 
source image.  Compute all occurrences in <i>P</i> with another point set, 
<i>T</i>, spots in a target image.  The pattern search can be likened to looking 
for a given star constellation in a celestial chart. There is an admissible 
space <i>A</i> representing transformations (translations, rigid motions and/or 
scalings, rotations) which can be used to map the pattern as close as possible 
to point set <i>T</i>.  In addition there is a distance measure <i>d</i> between 
patterns.  Also the Hausdorff distance <i>H</i> is often considered.  In general 
a search for an <i>f</i> &isin; <i>A</i> and a pattern <i>Q</i> in <i>T</i> for 
which d(<i>f(P),Q</i>) &le; &epsilon;, where &epsilon; is an error tolerance. 
For exact matching &epsilon; = 0.
<p>
Two approaches have proved useful in point matching, the alignment method and 
geometric hashing.  The alignment method uses a similarity transformation based 
on a reflection through the mapping of a single line segment.  Thus two points 
<i>a</i> and <i>b</i> in pattern <i>P</i> are chosen and the edge <span 
style="text-decoration:overline;"><i>ab</i></span> is mapped to all edges <span 
style="text-decoration:overline;"><i>uv</i></span> in target set <i>T</i>.  In 
each mapping a check is made if it induces a matching (large partial 
approximate) of <i>P</i>.  Even in partial matching it is not sufficient to 
consider only one edge <span style="text-decoration:overline;"><i>ab</i></span>. 
In a worst-case situation it would be necessary to map all pattern edges to 
edges in <i>T</i>. The use of scalings would make the process more complex, if 
they are allowed or not strongly restricted;  for  a given edge <span 
style="text-decoration:overline;"><i>ab</i></span> it it enough to search for a 
target edge of the same length with no scaling.  If rotations are forbidden, all 
target edges with approximately same slope as <span 
style="text-decoration:overline;"><i>ab</i></span> are checked. The alignment 
method can be accelerated if points in the pattern and the target are labeled 
with positive values (intensities) and used such that valid matching requires 
that the intensity order of the pattern is consistent with order in the target. 
The quadratic size search space of all target edges can be reduced under certain 
cases to the set of all edges in the history of the incremental Delaunay 
triangulation of <i>T</i> which has an expected linear size. There is a 
considerable expenditure of time for doing the matching for each pair of edges 
<span style="text-decoration:overline;"><i>ab</i></span> in <i>P</i> and <span 
style="text-decoration:overline;"><i>uv</i></span> in <i>T</i>.
<p>
Avoiding the time spent can be helped by geometric hashing.  A hash table is
produced in preprocessing, then the number of pattern point matched by a
transformation resulting from a legal edge pair can be counted efficiently.
At the end the matches only for the best transformations are used in determination.
<p>
<b>Local Matching Queries</b>.  Given a local spot pattern <i>P</i> in a 2D gel
image, the task is to find all local spot patterns in target image <i>T</i>
that resemble at least partially the geometric shap and spot intensities of
<i>P</i>.  Results of the local match are later used to form global match between
source and target images.  Algorithm to answer local matching queries starts from
two assumptions: (i)  the geometry of spot patterns is given by point patterns;
(ii) the algorithm should not use relative position of the selected pattern within the
source image or its location with respect to possibly given landmarks.  The source
and target images can be assumed to have the same bounding box whose size can
be used to limit the search range and speed up matching.  Linear scaling can be
done to alter the bounding box.
<ol style="list-style-type:lower-roman;">
<li> A small rectangular window in source <i>S</i> is chosen because a pattern
<i>P</i> is identified within it.  The pattern of the most intense spots are
chosen first (it is not important if the spot counts differ between gels).
Pattern size default is eight (8).
<li>In target <i>T</i> a matching pattern <i>P</i>&prime; can be found composed
of local intense spots.  Solutions are accepted in which <i>P</i>&prime;
resembles only a large portion of <i>P</i>.  This can help correct errors
made by the spot detection algorithms which have problems resolving spots
close to each other or which partially overlap.
<li> A simple rule incorporating two tolerance parameters &lambda; and &alpha; 
are used to model the pure geometric resemblance between <i>P</i> and 
<i>P</i>&prime;.  Two line segments <span 
style="text-decoration:overline;"><i>st</i></span> and <span 
style="text-decoration:overline;"><i>s&prime;t&prime;</i></span> are said to be 
similar for &lambda; and &alpha; if the differences in their slopes is &lt; 
&alpha; and the length meets the condition: 1 &minus; &lambda; &le; |<span 
style="text-decoration:overline;"><i>st</i></span>| / |<span 
style="text-decoration:overline;"><i>s&prime;t&prime;</i></span>| &le; 1 + 
&lambda;.  Two point patterns <i>P</i> and <i>P</i>&prime; are similar for 
(&lambda;, &alpha;) if there is bijection <i>f</i> between points sets such that 
<span style="text-decoration:overline;"><i>st</i></span> and <span 
style="text-decoration:overline;"><i>f</i>(<i>s</i>)<i>f</i>(<i>t</i>)</span> 
are (&lambda;, &alpha;)-similar for all <i>s,t &isin; P</i>.  In total the 
search is for (&lambda;, &alpha;)-matchings between as large as possible 
subpatterns <i>P</i>&prime;&prime; &sub; <i>P</i> and target patterns 
<i>P</i>&prime;.
<li>The absolute intensity <i>i</i>(<i>s</i>) values are not used directly
in modeling intensity resemblance between spots, but a robust heuristic
ranking rule is used assigning to each spot a discrete integer from 1 to 10.
The 500 most intense spots in the image are distributed equallly according to
cardinality between 10 and 6.  Spots that remain are given integer values
&le; 5 so that the total intensity sum in each class is the same.  In matching
the criterion that a pattern spot <i>s</i> can only be matched to a spot in
<i>P</i>&prime; can be done if the discrete intensity differs by at most 2-fold.
<li>The edge similarity constants &lambda; and &alpha; are usually small
(default values are 0.2 for both), and it is known that each (&lambda;,&alpha;)-matching
between <i>P</i>&prime;&prime; &sub; <i>P</i> and <i>P</i>&prime; is close to
a translation <i>t</i>;  more exactly it is the Hausdorff distance
<i>H</i>&tilde;(<i>t</i>)(<i>P</i>&prime;&prime;,<i>P</i>)  between a translated
<i>P</i>&prime;&prime; and <i>P</i>&prime; representing the worst case bound from
above by max<sub>s,t&isin;P<sup>2</sup></sub>&epsilon;|<i>st</i>|.  Other that size of
<i>P</i>&prime;&prime;, an evaluation of the match could be the Euclidean distance
of the center of <i>P</i>&prime; from an expected center position of transformed
<i>P</i> within the target image, given its source position is known.
</ol>
<p>
The key idea, first used in a cited reference in finding a star constellation
in a star atlas, is as follows.  Call a triplet of spots in a gel image
the most intense if its circumcircle does not contain a spot that is more intense
than any of the triplet.  An edge which connects spots <i>s,t</i> is intensive
if there exists a third spot forming an intensive triplet together with <i>s</i>
and <i>t</i>.  Intensive edges described here are related to Delaunay triangualtion
construction understood in computational geometry.  Triangulation of point set
<i>S</i> in the plane is called Delaunay triangulation if for each triangle in
the triangulation its circumcircle contains only the triangle points.  The
triangulation is done in steps (incrementally) by adding one point after another.
Assume that the Delaunay triangulation of a gel image is computed incrementally
but inserting spots in order of decreasing intensity.  Then the set of all
Delaunay triangles and edges occurring during the history of that process is
exactly the set of intensive triangles and intensive edges.
<p>
Hist* (<i>T</i>) is a data structure representing all intensive edges together
with their lengths and slopes and so-called flipped diagonals.  With a new
line segment creating a bisection of the triangle, any segment that crosses
it is deleted from the current triangulation.  Four Delaunay edges (outer
segments of the bisected triangle) and two flipped diagonals are added to
Hist*(<i>T</i>).  The edge set Hist*(<i>T</i>)is then used instead of the set
of all edges in the target for the following reasons: (i) While the count of
all edges is quadratic wrt to <i>n</i> count of spots, the expected count
of edges in Hist*(<i>T</i>) is bounded by 12<i>n</i>. (ii) If a pattern <i>P</i>
of locally intensive spots occurs in <i>T</i>, then one expects that despite
the possible noise that cause changes in intensity order, many of the edges
connecting spots in <i>P</i> will be (&lambda;,&alpha;)-similar to edges in
Hist*(<i>T</i>).  Up to this point our technique is like Weber et al.
<p>
For Weber et al., each occurrence of a target edge that is of equal length is
extended with a source pattern edge to a matching of the complete source pattern.
A different strategy is called for because a small but nevertheless considerable
length and slope tolerance that imply an edge search range in Hist*(<i>T</i>)
which is too large although already linear in size.  The alternative to the
alignment technique is the two-step variant of geometric hashing.  First
compute all locations in a target image where good matching with the pattern
is likely to occur, and then compute actual patterns that answer the local matching
query.
<p>
For each pattern edge <i>e</i> target edges <i>e</i>&prime; in Hist*(<i>T</i>)
meeting tolerance bounds wrt to length, slope, and discrete spot intensities
are found and the vector <i>t</i>(<i>e,e</i>&prime;) translating the midpoint
of <i>e</i> to that of <i>e</i>&prime; is computed.  A scoring table is kept
storing translation vectors indirectly and yielding clusters of such vectors.
Such clusters may correspond to possible matching locations:  the bounding box
of <i>T</i> is interpreted as the possible space for translation vectors
<i>t</i>(<i>e,e</i>&prime;), and then a regularly spaced grid is overlaid
on the translation space with an integer scores data structure, initially zero,
all maintained and defined for each grid node.  Each translation vector increases
the score among the four grid nodes defining the grid cell into which the vector
occurs.  Cell is divided into four rectangles by <i>t</i>(<i>e,e</i>&prime;).
Each of the four grid nodes adds it current score an amount proportional to
area of the opposite rectangle given by the total area by 100.
Score(<i>i</i>,<i>j</i>) is the total score accumulated in grid node (<i>i</i>,<i>j</i>)
after probing all <sub>|<i>P</i>|</sub><i>C</i><sub>2</sub> pattern edges.
All local maixma that are greater than a threshold value depending on the
cardinality of <i>P</i> are considered to correspond to potential matching
locations.
<p>
After scoring, a list of putative locations of matching pattern centers (<i>c</i>)
in the target is generated with each point <i>c</i> corresponding to a
translation <i>t<sub>c</sub></i> for which <i>t<sub>c</sub></i> (<i>P</i>)
is computed, mapping the pattern into the target.  for each source spot <i>s</i>
&isin; <i>P</i>, neighbors of <i>t<sub>c</sub></i>(<i>s</i>) in the Delaunay
triangulation of <i>T</i> are sought, constituting a list of matching candidates
for pattern spot <i>s</i>.  A maximal subste that defines (&lambda;,&alpha;)-matching
with subpattern of <i>P</i> is selected using an exhaustive search.
<p>
A geometric distortion between pattern <i>P</i> and counterpart <i>P</i>&prime;
in the target can be more severe, done by increasing tolerances in the values
for slope and length, increasing time bounds for geometric hashing process.
A standard heuristic trick is to distort the pattern <i>P</i> iteratively in a
typical (application-dependent) way and search for the distorted pattern but keep
similarity tolerances low.
<!-- ============================================================= -->
<p class="tags">
silver, peptides, fragmentation, sequencing, FD
<p class="ref">
 <span class="authors">
Chu IK, Guo X, Lau T-C, Siu KWM
 </span>
 <span class="year">
(1999)
 </span>
 <span class="jname">
Anal. Chem.
 </span>
 <span class="jvol">
71,
 </span>
 <span class="pages">
2364-2372.
 </span>
<p class="reftitl">
Sequencing of Argentinated Peptides by Means of Electrospray Tandem Mass
Spectrometry
<p class="Para1">
Two PE SCIEX triple quad (API 365 and API 3000) mass spectrometers were used.
Peptides were prepared at 2 pmol/&micro;l in 50% MeOH with 10 &micro;M AgNO<sub>3</sub>.
Infusion is 2 &micro;l/min using air as nebulizer.  Nitrogen pressure in Q2 is
3 mTorr and collision energy at center-of-mass frame (<i>E</i><sub>cm</sub>) is
1-3 eV.  1 mg proteins (bov cyt c, ubiquitin) are dissolved in 0.1 ml 8 M urea,
40 &micro;l 800 mM ammoninum carbonate, and then 22 &micro;l 2 mg/ml trypsin added,
left 24 h at 37&deg;.  Water added to 1 ml, then 10 &micro;l glacial acetic acid
added.  Digests are sampled, silver nitrate added to 10 &micro;M.
<p>
The first figure shows product ion spectra of [M+<sup>107</sup>Ag]<sup>+</sup>. 
Note Ag has two stable isotopes, 107 and 109, which have about equal abundance. 
Three spectra with energies of <i>E</i><sub>cm</sub> = 1.5, 2.0 and 2.5 eV are 
shown.  Ions are [a<sub>n</sub>-H+Ag]<sup>+</sup>, 
[b<sub>n</sub>-H+Ag]<sup>+</sup> and [b<sub>n</sub>+OH+Ag]<sup>+</sup>.  Figure 
shows product ion yields increase with collisional energy.  If spectra at each 
energy level are summed to produce a composite, it is a convenient way to 
present minimal number of search spectra to triplet/doublet identification 
algorithm.  Alternatives include acquiring product-ion spectra where 
<i>E</i><sub>cm</sub> depends on <i>m/z</i>:  a mass spectrum is shown in which 
<i>E</i><sub>cm</sub> is linear function of <i>m/z</i>: <i>E</i><sub>cm</sub> = 
(-0.0015798)(<i>m/z</i>) + 2.5474, in which <i>E</i><sub>cm</sub> = 2.5 eV at 
<i>m/z</i> 30 and 1.5 eV at <i>m/z</i> 663.  Spectra are of quality that the 
masses are easily determined without computation.  Triplet patterns of peaks can 
be found which are 18 and 28 units apart, and correspond to the three ion types 
(one <i>a</i>, and two <i>b</i> ions) above.  For example, for Leu-enkephalin, 
the [M+Ag]<sup>+</sup> ion is at <i>m/z</i> 662.2 and the first triplet 
<i>m/z</i> 548.9, 531.2, and 503.1 show differences of 17.7 and 28.1 units 
between the three, and representing [b<sub>n</sub>+OH+Ag]<sup>+</sup> for the 
largest ion of the three.  This latter ion is the truncated peptide after 
cleavage of the C-terminal residue. Difference between [LeuEnk+Ag]<sup>+</sup> 
and [b<sub>4</sub>+OH+Ag]<sup>+</sup> is 662.2 &minus; 548.9 = 113.3, which 
identifies either Leu or Ile (theoretical is 113.1 for +1 ions). Continued mass 
difference determinations between [b<sub>n</sub>+OH+Ag]<sup>+</sup> ions gives a 
proper sequence tag.  The last triplet pattern has <i>m/z</i> 300, which 
indicates probably two amino acids from the N-terminus.  The third peak in the 
triplet has <i>m/z</i> 299.1 and would be the [a<sub>2</sub>-H+Ag]<sub>+</sub> 
ion.  Cleavage between residue 1 and 2 would not produce a 
[b<sub>1</sub>-H+Ag]<sup>+</sup> ion since most <i>b</i> ions are oxazolones and 
must be two residues in this state, and [b<sub>1</sub>+OH+Ag]<sup>+</sup> is 
usually missing. The [a<sub>1</sub>-H+Ag]<sup>+</sup> ion is present however at 
<i>m/z</i> 242.1, and the different with [a<sub>2</sub>-H+Ag]<sup>+</sup> at 
<i>m/z</i> 299.1 is 57.0, identifying 4th residue from C terminus as glycine. 
The N-terminal residue can be found by substracting Ag isotope mass from proton 
(106.9 - 1.0 = 105.9) and then this from the mass of the m/z 242.1 ion, leaving 
136.2, the immonium ion of Tyr.  Thus a sequence of YGGF(L/I) is determined. The 
same kind of analyses are shown for other pure peptides and for tryptic 
peptides.
<p>
The [M+<sup>107</sup>Ag]<sup>+</sup> product spectrum of GGEGG.  The difference
between [M+Ag]<sup>+</sup> and [<i>b<sub>n</sub></i> + OH + Ag]<sup>+</sup>
482.1 &minus; 425.1 = 57.0, making it Gly.  The spectrum also shows that the
next <i>b</i> ion is also Gly.  These residues indicate fragmentation from the
C-terminus.  The next degradation indicates Glu.  The [<i>a</i><sub>1</sub>
&minus; H + Ag]<sup>+</sup> is weak and the small <i>m/z</i> 193.1 of 
[<i>a<sub>n</sub></i> &minus; H + Ag]<sup>+</sup> indicates it is 
[<i>a</i><sub>2</sub> &minus; H + Ag]<sup>+</sup>, which for GG is a theoretical
193.0.  Finally there is a peak <i>m/z</i> 136.0 to indicate an N-terminal G
<i>a</i><sub>1</sub> ion.
<p>
Looking at bradykinin (RPPGFSPFR), the sequencing of the protonated 
[M+H]<sup>+</sup> is difficult because of the proton is sequestered by guanidine 
of one or both of the two Arg residues, and so it does not affect amidic 
functional groups. In contrast silver ion binds to different sites on peptide. 
The spectrum reveals for each residue broken down a pattern of triplet peaks 
([<i>b<sub>n</sub></i> + OH + Ag]<sup>+</sup>, [<i>b<sub>n</sub></i> &minus; H + 
Ag]<sup>+</sup>, and [<i>a<sub>n</sub></i> &minus; H + Ag]<sup>+</sup>)
to reveal the tag FSPFR.  For peptides longer than 6 residues in fact, it is
not possible to sequence beyond 5 or 6 residues.  An advantage to sequencing with
silver ion is that it provides the signature triplet pattern as opposed to protonated
peptides, and the pattern helps to identify fragment ions for sequencing.
It was observed that Ag<sup>+</sup> bound peptides with C-terminal Arg, Met,
and Lys residues show strong [<i>y<sub>n</sub></i> + H + Ag]<sup>+</sup> product
ions, and this confirms the cleaved residue.  Bradykinin shows these 
[<i>y<sub>n</sub></i> + H + Ag]<sup>+</sup> ions for <i>n</i> = 3 to 5, and this
too is matched by a triplet peak search.  For the Pro residue the triplet signal
is weak, which itself confirms such a residue.  An oxazolone 
[<i>b<sub>n</sub></i> &minus; H + Ag]<sup>+</sup> cannot be formed from the Pro
and the weak [<i>b<sub>n</sub></i> &minus; H + Ag]<sup>+</sup> is probably a 
ketene.  Pro residues can also be identified by 
[<i>y<sub>n</sub></i> + H + Ag]<sup>+</sup> ions or identification of next triplet.
The difference between [<i>b<sub>n</sub></i> + OH + Ag]<sup>+</sup> for <i>n</i>
= and <i>n</i> = 3 for the spectrum computes to 243.8, which sums to Phe + Pro.
<p>
The dynorphin A fragment 1-7 YGGFLRR product spectrum for [M + 
<sup>107</sup>Ag]<sup>+</sup> is another highly basic peptide.  A partial 
sequence of FLRR was determined by the [<i>y<sub>n</sub></i> + H + 
Ag]<sup>+</sup> product series.
<p>
To determine whether an unknown (not known <i>a priori</i>) could have its sequence
determined by the search algorithm, a blinded scientist was given the task of
sequencing. The partial sequences determined were correctly.
<p>
The greatest potential of sequencing argentiated peptides involves identifying
<ul>
<li>[<i>b<sub>n</sub></i> + OH + Ag]<sup>+</sup>
<li>[<i>b<sub>n</sub></i> &minus; H + Ag]<sup>+</sup>
<li>[<i>a<sub>n</sub></i> &minus; H + Ag]<sup>+</sup>
<li>[<i>y<sub>n</sub></i> + H + Ag]<sup>+</sup>
</ul>
<p>
where the first three are the triplet particularly observed.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Cole RB
 </span>
 <span class="year">
(2000)
 </span>
 <span class="jname">
J. Mass Spectrom.
 </span>
 <span class="jvol">
35,
 </span>
 <span class="pages">
763-772.
 </span>
<p class="reftitl">
Some Tenets Pertaining To Electrospray Ionization Mass Spectrometry
<p class="abstract">
<b>ABSTRACT</b>.
The field of electrospray ionization mass spectrometry is reviewed with emphasis placed upon advances in the
elucidation of fundamental mechanistic aspects of the ionization process that have been reported over the past
10 years. The analytical consequences of these findings are also examined. Eight central conclusions or tenets
are presented, as deduced from the body of work contained in 80 references.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Kebarle P
 </span>
 <span class="year">
(2000)
 </span>
 <span class="jname">
J. Mass Spectrom.
 </span>
 <span class="jvol">
35,
 </span>
 <span class="pages">
804-817.
 </span>
<p class="reftitl">
A brief overview of the present status of the mechanisms involved
in electrospray mass spectrometry identification technology
<p class="Para1">
Two models have been proposed for the mechanism by which electrospray ionization 
occurs.  The <b>ion evaporation model</b> (IEM) proposes that ions move into the 
gas phase from droplets whose radii has decreased to &le; 10 nm because of 
charge repulsion.  The <b>charged residue model</b> (CRM) proposes instead that 
analyte ions themselves never transit from the liquid phase to the gas phase, 
but that droplet fission proceeds rapidly to leave nothing but a charged single 
ion (i.e., the evaporation is of the solvent molecules, not the analyte ion). 
The steps in both models require the production of charged droplets at the ES 
capillary tip; the evolution of the charged droplets and the formation of gas 
phase ions.  Sensitivity in ESMS depends on the analyte's chemical properties 
and concentration used, and on the presence of other analytes.
<p>
An electric field in the ES capillary causes a partial separation of positive 
and negative ions in solution. In positive ion mode, positive ions are driven to 
the surface of the liquid and negative ions toward the center.  As fluid emerges 
from the tip of the capillary, the positive ions at the surface and the pull of 
the electric field at the tip is greater than the surface tension of the liquid. 
These forces cause the liquid to expand into a cone called the <q>Taylor</q> 
cone.  The tip of the cone is the least stable point elongates into a liquid 
filament.
<p>
To enable onset of electrospray requires an electric field
<i>E</i><sub>0</sub>
expressed as:
<table class="eqnserif" style="text-align:center;">
<tr><td rowspan="2"><i>E</i><sub>0</sub> =
 <span style="font-size:200%;">(</span>
 <td class="numer">2&gamma; cos 49&deg;
 <td rowspan="2"><span style="font-size:200%;">)<sup>&frac12;</sup></span>
<tr><td>&epsilon;<sub>0</sub><i>r</i><sub>c</sub>
</table>
<p>
where &gamma; is the surface tension of the liquid, cos 49&deg; is the 
half-angle of the Taylor cone, &epsilon;<sub>0</sub> is the permittivity of 
vacuum, and <i>r</i><sub>c</sub> is the capillary radius.  Note the dependence 
on the surface tension of the liquid.  Water has a higher surface tension, and a 
flow with a higher aqueous component will require higher voltage.  The field is 
often higher than that which will cause corona discharge, and so special 
arrangements are needed to suppress discharges (arcing) with a water solvent. 
This is particularly true in negative ion mode, as there is a lower voltage 
threshold for electric discharge.
<p>
Successful formation of the Taylor cone in ES requires a certain ionic strength, 
with a minimum electrolyte concentration of 10 &micro;M.  <i>Total droplet 
current</i> (TDC) increases rapidly with change in the electrolyte concentration 
up to 10 &micro;M, but the rate of increase falls off above that concentration. 
TDC or <i>I</i> can be defined as follows:
<p class="eqnserif">
<i>I</i> &cong; constant &times; (&gamma;<i>KV</i><sub>f</sub>&epsilon;/&epsilon;<sub>0</sub>)<sup>&frac12;</sup>
<p>
(note &cong; means <q>approximately equal to</q>) where <i>K</i> = solution 
conductivity, <i>V</i><sub>f</sub> = solution flow rate, &epsilon; = 
permittivity of solution, and &gamma; and &epsilon;<sub>0</sub> have been 
defined above.  The constant is usually 18 for liquids such as water, methanol, 
acetonitrile and formamide, which have a high dielectric constant 
(&epsilon;/&epsilon;<sub>0</sub>).  This current equation and radius and charge 
equations below are valid when a <q>cone-jet</q> electrospray is attained:  this 
is the cone from the capillary tip becoming a liquid filament (jet) where flow 
rates are few &micro;l/min and electrolyte concentrations are &lt; 
10<sup>&minus;3</sup>, typical electrospray MS conditions.
<p>
Electrospray devices using metal capillaries deliver continuous current, and so 
an electrochemical oxidation reaction must occur (the negative electrode must be 
the spectrometer ground wherever positive ions hit). Use of a Zn electrode 
showed the production of Zn<sup>2+</sup> ions at the capillary tip.  The 
[Zn<sup>2+</sup>] was found to be equal to [<i>Q</I>], which is the 
concentration of charge in moles charge per liter of charged droplets. 
[<i>Q</i>] is defined as:
<p class="eqnserif">
[<i>Q</i>] = <i>I</i> / (<i>FV</i><sub>f</sub>)
<p>
where <i>F</i> is Faraday's constant and <i>I</i> and <i>V</i><sub>f</sub>
have been defined.  That this equation applies indicates that the
electrospray works as an electrochemical cell.  It has been shown that
some compounds not ionized normally in solution are ionized by oxidation
in positive ion mode, or by reduction in negative ion mode.  Polycyclic
aromatic hydrocarbons (PAHs) are quite susceptible to free radical
formation, and one-electron oxidations of PAHs such as perylene
([C<sub>20</sub>H<sub>12</sub>]<sup>+&bull;</sup>) and rubrene
([C<sub>42</sub>H<sub>28</sub>]<sup>+&bull;</sup>) are possible with ES MS.
&beta;-Carotenes and other carotenoids are as efficient.  Electrochemical
oxidation-reduction is unwanted in some cases, such as metal analyte
quantitation of Ag<sup>+</sup>, Cu<sup>2+</sup>, and Hg<sup>2+</sup>.
<p>
The radius <i>R</i> and charge <i>q</i> of the initial droplets formed
at the capillary tip have been deduced:
<p class="eqnserif">
<i>R</i> &cong; (<i>V</i><sub>f</sub><i>&epsilon; / K</i>)<sup>1/3</sup><br>
<i>q</i> &cong; 0.7[8&pi;(&epsilon;<sub>0</sub>&gamma;<i>R</i><sup>3</sup>)<sup>&frac12;</sup>]
<p>
where all symbols and variables have been previously defined
(&cong; is <i>approximately equal to</i>).  It is useful to note that
smaller droplets (lower radii) can be achieved by higher electrolyte
concentrations (higher <i>K</i>).  Applying the equation for changing
concentrations of NaCl, the following table is produced:
<table style="text-align:center;">
<tr><th>[NaCl] <td>0.5 mM  <td>5 mM   <td>20 mM
<tr><th><i>R</i> (&micro;m)     <td>0.96    <td>0.40    <td>0.28
<tr><th><i>N</i><sub>P</sub> <td>83,000  <td>26,000  <td>13,000
</table>
<p>
<i>N</i><sub>P</sub> is the number of elementary charges, which indicates
the charge of the parent droplets.
<p>
Evaporation of solvent (from use of hot nitrogen desolvation gas) causes
decreased droplet radius.  With 100 &micro;s, the radii are close the
Rayleigh limit:
<p class="eqnserif">
<i>q</i> = 8&pi;(&epsilon;<sub>0</sub>&gamma;<i>R</i><sup>3</sup>)<sup>&frac12;</sup>
<p>
This defines the point where Coulombic repulsion overcomes surface tension
and droplet fission occurs.  Solvents of low volatility and radii &gt;
1 &micro;m show such fission.  Budding from the droplet occurs, with
offspring radius about 0.1 of the parent radius (<b><i>R</i><sub>P</sub></b>).
Offspring droplets represent 2% of the mass and 15% of the charge
of the parent droplet.  Several nanometer-sized droplets with only a
few elementary charges eventually make the gas phase.
<p>
CRM proposes production of gas phase ion in a scheme as that just mentioned
until no further evaporation of solvent occurs.  In single analyte
solutions of species MX, offspring droplets will be composed of mostly
M<sup>+</sup> and some (M<sup>+</sup>)<sub><i>x</i></sub>(MX)<sub><i>n</i></sub>
where <i>x</i> and <i>n</i> are low, but for the parent droplet, both are
high.
<p>
IEM proposes that ions directly enter the gas phase after a combination
of solvent evaporation and Coulombic-based droplet fission bring the
size of charged droplets to a limit radius (about 10-20 nm).  Droplet
charge required for IEM is lower than that required for Coulombic-based
fission.  Ion evaporation replaces fission and M<sup>+</sup> evaporation
continues until the solid residue M<sup>+</sup>)<sub><i>x</i></sub>(MX)<sub><i>n</i></sub>
forms.  Ion evaporation rate was based on transition state theory:
<table class="eqnserif" style="text-align:center;">
<tr><td rowspan="2"><i>k</i><sub>I</sub> =
 <td class="numer"><i>k</i><sub>B</sub><i>T</i>
 <td rowspan="2">e<sup>&minus;&Delta;<i>G</i><sup>&Dagger;</sup>/<i>RT</i></sup>
<tr><td><i>h</i>
</table>
<p>
Here <i>k</i><sub>I</sub> is the ion evaporation rate from droplets,
<i>k</i><sub>B</sub> is Boltzmann's constant, <i>h</i> is Planck's constant,
and &Delta;<i>G</i><sup>&Dagger;</sup> is an expression of the free energy
of the factors that result in a late transition state
(i.e., after the ion has left the droplet).  The barrier is represented by
attraction of the ion to the counterions in the polarizable droplet
and the repulsive forces between ion and charges in droplet,
&Delta;<i>G</i><sup>&Dagger;</sup>(<i>E</i>).  There is also a
solvation free energy term, &minus;&Delta;<i>G</i>&deg;<sub>sol</sub>:
<p class="eqnserif">
&Delta;<i>G</i><sup>&Dagger;</sup> =
&minus;&Delta;<i>G</i>&deg;<sub>sol</sub> + &Delta;<i>G</i><sup>&Dagger;</sup>(<i>E</i>)
<p>
&Delta;<i>G</i><sup>&Dagger;</sup>(<i>E</i>) is a function of <b><i>E</i></b>,
the electric field perpendicular to the charged drop surface.  Because
charges are distributed on the surface of a sphere, Coulomb's Law
is thus:
<table class="eqnserif" style="text-align:center;">
<tr><td rowspan="2"><i>E</i> =
 <td class="numer"><i>q</i>
 <td rowspan="2">=
 <td class="numer"><i>Ne</i>
<tr>
 <td>4&pi;&epsilon;<sub>0</sub><i>R</i><sup>2</sup>
 <td>4&pi;&epsilon;<sub>0</sub><i>R</i><sup>2</sup>
</table>
<p>
<i>R</i> is the droplet radius, <i>q</i> the charge, and <i>N</i> is
the number of elementary charges <i>e</i>.
&minus;&Delta;<i>G</i>&deg;<sub>sol</sub> is effectively the free energy to desolvate ion (transfer
it to gas phase if the droplet were uncharged (neutral).  Li<sup>+</sup>,
Na<sup>+</sup>, K<sup>+</sup> and all alkali metals are strongly solvated.
&minus;&Delta;<i>G</i>&deg;<sub>sol</sub> is lowest and better for IEM
thusly when the ion takes (several) solvent molecules with it.
<p>
Rate constants <i>k</i><sub>I</sub> for droplet radii <i><b>R</b></i>
smaller than 10-20 nm were &ge; 10<sup>6</sup> s<sup>&minus;1</sup>,
meaning that ion evaporation is in the microsecond scale.  Moreover
charge <i>q</i><sub>I</sub> required for ion evaporation is lower than
Coulombic fission, meaning charge will be kept low and fission not taking
place.
<p>
Experiments to validate models made use of NaCl as the MX and made reasonable
assumptions about the distribution of gas phase ions and those trapped
in solid residues.  The distribution of spectra and ion counts matched
IEM models and agreement was worse for CRM.  Uncertainties that remained
did not allow the IEM model to be proved unequivocally however.
<p>
In other experiments, there was a focus on the size and charges of solid
residues formed after evaporation of solvent.  These are fair approximations
of the evolving charged droplet.  Again the results were in agreement with
IEM.  Very small initial droplets were produced by low flow rate, high
conductivity electrospray.  The radius of these droplets falls close to
the ion emission radius for IEM before Rayleigh disintegration occurs.
Radius and charge could be determined from the mobility and mass of charged
residues, using a <i>hypersonic impactor</i> apparatus measuring mass, and
assuming density was same as solid salt.
<p>
<b>Sensitivity</b>.  Different analytes will have different thresholds
of detection, and this will be complicated by the presence of other
ions not of interest (electrolytes, contaminants, interfering substances).
Consider a two-component solution with analyte AX and electrolyte EX
(for simplicity, in positive ion mode) and the dissociation is
A<sup>+</sup> and E<sup>+</sup> in solution (there is no induced ionization
electrochemically or otherwise).  The counterion X<sup>&minus;</sup>
in AX and EX may not be the same.  The ion current produced by A<sup>+</sup>
in the two-component system is:
<table class="eqnserif" style="text-align:center;">
<tr><td rowspan="2" style="padding-right:0.3em;"><i>I</i><sub>A<sup>+</sup></sub> = <i>pf</i>
 <td class="numer"><i>k</i><sub>A</sub><i>C</i><sub>A</sub>
  <td rowspan="2"><i>I</i>
<tr><td><i>k</i><sub>A</sub><i>C</i><sub>A</sub> +
  <i>k</i><sub>E</sub><i>C</i><sub>E</sub>
</table>
<p>
For a three-component system of AX, BX, and EX:
<table class="eqnserif" style="text-align:center;">
<tr><td rowspan="2" style="padding-right:0.3em;"><i>I</i><sub>A<sup>+</sup></sub> = <i>pf</i>
 <td class="numer"><i>k</i><sub>A</sub><i>C</i><sub>A</sub>
  <td rowspan="2"><i>I</i>
<tr><td><i>k</i><sub>A</sub><i>C</i><sub>A</sub> +
	<i>k</i><sub>B</sub><i>C</i><sub>B</sub> +
  <i>k</i><sub>E</sub><i>C</i><sub>E</sub>
</table>
<p>
<i>I</i><sub>A</sub> is the ion current registered by the MS, and 
<i>C</i><sub>A</sub>, <i>C</i><sub>B</sub>, <i>C</i><sub>E</sub> are the 
concentrations in the electrosprayed solution.  <i>I</i> is the total 
electrospray capillary current.  <i>p</i> is the sampling efficiency of the ES 
MS, or ions detected relative to total ion in gas phase (a number that reaches 
0.5, and rarely 0.8), and <i>f</i> is fraction of charge converted to gas phase 
ions, usually much smaller than <i>p</i> and dependent on sampling arrangements 
used. <i>k</i><sub>A</sub> refers to the efficiency of species A to become a gas 
phase ion. &Delta;<i>I</i>/&Delta;<i>C</i> is higher when <i>C</i><sub>E</sub> 
is &lt; 10 &micro;M, and much lower when <i>C</i><sub>E</sub> is &gt; 10 
&micro;M (a point stated earlier).
<p>
The relative sensitivities of any two analytes A and B can determined by 
applying the three component equation to derive:
<table class="eqnserif" style="text-align:center;">
<tr><td class="numer"><i>I</i><sub>A</sub>
 <td rowspan="2">=
 <td class="numer">
 	<i>k</i><sub>A</sub><i>C</i><sub>A</sub>
<tr><td><i>I</i><sub>B</sub>
 <td><i>k</i><sub>B</sub><i>C</i><sub>B</sub>
</table>
<p>
It is convenient to set <i>C</i><sub>A</sub> = <i>C</i><sub>B</sub> for the 
determination of <i>k</i><sub>A</sub>/<i>k</i><sub>B</sub>. These concentrations 
are increased while <i>C</i><sub>E</sub> is kept constant and 
<i>I</i><sub>A</sub> and <i>I</i><sub>B</sub> determined using the ES MS.  Good 
fits were obtained generally, with exacts fits when <i>C</i><sub>A</sub> and 
<i>C</i><sub>B</sub> were high (&gt; 10 &micro;M).  Only approximate fits were 
obtained with <i>C</i><sub>A</sub>, <i>C</i><sub>B</sub> &lt; 10 &micro;M. 
Plots show that below 10 &micro;M  where <i>C</i><sub>A</sub> = 
<i>C</i><sub>B</sub>, the ratio decreases to the limit 
<i>k</i><sub>A</sub>/<i>k</i><sub>B</sub> = 1.  When <i>C</i><sub>A</sub> and 
<i>C</i><sub>B</sub> are both &lt;&lt; 10 &micro;M, the current <i>I</i> and the 
droplet charge <i>Q</i> are carried by the electrolyte E with its higher 
concentration.  At low concentrations of A<sup>+</sup> and B<sup>+</sup>, ions 
are quickly depleted from the droplet with high <i>k</i><sub>A</sub> and 
<i>k</i><sub>B</sub>, leading to an apparent 
<i>k</i><sub>A</sub>/<i>k</i><sub>B</sub> = 1. The ion signal intensities 
<i>I</i><sub>A</sub> and <i>I</i><sub>B</sub> depend only on the amount of A and 
B available, i.e. they are mass-dependent. and this provides maximum 
sensitivity.
<p>
The convention of expressing current <i>I</i> as a concentration of charge 
[<i>Q</i>] represents an advance in ES MS such that currents are really 
fractions of charges on droplets due to an ion species. Analyte AX is 
distributes within the interior with concentration 
[A<sup>+</sup>]<sub><i>i</i></sub>[X<sup>&minus;</sup>]<sub><i>i</i></sub> and 
surface concentration of [A<sup>+</sup>]<sub><i>s</i></sub>. Equilibrium 
<i>K</i><sub>A</sub> = 
[A<sup>+</sup>]<sub><i>s</i></sub>/[A<sup>+</sup>]<sub><i>i</i></sub>. Charge 
and mass balance equations become [<i>Q</i>] = [A<sup>+</sup>] + [B<sup>+</sup>] 
+ ... and <i>C</i><sub>A</sub> = [A<sup>+</sup>]<sub><i>i</i></sub> + 
[A<sup>+</sup>]<sub><i>s</i></sub>, respectively.  Solutions for equations 
become apparent.  Certain assumptions are made, such that 
[A<sup>+</sup>]<sub><i>s</i></sub>, [B<sup>+</sup>]<sub><i>s</i></sub>, etc. 
transfer to the gas phase proportional to the constant product <i>pf</i>.
<p>
A number of analytes were examined in methanol to determine the 
<i>k</i><sub>A</sub>/<i>k</i><sub>B</sub> working at high concentrations 
<i>C</i><sub>A</sub>, <i>C</i><sub>B</sub>. Singly charged inorganic ions 
Na<sup>+</sup>, K<sup>+</sup>, Rb<sup>+</sup>, Cs<sup>+</sup>, and 
NH<sub>4</sub><sup>+</sup> had nearly the same coefficients.  However 
coefficients of analytes expected to concentrate mostly on the surface (i.e. 
surface active agents) have higher coefficients.  Values for <i>k</i> are &cong; 
1 for Na<sup>+</sup>, K<sup>+</sup>, Rb<sup>+</sup>, NH<sub>4</sub><sup>+</sup> 
and Cs<sup>+</sup>, Et<sub>4</sub>N<sup>+</sup> = 3, Pr<sub>4</sub>N<sup>+</sup> 
= 5, Bu<sub>4</sub>N<sup>+</sup> = 9, Pen<sub>4</sub>N<sup>+</sup> = 16, and 
heptNH<sub>3</sub><sup>+</sup> = 8.  If <i>K</i><sub>SA</sub> represents surface 
activity and is represents an equilibrium ratio of surface to bulk 
concentrations of A<sup>+</sup>X<sup>&minus;</sup>, and <i>k</i><sub>IA</sub> is 
a rate constant for ion evaporation, then the equation
<table class="eqnserif" style="text-align:center;">
<tr><td class="numer"><i>k</i><sub>A</sub>
 <td rowspan="2"> =
 <td class="numer"><i>K</i><sub>SA</sub><i>k</i><sub>IA</sub>
<tr><td><i>k</i><sub>B</sub>
 <td><i>K</i><sub>SB</sub><i>k</i><sub>IB</sub>
</table>
<p>
is an expression of the relative transition to the gas phase of two
different species A and B in IEM.  If instead CRM holds, then
<table class="eqnserif" style="text-align:center;">
<tr><td class="numer"><i>k</i><sub>A</sub>
 <td rowspan="2"> =
 <td class="numer"><i>K</i><sub>SA</sub>
<tr><td><i>k</i><sub>B</sub>
 <td><i>K</i><sub>SB</sub>
</table>
<p>
The rate constants <i>k</i><sub>IA</sub>, <i>k</i><sub>IB</sub> depend on the 
solvation free energies &Delta;<i>G</i>&deg;<sub>sol</sub> for A<sup>+</sup> and 
B<sup>+</sup>.  Low &minus;&Delta;<i>G</i>&deg;<sub>sol</sub> mean high 
evaporation constants, and &minus;&Delta;<i>G</i>&deg;<sub>sol</sub> decreases 
when surface activity increases.  If IEM is true, surface active ions have high 
<i>k</i><sub>A</sub> because <i>K</i><sub>SA</sub> is high (the surface 
concentration in the numerator is great) and <i>k</i><sub>IA</sub> is high.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Berndt P, Hobohm U, Langen H
 </span>
 <span class="year">
(1999)
 </span>
 <span class="jname">
Electrophoresis
 </span>
 <span class="jvol">
20,
 </span>
 <span class="pages">
3521-3526.
 </span>
<p class="reftitl">
Reliable Automatic Protein Identification from Matrix-Assisted Laser
Desorption/Ionization Mass Spectrometric Peptide Fingerprints
<p class="Para1">
Correct identification of MS data depends on several factors.  Size of
database is a factor:  SWISS-PROT has &gt; 70,000 protein and the likelihood
of false positive is greater with that number.  Protein size is important
since smaller proteins are less likely to generate more peptides and the
increase in peptide count creates opportunities for mismatch.  Mass measure
to a precision of 0.05% (0.05 in 100 or 5 in 10000 or 500 ppm) or 0.0075%
(or 0.0075 in 100 or 75 in 1000000 or 75 ppm) is critical.
<p>
The distribution of masses of queried peptides is important.  Most fragments
have masses in lower range (close to 800 Da) and few have higher mass at
4000 Da.  Peptide number in matching is critical, with a protein matching
more peptides giving a higher confidence result.
<p>
Suppose <i>P</i>(<i>m</i>, &Delta;) is the estimated probability for
choosing a mass in the range [<i>m</i> &minus; &Delta; <i>m</i> + &Delta;],
perhaps computed as number of peptides in the mass range as a proportion of
all peptides (<i>N</i><sub>[<i>m</i> &minus; &Delta; <i>m</i> + &Delta;]</sub>/
<i>N</i><sub>pept</sub>).  The database is queried with <i>k</i> masses.
The probability of finding one protein with any masses in the <i>k</i>
mass range = &sum;<sup><i>k</i></sup><sub>i=0</sub> <i>P</i><sub>i</sub>(<i>m</i>,&Delta;).
The probability of finding a single combination of two masses in <i>k</i>
masses is the sum of the products of the probabilities:
<p class="eqnserif">
&sum;<sup><i>k</i></sup><sub><i>i</i>=0</sub>
&sum;<sup><i>k</i></sup><sub><i>j=i</i></sub>
<i>P</i><sub>i</sub><i>P</i><sub>j</sub>.
<p>
So to find <i>n</i> masses out of <i>k</i> masses is the sum of all the
products of the probability for all <i>n</i>-ary mass combinations:
<i>T</i><sub>1</sub> = &sum;<i>P</i><sub>i</sub><i>P</i><sub>j</sub>...<i>P</i><sub>n</sub>.
<p>
For protein with <i>p</i> peptides, one can select <i>T</i><sub>2</sub> =
<sub><i>p</i> + <i>n</i> + 1</sub><i>C</i><sub><i>n</i></sub> combination of
peptides for any <i>m</i> &lt; <i>p</i>.
<p>
So the probability of finding a single protein with a peptide list of
<i>p</i> peptides from database of <i>N</i><sub>prot</sub> protein with
only <i>p</i>-membered lists is <i>T</i><sub>1</sub><i>T</i><sub>2</sub>.
And finally the probability of finding the protein <i>P</i> in the database
is sum of all probability for all single proteins:
<i>P</i> = <i>N</i><sub>prot</sub><i>T</i><sub>1</sub><i>T</i><sub>2</sub>.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Pfeifer T, R&uuml;cknagel P, Kuellertz G, Schierhorn A
 </span>
 <span class="year">
(1999)
 </span>
 <span class="jname">
Rapid Commun. Mass Spectrom.
 </span>
 <span class="jvol">
13,
 </span>
 <span class="pages">
362-369.
 </span>
<p class="reftitl">
A Strategy for Rapid and Efficient Sequencing of Lys-C Peptides
by Matrix-Assisted Laser Desorption/Ionization Time-of-Flight
Mass Spectrometry Post-Source Decay
<p class="abstract">
<b>ABSTRACT</b>.
A modification procedure for Lys-C peptides is described which
simplifies the correct assignment of the
amino acid sequence. Release of the C-terminal lysine from Lys-C
peptides by carboxypeptidase B and
subsequent N-terminal acetylation of the resulting peptides leads
to predictable shifts of the C- and Nterminal
fragment ions in Matrix-assisted laser desorption/ionisation
time-of-flight post-source decay mass
spectra and facilitates the correct assignment of mostly complete
amino acid sequences for oligopeptides.
The derived sequences of peptides from unknown proteins were used
to search in databases for homologous
protein sequences. Our method was applied to an unknown protein
isolated from eggs of Drosophila
melanogaster, resulting in the identification of a peptidyl prolyl
cis-trans-isomerase.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Zissis KD, Brereton RG, Dunkerley S
 </span>
 <span class="year">
(1999)
 </span>
 <span class="jname">
Rapid Commun. Mass Spectrom.
 </span>
 <span class="jvol">
13,
 </span>
 <span class="pages">
1755-1761.
 </span>
<p class="reftitl">
Principal Components Scores and Loading Plots for Visualization of the
Electrospray Ionization Liquid Chromatography Mass Spectra of a Mixture
of Chlorophyll Degradation Products at Different Cone Voltages
<p class="Para1">
PCA is used in LC-MS and GC-MS for classifying/grouping peaks and
compounds in complex chromatographic mixture.  When MS/MS is not
available, PCA can be used to assign ions belonging to resolved
compounds with identical masses.  Spinach leaf Chl a is extracted
by a cited method, and Chl a standard dissolved in MeOH is
degraded by 4000 lux light in 40 deg water bath with oxygen
present for 60 min.  Aliquot is dried down and resuspended in
acetone and then injected on LC-MS.  LC-MS C18 4.6 &times; 250 mm (5 &micro;m)
with gradient elution involving acetone, MeOH, and water, with a
20 ul full loop.  ESI+ was done with MS m/z 500-1000 with 1.25 s
scan.  Cone V varied from 60, 65, and 70 V.  A splitter was used
to set LC flow @ 0.5 ml/min.  Chlorophyll has many allomers
which typically varies by the presence certain groups on C13
of the closed ring formed by one of the 4 rings in the porphyrin.
<p>
Mass spectral data are collected by MassLynx and the raw data
decoded using a C++ program.  A 5.5 min in chromatogram is
scanned containing peak(s) of interest, with background
subtraction.  Mass spectral processing finds 70 <i>m/z</i> peaks
attaining threshold, determined by a peak variance-to-mean ratio,
with a threshold of 1 million ion counts.  Data is then
scaled by normalizing all selected masses to be equal in
intensity despite the relative intensities in the raw data.
After normalization, a <q>standardization</q> was done by subtracting
the mean and dividing by standard deviation.  This puts each of
the selected m/z peaks (70) on same scale, and will give weight
to low abundance ions (fragments?).  For PCA, matrix <b>X</b> is the 70
selected masses and 360 time points (360 seconds), described by
the equation <b>X = T.P + E</b>, where <b>T</b> is the scores matrix
corresponding to elution profiles, <b>P</b> is loadings matrix
corresponding to selected masses, and <b>E</b> is error term.
Three components are determined.  3-D projections are generated
by projecting Scores or Loadings values on to 2 dimensions using
rotations in XY, XZ, and YZ planes.
<p>
Cone voltage increases from 60&rarr;70 V result in more ions
in the spectrum (regarded as noise).  In the chromatogram,
5 peaks are seen, very smooth at 60 V, very unsmooth at 70 V.
Peak 4 is Chl a, and peak 5 is an isobaric epimer (mass 892).
Peaks 1-3 are a hydroxy allomer, a methoxylacetone variant,
and a ring E-opened variant (see previous papers for identification).
<p>
3-D plots of the Scores were generated on processed and scaled data.  The locations of
the five peaks are indicated on the plots.  The 70 V 3-D plot clearly
shows the effects of the noise, but the plot also shows a greater spatial
separation of the 5 peaks, even for components have identical mass (such
as peaks 4 and 5).  At 60 V, peaks 2 and 3 are clustered in one group,
peaks 4 and 5 in another.  This separation increases with cone V.
This is because of more information resulting from fragmentation of ions
occurring at the higher voltage.
<p>
3-D plots of the Loadings show the 70 peaks, which are the protonated
main mass, allomeric variants, and associated K and Na adducts.
Three main clusters are found.  One cluster contains <i>m/z</i> 962 and
978, two allomers.  From the 60 to 70 V analyses, there is a better
discrimination in ion clusters, in which some ions are regrouped
(for example, <i>m/z</i> 595-597 to chlorophyll IV, while ions
between <i>m/z</i> 600-603 and the ion at <i>m/z</i> 629 segregate to
chlorophyll III.  Ions at <i>m/z</i> 962 lie at the center, indicating
the parent nature of this group.
<p>
Typical data analysis would involve looking at mass spectra for all
four chlorophylls I-IV at the 3 different cone voltages in a conventional
approach.  The spectra show increased relative intensity for ions with a
lower <i>m/z</i>.  However, all these peaks may represent more than
one compound in a co-eluting in the chromatogram.	Chromatographic
reproducibility and resolution is complicated by plumbing systems
in which turbulent flow is more likely in capillary systems than in those
of larger diameter.  MS detects these mixing phenomena.  PCA to produce
the loadings plot visually clarifies the differences and confusion that
might be caused by misassigments in the spectrum.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Knochenmuss R, Stortelder A, Breuker K, Zenobi R
 </span>
 <span class="year">
(2000)
 </span>
 <span class="jname">
J. Mass Spectrom.
 </span>
 <span class="jvol">
35,
 </span>
 <span class="pages">
1237-1245.
 </span>
<p class="reftitl">
Secondary Ion-Molecule Reactions in Matrix-Assisted Laser
Desorption/Ionization
<p class="abstract">
<b>ABSTRACT</b>.
Ion-molecule charge- and proton-transfer reactions in the desorption plume are considered for the case of
matrix-assisted laser desorption/ionization (MALDI) with ultraviolet laser excitation, and it is proposed that
they are major determinants of the observed mass spectrum. Specific MALDI phenomena which are discussed
include the dominance of singly charged ions and analytematrix or analyteanalyte signal suppression. Should
any be formed, highly charged products can be reduced by reaction with neutral matrix, yet singly charged ions
cannot generally be neutralized in the same manner. Ion suppression effects can also be explained by similar
reactions, which in some cases involve interconversion of dissimilar ion types. The plume is proposed often to be
more under thermodynamic rather than kinetic control owing to these secondary reactions. UV/MALDI mass
spectra should therefore be largely predictable, given sufficient thermodynamic information, and appropriate
experimental conditions of sufficient analyte and plume density.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
G&ouml;rg A, Obermaier C, Boguth G, Harder A, Scheibe B, Wildgruber R,
Weiss W
 </span>
 <span class="year">
(2000)
 </span>
 <span class="jname">
Electrophoresis
 </span>
 <span class="jvol">
21,
 </span>
 <span class="pages">
1037-1053.
 </span>
<p class="reftitl">
The Current State of Two-Dimensional Electrophoresis with
Immobilized pH Gradients
<p class="Para1">
<b>REVIEW</b>.  Sample preparation varies depending on species and tissue.
The various techniques available include grinding in liquid nitrogen-cooled
mortar, sonication, shearing-based methods, or homogenization.  Solubilization
can be in a lysis buffer containing 9 M urea, 1% DTT, 2-4% CHAPS, 2%
3-10 ampholytes, and 10 mM Pefabloc inhibitors, or in 2 M thiourea, 7 M urea,
with CHAPS, ampholytes, DTT, and Pefabloc the same.  Alternatively a
hot SDS sample buffer (1% SDS in 0.1 M TrisHCl pH 7.0) might be used, and
then diluted with 3-fold or more of the lysis buffer or thiourea lysis buffer.
Mouse liver was ground in liq. N<sub>2</sub>-cooled mortar and the powder
suspended in 20% TCA in acetone (&minus;18&deg;) containing 0.2% DTT and
kept overnight at &minus;18&deg;.  The pellet from centrifugation
is resuspended in acetone with 0.2% DTT, and sample spun again.  The pellet
is dried under vacuum and solubilized in a lysis buffer.  This is then
centrifuged 40,000 <i>g</i> 60 min at 15&deg; and can be stored at
&minus;80&deg;.  IEF is done on 3 mm strips either ready-made or prepared.
Sample loading is by cup or in-gel rehydration.
<p>
IPG gel casting is with 4% polyacrylamide (3% C) in 0.5 mm thick sheets
and lengths of 4, 11, 18, and 24 cm on GelBond PAGfilm.  For the pH 9-12
and pH 10-12 range, acrylamide is substituted with DMAA with monomer increased
to 6% T.  Casting is with two glass plates with one sheeted with GelBond
PAGfilm and the other with a 0.5 mm thick U-frame; the U-frame plate
is treated with Repel Silane to prevent gel sticking to the plate.  Immobiline
acidic and basic starter solutions are used to prepare a gradient, and
a gradient mixer is used to load monomer from the top.  Since polymerization
is pH-dependent, the starter solutions are adjusted to pH 7 (NaOH to the
acidic, and acetic acid to the basic);  the acid solution at a heavier
density is placed in the mixing chamber, while basic solution (at a lighter
density) is placed into the reservoir.  The mold for the cast should be
pre-cooled in the refrigerator and the gradient poured, then the mold
is left 15 min at room temp to permit leveling of density gradient prior
to polymerization, where mold is placed in 50&deg; oven for 60 min.
The IPG gel is removed from the mold was washed extensively (6 &times;
10 min each) in deionized water, then kept 30 min in 2% glycerol, and then
dried at room temp in dust-free environment.  Cover with plastic film and
stored at &minus;20&deg;.  Just before use, the IPG gel is cut into 3 mm
strips with a paper cutter.
<p>
Rehydration of strips is done by mixing total cell/tissue lysates
between 0.5-5 mg protein into 0.5 ml of 8 M urea (or 6 M urea + 2 M thiourea),
0.5-4% CHAPS, 0.4% DTT, and 0.5% 3-10 ampholytes.
<p>
After rehydration, strips might be dipped 1 second in distilled water and
blotted between filter paper to remove excess swelling solution.  This
avoids urea crystallization on the gel surface, causing prolonged IEF and
empty vertical lanes on the 2-D gel.
<p>
SDS-PAGE is described in a horizontal and vertical format, without the
use of a stacking gel for the vertical format.
<p>
After the bromophenol blue dye is allowed to migrate out from the SDS gel,
the gel is fixed in 40% ethanol with 10% acetic acid for at least 1 hour
but usually overnight.  Gels for analytical purposes are stained by
silver whereas preparative gels are stained with CBB G-250 (0.1% in the
ethanol/acetic acid mix for 1-3 h.  If SYPRO Ruby staining is used
fix is in 10% MeOH with 7% acetic acid for 30 min prior to staining.
<p>
O'Farrell proposed the first IEF sample buffer of 9 M urea, 4% NP-40,
and 50-100 mM DTT, which has been modified to produce solubilizaition
of different classes of proteins.  Modifications include use of 2-4%
CHAPS, 1% DTT, and 2% ampholytes, and thiourea in a reduced urea solution.
In addition TCA precipitation before solubilization is possible.  In some
cases, SDS sample buffer gets at a wide variety of tough proteins.
<p>
In making an IPG gel, the thickness and width of the strip should be
kept to a minimum since electroendosmotic effects from immobilized
buffer groups appear, causing gel gluing and hindrance of protein transfer
from 1st to 2nd dimension.  Adding urea and glycerol are the only remedy
for this.
<p>
As for IEF sample loading, when using cup loading, apply at either the
anode or cathode, although anode end application appears to give better
results (some exceptions, such as human cardiac proteins).  When using
basic end pH gradients (6-10, 7-10, 6-12) prefer the anodic end.
<p>
For analytical 2DGE using silver staining, 50-100 &micro;g loads are best
for 18 cm long strips that are 3-6 pH units wide.  Either cup-loading
or simple rehydration works, although for preparative work, prefer
gel rehydration.  Up to 1 mg protein in a 100 &micro;l volume may be better
for preparative work.  Up to 10 mg protein might be loaded for even 1 pH
unit narrow-range strips.  Excessive protein loading causes problems in the
2nd dimension more than the 1st, but protein concentration should not exceed
10 mg/ml.
<p>
Note that when the sample has excessive salt, it is best to reduce the
voltage to 50-100 V even for several hours before ramping up the voltage.
When applying large sample volumes, voltage should be limited to 50 V overnight
for improved sample entry.
<p>
The total number of volt-hours to do IEF depends on many factors:
sample load, pH range interval, strip length.  For pH 4-7, 4-9, and 3-10
strips used mostly for analytical purposes (50-100 &micro;g) or
micropreparative (up to 1 mg), the final focusing voltage is much lower
(to 3500 V) and the times are fairly long the narrower the pH range
(20-24 h for 1-1.5 pH units strips, 12 hours for 3 pH wide strips,
10 h for 4 pH wide strips, 8 h for 5-6 pH unit wide strips, 6 h for
7, 8, and 9 pH unit wide strips usually).  If the strip is longer, the
time is also increased (as for 24 cm strips).  If there are greater
protein loading, the time added may be 50% of that above for analytical
separations.
<p>
Electrode wicks may need to be replaced for very long runs, and this
is particularly true when the sample contains high amounts of salt.
<p>
When running narrow or medium-range pH strips in the alkaline range
(typically for ribosomal and nuclear proteins), the acrylamide monomer
should be DMAA and isopropanol may need to be added to the rehydration
solution to avoid severe streaking.
<p>
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Schilling B, Wang W, McMurray JS, Medzihradszky KF
 </span>
 <span class="year">
(1999)
 </span>
 <span class="jname">
Rapid Commun. Mass Spectrom.
 </span>
 <span class="jvol">
13,
 </span>
 <span class="pages">
2174-2179.
 </span>
<p class="reftitl">
Fragmentation and Sequencing of Cyclic Peptides by Matrix-Assisted
Laser Desorption/Ionization Post-Source Decay Mass Spectrometry
<p class="abstract">
<b>ABSTRACT</b>.
A series of synthetic cyclic decapeptides and other smaller cyclic peptides were analyzed using matrixassisted
laser desorption/ionization time-of-flight (MALDI-TOF) mass spectrometry. The investigated
compounds were cyclized in a head-to-tail manner and contained non-proteinaceous amino acids, such as Dphenylalanine,
D,L-4-carboxyphenylalanine, e-aminocaproic acid, and g-aminobutyric acid, and were
synthesized in a program to develop inhibitors of pp60c-src (Src), a tyrosine kinase that is involved in signal
transduction and growth regulation. Post-source decay (PSD) spectra of the cyclic peptides featured
abundant sequence ions. Two preferential ring opening reactions were detected resulting in linear fragment
ions with an N-terminus of proline and a C-terminus of glutamic acid, respectively. MALDI-PSD spectra
even permitted de novo sequencing of some cyclic peptides. Systematic studies on cyclic peptides using this
method of fragmentation have not been reported to date. This work presents an easy mass spectrometric
method, MALDI-PSD, for the characterization and identification of cyclic peptides.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Corthals GL, Wasinger VC, Hochstrasser DF, Sanchez J-C
 </span>
 <span class="year">
(2000)
 </span>
 <span class="jname">
Electrophoresis
 </span>
 <span class="jvol">
21,
 </span>
 <span class="pages">
1104-1115.
 </span>
<p class="reftitl">
The Dynamic Range of Protein Expression:  The Challenge for Proteomic Research
<p class="Para1">
<b>REVIEW</b>.  Genomic information provides good foundation for cross
correlation between transcriptome and proteome-generated data.  A powerful
set of tools to study global and quantitative genome expression include
differential display-PCR, cDNA microarrays, and serial analysis of gene
expression (SAGE).  The proteome itself is a dynamic, spatially and
temporally expressed (nuclear proteins control transcription, and mitochondiral
proteins regulate bioenergetics).  Proteins are expressed in
<q>cluster</q> or <q>modules</q>, usually because they
have functional relatedness such as signal transduction.  Proteins are
also modified in dynamic (phosphorylation) and static (disulfide links) ways.
<p>
Proteomics methods are two-phased.  The first phase is 2-DE separations
with an intermediate analytical step to identify proteins of interest.
The second phase is analytical at the MS.  Raymond and Weintraub (1959)
and Ornstein &amp; Davis (1962) showed that polyacrylamide did better than
starch.  Margolis and Kenrick demonstrated a 2-D separation with a first
dimension disc and then 2nd dimension gradient gel.  O'Farrell and Klose
independently demonstrated IEF combined with SDS-PAGE gave excellent
multidimensional separations.
<p>
There was not much use for 2D gels as a useful tool until the sensitivity
of instruments improved, making 2D gels a good semipreparative method.
Edman sequencing could be done on high picomole amounts (this required
a Western blotting first however.  Other improvements include in-gel
rehydration into the IPG strips, which avoided streaking caused by
poorly soluble proteins.
<p>
Wide-range (&gt; 3 pH units per 20 cm) IPGs present the problem of
spot comigration (protein overcrowding when large loads are done).
Multiple overlapping medium-range (1-3 pH units per 20 cm) IPGs have
been considered as an answer to avoiding overcrowding.  Unfortunately
the number of proteins has not increased compared to wide range.
Narrow-range (&lt; 1 pH unit in a 20 cm distance) have been looked at.
Tonella et al. have found thousands of proteins within a pH 5.1-5.6
narrow-range IPG in <i>E. coli</i>, based on a genomic analysis of
804 ORFs with calculated pI 5.09-5.59, predicted to produce 1029 polypeptides
(based on a 1.28 peptides per ORF determination).  3.2 mg protein from
lysate was placed and Ag-stained to show 949 spots.  With 7% being
fragments, that meant 878 proteins, or about 85% of predicted for the range.
That is excellent coverage.
<p>
The yeast genome has 6139 ORFs.  IPGs (25 cm) were poured ranging from
pH 4.5-5.5 (1 cm = 0.04 pH units).  These were cut to size (~18 cm)
to feature a range of 4.8-5.5.  Predicted ORFs with pI values in that
range was 852; with 200 post-translational modifications possible for this
subset, more spots are possible.  500 &micro;g total yeast lysate was
loaded on to the narrow-range IPG and visualized with Ag.  The Melanie
Bio-Rad package found 2700 spots (!) in which 852 were expected.
<p>
50 of the most intensely Ag-stained spots from a 4 &times; 4 cm section were
selected and 39 identified as unique.  58 ORFs were predicted for the range,
and 14 of the 39 predicted to fall within the pI range.  Such failure to
predict could be due to unanticipated posttranslational modifications.
Additional gel loading does not help to illuminate proteins present
in smaller amounts, as comigration and spot fusion occurs.  In this
case prefractionation or other purification might make additional loading
possible.
<p>
Developments that advanced the used of ESI and MALDI-MS include use
of microscale capillary RP-HPLC coupled to ESI; development of
algorithms that correlated MS and MS/MS data to databases; better software
and interfaces to instrument control; use of post-source decay (PSD) in
MALDI-ToF permitting sequencing; cleanup of small 1-5 &micro;l samples
for ESI and MALDI; ability to use low ESI flow rates (10-25 nl/min) or
nanoelectrospray; liquid extraction of peptides from in-gel proteolytic
digestion; and simultaneous digestion and electrotransfer to PVDF for
protein scanning.
<p>
For a comprehensive proteome analysis, at least 1 billion cells
are necessary.  This assumes 100% protein recovery and that 1000 copies
of a protein are present per cell to produce 1.6 pmol protein.
This is 4, 8, and 16 ng of 25, 50, and 100 kDa proteins, resp.
Only a &micro;LC-MS/MS or MALDI-ToF MS can analysis this low.  If
only 10<sup>6</sup> cells are present, only the most highly abundants
are visualizable and amenable to MS.  Staining is important too.
The limits of Coomassie are at 1 &micro;g, while for silver are 1 ng;
radionuclides can visualize only smaller proteins present (for example
at only 10 copies per cell with 1 billion cells, radioisotopes are necessary).
<p>
[The paper includes a most interesting Table 1 that shows the limits of
detection and analysis and the amount of cells that must be used for
predicted copies of proteins.]
<p>
2-DE prefractionantion is best done in solution to minimize loss of protein
and minimize surface interaction.  Liquid phase IEF is efficient and when
done with carrier ampholytes and compatible compounds, transfer to IPGs
is possible.  <b>Recycling IEF</b> is where sample is re-introduced
into a focusing cell.  Screen elements prevent convective mixing.  The
Bio-Rad Rotofor and Dr. Weber Octopus can be used to prefractionate.
<p>
The multicompartment electrolyser is a form of IEF and descends from Bier's
RIEF, and IEF in gel-layer approach.  Here the protein is kept apart from
the IPG matrix allowed to flow freely.  Isoelectric membranes instead
compartmentalize or stabilize a pI region.  The membranes have IPG
technology and proteins flow through.  The Hoefer IsoPrime system works
using this technology.  The Gradipore Gradiflow system is a multifunctional
electrokinetic membrane apparatus that processes and purifies in a number
of ways:  pI, mobility, size, and affinity.  A thin polyacrylamide membrane
on a support separates two fractions.  It can prefractionate on a relatively
large (3 ml to hundreds of mL).
<p>
Three different chromatographic techniques can be used to prepurify
proteins:  elution, frontal, and displacement.  Step-gradient elution
techniques are the most common.  These include IEC, HIC, RP, and affinity.
Size-exclusion has been replaced by membrane ultrafiltration.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Gygi SP, Corthals GL, Zhang Y, Rochon Y, Aebersold R
 </span>
 <span class="year">
(2000)
 </span>
 <span class="jname">
Proc. Natl. Acad. Sci. USA
 </span>
 <span class="jvol">
97,
 </span>
 <span class="pages">
9390-9395.
 </span>
<p class="reftitl">
Evaluation of Two-Dimensional Gel Electrophoresis-Based Proteome Analysis
Technology
<p class="Para1">
A typical 2D gel with IEF in the pH 3-10 range and 50 &micro;g load produced
1000 Ag-stained proteins.  If the pH range is narrowed from 7 to 1 pH unit
over a 20 cm rang and load increased to 500 &micro;, more than 1500 Ag-stained
spots are seen.  If seven 2D gels are run each with a 1 pH range,
the expectation is that 10,000 proteins from a cell lysate can be seen by
extrapolation.  This surpasses the 6139 proteins predicted from the genome.
An arbitrary 4 cm<sup>2</sup> region was taken from the gel and 50 spots
in the area identified by MS.  MS/MS spectra were searched using SEQUEST
against the yeast database.
<p>
Codon bias represents the preference of one codon among many in specifying
an amino acid.  The more highly expressed proteins have large codon biases
(&gt;0.2).  Plots of the number of genes against the codon bias were
made for the entire yeast genome, the genes predicted in the area that
was cutout, those genes actually identified, and those both predicted and
identified.  For the entire genome, the largest number of genes have a
bias values of 0.0 and 0.1.  For the cutout area 58 genes are predicted
and have a similar distribution.  The codon bias distribution (CBD) was
entirely different for the genes identified however, shifted well above
0.1 (no proteins had a bias value &lt; 0.1, although more than half of
genes fall into this range).  The CBD of genes predicted and found
(14 of the 58) also showed a similar distribution.  Proteins of lesser
abundance were not detected and there was a larger number of proteins
migrating to the selected region than were predicted.
<p>
While the products of a single gene can be separated on a gel, the products
of different genes can be found to comigrate to the same spot.  This difference
and identity in migration can complicated the comparative quantitative
pattern in 2D gel databases.
<p>
In order to detect low abundance proteins (those with &lt;0.1 codon bias
values), 50 mg of protein was loaded on to 10% SDS-PAGE 15 &times; 12 &times;
1.5 cm gel run at 40 mA constant and stained with colloidal Coomassie
Blue.  A 3 &times; 100 mm in the 68-85 kDa range was excised, the strip
diced into 1 mm<sup>3</sup> pieces and in-gel trypsin digestion done.
Extracted peptides were lyophilized and resuspended in 5 mM potassium
phosphate/25% MeCN pH 3.0, loaded on to polysulfoethyl A (2.1 &times; 250 mm)
and eluted with buffer containing 0.25 M KCl over a 60 min gradient at
0.2 ml/min.  Off the SCX column, selected fractions were run on
RP-microcapillary LC-MS/MS.  193 proteins were detected in the 68-85 kDa
range (more than 1/3 of proteins predicted in this range).  The CBD was
of this set was similar (not identical) to the CBD for the entire genome,
although many more proteins with bias value &lt; 0.1 were detected this
time.  Low abundance protein range from 10-10<sup>3</sup> copies per cell,
whereas high abundance are 10<sup>4</sup>-10<sup>6</sup> copies per cell.
With standard 2DE gels with a 40 &micro;g load of whole lysate, the average
abundance is 51,200 copies per cell and no proteins with &lt; 1000 copies/cell.
At 500 &micro;g, proteins at 1000 copies/cell can be seen by Ag staining,
but less than that are not detected.  Pre-fractionation in milligram
amounts is required to enrich loads for 2DE is essential;  this compromises
any ability to quantify amounts accurately however.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Santoni V, Molloy M, Rabilloud T
 </span>
 <span class="year">
(2000)
 </span>
 <span class="jname">
Electrophoresis
 </span>
 <span class="jvol">
21,
 </span>
 <span class="pages">
1054-1070.
 </span>
<p class="reftitl">
Membrane Proteins and Proteomics:  Un Amour Impossible?
<p class="Para1">
<b>REVIEW</b>.  Membrane proteins serve functions in signal transduction,
cell adhesion, metabolite and ion transport, endocytosis, etc.
From the sequencing of several genomes it is estimated that transmembrane
proteins represent 30% of the total proteins.   Proteins can be
<q>associated</q> with a membrane, but there is a great deal
of argument about the word <q>associated.</q>  Typically
intrinsic or <i>integral</i> membrane proteins are those that span
the lipid bilayer a number of times; this association is with doubt.
The polypeptide chain must organize into an &alpha;-helix or &beta;-sheet
in order to span the membrane.
<p>
&alpha;-helical proteins typically have long sequences of mostly
nonpolar amino acids, the 15-25 required to span the bilayer.
Their hydropathy values are quite positive (making them hydrophobic).
In <i>E. coli</i> the count of &alpha;-helical transmembrane segments
ranges from 3 for NADH dehydrogenase component NuoA to up to 15
for cytochrome <i>o</i> ubiquinol oxidase subunit-1, CyoB.  380 open
reading frames encode these types of proteins and the hydropathy is
skewed toward hydrophobicity.
<p>
Gram-negative bacteria possess a bilayer membrane external to the
cytoplasmic membrane.  This membrane is rich in integral proteins, but
these are not &alpha;-helical but are usually &beta;-sheet.  The tertiary
structure is usually a &beta;-barrel, forming a pore that allows
certain molecules past the outer membrane.  These transmembrane domains
form by alternating polar and nonpolar amino acids.  Their hydropathy
profiles show much lower hydrophobicity.
<p>
Proteins also become associated with membranes through post-translational
modifications that include myristylation or polyisoprenylation.  The
latter modification occurs for G proteins.  Other associations are achieved
with glycolipid anchors such as glycosylphosphatidyl inositol (eukaryotes)
or <i>N</i>-acyl diglyceride to an <i>N</i>-terminal cysteine (typical
for bacterial lipoproteins).  Acylation can also occur to simple
fatty acid chains a part of the membrane.
<p>
With integral membrane proteins, there is no option to be associated or
not associated.  These are embedded and removal from the membrane ends
the function, because denaturation is usually required.  Other associations
are provided by these post-translational modifications that cause partitioning
between membrane and membrane-free phase.  Yet another is caused by
protein-protein interaction:  for instance, ubiquinol-cytochrome c
oxidoreductase core proteins 1 and 2 do not interact with the mitochondrial
inner membrane, but do with the proteins that are embedded (cytochrome <i>b</i>).
<p>
In order to separate membrane proteins from soluble proteins, techniques
such as salt washing and differential centrifugation are used.  Use
of chaotropic salts (such as KBr) strip proteins clinging to membranes
away from them if they are not truly embedded.  Use of sodium carbonate
makes of pH in order to achieve the removal.
<p>
Bordier made use of two-phase detergents to achieve membrane protein
purifications.  Oligooxyethylene detergents (Tritons) show two transition
temperatures in aqueous solution: one is a lower transition temperature
in which they separate into a pure phase out of water, and an upper
transition temperature in which a phase separation occurs between
a detergent-rich and -poor phase, and compositions are constant at a given
temperature.  Triton X-114 has an upper transition of 20&deg; (which is
60&deg; for Triton X-100).  If the ionic strength is raised, the transition
temperature is lowered the higher salt concentration promotes the detergent
transition.  For example, in 3 M NaCl, Triton X-100 critical temperature
goes from 60&deg; to 20&deg;.  The problem is that at higher ionic strength,
proteins are driven into the detergent phase;  50% of total cell proteins
move into the detergent phase at 1.2 M ammonium sulfate.
<p>
<b>Animal cells/tissues</b>.  When 2DE was first used, there was difficulty
in preparing membrane proteins.  When trying to solubilize erythrocyte
membranes, Band III is not amenable to classical urea-nonionic detergents.
In a subsequent paper integral membranes were separated by 2DE.
Histocompatibility antigens prepared from radiolabeled cultured cells
were immunopptd and the ppt. analyzed by 2DE, with spots at the expected
pI and <i>M</i><sub>r</sub>.  Postion of the protein was important in
establishing that the separated proteins are proteins co-immunoprecipitated
with the true antigen.
<p>
In some cases in order to characterize membrane proteins on 2DE partial
purification may be necessary.  A glucose transporter could be separated
on 2DE in purified but not total cell extracts.  The fact that hydrophobic
proteins are poorly represented in the 2DE protein sets could be due to
their belonging to low abundance proteins, but it could also be due to
poor extraction using classical urea-detergent measures.
<p>
<b>Yeast/Plant Cells/Tissues</b>.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Merchant M, Weinberger SR
 </span>
 <span class="year">
(2000)
 </span>
 <span class="jname">
Electrophoresis
 </span>
 <span class="jvol">
21,
 </span>
 <span class="pages">
1164-1167.
 </span>
<p class="reftitl">
Recent Advancements in Surface-Enhanced Laser Desorption/Ionization
Time-of-Flight Mass Spectrometry
<p class="abstract">
<b>REVIEW</b>.
<p>
<b>ABSTRACT</b>.
The overall history and recent advances in surface enhanced laser
desorption/ionization-time of flight-mass spectrometry (SELDI-TOF-MS)
technology is reviewed herein. Fundamentals of SELDI-TOF analysis are
presented while drawing comparisons with other laser-based mass
spectrometry techniques. The application of SELDI-TOF-MS to functional
genomics and biomarker discovery is discussed and exemplified by
elucidating a biomarker candidate for prostatic carcinoma. Finally,
a short discussion regarding future SELDI requirements and developments
is supplied.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Kinumi T, Niwa H, Matsumoto H
 </span>
 <span class="year">
(2000)
 </span>
 <span class="jname">
Anal. Biochem.
 </span>
 <span class="jvol">
277,
 </span>
 <span class="pages">
177-186.
 </span>
<p class="reftitl">
Phosphopeptide Sequencing by In-Source Decay Spectrum in Delayed Extraction
Matrix-Assisted Laser Desorption Ionization Time-of-Flight Mass Spectrometry
<p class="abstract">
<b>ABSTRACT</b>.
Protein phosphorylation underlies numerous cellular
signaling processes. Since a reliable prediction of
phosphorylation site(s) based on a consensus amino
acid sequence is rather difficult to date, determination
of phosphorylation site(s) in phosphoproteins is a crucial
step toward the understanding of their function at
the molecular level. A conventional protocol for the
determination of phosphorylation sites utilizes radioactive
labeling of a phosphoprotein by 32P and purification
of digested peptides carrying radioactivity, followed
by Edman degradation. This method is not only
tedious, but also indirect because the evidence will be
based on disappearance of a phenylthiohydantoin signal
from the degradation cycle where the 32P radioactivity
is eluted. Several methodologies have been developed
to determine the phosphorylation sites
directly by using mass spectrometry. These include
collision-induced dissociation (CID) and post-source
decay (PSD), both of which tend to produce fragment
ions less efficiently as the number of residues exceeds
20. Moreover, in both decay processes, there is a tendency
for the phosphate group to be removed during
the breakdown of the main peptide chain. We report a
method that allows direct observation of phosphorylated
peptide fragments of phosphopeptides exceeding
20 residues by using an in-source decay fragmentation
by matrix-assisted laser desorption ionization
time-of-flight mass spectrometry, yielding results
which are difficult or impossible to obtain by existing methods using CID or PSD.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Wittmack K, Szymczak W, Hoheisel G, Tuszynski W
 </span>
 <span class="year">
(2000)
 </span>
 <span class="jname">
J. Am Soc. Mass Spectrom.
 </span>
 <span class="jvol">
11,
 </span>
 <span class="pages">
553-563.
 </span>
<p class="reftitl">
Time-of-Flight Secondary Ion Mass Spectrometry of Matrix-Diluted
Oligo- and Polypeptides Bombarded with Slow and Fast Projectiles:
Positive and Negative Matrix and Analyte Ion Yields, Background Signals,
and Sample Aging
<p class="abstract">
<b>ABSTRACT</b>.
Human angiotensin II, chain B of bovine insulin, and porcine insulin were determined by
time-of-flight secondary ion mass spectrometry under impact of ;25 keV Xe1 and SF5
1 ion
beams and ;100 MeV 252Cf fission fragments. Matrix-embedded samples, dissolved in a large
surplus of a-cyano-4-hydroxycinnamic acid, were prepared by nebulizer spray deposition,
neat samples by the droplet technique. It is shown that the status of the sample can be assessed
by evaluating the matrix-specific features of the mass spectra. The beneficial effect of matrix
isolation was small for angiotensin but large for the insulin samples, which did not show
parent peaks from neat material. Negative ion yields under SF5
1 impact were up to a factor of
50 higher than with Xe1. For positive secondary ions, the enhancement was much smaller. The
mass spectra produced by slow ion beams or fast fission fragments were qualitatively similar.
Quantitative differences include the following: with fast projectiles the yields were about
1030 times higher than with slow ions, but similar for negative ion emission under SF5
1
bombardment; the analyte-to-matrix yield ratios were higher with slow ions and up to 250
times higher than the molar analyte concentration; for analyte ions the peak-to-background
ratios were higher using slow projectiles; the fraction of carbon-rich collisionally formed
molecular ions was much higher with fast projectiles. Sample aging in vacuum for up to five
weeks strongly reduced the yield of protonated analyte molecules ejected by slow ion impact,
but not of deprotonated species. Hence protonation seems to correlate with sample wetness
or the presence of volatile proton-donating additives.

<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Gevaert K, Vandekerckhove J
 </span>
 <span class="year">
(2000)
 </span>
 <span class="jname">
Electrophoresis
 </span>
 <span class="jvol">
21,
 </span>
 <span class="pages">
1145-1154.
 </span>
<p class="reftitl">
Protein Identification Methods in Proteomics
<p class="abstract">
<b>ABSTRACT</b>.
<b>REVIEW</b>. A combination of high-resolution two-dimensional (2-D)
polyacrylamide gel electrophoresis, highly sensitive biological mass
spectrometry, and the rapidly growing protein and DNA databases has
paved the way for high-throughput proteomics. This review concentrates
on protein identification. We first discuss the use of protein
electroblotting and Edman sequencing as tools for de novo sequencing
and protein identification. In the second part, we highlight
matrix-assisted laser desorption/ionization mass spectrometry
(MALDI-MS) as one of the main contemporary analytical methods
for linking gel-separated proteins to entries in sequence databases.
In this context we describe the two main MALDI-MS-based identification
methods: (i) peptide mass fingerprinting, and (ii) post-source decay
(PSD) analysis. In the last part, we briefly emphasize the importance
of sample preparation for obtaining highly sensitive and high-quality
MALDI-MS spectra.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Breen EJ, Hopwood FG, Williams KL, Wilkins MR
 </span>
 <span class="year">
(2000)
 </span>
 <span class="jname">
Electrophoresis
 </span>
 <span class="jvol">
21,
 </span>
 <span class="pages">
2243-2251.
 </span>
<p class="reftitl">
Automatic Poisson Peak Harvesting for High Throughput Protein Identifcation
<p class="abstract">
<b>ABSTRACT</b>.
High throughput identification of proteins by peptide mass fingerprinting
requires an efficient means of picking peaks from mass spectra. Here,
we report the development of a peak harvester to automatically pick
monoisotopic peaks from spectra generated on matrix-assisted laser
desorption/ionisation time of flight (MALDI-TOF) mass spectrometers.
The peak harvester uses advanced mathematical morphology and watershed
algorithms to first process spectra to stick representations.
Subsequently, Poisson modelling is applied to determine which peak
in an isotopically resolved group represents the monoisotopic mass
of a peptide. We illustrate the features of the peak harvester with
mass spectra of standard peptides, digests of gel-separated bovine
serum albumin, and with Escherictia coli proteins prepared by
two-dimensional polyacrylamide gel electrophoresis. In all cases,
the peak harvester proved effective in its ability to pick similar
monoisotopic peaks as an experienced human operator, and also proved
effective in the identification of monoisotopic masses in cases
where isotopic distributions of peptides were overlapping. The
peak harvester can be operated in an interactive mode, or can be
completely automated and linked through to peptide mass fingerprinting
protein identification tools to achieve high throughput automated
protein identification.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Keough T, Lacey MP, Fieno AM, Grant RA, Sun Y, Bauer MD, Begley KB
 </span>
 <span class="year">
(2000)
 </span>
 <span class="jname">
Electrophoresis
 </span>
 <span class="jvol">
21,
 </span>
 <span class="pages">
2252-2265.
 </span>
<p class="reftitl">
Tandem Mass Spectrometry Methods for Definitive Protein Identification
in Proteomics Research
<p class="abstract">
<b>ABSTRACT</b>.
Optimized procedures have been developed for the addition of
sulfonic acid groups to the N-termini of low-level peptides.
These procedures have been applied to peptides produced by tryptic
digestion of proteins that have been separated by two-dimensional
(2-D) gel electrophoresis. The derivatized peptides were sequenced
using matrix-assisted laser desorption/ionization (MALDI)
post-source decay (PSD) and electrospray ionization-tandem mass
spectrometry methods. Reliable PSD sequencing results have been
obtained starting with sub-picomole quantities of protein. We
estimate that the current PSD sequencing limit is about 300
fmol of protein in the gel. The PSD mass spectra of the
derivatized peptides usually allow much more specific protein
sequence database searches than those obtained without derivatization.
We also report initial automated electrospray ionization-tandem mass
spectrometry sequencing of these novel peptide derivatives. Both
types of tandem mass spectra provide predictable fragmentation
patterns for arginine-terminated peptides. The spectra are
easily interpreted de novo, and they facilitate error-tolerant
identification of proteins whose sequences have been entered
into databases.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Karas M, Bahr U, D&uuml;lcks T
 </span>
 <span class="year">
(2000)
 </span>
 <span class="jname">
Fresnius J. Anal. Chem.
 </span>
 <span class="jvol">
366,
 </span>
 <span class="pages">
669-676.
 </span>
<p class="reftitl">
Nano-electrospray Ionization Mass Spectrometry: Addressing Analytical
Problems Beyond Routine
<p class="abstract">
<b>REVIEW</b>.
<p>
<b>ABSTRACT</b>.
The advent of nano-electrospray ionization
(nano-ESI) has considerably extended the usability of ESI
in the analytical mass spectrometric laboratory. One of the
remarkable features of nano-ESI is its extremely low sample
consumption. Only a few microliters of analyte solution
(105108 M) are sufficient for molecular weight determination
and structural investigations by MS/MS. But
nano-ESI is more than just a minimized-flow ESI; the low
solvent flow rate also affects the mechanism of ion formation.
As a consequence, the area of ESI-MS applications
is significantly enhanced. Oligosaccharides, glycosides
as well as glycoproteins can be analyzed more easily
than with normal ion spray. The same holds for the
analysis of non-covalent complexes sprayed directly from aqueous solutions.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Dalluge JJ
 </span>
 <span class="year">
(2000)
 </span>
 <span class="jname">
Fresnius J. Anal. Chem.
 </span>
 <span class="jvol">
366,
 </span>
 <span class="pages">
701-711.
 </span>
<p class="reftitl">
Mass Spectrometry for Direct Determination of Proteins in Cells:
Applications In Biotechnology and Microbiology
<p class="abstract">
<b>REVIEW</b>.
<p>
<b>ABSTRACT</b>.
A number of different procedures have been
developed in recent years that utilize mass spectrometry
for the direct determination of proteins in complex mixtures
of biological origin. Specific examples of these include
the use of matrix-assisted laser desorption/ionization
mass spectrometry (MALDI-MS) or directly combined
liquid chromatography-electrospray ionization mass spectrometry
(LC/ESI-MS) for rapid profiling of protein expression
in bacterial and eucaryotic cells and cell-free extracts.
Approaches to sample cleanup, contaminant removal,
and initial separation of analytes on-line for the direct
determination of proteins in cells using MALDI- and
ESI-MS are discussed. Advantages of these techniques
over traditional biochemical methods are highlighted, and
a critical review of their utility and potential as standard
tools in the biomolecular and microbiological research
laboratory is presented.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Gygi SP, Rist B, Aebersold R
 </span>
 <span class="year">
(2000)
 </span>
 <span class="jname">
Curr. Opin. Biotechnol.
 </span>
 <span class="jvol">
11,
 </span>
 <span class="pages">
396-401.
 </span>
<p class="reftitl">
Measuring Gene Expression by Quantitative Proteome Analysis
<p class="abstract">
<b>REVIEW</b>.
<p>
<b>ABSTRACT</b>.
Proteome analysis is most commonly accomplished by the combination
of two-dimensional gel electrophoresis for protein separation,
visualization, and quantification and mass spectrometry for
protein identification. Over the past year, exceptional progress
has been made towards developing a new technology base for the
precise quantification and identification of proteins in complex
mixtures, that is, quantitative proteomics.
<p class="Para1">
The intention of 2DE-MS is to provide a global proteome analysis, but
closer examination reveals it is not truly global.  Many protein classes
are excluded or underrepresented in the 2D gel patterns.  Very acidic
or basic proteins, extremely large or small, and membrane proteins
are not typically represented.  Pre-gel enrichment is usually necessary
to look at low abundance proteins, for example, as shown in a study of
codon bias values.
<p>
Codon bias values generally correlate with gene expression.  Highly
expressed proteins generally have a large codon bias value.  61 codons
code for 20 amino acids.  Low abundance proteins (transcription factors,
protein kinases) typically have codon bias values &lt; 0.1.  Of the
6139 genes in the known yeast genome, one half of the gens have values
&lt; 0.2 and more than 2500 gene are in the range of 0.0 to 0.1.
In 2D gel studies of 343 detected proteins, most of these have high
codon bias values.
<p>
Alternative to classical 2DE have been sought.   Andrews and co-workers
used a technique first to separate proteins by IEF then analyze to
create <q>virtual</q> 2D gels.  In one method prep HPLC
is used to fractionate whole yeast lysate prior to SDS-PAGE.
<p>
In another method protein mixtures are digested into even more complex
peptide mixtures and analyzed by LC-MS/MS.  Co-eluting peptides can
be filtered in the MS one at a time and fragmented and their fragmentation
patterns studied for sequence information.  The count of proteins that
can be identified is only constrained by the lengh of the analysis and
the complexity of the mixtures.	Others have increased the number of
co-eluting peptides by doing something called <q>peak parking</q>
in which the flow in the capillary or nano-colums is reduced from
200 nl/min to 20 nl/min which extends elution time for particular peaks.
Up to 10,000 sequencing attempts can be made in a single run.
<p>
Link et al. have reported the use of DAPLC (direct analysis of protein
complexes) in which they separate proteins by charge and then hydrophobicity
before MS, a multidimensional chromatographic technique.  The mixture
is denatured and reduced and then digested to peptides.  The peptides
are acidified and passed over a SCX column, and then discrete fractions
eluted to an RP column with a salt step gradient.  Peptides are then
analyzed by LC-MS/MS.  This successfully identified all proteins in the
80S yeast ribosome.  This 2D technique gets 4 times more proteins than
a 1D technique.
<p>
Using a preparative SDS-PAGE technique, a strip of the gel containing
proteins in the region 68-85 kDa were removed from the Coomassie-stained
gel and digested with trypsin.  Peptides were then separated and analyzed
by gradient SCX and then by capillary RP-LC-MS/MS.  193 proteins were
identified and 63 had a codon bias &lt; 0.1.
<p>
One difficulty with directly LC-MS/MS analysis is a complete loss of the
ability to quantify levels of protein.  In 2DE-MS/MS quantification is
at the level of the 2D gel.  New methods make use of <b>stable isotope
dilution</b> techniques.  Stable heavy isotopes (<sup>2</sup>H,
<sup>13</sup>C, <sup>15</sup>N, etc.) are used because the ionization
efficiency is quite variable among the different peptides of a protein,
and thus alone present a confusion of the expression.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Gygi SP, Aebersold R
 </span>
 <span class="year">
(2000)
 </span>
 <span class="jname">
Curr. Opin. Chem.  Biol.
 </span>
 <span class="jvol">
4,
 </span>
 <span class="pages">
489-494.
 </span>
<p class="reftitl">
Mass Spectrometry and Proteomics
<p class="abstract">
<b>REVIEW</b>.
<p>
<b>ABSTRACT</b>.
Proteomics is the systematic analysis of the proteins expressed
by a cell or tissue, and mass spectrometry is its essential
analytical tool. In the past two years, incremental advances in
standard proteome technology have increased the speed of protein
identification with higher levels of automation and sensitivity.
Furthermore, new approaches have provided landmark advances in
determining functionally relevant properties of proteins,
including their quantity and involvement within protein complexes.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Gygi SP, Aebersold R
 </span>
 <span class="year">
(2000)
 </span>
 <span class="jname">
Ann. N. Y. Acad. Sci.
 </span>
 <span class="jvol">
919,
 </span>
 <span class="pages">
33-47.
 </span>
<p class="reftitl">
Quantitative Proteome Analysis:  Methods and Applications
<p class="abstract">
<b>REVIEW</b>.
<p>
<b>ABSTRACT</b>.
With the completion of a rapidly increasing number of complete
genomic sequences, much attention is currently focused on how
the information contained in sequence databases might be
interpreted in terms of the structure, function, and control
of biological systems. Quantitative proteome analysis,
the global analysis of protein expression, has been proposed
as a method to study steady-state gene expression and
perturbation-induced changes. Here, we discuss the rationale
for quantitative proteome analysis, highlight the limitations
in the current standard technology, and introduce a new
experimental approach to quantitative proteome analysis.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Airs RL, Keely BJ
 </span>
 <span class="year">
(2000)
 </span>
 <span class="jname">
Rapid Commun. Mass Spectrom.
 </span>
 <span class="jvol">
14,
 </span>
 <span class="pages">
125-128.
 </span>
<p class="reftitl">
A Novel Approach for Sensitivity Enhancement in Atmospheric Pressure
Chemical Ionization Liquid Chromatography/Mass Spectrometry of Chlorophylls
<p class="Para1">
Pure chlorophyll <i>a</i> was used to prepare pheophytin <i>a</i> by shaking
a 1 &micro;M solution in Et-O-Et (30 ml) with 2 M HCl (15 ml).  The organic
layer is washed 5 times in water, then dried in rotovap.  Spectrophotometric
scans are obtained with 1 cm cell using 2 nm slit a 300 nm/min, and
concentration determined using &epsilon;<sub>662</sub> = 81300 for
chl <i>a</i> and &epsilon;<sub>662</sub> = 46000 for pheo <i>a</i>.
Spinach leaves (20 g) are blanched in boiling salted water for 20-30 s.
These are then macerated in 50 ml HPLC acetone.  The acetone is centrifuged
at 2000 <i>g</i> for 5 min.  Supernatant is passed twice through
solvent-pretreated cotton wool, then dried by rotovap.  HPLC was done
on two 3 &micro;m 4.6 &times; 150 mm Waters Spherisorb ODS2 column in series.
The gradient elution is published.  For LC-MS, a Thermo Finnigan system is
used with AS3000 autosampler, ScM degasser, P4000 gradient pump, UV2000
detector.  The Finnigan MAT LCQ ion trap with APcI source is used.
Capillary temp is 150&deg;, APCI vaporizer temp 450&deg;, discharge current
is 5 &micro;A.
<p>
In the method, 5 &micro;l of a 11 &micro;M chl <i>a</i> solution is injected
to the spray at 300 &micro;l/min, using solvent composition expected in
the gradient program, in order to optimize spectral data.  The major ion
is the [M-H]<sup>+</sup> form.  Peak area measurements show that chl <i>a</i>
had the weaker ionization at 15% of that of pheo <i>a</i>.  The weaker
chl <i>a</i> response might be due to Mg<sup>2+</sup> presence.  APCI
likely adds protons to the molecule.  Protonation of porphyrin ring nitrogens
would be disfavored since non-bonding electron pairs on a Mg-associated
nitrogens contribute to the conjugated &pi; electron system (16 members with
18 electrons).  The lone pairs do not contribute to this in pheo <i>a</i>
and so more readily protonate.  Conversion of chlorophylls to pheophytins
would achieve better ionization, except when analyses involve determination
of mixtures.  The latter problem is circumvented with on-line demetallation
systems:  use of formic acid introduced into solvent flow at 5 &micro;l/min
would remove Mg, showing a <i>m/z</i> 893 shift to <i>m/z</i> 871 and
increase in relative intensity.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Airs RL, Jie C, Keely BJ
 </span>
 <span class="year">
(2000)
 </span>
 <span class="jname">
Org. Geochem.
 </span>
 <span class="jvol">
31,
 </span>
 <span class="pages">
1253-1256.
 </span>
<p class="reftitl">
A Novel Sedimentary Chlorin: Structural Evidence for A Chlorophyll Origin
for Aetioporphyrins
<p class="Para1">
<b>NOTE</b>.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Wildgruber R, Harder A, Obermaier C, Boguth G, Weiss W, Fey SJ, Larsen PM,
G&ouml;rg A
 </span>
 <span class="year">
(2000)
 </span>
 <span class="jname">
Electrophoresis
 </span>
 <span class="jvol">
21,
 </span>
 <span class="pages">
2610-2616.
 </span>
<p class="reftitl">
Towards Higher Resolution:  Two-Dimensional Electrophoresis of
<i>Saccharomyces cerevisiae</i> Proteins Using Overlapping Narrow
Immobilized pH Gradients
<p class="Para1">
The following conditions were used for both broad and narrow range
IEF IPG strips:

<table>
<tr><th>IPG Strips
 <td>Broad range:  pH 3-10L, 4-7, 6-9<br>
     Narrow range:  pH 3.5-4.5, 4-5, 4.5-5.5, 5-6, 5.5-6.7
<tr><th>Gel length           <td>18 cm
<tr><th>Temperature          <td>20&deg;
<tr><th>Current              <td>50 &micro;A per strip
<tr><th>Power                <td>5 W max.
<tr><th>Sample Volume        <td>20-140 &micro;l by cup loading
<tr><th>Focusing
  <td>Broad range:  1 h @ 150 V, 1 h @ 300 V, 1 h @ 600 V, to 3500 V<br>
      Narrow range:  12 h @ 50 V, 1 h @ 300 V, 1 h @ 600 V, to 3500 V<br>
		pH 3-10:  18,000 Vh<br>
		pH 4-7:   45,000 Vh<br>
		pH 6-9:   45,000 Vh<br>
		Narrow range: 70,000 Vh
</table>
<p>
A Multiphor II chamber with dry strip kit and gradient mixer was used with
GelBond PAGfilm.  Haploid yeast wild-type FY1679 of <i>S. cerevisiae</i>
were grown 30&deg; in synthetic YNB medium (6.7 g/L) with 20 g/L glucose
and buffered with NaOH/succinic acid (6 g/L, 10 g/L) to pH 5.8.  When cells
reached OD 1.0, they were washed in ice-cold deionized water and spun down
at 5000 <i>g</i> in 5 min.  Pellets were put in 1.5 ml conical tubes and stored
at &minus;80&deg; unless used immediately.  200 &micro;l hot (95&deg;)
SDS sample buffer (1% SDS in 100 mM Tris HCl pH 8.5) was added to pellet
and sonicated 15 &times; 1 sec.  Sample then boiled 5 min, cooled in ice-bath.
500 &micro;l 2 M thiourea, 7 M urea, 4% CHAPS, 1% DTT, and 2% ampholytes 3-10
was added.  This is re-spun at 16,000 <i>g</i> for 5 min.
<p>
IPG gels with many pH ranges narrow and broad were cast (listed in table).
These were 250 &times; 180 &times; 0.5 mm in size.  Strips were cut
and rehydration was in 6 M urea, 2 M thiourea, 0.5% CHAPS, 0.4% DTT and
0.5% pH 3-10 Pharmalytes.  Samples were loaded by cups on the anode side
with total 40 (broad range) or 140 (narrow-range) &micro;l volumes, all
concentrations at 1 &micro;g/&micro;l.  Except for 4-7 and 3-10 strips,
all IEF was under silicone oil.  Strips are soaked in 10 ml, 15 min each,
of equlibration buffer:  50 mM Tris pH 8.8, 6 M urea, 30% (w/v) glycerol,
2% SDS, with 1% DTT in the first solution and 4% iodoacetamide in the 2nd.
13% polyacrylamide gels were cast (210 &times; 185 &times; 1 mm).  These
were silver-stained by an older method.
<p>
The study shows that the narrow-range gels produced many more resolvable
proteins than the broad range gels.  With the pH 3-10 strips only 755 spots
were detectable in the pH 4-7 range.  In the pH 4-7 gel, this became 1071
using identical spot detection parameters.  When the narrow 4-5, 5-6,
and 5.5-6.7 strips are combined, a total of 1754 proteins are resolved.
This increase is from 1.6 to 2.3 times from the pH 3-10 strip.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Santoni V, Kieffer S, Desclaux D, Masson F, Rabilloud T
 </span>
 <span class="year">
(2000)
 </span>
 <span class="jname">
Electrophoresis
 </span>
 <span class="jvol">
21,
 </span>
 <span class="pages">
3329-3344.
 </span>
<p class="reftitl">
Membrane Proteomics:  Use of Additive Main Effects with Multiplicative
Interaction Model to Classify Plasma Membrane Proteins According to Their
Solubility and Electrophoretic Properties
<p class="abstract">
<b>ABSTRACT</b>.
Recent efforts at the proteomic level were employed to describe
the protein equipment of the plasma membrane of the model plant
Arabidopsis thaliana. These studies had revealed that the plasma
membrane is rich in extrinsic proteins but came up against two major
problems: (i) few hydrophobic proteins were recovered in
two-dimensional electrophoresis gels, and (ii) many plasma
membrane proteins had no known function or were unknown in the
database despite extensive sequencing of the Arabidopsis genome.
In this paper, several methods expected to enrich a membrane sample
in hydrophobic proteins were compared. The optimization of
solubilization procedures revealed that the detergent to be
used depends on the lipid content of the sample. The corresponding
proteomes were compared with the statistical model AMMI
(additive main effects with multiplicative interaction) that aimed
at regrouping proteins according to their solubility and
electrophoretic properties. Distinct groups emerged from this
analysis and the identification of proteins in each group allowed
us to assign specific features to several of them. For instance,
two of these groups regrouped very hydrophobic proteins, one group
contained V-ATPase subunits, another group contained proteins with
one transmembrane domain as well as proteins known to interact with
membrane proteins. This study provides methodological tools to study
particular classes of plasma membrane proteins and should be applicable
to other cellular membranes.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Kaufman E, Lamster IB
 </span>
 <span class="year">
(2000)
 </span>
 <span class="jname">
J. Clin. Periodontol.
 </span>
 <span class="jvol">
27,
 </span>
 <span class="pages">
453-465.
 </span>
<p class="reftitl">
Analysis of Saliva for Periodontal Diagnossis
<p class="Para1">
<b>REVIEW.</b>
Probing depth, attachment level, bleeding on probing (BOP), plaque index (PI) and
radiographic loss of alveolar bone are used to assess clinical periodontal disease.
Microbial infection and environment in the gingival crevicular fluid (GCF) is
used to assess risk of potential disease or evaluate current disease.
<p>
Analysis of saliva provides great opportunities.  It contains proteins of
all classes (transporters, buffering agents, growth factors, matrix beds,
antimicrobial enzymes and immunoglobulins), small organic molecules that
include all classes of metabolites, and virtually every major eleectrolyte
and trace element.
<p>
Enzymes present in saliva are likely produced by many differetnt sources
besides saliva glands; microorganisms and polymorphonuclear leukocytes,
epithelial cells and from GCF.  Saliva from adult patients with periodontitis
shows higher levels of alkaline phosphatase, esterase, &beta;-glucuronidase,
&beta;-glucosidase, other aminopeptidases.  These levels are higher than for
healthy patient saliva; juveniles with periodontitis show the same high levels.
Enzyme levels are reduced with treatment (root planing &amp; scaling plus
tetracycline);  the belief is that microbes contribute to this enzyme pool.
Effective nof treatment might be determined by monitoring bacterial enzymes or
protein levels in saliva.  Whole saliva in juvenile periodontitis shows higher
Ig-degrading enzyme levels, and presence of these enzymes may provide bacteria
an ecological advantage.
<p>
The matrix glycoprotein fibronectin offers an advantage to some bacteria while
inhbiting others.  Degrading enzymes made by bacteria might help them adhere
and colonize.  Whole saliva collected immediately after awakening and after
toothbrushing shows a higher level of proteolytic activity in the saliva
collected before brushing.
<p>
In a long term study of 24 adults with advanced periodontitis and 25 healthy
subjects, the diseased group was treated and evaluated over 20 months.
Salivary elastase levels correlated with deep pocket count and percentage
of bleeding sites.  Proteolytic activity (trypsin-like, collagenase,
elastase-like) was higher in adults with periodontitis.  Studies show
collagenase of host source is found in whole, but not parotid, sublingual,
or submandibular saliva.  With treatment collagenase level falls.  The authors
conclusion that this collagenase originated from PMNs entering oral cavity
through gingival sulcus;  fibroblasts were ruled out.  In particular Downs
syndrome children show elevated collagenase levels:  the activation of
matrix metalloproteinases from PMNs and/or cytokine-activated fibroblasts
may account for this early onset destruction.
<p>
Reduced levels of tissue inhibitor of metalloproteinases I (TIMP-1) was lower
in patients with periodontal disease.  Collagenase and gelatinase levels
in saliva of 23 adult periodontitis patients and 7 juveniles were higher
than controls in one study, and reduced after treatment, although there were
no significant correlations of enzyme activity and clinical parameters (
gingival indices, plaque index, pocket depth).  Patients with periodontitis
have shown decreased lysozyme levels compared to healthy patients;  however
another study of 7 juveniles and 7 healthy children shows lysozyme, myeloperoxidase,
and lactoferrin as significantly elevated in peripheral blood PMNs and in
GCF.  Thiocyanate levels show an inverse relationship to presence of plaque
and of gingival inflammation when measured in both resting and stimulated
saliva.  Lactoferrin levels in stimulated saliva are directly correlated to
gingival disease severity and plaque.  Lysozyme was correlated to plaque index,
and free lysozyme related to gingivitis severity.  Other proteins studied
include kallikrein and kininogen.
<p>
Plasma cells in the salivary glands are responsible for the presence of
secretory IgA (sIga).  The majority of sIgA comes from the parotid.  sIgA
probably works by interfering with the ability of bacteria to adhere and
colonize.  IgA is composed of two classes:  IgA<sub>1</sub> predominates
in serum while IgA<sub>2</sub> generally is found in secretions (tears,
milk, saliva.  Presence of IgG is derived directly from serum via the GCF,
is in low levels in healthy gums, higher levels with periodontitis.
Correlations have been shown between IgA levels and inflammation severity.
IgA, IgG, and IgM levels in another study were correlated with juvenile
periodontitis, although another study reveals lower IgA concentrations
in mixed unstimulated saliva.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Regula JT, Ueberle B, Boguth G, G&ouml;l A, Schn&ouml;er, Herrmann R,
Frank R
 </span>
 <span class="year">
(2000)
 </span>
 <span class="jname">
Electrophoresis
 </span>
 <span class="jvol">
21,
 </span>
 <span class="pages">
3765-3780.
 </span>
<p class="reftitl">
Towards A Two-Dimensional Proteome Map of <i>Mycoplasma pneumoniae</i>
<p class="abstract">
<b>ABSTRACT</b>.
A Proteome map of the bacterium Mycoplasma pneumoniae was constructed
using two-dimensional (2-D) gel electrophoresis in combination with
mass spectrometry (MS). M. pneumoniae is a human pathogen with
a known genome sequence of 816 kbp coding for only 688 open reading
frames, and is therefore an ideal model system to explore the scope
and limits of the current technology. The soluble protein content
of this bacterium grown under standard laboratory conditions was
separated by 1-D or 2-D gel electrophoresis applying various pH
gradients, different acrylamide concentrations and buffer systems.
Proteins were identified using liquid chromatography-electrospray
ionization ion trap and matrix-assisted laser desorption/ionization-MS.
Mass spectrometric protein identification was supported and controlled
using N-terminal sequencing and immunological methods. So far,
proteins from about 350 spots were characterized with MS by
determining the molecular weights and partial sequences of their
tryptic peptides. Comparing these experimental data with the DNA
sequence-derived predictions it was possible to assign these 350
proteins to 224 genes. The importance of proteomics for genome
analysis was shown by the identification of four proteins, not
annotated in the original publication. Although the proteome map
is still incomplete, it is already a useful reference for
comparative analyses of M. pneumoniae cells grown under modified
conditions.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Lehmann WD, Schlosser A, Erben G, Pipkorn R, Bossemeyer D, Kinzel V
 </span>
 <span class="year">
(2000)
 </span>
 <span class="jname">
Prot. Sci.
 </span>
 <span class="jvol">
9,
 </span>
 <span class="pages">
2260-2268.
 </span>
<p class="reftitl">
Analysis of Isoaspartate in Peptides by Electrospray Tandem Mass Spectrometry
<p class="Para1">
IsoAsp (&beta;-Asp) is found naturally and its occurrence impairs protein
function. For example, in tau protein accumulation of isoAsp is associated with
aggregation seen as paired helical filaments.  Moreover knockouts of the known
repair enzyme for isoAsp. protein L-isoapartate (D-aspartate) O-
methyltransferase (PIMT) results in severe seizures and reduced life span in
mice, and isoAsp accumulation is seen.  IsoAsp forms by spontaneous deamidation
via &beta;-aspartyl shift in which a cyclic succinimide intermedate forms, and
the ring is hydrolyze with a 3:1 IspAsp:Asp product ratio; racemization occurs
when the ring intermediate is formed.  PIMT methylates L-isoAsp and D-Asp
residues.  The isoAsp methyl esters then become the succinimide structure and
re-enter the &beta;-aspartyl shift.  With repeated cycling, a portion is
converted back to Asp. Some methods assay the level of L-isoAsp (not D-form).
PIMT is utilized with radiolabeled S-adenosylMet in the methylation reaction and
the quantity of radiolabeled MeOH reports the L-isoAsp present.  An alternative
is an HPLC-based measure of S-adenosylHomoCys product released (no radiolabel
necessary).  Using radiolabeling and then protein digestion, the site of isoAsp
can be determined. Edman sequencing can be used since it stops at the isopeptide
bond.
<p> Using low energy CID, fragment ions can be produced that are <i>b</i>- and
<i>y</i>-ions generated by cleavage at N-terminal side (X-Asp/IsoAsp bond),
corresponding <i>b</i>- and <i>y</i>-ions from cleavage at C-terminal side
(Asp/isoAsp-X), and the Asp/isoAsp immonium at <i>m/z</i> 88.
<p> The myristoylated N-terminal petpide (1-7) of protein kinase A catalytic
subunit was the model.  Asn-2 will see partial deamidation/isomerization and the
product ion spectra of synthetic myrG-D-AAAK and myrG-isoD-AAAAK were recorded
and collision energy changed from &minus;10 to &minus;110 V (spectrum shows
accumulation of scans at a change of 1 V per scan).  Fragment ions specific for
Asp or IsoAsp are absent; relative abundances rather than absolute intensities
are measured, which are reproducible for product ion spectra under same
conditions of gas pressures and collision energies.  in the spectra (1) the
<i>b</i><sub>2</sub> (<i>m/z</i> 383) over <i>y</i><sub>5</sub> (<i>m/z</i> 431)
decreases for the isoAsp petpide, (2) <i>b</i><sub>1</sub> (<i>m/z</i> 268) over
<i>y</i><sub>6</sub> (<i>m/</i>z 546) is changed in same way, and (3) Asp immonium
at <i>m/z</i> 88.
<p>
With collision offset set to &minus;80V, there was in intention to obtain
immonium ions.  Three forms of T1 were studied.  The D2 (Asp in 2nd position),
isoD2, and the succinymidyl (dehydrated) forms of the heptapeptide were
evaluated.  The m/z 80-90 fine mass range is shown.  In D2, the immonium-D
peak at m/z 88 is relatively intense compared to the base peak immK-NH<sub>3</sub>
at m/z 84 (Lys is the C-term residue).  A signal of imm-D is still observed
with the iso-D2 peptide, but no m/z 88 signal is seen with succ-D form.
Thus immonium forms only original from noncyclic forms.  Skimmer CID was
used to purify the ion stream of D and isoD isomers.  The fragment ion spectra
were identical, and so immonium ions form equally from Asp or isoAsp peptides.
<p>
Additional peptides with Asn sites known to form isoAsp were studied.
To understand mechanisms, it is important to examine relative intensity ratios
with reference ions:  immonium ion of C-term Lys &minus;17 Da at <i>m/z</i> 84,
immonium ion of Pro and immonium ion of Arg &minus; 59 Da at <i>m/z</i> 70,
immonium of Leu/Ile at <i>m/z</i> 86, and immonium of Val at <i>m/z</i> 72.
IsoAsp peptides consistently show a lower <i>m/z</i> 88 relative intensity
than the Asp-containing peptides.  Complementary b and y ions were also
examined.  All peptide paris showed a higher <i>b/y</i> ratio in Asp form compared
to isoAsp form.  15 Asp/IsoAsp peptides pairs showed big differences  in
<i>m/z</i> 88 abundances and <i>b/y</i> intensity ratios.
<p>
IsoAsp formation results in a consistently reduced <i>b</i> ion formation (see the
<i>b/y</i> intensity ratio) and reduction of the Asp immmonium at <i>m/z</i> 88.  This
relationship shows that immonium ions result from <i>b</i> ions.  The currently
accepted theory is that C-N peptide bond cleavage  involves protonation of amide
nitrogen in the first step, weakening the C-N bond and increasing electrophilic
character of carbonyl atom.  The C-N bond is broken directly or by nucleophilic
attack on the carbonyl oxygen at N-terminal side of carbonyl carbon of the
protonated amide  Oxygen-to-carbon attack leads to five-centered oxazolone,
with <i>b</i>-ion formation favored.  Direct bond cleavage favors <i>y</i>-ion formation.
<p>
When Asp is replaced with isoAsp in the structure, the reaction mechanism of
oxazolone intermediate to <i>b</i>-ion formation is impeded.  New interactions are
created.  With an extra methylene group inserted, a less-favored six-centered
interaction is required and hampers <i>b</i>-ion formation.
<p>
The approach to examining Asp and IsoAsp isomers is first to examine product
ion spectra of the pure isomers.  The most pronounced quantitative differences
are selected as the discriminatory parameter.  Thus the immoniumm Asp at
m/z 88 and the b/y ion pairs to Asp/isoAsp show the pronounced quantitative
differences.
<p>
To determine the optimal collision energy for distinguishing peptide pairs,
a plot of the ratio of ratios of intensities is done.  Calculate the
<i>m/z</i> 88 / <i>m/z</i> 84 of the D peptide, and the <i>m/z</i> 88 /
<i>m/z</i> 84 of the iso-D peptide, and then plot the ratio of D to isoD ratios.
This is for the K-terminated peptides.  For the R-termininated peptides,
instead of the m/z 84 peak, use the m/z 70 peak.  When these are plotted against
collision energies (from &minus;40 to &minus;150 V, in roughly 10-15 V steps),
A broad maximum between &minus;60 and &minus;120 V is found for the pairs
myrG-X-AAAAR, myrG-X-AAAAK, and GA-X-TAAK.  For pairs GA-X-GAAK and VSI-X-QK,
increasing collision energy was proportional to increasing discriminatory power.
As an additional indicator absolute (not relative) intensity at <i>m/z</i> 88 at
80 V is useful.
<p>
A series of test mixtures with varying amounts of Asp- and isoAsp-isomer
of the PKA fragment myrG-X-AAAAK were done on volumetric basis and analyzed
by ESI-MS/MS with <i>m/z</i> 88 normalized to <i>m/z</i> 84.  Then a plot of
mole percent of Asp-peptide in a mixture based on the <i>m/z</i> 88/84 ratio
against the volume percenter of Asp-peptide solution done.  This shows a line
of increasing slope which indicates a correlation between volumetric data
and the 88/84 ratio.  Using the VQ-X-GLR peptide (N-terminal region being
residues 41-46 of human procollagen, which quickly forms isoAsp, Table 2 shows
the <i>b</i><sub>2</sub>/<i>y</i><sub>4</sub> intensity ratio decreases
by factor of 40 for isoAsp compared to
Asp.  Plots of the ratio of intensity of <i>m/z</i> 228 / <i>m/z</i> 460 were done against
the volume percent of the isoAsp peptide:  a parabolic curve upward is found,
indicating that there is a relationship, not always linear, between these
intensity ratios and the mix of peptides.  These peptides had a concentration
of 5 pmol/&micro;l, such that each scan was &lt; 1 pmol peptide consumed.
<p>
ESI-MS/MS can thus be used to track deamidation and isomerization, and several
components at the same time, so long as peptide mass differs for the components.
<p>
M &amp; M.  A Finnigan MAT TSQ 7000 triple quad with nanoESI (EMBL, Germany) was
used.  Spray capillaries were made in house with Sutter Instruments (Novato, CA)
micropipette puller and surface-coated semitransparent gold film using sputter
unit.  Voltage on spray was +400 to +700 V; argon was used at 2e-3 millitorr.
Solvent was 2:1 MeOH:water with 2% formic acid and sprayed at 5 pmol/&micro;l.
50 consecutive scans with 4 s duration were acquired and averaged to get
single data point.  Data are the means of 3 data points.  Peptides are synthesized
on solid phase using Fmoc (9-fluorenylmethyloxycarbonyl) principles using
Abimed AMS 422.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Berggren K, Chemokalskaya E, Steinberg T, Kemper C, Lopez M, Diwu Z, Haugland R,
Patton W
 </span>
 <span class="year">
(2000)
 </span>
 <span class="jname">
Electrophoresis
 </span>
 <span class="jvol">
21,
 </span>
 <span class="pages">
2509-2521.
 </span>
<p class="reftitl">
<p class="Para1">
<!-- ============================================================= -->
<p class="tags">
peptides, sodium and potassium adducts, FD
<p class="ref">
 <span class="authors">
Rodriquez CF, Guo X, Shoeib T, Hopkinson AC, Siu KWM
 </span>
 <span class="year">
(2000)
 </span>
 <span class="jname">
J. Am. Soc. Mass Spectrom.
 </span>
 <span class="jvol">
11,
 </span>
 <span class="pages">
967-975.
 </span>
<p class="reftitl">
Formation of [M &minus; <i>n</i>H + 
<i>m</i>Na]<sup>(<i>m</i>&minus;<i>n</i>)+</sup> and [M &minus; <i>n</i>H + 
<i>m</i>K]<sup>(<i>m</i>&minus;<i>n</i>)+</sup> Ions in Electrospray Mass 
Spectrometry of Peptides and Proteins
<p class="Para1">
The formation of adduct cations to peptides and proteins particularly include
those interactions with alkali metals (sodium, potassium) especially when present
as impurities or additives.  These ions replace protons.  Cu<sup>2+</sup> and
Ni<sup>2+</sup> will induce abstraction of amide protons from oligopeptides
in solution, after ligation to N-terminal nitrogen atom.  Complex ions of the
type [M&minus;<i>n</i>H+<i>m</i>X]<sup>(<i>m</i>&minus;<i>n</i>)+</sup> could
be formed in the lens of the MS after collisional activation, as well as from
collisions between peptide and neutral alkali metal compounds.  Using lithium
hydroxide, chloride and hydrate as examples, enthalpies were calculated for 
reactionswith N-methylacetamine, acetic acid, acetate ion, 1-propanamine
(these all models of functionl groups found on peptides) and the majority of
reactions are exothermic or slightly endothermic.  In a model of reaction between
HCONH<sub>2</sub> and LiOH, reactions are dominated by relatively deep well from
formation of binary complex and are devoid of barriers.  Reaction rates are thus
determined by reaction enthalpies.  Using typical electrospray sampling condtions
with endothermicities &lt; 20 kcal/mol, peptide-lithium salt reactions which
are exothermic or endothermic by less than 20 kcal/mol proceed to form 
[M&minus;<i>n</i>H+<i>m</i>Li]<sup>(<i>m</i>&minus;<i>n</i>)+</sup>.  The
study of lithiated products was based on computational economy. (see
Rodriquez et al, Int J. Mass Spectrom. 192: 303, 1999).
<p>
Experiments done here are with PE-SCIEX API 3000 prototype triple quad MS
using ubiquitin at 10 pmol/&micro;l + 1 mM (1000 pmol/&micro;l) in 50% MeOH.
Infusion was at 2 &micro;l/min into pneumatically assisted electrospray probe.
Mass spectra were acquired in range <i>m/z</i> 600-3000, with step size of 0.2 u,
dwell of 10 ms, and 10 scans summed to produce mass spectrum.  Density
functional theory (DFT) was done using the B3LYP method, with Becke's 3-parameter
exchange function (see Becke, J. Chem. Phys. 98: 5648, 1993) and the correlation
functional of Lee et al (Phys Rev B: 37: 785, 1988). with the 6-311++G(d,p)
basis set (Krishan et al J. Chem Phys 72: 650, 1980; McLean and Chandler,
J Chem Phys 72: 5639, 1980; Chandresekhar et al, JACS 103: 5609, 1981; 
Chandresekhar et al J. Comput Chem. 4: 294, 1983; Curtiss et al, J Chem Phys
103: 6104, 1995) in Gaussian 98, (Gaussian Inc.) and used to calculate optimized
geometries and vibrational frequencies.  First-order saddle points were found
using Berny transition state algorithm and the CALCALL method.
<p>
The ESI spectrum of Ubq with KCl and KOH show several charge states, and at each
charge state are several ions forming an envelope that fits the patterns of
[M&minus;(<i>m</i>&minus;<i>z</i>)H+<i>m</i>K]<sup><i>z</i>+</sup> where <i>z</i>
represents the charge state.  More K<sup>+</sup> was incorporated with its hydroxide
compared to its chloride.  Ubq is a 76 residue protein with avg mass of 8564.9 Da.
The solution to the count of K<sup>+</sup> is given by 
<p class="eqnserif">
<i>m</i> = (5 &times; (<i>m/z</i>) &minus; 5 &minus; 8564.9) / (39.1 - 1.0)
<p>
where <i>m/z</i> is the ion in the charge state of +5 cluster.  With chloride
salt, <i>m</i> ranges from 0 to 20 where <i>m</i>=5 is the most abundant ion,
but with the hydroxide, <i>m</i> ranges from 9 to 21 with <i>m</i> = 14 being the
most abundant.  There is less K<sup>+</sup> incorporation with the chloride
with increasing cluster charge.  This is because of differences in reaction 
enthalpies with model functional groups of Ubq.  KOH is able to react with more
different functional groups than KCl; there are also strong resemblances between
NaCl and NaOH as with the potassium counterpart.  The resolution between the 
potassium adducts compared to sodium adducts are because of the higher mass of
potassium.
<p>
The reaction enthalpies of KOH, KCl, K(H<sub>2</sub>O)<sup>+</sup>, NaOH, NaCl, 
and Na(H<sub>2</sub>O)<sup>+</sup> with compounds which model function groups on 
the peptide are done.  These are N-methylacetamide 
[CH<sub>3</sub>-NH<sub>2</sub>-(C=O)-CH<sub>3</sub>] (a peptide bond mimic), 
acetic acid and acetate ion (mimics of Asp and Glu side chains and the 
C-terminus), propanamine [CH<sub>3</sub>-CH<sub>2</sub>-CH<sub>2</sub>-NH<sub>2</sub>] 
(mimics amino terminus and side chain of various basic residues).  For each
alkali methods, the following 12 reactions are considered:
<ol class="eqnserif">
<li>CH<sub>3</sub>CONHCH<sub>3</sub> + KOH &rarr; CH<sub>3</sub>CON(K)CH<sub>3</sub>
+ H<sub>2</sub>O
<li>CH<sub>3</sub>CONHCH<sub>3</sub> + KCl &rarr; CH<sub>3</sub>CON(K)CH<sub>3</sub>
+ HCl
<li>CH<sub>3</sub>CONHCH<sub>3</sub> + K(H<sub>2</sub>O)<sup>+</sup> &rarr; 
CH<sub>3</sub>CON(K)CH<sub>3</sub> + H<sub>3</sub>O<sup>+</sup>
<li>CH<sub>3</sub>COOH + KOH &rarr; CH<sub>3</sub>COOK + H<sub>2</sub>O
<li>CH<sub>3</sub>COOH + KCl &rarr; CH<sub>3</sub>COOK + HCl
<li>CH<sub>3</sub>COOH + K(H<sub>2</sub>O)<sup>+</sup> &rarr; 
CH<sub>3</sub>COOK + H<sub>3</sub>O<sup>+</sup>
<li>CH<sub>3</sub>COO<sup>&minus;</sup> + KOH &rarr; CH<sub>3</sub>COOK + OH<sup>&minus;</sup>
<li>CH<sub>3</sub>COO<sup>&minus;</sup> + KCl &rarr; CH<sub>3</sub>COOK + Cl<sup>&minus;</sup>
<li>CH<sub>3</sub>COO<sup>&minus;</sup> + K(H<sub>2</sub>O)<sup>+</sup> &rarr; 
CH<sub>3</sub>COOK + H<sub>2</sub>O
<li>CH<sub>3</sub>CH<sub>2</sub>CH<sub>2</sub>NH<sub>2</sub> +  KOH &rarr; 
CH<sub>3</sub>CH<sub>2</sub>CH<sub>2</sub>NHK + H<sub>2</sub>O
<li>CH<sub>3</sub>CH<sub>2</sub>CH<sub>2</sub>NH<sub>2</sub> +  KCl &rarr; 
CH<sub>3</sub>CH<sub>2</sub>CH<sub>2</sub>NHK + HCl
<li>CH<sub>3</sub>CH<sub>2</sub>CH<sub>2</sub>NH<sub>2</sub> +  
 K(H<sub>2</sub>O)<sup>+</sup> &rarr; 
CH<sub>3</sub>CH<sub>2</sub>CH<sub>2</sub>NHK + H<sub>3</sub>O<sup>+</sup>
</ol>
<p>
Standard enthalpies can be calculated from thermochemical data, if the potassium
ion affinities (KA values) of the anions of N-methylacetamide, acetate, and
propanamine are independently known.
<ul>
<li>&Delta;<i>H</i><sup>0</sup><sub>2</sub> = 
&Delta;<i>H</i><sup>0</sup><sub>acid</sub> (N-methylacetamide) + 
KA (OH<sup>&minus;</sup>) &minus; &Delta;<i>H</i><sup>0</sup><sub>acid</sub>
(H<sub>2</sub>O) &minus; KA (1)
<li>&Delta;<i>H</i><sup>0</sup><sub>8</sub> = 
&Delta;<i>H</i><sup>0</sup><sub>acid</sub> (acetic acid) + 
KA (Cl<sup>&minus;</sup>) &minus; &Delta;<i>H</i><sup>0</sup><sub>acid</sub>
(HCl) &minus; KA (2)
<li>&Delta;<i>H</i><sup>0</sup><sub>12</sub> = 
&Delta;<i>H</i><sup>0</sup><sub>acid</sub> (propanamine) + 
KA (H<sub>2</sub>O) &minus; PA (H<sub>2</sub>O) &minus; KA (3)
</ul>
<p>
where PA is proton affinity.  Calculations included both KA and SA (sodium 
affinity) values of NMAA (N-methylacetamide), acetate, and PA, and also 
affinities for hydroxide, chloride, and hydrate using MO calculations.  The 
latter are included as references to permit gauging accuracy of DFT method at 
B3LYP/6-311++G(d,p) level of theory. Electronic energies, zero-point vibrational 
energies, and thermal corrections (adjusted to 298 K) are shown in Table 1 of 
the potassiated forms of these three molecules, and also of the sodiated forms, 
in addition to the hydroxides, chlorides, and hydrates.
<p>
The attachment of Na and K to NMAA anion will produce conformational change
which is same as when Li attaches, in which two methyl groups are <i>cis</i> to
one another (compare structures <b>4, 7,</b> and <b>10</b>).  In <b>10</b> there
is a minor difference in which C-methyl group is staggered about O-Li bond,
and in Na- and K-containing analogs <b>4</b> and <b>7</b>, C-methyl group 
eclipses the O-metal bond.  Replacing Li with Na or K leadst to lenghtening of
X-O (X= alkali metal) bond and the X-N bonds, and widening of X-O-C, O-C-N,
and C-N-X bond angles.  otherwise salt structures of <b>10, 7</b> and <b>4</b>
are identical.  Increases in metal-ligand bond lengths and metal-related bond
angles in accordance with increase in atomic radii from Li to Na and K.  Similar
effects with acetate ion (<b>2</b>) and 1-propanamide ion (<b>3</b>). 

<table class="td-center">
<caption style="text-align:justify;">Calculated Li, Na, and K ion affinities of anions (kcal/mol) using DFT 
at the B3LYP/6-311++G(d,p) level of theory.  Experimental values (Expt.) are also 
shown for comparison</caption>
<tr><th rowspan="2">Anion 	<th colspan="2">Lithium<sup><i>a</i></sup>  
  <th colspan="2">Sodium	<th colspan="2">Potassium
<tr><th>theory  <th>Expt. 	<th>theory	<th>Expt. 	<th>theory 	<th>Expt 	 
<tr><td>1     <td>170.4   <td>       <td>170.4   <td>       <td>124.0  <td>     
<tr><td>2     <td>170.5   <td>       <td>170.5   <td>       <td>127.0  <td>      
<tr><td>3     <td>175.6   <td>       <td>175.6   <td>       <td>128.7  <td>      
<tr><td>OH<sup>&minus;</sup>
   <td>187.9   <td>185.7<sup><i>a</i></sup>  
   <td>158.0   <td>158.7<sup><i>b</i></sup>  
   <td>141.5  <td>144.7<sup><i>a</i></sup>
<tr><td>Cl<sup>&minus;</sup>  
   <td>        <td>       <td>131.8        
   <td>132.7<sup><i>c</i></sup>
   <td>119.8    <td>118.3<sup><i>f</i></sup>
<tr><td>H<sub>2</sub>O 	
   <td>35.5    
   <td>33.9<sup><i>a</i></sup>   <td>25.0    
   <td>24.1<sup><i>d</i></sup>   <td>18.0   
   <td>17.2<sup><i>g</i></sup>
<tr><td colspan="7" style="font-size:85%;">
<sup><i>a</i></sup> Rodriguez et al <i>Int. J. Mass Spectrom.</i> <b>192</b>, 303-317. 
<br>
<sup><i>b</i></sup>&Delta;<i><sub>f</sub>H</i> <sup>0</sup>  (Na<sup>+</sup>) + 
&Delta;<sub><i>f</i></sub><i>H</i> <sup>0</sup> (OH<sup>&minus;</sup>) &minus; 
&Delta;<i><sub>f</sub>H</i> <sup>0</sup> (NaOH) = 144.1 &minus; 32.7 + 47.3 = 
158.7
<br>
<sup><i>c</i></sup>&Delta;<i><sub>f</sub>H</i> <sup>0</sup>  (Na<sup>+</sup>) + 
&Delta;<sub><i>f</i></sub><i>H</i> <sup>0</sup> (Cl<sup>&minus;</sup>) &minus; 
&Delta;<i><sub>f</sub>H</i> <sup>0</sup> (NaCl) = 144.1 &minus; 54.4 + 43.0 = 
132.7
<br>
<sup><i>d</i></sup>&Delta;<i><sub>f</sub>H</i> <sup>0</sup>  (Na<sup>+</sup>) + 
&Delta;<sub><i>f</i></sub><i>H</i> <sup>0</sup>(H<sub>2</sub>O) &minus; 
&Delta;<i><sub>f</sub>H</i> <sup>0</sup> (NaOH<sub>2</sub><sup>+</sup>) = 144.1 
257.8 &minus; 62.2 = 24.1
<br>
&nbsp;&nbsp;&nbsp;&Delta;<i><sub>f</sub>H</i> <sup>0</sup> 
(NaOH<sub>2</sub><sup>+</sup>) = &Delta;<i><sub>f</sub>H</i> <sup>0</sup> 
(H<sup>+</sup>) + &Delta;<sub><i>f</i></sub><i>H</i> <sup>0</sup> (NaOH) &minus; 
PA (NaOH) = 365.7 &minus; 47.3  &minus; 256.2 = 62.2
<br>
<sup><i>e</i></sup>&Delta;<i><sub>f</sub>H</i> <sup>0</sup>  (K<sup>+</sup>) + 
&Delta;<sub><i>f</i></sub><i>H</i> <sup>0</sup> (OH<sup>&minus;</sup>) &minus; 
&Delta;<i><sub>f</sub>H</i> <sup>0</sup> (KOH) = 121.4 &minus; 32.7 + 56.0 = 
144.7
<br>
<sup><i>f</i></sup>&Delta;<i><sub>f</sub>H</i> <sup>0</sup> (K<sup>+</sup>) + 
&Delta;<sub><i>f</i></sub><i>H</i> <sup>0</sup> (Cl<sup>&minus;</sup>) &minus; 
&Delta;<i><sub>f</sub>H</i> <sup>0</sup> (KCl) = 121.4 &minus; 54.4 + 51.3 = 
118.3
<br>
<sup><i>g</i></sup>&Delta;<i><sub>f</sub>H</i> <sup>0</sup> (K<sup>+</sup>) + 
&Delta;<sub><i>f</i></sub><i>H</i> <sup>0</sup>(H<sub>2</sub>O) &minus; 
&Delta;<i><sub>f</sub>H</i> <sup>0</sup> (KOH<sub>2</sub><sup>+</sup>) = 121.4 
&minus; 57.8 &minus; 46.4 = 17.2
<br>
&Delta;<i><sub>f</sub>H</i> <sup>0</sup>  (KOH<sub>2</sub><sup>+</sup>) 
= &Delta;<i><sub>f</sub>H</i> <sup>0</sup> (H<sup>+</sup>) + 
&Delta;<sub><i>f</i></sub><i>H</i> <sup>0</sup> (KOH) &minus; PA (KOH) = 365.7 
&minus; 56.0 &minus; 263.3  = 46.4
<div style="margin:0.5em 0 0 0;font-size:83%;">
All reference thermochemical data from:
<ul style="margin:0;">
<li>Lias, SG et al (1988) <i>J. Phys. Chem. Ref. Data</i> <b>17</b>, Suppl. 1.
<li>Lias SG et al (1994) NIST Standard Reference Database 25, Version 2.02
<li>Bartmess, J. E. (1993) NIST Standard Reference Database 19B, Version 3.01.
<li>Hunter EP, Lias SG In <a href="http://webbook.nist.gov/chemistry/">NIST 
Chemistry WebBook</a>;  Hunter EP, Lias SG (1998) <i>J. Phys. Chem. Ref. 
Data</i> <b>27</b>, 413-656.
</ul>
</div>
</table>
<p>
The degree of agreement between calculated and experimental affinities from the 
table shows that the DFT B3LYP/6-311++G(d,p) method is adequate.  Another table 
shows reaction enthalpies for XOH, XCl, and X(H<sub>2</sub>O)<sup>+</sup> where 
X = Li, Na, and K, with N-methylacetamide, acetic acid, and acetate anion, and 
1-propanamine.  Reaction enthalpies which are exothermic less than 20 kcal/mol 
can be sufficiently fast to be observable because of multiple collision 
activation in the lens region, if barriers to reaction are not significant.  The 
20 kcal/mol limit is not arbitrary:  in a recent study with 
Ag(NH<sub>3</sub>)<sup>+</sup><sub><i>n</i></sub>, very few 
Ag(NH<sub>3</sub>)<sup>+</sup><sub>3</sub> ions and no 
Ag(NH<sub>3</sub>)<sup>+</sup><sub>4</sub> ions were observed when a solution of 
AgNO<sub>3</sub> was electrosprayed with 5% ammonia; the sampling conditions 
were mild.  With regular lens conditions, no 
Ag(NH<sub>3</sub>)<sup>+</sup><sub>3</sub> ions are observed, and abundances for 
Ag(NH<sub>3</sub>)<sup>+</sup><sub>2</sub> and Ag(NH<sub>3</sub>)<sup>+</sup> 
are very high.  DFT at B3LYP/DZVP level of theory show reaction enthalpies 
(&Delta;<i>H</i><sup>0</sup><sub><i>n,n+1</i></sub>) for reaction 
Ag(NH<sub>3</sub>)<sup>+</sup><sub><i>n</i></sub> + 
NH<sub>3</sub>&nbsp;&rarr;&nbsp;Ag(NH<sub>3</sub>)<sup>+</sup><sub><i>n</i>+1</sub> 
which are &Delta;<i>H</i><sup>0</sup><sub>0,1</sub> = &minus;40.1, 
&Delta;<i>H</i><sup>0</sup><sub>1,2</sub> = &minus;36.1, 
&Delta;<i>H</i><sup>0</sup><sub>2,3</sub> = &minus;15.1, 
&Delta;<i>H</i><sup>0</sup><sub>3,4</sub> = &minus;11.0 kcal/mol.  Last three 
values are in agreement with experimental values.  The absence of 
Ag(NH<sub>3</sub>)<sup>+</sup><sub>3</sub> and 
Ag(NH<sub>3</sub>)<sup>+</sup><sub>4</sub> is because of CID in lens region, 
which indicates energies of between 15 and 36 kcal/mol are available in 
collisional activation.  Ag(H<sub>2</sub>O)<sup>+</sup> and 
Na(CH<sub>3</sub>OH)<sup>+</sup> complexes have bond enthalpies of 28.4 and 22.3 
kcal/mol, respectively, and show intense signals.  The abundance ratio of 
Na(CH<sub>3</sub>OH)<sup>+</sup>:Na<sup>+</sup> is 1.2 to 1.5, and all these 
observations support a collisional activation of about 20 kcal/mol in lens region.
<p>
Count of functional groups whose reactions with alkali metal-containing reagents are
either exo- or endothermic by &lt;20 kcal/mol decreases from reagent being
XOH to XCl to X(H<sub>2</sub>O).  Most abundant functional group in peptide is
amide bond, and reactions of CH<sub>3</sub>NH-C(=O)-CH<sub>3</sub> with all
three alkali hydroxides are exothermic, while with alkali chlorides are 
endothermic but accessible for LiCl and maybe NaCl.
<p>
In the previous study the potential energy hypersurface of LiOH + HCONH<sub>2</sub>
(formamide) was the prototypical example for formation of 
[M&minus;<i>n</i>H + <i>m</i>Li]<sup>(<i>m&minus;n</i>)+</sup>.  Reaction profile
shows deep well of ~35 kcal/mol, and this is because of formation of 
formamide-lithium hydroxide complex with Li-O-C (lithiation of carbonyl oxygen)
then transfer of proton from amide to oxygen of LiOH, di-coordiation of Li
by amide O and N atoms, then elongation of Li-OH<sub>2</sub> bond to yield
HCON(Li)H and H<sub>2</sub>O complete reaction profile.  None of reactions have
large barriers and are driven by exothermicity.
<p class="eqnserif">
HCONH<sub>2</sub> + NaOH &rarr; HCON(Na)H + H<sub>2</sub>O
<br>
HCONH<sub>2</sub> + NaCl &rarr; HCON(Na)H + HCl
<p>
The structures of these molecules and their transition states as well as
many aspects of the energetics of the reaction are given.
<!-- ============================================================= -->
<p class="tags">
collision-induced dissociation (CID), alkali metal ions, bond energies,
guided ion beams, noncovalent interactions, thermochemistry, transtion
metal ions, FD
<p class="ref">
 <span class="authors">
Rodgers MT, Armentrout PB
 </span>
 <span class="year">
(2000)
 </span>
 <span class="jname">
Mass Spectrom. Rev.
 </span>
 <span class="jvol">
19,
 </span>
 <span class="pages">
215-247.
 </span>
<p class="reftitl">
Noncovalent Metal-Ligand Bond Energies As Studied By Threshold Collision-Induced 
Dissociation
<p class="Para1">
<b>REVIEW</b>.  Noncovalent interactions account for most molecular chemistry
such as condensation and solvation or conformation.  Metal ion-ligand binding
is a major type of noncovalent bonding.  Multiple metals are involved in many
types of noncovalent interactions:  dipole and polarizability effects, chelation,
conformers, tautomers.  Metal-ligand bond dissociation energies (BDEs) are
determined according to the following reaction:
<p class="eqnserif">
M<sup>+</sup>(L)<sub>x</sub> + Xe &rarr; (L)<sub>x&minus;1</sub>M<sup>+</sup> + L
+ Xe
<p>
This endothermic reaction shows a Xe atom knocking off a ligand from the metal.
A plot of the cross section of the reaction (from 0 to 30 &times; 10<sup>&minus;16</sup>
cm<sup>2</sup>) vs. kinetic energy (from 0.0 to 5.0 eV for center-of-mass frame,
0.0 to 8.0 eV for laboratory frame) 
<!-- ============================================================= -->
<p class="tags">
effective temperature, computational chemistry, microcanonical analysis
<p class="ref">
 <span class="authors">
Ervin KE
 </span>
 <span class="year">
(2000)
 </span>
 <span class="jname">
Int. J. Mass Spectrom.
 </span>
 <span class="jvol">
195/196,
 </span>
 <span class="pages">
271-284.
 </span>
<p class="reftitl">
Microcanonical analysis of the kinetic method. The meaning of the <q>effective 
temperature</q>
<p class="Para1">
Cooke's <q>kinetic method</q> is an MS technique used widely to obtain
thermochemical information about ion-molecule systems.  The metastable or
CID of mass-selected cluster ion having two product channels is followed by
MS/MS such as proton-bound anion A<sub>1</sub>HA<sub>2</sub><sup>&minus;</sup>:
<p class="eqnserif" style="margin:1em 25%;text-align:right;">
A<sub>1</sub>HA<sub>2</sub><sup>&minus;</sup> &rarr; A<sub>1</sub><sup>&minus;</sup>
+ HA<sub>2</sub>&nbsp;&nbsp;&nbsp;(<i>k</i><sub>1</sub>)&nbsp;&nbsp;&nbsp;[1]
<br>
&nbsp;&nbsp;&nbsp;&rarr;HA<sub>1</sub> + A<sub>2</sub><sup>&minus;</sup>
&nbsp;&nbsp;&nbsp;(<i>k</i><sub>2</sub>)&nbsp;&nbsp;&nbsp;[2]
<p>
These two reactions compete for the anion.  The correlation between ratio 
of intensities of two product ions and the enthalpy or Gibbs free energy 
difference between to product channels is expressed as:
<p class="eqnserif">
<b>ln</b> <i>I</i><sub>2</sub>/<i>I</i><sub>1</sub> &#8776; 
&Delta;&Delta;<i>H</i>/<i>RT</i><sub>eff</sub> &#8776; 
&minus;&Delta;&Delta;<i>G</i>/<i>RT</i><sub>eff</sub>&nbsp;&nbsp;&nbsp;[3]
<p>
where <i>I<sub>j</sub></i> is the intensity of ion 
A<sub><i>j</i></sub><sup>&minus;</sup>, &Delta;&Delta;<i>H</i> = 
&Delta;<sub>2</sub><i>H</i> &minus; &Delta;<sub>1</sub><i>H</i> 
&Delta;&Delta;<i>G</i> = &Delta;<sub>2</sub><i>G</i> &minus; 
&Delta;<sub>1</sub><i>H</i> <q>Effective temperature</q> 
(<i>T</i><sub>eff</sub>) is a correlation parameter which is determined from the 
slope of <b>ln</b> <i>I</i><sub>2</sub>/<i>I</i><sub>1</sub> vs. either 
&Delta;&Delta;<i>H</i> or &minus;&Delta;&Delta;<i>G</i>.  In measuring the
product intensity ratios for proton-bound dimers composed of partners of known
acidity, slope of the correlation is found and the acidity of the unknown can be
assessed against a reference acid.  Besides proton bound dimers, other types
of clusters have been studed by the kinetic method to determine other 
thermochemical quantities, including electron affinities for electron-bound
dimers (see references).  This paper reports a microcanonical kinetics analysis
of the kinetic method using classical RRK (Rice-Ramsperger-Kassel) theory
of statistical unimolecular dissociation.  A simple analytical expression
for effective temperature for case of metastable ion dissociation is derived.
The ratio of ion intensities observed for two pathways to products is 
equal to the ratio of macroscopic rate coefficients:
<p class="eqnserif">
<i>I</i><sub>2</sub> / <i>I</i><sub>1</sub> = 
<i>k</i><sub>2</sub>(<i>T</i>) / <i>k</i><sub>1</sub>(<i>T</i>)&nbsp;&nbsp;&nbsp;[4]
<p>
Rates for competing decomposition channels <i>j</i> = 1 and 2 are calculated
using canonical transition state theory:
<p class="eqnserif">
<i>k<sub>j</sub></i>(<i>T</i>) = { [ <i>k</i><sub>B</sub><i>T</i>  / <i>h</i>] 
[<i>Q<sub>j</sub></i><sup>&Dagger;</sup>(<i>T</i>) / 
<i>Q</i><sub>0</sub>(<i>T</i>) ] } 
<i>e</i><sup>&minus;&Delta;<sub><i>j</i></sub><i>E</i><sub>0K</sub><sup>&Dagger;
</sup> / <i>RT</i></sup>&nbsp;&nbsp;&nbsp;[5]
<p>
where <i>T</i> is temperature, <i>h</i> is Planck's constant, 
<i>k</i><sub>B</sub> is Boltzmann's constant, 
<i>Q<sub>j</sub></i><sup>&Dagger;</sup>(<i>T</i>) is partition function for 
transition state leading to channel <i>j</i>, <i>Q</i><sub>0</sub>(<i>T</i>) is 
partition function for cluster ion, and 
&Delta;<sub><i>j</i></sub><i>E</i><sub>0K</sub><sup>&Dagger;</sup> is the 
energy difference between the cluster and transition state at zero temperature.
The rate coefficient, <i>k</i>(<i>T</i>), applies strictly to canonical ensemble
of reactant molecules (that is system in thermal equlibrium at temp <i>T</i>).
The logarithm of the ratio of the rates will evaluate to:
<p class="eqnserif">
<b>ln</b> <i>k</i><sub>2</sub>(<i>T</i>) / <i>k</i><sub>1</sub>(<i>T</i>) = 
<b>ln</b> [ <i>Q</i><sub>2</sub><sup>&Dagger;</sup>(<i>T</i>) / 
<i>Q</i><sub>1</sub><sup>&Dagger;</sup>(<i>T</i>) ] &minus;
{ &Delta;<sub>2</sub><i>E</i><sub>0K</sub><sup>&Dagger;</sup> &minus; 
&Delta;<sub>1</sub><i>E</i><sub>0K</sub><sup>&Dagger;</sup> } / 
<i>RT</i>&nbsp;&nbsp;&nbsp;[6]
<p>
Assumptions are made
<ol>
<li>no reverse activation energies: 
&Delta;<sub><i>j</i></sub><i>E</i><sub>0K</sub><sup>&Dagger;</sup> = 
&Delta;<sub><i>j</i></sub><i>E</i><sub>0K</sub>
<li>entropy differences between two channles are zero:  &Delta;&Delta;<i>S</i>
= <i>R</i> <b>ln</b>(<i>Q</i><sub>2</sub> / <i>Q</i><sub>1</sub>) = 0, or else
same for all dimer pairs:  &Delta;&Delta;<i>S</i> = &Delta;<sub>2</sub><i>S</i>
&minus; &Delta;<sub>1</sub><i>S</i>
<li>integrated heat capacity and entropy differences between internal energ,
enthalpy, and Gibbs energy at various temperatures are equal for the two channels
and they cancel:  &Delta;&Delta;<i>E</i><sub>0K</sub> = 
&Delta;<sub>2</sub><i>E</i><sub>0K</sub>  &minus; &Delta;<sub>1</sub><i>E</i><sub>0K</sub> 
&#8776; &Delta;&Delta;<i>H<sub>T</sub></i> &#8776; &Delta;&Delta;<i>G<sub>T</sub></i>
</ol>
<p>
With the approximations, the following expression is obtained:
<p class="eqnserif">
<b>ln</b> <i>k</i><sub>2</sub>(<i>T</i>) / <i>k</i><sub>1</sub>(<i>T</i>) &prop; 
&minus; &Delta;&Delta;<i>H</i> / <i>RT</i> &#8776; &minus; 
&Delta;&Delta;<i>G</i> / <i>RT</i>&nbsp;&nbsp;&nbsp;[7]
<p>
The temperature in the expression is the effective temperature because clusters
in the ion beam are isolated species and not in thermodynamic equilibrium 
at any temperature.  Combining Eqs [4] and [7] thus show how Eqn [3] is derived.
<p>
When an ion leaves the source of the MS or undergoes its last collision, it is
finally isolated and its total internal energy (<i>E</i>) conserved.  Decomposition
is described by the microcanonical rate coefficient, <i>k</i>(<i>E</i>), than
the canonical rate coefficient, <i>k</i>(<i>T</i>).  Products are detected
in MS/MS if cluster ion lifetime falls within narrow time window corresponding
to field-free region between two mass-selection regions. With sector instruments
a typical time window is defined by the time of flight travel from the source
of 20-40 &micro;s, and ions with shorter liftimes will have higher internal 
energies and dissociate in first mass-selection region and are ejected.
Ions with longer lifetimes (lower internal energies) do not dissociate in second
mass-selected region.  If ions dissociate within the time window of a MS/MS
have one internal energy of <i>E</i> = <i>E</i><sub>d</sub>, the ion intensity
ratio is:
<p class="eqnserif">
<i>I</i><sub>2</sub> / <i>I</i><sub>1</sub> = 
<i>k</i><sub>2</sub>(<i>E</i><sub>d</sub>) / 
<i>k</i><sub>1</sub>(<i>E</i><sub>d</sub>)&nbsp;&nbsp;&nbsp;[8]
<p>
and this is the microcanonical equivalent of Eqn [4].
<p>
Classical RRK theory (also called classical quasiequilibrium theory, QET)
provides a useful approximation for calculating microcanonical unimolecular
dissociation rate.  The RRK model is not adequate for quantitative calculation
of absolute rates, but provides function that reproduces the behavior of the 
unimolecular dissociation rate as function of internal energy.  RRK rate
constant <i>k<sub>j</sub></i>(<i>E</i>) for channel <i>j</i> = 1 or 2
is:
<p class="eqnserif">
<i>k<sub>j</sub></i>(<i>E</i>) = &nu; &times; [ (<i>E</i> &minus; 
&Delta;<sub><i>j</i></sub><i>E</i><sub>0K</sub><sup>&Dagger;</sup>) / <i>E</i> 
]<sup><i>s</i>&minus;1</sup>
<p>
where <i>s</i> identical classical harmonic oscillators of frequency <i>&nu;</i>
make up the dissociating molecule.  There are no reverse activation energies
&Delta;<sub><i>j</i></sub><i>E</i><sup>&Dagger;</sup> =
&Delta;<sub><i>j</i></sub><i>E</i>.  Frequency factor and oscillator count
is same for both channels, which is same assuming &Delta;&Delta;<i>S</i> = 0.
As RRK model tends to underestimate rates, number of oscillators <i>s</i>
is reduced empirically by factor of 2-5.
<!-- ============================================================= -->
<p class="tags">
metal cationization, CID, ESI-MS/MS, FD
<p class="ref">
 <span class="authors">
Madhusudanan KP, Raj K, Bhaduri AP
 </span>
 <span class="year">
(2000)
 </span>
 <span class="jname">
J. Mass Spectrom.
 </span>
 <span class="jvol">
35,
 </span>
 <span class="pages">
901-911.
 </span>
<p class="reftitl">
Effect of metal cationization on the low-energy collision-induced dissociation 
of loganin, <i>epi</i>-loganin and ketologanin studied by electrospray 
ionization tandem mass spectrometry
<p class="Para1">
Loganin and its epimer are glycosides with therapeutic activity in liver disorders.
They form strong [M+Na]<sup>+</sup> ions and weak [M+H]<sup>+</sup> ions in
methanolic ESI-MS.  CID MS/MS was done to obtain [M+H]<sup>+</sup> by inducing
formation of protonated molecules using cone voltage fragmentation of 
[M+NH<sub>4</sub>]<sup>+</sup> ions.  With a cone voltage of &lt; 20 V in
5 mM ammonium acetate, just [M+NH<sub>4</sub>]<sup>+</sup> are observed.  Cone
V was raised to &gt; 25 V to enhance the analysis of [M+H]<sup>+</sup>, observed
at <i>m/z</i> 391.  The protonated aglycone [Agly + H]<sup>+</sup> of the 
two forms has intensities of 100 and 22% respectively.  For the ion 
[Agly + H &minus; H<sub>2</sub>O]<sup>+</sup>, the epi-loganin has a 100% 
intensity and the loganin form is 3%.  The epimers differ in that loganin has
the C-7 OH above the plane as for the C-1 substituent.  This difference may involve
H-bonding involving the C-4 substituent and the C-3/C-4 bond stabilizing 
[Agly + H]<sup>+</sup>.  This stabilization does not occur with the epi-loganin
with C-7 OH below plane.
<p>
With deuterium-exchanged structures the [Agly+D]<sup>+</sup> and 
[Agly+D&minus;D<sub>2</sub>O]<sup>+</sup> structures show at <i>m/z</i> 232 and
212, respectively.  Most likely an exchangeable hydrogen from C-2 OH of the glucose
moves to the aglycone moiety during formation of [Algy + H]<sup>+</sup> ion at
<i>m/z</i> 229 which eliminates anhydroglucose, presented with scheme including
deuterated spectrum analysis.
<p>
The keto-loganin form (keto on C-8 of aglycone with <i>m/z</i> 389 in [M+H]<sup>+</sup>
shows two fragmentation pathways.  Two ions are <i>m/z</i> 227 and 195 for
[Agly + H]<sup>+</sup> and [Agly + H - MeOH]<sup>+</sup>, respectively.
Fragmentation pathways generally involve detection only of the aglycone.
<p>
Low energy CID spectra were taken of [M+Li]<sup>+</sup> and [M+2Li&minus;H]<sup>+</sup>
forms.  Both <sup>6</sup>Li<sup>+</sup> and D-exchanged forms of molecule with 
methanolic LiCl show lithiated aglycone, [Agly + Li]<sup>+</sup> at <i>m/z</i> 235.
Elimination of MeOH is favored equally by both epimers.  The [Agly + Li]<sup>+</sup>
ion at <i>m/z</i> 235 becomes <i>m/z</i> 237 with D exchange, and in cleavege there
is transfer of D to the aglycone.  With Li and not protonated molecule, there is
also  [Glc+Li]<sup>+</sup> ion at <i>m/z</i> 187, the intensity higher for epi-loganin
than loganin.  Presumably chelating of Li ion is by C-1 and C-7 oxygens, a C3-C4
double bound and also C-4 substituent.  In epi-loganin, C-7 OH falls below plane
and cannot perform chelating.  This means more Li<sup>+</sup> is available to the 
sugar for bindding, and so [Glc+Li]<sup>+</sup> has stronger signal.  For D-exchanged,
the [Glc+Li]<sup>+</sup> is at <i>m/z</i> 191 instead of 187, and an exchangable 
hydrogen from C-9 of aglycone likely transfers to glycone moiety.  The D-exchanged
lithiated molecule normall at [Glc+Li&minus;H<sub>2</sub>O]<sup>+</sup> moves up to
<i>m/z</i> 172 from 169.  Thus [Glc+Li&minus;H<sub>2</sub>O is [anhydroglucose + Li]<sup>+</sup>.
[More studies of Li<sup>+</sup> ion interaction explained here.]
<p>
Compared to [M+Li]<sup>+</sup>, intensity of [M+Na]<sup>+</sup> ions for the 
three glycones is weak while fragmentation pathways are similar.  Na<sup>+</sup>
appears to have equal affinity for aglycone and glucose.  The ketologanin form
readily loses MeOH compared to loganin.  As the alkali metal size increases with
K<sup>+</sup>, Rb<sup>+</sup>, and Cs<sup>+</sup>, but the fragmentation pattern
involves only the loss of the association of metal ions.  No fragments associated
with K<sup>+</sup>, Rb<sup>+</sup>, and Cs<sup>+</sup> are seen in spectra.
Weaker bond energies are likely the cause.
<p>
The Ag<sup>+</sup> ion configuration is of a <i>d</i><sup>10</sup> with empty
<i>s</i> orbitnals.  Configuration is similar to Cu<sup>+</sup>.  Silver-bound
methyl glycosides fragment with elimiantion of AgH similar to abstracting 
&alpha;-hydrogen from organic amine complexes of Ag<sup>+</sup> and Cu<sup>+</sup>.
In high enegy CID, Ag<sup>+</sup> bound causes specific carbohydrate fragmentations.
Methanolic solutions of the three molecules in AgNO<sub>3</sub> or with MeOD
for deuterated exchanges were done.  The <i>d</i><sub>3</sub> was made in LiOH
in D<sub>2</sub>O and spiked with AgNO<sub>3</sub> to see the [M+Ag]<sup>+</sup>
ion at <i>m/z</i> 502.  Fragmentation involves silver associated with the aglycone
after which the silver dissociates as AgH.  Some fragment sizes differ between
loganin and the epiform.
<p>
In summary protonation leads to fragmentation with the proton transferring to
aglycone.  Alkali metal and silver cationization cause cleavage at the glycosidic
bond.  With lithiated species, ring cleavages are more probable.
<!-- ============================================================= -->
<p class="tags">
guanidination, MALDI MS
<p class="ref">
 <span class="authors">
Brancia FL, Oliver SG, Gaskell SJ
 </span>
 <span class="year">
(2000)
 </span>
 <span class="jname">
Rapid Commun. Mass Spectrom.
 </span>
 <span class="jvol">
14,
 </span>
 <span class="pages">
2070-2073.
 </span>
<p class="reftitl">
Improved matrix-assisted laser desorption/ionization mass spectrometric 
analysis of tryptic hydrolysates of proteins following guanidination of 
lysine-containing peptides
<p class="Para1">
Ions accelerated in linear mode may show in-flight instability and decompose
(fragment).  They will be detected whether ionized or neutral.  In refletron
mode, neutral products should not be detected.  Thus a comparison of 
linear and reflectron mode intensities should show the difference.
The yeast ADH tryptic digest shows signed derived from Lys- or Arg-containing
peptides.  Lys-containing peptide intensities are greater in linear than
in reflectron mode.  The <i>m/z</i> 2701 ion is the 13th tryptic peptide
and is not observed in reflectron mode.  Since 
<!-- ============================================================= -->
<p class="tags">
structure prediction, membranes
<p class="ref">
 <span class="authors">
M&ouml;ller S, Croning MDR, Apweiler R
 </span>
 <span class="year">
(2001)
 </span>
 <span class="jname">
Bioinformatics
 </span>
 <span class="jvol">
17,
 </span>
 <span class="pages">
646-653.
 </span>
<p class="reftitl">
Evaluation of Methods for the Prediction of Membrane-Spanning Regions
<p class="Para1">
Several well known methods for prediction of MSRs were evaluated:
<ul>
<li>TMHMM 1.0, 2.0, and retrained 2.0 (Sonnhammer et al. 1998 and Krogh
et al. 2001)
<li>MEMSAT 1.5 (Jones et al 1994)
<li>Eisenberg (Eisenberg et al. 1982)
<li>Kyte &amp; Doolittle (1982)
<li>TMAP (Persson and Argos, 1997)
<li>DAS (Cszero et al 1997)
<li>HMMTOP (Tusnady and Simon 1998)
<li>SOSUI (Hirokawa et al 1998)
<li>PHD (Rost et al 1996)
<li>TMpred (Hoffman and Stoffel 1993)
<li>KKD (Klein et al 1985)
<li>ALOM2 (Nakai and Kaneisha, 1992)
<li>Toppred 2 (Claros and Heijne, 1994)
</ul>
<p>
Alom2, Eisenberg, Kyte/Doolittle, TMpred and DAS typically look at properties of 
amino acids to find sub-sequences most likely to span a membrane, with a sliding 
window analysis.  PHD uses a neural network (a nonetheless local approach like 
the others).  Global approaches include HMMTOP and TMHMM usuing circular Hidden 
Markov models. TMAP, Toppred2 and MEMSAT combine features for both local and 
global heuristics.  TMAP can make use of sequence alignments, but this data was 
withheld so that the data was reproducible.
<p>
The test set was 188 proteins with 883 MSRs determined from elucidated 
structures or by fusion experiments.  Correct predictions were judged to be 
those in which at least 9 of the amino acids belonged to the MSR in the 
reference annotation;  an MSR is expected to contain at least 20 residues. 
Ratings included three values:
<ol>
<li>True positive predictions:  that is the percentage of predicated
transmembrane regions assignable to a reference MSR
<li>False negatives:  percentage of reference MSRs not predicted
<li>False positives:  percentage of predicted MSRs not existent in the
reference protein set.
</ol>
<p>
The sidedness of the membrane protein was also tested for reliability, but is 
not evaluated by all methods.  Both mitochondrial inner membrane and plasma 
membrane proteins were tested.  Kyte/Doolittle was evaluated on a 5- and 
9-residue window system (KD5 and KD9).
<p>
A table is presented to show the TP, FP, and FN, and the methods are ranked from 
lowest to highest with respect to errors detected (FP + FN). By this ranking, 
all three versions of TMHMM were better than any other evaluation method. 
MEMSAT was second, but had twice as many errors. ALOM had a low false positive 
rate, and its FP/TP ratio is lower than the one of the TMHMM.
<p>
The average correct prediction of MSRs within a protein was evaluated, and again 
TMHMM versions were the best (with 2 in 3 correct predictions). In 7-8 of 10 
predictions, the sidedness was also correctly determined (the retrained TMHMM 
did better in sidedness).  Toppred, TMAP, and TMpred did worse in prediction 
than in mere random guessing.
<p>
A table of MSRs not presented to the program for training and analysis shows 
that the programs identified before did best in predictions. TMHMM has 80-85% 
correct prediction, with MEMSAT following at 76%. Eisenberg and Kyte/Doolittle 
were runners-up.
<p>
Many programs also interpret the signal/transit peptide sequences as part of the 
hydrophobic spanning core.  These are hydrophobic, but are also removed from the 
mature protein.  A test set of 34 proteins containing cleavable signals, and 8 
proteins with transit peptides were included. Only the ALOM2 program correctly 
predicted that not a single signal sequence was part of the transmembrane 
spanning of the mature protein. This was followed by PHD with one error, and 
Toppred2 with 3 errors. TMHMM 2.0 had 7 errors, responsible for 1 in 6 of the 
false positives this program made.  Kyte/Doolittle predicted 8 mitochondrial 
transit peptides as transmembranous.
<p>
The tendency of the software to overpredict proteins with MSRs was tested by 
submitting known soluble proteins.  While globular proteins have hydrophobic 
sequences that form inner folds, these must not be predicted to be MSRs.  A 
total of 634 cytoplasmic or periplasmic soluble proteins from SWISSPROT release 
38 were tested.  TMHMM had the lowest errors (8, or 1% false annotations), 
followed by SOSUI (19, or 3%).  ALOM2, which has the lowest false positives on 
real MSRs, found 61 soluble proteins with MSRs (a poor 10% rate).
<!-- ============================================================= -->
<p class="tags">
principal components analysis (PCA), mass spectrometry
<p class="ref">
 <span class="authors">
Bryant DK, Mont&eacute; S, Man WJ, Kramer K, Bugelski P, Neville W, White IR,
Camilleri P
 </span>
 <span class="year">
(2001)
 </span>
 <span class="jname">
Rapid Commun. Mass Spectrom.
 </span>
 <span class="jvol">
15,
 </span>
 <span class="pages">
418-427.
 </span>
<p class="reftitl">
Principal Component Analysis of Mass Spectra of Peptides Generated
from The Tryptic Digestion of Protein Mixtures
<p class="abstract">
<b>ABSTRACT</b>.
Principal component analysis (PCA) has been used to analyse mass
spectral peptide profiles obtained from the enzymatic digestion
of standard protein mixtures. Scores and loadings plots clearly
revealed peptide fragments that differentiated one protein mixture
from another. Peptide map search results identified with a high
degree of certainty any additional proteins in these mixtures.
As a proof-of-concept this methodology was applied to hepatic
protein mixtures obtained from rats treated with two hepatotoxic
compounds: methapyriline and SB-219994. Liver proteins were
extracted, pre-separated by one-dimensional polyacrylamide
gel electrophoresis, subjected to tryptic digestion and
analysed by mass spectrometry. Two up-regulated proteins,
glutathione S-transferase with methapyrilene and peroxisomal
bifunctional enzyme with SB-219994, were identified in this manner.
<!-- ============================================================= -->
<p class="tags">
salt cluster ions, FD
<p class="ref">
 <span class="authors">
Hao C, March RE, Croley TR, Smith JC, Rafferty SP
 </span>
 <span class="year">
(2001)
 </span>
 <span class="jname">
J. Mass Spectrom.
 </span>
 <span class="jvol">
36,
 </span>
 <span class="pages">
79-96.
 </span>
<p class="reftitl">
Electrospray ionization tandem mass spectrometric study of salt cluster ions. 
Part 1&mdash;Investigations of alkali metal chloride and sodium
<p class="Para1">
Electrospray ionization was reported by Dole et al in 1968 although used for 
biomolecules much later.  Anacleto et al (1992) investigated protonated water 
clusters and salt clusters of NaF, KF, NaI, KI, RbI, CsI, CsNO<sub>3</sub> and 
CsCO<sub>3</sub> and found they could make calibration standards in positive and 
negative ion mode.  Moini et al reported NaOOCCF<sub>3</sub> could be a 
calibration standard from 100-4000 Th.  ESI may work from either the ion 
evaporation model or by the charge residue model.  Kebarle and Peschke found 
that hydrates of metal [M(H<sub>2</sub>O)<sub>x</sub>]<sup>+</sup> and salt 
clusters [M(MX)<sub>n</sub>]<sup>+</sup> in low <i>m/z</i> region.  In one case, 
the study of alkali metal halides were given as support for charge residue. 
Additional studies with sodium ions clustered with their chlorides, formates, 
and combination formates and acetates although they were in low <i>m/z</i> 
region.  Cases of [Na<sub>x</sub>(NaCl)<sub>n</sub>]<sup>x+</sup> for <i>x</i> = 
1 or 2 have been shown under nanospray.  MS/MS spectra revealing 2+ ions of NaCl 
were recently reported, with magic numbers of molecules, and collision 
fragmentation channels.  In this study are alkali metal chlorides and sodium 
fsalt with &gt;2&plusmn; charge and studied as CID with triple quad and QToF 
instruments.
<p>
In general the total ion current (TIC) increases with flow rate, temperature, 
and capillary voltage.  The distribution of ions however are affected  by cone 
voltage.  The spectrum of a 50 mM NaCl in 50% MeOH is shown with cone V = 10 and 
75 V.  Note that a <q>normal cone voltage</q> (NCV) is thought to be in range of 
45-80 V, while a <q>low cone voltage</q> (LCV) is any setting &lt; 30 V.  Two 
tables are shwonw for positive and negative ion mode.  The data show that for 
NaCl and KCl as singly charged [Me(MeCl)<sub>n</sub>]<sup>+</sup>, the <q>magic 
numbers</q> are <i>n</i> = 1, 4, 9, 13, 22, 28, 37, and 47, although the 
<i>n</i> = 9 cluster for NaCl is not apparent in full-scan mass spectra.  Very, 
very low intensities (barely detectable) can be observed for <i>n</i>  = 14, 15, 
24, and 32 and seen under NCV conditions.  The magic and absent numbers are 
thought to correspond to stable and unstable clusters, respectively, and may 
relate to geometry of salt cluster ions.  Doye and Wales have suggested 
[(NaCl)<sub>n</sub>Cl]<sup>&minus;</sup> for <i>n</i> = 1-35 with <i>n</i> = 13 
and 22 having regular compact structures with greater stability.  A cluster of 
13 NaCl plus an ion (total 27 ions) has a 3 &times; 3 &times; 3 cubic-like 
structure.    For <i>n</i> = 22, the structure shows a 3 &times; 3 &times; 5. 
It is not clear why <i>n</i> = 4 and 9 cluster ions are quite stable according 
to Doye and Wales. Clusters having <i>n</i> = 14, 15, 24 and 32 had particles 
protruding from regular clusters and might be lower stability.
<p>
Under NCV conditions, clusters ranged up to <i>m/z</i> 4000, but under LCV 
conditions, clusters in the low <i>m/z</i> region predominated, usually &lt;
<i>m/z</i> 1700.  At LCV, clusters having magic numbers are no longer dominant.
<p>
For ions in the low <i>m/z</i> region subjected to CID, these ions are actually
multiply charged clusters.  Dications have been reported but multiply charged cluster
ions have not been obesrved before although they have been reported for arginine
solutions.
<p>
In a plot of intensity vs cluster number <i>n</i> for the formula 
Na<sup>+</sup>(NaCl)<sub><i>n</i></sub>, where the cone voltages are 10 (LCV) 
and 75 (NCV) volts, and <i>n</i> ranges from 1 to 29.  Because 
(Na<sup>+</sup>)<sub>2</sub>(NaCl)<sub><i>n</i></sub> dications overlapped the 
monocation clusters, their intensities had to be subtracted.  The chart shows 
that the magic numbers are <i>n</i> = 4, 13, 22, 28 with signficant intensities 
at NCV than at LCV.  Sums of the cluster ion intensities are 138 &times; 
10<sup>6</sup> counts for LCV, and 220 &times; 10<sup>6</sup> counts for NCV.
<p>
The count of energetic collisions (those with vibrational energy increases in
ionic species) that occurs in the space/volume is dependent on cone voltage;
total number of collisions is <em>not</em> dependent on cone voltage however,
only on energy.  With LCV, where sample and extraction cone voltage is ~10 V,
energetic collisions are sharply reduced.  This allows multiply charged cluster
ions that are somewhat labile to survive which are collisionally cool.  With 
higher cone voltage, only the really stable clusters (with magic numbers) are
detected, so their intensities are maintained.  Kebarle and Peschke have favored
the CRM for salt concentrstions &gt; 10 mM although the IEM is not ruled out.
Thus charged droplets are formed first, with salt molecules and solvent molecules
with excess cations in positive mode and anions in negative mode.  Charged droplets
then undergo evaporation and uneven fission until salt cluster ions form.  The 
nascent cluster ions collide with gas molecules in the volume between sample and
extraction cone, and collision energies determine distribution of cluster ions
detected.  When cone V is reduced in steps down to 0 V, cluster ions detected 
represent distribution of droplets in vicinity of sample cone although the distribution
might be modified slightly because of non-uniform ion transmission.  Few if any
energetic collisions occur between ions in the volume between sample and extraction
cones.  For positively charged clusters of NaCl, the nascent ion ensemble is composed 
principally of numerous doubly, triply, and quadruply charged species and two 
additional species with <i>n</i> = 52 and 53, carrying six charges each.
<p>
The preponderance of ions with low <i>m/z</i> is because nascent ions having high
number of charges (<i>z</i>) rather than low mass and single charge.  Under LCV
conditions there are very few +1 charged clusters.  At LCV the intensity of the 
cluster ions at magic number <i>n</i> = 13 indicates that few collisions if any 
are energetic to induce dissociation.  The data indicate that ESI itself does not
produce the clustering conditions, but are conducive to formation of cluser ions
that are thermodynamically less stable.
<p>
With sample cone V increases, ion-neutral collision fraction of total collisions
being energetic or energy-rich is increased.  Ions in nascent ensemble formed
in ESI soruce under partial charge-transfer reactions and fragmentation.  Secondary
ion ensemble is now of higher average <i>m/z</i> because lower number of charges
are associated with molecular clusters.  The sample cone voltage controls the types
of cluster ions that enter the MS.  Cone voltage magnitude determines size (mass)
and charge state of detected cluster ions.  Plot of variations in signal intensity
of Na<sup>+</sup>(NaCl)<sub>13</sub> (a magic number) vs. cone voltage shows that
signal intensity increases with increasing cone V until 55 V, then it decreases due
to collisional dissociation and scattering.
<p>
CID helps to give information on structure through fragmentation pathways and
thermodynamic behavior of gas-phase ions.  With MS/MS analysis, ions are selected
or isolated which fall within a 1-2 Da narrow band of <i>m/z</i> ratios in the 
analysis of the product ions formed.  Product ion spectra were obtained for
both positive and negative ions:  
(NaCl)<sub><i>n</i></sub>Na<sub><i>x</i></sub><sup><i>x</i>+</sup> for 
<i>x</i> = 1-4, 6 and 
(NaCl)<sub><i>n</i></sub>Cl<sub><i>x</i></sub><sup><i>x</i>&minus;</sup> for 
<i>x</i> = 1-3.  Product ion spectra were also checked for potassium chloride
with <i>x</i> = 1-3 for both the cations and anions.  The base peaks were 
clusters having magic numbers in both &plusmn;1 or &plusmn;2 ions.
<p>
The &plusmn;1 ions entering the filter produce &plusmn;1 fragment ions 
naturally, losing MCl paired salt molecules in CID.  For +2 ions with <i>n</i> 
&lt; 28, only +1 are seen (never +2 ions).  Thus the internal energy in 
collision + the energy of charge repulsion enable cluster fragmentation.  When 
<i>n</i> &ge; 28, both +1 and +2 product ions are seen.  Considering the 
<i>n</i> = 28 +2 ion is stable, what would be its structure?  Most likely +2 
ions and other multiply charged ions have a regular flat structure with 
thickness equal to size of single molecule being preferred.  The two faces would 
have a relatively large area and look like a rectangular wafer.  For the NaCl +2 
ion with <i>n</i> = 28, it might look like a 2 &times; 4 &times; 7, with 56 
positive and negative ions and lengths where the sides are approximately equal. 
The two excess sodium ions are connected to chloride ions on opposite corners, 
diagonally opposed for maximum charge separation.  This would be 24.8 &Aring; 
intercharge separation.  The coulombic energy <i>T</i> is related to the 
intercharge distance <i>R</i> such that <i>TR</i> = 14.39 eV &Aring;. Thus the 
coulombic repulsion energy for this dication is <i>T</i> = 0.58&nbsp;eV.
<p>
CID spectra are collected for the following cluster ions: 
(NaCl)<sub>4</sub>Na<sup>+</sup>, 
(NaCl)<sub>21</sub>Na<sub>2</sub><sup>2+</sup>, 
(NaCl)<sub>45</sub>Na<sub>2</sub><sup>2+</sup>, 
(NaCl)<sub>50</sub>Na<sub>3</sub><sup>3+</sup>, and 
(NaCl)<sub>53</sub>Na<sub>6</sub><sup>6+</sup>. Domimant product ions for parent 
(NaCl)<sub>4</sub>Na<sup>+</sup> (<i>m/z</i> 257) are (NaCl)Na<sup>+</sup> 
(<i>m/z</i> 81), (NaCl)<sub>2</sub>Na<sup>+</sup> (<i>m/z</i> 139, and 
(NaCl)<sub>3</sub>Na<sup>+</sup> (<i>m/z</i> 197).  Note that the <i>m/z</i> 257 
value is a [M + 2]<sup>+</sup> value, because it represents the greatest 
abundance of the molecular ion cluster, largely due to the dual isotopes of 
chlorine.  With +1 ions, the only possible energy change is that resulting from 
loss of paired ions.  With (NaCl)<sub>21</sub>Na<sub>2</sub><sup>2+</sup> 
<i>m/z</i> 637, all +1 ions with <i>n</i> 1-20 are observed except for <i>n</i> 
= 14, which is reported to be unstable and seen only in low abundance under NCV 
conditions.  In contrast (NaCl)<sub>13</sub>Na<sup>+</sup> is quite dominant and 
corresponds to a magic number.  It should also be noted that several product 
ions have <i>m/z</i> larger than the precursor. 
(NaCl)<sub>45</sub>Na<sub>2</sub><sup>2+</sup> (<i>m/z</i> 1339) has a spectrum 
of both +1 and +2 fragment ions, where +1 ions have <i>n</i> = 1-4 and 28-44, 
with <i>n</i> = 37 having highest intensity and being a magic number.  Why are 
fragment ions with magic number <i>n</i> = 13 not observed? Answer is addressed 
later.  Ions of 2+ charge and <i>n</i> = 33-44 are also part of fragment 
spectrum.  Four of these 2+ ions with greatest signal intensities have a loss of 
1-4 NaCl molecules from the parent 2+ ion.
<p>
(NaCl)<sub>50</sub>Na<sub>3</sub><sup>3+</sup> (<i>m/z</i> 998) undergoes CID to 
produce +1 ions with 1-4 NaCl molecules and complementary 2+ ions with 46-49 
NaCl clusters.  One fragment is the loss of a single Na<sup>+</sup> to form 
(NaCl)<sub>50</sub>Na<sub>2</sub><sup>2+</sup>.  There are no 3+ fragments.
<p>
There is a 6+ ion of (NaCl)<sub>53</sub>Na<sub>6</sub><sup>6+</sup> (<i>m/z</i> 
539) is simpler:  there are three paths to dissociation.  One is a loss of three 
Na<sup>+</sup> ions  (NaCl)<sub>53</sub>Na<sub>3</sub><sup>3+</sup> (<i>m/z</i> 
1051), another is a loss of two Na<sup>+</sup> (<i>m/z</i> 795), and then a loss 
of six NaCl and two Na<sup>+</sup> for an ion <i>m/z</i> 707.  Low <i>m/z</i> 
fragment ions complementary to these are not seen.  As charge states are 
determined by isotopic peak cluster difference, so long as resolution allows. In 
one particular cluster there are 45 peaks spread out over 29.333 Th.
<p>
Solution conditions such as solvent, concentration, and pH also affect ion 
clustering in ESI-MS.  Earlier attempts to observe cluster ions in alkali metal 
halides may have been due to concentrations.  NaCl at 0.10, 0.05, and 0.01 M 
aqueous in MeOH or MeCN at 25, 50, and 75% levels were studied.  The 
distribution of ions was found to change in going from 10 to 100 mM.  It was
also affected by a changes in solvent.  Total ion current was affected by both
a change in concentration and in solvent combined are necessary.  TIC increases
with concentration and when MeOH or MeCN increases.  A 75% MeOH solution
gives best ion intensities in positive ion mode, and 25% MeOH gives best intensities
in negative ion mode.  Addition of HCl to pH 3 and NaOH to pH 11 in 50 mM
NaCl was done:  at pH 3 positively charged NaCl clusters show the presence of 
one or two HCl molecules.  (NaCl)<sub>17</sub>(HCl)<sub>2</sub>Na<sup>+</sup>
and (NaCl)<sub>18</sub>(HCl)Na<sup>+</sup>.  There is large reduction of ions
seen in the high <i>m/z</i> region compared to pH neutral.  This could be due
to unresolvable combinations of (NaCl)<sub><i>n</i></sub>(HCl)<sub><i>m</i></sub>Na<sup>+</sup>.
Negatively charged clusters were observed at pH 3 and similar to the spectrum
of neutral molecules.  At pH 11 the spectrum of positively and negatively charged
culster is different from the neutral.  There is a significant absence of large
cluster ions in the high <i>m/z</i> range.  Peaks are observed in low <i>m/z</i>
range with ions having mixed clusters of NaCl and NaOH.  Clusters of 
(NaCl)<sub>3-5</sub>(NaOH)<sub>1-2</sub> with either Na<sup>+</sup> or Cl<sup>&minus;</sup>
are prominent in the spectra.  Differences in stability of the cluster are clearly
related to size of the chloride and hydroxyl anions.
<p>
<b>Ionic Radius</b>.  Salt cluster stability is a function of ionic radii and how
it affects solvation energy.  This is more significant when cluster size is <i>n</i> &lt; 10.
In a solvent of 50% MeOH at 50 mM concentration, with NCV (cone= 75 V) conditions, 
alkali metal chlorides (LiCl, RbCl, CsCl) are analyzed.  LiCl for different values of
<i>n</i> (zero charge cluster component) and <i>x</i> for different values of Li<sup>+</sup>
or Cl<sup>&minus;</sup> are similar to NaCl behavior.  In the previous work NaCl and
KCl show a wide distribution of ions up to a limit of <i>m/z</i> 4000, depending on 
spectrometer.  For heavier Rb and Cs, both positive and negative ions are seen as
+1 or &minus;1 ions ranging form <i>n</i> = 13.  Ions at <i>n</i> = 16, 17, 22 and 23
are also seen.  For 2+ ions of Rb and Cs clusters, ions for <i>n</i> &ge; 26 are only
observed.  The count of RbCl and CsCl clusters in absolute intensity are far lower
than for NaCl and KCl cluster ions however.  There is a minor series of cluster ions
containing other cations (Na<sup>+</sup> and K<sup>+</sup>) and may arise from impurities
in the CsCl and RbCl.
<p>
An anionic series was also test against sodium (NaX), where X is iodide 
(I<sup>&minus;</sup>), formate (HCOO<sup>&minus;</sup>), acetate 
(CH<sub>3</sub>COO<sup>&minus;</sup>), nitrite 
(NO<sub>2</sub><sup>&minus;</sup>, or nitrate (NO<sub>3</sub><sup>&minus;</sup>) 
was test with cone V = 75.  NaI produced cluster ions up to <i>m/z</i> 6000, and 
now NaI is used for calibration for these reasons.  Both sodium formate and 
sodium acetate shows clusters quite similar to each other.  The largest ion has 
a <i>n</i> = 31 for formate and <i>n</i> = 32 for acetate.  Under NCV conditions 
cluster ions having the normal magic number of molecules do not dominate the 
mass spectrum.  2+ ions are obaserved with <i>n</i> in region of 25 and one 
identified in positive ion spectra of NaNO<sub>2</sub> and NaNO<sub>3</sub>. 
Also a (NaX)<sub>61</sub>Na<sub>3</sub><sup>3+</sup> ion is found in all 
systems.  The nitrite and nitrate spectra are remarkably similar to each other 
and to sodium formate and sodium acetate.
<p>
Mixes of clusters also occurs.  For example, 
(CH<sub>3</sub>COONa)<sub><i>n</i></sub>(HCOONa)CH<sub>3</sub>COO<sup>&minus;</sup> 
where <i>n</i> = 8-14 is unique; the presence of a single formate. 
Observations for NaCl and LiCl stand out among all others.  LiCl produces no 
fewer than 246 different cluster ions in all, with 167 positively charged, and 
79 negatively charged.  It also produces the cluster ion with the greatest 
number of molecules (<i>n</i> = 152). NaI produces a series of 4+ ions counting 
40 total (<i>n</i> = 65-104, <i>x</i> = 4) unlike no others.  Note this is true 
only in positive ion mode, with 4 Na<sup>+</sup> ions. What is the structure of 
a (NaI)<sub>65</sub>Na<sub>4</sub><sup>4+</sup> cluster? Likely a wafer of 2 
&times; 5 &times; 13 atoms (sodium and iodide), and the four Na<sup>+</sup> 
should be located at corner of one of the two 5 &times; 13 faces.  This provides 
maximum separation.  What about the upper limit 
(NaI)<sub>104</sub>Na<sub>4</sub><sup>4+</sup>? This might be a wafer of 2 
&times; 8 &times; 13 atoms.  The sodium cations would be given the widest 
possible separation.  These structures of 4+ ions cann be regular and compact as 
proposed for (NaCl)<sub>47</sub>Na<sub>4</sub><sup>4+</sup>.  The four sodium 
ions in the <i>n</i> = 47 are part of regular rectangular structure than being 
attached to corners of wafer-like crystal. 
(NaCl)<sub>47</sub>Na<sub>4</sub><sup>4+</sup> is quite stable as it is a CID 
product of the precursor (NaCl)<sub>53</sub>Na<sub>4</sub><sup>4+</sup>,
As for negative cluster ions, those with charge &gt; 2 are relatively rare.
While the 4+ ion (NaI)<sub>65</sub>Na<sub>4</sub><sup>4+</sup> was observed,
not 3+ ion was.
<p>
Three principles can be stated about the formation of cluster ions:
<ol>
<li>10 of the 12 types of sodium salt cluster ions are 1&plusmn; have a maximum
in range of 25-37 molecules of NaX, depending on X.
<li>9 of the 12 types have 2&plusmn; have a minimum of 21-27 molecules, again
depending on X.
<li>5 of the 12 types of cluster ions with 3&plusmn; ions have a minimum of 
61-67 molecules, depending on X.
</ol>
<p>
It can be asserted that when a Na<sup>+</sup> must leave a cluster, it is forced
by cluster instability, when the number of molecules in the cluster falls below
a threshold (critical value).  Suppose <i>n</i> = 70 and <i>x</i> = 3+.  When
<i>n</i> falls below a critical value, <i>x</i> will decrease by 1, where <i>n</i>
will be somewhere between 61 and 67 depending on the anion.  Then dications are
observed.
<p>
Even more interesting observations for LiCl, NaCl, and KCl.  It can be observed
that the maximum value <i>n</i> for <i>x</i> = 1 is half that of the <i>n</i><sub>max</sub>
for <i>x</i> = 2, and a third of that for <i>n</i><sub>max</sub> for <i>x</i> = 3,
so that the <i>m/z</i> value is the same.  It might be a property of the mass transmission
function of the spectrometer or a property of the cluster ions.
<!-- ============================================================= -->
<p class="tags">
salt cluster ions, FD
<p class="ref">
 <span class="authors">
Hao C, March RE
 </span>
 <span class="year">
(2001)
 </span>
 <span class="jname">
J. Mass Spectrom.
 </span>
 <span class="jvol">
36,
 </span>
 <span class="pages">
500-521.
 </span>
<p class="reftitl">
Electrospray ionization tandem mass spectrometric study of salt cluster ions. 
Part 2&mdash;Salts of polyatomic acid groups and of multivalent metals
<p class="Para1">
A Micromass Quattro LC triple-stage quadrupole (TSQ) equipped with Z-spray interface
is used.  Salt solutions are pumped with Harvard Model 11 syringe pump at 5 &micro;l/min.
Source block is 90&deg; and desolvation 150&deg;C.  Capillary voltage is 3000 V and
cone = 60 V.  Argon @ 10 psi is the collision gas for MS/MS work.  Beam of 
mass-selected ions is attenuated by about half.  A 5 Th <i>m/z</i> range was isolated
for MS/MS work.  Collision voltage is adjusted  to an optimum.
<p>
<b>Results</b>.  A 50 mM CaCl<sub>2</sub> solution is scanned in positive and negative
ion mode in range <i>m/z</i> 0-2500.  The ions present show positive clusters 
of the form (CaCl<sub>2</sub>)<sub><i>n</i></sub>(CaCl)<sub><i>x</i></sub><sup><i>x</i>+</sup>,
where <i>x</i> = 1-3.  In negative ion mode, the form is
(CaCl<sub>2</sub>)<sub><i>n</i></sub>(Cl)<sub><i>y</i></sub><sup><i>y</i>&minus;</sup>
where <i>y</i> = 1-3.  All cluster ions are identifed based on <i>m/z</i> ratio and
intuition based on chemistry of material.  Fragmentation studies by MS/MS also give
clues about the structure.  Five precursor ions were selected for CID fragmentation:
(CaCl<sub>2</sub>)<sub>3</sub>(CaCl)<sup>+</sup>, 
(CaCl<sub>2</sub>)<sub>5</sub>(CaCl)<sup>+</sup>,
(CaCl<sub>2</sub>)<sub>10</sub>(CaCl)<sub>2</sub><sup>2+</sup> (isobar to previous),
(CaCl<sub>2</sub>)<sub>13</sub>(CaCl)<sub>2</sub><sup>2+</sup>, and
(CaCl<sub>2</sub>)<sub>25</sub>(CaCl)<sub>3</sub><sup>3+</sup>.  The fragmentation
was optimized to retain some of the precursor in the spectrum.  Product ions appearing
for the <i>n</i> = 3/<i>x</i> = 1+ ion include <i>n</i> = 1 and 2/<i>x</i> = 1+, thus
losing CaCl<sub>2</sub> molecules successively.  For <i>n</i> = 5/<i>x</i> = 1+ and
isobaric <i>n</i> = 10/<i>x</i> = 2+, the products are <i>n</i> = 1-8/<i>x</i> = 1+; no
dications.  For <i>n</i> = 13/<i>x</i> = 2+, again only 1+ cations with <i>n</i> = 1-12,
indicating the energy to knock off CaCl<sub>2</sub> one at a time.  It thus appears
that there is a critical value / threshold of <i>n</i> = 13 for the dication form to
be possible.  For <i>n</i> = 25/<i>x</i> = 3+, 1+ cations with <i>n</i> = 1-4 are
observed and 2+ ions with <i>n</i> = 10-24 are observed.  In addition, 3+ ions with
<i>n</i> = 22-24 are observed, indicating a critical threshold for 22 ions before a
CaCl<sup>+</sup> is discharged.  All this is similar to the results for MX, where 
M is alkali metals.  The table below summmarizes the 50 mM CaCl<sub>2</sub> data:
<table class="td-center">
<tr><th rowspan="2">Charge number <br> <i>x</i> or <i>y</i>
 <th colspan="2">Observed <i>n</i>
<tr><th>Positive ions <br> (CaCl<sub>2</sub>)<sub><i>n</i></sub>(CaCl)<sub><i>x</i></sub><sup><i>x</i>+</sup>
  <th>Positive ions <br> (CaCl<sub>2</sub>)<sub><i>n</i></sub>(Cl)<sub><i>y</i></sub><sup><i>y</i>&minus;</sup>
<tr><td>1 <td>1-17 <td>1-22
<tr><td>2 <td>9-33 <td>9-39
<tr><td>3 <td>25-50 <td>49-65
</table>
<p>
For 50 mM CuCl<sub>2</sub> the ion pattern in MS mode is more complex.  Formations of
ions fitting the following five patterns were observed:
<ul>
<li>(CuCl<sub>2</sub>)<sub><i>n</i></sub>(CuCl)Cu<sup>+</sup>
<li>(CuCl<sub>2</sub>)<sub><i>n</i></sub>(CuCl)<sup>+</sup>
<li>(CuCl<sub>2</sub>)<sub><i>n</i></sub>Cu<sup>+</sup>
<li>(CuCl)<sub><i>n</i></sub>(CuCl)<sup>+</sup> / 
(CuCl<sub>2</sub>)(CuCl)<sub><i>n</i> &minus; 1</sub>Cu<sup>+</sup>
<li>(CuCl)<sub><i>n</i></sub>Cu<sup>+</sup>
</ul>
<p>
Thus in positive mode, there is are losses of chloride anions to form CuCl<sup>+</sup>
of Cu<sup>+</sup>.    The table below summarizes the cluster formula and observed
<i>n</i>:

<table class="td-center">
<tr><th>Mode  <th>Cluster formula  <th>Observed <i>n</i>
<tr><td rowspan="5">Positive
  <td>(CuCl)<sub><i>n</i></sub>Cu<sup>+</sup>  <td>1-6
<tr><td>(CuCl)<sub><i>n</i></sub>(CuCl)<sup>+</sup> / 
(CuCl<sub>2</sub>)(CuCl)<sub><i>n</i> &minus; 1</sub>Cu<sup>+</sup> <td>1-4
<tr><td>(CuCl<sub>2</sub>)<sub><i>n</i></sub>Cu<sup>+</sup> <td>1-17
<tr><td>(CuCl<sub>2</sub>)<sub><i>n</i></sub>(CuCl)<sup>+</sup><td>1-17
<tr><td>(CuCl<sub>2</sub>)<sub><i>n</i></sub>(CuCl)Cu<sup>+</sup> <td>1-17
<tr><td rowspan="4">Negative
  <td>(CuCl)<sub><i>n</i></sub>Cl<sup>&minus;</sup>  <td>1-3
<tr><td>(CuCl<sub>2</sub>)<sub><i>n</i></sub>Cl<sup>&minus;</sup>  <td>1-16
<tr><td>(CuCl<sub>2</sub>)<sub><i>n</i></sub>(CuCl)Cl<sup>&minus;</sup>  <td>1-12
<tr><td>(CuCl<sub>2</sub>)<sub><i>n</i></sub>(CuCl)<sub>2</sub>Cl<sup>&minus;</sup>  <td>1-14
</table>
<p>
Note that both positive and negative mode clusters featured CuCl, indicating that
Cu has been reduced from Cu(II) to Cu(I).
<p>
Product ion spectra were collected on the following precursors:
<ol>
<li>(CuCl<sub>2</sub>)(CuCl)<sub>3</sub>Cu<sup>+</sup>
<li>(CuCl<sub>2</sub>)<sub>5</sub>(CuCl)Cu<sup>+</sup>
<li>(CuCl<sub>2</sub>)<sub>11</sub>(CuCl)<sup>+</sup>
</ol>
<p>
For precursor 1,  there are losses of one, two, and three CuCl molecules,
and also losses of (CuCl)<sub>1-2</sub>(CuCl<sub>2</sub>).  In addition
ions appear that represent single Cl atom loss and the formation of 
three Cu<sup>+</sup> ions (Cu<sup>+</sup> appears at <i>m/z</i> 63.4).
A particular problem is a distinction between empirical (ion) formula and 
molecular formula.  The distinction might need to be resolved by understanding
energy states.  A particular ion that can be shown to be 
(CuCl)<sub>4</sub>(CuCl)<sup>+</sup>
might also be represented as (CuCl<sub>2</sub>)(CuCl)<sub>3</sub>Cu<sup>+</sup>.
<p>
Precursor 2 was determined based on a loss of five CuCl<sub>2</sub> and one
CuCl moiety.  It loses two Cl atoms and produces no free Cu<sup>+</sup> ions.
<p>
The species determined to be precursor 3 was determined in the same way, observing
successive CuCl<sub>2</sub> losses.
<p>
How does the reduction occur?  Perhaps at the expense of chloride ion in the gas
phase, which forms a radical Cl&bull; or produces &frac12; Cl<sub>2</sub> for a 
thorough oxidation.  It was observed with CuCl<sub>2</sub> sprays that there was a
discharge between the ESI capillary and sampling cone, visible to naked eye, and 
8 mm long, intensely blue in region of sample cone.  The discharge was most intense
at 3000 V, and loses intensity as the potential is decreased (at 2500 V it is no
longer visible to dark-adapted eye).  This discharge is likely attributable to 
the first negative system of nitrogen, N<sub>2</sub><sup>+</sup> 
(<i>B</i><sup>2</sup> &sum;<sub><i>u</i></sub><sup>+</sup> &rarr; 
<i>X</i><sup>2</sup> &sum;<sub><i>g</i></sub><sup>+</sup>).  This is a common component
of aurorae. Ground state nitrogen forms the first state as follows:
<p class="eqnserif">
N<sub>2</sub> + e<sup>&minus;</sup> &harr; N<sub>2</sub><sup>+</sup> 
(<i>B</i><sup>2</sup> &sum;<sub><i>u</i></sub><sup>+</sup>) + 2 e<sup>&minus;</sup>
<p>
Thus gaseous electrons ionize dinitrogen.  A blue emission occurs when there is
a transition from excited to ground state ion:
<p class="eqnserif">
N<sub>2</sub><sup>+</sup>  (<i>B</i><sup>2</sup> &sum;<sub><i>u</i></sub><sup>+</sup>) 
&harr; 
N<sub>2</sub><sup>+</sup>  (<i>X</i><sup>2</sup> &sum;<sub><i>g</i></sub><sup>+</sup>) 
+ <i>h&nu;</i>
<p>
The (0,0) transition at 3916 &Aring; accounts for the blue flame.  Differences between
CaCl<sub>2</sub> and CuCl<sub>2</sub> are attributable to <i>d</i> orbital chemistry.
Valence shell configuration is 4<i>s</i><sup>2</sup> for Ca and 
3<i>d</i><sup>10</sup>4<i>s</i><sup>1</sup> for Cu.  When ionized to 2+ ions, 
Ca<sup>2+</sup> is 3<i>s</i><sup>2</sup>3<i>p</i><sup>6</sup>, which is quite stable.  In
contrast Cu<sup>2+</sup> is 3<i>d</i><sup>9</sup>, less stable than 3<i>d</i><sup>10</sup>,
which would be Cu<sup>+</sup>.  However, Cu<sup>+</sup> is not stable in aqueous
solution (2 Cu<sup>+</sup> &harr; Cu<sup>2+</sup> (aq) + Cu(s)).  But in electrospray
in the gas phase, it will form the more energetically favorable state CuCl.
<p>
<b>ESI-MS of Na<sub><i>x</i></sub>Y, where <i>x</i> = 1 or 2 and Y = CO<sub>3</sub><sup>2&minus;</sup>,
HCO<sub>3</sub><sup>&minus;</sup>, H<sub>2</sub>PO<sub>4</sub><sup>&minus;</sup>,
and HPO<sub>4</sub><sup>&minus;</sup></b>.  A 50 mM NaCO<sub>3</sub> solution 
showed a spectrum of cluster ions quite similar to CaCl<sub>2</sub> with a pattern
shown in the table, with +1 and +2 ions in positive mode and &minus;1, &minus;2
and &minus;3 ions in negative mode.
<table class="td-center">
<tr><th rowspan="2">Charge number <br> <i>x</i> or <i>y</i>
 <th colspan="2">Observed <i>n</i>
<tr><th>Positive ions <br> (Na<sub>2</sub>CO<sub>3</sub>)<sub><i>n</i></sub>(Na)<sub><i>x</i></sub><sup><i>x</i>+</sup>
  <th>Negative ions <br> (Na<sub>2</sub>CO<sub>3</sub>)<sub><i>n</i></sub>(NaCO<sub>3</sub>)<sub><i>y</i></sub><sup><i>y</i>&minus;</sup>
<tr><td>1 <td>1-28 <td>1-25
<tr><td>2 <td>15-51 <td>9-43
<tr><td>3 <td>&mdash; <td>19-63
</table>
<p>
When 50 mM NaHCO<sub>3</sub> was tested in positive and negative ion mode, peaks in 
respective positive and negative ion modes appeared similarly to those in NaCO<sub>3</sub>
indicating the common nature of the chemistry.  More peaks were apparent in the 
hydrogencarbonate though.  Selection for MS/MS by CID shows that the additional minor
peaks were because of +1 and more charge state cluster ions with several NaHCO<sub>3</sub>
molecules.  Note that there is a 22 Da difference between the lower mass hydrogencarbonate
compared to the disodium form.
<p>
What the bicarbonate solution behavior shows is that there is a change of pH 
during electrospray.  The quadrupole filter was set for parent ions around 
<i>m/z</i> 1083, 1048, 1028, and 1013, and product ion spectra recorded.  For 
parent <i>m/z</i> 1083 fragments fit the patterns 
(Na<sub>2</sub>CO<sub>3</sub>)<sub>1-37</sub>Na<sup>+</sup>, 
(Na<sub>2</sub>CO<sub>3</sub>)<sub>29</sub>Na<sub>2</sub><sup>2+</sup>, and 
(Na<sub>2</sub>CO<sub>3</sub>)<sub>39</sub>Na<sub>3</sub><sup>3+</sup>.  The 
precursor must have been composed of what amounted to four isobaric species: 
(Na<sub>2</sub>CO<sub>3</sub>)<sub>10</sub>Na<sup>+</sup>, 
(Na<sub>2</sub>CO<sub>3</sub>)<sub>20</sub>Na<sup>2+</sup>, 
(Na<sub>2</sub>CO<sub>3</sub>)<sub>30</sub>Na<sup>3+</sup>, and 
(Na<sub>2</sub>CO<sub>3</sub>)<sub>40</sub>Na<sup>4+</sup>.  The products then 
represent a common loss of (Na<sub>2</sub>CO<sub>3</sub>)Na<sup>+</sup> from all the 
precursors (<i>n</i> = 20, 30, 40 for <i>x</i> = 2, 3, 4) in order to 
produce <i>n</i> = 19, 29, 39 for <i>x</i> = 1, 2, 3, respectively.  With bicarbonate
the ion cluster may be 
(Na<sub>2</sub>CO<sub>3</sub>)<sub>16</sub>(NaHCO<sub>3</sub>)<sub>3</sub>Na<sub>2</sub><sup>2+</sup>
and the product will be 
(Na<sub>2</sub>CO<sub>3</sub>)<sub>15</sub>(NaHCO<sub>3</sub>)<sub>3</sub>Na<sup>+</sup>
with the loss of (Na<sub>2</sub>CO<sub>3</sub>)Na<sup>+</sup>.
<p>
Filtering at <i>m/z</i> 1013 &plusmn; 5 Th, this takes as precursor both
(Na<sub>2</sub>CO<sub>3</sub>)<sub>28</sub>Na<sub>3</sub><sup>3+</sup> (<i>m/z</i> 1012)
and (Na<sub>2</sub>CO<sub>3</sub>)<sub>35</sub>(NaHCO<sub>3</sub>)<sub>3</sub>Na<sub>4</sub><sup>4+</sup>
(<i>m/z</i> 1014).  Product spetra show +1 ions 
(Na<sub>2</sub>CO<sub>3</sub>)<sub>5-9,12-14,21-27</sub>Na<sup>+</sup> and
(Na<sub>2</sub>CO<sub>3</sub>)<sub>27</sub>Na<sub>2</sub><sup>2+</sup>.
From (Na<sub>2</sub>CO<sub>3</sub>)<sub>35</sub>(NaHCO<sub>3</sub>)<sub>3</sub>Na<sub>4</sub><sup>4+</sup>
the ions (Na<sub>2</sub>CO<sub>3</sub>)<sub>34</sub>(NaHCO<sub>3</sub>)<sub>3</sub>Na<sub>3</sub><sup>3+</sup>,
(Na<sub>2</sub>CO<sub>3</sub>)<sub>33</sub>(NaHCO<sub>3</sub>)<sub>3</sub>Na<sub>2</sub><sup>2+</sup>,
(Na<sub>2</sub>CO<sub>3</sub>)<sub>35</sub>Na<sub>2</sub><sup>2+</sup>, and
(Na<sub>2</sub>CO<sub>3</sub>)<sub>33-35</sub>Na<sup>+</sup> are formed from a combination
of molecules and atoms lost:  these combinations include sodium carbonate, sodium
bicarbonate, and an attached sodium ion.  The main series of ions are composed of 
sodium carbonate molecules whether the solution is carbonate or bicarbonate.
<p>
So why does bicarbonate form mostly carbonate?  The pH of 50 mM NaHCO<sub>3</sub> is
8.42 and that of 50 mM Na<sub>2</sub>CO<sub>3</sub> is 11.21.  Bicarbonate forms
carbonate by the abstraction of the proton with hydroxide.  Thus a pH increase will
favor carbonate formation.  Clearly in electrospray, there is an increase in pH
causing this conversion.
<p>
If this is true, then what about another polyprotic acid, for example sodium 
phosphate. The full spectrum of 50 mM NaH<sub>2</sub>PO<sub>4</sub> in both 
positive and negative mode was obtained.  Nearly two dozen ions were evaluated 
as clusters in positive ion mode of NaPO<sub>3</sub>, 
NaH<sub>2</sub>PO<sub>4</sub>, Na<sub>2</sub>HPO<sub>4</sub>, 
Na<sub>3</sub>PO<sub>4</sub>, and Na<sup>+</sup>; and in negative ion mode of 
NaPO<sub>3</sub>, NaH<sub>2</sub>PO<sub>4</sub>, Na<sub>2</sub>HPO<sub>4</sub>, 
Na<sub>3</sub>PO<sub>4</sub>, PO<sub>3</sub><sup>&minus;</sup> and 
H<sub>2</sub>PO<sub>4</sub>.  Included in the spectrum was 
(H<sub>3</sub>PO<sub>4</sub>)Na<sup>+</sup> and 
(H<sub>3</sub>PO<sub>4</sub>)<sub>2</sub>Na<sup>+</sup>. In the negative ion 
mode, (HPO<sub>3</sub>)PO<sub>3</sub><sup>&minus;</sup>, 
(H<sub>3</sub>PO<sub>4</sub>)PO<sub>3</sub><sup>&minus;</sup>, 
(H<sub>3</sub>PO<sub>4</sub>)H<sub>2</sub>PO<sub>4</sub><sup>&minus;</sup>, and 
(H<sub>3</sub>PO<sub>4</sub>)<sub>2</sub>H<sub>2</sub>PO<sub>4</sub><sup>&minus;</sup>
The formation of H<sub>3</sub>PO<sub>4</sub> indicates that pH is falling within the
electrospray medium.  Product ion spectra were obtained for the following precursors:
<ul style="text-align:left;">
<li>(NaPO<sub>3</sub>)(Na<sub>2</sub>HPO<sub>4</sub>)Na<sup>+</sup>
<li>(NaPO<sub>3</sub>)(NaH<sub>2</sub>PO<sub>4</sub>)<sub>2</sub>(Na<sub>3</sub>PO<sub>4</sub>)Na<sup>+</sup>
<li>(NaPO<sub>3</sub>)<sub>2</sub>(Na<sub>2</sub>HPO<sub>4</sub>)(Na<sub>3</sub>PO<sub>4</sub>)Na<sup>+</sup>
and its isobar (NaPO<sub>3</sub>)<sub>4</sub>(Na<sub>2</sub>HPO<sub>4</sub>)<sub>2</sub>(Na<sub>3</sub>PO<sub>4</sub>)<sub>2</sub>Na<sub>2</sub><sup>2+</sup>
</ul>
<p>
(NaPO<sub>3</sub>)(Na<sub>2</sub>HPO<sub>4</sub>)Na<sup>+</sup> (<i>m/z</i> 
266.8) show fragmentations involving neutral losses of 
Na<sub>2</sub>HPO<sub>4</sub> and one or two NaPO<sub>3</sub> moieties.  The 
loss of two NaPO<sub>3</sub> moieties leaves an ion looking like 
(NaOH)Na<sup>+</sup>.  For precursor 
(NaPO<sub>3</sub>)(NaH<sub>2</sub>PO<sub>4</sub>)<sub>2</sub>(Na<sub>3</sub>PO<sub>4</sub>)Na<sup>+</sup> 
(<i>m/z</i> 529.1) were losses of H<sub>2</sub>O, 
H<sub>3</sub>PO<sub>4</sub>, NaPO<sub>3</sub> and NaH<sub>2</sub>PO<sub>4</sub>. 
The precursor 
(NaPO<sub>3</sub>)<sub>2</sub>(Na<sub>2</sub>HPO<sub>4</sub>)(Na<sub>3</sub>PO<sub>4</sub>)Na<sup>+</sup> 
(<i>m/z</i> 532.7) involved neutral losses of 
NaPO<sub>3</sub>, Na<sub>2</sub>HPO<sub>4</sub>, and 
Na<sub>3</sub>PO<sub>4</sub>.  For fragmentation in negative ion mode, the 
precursor 
(NaPO<sub>3</sub>)<sub>3</sub>(Na<sub>2</sub>HPO<sub>4</sub>)PO<sub>3</sub><sup>&minus;</sup> 
there were losses of Na<sub>2</sub>HPO<sub>4</sub> and 
NaPO<sub>3</sub>. 
(NaPO<sub>3</sub>)<sub>2</sub>(NaH<sub>2</sub>PO<sub>4</sub>)<sub>2</sub>(Na<sub>2</sub>HPO<sub>4</sub>)PO<sub>3</sub><sup>&minus;</sup> 
fragmentation involved 
H<sub>2</sub>, NaPO<sub>3</sub>, NaH<sub>2</sub>PO<sub>4</sub>, and the disodium 
phosphate too.  Noticeably there is a reduction of 
PO<sub>3</sub><sup>&minus;</sup> (<i>m/z</i> 78.9) to 
PO<sub>2</sub><sup>&minus;</sup> (<i>m/z</i> 63.1).  The negative ion MS/MS 
spectrum of 
(H<sub>3</sub>PO<sub>4</sub>)H<sub>2</sub>PO<sub>4</sub><sup>&minus;</sup> 
(<i>m/z</i> 194.6) shows a very strong intensity at 
H<sub>2</sub>PO<sub>4</sub><sup>&minus;</sup> (96.8) with much smaller 
intensities for ions PO<sub>3</sub><sup>&minus;</sup> (79.0), 
(HPO<sub>3</sub>)H<sub>2</sub>PO<sub>4</sub><sup>&minus;</sup> (176.9), and 
(HPO<sub>3</sub>)PO<sub>3</sub><sup>&minus;</sup> (158.0).
<p>
Certain patterns emerge.  NaPO<sub>3</sub> molecules were more prominent in
cluster ions when NaH<sub>2</sub>PO<sub>4</sub> solutions were sprayed, 
while Na<sub>3</sub>PO<sub>4</sub> moieities were prominent in ions in which
Na<sub>2</sub>HPO<sub>4</sub> were sprayed.  In ions where NaOH was also part
of the cluster, it is clear that pH increased during electrospray.  What is 
clear is that if monosodium phosphate is used, the pH goes down in ESI, while
in disodium phosphate, the pH goes up.  pH change may result from two mechanisms.
One is by the electrochemical process accompanying ESI.  Solvent composition
affects the potential and type of electrolytic events.  It also depends on if
there is positive mode, when oxidation is likely to occur, and possible reactions
lowering pH.  With negative ion mode, reduction is likely to occur and cause
increases to pH.
<p>
In positive ion mode, where electrons are pulled from solution, water 
electrolysis occurs to produce protons:  2 H<sub>2</sub>O &rarr; O<sub>2</sub>
+ 4 H<sup>+</sup> + 4 e<sup>&minus;</sup> where <i>E</i><sub>0</sub> = 1.229 V.
The concentration of electrons can be computed with Faraday's law:
[H<sup>+</sup>] = <i>i</i><sub>electrospray</sub> / <i>nFv</i><sub>f</sub>,
where the factors of electrospray current, molar equivalent of electrons
to produce 1 mol H<sup>+</sup> (<i>n</i> = 1 in this case), and <i>F</i>
= 96,480 C mol<sup>&minus;1</sup>.  The factor <i>v</i><sub>f</sub> is the 
solution flow rate.  At 5 &micro;l/min and current estimated to be 0.1 &micro;A,
the [H<sup>+</sup>] = 12.4 &micro;M, producing a final pH 4.9, a decrease
of 2.1 units.  What about pH as solvent evaporates?  If monosodium phosphate
is 1 mM, and the ratio of undissociated salt molecules to solvent is 
~1:55000, then the bulk pH solution wil be 5.1 if <i>K</i><sub>h</sub> =
1.3 &times; 10<sup>&minus;12</sup> and K<sub>2</sub> = 6.3 &times; 10<sup>&minus;8</sup>.
As droplet evaporation occurs in hot gas, the solubiliity limit of 372 g
per 100 ml at 100&deg; would be reached at 30.9 M, and the calculated pH
would decrease to ~2.86 (neglecting activity coefficients).  The ratio would 
of undissociated salt to solvent molecules is ~1:2.  For disodium phosphate,
the solubility is 117 g / 100 ml at 80&deg; and would be a concentration of 
6.5 M, with undissociated salt to solvent molecues being ~1:8.
<p>
In computing the proton and hydroxide forming constants in the equilibrium
reactions for phosphoric acid, dihydrogen phospate, monohydrogen phosphate, 
the dissociation of NaH<sub>2</sub>PO<sub>4</sub> while hydrolysis is the 
dominant process for Na<sub>2</sub>HPO<sub>4</sub>.  Thus a 50 mM 
NaH<sub>2</sub>PO<sub>4</sub> will be acidic and a 0.05 M Na<sub>2</sub>HPO<sub>4</sub>
will be basic:  their pH is 4.71 and 9.30, respectively.  What happens as
the solvent evaporates is that pH &lt; 7 will decrease more, pH &gt; 7 will
increase more, and when pH = 7, there is no change.  Relative proportions
of the four phosphate species (H<sub>3</sub>PO<sub>4</sub>, 
H<sub>2</sub>PO<sub>4</sub><sup>&minus;</sup>, HPO<sub>4</sub><sup>2&minus;</sup>,
and PO<sub>4</sub><sup>3&minus;</sup>) as a function of pH can be computed
based on the equilibrium constants.  When pH &lt; 7, clearly the more acidic
forms dominate (0, &minus;1);  at pH &gt; 7, then the more alkaline forms
(2&minus;, 3&minus;).  When 50 mM monosodium is present, NaPO<sub>3</sub>,
NaH<sub>2</sub>PO<sub>4</sub>, and H<sub>3</sub>PO<sub>4</sub> dominate the
clusters by 53 to 31 for positive ions and 43 to 18 for negative ions.
<!-- ============================================================= -->
<p class="tags">
salt cluster ions, FD
<p class="ref">
 <span class="authors">
Charles L, P&eacute;pin D, Gonnet F, Tabet J-C
 </span>
 <span class="year">
(2001)
 </span>
 <span class="jname">
J. Am. Soc. Mass Spectrom.
 </span>
 <span class="jvol">
12,
 </span>
 <span class="pages">
1077-1084.
 </span>
<p class="reftitl">
Effects of Liquid Phase Composition on Salt Cluster Formation in Positive Ion 
Mode Electrospray Mass Spectrometry: Implications for Clustering Mechanism in 
Electrospray
<p class="Para1">
Another article on ion cluster formation, this time with potassium bromate.
Positive ion mode was assessed, and from one to four cations (K<sup>+</sup>)
were associated with the cluster.  Effect of seven solvents differing in 
dielectric constant and viscosity were tested:  lower dielectric and 
higher viscosity favored clustering, but limited the charge state.  Reduction
in cluster charge state during desolvation does not correlate with solvent
cation affinity.  Ion distribution observations show that with increasing
cluster size comes increasing charge state.
<!-- ============================================================= -->
<p class="tags">
principal components analysis (PCA), clustering, gene expression analysis
<p class="ref">
 <span class="authors">
Yeung KY, Ruzzo WL
 </span>
 <span class="year">
(2001)
 </span>
 <span class="jname">
Bioinformatics
 </span>
 <span class="jvol">
17,
 </span>
 <span class="pages">
763-774.
 </span>
<p class="reftitl">
Principal Component Analysis for Clustering Gene Expression Data
<p class="abstract">
<b>ABSTRACT</b>.
MOTIVATION: There is a great need to develop analytical methodology
to analyze and to exploit the information contained in gene expression
data. Because of the large number of genes and the complexity of
biological networks, clustering is a useful exploratory technique
for analysis of gene expression data. Other classical techniques,
such as principal component analysis (PCA), have also been applied
to analyze gene expression data. Using different data analysis
techniques and different clustering algorithms to analyze the
same data set can lead to very different conclusions. Our goal
is to study the effectiveness of principal components (PCs)
in capturing cluster structure. Specifically, using both real
and synthetic gene expression data sets, we compared the
quality of clusters obtained from the original data to
the quality of clusters obtained after projecting onto
subsets of the principal component axes. RESULTS: Our
empirical study showed that clustering with the PCs
instead of the original variables does not necessarily
improve, and often degrades, cluster quality. In particular,
the first few PCs (which contain most of the variation in
the data) do not necessarily capture most of the cluster
structure. We also showed that clustering with PCs has
different impact on different algorithms and different
similarity metrics. Overall, we would not recommend PCA
before clustering except in special circumstances.
<!-- ============================================================= -->
<p class="tags">
mass spectrometry, carbohydrates
<p class="ref">
 <span class="authors">
Harvey DJ
 </span>
 <span class="year">
(2001)
 </span>
 <span class="jname">
Proteomics
 </span>
 <span class="jvol">
1,
 </span>
 <span class="pages">
311-328.
 </span>
<p class="reftitl">
Identification of Protein-Bound Carbohydrates by Mass Spectrometry
<p class="Para1">
<b>REVIEW</b>.
The higher eukaryotes have three types of glycans.  N-linked glycans
modify Asn using N-acetylglucosamine (GlcNAc) on the motif NX(S/T),
where X is any residue but Pro.  It also modifies proteins via Ser or Thr
in an O-linked glycan.  There are also oligosaccharides that modify
glycosylphosphatidylinositol:  GPI anchors.  In Gram-positive and archaea
bacteria, glucose attaches to Asn, galactose to Thr, GlcNAc to Asn,
rhamnose to Asn, and Glc or Gal to Tyr.  The N-linked sugars typically
form a tri-mannosyl chitobiose core with antennae attaching to each of the
two outer mannoses.  Antennae extend with N-acetyllactosamine (1 or more)
and then terminate with sialic acid or &alpha;-linked Gal.  Three general
types of N-linked glycans are usual.  A <q>high-mannose</q> in which
only mannose form the antenna.  <q>Hybrid glycans</q> are those
in which one antenna is processed to remove the mannose.  <q>Complex
glycans</q> represent processing of both antenna.
O-linked glycans are more diverse, from monosaccharides to large sulfated
polysaccharides, and no common core or consensus.
<p>
As for structural analysis techniques, NMR does an almost complete job
although it is less sensitive than MS and chromatographic analysis.
Proteomic scale samples are insufficient for NMR workup.  The steps usually
involved are determining the sites of glycosylation on the polypeptide,
determining the extent of occupation (occupancy) at each site, determining
structures and amounts of each glycan at each site, and determing total
count of individual species (glycoforms).  Trypsin and other polypeptide
digestion techniques are used in the first two determinations.  The
third determination is done on glycopeptides or released glycans.  The
last determination is difficult because of the complexity of mixtures.
<p>
MS can do pretty much anything but separate isomers.  However it can be
combined with HPLC to an ESI source to get glycan profiles.  More sensitive
than MS on an HPLC system is fluorescence detection of tagged glycans.
Before 1988, MS work made use of electron ionization, typically with
GC-based separations.  Intact glycans are too large to be examined directly.
In-beam EI has been used for large glycolipids.  FAB was popular for intact
glycans, but sensitivity is reduced by high matrix background.
<p>
Neutral sugars can be analyzed in MALDI MS as [M + Na]<sup>+</sup> ions in
the 1 pmole range.  Glycans with masses less than 1 kDa will show less
intensity than the larger glycans.  In both positive and negative ion modes,
acidic glycans show a lower signal than do neutral sugars.  Other peaks
in the spectra may result from sodium or potassium salts as well as
adducts of the free acidic sugar.  With negative ion spectra a proton is
lost from anionic groups.  Sialic acid groups are unstable and decompose
in flight.  Linear mode analysis may be better than reflectron mode since
post-source decay is more significant in the latter.
<p>
DHB is the more common matrix producing strong [M + Na]<sup>+</sup> ions.
Super DHB is 9:1 mix of DHB:2-OH-5-methoxyDHB.  1-hydroxyquinolone also
is an additive that has some effect.  Recrystallization of sample from
EtOH has worked as well.  The idea is to reduce crystal size, and EtOH
may promote mixing of glycan with matrix.  Spermine addition may improve
negative ion spectra of anionic glycans by binding salt.  Other matrices:
5-chloro-2-mercaptobenzithiazole, various &beta;-carbolines, 3-aminoquinoline,
arabinosazone, and 2-(4&prime;-hydroxyphenylazo)benzoic acid.  3-AQ supposedly
gives lower background, increased resolution and is better for sialylated
sugars than DHB.  HABA gives good fragment ions but also produces many
matrix ions up to <i>m/z</i> 1000, making it impossible for small glycan
analysis.  6-aza-2-thiothymine increases sensitivity for anionic glycans.
<p>
Human urinary EPO analysis shows that the glycoprotein has fucosylated
tetra-antennary glycans on Asn-38 and -83 but mixtures of fucosylated bi-, tri-,
and tetra-antennary glycans on Asn-24.  CD59 shows significant heterogeneity
with 130 N-linked glycans having up to 25 monosaccharides.
<p>
ESI-MS is less sensitive than MALDI to glycans, particularly larger ones.
Often permethylation (with diazomethane) is useful.  Capillaries are charged
at 3000 v in 1:1 MeOH/MeCN:water.  Underivated glycans show [M + Na]<sup>+</sup>
ions.
<p>
To determine site occupancy, glycopeptides need to be generated, usually
trypsinizing reductively alkylated protein.  LC-ESI-MS is then applied on the
digest.  Oxonium ions such as HexNAc (<i>m/z</i> 204), hexose (<i>m/z</i>
163), sialic acid (<i>m/z</i> 292), or Hex-HexNAc (<i>m/z</i> 366) can
be used to identify sugars.
<p>
To release glycans from N- and O-linked glycoproteins, hydrazine is effective,
either manually or with automated equipment.  O-linked are more sensitive
at 60&deg;, and N-linked require 95&deg;.  The peptide bond is destroyed
in the process.  Partial cleavage of acyl groups (N-acetylamino sugars,
sialic acids) occurs as well.
<p>
Use of alkali releases O-linked glycans through beta-elimination.
The presence of sodium borohydride can be used to reduce them to alditols
as the reducing sugars are unstable at high pH.  Disadvantages are that
borohydride releases some N-linked glycans, the reducing sugar terminus
is reduced to prevent tagging with chromophores, and O-acetyl groups are
lost.  Hydrazine at 65&deg; under anhydrous conditions overcomes these
problems, and O-linked glycans can be released by heating  at 45&deg; for
18 h in 50% aqueous hydrazine with 0.2 M triethylamine.  The hydrazones
are acetylatd and incubated in aqueous acetone 24 h at 55&deg; to release
glycan.
<p>
Thin (0.75 mm) 12.5% and 17% SDS-PAGE gels were used in a system that
employed glycosidase PNGase F to remove N-linked sugars.  Glycan is
extracted in water and passed through AG-3/AG-50 mixed bed resin and
C18 packed into gel loader tips.  MALDI-MS or HPLC is then used to analyze.
Removal of anions (by AG-3) was helpful for MALDI although it removes
acidic glycans.  Methylation prior to ion-exchanges is helpful to keep
sialic and uronic acid sugars.  Graphitized carbon can be used to clean
up sulfated sugars.
<p>
For looking at the structure of glycans, a number of chemical methods
are essential.  Methanolysis produces methyl glycosides which can be
analyzed by GC/MS as trimethylsilyl or acetyl derivatives.  Alternatively
hydrolysis to monosaccharides followed by high pH anion-exchange
chromatography (HPAEC) coupled with pulsed amperometric detection (PAD)
is also possible.   For branch analysis, free hydroxyls must be
permethylated after reducing end reduction.  Hydrolysis to monosaccharides
and reduction to alditols is done.  These are then analyzed by GC/MS.
<p>
Additional structural analysis is by specific glycosidases.  Digestion
is probably best done in a matrix or sequentially.  Mass spectrometry
prior to digestion might help in glycosidase selection.
<p>
Periodate oxidation can provide information about linkage, as it is specific
for cis-diols.  Dialdehydes are created that are reduced with sodium
borodeuteride and then methylated.  Product mass then differs in a way
that reflects linkage.  If an interal monosaccharide has link at C1 and C4,
there will be cleavage at C2 and C3.  Linkage at C6 will result in
peroxidation at C2, C3, and C4, with loss of C3.
<p>
Polysaccharides treated with acetic anhydride, acetic acid, and sulfuric
acid in a (10:10:1) mixture results in peracetylation and cleavage of
glycosidic bonds.  1&minus;&gt;6 branches are most susceptible, while
1&minus;&gt;2 and 1&minus;&gt;3 are more resistant.  &alpha; anomers are
also more susceptible than &beta; anomers.
<p>
Fragmentation spectra may be useful in determining structure.  The
particular type of ion formed ([M + H]<sup>+</sup>, [M + Na]<sup>+</sup>),
the charge state, collision energy, and time for fragmentation are all
factors determining the type of dissociation.  Mass changes may result
from loss of adduct.  Typical cleavages will be intermonosaccharidal,
but not always.  A nomenclature has been developed by Domon and Costello
similar to peptide ions (see table)
<table>
<caption>Fragment Ion Nomenclature for Glycan Monosaccharides</caption>
<tr><th>Ion <th>Bonds Cleaved <th>Reducing/Nonreducing End?
<tr><td><sup><i>s,t</i></sup>A<sub><i>n</i></sub>
  <td>any two cross ring bonds
  <td>Non-reducing end ion
<tr><td>B<sub><i>n</i></sub>
  <td>glycosidic bond, non-reducing side monosaccharide
  <td>Non-reducing end ion
<tr><td>C<sub><i>n</i></sub>
  <td>glycosidic bond, reducing side monosaccharide
  <td>Non-reducing end ion
<tr><td><sup><i>s,t</i></sup>X<sub><i>n</i></sub>
  <td>any two cross ring bonds
  <td>Reducing end ion
<tr><td>Y<sub><i>n</i></sub>
  <td>glycosidic bond, non-reducing side monosaccharide
  <td>Reducing end ion
<tr><td>Z<sub><i>n</i></sub>
  <td>glycosidic bond, reducing side monosaccharide
  <td>Reducing end ion
</table>
<p>
The subscript <i>n</i> indicates the numbering of the rings from the 
non-reducing end if they are A, B, or C, ions, and from the reducing end if they 
are X, Y, or Z ions.  The <i>s,t</i> superscript on the cross-ring ion indicates 
the two bonds of the ring that are broken in forming the ion.  When fragments 
are from branched chains, Greek letters are used to designate ions, with &alpha; 
being that of the largest chain.  Fragmentation is more easily achieved in 
protonated than in metal cationized species, and in metal cationized species the 
order of easier-to-harder decomposition is 
Li<sup>+</sup>&nbsp;&gt;&nbsp;Na<sup>+</sup>&nbsp;&gt;&nbsp;K<sup>+</sup>&nbsp;&gt;&nbsp;Cs<sup>+</sup>. 
Metal-cationized sugars produce more cross-ring fragments (particularly 
Li<sup>+</sup>).  PSD spectra reveal that CHCA matrix protonates the nitrogen 
atom of 2-AP efficiently, and the [M+H]<sup>+</sup> ion fragments only into a Y 
series of ions because of charge localization on the reducing terminus.  DHB 
produces a [M+Na]<sup>+</sup> with no localized charge, and this gives a mixture 
of B and Y ions with some cross-ring fragments.
<p>
<b>PSD</b>.  Matrix selection is quite important.  Some matrices are <q>hot</q> 
in that readily produce extensive fragmentation, but are less efficient with 
respect to ionization efficiency.  CHCA is better at causing PSD but does not 
ionize carbohydrates as well as DHB. Typical tens of pmoles are necessary to see 
PSD spectra in neutral glycans. Acidic glycans more readily fragment, giving 
strong signals when sialic acid or 2-keto-3-deoxyoctonate are lost.  Some even 
use linear mode to record spectra so that uncharged fragments can be seen.  With 
reflectron spectra, the data must be collected in segments (different mass 
ranges) to produce a complete spectrum, and this is inefficient in terms of time 
and sample consumed.
<p>
<b>ISD</b>.  In-source decay (ISD) is the <q>prompt</q> fragmentation or rapid 
decay occurring in the source which gives fragments focused with their parent 
ions.  These are typically seen in FAB but not so much in MALDI spectra.  They 
would be present in MALDI where acidic carbohydrates are rich in the sample or 
if there is a <q>time-lag focusing</q> feature (delayed extraction) used to 
increase resolution.  ISD on an ESI-MS is controlled by adjustments to the 
sample cone voltage.
<p>
<b>CID</b>.  This is essentially a directed fragmentation, and as such
is much more reproducible.  Today more work is done with an ESI-QToF
using a high cone voltage where [M+Na]<sup>+</sup> are the major species
with complex spectra involving glycosidic and cross-ring fragment ions.
Only 5 pmol are necessary.  At lower cone voltages, the spectra are dominated
by <i>z</i> = 2 ions, with hydrogen as adduct, and the fragments are
largely glycosidic cleavage ions in CID.  When tertiary amines are present
there is an intense <i>z</i> = 2 as a result of amine protonation, and
only 500 fmol sample is required.  Sialylated glycans might benefit from
permethylation or methyl ester formation to achieve sensitivity.
When ESI in the high mass range is not useful, consider using high energy
CID spectra from glycans in a MALDI-MS/MS system.
<p>
Generally carbohydrate fragmentation spectra are heavily represented by
B and Y ions, indicating fragmentation on the glycosidic bond.  This
is more intense when the sugar is adjacent to sialic acid, and cleavages
adjacent to HexNAc residues are favored over losses of hexose.
N-linked glycans show prominent B-type fragments when the GlcNAc reducing
termnal residue is lost, or when ever reducing terminal derivative or
fucose is lost.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
K&uuml;ster B, Krogh TN, Mortz E, Harvey DJ
 </span>
 <span class="year">
(2001)
 </span>
 <span class="jname">
Proteomics
 </span>
 <span class="jvol">
1,
 </span>
 <span class="pages">
350-361.
 </span>
<p class="reftitl">
Glycosylation Analysis of Gel-Separated Proteins
<p class="abstract">
<b>REVIEW</b>.
<br>
<b>ABSTRACT</b>
Beyond the identification of proteins involved in a particular physiological situation,
many aspects of proteomics require more detailed characterization of the proteins
involved. Post-translational modifications (PTMs) of proteins are a common means to
target proteins, regulate their activities and to mediate communication between proteins
and cells. Owing to the much higher analytical complexity of glycan analysis compared
to e.g. protein identification, PTM analysis in general and glycosylation analysis
in particular is largely neglected in proteomics. In this review, the current technological
status of global and site-specific glycosylation analysis of gel-separated proteins is
described and the way in which the available technology can be employed in proteomics
is critically discussed.
<p class="Para1">
Advances in delayed extraction MALDI-MS, nano-ESI-MS and HPLC with
fluorescence detection have improved analysis.  Glycosylation analysis
typically requires very pure protein, typically several chromatographic
steps if the protein is in solution.  Recombinants do not require as much
purification, but the problem is that biologically relevant glycosylation
is not really achieved in recombinants, so the protein in its natural
state is required.  For large membrane glycoproteins, 1-D SDS-PAGE is the
one way of trying to purify.  Attempts to use 2-D gels often generate
smears, making handling more difficult.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Bakhtiar R, Nelson RW
 </span>
 <span class="year">
(2001)
 </span>
 <span class="jname">
Molec. Pharmacol.
 </span>
 <span class="jvol">
60,
 </span>
 <span class="pages">
405-415.
 </span>
<p class="reftitl">
Mass Spectrometry of the Proteome
<p class="abstract">
<b>MINIREVIEW</b>.

<!-- ============================================================= -->
<p class="tags">
MudPIT, 2DLC-MS/MS, LC-MS/MS
<p class="ref" id="washburn-2001">
 <span class="authors">
Washburn MP, Wolters D, Yates 3rd JR
 </span>
 <span class="year">
(2001)
 </span>
 <span class="jname">
Nat. Biotechnol.
 </span>
 <span class="jvol">
19,
 </span>
 <span class="pages">
242-247.
 </span>
<p class="reftitl">
Large-scale analysis of the yeast proteome by multidimensional protein
identification technology
<p class="Para1">
Multidimensional protein identification technology (MudPIT) is reproducible
chromatographically and in the final protein list.  MS/MS data was generated
from three different samples and assigned to 5540 peptides to identify
1484 proteins in the <i>S. cerevisiae</i> genome.  3 fractions represented
soluble fraction, lightly washed insoluble, and heavily washed insoluble.
Lysis was in 0.31 M sodium fluoride, 3.45 mM sodium vanadate, 250 mM sodium
chloride, 12 mM EDTA, 50 mM Tris, and 0.14 M sodium phosphate dibasic,
pH 7.6.  Glass beads were added and extraction in BioSpec Mini-BeadBeater
(see method in text).  The pellet was washed in PBS (1.4 mM NaCl, 0.27 mM KCl,
1 mM disodium phosphate, and 0.18 mM dibasic potassium phosphate, pH 7.4)
by vortexing 2 min.  After spinning 14000 rpm 10 min in Eppendorf microfuge,
one of these pellets (<i>lightly washed insoluble</i>) was then dried
in SpeedVac SC110 (Savant).  Another pellet was then washed three more
times in this way, and was called <i>heavily washed insolube</i>.
Soluble samples are adjusted to pH 8.5 with 1 M ammonium bicarbonate and then
protein assayed according to Bradford.  Samples are treated to urea (to 8 M),
reduced with DTT (to 1 mM), and carboxyamidomethylated with 10 mM iodoacetamide.
Endoproteinase Lys-C is used for the digestion of the soluble fraction.
<p>
Whole cell lysates (from yeast) are loaded separately on to biphasic 
microcapillary column and the column inserted into the instrumental setup. 
Elution from the experimental setup is with salt gradient on to the RP phase, 
and then from a RP into an ESI-IonTrap MS.  A 4-way union permits HPLC gradient 
in opposite the fitting of the SCX/RP column, and a voltage wire opposite a 
waste/splitter port.  The voltage behind the column supplies the ESI.
<p>
The yeast proteome was searched against the MIPS database which provides data on 
proteins according to cell localization.  The two insoluble fractions identified 
may organellar proteins.  Heavily washed insoluble represented nuclear, 
mitochondrial, ER, PM, and Golgi membrane proteins. The majority of proteins 
found for the vacuole were seen in the lightly washed insoluble fraction, 
including H<sup>+</sup>-ATPase domains VMA4 (CAI=0.27) and VMA5 (CAI=0.24).  The 
analysis showed that every major functional category and protein class was 
represented in the data. 32 protein kinases, rarely seen in 2D-PAGE, including 
MAP kinase signal pathway kinases STE7, STE11, STE20, and FUS3 were identified. 
Low abundance proteins such as 45 transcription factors including SW1-SNF 
complex SNF5, SWI4, and SWI6.  With 6216 open reading frames in the yeast 
genome, 83% have CAI values between 0-0.20.  Most proteins found by 2D-PAGE 
could identify few proteins with CAI&lt;0.20, indicating that it is likely 
incapable of detecting low abundance proteins.  MudPIT found a proteome in which 
791 or 53.3% of proteins had CAI&lt;0.20.  1347 peptides from the 791 proteins 
were identified that had a CAI&lt;0.20., an average of 1.7 peptides per protein. 
The peptides per protein increases with increasing CAI.  CAI is considered a 
predictor of protein abundance, and it is not difficult to find the most 
abundant of proteins.
<p>
This study found 72 of 231 possible peripheral membrane proteins predicted in 
the Yeast Proteome Database.  23 from the heavily washed insoluble fraction and 
14 from the lightly washed were uniquely identified.  MIPS database has searched 
entire yeast genome for predicted transmembrane domains numbering from 1 to 20 
using criteria of Klein et al. and Goffeau et al.  697 proteins in <i>S. 
cerevisiae</i> genome have &ge;3 predicted Tm domains, and this analysis found 
131 or 19%.  Of the 131, 44 were in the heavily washed insoluble, and 33 in the 
lightly washed insoluble.  Several had low abundance based on CAI.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Puig O, Caspary F, Rigaut G, Rutz B, Bouveret E, Bragado-Nilsson E,
Wilm M, S&eacute;raphin B
 </span>
 <span class="year">
(2001)
 </span>
 <span class="jname">
Methods
 </span>
 <span class="jvol">
24,
 </span>
 <span class="pages">
218-229.
 </span>
<p class="reftitl">
The Tandem Affinity Purification (TAP) Method:  A General Procedure of
Protein Complex Purification
<p class="Para1">
TAP method involves fusion of the TAP tag to the target protein and then
placing in the host cell or organism.  Expression should be controlled
to natural levels or the protein associates with nonnatural partners
(heat shock proteins, proteasome).
<p>
The TAP tag consists of two IgG binding domains of <i>S. aureus</i>
protein A and a calmodulin binding peptide (CBP) separated by a TEV
protease cleavage site.  A C-terminal TAP tag had been described previously
and this report describes an N-terminal tag.  The ProtA binding domain
must be located at the extreme terminus, whether at the C- or N-end.
<p>
The material is passed over an IgG affinity column, washed thoroughly.
The TEV protease is required to elute the material.  The eluate is
then incubated with calmodulin-coated beads in the presence of calcium.
Washing removes the TEV protease and other contaminants.  The bound material
is released by adding EGTA.
<p>
In deciding how to make the fusion protein with TAP tag, consider that
there may be a need to put the tag on the N- and C-terminus to see which
one works.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Tonge R, Shaw J, Middleton B, Rowlinson R, Rayner S, Young J, Pognan F,
Hawkins E, Currie I, Davison M
 </span>
 <span class="year">
(2001)
 </span>
 <span class="jname">
Proteomics
 </span>
 <span class="jvol">
1,
 </span>
 <span class="pages">
377-396.
 </span>
<p class="reftitl">
Validation and Development of Fluorescence Two-Dimensional Differential
Gel Electrophoresis Proteomics Technology
<p class="abstract">
<b>ABSTRACT</b>
Fluorescence two-dimensional differential gel electrophoresis (2-D DIGE *) is a new
development in protein detection for two-dimensional gels. Using mouse liver homogenates
(control and paracetamol (N-acetyl-p-aminophenol, APAP)-treated), we have
determined the quantitative variation in the 2-D DIGE process and established statistically
valid thresholds for assigning quantitative changes between samples. Thresholds
were dependent on normalised spot volume, ranged from approximately 1.2 fold for
large volume spots to 3.5 fold for small volume spots and were not markedly affected
by the particular cyanine dye combination or by multiple operators carrying out the dye
labelling reaction. To minimise the thresholds, substantial user editing was required
when using ImageMaster 2D-Elite software. The difference thresholds were applied
to the test system and quantitative protein differences were determined using replicate
gels of pool samples and single gels from multiple individual animals (control vs treated
in each gel). Throughout, the differences revealed with a particular cyanine dye combination
were mirrored almost without exception when the dye combination was
reversed. Both pool and individual sample analyses provided unique data to the study.
The inter-animal response variability in inbred mice was approximately nine times that
contributed by the 2-D DIGE process. A number of the most frequently observed protein
changes resulting from APAP-treatment were identified by mass spectrometry.
Several of these can be rationalised based on available data on the mechanism of
APAP hepatotoxicity but others cannot, indicating that proteomics can provide further
insights into the biochemical basis of APAP toxicity.
<p class="Para1">
Mice were treated with 500 mg/kg APAP for 4 h and liver sections examined
under light microscopy.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Gobom J, Schuerenberg M, Mueller M, Theiss D, Lehrach H, Nordhoff E
 </span>
 <span class="year">
(2001)
 </span>
 <span class="jname">
Anal. Chem.
 </span>
 <span class="jvol">
73,
 </span>
 <span class="pages">
434-438.
 </span>
<p class="reftitl">
&alpha;-Cyano-4-hydroxycinnamic Acid Affinity Sample
Preparation. A Protocol for MALDI-MS Peptide
Analysis in Proteomics
<p class="Para1">
High throughput sample analysis by MALDI MS requires some adaptations.
Spectral reproducibility is enhanced when sample diameter falls below
250 &micro;m.  Pre-structured sample supports have been previously prepared
with a strongly lyophobic (solvent-repelling) Teflon coating and an
array of gold spots as hydrophilic sample anchors.  Each sample spot
drops onto one anchor.  In this study, the sample supports are coated
with ~5 &micro;m thick layer of Teflon-like composite and an array of
384 circular <q>interruptons</q> having 400 &micro;m diameter.
The underlying stainless steel surface is the anchor.
<p>
The CHCA surface affinity sample prep method resembles the fast evaporation
method such than an acidified, aqueous sample is deposited on a
microcrystalline surface of CHCA.  But in this method, sample deposited
is not allowed to dry, but is removed after a short contact time on the
sample surface.  The following steps are done:
<ol>
<li>Dispense 100 &micro;l of 100 mg/ml CHCA in 90% acetone/0.005% TFA
near one of the short edges of the support.
<li>Place a Teflon (PTFE) rod  horizontally across the sample plate surface
in contact with the matrix solution in order to spread it across the length
of the rod to create a liquid film between rod and sample support.
<li>Draw the rod across the sample support, dragging the matrix solution
along, to let each anchor absorb a small volume of matrix.
</ol>
<p>If only a small number of samples is desired
<ol>
<li>Take a narrow pipet tip (GELoader, Eppendor) and immerse in a
1 mg/ml CHCA in 99% acetone / 0.2% TFA) and let capillary force aspirate it.
<li>Let the pipet tip touch an anchor, to deposit a small amount of matrix.
</ol>
<p>To deposit the peptides on the surface, acidify peptides in minimum 0.2% TFA
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Beck JL, Colgrave ML, Ralph SF, Sheil MM
 </span>
 <span class="year">
(2001)
 </span>
 <span class="jname">
Mass Spec. Rev.
 </span>
 <span class="jvol">
20,
 </span>
 <span class="pages">
61-87.
 </span>
<p class="reftitl">
Electrospray Ionization Mass Spectrometry of Oligonucleotide Complexes
with Drugs, Metals, and Proteins
<p class="abstract">
<b>REVIEW</b>.  Others have reviewed the use of ESI and MALDI for nucleic acid
analysis (see McCloskey &amp; Crain, 1992; Limbach, Crain, McCloskey, 1995;
Nordhoff, Kirpekar, Roepstorff, 1996; Crain and McCloskey, 1998).  Noncovalent
interactions using ESI-MS have also been done: protein-protein, DNA-protein,
DNA-drug, and Ab-Ag complexes.  Particularly DNA-metal ion interactions since
they are criticla in transcription and replication.
<p>
Antitumor drugs are useful and bind directly to DNA.  Some drugs are designed
using ADEPT or GDEPT to promote binding action.  Drug design for DNA binders
starts with sequence specificity, while also considering major or minor groove
binding, determining mono- vs. bifunctional binding, and compare intrastrand
vs. interstrand crosslinking for bifunctionals.
<p><b>Alkylation</b> is a natural event affecting carcinogenesis, mutagenesis.
It occurs preferentially on N and O atoms in purines (N7 of guanine, the
most nucleophilic).  Reports have come out of the use of electron capture
negative chemical ionization, thermospray, FAB, and API to characterize adducts
with bases or nucleosides.  ESI has been used to characterize addutts formed from
malondialdeyde reaction with deoxynucleosides.  Tamoxifen activation was studied
in in rats, mice, and humand with <sup>32</sup>P-postlabeled material and
LC-MS.
<p>
Information about sequence is important because detailed information on ligand-DNA
complexes by NMR and XRD requires shorter target sequences.  Variations
in local structure, (frayed ends in short sequences, higher order structure
in longer ones) and the requirements for large amounts of sample (NMR) or
difficulties in crystallization (XRD) make these techniques difficult.
<p>
Preliminary studies reveal that MS/MS spectra of alkylated oligos show a
highly specific fragmentation.  Hedamcyin (covalent) adducts were made with
two different 6-mers.  G akylation makes glycosidic bond labile, with depurination
and then cleavage of 3'-C-O bond next to akylated residue.  This compares much
better than the unmodfified oligo MS/MS spectra.  Electron-withdrawing groups
on the base enhance fragmentation of the 3'-C-O to give <i>w</i> ions.
<p>
A number of potential binding sites exist on DNA.  Electrostatic interaction between
cationic ligands and polyanionic backbone.  Intercalation in the planar rings
between bases.  Hydrogen bonding with bases through the minor groove.  Many
drugs have combinations:  safranine T binds by intercalation and electrostatic
interaction.  In fact the general feature of intercalators is planarity like
the bases and cationicity.  Ethidium bromide, daunomycin are simple intercalators.
Nogalamycin is a threading intercalator.  Ditercalinium are bis-intercalators.
<p>
The minor groove is a preferred association because they have torsional freedom
permitting twist complemetary to minor groove.  Minor groove  ligands interact
with 3-4 bases in sequence-dependent manner, usually with A and/or T combinations,
as alignment of hydrogen bond donors and acceptors on the ligand with those on the
bases in the minor groove.  AT-rich regions have larger electrostatic potential,
strengthening the interaction, and the ligand has a positive charge on one or
both ends.
<p>
The stability of DNA duplexes is improved by the presence of monocations in
relatively high concentrations to minimize the repulsion between anionic
backbones.  But high salt concentrations cause ion suppression and to dispersion
of the ion current among multiple adduct peaks for each species.  Oliog length
and sequence are important.  Short oligos (&le; 10-mer) have been used to
simplify data interpretation and control cost.  But this is often shorter than
a turn of the helix (requires 10 bp).
<p>
Drug studies have been done with ss DNA as well.  Distamycin A binds to
14-mer self-complementary ds DNA in 1:1 or 2:1 ratios.  This was followed by
other minor groove binders (pentamidine and Hoechst 33258) and showed that
non self-complementary oligos provide advantages.
<p>  Few studies have shown noncovalent interactions because of harsh ESI conditions
do not promote them.  Generally low capillary and cone voltages and lower
source and probe temperatures.  In one case DNA (5'-d[GGCTAGCC]<sub>2</sub>-3')
was annealed in the presence of intercalator (nogalamycin) in 100 mM
ammonium acetate.  Just before analysis, it is diluted 10-100 fold to
get away from signal suppression.  Recent studies done with minor-groove binders
and intercalators using non-self-complementary 12-mers:  minor-groove binders
showed sequence-specific binding but the intercalators did not.  Studies with
UV-vis specttroscopy and ESI showed that melting studies with minor-groove binders
and intercalators was the same in solution and gas phase.
<p>
Metal ions complex with DNA in many ways.  The antitumor agent cisplatin
Pt(NH<sub>3</sub>)<sub>2</sub>Cl<sub>2</sub> forms a covalent bond between
the platinum metal and the N7 of purines.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Toby GG, Golemis EA
 </span>
 <span class="year">
(2001)
 </span>
 <span class="jname">
Methods
 </span>
 <span class="jvol">
24,
 </span>
 <span class="pages">
201-217.
 </span>
<p class="reftitl">
Using the Yeast Interaction Trap and Other Two-Hybrid-Based Approaches To
Study Protein-Protein Interactions
<p class="Para1">
<b>REVIEW</b>.  The popularity of the yeast two-hybrid system in the
decade since its first description is indicated by more than 3000 articles.
The technique is rapid, inexpensive, simple to perform.  Two observations
in the 1980s led to the development of the yeast two-hybrid system.
One was the generation of a hybrid protein that could serve to activate
transcription.  A hybrid of the <i>E. coli</i> LexA repressor DNA binding
domain and the yeast Gal4p transcriptional activation domain was
expressed in <i>S. cerevisiae</i> and the chimera could induce transcription
when <i>lexA</i> operator sites were present.  Another is that transcriptional
activators do not have to bind to DNA to work:  they can interact with
other proteins that themselves bind to DNA.  Suppose protein X is translationally
fused to a DNA-binding domain (DB) of a transcription factor; this is
the <q>bait</q> protein.  A second hybrid, a fusion of known
protein Y or a library of proteins, is made to the transcriptional activation
domain (AD); this is the <q>prey</q> protein.  The bait must
interact with the prey in order to bring DB and AD together to activate
transcription.  Usually the transcribed gene is a reporter (<i>lacZ</i>
on XGal), or auxotrophic genes (<i>HIS3</i>, <i>LEU2</i>) are used  to
permit growth in medium with no His or Leu.  The original yeast two-hybrid
system used the Gal4p transcriptional activator.  The LexA DNA-binding
domain and either viral VP16 or bacterial B42 sequences act as the activation
domain.
<p>
The system has been used to look at regulation of telomere structure,
identification of substrate-kinase interaction, homodimerization of
gag proteins, heterodimerization of myogenin with E2A products.
Analyses that were not suitable were strong transcriptional activators,
extracellular proteins, and membrane-associated proteins.
<p>
Recent work has seen the development of a dual-bait system.  Two
independent baits are fused to different DNA-binding domains (e.g.,
LexA and cI).
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Dell A, Morris HR
 </span>
 <span class="year">
(2001)
 </span>
 <span class="jname">
Science
 </span>
 <span class="jvol">
291,
 </span>
 <span class="pages">
2351-2356.
 </span>
<p class="reftitl">
Glycoprotein Structure Determination by Mass Spectrometry
<p class="Para1">
<b>REVIEW</b>.  About 25 years earlier the first glycoprotein was determined
by MS in which multiple Thr O-linked disaccharides were found in antifreeze
proteins.  Electron impact and chemical ionization were used on permethylated
samples, allowing volatization and &beta;-elimination release.
Advanced <q>soft</q> ionization methods are used now (FAB, ESI, MALDI) although
permethylation is a strategy.
<p>
O- and N-type glycosylation are the two main types.  Mammalian N-type
glycosylation feature a trimannosyl-chitobiose core derived from
Glc<sub>3</sub>Man<sub>9</sub>GlcNAc<sub>2</sub> add cotranslationally.
ER-resident glucosidases then trim to produce Man<sub>9</sub>GlcNAc<sub>2</sub>
which is further processed.
<p>

<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Herbert BR, Harry JL, Packer NH, Gooley AA, Pedersen SK, Williams KL
 </span>
 <span class="year">
(2001)
 </span>
 <span class="jname">
Trends Biotechnol.
 </span>
 <span class="jvol">
19,
 </span>
 <span class="pages">
S3-S10.
 </span>
<p class="reftitl">
What place for polyacrylamide in proteomics?
<p class="abstract">
<b>ABSTRACT</b>.
Two-dimensional polyacrylamide gel electrophoresis (2D-PAGE) continues to deliver
high quality protein resolution and dynamic range for the proteomics researcher. To
remain as the preferred method for protein separation and characterization, several key
steps need to be implemented to ensure quality sample preparation and speed of
analysis. Here, we describe the progress made towards establishing 2D-PAGE as the
optimal separation tool for proteomics research.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Sinha P, Poland J, Schn&ouml;lzer M, Rabilloud T
 </span>
 <span class="year">
(2001)
 </span>
 <span class="jname">
Proteomics
 </span>
 <span class="jvol">
1,
 </span>
 <span class="pages">
835-840.
 </span>
<p class="reftitl">
A new silver staining apparatus and procedure
for matrix-assisted laser desorption/ionization-time
of flight analysis of proteins after two-dimensional
electrophoresis
<p class="Para1">
Gels were placed in a special staining apparatus for staining constructed
with plexiglas platforms with perforations or grids.  The gels are clamped into
the system to prevent movement during shaking.  An orbital shaker shakes the
entire body.  There is a drain to allow removal of 8 liters of solution
(enough for 10 gels) in about 1 min.
<p>
Proteins were randomly excised from silver-stained gels
and cleaned on ZipTip C18 then analyzed on MALDI
using Reflex II (db search is of NCBInr db with <a
href="http://prowl.rockefeller.edu/cgi-bin/ProFound">ProFound algorithm</a> and
<a href="http://prospector.ucsf.edu/uscfhtml3.2/msfit.htm">MS-Fit</a>.
Coomassie gels were overloaded with same sample and excised proteins showed
identical MS spectra.
<p>
Several silver stains have been attempted and compatibility with MS is a
problem.  Schevchenko et al method is often used. Fixation is in MeOH/acetic
acid and then acid is removed with MeOH wash, sensitized in thiosulfate,
treated with AgNO3 then reduction with HCHO in sodium carbonate.  This
is a rapid (1 h) stain especially with thin gels.  Thiosulfate helps to produce
a clear background and treatment with silver nitrate is at 4&deg; to avoid
oxidation reactions.  There is 1.6 ng detection limit for BSA.
<p>
This stain is the most sensitive to date (100 times more sensitive than
Coomassie).  Fixation is done 1 h + overnight or 4 &times; 30 min in
30% EtOH/10% acetic acid.  Gels are soaked 45 min in 3 g/L potassium
tetrathionate, 0.5 M potassium acetate, 30% EtOH.  Gels are rinsed
in 6 &times; 10 min water.  Impregnation is done with 0.2% silver nitrate
for 1-2 h.  Very brief (5-15 s) rinsing in water is done.  Development
is with 3% potassium carbonate with 300 &micro;l formaldehyde and
125 &micro;l 10% sodium thiosulfate pentahydrate (hypo) per liter for
up to 45 min.  Reaction is stopped with 30 min soaking in 40 g Tris and
20 ml acetic acid per liter.  Gels are rinsed in water before excision
for MS.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Rabilloud T, Strub J-M, Luche S, van Dorsselaer A, Lunardi J
 </span>
 <span class="year">
(2001)
 </span>
 <span class="jname">
Proteomics
 </span>
 <span class="jvol">
1,
 </span>
 <span class="pages">
699-704.
 </span>
<p class="reftitl">
A comparison between Sypro Ruby and ruthenium II tris (bathophenanthroline 
disulfonate) as fluorescent stains for protein detection in gels
<p class="Para1">

<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Steinberg TH, Pretty On Top K, Berggren KN, Kemper C, Jones L, Diwu Z,
Haugland RP, Patton WF
 </span>
 <span class="year">
(2001)
 </span>
 <span class="jname">
Proteomics
 </span>
 <span class="jvol">
1,
 </span>
 <span class="pages">
841-855.
 </span>
<p class="reftitl">
Rapid and Simple Single Nanogram Detection of Glycoproteins in Polyacrylamide
Gels and On Electroblots
<p class="Para1">
Glycans on glycoproteins vary in weight from 1% in cytokeratin-18 to as much
as 80% in mucin.  N-glycosidic bonds occur through Asn, but O-glycans bind
to proteins through the -OH group of Ser, Thr, HO-Lys, and HO-Pro, and through
the glycosyl phosphatidylinositol (GPI) anchor found on the COOH terminus of
some proteins with the right signal sequence (however, removed upon attachment).
Glycosylation has a great many biological effects:  immunogenicity, clearance
rate from blood, solubility, stability, and protease resistance.  In addition
it is involved in cell adhesion.
<p>
Methods to detect glycoproteins in gels and electroblots have proved very
insensitive.  Chromogenic detection of periodic acid-treaed with a Schiff
stain like acid fuchsin has been tried.  Also digoxigenin hydrazide after
oxidation, followed with anti-digoxigenin Ab conjugated to alkaline phosphatase.
Also use of biotin hydrazide conjugated to HRP or AP-streptavidin has been used.
Alcian Blue staining has been done for acidic glycoproteins.
<p>
A new fluorescent hydrazide is used on PAS-treated proteins; it does not
require a reduction step (sodium metabisulfite or sodium borohydride).
The Pro-Q Emerald 300 was developed to be compatiable with usual UV transillumination
sources.  The dye absorbs maximally at 288 nm (while Sypro Ruby absorbs with a
major peak at 282 nm and a broader peak at 470 nm), and so it can be excited by
mid-range UV (UV-B).  It emits at 533 nm (Sypro Ruby emits at 613 nm).  Green
illumination can be visualized with fluorescein band-pass emission filter and
the Sypro Ruby can be controlled with Texas Red band-pass emission.  Any Polaroid
or CCD camera can visualize either fluorescent dye.
<p>
<b>Comparison to other hydrazide detections</b>.  As the chemistry of hydrazide-based
analysis is the same, the question is of sensitivity.  Serial 2-fold diltuions
of Candy Cane MW markers (mix of glycosylated and unglycosylated proteins) were
run on SDS-PAGE and EITHER stained directly with the gel stain kit or proteins
stained with the blot kit after electroblotting.  The flourescent stain for the
gel kit proved to be 150-200 times more sensitive than colorimetric acid fuchsin
sulfite-based method, which varied with the glycoprotein evaluated.  Alexa
Fluor 350 hydrazide had 5-15-fold poor sensitivity that Pro-Q Emerald 300,
and also showed nonspecific staining of unglycosylated proteins.  Other
fluorescent dyes evaluated include hydrazides of Alexa Fluor 488, Cascade
Blue, dapoxyl sulfonyl, and Marina Blue, which all had worse nonspecific
staining.  Dansyl hydrazine had good specificity but 4-16-fold less sensitivity,
not to mention that it requies heating gels in acidified DMSO.
<p>
Run TWO gels at the same time with the sample.  One sample will be treated with
periodic acid and the other will NOT be.  This must be done to control possible
artifacts (non-specific staining).  Perhaps some proteins also have inherent
fluorescence.
<p>
On PVDF blots, the Pro-Q Emerald dye is 8-10 times more than sensitive than
acid- fuchsin sulfite methods.  Biotin-hydradize with ALP-conjugated strepavidin
and digoxigenin hydrazide with anti-Dig-conjugated ALP had similar sensitivity
as the Pro-Q Emerald 300 dye, but 6-18-fold less sensitivity than stains
in gels.  The biotin and digoxigenin-hydrazide methods resulted in nonspecific
staining of carbonic anhydrase, which lacks carbohydrate chains.
<p>
Other methods require a following reduction step to stabilize or visualize the
chromphoric molecule.  Pro-Q Emerald does lose its fluorescence 20-40% after
storage in 3% acetic acid for 18-48 h.  This is not prevented by using reduction
steps, indicating that this occurs because of an instability in the molecule
itself rather than rehydrolysis.
<p>
<b>Quantitation</b>.  Several glycoproteins were titrated: &alpha;-2
macroglobulin (9-10% CHO), glucose oxidase (12-13% CHO), &alpha;-1 acidic
glycoprotein (38-42% CHO), avidin (7% CHO), fetuin (22% CHO), and
ovalbumin (3-4% CHO).  All glycoproteins except ovalbumin and &alpha;-2 macroglobulin
could be quantified in a 500-1000 fold range, while the linear range of the
other two was about 125.
<p>
<b>Comparison to Lectin</b>.  Ovalbumin could be detected using ConA and
WGA but many glycoproteins were not.  Con A binds to unsubstitued and
2-O-substituted &alpha;-mannosyl residues, while WGA bind with high affinity
to tri-<i>N</i>-acetylglucosamine and sialic acid residues.  Horseradish
peroxidase is illuminated with Con A but not WGA, while fetuin is observed
by WGA but not Con A.  WGA detects &alpha;-1 acid glycoprotein (42 kDa), but
Con A does not.  Ovomucoid and ovotransferrin (28 and 76 kDa) are also
detected by WGA, but not Con A.  Ovalbumin is a mix of oligomannose, hybrid
and multi-antennary complex N-linked glycans, and so it is a good subtrate
for both lectins.  &alpha;-1 acid glycoprotein has N-linked complex type
heteroglycans present as tri- and tetra-antennary structures, and fetuin
is mostly sialylated tri-antennary N-linked glycans and also sialylated
O-linked glycans, which explains why Con A does not detect these.  HRP
is generally small, neutral N-linked glycans, usually not bound well by WGA.
<p>
Comparing blotting to direct gel staining, the latter is more sensitive
since blotting represents an extra step wherein yields are reduced.  Moreover
glycoproteins transfer poorly to membranes.  But with blotting there is the
additional advantage in that multiple immunochemical analyses are possible.
<b>Caution</b>:  any step that uses periodate oxidation effectively destroys
the sugar and binds it to the Pro-Q E300 molecule, and so lectins lose their
affinity as the structure is destroyed.
<p>
Endoglycosidase F, Endo-O-glycosidase, and sialidase A were used in a cocktail
to enzymatically deglycosylate several pure glycoproteins.  The untreated and
treated glycoproteins are then run in SDS-PAGE.  SYPRO Ruby staining shows that
treated proteins had a faster mobility shift with loss of the green
flourescence seen with Pro-Q staining.  The exception was HRP, in that it
did not show a mobility shift but still showed loss of green fluorecence.
HRP has an &alpha;(1,3)-fucosylated Asn N-acetylglucosamine linkage resistant
to these enyzmes.
<p>
In gel staining SYPRO Ruby quenches Pro-Q 300 Emerald fluorescence, and so simultaneous
visualization is not possible.  For this reason, Pro-Q staining should precede
SYPRO Ruby staining.  With electroblotting, simultaneous visualization of
red and green signal is possible.  Hardware/equipment that does well
(1024 &times; 1024 pixel systems with 16-bit intensities) incude Roche
Lumi-Imager, Bio-Rad Fluor-S Max Multi-Imager, Perkin-Elmer 1442 Multiwavelenght
Fluorimager, Fuji FLA-3000G Fluorescent Image Analyzer, and Genomic Solutions
BioImage Analyzer.  Other post-Pro-Q 300 staining could include Coomassie
Brilliant Blue and its colloidal form and Ag staining.
<p>
<b>Other Pro-Q Emerald Uses: LPS</b>.  Besides glycoproteins, PQE-300 can be
used to check any glycoconjugates.  All that is necessary is that it be
periodate oxidizable.  This stain was compared to an alkaline silver diammine
stain (superior to many other stains, including acidic AgNO<sub>3</sub>,
zinc-imidazole, SYPRO Orange and Ruby staining).  The sensitivity of the PQE300
stain was as little as 2-4 ng total, which 250-1000 ng was required for the silver
stain.
<p>
Radiolabeling of cultured cells with <sup>3</sup>H and <sup>14</sup>C and
also specific doubling labeling with [<sup>35</sup>S]Met (for all proteins)
and [<sup>3</sup>H]fucose has been done, but hazards and expense are a big
disadvantage, and not all systems can be labeled in vitro.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Premstaller A, Huber CG
 </span>
 <span class="year">
(2001)
 </span>
 <span class="jname">
Rapid Commun. Mass Spectrom.
 </span>
 <span class="jvol">
15,
 </span>
 <span class="pages">
1053-1060.
 </span>
<p class="reftitl">
Factors Determining The Performance of Triple Quadrupole, Quadrupole
Ion Trap and Sector Field Mass Spectrometer in Electrospray Ionization
Mass Spectrometry.  2. Suitability for <i>De Novo</i> Sequencing
<p class="Para1">
Fragment spectra of a decameric oligonucleotide (ATGCTACGAG, mass
3052.08) were generated in a triple-stage quadrupole (TSQ, Finnigan
MAT TSQ 7000), quadrupole ion trap (QIT, Finnigan MAT LCQ),
and double-focusing sector field (DFSF, Finnigan MAT 95S)
mass spectrometers.  Electrospray was used from a 250 &micro;l Unimetrics
glass syringe on Harvard model 22 pump, with a 10 mM triethylamine
in 50% MeCN solvent with flow 3-20 &micro;l/min.  Cations were removed
inline with a 20 &times; 0.5 mm Dowex 50WX8 microcolumn (Bio-Rad),
and plumbing was with 50 &micro;m (150 &micro;m o.d.) capillaries.
Heat was 200-210&deg; on the spray, and negative mode with capillary
voltage of 3200-4000 V used.  Nitrogen sheath gas pressure was 345 kPa
(TSQ), 65-93 units (QIT), and 410 kPa (DFSF).  Precursor ion was selected
with a 5 mass unit isolation width, argon set to 400 Pa for collision,
and the energy from 20-50 V for the TSQ.  For QIT, isolation width=3.6 u
and collision energy is 20% relative (0-100% corresponds to rf voltage
for resonance excitation from 0-5 V); Helium gas used for collision at
0.1 Pa.
<p>
In the TSQ and QIT instruments, the -4 ion was selected as precursor since
it gives better sequence info compared to lower charge precursors.
83 fragments were identified on the TSQ.  The (a-B) and <i>w</i> series
were complete starting from (a<sub>3</sub>-B<sub>3</sub>(G)) to
(a<sub>9</sub>-B<sub>9</sub>(A)) fragment, and from (w<sub>1</sub>)
to (w<sub>9</sub>).  Fragments with different charge states help to
make sequence identification quite complete.  With the exception of
(a<sub>5</sub>), a-type ions from (a<sub>2</sub>) to (a<sub>8</sub>) were
detected, as well as b-, c-, and d-series ions.  The b- and c-series
often lose the nucleobase at the 5&prime;-end, giving rise to three
fragments of the (b-B)- and most prominently, as many as 9 fragments
of the (c-B)-series.   3&prime; fragments were seen in a nearly complete
y-series, with only (y<sub>5</sub>) missing; additionally ions in the
x- and z-series were seen, with or without loss of base at 3&prime;-end
of fragment.  In the low mass end nucleobases adenine, guanine, and thymine
were represented at good intensity.  The drawbacks of looking at
a 10-mer on the TSQ include substantial spectral complexity (83 fragments)
and reduced signal-to-noise.
<p>
On the QIT only 28 fragments were seen.  The product ions involved
neutral loss of a nucleobase to form [M-A]<sup>4&minus;</sup>,
[M-G]<sup>4&minus;</sup>, and [M-C]<sup>4&minus;</sup> species.
Some double neutral loss species were observed as well:
[M-A-G]<sup>4&minus;</sup> (note no [M-A-G]<sup>3&minus;</sup> was found).
Complete series of w-fragments (w<sub>1</sub><sup>1&minus;</sup>) to
(w<sub>9</sub><sup>4&minus;</sup>) allowed the unambiguous determination
of the exact sequence of the decanucleotide.  When walking up the fragment
series, certain fragments will be key in mass determinations.
For the 10-mer, eight (a-B) type fragments, namely
(a<sub>3</sub>-B<sub>3</sub>(G))<sup>1&minus;</sup>,
(a<sub>4</sub>-B<sub>4</sub>(C))<sup>2&minus;</sup>, and
(a<sub>6</sub>-B<sub>6</sub>(A))<sup>2&minus;</sup> to
(a<sub>9</sub>-B<sub>9</sub>(A))<sup>3&minus;</sup> corroborate other
findings.  Elimination of PO<sub>3</sub> from w-fragments is indicated
by (y<sub>4</sub>)<sup>1&minus;</sup> and (y<sub>7</sub>)<sup>2&minus;</sup>.
Gaps in the (a-B) series resulted from missing ions for fragments
(a<sub>2</sub>-B<sub>2</sub>(T)) and (a<sub>5</sub>-B<sub>5</sub>(T)),
which are formed after elimination of the thymine base in a first step
fragmentation.  (M-T) base loss ions are not seen because of a low reactivity
of thymine in gas-phase collisions.  Backbone cleavage at the position
of this base is a less-favored pathway.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Griffin TJ, Gygi SP, Rist B, Aebersold R, Loboda A, Jilkine A, Ens W,
Standing KG
 </span>
 <span class="year">
(2001)
 </span>
 <span class="jname">
Anal. Chem.
 </span>
 <span class="jvol">
73,
 </span>
 <span class="pages">
978-986.
 </span>
<p class="reftitl">
Quantitative Proteomic Analysis Using a MALDI Quadrupole Time-of-Flight
Mass Spectrometer
<p class="abstract">
<b>ABSTRACT</b>.
We describe an approach to the quantitative analysis of complex
protein mixtures using a MALDI quadrupole time-of-flight (MALDI QqTOF)
mass spectrometer and isotope coded affinity tag reagents
(Gygi, S. P.; et al. Nat. Biotechnol. 1999, 17, 994-9.). Proteins
in mixtures are first labeled on cysteinyl residues using an isotope
coded affinity tag reagent, the proteins are enzymatically digested,
and the labeled peptides are purified using a multidimensional
separation procedure, with the last step being the elution of the
labeled peptides from a microcapillary reversed-phase liquid
chromatography column directly onto a MALDI sample target.
After addition of matrix, the sample spots are analyzed using a
MALDI QqTOF mass spectrometer, by first obtaining a mass spectrum
of the peptides in each sample spot in order to quantify the ratio
of abundance of pairs of isotopically tagged peptides, followed by
tandem mass spectrometric analysis to ascertain the sequence of
selected peptides for protein identification. The effectiveness
of this approach is demonstrated in the quantification and
identification of peptides from a control mixture of proteins
of known relative concentrations and also in the comparative
analysis of protein expression in Saccharomyces cerevisiae grown
on two different carbon sources.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Lilley KS, Razzaq A, Dupree P
 </span>
 <span class="year">
(2001)
 </span>
 <span class="jname">
Curr. Opin. Chem. Biol.
 </span>
 <span class="jvol">
6,
 </span>
 <span class="pages">
46-50.
 </span>
<p class="reftitl">
Two-Dimensional Gel Electrophoresis:  Recent Advances in Sample Preparation,
Detection, and Quantitation
<p class="Para1">
Proteins predicted from genome sequence data have recently been placed
into three classes:  those with known function, those with recognizable
motifs (a vague idea of function), and those with no recognizable
sequence similarities and therefore having no known function.  30% of
<i>Arabidopsis</i> predicted proteins fall into the latter category.
Transcript level studies are not really a clue as to protein levels.
<p>
Approximately 90% of the total protein cell is coded by only 10% of the
total protein species.  Low-abundance proteins, such as transcription
factors, are undetectable by 2D PAGE.  A single 2D PAGE with standard
visualization will maximally show 3000 spots, and many of those are
proteins co-migrating to the same spot with others.  Gygi et al. reported
that one faint spot held the products of six genes on Ag-stained, narrow
pH range gel in a yeast cell extract.  It is estimated that at least
1000 copies of a protein should be present to reach the level of detection
on 2D PAGE, but Hoving et al. reported detection of down to 300 copies
per cell on 1 pH unit zoom strips.  Tonella et al. estimated that 70%
of the <i>E. coli</i> proteome was seen on six different pH range gels.
Pre-focusing fractionation (on the Bio-Rad Rotofor) has been found to
improve proteome visualization as well.  Anion-exchange chromatography
is also useful in enriching certain proteins, and can work with lower
abundance proteins.
<p>
Membrane proteins are barely soluble in the buffers used in IEF.
Prefractionation with Triton X-114, chloroform/MeOH, or washes with
sodium carbonate, with sequential detergent extraction helps.  Santioni et al.
used zwtterionic detergents achieve this effect.  ASB14 and TX-100
combinations have helped to solubilize bacterial outer membrane proteins.
A study of detergents generally found that the stronger detergents helped
more in the detection of certain classes of membrane proteins.
<p>
As far as staining techniques, silver is more sensitive but has a very limited
linear range and is unusable for quantitation.  Sypro dyes are even
more sensitive.  Sypro red and orange give best results although Sypro
Ruby is used.
<p>
Quantitative studies of protein abundance indicate the more sensitive
dyes would be best at looking at low levels of protein expression.
Certain software is being used to provide accurate estimations of protein
levels over a linear range of 3 orders of magnitude, but pitfalls have
been described by others.
<p>
Difference gel electrophoresis (DIGE) involves labeling samples with
one of three distinct fluorescent dyes: Cy2, Cy3, and Cy5.  Labeled
samples are then run in a single gel and the gel scanned at different
wavelengths.  Since labeling requires Lys residues, Lys-rich proteins
will be over-represented.  Detection limits are estimated to be 0.1-0.2 ng,
and the linear range is estimated to be over 5 orders of magnitude.
A study of the heart mitochondrial proteome in mice deficient in creatine
kinase made use of a looped two-frame movie that showed the simplicity
of the technique but did not represent the best method for doing quantitation.
In another analysis of mouse liver before and after paracetamol admin.,
the use of Phoretix/ImageMaster software showed that significance thresholds
for changes indicate thresholds are dependent on normalized spot volumes,
and lower volumes have higher thresholds.  Also variation in levels
was found in the mice and not in error associated with DIGE.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Sonnhammer ELL, Wootton JC
 </span>
 <span class="year">
(2001)
 </span>
 <span class="jname">
Proteins
 </span>
 <span class="jvol">
45,
 </span>
 <span class="pages">
262-273.
 </span>
<p class="reftitl">
Integrated Graphical Analysis of Protein Sequence Features Predicted from
Sequence Composition
<p class="abstract">
<b>ABSTRACT</b>.
Several protein sequence analysis
algorithms are based on properties of amino acid
compositionandrepetitiveness. Theseincludemethods
for prediction of secondary structure elements,
coiled-coils, transmembrane segments or signal peptides,
and for assignment of low-complexity, nonglobular,
or intrinsically unstructured regions. The
quality of such analyses can be greatly enhanced by
graphical software tools that present predicted sequence
features together in context and allowjudgment
to be focused simultaneously onseveral different
types of supporting information. For these
purposes, we describe the SFINX package, which
allows many different sets of segmental or continuous-
curve sequence feature data, generated by individual
external programs, to be viewed in combination
alongside a sequence dot-plot or a multiple
alignment of database matches. The implementation
is currently based on extensions to the graphical
viewers Dotter and Blixem and scripts that
convert data from external programs to a simple
generic data definition format called SFS. We describe
applications in which dot-plots and flanking
database matches provide valuable contextual information
for analyses based on compositional and
repetitive sequence features. The system is also
useful for comparing results from algorithms run
with a range of parameters to determine appropriate
values for defaults or cutoffs for large-scale genomic analyses.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Belghazi M, Bathany K, Hountondji C, Grandier-Vazeille X, Manon S,
Schmitter J-M
 </span>
 <span class="year">
(2001)
 </span>
 <span class="jname">
Proteomics
 </span>
 <span class="jvol">
1,
 </span>
 <span class="pages">
946-954.
 </span>
<p class="reftitl">
Analysis of Protein Sequences and Protein Complexes by Matrix-Assisted
Laser/Desorption Ionization Mass Spectrometry
<p class="abstract">
<b>ABSTRACT</b>
In the context of proteome analysis, matrix-assisted laser desorption/ionization mass
spectrometry (MALDI-MS) can fulfil the two tasks of primary structure verification and
protein identification. As an illustration of the first of these tasks, the sequence of
Eschericha coli isoleucyl-tRNA synthetase, a protein with 15 reported sequence conflicts,
has been established by means of MALDI mass mapping. The identification
of mitochondrial proteins participating in a yeast supramolecular complex exhibiting
NADH dehydrogenase activity highlights the performances of MALDI-MS for the
second task. The spectral suppression phenomenon occurring for complex peptide
mixtures analysed by MALDI is discussed, as well as the role of post-source decay
analysis for confident protein identification.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Hamdan M, Galvani M, Righetti PG
 </span>
 <span class="year">
(2001)
 </span>
 <span class="jname">
Mass Spec. Rev.
 </span>
 <span class="jvol">
20,
 </span>
 <span class="pages">
121-141.
 </span>
<p class="reftitl">
Monitoring 2-D Gel-Induced Modifications of Proteins by MALDI-TOF Mass
Spectrometry
<p class="Para1">
<b>REVIEW</b>.  With 2D gel/MALDI MS analysis the intact protein mass and
its pI are information used for protein identification.  The accuracy of
pI prediction and the matching of mass are crucial.  Modifications of
proteins affecting mass determination are frequent.  High pH conditions
can for example cause reactions between IEF and polyacrylamide gel chemicals
with amino acids (Cys can form cysteinyl-S-&beta;-propionamide in reaction
with acrylamide, and can form an adduct with 2-mercaptoethanol.  Methionine
can be oxidized to form a sulfoxide adding a 16 Da mass;  usually the
reaction of ammonium persulfate remaining.  Coomassie-stained proteins
may have tightly bound and affect the mass.  Formic acid presence can
cause formylation of Ser and Thr residues.
<p>
With gel creation, acrylamide (FW=71, Partition Coefficient=0.2)
is not the only monomer used.  Many <i>N</i>-substituted monomers
have interesting properties.  <i>N,N</i>-Dimethylacrylamide (DMA,
FW=99, PC=0.5), <i>N</i>-acryloyl-2-amino-hydroxymethyl-1,3-propane
(TRISACRYL, FW=175, PC=0.01), <i>N</i>-acryloyl-aminoethoxyethanol (AAEE,
FW=159, PC=0.13), and <i>N</i>-acryloyl aminopropanol (AAP, FW=129,
PC=0.1).
<p>
Polymerization is not always 100%, but only about 90%, with about 30 mM
free unreacted acrylamide in the gel.  To see the effect this might have
on proteins, a 50 pmol/&micro;l bovine &beta;-lactoglobulin B (BLGB) solution
was prepared with different levels from 0.5 to 30 mM of the monomers named
above.  These were incubated at room temperate at approx. pH 9 and then
examined on MALDI ToF for the intact protein.  Spectra show a more
extensive reaction with acrylamide than with AAP;  this is based on
changes to relative peak heights of the major peaks and appearances of new
peaks.  The new peaks are consistent with the addition of from 2-4
acrylamide adducts.  Eventually all monomers were compared and the
reactivity scale appears to be acrylamide &gt; DMA &gt; AAEE &gt; trisacryl
&gt; AAP.
<p>
It is thought that Cys is the reactive center, but Tyr hydroxyl, Lys
&epsilon;-amino, and the free N-terminus are also implicated.  For this
study, bovine and human hemoglobin &alpha;-chains were used since they
have only a single Cys residue.  50 &micro;M proteins in 30 mM monomer
were incubated 1 h (human) and 6 h (bovine), then washed to remove excess
acrylamide, digested with trypsin, and then checked on MALDI.  Before
and after spectra were compared for the bovine:  two new peaks appear
at <i>m/z</i> 2114.0 and 2707.31, indicating modification of the
sequences 12-31 and 69-92 with the added mass of <i>m/z</i> 71.
There is no Cys in either fragment.  Lys<sup>16</sup> is present in
12-31 and Lys<sup>90</sup> in 69-92.  Human tryptic peptide analysis
also shows <i>m/z</i> 71 transition in peak <i>m/z</i> 2531.31 to
2602.23.  A single Cys 104 is present and is supposed to be the modified
site.  The longer incubation time for bovine vs. human may indicate
that Lys reactivity is slower.  It is also noted that a free -SH group
is not a precondition, but Cys residues in -S-S- bridges are just as
reactive, perhaps more so.
<p>
The polymers that make up immobilized pH gradients also were investigated
for the potential to modify proteins.  Immobilines include a great
many chemicals:  2-acrylamido-2-methylpropane sulfonic acid (MW 207,
pK 1), N-acryloylglycine (MW 129, pK 3.6), 4-acrylamidobutyric acid
(MW 157, pK 4.6), 2-morpholinoethyl acrylamide (MW 184, pK 6.2),
2-thiomorpholinoethylacrylamide (MW 200, pK 6.6), 3-morpholinopropylacrylamide
(MW 198, pK 7.0), 3-thiomorpholinopropylacrylamide (MW 214, pK 7.4),
N,N-dimethylaminoethylacrylamide (MW 142, pK 8.5),
N,N-dimethylaminopropylacrylamide (MW 156, 9.3), N,N-diethylaminopropylacrylamide
(MW 184, pK 10.3).  20 mM of these chemicals were incubated 1 h at
alkaline pH (9-10) with 50 &micro;M proteins, bovine &alpha;-lactalbumin (bLA) and
chicken egg lysozyme.  1 &micro;l was taken and mixed with eq vol sinapinic
acid and spotted on MALDI.  It should be noted that at the working pH
bLA with a low pI has a very negative charge, while the lysozyme is neutral
or slightly positive;  hence at alkaline pH acidic Immobilines, themselves
highly negative, are unlikely to react with bLA at least.  This was
shown by reaction of the pK 3.6 Immobiline more vigorously with lysozyme
than bLA.
<p>
At the higher pH there is strong evidence to show that Cys-SH groups are
targets of acrylamide alkylation.  This was found true with incubation of
Immobiline and the change in <i>m/z</i> associated with tryptic peptides
known to contain one or more Cys residues.  The reaction was fairly complete.
<p>
<b>Following Unfolding By Cys Alkylation</b>.  Protein folding and unfolding
can be monitored by UV difference spectroscopy (Gratzer 196), fluorescence
(Cantor &amp; Timasheff 1982), circular dichroism (Bayley 1980), optical
rotatory dispersion (Pace et al 1989), NMR (Jennings &amp; Wright 1993,
Bai et al 1994) and also MS (Miranker et al 1993, Zhang &amp; Smith 1993,
Konermann et al 1997).  The MS technique is done largely through ESI
and using hydrogen/deuterium exchange techniques.  Use of immobiline-type
chemicals has been used to study changes in the alkylation of Cys
kinetically and with different levels of denaturants present.
<p>
<b>Gel Crosslinker Reactivity</b>.  To test the potential of crosslinkers
to modify proteins, a 5 mM acrylamide monomer solution with a range of
1-10% PEGDA crosslinker was incubated 15 min in 50 &micro;M &beta;-lactoglobulin B
(bLGB).  MALDI spectra show a single new peak emerging:  from <i>m/z</i>
18281 to 18353, indicating acrylamide addition when no crosslinker is
present, but two additional peaks when 5% C is present, indicating
crosslinker reaction.  In fact relative to acrylamide present the
peak height suggests that the crosslinker has a more efficient reaction.
One result found was that the reactivity of the other end of the crosslinker
was altered after bonding with the first end.  Longer incubation times
were necessary to achieve modification.
<p>
bLGB was separated in SDS-PAGE and then extracted with formic acid:MeCN:isopropanol:water
after 10 min and 2 h in ultrasonic bath.  MALDI spectra of intact protein
shows a series of peaks on an envelope.  Some resulted from acrylamide
modification, others from formylation of Ser/Thr residues with the presence
of formic acid.  Coomassie blue interference may be seen with particularly
hydrophobici proteins.
<p>
How does one avoid all these potential modifications from 2-D gel work?
One is to make sure the protein is sufficiently denatured.
Urea (7 M), thiourea (2 M), SDS or other detergent (2%), DTT (65 mM),
and 30% glycerol are used in the IEF process, and they all stabilize
the unfolding of proteins.  With unfolding, alkylation of Cys residues
in particular should occur.  Use of iodoacetamide or acrylamide at pH 9
will efficiently alkylate protein.  One problem is that thiourea is quite
reactive with iodoacetamide.   If thiourea is present, proteins should
be alkylated at pH 9 with acrylamide.  Another factor to consider is
that SDS may have a negative influence (as opposed to other detergents),
on alkylation efficiency.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Zaia J, Boynton R, Heineg&aring;d D, Barry F
 </span>
 <span class="year">
(2001)
 </span>
 <span class="jname">
Biochemistry
 </span>
 <span class="jvol">
40,
 </span>
 <span class="pages">
12983-12991.
 </span>
<p class="reftitl">
Posttranslational Modifications to Human Bone Sialoprotein Determined
by Mass Spectrometry
<p class="Para1">
Bone sialoprotein (BSP) was obtained from human bone with GuanHCl extraction
and then purifying on anion IEX chromatography.  Analytical gel filtration
showed single peak with amino acid analysis showing homogeneity.
Amino acid similarity search with SwissPROT db shows topscore for BSP.
A series of treatments is applied before MALDI-ToF analysis.  Untreated
shows peak at m/z 52533;  N-Glycanase (2.5 &micro;g substrate with 75 mU
enzyme, 100 mM Tris pH 8.5, ON @ 37&deg;) produced m/z 46077 protein;
with neuraminidase alone, m/z 45771; combination of N-glycase and neuraminidase,
m/z 41456, and with an N- and O-glycanase and neuraminidase, m/z 34799.
The untreated peak is quite broad, reflecting significant natural
microheterogeneity in BSP.  Significant peak narrowing occurred with
single and combined deglycosylations.  Loss of mass calculations show
that an average of 7 sialic acid (NAcNeuram) residues per molecule BSP.
Total BSP oligosaccharide was about 17,734 &plusmn; 423 Da.
<p>
Hydrolysis of BSP (2 M TFA for 3 h for neutral sugars, 4 M HCl for 5 h
for hexosamines, 6 M HCl @ 105&deg; for 24 h for amino acids)
reduces it to monosaccharides and amino acids, which can be analyzed.
The saccharides are dissolved in water after drying and then separated
on CarboPac PA-1, eluted with 50 mM NaOH and sensed by pulsed amperometry.
A molar CHO to core protein ratio is found.  Calculated CHO is 12,020
&plusmn; 2300 (oligo less sialic acid), comparable to MS determination
of 10972 &plusmn; 1138 Da.  BSP has 4 potential N-glycosylation sites
and the 6.1 mol mannose/mol protein suggests two are glycosylated.
If they are complex type oligosaccharides with no mannose other than core.
Some sialic acid residues could belong to O-linked sugars.  O-linked
saccharides that are desialylated have a mass of 6660 Da, with 16 sialic
acid residues.
<p>
Sulfated tyrosines in a peptide will fragment in positive ion mode to
produce ion mass consistent with unmodified peptide.  In negative ion
mode there is less fragmentation, and pair of ions separated by 80.06 Da
are seen (sulfated and desulfated molecules).  Phosphate ions do not
decay and their presence is confirmed with alkaline phosphatase treatment.
Use of positive and negative ion mode spectral acquisition is useful
in
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Doucette A, Li L
 </span>
 <span class="year">
(2001)
 </span>
 <span class="jname">
Proteomics
 </span>
 <span class="jvol">
1,
 </span>
 <span class="pages">
946-954.
 </span>
<p class="reftitl">
Investigation of the Applicability of A Sequential Digestion Protocol
Using Trypsin and Leucine Aminopeptidase M for Protein Identification
by Matrix-Assisted Laser Desorption/Ionization Time of Flight Mass Spectrometry
<p class="abstract">
<b>ABSTRACT</b>
An investigation of the applicability of a sequential digestion procedure involving endoand
exoprotease digestion of proteins is reported. The procedure involves the digestion
of a protein sample with trypsin, yielding peptide fragments characteristic of the
protein. The resulting mixture of peptide fragments is then subjected to N-terminal
sequencing with leucine aminopeptidase M (LAP), with matrix-assisted laser desorption
ionization-time of flight mass spectrometric analysis of the various digestion products.
Several proteins in solution, as well as gel extracted proteins were subjected to
this sequential enzyme digestion procedure. The results of these experiments reveal
that LAP will preferentially cleave specific peptides of the trypsin digested sample
with high efficiency, while leaving other peptides undigested. Also, the length of the
amino acid sequence tags that can be generated with this method is limited; the longest
sequence tag generated from a single tryptic peptide was four amino acids, even
though the digestion was allowed to proceed for long times. In the experiments,
N-terminal digestion products were detected as early as two minutes, or as late as
90 minutes, following the addition of LAP to the sample. The method was shown to
be effective for subpicomole starting quantities of protein, although with some loss in
digestion efficiency at lower concentrations of starting material. This method is useful
in providing additional sequence information to increase the level of confidence in protein
identification, as illustrated in the identification of bacterial proteins fractionated by
high-performance liquid chromatography. In some instances, this method can provide
additional sequence information where post source decay and nanospray mass spectrometry
failed to generate fragment ion spectra. This is illustrated in an example where
the procedure was applied to a membrane protein, CD9, that had been isolated by
sodium dodecyl sulfate polyacrylamide gel electrophoresis. Although the sequential
digestion procedure requires more human intervention, it is a straightforward method
and can be readily implemented.
<!-- ============================================================= -->
<p class="tags">
shotgun proteomics, MudPIT
<p class="ref" id="wolters-2001">
 <span class="authors">
Wolters D, Washburn MP, Yates 3rd JR
 </span>
 <span class="year">
(2001)
 </span>
 <span class="jname">
Anal. Chem.
 </span>
 <span class="jvol">
73,
 </span>
 <span class="pages">
5683-5690.
 </span>
<p class="reftitl">
An Automated Multidimensional Protein Identification Technology for
Shotgun Proteomics
<p class="Para1">
Peptide separation and protein identfication methods were compared using
previously described buffer conditions: in that study solvent A was 5% MeCN + 0.5%
acetic acid, solvent B was 80% MeCN + 0.5% AA, solvent C was 5% MeCN +
0.5% AA + 0.25 M KCl, and solvent D the same as solvent C but containing
1 M KCl.  However, KCl is incomptable with ESI-MS.  Ammonium acetate was
substituted as a volatile alternative.
<p>
Soluble <i>S. cerevisiae</i> protein (420 &micro;g) was digested and
loaded to 100 &times; 365 &micro;m fused silica capillary with tip
pulled with P-2000 laser (Sutter Instruments) in which the fritless
capillary had been packed with 10 cm of 5 &micro;m Zorbax Eclipse XDB
C<sub>18</sub> (Hewlett-Packard) and then with 4 cm of 5 &micro;m
Partisphere SCX (Whatman).  The column was connected to PEEK microcross
to split flow of pump so that flow was 150-250 nl/min and spray set at
1800 V to a Finnigan LCQ ion trap mass spectrometer.  Desolvation temperature
was set to 180&deg;.  SEQUEST was used to interpret MS/MS data: all
accepted results had a &Delta;Cn of 0.1 or more, which leads to high
confidence in SEQUEST result.  After equilibrating column for 5 min with
solvent A, a 2 min step each for 10, 15, 20, 30, 40, 50, 60, 70, 80, 90,
90, 100, 100% solvent C was done to move SCX proteins to the RP phase.
A 3 min flush then removes the ammonium acetate.  The peptides on
the RP resin are eluted by linear gradient of 0-10% B in 10 min, then
to 45% B over 80 min.  A new cycle starts with 100% solvent A and another
fraction is eluted with a higher salt concentration from the SCX.  In the
last two cycles, the column is washed with 100% solvent C and 100%
solvent D for 20 min to remove all peptides from the SCX.  The entire
analysis takes 15 cycles.  Data for 666 peptides was obtained by MS/MS
and searched against the database using SEQUEST, with identification of
288 proteins.
<p>
An offline prefractionation of peptides on 4.6 &times; 30 mm Zorbax
Eclipse XDB C<sub>18</sub> before loading on to the LC/LC columns.
2.1 mg protein from the soluble protein was loaded on to the column,
the peptides eluted in 3.5 ml steps of 20, 30, 50, 70, 100% solvent B.
Fractions were centrifugally evaporated then suspended in 20 &micro;l
solvent A, which was then run on the biphasic capillary column.
A 15-cycle MudPIT was then done on each of the five fractions, and more
than twice the number of unique peptides (1625) were found, with 607
proteins characterized.  Although not orthogonal, it avoids signal overload,
indicating chromatographic performance limiting analysis.
<p>
If peak resolution was the problem, a suitable ion-pairing agent was
necessary.  Alkanesulfonic acids make decent ion-pairing agents, but
perfluorinated carboxylic acids up to 4 carbons are volatile and best
for ESI-MS.  A downside is suppression of ionization, which can be nonetheless
minimized if concentrations are kept low.  Substitution of acetic acid
with 0.02% heptafluorobutyric acid (HFBA) was used with a 400 &micro;g
sample with MudPIT and 630 proteins were identified by 1665 peptides,
nearly 3 times more than with acetic acid.
<p>
In other work, a larger set of proteins in the yeast proteome were analyzed
by attempting to recover more than soluble extracts.  These included
lightly and heavily washed insoluble fractions.  After drying them in
centrifugal evaporator, these are resolubilized in 90% formic acid,
then digested with CNBr.  This likely presents a substrate amenable to
tryptic digestion.  Combining all results and doing LC using HFBA, a total
of 974 protein based on 2169 peptides were identified in the lightly washed
insoluble and 1036 proteins detected from 2114 peptides identified from
heavily washed insoluble fraction.  For each of the 15 cycles in a fraction
analysis, the number of peptides identified was fairly constant, averaging
383 proteins, with a range of 218 (cycle 14) to 505 (cycle 2) peptides.
Moreover each cycle provided unique peptide hits added to the final count,
from a low of 38 unique hits in cycle 14 to 342 hits in cycle 2.
<p>
Thus MudPIT works by separating by charge in the 1st dimension and
hydrophobicity in the 2nd.  The effectiveness of this approach was
tried on test proteins. Phosphorylase B was digested and run.  A plot
of peptide pI value vs. salt step cycle showed that peptides with
lower pI values elute early from the SCX column, those with higher
pI values in later cycles (solvent pH is about 3 in all washes).
The peptides K.DFYELEPHK.F and K.DFYELEPHKFQNK.T differ by four
amino acids at the C-terminal end.  The former has a pI 4.65 and elutes
in cycle 3 and the latter has a pI 5.45 and elutes in cycle 6.
In addition, the count of singly charged peptides eluting in the first
cycle is much higher than in subsequent cycles; 19% of peptides in the
1st cycle are singly charged, whereas in any subsequent cycle, at most
5% are singly charged.
<p>
Reproducibility in mapping is largely a function of chromatographic
performance (less about MS performance).  Two runs were done on
consecutive days to evaluate reproducibility, and peptides usually
eluted within a time window of less than 1 min.  Where variation is
seen, it is due to the difficulties in producing a consistent flow
rate with a microcross split (used to make nanoliter/min flow rates).
That all peaks were shifted in time was an indication of this.
Even with a particular retention time shift, the SEQUEST algorithm
correctly identifies the peptide from the spectrum.
<p>
An important aspect of LC/LC is the ability to improve the
detection of components present in low quantities of analyses of
complex peptide mixtures or the dynamic range. A theoretical
measure of the performance of multidimensional separations is
peak capacity.  Peak capacity of the reversed phase is determined by:
<p style="text-align:center;">
<i>n</i><sub>c</sub> = <i>L</i> / (4&sigma;)
<p>
where <i>L</i> is the total time over which the peptide elutes,
&sigma; is the average standard deviation of the peaks.  Peak width,
represented by 4&sigma; for a peptide eluting from reversed phase
is ~25 s and the gradient time is 90 min, providing an <i>n</i><sub>c</sub>
of 216.  Giddings has said that a peak capacity for a 2-D LC system is
the sum of the capacity in each cycle if a column is used repetitively
on successive cuts from a feeder column.  Given 15 fractions came
off the SCX column, the peak capacity is thus 3240.  The mass spectrometer
also represents a dimension with a peak capacity of at least 7
(the number of MS/MS spectra which can be acquired while a peak with
width 25 s is eluting).  Hence an LC/LC/MS/MS system has ~23,000
peaks it can analyze.
<p>
The <b>dynamic range</b> of a system is its ability to analyze both
low and high abundance substances, and is the ratio of a protein of
high abundance over a protein of low abundance.  9.8 fmol of G6P dehydrogenase
from <i>Leuconostoc mesenteroides</i> and 94.9 pmol of &alpha;-casein
were spiked into soluble <i>S. cerevisiae</i> cell lysate.  Three
peptides from the G6PDH were found, giving the system a dynamic range
of 10,000:1.  TDH2 from <i>S. cerevisiae</i> is one of the most abundant,
present at 10<sup>6</sup> copies per cell.  If the dynamic range is such,
then the system can sense as little as 100 copies per cell, although
this applies only to the method.  If a sample were applied where proteins
expressed at 100 copies per cell and the mass was at the attomole level,
this is below the ability of the system to detect.
<!-- ============================================================= -->
<p class="tags">
ICAT, quantitative proteomics
<p class="ref">
 <span class="authors">
Smolka MB, Zhou H, Purkayastha S, Aebersold R
 </span>
 <span class="year">
(2001)
 </span>
 <span class="jname">
Anal. Biochem.
 </span>
 <span class="jvol">
297,
 </span>
 <span class="pages">
25-31.
 </span>
<p class="reftitl">
Optimization of the Isotope-Coded Affinity Tag-Labeling Procedure for
Quantitative Proteome Analysis
<p class="Para1">
A set of standard proteins (BSA, ovalbumin, bovine &beta;-lactoglobulin,
and bovine &alpha;-lactalbumin)
were labeled using 20 &micro;g protein dissolved in 0.05% w/v SDS,
5 mM EDTA and 50 mM TrisHCl pH 8.3.  5 mM TBP for 10 min, then the ICAT
(ABi) is added for 2 h at room temperature (the reaction is done in the
dark).  The reaction is halted with DTT to 10 mM. 12.5% SDS-PAGE gels
are run and stained to look for a mobility shift caused by the added
442.2 Da of the ICAT reagent to each Cys.  Several different concentrations
of ICAT reagent were used to observe the partial labeling reaction:
these were 0, 50, 150, 300, 600, and 1200 &micro;M.  The reaction was
complete with 1200 &micro;M (1.2 mM) under these conditions.
To verify that the migration shift was due to complete labeling, the
band was digested in gel and extracted peptides examined on MS.  All
peptides predicted to have Cys showed a mass shift of 442.2 Da, thus
indicating that all residues were labeled.  &alpha;-Lactalbumin has
9 theoretical peptides, of which 7 are detected, in the MS set at its
range (600-2800).  One Cys containing peptide was not detected in the QSTAR
Pulsar Q-TOF using a MALDI source.  This was because its mass was beyond
the measured range (<i>m/z</i> 5096.35).  Another peptide (pentapeptide)
was also not detected, although it has no Cys.
<p>
In looking at other proteins, the concentration range was extended from
0 to 3 mM ICAT, and the proteins show variable reactivity.  At 3 mM
the ratio is more than 20-fold molar excess with the free thiol.
A 50:1 reagent concentration was also tried in a 1 ml volume, the ICAT
being 0.4 mM, but the labeling is incomplete.  Thus a minimal ICAT
concentration rather than an optimized ICAT:target ratio is more important.
<p>
The effect of denaturants was assessed in their ability to open up the
protein to reaction.  Different concentrations of ICAT (0-3 mM) in
0.5% SDS or 6 M urea were tried on the four proteins and the gels run.
In the end generally both urea and SDS at these concentrations were
effective, but urea did a better job of improving the reactivity of the
ICAT at lower concentrations.  For example, urea could effectively
allow complete reaction as 1.2 mM, but SDS use resulted in a less than
complete action.  A reaction of 0.5% was compared to 0.05% and there
was better partial labeling at a lower ICAT concentration:  this indicates
that SDS at high concentration interferes with the reaction, aside from
opening up the protein to make the Cys accessible.  It is advised that
any preliminary extraction or denaturation with high SDS be diluted
or dialyzed to remove possibly interfering concentrations.
With ovalbumin it was observed that neither urea nor SDS at any concentration
helped it to attain complete labeling.
<p>
A time course was also done using 1 mM ICAT with a solution
containing 0.05% SDS, 6 M urea, 5 mM EDTA, 50 mM TrisHCl pH 8.3,
and 5 mM TBP.  Reaction times were 10, 30, 60 min, 2 h, 6 h, and 24 h.
Results show that the reaction is nearly complete in 10 min, and complete
by 60 min.
<p>
Since high ICAT concentrations are required for labeling, minimal volumes
are thus necessary since ICAT costs can be come high.  With lower volumes
in the face of constant protein mass, the protein concentration increases
as well.  &beta;-Lactoglobulin was dissolved between 0.1 and 3.6
&micro;g/&micro;l in SDS, urea, EDTA, Tris as before, reduced with TBP,
and then treated with 1 mM ICAT for 2 h.  Results show that even at
3.6 &micro;g/&micro;l, which is a 1:1 molar ratio of ICAT:sulfhydryls,
all the protein is labeled, and none of it remains unlabeled or partially
labeled.
<p>
A yeast total protein extract was then treated.  300 &micro;g of
yeast protein precipitate was dissovled in 50 &micro;l of the ICAT
reaction buffer and then reduced with TBP.  The ICAT was 3 mM for 2 h.
Yeast proteins have an average mass of 40 kDa and contain 7 Cys residues,
and so the ICAT (175 nmol) was in 3-fold excess of predicted sulfhydryls.
2DGE with Ag staining was done.  The image showed no evidence of partial
labeling.  Unlabeled and labeled 2-D maps were compared to look for
shifts;  the shifts that were seen were strictly 2nd dimensional, and
not in the isoelectric points.  Thus the ICAT does not affect charged
groups, nor does urea cause carbamylation.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Beck JL, Colgrave ML, Ralph SF, Sheil MM
 </span>
 <span class="year">
(2001)
 </span>
 <span class="jname">
Mass Spec. Rev.
 </span>
 <span class="jvol">
20,
 </span>
 <span class="pages">
362-387.
 </span>
<p class="reftitl">
Practical Implications of Some Recent Studies in Electrospray Ionization
Fundamentals
<p class="abstract">
<b>REVIEW</b>.
<p>
<b>ABSTRACT</b>.
In accomplishing successful electrospray ionization analyses, it
is imperative to have an understanding of the effects of
variables such as analyte structure, instrumental parameters,
and solution composition. Here, we review some fundamental
studies of the ESI process that are relevant to these issues. We
discuss how analyte chargeability and surface activity are
related to ESI response, and how accessible parameters such as
nonpolar surface area and reversed phase HPLC retention time
can be used to predict relative ESI response. Also presented is a
description of how derivitizing agents can be used to maximize
or enable ESI response by improving the chargeability or
hydrophobicity of ESI analytes. Limiting factors in the ESI
calibration curve are discussed. At high concentrations, these
factors include droplet surface area and excess charge
concentration, whereas at low concentrations ion transmission
becomes an issue, and chemical interference can also be
limiting. Stable and reproducible non-pneumatic ESI operation
depends on the ability to balance a number of parameters,
including applied voltage and solution surface tension, flow
rate, and conductivity. We discuss how changing these
parameters can shift the mode of ESI operation from stable to
unstable, and how currentvoltage curves can be used to
characterize the mode of ESI operation. Finally, the characteristics
of the ideal ESI solvent, including surface tension and
conductivity requirements, are discussed. Analysis in the
positive ion mode can be accomplished with acidified
methanol/water solutions, but negative ion mode analysis
necessitates special constituents that suppress corona discharge
and facilitate the production of stable negative ions.
<!-- ================================================================ -->
<p class="tags">

<p class="ref">
 <span class="authors">
von Brocke A, Nicholson G, Bayer E
 </span>
 <span class="year">
(2001)
 </span>
 <span class="jname">
Electrophoresis
 </span>
 <span class="jvol">
22,
 </span>
 <span class="pages">
1251-1266.
 </span>
<p class="reftitl">
Recent Advances in Capillary Electrophoresis/Electrospray Mass Specrometry
<p class="abstract">
<b>ABSTRACT</b>.
In this review, the progress in hyphenation of capillary electrophoresis (CE) with electrospray
ionization-mass spectrometry (ESI-MS) since the article of Banks (Banks,
J. F., Electrophoresis 1997, 18, 22552266) is reported. In all capillary-based electromigration
techniques, such as capillary gel electrophoresis (CGE), capillary isotachophoresis
(CITP), capillary isoelectric focussing (CIEF), micellar electrokinetic chromatography
(MEKC), affinity capillary electrophoresis (ACE), as well as in the hybrid
techniques capillary electrochromatography (CEC), and pressurized capillary electrochromatography
(pCEC) progress has been made in experimental setups, and for
many groups of analytes, such as peptides, proteins, nucleotides, saccharides, drugs
and their metabolites, CE/ESI-MS has been successfully applied. Electromigration is
further miniaturized. New preconcentration methods allow the investigation of compounds,
which are not sensitively detected with ESI-MS. Coordination ion spray (CIS)
MS is another method for sensitivity enhancement by on-line formation of charged
coordination compounds.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Wehofsky M, Hoffmann R, Hubert M, Spengler B
 </span>
 <span class="year">
(2001)
 </span>
 <span class="jname">
Eur. J. Mass Spectrom.
 </span>
 <span class="jvol">
7,
 </span>
 <span class="pages">
39-46.
 </span>
<p class="reftitl">
Isotopic Deconvolution of Matrix-Assisted Laser Desorption/Ionization
Mass Spectra for Substance-Class Specific Analysis of Complex Samples
<p class="Para1">
With the increasing mass of the observed molecule, the mass signal
patterns become more complex.  Up to 1800 u the monoisotopic mass
is usually the most intense. Beyond that the monoisotopic signal becomes
less intense to the other isotopic variants, and even becomes lost
in the noise.  With FTICR-MS mass accuracy of large proteins is about 1 ppm
but the monoisotopic mass approaches 1 u of error.  Some methods make
use of determining monoisotopic mass from the average mass.  Analysis of
the isotopic cluster can also lead to monoisotopic mass determination.
Horn et al. use a least squares fitting approach to derive the monoisotopic
mass from ESI spectra of larger proteins.  Zhang et al. use maximum
entropy-based deconvolution of high mass molecules in ESI-FTICR-MS spectra.
<p>
<b>Materials and Methods</b>.  If the atomic composition of a molecule
is known, the natural isotopic distributions of atoms can be calculated
using probabilistic calculations.  For the average isotopic distribution
of peptides and variance for a given molecular mass, the distributions
are calculated for any possible amino acid in a given mass interval.
The count of compositions increases exponentially with mass.  Thus if
<i>N</i> is the count of residues in a peptide and <i>A</i> is the
count of possible components (20, for proteogenic peptides), the count
of potential peptides is:
<p style="text-align:center;" class="eqnserif">
&nu; = <i>A<sup>N</sup></i>
<p>
For position-insensitive counts (count of different amino acid sum
formulae), this is the number of combinations <i>c</i>:
<p style="text-align:center;">
<table class="eqnserif">
<tr><td rowspan="2"><i>c</i> = <span style="font-size:200%;">(</span>
 <td> <i>A</i> + <i>N</i> &minus; 1
 <td rowspan="2"><span style="font-size:200%;">)</span> =
 <td class="numer"> (<i>A</i> + <i>N</i> &minus; 1)!
<tr><td><i>N</i>
 <td><i>N</i>! (<i>A</i> &minus; 1)!
</table>
<p>
These are the count of masses that can be formed from a set of amino acids.
A plot from 600 to 1100 u with a mass interval of &Delta;M = 0.05 u shows
local maxima every 14 u.  This oscillation at every 14 u is logical as
this represents the mass of CH<sub>2</sub>, a very common structural repeat
in organics.  Below 400 u the periodicity disappears as there are lower
number of possible combinations.  Zoom in closer to a very narrow mass
range and it is clear that some masses drop to zero with respect to
possible combinations.  As mass increases, there are fewer zero population
intervals.  At 800 u, peaks are 0.3 u wide at base (0.14 u at half height),
with a 0.7 u non-populated area, but at 1500 u, the nonpopulated clusters
vanish.  Note that sequence permutations only show up as increasing the
peak height, not in a wider variation of masses.  Each cluster can
be reduced to a nominal mass, an integer value, and the average mass
of the cluster can approximated as the nominal mass &times; 1.00043.
<p>
Generally proteins will be considered to consist of C, H, N, O, and S.
The first 3 have only two isotopes, with the lightest having abundances
of &gt;98%.  The lightest isotopes or O and S are also the most abundant, but
three isotopes are to be considered for each.  The heaviest stable isotope
of S has a relatively high abundance of 4.22%.  The probability of certain
isotope combinations can be calculated from binomial distributions.
The isotopic with the lowest mass, the monoisotopic mass, has mass M
(to be considered the mass of the molecule).  Masses of 1 u more represent
a mix of molecules with atoms with the next heavier isotope.  There is
only one isotopic set of atoms for the monoisotopic mass M, but others
represent a heterogeneity of the atoms with different isotopes.  The
exchange of a <sup>12</sup>C for <sup>13</sup>C, a <sup>14</sup>N
for a <sup>15</sup>N, or a <sup>1</sup>H for a <sup>2</sup>H will be a
factor in determining the intensity of the heavier isotopes.  For the elements
with three or more significant representations of stable isotopes,
a polynomial distribution is used in the calculation.  A plot of the
ratio (as a percent) of the predicted intensity of the 2nd isotopic peak
relative to the predicted intensity of the 1st isotopic peak against the
monoisotopic mass (u) illustrates the pattern for all possible amino
acid combinations (not sequence permutations).  A cross in the plot
represents one such mass.  One then can plot the count of the crosses as
a function of the calculated intensity ratios.  For example one can
show the distribution of amino acid combinations with a 0.1% intensity
interval plotted for the cluster at mass 1000.4 u.  The average of the
distribution can then be calculated at 54 &plusmn; 3% (range 45-70%).
The presence of a particular atom can be determined from the isotopic
pattern.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Chernuschevich IV, Loboda AV, Thomson BA
 </span>
 <span class="year">
(2001)
 </span>
 <span class="jname">
J. Mass Spectrom.
 </span>
 <span class="jvol">
36,
 </span>
 <span class="pages">
849-865.
 </span>
<p class="reftitl">
An Introduction to Quadrupole-Time-of-Flight Mass Spectrometry
<p class="Para1">
<b>TUTORIAL</b>.  Q-ToF hybrid MSes were first commercially introduced
6 years ago, combining ToF in MS and MS/MS modes with ESI and APCI.
QqTOF will be used to refer to the technique and instrument where Q is
the mass-resolving quadrupole, q is the r.f.-only quadrupole or
hexapole collision cell, and TOF is the time-of-flight spectrometer.
ESI-TOF was first used to demonstrate orthogonal injection from
a high-pressure ion source, leading to QqTOF.  The configuration can
be seen as adding a mass-resolving quadruople to an ESI-TOF or as replacing
the third quadrupole in a triple quad by a TOF MS.  Sensitivity, mass resolution,
and mass accuracy all increase for both precursor and product ions.
More specialized modes of the precursor such as neutral loss
and multiple reaction monitoring (MRM) scans in triple quads are not
replaceable.  In general the QTOF was generally seen as a qualitative
analyzer, but can be used quantitatively once dynamic range issues are
resolved.  While the triple quad still has the better sensitivity,
the QqTOF provides better specificity through its better resolution.
MALDI ion sources have also been developed for QqTOF.  The explanation
of how the QqToF works is fairly much the same as the triple quad;
the triple quad has been described by Yost and Boyd.
<p>
The usual QqTOF has a r.f. Q0 quadrupole for collisional damping, and
so typically has a triple quad with TOF.  A vacuum in the extraction
cone of the ion source is set up at 0.5 torr.  At Q0 the vacuumm is
set at 1e-2 torr.  The resolving Q1 quadrupole is set up at 4e-5 torr,
while the Q2 collision quad has argon gas contributing to 1e-2 torr.
The TOF tube is set up at 5e-7 torr.  In the case of one system,
Q0 and Q2 are hexapoles.  For MS mode only instruments Q1 is operated in
r.f. mode as a mass filter.  Spectra from the TOF have the power of its
resolution and mass accuracy; all ions are measured in a pulse
rather than continuous (scanning) mode.  Q1 scans can still be done
using th TOF as ion current detector only; generally this mode is used
for Q1 calibration and tuning only.
<p>
Q0 is used for collisional cooling of ions from the ESI source and for
focusing.  Q0 and Q2 are operated in r.f.-only mode, creating a potential
well with radial confinement of precursor and/or fragment ions.  These
r.f. quadrupoles or hexapoles are operated at several millitorr and
both radial and axial collisional damping of ion motion is obtained.
Thermalization of ions by collision reduces energy spread and beam
diameter.  Re-acceleration of ions axially upon r.f. quad exit provides
necessary energies and near-thermal energy spreads.  In MS/MS mode Q1
is set to transmit the parent ion only with mass window of 1-3 thomson
(Th) depending on whether isotope cluster transmission is desired.
Acceleration of 20-200 eV into the Q2 collision cell containing argon
or nitrogen produces fragments that are cooled and focused.  Since TOF
is more sensitive to incoming ion beam <q>quality</q> than a Q3
triple quad, fine tuning in this step is essential.
<p>
Collision cells are still pressurized with non-reactive gas to provide
the collisional cooling that also helps focusing.  After leaving the
collision cell the ion is focused.  Initially it is a field-free entry.
Then a pulsed electric field is applied (several kHz) to push ions
orthogonally where ions acquire several keV per unit charge. Ions travel
through a field-free drift space where TOF separation occurs.  Detectors
are composed of aligned microchannel plates configured as a chevron.
The electrostatic mirror and detector are similar to those used with
pulsed ionization sources (MALDI).  Note that positive ions (in that mode)
are accelerated from ground to high negative voltages in the TOF of
a QqTOF instrument while high positive to ground voltages are used in
the MALDI-TOF.  This inverted configuration requires and extra shield
in the TOF chamber to create field-free drift region floated at high
potential.  All spectra in the TOF are recorded using time-to-digital
(TDC) converter.
<p>
In mass analyzer mode with both r.f. and d.c. components applied to rods,
and the reduced Mathieu parameters are used to characterize amplitudes
of both components:
<table style="text-align:center;" class="eqnserif">
<tr><td rowspan="2"><i>q</i><sub>M</sub> =
 <td class="numer">4<i>eV</i>
 <td rowspan="2" style="width:3em;">&nbsp;
 <td rowspan="2"><i>a</i><sub>M</sub> =
 <td class="numer">8<i>eU</i>
<tr><td>(<i>m/z</i>)&omega;<sup>2</sup><i>r</i><sub>0</sub><sup>2</sup>
 <td>(<i>m/z</i>)&omega;<sup>2</sup><i>r</i><sub>0</sub><sup>2</sup>
</table>
<p>
Here <i>e</i> is the electron charge constant in coulombs, <i>V</i> and
&omega; are amplitude and frequency of the r.f. voltage, respectively,
<i>U</i> is value of d.c. voltage, and <i>r</i><sub>0</sub> is inscribed
radius of quadrupole.  Typically with <i>q</i><sub>M</sub> = 0.706
and <i>a</i><sub>M</sub> = 0.237, a narrow <i>m/z</i> window is obtained.
In r.f. mode only, <i>a</i><sub>M</sub> = 0 and the quadrupoles are
high-mass filters:  ions with <i>m/z</i> values below cut-off value corresponding
to <i>q</i><sub>M</sub> = 0.908 are rejected.  Transmission of ions
with high <i>m/z</i> values suffers from poor focusing:  the depth of the
effective potential well is inversely proportional to <i>m/z</i>.  Thus
ions can be lost in any of the three multipoles on their way to the TOF
section.  In single MS mode set for r.f.-only mode for Q0, Q1 and Q2,
ions covering an order of magnitude in <i>m/z</i> are transmitted simultaneously
into the TOF.  For a wider <i>m/z</i> range, the r.f. voltage is modulated
(stepped or ramped) between two or more r.f. levels during acquisition,
providing larger transmission window over time.
<p>
With scanning mass analyzers, only one ion type with particular <i>m/z</i>
is recorded at any given time; the duty cycle is brought to below 0.1%
with full spectrum recording, and is inversely proportional to mass
resolution since narrow transmission window and smaller step size are
required for higher resolution in a quadrupole.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Cramer R, Corless S
 </span>
 <span class="year">
(2001)
 </span>
 <span class="jname">
Rapid Commun. Mass Spectrom.
 </span>
 <span class="jvol">
15,
 </span>
 <span class="pages">
2058-2066.
 </span>
<p class="reftitl">
The Nature of Collision-Induced Dissociation Processes of Doubly
Protonated Peptides:  Comparative Study for the Future Use of
Matrix-Assisted Laser Desorption/Ionization On a Hybrid Quadrupole
Time-Of-Flight Mass Spectrometer in Proteomics
<p class="Para1">
MALDI-QTOF was done with Pulsar (MDS Sciex) mass spectrometer with prototype
UV-MALDI source.  ESI-MS/MS was on MDS Protana A/S with nanospray source.
Sample introduction with syringe at 0.2 &micro;l/min was done.  MALDI-TOF
was on PerSeptive Biosystems Voyager Elite XL.  Analytes were purchased
except for phosphorglycerate mutase 1 from 2Dgel.
<p>
Matrices were first examined for optimal selection.  Tryptic &alpha;-casein
digests were measured in linear (axial) mode in UV-MALDI-TOF MS because
there is metastable decay in reflectron (lower resolution).  There is
a lower sensitivity in high mass range for MALDI QTOF MS based on the
comparison of instrument modes.  S/N dynamic range was better than for
linear MALDI-TOF-MS.  DHB-matrix peptides had better intensities than
from CHCA or caffeic acid, possibly because of a lower background.
This is because <q>cold</q> matrices probably have less decomposition.
The decoupling of the desorption event from mass analysis in the QTOF
but not linear TOF, converting MALDI pulse into quasi-continuous ion beam
allows for much higher laser desorption.  DHB was also better at yielding
+2 ion and thus used for ESI and MALDI-MS/MS.
<p>
+1 charge Glu-Fib B was recorded on UV-MALDI-PSD-TOFMS, UV-MALDI-QTOF-MS/MS,
and ESI-QTOFMS/MS, and +2 ion on UV-MALDI-QTOF-MS/MS and ESI-QTOF-MS/MS.
The fragmentation spectra for +1 ion precursor look almost identical
and independent of instrument or fragmentation method.  Increasing
collision energy decreases absolute signal intensities of precursor
and high mass range fragments, shifting to higher yields of low mass
fragments, without really causing new ions to form.  With the +2 precursor
the fragmentation pattern changes significantly, with 8 or 9 of 10 fragments
being <i>y</i> ions (only 5 for +1 precursor).
<p>
The tryptic peptide of horse myoglobin (V<sup>17</sup>-R<sup>31</sup>)
was examined with +1 and +2 precursors.  In the +1 ion, all fragmentation
spectra (including PSD) show high intensities for <i>b</i><sub>10</sub>
and <i>b</i><sub>11</sub> along with <i>y</i><sub>11</sub> above
<i>m/z</i> 900.  Only <i>y</i>-type ions are seen with the +2 ion.
The database search showed a significantly higher probability for
horse myoglobin match from fragmentation data of the +2 precursor.
In contrast +1 ion data showed typically very low confidences (probabilities),
even though twice as many fragment ions are observed for +1 ion MS/MS
spectra.  Matching for the +2 ion data was to <i>y</i>-type ions,
twice as many compared to <i>a</i> and <i>b</i> type ions combined.
<p>
Ag-stained protein was in-gel digested (known to be phosphoglycerate mutase),
and analyzed by linear MALDI TOF and MALDI-QTOF.  Both gave good PMF
spectra.  The calibration on the instrument was even 2 days old, and more
than 80% of the matched peptides were within 5 ppm.  Those peptides
matched above the calibration range (&gt; <i>m/z</i> 1700) were still matched
within 15 ppm.
<p>
CID-MS/MS spectra of the tryptic peptide H<sup>11</sup>-R<sup>21</sup>
+1 and +2 precursor were more complex.  More than twice as many <i>a</i>
and <i>b</i> type ions were identified and only half as many <i>y</i> type
for the +1 compared to the +2 precursor, although both spectra were good
enough for unambiguous identification.  For all UV-MALDI-QTOF the
rel. abund. of +2 ion did not exceed 20% of +1 ion, and when laser
intensity was raised to improve ion count, the rel. abund. fell to 10%.
Production of +2 ions in MALDI is a function of both the peptide sequence
and the matrix.  The cold type matrix DHB does better at producing +2 ions.
<p>
UV- and IR-MALDI were compared including their special matrices.
For Glu-Fib, the +2 ion was about 20-25% of that of the +1 in IR-MALDI
using succinic acid matrix.  No +2 ions were seen with DHB in UV-MALDI.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Macmillan D, Bill RM, Sage KA, Fern D, Flitsch SL
 </span>
 <span class="year">
(2001)
 </span>
 <span class="jname">
Chem. Biol.
 </span>
 <span class="jvol">
8,
 </span>
 <span class="pages">
133-145.
 </span>
<p class="reftitl">
Selective In Vitro Glycosylation of Recombinant Proteins:  Semi-Synthesis
of Novel Homogeneous Glycoforms of Human Erythropoietin
<p class="Para1">
EPO is a glycoprotein product of fetal liver and adult kidney.  Elevated
serum oxygen results in reduced EPO expression and low oxygen results in
increased EPO.  When the kidney fail chronically EPO is lowered and anemia
can result.  I.V. admin of recombinant human EPO eliminates the need for
transfusions.  As many as 58 glycosylation variants have been identified.
Glycosylation does improve biological activity.  It has been shown that
<i>O</i>-linked glycosylation at Ser-138 is not essential, but it may
serve in folding stabilization.  <i>N</i>-linked glycans at Asn-24, -38,
and -83 are essential.  Sialylation increases circulatory half-life.
One group has made it possible to insert Cys in the sequence by
site-directed mutagenesis, and these can be glycosylated using iodoacetamide
derivatives in a chemical synthesis.  rhEPO with Asn&minus;&gt;Cys mutations
(N24C, N38C, N83C).  One problem in the use of rhEPO is that it normally
contains Cys for disulfide bond formation.
<p>
His<sub>10</sub>-WT EPO with mutations N24C, N38C, and N83C were overexpressed
from <i>E coli</i> B834(DE3) using pET16b expression system.  The EPO
was highly aggregated due to disulfide formation, sometimes with itself,
sometimes to glutathione.  The N-terminal His tag allows for one-step
cleaning by metal chelation (Ni<sup>2+</sup>) affinity chromatography.
Isoforms having similar MW were obtained with the overexpression.
These were detected using mAbs at the N-terminus of rhEPO.  The lower
band had a mass of 21097 Da by MALDI-ToF and is most likely
His<sub>10</sub>-C38hEPO (calcd mass 20906 Da).  The upper band carries
an extended sequence, which results from a failure of an inefficient stop
(TGA) sequence that has been see with pET vectors.  26 bp downstream
from 3&prime;-end is an <i>Stu</i>I site into which is inserted a stop
sequence (TAA).  Protein yields were 13 &micro;g/ml.
<p>
Mass accuracy of the MALDI ToF is about 50-150 Da within the calcd masses.
A better resolution was obtained with LC-ESI-MS on reduced protein samples.
These are eluted from Ni<sup>2+</sup> affinity as aggregates, then
reduced in 10-50 mM DTT or TCEP before LC-MS to provide monomers in highly
charged states.  The mass was within 5-20 Da of the calculated.
<p>
Affinity-purified rhEPO Cys mutants were dialyzed against 6 M guanidine
HCl in 50 mM Tris HCl pH 8.0 in a nitrogen atmosphere.  The protein
then was diluted 1:50 (to about 1 &micro;M), then refolded in 2%
<i>N</i>-lauroyl sarcosine in 50 mM Tris HCl pH 8.0 with 40 &micro;M
copper sulfate.  A syringe filter with a 1.5 ml Ni<sup>2+</sup>-charged
affinity resin was then used for concentrating the dilute protein as
it was passed over.  Another <q>pulsed</q> refolding strategy
was used prior to concentration (see Lilie et al. <i>Curr. Opin.
Biotechnol.</i> <b>9</b>: 497-501, 1998).  Elution from the nickel
affinity columns is using 200 mM imidazole in 6 M guanidine HCl.
The glycosyl iodoacetamide is then added to 10 mM.  The reaction is
monitored by sampling aliquots over time and then checking reaction extent
on LC-ESI-MS.  Even as high as 50 mM the reactions did not go to
stoichiometric completion, but at least one glycosyl (GlcNAc, in this case)
was added.
<p>
Note that the presence of imidazole assisted the glycosylation.  In reactions
where protein was concentrated by precipitation instead affinity resin,
with resuspension in buffer having no imidazole, glycosylation could be
seen on the N-terminal and possibly on the His<sub>10</sub> tag.
This was verified by use of 2-nitro-5-thiocyanaot-benzoic acid (NTCBA)
cleavage of the peptide backbond and then ESI-MS.  There were differences
in the way the mutants glycosylated:  the N38C mutant was &gt;60%,
while the other two were 30%.  And glycosylation was better under denaturing
conditions;  samples eluted from affinity resin and refolded before
glycosylation did not modify as much.
<p>
The neoglycoproteins were separated from non-modified using a lectin
column (<i>T. avesticum</i> wheat germ agglutinin immobilized to 4% agarose
beads).  After glycosylation, products are dialyzed against 3 M guanidine
HCl in 50 mM Tris HCl pH 7.5 with 1 M NaCl, 1 mM MnCl<sub>2</sub>, and
1 mM CaCl<sub>2</sub> and then applied to column.  Unbound material is
washed out, and then the glycoprotein is eluted with 0.5 M GlcNAc.
Fractions of elution are monitored by LC-MS.
<p>
The MS work established that only one residue had been glycosylated, so
it became necessary to determine which site was modified.  Unglycosylated
protein is also employed since it is useful in comparison as an internal
calibrant.  Material was prepared at 60 &micro;M in a 3 M guanidine HCl
solution and digested with trypsin or chymotrypsin for 1-16 h.  Unfortunately
the generated peptides produced regions of interest that did not bind to
reversed phase (N24C and N38C mutants).
<p>
Proteins bound to the affinity column were directly digested on-column.
The material coming off the column was collected, and then bound material
removed with imidazole.  Chymotrypsin was effective at creating fragments
of 4300 and 8000 Da.  These were further treated with trypsin or V8 protease
in 50 mM ammonium bicarbonate pH 8.  MS analysis showed that glycosylation
occurred between residues 21 and 45 according to the trypsin digest,
and residues 24 and 43 on the V8 digest.  The fragments not hold C24 and
C38 mutants, but also the C29-C33 disulfide bond.  The peptide resists
V8 treatment (targets Glu).  The ion is a +3 ion whose mass is two units
smaller than the calculated, indicating that a disulfide bond is present.
Glycosylation does not occur at C7 and C161, which form disulfides linking
N- and C-termini in the native EPO.
<p>
Also tried was NTCBA treatment:  this chemical cleaves peptide bonds
adjacent to free cysteine thiols.  In reduced EPO, NTCBA is effective
in producing peptides cleaved at 7, 29, 33, and 161 in wild-type EPO;
for the mutants, it cleaved at their sites as well.  It was not possible
to determine if C38 glycosylation occurred with this chemical because its
use resulted in deglycosylation and cleavage.  One glycosylated fragment
appeared in the N-terminus (before Ile-6), and since there is no Cys here,
it indicates another nucleophile.
<p>
The aggregation of recombinant EPO expressed in <i>E coli</i> has been
subjected to structural analysis.  Crystal structures of EPO/EPO-R complex
and NMR analysis of recombinant EPO with hydrophilic subsitutions and additions
was done.  Dynamic light scattering (DLS) reveals the effect of glycosylation
since it is insensitive to high ionic strength buffers necessary to solubilize
EPO.  Light scattered to a right (90&deg;) angle can be assayed as it
passes through an avalanche photodiode into autocorrelator.  Changes in
light scattering over time allow the determination of the translational
diffusion coefficient (<i>D</i><sub>T</sub>) and then the hydrodynamic
radius (<i>R</i><sub>H</sub>) with the Stokes-Einstein equation:
<p class="eqnserif">
<i>R</i><sub>H</sub> = <i>k</i><sub>B</sub><i>T</i>/6<i>&pi;&eta;D</i><sub>T</sub>
<p>
<i>k</i><sub>B</sub> is Boltzmann constant, <i>T</i> is absolute temperature,
<i>&eta;</i> is the solvent viscosity.
<p>
Non-reducing SDS-PAGE already indicated that the refolded proteins were
monomeric and not aggregated.  DLS was used to get <i>D</i><sub>T</sub>
when was used to calculate <i>R</i><sub>H</sub>, from which an estimated
MW was made.  DLS was done with increased concentration of guanidine HCl
(from 0 to 6 M guanidine HCl in 0.2 M steps).  At about 2 M precisely, there
was a sharp transition from a monomer weight of 25-30 kDa to about
250 kDa.  Circular dichroism spectroscopy had shown such a transition
at 1.75 M guanidine HCl for unfolding in EPO derived from CHO cells, and
at 1.2 M GuanHCl for <i>E coli</i>-derived EPO.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Schwarz-Selinger T, Preuss R, Dose V, von der Linden W
 </span>
 <span class="year">
(2001)
 </span>
 <span class="jname">
J. Mass Spectrom.
 </span>
 <span class="jvol">
36,
 </span>
 <span class="pages">
866-874.
 </span>
<p class="reftitl">
Analysis of Multicomponent Mass Spectra Applying Bayesian Probability
Theory
<p class="abstract">
<b>ABSTRACT</b>.
In this paper we develop a method for the decomposition of mass spectra of gas mixtures, together with
the relevant calibration measurements. The method is based on Bayesian probability theory. Given a set
of spectra, the algorithm returns the relative concentrations and the associated margin of confidence for
each component of the mixture. In addition to the concentrations, such a data set enables the derivation
of improved values of the cracking coefficients of all contributing species, even for those components for
which the set does not contain a calibration measurement. This latter feature also allows one to analyze
mixtures that contain radicals in addition to stable molecules. As an example, we analyze and discuss
the mass spectra obtained from the pyrolysis of azomethane, which contain the radical CH3 apart from
nitrogen and C1- and C2-hydrocarbons.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Griffiths WJ, Jonsson AP, Liu S, Rai DK, Wang Y
 </span>
 <span class="year">
(2001)
 </span>
 <span class="jname">
Biochem J.
 </span>
 <span class="jvol">
355,
 </span>
 <span class="pages">
545-561.
 </span>
<p class="reftitl">
Electospray and Tandem Mass Spectrometry in Biochemistry
<p class="abstract">
<b>REVIEW</b>.  Proteins can be identified at the sub-picomole level with
MS (neuropeptides have been identified at the zeptomole&mdash;10<sup>&minus;21</sup>
mol&mdash;level).   Tandem MS (MS/MS)  takes a precursor ion and selects
it for fragmentation to provide product ions with structural information.
Ionization was usually done by <b>electron impact (EI)</b> or <b>chemical
ionization (CI)</b> methods, but these methods require that the molecule
to be ionized already be in the gas phase.  Polar thermally labile samples
are not amenable to these ionization methods without prior derivatization.
<p>
Barber and others (1981) introduced fast atom bombardment (FAB) which
permitted the ionization of polar thermally labile molecules up to
a few thousand daltons.  This worked well for already preformed ions
(protonated, deprotonated, sodiated) in solution.  Generally a 1 &micro;l
sample with perhaps [M+H]<sup>+</sup> ion and their negatively charged
counter ions are dissolved in 2 &micro;l FAB matrix, typically glycerol,
and then put into the vacuum.  These are then bombarded with Xe atoms
accelerated to 8 keV or Cs<sup>+</sup> ions energized to 20 keV.  The
[M+H]<sup>+</sup> ions are put into gas phase with protonated matrix
clusters.  This soft ionization method worked for species below about
1000 Da.  Chemical noise can be significant with FAB however, although
a continuous flow system helps to minimize the problem.
<p>
Karas and Hillenkamp in 1988 at an international conference described
MALDI of proteins with masses greater than 10 kDa.  Typically MALDI
is used with TOF analyzers since they are both work using pulsed streams
of ions rather than a continuous stream.  Sample is co-crystallized
with matrix (like FAB, a matrix is the mediator), and a laser supplies the
energy.  Ionized (protonated and deprotonated) sample and matrix molecules
then are vaporized and mass-analyzed in the TOF which has a theoretically
unlimited mass range measurement ability.  Peptide mass fingerprinting
(also called peptide mass mapping) is the typical use of MALDI-TOF of peptides.
The masses are searched against a protein or DNA sequence database and the
protein identified by this means.  Like FAB, chemical noise from matrix
makes the low mass range impossible to use to get data.  But peptides tend
to have <i>m/z</i> greater than the region.
<p>
Electrospray ionization is more versatile than MALDI in ionizing a wider
range of polar molecules.  Sample is ionized to [M+H]<sup>+</sup> and/or
[M-H]<sup>&minus;</sup> ions
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Bafna V, Edwards N
 </span>
 <span class="year">
(2001)
 </span>
 <span class="jname">
Bioinformatics
 </span>
 <span class="jvol">
17,
 </span>
 <span class="pages">
S13-S21.
 </span>
<p class="reftitl">
SCOPE: A Probabilistic Model for Scoring Tandem Mass Spectra Against A
Peptide Database
<p class="Para1">
MS/MS spectra generated from CID show fragmentation of peptides in
a stochastic process whose factors include the physicochemical properties
of the peptide and the collision energy.  N-terminal ions form an
<i>a, b, c</i> ion series while C-terminal fragments form an <i>x, y, z</i>
ion series.  The series <i>a, b</i> and <i>y</i> ions are the most common.
<p>
The core of most MS/MS spectrum analysis software involves three modules:
<ul>
<li><b>interpretation</b>:  the input is an MS/MS spectrum and output
<i>interpreted</i> MS/MS data, which include sequence tags, parent peptide
mass, combinations of tags, and any molecular masses
<li><b>filtering</b>:  the input is the interpreted MS/MS data and a
peptide sequence database, and the output is a list of candidate peptides
which accounts for the MS/MS spectra
<li><b>scoring</b>:  input is the candidate peptide list and the MS/MS
spectrum, and the output is a score (possibly including <i>p</i> value)
of the correlation; candidate peptides are sorted/ranked by score.
</ul>
<p>
The filter would do more to eliminate peptides that could not have generated
the spectra rather than looking for those that would.
<p>
Some <i>de novo</i> sequencing algorithms rely entirely on the MS/MS
spectrum as data.  They depend on very clean spectra, with the greater
intensities corresponding to peptide bond breaks and also on little noise.
Database searching algorithms rely more on scoring such that the peptide
with the highest score or lowest <i>p</i> value explains the spectrum;
it depends on database completeness and a good scoring mechanism.
This paper focuses on scoring.
<p>
First some terms should be defined.  A <b>MS/MS spectrum</b> <i>S</i>
&isin; <b>R</b><sup><i>k</i></sup><sub>+</sub> is a vector of real
positive numbers specifying <i>k</i> observed <i>m/z</i> values of the
spectral peaks.  A <b>peptide</b> <i>p</i> &isin; &xi;<sup><i>n</i></sup>
is a sequence of <i>n</i> amino acid residues from the alphabet of
amino acid symbols.  <i>Fragment space</i> &phi;(<i>p</i>) is an enumeration
of all fragment <i>m/z</i> values produced by peptide <i>p</i>.
All possible <i>m/z</i> values are generated and each element of
&phi;(<i>p</i>) is a fragment-charge state pair:
<p class="eqnserif">
 &phi;(<i>p</i>) = {(<i>a</i><sub>1</sub>, <i>i</i>), (<i>b</i><sub>1</sub>, <i>i</i>),
  (<i>y</i><sub>1</sub>, <i>i</i>),..., (<i>a<sub>n</sub></i>, <i>i</i>),
  (<i>b<sub>n</sub></i>, <i>i</i>), (<i>y<sub>n</sub></i>, <i>i</i>),
  <i>i</i> = 1, 2, 3}
<p>
Where higher energy ionization occurs, double cleavage and no cleavage
events and more ion types are variables.  A mass-to-charge ratio
of fragment <i>f</i> &isin; &phi;(<i>p</i>) is shown as <i>m/z</i>(<i>f</i>).
<p>
<b>Fragmentation Pattern</b> <i>F</i> &sube; &phi;(<i>p</i>) is a subset
of the fragment space respresenting observed fragments generated from
peptide <i>p</i>.  <b>Fragmentation Space</b> &xi;(<i>p</i>) of peptide
<i>p</i> is the set of all fragmentation patterns of <i>p</i>, or
&xi;(<i>p</i>) = {F : F &sube; &phi;(<i>p</i>)}.  <b>Noise</b> is
considered to be any peak of <i>S</i> for which &phi;(<i>p</i>) provides
no explanation of the peak other than it is noise.
<p>
A peptide <i>p</i> is scored by the probability that the observed spectrum
<i>S</i> was generated by <i>p</i>.  &psi;(<i>S</i>|<i>p</i>) is the
probability density function for a random vector <i>S</i> representing
the MS/MS spectrum given <i>p</i>.  A database is searched for petpide
<i>p</i>* which satisfies:
<pre class="eqnserif" style="text-align:left;">
                       <i>p</i>* = arg max &psi; (<i>S</i>|<i>p</i>)
                                       <i style="font-size:70%;">p</i>
</pre>
<p>
The formal description of the two-step model of fragmentation is:
<p class="eqnserif">
 &psi; (<i>S</i>|<i>p</i>) = <span style="font-size:125%;">&sum;</span><sub
 style="margin-left:-2em;vertical-align:-1.5em;"><i>F</i>&sube;&phi;(<i>p</i>)</sub>
 &psi;(<i>S</i> |<i>F</i>, <i>p</i>) Pr(<i>F</i> |<i>p</i>)
<p>
Pr(<i>F</i> |<i>p</i>) is the probability of a particular fragmentation
pattern of a peptide, the complex process of fragmentation is modeled
in this value.  &psi;(<i>S</i> |<i>F</i>, <i>p</i>) is the probability
that spectrum <i>S</i> was generated by fragmentation pattern <i>F</i>
of peptide <i>p</i>.  The equation for &psi; (<i>S</i>|<i>p</i>) shows
that it is computed as the summation over fragmentation space, which is
an exponentially sized set.  It cannot be computed as part of a database
search since the cost is too high.
<p>
For getting Pr(<i>F</i> |<i>p</i>), we generate the list of observed
fragments.  Estimate likelihood of a peptide bond cleavage by counting
the number of times it is observed.  Some peptides will favor one cleavage
or another.  Clustering and data mining on the dataset will yield properties
causing changes in the likelihood.  Acidic amino acids in peptide make
the neutral water loss ion type cleavages more likely.  A database
of expertly curated spectra of peptides does not exist, so probabilities
are estimated by other means.
<p>
For the probability density &psi;(<i>S</i> |<i>F</i>, <i>p</i>), one must
resolve the issue of peaks as noise rather than signal.  In general
a unique <i>m/z</i> value in fragment space generates one spectral peak.
Each spectral peak is the observed <i>m/z</i> of at most one of the unique
<i>m/z</i> values in fragment space.  Assignment of peaks to fragments
must be <i>non-crossing</i>.  Thus for all fragments <i>f</i><sub>1</sub>,
<i>f</i><sub>2</sub> and spectral peaks <i>S</i><sub>1</sub>, <i>S</i><sub>2</sub>,
if <i>m/z</i>(<i>f</i><sub>1</sub>) &lt; <i>m/z</i>(<i>f</i><sub>2</sub>)
and <i>S</i><sub>1</sub> &lt; <i>S</i><sub>2</sub>.  Zero probability
is assigned to fragmentation patterns violating the assumptions.
Peaks in the spectrum are then assigned either to noise fragments or
to a unique from the fragment space &phi;(<i>p</i>).  Only fragmentation
patterns <i>F</i> &sube; &phi;(<i>p</i>) with |<i>F</i> | = <i>k</i>
have non-zero probability mass.
<p>
A stronger statement is that peak <i>S<sub>i</sub></i> is equal to fragment
<i>f</i> such that it is generated by it, and <i>S</i> = (<i>S</i><sub>1</sub>,
<i>S</i><sub>2</sub>,...,<i>S<sub>k</sub></i>) is tandem MS spectrum ordered
with respect to <i>m/z</i>.  The fragmentation pattern <i>F</i> &sube;
&phi;(<i>p</i>), |<i>F</i> | = <i>k</i> is an arbitrary pattern
with observed fragments <i>f</i><sub>1</sub>,...,<i>f<sub>k</sub></i> &isin;
<i>F</i> ordered by <i>m/z</i> values.
<p>
The non-crossing and uniqueness assumptions means only one assignment of
spectral peaks to fragments.  With a Bayesian notion of valid assignments,
then
<p class="eqnserif">
&psi;(<i>S</i> |<i>F</i>, <i>p</i>) = &psi;(<i>S</i> | &cap;<sup><i>k</i></sup><sub><i>i</i>=1</sub>
[<i>S<sub>i</sub></i> =<span
style="vertical-align:1.1em;margin-left:-0.6em;font-size:60%;"><i>M</i></span> <i>f<sub>i</sub></i>], <i>F</i>, <i>p</i>)
<p>



<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Han DK, Eng J, Zhou H, Aebersold R
 </span>
 <span class="year">
(2001)
 </span>
 <span class="jname">
Nat. Biotechnol.
 </span>
 <span class="jvol">
19,
 </span>
 <span class="pages">
946-951.
 </span>
<p class="reftitl">
Quantitative Profiling of Differentiation-Induced Microsomal Proteins
Using Isotope-Coded Affinity Tags and Mass Spectrometry
<p class="Para1">
Cell line HL-60 is used an in vitro model for cell differentation and
there is much data with this cell line.  Treatment with 12-phorbol-13-myristate
(PMA) will cause change from a nonadherent state to monocyte-like,
adherent, spreading  and morphologically altered cell.  One would assume
with the transition from nonadherence to adherence, the cell surface protein
profile would be extensive.  The microsome fraction was isolated from
untreated and treated HL-60 cell as follows.  Lysates from 100 million
cells of control or treated were differentially ultracentrifuged.
Pellets were resuspended in 0.5% SDS in buffer, and 100 &micro;g protein
was reduced and labeled with either a light d0 (control) or heavy
d8 (PMA) form of a sulfhydryl-specific ICAT reagent.  Samples are combined,
trypsinized, and then resolved on multidimensional chromatography:
cation-exchange chromatography is used to separate into 30 fractions and
remove remaining proteases, ICAT, and SDS.  Then each of these fractions
is passed through avidin affinity column.  Biotinylated peptides are
recovered and run on microcapillary LC-ESI-MS/MS.  If fractions were
taken directly from the cation column, the mass spectrum would be too messy
with multiple contaminants.
<p>
The mass spectrum is full of paired signals, differing by 8 u from one another
for each Cys present on a peptide.  Automated precursor ion selection
with an exclusion list of peptides already sequenced in a user-defined
time window (dynamic exclusion) permitted analysis by CID of 12 of
the peptides in a 30 s time window.  Two peptides from the RP column
separated by 4 s show CID spectra which identifies them as the transmembrane
tyrosine phosphatase CD45 and the calcium pump ATC2.  Their
d0:d8 ratios are 1:0.72 and 1:1.2, resp., indicating that differentiation
reduces the amount of CD45 while increasing the calcium pump.
Analysis of all 30 cation-exchange fractions produced 25,891 CID spectra.
Each fraction consumes 100 min of automated sequencing time, and thus
about 50 h to complete.  The NCI human sequence database of 73,656 proteins
was searched with SEQUEST and identified more than 5000 Cys-containing
peptides.  These corresponded to 491 absolutely identified proteins.
Because of so much data generated, software had to be created to manipulate
this:  XPRESS and INTERACT, which quantify and organize the peptides
identified by SEQUEST.
<p>
The d0:d8 ratios were reproducible.  CD45 has 23 theoretical Cys-containing
peptides and 13 were found, some detected multiple times (between once
and 10 times was the range of the 13, for a total of 65 analyses).
Variability in quantification was &lt;20%.  Consistent variability
is found when two additional small-scale experiments were done in a single
LC-MS/MS analysis for each experiment.  Such precision is the highest
obtainable when different samples from different experiments are compared.
<p>
This technique does not or cannot quantify proteins without Cys residues,
very low abundance proteins, and very hydrophobic proteins.  The database
searching of the CID spectra found ~5000 Cys-containing peptides matching
to 1025 proteins.  491 proteins were conclusively identified and
quantified from the d0:d8 ratios.  Broad classes of proteins were identified,
including moderate and low abundance, regulatory proteins (protein
kinases and phosphatases, GTPases), membrane receptors, channels.
Changes in abundance of most microsomal fraction proteins are not significant
between control and treated cells.  For example, proteins of ribosome,
cytoskeleton, metabolism and a majority of cell surface receptors and
channel proteins are not altered.  Many signal transduction-associated
proteins and enzymes showed significant changes, particularly those
in the downstream pathway of PMA stimulation.  Farnesyl-diphosphate
farnesyl transferase (squalene sythetase, SQS) shows PMA-induced downregulation
by 20-fold;  in addition membrane-associated protein connected by a lipid
anchor involving protein prenylation requiring farnesylpyrophosphate (FPP),
a metabolite whose presence is regulated by SQS, are altered with
respect to their subcellular location.  SQS is a branchpoint of the
mevalonate pathway playing a role in regulating the level of FPP pool
either for cholesterol/sterol biosynthesis or nonsteroidal protein prenylation.
SQS is activated coincident with a proteolysis-induced release from the
membrane, consuming FPP for producing squalene, precursor to cholesterols
and sterols.  Such activation depletes FPP available for protein prenylation,
and thus the PMA-induced downregulation of such prenylated proteins as
RhoG and Rac1 in the microsome fraction is explained coincidental with
loss of microsomal SQS.  Downregulating SQS in response to cytokines
and endotoxin, and the coordinated increase in HMG-CoA reductase activity
is thought to increase isoprenoid intermediates for an acute inflammatory
response.
<p>
PMA also downregulates Src family kinases, PKC isoforms, small GTPases,
and phosphatases in microsomal fraction of treated HL-60 cells.
Increased expression was exceptional:  PKC-&theta;, Yes, and the PTN-1
and -3 phosphatases.  PKC-&alpha;, -&beta;1, Lck, Src, and phosphatases
PP1A and PTN-2 were found in decreased amount after PMA treatment.
Together these results indicate that there are unidentified mechanisms
for the regulation of specific proteins that likely regulate cellular
attachment and differentiation in HL-60 cells.  This proteome profiling
data now reveals study of the PKC isoforms to determine why these differences
occur with PMA treatment.
<!-- ================================================================ -->
<p class="tags">

<p class="ref">
 <span class="authors">
Badock V, Steinhusen U, Bommer K, Otto A
 </span>
 <span class="year">
(2001)
 </span>
 <span class="jname">
Electrophoresis
 </span>
 <span class="jvol">
22,
 </span>
 <span class="pages">
2856-2864.
 </span>
<p class="reftitl">
Prefractionation of Protein Samples for Proteome Analysis Using
Reversed-Phase High Performance Liquid Chromatography
<p class="abstract">
<b>ABSTRACT</b>.
We describe an approach for fractionating complex protein samples prior to twodimensional
gel electrophoresis using reversed-phase high-performance liquid chromatography.
Whole lysates of cells and tissue were prefractionated by reversed-phase
chromatography and elution with a five-step gradient of increasing acetonitrile concentrations.
The proteins obtained at each step were subsequently separated by
high-resolution two-dimensional gel electrophoresis (2-DE). The reproducibility of this
prefractionation technique proved to be optimal for comparing 2-DE gels from two different
cell states. In addition, this method is suitable for enriching low-abundance proteins
barely detectable by silver staining to amounts that can be detected by Coomassie
blue and further analyzed by mass spectrometry.
<!-- ============================================================= -->
<p class="tags">
silver staining
<p class="ref">
 <span class="authors">
M&ouml;rtz E, Krogh TN, Vorum H, G&ouml;g A
 </span>
 <span class="year">
(2???)
 </span>
 <span class="pages">
Poster (unknown presentation)
 </span>
<p class="reftitl">
Improved Silver Staining Protocols Compatible with Large-Scale Protein 
Identification
<p class="Para1"> Proteins were stained and the digested in automated manner 
using Protana A/S workstation (can process eight 96-well plates in parallel).
This workstation has high throughput, sensivitity and reproducibility.
Samples are concentrated on ZipTips.  10% of sample applied to tip and eluted
to 400 &micro;m Bruker Anchorchip target with 2,5-DHB in MeOH:0.1% TFA (1:1).
A modified fuzzy logic algorithm was run on Bruker REFLEX III MALDI with 384
sample Scout source.  Software for queries and db managment is in-house, part
of Protana Proteomics Software Suite (PPSS 2.2).
<p>
Three stadnard proteins were diluted in a series from 1 pmol to 5 fmol.
They had been run on 1D gels, stained using Coomassie, and by silver 
using Schevchenko et al, Bio-Rad's kit, Blum and Vorum.  Destaining of silver
was also evaluated.  Peptide digests were either used directly (2% of sample)
or purified on ZipTips (10% of sample).  Staining protcols were modified based
on intermediate results to optimize sensitivity and recovery of peptides.
<p>The following silver stain protocols were done:
<ul>
<li>EMBL Silver stain
 <ol>
 <li>20 min in 50% MeOH, 5% acetic acid
 <li>10 min in 50% MeOH
 <li>2 h in water (overnight washing reduces background even more)
 <li>1 min in 0.02% (20 mg/100 ml) sodium thiosulfate (Na<sub>2</sub>S<sub>2</sub>O<sub>3</sub>)
 <li>1 min two times in water
 <li>20 min in cold 0.1% silver nitrate
 <li>1 min in water
 <li>Transfer to new container
 <li>1 min in water
 <li>0.04% (40 ul per 100 ml) formaldehyde in 2% sodium carbonate:
   monitor the development:  if developer turns yellow, remove it and
   put in fresh developer.
 <li>Stop:  5% acetic acid:  change it a couple of times
 <li>Keep gel in 1% acetic acid at 4&deg;
 </ol>
<li>Blum silver stain
 <ol>
 <li>at least 1 h in 40% EtOH, 10% acetic acid fix
 <li>20 min twice in 30% EtOH
 <li>20 min in water
 <li>1 min in 0.02% sodium thiosulfate
 <li>20 s three times in water
 <li>20 min in cold 0.1% silver nitrate
 <li>20 s three times in water
 <li>Move to new gel chamber
 <li>1 min in water
 <li>Develop in 0.05% formaldehyde and 3% sodium carbonate;
   change developer if it turns yellow
 <li>wash gel in water for 20 s
 <li>Add 5% acetic acid to stop
 <li>Wash in water three times for 10 min
 <li>Keep gel in 1% at 4&deg;
 </ol>
<li>Vorum silver stain
 <ol>
 <li>Fix gel in 50% MeOH, 12% acetic acid, 0.05% formalin for 2 h or
  overnight
 <li>Wash gel three times in 35% EtOH for 20 min each
 <li>Sensitizer:  0.02% sodium thiosulfate for 2 min
 <li>Wash three times in water, 5 min each
 <li>20 min in 0.2% silver nitrate contain 0.076% formaldehyde
 <li>Twice for 1 min each in water
 <li>Develop in 0.05% formaldehyde, 6% sodiumm carbonate, 0.0004%
sodium thiosulfate.
 <li>Stop. 50% MeOH, 12% acetic acid at least 5 min
 <li>Keep gel 4&deg; in 1% acetic acid
 </ol>
</ul>
<p>
The most significant changes were:
<ul>
<li>After acetic acid fixation, the washing was improved, which reduces 
background and increases sensitivity.
<li>Limits to sodium thiosulfate were done:  it increases recovery of peptides
but it also increases background.
<li>Alterations to the stop solution were done to make it MS-compatible
</ul>
<p>
The three proteins were BSA, transferrin and carbonic anhydrase.  With the
<b>EMBL silver stain</b>, transferrin was detectable down to 25 fmol, with BSA
just barely detectable.  Carbonic anhydrase could be detected to 50 fmol.
The modified Blum had best sensitivity and less background than EMBL.
Vorum protocol was less sensitive, but provided black bands and a very clear
background (advantage in image analysis).  The Bio-Rad kit was comparable to
Coomassie staining.
<p>
Recovery of peptides with EMBL and Blum was quite good for faint bands.  
Of interest, the recovery from Coomassie-stained gel or <em>destained</em>
silver-stained gels is not better than from silver-stained gels.
The best spectra were obtained with micropurified (ZipTip) aliquots and
using the Anchorchip, which concentrates the target better.  Sequence 
coverage was better in these cases, which enables identification.
<p>
2D gel maps were made with different stains.  The background was much better
with Vorum than other protocols.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
M&ouml;rtz E, Krogh TN, Vorum H, G&ouml;g A
 </span>
 <span class="year">
(2001)
 </span>
 <span class="jname">
Proteomics
 </span>
 <span class="jvol">
1,
 </span>
 <span class="pages">
1359-1363.
 </span>
<p class="reftitl">
Improved Silver Staining Protocols for High Sensitivity Protein
Identification Using Matrix-Assisted Laser Desorption/Ionization-Time of
Flight Analysis
<p class="abstract">
<b>ABSTRACT</b>
In proteomics it is essential to be able to detect proteins separated by gel electrophoresis
at high sensitivity. Silver staining is currently the most popular method. Here we
present silver staining protocols that are optimized for staining sensitivity, peptide
recovery and compatibility with digestion and mass spectrometry.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
<a name="smilansky"></a>Smilansky Z
 </span>
 <span class="year">
(2001)
 </span>
 <span class="jname">
Biochemistry
 </span>
 <span class="jvol">
22,
 </span>
 <span class="pages">
1616-1626.
 </span>
<p class="reftitl">
Automatic Registration for Images of Two-Dimensional Protein Gels
<p class="Para1">
Between-image registration (alignment of similar or identical patterns)
is a complex matter.  In 2D gel analysis programs it is one step in the
process.  In customary applications, spot detection, quantitation, editing
and landmarking are done, and the registration is done as part of the matching
process, followed by spot comparison.  In the Z3 algorithm discussed here
image registration precedes spot detection or it goes directly to computing
differential expression in raw images rather than creating a synthetic spot
image distinct from the raw image.  The basis of Z3 is region-based comparison
algorithm within the raw image rather than use of spot-cluster matching algorithms.
<p>
Image registration can be done using a number of gel features.  Spots themselves
have shapes, streaks, smears, spot tails, and other special background elements
can be used for alignment.  Theory of registration recognizes two broad
approaches:  feature-based and direct methods.  Feature-based approaches
start with a preliminary stage of feature extraction and then register using
the feature list.  Spot centers are the usual high-priority features.  Direct
approaches use intensity data from the raw image rather than working through
feature detection.  Transformations between gels are generally heuristic
and not amenable to analytic modeling.  Where a gel has suffered a crack
the transformation must be robust.  Small regions are expected to correspond
and must be locally smooth.  A region's small size allows compuation of
a transformation as a simple shift vector, and geometric uniqueness enables
high-confidence determination.  The entire set of shift vectors is combined
wtih global transformation using the obvious choice of Delaunay triangulation
transformation.
<p>
The new method works by comparing regions of gels as opposed to doing registration
spot-by-spot.  Confidence of matching improves by comparing spot lists region
by region rather than spot by spot.  Current system implement a graph algorithm
where vertex clusters are matched, but the new approach uses a rectangular
region of the raw image contain several spots making a unique geometric
signature (like a constellation).  Direct registration techniques are
applied to the subimage defined by this rectangular region and the correct
transformation at the center of the region determined.  This approach
works better for detecting isoelectric shifts.
<p>
In the system to compare gel images, the reference is green and the
comparative gel is magenta.  Overlapping of the green and magenta produce
a gray, with weak spots light gray and strong spots being dark gray to
black.  Similar gels will look entirely black-and-white, although
differential expression leads to coloring (greenish when proteins are
overexpressed, and magenta for underexpressed).  This system is done
because of theories on visual perception related to attentive and pre-attentive
tasks.  Pre-attentive are instantaneous tasks and time of performance is
unrelated to complexity.  Finding a red letter among a large field-of-view
of green letters or a diagonal among vertical line segments is a task of
this type.  Detection of a difference between two adjacent similar
configuration of letters or spots is an attentive task.
<p>
The basic step is an optimal shift computation in a small rectangular subimage.
Output is a shift vector (<i>x,y</i>)&rarr;(<i>u,v</i>), this being a quadruple
(<i>x,y,u,v</i>).  The list of all shift vectors creates a global transformation;
the more vectors, the more complex the transformation.  The algorithm starts
with the identity transformation, progresses to shift/translation, semi-affine,
affine, and then to Delaunay type transformations.
<p>
If the size of the basic rectangle is too large, it might not correctly register
the entire region.  Geometric warping can result from uneven polymerization,
uneven coolant flow, voltage leaks, even cuts or tears in the gel.  Rectangles
too small may not have a sufficiently unique geometric signature; they may have
only one spot.  Establishing a sequence of covering rectangles (SCR) thus
should be such that there are at least 2 and ideally 3 or 4 spots contained.
Too many spots runs the risk of having multiple solutions and a nonunique
pattern.  For the rectangle sequence, apply first a simple spot detection
fast and robust without a need to be precise.  A local threshold on the 2nd
derivative would be an example of a mechanism.  A segmentation algorithm then
produces a list of spot segments with 3 params:  area, bounding rectangle,
and contrast value.  The area <i>S</i> would contain 4 spots and then
constructed.  A score for the triangle is computed based on spot count, contrast,
and area.  If overlap occurs between triangles,  the higher scoring one is
kept and the other removed.  Rectangles for the entire image are sequenced,
with number 1 being the rectangle closest to the center of the image, and will
be seed for the registration loop.  Sorting occurs by determining the distance
of all rectangles from the seed rectangle.
<p>
<b>Shift Vector List</b>.  This represents the first step in the loop of the
algorithm.  The shift vector of the first rectangle is computed with the global
transformation being identity.  A discrepancy is a fixed parameter (<q>maximum
positional uncertainty</q>) which indicates the maximal expected global shift
between two images.  If it fails in the 1st rectangle, then the next rectangle
is regarded as the first.  If it fails for the initial 10 rectangles, the
algorithm stops and reports an error.  If the solution in the seed rectangle
is restricted to a simple shift/translation, then there is a tradeoff between
the magnitude of global scaling and rotation and the size of the seed rectangle.
Experience shows that the running and scanning and gel image result in relative
scaling not larger than 10% and rotations not larger than 5&deg;.
<p>
Using the global transformatin <i>T</i> determined by the previous step in the
loop, the expected shift vector (<i>x,y,u<sub>0</sub>,v<sub>0</sub></i>)
is calcuated by (<i>u<sub>0</sub>,v<sub>0</sub></i>) = <i>T</i>(x,y).
The local shift is d<i>x</i> = <i>u</i><sub>0</sub> &minus; <i>x</i>,
d<i>y</i> = <i>v</i><sub>0</sub> &minus; <i>y</i>.  The expected
<i>I</i><sub>dst</sub> corresponding to <i>R</i><sub>i</sub> in the source
image is <i>R</i><sub>i</sub><sup>dst</sup> = <i>R</i><sub>i</sub> +
(d<i>x</i>, d<i>y</i>).
<p>
The allowed maximal horizontal and vertical discrepancy (<i>D</i><sub>u</sub>)
between the exact solution and the expected shift vector is estimated and for
the first few points it is fixed, but with accumulation of several points, it
is computed from previous values using principles of statistical error.
<p>
A set of rectangles is defined {<i>R</i><sub>kl</sub> =
<i>R</i><sub>i</sub><sup>dst</sup> + <i>s</i>(<i>k, l</i>): <i>k,l</i> =
0, &plusmn;1, &plusmn;2,..., |<i>sk</i>|1&le; <i>D</i><sub>u</sub> + 2<i>s</i>,
|<i>sl</i>| &le; <i>D</i><sub>v</sub> + 2<i>s</i>}.  These defined in the
destination image and together form the complete set of candidate matches to
the original rectangle in source image <i>R</i><sub>i</sub>.  The series
of rectangles covers a region with sides 2(<i>D</i><sub>u</sub> + 2<i>s</i>),
2(<i>D</i><sub>v</sub> + 2<i>s</i>) in jump of size <i>s</i> pixels.  The step
size <i>s</i> is set to radius of smallest expected spot on the gel and its purpose
is to decrease computational load by <i>s</i><sup>2</sup> where typically
<i>s</i>=3.  The enlargement two by regions on each side helps verify that the
solution that was obtained is a real one and not an artifact.  If the best solution
turns out to have a discrepancy larger than (<i>D</i><sub>u</sub>,
<i>D</i><sub>v</sub>), it is discarded.
<p>
For each rectangle <i>R</i><sub>kl</sub>, a comparison to <i>R</i><sub>i</sub>
is done and similarity score is computed.  This score function <i>F</i> is
the sum of a pixel score function <i>G</i><sub>P</sub> over all pixel pairs
<i>p</i> &isin; <i>R</i><sub>i</sub> and the correponding pixel <i>q = p +
s</i>(<i>k, l</i>) &isin; <i>R</i><sub>kl</sub>.  Function <i>G</i><sub>p</sub>
is specific to gel image registration and has 3 factors:  (i) a fixed
coontribution to the score when both pixels have some minimal contrast, allowing
spots representing differentially expressed proteins to match, (ii) in cases
where both spots have high constrast a <q>bonus</q> is given to give a match
between strong spots, (iii) A contribution also occurs from a scalar product
of two gradient vectors, increasing the score for spots having same size and
shape, and decreases socre for spots whose boundaries overlap incorrectly.
The edge pixels provide the contribution:  where edges overlap and are similarly
oriented, score is increased and where there is negative correlation, score
is diminshed.  Transformations are preferred where spot matches have similar
contrast, size and shape compared to spot matches vastly different.  Proteomes
comparing different types of tissues or even organisms are inconceivable.
In a typical case, at least 70-80% of the spots to match in position, if not
size.
<p>
With a score assigned to all <i>R</i><sub>kl</sub> the best rectangle is chosen,
considering both the score and discrepancy (although the initial rectangle
has only a score).  Discrepancy is implemented as a <q>central preference</q>
factor in a parabolic shape and used in factoring scores.  In case of two
high scores, one central and one peripheral, the central one wins.
<p>
When the chosen solution fails the discrepancy limit, then the test fails.
The test at this point is supposed to eliminate problematic areas (random
matches) and one happens to be maximal.  Scores are sorted by decreasing the
distance from the position having the highest score.  Sorted scores must
form a distribution which has a minimal slope and with a winning score
corresponding to a real peak.
<p>
A solution is now fine-tuned using optical
flow techniques which assume that two images of distinct 2D gels are the same
physical data observed twice; standard optical flow equations, the choice solution
is fine-tuned to sub-pixal accuracy.  Let &nabla;<i>g</i><sub>i</sub> denote
a gradient vector at pixel <i>i</i> and <i>dg</i><sub>i</sub> be the gray level
difference between source pixel <i>P</i><sub>i</sub><sup>src</sup> and the
corresponding destination pixel <i>P</i><sub>i</sub><sup>dst</sup>.  <i>w</i>
denotes the required shift.  The <q>brightness constancy</q> equation explains
the gray level difference <i>dg</i><sub>i</sub> resulting from an image shift.
This is estimated using gradient &nabla;<i>g</i><sub>i</sub>, thereby enabling
the determination of <i>w</i> through minimizing
&sum;(&nabla;<i>g</i><sub>i</sub><i>w</i> &minus; <i>dg</i><sub>i</sub>)<sup>2</sup>,
the summation being over all pixels in the region of interest.  Least squares
fitting is used in solution, iterating 2-3 times to gain robustness and precision.
If convergence occurs, the analysis is a success; otherwise an error is generated
and "failure" reported.  If the shift vector was computed successfully and passed
all test, the vector is added to SVL and transformation (discussed below) is
applied/updated.
<p>
With all shift vectors computed, it is possible to analyze for an irregular
vector as a new way of detecting errors.  In some cases however large deviations
(large 2nd derivatives) should be allowed (cuts/tears).  A singleton is not allowed:
this is a shift vector not supported by even one other vector in its neighborhood.
Even with a tear or crack several vectors should be similar.  This particular test
is done at the end of the main loop.  If there is no change in the shift vector
list (SVL), then the main loop is exited; otherwise there is a chance to add more
anchors.
<!-- ============================================================= -->
<p class="tags">
silver-associated peptides, CID, FD
<p class="ref">
 <span class="authors">
Chu IK,  Shoeib T, Guo X, Rodriquez CF, Lau T-C, Hopkinson AC, Siu KWM
 </span>
 <span class="year">
(2001)
 </span>
 <span class="jname">
J. Am. Soc. Mass Spectrom.
 </span>
 <span class="jvol">
12,
 </span>
 <span class="pages">
163-175.
 </span>
<p class="reftitl">
Characterization of the Product Ions from the Collision-Induced Dissociation of
Argentinated Peptides
<p class="Para1">
Low (&lt;100 eV) energy CID spectra of protonated peptides produce a series of ions
that are typically <i>a</i>, <i>b</i>, and <i>y</i> type fragments.  In solution
the more basic sites are N atoms on the N-terminal amino, and side chains of
Arg, Lys, and His.  In gas phase however, the energy barriers are altered, particularly
with collision so that the proton bonds to amide N atoms, which is the first step
in promoting fragmentation.  Charge retained on the N-terminal peptide becomes a
<i>b</i> ion, and on the C-terminal side a <i>y </i>ion.  A b ion can typically
undergo neutral loss of CO and become an <i>a</i> ion.  The type b ion was thought to
be an acylium ion, but several groups (Harrison's, Wesdemiotis', and Hunt's) have
shown evidence it is an oxazolone ring, which fragments under CID to produce
immonium via acylium activated complex.  Silver ion has been used for heavy atom
derivatization in crystallography and metallothioneins have a high affinity for Ag ion.
In solution Ag(I) has a high affinity for Met thioether, but in gas phase, it
formed a complex with the amidic carbonyl groups probably as a result of an attempt
to solvate with ion.  Fragmentation showed four ion types: [b<sub>n</sub>-H+Ag]<sup>+</sup>,
[a<sub>n</sub>-H+Ag]<sup>+</sup>, [y<sub>n</sub>-H+Ag]<sup>+</sup> and
[b<sub>n</sub>+OH+Ag]<sup>+</sup>
<p>
The model peptide AAA shows a product ion spectrum of the ion
[b<sub>2</sub>+OH+<sup>107</sup>Ag]<sup>+</sup>, filtered in the triple quad (the
<sup>109</sup>Ag form of peptide ion was filtered out).  Another spectrum is of the
product ions of [M+<sup>107</sup>Ag]<sup>+</sup> of dipeptide AA.  These two spectra
are almost identical.  The most intense peak at m/z 196 would correspond to two
isobaric fragments, [b<sub>1</sub>+OH+Ag]<sup>+</sup> and [y<sub>1</sub>-H+Ag]<sup>+</sup>.
This spectral similiarity for [b<sub>n</sub>+OH+Ag]<sup>+</sup> product ion and
[M+Ag]<sup>+</sup> with identical residues
<!-- ============================================================= -->
<p class="tags">
GGG peptides, mobile proton theory, FD
<p class="ref">
 <span class="authors">
Rodriquez CF, Cunje A, Shoeib T, Chu IK, Hopkinson AC, Siu KWM
 </span>
 <span class="year">
(2001)
 </span>
 <span class="jname">
J. Am. Chem. Soc.
 </span>
 <span class="jvol">
123,
 </span>
 <span class="pages">
3006-3012.
 </span>
<p class="reftitl">
Proton Migration and Tautomerism in Protonated Triglycine
<p class="Para1">
Different computed structures of GGG are presented using computational
chemistry.  Relative (1) electronic energies, (2) enthalpies, and (3)
free energies are given at 298 K in tabular format.  Initial structures
computed using HF/6-31G(d) by Zhang et al were the basis for geometric
optimization of neutral and protonated GGG calculated at B3LYP/6-31++G-(d,p).
These results were then modified to produce isomers of GGG-H<sup>+</sup>
found on an energy hypersurface, which would generate a b<sub>2</sub> ion
Of the many isomers that could be structured, only the lowest energy structures
are shown.  Three neutral GGG are presented, two folded, and one linear.
One of the folded structures (N2) is an optimization of the lowest energy
HF/6-31G(d) minimum of Zhang et al; the only significant difference between
the Zhang et al structure and N2 is the distance between N-terminal
amino H atom and the C-terminal carbonyl O atoms.  For Zhang et al, the distance
is 2.654 &Aring; but in N2 it is 3.683 &Aring;.  The use of
B3LYP/6-31++G(d,p) higher level of theory is a more accurate description of
long-range interaction. where electron correlation and diffuse functions are
incorproated.  N2 is also closed to the Strittmatter and Williams neutral
GGG structure via molecular mechanics, with the major difference between the
orientation of COOH group.  Both folded structures (N1 and N2) are lower in
energy than the folded (N3) by 2.9 kcal/mol.  The results are not different
for enthalpy changes at 298 K.  Introduciing entropy terms, there is only a
0.2 kcal/mod difference in free energies among isomers (meaning differences
are negligible).
<p>
Proton transfer from N-terminal amine N atom to C-terminal peptide bond
amide N atom, followed by cleavage to form b<sub>2</sub> ion is a long
sequence involving 8 structures and 6 described transition states, all
of which are computed.  The free energy profile is shown computed
in B3LYP/6-31++G(d,p), with a final two product yield of 2-aminomethyl-5-oxazolone
(ring form GG) and glycine.  Protonated GGG has sufficient flexibility to form
long-range H-bonds.  Four H-bond types are considered:  N-H&#9481;N,
O-H&#9481;N, N-H&#9481;O, O-H&#9481;O.
H-bonding distances &lt; 2.0 &Aring;.  Two low-energy N-terminal
nitrogen-protonated structures designated P1 and 1 were computed:  structure 1
is lower than P1 by 0.4 kcal/mol in energy and similar to lowest energy structure
found by Zhang et al. while P1 is similar to the optimized structure of Strittematter
and Williams.  Hydrogen bond of type N<sub>1</sub>-H&#9481;O<sub>2</sub>
(subscripts are residue number) of length 1.840 &Aring; in structure P1, while
it is shorter in structure 1 at 1.777 &Aring; for
N<sub>1</sub>-H&#9481;O<sub>3</sub>.  With hydrogen bonding, proton
transfer to carbonyl O atom on 2nd peptide bond proceeds.  The proton must
then undergo a 1,3-shift to place itself on amide N<sub>3</sub> atom for
fragmentation to occur to produce b2 ion, but this takes 40 kcal/mol.
From 1 to structure 2, the amino nitrogen proton moved to O<sub>1</sub> atom
is a 1,4-transfer requiring 4.3 kcal/mol; this structure is a global minimum
and lower than structure 1 by 1.3 kcal/mol.  2 is more linear than 1 and
thus has a greater entropy contribution;  the
O<sub>1</sub>-H&#9481;N<sub>1</sub> H bond is 1.748 &Aring;, the
shorter bond giving more stability.  The 5-membered ring in 2 has a more
compact (shorter bond lengths) than the ring in 1.  The final structure
is the 2-aminomethyl-5-oxazolone which requires rotation of the N-termainal
amino so it is trans to carbonyl O atom (&ang;N<sub>1</sub>CCO<sub>1</sub> &#8776;
180&deg;)
[The text of this article goes into detail describing the transition and energy
in the reaction which involves b2 ion formation.  There are alternative structures
considered, such as the transition from 4&rarr;6 going through 9 instead of 5.]
<p>
In collisionally activated decomposition, enough energy will cause progress
from structures 1&rarr;4.  From 4 (where protonation is on carbonyl O atom), there
is a 32.5 kcal/mole barrier representing TS(6&rarr;7).  Threshold energy experiments
found an estimate of 54.5 kcal/mol, which is the upper limit as the kinetic shift,
large for an ion with as many bonds as protonated GGG, was ignored; incorporating
kinetic shift reduces the estimate drastically.
<p>
Many studies have looked at the probable protonation site of a petpide.  Structure
1 suggests the N-terminal amino N atom, 1 kcal/mol higher than structure 2 which
is amide O<sub>1</sub> atom (with solvation by amino N atom).  Structure 4 shows
protonation on same O atom but H-bonding  to carbonyl O atom of 2nd residue.
Structure 2 has relative stability because it has larger entropy.
Gas phase basicity of base B is the free energy change
&Delta;<i>G</i><sup>0</sup><sub>r,298</sub> while proton affinity is the enthalpy
change, &Delta;<i>H</i><sup>0</sup><sub>r,298</sub> of the reaction
<p class="eqnserif">
BH<sup>+</sup> &rarr; B + H<sup>+</sup>
<p>
and they are related by the simple equation
<p class="eqnserif">
&Delta;<i>G</i><sup>0</sup><sub>r,298</sub> = &Delta;<i>H</i><sup>0</sup><sub>r,298</sub>
&minus; <i>T</i>&Delta;<i>S</i><sup>0</sup><sub>r,298</sub>
<p>
&Delta;<i>H</i><sup>0</sup><sub>r,298</sub> is arrived by molecular orbital
calculations:
<p class="eqnserif">
&Delta;<i>H</i><sup>0</sup><sub>r,298</sub> = &Delta;<i>E</i><sub>elec</sub> +
&Delta;<i>E</i><sub>ZVPE</sub>(0) + &Delta;<i>E</i><sub>int</sub>(298) +
5<i>RT</i>/2
<p>
First three terms oare electronic energy, zero-point vibrational energy, and
thermal energy needed for reaction 1 at 298.15 K.  The 5<i>RT</i>/2 constant
is estimation of three translational degrees of freedome (3<i>RT</i>/2) for the
proton plus <i>RT</i> which is the <i>PV</i> work term for the proton.  Substituting
the free-energy and enthalpy equations above along with
<p class="eqnserif">
<i>T</i>&Delta;<i>S</i><sup>0</sup><sub>r,298</sub> = [298.15][<i>S</i>(BH<sup>+</sup>
&minus; <i>S</i>(B)] &minus; 7.8 kcal/mol
<p>
where the 7.8 kcal/mol is the entry of the proton at 298.15 K.  With structures
N1 and 2, the basicity of GGG is 216.3 kcal/mol, while for N1 and structure 1,
basicity is 215.0 kcal/mol.  Zhang et al determined basicity for the amino N atom
and found 219.6 kcal/mol.  Several different theoretical methods are used, and
a couple agree with our own calcuations.
<!-- ============================================================= -->
<p class="tags">
alpha<sub>2</sub>-macroglobulin, 2DGE
<p class="ref">
 <span class="authors">
Kondo T, Sakaguchi M, Namba M
 </span>
 <span class="year">
(2001)
 </span>
 <span class="jname">
Exp. Gerontol.
 </span>
 <span class="jvol">
36,
 </span>
 <span class="pages">
487-495..
 </span>
<p class="reftitl">
Two-Dimensional Gel Electrophoresis Studies on The Cellular Aging:
Accumulation of Alpha-2-Macroglobulin in Human Fibroblasts with Aging
<p class="Para1">
KMS-6 and OUMS24 are normal human fibroblast cell lines from 9- and 6-week
old female human embryos, respectively.  KMST-6 was produced from KMS-6
by <sup>60</sup>Co irradiation and OUMS24F from OUMS24 by 4-nitroquinoline
1-oxide treatment.  Cells are maintained in EMEM with 10% hi-FBS and
subcultured at confluence.  15th passage cells are <q>young</q> while
53rd passage KMS-6 and 74th passage OUMS24 cells are senescent.  Aged
cells were <q>confirmed</q> by incubating fixed cells (2% HCHO/2% glutaraldehyde
5 min RT) at 37&deg; with 1 mg/ml BCI-galactose in pH 6.0 (40 mM citrate
acid/sodium phosphate) buffer containing 5 mM K<sub>3</sub>Fe(CN)<sub>6</sub>,
5 mM K<sub>4</sub>Fe(CN)<sub>6</sub>, 150 mM NaCl, 2 mM MgCl<sub>2</sub>.
<p>
Proteins were obtained from thrice PBS-washed cells, treated with 10% TCA
ice-cold 30 min on ice and pellets washed with PBS.  Pelleted cells were
solubilized in 5 M urea, 2% Triton X-100, 70 mM DTT, 2% ampholytes and
1 mM PMSF for 30 min on ice at 4&deg;.  2DGE was on pH 4-7 Immobiline
and run on a Multiphor.  Second dimension was a 12% gel, with silver staining
as done by Hochstrasser et al.  Duplicate gels were run.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Stern DF
 </span>
 <span class="year">
(2001)
 </span>
 <span class="jname">
Exp. Molec. Pathol.
 </span>
 <span class="jvol">
70,
 </span>
 <span class="pages">
327-331.
 </span>
<p class="reftitl">
Phosphoproteomics
<p class="Para1">
<b>REVIEW</b>.  In pathology there is an effort to distinguish disease into
subtypes and to make prognostic predictions.  Tumor analysis has two approaches:
(1) develop a taxonomy for tumors involving tissue of origin and look at
common biology, and (2) try to find biomarkers (individuals or groups) with
strong prognostic or predictive significance.  Transcriptomics and RNA profiling
may hold promise, but these don't predict post-translational modifications that
may originate a disease course.
<p>
Prototypic phosphoproteomic analysis.  <i>HER2/NEU/ERB2</i> is a gene for
the receptor tyrosine kinase in the EGF receptor family. In 1 in 5 to 1 in 4
breast and ovarian (and other) cancers, this gene is amplified.  Its presence
is associated with a poor prognosis and chemotherapeutic resistance.
Herceptin (Tratuzamab) is a humanized monoclonal approved for use in patients
that overexpress Her2.  Oncogene models in animals with Her2 overexpression
show aggressive cancer.  Her2 is weakly activated when overexpressed, but
strongly activated by EGF family growth factors and maybe by splice alterations
and proteolytic processing.
<p>
RTKs are phosphorylated on specific Tyr residues when activated.  The
phosphopeptides produced are docking sites for downstream signaling processes.
The antibody PN2A is a mAb directed against phospho-Her2.  It detects a
heterogeneous signal in breast carcinomas, indicating a signal heterogeneity
of Her2.  With overexpression of Her2, there was only a weakly predicted
poor prognosis.  With a small subset of patients with high Her2 phosphorylation
there was a shorter disease-free and disease-significant survival.  Thus
antiphosphate specific antibodies have strong predictive value.
<p>
A major function of Her2 is to stimulate Ras in cancer.  Looking at downstream
components such as MAP kinase could prove useful.  Because it is an endpoint
in the signaling system, it might be a superior marker to examine effects.
But is MAP kinase a good indicator of Her2 activity?  Hardly, since Her2
has other many downstream effects.
<p>
The hallmark of cancer is that there is a disruption of the growth factor
(mitogen) requirement for cell proliferation.  This can be achieved by
mutations in Ras or by constitutive growth factor receptor activation by
autocrine factors, receptor overxpression, and receptor mutation.
<!-- ============================================================= -->
<p class="tags">
SPR, DNA-protein complexes
<p class="ref">
 <span class="authors">
Buckle M
 </span>
 <span class="year">
(2001)
 </span>
 <span class="jname">
Methods in Molecular Biology
 </span>
 <span class="jvol">
148,
 </span>
 <span class="pages">
535-546.
 </span>
<p class="reftitl">
Surface Plasmon Resonance Applied to DNA-Protein Complexes
<p class="Para1">
Surface plasmon resonance (SPR) measures the refractive index changes
(&delta;n) at or near a surface and relates those changes to changes
in the mass at the surface.  The relationship is described through
the Mossotti form [Eq 2] of the Debye equation [Eq 1]:
<table class="eqnserif">
<tr><td style="border-bottom:1px solid black;">&epsilon; &minus; 1
 <td rowspan="2">=
 <td style="border-bottom:1px solid black;">N
 <td rowspan="2" style="font-size:150%;padding:0;">(
 <td style="border-bottom:1px solid black;">&alpha; + &mu;
 <td rowspan="2" style="font-size:150%;padding:0;">)&nbsp;&nbsp;
  <td rowspan="2">[1]
 <tr><td>&epsilon; + 2
  <td>3&epsilon;<sub>0</sub>
  <td><i>kT</i>
</table>

<table class="eqnserif">
<tr><td style="border-bottom:1px solid black;">&epsilon; &minus; 1
 <td rowspan="2">=
 <td style="border-bottom:1px solid black;">N&alpha;
  <td rowspan="2">[2]
 <tr><td>&epsilon; + 2
  <td>3&epsilon;<sub>0</sub>
</table>
<p>
where &epsilon; is the real part of the dielectric or permittivity constant
related to the refractive index by &epsilon; = <i>n</i><sup>2</sup>, <i>N</i>
is the number density given by <i>N<sub>A</sub>&rho;/M<sub>a</sub></i>
(<i>N<sub>A</sub></i> is Avogadro's number, &rho; is density, and
<i>M<sub>a</sub></i> is molecular mass).  The value
&Delta;<i>n</i>/&Delta;<i>C</i> is assumed to be constant.
<p>
The BIACORE instrument is an SPR device made with a monochromatic,
plane-polarized light source with photodetector, these connected optically
through a glass prism.  A 50 nm thin gold film is deposited on one side
of the prism, and this is covered with long-chain hydroxyalkanethiol
forming a monolayer 100 nm thick.  The layer is an attachment point for
carboxymethylated dextran chains creating a hydrophilic surface to which
ligands can couple.  Light incident to the back side of the metal film
is internally reflected on the diode array detector.  An evanescent wave
that is nonpropagative penetrates to the solution side of the prism away
from light source.  In the gold layer free electrons resonate with the
evanescent wave.  The amplitude vector of the transversal wave propagating
along the gold surface (<i><b>ksp</b></i>) is equal to the component of the
evanescent wave (<i><b>kx</b></i>).  Since &epsilon; = <i>n</i><sup>2</sup>,
if &omega; is the wave frequency and <i>c</i> speed of light, then:
<p class="eqnserif">
|<i><b>ksp</b></i>| = (&omega;/<i>c</i>)[(&epsilon;<sub>1</sub> &minus;
&epsilon;<sub>2</sub>) / (&epsilon;<sub>1</sub> + &epsilon;<sub>2</sub>)]<sup>1/2</sup>
<p>
The evanescent wave is described by:
<p class="eqnserif">
|<i><b>kx</b></i>| = (&omega;/<i>c</i>) sin&theta; (&epsilon;<sub>g</sub>)<sup>1/2</sup>
<p>
When resonance occurs, |<i><b>ksp</b></i>| = |<i><b>kx</b></i>|, and the
intensity of reflected light decreases at a sharply defined angle of incidence,
the SPR angle, given by the expression:
<p class="eqnserif">
sin&theta;<sub>0</sub> = (&epsilon;<sub>1</sub>&epsilon;<sub>2</sub>) /
[&epsilon;<sub>g</sub>(&epsilon;<sub>1</sub> + &epsilon;<sub>2</sub>)]
<p>The SPR angle &theta;<sub>0</sub> is where there is a decrease in intensity
of reflected light, measuring the refractive index of the solution in contact
with the gold surface and depends on the wavelength of the light source
and the metal used in the film.  When those parameters are kept constant
the angle shift in SPR depends only on refractive index changes in the thin
layer on the metal surface.  Increases of surface material should result in
increases in the SPR angle, detected as a shift of the position of the
light intensity minimum on the diode array.  Changes in surface concentration
over time have been correlated with radiolabeling methods and linearity is shown
over a wide range of concentration.  Instrument output is given in resonance
units (RU) with a 1000 RU corresponding to a 0.1&deg; shift in SPR angle,
which for the average protein is a change in concentration of 1 ng/mm<sup>2</sup>.
The Biacore 2000 can measure a deviation of 0.001&deg;, thus being a variation
in refractive index of 0.00001.
<!-- ============================================================= -->
<p class="tags">
computational chemistry, 2+ NaCl cluster ions, FD
<p class="ref">
 <span class="authors">
Aguado A
 </span>
 <span class="year">
(2001)
 </span>
 <span class="jname">
J. Phys. Chem. B
 </span>
 <span class="jvol">
105,
 </span>
 <span class="pages">
2761-2765.
 </span>
<p class="reftitl">
An ab Initio Study of the Structures and Relative Stabilities of Doubly Charged
[(NaCl)<sub><i>m</i></sub>(Na)<sub>2</sub>]<sup>2+</sup> Cluster Ions
<p class="Para1">
Alkali halide clusters have many useful properties and have been investigated in 
multiple ways.  Much about singly charged clusters 
(AX)<sub><i>n</i></sub>A<sup>+</sup> has been analyzed, as well as neutral 
clusters.  Small clusters of sodium iodide and lithium halide would adopt ground 
state structures which are stacks of hexagonal rings while others adopt rock 
salt-like ground state structures. Theoretical calculations on the +1 charged 
clusters verify experimental records of bulk rocksalt symmetry even for elements 
crystallizng in CsCl-type structure: namely CsCl, CsBr, CsI.  Doubly charged 
clusters present the problem of excess charges adding to instability.  With NaI 
clusters a critical size was deduced for stability in which <i>m</i> = 18 for 
[(AX)<sub><i>m</i></sub>(A<sub>2</sub>)]<sup>2+</sup>.  Pair potential calcuations
on structures of 2+ NaCl clusters were used to explain experimental findings;
cluster sizes had at least one metastable bound state, even though <i>m</i> &lt;
8 caused removal of Na<sup>+</sup> cation leading to exothermic reaction.
<p>
The model used is the ab initio perturbed ion (aiPI) model which is good 
computational framework for ionic systems in both crystals and clusters.
aiPI has theoretical foundation in theory of electronic separability.  The
Hartree-Fock (HF) equations are solved stepwise, breaking the cluster wave
function into local group functions (ionic characteristics).   With each iteration
total energy is minimized with respect to variations of the electron density
localized in given ion, and densities of other ions are not varied (kept frozen).
The self-consistent process finishes, and outputs are total cluster energy and
set of localized wave functions, one for each geometrically nonequivalent ion
of the cluster.  Localized cluster-consistent ionic wave functions are used to 
estimate intraatomic correlation energy correction through Clementi's 
Coulomb-Hartree-Fock method.  Large multizeta basis sets of Clementi and Roetti
are used to describe the ions.  In ths respect optimizations are performed
using basis sets (5s4p) for Na<sup>+</sup> and (7s6p) for Cl<sup>&minus;</sup>.
Use of diffuse basis functions was found unnecessary.  Use of localized functions
makes it possible to scale linearly the computational effort with number of atoms
in cluster.  Full structural relaxations of clusters have been done with as many
as 58 ions with reasonable computational cost.  Isomers (about 10-15) have been
investgated for each cluster size.  Generation of initial cluster geometries 
done using pair potential.  Optimization of geometries was achieved using downhill
simplex algorithm.
<p>
Optimized aiPI structures for ground state and the lowest lying isomers of 
[(NaCl)<sub><i>m</i></sub>Na<sub>2</sub>]<sup>2+</sup> where <i>m</i> = 8-28 
were determined.  Structures for <i>m</i> = 6 and 7 were calculated just to show 
the special stability of <i>m</i> = 8 size.  The low-lying isomers with <i>m</i> 
= 8 have a 3 &times; 2 &times; 2 structure with a a (NaCl)<sub>6</sub> neutral 
cluster (notation indicates number of ions along each of three perpendicular 
Cartesian axes), and they differ in the way six extra ions are added.  Most 
favorable addition of extra ions is along two opposite edges of the neutral. 
The two triatomic NaClNa<sup>+</sup> units minimize mutual repulsion  without 
distorting too much of the neutral cluster.  Zhang and Cooks visualized 
structure as combination of two 3 &times; 3 &times; 1 planar sheets.  Since 
sheets bend in the ab initio calculations, the notation 3 &times; 2 &times; 2 + 
3 + 3 is preferable to describe the cluster; although calculations agree with 
their suggestion.  Ground state structures of <i>m</i> = 9 and 10 are more 
visually difficult, resulting in addition of one and two NaCl atom pairs.  The 
GS isomer of <i>m</i> = 11 constructs a GS isomer as 3 &times; 3 &times; 2 + 3 + 
3; these added triatoms are on same face as neutral structure.  With <i>m</i> = 
12, a compact structure of different kind appears:  the GS structure can be 
formed from singly charged (NaCl)<sub>13</sub>Na<sup>+</sup> cluster and 
removing the inner chloride anion coordinated to six atoms.  Pair potential
calculations by Martin 
<!-- ============================================================= -->
<p class="tags">
radical cations, collision-induced dissociation, copper(II)
<p class="ref">
 <span class="authors">
Chu IK,  Rodriquez CF, Hopkinson AC, Siu KWM, Lau T-C
 </span>
 <span class="year">
(2001)
 </span>
 <span class="jname">
J. Am. Soc. Mass Spectrom.
 </span>
 <span class="jvol">
12,
 </span>
 <span class="pages">
1114-1119.
 </span>
<p class="reftitl">
Formation of Molecular Radical Cations of Enkephalin Derivatives via Collision-Induced
Dissociation of Electrospray-Generated Copper (II) Complex Ions of Amines and Peptides
<p class="Para1">
PE-SCIEX mass spectrometers were used:  API III, and prototypes of API 3000 and Q/STAR;
first two are triple quads, and latter is Q-ToF.  600 &micro;M Cu(II)-amine perchlorate
and 50 &micro;M are in 50% MeOH, infused at 2 &micro;l/min with air nebulization.
Product ions are generated by nitrogen or argon with center-of-mass energies
(<i>E</i><sub>cm</sub>) at 1-3 eV.  Cu(II)-amine perchlorate is synthesized according
to published procedures (perchlorate intead of nitrate).  Amines tested are dien,
Me5dien (N,N,N',N',N"-pentamethyldiethylenetriamine), terpy (2,2':6',2"-terpyridine),
and 4-Cl-terpy (4-Cl 2,2':6',2"-terpyridine).
<p>
Peptide YAGFLR (= M) is mixed with [Cu<sup>II</sup>(dien)](ClO<sub>4</sub>)<sub>2</sub>
in solution and submitted to ESI.  The ion [<sup>63</sup>Cu<sup>II</sup>(dien)M]<sup>2+</sup>
is most prominent at m/z 445.73 Th, with minor ions of [M+H]<sup>+</sup> at 726.40 Th,
M<sup>&bull;+</sup> at 725.38 Th, [Cu<sup>I</sup>(dien)]<sup>+</sup> at 166.04 Th, and
[Cu<sup>II</sup>(dien-H)]<sup>+</sup> at 165.03 Th.  CID of
[<sup>63</sup>Cu<sup>II</sup>(dien)M]<sup>&bull;2+</sup> shows that radical cations
of the peptide M<sup>&bull;+</sup> are formed along with [Cu<sup>I</sup>(dien)]<sup>+</sup>
when [<sup>63</sup>Cu<sup>II</sup>(dien)M]<sup>&bull;2+</sup> fragments.
Ion [M-106]<sup>&bull;+</sup> is a 2nd generation product ion as well.  Note that the
radical cation of M is formed from YAGFLR, not YAGFL.  With the latter peptide, there
are two reaction pathways: (1) dissociation into [Cu<sup>II</sup>(M-H)]<sup>+</sup> and
[dien+H]<sup>+</sup>, and (2) dissociation into [M+H]<sup>+</sup> and
[Cu<sup>II</sup>(dien-H)<sup>+</sup>].  Latter is intra-ionic proton transfer in complex
ions.
<p>
Radical formation of the cation is not dependent on complexing amine of the metal
(only peptide sequence). [<sup>63</sup>Cu(Me<sub>5</sub>dien)M]<sup>&bull;2+</sup>
product ion for M=YGGFLR shows prominent M<sup>&bull;+</sup> and [M-106]<sup>&bull;+</sup>.
Also terpy and 4-Cl-terpy ligands were as effective.  Comparing spectra shows that
Arg and Tyr residues are important.
Product ion spectra filtering on [<sup>63</sup>Cu(Me<sub>5</sub>dien)M]<sup>&bull;2+</sup>
for M = MGGFLR (M=Met in peptide) produces [Cu<sup>II</sup>(M-H)]<sup>+</sup> and
[dien+H]<sup>+</sup>, as well as [M+H]<sup>+</sup> and
[Cu<sup>II</sup>(dien-H)]<sup>+</sup>, but no M<sup>&bull;+</sup> or
[M-106]<sup>&bull;+</sup>.  30 peptides were surveyed at the N-terminal, and either
Tyr or Trp are necessary for radical cation production.  Ionization energies for these
two amino acids are the lowest among the amino acids.  For the Trp-peptide complexed
to Cu(II)(dien) as a radical 2+ ion, CID leads to formation of three different types of
channels: (1) [Cu<sup>II</sup>(M-H)]<sup>+</sup> and [dien+H]<sup>+</sup>, (2)
[M+H]<sup>+</sup> and [Cu<sup>II</sup>(dien-H)]<sup>+</sup>, and (3) some
M<sup>&bull;+</sup> plus a prominent [M-129]<sup>&bull;+</sup> ion.  Quinmethides
form from oxidatin of phenolic compounds in biological reactions.  Coenzyme
PQQ (pyroloquiolinequinone) oxidation of Tyr and Trp can lead to
C<sub>&alpha;</sub>-C<sub>&beta;</sub> bond cleavage producing
<i>p</i>-quinmethide and 3-methylene indolenine respectively.  The loss of 106 in
YGGFLR and 129 Th from WGGFLR is because of the elimination of these side chains.
With the product spectra of YGGFLR and WGGFLR looking virtually identical with
relative abundance differences, it is likely the precursor ions have the same structure.
Previous reports indicate that <em>both</em> Tyr and Trip <em>and</em> a basic
residue like Arg in order to show radical cations, with radical cations showing loss
of Tyr or Trp.  These experiments show radical cations are produced in Tyr-
and Trp-containing peptides without Arg, Lys, or His; moreover side chain
cleavage of Tyr and Trp is not always seen.  The way in which fragmentation
occurs in copper-liganded peptides is only residue- and sequence-dependent.
<p>
Radical ion production for organics occurs through electron impact resulting
in fragmentation.  Dissociation mechanisms track the radical and charge
sites (<q>electronic hole</q>) as mobile.  Using resonant UV two-photon
ionization plus resonant visible photon absorption to follow dissocation of
small peptides with one Tyr or Trp residue, the charge is seen to migrate
over several residues to effect fragmentation distant from chromophore.
Charge is initially localized to chromophore as a ground state cation.
Photoexcitation promotes unpairing of an electron to charge transfer state
and the new hole hopes between sites that relatively stabilize it.
Charge transfer between residues is dependent on conformation; favorable
conformations enable sufficiently strong coupling strength and high probability
of charge transfer.  Two precursor radical cations containing peptide
M = YGGFLK, the Lys containing peptide, are checked for product ions:
[Cu<sup>II</sup>(terpy)M]<sup>&bull;2+</sup> and [M-106]<sup>&bull;+</sup>.
For the [M-106]<sup>&bull;+</sup> ion and that for M=YGGFLR, similar ion
types are observed, particularly y<sub>n</sub><sup>*</sup> and z<sub>n</sub>,
not commonly seen in protonated peptides.  The YGGFLR y<sub>2</sub><sup>*</sup>
ion is believed to be cyclic, while z<sub>5</sub> is thought to be a ketene.
Ions of type a<sub>n</sub> and b<sub>n</sub> are notably missing from
protonated peptide spectra.  N-terminal fragments like (b<sub>4</sub><sup>*</sup>-H),
(a<sub>4</sub><sup>*</sup>-H), and (b<sub>3</sub><sup>*</sup>-H).
<p>
Shorter peptides are also being investigated, particularly as they produce
fewer ions and reduce the complexity of interpretation.  Peptides YSK
and YTK generated [M-106]<sup>&bull;+</sup> and this ion was filtered for
its product ions.  The two spectra are quite similar.  They differ only
in that the &beta;-C of residue 2 has a methyl group vs. a hydrogen.
Ions from m/z 234.18 and 248.17 in the spectra of YSK and YTK, respectively,
differ by 14 Da and are y<sub>2</sub> ions;  corresponding ions at m/z 144.08
and 158.08 also differ by 14 Da, the different between -CH<sub>3</sub> and
-H atoms and are (b<sub>2</sub><sup>*</sup>-H) ions.  The latter ions lose the
Tyr residue side chain, and this seems to be the characteristic of radical
peptide cations.
<!-- ================================================================ -->
<p class="tags">

<p class="ref">
 <span class="authors">
Gygi SP, Rist B, Griffin TJ, Eng J, Aebersold R
 </span>
 <span class="year">
(2002)
 </span>
 <span class="jname">
J. Proteome Res.
 </span>
 <span class="jvol">
1,
 </span>
 <span class="pages">
47-54.
 </span>
<p class="reftitl">
Proteome Analysis of Low-Abundance Proteins Using Multidimensional Chromatography
and Isotope-Coded Affinity Tags
<p class="abstract">
<b>ABSTRACT</b>.
The effectiveness of proteome-wide protein identification and
quantitative expression profiling is dependent on the ability of
the analytical methodologies employed to routinely obtain information
on low-abundance proteins, as these are frequently of great biological
importance. Two-dimensional gel electrophoresis, the traditional method
for proteome analysis, has proven to be biased toward highly expressed
proteins. Recently, two-dimensional chromatography of the complex
peptide mixtures generated by the digestion of unseparated protein
samples has been introduced for the identification of their components,
and isotope-coded affinity tags (ICAT) have been introduced to allow
for accurate quantification of the components of protein mixtures
by mass spectrometry. Here, we demonstrate that the combination
of isotope coded affinity protein tags and multidimensional
chromatography/mass spectrometry of tryptic peptide mixtures is
capable of detecting and quantifying proteins of low abundance
in complex samples.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Petricoin 3rd EF, Ardekani AM, Hitt BA, Levine PJ, Fusaro VA, Steinberg SM,
Mills GB, Simone C, Fishman DA, Kohn EC, Liotta LA
 </span>
 <span class="year">
(2002)
 </span>
 <span class="jname">
Lancet
 </span>
 <span class="jvol">
359,
 </span>
 <span class="pages">
572-577.
 </span>
<p class="reftitl">
Use of Proteomics Patterns in Serum to Identify Ovarian Cancer
<p class="Para1">
In more than 80% of patients ovarian cancer is presented at late clinical
stage with a 35% 5-year survival.  Those presenting in stage I have
a 90% 5-year survival with cure from surgery alone.  CA125 antigen
is most widely used, abnormally high in 80% of advanced stage disease,
but has a 10% positive predictive value as a single marker; this
improves to 20% with ultrasound added to the screening.  Use of
MALDI-TOF and SELDI-TOF might help with low MW serum protein profiling.
<p>
In this study 100 controls (50 preliminary and 50 masked) were available
from an ovarian cancer program.  17 controls showing other problems were
also avaiable:  endometriosis (7), uterine fibroids (3), sinusitis (4),
rheumatoid arthritis (2), ulcerative colitis (1) and included in masked
validation.  Cases from program were self-referred with at least one
of following criteria:  first-degree relative diagnosed, familial breast
or ovarian cancer syndrome, positivity for <i>BRCA1</i> or <i>BRCA2</i>,
personal history of breast cancer.  All women received annual 3-D
color doppler flow ultrasound and CA125 measurements.  Simple ovarian
cysts were found by ultrasound in 38% of unaffected women (those
having a min. 5 year follow-up without ovarian cancer).  All epithelial
subtypes of cancer were represented;  stage I cancers were representative
as they are in the community.  The median age in the healthy controls
was 49 (range 21-75) in the preliminary set and 48 (25-73) in the masked
validation set.  Median age in affected preliminary of 58 y (range 29-82)
and masked validation  of 59 (30-80) was not significantly different.
Serum was obtained before exam, diagnosis or treatment and flash frozen
in liq. nitrogen.  Samples were thawed divided into 10 &micro;l portions
and re-frozen.
<p>
Analyzed serum was thawed and added to C16 hydrophobic interaction chip
and analyzed on the Protein Biology System 2 SELDI-TOF (Ciphergen).
The mass resolution is about 400 or less, and mass accuracy calibrated
with angiotensin peptide (about 1000 ppm or 0.1%).  Data were analyzed
according to Holland's genetic algorithms and Kohonen cluster analysis.
ASCII format spectral <i>m/z</i> and intensities from SELDI-TOF (about
15,200 values) are run through the algorithm whose output is the best
fit of intensities which segregates data.  Spectra from 50 healthy
and 50 affected with biopsy-proven cancer were compared where the
algorithm identifies the <i>m/z</i> values of signficance through
iterative search.  Once key subset of the <i>m/z</i> values are
identified in the first phase analysis, a second phase to classify
unknowns is done.  Patterns of the relative intensities is matched
to an optimum pattern from the first phase analysis.  Unknowns fall
into three categories: cancer, unaffected, or new cluster.  <em>New
cluster</em> indicates a point in <i>N</i>-space outside the defined
likeness boundaries of the cancer and unaffected clusters.  A
more detailed description of results and analysis is available <a
href="http://clinicalproteomics.steem.com">online</a>.
<p>
Results show high reproducibility (&lt;10% CV) with selected
<i>m/z</i> peaks of the higher intensities.  The analysis software
took random samples and could reliably assign 100% of samples into
the proper cluster.  Two healthy and two affected samples were analyzed
and intensities at <i>m/z</i> 534, 989, 2111, 2251, and 2465
discriminatory between healthy and diseased.  There was a 100%
sensitivity and 95% specificity for this set, with positive predictive
value of 94%, compared with 35% for CA125.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Griffin TJ, Han DK, Gygi SP, Rist B, Lee H, Aebersold R, Parker KC
 </span>
 <span class="year">
(2001)
 </span>
 <span class="jname">
J. Am. Soc. Mass Spectrom.
 </span>
 <span class="jvol">
12,
 </span>
 <span class="pages">
1238-1246.
 </span>
<p class="reftitl">
Toward A High-Throughput Approach To Quantitative Proteomic Analysis:
Expression-Dependent Protein Identification by Mass Spectrometry
<p class="abstract">
<b>ABSTRACT</b>.
The isotope-coded affinity tag (ICAT) technology enables the
concurrent identification and comparative quantitative analysis
of proteins present in biological samples such as cell and tissue
extracts and biological fluids by mass spectrometry. The initial
implementation of this technology was based on microcapillary
chromatography coupled on-line with electrospray ionization
tandem mass spectrometry. This implementation lacked the ability
to select proteins for identification based on their relative
abundance and therefore to focus on differentially expressed proteins.
In order to improve the sample throughput of this technology,
we have developed a two-step approach that is focused on those
proteins for which the abundance changes between samples: First,
a new software program for the automated quantification of ICAT
reagent labeled peptides analyzed by microcapillary electrospray
ionization time-of-flight mass spectrometry determines those
peptides that differ in their abundance and second, these peptides
are identified by tandem mass spectrometry using an electrospray
quadrupole time-of flight mass spectrometer and sequence database
searching. Results from the application of this approach to the
analysis of differentially expressed proteins secreted from
nontumorigenic human prostate epithelial cells and metastatic
cancerous human prostate epithelial cells are shown.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Barth A, Zscherp C
 </span>
 <span class="year">
(2002)
 </span>
 <span class="jname">
Quart. Rev. Biophys.
 </span>
 <span class="jvol">
35,
 </span>
 <span class="pages">
369-430.
 </span>
<p class="reftitl">
What Vibrations Tell Us About Proteins
<p class="Para1">
<b>REVIEW</b>.  Avg yeast proteins has 20,000 vibrational degrees
of freedom, the <q>normal modes</q> of vibration.  A normal mode couples
movements of several internal coordinates (bond lengths and angles).
The estimate is based on 500 amino acids in a protein with an avg
15 atoms per amino acid for a total 7500 atoms, assuming each has 3 degrees
of freedom.  6 degrees of freedom correspond to translation and rotational
movements, with the balance vibrational.  Dominant vibrations include
the C=O stretch and atoms belonging to functional groups.
<p>
Vibrational spectra can be acquired using Raman scattering and IR light
absorption.  Spectral recordings plot intensity or absorbance against
wavenumbers (inverse wavelength).  This spectroscopy is very sensitive
to bond strength changes as 0.02% can be detected (based on spectral
resolution of 5 cm<sup>&minus;1</sup>.  Bond length and energy are directly
related so distortions during catalysis or binding can be monitored with
significant accuracy.
<p>
The frequency of oscillation of two atoms is given by the equation
<p class="eqnserif">
&nu; = (<i>k</i>/<i>m</i><sub>r</sub>)<sup>0.5</sup>/2&pi;
<p>
where <i>k</i> is force constant between two atoms and <i>m</i><sub>r</sub>
is a reduced mass (1/<i>m</i><sub>r</sub> = 1/<i>m</i><sub>1</sub> +
1/<i>m</i><sub>2</sub>).  An increase in <i>k</i> clearly increases
frequency, and this means electron density in bond increases.  Alterations
in density are caused by inter- and intramolecular factors.  A larger
the atomic mass, the slower the vibration.  Indeed, isotopic labeling
causes frequency shifts that are used in structural determinations.
<p>
Chemical structure of the molecule dominates spectral features and alterations
in structure are sensitive to spectral interpretation.  Protonation of
catalytically active residues can be seen in IR.  Changes in redox state
cause changes in electron density and affect vibrations.
<p>
Stretching vibrational frequencies are a direct indicator of bond force
constants.  Relationships between vibrational frequency and bond strength
(&Delta;H) were studied for serine proteases and proteins.  Proteins form
acyl-enzyme intermediates which breakdown based on interaction with acyl-enzyme
ester carbonyl.  For a dhydrocinnamoyl-chymotrypsin acyl-enzyme intermediate
a nonbonded conformation and a productive conformation were identified,
and carbonyl stretching wavenumbers shift by 39 cm<sup>&minus;1</sup>,
and this indicates a carbonyl bond strength of 26.9 kJ/mol.  Shifts in
frequency can correlated with reaction rates which are affected by changes
in bond length.
<p>
Vibrations are <i>coupled</i> typically which depends on molecular geometry
and geometry and frequency are correlated for many chemical groups.
Thus coupling provides 3D structure information.  The two C-O vibrations
in COO<sup>&minus;</sup> are coupled with stretching seen at 1400 and
1570 cm<sup>&minus;1</sup> but changing with electron density and
the O-C-O angle.  A hypothetical extreme of 90&deg; and 180&deg; gives
no coupling at 90&deg; and the strongest coupling at 180&deg;.  When the
bonds oscillate at the same frequency coupling is its strongest.
Chelation to a cation would cause changes in bond lengths and angles and
therefore frequency shifts.  Symmetric and asymmetric stretches will
report the angle between P-O and V-O in phosphate and vanadate, used in
myosin studies.  Finally the amide groups in the protein backbone will
couple with amide oscillators, the coupling dependent upon 3D structure.
<p>
<b>Hydrogen bonds</b> stabilize structures and are vital to catalysis, and
vibrational spectroscopy directly indicates their bond strength.  Bonding
lowers the frequency of stretch vibrations because it lowers restoring
force; in contrast it increases bending vibration frequencies since
it produces additional restoring force.  C=O groups that downshift
1 cm<sup>&minus;1</sup> correspond to favorable binding enthalpy of
1600-2600 J/mol for several aliphatics.  In substrate binding to enzyme
favorable enthalpies with respect to water are as large as 60 kJ/mol
when C=O groups are bound between enzyme and substrate.  For example
pyruvate binding to LDH causes downshift of substrate C=O band by
35 cm<sup>&minus;1</sup>, and the bond length only changes 0.02 &Aring;!
H-bonding to phosphate (PO<sub>2</sub><sup>&minus;</sup>) lowers the
band position in symmetric stretching by 3-20 cm<sup>&minus;1</sup> and
antisymmetric stretching by 20-34 cm<sup>&minus;1</sup>.  This is because
electron density is reduced in P-O bonds and there is a relaxation in
phosphate geometry toward the ideal tetrahedral as Coulombic repulsion
is decreased.
<p>
<b>Electric fields</b> induce alterations in electron density distribution.
These fields are natural within large molecules, such as the active site
of a dehalogenase which strongly polarizes the product of the catalysis.
<p>
<b>Bandwidth</b> in addition to band position and intensity is a third
spectral parameter useful in interpretation.  A particular conformer of
the molecule exists for 10<sup>&minus;18</sup> seconds, and the band position
for a given vibration is slightly different for every conformer.
This heterogeneity gives rise to the band broadening.  The more flexibility
in structure, the broader the band compared to more rigid structures;
hence bandwidth is a measure of conformational freedom.  Others have
related bandwidth to entropy and the effects of entropy are quantified
in catalysis.  Phosphate bands of GTP will narrow (or become sharper)
when the nucleotide binds Ras protein.  Bandwidth studies have allowed
determinations of whether a functional group or amino acid side chain
is exposed to solvent or confined to an inner part of a protein.
<p>
With infrared spectroscopy, the electric field vector <i>E</i>(<i>t</i>)
of the electromagnetic wave couples with the oscillating (vibrating)
dipole moment &mu;(<i>t</i>) of the molecule.  When the light and vibrational
frequencies are the same, the electric field amplifies the movement of
the oscillating charges.  An additional point is that the larger the point
charges +<i>q</i> and &minus;<i>q</i>, the stronger the interaction with the
electric field.  Harmonic oscillation assumes discrete energy levels
separated by <i>h&nu;</i> where <i>&nu;</i> is vibrational frequency.
Fermi's golden rule states that the transitional probability between two
vibrational levels is proportional to the square of the transition dipole
moment (TDM).  For transition from vibrational level <i>n</i> to <i>m</i>
of the electronic ground state &psi;<sub>0</sub> the Born-Oppenheimer
approximation that separates the nuclear wavefunctions <i>&phi;<sub>n</sub></i>
and <i>&phi;<sub>m</sub></i> from the electronic wavefunction &psi;<sub>0</sub>
and using U = <b>&mu;</b>(<i>t</i>)<i>E</i>(<i>t</i>) for operator of
interaction potential U:
<p>
TDM = &lang;&psi;<sub>0</sub>&phi;<sub><i>m</i></sub>|<b>&mu;</b>|&psi;<sub>0</sub>&phi;<sub><i>n</i></sub>&rang;
<p>
TDM has a electronic and nuclear component in the equation with two
selection rules:  the nuclear term is zero except when <i>m = n</i> &plusmn; 1
and indicates that a jump can only be made to the next level: &Delta;<i>n</i>
= &plusmn;1.  Most oscillators are in the ground state so it is the first
excited state that represents the transition.  For a diatomic oscillator
the TDM to this first excited state is:
<p>
TDM = &lang;&part;&mu;/&part;R(<i>R</i><sub>0</sub>)&rang;(<i>h</i>/8&pi;<sup>2</sup><i>m</i><sub>r</sub><i>&nu;</i>)<sup>0.5</sup>
<p>
The right term is the nuclear contribution, and the left term electronic,
being the expectation value for change of dipole moment at equilibrium
position <i>R</i><sub>0</sub> calculated with electronic wavefunctions.
The second selection rule is revealed here in which IR wave absorption
occurs  when dipole moment of molecule changes with vibration.  The larger
the change the stronger the absorption; a large change is correlated with
a large bond polarity.  Consider a dipole moment with partial charges
+<i>q</i> and &minus;<i>q</i> with a distance <i>R</i> separating them
and the moment <i>&mu; = qR</i>.  The change in dipole moment with any
change in <i>R</i> (effectively bond length) is &part;&mu;/&part;<i>R</i>
= <i>q</i>.  Hence the larger the partial charge, the larger the bond
polarity and the stronger the absorption.  More intense (absorbed) bands
are seen with C=O than for C=C in molecules like HFC=CH<sub>2</sub>, as
H<sub>2</sub>C=CH<sub>2</sub> does not absorb at all.  Intermolecular
actions (solvent actions) that contribute to polarization will be indicated
in absorption magnitude.
<p>
A huge table is included in this article to list the IR absorption
of amino acid side chains in the presence of water (H<sub>2</sub>O) and
deuterated water (D<sub>2</sub>O).
<table id="ir-table">
<caption>Amino acid side chain IR bands</caption>
<tr><td colspan="4">from Barth, 2000
<tr><th>Assignment
   <th>Band position in cm<sup>&minus;1</sup><br>
   (and extinction &epsilon; in M<sup>&minus;1</sup> cm<sup>&minus;1</sup>)
	<br>in <sup>1</sup>H<sub>2</sub>O
   <th>Band position in cm<sup>&minus;1</sup><br>
   (and extinction &epsilon; in M<sup>&minus;1</sup> cm<sup>&minus;1</sup>)
	<br>in <sup>2</sup>H<sub>2</sub>O
	<th>Remarks
<tr><td style="width:15em;">Cys, <i>&nu;</i>(SH)
  <td style="width:14em;">2551
  <td style="width:14em;">1849
  <td style="width:25em;">&nbsp;
<tr><td>Asp, <i>&nu;</i>(C=O) <td>1716 (280)  <td>1713 (290)
   <td>if no H-bond 1762 cm<sup>&minus;1</sup> in proteins; shift
	25 cm<sup>&minus;1</sup> down if H-bond; above 1740 cm<sup>&minus;1</sup>
	there is inverse correlation of &nu;(C=O) with dielectric constant &epsilon;
<tr><td>Glu, <i>&nu;</i>(C=O) <td>1712 (220)  <td>1706 (280)
  <td>Expected to be up to 50 cm<sup>&minus;1</sup> higher without H-bond
<tr><td>Asn, <i>&nu;</i>(C=O) <td>1677-1678 (310-330)  <td>1648 (570)
  <td>Up to 1704 cm<sup>&minus;1</sup> in proteins
<tr><td>Arg, <i>&nu;</i><sub>as</sub>(-NH-C[=NH<sub>2</sub><sup>+</sup>]-NH<sub>2</sub>)
    <td>1652-1695 (420-490)  <td>1605-1608 (460)
  <td>Postion dependent upon salt bridges between Arg and other residues
  in <sup>1</sup>H<sub>2</sub>O, not for <sup>2</sup>H<sub>2</sub>.
  In <sup>1</sup>H<sub>2</sub>O without a salt bridge, it is near 1672
  cm<sup>&minus;1</sup>.  In proteins in water it is up to 1695 cm<sup>&minus;1</sup>
  and is down to 1595 cm<sup>&minus;1</sup> in D<sub>2</sub>O.
<tr><td>Gln, <i>&nu;</i>(C=O) <td>1668-1687 (360-380)  <td>1635-1654 (550)
  <td>Between 1659 and 1696 cm<sup>&minus;1</sup> in proteins
<tr><td>Arg, <i>&nu;</i><sub>s</sub>(-NH-C[=NH<sub>2</sub><sup>+</sup>]-NH<sub>2</sub>)
  <td>1614-1663 (300-340)  <td>1581-1586 (500)
  <td>Postion dependent upon salt bridges between Arg and other residues
  in <sup>1</sup>H<sub>2</sub>O, not for <sup>2</sup>H<sub>2</sub>.
  In <sup>1</sup>H<sub>2</sub>O without a salt bridge, it is near 1635
  cm<sup>&minus;1</sup>.  In deuterated proteins
  there is shift down to 1595 cm<sup>&minus;1</sup>.
<tr><td>HisH<sub>2</sub><sup>+</sup>, <i>&nu;</i>(C=C)
  <td>1631 (250)  <td>1600 (35), 1623 (16)
  <td>Only one strong band seen for 4-methylimidazole at 1633 (H<sub>2</sub>O)
  and 1605 cm<sup>&minus;1</sup> (<sup>2</sup>H<sub>2</sub>O).
<tr><td>Lys, <i>&delta;</i><sub>as</sub>(NH<sub>3</sub><sup>+</sup>)
  <td>1626-1629 (60-130)
  <td>1201
  <td><sup>2</sup>H<sub>2</sub>O band position based on shift observed for
  CH<sub>3</sub>NH<sub>3</sub>Cl and CH<sub>3</sub>N<sup>2</sup>H<sub>3</sub>Cl
<tr><td>Tyr-OH, <i>&nu;</i>(C-C) <i>&delta;</i>(C-H)
 <td>1614-1621 (85-150)
 <td>1612-1618 (160)
 <td>&epsilon; estimated relatve to 1517 cm<sup>&minus;1</sup> band,
  Tyr or <i>p</i>-cresol
<tr><td>Asn, &delta;(NH<sub>3</sub>)
 <td>1612-1622 (140-160)
 <td><td>
<tr><td>Trp, <i>&nu;</i>(C-C), <i>&nu;</i>(C=C)
 <td>1622  <td>1618 <td>
<tr><td>Tyr-O<sup>&minus;</sup>, <i>&nu;</i>(C-C)
 <td>1599-1602 (160) <td>1603 (350)
 <td>Tyr or <i>p</i>-cresol
<tr><td>Tyr-OH, <i>&nu;</i>(C-C)
 <td>1594-1602 (70-100)  <td>1590-1591 (&lt;50)
 <td>&epsilon; estimated realtive to 1517 cm<sup>&minus;1</sup> band,
   Tyr or <i>p</i>-cresol
<tr><td>Gln, <i>&delta;</i>(NH<sub>2</sub>)
 <td>1586-1610 (220-240)
 <td>1163
 <td>
<tr><td>HisH, <i>&nu;</i>(C=C)
 <td>1575, 1594 (70)
 <td>1569, 1575
 <td>Doublet is because of two protonated tautomers of His
<tr><td>Asp, <i>&nu;</i><sub>as</sub>(COO<sup>&minus;</sup>)
 <td>1574-1579 (290-380)  <td>1584 (820)
 <td>Shift might be +60 or &minus;40 cm<sup>&minus;1</sup> when cation
 chelation occurs, and in extreme cases the band position is as
 for <i>&nu;</i>(C=O)
<tr><td>Glu, <i>&nu;</i><sub>as</sub>(COO<sup>&minus;</sup>)
 <td>1556-1560 (450-470)  <td>1567 (830)
 <td>See Asp <i>&nu;</i><sub>as</sub>(COO<sup>&minus;</sup>)
<tr><td>Lys, <i>&delta;</i><sub>s</sub>(NH<sub>3</sub><sup>+</sup>)
 <td>1526-1527 (70-100) <td>1170
 <td><sup>2</sup>H<sub>2</sub>O band position is based on the shift
   observed for CH<sub>3</sub>NH<sub>3</sub>Cl and
	CH<sub>3</sub>N<sup>2</sup>H<sub>3</sub>Cl
<tr><td>Tyr-OH, <i>&nu;</i>(C-C), <i>&delta;</i>(C-H)
 <td>1516-1518 (340-430)
 <td>1513-1517 (500)
 <td>Tyr or <i>p</i>-cresol
<tr><td>Trp, <i>&nu;</i>(C-N), <i>&delta;</i>(C-H), <i>&delta;</i>(N-H)
 <td>1509
 <td>
 <td>indole IR spectrum
<tr><td>Tyr-O<sup>&minus;</sup>, <i>&nu;</i>(C-C), <i>&delta;</i>(C-H)
 <td>1498-1500 (700)
 <td>1498-1500 (650)
 <td>Tyr or <i>p</i>-cresol
<tr><td>Trp, <i>&nu;</i>(C-C), <i>&delta;</i>(C-H)
 <td>1496
 <td>
 <td>Trp Raman spectrum, observed in the indole IR spectrum at 1487
  cm<sup>&minus;1</sup>
<tr><td>Phe, <i>&nu;</i>(C-C ring)
 <td>1494 (80) <td><td>
<tr><td>Phe, <i>&delta;</i>(CH<sub>3</sub>)
 <td>1445-1480 <td><td>
<tr><td>Trp, <i>&delta;</i>(C-H), <i>&nu;</i>(C-C), &nu;(C-N)
 <td>1462 <td>1455 (200)
 <td>&epsilon; estimated from comparison with 1571 cm<sup>&minus;1</sup>
<tr><td>His<sup>&minus;</sup>, <i>&delta;</i>(CH<sub>3</sub>),
 <i>&nu;</i>(C-N)
 <td>1439 <td>1439
 <td>Observed for 4-methylimidazole with strong contribution from
  <i>&delta;</i>(CH<sub>3</sub>).  Thus, position for His may differ
<tr><td>Pro, <i>&nu;</i>(C-N)
 <td>1400-1465 <td>
 <td>Sensitive to backbone conformation
<tr><td>Pro, <i>&delta;</i>(CH<sub>3</sub>)
 <td>1425-1475 <td>
 <td>Good group frequency, normally at 1463, near 1425 cm<sup>&minus;1</sup>,
  and more intense when next to a C=O group
<tr><td>Trp, <i>&delta;</i>(N-H), <i>&nu;</i>(C-C), <i>&delta;</i>(C-H)
 <td>1412-1435  <td>1382
 <td><sup>1</sup>H<sub>2</sub>O: higher number for Raman spectrum of Trp,
   lower number for IR imidazole spectrum.  <sup>2</sup>H<sub>2</sub>O:
	Raman spectrum of Trp.
<tr><td>Gln, <i>&nu;</i>(C-N)
 <td>1410 <td>1409 <td>
<tr><td>Glu, <i>&nu;</i><sub>s</sub> (COO<sup>&minus;</sup>)
 <td>1404 (316) <td>1407
 <td>see Asp <i>&nu;</i><sub>s</sub>(COO<sup>&minus;</sup>)
<tr><td>Asp, <i>&nu;</i><sub>s</sub>(COO<sup>&minus;</sup>)
 <td>1402 (256) <td>1404
 <td>could shift +60/&minus;90 cm<sup>&minus;1</sup> with cation chelation,
  and in extreme cases the band position is as for <i>&nu;</i>(C-O)
  of COOH group.  Band position of Asp in D<sub>2</sub> is estimated
  from shift observed for CH<sub>3</sub>COO<sup>&minus;</sup>
<tr><td>Asp, <i>&delta;</i><sub>s</sub>(CH<sub>3</sub>)
 <td>1375 or (1368, 1385) <td>
 <td>1 band for one CH<sub>3</sub> group, 2 bands for two adjacent
 groups (Val, Leu), narrower than <i>&delta;</i>(CH<sub>3</sub>) but
 same intensity.  Insensitive to hydrocarbon chain conformation.
<tr><td>Trp <td>1352-1361 <td><td>Higher number for Raman spectrum of
 Trp, lower value for IR imidazole
<tr><td>Trp <td>1334-1342 <td>1334 (100) <td>
</table>
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Rabilloud T
 </span>
 <span class="year">
(2002)
 </span>
 <span class="jname">
Proteomics
 </span>
 <span class="jvol">
2,
 </span>
 <span class="pages">
3-10.
 </span>
<p class="reftitl">
Two-Dimensional Gel Electrophoresis in Proteomics:  Old, Old-Fashioned,
But It Still Climbs Up the Mountains
<p class="Para1">
<b>REVIEW</b>. 2DGE became a more useful tool when microanalytical
techniques (such as Edman sequencing at picomole level) became available
to make use of it as a preparative tool in protein identification.
With the use of MS as a more sensitive and effective tool, particularly
with identifying post-translational modifications, the usefulness of 2DGE
increased measurably.
<p>
Disadvantages in 2DGE include inability to automate and get high throughput.
Although resolution in 2DGE is still impressive, it is insufficient for
looking at all cellular proteins.  Zoom/narrow range pH gels have helped
a great deal.  Other problems have to do with preventing abundant
proteins (such as actin, present at 10<sup>8</sup> molecules per cell)
from overwhelming detection of low abundance proteins (such as
receptors or transcription factors, present at 100-1000 molecules per cell).
In serum, albumin, at 40 mg/ml, can make it impossible to find cytokines,
at pg/ml levels.  Enrichment or prefractionation strategies must be
considered.  Other problems occur because of protein insolubility during 2DGE or with
protein extraction failing to get some proteins.
<p>
Consider trying to do work without using 2DGE.  SDS-PAGE coupled with MS/MS
is one.  The problems in focusing are eliminated, and one band on the gel
theoretically represents a single protein.  This approach worked on characterizing
spliceosomes and highly enriched chloroplast envelope preparations.
The problem is, most preparations have greater complexity, and it is
rare that an isolated band contains a single species.  SDS-PAGE has no
other role than in deconvoluting the protein mixture, to prevent <q>overcrowding</q>
of the MS/MS spectra.  One problem with gel use and extraction of peptides
is that yields are quite low (usu. &le; 1%).
<p>
Because of this, others have decided to use gel-free methods.  Yates and
coworkers have introduced MudPIT, in which extracts are digested first
into peptides (often with different enzymes), and the total digest
separated multidimensionally by chromatography and then sequenced.
Complexity reduction is in the chromatography step, but the MS/MS system
also handles several peptides at a time.  One small limitation is to
be able to automate the system such that a nanoflow ESI runs without
clogging for an extended period of time.  MudPIT also has no ability to
quantitatively assess the proteome.
<p>
Quantitative analysis of proteins using a non-gel approach would require
use of ICAT labeling, although 1 in 7 proteins has no cysteines.  Also
such agents require a reactive cysteine for proteins not too hydrophobic.
<p>
Array-based approaches like that of DNA microarrays would involve use
of a ligand bound to array substrate to bind to a labeled protein, in
something quite similar to an ELISA (antibodies to antigens).  Indeed
use of antigens or antibodies to capture materials of interest is the
focus.  The best approach may be phage display, which can possibly
generate several thousand specific binding sequences.  Other issues arise
however that mean doing several thousand immunoassays on a microscale
while retaining specificity.
<p>
One of the great advantages of 2DGE is the ability to analyze modified
proteins.  Phosphoproteins are a good example.  The complex mixture of
proteins is digested with protease(s) after reductive alkylation.
All amino groups are protected with tert-butyl oxycarbonyl (tBOC) first,
then all acid functional groups (carboxyl and phoshoryl groups) are
activated with EDC and then reacted with ethanolamine to form
ethanolamide derivatives.  The ethanolamide derivative from phosphoryl
groups are selectively removed by mild acid treatment (the phosphoramide
is more reactive than the amide of carboxyl groups) to yield the free
phosphate group on peptides.  The phosphate is then activated with EDC
again and cystamine added to derivatize it.  The disulfide in the cystamine
is then reduced (with phosphine) and an SH-reactive biotin probe (namely
an ICAT probe) is added.  The phosphopeptide has now in these multiple
reactions has been biotinylated and can be selectively retained on an
avidin column.  A strong acid can be added to deblock all derivative
groups (the phosphoramide, the carboxylamide, and the tBOC-amine).
The peptide is then subjected to MS/MS analysis.
<p>
Array-based methods for modified proteins is not yet ready.  Consider
the use of anti-phospho amino acid antibodies which work well for
phosphotyrosine but weakly bind for phosphoserine and phosphothreonine.
There is no concept for array-based methods involving other, usually
very small derivative modifications such as acylation.
<p>
The ability of 2DGE small modifications will not come in the 2nd dimension
since mass is hardly changed.  Rather many such modifications alter
charged residues and thus the pI.
<!-- ============================================================= -->
<p class="tags">
FD
<p class="ref">
 <span class="authors">
Sonsmann G, R&ouml;mer A, Schomburg D
 </span>
 <span class="year">
(2002)
 </span>
 <span class="jname">
J. Am. Soc. Mass Spectrom.
 </span>
 <span class="jvol">
13,
 </span>
 <span class="pages">
45-78.
 </span>
<p class="reftitl">
Investigation of the Influence of Charge Derivatization on the Fragmentation
of Multiply Protonated Peptides
<p class="Para1">
Tryptic peptides subjected to ESI-MS conditions tend to be doubly charged
in positive mode (doubly protonated).  When fragmented, they yield
singly-charged <i>b</i> and <i>y</i> ions.  In cases of missed tryptic
cleavage, these peptides (<q>partials</q>) contain more than one basic
amino acid and the protonation can reach 4 or 5.  Fragmentation is more
intense in this charge state, and the product ions also have higher charge
states to complicate spectrum interpretation.  Chemical derivatization of
peptides to increase charge state also improves fragmentation.  Several
mechanisms have been proposed to explain fragmentation, including
<b>charge site-induced</b> and <b>charge site-initiated</b>, with the idea
that a mobile proton is randomly distributed over the peptide backbone.
Many studies have limited themselves to one and two basic residue peptides,
but multiple basic residues would be of interest since a maximum count
of charges is expected with increased Coulombic repulsion.  This increased
protonation would also limit the movement of a mobile proton.  Because
the C-terminus is a localization of negative charge and attracts the proton,
greater fragmentation is expected near this site.  To see the influence
of negative charges, peptides with Cys were oxidized to cysteic acid.
A negative charge brings in another proton for a net single positive
charge on the molecular ion, and <i>y</i> ions (N-terminal charges)
predominate.
<p>
Native glucagon (HSQGTFTSD<b>Y</b>SKYLDSRRA<b>Q</b>DFVWLMNT, every 10th
residue in bold) was sequenced by ESI-MS/MS on a Thermoquest LCQ
with flow 1-5 &micro;l/min and capV=4500 V, temp=200&deg; and sheath
gas set to 60 (instrument value).  Range was <i>m/z</i> 250-2000,
isolation width was 3 Da.  Increasing charge states of precursor were
set with 25-50% lower collision energies, but adjusted to maximum fragmenation
efficiency.  Final spectra were acquired over 1 min (20-25 scans).
The MS/MS spectra of charge states +2, +3 and +4 showed significant
differences in fragmentation behavior.  A +5 ion was not observed in
glucagon although there are five titratable sites (bovine insulin B-peptide
is reported to have +5 ion).  Likely because Arg-17 and -18 are too close,
one of them is not protonatable.
<p>
The +2 ion of glucagon is poorly fragmented.  Ions are mainly products
from cleavage at the COOH side of Asp;  fragmentation at D residues is
observed and may be because of nucleophilic attack
on the deprotonated carboxyl group.  What likely happens is that
Asp lends its COOH proton to the amine nitrogen of the C-terminal
residue, then the oxyanion binds to the carbonyl carbon of the own
residue of Asp, forming an acid anhydride 5-membered ring, and cleaving
the peptide.
<p>
Selection of the +3 precursor glucagon shows more thorough fragmentation
with <i>b</i> and <i>y</i> ions present, these product ions also doubly
charged.   This might indicate a mobile proton on both ends is involved
in fragmentation, where fragmentation involves a charge separation mechanism.
Smaller singly charged peptide products are not detectable in the ion
trap, and the double charged fragments are of detectable intensity.
<p>
The +4 precursor shows only intense fragmentation of the C-terminus.
+3-charged <i>b</i> ions are explained by a mobile proton on the C-terminus
whereas basic residues (His, Lys, Arg) are protonated by an immobile proton.
In addition an immobile proton occupies the N-terminal His, withdrawn
from peptide backbone.
<p>
The N-terminus was derivatized with chlorosulfonylacetyl chloride (CSAC,
Cl-SO<sub>2</sub>-CH<sub>2</sub>-C(=O)-Cl) in a previously described
method:  1 nmol peptide is dissolved in 2 &micro;l 20 mM sulfonic acid
(2 &micro;l CSAC in 500 &micro;l water), which is evaporated, then
re-dissolved in 20 &micro;l THF/ethyl-diisopropylamien (4:1) and 2 &micro;l
0.1 M CSAC n dry THF added, vortexed for 30 s after 1-2 min reaction time,
vacuum dried then dissolved in 50% MeCN with 0.1% formic acid.  For
acetylation, 10% acetic anhydride in acetic acid is used for 2 h,
evaporated, then suspended in 1 M ammonium hydroxide for 20 min to
remove O-acetylations, then evaporated and stored in freezer.  It is
dissolved in same solvent just before ESI-MS.  In addition a
triphenylphosphine (Br-TPP, bromoethyl-triphenylphosphine) was prepared
and used with peptide (0.5-2 nmol) in a pH 9 Tris buffer, with a
10 nmol Br-TPP/&micro;l MeCN solution added in 20-fold molar excess,
and reacted for 3 h at 37&deg;;  derivatized peptide is collected on
SepPak or BondElut CS to remove organic products.
<p>
The +2, +3, and +4 precursor ions of derivatized glucagon were then analyzed.
The +2 precursor showed improved fragmentation with N-terminus <i>y</i>
ions of +2 charge evident.  Asp-induced fragmentation was less of a factor
based on lower intensities of these ions.  +3 charged precursor showed
additional <i>y</i> ions and a significant increase in <i>b</i> ions
with fragmentations at the C-terminus.  Hence adding a negative charge
at the N-terminus improved <i>y</i> ion appearance as expected.  As
for the C-terminus fragmentation, the second proton may locate at basic
sites in the middle of the peptide and 3rd proton must be distributed
along the C-terminus.  A plot of the +2, +3, and +4 <i>b</i> and <i>y</i>
ions of native and sulfonylated glucagon shows that sequence information
(MS/MS spectra) improved considerably with derivatization.
<p>
The bovine insulin B chain (FVNQHLC<sub>CAM</sub>GSH<b>L</b>VEALYLVC<sub>CAM</sub>G<b>E</b>RGFFYTPKA)
resembles a tryptic peptide with a Lys at position 28 near C-terminus,
and the Arg-22 represents a missed cleavage.  Charge states from +2 to
+5 were examined.  The +2 ion MS/MS spectrum reveals poor fragmentation,
with a main <i>b</i><sub>29</sub><sup>+2</sup> product.  The N-terminal
side of Pro-27 fragmented easily with neutral loss of water
(<i>b</i><sub>27</sub><sup>2+</sup> &minus; H<sub>2</sub>O).
There is very low abundance fragmentation of N- and C-terminus.
Thus with two protons at Arg and Lys in the C-terminus, fragmentation
along the backbone is unfavorable.
<p>
The +3 ion shows greater distribution of fragment ions.  Doubly charged
<i>b</i> ions (from <i>b</i><sub>14</sub><sup>2+</sup> to
<i>b</i><sub>27</sub><sup>2+</sup>) series and a <i>y</i> ion series from
<i>y</i><sub>28</sub><sup>2+</sup> to <i>y</i><sub>17</sub><sup>2+</sup>,
and so a mobile third proton is explained.  Only the Pro-Lys C-terminal
bond resisted cleavage.   The greater intensities were at both ends
of the peptide.
<p>
The +4 ion MS/MS spectrum was more difficult to interpret, because of
fragments in different charge states overlapping.  The greater intensities
are for cleavage in the middle of the peptide.  In the +5 ion precursor
spectrum only a few highly charged fragments can be seen.  The more intense
fragments are doubly charged with sequence of the center of the peptide
being given.  If the mobile proton theory is true, then a restricted mobility
of the proton would account for the dissociation distribution.  If
His at 4 and 9 are protonated in the +5 ion, and the next basic residue is
Arg-22, the fragmenation pattern shows a mobile proton trapped between
His-9 and Arg-22.
<p>
Derivatized insulin peptide at the N-terminus puts a fixed positive charge
on it (Br-TPP), with a +6 precursor now selectable.  However, this
derivative is not stable.  The +2 ion selection yielded a high intensity
fragment showing a 289 Da loss, corresponding to cleavage of TPP off the
peptide, and the same was seen for +3 and +4 precursors.  The loss is
described with plasma desorption MS as a reversed Michael-type reaction.
The MS/MS of +4 ion is quite different:  the N-terminus now has a fixed
charge and so Lys dissociation is not seen at N-terminus;  <i>y</i>
ion series at the N-terminus has been suppressed and now a <i>b</i>
ion series at the C-terminus is favored, with some +2 <i>y</i> product
ions from the C-terminus also observed.
<p>
Acetylation of the primary amines (N-terminus, &epsilon;-NH<sub>2</sub>
of Lys) should lower basicity.  Only a +2 and +3 precursor ion can be
seen.  There is better fragmentation along the middle of the peptide for
the +2 ion, and the +3 ion produces only a <i>b</i> series
(<i>b</i><sub>12</sub><sup>2+</sup> to <i>b</i><sub>27</sub><sup>2+</sup>).
A small <i>y</i> series (<i>y</i><sub>3</sub><sup>+</sup> to
<i>y</i><sub>6</sub><sup>+</sup>) derived from charge separation reactions
confirm the C-terminal sequence.  With suppression of <i>y</i> fragmenting
in the middle, a more easily intepretable spectrum is obtained.
Bromo-nicotinic acid acylation produced similar results (note the isotope
distribution of bromine for modification of the N-terminus can be used
to help distinguish <i>b</i> and <i>y</i> ions).
<p>
When derivatized with CASC a mixture of mono- and bis-derivatives were
found.  Only +2 and +3 ion precursors were observed, indicating how the
negative charge decreases overall charge state of precursors.  Only
a mono adduct (modified N-terminus) was seen.  One intense ion showed
a loss of the adduct, providing a marker for stepwise searching from
N-terminal amino acid;  the <i>y</i><sub>29</sub><sup>2+</sup> ion
had a mass difference of 78.5 amu, which is exactly the correct mass
difference for a +2 ion indicating Phe.  The next seven
(<i>y</i><sub>29</sub><sup>2+</sup> to <i>y</i><sub>22</sub><sup>2+</sup>)
fragments were determinable.  The series <i>y</i><sub>17</sub><sup>+</sup>
to <i>y</i><sub>7</sub><sup>+</sup> with charge localized to C-terminus
were readable also.  Thus derivatization improved fragmentation at the
N-terminus and the abundance of <i>y</i> ions.  Fragmentation of
the +3 precursor also showed the same dissociation pattern, with
mostly <i>y</i> series ions sequencing the N-terminus, with diminished
<i>b</i> ion series intensities.  Putting a negative charge on the N-terminus
directs fragmentation to the N-terminus, and is independent of precursor
ion charge state.  There is additional help when the adduct is lost in
some ions to help identify N-terminus ions.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Yarmush ML, Jayaraman A
 </span>
 <span class="year">
(2002)
 </span>
 <span class="jname">
Annu. Rev. Biomed. Eng.
 </span>
 <span class="jvol">
4,
 </span>
 <span class="pages">
349-373.
 </span>
<p class="reftitl">
Advances in Proteomic Technologies
<p class="Para1">
Advances in functional genomics do not obviate the need to study the
structure, function, and interaction of proteins that come into the
realm of <q>proteomics.</q> Proteomics is essential microarrays
and PCR-based methods do not provide information about the proteome
nor regulation at the transcriptional, translational, and posttranslational
levels. Studies of yeast and liver show no direct relationship between
transcriptome and proteome.  Transcripts can be processed to different
final messages that produce different protein isoforms.
<p>
The classical description of proteomics is (1) protein profiling
(2) protein identification and (3) elucidation of protein-protein interaction.
Proteome maps or fingerprints are obtained from two-dimensional gel
electrophoresis, in which an orthogonal separation according to isoelectric
point (pI) and molecular mass (MW) of subunit polypeptides.  Fingerprints
of different cellular or tissue states can be compared to identify relevant
proteins particular to that state or phenotype.  Major limitations of 2DE
as a protein profiling method is its ability to represent the entire
proteome.  Proteins have widely varying solubilities and much of the
protein is not seen with 2DE.  Solubility in IEF is particularly a problem.
Membrane proteins cannot really be analyzed by IEF and are lost in sample
preparation.  Hydrophobic proteins, basic proteins (pI &gt; 8), and high
MW (~200 kDa) proteins typically precipitate during IEF and are poorly
resolved.  Zwitterionic detergents (e.g., ASB) has improved solubilization
of hydrophobic detergents, but affect resolution of other protein classes.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Fryksdale BG, Jedrzejewski PT, Wong DL, Gaertner AL, Miller BS
 </span>
 <span class="year">
(2002)
 </span>
 <span class="jname">
Electrophoresis
 </span>
 <span class="jvol">
23,
 </span>
 <span class="pages">
2184-2193.
 </span>
<p class="reftitl">
Impact of Deglycosylation Methods On Two-Dimensional Gel Electrophoresis and
Matrix Assisted Laser Desorption/Ionization-Time of Flight-Mass Spectrometry for
Proteomic Analysis
<p class="Para1">
A great deal of papers indicate LC-MS/MS is about to replace 2DE although 2DE
still has high resolving power.  Many studies today focus on comparative expression
of proteins and few focus on PTMs in proteins.
<p>
LAMINEX cellulase mix was desalted with PD-10 using either a water or pH 7.5
sodium phosphate eluent.  Desalted samples were then concentrated on Ultrafree-15
centrifugal concentrators.  Digestion was done with PNGase F and Endo H using
manufacturer's instructions (see text for details).  O-Glycanase and sialidase
II, with &beta;-galactosidase and &beta;-N-acetylhexosamindiase were used in a
cocktail.  After all digestions, proteins are precipitated with 10% TCA/20 mM DTT
in acetone, leaving for 1 h at &minus;20&deg; collecting precipitate and washing
twice with 20 mM DTT in acetone then air-drying.  The Glyko-Free kit is TFMS +
toluene, neutralized with pyridine.  63 mM ammonium bicarbonate is used to form
a precipitate that contains the protein.  GC-MS is used to measure mole percent
of mannose, galactose, glucose, N-AG.
<p>
PNGase F and Endo H digests were examined on 1D gels with a 62 kDa standard (HRP?)
PNGase F at 36 kDa was stained as was Endo H at 29 kDa;  band shifting to a lower
MW region was clear. Samples digested with O-glycanases showed no band shifting.
The total carbohydrate and mole percent of the glycan removed was determined using
GC-MS.  Untreated protein has a 21% carbohydrate by weight and enzyme treated
had 12% by mass.  In stark contrast, TFMS-treated had very little (~2%) and each
of the carbohydrate peaks present was found only in trace amounts, indicating that
the glycoprotein gel stain was quite informative about the glycan content of treated
protein.  A table shows the mole percent:  untreated shows 66.4% mannose, 8.8%
galactose, 22.9% glucose, 1.9% N-acetylglucosamine, with no xylose detected.
Enzyme treated protein showed 73.0% mannose, 8.8% galactose, 15.9% glucose,
2.3% NAGamine, and no detected xylose.  TFMS-treated protein had no specific sugar
at all detectable.
<p>
The cellulase samples were enzymatically treated (PNGase F, O-glycanses,
and Endo H) and separated on 2D gels.  Generally few if any difference were seen
between the pre-treated and treated samples.  Note that while all three
pre-treated gels should theoretically look the same, the "pre-treatments" included
additions of reagents or of denaturing conditions, not to discount the usual
gel-to-gel variation that notoriously occurs in 2D gels.  Isoform trains or
spot patterns were seen in both pre-treated and treated.  Molecular weight changes
were not particularly noticeable.  Two small spot trains at the extreme alkaline
region disappeared with PNGase F, Endo H, and more faintly with O-glycanases.
A new spot train at a lower MW appeared:  this train probably represents
endoxylanase II as seen on the TFMS-treated control.
<p>
Contrasting enzyme and TFMS treatments, TFMS treatments resulted in visible
decreases in MW and tight charge trains.  Eight trains were visible and also
can be related to bands in the 1D Coomassie-stained gels.  2D gel pattern complexity
was reduced.  Spots were "most likely" consisting of same polypeptide or simple
mixtures.  Lower MW spots are single spots or smaller trains.  Results show
less effectiveness for enzyme treatments (probably steric hindrance);
it is reported that glycan removal is more effective if the polypeptide is
digested into peptides.  Another report shows that a cellobiohydrolase I (CBH I)
cellulase is reduced by 10 kDa when digested with Endo H and PNGase F, but
there was no such loss of mass in these experiements.  It is possible that
an inhibition occurred since our preparation was not purified.
<p>
Curiously charge trains were still evident with TFMS treatment.  Another group
showed that TFMS treatment of CBH I result in loss of isoforms and formation
of a compact spot.  Do the charge effects result from acid treatment, or are they
do to modifications not affected by TFMS degradation?
<p>
Cellulase control was run in 1D gels and stained with a phosphoprotein-specific
dye but no banding was evident (absence of sensitivity).   Sample was then
analyzed using direct inductively coupled plasma-optical emission spectrometry
and there was 327 &micro;g phosphorus per gram protein, or 0.033% phosphorus
by weight.  If the avg MW of a protein is 50 kDa, this is 0.5 mole P per mole
protein.  This would likely make a protein insensitive to a phosphoprotein stain
If phosphorylations alone accounted for the isoform train, there should be
enough phosphorylations present to be detectable by the stain.  For example
a protein with 4.39 mol P per mol protein present at 160 ng would be detected.
It thus seems unlikely that the isoform charge separation results from phosphate.
<p>
Proteins were identified using peptide mass fingerprinting with MALDI-TOF MS.
Since the protein identity was already established, this amounts to a verification
of the system.  Two proteins in the spot train are unknown, whereas 5 are
identified (&beta;-glucosidase I, CBH I, CBH II, endoxylanase II, and
endoglucanase III.  Several plugs were taken from larger protein spots (five
per protein).  A table is shown of the avg number of total peptide seen
in spectra using the three enzymes (and also three pre-treated).  Enzyme treatment
increased the number of peptides seen.  Where a protein is heavily glycosylated
there is little to no enhancement of ionization.  As for increased peptide
in mass spectra it is possible that a small amount of deglycosylation by
enzymes results in better trypsin digestion.
<p>
An interesting finding was that spot plug excision location dependence was
found for protein spots.  20 plugs were excised from random locations from
the CBH I spot, and masses compared to theoretical digets of CBH I.  The finding
was that the avg number of total peptides matched to CBH I did no vary
significantly between matching pre-treated and treated samples.  It shows that
spot samples for some proteins involves finding <q>sweet spots</q> and especially
those in a charge train.  Enzyme treatment generally did not improve the ability
to identify proteins (except in case of PNGase F).
<p>
With respect to TFMS-treated, identifications were more problematic.  Gel
spots yielded a lack of peptides specific to protein using the normal search
system.  When spectra were processed through Proteometics <i>m/z</i> software
and searched using ProFound with a public fungal database.
<p>
<b>Methods</b>.  Excised gels spots are washed in three alternate cycles of
25 mM ammonium bicarbonate and 100% MeCN, then reductively akylated with DTT
and iodoacetamide.  Digestion is with trypsin in 25 mM ammonium bicarb for 4 h
at 37&deg;.  Peptides are extractd in 3% formic acid.  0.5 &micro;l sample (from
20 &micro;l extract) is mixed with 0.5 &micro;l of CHCA satd in 50% MeCN.
The ABi Voyager DE-STR MALDI ToF MS in + ion mode with 20 kV, a 67% grid voltage,
185 ns delay, and 100 scans per spectrum in the <i>m/z</i> 700-3000.  Calibration
was internal using trypsin autolytic peaks.  The ABi PMF system uses
Protein Prospector MS-Fit at <a
href="http://prospector.ucsf.edu/">http://prospector.ucsf.edu/</a> and some
data during comparison are processed using Proteometric, Inc. <i>m/z</i> Knexus
Edition software and searched with <a
href="http://prowl.rockefeller.edu">ProFound</a>.
<p>
<b>Glycan analysis</b>.  Free glycan can be precipitated with 10% TCA in acetone
if 100 ug protein used.  Pellets are hydroylyzed with 1 M methanolic HCl, then
dried, N-acetylated with MeOH and acetic anhydride, then trimethyl-sialylated
wtih Tril-Sil.  Samples are resolved on a 30 m DB-1 column with 0.25 mm i.d.
using GC-MS.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Wehofsky M, Hoffmann R
 </span>
 <span class="year">
(2002)
 </span>
 <span class="jname">
J. Mass Spectrom.
 </span>
 <span class="jvol">
37,
 </span>
 <span class="pages">
223-229.
 </span>
<p class="reftitl">
Automated Deconvolution and Deisotoping of Electrospray Mass Spectra
<p class="Para1">
Experiments were done on quadrupole ToF with peptides in 50% MeOH
with 1% formic acid at low picomole level.  The <i>Isoconv</i> algorithm
uses the theoretical isotopic cluster and variance of peptides to deisotope
MALDI- and MALDI-PSD spectra.  Monoisotopic masses are determined
to simply interpretation of PSD spectra.  The program is modified to
deisotope multiply charged clusters in ESI-MS and ESI-MS/MS spectra, and
is called <i>ESI-Isoconv</i>.
<p>
Analyzing lower charge states (singly and doubly, for example) might
result in incorrectly calculated signals in the deconvoluted spectrum.
In contrast deconvoluting spectra from high to low charge states gets
past incorrect processing because intensities calculated for higher
charge states fall between the intensities of the recorded signals.
Only the maximum charge state <i>z<sub>h</sub></i> is required for
deconvolution and must be estimated or calculated from the spectrum.
For MS/MS spectra <i>z<sub>h</sub></i> is identical with charge of
selected precursor ion.  The basic algorithm loads the spectrum,
subtracts background, and analyzes peak form.  From low to high,
the <i>Z = z<sub>h</sub></i> is found.  The spectrum is then deconvoluted
and deisotoped with charge <i>Z</i> and stored.  <i>Z</i> is then
decremented by 1.  The deconvolution and deisotoping and saving of the
spectrum continues for all charge states down to <i>Z</i> = 1.
Finally the individual charge spectra are deconvoluted and deisotoped
for a combined spectrum.
<p>
For each data point in a signal , the mass is calculated based on <i>Z</i>
as <i>m/z</i> &times; <i>Z</i> &minus; (<i>Z</i> &minus; 1), assuming
ion [M + <i>z</i>H]<sup><i>z</i>+</sup>.  The mean isotopic pattern is
then determined (see Wehofsky et al. <i>Eur. Mass Spectrom</i> <b>7</b>:
30, 2001).  Assuming a monoisotopic peak, the mean pattern is normalized
to the intensity of the current data point and resulting intensities of the
isotopic signals of the cluster at distances 1/<i>Z</i> are subtracted
from processed spectrum (i.e. original spectrum for <i>Z = z<sub>h</sub></i>).
If assumption is that detected signal is monoistopic is correct, processed
channels contain positive (other peptide signal present at this mass)
or zero (no signal present at mass) signal intensity.  The sum of
signal intensities of detected cluster is written as monoisotopic
singly charged signal [M + H]<sup>+</sup> at mass <i>m/z</i> &times;
<i>Z</i> &minus; (<i>Z</i> &minus; 1) to a new spectrum <i>S<sub>Z</sub></i>
containing only signals of currently processed charge state <i>Z</i>.
Intensities of all signals corresponding to the cluster are substracted
from the processed spectrum <i>S</i><sub>p</sub>.  Negative intensities
indicate one or more processed channels are not part of the cluster
for that charge state.  Intensities of these data points are not altered
in processed spectrum and the data point in spectrum <i>S<sub>Z</sub></i>
would be set at zero.  The final deconvoluted and deisotoped spectrum
is calculated as the sum of the <i>z<sub>h</sub></i> spectra at each mass
<p class="eqnserif">
<i>S</i><sub>d</sub>(<i>m</i>) = <i>S</i><sub>1</sub>(<i>m</i>) + ... +
<i>S<sub>zh</sub></i>(<i>m</i>)
<p>
If signals in a cluster are not baseline-separated, any charge state
<i>z<sub>h</sub></i> selected would produce a false signal at
<i>m/z</i> &times; <i>z<sub>h</sub></i> &minus; (<i>z<sub>h</sub></i> &minus; 1).
Subtraction of intensities in processed spectrum could affect deconvolution
of incorrectly processed signals at lower charge state resulting in more
background noise in deconvoluted spectrum.  The program scans signals
from low to high and always assumes a threshold signal represents
a monoisotopic peak.  Shape and width of a monoisotopic peak should
resemble the other signals at higher <i>m/z</i> in the cluster.
The form of the peak is decided based on the following exclusion criteria
in the table below:
<table class="centertd" style="text-align:center;">
<caption>Exclusion Criteria for Checking Peak Form</caption>
<col style="width:5em;"><col style="width:2em;">
<col span="4" style="width:3em;">
<tr><th rowspan="2" colspan="2"><th colspan="4">Monoisotopic Peak
<tr><th>/  <th> \ <th> ^ <th> v
<tr><th rowspan="4">Higher Isotopic Peaks <th> / <td> + <td> &minus; <td> + <td> +
<tr><th> \ <td> &minus; <td> + <td> + <td> +
<tr><th> ^ <td> + <td> + <td> + <td> &minus;
<tr><th> v <td> + <td> + <td> &minus; <td> +
</table>
<p>
In this table, a &lsquo;+&rsquo; means that the peak form of an isotope
cluster, while &lsquo;&minus;&rsquo; indicates the signal is not part  of
the cluster and no longer considered for processed charge state.
&lsquo;/&rsquo; and &lsquo;\&rsquo; indicate ascending and descending
intensity signals, and &lsquo;^&rsquo; indicates the top of the peak
while &lsquo;v&rsquo; is the minimum between two peaks.  If processed
signals are not excluded in peak form analysis, the sum of the signal
intensities are stored in the spectrum.
<p>
In testing the algorithm on insulin, limitations were found for higher
peptides masses as higher charge states result in incorrectly assigned
signals at higher masses than the exactly assigned dominant [M + H]<sup>+</sup>
ion at 5804.7 u.  Examination of processed spectra <i>S<sub>zh</sub></i>
to <i>S</i><sub>1</sub> showed that misinterpretations occurred usually
in only one charge state (<i>S<sub>n</sub></i>) but not in the next
lower or higher charge state (<i>S</i><sub><i>n</i>&minus;1</sub> or
<i>S</i><sub><i>n</i>+1</sub>).  In contrast correctly assigned signals
of higher charged species were always present in several spectra
<i>S<sub>n</sub></i> resembling the ion seris of ESI spectra.  Thus
intensities of all corresponding data points (same masses) in both
spectra <i>S<sub>n</sub></i> and <i>S</i><sub><i>n</i>&minus;1</sub>
were multiplied:
<p class="eqnserif">
<i>P</i>(<i>n</i>,<i>n</i>&minus;1)(<i>m</i>) =
<i>S<sub>n</sub></i>(<i>m</i>) &times; <i>S</i><sub><i>n</i>&minus;1</sub>(<i>m</i>)
<p>
The <i>P</i>(<i>n</i>,<i>n</i>&minus;1) for the <i>z<sub>h</sub></i>&minus;1
spectra had signal present only from the charge-dependent spectra
<i>S<sub>n</sub></i> and <i>S</i><sub><i>n</i>&minus;1</sub>.
Removed are the singly charged impurities from solvents and buffers
as well as typical background noise.  The final deconvoluted and deisotoped
spectrum is calculated as the sum of all intensities of all data points
of spectra <i>P</i>(<i>z<sub>h</sub></i>,<i>z<sub>h</sub></i> &minus; 1)
to <i>P</i>(2,1):
<p class="eqnserif">
<i>S</i><sub>d</sub>(<i>m</i>) =
<i>P</i>(2,1)(<i>m</i>) + <i>P</i>(3,2)(<i>m</i>) + ... +
<i>P</i>(<i>z<sub>h</sub></i>,<i>z<sub>h</sub></i> &minus; 1)(<i>m</i>)
<p>
Mass accuracy to assign two data points in different spectra to the same
mass must be known before adding spectra.  If <i>m</i><sub>d</sub>
is the mass difference in the data points, then the mass accurayc has
to be at least <i>m</i><sub>d</sub> &times; <i>z<sub>h</sub></i> because
data points in <i>S<sub>zh</sub></i> are going to be very far apart.
<p>
The modified <i>Isoconv</i> worked well for small peptides so long as
isotope clusters of ion species were baseline resolved and signal
intensity between the isotopic peaks was zero.  Deconvolution of
higher charge state signals of mellitin  and insulin
[M + H]<sup>+</sup> was 2845.76 u and 5804.75 u for initial charge
states of <i>z<sub>h</sub></i> = 5 and 7, resp.  Thus additional parameters
were added that included peak form analysis to deconvolute higher charge
states with overlapping isotopic signals to suppress misinterpretations
for higher charge states.  A second parameter added was to
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Yuan X, Desiderio DM
 </span>
 <span class="year">
(2002)
 </span>
 <span class="jname">
J. Mass Spectrom.
 </span>
 <span class="jvol">
37,
 </span>
 <span class="pages">
512-524.
 </span>
<p class="reftitl">
Protein identification with Teflon as matrix-assisted
laser desorption/ionization sample support
<p class="abstract">
<b>ABSTRACT</b>.
Protein identification is a critical step in proteomics, and matrix-assisted laser desorption/ionization
time-of-flight mass spectrometry (MALDI-TOFMS) plays an important role in that identification.
Polytetrafluoroethylene (Teflon) was tested as a new MALDI sample support to improve protein
identification. The tryptic peptides obtained from a model protein were bound to the surface of a
modified MALDI sample holder via the hydrophobic interactions that occur between the Teflon surface
and the peptide ion-pairs, and the affinity of a-cyano-4-hydroxycinnamic acid for the peptides. During that
surface-binding step, the peptidemixture was also desalted and concentrated.Agreater number ofmatched
peptides and a larger sequence coverage were obtained for the proteins when Teflon was used as the
sample support compared with conventional sample preparation methods and a stainless-steel surface. In
addition, the characterization of a small amount of protein was improved with Teflon. Nine silver-stained
protein spots obtained from 2-D gel of a human cerebrospinal fluid (CSF) proteome were identified by
this method. Among the nine protein spots, peptide 6 : c3c fragment and procollagen c-proteinase enhancer
were not annotated in any published 2-D map of human CSF. A Teflon MALDI sample support is a
low-cost, simple, and effective method that can be used to improve the quality of the MALDI mass
spectrum of a complex tryptic peptide mixture, and to achieve a higher level of reliability and success in
protein identification.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Nijtmans LGJ, Henderson NS, Holt IJ
 </span>
 <span class="year">
(2002)
 </span>
 <span class="jname">
Methods
 </span>
 <span class="jvol">
26,
 </span>
 <span class="pages">
327-334.
 </span>
<p class="reftitl">
Blue Native Electrophoresis To Study Mitochondrial and Other Protein Complexes
<p class="abstract">
<b>ABSTRACT</b>.
The biogenesis and maintenance of mitochondria relies on a sizable number of proteins. Many of these proteins are organized
into complexes, which are located in the mitochondrial inner membrane. Blue Native polyacrylamide gel electrophoresis (BNPAGE)
is a method for the isolation of intact protein complexes. Although it was initially used to study mitochondrial respiratory
chain enzymes, it can also be applied to other protein complexes. The use of BN-PAGE has increased exponentially over the past
few years and new applications have been developed. Here we review how to set up the basic system and outline modications that
can be applied to address specic research questions. Increasing the upper mass limit of complexes that can be separated by BNPAGE
can be achieved by using agarose instead of acrylamide. BN-PAGE can also be used to study assembly of mitochondrial
protein complexes. Other applications include in-gel measurements of enzyme activity by histochemical staining and preparative
native electrophoresis to isolate a protein complex. Finally, new ways of identifying protein spots in Blue Native gels using mass
spectrometry are briefly discussed.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Oh-Ishi M, Maeda T
 </span>
 <span class="year">
(2002)
 </span>
 <span class="jname">
J. Chromatog. B
 </span>
 <span class="jvol">
771,
 </span>
 <span class="pages">
49-66.
 </span>
<p class="reftitl">
Separation Techniques for High-Molecular-Mass Proteins
<p class="abstract">
<b>Abstract</b>.
Many high-molecular-mass (HMM) proteins (MW.100 kDa) are known to be involved in cytoskeleton, defence and
immunity, transcription and translation in higher eukaryotic organisms. Even in the post-genomic era, purification of HMM
protein is the first important step to analyze protein composition in a tissue or a cell (proteomics), to determine protein
tertiary structure (structural biology), and to investigate protein function (functional genomics). To separate a HMM protein
from a protein mixture, ions, chaotropes (urea and thiourea), detergents and protease inhibitors in extraction media and buffer
solutions either for liquid chromatography or for gel electrophoresis should be carefully chosen, since HMM proteins tend to
be aggregates under denatured condition and their long polypeptide chains are easily attacked by intrinsic proteases during
separation procedure. Among many liquid chromatography techniques, affinity chromatography either with sequence-specific
DNA for transcription factor, or with monoclonal antibody specific for myosin heavy chain has been used for preparative
isolation of the respective HMM proteins. Though SDSPAGE could analyze the size and the quantity of megadalton
proteins, the resolution of HMM proteins is relatively poor. A newly developed pulse SDSPAGE would be able to raise the
resolution of HMM proteins compared with the conventional SDSPAGE. The 2-DE method is not particularly suitable in
analyzing HMM proteins larger than 200 kDa. However, a 2-DE method that uses an agarose IEF gel in the first dimension
(agarose 2-DE) has been shown to produce significant improvements in 2-DE separation of HMM proteins larger than 150
kDa and up to 500 kDa.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Zhou H, Ranish JA, Watts JD, Aebersold R
 </span>
 <span class="year">
(2002)
 </span>
 <span class="jname">
Nat. Biotechnol.
 </span>
 <span class="jvol">
19,
 </span>
 <span class="pages">
512-515.
 </span>
<p class="reftitl">
Quantitative Proteome Analysis by Solid-Phase Isotope Tagging and
Mass Spectrometery
<p class="Para1">
A solid-phase isotope tagging technique is described.  Beads
covalently bonded to, in this order, a photocleavable linker, the
isotope tag, and a protein thiol-reactive group are produced by
attaching a <i>o</i>-nitrobenzyl-based photocleavable linker to
aminopropyl-coated glass beads using solid-phase peptide synthesis
(see Holmes and Jones, <i>J. Org. Chem.</i> <b>60</b>: 2318-2319,
1995).  0.5 g of aminopropyl-coated controlled-pore glass beads with
amin content of ~400 &micro;mol/g (Sigma) are washed with anhydrous
DMF.  600 &micro;mol each of 1-hydroxybenzotriazole (HOBt, NovaBiochem),
Fmoc-aminoethyl photolinker (NovaBiochem), and diisopropylcarbodiimide
(DIC) were mixed 30 at room temperature, then added to beads for 90 min.
Beads are washed sequentially with DMF and dichloromethane, then capped
with 2 ml 40% acetic anhydride/60^% pyridine in dichloromethane for 30 min.
Beads again washed with DMF and treated with 20% piperidine in DMF for 30 min,
which removes Fmoc protection.  The process is repeated to add Fmoc-Leu
as the isotope tag (either the light or heavy form).  The iodoacetyl group
is attached to the beads using the method of Zhou et al. (<i>Nat.
Biotechnol.</i> <b>19</b>: 375-378, 2001):  3 equivalents of iodoacetic
anhydride are added to 1 equivalent of amino-derivatized beads (Sigma G4643)
with the presence of 3.3 equivalents of diisopropylethylamine in
dimethylformamide (DMF).  Beads are then washed successively in DMF,
water, and methanol, dried under slight vacuum, and stored at room
temperature in the dark.  The heavy form of derivatized Leu is prepared
using d7-Leu (Isotec) and Fmoc-N-hydroxysuccinimide (NovaBiochem) according
the method of Lapatsanis et al (<i>Synthesis</i> <b>1983</b>: 675-677, 1983)
with the exception that the recrystallization step was omitted for purifying
the product.
<p>
A 1 nmol Cys-containing laminin B peptide (CDPGYIGSR) and
500 pmol of phosphoangiotensin (DRVY*IHPF) were treated first with
TCEP (5 mM in 100 &micro;l 0.2 M Tris pH 8.0, 10 mM EDTA) for 30 min,
then 5 mg beads (2 &micro;mol binding capacity) were used in a 100 &micro;l
volume with constant agitation.  1 &micro;l aliquots of a supernatant
were analyzed both before and after addition of beads using &micro;LC-MS.  After 15 min
2 &micro;l of 2-mercaptoethanol is added for 5 min, then beads washed
with 2 M NaCl, methanol, and water.  Beads are suspended in 100 &micro;l
of 0.2 M Tris pH 8, 10 mM EDTA, 2% &beta;-mercaptoethanol.  500 pmol
phosphoangiotensin is added as internal standard.  A 100 watt Blak-Ray
long-wave UV lamp filtered with a 1 cm deep 10% copper (II) sulfate solution
is then used to illuminate the beads from 10 cm distance, using different
times.  1 &micro;l aliquots are analyzed on &micro;LC-MS.  Addition of
&beta;-mercaptoethanol prevents Met oxidation during photocleavage.
<p>
Note that the 360 nm light used in cleavage is chosen to minimize
other photocatalyzable side reactions.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Shimizu A, Nakanishi T, Kishikawa M, Miyazaki A
 </span>
 <span class="year">
(2002)
 </span>
 <span class="jname">
J. Chromatog. B
 </span>
 <span class="jvol">
776,
 </span>
 <span class="pages">
15-30.
 </span>
<p class="reftitl">
Detection and Identification of Protein Variants and Adducts in Blood
and Tissues:  An Application of Soft Ionization Mass Spectrometry to
Clinical Diagnosis
<p class="abstract">
<b>REVIEW</b>.
<p>
<b>ABSTRACT</b>.
The detection and identification of protein variants and abnormally increased modified proteins are important for clinical
diagnosis.We applied soft ionization mass spectrometry (MS) to analyze proteins in blood and tissues from various patients.
Over the past 8 years, we diagnosed 132 cases (55 kinds) of variant proteins including hemoglobin (Hb), transthyretin
(TTR), and Cu/Zn-superoxide dismutase (SOD-1), using MS as the leading technology. Of these variants, eight were new,
and nine were the first cases in Japan. Some abnormal Hb cause diseases, and most of them cause erroneous levels of
glycated Hb, HbA1c, i.e., a popular index of diabetes. Most of the variant TTR causes amyloidotic polyneuropathy.Variant
SOD-1 causes amyotrophic lateral sclerosis. We first showed that immunoprecipitation by a specific antiserum is a reliable
and simple method to prepare protein from sera and tissues for analysis by matrix-assisted laser desorption time-of-flight
MS, and liquid chromatographyelectrospray ionization MS (LCESI-MS). The use of this technology has become
widespread. Using an immunoprecipitated target protein and LCESI-MS, we showed that the ratios of tetra-, di- and
a-sialo-transferrin from two cases of congenital glycoprotein deficient syndrome were clearly distinguishable from those of
control samples.We first reported a unique modified form of TTR, that is, S-sulfonated TTR, which increased markedly and
specifically in three cases with molibdenum cofactor deficiency.We proposed that S-sulfonated TTR is a useful marker for
screening this disease. ESI-MS was successfully used for the accurate determination of HbA1c, and we clarified the extent of
discrepancies between the HbA1c value measured by conventional methods and the accurate values for samples containing
various Hb variants determined by the MS method.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Krutchinsky AN, Chait BT
 </span>
 <span class="year">
(2002)
 </span>
 <span class="jname">
J. Am. Soc. Mass Spectrom.
 </span>
 <span class="jvol">
13,
 </span>
 <span class="pages">
129-134.
 </span>
<p class="reftitl">
On The Nature Of The Chemical Noise in MALDI Mass Spectra
<p class="Para1">
Sensitivity in the mass spectrum is usually limited by background ions
rather than inherent limits in the detection system.  Previous work in
MALDI/ESI-QqTOF and MALDI/ESI-ion trap show background ion peaks at
every <i>m/z</i> value.  Techniques to improve signal-to-noise include
accumulation of longer-time spectra (or multiple spectra), but the
improvement is only &prop; <i>t</i><sup>0.5</sup>.  Linked scan modes
can also overcome these limitations but there is a low duty cycle because
of the need to scan the spectrometer.
<p>
A Thermo Finnigan LCQ Classic commercial ion trap MS was installed with a
new ion interface.  The standard LCQ has a set of octapoles leading to
ion trap that leads to the detector.  The ion optics for the MALDI
were fit with a quadrupole (0.635 cm dia &times; 20 cm long) between
octapoles and skimmer.  An aperture plate (~0.3 cm dia aperature)
separates the quadrupole and octapole.  The added quadrupole is a
high pressure ion guide.  A 500 kHz crystal oscillator controlled
sine wave generator and power amplifier sets an RF voltage:  amplitude
on sine wave is +300-500 V.  An accelerator is placed in the ion guide
to drag ions toward the exit:  these are four ~18 cm rods 0.32 cm dia
inserted between quadrupole rod.  These are closer to the axis of the
ion guide at the entrance, and further from it at the exit.  A constant
+100 V in all four rods creates a small electrical field along the ion
guide which improves transport.  An atmosphere of 70 &plusmn; 10 mTorr
is introduced by a small orifice, and a mechanical pump of 8 l/s drives
the turbomolecular pump of the system pushing the pressure down to
~2 &times; 10<sup>&minus;5</sup>.
<p>The MALDI target plate surface is the polycarbonate surface of a CD.
The CD is screwed to a metal plate and placed in the vacuum lock.  A
potential is applied by contact of the metal support plate with a spring
electrode.  The distance between the CD and ion guide entrace is ~1 mm.
A special shaft-wheel assembly allows rotation of the CD to align the
spot with guide entrance.  The light of a 337 nm nitrogen laser
(Laser Science, Inc.) with 10-20 Hz strobe is reflected towards a
collimating lens (<i>f</i> = 15 cm) and then through sapphire window
with angle of incidence ~60&deg; on the CD.  Light spot diameter is about
300-500 &micro;m, with power density ranging from 2-5 &times; 10<sup>7</sup>
W/cm<sup>2</sup> and controlled by a variable attenuator (Newport Corp.
Model 935-5).  A video cam allows visualization of laser light and sample
position.
<p>
Ion injection time (<q>Maximum Inject Time</q>) is 0.5 s, automatic gain
control (AGC) is set on, maximum count of ions allowed to fill trap (<q>Full MS
Target</q>) is 5 &times; 10<sup>9</sup>. MS/MS spectra were obtained with these
settings: <i>m/z</i> window (<q>Isolation Width</q>) was 3-4; activation energy
(<q>Normalized Collision Energy</q>) was 25%, <i>q</i> of activation
(<q>Activation <i>q</i></q>) was 0.25, and activation time (<q>Activation Time</q>)
was 0.3 s.
<p>
A mixture of 6 peptides; bradykinin 2-9 [PPGFSPFR], substance P [RPKPQQFFGLM],
neurotensin [pyroELYENKPRRPYIL], amyloid &beta;-protein 12-28
[VHHQKLVFFAEDVGSNK, CAS 107015-83-8], ACTH 1-24 [SYSMEHFRWGKPVGKKRRPVKVYP],
oxidized bovine insulin B chain [FVNQHLC(ox)GSHLVEALYLVC(ox)GERGFFYTPKA] were
made at 200 fmol/&micro;l each win 35% MeOH + 5% acetic acid; dilutions were
made just prior to analysis. Fresh saturated matrix solutions of 2,5-DHB, CHCA,
and sinapinic acid (Sigma) were made up in solvent just before measurement. An
equal volume of satd DHB and diluted peptide mix is prepared and 1 &micro;l is
spotted to CD target.
<p>
This system not only produces high quality spectra from low femtomole
amounts of sample peptides;  its principal feature is high speed analytical
performance.  A decent spectrum can be obtained after a 2 s acquisition,
with 5 of 6 peptides showing sufficient signals above the noise.  An
MS/MS spectrum of the <i>m/z</i> 1673 peak (neurotensin) indicated that
a 5 s acquisition gave sufficient S/N.  Sample was diluted 1/10 (to
100 attomole spotting) permitted no discernible peaks even with 60 s
acquisition.  Background ions interfere with peptide ion signals.
Their presence is largely the result of sample handling and preparation,
but the resolution and type of ion detection of the MS is also a factor.
Although no peak signals in the 60 s 0.1 fmol MS spectrum were seen,
a 60 s MS/MS acquisition of the <i>m/z</i> 1673 peak produced fragments
from the ion trap whose signals were sufficient.  Certain characteristic
fragmentations occur:  N-terminal side of proline (<i>b</i><sub>9</sub>,
<i>y</i><sub>7</sub>) and C-terminal side of Glu (<i>y</i><sub>9</sub>).
Starting from the parent ion are also a fragment series whose mass
differences are 154 and 136 Da.  DHB has a mass of 154.1 Da, while loss
of water from DHB would be 136.1 Da.  Similar losses were seen in MS/MS
spectra of every <i>m/z</i> tested.  These cluster ions have a composition
of (DHB)<sub><i>n</i></sub>XC<sup>+</sup> where X is an unknown species
and C<sup>+</sup> is a cation.  The exact composition of the cluster
species was not determined through MS<sup>n</sup> experiments because
dissociation channels increase as a function of the order of MS<sup>n</sup>
experiments.  Other matrices were used but were inferior to DHB
with respect to less intense signals, with lower shot-to-shot reproducibility,
and higher noise level.  This is probably due to the hotter nature of
CHCA and sinapinic acid.  Even so MS/MS spectra from these matrices
showed characteristic mass losses indicating the clustering nature.
<p>
<q>Broadband</q> collisional activation of part of the spectrum
was done at <i>m/z</i> 1673.  Ions isolated in a window of &plusmn;
50 <i>m/z</i> units were activated.  In unactivated controls the same
chemical noise background is observed.  A distribution outside the window
forms whose center is shifted by ~150 <i>m/z</i> units, probably representing
the loss of a single matrix molecule.  If the activation energy is increased
to the threshold for peptide ion fragmentation.  This actually reveals
the peptide more clearly at its parent mass, since the cluster is more
effectively dispersed.  Further increases do not increase relative
intensity of the parent, since fragmentation ensues.  In an MS<sup>2</sup>
stage the &quot;normalized collision energy&quot; of 20% is applied
for 0.3 s which disperses/cleans the background ion cluster from the
peptide, and in an MS<sup>3</sup> stage, the energy is increased to 25%
which results in peptide ion fragmentation, as shown in spectra.
Degradation of the cluster is not complete, showing a stable component
even at peptide fragmentation energies.  About half of the noise caused
by background is removed, resulting in a final ~3:1 signal-to-noise.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Papantonakis MR, Kim J, Hess WP, Haglund Jr RF
 </span>
 <span class="year">
(2002)
 </span>
 <span class="jname">
J. Mass Spectrom.
 </span>
 <span class="jvol">
37,
 </span>
 <span class="pages">
639-647.
 </span>
<p class="reftitl">
What Do Matrix-Assisted Laser Desorption/Ionization Mass Spectra
Reveal About Ionization Mechanisms?
<p class="Para1">
The MALDI spectra of peptides and oligosaccharides was studied using the
widest range of wavelengths and pulse durations tested to now.
MALDI is more properly described as matrix-assisted laser <i>ablation</i>
ionization since materials are actually removed with laser pulses.
In general once pulse durations exceed ~10 ps, energy is thermalized
and ablation takes place in thermal equilibrium.  The mass spectra for
erythromycin, &beta;-cyclodextrin, and angiotensin II  are obtained in
DHB generated from 266 nm with pulse lengths of 120 fs and 2 ns.
Below <i>m/z</i> 300 in the spectrum the alkali metals sodium and potassium,
matrix, and matrix fragment peaks dominate.  [DHB + H]<sup>+</sup>,
[DHB]<sup>+&bull;</sup> radical, [DHB + Na]<sup>+</sup>, [DHB-H + Na]<sup>+&bull;</sup>
radical cations are these matrix peaks.  Matrix fragment peaks involve
losses of hydroxyls, water, and carboxyl groups from 2,5-DHB parent
(MW=154).  Peak <i>m/z</i> 273 corresponds to [2DHB-OH-H<sub>2</sub>]<sup>+</sup>.
The region above <i>m/z</i> 300 is protonated analyte peaks particularly
for erythromycing and angiotensin II.  The &beta;-cyclodextrin is dominated
by sodium and potassium adducts, rather than just plain protonated analyte
which is absent.  Absence of protonated forms of oligosaccharides is not
unusual in MALDI, for sodium and potassium adduction is typical.
<p>
For erythromycin, the matrix-to-analyte intensities do not change with
variation in pulse durations
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Lee H, Griffin TJ, Gygi SP, Rist B, Aebersold R
 </span>
 <span class="year">
(2002)
 </span>
 <span class="jname">
Anal. Chem.
 </span>
 <span class="jvol">
74,
 </span>
 <span class="pages">
4353-4360.
 </span>
<p class="reftitl">
Developement of a Multiplexed Microcapillary Liquid Chromatography
System for High-Throughput Proteome Analysis
<p class="abstract">
<b>ABSTRACT</b>.
Comprehensive proteome analysis requires the identification
(and quantification) of the proteins in samples consisting of
thousands of proteins spanning a range of abundance of several
orders of magnitude. The currency of proteome analysis by mass
spectrometry is the peptides generated by protein proteolysis.
The high sample complexity of such samples requires a large
separation capacity, which is commonly achieved by fractionation
of the mixture followed by further serial separations of each
fraction. The sample throughput of proteome analysis is therefore
limited by the need to sequentially process large numbers of samples.
We have developed a novel four-plexed microcapillary liquid
chromatography system for automated, high-throughput separation
of complex peptide samples. The system supports the concurrent
separation of four different samples by directing identically
split solvent-gradient flows into four microcapillary C18 columns.
The simple design of the system achieves multiplexed separation
without the need for extra solvent pumps. Peak resolution,
reproducibility, and parallel separating capacity of the system
were investigated using standard peptides. The applicability of
the system to high-throughput protein expression profiling was
demonstrated in qualitative and quantitative analyses of protein
expression in S. cerevisiae grown on two different carbon sources
using the isotope-coded affinity tag (ICAT) reagent and
matrix-assisted desorption/ionization quadrupole time-of-flight
mass spectrometry.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Lubman DM, Kachman MT, Wang H, Gong S, Yan F, Hamler RL, O'Neil KA,
Zhu K, Buchanan NS, Barder TJ
 </span>
 <span class="year">
(2002)
 </span>
 <span class="jname">
J. Chromatog. B
 </span>
 <span class="jvol">
782,
 </span>
 <span class="pages">
183-196.
 </span>
<p class="reftitl">
Two-Dimensional Liquid Separations&mdash;Mass Mapping of Proteins from
Human Cancer Cell Lysates
<p class="Para1">
<b>REVIEW</b>.  The ability to identify key protein biomarkers involved
in cancer-transforming processes may serve as diagnostic tool in early
detection; molecular profiling of the tumor.  Standard format 2D-PAGE
reveals about 1000 proteins detected with Ag staining to 1-10 ng level
(about 500-1000 copies per cell).  Staining protocols also attempt to
quantify differences in expression.  Drawbacks of the method include
spot position and intensity reproducibility; interlab comparisons are
difficult.  The DIG method attempts to compensate:  two lysates are
labeled with different dyes and then run on the same gel, and a differential
color display is evaluated.  Other problems include labor-intensive
methods, <i>M</i><sub>r</sub> accurate to only 5-10%.  Mass-scale
automated analysis is much preferred.
<p>
pI-based separations are useful but in gels, the standard analysis
effectively separates only in the pI 4-9 range, and only the lower
pH range is separated while the higher end shows cathodic drift, resulting
in band smearing.  IPG strip use has much better performance and a greater
loadability (in milligrams), but precipitation of proteins is still a
problem;  reproducibility is also a problem.  Liquid-based IEF
(with the Bio-Rad Rotofor) may be a better alternative: stability zones
are created in 20 compartments created by monofilament polyester membranes.
Preparative scale uses a 55 ml volume, and the mini-scale is 16 ml.
Constant power is applied.  The continuous rotation of the device and
a 4-10&deg; is thought to maintain solubility as the isoelectric point.
A separation takes 3-5 h.  Liquid are drained/suctioned from each compartment
at the end.  A problem with the Rotofor is that separatiosns are not as
good as with IPG, although liquid-based seps are readily adaptable to
HPLC separations.
<p>
Chromatofocusing is an ion exchange technique where proteins bind to
an anion-exchanger and then eluted by a continuous fall in pH.
pH 4-11 ranges can be achieved.  Fractions from IEF or CF methods
separating whole cell lysates of human cell lines will have 50-150
proteins, particularly numerous in the pH 5.5-7.0 midrange of the method,
less on the basic and acidic ends.  To separate such large numbers
a nonporous C<sub>18</sub> column (1.5 &micro;m) is necessary.  Pore-based
particle columns generate smeared peaks and loss of resolution, poor
recovery.  Particularly low MW proteins (&lt;40 kDa) are recoverable
up to 90%, but separations of over 200 kDa are good.  Larger particle
columns provide good separation of over 400 kDa proteins.
<p>
Use of SDS is incompatible with RP-HPLC and ESI-MS.  Triton X-100
and Tween-20 are not compatible with either IEF or RP-HPLC (also may make
polymers interfering with MS).  The nonionic
<i>n</i>-octyl-&beta;-D-galactopyranoside or
<i>n</i>-octyl-&beta;-glucopyranoside detergents are particularly good
at solubilizing large and hydrophobic membrane proteins for IEF (Rotofor)
and RP-HPLC.  Other possibilities include sulfobetaine series, although
they interfere with protein quantitation.  Denaturing agents and chaotropes
prevent protein precipitation (mixes of urea/thiourea coupled with
DTT or tris(2-carboxyethyl)phosphine (TCEP)
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Hansen RK, Broadhurst RW, Skelton PC, Arkin IT
 </span>
 <span class="year">
(2002)
 </span>
 <span class="jname">
J. Am. Soc. Mass Spectrom.
 </span>
 <span class="jvol">
13,
 </span>
 <span class="pages">
1376-1387.
 </span>
<p class="reftitl">
Hydrogen/Deuterium Exchange of Hydrophobic Peptides in Model Membranes
by Electrospray Ionization Mass Spectrometry
<p class="abstract">
<b>ABSTRACT</b>.
We demonstrate here that the hydrogen/deuterium solvent exchange (HDX) properties of the
transmembrane fragment of the M2 protein of Influenza A (M2-TM) incorporated into lipid
vesicles or detergent micelles can be studied with straightforward electrospray (ESI) and
nanospray mass spectrometry (MS) configurations provided that key factors, including sample
preparation techniques, are optimized. Small unilamellar vesicle preparations were obtained
by solubilizing dimyristoyl phosphatidylcholine (DMPC) and the M2-TM peptide in aqueous
solution with n--D-glycopyranoside, followed by dialysis to remove the detergent.
Electron microscopy experiments revealed that subsequent concentration by centrifugation
introduced large multilamellar aggregates that were not compatible with ESI-MS. By contrast,
a lyophilization-based concentration procedure, followed by thawing above the liquid crystal
transition temperature of the lipid component, maintained the liposome size profile and
yielded excellent ion fluxes in both ESI-MS and nano-ESI-MS. Using these methods the global
HDX profile of M2-TM in aqueous DMPC vesicles was compared with that in methanol,
demonstrating that several amide sites were protected from exchange by the lipid membrane.
We also show that hydrophobic peptides can be detected by ESI-MS in the presence of a large
molar excess of the detergent Triton X-100. The rate of HDX of M2-TM in Triton X-100 micelles
was faster than that in DMPC vesicles but slower than when the peptide had been denatured
in methanol. These results indicate that the accessibility of backbone amide sites to the solvent
can be profoundly affected by membrane protein structure and dynamics, as well as the
properties of model bilayer systems.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Reid GE, McLuckey SC
 </span>
 <span class="year">
(2002)
 </span>
 <span class="jname">
J. Mass Spectrom.
 </span>
 <span class="jvol">
37,
 </span>
 <span class="pages">
663-675.
 </span>
<p class="reftitl">
&lsquo;Top Down&rsquo; Protein Characterization Via Tandem Mass Spectrometry
<p class="abstract">
<b>ABSTRACT</b>.
Technological and scientific advances over the past decade have enabled protein identification and
characterization strategies to be developed that are based on subjecting intact protein ions and large protein
fragments directly to tandemmass spectrometry. These approaches are referred to collectively as top down
to contrast them with bottom up approaches whereby protein identification is based on mass spectrometric
analysis of peptides derived from proteolytic digestion, usually with trypsin. A key step in enabling top
down approaches has been the ability to assign tandem mass spectrometer product ion identities, which
can be done either via high resolving power or through product ion charge state manipulation. The ability
to determine product ion charge states has permitted studies of the reactions, including dissociation,
ionmolecule reactions, ionelectron reactions and ionion reactions of high-mass, multiply charged
protein ions. Electrospray ionization combined with high magnetic field strength Fourier transform ion
cyclotron resonance has proven to be particularly powerful for detailed protein characterization owing to
its high mass resolution and mass accuracy and its ability to effect electron capture-induced dissociation.
Other types of tandem mass spectrometers are also beginning to find increasing use in top down protein
identification/characterization studies. Charge state manipulation via ionion reactions in electrodynamic
ion traps, for example, enables top down strategies to be considered using instruments with relatively
modest mass resolution capabilities. Precursor ion charge statemanipulation techniques have also recently
been demonstrated to be capable of concentrating and charge-state purifying proteins in the gas phase.
Advances in technologies applied to the structural analysis of whole protein ions and in understanding
their reactions, such as those described here, are providing new options for the study of complex protein
mixtures.
<!-- ============================================================= -->
<p class="tags">
FD
<p class="ref">
 <span class="authors">
Bowie JH, Brinksworth CS, Dua S
 </span>
 <span class="year">
(2002)
 </span>
 <span class="jname">
Mass Spec. Rev.
 </span>
 <span class="jvol">
21,
 </span>
 <span class="pages">
87-107.
 </span>
<p class="reftitl">
Collision-Induced Fragmentations of The [M-H]<sup>&minus;</sup> Parent
Anions of Underivatized Peptides:  An Aid to Structure Determination
and Some Unusual Negative Ion Cleavages
<p class="Para1">
<b>REVIEW</b>.  Positive mode MS for the sequencing of peptides and proteins
has been of great utility for this technique (Biemann &amp; Martin).
Negative mode MS for proteins and peptides has typically involved
mass determinations of species with rare reports of fragmentation being done
in this mode.  A survey of fragmentation of deprotonated amino acids
and peptides indicates use of FAB-MS using a glycerol matrix.  In some
reports peptide fragmentation produced breaks along the peptide backbone
allowing generation of sequence information comparable to positive ion mode
analysis.  Negative ion mode can also produce more information than positive
mode since amino acid side chains show a characteristic fragmentation
pattern more consistent than positive mode.  The tendency of Ser to lose
CH<sub>2</sub>O and Thr to lose MeCHO even dominate peptide bond fragmentation.
<p>
A complex nomenclature is used to describe peptide cleavages along the
backbone.  For MH<sup>+</sup> ions, the <i>b</i> and <i>y</i> + 2 are the
most interesting.  For negative ion spectra of deprotonated peptides,
the &alpha; and &beta; cleavages provide the corresponding equivalent.
However certain processes occur with negative ion fragmentation that do
not occur with positive ions, and mechanisms are completely different, so
a distinct terminology is called for.	The following scheme is considered:
<div style="font-size:90%;text-align:left;" class="eqn">
<ol>
<li>R<sub>1</sub>NH-<sup>&minus;</sup>C(R<sub>2</sub>)CONHCH(R<sub>3</sub>)CO<sub>2</sub>
<br>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&darr;<br>
[R<sub>1</sub>NHC(R<sub>2</sub>)=C=O&nbsp;&nbsp;&nbsp;&nbsp;<sup>&minus;</sup>NHCH(R<sub>3</sub>)CO<sub>2</sub>H]&nbsp;&nbsp;&nbsp;&nbsp;&rarr;<span style="margin-left:-1em;vertical-align:0.75em;">&alpha;</span>&nbsp;&nbsp;&nbsp;&nbsp;<sup>&minus;</sup>NHCH(R<sub>3</sub>)CO<sub>2</sub>H&nbsp;&nbsp;+&nbsp;&nbsp;R<sub>1</sub>NHC(R<sub>2</sub>)=C=O
<br>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&darr;&nbsp;&beta;<br>
[R<sub>1</sub>NHC(R<sub>2</sub>)=C=O&nbsp;&bull;&nbsp;H]<sup>&minus;</sup>&nbsp;&nbsp;+&nbsp;&nbsp;NH<sub>2</sub>CH(R<sub>3</sub>)CO<sub>2</sub>H
<li><sup>&minus;</sup>NHCH(R<sub>3</sub>)CO<sub>2</sub>H&nbsp;&nbsp;&rarr;&nbsp;&nbsp;NH<sub>2</sub>CH(R<sub>3</sub>)CO<sub>2</sub><sup>&minus;</sup>
</ol>
</div>
<p>
The &alpha; and &beta; fragmentation reaction mechanism is shown to proceed
by enolate anions.  Reaction 2 is just
a rearrangement of charge/proton from the &alpha; product.  Enolate
anions form from deprotonation or by protein transfer to -COO<sup>&minus;</sup>
or to CONH<sup>&minus;</sup>.  These anions might also form with successive
proton transfer when the peptide chain has flexible or random conformations.
<p>
Consider the dipeptide anion &alpha;-methylglycylglycinate [1 below]:
<ol>
<li>CH<sub>3</sub>NHCH<sub>2</sub>C(=O)NHCH<sub>2</sub>CO<sub>2</sub><sup>&minus;</sup> (0)
<li>CH<sub>3</sub>NH-<sup>&minus;</sup>CHC(=O)NHCH<sub>2</sub>CO<sub>2</sub>H (22)
<li>[CH<sub>3</sub>NHCH=C=O &nbsp;&nbsp; <sup>&minus;</sup>NHCH<sub>2</sub>CO<sub>2</sub>H] (75)
<li>CH<sub>3</sub>NHCH=C=O &nbsp;&nbsp;+&nbsp;&nbsp; <sup>&minus;</sup>NHCH<sub>2</sub>CO<sub>2</sub>H (95)
<li>&beta;:&nbsp;&nbsp;CH<sub>3</sub>NHC=C=O<sup>&minus;</sup>&nbsp;&nbsp;+&nbsp;&nbsp;NH<sub>2</sub>CH<sub>2</sub>CO<sub>2</sub>H (68)
<li>&alpha;: CH<sub>3</sub>NHCH=C=O &nbsp;&nbsp;+&nbsp;&nbsp; NH<sub>2</sub>CH<sub>2</sub>CO<sub>2</sub><sup>&minus;</sup> (32)
</ol>
<p>The geometries are determined at the AM1 theoretical level and
relative energies in kcal/mol are calculated using HF/6-31G(d)//AM1 theoretical
level (shown in parentheses at the end of the molecular forms).  In the
reaction, the carboxylate anion becomes an enolate anion via proton transfer.
This moves to a more activated transition state molecule [3] which is
fragmented and produces even more highly energetic products.  It is true
that the fragmentation might occur with charge in a position remote from
the fragmentation site (a <q>charge-remote</q> process), but no model
can be found to indicate an energetically process that is charge-remote.
The supposition of enolate ion formation at the fragmentation is plausible
given the energies involved.  The final products can track in two paths.
The &beta; fragmentation means that the ion (&beta; ion) is an enonide
less energetically stable than formation of a carboxylate  on the C-terminal
fragmentation product.  Experiments confirm that the &alpha; fragmentation
is more likely (being energetically favorable), with more numerous
&alpha;-ions and these ions being more intense.
<p>
The peptide pEPQVFV-nh<sub>2</sub> proved the fragmentation pattern.
The parent (M-H)<sup>&minus;</sup> has <i>m/z</i> 697 peak.  In the
range <i>m/z</i> 54-700, the following spectrum is described:
<table class="hrules" style="text-align:center;">
<tr><th><i>m/z</i>  <th>Rel. Int.  <th>Fragmentation <th>N-terminal Product
<tr><td>697         <td>40%        <td colspan="2">none (parent ion)
<tr><td>586         <td>10%        <td>&alpha;       <td>pE
<tr><td>489         <td>20%        <td>&alpha;       <td>P
<tr><td>472         <td>100%       <td colspan="2">&gamma;<b>Q<sup>3</sup></b>
<tr><td>451         <td>25%        <td colspan="2">&delta;<b>F<sup>5</sup></b>
<tr><td>434         <td>5%         <td>&beta;       <td>[FV-nh<sub>2</sub>]
<tr><td>361         <td>15%        <td>&alpha;      <td>Q
<tr><td>262         <td>5%         <td>&alpha;      <td>V
<tr><td>245         <td>5%         <td colspan="2">&gamma;<b>F<sup>5</sup></b>
<tr><td>207         <td>10%        <td>&beta;       <td>(QV)
</table>
<p>
The peptide GLFAVIKKVAKVIKKL-nh<sub>2</sub> is similarly analyzed.
<p>
The positive ion spectrum for the pyroglutamate peptide sequences the entire
peptide from the <i>b</i> and <i>y</i> + 2 cleavages taken together.
The negative ion spectrum identifies all residues, but does not indicate
the orientation of the final two (FV-nh<sub>2</sub>).  Taken together, both
the negative and positive ion spectra sequence the longer peptide. The
&alpha; fragmentation information provides all but the identity of the final
two residues.  No &beta; ions (&beta; fragmentation) can be observed however.
From the positive ion spectrum, <i>b</i> fragmentation provides sequence
for all but the first two residues, and there is no <i>y</i> + 2 peak
information.
<p>
Other backbone cleavages occur in the longer peptide.  The losses involve
Leu (not Leu-nh<sub>2</sub>), Leu-Lys, Leu-Lys-Lys, etc. and they occur
irrespective of the nature of the C-terminal group as an anion
(CO<sub>2</sub><sup>&minus;</sup> or CONH<sup>&minus;</sup>).  This
involves amide bond cleavage with migration of NH from the C-terminal
CONH<sup>&minus;</sup> group.  A cyclic intermediate (or transition) is
envisioned.  A charge-initiated process indicates that nucleophilic
addition occurs to the carbonyl group of any amide bond in the backbone.
Fragmentation occurs on the C-terminal amino acid and the charge is
left on the remaining larger peptide in a process that is a type of
&beta; process, called &beta;&prime; (beta prime).  This type of cleavage
occurs in peptides lacking Ser, Thr, Asp, Asn, Glu, or Gln (functions
that effect facile side chain fragmentation) and also those peptides
show &alpha;-cleavage but minor or no &beta;-backbone cleavage.
<p>
<b>&gamma; and &delta; cleavages with Asp and Asn (also Phe, Tyr, His, Trp)</b>.
The most intense peaks in a spectrum (above the parent intensity) are the
result of the presence of Asp and Asn residues.  Side chain fragmentation
occurs in these residues, with loss of H<sub>2</sub>O in the case of Asp
and NH<sub>3</sub> in the case of Asn.  It is proposed that for Asp
the mechanism starts with an enolate anion:
&mdash;<sup>&minus;</sup>CH-CO<sub>2</sub>H &rarr; [(&mdash;CH=C=O)
HO<sup>&minus;</sup>] &rarr; &mdash;C&equiv;C-O<sup>&minus;</sup> + H<sub>2</sub>O
or perhaps cyclization between side chain and adjacent amide with
release of water.  Cleavage always occurs between the &alpha;-amino
nitrogen and &alpha;-carbon of Asp (or Asn, Glu, or Gln, etc.):
the &gamma; fragmentation leaves the
anion on the Asp side chain, whereas the &delta; fragmentation leaves
it on the amino nitrogen (yielding an amide &delta;-ion).  This mechanism
involves a side chain with an enolate elimination (water loss) product
which can also occur with amino acids such as Phe, Tyr, His, and Trp.
Indeed this occurs with these amino acids when peptides are &le; four
residues.  It also occurs with Phe residues in larger peptides.
Molecular modeling with 6-31G(d)/AM1 was done to characterize the process
energetically.  The side chain carboxylate ion involving cyclization
to form a four-membered ring lactone would rule out the &gamma; cleavage
and so is dismissed as a mechanism requiring too much activation.
Rather enolate anion formation is considered more favorable (barrier
of 69.4 kcal/mole compared to lactone ring barrier of 77.1 kcal/mole).
Charge delocalization activates the amino-&alpha;-carbon bond with
subsequent cleavage with elimination.  The &gamma; fragmentation is
very much energetically favored.  The &gamma; ion likely cyclizes
by nucleophilic substitution with the  <i>n</i> + 1 amino and the
side chain Asp carboxyl to form a succinimide and release of water
and the anion on the &alpha;-carbon of the <i>n</i> + 1 amino acid
(enolate anion).  If the residue is Asn, the nucleophilic substitution
proposes release of NH<sub>3</sub> instead.
<p>
With the extra methylene carbon on Glu and Gln, the mechanism of &gamma;
and &delta; cleavage is different than for Asp and Asn.  Molecular modeling
proposes two mechanisms and the relative energetic favorability.
Again, an enolate anion is proposed in which the side chain
CO<sub>2</sub><sup>&minus;</sup> (for Glu) or CONH<sup>&minus;</sup> (for
Gln) draws a slightly acidic proton from the &alpha;-carbon, and in which
the O or N atom (Glu or Gln) bonds to form a 5-membered ring (lactone
or imide), followed by fragmentation with charge transfer to either the
enolate carbon on the side chain or to the nitrogen on the leaving amide.
An alternative mechanism (less favorable energetically) is formation
of the enolate on the side chain making it the nucleophile and straining
to form a three-membered ring in a very endothermic &gamma; or &delta;
fragmentation product.
<p>
Presence of Ser and Thr produce profound side chain-induced cleavages
with a loss of CH<sub>2</sub>O from Ser and MeCHO from Thr.  These
do not identify the position of these residues however.  Consider the
peptide GLFAVIKKVASVIKGL-nh<sub>2</sub>.  The base peak in the
spectrum is <i>m/z</i> 1609 and not the parent (M-H)<sup>&minus;</sup>
at 1639.  Only &alpha; ions are present and backbone fragmentation occurs
from the 1609 peak.  There are no &beta; ions, but &beta;&prime; ions
can be observed from <i>m/z</i> 1609.  A &gamma; cleavage of Phe
is seen with <i>m/z</i> 1422.
<p>
With the peptide GLLEGLLGTLGL-nh<sub>2</sub>, the parent ion at <i>m/z</i>
1152 is nearly the most intense.  The <i>m/z</i> 1108 representing loss
of CH<sub>3</sub>CHO is present but at only 15-20% intensity.  The spectrum
shows both &alpha;- and &beta;-cleavages.  The &gamma;-cleavage of E however
comes from the (M-H MeCHO)<sup>&minus;</sup> ion and not the parent.
<p>
A proposed mechanism for Ser-mediated cleavage involves a charge-remote
process in the molecular model.  The side chain forms a methoxy anion (methoxide)
which delocalizes on the methylene and alpha-carbon.  Formaldehyde is
released to form the enolate on the backbone and charge transfer occurs
to the C-side amide, with fragmentation between the &alpha;nitrogen
and carbon on the Ser + 1 amino acid, and charge retained on the Ser (now
Gly) amide nitrogen.  Explaining the backbone cleavage induced by
side chain activation is more interesting when account is taken that
they are more likely to occur as at the peptide is larger;  indeed
fragmentations did not occur for peptides fewer than 16 residues.
If losses of CHOH and MeCHO are so easily obtained from Ser and Thr,
then how can backbone cleavage compete energetically?  Perhaps the more
likely mechanism is that backbone cleavage actually facilitates
alkoxide removal from Ser and Thr, and that alkoxide elimination does
not occur independent of it.  The mechanism proposes that a nucleophile
is generated on the backbone, and in a charge-remote process, generates
the alkoxide largely through hydrogen bonding to it with concomitant
charge delocalization.  This nucleophile could be a C-terminal anion
(CO<sub>2</sub><sup>&minus;</sup> or CONH<sup>&minus;</sup>).
<p>
Other cleavages are structure-dependent cleavages that depend on a particular
3-D conformation.  Caerin-1.1 (GLLSVLGSVAKHVLPHVVPVIAEHL-nh<sub>2</sub>)
is an example.  Much of the fragmentation
in the spectrum follows standard principles but two peaks in the
spectrum at <i>m/z</i> 1957 is formed from a daughter product
representing parent less two CHOH groups
and one at <i>m/z</i> 2481, representing parent less Val.  Curiously
the relative intensity of these peaks falls when the internal energy
of the decomposing parent is increased.  The <i>m/z</i> 1957 ion is
GLLGVLGGVAKHVLPHVVPV-nh<sup>&minus;</sup>:  it forms by cleavage of
the N-C bond on Ile in the [M-H &mdash; 2CHOH]<sup>&minus;</sup>.
Using many synthetic variations of caerin 1.1, one can discern a
unique fragmentation process.  The evidence is as follows:
<ul>
<li>Changing the last two residues has no effect on the generation
of <i>m/z</i> 1957.
<li>Changing residue 23 Glu results in the absence of 1957, even if
the substitution is to Asp.
<li>Replacing His at 24 with Glu results in the absence of the <i>m/z</i>
2070 peak, which is GLLGVLGGVAKHVPHVVPVI-nh<sup>&minus;</sup>
</ul>
<p>
The only mechanism that might explain what happens is a particular
conformation that juxtaposes Ile21 and Glu23.  3-D structure analysis
of the peptide using 2-D NMR indicates that it is an amphipathic
peptide with well defined hydrophobic and hydrophilic zones with two
helical regions and a hinge near the center.  I21 and E23 are in the same
region of a helix and the E23 side chain would be positioned away from
I21.  E23 is flexible enough to bend toward I21, but a substituted
D23 would not (explaining one result).  So why does this not occur with
E24 and A22?  Because these would be too far apart.  An increase in the
internal energy of the molecule would unravel the helix and affect
the special proximity of the carboxylate group on the hydrogen of the
&alpha;-carbon on I21.  As for the second unusual peak, the loss of
the Val from the parent ion, the identification of the Val that is lost
will be necessary to arrive at some sort of explanation.  Again this
may involve an understanding of the structure.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Witzmann FA, Li J
 </span>
 <span class="year">
(2000)
 </span>
 <span class="jname">
Am. J. Physiol. Gastrointent. Liver Physiol.
 </span>
 <span class="jvol">
282,
 </span>
 <span class="pages">
G735-G741.
 </span>
<p class="reftitl">
Phosphopeptide Sequencing by In-Source Decay Spectrum in Delayed Extraction
Matrix-Assisted Laser Desorption Ionization Time-of-Flight Mass Spectrometry
<p class="abstract">
<b>ABSTRACT</b>.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Steen H, Jensen ON
 </span>
 <span class="year">
(2002)
 </span>
 <span class="jname">
Mass Spec. Rev.
 </span>
 <span class="jvol">
21,
 </span>
 <span class="pages">
163-182.
 </span>
<p class="reftitl">
Analysis of Protein-Nucleic Acid Interactions by Photochemical
Cross-linking and Mass Spectrometry
<p class="abstract">
<b>ABSTRACT</b>.
Photochemical cross-linking is a commonly used method for
studying the molecular details of protein-nucleic acid interactions.
Photochemical cross-linking aids in defining nucleic acid
binding sites of proteins via subsequent identification of crosslinked
protein domains and amino acid residues. Mass spectrometry
(MS) has emerged as a sensitive and efficient analytical
technique for determination of such cross-linking sites in proteins.
The present review of the field describes a number of
MS-based approaches for the characterization of cross-linked
protein-nucleic acid complexes and for sequencing of peptidenucleic
acid heteroconjugates. The combination of photochemical
cross-linking and MS provides a fast screening method to
gain insights into the overall structure and formation of proteinoligonucleotide
complexes. Because the analytical methods are
continuously refined and protein structural data are rapidly
accumulating in databases, we envision that many proteinnucleic
acid assemblies will be initially characterized by combinations
of cross-linking methods, MS, and computational
molecular modeling.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Gorman JJ, Wallis TP, Pitt JJ
 </span>
 <span class="year">
(2002)
 </span>
 <span class="jname">
Mass Spec. Rev.
 </span>
 <span class="jvol">
21,
 </span>
 <span class="pages">
183-216.
 </span>
<p class="reftitl">
Protein Disulfide Bond Determination by Mass Spectrometry
<p class="abstract">
<b>ABSTRACT</b>.
The determination of disulfide bonds is an important aspect
of gaining a comprehensive understanding of the chemical
structure of a protein. The basic strategy for obtaining this information
involves the identification of disulfide-linked peptides in
digests of proteins and the characterization of their half-cystinyl
peptide constituents. Tools for disulfide bond analysis have
improved dramatically in the past two decades, especially in
terms of speed and sensitivity. This improvement is largely due
to the development of matrix-assisted laser desorption/ionization
(MALDI) and electrospray ionization (ESI), and complementary
analyzers with high resolution and accuracy. The process
of pairing half-cystinyl peptides is now generally achieved by
comparing masses of non-reduced and reduced aliquots of a
digest of a protein that was proteolyzed with intact disulfide
bonds. Pepsin has favorable properties for generating disulfidelinked
peptides, including its acidic pH optimum, at which
disulfide bond rearrangement is precluded and protein
conformations are likely to be unfolded and accessible to
cleavage, and broad substrate specificity. These properties
potentiate cleavage between all half-cystine residues of the
substrate protein. However, pepsin produces complex digests
that contain overlapping peptides due to ragged cleavage. This
complexity can produce very complex spectra and/or hamper
the ionization of some constituent peptides. It may also be more
difficult to compute which half-cystinyl sequences of the protein
of interest are disulfide-linked in non-reduced peptic digests.
This ambiguity is offset to some extent by sequence tags that
may arise from ragged cleavages and aid sequence assignments.
Problems associated with pepsin cleavage can be
minimized by digestion in solvents that contain 50% H<sub>2</sub><sup>18</sup>O.
Resultant disulfide-linked peptides have distinct isotope profiles
(combinations of isotope ratios and average mass increases)
compared to the same peptides with only <sup>16</sup>O in their terminal
carboxylates. Thus, it is possible to identify disulfide-linked
peptides in digests and chromatographic fractions, using these
mass-specific markers, and to rationalize mass changes upon
reduction in terms of half-cystinyl sequences of the protein of
interest. Some peptides may require additional cleavages due to
their multiple disulfide bond contents and/or tandem mass
spectrometry (MS/MS) to determine linkages. Interpretation of
the MS/MS spectra of peptides with multiple disulfides in
supplementary digests is also facilitated by the presence of <sup>18</sup>O
in their terminal carboxylates.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Shimura K, Kamiya K, Matsumoto H, Kasai K
 </span>
 <span class="year">
(2002)
 </span>
 <span class="jname">
Anal. Chem.
 </span>
 <span class="jvol">
74,
 </span>
 <span class="pages">
1046-1053.
 </span>
<p class="reftitl">
Fluorescence-Labeled Peptide p<i>I</i> Markers for Capillary Isoelectric Focusing
<p class="Para1">
Isoelectric point markers should quickly focus to form sharp bands in a 
gradient.  The focusing of small ampholytes is dependent on 
&minus;d<i>z</i>/d(pH) at the pI value, that is 
[&minus;d<i>z</i>/d(pH)]<sub>pH=pI</sub>, where <i>z</i> = net charge of
the ampholyte.  Assuming no interdependent effects on ionizable groups in
molecule on each other, then:
<p class="eqnserif">
<i>z</i> = &sum;<sub><i>i</i></sub> (<i>n<sub>i</sub></i> / (1 + 
<i>K<sub>i</sub></i>/[H<sup>+</sup>])) &minus; &sum;<sub><i>j</i></sub>
(<i>n<sub>j</sub></i> / (1 + [H<sup>+</sup>]/<i>K<sub>j</sub></i>))
<p>
where <i>K<sub>i</sub></i> is acid dissociation constant for conjugate
acid of basic group and <i>K<sub>j</sub></i> is that constant for the
acidic groups, <i>n<sub>i</sub></i> and <i>n<sub>j</sub></i> are number
of ionizable groups in a peptide.  The value <i>z</i> = 0 should obtain
for pH = pI.  p<i>K<sub>a</sub></i> values assumed are C-term=3.6,
Asp=3.95, Glu=4.45, His=6.45
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Cooper JW, Gao J, Lee CS
 </span>
 <span class="year">
(2002)
 </span>
 <span class="jname">
Anal. Chem.
 </span>
 <span class="jvol">
74,
 </span>
 <span class="pages">
1182-1186.
 </span>
<p class="reftitl">
Gel Protein Capillary Extraction Apparatus.  Electronic Protein Transfer
<p class="Para1">
A gel protein capillary extraction apparatus is described to remove proteins
from polyacrylamide.  Fused silica capillaries with 50 &micro;m i.d./192
&micro;m o.d. are internally coated with polyacrylamide and filled with
a 20 mM Tris pH 6.8 buffer.  Modified Teflon sleeve permits extraction
end of capillary to contact the gel.  200 &micro;l buffer is spotted to
create electrical contact between capillary and gel, and the gel is connected
to ground via Pt wire.  A CZE 1000R HV power supply (Spellman HighVoltage
Electronics) is used to supply positive 7.4 kV at the outlet through
a Pt electrode, establishing a 200 V/cm field in the 37 cm capillary.
After the extraction, the capillary is transferred to buffer which has
the common ground.  A Linear UVIS 200 detector is on the outlet end
7 cm from end of capillary to monitor 214 nm.
<p>
Cytochrome c was run on 4-20% SDS-PAGE gels from Bio-Rad at 200 V constant.
Protein was stained in 0.1% CBB-R in 30% MeOH.  Loads from 2.5-50 &micro;g
were done.  Cytochrome c from gel was then extracted by capillary.
In addition a CZE capillary was set up to analyze SDS-treated
cytochrome c samples at 4 mg/ml loading.  Migration to UV detector
for gel-extracted material is about 8.1 min, slightly shorter than for CZE.
For a 2 min extraction, the base width is 0.5 min, corresponding to solution
plug of ~30 nl in the capillary.  The current fell from 9 to 6 &micro;A
during extraction.  After the 2 min, the extraction end of the capillary
was moved to a reservoir with buffer, and the current was restored to
9 &micro;A.  A calibration curve of SDS-cyt c complexes indicated that
a protein load of 25 &micro;g could be concentrated to 3.5 &micro;g/&micro;l
in a 30 nl solution plug.
<p>
Spots of cytochrome c were done at low protein loading and stained with
modified Coomassie blue, the limit being 50-100 ng/spot and in agreement
with a 100-200 ng/spot reported by others.  At 2.5 &micro;g load, the
migration time and peak profile of the stained SDS-cyt <i>c</i> were identical
to the non-stained.  The dye permits visualization with little effect on
mobility or UV absorbance.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Shevchenko A, Schaft D, Roguev A, Pijnappel WWMP, Stewart AF, Shevchenko A
 </span>
 <span class="year">
(2002)
 </span>
 <span class="jname">
Molec. Cellular Proteomics
 </span>
 <span class="jvol">
1,
 </span>
 <span class="pages">
204-212.
 </span>
<p class="reftitl">
Deciphering Protein Complexes and Protein Interaction Networks by Tandem
Affinity Purification and Mass Spectrometry
<p class="Para1">
Homologous recombination in <i>S. cerevisiae</i> is the method to
integrate fragments into the genome, permitting endogenous in-frame
fusions between ORF of choice and PCR-generated epitope tag cassettes.
There are some limitations.  It is easier to fuse tag into the C-terminus
rather than N terminus.  The selectable marker gene can be placed
beyond the 3&prime; part of the coding region, but for N-terminal
fusions the marker is between the promoter and targeted ORF, and then
has to be removed.  Generally N-terminal tagging was done only when
C-terminal tagging presented a problem.
<p>
TAP tagging has several important advantages.  The tag has two high
affinity modules, a calmodulin (CaM)-binding peptide and double
protein A epitope, both separated by a TEV protease cleavage site.
Protein complex analysis is done by loading extract supernatants on to a
IgG Sepahrose column, which is then washed with one buffer, then
equilibrated in a TEV protease cleavage buffer.  A recombinant
TEV protease solution is then added to the column, and this washed
into a CaM-Sepharose column using calcium-containing buffer.
This purification results in less background than standard immunoprecipitation.
<p>
53 genes were tagged with the TAP epitope, with 52 tagged successfully
(98%), as shown by Western blotting.  One case showed successful homologous
recombination although this was not shown by Western; caused by either
incorrect prediction of ORF or absence of protein expression.  Tagging
did not cause any lethality or severely abnormal phenotype.  For the
Set1 fusion, tagging did not affect complex formation, but did interfere
with enzymatic activity;  when tagged on the N-terminal, normal enzymatic
activity was restored.  Bait proteins were observed in 4 of 52 cases
by Western blotting.  Insufficient amounts of iPed made them undetectable
by mass spectrometry.  The codon bias index of these four proteins
is &lt;0.1 indicating they are low abundance.  The tagged proteins
ranged from 9-175 kDa and pI 4.5-10.0 with CBI from &minus;0.064 to
0.16.
<p>
Protein complex analysis by MS is complicated by the fact that
gel analysis means extracted peptides can vary.  Immunoaffinity
isolation also tends to recover non-specifically bound proteins and
are tough to scale up.  Use of large volumes of culture media cause
loss of binding specificity and increased background.  A ~20 kDa
Coomassie-stained band isolated from tagged Set1 immunoprecipitate
was found to contain 25 peptides on MALDI MS spectrum of the digest,
with the search find the S24 protein, a 15 kDa subunit of the 40S ribosome.
8 peptides were matched with 100 ppm accuracy and &gt; 50% coverage,
but the most intense peaks were not matched.  Analyzed by nanoES MS/MS
another 6 ribosomal proteins were found from sequenced peptides
which had low confidence from MALDI:  none of these identified S24
protein however.  A +2 ion with <i>m/z</i> 382.0 matched a 15 kDa
YBR258c with low confidence (because sequence was short and degenerate).
Large multiply charged peptides undergo partial fragmentation to
produce single and double-charged <i>y</i>-ions, thus making it difficult
to do searches based on tryptic digestion.  Re-checking the MALDI MS
data showed that four peptides of YBR258c had mass matches.  Hence
MALDI and ESI MS alone can leave some proteins unidentified, but together
produce greater confidence.  Tagging of YBR258c confirmed it to be
a subunit of the protein complex.
<p>
Immunoprecipitates were run on SDS-PAGE with controls.  All bands not
present in controls were analyzed by MS, as well as analyzing all bands.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Mitaku S, Hirokawa T, Tsuji T
 </span>
 <span class="year">
(2002)
 </span>
 <span class="jname">
Bioinformatics
 </span>
 <span class="jvol">
18,
 </span>
 <span class="pages">
608-616.
 </span>
<p class="reftitl">
Amphiphilicity Index of Polar Amino Acids As An Aid In The Characterization
of Amino Acid Preference at Membrane-Water Interfaces
<p class="abstract">
<b>ABSTRACT</b>.
MOTIVATION: An amphiphilicity index of amino acid residues was developed
for improving the method of transmembrane helix prediction.
RESULTS: The transfer energy of a hydrocarbon stem group beyond
the gamma-carbon was calculated from the accessible surface area,
and used to index the amphiphilicity of the residue. Non-zero
amphiphilicity index values were obtained for lysine, arginine,
histidine, glutamic acid, glutamine, tyrosine and tryptophan.
Those residues were found to be abundant in the end regions of
transmembrane helices, indicating their preference for the
membrane-water interface. The moving average of the amphiphilicity
index actually showed significant peaks in the end regions of most
transmembrane helices. A dispersion diagram of average amphiphilicity
index versus average hydrophobicity index was devised to facilitate
discrimination of transmembrane helices. AVAILABILITY: The
amphiphilicity index has been incorporated into a system,
SOSUI, for the discrimination of membrane proteins and the
prediction of tranmembrane helical regions
(http://sosui.proteome.bio.tuat.ac.jp/sosuiframe0.html).
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Adam GC, Sorensen EJ, Cravatt BF
 </span>
 <span class="year">
(2002)
 </span>
 <span class="jname">
Molec. Cell. Proteomics
 </span>
 <span class="jvol">
1,
 </span>
 <span class="pages">
781-790.
 </span>
<p class="reftitl">
Chemical Strategies for Functional Proteomics
<p class="Para1">
<b>REVIEW</b>.  Strategies in proteomics can be divided into two areas
of complementary objectives:  (1) the global characterization of
protein expression, (2) global characterization of protein function.
With respect to expression, 2DE and MS are used for separation, detection,
and identification.  Low abundance and membrane-associated proteins
are not well characterized by 2DE approaches though.
<p>
Global, high throughput methods are being used to look at function.
Large scale yeast two-hybrid screens show protein-protein interactions,
and the development of protein chip microarrays look at recombinantly
expressed proteins.  Nonetheless, even these methods do not look at
proteins in their native environment.
<p>
Additional methods include chemical tools to look at function and
interaction.  Changes in protein abundance are currently examined
by staining which are cost-effective but Coomassie and silver staining
response is nonlinear or insensitive or has a limited dynamic range.
SYPRO Ruby offers new possibilities but there is still problems with
resolution of membrane-associated and low abundance proteins.
<p>
<b>ICAT</b>.  Isotope-coded affinity tags (ICATs) are chemical probes
of proteins that aid in measuring abundance.  Light- and heavy-isotope
labeled iodoacetamide are used to tag two different proteomes.  The
tagged material is then combined, digested with protease (trypsin),
and then passed through an avidin affinity column to trap tagged peptides
only (the label contains a biotin).  These are run on LC-MS and the ratio
of signal intensities shows the relative level of expression.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Royle L, Mattu TS, Hart E, Langridge JI, Merry AH, Murphy N, Harvey DJ,
Dwek RA, Rudd PM
 </span>
 <span class="year">
(2002)
 </span>
 <span class="jname">
Anal. Biochem.
 </span>
 <span class="jvol">
304,
 </span>
 <span class="pages">
70-90.
 </span>
<p class="reftitl">
An Analytical and Structural Database Provides a Strategy
for Sequencing O-Glycans from Microgram Quantities
of Glycoproteins
<p class="Para1">
O-glycans are released by hydrazinolysis using strict anhydrous conditions
(GC shows water content &lt; 0.8%) since water presence causes &beta;-elimination
(peeling).  Peeling results in elimination of terminal (core) GalNAc from some
susceptible oligosaccharides, and 2-aminobenzamide can be labeled on teh exposed
galactose reducing terminal.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Forde CE, McCutchen-Maloney SL
 </span>
 <span class="year">
(2002)
 </span>
 <span class="jname">
Mass Spec. Rev.
 </span>
 <span class="jvol">
21,
 </span>
 <span class="pages">
419-439.
 </span>
<p class="reftitl">
Characterization of Transcription Factors by Mass Spectrometry
and the Role of SELDI-MS
<p class="abstract">
<b>ABSTRACT</b>.
Over the last decade, much progress has been made in the field
of biological mass spectrometry, with numerous advances in
technology, resolution, and affinity capture. The field of genomics
has also been transformed by the sequencing and characterization
of entire genomes. Some of the next challenges lie
in understanding the relationship between the genome and the
proteome, the protein complement of the genome, and in characterizing
the regulatory processes involved in progressing from
gene to functional protein. In this new age of proteomics, development
of mass spectrometry methods to characterize transcription
factors promises to add greatly to our understanding of
regulatory networks that govern expression. However, at this
time, regulatory networks of transcription factors are mostly
uncharted territory. In this review, we summarize the latest
advances in characterization of transcription factors by mass
spectrometry including affinity capture, identification of complexes
of DNA-binding proteins, structural characterization,
determination of proteinDNA and proteinprotein interactions,
assessment of modification sites and metal binding, studies
of functional activity, and the latest chip technologies that use
SELDI-MS that allow the rapid capture and identification of
transcription factors.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Hancock WS, Wu S-L, Shieh P
 </span>
 <span class="year">
(2002)
 </span>
 <span class="jname">
Proteomics
 </span>
 <span class="jvol">
2,
 </span>
 <span class="pages">
352-359.
 </span>
<p class="reftitl">
The Challenges of Developing A Sound Proteomics Strategy
<p class="Para1">
<b>REVIEW</b>.  Many aspects of proteomic studies are difficult than genomic
analysis.  For example sample preparation is more variable and typically
difficult:  proteins have much more complicated chemistry and a representative
sample can be hard to come by.  A typical protocol is a mixture
of human plasma protein (20 &micro;g/&micro;l).  Human growth hormone
is present at 0.6 ng/&micro;l and is reductively alkylated (DTT/iodoacetate).
A 1 ml sample is run on Vydac C-4 column (2.1 &times; 150 mm) with
TFA/acetonitrile and elutes in 2 ml, is dried down and then digested
with trypsin (1:50 w/w in 1 ml total in 0.1 M ammonium bicarb for 12 h
at RT, a 2nd aliq. added for another 12 h).  A 1:100 splitter reduces flow
from 150 &micro;l/min to 1.5 &micro;l/min.  A 2% MeCN with 0.1% formic acid
flow is done (3 min)  then ramped to 60% MeCN in either 180 or 590 min,
then ramped to 80% MeCN for 20 min.  5 &micro;l is loaded into 10 &micro;l
loop on to LCQ Deca-XP Microtech C-18 capillary column (150 &micro;m &times;
10 cm) connected orthogonally to ion source with sheath flow at 3 units
and temp at 140&deg;, capillary voltage 2800 V and collision energies
being 35% of MS/MS mode setting.
the integration of the two-dimensional (2-D) gel platform with mass spectrometry measurements.
The use of both matrix-assisted laser/desorption ionization (on off-line coupling)
and electrospray (on-line) ionization are complementary. While the use of onedimensional
and 2-D gels are essential to many aspects of proteomics research (sample
preparation, preliminary fractionation and quantitation, storage of protein components),
the emergence of shotgun sequencing based on high performance liquid chromatography
and tandem mass spectrometry offers a powerful new approach. The latter has
particular utility in the characterization of low level samples and complex post-translational
modifications. The development of capillary columns, such as 75 to 150 micron,
that can be packed in a reproducible manner has been a key step in the development of
high sensitivity liquid chromatography/mass spectrometry analysis.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Jonsson A, Hjelmqvist L, J&ouml;rnvall H, O'Malley R, Swainston N, Young P,
Millar A, Uria D, Langridge J
 </span>
 <span class="year">
(2002)
 </span>
Presented at 5<sup>th</sup> Siena Meeting, Italy 2-5 September 2002
<p class="reftitl">
Automated LC-MS/MS Data Analysis of A Novel Alcohol Dehydrogenase Form
Isolated from Pigeon Liver
<p class="Para1">
<b>POSTER</b>.  ESI LC-MS/MS at nanoscale level is important in the high
throughput identification of proteins in a proteome whose complex mixture
is full of tryptic peptides.  Some peptides remain <q>orphaned</q>:
that is, their masses and (partial) sequences were not matched to
database proteins.  Proteomes of many species are not well-represented
in databases, and a single amino acid substitution (quite common
with differences at the genera or class level) leaves the peptide
orphaned.  The number of spectra that pile up for manual handling
becomes enormous.  The use of an <i>automated</i> approach to
MS/MS spectrum processing can be done using a tiered series of
algorithms.  Unannotate MS/MS spectra emerging from the initial
database search are passed to a modification search workflow;
this is set to consider amino acid substitutions and other modifications
of proteins already identified by 1st pass matched peptides.
Simultaneously a sequence analysis routine can be applied to
all MS/MS spectra.  Assembly of the overlaps of peptide sequences
is then done.
<p>
The setup includes a CapLC HPLC system, a 320 &micro;m &times; 5 mm
PepMap C<sub>18</sub> trap and a 75 &micro;m &times; 100 mm
analytical C<sub>18</sub> column.  Solvent A is 5% MeCN + 0.1%
formic acid and solvent B is 95% MeCN + 0.1% formic acid and
the program starts sample loading on to the trap at 30 &micro;l/min
followed by running at a 200 nl/min flow at 0% B, then applying
a 40 min gradient to 60% B.  1 second MS scans on the QToF Ultima
were obtained with DDA-switching to MS/MS, 3 channels of 5 seconds
each.  Spectra not representing peptides sufficiently are discarded
by the query interface (ProteinLynx).  MS/MS spectra passing filter
are transposed to single charge state and centroided by MaxEnt 3.
Data are represented in XML format and searched against SWISS-PROT.
A post-search filter is applied to hits in which partial sequences
from spectra are compared to database sequences, and any mismatch
disqualifies the hit.  Spectra disqualified are then run through
a modifications search which includes amino acid substitutions
and non-specific cleavage.  In ProteinLynx, the Databank Search
Query has the following parameters set aside different from
defaults:  peptide tolerance: 50 ppm; fragment tolerance: 0.5 Da;
fixed modifications: carbamidomethyl C.  The AutoMod Query is
set for the same peptide and fragment tolerances;  modifications
and substitutions are set to be considered with maximum modifications
or substitutions per peptide set at 2 and subsitution likelihood
set at &minus;4.  A filter is also applied.
<p>
An alcohol dehydrogase was purified from pigeon liver proteome,
tryptically digested and applied to system.  123 MS/MS spectra
were obtained on peptides.  ADH from Japanese quail was matched
on only 1 peptide with no missed cleavages or modifications set
(2% coverage, score 67.5, submitted <i>m/z</i> 450.29,
charge state 2, mass 898.549, sequence IIAIDINK).  When resubmitted
with Cys modification considered and 1 missed cleavage, 6 forms
of ADH from different species were offered.  One 2+ peptide at
<i>m/z</i> 465.2 identified to try substitutions (here at position
8 in D to E) was found.  Double substitutions also yielded results
for two 2+ peptides, <i>m/z</i> 538.8 and 592.3 which included
cleavage at non-tryptic site.  The coverage was increased to include
ADH3 Cotja sequence.  The Bayesian probabilistic sequencing algorithm
MassSeq was used for all MS/MS spectra.  77 of 123 MS/MS spectra
provided sequence information.  38 peptides were annotated after all
passes.  A novel ADH sequence is obtained from all overlapping peptide
sequence information.  In addition to trypsin, Lys C, chymotrypsin
and thermolysin digests were done.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Belgacem O, Buchacher A, Pock K, Josic D, Sutton C, Rizzi A, Allmaier G
 </span>
 <span class="year">
(2002)
 </span>
 <span class="jname">
J. Mass Spectrom.
 </span>
 <span class="jvol">
37,
 </span>
 <span class="pages">
1118-1130.
 </span>
<p class="reftitl">
Molecular Mass Determination of Plasma-Derived Glycoproteins by Ultraviolet
Matrix-Assisted Laser Desorption/Ionization Time-of-Flight Mass Spectrometry
with Internal Calibration
<p class="Para1">
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Drews O, Weiss W, Reil G, Parlar H, Wait R, G&ouml;rg A
 </span>
 <span class="year">
(2002)
 </span>
 <span class="jname">
Proteomics
 </span>
 <span class="jvol">
2,
 </span>
 <span class="pages">
765-774.
 </span>
<p class="reftitl">
High pressure effects step-wise altered protein expression in <i>Lactobacillus 
sanfranciscensis</i>
<p class="Para1">
High hydrostatic pressure treatments of the bacteria were optimized and
standardized.  Stress from 25 to 200 MPa were applied and proteomes of
Cells were grown in numbers to the same optical density as that of cells
that were grown after pressure treatment.  Nonetheless there were no
difference in proteome maps of bacteria grown under normal conditions and those
grown to lower numbers in order to eliminate confounding variables.  Proteome
maps of <i>Lactobacillus sanfranciscensis</i> were made of bacterial culture
that was split and one of the fractions subjected to high pressure.
Proteome patterns from 0.1 to 200 MPa in 25 MPa steps permits a pressure-dependent
analysis of expression changes also.  Pressure ramping was randomized as well
since there can be heating and cooling of the bacteria and no temperature
effects can be discounted from occurring.  In optimizting pressure treatment,
the bacteria were assessed before and after under the microscope, pH was
measured
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Cohen AM, Rumpel K, Coombs GH, Wastling JM
 </span>
 <span class="year">
(2002)
 </span>
 <span class="jname">
Int. J. Parasitol.
 </span>
 <span class="jvol">
32,
 </span>
 <span class="pages">
39-51.
 </span>
<p class="reftitl">
Characterisation of global protein expression by two-dimensional electrophoresis 
and mass spectrometry: proteomics of <i>Toxoplasma gondii</i>
<p class="Para1">
Tachyzoites from cloned RH strain of <i>T. gondii</i> (obtained from 
Pfefferkorn) were maintained in African green monkey kidney cells.  Culture
medium is Iscove's MDM with 50 U/ml-50 &micro;g/ml pen-strep in 5% FCS
at 37&deg; in 5% CO<sub>2</sub> humidified incubator.  Infection ratio is
4:1 parasite:host and tachyzoites are harvested at 3 d.  Parasites are 
separated from host cells by filtering through 3 &micro;m polycarbonate
Nucleopore Track-Etch membranes (Whatman).  Filtered tachyzoites are washed
in PBS (pH 7.4) by spinning at 1500 <i>g</i> for 20 min at 4&deg;.  A final
wash is done with centrifugation at 16,000 <i>g</i> for 3 min at 4&deg; and
stored in batches of 50 million @ &minus;80&deg;.  Protein assays are done
using BCA (Pierce); these are read @560 nm in Titertek Multiskan MCC/340 MTP
reader.
<p>
Frozen tachyzoite pellets were disrupted in 40 &micro;l 8 M urea, 4% CHAPS,
40 mM Tris, and frozen (in liq N<sub>2</sub>) and thawed in 3 cycles.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Hastings CA, Norton SM, Roy S
 </span>
 <span class="year">
(2003)
 </span>
 <span class="jname">
Rapid Comm. Mass Spectrom.
 </span>
 <span class="jvol">
16,
 </span>
 <span class="pages">
462-467.
 </span>
<p class="reftitl">
New Algorthms for Processing and Peak Detection in Liquid Chromatography/Mass 
Spectrometry
<p class="Para1">
LC/MS enables id and quantification of thousands of analytes in single injection.
Present peak detection algorithms  attempt to identify peaks in individual
ion chromatograms, or ion chromatograms made by averaging all signals in a mass
range.  With smaller sample volumes and/or lower concentrations, noise generally
increases against signal.  Detection in MS involves numerous physical mechanisms
before ion is detected.  At each point there is noise with a particular distribution,
and this accumulates to final measured ion current.  Noise is basically chemical
and electrical. Histograms (plots of # of occurrences vs. log intensity) of
two samples, tryptic digests of high MW fraction of human serum and 5-protein
mixtures are shown.  The histogram is produced by treating LC/MS data as 2D
matrix.  Signal intensity of matrxi points is on x-axis, and frequency is 
in which intensity range occurred is on y-axis (histogram described).  Figure
shows only small fraction of matrix point have signal because of analytes of
interest, and they do not significantly affect histogram. Spectra acquired
on Finnegan LCQ Deca with or with nanoESI heads (Protana, Denmark; New Objective,
USA). and data not normalized before making histogram.  Serum had been ultrafiltered
in 10 kDa cutoff.  Separately a 5-protein mix (equine myoglobin, bovine RNase A,
BSA, bov. cytochrome c, human Hb) prepared in DTT/iodoacetic acid+NaOH before
trypsinization.
<p>
Histograms of serum and 5-protein mix quite similar, described by sum of
two normal distributions in log scale.  There is a separation of the peak
by a few orders of magnitude in intensity.  Major noise component is chemical,
and the log-normal distribution at higher intensity is associated with this
noise type.  In the 5-protein mix there are lower ion current intensities and
so the noise is likely to be electrical.
<p>
<b>Processing</b>.  Generally noise and artifact removal focuses on peak
sharpening or removing nose and artifacts in corrupted chromatograms.
Backfolding methods are used to sharpen peaks and increase intensity of signal
by concentrating them in single scan.  Signals are located in 2D dataset since
sharpened peaks are prominent in total ion chromatogram, but loss of lineshape
info complicates peak verification when nose on par with intensity of sharpened
signals.  When noise is significantly stronger than intensity of sharpened
analyte signals this method fails, since noise obscures analytes in TIC after
backfolding.  The latter algorithm, called CODA, depends on averaging filter
to identify noisy runs and handles non-normally distributed noise since smoothed
and unsmoothed chromatograms differ.  But the entire ion chroamtogram and not
just noise component is removed from filtered dataset.  Nonlinear filters are
proper for handlings data with noise not normally distributed in linear scale.
A filtering scheme based on median filter in which a simple nonlinear filter
replaces each point in a data vector with the median of the point and some 
number of surrounding points.  Such a filter has not been reported on the 
retention time axis, although used in mass spectra and cyclic voltammetry data.
It should completely remove spectral features narrower than half-width of filter,
and those wider than the filter width pass unchanged, although intenisty of
signal slightly attenudated at or near peak maximum.  Since it is an averaging
filter, performance results improve from increased sampling (assumes increased
sampling does not result in degraded signal-to-noise in individual scans).
<p>
Relative performance of mean (boxcar averaging) and median filters studied by
use of a protein mixture dataset with noise distriubitons described above.
Analytes elute typically in a period where 20-25 scans are collected.
Dataset were mean or median filtered along time axis with 9-scan filter width.
A filter width of half analyte signal width in scans maximizes discrimination
between analyte and noise, but small variations in this value do not impact 
results.  The base peak intensity is normalized to 10<sup>9</sup> count and
the spectrum converted to log scale, then noise distributions calculated.
Both mean and median filtering shifted noise distribution peak to the left,
decreasing noise level.  Signal-to-noise is increased if signal remains 
unchanged.  Filter effectiveness is judged by shift magnitude of noise distributions.
Median filtering is more effective for both distributions.  Using a bimodal
Gaussian distribution with software Grace, an improvement of fitting the curves
was obtained.  The mean filter shifts high intensity noise distribution to left
by 1.26 and shifts low intensity to left by factor of 1.32.  Median filter
shifts noise to left by factor of 1.78 and low intensity noise by 1.86.  Generally
higher intensity (chemical) noise distribution is responsible for obscuring analyte
signals and in this case the median filter is 1.78/1.26 = 1.4 times more 
effective in reducing noise than the mean filter.
<p>
NMR software packages are quite developed in looking at variation and detecting
local maxima in multidimensional data exceeding a signal-to-noise.  Candidate
peaks are then subjected to additional filtering routines.  Noise level is 
assessed from signal-free part of spectrum.  The program AUTOPSY determines noise
value as a geometric mean of noise levels of all vectors intersecting that point.
The noise threshold varies from point to point in response to noise concentrated
in regions of the dataset.  LC-MS spectral noise and signals is not always 
amenable to algorithms for NMR data.  Artifacts from mobile phase clusters and 
column bleed occur at discrete <i>m/z</i> ratios.  Chemical noise occurs in
particular <i>m/z</i> ranges and particular compositions of mobile phase.
Peak detection can be done on the ion chromatogram itself or by first averaging
ion chromatograms in the <i>m/z</i> range of interest.
<p>
The peak detection algorithm developed here takes a point (represented by 
<i>m/z</i> and retention time) and for it to be a peak, there must be a peak at
that <i>m/z</i> in the mass spectrum for that retention time, and there must be
a peak at that retention time in the ion chromatogram containing that <i>m/z</i>
ratio.  Clustered points (<i>m/z</i> and time) passing the criteria applied along
ion chromatogram vector intersecting that point and also applied along the mass
spectrum vector intersecting that point comprise peaks.  These are then further
optimized and filtered.  This vectorized peak detection has two advantages.
One: it is more robust to variation in noise and artifact distribution.  Unlike
in AUTOPSY NMR data, points on ion chromatograms corrupted by solvent clusters
or column bleed are identified as candidate peaks if they exceed a threshold
defined by level of chemical noise in that ion chromatogram, regardless of how
noise-free adjoining mass spectra may be at that <i>m/z</i> value.
<p>
Vectorized peak detection also permits existing algorithms for peak detection
in ion chromatograms and mass spectra to be used.  Estimating noise in LC-MS
data is complicated by centroiding and thresholding done at data acquisition time.
This method usese only simple estimate: third-median of all nonzero points in
vector under consideration.  Bayesian or other proprietary methods in commercial
MS software packages identifies candidate peaks in either ion chromatogram or
mass spectra.  Line shape and isotope distribution can be included in peak
detection.
<p>Comparing vectorized and global threshold methods is shown in fig 4 of LC-MS
of human serum.  A trade-off between detecting spurious peaks and missing peaks
due to analytes of interest.  The contamination caused by chemical noise is
apparent in upper right.  With a global threshold some spurious peaks are
detected.  With vectorized method, the threshold is varied (not set at a global
level) and adapts to noise, avoiding the detection of noise as peak.  when chemical
noise is greater in intensity than weak peaks in uncontaminated regions, using
single global threshold cannot identify all peaks simultaneously in uncontaminated
regions, avoid identifying spurious, and pick the intense peaks in contaminated
regions
<!-- ============================================================= -->
<p class="tags">
silver-associated peptides, MALDI, FD
<p class="ref">
 <span class="authors">
Chu IK, Cox DM, Guo X, Kireeva I, Lau T-C, McDermott JC, Siu KWM
 </span>
 <span class="year">
(2003)
 </span>
 <span class="jname">
Anal. Chem.
 </span>
 <span class="jvol">
74,
 </span>
 <span class="pages">
2072-2082.
 </span>
<p class="reftitl">
Sequencing of Argentinated Peptides by Means of Matrix-Assisted Laser
Desorption/Ionization Tandem Mass Spectrometry
<p class="Para1">
In a previous work, we prepared oligopeptides (tryptic digests) and then added
silver(I) nitrate before doing ESI-MS/MS using low energy CID.  Unlike protonated
peptides, the argentinated peptides produced predominantly N-terminal ions
(one <i>a</i> type, two <i>b</i> types) into a recognizable triplet pattern such that
[b<sub>n</sub>+OH+Ag]<sup>+</sup> is 18 u higher than [b<sub>n</sub>-H+Ag]<sup>+</sup>,
and [b<sub>n</sub>-H+Ag]<sup>+</sup> is 28 u higher than [a<sub>n</sub>-H+Ag]<sup>+</sup>,
the differences being H<sub>2</sub>O and CO loss, respectively.  An algorithm is
used to identify the triplet and doublets of fragment ion peaks, computes mass differences
between neighboring triplets, and then proposes the residue removed.  Silver ion
in ESI might reduce ion intensity and analytical sensitivity.  Another problem in
ESI is multiple charge formation that includes Ag<sub>+</sub> and H<sub>+</sub>,
which complicates use of a simple algorithm.  In this study, a MALDI MS/MS is used
(ABi/MDS SCIEX QSTAR Pulsar i QqTOF MS and a ABi/Perceptive Voyager DE-STR MALDI ToF).
Peptides are prepared at 10 &micro;mol/L in 50% MeOH with 2 mM AgNO<sub>3</sub> or
with 0.1% TFA when they are to be protonated.  1 &micro;l petpide solution mixed with
9 &micro;l satd CHCA or DHB, then dried-droplet spotted to target plate (1 &micro;l).
<p>
MALDI-MS spectra are compared for protonated and argentinated peptides ranging from
50-1140 fmol peptide.  Sensitivity is comparable for both cationizations:
[M+Ag]<sup>+</sup>/[M+H]<sup>+</sup> ratios are from 0.2 to 0.6, showing that a significant
fraction of peptides are argentinated.  Angiotensin I has the highest affinity for Ag ion,
not only in the intensity ratio, but also producing [M-H+2Ag]<sup>+</sup> and
[M-2H+3Ag]<sup>+</sup> ionic forms.
<p>Bradykinin shows a typical spectrum in which the peak cluster of [M+Ag]<sup>+</sup> shows
the <sup>107</sup>Ag and <sup>109</sup>Ag isotopes of equal abundance (107/109 ratio
is 1.076).  A third peak is in that cluster is often the most abundant though,
because it is the sum of the intensity of the
[<sup>13</sup>C<sub>2</sub>-M+<sup>107</sup>Ag]<sup>+</sup> with that of
[M+<sup>109</sup>Ag]<sup>+</sup>
<p>CID spectra were collected on peptide MGGFLR as [M+Ag]<sup>+</sup>.  The quadrupole
was set to filter at 788.25 Th to remove one of the Ag doublets.  The classic
triplet pattern of two <i>b</i> and one <i>a</i> type ions is present, and the tag
-GFL/IR is readable from the spectrum.
<p>
Previous work shows that N-terminal ions corresponding to the C-terminal side fragment
of a proline residue are low in intensity.  This is beccause an argentinated oxazolone
cannot be produced by cleavage of proline on C-terminal side: prolyl residue lacks
an N-H hydrogen for oxazolone formation.  Bovine casein 192-198 is a tryptic peptide
AVPYPQR.  The [b<sub>5</sub>-H+Ag]<sup>+</sup> has very low intensity, while
[b<sub>5</sub>+OH+Ag]<sup>+</sup> and [a<sub>5</sub>-H+Ag]<sup>+</sup> have better
intensity, and enabled assignment of the other b<sub>5</sub> ion.  The complement of
b5 is [y<sub>3</sub>-H-Ag]<sup>+</sup> and this C-terminal ion was in greater
abundance:  its mass confirms the -PQR tag.  It was unusually intense for an
R-terminating peptide.  There is also a rather abundant N-terminal prolyl peptide
corresponding to [y<sub>5</sub>-H+Ag]<sup>+</sup>, not much apparently because the
complement N-terminal triplet is missing.  Note that intense
[y<sub>n</sub>-H+Ag]<sup>+</sup> ions with N-terminal proline parallels the finding
in protonated peptides.
<p>
CID spectra also show that protonated and argentinated peptides coexist, with the
abundance ratio of [M+Ag]<sup>+</sup>/[M+H]<sup>+</sup> being from ~0.2 to 1.0.
The bovine &alpha;-casein tryptic peptides were studied.  Peptides 140-147, 106-115,
38-49, 121-134, 23-37, and 119-134 make up 28% of the residues.  Abundance ratios
were hard to determine in this mix.  37 fmol of BSA digest was analyzed and the
spectrum shows the mix of protonated and argentinated peptides.  9 ions of tryptic
peptides (18% of sequence) were seen in H<sup>+</sup> and Ag<sup>+</sup> forms.
Four Ag<sup>+</sup> peptide ions were fragmented and gave minimum of three residues
from C-terminal ions.  This combination of tags and tryptic masses run through
MASCOT gave unambiguous identifcation (MOWSE=307, with &gt;71 being significant).
<p>
MEF2A protein was digested and mixed with silver nitrate and both MS and MS/MS spectra
were collected.  Two MS/MS spectra focused on m/z 1545.59 and 1532.51 Th gave partial
sequences of -GN(I/L)GMNSR and -PHESR, respectively.  The MASCOT processing produced
a score of 165 (&gt;65 significant) for MEF2A.  The protonated MS/MS spectra were less
informative.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Candiano G, Musante L, Bruschi M, Ghiggeri GM, Herbert B, Antonucci F,
Righetti PG
 </span>
 <span class="year">
(2002)
 </span>
 <span class="jname">
Electrophoresis
 </span>
 <span class="jvol">
23,
 </span>
 <span class="pages">
292-297.
 </span>
<p class="reftitl">
Two-Dimensional Maps in Soft Immobilized pH Gradient Gels:
A New Approach To The Proteome of The Third Millennium
<p class="abstract">
<b>ABSTRACT</b>.
Same major improvements in proteome analysis of cytosolic and membrane proteins
by two-dimensional mapping are here reported. A much improved transfer of proteins
from the first to the second dimensional sodium dodecyl sulfate (SDS)-gel is obtained
by simply diluting the gel matrix, normally composed of 4%T polyacrylamide in all
commercially available Immobiline strips down to as low as 3%T. In the analysis of total
lysates of platelets, this augmented transfer has been evaluated as being 23 times
higher than in standard 4%T gels. A second major improvement, in the case of analysis
of membrane protein preparations, has been demonstrated to consist in a delipidation
step in a tertiary solvent mixture composed of tri-n-butyl phosphate:acetone:methanol
in a 1:12:1 ratio. By adopting this protocol, large amounts of spectrins (240220 kDa,
filamentous proteins of the red blood cell membranes) could be transferred vs. essentially
none when delipidation was omitted. The present report also confirms the importance
of a reduction and alkylation step of the protein sample prior to all electrophoretic
steps, including focusing in the Immobiline gel, as recently reported by Herbert et al.
(Electrophoresis 2001, 22, 20462057).
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Langridge J, Wallace A, Millar A, Young P, O'Malley R, Swainston N,
Skilling J, Hayes J, Richardson K
 </span>
 <span class="year">
(2002)
 </span>
Presented at EPS, Sorrento, Italy 1-7 September 2002
<p class="reftitl">
A Fully Automated Software Strategy for <i>De Novo</i> Sequencing
of Whole LC-MS/MS Datasets
<p class="Para1">
<b>POSTER</b>.  A problem of MS/MS spectra is that matches are
not made in searches of known protein sequences.  Only peptides
within the database are identified, and good quality spectra with
single amino acid differences are unmatched.  Sequences can be
used for BLAST-based homology searches or for preparing degenerate
primers.  An automated, high-throughput sequence analysis system
is provided.  The Q-ToF Ultimate API hybrid quadrupole oa-ToF
is used for data generation.  Two proteins were digested with
trypsin, mixed in equimolar amounts.  100 fmoles were injected.
System is fitted with 10-port valve, stream select module.
Trap (300 &micro;m &times; 5 mm) and analytical (75 &micro;m
&times; 150 mm) containing PepMap C18 (LC Packings) were used.
The nano-LC sprayer is set up at 3000 V with a 5 psi nebulizing
gas.  Flow to sprayer is 200 nl/min with pump set at 2 &micro;l/min
with 1:9 splitter.  An internal reference is used to provide
mass accuracy tolerance of 5 ppm in both MS and MS/MS mode.
Workflow is such that a filter is applied to dataset to remove
spectra insufficient to represent a peptide; remaining spectra
are recalculated using MaxEnt3 to zero-charge spectra;
XML submissions are made and post-search filter applied to
returned hits for validation.
<p>
The two proteins produced 30 ions, and 25 of these were matched
to either pyruvate kinase or FBP aldolase A, the proteins.
11 ions produced identical sequences.  With 14 cases, a high
degree of homology was found. 5 ions had not database entry:
these were re-submitted to a BLAST-P search against nrDB, where
a match was made against the two proteins.  It was found that
the sequence was non-tryptic, explaining why no match was made.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Gobom J, Mueller M, Egelhofer V, Theiss D, Lehrach H, Nordhoff E
 </span>
 <span class="year">
(2002)
 </span>
 <span class="jname">
Anal. Chem.
 </span>
 <span class="jvol">
74,
 </span>
 <span class="pages">
3915-3923.
 </span>
<p class="reftitl">
A Calibration Method That Simplifies and Improves Accurate Determination
of Peptide Molecular Masses by MALDI-TOF MS
<p class="abstract">
<b>ABSTRACT</b>.
The use of delayed ion extraction in MALDI time-of-flight
mass spectrometry distorts the linear relationship between
m/z and the square of the ion flight time (t2) with
the consequence that, if a mass accuracy of 10 ppm or
better is to be obtained, the calibrant signals have to fall
close to the analyte signals. If this is not possible,
systematic errors arise. To eliminate these, a higher-order
calibration function and thus several calibrant signals are
required. For internal calibration, however, this approach
is limited by signal suppression effects and the increasing
chance of the calibrant signals overlapping with analyte
signals. If instead the calibrants are prepared separately,
this problem is replaced by an other; i.e., the ion flight
times are dependent on the sample plate position. For this
reason, even if the calibrants are placed close to the
sample, the mass accuracy is not improved when a higherorder
calibration function is applied. We have studied this
phenomenon and found that the relative errors, which
result when moving from one sample to the next, are
directly proportional to m/z. Based on this observation,
we developed a two-step calibration method, that overcomes
said limitations. The first step is an external
calibration with a high-order polynomial function used for
the determination of the relation betweenm/z and t2, and
the second step is a first-order internal correction for
sample position-dependent errors. Applying this method,
for instance, to a mass spectrum of a mixture of 18
peptides from a tryptic digest of a recombinant protein
resulted in an average mass error of 1.0 ppm with a
standard deviation of 3.5 ppm. When instead using a
conventional two-point internal calibration, the average
relative error was 2.2 ppm with a standard deviation of
15 ppm. The new method is described and its performance
is demonstrated with examples relevant to proteome research.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Carpenter KH, Wiley V
 </span>
 <span class="year">
(2002)
 </span>
 <span class="jname">
Clin. Chim. Acta
 </span>
 <span class="jvol">
322,
 </span>
 <span class="pages">
1-10.
 </span>
<p class="reftitl">
Application of Tandem Mass Spectrometry to Biochemical Genetics and
Newborn Screening
<p class="abstract">
<b>REVIEW</b>.
<p>
<b>ABSTRACT</b>.
Tandem mass spectrometry (MS/MS) has become a key technology in the fields of biochemical genetics and newborn
screening. The development of electrospray ionisation (ESI) and associated automation of sample handling and data
manipulation have allowed the introduction of expanded newborn screening for disorders which feature accumulation of
acylcarnitines and certain amino acids in a number of programs worldwide. In addition, the technique has proven valuable in
several areas of biochemical genetics including quantification of carnitine and acylcarnitines, in vitro studies of metabolic
pathways (in particular h-oxidation), and diagnosis of peroxisomal and lysosomal disorders. This review covers some of the
basic theory of MS/MS and focuses on the practical application of the technique in these two interrelated areas.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Hack CA, WH Benner
 </span>
 <span class="year">
(2002)
 </span>
 <span class="jname">
Rapid Commun. Mass Spectrom.
 </span>
 <span class="jvol">
16,
 </span>
 <span class="pages">
1304-1312.
 </span>
<p class="reftitl">
A Simple Algorithm Improves Mass Accuracy to 50-100 ppm for Delayed
Extraction Linear Matrix-Assisted Laser Desorption/Ionization Time-of-Flight
Mass Spectrometry
<p class="Para1">
A linear MALDI-ToF with delayed extraction was built in-house.  Sample is
placed on circular stainless steel probe 4 mm dia attached to end of long
rod inserted into vacuum chamber.  Rod is mounted so that probe center is
1.5 mm from centerline of vacuum chamber, allowing probe rotation to obtain
fresh spots.  Rotation is done after 50-100 pulses.  A Voyager Biospectrometry
Workstation (PerSeptive Biosystems) operated in linear mode with DE; it
takes a 192-spot plate ea. with 1 mm dia spots surrounded by hydrophobic
surface.  Sample irradition is in 3 ns pulses with 337 nm nitrogen laser.
Spots are made in sandwich method: base layer is satd matrix solution alone
(in some cases, 1/20 dilution of CHCA in acetone).  The second layer includes
peptides or proteins mixed to 0.1-5 &micro;M in a solution with 20-fold dilute
satd CHCA with equal vols water and MeCN containing 0.1% TFA (v/v).  In
some cases samples are dried under a nitrogen stream.
<p>
A linear MALDI ToF has the following components.  A sample plate and
extraction grid with the distance between them called the extraction
region (for the Voyager workstation, it is 2.5 mm).  A grounded grid
entry plate is located 17.2 mm further.  After that is the free flight
tube or acceleration region whose length is 1228.8 mm.  The in-house
built system values are 9.5, 27.9 and 1189.6 mm, resp.  Acceleration
voltage is 15,000 V, and after the delay time the extraction grid is
reduced to 95% of the sample plate voltage.  A guide wire is positioned
on the beamline axis between grounded grid and the MCP, set to &minus;3 V.
Delayed extraction times are 200-500 ns.  As the extraction and accleration
regions are longer in the in-house built system, larger voltage drops and longer
delayed extractions times are necessary.
<p>
The initial velocity of ions (<i>v<sub>0</sub></i>) can be calculated
by observing changes in the ion TOF resulting from changes in extraction
delay times:
<p class="eqnserif">
<i>v<sub>0</sub></i> ~ &beta;/<i>t</i> (<i>dt / d&tau;</i>)
<p>
where <i>t</i> is time of flight, <i>dt / d&tau;</i> is change in
time of flight with respect to delayed extraction time, and &beta; is
property of instrument geometry and ratio between voltages on sample
electrode and extraction grid.  With voltage ratio of 89% in in-house
system, &beta;=0.0356;  for the Voyager with ratio of 95%, &beta;=0.0709.
The ion initial velocity must be known to use the Juhasz calibration
equation.  Nitrocellulose was included in a matrix and this might have
an effect on ion initial velocity.  Using Substance P[1-7] fragment,
an initial velocity of 300 &plusmn; 10 m/s was obtained with CHCA alone
and 310 &plusmn; 13 m/s (each <i>n</i> = 5) with CHCA + nitrocellulose.
The difference is not significant by Student's <i>t</i> test.
While initial velocities were not statistically significant, the flight
times of ions with nitrocellulose were always longer than for samples
without nitrocellulose.  There was a 99% probability that these flight
times differed with the Substance P fragment, and 82% with a dynophorin
peptide (this peptide has a 1600 Da mass compared to the 900 Da mass of
substance P frag).  The difference in time is 1-2 ns, which is not
much when the electronics digitize at 2 ns intervals.  These differences
might result in errors of up to 200 ppm in calibration equations however.
<p>
Five peptides with masses in the range 500-3000 Da were measured in the
in-house instrument with CHCA matrix (no nitrocellulose).  The calibration
would be &radic;<i>m/z</i> = <i>At</i> + <i>B</i> for a linear fit.
<p>
The mass error (Da) is then plotted against the peptide mass (Da) and
reveals peptides at the extreme of the mass range with &delta;<i>m</i>
&gt; 0 and those in the middle having &delta;<i>m</i> &lt; 0.   The
lowest absolute deviation is 0.9 Da.  The mean absolute deviation is
1262 ppm.  A parabola can be fit to the deviation; the quadratic equation
then shows the deviation from expected values.  The mass can then be
corrected using the quadratic equation.  The accuracy of each mass
is then increased dramatically, with all corrected values within 0.12 Da
of theoretical and an average error of 59 ppm, a 21-fold increase.
<p>
The same ToF data were then calibrated with the Juhasz et al equation
which adds second and third order terms to linear calibration, but
which do not depend on the ion ToF, but rather initial velocity and
instrument geometry, and are essentially constants.  With this third
order calibrated mass, errors are calculated and still the average
error is 1150 ppm, hardly any improvement.  A plot of the error
curve of the adjusted masses is not really different than the uncorrected
plot.  This plot was used to create a quadratic equation to correct the
data just as with the data not corrected by third order equation.
The correction had an average error of 60 ppm like the previous correction.
The third order correction becomes unnecessary and removes the need to know
the initial ion velocity, which is not known with a high degree of accuracy
anyway.
<p>
The calibrants used were not exactly evenly spaced.  The span was from
500-3000 Da but four were &lt; 2000 Da and three were spaced 600 Da apart.
The parabolic/quadratic fit then was weighted to the 1000-1600 Da region,
and <i>R</i><sup>2</sup> for the correction was 0.9979.  When calibrants
were chosen with more evenly spaced masses, the weighting was more equal
and <i>R</i><sup>2</sup> became 0.9992.
<p>
Tryptic peptides of myoglobin were mixed with the 5-peptide mix.
ToF data for the 5 calibrants were generated for both first order (linear)
and third order Juhasz equation calibration curve.  The average error
of the linear calibrated (fit) masses is 230 ppm.  With the Juhasz calibration
the average error is 150 ppm.  When quadratic correction is applied, the
error is reduced to 4 ppm, a 50-fold improvement over the linear, and 30-fold
over the third order calibration.  Applying the correction to the myoglobin
peptide masses, the masses are submitted to MS-Fit in Protein Prospector
with 100 ppm tolerance.  If the masses had been calibrated linearly, the
average error would be 204 ppm, requiring a tolerance of 300 ppm to make
the identification.  With quadratic correction, the average error is reduced
to 28 ppm.
<p>
Cytochrome c from horse and cow were analyzed to distinguish digests from
highly similar amino acid sequences.  There is only a 3 amino acid difference
in the 154 residues of the protein.  Quadratic corrections applied in the
four internal calibrants reduced differences to 10-30 ppm of theoretical.
A db search with 100 ppm tolerance then 500 ppm was done.  Only one hit
was found for the 100 ppm search, which was the correct identification
for horse cytochrome c.  The other cytochrome c 100 ppm search found cow and
dog cytochrome c, with identical sequences in the regions of digest
fragments.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Langridge J, Wallace A, Millar A, Hughes C, Vissers H, Dourdeville T,
Young P
 </span>
 <span class="year">
(2002)
 </span>
Presented at 19<sup>th</sup> Montreux Symposium, Montreux, Switzerland,
6-8 November 2002
<p class="reftitl">
High Sensitivity Phosphoprotein Analysis Using A Combination of Variable
Flow Chromatography and Precursor Ion Discovery on a Q-ToF Mass
Spectrometer
<p class="Para1">
<b>POSTER</b>.  Precursor ion scanning is the best approach for
analyzing post-translational modifications.  Typical applications
will scan through a protein digest and search only for peptides
that are potentially modified (phosphorylated).  During an HPLC
run the MS is switched at one-second intervals between low and
high energy collision energy.  The first data set is low (10 eV)
energy and shows normal pseudo molecular ions.  The second data
set is at higher energy to look at fragments.
<p>
As the HPLC gradient develops the energy is switched at 1-second
intervals between low and high energy, and the quadrupole (MS1)
is set to operate in RF mode only:  the full mass range is
passed to the ToF analyzer.  Product ions seen in the high
energy dataset  may cause a switch to MS/MS mode to select precursors
to find true parent.
<p>
The following workflow shows what happens:
<ol>
<li>HPLC run is started
<li>MS Survey (low energy) is done
<li>Initial precursor mass list created
<li>Selection criteria applied:  charge state, etc.
<li>Short list of precursors and expected products from neutral loss created
<li>Acquire high energy MS survey
<li>If there is a product due to neutral loss present, switch to MS/MS
<li>If there no product due to neutral loss, then is prodct ion present??
<li>In MS/MS mode of neutral loss products, is there still neutral loss
present?  If yes, acquire MS/MS on identified precursor, then return to
lower energy MS survey.  If no, return to lower energy survey without
doing MS/MS on any precursor.
<li>If a product ion without neutral loss is present, switch to MS/MS
mode on first shortlist precursor:  if no product ion present, switch
to MS/MS mode on next shortlist precursor.  If no product ion present,
acquire MS/MS on identified precursor, then go back to lower energy MS
scans.
</ol>
<p>
Variable flow LC-MS/MS was set up with two 10-port valves of the Stream
Select module and CapLC.  A preconcentration trap was set up with
nano-analytical column.  The trap was 300 &micro;m &times; 5 mm
C<sub>18</sub> and the separation column 75 &micro;m &times; 150 mm
PepMap C<sub>18</sub>.  The spray tip was a PicoTip&trade; capillary
with 10 &micro;m orifice diameter.  A 5 psi nebulizing gas was used.
To run 200 nL/min through the column, a splitter was required
and pump set to 7 &micro;l/min.
<p>
The first 10-port valve was put on a switch that was triggered when
low mass product ions were detected and precursors were interrogated,
and the valve was switched to divert flow to waste and stop flow to
the column.  Two restrictors are placed on the first 10-port which
slow the dissipation of the system backpressure created by the
two column and moving back through the splitter to the two lines on the
valve.  Hence flow in the system is dictated by the rate of pressure
dissipation and also by the continuing potential at the spray tip.
<p>
Five proteins (8 fmol ADH, 140 fmol &beta;-lactoglobulin, 110 fmol
pyruvate kinase, 8 fmol phosphorylase <i>b</i>, and 18 fmol
glyceraldehyde 3-phosphate dehydrogenase/aldolase C) were analyzed
along with 30 fmol of synthetic phosphopeptide TNLSEQ(Yp)ADVYR.
The software was set to look for a phosphotyrosyl immonium ion
of <i>m/z</i> 216.0430 (tolerance &plusmn; 0.020 Da) in the high
energy data set since neutral loss of H<sub>3</sub>PO<sub>4</sub>.
5 possible precursors were then selected for MS/MS analysis, selected
from low energy data set if ion intensity and charge state could produce
the <i>m/z</i> 216 ion.
<p>
The <i>m/z</i> 216.0430 ion was seen at 17.4 min in the chromatogram.
The low energy set was immediately to select candidates.  Each of the
5 ions were selected for 12 sec MS/MS analysis, for 1 min in the cycle.
Since the peak width is typically 20-30 s that is time for only
interrogating 1 or 2 ions.  Reducing flow enables time for interrogation
for more.    The MS/MS spectra reveal that an ion at <i>m/z</i> 769.83
is the true precursor.  Voltage was alternated at 1 second intervals
between 30 and 70 V, necessary to confirm the presence or absence of
the <i>m/z</i> 216.0430 peak.  The odd numbered spectra (1, 3, 5, 7, 9, 11)
are 70 V spectra and an isotope cluster at <i>m/z</i> 216.04 (monoisotopic)
is the only evidence of the phosphtyrosylimmonium ion.  The even-numbered
spectra (2, 4, 6, 8, 10, 12) are the 30 V spectra and report the
<i>b</i><sub>2</sub> sequence ion.  Finally the MS/MS spectrum of
selected precursor <i>m/z</i> 769.83 reveals a series of sequenceable
<i>y</i>&prime;&prime; fragment ions.  10 of 12 mass differences
can be determined and correspond to the sequence, including the
phosphoTyr.
<p>
The important point is the use of variable flow to increase the analytical
time in MS/MS mode.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Choi B-K, Bobrowicz P, Davidson RC, Hamilton SR, Kung DH, Li H, Miele RG,
Nett JH, Wildt S, Gerngross TU
 </span>
 <span class="year">
(2002)
 </span>
 <span class="jname">
Proc. Natl. Acad. Sci. USA
 </span>
 <span class="jvol">
100,
 </span>
 <span class="pages">
5022-5027.
 </span>
<p class="reftitl">
Use of Combinatorial Genetic Libraries to Humanize N-Linked Glycosylation
in the Yeast <i>Pichia pastoris</i>
<p class="Para1">
Recombinant gene expression in fungi have several advantages when the
therapeutic product is a glycoprotein, particularly N-linked.  One problem
in yeast expression is that the N-glycans are of the high mannose (Man) type.
These are immunogenic in humans and thus a problem.  Mammals and fungi
share the first steps of N-linked glycosylation, namely the transfer
of (Glc)<sub>3</sub>-(Man)<sub>9</sub>-(GlcNAc)<sub>2</sub> from the
ER lumen to the nascent polypeptide via an oligosaccharyltransferase.
These are trimmed by glucosidases I and II and a specific &alpha;-1,2-mannosidase
to (Man)<sub>8</sub>-(GlcNAc)<sub>2</sub> (isomer Man8B), and then exit the
ER to the Golgi.  In the human Golgi, &alpha;-1,2-mannosidases remove the
mannose to produce (Man)<sub>5</sub>-(GlcNAc)<sub>2</sub> for its core.
These mannosidases are type II membrane proteins with N-terminal in cytosol
and C-terminal catalytic domain, featuring transmembrane domain and stem
region;  sequences in all parts to the polypeptide but the amino terminus
dictate the enzyme localization.  In <i>S. cerevisiae</i> glycosylation
in contrast involves addition of more Man monomers, the final product
having &gt; 100 Man monomers.  The enzyme &alpha;-1,6-mannosyltrnasferase
(Och1p) has a broad substrate except for the human (Man)<sub>5</sub>-(GlcNAc)<sub>2</sub>
core.  Addition of the first Man monomer by Och1p, additional &alpha;-1,6
mannosidases add more Man in &alpha;-1,6 manner, then &alpha;-1,2 and
&alpha;-1,3 mannosyltransferase in the yeast Golgi continue the hypermannosylation.
In <i>P. pastoris</i> the process is similar though there is much less
mannosylation (also it lacks the &alpha;-1,3 transferase; confirmed by
absence of &alpha;-1,3 branching).  The first steps of humanizing the
glycosylation apparatus involve creating gene knockouts of the &alpha;-1,6
and &alpha;-1,3 transferases.  Subsequent steps are to create fusion
genes that do everything possible to create active enzymes that localize
in their proper position.
<p>
In this study a combinatorial genetic library generating hundreds of
fusion constructs, with a high-throughput screen to analyze large numbers
of strains.  The library is an array of different fusion constructs,
each containing a fungal cellular targeting sequence fused in frame to
a catalytic domain.  Each of the 608 constructs was tested for the ability
to trim high Man structures to (Man)<sub>5</sub>-(GlcNAc)<sub>2</sub>.
Strains able to produce this core were then given a second confirmatory
screen.  They were then engineered to produce GlcNAc-(Man)<sub>5</sub>-(GlcNAc)<sub>2</sub>
in which a &beta;-1,2-<i>N</i>-acetylglucosaminyltransferase I/(GnTI)/leader
library was used.
<p>
<i>P. pastoris</i> GS115 (<i>his4</i>, Invitrogen) or JC308 (<i>ura3,
ade1, arg4, his4</i>, gift of J.M. Cregg).  Medium is 1% yeast extract,
2% peptone, 0.1 M potassium phospate pH 6.0, 1.34% yeast nitrogen base,
0.4 mg/L biotin, and 1% glycerol.  Cells are grown at RT in 96-well plates.
For induction, the glycerol is replaced with 1.5% methanol.  The minimal
medium is 1.4% yeast nitrogen base, 2% dextrose, 1.5% agar, 0.4 mg/L
biotin and amino acids.
<p>
The 1215 bp orf of <i>P. pastoris OCH1</i> codes for a &alpha;-1,6-mannosyltransferase.
Primers based its sequence were used to amplify genomic DNA from
strain X-33 (Invitrogen).  Another library (gift from J. Folkman)
was used to amplify segments 2685 upstream and 1175 bp downstream of the
orf of <i>OCH1</i> gene:  internal oligos of the gene were used with T3
and T7 oligos in the backbone of the library bearing a plasmid
&lambda; ZAP II (Stratagene).  This makes a 5075 bp fragment placed into
pCR2.1-TOPO vector (Invitrogen), called pBK9.  The plasmid pJN329
contains a <i>och1::URA3</i> mutant allele, and 100 &micro;g is digested
with <i>Sfi</i>I, transfected into <i>P. pastoris</i> JC308 by electroporation.
Selection on a uracil-deficient medium is done for 10 d.  1000 colonies
were re-streaked.  <i>URA<sup>+</sup></i> clones unable to grow at 37&deg;
but growing at RT were checked by colony PCR.  One clone YJN153 was
selected.
<p>
DNA encoding the Kringle 3 domain of human plasminogen was amplified with
<i>Pfu</i> turbo polymerase (Stratagene) and clonded into the EcoRI
and <i>Xba</i>I sites of pPICZ&alpha;A (Invitrogen), which creates a
C-terminal 6-His tag.  Pro-46 was replaced with a serine to improve
N-linked glycosylation.  The plasmid (pBK64) was transformed into P.
pastoris strains as a means of monitoring changes in glycosylation
engineering.  In the wild-type P. pastoris with the <i>och1</i> mutant
gene, hyperglycosylation was eliminated on K3 protein, as seen by
westerns:  4-20% SDS-PAGE separated proteins transferred to nitrocellulose
were probed with anti-peptide antibody (Sigma-Genosys) raised against
CQQLSSPKIDYDPLTL, part of <i>P. pastoris</i> Och1p.
<p>
The N-glycans from K3 were removed by reductively alkylating Cys of the
protein and then blocking the membranes. washing wells three times with
water.  30 &micro;l of 10 mM ammonium bicarbonate pH 8.3 with 1 mU
<i>N</i>-glycanase (Glyko) is added and after 16 h at 37&deg; the
solution containing glycans is removed by centrifugation and evaporated
to dryness.  Glycans were resuspended in 15 &micro;l water and spotted
0.5 &micro;l on target plates, mixed with 0.5 &micro;l of super DHB
in 50% MeCN containing 0.1% TFA.  (Man)<sub>5</sub>-(GlcNAc)<sub>2</sub>
was used as external standard.  MALDI-ToF spectra showed a molecular
mass consistent with (Man)<sub>10-16</sub>-(GlcNAc)<sub>2</sub> for
K3 expressed in wild-type.  In K3 in strains with <i>och1</i> mutant,
predominantly (Man)<sub>6-12</sub>(GlcNAc)<sub>2</sub> was found, with
associated loss of smearing in westerns.
<p>
A leader library consisting of N-terminal peptides of type II membrane
proteins which localize in the ER or Golgi of yeast was constructed.
The catalytic domains of &alpha;-1,2-mannosidases from several species
(human to single-cell eukaryotes) formed a second library.  A third
library of the catalytic domains of GnTI genes from a wide variety
of eukaryotes was also prepared.  The combination of constructs was
such that the leader was fused with a catalytic domain.
<p>
The leader itself was of three classes: (1) a short length being the
cytoplasmic tail and transmembrane domain, (2) a long form which added
the complete stem up to the catalytic domain, and (3) a medium form
containing half the stem along with the short form.  The catalytic domain
was obtained from different species to find an active form that could
exist at different pH and temperature.  The best chimeric constructs
were those that had a high degree of Man-trimming activity and
UDP-GlcNAc transfer activity on the K3 reporter.
<p>
608 total chimera were thus screened.  Homologs of <i>C. elegans</i>
had a high degree of trimming activity down to (Man)<sub>5</sub>-(GlcNAc)<sub>2</sub>;
its leader in the fusion was the <i>MNS1</i> or <i>MNN10</i> fragments from
<i>S. cerevisiae</i>.  More than half (56%) of fusions formed only
very low (&lt;10%) amounts of the desired Man core.  The constructs
that worked accounted for 70-80% trimming.  The results show the importance
of picking a good localization sequence and a catalytic domain of proper length.


<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Andon NL, Hollingsworth S, Koller A, Greenland AJ, Yates JR 3rd, Haynes PA
 </span>
 <span class="year">
(2002)
 </span>
 <span class="jname">
Proteomics
 </span>
 <span class="jvol">
2,
 </span>
 <span class="pages">
1156-1168.
 </span>
<p class="reftitl">
Proteomic Characterization of Wheat Amyloplasts
Using Identification of Proteins by Tandem Mass Spectrometry
<p class="abstract">
<b>ABSTRACT</b>.
We describe the initial characterization of the wheat amyloplast proteome, consisting
of the identification and classification of 171 proteins. Whole amyloplasts and purified
amyloplast membranes were prepared from wheat (Triticum aestivum). Protein extracts
were examined by one-dimensional and two-dimensional electrophoresis, followed by
high performance liquid chromatography-tandem mass spectrometry of separated
proteins. Tandem mass spectrometry data of individual peptides was then searched
by SEQUEST, using a database containing known protein sequences from both wheat
and other homologous cereal crops. Using this approach we identified 108 proteins
from whole amyloplasts and 63 proteins from purified amyloplast membranes. The
majority of protein identifications were derived from protein sequences from cereal
crops other than wheat, for which relatively little gene sequence data is available. The
highest percentage of protein identifications obtained from any individual species was
46% of the total number of proteins identified, using sequence data found in our proprietary
rice (Oryza sativa) genome database.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Keller A, Nesvizhskii AI, Kolker E, Aebersold R
 </span>
 <span class="year">
(2002)
 </span>
 <span class="jname">
Anal. Chem.
 </span>
 <span class="jvol">
74,
 </span>
 <span class="pages">
5383-5392.
 </span>
<p class="reftitl">
Empirical Statistical Model To Estimate The Accuracy of Peptide
Identifications Made By MS/MS and Database Search
<p class="abstract">
<b>ABSTRACT</b>.
We present a statistical model to estimate the accuracy
of peptide assignments to tandem mass (MS/MS) spectra
made by database search applications such as SEQUEST.
Employing the expectation maximization algorithm, the
analysis learns to distinguish correct from incorrect
database search results, computing probabilities that
peptide assignments to spectra are correct based upon
database search scores and the number of tryptic termini
of peptides. Using SEQUEST search results for spectra
generated from a sample of known protein components,
we demonstrate that the computed probabilities are
accurate and have high power to discriminate between
correctly and incorrectly assigned peptides. This analysis
makes it possible to filter large volumes of MS/MS
database search results with predictable false identification
error rates and can serve as a common standard by
which the results of different research groups are compared.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Dickinson DP
 </span>
 <span class="year">
(2002)
 </span>
 <span class="jname">
Crit. Rev. Oral Biol. Med.
 </span>
 <span class="jvol">
13,
 </span>
 <span class="pages">
485-508.
 </span>
<p class="reftitl">
Salivary (SD-Type) Cystatins:  Over One Billion Years In the Making&mdash;But
To What Purpose?
<p class="Para1">
<b>REVIEW</b>.  Barrett et al (1986) reported a trypsin inhibitor in egg white
which could inhibit plant proteases like papain and ficin.  The inhibitor was
about 12.7 kDa and potent (<i>K<sub>i</sub></i> = 10 nM).  Binding is a 1:1
reversible complex, although in many cases affinity is so high it is essentially
irreversible.  A 13.3 kDa protein in normal CSF and in the urine of renal failure
patients was identified as cystatin C.  Cystatins are found all throughout
eukaryotic species:  orthologs are in many species, and within many species are
paralogs (occur by duplication).   The proteins has 3 distinct structural features:
an N-terminal glycine (posn 11 in cystatin C), a QXVXG motif in a hairpin loop,
and a PW motif in a second loop in the C-terminus.  All of these come together to
dock with the active site of cysteine proteinases.
<p>
The chicken egg white protein (ortholog is human cystatin C) is a family 2 protein
also found in fish, other birds, and mammals.  These are 120-125 residues long with
two disulfide bonds.  A secretory peptide leader is present.  Both x-ray crystals
and NMR in solution have revealed structure.  It is five-straned &beta;-sheet
wrapped with a five-turn &alpha;-helix.  N-terminal 10 residues are disordered
and flexible.  G11 is in the first short &beta;-strand A, connected to a five-turn
&alpha;-helix 1.  QLVSG (the QxVxG match) is in a hairpin between &beta;-strands
B and C.  A short second &alpha;-helix 2 connects strands C and D; this &alpha;-helix
is not seen in solution by NMR, forming a loop lacking secondary structure.
The &alpha;-helix 2/loop is anchored by disulfide #1.  Conserved PW motif is
in a 2nd hairpin loop between &beta; strands D and E, linked by 2nd disulfide.
All three conserved regions come together to form a wedge that docks to cysteine
proteinase active site.  The RLV (pos 8-10) sequence in human cystetin C interact
with S<sub>4</sub>, S<sub>3</sub>, and S<sub>2</sub> substrate-binding subpockets,
respectively, on the target enzyme.  G11A12 from wedge-shaped edge that binds
to active site cleft of papain (G11 binds S<sub>1</sub> site.  The bond is
too far away for cleavage by the active site.  Both hairpins interact with residues
in the vicinity of the reactive site cysteine.  All three domains act rather
independently.  Binding occurs without induced conformational changes, but the
binding of cathepsin B is a two-step mechanism.  The proteinase has a loop
occluding the active site: there is probably a weak interaction with the N-terminal
region of the cystatin, then a conformational change to displace the occluding
loop, followed by tight binding.
<p>
cDNAs of the three human salivary cystatins S, SA, and SN and a distantly related
cystatin D have been cloned, each about 740 bp long.  Comparing mature protein
to preprotein shows a 20-residue long secretory signal peptide, then an 11-residue
N-terminal extension to the conserved G11.  Cystatins S, SA, and SN have an 88%
identity.  Cystatins SA and SN have a greater similarity to chicken cystatin than
to cystatin S, which has poorer inhibitory properties.
<p>
Cystatins from saliva in other animals have been noted.  Snake venom (its glands
are modified salivary glands) contain family 2 cystatins.  One different from
cystatin C by a 6-residue insertion in the loop 2 helix.  EST seqeuncing of
human amniotic and fetal skin epithelial cDNA libraries revealed a novel human
cystatin E, as well as a down-regulated gene in human breast metastatic tumor
cells.
<p>
Genetic expression diversity is rare in these small, highly conserved family 2
proteins.  Haplotypes A and B differ at three sites in gene <i>CST3</i>.  Two
differences are in promoter region and one in signal peptide (A&rarr;T base change).
there is no difference in the amino acid sequence.  The mutation L68Q causes
an autosomal dominant disease:  hereditary cerebral hemorrhage with amyloidosis,
Icelandic type.  <i>CST2</i> (cystatin SA) has a G&rarr;A transition in exon 2
and A&rarr;T transversion in exon 3 to produce G59D and E120D in SA1 and SA2
proteins.  Substitution at 59 is within the QxVxG sequence: SA1 QIVGG&rarr; SA2
QIVDG. with clearly different inhibitory properties.  A C25R change in <i>CST5</i>
exon 1 is within the &alpha;-helix 1 opposite the inhbitory binding region and the
Ki values remain unchanged.
<p>
About 20% of rat cystatin C is N-glycosylated at a NLT consensus site in the
&alpha;-helix 2/loop region.  Other cystatins have glycosylation sites although
they are though of as nonglycosylated.  The effect of glycosylation is not
understood.
<p>
Within chicken cystatin (not human cystatin C) is a S82 phosphorylation site
in the helix 2/loop.  Human S and SA are partially phosphorylated, but cystatin
SN is not.  Phosphorylation sites at S2, S98, S111, and S114 have been identified.
The SN also has a S98 consensus site.  The S111 is a casein kinase 2 site
[(S/T)xx(D/E/Sp)].  The S2 and S98 sites have a Golgi kinase site [Sx(E/Sp)].
<p>
Other cystatin families include Family 1, the stefins, of which humans have
two, stefins A and B.  Structurally they differ from family 2 inhibitors in that
they lack disulfide bonds and the &alpha;-helix 2/loop, have a kink in
&alpha;-helix 1, and have a 9-residue added C-terminus.  Gene structure
differences include different intron positions and the absence of a secretory
signal sequence.  Although they are thought to be intracellular, evidence shows
secretion in some conditions.  The Family 3 cystatins are the kininogens:
these are a low and high MW form and the T-kininogen and are made up of three
tandem Family 2 cystatin domains followed by a kinin fragment, with different
C termini;  all of them are inhbitors.  Family 4 cystatins are the fetuins,
which are abundant mammalian fetal serum and bone  glycoproteins.  These are
N- and O-glycosylated and phosphorylated.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Cutler P, Heald G, White IR, Ruan J
 </span>
 <span class="year">
(2003)
 </span>
 <span class="jname">
Proteomics
 </span>
 <span class="jvol">
3,
 </span>
 <span class="pages">
392-401.
 </span>
<p class="reftitl">
A novel approach to spot detection for twodimensional gel electrophoresis
images using pixel value collection
<p class="Para1">

<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Lin D, Tabb DL, Yates JR 3rd
 </span>
 <span class="year">
(2003)
 </span>
 <span class="jname">
Biochim. Biophys. Acta
 </span>
 <span class="jvol">
1646,
 </span>
 <span class="pages">
1-10.
 </span>
<p class="reftitl">
Large-Scale Protein Identification Using Mass Spectrometry
<p class="abstract">
<b>ABSTRACT</b>.
Recent achievements in genomics have created an infrastructure of biological information. The enormous success of genomics promptly
induced a subsequent explosion in proteomics technology, the emerging science for systematic study of proteins in complexes, organelles,
and cells. Proteomics is developing powerful technologies to identify proteins, to map proteomes in cells, to quantify the differential
expression of proteins under different states, and to study aspects of proteinprotein interaction. The dynamic nature of protein expression,
protein interactions, and protein modifications requires measurement as a function of time and cellular state. These types of studies require
many measurements and thus high throughput protein identification is essential. This review will discuss aspects of mass spectrometry with
emphasis on methods and applications for large-scale protein identification, a fundamental tool for proteomics.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Hoofnagle AN, Resing KA, Ahn NG
 </span>
 <span class="year">
(2003)
 </span>
 <span class="jname">
Annu. Rev. Biophys. Biomol. Struct.
 </span>
 <span class="jvol">
32,
 </span>
 <span class="pages">
1-25.
 </span>
<p class="reftitl">
Protein Analysis by Hydrogen Exchange Mass Spectrometry
<p class="abstract">
<b>ABSTRACT</b>.
Mass spectrometry has provided a powerful method for monitoring
hydrogen exchange of protein backbone amides with deuterium from solvent. In comparison
to popular NMR approaches, mass spectrometry has the advantages of higher
sensitivity, wider coverage of sequence, and the ability to analyze larger proteins. Proteolytic
fragmentation of proteins following the exchange reaction provides moderate
structural resolution, in some cases enabling measurements from single amides. The
technique has provided new insight into protein-protein and protein-ligand interfaces,
as well as conformational changes during protein folding or denaturation. In addition,
recent studies illustrate the utility of hydrogen exchange mass spectrometry toward
detecting protein motions relevant to allostery, covalent modifications, and enzyme
function.
<!-- ================================================================ -->
<p class="tags">

<p class="ref">
 <span class="authors">
Dowsey AW, Dunn MJ, Yang G-Z
 </span>
 <span class="year">
(2003)
 </span>
 <span class="jname">
Proteomics
 </span>
 <span class="jvol">
3,
 </span>
 <span class="pages">
1567-1596.
 </span>
<p class="reftitl">
The Role of Bioinformatics in Two-Dimensional Gel Electrophoresis
<p class="Para1">
<b>REVIEW</b>.  A single gene might be able to encode as many as six
different protein species, indicating that a study of genomics alone is
insufficient to understand what the science of proteomics brings to knowledge.
Proteomics begins with sample collection, after which samples must be
solubilized, denatured, and the proteins reduced to eliminate structural
features that inhibit further analysis.  2-DE is the only method able to
separate and quantitate as many as 10,000 proteins.  The use of stable isotope
tagging in MS is making this part of proteomics able to quantify proteins
but 2-DE remains the main method to study differential expression.  Staining
of 2-DE primarily involves use of silver to detect minor proteins ("gold
standard"), but Coomassie Brilliant Blue and SYPRO Ruby have their uses.
Imaging of gels is done using 12- to 16-bit grayscale scanners.
<p>
The most intense third of spots seen on a gel account for 75% of the total
protein loaded on to a gel, and the least intense third of spots make up only
about 6% of the mass.  The range between the most and least intense spot is
probably million-fold for cells and tissues, and 10<sup>12</sup> for body
fluids (e.g., plasma).  The effective staining range is 10,000-fold, which means
the least abundant proteins can scarcely be distinguished from noise.
<!-- ============================================================= -->
<p class="tags">
FD
<p class="ref">
 <span class="authors">
Anbalagan V, Patel JN, Niyakorn G, Van Stipdonk MJ
 </span>
 <span class="year">
(2003)
 </span>
 <span class="jname">
Rapid Comm. Mass Spectrom.
 </span>
 <span class="jvol">
17,
 </span>
 <span class="pages">
291-300.
 </span>
<p class="reftitl">
McLafferty-type rearrangement in the collision-induced dissociation of
Li<sup>+</sup>, Na<sup>+</sup> and Ag<sup>+</sup> cationized esters of
N-acetylated peptides
<p class="Para1">
N-acetylated GGG peptide (AcGGG) is esterified at the C-terminal end with
ethyl, propyl, and butyl groups and studied in presence of cations (Li,
Na, Ag).  CID spectra for AcGGGOEt show predominantly is y<sub>2</sub><sup>+</sup>
which occurs when the AcG residue is lost as a oxazolinone.  Ions of type
[y<sub>2</sub>-1+Me]<sup>+</sup> where Me is Li, Na, or Ag metal appear
but at lower intensities.  The Na- and Ag-cationized peptides also fragment
into product ions [b<sub>2</sub>-1+Me]<sup>+</sup> and [a<sub>2</sub>-1+Me]<sup>+</sup>.
All metal forms of the AcGGGOEt produced a water loss ion:
[AcGGGOEt-H<sub>2</sub>O+Me]<sup>+</sup> was the most abundant peak of
Li and Ag cationized ions, and major peak in the Na CID spectrum.  Product
ions were also seen for the methyl esters.  In the unesterified peptide,
a water loss ion would be predicted, but not in esters.  With free C-terminal
-COOH, the mobile proton bonds to the carbonyl O atom on -COOH, making the
carbonyl C nucleophilic to attack by the amide O of the n - 1 residue.  Water
is eliminated with the creation of the charged oxazoloninone ring in what
is called the <b>Retro-Koch</b> reaction.  Alternatively the mobile proton
can bond to a more internal amide O atom, and the proton on the amide N
shifts to the other nonbonding pair on the amide O atom.  Water is eliminated
and a charged triple bond C-N is formed with the charge on the electron-deficient
N atom.  This is known as a <b>Retro-Ritter</b> reaction.  This reaction
requires a proton however, not present in case of metal-cationized peptides.
<p>
Another mechanism is proposed for water loss, one similar to that proposed
in formation of b<sub>n</sub><sup>+</sup> or [b<sub>n</sub>-1+Me]<sup>+</sup>
product ions with cyclization to form  5-member ring, with protein transfer
and 1,2-elimination to remove H<sub>2</sub>O.  Evidence for the mechanism:
(a) D2O is lost from esters with exchangable protium replaced by deuterium
indicating two amide H/D atoms in mechanism, (b) if Pro, which lacks
amide nitrogen proton required for 1,2-elimination, is the following residue
after the N-acetyl terminus, there is no H2O loss.  In the mechanism,
The acetyl group isomerizes with N-terminus nitrogen:
CH<sub>3</sub>-C(=O)-NH- &rarr;
CH<sub>3</sub>-C(-O<sup>&minus;</sup>)=N<sup>+</sup>H-, followed by the
oxyanion attacking the carbonyl C atom of the n+1 residue, which forms
a 5-member ring.  The proton on the ring N atom moves to the n+1
carbonyl O atom (oxyanion) to form the hyxdroxyl.  The n+2 nitrogen
gives a proton up as the -OH on the ring becomes a good leaving group
(1,2-elimination).
<p>
To verify structure of the remaining ion, MS<sup>3</sup> analysis was
done on metal-cationized precursor, with a neutral loss consistent with
alkene elimination of 28, 42, and 56 Th, depending on whether the ester
was ethyl, n-propyl, and n-butyl (a McLafferty-type rearrangement).
Additional CID in MS<sup>4</sup> resulted in 46 Th loss consistent
with H2O and CO loss of the C-terminus, or loss of residue of C-terminal
acid from tri- and tetrapeptide esters by a reaction which generates
[b<sub>n</sub>+17+Me]<sup>+</sup> from peptides with unesterified
COOH.  The loss of H2O is neither the carbonyl nor the ester O atoms
on the C-terminus; combined with Pro being the 2nd atom and the absence
of H<sub>2</sub>O loss, then the N-terminal amide carbonyl O is eliminated as
indicated.  Additional experiments (isotope labeling) should verify
this mechanism.
<p>
A 28 mass unit loss in peptide dissociation is usually CO from the
b<sub>n</sub><sup>+</sup> or analogous [b<sub>n</sub>-1+Me]<sup>+</sup> product ion representing substituted
oxazolinones to produce a<sub>n</sub><sup>+</sup> or
[a<sub>n</sub>-1+Me]<sup>+</sup> ions.  The reaction results in peptide
bond cleavage in which the C-terminal side takes the neutral peptide.
CO elimination from cationized oxazolinone is an exothermic process.
Direct 28 amu elimination in the peptide ethyl ester without
<i>b</i> ion (metal-ionized or not) can only be explained by the
McLafferty-type rearrangement, used to explain odd-electron ions from
electron impact, with cyclization to form 6-member ring, then transfer
of &gamma;-C hydrogen to carbonyl O atom, followed by alkene elimination
of the ester.  Ethylene is a 28 Da neutral species that could be eliminated.
Methyl ester of AcGGG in Li<sup>+</sup> and shows H2O elimination without
any Mclafferty since no transferable H atom is on a &gamma;-carbon.
With isopropyl and tert-butyl esters, losses of 42 and 56 amu represent
propylene and isobutylene, respectively, consistent with McLafferty;
the ion products are both m/z 238.  And the relative intensities of the
branched esters were higher since the formation of their alkenes has
a lower energy barrier.  The nature of the metal did not matter, since
Na<sup>+</sup> and Ag+ gave the same result.  Expecting the product
ions to be acetylated metal-cationized having COOH C-terminals, MS<sup>3</sup>
spectra of rearrangement products were compared with MS/MS of the
metal-cationized peptides.  The spectra are similar with generation of
sequence ions (e.g., [b<sub>2</sub>+17+Li]<sup>+</sup> and
[b<sub>2</sub>-1+Li]<sup>+</sup>) to support a mechanism where the free
COOH is produced from the esters by McLafferty rearrangment.
<p>
This rearrangement requires H atom transfer from &gamma;-position.
To verify 2,2,2-<i>d</i><sub>3</sub>-EtOH was used to esterify
CH<sub>3</sub>CO-peptides.  If McLafferty-type rearrangement is true,
the product ion should be 1 amu higher as it takes up a D atom, and
the neutral loss will be 30 amu (not 28) as it takes two D atoms in the
alkene.  This was found in the spectrum.  Intensities of fragment ions
in CID spectra are function of precursor ion internal energy and rate
of dissociation reactions.  The McLafferty depends on the 6-member ring
and branched alkyl esters (isopropyl, tert-butyl) offer more H atoms
in the &gamma;-carbon.  This is shown also.
<p>
Dissociation profiles, which are plots of the fraction of ion abundance
against the corrected activation amplitude (in arbitrary units) in an
ion trap, show the energy needed to produce the dissociation.
Different metal-cationized (Li, Na, Ag) forms of the acetylated peptides
APF and GGGG had their profiles determined.  For ethyl esters there is
a decrease in precursor ion intensity and an increase in McLafferty-type
rearrangment ions and Li and Na require higher corrected activation
amplitudes than did Ag.  The Na cationzed peptides had slightly
lower activation amplitides than did Li-cationized peptides.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Wang H, Hanash S
 </span>
 <span class="year">
(2003)
 </span>
 <span class="jname">
J. Chromatog. B
 </span>
 <span class="jvol">
787,
 </span>
 <span class="pages">
11-18.
 </span>
<p class="reftitl">
Multi-Dimensional Liquid Phase Based Separations in Proteomics
<p class="abstract">
<b>REVIEW</b>.
<p>
<b>ABSTRACT</b>.
This review covers recent developments towards the implementation of multi-dimensional (MuD) liquid phase based
systems for proteome investigations. Although two-dimensional polyacrylamide gel electrophoresis (2D-PAGE) has been
used as a standard approach in proteomics, its drawbacks including the limited dynamic range and molecular mass range,
together with lack of on-line integration with biological mass spectrometery (Bio-MS) have limited its widespread use and
applications in proteomics. In the meantime, various liquid-phase based multi-dimensional separation techniques have been
explored. Especially, with the emergence of the combination of nanoflow capillary high-performance liquid chromatography
(cHPLC) and Bio-MS, attention is again refocused on utilizing multi-dimensional liquid-phase based separation of proteins.
Some remarkable applications of on-line analysis of intact proteins and on-column digested proteins, and the emergence of
approaches such as multiple HPLCelectrospray ionization tandem MS and capillary array electrophoresis-matrix assisted
laser desorption ionization MS, have stimulated thinking towards developing a automated multi-dimensional system
(MuDSy) that integrates liquid phase based separation, digestion and identification of proteins in complex biological
mixtures.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Luche S, Santoni V, Rabilloud T
 </span>
 <span class="year">
(2003)
 </span>
 <span class="jname">
Proteomics
 </span>
 <span class="jvol">
3,
 </span>
 <span class="pages">
249-253.
 </span>
<p class="reftitl">
Evaluation of Nonionic and Zwitterionic Detergents As Membrane Protein
Solubilizers in Two-Dimensional Electrophoresis
<p class="Para1">
Methods in the past that have done much to reveal membrane-embedded
proteins including SDS-PAGE and strategies such as MudPIT (MDLC-MS/MS), but
these methods have some disadvantages such as not being quantitative.
Use of thiourea as a chaotrope and the development of zwitterionics
to extract proteins has enabled better analysis.  Electrically neutral
detergents do not repel each other as SDS does, and this leads to poorer
solubility.  Many experimental detergents are not commercially available
now.  So several nonionic, commercially available detergents were
evaluated for their efficiency in membrane protein solubilization.
<p>
<i>Arabidopsis thaliana</i> leaf membranes were incubated in Triton X-100
with mixing 1:1 such that detergent concentration was 0.2% w/v,
at 4&deg; for 10 min with gentle shaking.  The preparation was centrifuged
at 120,000 <i>g</i> for 15 min.  The pellet was resuspended in the
storage buffer and centrifuged again at the same speed and time.  The
final membrane pellets were then solubilized in an extraction medium
of 7 M urea, 2 M thiourea, 20 mM DTT, 0.4-1% carrier ampholytes, and 2-4%
detergent.  Zwitterionics included tetradecanoyl propyl dimethyl ammonio
propane sulfonate (ASB14), 3-(4-heptyl)phenyl-3-hydroxylpropyldimethyl
ammoniopropanesulfonate (C7BzO) that were synthesized previously.
Glucopon and Plantacare were from Fluka/Sigma-Aldrich.  Glycolipid
detergents include dodecyl and tetrdecyl mellbionamide.  At least 98%
pure alkyl maltosides both dodecyl and tetradecyl were also available.
<p>
Protein amounts in sample were kept constant.  Membranes were solubilized
and centrifuged 250,000 <i>g</i> 30 min and supernatant directly focused.
IEF was by home-made gradients from pH 4-8 and commercial 3-10 strips.
Strip rehydration was on 3%T overnight in 7 M urea, 2 M urea, 20 mM DTT,
and 0.4% Pharmalytes 3-10.  Detergents were 2% although CHAPS was 4%.
Focusing was from 0-300 V in 1 min, kept at 3 hours, then ramped to 3500 V
over 60 min, and then kept at that for 20 h.  For the 2nd dimension
strips are incubated 10 min RT in 6 M urea, 30% w/v glycerol, 2% w/v SDS
13 mM DTT, and 0.125 M Tris 0.1 M HCl.  An alkylating solution leaves
out DTT and uses 2.5% iodoacetamide.  The 2nd dimension is 10% T without
a stack.
<p>
Band III protein in RBC membranes is easily identifiable (by MS) and the
effectiveness of its extraction was the basis of comparison.  CHAPS
alone was not effective as no band III was detectable.  The band for this
membrane is not normally sharp probably because of many modifications
(glycosylation, palmitylation).  Tween family detergents were thoroughly
ineffective (no band III was seen with polyoxyethylene sorbitan
monopalmitate (Tween 40) nor Tween 20 or 80.  Maltoside detergents were
good (&alpha; and &beta;-dodecyl maltosides).  Tetradecyl maltoside
and 6-cyclohexyl-hexyl-&beta;-maltoside (Cymal-6) also gave good results.
However these detergents are expensive.  Cheaper substitutes like
Glucopon and Plantacare are mixtures of alkyl glucosides and maltosides
but these were ineffective.  The dodecyl and tetradecyl mellbionamides
were also ineffective.
<p>
Some detergents have an oligoethylene glycol polar head and a linear
alkyl tail.  Triton X-100 has such a polar head but its hydrophobic
portion is tert-octylphenyl.  Tetraethylene glycol monododecyl ether
(Brij 30) did not solublized Band III, and docosaethylene glycol
monohexadecyl ether (Brij 58) and docosaethylene glycol octadecyl ether
(Brij 78) did little to solubilze and gave poor focusing.  Decaethylene
glycol monooleylether (Brij 96) solubilized a limited amount of protein
and gave better focusing results.  Brij 56, TX-100, and decaethylene glycol
monotridecyl ether (C13E10) were as good as the zwitterionics and glycosidic
detergents.
<p>
As a control, <i>A thaliana</i> membranes, rich in aquaporins and
H<sup>+</sup>-ATPase, were compared as well.  ASB14 is superior for
C7BzO for band III extraction, but the opposite was true for plant
membrane preparation.  With respect to the nonionic detergents, Brij 78
was ineffective (as for RBC membranes) while Brij 56 was intermediate
between ASB14 and C7BzO.  Dodecyl maltoside was the most efficient in this
panel however.  Thus efficency of membrane protein solubilization is
a factor of the membrane protein structure itself, the lipid content of the
membrane, and the method used to prepare the membrane before final
solubilization.  Triton X-100 can partially delipidate plant membranes
but have no effect on RBC membranes.
<p>
It may be necessary to try a panel of nonionic and zwitterionics as part
of the process of selecting the best tool.  Saccharide-derivative detergents
should also be considered despite their costs.  However there is
batch-to-batch variability, and the detergents of highest purity should be
used in this case.
<p>
One factor to consider is the hydrophilic-lipophilic balance (HLB).
Detergents at the extremes with a low HLB (Brij 30, HLB=9.7) or high
HLB (Brij 58, HLB=15.7; Brij 78, HLB=15.3; Tween 40, HLB=15.6) are
inefficient, whereas those with intermediate HLB (Brij 56, HLB=12.9;
Triton X-100, HLB=13.5; C13E10, HLB=14.1) were better.  Brij 96 (HLB=12.4)
was an exception with its oleyl hydrophobic moiety.  Usually these
principles hold true for extraction in the presence of denaturing chaotropes
(urea-thiourea mixtures).  Brij 30 and Brij 96 were more efficient
under non-denaturing conditions however.   Triton X-100 is better than
CHAPS at solubilizing membranes in urea-thiourea mixtures.
<!-- ================================================================ -->
<p class="tags">

<p class="ref">
 <span class="authors">
Bruschi M, Musante L, Candiano G, Ghiggeri GM, Herbert B, Antonucci F,
Righetti PG
 </span>
 <span class="year">
(2003)
 </span>
 <span class="jname">
Proteomics
 </span>
 <span class="jvol">
3,
 </span>
 <span class="pages">
821-825.
 </span>
<p class="reftitl">
Soft Immobilized pH Gradient Gels in Proteome Analysis: A Follow-Up
<p class="abstract">
<b>ABSTRACT</b>.
As a follow-up of a previous work on two-dimensional map analysis
utilizing soft (&lt; 4%T) immobilized pH gradient (IPG) matrices in
the first dimension (Candiano et al., Electrophoresis 2002, 23, 292297),
we have further optimized the preparation of such
dilute IPG gels. One important step for obtaining an even reswelling
of the entire IPG strip along the pH 3-10 interval is a washing step
in 100 mM citric acid. It appears as
though after rinsing off the excess acid in distilled water, a gradient of this tricarboxylic
acid remains trapped into the IPG matrix, from almost nil at the acidic gel region to
substantially higher amounts in its basic counterpart. This gradient helps in obtaining
a uniform reswelling of the IPG strip, since carboxyl groups are more heavily hydrated
than amino groups. The combined effects of uniform reswelling and of diluting the gel
matrix favor penetration of large macromolecules (&gt; 200 kDa) and allow for better
spot resolution and for the display of a substantially higher number of spots also in
the 3060 000 Da region. A delipidation step in tri-n-butylphosphate:acetone:methanol
(1:12:1) appears to substantially improve spot focusing and greatly diminish streaking
and smearing of spots in all regions of the pH gradient.
<!-- ================================================================ -->
<p class="tags">

<p class="ref">
 <span class="authors">
Herbert B, Hopwood F, Oxley D, McCarthy J, Laver M, Grinyer J, Goodall A,
Williams K, Castagna A, Righetti PG
 </span>
 <span class="year">
(2003)
 </span>
 <span class="jname">
Proteomics
 </span>
 <span class="jvol">
3,
 </span>
 <span class="pages">
826-831.
 </span>
<p class="reftitl">
&beta;-Elimination:  An Unexpected Artefact in Proteome Analysis
<p class="abstract">
<b>ABSTRACT</b>.
Two persistent myths, ingrained in the electrophoretic literature of the last thirty years,
namely carbamylation and deamidation, have been recently dispelled (Herbert et al., J.
Proteome Res. 2002, in press). We report here, for the first time, a noxious and unexpected
artefact in proteome -elimination (or desulfuration), which results on
the loss of an H2S group (34 Da) from cysteine (Cys) residues for protein focusing in the
alkaline pH region. With such an elimination event, a dehydro alanine residue is generated
at the Cys site. In turn, the presence of a double bond in this position elicits lysis of
the peptide bond, generating a number of peptides of fairly large size from an intact
protein. The first process seems to be favored by the electric field, probably due to
the continuous harvesting of the  anion produced. The only remedy found to this
noxious degradation pathway is the reduction and alkylation of all Cys residues prior
to their exposure to the electric field. Alkylation appears to substantially reduce
both-elimination and the subsequent amido bond lysis.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Grant BD, Wilkinson HA
 </span>
 <span class="year">
(2003)
 </span>
 <span class="jname">
Curr. Opin. Cell Biol.
 </span>
 <span class="jvol">
15,
 </span>
 <span class="pages">
206-212.
 </span>
<p class="reftitl">
Functional Genomic Maps in <i>Caenorhabditis elegans</i>
<p class="Para1">
The nematode <i>C. elegans</i> is the first metazoan whose genomic sequence
is complete.  Half of the genes have been characterized in a large-scale
EST project.  A high percentage of genes have counterparts in humans.
Gene expression is being studied by DNA microarray, yeast two-hybrid,
and RNA-mediated interference (RNAi).
<p>
Microarrays containing 1-2 kb genomic sequence covering coding regions
from about 94% of known or predicted genes are used for profiling.
Two RNA samples are then compared as a ratio of a cy3-to-cy5 labeled
template.  Hill has developed a microarray of ologo containing 98%
of known or predicted ORFs.  Parallel chips are hybridized with different
RNA samples and absolute levels of gene exression.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Hamdan M, Righetti PG
 </span>
 <span class="year">
(2003)
 </span>
 <span class="jname">
Mass Spec. Rev.
 </span>
 <span class="jvol">
21,
 </span>
 <span class="pages">
287-302.
 </span>
<p class="reftitl">
Modern Strategies for Protein Quantification in Proteome Analysis:
Advantages and Limitations
<p class="Para1">
<b>REVIEW</b>. Advances in MS and development of protein/DNA databases
and their search algorithms has enabled global identification and quantification
of proteins.  Two strategies in global analysis are 2DGE combined with
MS analysis.  Gygi et al and Washburn et al. have recently reported the
value of multidimensional chromatography of protein digest complex mixtures
combined with MS analysis.  Although both approaches have strong and
weak points, there are common characteristics.  Proteins should be soluble
(dissolved), denatured, and alkylated...a very critical step.  For
quantification, either gel spot analysis (with a non-saturable stain)
or use of ICATs is possible.  Other tagging methods include esterification
of Glu and Asp using deuterated forms of MeOH have been used (see
Goodlett et al.).  Derivatization of Lys to homoArg has been proposed by
several particularly as it increases peak intensities for MALDI-MS.
Cagney and Emili have taken the approach of modifying the C-terminal
Lys of tryptic peptides by differential guanidination (a process they
call <q>mass-coded abundance tagging</q> (MCAT).  Acrylamide
labeling of Cys has also been used with isotope labeling to make
control-treated quantitative comparisons.
<p>
With respect to sample preparation, there are several steps to be
considered.  Complete alkylation is essential.  There are many competing
processes for the nucleophilic leaving group iodine atom in the alkylating
reagent iodoacetamide (IAA).  Thiourea can react as much as ionized
sulfhydryls on Cys in proteins at alkaline pH.  Hence IAA is rapidly
depleted by thiourea with formation of an adduct that deamidates to form
cyclic thiazolinidone monoimine:
<p class="eqnserif">
  ICH<sub>2</sub>C=ONH<sub>2</sub> + S=C(NH<sub>2</sub>)<sub>2</sub> &rarr;
  (NH<sub>2</sub>)<sub>2</sub>C-S-CH<sub>2</sub>C(=O)NH<sub>2</sub> &rarr;
  [-NH-C(-NH<sub>2</sub>)-S-CH<sub>2</sub>-C(=O)-] cyclic + NH<sub>3</sub>
<p>
Depletion of IAA in a solubilization medium containing thiourea was complete
in &lt; 5 min.  Acrylamide might be added as a scavenger, but then there
was the problem of substantially increased alkylation time (&gt; 6 h).
Studies with the alkylation of lactalbumin (with 8 Cys residues and a mass
of 14186 Da) show quite variable results using alkylators such as
acrylamide and 2- and 4-vinylpyridine.
<p>
Surfactants (detergents) are usually present at 2% concentration
which help to solubilize the protein and make it amenable to better
alkylation as well.  The influence of SDS and CHAPS in the alkylation
of reduced &alpha;-lactalbumin (at 50 pmol/&micro;l) was tested in a
1 h incubation of 260 mM IAA at pH 8.8 with 6 M urea present, and with
either 2% SDS or CHAPS.  SDS has a substantial negative influence.
Small angle neutron scattering studies indicate that a <q>protein-decorated
micelle</q> is formed with the polypeptide effectively buried in the
micelle and short (4-5 residues) outside the micelle and reactive to
an alkylator.
<p>
Quantifying protein expression involves several possible strategies.
Gygi et al. used light and heavy isotopic reagents to modify Cys-containing
proteins, then combined them and proteolyzed, and then used affinity
chromatography followed by MS.  The ICAT has a thiol-reactive group
to alkylate Cys, with a polyether link permitting replacement of eight
H atoms with D atoms, and a biotin function to permit enrichment on
an avidin affinity column.  The ICAT group is substantial in size
however (570.5 Da) and can overwhelm short peptides with collision-induced
dissociation within the ion source itself.  Furthermore complete alkylation
is a problem with ICATs since the function is of the iodoacetamide type,
which rarely achieves 80% blocking within intact proteins.  Other considerations
of ICAT-labeled peptides is an isotope effect seen with RP chromatography:
deuterated peptides elute earlier than non-deuterated species, which can
lead to misleading isotope ratio results critical to quantitation.
This effect is more pronounced with the smaller peptides.  The separations
are not just in chromatography:  isotope effects are more pronounced
with micellar electrokinetic capillary electrophoresis of small tagged
organic molecules.
<p>
MALDI ionization is more efficient with Arg than Lys residues (as reported
by Krause et al 1999).  Arg-containing peptides in one study showed 4-18-fold
greater intensity in positive ion mode than Lys-containing peptides.
This knowledge has spurred the development of derivatization procedures
to permit de novo sequencing of Lys-terminated peptides by guanidination
of the &epsilon;-amine.  Optimization of methods have focused on reaction
time and pH.  Too high a pH resulted in hydrolysis products, and slightly
lower pH brought more desirable results.  Cagney and Emili have attempted
to derivatize the terminal Lys with an MCAT reagent that adds 42 Da
rather than the 8 Da with ICAT, permitting better discrimination.
The guanidination reaction reaches near completion in 1 h at pH 10/37&deg;
and is specific for Lys side chain.  It can be of help in identifying
<i>y</i> ions in MS/MS spectra:  fragmentation of ions before and after
modification would be seen in the <i>y</i> ion as a 42 Da shift.
<p>
Other means of quantifying proteins include stable isotope labeling
for protein profiling.  Groups have used <sup>2</sup>H, <sup>13</sup>C,
<sup>15</sup>N, <sup>18</sup>O, etc. in growth medium, then pooled protein
from labeled and unlabeled and enriched on RP HPLC and SDS-PAGE and analyzed
by MS.  Yao et al have used <sup>18</sup>O-labeled H<sub>2</sub>O in
tryptic digestion to label the peptides during hydrolysis.  Others
have permethylated carboxylic acid groups with either <i>d</i><sub>0</sub>-methanol
or <i>d</i><sub>3</sub>-methanol; ratios are obtained by normalization
of the areas under the curves for the same charge states associated with
the <i>d</i><sub>0</sub>- and <i>d</i><sub>3</sub>-methanol.
<p>
<q>Global internal standard strategy</q> or GIST is the description
for the labeling strategies above.  The idea is to get all peptides
labeled independent of some structural feature which may not be present
on all of them, such as depending on Cys for ICAT labeling.  In a first
try at this two groups showed that acylation of primary amino groups
would be tried with <i>N</i>-acetoxysuccinimide or
<i>N</i>-acetoxy-[<sup>2</sup>H<sub>3</sub>]succinimide.  Reagents are
added to experimental or controls, respectively, after extensive tryptic
digestion.  If the peptide had a C-terminal Arg, its mass increases by
42 Da.  Peptides with a free amino terminus and C-terminal Lys would be
acetylated twice (increase by 84 amu).
<p>
These experiments in labeling and mixing the variably treated samples
probably owes its origins to a study of mRNA expression in which
mRNA was labeled by different fluorescent dyes and then mixed, and individual
species isolated by hybridization, with the relative degree of the change
in mRNA concentration determined by the fluorescence ratio.
Differential in-gel electrophoresis (DIGE) is a direct extension of this
technique used for proteins.  Samples are labeled with mass- and charged-matched
fluorophores Cy3 and Cy5, then mixed and run on a 2D gel.  Because the dyes
are charge- and mass-matched, proteins labeled with them should migrate to
the same spot on the pI axis, but &gt;95% of labeled protein is shifted
from its unlabeled protein (particularly at lower masses).  Matched spots
expressed as a violet color generally are in a 1:1 ratio.  Cy5-dominant
spots produce redder hues, while Cy3-dominant spots show bluer hues.
Unlike attempts to match spots between different gels, no spot matching is
required since one gel allows complete quantitative analysis.  A major
disadvantage is the high cost of equipment, software, and reagents.  In
general, the process should be repeated 3-5 times to provide a proper
statistical evaluation.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Shaw J, Rowlinson R, Nickson J, Stone T, Sweet A, Williams K, Tonge R
 </span>
 <span class="year">
(2003)
 </span>
 <span class="jname">
Proteomics
 </span>
 <span class="jvol">
3,
 </span>
 <span class="pages">
1181-1195.
 </span>
<p class="reftitl">
Evaluation of Saturation Labelling Two-Dimensional Difference Gel
Electrophoresis Fluorescent Dyes
<p class="abstract">
<b>ABSTRACT</b>.
Two-dimensional difference gel electrophoresis (2-D DIGE) enables an
increased confidence in detection of protein differences. However,
due to the nature of the minimal labelling where only approximately
5% of a given protein is labelled, spots cannot be
directly excised for mass spectrometry (MS) analysis and detection
sensitivity could be further enhanced. Amersham Biosciences have
developed a second set of CyDye DIGE Cy3 and Cy5 dyes, which aim
to overcome these limitations through saturation-labelling of
cysteine residues. The dyes were evaluated in relation to their sensitivity
and dynamic range, their useability as multiplexing reagents and
the possibility of direct spot picking from saturation-labelled
gels for MS analysis. The saturation-labelling dyes were superior
in sensitivity to their minimal-labelling counterparts, silver stain
and Sypro Ruby, however, the resulting 2-D spot pattern was
significantly altered from that of unlabelled or minimal-labelled
protein. The dyes were found to be useful as multiplexing reagents
although preferential labelling of proteins with one dye over
another was observed but was controlled for through experimental
design. Protein identities were successfully obtained from material
directly excised from saturationlabelled
gels eliminating the need for post-stained preparative gels.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Rappsilber J, Ishihama Y, Mann M
 </span>
 <span class="year">
(2003)
 </span>
 <span class="jname">
Anal. Chem.
 </span>
 <span class="jvol">
75,
 </span>
 <span class="pages">
663-670.
 </span>
<p class="reftitl">
Stop and Go Extraction Tips for Matrix-Assisted
Laser Desorption/Ionization, Nanoelectrospray,
and LC/MS Sample Pretreatment in Proteomics
<p class="abstract">
<b>ABSTRACT</b>.
Proteomics is critically dependent on optimal sample
preparation. Particularly, the interface between protein
digestion and mass spectrometric analysis has a large
influence on the overall quality and sensitivity of the
analysis. We here describe a novel procedure in which a
very small disk of beads embedded in a Teflon meshwork
is placed as a microcolumn into pipet tips. Termed Stage,
for STop And Go Extraction, the procedure has been
implemented with commercially available material (C18
Empore Disks (3M, Minneapolis, MN)) as frit and separation
material. The disk is introduced in a simple and
fast process yielding a convenient and completely reliable
procedure for the production of self-packed microcolumns
in pipet tips. It is held in place free of obstacles solely by
the narrowing tip, ensuring optimized loading and elution
of analytes. Five disks are conveniently placed in 1 min,
adding <0.1 cent in material costs to the price of each
tip. The system allows fast loading with low backpressure
(&gt;300 L/min for the packed column using manual force)
while eliminating the possibility of blocking. The loading
capacity of C18-StageTips (column bed: 0.4mmdiameter,
0.5 mm length) is 2-4 g of protein digest, which can be
increased by using larger diameter or stacked disks. Five
femtomole of tryptic BSA digest could be recovered
quantitatively. We have found that the Stage system is wellsuited
as a universal sample preparation system for proteomics.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Finehout EJ, Lee KH
 </span>
 <span class="year">
(2003)
 </span>
 <span class="jname">
Electrophoresis
 </span>
 <span class="jvol">
24,
 </span>
 <span class="pages">
3508-3516.
 </span>
<p class="reftitl">
Comparison of Automated In-Gel Digest Methods for Femtomole Level Samples
<p class="abstract">
<b>ABSTRACT</b>.
A comparison of automated in-gel digestion methods for low picomolar to femtomolar
levels of protein is presented. Gel spots with 4 pmol to 120 fmol of protein were stained
with either Coomassie colloidal blue or SYPRO Ruby and digested using an automated
platform. The sequence coverages and average peak intensities obtained from a
matrix-assisted laser desorption/ionization-mass spectrometry (MALDI-MS) analysis
are compared. Results show that methods using an acetonitrile extraction or digest
times greater than the standard 4 h give no significant increase in peptide sequence
coverage for automated digestion of low protein level samples. It is also shown that
digests from SYPRO Ruby-stained gels show a greater improvement upon ZipTip
cleanup than digests from Coomassie colloidal blue-stained gels. The digests from
SYPRO Ruby-stained gels are also shown to give a higher average peptide intensity
if a method with minimal gel plug washing is used.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Bodnar WM, Blackburn RK, Krise JM, Moseley MA
 </span>
 <span class="year">
(2003)
 </span>
 <span class="jname">
J. Am. Soc. Mass Spectrom.
 </span>
 <span class="jvol">
14,
 </span>
 <span class="pages">
971-979.
 </span>
<p class="reftitl">
Exploiting the Complementary Nature of LC/MALDI/MS/MS and LC/ESI/MS/MS
for Increased Proteome Coverage
<p class="Para1">
Most significant figure of merit in proteome analysis is proteome coverage.
A proteolytic digest of a cell lysate might have several hundred thousand
different peptides.  1000 precursors per hour can be identified.
For this work analytical grade reagents/chemicals were from Sigma,
HPLC grade solvents from Burdick &amp; Jackson, and Promega trypsin
was used.  Ribosomes from mitochondria were prepared from a method;
a 1:10 protein:trypsin by weight was used in the digest, with DTT/iodoacetamide
treatment before digestion.  An UltiMate system featuring FAMOS micro-
autosampler and SWITCHOS valves.  A 1 mm precolumn cartridge was used
for sample preconcentration and desalting, and a 180 &micro;m id &times;
15 cm PepMap C<sub>18</sub> column for separation was used.  On
the elution end a Probot system features a coaxially mix 7 mg/ml
CHCA solution and deposited on MALDI target.  Flow at 1.5 &micro;l/min
meant that each spot was a 1 min fraction over a 180 min RP gradient
of 5-40% MeCN.  The Q-ToF Ultima API from Micromass was used., and
up to 8 precursors were examined from each MS scan.  Selection of precurors
was a function of ion intensity, charge state, and dynamic exclusion
(was the precursor previously selected?).  Collision energy setting
was a function of charge state and <i>m/z</i> value.  The mass range
was 700-3500.  Scan combinations were a rolling average, and scans
considered complete when any peak inside the 1000-2500 range obtained
an intensity greater than a user-defined threshold.  Variable collision
energies (up to 5) could be set to determine an optimum from a profile:
MS/MS acquisitions were stopped when either the end of the precursor
list was reached, or the sample depleted, or the time for acquisition
expired.
<p>
The LC program involved a split flow to the MALDI spotting plate
or to the online ESI-MS.  39S bovine mitochondrial ribosomes were
digested and separated in 3 h gradient with 20% flow to ESI and
80% to the MALDI target.  Better intensities were obtained from
MALDI than ESI (base peak intensity ~5 &times; greater).  Differences
in abundance are attributable to ionization process and experimental
method.  MALDI MS keeps MS scans to a minimum so that more sample
is available for MS/MS; e.g. 260 scans (22 min) were spent on 15
precursors selected from 2 scans (10 s).  Matrix ion clusters
adducts formed during ionization contribute to background and can
lead to ion suppression in MALDI-MS.  As for the ribsome analysis,
a selected 3 min chromatogram section showed that
3 peptides were seen in ESI-MS not in the MALDI analysis, 6 peptides
seen in MALDI not in ESI, and 16 peptides were seen in both methods.
<p>
MS/MS spectra for two precursors were selected for MALDI/ESI
comparison.  Peptide mass error was &le; 0.03 Da for each case,
and MASCOT scores were &ge; 95% confidence.  z=2 ESI and z=1
MALDI peptides differed in fragmentation patterns and ion intensities.
The <i>z</i> &le; 2 peptides give cleaner spectra with fewer
internal fragments, but less structural info in some cases.
Degree of fragmentation is a factor of sequence, but collision
energy profiling overcomes difficulty.  With MALDI Q-ToF Ultima,
increasing collision energy for some peptide ions reduces precursor
signal without a concomitant increase in fragment ion signal.
<p>
An interesting finding in the MASCOT search shows that version
1.7 and 1.8 yielded improvements in the score in the later version
which allowed instrument type settings (MALDI/TOF/TOF was set in
1.8).  This indicates a higher degree of fragmentation for singly
charged peptides than expected.  A significant amount of internal
fragments and neutral losses (<i>a, y*</i>, and <i>b*</i> type
ions) are seen.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Laugesen S, Roepstorff P
 </span>
 <span class="year">
(2003)
 </span>
 <span class="jname">
J. Am. Soc. Mass Spectrom.
 </span>
 <span class="jvol">
14,
 </span>
 <span class="pages">
992-1002.
 </span>
<p class="reftitl">
Combination of Two Matrices Results in Improved Performance of MALDI MS
for Peptide Mass Mapping and Protein Analysis
<p class="Para1">
Karas and Hillenkamp embedded tryptophan and nicotinic acid in a UV-absorbing
matrix while Tanaka et al. tried glycerol and finely divided powder in
the first experiments to develop MALDI.  This led to changes in the laser
wavelength, discovery of new matrices and sample prep methods.
<p>
The use of TFA or FA has significant effects on the spectrum.  The
best results (S/N, reproducibility, and reduced ion suppression)
are seen with CHCA is prepared with FA, and DHB with TFA.  Inspection
of the dried droplets shows that CHCA forms small but unevenly sized
crystals with larger crystals and lower densities at the center.
DHB forms large crystal needles pointing inward from the outer edge.
Combinations of the matrices showed better crystal distribution, with
needle-like crystals from the rim of shorter length and evenly distributed
CHCA toward the center.  Different ratios (CHCA:DBH 2:1) resulted in
one crystal type favored over another.  Despite surface heterogeneity
analyte formation occurred although ion intensity varied depending upon
target.  DHB matrix use typically resulted in sweet spot searching
along the rim.  Matrix mixes typically provided intense signals
over the entire well surface.  Examination of the <i>m/z</i> 0-500
range was done to check for matrix-related ions.  Laser fluence
required to get ions from the rim was twice needed for that in the center.
<p>
BSA, ADH, and lactoglobulin were run on 1-D gel, Coomassie-stained,
in-gel trypsin-digested, and extracted peptides set up in matrix and mixes.
150 fmol (in 0.5 &micro;l) was mixed with 0.5 &micro;l matrix and
applied to target.  For all peptides more peptides could be seen with
matrix mixture than with matrixes alone.  CHCA produced more chemical
noise in the low mass region (500-1500), while DHB had fewer matrix
peaks and better S/N but was more prone to sweet-spotting.  With the
matrix mix, peaks &gt; <i>m/z</i> 1000 generally show a 2- to 3-fold
increase in S/N while those lower show an ion suppression.  The
<i>m/z</i> 1907.9 showed a better resolution with matrix mix and
CHCA (9200) than did DHB (5600);  below <i>m/z</i> 1500 no clear tendency
is seen.
<p>
Ten faintly silver stained spots from a 2-D gel of the barley grain
proteome were selected, in-gel digested, and 10% of volume of extracted
peptides used for MALDI.  Sequence coverage was then compared for
the dried droplet using CHCA, DHB, and matrix mix.  In all cases,
the matrix mix was equal to or greater than CHCA or DHB alone.
In one high coverage protein, CHCA and DHB are missing peptides found
in the other, but all present in the mix.  S/N for DHB and mix are
generally comparable, while for CHCA it is low.  The presence of
impurities was also assessed.  Peptides from digestion supernatant
were loaded directly on MALDI or desalted and concentrated using
an R2 GELoader tip nano-column.  S/N is better after purification
but fewer peaks are seen.  The lost signals correspond to small hydrophilic
and large hydrophobic peptides, either because they are not retained
or not eluted from the column, respectively.  However, for faint
spots, this purification was necessary to get a protein ID.
<p>
The effectiveness of matrix mix on intact proteins was also evaluated.
Ribonuclease B was examined using different matrix preps.  50 fmol protein
premixed with matrix and applied by dried droplet.  100 laser shots
randomly over the surface were accumulated with CHCA/DHB and CHCA.
DHB matrix alone formed sweet spots.  Ribonuclease B is an Asn-60-linked
glycoprotein with the GlcNAc<sub>2</sub>Man<sub>5-9</sub> oligo (the variable
mannose units produce up to five different glycosylated forms).  All MS
spectra revealed the different forms (mass difference of a mannose unit
is 162 Da).  The better resolution was with matrix mix than pure forms.
CHCA provided poorer resolution, while DHB caused adducting with
high mass tailing.  With test of nonglycosylated proteins (insulin,
myoglobin, trypsin, BSA) results were similar, with improved spectra.
The improvement effect was more pronounced with ribonuclease B.
<p>
More extensive testing with 500 fmol of well-purified recombinant
glycoproteins was done.  A matrix mix produced more intense spectra
than CHCA or DHB alone.  It was believed that urea might have been responsible
for the ineffectiveness of pure matrices.  Thus urea was added to the
preparation:  while crystals were poorly formed in urea presence, spectra
could still be obtained.  With 300 fmol of riboB in the presence of 80 mM
urea, the three matrices were tried.  Only the matrix mix produced well
resolved spectral peaks.  DHB alone produced signals at 300 fmol, and
only a broad peak was found at 600 fmol.  Higher amounts of urea (up to 8 M)
could be tolerated so long as the protein was increased (to 6 pmol).
<p>
Peptides/proteins are modified or fragmented and much depends on matrix
conditions.  Tryptically digested lactoglobulin has a peptide from
A<sub>131</sub>-R<sub>137</sub> with mass 837.06 Da and containing a methionine.
Oxidized and non-oxidized forms are seen with <i>m/z</i> 853.48 and
837.49, respectively.  The oxidized form fragments:  there is a loss of
methanesulfenic acid with resulting <i>m/z</i> 789.51 peak because of
in-source decay (ISD), and then a broad peak at <i>m/z</i> 796.52
due to post-source decay (PSD).  The latter is dependent on MALDI ToF
parameters.  ISD was more likely with matrix mix than with CHCA, and with
DHB was not present.  With intact proteins riboB, a narrow peak at
<i>m/z</i> 13682 is seen, most likely because glycan is lost by ISD caused
by matrix mixture.  With CHCA, there is broad peak with loss of glycan.
<p>
The sensitivity of the matrix preparations is now evaluated using
MALDI-quality standard peptides prepared by a successive dilution of
1 &micro;M stock solution in 5% formic acid.  This is a seven peptide
mix spanning 1-7 kDa.  For targets containing 100-500 fmol, signals for
<em>all</em> peptides were observed with matrix mix and DHB.  At or
below 25 fmol, signals were weak or absent for peptides <i>m/z</i>
3159.4, 5803.6, and 6959.9 for the matrix mix and DHB.  With CHCA
alone, the signals for these three peptides was not even detected up
to the 500 fmol range;  the peptide at <i>m/z</i> 1007.4 was also not
present with CHCA, and was of low intensity with matrix mix and DHB
and undetectable below 50 fmol (ionization is probably suppressed because
of the absence of basic amino acids).  At the lowest level (6 fmol applied
to target), only two of the most intense (<i>m/z</i> 1084.4 and 1281.7)
could be seen, and with a very deteriorated signal-to-noise.  As for
the sensitivity of proteins, sinapinic acid was included because of
its preference as a matrix for proteins.  100 fmol protein on the target
and all five glycosylated forms are seen in spectra.  Similar results
are seen with matrix mix and with DHB alone at 50 fmol, and deterioration
is seen with CHCA and SA alone.  At 25 fmol, only a barely perceptible
signal is seen with mix, and no signal with CHCA, DHB, and SA alone.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Glish GL, Vachet RW
 </span>
 <span class="year">
(2003)
 </span>
 <span class="jname">
Nature Reviews | Drug Discovery
 </span>
 <span class="jvol">
2,
 </span>
 <span class="pages">
141-150.
 </span>
<p class="reftitl">
The Basics of Mass Spectrometry in The Twenty-First Century
<p class="Para1">
<b>REVIEW</b>.  A mass spectrometer does not measure mass directly.
Rather is measures the mass-to-charge (<i>m/z</i>) of ions, or a property
related to <i>m/z</i>.  The spectrum is a plot of the abundance of ions
against <i>m/z</i>.  An important effect of MS is isotopes:  chlorobenzene
has a molecular weight of 112.56, but there are no ions of <i>m/z</i> 112.56.
The formula weight is determined by the weight average of all isotopes of
the atoms present in a molecule.  Chlorine has two isotopes:  <sup>35</sup>Cl
makes up 74.5% of one of the isotopes, and <sup>37</sup>Cl makes up
24.6%.  The ions in chlorobenzene will be seen in the MS as <i>m/z</i>
112.01 or 114.01, and the peak height of the <i>m/z</i> 112.01 ions will
be 3 times higher than the <i>m/z</i> 114.01 ions, reflect the 74.5:24.6
or 3:1 ratio of abundance.
<p>
An MS instrument must achieve several things.  First it can only detect
gas phase ions.  Thus its source must ionize and then put the ions of
interest in the gas phase.  Historically this was achieved by attempting
to volatilize the sample and then ionize it, usually by beam bombardment
(with electrons usually).  This limited MS to compounds that were low
molecular weight and that could be volatilized.  Most biological substances
are high MW and have high polarities which limit volatility.  However
recent work has enabled biological substances of all kinds to be manipulable.
Electrospray ionization (ESI) and matrix-assisted laser desorption ionization
(MALDI) now permit analysis (and their development led to a share of the
2002 Nobel in Chemistry).  ESI has also led to the renaissance of the use
of atmospheric pressure chemical ionization (APcI).
<p>
With ESI, ion generation is achieved by passing the sample in a solution
through a small capillary (&lt; 250 &micro;m i.d.) that is electrically
charged to +500 to +4500 V.  Voltage settings will be a function of the
actual capillary internal diameter and the solvent composition of the sample;
capillaries with larger diameters and higher boiling points require higher
voltages.  In addition a concentric flow of a non-reactive hot gas (typically
N<sub>2</sub>) facilitates the conversion of ions to the gas phase through
nebulization.  Droplets emerge from the tip of the capillary as flow is
started containing the ions, the net charge depending on the polarity of
the capillary.  Ionization of analytes is typically achieved by either their
<b>protonation</b> or <b>deprotonation</b>.  With large biomolecules,
typically polymers composed of ionizable functional groups with acid/base
functionality, multiple ionization is likely.  Multiple charging of single
molecules actually enhances mass measurment.  It allows several peaks of
the same molecule to be seen in the spectrum, since there are several
ions of different <i>m/z</i> and only <i>z</i> is different.  Also, most
mass spectrometers are built to resolve to the highest levels peaks in
a particular range of <i>m/z</i> (usually 500 to 5000).  Several ESI-generated
<i>m/z</i> peaks of ions of the same molecule fall within that range, allowing
accurate mass measurement.  For example, cytochrome <i>c</i> has a molecular
mass of 12,360 Da, but if it is ionized with 10-20 protons, the <i>m/z</i>
is brought to below 2000, in which most MS instruments measure masses with
extremely high accuracy.   ESI is also a <q>soft</q> form of ionization,
in which there is no destruction or fragmentation of the molecule, nor
are multimeric complexes or associations destroyed.  Thus protein-protein
complexes, ds DNA, protein-drug binding are possible to measure.  Indeed
tobacco mosaic virus can be mass measured not only intact, but it even
retains its viability.	MS instruments had to be adapted to ESI since
ESI continously generates ions from samples but MS instruments are not
constructed to continuously analyze them;  MS instruments have a high
duty cycle and ESI sources must be pulsed.  ESI also is susceptible to
ion suppression in which salts in particular can interfere with ion
generation into the gas phase; thus the final stage should have ions of
interest is solvents with little to no salt and with a high nonionized
organic component.
<p>
APcI is an option with ESI.  APcI works by ionization using a corona
discharge enabled by a fine needle.  Voltages between &minus;1000 and
+5000 V and a corona with a current of 2-5 &micro;A is obtained.
Electrons ionize nitrogen, oxygen, and water and solvent molecules
around the needle.  Molecules in nebulized sample are also ionized in
a series of complex reactions as they pass into the discharge region.
Both positive and negative ions are formed (usually by protonation
or deprotonation).  A distinct advantage of APcI is that it is less
susceptible to matrix interference from salts, and weakly polar analytes
not ionizable by ESI are ionized by APcI.  Disadvantages of APcI include
thermal decompostion of labile compounds with nebulizing gas that is
too hot, and the higher sensitivity of APcI requires exacting solvent
purity:  ionization is achieved thermodynamically in APcI, and some
compounds can cause ion suppression.  For example, amines are typically
added to improve chromatography, but can compete with protonation of the
analyte in APcI.
<p>
MALDI is used where samples are generated by pulsed laser irradition.
It is most suitable to MS because MS, particularly time-of-flight, requires
that ions be generated in a pulse and analyzed in that fashion.
Sample is co-crystallized with a matrix that acts as an antenna for the
laser light.
<p>
[continued]
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Wang H, Hanash S
 </span>
 <span class="year">
(2003)
 </span>
 <span class="jname">
J. Chromatog. B
 </span>
 <span class="jvol">
793,
 </span>
 <span class="pages">
15-36.
 </span>
<p class="reftitl">
Derivatization of Carbohydrates for Chromatographic, Electrophoretic
and Mass Spectrometric Structure Analysis
<p class="abstract">
<b>REVIEW</b>.
<p>
<b>ABSTRACT</b>.
Carbohydrates, either alone or as constituents of glycoproteins, proteoglycans and glycolipids, are mediators of several
cellular events and (patho)physiological processes. Progress in the glycome project is closely related to the analytical
tools used to define carbohydrate structure and correlate structure with function. Chromatography, electrophoresis and mass
spectrometry are the indispensable analytical tools of the on-going research. Carbohydrate derivatization is required for most
of these analytical procedures. This review article gives an overview of derivatization methods of carbohydrates for their
liquid chromatographic and electrophoretic separation, as well as the mass spectrometric characterization. Pre-column and
on-capillary derivatization methods are presented with special emphasis on the derivatization of large carbohydrates.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Marvin LF, Roberts MA, Fay LB
 </span>
 <span class="year">
(2003)
 </span>
 <span class="jname">
Clin. Chim. Acta
 </span>
 <span class="jvol">
337,
 </span>
 <span class="pages">
11-21.
 </span>
<p class="reftitl">
Matrix-Assisted Laser Desorption/Ionization Time-of-Flight Mass Spectrometry
in Clinical Chemistry
<p class="abstract">
<b>REVIEW</b>.
<p>
<b>ABSTRACT</b>.
Matrix-assisted laser desorption/ionization time-of-flight mass spectrometry (MALDI-Tof-MS) has recently become a
popular and versatile method to analyze macromolecules from biological origin. In this paper, we will review the application of
MALDI-Tof-MS in clinical chemistry and biology. MALDI-Tof-MS is used in clinical chemistry, e.g. disease markers can be
identified with MALDI-MS analysis in combination with 1-D and 2-D gel electrophoresis separations thanks to either peptide
mass fingerprinting (PMF) or peptide sequence tag (PST) followed by data base searching. In microbiology, MALDI-Tof-MS is
employed to analyze specific peptides or proteins directly desorbed from intact viruses, bacteria and spores. The capability to
register biomarker ions in a broad m/z range, which are unique and representative for individual microorganisms, forms the
basis of taxonomic identification of bacteria by MALDI-Tof-MS. Moreover, this technique can be applied to study either the
resistance of bacteria to antibiotics or the antimicrobial compounds secreted by other bacterial species. More recently, the
method was also successfully applied to DNA sequencing (genotyping) as well as screening for mutations. High-throughput
genotyping of single-nucleotide polymorphisms has the potential to become a routine method for both laboratory and clinical
applications. Moreover, posttranscriptional modifications of RNA can be analyzed by MALDI using nucleotide-specific
RNAses combined with further fragmentation by post source decay (PSD).
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Righetti PG, Castagna A, Herbert B, Reymond F, Rossier JS
 </span>
 <span class="year">
(2003)
 </span>
 <span class="jname">
Proteomics
 </span>
 <span class="jvol">
3,
 </span>
 <span class="pages">
1397-1407.
 </span>
<p class="reftitl">
Prefractionation Techniques in Proteome Analysis
<p class="abstract">
<b>REVIEW</b>.
<p>
<b>ABSTRACT</b>.
The present review deals with a number of prefractionation protocols
in preparation for two-dimensional map analysis, both in the fields
of chromatography and in the field of electrophoresis. In the first
case, Fountoulakis groups has reported just about any chromatographic
procedure useful as a prefractionation step, including affinity,
ion-exchange, and reversed-phase resins. As a result of the various
enrichment steps, several
hundred new species, previously undetected in unfractionated samples,
could be
revealed for the first time. Electrophoretic prefractionation protocols
include all those
electrokinetic methodologies which are performed in free solution,
essentially all relying
on isoelectric focusing steps. The devices here reviewed include
multichamber apparatus,
such as the multicompartment electrolyzer with Immobiline membranes,
Off-Gel
electrophoresis in a multicup device and the Rotofor, an instrument also based on a
multichamber system but exploiting the conventional technique of carrier-ampholytefocusing.
Other instruments of interest are the Octopus, a continuous-flow device for
isoelectric focusing in a upward flowing liquid curtain, and the Gradiflow,where different
pI cuts are obtained by amultistep passage through two compartments buffered at differentpHvalues.
It is felt that this panoply of methods could offer a strong step forward in
mining below the tip of the iceberg for detecting the
<q>unseen proteome.</q>
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Molloy MP, Brzezinski EE, Hang J, McDowell MT, VanBogelen RA
 </span>
 <span class="year">
(2003)
 </span>
 <span class="jname">
Proteomics
 </span>
 <span class="jvol">
3,
 </span>
 <span class="pages">
1912-1919.
 </span>
<p class="reftitl">
Overcoming technical variation and biological variation in quantitative 
proteomics
<p class="Para1">
Quantitative proteomics requires the careful determination of technical
variation between two test sets.  Method comparisons involve pairwise
of spot volumes on 2-D gels (two replicates per method) with correlation
coefficients (<i>r</i><sup>2</sup>) determined.  A coefficient of
variation (CV) was evaluated for repeat gels to examine reproducibility.
Five different sample types (bacteria, cultured cells lines, primary
cultured cells) were evaluated.  <i>r</i><sup>2</sup> values were between
0.83-0.91, while CV values were between 18.7-26.4%.  The technical variation
had an average CV of 23% (average CV of replicate 2-D gels for all five
sample types).  Samples prepared with the IPG format IEF strips did not
differ from a tube gel system used.
<p>
With the degree of technical variation assessed, biological variation
can be evaluated.  The between-experimental variation for bacterial
proteomes showed a <i>r</i><sup>2</sup> = 0.74, CV = 31.2% and for
mammalian cultured cells a <i>r</i><sup>2</sup> = 0.78, CV = 26.0%.
Because the cells are essentially clonal, biological variation is
expected to be minimal, and these values set a mark for making
comparative assessments of biological variation.
<p>
Samples of mouse bone marrow cells and macrophages in primary culture,
often used in drug toxicity studies, were examined for proteomic
variation.	Spot volumes from different experiments, representing
cultures established from different animal pools, showed low reproducibility
as shown by lower <i>r</i><sup>2</sup> and higher CV values (the latter
averaging 40-50%).  Deviation from the mean CV was also high, about 70%,
thus representing the sum of the average CV and the average SD.
<p>
Human serum proteins were quantitated based on clinical drug test.
The drug showed good preclinical efficacy, no Phase I toxicity, and it
required a good serum biomarker to assist monitoring pharmacodynamic drug
activity.  A serum baseline was taken from 4 same-sex patients and then
at 24 and 48 h post-treatment.  30 spots from 2-D gels were found to have
changed greater than 2-fold in any of the patients, but 3 of the 30
(spots 14, 15, 16) were 2-fold changed in all patients at both 24 and 72
hours.  These proteins were repressed in 6 of 8 samples and induced in
the remaining two.
<p>
2-D gels represent the most robust high-throughput method in quantitative
proteomics.  Replicate gels show an <i>r</i><sup>2</sup> of ~0.85.
The average CV for spot volumes in replicate gels is 18-27, consistent
with previous reports.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Vollmer M, N&auml;gele E, H&ouml;rth P
 </span>
 <span class="year">
(2003)
 </span>
 <span class="jname">
J. Biomolec. Tech.
 </span>
 <span class="jvol">
14,
 </span>
 <span class="pages">
128-135.
 </span>
<p class="reftitl">
Differential Proteome Analysis:  Two-Dimensional Nano-LC/MS of
<i>E. coli</i> Proteome Grown on Different Carbon Sources
<p class="abstract">
<b>ABSTRACT</b>.
Different sugars provided to bacteria as single sources of carbon
and energy require the induction of different metabolic
enzymes, transporters, and uptake systems in order to support
growth and cell survival. Using a nanohigh-performance
liquid chromatography/mass spectrometry (nano-
HPLC/MS) system we constructed comprehensive peptide
maps for Escherichia coli grown with either lactose or glucose
in minimal medium. Digested bacterial samples were separated
in a two-dimensional manner by combining strong
cation exchange (SCX) and reversed-phased (RP) chromatography.
Peptides were eluted online to an iontrap MS
instrument and further analyzed by tandem MS fragmentation.
Bacterial proteins originating from the differing samples
were analyzed by searching the Swiss Prot Database. Data
are presented that show the ability to detect several hundred
different proteins significantly expressed under both conditions.
Several enzymes and binding proteins related to the
lactose metabolism were only identified in the sample grown
with this carbon source.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Patterson SD, Aebersold R
 </span>
 <span class="year">
(2003)
 </span>
 <span class="jname">
Nat. Genet. Suppl.
 </span>
 <span class="jvol">
33,
 </span>
 <span class="pages">
311-323
 </span>
<p class="reftitl">
Proteomics: The First Decade and Beyond
<p class="Para1">
<b>REVIEW</b>.  Discovery science aim at assaying function and control
of biological system based on less well-defined technologies.
Proteomics at one point thought to identify all proteins in a cell
or tissue, but currently serves more useful purposes including
sequencing, quantifying, assessing modification states, interactions
with other proteins, biological activity, subcellular distribution,
and structure.
<p>
<q>Forward genetics</q> refers to the standard process of using
protein chemistry as an expression of phenotype to get at the genes that
were responsible for the phenotype.  A glossary of terms is included
that are useful in understanding proteomics, including 2DE, discovery
science, DNA microarrays, ESI, ICAT, ion source, ionization, MALDI,
mass analyzer, mass spectrometry, MS spectra, MS/MS (or CID) spectra,
protein identification, and yeast two-hybrid.
<p>
ESTs represent large-scale efforts to sequence partially the cDNAs of
cells and tissues.  These and genomic libraries provide information
about possible proteins.  Then mass spectrometry of peptides and proteins
provide mass information useful in identification of proteins, reported
in 5 different pubs in 1993.  MALDI-ToF has a greater ease of use and
is the method of choice.  ESI was used mostly with triple quad and ion trap
systems.
<p>
Better algorithms can now match MS/MS spectra to sequence databases,
and are especially suitable for partial sequence databases such as ESTs.
<p>
Gel proteomics is not as promising as thought.  2DE is technically
complex and labor-intensive.  The resolution of gels is not what was
once thought, with MALDI-MS and ESI-MS/MS revealing a great many
spots with comigrating proteins.  This is especially so when protein
sets are unfractionated and using conventional staining which is relatively
selective in illumination as well as insensitive to low-abundance proteins.
<p>
With the demonstration of LC-MS/MS to show parts of the proteome not
shown by gel-based methods, a gel-independent approach was realized.
Hunt and coworkers uses immunoprecipitation to isolate peptide-MHC
complexes and then subjected antigenic peptides to LC-MS/MS.  They
also used a cytotoxic T cell (T<sub>c</sub>) bioassay to verify
antigenic peptides in each fraction, correlating functional data with
the mass spectrometric data.  The use of LC-MS/MS is called <q>shotgun
proteomics</q> and can catalog hundreds, even thousands, of peptides.
Proteins identified by this process include subcellular fractions, including
periplasmic space of bacteria, yeast ribosomal complexes, murine nuclear
interchromatin granule clusters (nuclear speckles), murine mitochondrial
soluble intermembrane proteins, and human urinary proteins, and many more.
A limitation of this method is the ability to get MS/MS (CID) spectra
for all peptides in a sample, of which tens of thousands can be generated
in a single experiment.  One protein digested by trypsin can generated
30-50 peptides (theoretically), and so a digest of the entire proteome
generates hundreds of thousands of these.
<p>
Complexity reduction approaches are therefore considered.  Typically the
simplest is fractionation&mdash;selection of the sub-proteomic set.
One method might be to target proteins with reactive cysteine sulfhydryls,
or residues modified with phosphate (phosphoproteins) or
carbohydrate (glycoproteins).  Multidimensional chromatography has
also helped reduce complexity with use of cation-exchange and capillary
RP columns interfaced to ESI.
<p>
One problem with mass spectrometer is the difficulty of quantitating
differences in expression.  A problem is that the mass spectrometer
will detect peptides with different signal intensities based on chemical
composition, and signal intensity is also a function of the matrix and
other variables not really understood.  The intensity of a peptide signal
is not quantitatively associated, and certainly there is no reasonable
expection of a linear response.  One means of making MS quantitative
however is to use isotope dilution techniques.  Isotope-coded affinity
tags (ICAT) was the first approach tried.  The ICAT reagent is composed
of iodoacetamide group to allow reaction with the protein (by cysteine
thiols), a polyether linker containing either eight <sup>1</sup>H or
<sup>2</sup>H atoms, and a biotin affinity tag.  The tags are labeled
d0-ICAT or d8-ICAT, depending on whether they contain hydrogen or
deuterium, respectively.  Reagent is added to protein from the reference
(control/untreated) set, and the other reagent is added to protein
from the study (treated) set.  These ICAT-labeled samples are now
combined.  This is possibly followed
by enrichment, then digestion with protease.  An avidin column is used
to recover the labeled tags now bound to peptides, and the peptides
analyzed by LC-MS/MS.   The ratio of the abundance d0- to d8- peptides
is the ratio of the relative abundance of proteins, and thus serves
to quantitate differences.  Identification is done by MS/MS spectra.
Other methods include solid-phase isotope tag transfer, <sup>16</sup>O
and <sup>18</sup>O incorporation from water containing enriched isotopes
during trypsin digestion (which incorporates the isotopes into the
carboxyl terminus), and stable-isotope metabolic protein labeling.
<p>
Systems biology takes the approach that as much data that can be obtained
globally as possible should be done.  Identification of interesting
components or processes is then done by mathematical tools such as
cluster analysis or singular value decomposition.
<p>
Another area of activity in proteomics is actually the evaluation
of activity, biological activity.  Activity-based reagents react with
only the biologically active forms of protein.  The tags are either
fluorophores or biotinylated probes in cases where MS is not used,
but isotope-tagged versions are being developed.
<p>
(Micro)array-based profiling is also being done for proteins as it
is done for transcripts.  Ordered arrays of specific protein-binding
modules are being prepared, with modules including phage library-selected
scFv antibodies, minibodies, cyclic peptides, atpamers, antibodies, etc.
Problems with array-based profiling include the need to maintain the
native (binding) state during imprinting (not a problem with cDNA),
and to keep sensitivity since amplification of proteins is not possible
unlike with DNA-based microarrays.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Lehner I, Niehof M, Borlak J
 </span>
 <span class="year">
(2003)
 </span>
 <span class="jname">
Electrophoresis
 </span>
 <span class="jvol">
24,
 </span>
 <span class="pages">
1795-1808
 </span>
<p class="reftitl">
An Optimized Method for The Isolation and Identification of Membrane Proteins
<p class="Para1">
Three methods were used to extract membrane proteins from A549 cultured
cells.  Each method was done three times.  The <b>sequential</b> method
was that of Molloy et al (1998 <i>Electrophoresis</i>).  Thawed frozen
cells are extracted in Tris (vortex 10 min, 40 interval sonications),
then centrifuged at room temp for 10 min max speed in microfuge, and
then the pellet is extracted in urea/thiourea/CHAPS/SB3-10/DTT/ampholytes
with the same vortex and sonications, and centrifuged such that
the supernatant is the membrane fraction.  With <b>detergent-based</b>
extraction, the cells are lysed in buffer with digitonin (10 mM PIPES pH 6.8,
0.015% digitonin, 0.3 M sucrose, 0.1 M NaCl, 3 mM MgCl<sub>2</sub>,
5 mM EDTA, 10 &micro;l/ml Pefabloc SC, 10 &micro;l/ml aprotinin).
The supernatant is recovered as cytosolic extract after low speed spin
(480 <i>g</i>, 5 min, 4&deg;).  Pellet is suspended in Triton buffer
(10 mM PIPES pH 7.4, 0.5% v/v TX-100, sucrose, NaCl, MgCl<sub>2</sub>,
EDTA, Pefabloc SC and aprotinin as before), shaken for 30 min at 4&deg;
then spun 5000 <i>g</i> 10 min 4 &deg;.  The supernatant is recovered
as lyophilized.  All fractions are stored at &minus;80&deg;.  In a
<b>centrifugal</b> extraction method, cells are suspended in 150 mM
KCl (3 ml per gram cells) and homogenized in glass homogenizer (20 strokes).
These are centrifuged in Optima LE-80K with Ti70 rotor at 15,000 rpm
for 30 min (4&deg;), equiv. to 11,000 <i>g</i> (the pellet is DNA-rich).
The supernatant is recovered and spun 55,000 rpm (170,000 <i>g</i>)  for
60 min (4&deg;).  The pellet is then resuspended in 2D loading buffer
(5 M urea, 2 M thiourea, 2% CHAPS, 2% SB3-10, 40 mM Tris, 100 mM DTT,
0.5% 3-10 ampholytes).  In a <b>Whole-Cell</b> extraction method, cells
are thawed and given the 2-D loading buffer, vortexed 5 min vigorously at
4&deg;, sonicated 20 times, and then microfuged 10 min at RT before storing
frozen.  For an <b>SDS-based total protein</b> extraction, cells are suspended
in a lysis buffer (only Tris and SDS) and then boiled for 10 min.  This is
then followed by sonication on the cooled sample, spinning on microfuge 10
min and then collecting the supernatant.  A small aliquot is taken for
protein assay, and then sample buffer is added, boiled 3 min, then stored
frozen.  Protein assays consisted of the Bradford and BCA assays using
microtiter plate format.
<p>
Results show that the best total yield was a detergent-based method
(32.3 &micro;g protein/million cells), followed by the sequential method
with 16.2 &micro;g protein/million cells, then centrifugal method
at 9.0 &micro;g protein/million cells.  Another method (sucrose/percoll)
produced yields too low to be seriously considered (0.17 &micro;g/million cells).
<p>
A marker for testing membrane protein extraction was E-cadherin (120 kDa)
for which anti-E-cadherin is available for Western blotting (1D).
It was detectable in the SDS, sequential, detergent, and centrifugal
extractions in the membrane fractions, and not detectable in the cytosolic
fractions of the sequential and centrifugal, but present in the
detergent-based cytosolic fraction.  The sequential and detergent-based
fractions did better at enriching E-cadherin compared to total protein.
The centrifugal and sequential fractionation clearly put the protein
in the membrane fraction.
<p>
IEF was done on 7 cm IPG 3-10 strips ranging from 8000, 20,000, and
35,000 Vh.  The best time was 20,000 Vh based on analysis of the
membrane fraction of the sequential method.  Medium-range gels (pH 4-7,
5-8, 7-10) along with pH 3-10 were run on the same sample.  In the pH 3-5
region there were few if any spots in all gels.  The pH 5-8 region was
very crowded.  Streaking was evident at the gel edges, and half of the
pH 7-10 gel showed nearly nothing  but streaking.
<p>
A comparison was made of 2-D gels run with 17 cm pH 3-10 strips using
different samples (whole cell extracts, cytosolic and membrane fractions)
in which the sequential extract was used for the latter two.  The whole
cell extracts should contain both cytosolic and membrane fractions ideally
and the 2-D gel images did show where proteins segregated according to
their selective solubility.  In a comparison of the extraction method,
the sequential, detergent, and centifugal extractions all gave distinctive
2-D gels, and so it appears that not all proteins are obtained by one
strict method.
<p>
Clearly the size of the gel mattered as well.  Using narrower range pH
IEF strips and larger strips (gel formats) always produces greater spots,
as much as twice the amount.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors" id="lebihan-2003">
Le Bihan T, Duewel HS, Figeys D
 </span>
 <span class="year">
(2003)
 </span>
 <span class="jname">
J. Am. Soc. Mass Spectrom.
 </span>
 <span class="jvol">
14,
 </span>
 <span class="pages">
719-727.
 </span>
<p class="reftitl">
On-Line Strong Cation Exchange &micro;-HPLC-ESI-MS/MS for Protein Identificatioin
and Process Optimization
<p class="Para1">
Sensitivity for discovery of low level proteins is a problem with gel-based
proteomics methods.  Membrane proteins are usually not detected by
2-D gel systems.  Concentration levels among proteins can differ by
as much as 10<sup>5</sup>, with total number of protein species in the
thousands.
<p>
New strategies including targeting specific classes of proteins, use of
affinity purification, and multidimensional fractionation are being reported.
Subsequent to enrichment, (usually trypic) proteolysis followed by separation
of peptides and identification by MS is used.  MudPIT is a strategy
used that couples online SCX chromatography to RP
columns; it is also done offline.  Little optimization of the first
dimension SCX is done because of time.
<p>
The system used here is Picotip capillary column
(75 &micro;m i.d./360 &micro;m o.d. &times; 7 cm, with 15 &micro;m tip opening,
New Objective, Inc.)
fitted with a precolumn (200 &micro;m &times; 3 cm fused silica tubing,
Polymicro Technologies),
with frit created in precolumn according to Zhang &amp; Huang.  Connectors
or fittings were from Upchurch or Valco.  SCX
buld resin was polysulfoethyl aspartamide (5 &micro;m, 300 &Aring; pore,
PolyLC, Inc.) were conditioned with 0.2 M NaH<sub>2</sub>PO<sub>4</sub>, 0.3
sodium acetate (see <a href="http://www.westernanalytical.com/">Western
Analytical</a> web site for method).  Zorbax SB-C<sub>18</sub> (5 &micro;m)
bulk came from Agilent Technologies.  Conditioned resin is suspended then
in 1 M ammonium formate, 0.25 M formic acid and then slurry-packed in column
and pre-column using home-built pressure vessel (pressure vessel connected
to Perkin-Elmer series 200 Micropump), delivering 0.1% (v/v) formic acid
with max press 400 bar.  For the RP packing, slurry was in chloroform and
the pump delivered acetonitrile.
<p>
A gradient elution for SCX was 0.25 M formic acid to 0.25 M formic acid +
1.0 M ammonium formate was designed for direct delivery to ESI-MS.
The precolumn is 200 &micro;m i.d. &times; 360 &micro;m o.d. &times; 3 cm.
The column is 75 &micro;m i.d. &times; 360 &micro;m o.d. &times; 5 cm
with 15 &micro;m tip opening.  A 3-tee connects the HPLC pump to the
pre-column, and the perpendicular goes to one port of a 6-port valve.
A 4-tee (cross) connects to the other end of the pre-column straight through
to the column; the other connections are to another port of the 6-port valve
and straight through to a platinum wire.  The other ports of the 6-port
valve are connected to (1) waste, (2) a loop to two of the ports, and
(3) a line to waste but one which features a <i>restrictor</i> capillary
which is 20 &micro;m i.d. &times; 360 &micro;m &times; 15 cm
fused silica.  In the load position, sample comes through the restrictor
line through the loop and into the pre-column via the 3-tee.
In the inject position, flow through the loop goes out to waste from the
4-tee.  (If port 1 is at the top, connect port 1 and 4 to the loop, port
2 to the line from the 4-tee, port 3 to waste, port 5 to the restrictor
line that goes to waste, and port 6 to the line going to the 3-tee.)
<p>
Performance of the system was tested on a BSA tryptic digest.  950 fmol
was injected on to the SCX precolumn + cap column setup.  The gradient
program was a wash for the first 4 min, then a 10 min increase to 0.2 M
(amm formate), then a 3 min increase to 0.6 M, then 1 min to 1.0 M, which
remained there for 4 min before a 1 min decrease back to zero salt.
The total program was 30 min.  This represented an optimized gradient
with 30 peptides separated and identified for 41% sequence coverage.
ESI was stable at 1.0 M amm formate.  A MS/MS spectrum was done of the
eluate.  Retention time reproducibility between times 16-22.5 min was
2-10 seconds, peak to peak.  Peak width at half height was 10-15 seconds
for all peaks evaluated by MS.
<p>
Because tryptic digests are usually done in ammonium bicarbonate and this
is incompatible with SCX without dilution.  Samples were brought to 0.25 M
formic acid after trypsin digestion containing various levels of Amm Bicarb.
At 50 mM Amm Bicarb, sample loss and broadening was significant, and
was noticeable at 25 mM as well.  Better recovery was seen at 5-10 mM Amm
Bicarb.  Peak broadening in generally is seen in samples eluting early
in the gradient, and is even not really observed with late eluters (even
with 50 mM Amm Bicarb in the load).
<p>
Step-gradient elution from the SCX to RP columns makes it easier to optimize
the ion-exchange elution profile.  With 50 mM ammon. formate in 250 mM FA,
only 6 major components are seen, but when ammon. formate is increased
to 100 or 150 mM ammon. formate, 9-10 distinct components appear.  Early
eluting peptides elute no matter what the concentration of NH<sub>4</sub>OOCH.
No peptide binds to the column indefinitely; if the duration is sufficiently
long, the peptide elutes.  Stronger interacting peptides will show peak
broadening at lower concentration steps.
<p>
Tryptic peptides of BSA and phosphorylase <i>b</i> were examined by SCX-MS
and the calculated pI plotted against retention time using the Bjellqvist
algorithm.  With peptides having pI &lt; 6, an increasing pI was seen in an
increasing retention time. Little or no dependency is seen with peptides
&gt; 6.  No trend was observed between peptide time on RP vs. SCX column.
<p>
The limit of detection was determined to be the lowest amount of protein
that can analyzed and identified from a database search.  Different
amounts of phosphylase <i>b</i> were loaded and the search results plotted
against the mass loaded.  A relationship was clear between score and load,
and a relationship between score and number of peptides as well.  As little
as 17 fmol was sufficient to identify phosphorylase <i>b</i>.
<p>
Online SCX-MS/MS and RP-MS/MS were compared with respect to complex sample
profiling.  Human K562 cells were cultured in DMEM + 10% FBS until 100 million
cells were grown;  cells were spun (410 <i>g</i>) and washed in PBS twice.
TRIzol was used on a cell pellet, and the final protein pellet solubilized
in 1% SDS (1 ml).  The BioRad DC protein assay was used to assess protein.
200 &micro;l (about 1 mg) of solublized protein was diluted in 500 &micro;l
water and 75 &micro;l of 1 g/ml TCA, and sample left at &minus;20&deg; for
5 min.  Precipitate was pelleted 20,000 <i>g</i> for 10 min at 4&deg;.
Pellet was resuspended in 0.5 mM HCl, 50:50 acetone:MeOH (300 &micro;l) and
after 5 min spun 15,000 <i>g</i> 10 min at 4&deg;.  200 &micro;l cold
acetone was added to the pellet and sample kept at &minus;20&deg; for 10-20
min.  This was again spun 20,000 <i>g</i> 10 min at 4&deg;.  Thus the
pellet was washed several times and dried on a centrifugal evaporator.
Samples were digested with trypsin:  pellet was suspended in 0.1 M ammonium
bicarbonate containing 4 mM DTT and incubated 55&deg; for 30 min, followed
by alkylation with 12 mM iodoacetamide 60 min at 37&deg; in the dark.
Sample is then digested overnight at 37&deg; using a trypsin:substrate
1:50 (w/w).  About 2 &micro;g of cell protein (the amount in 22,000 cells)
was submitted to SCX-MS/MS and RP-MS/MS and the top 25 scoring proteins
identified.  Both SCX and RP identified the some of the same top
10 proteins, but in different order, and there were unique peptides.
The dnaK-type chaperone ranked 5 on RP-MS but was not in the top 25 of
SCX-MS as an example, and polypeptidyl-tRNA translocase and cyclophilin A
were 9 and 10 on SCX-MS but not in top 25 of RP-MS.  Few peptides per protein
were obtained with SCX as opposed to RP.
<p>
Both SCX- and RP-MS tend to favor identification of peptides depending on
their average molecular weight.  A plot of relative distribution percent
against the peptide length of peptides analyzed show an asymmetric distribution
for both modes.  The peak maximum falls in the range 8-12 residues for RP-MS,
and 14-18 residues for SCX-MS.  A plot of integrated (cumulative)
distribution percent against peptide length allows one to find the
peptide lengths at the 50% point, an averge:  for RP-MS, it is
about 13 residues, and for SCX-MS, about 15.4 residues.  The higher the
mass of the peptide, the more like SCX-MS is to identify it, but more
importantly, how RP- and SCX-MS make complementary coverage.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
M&uuml;ller WEG, M&uuml;ller IM <i>in</i>  W.E.G. M&uuml;ller, ed.,
<i>Marine Molecular
Biotechnology</i>, Progress in Molecular and Subcellular Biology series,
(Berlin: Springer-Verlag), 2003, pp 1-33.
 </span>
<p class="reftitl">
Analysis of the Sponge [Porifera] Gene Repertoire:  Implications
for the Evolution of the Metazoan Body Plan
<p class="Para1">
All metazoan phyla likely originate from a hypothetical common ancestor,
the Urmetazoa.  Gene analysis and cytological data now disprove the posited
view that the Choanoflagellida are the closest to Urmetazoa.  Some
gene fine structure analyses of protein kinase C, hsp70 and others suggest
Metazoa have a common ancestry with Fungi.  Analysis of coding regions
in sponges (primarily the marine <i>Suberites domuncula</i> and
<i>Geodia cyodonium</i>) show an unexpected similarity to humans and more
divergence from <i>C. elegans</i> and <i>D. melanogaster</i>.  Additional
divergence from protostomians is found in molecules of the immune system
and the related apoptotic pathway.  The genome of <i>S. domuncula</i>
is being mapped in toto and can be
<a href="http://spongebase.uni-mainz.de/cgi-bin/blast/blastserver.cgi">tracked</a>.
Fungi and Urmetazoa diverged probably after establishment of signal
transduction and adhesion molecules (galectins).  The Urmetazoa
benefitted from development of cell-cell/matrix adhesion molecules,
immune molecules, and neuronal receptors and morphogens.  The Porifera
then emerged (Hexactinellida, Demospongiae, and later the Calcarea.
Prior to the emergence of the Radiata, a nerve system, germ-somatic
cell lines, and homeotic genes emerged.  From these the Cnidaria and
Ctenophora diverged from Radiata.  With the development of a dorso-ventral
axis, the Urbilateria then emerged, diverging into the Protostomia
and Deuterostomia body plans.
<p>
Sponges are sedentary filter-feeders with well-constructed and complex
network of water-conducting canals and choanocyte chambers.  The chambers
are lined with flagellated choanocytes.  The entire system clears a water
mass equal to the volume in the sponge every 5 seconds.  Two cell layers
envelope a <q>ground</q> substance, the pinacoderm on the external
sponge surface, and the choanoderm (also called endopinacoderm) on the
internal lining.  The ground substance is the <i>mesohyl</i>.
Analysis of the protein complement of sponges has indicated how they
have a body plan.  Proteins critical to cell-cell/matrix connections
are found:  integrin, collagen, FN<sub>3</sub>, SRCR- and SCR-domain-
containing proteins.  Signal transduction tyrosine kinases are present.
Immune-related proteins: cytokines, variable Ig-like molecules,
(2-5)A synthetase.
<p>
Sponge classification is not quite resolved.  Demospongiae and Calcarea
are cellular sponges, and Hexactinellida is a synctial group.  Another
is based on the fact that Demospongiae and Hexactinella are siliceous
while the Calcarea are removed from this clade and linked to Cnidaria.
Coding region gene analysis however suggests that the Calcarea are
likely more related to other nonsponge metazoans.  Thus Porifera are
paraphyletic with Calcarea forming a clade with other Metazoa.
Homeobox genes are indicators of anterior/posterior axis although no
morphological evidence indicates such an axis; related genes appear
to be transcription factors with a general function, and are also present
in fungi and plants.
<p>
A mineral skeleton is created by the secretion of spicules from sclerocytes.
In the Demospongiae, silica is crystallized around an organic nucleating
filamentous element, but no such filament is the basis of nucleation in
Calcarea.  Demospongiae also form their spicules intracellularly, while
Calcarea use a cluster of sclerocytes to form the spicule extracellularly.
Demonspongiae and Hexactinellida are alone in using silica to form a skeleton,
as the rest of the animal kingdom uses calcium.  Calcium in seawater is
orders of magnitude higher in concentration than silicon (10 mM vs.
5-20 &micro;M).  Demospongiae also have high amounts of polyphosphate
(10 &micro;g/g wet weight), which chelates calcium and prevents its
precipitation.  Demospongiae also harbor symbiotic bacteria and Calcarea
do not, possibly indicating that polyphosphate is of bacterial origin
<p>
The many studies of <i>Microciona prolifera</i> and <i>G. cydonium</i>
have been useful in studying cell-cell and cell-matrix adhesion.
A special <b>aggregation factor</b> (AF) was isolated from these sponges.
In <i>M prolifera</i> this had a relative molecular mass of 20 million
daltons!  AF mediates cell-cell aggregation and is calcium-dependent.
Later the first cell-cell adhesion (galectin) was cloned as was
a cell-matrix adhension integrin.
<p>
AF is composed of a proteoglycan core.  Protein MAFp3 has been cloned
as one member of the core.  It is likely entrapped in a polysaccharide
cover.  The gene encoding MAFp3 is quite polymorphic.  An 86 kDa
polypeptide is thought to be associated with the core (peripherally)
and its C-terminal has a significant hydrophobic stretch (residues
786-806 for a 807 aa long polypeptide);  on the N-terminus are nine
short consensus repeats (SCR), each separated by 4-aa long spacers
(the spacers also have an interesting combination:  NRVQ, QIVA, VRNQ, EIVM,
IRNQ, EIVL, NRVQ, QAVE).  Generally well conserved in the SCR
are four Cys, two Gly, one Trp, one Phe, and two Pro residues, something
typical in the mammalian complement-control protein family.
<p>
Antibodies to AF have been found to eliminate its aggregating function, and these
antibodies were used to find a 1.7 kb cDNA whose N-terminus coding region
has high similarity to amphiphysin/BIN1 of both Proto- and Deuterostomia,
but which lacks a characteristic SH3 domain in the C-terminus.  A 47 kDa
protein is predicted, and a 36 kDa protein recombinant was expressed in
E. coli.  This protein competes for binding with the core, and is also
associated with galectin.  It is predicted to connect the AF core to
galectin.
<p>
Galectin is a carbohydrate-binding protein that links the AF core,
probably through a 36 kDa associated polypeptide, to AR, the aggregation
receptor.  It has three isoforms, from soluble to membrane-associated.
<p>
The aggregation factor receptor, AR, is a membrane-associated protein
that links the cells of the pinacoderm to the AF complex.  The protein
is composed of 14 scavenger receptor cysteine-rich (SRCR) domains,
6 short consensus repeats (SCR), a transmembrane region in the C-terminus,
and a C-tail that is intracytoplasmic.  Several experiments have shown
that AR binds to AF complex and that it is strengthened by galectin.
<p>
<b>Cell Adhesion Matrix</b>.
The mesohyl is not really a homogeneous ground substance but is composed
of galectin, collagen, fibronectin-like molecules, and minor component
recently identified as dermatopontin.  Collagen is present only in
metazoa.  At least 20 different types of collagen can be found in the
higher metazoa, but only two types are in sponges:  fibrillar and
type IV-related collagen.  EM studies show sponge collagen is made
of 20-25 nm thin fibrils which have a regular banding pattern and
period length of 60 nm that are three periods long;  intraperiod
bands dissect the period.	Molecular analysis of both marine and freshwater
sponges shows typical G-x-y triplets forming internally and terminally
helixed strands.  One species shares a feature with type IV collagen
found in basement membranes, one associated with integrins.
<p>
Long ago sponge have a protein that cross-reacts with antibody to
firbronectin.  One cDNA from <i>G. cydonium</i> codes for an RGD
domain, and another has a cDNA for the &alpha;-subunit of integrin.
Three of the sponge's proteins cross-react with human anti-fibronectin
antiserum.
<p>
Integrins are membrane proteins composed of &alpha;&beta; heterodimers.
16 different &alpha; and 8 different &beta; subunits have been identified
that yield more than 20 types of heterodimeric receptors.  When integrins
bind, they activate intracellular signalling pathways (kinases, SH2
and SH3 signaling and phospholipid mediators).  With collagen in the ECM
of sponge, cDNAs were found that encode &alpha; subunit of integrin.
These subunits show presence of 7-8 repeating domains and putative
divalent cation binding sites.  The greatest sequence similarity is
with <i>D. melanogaster</i> and <i>C. elegans</i> from invertebrates,
and mouse integrin &alpha;V, chick integrin &alpha;VIII, and human
fibronectin receptor &alpha; subunit.
<p>
A &beta; integrin 878 aa long and 95,215 Da has three Cys-rich domain
signatures.  Also an EGF-like domain Cys-pattern signature, transmembrane
segment, and Ca<sup>2+</sup>-binding site.  Cytoplasmic domains interact
with the cytoskeleton.  35% sequence identity and 55% sequence similarity
with other &beta; integrins in higher metazoa complete the description.
Sponge integrins trigger signal transduction pathways.  AF-mediated
cell-cell adhesion is blocked by RGDS peptide, which interacts with
&beta; integrin.  RGDS and AF stimulated DNA synthesis as well.
RGDS or AF addition to isolated cells of <i>S. domuncula</i> induces
increases in intracellular Ca<sup>+</sup> within 1-2 min.  There is also
an activation of GTP-binding protein Ras.  Prolonged incubation of
the cells with RGDS and AF also upregulates Ras and calmodulin.  Integrin
is thus an adhesion molecule and signal mediator.
<p>
Proliferation is associated with differentiation in higher metazoans.
Moreover, ageing and senesence is associated with reduction in telomerase
activity.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Wei YQ, Feng R, Yi ZS, Liu QF
 </span>
 <span class="year">
(2003)
 </span>
 <span class="jname">
Di Yi Jun Yi Da Xue Xue Bao
 </span>
 <span class="jvol">
23,
 </span>
 <span class="pages">
1161-1164.
 </span>
<p class="reftitl">
Establishment of two-dimensional gel electrophoresis profiles of
proteome from CD34<sup>+</sup> hematopoietic stem/progenitor cells
<p class="abstract"><b>Abstract</b>. <i>Article in Chinese</i>.
OBJECTIVE: To investigate the differential expression of proteins
in bone marrow (BM) and mobilized peripheral blood (MPB)
CD34<sup>+</sup> cells.
METHODS: Immunomagnetic beads were used to separate and purify
CD34<sup>+</sup> cells from the BM and the MPB mononuclear cells
(MNCs) mobilized by granulocyte colony-stimulating factor (G-CSF)
of normal subjects.
The whole cell proteins in CD34<sup>+</sup> cells were extracted by freeze-thaw
lysis, and applied for two-dimensional electrophoresis (2-DE),
the results analyzed with image analysis software.
RESULTS: The average purity of CD34<sup>+</sup> cells was 92.33% &plusmn; 2.65%
in output, with an average cell number of (1.12 &plusmn; 0.42) &times; 10<sup>6</sup>.
2-DE techniques were optimized, which yielded satisfactory 2-DE
profiles of the proteome, showing the different expressions of
the proteins between different CD34<sup>+</sup> cells.
CONCLUSIONS: Immunomagnetic beads in combination with 2-DE is
applicable for research of hematopoietic stem/progenitor cells
proteome. There are differences in the protein expressions
between BM- and MPB-derived CD34<sup>+</sup> cells.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Shaw MM, Riederer BM
 </span>
 <span class="year">
(2003)
 </span>
 <span class="jname">
Proteomics
 </span>
 <span class="jvol">
3,
 </span>
 <span class="pages">
1408-1417.
 </span>
<p class="reftitl">
Sample Preparation for Two-Dimensional Gel Electrophoresis
<p class="Para1">
<b>REVIEW</b>.  Sample prep in 2DGE depends largely on the aim of the
separation.  If soluble aggregates are to be separated under IEF, the
prep method should be nondenaturing, nonreducing.  Thylakoid membrane
proteins have been prepared in nonionic detergents (Triton X-100 and
<i>N</i>-dodecyl-&beta;-D-maltoside);  these disrupt lipid-lipid
and lipid-protein as opposed to protein-protein interactions.  In most
cases however, sample prep is denaturing and reducing to achieve the
solubilization of the most proteins.
<p>
Urea disrupts hydrogen bonds and should be present between 5 and as
high as 9.8 M.  Urea breaks down by hydrolyzing to cyanate and
ammonia.  Left overnight, a 10 M urea solution at room temperature will
produce 20 mM cyanate ion.   Cyanate can also react with primary amine
to form urea monosubsitutes (carbamylation);  these amine groups
are normally positively charged on proteins, but are lost and can
affect IEF results.  Reaction can also occur under slightly acidic
conditions (at pH 6) with sulfhydryl groups to form substituted thioureas.
Spermine is a cyanate scavenger that can be added during strip rehydration.
<p>
Thiorea was first used in 1996 in membrane solubilization.  At pH 8.5-9
the sulfur on thiourea becomes as reactive as the cysteine sulfhydryl.
This happens to be the pH of the 2nd dimension equilibration buffer
(pH 8.8).  Thus thiourea will scavenge iodoacetamide and result in
poor alkylation.  It has been recommended that reductive alkylation be
done prior to IEF.  DTT itself is a scavenger of cyanate, and alkylation
will prevent carbamylation of the protein primary amines.  Thiourea
is used at 2 M along with urea at 5-7 M;  Musante et al. report that
protein resolution is seriously affected with high thiourea, so they limited
it to 0.5 M.  One problem with protein resolution may be that thiourea
inhibits SDS-protein binding or that the lipid content is too high,
which also reduces 2-D resolution.
<p>
The use of SDS in IEF appears illogical since it imparts a negative
charge to proteins.  However, SDS will not interfere with IEF if used
together with an excess of nonionic or zwitterionic detegents such as
NP-40, Triton X-100, CHAPS.  SDS will form micelles with NP-40 and
migrate to the anode;  an NP-40:SDS ratio of 8:1 is necessary to avoid
streaking.  Moreover if SDS is used, it should be &lt; 0.25%.
Other problems with using SDS is that its electrophoretic removal during
focusing might result in <i>in-gel</i> protein precipitation.
<p>
Nonionic detergents are used in nondenaturing IEF, such as Tween 80,
NP-40, Triton X-100.  These are used at 0.4-4%, but 10% NP-40 has been
used.  As for zwitterionics, CHAPS has proven the most effective,
but SB3-10 and ASB-14 are useful as well.  Compatibility of detergents
with high urea concentrations is a factor.  SB3-10 is compatible with
only low concentrations of urea.
<p>
<b>Reducing agents</b>.  2-mercaptoethanol is little used because of its
tendency to ionize at the alkaline end of the IEF gel, causing it to
electrophorese to the cathode and to disturb carrier ampholytes in the
region, resulting in streaking.  This also happens with DTT, DTE,
and TCEP (tris[2-carboxyethyl]phosphine).  Often disulfide-rich
prtoteins, incuding and especially keratins, form spurious spots in the
alkaline region.  Use of TBP apparently overcomes the problem.
<p>
<b>Carrier ampholytes</b>.  Carrier ampholyte use appears to inhibit
interactions between hydrophobic proteins and immobilines, often occurring
at the basic end of the gel and leading the streaking.  They also help
to scavenge cyanates and to precipitate nucleic acids.  Used at 0.5-2%,
although 2% is recommended if proteins solubilize poorly.
<p>
<b>Protease inhibitors</b>.  Denaturants will inhibit proteases but
even in 1 mg/ml SDS, cathepsin C, several carboxypeptidases and endoproteinases,
chymotrypsin, trypsin, plasmin, and proteinase K are among the examples.
Inclusion of protease inhibitors is recommended (see table)
<table style="text-align:center;">
<caption>Protease Inhibitors and Their Targets</caption>
<tr><th>Inhibitor<th>Target<th>Recommended Concentration
<tr><td>APMSF <td>Plasma serine proteases <td>10-20 &micro;M
<tr><td>Aprotinin <td>Serine proteases  <td>0.01-0.3 &micro;M
<tr><td>Bestatin <td>Aminopeptidases <td>40 &micro;g/ml
<tr><td>Dichloroisocoumarin <td>Serine proteases <td>1-43 &micro;g/ml
<tr><td>Disodium EDTA  <td>Metalloproteases <td>100 &micro;M
<tr><td>E-64  <td>Thiol proteases  <td>1.4-2.8 &micro;M
<tr><td>Leupeptin <td>Serine and thiol proteases <td>1 &micro;M
<tr><td>Pepstatin <td>Acidic proteases  <td>1 &micro;M
<tr><td>PMSF      <td>Serine proteases  <td>100-1000 &micro;M
<tr><td>Phosphoramidon  <td>Thermolysin<br>Collagenase<br>Metalloendoproteinases<td>7-569 &micro;M
<tr><td>TLCK-HCl    <td>Trypsin<br>Thiol proteases <td>37-50 &micro;g/ml
<tr><td>TPCK       <td>Chymotrypsin<br>Thiol proteases <td>70-100 &micro;g/ml
</table>
<p>
<b>Sample Prefractionation</b>.  Prefractionation can be done not just
chemically, but anatomically (careful dissection of tissues) or
cellularly (such as cell sorting).  The problem with such manipulations
is that they may induce stress signalling pathways that alter the
natural protein profile.
<p>
Subcellular fractionation from homogeneous cell types or from tissue
are useful.  Centrfiugation has been used to achieve physically based
separations.  Other techniques include the resistance of subcellular
structures to extraction buffer components, such as actin and intermediate
filaments and nuclei to nonionic detergents;  and of the resistance
of intermediate filament proteins, vimentin, and nuclear proteins to
the weakly ionic detergent sodium deoxycholate (actin is susceptible
to solubilization).
<p>
Removal of high abundance proteins (albumin in serum, fibrinogen in
plasma) may be necessary in prefractionation.  Also reduction of salts
may be necessary.  Dialysis, gel filtration, TCA precipitation or
centrifugal ultrafiltration (Centricon).  Excess lipid, particularly
in lipid-rich tissues such as adipose and brain tissue, can be removed
by acetone precipitation.  Polysaccharides can obstruct gel pores;
typically ultracentrifugation can remove the higher MW polysaccharides.
Lower MW polysaccharides can be removed by TCA, ammonium sulfate,
or phenol/ammonium acetate precipitation.  As for polynucleotides
they can be removed by nucleases, addition of carrier ampholytes
followed by ultracentrifugation, or by a basic polyamine at high pH.
<p>
Preparative IEF systems such as Bio-Rad's Rotofor, Amersham Bioscience's
Isoprime, and Gradipore's Gradiflow are useful for high capacity applications.
The Rotofor rotates during its work but the isoelectric zones are not
stable nor are proteins well-resolved.  The Isoprime system is a
multicompartment system with each compartment separated by gel membrane
and possessing special immobilines at a certain pH;  fractions are high
quality, but the sample usually already partially purified for large
scale applications, and does not work well with crude extracts.
A smaller scale version of solution IEF is by Zou and Speicher, and
a microscale solution or <q>nusol</q>  IEF technique has
been developed.
<p>
<b><q>Elusive</q> Proteins</b>.  Proteins may fail to enter the
IEF gel because of several factors.  Size may be the most significant.
Spectrins are large filamentous proteins (280 kDa) which do not appear
on the 2-D map unless active sample rehydration is done.  Some proteins
lose solubility and fail to enter the gel.  The use of thiourea has made
it possible to solubilize many proteins, particularly polymeric
ones like tubulin and fibronectin.
<p>
Detergents also improve solubilization.  CHAPS-containing solutions
with urea and thiourea are effective at solubilizing high MW integral
membrane protein than are SB3-10-containing solutions.  The urea has
to be reduced to 5 M when SB3-10 is included because of its incompatibility.
Molloy et al. have used detergent combinations of 2% CHAPS, 2% SB3-10
with 1% ASB14.  ASB14 with 7 M urea and 2 M thiourea produces larger
spots on 2-D gels, indicating a greater extraction;  this may have been
due to the greater urea concentration.  A buffer containing
2/7 thiourea/urea + 4% CHAPS, and one containing all those +
NDSB256 (Calbiochem) yielded many more spots in one study involving
2-D maps.  The alkylaryl aminosulfobetaine C80 failed to extract
a number of membrane-associated proteins, although it was effective
at extracting the water channel and H<sup>+</sup>-ATPase complexes
in plant cell membranes.  Washing with sodium carbonate at pH 11
removes peripheral membrane proteins.
<p>
Organic solvents can be used to extract proteins as well.  <i>E coli</i>
was lyophilized and then extracted in 1:1 (v/v) chloroform:methanol.
The extract was evaporated and resuspended in ASB14 solubilizing buffer,
and new hydrophobic proteins were observed.  A lipid smear was present
on 2-D gels and highly hydrophobic proteins could not be resolved.
Without the solvent, these proteins were resolved.
<p>
<b>Prelabeling of proteins</b>.  The first attempts to label proteins
involved radiolabeling followed by autoradiography.  Fluorescent dyes
have now replaced radiolabels.  The first was monobromobimane and
2-methoxy-2,4-diphenyl-3(2H)-furanone (MDPF).  MPDF reacts with primary
amines and can alter charge/pI, so cannot be used prior to IEF.
Monobromobimane reacts with free sulfhydryls, which makes it better for
IEF;  a problem however is that not all proteins have cysteines and
many proteins are variable in number, so it can not be used to quantitate
proteins.  PerkinElmer UK is currently developing <q>Superbright</q>
as an alternative to monobromobimane, and Amersham Biosciences has
produced a series of Cy dyes that bind to sulfhydryl groups
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Burkitt WI, Derrick PJ, Lafitte D, Bronstein I
 </span>
 <span class="year">
(2003)
 </span>
 <span class="jname">
Biochem. Soc. Trans.
 </span>
 <span class="jvol">
31,
 </span>
 <span class="pages">
985-989.
 </span>
<p class="reftitl">
Protein-Ligand and Protein-Protein Interactions Studied by
Electrospray Ionization and Mass Spectrometry
<p class="abstract">
<b>ABSTRACT</b>.
Electrospray ionization has made possible the transference of non-covalently bound complexes from solution
phase to high vacuum. In the process, a complex acquires a net charge and becomes amenable to
measurement by MS. FTICR (Fourier-transform ion cyclotron resonance) MS allows these ions to be measured
with sufficiently high resolution for the isotopomers of complexes of small proteins to be resolved from
each other (true for complexes up to about 100 kDa for the most powerful FTICR instruments), which is of
crucial significance in the interpretation of spectra. Results are presented for members of the S100 family
of proteins, demonstrating how non-covalently bound complexes can be distinguished unambiguously from
covalently bound species. Consideration relevant both to determination of binding constants in solution
from the gas-phase results and to the elucidation of protein folding and unfolding in solution are discussed.
The caveats inherent to the basic approach of using electrospray and MS to characterize protein complexes
are weighed and evaluated.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Standing KG
 </span>
 <span class="year">
(2003)
 </span>
 <span class="jname">
Curr. Opin. Struct. Biol.
 </span>
 <span class="jvol">
13,
 </span>
 <span class="pages">
595-601.
 </span>
<p class="reftitl">
Peptide and Protein <i>De Novo</i> Sequencing by Mass Spectrometry
<p class="abstract">
<b>ABSTRACT</b>.
Although the advent of large-scale genomic sequencing has greatly simplified the 
task of determining the primary structures of peptides and proteins, the genomic 
sequences of many organisms are still unknown. Even for those that are known, 
modifications such as post-translational events may prevent the identification 
of all or part of the protein sequence. Thus, complete characterization of the 
protein primary structure often requires determination of the protein sequence 
by mass spectrometry with minimal assistance from genomic data  de novo protein 
sequencing. This task has been facilitated by technical developments during the 
past few years: soft ionization techniques, new forms of chemical modification 
(derivatization), new types of mass spectrometer and improved software.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Lill J
 </span>
 <span class="year">
(2003)
 </span>
 <span class="jname">
Mass Spec. Rev.
 </span>
 <span class="jvol">
22,
 </span>
 <span class="pages">
182-194.
 </span>
<p class="reftitl">
Proteomic Tools for Quantitation by Mass Spectrometry
<p class="Para1">
<b>REVIEW</b>.  Protein identification and characterization are the initial
steps in a proteomic strategy, but quantitation also becomes important.
Ion current intensities do not always correlate.  For one thing, not
all peptides ionize equally, and so within a proteome, one protein
cannot be said to be expressed relative to another protein.
Moreover some differences are subtle or slight, sufficient to produce
a biological effect, but not subject to analytical distinction if
mass spectral intensities are the only data available.
<p>
Quantitation of protein expression has been done using Westerns,
autoradiography of metabolically labeled samples, fluorography, and
protein stains in gels.   In addition to 2-D gel/MS, activity-based
profiling helps in at least the detection activities.
<p>
<b>Relative Quantitation</b>.  The relative abundance of proteins is
of concern for drug discovery.  Comparison of diseased and control cells
or tissues is involved. 2DGE works well in this regard.  Samples are loaded
with equal amounts of protein that allow relative comparisons between
proteomes and not just within them.  2DGE is relatively reproducible.
Quantitatively the best stains provide differences of expression that
are 3 orders of magnitude;  low abundance proteins are undetectable.
Other disadvantages are that protein spots may not be homogeneous, or
some proteins are distributed in several bands.  From run to run, and
protein to protein there may be variable staining intensity.
Post-translational modifications can often be the cause of streaking
of spots.
<p>
<b>Metabolic Labeling</b>.  Media containing <sup>15</sup>N can be used
in a treatment sample while normal media be used in the untreated control.
This can compare treatments, or treated vs. untreated.  The two sets
are equalized either by cell number or cell weight, and then pooled.
Sample pooling ensures than any processing that causes losses occurs
equally, so relative expression information is not lost.  <sup>13</sup>C
is another isotope for which different incorporation media can be used.
Stocklin et al. have used a technique where one sample is grown in
99.98% <sup>12</sup>C (<sup>13</sup>C-depleted); an internal standard
is monitored as well.  The two labeled peptides show a non-natural
isotopic distribution in the MALDI-TOF.  Several amino acids are
available that have stable heavy isotopes.  Generally for a total proteome
analysis, the most abundant amino acid should be selected as the label
since it will create significant mass difference between the compared
proteomes.  A specific amino acid might be chosen if a peptide of interest
is targeted and a specific amino acid is present.  SILAC (Stable Isotope
Labeling by Amino Acids in Cell Culture)  is basically the use of
heavy stable isotopes for labeling, and use of these amino acids does
not alter cell metabolism.  Ong et al. have reported that 5 cell doublings
are sufficient to completely (97% or 31/32) of molecules.  The use
of isotopic incorporation of <sup>15</sup>N or amino acids is fairly simple
to do.  If collision-activated dissociation is done, the fragmentation
is not distinct between <sup>15</sup>N and <sup>14</sup>N.
<p>
<b>Chemical Labeling</b>.  Metabolic labeling is only possible with
cell cultures.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Andersen H, Jensen ON, Moiseeva EP, Eriksen EF
 </span>
 <span class="year">
(2003)
 </span>
 <span class="jname">
J. Bone Miner. Res.
 </span>
 <span class="jvol">
18,
 </span>
 <span class="pages">
195-203.
 </span>
<p class="reftitl">
A Proteome Study of Secreted Prostatic Factors Affecting Osteoblastic
Activity:  Galectin-1 Is Involved in Differentiation of Human Bone Marrow
Stromal Cells
<p class="Para1">
PC3 cells are a human prostate cancer cell line that can be grown in
MEM with 10% FBS.  Human bone marrow stromal cells were isolated by
Lymphoprep and plated at 10<sup>6</sup>/ml in MEM + 10% FBS.
PC3-conditioned medium (48 h in MEM with 0.1% BSA in cells 80% confluent)
was pooled, centrifuged to remove debris, and then precipitated with
25% (v/v) TCA on ice 30 min.  Precipitated protein was centrifuged and
the pellet washed with ice-cold 96% EtOH (to remove TCA) and then
resuspended in 1 ml of 9 M urea, 2% (v/v) Triton X-100, 0.13 M DTT,
8 mM PMSF, traces of bromophenol blue, and 2% (v/v) immobilines pH 3-10.
Samples are stored at &minus;80&deg; until analysis by 2D-PAGE.
<p>
First-dimensional IEF was using pH 3-10 linear immobiline strips:
these are rehydrated 15-20 h at room temp in 0.4 ml protein sample,
and then focusing done with 500 V for 3 h, 3500 V for 5 h, followed by
3500 for 20 h in a gradient mode (at 22&deg;).  Focused strips are
rinsed twice 15 min each in 10 ml 0.1% Tris HCl, pH 6.8, 5.5 M urea,
0.3% glycerol, 35 mM SDS, and 65 mM DTT (in the 2nd rinse, DTT is replaced
by 14 mM iodoacetamide and traces of bromophenol blue).  These are transferred
to ExcelGel XL SDS 12-14% gradient gels and electrophoresis done at 15&deg;
using the Excel SDS Buffer Strips for horizontal electrphoresis with
maximum 1000 V (20 mA) for 45 min.  The strips are removed and electrophoresis
resumed for 2 h.  Proteins are illuminiated by silver staining:  fix in
45% MeOH + 5% acetic acid 30 min, then double distilled water overnight;
sensitize with 0.02% sodium thiosulfate 2 min, incubate in 0.1% silver
nitrate 30 min, develop with 0.04% formalin and 2% sodium carbonate 10 min;
stop with 1% acetic acid.
<p>
Spots ranging from 5-30 kDa, 30 in all, were excised, destained with
acetonitrile, then 100 mM ammonium bicarbonate, then dried.  Pieces
were re-swelled in 30 &micro;l 100 mM amm. bicarb. with 10 mM DTT for 45 min
at 56&deg; (reduction), then 55 mM iodoacetamide in 100 mM amm. bicarb.
for 30 min (alkylation), then washed 1:1 MeCN:100 mM amm. bicarb.  80 &micro;l
MeCN was added to shrink gel pieces before vacuum drying.  Digestion of
proteins with 15.5 ng/&micro;l trypsin in 50 mM amm. bicarb. overnight at
37&deg;.  Peptide extraction was by 25 &micro;l 5% formic acid (5FA) 10 min
and 25 &micro;l MeCN for another 10 min; this procedure was done twice
and extracts pooled, dried under vacuum, and then resuspended in 20 &micro;l
5FA.  GelLoader tips were given a slurry of 100-200 &micro;l
Poros 50R2 resin in 1:1 5% formic acid:MeCN to make microcolumns, which
were washed in MeCN followed by 5FA.  3 &micro;l of peptide was
mixed with 15 &micro;l 5FA, followed by washing with 15 &micro;l 5FA.
CHCA matrix in MeCN was then used to elute the columns directly on to
MALDI targets.  A Bruker-Reflex MALDI-ToF was used to get spectra (200
laser shots with internal calibration of autolytic tryptic fragments at
<i>m/z</i> 842.51 and 2211.10.  The database search tool MASCOT and
ProFound software were used for identification of peptides (see
<a href="http://www.jbmr-online.com/fulltext/01802/01950/www.matrixscience.com">http://www.jbmr-online.com/fulltext/01802/01950/www.matrixscience.com</a>
and <a href="http://www.jbmr-online.com/fulltext/01802/01950/www.proteomics.com">http://www.jbmr-online.com/fulltext/01802/01950/www.proteomics.com</a>).
<p>
Three 2-D gels were analyzed independently to ensure reproducibility of
the profiles.  MALDI-ToF spectra were searched for all 30 spots excised.
Spot #3 was identified as galectin-1, and #26 as cyclophilin A.
Spots #1, 2, 4, 9, 13, 19, 20, 21, and 22 were identified as fragments
of the BSA that had been in the culture medium.  The remaining spots were
not identified largely because of interference from the BSA and largely
because the protein amounts were too low to be analyzable.
<p>
A human cDNA library was screened to obtained the full size of galectin-1.
A Pfu polymerase was used in the PCR with primers featuring 5' <i>Nco</i>I
and 3' <i>Bgl</i>II sites was used in the amplification.  The amplified
product was inserted into the <i>Nco</i>I-<i>Bgl</i>II site of pQE60
(Qiagen).  A recombinant with modified COOH-terminus was obtained
(ArgSerHis<sub>6</sub> at the tail).  E. coli was transformed with the
plasmid and expression controlled by IPTG induction (2-3 h).  The protein
is purified with a HisTrap (AmershamPharmacia);  in PBS it is refined on
PD10 columns, concentrated on Centriprep-3, and frozen to prevent multimeric
formation.  Galectin-1 is a 14 kDa protein and found in many mammals.
It will bind to matrix proteins laminin and fibronectin in a
&beta;-galactoside-dependent way in ELISA.  The recombinant galectin-1
forms dimers similar to the native form.
<p>
10 and 1000 ng/ml alone or in combination with IGF-I (10 ng/ml) was
used to treat hBMS cells (IGF-I is used because high plasma levels
of this hormone are associated with increased risk of developing malignant
prostate cancer).  Proliferation of hBMS cells increased 3-fold in
in PC3 cell-conditioned medium as shown by [<sup>3</sup>]H-thymidine.
But recombinant galectin-1 alone or in combination with IGF-I had no effect
on proliferation;  in contrast, proliferation was inhibited by up to 70%
compared to control cells with 1000 ng/ml alone, although this was
blocked by the presence of IGF-I.  When 100 mM lactose was added to the
PC3-conditioned medium, proliferation was increased but to only 2.6-fold,
not significantly different from untreated medium.
<p>
The conditioned medium and recombinant galectin-1 were then used in
differentiation assays of hBMS cells.  ALP activity is increased 1.7-fold
in hBMS cells with CM, and increased 1.3-fold in medium with 10 and 1000 ng/ml
recominbant galectin-1.  These increases were inhibited by IGF-I up to
101 and 91%, resp.  IGF-I alone increases ALP by 1.3-fold.  When cells
are treated with CM containing 0.1 M lactose, hBMS ALP activity goes up
1.6-fold, the same as for CM from untreated cells.
<p>
Osteocalcin secretion is reduced 81% by conditioned medium.  Recombinant
galectin-1 at 10 and 1000 ng/ml reduces secretion by 93% and 58%, resp.
In contrast IGF-I alone increases osteocalcin up to 130%, and the combination
with recomb. galectin-1 produces cells with 1.2-fold increase in secretion
at 10 ng/ml, but a reduced secretion at 82% of control at 1000 ng/ml.
Lactose in conditioned medum gave a secretion of 94% of control, effectively
only slightly different.  Other results show that the presence of lactose
in medium prevents the adhesion of PC3 cells to laminin, fibronectin, and
collagen type I.
<p>
Galectin-1 can modulate differentiation in a large number of cells, and
has been shown to be involved in fibroblast-to-myocyte differentiation.
Galectin-1 had a counteracting effect with IGF-I in the hBMS cells:
IGF-I is known to induce tyrosine phosphorylation cascade leading to
transcriptional activation.  Galectin-1 binds to CD45 in Jurkat T cells
and Burkitt lymphoma B cells; CD45 is a receptor-type protein tyrosine
phosphatase, and galectin-1 binding inhibits the activity.
<p>
Galectin-1 stimulated ALP and inhibited osteocalcin secretion, consistent
with its possible role in promoting mineralization around metastatic
tissue.  ALP produces hydroxyapatite while osteocalcin delays its nucleation
and may be a chemoattractant for osteoclasts.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Westbrook J, Feng Z, Chen L, Yang H, Berman HM
 </span>
 <span class="year">
(2003)
 </span>
 <span class="jname">
Nucl. Acids Res.
 </span>
 <span class="jvol">
31,
 </span>
 <span class="pages">
489-491.
 </span>
<p class="reftitl">
The Protein Data Bank and Structural Genomics
<p class="Para1">
PDB (http://www.pdb.org/) has grown from an archive of 3D structural data
having 7 files in 1971 to one with 18,800 in late 2002.  Earlier work
with target databases (http://www.genome3d.org/) has led to a centralized
target registration database (TargetDB, http://targetdb.pdb.org/).
Targets are loaded into a relational database with sequences from
experimentally determined structures.  All or subsets of sequences
can be searched with FASTA sequence method.  A simple search
permits queries of target data including the contributing site,
protein name, sequence, project tracking, last mod date.  Search
results are reported in HTML, FASTA file, or XML docs.
<p>
The International Task Force on Deposition, Archiving and Curation
of Primary Information recommends all depositions in structural genomics
include information normally found in Materials and Methods sections
of any article reporting structure determination.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Millea K, Chakraborty AB, Wall DB, Cohen SA, Krull IS, Berger SJ
 </span>
 <span class="pages">
Presented at ASMS, Montr&eacute;al, Canada, 8th-12th June 2003
 </span>
<p class="reftitl">
A Multidimensional LC/ESI-TOF MS Prefractionation Approach for The
Analysis of Intact Proteins in Complex Proteomes
<p class="Para1">
A Waters 2796 sample handling and pump system was configured to
a 4.6 x 35 mm nonporous sulfopropyl column.  A 1525&micro; binary
pump was used to pump a gradient to RP columns (2.1 &times; 50 mm)
Symmetry300 C<sub>4</sub>.  A four-solvent AutoBlend system was
set up to allow control of both pH and ionic strength.  Solvent A
was 0.1 M acetic acid, B was 0.1 M sodium acetate, C is 1.25 M NaCl
in 12.5% MeCN, and solvent D is 12.5% MeCN.  A and B are used to
control pH and C and D used to control ionic strength.
The first dimension program loads the SCX column and then elutes
with salt steps at a constant pH 4.  At the end the pH is raised to
remove proteins not affected by salt but ionically held.  The
2796 is fitted with a pH and conductivity meter to monitor changes.
The period for each step is the sum of the salt divert time
and the time for a gradient in the 2nd dimension of the step.
At the end we have several plots:  one a TIC of ions in the MS,
another of the salt steps for the SCX colum, another for each
RP gradient of the 2ndD, another of the pH change of the 1stD,
another two of the valve changes for the 2ndD column select and
of the salt diversion.
<p>
12 mg of <i>E. coli</i> cytosol was acidified with 20% acetic acid
which precipitates nucleic acids and other debris, removed by
centrifugation.  The supernatant is usally richer in smaller
polypeptides.  This supernatant was then run on 2DLC with the SCX
and RP column.  Spectra were acquired at 2 seconds per scan over
a <i>m/z</i> 500-3000 range.  Fractions from the post-valve split
were collected at ~3 min intervals.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Craig R, Beavis RC
 </span>
 <span class="year">
(2003)
 </span>
 <span class="jname">
Rapid Commun. Mass Spectrom.
 </span>
 <span class="jvol">
17,
 </span>
 <span class="pages">
2310-2316.
 </span>
<p class="reftitl">
A Method for Reducing the Time Required to Match Protein Sequences
with Tandem Mass Spectra
<p class="abstract">
<b>ABSTRACT</b>.
An algorithm for reducing the time necessary to match a large set
of peptide tandem mass spectra with a list of protein sequences
is described. This algorithm breaks the process into multiple steps.
A rapid survey step identifies all protein sequences that are reasonable
candidates for a match with a set of tandem mass spectra. These
candidates are then used as models, which are refined by
detailed analysis of the set of tandem mass spectra for evidence
of incomplete enzymatic hydrolysis, non-specific hydrolysis and
chemical modifications of amino acid residues resulting from either
post-translational modifications or sample handling. Compared
with current one-step methods for matching proteins to mass spectra,
this multiple-step method can decrease the time required for the
calculation by several orders of magnitude.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Suckau D, Resemann A, Schuerenberg M, Hufnagel P, Franzen J, Holle A
 </span>
 <span class="year">
(2003)
 </span>
 <span class="jname">
Anal. Bioanal. Chem.
 </span>
 <span class="jvol">
376,
 </span>
 <span class="pages">
952-965.
 </span>
<p class="reftitl">
A novel MALDI LIFT-TOF/TOF mass spectrometer for proteomics
<p class="Para1">
<b>REVIEW</b>.  It is estimated that the number of expressed proteins
is &gt; 100,000 which by far exceeds number of estimated genes
(ca. 30,000).  Thus systems to determine sequence and expression level
of mRNA and proteins are being sought.  There is also a poor correlation
between protein and mRNA levels, so quantitating mRNA is no clue about
quantitating protein.  Posttranslational modifications also mean that
intensive study of the proteins themselves is necessary.
<p>
Most widely adopted strategy is 2D-gel electrophoresis combined with MS.
Spots in gels are usually digested in situ (typically with trypsin),
and peptides eluted/extracted and prepared on a MALDI plate because of
its high throughput and good sensitivity in femtomole (sometimes subfemtomole)
range.  Peptide mass fingerprint (PMF) used to screen protein sequence database.
Predicted masses of peptides from proteins in the database are matched
to the actual masses obtained from the PMF.  A scoring is done.  Success
of identification depends on the following variables:
<ul>
<li>database size and its intrinsic error rate
<li>count of matching peptides and their predicted masses
<li>accuracy of the peptide masses provided by MS
<li>control of the digest chemistry
</ul>
<p>
The best results come with a bigger count of peptides and a good MS
calibration.  Poorer results are seen if mass tolerance is increased
and there is more chemical heterogeneity.  More problems arise when:
<ul>
<li>proteolysis is incomplete
<li>there are non-specific cleavages
<li>side chains are modified that affect the mass (e.g. oxidation of Met,
deamidation, acrylamidation of Cys)
</ul>
<p>
When PMF data are insufficient, MS/MS fragment ion analysis will improve
the chance of success.  Problems with PMF alone can occur when:
<ul>
<li>There are a low number of digest peptides in the standard mass range
of 800-3500 Da, probably because the protein itself is of low MW (&lt; 15 kDa).
When cleavage sites are not distributed to give good spectra, this is also
a problem.  Also suppression of peptides is seen in the matrix/contaminant
range.  Low amounts of protein and high levels of detergents and other
preparation materials can lead to poor PMF spectra.
<li>The EST or genomic databases being screened have high error rates,
usually &ge;0.3%, which for a protein of 400 amino acids can mean as much
as 2 amino acids in error at 0.3%, or 4 amino acids at 1% error rates.
Any sequence error can cause frame shifts as well.  Moreover, peptides
whose coding spans the splice between two exons are usually not matched.
<li>Protein idents based on mass spectra representing a set of proteolytic
petides from several proteins cannot be based only on PMFs.  These include
1-D chromatography or multidimensional chromatography of complex protein
mixtures.  Sample complexity is too hight to be certain that peptides
correspond to proteins.  Isotope-coded affinity tags (ICAT) and diagonal
chromatography are being used to make identification possible.
</ul>
<p>
MALDI-MS/MS tends to generate poor fragment ions compared to ESI.
Ions with <i>z</i>=1 are more common in MALDI whereas ESI generates
<i>z</i>&ge;2 ions.  Sequence-specific fragments are more easily generated
from multiply protonated ions.  <q>Hot</q> MALDI matrices are
used in post-source decay (PSD) for metastable fragmentation of MALDI
generated ions.  It has sensitivity to the mid-attomole range and can
characterize modifications such as phosphorylations.
<p>
MALDI-TOF/TOF are colinear arrangements of two TOF mass analyzers, each
with ion source accelerating and focusing ions.  TOF1 accelerates
and precursors are selected and fragmented, and fragments move to source
of TOF2 to be accelerated and mass analyzed.  Two MS/MS modes are available:
LID (laser-induced dissocation of metastable ions) for protein ident
and CID (collision-induced dissocation) for de novo sequencing, and for
resolving Leu/Ile ambiguity.
<p>
The <b>MALDI LIFT TOF/TOF</b> (Ultraflex TOF/TOF from Bruker) is a gridless
MALDI source with delayed extraction electronics, high-resolution timed
ion selector, <q>lift</q> to raise potential energy of ions,
stage for focusing velocity and a post-acceleration (source 2), a post lift
metastable suppressor (PLMS), gridless space-angle and energy focusing
reflector, fast ion detector for linear and reflector mode.
<p>
Ion source takes target plates same size and shape of microtiter plates.
Anchor chip plates of 384 and 1536 hydrophilic anchors enhance sensitivity
10-fold.  Image recognition does plate positioning, and target to
target movement time is 300 ms.
<p>
In MS mode source voltage can be up to 15 kV in linear, reflector,
negative and positive.  A gated MCP used in linear mode; gating prevents
saturation from low-mass ions and neutrals.  In reflector mode a fast
dual MCP used with microchannels having 5 &micro;m diameter and 12&deg;
bias angle (better subnanosecond pulse widths for single ions).
Optimized for symmetrical peaks and minimal ringing, better for
finding exact mass using software algorithms such as SNAP.
<p>
Pulse voltage modulation of delayed extraction permits best resolution
over the 500-4000 Da range.  At lower mass range resolution limited
by detection system adding constant signal width to ion signal.
By increasing laser fluence to provide larger count of precursor ions
per shot, high fragment ion yields are increased.  A low initial
accelerating voltage of 8 kV permits longer flight times for fragmentation
to occur.  Note that fragments formed after acceleration have same
velocity as precursor.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors" id="wu-2003">
Wu CC, MacCoss MJ, Howell KE, Yates 3rd JR
 </span>
 <span class="year">
(2003)
 </span>
 <span class="jname">
Nat. Biotechnol.
 </span>
 <span class="jvol">
21,
 </span>
 <span class="pages">
532-538.
 </span>
<p class="reftitl">
A Method for the Comprehensive Proteomic Analysis of Membrane Proteins
<p class="Para1">
The objective of proteomics is a systematic identification of all proteins
expressed in a cell or tissue, including posttranslational modifications.
This includes membrane proteins.  Integral membrane proteins are usually
extracted apart from soluble proteins using alkaline pH conditions;
electron micrographs show that high pH prevents resealing of membrane
structures after mechanical agitation which favors presence of membrane
sheets with free edges.  To solubilize the membrane proteins, detergents,
organic solvents, and organic acids can be used with loss of native structure.
If instead protease are used to cleave extramembrane segments, native
structure is preserved.
<p>
Rat brain was obtained from halothane-killed animals and regions separated
(cortex, cerebellum, striatum/hypothalamus, hippocampus).  These were
homogenized in 10 volumes of ice-cold 100 mM potassium phosphate, pH 6.7
containing 5 mM MgCl<sub>2</sub>, 0.25 M sucrose with 30 strokes of tight
Dounce homogenizer.  A 15 min, 4&deg; spin at 3000 <i>g</i> was done,
and supernatant adjusted to 1 mg/ml with buffer (protein assay by Lowry
H Protein from Bio-Rad).  Golgi fraction were also prepared from livers
of cycloheximide-treated rats (see Taylor et al., <i>Mol. Biol. Cell</i>
<b>8</b>: 1911, 1997).  Homogenates with 500 &micro;g protein were diluted
with equal vol. of 0.1 M potassium phosphate pH 6.7 and spun 18000 <i>g</i>
for 30 min at 4&deg;.  The pellet is resuspended to 1 mg/ml in 200 mM
sodium carbonate pH 11 by five passes through an insulin syringe; it
is left on ice for an hour.  Urea is then added to 8 M, and then reduced
and alkylated.  5 &micro;g proteinase K is added and sample incubated 3 h
at 37&deg; in Thermomixer (Brinkmann).  Another 5 &micro;g proteinase K
is added and incubated 37&deg; for 90 min.  The reaction is quenched with
formic acid (to 5%) and microfuged at 18,000 <i>g</i> for 4&deg; 15 min.
Golgi membranes were worked up according the method described by Washburn
et al.
<p>
<b>MudPIT</b>.  Digests are pressure-loaded on to a 5 cm of a 250 &micro;m
id with 2 &micro;m filtered union (Upchurch) and containing 5 &micro;m
Polaris C18-A (Metachem), serving as a desalting column.  The loaded column
is washed with 5% MeCN + 0.1% formic acid, and proteins eluted into the
rear of a triphasic 100 &micro;m id capillary with 5 &micro;m pulled tip
and packed from the tip as follows: 7 cm of 5 &micro;m Aqua C18 (Phenomenex);
3 cm of 5 &micro;m Partisphere SCX (Whatman), and 3 cm 5 &micro;m hydrophilic
interaction chromatography material (PolyLC).  The column is then brought
into line with quaternary system and eluted over 12 steps.  Solvent A
is 5% MeCN + 0.1% FA, solvent B is 80% MeCN + 0.1% FA, solvent C is
500 mM ammonium acetate + 5% MeCN + 0.1% FA.  The program was:
<ol>
<li>100 min from 0 to 100% solvent B
<li>3 min of solvent A at 100%
<li>2 min each of 10, 15, 20, 25, 30, 35, 40, 45, 50 and 60% solvent C
<li>10 min of a gradient from 0-15% solvent B
<li>97 min gradient from 15-45% solvent B
<li>For the final step the gradient contained 3 min of 100% solvent A, 20
min 100% solvent C, a 10 min gradient from 0-15% solvent B, and 107 min
gradient from 15-70% solvent B.
</ol>
<p>
Elution is directly to electrospray into a ThermoFinnigan LCQ-Deca MS with
a 2400 V spray voltage.  The mass range is <i>m/z</i> 400-1400 with
three data-dependent MS/MS spectra at 35% normalized collision energy
was done continuously.
<p>
1610 proteins were identified with &gt;95% confidence with a two-peptide
minimum.  28.2% of these (454 proteins)  had predicted transmembrane domains
(TMD) using the analyzer TMHMM 2.0.  Many had only 1 TMD, but most known
proteins have 2-23 TMD (54% of proteins).  Reports on genome analysis predict
that 20-30% of all ORFs encode integral membrane proteins.  This range
agrees with the 28.2% of proteins with TMDs.
<p>
The nonspecific protease approach is related to redundant sequence coverage.
Most proteins had &lt;20% sequence coverage.  79 modifications on 51
proteins (24 assigned to 18 membrane proteins) were identified.  These
included 22 phosphorylations (16 predicted by NetPhos 2.0), 14 monomethylations,
39 dimethylations, and 4 trimethylations or acetylations.  Assignments were
made based on mass shifts seen in MS/MS.
<p>
Membrane proteins (STX2, AQP4, GRID2, ATP2b2) and soluble proteins (NVP2,
VSNL1, CALM1) were particularly selected to examine the effectiveness of
proteomic analysis.  STX2 (syntaxin 2) is a SNARE protein with central
role in exocytosis of synaptic vesicles.  It contains 1 TMD at the COOH
end.  Its sequence coverage was 54.2%.  Three peptides show phosphorylation
of Ser14 with 99.7% confidence.  Tandem MS of nontryptic +2 phosphopeptide
KDSDDEEEVVHVD (S is modified) shows a loss of [M-98]<sup>2+</sup>
from the precursor.  Fragment ions of the b- and y-ion series are
identifiable completely.
<p>
AQP4 (aquaporin 4) is the predominant water channel in brain expressed
only on the basolateral surface of epithelial cells.  It has 6 predicted
TMDs.  Coverage was 26.3%, and localized redundant coverage was 45.7%
on COOH soluble domain.  Five peptides overlap into the C-terminal-most
predicted TMD (thus the domain is exposed and may not be embedded).
Ser180 and Ser276 are thought to control the mechanism of action and cell
surface localization when phosphorylated., but were not detected by this
work because coverage was missing.  Ser285 was not previously reported to
be phosphorylated, but three peptides identified its modification with 98%
confidence.  The tandem MS of the +2 peptide DNRSQVETEDLILKPGVV showing
a base peak that has a loss of -H<sub>2</sub>PO<sub>4</sub>.
<p>
Protease protection experiments help determine sidedness as long as membrane
compartments remain sealed.  Complete digestion with the high pH-proteinase K
method was done in 3 steps.  The protease is added at neutral pH first.
Intact membrane structures are re-isolated in the 2nd step.  In the 3rd,
high pH-protease unseals membranes and digests luminal domains.  Golgi
membrane proteins are most usefully analyzed by protease protection.
<p>
SIAT1 (sialyltransferase 1) is a resident Golgi type II membrane protein
involved in terminal sialylation of N-linked carbohydrate groups of
glycoproteins and glycolipids.  Identification was made with 47% coverage.
With the use of high pH-protease all peptides over the COOH domain were
found.  A +2 HLNEGTDEDIY*LFGK peptide identifed Tyr388 phosphorylation with
90% confidence.  Fragment ions b10, b11, and y5 identified the localized
phosphorylated group.  The majority of SIAT1 protein is located on the
luminal side and so were protected against digestion from protease before
high pH treatment.
<!-- ================================================================ -->
<p class="tags">

<p class="ref">
 <span class="authors">
Wright ME, Eng J, Sherman J, Hockenbery DM, Nelson PS, Galitski T, Aebersold R
 </span>
 <span class="year">
(2003)
 </span>
 <span class="jname">
Genome Biol.
 </span>
 <span class="jvol">
5,
 </span>
 <span class="pages">
R4.
 </span>
<p class="reftitl">
Identification of Androgen-Coregulated Protein Networks From The Microsomes
of Human Prostate Cancer Cells
<p class="Para1">
LNCaP cells are a continuous human prostate cancer cell line which were
treated with 10 nM R1881 androgen (methyltrienolone) for 72 hours.
Control (AS) and treated (AD) cells after growth in RPMI 1640 with
10% charcoal-treated FBS.  Microsomes were prepared and treated with
5 mM TBP for 3 h, then with ICAT reagent for 3 h and then digested overnight
with trypsin at 37&deg;.  The peptide digest is resolved on cation-exchange
column with fractionation.  53 fractions are then avidin-purified and
subjected to &micro;LC-ESI-MS/MS.  MS/MS spectra are searched against
73,600 human sequences of the NCBI nrDB.  SEQUEST was used in the search.
Fixed/static modifications include 503 Da for the Cys-light ICAT coupling
and variable modification of +8 Da for the heavy ICAT coupling.
SEQUEST output is passed through INTERACT which filters peptide
identification as correct:  Xcorr &gt; 1.5, &Delta; correlation scores
&ge; 0.1, Sp rank &le; 8, singly tryptic, and at least one Cys.
The list contained 5450 MS/MS spectra.  When filtered through
PeptideProphet, 93% of entries are &ge; 0.99 probability and 97% of
entries had 0.90 or higher.  The XPRESS quantification program was
used to analyze, validate, correct quantification.
<p>
Lysates from the AD and AS 72 h-treated cells (16 &micro;g) were also
immunoblotted with 1 &micro;g/ml mouse anti-PSA (Res. Diag Inc.
RDI-PSA85abm/RDI-PSA85abm-1), 1:1500 rabbit anti-TMPRSS<sub>2</sub>,
1:200 rabbit anti-LDLR (all Res. Diag. Inc) and 1 &micro;g/ml
mouse anti-MST-3 (BD Transduction lab clone 47), 1 &micro;g/ml
mouse anti-FAS (BDTL clone 23), 2 &micro;g/ml anti-&beta;-catenin (BDTL
clone 14), 1 &micro;g/ml anti-FasL (BDTL clone 13), and 50 ng/ml
mouse anti-Hsp60 (Stressgen clone LK<sub>1</sub>).  All cells
were then gorown as usual and labeled with Mitotracker Red CMXRos
and Alexa Fluor 488 phalloidin and DAPI.  Fluorescent images were
obtained and deconvolved.  LNCaP cells were also exposed to three
growth conditions (androgen-depleted and -treated, as well as normal
serum) then incubated with respiratory chain inhibitors for 24 h:
10 &micro;M rotenone, 50 &micro;M carbonyl cyanide
<i>m</i>-chlorophenylhydrazone (CCCP), 10 &micro;g/ml antimycin A,
5 &micro;g/ml oligomycin.  Controls were treated with EtOH 24 h (vehicle).
At time of drug addition, cell densities were 1000/cm<sup>2</sup>
for the standard cells, and 3000/cm<sup>2</sup> for the AS and AD cells,
which are equivalent cell densities.
<p>
Proteins identified in SWISS-PROT were searched at Incyte Human Proteome
Database and assigned to a class of functions.  Proteins with a GenBank
GenPept accession assignment were also searched.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Kawasaki N, Itoh S, Ohta M, Hayakawa T
 </span>
 <span class="year">
(2003)
 </span>
 <span class="jname">
Anal. Biochem.
 </span>
 <span class="jvol">
316,
 </span>
 <span class="pages">
15-22.
 </span>
<p class="reftitl">
Microanalysis of N-Linked Oligosaccharides in A Glycoprotein by Capillary
Liquid Chromatography/Mass Spectrometry and Liquid Chromatography/Tandem
Mass Spectrometry
<p class="Para1">
A Microchrom BioResources Magic 2002 LC system connected to ThermoFinnigan
TSQ 7000 triple quad was set in positive ion mode with <i>m/z</i> 800-2000
equippd with AMR nanonelectrospray ion source.  A graphitized carbon column
(GCC) of Hypercarb 5&micro; (200 &micro;m &times; 150 mm) from ThermoFinnigan
was used with mobile phase of 5 mM ammonium acetate pH 9.6 with 2% MeCN.
Sugars were reduced and eluted at 2 &micro;l/min in 18-38% gradient 40 min
(solvent B contains amm acetate with 80% MeCN).  Capillary voltage is 2000
V with metal needle and capillary temperature is 175&deg;.  Detector is
1200 V.  Argon gas is 2 mTorr.  Collision energy is &minus;25 eV and 3
second scans are collected.
<p>
EPO oligosaccharides are separated after PNGase F treatment and reduction
with borohydride (so anomers are not separated).  50 ng EPO was injected
to 200 &micro;m column and 20 &micro;g to 2.1 mm column, and were detectable
in the mass chromatogram (&gt; 10:1 S/N).
<p>
Recombinant hepatocyte growth factor (HGF) produced in NS0 cells were
studied for carbohydrate.  Reduced oligosaccharides from 200 ng HGF
were run with 11 peaks identified (two major peaks 5 and 9).
&alpha;-galactosidase treated HGF yield about six peaks, with barely
resolved peaks <i>c</i> and <i>d</i> as major.  Peak 5 was an <i>m/z</i>
1467 corresponding to [M+2H]<sup>2+</sup> ion of
[dHex]<sub>1</sub>[Hex]<sub>7</sub>[HexNAc]<sub>5</sub>[NeuGc]<sub>2</sub>,
being a fucosyl triantennary complex-type with additional Hex and two
<i>N</i>-glycoylneuraminic acids.  Peak 9 was 2+ ion with <i>m/z</i>
1394 consistent with [dHex]<sub>1</sub>[Hex]<sub>8</sub>[HexNAc]<sub>5</sub>[NeuGc]<sub>1</sub>.
Peaks 1-4, 6-8, and 10 and 11 were probably fucosyl bi-, tri-, and tetraantennary
types with 0-3 Hex and 0-3 NeuGc.  Thus the majority of N-linked glycan
types on HGF appear to be triantennary complex-type with NeuGc and additional
Hex.
<p>
&alpha;-Galactosidase will hydrolyze Gal&alpha;1&rarr;3,4,6Gal/Glc bonds
as an exoglycosidase.  Peak <i>c</i> had <i>m/z</i> 1386 2+ ion, and was
deduced to be fucosyl triantennary bearing two NeuGc, possibly produced from
FucTri-Gal<sub>1</sub>NeuGc<sub>2</sub> (peak 5) with loss of a Gal.
Peak <i>d</i> was a 2+ ion with <i>m/z</i> 1232 and is triantennary with
one NeuGc with loss of two Gal from peak 9.  Thus &alpha;-linked galactosylation
on the nonreducing ends of HGF oligosaccharides is likely.
<p>
LC-MS/MS was done on peaks 5 and 9.  Fragment spectra show clean <i>b</i>
ion series at <i>m/z</i> 528 (Hex<sub>2</sub>HexNAc<sup>+</sup>), 673
(NeuGc Hex HexNAc<sup>+</sup>), 690 (Hex<sub>3</sub> HexNAc<sup>+</sup>),
836 (NeuGc Hex<sub>2</sub> HexNAc<sup>+</sup>, 1362 (NeuGc Hex<sub>4</sub>
HexNAc<sub>2</sub><sup>+</sup>), and 1209 (NeuGc Hex<sub>8</sub>
HexNAc<sub>4</sub> <sup>2+</sup>).
<p>
Four isomers, peaks <i>a</i> to <i>d</i>, were seen at <i>m/z</i> 1467 (2+ ion).
Fragmentation was able to distinguish peaks <i>a</i> and <i>c</i>
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Osinga R, Belarbi EH, Grima EM, Tramper J, Wijffels RH
 </span>
 <span class="year">
(2003)
 </span>
 <span class="jname">
J. Biotechnol.
 </span>
 <span class="jvol">
100,
 </span>
 <span class="pages">
141-146.
 </span>
<p class="reftitl">
Progress Towards A Controlled Culture of The Marine Sponge
<i>Pseudosuberites andrewsi</i> in A Bioreactor
<p class="Para1">
<i>P. andrewsi</i> sponge samples were obtained from a Rotterdam zoo,
which originated from a single parent.  Explants with known volume
was tied to nylon fishing line on to a plastic carrier (healthy fragments
attach).  Special PVC bioreactors (880 ml) were constructed with overflow
device and magnetic stirrers and thermostated bath set at 26&deg;.
The food source was the marine diatom <i>P. tricorotum</i>;  it was cultured
in 5 L Erlenmeyers using ~100 &micro;mol m<sup>&minus;2</sup>
s<sup>&minus;1</sup> at 22&deg; for 1 week, and product stored at 4&deg;.
Food particle concentration was quite important.  Food particle concentration
in the water surrounding the sponge can be described as:
<p class="eqnserif">
 [<i>P</i>] = [<i>P</i><sub>in</sub>]&phi;(&phi; +
<i>FV</i><sub>sp</sub>)<sup>&minus;1</sup>
<p>
where [<i>P</i>] represents food particle concentration in the bioreactor,
[<i>P</i><sub>in</sub> represents the concencentration of particles in
the inflowing stream, &phi; is the flow through the bioreactor, <i>F</i>
is the filtration rate (in cm<sup>3</sup> seawater per minute per
cm<sup>3</sup> sponge volume), and <i>V</i><sub>sp</sub> is volume of all
sponge explants in the bioreactor.  Explants were added to the reactor
such that <i>V</i><sub>sp</sub> = 14.42 cm<sup>3</sup> initially.
A value <i>F</i> = 3.5 cm<sup>3</sup> SW min<sup>&minus;1</sup>
cm<sup>&minus;3</sup> sponge was assumed (this is the known average for
<i>P. andrewsi</i> from other work).  [<i>P</i>] was to be set at
2 &times; 10<sup>5</sup> diatoms cm<sup>&minus;3</sup> by varying
both &phi; and [<i>P</i><sub>in</sub>], in this case &phi; = 0.34 cm<sup>3</sup>
min<sup>&minus;1</sup> and [<i>P</i><sub>in</sub>] = 3 &times; 10<sup>7</sup>
diatoms cm<sup>&minus;3</sup>.
<p>
All sponges in these experiments grew.  Increases in explant weight
averaged 255 &plusmn; 112%.  Sponges fed on live diatoms and crude extracts
showed no difference in growth (crude extracts of diatoms were prepared
by sonicating the diatoms at 200 W (max power) for 10 min, and then filtering
on Whatman GF/F).  In about a month, explants fed crude extracts doubled
their sizes.  In public aquaria, <i>P. andrewsi</i> grow at about 160%
in 50 days (<i>n</i>=4, SD=33%), and was surviving on adsorbed organic
materials rather than living forms.
<p>
There is a question about the filtration activity of sponges relative to
their size (volume).  To avoid drastic changes, a tight control of
[<i>P</i>] is necessary.  Dissolved organics are likely taken up the
outer surface layer of the sponge body, the <b>exopinacoderm</b>.
Attempts to change the food source of the sponge from <i>P. tricornotum</i>
to the chlorophycae <i>Nannochloropsis</i> sp. resulted in sponges losing
weight rapidly, indicating a difference in the nutritive value of
the two microalgal species.  A search of the literature to compare the
two algae for their composition indicates both produce amino acids
and are rich in fatty acids.  One component that <i>Nannochloropsis</i>
was missing was the polyunsaturated fatty acid docosahexanoic acid
(DHA).  When DHA is supplemented to 4 mg per liter along with the
<i>Nannochloropsis</i> feed, there was no growth in the sponges.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Lopez MF, Pluskal MG
 </span>
 <span class="year">
(2003)
 </span>
 <span class="jname">
J. Chromatog. B
 </span>
 <span class="jvol">
787,
 </span>
 <span class="pages">
19-27.
 </span>
<p class="reftitl">
Protein Micro- and Macroarrays:  Digitizing the Proteome
<p class="Para1">
<b>REVIEW</b>.  Arrays of cDNAs sequences and oligonucleotides are in
routine use in large-scale analysis of gene expression and variations
in gene sequence.  Expression proteomics largely involves serial processing
of 2DGE.  The high throughput methods useful in nucleic acids are now
being used for identifying useful functions in proteins.
<p>
Being able to amplify DNA in vitro by PCR is a great asset in an analytical
approach.  The closest equivalent is specific affinity selection or
isolation of defined charge class by IEF from complex mixture.  This
is the best that can be done now for complexity reduction.
<p>
Proteins, like DNA, can also be labeled with fluorescent molecules
to improve the sensitivity of detection, or to multiplex detection of
several targets for easy interfacing with instrumentation and automation.
The finding that MALDI-MS can be used on oligonucleotides to resolve
single bases for sequencing shows its potential.
<p>
<b>Antibody arrays/Phage Display/Antibody Mimics</b>.  Antibodies are
relatively manipulable without loss of the ability to bind ligands.
A problem is they cannot recognize all protein isoforms or post-translational
modifications.  Nonetheless one group printed 115 antibody/antigen
pairs on derivatized glass slides with a robotic arrayer and could detect
half of the arrayed antigens and one-fifth of the arrayed antibodies
in complex mixtures at concentrations ranging from 1 to 340 ng/ml.
Others have shown similar results, and some companies have formed to
develop protein chips based on antibody arrays (Zyomyx, BioSite,
Hypromatrix &amp; Diversys).   Rather than whole antibody molecules,
which have large molecules and often lack specificity, others are using
antibody fragments, Ab mimics and phage display.
<p>
Phage display creates
antibody libraries with genetically modified phages that express the
variable fragments of Ig molecules in the capsid surface.  Companies
such as Dyax, Cambridge Antbiody Technology, and Affitech are using this
approach or its modifications.  Phage display has problems with specificity
as do conventional antibodies.  These display libraries are not limited
to antibodies and can consist of non-biological antibody mimics and diverse
molecules.  Aptamer (single-stranded oligonucleotides expressing high
affinity for protein molecules) libraries are easily synthesized
and less labile than antibodies but are subject to nucleases that might
destroy them when present in complex mixtures from body fluids use to
screen chips.
<p>
Ciphergen Biosystems pioneered a technique called SELDI in which a MALDI
target surface is modified with affinity ligands in order to capture proteins.
The target is directly inserted into a mass spectrometer for analysis.
Another approach uses surface plasmon resonance (SPR) to quantify the
target capture.
<p>
In other approaches, glass slides are activated with cross-linking agents
that react with primary amines used to attach proteins.  Protein-protein
interactions, enzyme-substrate and protein-ligand reactions using fluorescent
probes or radiolabeled ATP.  In a single experiment one protein was
identified within an array of 10,000 other proteins, which shows the utility
of using protein microarrays for large-scale, high-throughput screening.
Polyacrylamide gel pads fixed on to glass slides are being used to minimize
changes in the 3-D integrity of proteins.  Others have developed nanowell
technology to assay 117 different yeast kinases with 17 different substrates.
A novel Tyr kinase was found as a result.  In another study, a library
of about 15,000 nonredundant His-tagged protein from human fetal brain
cDNA library; the fusion proteins are arrayed onto glass and membrane
and then probed with Ig, with successful screening shown despite the likely
high amount of denaturation.
<p>
Some companies are using microfluidic printing or arrayed chemistries
to individual protein spots blotted on membranes.  Thus each spot on a
2-D gel blot is a protein chips (a 2-D macroarray).  Since the proteins
are biologically genuine (from an extract separated by pI and MW), they
include posttranslational modifications.  Proteins are membranes are stable
for indefinite periods of time.  Using a piezo-electric chemical printing
technique, several chemistries can be applied to a single spot, such
as trypsin digesting and PMF, glycosidase digestion, glycopeptide analysis,
etc.
<p>
[Fig 1 shows proteins in microarray format.  &alpha;-1 acid glycoprotein
(AAG), horseradish peroxidase, IgG, and soybean trypsin inhibitor are
arrayed on PVDF adhered to glass slide from a 384-well source plate
with range of concentration from 62.5 to 4000 &micro;g/ml in PBS.  Spotting
was done with V&amp;P Scientific manual glass slide arrayer fixed with
four rows of eight pins (32 total), ~500 &micro;m dia. spot size,
1125 &micro;m horizontal pitch and 750 &micro;m vertical pitch (where
pitch is the center-to-center spacing of spots).  Replicates of 6 are
arrayed, with a total of 168 spots.]
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Finehout EJ, Lee KH
 </span>
 <span class="year">
(2003)
 </span>
 <span class="jname">
Electrophoresis
 </span>
 <span class="jvol">
24,
 </span>
 <span class="pages">
3508-3516.
 </span>
<p class="reftitl">
Comparison of Automated In-Gel Digest Methods for Femtomole Level Samples
<p class="abstract">
<b>ABSTRACT</b>.
A comparison of automated in-gel digestion methods for low picomolar to femtomolar
levels of protein is presented. Gel spots with 4 pmol to 120 fmol of protein were stained
with either Coomassie colloidal blue or SYPRO Ruby and digested using an automated
platform. The sequence coverages and average peak intensities obtained from a
matrix-assisted laser desorption/ionization-mass spectrometry (MALDI-MS) analysis
are compared. Results show that methods using an acetonitrile extraction or digest
times greater than the standard 4 h give no significant increase in peptide sequence
coverage for automated digestion of low protein level samples. It is also shown that
digests from SYPRO Ruby-stained gels show a greater improvement upon ZipTip
cleanup than digests from Coomassie colloidal blue-stained gels. The digests from
SYPRO Ruby-stained gels are also shown to give a higher average peptide intensity
if a method with minimal gel plug washing is used.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Raffaelli A, Saba A
 </span>
 <span class="year">
(2003)
 </span>
 <span class="jname">
Mass Spec. Rev.
 </span>
 <span class="jvol">
22,
 </span>
 <span class="pages">
318-331.
 </span>
<p class="reftitl">
Atmospheric Pressure Photoionization Mass Spectrometry
<p class="abstract">
<b>ABSTRACT</b>.
Atmospheric pressure photoionization (APPI) is the last arrival
in the family of atmospheric pressure ionization (API) methods
to couple mass spectrometry (MS) to liquid-phase separation
techniques. The basic idea was to further extend the fields of
application of liquid chromatography (LC)MS to those
molecules that are not, or are poorly amenable, to electrospray
(ESI) or APCI. The present review explores the literature. After
a short introduction with an historical background and the
premises for its development, we describe the technique, its
physical principles, and the factors that affect its efficiency. The
review also presents a survey of applications in different fields.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Rabilloud T
 </span>
 <span class="year">
(2003)
 </span>
 <span class="jname">
Nat. Biotechnol.
 </span>
 <span class="jvol">
21,
 </span>
 <span class="pages">
508-510.
 </span>
<p class="reftitl">
Membrane Proteins Ride Shotgun:  A New Mass Spectrometry-Based Approach
Identifies and Characterizes Membrane Proteins On A Large Scale
<p class="abstract">
<b>NEWS</b>.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Kuyama H, Toda C, Watanabe M, Tanaka K, Nishimura O
 </span>
 <span class="year">
(2003)
 </span>
 <span class="jname">
Rapid Commun. Mass Spectrom.
 </span>
 <span class="jvol">
17,
 </span>
 <span class="pages">
1493-1496.
 </span>
<p class="reftitl">
An Efficient Chemical Method for Dephosphorylation of Phosphopeptides
<p class="Para1">
Probably 1 in 3 eukaryotic proteins are modified by phosphate at any one
time.  While there are many enzymatic treatments described, a chemical
method that is specific can be preferable and more complete.
<p>
The method described takes 100 &micro;g of lyophilized peptide in 50 &micro;l
of 50% hydrofluoric acid (HF) or 70% HF in pyridine (Fluorochem Ltd.,
Derbyshire UK).  Peptide is left standing on ice bath or at room temperature,
and 7 &micro;l aliquot is taken and dried under vacuum with a CaO and
NaOH trap to capture the HF.  The dried residue is resuspended in 20
&micro;l water for MALDI-ToF  MS.
<p>
HF is used in the final cleavage step of peptide synthesis since it
removes a wide variety of protecting groups.  It is also volatile
and easily removed  by vacuum.  It is corrosive however and toxic
to humans, so it must be trapped in alkali.  At lower concentrations
it is less toxic but is still effective at achieving nucleophilic attack.
A 50% HF has been found to remove phosphate esters of saccharides without
causing cleavage of the O-glycosidic bond, maintaining oligosaccharide
structure.
<p>
The model peptides used in these experiments included WAGGDApSGE,
Ac-IpYGEF-nh<sub>2</sub>, and GFETVPEpTG-nh<sub>2</sub>.  Using 50%
HF, cleavage of the single phosphates was completed within 3 h on an
ice bath as shown with MALDI MS spectra.  The first peptide produced
an [M+H-80]<sup>+</sup> peak (the mass of the removed phosphate ester
is 80 Da).  The second peptide produces before digestion both
[M+Na]<sup>+</sup> and [M+K]<sup>+</sup> peaks in the Shimadzu AXIMA-CFR
(20 kV acceleration in gridless electrode, with satd CHCA matrix in 50%
MeCN + 0.05% TFA, mixing of peptide and matrix not on-plate).  These
Na and K adduct peptides are present at 80 Da less.  For the third peptide,
it is present as a [M+H]<sup>+</sup>, [M+Na]<sup>+</sup>, and
[M+K]<sup>+</sup> ions.  With phosphothreonine residues, 3 h on ice
bath is insufficient, with part of the starting peptide still present.
At 1.5 h at room temperature, the reaction finishes well.
<p>
With 70% HF in pyridine, dephosphorylation was complete in 60 min on
an ice bath for all three phosphopeptides according to spectra.
<p>
Other conditions tried includes 95% TFA for 3 h on ice, with no appearance
of spectra peaks at 80 Da less.  This shows that acid catalysis alone
is not the factor by which cleavage occurs, and that it is related to
a specific affinity of fluoride anion for the oxygen-phosphorus bond.
<p>
A more complex mixture of peptides was tried.  &alpha;-casein was digested
with trypsin and the resulting peptides treated 3 h on ice bath with 50%
HF.  The pre-digest MALDI spectra show <i>m/z</i> 1661.21 and 1952.35
corresponding to VPQLEIVPNpSAEER and YKVPQLEIVPNpSAEER, respectively.
Post-digest spectra show these two peaks exactly at 80 Da less, with no
peaks of the precursors.
<p>
<b>Sulfation</b>.  For sulfated peptides/proteins, it must be considered
that the mass loss of a phosphate versus sulfate ester are nearly
identical:  sulfate loss is 79.957 while phosphate loss is 79.966, a
difference too small to resolve on MS reliably.  However, the
sulfate ester modification is much more acid labile, and so use of
the 95% TFA treatment prior to HF treatment makes it possible to distinguish
between modifications.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Zhang H, Li X-J, Martin DB, Aebersold R
 </span>
 <span class="year">
(2003)
 </span>
 <span class="jname">
Nat. Biotechnol.
 </span>
 <span class="jvol">
21,
 </span>
 <span class="pages">
660-666.
 </span>
<p class="reftitl">
Identification and Quantification of N-linked Glycoproteins Using
Hydrazide Chemistry, Stable Isotope Labeling and Mass Spectrometry
<p class="Para1">
The top-down approach in global protein analysis in proteomics unfortunately
typically catches on the most abundant proteins.  Because of these
limitations, strategies have been employed which focus more on enrichment
of proteome subsets, such as isolation/purification of organelles or
cellular substructures (ribosome, spliceosome, nuclear pore complex,
mitochondria, peroxisomes, etc.), before doing the analysis.
Chemical specificity has also been used to enrich proteomes selectively
(such as phosphates or thiols on proteins) as they have affinity for
particular ligands.
<p>
Glycosylated proteins are another chemically specific subproteome.
Carbohydrates can be O-linked to Ser or Thr or N-linked to Asn.
The usual motif for N-linked glycosylation is -N-X-S/T- where X is
any amino acid except Pro.  N-linked glycosylation is generally found
on proteins with extracellular domains, there the oligosaccharide
is likely to be located;  it also includes secreted proteins.
Many clinical biomarkers and therapeutic proteins are such glycoproteins:
Her2/neu for breast cancer, &beta;-human chorionic gonadotropin and
&alpha;-fetoprotein in germ cell tumors, PSA in prostate cancer, CA125
in ovarian cancer.
<p>
A method for quantitative glycoprotein profiling is described.
A vicinal (cis)-diol is oxidized by periodate to produce a dialdehyde.
An unwanted side reaction might occur if a primary amine and secondary
hydroxyl groups are on adjacent carbon atoms (such as the presence of
an N-terminal Ser), but this would be rare in proteins.  Different
oligosaccharides have different reaction conditions (periodate concentration,
reaction time).  15 mM sodium periodate for 60 min was used here,
intended to assure reaction with all types.
<p>
The oxidized glycoprotiens are then passed over a solid support
with hydrazide groups (Bio-Rad).
The hydrazide forms a hydrazone with the sugar, coupling it to the
support, and washing away nonglycosylated proteins.  The coupled
glycoproteins are then digested with a protease, and nonglycosylated
peptides are washed away.  The nonglycosylated peptides can be collected
for further analysis.  &epsilon;-amino groups on Lys are then converted
to homoarginine.  The &alpha;-amino groups of the glycosylated
coupled peptides are then treated with one of two isotopic forms
of succinic anhydride:  either a <i>d</i><sub>0</sub> (light)
or <i>d</i><sub>4</sub> (heavy) form, the latter with four deuterated
atoms (C/D/N Isotopes).  The N-linked glycopeptides are
then released from the
support with PNGase (specific for N-linked glycosylation sites, but
those with fucose cores are unreactive, New Eng. Biolabs).
Recovered peptides are passed through
&micro;LC-ESI MS/MS or offline &micro;LC MALDI MS/MS.
<p>
Consider a model analysis with three glycoproteins &alpha;-1-antichymotrypsin,
&alpha;-1-antitrypsin, and &alpha;-2-hs-glycoprotein.  Four glycosylation
sites were identified, although only three had been reported in the
literature (one for each glycoprotein).  An additional peptide
FN#LTETSEAEIHQSFQH represents a glycosylation site in &alpha;-1-antichymotrypsin.
Abundance ratios calculated from the isotopic ratios agreed with expected
values.
<p>
For O-linked sugar analysis, there is no enzyme like PNGase F that
can release O-linked sugars.  It is thus necessary to sequentially
remove monosaccharides with a panel of exoglycosidases until only
the Gal&beta;1,3GalNAc core is left on Ser/Thr, and then release it
with an O-glycosidase.  Since not all sugars have this core, it might
be necessary to use &beta;-elimination to release the peptide as well.
<p>
The human serum proteome was now tested.  Oxidized serum proteins were
coupled to hydrazide beads.  Samples were then analyzed on 9% SDS-PAGE
and silver-stained or glycoprotein-stained (Gel Code Blue, Pierce).
Control lanes in which proteins are run without incubation with coupling
beads show the glycoproteins to be coupled.  The post-coupling solution
was examined and the bands have disappeared, showing the effectiveness
of the method in coupling.   Bands appeared which were not coupled
but were clearly not glycoproteins.
<p>
To compare conventional serum protein analysis with the selective
subproteome method, the conventional approach used Cys-labeled
ICAT reagents to select proteins.  Both products were then subjected
to &micro;LC-ESI-MS/MS.  ICAT-labeled peptides were also examined
a 3-D chromatographic approach (cation exchange, biotin affinity,
RPLC) with 17 cation-exchange fractions then analyzed by ESI-MS/MS.
A single run of LC-ESI-MS/MS requires 2 h on the mass spectrometer;
145 unique peptides were found corresponding to 57 unique proteins
with glycopeptide capture (2.5 peptides/protein).  With ICAT,
72 unique peptides corresponding to 23 unique proteins, of which 15
were identifed also by glycopeptide capture.  A more extensive
separation protocol for ICAT-labeled proteins used 34 h mass spec time,
identifying 356 unique peptides mapping to 97 serum proteins.
Of 57 proteins identfied by glycopeptide capture, 23 of these were
<i>not</i> identified using ICAT approach.
<p>
There are 4 major conclusions:
<ol>
<li>ICAT-labeling approaches can be limited substantially by the presence
of some high-abundance proteins (with serum there are five major plasma
proteins, making up 80% of the total protein:  albumin, &alpha;-1-antitrypsin,
&beta;-2-macroglobulin, transferrin, and &gamma;-globulins).  The MS
spent one-third of its acquisition time on CID spectra of albumin (39%
of peptides identified by Cys-tagging were from albumin);  in contrast
glycopeptide capture eliminates albumin, and only 1% of the time was
spent identifying it.
<li>Proteins not identified by Cys-tagging were readily identified by
glycopeptide capture.
<li>Glycopeptide capture reduces sample complexity and thus background
and signal loss of low-abundance proteins.  2.5 peptides per protein
were seen.
<li>Presence of N-glycosylation sequence motif in identified peptides
is further validation of specific isolation and increased confidence
in results.
</ol>
<p>
Not all glycoproteins may be identified.  The coupled peptide may be
too short or too long for MS/MS analysis and is thus lost.  In this
case, perhaps alternate digestion schemes (aside from trypsin) might
cover those proteins not identified.
<p>
<b>Quantitative Analysis</b>.  Having established the principle, it
now becomes important to assess its potential in reporting quantitative
differences.  Crude membrane fractions from LNCaP prostate cancer
epithelial cell line were subjected to the protocol.  104 unique
peptides mapped to 64 unique protein (1.6 peptides/protein), and all
peptides featured the -N-X-S/T- motif.  Identified proteins were
assessed for subcellular location:  of the 64, 45 (70%) were true or
predicted transmembrane proteins.  The nontransmembrane proteins
were either secreted (7 proteins, 11%) or lysosomal (9 proteins, 14%);
these are compartments known for being rich in glycoproteins.  Only
3 (5%) proteins were considered cytoplasmic.  These proteins also
have high MW or extreme pI and would not likely be seen in a gel approach.
For example, heparan sulfate proteoglycan core protein (a basement
membrane-specific protein) with mass 470 kDa, and the acidic transmembrane
signal sequence receptor &alpha; subunit (pI 4.39) were identified.
<p>
<b>Methods</b>.  Isotopic labeling is done by washing coupled
glycopeptides in a high pH (&gt; 11) 15% ammonium hydroxide,
then 1 M methylisourea in 15% ammonium hydroxide is added at 100-fold
excess over amine groups and incubated 55&deg; for 10 min.  Beads
are not washed twice in water, and then twice with DMF:pyridine:water
at 5:1:4 (v/v/v), and then suspended in DMF:Pyr:water.  Succinic
anhydride is added to 2 mg/ml, left at RT for 60 min, washed 3 times
with DMF, 3 times with water, and 6 times with 0.1 M ammonium bicarbonate.
Peptides are released by adding 1 &micro;l PNGase F for every 40 mg
protein used and leaving overnight.  Released peptides are recovered,
dried and resuspended in 0.4% acetic acid for LC-MS/MS.  For coupling
the glycoproteins to hydrazide resin, the oxidation is first done
in 15 mM sodium periodate in pH 5.5 0.15 M NaCl and 0.1 M sodium acetate
(1 h, RT), which is washed away after.  The oxidized protein is treated
to hydrazide beads at 20&deg; for 10-24 h.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Sipkema D, Heilig HGHJ, Akkermans ADL, Osinga R, Tramper J, Wijffels RH
 </span>
 <span class="year">
(2003)
 </span>
 <span class="jname">
Mar. Biotechnol.
 </span>
 <span class="jvol">
5,
 </span>
 <span class="pages">
443-449.
 </span>
<p class="reftitl">
Sponge-Cell Culture? A Molecular Identification Method for Sponge Cells
<p class="Para1">
<i>Dysidea avara</i> specimens were obtained from Gulf of Naples,
dissociated into single cells and suspended in natural seawater filtered
and containing an antibiotic cocktail.  Cells were transported and washed
in artificial seawater (33 g/l instant ocean reef crystals in demin. water);
one fraction set aside for DNA isolation and the balance cultured.
Artificial seawater (ASW) enriched in the same antibiotics (kanamycin,
tetracycline, gentamycin, tylosin).  ASW was replaced twice to ensure
that only attached cells were in culture.  Cells were then scraped and
incubated in 50% DMEM w/ glucose, Gln, kanamycin, tylosin, Tet, gentamycin,
but the osmolarity was adjusted to seawater with NaCl.  Medium was replaced
weekly, cultures maintained at 18&deg;.  After 4 weeks cells were harvested
for genomic analysis.
<p>
The gene for 18S ribosomal RNA was amplified from genomic DNA by PCR using
universal primers (see Collins 1998).  Products were cloned into <i>E. coli</i>
JM109 using Promega pGEM-T vector.  Ten transformants were selected.
White colonies were transferred using
toothpick into 20 &micro;l TE, and cells lysed at 95&deg;/15 min.  PCR
was then done on lysates with pGEM-T-specific primers T7 and Sp6, confirming
insert sizes.  Only two of the 10 transformants contained an 1800 bp insert.
Transformant #7 was selected for further workup, although #10 was also
selected for partial sequencing for confirmation.
RFLP analysis using <i>Cfo</i>I was done to determine diversity;
transformants #7 and #10 had identical patterns.
Inserts with unique patterns were purified on spin minipreps (Qiagen)
and the DNA sequenced by taking 1 &micro;g pure pGEM-T plasmid and the
T7 sequenase sequencing kit (Amersham Life Sci).  The complete 18S rDNA
was deposited into GenBank and assigned AF456236.  BLAST analysis
of the complete sequence came up with the highest similarity scores with
18S rRNA genes for the marine sponges <i>Pleraplysilla spinifera</i> and
<i>Hippospongia communis</i> (98% and 97%, resp.).
<p>
Cells taken through the culture regime were generally pure after a single
medium replacement despite doing two replacements.  Cells grew in small
aggregates loosely attached to petri dishes.  Cells had a uniform appearance
and a growth rate of 0.13 per day, corresponding to a doubling rate of
5-6 days.  Restriction analysis of 18S rDNA of the growing organisms
indicated that the proliferating cells were not <i>D. avara</i>, but that
of another eukaryote although the similarity was 85%.  Search analysis
indicated the rDNA had 99.4% similarity to rRNA gene from <i>Candida
parapsilosis</i>.
<p>
A difficulty of culturing sponge cells is that they seem to be as susceptible
to antibiotics specific for yeasts and fungi.  Use of 18S rRNA gene for
identification is useful for verifying sponge cells.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Stark J, Callard R, Hubank M
 </span>
 <span class="year">
(2003)
 </span>
 <span class="jname">
Trends Biotechnol.
 </span>
 <span class="jvol">
21,
 </span>
 <span class="pages">
290-293.
 </span>
<p class="reftitl">
From The Top Down:  Towards A Predictive Biology of Signalling Networks
<p class="abstract">
<b>ABSTRACT</b>.
True quantitative and predictive biology requires the
ability to interpret increasingly complex datasets in
new ways to reveal underlying functional interactions.
This has come a step closer with two recent articles
that describe a top-down modelling approach to
reconstructing functional networks from microarray data.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Rappsilber J, Ishihama Y, Mann M
 </span>
 <span class="year">
(2003)
 </span>
 <span class="jname">
Anal. Chem.
 </span>
 <span class="jvol">
75,
 </span>
 <span class="pages">
663-670.
 </span>
<p class="reftitl">
Stop and Go Extraction Tips for Matrix-Assisted Laser Desorption/Ionization,
Nanoelectrospray, and LC/MS Sample Pretreatment in Proteomics
<p class="Para1">
StageTips are reversed-phase beads placed in Teflon matrix mesh with a small
disk having a 60 nL volume; useful in MALDI and nano-ESI work.  The disk is
punched out with a needle using an Empore Disk C18 (3M).  The disk is positioned
in the tip, serving as packing material and frit simultaneously.  The Teflon
minimizes peptide absorption
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Chamrad DC, Koerting G, Goborn J, Thiele H, Klose J, Meyer HE, Blueggel M
 </span>
 <span class="year">
(2003)
 </span>
 <span class="jname">
Anal. Bioanal. Chem.
 </span>
 <span class="jvol">
376,
 </span>
 <span class="pages">
1014-1022.
 </span>
<p class="reftitl">
Interpretation of Mass Spectrometry Data for High-Throughput Proteomics
<p class="Para1">
Mouse brain extracts were run on 2D gel and illuminated by CBB G250.  1336
spots were automatically cut using the Bruker Daltonics Proteineer and
Protagen (Bochum).  MALDI-ToF of tryptic peptides was collected in reflectron
mode in the Bruker Daltonic Reflex III with Scout 384 ion source.  The
SNAP algorithm was used to detect peptide signal.  Peak lists from spectra,
2DE images and gel spot data were put into ProteinScape database.
ProteinScape is a relational database  MS data interpretation is done with a
module called ScoreBooster.
<p>
Prior to PMF identification, MS data are processed by a calibration, peak
rejection, triggering of search engines, result evaluation, and then annotation
and storage in PorteinScape.  Three identification approaches are undertaken:
one involves external calibration with no further data processing, with the
search engine ProFound being used.  Approach 2 involves automatic calibration
and peak rejection.  Approach 3 involves a meta search strategy making use
of the three available PMF search engines:  Mascot, ProFound, and MS-Fit.
<p>
Many signals in the spectra other than the peptides of the protein of interest,
such as autolytic products of cleavage enzyme and the MALDI matrix and the
2D gel stain.  Other contaminants include keratin from hair, nails, skin,
PEG in hand creams and plastic softeners.  The mass of some contaminants if
known can be useful in a multipoint calibration.  Alternatively the peaks
can overlap important peptide signals or generate false matches in database
searches.
<p>
Calibration constants of MALDI-TOF-MS are affected by instrument parameters,
sample position and morphology, desorption laser anergy, and temp variations.
Calibration is achieved in three common ways:  through external calibrants,
internal calibrants, and statistical calibration by the peptide mass
distribution rule.
<p>
To prevent wrong calibrations and keep track of the rendered calibration,
the quality of the calibration was assessed by these formula:
<!--
<table class="eqnserif">
</table>-->
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Osinga R
 </span>
 <span class="year">
(2003)
 </span>
 <span class="jname">
J. Biotechnol.
 </span>
 <span class="jvol">
100,
 </span>
 <span class="pages">
91-92.
 </span>
<p class="reftitl">
Biotechnological Aspects of Marine Sponges
<p class="Para1">
<b>EDITORIAL</b>.
Porifera are the oldest of metazoans.  Besides being natural sponges,
their secondary metabolites are useful as natural products in
medicine and other areas.
<p>
New cultivation (farming) techniques are sought.  40 scientists in
2001 attended international meeting dedicated to subject.
<table class="hrules" style="text-align:center;">
<thead>
<tr><th>Culture Method   <th>Advantages   <th>Disadvantages
<tbody>
<tr><td>Sea-based aquaculture
 <td><ul><li>low cost  <li>easy </ul>
 <td><ul><li>exposure to pathogens of sponges
    <li>lack of control of adverse growing conditions</ul>
<tr><td>Semi-enclosed unfiltered seawater culture
 <td><ul><li>control of most growth conditions  <li>natural food </ul>
 <td><ul><li>higher cost  <li>continued susceptibility to pathogens </ul>
<tr><td>Closed marine systems
 <td>completely controlled growth
 <td>ignorance of defined optimal conditions
<tr><td>culture expansion from totipotent cells
 <td>better control against pathogens
 <td>little data to date
</table>
<p>
Multicellular aggregates (primmorphs) are the best starters for
culture than infinite cell lines.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Zenzmaier C, Kollroser M, Gesslbauer B, Jandrositz A, Preisegger K-H, Kungl AJ
 </span>
 <span class="year">
(2003)
 </span>
 <span class="jname">
Biochem. Biophys. Res. Comm.
 </span>
 <span class="jvol">
310,
 </span>
 <span class="pages">
483-490.
 </span>
<p class="reftitl">
Preliminary 2-D Chromatographic Investigation of The Human Stem Cell Proteome
<p class="abstract">
<b>ABSTRACT</b>.
Stem cells represent a promising tool for the treatment of various hematopoietic diseases. In order to identify stem cell-specific
proteins, the proteome of human stem cells from umbilical cord blood was explored for the first time. For this purpose, the crude
lysate of 4 &times; 10<sup>5</sup> CD34+ cells was subjected to in solution trypsin digestion. The resulting peptides were then separated via cation
exchange followed by reversed phase chromatography and analyzed by nanospray MS/MS. Database search revealed a total of 215
proteins which could be reliably identified. To obtain a more complete picture of the human stell cell proteome and to also access
low abundant proteins, pooling of more than one CD34+ preparations seems necessary in order to increase the cell number and thus
the protein content.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Schmitt-Kopplin P, Frommberger M
 </span>
 <span class="year">
(2003)
 </span>
 <span class="jname">
Electrophoresis
 </span>
 <span class="jvol">
24,
 </span>
 <span class="pages">
3837-3867.
 </span>
<p class="reftitl">
Capillary Electrophoresis-Mass Spectrometry:  15 Years of Developments
and Applications
<p class="abstract">
<b>REVIEW</b>.
<p>
<b>ABSTRACT</b>.
Since its introduction in 1987, capillary electrophoresis-mass spectrometry (CE-MS)
has developed to a well accepted multidimensional analytical approach complementary
and/or competitive to classical MS-hyphenated separation techniques. The threefold
combination of rapid developments of an exceptional separation technique, of
selective mass detection possibilities, and of very mild ionization modes first allowed
these progresses. This article shows the CE specificities that need to be well controlled/
known, compared to classical and more routinely used liquid chromatography
in the light of its coupling to MS. The major trends and developments over the last
15 years and most of the reviews and applications found in ISI Web of science and
publisher databases are presented in a tabulated way. The reader can thus rapidly
find existing CE-MS analysis techniques in his field of research and application (forensics,
environment, bioanalytics, pharmaceutics, and metabolites).
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Cornwall GA, Hsia N
 </span>
 <span class="year">
(2003)
 </span>
 <span class="jname">
Molec. Cell. Endocrinol.
 </span>
 <span class="jvol">
200,
 </span>
 <span class="pages">
1-8.
 </span>
<p class="reftitl">
A New Subgroup of the Family 2 Cystatins
<p class="Para1">
<b>REVIEW</b>.  Cystatins are superfamily of reversible competitive inhibitors
of C1 cysteine proteases;  they inhibit proteases like plant papain and the
mammalian cathepsins B, H, and L.  The superfamily consists of stefins (family 1),
cystatins (family 2) and kininogens (famiy 3).  Family 1 cystatins have a size
of about 11 kDa and no disulfides or oligosaccharide groups and include cystatins
A and B.  Family 2 include cystatins C, D, E/M, F, S, SN, SA and are 13 kDa
secretory proteins with two disulfide bonds in the C-terminus and some family
members being glycosylated.  Faimly 3 are low MW (50-68 kDa) and high MW (88-115 kDa )
forms also secreted and glycosylated and contain three family 2 domains, two of
which have protease inhibitory activity.
<p>
Both mutagenesis and X-ray crystallography show three conserved regions in
cystatins forming a wedge-shaped structure blocking C1 cysteine protease active site.
The three regions are (1) N-termainal glycine, (2) a Gln-Val-Gly loop (QXVXG),
and (3) a C-terminal hairpin with Pro-Trp (PW) residues.  Roles in tumor invasion,
inflammation, and proprotein processing have been proposed, although there is not
a great deal understood with respect to in vivo biological function.  Gene knockout
of murine cystatin B and loss-of-function mutations in human cystatin B result
in neurological dysfunciton as a form of epilepsy.  Mutations in cystatin C
cause a neurological disease amyloid angiopathy.
<p>
The archetypal cystatin was probably a family 1 gene.  Disulfide bonds were added
with the emergence of family 2 genes, and two gene duplication events likely gave
rise to the higher MW forms of family 3 genes.  Fetuin and His-rich glycoprotein
(HRG) contain two cystatin-like domains and conserved disulfide bonds but have
no inhibitory activity indicates a 4th family with possibly new functions.  HRG
has no known function but fetuin will bind TGF-&beta;1 and -&beta;2 and might be
a cytokine antagonist.
<p>
Witin family 2 cystatins are the cystatin-related epididymal spermatogenic (CRES)
subgroup, identified by subtractive hybridization for genes preferentially expressed
in the proximal portion of the murine epididymis.  The CRES and cystatin group
had a low total sequence similarity (&lt; 30%), but had the greatest identity
with family 2 cystatins.  CRES is a <i>N-glycosylated</i> secretory protein with
similar size to cystatin C and conserved Cys residues.  <i>Cres</i> gene structure
shows an additional exon with 5'-untranslated sequences and the 3 exons downstream
have sizes similar to mouse cystatin C; intron/exon split sites are also conserved;
finally it maps to chromosome 2 and co-segregates with cystatin C.  CRES represents
a subgroup also in that it contains the C-terminal PW site but lacks teh Q-V-G
sequence required for Cys protease inhibition.  The N-terminal region, required for
high affinity binding to Cys proteases, is poorly conserved and thus specificity
may be entirely different.   Cystatin C is also ubiquitous whereas Northern blots
show high levels of <i>Cres</i> mRNA in proximal caput epididymis and testes but
not in 28 other tissues probed.  ISH shows <i>Cres</i> expression in epithelial cells
of epididymis and in stage-specific germ cells of the testes.  CRES protein is found
in tissue and luminal fluid of proximal epididymis and in the sperm acrosome,
indicating importance in sperm maturation and fertilization.  RT-PCR and
immunohistochem work shows <i>Cres</i> mRNA and protein in the anterior pituitary
gonadotroph cells and a colocalization of CRES with LH&beta; in the secretory
pathway.
<p>
Two genes later emerged as members of the CRES subgroup.  Testatin has been shown
in differential display for a search of genes in mouse testicular development,
and cystatin T emerged from a database search of cystatin-related genes.  Testatin
and cystatin T have a 23 and 38% protein sequence identity to CRES, and 21% to each
other.  While that percentage is low, what is interesting is the sharing of
unique features:  they encode secretory proteins of similiar MW and they possess
conserved Cys residues.  They contain PW at C-term but lacks the Q-V-G and have
divergent N-terminal sequences.  As for gene structure, testatin has two introns
of similar alignment and cystatin T maps to chromosome 2 near cystatin C and <i>Cres</i>.
More compelling is the expression seems focused in the reproductive tract.  Testatin
is expressed according to RT-PCR in mouse testis just after <i>Sry</i> (sex-determining
region) gene is.  Cystatin T is expressed in pachytene spermatocytes and round
spematids.
<p>
A study of inhibition found that CRES does not inhibit cysteine proteases like
papain or cathepsin B.  However at nanomolar concentrations it inhibits the prohormone
convertase, involved in proteolytic prohormone forms of maturation.  Thus CRES
is a new functional family of Ser protease (serpins) inhibitors.  It lacks the
Asn39 residue said found as important in cystatin C inhibition mechanism.
<p>
Database search of genomic sequences shows that CRES-like sequences even in the
nematpode are found.  Several orthologs are found in different species including
catfish, zebrafish, and carp.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Mano N, Iijima S, Kasuga K, Goto J
 </span>
 <span class="year">
(2003)
 </span>
 <span class="jname">
Analyt. Sci.
 </span>
 <span class="jvol">
19,
 </span>
 <span class="pages">
1469-1472.
 </span>
<p class="reftitl">
Exopeptidase Degradation for the Analysis of Phosphorylation Site in a
Mono-phosphorylated Peptide with Matrix-assisted Laser Desorption/Ionization
Mass Spectrometry
<p class="Para1">
Phosphopeptide detection in MALDI applications differs from that of other peptides.
Every peptide fragment produced by enzymatic cleavage has different charge and
physicochemical properties.  HPLC with UV detection at 210-220 nm will detect
P-peptides similiar to non-P-peptides.  Tryptic peptides of &beta;-casein were
treated with alkaline phosphatase.  HPLC runs show a peak with retentiom time
of 10.0 min peak disappears in the ALP-treated while a peak at 10.4 min appears
in the sample not treated with ALP.  The 10.0-min peak in positive ion mode shows
a [M+H]<sup>+</sup> peak at <i>m/z</i> 2063, which would be a singly phosphorylated
33-48 fragment, which the 10.4-min peak shows a signal at <i>m/z</i> 1983, fully
80 amu different.  In negative ion mode, a [M-H]<sup>&minus;</sup> peak has
an <i>m/z</i> 2061 with very high sensitivity.  Together these confirm the
fragment nature and its phosphorylation.
<p>
Carboxypeptidase W (CPW, Seikagaku, Tokyo) and aminopeptidase M (APM, Roche Diag) were
used in ladder sequencing.  For CPW, 10 pmol peptide in 15 &micro;l 50 mM ammonium
acetate pH 4.0 is mixed with 1.2 &micro;g CPW (20:1 substrate:enzyme),
and 3 &micro;l aliquots are removed and mixed with 3 &micro;l 0.2% TFA at
37&deg; at various time intervals.  For APM, 10 pmol substrate mixed with
12 &micro;l 50 mM TrisHCl pH 7.5 with 1 mM MgCl<sub>2</sub> and 64 &micro;l
APM (20 U/ml).  This is incubated at 37&deg; for various intervals, and
11 &micro;l alquots taken and mixed with 8 &micro;l 0.2% TFA.  The various
times were from 0-300 min.  Peptides that included the Lys residue were
strong in signal, while without it the signal was less intense because of
phosphate negative charge.  In negative ion mode the ladder peptides had predictably
stronger signals.
<p>
With peptides FQpSEEQQQTEDE and FQpSEEQQQTEDELQ there are no positive ion signals,
but the negative ion mode signals were barely detected.  Two potentially phosphorylated
residues, Ser-35 and Thr-41, are present.  Spectra suggest no phosphate at Thr-41.
APM digestion was unhelpful since it was blocked by the phosphorylated Ser.
It has been found that phosphorylated residues resist amino- and carboxypeptidase action.
After treating with ALP, APM and CPW digestion was done and peptides formed between
Thr 41 and Lys 48 were easily detectable.  The ladder formed by APM treatment easily
clarified the phosphorylated residue from the dephosphorylated peptide.  Good ladders
from Phe 33 to Glu 37 were formed.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Storey JD
 </span>
 <span class="year">
(2003)
 </span>
 <span class="jname">
Ann. Stat.
 </span>
 <span class="jvol">
31,
 </span>
 <span class="pages">
2013-2035.
 </span>
<p class="reftitl">
The Postive False Discovery Rate:  A Bayesian Interpretation and The
<i>q</i>-Value
<p class="Para1">
Single hypothesis testing concerns the control of the false positive rate (FPR)
while maximing probability of detection when there is a condtion.  This
is maximing the power conditional on Type I error being at or below some
level.  With multiple hypothesis testing (testing them simultaneously)
a compound error measure is defined according th FPR one is willing to encounter.
One sets the error rate at desired level and maintains power of each test
as much as possible.  The family-wise error rate (FWER) is most commonly
controlled in multiple hypothesis testing, with Bonferroni test the typical.
With <i>m</i> hypothesis test, each test is controlled so that the probability
of FP &le; &alpha;/<i>m</i> for a chosen value of &alpha;.  Then overall
FWER &le; &alpha;.
<p>
In some cases we are concerned about FP rates among rejected hypotheses
rather than probability of making one or more Type I errors.  Today a dataset
presents so much information that several hypotheses can apply to it
simultaneously.  The concern is over false positives: the rejection of the
null hypothesis when it is true.  There is less concern over the significance
of all items.
<table>
<caption>Possible Outcomes from <i>m</i> Hypothesis Tests</caption>
<tr><th>          <th>Accept null <th>Reject null <th>Total
<tr><td>Null true <td><i>U</i>    <td><i>V</i>    <td><i>m</i><sub>0</sub>
<tr><td>Alternative true
                  <td><i>T</i>    <td><i>S</i>    <td><i>m</i><sub>1</sub>
<tr><td>          <td><i>W</i>    <td><i>R</i>    <td><i>m</i>
</table>
<p>The table above gives various outcomes that occur when <i>m</i> hypothesis
tests are done according to some significance rule, fixed or data-dependent.
FWER can be stated to have probability Pr(<i>V</i>&ge;1).  Benjamini and
Hochberg introduced the multiple hypothesis testing error measure called
the <b>false discovery rate</b> (FDR) defined as:
<p class="eqn">
FDR = E [ <i>V</i> / (<i>R</i> &#8744; 1) ]<br>
= E[ (<i>V</i> / <i>R</i> | <i>R</i> &gt; 0] Pr(<i>R</i> &gt; 0)
<p>
Only effect of <q><i>R</i> &#8744; 1</q> in denominator of first expectation
is that ratio <i>V / R</i> is set to zero when <i>R</i> = 0.
<p>
The Benjamini and Hochberg procedure works thusly:  suppose the <i>p</i>-values
resulting from <i>m</i> tests are ordered so that <i>p</i><sub>(1)</sub> &le;
<i>p</i><sub>(2)</sub> &le; ... &le; <i>p</i><sub>(<i>m</i>)</sub>.
If the following is calculated:
<p class="eqn">
<i>k</i> = arg max{<i>k</i>:<i>p</i><sub>(<i>k</i>)</sub> &le;&alpha;&bull;
<i>k</i>/<i>m</i>},
<p>
for 1&le;<i>k</i>&le;<i>m</i> for the arg max.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Edge ASB
 </span>
 <span class="year">
(2003)
 </span>
 <span class="jname">
Biochem. J.
 </span>
 <span class="jvol">
376,
 </span>
 <span class="pages">
339-350.
 </span>
<p class="reftitl">
Deglycosylation of glycoproteins with trifluoromethanesulphonic acid:
elucidation of molecular structure and function
<p class="Para1">
<b>REVIEW</b>.  An advantage of TFMS chemical deglycosylation over enzymatic
removal is that all oligosaccharide groups can be removed in a single step
with no requirement about the structure or type of glycan.  TFMS has been
show to work on O- and N-linked glycans as well as those collagen saccharides
attached by hydroxyproline linkage.  Gal&rarr;Ser, Ara&rarr;HO-Pro, Man&rarr;Ser,
Glc&rarr;Tyr linkages in all manner of living species have been broken.
<p>
Amide bonds are stable under TFMS, which is why the polypeptide is not damaged.
In fact, TMFS is unable to break the Asn and N-acetylglucosamine (GlcNAc)
bond.
<p>
The reaction is done in anhydrous conditons with a free radical scavenger (often
anisole) to protect amino acids.  Solvolytic cleavge of glycosidic bonds occurs,
done in absence of water, or else this can induce protein cleavage.  In the
solid phase synthesis of peptides, TFMS is used to cleave the finished peptide
from the resin.  With anisole present damage to side chains of Tyr, Trp,
Cys, Met, Ser, and Thr are kept to a minimum.  Deprotection is also achieved
by HF with same methods, but TFMS is less toxic and more powerful.
<p>
TFMS varies on how it takes apart the sugars.  The GalNAc that is directly
connected to Ser/Thr in O-linked sugars is rather resistant to TFMS, also
observed with HF.  All Gal and Man chains in fetuin were removed at 0&deg;
in less than an hour, while GalNAc at 25&deg; was 75% removed after 3 h.
TFMS has fully deglycosylated plant glycoproteins and proteoglycans, distinct
from the more resistant mucins, whose resistant is more structure-based than
the extent of glycosylation.
<p>
The scavenger provides a sink for reactive electrophiles.  Severe side reactions
are seen in the absence, with amino acids Ser, Thr, Asp, Cys, and Met, and the
aromatics.  Others have reported that phenol and toluene give superior results.
TFMS reactivity is reduced if the scavenger concentration is too high.  Generally
the anisole is set at 10% of level of TFMS mole-for-mole.  Reaction volumes are
usually less than 100 &micro;l and reaction temperature should be 0&deg; unless
O-linked GalNAc must be removed, in which case the temperature should be higher.
<p>
TFMS is an option when enzymic deglycosylation has proven difficult or useless.
N-glycanase (PNGase F) and endo F and endo H (endo-&beta;-<i>N</i>-acetylglucosaminidases
F and H) are typically used enzymes.  Often these enzymes will not work without
using denaturing agents to open access to the cleavage site.  Enzymes of
O-linked sugars are much more limited.  The commercially available O-glycanase
is specific for the disaccharide Gal-GalNAc.  Mucins generally are untouched
because of their complexity.
<p>
Treatment of proteoglycans with heparanases, chondroitinases, hyaluronidases,
and keratanases will remove the GAGs, but it is often necessary to know something
about the sugar structure before using these enzymes.  The core saccharide
GlcA-Gal-Gal-Xyl is not removed by enzymes and remains.
<p>
Following deglycosylation, mass determinations using MS have been done and
the predicted and measured masses corresponded quite well.  The fungal glycoprotein
hydrophobin S3 was TFMS-treated and MALDI-ToF MS measured a 10854 Da mass
against the theoretical 10856 Da.  The pre-TFMS mass is 14,400 Da.
<p>
<b>Sequencing of glycoproteins</b>.  Glycoproteins are less susceptible to
trypsin and other proteases used in sequencing or protein identification.
In Edman degradation, the PTH-amino acid of the glycosylated residue is
not determined.  Fragmentation of MS ions is complicated and gives uninterpretable
spectra.  Mucin analysis was impossible since they are protease-resistant, whereas
TFMS made it entirely analyzable.  Glycsoylation sites are all determinable
for O-linkages if TFMS treatment is limited or mild (1 h, 4&deg;).  If full
removal of O-linked sugars is desirable, periodate treatment subsequent to
TFMS treatment can be done.
<p>
It is ofen useful to use N-glycanase treatment and TFMS to look for mass
differences between the products.  TFMS removes all O- and N-linked, while
N-glycanase removes only N-linked.  A significant mass difference indicaes
O-linked sugars.  Combined with proteolysis, mass analysis of the peptides
can suggest locations of where oligosaccharides are attached.  Confirmation
of O-linkage can be done with the use of lectins specific for many O-linked
sugars.
<p>
The difference of glycosidic bond susceptibility to TFMS has been exploited
to characterize the oligosaccharides structurally.  Substituents at C-2
which include amido and acetyl groups alter TFMS reactivity at C-1.
With the complete reaction, monosaccharides are obtained that can be
analyzed quantitatively.
<p>
TFMS does not appear to affect other post-translational modificiations.
Thus if the mass of a protein is reduced by TFMS, it is safe to say that the
modification was a glycan.  <sup>32</sup>P-labeled proteins retain the
radiolabel under TFMS, as shown after acid hydrolysis of phosphoamino acids.
Sulfates on sugars are cleaved by TFMS, but not on tyrosine.  Haem groups
are not affected by TFMS, and Ca<sup>2+</sup> ion is stably associated with
their proteins.  Glycosylphosphatidylinositol (GPI) anchors are expectedly
removed by TFMS.  Fatty acid acyl linkages are stable.  Disulfide linkages
are also unaffected.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Wang W, Zhou H, Lin H, Roy S, Shaler TA, Hill LR, Norton S, Kumar P, Anderle M, 
Becker CH 
 </span>
 <span class="year">
(2003)
 </span>
 <span class="jname">
Anal. Chem.
 </span>
 <span class="jvol">
75,
 </span>
 <span class="pages">
4818-4826.
 </span>
<p class="reftitl">
Quantification of Proteins and Metabolites by Mass Spectrometry without Isotopic 
Labeling or Spiked Standards
<p class="Para1">
In this study human serum is pooled (from four anonymous healthy donors) and 
several metabolites and proteins will be spiked into it.  A 5-protein mix
(eq. myoglobin, BSA, bov RNase A, bov cytochrome c, human Hb) is also used, some
proteins not varied, others varied.  A binary HP 1100 Series HPLC is interfaced
to ESI to ThermoFinnigan LCQ DECA with automatic gain or Micromass LCT ESI-ToF
with New Objective nanospray.  PicoFrit capillary columns (5 &micro;m BioBasic
C18, 75 &micro;m &times; 10 cm) is run at 300 nl/min in a split flow. 20 &micro;g
of tryptic peptides are injected in 20 &micro;l loop.  Gradient is 0.1% formic
acid to 40% B (= 0.1% formic acid in MeCN) over 100 min period.  For metabolome
work, a 20 &micro;l injection is done:  this is 10-25% solvent B in 40 min, 
then 25-90% solvent B in 30 min.  With these programs, 50 samples/week can be
analyzed.
<p>
5-protein mixtures were prepared so that eq myoglobin, RNase A, and BSA are
constant in three preparation.  Cytochrome c and Hb were however varied
5-fold in the two other preprations (in one prep 5&times; cytochome c and 
1/5&times; Hb; in other 1/5&times; cytochrome c and 5&times; Hb).  Proteins
are alkylated after reduction, digested with trypsin, and then analyzed on
ion trap.  Each preparation is run in four replicates.  Intensities of tryptic
peptides of the proteins kept constant in all three mixes show that the signal
intensities for the two proteins changed were close to linear in the known
relative concentrations, within an acceptable coefficient of variance (CV).
<p>
With human serum, the proteome (high MW) and metabolome (low MW) are first
separated and both fractions have significant complexity as shown by the ion
chromatogram.  With the ToF tracking and quantifying large counts of mass
spectral peaks is advantageous.  Software was customized for doing analysis.
MS scans of 1 s period are taken during chromatography:  noise reduction involves
background subtract and signal smoothing.  Peak detection is by vectorized method
described in another publication (particular good in high intensity or chemical
noise).  De-isotoping is done (requires idealizing an isotope pattern for all
peaks).  Reference sample is arbitrarily chosen for adjusting retention times
where small variations exist.  Centroided intensities are normalized globally
(one normalization constant for the sample).  Where MS peaks are in multiple
samples, these become components of the study, and final intensities are determined
in each sample, and CV computed.  Paired or group statistical tests (such as
<i>t</i> test) are used to determine significant differences.  In baseline
subtraction, a low-order polynomial smoothing filters (based on Savitzky-Golay)
is applied to remove baseline noise and spikes, without loss of signal frequency
(see Chapt 14.8 of Numberical Recipes in C by Press et al).  A smoothed baseline
with well-defined starting and end points for all peaks results.  Then sets of
points belonging to baseline selected, and low-order polynomial interpolation
results in new baseline approximation, which is subtracted from original
spectrum.  Methods described in Savitzky, Golay (Anal Chem 36: 1627, 1964),
Steiner et al (Anal Chem 44: 1906, 1972), and Madden (Anal Chem 50: 1383, 1978)
are used to process peak and baseline signal, with different filter parameters
for baseline and signal, prove good for multifrequency sginals (multiple peaks
with different peak width).  With vectorized peak detection, local maxima are
found in two dimensions.  Ion counts must be greater than a varying threshold
over a limited range of <i>m/z</i> and time values, and this phase is peak
sensing.  Points seen in both dimensions pass the criteria.  there is a robust
elimination of chemical noise while accepting weak analyte peaks.  Chemical noise
peaks can be elimianted or at least greatly reduced in that they show no local
maxima in the chromatographic dimension.
<p>
The maximum intensity isotope in a spectral region is located and then a template
matching tool is applied (often used in image processing for object recognition)
and an isotope pattern is found.  Templates derived from an isotopic table search
for patterns of peaks in the original spectra.  The template or pattern is 
mass-dependent and based on nominal peptide elemental composition associated with
avg occurrence of various amino acids determined for large number of proteins
(see Mathews &amp; van Holde, <i>Biochemistry</i>, Chap 5, 1990).  Measure of fit
between original data and isotopic pattern template is cross-correlation 
(convolution) between the two, and is maximized by the isotopic template that
matches the one found in the original data.  A variety of patterns associated with
different charge states (and different masses for <i>m/z</i> region) are examined
systematically up to z = 5 or 6, and best fit chosen.  Isotopes then are found
to belong to single molecular ion and then removed from peak list and no longer
considered.
<p>
Small variations also occur in the 100 min chromatographic run.  Tracking ions
can be difficult when many elute in a dense time period.  <q>Dynamic time warping</q>
(DTW) is an approach developed by us where a sample file is compared to the 
reference; for each comparison, a set of peaks in common between the two files
are selected by software based on retention time windows (in the first stage
&plusmn; 5 min) and on <i>m/z</i> (&plusmn; 0.10 Da).  All mass chromatograms
spanning elution times at these masses are used in compututation rather than 
just peaks themselves.  The number of mass chromatograms is limited to ~200
since computations are time-consuming.  The 200 mass chromatograms are combined
into a matrix: each element is difference in intensity at time point and <i>m/z</i>
between two files, each matrix row originates from mass chromatogram.  Using
dynamic programming techniques (Rabiner &amp; Juang, <i>Fundamentals of Speech
Recognition</i>, 1993, Ch 4) a path is found through the matrix minimizing
intensity differences.  This path provides a warping function to apply to the
nonreference file.  Wang &amp; Isenhour (Anal Chem 59: 649, 1987) applied a 
similar approach but only for TIC plots, which is not enough for this type of 
analysis.  Peak intensities are next determined for quantification using smoothed
data, by using eitehr peak area (summed over elution time of component which
is mass chromatogram, or by max peak height in ion counts.  Software can sum
over all isotopes or use intensity of monoisotopic peak.  In this report
the TOF data took as the peak intensity the max valu of smoothed monoisotopic
peak in ion counts.
<p>
Data is normalized to correct for instrument sensitivity drift over time and
sample-to-sample concentration variations.  One file is chosen as reference
and then other files refactored (normalized) relative to it.  There is no
prior evaluation in choosing the reference file.  Retention time and <i>m/z</i>
values are matched by a window.  Unlike size-constrained list used for DTW,
no need to constrain count of ratios, thus 1000-3000 peak pairs used.  The
single normalization constant for each file is taken as median of the ratios
of intensities for all componnents between file being examined and the reference.
The software does not select peaks in making comparison, but the median value
of intensity ratio of all spectral peaks between the sample and reference file
is used as normalization for that file.  In using the median instead of average,
strongly changing peaks get a better representation.  Component building now
occurs by scanning de-isotoped peak maximums according to <i>m/z</i> and retention
time values.  If a peak in another sample is with user-settable <i>m/z</i>
and time windows, the software calls it an identity.  The ToF <i>m/z</i> window
is 0.1 Da or less than for ion trap.  As for intensity, a threshold is set
by user (usually ~20 ion counts); if more conservative, more reliable and robust.
<p>
In the serum proteome analysis (20 &micro;g trypsin-digested), about 2700
molecular ions were found in the file.  In the metabolome fraction of 100 &micro;l
serum, there were 730 molecular ions detected and tracked.
<p>
Now to quantify, stability and reproducibility of analyte signal and linear
(or near-linear) dependence on control is required.  105 proteome samples were
were processed and analyzed consecutively on nano-LC-MS in 13 days, 80 on
ion-trap and 25 on ToF.  115 samples were analyzed for metabolome analysis over
17 days.  MassView software was used to quantify intensities relative to
reference in order to measure CV for larger numbers of molecular components.
A histogram of the CV values of normalized intensities are then plotted for
about 3400 ions from 25 samples, and with about 85,000 ion measurements
(each molecule de-isotoped).  Median CV was 25.7% and mean CV was 29.0% for 
proteome and 23.8% and 28.5% for metabolome, respectively.  This histogram
distribution was divided into quartiles on the basis of average intensity
for components in order to compare high and low-intensity ions.  New histograms
were prepared for lowest to highest intensity ions.  Plots show that there is
no real increase in CV values for the lowest intensity ions:  some narrowing
of CV distribution occurs in lowest quartile because of the influence of 
threshold eliminating small intensity data.
<p>
Two proteins (eq myoglobin and bov carbonic anydrase II) were spiked into 
20 &micro;g unprocessed human serum in amts ranging from 100 fmol &rarr; 100 pmol.
Samples were denatured and digested with trypsin and analyzed on ion-trap and 
ToF MS instruments using nano-ESI-LC systems.  Plots of intensity vs pmol 
spiked protein show that the signal is quite linear over the entire range.
With the ESI-ToF, at 50 fmol protein, signal-to-noise was more than 60, showing
that lower masses could be used.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Piraud M, Vianey-Saban C, Petritis K, Elfakir C, Steghens J-P, Morla A, Bouchu D
 </span>
 <span class="year">
(2003)
 </span>
 <span class="jname">
Rapid Comm. Mass Spectrom.
 </span>
 <span class="jvol">
17,
 </span>
 <span class="pages">
1297-1311.
 </span>
<p class="reftitl">
ESI-MS/MS analysis of underivatised amino acids: a new tool for the diagnosis of 
inherited disorders of amino acid metabolism. Fragmentation study of 79 
molecules of biological interest in positive and negative ionisation mode
<p class="Para1">
Many amino acids and their catabolic products can be monitored for evidence of
disorders of metabolism.  All manner of fluids can be followed.  Automated
AA analyzers are ion-exchange chromatography-based after Spackman et al.
This technique can monitor 50 AA and derived compounds, with post-column 
derivatizatin (ninhydrin).  It is not high throughput (only 8-10 runs a day)
with high operating costs on dedicated system.  Most AA methods would always
use pre- or post-analytical modification using UV-absorbing or fluorescent
constituents; derivatization improves sensitivity but this introduces variation.
Tandem MS is being used to analyze AA, organic acid, purines, pyrimidines,
fatty acid oxidation to study inborn errors of metabolism.  FAB ionization
and then ESI methods were developed to look at inherited disorders.
Butyl esters of AA improve detection, minimize ion suppression with complex
biological matrices.
<p>
Attempts have been made to analyze AA without derivatization. CE-ESI-MS/MS of
20 proteinogenic and 31 physiological AA in standard mixture can be done.
Different ion-paired RPLC methods for the 20 AA are also avialable using C18
and porous graphitic carbon with perflourinated carboxylic acids in the ion pair
interfaced to ESI-MS/MS.  Our lab detects and quantifies 80 molecules routinely
using Beckman 6300 AA analyzer.  This study hopes to use LC-ESI-MS/MS to replace
it.  This report looks at 79 molecules of interest using pos, and neg ion mode.
Petritis et al had previously look at 42 in pos mode.  The instrument is API 2000
triple quad MS with ESI. Nitrogen is collision and curtain gas.
<p>
10 &micro;M solutions were prepared in MeOH/water and infused by syringe into
ESI.  It has an MRM to select most abundant or most specific transition.
Gas flow and voltage parameters are similar for all ions.  The declustering
potential (DP) was specific for each precursor ion, and collision energy (CE)
specific for each fragment ion.  A table is produced which shows the results
of negative ion mode, showing DP values, nominal mass [M-H]<sup>&minus;</sup>
and the produced fragment ions with at least 10% RI.  Small-mass molecules usually
show no [M-H]<sup>&minus;</sup> ion in neutral solvents or detectable fragments.
Basic AA do not ionize expectedly.  All these results show AA are better studied
in pos mode.  A total of 41 of the 79 molecules produce [M-H]<sup>-</sup> ions
and fragments.  Leu, Ile, Val, Arg, Orn, Lys, and Tyr do not show in neg mode
and these are typical AA in metabolic problems.
<p>
Another table shows the results of positive ion mode.  Fully 67 of the 79 
molecules show [M+H]<sup>+</sup> ions in neutral conditions.  Of the 12 not
detected, they can be detected in neg mode.  When solvent was acidified, 9 of
the 12 were detectable.  The 3 not detected are orotic acid, orotidine, and 
phosphoethanolamine (PEA).  None of these are true AA and they are highly
acidic, better seen in neg mode.
<p>
Fragment ion spectra are in agreement with previous reports for some molecules
under specified conditions.  Some fragments (not reported) were obtained at
high CE (&gt; 40 eV).  Differences in relative intensities may be because of
use of optimized CE values rather than presets.  Optimization involved setting
curtain/cone gas, ion source gas 1, ion spray voltage (IS), declustering potential
(DP), focusing potential (FP), and entrance potential (EP).  Mechanisms of
fragmentation have been studied.
<p>
38 of the 79 molecules are detected in both pos and neg mode.  Taurine,
S-sulfo-L-cysteine, L-citrulline, &gamma;-carboxyglutamic acid are a few that
are better detected in neg mode.  Sulfur-containing AA (Cys, homoCys) have 
low intensity signals but are better detected in oxidized form (Cys-S-S-Cys,
HCys-S-S-CysH).
<p>
A next step was flow injection analysis (FIA).  Molecules that are poorly
or not detectable in ESI(+) mode have highly electronegative groups (-COOH,
phosphate, sulfate).  These are delivered as zwitterions and thus do not
get drawn into MS path.  Tridecafluoroheptanoic acid (TDFHA) is an extremely
strong acid and volatile ion-pairing agent used for underivaized AA for separation
and MS/MS.  Both neg and pos ion mode show signals, although weaker in neg mode.
Solvent was made alkaline in 0.25% ammonium hydroxide for some molecular analyses
and there was a signal enhancement, indicating that mobile phase clearly has
a significant effect on analysis. In assessing limits of detection (sensitivity),
rough estimates of pos mode for a molecule is about 0.1-0.2 pmol (piperidine,
Leu, Pro, Lys) to 30-40 pmol (Gly, Tau).  In biological fluids under physiological
conditions, plasma AA can range from 10-700 &micro;M, which would be 5-350 pmol
in 5 &micro; of a 1:10 diluted sample:  detection would be possible in plasma
or urine at even 1:10 dilution with 5 &micro;l injection.
<p>
As the AA were not in matrix but merely solvents, it is important to study them
in biofluids for ion (signal) suppression. MS/MS specificity is linked to 
a combined analysis of precursor and product ion.  A problem arises if two
molecules select on the same precursor and product ions.  20 &micro;L of 
50 &micro;M solution of all analytes were injected by FIA under acidic conditions.
Signal (peak area) obtained with each transition was measured.  If signal was
&gt; 0.5% of signal obtained with molecule for which transition was selected,
it is an interference.  Results indicate negligible (&lt;1%) to significant
(&gt;15%) interference, sometimes higher than the analyte signal.  For example
HCy(O<sub>3</sub>H) (<i>m/z</i> 184) and Met-sulfone (<i>m/z</i> 182) 
both produce a <i>m/z</i> 56 fragment (the transition) and this represents 
136% of the signal.
<p>
Isobars include Gln and Lys and Leu, Ile, and (<i>a</i>)Ile, which are even 
isomers.  At low CE, they have identical fragmentation patterns.  There are 
12 groups of isobars.  16 molecules have transitions (fragment in a way) that
allows their distinction and identification.  8 pairs would need separating
before analysis.  6 molecules in have a [M+H]<sup>+</sup> at <i>m/z</i> 132.
Most have transitions that allow distinction.  The two diastereomers of Ile
would not be distinguished unless a prior chromatographic step was done.
<p>
Some interference is the result of in-source CID fragmentation.  These can
interfere with (be mistaken for) other precusors and interfere with other
transitions:  for example, Orn gives 133 &rarr; 70 and 133 &rarr; 116, and
these ions become precursors interfering with Pro (116 &rarr; 70).
<p>
Other interferences arise from the natural isotope distributions, mostly because
of <sup>13</sup>C (1.1%).  In some cases where sulfur is present, its <sup>34</sup>S
is present at 4.5% abundance.  When molecular masses differ by 1 or 2 u, the
interference occurs.  Met-sulfone (182) can interfere with Hcy(O<sub>3</sub>H)
(184).  Gln and Lys (147 &rarr; 84) will interfere with Glu (184 &rarr; 84)
because of <sup>13</sup>C.  8 other situations arise from isotope abundance.
<p>
&Delta;<sup>1</sup>-Pyrroline-5-carboxylic acid interferes with 132 &rarr; 68 
and 132 &rarr; 69 and 132 &rarr; 114 transitions, but there is no ability to
find the reason.  Cross-talk effect as a result of slow ion flight times was
ruled out.  Interference might be due to impurity present in the chemical compound
or a PC degradation product.  If true, then it should not occur in biological
samples.
<!-- ============================================================= -->
<p class="tags">
cluster ion, FD
<p class="ref">
 <span class="authors">
Charles L , Chiron J, Galy J-P
 </span>
 <span class="year">
(2003)
 </span>
 <span class="jname">
Rapid Comm. Mass Spectrom.
 </span>
 <span class="jvol">
17,
 </span>
 <span class="pages">
2471-2474.
 </span>
<p class="reftitl">
Characterization of ammonium chloride derivatives by salt clustering in 
electrospray mass spectrometry
<p class="Para1">
Cluster ions are of the form [CA]<sub>x</sub>C<sub>m</sub><sup>m+</sup>
in positive ion mode and [CA]<sub>y</sub>A<sub>n</sub><sup>n+</sup>
in negative ion mode where C is the cation and A is the anionic part of the 
salt.  Both the charged residue and ion evaporation models can account for
cluster ion formation.  Factors affecting salt cluster ion formation include:
<ul>
<li>salt concentration
<li>solution conductivity
<li>viscosity
<li>mole percent of organic solvent
<li>solvent polarity
<li>counterion characteristics
</ul>
<p>
Chirality of molecules (e.g. amino acids) is also a factor is clusters showing
homochirogenesis.  Homochiral serine clusters formed a magic number protonated
octamers.
<p>
All experiments are done on Sciex API III Plus TQ with pneumatically assisted
electrospray.  Interface temperature is 54&deg;.  A curtain gas of 600 ml/min
nitrogen is used with 800 ml/min zero-grade air.  Spray voltage is 5000 V
positive and orifice voltage is 35 V.  Negative spray is &minus;4800 V and 
orifice is &minus;35 V.  Sample flow rate is 5 &micro;l/min.
<p>
A spectrum of A 1 mmM 4-aminomethyl HCl in isopropanol with range starting
at <i>m/z</i> 200 to 1600 produced numerous peaks.  The use of isopropanol
was expected to favor the formation of clusters.  A base peak 
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Heck AJR, van den Heuvel RHH
 </span>
 <span class="year">
(2004)
 </span>
 <span class="jname">
Mass Spec. Rev.
 </span>
 <span class="jvol">
23,
 </span>
 <span class="pages">
368-389.
 </span>
<p class="reftitl">
Investigation of Intact Protein Complexes By Mass Spectrometry
<p class="abstract">
<b>REVIEW</b>.
<p>
<b>ABSTRACT</b>.
Mass spectrometry has grown in recent years to a well-accepted
and increasingly important complementary technique in structural
biology. Especially electrospray ionization mass spectrometry
is well suited for the detection of non-covalent protein
complexes and their interactions with DNA, RNA, ligands, and
cofactors. Over the last decade, significant advances have been
made in the ionization and mass analysis techniques, which
makes the investigation of even larger and more heterogeneous
intact assemblies feasible. These technological developments
have paved the way to study intact non-covalent proteinprotein
interactions, assembly and disassembly in real time, subunit
exchange, cooperativity effects, and effects of cofactors, allowing
us a better understanding of proteins in cellular processes. In this
review, we describe some of the latest developments and several highlights.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Zhang R, Barker L, Pinchev D, Marshall J, Rasamoelisolo M, Smith C,
Kupchak P, Kireeva I, Ingratta L, Jackowski G
 </span>
 <span class="year">
(2004)
 </span>
 <span class="jname">
Proteomics
 </span>
 <span class="jvol">
4,
 </span>
 <span class="pages">
244-256.
 </span>
<p class="reftitl">
Mining Biomarkers in Human Sera Using Proteomic Tools
<p class="abstract">
<b>ABSTRACT</b>.
One of the major difficulties in mining low abundance biomarkers from serum or
plasma is due to the fact that a small number of proteins such as albumin, a2-macroglobulin,
transferrin, and immunoglobulins, may represent as much as 80% of the total
serum protein. The large quantity of these proteins makes it difficult to identify low
abundance proteins in serum using traditional 2-dimensional electrophoresis. We
recently used a combination of multidimensional liquid chromatography and gel electrophoresis
coupled to matrix-assisted laser desorption/ionization-quadrupole-time of
flight and Ion Trap liquid chromatography-tandem mass spectrometry to identify protein
markers in sera of Alzheimers disease (AD), insulin resistance/type-2 diabetes (IR/
D2), and congestive heart failure (CHF) patients. We identified 8 proteins that exhibit
higher levels in control sera and 36 proteins that exhibit higher levels in disease sera.
For example, haptoglobin and hemoglobin are elevated in sera of AD, IR/D2, and CHF
patients. The levels of several other proteins including fibrinogen and its fragments,
alpha 2-macroglobulin, transthyretin, pro-platelet basic protein, protease inhibitors
clade A and C, as well as proteins involved in the classical complement pathway such
as complement C3, C4, and C1 inhibitor, were found to differ between IR/D2 and control
sera. The sera levels of proteins, such as the 10 kDa subunit of vitronectin, alpha
1-acid glycoprotein, apolipoprotein B100, fragment of factor H, and histidine-rich glycoprotein
were observed to be different between AD and controls. The differences
observed in these biomarker candidates were confirmed by Western blot and the enzyme-
linked immunosorbent assay. The biological meaning of the proteomic changes
in the disease states and the potential use of these changes as diagnostic tools or for
therapeutic intervention will be discussed.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Evans CA, Tonge R, Blinco D, Pierce A, Shaw J, Lu Y, Hamzah HG, Gray A,
Downes CP, Gaskell SJ, Spooncer E, Whetton AD
 </span>
 <span class="year">
(2004)
 </span>
 <span class="jname">
Blood
 </span>
 <span class="jvol">
103,
 </span>
 <span class="pages">
3751-3759.
 </span>
<p class="reftitl">
Comparative proteomics of primitive hematopoietic cell populations reveals
differences in expression of proteins regulating motility
<p class="Para1">
Hematopoietic cells from marrow of C57Bl/6 mice were isolated on Optiprep
and enriched for Lin<sup>&minus;</sup> cells with a linear marker antibody
cocktail (Pharmingen) using magnetic sheep antirat beads.  These cells
were then labeled with FITC-anti-Sca-1 and PhycoE-anti-cKit and sorted
to prepare Lin<sup>&minus;</sup> Sca-1<sup>+</sup> c-Kit<sup>+</sup>
(lSK) or Lin<sup>&minus;</sup> Sca-1<sup>+</sup> c-Kit<sup>&minus;</sup>
(lSk) cells. lSK cells show a reduced motility (chemokinetic) response
to SDF-1 compared to lSk cells.  SDF-1 attracts hematopoietic progenitors
by activating the PI-3K/PIP<sub>3</sub> signal cascade.  Cells were treated with
SDF-1 and then fixed and stained with a GFP-tagged form of the
pleckstrin homology (PH) domain of GRP1 (the PH doman binds selectively
to PIP<sub>3</sub> ).  A mutant PH GRP1 GFP-tagged form unable to bind
PIP<sub>3</sub> was used to determine background fluorescence.  The data
show that cells were identical in fluorescent response, and that both
lSK and lSk cells produce PIP<sub>3</sub>.  The use of PI-3K inhibitors
completely blocked the motility and chemotaxis of both lSK and lSk cells.
<p>
Between 0.1-1 million cells were washed 3x in PBS and pellets stored frozen
at &minus;80&deg;.  Pellets were resuspended in 9 M urea, 2 M thiourea,
4% CHAPS, 0.5% TX-100 to a 2.75 mg/ml protein concentration.  These are
vortexed on platform for 2 min at room temp to complete lysis and spun
13,500 rpm on microfuge 10 min.  Supernatants are transferred to fresh
tube and samples treated with 1 mM Tris(tris2-carboxyethyl)phosphine for
60 min at 37&deg;.  For every 25 &micro;g protein, 20 nmol of Cy3 or Cy5
labeling dye (Amersham Biosciences) in DMF is added, reaction done for 30 min
at 37&deg;.  Lysis buffer with 4% (v/v) immobilized pH gradient (IPG)
3-10 nonlinear containing 20 mM DTT is added.  Samples are iced before
storage at &minus;80&deg;.  2D-separation is done using the IPG 3-10NL
24 cm strips of Amersham Biosci using Multiphor II (for 72 kVh) and then
the 2nd dimension of the ISODALT SDS-PAGE system (Amersham Biosci).
Visualization is by scanning gels on 532 nm (Cy3) and 633 (Cy5)
Typhoon 8600 flourescent scanner in gel format.
<p>
2D-DIGE (2D-difference gel electrophoresis) can be done using maleimide
CyDye labeling.  This might pick up the difference in proteins responsible
for the difference in motility between lSK and lSk.  The first generation
of Cy dyes are <i>N</i>-hydroxysuccinimidyl esters that label
&epsilon;-amino groups on Lys residues which label 5% of proteins.
The maleimide dyes label Cys sulfhydryl protein to saturation and they
provide enhanced sensitivity.  Cy3 and Cy5 maleimide dyes are mass- and
charge-matched.  A cell line was test labeled with Cy3 and Cy5 to see
if there were any differences in labeling properties, and there were none.
<p>
25 &micro;g protein samples representing about 250,000 cells from lSK and
lSk cells labeled with Cy3 or Cy5, resp., and run on 2D gel.  Both dyes
were used on both cell types to control for dye-labeling artefacts.
Five sets of cells were compared to establish significant differences in
expression (four Cy5 and one Cy3).  A mean value of 550 &plusmn; 32
protein spots was found, and the gel image analysis showed major differences
between the two cell populations. 122 spots were unique in the lSK cells,
and 41 spots in teh lSk cells.  Difference maps were constructed to
look at the relative fold differences.
<p>
One set of proteins has a mass of about 80 kDa and form 7 spots between
pI 5-7.  These proteins may be splice variants with minor changes in mass
or might be posttranslationally modified.  The database search of mass
fingerprints identified <i>adseverin</i>, also known as scinderin or
gelsolin-like protein.  These mediate actin capping, nucleation, and
filament splicing.  Spot volumes were greater in lSk cells than lSK.
And so were spots slightly higher above these in the 2D gel (slightly
higher mass), corresponding to gelsolin.  mRNA level expression also
correlated with protein level expression:  1.3 fold in adseverin vs.
4-fold in gelsolin.
<p>
A variety of mass spectrophotometric techniques were used to determine
the nature of the pI difference in spots.  N-acetylation was observed
for adseverin, and it is known that acetylation is used to modify
cytoskeletal proteins.  The acetylated peptide AQELQHPEFAR corresponds to
amino acids 2-12 of adseverin:  Met is cleaved from the amino terminus and
the alanine residue is acetylated (uslly happens to 2nd amino acid when it
is small and neutral).  Actin filament regulation of this kind alone could
account for differences in motility of the two cell types.  The actin
levels are equivalent.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Friedman DB, Hill S, Keller JW, Merchant NB, Levy SE, Coffey RJ, Caprioli RM
 </span>
 <span class="year">
(2004)
 </span>
 <span class="jname">
Proteomics
 </span>
 <span class="jvol">
4,
 </span>
 <span class="pages">
793-811.
 </span>
<p class="reftitl">
Proteome Analysis of Human Colon Cancer by Two-Dimensional Difference
Gel Electrophoresis and Mass Spectrometry
<p class="abstract">
<b>ABSTRACT</b>.
Two-dimensional difference gel electrophoresis (2-D DIGE) coupled with mass spectrometry
(MS) was used to investigate tumor-specific changes in the proteome of
human colorectal cancers and adjacent normal mucosa. For each of six patients with
different stages of colon cancer, Cy5-labeled proteins isolated from tumor tissue were
combined with Cy3-labeled proteins isolated from neighboring normal mucosa and
separated on the same 2-D gel along with a Cy2-labeled mixture of all 12 normal/tumor
samples as an internal standard. Over 1500 protein spot-features were analyzed in
each paired normal/tumor comparison, and using DIGE technology with the mixedsample
internal standard, statistically significant quantitative comparisons of each protein
abundance change could be made across multiple samples simultaneously without
interference due to gel-to-gel variation. Matrix-assisted laser desorption/ionization-
time of flight (MALDI-TOF) and tandem (TOF/TOF) MS provided sensitive and
accurate mass spectral data for database interrogation, resulting in the identification
of 52 unique proteins (including redundancies due to proteolysis and post-translationally
modified isoforms) that were changing in abundance across the cohort. Without
the benefit of the Cy2-labeled 12 sample mixture internal standard, 42 of these proteins
would have been overlooked due to the large degree of variation inherent between
normal and tumor samples.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Hirabayashi J
 </span>
 <span class="year">
(2004)
 </span>
 <span class="jname">
Glycoconj. J.
 </span>
 <span class="jvol">
21,
 </span>
 <span class="pages">
35-40.
 </span>
<p class="reftitl">
Lectin-Based Structural Glycomics:  Glycoproteomics and Glycan Profiling
<p class="abstract">
<b>ABSTRACT</b>.
Structural glycomics (SG) plays a fundamental part of concurrent glycobiology aiming at comprehensive elucidation of
glycan functions (i.e., functional glycomics) in the context of post-genome sciences. The SG project started in April 2003
and will continue for 3 years in the framework of NEDO (New Energy and Industrial Technology Organization) under the
METI (the Ministry of Economy, Trade, and Industry), Japan. The main purpose of the project is the development of highthroughput
and robust machines, which should greatly contribute to the structural analysis of complex glycans. In this
chapter, 2 major research items, i.e., (1) glycoproteomics, which enables comprehensive analysis of glycoproteins, and
(2) glycan profiling by means of lectins, are described. For the latter, frontal affinity chromatography has been adopted
as a starting tool for comprehensive analysis of the interaction of 100 lectins and 100 oligosaccharides under the concept
of hect-by-hect, which refers to 100 &times; 100.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Balogh MP
 </span>
 <span class="year">
(2004)
 </span>
 <span class="jname">
LC GC Europe
 </span>
 <span class="jvol">
17,
 </span>
 <span class="pages">
152-159.
 </span>
<p class="reftitl">
MS In Practice:  Debating Resolution and Mass Accuracy
<p class="Para1">
With the use of an MS as a detector in LC systems, there are increased
expectations regarding its performance.  Its primary functions are to
produce accurate mass signals at high resolution.  But as a detector,
sensitivity and dynamic range also become important.  One important thing
overlooked however in LC-MS work is the expectation that a single run
provide all the needed work.  Instead, runs should be in triplicate
and they should have well-prepared samples, with accurate mass.
<p>
An anecdote from K. Biemann from the early 1960s is given.  The drug
reserpine was marketed by CIBA starting from the 1930s but not structurally
determined until the 1940s.  Techniques such as Zn dust distillation
involved purification of alkyl indoles and pyridines through their crystallization
in picrates, which are then identified by melting point and elemental analysis.
Biemm used MS with something called mass spectrometic shift technique instead
of mp determinations and IR.  One reviewer asked Biemann if the appropriate
level of characterization had been done.  One journal (J Am Soc Mass Spectrom)
requires consideration of <q>acceptable uncertainty</q> for any method
used to support structural determinations.  For example, when parent
masses having C<sub>0-100</sub>, H<sub>3-74</sub>, O<sub>0-4</sub>,
N<sub>0-4</sub> show an <i>m/z</i> 118, no alternative formulae within
34 ppm can exist.  If the mass becomes <i>m/z</i> 750 for
C<sub>0-100</sub>, H<sub>25-110</sub>, O<sub>0-15</sub>, N<sub>0-15</sub>,
there are now 626 alternatives and all within 5 ppm.  For <i>m/z</i>
118, this now becomes 0.018 ppm to eliminate alternatives.
<p>
FTICR MS instruments provide both sensitivity and high resolution.  Bruker
and others have been market leaders in FTICR although Thermo recently
introduced a model.  AG Marshall at the National High Field Fourier
Transform ICR Center in Florida plotted the logarithm of the spacing
between resolved peaks in datltons (Y) against the ratio of instrument resolution
to molecular weight (X).  Bovine ubiquitin in an electrospray is the example.
At a plot point up to (X = 10<sup>&minus;4,</sup>, Y=10<sup>&minus;3</sup>),
the resolution is weakest but able to resolve charge states.  Between
the last X,Y and (10<sup>&minus;9</sup>, 10<sup>&minus;2</sup>),
resolution of identically charged adducts ([M+<i>n</i>H)]<sup><i>n+</i></sup>
and [M+<i>n</i>&minus;1H+Na]<sup><i>n+</i></sup> is enough when <i>n</i>
is large.  Between the last (X,Y) and (1, 10<sup>&minus;10</sup>),
isotopic peaks of an ion are resolved as baseline resolution of
highly charged ions (multiple peaks seen within an <i>m/z</i> 1 scale unit.
And beyond the last (X,Y) one can resolve isomers to 0.005 <i>m/z</i> units.
<p>
The ion cyclotron was first reported by Hipple, Sommer and Thomas in the late
1940s and then Cooley and Tukey developed the fast FT algorithm.  Bendix
Corp. made the first ToF instrument, and later a more compact instrument
with reflectron to correct for effects of kinetic energy distribution of ions
was made based on Boris Mamyrin's work.  In general ToF offers higher
resolution than quadrupoles.
<p>
Mass accuracy is extremely critical.  It is the ability to calibrate or
measure the instrument response against a known entity.  Tolerance is measured
in ppm, or the deviation from a response to the known mass.  To achieve the
highest accuracy, continuous calibration with a lockmass is necessary.
<p>
LC-ToF and Q-ToF provide the best mass range.  ToF is the best choice for
broad or general applications, whereas FT-MS and magnetic sector are specialized.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Vitorino R, Lobo MJC, Ferrer-Correira AJ, Dubin JR, Tomer KB, Domingues PM,
Amado FML
 </span>
 <span class="year">
(2004)
 </span>
 <span class="jname">
Proteomics
 </span>
 <span class="jvol">
4,
 </span>
 <span class="pages">
1109-1115.
 </span>
<p class="reftitl">
Identification of Human Whole Saliva Protein Components Using Proteomics
<p class="Para1">
Whole saliva contains mucins and other high MW glycoproteins
which create problems for 1st dimension IEF.  1 ml whole saliva was
obtained from healthy gums at 8 am after overnight fasting, immediately
spun max speed in microfuge for 20 min at 4&deg;, and Lowry assay done.
60 &micro;g was loaded to 13 cm pH 3-10 IPG strip (Amersham Pharmacia)
with 3-10 Immobilines, thiourea, CHAPS, and urea.  This was run in a
12.5% gel and then silver-stained.  For tryptic peptide preparation,
150 &micro;g was loaded, stained with Coomassie.
<p>
With the analytical method, 200 spots could be detected.  Good resolution
was obtained with a preparative gel technique.  60% of total protein
is &alpha;-amylase in the 50-70 kDa range with multiple isoenzymes.
Keller et al. have reported that the amylase fractionates into two major
groups, one at 59 and another at 63 kDa.  The higher MW group is because
of N-linked sugars.  The resolution of the gel is sorely affected as
a result.  With optimized protein levels more proteins are visible on
the acidic zone compared with earlier reports.  About 100 spots were
selected for trypsin digest and MALDI-ToF-ToF (ABi 4700 Proteomics
Analyzer) in the <i>m/z</i> 800-4000 ranges.  Resolution was 15000 with
mass accuracy of 25 ppm.  A switch to MS/MS mode (excluding known background
or contaminant peaks) was of the two most intense peaks in the parent
spectrum.  The MS/MS range was 70 (to include immonium ions) to the parent
mass; resolution was 8000 and mass accuracy 75 ppm.  Fragmentation
spectra saw peaks with <i>y</i> and <i>b</i>-series, immonium, amine,
and water loss ions.  40% of spectra produced at least a partial sequence
even for peptides with <i>m/z</i> values &gt; 2000, and 15% of peptides
were fully sequenced.  Identification was accepted with a Mascot score of
90 with confirmation using MW/pI values.  43 spots were identified this
way, with identical identifications for several spots.  Glycosylation
and phosphorylation account for PTMs.
<p>
Lipocalin-1 was seen as four spots, prolactin with 5 spots, carbonic
anhydrase VI with five spots, lysozyme with 2, histatin 1 with one,
PLUNC-palate lung nasal epithelial clone, lactotransferrin, secretory
component, common cystatins (SN, three spots; SA, one; SA-III, -D, and
-C, one), two uncommon cystatins, several fragments of &alpha;-amylase.
<p>
12 new proteins were identified.  Statherin and histatin-3 found by others
were not found.  The same is true for interleukin-1, IgJ, and fatty-acid
binding protein, which might be suppressed by peptide masses of
co-migrating proteins or are subject to individual variation.
Proline-rich proteins are more difficult to analyze since they produce
few tryptic digest peptides (3 or 4).  Four spots of carbonic anhydrase
VI were seen.

<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Mitulovi&#263; G, Stingl C, Smoluch M, Swart R, Chervet J-P, Steinmacher I,
Gerner C, Mechtler K
 </span>
 <span class="year">
(2004)
 </span>
 <span class="jname">
Proteomics
 </span>
 <span class="jvol">
4,
 </span>
 <span class="pages">
2545-2557.
 </span>
<p class="reftitl">
Automated, On-Line Two-Dimensional Nano Liquid Chromatrograph Tandem
Mass Spectrometry for Rapid Analysis of Complex Protein Digests
<p class="Para1">
2-D chromatography was done on a capillary HPLC from LC Packings featuring
FAMOS autosampler, UltiMate micropump with UV detector and SWITCHOS
column switcher.  All MS data were acquired using a Finnigan LCQ classic
ion trap MS that included a homemade nano-ionization source with a
luminary strip and red diode to avoid reflection. MS/MS data were obtained
in data-dependence setting with dynamic exclusion.  Database searches
of MS/MS were done using MASCOT (on a Compaq alpha cluster 8 processor
running under Linux).  UltiMate was in nanomode at 200 nl/min.  SWITCHOS
loading ran at 35 &micro;l/min and its flow transported sample from the
loop to SCX column.  Its flow also washed the C<sub>18</sub> trap column
to prevent matrix and/or salt from reaching the nano sep column and MS.
The FAMOS was operated in large inject volume mode with 250 &micro;l loop.
Columns were BioX-SCX 500 &micro;m i.d. &times; 15 mm LC Packings;
300 &micro;m i.d. &times; 5 mm cartridge with PepMap100 3 &micro;m
C<sub>18</sub> LC Packings; a PepMap nano sep 75 &micro;m i.d. &times;
15 cm PepMap100 3 &micro;m C<sub>18</sub> LC Packings.  The system was
tested with tryptic digests of bovine heart cytochrome <i>c</i> (Sigma
C-2037), BSA Fraction V (Fluka 05480), horse liver alcohol dehydrogenase
(Sigma 17233), &beta;-galactosidase and serotransferrin (Sigma).
Protein were reduced with DTT (Sigma D-9163) and alkylated with IAA
(Sigma I-4386).  These were dialyzed away with 0.1 M ammonium bicarbonate
pH 8; and trypsinized overnight at 37&deg; using Promega sequencing grade
trypsin.  A 1:10 dilution with 0.5% TFA was done and loaded to SCX equilibrated
with 0.1% formic acid (Fluka 06440),
<p>
RP columns alone (1D chromatography), even with long gradient elutions
yield poor results in MS/MS and database searching.  Alternatively, tryptic
peptides can be held on to SCX trap and then sep column, and then eluted to RP
trap and sep.  The peak capacity of a multidimensional chromatography
system is effectively the product of peak capacities of each dimension
in the system.  Column switching permits use of loading columns with larger
i.d.s in first dimension (the SCX had a 0.5 mm i.d.) and this leads to more
data points in later dimensions.  The plumbing config was setup with
the FAMOS 6-port valve and the two 10-port valves on SWITCHOS.  For the
FAMOS, port 3 leads to the injection needle, ports 2-5 to holds a sample
loop, port 1 to the SWITCHOS loading solvent (at 35 &micro;l/min), port
6 to port 1 of valve B on SWITCHOS, (and port 4 to waste?).  On SWITCHOS
valve B, port 2 runs to top of SCX column, port 9 to the bottom of the column,
and port 10 to port 9 of valve A of the SWITCHOS.  SWITCHOS valve A features
ports 4 and 8 to waste, port 3 to one end of an RP 300 &micro;m i.d. trap
column, port 10 to the other end of the trap column, port 2 to the gradient
pump running at 200 nl/min, and port 1 leads to the RP 75 &micro;m i.d.
nano sep column on the UltiMate.
<p>
With inject mode, peptides in the FAMOS loop are loaded on to the SCX
column where positively charged proteins are bound.  Unbound proteins
continue toward valve A of the SWITCHOS which holds an RP trap column.
Peptides that continue to remain unbound goes out the waste line (port 4).
In a wash mode, valve B is switched while valve A is not:  the 35 &micro;l/min
wash line comes in at port 1 and goes out port 10 of valve B and directly
to port 9 and to 10 of valve A, causing washing of the RP trap column to
remove matrix and salts.  In elute mode, valve A is now switched and
the RP trap eluted via the gradient line on to the RP nanosep column, and
peptides separated on to the detector (MS).  For step 4, a conditioning
mode switches both valves A and B and the columns are reconditioned.
<p>
500 fmol of each of 5 proteins was digested and run on to a nanosep RP
column (1D sep) alone, and the base peak chromatogram shows a high number
of peptides eluting within a short range of time (from 18-48 min in a
55 min run).  0.1% formic acid and 50 mM phosphate were then used as loading
mobile phases for the SCX column set up in line at the top of the RP column:
thus all peptides off the RP column represent unbound peptides to the SCX.
A greater amount of peptides remained unbound to SCX with 0.1% FA,
as the base peak had a higher intensity than in direct injection of the RP
column.  With phosphate as the loading mobile phase to SCX, more peptides
bound to SCX.  These were eluted from SCX to MS detection with increasing
steps of 50, 100, 200, and 500 mM ammonium acetate salt plugs.
A disadvantage of off-line loading is that it takes several hours
and sample loading requires some experience to be successful.  Multiphase
single column use results in faster sample loading and less loss but
does not allow removal of salts before the RP separation.
<p>
Loading buffer selection is not easy when coupling an SCX to an RP trap.
Conditions that improve trapping on RP reduce the amounts of peptides that
bind to SCX.  0.05% TFA resulted in low SCX peptide binding even though
it helps create ion pairs for RP.  Use of formic acid represented only a
slight improvement in peptide binding to SCX.  Phosphate represented
a better compromise.  It stabilizes pH over a large range and can promote
RP trapping.  The problem is that phosphate is not compatible with
ESI-MS, and so the RP trap requires washing prior to elution.  A switch
to 0.1% FA is appropriate, but wash time needs to be optimized, since
peptides will eventually elute from the RP trap if prolonged beyond removal
of all phosphate.  Each HPLC parameter depends on several factors.
Flow rates are set according to column (SCX) dimensions, column capacity,
and connecting tube types.  Loading times will generally be affected by
sample volume, but complexity and matrix of the sample is important,
as is the loading column capacity.  Matrix and salts can displace peptides
on the columns if loading times are too long and there is competitve affinity.
Note that in the plumbing configration that the RP trap column is
bidirectional and eluted through a back flush onto the RP nanosep column:
this reduces peak broadening.
<p>
Step elution (salt plugs) was not always clean.  Some proteins eluted in more
than one fraction (step elution).  Very abundant proteins even eluted in
several fractions if strongly bound to SCX.  Low abundance proteins,
the aim of this work, are often obscured in complex samples in which
a large number of proteins predominate.  Comparisons were made to a 1D
separation and the sensitivity of the low abundance protein increased 2-fold
with a 1D separation.  Use of phosphate pH 3.0 resulted in higher MASCOT
scores (sequence coverage), but at the expense of a higher MS/MS background
compared to FA.  It was clear that not all of the phosphate was or could be
washed free.  Optimization of the phosphate concentration and pH showed
that as little as 2.5 mM phosphate at pH 2.5 could result in reduced
effects of MS without affecting ion exchange.
<p>
Jurkat cell nuclear matrix protein was prepared and separated on 2-D gels
run from pH 3-8 and separated between 20-250 kDa.  94 gel spots were analyzed
by MS whereas in contrast 174 proteins could be identified in a single
10 &micro;g run on LC.  108 of the 130 proteins found by LC-MS/MS were
not found in the gels, largely because of high basic pI values and low
<i>M</i><sub>r</sub>.  46 of the 68 proteins that could be theoretically
resolved on 2-D gels were in fact resolved;  this score is higher than others
who have done comparisons of LC-MS/MS and 2DGE-MS.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Hopfgartner G, Varesio E, Tscha&auml;pp&auml;t V, Grivet C, Bourgogne E,
Leuthold LA
 </span>
 <span class="year">
(2004)
 </span>
 <span class="jname">
J. Mass Spectrom.
 </span>
 <span class="jvol">
39,
 </span>
 <span class="pages">
845-855.
 </span>
<p class="reftitl">
Triple Quadrupole Linear Ion Trap Mass Spectrometer For The Analysis
of Small Molecules and Macromolecules
<p class="abstract">
<b>ABSTRACT</b>.
Recently, linear ion traps (LITs) have been combined with quadrupole (Q), time-of-flight (TOF) and
Fourier transform ion cyclotron resonance (FT-ICR) mass spectrometery (MS). LITs can be used either
as ion accumulation devices or as commercially available, stand-alone mass spectrometers with MSn
capabilities. The combination of triple quadrupole MS with LIT technology in the form of an instrument
of configuration QqLIT, using axial ejection, is particularly interesting, because this instrument retains the
classical triple quadrupole scan functions such as selected reaction monitoring (SRM), product ion (PI),
neutral loss (NL) and precursor ion (PC) while also providing access to sensitive ion trap experiments.
For small molecules, quantitative and qualitative analysis can be performed using the same instrument.
In addition, for peptide analysis, the enhanced multiply charged (EMC) scan allows an increase in
selectivity, while the time-delayed fragmentation (TDF) scan provides additional structural information.
Various methods of operating the hybrid instrument are described for the case of the commercial Q
TRAP (AB/MDS Sciex) and applications to drug metabolism analysis, quantitative confirmatory analysis,
peptides analysis and automated nanoelectrospray (ESI-chip-MS) analysis are discussed.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Choi J-K, Chae H-Z, Hwang S-Y, Choi H-I, Jin L-T, Yoo G-S
 </span>
 <span class="year">
(2004)
 </span>
 <span class="jname">
Electrophoresis
 </span>
 <span class="jvol">
25,
 </span>
 <span class="pages">
1136-1141.
 </span>
<p class="reftitl">
Fast Visible Dye Staining of Proteins in One- and Two-Dimensional
Sodium Dodecyl Sulfate-Polyacrylamide Gels Compatible With Matrix-Assisted
Laser Desorption/Ionization-Mass Spectrometry
<p class="abstract">
<b>ABSTRACT</b>.
A fast and matrix assisted laser desorption/ionization-mass spectrometry (MALDI-MS)
compatible protein staining method in one- and two-dimensional sodium dodecyl sulfate-
polyacrylamide gel electrophoresis (1- and 2-D SDS-PAGE) is described. It is
based on the counterion dye staining method that employs oppositely charged two
dyes, zincon (ZC) and ethyl violet (EV) to form an ion-pair complex. The protocol,
including fixing, staining and quick washing steps, can be completed in 11.5 h
depending upon gel thickness. It has a sensitivity of 48 ng, comparable to that of
colloidal Coomassie Brilliant Blue G (CBBG) staining with phosphoric acid in the staining
solution. The counterion dye stain does not induce protein modifications that complicate
interpretation of peptide mapping data from MS. Considering the speed, sensitivity
and compatibility with MS, the counterion dye stain may be more practical than
any other dye-based protein stains for routine proteomic researches.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Tang N, Tornatore P, Weinberger SR
 </span>
 <span class="year">
(2004)
 </span>
 <span class="jname">
Mass Spec. Rev.
 </span>
 <span class="jvol">
24,
 </span>
 <span class="pages">
34-44.
 </span>
<p class="reftitl">
Current Developments in SELDI Affinity Technology
<p class="abstract">
<b>ABSTRACT</b>.  The overall history and recent advancements in Surface-
Enhanced Laser Desorption/Ionization (SELDI) affinity technology
is reviewed. A detailed account of SELDI technology,
utilizing Immobilized-Metal Affinity surfaces, pseudo-specific
chromatographic surfaces, and biospecific interactive surfaces,
is presented with particular emphasis placed upon examination
of fundamental characteristics as well as specific applications for
each. Finally, a detailed review of the specific use of such affinity
surfaces in fundamental aspects of clinical, process, and research
proteomics activity is presented.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Zaia J
 </span>
 <span class="year">
(2004)
 </span>
 <span class="jname">
Mass Spec. Rev.
 </span>
 <span class="jvol">
24,
 </span>
 <span class="pages">
161-227.
 </span>
<p class="reftitl">
Mass Spectrometry of Oligosaccharides
<p class="abstract">
<b>REVIEW</b>.
<p>
<b>ABSTRACT</b>.
Glycosylation is a common post-translational modification to cell
surface and extracellular matrix (ECM) proteins as well as to
lipids. As a result, cells carry a dense coat of carbohydrates on
their surfaces that mediates a wide variety of cellcell and cell
matrix interactions that are crucial to development and function.
Because of the historical difficulties with the analysis of complex
carbohydrate structures, a detailed understanding of their roles
in biology has been slow to develop. Just as mass spectrometry
has proven to be the core technology behind proteomics, it stands
to play a similar role in the study of the functional implications of
carbohydrate expression, known as glycomics. This review
summarizes the state of knowledge for the mass spectrometric
analysis of oligosaccharides with regard to neutral, sialylated,
and sulfated compound classes. Mass spectrometric techniques
for the ionization and fragmentation of oligosaccharides are
discussed so as to give the reader the background to make
informed decisions to solve structure-activity relations in glycomics.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Li J, Liu S, Osterman T, Zhang Y, Coppage H, Pedrick N, Witzmann FA
 </span>
 <span class="year">
(2004)
 </span>
 <span class="jname">
Anal. Biochem.
 </span>
 <span class="jvol">
332,
 </span>
 <span class="pages">
187-190.
 </span>
<p class="reftitl">
A Software Utility for Creating Interactive Maps for 2D Gel-Based Proteomics
<p class="abstract">
<b>Notes &amp; Tips</b>.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Bogdanov B, Smith RD
 </span>
 <span class="year">
(2004)
 </span>
 <span class="jname">
Mass Spec. Rev.
 </span>
 <span class="jvol">
24,
 </span>
 <span class="pages">
168-200.
 </span>
<p class="reftitl">
Proteomics By FTICR Mass Spectrometry:  Top Down and Bottom Up
<p class="Para1">
<b>REVIEW</b>.  FTICR provides the highest mass resolution, resolving power, mass accuracy,
and sensitivity of all MS technologies.  One field that has benefitted
from FTICR is proteomics.  Proteomics can be defined as the direct
qualitative and quantitative analysis of the full complement or subset
of proteins present in an organism, tissue, or cell under given set of
physiological or environmental conditions.  Analyses can be from the
the bottom up (peptide to protein) or top down (protein to peptide).
<p>
Identification of peptides (the <q>bottom up</q> approach) describes
the determination of the peptides masses from a parent protein and the
other from MS/MS fragmentation of one or more of the peptides.
The first approach is peptide mass fingerprinting.  The second approach
is based on information from dissociation (collisional activation)
of one or more polypeptides isolated in the first stage of MS analysis.
Bottom-up approaches do not provide info on the whole protein sequence.
<p>
Peptide fragmentation in the FTICR is achieved by sustained off-resonance
irradiation (SORI) collision-induced dissocation (CID), infrared
multi-photon dissociation (IRMPD), both with CO<sub>2</sub> laser and
blackbody radiation, and surface-induced dissociation.  Peptides fragment
primarily into <i>b</i>- and <i>y</i>-type ions for slower mutli-step
activation processes;  fragmentation efficiency and relative fragment
ion abundances are dependent upon amino acid sequence.  Peptide CID
in Q-ToF or QIT (quadrupole ion-trap) is generally faster because FTICR
requires re-establishment of high vacuum before excitation and detection.
CID is also done outside the FTICR, in a rf-multipole device.
<p>
To achieve controlled, predictable fragmentation, Lee et al prepared
single and double sodiated peptides on MALDI and ESI using SORI.
Sodiated peptides showed highy selective cleavage at Asp residues.
Peptides without acidic residues show sequential cleave from C-term.
The mechanims was described as a salt-bridge chemistry, where Na ion
stabilizes proton transfer from Asp residue to adjaent amide
nitrogen.
<p>
SORI-CID has been used with negative mode FTICR to assess the expectation
that negative ions would produce good spectra.  Dominant fragments were
<i>c</i>, <i>y</i> and internal fragments, with <i>a</i>, <i>b</i>, <i>x</i>,
and <i>z</i> ions formed.  Much of the fragments were cleavage next
to acidic (Asp, Glu, and cysteic acid) residues on either side (N or C)
of the residue.  Acquiring a single spectrum takes &gt;2 s and even with
that long a scan, there are many problems that give low fragmentation
efficiency;  FTICR with in-cell SID was designed with a high acquisition
rate (300 ms/spectrum) which traps fragment ions with a high efficiency,
well controlled kinetic energy of the parent, and high fragmentation.
A modified SORI-CID was used for a MALDI source.
<p>
In 1998 electron capture dissociation was pioneered using FTICR
and remains unique to it.  ECD was applied to larger multiply charged
protein molecules, and was applied to peptides having at least two charges
in order to induce fragmentation.  Low-energy (&lt;1 eV) electrons
were captured by [M + nH]<sup>n+</sup> ions to produce
[M + nH]<sup>(n&minus;1)+&bull;</sup>.  These fragment rapidly and randomly
(fast recombination of thermal electrons and protonated sites) than that
by CID, with cleavage of N-&alpha;C to produce N-terimal <i>c</i>-type
and C-terminal <i>z</i>-type ions.  Post-translational modifications that
are labile to other fragmentation processes are retained with ECD.  ECD
also cleaves disulfide bonds.
<p>
<b>Accurate Mass and Time (AMT) Tag</b>.  Polypeptide AMT tags are a
bottom-up approach in which proteins in large numbers (as in cell lysates)
are digested
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Candiano G, Bruschi M, Musante L, Santucci L, Ghiggeri GM, Carnemolla B,
Orecchia P, Zardi L, Righetti PG
 </span>
 <span class="year">
(2004)
 </span>
 <span class="jname">
Electrophoresis
 </span>
 <span class="jvol">
25,
 </span>
 <span class="pages">
1327-1333.
 </span>
<p class="reftitl">
Blue Silver:  A Very Sensitive Colloidal Coomassie G-250 Staining for
Proteomic Analysis
<p class="Para1">
A study was done to see assess how quickly the various stains reached
a certain staining intensity over time.  Three staining preparations
were tested:  (1) the original recipe of Neuhoff et al., (2) the
<q>blue silver</q> stain reported here, (3) a commercial Neuhoff-like
recipe from Sigma.  The fastest and most intense staining was with the
blue-silver:  80% of the dye is taken up in the 1st hour vs. 40% with
the Neuhoff staining, and virtually nil with the commercial dye.
After 50 hours of staining, the Neuhoff stain is 2.5 times more intense
and blue-silver is 3.3 times more intense.  A dilution curve of BSA runs
as bands on a 1-D gel shows that staining can be sensed with as little
as 1 ng, possiby 0.75 ng.
<p>
2-D maps of human kidney tubular epithelial cells were run and stained
with Neuhoff, silver, and <q>blue silver.</q>  The blue-silver
showed far more sensitivity than did the Neuhoff.  To prepare gels,
the cultured cells were lysed in a 5 mM phosphate pH 8 buffer with
inhibitors of phosphatases and proteases.  Protein content was adjusted
to 5 mg/ml, adding 10% SDS, 3% DTT, 40 mM Tris, and 0.1 mM EDTA, heating
for 5 min at 100&deg;.  Soluble sample was treated with a delipidating
extraction solution of 1:12:1 tri-<i>n</i>-butylphosphate:acetone:methanol
and cooled on ice.  14 ml of this was added to SDS-solubilized cytosolic
epithelial cell with a final acetone concentration of 80% and left at
4&deg; for 90 min.  Centrifugation was at 2800 &times; <i>g</i> for 20 min
at 4&deg;.  Pellet was re-washed with delipidizing solution and spun and
then air-dried.  Pellet was re-suspended in 7 M urea, 2 M thiourea,
4% CHAPS, 5 mM TBP, 20 mM iodoacetamide, 40 mM Tris, and 0.1 mM EDTA
pH 8.8.  The solution sat 3 h before focusing was started.  DTT was
added to neutralize iodoacetamide before focusing, and 100 &micro;g was
loaded on to an IPG strip.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Hirano H, Islam N, Kawasaki H
 </span>
 <span class="year">
(2004)
 </span>
 <span class="jname">
Phytochem.
 </span>
 <span class="jvol">
65,
 </span>
 <span class="pages">
1487-1498.
 </span>
<p class="reftitl">
Technical Aspects of Functional Proteomics in Plants
<p class="abstract">
<b>REVIEW</b>.
<p>
<b>ABSTRACT</b>.
Since the completion of genome sequences of several organisms, attention has been focused to determine the function and
functional network of proteins by proteome analysis. This analysis is achieved by separation and identification of proteins, determination
of their function and functional network, and construction of an appropriate database. Many improvements in separation
and identification of proteins, such as two-dimensional electrophoresis, nano-liquid chromatography and mass
spectrometry, have rapidly been achieved. Some new techniques which include top-down mass spectrometry and tandem affinity
purification have emerged. These techniques have provided the possibility of high-throughput analysis of function and functional
network of proteins in plants. However, to cope with the huge information emerging from proteome analyses, more sophisticated
techniques and software are essential. The development and adaptation of such techniques will ease analyses of protein profiling,
identification of post-translational modifications and proteinprotein interaction, which are vital for elucidation of the protein
functions.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Cort&oacute;n M, Villuendas G, Botella JI, San Mill&aacute;n,
Escobar-Morreale HF, Peral B
 </span>
 <span class="year">
(2004)
 </span>
 <span class="jname">
Proteomics
 </span>
 <span class="jvol">
4,
 </span>
 <span class="pages">
438-441.
 </span>
<p class="reftitl">
Improved Resolution of The Human Adipose Tissue Proteome at Alkaline
and Wide Range pH by Addition of Hydroxyethyl Disulfide
<p class="Para1">
Samples from 10 morbidly obese women and 10 female controls were
cut into small pieces and frozen at &minus;75&deg; until processed.
A Polytron PT-1200C with a 8.4 M urea, 2.4 M thiourea, 5% CHAPS,
50 mM DTT and 1% 3-10 ampholyte was used for lysis (2 ml/g fat).
Thiourea used in combination with urea helps to improved protein
solubility.  The suspension was shaken at room temp for 60 min,
spun at 70,000 rpm for 1.5 h at 19&deg; using Beckman TL-100.
Solubilization of lipid-rich proteins is best done by taking the
infranatant (part below the fat supernatant).  A Bradford assay is
used for protein.  The original extract was solubilized in 8 M urea,
2 M thiourea, 2% CHAPS, 0.5% 3-10 ampholyte, 0.002% bromophenol blue,
and 10 mM DTT (alternatively 50 mM DTT or 10 mM hydroxyethyl disulfide
[HED] was sometimes used in place of DTT.  IPG strips with pH 6-11 range
were loaded with 100 &micro;g protein in 100 &micro;l rehydration solution
using a cup-loading method positioned on the anode.  For the IPG pH 3-10
80 &micro;g protein was loaded using in-gel sample rehydration in
350 &micro;l solution.  The 18 cm strips were focused at 50 &micro;A
per strip with the following running conditions:  for IPG 6-11, 0-300 V
in 60 sec, then 300 V for 3 h, 300 to 600 V in 2 h, 600 to 1000 V in 2 h,
and 1000 to 8000 V in 3 h, and then 5 h at 8000 V.  The IPG 3-10 strips
were run in conditions specified by G&ouml;rg et al. with the modification
that there was an additional step at 120 V for 60 min, and 90 min ramp from
1000-8000 V rather than 30 min.  The 8000 V step was maintained until
35 kVh was reached.  For the 2nd dimension, gels were run in a Protean
II XL system with 12.5% gels in a 185 &times; 200 &times; &times; 1 mm
and fixed overnight and then silver-stained.
<p>
In the alkaline range of 2-DE maps horizontal streaking is common.
This is because the reducing agent is degraded and because protein thiols
oxidize, and because there is water transport to the anode because of
reverse electroendosmotic flow.  Some use paper wicks immersed in DTT
at the cathode during focusing, and it is thought that adding 10% or
an isopropanol/glycerol mixture reduces water transport.  Others compensate
by reducing the total focusing time to a minimum (ramping the final voltage
quickly in doing so).
<p>
Fig. 1 shows that the use of the Amersham-Biosciences DeStreak reagent
in the pH 6-11 strips shows improved separation efficiency.  This confirms
work by others that HED eliminated horizontal streaking.  The same was
observed for pH 3-10 strips (Fig. 2) with much better maps when HED rather
than 10 and 50 mM DTT being used.  HED works by oxidizing protein thiol
groups to disulfides and improving gel reproducibility.  Different
concentrations of HED (50, 100, 200 mM) were tried, and they all led to
the same spot patterns.  PDQuest software found 110 and 123 additional
spots with pH 3-10 strips when HED was used rather than 10 and 50 mM DTT,
respectively.   For pH 6-11 strips, only the HED map was analyzed, finding
a total of 488 spots.
<p>
The Shevchenko et al. method with minor variations was used for in-gel
trypsin digestion.  Extracts were spotted on 400 mm AnchorChip MALDI
plate (Bruker) and analyzed on Bruker Reflex IV MALDI-ToF in positive
reflectron mode.  Two tryptic autolytic products were used to calibrate
masses (<i>m/z</i> 842.510 and 2211.105).  A mass accuracy of 30 ppm was
found.  The Bruker MS BioTools interface was used to search NCBInr
database (http://www.ncbi.nlm.nih.gov) with Mascot, with no species restriction
or variable modification, 1 missed cleavage, and a mass accuracy of 50 ppm
set.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Cooper JW, Gao J, Lee CS
 </span>
 <span class="year">
(2004)
 </span>
 <span class="jname">
Electrophoresis
 </span>
 <span class="jvol">
25,
 </span>
 <span class="pages">
1379-1385.
 </span>
<p class="reftitl">
Electronic Gel Protein Transfer and Identification Using Matrix-Assisted
Laser Desportion/Ionization-Mass Spectrometry
<p class="Para1">
Fused silica capillaries (50 &micro;m id/192 &micro;m od,
100 &micro;m id/200 &micro;m) were coated internally with linear
polyacrylamide (Kilar &amp; Hjert&eacute;n, 1989).	A gel protein capillary
extraction apparatus has been described previously.  The 50 or 100 &micro;m
coated capillaries are filled with 20 mM Tris pH 8.0 and a modified Teflon
sleeve fit over and placed in contact with the gel.  Tris buffer or deionized
water is placed on gel-capillary interface to establish electrical contact.
The gel itself is set to common ground with Pt wire, set to positive.
Field strength is varied from 100 to 500 V/cm over a 37 cm capillary
length.  To study the effects a detection window is burned 7 cm from
capillary outlet and 214 nm lamp and detector aligned.  The outlet end
of the capillary can be postioned in a drop of solution or placed on
top of a MALDI target (a 20 cm instead of 37 cm capillary is used when
a MALDI target is used; note that the entire solution volume of 20 cm
long &times; 100 &micro;m id has an est. vol of 1.5 &micro;l).  After
extraction, the capillary is fit to a syringe pump and placed on the target
at 1.5 &micro;l/min (for 1 min).  After drying, 1 &micro;l matrix is
pippeted to and allowed to crystallize.  Matrix per spot is either
10 &micro;g CHCA in (2:3:1 v/v/v) isopropanol:water:formic acid or
in 40:60:0.1 MeOH:water:acetic acid or in 50:50:0.1 MeCN:water:TFA;
also 20 &micro;g DHB in 1:9 MeOH:water; or in 10 &micro;g sinapinic acid
in MeCN:water:TFA (30:70:0.1).  An ABi Voyager DE Linear MALDI-ToF with
50 shots per spectrum was done.
<p>
Previous work showed that there is a continuous current drop observed
at the capillary-gel interface during protein extraction, caused by
a depletion of limited carrying electrolytes.  There is also generation
of low conductivity solution plug at the capillary extraction end.
A higher voltage drop per capillary length occurs across small plug of
low conductivity buffer than in the rest of the capillary.  This
higher electric field strength near extraction end promotes electroelution,
but also promotes sample stacking because of the field amplification.
Complexes (SDS-protein) move rapidly across the low conductivity drop
and halt.
<p>
In the system, the gel containing the protein is stained, destained, and
then equilibrated with 20 mM Tris before extraction.  After 2 min of
extraction set at 300 V/cm, the voltage is turned off and the extraction
end moved from gel to reservoir with 20 mM Tris buffer.  Positive voltage
is re-applied and the complexed pushed to the UV detection end.  Peak
heights increased up to 4 min of extraction time; longer times resulted
in tailing of non-stacked proteins.
<p>
Raising the field strength continuously from 100-500 V/cm reduced migration
time.  Peak heights increased with strength and reach a plateau with
300 V/cm.  A constant 300 V/cm was used for further work to avoid Joule
heating.  Peak heights increased 3-fold by varying buffer strength from
20 mM Tris down to deionized water;  the de-I water had the effect of causing
great stacking and greater extraction.
<p>
[continued]
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Lamanda A, Zahn A, R&ouml;der D, Langen H
 </span>
 <span class="year">
(2004)
 </span>
 <span class="jname">
Proteomics
 </span>
 <span class="jvol">
4,
 </span>
 <span class="pages">
599-608.
 </span>
<p class="reftitl">
Improved Ruthenium II Tris (Bathophenanthroline Disulfonate) Staining
and Destaining Protocol for A Better Signal-To-Background Ratio and
Improved Baseline Resolution
<p class="Para1">
Proteins labeled covalently with fluorescent dyes can be detected with
high sensitivity.  Noncovalent stains such SYPRO Ruby, along with other
SYPRO dyes and ruthenium II tris-bathophenanthroline disulfonate (RuBP)
are also effective.  RuBP is cheaper and offers several possibilities
for improvement compared to SYPRO Ruby.
<p>
RuBP was prepared according to Rabilloud et al and the UV/Vis spectral
analysis was identical with published spectrum.  SYPRO Ruby was from
Bio-Rad with a new formulation.  Marker proteins were run on 1D 17.5% gels
in a serial double dilution in the lanes for sensitivity testing.
For 2D gels, pH 3-10 strips are rehydrated 10 h with 300 &micro;l
sample in 8 M urea, 2% CHAPS, 18 mM DTE, 0.5% IPG buffer 3-10 and with
some bromophenol blue dye in which 80 &micro;g <i>E. coli</i> proteins
are present.  The focusing program is 200 &micro;A per strip, 150 V for
60 min, 300 V for 1 h, 3500 V for 5 h.  These are the analytical strips.
The preparative strips have 10-fold more protein in the same volume
which are loaded 30 V over 10 h (active rehydration).  The 2nd dimension
is 12% gel run at 2&deg; for 5 h at 50 mA per gel with a 500 V.
<p>
Silver staining was done using the Swain and Ross method with some
modifications.  Gels are fixed in 40% EtOH/10% acetic acid for at least
2 h (overnight okay), then incubated in 40% EtOH + 10 &micro;l of
37% HCHO for 5 min, washed in 40% EtOH for 20 min, then 20 min in water,
then in 0.02% sodium thiosulfate for 1 min, then 0.1% silver nitrate
for 20 min, then three times with water for 1 min each (very critical
not to overwash!), developed with 2.5% sodium carbonate containing
40 &micro;l formaldehyde per 100 ml) for 4.25 minutes.  If the solution
turns yellow after the first 1.25 minutes (when spots typically appear),
change it quickly with fresh solution of the same volume.  Stop with
10% acetic acid containing 0.6% Tris base (leave 5 min).  Rinse water
for at least 5 min and get an image right away.  Store in 1% acetic acid
at 4&deg;.
<p>
The SYPRO Ruby protocol is typical:  gels are fixed in 10% MeOH and 7%
acetic acid for 30 min, and then incubated for at least 3 h (overnight
might be better), then washed in solvent for 30 min and imaged immediately.
<p>
For RuBP, fix in 30% EtOH 10% acetic acid overnight, rinse gel in 20% EtOH
for 30 min, repeating 3 times.  The 20 mM stock is diluted to a working
solution of 100 nM (5 &micro;l in 1 liter 20% EtOH) just before use, and
the gel left 6 hours.  Wash twice in water for 10 min each, and image
immediately.  This method is that of Rabilloud et al.
<p>
The modified RuBP staining protocol performs the fixing, acid-removal,
RuBP staining, and brief water washing steps, but also destains additionally
in 40% EtOH + 10% acetic acid for 15 hours, and repeats the brief water
washing steps before imaging.
<p>
Note that RuBP-stained gels can be stored in water for several days, and
even silver-stained additionally.
<p>
The silver-stained gels produced uneven staining (not unexpected), whereas
the fluorescently stained proteins were quantitative.  Peak (cone) volumes
over the background levels were calculated to assess quantitative results.
The silver stain threshold detection was 2 ng (lactalbumin 14.1 kDa marker),
4 ng for SYPRO Ruby (trypsin inhibitor 20.1 kDa marker), 16 ng for
RuBP (carbonic anhydrase 30 kDa marker), and 8 ng for the modified
RuBP staining (trypsin inhibitor).  Silver stain was the most sensitive,
and although SYPRO Ruby was second best, better linearity and more
intense signal were seen with RuBP modified protocol.  Curiously additional
silver staining of modified RuBP produced a 60 kDa artifact stain probably
from traces of a detergent used clean glass plates.
<p>
Profiling and 2D densitometry were done to characterize signal-to-background.
Areas at the upper and lower edge of lanes with no protein represented the
background.  A signal to background of 60/0.2 for the improved RuBP was
found, 7/0.8 for SYPRO Ruby, and 23/3 for standard RuBP staining.  Silver
staining is impossible to quantitate because of peak saturation (flattened
peaks along the lane).  Fluorescent staining even with the highest load
amounts in a lane never caused peak saturation.
<p>
The lesser sensitivity of the modified RuBP procedure compared to SYPRO
Ruby is a result of improving contrast (resolution) by washing out the
background of RuBP with the extra step.  If low abundance proteins are to be
detected, the modified staining procedure might fail to detect them.
Either shorten or even eliminate the destaining step to improve their
detectability.
<p>
For 2D gels intended to be compared quantitatively for expression,
fluorescence staining should be considered in preference to silver staining.
The development of the silver stain is time-dependent and makes gel
comparison difficult to impossible.  Fluorescent staining depends
on equilibration and is independent of time.  2D gels were stained the
four ways indicated.  Spot number detection was compared:  silver found the
most at 1000, SYPRO Ruby revealed 650, RuBP found 430, and the modified
RuBP found 750.  Signal-to-background was determined although visual
inspection shows that modified RuBP had the most obvious highest contrast.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Craig R, Beavis RC
 </span>
 <span class="year">
(2004)
 </span>
 <span class="jname">
Bioinformatics
 </span>
 <span class="jvol">
20,
 </span>
 <span class="pages">
1466-1467.
 </span>
<p class="reftitl">
TANDEM: Matching Proteins with Tandem Mass Spectra
<p class="Para1">
The fundamental idea of tandem mass spectrometry (MS/MS) is that particular
intact peptide parent ion is isolated with one MS and electronic or
vibrational energy added to it (such as with collision with argon) and the
mass of the resulting fragment ions measured with the 2nd MS.  Hopefully
most of the signal peaks break along the peptide backbone and the masses
reveal the differences in residues, permitting sequencing (only Leu and Ile
cannot be distinguished).
<p>
Because MS/MS spectrum are not entirely clear, an alternative was developed
in which the same information could be compared to the spectra of every
known peptide sequence generated from the known proteome of the organism
being studied.  This sort of analysis cannot be done manually, but software
solutions, many commercial, are available although the algorithm is not.
<p>
TANDEM is an open source project running from the command line and
takes an input XML file.  The code is composed of classes that do the following:
<ol>
<li>read XML input parameters
<li>read protein sequences from FASTA files
<li>read MS/MS spectra in DTA, PKL, and Matrix Science formats (all ASCII)
<li>process the MS/MS psectr to remove noise and artifacts
<li>manipulate peptide sequences with cleavage reagents, and posttranslational
and chemical modifications
<li>score peptide sequences
<li>create XML output with the best scoring and statistics relevant to scoring
process
</ol>
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Benkenstock K, Sundqvist G, Edlund P-O, Roeraade J
 </span>
 <span class="year">
(2004)
 </span>
 <span class="jname">
J. Mass Spectrom.
 </span>
 <span class="jvol">
39,
 </span>
 <span class="pages">
1059-1067.
 </span>
<p class="reftitl">
Influence of Droplet Size, Capillary-Cone Distance and Selected Instrumental
Parameters for The Analysis of Noncovalent Protein-Ligand Complexes by
Nano-Electrospray Ionization Mass Spectrometry
<p class="abstract">
<b>ABSTRACT</b>.
It has been suggested in the literature that nano-electrospray ionization (nano-ESI) mass spectrometry
better reflects the equilibrium between complex and free protein in solution than pneumatically assisted
electrospray ionization (ESI) in noncovalent interaction studies. However, no systematic studies of the
effects of ionization conditions have been performed to support this statement. In the present work,
different instrumental and sample-derived parameters affecting the stability of noncovalent complexes
during analysis by nano-ESI were investigated. In general, increased values of parameters such as drying
gas flow-rate, ion-source temperature, capillary tip voltage and buffer concentration lead to a dissociation of
ribonucleaseA(RNAse)cytidine 2-monophosphate (CMP) and cytidine 5-triphosphate (CTP) complexes.
The size of the electrosprayed droplets was shown to be an important issue. Increasing the capillary to
cone distance yielded an increased complex to free protein ratio when a hydrophilic ligand was present
and the reverse effect was obtained with a hydrophobic ligand. Important in this regard is the degree of
sampling of ions originating from late-generation residue droplets, that is, ions present in the droplet bulk.
Sampling of these ions increases with longer capillarycone distance (flight time). Furthermore, when the
sample flow-rate was increased by increasing the capillary internal tip i.d. from 4 to 30 m, a decreased
complex to free protein ratio for the RNAseCTP system was observed. This behavior was consistent with
the change in surface to volume ratio for flow-rates between 2 and 100 nl min1. Finally, polarity switching
between positive and negative ion modes gave a higher complex to free protein ratio when the ligand and
the protein had the same polarity.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Russell RB, Alber F, Aloy P, Davis FP, Korkin D, Pichaud M, Topf M, Sali A
 </span>
 <span class="year">
(2004)
 </span>
 <span class="jname">
Curr. Opin. Struct. Biol.
 </span>
 <span class="jvol">
14,
 </span>
 <span class="pages">
313-324.
 </span>
<p class="reftitl">
A Structural Perspective on Protein-Protein Interactions
<p class="Para1">
<b>REVIEW</b>.  There are 12,000 known complexes of proteins involving
two or more protein chains identified from a variety of organisms.  Based
on sequence similarity they can be organized into 3500 groups.
The most comprehensive info comes from yeast <i>Saccharomyces cerevisiae</i>
consisting of 6200 proteins, provided by the yeast two-hybrid system
and affinity purification followed by MS.  It is believed that the yeast
may have complexes of proteins with a lower limit of 30,000.  The human
proteome may have an order of magnitude more.
<p>
X-ray crystallography is the gold standard with respect to accuracy and
resolution in providing structure of proteins and their complexes.
A great many complexes (RNA polymerase, photosystem I and light-harvesting
complex of photosystem II, the signal recognition particle complex in
nascent protein targeting) have been determined.  NMR has been used
for larger subunits and can provide info on the residues
involved in interactions.
<p>
Electron microscopy also provides information through several variants.
Single-particle EM, electron tomography, and electron crystallography
are ways of looking at two-dimensional arrays of sample.  Single particle
cryo-EM can look at assemblies of 250-500 kDa with resolutions as high
as 5&Aring;.  Two-dimensional images are created in a stack or layer
from which 3-D projections are computed, capable of showing the structure
at any angle/perspective.  And sample does not have to be prepared as
a crystal or in large quantities.  X-ray crystal structures require
crystalline samples impossible to attain with macromolecular assemblies.
<p>
Protein interactions can be obtained using <q>low resolution</q>
experimental techniques.  Two-hybrid systems, tagged affinity chromatography,
phage display, and synthesis of peptides on cellulose membranes.
Biophysical, biochemical, and molecular biology methods include site-directed
mutagenesis; forms of footprinting that include hydrogen-deuterium exchange,
hydroxyl radical footprinting; chemical crosslinking; fluorescence resonance
energy transfer (FRET); Fourier transform IR spectroscopy (FTIR);
small angle X-ray scattering (SAXS).
<p>
Methods to assess protein-protein docking have been developed.  One method
generates many possible orientations of two docked proteins using
fast Fourier transforms, then ranks them with the hope that the
highest ranked one is the native (Katchalski-Katzir et al., 1992
<i>PNAS</i> <b>89</b>: 2195).  Many docking programs are being combined with
experimental techniques to assess their ability to predict interactions.
Many docking programs include FTDock, 3D-Dock, GRAMM,
DOT, ZDOCK, HEX, ICM, and CAPRI.
<p>
Often it is possible to predict interaction by using what is known about
the structure of interacting domains.  Homology searches can be used to
determine whether proteins have potential interaction domains when compared
to known structures.  Specific interaction predicting programs based on
homology templates include InterPReTS and ModBase.

<!-- ================================================================== -->
<p class="tags">

<p class="ref">
 <span class="authors">
Constantini M
 </span>
 <span class="year">
(2004)
 </span>
 <span class="jname">
Gene
 </span>
 <span class="jvol">
342,
 </span>
 <span class="pages">
321-325.
 </span>
<p class="reftitl">
An Analyis of Sponge Genomes
<p class="Para1">
<i>S. domuncula</i> (Porifera, Demospongiae, <i>Tetractinomorpha</i>,
<i>Hadromerida</i>, <i>Suberitidae</i>) and <i>G. cydonium</i>
(Porifera, Demospongiae, <i>Astrophorida</i>, <i>Geodidae</i>)
were collected from Bay of Naples at 20 m, with specimens collected
in individual bags and kept in seawater basins between 15-20&deg;.
DNA was extracted from internally in body (avoids epibionts).
5 g samples ground in liq. nitrogen and dispersed in 10 ml
of 0.1 M NaCl, 50 mM EDTA, pH 8.  SDS was brought to 2% with 20% stock,
and this heated to 60&deg; for 30 min and treated with proteinase K
(3 h @ 50&deg;) and then RNase at 37&deg; for 3 h.  Phenol/chloroform
and chloroform/isoamyl alcohol extracted proteins.  Sodium acetate
pH 5.9 was added to 3 M and then ethanol pptn.  DNA redissolved in
10:50 TE pH 8 and stored 4&deg;  EtBr-included 0.7% AGE was used to
view DNA.  DNA was also separated on a pulsed-field system to estimate
weight distribution.
<p>
To identify sponge DNA, two sponges are cut into pieces with external
layer not used.  These are put into filtered seawater in the dark
(prevents photosynthetic endobiont growth) for 4 days.  Tissue is dissociated
into cells and washed in artificial seawater (ASW:
7 mM Na<sub>2</sub>SO<sub>4</sub>, 2 mM NaHCO<sub>3</sub>, 50 mM MgCl<sub>2</sub>,
10 mM CaCl<sub>2</sub>, 540 mM NaCl, 10 mM KCl, 20 mM TrisHCl pH 8.2);
cell dissociation on 5 g samples of <i>S. domuncula</i> is done in
Ca/Mg-free ASW containing EDTA (CMFSW-E), which is just ASW without the
calcium and magnesium chloride and supplemented with 20 mM EDTA, with
gentle shaking at 20&deg;.  For the silicious sponge <i>G. cydonium</i>
dissociation is done in 100 &micro;g/ml trypsin, 100 IU/ml penicillin, and
100 &micro;g/ml streptomycin; cells are filtered through a 20 &micro;m
nylon mesh.  Cells are centrifuged 15 min at 800 <i>g</i> and dispersed
in CMFASW-E.  Lysis is 4 M GuanSCN, 25 mM sodium citrate pH 7, 0.5% sarcosyl,
0.1 M 2-mercaptoethanol.
<p>
Analytical ultracentrifugation of the DNA was done in CsCl gradients run
to equilibrium.  The speed was 44,000 rpm in the Beckman XL-A with
wavelength set to 260 nm;  equilibrium is reached in 24 h.  GC content
was assessed by the Schildkraut et al. relationship:
<p class="eqnserif">
&rho; = (GC &times; 0.098)/100 + 1.66
<p>
Results of ultracentrifugation of genomic DNA of <i>G. cydonium</i>
show that it had a greater buoyant density
with a peak of &rho; = 1.7030 g/cm<sup>3</sup>, while <i>S. domuncula</i> was
&rho; = 1.6987.  Two additional heavier peaks were observed
(at 1.7173 and 1.7253 g/cm<sup>3</sup>) for <i>G. cydonium</i>,
indicating a rather heterogeneous DNA.  A previous report
indicated that five different densities were
apparent, although the profile showed them to be discrete, and the
density profile in this study showed a broad range of heterogeneity.
<i>S. domuncula</i> showed two additional minor peaks, one heavier and
one lighter than the main peak.
<p>
The additional peaks have two possible explanations.  One is that the
DNA truly is heterogeneous.  The other is that the DNA is of endosymbiotic
organisms, and thus contaminant.
<p>
Genomic DNA from purified cells was then analyzed by CsCl
ultracentrifugation.  For <i>S. domuncula</i> a single peak was clear
at 1.6987 g/cm<sup>3</sup>, corresponding to the second and main peak in DNA
harvested from the interior of sponge body.  For <i>G. cydonium</i>
purified cells, the DNA re-analysis showed a single large peak
at 1.7030 g/cm<sup>3</sup>, again for DNA having the highest peak; two
additional but very small peaks relative to the main were also seen.
In addition <i>G. cydonium</i> was analyzed on ethidium bromide-containing
CsCl gradients, and the peak of 1.7030 g/cm<sup>3</sup> was seen with
no detection of the smaller peaks.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Baldwin MA
 </span>
 <span class="year">
(2004)
 </span>
 <span class="jname">
Molec. Cell. Proteomics
 </span>
 <span class="jvol">
3,
 </span>
 <span class="pages">
1-9.
 </span>
<p class="reftitl">
Protein Identification by Mass Spectrometry
<p class="Para1">
It is vital that there be a non-zero probability that a protein match
from a database search.  The data quality can not be ignored despite
hits being returned from these searches.
<p>
Typically the analysis of peptides is the basis for producing data.
Proteins are hydrolyzed by trypsin, but only a subset of these peptides
are detectable above the noise.  Sequence coverage is improved significantly
by analysis peptides from more than one type of digest, such as adding
chymotrypsin digests in parallel.  Factors that affect where a large or
small fraction of peptides are detected include the amount of protein
in original sample, efficiency of extraction, digest, and peptide
extraction, presence of other proteins and impurities, the sensitivity
of the mass spectrometer and mode of ionization, mass separation and
ion detection.  Early implementation of MALDI-TOF provided mass accuracies
of &plusmn;2 Da, no longer an acceptable standard since the reliability
of database searching by limiting possible compositions of peptides for
any given mass.  Delayed extraction MALDI-ToF in reflectron mode should
provide better than 50 ppm accuracy; with careful internal calibration
the error is ~10 ppm.  ESI-MS is used with single or triple quads
or quad ion traps with modest resolution.  With a quad mass selection
and quad collision cell with orthogonal acceleration ToF (QqTOF)
might produce resolution as high as 10,000 with 5 ppm accuracy.  FT-MS
provides a mass accuracy of about 1 ppm, the ultimate, but these
are expensive, technically demanding, and usually only found in large
MS facilities.
<p>
With the growth of the size of databases, the likelihood of finding an
incorrect hit with a mass associated with a high error (100 ppm or greater).
The NCBInr has grown by 50% in a year from 2002 to 2003 (having 1.5 million
entries at that time).  The need to have more peptides above background
threshold and larger sequence coverage is important.  Moreover, adding
CID spectra from MS/MS mode analysis provides tags to narrow the possibilities.
<p>
MALDI-ToF with PSD
is difficult to collect and the mass accuracy is low and so this
is little used.  MALDI configured QqTOF and TOF/TOF is a more desirable
configuration.  Also popular are low-resolution 3-D ion traps which
work well with LC-MS/MS;  these cannot determine charge states in
full-scan mode.  Libary searches on this data therefore cover the search
range by do analyses assuming one, two, and three protons are attached.
<p>
One uncontrolled variable is the choice of database search engine.
Searching with PMF and sequence tags, programs for calculating isotope
patterns, and predicting digestion are routine.
<p>
CID spectra are too complicated to analyze without software scanning.
Certain peak picking algorithms are propietary but others have
developed mathematical approaches.  Such algorithms produce lists
of monoisotopic mass and intensity and reduce electronic and random noise.
The <i>m/z</i> is defined by the centroid, determined above something like
50%
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Chertov O, Biragyn A, Kwak LW, Simpson JT, Boronina T, Hoang VM, Prieto DA,
Conrads TP, Veenstra TD, Fisher RJ
 </span>
 <span class="year">
(2004)
 </span>
 <span class="jname">
Proteomics
 </span>
 <span class="jvol">
4,
 </span>
 <span class="pages">
1195-1203.
 </span>
<p class="reftitl">
Organic Solvent Extraction of Proteins and Peptides from Serum As An
Effective Sample Preparation for Detection and Identification of Biomarkers
by Mass Spectrometry
<p class="Para1">
The effect of organic solvent extraction efficiency on the peptides and
proteins from serum was analyzed.  Acetonitrile was a natural choice
since it is miscible and used as a solvent in analytical processes,
particularly in mass measurement.  Different volume ratios of
0.1% TFA in acetonitrile:serum (1:1, 2:1, 3:1) were investigated using
100 &micro;l serum and 100, 200, 300 &micro;l solvent.  Precipitates
were centrifuged, and additional volumes were added to collected
supernatants.  Because there was only one precipitate additionally
formed after the initial extraction, it was decided that a 2:1 volume
ratio was sufficient to generate all obtainable precipitate.  Use of methanol
produced a larger but less dense precipitate that was difficult to remove
from the supernatant.   SDS-PAGE on precipitate extractions of human
and mouse serum.  High MW proteins were effectively precipitated, but
low MW proteins were clearly retained in the supernatant.
<p>
Acid hydrolysates of mouse serum proteins retained in the supernatant
by doing the 2:1 TFA/MeCN:serum extraction and drying down the supernatant
before acid hydrolysis.  A control of free amino acids was obtained by
analysis of resuspended supernatant which was not acid hydrolyzed, and
then subtracting total (acid) from free (no acid) amino acids.  The estimate
was that there was 40 &micro;g peptide per 100 &micro;l serum, or about
0.6% of total protein, assuming 65 mg total protein per milliliter
in mouse serum.
<p>
Aliquots of mouse sera and extracts of the sera were analyzed by SELDI-MS,
effective at looking at peptides up to 10-15 kDa.  The spectrum generated
is an image of visual contrast that reveals difference quickly.  In
particular an extracted sample with mass 9348 Da went undetectable in
analysis of whole serum samples.  Hence this extraction technique improves
detectability of some proteins/peptides as it removes large abundant proteins.
Spectra were then generated of sera of tumor-bearing mice to see if markers
could be revealed.  BALB/c mice had been inoculated with different counts
of syngeneic B lymphoma (A20) cells:  25; 2500; 250,000, with the minimal
lethal dose being 100,000.  Control mice were injected with vehicle (PBS).
Sera were collected on day 0, 25, and 40, and 20 &micro;l aliquots
precipitated with TFA in MeCN.  Spectral analysis revealed that peaks
at 8760 and 9340 Da are absent at day 40 in the mice injected with 250,000
cells, and absence correlating with tumor development in the animals.
Extracted and unextracted were compared in the sera of these animals both
on days 0 and 40.  The intensity of a peak at 8762-8768 is significantly
reduced.
<p>
To identify the proteins of interest, 150 &micro;l serum of control mice
was extracted with TFA/MeCN, the extract dried and resuspended for RP-HPLC
and fractions run on SDS-PAGE.  Bands from the fraction 15 (with two
major components) were analyzed by Edman sequencing.  One was not
sequenceable (N-terminal modification).  An aliquot was then digested
and and run on MALDI-ToF MS with five peaks indicate a high probability
that the 8700 Da protein was apolipoprotein A-II, accounting for the
fact that C-terminal Lys is often removed.  Analysis by LC-MS/MS also
indicated apo A-II.  The murine Apo A-II apparently exists as a monomer,
while human Apo A-II is a disulfide-linked dimer of two chains with 77
amino acids.  If the N-terminal Gln is cyclized to pyroglutamate and the
Lys is removed, the theoretical mass is exact to the 8718.81 Da mass in the
MALDI.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Jiang L, He L, Fountoulakis M
 </span>
 <span class="year">
(2004)
 </span>
 <span class="jname">
J. Chromatog. A
 </span>
 <span class="jvol">
1023,
 </span>
 <span class="pages">
317-320.
 </span>
<p class="reftitl">
Comparison of protein precipitation methods for sample preparation
prior to proteomic analysis
<p class="abstract">
<b>ABSTRACT</b>.
Protein samples should be free of salt and other disturbing agents and have an appropriate concentration to be suitable for two-dimensional
(2D) electrophoresis, the principal step of proteomics. To find the most efficient method for sample preparation, we used human plasma and
compared four widely applied precipitation methods, using trichloroacetic acid (TCA), acetone, chloroform/methanol and ammonium sulfate,
as well as ultrafiltration. Precipitation with TCA and acetone and ultrafiltration resulted in an efficient sample concentration and desalting.
We also found that ammonium sulfate fractionation can efficiently remove albumin, which represents more than 50% of plasma proteins.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Colland F, Daviet L
 </span>
 <span class="year">
(2004)
 </span>
 <span class="jname">
Biochimie
 </span>
 <span class="jvol">
86,
 </span>
 <span class="pages">
625-632.
 </span>
<p class="reftitl">
Integrating A Functional Proteomic Approach Into The Target Discovery Process
<p class="Para1">
<b>REVIEW</b>.  Protein-protein interactions are being described through
tools such as yeast two-hybrid (Y2H) system, protein chips, phage display,
or systematic analysis by tandem affinity purification (TAP) and mass
spectrometry.  But the study of these is a means to the end of discovery
targets.
<p>
Y2H is a relatively easy way of identifying protein-protein interactions
in a high throughput, low cost fashion.  It identifies a large number,
but not even a majority of interactions, as they must be seen in the
yeast nucleus if they occur at all.  A significant count of false negatives
and positives are likely.  Intrinsic limitations of Y2H is possible
misfolding and instability of hybrid polypeptides, inappropriate
subcellular localization, absence of posttranslational modifications
(tyrosine phosphorylation, complex glycosylation).  False positives
occur when bait proteins are <q>auto-activated</q> such that they
start reporter gene transcription without prey, and when prey proteins
bind nonspecifically to baits.
<p>
Other kinds of screening include the <b>matrix</b> and <b>library</b>
formats.  The matrix approach takes a collection of predefined
open-reading frame products (full-length proteins) and use these
as bait and prey for interaction assays.  The combination of bait and
prey is either individual or in pools.  A limitation is that it tests
only full-length proteins.  With the prey library format, randomly generated
prey protein fragments are screened against given bait polypeptide.
The interaction map quality is much better and minimizes false negatives.
Use of truncated (domain length) rather than full-length proteins
maximizes the chance of obtaining properly folded bait and prey and of
looking at what really happens in interactions:  the binding of domains
rather than full-length proteins.  With randomly generated prey fragments,
the region of overlap helps to narrow down the actual interaction domain.
<p>
The library approach has its limitations too.  One is the need to prepare
highly complex prey fragment libraries for comprehensive coverage of the
transcriptome of interest (for rare mRNAs in particular).  Another is
the technical difficulty and cost of interaction screens required to
cover complex libraries exhaustively.
<p>
It is important to score each interaction with a certain level of
confidence as well.  A scoring system called PBS has been shown
in several studies to correlate the biological significance of interactions.
Other attempts at confidence scoring used only the redundancy of the
prey fragments or identified the parameters used in PBS calculation
as global predictors in a trained linear model.
<p>
PIMRider is a tool which is a web-based graphical interface that
visualizes, explores, and analyzes interaction maps from raw
interaction networks.  It features a ProteinViewer component that
annotates a given protein, its sequence, and list of interacting
partners; the PIMViewer which shows dynamical graphical view of the
interaction network;  InteractionViewer, providing access to raw
experimental data on prey, bait, and SID sequences; DomainViewer.
which shows domains and motifs revealed from experimental (bait and
SID) and calculated (transmembrane segments, signal peptides, and
InterPro domains).  GenoLink from the Genostar Consortium is a
graph pattern query engine that builds up queries of protein interactions.
<p>

<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Challapalli KK, Zabel C, Schuchhardt J, Kaindl AM, Klose J, Herzel H
 </span>
 <span class="year">
(2004)
 </span>
 <span class="jname">
Electrophoresis
 </span>
 <span class="jvol">
25,
 </span>
 <span class="pages">
3040-3047.
 </span>
<p class="reftitl">
High reproducibility of large-gel two-dimensional electrophoresis
<p class="abstract">
<b>ABSTRACT</b>.
Two-dimensional gel electrophoresis (2-DE) facilitates the separation of thousands of
proteins from highly complex protein mixtures and has become a central method in
proteomics in recent years. In the present study, we examined the technical variability
of large 2-DE gels with respect to sample preparation, electrophoresis procedure, data
acquisition, and biological variation by analyzing a disease (Huntingtons disease) and
control state with a commercially available software package, PROTEOMWEAVER.
Scatter plots and correlation coefficients were obtained to quantify both technical
and biological variation. Even 2-DE gels run separately in both dimensions yielded correlation
coefficients around 0.88 and deviations from the mean close to 20% for lowintensity
spots. This indicates a high technical reproducibility of the 2-DE procedure
developed in our laboratory. Variability within a biological condition was low and comparable
to technical variation (at least 0.87). Two-dimensional (2-D) gels obtained from
samples of different biological conditions (health vs. disease) achieved a variability
similar to intracondition and technical variability. These findings highlight the importance
of multiple gel and spot-by-spot comparisons to identify biological significant
changes. Minor errors introduced by technical and biological variation allow a comparison
of all gels within a study which facilitates the tackling of complex biological
problems.
<p class="Para1">
Digital images were made from silver-stained large format 2D gel.
In the figure are the acidic pI 3.5-6 halves of brain extracts of
a transgenic mouse that is a Huntington's disease model,
and with extracts of a mouse control.	Important to this study is how
the image was acquired:  Umax Mirage-II scanner saved at 16-bit, 200 dpi
grayscal TIFF, images analyzed with Definiens PROTEOMWEAVER 1.3 software,
with spot detection and gel/spot matching/warping set on automatic.
<p>
Gel-to-gel comparisons were done on about 3500 resolved spots.  Scatter
plots and correlation coefficients were determined by
(<a href="http://plasma-gate.weizmann.ac.il/Grace/">Xmgrace</a> (available
online), data plotted on a log scale and a Perl program used for integration.
Mean, SD, and nested <i>t</i> test (difference of means between two groups
per standard error of difference) were used in plot analysis.  The standard
deviation of a pair of spots in gel comparisons was estimated as
(|<i>x</i>&minus;<i>y</i>|)/&radic;2 where <i>x</i> and <i>y</i> are
spot intensities.  73 comparisons were made of 32 different gel images.
Extracts of three different C57Bl/6 mice were pooled and differences between
three gels made from the same pooled sample analyzed.  The influence of the
2nd dimension was also studied, in addition to repeated scanning to
assess differences.  66 comparisons were done for all possible combinations
of six control and six HD gels.
<p>
The software PROTEOMWEAVER also includes an <q>averaged gel</q> feature,
with the standard setting being that when a matched spot is found in at
least 50% of the gels, it will be present in the average gel.  Spot matching
itself was set to indicate matching when spot centers fall within a
250 pixel radius of each other.
<p>
<b>Results</b>. Pixelation is the assignment of the pixels to a spot during
scanning; scanner cleanliness clearly contributes to variability in scanning
procedures.  Single pairs of gels contained about 3500 spots able to be
matched.  Plots of spot intensity data on a linear scan with regression
analysis revealed an <i>r</i> = 0.988.  Plotted on a log scale, greater
variability was revealed in spots with lower intensities:  below an intensity
of 0.1, the scatter in the log plot was obvious.  A plot of the coefficient
of variation (CV = std dev/mean) against the mean spot intensities shows that
CV is 20% at the lower intensities and 5% at the higher.  For the log plot
<i>r</i> = 0.952 to reveal the effect of variation in the lower intensities.
Nonetheless the log plot showed reproducibility of scanning was adequate
over 3 orders of magnitude.  Another possibility with respect to spot
intensity calculation is variability introduced by the software in how to
include pixels in a spot during detection or variations in digitization of
spots during image file creation.  Redetection and matching by the software
of the same gel raw image showed variation in the lower intensity spots.
Nonetheless, the variation introduced here is insignificant to other technical
and biological variation.
<p>
Several gels of the same sample were then run.  Brain extracts of
three male mice were pooled.  A scatter plot of spot intensities (gel 2 vs
gel 1) was made and the <i>r</i> = 0.926 found over three orders of
magnitude.  A plot of CV vs. mean spot intensity now reveals that the
CV is raised at the higher intensities, but that the CV running average is
only about 20%, 10% at the higher intensities.  This variability is
termed <q>multiplicative</q> and is seen with DNA chips as well.
<p>
Ways to limit the variables that can introduce significant variability in
the production of 2D gel data include using the same technician or
a standardized method, preparation of chemicals in large batches and/or
from the same commercial lots.
<p>
Gels were then analyzed with respect to a grouping of homogeneous samples
(health or disease).  Six gels from male 8 week CBA &times; C57BI/6 controls
and those with transgenic HD models (R6/2) were used.  12 gels allowed for
66 paired comparisons.  In additions, 15 control-control and R6/2-R6/2
matches were done as this determines the biological background variation.
Plot of CV vs. mean spot intensity show that the relative error was about
20% for all spot intensities.  Estimations of standard deviations from
just two measurements is not good.  Running averages for 100 serial points
were determined and the average correlation coefficients ranged from
0.87-0.89, somewhat smaller than those of pooled sample of with <i>r</i> =
0.93.  Average correlation of all 15 control-control comparisons was 0.88,
and that for the diseased-diseased was 0.87.  <i>r</i> values were slightly
better for 2nd dimension runs that were run in the same run (buffer +
gel cast) than for those with different runs (buffer + gel cast).  Thus
most variability was biological rather than technical.
<p>
Matching spot intensities from a large 2D gels of HD vs. control were plotted
and the <i>r</i> = 0.877 found.  This is remarkable given that only a very
few proteins are differentially expressed.  Hence in diseased and healthy
tissues, high abundance and/or housekeeping proteins do not alter their
expression significantly.
<p>
To reduce group heterogeneity, gel averaging was done as a step toward
automated analysis.  In doing single gel pairing, 3500 spots were matched.
With three gels used in creating an average gel, spot matching is 1700.
With 6 gels used in the average, the reduction in spots is 460, with very
low intensity spots missing entirely from the gel, since matching occurs
in only a minority of cases.  Since most spots are of low intensity, this
means elimination of those spots from consideration.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Bashir S, Mutter R, Derrick PJ
 </span>
 <span class="year">
(2004)
 </span>
 <span class="jname">
Eur. J. Mass Spectrom.
 </span>
 <span class="jvol">
10,
 </span>
 <span class="pages">
487-493.
 </span>
<p class="reftitl">
Parameterizing Matrix-Assisted Laser Desorption/Ionization:  Effects of
Solvents and Co-Additives on Analyte Peak Intensities
<p class="Para1">
DHB spotting properties depend on solvent system used.  In MeOH/water,
small crystals form, and in DMF these crystals are slightly larger
(viewed using optical microscope).  The ratio of
[DBH-OH + H]<sup>+</sup>/[DHB + Na]<sup>+</sup> for plain laser desorption
and matrix-assisted laser-desorption (NLDI and MALDI) was determined
for various solvent systems.  No clear pattern could be seen with respect
to the solvent used except that the solvent used affected LDI but not
MALDI.  Use of DMF seemed to give higher peak intensities in work with
MALDI of substance P and betacyclodextrin.
<p>
Besides use of particular solvents, co-additives also help.  Adding
2-hydroxy-5-methoxybenzoic acid (HMBA) disrupts the DHB crystallization
and improves matrix-analyte mixing.  DHB may coordinate by hydrogen
bonding to the analyte.  Breaking of the H-bonds is useful and can be
assisted by use of DMF.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Huang R-H, Wang D-C
 </span>
 <span class="year">
(2004)
 </span>
 <span class="jname">
J. Chromatog. B
 </span>
 <span class="jvol">
803,
 </span>
 <span class="pages">
167-172.
 </span>
<p class="reftitl">
Mass spectrometric strategy for primary structure determination
of N-terminally blocked peptides
<p class="Para1">
EAFP1 and EAFP2 are antifungal peptides previously purified and blocked
to Edman sequencing.  Rather sequencing was attempted from the C-terminus end.
Patterson et al previously presented the use of C-terminal ladder
sequencing using MALDI-MS.  This successfully found the sequence of
19 C-terminal residues.  Only a few peaks are present at a short CPY
digestion time;  the low rate of CPY digestion is attributed largely to
the high amount of glycyl residues in these peptides.  Sequence determinations
were for EAFP1 RCPRPCNAGLCCSIYG..gSGNAYcgAGNCRCQCRG and for EAFP2
nagLCCSIYGY..SgAAYC...NCRCQCRG. Lowercase letters represent residues identified
in spectra of different CPY quenching time, and the dots are gaps unidentified
in the mass spectra.  Generally sequences can be read down to about 600 Da,
where the matrix CHCA ions begin to interfere with reading.  An advantage
of CPY is its sensitivity (use of very small amounts of sample).  A disadvantage
is that it is quite sequence-dependent and additional methods to confirm
results are often necessary.
<p>
This confirmation of C-terminal sequencing by CPY can come in the form
of protease enzyme selection based on the sequencing results.  EAFP1 and
EAFP2 both contain sites amenable to trypsin cleavage.  After trypsin
treatment and HPLC separation, peptides were measured on MALDI-MS and then
Edman sequencing.  Results were used to build the final sequences (EAFP1:
RCPRPCNAGLCCSIYGYCGSGNAYCGAGNCRCQCRG, EAFP2:
RCPRPCNAGLCCSIYGYCGSGAAYCGAGNCRCQCRG).  Indeed more peaks in the HPLC were
observed than expected for trypsin cleavage, probably because of trypsin
impurity.  One peptide with MALDI-MS mass of 704 Da eluting at 12 min in the
HPLC was resistant to Edman sequencing and is likely the N-terminal peptide.
The tandem FTICR MS/MS was used to read the monoisotopic masses ([M + H<sup>+</sup>]).
Parent ion mass was 705.2997 Da.  <i>Y</i>-series ion masses could be read
easily.  A sequence of PyroGul-Thr-Cys-Ala-Ser-Arg was determined.
There was a 10 Da difference observed between calculated and experimental
average masses of the EAFP1 and EAFP2 peptides.  The calculated mass is higher
probably because of the 10 protons on Cys residues that would be removed
if five disulfide bridges were formed.
<p>
<b>Methods</b>.  Peptides are routinely reductively alkylated.  Generally
100 &micro;g peptide are dissolved in 100 &micro;l of 0.1 M TrisHCl pH 8.3
buffer containing 6 M guanidine HCl. DTT is added to a 100:1 molar ratio
under nitrogen gas at 37&deg; for 30 min, and then iodoacetamide at 500:1
molar ration under nitrogen gas 37&deg; for 4 h.  The mixture is diluted to
1 ml and cleaned on reversed phase (Waters YWG-PAK C18, 3.9 &times; 250 mm)
with 5-45% MeCN in 0.1% TFA linear gradient for 60 min.
<p>CPY digestion is done as described by Patterson et al. 500 pmol peptide
is suspended in 20 &micro;l water and 160 mU CPY (Sigma) in 30 &micro;l water.
These are then mixed at 37&deg; to produce a 10 pmol peptide and 3.2 mU CPY
per microliter concentration.  1 &micro;l aliquots are removed at 20 and 60
seconds, 2, 5, 10, and 30 minutes, and 2 and 16 hours for 8 time points.
Each aliquot is mixed immediately with 9 &micro;l CHCA saturated in 50% MeCN
+ 0.1% TFA, and 1 &micro;l of these are placed on target wells.
<p>A Bruker Proflex III MALDI-MS was used for MALDI-MS and a ABi 491 sequencer
for Edman degradation.  A Bruker APEX II FTICR mass spectrometer is used for mass measurement.
500 pmol peptide is dissolved in 25 &micro;l 50% MeOH with 2% acetic acid
and injected at 30 &micro;l per hour. MS/MS fragment ions are generated
with 3500 &micro;s argon gas pulse at 1 mbar
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Bafna V, Reinert K
 </span>
 <span class="year">
(2004)
 </span>
 <span class="jname">
&nbsp;
 </span>
 <span class="jvol">
&nbsp;
 </span>
 <span class="pages">
&nbsp;
 </span>
<p class="reftitl">
Mass Spectrometry and Computational Proteomics
<p class="Para1">
The mass spectrometer has a source for producing molecular ions in the
gas phase (MALDI, ESI) and then a mass analyzer for measuring mass:
ion trap, ToF, quadrupole, fourier transform.
<p>
Fragmentation in CID MS is a stochastic process.  Properties of the peptide
and collision energy are important.  Ions in the <i>a, b, c</i> refer to
ionized N-terminal fragments while <i>x, y, z</i> ions are C-terminal
fragments.  The most common ions are <i>a, b,</i> and <i>y</i>.
<p>
Bayesian probabilistic models are used to assign a likelihood of a sequence
based on peptide fragment masses.  Data is analyzed in three modules:
interpretation, filtering, and scoring.  In interpretation, the MS/MS
spectrum data are input and an <i>interpreted</i> spectrum is output.
Parent peptide mass, partial or complete sequence tags, and combinations
of tags are used.  In filtering the interpreted spectrum and a peptide
sequence database are brought together, and those peptides not in the
database that match a mass closely are removed, and a small list of candidates
remains that is most likely to represent the spectrum.  A scoring module
takes the candidate peptide list and then ranks and scores the peptides
in the list.  The highest scoring peptide is returned, possibly with a
<i>p</i> value of significance.
<p>
Starting with the MS/MS spectrum, interpretation is any sequence information
obtained from it.  Interpretation is done without assistance from a peptide
database comparison, as in filtering.  A spectrum might be composed of
<i>b</i> ions and their intensities used to locate them.  One problem
is that spectra are not just composed of the fragments of interest.
There are internal fragments and ions with different charge states,
and some peptide bonds do not fragment.  The use of a <b>prefix residue
graph</b> and creation of a list of prefix-suffix pairs that are possible
and impossible is done.
<p>
In referring to a database of all combinations of peptides, the actual
spectrum and a hypothetical spectrum are correlated, a technique used
regularly in database searches of proteins.  SEQUEST works by generating
a hypothetical spectrum, then scores it against the actual spectrum.
In scoring, a raw numerical score and a significance (<i>p</i>) value
are given.  SCOPE is a probabilistic model for scoring tandem mass spectra
against a peptide database.  Scoring improvements have been made given
conditions such as neutral loss more likely when acidic or basic residues
present, or when there is Pro-directed fragmentation.
<p>
Filtering scans a database of peptide candidates and removes the vast
majority of those peptides not likely to account for the spectrum.
Simple filters for parent mass, immonium ions, matching peaks are used.
One useful approach is the use of suffix trees with respect to pre-indexing
sequence databases.  Sequence tags are often used as filters in the database
search.
<p>
Peptides that elute from LC system to MS as a detector have their spectra
collected as elution occurs (<q>real time</q>).
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Henkin JA, Jennings ME, Matthews DE, Vigoreaux JO
 </span>
 <span class="year">
(2004)
 </span>
 <span class="jname">
J. Biomolec. Tech.
 </span>
 <span class="jvol">
15,
 </span>
 <span class="pages">
230-237.
 </span>
<p class="reftitl">
Mass Processing&mdash;An Improved Technique for Protein Identification
with Mass Spectrometry Data
<p class="abstract">
<b>ABSTRACT</b>.
Matrix-assisted laser desorption/ionization time-of-flight mass
spectrometry analysis following tryptic digestion of polyacrylamide
gel pieces is a common technique used to identify
proteins.  This approach is rapid, sensitive, and user friendly, and
is becoming widely available to scientists in a variety of biological
fields. Here we introduce a simple and effective strategy
called mass processing where the list of masses generated
from a mass spectrometer undergoes two stages of data
reduction before identification. Mass processing improves the
ability to identify in-gel tryptic-digested proteins by reducing
the number of nonsample masses submitted to protein identification
database search engines. Our results demonstrate
that mass processing improves the statistical score and rank
of putative protein identifications, especially with low-quantity
samples, thus increasing the ability to confidently identify proteins
with mass spectrometry data.
<!-- ============================================================= -->
<p class="tags">

<p class="ref" id="vollmer-2004">
 <span class="authors">
Vollmer M, H&ouml;rth P, N&auml;gele E
 </span>
 <span class="year">
(2004)
 </span>
 <span class="jname">
Anal. Chem.
 </span>
 <span class="jvol">
76,
 </span>
 <span class="pages">
5180-5185.
 </span>
<p class="reftitl">
Optimization of Two-Dimensional Off-Line LC/MS Separations To Improve
Resolution of Complex Proteomic Samples
<p class="abstract">
<b>ABSTRACT</b>.
In off-line 2D-HPLC a continuous salt gradient is applied
in the first separation dimension. This increases the
number of identified proteins from complex samples
significantly due to higher chromatographic resolution
compared to stepwise elution. Achievement of optimal
resolution requires the optimization of the two separation
dimensions. The influence of LC elution gradients in the
first and second dimensions, of analysis time, of stationary-
phase material, and of column dimensions was systematically
investigated in order to obtain information on
the overall peak capacity of the separation system. Provided
data indicate that for complex samples such as an
E. coli cell extract, a shallow LC SCX gradient with a high
number of collected fractions significantly increases the
overal peak capacity while for lower complexity samples
short gradients with few fractions were sufficient to obtain
a maximum of identified peptides. In addition, column
dimensions and materials exihibited a strong effect on the
overall efficiency of the 2D HPLC separation. The outcome
of these experiments could hence serve as a guideline for
investigators to adapt their method for the separation of
their specific proteome sample to achieve a maximum of
peptide sequence information by 2D LC MS/MS analysis.
<p class="Para1">
Samples of a digest of 10 bovine proteins and an E coli lysate were used to
test the SCX gradient comparison in a 2DLC system.  The SCX is a
0.3 &times; 35 mm with 3.5 &micro;m particles.  A 0-100 mM NaCl gradient
programmed for 15, 30, 60, and 120 min were tried, with a steep short ramp
to 500 mM in 2.5, 5, 10, and 20 min, respectively.  The SCX column resolution
is good even for a 120 min gradient.  Runs were done in triplicate and
the average relative std deviations of amount of identified peptides and proteins
were 3 and 6%, respectively.  For E coli lysate proteins, the peptide and
protein rel std dev were 4.3 and 9.8% on peptide and protein level.
<p>
Column dimensions is important feature.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Benkestock K, Sundqvist G, Edlund P-O, Roeraade J
 </span>
 <span class="year">
(2004)
 </span>
 <span class="jname">
J. Mass Spectrom.
 </span>
 <span class="jvol">
39,
 </span>
 <span class="pages">
1059-1067.
 </span>
<p class="reftitl">
Influence of droplet size, capillary-cone distance and
selected instrumental parameters for the analysis of
noncovalent proteinligand complexes by
nano-electrospray ionization mass spectrometry
<p class="abstract">
<b>ABSTRACT</b>.
It has been suggested in the literature that nano-electrospray ionization (nano-ESI) mass spectrometry
better reflects the equilibrium between complex and free protein in solution than pneumatically assisted
electrospray ionization (ESI) in noncovalent interaction studies. However, no systematic studies of the
effects of ionization conditions have been performed to support this statement. In the present work,
different instrumental and sample-derived parameters affecting the stability of noncovalent complexes
during analysis by nano-ESI were investigated. In general, increased values of parameters such as drying
gas flow-rate, ion-source temperature, capillary tip voltage and buffer concentration lead to a dissociation of
ribonucleaseA(RNAse)cytidine 2&prime;-monophosphate (CMP) and cytidine 5&prime;-triphosphate (CTP) complexes.
The size of the electrosprayed droplets was shown to be an important issue. Increasing the capillary to
cone distance yielded an increased complex to free protein ratio when a hydrophilic ligand was present
and the reverse effect was obtained with a hydrophobic ligand. Important in this regard is the degree of
sampling of ions originating from late-generation residue droplets, that is, ions present in the droplet bulk.
Sampling of these ions increases with longer capillarycone distance (flight time). Furthermore, when the
sample flow-rate was increased by increasing the capillary internal tip i.d. from 4 to 30 m, a decreased
complex to free protein ratio for the RNAseCTP system was observed. This behavior was consistent with
the change in surface to volume ratio for flow-rates between 2 and 100 nl min1. Finally, polarity switching
between positive and negative ion modes gave a higher complex to free protein ratio when the ligand and
the protein had the same polarity.
<!-- ================================================================ -->
<p class="tags">

<p class="ref">
 <span class="authors">
Zehl M, Allmaier G
 </span>
 <span class="year">
(2004)
 </span>
 <span class="jname">
Rapid Commun. Mass Spectrom.
 </span>
 <span class="jvol">
18,
 </span>
 <span class="pages">
1932-1938.
 </span>
<p class="reftitl">
Ultraviolet Matrix-Assisted Laser Desorption/Ionization Time-of-Flight
Mass Spectrometry of Intact Hemoglobin Complex from Whole Human Blood
<p class="Para1">
Fresh human blood from fingertip pricks were diluted in 20 mM ammonium
acetate or in 0.1% TFA (latter for subunit and cluster ion formation)
with dilution factors of 1:50, 1:100, 1:500, 1:1000 and 1:5000 for effective
Hb concentrations of 46, 23, 4.6, 2.3, and 0.46 &micro;M.  Diluted blood
was mixed with 0.17 M ferulic acid in EtOH in 7:3 ratio (Houston and Reilly),
and 0.5 &micro;l applied to steel target, dried under RT under air stream.
For other spotting: 0.5, 1.0, or 1.5 &micro;l sample is mixed with equal
volume satd 2,6-DHAP in 25% MeCN in 20 mM ammonium acetate, and another
was satd 2,6-DHAP in 25% MeCN in 0.1% TFA, using dried droplet.
<p>
Solution acidity leads to dissociation of noncovalent complexes. 2,6-DHAP
is less acidic a matrix than cinnamic acid.  When used with TFA, Hb subunits
are fully dissociated.  The &alpha; and &beta; subunits could be resolved
well (resolution is 1518 and 1785 for alpha and beta chains respectively).
The theoretical mass of HbA is 15216.36 and HbB is 15867.22 Da (apoprotein)
and the experimental is 15216.45 and 15867.22, mass differences of 6.4
and 0 ppm!
<p>
Cluster ions formed, with decreasing intensity as to size of cluster.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Freeman WM, Hemby SE
 </span>
 <span class="year">
(2004)
 </span>
 <span class="jname">
Neurochem. Res.
 </span>
 <span class="jvol">
29,
 </span>
 <span class="pages">
1065-1081.
 </span>
<p class="reftitl">
Proteomics for Protein Expression Profiling in Neuroscience
<p class="abstract">
<b>ABSTRACT</b>.
As the technology of proteomics moves from a theoretical approach to a practical reality, neuroscientists
will have to determine the most appropriate applications for this technology. Neuroscientists
will have to surmount difculties particular to their research, such as limited sample amounts,
heterogeneous cellular compositions in samples, and the fact that many proteins of interest are rare,
hydrophobic proteins. This review examines protein isolation and protein fractionation and separation
using two-dimensional electrophoresis (2-DE) and mass spectrometry proteomic methods.
Methods for quantifying relative protein expression between samples (e.g., 2-DIGE, and ICAT) are
also described. The coverage of the proteome, ability to detect membrane proteins, resource requirements,
and quantitative reliability of different approaches is also discussed. Although there are
many challenges in proteomic neuroscience, this eld promises many rewards in the future.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Kreil D, Karp NA, Lilley KS
 </span>
 <span class="year">
(2004)
 </span>
 <span class="jname">
Bioinformatics
 </span>
 <span class="jvol">
20,
 </span>
 <span class="pages">
2026-2034.
 </span>
<p class="reftitl">
DNA microarray normalization methods can remove bias from
differential protein expression analysis of 2D difference
gel electrophoresis results
<p class="abstract">
<b>ABSTRACT</b>
Motivation: Two-dimensional Difference Gel Electrophoresis
(DIGE) measures expression differences for thousands of
proteins in parallel. In contrast to DNA microarray analysis,
however, there have been few systematic studies on the validity
of differential protein expression analysis, and the effects
of normalization methods have not yet been investigated. To
address this need, we assessed a series of samesame
comparisons, evaluating how random experimental variance
influenced differential expression analysis.
Results: The strong fluctuations observed were reflected
in large discrepancies between the distributions of the spot
intensities for different gels. Correct normalization for pooling
of multiple gels for analysis is, therefore, essential. We
show that both dye-specific background levels and the differences
in scale of the spot intensity distributions must be
accounted for. A variance stabilizing transform that had been
developed for DNA microarray analysis combined with a robust
Z -score allowed the determination of gel-independent signal
thresholds based on the empirical distributions from same
same comparisons. In contrast, similar thresholds holding up
to cross-validation could not be proposed for data normalized
using methods established in the field of proteomics.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Yang Z, Hancock WS
 </span>
 <span class="year">
(2004)
 </span>
 <span class="jname">
J. Chromatog. A
 </span>
 <span class="jvol">
1053,
 </span>
 <span class="pages">
79-88.
 </span>
<p class="reftitl">
Approach to The Comprehensive Analysis of Glycoproteins Isolated from
Human Serum Using A Multi-Lectin Affinity Columnh
<p class="abstract">
<b>ABSTRACT</b>.
Glycosylation is one the most common post-translational modifications (PTM) and glycoproteins play fundamental roles in a diversity of
biological processes. The development of an analytical approach to the study of variation of glycosylation patterns in serum samples has
been hindered by the structural heterogeneity of this post-translational modification and the complexity of serum proteome.We have used the
ability of different lectins to recognize specific glycosylation motifs to develop a specific affinity system that can achieve a comprehensive
capture of serum glycoproteins. In a preliminary investigation, we evaluated the ability of five commonly used immobilized lectins to capture
glycoproteins from human serum. SDS-PAGE analysis showed each lectinwas able to enrich a subset of the serum glycoproteome and overlaps
in lectin specificity were indeed observed. Based on these results and with the goal of studying the extent of the human serum glycoproteome,
we then developed a multi-lectin affinity column containing Concanavalin A (Con A), Wheat germ and Jacalin lectin. The selection of lectins
was also based on the known N-linked and O-linked glycan structures that are considered representative of the serum proteome. We then
demonstrated that the capture of glycoproteins was specific, efficient and reproducible with this multi-lectin column. The results obtained
with this affinity step indicated that about 10% of human serum proteins are glycosylated (weight/weight) and, after removal of six high
abundance proteins, including albumin, at least 50% of the remaining proteins were glycosylated. We then evaluated the use of this affinity
column to monitor changes in the pattern of glycosylation in serum samples by a controlled, stepwise release of the captured proteins from
the multi-lectin affinity column with specific displacers.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Cooper JW, Lee CS
 </span>
 <span class="year">
(2004)
 </span>
 <span class="jname">
Anal. Chem.
 </span>
 <span class="jvol">
76,
 </span>
 <span class="pages">
2196-2202.
 </span>
<p class="reftitl">
Integrated and Ultrasensitive Gel Protein Identification
<p class="Para1">
Milligram amounts of protein can be loaded and separated on 2D-PAGE.
Gygi et al reported that current gel/MS identification identifies only the
most abundant proteins despite 1500 Ag-stained spots in a narrow pH range.
Regulatory proteins typically are undetectable because less than 100
copies are present in a cell.
<p>
A system that combines electronic protein transfer with in situ proteolysis
was designed to permit ultrasensitive identification.  A continuous current
drop is monitored during the protein extraction period which indicates
depletion of limited electrolytes at the capillary/gel interface and the
generation of a low-conductivity solution plug at the extraction end of the
capillary.	Elution is promoted by the high voltage drop across the low
conductivity buffer, and sample stacking is also achieved.  Modified
Coomassie Blue and SYPRO Red staining were developed to enhance visualization
on low protein loadings.  Electronic transfer was done for 2 min for
3 model protein (cytochrome c, 14 kDa; ovalbumin, 45 kDa; &beta;-galactosidase,
116 kDa) with gel loadings from 5 &micro;g down to 1 ng.  The peak height
and concentration decrease proportionally with decreases in protein loading.
If deionized water is used at the capillary/gel interface both the protein
concentration and recovery increase 4- to 10-fold, resp.
<p>
Use of the electronic capillary extraction technique with modified SYPRO
Red staining enable visualization and extraction as low as 1 ng and with
90% recovery of proteins.  PVDF membrane was placed inside a commonly used
capillary fitting to serve as a miniaturized trypsin membrane reactor.
Trypsin is immobilized on to the porous membrane.  As protein substrate
passes, it is digested, and fragments directly loaded to MALDI target plates.
As little as 1 ng of ovalbumin (22 fmol) and &beta;-galactosidase (8 fmol)
could be evaluated.  In an old paper by Gygi et al, 14 of 58 predicted
yeast proteins were identified and were high abundance proteins, with
CAI values &gt; 0.2.  Using this concentration/extraction technique,
11 additional, 8 of them low-abundance with CAI &lt; 0.2 were identified.
These include RNA polymerase I-specific transscription initiation factor.
10 low-abundance proteins were actually predicted for the cut-out region,
and 4 of them were found using this sampling technology.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Canelle L, Pionneau C, Marie A, Bousquet J, Bigeard J, Lutomski D,
Kadri T, Caron M, Joubert-Caron R
 </span>
 <span class="year">
(2004)
 </span>
 <span class="jname">
Rapid Commun. Mass Spectrom.
 </span>
 <span class="jvol">
18,
 </span>
 <span class="pages">
2785-2794.
 </span>
<p class="reftitl">
Automating Proteome Analysis:  Improvements in Throughput, Quality,
and Accuracy of Protein Identification by Peptide Mass Fingerprinting
<p class="abstract">
<b>ABSTRACT</b>.
The use of robots has major effects on maximizing the proteomic workflow required in an increasing
number of high-throughput projects and on increasing the quality of the data. In peptide mass
finger printing (PMF), automation of steps downstream of two-dimensional gel electrophoresis is
essential. To achieve this goal, the workflow must be fluid. We have developed tools using macros
written in Microsoft Excel and Word to complete the automation of our platform. Additionally,
because sample preparation is crucial for identification of proteins by matrix-assisted laser desorption/
ionization (MALDI) mass spectrometry, we optimized a sandwich method usable by any
robot for spotting digests on a MALDI target. This procedure enables further efficient automated
washing steps directly on the MALDI target. The success rate of PMF identification was evaluated
for the automated sandwich method, and for the dried-droplet method implemented on the robot
	as recommended by the manufacturer. Of the two methods, the sandwich method achieved the
	highest identification success rate and sequence coverage of proteins.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Karp NA, Kreil DP, Lilley KS
 </span>
 <span class="year">
(2004)
 </span>
 <span class="jname">
Proteomics
 </span>
 <span class="jvol">
4,
 </span>
 <span class="pages">
1421-1432.
 </span>
<p class="reftitl">
Determining a significant change in protein expression
with DeCyder during a pair-wise comparison using
two-dimensional difference gel electrophoresis
<p class="abstract">
<b>ABSTRACT</b>.
Two-dimensional difference gel electrophoresis (DIGE) is a tool for measuring changes in protein
expression between samples involving pre-electrophoretic labeling with cyanine dyes.
Here we assess a common method to analyze DIGE data using the DeCyder software system.
Experimental error was studied by a series of same sample comparisons. Aliquots of
sample were labeled with N-hydroxyl succinimidyl ester-derivatives of Cy2, Cy3, and Cy5
dyes and run together on one gel. This allowed assessment of how experimental error influenced
differential expression analysis. Bias in the log volume ratios was observed, which
could be explained by differences in dye background. Further complications are caused by
significant gel-to-gel variation in the spot volume ratio distributions. Using DeCyder alone
results in an inability to define ratio thresholds for 90 or 95% confidence. An alternative normalization
method was thus applied which resulted in improved data distribution and
allowed greater sensitivity in analysis. When combined with a standardizing function, this
allowed gel-independent thresholds for 90% confidence. The new approach, detailed here,
represents a method to greatly improve the success of DIGE data analysis.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Halligan BD, Ruotti V, Jin W, Laffoon S, Twigger SN, Dratz EA
 </span>
 <span class="year">
(2004)
 </span>
 <span class="jname">
Nucl. Acids Res.
 </span>
 <span class="jvol">
32,
 </span>
 <span class="pages">
W638-W644.
 </span>
<p class="reftitl">
ProMoST (Protein Modification Screening Tool):
a web-based tool for mapping protein modifications
on two-dimensional gels
<p class="abstract">
<b>ABSTRACT</b>.
ProMoST is a flexible web tool that calculates the
effect of single or multiple posttranslational modifications
(PTMs) on protein isoelectric point (pI) and
molecular weightanddisplays the calculated patterns
as two-dimensional (2D) gel images. PTMs of proteins
control many biological regulatory and signaling
mechanisms and 2D gel electrophoresis is able to
resolve many PTM-induced isoforms, such as those
due to phosphorylation, acetylation, deamination,
alkylation, cysteine oxidation or tyrosine nitration.
These modifications cause changes in the pI of the
protein by adding, removing or changing titratable
groups. Proteins differ widely in buffering capacity
and pI and therefore the same PTMs may give rise
to quite different patterns of pI shifts in different
proteins. It is impossible by visual inspection of a
pattern of spots on a gel to determine which modifications
aremost likely tobepresent.ThepatternsofPTM
shifts for different proteins can be calculated and are
often quite distinctive. The theoretical gel images
produced by ProMoST can be compared to the experimental
2D gel results to implicate probable PTMs and
focus efforts on more detailed study of modified
proteins. ProMoST has been implemented as cgi
script in Perl available on a WWW server at http://
proteomics.mcw.edu/promost.


<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Wang D, Park JS, Chu JSF, Krakowsik A, Luo K, Chen DJ, Li S
 </span>
 <span class="year">
(2004)
 </span>
 <span class="jname">
J. Biol. Chem.
 </span>
 <span class="jvol">
279,
 </span>
 <span class="pages">
43725-43734.
 </span>
<p class="reftitl">
Proteomic Profiling of Bone Marrow Mesenchymal Stem Cell upon Transforming
Growth Factor &beta;1 Stimulation
<p class="Para1">
Bone marrow stem cells are nonhematopoietic pluripotent stromal cells
which can expand in culture and differentiate to osteoblasts, chondrocytes,
adipocytes, skeletal and smooth muscle cells in response to microenvironmental
cues.  TGF-&beta;1 are multifunctional proteins regulating growth,
differentiation, migration, ECM production.  TGF-&beta; increases smooth
muscle &alpha;-actin in MSCs, but will induce chondrocyte differentiation
in presence of dexamethasone or cell aggregates.  Thus the effect depends
on microenvironmental cues not understood.
<p>
Proteomics is a systematic approach for the quantiative and qualitative
mapping of the whole proteome.  2D-PAGE is done because of its ease of
use and rapidity.
<p>
Human bone marrow MSCs were purchased and characterized by the company:
positive for CD105, CD166, CD29, CD44; negative for CD34, CD14, CD45.
The company supplied the medium and FBS (MSCGM medium).  Detached expansions
of cells were washed with PBS with 1% BSA for 30 min, then FITC-conjugated
antibodies to CD14, CD34, CD45, CD105, and CD166 incubated for 30 min
before characterization on Beckman-Coulter EPICS XL flow cytometer.
Antibodies to CD29 and CD44 were incubated for 30 min followed by 30 min
with FITC-conjugated 2nd antibody and then flow cytometry; negative controls
were cells incubated only with FITC-2nd Ab.  Antibodies and FITC-antibodies
came from many vendors (Santa Cruz Biotech, BD Biosci, Chemicon, Serotec,
BIOSOURCE, and Molecular Probes).  Sigma TGF-&beta;1 at 10 ng/ml was
used for treatment (5 and 20 ng/ml were tried with same effect).  Medium
was changed every 2-3 d for longer incubations.  For immunostaining and
confocal work, cells were fixed in 4% paraformaldehyde in PBS 15 min,
permeabilized with 0.5% TritonX-100 in PBS 10 min.  Antibodies to gelsolin,
&alpha;-actin, and FLAG tag were incubated 2 h and then FITC- or rhodamine-conj.
2nd for 1 h.  F-actin was stained with rhodamine-phalloidin 30 min.
The Leica TCL SL confocal with Ar and He/Ne laser, a scanner and the Leica
DM IRB microscope were used.  Projection planes were 0.3 &micro;m thick.
Immunoblots were prepared in cells lysed with 25 mM TrisHCl pH 7.4,
0.5 M NaCl, 1% TX-100, 0.1% SDS, 1 mM PMSF, 10 &micro;g/ml leupeptin, and
1 mM sodium vanadate; with supernatants run on SDS-PAGE, transferred to NC
membrane, blocked (3% nonfat milk), and primary Ab incubated in Tris-buffered
saline with 0.05% Tween-20 and 0.1% BSA; goat anti-mouse or anti-rabbit
IgG-HRP with ECL detection used.
<p>
Cells passaged and expanded up to 10 times were used.  MSCs at passage 10
were positive for CD105, CD166, CD29, and CD44 but negative for CD14, CD45,
and CD34, indicating no change in phenotype.  Cells cultured 2 weeks
in chondrogenic medium became rounded and synthesized large amounts of
glycosaminoglycans, indicating chondrogenic potential.  Cells in osteogenic
medium for 2 weeks stained with Alizarin Red indicating matrix mineralization.
<p>
Cells were treated long term with TGF-&beta;.  After 2 d, MSC show a more
polygonal broad shape with myoblast-like morphology and which was maintained
by 6 d.  The fibrous-like appearance stained positive for actin filaments
and thick fibers.  Immunoblotting indicated a quantitative increase
in &alpha;-actin while total actin was not changed;  tubulin expression
was seen to be constant (internal control).  TGF-&beta; was seen to affect
SM &alpha;-actin and SM-22&alpha; at transcriptional level.
<p>
Proteome characterization involved 2D-PAGE and characterization of select
silver-stained proteins, determined by treated and untreated cells.
1500 spots were resolved (confidence &gt; 95%), and 60 showed up- or
down-regulation by &gt;2-fold in triplicate studies of 4-day treated cells.
30 of the spots were selected for identification using in-gel digestion
and LC-MS/MS on Q-ToF API US from Waters.  The LC system was a
20 &micro;l/min with pre-concentration on 300 &micro;m &times; 5 mm
PepMap C18 (Dionex) with peptides eluted to 75 &micro;m &times; 15 cm
PepMap C18 analytical.  Buffer A is 3% MeCN + 0.1% formic acid and Solvent
B is 95% MeCN + 0.1% FA; the program is loading with Solvent A and
elution with 3-40% gradient of Solvent B over 35 min at 0.25 &micro;l/min,
achieved by flow splitting pumps A and B running at 8 &micro;l/min.
A PicoTip emitter (New Objectives) was used to direct eluent to ESI source.
The MS had source temp of 100&deg; with 40 V on the cone; the PicoTip
was set at 2000 V.  The ToF was in V mode.   Calibration was of
Glu-fibrinopeptide B fragment ions obtained by CID.  MS/MS were obtained
by directed data acquisition mode in which peaks with <i>z</i>= +2, +3, +4
with highest intensity were selected for CID, collision energies set
at 10 and 30V for MS and MS/MS scans.  The NCBI nrdb (non-redundant database)
with masses from 1000-500,000 Da and pI 3-10 were selected; modifications
set for Cys (carbamidomethylation), N-terminus (acetylation), N-Gln (to
pyro-Glu), Met (oxidation), and Ser/Thr/Tyr (phosphorylation).  Predicted
MW and pI of the 30 spots using Z3 program agreed with theoretical values
(&plusmn;10%).  Proteins affected by TGF-&beta; include cytoskeletal
proteins gelsolin, T-plastin (and an isoform), &gamma;-actin and
&alpha;-cardiac actin. A series of annexin molecules (cell membrane bound).
Matrix synthesis proteins:  collagen-binding protein 2 precursors and isoforms.
Metabolic enzymes change include GA3P DH, thioredoxin reductase, malate DH,
Tyr-tRNA synthetase.  TGF-&beta; also modulates post-translational
modifications, e.g. HSP27 phosphorylation at Ser-82.
Both ProteinLynx and MASCOT agreed on protein
spot identity and scoring; two peptides were sequenced with good MS/MS
spectra.  Some spots had more than one protein co-migrating.
<p>
Gelsolin expression was reduced by TGF-&beta; according to the 2D gel.
Gelsolin severs actin, disrupting the cytoskeleton, but having roles in
cell motility and apotosis.  Its role in differentiation is unknown.
A time course study indicates expression falling off with 4-6 d growth factor
treatment.  qPCR was used to study transcriptional regulation.
After 24 h, gene expression decreases by 50% for gelsolin, with &alpha;-actin
up 4-fold.  At 48 h, the trend continues, indicating TGF-&beta; alters
both proteins through transcription.  Immunocytohistochemistry of 4 d
treated and untreated cells shows gelsolin in untreated cells co-localized
with actin (a weak background in cytoplasm).
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Olsen JV, Ong S-E, Mann M
 </span>
 <span class="year">
(2004)
 </span>
 <span class="jname">
Molec. Cell. Proteomics
 </span>
 <span class="jvol">
3,
 </span>
 <span class="pages">
608-614.
 </span>
<p class="reftitl">
Trypsin Cleaves Exclusively C-Terminal to Arginine and Lysine Residues
<p class="Para1">
Proteins were extract from young mouse liver (3 wk old male).  Tissue was
crushed with mortar and pestle under liquid nitrogen and resolublized
with 6 M urea (Invitrogen), 2 M thiourea (Fluka),
3-[(3-cholamidopropyl)dimethylammonio]-1-propanesulfonate (CHAPS, Invitrogen),
and protease inhibitors (Complete Tablet, Roche).  Benzonase (Roche) was
added to degrade DNA and RNA, left on ice 30 min.  Sample was spun
10,000 <i>g</i> to get a supernatant.  90 &micro;g protein was run on a
4-12% Bis-Tris gel (Novex, Invitrogen), stained with colloidal Coomassie.
The lane was cut to 10 equally sized pieces and in-gel tryptic digestion
done:  piece were destained, washed, reductively alkylated (DTT/iodoacetamide),
and digested with Promega trypsin overnight/37&deg;.  Extraction was
with 30% MeCN/0.3% TFA, followed by vacuum evaporation.  Peptides are desalted
and concentrated on C18 StageTips.  Nanoflow LC-MS/MS is on a 7 T ThermoElectron
Finnigan LTQ-FT MS, using Agilent 1100 nanoflow system (degasser, nanoflow
pump, thermostated-microautosampler.  A 75 &micro;m &times; 20 cm capillary
was packed with MeOH slurry of 3 &micro;m Repro-Sil-Pur C18-AQ at 50 bar
under helium.  6 &micro;l of the tryptic mix was autosampled with 500 nl/min
flow for 20 min, eluted with 90 min gradient from 4-40% MeCN in 0.5% acetic
acid at 200 nl/min.  The mass spec switches from MS to MS/MS mode in
data-dependent way, with survey from <i>m/z</i> 300-1500 in FTICR
with <i>r</i>=25,000 at <i>m/z</i> 400.  Three most intense ion sequentially
isolated by FTICR <q>SIM scan</q> consisting of 10-Da mass range,
<i>r</i>=50,000 and target accum. value = 50,000.  These are fragmented
in linear ion trap using CID with normalized collision energy of 30% and
target value = 2000.  Former target ions selected for MS/MS were dynamically
excluded for 30 s, and total cycle time was 3 s.  4755 MS/MS mass spectra
were acquired.  Identification involved searching the Mouse International
Protein Index protein sequence database (v. 2.18 with 40k+ sequences,
curated  by European Bioinformatics Institute; http://www.ebi.ac.uk/IPI/)
which includes SWISSPROT, TrEMBL, RefSeq, and Ensembl and the contaminants
porcine trypsin and human keratins.  Fixed modifs were carbamidomethyl Cys
and variable modifs were oxid Met and N-acetylation.  Tolerances were
3 ppm for MS and 0.8 Da for MS/MS peaks.  MASCOT was set for &quot;ESI-Trap&quot;.
<p>
Mouse liver proteins were run on SDS-PAGE (1D gel), and 10 equally spaced
bands excised.  Highest apparent molecular mass was selected for proteomic
analysis.  FTICR-MS can have very high mass accuracies in low or sub-ppm
range if ion counts are strictly controlled.  With the ion trap, precursors
selected for MS/MS analysis can be accumulated within the linear trap,
then fragmented and the fragments analyzed within the linear trap or
in the FTICR-MS.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Maurer MH, Feldmann Jr RE, F&uuml;tterer CD, Butlin J, Kuschinsky W
 </span>
 <span class="year">
(2004)
 </span>
 <span class="jname">
Neurochem. Res.
 </span>
 <span class="jvol">
29,
 </span>
 <span class="pages">
1129-1144.
 </span>
<p class="reftitl">
Comprehensive Proteome Expression Proling
of Undifferentiated versus Differentiated Neural
Stem Cells from Adult Rat Hippocampus
<p class="abstract">
<b>ABSTRACT</b>. Adult neural stem cells can be isolated from various regions of the rat brain and seem to have
multilineage differentiation potential. In this study, we investigated the hypothesis that global protein
expression of adult neural stem cells isolated from rat hippocampus is changed during in vitro differentiation.
After 2 days of differentiation, we separated total protein extracts by two-dimensional
gel electrophoresis and used mass spectrometry and gel-matching for identication. We detected a
differential expression in 367 regulated protein spots, of which 128 could be identied. Regulated
proteins included participants in transcription and DNA metabolism; signal transduction and
Ca<sup>2+</sup>-signaling; MAP kinase pathways; cytoskeletal rearrangement; regulation of cell cycle, proliferation,
and survival; protein biosynthesis, folding, and degradation; and glycine and glutamate metabolic
pathways. These results suggest a massive reorganization of the stem cell proteomic prole upon differentiation
and indicate potential cellular targets mediating the differentiation of neural stem cells.

<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Ng TB
 </span>
 <span class="year">
(2004)
 </span>
 <span class="jname">
Peptides
 </span>
 <span class="jvol">
25,
 </span>
 <span class="pages">
1055-1073.
 </span>
<p class="reftitl">
Peptides and Proteins from Fungi
<p class="Para1">
Mushrooms are a class of fungi that are quite protein-rich and used in
diets in all cultures around the world.  Among proteins from fungi of
interest are ribsome-inactivating proteins, antifungal proteins, ribonucleases,
ubiquitin-like proteins and peptides, lectins, cellulases, xylanases,
laccases, invertases, and trehalose phosphorylases.
<p>
Of the ribosome-inactivating proteins, many are found in flowering plants
which are <b>type 1</b> proteins with molecular masses of 25-32 kDa.
These are immunosuppressive, antimitogenic, anti-tumor/anti-proliferative,
and antiviral.  Type 2 inactivating have an inactivating protein chain
and lectin chain.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Rejtar T, Chen H-S, Andreev V, Moskovets E, Karger BL
 </span>
 <span class="year">
(2004)
 </span>
 <span class="jname">
Anal. Chem.
 </span>
 <span class="jvol">
76,
 </span>
 <span class="pages">
6017-6028.
 </span>
<p class="reftitl">
Increased Identification of Peptides by Enhanced Data Processing of
High-Resolution MALDI TOF/TOF Mass Spectra Prior to Database Searching
<p class="Para1">
Mass spectra were obtained with a ABi 4700 Proteomics Analyzer using
Nd:YAG laser at 200 Hz using ABi Explorer v1.1 (resolution 15,000 and
4000 in MS and MS/MS modes, resp).  Accuracy is &plusmn; 10 ppm in MS
(internal calib) and 0.2 Da in MS/MS.  Agilent Tech Series II 1090
liquid chromatography on to 2.1 mm &times; 15 cm polysulfoethyl A (PolyLC)
and separated with 60 min linear gradient at 200 &micro;l/min.
Sample is 2 mg/ml yeast lysate with 0.5 ml injection.  Solvent A
is 25% MeCN, 10 mM phosphate pH 3.0; solvent B is solvent A + 0.35 M KCl.
From twenty 0.6 ml (3 min) fractions obtained, 10 with highest UV selected
and diluted 10-fold and loaded to 75 &micro;m &times; 15 cm column with
3 &micro;m 100 &Aring;.  10 &micro;l sample is loaded manually, to
precolumn of 1 mm &times; 300 &micro;m (UltiMate).  Sample was desalted
and eluted to analytical column at 200 nl/min with solvent A of 0.1% (v/v)
TFA in 2% MeCN, and solvent B of 0.1% TFA, 5% IPA, 85% MeCN with 60 min
linear gradient.  An Upchurch MicroTee mixed eluent with 7 mg/ml
CHCA in 50% MeCN + 1 mM ammonium citrate + 0.1% TFA MALDI matrix at
1.3 &micro;l/min which is streaked on plate with 20 cm &times; 40 &micro;m
(i.d)/150 &micro;m (o.d.) capillary, with 250 &micro;m wide streak.
<p>
A model sample of yeast soluble protein digest was run through an SCX
column and 10 of 20 fractions run through a reversed-phase nano-LC deposited
as narrow streaks on target plates under rough vacuum.  Spectra were
denoised with MEND and precursor ions for MS/MS selected using PRESEL.
The MS/MS spectra were processed by generation of peak lists, which
are centroided and then de-isotoped; intensity of deisotoped peaks
is the sum of the area of the peaks in a particular isotope cluster.
Peaks make the list only if a S/N &gt; 5 threshold is reached and the
cluster area &gt; 100, and up to 20 of most intense peaks in a 200 Da
region.  Searches were done on Mascot with precursor mass tolerance of
50 ppm (external calibration mass accuracy, 6 calib targets per plate).
9504 spectra resulted in 1077 peptides identified.
<p>
Enhanced processing of MS/MS spectra involved generating three separate
peak lists for each MS/MS spectrum based on increasing cluster area threshold.
Peak lists are submitted to search with 10 ppm tolerance, involving
closely placed external std streak over entire chromatographic run.
Identified spectra are removed and search using semitryptic specificity
conducted for all three cluster area thresholds.  Spectra not identified
yet are denoised with wavelet transformation.  With peak detection,
another search at 10 ppm mass tolerance.
<p>
The peak lists were intially generated by ABi 4700 software.  Intensity
and count of random peaks are factors used to optimize selection
of fragment ions in the Mascot score.  A model peptide FAHEGGYYIVPLSSK
was used for calculating theoretical mass values of six consecutive
<i>y</i> ions, and the intensities normalized to 1.  Also added to the peak
list were 1-19 peaks with random <i>m/z</i> values and intensities
one-tenth (0.1) of those in the <i>y</i>-ion series.  20 peaks resulted
and were submitted to Mascot.  The Mascot score decreased systematically
from its high of 59 when there are no random peaks, to  28 when 19 random
peaks are present at the 10-fold lower intensities.  Thus random,
noisy peaks do have a negative effect on Mascot scoring, and an effort
should be made to reduce their inclusion when possible.
<p>
One way of excluding random peptides is by excluding masses that cannot
possibly ever be present as calculated by all potential combination of
small peptides with the 20 amino acids.  All theoretical tryptic peptides
from the ORFs of yeast were generated and the masses of <i>b</i>-
and <i>y</i>-ion fragments calculated.  Intervals with the mass range
70-3000 Da were next determined which would have the theoretical fragment
ions, which included the error of &plusmn;0.2 Da for the mass accuracy
of the MS in MS/MS mode.  About 70% of the mass range included potential
masses, but at least 30% of the masses could be eliminated as forbidden.
<p>
Then the mass spectra were acquired.  About 7% of the total masses were
eliminated as forbidden.  This indicates that the standard analysis
software of the MS system is rather effective in identifying fragment
ions.  Database searching was done on the filtered set, and there was a
4% increase in identified peptides.  In a reversed database search
which catches false positives, the removal of random peaks from the search
caused a negligible increase in the count of false positives (from 2.7
to 2.8%).
<p>
Multiple threshold values were set for fragment ion peaks in MS/MS spectra.
This is because a single threshold often lets in too much background in
some spectra while it might reduce it in other spectra.  Spectra were
centroided and deisotoped, and peaks with S/N &gt; 3 chosen.  A histogram
was constructed of the dependence of the different cluster areas (peak
intensities):  it reveals a 2-fold increase in the number of peaks with
cluster area of 100 relative to those with area of 200.  Thus background
noise peaks have a cluster area &lt;200.
<p>
Three peak lists were then generated, with thresholds for peak inclusion
being cluster areas of 100, 300, and 500.  Peaks with S/N &gt; 3 were
preselected, then forbidden <i>m/z</i> values filtered out, then the 20
most intense peaks in a 200 Da mass interval selected.  After this
step, the cluster threshold was applied for the three variable thresholds.
The search was then applied with a &plusmn; 50 ppm tolerance.  70%
of all spectra obtained a significant score with 3 peak lists, 20% for
two peak lists, and 10% for only one.  The three-peak list approach provided
11.3% more identifications.
<p>
A low intensity MS/MS spectrum was submitted to this approach using a
peak list with cluster area threshold 100.  A Mascot score of 56 (highly
significant) was obtained for this peptide (QLYSFDLEGFWMDVGQPK) but dropped
to 6 with a threshold of 300, as far few fragment masses were available.
The measured mass was only 2.2 ppm from the theoretical.  Thus lowering
threshold increases the Mascot score.
<p>
But suppose the lower threshold was used on all MS/MS spectra.  Increased
noise also results in lower Mascot scoring.  For instance peptide
EVSDGVIAPGYEPEALAILSK obtained a significant Mascot score (30) only when the
cluster threshold was raised to 500 from 300, where its score was an
insignificant 15.
<p>
Precursor ion mass accuracy is a factor in the improvement of identification.
1000 MS/MS spectra were taken on proteins from a single SCX fraction.
The dependence of the Mascot score on a search against the MSDB for all
organisms as opposed to only ORFs for the yeast genome.  The significant
score is calculated from the theoretical count of peptides with the mass
tolerance.  Generally it is a better idea to reduce the tolerance of the
precursor ion to get high scoring hits.  Good MS calibration becomes a
factor in that case.  Internal calibration is more desirable since
the tolerance falls from &plusmn;50 to 10 ppm, but addition of mass
standards can lead to ion suppression in MALDI samples.  Placing external
calibrants in close proximity (i.e. in the same <q>well</q>) to
the sample such that they can be acquired in a single sequence allows
a tolerance of 10 ppm without the associated ion suppression.  Comparison
of the theoretical with the measured precursor masses showed all masses
within 10 ppm tolerance for peptides with highly significant identifications.
<p>
Thus after doing a three peak list searches based on cluster area thresholds,
a filter was set for precursor mass to be within &plusmn; 10 ppm tolerance.
2101 peptides has a Mascot score &gt; 17 (the <i>significant</i> level).
5% of precursors were outside the 10 ppm tolerance, so the search was
repeated at 30 ppm level, and hits with only 99% (Mascot &ge; 26.5)
significance were accepted to reduce false positives.  19 additional
significant identifications were made.
<p>
The standard MS software and the enhanced processing system were compared,
and the variable threshold plus precursor mass accuracy changes results
in 377 more identifications with only a slight increased risk of false
positives (from 2.7 to 3.6%, still below the 5% level of confidence).
<p>
Additional complicating factors are the presence of proteases that cleave
nonspecifically.  Additional peptides (96, or about 5%) were obtained
with Mascot scores &gt; 35 which were not produced strictly by tryptic
expectations.  With all these still ~75% of MS/MS spectra did not yield
identifications.  <b>Wavelet denoising</b> is known to be very effective
for processing MS/MS spectra, although care must be taken not to produce
false negatives, and so it should only be done after doing the other searches
first, which is why it is part of the enhanced method at the final stage.
Optimization of wavelet denoising is done on a subset of MS/MS spectra
by setting the different variables that are the base functions, order,
and decomposition level.  For this study, the Sym8 function with
decomposition level = 5 was used.  The peak list was then generated
after denoising with threshold of S/N 15.  Database searching with 10 ppm
tolerance and no missed cleavages was set, and 178 additional peptides.
One example of an MS/MS spectrum before denoising had a Mascot of only
17, which improved to 47 after several <i>y</i>-ion fragments emerged.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Whitelegge JP
 </span>
 <span class="year">
(2004)
 </span>
 <span class="jname">
Plant Physiol. Biochem.
 </span>
 <span class="jvol">
42,
 </span>
 <span class="pages">
919-927.
 </span>
<p class="reftitl">
Mass Spectrometry for High Throughput Quantitative Proteomics in Plant
Research:  Lessons from Thylakoid Membranes
<p class="abstract">
<b>REVIEW</b>.
<p>
<b>ABSTRACT</b>.
Proteomics seeks to monitor the flux of protein through cells under variable developmental and environmental influences as programmed
by the genome. Consequently, it is necessary to measure changes in protein abundance and turnover rate as faithfully as possible. In the
absence of non-invasive technologies, the majority of proteomics approaches involve destructive sampling at various time points to obtain
snapshots that periodically report the genomess product. The work has fallen to separations technologies coupled to mass spectrometry, for
high throughput protein identification. Quantitation has become the major challenge facing proteomics as the field matures. Because of the
variability of day-to-day measurements of protein quantities by mass spectrometry, a common feature of quantitative proteomics is the use of
stable isotope coding to distinguish control and experimental samples in a mixture that can be profiled in a single experiment. To address
limitations with separation technologies such as 2D-gel electrophoresis, alternative systems are being introduced including multi-dimensional
chromatography. Strategies that accelerate throughput for mass spectrometry are also emerging and the benefits of these shotgun protocols
will be considered in the context of the thylakoid membrane and photosynthesis. High resolution Fourier-transform mass spectrometry is
bringing increasingly accurate mass measurements to peptides and a variety of gas-phase dissociation mechanisms are permitting top-down
sequencing of intact proteins. Finally, a versatile workflow for sub-cellular compartments including membranes is presented that allows for
intact protein mass measurements, localization of post-translational modifications and relative quantitation or turnover measurement.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
van Wijk, KJ
 </span>
 <span class="year">
(2004)
 </span>
 <span class="jname">
Plant Physiol. Biochem.
 </span>
 <span class="jvol">
42,
 </span>
 <span class="pages">
963-977.
 </span>
<p class="reftitl">
Plasmid Proteomics
<p class="abstract">
<b>REVIEW</b>.
<p>
<b>ABSTRACT</b>.
Plastids are essential organelles present in virtually all cells in plants and in green algae. The proteomes of plastids, and in particular of
chloroplasts, have received significant amounts of attention in recent years. Various fractionation and mass spectrometry (MS) techniques
have been applied to catalogue the chloroplast proteome and its membrane compartments. Neural network and hidden Markov models, in
combination with experimentally derived filters, were used to try to predict the chloroplast subproteomes. Some of the many proteinprotein
interaction, as well as post-translational modifications have been characterized. Nevertheless, our understanding of the chloroplast proteome
and its dynamics is very incomplete. Rapid improvements and wide-scale implementation of MS and new tools for comparative proteomics
will undoubtedly accelerate this understanding in the near future. Proteomics studies often generate a large amount of data and these data are
only meaningful if they can be easily accessed via the world-wide-web and connected to other types of biological information. The plastid
proteome data base (PPDB at http://www.ppdb.tc.cornell.edu/) and other web resources are discussed. This review will briefly summarize
recent experimental and theoretical efforts, attempt to translate these data into the functions of the chloroplast and outline expectations and
possibilities for (comparative) chloroplast proteomics.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Lee S-J, Saravanan RS, Damasceno CMB, Yamane H, Kim B-D, Rose JKC
 </span>
 <span class="year">
(2004)
 </span>
 <span class="jname">
Plant Physiol. Biochem.
 </span>
 <span class="jvol">
42,
 </span>
 <span class="pages">
979-988.
 </span>
<p class="reftitl">
Digging Deeper Into The Plant Cell Wall Proteome
<p class="abstract">
<b>REVIEW</b>.
<p>
<b>ABSTRACT</b>.
The proteome of the plant cell wall/apoplast is less well characterized than those of other subcellular compartments. This largely reflects
the many technical challenges involved in extracting and identifying extracellular proteins, many of which resist isolation and identification,
and in capturing a population that is both comprehensive and relatively uncontaminated with intracellular proteins. However, a range of
disruptive techniques, involving tissue homogenization and subsequent sequential extraction and non-disruptive approaches has been developed.
These approaches have been complemented more recently by other genome-scale screens, such as secretion traps that reveal the genes
encoding proteins with N-terminal signal peptides that are targeted to the secretory pathway, many of which are subsequently localized in the
wall. While the size and complexity of the wall proteome is still unresolved, the combination of experimental tools and computational
prediction is rapidly expanding the catalog of known wall-localized proteins, suggesting the unexpected extracellular localization of other
polypeptides and providing the basis for further exploration of plant wall structure and function.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Stauber EJ, Hippler M
 </span>
 <span class="year">
(2004)
 </span>
 <span class="jname">
Plant Physiol. Biochem.
 </span>
 <span class="jvol">
42,
 </span>
 <span class="pages">
989-1001.
 </span>
<p class="reftitl">
<i>Chlamydomonas reinhardtii</i> Proteomics
<p class="abstract">
<b>REVIEW</b>.
<p>
<b>ABSTRACT</b>.
Proteomics, based on the expanding genomic resources, has begun to reveal new details of Chlamydomonas reinhardtii biology. In particular,
analyses focusing on subproteomes have already provided new insight into the dynamics and composition of the photosynthetic
apparatus, the chloroplast ribosome, the oxidative phosphorylation machinery of the mitochondria, and the flagellum. It assisted to discovered
putative new components of the circadian clockwork as well as shed a light on thioredoxin proteinprotein interactions. In the future, quantitative
techniques may allow large scale comparison of protein expression levels. Advances in software algorithms will likely improve the use
of genomic databases for mass spectrometry (MS) based protein identification and validation of gene models that have been predicted from
the genomic DNA sequences. Although proteomics has only been recently applied for exploring C. reinhardtii biology, it will likely be
utilized extensively in the near future due to the already existing genetic, genomic, and biochemical tools.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Riccardi F, Gazeau P, Jacquemot M-P, Vincent D, Zivy M
 </span>
 <span class="year">
(2004)
 </span>
 <span class="jname">
Plant Physiol. Biochem.
 </span>
 <span class="jvol">
42,
 </span>
 <span class="pages">
1003-1011.
 </span>
<p class="reftitl">
Deciphering Genetic Variations of Proteome Responses To Water Deficit
in Maize Leaves
<p class="abstract">
<b>ABSTRACT</b>.
The proteome of the basal part of growing Zea mays leaves was analyzed from 4 to 14 d after stopping watering and in well watered
controls. The relative quantity of 46 proteins was found to increase in leaves of plants submitted to water deficit. Different types of responses
were observed, some proteins showing a constant increase during water deficit, while others showed stabilization after a first increase or a
transient increase. Isoforms encoded by the same gene showed different responses. The response to water deficit showed genetic variation.
Some increased proteins were induced specifically in one of the two studied genotypes (e.g. ASR1) while others were significantly induced in
both genotypes but to a different level or with different kinetics. Analyses of relations between protein quantities, relative water content
(RWC) and abscisic acid (ABA) concentration allowed us to show that the quantitative variation of some proteins (e.g.ABA45 and OSR40 proteins)
was linked to differences in ABA accumulation between the genotypes. Other proteins showed genetic variations that were not related
to differences in water status or ABA concentration (e.g. a cystatin). Data obtained from these experiments, together with data from other
experiments, contribute to the characterization of maize proteome response to drought in different conditions and in different genotypes. This
characterization allows the search for candidate proteins, i.e. for protein whose genetic variation of expression could be partly responsible for
the variability of plant responses to drought.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Schneider M, Tognolli M, Bairoch A
 </span>
 <span class="year">
(2004)
 </span>
 <span class="jname">
Plant Physiol. Biochem.
 </span>
 <span class="jvol">
42,
 </span>
 <span class="pages">
1013-1021.
 </span>
<p class="reftitl">
The Swiss-Prot Protein Knowledgebase and ExPASy:  Providing The Plant
Community with High Quality Proteomic Data and Tools
<p class="abstract">
<b>REVIEW</b>.
<p>
<b>ABSTRACT</b>.
The Swiss-Prot protein knowledgebase provides manually annotated entries for all species, but concentrates on the annotation of entries
from model organisms to ensure the presence of high quality annotation of representative members of all protein families. A specific Plant
Protein Annotation Program (PPAP) was started to cope with the increasing amount of data produced by the complete sequencing of plant
genomes. Its main goal is the annotation of proteins from the model plant organism Arabidopsis thaliana. In addition to bibliographic
references, experimental results, computed features and sometimes even contradictory conclusions, direct links to specialized databases
connect amino acid sequences with the current knowledge in plant sciences. As protein families and groups of plant-specific proteins are
regularly reviewed to keep up with current scientific findings, we hope that the wealth of information of Arabidopsis origin accumulated in our
knowledgebase, and the numerous software tools provided on the Expert Protein Analysis System (ExPASy) web site might help to identify
and reveal the function of proteins originating from other plants. Recently, a single, centralized, authoritative resource for protein sequences
and functional information, UniProt, was created by joining the information contained in Swiss-Prot, Translation of the EMBL nucleotide
sequence (TrEMBL), and the Protein Information ResourceProtein Sequence Database (PIRPSD). A rising problem is that an increasing
number of nucleotide sequences are not being submitted to the public databases, and thus the proteins inferred from such sequences will have
difficulties finding their way to the Swiss-Prot or TrEMBL databases.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Kim SI, Voshol H, van Oostrum J, Hastings TG, Cascio M, Glucksman MJ
 </span>
 <span class="year">
(2004)
 </span>
 <span class="jname">
Neurochem. Res.
 </span>
 <span class="jvol">
29,
 </span>
 <span class="pages">
1317-1331.
 </span>
<p class="reftitl">
Neuroproteomics:  Expression Profiling of the Brain's Proteomes in
Health and Disease
<p class="abstract">
<b>ABSTRACT</b>.
The term proteome describes the protein complement of a genome. Proteomes of cells are
dynamic and are directly affected by environmental factors, such as stress and/or drug treatment,
or as a result of aging and disease. One of the distinct advantages of proteomic analysis, not
attainable with RNA expression data, is the ability to fractionate the cells proteins into various
subpopulations. In neuroscience, neuromics (proteomics in the central nervous system) is in its
infancy, with a paucity of studies in the context of the brain. One of the objectives of this review
is to illustrate the potential of neuromics to profile differences in the distribution of thousands of
proteins as a function of disease markers. We have previously used this approach to determine
the effects of varied postmortem interval in examining human brain tissue and to identify biomarkers.
Here we review proteomic studies of schizophrenia, Alzheimers disease, and Parkinsons
disease. Experimental results regarding Parkinsons disease are presented to illustrate the potential
of neuromics to identify pathways of pathogenesis and novel therapeutic targets.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Saravanan RS, Rose JKC
 </span>
 <span class="year">
(2004)
 </span>
 <span class="jname">
Proteomics
 </span>
 <span class="jvol">
4,
 </span>
 <span class="pages">
2522-2532.
 </span>
<p class="reftitl">
A critical evaluation of sample extraction techniques for enhanced
proteomic analysis of recalcitrant plant tissues
<p class="abstract">
<b>ABSTRACT</b>.
Most published proteomics studies of bulk plant tissues use a procedure in which proteins are
precipitated with trichloroacetic acid (TCA) and acetone (TCA-A), but few attempts have been
made to contrast this approach in a systematic way with alternative methods against a spectrum
of tissues. To address this, TCA-A was compared with another acetone-based protocol
(TCA-B) or a phenol (Phe)-based method, targeting a range of tomato tissues and three species
of fruits that contain high levels of contaminating compounds: banana, avocado and
orange. The Phe method gave a higher protein yield and typically greater resolution and spot
intensity, particularly with extracts from tissues containing high levels of soluble polysaccharides.
The methods also generated remarkably different two-dimensional gel electrophoresis
(2-DE) protein spot patterns. Peptide mass fingerprinting was used to identify polypeptides
that were common to multiple extracts or uniquely present in one extract type. While no clear
pattern emerged to explain the basis for the differential protein extraction, it was noted that
the Phe method showed enhanced extraction of glycoproteins. These results suggest that the
Phe protocol is highly effective with more recalcitrant tissues and that a combination of TCA-A
and Phe methods provides enhanced 2-DE based proteomic analyses of most plant tissues.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Seike M, Kondo T, Fujii K, Yamada T, Gemma A, Kudoh S, Hirohasi S
 </span>
 <span class="year">
(2004)
 </span>
 <span class="jname">
Proteomics
 </span>
 <span class="jvol">
4,
 </span>
 <span class="pages">
2776-2788.
 </span>
<p class="reftitl">
Proteomic Signature of Human Cancer Cells
<p class="abstract">
<b>ABSTRACT</b>.
We assessed proteomic profiles as biomarkers for monitoring cell phenotypes. Protein
expression profiles were obtained by fluorescence two-dimensional difference gel electrophoresis
(2-D-DIGE), in which quantitative ability is improved by labeling proteins with fluorescent
dyes prior to electrophoresis. Integrated protein spot intensities were analyzed by a statistical
approach. The proteomic data of two groups of cell lines: (1) adenocarcinoma (AC) cell lines
derived from lung, pancreas and colon tissues and (2) lung cancer cell lines with different histological
backgrounds, including AC, squamous cell carcinoma and small cell carcinoma,
were assessed on the basis of prior biological information. Hierarchical clustering analysis
and principal component analysis were used to divide the cell lines into subgroups on the
basis of similarities between their protein expression profiles. The majority of cell lines were
grouped according to their organ of origin or histological background. A machine-learning
algorithm selected 32 protein spots that were responsible for the classification. The results
indicate that proteomic data generated by 2-D-DIGE can provide a signature of essential cell
phenotypes, suggesting that it might be possible to apply this technique to developing tumor
markers that could identify the organ of origin of metastatic tumors and contribute to the differential
diagnosis of lung cancer.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Causier B
 </span>
 <span class="year">
(2004)
 </span>
 <span class="jname">
Mass Spec. Rev.
 </span>
 <span class="jvol">
23,
 </span>
 <span class="pages">
350-367.
 </span>
<p class="reftitl">
Studying The Interactome with The Yeast Two-Hybrid System and Mass Spectrometry
<p class="Para1">
Protein function can often be determined through interaction with other
proteins whose functions are known.  One approach is to analyze purified
protein complexes by MS.  Another is the yeast two-hybrid <i>in vivo</i>
assay.  Yeast two-hybrid studies evolved from study of eukaryotic
transcription factors.  Transcription factors have two domains:
a DNA-binding and activation domain (BD and AD, respectively).
It was discovered that the domains were both functionally and physically
separable.	Ptashne and co-workers found that the BD of one protein
could be fused to the AD of another, making a new transcription factor.
Fields &amp; Song (1989) found that two proteins that interacted, one
linked to the BD and another to an AD, would elicit transcription.
In a proof-of-principle study, SNF1 was bound to the BD C-terminus of
GAL4 and called the <q>bait</q>, and SNF4 was constructed to
the GAL4 AD, called the <q>prey</q> protein.  Plasmids were
transformed into yeast that had a <i>lacZ</i> reporter under the control
of GAL4.  This is now known as the GAL4 system.  Other systems include
LexA or <i>interaction trap</i> system, utilizing the BD of the bacterial
repressor LexA and the AD of <i>E. coli</i> B42.
<p>
The nature of this allows the generation of many prey constructs in order
to screen a library of prey proteins.  In early work there were a large
number of false positives, but methods have evolved which put a higher
stringency on interactions.  One means of producing higher stringency
is to use yeast that carry several reporters (<i>lacZ</i>, <i>HIS3</i>,
<i>LEU2</i>, <i>URA3</i>, <i>ADE2</i>, <i>gusA</i>, <i>GFP</i>,
<i>MEL1</i>).  GAL4 stringency is increased by adding 3-amino-1,2,4-triazole
(3-AT) to millimolar concentrations, a competitive inhibitor of HIS3
gene product;  this reduces background caused by basal <i>HIS3</i> expression.
Stringency in LexA is determined by the use of reporter genes that have
different numbers of LexA operators.	It was also an improvement in
developing tow mating types, MAT&alpha; and MATa as opposed to using
the original co-transformation method, because expression in the haploids
are already at a steady state level.  Sensitivity of the system is
such that it can determine differences of a single amino acid.
<p>
Among the difficulties with using yeast two-hybrid are the false positives,
as well as the rare false negative.  One problem is the ability of
constructs to localize to the yeast nucleus, which they must if the
system is to function.	Proteins have <i>nuclear localization signals</i>
(NLS), but the bait and prey themselves may have signals that cause
localization to other compartments.  An assay was developed to determine
if the bait localizes to the nucleus.
<p>
<b>Sos Recruitment</b>  The yeast strain cdc25H carries a
temperature-sensitive mutation in the Ras guanyl nucleotide exchange
factor (GEF).  It can be rescued with human Ras GEFhSos, which
is recruited to the plasma membrane, activates yRas, and reconstitutes
signaling.  hSos is fused to a test protein, and the second is
fused to a v-Src myristilation signal that localizes to the yeast
plasma membrane.  At the restrictive temperature, cdc25 is no functional
and cannot activate Ras.  After mating hSos is recruited to the membrane
and substitutes for cdc25 at the restrictive temp.  Only interactions
that occur permit yeast growth, and the advantage here is that nuclear
localization is not required.  In addition, post-translational modifications
important for some interactions are more likely in the cytoplasm.
A hybrid using mRas to rescue cdc25 has also been constructed.
Problems with the cdc25H system include reversion of temperature
sensitivity, in which case the untransformed control must be tested.
Expression libraries used with the systems encode proteins fused with the
signal, and Ras proteins are common false positives; procedure can be
used to eliminate proteins encoding false positives.
<p>
<b>Split-ubiquitin</b>.  Ubiquitin can tag proteins for destruction in
the 26S proteosome, and ubiquitin is recycled by ubiquitin-specific
proteases (UBPs) back into the cytoplasm.  One group produced a
split-ubiquitin construct by fusing protein X to the C-terminal half
of ubiquitin (Cub) with a reporter attached.  The second test protein Y
was fused to the N-terminal half of ubiquitin (Nub).  If X and Y interact,
the split ubiquitin join to form an active ubiquitin recognized by UBPs,
and the reporter protein is released.  The first system made use of
an immunochemically identifed reporter.  Other systems have been constructed,
such as an A-LexA-VP16 transcription factor which when released activates
<i>lacZ</i> and <i>HIS3</i>.  In another system Ura3 converts 5-fluoroorotic
acid to 5-fluorouracil which is toxic:  Cub alone is not recognized by
UBPs, but Cub-Ura3 is toxic to cells grown on 5-FOA, and if Cub-Ura3
is a fusion of protein X, interacting with protein Y fused to Nub,
Ura3 is released by UBPs and is degraded in the proteosome such
that it can not be toxic.
<p>
<b>Dual-Bait</b>.  In this system, <i>lacZ</i> and <i>LEU2</i> are
put under the control of LexA, and LYS2 and gusA are bound to
&lambda;cI.  Bait protein X1 is fused to LexA BD and second bait
protein X2 is a fusion of &lambda;cI BD.  The appropriate yeast strain
is co-transformed, and interaction is done with prey protein fused to
B42 AD.
<p>
<b>Yeast One-Hybrid System</b>.  A variation which studies protein-DNA
interactions.  Here the bait is not a protein but known DNA sequence.
Multiple copies of bait DNA are inserted upstream of a reporter (<i>lacZ</i>,
<i>HIS3</i>).
<p>
Numerous <i>in vitro</i> methods are being used to study interactions
including co-immunoprecipitation, pull-down assays, affinity chromatography,
protein array chips, and surface plasmon resonance.  Bait proteins
can be modified with affinity tags (GST, His6, CaM-binding peptide,
and common epitope tags such as HAgg, myc, FLAG).  Some tags are used in
combination on the same protein to permit extensive purification
(tandem affinity purification).
<p>
<b>Large-scale two-hybrid screens</b>.  A matrix-based approach clones
full-length open reading frames into bait and prey two-hybrid vectors
each transformed into MATa and MAT&alpha; strains.  Each bait strain
is mated with an array of prey strains, diploids identified by reporter
gene products.  In the library-based approach, undefined prey libraries
expressing either full-length ORFs or randomly generated cDNAs are
screened with bait strains.  Screen simplification occurs when pools
of the library are screened against each bait.  The first eukaryotic
genome to be completely sequenced and annotated was <i>S. cerevisiae</i>
in 1996.  This led to the first comprehensive analysis of the interactome
using matrix- and library-based approaches of proteins encoded by
6000 predicted ORFs.  Using matrix-based 192 bait proteins were screened
against array of 6000 prey proteins, and 281 interacting protein
pairs identifed.  In the library-based approach, 6000 prey proteins
were pooled to generate a complex library.  Using a partially automated
system, the library was screened with individual bait representing
6000 ORFs;  692 interactions were found.  Comparisons show that
matrix-based approach showed 3.4 interactions per bait, while 1.8
interactions per bait were seen with library-based, making the former
more sensitive.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Kakniashvili DG, Bulla LA Jr, Goodman SR
 </span>
 <span class="year">
(2004)
 </span>
 <span class="jname">
Molec. Cell. Proteomics
 </span>
 <span class="jvol">
3,
 </span>
 <span class="pages">
501-509.
 </span>
<p class="reftitl">
The Human Erythrocyte Proteome
<p class="abstract">
<b>ABSTRACT</b>.
This report describes an analysis of the red blood cell
proteome by ion trap tandem mass spectrometry in line
with liquid chromatography. Mature red blood cells lack
all internal cell structures and consist of cytoplasm within
a plasma membrane envelope. To maximize outcome,
total red blood cell protein was divided into two fractions
of membrane-associated proteins and cytoplasmic proteins.
Both fractions were divided into subfractions, and
proteins were identified in each fraction separately
through tryptic digestion. Membrane protein digests were
collected from externally exposed proteins, internally exposed
proteins, spectrin extract mainly consisting of
membrane skeleton proteins, and membrane proteins minus
spectrin extract. Cytoplasmic proteins were divided
into 21 fractions based on molecular mass by size exclusion
chromatography. The tryptic peptides were separated
by reverse-phase high-performance liquid chromatography
and identified by ion trap tandem mass
spectrometry. A total of 181 unique protein sequences
were identified: 91 in the membrane fractions and 91 in the
cytoplasmic fractions. Glyceraldehyde-3-phosphate dehydrogenase
was identified with high sequence coverage
in both membrane and cytoplasmic fractions. Identified
proteins include membrane skeletal proteins, metabolic
enzymes, transporters and channel proteins, adhesion
proteins, hemoglobins, cellular defense proteins, proteins
of the ubiquitin-proteasome system, G-proteins of the
Ras family, kinases, chaperone proteins, proteases,
translation initiation factors, and others. In addition to the
known proteins, there were 43 proteins whose identification
was not determined.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
G&ouml;rg A, Weiss W, Dunn MJ
 </span>
 <span class="year">
(2004)
 </span>
 <span class="jname">
Proteomics
 </span>
 <span class="jvol">
4,
 </span>
 <span class="pages">
3665-3685.
 </span>
<p class="reftitl">
Current two-dimensional electrophoresis technology for proteomics
<p class="abstract">
<b>REVIEW</b>.
<p>
<b>ABSTRACT</b>.
Two-dimensional gel electrophoresis (2-DE) with immobilized pH gradients (IPGs) combined
with protein identification by mass spectrometry (MS) is currently the workhorse for proteomics.
In spite of promising alternative or complementary technologies (e.g. multidimensional
protein identification technology, stable isotope labelling, protein or antibody arrays) that have
emerged recently, 2-DE is currently the only technique that can be routinely applied for parallel
quantitative expression profiling of large sets of complex protein mixtures such as whole cell
lysates. 2-DE enables the separaration of complex mixtures of proteins according to isoelectric
point (pI), molecular mass (Mr), solubility, and relative abundance. Furthermore, it delivers a
map of intact proteins, which reflects changes in protein expression level, isoforms or posttranslational
modifications. This is in contrast to liquid chromatography-tandem mass spectrometry
based methods, which perform analysis on peptides, where Mr and pI information is
lost, and where stable isotope labelling is required for quantitative analysis. Todays 2-DE
technology with IPGs (Grg et al., Electrophoresis 2000, 21, 10371053), has overcome the
former limitations of carrier ampholyte based 2-DE (OFarrell, J. Biol. Chem. 1975, 250, 4007
4021) with respect to reproducibility, handling, resolution, and separation of very acidic and/or
basic proteins. The development of IPGs between pH 2.512 has enabled the analysis of very
alkaline proteins and the construction of the corresponding databases. Narrow-overlapping
IPGs provide increased resolution (DpI = 0.001) and, in combination with prefractionation
methods, the detection of low abundance proteins. Depending on the gel size and pH gradient
used, 2-DE can resolve more than 5000 proteins simultaneously (,2000 proteins routinely),
and detect and quantify,1 ng of protein per spot. In this article we describe the current 2-DE/
MS workflow including the following topics: sample preparation, protein solubilization, and
prefractionation; protein separation by 2-DE with IPGs; protein detection and quantitation;
computer assisted analysis of 2-DE patterns; protein identification and characterization by
MS; two-dimensional protein databases.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Meng Z, Simmons-Willis TA, Limbach PA
 </span>
 <span class="year">
(2004)
 </span>
 <span class="jname">
Biomolec. Eng.
 </span>
 <span class="jvol">
21,
 </span>
 <span class="pages">
1-13.
 </span>
<p class="reftitl">
The Use of Mass Spectrometry in Genomics
<p class="abstract">
<b>REVIEW</b>.
<p>
<b>ABSTRACT</b>.
Recent advances in the development of electrospray ionization (ESI) and matrix-assisted laser desorption/ionization (MALDI) now permit
the near routine analysis of oligonucleotides and intact nucleic acids. These developments have led to the use of mass spectrometry (MS)
as a detection platform for genomics studies. Among the various uses of mass spectrometry in genomics, applications focused on the
characterization of single nucleotide polymorphisms (SNPs) and short tandem repeats (STRs) are particularly well-suited to MALDI or
ESI-based analysis. It is predicted that continued developments in methodology and instrumentation will further improve the capabilities of
mass spectrometry for nucleic acid analysis.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Smith RD, Shen Y-F, Tang K
 </span>
 <span class="year">
(2004)
 </span>
 <span class="jname">
Acc. Chem. Res.
 </span>
 <span class="jvol">
37,
 </span>
 <span class="pages">
269-278.
 </span>
<p class="reftitl">
Ultrasensitive and Quantitative Analyses from Combined Separations-Mass Spectrometry
for the Characterization of Proteomes
<p class="abstract">
<b>ABSTRACT</b>.
This article describes developments in fundamental and applied
aspects of separations, electrospray ionization phenomena, and
mass spectrometric instrumentation that are interrelated and
important for making more effective and quantitative measurements,
particularly for proteomics applications. The basis for better
quantitation and ultrahigh sensitivity is highlighted for highresolution
capillary liquid chromatography separations that provide
low nanoliter per minute flow rates to an electrospray ionization
interface. The increased dynamic range of measurements and low
zeptomole regime detection limits obtainable open new avenues
for biological research.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Balmer Y, Koller A, del Val G, Sch&uuml;rmann P, Buchanan BB
 </span>
 <span class="year">
(2004)
 </span>
 <span class="jname">
Anal. Chem.
 </span>
 <span class="jvol">
79,
 </span>
 <span class="pages">
275-280.
 </span>
<p class="reftitl">
Proteomics Uncovers Proteins Interacting Electrostatically with Thioredoxin
in Chloroplasts
<p class="abstract">
<b>ABSTRACT</b>.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Reinders J, Lewandrowski U, Moebius J, Wagner Y, Sickmann A
 </span>
 <span class="year">
(2004)
 </span>
 <span class="jname">
Proteomics
 </span>
 <span class="jvol">
4,
 </span>
 <span class="pages">
3686-3703.
 </span>
<p class="reftitl">
Challenges in Mass Spectrometry-Based Proteomics
<p class="abstract">
<b>REVIEW</b>.
<p>
<b>ABSTRACT</b>.
During the last decade, protein analysis and proteomics have been established as new tools
for understanding various biological problems. As the identification of proteins after classical
separation techniques, such as two-dimensional gel electrophoresis, have become standard
methods, new challenges arise in the field of proteomics. The development of functional
proteomics combines functional characterization, like regulation, localization and modification,
with the identification of proteins for deeper insight into cellular functions. Therefore, different
mass spectrometric techniques for the analysis of post-translational modifications,
such as phosphorylation and glycosylation, have been established as well as isolation and
separation methods for the analysis of highly complex samples, e.g. protein complexes or cell
organelles. Furthermore, quantification of protein levels within cells is becoming a focus of
interest as mass spectrometric methods for relative or even absolute quantification have currently
not been available.
Protein or genome databases have been an essential part of protein identification up to now.
Thus, de novo sequencing offers new possibilities in protein analytical studies of organisms
not yet completely sequenced.
The intention of this review is to provide a short overview about the current capabilities of
protein analysis when addressing various biological problems.
<!-- ============================================================= -->
<p class="tags">

<p class="ref" id="stasyk-2004">
 <span class="authors">
Stasyk T, Huber LA
 </span>
 <span class="year">
(2004)
 </span>
 <span class="jname">
Proteomics
 </span>
 <span class="jvol">
4,
 </span>
 <span class="pages">
3704-3716.
 </span>
<p class="reftitl">
Zooming In:  Fractionation Strategies in Proteomics
<p class="Para1">
<b>REVIEW</b>.  Fractionation strategies must be employed to attempt to
recover low abundance proteins such as regulatory proteins (kinases,
phosphatases, etc.).  It is also necessary to enrich certain proteins
corresponding to a particular subproteome of interest.
<p>
<b>Organelle proteomics</b>.  Subcellular fractionation allows the
separation of organelles based on physical and biological properties
and involves first the disruption of the cell (homogenization), and then
fractionation of the homogenate into its component organelles.  Centrifugation
is the typical approach.  First the intact cells are collected by low
speed spin.  After disruption (mechanical usually), nuclei are removed with
a low speed spin and then possibly purified to remove debris and unbroken
cells.  The postnuclear supernatant is cytosol and other organelles which
can be separated in an equilibrium gradient medium, or possibly by
rate zonal method.  Sucrose is the typical gradient medium, but Ficoll,
Percoll, Nycodenz or Metrizamide are also used.  Free flow electrophoresis
and immunoisolation are also used.  Although the organelles can be
relatively enriched, they cannot be completely pure.  Enrichment is still
valuable since it can bring low abundance proteins to signal levels.
For example, p14 is a low <i>M</i><sub>r</sub> protein found in late
endosomes, a fact obtained by enrichment.  The protein is an adaptor
protein for targeting MAPK signaling to the late endosomal compartment
in an alternative EGFR pathway.
<p>
Enriched endosomal fractions were studied using 2D-DIGE and showed an
enrichment of 305 (2- to 120-fold)  and 292 (2- to 25-fold) protein
spots in purifed late and early endosomal fractions in comparison to
protein of the PNS serving as the starting fraction.  Wu et al. took
an organelle proteomics approach by first isolating the Golgi fraction
from rat liver using sucrose step gradient centrifugation.  Golgi
samples were digested and then analyzed by MudPIT using triphasic
chromatography (reverse phase, SCX, and HIC).  In 421 identified proteins,
110 were known to be resident in Golgi, and 64% were predicted
transmembrane proteins.  In other studies Kikuchi et al. used Nycodenz
density gradient centrifugation to isolate peroxisomes which were
further purified with anti-PMP70 antibodies (70 kDa peroxisome membrane
protein) immunomagnetically.
<!-- ================================================================ -->
<p class="tags">

<p class="ref">
 <span class="authors">
Drake SK, Bowen RAR, Remaley AT, Hortin GL
 </span>
 <span class="year">
(2004)
 </span>
 <span class="jname">
Clin. Chem.
 </span>
 <span class="jvol">
50,
 </span>
 <span class="pages">
2398-2401.
 </span>
<p class="reftitl">
Potential Interferences from Blood Collection Tubes in Mass Spectrometric
Analyses of Serum Polypeptides
<p class="Para1">
Greatest sensitivity and resolution of MALDI-ToF is in the mass range
500-20,000 Da.  Many blood collection tubes contain components that
might interfere with these analyses.  Silicones for lubricants for
stoppers or coating on internal surfaces.  Polyvinylpyrrolidones
or polyethylene glycols are useful in surface wetting.  Plus clot
inhibitors or activators.
<p>
1 ml PBS (pH 7.2, KD Medical)  was added to collection tubes and incubated
at RT for ~4 h with gentle rocking with contact on all surfaces, simulating
real conditions of blood handling.  Tubes studied included BD, Greiner
Bio-One Vacuette North America.   These are mixed with 0.5 vol of
6 M guanidine HCl + 3 g/L TFA (0.3%).  30 &micro;l of these were extracted
to C18 ZipTips (0.6 &micro;l).  Tips are washed three times with 0.1% TFA,
then eluted with 3 &micro;l of 50% MeCN + 0.1% TFA.  1 &micro;l of this
and of 50 mM CHCA or sinapinic acid in 50% MeCN + 0.1% TFA are mixed
on 384-well target.  Analysis is with Bruker UltraFlex MALDI-ToF in
positive linear mode;  300 shots on 10 positions done, on external calibrated
system.
<p>
The serum tubes (except for no-additive glass) contributed much background
in the <i>m/z</i> 1000-3000 range.  A series of 5-6 peaks differing by
14 Da (assuming +1 state) and repeating every 111 Da were found.  The
111 Da repeat is consistent with polyvinylpyrrolidone monomer, while the
14 Da difference represents a -CH<sub>2</sub>- group.  CHCA was better than
sinapinic for producing higher intensities in range &lt; <i>m/z</i> 5000.
BD plastic serum separator showed peaks in the <i>m/z</i> 1800-1900 range,
while the Greiner Bio-One showed peaks at about <i>m/z</i> 1400.
<p>
Extracts of EDTA-containing plasma tubes and no-additive glass with glycerol
as lubricant showed no significant background peaks that were obvious.
An analysis program that looks for S/N &gt; 20 found nothing.
<p>
One plasma tube for research use created to stabilize proteins for analysis
had several contaminating peaks (<i>m/z</i> 1052, 1066, 6509, 6696, and
6880).  These may correspond to protease inhibitor additives.
<p>
Conclusions include testing all containers with saline extraction technique
to test for potential interferants in the analysis range.  There is a
general recommendation that plasma collection tubes contribute less potential
interferents compared to serum tubes.
<!-- ================================================================ -->
<p class="tags">

<p class="ref">
 <span class="authors">
Giles K, Pringle SD, Worthington KR, Little D, Wildgoose JL, Bateman RH
 </span>
 <span class="year">
(2004)
 </span>
 <span class="jname">
Rapid Commun. Mass Spectrom.
 </span>
 <span class="jvol">
18,
 </span>
 <span class="pages">
2401-2414.
 </span>
<p class="reftitl">
Application of A Travelling Wave-Based Radio-Frequency-Only Stacked Ring
Ion Guide
<p class="Para1">
Radiofrequency (RF) multipole ion guides are used for transportation of ions.
Gerlich has detailed of these ion guides and says that at pressures low
enough, collisions with background gas molecules are unimportant.  In some
cases these guides can be used to cool ions to permit thermal ion-molecule
reaction studies.  Quadrupole, hexapole and octapole guides are useful
when the gas pressure is high 10<sup>&minus;3</sup> to 10<sup>&minus;3</sup>
mbar.  Electrostatic lens for focusing and transport is difficult because
of scattering at moderate background pressures.  By reducing the effective
potential well using RF-only multipoles, these losses in ions are reduced.
Bahr et al (1969) first discussed the stacked ring ion guide (SRIG)
and Gerlich and coworkers developed it.  Smith and coworkers have pursued
the ion funnel concept, in which the apertures of the ring electrodes
are consecutively reduced in diameter from entrance to exit.
<!-- ================================================================ -->
<p class="tags">

<p class="ref">
 <span class="authors">
Khoudoli GA, Porter IM, Blow JJ, Swedlow JR
 </span>
 <span class="year">
(2004)
 </span>
 <span class="jname">
Proteome Sci.
 </span>
 <span class="jvol">
2,
 </span>
 <span class="pages">
6-17.
 </span>
<p class="reftitl">
Optimization of The Two-Dimensional Gel Electrophoresis Protocol Using
The Taguchi Approach
<p class="Para1">
Rehydration buffers typically contain chaotropes, detergents, reducing
agents, and carrier ampholytes.  The combination 7 M urea/2 M thiourea
reportedly produces better 2-D gels with IPG strips compared to 8 M urea.
The reducing agents DTT, TBP, anc TCEP have all been used and were
test as 20 mM DTT, 2 mM TBP, or 10 mM TCEP in our urea/thiourea/4% CHAPS/0.5%
ampholyte preparation.  50 &micro;g <i>Xenopus</i> egg protein was
obtained by MeOH/chloroform precipitation and used in the rehydration.
DTT showed the best results (TBP and TCEP) reduced focusing.
<p>
When carrier ampholytes were added at higher concentrations to determine
optimal, the focusing time was lengthened.  A one-step fast-ramping gradient
from 250-5500 V after 30 min at 250 V works well for 7 cm strips at all
pH ranges.  A total 33000 Vh produced good results (with a restriction
to 50 &micro;A per strip); this was sufficient for the highest carrier
ampholyte concentration.
<p>
It is thought that the presence of SDS in alkylation buffer prior to
2nd dimension separation reduces the completeness of the alkylation.
It has been proposed that alkylation be done before IEF solubilization,
but thiourea can also prevent alkylation by iodoacetamide.  Acrylamide
is an alternative, so it was used to 60 mM concentration in the rehydration
buffer after precipitated protein was solubized for 2 h in DTT.  Acrylamide
was also added to the 2nd equilibration buffer before SDS-PAGE.
<p>
The Taguchi experimental optimization arranges variables in an orthogonal
array.  For 2DGE, each column in the array (table) is the individual
buffer component of the IEF rehydration buffer, and each row represents the
individual IEF rehydration buffer.  Each component is defined at 3 possible
concentrations (A, B, C) to cover the range of varying effects.  Effectiveness
is determined by a count of spots on the 2D gel, and Taguchi's signal-to-noise
ratios (SNR) are calculated for all components:
<table class="eqnserif" style="text-align:center;">
<tr><td rowspan="2" style="vertical-align:middle;"><i>SNR</i> = &minus;10 log
 <span style="font-size:250%;">[</span>
 <td class="numer">1
 <td rowspan="2"><span style="font-size:150%;">&sum;</span><sub
 style="margin-left:-1.25em;vertical-align:-1.25em;"><i>i</i>=1</sub>
	<sup style="margin-left:-1.5em;vertical-align:1.75em;"><i>n</i></sup>
 <td rowspan="2" style="vertical-align:middle;">
 <td class="numer">1
 <td rowspan="2"><span style="font-size:250%;">]</span>
<tr><td><i>n</i>
 <td><i>Y<sub>i</sub></i><sup>2</sup>
</table>
<p>
where <i>n</i> is number of trials with given concentration,
and <i>Y<sub>i</sub></i> is yield on corresponding trials.  As an example
to calculate <i>SNR</i> for 1% CHAPS, the total spot counts of gels
2, 5, 8 (<i>n</i> = 3)  were used since they used 1% CHAPS.  A 2nd order
polynomial was used to determine the concentration corresponding to
maximum <i>SNR</i> as if were plotted.  The Taguchi method factors
performance as a measure of mean and variability (SNR is ratio of mean
to standard deviation in simplest form).  SNR plots were determined for
CHAPS and DTT, and determination of 1.32% CHAPS and 34 mM DTT was found
as maximal SNR values.  When using the lowest concentration of ampholytes
and ASB14, the highest SNR values are seen.  When experiments were repeated
three time, optimal CHAPS is found to be 1.20%, DTT to be 43 mM, 0.25%
ampholytes, and 0.4% ASB14.
<p>
While spot count was the measure of performance, spot circularity, streaking,
spot intensity, and so on could have been factored in as well.  When
ASB14 was greater than 2.0% significant streaking and spot shape changes
occurred (thus reducing spot number).
<p>
Lower concentrations of ampholytes and ASB14 may improved detected spot
number.  From 0.1% to 0.4% ASB14 and 0.05% to 0.25% ampholytes, there
was little change in spot number.  Was ASB14 even useful?  A rehydration
buffer of 8 M urea, 4% CHAPS, 0.5% ampholytes and 20 mM DTT (called sRB)
produces 425 spots, while 7 M urea, 2 M thiourea, 1.2% CHAPS, 0.4% ASB14,
and 0.25% ampholytes (called oRB) produces 659 spots, a 50% increase.
Since our interest was in chromatin-associated proteins (Lys and Arg-rich),
it was observed that there was a paucity of basic and high MW proteins
on the gels with the 1st buffer, but a larger number were seen with
the 2nd buffer.
<p>
The improved oRB buffer was able to reveal more about the chromatin proteins,
eliminating ambiguities on immunoblots of high MW proteins. Condensin
component XCAP-E (140 kDa)
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Wilmarth PA, Riviere MA, Rustvold DL, Lauten JD, Madden TE, David LL
 </span>
 <span class="year">
(2004)
 </span>
 <span class="jname">
J. Proteome Res.
 </span>
 <span class="jvol">
3,
 </span>
 <span class="pages">
1017-1023.
 </span>
<p class="reftitl">
Two-Dimensional Liquid Chromatography Study of the Human Whole Saliva
Proteome
<p class="Para1">
Unstimulated whole saliva from healthy 28 y o male nonsmoker was spun
20000 g for 15 min and supntnt dialyzed against 12 mM ammonium bicarbonate
overnight with 10000 MWCO.  BCA assay shows 1.1 mg/ml.  Sample dried by
vacuum centrifugation.  Sample dissolved in 8 M urea, 0.4 M ammonium bicarb,
80 mM methylamine in 0.5 ml total.  125 &micro;l of 0.9 M DTT for 15 min
at 50&deg; followed by 250 &micro;l 1 M iodoacetamide at room temp in dark
15 min.  Another 125 &micro;l 0.9 M DTT, then 500 &micro;l 1 M ammonium bicarb,
then 100 &micro;g seq.-grade trypsin (Promega V5113) in 500 &micro;l water,
left 18 h at 37&deg; with shaking.  75 &micro;l of 88% formic acid is added
and peptides purified on SepPak Light cartridge, eluting with 0.5 ml 50% MeCN,
then 0.5 ml 100% MeCN.  Digest (vol=2.0 ml) injected on to 100 &times; 2.1 mm
sulfoethyl column at 0.2 ml/mi with solvent A (10 mM NaPi pH 3.0 + 25% MeCN)
for 10 min isocratic and gradient with solvent B (A + 0.35 M KCl) from 0-50%
over 40 min, then 50-100% over 20 min.  2-min fractions collected and dried
on vacuum, redissolved in 160 &micro;l 5% formic acid, then run on
3.0 cm &times; 180 &micro;m trap with 5 &micro;m Zorbax SB-C18 using a 0.2%
acetic acid mobile phase.  Sample is loaded over 10 min, the flow rate slowed
to 1.5 &micro;l/min and then the trap switched  inline to a 10 cm &times; 180
&micro;m column with same packing.  Peptides are eluted using mobile phase B
with 75% MeCN and 0.2% acetic acid.  Gradient consisted of 0-10% B over 5 min,
10-40% B over 60 min, and 40-100% B over 25 min.  MS scans from <i>m/z</i>
400-2000 were followed by single data-dependent MS/MS scans with dynamic
exclusion with repeat count of 1, repeat duration of 1.0 min, exclusion duration
of 3 min, and exclusion mass width of 3.
<p>
37 SCX fractions were generated and 36,000 MS/MS spectra were analyzed by
Sequest searches.  Outputs were filtered with DTASelect, and 892 nonredundant
peptides mapped to 268 proteins.  Rather strict DTAselect filtering thresholds
were placed and additional steps taken to ensure reliability of protein
identifications.  SwissProt entries were consulted before the nonredundant
NCBI human entries, used as a second resort. 84 proteins were identified with
three or more peptides using the DTASelect cutoffs, accepted without further
validation.  28 protein identifications based only on two peptides were
manually validated.  7 proteins were removed from a total of 105 valid
database entries.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Tuteja R, Tuteja N
 </span>
 <span class="year">
(2004)
 </span>
 <span class="jname">
BioEssays
 </span>
 <span class="jvol">
26,
 </span>
 <span class="pages">
916-922.
 </span>
<p class="reftitl">
Serial Analysis of Gene Expression (SAGE):  Unraveling The Bioinformatics Tools
<p class="Para1">
<b>REVIEW</b>.  Many important cell biology events are associated with
the expression of key genes:  mitosis/division, differentiation, signal
response to hormones and growth factors, apoptosis.  In understanding
disease etiology the comparison of different expression profiles under
different conditions identifies individual and groups of genes.  Methods
for assaying expression fall into two classes: <b>closed</b> in which
expression levels are measured by collected clones and sequences, and
<b>open</b> where prior knowledge of genes to be measured is not required.
Subtractive hybridization and differential display are methods used to
screen transcript populations.  They are however insensitive to low level
transcription.  Use of microarrays of oligonucleotides can catch these
low level transcripts but limit themselves to previously identified
transcripts (cDNA or EST libraries).  Massively parallel signature
sequencing (MPSS) is another global approach combining non-gel-based
signature sequencing with the in vitro cloning of millions of templates
on separate microbeads.
<p>
SAGE is a high throughput, high efficiency way to detect expression at
any point in time and particularly of low level gene expression.
SAGE works by generating tags that are 9-10 bases long.  These are then
concatenated serially into long DNA molecules before a final sequencing
of the serialized tags.  The frequency of the occurrence of tags is
a direct function of its level of expression.  Thus the gene belonging
to the tag is both identified and quantitated for its expression level.
With just one clone, 30-50 different mRNA can be identified.  The
development of LongSAGE was recently described in which 21 base-long tags
are made and rapidly serialized to genomic sequence data.  Results in
SAGE are digital, making analysis and data management more amenable to
bioinformatic manipulation.
<p>
A 10-11 base (pair) long tag is sufficient for unique identification
of a transcript.	This size of the tag is convenient for serializing
into sequencable samples whose length allows sequencing of between 25-50
tags in the lane of a DNA sequencer.  In the SAGE method a 9-10 base
(pair) long tag is made by reverse transcription (first and
second strand synthesis) of poly(A)<sup>+</sup> RNA into a biotinylated
ds cDNA.  Fragments contain an anchoring enzyme <i>Nla</i>III linker
to generate fragments whose average length is 256 bp.  Biotinylated
fragments are isolated using strepavidin magnetic beads.  The
fragments are divided into two, and each half is ligated to
two different linkers.  Both linkers have sites for a tagging
enzyme (<i>Bsm</i>FI), an anchoring enzyme overhang, and priming site
for PCR amplification.  The tagging enzyme is then used to digest
the bound linker-cDNA sequences to generate fragments composed of linker
and bound cDNA tag, which is then released from the bead.  The linker
tages are then blunt-ended (ss nuclease).  The two sets of linker-tags
are ligated to form linker-ditag-linker constructs and then PCR-amplified
with primers specific to the linkers.  The anchoring enzyme is then used
to release the ditags.  Ditags are isolated, ligated to concatemers, cloned,
and then sequenced.  Many modifications have been made to the original
SAGE method and these modifications are described by various names:
SAGE-lite, micro-SAGE, mini-SAGE, SADE (an adaptation for downsized
extracts), and GLGI (Generation of Longer cDNA fragments from SAGE tags
for Gene Identification).
<p>
Sequence results from SAGE are first inspected to confirm tag content.
Software first extracts and tabulates the sequences and then counts their
occurrences.  It then should compare tag abundance between projects/experiments.
It should also match tags to reference sequences in other databases.
<p>
The company Genzyme has produced a propietary SAGE database as part of their
therapeutic discovery process.  It has over 4 million tags probably representing
100,000 genes from tissues both normal and cancerous.  For a license fee
it will make the library information available.
<p>
SAGEnet is a public database (<a href="http://www.sagenet.org/">http://www.sagenet.org/</a>)
with tags from tissues of colon and pancreatic cancers as well as control
(healthy) tissues.  It is maintained by the Vogelstein/Kinzler lab at
Johns Hopkins.
<p>
SAGEmap is a public repository maintained by NIH NCBI in collaboration with
the NIH Cancer Genome Anatomy Project (CGAP).  The database is accessible
through both <a href="http://www.ncbi.nlm.nih.gov/sage">WWW</a> and
<a href="ftp://ncbi.nlm.nih.gov/pub/sage">FTP</a> servers.  Procedures to
submit data, construct and characterize SAGE tags to gene assignments,
the derivation and use of a special statistical test to analyze
differential-type SAGE data are described.  CGAP SAGE created <a
href="http://cgap.nci.nih.gov/SAGE">SAGE Genie</a> for analyzing
and presenting SAGE data.  One of the advantages of SAGEmap is that it
can map SAGE tags to UniGene EST clusters.  It can incorporate EST frequency
location, and orientation info as well as correct sequencing error in
refining mapping of tags to genes.
<p>
The <a href="http://mouse.biomed.cas.cz/sage/">mouse SAGE</a> site
is a database of all available SAGE libraries of mouse tissues and cell
lines.  This is maintained by the Institute of Molecular Genetics at
the Academy of Sciences of the Czech Republic.  At Stanford, there
is a Saccharomyces Genome Database the includes a <a
href="http://genome.www.stanford.edu/cgi-bin/SGD/SAGE/query/SAGE">SAGE
resource</a>.
<p>
The program SAGE300 was developed to compile a database of tags extracted
from human sequences in Genbank.  It provides a statistical analysis
to report a <i>p</i> value in the event of differences between sequenced
tags and expected tags.  Man et al. compared three commonly used
statistical tests that deal with pairwise sequence comparisons:
the chi-square test (test for equality of two proportions),
Fisher's exact test, and the Audic-Claverie Bayesian method (see
<a href="http://igs-server.cnrs-mrs.fr~audic/significance.html">site</a>),
and they found that the chi-square test has the greatest power and robustness,
especially for lower expression levels.
<!-- ============================================================= -->
<p class="tags">
discriminant function, peptide identification
<p class="ref">
 <span class="authors">
Strittmatter EF, Kangas LJ, Petritis K, Mottaz HM, Anderson GA, Shen Y, Jacobs 
JM, Camp II DG, Smith RD
 </span>
 <span class="year">
(2004)
 </span>
 <span class="jname">
J. Proteome Res.
 </span>
 <span class="jvol">
3,
 </span>
 <span class="pages">
760-769.
 </span>
<p class="reftitl">
Application of Peptide LC Retention Time Information in a Discriminant Function 
for Peptide Identification by Tandem Mass Spectrometry
<p class="Para1">
Complete genomes done for more than 150 organisms.  Much proteomic data coming
in using peptide identfication from MS/MS using pattern matching algorithms such
as SEQUEST and Mascot, where scores indicate confidence.  In order to determine
better thresholds between poor and good scores, Keller et al used a discriminant
function based on several SEQUEST parameters and Bayesian statistics for creating
probabilistic model for accepting or rejecting peptide identfiications.
<p>
A neural network-based model for determining peptide retention time was derived 
from large collection of data on peptides from digests of <i>D radiourans</i> 
and <i>S oneidensis</i> proteins.  The model predicts retention time based on 
the amiono acid composition of a peptide:  this is the normalized elution time 
(NET<sub>pred</sub>) using specific separation conditions.  Difference between 
NET<sub>pred</sub> and NET<sub>obs</sub> are calculated for all tentative 
identifications, which includes those of 17 control proteins.  Assignments for
the control proteins showed a less broad (errant) histogram/distrbution than
for <i>D melanogaster</i> proteins for which the training set was not developed.
90% of the control proteins had a deviation &le; 0.10, and 50% of identifications
of within 2.5%.  Note that all NET<sub>obs</sub> results are the mean of several
LC/MSMS experiments which were quite reproducible (median standard deviation for
the peptides was 0.0034 NET units).
<p>
Nine LC-MS/MS datasets with 14,762 spectra were used to develop a NET-based 
discriminant. Of the spectra 3420 have top-scoring spectrum assigments (648 
unique sequences) to one of the 17 control proteins.  The ideal score criteria 
would make a distinction between correct and incorrect assignments.  There is no 
perfect discrimination however because data quality and noise among other 
factors.
<p>
Washburn et al defined a cross-correlation (X<sub>corr</sub>) and 
&Delta;<i>C<sub>n</sub></i> criteria, making 2241 nonunique peptide 
identifications to the control set, across all charge states.  If the data of 
these experiments is applied using Washburn algorithms, the false positive rate 
is ~5%.  The effect of applying a <i>d</i><sub>NET</sub> (<i>d</i><sub>NET</sub> 
= |NET<sub>pred</sub> &minus; NET<sub>obs</sub>|), which for <i>D 
melanogaster</i> halves the number of incorrect identifications and reducing the 
false positive rate.  Looking at peptide identifications grouped by charge state,
there was improvement for +1 and +3 charge state peptides.  The count of control
proteins peptide identifications as a function if X<sub>corr</sub> grouped by 
raw number and as fraction of identifications of +1 protonated peptides was done
by binning data in 0.2 Xcorr intervals.  Results with and without the <i>d</i><sub>NET</sub>
criteria show that the <i>d</i><sub>NET</sub> &lt; 0.10 has a limited impact on 
number of control protein set identifications while the false positive reate is
reduced by nearly half.  Using a &Delta;<i>C<sub>n</sub></i> &gt; 0.1 restriction
greatly reduces false positives above Xcorr = 2.0.  Results indicate that many 
valid peptide identifications may have Xcorr &lt; 1.9.  Similar plots are produced
for +2 and +3 peptides with similar trends to +1 peptides.
<p>
Parameters <i>d</i><sub>M</sub> and ln(RankSp) can be used to judge match between 
peptide and mass spectrum.  A discriminant function combining the factors 
Xcorr&prime;, &Delta;C<sub>n</sub>, <i>d</i><sub>M</sub>, and <i>d</i><sub>NET</sub>
was determined.  This is 
<p class="eqnserif">
discrim score = sigmoid(Xcorr&prime;) + sigmoid(&Delta;<i>C</i><sub>n</sub> +
sigmoid(ln(RankSp)) + sigmoid(<i>d</i><sub>M</sub> + sigmoid(<i>d</i><sub>NET</sub>)
+ tryptic parameter
<p>
where the sigmoid function is:
<p class="eqnserif">
<i>sigmoid</i>(<i>x</i>) = &beta;<sub>1</sub> / {1 + exp[(&beta;<sub>2</sub> &minus; x) / &beta;<sub>3</sub>] }
<p>
and &beta;<sub>1</sub>, &beta;<sub>2</sub>, and &beta;<sub>3</sub> are 
coefficients optimized to produce maximum separation between confident 
identifications to control protein set and the supposed errant assignments. 
Note that the SEQUEST software produces the values Xcorr (cross-correlation 
score between actual and predicted spectrum), &Delta;<i>C</i><sub>n</sub> 
(relative difference between correlation score for highest scoring peptide match 
and second hight peptide match, RankSp (rank based on preliminary Sp score for 
peptide; Sp score = preliminary score of peptide in search procedure) and 
<i>d</i><sub>M</sub> (absolute difference between precursor ion mass and mass 
derived from assigned peptide).  Xcorr and RankSp values are transformed also.
Xcorr&prime; is a transformation of Xcorr and number of expected fragment ions in
the assigned peptide:
<p class="eqnserif">
Xcorr&prime; = ln(Xcorr) / ln(<i>N</i><sub>L</sub>)
<p>
which reduces the length dependence of Xcorr (proposed first by Keller et al).
RankSp is used as its natural log.
<p>
The discriminant function provide new value <i>F</i> which separates identifications
of control proteins from false positives by combining scores of its five components.
Discriminant scores were set for each MS/MS spectra, ranging from &minus;3 to +7 (max).
Spectra are placed in distibution with bin width of 0.2 and counting number of peptides
that had <i>F</i> scores in a particular bin.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Baggerly KA, Morris JS, Coombes KR
 </span>
 <span class="year">
(2004)
 </span>
 <span class="jname">
Bioinformatics
 </span>
 <span class="jvol">
20,
 </span>
 <span class="pages">
777-785.
 </span>
<p class="reftitl">
Reproducibility of SELDI-TOF protein patterns in serum: comparing datasets from 
different experiments
<p class="Para1">
Petricoin et al (2002, Lancet) reported different patterns in SELDI-TOF MS
spectra of women with and without ovarian cancer.  Using 100 spectra from
cancer patients, 100 from healthy and 16 with <q>benign</q> disease, they
split them, using 50 cancer and 50 healthy spectra to train a classification
algorithm.  The algorithm was then used to classify remaining spectra.
50 of 50 cancer spectra were correctly classified, and 47 of 50 normals
were classed, with 3 false positives.  Of the 16 benign, it classed them
as <q>other than normal or cancer.</q>
<p>A question arises about whether differences can reflect biological 
variation or artifacts related to the technology.  Artifacts include
<ul>
<li>handling or processing samples differently
<li>changes in type of ProteinChip array
<li>mechanical adjustments to the mass spectrometer
<li>shift to the use of another instrument or lab
</ul>
<p>Three datasets of SELDI mass spectra to study ovarian conditions were
obtained.  Set 1 is from Petricoin et al 2002 with 216 spectra in five groups:
training set for cancer and normal, test/validation for cancer and normal,
and a set for benign disease.  The H4 ProteinChip was used, and all data
baseline-subtracted.  Dataset 2 is the same groups but the WCX2 chip was
used.  Dataset 3 are new samples, 91 healthy and 162 cancer; these samples
were processed robotically (the previous by hand), and run on WCX2 chip and
data are not baseline-subtracted.  File contains first column of m/z values
and second column of relative intensities.
<p>
Data <q>normalization</q> involves altering the maximum and minimum intensities
to be equal to 1 and 0, respectively, and recalculating the intensity by the
normalization factor [= <i>NV<sub>i</sub></i> = (<i>V<sub>i</sub></i> &minus;
<i>V</i><sub>min</sub>) / (<i>V</i><sub>max</sub> &minus; <i>V</i><sub>min</sub>) ]
where <i>V<sub>i</sub></i> = raw intensity and <i>NV<sub>i</sub></i> is
the normalized intensity.  The fitness of a feature set with <i>N</i> features
is assessed using associated scaled intensities to define locations in the
<i>N</i>-dimensional unit cube:  start with sample 1; if the Euclidean distance
between sample 2 and sample 1 &lt; 0.1 &times; &radic;<i>N</i>, the samples
are placed in common cluster, and the means of the samples are the center.
When sample 2 is farther is beyond the cluster test limit, a new cluster is
started.  Samples are processed until complete.  After all clusters are formed,
they are labeled <q>cancer</q> or <q>normal</q> by majority vote, and fitness
defined as number of samples correctly classified.
<p>
<b>Feature sets are then selected</b> with a genetic algorithm.  The algorithm
run starts with 1500 logical chromosomes (feature sets), the size ranging from
5-20 index values.  New populations are produced by preferential combination
of pieces of the <q>most fit</q> members of the current generation.  The process
evolves for 250 generations, and a 0.02% mutation rate and random crossover
locations.  All 15,154 spectral featuers are available for inclusion in a
feature set; with no initial peak finding step.  For each of the experiments
a single feature set that allows the cancers to be separated from the normals
is reported.
<p>
In examining the ovarian datasets for features, dataset 1 was processed
according to methods described above.  In the original report were five 
<i>m/z</i> values, so they were put into a feature set, the distance between all 
pairs was computed for a distance matrix.  The distance between cancer samples 
and normal samples was not different than between two cancer samples or between 
two normal samples.  The pattern should be <q>plaid</q>:  with small distances 
between samples of the same type and large distances between samples of 
different type.  There were only four pairwise distances &gt; 0.1 &times; 
&radic;5, the cutoff distance for new cluster with five peaks.  Clusters will 
not be formed at all.  The problem here is <b>baseline subtraction</b>.  The 
correction was noted by authors to <q>create negative intensities.</q>
The correction is an irreversible nonlinear operation: the raw values cannot
be reconstructed.  In the three datasets supplied, only Dataset 3 was not
baseline-subtracted.
<p>
In the original study, 16 samples were identified as being <q>other</q> than 
cancer or normal.  When spectra are plotted side-by-side, a <q>heat map</q> is 
produced such that regions of higher intensity show as darker bands.  This heat 
map for Dataset 1 shows the <i>other</i> group with a clearly distinct pattern, 
whereas the normal and cancer share significant similarity.  Dataset 2
however showed no significant differences in the heat map:  it differs in the
chip used.  In the original study, the analyses on both chips were concurrent
and the technicians blinded.  An analysis of data shows that Dataset 1 and 
Dataset 2 are not numerically identical.
<p>
Since Datasets 2 and 3 were done with the same chip.  While Dataset 2 was 
baseline-subtracted, qualitative features were assessed for looking at 
similarity.  In looking at indices such as major high intensity peaks, the 
spectra from the datasets appeared offset by 50-60 clock ticks.  This corresponds
to a 1% imprecision although the stated mass accuracy is 0.1%.  A quadratic
offset to the masses of dataset 3 were fitted using four peaks in calibration.
These are <i>m/z</i> 6773, 7269, 9566, and 10,260 in dataset 2 and <i>m/z</i>
6823, 7321, 9632, and 10,337 in dataset 2.  The peak matching produced and
offset of 200 Da at the spectral low end.
<p>
In comparing datasets 2 and 3, which should be comparable if the chip type
is the same, the distance matrix shows randomness and no distinct clustering.
One feature at a time was checked, starting with the seven in Dataset 3, because
Dataset 2 had been baseline corrected.  The peak <i>m/z</i> 435.46 was the
most useful in distinguising cancer and normal samples.  Other problems became
evident however, such as the presence of valleys in one dataset where there
was a peak in other dataset.  And some raw spectral signals were saturated 
(flat regions of high intensity).  Several mathematical efforts were made to
align and match signals and features.
<p>
The problem with much of the data was the presence of a maximum recorded raw
intensity (signal saturation).  Thus normalzing or correcting data can be a
problem in looking for features.  The raw data was searched uncorrected.  
A two-sample <i>t</i>-test was used to look for differences between cancer
and normal samples at every <i>m/z</i> value in Dataset 3, with emphasis on
seven <i>m/z</i> values reported on the author's website.  The most extreme
<i>t</i> values are huge, and the focus was at <i>m/z</i> 435.46, with
<i>t</i> = 22.3463.  238 of 253 samples were correctly classified using this
peak.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Severcan M, Haris PI, Severcan F
 </span>
 <span class="year">
(2004)
 </span>
 <span class="jname">
Anal. Biochem.
 </span>
 <span class="jvol">
332,
 </span>
 <span class="pages">
238-244.
 </span>
<p class="reftitl">
Using Artificially Generated Spectral Data To Improve Protein Secondary
Structure Prediction from Fourier Transform Infrared Spectra of Proteins
<p class="Para1">
A data set prepared by Lee et al. (Biochemistry 29: 9185, 1990)
<!-- ================================================================ -->
<p class="tags">

<p class="ref">
 <span class="authors">
Morelle W, Michalski J-C
 </span>
 <span class="year">
(2005)
 </span>
 <span class="jname">
Curr. Anal. Chem.
 </span>
 <span class="jvol">
1,
 </span>
 <span class="pages">
29-57.
 </span>
<p class="reftitl">
The Mass Spectrometric Analysis of Glycoproteins and Their Glycan Structures
<p class="Para1">
<b>REVIEW</b>.  Glycosylated molecules participate in many biological
processes, including coordination of immune functions, pathogens targeting
host tissues, protein regulation and interaction.  Glycosylation occurs
in all eukaryotes and is found in prokaryotes.  Proteins are glycosylated
at specific target sites.  N-glycosylation occurs at Asn residues with
a consensus of Asn-X-Ser/Thr, where X is any amino acid but Pro.  O-linked
modification is on Ser or Thr.  Cys and Lys are glycosylated in rare
instances.  Glycosylation is not always structually identical even for
the same protein.  Human RBC CD59 shows 100 different oligosaccharide
structures at a single glycosylation site, demonstrating an important feature
of glycosylation which is <i>microheterogeneity</i>.
<p>
There are some common features.  <i>N</i>-linked modification is usually by
an amide bond of <i>N</i>-acetylglucosamine (GlcNAc) to Asn.  A <b>core</b>
is then built up to be Man<sub>5-9</sub>GlcNAc<sub>2</sub>.  These glycans
have a <q>high mannose type</q> cor <q>oligomannose</q> core,
formed by trimming the larger core with &alpha;-glucosidase and -mannosidases.
Glycans can also contain <i>N</i>-acetyllactosamine (Gal&beta;1-3/4GlcNAc)
in the antennal region, making them <q>complex type.</q>  Mammals
may add antenna composed of addtion of &beta;-Gal and GlcNAc, with
tandem repeats of LacNAc.  The <q>hybrid type</q> has both mannose
and N-acetyllactosamine attached to trimannosyl chitobiose core.  Core and
antenna can be capped with fucose or sialyl groups by &alpha; linking,
in contrast to the use of &beta; linking for the backbone sugars.
<p>
O-linked sugars show more diversity, from monosaccharides to large sulfated
polysaccharides.  The connecting sugar is GlcNAc as in N-linked, although
other sugars have been reported.  Cytosolic and nuclear glycoproteins
show O-GlcNAc.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Ahnert N, Patton WF, Schulenberg B
 </span>
 <span class="year">
(2004)
 </span>
Presented at the Association of Biomolecular Resource Facilities Annual Meeting:
Portland, Oregon, February 28-March 2, 2004
<p class="reftitl">
Optimizing Conditions for Diluting, and Reusing SYPRO Ruby Protein Gel Stain
<p class="Para1">
Berggren et al (2000) introduced SYPRO Ruby which can detect as little as 1 ng
protein and still permits Edman sequencing and MS (Finehout and Lee 2003).
The formulation was later improved (Berggren et al 2002) for use with other 
stains like Pro-Q Emerald 300 glycoprotein and Pro-Q Diamond phosphoprotein stain.
Unfortunately it is rather expensive.  A more economical ruthenium-based
fluorescent stain (Berggren et al 2002, Rabilloud et al 2001) and dilution
of the stain (Krieg et al 2003).  Figures are presented which show the following:
<ul>
<li>Gels should be prefixed rather than unfixed.  50:10% MeOH:MeCOOH consistently
gives better results than unfixed gels.
<li>In a plot of fluorescence intensity vs. ng BSA, any kind of fixation 
produced the same results:  50:3% EtOH:MeCOOH, 10:7% MeOH:MeCOOH, and 50:10%
MeOH:MeCOOH, compared to no fixation.  Thus 10:7% MeOH:MeCOOH is the most 
economical
<li>A time course shows that fluorescence consistently improves with staining
time.  At 7 h, intensity is generally twice that of 3 h, and overnight is usually
twice that of 7 h.
<li>The stain was diluted 1/2, 1/3, 1/5, 1/10, and 1/20.  There were clear 
reductions in fluorescence intensity at higher amounts of protein, but the
reductions were less so with smaller amounts of protein.
<li>In looking at re-used stains, there was a clear reduction of fluorescence
intensity with each re-use, but when the volumes used were in greater volumes,
more fluorescence intensity was retained (see table)
 <table>
<tr><th>ng BSA: <th>62.5  <th>125   <th>250   <th>500    <th>1000
 <tr><th colspan="6">50 ml
 <tr><th>unused <td>1500  <td>2500  <td>3900  <td>6600   <td>11,000
 <tr><th>1X     <td>800   <td>1200  <td>2200  <td>3300   <td>6100
 <tr><th>2X     <td>700   <td>1100  <td>1800  <td>2800   <td>5000
 <tr><th colspan="6">100 ml
 <tr><th>unused <td>1300  <td>2200  <td>3600  <td>6000   <td>9500
 <tr><th>1X     <td>1000  <td>1700  <td>3000  <td>5300   <td>8500
 <tr><th>2X     <td>800   <td>1400  <td>2500  <td>4100   <td>6700
 </table>
Note that gels were preincubated in two changes of 50:10% MeOH:MeCOOH, for 30
min each.
</ul>

<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Maleknia SD, Downward KM
 </span>
 <span class="year">
(2005)
 </span>
 <span class="jname">
Anal. Chem.
 </span>
 <span class="jvol">
77,
 </span>
 <span class="pages">
111-119.
 </span>
<p class="reftitl">
Charge Ratio Method:  Approach for Deconvolution of Electrospray Mass Spectra
<p class="Para1">
ESI-MS produces multiply charged ions, and so deconvolution algorithms are
used to calculate charge states, then the mass.  The following equation
is fundamental to their determination:
<p style="text-align:center;" class="eqnserif">
<i>R<sub>z</sub></i> = (<i>M &plusmn; zm</i>)/<i>z</i> &nbsp;&nbsp; [1]
<p>
From the <i>m/z</i> value for any two ions (<i>R<sub>z</sub></i>)<sub>a</sub>
and (<i>R<sub>z</sub></i>)<sub>b</sub> within the spectrum and the
corresponding charge difference (<i>z</i><sub>b</sub> = <i>z</i><sub>a</sub>
+ integer) along with assumptions about the charge-carrying species <i>m</i>,
the equations constructed can be used to solve <i>M</i>.
In addition, entropy-based calculations are used first to predict a molecular
weight range and then hypothetically multiply charged species are fit to the
data.
<p>
The charge ratio analysis method (CRAM)  is the ratio of two multiply
charged ions, a and b, originating from the same compound derived from
eq. 1
<table class="eqnserif" style="text-align:center;">
<tr><td style="border-bottom:1px solid black;">
  (<i>R<sub>z</sub></i>)<sub>a</sub>
 <td rowspan="2"> =
<td style="border-bottom:1px solid black;"><i>z</i><sub>b</sub>(<i>M &plusmn;
	z</i><sub>a</sub><i>m</i>)
<td rowspan="2">&nbsp;&nbsp;&nbsp;[2]
<tr><td>(<i>R<sub>z</sub></i>)<sub>b</sub>
 <td><i>z</i><sub>a</sub>(<i>M &plusmn; z</i><sub>b</sub><i>m</i>)
</table>
<p>
When the mass of the species <i>M</i> is very large, as for proteins,
that is when <i>M</i> &gt;&gt; <i>z</i><sub>a</sub><i>m</i> or
<i>z</i><sub>b</sub><i>m</i>, then eq 2 simplifies to

<table class="eqnserif" style="text-align:center;">
<tr><td style="border-bottom:1px solid black;">
  (<i>R<sub>z</sub></i>)<sub>a</sub>
 <td rowspan="2"> =
<td style="border-bottom:1px solid black;"><i>z</i><sub>b</sub>
<td rowspan="2">&nbsp;&nbsp;&nbsp;[3]
<tr><td>(<i>R<sub>z</sub></i>)<sub>b</sub>
 <td><i>z</i><sub>a</sub>
</table>
<p>
A comparison of the ratio of <i>m/z</i> values (<i>R<sub>z</sub></i>)
to the unique ratio of any two integers identifies the charge state
values for each of the ions.  Once the charge states are identified,
the <i>R<sub>z</sub></i> values for two ions are multiplied by their
charge state values.
<p style="text-align:center;" class="eqnserif">
(<i>R<sub>z</sub></i>)<sub>a</sub> <i>z</i><sub>a</sub> =
		<i>M &plusmn; z</i><sub>a</sub><i>m</i> &nbsp;&nbsp; [4]<br>
(<i>R<sub>z</sub></i>)<sub>b</sub> <i>z</i><sub>b</sub> =
		<i>M &plusmn; z</i><sub>b</sub><i>m</i> &nbsp;&nbsp; [5]<br>
(<i>R<sub>z</sub></i>)<sub>a</sub><i>z</i> &minus;
   (<i>R<sub>z</sub></i>)<sub>b</sub><i>z</i>  =
		&plusmn;<i>m</i> (<i>z</i><sub>a</sub> &minus;
					<i>z</i><sub>b</sub>) &nbsp;&nbsp; [6]
<p>
When <i>z</i> and <i>m</i> are obtained, eq. 1 can be used to evaluate
<i>M</i>.
<p>
An example of lysozyme is used.  Its average mass is 14305.1438 Da.
A table lists the theoretically generated
(<i>R<sub>z</sub></i>)<i><sub>n</sub></i>) values for the lysozyme ions
with 10-13 charges formed from addition of protons, and sodium and
potassium ions.  For a ratio of 1301.47542/1431.52218 = 0.90915, this
is closest to 0.90909 obtained by dividing 10 by 11.  Thus the calculation
of these means that these species represent +10 and +11 ions.
<p>
Now eq. 6 is applied to find the difference in the mass:
<p style="text-align:center;">
  (1301.47542 &times; 11) &minus; (1431.52218 &times; 10) = 1.0078
<p>
This represents the difference for the mass of a proton, which this
must be the charge-carrying species that accounts for the difference
in the +10 and +11 species.
<p>
The errors associated with these determinations are calculated as:
<p class="eqnserif" style="text-align:center;">
| (<i>R<sub>z</sub></i>)<sub><i>n</i> + 1</sub>/(<i>R<sub>z</sub></i>)<sub><i>n</i></sub>
&minus; <i>z<sub>n</sub> / z</i><sub><i>n</i> + 1</sub> |
<p>
When the adducts are protons, the errors are 6 &times; 10<sup>&minus;5</sup>
(60 ppm) of their theoretical values;  when potassium, then
244 &times; 10<sup>&minus;5</sup> of the theoretical values.  This
follows from the assumption of <i>M</i> &gt;&gt; <i>z<sub>n</sub>m</i>.
As <i>z<sub>n</sub>m</i> increases with the adduct size however,
this value becomes more significant relative to <i>M</i>.
<p>
CRAM is not limited to calculations of adjacent/consecutive <i>m/z</i>
species.  It can calculate using any two <i>m/z</i> for which &Delta;<i>z</i> &gt; 1.
<p>
<b>Correlating Isotopic Compositions of the Charged Species Using CRAM</b>.
Another feature of CRAM is its ability to find within isotope clusters
of the charged species the match between those species representing
identical isotopic compositions with but the exception of the addition
of the charged species.  The process requires acquisition of high
resolution mass spectra to see accurate masses of the isotopes of
the multiply charged species.  A table is constructed with the
<i>m/z</i> values of each isotope, its relative intensity, the isotope
spacing (1/<i>z</i><sub>n</sub>), and the calculation of the error.
In addition the ratios of all isotopes in one cluster of a multiply
charged species and the isotopes in another cluster are determined.
Suppose that A and B represent one multiply charged species, and
A<sub><i>m</i></sub> and B<sub><i>n</i></sub> represent the <i>m</i>th
and <i>n</i>th isotope in their clusters (it could represent only isotopes
with <i>m/z</i> that meet a minimum threshold relative intensity).
The ratio A<sub><i>m</i></sub> / B<sub><i>n</i></sub> is determined
for all <i>m</i> and <i>n</i>.  If the ratio of A<sub>1</sub> / B<sub>3</sub>
= 0.92857 (i.e., the 1st isotope in the cluster of species A and the
3rd isotope in cluster B is that ratio value), then that corresponds
to quotient of the division of integers 13 / 14.  Thus the isotopes
A<sub>1</sub> and B<sub>3</sub> have identical isotopic composition.
It can also be seen that A<sub>2</sub> / B<sub>4</sub> have the same
ratio within experimental error, and are thus isotopically identical too,
as would be expected.
<!-- ================================================================ -->
<p class="tags">

<p class="ref">
 <span class="authors">
St&uuml;biger G, Marchetti M, Nagano M, Reichel C, Gmeiner G, Allmaier G
 </span>
 <span class="year">
(2005)
 </span>
 <span class="jname">
Rapid Commun. Mass Spectrom.
 </span>
 <span class="jvol">
19,
 </span>
 <span class="pages">
728-742.
 </span>
<p class="reftitl">
Characterisation of Intact Recombinant Human Erythropoietins Applied in
Doping By Means of Planar Gel Electrophoretic Techniques and Matrix-Assisted
Laser Desorption/Ionization Linear Time-of-Flight Mass Spectrometry
<p class="Para1">
Human erythropoietin (EPO) is produced in human adult kidney and coded
by the chromosome 7 into 193 amino acid long pre-protein with 27 aa signal
peptide.  The mature form is 165 aa (with Des-Arg, 166) long and has
N-glycosylation at Asn-24, -38, and -83, and O-glycosylation at Ser-26,
MW calculated to be 18240 Da.  Hypoxia leads to transcriptional induction,
as well as effects of hGH, ACTH, IL-3, angiotensin II, and catecholamines.
First recombinants in mammalian cells were reported in 1985, and FDA
approved drug in 1987.  Natural and derivative hEPO (NESP, pegylated EPO,
continuous erythropoiesis receptor activator [CERA]) are widely available
on market (Erypo, ESPO, Epogin, Procrit, NeoRecormon, Aranesp).
<p>
For this study Janssen-Cilag Erypo (2000 IU EOP-&alpha; injection, in 500
&micro;l), Roche NeoRecormon (500 IU EPO-&beta; in 300 &micro;l) and Amgen
Aranesp (10 &micro;g novel erythropoiesis stimulating protein in 400 &micro;l)
were studied.  1 IU is about 8.4 ng (~280 fmol).  Most chemicals are Sigma.
MALDI matrices are Fluka.  See M&amp;M for other suppliers.
<p>
The commercial products were de-salted on MicroSpin G-25 columns (Amersham
Biosciences) and analyzed on 10% bis-Tris-SDS-PAGE.  Detection limits were
400 and 80 ng on Coomassie- and silver-stained gels, respectively.  MW
of bands was estimated from log MW vs. Rm plot: EPO-&beta; has a lower
36 kDa with than EPO-&alpha; at 37 kDa.  These are 6-7 kDa higher than
seen on MALDI-MS, probably because of non-ideal mobility by failure of SDS
to bind to oligosaccharides.  Slab gel (planar) IEF was done with
electrotransfer to PVDF and immunostaining.  NESP was far more highly
acidic than EPO-&alpha; or -&beta;, probably because it is highly sialyted.
EPO-&alpha; shows 4-5 isoforms, and EPO-&beta; about 7 to 8.  Two EPO-&beta;
isoforms were quite basic.  NESP resolved 4 bands.  Detection limit
for immunostaining is about 20 pg using chemiluminescent detection
(strepavidin-biotin HRP + Covalight chemilum kit).  With increases in loaded
amount, other bands were detected.
<p>
The EPO products were de-sialyated (in 10 mU Roche neuraminidase per
&micro;g protein in 10 mM sodium bicarbonate pH 8.0, 18 h at 37&deg; rocked
at 350 rpm on thermomixer).  Bands of identical mobility are seen on SDS-PAGE
in enzyme-treated samples.  Deglycosylation (Roche PNGase F 1-3 U/&micro;g
proteins 18 h at 37&deg; in pH 8 product buffer) produces EPO-&alpha; running
at 19 kDa, the O-glycosylated product.  NESP deglycosylation (no
prior reduction alkylation) showed incomplete reaction with two bands at
28 and 19 kDa; the 28 kDa band is still N-glycosylated at one point.
With reductive-alkylation, the 28 kDa band disappears.
<p>
MALDI-MS at threshold energy shows an average MW of 30.2 &plusmn; 0.4 kDa
for EPO-&alpha; and -&beta; and 37.6 &plusmn; 0.9 kDa for NESP.
Measurements (<i>n</i> = 12-17) done with an external calibration of
[M]<sup>+</sup>  signals using sinapinic acid.  Weight can fluctuate
&plusmn; 1.3% for EPO-&alpha; and -&beta; and &plusmn;2.4% for NESP if
laser energy is beyond threshold.  Internal calibrants trypsin and
enolase provide a higher mass accuracy.
<p>
Purified proteins from the bottle were ZipTipped and spotted for MALDI-MS.
Spectra of NESP showed an [M]<sup>+</sup>
of <i>m/z</i> 36807.1 corrected by use of 23.3 kDa trypsin and 46.7 kDa
enolase internally.  EPO-&alpha; [M]<sup>+</sup> is <i>m/z</i> 29869.0.
<p>
Proteins were run on SDS-PAGE, extracted from
gel slices cut with scalpel, destained with 40% MeCN/60% 50 mM ammonium
bicarbonate, washed in water, crushed with glass rod, then extracted in
1:3:2 formic acid:water:isopropanol.  Gel particles were spun down
and supernatants evaporated.  Proteins are resuspended in 1% formic acid
and cleaned with ZipTips.  MALDI-MS spectra show an [M]<sup>+</sup> with
<i>m/z</i> 36092.3 and [M]<sup>2+</sup> of <i>m/z</i> 17984.8 for NESP, and
[M]<sup>+</sup> with <i>m/z</i> 28971.3 and [M]<sup>2+</sup> of <i>m/z</i>
14498.7 for EPO-&alpha;.  In some cases the weight differences are due
to sodium and/or potassium adduct formation despite extensive desalting.
The 6.7 kDa mass difference between NESP and EPO-&alpha; and -&beta; is
due to additional N-glycations.
<p>
Matrices such as sinapinic and ferulic acid, and the benzoic acids (DHB,
and super-DHB) did well in ionizing EPO products.  The trans-cinnamic
acids crystallized well:  sinapinic acid shows disperse distributed
rhombic crystal, while ferulic is more needle-like (from 0.5-1 mm long)
around rim.  Laser aims at crystals produce sustained spectra.  With
respect to S/N and peak resolution, no one matrix stood out from the others.
Metastable fragmentation is seen when there is an asymmetric peak tailing
that results from raising the laser energy especially on DHB matrices.
3-5 mg/ml matrix concentrations were good, and higher (&ge; 10 mg/ml) may
contribute to noise, drowning out sample signals in the fmol range.
At &le; 1 mg/ml, crystal formation is impaired with signal loss.
A DHB/CHCA combination in 5% formic acid/0.1% TFA has been used:  less
laser energy is required but there are multiple charged forms.
Calculation of the 1+ ion from multiple charged states shows a difference
from the direct measurement of a [M]<sup>+</sup> ion, but may be more
accurate.  Other matrices caffeic acid, 2,6-DHAP, 3-HPA and &beta;-carbolines
(harmaline, nor-harmane) failed because there were too many laser aims to
find a sweet spot.  In negative ion mode THAP with ammonium citrate produced
ionization; shown to work with sialyated glycoproteins and glycopeptides;
it worked in positive ion mode too, but the other matrices were far better.
<p>
C18 ZipTip binding capacity for rhEPO was shown to be &le; 1 &micro;g, and
similar to findings for C4 ZipTips.  MALDI-MS (on Shimadzu Biotech
Kratos Analytical AXIMA-LNR, 10 Hz firing, 3 ns pulse with 1.2 m linear
tube with delayed extraction, 20 kV acceleration, externally calibrated
with +1 and +2 ions of BSA) was done on stainless steel with different
target surface types.  CS4- and KA4-type plates gave the best results
whereas KA-2 gnd gold-plated were adequate down only to upper fmol range.
Use of a DropStop hydrophobic surface layer gave improved sensitivtiy in
the 50-100 fmol range.  Sinapinic and ferulic were comparable on different
plates, the latter better on stainless steel, the former on DropStop.
<p>
It is generally indicated that small crystal homogeneity works best
for MALDI spectrum quality and sensitivity, but results here show that
larger crystals with higher local concentrations of analytes improve
sensitivity.  Slower evaporation on DropStop plates provide better crystals;
slower crystallization may also help in salt exclusion.
<p>
The MALDI peaks of rhEPO are quite broad (R<sub>FWHM</sub> ~ 15).
With desialylation, there is a shift in the peak maxima down by 3-6 kDa,
which indicates a loss of 10-20 sialic acid residues (MW 291 Da).
When the sialic acids are removed with their attached alkali salts
there is a signficant resolution enhancement (R ~ 190).  Hence the
broad peak is the microheterogeneity caused by sialic acid presence
producing unresolved signal.  8-10 individual well-separated peaks
are then observed which represent permutations of O- and N-glycosylation.
The 370 Da mass difference between the peaks indicates differences in
N-acetyllactosamine (365 Da).  The various peaks could be assigned to
1-2 tri- and 2-3 tetraanntenary N-glycan forms of EPO-&alpha; and -&beta;.
Another series of peaks separated by 150 Da on the main peak might
be differences in fucosylated and non-fucosylated forms (fucose 146 Da),
expressed to lesser degree in CHO cells.
<p>
N-deglycosylation produced molecules with only O-glycosylation, but the
spectrum indicated variable degrees of sialylation, so the protein was
treated with both neuraminidase and PNGase F.  With both enzyme treatments,
the very predominant peak is [M]<sup>+</sup> at <i>m/z</i> 18519.9
with significant peak tailing.  A slight peak at baseline at
<i>m/z</i> 18157.6 represents the completely unglycosylated form.
When EPO-&alpha; is treated only with PNGase F, several peaks around
<i>m/z</i> 18500-1900 are seen:  <i>m/z</i> 19110.3 with two sialic acids
on the O-glycan, <i>m/z</i> 18817.1 with one sialic acid on the O-glycan,
<i>m/z</i> 18533.9 (not labeled), and the non-glycosylated one.
<p>
MALDI-MS spectrum is different with reductive alkylation then PNGase:
a two sialic acid O-glycan product is seen at <i>m/z</i> 19189.7.
The spectrum of NESP was quite complex, with PNGase F treatment still
showing mono-N-glycosylated envelope of peaks with a maximum of <i>m/z</i>
21959.0.  IEF-separated isoforms were processed to MALDI-MS
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Qian W-J, Monroe ME, Strittmatter EF, Jacobs JM, Kangas LJ, Petritis K,
Camp II DG, Smith RD
 </span>
 <span class="year">
(2005)
 </span>
 <span class="jname">
J. Proteome Res.
 </span>
 <span class="jvol">
4,
 </span>
 <span class="pages">
53-62.
 </span>
<p class="reftitl">
Probability-Based Evaluation of Peptide and Protein Identifications
from Tandem Mass Spectrometry and SEQUEST Analysis:  The Human Proteome
<p class="Para1">
Proteome samples from three human-derived samples were generated:
blood plasma, a human mammary epithelial cell (HMEC) line 184 A1L5,
and human hepatocyte cell line Huh-7.5.  Cell lysates are denatured
in 8 M urea in 50 mM NH<sub>4</sub>CO<sub>3</sub> pH 8.2 and reduced
in 10 mM DTT for 1 h 37&deg;, followed by alkylation in 40 mM iodoacetamide
90 min RT, shaking in the dark.  Samples are diluted 8-fold in 50 mM
NH<sub>4</sub>CO<sub>3</sub>, and digested in Promega porcine trypsin
with 1:50 (w/w) trypsin:protein.  Samples are then boiled 5 min and placed
on ice.  In addition a tryptic peptide mix of 17 pure proteins is produced:
proteins are BSA, bovine casein (contains 4 caseins), ovalbumin, horse
myoglobin, bovine lactoglobulin, bovine cardiac anhydrase, bovine Hb,
chicken lysozyme, bovine ubiquitin, bovine catalase, rabbit G3PDH, and
bovine ribonuclease 1-A.  For offline SCX, peptides are reconstituted
in 1.0 ml of 10 mM HCOONH<sub>4</sub>, 25% MeCN pH 3.0 (solvent A)
and injected on
Polysulfoethyl A 200 in 4.6 mm column at 1 ml/min.  Solvent B is
500 mM HCOONH<sub>4</sub> with 25% MeCN pH 6.8.  The gradient program
is 0-50% over 40 min and then 50-100% for 10 min.  35-70 fractions are
taken and lyophilized.  The 17-protein peptide mix was analyzed 12 times.
RPLC-MS/MS was then done using an LCQ Deca XP ion-trap MS (Thermo Finnigan).
The capillary column is slurry-packed 5 &micro;m Jupiter C<sub>18</sub>
(Phenomenex) into 150 &micro;m i.d. (360 &micro;m o.d.) &times; 65 cm
fused silica (Polymicro Tech).  Mobile phase is 0.2% acetic acid and
0.05% TFA in water (solvent A) and 0.1% TFA in 90% MeCN (solvent B).
The SCX fractions are dissolved in 50 &micro;l 25 mM HCOONH<sub>4</sub>
pH 8.0, and 10 &micro;l injected.  Gradient is exponential, increasing
to 70% B over 150 min.  MS is operated in data-dependent MS/MS mode
(<i>m/z</i> 400-2000) with full MS scan followed by three MS/MS scans
and normalized collision of 45%.  MS/MS  spectra are searched against
human IPI database and the reversed database with SEQUEST algorithm.
Searches included the fixed modification of carbamidoyl-cysteine.
Finally the count of unique peptide identifications from the reversed database
(<i>N</i><sub>N</sub>) was divided by the count from the normal database
(<i>N</i><sub>R</sub>) to get the false positive rate
(=<i>N</i><sub>R</sub>/<i>N</i><sub>N</sub>).
<p>
The number of MS/MS spectra generated from LC-MS/MS analyses
of the SCX fractions (run in duplicate) is shown in the table:
<table style="text-align:center;">
<tr><th>Proteome       <th>LC-MS/MS analyses <th>MS/MS spectra
<tr><td>human plasma   <td>140               <td>470,000
<tr><td>HMEC lysate    <td>69                <td>230,000
<tr><td>Huh-7.5 lysate <td>74                <td>250,000
</table>
<p>
Spectral data are submitted to normal and reversed databases by SEQUEST.
<i>Xcorr</i> and &Delta;<i>Cn</i> are used to examine correlation between
peptide scores and false positive rates. <i>Xcorr</i> is cross-correlation
score between actual and predicted MS/MS spectrum, and &Delta;<i>Cn</i>
is relative difference between correlation score for top-matching peptide
and second-best match.  Usefulness of a reversed database for yeast proteomes
has been discussed elsewhere.  The count of unique peptide identifications
from a reversed database (<i>N</i><sub>R</sub>) that pass the filtering
criteria is an estimate of the number of false positives.  Since the count
of unique IDs from the normal database is the count of true positives
plus the number of false positives, and the false positive rate is defined
as the proportion of false positives among all positives (true and false),
then the rate = FP/(TP + FP), <i>N</i><sub>R</sub> = FP, and
<i>N</i><sub>N</sub> = TP + FP;  thus the rate =
<i>N</i><sub>R</sub>/<i>N</i><sub>N</sub>.  As verification of this
model, searching including the 17 known protein peptide mix with nonhuman
sequences.
<p>
Table 1 shows a list of the count of true and false positives from the
normal and reversed databases.  The data shows that the count
of FP and the FP rate agree between the two dbases.  Most false peptide
assignments result from random matching.  This means searching against
the reversed dbase gives a reliable estimation of the FP count.
<p>
The FP rate was evaluated for peptide IDs from the three proteomes
based on several different parameters include peptide ion charge states,
tryptic cleavage states (full vs. partial), <i>Xcorr</i>, and
&Delta;<i>Cn</i>.  The estimated proportion of FP was plotted against
&Delta;<i>Cn</i> for different <i>Xcorr</i> scores greater than a certain value
(1.5, 1.8, 2.1, 2.4) both for human plasma and HMEC lysates with
doubly charged fully tryptic peptides selected.
The FP rate was then normalized between the two proteomes and
plotted against &Delta;<i>Cn</i>.  The plots had the same pattern:
the FP rate decreases as the cutoff value of &Delta;<i>Cn</i> increases.
A polynomial fit revealed <i>y</i> = 33.183<i>x</i><sup>2</sup>
&minus;11.931<i>x</i> + 1.1036 with residual (R<sup>2</sup>) = 0.9943, where
<i>y</i> is the normalized FP rate and <i>x</i> is &Delta;<i>Cn</i>.
<!-- ================================================================ -->
<p class="tags">

<p class="ref">
 <span class="authors">
Delany J, Floyd ZE, Zvonic S, Smith A, Gravois A, Reiners E, Wu X,
Kilroy G, Lefevre, Gimble JM
 </span>
 <span class="year">
(2005)
 </span>
 <span class="jname">
FASEB J.
 </span>
 <span class="jvol">
&nbsp;
 </span>
 <span class="pages">
<a href="http://dx.doi.org/10.1096/fj.03-1100fje">DOI:10.1096/fj.03-1100fje</a>
 </span>
<p class="reftitl">
Proteomic Analysis of Primary Cultures of Human Adipose-Derived Stem
Cells:  Modulation by Adipogenesis
<p class="Para1">
Notes are located <a href="file://c:/Documents%20and%20Settings/Proteomist/My%20Documents/AUBI/Projects/Stem%20Cell/StemCellLit.html#delany">here</a>
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
M&ouml;hring T, Kellmann M, J&uuml;rgens M, Schrader M
 </span>
 <span class="year">
(2005)
 </span>
 <span class="jname">
J. Mass Spectrom.
 </span>
 <span class="jvol">
40,
 </span>
 <span class="pages">
214-226.
 </span>
<p class="reftitl">
Top-Down Identification of Endogenous Peptides Up to 9 kDa in Cerebrospinal
Fluid and Brain Tissue by Nanoelectrospray Quadrupole Time-of-Flight
Tandem Mass Spectrometry
<p class="abstract">
<b>ABSTRACT</b>.
Recent work on protein and peptide biomarker patterns revealed the difficulties in identifying their
molecular components, which is indispensable for validation of the biological context. Cerebrospinal fluid
and brain tissue are used as sources to discover new biomarkers, e.g. for neurodegenerative diseases.
Many of these biomarker candidates are peptides with amolecular mass of <10 kDa. Their identification is
favourably achieved with a top-down approach, because this requires less purification and an enzymatic
cleavage will often not yield enough specific fragments for successful database searches. Here, we describe
an approach using quadrupole time-of-flight mass spectrometry (TOFMS) as a highly efficient mass
spectrometric purification and identification tool after off-line decomplexation of biological samples by
liquid chromatography.After initial peptidomic screeningwith matrix-assisted laser desorption/ionization
(MALDI) TOFMS, the elution behaviour in chromatography and the exact molecular mass were used to
locate the same signals in nanoelectrospray measurements.Most of the peaks detected inMALDI-TOFMS
could be retrieved in nanoelectrospray quadrupole TOFMS. Suitable collision energies for informative
fragment spectra were investigated for different parent ions, charge states and molecular masses. After
collision-induced dissociation, the resulting fragmentation data of multiply charged ions can become
much more complicated than those derived from tryptic peptide digests. However, the mass accuracy
and resolution of quadrupole TOF instruments results in high-quality data suitable for determining
peptide sequences. The protein precursor, proteolytic processing and post-translational modifications
were identified by automated database searches. This is demonstrated by the exemplary identifications
of thymosin beta-4 (5.0 kDa) and NPY (4.3 kDa) from rat hypothalamic tissue and ubiquitin (8.6 kDa)
from human cerebrospinal fluid. The high data quality should also allow for de novo identification. This
methodology is generally applicable for peptides up to amolecular mass of about 10 kDa from body fluids,
tissues or other biological sources.
<!-- ================================================================ -->
<p class="tags">

<p class="ref">
 <span class="authors">
Zehl M, Allmaier G
 </span>
 <span class="year">
(2005)
 </span>
 <span class="jname">
Anal. Chem.
 </span>
 <span class="jvol">
77,
 </span>
 <span class="pages">
103-110.
 </span>
<p class="reftitl">
Instrumental Parameters in the MALDI-TOF Mass Spectrometric Analysis of
Quaternary Protein Structures
<p class="Para1">
Chicken avidin is homotetramer strong in association that has alkaline pI.
It has a high affinity for biotin.  It has 128 amino acids with estimated
calculated weight of 14,342.14 Da.  Iti is glycosylated on Asn-17, but
glycan not really analyzed.  MALDI glycoprotein analysis will resolve
11 masses ranging from 15,396.66 to 15965.08 Da, with 41 and 162 Da
differences indicating differenes in hexoses and N-acetylhexosamine units.
The tetramer is stable in wide range of pH and temperature:  association
stablized by hydrophobic contacts, hydrogen bonds, salt bridges.
<p>
MALDI was a AXIMA-LNR and AXIMA-CFR ToF from Shimadzu Biotech-Kratos Analytical
with 1 GHz TDC, using 20 kV accelerating with delayed extraction time set.
Pulse width of laser is 4 ns with 150 &micro;m dia. spot aim.
<p>
Spectra were acquired by spotting with 2,6-DHAP in 25:75 MeCN:20 mM
ammonium acetate.  The tetramer had the greater intensity compared to the
monomer.  Also 2+ and cluster ions were detected.  Because of carbohydrate
microheterogeneity, a broad peak width was seen.  Average mass of the assembly
was 63914 Da, which is higher than the expected 63292 Da with the
isoform of (Hex<sub>4</sub>HexNAc)<sub>2</sub>(Hex<sub>7</sub>HexNAc<sub>2</sub>)<sub>2</sub>
accounted for.  Possibly from the specific and nonspecific association
with low neutral MW substances this shift is caused.  Peak broadening
and mass shift is seen in ESI-MS spectra of assemblies, particularly if
desolvation is incomplete.
<p>
When sample was prepared in 25 mg/ml 2,6-DHAP in THF and completely
different spectrum with a series of cluster ions up to 200,000, all 1+
ions representing subunit associations that are not natural.
<p>
Equine liver ADH is a homodimer, each subunit with two Zn<sup>2+</sup> atoms.
Subunit is 374 amino acids with N-acetylserine N-terminus, and theoretical
mass of 39846.4 Da.  MALDI mass measurement average was 39861.8 which
0.04% off.  The homodimer was more intense than the monomer, with the benefit
of ammonium acetate improving desorption and ionization.
<p>
Bovine liver catalase was tried with both TFA-containing matrix (2,6-DHAP
saturated in 25% MeCN in 0.1% TFA) and only the monomer was seen.
When ammonium acetate was used instead the aposubunit peak and holosubunit
(polypeptide with heme b) is seen.  Also seen is the intact subunit assembly.
<p>
Repeated irradiation did not reveal a <q>first shot</q> phenomenon since
some residual peak of the oligomer was always seen.  To determine effect
of laser pulse energy, a single spectrum was made from 50 unselected
laser shots with both continuous and delayed extraction tried.  An optical
density filter on the laser was adjusted from 90 to 170 by steps of 10
(arbitrary units) corresponding to exponential energy increases from
6.3 to 60 &micro;J.  Peak intensity correlated with input laser energy
with saturation reached at about 150 on laser wheel (~34 &micro;J).
At laser energy threshold virtually on the subunit assembly was detected
while increases in laser energy result in monomeric subunits and cluster
ions being seen.  Cluster ion formation with higher laser energy is likely.
These results are seen with continuous or delayed extraction.
<p>
As for cluster ions, low laser energy pulses show formation of clusters
with multiple intact assemblies whereas higher energy pulses for clusters
made of multiple monomeric subunits.  The 2M<sup>+</sup> assemblies reach
maximum intensity ratios at lower energy pulses compared to clusters ion
of nonassemblied subunits (3suunit<sup>+</sup>) likely because of
high energy pulse disrupting association of the intact assembly while
cluster ion formation still occurs.  The reason that matrix/solvent did
not cause preformed clusters is that these were not present at lower
energy pulses.
<p>
Delayed extraction time allows for enhanced resolution by allowing
internal energies of ions to even out and reducing possibilities of
collisions that cause change in resolution.  However, the time delay
permits in-source decay as well.  Delay times of 416, 588, and 832 ns
were applied.  This variation had little effect on cluster ion or
intact assembly peak intensity compared to laser energy as the variable.
While no trend was seen with intact assemblies, cluster ion intensity
decreased with increasing delay time.  These data indicate that laser-induced
dissocation occurs prior to any effect of delayed extraction time.
<p>
Use of reflectron mode is not warranted with high mass ions.  No resolution
enhancement is found and sensitivity and S/N are decreased.  Nonetheless
reflectron can be used when metastable fragmentation of the ion is suspected.
Spectra of avidin show no change in mass and thus no in-source instability
of subunit association.
<!-- ================================================================ -->
<p class="tags">

<p class="ref">
 <span class="authors">
Churchward MA, Butt RH, Lang JC, Hsu KK, Coorssen JR
 </span>
 <span class="year">
(2005)
 </span>
 <span class="jname">
Proteome Sci.
 </span>
 <span class="jvol">
3,
 </span>
 <span class="pages">
5.
 </span>
<p class="reftitl">
Enhanced Detergent Extraction for Analysis of Membrane Proteomes by
Two-Dimensional Gel Electrophoresis
<p class="Para1">
RBC extracts can be screened using 1D SDS-PAGE to determine the effects
of extraction reagents, the effects being compared to controlled extractions
with detergents like CHAPS and SDS.  Band III was seen in SDS extracts
at ~110 kDa, but not with CHAPS; so was a band at 28 kDa.  Detergents
showing improvements in the ability to extract more proteins than CHAPS
included L-lauryl lysophosphatidylcholine (LPC), the N-methylglucanamide
detergents MEGA 8, 9, and 10, and the sulfobetaine-based detergents
ASB-14 and SB 3-10.  Natural source lipids lysophosphatidylethanolamine (LPE),
lysophosphatidylglycerol (LPG), LPC from egg, lysophosphatidylserine (LPS)
from bovine brain, and cardiolipin from bovine heart were generally of
limited general usefulness, largely because they are charged at neutral
pH and would interfere with IEF and difficult for the purposes here.
<p>
RBC membranes were extracted in a 2DGE buffer containing 4% CHAPS or
a 1-2% total detergent (CHAPS is one of the few detergents soluble at
such a high concentration).  Extraction with LPC and MEGA 10 showed
improvement in the maps compared to CHAPS.   The maps with SB 3-10 were
similar to the CHAPS, but this detergent is not too soluble in high urea.
Use of ASB-14 did not improve the results found with CHAPS or SB 3-10,
contrary to previous reports.
<p>
Experiments were now tried with 3% CHAPS and 1% of another detergent,
being either LPC or MEGA 10, and using membranes of RBCs, mouse brains,
or mouse liver.  With CHAPS and LPC, spot densities were improved
and there was some reduced horizontal streaking in the intermediate
MW range.  Some areas of the gel clearly revealed more proteins.
Band III was a prominent spot, and there wasa 2.2-fold increase in spot
density for a string of spots compared to the CHAPS gel.  Three more
spots are seen with 3% CHAPS:1% LPC.  Maps of RBC membranes extracted with
3% CHAPS:1% MEGA 10 were equivalent with 3% CHAPS:1% LPC and 4% CHAPS.
The combination of 3% CHAPS:0.5% LPC:0.5% MEGA 10 did not really improve
upon 3:1 CHAPS:LPC, although it was still an improvement over CHAPS alone,
with a density increase of 1.7-fold.  LPC addition alone enhances recovery
and resolution on 2D gels.
<p>
These experiments were only done on RBC membranes, and so it was necessary
to test perhaps more complex tissue types.  The four extraction conditions
(4% CHAPS, 3% CHAPS:1% LPC, 3% CHAPS:1% MEGA 10, 3% CHAPS:0.5% LPC:0.5% MEGA 10)
were tried.  All results were similar to those found for RBC membranes.
Spot number, density and resolution was better with CHAPS:LPC compared to
CHAPS alone.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Wu L, van Winden WA, van Gulik WM, Heijnen JJ
 </span>
 <span class="year">
(2005)
 </span>
 <span class="jname">
Metab. Engin.
 </span>
 <span class="jvol">
7,
 </span>
 <span class="pages">
302-310.
 </span>
<p class="reftitl">
Application of Metabolome Data in Functional Genomics:  A Conceptual Strategy
<p class="abstract">
<b>REVIEW</b>.
<p>
<b>ABSTRACT</b>.
A gene with yet unknown physiological function can be studied by
changing its expression level followed by analysis of the
resulting phenotype. This type of functional genomics study can be
complicated by the occurrence of silent mutations, the
phenotypes of which are not easily observable in terms of metabolic
fluxes (e.g., the growth rate). Nevertheless, genetic alteration
may give rise to significant yet complicated changes in the metabolome.
We propose here a conceptual functional genomics strategy
based on microbial metabolome data, which identifies changes in
in vivo enzyme activities in the mutants. These predicted changes
are used to formulate hypotheses to infer unknown gene functions.
The required metabolome data can be obtained solely from highthroughput
mass spectrometry analysis, which provides the following in vivo
information: (1) the metabolite concentrations in the
reference and the mutant strain; (2) the metabolic fluxes in both
strains and (3) the enzyme kinetic parameters of the reference strain.
We demonstrate in silico that changes in enzyme activities can be
accurately predicted by this approach, even in silent mutants.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Ram RJ, VerBerkmoes NC, Thelen MP, Tyson GW, Baker BJ, Blake RC 2nd,
Shah M, Hettich RL, Banfield JF
 </span>
 <span class="year">
(2005)
 </span>
 <span class="jname">
Science
 </span>
 <span class="jvol">
308,
 </span>
 <span class="pages">
1915-1920.
 </span>
<p class="reftitl">
Community Proteomics of a Natural Microbial Biofilm
<p class="abstract">
<b>ABSTRACT</b>.
Using genomic and mass spectrometrybased proteomic methods, we evaluated
gene expression, identified key activities, and examined partitioning of
metabolic functions in a natural acid mine drainage (AMD) microbial biofilm
community. We detected 2033 proteins from the five most abundant species in
the biofilm, including 48% of the predicted proteins from the dominant biofilm
organism, Leptospirillum group II. Proteins involved in protein refolding and
response to oxidative stress appeared to be highly expressed, which suggests
that damage to biomolecules is a key challenge for survival. We validated and
estimated the relative abundance and cellular localization of 357 unique and
215 conserved novel proteins and determined that one abundant novel protein
is a cytochrome central to iron oxidation and AMD formation.
<!-- ================================================================ -->
<p class="tags">

<p class="ref">
 <span class="authors">
Hu S, Xie Y, Ramachandran P, Ogorzalek, Loo RR, Li Y, Loo JA, Wong DT
 </span>
 <span class="year">
(2005)
 </span>
 <span class="jname">
Proteomics
 </span>
 <span class="jvol">
5,
 </span>
 <span class="pages">
1714-1728.
 </span>
<p class="reftitl">
Large-Scale Identification of Proteins in Human Salivary Proteome by
Liquid Chromatography/Mass Spectrometry and Two-Dimensional Gel
Electrophoresis-Mass Spectrometry
<p class="Para1">
The human salivary proteomes has a mix of proteins and peptides that maintain
oral cavity homeostasis.  About 80 such proteins have been identified.
This compares to about 1000 proteins in plasma (human).  A more global
analysis was sought by taking whole saliva from healthy, non-smoking
male in morning 2 h AFTER eating and rinsing water with mouth.
1 &micro;l/ml saliva of Sigma inhibitor cocktail was used and sodium
vanadate added to 1 mM.  Saliva having a 5 ml volume was obtained
and and centrifuged at 1300 <i>g</i> for 5 min, then centrifuged again
at 14,000 <i>g</i> for 15 min at 4&deg;.  Bradford showed a protein
concentration of 350 &micro;g/ml.  Samples are divided into 1 ml aliquots
and stored at &minus;80&deg;.
<p>
<b>LC-MS/MS</b>.  Samples were fractionated using centrifugal ultrafilters:  they were first
concentrated on YM-10.  YM-10 retentate and filtrate were then
concentrated on YM-3.  Fraction 1 represents filtrate from the YM-10f/YM-3f
where r=retentate and f=filtrate and the serial order of filtration is
shown.  Fraction 2 is the YM-10f/YM-3r.  Fraction 3 is YM10r.	For 0.5
ml saliva, the final volumes are 50 &micro;l for the retentates (fracs 2
and 3) and 400 &micro;l for the one filtrate (frac 1).  200 &micro;l
of f1 was dried in vacuum centrifuge then stored at &minus;80&deg;.
The other 200 &micro;l was given 40 &micro;l MeCN and 50 &micro;l
trypsin and incubated 37&deg; overnight.  f2 and f3 are treated with
10 mM DTT 30 min, then 50 mM iodoacetamide 30 min in dark.  10 &micro;l
MeCN and 50 &micro;l 20 &micro;g/ml trypsin added for overnight digestion
at 37&deg;.  Samples are dried after digestion and stored &minus;80&deg;.
LC-MS/MS was done with nano-LC system (LC Packings) coupled to Protana
electrospray interface and QSTAR XL (ABi) quadrupole ToF MS.  Samples are
passed to PepMap C18 5 &micro;m (300 &micro;m &times; 1 mm) for 5 min
in 3% MeCN + 0.1% formic acid + 0.01% TFA.  Samples are then injected
on to PepMap C18 5 &micro;m (75 &micro;m &times; 150 mm) for separation
at a flow of 180 nl/min.  A program starts with 5% solvent B to 60% B
in 55 min, then to 95% B in 0.1 min, then dropped to 90% B for 15 min,
and then prepared 15 min for next run (solvent A = 5% MeCN + 0.1% FA +
0.01% TFA, solvent B = 95% MeCN + 0.1% FA + 0.01% TFA).  A PicoTip
8 &micro;m tip was used for ESI at 2000 V.  Resolution is 5000-1000.
Dynamic exclusion reduces repeat peptide analyses.  Identification was
through MASCOT and Pro ID (ABi).
<p>
<b>2DGE and MALDI-MS</b>.  150 &micro;l whole saliva was treated with
10% TCA/90% acetone/20 mM DTT at &minus;20&deg; for 2.5 h.  After
spinning 14 krpm for 15 min, pellet was washed in cold acetone with
20 mM DTT and then resuspended in rehydration buffer.  IEF was done on
11 cm pH 3-10NL IPG strips (Bio-Rad) and the 2nd dim was 12.5% precast
using the Dodeca.  Gels were stained with SYPRO Ruby.  After imaging and
analysis, spots were cut using spot cutter and then depositing in 96-well
plate; digesting was on MassPREP and then analysis on M@LDI-LR using
CHCA matrix.  Additionally an ABi nano-LC ESI Q-ToF was used in MS/MS mode
to get sequencing info.  Data was searched against human database via
MASCOT and Spectrum Mill (Agilent).
<p>
Fraction 1 (&lt; 3000 Da by ultrafiltration) was run on LC-MS/MS, half
undigested, and the other half on another run with trypsin digestion.
Fraction 2 (between 3000 and 10000 Da) and Fraction 3 (&gt; 10000 Da) were
digested with trypsin.  Each analysis was done 3 times.  The LC run
is 90 minutes long, although most peptides elute within 50 min, with
MS/MS analyses between 10-50 min.  600 MS/MS spectra were produced in
Fraction 1 data alone, and 800 and 1700 for fractions 2 and 3, respectively.
For every Fraction 1 sequence produced, there was a database match.
For Frac 2 and 3, &gt; 10% of spectra had no peptide:  could result
from posttranslational modifications (glycosylation).  Of the 3100 MS/MS,
about 600 were reliable for database searching.
<p>
One example is of a Fraction 1 precursor +2 peptide with <i>m/z</i>
659.85 (note mass accuracy is 19 ppm).  The MS/MS spectrum produced
the range of <i>b</i><sub>m</sub> and <i>y</i><sub>n</sub> fragment
ions to identify a proline-rich peptide GPGFVPPPPPPPY, derived from
salivary proline-rich peptide P-B.  Another MS/MS of the <i>m/z</i>
593.36 precursor (2+) produced sequence GLSTESILPR, originating from
lacrimal lipocalin, a fairly common salivary protein.  Totaled, 79
proteins were identfied from Fraction 1 (only 26 could be found with the
trypsin digests however, whether enzymatic digestion was a factor in the
search or not).  With Fraction 2, 19 proteins were identifed, and
with Fraction 3, 205 proteins.  When removing redundant assignments,
a total of 266 proteins were identified, with about 2.0-2.5 peptides
used per protein.  At least 46% of the proteins were matched with 2 or
more peptides.  Insofar as reproducibility, the sample was analyzed
on three different days.  With the triplicate studies, 65% of the proteins
were identified in all three studies.
<p>
2DGE was done with SYPRO Ruby staining, revealing 300 spots total.
Of these 105 spots were actually excised from the gel, destained and then
digested in situ.  Fragments were extracted from gel piece and then
run on either MALDI-MS or LC-MS/MS.  Spot 43 produces several peptides
in the <i>m/z</i> 800-4000 range (resolution = 14,000, mass accuracy
~30 ppm).  24 spectral peaks were matched with 51% coverage to
salivary amylase, confirmed by LC-MS/MS of this spot.  64 distinct
proteins were identified (more than reported previously using this
approach for the salivary proteome).  21 of these proteins (33% of the
MALDI total) had already been identified by the shotgun approach.
Hence perhaps both should be used when characterizing the proteome.
<!-- ================================================================ -->
<p class="tags">

<p class="ref">
 <span class="authors">
Campostrini N, Areces LB, Rappsilber J, Pietrogrande MC, Dondi F,
Pastorino F, Ponzoni M, Righetti PG
 </span>
 <span class="year">
(2005)
 </span>
 <span class="jname">
Proteomics
 </span>
 <span class="jvol">
5,
 </span>
 <span class="pages">
2385-2395.
 </span>
<p class="reftitl">
Spot Overlapping in Two-Dimensional Maps:  A Serious Problem Ignored for
Much Too Long
<p class="abstract">
<p class="Para1">
In using 18 &times; 20 cm 2D gels loaded with minute amounts of protein from
biopsies, we observed 264 spots instead of the usual 750  spots when loading
standard amounts.  This would be a minimal number with colloidal Coomassie
stained gels as opposed to silver stained.  Of the 264 spots, 85 were
found to be up- or down-regulated in the tumor tissue.  These were eluted
for LC-MS/MS and 74 of them could be identified.  52 of the 74 were found
to be single proteins, 14 had two polypeptides, 6 had three, 1 had 4, and
another had 5 peptides.
<p>
To estimate the degree of overlapping of spots, there is a statistical
procedure that can be applied.  The 2D gel map is divided into a proper
number of strips (4) so that each strip contained 20 single components.
For each 1-D separation, the statistical procedure was applied to estimate
the count of proteins present in each strip and the sum yielded the total
number in the map.  In a strip, <i>p</i> is the total number of detectable
spots, <i>m</i> is the total number of single components (must by &ge; <i>p</i>).
For four strips, the <i>p</i> values are 21, 18, 19, and 16, with
corresponding <i>m</i> values of 28, 21, 31, and 23, for a total of
74 spots containing a total of 103 (single) components.  The degree of
overlapping (&gamma;) is <i>p/m</i> = 71%.
<!-- ================================================================ -->
<p class="tags">

<p class="ref">
 <span class="authors">
Bruschi M, Seppi C, Arena S, Musante L, Santucci L, Balduini C, Scaloni A,
Lanciotti M, Righetti PG, Candiano G
 </span>
 <span class="year">
(2005)
 </span>
 <span class="jname">
J. Proteome Res.
 </span>
 <span class="jvol">
4,
 </span>
 <span class="pages">
1304-1309.
 </span>
<p class="reftitl">
Proteomic Analysis of Erythrocyte Membranes by Soft Immobiline Gels Combined
with Differential Protein Extraction
<p class="abstract">
<b>ABSTRACT</b>.
Improvements in proteome analysis of erythrocyte membrane proteins by
two-dimensional electrophoresis are here reported. In particular,
a differential extraction procedure was set up allowing separation
of integral membrane proteins from peripheral species. Moreover,
the use of dilute Immobiline gels (down to as low as 3% T matrix)
permitted a better penetration and transfer inside the gel of
proteins with large M(r). These protocol modifications,
combined with sample delipidation and alkylation prior
to electrophoresis, which prevented generation of homo-
and hetero-oligomers following disulfide scrambling phenomena,
allowed the display of more than 500 spots in the pI/M(r) plane.
Among those, noteworthy was the presence of high levels of
filamentous proteins, such as alpha-spectrin and ankyrins,
or integral membrane proteins, such as band 3, band 4.1 and 4.2,
not displayed or barely present in other maps exploiting
immobilized pH gradients in the first dimension. Accordingly,
our results show that this 2D mapping technique is a valuable
tool in exploring pathologies related to genetic defects
associated to membrane proteins.
<!-- ================================================================ -->
<p class="tags">

<p class="ref">
 <span class="authors">
Heremans K
 </span>
 <span class="year">
(2005)
 </span>
 <span class="jname">
Braz. J. Med. Biol. Res.
 </span>
 <span class="jvol">
38,
 </span>
 <span class="pages">
1157-1165.
 </span>
<p class="reftitl">
Protein Dynamics: Hydration and Cavities
<p class="Para1">
<b>REVIEW</b>.  Cell lips, proteins and nucleic acids are sensitive to
high temperature.  Hyperthermophile bacteria adapt using ether lipids
instead of more hydrolysis-sensitive ester lipids in mesophiles.
There is a correlation between the unfolding of proteins and the
inactivation (death) kinetics of microorganisms, indicating that
proteins are the major targets in temperature-sensitive loss of life.
Chaperones are proteins induced by heat found in hypertheromphile crude
cell extracts whose function is to assist in folding during protein
synthesis.
<p>
A new archeal species called Strain 121 is found to grow at 121&deg;C.
Survival at 130&deg;C is for 2 h and 121&deg;C for 1 h.
<p>
Microbial cells can bear a max pressure of 100 MPa, although motility,
transport, cell division, cell growth, DNA synthesis, transcription,
translation are all affected at lower pressures.  Much depends on the
state of the organism.  Many have the ability to withstand high temp
and pressure by going into a dehydrated or a sporulated state.  In that
way, pressures beyond 600 MPa and even 1 GPa have no effect, and organisms
can survive from &minus;253&deg;C to 151&deg;C.
<p>
Bridgman observed that a pressure of several hundred MPa will transform
an egg white from raw to cooked look.  It is basically an aqueous solution
of protein, and pressure-induced coagulation becomes easier at low
temperatures.  Other studies have shown that during hard boiling of
an egg, the yolk solidifies at a higher temperature than the white,
while the reverse is for pressure when the white solidifies after the yolk.
<p>
A P vs. T (pressure vs. temperature) diagram has been developed for the
generalized protein.  An ellptical curve shows the phase transitions,
the curve resulting from a Taylor expansion of the free energy difference
between unfolded and native state;  the second order terms make a
substantial contribution.  The volume change is dependent on both
pressure and temperature (for gases, usually just pressure):
<p class="eqnserif">
&Delta;V(p,T) = &Delta;V&deg; + &Delta;&alpha;(T&minus;T<sub>0</sub>)
&minus;&beta;(p&minus;p<sub>0</sub>)
<p>
&Delta;V&deg; refers to reference conditions.  &Delta;&beta; is a
compressibility factor difference, and &Delta;&alpha; is a difference
of the thermal expansion factor.  The correlation between enthalpy and
volume change of the unfolding transition accounts for the elliptical
phase diagram, with the major axes being the line where &Delta;H = 0
and &Delta;V = 0, and regions of the diagram involving &Delta;H either
&gt;0 or &lt;0, and the same for &Delta;V.  The diagram shows that
pressure stabilizes the protein against thermal unfolding.  In many
cases the thermal denaturing transition temperature is raised with
increased pressure.
<p>
Fluorescence spectroscopy indicates that heat-induced folding may arise
from an exposure of protein core to water, while pressure-induced unfolding
occurs from pressure-mediate penentration of water into protein matrix
and then chain-chain interaction disruption.
<p>
Studies of water-soluble synthetic polymers with the main chain interacting
hydrophobically and the side chains hydrophilically show that at low T and
P, polymers are random coil (homogeneous one-phase).  There is a
formation of 2nd phase at high T or P.  Polymers show aggregation in the
2nd phase, reduced by detergents, but which does not affect any change
in the transition T or P points.
Phase transition states monitored by IR also show similar results for
gelatinazation of starch.
<p>
The partial molar volume of a protein is probably the sum of the partial
molar volumes of the atoms, cavities, and hydration sphere (Kauzmann).
Volume of atoms is independent of pressure and temp, but not so for the
other two terms.
<p>
<!-- ================================================================ -->
<p class="tags">

<p class="ref">
 <span class="authors">
Dodds ED, Hyun JA, Hagerman PJ, Lebrilla CB
 </span>
 <span class="year">
(2005)
 </span>
 <span class="jname">
J. Proteome Res.
 </span>
 <span class="jvol">
5,
 </span>
 <span class="pages">
1195-1203.
 </span>
<p class="reftitl">
Enhanced Peptide Mass Fingerprinting Through High Mass Accuracy:
Exclusion of Non-Peptide Signals Based on Residual Mass
<p class="Para1">
MOWSE scores are probalistic molecular weight search values reported by MASCOT.
They are logarithmic transforms of the random match probability:f
<i>S</i><sub>M</sub> = &minus;10 log(<i>P</i>) where <i>P</i> is the probability
that the protein has been identified through a random guess.  Factors involved
in score calculation include searches optionally involving variable amino
acid modifications, tolerance of missed proteolytic cleavage, and tolerance
of mass error, count of proteins in the database, count of peptides matched,
relative frequency of occurrence of peptide masses, and count of unassigned
peaks in submitted query.
<p>
Sequence coverage usually less than 50% even under most favorable circumstances.
Reasons why no assignment made to other peaks:  unknown posttranslational
modifications, variable modifications caused by sample preparation,
mutations or splice variants, errors in database catalog used to predict masses,
zero to low ionization efficiency (no peak to match a predicted peptide).
<p>
Algorithms are developed to refine PMF peak lists such as those making use of
an isotopomer distribution fitting based on avg amino acid elemental composition
(the <q>averagine</q> residue).  Poisson statistics can be use do extract
monoisotopic peptide masses.  Iterative, successive rounds of peak list
refinement to optimize outcome of PMF searches has been suggested.
Combining multiple peak lists to arrive at a representative list have been tried.
<p>
Monoisotopic masses of peptides are normally distributed about specific
fractional values for given nominal mass.
<!-- ================================================================ -->
<p class="tags">

<p class="ref">
 <span class="authors">
Torres MP, Thapar R, Marzluff WF, Borchers CH
 </span>
 <span class="year">
(2005)
 </span>
 <span class="jname">
J. Proteome Res.
 </span>
 <span class="jvol">
4,
 </span>
 <span class="pages">
1628-1635.
 </span>
<p class="reftitl">
Phosphatase-Directed Phosphorylation-Site Determination:  A Synthesis
of Methods for the Detection and Identification of Phosphopeptides
<p class="Para1">
Phosphoproteins were digested and then incubated with IMAC beads.
The IMAC was made by getting 0.5-2.5 &micro;l metal-free
nitrilotriacetic acid beaded agarose with ~425 beads/&micro;l
in compact reactions columns 35 &micro;m frit, USB Corp., and washing
with ultrapure water, then incubating with 30 &micro;l 50 mM ferric(III)
nitrate in 12 mM HCl three times, rinsing with 30 &micro;l water, then
30 &micro;l 0.1% acetic acid.  Peptides are diluted to 40 &micro;l with
50 mM acetic acid and then incubated shaking at RT overnight.  Unbound
material is washed with 30 &micro;l 0.1% acetic acid, 30 &micro;l
30% MeCN/0.1% TFA, and 30 &micro;l 0.1% acetic acid.  Final volume is
reduced to 5-10 &micro;l and 0.25-0.4 &micro;l of slurry (20-70 beads
 or 5-10% of total bead slurry) is spotted to MALDI plates.   Alkaline
phosphatase at various concentrations in 100 mM ammonium bicarbonate is
spotted in identical volume (0.25-0.4 &micro;l) with peptide or
IMAC-bound peptides.  After incubation time, a 0.5-0.8 &micro;l satd
CHCA matrxi in 50% MeCN + 0.1% TFA is spotted.  As an alternative,
CHCA in 50% MeCN/0.1-1.0% phosphoric acid was tried.  A Bruker Daltonics
Reflex III MALDI ToF was used with AnchorChip plate; for phosphatase
optimization, and a ABi 4700 MALDI ToF/ToF for other work.
<p>
One method for phosphatase treatment is to incubate 0.75-1 h at 37&deg;
with 1 U ALP and then clean peptide over POROS R3 reversed phase resin,
then do MS.  For on-target digestion on plates, incubation times must
be reduced to compensate for evaporation and enzyme quantities would be
reduced to deal with possible suppression effects.  Different enzyme
dilutions were mixed in buffer in equal volumes with the model
phosphopeptide PKA-I.  Matrix addition stops the reaction since a test
reaction was done with adding matrix first then the ALP and no dephosphorylation
was seen.  Different dilutions of enzyme affected the matrix crystallization
(1:10 dilution of enzyme had significant inhibition effect on crystallization,
but 1:100 enzyme did not).  Complete dephosphorylation is found with 1:10
and 1:100 (0.25 and 0.025 U respectively), but ion signals from 1:10 are
much less intense.  All reactions were within 5 minutes.  Note with 1:500
it could be seen that reactions were incomplete.  Since only a 0.6 &micro;l
volume is added, it is fully evaporated in 5-6 min, and so dephosphorylation
should be completed at a rate of 500 fmol/min.
<p>
KDIR-4 phosphopeptides were bound to immobilized Fe(III); a control
using uncharged IMAC beads was also done, and beads spotted (20-70 beads
per spot).  KDIR was seen only on charged IMAC beads and its phosphate
disappeared with phosphatase treatment.  Ionization was seen only when the
laser is aimed directly at the beads in peptides not treated with ALP.
In an earlier study with MALDI-QToF the matrix caused elution from
the beads, although the matrix was different (2,5-DHB) since DHB
was better than CHCA.  When peptides were ALP-treated, the better
ionization was when the laser is aimed the edges of bead clusters, indicating
that either the phosphatase reaction or ammonium bicarbonate buffer caused
elution.
<p>
The KDIR-4 phosphopeptide on the IMAC beads shows a <i>m/z</i> of 1702.7.
This peak is not detected after sequentially spotting beads followed by
ammonium bicarb without phosphatase.  When phosphatase was in the ammonium
bicarb buffer, KDIR-4 dephosphorylation is seen.  IMAC beads remaining
in the compact column were washed with ammonium bicarb buffer and no
peptides was found in wash solution (passed through C18 zip-tip to desalt);
KDIR-4 phosphopeptide does bind to C18 zip-tip in ammonium bicarb buffer.
No peaks are seen from IMAC beads washed with ammonium bicarb, nor
after re-equilibration of beads with 0.1% acetic acid.  These results
appear to show that there is a strong immobilization of phosphopeptide to
Fe(III)-charged IMAC beads which is irreversible with acetic acid and which
thus prevents ionization from beads.  This can be overcome by using the
phosphatase treatment.
<!-- ============================================================= -->
<p class="tags">
Proline effect, Fragmentation, FD
<p class="ref">
 <span class="authors">
Harrison AG, Young AB
 </span>
 <span class="year">
(2005)
 </span>
 <span class="jname">
J. Mass Spectrom.
 </span>
 <span class="jvol">
40,
 </span>
 <span class="pages">
1173-1186.
 </span>
<p class="reftitl">
Fragmentation reactions of deprotonated peptides containing proline. The proline 
effect
<p class="Para1">
In previous work by AGH, [M-H]<sup>&minus;</sup> deprotonated ions of ProVal and 
AlaPro were studied and the major fragmentation involved loss of CO<sub>2</sub> 
for forming <i>a</i><sub>2</sub> and <i>y</i><sub>1</sub> ions.  The Pro-Ala and 
Pro-Val peptides are studied here using a Q-ToF.  Spectra containing <i>m/z</i> 
113 and 72 were not expected in high energy CID of Ala-Pro, nor for <i>m/z</i> 
141 in Pro-Val.  For Pro-Ala under CID at 20 eV, six ions are produced, with 
<i>m/z</i> 141 being an <i>a</i><sub>2</sub> ion and <i>m/z</i> 88 being a 
<i>y</i><sub>1</sub> ion.  In MS<sup>3</sup> spectrum of <i>a</i><sub>2</sub> 
produces three ions at <i>m/z</i> 47.4, 100, and 15.2.  For Pro-Val, also six 
ions with <i>m/z</i> 169 being <i>a</i><sub>2</sub> and <i>m/z</i> 116 being 
<i>y</i><sub>1</sub>.  Four ions from <i>a</i><sub>2</sub> are at <i>m/z</i> 
100, 59.1, 13.9, and 9.4.  Loss of CO<sub>2</sub> accounts for 
<i>a</i><sub>2</sub>, and elimination of Pro accounts for <i>y</i><sub>1</sub>. 
The reaction scheme for the disintegration of the <i>a</i><sub>2</sub> ion is 
depicted with a proton from the N-terminal proton moving to the &alpha;-C 
carbanion formed in <i>a</i><sub>2</sub>.  This then leads to a break break of 
the &alpha;-C and carbonyl carbon-carbon bond and simultaneous break of peptide 
(amide) bond with formation of 1-pyrroline (also 3,4-dihydro-2<i>H</i>-pyrrole, 
CAS 5724-81-2) and CO, both neutral losses. What remains is an 
R<sub>1</sub>CH<sub>2</sub>NH<sup>&minus;</sup> anion detected as <i>m/z</i> 72 
if R<sub>1</sub> is CH<sub>3</sub> of Pro-Ala or <i>m/z</i> 100 if R<sub>1</sub> 
is isopropyl group of Pro-Val.  The elemental composition of <i>m/z</i> 113 in 
the Pro-Ala fragmentation should be 
[C<sub>5</sub>H<sub>9</sub>N<sub>2</sub>O]<sup>&minus;</sup>, which is an 
elimination of H<sub>2</sub>C=CH<sub>2</sub> from <i>a</i><sub>2</sub> ion.
Similarly the elimination of H<sub>2</sub>C=CH<sub>2</sub> in Pro-Val should
produce a [C<sub>7</sub>H<sub>13</sub>N<sub>2</sub>O]<sup>&minus;</sup> ion.
This elimination of ethylene would be possible if the anionic proline
ring has a C2-C3 carbon bond break to produce an enamine N-terminus with
the &alpha; carbon branched with ethyl anion, leading to ethylene loss
and the carbanion left on the &alpha;-C.
<p>
The next study is with a Pro-Gly-Gly tripeptide and the product ion spectrum of 
its <i>a</i><sub>3</sub> and <i>b</i><sub>2</sub> products.  Five product ions 
are seen in a 20 eV fragmentation: <i>a</i><sub>3</sub>, 
&prime;&prime;<i>b</i><sub>2</sub>, <i>a</i><sub>2</sub>, 
<i>a</i><sub>3</sub>&minus;P, and <i>y</i><sub>1</sub>, with <i>m/z</i> values 
of 184, 153, 127, 87, and 74, respectively.  The <i>a</i><sub>3</sub> ion
was subjected to further fragmentation and produced a &prime;&prime;<i>b</i><sub>2</sub>
ion by loss of CH<sub>3</sub>NH<sub>2</sub>;  it also loses 57 Da to form 
an <i>a</i><sub>2</sub> ion, and also a Pro residue to create <i>m/z</i> 87 peak
the base peak.  Others have reported that N-terminal residue is lost from the
<i>a</i><sub>3</sub>, and that &prime;&prime;<i>b</i><sub>2</sub> is formed from
neutral amine loss.  Glycine loss from <i>a</i><sub>3</sub> has not been reported
before, and it apparently involves elimination of imine and CO.  The 
&prime;&prime;<i>b</i><sub>2</sub> <i>m/z</i> 153 fragments to lose NH<sub>3</sub>,
CO, and [H<sub>2</sub>, CO].
<p>
PGG exchangeable hydrogens were changed with deuterium to make the 
<i>d</i><sub>4</sub> molecule to obtain the [M-D]<sup>&minus;</sup> spectrum. 
Results show that the &prime;&prime;<i>b</i><sub>2</sub> ion has one labile 
hydrogen, the <i>a</i><sub>2</sub> has two labile H atoms, and 
<i>a</i><sub>3</sub>&minus;P has 3 labile hydrogens.  There is a signal at 
<i>m/z</i> 89 showing incorporation of only two labile H atoms.  Some 
differences in fragmetnation pathways from previous reports might be related to 
the type of dissociation (CID vs. FAB).  Ionization of peptide methyl esters
is different for FAB and ESI:  for ESI it is deprotonation of amide nitrogen
but for FAB involves the &alpha;-carbon.  Such schemes must account for this.
<p>
[M-H]<sup>&minus;</sup> CID spectra of PVG and its <i>a</i><sub>3</sub> ion
show primary fragmentation to <i>a</i><sub>3</sub> and <i>y</i><sub>1</sub>.
Additional ions develop from <i>a</i><sub>3</sub>.  Peak at <i>m/z</i> 183
involves a C<sub>3</sub>H<sub>7</sub> side chain loss from <i>a</i><sub>3</sub>,
followed by neutral loss of CO to form <i>m/z</i> 155.  Ion <i>a</i><sub>3</sub>
fragmentation leads to &prime;&prime;<i>b</i><sub>2</sub> and formation of
<i>a</i><sub>2</sub> and lose of N-terminal Pro to form <i>m/z</i> 129.
<p>
The spectrum of APG and its <i>a</i><sub>3</sub> and <i>y</i><sub>2</sub>
product ions were examined.  The [M-H]<sup>&minus;</sup> major products are
<i>a</i><sub>3</sub>, <i>y</i><sub>2</sub> and <i>y</i><sub>1</sub>.  Cleavage
at Ala-Pro amide bond to produce <i>y</i><sub>2</sub> and charge retention
on C-terminus contrasts with findings of deprotonated tripeptides having only
H (Gly) or alkyl side chains which do not form these ions.  Signal at <i>m/z</i>
127 indicates that either Ala is lost from <i>a</i><sub>3</sub> or that 
<i>y</i><sub>2</sub> loses CO.  As the product spectrum of the <i>y</i><sub>2</sub>
ion (deproonated PG) has similarites with PA and PV dipeptides [M-H]<sup>&minus;</sup>
spectra with loss of ethylene from <i>a</i><sub>2</sub> (<i>m/z</i> 127) to form
<i>m/z</i> 99 ion.   The [M-H]<sup>&minus;</sup> and [M-D]<sup>&minus;</sup> 
were examined to show formation of <i>y</i><sub>1</sub>, <i>y</i><sub>2</sub>,
and <i>m/z</i> 141 (loss of CO<sub>2</sub> from <i>y</i><sub>2</sub>, and
possible loss of Gly from <i>a</i><sub>2</sub>).  Others have reported GPA
at high energy producing <i>y</i><sub>2</sub> and minor formation of <i>y</i><sub>1</sub>
and the <i>m/z</i> 141 fragment.  With deuterated peptide, <i>y</i><sub>1</sub>
retains a labile H and <i>y</i><sub>2</sub> retains two label H.
<p>
Tripeptide AAP was next studied and its <i>a</i><sub>3</sub> and 
&prime;&prime;<i>b</i><sub>2</sub> product spectrum also.  [M-H]<sup>&minus;</sup>
products are loss of CO<sub>2</sub> to form <i>a</i><sub>3</sub> and a <i>y</i><sub>1</sub>
deprotonated Pro.  The route to &prime;&prime;<i>b</i><sub>2</sub> is by loss
of neutral pyrrolidine from <i>a</i><sub>3</sub>.  In addition a <i>m/z</i>
87 ion indicates <i>c</i><sub>1</sub>, formed from ;<i>a</i><sub>3</sub>.
CID activation of &prime;&prime;<i>b</i><sub>2</sub> leads to <i>m/z</i>
124, 113, 111, and 97 which are respectively NH<sub>3</sub> loss, CO loss,
loss of H<sub>2</sub>, C, and O, and then loss of C<sub>2</sub>, H<sub>4</sub>,
and O.
<p>
The tripeptide LAP and its <i>a</i><sub>3</sub> and &prime;&prime;<i>b</i><sub>2</sub>
product spectra were checked.  Again the [M-H]<sup>&minus;</sup> involve loss
of <i>a</i><sub>3</sub> and <i>y</i><sub>1</sub>, and loss of pyrrolidine 
leads to formation of &prime;&prime;<i>b</i><sub>2</sub> from <i>a</i><sub>3</sub>.
The &prime;&prime;<i>b</i><sub>2</sub> further fragments with loss of NH<sub>3</sub>
to <i>m/z</i> 166, loss of CO to <i>m/z</i> 155, loss of [H<sub>2</sub>,C,O]
to <i>m/z</i> 153, and loss of [C<sub>2</sub>,H<sub>4</sub>,O] to <i>m/z</i> 139.
A [&prime;&prime;<i>b</i><sub>2</sub>-NH<sub>3</sub>]<sup>&minus;</sup> at
<i>m/z</i> 166 loses CO<sub>2</sub> to produce ion at <i>m/z</i> 122.  The 
[M-H]<sup>&minus;</sup> and [M-D]<sup>&minus;</sup> spectra were compared:
this shows that &prime;&prime;<i>b</i><sub>2</sub> (<i>m/z</i> 183) retains 
two label H atoms, and <i>y</i><sub>1</sub> ions retain one labile H atom, and
<i>a</i><sub>3</sub> has three labile H atoms as expected.
<p>
Product ion spectrum for tripeptide AFP [M-H]<sup>&minus;</sup> show two
primary pathways forming <i>a</i><sub>3</sub> and <i>y</i><sub>1</sub> ions.
Product ion spectrum for <i>a</i><sub>3</sub> indicates loss of pyrrolidine
to form &prime;&prime;<i>b</i><sub>2</sub>, but a benzyl radical is lost leading
to an ion at <i>m/z</i> 197 (side chain radical loss is often seen).  From
&prime;&prime;<i>b</i><sub>2</sub> there is a minor loss of ammonia forming
a <i>m/z</i> 200 ion and a benzyl anion <i>m/z</i> 91, but major fragment is
a <i>m/z</i> 97 ion resulting from neutral (or neutrals) whose loss is 120 Da.
Elemental composition of these neutrals would be C<sub>8</sub>H<sub>8</sub>O,
which mostly likely is C<sub>7</sub>H<sub>8</sub> (toluene) + CO.
<p>
In looking at all &prime;&prime;<i>b</i><sub>2</sub> fragmentation, the following
patterns are seen:
<ol>
<li>With an N-terminal Pro, a labile H is retained in 
&prime;&prime;<i>b</i><sub>2</sub> ion. If there is primary amine at N-terminus 
the &prime;&prime;<i>b</i><sub>2</sub> ion incorporates two labile H atoms
The labile H atom incorporated to &prime;&prime;<i>b</i><sub>2</sub> are from
the N-terminal amine.
<li>With the &prime;&prime;<i>b</i><sub>2</sub> a peak showing neutral loss
of NH<sub>3</sub> is evident, meaning that there is free amino group in the
&prime;&prime;<i>b</i><sub>2</sub> ion.
<li>CO loss from &prime;&prime;<i>b</i><sub>2</sub> represents a major 
fragmentation pathway, which is followed by elimination of H<sub>2</sub>. 
Moreover with [&prime;&prime;<i>b</i><sub>2</sub>-CO]<sup>&minus;</sup> ion 
there is loss of R<sub>2</sub>H where R<sub>2</sub> is a side chain of the 2nd 
amino acid of a tripeptide.  With Ala-Phe-Pro, C<sub>7</sub>H<sub>8</sub> is 
lost from the [&prime;&prime;<i>b</i><sub>2</sub>-CO]<sup>&minus;</sup> to make 
a <i>m/z</i> 97 base peak ion, and there is a complementary 
[C<sub>7</sub>H<sub>7</sub>]<sup>&minus;</sup> ion.  Similarly for Leu-Ala-Pro 
there is loss of CH<sub>4</sub> but no loss of C<sub>4</sub>H<sub>10</sub> butyl 
side chain of Leu.  The 
[&prime;&prime;<i>b</i><sub>2</sub>-CO]<sup>&minus;</sup> from Gly-Leu-Sar has 
an abundant ion 
[&prime;&prime;<i>b</i><sub>2</sub>-CO-C<sub>4</sub>H<sub>10</sub>]<sup>&minus;</sup>
<i>m/z</i> 83 but the peptide Leu-Gly-Pro shows only a minor formation of the
<i>m/z</i> 83 ion.
</ol>
<p>
The &prime;&prime;<i>b</i><sub>2</sub> structures are not fully determined.
Others have proposed the structure H<sub>N</sub>CH<sub>2</sub>C(=O)N<sup>&minus;</sup>CH=C=O.
Calculations using <i>ab initio</i> suggest a cyclic structure however:
H<sub>N</sub>CH<sub>2</sub>C(-*)=N-CH<sup>&minus;</sup>=C(-O-*)=O (the two
asterisk join).  The simplest &prime;&prime;<i>b</i><sub>2</sub> derived
from Gly-Gly-Xxx has a computational structure of the six-membered ring
H<sub>N(-*)</sub>CH<sub>2</sub>C(=O)-N<sup>&minus;</sup>-CH<sub>2</sub>-C(-*)=O
which as 9.5 kcal/mol lower energy.  Deuterium studies however rule this
structure out and another not shown.
<p>
In tripeptides, &prime;&prime;<i>b</i><sub>2</sub> ion formation less through a 
<i>a</i><sub>3</sub> parent.  Mechanism starts with loss of CO<sub>2</sub> 
leaving carbanion on &alpha;-C of 3rd residue.  The proton from amide N of 2nd 
residue moves to neutralize the &alpha;-C of 3rd residue, leading to amide 
tautomer form (C=N with oxyanion), and the 1st residue oxyanion forms 
nucleophilic addition to 2nd residue for a oxazolone 5-membered ring with an 
amide (C-N) bond break between 2nd and 3rd residues.  The negative charge is 
delocalized in the ring (as the &prime;&prime;<i>b</i><sub>2</sub> ion now), 
probably on the &alpha;-C of residue 2.  This mechanism is also consistent with 
deuterium studies.  In another scheme, the <i>a</i><sub>3</sub> parent sees the 
proton coming from the N-terminal amine N (not the second residue), and the 
N-terminal anionic N shows nucleophilic addition to 2nd residue carbonyl C, 
which forms a nice 6-membered ring with the 2nd-3rd residue amide bond break and 
either deprotonation of a -CH<sub>2</sub>- or -NH- group in the ring to form 
&prime;&prime;<i>b</i><sub>2</sub> ion (ring is diketopiperazine). However, this 
pathway would most like lead to deprotonation of the -NH- in the ring, and 
retention of only one labile H atom, not consistent with deuterium results, so 
the oxazolone ring mechanism seems more likely. Further 
&prime;&prime;<i>b</i><sub>2</sub> fragmentation is speculated to involve 
CO<sub>2</sub> loss, which has been shown for the &prime;&prime;<i>b</i> itself, 
but the spectrum shows a loss of 44 Da, but this is due to CO + CH<sub>4</sub>: 
the oxazolone ring in &prime;&prime;<i>b</i><sub>2</sub> first loses C=O with 
ring break and carbionic enamine on residue 2 with the charge (now the 
[&prime;&prime;<i>b</i><sub>2</sub>-CO]<sup>&minus;</sup> ion):  this can lead 
to a ring formation by N-terminal N atom with &alpha;-C in the anionic enamine 
and to loss of H<sub>2</sub> as the ring stabilizes the anion (probably on 
&alpha;C carbanion of 1st residue), or with proton transfer from N-terminal N 
atom against leading to ring but with loss of 2nd residue side chain and a 
proton to form same ring.
<p>
The 4-residue Ala-Ala-Pro-Ala and its <i>y</i><sub>2</sub> ion had their
deprotonated spectra analyzed.  The [M-H]<sup>&minus;</sup> major fragment
ion is <i>y</i><sub>2</sub>.  An Ala<sub>4</sub> peptide as well as other
side chains that are only alkyl have only minor formation of <i>y</i><sub>2</sub>
from deprotonated precursor.  This <i>y</i><sub>2</sub> is similar to 
spectrum from Pro-Ala, which has equal probability of having one or two labile
hydrogen atoms, indicating two pathways to <i>y</i><sub>2</sub>.  Schemes are
shown that show how one and two labile H atoms can be part of the <i>y</i><sub>2</sub>
ion product.
<p>
The 5-residue Ala-Ala-Pro-Ala-Ala was subjected to [M-H]<sup>&minus;</sup> 
ESI-MS.  In this case a <i>y</i><sub>3</sub> and 
[<i>y</i><sub>3</sub>-CO<sub>2</sub>]<sup>&minus;</sup> are prominent ions,
and cleavage in on amino side of Pro.  The CID spectrum of deprotonated A<sub>5</sub>
shows <i>y</i><sub>3</sub> as very minor.  Also there is a <i>c</i><sub>2</sub>
ion as well with AAPAA, where the charge goes to the N-terminal side.
The 4-peptide GPGG CID spectrum however gives an unexpected weak <i>y</i><sub>3</sub>
signal; the major ion is actually <i>a</i><sub>4</sub> which fragments further.
This is likely because at least two amino acids N-terminal to Pro are required
probably to stabilize the oxazolone or diketopiperazine neutral rings will form.
These are proven with Xxx-Pro-Yyy and Xxx-Yyy-Pro studies.
<p>
The CID [M-H]<sup>&minus;</sup> spectrum of peptide VGVAPG has <i>y</i><sub>2</sub>
as its major ion, again showing a pathway N-terminal to Pro residue.  A series of
<i>c</i> ions are also produced:  <i>m/z</i> 115 (<i>c</i><sub>1</sub>), 
172 (<i>c</i><sub>2</sub>), 271 <i>c</i><sub>3</sub>, and very weak 342 
(<i>c</i><sub>4</sub>).
<p>
The proline effect has been seen in [M+H]<sup>+</sup> and 
[M+<i>n</i>H]<sup><i>n</i>+</sup> (i.e, single and multiply protonated peptides
with Pro).  With VGVAPG a dominant <i>y</i><sub>2</sub>&prime;&prime; and 
<i>b</i><sub>4</sub> ions are proof (the significant <i>b</i><sub>3</sub> is because
of <i>b</i><sub>4</sub> continued fragmentation).  Also [M+H]<sup>+</sup> in tripeptides
Xxx-Yyy-Pro produces signficant <i>y</i><sub>1</sub>&prime;&prime; ions.
In addition there must be two amino acids N-terminal to the Pro residue. Studies
of [M+H]<sup>+</sup> ions of Xxx-Pro-Yyy show more cleavage on amide bond on the C-side
of Pro and this N-terminal side.  With GPGG, the cleavage is [M+H]<sup>+</sup>
&rarr; <i>b</i><sub>2</sub> &rarr; <i>ay</i><sub>2</sub> &rarr; P (proline iminium).
The <i>b</i><sub>2</sub> ion is major for APLG too.  The proline effect may be
due to proton affinity for Pro or because of the strain required to form a bicyclic
oxazolone with fragmenting on C-side of Pro.  Whe Pro is residue 2, formation of
<i>y<sub>n</sub></i>&prime;&prime; neutrals imine and CO are formed (requires higher
energy).  With deprotonated peptides however these neutrals are likely a ketene
than an imine + CO.  The mechanism is likely to involve the ability of Pro to stabilize
the negative charge with its secondary amine as opposed to a primary amine, enabling
the favored break at the N-terminal side of Pro.  CID MS of 

<!-- ================================================================ -->
<p class="tags">

<p class="ref">
 <span class="authors">
Xie H, Rhodus NL, Griffin RJ, Carlis JV, Griffin TJ
 </span>
 <span class="year">
(2005)
 </span>
 <span class="jname">
Molec. Cell. Proteomics
 </span>
 <span class="jvol">
4,
 </span>
 <span class="pages">
1826-1830.
 </span>
<p class="reftitl">
A Catalogue of Human Saliva Proteins Identified by Free Flow Electrophoresis-Based
Peptide Separation and Tandem Mass Spectrometry
<p class="Para1">
Whole unstimulated saliva was taken from healthy female.  1 ml was taken and 
spun 25000 <i>g</i> 30 min at 4&deg;., assayed by BCA (=1.05 mg). HEPES pH 8.0 
was added to 100 mM and TCEP to 5 mM.  20 &micro;g trypsin was added and left 
overnight at 37&deg;.  Peptides are collected on SepPak, and dried by vacuum 
centrifugation after elution.  The BD Biosciences ProTeam free flow 
electrophoresis (FFE) system was used to separate peptides, which are dissolved 
in 250 &micro;l of FFE separation buffer and then fractionated by FFE into 96-
well microplates (described previously).  A microelectrode was used to 
immediately measure pH in each FFE fraction (Fisher Accument combitrode). 50 
&micro;l (about 10%) aliquot is taken and processed for MS.  An automated 
Michrom Bioresources, Inc Paradigm MS4 System was used:  each FFE fraction 
loaded on Paradigm Platinum Peptide Nanotrap precolumn (150 &micro;m &times; 5 
cm, 0.4 ml volume) at 50 &micro;l/min in buffer A.  5 &micro;m, 200-&Aring; 
Magic C18AG C<sub>18</sub> resin was packed into 75 &micro;m &times; 12 cm. 
Peptides are eluted with 10-35% buffer B over 1 h, then isocratically at 80% B 
for 5 min at constant 250 nl/min flow.  This goes to electrospray on Thermo LTQ 
linear ion trap MS with voltage 2000 V and collision energy at 29%, scanning in 
<i>m/z</i> 400-1800 range, with four MS/MS scans for four most abundant 
precusors detected in a single MS scan.  Fill time of ion trap was 50 ms maximum,
and precursor <i>m/z</i> were excluded for 30 s before MS&rarr;MS/MS switching.
Databases for MS/MS spectra were searched using TurboSEQUEST (Thermo) against
the non-redundant human International Protein Index (~50,000 protein sequences,
a reverse version of database is also available).  Mass accuracy tolerance of 
2.0 for precursor ion, with Met oxidation as a differential modification.
<b>Interact</b> software tool used organize/view match results.  False positive
rates were determined.  Shimura et al used to calculate predicted pI values
of sequences.
<p>
<b>Results</b>.  Since initial fractionation of peptides based on pH separation
using FFE, it will be interesting to see how pI-predicting programs compare
to pH-measured fractionation.  First sequence matches were filtered using
Peptide Prophet, which computes a probability <i>p</i> for the match.
A strict <i>p</i> &ge; 0.90 threshold match level was used.  Fig 1A is a plot
of measured pH and computed pI vs FFE fraction number, with a plot of the number
of peptides in each fraction (max found was about 200 in one fraction).  Plot
does show a correlation or correspodence between <em>average</em> of the
computed pI of each sequence found in a fraction, and the measured pH of the
fraction, although the variation was larger on the alkaline end with respect
to average computed pI; thus confidence of computation may not apply in all
pH ranges.  A large number of unique peptides elute in the pH 3.5-5.0 region,
and only a very few peptides in the pH ~7-8 range (similar to a preparative
IEF study).
<p>
Fig 1B plots the false positive rate FPR (%) against a probability score.
A filter is applied as to the &Delta;pH between calculated peptide pI value
and the FFE fraction (measured pH) in which the peptide elutes.  False matches
are scored if random computed pI values are observed, and these are eliminated
in the filter.  The initial filter is high (&Delta;pH &plusmn; 0.5), a tolerance
optimal based on FFE-based IEF resolution.  The <i>p</i> score threshold can be
reduced in this filtering step while maintaining low 1% FPR.  Optimal <i>p</i>
threshold should differ for each dataset analyzed.  For this dataset, reduction
of <i>p</i> to 0.76 could be done with &Delta;pH filter applied.  A second-step
filtering used a lower <i>p</i> threshold of 0.2 and peptide pI with &Delta;pH
&plusmn; 0.5 filter; the protein would only be catalogued only if match by
&ge; 2 unique peptide sequences.  With first-step filtering, 433 proteins
were identified in peptide matches with <i>p</i> &ge; 0.76.  181 of the proteins
had two matches of peptides; the rest only one match.  Another four proteins
were added in second step.
<!-- ================================================================ -->
<p class="tags">

<p class="ref">
 <span class="authors">
Baczek T
 </span>
 <span class="year">
(2005)
 </span>
 <span class="jname">
Curr. Pharmaceut. Anal.
 </span>
 <span class="jvol">
1,
 </span>
 <span class="pages">
31-40.
 </span>
<p class="reftitl">
Improvement of Peptide Identification in Proteomics with Use of New
Analytical and Bioinformatic Strategies
<p class="Para1">
<b>REVIEW</b>.  Reports on modalities for achieving high resolution separation
of proteins prior to PMF or de novo sequencing.  This review contains
serious errors about methodology of preparation of peptides (to be trusted?).
<p>
The Sequest algorithm was developed in 1994 by Yates and co-workers.
It assumes that amino acid sequences can be inferred from tandem mass spectra.
The algorithm automates the inference by enumerating candidates from the
database that match the observed peptide's mass.  Sequences are checked
against the spectrum by a preliminary score and the first nonmatches are
removed.  More extensive cross-correlation scoring is used, and this
evaluates the sequence-derived theoretical spectra.  This compares against
the observed spectra and sequences are then ranked.  Statistics accumulate
help to classify matches.  Difference between normalized cross-correlation
functions and the first and second ranked results (&Delta;C<sub>n</sub>) is
used to indicate a correctly selected peptide sequence.  Cross-correlation
score between observed fragment spectrum and theoretically predicted one
(X<sub>corr</sub>) the preliminary score bsed on count of ions in MS/MS
spectrum matching the experimental data (S<sub>p</sub>), the rank of the
particular match during the preliminary scoring (RS<sub>p</sub>) and the
ions value (I) describing how many of the detecte (observed) match the
theoretical ions for the peptides are then listed.
<p>
Post-db search evaluation include the use of learning algorithm called
a support vector machine (SVM).  SVM distinguishes between correctly and
incorrectly identified peptides in looking for subtle patterns in a complex
data set.  It requires an appropriate training set of MS/MS data, and the
analysis does a better match between MS/MS and theoretical sequences.
In cases where SVM scores are low, the user needs to control noisy or
poorly fragmenting spectra which could compromise peptide identfication.
Additional study with <i>artificial neural network</i> (ANN) has been
proposed.  ANN works much like the human brain:  it is compact group of
connected, ordered layers able to process information.  There are three
types of layers:  input, one or more hidden, and then an output.  Elements
in the network are <q>artificial neurons</q> and each element connects
with a different strength called a <q>synaptic weight</q>.  The important
feature of ANN is its learning ability:  learning ability is determined
by alteration in synaptic weight values for connections applying a specific
algorithm.	The most commonly used algorithm is the <b>back-propagation
training algorithm</b>.  Learning in the use of that algorithm involves
analyzing error between the current and desirable output and then altering
synpatic weights to move toward the desirable output.  In analzying MS/MS
spectra of a tryptic digest of a yeast extract using LC-ESI-MS/MS,
each individual peptide was characterized by its peptide MW and peptide
charge (MW and CH, resp.) and Sequest-generated values (X<sub>corr</sub>,
&Delta;C<sub>n</sub>, S<sub>p</sub>, RS<sub>p</sub>, and I) and
atom-structure-calculated values of pI, change in charge with pH at the
pI (dz/dpH), and hydrophobicity (H).  Using these values in a training
set, an ANN could be constructed assign an MS/MS spectrum as <q>good</q>
or <q>bad</q> and eliminate its consideration in further analysis.
<p>
Prediciting a peptide's retention time on chromatography columns has been
studied extensively in connection with known important parameters of a
peptide such as amino acid composition.  Retention coefficients were
determined for each of the amino acids and end groups, derived from
experimental evidence of amino acids themselves.  Many groups reported,
all giving varying values however.  The influence of sequence too was
studied.  One group found that presence of a binding domain in an
amphipathic &alpha;-helical peptide produces greater retention than
predicted by other models, so amino acid composition may rank high.
<p>
QSRR, or quantitative structure-retention relationships were obtained using
statistically derived relationships between chromatographic parameters
and descriptors of molecular structure.  In the derivation, the
logarithm of the sum of the retention times of amino acids composing the
peptide, <i>log Sum<sub>AA</sub></i>, the logarithm of the van der Waals
volume of the peptide, <i>log VDW<sub>vol</sub></i>, and the logarithm of
the calculated <i>n</i>-octanol-water partition coefficient, <i>clog P</i>,
to come up with the following equation:
<p class="eqnserif">
<i>t<sub>R</sub></i> = 8.02 (&plusmn; 2.04) + 14.86 (&plusmn; 0.93)
<i>log Sum<sub>AA</sub></i> &minus; 5.77 (&plusmn; 1.16) <i>log VDW<sub>Vol</sub></i>
+ 0.28 (&plusmn; 0.06) <i>clog P</i>
<p>
There are four terms in the equation and each has a significance level
of <i>p</i> = 1 &times; 10<sup>&minus;4</sup>, 6 &times; 10<sup>&minus;29</sup>,
3 &times; 10<sup>&minus;6</sup>, 3 &times; 10<sup>&minus;6</sup>, with
a total significance in the equation of <i>p</i> &lt; 5 &times;
10<sup>&minus;55</sup> for <i>n</i> = 101 peptides used in the studies,
a multiple correlation coefficient of <i>R</i> = 0.963, standard error
of estimate <i>s</i> = 0.97, and <i>F</i> test ratio = 411.
<!-- ================================================================ -->
<p class="tags">

<p class="ref">
 <span class="authors">
Ji C, Guo N, Li L
 </span>
 <span class="year">
(2005)
 </span>
 <span class="jname">
J. Proteome Res.
 </span>
 <span class="jvol">
4,
 </span>
 <span class="pages">
2099-2108.
 </span>
<p class="reftitl">
Differential Dimethyl Labeling of N-Termini of Peptides After
Guanidination for Proteome Anaylsis
<p class="Para1">
Guanidination and reductive amination are chemistries with a long history.
In this report a first step is taken to block the &epsilon;-amino groups
of Lys using <i>O</i>-methylisourea at high pH.  After guanidination,
dimethyl labeling of the <i>N</i>-termini of the tryptic peptides with
either d(0),<sup>12</sup>C- or d(2),<sup>13</sup>C-formaldehyde.
This adds 28 or 34 Da to each peptide.  The guanidination converts
Lys to homoarginine, adding 42 Da to it, and does not at all modify the
N-terminus, probably because of steric hindrance from the R group.
The increased basicity of the Lys probably improves peptide ionization
for MS.
<p>
Myoglobin was used as a model to determine labeling efficiency.
100 &micro;l of 0.5 mg/ml Mb was heated to 95&deg; for 10 min, cooled
and 10 &micro;l 1 M sodium bicarbonate added to adjust pH to ~8.5.
Mb was digested overnight at 37&deg; at a 1:50 (w/w) enzyme/substrate
ratio.  Digests are stored at &minus;78&deg;.  For guanidination,
10 &micro;l of 2 M NaOH is added to irreversibly inactivate trypsin.
40 &micro;l of 2 M <i>O</i>-methylisourea in 100 mM NaHCO<sub>3</sub>
and adjusted to pH 11 with 2 M NaOH.  This is incubated 37&deg; for 2 h
or 65&deg; for 10 min.  The reaction was stopped and pH adjusted to 8
with 10% TFA.  15 &micro;l of 2 M NaBH<sub>3</sub>CN was added and vortexed.
3 &micro;l of 4% (w/w) either d(0),<sup>12</sup>C or d(2),<sup>13</sup>C
formaldehyde.  After vortexing mixtures are left at 37&deg; for 60 min.
3 &micro;l of 1 M NaHCO<sub>3</sub> was added to consume excess formaldehyde.
Note that it is critical that the dimethyl labeling be done under a hood.
MALDI MS was done with treated peptide using a Bruker Reflex III MALDI TOF
with 384 multiprobe inlet and 337 nm nitrogen laser with 3 ns pulse in
positive ion mode featuring delayed extraction.  Two-layer sample deposition
was used in preparing target plates, with first layer 20 mg/ml CHCA matrix
in 20% MeOH/acetone and second layer a saturating CHCA solution in
30% (v/v) methanol/water.  Second layer was mixed in 4:1 matrix:analyte
with dilute and acidified analyte.  0.5 &micro;l of first layer is deposited
and dried, then 0.5 &micro;l of sample and second layer, containing 5 ng
of Mb digest (~300-500 fmol).  After the 2nd layer was air-dried,
three &times; 1 &micro;l water was used to wash the dried layers.
<p>
Whey fractions of skim milk were labeled in a similar way.  About 1 mg
proteins in 30 &micro;l was diluted 1/50 with ultrapure and then
ultrafiltered with Microcon-3 (3000 Da cutoff).  Whey was prepared using
a report in literature.  caseins (making up 80% of total protein) were
prepared by acidifying milk to pH 4.6 with 2 M acetic acid and then
spun out in microfuge for 10 min (max speed).  Supernatant was dried
in 2 ml siliconized vial to 100 &micro;l.  Reductive alkylation with
DTT/iodoacetamide was done.  Acetone precipitation was done to remove
low MW reactants (it was precooled to &minus;80&deg; then added to
final 83% (v/v); vortexed 1 min; held at &minus;20&deg; for 2 h;
microcentrifuged at 4&deg; for 10 min max speed.  Pellets were solubilized
by adding 5 &micro;l of 1% SDS and vortexing.  The solution was then
diluted to 0.05% SDS and adjusted to pH 8.5 with 10 &micro;l 1 M
sodium bicarbonate.  2.5 &micro;l of 1 mg/ml trypsin and 5 &micro;l
20 mM calcium chloride were added for overnight digestion at 37&deg;.
<p>
For Mb, 15 peptides were seen in unlabeled Mb treatments whereas
20 peptides were seen with labeling.  Sequence coverage for the
tryptic, guanidinated, dimethylated peptides was up to 96.2% compared
to a coverage of 58% for unlabeled.  The 5 extra peptides were all
the result of labeling, with signal intensities of labeled peptides
comparable to unlabeled.  With some Lys-containing peptides increased
intensity is only seen with guanidination:  homoArg increases basicity
of peptides.  When Gly is the N-terminal amino acid, partial guanidination
is observed largely because the absence of a bulky R group.  Keeping
reaction temperature to &le;37&deg; will also help to minimize
the unwanted N-terminal guanidination where it might occur (the
use of 10 min at 65&deg; as opposed to 2 h at 37&deg; was found to
result in no side reaction as well).
<p>
As for the completeness of the reactions, spectra show that all peptides
have increased shifted masses indicating that guanidination and dimethyl
labeling are quantitative.  Peptides guanidinated alone and those
reacted as a combination show completeness of reaction.  The absence
of signal differences in the MALDI spectra indicates that dimethyl
labeling does not change the basicity of the peptides.
<p>
An isotope effect is observed with respect to chromatography of labeled
peptides.  There is a slight difference in retention times between the
deuterated and non-deuterated times.  The peak separation between
d(0),<sup>12</sup>C- and d(2),<sup>13</sup>C-formaldehyde labeled peptides
was 1 s on peaks which are 20-25 s wide on half height.  Deuterium tags
are added to charged site of peptide and these do not usually interact
with a nonpolar phase anyway, and the addition of 4 deuterium atoms does
not generally create a significant istope effect.
<p>
CID spectra were generated from unlabeled, light labeled, and heavy-labeled
peptides using the peptide YKELGFQG.  One major difference is seen
between the native/untreated/unlabeled spectra and those in which
guanidination occurs: fragment ions stand out better in labeled treatments.
The signal intensities of the <i>a</i><sub>1</sub> ion are substantially
increased 10-fold, which easily distinguishes it from immonium ion.
It is possible that resulting <i>b</i><sub>1</sub> ions form a stable
alkylated immonium ion <i>a</i><sub>1</sub> in losing a molecule of CO.
For some fragment ions, the count of <i>b</i> ions increased with labeling.
<p>
Skim milk proteins were used because they are less complex than serum
and have high abundance proteins which overwhelm low abundance proteins.
Differently labeled proteins were combined and injected on to microbore
LC-MALDI-MS/MS (ABi/MDS-Sciex QSTAR Pulsar QqTOF equipped with
orthogonal MALDI source).  The theoretical quantitative ratios should
be equal to 1.  All peptide pairs with good signal-to-noise were selected
for CID, making 56 peptide pairs.  From these 14 proteins were
unambiguously identified from the sequence info.  In some cases,
peptide cleavage was not along tryptic sites (nonspecific cleavage).
Lys guanidination did not interfere with MALDI MS/MS analysis.
The measured ratio for the heavy-light pairs was 1.02, close the
theoretical 1.00.  thus the error is 2.0%, and relative standard
deviation is 4.6%, well be 15% used as a threshold in quantitative
bioanalysis.  Phosphopeptides eluting at 33 and 35 min in the gradient
were sequenced in MS/MS easily.  The heavy-light ratio was 1.01.
<p>
The milk proteome is dominated by six gene products, four caseins
(&alpha;-S1, &alpha;-S2, &beta; and &kappa;), &beta;-lactoglobulin,
and &alpha;-lactalbumin which are 95% of the protein.  Identification
of minor proteins requires removing caseins by acid precipitation
from whey.  2DGE of the whey shows several isoforms of low abundance
lipocalin-type prostaglandin D synthase.  Still a great deal of caseins
were present.  Others have tried immunoadsorption to remove caseins
followed by 2DGE and microsequencing, which helped to identify
BSA, lactoferrin, ans serotransferrin.  The use of a high capacity of
microbore LC as opposed to capillary LC will provide accurate quanitation
because of an increased signal-to-noise (gain of 10<sup>4</sup>).
Here 8 low abundance proteins were identified, among them osteopontin-K
and cyclic dodecapeptide for the first time in mature bovine milk.
<p>
The steps for peptide sequencing in 2-MEGA labeling are:
<ol style="list-style-type:lower-roman;">
<li>digest the protein with protease (trypsin)
<li>block Lys residues by guanidination with <i>O</i>-methylisourea
<li>split guanidinated reaction equally into new reaction vials
and label these with d(0),<sup>12</sup>C-formaldehyde and
d(2),<sup>13</sup>-formaldehyde
<li>combine the peptide reactions
<li>analyze on combined material on microbore LC-MALDI MS
<li>select peptide pairs for CID
<li>overlay spectra of pairs and compare to delineate derived fragment
ions from N-terminal (<i>a</i> and <i>b</i> ions).
</ol>
<p>
The theoretical masses of the <i>a</i><sub>1</sub> peptide are given
in the table below for an N-terminal residue having one of the 20
amino acids.  When no labeling occurs, the <i>a</i><sub>1</sub> ion
signals in low mass region rarely show up or are indistinguishable from
immonium ion signals from other amino acid residues in the sequence.
In contrast labeled peptides have strong, distinguishing signals
for <i>a</i><sub>1</sub> ions in the low mass region, with unambiguous
fingerprinting of the N-terminal residue.
<p>
An example is studied with a peptide with retention time of 43 min.
The peptide pair with <i>m/z</i> 1165.58 and 1171.62 shows good
CID MS/MS spectra with identifiable <i>b</i> and <i>y</i>-series ions,
and an <i>a</i><sub>1</sub> of 164.10 and 170.11, respectively.  The
N-terminal residue can be assigned Y right away.
The separate MS/MS spectra are overlaid to show the labeling mass
difference, further verifying any <i>b</i>- and <i>y</i> series whose
signal-to-noise might be small.  The <i>b</i>-series indicates PVEPFTER.
Put together the YPVEPFTER is searched with BLAST and sufficient to
hit on bovine &beta;-casein, conforming to expectations of a milk protein.
Note that an Ser&rarr;Arg 137 transformation is observed, explaining
generation of a tryptic site.  Another variant (polymorphism) was seen
in &beta;-lactoglobulin with a Met&rarr;Val 23 conversion.

<table>
<caption>Theoretical Masses of the <i>a</i><sub>1</sub> Ions Derived
From The Twenty Amino Acid Residues Without and With <i>N</i>-Terminal
Dimethyl Labeling by Either d(0),<sup>12</sup>C- or d(2),<sup>13</sup>C-
Formaldehyde</caption>
<col>
<col span="3" style="text-align:right;" align="right">
<tr><th rowspan="2">N-terminal amino acid residues
 <th rowspan="2">Unmodified <i>a</i><sub>1</sub> mass
 <th colspan="2"> Modified <i>a</i><sub>1</sub> mass
<tr><th>+H<sub>4</sub><sup>12</sup>C<sub>2</sub>
    <th>+D<sub>4</sub><sup>13</sup>C<sub>2</sub>
<tr><td>Alanine (A)       <td> 44.050 <td> 72.081  <td> 78.113
<tr><td>Arginine (R)      <td>129.114 <td>157.145  <td>163.177
<tr><td>Asparagine (N)    <td> 87.006 <td>115.037  <td>121.069
<tr><td>Aspartic acid (D) <td> 88.040 <td>116.071  <td>122.103
<tr><td>Cysteine (C)*     <td>133.056 <td>161.087  <td>167.119
<tr><td>Glutamic acid (E) <td>102.056 <td>130.087  <td>136.119
<tr><td>Glutamine (Q)     <td>101.071 <td>129.103  <td>135.135
<tr><td>Glycine (G)       <td> 30.034 <td>58.066   <td>64.098
<tr><td>Histidine (H)     <td>110.072 <td>138.103  <td>144.135
<tr><td>Isoleucine (I)    <td> 86.097 <td>114.128  <td>120.160
<tr><td>Leucine (L)       <td> 86.097 <td>114.128  <td>120.160
<tr><td>Lysine (K)        <td>101.108 <td>171.161  <td>177.193
<tr><td>Methionine (M)    <td>104.053 <td>132.085  <td>138.117
<tr><td>Phenylalanine (F) <td>120.081 <td>148.113  <td>154.144
<tr><td>Proline (P)       <td> 70.066 <td> 84.081  <td>87.097
<tr><td>Serine (S)        <td> 60.045 <td> 88.076  <td>94.108
<tr><td>Threonine (T)     <td> 74.061 <td>102.092  <td>108.124
<tr><td>Tryptophan (W)    <td>159.092 <td>187.124  <td>193.155
<tr><td>Tyrosine (Y)      <td>136.076 <td>164.108  <td>170.139
<tr><td>Valine (V)        <td> 72.081 <td>100.113  <td>106.144
<tr><td colspan="4">
* Side chain of cysteine was blocked by iodoactemide.
</table>
<!-- ================================================================ -->
<p class="tags">

<p class="ref">
 <span class="authors">
Gilar M, Olivova P, Daly AE, Gebler JC
 </span>
 <span class="year">
(2005)
 </span>
 <span class="pages">
Waters Posters
 </span>
<p class="reftitl">
Development of Orthogonal Separation Methods for 2D-HPLC (MS/MS) Analysis
of Peptides
<p class="Para1">
In a 2D separation, the theoretical number of resolved components,
or <em>peak capacity</em>, is described as <i>P</i><sub>2D</sub> =
<i>P</i><sub>1</sub> &times; <i>P</i><sub>2</sub> where <i>P<sub>n</sub></i>
is the number of resolved peaks in dimension <i>n</i>.  This is true
only in LC when the modes are ideally orthogonal.  Retention times for
196 tryptic peptides were obtained in a 1D-LC-MS.  To describe the orthogonality
a geometric approach was used with a surface area of retention plots.
Retention maps are normalized by the equation:
<table class="eqnserif">
<tr><td rowspan="2"><i>RT</i><sub><i>i</i>(<i>norm</i>)</sub> =
<td class="numer"><i>RT<sub>i</sub></i> &minus; <i>RT</i><sub>min</sub>
<tr><td><i>RT</i><sub>max</sub> &minus; <i>RT</i><sub>min</sub>
</table>
<p>
where <i>RT</i><sub>min</sub> and <i>RT</i><sub>max</sub> are the retention
times of the first and last eluting peptides, and thus the equation evaluates
to a value in the range 0 to 1.  The 2D space is then divided into 14
&times; 14 bins for the total peak capacity of 196.
<p>
To make the map, construct a multilined grid with normalized retention
of the 1st D and 2nd D plotted on the vertical and horizontal axis,
respectively.  Divide the range 0-1 in 0.1 units, giving a 10 &times; 10
separation space with peak capacity being 100.  Suppose the peak plots
all fall on a 45&deg; line, covering the 10 squares that make up the line.
Thus 10% of the normalized area is used, but the 1st and 2nd dimensions do
not really differ with respect to chromatographic separation, and are thus
not orthogonal.  In another system, suppose that the plot points are all
over the map.  Where a point falls, the square is said to be used.  If
63 of the squares are used, 63% of the normalized area is covered, and
the orthogonality is considered to be 100%.
<p>
In a completely random distribution of 100 points, 63% of the map should
be covered.  Since orthogonality lies between 0-100% in a space from 10 to
63
<table class="eqnserif">
<tr><td rowspan="2"><i>O</i> % = 100 &times;
<td class="numer">&sum;<i>bins</i> &minus; &radic;<i>P</i><sub>max</sub>
<tr><td>0.63 &times; <i>P</i><sub>max</sub>
</table>
<p>
where &sum;<i>bins</i> is the total count of bins with at least one plot
point in them and <i>P</i><sub>max</sub> is the total count of all bins
representing the 2D peak capacity.  Practical peak capacity can be deterimed
from the mean peak width <i>w</i> represent the 4 &times; &sigma; spanning
the peak and in the useful gradient time <i>t<sub>g</sub></i>.
<table class="eqnserif">
<tr><td rowspan="2"><i>N<sub>P</sub></i> = <i>P</i><sub>1</sub> &times;
  <i>P</i><sub>2</sub> &times;
<td class="numer">&sum;<i>bins</i>
<tr><td><i>P</i><sub>max</sub>
<tr><td><i>P</i><sub>1,2</sub> = 1 + <i>t<sub>g</sub></i> / <i>w</i><sub>13.4%</sub>
</table>
<p>
Five protein digests of MassPREP standard proteins were produced and
run in 1D-LC-MS, and then 2D plots were made from a combination assessment:
phenyl &times; C<sub>18</sub>, pentafluorophenyl &times; C<sub>18</sub>,
C<sub>18</sub> @ pH 10 &times; C<sub>18</sub> @ pH 2.6 (formic acid and
ammonium formate), SEC (using YMC diol 4.6 mm &times; 75 cm 60 &Aring;
column) &times; C<sub>18</sub>, HILIC &times; C<sub>18</sub>, and
SCX &times; C<sub>18</sub>; all columns not specified for dimensions were
2.1 &times; 150 mm, with a 200 &micro;l/min flow, 40&deg; column temp
(SCX was 30&deg;), using solvent A = 0.2% formic acid and solvent B = 100% MeCN,
and when RP used 0.8% MeCN increase per min gradient over 50 min.  SEC
was 20% MeCN in 40 mM ammonium formate.  HILIC went from 90% MeCN down
by 0.8% per minute.  SCX was a PolyLC polysulfoaspartamide with 25% MeCN
constant with 40-300 mM ammonium formate pH 3.25 gradient.
<p>
2D space was divided into a 14 &times; 14 bin.  Of the six 2D maps made,
the first had no orthogonality (no difference between phenyl and C<sub>18</sub>).
The modified phenyl column showed only slightly more orthogonality.
Doing C<sub>18</sub> at different pH had better orthogonality.   When the
1st D had different physicochemical separation principles, orthogonality
was very much improved.  Ultimately there was a strong effect of pH in
separation and improved orthogonality.
<p>
Results allow several conclusions:
<ul>
<li>RP &times; RP 2D separations show low orthogonality.  It was not
useful to change ion-pairing buffer composition, or alter the RP stationary
phase (say from phenyl to C<sub>18</sub>).  The best orthogonality in
a RP &times; RP trial was with pentafluorophenyl &times; C<sub>18</sub>.
<li>Peptides are ionizable, and so pH is perhaps the best way to alter
selectivity.  Acidic (pH &lt; 5.5) allowed better retention for RP modes
when the peptides are acidic, and basic peptides are retained when the pH
is basic.
<li>Size exclusion (SEC) produced decent orthogonality with some interaction
with the stationary phase.  Some hydrophobic peptide loss is observed.
<li>HILIC &times; RP modes had the highest orthogonality.  A silica-based
Atlantis HILIC produced some of the best recovery.  A secondary interaction
with silanols produced cation-exchange behavior.
<li>SCX using NaCl is not compatible with MS, and ammonium formate increases
were adequate.  Higher charge density (charge-to-mass) often showed longer
retention (2+ short peptides retained longer than 2+ longer peptides).
2+ and 3+ peptides tended to cluster in a smaller region of 2D separation
space, reducing overall orthogonality of a SCX &times; RP system.
</ul>
<!-- ================================================================ -->
<p class="tags">

<p class="ref">
 <span class="authors">
Gedcke DA
 </span>
 <span class="year">
(????)
 </span>
 <span class="pages">
ORTEC&reg; Application Note 58 (AN58)
 </span>
<p class="reftitl">
How Histogramming and Counting Statistics Affect Peak Position Precision
<p class="Para1">
Mass spectra are effectively histograms (or frequency distributions),
in which ions falling at a
certain <i>m/z</i> (or class of ion) are counted within that position
or class, and the spectra is constructed as the counts at each class
(<i>m/z</i>) against the class (<i>m/z</i>).  A histogram is a plot
in which the horizontal access is separated at equal intervals and
each interval is an index into an array.  The vertical axis represents
the probability of recording an event in each interval.
But in creating histograms, the setting of the interval size will
affect the resolution or accuracy of measuring a peak position in
the spectrum.  These counting statistics are significant.  In TOF
mass spectrometers, time digitizers decide the interval.  The same is
true for any system using a counter in a time or pulse/amplitude/energy
domain.
<p>
The number of events counted over a duration with a time domain exhibits
a statistical fluctuation described as a Poisson probability distribution:
<table class="eqnserif" style="text-align:center;">
<tr><td rowspan="2"><i>P</i>(<i>N</i>) =
 <td class="numer"> <i>&mu;<sup>N</sup> e<sup>&minus;&mu;</sup></i>
<tr><td><i>N</i>!
</table>
<p>
<i>P(N)</i> is the probability of counting <i>N</i> events in a single
measurement.  As the count increases and the values of <i>N</i> are
averaged, the value of <i>N</i> approaches the mean &mu; of the distribution.
The standard deviation can be computed as:
<p style="text-align:center;" class="eqnserif">
&sigma;<sub><i>N</i></sub> = &radic;&mu; = &radic;<i>N</i>
<p>
Thus <i>N</i> is an adequate accurate estimate of &mu;.  The precision
of measurement is often expressed as a percentage of <i>N</i>, which
is &sigma;<sub><i>N</i></sub> / <i>N</i> &times; 100% =
100% / &radic;<i>N</i>.  For <i>N</i> = 1, &sigma;<sub>N</sub>% = 100.0%.
For <i>N</i> = 100, &sigma;<sub>N</sub>% = 10.0%.
For <i>N</i> = 10,000, &sigma;<sub>N</sub>% = 1.0%.
And for <i>N</i> = 1,000,000, &sigma;<sub>N</sub>% = 0.1%.
<p>
Many spectra have narrow peaks and the positions of those define the
physical quantity to be measured.  The peak centroid can affect the
final <i>m/z</i> determination of an ionized molecule.  To evaluate
the proper position, the bin interval (<q>channel</q>) of
the spectrum analyzer should be a fraction of the extent or range of a
peak (peak start to stop).  In a figure shown, the bin width interval
is 5% of the standard deviation, with is almost 1% of the entire
peak span.  The horizontal axis is calibrated to read such that
<p class="eqnserif" style="text-align:center;">
<i>x<sub>i</sub> = k i</i>
<p>
where <i>x<sub>i</sub></i> is the calibrated value for the center of the
<i>i</i><sup>th</sup> bin, and <i>k</i> is the calibration factor.
<p>
When the peak area is normalized to unity, the peak now represents an
empirical probability that a single even will be measured with a
particular value <i>x<sub>i</sub></i>, and so the peak is a probability
distribution for <i>x<sub>i</sub></i>.  The mean is computed to be
<table class="eqnserif" style="text-align:center;">
<tr><td rowspan="2"><i style="text-decoration:overline;">x</i> =
 <td class="numer">&nbsp;&sum;<sub
 style="margin-left:-0.5em;vertical-align:-1em;"><i>i</i></sub>
 &nbsp;<i>x<sub>i</sub> y<sub>i</sub></i>
<tr><td><i>N</i>
</table>
<p>
where <i>y<sub>i</sub></i> is the number of events counted in bin
<i>i</i>, and <i>N</i> is the total number of events forming the peak,
<i>&sum;<sub style="margin-left:-0.5em;vertical-align:-1em;"><i>i</i></sub>
&nbsp;y<sub>i</sub></i>.  The mean is the centroid of the peak, and will be the
determined value for the peak on the <i>x</i> axis.
<p>
The standard deviation for the probability distribution will be:
<table class="eqnserif" style="text-align:center;">
<tr><td rowspan="2"> &sigma;<sub><i>x</i></sub><sup>2</sup> &asymp;
 <i>s</i><sup>2</sup> =
 <td class="numer">&nbsp;<i>&sum;<sub
 style="margin-left:-0.5em;vertical-align:-1em;"><i>i</i></sub>
 &nbsp; y<sub>i</sub></i> (<i>x<sub>i</sub></i> &minus;
			<i style="text-decoration:overline;">x</i>)<sup>2</sup>
<tr><td><i>N</i> &minus; 1
</table>
<p>
(The denominator is <i>N</i> &minus; 1 because <i
style="text-decoration:overline;">x</i> has to be calculated from
the <i>N</i> values, and this removes one degree of freedom.
<p>
What is the effect of the bin width on the error in measuring the
peak centroid.  Consider a Gaussian peak with the shape having the
form
<table class="eqnserif" style="text-align:center;">
<tr><td rowspan="2"> <i>y</i> =
 <td class="numer">&nbsp;exp [&minus;(<i>x</i> &minus; <i>X</i><sub>C</sub>)
 <sup>2</sup> / 2 &sigma;<sub><i>x</i></sub><sup>2</sup> ]
<tr><td>&radic;<span style="text-decoration:overline;">2
&pi;&sigma;</span><sub><i>x</i></sub><sup>2</sup>
</table>
<p>
This can be plotted on Excel spreadsheet, with <i>X</i><sub>C</sub>
set to zero and <i>x</i> values computed in units of the standard
deviation.  A bin width of 0.05&sigma;<sub><i>x</i></sub>, small enough
to create negligible errors in conversion.  The bin overlapping the
centroid of the peak is centered at <i>x</i> = 0 and the remaining
bins extend up to &plusmn;8.2 &sigma;<sub><i>x</i></sub> about the centroid.
The value of the equation above is computed for the center of each bin
and multiplied by the bin width, which is the value of <i>y<sub>i</sub></i>
for each bin.  Note that the accuracy of this calculation was off by the
exact value of 1.000 with an error of 7 &times; 10<sup>&minus;10</sup>.
<p>
The error introduced can be analyzed by adjusting the bin width.  Suppose
only two bins form the peak, and their adjacent boundary coincides with
the centroid.  If bins are not symmetrically positioned then the
calculated centroid is offset from the true centroid.  One can determine
the offset by choosing different bin widths and sliding the bins
across the centroid in 8 steps, with step size being one-eighth of the
bin width.  The centroid of the resulting histogram is evaluated
and compared to the true centroid.  The bin widths used were 0.2, 0.4,
0.8, 1.0, 1.2, 1.6, 2.0, 2.4, and 3.2 of the &sigma;<sub><i>x</i></sub>.
These were divided 2.35 to produce the ratio of the FWHM to the bin width,
and thus are the number of bind that span the FWHM of the peak.
One can plot the maximum centroid error as a percent of the FWHM
against the ratio of the FWHM/(bin width).  When the count of bins
spanning FWHM exceeds 1.5, the max error in the centroid due to bin
width is &lt; 0.01% of the FWHM.  As the bin count across FWHM increases,
so does the error decrease steeply.  Beyond 2 bins, the error is so
miniscule that precision of Excel in determining the small differences
begins to limit accuracy of the calculation.
<p>
Does the error from bin width selection or the error from counting statistics
dominate?  Create a plot with log event counts against the bin width as a proportion
of the FWHM.  One can plot a reverse sigmoidal line in which the errors
of the two sources are equal.  Below and to the left of this sigmoidal
line the random error of counting statistics will dominate, whereas
above and right of the curve the systematic error from bin width is
dominant.  A spectrometer will usually acquire 5 or more than the
10<sup>7</sup> event counts in a peak to be analyzed.  The horizontal
limits of the rectangle extend from 0.05 FWHM to about 0.33 FWHM in the
typical operating range; this means from 3-20 bins spanning the FWHM.
Since the operating range box defined by these falls below and to
the left of the curve, the random error from counting statistics dominates
in causing centroid error.
<p>
The general presumption is that all bin widths are equal, but in practice
bin widths will be varied in the spectrometer.  The variation is called
Differential Non-Linearity (DNL), and percent DNL is calculated as:
<table class="eqnserif" style="text-align:center;">
<tr><td rowspan="2"> %DNL = &plusmn;
	<td class="numer"><i>W<sub>max</sub> &minus; W<sub>min</sub></i>
	<td rowspan="2"> &times; 100%
<tr><td><i>W<sub>max</sub> + W<sub>min</sub></i>
</table>
<p>
<i>W</i><sub>max</sub> is the maximum bin width and <i>W</i><sub>min</sub>
the minimum.  High quality spectrometers have a DNL within &plusmn;1%,
meaning the widest bin is no more than 1% wider than the average bin
width, the narrowest no more than 1% smaller than the average.
To investigate bin width-dominant error, and odd-even and random pattern
were investigated.  An average bin width equal to 0.4 &sigma;<sub><i>x</i></sub>
was used in both patterns, corresponding to 5.9 bins spanning the FWHM
of the Gaussian peak.  With odd-even pattern, the %DNL is &plusmn;12.5%,
and the centroid error is 4.7 &times; 10&minus;12% of the bin width
8.1 &times; 10<sup>&minus;13</sup>% of the FWHM, with an equivalent
<i>N</i> = 2.8 &times; 10<sup>27</sup>.  With the random pattern,
the %DNL is &plusmn;1.3%, the centroid error is 0.28% of the bin width
0.047% of the FWHM, with an equivalent <i>N</i> = 820,000.  In the odd-even
pattern the bin widths alternated between being 12.5% narrower (for odd)
and 12.5% wider (even) than the average.  Since this produces even spacing
between bin centers, the contribution to centroid error is negligible
although DNL is high (even extreme) at 12.5%.  For the random error,
the change in bin width was limited to &plusmn;1.3% but each succeeding
bin was changed randomly.  The centroid from the random DNL pattern is
small and can be ignored up to ca. 820,000 events.
<!-- ================================================================ -->
<p class="tags">
FD
<p class="ref">
 <span class="authors">
Paizs B, Suhai S
 </span>
 <span class="year">
(2005)
 </span>
 <span class="jname">
Mass Spec. Rev.
 </span>
 <span class="jvol">
24,
 </span>
 <span class="pages">
508-548.
 </span>
<p class="reftitl">
Fragmentation Pathways Of Protonated Peptides
<p class="Para1">
<b>REVIEW</b>.
Fragmentation within the mass spectrometer depends on a number of factors
including amino acid composition, peptide length, method of excitation,
time scale of instrument, ion charge state, and others.  Fragmentation
ideally in low energy collisions results in b and y ions that cleave along
the peptide bond, but also neutral losses of water and ammonia occur.
In addition to actual MS/MS spectral observation, spectra can be generated
in silico which predict peaks based on isotopic abundances.  A top-down
strategy is a statistical approach based on systematic assessment of large
databases containing MS/MS spectra of protonated peptides to derive
fragmentation rules.  The bottom-up approach does a systematic investigation
of the major pathways of fragmentation of protonated peptides to increase
knwoledge of dissocation chemistry.
<p><b>Mobile Proton Model</b>.  In this model protonated peptides activated
under low-energy collision conditions will fragment mainly by charge-directed
reactions.  Protonation sites exist in various functional groups of the peptide.
Some are more or less energetically favorable.  With the internal energy
of ions increased by collision, less energetically favorable sites can
accept a proton.  Charge-remote fragmentation (e.g., the <q>aspartic acid
effect</q>) can occur.  Molecular orbital calculations show protonation
of amide nitrogen leads to weakening of amide bond, while protonation of
amide oxygen makes amide bond stronger than in neutral species.  Generally
protonation is the least favorable on the amide N, while it is much more
thermodynamically favorable on amide O or on side chain nitrogens of Lys
and Arg.  Because it suggests an unstable state to have amide N protonated,
from the point of view of decomposition this state is favorable.
The mobile proton model suggests that unless protons are sequestered by
basic side chains of R and K, protons will move along the peptide backbone
and create the likelihood of decomposition.
<p>
Experimental evidence of the model comes from deuterium labeling.  Results
show strong H/D mixing (isotopic exchange) before collisionally activated
dissocation of [M+D]<sup>+</sup> ions.  Tsang and Harriso show facile H/D
mixing occurs in D<sub>2</sub> and CD<sub>4</sub> chemical ionization of
amino acids.  In peptides lacking Arg, [M+D]<sup>+</sup> show high proton
mobility between terminal amion groups and the amide N and O atoms.
In a study of the peptide FFF, fragmentation produces [M+H]<sup>+</sup>
and [M+D]<sup>+</sup> show complete exchange of added deuterons with labile
hydrogens in gas phase.  Deuterons move from the amino acid they had labeled
to fragments of other amino acids.
<p>
The energy required to move a proton from the amino terminus or basic side
chain depends on amino acid composition.  Dissociation requirements are
highest for Arg-containing peptides, decreasing in order of R-containing
&gt; K-containing &gt; non-basic peptides.  The mobile proton model adequately
explains even charge-state dependent fragmentation.  In cases where ionizing
proton number exceeds number of Arg residues, fragmentation is non-selective
or non-specific.  When Arg residues &ge; ionizing protons, then fragmentation
can occur specifically via the Asp effect.  Quantum chemistry has been used
to explain the structure and energetics of various protonated forms of model
peptides.  Transitions between chemical species are explained by internal
rotations, proton transfer reactions, etc.  Structures and energetics of
transition states are determined, then quantum chemical data (relative
energies, vibrational frequencies, etc) in RRKM formalism are applied to
approximate unimolecular reaction rates and time scale of underlying process.
Proton traffic maps can be constructed to show energy required to mobilize
additional protons between protonation sites.
<p>
Dissociation of bonds in fragmentation depends on energy and kinetic
accessibility of the reactive configurations and on the actual rate constants
of the bond cleavages themselves.  After fragmentation fate of the separating
fragments is determined by thermodynamics:  fragment with larger proton affinity
keeps the added proton responsible for charge-directed dissociation.
<p>
Peptide fragmentation pathways (PFPs) likely adhere to the Pathways in Competition
(PIC) model.  PFPs can be divided into three categories: (1) mechanistic,
(2) energetic, (3) kinetic.  Kinetics often explains competitive pathways.
Protonated peptide dissociation can be divided into either charge-directed
 or charge-remote PFPs.  Ions with low internal energy are generally fragmented by
charge-directed pathways although the charge-remote pathways described by the Asp
effect and the action of the oxidized Met side chain also work in low energy ions.
Charge-directed pathyways can be described as sequence-independent and -dependent
pathways.  Sequence-independent pathways explain neutral losses, while sequence-dependent
explain the alphabet (a, b, c, x, y, z) ions.
<p>
The <i>b/y</i> ion series are the most structurally informative and desirable.
It requires protonation of the amide N.  Several pathways are possible for the
protonated amide N.  These include the <i>a</i><sub>1</sub>-<i>y</i><sub>x</sub> pathway.
Such pathways require high energy ions however, and lower energy ions break on the
peptide bond only through complex rearrangement-type reactions.  Functions involved
include the oxygen of the N-terminal neighbor amide bond
(<i>b</i><sub>x</sub>-<i>y</i><sub>z</sub> pathway), the nitrogen of the N-terminal
amino group (aziridinone and diketopiperazine pathyways), and side chain nucleophiles.
These <i>b</i> and y ions can further fragment to <i>b</i> ions (<i>b</i><sub>x</sub>
&rarr; <i>b</i><sub>x-1</sub>), <i>a</i> ions (<i>b</i>x &rarr; <i>a</i><sub>x</sub>),
internal fragments and immonium ions.  Fragments with larger proton affinity will
hold the mobile hydrogen ion (thermodynamics rules).  Among the tools that can be
used to study the thermodynamics and kinetic of fragmentation, transition state-like
structures, isotope-labeling, controlled temperature variation in the fragmentation
chamber combined with theoretical evaluations can be used.
<p>
Additional theoretical tools requires scanning of potential energy surfaces (PES)
of protonated peptides, evaluating the characteristics/state of transition structures
from pre-dissociation to dissociations events and assessment of thermochemical
quantitite (proton affinities) of final products.  PES scanning is used to probe
energetic of different protonation sites, charge-solvated and salt-bridge forms.
Protein force fields are computed in low energy in molecular dynamics simulations.
Current methods involve first determining force fields on neutral (unprotonated)
species, then doning dynamics calculations by protonating collection of neutral
structures in second step.  A probably more practical approach is to use quantum
mechanical calculations on various levels with less accurate models and then finish
with density functional theory (DFT) on more interesting species that come out of the
rough work.
<p>
MS/MS ion abundances are determined by kinetics of PFPs.  Transition structures belonging
to fragmentaiton pathwyas, proton transfer reactions, intramolecular rearrangements,
and so are searched for.  Moderate basis sets in conjunctionw with DFT for protonated
peptide chemistry give good results, mainly because positive charge in ion forces
electrons close to nuclei and heterolytic bond cleavages are most important, described
by DFT reasonably.  With results of quantum chemical calculations, rate coefficients
for transitions between minima of PES can be calculated using RRKM method.
<p>
<b>Charge-Directed PFPs</b>.  Protonated peptides can be cleaved by either rearrangment-type]
or direct-bond cleavage.  Low energy ions are done by the former class, with exception
of <i>a</i><sub>1</sub>-<i>y</i><sub>x</sub> pathway.  Direct bond cleavage typically
requires high energy (excited) ions.  The <i>b</i><sub>x</sub>-<i>y</i><sub>z</sub>
pathway has the mobile proton on the amide N of residue <i>n</i>, which makes the carbonyl
C of residue <i>n</i> - 1 susceptible to nucleophilic bond formation.  The carbonyl O of
residue <i>n</i> - 2 obliges, creating two products, an ioniized (protonated) 5-member
N-terminal-side peptide (<i>b</i> ion) with oxazolone ring formed from residue <i>n</i> - 1, and a
C-terminal side product with N-terminus at residue <i>n</i>.  The C-terminal product
abstracts the proton from the ring N of the N-terminal product as well to form a
<i>y</i> ion in some cases.  The proton affinity of the generated fragments can be
determined from the relationship ln (<i>b</i><sub>x</sub>/<i>y</i><sub>z</sub>) ~
[(PA<sub>N-term</sub> &minus; PA<sub>C-term</sub>) / RT<i>eff</i>] where
<i>b</i><sub>x</sub>/<i>y</i><sub>z</sub> is ratio of abundances of the fragment
<i>b</i> and <i>y</i> ions, PA<sub>N-term</sub> and PA<sub>C-term</sub>
are proton affinitis of neutral fragments, and T<i>eff</i> is the effective temperature.
Mechanistic considerations explain what is observed.  CID spectra of protonated
C<sub>6</sub>H<sub>5</sub>CO-Gly-Gly-OH showed <i>b</i>-type ions and this matches the CID spectra of protonated
2-phenyl-5-oxazolone which supports oxazolone structure stabilizing <i>b</i> ion formation.
Study of N-methylated peptides (e.g., H-Gly-Sar-Sar-OH) the formation of <i>y</i><sub>1</sub>
ions from the <i>b</i><sub>2</sub>-<i>y</i><sub>1</sub> pathway is not possible
since the oxazolone product has a fixed charge and no proton to share with the leaving
H-Sar-OH.  The spectrum is dominated by b2 ions; y1 only appears when high
energies are input, originating probably from secondary fragmenation of y2 ions.
<p>
There is a linear relationship between log (y1/b2) and PA of C-terminal amino
acid for protonated tripeptides with series Gly-Gly-Xxx.  Fragmentation leads to
a b2 ion (protonated 2-aminomethyl-5-oxazolone).  Linearity of log(y1/b2) and
Xxx PA by using above equation:  PA<sub>N-term</sub> (PA of
2-aminomethyl-5-oxazolone) is constant while PA<sub>C-term</sub> is varied,
which makes the equation linear. Similar results fround for other peptide
series:  Gly-Xxx-Phe, benzoyl-Gly-Gly-Xxx, and benzoyl-Xxx-Gly-Gly.
<p>
<b>Diketopiperazine PFPs</b>.  Like the others, this starts with protonation of
an amide N.  Starting at the N-terminal residue (<i>n</i>), the <i>n</i>+1 amide
N is protonated (mobile proton).  This leads to <i>trans-cis</i> isomerization
of the peptide bond. The residue <i>n</i> amino N is re-protonated, but with the
geometric bond change, it is brought closer to the <i>n</i> + 1 and <i>n</i> + 2
peptide bond:  the <i>n</i> + 2 amide N takes the proton, making the N-terminal
amino N nucleophilic, and so it attacks the carbonyl C of residue <i>n</i> + 1,
resulting in a 6-member diketopiperazine ring composed of residues <i>n</i> and
<i>n</i> + 1, with product peptide having residue <i>n</i> + 2 as new
N-terminus.  It also takes the proton, and so a <i>y</i><sub>z-2</sub> ion is
produced (where z was original length).  PA values of cyclic peptides are lower
than those of linear, explaining why the leaving linear peptide abundance is
prominent.  This mechanism is not limited to n - 2 diketopiperazine formations.
It requires N-terminal amino N attack on the carbonyl C of the protonated amide
bond, which can be between the 3rd and 4th, 4th and 5th, and 5th and 6th residues,
leading to cyclo-tri-, cyclo-tetra-, and cyclo-pentapeptides.  Studies with
Leu-Gly-Pro (for neutral fragment reionization) show that the neutral counterpart
to <i>y</i><sub>1</sub> ion formatin is the diketopiperazine derivative.
<p>
<b>Amide Oxygen PFPs</b>.  Protonation of amide O atoms is more favored compared to
amide N atoms.  Others have proposed that b and y ions form not as a result of
amide N protonation.  Suppose amide O of residue n is protonated.  This makes the
amide O of residue <i>n</i> - 1 attracted to the carbonyl C containing the amide bond O
now protonated.  A 5-membered ring is formed, followed by a 1,1-elimination reaction
with a protonated oxazolone on the N-terminal fragment and linear C-terminal fragment.
Without proton transfer, the <i>b</i> ion is formed, and with it, the <i>y</i> ion is formed.
PES investigations for a large number of protonated peptides (GGG, GGP, AGG, GAG,
GGA, GGGG, AAAAA) shows that 5-membered rings made of amide O-protonated species
exist but are not energetically favored.  The 1,1-elimination requires bond
rearrangements to create a 4-membered highly strained ring which has a very large
energy barrier.  In larger peptides the barrier could be reduced with catalysis of
proton transfer step involving basic sites of the peptide.  It is unlikely that such
ion formations proceed by such a pathway.
<p>
<b>Dissociation of N-terminal Amide Bond in Underivatized Peptides</b>.  Charge-directed
dissociation of N-terminal amide bond differs from cleavage at other amide bonds.
Most dissociations of low energy ions happen by the rearrangements already described
rather than by direct bond cleavage.  Here the <i>a</i><sub>1</sub>-<i>y</i><sub>x</sub>
pathway is described as the exception.  If a rearrangement is posited since the
only available backbone nuclephile is the N-terminal amino group, the aziridinone
pathway is described, with a highly strained 3-membered ring.  Either the
<i>a</i><sub>1</sub>-<i>y</i><sub>x</sub> or aziridinone pathways can be inhibited
by acetylating the N-terminal amino group. For the <i>a</i><sub>1</sub>-<i>y</i><sub>x</sub>
pathway the proton travels to the <i>n</i> + 1 amide N.  A direct bond cleavage
occurs as an immonium (<i>a</i><sub>1</sub>) ion attempts to form (H2N=CH-R),
becoming the charged group, with a neutral loss of carbon monoxide (CO).
If there is proton transfer, then the <i>y</i><sub>x-1</sub> ion forms.  Whether
proton transfer occurs depends on a number of factors: internal energy distribution
of ion population, PA of fragments  (this can be approximated by the
ln (<i>a</i><sub>1</sub>/<i>y</i><sub>x</sub>) &#126; [(PA<sub>Nterm</sub> &minus;
PA<sub>Cterm</sub>)/RT<sub>eff</sub>] relation.)  Studies with dipeptides and small
peptides show <i>y</i><sub>x</sub> ions with added proton as well as H-atom attached
to N-terminal nitrogen, with migration.  Neutral fragment reionization studies with
AA dipeptide strong show ions dominated by MeCH=NH imine as a neutral counterpart
to y1 ion.
<p>
<b>Aziridinone PFPs</b>.  This is simply an attack of a nucleophilic uncharged
amine at the N-terminus on the carbonyl C of its residue so long as the amide
bond has been protonated on the amide N.  A 3-membered aziridinone ring forms
taking the proton with it, and this eventually leads to CO neutral loss with a1
ion formation. However, in the case of proton transfer, the y ion is formed.
Energetically and kinetically such a pathway is far less favored than
<i>a</i><sub>1</sub>-<i>y</i><sub>x</sub>. However, some experimental evidence
could be explained by the aziridinone pathway based on some dipeptide studies
with Val-Xxx, where Xxx is varied and proton affinities are known.
<p>
<b>Charge-Directed Amide Bond Cleavage by Attack of Nucleophilic Side
Chains</b>. The effect of some reactive side chains in directing cleavage cannot
be underestimated. Nucleophilic-based cleavage will involve His, Glu, Asn, Lys,
and Arg.  The mobile proton charges the amide bond with the nucleophile on the
side chain of the residue on the N-terminal side.  This creates a cyclic
intermediate effectively being the <i>b</i> ion.  This must be distinguished from the
<i>b</i><sub>x</sub>-<i>y</i><sub>z</sub> pathway in which the carbonyl O of
residue <i>n</i> - 1 attacks the carbonyl carbon to form the 5-membered
oxazolone ring. For His residues, a clear <q>histidine effect</q> is seen so
long as the protons added is larger than the number of Arg residues present.
His has a high PA and is likely charged in gas phase low energy ions.  With His
as a nucleophile, the <i>b</i> ion is a bicyclic, and there is an association
long enough for proton transfer to allow distribution of protons to either
product ion.  The His effect has been studied by varying the C-side adjacent
residue, by alkylating the imidazole ring, and so on. MS/MS spectra of of 2+ ion
RVYIHPF shows <i>b</i><sub>5</sub> and <i>y</i><sub>2</sub> ion pairs dominate,
indicating both proton transfer and His effect.  for the substitution of Pro for
Ala (RVYIHAF), dominance of <i>b</i><sub>5</sub> and <i>b</i><sub>6</sub> are
found, explained by PA difference for PF and AF dipeptides.  If alkylation of
ring is done, a <i>b</i><sub>x</sub><sup>2+</sup> ion dominates, indicating
proton transfer does not occur.
<p>
The amide side chains of Gln and Asn contribute to fragmentation as well.  The side
chain amide O atom nucleophilically bonds to its carbonyl C on the carboxyl side
of the protonated amide bond to form a cyclic isoimide and the leaving C-terminal
portion.  With proton transfers, either <i>b</i> or <i>y</i> ions are formed.
b2 ions derived from N-acyl Gln and Asn methyl esters were studied using MS<sup>n</sup>
and ab initio techniques.  The <i>b</i><sub>2</sub> ions fragment further to
produce CO and CH<sub>2</sub>CO, which indicate oxazolone and isoimide forms.
Ab initio calculations indicate the oxazolone is more stable than isoimide, although
the Gln form difference is smaller (~2 kcal/mol) than for the Asn form (~11 kcal/mol).
Study with GQG peptide using MS<sup>n</sup> shows b2 further fragments which includes
CO neutrals and a Gly residue.  The CO loss suggest classical oxazolone while latter
indicates isoimide.  Studies of peptides with a QG motif, name jQGb where j is any
amino acid and b is any except G indicates Gln-based cleavage mechanism.
<p>
With Lys and Arg side chains, if mobile proton gets to a Lys-Xxx amide bond, the
Lys side chain can form a caprolactam.  Lys-Gly and other derivatives have been
studied under MS. An ion with <i>m/z</i> 129 probably represents the
caprolactam.  The dipeptide has been studed using quantum chemical and RRKM
calculations as well.  Studies with b2 ions indicate that there is no CO loss
and only the caprolactam is avaialble. Ab initio indicates oxazolone less stable
than caprolactam. The peptide GGKAA was examined using ion trap MS:
<i>b</i><sub>3</sub> ion was formed more abundantly than YGGFL.  The GGKAA
<i>b</i><sub>3</sub> ion loses H2O and GG with no dissociation signs (loss of CO
and <i>b</i><sub>x</sub> &rarr; <i>b</i><sub>x-1</sub>) suggestive of oxazolone
intermediate.  With Arg side chains, there is a great deal of evidence that it
too forms a 6-membered ring structure with a methyl diamine substituent.
<p>
<b>Loss of Small Neutrals</b>.  MS/MS spectra provide much evidence of the loss
of small neutrals such as water and ammonia.  In low energy conditions
[MH-H<sub>2</sub>O]<sup>+</sup>, [MH-NH<sub>3</sub>]<sup>+</sup>,
[<i>y</i><sub>z</sub>-H<sub>2</sub>O]<sup>+</sup>,
[<i>y</i><sub>z</sub>-NH<sub>3</sub>]<sup>+</sup>,
[<i>b</i><sub>x</sub>-H2O]<sup>+</sup>,
[<i>b</i><sub>x</sub>-NH<sub>3</sub>]<sup>+</sup> result from charge-directed
pathways. Water loss can occur from C-terminal, D and E carboxyl groups, from
backbone amide O atoms, and Ser and Thr side chains.  Ammonia loss can occur
from Arg, Lys, Asn, and Gln.  What mechanisms are proposed for neutral losses?
When lost from the C-terminal COOH, one mechanism is proposed in which the n - 1
carbonyl O atom is protonated; this makes it nucleophilic, and it attacks the
carboxyl C atom of the terminus.  A concerted reaction of the COOH hydroxyl with
the mobile proton causes loss of water, and the C-terminal amino acid forms a
5-membered oxazolone ring. Experiments with <sup>18</sup>O and blocked carboxyl
group show for instance that the peptide TRKR dehydrates exclusively at the
C-terminal COOH group.  This mechanism also explains water loss from Asp and Glu
side chains although there is a different cyclic product.  Peptides or their y
ions that have Glu at the N-terminal position show significant side chain water
loss, which involves pyroglutamate formation. MS/MS of
[<sup>18</sup>O<sub>2</sub>]-RPPGF peptide shows dehydration occurring primarily
through loss of H<sub>2</sub><sup>16</sup>O involving backbone amide O atoms.
MS/MS/MS spectra show the oxygen comes from the carbonyl of amide bond nearest
to N-terminus.  It was shown that b4 from GGGG and b5 from GGGGG do not have
oxazolone structures, expected if dehydration involves C-terminal COOH.  MS/MS
of methylated GGGG indicates water elimination by an amide O atom on backbone.
Dehydration of Ser and Thr is thought to occur when the mobile proton bonds to
the hydroxyl, forming a cationic leaving group;  the amide O atom of the
previous residue bonds to the methylene, creating the water loss, and forming a
5-membered ring in the case of Ser, 6-membered for Thr.
<p>
Ammonia loss comes strictly from side chains of basic amino acids and amides.
From Lys side chains it occurs through S<sub>N</sub>2-type reactoins.
<sup>15</sup>N-labeling experiments show strict elimination, although there is
position dependence. When Lys is at N-terminus, the amino terminus attacks the
delta-methylene of the side chain, with formation of the 6-membered pipecolic
acid at N-teriminus; this is more kinetically favored than an amide O atom
attack.  Unfavorable entropic factors prevent amino terminal attack of Lys side
chain not at the terminus.  Theoretical methods were used to determine mechanism
for Arg.  Its protonated guanidino function is capable of multiple tautomer
formation:
-N<sub>&epsilon;</sub>H-C(N<sub>&eta;</sub>H<sub>3</sub><sup>+</sup>)(N<sub>&eta;&prime;</sub>H) or
-N<sub>&epsilon;</sub>-C(N<sub>&eta;</sub>H<sub>3</sub><sup>+</sup>)(N<sub>&eta;
&prime;</sub>H<sub>2</sub>). In this case ammonia loss probably proceeds via
S<sub>N</sub>1-type reaction. Finally, the amide O atom (probably on the
residue) attacks the guanidinyl carbon to form an 8-membered ring with ammonia
loss.  With Gln and Asn, then N atom of the amide side chain is protonated.  If
the residue is N-terminal Gln, then pyroglutamate is likely since the amino terminus
is a good nucleophile.  If Gln or Asn are within the peptide, attack of the amide
carbon is probably from peptide backbone amide O atom.
<p>
<b>Continuing Fragmentation</b>.  Fragmentation is not limited to bond
dissociations of the parent peptide.  Ions in the <i>b</i> and <i>y</i> series
can fragment further to smaller <i>b</i> and <i>y</i> ions.  <i>y</i> Ions have
no special structure, so their continued fragmentation can follow
<i>b</i><sub>x</sub>-<i>y</i><sub>z</sub>,
<i>a</i><sub>1</sub>-<i>y</i><sub>z</sub>, etc. PFPs.  <i>b</i> Ions have the
oxazolone ring and so follow additional specific fragmentation such as CO
elimination by the <i>b</i><sub>x</sub>&rarr;<i>a</i><sub>x</sub> pathway and
formation of smaller <i>b</i> ions by the
<i>b</i><sub>x</sub>&rarr;<i>b</i><sub>x-1</sub> pathway.  Doubly protonated
tryptic peptides show have weak b ion signals in triple quads or Q-ToF from
initial fragmentations; this is likely because of b ion subfragmentation is
more favored in these systems.  Singly charged y ions from initial fragmentation
are more likely to be seen on MS/MS spectra because of apparent stability,
probably because added protons on the C-terminal side are sequested by Lys and
Arg residues of tryptic peptides.
<p>
<b>Subfragmentation of <i>b</i> ions</b>.  Higher <i>b</i> ion fragmentation of
GGGG, GGGGG, and YGGFL have been studied. <i>b</i><sub>3</sub> and
<i>b</i><sub>4</sub> ions are likely to fragment in a sequence-dependent way to
form lower b ions.  There are strong indications of formations of oxazolone
rights particularly in penta-Ala homopolymer.  A mechanism has been proposed in
which an amide O atom on the N-terminal side is nucleophilically attracted to
the -C=N<sup>+</sup>- moiety in the oxazolone ring.  This actually leads to
formation of an aziridinone tricyclic ring with the C-terminal residue of the
<i>b</i><sub>x</sub> ion, and a <i>b</i><sub>x-1</sub> ion is formed with
another oxazolone ring.  This particular reaction had an activation energy of 77
kcal/mol, too high for mass spectrometers under low-energy.  A pathway of
<i>b</i><sub>x</sub>&rarr;<i>a</i><sub>x</sub>&rarr;<i>b</i><sub>x-1</sub>
requires 40 kcal/mol internal energy, but metastable ion studies show the
<i>b</i><sub>x</sub>&rarr;<i>a</i><sub>x</sub> reaction occurs with a
substantial release of kinetic energy (T<sub>&frac12;</sub> = 0.3-0.5 eV),
indicating that the bx&rarr;ax&rarr;bx-1 pathway is unlikely as there are low
KER values.  Another group using stored waveform Fourier transform and double
resonance techniques with quadrupole ion-trap showed that 50% of
<i>b</i><sub>3</sub> ions of Leu-enkephalin originate from <i>b</i><sub>4</sub>
ions directly under moderately energetic conditions, and so the
<i>b</i><sub>x</sub>&rarr;<i>b</i><sub>x-1</sub> pathway is yet unknown.
<p>
The <i>b</i>x&rarr;ax pathways produce ion pairs differing by 28 Da and this
helps the identification of the b ion series, as these immonium (ax) ions point
to their parent b ions.  Protonation of the N atom in the oxazolone ring leads
to weakening of -O-CO- bond and CO elimination by a concerted pathway, which
requires breaking of two covalent bonds of the cyclic <i>b</i><sub>x</sub> ion
to form a linear <i>a</i><sub>x</sub> ion.  Activation energies for the
<i>b</i><sub>x</sub>&rarr;<sub>a</sub><sub>x</sub> paths of
HCO-NH-CH<sub>2</sub>CO<sup>+</sup>, MeCO-NH-CHMeCO<sup>+</sup>,
NH<sub>2</sub>-iBuCH-CO-NH-CH<sub>2</sub>-CO<sup>+</sup>, and
NH<sub>2</sub>-CH<sub>2</sub>-CO-NH-iBuCHCO<sup>+</sup>
<i>b</i><sub>2</sub><sup>+</sup> ions are 29, 30.5, 34.6, and 30.3 kcal/mol,
respectively.  In general <i>b</i><sub>x</sub>&rarr;<i>b</i><sub>x-1</sub>
and <i>b</i><sub>x</sub>&rarr;<i>a</i><sub>x</sub> dissociations are the
main pathways.
<p>
<b>Internal fragment</b> formation requires cleavage of at least two backbone
amide bond and leads to inos of underivatized N- and oxazolone or immonium
C-terminus.  In low energy collision spectra these fragments appear when
y ions are present in unusual abundance.  It occurs very frequently with
peptides containing Pro and His (see discussion below of this effect).
Formation of these has been studied using YGGFL and des-Arg-substance P
in product ion scanning mand reaction intermediate scanning modes.  Certain
yz ions fragment further.  In des-Arg-subP, the Lys-Pro amide bond cleaves
(Pro effect) with <i>a</i><sub>x</sub><i>y</i><sub>8</sub> and
<i>b</i><sub>x</sub><i>y</i><sub>8</sub> formation.  N-terminal residue
immonium ion formation comes down from the
<i>a</i><sub>1</sub>-<i>y</i><sub>z</sub> and
<i>b</i><sub>2</sub>&rarr;<i>a</i><sub>1</sub> pathways, although immonium
ions in the low mass range may also originate from amino acids in other
positions.  Internal immonium ion (<i>I</i><sub>x</sub>) formation comes
two pathways: <i>a</i><sub>x</sub> ions fromed from
<i>b</i><sub>x</sub>-<i>y</i><sub>z</sub> and
<i>b</i><sub>x</sub>&rarr;<i>a</i><sub>x</sub> fragment to produce
<i>a</i><sub>x-1</sub> and <i>I</i><sub>x</sub>.
<p>
While the Asp and His effects clearly result from chemical activity which
can be shut down by chemical modification (Asp esterification, for ex), the
<b>Proline Effect</b> works by altering other pathways leading to dominance of
a few ions, especially in the <i>b</i><sub>x</sub>-<i>y</i><sub>z</sub> and
diketopiperazine pathways.  Peptides AAPAA and AAAPA were found to produce
abundant <i>y</i><sub>3</sub> and <i>y</i><sub>2</sub> ions, respectively, and
this is because of a high PA by Pro.  Peptide VAPLG forms abundant <i>y</i><sub>3</sub> ion,
whereas Val-Ala-NMeAla-Leu-Gly and Val-Ala-Pip-Leu-Gly produce significant
<i>b</i><sub>3</sub> ions.  VAPLG and VA-Pip-LG differ by only a methylene
group, and the other peptide is N-methylated, indicating an N-methylation
effect.  Simple proton affinity differences cannot account for this.
A b3 ion formation on C-side of Pro requires formation of highly strained
bicyclic [3.3.0] moiety.  However, if the residue is Pip, a less strained
bicylic [4.3.0] moiety is formed; in the other case, N-methylation dictates
fragmentation.  The tripeptide GPG shows a dominant <i>b</i><sub>2</sub> ion which contains
a bicyclic [3.3.0] moiety.  We contend that there is no special Pro effect
on cleavage of N-terminal amide bond and that this cleavage is described by the
<i>a</i><sub>1</sub>-<i>y</i><sub>z</sub> pathway.  The dipeptide VP conforms
to <i>a</i><sub>1</sub>-<i>y</i><sub>1</sub> fragmentation rules in that
<i>y</i><sub>1</sub> is favored at low energies (PA of Pro higher than imine
derived from Val).  At C-terminal of tripeptides it may have a special effect:
in LGP tripeptide, the majority of <i>y</i><sub>1</sub> ions form via the diketopiperazine
pathway whereas with GGG and GGL, <i>y</i><sub>1</sub> ions form from a
<i>b</i><sub>2</sub>-<i>y</i><sub>l</sub> pathway.
<p>
In cyclic peptides because there is no free amino terminal, pathways that
involve internal amide bond dissociation dominate.  The initial fragmenation
is by loss of CO and production of a sequence ion:
<i>b</i><sub>x</sub><sup>n-m</sup> and <i>y</i><sub>z</sub><sup>n-m</sup>
where <i>n</i> is the N-side amino acid and <i>m</i> is the C-side amino acid of the amide
bond broken.  Prior to fragmentation, an amide O atom is protonated with
mobilization to an amide N atom creating the high potential energy.  Ring
opening is following by oxazolone formation of the new C-terminus.  Linearized
peptide can cause <i>b</i><sub>x</sub>&rarr;<i>a</i><sub>x</sub> formation
(CO loss) with formation of a lower <i>b</i> ion
(<i>b</i><sub>x</sub>&rarr;<i>b</i><sub>x-1</sub>).  With N-methylation on
residue <i>n</i>, amide bond cleavage then occurs between residues <i>n</i> and
<i>n</i> + 1, producing and N-methylated oxazolone ring;  this ring degrades
further with lose of CO (bx&rarr;ax) or by losing the resiude (bx&rarr;bx-1).
<p>
<b>Charge-Remote PFPs</b>.  The <b>aspartic acid effect</b> is seen to produce
abundant <i>b</i> ions on the C-side of Asp residues.  Investigations of
Asp-Pro and Asp-Xxx in MALDI MS shows great lability.  If Asp is esterified,
b ion formation produced by this effect is eliminated.  The Asp75-Met76 bond
in Hb alpha chain undergoes this effect particularly for 11+ and 12+ ions.
In ubiquitin the 11+ ion fragments easily into the
<i>b</i><sub>52</sub><sup>7+</sup>/<i>y</i><sub>24</sub><sup>4+</sup>
ion pair, because of the presence of Asp-52.
<!-- ================================================================ -->
<p class="tags">

<p class="ref">
 <span class="authors">
Kiri AN, Tran H-C, Drahos KL, Lan WK, McRorie DK, Horn MJ
 </span>
 <span class="year">
(2005)
 </span>
 <span class="jname">
J. Biomolec. Techniques
 </span>
 <span class="jvol">
16,
 </span>
 <span class="pages">
369-377.
 </span>
<p class="reftitl">
Proteomic Changes in Bovine Heart Mitochondria with Age:  Using A
Novel Technique for Organelle Separation and Enrichment
<p class="Para1">
Bovine hearts from 3-8 mon young and 18-24 mon adult were harvested
and whole hearts rapidly frozen in liq. nitrogen for storage.
Tissue was then thawed at 4&deg; in PBS.  100 g myocardium free of
connective and vascular tissue was homogenized in blender in 0.5 L
0.5 M sucrose buffered with 20 mM HEPES pH 7.2, 5 mM MgCl<sub>2</sub>,
and inhibitors (Roche Complete); blending was 30 sec high, 30 sec low.
Material was filtered through 4 layers cheesecloth, then spun 750 g 10 min
to remove nuclei.  Pellets were resuspended in extraction buffer and
blended and re-spun again.  Supernatants are combined and diluted 2-fold
in 20 mM HEPES pH 7.2, 5 mM MgCl<sub>2</sub> so that sucrose is now 0.25 M.
<p>
An Alfa Wasserman pCFU continuous-flow ultracentrifuge was equipped with
PKII 800 ml rotor and loaded with 20 mM HEPES pH 7.2, 5 mM MgCl<sub>2</sub>,
0.25 M sucrose buffer using peristaltic pump.  Air bubbles are removed
by spinning at 20000 rpm and pumping buffer up through bottom.  The rotor
is stopped and a higher density solution is introduced to the bottom
(60% sucrose in buffer) at 10 ml/min.  The rotor is accelerated to 3500 rpm
to create the linear gradient.  Acceleration continues to 20000 rpm and
then the homogenate is loaded.  Material from the top is recirculated to
the bottom at 35000 rpm permitting light particles to get into gradient.
Banding occurs for another 2 h at 35000 rpm.  When stopped, the rotor
is fractionated by collection from the bottom.  1 ml from each fraction
was kept unfrozen, while the rest was deep-frozen.
<p>
Continuous flow ultracentrifugation was pioneered by Norman Anderson in
the 1960s and commercialized by Alfa Wasserman.  Sample is first put
into the unmoving rotor.  Spinning is started and a gradient forms
vertically along the wall.  A continuous flow is provided to allow
particles to enter and move to equilibrium point.  The centrifuge is
stopped without loss of the formed gradient.  A refractometer (Milton Roy)
was used to verify the formation of a gradient (samples could have
been weighed too).  Anti-Tom20 antibody was used in western blotting
of each of the fractions, revealing the mitochondria banded across
about 10 fractions (8-17) from 1.11 to 1.18 g/ml.  Results seem to indicate
that adult mitochondria are slightly more dense than those from young hearts.
<p>
Fractions from the adult and young heart ultracentrifuge spin were then
run on 2D gels (strips from 3-10, 5-8, 7-10) in triplicate.  Replicate
gels showed 144 matching spots between identical fractions from young and adult
in one faction.
For the young, there were 103 unique spots in one fraction, 190 in another.
Significant differences exist in the protein mapping profiles based on age.
<p>
pCFU will separate and enrich on the organelle subtype.  Heart mitochondria
and subtypes were separated based on shape and densities.  Comparisons
were then made on spots between densities but within the same age group,
and the number of matched spots were significantly greater than between
age groups but at similar densities.  NanoLC-ESI-MS/MS was used for selected
spots that differed by at least 2-fold expression.  Proteins found in
Adult fraction 9 and 12 but not in the young bovine animal were
NADH dehydrogenase (ubiquinone) Fe-S 3-30 kDa, desmin, a D-Asp oxidase-like
enzyme, voltage-dependent anion channel 2, thymidine kinase bovine
herpesvirus 4; identified from a pH 5-8 IPG.  Protein in the young bovine
fractions 9 and 12 but not in adult were translation elongation factor
EF-Tu precursor, caldesmon, thymidine kinase, Ig &lambda; light chain,
and enoyl-CoA hydratase precursor.  For proteins present in both with
2-fold differences in both fractions of both ages, these were four
proteins greater in adults:
NADH dehydrogenase (ubiquinone) Fe-S protein 1-75 kDa; S chain S, bovine
heart cytochrome C oxidase at fully reduced state; serum albumin precursor;
casein alpha-S1.  The next four proteins are greater in the young animal:
E Chain E, structure of bovine F1-ATPase covalently inhibited  with
4-chloro-7-nitrobenzofuran; adenylate kinase 2; multicatalytic protease
complex, macropain, proteosome subunit 13 peptide; bovine viral diarrheal
virus.
<!-- ================================================================ -->
<p class="tags">

<p class="ref">
 <span class="authors">
Williams B, Cornett S, Crecelius A, Caprioli R, Dawant B, Bodenheimer B
 </span>
 <span class="year">
(2005)
 </span>
 <span class="jname">
Proceeedings of The ACM 43rd Annual Southeast Regional Conference
 </span>
 <span class="jvol">
1,
 </span>
 <span class="pages">
137-142.
 </span>
<p class="reftitl">
An Algorithm for Baseline Correction of MALDI Mass Spectra
<p class="Para1">
A new baseline correction (background subtraction) algorithm is presented,
particular to compensate for convex shapes that occur to signal in the
low end of the mass range.  The spectrum is first divided into
<b>local areas</b>, and then <i>local areas</i> are divided into
<b>windows</b>.  Then the minima are found in the windows.  The local
area range is changed adaptively to ensure that minima lying on peaks
are not selected.  A function for the set of minima is then derived to
produce the final baseline.
<p>
Windows within local areas are constructed so that a polynomial is
fit through median points.  If too many medians lie above the polynomial,
a peak is detected and the local area expands.  As an example suppose
a local area of 1000 is started at <i>m/z</i> 20,200.  The local area
is divided into 20 windows, each being 50 units wide.  The median signal
is then computed in each of the windows.  Within the local area, a polynomial
line is drawn that fits the median points created in each window.
The count of median points above and below the first iterated line is
determined.  If too many points lie above the line, then the local area
is increased until a percentage of local medians are below the polynomial
or that lie within a distance &epsilon; from the polynomial.  Medians
above the distance &epsilon; are discarded; remaining medians are added
to list of baseline points.
<p>
<b>Finding the Offset</b>.  <ol><li>Define a window size of fixed width
<i>w</i>, to be stepped across the spectrum.  In practice a value of
1000 <i>m/z</i> units averages out the noise fairly well. <li>
Initialize the location of the window <i>w_loc</i> to the start of the
spectrum. <li>Set current intensity <i>curr_int</i> equal to the mean
intensity of <i>m/z</i> values in the first window. <li>Move the
window <i>w</i> m/z values such that <i>w_loc = w_loc</i> + <i>w</i>.
<li>Set previous intensity <i>prev_int</i> = <i>curr_int</i>, and then
set <i>curr_int</i> to mean intensity of the moved window.
<li>Repeat the last previous two steps until <i>prev_int &gt; curr_int</i>,
which means that the previous window contains essentially a peak intensity.
<li>An area to search for the window of highest intensity will be
<i>startsearch = w_loc</i> &minus; 2 &times; <i>w</i>, and the limit
<i>endsearch = w_loc</i>.
<li>A window having width <i>w</i> is started at <i>startsearch</i>.
<li>The window within the search region is moved one <i>m/z</i> unit,
and the mean of intensity values in the window is evaluated.  The window
having the highest mean value is tracked. <li>  The <b>median</b> of the
window having the highest mean intensity is evaluated, and used as the
starting point for the baseline algorithm.</ol>
<p>
<b>Calculating the baseline</b>.  The baseline will follow the determined
median values for each window.  If windows do not span peak widths, the
baseline will be radically shifted or altered, and so to avoid this,
and adaptively expanding window is employed.  This adaptive expansion
is employed in the following algorithm: <ol><li>Set <i>loc_area</i> to the
intial local area size, <i>init_loc_size</i>; it should be as small as
possible but large enough so as not to be far less than typical peak width.
In practice a <i>init_loc_size</i> = 1000 works as a tradeoff between
precision and speed. <li>The local window size <i>loc_wind</i> =
<i>loc_area</i> / <i>num_wind</i> (number of windows).  In practice
<i>num_wind</i> set at 20 is reasonable. <li>Now set <i>loc_area</i>
= <i>num_wind</i> &times; <i>loc_wind</i>. <li>Find <i>m/z x</i> and
intensity <i>y</i> of the medians in the window (<i>x<sub>i</sub></i>,
<i>y<sub>i</sub></i>) where <i>i</i> = 1 to <i>num_wind</i>.
<li>Fit a fourth order polynomial, <i>P</i>, to the medians using least
squares to solve a system of linear questions at points (<i>x<sub>i</sub></i>,
<i>y<sub>i</sub></i>).  The 4th order polynomial compromises speed and
smoothness. <li>Let <i>y<sub>i</sub></i> = <i>P</i>(<i>x<sub>i</sub></i>).
Count the number of points at which <i>y<sub>i</sub></i> &minus;
<i>y&prime;<sub>i</sub></i> &gt; &epsilon;, i.e. the number the medians
computed in step 3 that are above the tolerance.  Use of an &epsilon;
of 5% is reasonable.
 <ol><li>If the medians above the polynomial exceed 25% of the points in
the local area (<i>num_wind</i>), then all median points in the local area
are discarded.  The local area size is then increased by <i>init_loc_size</i>
and step 2 is resumed.  The local area expands until &lt; 25% of point lie
distance &epsilon; above the fitted polynomial line. <li>Otherwise,
add the median points which are &lt; distance &epsilon; above the polynomial
to the set of baseline points, then continue to step 6.</ol>
The value of 25% is selected as it represents a value that ignores points
lying on peaks without compromising speed.
<li>Reset <i>local_area</i> to <i>init_loc_size</i>.  Move to next local
area, and repeat steps 2-5 until end of spectrum is reached.
<li>The baseline is calculated by fitting the exponential
<i>y = ae<sup>bx</sup>
+ ce<sup>dx</sup></i> to the set of data points using least squares.
</ol>
<!-- ================================================================ -->
<p class="tags">

<p class="ref">
 <span class="authors">
Higgs RE, Knierman MD, Gelfanova V, Butler JP, Hale JE
 </span>
 <span class="year">
(2005)
 </span>
 <span class="jname">
J. Proteome Res.
 </span>
 <span class="jvol">
4,
 </span>
 <span class="pages">
1442-1450.
 </span>
<p class="reftitl">
Comprehensive Label-Free Method for the Relative Quantification of Proteins
from Biological Samples
<p class="Para1">
Biological fluid samples for biomarker discovery were studied by LC-MS/MS
using different modalities.  50 &micro;l rat serum is albumin-depleted
(Montage, Millipore) then treated with 2% iodoethanol and 0.5%
triethylphosphine in MeCN in 8 M urea in pH 11 0.1 M ammonium bicarbonate
for reduction-alkylation.  Treated to Worthington trypsin 1:50 (w/w) overnight
assuming 50% protein depletion.  The LC-MS/MS setup is a Surveyor autosampler
with MS HPLC pump (Thermo-Finnigan) with tryptic digests (10 &micro;l)
injected to Zorbax SB300 1 &times; 50 mm C18 RP (Agilent) at 50 &micro;l/min.
with 10-95% B gradient over 2 h with 0.1 min to 100% B, 5 min at 100% C,
0.1 min to 10% B, equilibrate for 17 min.  Solvent A is 0.1% FA, solvent
B is 50% MeCN + 0.1% formic acid, C is 80% MeCN + 0.1%FA.  All effluent
was sent to electrospray on LCQ Deca XP + ion trap MS (ThermoFinnigan)
Source is +4.0 kV, sheath gas 20 units, capillary 150&deg;.  Maximization
of the +2 charge ion of angiotensin was done.  Data collection is triple
play mode:  centroid parent scan with 3 microscans 200 ms maximum injection
time, profile zoom scan with 3 microscans and 200 ms max injection time,
centroid MS/MS with 3 microscans and 400 ms max injection time.
Dynamic exclusion were set to repeat count of one, exclusion list duration
of 2 minutes, and rejection widths of -0.75 <i>m/z</i> and +2.0 <i>m/z</i>.
Relative collision is 35% and exclusion width of 4 <i>m/z</i>.
<!-- ================================================================ -->
<p class="tags">

<p class="ref">
 <span class="authors">
Agrawal GK, Thelen JJ
 </span>
 <span class="year">
(2005)
 </span>
 <span class="jname">
Proteomics
 </span>
 <span class="jvol">
5,
 </span>
 <span class="pages">
4684-4688.
 </span>
<p class="reftitl">
Development of a Simplified, Economical Polyacrylamide Gel Staining
Protocol for Phosphoproteins
<p class="Para1">
The Pro-Q Diamond phosphoprotein stain for gels from Molecular Probes
is quite costly.  The procedure given by the manufacturer has been
modified after 1 year in production.  The modification was essentially
to the fixation (recommended now to be overnight, as opposed to
90 min-to-overnight).  The latest revision indicates use of a fix
that is 50% MeOH/10% acetic acid as opposed to 45% MeOH/5% acetic acid.
Changes to destaining and washing protocols are also modified in the
revisions.  All these continuing revisions suggest that optimization is
still an open question.  Although various destains have been reported,
the best ones contain MeCN which help to remove background quickly.
The latest and best destain is 20% MeCN in 50 mM sodium acetate, pH 4.0,
which is done four times for 30 min each.
<p>
Peppermint Stick phosphoprotein MW markers from Molecular Probes were
loaded in serial 2-fold dilutions from 1000 down to 2 ng and run
on a 1D gel; the markers have two phosphorylated proteins (ovalbumin
and &beta;-casein at 45 and 23.6 kDa, resp.) and four nonphosphorylated
proteins (&beta;-galactosidase, BSA, avidin and lysozyme at 116, 66.2,
18.0, and 14.4 kDa).  It was found that a 3-fold dilution of the recommended
ProQ Diamond stain still gave a fluorescence intensity equivalent to the
rest:  there was no change in sensitivity, fluorescence intensity or linear
dynamic range.  Staining time was also checked (2, 6, 12 h) and staining
longer than 2 h did not alter the results;  this is unlike many typical
total protein stains in which longer incubation times typically improve
the results because the stain has different kinetics with different amino
acids.  A time course was done on fixation (1, 6, 15 h) and 1 h was found
sufficient.  The standard (manufacturer) and modified protocols were compared
using the markers ovalbumin and &beta;-casein, and plots of fluorescence
vs. protein amt were virtually identical.
<p>
The advantages of the modified protocol (diluted stain reagent) are:
(1)
<!-- ============================================================= -->
<p class="tags">
FD
<p class="ref">
 <span class="authors">
Cooper TJ, Talaty ER, Van Stipdonk MJ
 </span>
 <span class="year">
(2005)
 </span>
 <span class="jname">
J. Am. Soc. Mass Spectrom.
 </span>
 <span class="jvol">
16,
 </span>
 <span class="pages">
1305-1310.
 </span>
<p class="reftitl">
Novel Fragmentation Pathway for CID of (b<sub>n</sub> &minus; 1 + Cat)<sup>+</sup>
Ions from Model, Metal Cationized Peptides
<p class="Para1">
<b>Short Comm.</b>.  The peptides AAAG and A(&beta;A)AG were ionized in presence
of Li<sup>+</sup> in ESI-MS/MS with CID.  Prominent in the product spectra were
ions at m/z 238 [<i>b</i><sub>3</sub> + 17 + Li]<sup>+</sup>, 220
[<i>b</i><sub>3</sub> &minus; 1 + Li]<sup>+</sup>, and 192 [<i>a</i><sub>3</sub>
&minus; 1 + Li]<sup>+</sup>.  Peaks at m/z 277 and 260 corresponded to neutral
loss of H<sub>2</sub>O (18 amu) and a combination of H<sub>2</sub>O and
NH<sub>3</sub> loss (35 amu) were also found for AAAG but not the
&beta;A-containing peptide.  Loss of H<i>2</i>O in combination
with NH<sub>3</sub> or not might occur by nucleophilic attack of <i>N</i>-terminal
carbonyl oxygen on the <i>n</i> + 1 carbonyl carbon.  Absence of these two reaction
products for the &beta;A peptide consistent with mechanism in which cyclization
and intramolecular bonding involves less kinetically and entropically favored
6-member ring and with earlier study of influence of amino acids on generating
[<i>b</i><sub>n</sub> + 17 + Cat]+ products.
<p>
The [<i>b</i><sub>3</sub> &minus; 1 + Li]<sup>+</sup> from AAAG and A(&beta;A)AG
is the lithium-cationized oxazolinone.  The most prominent dissociation
pathway for [<i>b</i><sub>n</sub>]+ and [<i>b</i><sub>n</sub> &minus; 1 + Cat]<sup>+</sup>
is CO elimination to produce [<i>a</i><sub>n</sub> &minus; 1 + Cat]<sup>+</sup>.
A scheme shows the mechanism by which that can be accomplised.  MS<sup>3</sup>
spectra are presented with the <i>m/z </i>220 [<i>b</i><sub>3</sub> &minus; 1 + Li]<sup>+</sup>
presented.  Most abundant was m/z 192 [<i>a</i><sub>3</sub> &minus; 1 + Li]<sup>+</sup>
for both AAAG and A(&beta;A)AG, being a loss of CO (28 amu).  Additionally for
AAAG peptide showed an <i>m/z</i> 149 ion while A(&beta;A)AG showed a <i>m/z</i> 148,
losses of 71 and 72 amu, respectively.  The 72 amu is equivalent to loss of a
C-terminal Ala (71 amu) plus one.  We compared 4 peptides with the motif AxAG
where x = &beta;-Ala, &gamma;-aminobutyric acid, &epsilon;-amino-n-caproic acid,
and 4-aminomethylbenzoic acid.  In all cases, the [<i>b</i><sub>3</sub> &minus; 1 + Li]+
had an increasing tendency to lose 72 amu and less so the 28 amu CO.
<p>
The peptide motif A(&epsilon;-Cap)-x-G was tried, where x was Val, Gly,
Gly&minus;1&minus;<sup>13</sup>C, and Gly&minus;<sup>15</sup>N.  For Val, there
was a 100 amu loss product ion (= mass Val + 1) at m/z 190; for Gly, there as a 58 amu
loss product (= mass Gly + 1) at m/z 190 and a -44 amu loss ion (m/z 204).
These ions were also found for the Gly&minus;1&minus;<sup>13</sup>C.  With the
<sup>13</sup>C isotope, neutral losses were 29, 45, and 59, namely all +1 for each
of the product ions, which indicates its loss and which carbon is affected.  For the
<sup>15</sup>N-labeled glycine, the dominant product ion shifted by 1 to m/z 149,
which indicates the labeled nitrogen is retained in the product.
<p>
For the m/z 262 (A-&epsilon;Cap-AG) to m/z 190 transition, formation of a nitrile
with loss of 3-membered ring CH3CH(O)C=O &alpha;-lactone.  Formation of &alpha;lactones
was suggested as a dissocation pathway for Ag<sup>+</sup>-cationized phenylalanine.
Other product ions include -44 and -45 amu ions (latter in the <sup>13</sup>C-labeled
amino acids.  This suggests a neutral loss of CO<sub>2</sub> from the oxazolinone
ring.  The m/z 218 ion that forms might by a 2<i>H</i>-azirine (3-membered) ring.
This 44 amu loss is seen only for AxGG peptides, in which the stability of the
neutral &alpha;-lactone is the key to the formation of a nitrile product.
2<i>H</i>-azirines are more stable than anti-aromatic 1<i>H</i>-azirines;
2<i>H</i>-azirines can be isolated at low temperature, whereas &alpha;-lactones
cannot be.
<p>
Thus the [<i>b</i><sub>3</sub> &minus; 1 + Li]<sup>+</sup> ion fragments
in several ways: (1) loss of &alpha;-Ala residue mass to produce
[<i>b</i><sub>2</sub> &minus; 1 + Li]<sup>+</sup>, (2) CO neutral loss
to produce [<i>a</i><sub>2</sub> &minus; 1 + Li]<sup>+</sup>, (3) ejection
of an &alpha;-lactone to produce lithiated nitrile, (4) elimination of
CO<sub>2</sub> to produce Li-cationized 2H-azirine.
<!-- ============================================================= -->
<p class="tags">
FD
<p class="ref">
 <span class="authors">
Wells JM, McLuckey SA
 </span>
 <span class="year">
(2005)
 </span>
 <span class="jname">
Meth. Enzymol.
 </span>
 <span class="jvol">
402,
 </span>
 <span class="pages">
148-185.
 </span>
<p class="reftitl">
Collision-Induced Dissociation (CID) of Peptides and Proteins
<p class="Para1">
Most commonly used activation method in tandem MS of peptides and proteins
is energetic collisions with neutral target gas (rather than <q>neutral</q>
better to use <q>chemical non-reactive</q>?).  This is called <em>collision-induced
dissociation</em> (CID).
<p>
When J. J. Thomson was undertaking the first studies of mass spectrometry
he observed gas-phase CID in his parabola mass spectrograph.  Summary
of early work on CID was published (Cooks, 1995).  Closely related to
CID is <b>metastable ion dissociation</b>.  Tandme MS has been very useful
in identification and characterization of ions and complex mixture analysis
where collision product product ions with understanding of fragmentation
theory.  Other dissociation methods include surface-induced dissociation
(SID) and photo-induced dissociation (PID) but CID is far easier to implement.
20 years ago the emergence of both MALDI and ESI was reported for biomolecule
analysis.
<p>
Use of technique has outpaced the understanding of its mechanisms.  The
energy transfer mechanisms that operate during collisions of large (&gt; 1000 Da)
possibly multiply charged with small target gas atoms or molecules are
not as well understood as they are for smaller ions.
<p>
Figures of merit for CID are outlined here of how peptides/proteins dissociate
as function of CID conditions uses:
<ol>
<li>Time over which activation occurs relative to time for unimolecular
dissociation  or rearrangement
<li>Amount of energy deposited during activation
<li>Distribution of deposited energy
<li>Variability of deposited energy
<li>Form of deposited energy (vibrational vs. electronic)
<li>Time scale of instrument used (kinetic window within which dissociation
reactions must occur in order to be observed)
<li>Efficiency of CID process in terms of cross section of rate constant
</ol>
<p>First figure of merit, time scale of activation, refers to how fast energy
is deposited in ion relative to how fast energy equilibrates through degrees
of freedom of ion, how fast it dissociates or rearranges, and how fast energy
is removed by cooling processes such as deactivating collisions or photon
emission.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Salih E
 </span>
 <span class="year">
(2005)
 </span>
 <span class="jname">
Mass Spect. Rev.
 </span>
 <span class="jvol">
24,
 </span>
 <span class="pages">
828-846.
 </span>
<p class="reftitl">
Phosphoproteomics by Mass Spectrometry and Classical Protein Chemistry
Approaches
<p class="Para1">
<b>REVIEW</b>.  Generally phosphorylation occurs at a low level, perhaps
too low to be observed without sensitive methods such as radiolabeling.
Use of pulse-chase experiments are informative for cell/tissue culture
based experiments with <sup>32</sup>P.  <sup>32</sup>P-&gamma;-ATP mixed
with cell lysates or in the presence of purified or enriched protein kinases
can be used.  Proteins are isolated, identified, digested and radiolabeled
peptides separated by TLC or HPLC.  N-terminal sequence analysis is then
done using Edman method.  This was the classical approach to identifying
phosphorylated residues.  A problem is that P-Ser residues degrade in the
Edman method.
<p>
Meyer et al. (1993) used a method in which they achieved the &beta;-elimination
of phosphate from P-Ser and P-Thr residues with alkaline medium and then
add a thiol (e.g., ethanethiol).  The peptide is marked and stabilized and
then isolated by the usual methods above (digestion, resolution of peptide
by chromatography, then Edman sequencing).  Unfortunately, such methods are
quite insensitive.  Sensitivity can be improved however when radiolabeled
thiol agents (e.g, 1-S-[<sup>3</sup>H]carboxymethyl-dithiothreitol, or
[<sup>3</sup>H]CM-DTT).  This reagent can actually enable qualitative,
and relative and absolute quantitation of proteins for cell-labeling studies.
In addition non-labeled form of CM-DTT and its precursor DTT were used
to study P-Ser and P-Thr-containing phosphoproteins analyzed by MALDI-MS.
Note that in using thiols, a mono- and di-thiol approach might be taken.
Dithiols present the possibility of linking other components (affinity
tags like biotin, or thiol-reactive radiolabeled components such as
deuterated iodoacetamide or ICATs (also deuterated).  Recently ECM
phosphoprotein studies led to a rapid ID of the complete topographical
distribution of in vivo and in vitro phosphorylation sites of bone
sialoprotein (BSP).  All the phosphorylation sites were determined using
a combination of approaches:  [<sup>32</sup>P]ATP in vitro using Edman sequencing,
[<sup>14</sup>C]-CM-DTT after base-catalyzed elimination and Edman sequencing,
and MALDI-MS.  It is not enough to identify phosphorylation but it is
essential to indicate the number of moles P per mole peptide per site.
<p>
A monophosphorylated peptide and its unphosphorylated form will differ in
mass by exactly 80 Da.  When doing peptide mass fingerprinting, the assumption
of a phosphorylated peptide can be factored in during database searching for
the purpose of identification.  Moreover, the report comes back that the
ID was made because phosphorylation was assumed.  In this way, a protein is
marked as potentially phosphorylated.
<p>
Immobilized-metal affinity columns (IMAC) with Zn<sup>2+</sup>, Fe<sup>3+</sup>,
and Ga<sup>3+</sup> at the center have a particular affinity for phosphate.
Peptides can be run over the column and the many bound peptides not washed
off are likely to be phosphorylated.  Unfortunately the affinity for phosphate
is not all that specific:  peptides with a high number of acidic,
histdine, and cysteine residues also bind.   If analyzed on MALDI-MS, a
phosphatase-treated sample will also need to be run to look for peak shifts
that are relevant.  Alternatively direct sequence analysis may be necessary.
However a recent IMAC approach by Ficarro et al (2002) eliminated nonspecific
binding, with a large number of phosphopeptides site-identified.
<p>
Radiolabeled proteins have also been 2-D gel-separated followed by autoradiography.
These proteins can be excised, digested and MALDI-MS-analyzed.
<p>
Additionally high-affinity anti-P-Tyr antibodies can be used to enrich P-Tyr-containing
proteins and peptides.  It is less effective for P-Ser and P-Thr-containing
proteins and peptides.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Vogel RF, Pavlovic M, H�rmann S, Ehrmann MA
 </span>
 <span class="year">
(2005)
 </span>
 <span class="jname">
Brazilian J. Med Biol. Res.
 </span>
 <span class="jvol">
38,
 </span>
 <span class="pages">
1247-1252.
 </span>
<p class="reftitl">
High pressure-sensitive gene expression in <i>Lactobacillus sanfranciscensis</i>
<p class="Para1">
High hydrostatic pressure (HHP) is used in food processing is intened to
inactivate (kill) bacteria and viruses without loss of the desirable nutritent
or aesthetic properties.  Bacteria in  their vegetative form are usually killed,
but those in (endo)sporulating conditions resist.  Kinetics is such that a
small <q>resistant</q> fraction may survive HHP. <i>Lactobacillus
sanfranciscensis</i> DSM 20451<sup>T</sup> is grown at 30&deg; in mMRS pH 6.1
medium and cells harvested in exponential phase.  Cells are transferred to
fresh 30&deg; medium and subjected to pressuures of 80 MPa for 60 min,
at pressure rate increase of 30 MPa/min.  2D gels were run in pH 3-10, 4-7,
4.5-5.5 and 5.5-6.5.  MALDI-MS and N-terminal sequence was provided through
a service (TopLab).  In addition genomic DNA was isolated by Lewington et al
using DSM 20451 strain; shotgun libraries were made in E coli DH5&alpha;
using <i>E. cloni</i>&trade; and plasmid psmart-HCAmp (Lucigen) using partial
digest (BsuRI, AluI) and sonicated DNA.  Random sequencing of 100 inserts and
inserts of 2000 clones were PCR-amplified and spotted to silylated microscope
slides to form array.  Hybridizations were done according to Huang et al with
cDNAs labeled with Alexa Fluor 555 from untreated cells and AlexaFluor 647
from pressure-treated cells.
<p>
Five experiments to grow and pressure-treat cells were performed, and proteins
from them run on 2DE.  16 proteins were identified as having their expression
altered by pressure;  of these 9 showed 2- to 12-fold increase while 7
proteins were reduced by 50% or more.  N-terminal sequencing was done on
internal peptides of four proteins.  BLAST analysis showed two proteins P1 and
P2 had homology to ribokinase and ATP-dependent protease ClpL.  Sequences of
P9, P13, and P16 showed homology to general stress protein GroEL (hsp60),
EF-Tu, and GA3PDH, respectively.  "Reversed genetics" was used to find and code
genes.  P1 is a 540 amino acid proteiin coded by 1620 bp segment with calculated
mass of 58 kDa, larger than any known ribokinase.  The first 308 aa show
homology to ribokinase and the latter 232 to ribose-5-phosphate isomerase.
<p>
DNA microarrays were used to analyze transcriptome.  750 spots were judged
quality spots and showed that mRNA levels of genes or operons was affected in 48.
6% of genes showed 2-fold or more induction, and 1% showed repression at 2-fold
or more.  The phosphoketolase <i>xpk</i> gene showed no affect of HHP and
was used in reference for using real-time PCR.  Real-time PCR was used as an
independent confirmation of the microarray data.  The most upregulated gene
was an ORF with a DEAD-ATP-dependent helicase motif (3.9-fold increased)
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Eiceman GA, Karpas Z
 </span>
 <span class="year">
(2005)
 </span>
 <span class="pages">
2nd Ed., (Boca Raton: Taylor &amp; Francis)
 </span>
<p class="reftitl">
Ion Mobility Spectrometry
<p class="Para1">
<b>BOOK</b> (350 pages).
Ion mobility spectrometry (IMS) involves characterizing chemicals substances
by the velocity of their gas-phase ions in an electric field.  A <q>swarm</q>
of ions are introduced to voltage gradient or electric field (V/cm) and attain
a constant <b>drift</b> velocity (cm/s) at ambient pressure in a gas, usually
air, and the velocity is proportional to electric field:
<p class="eqnserif">
<i>v</i><sub>d</sub> &prop; <i>E</i>, <i>v</i><sub>d</sub> = <i>kE</i> 
<p>
where <i>k</i> has units of cm<sup>2</sup> / [V-s].  Velocity applies
only to swarm and not to individual ions.  For swarms of ions from 14 to 500
amu, velocities are 1-10 m/s in electric fields of 150-300 V/cm at temps from
25-250&deg; C.  Mobility coefficients would be 0.8-2.4 cm<sup>2</sup>/V-s
and are normalized to T=273 K and P=760 torr:
<p class="eqnserif">
<i>k</i><sub>0</sub> = <i>k</i>(273/<i>T</i>)(<i>P</i>/760)
<p>
A sample is put in the gas phase into a portion of the <b>drift tube</b>
called the <b>reaction region</b>.   In this region, some of the sample
molecules get ionized.  An <b>ion shutter</b> is activated to draw the
ionized (not neutral) molecules into the <b>drift region</b> part of the tube.
In this they have a constant velocity and strike a detector at the other
end.  Drift tubes have a 4-20 cm length usually.  At ambient pressure and
temperature, a swarm with <i>k</i><sub>0</sub> = 2 cm<sup>2</sup>/V-s 
moving at 4 m/s in a E=200 V/cm field should take 15 ms to complete travel time
when the tube is 6 cm long.  The <b>mobility spectrum</b> is often presented as detector
response (current or voltage) vs. drift time (in milliseconds).  When ions
collide with detector, they become neutralized, causing a current flow of
10-1000 pA, which becomes amplified and transformed to a 0-10 V full range.
<p>
The mobility spectrum often contains selectivity that includes fragmentation
and peak shape (performance measure).
<p>
Original ionization sources were 10 mCi <sup>63</sup>Ni beta electron emittes 
with 17 keV mean energy.  These collide and ionize atmosphere gas and release 
secondary electrons.  Ion-molecule reactions with N<sub>2</sub>, O<sub>2</sub>, 
and H<sub>2</sub>O vapor causing formation of 
H<sup>+</sup>(H<sub>2</sub>O)<sub>n</sub> and 
O<sub>2</sub><sup>&minus;</sup>(H<sub>2</sub>O)<sub>n</sub>.  These are 
reactant ions which then ionize sample (M) ions colliding with them.
The reaction becomes part of a cluster that includes displacement.
<p class="eqnserif">
M + H<sup>+</sup>(H<sub>2</sub>O)<sub>n</sub> &rarr; 
MH<sup>+</sup>(H<sub>2</sub>O)<sub>n</sub> &rarr;
MH<sup>+</sup>(H<sub>2</sub>O)<sub>n&minus;x</sub> + x H<sub>2</sub>O
<p>
The cluster ion might be stabilized by collision with another M, or third
body collision usually from supporting atmosphere, with loss of water adducts.
<p>
<b>History</b>.  From 1850 to 1938 represents the period of innovation and
discovery.  Early researchers found that conductivity in air developed by
formation of electrons and gaseous ions.  X-rays focused on gases made them
ionized: Rutherfored measured mobility of ions formed by x-rays within two
years of the discovery of x-rays.  Additionally, radioactivity and ionization
by flame and UV light could be observed.  By 1938, there was much data on
ions formed in gases and their mobilities, and effects of temperature, pressure
and gas purity.  Also clusters of ions formed with polar neutral molecules
was studied.   For dried gases, positive ions had an envelope of molecules.
For negative ions, mobilities were altered when placed in water vapor.
<p>
Achievements made included Langevin's findings on the collisional nature
of mobility, and role of attractive forces on effective collision cross 
sections.  Another important advance was pulsed injection of ions using a
shutter; today a linear field with two-shutter design would be employed.
Additionally an understanding of electric fields and gas pressure on mobility
of ions in gases was important.  For example, mobility was found to be
independent of the E/N (field voltage/gas pressure) ratio up to several 
<b>Td</b> (1 Td = 10<sup>&minus;17</sup> V/cm<sup>2</sup>); beyond 2-4 Td,
mobility becomes dependent on E/N.
<p>
In 1948 Lovelock was studying if breezes were related to human illness so he
developed a <q>vapor anemometer</q> to look at changed in wind direction or
speed.  A radioactive source ionizes the air space between two electric field
plates or grids, and changes in the current that might be caused by a breeze
or wind can be detected.  Some of this technology became part of electron
capture detection (ECD).
<p>
Gas phase ion-molecule reactions at elevated (atmospheric) pressures were
an interest in 1950s experiments looking at radiolysis of air, and in reactions
at Earth's upper atmospheric layers.  These studies were facilitiated by 
identifying ions produced in flames and corona discharged using mass spectrometers.
Shahin at Xerox sampled ions in corona discharge using pinhole orifice of
MS to look at chemistry of photocopiers and laser printers.  This occurred
at same time as study of ion-molecule reactions in chemical ionization mass spectrometry.
Kebarle looked at reactions and kinetic of these reactions at elevated pressures.
<p>
Drift tube design was also changed.  One was a stack of electrically isolated
rings with linear electric field established by voltage divider.  They created
a system called Plasma Chromatography.  IMS and ion chemistry in air at
ambient pressure has complicated charge-transfer reaction and ion cluster
formations.  While it might be fused with an MS, the mobility spectrometer
did not have much analytical value.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Cottingham K
 </span>
 <span class="year">
(2005)
 </span>
 <span class="jname">
Anal. Chem.
 </span>
 <span class="jvol">
77,
 </span>
 <span class="pages">
227A-231A.
 </span>
<p class="reftitl">
Versatile TOFMS
<p class="Para1">
<b>PRODUCT REVIEW</b>.  TOF instruments are superior in exact mass 
determinations which solve problems and determine structures.  Accurate mass 
measuments to four decimal places narrow down identities of unknowns and can set 
elemental compositions.  TOFs are now choices for biomarker discovery where as 
FTMS, sector, and triple quads had been used earlier.  
<p>
<b>Reflectrons</b>.  With the use of reflectrons (<q>ion mirrors</q>) resolution 
can be improved, but that an intended effect is to <q>refocus ions of different 
energies.</q>  The analogy is of the bicycle race:  really fast guys missed the 
turn, had gone too far, then turned around.  The slower ones made the turn, and 
so at one point the fast and slow racers are together for just a moment, which 
is how the reflectron works.  Reflectrons can actually work as single- or 
dual-stage reflectons, which are one- or two-constant field regions separated by 
grids.  Quadratic and curved field reflectrons have more complex fields. 
Dual-stage reflectrons have higher mass resolution than single stage, but the 
single stage can focus a broader range of energies.  The curved field reflectron 
is similar to the single stage but has a higher bandwidth; it is efficient as 
focusing fragment ions.  Reflectrons usually have grids, but there are gridless
reflectrons;  grids show a change in field at the end of the drift tube similar
to hitting a curb, but gridless systems are not at any advantage since ions can
deviate if entering at an angle or off-axis.
<p>Since TOF relies on a pulse, it is often mentioned with MALDI.  These are 
thus axial TOF systems.  With orthogonal TOF systems, a continous stream of 
ions as produced by electrospray can be sampled into a ToF.  However, atmospheric
pressure MALDI can now be used, in which formed ions enter the MS in a continuous
stream and are orthogonally sampled.
<p>
Some ToF systems have the ability to rapidly alternate between ESI and APCI modes
such at the Water's LCT Premier.  Agilent's LC/MSD ToF system simultaneously
ionizes using ESI and APCI.  JEOL recently pioneered Direct Analysis in Real Time
(DART)
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Simpson DC, Smith RD
 </span>
 <span class="year">
(2005)
 </span>
 <span class="jname">
Electrophoresis
 </span>
 <span class="jvol">
26,
 </span>
 <span class="pages">
1291-1305.
 </span>
<p class="reftitl">
Combining Capillary Electrophoresis with Mass Spectrometry for Applications
in Proteomics
<p class="Para1">
<b>REVIEW</b>.  In capillary electrophoresis (CE), the techniques with the 
greatest potential are capillary zone electrophoresis (CZE) and capillary 
isoelectric focusing (CIEF).  In interfacing to MS, the predominant is by ESI,
although MALDI has been used too.  The CE-separated interface usually requires
supplemental fluid, if only to make it compatible with ESI since background
electrolyte (BGE) for CZE may not be ideal.  However, when no supplemental fluid
is required, sensitivity is clearly higher.  A problem occurs with differences
in BGE and supplemental fluid, coupled with low EOF (typical for peptide and
protein separations) in which moving ionic boundary complicates separation 
process.  The first attempts did not use supplemental fluid, but now most do
with a design of the sheath flow made by Smith et al (1988).  Lee et al introduced
a liquid junction interface.  When ESI current is greater than CZE current there
is optimum sensitivity: this is achieved by using as narrow possible capillaries
in the CE phase.  Early uses of CE with no sheathing used capillaries tapered by
40% HF etching and then silvered to convert CE terminus to ESI emitter; SF<sub>6</sub>
is used to suppress corona discharge.  This achieved resolution of tryptic digests
of cytochrome <i>c</i> of bovine, equine, and Candida krusei type with capillary
of 10 &micro;m i.d. and 150 &micro;m o.d.  In another design, the capillary tip
is ground and the tip coated with gold to make an electrical connection, which
was tested using protein standards and shown to have 100 h lifetime on the ESI
emitter.  While greater narrowing of the tip improve ESI performance, a back
pressure builds up that results in mixing.
<!-- ================================================================ -->
<p class="tags">

<p class="ref">
 <span class="authors">
Monti M, Orr&uacute; S, Pagnozzi D, Pucci P
 </span>
 <span class="year">
(2006)
 </span>
 <span class="jname">
Biosci. Rep.
 </span>
 <span class="jvol">
25,
 </span>
 <span class="pages">
45-56.
 </span>
<p class="reftitl">
Interaction Proteomics
<p class="Para1">
<b>REVIEW</b>.

Genome sequencing projects are leading to the prediction of coded sequences 
(proteins) for which if they exist have unknown function.  Much protein 
identification is done using classical approaches such as 1D/2DGE-MALDI-TOF-MS 
and/or 2DLC-ESI-MS/MS.  Functional approaches use a bait-prey approach. 
Alternatives to two-hybrid system including cloning the protein with a tags both 
large and small (GST as large;  FLAG, hemaglutanin (HA), c-myc, poly-His-tail, 
or biotinylated).  Tagged bait then immobilized to agarose beads derivatized 
with the anti-tag ligand (glutathione, anti-epitope Abs, nickel ions, 
streptavidin, etc).  Entire cellular extract or extracts of specific organelles 
(when appropriate) ae incubated with immobilized bait.  After washing away 
unbound material, interacting proteins are elutded, fractionated on SDS-PAGE, 
stained and examined by MS for protein ID.  This was used for ZnF224, and 82 kDa 
Zn finger belong to Kr&uuml;ppel-like Zn-finger proteins family (KRAB), one of 
largest classes of transcription factors.  Protein has 45-aa box A of 
Kr&uuml;ppel-associated box (KRAB) domain at N-terminus, an evolutionarily 
conserved doman, and 19 Cys<sub>2</sub>His<sub>2</sub> Zn-finger motifs at 
C-terminus.  The protein binds to AldA-NRE negative regulatory element, found
in human aldolase A gene promoter; it inhibits transcription throuhg the binding 
to the element DNA by the 45-aa KRAB-A domain with Zn fingers.  It is presumed
that interacting proteins would inhibit or regulate the activity of ZnF224.
Full-length cDNA of ZnF224 and a deletion mutant ZnF224-M1 (only 8 Zn fingers)
were fused to GST C-terminus and purified on glutathione-Sepharose.  this
bait is then linked to GSH-activated beads.  Nuclear 293 cell
extracts are incubated with GST-beads which retains proteins nonspecifically
interacting with GST-beads and matrix.  This pre-incubated extract is then 
incubated with GST-ZnF224 immobilized bait.  Several washes are done, the beads
eluted, then run on SDS-PAGE followed by colloidal Coomassie staining.  Samples
of extracts purified once and twice with GST-bait as well as samples preincubated
with GST only and also the unbound protein fraction were analyzed on gel.
A problem occurs when both the sample and controls identical bands.  Since total
cellular extracts are used, this creates non-physiological interactions of
proteins normally segregated in different organelles.  Another problem is that
bait-prey interactions occur in vitro, so there is always a possibility that
interactions are not physiological.
<p>
Using another fishing strategy, randomized oligonucleotides immobilized on
a support can be used to bind to any protein to the NRE of aldolase gene
promoter, eventually identified as ZnF224.  To confirm the specificity
recombinant wild-type ZnF224 and deletion mutants were prepared and incubated
with AldANRE in band shift experiments, where the wt caused shifting and 
deletion mutants did not.  A CAT reporter was set up in a plasmid using 
ZNF224 with tow AldA-NRE upstream and the expression of ZNF224 caused
negative modulation of the CAT reporter in dose-dependent manner.
<p>
<b>Immunoprecipitation</b>.  To answer questions involving fishing strategies,
the gene for the bait is modified to include epitope tags to commonly available
antibodies (FLAG, HA, c-myc, etc) then transfected into cognate host cell.
As protein complexes form naturally within the cell (not in vitro), cell 
extracts are produced and immunoprecipitated with anti-tag antibodies bound
to solid phase.  These are then run on SDS-PAGE followed by protein ID by MS.
This strategy was used

<!-- ================================================================ -->
<p class="tags">

<p class="ref">
 <span class="authors">
Shen H, Cheng G, Fan H, Zhang J, Zhang X, Lu H, Liu C, Sun F, Jin H,
Xu X, Xu G, Wang S, Fang C, Bao H, Wang Y, Wang J, Zhong H, Yu Z, Liu Y,
Tang Z, Yang P
 </span>
 <span class="year">
(2006)
 </span>
 <span class="jname">
Proteomics
 </span>
 <span class="jvol">
6,
 </span>
 <span class="pages">
528-537.
 </span>
<p class="reftitl">
Expressed Proteome Analysis of Human Hepatocellular Carcinoma in Nude
Mice (LCI-D20) with High Metastasis Potential
<p class="abstract">
<b>ABSTRACT</b>.
We report for the first time an expressed proteome for human hepatocellular carcinoma (HCC)
in nude mice model. Most cases of human liver cancer are HCC with highly metastatic ability.
Therefore, the early prediction or diagnosis and effective treatment are the key points of research.
We have previously successfully established a human HCC nude mice model (LCI-D20) with
high metastasis potential. To understand better the tumor biology of HCC it is worth to explore
the relativity of all expressed protein profiles in the LCI-D20 HCC nude mice model. With
advanced proteomics technologies, we have carried out a proteomic analysis with following
stages: protein sample preparation of cancer tissue, including total cellular extraction and
sequential fractionation, 2-DE and 2-D LC separation, ESI/MALDI-MS/MS identification, as well
as data-dependent bioinformatics. The identified proteins were classified bioinformatically respective
to their function, biological process and intracellular localization. Some important proteins
found in HCC, e.g. metabolism enzymes, proteins regulating cell motility, signaling proteins,
and heat shock proteins, are discussed in terms of their metastasis.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Saha RP, Bahadur RP, Chakrabarti P
 </span>
 <span class="year">
(2006)
 </span>
 <span class="jname">
J Proteome Res.
 </span>
 <span class="jvol">
4,
 </span>
 <span class="pages">
1600-1609.
 </span>
<p class="reftitl">
Interresidue contacts in proteins and protein-protein interfaces and
their use in characterizing the homodimeric interface
<p class="abstract">
<b>ABSTRACT</b>.
The environment of amino acid residues in protein tertiary structures
and three types of interfaces formed by protein-protein
association--in complexes, homodimers, and crystal lattices
of monomeric proteins--has been analyzed in terms of the
propensity values of the 20 amino acid residues to be in
contact with a given residue. On the basis of the similarity
of the environment, twenty residues can be divided into nine classes,
which may correspond to a set of reduced amino acid alphabet.
There is no appreciable change in the environment in going
from the tertiary structure to the interface, those participating
in the crystal contacts showing the maximum deviation.
Contacts between identical residues are very prominent in
homodimers and crystal dimers and arise due to 2-fold related
association of residues lining the axis of rotation. These
two types of interfaces, representing specific and nonspecific
associations, are characterized by the types of residues that
partake in "self-contacts"--most notably Leu in the former and
Glu in the latter. The relative preference of residues to be
involved in "self-contacts" can be used to develop a scoring
function to identify homodimeric proteins from crystal structures.
Thirty-four percent of such residues are fully conserved among
homologous proteins in the homodimer dataset, as opposed to only
20% in crystal dimers. Results point to Leu being the stickiest
of all amino acid residues, hence its widespread use in motifs,
such as leucine zippers.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Sun W, Gao S, Wang L, Chen Y, Wu S, Wang X, Zheng D, Gao Y
 </span>
 <span class="year">
(2005)
 </span>
 <span class="jname">
Molec. Cell Proteomics
 </span>
 <span class="jvol">
XX,
 </span>
 <span class="pages">
xxxx-xxxx.
 </span>
<p class="reftitl">
Microwave-Assisted Protein Preparation and Enzymatic Digestion in Proteomics
<p class="Para1">
BSA was digested with trypsin using different protocols.  The standard
protocol involves 10 mM DTT reduction at 57&deg; for 1 h, alkylation with
55 mM iodoacetamide for 30 min, then 16 h digestion with 0.5 &micro;M trypsin
in 25 mM ammonium bicarbonate.  Variations on the use and timing of these
steps were tried, including MAPED, or microwave-assisted protein
enzymatic digestion.  In MAPED, a container with 1.0 L water is placed
beside sample vials to act as a sink for most of the microwave radiation
of an 850 W device.  The oven is turned on for 1 minute.
<p>
When the spectra of the standard and MAPED protocols are taken, they are
nearly identical, indicating that 1 min of microwave energy can substitute
16 hours of digestion time!  Attempts to remove or eliminate use of
the Cys-modifying protocols were tried: elimination of iodoacetamide
alkylation, and decreasing the DTT reduction time to 5 min but in boiled
water while also
not doing alkylation gave similar spectra to the standard and the
MAPED with DTT/iodoacetamide with 60/30 min treatments.  1DLC-ESI-MS/MS
showed that the count of MS/MS spectra were similar for the 4 samples
injected at 100 fmol.  A sample that completely eliminated DTT reduction
however showed significant reduction in MS/MS spectra.
<p>
To ensure that this was not limited to BSA as the specific protein treated,
a human urinary protein mixture was tested.  Proteins were run on a 1D
tricine gel and visualized before and after standard and MAPED digestion
treatment.  Under MAPED, most proteins above 14 kDa disappeared and some
new bands appeared below 14 kDa.  Some proteins had not fully digested,
being at 31 kDa or so.  In 2DLC-ESI-MS/MS the count of proteins identified
using &ge; 2 MS/MS spectra were 153 for the standard protocol, and 139
for MAPED, with 2333 and 2541 MS/MS spectra for standard and MAPED treatments
respectively.  Thus the MS/MS spectra were lower for MAPED by 8.5%, but
this was not significant.  Similar results with respect to protein identification
were see with 1D tricine gel or 2DLC-ESI-MS/MS.  As the gel showed a
difference in efficiency between MAPED and standard protocol for some
proteins, other variables were tried: prolonging reduction from 5 to 30 min,
increasing digestion from 1 to 15 min, digesting with Lys-C then trypsin,
but there was no improvement.  Hence MAPED probably obtained the same
digestion efficiency.
<p>
With proteins in solution, MAPED works, but what about in-gel?   BSA
was run on SDS-PAGE and then subjected to digestion in-gel using standard
protocol for 19.5 h digestion time and MAPED for 25 min.  Extracted peptides
were then run on 1DLC-ESI-MS.  Previously it was reported that only 50%
of peptides could be recovered.  Identified MS/MS spectrum number from
300 fmol BSA by 1DLC-ESI-MS should be about 20;  in this study the standard
protocol generated about 11 spectra (55%), similar to a previous report.
With MAPED, about 14 spectra (70%) were recovered.
<p>
A yeast lysate was then run, which shows a good 5 thick bands which can be
cut and prepared in-gel.  Proteins with more than three MS/MS spectra
that were identified are shown in the table.  For bands 1-3 (higher MW),
the number of identfied proteins was the same for standard vs. MAPED,
but higher for MAPED with proteins of lower MW.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Cortese MS, Baird JP, Uversky VN, Dunker AK
 </span>
 <span class="year">
(2006)
 </span>
 <span class="jname">
J Proteome Res.
 </span>
 <span class="jvol">
4,
 </span>
 <span class="pages">
1610-1618.
 </span>
<p class="reftitl">
Uncovering the unfoldome: enriching cell extracts for
unstructured proteins by acid treatment
<p class="abstract">
<b>ABSTRACT</b>.
A method to enrich cell extracts in totally unfolded proteins was
investigated. A literature search revealed that 14 of 29 proteins
isolated by their failure to precipitate during perchloric acid
(PCA) or trichloroacetic acid (TCA) treatment where also shown
experimentally to be totally disordered. A near 100 000-fold
reduction in yield was observed after 5% or 9% PCA treatment
of total soluble E. coli protein. Despite this huge reduction,
158 and 142 spots were observed from the 5% and the 9% treated
samples, respectively, on silver-stained 2-D SDS-PAGE gels loaded
with 10 microg of protein. Treatment with 1% PCA was less
selective with more visible spots and a greater than 3-fold
higher yield. A substantial yield of unprecipitated protein
was obtained after 3% TCA treatment, suggesting that the
common use of TCA precipitation prior to 2-D gel analysis may
result in loss of unstructured protein due to their failure
to precipitate. Our preliminary analysis suggests that treating
total protein extracts with 3-5% PCA and determining the identities
of soluble proteins could be the starting point for uncovering
unfoldomes (the complement of unstructured proteins in a given proteome).
The 100 000-fold reduction in yield and concomitant reduction in
number of proteins achieved by 5% PCA treatment produced a fraction
suitable for analysis in its entirety using standard proteomic
techniques. In this way, large numbers of totally unstructured
proteins could be identified with minimal effort.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Torres MP, Thapar R, Marzluff WF, Borchers CH
 </span>
 <span class="year">
(2006)
 </span>
 <span class="jname">
J Proteome Res.
 </span>
 <span class="jvol">
4,
 </span>
 <span class="pages">
1628-1635.
 </span>
<p class="reftitl">
Phosphatase-directed phosphorylation-site determination: a synthesis
of methods for the detection and identification of phosphopeptides
<p class="abstract">
<b>ABSTRACT</b>.
Phosphopeptides can be difficult to detect and sequence by mass
spectrometry (MS) due to low ionization efficiency and suppression
effects in the MS mode, and insufficient fragmentation in the
tandem MS (MS/MS) mode, respectively. To address this problem,
we have developed a technique called Phosphatase-directed
Phosphorylation-site Determination (PPD), which combines
on-target phosphatase reactions, MALDI MS/MS of IMAC beads
on target, and hypothesis-driven MS (HD-MS). In this method,
on-target dephosphorylation experiments are conducted on
IMAC-bound phosphopeptides, because dephosphorylated peptides have,
in general, higher MS sensitivities than the corresponding
phosphopeptides. The detected dephosphorylated peptides are
sequenced by MS/MS, which identifies the potentially phosphorylated
peptide and the total number of Ser, Thr, or Tyr residues that
could hypothetically be phosphorylated within that peptide.
On the basis of this information, a mass list containing every
possible phosphorylation state of each observed peptide
(where 1 HPO(3) = 80 Da) is used to direct MALDI-MS/MS on the
phosphorylated peptides bound to IMAC beads at each theoretical
mass from the list. If the peptide is present, the resulting
MS/MS spectrum reveals the exact site(s) of phosphorylation
in the peptide. We have demonstrated the applicability of
PPD to the detection of in vivo phosphorylation sites on the
Drosophila Stem Loop Binding Protein (dSLBP), and the
complementarity of this new technique to conventional MS
phosphorylation site mapping methods, since the phosphorylation
sites in dSLBP could not be detected by other methods.
<!-- ================================================================ -->
<p class="tags">

<p class="ref">
 <span class="authors">
Horn A, Kreusch S, Bublitz R, Hoppe H, Cumme GA, Schulze M, Moore T,
Ditze G, Rhode H
 </span>
 <span class="year">
(2005)
 </span>
 <span class="jname">
Proteomics
 </span>
 <span class="jvol">
5,
 </span>
 <span class="pages">
0000-0000.
 </span>
<p class="reftitl">
Multidimensional Proteomics of Human Serum Using Parallel Chromatography
of Native Constituents and Microplate Technology
<p class="abstract">
<b>ABSTRACT</b>.
A versatile, multidimensional, and non-denaturing proteome separation procedure using
microplate technology is presented, yielding a digitized image of proteome composition. In the
first dimension, the sample under study is separated into 96 fractions by size exclusion chromatography
(SEC). In the second dimension, the fractions of the first dimension are transferred
by the liquid-handling device CyBi-Well (CyBio AG, Jena, Germany) to 96 parallel anion
exchange chromatography columns. In this way the proteins are conserved in their native states
and are distributed in 2400 liquid fractions with high recovery rates and sufficient reproducibility.
The resulting fractions are subjected to protein quantitation and identification. Spectrophotometrical
and immunological methods and enzyme activity measurements are used for
quantitation. To identify proteins, the fractions are subjected to MALDI-MS, and their tryptic
digests to both MALDI- and LC-ESI-MS/MS. All preparation steps except the first are applied in
parallel to sets of multiples of 96 samples. The procedure may be refined by adding more
separation steps and may be adapted to various protein amounts and to various proteomes.
Moreover, the method offers the opportunity to investigate functional protein complexes. The
method was applied to separate the normal human serum proteome. Within 255 fractions exhibiting
the highest protein concentrations, 742 proteins were identified by LC-ESI-MS/MS peptide
sequence tags.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Johnson RS, Davis MT, Taylor JA, Patterson SD
 </span>
 <span class="year">
(2005)
 </span>
 <span class="jname">
Methods
 </span>
 <span class="jvol">
35,
 </span>
 <span class="pages">
223-236.
 </span>
<p class="reftitl">
Informatics for Protein Identification by Mass Spectrometry
<p class="abstract">
<b>ABSTRACT</b>.
High throughput protein analysis (i.e., proteomics) Wrst became possible when sensitive peptide mass mapping techniques were
developed, thereby allowing for the possibility of identifying and cataloging most 2D gel electrophoresis spots. Shortly thereafter a
few groups pioneered the idea of identifying proteins by using peptide tandem mass spectra to search protein sequence databases.
Hence, it became possible to identify proteins from very complex mixtures. One drawback to these latter techniques is that it is not
entirely straightforward to make matches using tandem mass spectra of peptides that are modiWed or have sequences that diVer
slightly from what is present in the sequence database that is being searched. This has been part of the motivation behind automated
de novo sequencing programs that attempt to derive a peptide sequence regardless of its presence in a sequence database. The
sequence candidates thus generated are then subjected to homology-based database search programs (e.g., BLAST or FASTA).
These homology search programs, however, were not developed with mass spectrometry in mind, and it became necessary to make
minor modiWcations such that mass spectrometric ambiguities can be taken into account when comparing query and database
sequences. Finally, this review will discuss the important issue of validating protein identiWcations. All of the search programs will
produce a top ranked answer; however, only the credulous are willing to accept them carte blanche.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Delahunty C, Yates JR 3rd
 </span>
 <span class="year">
(2005)
 </span>
 <span class="jname">
Methods
 </span>
 <span class="jvol">
35,
 </span>
 <span class="pages">
248-255.
 </span>
<p class="reftitl">
Protein Identification Using 2D-LC-MS/MS
<p class="abstract">
<b>ABSTRACT</b>.
Multidimensional liquid chromatography techniques have been coupled to tandem mass spectrometry to provide a robust
method to identify proteins in complex mixtures. Data acquisition is interfaced directly with search algorithms for identiWcation
through cross-correlation with databases. This review describes the most recent advances in methodologies for protein identiWcation
by mass spectrometry and describes the limitations of the application of the technologies.
<!-- ================================================================ -->
<p class="tags">

<p class="ref" id="america-2005">
 <span class="authors">
America AHP, Cordewener JHG, van Geffen MHA, Lommen A, Vissers JPC,
Bino RJ, Hall RD
 </span>
 <span class="year">
(2005)
 </span>
 <span class="jname">
Proteomics
 </span>
 <span class="jvol">
5,
 </span>
 <span class="pages">
0000-0000.
 </span>
<p class="reftitl">
Alignment and Statistical Difference Analysis of Complex Peptide Data
Sets Generated by Multidimensional LC-MS
<p class="abstract">
<b>ABSTRACT</b>.
A method for high-resolution proteomics analyses of complex protein mixtures is 
presented using multidimensional HPLC coupled to MS (MDLC-MS). The method was 
applied to identify proteins that are differentially expressed during fruit 
ripening of tomato. Protein extracts from red and green tomato fruits were 
digested by trypsin. The resulting highly complex peptide mixtures were 
separated by strong cation exchange chromatography (SCX), and subsequently 
analyzed by RP nano-LC coupled to quadrupole-TOF MS. For detailed quantitative 
comparison, triplicate RP-LC-MS runs were performed for each SCX fraction. The 
resulting data sets were analyzed using MetAlign software for noise and data 
reduction, multiple alignment and statistical variance analysis. For each 
RP-LC-MS chromatogram, up to 7000 mass components were detected. Peak intensity 
data were compared by multivariate and statistical analysis. This revealed a 
clear separation between the green and red tomato samples, and a clear 
separation of the different SCX fractions. MS/MS spectra were collected using 
the data-dependent acquisition mode from a selected set of differentially 
detected peptide masses, enabling the identification of proteins that were 
differentially expressed during ripening of tomato fruits. Our approach is a 
highly sensitive method to analyze proteins in complex mixtures without the need 
of isotope labeling.

<!-- ================================================================ -->
<p class="tags">

<p class="ref">
 <span class="authors">
Nielsen J, Oliver S
 </span>
 <span class="year">
(2005)
 </span>
 <span class="jname">
Trends Biotechnol.
 </span>
 <span class="jvol">
23,
 </span>
 <span class="pages">
544-546.
 </span>
<p class="reftitl">
The Next Wave in Metabolome Analysis
<p class="abstract">
<b>ABSTRACT</b>.
The metabolome of a cell represents the amplification and integration of signals 
from other functional genomic levels, such as the transcriptome and the 
proteome. Although this makes metabolomics a useful tool for the high-throughput 
analysis of phenotypes, the lack of a direct connection to the genome makes it 
difficult to interpret metabolomic data. Nevertheless, functional genomics has 
produced examples of the use of metabolomics to elucidate the phenotypes of 
otherwise silent mutations. Despite several successes, we believe that future 
metabolomic studies must focus on the accurate measurement of the concentrations 
of unambiguously identified metabolites. The research community must develop 
databases of metabolite concentrations in cells that are grown in several 
welldefined conditions if metabolomic data are to be integrated meaningfully 
with data from the other levels of functional-genomic analysis and to make a 
significant contribution to systems biology.
<!-- ================================================================ -->
<p class="tags">
FD
<p class="ref">
 <span class="authors">
Barlow CK, McFadyen WD, O'Hair RAJ
 </span>
 <span class="year">
(2005)
 </span>
 <span class="jname">
J. Am. Chem. Soc.
 </span>
 <span class="jvol">
127,
 </span>
 <span class="pages">
6109-6115.
 </span>
<p class="reftitl">
Formation of Cationic Peptide Radicals by Gas-Phase Redox Reactions
with Trivalent Chromium, Manganese, Iron, and Cobalt Complexes
<p class="Para1">
The formation of cationic radical ions of peptides in the gas phase involves
the following scheme: (i) neutral peptide P is introduced into gas phase
coordinated to metal ion M with bound auxiliary ligand L; (ii)  CID leads
to homolytic bond cleavage, with reduction of the metal and oxidation of peptide
to produce cationic peptide radical P<sup>+&bull;</sup>; (iii) peptide radical
can be isolated and subjected to CID to probe its chemistry/structure.
For example, [Cu<sup>II</sup>(dien)-YGGFLR]<sup>2+</sup> will produce the
YGGFLR<sup>+*&bull;</sup> radical.  Since Cu(II) is an oxidant in peptides
and amino acids, it was thought that trivalent transition metals might be
better complexes of peptides from which to probe by CID.  Experiments were
done on Finnigan MAT model LCQ Q-IT MS.  100 &micro;l of 1 mg/ml peptide is
mixed with 100 &micro;l of 1 mg/ml metal complex in final 1 ml in MeOH.
The ligand is salen (N,N'-Ethylenebis(salicylimine), MW 263.8), so a
[M(salen)(P)]<sup>+</sup> complex results.
<p>
The [Cr<sup>III</sup>(salen)YGGFLR]<sup>+</sup> CID spectrum is show and
it shows many small molecule losses from the peptide.  Peaks at 18, 35,
and 60 Th less than the parent mass indicate H<sub>2</sub>O, H<sub>2</sub>O +
NH<sub>3</sub>, and H<sub>2</sub>O + NH=C=NH, respectively.  The NH=C=NH
originates from the R side chain.  Also in the spectrum is the peptide-free
metal-ligand complex [Cr<sup>III</sup>(salen)]<sup>+</sup> with the peptide
appearing as a cationic radical, P<sup>+&bull;</sup>, although it low abundance.
With salen derivatives as ligand there is only minor differences in CID spectra.
[Cr<sup>III</sup>(salenNO<sub>2</sub>)(P)]<sup>+</sup> produced enough
P<sup>+&bull;</sup> so that it could be further fragmented.  CID of
[Cr<sup>III</sup>(salenOMe)(P)]<sup>+</sup> does not yield radical cation
but just liberates the neutral peptide.
<p>
With [Mn<sup>III</sup>(salen)(P)]<sup>+</sup> complex the bound peptide no
longer shows a 60 amu (H2O+NH=C=NH) loss.  The complex
[Mn<sup>III</sup>(salenX)(P)]<sup>+</sup> dissociates to
[Mn<sup>III</sup>(salen)]<sup>+</sup> and both neutral peptide P
and radical cation P<sup>+&bull;</sup>.  An two hydrogen atom loss species
[Mn<sup>III</sup>(P-2H)]<sup>+</sup> ion complex is seen, indicating
neutral ligand salenH<sub>2</sub>X is lost with two protons.  The nature of
the loss is controlled more by X than by the peptide.
[Mn<sup>III</sup>(salenNO2)YGGFLR]<sup>+</sup> produces only P<sup>+&bull;</sup>
and some additional fragment ions.  [Mn<sup>III</sup>(salenCl)(P)]<sup>+</sup>
is similar but [Mn<sup>III</sup>(salenCl)]<sup>+</sup> and
[Mn<sup>III</sup>(P-2H)]<sup>+</sup> are in low abundance.  Compeition between
forming neutral peptide P and cationi radical P<sup>+&bull;</sup> in
salenX is governed by X electron-withdrawing capacity (NO<sub>2</sub> &gt; Cl
&gt; H &gt; OMe) which favors radical cation formation over that of the
neutral peptide.  The nature of the peptide structure P also a factor:
thus GGGFLR favors loss of the neutral peptide while YGGFLR and WGGFLR are
<p>
With Fe<sup>III</sup> complexes, the dominant product is P<sup>+&bull;</sup> and
its subfragmentation ions  From the peptide YGGFLR <i>p</i>-quinomethide
and from WGGFLR 3-methyleneindolenine are lost, something also noted
with Cu<sup>II</sup> complexes.  Fe(III) appears to make the peptide more
prone to fragmentation than Mn(III) through homolytic cleavage than
heterolytic bond cleavage for any particular ligand or peptide.  Unlike
for Mn(III), [Fe<sup>III</sup>(P-2H)]<sup>+</sup> is not present at all.
There is loss of water and of 60 amu (H<sub>2</sub>O + NH=C=NH).
The peptide radical is more likely to undergo subfragmentation when the ligand
X is electron-withdrawing.
<p>
Co(III) complexes of peptide provide a different kind of reaction, and
the sequence of the peptide is more significant for the peptides used here.
More radical cations are generated for
[Co<sup>III</sup>(salenX)YGGFLR]<sup>+</sup> with additional subfragmentation.
The radical also loses 44 and 45 amu which are not seen with other metal
complexes.  These correspond to CO2 and &bull;CO2H.  CID of
[Co<sup>III</sup>(salenX)GGGFLR]<sup>+</sup> do not produce GGGFLR<sup>+&bull;</sup>
in spectra except as the extensively subfragmented evidence of it to produce
[M-CO<sub>2</sub>]<sup>+&bull;</sup> and [M-&bull;CO<sub>2</sub>H]<sup>+</sup>.
<p>
In summary Mn and Fe are better than Cr and Co at producing cationic peptide
radicals although all metals produce it to some degree.  Dissociation of the
complex in Cr leads to neutral and radical peptide, with subfragmentation
dominating.  This indicates that the barrier to dissociation from Cr may be
higher and puts the complex in a higher energetic state necessary to achieve
dissociation, but with more fragmentation.
<!-- ================================================================ -->
<p class="tags">

<p class="ref">
 <span class="authors">
Lange E, Gr&ouml;pl C, Reinert K, Kohlbacher O, Hildebrandt A
 </span>
 <span class="year">
(2005)
 </span>
 <span class="jname">
Pac Sym Proceed
 </span>
 <span class="jvol">
&nbsp;
 </span>
 <span class="pages">
&nbsp;
 </span>
<p class="reftitl">
High-Accuracy Peak Picking of Proteomics Data Using Wavelet Techniques
<p class="Para1">
The algorithm proposed here starts by dividing the mass range into
10 Da <i>boxes</i>, but not in a way that a division occurs along a peak.
The wavelet transform in each box is then computed.  From the maximum position
in the wavelet transform every peak centroid, height and area can be estimated.
With these parameters the raw data peaks using an asymmetric sech<sup>2</sup>
and asymmetric Lorentzian function.  Improvement of fit quality comes from
correlating the resulting peaks with the experimental data can be increased
in an optional optimization step.  This can be important if neighboring
peaks overlap so strongly that they cannot be fit well individually, and
if resolution of the experimental data is low.
<p>
The algorithm works as follows:
<p style="white-space:pre;"><code>
 <b>for all</b> mass spectra <i>s</i> in <i>experiment</i> <b>do</b>
   <i>box_list</i> = splitSpectrum(<i>s</i>)
   <b>for all</b> boxes <i>b</i> in <i>box_list</i> <b>do</b>
     <i>peak_list</i> = [ ]
     <i>w<sub>b</sub></i> = continuousWaveletTransformation(<i>b</i>)
     <b>while</b> getNextMaximumPosition(<i>w<sub>b</sub>, b, x</i><sub>0</sub>) <b>do</b>
       (<i>x<sub>l</sub>,x<sub>r</sub></i>) = searchForPeakEndpoints(<i>b, x</i><sub>0</sub>)
       <i>c</i> = estimateCentroid(<i>x<sub>l</sub>,x<sub>r</sub></i>)
       <i>h</i> = intensity(<i>x</i><sub>0</sub>)
       (<i>A<sub>l</sub>, A<sub>r</sub></i>) = integrateAreas(<i>x<sub>l</sub>,x<sub>r</sub></i>)
       <i>f</i> = fitPeakShape(<i>A<sub>l</sub>,A<sub>r</sub>,x</i><sub>0</sub>,<i>h</i>)
       push(<i>f,peak_list</i>)
       removeRawDataPeak(<i>x<sub>l</sub>,x<sub>r</sub>,b</i>)
       <i>w<sub>b</sub></i> = continuousWaveletTransformation(<i>b</i>)
     <b>end while</b>
     optimizePeakParameters(<i>peak_list, b</i>)
   <b>end for</b>
 <b>end for</b>
</code>
<p>
In decomposing the spectrum, division of the mass spectrum into boxes
at point <i>x</i> if and only if intensity <i>y</i> is smaller than a given
noise threshold, or a minimum is searched using a moving average method.
When no minimum is inside a certain search radius <i>x</i> does belong to
a peak of sensible width, and so the division is made at <i>x</i>.
<p>
Continuous wavelet transform <i>W</i><sub><i>s</i>(<i>b</i>)</sub> of a signal
<i>s</i> at position <i>b</i> is defined as:
<p class="eqnserif">
<i>W<sub>&psi;</sub> s<sub>a</sub></i>(<i>b</i>) =
(1/&radic;a) &int;<sub>&minus;&infin;</sub><sup>&infin;</sup>
<i>s</i>(<i>x</i>)<i>&psi;</i>[(<i>x</i>&minus;<i>b</i>)/<i>a</i>] <i>dx</i>
<p>
with wavelet <i>&psi;</i> and the dilation parameter <i>a</i>.  The Marr
wavelet is chosen for <i>&psi;</i>:
<p class="eqnserif">
<i>&psi;</i>(<i>x</i>) = (1 &minus; <i>x</i><sup>2</sup>) exp
(<i>x</i><sup>2</sup> / 2) = <i>d</i><sup>2</sup>/<i>dx</i><sup>2</sup>
exp(&minus;<i>x</i><sup>2</sup>/ 2)
<p>
because for symmetric peaks the maximum position in the CWT coincides with
the maximum position in the data and is a good first estimate even for
asymmetric peaks.  In peak picking the scale of the wavelet (and its width)
should correspond to the typical width of peaks.  <i>&psi;</i> can only
be described by a few data points on this scale, and the convolution of
wavelet and signal can be computed efficiently with pre-tabulated values
of <i>&psi;</i>.
<p>
In searching for peak endpoints there is a difficulty with effects of
noise or the overlapping of peaks occurs.  Peak intensity will not drop
below a threshold when the next peak's area of influence is reached.
From the maximum, the algorithm moves both left and right until either
a minimum is reached or the intensity falls below a pre-defined noise
threshold.
<!-- ================================================================ -->
<p class="tags">

<p class="ref">
 <span class="authors">
le Guezennec X, Brinkman AB, Vermeulen M, Denissov SG, Gazziola C,
Lohrum ME, Stunnenberg HG
 </span>
 <span class="year">
(2005)
 </span>
 <span class="jname">
BJU Int.
 </span>
 <span class="jvol">
96 (Suppl 2),
 </span>
 <span class="pages">
16-22.
 </span>
<p class="reftitl">
Targeted Discovery Tools:  Proteomics and Chromatin Immunoprecipitation-On-Chip
<p class="abstract">
<b>ABSTRACT</b>.
Despite the availability of several completely sequenced genomes, we are still, 
for the most part, ignorant about how genes interact and regulate each other 
within a given cell type to specify identity, function and cellular memory. A 
realistic model of cellular regulation based on current knowledge indicates that 
many interacting networks operate at the epigenetic, transcriptional, 
translational and post-translational levels, with feedback between the various 
levels. Proteinprotein and proteinDNA interactions help to define which genes 
may be activated in a particular cell, and determine whether external cues cause 
activation or repression. New technologies, e.g. proteomics using mass 
spectrometry, high-density DNA or oligonucleotide microarrays (chips), and 
chromatin immunoprecipitation (ChIP), provide new and exciting tools for 
deciphering the pathways and proteins controlling gene expression. Analysis of 
these pathways offers new insight that aids targeted drug development.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Fandi&ntilde;o, Rais I, Vollmer M, Elgass H, Sch&auml;gger H, Karas M
 </span>
 <span class="year">
(2005)
 </span>
 <span class="jname">
J. Mass Spectrom.
 </span>
 <span class="jvol">
40,
 </span>
 <span class="pages">
1223-1231.
 </span>
<p class="reftitl">
LC-Nanospray-MS/MS Analysis of Hydrophobic Proteins from Membrane Proteins
from Membrane Protein Complexes Isolated by Blue-Native Electrophoresis
<p class="abstract">
<b>ABSTRACT</b>.
The application of two-dimensional electrophoresis for the identification of hydrophobic membrane
proteins is principally hampered by precipitation of many of these proteins during first-dimension,
isoelectric focusing. Therefore new strategies towards the identification and characterization ofmembrane
proteins are being developed. In this work we present a direct and rapid approach from blue-native
gels to mass spectrometry, which allows the analyses of complete complexes and prevents protein
aggregation of hydrophobic regions during electrophoresis. We combine blue-native gel electrophoresis
and liquid chromatographynanospray-iontrap tandem mass spectrometry to analyze the composition
of oxidative phosphorylation complexes I, III, IV and V from bovine-heart mitochondria as a model
system containing a number of highly hydrophobic proteins. Bands from blue-native gels were subjected
either to in-gel or to in-solution tryptic digestion. The obtained peptide mixtures were further analyzed
by liquid chromatographytandem mass spectrometry and the corresponding proteins were identified
by database search. From a total of 86 proteins, 67 protein subunits could be identified including all
highly hydrophobic components, except the ND4L and ND6 subunits of complex I. We demonstrate
that liquid chromatographytandem mass spectrometry combined to blue-native electrophoresis is a
straightforward tool for proteomic analysis of multiprotein complexes, and especially for the identification
of very hydrophobic membrane protein constituents that are not accessible by common isoelectric
focusing/sodium dodecyl sulphate gel electrophoresis.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Feldmann RE Jr, Bieback K, Maurer MH, Kalenka A, B&uuml;rgers HF,
Gross J, Hunzinger C, Kl&uuml;ter, Kuschinsky W, Eichler H
 </span>
 <span class="year">
(2005)
 </span>
 <span class="jname">
Electrophoresis
 </span>
 <span class="jvol">
26,
 </span>
 <span class="pages">
2749-2758.
 </span>
<p class="reftitl">
Stem Cell Proteomes: A Profile of Human Mesenchymal Stem Cells Derived
From Umbilical Cord Blood
<p class="abstract">
<b>ABSTRACT</b>.
Multipotent mesenchymal stem cells (MSCs) derived from human umbilical cord blood
(UCB) represent promising candidates for the development of future strategies in cellular
therapy. To create a comprehensive protein expression profile for UCB-MSCs, one
UCB unit from a full-term delivery was isolated from the unborn placenta, transferred
into culture, and their whole-cell protein fraction was subjected to two-dimensional
electrophoresis (2-DE). Unambiguous protein identification was achieved with peptide
mass fingerprinting matrix-assisted laser desorption/ionization  time of flight  mass
spectrometry (MALDI-TOF-MS), peptide sequencing (MALDI LIFT-TOF/TOF MS), as
well as gel-matching with previously identified databases. In overall five replicate 2-DE
runs, a total of 2037 6 437 protein spots were detected of which 205 were identified
representing 145 different proteins and 60 isoforms or post-translational modifications.
The identified proteins could be grouped into several functional categories, such as
metabolism, folding, cytoskeleton, transcription, signal transduction, protein degradation,
detoxification, vesicle/protein transport, cell cycle regulation, apoptosis, and
calcium homeostasis. The acquired proteome map of nondifferentiated UCB-MSCs is
a useful inventory which facilitates the identification of the normal proteomic pattern as
well as its changes due to activated or suppressed pathways of cytosolic signal
transduction which occur during proliferation, differentiation, or other experimental
conditions.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Wheelock AM, Buckpitt AR
 </span>
 <span class="year">
(2005)
 </span>
 <span class="jname">
Electrophoresis
 </span>
 <span class="jvol">
26,
 </span>
 <span class="pages">
4508-4520.
 </span>
<p class="reftitl">
Software-Induced Variance in Two-Dimensional Gel Electrophoresis Image
Analysis
<p class="abstract">
<b>ABSTRACT</b>.
Experimental variability in 2-DE is well documented, but little attention has been paid to
variability arising from postexperimental quantitative analyses using various 2-DE
software packages. The performance of two 2-DE analysis software programs,
Phoretix 2D Expression v2004 (Expression) and PDQuest 7.2 (PDQuest), was evaluated
in this study. All available background subtraction and smoothing algorithms
were tested using both data generated from one single 2-DE gel image, thus excluding
experimental variance, and with authentic sets of replicate gels (n = 5). A slight shift of
the image boundaries (the cropping area) caused both programs to induce variance
in protein spot quantification of otherwise identical gel images. The resulting variance
for PDQuest (CVmean = 8%) was approximately twice that for Expression (CVmean = 4%).
In authentic sets of replicate 2-DE gels (n = 5), the experimental variance confounded
the software-induced variance to some extent. However, Expression still outperformed
PDQuest, which exhibited software-induced variance as high as 25% of the total
observed variance. Surprisingly, the complete omission of background subtraction
algorithms resulted in the least amount of software-based variance. These data indicate
that 2-DE gel analysis software constitutes a significant source of the variance
observed in quantitative proteomics, and that the use of background subtraction
algorithms can further increase the variance.
<p class="Para1">
Proteins used were the pooled lysis-lavage of the airway epithelium of male rats.
Five 2D gels were run on sampling of the pool on IPG 4-7 and 6-9 strips.
A Multiphor II was used for IEF using EPS 3501 XL power supply.  500 &micro;g
protein is mixed in 350 &micro;l of rehydration buffer (7 M urea, 2 M thiourea,
0.5% TX-100, 4% CHAPS, 2% Calbiochem inhibitor mix, 65 mM DTT, 1% IPG buffer
for the strip with orange G tracking dye, using 18 cm 4-7 strips.  Electrophoresis
program:  0-50 V in 1 min, 50 V 1 h, 50 to 1000 V in 3 h, 1000 to 3500 V in
3 h, and 3500 V for 19 h, giving 75 kV-h total.  Second dimension was on 10%
Duracryl gels using Hoefer ISO-DALT at 10&deg; with 14 mA per gel until dye
front reached edge in about 18 h.  Fixation was in 20% EtOH/7% acetic acid for
&gt; 1 h.  Basic range 18 cm IPG pH 6-9 strips were rehydrated in special
DeStreak buffer with 2% protease inhibitor and 1% IPG buffer for pH 6-11 range
with 700 &micro;g protein spiked with DeStreak and diluted to 350 &micro;l
with lysis buffer without the IPG buffer.  Samples are loaded using a paper
bridge method with 5.5 cm paper wick on anodic end with IEF performed using
program 0 to 1000 V in 2.5 h, 1000 to 3500 V in 2.5 h, then 21 h at high voltage
for total 80 kVh.  Rhinohide-type acrylamide gels were run on the 2nd dimension.
Staining is with SYPRO Ruby; after destaining and soaking water in dark,
gels were scanned with Typhoon (Molecular Dynamics);  fluorescence material
is always wiped with isopropanol to remove background on glass platen; gels
are scanned 600 PMT with 200 &micro;m pixel size.  File format is GEL which is
variant of 16-bit TIFF, with square root rather than linear data storage and
better at preserving the 100,000-fold dynamic range of Typhoon, Storm, and
FluorImager systems.
<p>
A single 2D image file of the pH 4-7 range is cropped once (Imagequant) then
copied five times without modification.  These should be identical image
data files.  Another image of the basic pH range gel was produced with 5 copies.
Spot detection and matching are done as described below.  In addition,
a single gel image is produced with 5 copies, then each copy cropped;
cropped copies are similar but not identical.  This is done because it
represents better an authentic replicate set in which cropping area differs
between gels.  Finally, in another set, five 2D gels are produced from a
single sample, representhing an authentic experiment.
<p>
For all the image experiments described above, analysis was by Phoretix
Expression v2004 and by PDQuest 7.2.  The procedure of Marengo et al
for PDQuest was used with all available background subtraction and filtering
algorithms used.  Automatic spot detection was first done with careful
manual editing to eliminate spots matched incorrectly.  Delete/add/auto-split/
and auto-merge of spots were the only semi-automatic editing tools used
to minimize subjectivity.  The Expression software allows detecting and matching
of spots without background subtraction;  3 subtraction algorithms are available:
"mode of non-spot", "lowest on boundary", "average on boundary" after spot
matching and editing.  PDQuest always uses background subtraction with a floating
ball algorithm and applying five different smoothing filters (median, weighted
mean, power mean, contra mean, and adaptive) as well as a no-smoothing option.
The background subtract is kept at 21-27 for all gels and for spot detection,
sensitivity=10, size=3, min peak=700, and the speckle and streak filters are
applied.  Spot volumes are exported to Excel with computation of average, SD,
and CV for each spot and normality of CV distributions tested by normality
plots and Shapiro Wilks W test.  CV distributions are reported as median,
interquartile range, and range.
<p>
Both PDQuest and Expression reported 0.0% median, IQR, and range for gel
images that are 5 copies of the same cropped image for all matched spots.
Depending on the smoothing filter, matched spots ranged in count from
742 (power mean) to 938 (contra mean), with 921 spots with no smoothing.
The count of matched spots was 1312 spots for all modes of background
subtraction on Expression.  Since all gel images were identical, spot
count in PDQuest thus depended upon the use of a particular smoothing filter.
<!-- ================================================================ -->
<p class="tags">

<p class="ref">
 <span class="authors">
Valmu L, Kalkkinen N, Husa A, Rye PD
 </span>
 <span class="year">
(2005)
 </span>
 <span class="jname">
Biochemistry
 </span>
 <span class="jvol">
44,
 </span>
 <span class="pages">
16007-16013
 </span>
<p class="reftitl">
Differential Susceptibility of Transferrin Glycoforms to Chymotrypsin:
A Proteomics Approach to the Detection of Carbohydrate-Deficient Transferrin
<p class="Para1">
Human recombinant TRF was altered to change glycosylation sites
Asn-413 and -611 to Asp residues and expressed in BHK cells and purified
to homogeneity (verified by ESI-MS, anion exchange chromatography,
and urea gels).  Immunoreactivity and spectroscopy for Fe-binding presence
also show TRF.  rhTRF-NG (nonglycosylating) was treated with chymotrypsin
and peptides seen on MALDI.  Sequencing on LC-MS/MS was done, reporting
some missed cleavages of the protease.  Repetive digestion confirmed the
missed cleavage pattern; MALDI coverage was 40.8%.  TRF exists as a
bilobar globular protein and sites on the C-terminal half were preferentially
cleaved while N-side cleavage was on the linking sequence and N-terminus.
<p>
C<sub>8</sub> RP column separation of peptides of glycan and non-glycan
recombinant digested 2 h show differences in patterns.  1 pmol of
TRF-NG and TRF was digested with 30 min chymotrypsin and separated
on 0.75 &micro;m &times; 150 mm C<sub>18</sub>.  A &gt; 10-fold difference
is seen in the TRF-NG to TRF sample, with the glycans protecting the TRF
against digestion.  The differnces in digestion were found in both an
HPLC (detection OD214) and in MALDI.  The laser energy setting on the MALDI
was higher for TRF compared to TRF-NG peptides, probably because a lower
yield of peptides was obtained from digestion.  Some low intensity peaks
were detected for TRF-NG, one at <i>m/z</i> 1690.472, probably the
[M+H]<sup>+</sup> ion of the 413-427 peptide (this contains the Asn&rarr;Asp413)
substitution.  A peak at <i>m/z</i> 1464.239 would be the [M+H]<sup>+</sup>
ion corresponding to 609-622 peptide and have the 611 substitution;
this was missing in the TRF sample.  Signal-to-noise for these peptides
with altered Asn sites were 3-5 (mean 3.87) and 4-6 (mean 5.15), respectively.
<p>
LC-MS/MS of 10 lower intensity peaks was done.  Sequencing was difficult
for the 609-622 peptide since its ionization efficiency was a problem
(using chymotrypsin rather than trypsin to obtain certain positive ions).
The 413-427 produced good MS/MS spectra; its parent ion at <i>m/z</i>
845.86 [M+2H]<sup>2+</sup> gave good MS/MS with 32 eV collision energy
(and it contained a Lys residue) and good <i>b</i> product ions.
A sequence of SDNCEDTPEAG is readable with a C-terminal flanking dipeptide
of YF, and an N-terminal dipeptide of DK is seen.  N-terminal sequencing
of the peptide revealed DKSDN, as additional evidence.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Krokhin OV, Ens W, Standing KG
 </span>
 <span class="year">
(2005)
 </span>
 <span class="jname">
J. Biomolec. Tech.
 </span>
 <span class="jvol">
16,
 </span>
 <span class="pages">
427-438.
 </span>
<p class="reftitl">
MALDI QqTOF MS Combined with Off-Line HPLC for Characterization of
Protein Primary Structure and Post-Translational Modifications
<p class="Para1">
PRG03 is a lyophilized sample said to contain 5 pmol <q>unknown</q>
protein with two phosphorylated peptides at substoichiometric levels, and
included a contaminant.  It was suspended in 50 &micro;l 0.2% formic acid.
0.5 &micro;l was mixed 1:1 with 150 mg/ml DHB matrix in 50% MeCN and deposited
on MALDI target.  5 &micro;l (containing 500 fmol) was used for LC-MALDI.
PRG04 is a sample with 3 closely related proteins with 3 pmol each, with
possible contaminants; it was dissolved in 0.1 M ammonium bicarbonate,
and one-sixth of it was reduced 10 min in 30 mM DTT (57&deg;) and then
alkylate 30 min in dark at room temp in 50 mM iodoacetamide, then dialyzed
against 100 mM AmBicarb for 6 h in 7kDa MWCO tubing from Pierce. This
was digested in sequencing grade Promega trypsin 1:50 for 12 h 37&deg;.
After drying, material was resuspended in 5.5 &micro;l of 0.5% TFA.  0.5
&micro;l was used for MALDI and the balance of 5 &micro;l was run on
micro-LC fractionation followed by MALDI MS or MS/MS.  Three isoforms of
carbonic anhydrase were identified as components of the <q>unknown</q>
sample, but intensities were extremlely low;  as a control commercial
bovine carbonic anhydrase isoform I was prepared in 0.5% TFA to contain
3 pmol, and then processed as PRG03 and PRG04 above.
<p>
Chromatography was in two configurations:  a micro-Agilent 1100 was setup
for PRG03 and PRG04 analysis, with 5 &micro;l injection on to 150 &micro;m
&times; 15 cm Vydac 218 TP C18 5&micro;m and eluted with gradient of
1-80% MeCN containing 0.1% TFA over 60 min at 4 &micro;l/min.  OD214 was
monitored with 500 nl flow cell.  65 &micro;m PEEK and 50 &micro;m fused
silica was used for pre- and post-column connections, respectively.
To column effluent a Microtee P775 (Upchurch) was connected with flow
of DHB at 0.5 &micro;l/min.  This was deposited at 1 min intervals on
movable gold targets.  Most peptides elute inside 40 min
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Uetz P, Yu-An D, Zeretzke C, Atzler C, Baiker A, Berger B, Rajagopala SV,
Roupelieva M, Rose D, Fossum E, Haas J
 </span>
 <span class="year">
(2006)
 </span>
 <span class="jname">
Science
 </span>
 <span class="jvol">
311,
 </span>
 <span class="pages">
239-232.
 </span>
<p class="reftitl">
Herpesviral Protein Networks and Their Interaction with the Human Proteome
<p class="abstract">
<b>ABSTRACT</b>.
The comprehensive yeast two-hybrid analysis of intraviral protein interactions in two members of
the herpesvirus family, Kaposi sarcomaassociated herpesvirus (KSHV) and varicella-zoster virus
(VZV), revealed 123 and 173 interactions, respectively. Viral protein interaction networks resemble
single, highly coupled modules, whereas cellular networks are organized in separate functional
submodules. Predicted and experimentally verified interactions between KSHV and human proteins
were used to connect the viral interactome into a prototypical human interactome and to simulate
infection. The analysis of the combined system showed that the viral network adopts cellular
network features and that protein networks of herpesviruses and possibly other intracellular
pathogens have distinguishing topologies.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Ulintz PJ, Zhu J, Qin ZS, Andrews PC
 </span>
 <span class="year">
(2005)
 </span>
 <span class="jname">
Molec. Cell. Proteomics
 </span>
 <span class="jvol">
xxx,
 </span>
 <span class="pages">
yyyy-zzzz
 </span>
<p class="reftitl">
Improved Classification of Mass Spectrometry Database Search Results
Using Newer Machine Learning Approaches
<p class="Para1">
Better methods for checking/confirming correctness of db search results
are required.  SEQUEST scores of 2+/- tryptic peptides set an XCorr &gt; 2.2
and &delta;Cn of at least 0.1.  There is not wide agreement.  The
X!Tandem algorithm incorporates the hypergeometric distribution of
Sadygov and Yates on the frequency of fragment ion matches from a peptide
sequence database while the Poisson distribution has been put forward by
Geer et al.  Additional algorithms take as input the MASCOT or SEQUEST
search results and classify them as correct or incorrect.  These include
PeptideProphet and QScore.  PeptideProphet assesses validity of
MS db search results using supervised and unsupervised learning to
improve sensitivity than methods using scoring thresholds.  This paper
attempts to show the latest machine learning methods for classification,
including tree-based ensemble methods such as <i>boosting</i> and
<i>random forest</i>.
<p>
One popular method for peptide identification is PeptideProphet mixture
model approach.  A discriminant score function
<i>F</i>(<i>x</i><sub>1</sub>,<i>x</i><sub>2</sub>,...<i>x</i><sub>S</sub>) =
<i>c</i><sub>0</sub> + <i>c</i><sub>1</sub><i>x</i><sub>1</sub> + ...
<i>c</i><sub>S</sub><i>x</i><sub>S</sub> is defined for
<i>x<sub>i</sub></i> search scores having <i>c<sub>i</sub></i> weights.
A Gaussian distribution is modeled to the discriminant scores corresponding
to correct peptide assignment, and gamma distribution is used to model
asymmetric discriminant scores belong to wrong peptide assigments.
The scores are represented by <i>p</i>(<i>x</i>) =
<i>rf</i><sub>1</sub>(<i>x</i>) + (1 &minus; <i>r</i>)<i>f</i><sub>2</sub>(<i>x</i>)
where <i>f</i><sub>1</sub>(<i>x</i>) and <i>f</i><sub>2</sub>(<i>x</i>)
represent density functions of two types of discriminant scores and <i>r</i>
is the proportion of correct identifications.  With each new test dataset
the EM algorithm will estimate the probability that the peptide identified
is correct.
<p>
Choice of discriminant score function <i>F</i> is crucial.  The weight
<i>c<sub>i</sub></i> values are derived so as to maximize between-
vs. within-class variation under the multivariate normal assumption of
the training data.  This means training data and test data are generated
from same source.  When new discriminant score sets are generated and should
be classified one must retrain the <i>c<sub>i</sub></i> weight values
using new corresponding training set.  Because of several variables
(instrumentation, protocol, database, searching software) it is not unusual
to see different results.   As to modeling distributions, there is no
valid reason to prefer normal or gamma distributions for the two types
of discriminant scores.  Indeed gamma distribution may be better at modeling
positive and negative scores of the Mascot algorithm.
<p>
Determining correct and incorrect peptide assignments is a classification
problem, or supervised learning.  Methods such as CART, SVM, random forest,
boosting and bagging exist.  SVM is good for small sample size situations
with large feature spaces.
<p>
<b>Boosting</b> is a procedure that combines many weak classifiers to find a
final powerful classifier.  In a two-classification setting, suppose
<i>x<sub>i</sub></i> is a vector of input variables, which in this case
are scores and attributes of an individual MS db search result produced
from an algorithm such as SEQUEST, and <i>y<sub>i</sub></i> is the output
coded as &minus;1 or 1, indicating correct or incorrect assignment.
Suppose an algorithm that builds a classifier T(<i>x</i>) using weighted
training samples to that when there is new input <i>x</i>, T(<i>x</i>)
produces a prediction taking one of two values {&minus;1, 1} and so
classifier T(<i>x</i>) is a decision tree.  Boosting occurs by starting
with equal-weighted training samples and building classifer T<sub>1</sub>(<i>x</i>).
If a training sample is misclassified, an incorrect peptide being assigned
to the spectrum, the weight of the sample is increased (boosted).
A second classifier T<sub>2</sub>(<i>x</i>) is built with training samples
using new weights but which are not equal.  Misclassified samples have their
weights boosted again, and the procedure iterates <i>M</i> times.  Hundreds
or even thousands of classifiers are built this way.  The final score
is assigned for an input <i>x</i> which is a linear (weighted) combination
of classifers, with high score being that the sample is correctly assigned
and a low score an incorrect hit.
<p>
<b>Random forests</b> is similar to boosting but an ensemble that combines
many decision trees.  The trees grow differently however in three ways.
1. The method randomly selects with replacement <i>n</i> samples from the
original training data instead of assigning different weights to training
samples. 2. Instead of considering all input variables at each split of the
decision tree, a small group of input variables on which to split are
randomly selected.  3. Each tree is grown to the largest extent possible.
When classifying a new sample from an input, the input is run down each
of the trees in the forest, with each tree giving a classification (vote).
The forest chooses the classification having the most votes over all the
trees in the forest.
<p>
<b>Support vector machine</b> (SVM) is another learning technique.  It
makes a non-linear classification boundary in the original input space
by constructing a linear boundary in a transformed version of the original
input space.  Dimension of transformed space can be very large and infinit
in some cases.  The computation is achieved through a positive definite
reproducing kernel to give inner product of the transformed space.  SVM
finds the hyperplane in the transformed space separating two classes by the
biggest margin.  It has produced less false positives and negatives than
other approaches.  A weakness is that SVM only estimates the category of
classification, whereas the assignment probability <i>p</i>(<i>x</i>) is
of interest itself, where
<i>p</i>(<i>x</i>) = <i>P</i>(<i>Y</i>=1|<i>X</i>=<i>x</i>)
is the posterior probability of sample being in class 1 (correctly identified
peptide).  Another drawback is that the kernel and cost, tuning parameters,
are not easily selected.
<p>
An ESI-MS dataset was used in this study.  MS/MS spectra from 22 LC-MS/MS
runs on a control sample of 18 known proteins mixed in various concentrations
was used with a ThermoFinnigan ion trap MS generating the dataset.  There
were 37,044 spectra obtained for [M+H]<sup>+</sup>, [M+2H]<sup>2+</sup>,
and [M+3H]<sup>3+</sup> parent ions.  Each spectrum was searched by
SEQUEST on human database.  Top-scoring peptide was retained for each
spectrum, and top hits againt known 18 proteins labeled as <q>correct</q>.
2757 (7.44%) of peptide assignments were found to be correct.
<p>
A MALDI dataset was generated from 300 purified recombinants gel-purified
by 1D SDS-PAGE.  Band plugs were subjected to in-gel digestion, and
serially diluted to generate <q>realistic</q> spectra, peptides cleaned
by C18 ZipTip.  Digestions were spotted in 4 replicates on MALDI targets
and MS/MS spectra with ABi 4700 Proteomics Analyzer obtained.  The most
abundant ions were selected from each replicate and with previously selected
peaks excluded until MS peaks could no longer be detected.  Up to 24 MS/MS
spectra were obtained for each standard protein.  Results from 158 of the
protein standards were generated for the dataset.  Testing and training
datasets were generated by using Agilent Spectrum Mill with trypsin
as enzyme and oxidized Met and pyro-Glu as variable modifications searched
against NCBInr-human dataset.  The NCBInr database was altered to replace
entries corresponding to protein standard with annotated entries that
include the N-term affinity (T7 or 6xHis) tag, and appending E coli K12
sequences since E coli was the recombinant host.  Tolerances of 0.7 Da
for precursor and 0.3 Da for fragment ion search were used;  minimum
matched peak intensity was 40%.  11764 search results were found from
4340 spectra (up to 5 top ranking hits). 1044 were true positives and
111 of these were not top-ranking.
<p>
The .out files from SEQUEST and .spo files from SpectrumMill are parsed
into simple text row/column format for use in pattern classification
algorithms in Python-coded modules.  Only the top hit was parsed for
each SEQUEST spectrum, and for the MALDI set the top 5 ranking hits.
Attributes from SEQUEST assignments include typical parameters of the
SEQUEST algorithm:  Sp, Sp rank, deltaCn, XCorr and other statistics
of the report:  total intensity, number of matching peaks, fragment ion
ratios.  Length is included in the attributes since PeptideProphet
uses it when normalizing the XCorr attribute against peptide length.
Other attributes include factors related C-term residue (K='1', R='2',
others='0'), and count of prolines and arginines.  The Mobile Proton Factor
(MPF) is calculated to be
<p class="eqnserif">
(1.0 &times; <i>R</i>) + (0.8 &times; <i>K</i>) + (0.5 &times; <i>H</i>) / charge
<p>
MPF is measure of mobility of protons in peptide and theoretical measure
of ease of fragmentation in gas phase.  Smaller MPF values indicate
higher mobility whereas relative nonmobility is indicated when MPF &ge; 1.
The <i>RKH</i> refer to counts of residues in the peptide affecting its
basicity, and coefficients are reflective of relative basicity normalized
to the dissociation constant pKa 12, 10, and 5.9, respectively.
The <i>charge</i> is the charge of the parent/precursor to indicate count
of free protons available to assist charge-directed fragmentation.
<p>
SpectrumMill attributes are based on the search platform SpectrumMill
which works from ProteinProspector script sets.  It is a non-probabilistic
score, independent of database size.  Increases in score correlate
with peaks matching theoretical fragment ion types with penalty for
unmatched peaks.  Peak intenstity and peptide length affect score.  Scoring
system accounts for all fragment ion types and neutral losses, along with
internal and immonium ions.  Scored Peak Intensity (SPI) is second primary
scoring measure reflecting percentage of total peak intensity matching
predicted fragment ion masses.  The Background Cleavage Score (BCS) is the
count of cleavage events that generate <i>b</i> or <i>y</i> ions;
the unused ion ration (count of unused ions over count of total ions after
peak thresholding) is measure of amount of signal not accounted for in match.
The delta parent mass measures the difference between observed and
experimental peptide precursor masses.
<p>
One training and one testing dataset were constructed for the ESI-SEQUEST
and MALDI-SpectrumMill datasets by random selection.  For all results
the evaluation was done on a test so without overlap on training set.
2/3 of data were used for training, and 1/3 for testing.  SEQUEST .out
files for each test set are placed in separate diretcory and processed
with the out2summary.c script for making the PeptideProphet html input
file, which was run with default parameters.  The R language packages
for boosting and random forest approaches was used.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Kalia A, Gupta RP
 </span>
 <span class="year">
(2005)
 </span>
 <span class="jname">
Crit. Rev. Biotechnol.
 </span>
 <span class="jvol">
25,
 </span>
 <span class="pages">
173-198.
 </span>
<p class="reftitl">
Proteomics:  A Paradigm Shift
<p class="Para1">
<b>REVIEW.</b>  Genomics yields sequence information but we do not know the
biology of proteins coded by these sequences to predict many functional
aspects of proteins just from the sequence info.  In June 2000 the full
sequence was presented by Collins and Venter five years ahead of schedule.
Inventions and discoveries that helped included Hunkapiller's 1998 creation
of a laser beam-decoding instrument, and a shotgun sequencing technique from
Venter in 1999.  The next new movement is <b>functional genomics</b>, in which
we try to making functional sense of the genome sequence.  The genome
codes for 30,000-50,000 genes making 100,000 different proteins (owing to
alternative splicing).  There are trillions of cells in the body, with
46 chromosomes present in a 8 &micro;m space in cells.  With a 3.15 Gb
genomic size, that it about 150 million bases per chromosome.  The DNA
could be stretched out 2 meters.  About half a million proteins undergo
posttranslational modifications.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Jones P, Cote RG, Martens L, Quinn A, Taylor CF, Derache W, Hermjakob H,
Apweiler R
 </span>
 <span class="year">
(2006)
 </span>
 <span class="jname">
Nucl. Acids Res.
 </span>
 <span class="jvol">
34,
 </span>
 <span class="pages">
D659-D663.
 </span>
<p class="reftitl">
PRIDE: a public repository of protein and peptide
identifications for the proteomics community
<p class="abstract">
<b>ABSTRACT</b>.
PRIDE, the PRoteomics IDEntifications database
(http://www.ebi.ac.uk/pride) is a database of protein
 and peptide identifications that have been described
 in the scientific literature. These identifications will
 typically be from specific species, tissues and subcellular
 locations, perhaps under specific disease
 conditions. Any post-translational modifications
 that have been identified on individual peptides can
 be described. These identifications may be annotated
 with supporting mass spectra. At the time of writing,
 PRIDE includes the full set of identifications as submitted
 by individual laboratories participating in the
 HUPO Plasma Proteome Project and a profile of the
 human platelet proteome submitted by the University
 of Ghent in Belgium. Bylate 2005 PRIDE is expected to
 contain the identifications and spectra generated by
 the HUPO Brain Proteome Project. Proteomics laboratories
 are encouraged to submit their identifications
 and spectra to PRIDE to support their manuscript
 submissions to proteomics journals. Data can be
 submitted in PRIDE XML format if identifications are
 included or mzData format if the submitter is depositing
 mass spectra without identifications. PRIDE is a
 web application, so submission, searching and data
 retrieval can all be performed using an internet browser.
 PRIDE can be searched by experiment accession
 number, protein accession number, literature reference
 and sample parameters including species,
 tissue, sub-cellular location and disease state. Data
 can be retrieved as machine-readable PRIDE or
 mzData XML (the latter for mass spectra without
identifications), or as human-readable HTML.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Jones P, Cote RG, Martens L, Quinn A, Taylor CF, Derache W, Hermjakob H,
Apweiler R
 </span>
 <span class="year">
(2006)
 </span>
 <span class="jname">
Nucl. Acids Res.
 </span>
 <span class="jvol">
34,
 </span>
 <span class="pages">
D659-D663.
 </span>
<p class="reftitl">
MitoP2: the mitochondrial proteome database now including mouse data
<p class="abstract">
<b>ABSTRACT</b>.
The MitoP2 database (http://www.mitop.de) integrates
information on mitochondrial proteins, their
molecular functions and associated diseases. The
central database features are manually annotated
15 reference proteins localized or functionally associated
with mitochondria supplied for yeast, human
and mouse. MitoP2 enables (i) the identification of
putative orthologous proteins between these species
to study evolutionarily conserved functions and path-
20 ways; (ii) the integration of data from systematic
genome-wide studies such as proteomics and deletion
phenotype screening; (iii) the prediction of novel
mitochondrial proteins using data integration and the
assignment of evidence scores; and (iv) systematic
25 searches that aim to find the genes that underlie common
and rare mitochondrial diseases. The data and
analysis files are referenced to data sources in
PubMed and other online databases and can be easily
downloaded. MitoP2 users can explore the relation-
30 ship between mitochondrial dysfunctions and disease
and utilize this information to conduct
systems biology approaches on mitochondria.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Asara JM, Zhang X, Zheng B, Christofk HH, Wu N, Cantley LC
 </span>
 <span class="year">
(2006)
 </span>
 <span class="jname">
J Proteome Res.
 </span>
 <span class="jvol">
5,
 </span>
 <span class="pages">
155-163.
 </span>
<p class="reftitl">
In-Gel Stable-Isotope Labeling (ISIL): A Strategy for
Mass Spectrometry-Based Relative Quantification
<p class="abstract">
<b>ABSTRACT</b>.
Most proteomics approaches for relative quantification of protein
expression use a combination of stable-isotope labeling and
mass spectrometry. Traditionally, researchers have used difference
gel electrophoresis (DIGE) from stained 1D and 2D gels for
relative quantification. While differences in protein staining
intensity can often be visualized, abundant proteins can
obscure less abundant proteins, and quantification of
post-translational modifications is difficult. A method
is presented for quantifying changes in the abundance of
a specific protein or changes in specific modifications
of a protein using In-gel Stable-Isotope Labeling (ISIL).
Proteins extracted from any source (tissue, cell line,
immunoprecipitate, etc.), treated under two experimental
conditions, are resolved in separate lanes by gel electrophoresis.
The regions of interest (visualized by staining) are reacted
separately with light versus heavy isotope-labeled reagents,
and the gel slices are then mixed and digested with proteases.
The resulting peptides are then analyzed by LC-MS to determine
relative abundance of light/heavy isotope pairs and analyzed by
LC-MS/MS for identification of sequence and modifications.
The strategy compares well with other relative quantification
strategies, and in silico calculations reveal its effectiveness
as a global relative quantification strategy. An advantage of
ISIL is that visualization of gel differences can be used as
a first quantification step followed by accurate and sensitive
protein level stable-isotope labeling and mass spectrometry-based
relative quantification.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Ingvarsson J, Lindstedt M, Borrebaeck CA, Wingren C
 </span>
 <span class="year">
(2006)
 </span>
 <span class="jname">
J Proteome Res.
 </span>
 <span class="jvol">
5,
 </span>
 <span class="pages">
170-176.
 </span>
<p class="reftitl">
One-step fractionation of complex proteomes enables detection
of low abundant analytes using antibody-based microarrays
<p class="abstract">
<b>ABSTRACT</b>.
Antibody-based microarray is a novel technology with great promise
within high-throughput proteomics. The tremendous complexity
of all proteomes will, however, pose major technological challenges,
especially when targeting low-abundant analytes that remains
to be resolved. In this paper, we have shown that antibody microarrays
readily could be used for screening of low-abundant low molecular
weight analytes in complex proteomes by optimizing the sample format.
Focused antibody microarrays, based on human recombinant
single-chain Fv anti-cytokine antibodies on Ni2+-NTA
functionalized glass slides or black polymer Maxisorp substrates,
and crude cell supernatants from activated dendritic cells,
containing low levels of secreted cytokines, was used for
evaluation. The proteome was pre-fractionated based on size
in a simple one-step procedure using centrifugal filter devices
of various molecular weight cutoffs. The results showed that
the generation of a nondiluted low molecular weight (LMW) fraction,
corresponding to less than 2% of the original protein content,
was critical for the successful screening of cytokines in the
sub pg/mL range. The reduced complexity of the LMW fraction
significantly improved the assay sensitivity, by improving
the fluorescent tagging step and/or reducing the nonspecific
binding to the substrates.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Thongboonkerd V, Chutipongtanate S, Kanlaya R
 </span>
 <span class="year">
(2006)
 </span>
 <span class="jname">
J Proteome Res.
 </span>
 <span class="jvol">
5,
 </span>
 <span class="pages">
183-191.
 </span>
<p class="reftitl">
Systematic evaluation of sample preparation methods for
gel-based human urinary proteomics: quantity, quality, and variability.
<p class="abstract">
<b>ABSTRACT</b>.
We performed systematic evaluation of 38 protocols to concentrate
normal human urinary proteins prior to 2D-PAGE analysis.
Recovery yield and pattern of resolved protein spots were compared
among different methods and intra-/inter-individual variabilities
were examined. Precipitation with 90% ethanol provided the greatest
protein recovery yield (92.99%), whereas precipitation with
10% acetic acid had the least protein recovery (1.91%).
In most of precipitation protocols, the higher percentage of
applied organic compounds provided the greater recovery yield.
With a fixed concentration at 75%, the urine precipitated with
acetonitrile had the greatest number of protein spots visualized
in 2D gel, whereas the acetic-precipitated sample had the
smallest number of spots. For the intra-individual variability,
the first morning urine had the greatest amount of total protein
but provided the smallest number of protein spots visualized.
Excessive water drinking, not caffeine ingestion, caused alterations
in the urinary proteome profile with newly presenting spots and
also proteins with decreased excretion levels. As expected,
there was a considerable degree of inter-individual variability.
Coefficients of variation for albumin and transferrin expression
were greatest by inter-individual variables. Male urine had
greater amount of total protein but provided smaller number
of protein spots compared to female urine. These data offer
a wealth of useful information for designing a high-quality,
large-scale human urine proteome project.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Rauch A, Bellew M, Eng J, Fitzgibbon M, Holzman T, Hussey P,
Igra M, Maclean B, Lin CW, Detter A, Fang R, Faca V, Gafken P,
Zhang H, Whitaker J, States D, Hanash S, Paulovich A, McIntosh MW
 </span>
 <span class="year">
(2006)
 </span>
 <span class="jname">
J Proteome Res.
 </span>
 <span class="jvol">
5,
 </span>
 <span class="pages">
112-121.
 </span>
<p class="reftitl">
Computational Proteomics Analysis System (CPAS): an extensible,
open-source analytic system for evaluating and publishing proteomic
data and high throughput biological experiments
<p class="abstract">
<b>ABSTRACT</b>.
The open-source Computational Proteomics Analysis System (CPAS)
contains an entire data analysis and management pipeline for
Liquid Chromatography Tandem Mass Spectrometry (LC-MS/MS) proteomics,
including experiment annotation, protein database searching and
sequence management, and mining LC-MS/MS peptide and protein
identifications. CPAS architecture and features, such as
a general experiment annotation component, installation software,
and data security management, make it useful for collaborative
projects across geographical locations and for proteomics
laboratories without substantial computational support.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Nettikadan S, Radke K, Johnson J, Xu J, Lynch M, Mosher C, Henderson E
 </span>
 <span class="year">
(2006)
 </span>
 <span class="jname">
Molec. Cell Proteomics
 </span>
 <span class="jvol">
19 Feb 2006
 </span>
 <span class="pages">
xxx-xxx.
 </span>
<p class="reftitl">
Detection and Quantification of Protein Biomarkers from Fewer Than Ten Cells
<p class="Para1">
The substrates are silicon wafers patterned by reactive ion etching (RIE)
and diced into 4 mm squares (chips), then ultrasonicated in water
and absolute EtOH for cleaning, sputter-coated with 5 nm chromium and 10 nm
gold with dual gun ion beam sputterer operating at 4 mA and 7000 eV.
Reactive amine surfaces were created on the gold coat by immediate immersion
in prolinker B (Proteogen) in chloroform for an hour.  Surfaces are washed
in chloroform, then acetone, blown dry with nitrogen and stored under
nitrogen.  Mouse and goat antibodies to PSA are used for capture and detection,
respectively.  Rat and biotinylated anti-IL-6 are used for capture and
detection, respectively.  Fluorescence is generated by secondary
antibodies (donkey anti-goat Texas Red) and Alexa 594 strepavidin.
Alexa 488 rabbit anti-rat IgG is used to tag capture antibodies.
Ultramicroarrays are made with a NanoArrayer, a system including microfabricated
<q>print cartridges</q> (surface patterning tools, SPTs), having
precise motion and environmental control to transfer atto- to femtoliter
volumes from reservoir on the SPT on to a surface.  Protein is deposited
as a 1:1 mix (20% glycerol + 0.2 mg/ml n-octylglucose)
<p>
Capture antibodies are deposited in ultramicro domains and immobilized using
amine-reactive chemistry.  Antigens were captured and detected using
sandwich assay and immunoflourescence microscopy.  Direct imaging rather than
scanning of ultraminiaturized arrays makes data collection rapid and automatic
internal control with respect to instrument variability.  Mouse anti-PSA
and rat anti-IL 6 ultramicroarrays were constructed adjacent to each other,
both arrays with a 3 &times; 6 format and 15 &micro;m pitch and spot
size of 5-8 &micro;m dia.  Anti-rat IgG was Alexa 488-conjugated and also
cross-reacts with mouse IgG.  Fluorescent images show that capture
antibodies bound only to expected locales.
<p>
Detection sensitivity of the anti-PSA ultramicroarray was compared with
multiple PSA capture chips.  Chips were incubated with serial log dilutions
of PSA, with 3 chips per concentration.  The net (raw &minus; background)
fluorescence and data averaged (at least 15 datasets in the average).
Results show detection of PSA by the ultramicroarray at 10 pg/ml from
1 &micro;l, attomole sensitivity shown.  LNCaP cells produce PSA
constitutively.  Ten replicate 2-fold serial dilutions wre done starting
from 2000 cells per well and these incubated 24 h.  Media from each
cell dilution was pooled and stored.  Small volume of pooled media
is mixed with PBST buffer (1:1)
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Taylor AM, Holst O, Thomas-Oates J
 </span>
 <span class="year">
(2006)
 </span>
 <span class="jname">
Proteomics
 </span>
 <span class="jvol">
6,
 </span>
 <span class="pages">
xxx-xxx.
 </span>
<p class="reftitl">
Mass Spectrometric Profiling of <i>O</i>-linked Glycans Released Directly
From Glycoproteins in Gels Using In-Gel Reductive &beta;-Elimination
<p class="Para1">
For this work, 1 mg dry mucin (rich in O-linked glycans) from bovine
submaxillary mucin Type I-S was
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
L&oacute;pez-Ferrer D, Ramos-Fern&aacute;ndez A, Mart&iacute;nez-Bartolom&eacute; S,
Garc&iacute;a-Ruiz P, V&aacute;zquez J
 </span>
 <span class="year">
(2006)
 </span>
 <span class="jname">
Proteomics
 </span>
 <span class="jvol">
6,
 </span>
 <span class="pages">
xxx-xxx.
 </span>
<p class="reftitl">
Quantitative Proteomics Using <sup>16</sup>O/<sup>18</sup>O Labeling
and Linear Ion Trap Mass Spectrometry
<p class="Para1">
Standardized protocols for trypsin digestion with heavy oxygen have not
really been described.   Methods vary from sample prep to pH of digestion
to presence of salts and organic solvents.  BSA was dissolved in 8 M urea
with 25 mM ammonium bicarbonate and 10 mM DTT pH 8 for 60 min at 37&deg;.
After reduction, iodoacetamide was added to 50 mM and this incubated
at room temp in dark for 45 min.  A 4-fold dilution was made before
addition of trypsin at 1:25 overnight at 37&deg;.  1% TFA was added and
then sample lyophilized.  When labeling was done 5 &micro;g BSA
5 &micro;l 20% MeCN, 5 &micro;l 100 mM ammonium acetate pH 6.75,
5 &micro;l 50 mM CaCl<sub>2</sub>, and 0.2 &micro;g trypsin was added.
The mixture is then dried in vacuum centrifuge quickly.  To the sample
is added 5 &micro;l H<sub>2</sub><sup>16</sup>O or H<sub>2</sub><sup>18</sup>O
(Sigma, 95%) in 20% MeCN, and incubated 37&deg; for 24 h.  1% formic
acid is added to stop labeling.
<p>
The assumption is that one <sup>18</sup>O is incorporated into the COOH
of the C-terminal tryptic peptide.  The following equation helps to
understand the ratio of light to heavy isotope from the first five peaks
of an isotopic cluster:
<table class="eqnserif">
<tr><td rowspan="2">ratio<span style="font-size:200%;">(</span>
 <td class="numer"><sup>16</sup>O
 <td rowspan="2"><span style="font-size:200%;">)</span> =
 <td class="numer"><i>I</i><sub>0</sub>
<tr><td><sup>18</sup>O
 <td>
 <table class="eqnserif" style="font-size:80%;margin-top:0;">
 <tr><td rowspan="2"><i>I</i><sub>4</sub> &minus;
  <td class="numer"><i>M</i><sub>4</sub>
  <td rowspan="2"><i>I</i><sub>0</sub> + <i>I</i><sub>2</sub> (1
   &minus;
  <td class="numer"><i>M</i><sub>2</sub>
  <td rowspan="2">) &minus; (1 &minus;
  <td class="numer"><i>M</i><sub>2</sub>
  <td rowspan="2">)
  <td class="numer"><i>M</i><sub>2</sub>
  <td rowspan="2"><i>I</i><sub>0</sub>
 <tr><td><i>M</i><sub>0</sub>
  <td><i>M</i><sub>0</sub>
  <td><i>M</i><sub>0</sub>
  <td><i>M</i><sub>0</sub>
 </table>
</table>
<p>
where <i>I</i><sub>0</sub>, <i>I</i><sub>2</sub>, and <i>I</i><sub>4</sub>
are the relative intensities of the 1st, 3rd, and 5th peaks in the cluster,
and <i>M</i><sub>0</sub>, <i>M</i><sub>2</sub>, and <i>M</i><sub>4</sub>
are the theoretical relative intensities determined from the relative
abundance of the natural isotopes of C, O, H, N, and S.
<p>
BSA was treated in trypsin and examined by MALDI-ToF.  Without use of calcium
salts and organic solvents, complete incorporation of at least one <sup>18</sup>O
was seen in some peptides.  Adding calcium improved the labeling efficiency.
With the addition of an organic component (MeCN or DMSO), a complete
incorporation of at least one <sup>18</sup>O was seen.  Lowering the
pH to 6.75 did not help.  Problems with the use of DMSO included its
artifactual reactions with amine groups and N-termini and its lower degree
of purity and difficulty in removal, so MeCN was preferred in the final
reaction.
<p>
The use of mildly acidic conditions through sample processing did not
cause back-exchange of <sup>18</sup>O to <sup>16</sup>O.  Quenching the sample
with formic acid and storage in <sup>18</sup>O-water at &minus;20&deg;.
It was not possible to incorporate two <sup>18</sup>O atoms as achieved
by other researchers.  There is some information specific to the use of
ion trap mass spectrometers, settings for resolution.
<p>
Extracts from the nuclear proteome of stem cells were labeled during
digestion and the labeled sample analysis online by HPLC-LIT MS, and
the selected peptide MS/MS.  SEQUEST was used to search fragment spectra
databases.  Most peptides showed incorporation of &gt; 95%.  Completely
unlabeled peptides had not C-terminal basic residue and represented
those not digested by trypsin.
<p>
Labeled and unlabeled samples were mixed in different proportions to examine
the linearity of the method.  A plot of percent <sup>18</sup>O labeling
vs. percent labeled sample revealed a high degree of correlation (<i>r</i>
<sup>2</sup> = 0.9958).
<p>
High resolution scanning modes of ion trap MS are not used in quantitative
proteomics, so the robustness was tested in large-scale peptide analysis.
Two MSC cell cultures were prepared and nuclear extracts and protein
concentrations adjusted.  10 &micro;g each of the nuclear extracts were
trypsinized, desalted, dried and then subject to labeling either with or
without heavy isotope with trypsin present in the optimized protocol.
An RP-HPLC method with 5 hour gradient was used rather than a multidimensional
setup with SCX column.  8000 high-res scans were done with the ion trap
spending much time in MS/MS mode, identifying 390 peptides with a false
discovery rate (FDR) lower than 5%, and 308 peptides with FDR &lt; 1%.
227 and 183 peptides were unique, respectively.  Most identified proteins
were related to DNA/RNA replication, binding, and transcriptional regulation.
For proteins identified with FDR &lt;5%, 221 had good ZoomScan spectra
to allow quantitation (196 with FDR &lt; 1%), with 135 unique proteins
(120 with FDR &lt; 1%).  This method could find quantitative differences
of 0.5 or 2-fold at a 95% confidence level.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Xue Y, Li A, Wang L, Feng H, Yao X
 </span>
 <span class="year">
(2006)
 </span>
 <span class="jname">
BMC Bioinformatics
 </span>
 <span class="jvol">
7,
 </span>
 <span class="pages">
163 (11 pages)
 </span>
<p class="reftitl">
PPSP: Prediction of PK-Specific Phosphorylation Site with Bayesian Decision
Theory
<p class="Para1">
Protein phosphorylation one of most common posttranslational modifications,
playing crucial role in many biological processes:  transcription, translation,
mitosis/cell cycle, neurite outgrowth, cell signalling. Protein kinases
(PKs) target specific sites having a consensus sequence flanking with
canonical motif.  Consensus motifs for 30 PKs have been reported.  But
a consensus motif is sufficient but not necessary condition for phosphorylation
in vivo.  Many hypotheses have suggested that specificity is conferred by
co-complexes of PK with substrates, subcellular co-localization, interaction
with modular docking sites, phosphopeptide-binding mechanisms, and so on.
Many PKs are held to a scaffold protein, or to the substrate itself at a docking
site distinct from the phosphorylation site

<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Larsericsdotter H, Jansson &Ouml; Zhukov A, Areskoug D, Oscarsson S, Buijs J
 </span>
 <span class="year">
(2006)
 </span>
 <span class="jname">
Proteomics
 </span>
 <span class="jvol">
6,
 </span>
 <span class="pages">
0000-0000.
 </span>
<p class="reftitl">
Optimizing The Surface Plasmon Resonance/Mass Spectrometry Interface for
Functional Proteomics Applications:  How To Avoid and Utilize Nonspecific
Adsorption
<p class="Para1">
The yeast two-hybrid (Y2H) system is a means for mapping protein interaction
networks and large scale screening of those interactions.  Limitations
include the difficulties of revealing multiple component interactions,
those not involving proteins and interactions involving post-translational
modifications or built from multiple peptide chains.  Chip-based technologies
have emerged to cover part of the deficit of Y2H.  Surface plasmon
resonance (SPR)-based biosensors will detect mass changes on a sensor
surface when ligand in solution binds to a surface-immobilized receptor,
thus showing molecular interaction.  The technique typically involves
ligand fishing with a receptor of interest.  Biacore 3000 is a chip system
with that purpose in mind.  The bait molecule is immobilized on a sensor
surface and potential targets are mixed with it.  MS-compatible buffer
is used to wash away unbound without disturbing the interaction.
<p>
<b>SPR</b>.  Polarized light is sent to a prism at an angle causing
reflection rather than refraction.  The prism is placed at the underside
of a sensor chip.  Mounted on the fluidic side of the
sensor chip are immobilized receptors.  Flowing over the receptors on the
chip is a buffer solution containing potential receptor ligands.
The sensor chip is composed of gold layer that is electrically conducting.
The glass of the sensor surface has a high refractive index, while the
buffer flowing past as a low refractive index.  A detector is set up to
monitor the signal of reflected light.  When the light strikes the
glass, an <b>evanescent wave</b> is generated, corresponding to the
electric field vector of the light wave.  This wave interacts or is
absorbed by the conductive layer of electrons in the gold foil,
and this results in electron charge density waves (<b>plasmons</b>)
being formed.  These plasmons result in a reduction in the intensity
of the reflected light.  The angle of the incident light where
resonance is maximum and at which the reflected intensity is therefore
minimum is a function of the refractive index of the solution close
to the gold layer on the opposing facee of the sensor surface.
<p>
One problem with chip-based affinity separations and microfluidic devices
is the high nonspecific adsorption where surface area to volume ratio is
quite large.  Nonionic detergents are usually used to reduce this effect,
but these are MS-incompatible.  Salts are usually present because they
stabilize target-ligand interaction, but complicate MS analysis.
<p>
As a model for this study BSA is used because of its very high nonspecific
adsorption to many surfaces.  FITC-BSA was dissolved in HBS-N buffer
(10 mM HEPES pH 7.4 + 150 mM NaCl) and injected at 1 mg/ml  into four
flow cells for 5 min @ 10 &micro;l/min into a Sensor Chip CM5 mounted
into Biacore 3000.  Different wash solutions and MS-compatible buffers
were then used.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Walz A, St&uuml;hler K, Wattenberg A, Hawranke E, Meyer HE, Schmalz G,
Bl&uuml;ggel M, Ruhl S
 </span>
 <span class="year">
(2006)
 </span>
 <span class="jname">
Proteomics
 </span>
 <span class="jvol">
6,
 </span>
 <span class="pages">
1631-1639.
 </span>
<p class="reftitl">
Proteome Analysis of Glandular Parotid and Submandibular-Sublingual Saliva
in Comparison to Whole Human Saliva by Two-Dimensional Gel Electrophoresis
<p class="Para1">
Two female and two male, all healthy, provided all samples used in the
project; protein ID was from one of the females.  Saliva samples were
kept on ice.  Saliva was filter through 5 &micro;m and then through
0.2 &micro;m to remove bacteria.  Specific secretions from parotid
were obtained by Lashley cups by stimulating tongue with 2% citric acid.
The cups were then used to block parotid and cotton wool rolls used to
isolate sublingual area;  a 50 ml syringe with sterile silicon tube then
used to get the SMSL saliva.  Amicon Ultra with 5000 Da cutoff were used
to desalt and concentrate, providing final 5-25 mg/ml protein.  BCA assay
was used to assess protein.  Samples were stored at &minus;80&deg;.
<p>
IEF was on pH 3-10 NL 18 cm strip rehydrated with 300 &micro;g protein
in a 350 &micro;l buffer of 7 M urea, 2 M thiourea, 2% CHAPS, 1% DTT,
1% 3-10 ampholytes.  PRP-1 was a gift and used as a control (10 &micro;g
was dissolved in buffer and run).  Strip rehydration was active (30 V
overnight), followed by run start at 200 V for 60 min, 500 V and 1000 V
over 1 h, then a 30 min sloped rise to 8000 V, holding for 3 h.  2nd
dimension was in 12.5% T, 3% C gel (EttanDALT, Amersham), run at 1 W/gel
over 16 h at 25&deg;.
<p>
2-D gels of whole saliva were run and stained in Coomassie Blue R-250
(fix was 20% TCA for 60 min then staining in 0.1% R-250 in 40% EtOH/10%
acetic acid for 2 h).  In the middle of the gel at 60 kDa is salivary
amylase and there are proline-rich proteins.  PRPs are diffuse pink spots
in the acidic region (pI 3.2 - 4.9).  Other at the 20-37 kDa level appear
in the pI 5.3-6.2 region are are PRPs.  In the basic region, very near
pI 10, a spot at 25 kDa and one at 70-75 kDa appear, staining pink like
PRPs.  These spots only appear if the saliva is filtered with a 0.2 &micro;m
membrane.  192 spots were picked from the gel and digested with trypsin.
131 were identified by MALDI-MS.  Cyclophilin B (pI ~9.5/17-18 kDa)
was seen, as was prolyl-4-hydroxylase (pI 4-5/65 kDa), acidic PRP, basic
PRP, and the proline-rich glycoprotein at pI 10/70-75 kDa.  The salivary
amylase and PRPs are the major/dominant proteins.
<p>
Specific glandular secretions were done.  58 parotid gland spots were analyzed
and 43 identified.  63 spots from submandibular-sublingual (SMSL) were
analyzed and 53 identified.  Most spots were identical.  5 proteins found
in parotid not detected in whole saliva:  HSPC059 (pI 6-7, 25 kDa),
Ig&alpha;-2 heavy chain (pI 4.5-5, 55-60 kDa), Ig&kappa; light chain VLJ
region (pI 7.5-8, 25 kDa), lactoperoxidase (pI 7.5-8, 80 kDa), salivary
acidic proline-rich phosphoprotein 1/2 precursor (pI 4, 33 kDa).  In the
SMSL saliva spots were found not seen in the whole saliva:  CST1 (pI 6.5-7,
15 kDa), cystatin S (pI 3.5-4, 17-20 kDa), histatin 1 (pI 5-6.5, 10-12 kDa),
MUC7 (pI 3, 120 kDa), and 0508206A peptide PA, a PRP (pI 8.5-9, 13-15 kDa).
Lower amounts of salivary amylase were in the SMSL secretions;  also PRG
and basic PRPs were not in the basic pI region.
<p>
PRP-1 was purified and run on 2DGE.  Two pink spots at a pI ~4.5 and
~22 kDa were revealed. MS and MS/MS spectra indicated salivary acidic
proline-rich phosphoprotein 1/2 precursor (PRP-1).  These spots were
distinguished from those in the whole saliva and specific gland secretions.
Additional peptide of 2136 Da was not present in purified PRP-1.  MS/MS
spectra showed it to be PRP type Db (gene <i>PRH1</i>).  This is missing
a 21-aa repeat in the PRP-1 precursor.  In that MW range, from pI 3.2-6.2
there are spots seen in a horizontal line which are Db protein spots.
<p>
In parotid more acidic PRPs are seen of this PRP precursor in the pI 3-4
range.  N-terminus of the PRPs is not detectable in MALDI because it is
too large (not enough cleavage sites).
<p>
Samples from a single patient were collected on three different days to
see how the gel pattern might change.  The pattern rarely changed for
whole saliva, and parotid and SMSL secretions.
<p>
Samples from different individuals were then analyzed.  There were clear
differences both for whole saliva and for particular secretions.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Zhang X, Scalf M, Berggren TW, Westphall MS, Smith LM
 </span>
 <span class="year">
(2006)
 </span>
 <span class="jname">
J. Am. Soc. Mass Spectrom.
 </span>
 <span class="jvol">
17,
 </span>
 <span class="pages">
490-499.
 </span>
<p class="reftitl">
Identification of Mammalian Cell Lines Using MALDI-TOF and LC-ESI-MS/MS
Mass Spectrometry
<p class="Para1">
Except for BHK21, all cells are grown in RPMI-1640 + 1% Gln, 1% pen-strep,
and 15% FBS. BHK21 cells are grown in DMEM/F-12 with 5% FBS and 60 &micro;g/ml
pen-strep.  K562 cells from ATCC are a human myelo-monocytic continuous line
derived from chronic myelogenous leukemic patient.  About 1 million cells
are pelleted (1600 rpm/5 min at 5&deg;).  the pellet is resuspended in
100 &micro;l 10 mg/ml DHB (after  washing twice with it).  These are
then spotted on MALDI target.  Large number of discrete peaks appear in the
<i>m/z</i> 1000 to 20,000 range.  The same peak pattern is observed no
matter what the passage (three successive and 20 discontinuous on K562 cells).
Even with changes in protein expression because of growth conditions and
nonsynchronized cell growth cycles, a distinct mass spectral signature
is constant.
<p>
SDS-PAGE was also done, with a BSA band (from culture medium) apparent
and several less intense bands in the 30-60 kDa region.  High intensity
ions were all below 22 kDa.  The only peak above 22 kDa with any intensity
was the 66 kDa. Ions at higher <i>m/z</i> are not observed because
of less efficiency of ionization and lower detection efficiency of the
instrument.
<p>
DHB may be a better matrix than CHCA or SA as it stabilizes cell membranes
and eliminates salt effect.  CHCA produced matrix interference in the
<i>m/z</i> to 0-10,000 range, less so with SA in the 0-5000 range.
Pure DHB was best, but a mix of DHB and SA also was acceptable.
<p>
Cells are infiltrated with matrix and then centrifuged.  It is possible
the treatment causes lysis of cells with loss of protein in the supernatant.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Efrat A, Hoffmann F, Kriegel K, Schultz C, Wenk C
 </span>
 <span class="year">
(????)
 </span>
 <span class="jname">
no journal
 </span>
 <span class="jvol">
&nbsp;
 </span>
 <span class="pages">
&nbsp;
 </span>
<p class="reftitl">
Geometric Algorithms for the Analysis of 2D-Electrophoresis Gels
<p class="Para1">
Computational geometry techniques can be appled to image analysis with
respect to <b>spot detection</b> and <b>gel matching</b>.  Algorithms
used to implement the CAROL 2-DE analysis software system are explored.
With spot detection, problems arise such as interpreting twin spots,
streaks, and complex regions representing a union of ellipses.  In particular
there is the problem of dealing with saturated stains as well.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Wong DT
 </span>
 <span class="year">
(2006)
 </span>
 <span class="jname">
J. Am. Dental Assoc.
 </span>
 <span class="jvol">
137,
 </span>
 <span class="pages">
313-321.
 </span>
<p class="reftitl">
Salivary Diagnostics Powered by Nanotechnologies, Proteomics, and Genomics
<p class="Para1">
Saliva as a biofluid with diagnostic and health surveillance potential is
about five years away.  Saliva contains steroid hormones and HIV antibody
which can provide useful information.  Efforts are being made to use saliva
to test IL-1 genetic allele composition to susceptibility to oral pathogens
and lectin staining for oral risk.  The anatomy of the salivary gland
duct will be important.  The gland is surrounded by a vascular-rich
set of blood vessels.  From the blood and surrounding interstitial space,
acinar cells of the gland culdesac actively transport selective compounds
into the duct, and there is passive diffusion of lipid soluble compounds,
a simple filtration of selected compounds, and the active transport of
sodium ion (Na<sup>+</sup>) also moves water into the duct.  As the duct
proceeds from the acinar area, the cells are more cuboidal and these duct
cells transport the sodium out to make the saliva hypotonic.
<p>
OFNASET is the Oral Fluid NanoSensor Test.  This is intended to be a handheld
automated system that will allow simultaneous multiple salivary biomarker
analysis for identified biomarkers.  309 proteins have been found in the
human saliva which might be important diagnostically.  In addition
there is a transcriptome with about 3000 mRNA species, in which 180 commonly
occur in the healthy patients.  Consider patients with oral cancer.
In particular four genes (IL-8, Orn decarboxylase, spermidine acetyltransferase,
and IL-1&beta;) could be used to distinguish between cancer and healthy
patients, with sensitivity and specificity of 91% (ROC=0.95).
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Govorukhina NI, Reijmers TH, Nyangoma SO, van der Zee AGJ, Jansen RC, Bischoff R
 </span>
 <span class="year">
(2006)
 </span>
 <span class="jname">
J. Chromatog. A
 </span>
 <span class="jvol">
xxx,
 </span>
 <span class="pages">
xxx-xxx.
 </span>
<p class="reftitl">
Analysis of Human Serum by Liquid Chromatotgraphy-Mass Spectrometry: Improved
Sample Preparation and Data Analysis
<p class="Para1">
LC-MS dataset complexity can approach 100 million data points if the
spectral resolution is 0.1 amu in the <i>m/z</i> domain and the run
time is about 2 h (7000 data points).  Preprocessing algorithms reduce
complexity and multivariate statistics reduce dimensionality.
<p>
Serum was obtained and stored at &minus;80&deg; for long term, short term
at &minus;20&deg;.  Serum from a single cervical cancer patient with
squamous cell carcinoma antigen-1 (SCCA-1) level of 2.2 ng/ml (by ELISA)
was used.  60 &micro;l of serum was diluted with 180 &micro;l of 20 mM
pH 7 sodium phosphate and passed through 1 ml Bio-Rad Aurum Serum Protein
column
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Miller CS, King CP Jr, Langub M, Kryscio RJ, Thomas MV
 </span>
 <span class="year">
(2006)
 </span>
 <span class="jname">
J. Am. Dental Assoc.
 </span>
 <span class="jvol">
137,
 </span>
 <span class="pages">
322-329.
 </span>
<p class="reftitl">
Salivary Biomarkers of Existing Periodontal Disease:  A Cross-Sectional Study
<p class="Para1">
Saliva is produced by three major pairs of glands and hundreds of minor
glands in the mucosal surfaces of the mouth.  The secretions contain
proteins made both by the duct cells as well as those transported from
the blood (other sources/systems).  Blood, serum, serum products,
gingival crevicular fluid (GCF), electrolytes, epithelial and immune
cells, microorganisms, bronchial products, are present.
<p>
Some markers that are present should be indicators of disease, such as
periodontitis.  IL-1&beta; is a proinflammatory cytokine that stimulates
adhesion molecule expression and which encourages an inflammatory response.
Matrix metalloproteinase 8 (MMP-8) is involved in collagen matrix degradation,
so that polymorphonuclear leukocytes can gain entry during acute stages
of periodontal disease.  OPG is a glycoprotein is a decoy receptor which
also inhibits osteoclast differentiaton, binding the receptor activator
of the NF-&kappa;B ligand (RANKL).
<p>
Adults (age &gt; 18 y) in good health but with 20 or more erupted teeth
were enrolled through advertisements.  12 men and 16 women with existing
generalized moderate-to-severe periodontitis were part of the study.
Enrollment criteria:
<ul>
<li>At least 30% of peridontal probing caused bleeding
<li>At least 20% of probing depths were 4 mm or more
<li>At least 5% of periodontal sites had interproximal clinical attachment
loss (CAL) greater than 2 mm and radiographic bone loss was seen in
posterior vertical bitewing films
</ul>
<p>
Controls were 29 healthy adults of similar age, race, sex with no BOP
in less than 10% of periodontal sites, had probing depth of 5 mm or greater
in less than 2% of sites, had no probing depth &ge; 6 mm in any site,
had CAL of &gt; 2 mm in &lt; 1% of periodontal sites, and had not radiographic
bone less in any posterior bitewing view.  Patients with histories of
alcoholism, liver or kidney dysfunction, inflammatory bowel disease,
granulomatous disease or immunosuppression or undergoing organ transplantation
or cancer therapy were excluded.  Pregnancy, use of antibiotics or
immunosuppressants within last 6 months and many other exclusion criteria
are detailed in the text.
<p>
Saliva were collected from whole sample, unstimulated, by expectoration.
Patients first wash with tap water, then spit into sterile tubes.
Samples are immediately iced, then aliquoted and stored &minus;80&deg;.
Samples are all processed within 6 months.
<p>
The three salivary biomarkers described above were then assayed using
ELISA or EIA for IL-1&beta;, MMP-8, OPG in duplicate.  Kits were from
R &amp; D Systems or ALPCO Diagnostics.
<p>
Table 1 provides all the patient characteristic data (age range, mean
age, sex, race, smokers, drinkers, and the evaulation of the gums with
the four parameters (probing depths, CAL, bleeding on probing).
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Leptos KC, Sarracino DA, Jaffe JD, Krastins B, Church GM
 </span>
 <span class="year">
(2006)
 </span>
 <span class="jname">
Proteomics
 </span>
 <span class="jvol">
6,
 </span>
 <span class="pages">
1770-1782.
 </span>
<p class="reftitl">
MapQuant:  Open-Source Software for Large-Scale Protein Quantification
<p class="Para1">
Tryptic peptide samples of BSA, chicken conalbumin, bovine lactoperoxidase,
<i>E. coli</i> tryptophanase, human acid glycoprotein, rabbit aldolase,
yeast phosphoglucose isomerase were purchased and made up in 5% MeCN +
0.1% formic acid.  A ThermoElectron HPLC gradient pump, LCPacking autosampler,
and LCQ Deca XP + IT MS  (ThermoElectron) were used.  A 75 &micro;m capillary
was pulled and filled with 15 cm of Magic C<sub>18</sub> to 15 cm length
(5 &micro;m  200 &Aring; AQ-type).  Mobile phase is 0.1% formic acid
(solvent A) and 0.1% FA in MeCN (solvent B), using a 5-35% B gradient over
130 min with 85 &micro;l/min flow and 35-95% B rate over 30 min at 125
&micro;l/min.  1, 3.3, 10, 33, 100, 333, and 1000 fmol BSA was used in
calibration.  Low resolution analysis were done with four scans:  3 scans
in a full profile with <i>m/z</i> interval of 0.067 that included
one MS/MS scan per MS scan for quantification (q-experiments) purposes,
and then one scan in centroid mode with acquisition including five
MS/MS scans per MS scan for sequencing (s-experiments) purposes.  There
were about 3500 MS scans with a single chromatographic run.  The high
resolution MS runs were done on a hybrid linear IT/FTICR (ThermoElectron
LTQ-FT)  Signal acquisition for each data point was triplicated
and included two MS/MS scans in centroid mode for each MS scan in full
profile mode (average <i>m/z</i>interval = 0.0048).  These were used for
both quantification and sequencing.  A different chromatography (steeper
gradient, shorter program) was used, and there were ~1300 MS scans per run.
<p>
An application programming interface provided by ThermoElectron was used
to extract the LC/MS data into a structure called <q>OpenRaw</q>.
The OpenRaw structure holds archives of full scan MS and the higher
order CID spectra (MS<sup>n</sup>), in addition to the usual identifying
information about the experiment.
<p>
The MapQuant program will find unique organic species and quantify their
relative abundance from an LC-MS experiment.   Data from the experiment
is formatted as a 2-D map which shows a grayscale image.   The gray
levels represent a 3rd dimension in which intensity data from the
scans as a chromatogram is combined with the intensity data of the mass
spectrum.  The 2-D plot itself has <i>m/z</i> values on the ordinate and
the retention time on the abscissa.  The separation dimensions are
orthogonal as they describe two independent properties of the peptide:
mass and hydrophobicity (as a function of retention time on a reversed
phase column).  MSView, Spectromania, MosaiquesVisu are commercial programs
and MSight is a free program to look at large datasets but methods are
not sufficiently transparent (open source).
<p>
To produce the map, the following steps are taken:
<ol>
<li>smoothing by convolution
<li>watershed segmentation
<li>peak finding and fitting
<li>peak clustering
<li>peak refinement (i.e., deconvolving by fitting and subtracting) and
deisotoping
</ol>
<p>
The algorithms are ANSI C and the CLI (<i>MQParser</i>) was developed with
<i>bison</i>.  The MapQuant functions can be assembled into scripts readable
by MQParser.
<p>
An <i>experiment</i> is a data structure with information about a single
LC/MS run, with retention times and constituent MS scans, and the instrument
<i>m/z</i> sampling resolution.  A <i>scan</i> is a sampling unit in the
chromatography dimension that points to one mass spectrum, and each scan
has a time value.  The <i>mass bin</i> is the sampling unit of the
instrument when it measures the <i>m/z</i> of the ions.  The <i>2-D map</i>
is the matrix with rows and columns coinciding with the intensity values
of the mass spectra and the mass chromatograms of the experiment.
The <i>segment map</i> is a region in the 2-D map in which a segmentation
operation was done;  segmentation is done to partition the map into
tractable segments.  A <i>peak</i> is a local maximum in a 2-D map.
A <i>fitted peak (FPeak)</i> is a peak fitted to a particular mathematical
model.  An <i>isotopic cluster</i> is a cluster of fitted peak representing
candidate co-eluting isotopic clusters.  A <i>peak group</i> is a cluster
of fitted peaks representing candidate co-eluting isotopic clusters.
And a <i>peak group map</i> is the minimal 2-D map required to fit
the estimated number of isotopic clusters that a peak group might contain.
<p>
Data filtering is done, particularly smoothing by convolution since
this helps to detect local maxima (peaks).  Convolution functions include
box-car, Gaussian, and Savitzky-Golay.  With the low resolution spectrometer
a Gaussian was done on the retention time dimension:  the width of the filter
has an SD which is the average SD of the peaks.  The average peak SD will
not change significantly among  numerous runs of the same dataset since
it is dependent on chromatographic gradient.
<p>
The 2-D map is segmented using a watershed segmentation algorithm because
it is computationally too expensive to fill all 2-D peaks simultaneously.
The algorithm takes the noise-filtered map and produces a 2-D
labeled non-grayscale map in the form of a mosaic.  The segment is demarcated
by a rectangular circumscribed boundary which can be used to cut the segment
out of the map.  Well-resolved peaks usually get their own segment, while
overlapping peaks get grouped into a common segment.  A morphological
image opening operation is done in the overlapping peaks cases, prior to
the segmentation.  Confining overlapping peaks into same segment is required
during peak fitting since it is more accurate with simultaneous fitting.
<p>
Peak detection is done subsequent to segmentation.  Local maxima are
searched in every segment map, and their positions used as seeds for
the peak- (curve-) fitting algorithm.  Peak detection makes use of
mathematical morphology such as the <i>structuring element</i>.  A structuring
element is a small binary image <i>N</i> that an image operater (^) takes
as input along with the image of interest <i>I</i>, producing a binary
image <i>T</i>:
<p class="eqnserif">
<i>T = I</i> ^ <i>N</i>
<p>
Within the structuring element <i>N</i> is a sub-element <i>N<sub>i</sub></i>
with a value = 1 if it is to be used in deciding which points neighboring
each data point in the map are to be used in the image operation.  A
data point <i>s<sub>k</sub></i> is a local maximum <i>only if</i> the
value of <i>t<sub>k</sub></i> = 1:
<table class="eqnserif">
<tr><td rowspan="2"><i>t<sub>k</sub></i> = <span style="font-size:200%;">{</span>
 <td>1 <i>if</i> { &sum;
<sub style="margin-left:-0.75em;vertical-align:-1.25em;"><i>i</i></sub>
  &Lambda;(<i>s<sub>k</sub></i>, <i>s<sub>k</sub> N<sub>i</sub></i> } = |<i>N</i>|,
<tr><td style="text-align:left;">0 otherwise
</table>
<p>
where &Lambda;(<i>p, q</i>) = 1 if <i>p</i> &ge; <i>q</i>, or 0 otherwise,
and |<i>N</i>| = &sum;
<sub style="margin-left:-0.75em;vertical-align:-1em;"><i>i</i></sub>.
A threshold for abundance is set to avoid detection of maxima representing
noise; a threshold is set based on the mean, median, or standard deviation
of the 2-D map with and without considering zero values.  In this particular
study the threshold is set to the median plus two or three average absolute
deviations from the median, depending on the spectrometer.  Candidate peaks
are compiled into a list and fitted as a sum of curves described by a single
equation.  In a segment map if <i>n</i> candidate peaks, and if each peak
is fitted as curve <i>C</i>, the whole segment-map is fitted as
&sum; <sub style="margin-left:-0.75em;vertical-align:-1em;">i</sub>
<sup style="margin-left:-0.75em;vertical-align:1.2em;"><i>n</i></sup>
<i>C</i><sub>i</sub>.  Each peak was fit with a 2-D Gaussian (gaussioid),
which is bivariate with respect to retention time (<i>r</i>) and <i>m/z</i>
(<i>m</i>), described by:
<table class="eqnserif" style="text-align:center;">
<tr><td rowspan="2"><i>f</i>(<i>m, r; A, r</i><sub>0</sub>, <i>m</i><sub>0</sub>,
&sigma;<sub>m</sub>, &sigma;<sub>r</sub>) =
 <td style="border-bottom:1px solid black;"><i>A</i>
 <td rowspan="2">&nbsp;<i>e</i><sup>&minus;(<i>r &minus;
			r</i><sub>0</sub>)<sup>2</sup>/2&sigma;<sub>r</sub><sup>2</sup></sup>
	<i>e</i><sup>&minus;(<i>m &minus; m</i><sub>0</sub>)<sup>2</sup>/2&sigma;<sub>m</sub><sup>2</sup></sup>
<tr><td>2&pi;&sigma;<sub>m</sub>&sigma;<sub>r</sub>
</table>
<p>
Five parameters will be fitted for each peak:  abundance (A),
retention-time centroid (r<sub>0</sub>), <i>m/z</i> centroid (m<sub>0</sub>),
and the SD of the gaussioid in the retention time (&sigma;<sub>r</sub>)
and <i>m/z</i> (&sigma;<sub>m</sub>) dimensions.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Wyatt MF, Stein BK, Brento AG
 </span>
 <span class="year">
(2006)
 </span>
 <span class="jname">
J. Am. Soc. Mass Spectrom.
 </span>
 <span class="jvol">
17,
 </span>
 <span class="pages">
672-675.
 </span>
<p class="reftitl">
Investigation into Accurate Mass Capability of Matrix-Assisted Laser
Desorption/Ionization Time-of-Flight Mass Spectrometry, with Respect
to Radical Ion Species
<p class="Para1">
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Malmstrom L, Marko-Varga G, Westergren-Thosson G, Laurell T, Malmstrom J
 </span>
 <span class="year">
(2006)
 </span>
 <span class="jname">
BMC Bioinformatics
 </span>
 <span class="jvol">
7,
 </span>
 <span class="pages">
158-182.
 </span>
<p class="reftitl">
2DDB - A Bioinformatics Solution for Analysis of Quantitative Proteomics Data
<p class="Para1">
Typically MS/MS data is searched using SEQUEST or MASCOT with the sequence
data produced.  SwissProt has about 172,000 sequences as of spring 2005
and NCBI has its nonredundant database with 2.3 million unique sequences
and includes GenBank, SwissProt, EMBL and DDBJ.  For each protein it
becomes possible to store its MS spectra and mass and pI.  Data management
becomes an issue.  PEDRo is a data schema for the storage and sharing
of proteome data, and intends to integrate many proteomics technologies;
developed as data information exchange technology.  PRIDE works along the
same lines.  Accession number use becomes a problem since each database
has its own primary key for record access.
<p>
2DDB is implemented using Perl 5.8 as an object-oriented library using
a CGI script to manage the user interface.  It is a relational database
although some data exists outside in the filesystem.  It runs on Gnu/Linux.
Tables in the DB range from experimental meta-data tables with results from
statistical and bioinformatical analysis.  The amino acid sequence is
the core around which the data work.  A sequence will have one or more
accession numbers (keys) associated with it, usually from the common
annotated databases (SwissProt).  The sequence is database independent
and so can be compared if it is the accession number.  The accession
number based on sequence allows grouping of related sequences.
<p>
In typical PMF using MS work, the sequence coverage of identified/matched
peptides is low.  Two sequences differing by only one or more amino acids
can be identified or not disambiguated.  Suppose a composite sequence,
which is denoted by a multi-sequence identifier (MID) is created based
on a seed sequence, and any sequences similiar (95% identical) to the seed
become part of the MID which accounts for all possible peptides.
Clearly the MID will group together polymorphisms and sequence fragments.
The MID definition is based on sequence identity and not on evalues so
it is possible to obtain the data needed to create MIDs in incremental
steps, caching previous results in a lookup table, which is updated weekly
for a manageable computational cost.  The first detected sequence becomes the
seed for the MID, with longer sequences preferred when multiple sequence
belonging to same MID is imported at once.
<p>
Core data model has a few fundamental tables.  The <b>Experiment</b> table
contains several fields, with the key field <b>id</b>, and fields
<b>name, description, aim, conclusion, experiment_type, start_date</b>
and <b>finish_date</b>.  Essentially there is enough information to describe
a sample uniquely as it passes through separation, identification, and
a database search.  A protein in 2DDB is a connection between an experiment
and a protein sequence.  Sequences are stored in another table as they
can be associated with more than one experiment that identified them.
<p>
Additional tables are necessary, as demonstrated.  For example, an
experiment uses an SCX and RP before spotting on to MALDI plate.  Each
spot on the plate is represented as a <i>locus</i> in the <b>locus</b>
table.  Spectra are then acquired and placed in a <b>ms_spectra</b> table,
one or more of which may be associated with the locus.  Peaks, likely
representing peptides, are then selected for fragmentation.  A peptide
sequence is derived and a protein ID is made (by SEQUEST or MASCOT, as
an example).  The mzXML format has been specified and its use increases.
<p>
The <i>locus</i> can also be more than a spot on a MALDI plate.  It can
be a gel spot from a 2DE gel.  2DDB allows for the matching of master gels.
<p>
2DDB also works with the amino acid sequence and all the analyses involving
sequence:  secondary structure prediction, transmembrane domain prediction,
signal sequence prediction.  Functional and structural (secondary and
tertiary) annotations with respect to domains and sequences are
stored in other tables.  The information is both experimental and predicted.
Relations between sequences (sequence homologies/similarities, structural
and functional similarities) are annotated.  Gene Ontology terms are used.
<p>
<b>Protein Explorer</b> is a data analysis tool in 2DDB.  The user defines
a subset of proteins identified from one or more experiments in a
<b>protein list</b>.  A set of proteins is called a <b>project</b>.
The proteins in the list can then be grouped (into <b>GroupSet</b>s) in
a number of ways:  Composite GroupSet containing MIDs, Experiment GroupSet
in which proteins were grouped by experiment, Process GroupSet to indicate
function (splicing, DNA repair, etc.), Component GroupSet (nuclear vs.
non-nuclear), Regulation GroupSet (regulated vs. non-regulated proteins).
Groups and group sets reduce data complexity.
<p>
<b>Data Normalization and Statistics</b>.  2DGE data accumulates and is
full of spot intensities.  It then becomes necessary to normalize between
intensities of different gels for comparison.  Normalization helps to
bring out possibilities of differences in expression.
<p>
<b>Application</b>.  In one study human lung fibroblasts were stimulated
with TGF-&beta;<sub>1</sub>, lyesed, and proteins reduced and labeled with
ICAT, then digested.  Peptides were separated on LC-MS/MS in five
independent experiments.  4731 spectra were obtained and assigned to 1825
unique peptides.  From the 1825 peptides, 1568 proteins were identified,
and all data imported to 2DDB.  Protein count was reduced to 1517 MIDs
by sequence alignment and grouping with 95% sequencing identity explaining
all peptides.
<p>
To obtain 2DDB, see the <a href="http://www.2ddb.org">project homepage</a>
which can also be found at <a href="http://www.sourceforge.net/projects/twoddb">Sourceforge</a>.
It runs on Linux and requires MySQL 4.0 or higher.
<!-- ================================================================ -->
<p class="tags">

<p class="ref">
 <span class="authors">
Blonder J, Yu L-R, Radeva G, Chan KC, Lucas DA, Waybright TJ, Issaq HJ,
Sharom FJ, Veenstra TD
 </span>
 <span class="year">
(2006)
 </span>
 <span class="jname">
J. Proteome Res.
 </span>
 <span class="jvol">
6,
 </span>
 <span class="pages">
349-360.
 </span>
<p class="reftitl">
Combined Chemical and Enzymatic Stable Isotope Labeling for Quantitative
Profiling of Detergent-Insoluble Membrane Proteins Isolated Using Triton
X-100 and Brij-96
<p class="Para1">
Membrane proteins from mixture of high complexity were tagged with ICAT
while proteins and oxygen isotopes as peptides, detergent-resistant
membrane microdomains (DRMMs) were obtained from RBL-2H3 cells were
dually labeled in this way.  DRMMs are obtained from cells lysed in
0.5% w/v Brij96 or 1% w/v TX-100 in a Tris-saline buffer with PMSF,
vanadate and other inhibitors present.  A short spin to remove intact
nuclei and then an ultracentrifugal spin (400,000 <i>g</i>, 4 h)
to get membranes (VTi 65.2) using sucrose discontinuous gradients (40, 30,
5%).  Collected from top in 0.4 ml fractions for 13 total.  Frac 2-7
are pooled (low density) and BCA is done, pelleted at 400,000 <i>g</i>
for 1 h, stored &minus;80&deg;.  100 &micro;g membranes from TX-100 are
labeled with <sup>12</sup>C<sub>9</sub>-ICAT and light water, and the Brij96
isolatd DRMMs with heavy ICAT (<sup>13</sup>C<sub>9</sub>-ICAT and heavy
water).  Membranes brought to solution in 1.5 ml 25 mM ammonium bicarbonate
(pH 7.9) then pelleted 100,000 <i>g</i> 1 h; repeated two more times to
remove detergent.  The pellet is then solubilized in 200 &micro;l
50 mM ammonium bicarbonate containing 60% MeOH, using vortexing and sonication.
TCEP-HCl is added to 1 mM at left 37&deg; 30 min.  This is transferred
to vials with the ICAT in 20% &micro;l MeCN, left 37&deg; for 2 h.
Two exocellulose-packed SEC columns in 60% buffered MeOH are used to
remove salts.  The ICAT-labeled proteins are digested 1:20 w/w trypsin/protein
in same ammonium bicarbonate buffer.  After proteolysis heavy ICAT-labeled
digest was lyophilized and labeled with H<sub>2</sub><sup>18</sup>O using
trypsin-catalyzed exchange in 100 &micro;l 80% 25 mM ammonium bicarb and
20% MeOH, with 1:20 trypsin:protein ratio; stopped by boiling 3 min in water,
cooling, then added PMSF to 1 mM.  Peptides are then analyzed on LC-MS/MS
using QIT MS.	The mass difference between heavy and light doubly charged
pair will be <i>m/z</i> 6.53, labeling of both ICAT and heavy oxygen water.
An example is shown for R.C*SSILLHGK.E and the heavy form (R.C*SSILLHGK^.E)
of an Na/K-ATPase, and the ratio is 1.
<p>
Labeled peptides were then run on nanoflow-RPLCC-LIT-FTICR-MS using
SCX to compare TX-100 and Brij96-extracted proteins.  7105 peptides
matched to 783 unique proteins.  609 proteins were quantified by the
<sup>16</sup>O/<sup>18</sup>O ratio alone and 32 proteins from the dual
labeled fratction, and 97 proteins from both.  Certain proteins are
markers for DRMM (TEC-21, a cell-specific raft marker).  Cys-containing
proteins make up a small percentage, so relative to just the oxygen-labeled,
these should be a smaller proportion.  Only 129 proteins were seen by
ICAT-heavy oxygen alone.  But 32 were seen in the biotin-affinity
elution not found in heavy oxygen alone;  this is likely because the
purity obtained with avidin column eliminates interference by higher
abundance peptides with detection of lower abundance peptides.
A fragment spectrum of KYNFFCQGTRS orgininating from fatty acid elongase
is seen for both TX-100 and Brij96 isolated material, with a heavy-to-light
ratio of 0.31.  This shows a greater ability of TX-100 to isolate the protein.
<p>
25 of 609 proteins identified in the oxygen-labeled fraction have no Cys
and would not be identified by ICAT use.  A subset were identified strictly
from heavy Oxy labeling and the different detergent extractions compared.
DAD-1 (defender against cell death) is a 112-residue membrane protein
needed for cultured cell survival, containing two Cys residues.  With trypsin
digestion both C residues are contained in 49-aa peptide with mass 5355 Da,
usually outside mass detection range.  The protein was quantified instead
from 17 shorter tryptic peptides with the heavy Oxy labeled fraction,
as opposed to the heavy Oxy/ICAT fraction.  The biotin-free heavy Oxy
and biotin-labeled heavy Oxy fractions can be compared.  Seven proteins
were compared in this way, with good correlations in abundance ratios
using either detergent to isolate the proteins.  In contrast conventional
heavy Oxy and ICAT labeling in two separate experiments could have been
done, but more starting material and effort (labor) would be required.
With <em>simultaneous</em> use of heavy Oxy and ICAT labeling in a single
sample there is less time and more efficient quantitative profiling, as
collecting the avidin flow through (unbound) heavy Oxy fraction can be
utilized, while the dually labeled heavy Oxy-ICAT biotin-tagged  allows
a look at a smaller complement, with lower abundance proteins seen in
the same sample with single prep.
<p>
From Table 2 data, RBL-2H3 cells express a high level of Thy-1 (GPI-anchored
protein) which interacts with Lyn and forms detergent-insoluble complexes.
Both Lyn and Thy1 were quantified by multiple peptides, in which they ranked
22nd and 36th in 738 unique proteins ranked by identified peptides per
protein.  Examination of heavy Oxy and heavy Oxy-ICAT fractions in the
MS/MS spectra show that Lyn was enriched more greatly with Brij-96 extraction,
and these were seen in both heavy Oxy, and heavy Oxy-ICAT fractions.
Additionally western blotting with anti-Lyn primary against fractions
obtained from low density sucrose gradient fractions which were treated
separately with TX-100 and Brij-96.  Protein abundance ratio deviating
from 1 were statistically significant, according the nonlinear least squares
regression used to determine differences where ratios were 2 SD above or
below mean.
<p>
77 proteins showed greater abundance with Brij-96 and 533 proteins were
better with TX-100.  Hence more DRMMs (63%) are isolated with TX-100
compared to Brij-96.  Many of the proteins found in TX-100 are not
membrane proteins at all (for example, many ribosomal proteins are
quantitated; some cross-contamination with different subcellular fractions
is unavoidable).  Mean abundance ratio (Brij-96/TX100) for non-DRMM proteins
is 0.29, indicating a great capacity of TX-100 to solubilize nonDRMM proteins
as well.
<!-- ================================================================ -->
<p class="tags">

<p class="ref">
 <span class="authors">
Pasini EM, Kirkegaard M, Mortensen P, Lutz HU, Thomas AW, Mann M
 </span>
 <span class="year">
(2006)
 </span>
 <span class="jname">
Blood
 </span>
 <span class="jvol">
108,
 </span>
 <span class="pages">
791-801.
 </span>
<p class="reftitl">
In-Depth Analysis of the Membrane and Cytosolic Proteome of Red Blood Cells
<p class="Para1">
RBCs as cells were highly purified and controlled for purity (see Mat. Meth).
using automatic counting of WBCs and reticulocytes.  There was 1 WBC
per million RBCs, no granulocytes, monocytes, or reticulocytes.  Improvements
in cell enrichment were 1000-fold.  Digestion of RBC membranes with enzyme
alone resulted in identification of 35 protein.  95 proteins were found
with use of EtOH solubilization/precipitation (band 3 hits went from 6 to
36, but spectrin &alpha; and &beta; peptide fell from 45 to 11 and from 36
to 12, respectively.  Sodium carbonate and EtOH extraction were tried on
different levels (one get the membrane-associated and the other the integral
proteins).  With greater concentrations of carbonate, more loosely associated
proteins (e.g., Rab, annexins A11 and A4) are extracted, while integral
membrane proteins are unchanged.  No difference however between membrane-
and GPI-anchored proteins with carbonate extraction.  A 100 mM carbonate
solution alone will decrease count of identified protein compared to
EtOH/carbonate combination, but saturated carbonate reveals more proteins.
Removal of the cytoskeleton using a special procedure reduces identificaton
of cytosketelon and associate proteins, but also reduces the glycolytic
protein hit numbers.  Further study shows that with exception of GA3PDHase,
the glycolytic enzymes maybe be loosely associated with membrane as the
membrane skeleton to ankyrin.  This binding is sensitive to band 3 Tyr
phosphorylation and oxidation.
<!-- ================================================================ -->
<p class="tags">

<p class="ref">
 <span class="authors">
Nesvizskii AI, Roos FF, Grossman J, Vogelzang M, Eddes JS, Gruissem W,
Baginsky S, Aebersold R
 </span>
 <span class="year">
(2006)
 </span>
 <span class="jname">
Molec. Cell. Proteomics
 </span>
 <span class="jvol">
5,
 </span>
 <span class="pages">
652-670.
 </span>
<p class="reftitl">
Dynamic Spectrum Quality Assessment and Iterative Computational Analysis
of Shotgun Proteomic Data:  Toward More Efficient Identification of
Post-Translational Modifications, Sequence Polymorphisms, and Novel Peptides
<p class="Para1">
The scheme of this work is to produce peptides from a complex mixture and
get LC-MS/MS spectra.  A database search then makes use of SEQUEST, Mascot,
X Tandem and so on.  Results are statistically validated with PeptideProphet.
The probabilities are used to assemble a training dataset of assigned
(<i>p</i> &gt; 0.9) and unassigned (<i>p</i> &lt; 0.1) spectra.
Then for each MS/MS spectrum numerical descriptors of the spectrum are
obtained and statistical classifier developed.  Classifier is applied to
all unassigned spectra to find those of high quality.
<!-- ================================================================ -->
<p class="tags">

<p class="ref">
 <span class="authors">
Titulaer MK, Siccama I, Dekker LJ, van Rijswijk ALCT, Heeren RMA,
Sillevis Smitt PA, Luider TM
 </span>
 <span class="year">
(2006)
 </span>
 <span class="jname">
BMC Bioinform.
 </span>
 <span class="jvol">
7,
 </span>
 <span class="pages">
403 (16 pp).
 </span>
<p class="reftitl">
A Database Application for Pre-Processing, Storage and Comparison of
Mass Spectra Derived From Patients and Controls
<p class="Para1">
Comparison of mass spectra from patient and control groups is not reported.
Bruker Daltonics makes available a raw MS free induction decay for
peptide profiling (<a href="http://cabigcvs.nci.nih.gov/">http://cabigcvs.nci.nih.gov</a>)
in the RProteomics project featuring use of R language.
Software described here will provide storage of mass spectra and analysis
results.  Database holds all metadata.
<p>
CSF samples were obtained with each sample run in duplicate, and each
replicate spotted 3 times on AnchorChip target plate and measured three
times with MS, thus producing 18 replicate spectra for each sample.
Some measurements were <q>zero</q> raw fid files and no data;  if less
replicates are used, then data quality can be affected;  7 or more
spectra should be obtained for robust statistical comparison.
<!-- ================================================================ -->
<p class="tags">

<p class="ref">
 <span class="authors">
Titulaer MK, Siccama I, Dekker LJ, van Rijswijk ALCT, Heeren RMA,
Sillevis Smitt PA, Luider TM
 </span>
 <span class="year">
(2006)
 </span>
 <span class="jname">
BMC Bioinform.
 </span>
 <span class="jvol">
7,
 </span>
 <span class="pages">
403 (16 pp).
 </span>
<p class="reftitl">
A Database Application for Pre-Processing, Storage and Comparison of
Mass Spectra Derived From Patients and Controls
<p class="Para1">
<!-- ================================================================ -->
<p class="tags">

<p class="ref">
 <span class="authors">
Wong RL, Amster IJ
 </span>
 <span class="year">
(2006)
 </span>
 <span class="jname">
J. Am. Soc. Mass Spectrom.
 </span>
 <span class="jvol">
17,
 </span>
 <span class="pages">
1681-1691.
 </span>
<p class="reftitl">
Sub Part-Per-Million Mass Accuracy by Using Stepwise-External Calibration
in Fourier Transform Ion Cyclotron Resonance Mass Spectrometry
<p class="Para1">
FT-ICR MS was developed by Comisarow and Marshall and provides the highest
mass accuracy over a broad <i>m/z</i> range, as well as the highest mass
resolution.  Even peptide elemental composition possible.  Typical accuracy
is in the 1-10 ppm range, although sub-ppm precision is possible too.
External calibration is more difficult because of a space-charge effect
caused by the count of trapped ions, which results in frequency shifting.
Ion counts fluctuation in LC-MS can vary by 2-3 orders of magnitude.
Ultrahigh accuracy is obtainable with internal calibrant use, but ion
counts need to be kept low to avoid data scattering.  However some sensitivity
is lost.  Some have included ion intensity in the calibration equation:

<table class="eqnserif">
<tr><td rowspan="2" style="font-size:200%;padding:0;">(
 <td style="border-bottom:1px solid black;"><i>m</i>
 <td rowspan="2" style="padding:0;"><span
         style="font-size:200%;">)</span><sub><i>i</i></sub> =
 <td style="border-bottom:1px solid black;"><i>A</i>
 <td rowspan="2">+
 <td style="border-bottom:1px solid black;"><i>B</i>
 <td rowspan="2">+ <i>C</i> +
 <td style="border-bottom:1px solid black;"><i>I<sub>i</sub></i>
<tr><td><i>z</i>
 <td><i>f<sub>i</sub></i>
 <td><i>f<sub>i</sub></i><sup>2</sup>
 <td><i>f<sub>i</sub></i><sup>2</sup>
</table>
<p>
Here <i>f<sub>i</sub></i> is the measured cycltron frequency for a calibrant,
<i>I<sub>i</sub></i> is the ion intensity, and <i>A, B,</i> and <i>C</i>
are regression fitting parameters where <i>A</i> accounts for the magnetic
field effect, <i>B</i> accounts for global space-charge effect, and
<i>C</i> is for the local space-charge effect.  The local space-charge
effect is a frequency shift induced by ions of the same <i>m/z</i> wheras
the global effect is everything but the local.  The modified equation
improves internal mass accuracy by 1.5-6.7-fold, depending on mass range
and ion excitation radius.
<!-- ================================================================ -->
<p class="tags">

<p class="ref">
 <span class="authors">
Yanes O, Nazabal A, Wenzel R, Zenobi R, Aviles FX
 </span>
 <span class="year">
(2006)
 </span>
 <span class="jname">
J. Proteome Res.
 </span>
 <span class="jvol">
5,
 </span>
 <span class="pages">
2711-2719.
 </span>
<p class="reftitl">
Detection of Noncovalent Complexes in Biological Samples by Intensity Fading
and High-Mass Detection of MALDI-TOF Mass Spectrometry
<p class="Para1">
Human carboxypeptidase A (CPA) was tested with three inhbitors:  potato
carboxypeptidase inhibitor (PCI), leech carboxypeptidase inhibitor (LCI),
and tick carboxypeptidase inhibitor (TCI).  These were added to extracts
of leech <i>Hirudo medicinalis</i>.  Spectra are shown with CPA and inhbitors
mixed 1:1 in the unrefined mixtures.  The fading of intensities is clearly
demonstrated with PCI at <i>m/z</i> 4298 (600 fmol/&micro;l) LCI at
<i>m/z</i> 7395 Da (900 fmol/&micro;l), and TCI at <i>m/z</i> 7935 Da
(600 fmol/&micro;l).  As CPA is added in increasing amounts, PCI clearly
fades, with the signal disappearing almost completely at 1:1.  No other
protein signals fade at all.  Free CPA itself in the high mass range could
not be detected.

<!-- ================================================================ -->
<p class="tags">

<p class="ref">
 <span class="authors">
Khan A, Packer NH
 </span>
 <span class="year">
(2006)
 </span>
 <span class="jname">
J. Proteome Res.
 </span>
 <span class="jvol">
5,
 </span>
 <span class="pages">
2824-2838.
 </span>
<p class="reftitl">
Simple Urinary Sample Preparation for Proteomic Analysis
<p class="Para1">
Urine samples were collected from healthy male over 6 week period
in the presence of a protease inhibitor (AEBSF, aprotinin, leupeptin,
bestatin, pepstatin A, and E-64 used for mammalian tissues, Sigma, 1 &micro;l/ml)
and stored &minus;80&deg; in 20 ml aliquots.  One 20 ml sample is thawed
and spun 1500 <i>g</i> 10 min RT; the supernatant is collected and
pellet containing broken cells, tissues and other solid sediments discarded.
Precipitation of proteins is done with a 1:5 urine-to-solvent ratio using
acetone, EtOH, MeOH, MeCN, 50:50 acetone:EtOH for the solvents;  these
are left overnight at 4&deg;.  Ultrafiltration then acetone precipitation
is also done, using 3000 MWCO Amicon series 800.  After the overnight
precipitation, centrifuging is 4000 <i>g</i> for 40 min at 10&deg;.
In a serial precipitation step, proteins are pptd with MeCN at pH 6.2.
The supernatant is &minus;20&deg; fro 2.5 h in 100 mM Tris base, increasing
pH to 9.4.  Pellets are resuspended with 5 mM TBP, alkylated with 10 mM
acrylamide, desalted in centrifugal ultrafilter 5000 MWCO.  Desalting
was required because conductivity was too high.  Ultrafiltering also removed
TBP, acrylamide, and Tris.  Precipitated proteins are rehydrated in 200
&micro;l, and Bradford assay is done.  200 &micro;g protein was loaded
on 11 cm IPG strips pH 3-5, 4-7, 6-11 and these focused.  2nd dimenion
is 6-15% gradient gels (all gels and strips from Proteome Systems Ltd).
ProteomIQ Blue was used for staining.  Images were obtained with UMAX
PowerLook III flatbed at 300 dpi.  Images are uploaded into Progenesis
Discovery 2005.
<p>
The urine of the healthy subject contained 450-fold by weight more
pre-precipitation sediment than precipitated protein (in a 20 ml
sample, 90 mg vs. 200 &micro;g, the latter determined by Bradford).
Six different solvents were tried and triplicate 2-D gels were run.
MeCN-pptd spots produced the highest mean number of spots (567 &plusmn;
12 [SD]), followed by EtOH (442) and MeOH (365).  1:1 acetone:ethanol
and ultrafiltration/acetone pptn produced less spots (range 233-365).
Highest resolution was with MeCN-pptd urine also.  Serial/double precipitation
with MeCN first at urine pH then at alkaline pH with analysis of pellets
produced 707 then 133 spots.  The 2nd precipitate did not reveal new
spots (based on spot position).  These data were from the healthy
male subject.
<p>
More protein spots were seen with an ovarian cancer patient urine,
with 1098 spots in a first ppt and 707 in the alkalinized urine ppt.
Some new spots were observed in the 2nd ppt.
<p>
The proteome profile was checked with urine collected during different
times of the day, and also between days (four days over a 6 week period).
Most proteins have a pI 4-7 with fewer in the alkaline.  Greater resolution
is seen in the medium range strips pH 3-5, 4-7, 6-11, than the broad range.
Spot counts of triplicate gels differed significantly:  morning: 798;
afternoon: 703; evening: 729.  More variation was seen with the between-day
proteome in the long term:  the first 3 urine samples were all morning
urine samples while the last 6th week one was an afternoon sample.
<p>
MALDI ToF was done with a Shimadzu Kratos Axima CFR using 10 Hz laser
firing rate.  Internal mass calibration was on tryptic autolytic
peptide (<i>m/z</i> 842.51) and ACTH peptide (<i>m/z</i> 2466.2).
As a control, mass signals were subtracted from signals of gel plugs
taken outside the IPG strip.  Spots were excised with Schimadzu Biotech
Xcise robotic system.  Plugs are washed;  digestion is with 5 &micro;g/ml
trypsin overnight at 30&deg;.  Peptides were desalted and concentrated
with C18 ZipTips, then spotted.  The db search MS Fit in ProteinProspector
and IonIQ PMF (Proteome Systems) is used to search SwissPROT, NCBI, or
Owl with 50 ppm tolerance and one missed cleavage, with modifications
including Cys, oxidized Methionine, methylated Lysine.  In general a minimum
of 4 peptides matching were considered, but if modified and unmodified
matches were present, a minimum of 3 peptides were considered.  If
too many missed cleavages were present (more than 50%), the identification
was ruled out.  Also a match was ruled out if peptide spectral intensities
were less than 50% of highest intensity peptide.  If more than 33% of
the matched peptides were reported as modified, an identifcaiton was ruled
out.  Identification was acceptable for proteins &gt; 60 kDa if at least
10% sequence coverage was found;  between 30-60 kDa, sequence coverage
must be at least 20%.  Generally if matches involved modified and unmodified
peptides, the match was acceptable.
<p>
MALDI MS analyses were run on the pH 3-5, 4-7, and 6-11 gels.  395
proteins from 178 genes were identified with several proteins repeatedly
identified.  339 proteins (45 in pH 3-5, 272 in pH 4-7, 22 in pH 6-11)
from 159 genes (16, 125, 18 in respective pI ranges) were seen in these
gels.  37 proteins (for 29 genes) had more than one spot.  Of the 339
porteins, 63 (19%) are membrane-associated, 181 (53%) are cytoplasmic or
secreted, 11 (3%) are both membrane-associated and cytoplasmic or secreted,
and 83 (25%) are not determined for subcellular origin.  Further classifications
as to renal or plasma origin are made.
<!-- ================================================================ -->
<p class="tags">

<p class="ref">
 <span class="authors">
Grove H, Hollung K, Uhlen AK, Martens H, Faergestad EM
 </span>
 <span class="year">
(2006)
 </span>
 <span class="jname">
J. Proteome Res.
 </span>
 <span class="jvol">
5,
 </span>
 <span class="pages">
3399-3410.
 </span>
<p class="reftitl">
Challenges Related to Analysis of Protein Spot Volume from Two-Dimensional
Gel Electrophoresis As Revealed by Replicate Gels
<p class="Para1">
Twenty gels were run and silver-stained in two batches of ten, and 16 were
analyzed after removing
4 gels due to problems with distortions and staining.  Reference gel for spot
matching is from first batch, with mix of two samples from first extraction.
Spots are <q>fuzzier</q> in the outer regions of gels, more elongated, or missing
compared to those in center.  Saturation of some spots caused underestimation
of the spot volume.  Average spot count for 18 cm pH 4-9/12.5% 2D gel was 720
&plusmn; 120.  Automated matching without manual editing between sample and
reference gives 402 &plusmn; 63 pairs, and manual editing produces 460 &plusmn;
60 pairs.
<p>
770 spots on the reference gel are matching to spots on other 2D gels.  For
702 of the 770, these spots are missing in one or more gels before manual
editing, and after editing, 645 of 795 spots contained one missing spot.
there are 42% missing values total.  Thus 150 protein are matched to a spot
in all gels.  A distribution of the count of protein spots vs the number of
missing value per protein spot shows the maximum count of missing values is
15.  Four spots were selected at random for analysis. Spot 561 was present
in all 16 gels, while spot 25, 485, and 793 were missing in some.
<p>
If missing values are the result of biological differences, they should either
be present or absent in all gels produced of a sample.  None of the highlighted
spots showed complete presence or absence of spots, so biological differences
could not account.  Thus the differences arise from technical errors or
limitations.
<p>
What caused missing values in the gel images?  A partial least square regression
(PLSR) analysis was done with information about the images as predictor variables
(X).  Information included experimental design, total number of spots and
spot intensity, spot volume, and spot area.  The response (Y) is the count
of missing values (MV) on each gel.  A correlation-loading plot is drawn of
the PLSR analysis.  First component accounts for 77% of validated variability,
and the 2nd component accounts for 4%.  Predictor variables at the right-hand
side of the plot that are close the MV component, are those components that would
correlate positively with MV, while those on the left-hand are negatively
correlated: measured total intensity, volume, area and saliency (a measure of
the spot's curvature).  These parameters indicate that staining procedure is
an important source of missing values, since the lower the perceived level of
staining, then the lower the total intenisty, volume and area and the higher
number of missing values.  A high saliency value reflects a more defined spot
and a lower value indicates a spot is more likely to be background or artifact.
In addition mean volume percent of matched protein spots for each gel was
positively correlated to response (significantly).  When gels are faintly
stained some of the faintly stained proteins are lost (undetected), and this tends
to give a higher mean volume of proteins that are detected and matched.  Again
staining intensity parameter is reflected here.
<p>
We plotted the number of missing values against a sensory evaluation of the
amount of staining in 13 gels.  Three gels were excluded because of smears and
streaking that made evaluation difficult.  Even gels with optimal staining
contain missing values (the best gel from the plot has 200 missing values).
Thus staining alone is not responsible for missing values.  Technical variations
that relate to staining and the separation of spots create problems for the image
analyzer.  A very faint spot at the detection limit in four gels was used,
two from each sample and two from each batch.  Spot presence or absence is
highly dependent upon both staining and the system capable of detection.  In
one image, the failure to find a spot was a combination of the staining and of
being able to separate proteins.  The main peak (spot 726) has a shoulder
detected as a separate protein (spot 955) in two gels of the first batch, but
was not detected in either of the samples from the 2nd batch.  Another possibility
occurs that a poor gel alignment accounts for the missing spot.  Global
alignment can get most spots but locally there can be a failure; alternatively
too flexible an alignment could cause erroneous groups of different isoforms or
even different proteins.
<p>
Protein was separated into four groups on the basis of missing values they
contained.  A plot of the frequency of proteins vs their spot volumes (also
log transformed) was performed.  Examination of the distribution shows that
protein variables with many missing values do not have a volume distribution
different from those protein variables with few or no missing values.  This
shows that missing values are not solely found in low abundance values but
also in high volume proteins.  An F-test shows that portein variables with
no missing values had significantly higher protein volumes than those with
missing values.  This can also be seen in the log transformed plot.
<p>
Does the location of missing values in gels.  for each spot the distribution
of missing values was shown in a gel schematic view.  This showed that proteins
without any missing values were mostly centered within the gel, while those with
higher counts of missing values were on the edges, probably due to alignment
problems.
<p>
How can a gel be handled with missing values?  One option is to leave them out,
but this requires knowledge of how they come to be missing.  Some statistical
tests however cannot be done without complete data sets, while others are
weakened if missing values are factored in.  Inserting a missing value as
zero would skew results if it is missing because of technical error.
Use of a background value  can help in situations where software fails in
identifying a line that separates adjacent spots.  Yet another option is to
estimate the missing value, something used for DNA microarray experiments.
And yet another option is to exclude protein variables that have any missing
data, a very harsh treatment that might exclude important analyses.  And another
option is to circumvent the problem by not using spot detection but to analyze
the gel images as spectra after preprocessing and alignment.  This looks at
each pixel.
<p>
<b>Multivariate Data Analysis</b>.  Principal component analysis (PCA) can
require complete data sets or the removal or estimation of missing values with
some implementations.  PCA does not require normally distributed data, but
transformation to normal data can help interpretation (indeed, standard deviatin
of the untransformed variables increased with mean value, but transformation
results in a standard deviation becoming independent of mean value).  Data
transformations of this kind are discussed in Gustaffson et al  and
Meunier et al.  Again interpretations are easier in PCA when data is normally
distributed and SD independent of mean.  PCA was done on data with different levels
of missing values with or without standardization. (gives each variable equal
variance when standardization is used).  Score and loading plots from PCA
are given in the figure; changing the number of missing values has little effect
on analysis.  Score plots for the first two principal components are given.
Loading plots are not changed by reducing the count of missing values in the PCA.
In the first component all analyses show separation between samples in first
component.  If PCA is done without standardization the separation between samples
is more distinct for all gel then PCA done with standardized protein variables.
With standardization only a few abundant proteins guide the models
of the scoring plots; with standardization there was no problem with the model
at any rate.  With standardization, both low-abundant and abundant protein have
identical chance of influencing the first principal components.  While this
may be good if it puts low abundance at a higher level, it also risks increasing
the effect of noisy data.  Without standardization, proteins with the largest
variability dominate the analysis; in fact proteins with highest spot volumes
also show the greatest variation.
<p>
Data for some of the experiments was subjected to a shifted log transformation.
The score plots show the same result for the analysis as the result where
standardized data was used.d
<!-- ================================================================ -->
<p class="tags">

<p class="ref">
 <span class="authors">
Miller I, Crawford J, Gianazza E
 </span>
 <span class="year">
(2006)
 </span>
 <span class="jname">
Proteomics
 </span>
 <span class="jvol">
6,
 </span>
 <span class="pages">
5385-5408.
 </span>
<p class="reftitl">
Protein Stains for Proteomic Applications:  Which, When, Why?
<p class="Para1">
<b>REVIEW</b>.  Coomassie Brilliant Blue was first used in 1963 and is a
sulfonated triphenylmethane.  It binds in acid to protein using electrostatic
and hydrophobic interactions.  The G250 form was first tried in TCA and left
in dilute acetic acid where it forms a colloid in 12.5% TCA which has a low
background in gel but strong signal with protein.  The best preparation is
in the presence of phosphoric acid and ammonium sulfate.
<p>
Fast Green FCF is a dye that is more suited to staining proteins in an IEF
gel since it does not stain ampholytes whereas CBB does (see
<a href="http://dx.doi.org/10.1016/0003-2697(70)90196-X">Gorovsky et al.</a>
(1970) and <a href="http://dx.doi.org/10.1016/0003-2697(80)90106-2">Allen
et al.</a> (1980).
<p>
Silver stains are generally acidic or alkaline.  The latter is produced as
a diamine complex of silver nitrate using ammonia and sodium hydroxide.
Development is then in a dilute acidic solution of formaldehyde.  Acidic
methods use silver nitrate in water or weakly acidic solutions with development
in HCHO at an alkaline pH.  Methods are in four phases: (i) fixation, (ii)
sensitization, (iii)impregnation of silver as its nitrate or ammoniac, and
(iv) image development in dilute carbonate (acidic staining) or citric
acid (alkaline staining).  The ammoniacal silver may be better for basic
proteins, but staining reproducibility depends on ammonia concentration
(storage) and generally works better with a piperzzine diacrylamide crosslinker
and thiosulfate polymerized within the gel.
<p>
The first fluorescent dye to be used was fluorescamine but its results are
variable since it detects free amino groups.  2-Methoxy-2,4-diphenyl-3[2H]furanone
(MDPF)is more sensitive and stable and has also been used for 2D gels
[see Jackson et al. (1988)].  SYPRO Red and Orange were the
first in the noncovalent SYPRO family but are not selective for polypeptides.
These dyes excite at 300 nm and at visible wavelengths, detect as little as
1-10 ng and staining is reversible.  SYPRO Tangerine will work in nonfixative
solutions permitting electroblotting, electroelution and zymography.  SYPRO
Ruby is a ruthenium metal chelate binding to basic amino acids and in contrast
to other dyes, does not intercalate into SDS micelles, binding instead by direct
electrostatic interaction by a mechanism similar to Coomassie colloids.
<p>
Epicocconone is the reactive component in Deep Purple Total Protein Stain and
a nonfluorescent azophilone that becomes fluorescent when interacting with
protein.  It is quite sensitive and linear over four orders of magnitude,
but has a photostability lower than that of SYPRO Ruby and shows an 83%
loss of flourescence after 19 min of UV transillumination compared to 44% for
SYPRO Ruby.  It is thought to work by binding the SDS tail which is bound to
protein.
<p>
Negative stains are those with colorless or transparent protein bands on a
semi-opaque or white gel background.  Immersion in copper chloride or
(to lesser degree) copper sulfate for 5 min produces reverse staining with
sensitivity similar to CBB or silver.  This pattern is reversed and less
sensitive in nondenaturing gels.  Proteins are not fixed&mdash;the copper
is removed with EDTA&mdash;and can be eluted.  Other negative stains have
been tried with cobalt, nickel, zince and other heavy metals, and zinc chloride
has better sensitivity and reaction speed.  A zinc imidazole stain exploits
the ability of proteins to bind Zn<sup>2+</sup> and that of imidazole to react
with the <em>unbound</em> Zn<sup>2+</sup> to produce the deep white zinc
imidazolate, with cleared portions around the protein.  The stain is quite
reversible, and zinc ion also reacts with several modified groups on proteins
as well.  These reverse stained gels can be <q>toned</q> to make more permanent
staining by a redox reaction with ferricyanide/<i>o</i>-toluidine, the white
background now made deep blue and the protein bands still colorless and transparent.
<!-- ================================================================ -->
<p class="tags">

<p class="ref">
 <span class="authors">
Dowsey AW, English J, Pennington K, Cotter D, Stuehler K, Marcus K,
Meyer HE, Dunn MJ, Yang G-Z
 </span>
 <span class="year">
(2006)
 </span>
 <span class="jname">
Proteomics
 </span>
 <span class="jvol">
6,
 </span>
 <span class="pages">
5030-5047.
 </span>
<p class="reftitl">
Examination of 2-DE in the Human Proteome Organisation Brain Proteome
Project Pilot Studies with the New RAIN Gel Matching Technique
<p class="Para1">
The Human Proteome Organization (HUPO) Brain Proteome Project (BPP)
launched two pilot studies to standardize  techniques for phenotyping
patient and mouse models.  Brains of C57/Bl6 mice at three developmental
stages (E16, postnatal day 7, postnatal 54-56) were studied for both
proteome and transcriptome analysis.  5-10 different samples (no pooling)
for each stage.  Human brain from autopsy from epilepsy biopsy to compare
post-mortem protein stability was done.  Samples are are obtained and
distributed centrally to participants with no restriction on protocol
with respect to 2D on materials or methods nor software analysis.
A total 140 2D gels and 266 image were generated.  To revalidate all
these varied images from varied methods, a subset of these gels were
analyzed with a fully automatic gel matching algorith:  Robust Advanced
Image Normalization (RAIN).  It does not require prior spot detection.
The sample image is warped directly to the reference image.
<p>
2-DE remains the preferred core technology because of relative ease of
use and ability to identify changes in PTMs that alter both pI and relative
mass.  There still remains a problem of co-migrating spots even on large
gels and even factors unrelatd to loading of relatively low-abundance proteins
results in streaking or tailing.  The dynamic range of 2-DE is large in
assessing expression although the faintest spots will be hard to distinguish
from noise.
<p>
Image processing follows a pipeline of (i) noise reduction/background
subtraction, (ii) spot detection and modelling and expression quantification,
(iii) spot matchning to remove geometric deformation, (iv) identifying
differential expression (statistical outliers), (v) creating federated
2-DE databases.  With respect to spot matching and detection, a wide variety
of approaches are used: iterative closest point, alignment method, geometric
hashing, and robust pattern matching.  Only robust pattern matching interleave
image warping directly.  <q>With feature matching, a large number of candidate
transforms need to be evaluated due to combinational explosion when each arc
(drawn between two points) in the point pattern is mapped onto every arc
in the target point set. To reduce the search space, arc reduction techniques
such as Delaunay triangulation are regularly used.</q>
<p>
Manual matching is done when complex spot matching methods fail.  These are
pre-pended to compensate for severe geometric distortions.  The TT900 S2S
GUI does smooth landmark-based image warping with a facility to export
warp parameters and images into a proteomics analysis package.  This takes
20-30 min per gel pair.  It is better to focus on use of the raw pixel values
with the features of spot shape, streaks, smears, tails and other background
structures normally removed in spot detection.  The <a href="#smilansky">Z3
algorithm</a> incorporates pixel level information comparing rectangular image
regions with each containing a few spots.  MIR (Multi-resolution Image
Registration) and work by Gustafson et al. use registration techniques without
use of spot detection.  In these, <b>protein diffusion</b> (geometric and
intensity distortion attributed to experimental uncertainty) is eliminated,
while intrinsic differences are retained.  Source image <i>I</i><sub>s</sub>
is aligned to reference image <i>I</i><sub>r</sub> constrained by <i>M</i>
which is the space of allowable transformations (mappings) on <i>I</i><sub>s</sub>
guided by similarity measure function sim, a maximal value when images are
optimally aligned:
<p class="eqnserif">
<span style="font-size:85%;">argmax<sub
style="vertical-align:-1em;margin-left:-3em;">m&isin;M</sub></span>
sim(<i>I</i><sub>s</sub> &ordm; <i>m</i><sub>&theta;</sub>, <i>I</i><sub>r</sub>)
<p>
Note that (<i>f</i> &ordm; <i>g</i>)(<i>x</i>) = <i>f</i>(<i>g</i>(<i>x</i>)).
The maximum argument is found by an optimization procedure on parameters
&Theta; of the mapping function <i>m</i>, which transforms each pixel location
in the sample to a correspoinding location on the reference image space.
<i>M</i> has a tradeoff in finding a transformation with the highest similarity
yet ensures it is realistic.  In MIR a multi-resolution approach is used in
which low-resolution imaes are used to make initial matches with single
bilinear mapping, then finer and finer images are analyzed with subdivision
of the bilinear mapping to become more deformable.  The similarity measure is
normalized cross-correlation.  Between MIR and Z3, MIR took the same time and
scored 29 out of 30 times better with an expert quantifying the number of
misregistered spots.  Woodward et al have recently used wavelets as an alternate
formalism of multi-resolution analysis in which only a sub-band of image
frequencies (range of deformations) are kept at each iteration.
<p>
Although these algorithms have the highest possibility of fully automated gel
matching, it is possible to make further improvements:
<ol>
<li>accurate protein expression quantification is impossible on warped gels
because image-based registration algorithms in current form do not output
a smooth warp or ensure spot volumes remain consistent, and thus quantification
is only applied to unwarped originals.
<li>intensity inhomogeneities between gels are well know and suppressed with
statistical techniques with assumption they follow a Gaussian distribution.
</ol>
<p>
What about the significant variability even between technical replicates
with respect to spot volumes?  Current software does not correct for
expression bias related to pipetting, electrophoresis and staining errors,
but relies on extrapolation of the background variabilty of spot areas.
<p>
Rabilloud and then Gustafson et al. reported a spatial dependence in protein
quantification error over externally Studentized residuals in three representative
gel sets known as bias-field modeling.  Bias-field information cannot be
extracted on individual gels, so analysis is done relative to a second gel.
In normalizing spatial trends during between-treatment comparisons, some
biological conditions might be concealed.  As an example, a higher degree of
proteolysis can decrease spot volumes in full-length proteins enlarging in
degradation fragments.  Still bias-field modeling for image registration is
better posed than bias-field modelling for expression quantification.
<p>
<b>RAIN objectives</b>.  The objective of RAIN is to remove systematic geometric
and intensity distortions in the 2-DE process, with direct analysis of gel images.
RAIN is an image-based method to register simultaneously and bias-field correct
2-D gel pairs with smoot volume-invariant multi-resolution B-spline transformation.
A sum-of-squared-residuals similarity metric and Limited Memory BFGS optimizer
are used.  It includes processing of expression intensities to normalize
out systematic intensity inhomogeneity between gels, in order to ensure accurate
post-quantification.  Expression can then be accurately quantified using
SEA (statistical expression analysis) or conventional techniques.
<p>
B-splines were introduced by Schoeberg (1946) being connected piecewise
polynominal of order <i>n</i>.  A smoothness constraint is imposed on each
connection for spline continuity and its <i>n</i>&minus;1 derivatives, and
thus a <i>k</i> segment spline is defined by <i>k + n</i> parameters.
B-splines were chosen to optimize geometrical transformations for image
registration as each coefficient has a locality of effect of <i>n</i>+1
segments or less at the ends.  The Oslo algorithm allows the subdivision of
the B-spline curve without path alteration to increase the parameters and
thus deformability.  Modelling geometric deformation and bias-fields requires
piecewise surfaces, not segment-wise curves;  B-splines are linearly separable
so this is a simple extension.  Two B-spline surfaces are required to model
smooth deformation in the <i>x</i> and <i>y</i> axes.
<p>
Imaging warping corrects for optical distortion and not physical distortion
in 2DE, introducing protein overexpression in stretching and under-expression
in contraction.  To preserve invariance of the spot volume  the intensity of
each warped pixel is weighted by a change in density.  This is achieved
by considering the Jacobian, a set of partial derivatives of the mapping
with respect to the coordinate axes.  The derivatives correspond to the
change in density along respective axis, and the determinant of the Jacobian
is the total change in volume.  When intensities of the warped image are
normalized by the determinant at each pix, the integrated optical density
of each protein spot is constant even after warping.  For normalization to
be realistically smooth, the B-spline surface must be at least 3rd order,
which means partial derivatives are 2nd order and the change in density is
thus continuous.  With previous techniques 1st order transformations were
done (piecewise bilinear mappings and Delaunay triangulation, and are
discontinuous at piece boundaries leading to spot distortions for spots
lying across boundaries.
<p>
The bias-field correction technique of Lai and Fang is suitable for extracting
spatial bias from a list of spot matches in 2DGE.  But it is feature-based and
requires good manual verification of the spot lists.  Bias-field correction
is integrated in RAIN into the image registration phase by modelling multiplicative
and additive bias-fields by separate 3rd order B-spline surfaces.
A multiplicative bias-field &beta;<sub><i>U</i></sub> represents a smooth
variation in contrast over the image, while the additive bias-field
&beta;<sub><i>V</i></sub> provides the same for image brightness:
<p class="eqnserif">
	<i>I</i><sub>U,V</sub>(<i>x, y</i>) = (<i>I</i>(<i>x, y</i>) +
	&beta;<sub>V</sub>(<i>x, y</i>)) &bull; <i>e</i><sup>&beta;<sub>U</sub>(<i>x, y</i>)</sup>
<p>
Additive bias-field and multiplicative bias field each act in turn on
<!-- ================================================================ -->
<p class="tags">

<p class="ref">
 <span class="authors">
McCulloch CA
 </span>
 <span class="year">
(2006)
 </span>
 <span class="jname">
Periodontol. 2000
 </span>
 <span class="jvol">
40,
 </span>
 <span class="pages">
173-183.
 </span>
<p class="reftitl">
Proteomics for the Periodontium:  Current Strategies and Future Promise
<p class="Para1">
<b>REVIEW</b>.  Periodontal tissues are multicompartmental groups of interacting
cells and matrices providing continuous support, attachment, proprioception
and protection for the teeth.  Currently there is no catalog of the total
expression complement of matrix and cellular proteins in any alveolar bone,
cementum, gingiva and periodontal ligament.
<!-- ================================================================ -->
<p class="tags">

<p class="ref">
 <span class="authors">
Urfer W, Grzegorczyk M, Jung K
 </span>
 <span class="year">
(2006)
 </span>
 <span class="jname">
Pract. Proteomics
 </span>
 <span class="jvol">
1-2
 </span>
 <span class="pages">
48-55.
 </span>
<p class="reftitl">
Statistics for Proteomics: A Review of Tools for Analyzing Experimental Data
<p class="Para1">
<b>REVIEW</b>.  Proteomics experiments belong to two categories.  Either
it is to study protein expression differences in certain sample groups
(distinct disease classes) or it is to study relationship between the proteins
themselves.  In the first category data is acquired usualy protein expression
profiles.  Multiple hypothesis testing is done as part of the statistical
analysis.  Most samples are taken at a fixed time, and so addition of another
variable makes the statistics less complicated.  As for the second category,
it is usually desirable to group proteins with respect to their expression
profile;  this grouping might be by hiearchical ordered clusters or networks.
In the former, proteins with similar expression profile appear in the same
cluster.  With networks there is also information about directed interactions
between proteins.  Many methods applied to microarray can also be applied
to this aspect of proteomics.
<p>
<b>Preprocessing</b>.  Because raw data is usually not normally distributed,
the lognormal distribution may be a necessary transformation.  Usually
expression values in the higher quantity will have bigger variances, but
after the transformation, variances are quite uniform.  Additionally there
is the preprocessing of the calibration of expression values from a replicated
experiment.  This calibration eliminates technical variation so that only
biological variation is assessed.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Blackler AR, Klammer AA, MacCoss MJ, Wu CC
 </span>
 <span class="year">
(2006)
 </span>
 <span class="jname">
Anal. Chem.
 </span>
 <span class="jvol">
78,
 </span>
 <span class="pages">
1337-1344.
 </span>
<p class="reftitl">
Quantitative Comparison of Proteomic Data Quality Between A 2D and 3D Quadrupole 
Ion Trap
<p class="Para1">
The 2D quadrupole ion trap (2D-QIT) is a radiofrequency (rf)-based instrument. 
The multipole-based form is used in hybrid instruments such as FTICR, ToF, 
and TQ systems, but it is also standalone.  The LTQ from ThermoElectron is one
system, based on standard 3D-IT (its LCQ instrument).  The LCQ has ring electrode
with two end caps, but the LTQ has segmented hyperbolic quadrupole with three
distinct axial segments.  The quadrupole analyzer creates a 2D rf field, and then
discrete dc voltages applied to different axial segments to contain the ions
axially in the center segment.  Slits cut into two of the center rods facilitate
radial ion ejection and detection at two electron multiplier detectors.  Many
scan functions and electronics operate in similar manner to the 3D IT in which 
resonance excitation is used to isolate the ion, activate it, and then do mass
analysis.  But unlike the 3D there are performance advantages with the 2D IT:
<ol>
<li>there is no rf field in ion injection axis for ions to overcome before 
entering trap, and so trapping efficiency from external ion source is improved
and this result in faster ion injection time and contributes to rapid duty cycle;
<li>the linear configuration of mass analyzer results in larger volume (space)
to improve total ion capacity before space-charge effects degrade analysis;
moreover radial ejection is a design improvement as opposed to axial ejection 
in the 3D trap.  Radial ejection allows for use of two detectors also, resulting
in doubling of detection efficiency
<li>the LTQ has an approx 3-fold increase in ion ejection rate relative to the
LCQ while having the same resolution:  this should improve qualitative identification
of peptides by automated &micro;LC-MS/MS.
</ol>
<p>
In a perofmrance comparison, E coli strain OP50 is cultured in LB media in log
phase and bacteria pelleted and lysed in French press in a buffer of 50 mM 
ammonium bicarbonate pH 8.5 at 1000 psi.  Lysates are microfuged at 14,000 rpm
for 30 min at 4&deg;.  After assay, proteins are reduced in 5 mM DTT (30 min,
60&deg;) and then alkylated with 15 mM iodoacetamide (30 min, 25&deg; in dark).
Protein is then put in Rapigest and 2 mM CaCl<sub>2</sub> and digested with 
modified trypsin at 1:30 (w/w) overnight in 37&deg; (with shaking).  0.2 M HCl
is added and this is incubated 37&deg; for 45 min.  Sample microfuged for 4 min
and supernatant obtained.
<!-- ================================================================ -->
<p class="tags">

<p class="ref">
 <span class="authors">
Ramachandran P, Boontheung P, Xie Y, Sondej M, Wong DT, Loo JA
 </span>
 <span class="year">
(2006)
 </span>
 <span class="jname">
J. Proteome Res.
 </span>
 <span class="jvol">
5,
 </span>
 <span class="pages">
1493-1503.
 </span>
<p class="reftitl">
Identification of N-Linked Glycoproteins in Human Saliva by Glycoprotein
Capture and Mass Spectrometry
<p class="Para1">
<b>REVIEW</b>.  Multiple Affinity Removal System is a chromatographic system
for removing high abundance human proteins typical in body fluids: albumin,
transferrin, haptoglobulin, &alpha;-antitrypsin, IgA and IgG.  For analyzing
glycoproteins, Zhang et al. oxidized the oligosaccharides and coupled them to
hydrazide.  PNGase F was then used to release the protein part on the column,
and the proteins identified by mass spectrometry.  CA125 and CA15-3 are cancer
biomarkers which have glycosyl modifications, and it is believed that the modification
signifies the potential for metastasis.  Hydrazide binding can be a way to
reduce proteome complexity and enrich for this class of biomarkers.  Using
enrichment methods, Zhang identified 6 proteins whose expression differed
between cancer-containing and normal mice.  Liu and coworkers identified
more than 300 N-linked glycoproteins in plasma.
<p>
Salivary proteome is rich particularly in O- and N-linked glycoproteins
such as mucins (Muc5B, Muc7) and Pro-rich glycoproteins.  Mucins are O-linked
and constituting 70-80% of total protein weight.  Lectin probes have shown
presence of basic Pro-rich to be O-glycosylated.  Salivary amylasse is
N-glycosylated.
<p>
For this study N-linked glycoproteins were studied.  Salivary proteins were
treated with PNGase F and O-Glycanase and run on 1D gels and illuminated
with Pro-Q Emerald, revealing many N-glycosylated with PNGase F-treated alone
and those with both glycanases.  Many bands on the 1D gel still appear in the
PQ-E stain, indicating that deglycosylation was far from complete.  Lower MW
bands were scene with a re-staining of SyproRuby, indicating a large amount
of deglycosylation nonetheless.
<p>
On 2D gels, proteins previously identifed in another report showin PQ-E staining:
polymeric immunoglobulin receptor, &alpha;-amylase, Prolactin-Inducible Protein,
carbonic dehydratase, zinc &alpha;-2 glycoprotein.
<p>
Deglycosylated proteins reveal and acidic shift with reduced MW (faster migration):
The acidic shift is due to formation of Asp from Asn residues as the PNGase F
works in this way.  Some spots in gels were excised, trypsin-digested and peptides
analyzed on LC-MS/MS: identifications were found for transferrin, amylase,
PIP, zA2G, and pIg-R precursor.  This confirms effect of hydrazide-coupled
enrichment.
<p>
Other enrichment methods included EtOH precipitation and isolation of proteins
by pI.  pI-based enrichment improves 2D work allowing sample overloading and MS
analysis (the Zoom IEF-fractionator was used here).  Once proteins were isolated
into zoom disks they were oxidized with periodate in a pH 5.5 coupling buffer,
added to glycerol (reacts any unreacted periodate), dialyzed in coupling buffer,
passed over hydrazide agarose with column washed to remove unbound proteins,
then glycoproteins as hydrazones digested with trypsin.  Free peptides are
not attached to sugars.  PNGase F is then added to columns which removes the
peptides bound as hydrazones.  All peptides from both fractions are run on
LC-MS/MS.  during database searching, the sequence N-X-(S/T) was imposed on
the search as it is N-linked consensus site, with 1 Da added to reflect the
difference between Asn and Asp.  Note that washing of the agarose column prior
to the first trypsin digestion was modified from the original report.
<p>
Desalted peptides are run on the LC-MS/MS.  One peptide identified is
neutrophil gelatinase-associated lipocalin.
<!-- ================================================================ -->
<p class="tags">

<p class="ref">
 <span class="authors">
Song J, Patel M, Rosenzweig CN, Chan-Li Y,Sokoll LJ, Fung ET, Choi-Miura N-H,
Goggins M, Chan DW, Zhang Z
 </span>
 <span class="year">
(2006)
 </span>
 <span class="jname">
Clin. Chem.
 </span>
 <span class="jvol">
52,
 </span>
 <span class="pages">
1045-1053
 </span>
<p class="reftitl">
Quantification of Fragments of Human Serum Inter-&alpha;-Trypsin Inhibitor
Heavy Chain 4 by a Surface-Enhanced Laser Desorption/Ionization-Based Immunoassay
<p class="Para1">
ITIH4 is acute phase 120 kDa plasma glycoprotein made in liver.  The family is
multi-chain, but ITIH4 is single chain because C-terminus lacks DPHFII consensus
for bikunin assembly via glycosaminoglycan bridges.  there is a 120-residue
Pro-rich region with species- and tissue-specific domains.
<p>
Antibodies to three synthetic peptides (H4NT, H4CT, and H4CT-S) of human ITIH4
fragments were produced by attaching to keyhole limpet hemocyanin and
injected to rabbits with a standard 69-day protocol.  The antibodies were
collected with protein-A and affinity chromatography (Pierce Biotech).  In addition
polyclonal in rabbit to whole ITIH4 and a mAb (clone 1A4) were prepared.  The mAb
recognizes an epitope in the 35 kDa C-term fragment.
<!-- ================================================================ -->
<p class="tags">

<p class="ref">
 <span class="authors">
Hamberg A, Kempka M, Sj�dahl J, Roeraade J, Hult K
 </span>
 <span class="year">
(2006)
 </span>
 <span class="jname">
Anal. Biochem.
 </span>
 <span class="jvol">
357,
 </span>
 <span class="pages">
167-172.
 </span>
<p class="reftitl">
C-terminal ladder sequencing of peptides using an alternative
nucleophile in carboxypeptidase Y digests
<p class="Para1">
Edman chemistry can be used to generate a ladder of peptides whose mass
can be determined by mass spectrometry (see Chait et al, 1993).
Chemical degradation similar to Edman N-sequencing has been reported
by others.  For example, alkylation of the C-terminus and its reaction with
isothiocyanate have been reported by Boyd et al. (1992) (see also Samyn
et al, 2000).  Unfortunately, the chemistry requires stops at 3-5 cycles and
consumes nanomoles of sample in the process.  Mass spectrometry produces
alternative ladders by use of CID (with collision gases like argon), ECD,
or ETD with the mass spectrometer.  Zong et al (Nat Biotechnol, 2004) recently
reported ladder sequencing uisng microwave-assisted acdi hydrolysis with MALDI-MS.
Use of enzyme-catalyzed C-terminal sequencing is complicated by various rates of
hydrolysis, with gaps in the sequence.
<p>
CPY has broad substrate specificity with high sequence coverage in ladder sequencing.
However Hayashi et al (1975) did studies with dipeptides hydrolyzed by CPY and found
that the specificity constant <i>k</i><sub>cat</sub>/<i>K</i><sub>m</sub> differed
by 1000-fold for the variety of substrates.  Incomplete ladders are usual.
Amines added to the reaction mixture act as competing nucleophiles.  In the reaction
scheme with peptide WXYZ with Nu as the nucleophile, the nucleophile displaces the
C-terminal residue in a transpeptidation step:
WXYZ + Nu &harr; WXY-Nu + Z.  The next step is the slow hydrolysis
of Nu:  WXY-Nu + H<sub>2</sub>O &harr; WXY + Nu.  This cycle repeats for each
successive C-terminal displacement.  Two sets of products actually buildup:
one are the normal hydrolysis products (step 2 and the enzymatic) and aminolysis
products (step 1).  Eleven aminolytic candidates were tested as nucleophilic agent
candidates were also tested with an ACTH 7-38 peptide digested with CPY.  The largest
number of amino acids were obtained using 2-pyridylmethylamine (2-PMA);  since the amino
group of 2-PMA is closely positioned to the ring nitrogen, this typically reduces the
pKa of the amino group, as the negative charge is delocalized in the pyridyl ring.
Nevertheless the amino group is a good nucleophile at the lower pH, better than water,
and so the <i>n</i> - 1 C-terminal group carbonyl in the peptide bond is acted upon by
the amino group.  Enzyme binding would be difficult, and so peptides created by
aminolysis are resistant to further enzyme action.
<p>
Sample mass spectra are collected.  Aminolyzed peptides will have a different mass
compared to hydrolyzed (the difference between 2-PMA and water is 90 Da).  The mass
shift plus the mass of glycine is equal to the mass of phenylalanine, and this could
cause errors in data interpretation.
<!-- ================================================================ -->
<p class="tags">
FD
<p class="ref">
 <span class="authors">
Mujezinovic N, Raidl G, Hutchins JRA, Peters J-M, Mechtler K, Eisenhaber F
 </span>
 <span class="year">
(2006)
 </span>
 <span class="jname">
Proteomics
 </span>
 <span class="jvol">
6,
 </span>
 <span class="pages">
5117-5131.
 </span>
<p class="reftitl">
Cleaning of raw peptide MS/MS spectra: Improved protein identification
following deconvolution of multiply charged peaks, isotope clusters,
and removal of background noise
<p class="Para1">
With large datasets from LC-MS/MS work, the b- and y-ions are the ions of
interest since they represent fragmentation at the peptide bond.  Many peaks
are repeated shifted signals resulting from natural isotope distribution.
High resolution instruments can distinguish isotope clusters.  ESI helps the
process in that multiply charged ions of large molecules are brought into
measuring range.
<p>
In some cases, MS/MS spectra are so bad that selection criteria are used to
judge a spectrum as interpretable vs uninterpretable.  Clearly no time is
wasted with those judged bad.  Current methods have found a too high false
positive rate on uninterpretability:  that is the spectra were actually
interpretable when they were judged not interpretable.  A problemm is that
with many software packages, such as the MASCOT Distiller, which optimizes
peak location and intensity, the algorithms used are not published.
<p>
Spectra used in this clean-up procedure should be centroided, peak-list
transformed.  Four independent procedures can be performed on these spectra:
(i) detection of multiply charged peaks; (ii) removal of latent periodic
noise (including de-isotoping); (iii) removal of high frequency random noise;
(iv) detection of un-interpretable spectra.
<p>
In evaluating how background peaks can be selected for removal, we can use
the analogy of electric signal processing in which a series of peaks in the
mass spectrum is a signal compounded with noise after transfer through an
information channel, from which the original signal is to be recovered.
<a href="http://mendel.imp.univie.ac.at/mass-spectrometry/ANALYSIS/">Example
synthetic spectra</a> were created to illustrate the technique.  For example
isotope clusters as equidistant spacing of peaks are the source of latent
periodicity in the signal and would thus show up as maxima in the signal
frequency spectrum.  Fourier transformation shows this to be true in example
MS/MS spectra.  The dimishing signal of isotope clusters correlates with
signal dampening in the transform.
<p>
With noisy spectra there is a large number of low intensity peaks.  To
MASCOT-interpreted peaks random noise was added artificially.  Adding a
low-pass filter which removes high frequency part of the signal frequency
spectrum removes the artifically added noise.  These applications would
remove chemical or electronic noise from a spectrum but would not identify
b- and y-ions.  An algorithm which incorporates knowledge of chemical decay
processes would assist in the ion determination process.
<p>
MS/MS spectra are also complicated with ions superimposed upon each other
which have different charge states.  Generally the spectral intensities within
the isotope cluster are a function of the natural isotope abundances of carbon,
nitrogen, hydrogen, oxygen, and sulfur.  Prior to spectrum analysis, the
general form (etalon) of isotope cluster patterns is pre-computed for double
and triple-charged fragments.  While theoretical intensity patterns for larger
ions is more complicated, it is computable.  Rather than maintain a table, the
Wehofsky polynomial approximation in which the relative intensity of the
<i>n</i>th isotope variant peak (in a pattern of <i>N</i> &le; 7 peaks; <i>k</i> =
6) is
<p>
<i>I</i>(<i>n, M</i>) = <i>A</i>(<i>n</i>) + &sum;(<i>j</i> = 1 to <i>k</i>)
<i>B<sub>j</sub></i>(<i>n</i>)<i>M<sup>j</sup></i>
<p>
where <i>M</i> is mass corresponding to first, monoisotopic peak in the
cluster (<i>n</i> = 1).  Its relative intensity is assumed to be 1.
A(n) and B(n) are fitting parameters.  The distance between peaks in the
isotope cluster will be 1/<i>z</i> Da and the pattern length is
(<i>N</i> &minus; 1) / <i>z</i> Da.  The etalon pattner is complemented,
that is, made dense with a total of 20(<i>N</i> &minus; 1) / <i>z</i> &minus;
<i>N</i> + 1 additional peaks with a 50 mDa <i>m/z</i> step and
each intensity linearly interpolated from two surrounding pattern-defining
peaks with masses <i>M</i> + (<i>n</i> &minus; 1) / <i>z</i>
and <i>M</i> + <i>n</i> / <i>z</i>.  Intensity patterns have been tabulated
with accuracy of 100 Da.
<p>
Every peak in the experimental spectrum is assumed to be starting point of
an isotope cluster pattern.  The mass window with the length of target signal
is densified after each such peak with linearly interpolated additional peaks
in 50 mDa steps.  Addition of further peaks (a transformation to a semianalog
signal) compensates for small inaccuracies in resolving position of isotope-variant
peaks by the instrument's software.  A correlation coefficient is calcuated
between observed intensities from pre-computed pattern.  High correlations
(between 0.95 and even 0.99) show reoccurrence of target signal in computed
pattern.  Multiply charged clusters that are detected are converted to a singly
charged monoisotopic peak (deconvoluted).  In rare cases an ion's triply and
doubly charged peaks are found, the one with the higher coefficient is used.
There is no inspection of coefficient values between singly and multiply charged
states of the same ion.
<p>
So long as no very low intensity peaks are within ~0.25 Da of major intensities,
this algorithm works well.  Such peaks arise as random noise or from transforming
continuous signal to centroided discrete signal.  Small interfering peaks between
the main isotope cluster peaks are merged with the closest main peak before
densification, which is a procedure for reversal of small peak creation.
A weight-direcgted graph G(V,E) is constructed for peak merging where V is the
set of vertices which is the set of all m/z values in the window, and a
directed edge <i>e<sub>Ij</sub></i> &isin; <i>E</i> is added between the two
vertices <i>v<sub>i</sub></i>, <i>v<sub>j</sub></i> &isin; <i>V</i> if the
distance <i>d</i> between peaks <i>v<sub>i</sub></i>, <i>v<sub>j</sub></i> is
less than a user-defined accuracy value.  Direction of the edge is defined to
be from <i>v<sub>i</sub></i> to <i>v<sub>j</sub></i> if
intensity(<i>v<sub>i</sub></i>) &lt; intensity (<i>v<sub>j</sub></i>).
The weight wi of an edge eij is actively removed from the graph and its intensity
added to vertex vj, then edges to other vertices can also vanish.  Through
systematic enumeration (e.g. with topological ordering), and edge-free subgraph
is computed without large computational cost which fulfills condition that the
sum of weights of actively removed edges is minimal.
<p>
Because this procedure relies heavily on the resolution and accuracy of measurement
of isotope members and the correlation to an etalon.  There is also the high
likelihood that very dense noise in spectra will be seen as multiply charged ions
deconvoluted to a single charge ion representing the noise.  Some false positive
predictions might arise:
<ol>
<li>aggressive baseline suppression affects relative intensities of peaks in
an isotope cluster and reduces correlation with predefined etalon
<li>interfering peaks overlaying an isotope cluster result in a low correlation
and prevent detection of the multiply charged cluster when those interferences
have a high intensity leading to low coefficient, or the interferences disappear
in the peak merging algorithm (if of low intensity).
<li>Some interpretable ions are close in mass (e.g., y-NH<sub>3</sub> and
y-H<sub>2</sub>O differ by 1 Da) and could create false impression of isotope
cluster if relative intensities are commensurate with the etalon; results of
real applications show this to be rare
</ol>
<p>Measuring correlations between observed and calculated MS/MS is useful really
only for multiply charged peak clusters since the probability of find unrelated
peaks in spectrum which differ by 1 Da is high.  Thus for de-isotoping the
correlation analysis is not useful.  Assume the mass spectrum is a signal
with intensity being amplitude and the <i>m/z</i> being the time domain (true
for ToF-measured masses).  Single-charged ions can be described by a periodical
function with periodicity ~1 Da.  The transform produces a power spectrum in the
frequency domain where re-occurring elements are recognized.  Reoccurrences of
high amplitudes at multiples of a base frequency <i>f</i><sub>B</sub> in the
transform can be seen, and another Fourier transform can be applied to find
<i>f</i><sub>B</sub>.  Suppression of periodically reoccurring intensity maxima
in power spectraum can remove latent periodical noise including minor isotope
variant peaks;  this has been reported for removing chemical noise in MS fingerprints
of small organic and inorganic compounds.
<p>
The MS/MS spectrum (<i>S</i>) is converted to discrete Fourier transform <i>Y</i>
by taking the <i>N</i>-point fast Fourier transform <i>Y</i> &minus;
FFT(<i>S</i>,<i>N</i>).  The value <i>N</i> is <i>N</i> = 2<sup>n</sup> + 1,
where <i>n</i> is the smallest integer &gt;
log<sub>2</sub>[(<i>x</i><sub>max</sub>&minus;<i>x</i><sub>min</sub>)/0.05].
Values <i>x</i><sub>max</sub> and <i>x</i><sub>min</sub> are largest and
smallest <i>m/z</i> values in spectrum.  The power spectrum (a measurement
of power ast various frequencies) is <i>PS</i> = <i>Y</i> &bull; <i>Y</i>*/<i>N</i>.
The power spectrum is quasi-periodic with period length that of fB and determined
by transformation in which power spectrum is signal in a time domain.
Removal of re-occurring elements is done with multiband reject filter, which
is created by Yulewalk method of auto regressive moving average (ARMA) spectral
estimation.  Yulewalk designs recursive infinite impulse response (IIR) digital
filters using least-square fit to specified frequency response.  Frequencies
required by Yulewalk method are calculated by applying median filter to power
spectrum (with 300-500 discrete points and computing second power spectrum to
obtain most prominent frequence of first power spectrum.  Created IIR filter then
filters the MS/MS spectrum in time domain.  Filtered spectrum might have negative
intensity or some new postive intensity signals.
<p>Noisy spectra also have many superfluous peaks densely distributed over the
entire mass range.  Assuming this random noise has a high frequency of occurrence,
a Butterworth IIR low-pass filter is applied to spectrum in time domain.
Normalized stop frequency of filter is in range of 0.5 to 0.9 (best result
obtained with stop frequency of 0.8)  Empirical threshold of 99.99% applied
to remove all signals which have lost intensity above this threshold from
the raw spectrum.
<p>
The power spectrum can also be used to determine which MS/MS spectra are not
worth evaluating.  Hard-to-interpret MS/MS spectra whose two properties:
(i) first power spectrum has very low amplitudes for low frequencies
(ii)  finding the most prominent frequency in 2nd power spectrum can be
ambiguous (several similarly high peaks).  If so, the background removal
cannot be done and the mass spectra must be checked.  As fB is derived from
2nd power spectrum, the position of expected minima and maxima in 1st power
spectrum is computed.  Are the real minima and maxima within a half period
to the expected?  If the shift is more, then de-isotoping will be halted.
Large shifts between real and expected maxima and minima usually indicate
bad spectra.  The periodcity of the spectrum is also tested and a coefficient
of dispersion (<i>C</i><sub>d</sub>) of peak distances in the power spectrum
are calculated as the ratio of the std. dev. of peak distances (<i>s</i>) to the
mean valuve of the peak distances (x-bar):
<p class="eqnserif">
<i>C</i><sub>d</sub> = <i>s</i> / <i>x-bar</i>
<p>
A <i>C</i><sub>d</sub> that approaches or is close to zero indicates
good coincidence of distances between maxima and respectively minima of
consecutive periods with the exxpected distancee (equal to period length).
With quasi-periodic but shifted spectra, it is more complicated.  If
<i>C</i><sub>d</sub> is not larger than 3.3 (empirically derived threshold),
the algorithm predicts the MS/MS spectra cannot be analyzed.  This is generally
the case with the database search software (no protein hits found or hits with
low confidence).
<p>
In most cases, new software should be evaluated against a large panel of sample
spectra which cover the range of possible noise in real spectra, but these are
not available.  MS Cleaner was tested with known proteins.  100 fmol of
BSA, ADH or TRF were used.  Original and cleaned spectra are scored and
compared.  Results of these tests show that the MS Cleaner-processed specta
to remove background had increased scores on protein hits.  Cleaning reduced
the number of peaks considerably, which explains why scores would improve.
About 5-10% of spectra were judged bad and not scored: in only 1 case of 626
was MASCOT able to find the protein from a spectrum judged uninterpretable.
With the Cleaner program for every 10 spectra uninterpretable before Cleaning,
one spectra is returned to interpretable status.  In one example of 100 fmol
of BSA, 373 peaks are seen in the MS/MS spectra, 83 are recongized as background
and removed, leaving 290 peaks.  MASCOT did a better job of finding b and y
ions.
<p>
MASCOT can find multiply charged peaks.  The whole BSA dataset was manually
analyzed. MASCOT interpreted two peaks as doubly charged that had a remnant
isotope cluster.  MS Cleaner found seve doubly charged clusters, of which
two were found by MASCOT.  The other 5 are noise peaks.
<p>
Condensin is a set of two complexes involved in chromosome binding for mitosis
during the compacting of chromosomes.  Two ATPase units of structural maitenance
of chromosomse (SMC0 famly are Smc2 and Smc4, shared between condensin I and II.
A set of other proteins (kleisin and CAP proteins) are also involved.  Antibodies
to Smc2 were used to immunoprecipitate HeLa cell extracts and then SDS-PAGE was
done with Ag staining.  A solution digest for LC-MS/MS was also done.  Bands for
Smc2 and Smc4, and six non-Smc proteins were found.  Original, MS-cleaned,
and bad spectra were submitted to MASCOT MS/MS Ions Search against a small
database and against nonredundant that included all proteins and then all
human proteins.  MASCOT was used to search a database of only 146 sequences.
Each of the 8 condensin subunits had an increase in MASCOT score (8.2%) and
peptide matches increased 4.8% after cleaning.  Only kleisin-&beta; showed
a 2% reduction in sequence coverage after cleaning:  this was likely due to
a single petpide match generated by a single MS/MS spectrum that visually
appears to be of low quality.  MS Cleaner judged is noninterpretable because
it had very few significant peaks above baseline.  This artifact is result of
low frequency suppression in the PSPS-graph and can be avoided by slightly
reducing the threshold <i>f</i><sub>BT</sub> = 12.
<p>
MASCOT Distiller is the one commercial product able to clean backgrounds.
Making scientific explanations about differences in performance are not possible
since the algorithms are not public.  In one test, Distiller increased scoring
in one protein but decreased it in two others.  MS Cleaner did it in half the
time and all scores increased in the three cases.  In the large condensin dataset
Distiller halted on a run-time error.
<!-- ================================================================ -->
<p class="tags">

<p class="ref">
 <span class="authors">
Song J, Patel M, Rosenzweig CN, Chan-Li Y, Sokoll LJ, Fung ET, Choi-Miura N-H,
Goggins M, Chan DW, Zhang Z
 </span>
 <span class="year">
(2006)
 </span>
 <span class="jname">
Clin. Chem.
 </span>
 <span class="jvol">
52,
 </span>
 <span class="pages">
1045-1053.
 </span>
<p class="reftitl">
Quantification of Fragments of Human Serum Inter-&alpha;-Trypsin Inhibitor
Heavy Chain 4 by a Surface-Enhanced Laser Desorption/Ionization-Based Immunoassay
<p class="Para1">
Inter-&alpha;-trypsin inhibitor heavy chain 4 (ITIH4) is a plasma glycoprotein with
Mr 120,000 mostly made by liver in acute phase.  Other members of the ITI family
can assemble as multisubunits, but ITIH4 lacks a DPHFII consensus on the C-terminal
end for bikunin assembly and is a single chain.  It has a proline-rich region from
G6111 to Q730, ~120 residues long with species- and tissue-specific domains.  It
is a target of plasma kallikrein (pK), which may produce bioactive peptides from it.
pK divides it into a 85 kDa N-terminal doman and 35 kDa C-terminal part.  The N-terminal
fragment is then divided into a 57 kDa N-terminal part and 28 kDa C-terminal, which contains
the P-rich region, but the 28 kDa is not detectable and is probably quickly degraded
into smaller peptides.  Others have shown that ITIH4 cleavage fragment is a biomarker
for ovarian cancer:  it is M658-F687 with <i>m/z</i> 3272, and this encompasses the
pK-released, bradykinin-like P662-R688 fragment within the P-rich region.
<p>
For this study sera from 178 patients is obtained:  40 healthy women, 40 with
stage III or IV (23 + 17, resp.) ovarian cancer, groups of 19 patients with stage
0, I, II, or III (3, 5, 8, and 3 cases, resp.) breast cancer, and patients with
stage I, II, or III colon cancer (1, 10, 8 cases resp).  In addition there are cases
with prostate and pancreatic cancer, and diabetes.  Plasma EDTAK2 were also collected
in some cases of testing.  Monoclonals H4NT, H4CT, and H4CT-S were prepared against
human ITIH4 fragements:  H4NT was prepared against synthetic peptide CKIPKPEASFSPR,
H4CT against synthetic CMNFRPGVLSSRQLGPGPPDVPDHAAYHPF (proline-rich), and for H4CT-S
synthetic CLGLPGPPDVPDH which contains a sequence within H4CT in the middle.  Note
that all peptides contain N-terminal Cys and were conjugate to keyhole limpet
hemocyanin, then injected into rabbits using a 69-day protocol (used by SynPep).
Antibodies were purified by protein-A and by affinity chromatography.  In addition
rabbit antiserum was made and mAb 1A4 prepared against the 35 kDa C-termainal fragment.
<p>
Immunoblots were prepared from 1D gels and probed with the various antibodies.
The 120 kDa intact ITIH4 and the 85 kDa N-terminal fragment are recognized by
H4NT, H4CT, and H4CT-S.  The rabbit antiserum detects the 120 kDa protein and
the 85 and 57 kDa fragmentsl.  MAb 1A4 recognizes the 120 and C-terminal 35 kDa
fragments.  Using these antisera, it can be shown the serum of (ovarian) cancer
patients results in a rapid loss of the 120 kDa protein.
<p>
A ProteinChip used in SELDI analysis was used for the anti-synthetic peptide
antibodies in an immunoassay.  With H4CT, detection of the antigenic peptide
(<i>m/z</i> 3273) and a series of larger and smaller peptides with a potential
C-terminal cleavage and <q>sequential</q> N-terminal truncation was seen.
The pattern seen indicates that ITIH4 is cleaved at R688 and splits into an
85 and 35 kDa fragment.  With H4NT, a major peptide at <i>m/z</i> 3157 is found
with no other fragments, indicating relative stability.  This fragment is in
a domain with alternative splicing (missing G621-Q650) which is derived from
 a longer variant (isoform 1) of ITIH4.  Cleavages at N615 and R644 seem to be
targets in the Pro-rich region and these are trypsin and chymotrypsin consensus
sites.  Curiously H4CT-S recognizes fragments different from H4CT, although it
contains it, and this indicates the conformamtional differences occur that
partially block eptitope recognition by H4CT-S.
<p>
Protease inhibitors had little effect on intensity and on pattern of fragmentation.
Longer incubations decreased overall intensity but did not alter pattern.
4&deg; C temperatures had higher intensities than room temp.  Freeze-thaw cycles
counting up to 10 followed by incubation for different times (1, 3 h and overnight)
showed no obvious effects on intensity and pattern of fragmentation.  Neither the
immunblot or SELDI immunoassay could detect cleavage of intact 120 kDa protein in
freshly collected healthy control serum also subjected to these procedures.
Thus sample handling and processing do NOT account for fragmentation and all
such observations are related to biological processes in vivo well before collection
occurs.
<p>
Samples from different disease groups were recorded using the SELDI-based
immunoassay.  Both intensity and mass peaks are examined statistically.  Some
fragments are increased and decreased depending on cancer.  Fragmentation
pattern actually varied with the type of the disease.  One or all of fragments
recognized by H4CT are significantly decreased in ovarian and pancreatic cancer,
while several H4CT-detected fragments are increased in breast and prostate cancer
and in diabetes.  The <i>m/z</i> 3156.6 fragment bound by H4NT is decreased in
pancreatic cancer but significantly increased in prostate cancer.  ROC analysis
was done and significance could be shown for the use of these tests.
<!-- ================================================================ -->
<p class="tags">

<p class="ref">
 <span class="authors">
Xia Y, Chrisman PA, Erickson DE, Liu J, Liang X, Londry FA, Yang MJ, McLuckey SA
 </span>
 <span class="year">
(2006)
 </span>
 <span class="jname">
Anal. Chem.
 </span>
 <span class="jvol">
78,
 </span>
 <span class="pages">
4146-4154.
 </span>
<p class="reftitl">
Implementation of Ion/Ion Reactions in a Quadrupole/Time-of-Flight Tandem Mass
Spectrometer
<p class="Para1">
<b>Accelerated Article</b>.  Deprotonated perfluoro-1-octanol [PFO-H]<sup>&minus;</sup>
is an efficient proton transfer reagent that does not form adducts or
fragments protein/peptide cations.  200 &micro;M PFO solutions form dimer
1- anions as seen in netative mode ESI.  These dimer anions allow for use
of higher Q2 quadrupole RF than for monomer anions, which in turn permits
storage of high <i>m/z</i> ions.
<p>
Proton transfer reactions are done in a Q2 LIT (4 mTorr nitrogen) on proteins
ranging from 5-66 kDa (insulin, ubiquitin, myogloblin, serum albumin).  The
PFO anions reduced proteins to 1+ and 2+ states.  Resolution was 8300 for
[M+H]+ of insulin, although heavier proteins cannot even see isotopic clusters.
As for sensitivity, ability to detect ions is based on magnitude of RF
amplitudes (both radial and axial) during ion/ion reactions, which define the
m/z range over which ions of opposite polarity can be stored simultaneously.
The other is the detector's response to high m/z ions.  With previous 3-quad/LIT
instruments, upper limit of sensitivity is m/z ~33,000; this had an RF amplitude
of 185 V (<i>V</i><sub>r</sub>) during mutual storage and the Mathieu q value
for <i>m/z</i> 33,000 is 0.00748 (RF angular frequency &Omega; 650 kHz, field radius
<i>r</i><sub>0</sub> 4.17 mm).
[more on this paper]
<!-- ================================================================ -->
<p class="tags">
FD
<p class="ref">
 <span class="authors">
Shek PYI, Zhao J, Ke Y, Siu KWM, Hopkinson AC
 </span>
 <span class="year">
(2006)
 </span>
 <span class="jname">
J. Phys. Chem. A
 </span>
 <span class="jvol">
110,
 </span>
 <span class="pages">
8282-8296
 </span>
<p class="reftitl">
Fragmentations of Protonated Arginine, Lysine and Their Methylated Derivatives:
Concomitant Losses of Carbon Monoxide or Carbon Dioxide and an Amine
<p class="Para1">
An MDS SCIEX API 3000 ESI TQ MS/MS is used.  Amino acids and derivatives are
made up to 100 &micro;M in 50% MeOH with 0.1% acetic acid; dimethylated amino
acids at 100 &micro;M in sodium acetate.  Nitrogen is the collision gas.
DFT calculations using hybrid B3LYP method (using Becke's 3-parameter exchange
functional and correlation functional of Lee/Yang/Parr) with 6-31G* basis
set for calculating optimized geometries and energetics.  Critical points
are described by harmonic frequency calculations, intrinsic reaction coordinate
(IRC) calculations used to establish which two minima associated wtih given
transition state.  Zero-point energies and thermal corrections come from DFT
calculations and used to determine relative enthalpies at 298 K.  Entropy
terms then included to obtain free energies.
<p>
CID spectra of [Arg+H]+ was done and found to agree with previous reports.
At low energy NH<sub>3</sub> neutral loss from side chain to produce 158 Th ion.
With higher energy, the &alpha;-NH<sub>2</sub> substitutes guanidine to produce
N-protonated proline at 116 Th.  Computational chem has been used to examine
the proposed mechanisms.  Besides major products, an ion at 130 Th, a mass
loss of 45 Da.  These losses can be [C,H<sub>3</sub>,N,O] or [C,O<sub>2</sub>,H]
type.  Deuterated Arg with 8 D atoms shows the 130 Th ion to shift +5 to
135 Th, and so the neutral loss contains three <i>exchangeable</i> protons.
A precursor ion scan show that the 130 Th product come from a 175 Th ion.
The 158 Th ion does not fragment in a way to produce the 130 Th.  Together
this eliminates the [C,O<sub>2</sub>,H] as the 45 Da mass.  The most plausible
explanation is that the [C,H<sub>3</sub>,N,O] type is respresented by NH<sub>3</sub>
and CO.  <sup>15</sup>N-labeling of &alpha;-amino group shows that the
NH<sub>3</sub> loss to form 130 Th ion is from this function.  The sequence
indicates that protonated Arg first loses CO, then loses the guanidinyl NH<sub>3</sub>,
then the &alpha;-amino NH<sub>3</sub>.  The 130 Th is quite stable in low CID
energies and probably an aldehyde, 1-butanal with guanidinyl function on
&delta;-carbon.  Higher CID energies of the 130 Th iod give products at
71 and 70 Th, and lower abundance ions at 60, 88, 43, and 112 Th.  H/D
exchange studies and best theoretical guesses are that the 71 Th ion is
a cyclic oxonium with no exchangeable protons, and formed by nucleophilic
substitution by carbonyl O atom on &delta;-carbon, guanidine leaving.
Protonated guanidine could be 60 Th ion, with a very high proton affinity.
The 70 Th ion would be the product of two losses:  first of H2O from protonation
of carbonyl O atom of the aldehyde, in which the internal amine -NH- on &delta;
forms 5-membered ring, with loss of carbodimide, and then the ring hydroxyl
is lost as water, with the ion 1-pyrroline left.  Carbodiimide has low
proton affinity and  H/D exchange shows no 46 Th as would be expected in
deuterated precursor if it were the 43 Th ion.  Instead the 43 Th ion is
likely an isopropyl cation.
<p>
<b>N,N-dimethylarginine</b>.  Methylation increases basicity of primary
amino groups and increases likelihood of charge location on this atom
in CID spectra.  A 158 Th product is found in spectra, indicating that
there is a neutral loss of (CH<sub>3</sub>)<sub>2</sub>NH.  Losses of
NH<sub>3</sub> from any function are not seen.  Most abundant ion is
144 Th which is guanidine loss, which becomes N,N-dimethylproline.
<sup>15</sup>N-labeling shows a 145 Th product which is
<sup>15</sup>N,<sup>15</sup>N-dimethylproline.  In the Arg precursor
a 130 Th ion consistent with loss of CO then the &alpha;-amino NH3 was
observed.  With dimethyl-Arg and loss of CO and of dimethylamine, this
130 Th ion is still observed, additionally verifying that the NH<sub>3</sub> or
dimethyl-NH<sub>3</sub> loss is from this function.  Two abundant product
ions are 100 and 114 Th fomre from losses of CO<sub>2</sub> (46 Da) and
HN=C(NH<sub>2</sub>)<sub>2</sub> (103 Da) and CO<sub>2</sub> plus dimethylamine
(89 Da).  Precursor analysis of these products shows only protonated
N,N-dimethylArg.  The methyl ester of N,N-dimethylArg does not show these
neutral losses, which means CO<sub>2</sub> loss requires a mobile proton.
CO<sub>2</sub> loss is <em>not</em> seen with protonated Arg or Lys,
but as in dimethylArg it is seen in &alpha;-methylated Lys.  The indication
is that basicity increases in &alpha;-amino group lower energy for the
formation of zwitterion and this is the semi-transition stabilized for
CO<sub>2</sub> loss.  With the methyl ester, the zwitterionic tautomer
is not formable and the two channels to CO<sub>2</sub> loss are eliminated.
See more regarding theoretical/computational chemistry below.
<p>
<b>Trimethyl-Arginine</b>.  A charge is now fixed on the molecule.  Most
abundant is product ion 114 Th, resulting from loss of trimethylamine
[(CH<sub>3</sub>)<sub>3</sub>N] and CO<sub>2</sub>.  Second most abundant
is 158 Th, which is only the loss of 59 Da trimethylamine.  The loss of
guanidine, also an ion of 158 Th, is ruled out using <sup>15</sup>N-labeling of the
&alpha;-amino group.  Ions at 130 and 116 Th did not contain <sup>15</sup>N,
the former being the 130 Th ion described above, and the 116 Th being protonated
proline formed from trimethylamine and carbodiimide loss.
<p>
<b>Lysine and methylated &epsilon;-nitrogen derivatives</b>.  For these
precursors, dissociations are most from &epsilon;-carbon amino group.
All form the 130 Th ion, which is postulated to be protonated pipecolic acid
formed by nucleophilic substitution between &alpha;-N and &epsilon;-C,
forming the ring.  Also postulated is a 84 Th ion
[C<sub>5</sub>H<sub>10</sub>N]<sup>+</sup>.
with 46 Da loss by combined CO + H<sub>2</sub>O.  Pipecolic is the 6-member
ring compared to 5-membered proline.  Lower abundance ions represent
17 and 45 Da losses, being loss of NH<sub>3</sub> and CO + NH<sub>3</sub>.
NH<sub>3</sub> loss results in 144 Th and 158 Th for the mono- and dimethylated
&epsilon;-amino lysine, but no loss for the trimethylated amino group.
Additionally there is a 116, 130, and 144 Th for all three methylated forms,
indicating a CO loss.  Note that 130 Th form is isobaric with protonated
pipecolic acid:  to prove some of the abundance of the 130 Th from dimethylated
precusor is not just pipecolic acid, <i>d</i><sub>4</sub>-derivative (179 Da)
was used and the two product ions separated.  Methyl esters of the
e-amino methylated forms do not show CO + NH3 loss.
<p>
<b>&alpha;- and &epsilon;-amino methylated derivative of Lys</b>.
Methylation of &alpha;-amino group in Arg promoted its loss as a function;
methylation of &epsilon;-amino in Lys promoted its loss.  So what happens
in competitive dissociation for tetramethyl-Lys (Me<sub>4</sub>Lys,
dimethylated &alpha;N, dimethylated &epsilon;-N), pentamethyl-Lys
(Me<sub>5</sub>Lys, dimethylated &alpha;-N,
trimethylated &epsilon;-N), and hexamethyl-Lys (Me<sub>6</sub>Lys,
trimethylated on both amino groups)?  For Me<sub>4</sub>Lys,
CID spectra were obtained for it, the <sup>15</sup>N<sub>&alpha;</sub>-labeled,
and a <i>d</i><sub>2</sub>-labeled.  Most abundant is a 158 Th ion resulting
from dimethylamine loss.  The <i>15</i>N-labeled molecules shows peaks at 158 and
159 Th, and so there must be a loss of dimethylamine which is <sup>14</sup>N- and
<sup>15</sup>N-labeled.  In the deuterated molecule, there is an ion at 159 Th,
corresponding to ND(CH<sub>3</sub>)<sub>2</sub>.  The abundance ratio was
1:2 for the 158:159 Th ions, indicating twice as much <sup>15</sup>N-labeled
product ion.  Thus the dimethylated &epsilon;-amino group is more than twice
as likely to be lost, probably because of its higher proton affinity.
With protonation it becomes a better leaving group as a result of nucleophilic
substitution from &alpha;-amino group., forming the 158 Th dimehtylpipecolic
acid which then loses 46 Da (H2O and CO) to produce N,N-dimethyltetrahydropyridinyl
ion at 112 Th.  As for the 158 Th ion containing <sup>14</sup>N, its continued
dissociation indicates that it is not the dimethylpipecolic acid:  it produces
a 130 Th ion, probably a loss of 28 Da CO (established both with CID of
158 Th and by precusor scanning for 130 Th ion).  With protonation of
&alpha;-amino, then loss of 15N-dimethylamine, a carbocation forms which
would be destabilized by COOH, making this mechanism unlikely.  More likely
is the attraction of the carbonyl O atom to produce a 3-member lactone
with terminal protonated dimethylamino group with m/z 158.  CO would be eliminated
as a result of steric strain to produce ionic aldehyde at 130 Th.
Deuterated Me<sub>4</sub>Lys indicates that aldehyde ion has one exchangeable
proton, consistent with this mechanism.  Second most abundant is 114 Th
ion, 89 Da less in mass than Me<sub>4</sub>Lys.  Loss of CO<sub>2</sub> and
dimethylamine would produce this, but parent ion discovery for 114 Th shows
that 158 Th ion does not fragment to produce this ion (in other words,
elimination of dimethylamine does not precede CO2 loss).  The dissociation
is seen for Me<sub>5</sub>Lys and more prominent for Me<sub>6</sub>Lys.
With Me<sub>6</sub>Lys, ion at 128 Th is most abundant, with neutral loss
of CO2 and trimethylamine (totals 103 Da).  Relative abundance shows that
the loss of trimethylamine in Me<sub>6</sub>Lys is equally likely for the
&alpha;- and &epsilon; amino groups.  This is also shown in 15N&alpha;-labeled, with
equal abundances of 172 and 173 Th peaks.  Parent ion discovery on 144 Th
shows that only peak 172 Th results in CO elimination, which is the ion
still containing the dimethylated side chain, which means that the side
chain amino displaces the carboxyl group.  The second loss of trimethylamine
then occurs.  Ions at 129 Th represent loss of the side chain amino (<sup>14</sup>N)
while ions at 128 Th would be the &alpha;-amino (<sup>15</sup>N), and
abundance is in a 3:2 ratio, indicating again a greater likelihood of
&alpha;-amino loss.  The scheme is that loss of side chain amino results
from nucleophilic substitution from O atom on COO<sup>&minus;</sup>,
with formation of 6-hexanolactone and a 2-trimethylamino group (&alpha;-N
presence).  In contrast, &alpha;-amino (15N) trimethylamine loss is
promoted by also by the O atom in carboxyl group to form a propanolactone
with 2-trimethylbutylamino function;  this promote CO elimination in the
strained ring.  These lactones would not be prone to lose CO2, so
loss of CO2 must precede any trimethylamine loss.  In such a case,
a zwitterionic (two positive, one negative) hexamethylpentadiamine would
be created.  if there is no charge relocalization, the side chain
amino group would be lost to form a trimethylamino-cyclopentane ion with mass
129 Th (since the side chain nitrogen is <sub>14</sub>N).  If there is
rapid charge relocalization with proton shifting (1,4 proton transfer),
the same trimethylamino cyclopentane ion forms, but with mass 128 Th, since
the leaving group is <sup>15</sup>N-labeled &alpha;-amino.  With the 3:2
ratio, it shows that there is a proton shift, but immediate formation is
slightly more likely.
<p>
<b>Computational estimations</b>.  Energy profiles were calculated for
protonated Arg and the N<sub>&alpha;</sub>,N<sub>&alpha;</sub>-dimethylArg.
A previous report also looked at protonated Arg at B3LYP/6-31G* level.
Reactions are shown for loss of NH<sub>3</sub> and also both CO and
NH<sub>3</sub> together.  The canonical structure of protonated Arg is
its lowest energy isomer, with salt-bridged form being 10.0 kcal/mol higher.
From salt-bridged, an O atom on carboxyl leads to nucleophilic substitution
with C<sub>&alpha;</sub>-N bond lenghtening in preparation of NH3 leaving.
This leads to a lactone with propylguanidinium solvated by NH3.  NH3 migration
to guanidinium gives a lower energy complex.  To lose CO, the O atom making
the nucleophilic substitution to C<sub>&alpha;</sub> continues to bond to
it, coupled with lengthening of the C<sub>&alpha;</sub>-CO bond.
Transition and intermediate states are energetically consistent.
<!-- ================================================================ -->
<p class="tags">

<p class="ref">
 <span class="authors">
Beck HC, Nielsen EC, Matthiesen R, Jensen LH, Sehested M, Finn P, Grauslund M, 
Hansen AM, Jensen ON
 </span>
 <span class="year">
(2006)
 </span>
 <span class="jname">
Molec. Cell. Proteomics
 </span>
 <span class="jvol">
5,
 </span>
 <span class="pages">
1314-1325.
 </span>
<p class="reftitl">
Quantitative Proteomic Analysis of Post-translational Modifications of Human 
Histones
<p class="Para1">
Eukayotic DNA compacted in nucleosomes composed of octamers of core histones:
<ul>
<li>two heterodimeric formations of H2A and H2B
<li>a heterotetrameter of H3 and H4
<li>Joined by histone H1 and linker DNA to form chromatin.
</ul>
<p>
Gene activation involves posttranslational modifications of core histones
affecting their affinity for DNA and ability to form nucleosomes.  N- and C-terminal
ends are modified by methylation, acetylation, phosphorylation, ADP-ribosylation,
ubiquitination, and sumoylation.  The <b>histone code</b> specifies a combinatorial
system whereby covalent modifications occurs.  Lys acetylation seems to be a 
major part of the code:  acetylation is correlated with high transcriptional
activity whereas deacetylation (hypoacetylation) is involved in transcriptional
silencing.  Histone acyltransferases (HATs, histone acetylases) and deacetylases
(HDACs) have been identified.  Hyperacetylation of H3 and H4 occurs with HDAC
inhibition, and genes are expressed that induce growth arrest, cell differentiation,
apoptotic cell death in cultured tumor cells.  Several HDACs have been found to
have pharmacological properties that inhibit tumors in animal models. 
Proteomic study of histones recently showed 20 novel covalent modifications
in calf thymus histones H2A, H2B, H3 and H4, and showing more uniform modification
than understood; modifications would affect DNA binding to nucleosome lateral
surface.  Many histone modifications determined by western blotting with
site-specific antibodies.  MS-based methods have no limitations compared to
western analysis.
<p>
Human small cell lung cancer (SCLC) carcinoma OC-NYH cells are grown in 150 
cm<sup>2</sup> flasks (Nunclon) in RPMI-1640 with 10% FCS + 100 U/ml pen-strep in 
5% CO<sub>2</sub> humidified atmosphere.  HDACi PXD101 at 0, 0.1, 0.3, 1.0, 3.0, 
and 10 &micro;M were added when cell were 10 million per flask, and cells 
incubated additional 24 h before harvest.  Cells are pelleted and wash in cold 
PBS, lysed in 10 mM TrisHCl, pH 6.5, 50 mM sodium bisulfate, 1% TX-100, 10 mM 
MgCl<sub>2</sub>, and 8.6% sucrose at 4&deg; using 20 strokes Dounce with tight 
pestle.  Nuclei were recovered at 2500 &times; <i>g</i> 10 min at 4&deg;. 
Pellet is washed again in 10 mM TrisHCl pH 7.4 + 13 mM EDTA.  Pellet was 
resuspended in 100 &micro;l ice-cold 0.4 M sulfuric acid, left 1 h 4&deg;, then 
centrifuged 2500 <i>g</i> for 10 min.  Supntant is transferred to clean tube and 
1 ml ice-cold acetone added with overnight incubation.  This is centrifuged, the 
pellet air-dried, then resuspended in water, and protein assessed by Bradford. 
5 &micro;g purified protein boiled in SDS sample buffer (+DTT), run on 4-12% 
NuPAGE Bis-Tris gels, transferred to nitrocellulose; antihistone antibodies 
probed 4&deg; overnight, then HRP-conj goat antirabbit 45 min, detection by 
chemiluminescence (Amersham ECL Plus).  Rabbit anti-human pAbs to histone H2B, 
H4, acetyl-Lys5 H2B, acetyl-Lys12 H2B, acetyl-Lys15 H2B, acetyl-Lys20 H2B, 
acetyl-Lys16 H4 all from Upstate Cell Signaling solutions.  Also pAbs from 
Serotec for acetylated Lys-5, -8, -12, -16 for H4.  SDS-PAGE-separated protein 
bands for histones excised, trypsinated in-gel, removing Coomassie G-250 and 
R-250.  <b>q-Real Time PRC</b>: total RNA extracted from OC-NYH cells with 
Trizol using standard protcol; RNA sample quality check by EtBr-stained agarose 
gels.  5 &micro;g treated with TaqMan Reverse Transcription system using random 
hexamers in 50 &micro;l.  First strand synthesis was 25&deg; 10 min, then 
37&deg; 2 h.  TaqMan Gene Expression assays for &beta;-actin, 
p21<sup>C1P1/WAF1</sup>, cyclin A2 labeled with reporter dye FAM on 5' end, and 
TAMRA quencher on 3' end were used.  qPCR was done in 20 &micro;l with 9 
&micro;l 50&times; diluted cDNA, 1 &micro;l probe, 10 &micro;l TaqMan Universal 
PCR Master Mix.  ABi 7500 Real-Time PCR system programmed 2 min 50&deg;, 10 min 
95&deg;, then 40 cycles of 15 s 95&deg;/60 s 60&deg;.  All cDNA samples 
triplicate per experiment, and three experiments done.  The Pierson et al 
DART-PCR method was used to calculate absolute number of cDNA molecules. 
Relative expression of p21 and cyclin A2 genes was found by multiplying values 
by amount of endogenous control beta-action and normalizing to drug untreated 
sample. <b>Mass Spectrometry</b>:  QToF Ultimate or Q-ToF micro MS used.  The LC 
Packings UltiMate with Switchos2 and Famos options were a nano-flow HPLC with 
100 nl/min flow rate using homemade 2 cm capillary (75 &times; 360 &micro;m) 
filled with (Dr. Maisch GmbH) ReproSil-pur AC-C18 3 &micro;m in which peptides 
are loaded and then 30 min gradient 0&rarr;l50% B (A = 0.5 % acetic acid, B = A 
+ 80% MeCN), and this is inline to an 8 cm column with 50 &times; 360 &micro;m. 
Mass resolution was 5000-7000 for QToF micro and 8000-10,000 for QToF Ultima, 
temp source = 80&deg; and desolvation gas 60 L/h.  DDA was set to three most 
abundant ions using a 1 second MS mode scan in range <i>m/z</i> 350-1500 and 2 
second MS/MS mode scan <i>m/z</i> 50-2000, with 30-second dynamic exclusion. 
NaI was used for external mass calibration (error &lt; 50 ppm, but 15 ppm in 
<i>m/z</i> 50-2000); spectral processing using smooth 9/3 SavitzyGolay and 4 
channel/80% centroid top centering.  MS/MS data exported to PKI and a MASCOT 2.0 
and VEMS 3.0 server used for peptide identification.  Database was all known 
human histone variants and query set for tryptic peptides (up to 7 missed 
cleavages); 0.3 Da tolerance for MS, and 0.6 Da for MS/MS ions.  Fixed 
modification was carbamidomethylated Cys, wil variable mods are ox-Met, 
acetyl-Nterm, acetyl-Lys, methyl-Lys and -Arg, dimethyl-Lys and -Arg, 
trimethyl-Lys, P-Ser and -Thr, ubiquitinated-Lys. <b>Quantification</b>:  from 
the data of the MS/MS analysis, peptides with modified Arg or Lys could be 
quantified using VEMS 3.0 software, which is able to identify proteins, assign 
PTMs, align and calibrate LC-MS/MS data insofar as grouping retention times and 
masses.  This is done by looking at isotope cluster intensities from the LC-MS 
data (6 samples in triplicate). For doing normalization only peptides if they 
had CV &lt; 30%, where CV = standard deviation divided by mean of the ion 
intensities.  Four internal standard peptides for each LC-MS/MS run (6 &times; 3 
runs) had their intensities averaged, and then dividing the highest valued mean 
by the mean of the 17 remaining means to obtain the scaling factors for data 
normalization.  Tukey multiple comparison test was done to find significant 
differences in ion intensity between peptides in different groups; used also to 
analyze qRT-PCR results.  Single factor ANOVA used to determine if the 6 
treatment means were not significantly different; pairwise posthoc mean 
comparison in which <i>H</i><sub>0</sub>: &mu;<sub>A</sub> = &mu;<sub>B</sub> 
where A = 1 and B is an integer ranging from 2-6 corresponding to 6 different 
groups.  The Tukey test generates a <i>q</i> value for each of the 6 treatment 
group in each modified peptide identified: intensity from HDACi treatments is 
compared to intensity from control untreated. The <i>q</i> value = [ mean of 
treatment group &minus; mean of control/untreated ] / standard error.  If 
<i>q</i> &gt; 4.75 (<i>q</i><sub>0.05, 12, 6</sub> for &alpha; = 0.05 
significance level for 12 degrees of freedom and 6 groups) then treatment group 
significantly different from control.  <i>Significance analysis of 
microarrays</i> (SAM) was used to identify peptide showing significant abundance 
changes with HDACi treatment.  SAM analysis tests overall significance of 
difference in peak intensity in different run by compensating for multiple 
testing.
<!-- ============================================================= -->
<p class="tags">
guanidination, quantitative proteomics
<p class="ref">
 <span class="authors">
Warwood S, Mohammed S, Cristea IM, Evans C, Whetton AD, Gaskell SJ
 </span>
 <span class="year">
(2006)
 </span>
 <span class="jname">
Rapid Comm. Mass Spectrom.
 </span>
 <span class="jvol">
20,
 </span>
 <span class="pages">
3245-3256.
 </span>
<p class="reftitl">
Guanidination chemistry for qualitative and quantitative proteomics
<p class="Para1">
Many protein samples generated from 1D and 2D gels where the excised spot is 
digested with trypsin and peptide masses analyzed in 800-3000 Da range for 
peptide mass fingerprint.  Tandem MS can be used to get sequence tags for 
helping database searches.  Peptides with C-terminal Arg show superior detection 
compared to peptides with Lys residues.  This is related with the greater stability
of gas phase ions of Arg.  By converting Lys to homoArg using O-methylisourea
a guanidination of Lys occurs with enhanced detection of tryptic peptides.
This paper attempts to show the ability to do relative quantification using the
less expensive [<sup>15</sup>N]O-methylisourea forms.
<p>
[<sup>15</sup>N<sub>2</sub>]<i>O</i>-methylisourea was synthesized from 
[<sup>15</sup>N<sub>2</sub>]urea using a 6-h method involving DMSO heating then 
stirring in acetone and finally diethyl ether;  this produces an ~1 ml of a 0.5 
M solution.  A factor-dependent cell-Paterson mix (FCDP) was extracted and 1 mg 
protein separated on 2DGE.  46 spots were selected for analysis, with varying 
mobilities and intensities.  A MASCOT search was done on MALDI-MS masses 
obtained by in-gel trypsin digest.  Results of the searches were compared using 
digests that had been guanidinated and those which had not.  With unmodified 
peptides 36 proteins had significant MOWSE (molecular weight search) scores, but 
this increased to 42 with the guaninidinated peptides.  In fact MOWSE scores 
were increased for the majority of proteins, with number of peptides matched and 
sequence coverage improved.  Much of the improvement in results was related to 
enhanced detection in which signals representing noise may have been eliminated 
in spectral processing becuase they fall below a 10% relative intensity cut-off. 
It might overlook useful data but the approach was done to facilitate high 
throughput.
<p>
One spot (44) was identified only with a significant MOWSE score after guandination,
when 7 peptides (compared to 3) were matched with 24% sequence coverage (compared
to 12%).  Spot 17 (tropomyosin) was significant in MOWSE before and after
guanidintation, but 13 instead of 9 peptides were matched and sequence coverage
was 41% instead of 29%.
<p>
There are some cases where relevant peaks are qualified in the database search 
without guanidation by which fall below relative intensity cutoff after 
guanidination. This happened for an Arg-containing peptide in one spot.  If the 
database search is allowed to combine the pre- and post-guanidinated list of 
qualified peptides, it increases sequence coverage, peptides matched and MOWSE 
scores.
<p>
By using heavy methylisourea ([<sup>15</sup>N<sub>2</sub>]-O-methylisourea), 
it is possible to get relative quantitation along with better signal response.
The tryptic digestions are done and then the light and heavy methylisourea 
modifications are done, on different samples.  Thereafter the samples are combined,
desalted.  (Desalting is with C<sub>18</sub>ZipTips.  Modified samples are dried
down, dissolved in 10 &micro;l 0.1% TFA.  Tips are pre-wetted with MeOH and then
in 50% MeCN twice, then five in-and-out with 0.1% TFA.  Sample is passed in and
out 5-10 times to ensure binding.  Tips are washed twice with 0.1% TFA.  Peptides
on tips are then eluted with 80% MeCN + 0.1% TFA three times.)   Again the MALDI-MS
shows better detectability of Lys-containing peptides, which now appear as doublets;
not so for the Arg-containing.  One problem is that the mass difference is only
2 Da, which means that the monoisotopic mass of the molecule in one sample is part
of the isotopic cluster.  To determine the relative quantification, it might be
necessary to factor in natural abundances in elemental compositions.
<p>
In an initial assessment of this approach, 1 pmol/&micro;l BSA was modified with
light and heavy methylisourea in ratios mixed from 1:10 to 10:1.  Samples were
analyzed by MALDI-MS in triplicate.  In addition two separate experiments were done.
Seven peptides with C-terminal Lys were evaluated (<i>m/z</i> 965.4, 1016.4,
1056.6, 1205.6, 1347.7, 1637.8, 1681.9) with no missed cleavage although <i>m/z</i>
1637.8 has a mid-chain Arg, and <i>m/z</i> 1681.9 is C-terminal Arg with midchain
Lys.  Data show good agreement between observed and expected heavy and light
isotope ratio in relative intensities, which good CV values except at the extremes
of the mixed ratios.  Proteins from FDCP cell lysates were separated on 2DGE using
250 and 500 &micro;g and stained on Coomassie blue.  Entire spots were selected for
exicision and digested then guanidinated.  The 500 &micro;g sample was treated with
[<sup>15</sup>N<sub>2</sub>]-O-methylisourea and the 250 &micro;g sample treated
with [<sup>14</sup>N<sub>2</sub>]-O-methylisourea.  15 spots were so treated.
Data show that the expected isotope ratio of 2:1  for 500:250 &micro;g was 
observed.
<p>
Proteins were analyzed from rat liver cells in three different modalities:
(i) liver tissue, (ii) hepatocytes immediately after primary explant isoltion,
and (iii) hepatocytes at 24 h and 48 h after culture.  All samples were seaprated
on gels.  Guandination was done to enhance sensitivity.  Relative quantiation
using guandination showed that fructose-1,6-bisphosphatase (identified using MS/MS)
shows a 13.6% reduction in expression between freshly isolated hepatoctyes and
48 h after culture.  The densitometric analysis reveals a spot decrease of 
40% however.  This may be the result of saturation of the spot during staining.
However, accuracy and precision of differential isotope labelling at peptide level
might be due to variability in protein digestion and peptide recovery.  Another 
protein &beta;-actin was analyzed and found to increase in abundance between
liver tissue and 24 h in culture.
<p>
Of interest was the fact that guanidinated C-terminal Lys peptides are not detected
above noise level in ESI MS but are in MALDI MS.  However, based on the MALDI-MS
result, a precurors ion was selected on <i>m/z</i> 998.5 &plusmn; 0.75 to filter
on [M + 2H]<sup>2+</sup> ions containing both light and heavy modifired variants
of the undetectable peptide.  The CID MS/MS spectrum gives a spectrum of modest
quality, and the <i>y</i> ion search shows a sequence of VAPEEHPVLLTEAPLNP*K, indicating
a correct assignment.  In addition, the <i>y</i> ions appeared as doublets, and theya
all had similar ratios of relative intensities, giving high confidence for use of
the MS/MS spectrum for relative quantification.  The data show a 35% increase in
&beta;-actin in the cultured cells compered to freshly isolated, whcih compares
to a 30% value estimated from densitometry of the gels.
<p>

<!-- ================================================================ -->
<p class="tags">

<p class="ref">
 <span class="authors">
Witje D, de Jong AL, Mol MAE, van Baar BLM, Heck AJR
 </span>
 <span class="year">
(2006)
 </span>
 <span class="jname">
J. Proteome Res.
 </span>
 <span class="jvol">
5,
 </span>
 <span class="pages">
2033-2038.
 </span>
<p class="reftitl">
ProteomIQ Blue, a Potent Post-stain for the Visualization and Subsequent Mass 
Spectrometry Based Identification of Fluorescent Stained Proteins on 2D-gels
<p class="Para1">
Fluorescent dyes are used where quantitation is desired with low abundance or
low detection analytes.  However for spot excision a robotic device is required
since fluorescence is not visible during cutting.  Manual excision can only be
done with fluorescent staining by using various strategies, which include
aligning gel with printed scan image, putting gel on UV box (not possible with
the CyDyes), post-staining with silver, running separate preparative gels with
Coomassie.  None of these reproduce quality and sensitivity of fluorescent
staining, and protein id and quant should be done from same gel.  This method
describes a post-staining which is a reformulation of the Neuhoff et al
colloidal Coomassie Blue.  ProteomIQ Blue colloidal Coomassie is sold by
Proteome Systems (Woburn MA).  Bacteria were lysed in 0.2% SDS in 28 mM TrisHCl
and 22 mM Tris with 200 mM DTT, boiled 10 min/100&deg;.  A solutioin of 
200 U/ml DNase, 0.25 mg/ml RNase, 476 mM Tris HCl, 24 mM Tris, and 50 mM 
MgCl<sub>2</sub> and left on ice 10 min.  500 &micro;l sample is mixed with
500 &micro;l MeOH and 125 &micro;l CHCl<sub>3</sub>, mixed 30 min at 1800 rpm.
The tube was spun for 10 min at 20,800 <i>g</i>.  Pellet washed 1 ml MeOH, and
spun again.  The pellet was solubilized in 7 M urea, 2 M thiourea, 1% ASB,
1.2% Destreak, 1% carrier ampholytes.  This is stored at &minus;80&deg; after
protein assay.  NHS ester derivatives of Cy2 and Cy3 dyes are used to label
the proteins:  150 &micro;g protein mixed with 400 pmol CyDye DIGE Fluor 
freshly dissolved in DMF, left in dark 30 min, quenched by addition of 1.0 
&micro;l 10 mM L-Lysine and left on ice 10 min.  Samples are sonicated 
four cycles of 5 s each, iced at intervals, then mixed 35&deg; for 30 min and
then centrifuged 20 min at 20,080 <i>g</i>.  IEF is done in pH 3-10NL or pH 
4-7 in 450 &micro;l rehydration buffer with 150 &micro;g to a 24 cm strip.
Total run is 60-65 kVh with 50 &micro;A current limit, at 20&deg;.  The 
2nd dimension was done on 12.5% SDS-PAGE with strips sealed in 1% agarose
and gels run at 5 W per gel for 15 min, the 17 W per gel until tracking dye
reaches gel bottom.
<p>
<b>Gel Staining</b>.  Gels are fixed ing 10:7% MeOH:MeCOOH for 30 min, then
in Sypro Ruby (1:2 dilution as recommended by Krieg et al) overnight in dark.
Before post-staining with ProteomIQ Blue or silver, gels are incubated in
25:10% MeOH:MeCOOH for 60 min if ProteomIQ Blue, and in 40:10% EtOH:MeCOOH
overnight if silver.  After MeOH:MeCOOH for ProteomIQ, gels are washed 60 min
in water, 60 min in MeOH:MeCOOH, then 1 min in water, and then overnight in 
ProteomIQ Blue.  For plain Coomassie staining, gel are stained overnight in
45/10% MeOH:MeCOOH containing 0.2% CBB G-250 and then destained in 45:10%
MeOH:MeCOOH.
<p>
<b>Imaging</b>.  CyDye- and SYPRO Ruby-stained cells aer immaged with a 
Molecular Imager FX with 0.1 mm resolution and middle sensitivity; the
488 nm laser is used for Cy2 and 532 nm laser for Cy3 and SYPRO Ruby, then
processed with Quantity One v 4.4.0 (BioRad).  Silver, Coomassie and
ProteomIQ Blue are scanned on a ImageScanner and processed with PDQuest v 7.3.1.
Some spots are excised and digested in-gel with trypsin.  For MS in the 
Bruker Biflex III in DE and reflectron mode, calibration of mass scale was 
against trypsin autolysis products [M+H]<sup>+</sup> 2211.1 and a keratin 9
fragment at [M+H]<sup>+</sup> 2705.2 Da.
<p>
<b>Results</b>.  
<!-- ============================================================= -->
<p class="tags" id="shi-2006">
Salmonella
<p class="ref">
 <span class="authors">
Shi L, Adkins JN, Coleman JR, Schepmoes AA, Dohnkova A, Mottaz HM, Norbeck AD, 
Purvine SO, Manes NP, Smallwood HS, Wang H, Forbes J, Gros P, Uzzau S, Rodland 
KD, Heffron F, Smith RD, Squier TC
 </span>
 <span class="year">
(2006)
 </span>
 <span class="jname">
J. Biol. Chem.
 </span>
 <span class="jvol">
281,
 </span>
 <span class="pages">
29131-29140.
 </span>
<p class="reftitl">
Proteomic Analysis of <i>Salmonella enterica</i> Serovar Typhimurium Isolated 
from RAW 264.7 Macrophages:  Identification Of A Novel Protein That Contributes 
To The Replication Of Serovar Typhimurium Inside Macrophages
<p class="Para1">
<i>Salmonella enterica</i> serovar Typhimurium (STM) causes gastroenteritis in 
humans and lethal systemic infection in BALB/c mice lacking host-resistant factor
natural resistance-associated macrophage protein 1 (Nramp1, Slc11a1).  The symptoms
resemble typhoid fever in humans for the mice.  STM invades microfold (M) cells
in Peyer's patches after oral ingestion and is taken up by phagocytes adjacent to
the M cells.  FRom there it moves through lymphatic system and replicates in
macrophages in liver and spleen.  This work tries to identify proteins involved
in colonigzation of macrophage using an accurate mass and time (AMT) tag
approach to identify STM proteins obtained from RAW 264.7 macrophages with or
without functional Nramp1, taken at different time points of infection.  
<p>
<b>Methods</b>.  STM cells are lysed using Mini-Bead Beater-8 (BioSpec)
at maximum speed for 5 min at room temperature.  0.1 mm zirconia/silica beads
are used in 2 ml sterile microcentrifuge tubes.  Extracts are centrifuged 
15,000 <i>g</i> for 5 min.  Protein concentrations are determined.  Urea (7 M),
thiourea (2 M), and DTT (5 mM) are added.  These are incubated at 60&deg; for
30 min.  Samples are diluted in 0.1 M ammonium bicarbonate 10-fold containing
1 mM CaCl<sub>2</sub> and then treated 1:50 (w/w) trypsin:protein for 3 h
at 37&deg;.  Peptides are desalted on Supelco SCX columns then concentrated 
on SpeedVac.  A BCA protein assay is done on the peptides and digested stored
frozen at &minus;80&deg; after flash freezing.  For LC/MS studies samples are 
analyzed on ThermoElectron LTQ ion trap system interfaced to ultraHPLC system
using C18 reversed phase separation.  Peptides masses are obtained in MS mode
and CID MS/MS spectra also obtained.  SEQUEST is used to make identifications.
Washburn et al minimal SEQUEST scores were required for confidence to bring them
into the final database:  in addition a minimal discriminant score of 0.5 
(on 0-1 scale) was required.  Samples were also analyzed on 9.4 T FTICR MS
(Bruker Daltonics) after reversed phase nano HPLC with identical conditions.
Analysese were in triplicate.  Data included retention time, signal abundance
(peak area), monoisoptic mass (charge state + <i>m/z</i> value).  A match between
AMT tag base from LTQ and FTICR data was judged to be 5 ppm of peptide mass and
normalized retention time of 5%.  All proteins qualified if at least three of 
their peptides were identified.  The count of peptides identified for a protein
was divided by the total number of peptides determined from same sample to 
give estimate of relative abundance of each identified protein in the sample.
OmniViz 3.8 was used to generate heat maps.  The <q>relative risk</q> (RR)
of each data set from LC-FTICR is calculated as RR = (RRL &minus; RRT)/RRL
from number of matching AMT tags with a tight error window (RRT) of &plusmn;5 ppm
and &plusmn;3% normalized retetnion time to number matching in a more broad
error window (RRL) of &plusmn;10 ppm and &plusmn;6% normalized retention time.
Overall false postive rate of peptide identification is estimated by averaging
relative risks of all FTICR data sets.
<p>
<b>Results</b>.  Bacteria were isolated from their macrophage hosts in which 
they internalized in order to capture the proteome that enables them to infect 
the hosts.  Two extraction methods were used.  Method 1 is physical and uses 
gentle physical disruption while maintaing organelles that contain the bacteria. 
Method 2 is a fractional solubilization of host membranes while preserving 
detergent resistance of bacteria.  WT (N&minus;) macrophages were collected 2 h 
after infection.  In both methods, cells are washed with HBSS; all steps that 
follow are kept 4&deg; cold.  For Method 1  ml per culture well of 20 mM 
HEPES-KOH 250 mM sucrose, 0.5 mM EGTA pH 7.2 to suspend cells and then passed 30 
times through an Isobiotech ball bearing homogenizer having a clearance of 12 
&micro;m. Homogenates are spun 400 <i>g</i> 5 min.  Supernatants are frozen in 
liq nitrogen, thawed at room temp, diluted with 3 vol homogenization buffer, 
then spun at 12,000 <i>g</i> 1 min. Pellets are resuspended in homogenization 
buffer, loaded on to 12% sucrose cushion, spun 1700 rpm for 45 min.  STM are 
taken from bottom of tubes in 100 &micro;l of 0.1 M ammonium bicarbonate pH 8.4. 
For Method 2, HBSS-washed cells are lysed 30 min in 1 ml/well with 0.1% SDS, 1% 
(v/v) acidic phenol, 19% EtOH. Lysates from wells are pooled and spun 5000 
<i>g</i> 20 min.  Pellets are washed twice with HBSS, then suspended in 0.1 M 
ammonium bicarbonate.  
<p>
Vesicles from Method 1-isolated macrophages were examined using electron 
microscopy.  (Bacteria are diluted in 1 ml 0.1 M cacodylate pH 7.2, washed twice 
in this buffer, fixed in 1% glutaraldehyde in 0.1 M cacodylate at 4&deg; for 2 
h. Cells are washed three times in buffer, then treated with 1% osmium tetroxide 
for 1 h.  Cells are again washed three times in cacodylate, then dehydrated with 
increasing EtOH series, with three 100% EtOH washes for 10 min each.  Cells are 
treated with 1:1 EtOH:LR White resin (EMS), washed three times with LR White 15 
min each.  Cellular material is transferred to gelatin capsules filled with LR 
White and polymerized at 60&deg; overnight.  Hard blocks are cut on 
ultramicrotomes (Ultracut UCT) to 70 nm thirckness, which are mounted on 200 
mesh copper grids with Formvar film sputtered with carbon.  Sections are 
post-stainted with 1% uranyl acetate and lead citrate for contrast. Imaging was 
on JEOL 2010 HR transmission EM at 200 kV.  Images are collected on CCD camera.) 
EM results show large counts of vesicle-like structures from organelles within 
macrophages that were co-isolated with STM cells.  Only 34% of the proteins 
identified came from the bacteria, while the rest were mouse origin.  EM images 
show a cleaner preparation of bacteria and fewer vesicular remnants from 
macrophages.  The MS analysis shows 2 in 3 of all identified proteins are 
bacterial.  74% of proteins found in Method 1 were found in Method 2.
<p>
The (N&minus;) RAW 264.7 macrophages are infected rapidly and are viable for up 
to 24 h.  IN contrast (N+) RAW 264.7 cells make Nramp1 and they can rapidly 
clear bacterial cells from their environment.  Both mouse and bacterial proteins 
from N&minus; and N+ cells were identified using FTICR-MS.  Several time points 
post-infection were analyzed, and the count of bacterial proteins declines with 
time (to be expected). However after 24 h postinfection in N+ macrophages when 
95% of bacteria are killed, 40% of proteins are bacterial.  A total of 16,245 
peptides were identified in all samples from N&minus; and N+ macrophages.  The 
false positive rate was 4% for peptide identification.  66% of identified 
peptides (10,649) were from 315 bacterial proteins and the balance of (5596) 
peptides were from 371 mouse proteins.  The levels of mRNA associated with 296 
of 315 bacterial proteins (94%) have been previously studied in N&minus; 
macrophages and correlated.  About half of bacterial proteins were identified 
with 20 or more peptides per protein; only 16% of macrophage proteins were 
identifed with peptide abundances at that level, indicating better enrichment of 
bacteria.
<p>
Most of the identified proteins whose abundance does not decrease hours after 
infection have housekeeping functions:  biosynthesis and metabolism of amino acids,
carbohydrates, lipids, proteins, nucleotides, energy production, and other cellular
processes.  Nine bacterial proteins are part of pathogenicity islands (SPI-1, -2,
and -3) and they were changed in the time course.  11% of proteins are unknown
or unclassified to function.  27% of mouse proteins could not be assigned functions.
<p>
STM DnaK and mouse &alpha;-tubulin were found in abundance in every sample. 
Commercial antibodies to these proteins permits western blotting.  These confirm the
LC-MS findings for STM DnaK.  Both peptide counts from LC-MS and WB show that 
&alpha;-tubulin shows a dramatic increase at 24 h postinfection.  This is consistent
with finding that vesicles containing <i>Salmonella</i> form microtubules around them
during infection.
<p>
39 proteins (12%) found in LC-MS analysis were found on in samples 2, 4, and/or
24 h postinfection.  Previsou gene epxression profiles foiund in bacteria from
N&minus; macrophages show that 17 of the gens (43%)  are upregulated when bacteria
infect macrophages.  Of the 39, 7 are virulence factor including IHF&alpha;
(STM1339), IHF&beta; (STM0982), MgtB (STM3763), OmpR (ST3502) SitA (STM2861),
SitB (STM2862), SodCI (STM1044).  SerA (STM2062), and IHF&alpha; and IHF&beta;
are seen only in N+ macrophages.  To confirm upreguatlion of IroN (STM 2777), 
PhoN (STM4319), and STM3117 were tagged with FLAG epitope 3-fold:  IroN is
a TonB-dependent siderophore receptor protein; STM3117 has no known function
but might be a virulence factor;  PhoN is a nonspecific acid phosphatase and
is a positive control under regulation of PhoP/PhoQ two-component system.  WB
show that the three tagged protein increase in expression after entering 
macrpojhages.  This increase is seen after 2 h postinfection in STM3117 and PhoN,
peaking at 4 h.  The increase in IroN is detectable only at 4 h.  The LC-MS
and WB results agree as to quantitation.
<p>
As IroN, Mn<sup>2+</sup> transporter components SitA/B, and Mg<sup>2+</sup>
transporter protein MgtB are all upregulated, the bacteria are sensing divalent
metal ion limiting conditions in the vacuole.  PhoN and MgtB are regulated by
PhoP/Q which is known to be activated after infection of macrophages.  Increases
in OmpR also indicate that T3S of SPI-2 is upregulated.  One of functions of 
OmpR is to regulate SsrA/B, which in turn regulates T3S of SPI-2 expression.
No other SPI-2 proteins were found (consistent with other expression profiles).
T3S is translocated into the cytoplasm of host cells.
<p>
Nramp1 is a divalent metal transporter made by infected host cells.  Nramp1 is
recruited into the vacuole with bacteria present within.  Nramp1 action will
induce a response.  It should be noted that IHF&alpha; and IHF&beta; and SerA
were observed only when bacteria are taken from N+ macrophages.  SodCI, 
Upp (STM2498) and STM3117 are found in bacterial expression only from N&minus;
macrophages.  IHF&alpha;, IHF&beta;, SodC are involved in bacterial virulence.
IHF&alpha; and IHF&beta; associate as a heterodimer called integration host
factor, and is a DNA binding protein, regulating bacterial expression of genes
in SPI-1 through SPI-5.  SodCI is one of two periplasmic superoxide dismutase
(Cu,Zn-dependent) limiting the oxidatiave burst of NADPH oxidase in the vacuole
in an attempt to neutralize antibacterial ROS formed in the vacuole.  If <i>sodCI</i>
is inactivated in bacteria, they fail to proliferate in mouse spleen.  SodCI
is absent in bacteria isolated from N+ macrophages and thuse Nramp1 may affect
the ability of SodCI to associate with Cu/Zn cofactor.  Upp (uracily phosphoribosyltransferase)
regulates entry into pyrimidie salvage pathway commonly induced by starvation 
conditions, while SerA (D-3-phosphoglycerate dehydrogenase) regulates cellular
levels of 3-phosphoglycerate from serine; neither has clear role in pathogenesis.
<p>
Potein STM3117 is seen in LC-MS only from N&minus; macrophages and is in greater
abundance than in N+ macrophages.  It is tagged with 3 &times; FLAG epitope and
WB confirms quantiative findings of LC-MS relative to N+ macrophages.
The same is true for STM3118 and STM3119, indicating these three proteins are part
of an operon.  Using a &lambda;-phage recombinase system, STM3117 gene was deleted
during expression.  Bacteria are able to invade macrophages, with similar numbers
of bacteria seen in both N&minus; and N+ macrophages.  However, bacteria cannot
replicate inside N&minus; macrophages, and thus may be necessary for colonization.
STM3117 also gives no advantage to bacteria in resisting killing and clearing
in N+ macrophages.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Chevallet M, Luche S, Rabilloud T
 </span>
 <span class="year">
(2006)
 </span>
 <span class="jname">
Nat. Protoc.
 </span>
 <span class="jvol">
1,
 </span>
 <span class="pages">
1852-1858.
 </span>
<p class="reftitl">
Silver staining of proteins in polyacrylamide gels
<p class="Para1">
Proteins can bind to silver but in doing so they reduce the reactivity of
silver to reduction chemistry.  This explains why there is a hollow or doughnut
appearance in the spots/bands of proteins.  All staining protocols do the 
following: fixation which eliminates interfering chemicals; sensitization, 
followed by rinsing to improve sensitivitiy and contrast; impregnating
silver either with its nitrate or its ammonia complex form;  rinsing and
development; the stopping of development before excessive background occurs.
<p>
Silver nitrate methods are simplest, have moderate compatibility with MS.
They come in rapid and slower modes (expense of quality in rapid mode).
Background staining is seen when room temperatures are too high (&gt; 30&deg;).
Basic proteins are less efficiently stained than acidic ones.  Silver nitrate
methods are best choice when multiple gels must be stained and consistency
is required.
<p>
Silver ammonia complex do not work well with cold solutions (should be
20-25&deg; or above).  Basic proteins are stained better than for acidic
proteins.  Only gels made in Tris-glycine or taurine buffer systems can be
stained (bicine/tricine systems have backgrounds).  Polymerization must be with
70 &micro;l TEMED, 500 &micro;l 10% hypo (sodium thiosulfate pentahydrate), and
700 &micro;l ammonium thiosulfate per 100 ml gel mix, all added sequentially.
<p>
Some important considerations are 1D vs 2D gels.  The latter contain carrier
ampholytes which require removal with vigorous fixation.  Silver-ammonia
complex staining requires the use of fresh, in-house cast gels, and does
not work well with commercial gels.  Also fixes with aldehydes (formaldehyde,
glutaraldehyde) do well to improve fixation and sensitivity and uniform staining,
but are not MS-compatible.  Fixation processes that use a long time are good
at improving the image, but they lead to peptide loss and recovery for MS.
The use of fast vs slow protocols should favor slow protocols when gel 
replicates are made and consistency is desired;  faster protocols are okay
when the gel will be processed for MS and when image replication is not critical.
Room temperature consistency is vital too:  generally room temp should stay 
between 20-30&deg; for the best results.
<p>
The following selection procedure should work well:
<ul>
<li>Use Protocol A for fast and sensitive visualization where only a few gels
per series, with no special needs afterwards
<li>Protocol B is a slow silver nitrate method ideal for large series of gels
over extendded time period, maximal consistency and good linearity
<li>A variation of Protocol A is done when sensitive detection but no quantiation
giving maximal sequence coverage in MS work:  this is the ultrafast staining
to help prevent decreases in peptide recovery
<li>Protocol C provides sensitivity, linearity, and is MS compatible: 
this is aldehyde-free silver ammonia
<li>Protocol D gives excellent sensitivity, without requirements for quantitation
or MS-compatibility.  This is standard formaldehyde-based silver ammonia 
staining
</ul>
<p>
In general a batch of 3-5 gels of 20 cm &times; 16 cm &times; 1.5 mm can be
incubated at once in a 1 L solution of each step.  This is a solution:gel volume
ratio of &ge; 5.  500 ml solution is used for 1 or 2 gels of this size.  At 
least a 5:1 solution:gel ratio should be used.   The gels must also float
freely.  In general a batch processing (multiple gels in a container) can be
used for every step where the time period &ge; 5 min.
<p>
A rigid plastic sheet should be used for changing solutions, which can be used
to press gels down and hold them.  When adding solution it is advisable to 
keep the gels pressed so that the pouring of the new solution does not cause
gel breakage.
<p><b>Reagents</b>.
<ul>
<li><b>Ethanol</b>.  95% or absolute EtOH can be used without volume correction,
but denatured alcohol cannot be used.
<li><b>Water</b>.  Resistivity must be &gt; 15 M&Omega;-cm.
<li><b>Silver nitrate</b>.  A stock solution is usually 20% (w/v) and stored
in dar place (refrigeration is both dark and cold).  Alternatively a titrated
1 M solution (e.g., Riedel de Haen) can be used.
<li><b>Ammonium hydroxide</b>.  A titrated (5 M) concentrated stock is recommended.
When stored in fridge, stable for months.
<li><b>Sodium thiosulfate</b>.  A good pro-analysis grade pentahydrate can be
used.  A 10% stock solution can be made and kept at room temp for up to a week.
<li><b>Napthalene disulfonate</b>.  Existing as several isomers (acid or sodium
salt), the 2,7-isomer (Acros) is recommended but the 1,5-form (Aldrich) is okay
too.
<li><b>Formaldehyde</b>. The 37-40% solution is used.  It should not be stored 
low temperatures (cold environments cause polymerization).
</ul>
<p>
With respect to equipment, a rocker and not orbital shaker should be used.  It
should be capable of rocking 30-60 strokes per minute.  Containers can be 
glass or plastic, although plastic containers should be rinsed in acetone then
alcohol to remove plasticizers.  The area of the container should be 20% larger
than the area of the gel.
<p><b>Working Solutions</b>.
<ul>
<li><b>Tetrathionate Sensitizer</b> (1 L). 3 g potassium tetrathionate + 50 g 
potassium acetate in 500 ml water.  Bring the water to 700 ml, then add 300
ml EtOH.  This solution must prepared freshly (day of use).
<li><b>Silver-Ammonia</b>.  475 ml of water is placed in flask with strong
magnetic stirring.  7 ml of 1 N NaOH is added, then 7.5 ml of 5 N ammonium
hydroxide, followed by 12 ml of 1 N silver nitrate.  A brown precipitate appears
during silver nitrate addition, but it should disappear in a few seconds.  If
the brown precipitate persists, it indicates that ammonium hydroxide is 
exhausted (adding more NH<sub>4</sub>OH is useless as sensitivity will drop).
The ammonia-silver ratio is critical for good sensitivity; it is typically 3:1.
These solutions must not be allowed to dry out, or an explosive azide can form.
Unused ammoniated silver must have NaCl or ascorbic acid to precipitate silver
and deactivate it.
<li><b>Developer for AgNO<sub>3</sub></b>.  Prepare 3% potassium carbonate.  For 
each liter prepared, add 250 &micro;l 37-40% formaldehyde and 125 &micro;l 10% 
sodium thiosulfate pentahydrate.  Formaldehyde and thiosulfate must be added 
within 1 h of use.
<li><b>Developer for Ammoniated Silver</b>.  0.35 mM citric acid with 
1 ml formalin per liter.  Development of both image and background is more
rapid than with silver nitrate.  Thiosulfate helps to reduce the background but
cannot be used in developer as for silver nitrate, and so it must be preincubated
in the gel itself.  The developer must be prepared day of use, and formaldehyde
not added until 1 h before use.
<li><b>Stop solution for nitrate</b>.  4% Tris + 2% acetic acid.  Must be
fresh on day of use.
<li><b>Stop solution for silver nitrate (alternative)</b>.  0.5% (v/v)
ethanolamine and 2% acetic acid.  Must be made on day of use.  Add ethanolamine
to water, then add the acetic acid.  Do not mix concentrated chemicals together.
<li><b>Silver destaining</b>.  Prepare 30 mM potassium ferricyanide and also
100 mM sodium thiosulfate solution (both must be prepared on the day of use).
Mix equal volumes and use within 30 min.
</ul>
<p><b>Imaging and Mass Spectrometry</b>.  In all cases, imaging should be done
within a few hours (or at least same day).  If mass spectrometry is to be done,
imaging analysis and spot excision should be complete within 24-48 hours.
Keep the gel in 1% acetic acid until spot cutting is complete.  For even longer
term storage of the gel, soak the gel in 20% glycerol and remove all the water.
Keep the gel in an airtight container at 4&deg;.
<p><b>Protocol A</b>.
<ol>
 <li><b>Fixation</b>.  All gels are fixed in 30% EtOH + 10% MeCOOH.  The initial
fix must be 30 min long.  Fixation can occur for 24 h, with at least one solution
change.  In an ultrafast fixation of only 30 min, recovery for MS is better, but
there might be strong chromatic effects (yellow, orange, grey, brown spots), which
makes image analysis difficult.  Longer fixation (three changes over 30 min each)
gives better staining, but lower sequence coverage.  To remove carrier ampholytes,
an 18 hour fixation is necessary.
 <li><b>EtOH-only &amp; water rinsing</b>.  Gels are rinsed in 20% EtOH for 10 min 
each.  Then two times for 10 min in water.
 <li><b>Sensitization</b>.  Gel is now soaked for one minute only in 0.8 mM
sodium thiosulfate; this is 0.02% or 20 mg/100 ml of the pentahydrate salt.
This solution can be recovered if it will be used for more gels during that day.
 <li><b>Rinse</b>.  Rinse in double the usual volume of water two times, each 
for 1 minute.
 <li><b>Silvering</b>.  Place in 12 mM (0.2%) silver nitrate.  While gels might
appear yellow, this does not affect quality.  Time of incubation can be from
20 min to 2 h.
 <li><b>Development</b>.  Prepare 40 g Tris and 20 ml acetic acid per liter as
the stop solution.  Make sure the stop and plenty of water is ready.  Recover
the silver nitrate carefully.  Pour in a small amount of water (do not cover
the gel):  it is only to rinse out some silver nitrate solution.  Pour out the
water.  Pour in 10 gel volumes of water (do not be slow, but do not break the 
gel).  Rock 10 seconds by hand.  Pour it out.  Pour in a small amount of 
developer (about 1.5 depths of the gel).  Rock for 10-15 seconds.  If cloudiness
develops (it might be yellow or brown), pour it out quickly.  Quickly pour in
3 gel depths of developer.  You <em>MUST</em> be rapid in pouring out and pouring
in!  Do <em>NOT</em> allow the gel surface to remain exposed for any length
of time, or else surface deposition and background may occur.  Allow development
to proceed but not so the background accumulates.  If after 45 minutes nothing
appears, then nothing is detectable.  Discard developer (cannot be re-used).
<li><b>Stopping</b>.  Pour in stop solution.  Leave at least 30 min.  Can be
left up to 2 h (but not necessary).  Wash gels 30 min each in water.
</ol>
<p><b>Protocol B</b>.
<ol>
<li>Fix gels in 30% EtOH + 10% acetic acid for 60 min.  Change fix and leave
overnight.
<li>Sensitize in tetrathionate solution described above for 45 min.
<li>Rinse two times for 10 min each in 20% EtOH
<li>Rinse four times for 10 min each in water
<li>Incubate in 0.2% silver nitrate as described for Protocol A.
<li>Develop the spots/bands as described in the <b>Development</b> step in Protocol
A.
<li>Stop the development as described in Protocol A.  Image and process the gel
for MS as in Protocol A.
</ol>
<p><b>Protocol C</b>.
<ol>
<li>Fix gels in 30% EtOH + 10% MeCOOH + 0.05% (50 mg/100 ml) naphthalene disulfonic
acid.  Three times for 30 minutes each (rapid method) or 30 min + overnight 
two-time incubation will work.  To make sure low MW region is cleaned of ampholytes,
fixation must be 18 hours long (MS peptide recovery is at risk).
<li>Rinse gel in water six times for 10 min each.
<li>Incubate 30-60 min in silver ammonia complex solution
<li>Rinse three times in water, 5 min each
<li>Use the acidic developer solution above to develop the image.  It takes 5-10 
min.  The most intense bands appear inside 1-2 min.
<li>Stop development in the ethanolamine stop solution.  Keep in the solution for
30-60 min.
<li>Rinse in several changes of water before drying or doing densitometry.
</ol>
<p><b>Protocol D</b>.
<ol>
<li>Place the gel in water, and let is rinse for 5-10 min.
<li>Let gel incubate 1 h in 20% EtOH containing 10% (v/v) formalin
<li>Rinse twice in water, 15 min each
<li>Sensitize in 0.05% naphthalene disulfonate overnight.
<li>Rinse six times in water, 20 min each
<li>Incubate 30-60 min in silver-ammonia.
<li>Rince three times in water, 5 min each.
<li>Develop in acid developer.  It will take 5-10 minutes, and the most intense
spots/bands appear inside 1-2 min.
<li>Stop in ethanolamine stop solution, leaving it incubate for 30-60 min
<li>Rinse water several times before imaging.
</ol>
<p>For destaining spots or bands, the solution described above is used to cover the
cut-out gel and should be complete inside 5-10 min.  With the same volume used
to destain, rinse five times with water, 5 min each.  Soak gel pieces in 0.2 M
ammonium hydrogencarbonate (NH<sub>4</sub>HCO<sub>3</sub>) for 20 min.  Then
repeat the five times rinse with water for 5 min each.  The gel pieces can be
processed or stored at &minus;20&deg; until processed.
<p><b>Troubleshooting</b>.
<ul>
<li>If metallic silver (<q>mirroring</q>) occurs on the gel, this is likely due
to glass plates not being cleaned well enough.
<li>Pressure marks show the present of finger prints or oils on gloves.  Use
powder-free nitrile gloves, and minimize handling of the gels directly with
gloves.
<li>If a brown or black precipitate persists in the silver-ammonia solution, the
new ammonia reagent solution must be purchased.
<li>No development:  if no bands/spots are seen, it could be because the silver
solution or developer solution was made wrongly.  If so, stop development, and 
rinse ten times with water, 30 min each, then re-start the procedure from the 
sensitizer step.
<li>Weak image or slow development:  especially with silver-ammonia step, 
the temperature of the solution is critical.  Make sure that the water is 
pre-warmed (if necessary) to between 18-25&deg;.
<li>Gel is yellowed in background:  this indicates a problem with the thiosulfate.
Get new/fresh reagent.
<li>The black cloud appears during development and leaves a deposit on the gel.
Normally it should dissolve quickly with shaking.  If not, the thiosulfate is
exhausted.  Change it.
<li>What about a gel with bad background?  Stop development and rinse the gel.
Destain with 3 g/L potassium ferricyanide and 6 g/L sodium thiosulfate (better
is ammonium thiosulfate) and with 1% ammonia (10 ml/L).  Should be made just before
use (will have green-yellow color).  After removing the background, rinse several
times with water until background is gone.  Rinse 5 times with water, 30 min each,
then stain using protocol A, starting at sensitization step.
</ul>
<!-- ============================================================= -->
<hr style="margin-top:4em;">
<p>
This is the first in a series of documents containing notes (more than
abstracts but less than the full published reports) on the data and results
in papers relevant to research on methods and methodologies in proteomics,
or in which proteomics is used as a tool.  The (HTML) titles for these
documents are:

<ul>
<li>Notes on the Literature in Proteomics (Vol 1: 19xx-2006)
 <br>[this document:  <tt>ProteomicsLit(19xx-2006).html</tt>]
<li>Notes on the Literature in Proteomics (Vol 2: 2007-)
 <br>[document:
 <tt><a href="ProteomicsLit(2007-).html">ProteomicsLit(2007-).html</a></tt>]
</ul>

</body>
</html>