<!DOCTYPE html>
<html lang="en">
  <head>
    <title>Biology</title>
    <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
    <link href="./stdcss/std.css" type="text/css" rel="stylesheet" />
  </head>

  <body>
    <p id="title">Biology</p>
    <hr />
    <!--   E coli/Prokaryotic Biology  -->
    <h1>
      Biology of
      <i>Eschericia coli</i>
      and Prokaryotes
    </h1>
    <p>
      This is only scattered notes, and not intended to be a detailed review on
      the biology of
      <i>E. coli</i>
      .
    </p>
    <p>
      E. coli is used because it has a very short cell (division) cycle and can
      proliferate from thousands to billions in the matter of a day and night
      (24 h). In logarithmic growth phase, its division cycle period is on the
      order or 20 min to 1 h, depending upon growth medium and other factors. A
      minimal (defined) medium of glucose or glycerol as a carbon source, of
      ammonium salt (sodium ammonium phosphate) or histidine for a nitrogen
      source, and of salts including the ions sodium, potassium, magnesium,
      calcium, sulfate, chloride, and phosphate are used; some trace elements
      are also required. Alternatively a rich medium of partially hydrolyzed
      animal or plant protein, yeast extract (this provides vitamins, enzyme
      cofactors, etc.) as well as a carbon and nitrogen source with salts is
      used.
    </p>
    <p>
      When cells grow into the billions, these are essentially clones (their
      genomes are identical) with the single cell from which they expanded.
      However, with replication of any genome, there is a detectable mutation in
      every 1 in 10
      <sup>6</sup>
      to 10
      <sup>8</sup>
      replications of a gene. With about 3000 genes in
      <i>E. coli</i>
      , that means that by the time a cell has divided to a few thousand, some
      genes will differ. Therefore some means of selection are required to
      remove the cells whose genes of interest has mutated.
    </p>
    <h2>Genome</h2>

    <h3>Plasmids</h3>
    <p>
      Plasmids are extrachromosomal DNA occuring in a circular, superhelical
      double-stranded form ranging from 1 kb to 200 kb.
    </p>
    <p>
      A plasmid is a
      <b>replicon</b>
      if it contains an origin of replication and associated control elements,
      usually several hundred bases long. Today's plasmids originate from a
      single replicon, pMB1, although 30 or more have been identified (pMB1 is a
      close relative to the colicin E1 [colE1] replicon). This keeps about 15-20
      copies per cell. Specialized vectors exist to generate high or low copy
      numbers, depending upon the application.
    </p>

    <h2>Gene Transfer</h2>
    <p>
      Genetic material can be transferred into individual bacterium of E. coli
      and other species
      <i>horizontally</i>
      by three ways rather than merely
      <i>vertically</i>
      (by DNA replication and cell division to daughter cells).
    </p>

    <h3>Conjugation</h3>
    <p>
      Some
      <i>E. coli</i>
      carry an
      <i>F</i>
      plasmid that integrates into the chromosome, making the bacterium an
      <i>Hfr</i>
      <q>male</q>
      strain. Another bacterium necessarily lacking the integrated
      <i>F</i>
      plasmid (
      <q>female</q>
      ) becomes
      <i>conjugated</i>
      with the male bacterium through a
      <b>pilus</b>
      (about 1-2 percent of cell length). During gene transfer, a single strand
      in the male is nicked and this strand (5' or 3' end??) moves through the
      pilus into the female. It carries with it genes that trail the Hfr gene.
      The single strands in the male donor and female recipient are replicated.
      Transfer of the genes can be interrupted at random, with only some of the
      male genes transferred. In the female a recombination/replication cycle
      allows formation of two chromosomes, one containing the original female
      set, and the other incorporating the other the female set with recombined
      male genes. The bacterium is a
      <q>merodiploid</q>
      until cell division segregates the genes. A new third bacterium is now
      present in the soup, and can be selected.
    </p>

    <h3>Transduction</h3>
    <p>
      When bacteriophages infect bacteria, they can contain in their genome some
      genes taken from the previous host in which they grew. This gene transfer
      is known as
      <i>transduction</i>
      .
      <i>Specialized</i>
      or
      <i>restricted</i>
      transducing phages consistently incorporate a particular set of genes.
      These are because these phages may incorporate into particular sites of
      the host genome during
      <b>lysogeny</b>
      and then when a lytic event occurs, they take part of the host genome from
      that integrated site.
      <i>Generalized</i>
      transducing phages are not consistent in the host genes they transfer,
      probably because their biology allows them to incorporate any part of the
      host DNA.
      <i>Defective</i>
      transducing phages are phages incapable of producing lysis to kill their
      hosts during their replicative cycle, but they can still transfer genetic
      material.
    </p>

    <h3>Transformation</h3>
    <p>
      Bacteria can incorporate naked DNA from outside the cell into their cell
      membranes and even integrate it into the host genome.
    </p>

    <!--        Prokaryotic Transcription        -->

    <h2>Prokaryotic Transcription</h2>
    <p>
      This section describes the mechanism of transcription the RNA polymerases
      involved. Translation is coupled to (occurs immediately after)
      transcription in prokaryotes. There is no nucleus that segregates the
      translation machinery from the transcriptional machinery.
    </p>
    <h3>RNA Polymerase</h3>
    <p>
      The
      <a href="">
        RNA polymerase of
        <i>E coli</i>
      </a>
      is one of the first and most extensively studied. It is composed of four
      different proteins (α, β, β′, and ω) which assemble into five subunits
      total (one protein makes up two subunits). Its holoenzyme assembly is α
      <sub>2</sub>
      ββ′ω (M
      <sub>r</sub>
      390,000) and this transcribes all coding DNA It is a DNA-dependent
      polymerase, which requires a DNA template and four ribonucleotides ATP,
      GTP, CTP, and UTP. It also requires external Mg
      <sup>2+</sup>
      as a salt. The enzyme containa a Zn
      <sup>2+</sup>
      atom.
    </p>
    <p>
      The ds DNA template will run through a cleft formed by the interface of
      the large β and β′ subunits, where the esterification active site is for
      polymer extension.
    </p>
    <p>
      The two α subunits will interact probably with typically homodimeric
      DNA-binding proteins that act as transcriptional activators or enhancers
      or in positioning the polymerase.
    </p>
    <p>
      <b>σ factors</b>
      . The polymerase will also associate with other proteins which are σ
      factors and these direct which class of DNA genes the RNA polymerase binds
      to in the bacterium: normal transcription, heat- and cold-shock-based
      transcription. σ
      <sup>70</sup>
      (because its relative mass). The σ
      <sup>32</sup>
      (
      <i>M</i>
      <sub>r</sub>
      32,000) is specific for heat shock.
    </p>
    <p>
      <b>Errors in transcription</b>
      . About 1 for every 10,000 to 100,000 nts, higher than for replication,
      but since RNA is not the permanent source of the genome, it is acceptable.
      Mismatched pairing will cause polymerase pausing with a brief direction
      reversal to remove the mismatch.
    </p>
    <h3>Promoters</h3>
    <p>
      Directs the polymerase assembly to its start site. The promoter region
      begins usually 70 bp upstream and 30 bp downstream relative to
      transcription start because the polymerase binds these regions.
    </p>
    <p>Consensus sequences have been identified:</p>
    <ul>
      <li>
        <b>Pribnow box</b>
        : 6 base-long sequence located between −12 and −7 (centered at −10). The
        consensus sequences is
        <span class="tt">5′-TATAAT-3′</span>
        . The last base is always T, and others are &gt;60% the bases given.
      </li>
      <li>
        <b>−35 region</b>
        . This also contains a 6 bp region with a sequence that varies with a
        consensus of 5′-TTGACA-3′. Positions 2 and 5 are probably highly
        conserved as T and C, respectively. Positions 3, 4, and 6 are quite
        conserved, while position 1 might be G or C also.
      </li>
      <li>
        <b>UP element</b>
        . This has been identified and is present for some genes. It is
        separated by one base from the −35 region, and so it ends at position
        −37. It is 22 bp long with a consensus sequence identified as
        5′-NNAAA[A|T][A|T]T[A|T]TTTTNNAAAANNN3′. The sequence
        5′AGAAAATTATTTTAAATTTCCT-3′ is present in the P1 promoter for the rRNA
        gene
        <i>rrnB</i>
        .
      </li>
    </ul>
    Polymorphisms (natural variations) in the consensus sequences have effects
    on polymerase binding and on the initiation of transcription, which may be
    an intended effect of regulation of gene expression.

    <h3>Transcription Details</h3>
    <p>
      <b>Binding</b>
      . Footprinting studies (DNase cuts with polymerase bound) show that the
      assembly envelopes ~35 bp. Unwinding of the
      <b>closed complex</b>
      to form the
      <b>open complex</b>
      occurs, with a 12-15 bp region of DNA from within the Pribnow (−10) box un
      to position +2 or +3.
    </p>
    <p>
      <b>Initiation</b>
      . A purine (A or G) is usually the first base. It enters the enzyme at the
      triphosphate and the 5′ end retains the triphosphate during transcription
      (5′-pppN). With
      <b>promoter clearance</b>
      (i.e., the polymerase now directionally moved to expose the promoeter),
      which is about 8-9 nt transcribed, the σ subunit is released and
      elongation begins.
    </p>
    <p>
      <b>Elongation</b>
      . The polymerase needs to unwind and wind the DNA, forming a transcription
      <q>bubble</q>
      which is where the double-strandedness of the DNA is lost by opening it
      for transcription. About 17 bp are lose base pairing within the
      polymerase.
    </p>
    <p>
      As NTPs enter, they are correctly paired with the sense DNA strand. The
      3′-OH of the nascent RNA molecule is activated to esterify with the α-P of
      the rNTPs, leading to pyrophosphorlysis (loss of PP
      <sub>i</sub>
      ). The pyrophosphate will be hydrolyzed to drive the reaction
      energetically as well.
    </p>
    <p>
      During elongation, about 8 bp of the DNA-RNA hybrid exists in the
      polymerase. The RNA emerges with loss of base-pairing, and the ss RNA is
      free for immediate translation. The ds DNA strand binds to the surfaces of
      the β and β′, where a cleft formed by the subunits is active site for
      esterification of the rNTP to the growing RNA polymer. There is a Mg
      <sup>2+</sup>
      cation at the active site.
    </p>
    <p>The elongation proceeds at a rate of 50-90 nt per second.</p>
    <p>
      <b>Coiling</b>
      . Ahead of the transcriptional complex, the polymerase drives the positive
      supercoiling of DNA by strand separation, and negative supercoiling behind
      it (observed both in vivo and in vitro). Topoisomerases are required to
      deal with this.
    </p>
    <p>
      <b>Strand nomenclature</b>
      . Note that the strand used as template for transcription is the
      complementary strand to what is called the
      <i>coding</i>
      strand (which is also the non-template strand).
    </p>
    <h3>Regulation of Transcription</h3>
    <p>
      RNA polymerases can be
      <i>enhanced</i>
      or
      <i>activated</i>
      to promote high levels of gene expression. They can also be
      <i>inhibited</i>
      or
      <i>repressed</i>
      to stop or reduce the levels of expression.
    </p>
    <h4>cAMP Receptor Protein (CRP)</h4>
    <p>
      The
      <a href="http://www.uniprot.org/uniprot/P0ACJ8">
        <b>cAMP receptor protein</b>
        (CRP)
      </a>
      , also called
      <b>catabolite activator protein (CAP)</b>
      , was one of the first proteins discovered as a protein that alters the
      rate of transcription of some genes in bacteria. It was the first to have
      its tertiary and primary structure determined and fully characterized, and
      so it figures promimentally in the fundamentals of understanding
      regulation of prokaryotic transcription.
    </p>
    <p>
      The holoprotein is a homodimer, where the subunit is a 23.6 kDa 210 aa
      (209 aa with N-Met cleaved) polypeptide is coded by the
      <i>crp</i>
      in
      <i>E coli</i>
      and features a 124 aa cAMP-binding domain and the interface for homodimers
      (pos 10-133). A 73 aa ds DNA-binding domain (pos 138-210) contains a 20 aa
      helix-turn-helix motif (pos 170-189) that is the canonical site for DNA
      groove binding, binding the sequence 22 bp 5′-AAATGTGATCTAGATCACATTT-3′.
      This sequence is palindromic, with first 11 bases mirroring the latter 11
      bases.
    </p>
    <p>
      Several studies, including 3D structure, show that one half (11 bp) bind
      to one monomer and the other 11 bp bind the other monomer. A crystal
      structure of the homodimer with bound cAMP (one each monomer) to the ds
      sequence 5′-AAATGTGATCTAGATCACATTT-3′ reveals three important amino acids
      on each monomer that bind to specific bases. For the purpose of understand
      what protein interacts with the DNA position, the nomenclature for the
      double-strand is Bnn for
      <u>B</u>
      ase at position
      <u>nn</u>
      , and *Bnn when the base at position
      <u>nn</u>
      is on the complementary strand. The numbering of the DNA is only by the
      top 5′→3′ strand, so is it not the other way. The protein interaction will
      be Mx-Bnn or Mx-*Bnn where x is 1 or 2 for monomer 1 (M1) or 2 (M2). Thus
      M2-A15 would be monomer 2 interacting with base A at position 14 on the
      top (non-complementary) strand. The interactions found for CRP and its
      consensus are thus:
    </p>
    <ul>
      <li>
        M1-G5 and M2-*G18, which are symmetrical interactions. The R180 amino
        acid side chain is involved in this interaction, forming H-bonds with
        guanine O-6 and N-7.
      </li>
      <li>
        M1-G7 and M1-*T8, mediated by the single amino acid R185 on M1. The
        cognate reaction is M2-T15 and M2-*G16, mediated by R185 on M2. R185
        guanidinium interacts by H bone with thymine O-4 and with guanine O-6
        and N-7
      </li>
      <li>
        M1-*C7 mediated by E181 on M1, and M2-C16, mediated by E181 on M2. The
        -COO of Glu forms an H bond with cytosine N-4.
      </li>
    </ul>
    <p>
      Additionally there is strong contact with phosphate groups 3&amp;priime;
      to bases A12, G13, *A9, and *G10. The strongest preferences for binding
      are positions 1-2 and 4-8 for M1, and 15-19 and 21-22 for M2. Binding of
      CRP causes two types of strong bending (kinks) in the DNA between position
      1 and 2, position 6 and 7 by M1, and between position 16 and 17 and
      position 21 and 22 by M2. The inner kinks are primary and the outer are
      secondary.
    </p>
    <p>
      Binding of CRP to its DNA starts with a sequence-dependent formation of
      hydrogen bonds and van der Waal interactions (
      <q>direct readout</q>
      ). It then proceeds to interact with phosphates based on position and
      solvation, and causing deformation (
      <q>indirect readout</q>
      ). A bending of 80° is achieved. Each half has a
      <q>primary kink</q>
      of about 40° to compress major groove between position 6 and 7, then a
      <q>secondary kink</q>
      of about 9° compressing the minor group between positions 1 and 2. The
      T:A/G:C dinucleotide base pair is associated with high roll angles and
      susceptibility to roll deformation. Mutation studies such as E181D
      (shortens side chain by a -CH
      <sub>2</sub>
      -) show that bending still occurs, but the angle is a smooth bend.
      Nanosecond time-resolved fluorescence resonance energy transfer solution
      studies show formation of a DNA bend with mean angle of 77° (consistent
      with crystal data). Millisecond time-resolved luminescence shows the upper
      limit of the bend is 100°. Besides bending by the protein, the sequences
      itself shows inherent slight bending, up to 40-50°. Another factor might
      be numerous electrostatic interactions of the CRP (K22, K26, K44, K166,
      H199, and K201) with phosphates from positions 2 to 5. Mutation studies
      (K22, K26, K44, K166) shows loss of bend angle of 5° per residue per
      half-complex.
    </p>
    <p>
      CRP upregulates transcription in about 100 genes of
      <i>E coli</i>
      typically involved in energy metabolism (galactose, citrate, PEP group
      translocation); these are when energy sources other than glucose might be
      present. It however downregulates its own synthesis and for adenylate
      cyclase (
      <i>cyaA</i>
      ) which generates cAMP from ATP.
    </p>
    <p>
      There are several classes of activation elements (CRP-dependent promoters)
      on the DNA sequence for genes using CRP for activation.
    </p>
    <p>
      Class I elements are upstream the core promoter, and include the
      <i>lac</i>
      operon and the CC(−61.5). (CRP centers at position −61.5 in the case of
      CC.) The α subunit C-terminal domain (αCTD) which is 85 aa long and
      flexibly tethered to the RNA polymerase complex, binds via the
      <i>287 determinant</i>
      to the
      <i>activating region 1</i>
      of the downstream subunit of CRP. Note that αCTD is placed in interaction
      with σ
      <sup>70</sup>
      too as well as with the minor groove of DNA (via 265 determinant) next to
      CAP. The AR1-αCTD interaction as shown by crystal structures of CRP, DNA
      consensus, and αCTD (not whole RNAP) is an interaction of only 6 residues
      on both proteins with no conformational change in the binding: thus
      transcriptional activation is simple recruitment.
    </p>
    <p>
      Class II elements have CRP binding overlapping the −35 region and examples
      include the
      <i>galP1</i>
      and CC(−41.5) artificial promoters. Three sets of interactions occur
      compared to just one with the αCTD. There is an
      <i>activating region 2</i>
      (AR2) in the downstream subunit interacting with residues 162-165 in α
      subunit N-terminal domain (αNTD) of the RNA polymerase.
      <i>Activating region 3</i>
      (AR3) of downstream CRP subunit interacts with residues 593-603 of σ
      <sup>70</sup>
      (called region 4 or σR4), which is involved in −35 region. These AR2 and
      AR3 interactions activate via a post-recruitment mechanism, facilitating
      isomerization of the polymerase closed complex to form the open complex.
      The other αCTD of RNAP will interact with DNA minor groove: each αCTD can
      alternately interact with CRP or the DNA minor groove.
    </p>
    <p>
      There are also class III elements in which CRP works together with other
      co-activator proteins.
    </p>
    <h4>Operons and Repressors</h4>
    <p>
      Bacterial transcription also features
      <i>operons</i>
      , where several related genes are produced on a single transcript (
      <q>polycistronic</q>
      mRNA) and then then translated.
    </p>
    <p>
      <b>Repression</b>
      involves transcriptional regulation scheme whereby a protein (
      <i>repressor</i>
      ) binds to a region of DNA called the
      <i>operator</i>
      ) and the binding prevents RNA polymerase binding (and therefore
      transcription). An
      <i>effector</i>
      binds to a DNA-binding protein to either enhance or enable its binding or
      to reduce its binding. An
      <i>inducer</i>
      is an effector that binds to a repressor to reduce its ability to bind the
      operator; it effectively induces transcription. The
      <i>lac</i>
      operon is the best example of an inducible system (lactose binds to the
      repressor to cause it to release from its operator). A
      <i>co-repressor</i>
      is an effector that is required to bind to a repressor to enable the
      latter to bind to its operator region, and inhibit transcription. The
      <i>trp</i>
      operon is an example of a co-repressing regulation (in this case, the
      amino acid itself, tryptophan, binds to the repressor to effect
      repression).
    </p>
    <h5>
      The
      <i>lac</i>
      operon
    </h5>
    <p>
      The presence of lactose in medium growing
      <i>E. coli</i>
      allows for lactose to bind the inducible repressor of the operon. The
      operon consists of the
      <i>lacZ</i>
      gene coding for β-galactosidase, the
      <i>lacY</i>
      coding for a permease that binds to the inner side of the membrane to
      increase lactose uptake, and the
      <i>lacA</i>
      gene coding for a thiogalactoside transacetylase (GAT) which contains rare
      codons and probably is derived from a plasmid, transposon, or phage gene;
      this gene codes for a protein acetylates nonmetabolizable pyranosides to
      prevent cell re-entry and helps in detoxification. The
      <i>lacI</i>
      gene codes for the 360 aa repressor and is not part of the operon.
      Mutations in either the 25-30 bp operator DNA (
      <i>o</i>
      <sup>c</sup>
      ) sequence to which the lacI repressor binds or to the
      <i>lacI</i>
      gene causes constitutive transcription of the
      <i>lac</i>
      operon.
    </p>
    <h5>
      The
      <i>ara</i>
      operon
    </h5>
    <p>
      The arabinose (
      <i>ara</i>
      ) operon also consists of three genes,
      <i>araB</i>
      coding for an isomerase,
      <i>araA</i>
      coding for a kinase, and
      <i>araD</i>
      coding for an epimerase. All these convert arabinose into xylulose. The
      regulatory gene is coded by
      <i>araC</i>
      which acts as a required inducer; mutations in AraC shut down the
      <i>ara</i>
      operon. AraC in the absence of arabinose, AraC binds to a location
      immediately upstream the
      <i>ara</i>
      operon promoter. It also binds to a location −280 bp upstream, apparently
      by bending the DNA, and this represses transcription. In the presence of
      arabinose, the sugar binds to AraC and the subsequent allosterism prevents
      binding to the −280 bp site and turns AraC into an RNA polymerase binding
      enhancer.
    </p>
    <h5>
      The
      <i>gal</i>
      operon
    </h5>
    <p>
      The galactose operon is composed of a kinase, transferase, and epimerase.
      Induction the operon increase expression 10- to 15-fold, unlike the
      <i>lac</i>
      operon which induces by 1000-fold. When glucose is present, induction of
      <i>gal</i>
      and
      <i>lac</i>
      operons is reduced although there is basal expression. The basal level is
      because the epimerase is required for a component of the cell wall. It has
      been shown that
      <i>gal</i>
      operon has two transcriptional start sites (offset by 5 bp). This is
      because the promoter has two overlapping Pribnow boxes and two places (S
      <sub>1</sub>
      and S
      <sub>2</sub>
      ) where the polymerase can bind. In low levels of cAMP (for catabolite
      activation), the polymerase starts from S
      <sub>2</sub>
      and transcribes at a low level. When high levels of transcription occurs
      (with high cAMP levels), the polymerase binds S
      <sub>1</sub>
      .
    </p>
    <h5>The Arg repressor</h5>
    <p>
      Most repressors repress only one transcriptional unit (operon). The
      feature of the Arg repressor is that it regulates multiple operons
      (sites).
    </p>
    <h3>Phage T7 RNA Polymerase</h3>
    <p>
      Transcription rate significantly faster than
      <i>E coli</i>
      RNA polymerase, and it terminates less frequently, and can infect
      transcript a plasmid many times the size of that plasmid. It has its own
      promoter (
      <i>p</i>
      <sub>T7</sub>
      ) from which the polymerase is more efficient. And it is resistant to
      rifampicin (
      <i>E coli</i>
      polymerase is sensitive). A two-plasmid expression system with insert
      under control of T7 promoter on one plasmid and the T7 RNA polymerase on
      the second plasmid is induced. This second plasmid can be present in a
      host strain, or be constructed into M13 (vector mGP1-2) or λ (vector CE6).
    </p>
    <p>
      The pT7 and pET series vectors are available for use. The
      <i>E coli</i>
      host already has pGP1-2 which contains the T7 polymerase gene under the λ
      <i>p</i>
      <sub>L</sub>
      control. This is normally bound by the
      <i>c</i>
      I857 repressor which is temperature sensitive. pGP1-2 also has a p15A
      <i>ori</i>
      site compatible with ColE1
      <i>ori</i>
      . Both plasmids can be maintained using kanamycin (pGP1-2 has kanamycin
      resistance) and ampicillin (resistance on pT7 and pET vectors). Culture at
      non-inducing temperature of 30° is done for several hours, and then
      induced at 42°.
    </p>
    <p>The T7 promoter sequence is actually:</p>
    <p>
      TAATACGACTCACTATA
      <b style="color: red">GGGAGA</b>
    </p>
    <p>
      where the boldface red bases show where transcription actually starts and
      these are transcribed. Alternatively the
      <i>lac</i>
      I repressor can be used instead of the lambda repressor for induction by
      IPTG.
    </p>
    <table class="aligntop">
      <caption>Bacterial Strains Used In Molecular Cloning</caption>
      <colgroup>
        <col style="width: 6em; font-size: 90%" />
        <col style="width: 15em; font-size: 83%" />
        <col style="width: 6em; font-size: 90%" />
      </colgroup>
      <tbody>
        <tr>
          <th>Strain</th>
          <th>Genotype</th>
          <th>Remarks</th>
        </tr>
        <tr>
          <td>BL21(DE3)</td>
          <td>
            F
            <sup>−</sup>
            <i>dcm ompT</i>
            <i>hsdS</i>
            (r
            <sub>
              B
              <sup>−</sup>
            </sub>
            m
            <sub>
              B
              <sup>−</sup>
            </sub>
            )
            <i>gal</i>
            (λ
            <i>c</i>
            I
            <i>ts</i>
            857
            <i>ind</i>
            1
            <i>S</i>
            am7
            <i>nin</i>
            5
            <i>lac</i>
            UV5-T7 gene
            <i>1</i>
            )
          </td>
          <td>
            Used to express at high levels gene cloned into vectors under
            control of T7 promoter. Host chromosome integrates the λ DE3 phage,
            which carries the T7 RNA polymerase.
          </td>
        </tr>
        <tr>
          <td>BNN93 (C600)</td>
          <td>
            <i>hsdR supE</i>
            44
            <i>thi-</i>
            1
            <i>thr</i>
            -1
            <i>leuB</i>
            6
            <i>lacY</i>
            1
            <i>tonA</i>
            21 e14
            <sup>−</sup>
            (McrA
            <sup>−</sup>
            )
          </td>
          <td>
            Used with λgt10 cDNA library cloning. It is a permissive host.
          </td>
        </tr>
        <tr>
          <td>BNN102</td>
          <td>
            <i>hsdR supE</i>
            44
            <i>thi-</i>
            1
            <i>thr</i>
            -1
            <i>leuB</i>
            6
            <i>lacY</i>
            1
            <i>tonA</i>
            21
            <i>hflA</i>
            150[
            <i>chr</i>
            ::Tn
            <i>10</i>
            (
            <i>tet</i>
            <sup>r</sup>
            )]
          </td>
          <td>
            Used with λgt10 cDNA library cloning: it is a non-permissive host
            for λgt10 becoming permissive with cDNA insertion. Because it is a
            <i>hfl</i>
            mutant, it suppresses plaque formation in
            <i>c</i>
            I
            <sup>+</sup>
            phages. This strain is C600/BNN93 with the
            <i>hfl</i>
            mutation.
          </td>
        </tr>
        <tr>
          <td>DH1</td>
          <td>
            <i>supE</i>
            44
            <i>hsdR</i>
            17
            <i>recA</i>
            1
            <i>endA</i>
            1
            <i>gyrA</i>
            96
            <i>thi</i>
            -1
            <i>relA</i>
            1
          </td>
          <td>
            For plating and growth of plasmids and cosmids;
            recombination-deficiency can be suppressed.
            <i>gyrA</i>
            codes for a nalidixic acid-resistant form of DNA gyrase. Derives
            from the MM294 strain.
          </td>
        </tr>
        <tr>
          <td>DH5</td>
          <td>identical to DH1</td>
          <td>
            Used in the same way as DH1 strain, but has higher transformation
            efficiency
          </td>
        </tr>
        <tr>
          <td>DH5α</td>
          <td>
            DH1 type + Δ
            <i>lac</i>
            U169 (φ80
            <i>lacZ</i>
            ΔM15)
          </td>
          <td>
            Used as for DH1 strain. Integration of a φ80 phage with carboxyl
            terminus of β-galactosidase permits
            <i>α</i>
            -complementation with pUC vectors containing amino terminus of
            enzyme.
          </td>
        </tr>
        <tr>
          <td>HB101</td>
          <td>
            <i>supE</i>
            44
            <i>hsdS</i>
            20(r
            <sub>B</sub>
            <sup>−</sup>
            m
            <sub>B</sub>
            <sup>−</sup>
            )
            <i>recA</i>
            13
            <i>ara</i>
            -14 Δ(
            <i>gpt-proA</i>
            )62
            <i>lacY</i>
            1
            <i>galK</i>
            2
            <i>rpsL</i>
            20
            <i>xyl</i>
            -5
            <i>mtl</i>
            -1 Δ(
            <i>mcrC-mrr</i>
            ) (Str
            <sup>r</sup>
            )
          </td>
          <td>
            Useful for the large scale production of plasmids. Ability to
            restrict or modify DNA absent. The strain can be transformed without
            difficulty. A hybrid of the K12 and B strains of
            <i>E. coli</i>
            .
          </td>
        </tr>
        <tr>
          <td>JM101</td>
          <td>
            <i>supE</i>
            <i>thi</i>
            -1 Δ(
            <i>lac-proAB</i>
            ) [F´
            <i>traD</i>
            36
            <i>proAB</i>
            <sup>+</sup>
            <i>lacI</i>
            <sup>q</sup>
            <i>lacZ</i>
            ΔM15]
          </td>
          <td>
            To grow vectors with amber mutations. Is a male strain (contains
            <i>F</i>
            plasmid) that does not transfer. Features
            <i>lac</i>
            repressor overproducer, and
            <i>lac</i>
            α-complementing system
          </td>
        </tr>
        <tr>
          <td>JM103</td>
          <td>
            <i>supE</i>
            44
            <i>thi</i>
            -1
            <i>strA</i>
            <i>endA</i>
            1
            <i>sbcB</i>
            <i>hsdR</i>
            <sup>−</sup>
            Δ(
            <i>lac pro</i>
            ) F´[
            <i>traD</i>
            36
            <i>proAB</i>
            <i>lacI</i>
            <sup>q</sup>
            <i>lacZ</i>
            ΔM15]
          </td>
          <td>
            For plating M13 derivatives. Contains other activities as for JM101
            strain.
          </td>
        </tr>
        <tr>
          <td>JM109</td>
          <td>
            JM103 +
            <i>recA</i>
            1
          </td>
          <td>recombination-deficient form of JM103.</td>
        </tr>
        <tr>
          <td>TG1</td>
          <td>
            chromosome:
            <i>supE thi-1</i>
            Δ(
            <i>lac-proAB</i>
            ) Δ(
            <i>mcrB-hsdSM</i>
            )5, (
            <i>
              rK
              <sup>−</sup>
              mK
              <sup>−</sup>
            </i>
            )
            <br />
            plasmid: F′ [
            <i>tra</i>
            D36
            <i>proAB</i>
            <sup>+</sup>
            <i>lacI</i>
            <sup>q</sup>
            <i>lac</i>
            ZΔM15]
          </td>
          <td>useful for phage display</td>
        </tr>
        <tr>
          <td>Y1090</td>
          <td>
            Δ
            <i>lacU</i>
            169 Δ(
            <i>lon,araD</i>
            )139
            <i>strA</i>
            <i>supF</i>
            (
            <i>trpC</i>
            ::Tn
            <i>10</i>
            ) (pMC9)
          </td>
          <td>
            Used to screen (immunochemically) expression of cDNA inserts in
            λgt11 and λgt18-23. pMC9 codes for a
            <i>lac</i>
            repressor (
            <i>lacI</i>
            <sup>q</sup>
            ) in high levels. IPTG is added several hours after plaque
            formation.
            <i>lon</i>
            is a protease that interferes with expression or antibody screening.
            <i>supF</i>
            suppresses
            <i>S</i>
            am100 which would stop cell lysis.
          </td>
        </tr>
        <tr>
          <td colspan="3" style="font-size: 83%">
            Sources: Sambrook et al.,
            <i>Molecular Cloning: A Laboratory Manual</i>
            , 2nd Ed., 1989.
          </td>
        </tr>
      </tbody>
    </table>
    <p>
      A list of genotypes of E coli strains is available
      <a href="http://wheat.pw.usda.gov/ggpages/probes/strains.html">here</a>

      <!--        Prokaryotic Translation          -->
    </p>
    <h2>Prokaryotic Translation</h2>
    <p>
      Translation of the messenger RNA (mRNA) created by transcription is done
      by ribosomes that move along the strand, inserting a
      <q>charged</q>
      tRNA (aminoacyl-tRNA) into a space of the ribosome, and mediating the
      transfer of the amino acid to an ever increasing polymer of amino acids
      (the polypeptide).
    </p>
    <p>
      The prokaryotic ribosome is assembled having a size of 70S at initiation
      of translation. This assembly is of two subunits of sizes 30S and 50S,
      which are present in the cytosol and assembled on to the mRNA in stages.
    </p>
    <p>
      Acylation of the tRNA with the amino acid first begins with the
      esterification of the α-carboxyl with ATP, forming a phosphoanhydride bond
      with the α-phosphate, the products being AMP-amino acid and pyrophosphate
      (PP
      <sub>i</sub>
      ). The 3'-end of tRNA is composed of a constant trimer 5'-CCA-3'. The
      α-carboxyl of the amino acid transesterifies with the adenyl 2'-hydroxyl
      of the tRNA, releasing AMP and forming 2'-aminoacyl-tRNA, the fully
      <q>charged</q>
      tRNA.
    </p>
    <p>
      Initiation of the polypeptide begins with the first aminoacyl-tRNA, a
      formylated methionyl-tRNA (fMet-tRNA
      <sub>f</sub>
      ). The ribosome is assembled on the mRNA and moves to a special start
      sequence called the Shine-Dalgarno sequence which pairs with a small
      sequence on the 16S rRNA in the 30S subunit.

      <!--               Bacteriophages                 -->
    </p>
    <h1><a id="phages">Bacteriophages</a></h1>
    <p>
      The genome of bacteriophages can be modified to allow insertion of large
      pieces of DNA (cDNA and some high MW genomic DNA) for cloning, sequencing,
      expression, etc. This makes them more useful than plasmids, which are
      limited in the size of the inserts they can accomodate.
    </p>
    <h2>Known Phages</h2>

    <h3>
      T phages of
      <i>E. coli</i>
    </h3>
    <p>
      These DNA phages are lytic. T2, T4, and T6 (T-even) phages are about 200
      kb long and T1, T3, and T7 about 40 kb long. Each lytic cycle produces 100
      new phage per bacterium in about 20 min.
    </p>
    <h3>
      Temperate phages of
      <i>E. coli</i>
    </h3>
    <p>
      λ phage is the classical type. Viral DNA circularizes after infection and
      then can become integrated into the host chromosome using the host's
      recombination system.
    </p>
    <h3>
      Small phages of
      <i>E. coli</i>
    </h3>
    <p>
      The entire genomes of these phages have been sequenced and studied. φX174
      is classical
    </p>
    <h3>
      RNA phages of
      <i>E. coli</i>
    </h3>
    <p>
      When they infect the cell, they essentially have their mRNA ready. This
      can produce reverse transcriptases to convert the RNA into DNA form, from
      which multiple copies of DNA are made, and then RNA later transcribed from
      these for packaging into virion capsids.
    </p>
    <h2>λ Phage</h2>
    <p>
      λ phage is 49.4 kb but half of it can be removed to be substituted with
      foreign DNA and to allow the DNA to be packaged into phage capsids. λ is
      ideal for making genomic libraries. The minimum number of recombinant λ
      phage to cover a genome is the size of the genome divided by the insert
      size of λ recombinants. Therefore to cover the human genome of 4 × 10
      <sup>9</sup>
      bp, into fragments of 20 kbp means that 200,000 recombinants are
      necessary. In reality, about 10
      <sup>6</sup>
      recombinants will give a 90-95% chance that the entire genome is covered.
    </p>
    <h3>Infection</h3>
    <p>
      λ infects
      <i>E. coli</i>
      via transporters in the outer membrane used for maltose and coded by the
      bacterium's
      <i>lamB</i>
      gene. Glucose represses expression of the maltose transporters, and so in
      order to enable infection with λ,
      <i>E coli</i>
      host should be cultured in glucose-free maltose medium. Infection is
      facilitated by magnesium ions and happens within minutes at room temp or
      37°. Proliferation of phage does not occur at room temperature however, so
      the incubation for plaques on a bacterial lawn should be at 37°.
    </p>
    <p>
      The dsDNA λ circularizes in the host because of cohesive, sticky ends
      (representing the
      <i>cos</i>
      sequence). Host ligase ligates the nicks. λ then immediately begins
      transcription of products that compete for lysogenic or lytic phase.
    </p>
    <h3>Lysogeny vs. Lysis</h3>
    <p>
      There are two key promoters (P
      <sub>L</sub>
      and P
      <sub>R</sub>
      ) involved in the transcription of phage mRNA. These are regulated by a
      master transcriptional repressor, a product of the
      <i>c</i>
      I gene. Upon infection transcription of P
      <sub>L</sub>
      and P
      <sub>R</sub>
      begins immediately. The transcript of P
      <sub>L</sub>
      is about 1000 bases (12S) and codes for the N gene. The transcript of the
      P
      <sub>R</sub>
      is about 500 bases (8S-9S) and codes for the
      <i>cro</i>
      gene. The
      <i>cro</i>
      gene codes for a protein that binds the two promoters and represses early
      transcription. The N protein is an anti-transcriptional terminator protein
      that binds to terminators
      <i>t</i>
      <sub>L</sub>
      and
      <i>t</i>
      <sub>R1</sub>
      and
      <i>t</i>
      <sub>R2</sub>
      .
      <i>t</i>
      <sub>L</sub>
      and
      <i>t</i>
      <sub>R1</sub>
      , are sites located at the 3'-end of the N and
      <i>cro</i>
      genes, resp. N protein binds to ρ protein to prevent
      <i>rho</i>
      -dependent termination of transcription so that the polymerase continues
      transcription. There are
      <i>
        N
        <sup>−</sup>
      </i>
      mutants which can continue to lytic growth so long as
      <i>t</i>
      <sub>R2</sub>
      is deleted: these are known as
      <i>nin</i>
      (
      <i>N-in</i>
      dependent) mutants.
    </p>
    <p>
      This anti-termination of transcription by N protein thus permits
      expression of the
      <i>c</i>
      II and
      <i>c</i>
      III genes along with genes to the right and left of these genes, resp.
      Genes downstream of
      <i>c</i>
      III promote integration (lysogeny) of λ DNA, whereas genes downstream of
      <i>c</i>
      II codes for DNA replication and production and release of phage (lysis).
      <i>c</i>
      II is a DNA-binding protein that binds to the promoter site P
      <sub>E</sub>
      (promoter for repressor establishment). This increases transcription of
      the
      <i>c</i>
      I repressor, which is also under the control of the promoter P
      <sub>RM</sub>
      .
      <i>c</i>
      III interferes with the activity of H
      <sub>f</sub>
      l (a protease?) that decreases activity of
      <i>c</i>
      II gene. Thus an increase in
      <i>c</i>
      II activity leads to a burst in
      <i>c</i>
      I repressor synthesis.
      <i>c</i>
      I binds to P
      <sub>RM</sub>
      which is to the left of P
      <sub>R</sub>
      separated by three operator regions, and which is responsible for the
      transcription of
      <i>c</i>
      I (for repressor maintenance). Thus
      <i>c</i>
      I upregulates (maintains) its transcription.
    </p>
    <p>
      The three-operator region between P
      <sub>R</sub>
      and P
      <sub>RM</sub>
      is interesting. There is also one upstream the P
      <sub>L</sub>
      promoter. The order of the DNA sequences is:
    </p>
    <p>
      integration genes—
      <i>c</i>
      III—...—N←P
      <sub>L</sub>
      –O
      <sub>L</sub>
      1–O
      <sub>L</sub>
      2–O
      <sub>L</sub>
      3—
      <i>c</i>
      I←P
      <sub>RM</sub>
      – O
      <sub>R</sub>
      3–O
      <sub>R</sub>
      2–O
      <sub>R</sub>
      1– P
      <sub>R</sub>
      →
      <i>cro</i>
      ←P
      <sub>E</sub>
      —
      <i>c</i>
      II—DNAreplication
    </p>
    <p>
      Each operator is a 17 bp. P
      <sub>RM</sub>
      is close to the O
      <sub>R</sub>
      3 site.
      <i>c</i>
      I dimerizes into a high affinity DNA-binding protein. Binding is in the
      order of O
      <sub>R</sub>
      1, then O
      <sub>R</sub>
      2, then O
      <sub>R</sub>
      3. The dimer cooperatively binds with another, and so O
      <sub>R</sub>
      1 and O
      <sub>R</sub>
      2 are saturated, but not O
      <sub>R</sub>
      3. O
      <sub>R</sub>
      1/O
      <sub>R</sub>
      2 occupation stimulates transcription from P
      <sub>RM</sub>
      . When repressor gets so high that O
      <sub>R</sub>
      3 is occupied, this represses transcription from P
      <sub>RM</sub>
      , putting a negative feedback into the positive feedback system.
      Occupation of O
      <sub>R</sub>
      1 alone shuts down P
      <sub>R</sub>
      transcription, shutting off
      <i>cro</i>
      and the lytic phase.
      <i>cro</i>
      is a DNA-binding protein that binds with the following affinity: O
      <sub>R</sub>
      3 &gt; O
      <sub>R</sub>
      2 &gt; O
      <sub>R</sub>
      1. Its transcription becomes activated when some crisis occurs that
      triggers proteolytic breakdown of
      <i>c</i>
      I repressor (UV irradiation and DNA repair). As
      <i>cro</i>
      is produced it quickly shuts down
      <i>c</i>
      transcription by binding O
      <sub>R</sub>
      3 and inhibiting P
      <sub>RM</sub>
      . N gene activity is turned on to all antitermination of transcription for
      lytic genes.
    </p>
    <p>
      Integration in lysogeny is achieved by use of the host's recombination
      system. The
      <i>att</i>
      sequence on the circular phage aligns with the
      <i>att</i>
      sequence on the bacterial chromosome, which is between the
      <i>gal</i>
      and
      <i>bio</i>
      operons, for the recombination.
    </p>
    <p>
      DNA replication in active lytic growth is marked by the appearance under
      EM of Cairns or θ forms. This shows the phage in the process of activity
      DNA replication with the two synthetic forks. The
      <i>O</i>
      and
      <i>P</i>
      genes are the early genes that mediate this type of DNA replication.
      Later, expression of the phage
      <i>gam</i>
      gene converts the DNA replication from bidirectional replication of the
      circle to the
      <q>rolling circle</q>
      replication, in which circles are attached to capsids and linear strands
      are packaged into phage heads as replication rolls off the circle. The
      ends of the phage DNA start and end at
      <i>cos</i>
      sequence (
      <i>cos</i>
      L and
      <i>cos</i>
      R), with restriction-like cleavage by the product of the
      <i>A</i>
      gene at the capsid portal. The host's exonuclease V protein (encoded by
      <i>recB</i>
      and
      <i>recC</i>
      inhibits the rolling circle replication in
      <i>gam</i>
      mutants, as
      <i>gam</i>
      product specifically is involved with exonuclease V inhibition.
    </p>
    <p>
      The Q protein is expressed as part of the early transcriptional gene
      products, under control of P
      <sub>R</sub>
      promoter so long as termination is prevented by N protein. Its function is
      to prevent inhibited transcription from a promoter, P
      <sub>R′</sub>
      , involved in late phase synthesis of proteins involves in lysis (head and
      tail proteins as well as lysis).
    </p>
    <p>
      Viral assembly in lytic phase starts with the
      <b>scaffolded prehead</b>
      . Centered around the
      <i>Nu3</i>
      product, the
      <i>C</i>
      gene product is a long protein that extends radially as spokes in
      multimers from
      <i>Nu3</i>
      . The
      <i>B</i>
      gene product codes for a protein that extends as one of these spokes from
      <i>Nu</i>
      3. Encircling this assembly at the circumference are multimers of the
      <i>E</i>
      gene product. The scaffolded prehead thus forms. A host protein coded by
      <i>groE</i>
      cleaves the radial spokes and center from the scaffolded head, leaving a
      space within; this is now known as the head.
      <i>Nu</i>
      1 binds to the
      <i>A</i>
      gene product, and this complex then binds close to the left
      <i>cos</i>
      sites of the long strand of DNA forming multiple genomes (concatenated
      linear DNA) from the rolling circle. Under the control of the
      <i>F</i>
      I protein, the capsid takes in the DNA, swelling the head by 20%, up to
      the
      <i>Nu</i>
      1/
      <i>A</i>
      dimer. After filling, the
      <i>D</i>
      (for
      <q>decoration</q>
      ) attaches to the outside and finishes packaging the head around the DNA.
      At this time, the
      <i>ter</i>
      activity of the A protein creates the 12 bp sticky ends.
      <i>F</i>
      II then creates the base upon which an assembled tail is attached. The
      tail itself is assembled not part-by-part from the head, but as a complete
      unit separately. It is composed of the products of the genes
      <i>I</i>
      ,
      <i>J</i>
      ,
      <i>K</i>
      ,
      <i>L</i>
      ,
      <i>G</i>
      ,
      <i>H</i>
      ,
      <i>M</i>
      ,
      <i>V</i>
      ,
      <i>U</i>
      , and
      <i>Z</i>
      . The genes
      <i>R</i>
      and
      <i>S</i>
      are involved with lysis. Mutations in
      <i>S</i>
      can prevent, often delay, lysis. This can result in higher numbers of
      phage produced per bacterium, which can be recovered by artificial lysis
      with chloroform.
    </p>
    <h3>Lytic Induction</h3>
    <p>
      λ integrated into the
      <i>E. coli</i>
      chromosome can be induced to leave its lysogenic state and enter the lytic
      state with the production of progeny phage by environmental changes that
      might kill the host, Normally in the integrated state, the only phage
      genes expressed are
      <i>rexA</i>
      ,
      <i>rexB</i>
      , and the
      <i>c</i>
      I repressor. (See above how low and high concentrations of
      <i>c</i>
      I repressor serve as a positive and negative feedback on transcription
      from the P
      <sub>RM</sub>
      promoter to regulate levels.) The
      <i>c</i>
      I repressor can be inactivated in different ways. If it is
      temperature-sensitive, one is to raise the temperature to about 45° to
      inactivate the protein. Another is to use chemicals to cause DNA damage,
      which activates the host's RecA protein, which proteolyzes
      <i>c</i>
      I protein. With loss of
      <i>c</i>
      I, transcription starts at P
      <sub>L</sub>
      and P
      <sub>R</sub>
      . Synthesis of the Int and Xis (for e
      <i>X</i>
      c
      <i>is</i>
      e) permits recombination at
      <i>att</i>
      to remove the prophage DNA and start the lytic cycle.
    </p>
    <h3>λ As Tools in Molecular Cloning</h3>
    <p>
      Large regions of the phage genome can be substituted with foreign DNA
      inserts for cloning. The region from the
      <i>J</i>
      to
      <i>N</i>
      gene is about 16-17 kb (19000 to 35000 on the λ map, with the
      <i>cos</i>
      site at 0) were found to naturally carry
      <i>E coli</i>
      DNA as a result of specialized transduction, and it was thought that λ
      could be used as a vector for that reason.
    </p>
    <p>
      In the course of developing λ as a vector, all
      <i>Eco</i>
      RI sites were eliminated from the parts of the genome required for the
      vector (the parts that produce lysis). This is done by growing phage
      alternately in
      <i>E coli</i>
      that carry or do not carry the plasmid coding
      <i>Eco</i>
      RI. Thus any
      <i>Eco</i>
      RI sites are in the insert region, and so can be exploited in this way.
      Today insertion or elimination (by substitution or deletion) of
      restriction sites would be achieved using site-directed mutagenesis with
      synthetic oligonucleotides. Note that vectors having a
      <i>single</i>
      <b>unique</b>
      restriction site are
      <b><i>insertion</i></b>
      vectors, whereas those with
      <i>double</i>
      unique restriction sites at the ends of nonessential regions of a vector
      are called
      <i><b>replacement</b></i>
      vectors.
    </p>
    <p>When choosing a λ vector, the considerations are</p>
    <ul>
      <li>the restriction enzymes to be used</li>
      <li>the size of the insert or replacement</li>
      <li>
        whether any of the clone DNA sequences are to be expressed in the
        <i>E. coli</i>
        host
      </li>
    </ul>
    <p>
      About 60% of the genome is required as a vector. ~20 kb in the left arm
      codes for genes
      <i>A-J</i>
      (the head and tail structure), and 8-10 kb of the right arm, from the P
      <sub>R</sub>
      to the
      <i>cos</i>
      R, are necessary. When the insert exceeds 105% or is less than 78% of the
      length, empty or nonvirulent phage progeny are formed.
    </p>
    <p>
      Vectors are used to complement their hosts, particularly for screening or
      selection.
      <i>
        lac
        <sup>−</sup>
      </i>
      hosts will be used with λ vectors carrying
      <i>lacZ</i>
      so that screening can be done seeing blue or white plaques on plates
      containing X-gal: the
      <i>lacZ</i>
      will actually be a part of the DNA to be replaced in the λ vector by the
      insert, so blue plaques indicate non-replacement.
    </p>
    <p>
      Some
      <i>E. coli</i>
      carry a prophage P2 (a prophage a phage whose genome is integrated into
      the host genome) that interfere with the growth of λ phage. This host
      phenotype is Spi
      <sup>+</sup>
      (
      <i>s</i>
      ensitive to
      <i>P</i>
      2
      <i>i</i>
      nterference). If λ is missing the genes
      <i>red</i>
      and
      <i>gam</i>
      involved in recombination, it will grow in P2 lysogens nonetheless (Spi
      <sup>−</sup>
      ) if they carry the
      <i>chi</i>
      site (a recombination site controlled by
      <i>recBC</i>
      ) and the host is
      <i>rec</i>
      <sup>+</sup>
      . Vectors λ2001, λDASH, and the EMBL series carry
      <i>red</i>
      and/or
      <i>gam</i>
      in the stuffer, so non-recombinants will fail to grow whereas recombinants
      grow using E. coli containing P2 lysogens.
    </p>
    <p>
      Sometimes it is useful to maintain the
      <i>red</i>
      and/or
      <i>gam</i>
      genes in the vector. Since the
      <i>gam</i>
      gene encodes the exonuclease V that can degrade concatenates of λ DNA
      produced from the rolling circle, the only DNA available for packaging is
      from the closed circular dimers generated by θ-form replication and
      recombination caused by λ
      <i>red</i>
      or host
      <i>recA</i>
      system. In many vectors, the
      <i>gam</i>
      has been moved to an arm and is used in
      <i>recA</i>
      <sup>−</sup>
      hosts. This is especially necessary when the insert contains repeated
      sequences that are prone to any recombinatory process (recombinogenic).
    </p>
    <table class="topalign">
      <caption>λ Vectors</caption>
      <colgroup>
        <col />
        <col />
        <col style="text-align: left" />
      </colgroup>
      <tbody>
        <tr>
          <th rowspan="2">Series</th>
          <th rowspan="2">Species</th>
          <th>Genotype</th>
        </tr>
        <tr><th>Features</th></tr>
        <tr>
          <td rowspan="12">Charon</td>
          <td rowspan="2">4A</td>
          <td class="monospace">
            λ
            <i>A</i>
            am32
            <i>B</i>
            am1
            <i>lac</i>
            5
            <i>bio</i>
            256 KH54
            <i>sr</i>
            Iλ4°
            <i>nin</i>
            5
            <i>QSR</i>
            80
          </td>
        </tr>
        <tr>
          <td>
            Charon 4A used for eukaryotic genomic libraries. Partially digested
            genomic DNA are fitted with
            <i>Eco</i>
            RI linkers (if needed). Stuffer of vector composed of 6.9 and 7.8 kb
            <i>Eco</i>
            RI fragments. Left arm
            <i>Eco</i>
            RI site inside
            <i>lacZ</i>
            and right arm site inside
            <i>bio</i>
            operon, thus recombinants are Lac
            <sup>−</sup>
            Bio
            <sup>−</sup>
            . Single
            <i>Xba</i>
            I site allows vector to be used for maximum 6 kb insertion. Genes
            <i>A</i>
            and
            <i>B</i>
            with amber mutations requiring host with
            <i>supE</i>
            or
            <i>supF</i>
            suppressor. Vector is
            <i>
              red
              <sup>−</sup>
              gam
              <sup>−</sup>
            </i>
            requring
            <i>recA</i>
            <sup>+</sup>
            hosts (
            <i>E. coli</i>
            LE392 host recommended for all Charon type).
          </td>
        </tr>
        <tr>
          <td rowspan="2">21A</td>
          <td class="monospace">
            λ
            <i>W</i>
            am403
            <i>E</i>
            am1100
            <i>b</i>
            1007
            <i>sr</i>
            Iλ3°
            <i>imm</i>
            80 KH53 KH52
            <i>sr</i>
            Iλ4°
            <i>nin</i>
            5
            <i>shn</i>
            dIIIλ6°
            <i>sr</i>
            Iλ5°
          </td>
        </tr>
        <tr>
          <td>
            Charon 21A can act as an insertion vector (0-9 kb) or be replacement
            vector up to 15 kb. The vector is
            <i>
              Spi
              <sup>−</sup>
            </i>
            and the host must contain an amber suppressor (
            <i>supE</i>
            or
            <i>supF</i>
            ). The insertion site has four possible restrictions (
            <i>Eco</i>
            RI,
            <i>Hin</i>
            dIII,
            <i>Xho</i>
            I,
            <i>Sal</i>
            I). The replacement has three possible combinations:
            <i>Eco</i>
            RI +
            <i>Hin</i>
            dIII,
            <i>Eco</i>
            RI +
            <i>Xho</i>
            I,
            <i>Hin</i>
            dIII +
            <i>Xho</i>
            I (left arm is left of + sign).
          </td>
        </tr>
        <tr>
          <td rowspan="2">32</td>
          <td class="monospace">
            λ
            <i>sbh</i>
            Iλ1°
            <i>lac</i>
            5
            <i>sr</i>
            I
            <i>lacZ</i>
            &lt;mouse DNA&gt;
            <i>sr</i>
            Iλ3 KH54
            <i>nin</i>
            5
            <i>shn</i>
            dIIIλ6°
            <i>sr</i>
            Iλ4°
          </td>
        </tr>
        <tr>
          <td>
            Charon 32 clones larger fragments up to 19 kb and will grow in
            <i>
              recA
              <sup>−</sup>
            </i>
            host (the vector is
            <i>gam</i>
            <sup>+</sup>
            ). The
            <i>Eco</i>
            RI digest allows replacement of 8-19 kb (stuffer is mouse DNA), and
            the
            <i>Hin</i>
            dIII permits a 5-18 kb replacement. A 0-10 kb insertion is permitted
            at the
            <i>Sac</i>
            I site. Alternatively, digestion with
            <i>Eco</i>
            RI-
            <i>Sal</i>
            I,
            <i>Sal</i>
            I-
            <i>Xho</i>
            I,
            <i>Eco</i>
            RI-
            <i>Bam</i>
            HI permits this to be used as a replacement vector with loss of
            <i>gam</i>
            and a requirement for a
            <i>recA</i>
            <sup>+</sup>
            host (in this case, Spi-based selection possible).
          </td>
        </tr>
        <tr>
          <td rowspan="2">33</td>
          <td class="monospace">
            λ
            <i>sbh</i>
            Iλ1°
            <i>lac</i>
            5
            <i>sr</i>
            I
            <i>lacZ</i>
            &lt;mouse DNA&gt;
            <i>sr</i>
            Iλ3 ΔWL113 KH54
            <i>nin</i>
            5
            <i>shn</i>
            dIIIλ6°
            <i>sr</i>
            Iλ5°
          </td>
        </tr>
        <tr>
          <td>
            Charon 33 is identical to Charon 32 with a WL113 deletion
            corresponding to a
            <i>Sal</i>
            I-
            <i>Bam</i>
            HI removal in the right arm. This permits substitution with
            <i>Eco</i>
            RI fragments up to 20 kb and
            <i>Hin</i>
            dIII up to 19 kb. The
            <i>Sac</i>
            I insertion also possible.
          </td>
        </tr>
        <tr>
          <td rowspan="2">34, 35</td>
          <td class="monospace">
            λ
            <i>sbh</i>
            Iλ1°
            <i>lac</i>
            5
            <i>sr</i>
            I
            <i>lacZ</i>
            &lt;mcs - host DNA - mcs&gt;
            <i>sr</i>
            Iλ3 ΔWL113 KH54
            <i>nin</i>
            5
            <i>shn</i>
            dIIIλ6°
            <i>sr</i>
            Iλ5°
          </td>
        </tr>
        <tr>
          <td>
            Charon 34 and 35 changes the mouse DNA of Charon 32,33 to host DNA
            flanked by multiple cloning sites (6 restriction choices). Only the
            stuffer differs between 34 and 35.
          </td>
        </tr>
        <tr>
          <td rowspan="2">40</td>
          <td class="monospace">
            λ
            <i>sbh</i>
            Iλ1°
            <i>sap</i>
            Iλ1°
            <i>sk</i>
            Iλ1°
            <i>sk</i>
            Iλ2°
            <i>ssm</i>
            Iλ1 &lt;mcs -- polystuffer -- mcs&gt;
            <i>ssl</i>
            Iλ2 ΔWL113 KH54
            <i>ssm</i>
            Iλ3°
            <i>nin</i>
            5
            <i>shn</i>
            dIIIλ6°
            <i>sr</i>
            Iλ5°
          </td>
        </tr>
        <tr>
          <td>
            Each arm contains same polylinker inverted with more than a dozen
            restriction sites. The stuffer itself contains multiple
            <i>Nae</i>
            I sites to degrade it to small fragments, making removal of the
            stuffer easier (PEG pptn). The vector contains the
            <i>gam</i>
            gene and can be used on
            <i>recA</i>
            <sup>−</sup>
            hosts.
          </td>
        </tr>
        <tr>
          <td rowspan="6">EMBL</td>
          <td rowspan="2">3</td>
          <td class="monospace">
            <i>sbh</i>
            Iλ1°
            <i>b</i>
            189 &lt;mcs
            <i>int</i>
            29
            <i>nin</i>
            L44
            <i>trpE</i>
            mcs&gt; KH54
            <i>chi</i>
            C
            <i>sr</i>
            Iλ4°
            <i>nin</i>
            5
            <i>sr</i>
            Iλ5°
          </td>
        </tr>
        <tr>
          <td>
            Each arm contains same polylinker inverted with more than a dozen
            restriction sites. The stuffer itself contains multiple
            <i>Nae</i>
            I sites to degrade it to small fragments, making removal of the
            stuffer easier (PEG pptn). The vector contains the
            <i>gam</i>
            gene and can be used on
            <i>recA</i>
            <sup>−</sup>
            hosts.
          </td>
        </tr>
        <tr>
          <td rowspan="2">4</td>
          <td class="monospace">same genotype as EMBL3</td>
        </tr>
        <tr>
          <td>
            only difference with EMBL3 is that multiple cloning site is reversed
            in order:
            <span class="tt">
              <i>Eco</i>
              RI-
              <i>Bam</i>
              HI-
              <i>Sal</i>
              I-stuffer-
              <i>Sal</i>
              I-
              <i>Bam</i>
              HI-
              <i>Eco</i>
              RI
            </span>
          </td>
        </tr>
        <tr>
          <td rowspan="2">3A</td>
          <td class="monospace">
            λ(
            <i>A</i>
            am32
            <i>B</i>
            am1) + genotype of EMBL3
          </td>
        </tr>
        <tr><td>polylinker same as EMBL3.</td></tr>
        <tr>
          <td rowspan="2" colspan="2">λ2001</td>
          <td class="monospace">
            <i>sbh</i>
            Iλ1°
            <i>b</i>
            189 &lt;mcs--
            <i>sr</i>
            Iλ4°
            <i>nin</i>
            L44
            <i>bio</i>
            --mcs&gt; KH54
            <i>chi</i>
            C
            <i>sr</i>
            Iλ4°
            <i>nin</i>
            5
            <i>shn</i>
            dIIIλ6°
            <i>sr</i>
            Iλ5°
          </td>
        </tr>
        <tr>
          <td>
            Substitution inserts of 10-22 kb can be used for this vector. The
            vector is
            <i>gam</i>
            <sup>+</sup>
            but recombinants are
            <i>
              red
              <sup>−</sup>
              gam
              <sup>−</sup>
            </i>
            and Spi can be used for P2 lysogenic hosts (requires
            <i>recA</i>
            <sup>+</sup>
            hosts).
          </td>
        </tr>
      </tbody>
    </table>

    <h4>λgt10 and λgt11</h4>
    <p>
      λgt10 is a vector for cloning eukaryotic cDNA libraries and λgt11 designed
      for the expression of cDNA. These require infection in the
      <i>E coli</i>
      C600
      <i>hfl</i>
      <sup>−</sup>
      (
      <i>h</i>
      igh
      <i>f</i>
      requency of
      <i>l</i>
      ysogeny) mutant. Lambda has a repressor (
      <i>c</i>
      I) gene that shifts the phage toward lysogeny rather than lysis. This is
      seen as a reduced burst size (minute plaques). With λgt10, the presence of
      plaques indicates successful cloning, and the
      <i>c</i>
      I gene is disrupted by the DNA insert. With λgt11, the
      <i>hfl</i>
      mutant facilitates lysogenization for phage expressing fusion proteins. As
      a control, phage plated on non-
      <i>hfl</i>
      <sup>−</sup>
      mutants will not show plaques whether there are inserts or not. λgt11
      contains a β-galactosidase (
      <i>lacZ</i>
      ) carboxyl end at the site of the insertion, and generates a β-gal-cDNA
      fusion protein; expression is induced using
      isopropyl-β-D-thiogalactosylpyranoside (IPTG), and detection is with
      antibodies, or by detecting failure to turn
      5-bromo-4-chloro-3-indolyl-β-D-galactoside (Xgal) blue (clear plaques vs.
      blue plaques), since
      <i>lacZ</i>
      is inactivated by insertion. Note that the
      <i>E. coli</i>
      Y1090 host is used with λgt11.
    </p>
    <p>
      λ lysogens can be identified by spotting phage of the same immunity on a
      lawn of the test strain. Because of the repressor expressed by the
      prophage in the test strain, no plaques will form. Plating a phage with
      different immunity and seeing plaques shows that the strain is a true
      lysogen, since a failure to grow plaques with different immunity indicates
      that the phage has an adsorption/infection problem instead. P2 lysogens
      can inhibit growth of λ phage having the
      <i>red</i>
      and
      <i>gam</i>
      genes.
    </p>
    <h2>M13 Phage</h2>
    <p>
      This filamentous phage has a wild type DNA genome 6407 bp long, enclosed
      as a circular single-stranded molecule in the capsid. It specifically
      infects the male form of
      <i>E. coli</i>
      . Once in the host if forms a double-stranded circular replicative
      molecule.

      <!--         Eukaryotic Cell Biology              -->
    </p>
    <h1><a id="eukaryotes">Cell Biology of Eukaryotes</a></h1>

    <h2>Eukaryotic Genes: Structure, Organization</h2>
    <p>
      The DNA in eukaryotes is typically organized into multiple linear
      chromosomes, which become visible in the microscope when condensed in
      metaphase and stained properly. DNA is replicated in regions. When not in
      mitotsis (i.e., when in interphase), DNA is present as euchromatin, where
      gene expression is active. When not active, it is present as
      heterochromatin.
    </p>
    <p>
      Certain elements in the genome exist that may have contributed to its
      formation or might even be harmful. Transposons are integrated,
      transcribed, replicated, and reintegrated into multiple sites. Pseudogenes
      are likely vestiges of active genes losing function. Micro- and
      minisatellites are very short and somewhat longer sequences, respectively,
      found in numerous copies, very poorly conserved, and result from
      uncorrected transcription errors. These parts of the genome appear to
      serve no purpose and are called
      <q>selfish DNA</q>
      because they preserve themselves with no benefit.
    </p>
    <p>
      Genes are interrupted by non-coding sequences, introns, that are
      transcribed and eliminated during processing. Pre-mRNA is spliced by a
      spliceosome which is a ribonucleoprotein complex. Some proteins of the
      splicesoome remain bound to the first 20-24 nt of the processed mRNA for
      transport out to the cytoplasm. The spliceosome recognizes
      5′-CAGGU[A|G]AGU[A|U]GGG-3′ in the upstream exon and 5′-CAGG-3′ in the
      downstream exon.
    </p>
    <p>
      mRNA quality control is checked by a number of mechanisms. A translation
      of mRNA is done in the nucleus to control. Any stop codon followed by an
      intron at a distance &lt;s 50 nt is considered non-functional and
      destroyed by
      <b>nonsense-mediated decay</b>
      (NMD). Very long introns actually contain functional genes. Many upstream
      introns contain sites for binding transcription factors.
    </p>
    <p>
      The regulatory sequence that controls transcription and is just upstream
      of the transcription start site is the promoter. They are not usuallly
      longer than 150-200 nt. Upstream of promoters and present at variable
      sites ranging from a few hundred to 10,000 bp upstream are called
      <b>enhancers</b>
      . Enhancers do not increase transcription rate but instead the probability
      of occurrence: they do it by increasing the frequency by which a
      transcription complex is active.
    </p>
    <p>
      <b>Locus control regions</b>
      (LCRs) are found for a certain number of genes and as much as 30-100 kb
      upstream. These LCRs are at the border of different gene groups and
      contain enhancer and insulator elements. Insulators are a kind of silencer
      that inhibits enhancer activity on neighborning promoters. A function of
      LCR may be to keep chromatin in the active gene state. In particular LCRs
      have been shown to have a role in maintaining active chromatin for certain
      cell types during fetal differentiation.
    </p>
    <div style="margin: 1em -10%; text-align: justify; width: 45em">
      <img
        src="imagesBiologyHTML/major-gene-structural-elements.png"
        style="width: 100%; margin: 0"
        alt="structural elements of the eukaryotic gene"
      />
      <p
        style="
          color: navy;
          font:
            normal 90% &quot;Times New Roman&quot;,
            &quot;Times Roman&quot;,
            Times;
          margin-top: 0;
        "
      >
        Gene expression is controlled by sequences located upstream of the
        transcribed region. Promoters participate directly in the formation of
        the preinitiation transcription complex. Enhancers increase the
        frequency of promoter action. Distal regions, MAR (matrix attached
        region), chromatin openers and insulators maintain an open chromatin
        configuration and prevent gene silencing by the surrounding chromatin.
        [from L-M Houdebine,
        <i>Animal Transgenesis and Cloning</i>
        , (Wiley: Hoboken NJ), 2003]
      </p>
    </div>
    <p>
      The mature mRNA is translocated to the cytoplasm. Its 5′-UTR can be highly
      structured, GC-rich, and neither promote or inhibit translation. Ribosomal
      scanning is slowed by 2° structure in the 5′-UTR. In contrast an AU-rich
      5′-UTR will be less difficult. The 5′-UTR is important in
      posttanscriptional regulation. 3′-UTRs contain sequences that stabilize
      the mRNA and which may also direct it to the site where it is to be
      translocated. These features are important in transgenesis.

      <!--              Translation                     -->
    </p>
    <h2><a id="eukxlat">Eukaryotic Translation</a></h2>
    <p>
      Translation differs in eukaryotes in several respects, although there are
      many common features. Translation is done by ribosomes composed of
      multiple protein subunits and ribosomal RNA (rRNA), although the sizes of
      the rRNA polymers and proteins are different. Transfer RNA (tRNA) is
      charged with amino acids as in prokaryotes. That covers the similarities.
    </p>
    <p>
      Three phases mark translation of messenger RNA (mRNA) into a single
      protein polypeptide: initiation, elongation, and termination. mRNA is
      threaded through the ribosome at the 5'-end. The 5'-end undergoes a
      <i>capping</i>
      process as part of the post-transcriptional processing (discussed
      elsewhere).
    </p>
    <p>
      The ribosome is assembled having a size of 80S at initiation from two
      subunits of sizes 40S and 60S subunits, which are present in the cytosol
      and assembled on to the mRNA in stages. The 60S (2.82 MDa) is composed of
      three rRNA polymers of size 5S (120 nt), 5.8S (160 nt), and 28S (4718 nt);
      it is assembled using a scaffold of 49 proteins, names L1 to L49. (`L' is
      for `large,' referring to the large 60S subunit.) The 40S (1.4 MDa)
      subunit is assembled from 33 proteins (S1 to S33) around a single rRNA
      polymer of size 18S (1874 nt). By mass the 80S ribsome is 40% protein
    </p>
    <p>
      Acylation of the tRNA with the amino acid first begins with the
      esterification of the α-carboxyl with ATP, forming a phosphoanhydride bond
      with the α-phosphate, the products being AMP-amino acid and pyrophosphate
      (PP
      <sub>i</sub>
      ). The 3'-end of tRNA is composed of a constant trimer 5'-CCA-3'. The
      α-carboxyl of the amino acid transesterifies with the adenyl 2'-hydroxyl
      of the tRNA, releasing AMP and forming 2'-aminoacyl-tRNA, the fully
      <q>charged</q>
      tRNA.
    </p>
    <p>
      The ribosome is assembled on the mRNA strand at the capped 5'-end, and
      moves to the first in-frame AUG, found in a Kozak consensus sequence (A/G-
      <b>C-C</b>
      -A/G-
      <b>C-C-A-U-G-</b>
      G/A

      <!--                Plant Biology                 -->
    </p>
    <h2><a id="plantbio">Plant Biology</a></h2>
    <p>Notes from Botany 227 taught by John Harada in Spring 1988.</p>
    <p>
      Lec #1 (5 April 1988)
      <br />
      Relevant references:
    </p>
    <ul>
      <li>
        Goldberg et al. (1978)
        <i>Cell</i>
        <b>14:</b>
        123-131.
        <span class="tt">
          Sequence complexity of nuclear and polysomal RNA in leaves of the
          tobacco plant
        </span>
      </li>
      <li>
        Kamalay &amp; Goldberg (1980)
        <i>Cell</i>
        <b>19:</b>
        935-946. Regulation of structural gene expression in tobacco.
      </li>
    </ul>
    <p>
      <i>Gene expression</i>
      can be defined as the phenotypic (or protein) presence conveying the
      character of an organism. RNA and protein are expression products of
      genes. The processes that regulate this expression, of transcription and
      translation can be included within the definition.
    </p>
    <p>
      Cell structure and function is based upon the expression of different
      populations (subsets) of genes.
    </p>
    <p>What is the evidence of sporophytic DNA contents being the same?</p>
    <ol>
      <li>
        <b>totipotency</b>
        : differentiated tissue can be cultured in suspension and a whole plant
        can be grown from it
      </li>
      <li>
        <b>reassociation kinetics</b>
        :
        <i>C</i>
        <sub>0</sub>
        <i>t</i>
        curves for DNA isolated from two different tissue types will show
        identical patterns of reassocation
      </li>
      <li>
        cDNA probe corresponding to an embryonic protein will hybridize to DNA
        taken from different tissues, such as leaf and embryo tissue
      </li>
    </ol>

    <!--                Yeast Biology                 -->
    <h2><a id="yeast">Yeast Biology</a></h2>
    <p>
      <i>Saccharomyces cerevisiae</i>
      (baker's yeast) is useful in studying eukaryotic biology. These cells can
      divide by budding, which is not a true mitosis: chromosomes attach to a
      <b>spindle pole body</b>
      which replicates when the chromosomes (12-18 in number, depending on yeast
      species) do. When nutrient conditions are adverse, the yeast can
      <i>sporulate</i>
      , which involves a meiotic division to form four haploid spores. When
      conditions are better, two spores can fuse to form a diploid yeast, or a
      single spore can be propagated to form a haploid colony. Fusion of haploid
      spores is dependent upon them having distinguishing
      <b>mating types</b>
      .

      <!--            Eukaryotic Viruses                  -->
    </p>
    <h1><a id="eukvirus">Viruses of Eukaryotic Cells</a></h1>

    <h2>Baculovirus</h2>
    <p>
      (from M. D. Summers and G. E. Smith,
      <i>
        A Manual of Methods for Baculovirus Vectors and Insect Cell Culture
        Procedures
      </i>
      , Texas Agric. Exp. Station, 1987, 57 pp.)
    </p>
    <p>
      <i>Autographa californica</i>
      nuclear polyhedrosis virus (AcMNPV) is prototype of family Baculoviridae.
      Two forms of virus emerge after infection: extracellular (ECV) and
      occluded virus particles (OV).
    </p>
    <p>
      The 29,000 dalton polyhedrin protein makes up most (50-75%) of the total
      <q>stainable</q>
      protein in infected cells (1 mg/ml in 1-2 million cells). The viral
      occlusions are important to the viral life cycle, being necessary for
      horizontal transmission. Infected larvae die and millions of polyhedra can
      be found in the decomposing tissue. ECV forms are generally inactivated by
      environmental factors while the OV forms are ingested and the alkaline
      environment of the insect gut releases them. Once the organism is infected
      the ECV form serves to infect secondarily the other tissues.
    </p>
    <p>
      Viral particles enter the midgut cells of
      <i>Spodoptera frugiperda</i>
      or
      <i>Trichoplusia ni</i>
      by endocytosis or fusion and viral DNA is uncoated in the nucleus. DNA
      replication occurs 6 hr later. After 10 hr, ECV are released by budding.
      Occlusions are detected after 24 hr. After 36-48 hr, ECV are at a maximum
      of an avg. 8 × 10
      <sup>6</sup>
      pfu/ml.
    </p>
    <h3>Selection of Transfer Vector</h3>
    <p>
      Insertion of the gene of interest can be done to create a fusion although
      a mature protein (i.e., one with its own ATG start codon is preferable).
      An artificial BamHI linker has been inserted 8 bases upstream of the ATG
      site and about 50 bases downstream of the polyhedrin transcription start
      site.
    </p>
    <p>
      The lengths of the 5' and 3' non-coding sequences expressed have varied
      from 3 to 400 bases. It is recommended that the length of non-coding 5'
      leader be as short as possible as a rule. All transfer vectors constructed
      so far use the polyhedrin gene's own polyadenylation site. Use of the
      gene's own polyadenylation site tandem to the polyhedrin poly(A) site has
      worked.
    </p>
    <p>
      Vector pAc373 has a unique BamHI site -8 bases of the ATG site. Other
      vectors have sites for insertion of open reading frames into the
      polyhedrin coding sequence. Unique BamHI or SmaI sites from position +5 to
      +175 are here. Some evidence suggests that the polyhedrin's own
      translation initiator is more efficient that the gene's.
    </p>
    <h3>Production of recombinant AcMNPV</h3>
    <p>
      10 μg of highly purified plasmid DNA should be prepared since
      <i>S. frugiperda</i>
      cells are sensitive to contaminants in crude plasmid preps. CsCl-EtBr
      gradient centrifugation is essentially the best way to prepare the
      plasmid. All transfections should include a control transfection with
      AcMNPV DNA alone. 24 hr post-transfection, these cells can be compared to
      the transfected cells receiving recombinant DNA: Sf9 viability should by
      &gt;97%.
    </p>
    <p>
      <i>S. frugiperda</i>
      cells are transfected with 1 �g AcMNPV DNA and 2 �g plasmid DNA to produce
      the recombinant virus. After 3-4 days, the 10-50% of cells show evidence
      of viral occlusions and the ECV titer is ~10
      <sup>7</sup>
      pfu/ml. An inverted phase microscope set at 250-400 × identifies the
      infected cells (plaques). Plaques can be seen by placing them inverted on
      a black background and illuminating from the side at an acute angle with a
      strong light. Plaques are first formed 2-3 days post-infection and are
      fully formed 5 days p.i. Recombinants may make 0.1% to 5% of plaques.
      Purification requires 2-3 rounds of plaque purification. After 3-4 d
      post-transfection, spin cells down, store the virus-containing supernatant
      at 4�. Serially dilute virus into fresh medium to get 2 ml of 10-3, 10-4,
      and 10-5 dilutions, and then inoculate duplicate plates with 1 ml/plate
      and overlay with agarose.
    </p>
    <h3>Detection and purification of recombinant viruses</h3>
    <p>
      Detection of recombinant viruses can usually be done by visual screening
      of plaques of occlusion-negative viruses or by filter hybridization of
      plaques. Although the first method is easy and rapid, high quality
      dissecting scopes, optimal conditions of plaque formation, high levels of
      recombination, and experience in discriminating between plaques formed
      from occlusion-negative and -positive viruses.
    </p>
    <p>
      Plaques which are selected can be picked up from the agarose overlay with
      a Pasteur pipet and the agarose plug transferred to 1ml medium. 10
      <sup>4</sup>
      pfu can be recovered from a single plaque. Dilute to 10 and 100 pfu/ml,
      then inoculate duplicate plates with 1 ml of each dilution and overlay
      again with agarose. After another 4-5 d, plaques are visible and a high %
      are recombinant, occlusion-negative. Compare their appearance under the
      scope to agarose overlays infected with the wild-type virus to become
      familiar with occlusions. A purified plaque should be several millimeters
      from a wild-type plaque. A 100-200 µl pipet can be used to get an agarose
      plug which can be put in 5 ml medium containing 10
      <sup>6</sup>
      cells in a 25 cm
      <sup>2</sup>
      flask. After 3-4 d, a majority of the cells should be infected and titer
      should be 10
      <sup>8</sup>
      pfu/ml. 1 ml of the 5 ml of virus (infected cell supernatant) can by
      stored in 1.5 mL screw-top cryostat tubes for long-term. The other 4 ml
      can be stored at 4� and can be marked passage 1 virus. Southern
      hybridization analysis should be done to verify the presence of the
      foreign gene in its proper location in the viral genome. Sometimes false
      occlusion-negative plaques appear and the recombinant virus should be
      checked for presence and location of inserted genes. Total cell DNA from
      the 5 ml of infected cells contains sufficient viral DNA for blot
      analysis.
    </p>
    <p>
      Production of virus done by preparing the desired volume of cells to 10
      <sup>6</sup>
      cells/ml. In a 100 mL spinner flask, grow cells in 50 ml medium to 2 × 10
      <sup>6</sup>
      cells/ml. Pellet 1000 g for 10 min and resuspend in 19 ml medium + 1ml
      virus in which MOI is 0.1 to 1.0 pfu/cell (therefore, 2 × 10
      <sup>5</sup>
      to 2 × 10
      <sup>6</sup>
      pfu are needed). Incubate the 20 ml for 1 hr at room temperature, then add
      80 ml medium and place either in two 100 ml flasks or one 250 mL flask.
      After 2 days, the titer is ~2-3 × 10
      <sup>8</sup>
      pfu/ml. 2 × 10
      <sup>7</sup>
      cells can be placed in a 150 cm
      <sup>2</sup>
      flask and virus infected at low MOI. Virus passaged 6 times has shown no
      drop in protein expression. Loss of recombinant protein can be minimized
      by limiting passage and infecting at low MOI.
    </p>
    <h3>Methods</h3>
    <p>In general</p>
    <ol>
      <li>
        Fresh cell culture medium should be equilibrated to room temp before
        use.
      </li>
      <li>
        Do not pipet fresh medium into a flask which disturbs the cell
        monolayer. Pipet to a side away from the monolayer.
      </li>
      <li>
        Addition of liquid to a plate should be done dropwise at the center of
        the plate. Liquid flows toward the edges. An exception is with agarose,
        which should be added to the edge and allowed to flow to all edges.
      </li>
      <li>
        Dislodging cells from a monolayer is best accomplished by rapidly
        streaming media out of the pipet. Do not foam the cells and do not rap
        the flask to dislodge cells.
      </li>
      <li>
        Flasks or plates used in the infection process should be rocked in all
        directions by hand to make sure liquid reaches all areas of the growing
        surface.
      </li>
      <li>
        Note that some strains of insect cells are sensitive to centrifugal
        forces.
      </li>
    </ol>
    <p>
      Sf9 cells have a doubling time in TNM-FH medium containing 10% FBS of
      18-24 hours and should be subcultured three time per week. They grow in a
      temperature range of 25-30° and do not require CO
      <sub>2</sub>
      or other rigid culture conditions. Antibiotics are not routinely used.
    </p>
    <p>
      Subculturing is done by resuspending nearly confluent monolayers and
      recovering 0.5 to 1.0 ml of culture containing 2-2.5 × 10
      <sup>6</sup>
      cells to a new 25 cm
      <sup>2</sup>
      flask containing TNM-FH medium plus supplements.
    </p>
    <p>
      TNM-FH and Grace's medium do not contain pH indicators. When antibiotics
      are required, 50 �g/ml gentamycin and 2.5 µg/ml amphotericin B (Fungizone)
      are used. Healthy cultures double in 18-24 hours.
    </p>
    <p>
      Suspension cultures should be seeded to an initial density of 0.5 to 1.0 ×
      10
      <sup>6</sup>
      cells/ml. Constant stirring at 50-60 rpm is usual. Subculturing 2-3
      times/week is necessary, when cells reach 2-2.5 million/ml. 80% of the
      suspension medium is removed and replaced with fresh medium. About once
      every 2 weeks the suspension should be pelleted and the total medium
      removed as it accumulates waste products.
    </p>
    <table style="font-family: &quot;Courier New&quot;, Courier, monospace">
      <caption>The Genetic Code</caption>
      <tbody>
        <tr>
          <th style="background: none" colspan="2"></th>
          <th colspan="8">Second Nucleotide</th>
        </tr>
        <tr>
          <th style="background: none" colspan="2"></th>
          <th colspan="2">U</th>
          <th colspan="2">C</th>
          <th colspan="2">A</th>
          <th colspan="2">G</th>
        </tr>
        <tr>
          <th rowspan="16">
            F
            <br />
            i
            <br />
            r
            <br />
            s
            <br />
            t
            <br />
            <br />
            N
            <br />
            u
            <br />
            c
            <br />
            l
            <br />
            e
            <br />
            o
            <br />
            t
            <br />
            i
            <br />
            d
            <br />
            e
          </th>
          <th rowspan="4">U</th>
          <td>UUU</td>
          <td>Phe</td>
          <td>UCU</td>
          <td>Ser</td>
          <td>UAU</td>
          <td>Tyr</td>
          <td>UGU</td>
          <td>Cys</td>
        </tr>
        <tr>
          <td>UUC</td>
          <td>Phe</td>
          <td>UCC</td>
          <td>Ser</td>
          <td>UAC</td>
          <td>Tyr</td>
          <td>UGC</td>
          <td>Cys</td>
        </tr>
        <tr>
          <td>UUA</td>
          <td>Leu</td>
          <td>UCA</td>
          <td>Ser</td>
          <td>UAA</td>
          <td>ochre</td>
          <td>UGA</td>
          <td>opal</td>
        </tr>
        <tr>
          <td>UUG</td>
          <td>Leu</td>
          <td>UCG</td>
          <td>Ser</td>
          <td>UAG</td>
          <td>amber</td>
          <td>UGG</td>
          <td>Trp</td>
        </tr>
        <tr>
          <th rowspan="4">C</th>
          <td>CUU</td>
          <td>Leu</td>
          <td>CCU</td>
          <td>Pro</td>
          <td>CAU</td>
          <td>His</td>
          <td>CGU</td>
          <td>Arg</td>
        </tr>
        <tr>
          <td>CUC</td>
          <td>Leu</td>
          <td>CCC</td>
          <td>Pro</td>
          <td>CAC</td>
          <td>His</td>
          <td>CGC</td>
          <td>Arg</td>
        </tr>
        <tr>
          <td>CUA</td>
          <td>Leu</td>
          <td>CCA</td>
          <td>Pro</td>
          <td>CAA</td>
          <td>Gln</td>
          <td>CGA</td>
          <td>Arg</td>
        </tr>
        <tr>
          <td>CUG</td>
          <td>Leu</td>
          <td>CCG</td>
          <td>Pro</td>
          <td>CAG</td>
          <td>Gln</td>
          <td>CGG</td>
          <td>Arg</td>
        </tr>
        <tr>
          <th rowspan="4">A</th>
          <td>AUU</td>
          <td>Ile</td>
          <td>ACU</td>
          <td>Thr</td>
          <td>AAU</td>
          <td>Asn</td>
          <td>AGU</td>
          <td>Ser</td>
        </tr>
        <tr>
          <td>AUC</td>
          <td>Ile</td>
          <td>ACC</td>
          <td>Thr</td>
          <td>AAC</td>
          <td>Asn</td>
          <td>AGC</td>
          <td>Ser</td>
        </tr>
        <tr>
          <td>AUA</td>
          <td>Ile</td>
          <td>ACA</td>
          <td>Thr</td>
          <td>AAA</td>
          <td>Lys</td>
          <td>AGA</td>
          <td>Arg</td>
        </tr>
        <tr>
          <td><b>AUG</b></td>
          <td>Met</td>
          <td>ACG</td>
          <td>Thr</td>
          <td>AAG</td>
          <td>Lys</td>
          <td>AGG</td>
          <td>Arg</td>
        </tr>
        <tr>
          <th rowspan="4">G</th>
          <td>GUU</td>
          <td>Val</td>
          <td>GCU</td>
          <td>Ala</td>
          <td>GAU</td>
          <td>Asp</td>
          <td>GGU</td>
          <td>Gly</td>
        </tr>
        <tr>
          <td>GUC</td>
          <td>Val</td>
          <td>GCC</td>
          <td>Ala</td>
          <td>GAC</td>
          <td>Asp</td>
          <td>GGC</td>
          <td>Gly</td>
        </tr>
        <tr>
          <td>GUA</td>
          <td>Val</td>
          <td>GCA</td>
          <td>Ala</td>
          <td>GAA</td>
          <td>Glu</td>
          <td>GGA</td>
          <td>Gly</td>
        </tr>
        <tr>
          <td>GUG</td>
          <td>Val</td>
          <td>GCG</td>
          <td>Ala</td>
          <td>GAG</td>
          <td>Glu</td>
          <td>GGG</td>
          <td>Gly</td>
        </tr>
      </tbody>
    </table>

    <h1>Molecular Biology</h1>

    <h2>Gene Cloning</h2>
    <p>
      The following vectors can be used to clone DNA of any kind, gene or not,
      into bacteria.
    </p>
    <table>
      <tbody>
        <tr>
          <th>Vector</th>
          <th>Maximal Size (kbp)</th>
        </tr>
        <tr>
          <td>Plasmid</td>
          <td>20</td>
        </tr>
        <tr>
          <td>Cosmid</td>
          <td>40</td>
        </tr>
        <tr>
          <td>P1 Phage</td>
          <td>90</td>
        </tr>
        <tr>
          <td>Bacterial Artificial Chromosome (BAC)</td>
          <td>200</td>
        </tr>
        <tr>
          <td>Yeast Artificial Chromosome (YAC)</td>
          <td>1000</td>
        </tr>
      </tbody>
    </table>

    <h3>Transferring DNA Into Cells</h3>
    <p>
      DNA has to be transferred into cells when their genetic characteristics
      are to be changed.
    </p>
    <p>
      To transfer DNA into
      <b>cultured cells</b>
      , the following approaches might be used:
    </p>
    <ul>
      <li>cell fusion, rarely used but effective</li>
      <li>transfection, most common</li>
      <li>electroporation, also now common</li>
    </ul>
    <p>When done for gene therapy:</p>
    <ul>
      <li>vectors with specific ligands</li>
      <li>viral vectors</li>
    </ul>
    When done to achieve transgenesis:
    <ul>
      <li>viral vectors (as for gene therapy)</li>
      <li>DNA microinjection into zygotes or embryonic cells</li>
    </ul>
    <p>
      The chemical transfection method often uses calcium chloride. This causes
      DNA to precipitate with the excess calcium, and a microprecipitate gets
      endocytosed into the cell. Most might be degraded, but some becomes stably
      incorporated in the cell. Use of DMSO or glycerol may aid endocytosis; the
      latter reduce cell content in water and promote endocytosis. DNA may also
      be grouped with polycations (basic proteins, polyethyleneimine) which are
      themselves linked to lipids, which also induces endocytosis.
    </p>
    <p>
      <b>Electroporation</b>
      subjects cells to alternating electric field, causing transient porosity
      through which the DNA can pass. This method is most suitable for
      transferring genes into ES cells and to replace endogenous gene by
      homologous recombination.
    </p>
    <p>
      <b>Viral vectors</b>
      work on the principle that they infect the target cell easily getting the
      DNA through the membrane barrier. The DNA must be contained by the viral
      capsid. The viral genome must be modified to prevent normal pathogenesis.
      The parts of the viral genome responsible for pathogenesis are removed.
      Parts that produce particles containing foreign genes to be transduced are
      retained. Adenovirus is used as a vector because it infects cell in vitro
      or in vivo; it does not stably transfect cells however, rarely integrating
      into the genome. Retroviruses will essentially infect cultured cells and
      integrate into the genome. Both these are used for gene therapy, and for
      transferring genes into cell types for which no other has proved
      satisfactory. Adenovirus vectors are being used to transfer genes into
      animal organs in order to evaluate gene effects, probably a prelude or
      substitute to transgenesis.
    </p>
    <p>
      <b>DNA Microinjection</b>
      . DNA in solution is directly injected into cell cytoplasm or nucleus
      using advanced instruments (microscope and microinjector). Generally it is
      preferred to inject into nucleus. For cloning genes totally unknown, the
      hypervariable regions of the genome are used, typically microsatellites,
      which often have sequences like GT repeats 12-22 times serially. Known
      microsatellites can be analyzed with PCR to check for their diversity.
      This diversity occurs without selection pressure. A relationship of a
      microsatellite to a gen can be established when the distance between
      microsatellites is 10 centimorgans or more (10 cM is about 1 million kb).
    </p>
    <p>
      Once a gene is identified as having a major role in gene expression,
      alleles might be sequenced to understand their role. Testing of gene
      function can be done in bacteria to produce protein to study its
      properties, biochemically and structurally (crystals for XRD). Most genes
      in a genome can be identified by the EST library.
    </p>
    <div style="margin: 1em -5%; text-align: justify; width: 45em">
      <img
        src="imagesBiologyHTML/systematic-gene-study.png"
        style="width: 100%; margin: 0"
        alt="structural elements of the eukaryotic gene"
      />
      <p
        style="
          color: navy;
          font:
            normal 90% &quot;Times New Roman&quot;,
            &quot;Times Roman&quot;,
            Times;
          margin-top: 0;
        "
      >
        The classical method for gene cloning (1) is now followed by positional
        cloning based on the presence of microsatellites in the vicinity of the
        genes (2). Systematic sequencing of EST (expressed sequence tag) and
        genomes will eventually lead to the identification of all of the genes
        of a few living organisms (3). The study of gene function and regulation
        often includes transgenesis [from figure 1.6 in Houdebine]
      </p>
    </div>
  </body>
</html>
