<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01//EN"
   "http://www.w3.org/TR/html4/strict.dtd">
<html lang="en">
<head>
<title>Waters-Micromass M@LDI-ToF Use</title>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8">
<link href="./stdcss/std.css" rel="stylesheet">
<style>
 .result {color:green;border:1px solid maroon;background:none;}
</style>
<script src="/include/debug.js"></script>
<script src="/include/css.js"></script>
<script src="/include/dom-xml.js"></script>
<script src="/include/colorContrasting.js"></script>
<script src="/include/htmlTOCgenerator.js"></script>
<script src="/include/logos.js"></script>
<script src="../../proteomicsDocs.js"></script>
</head>
<body onload="initProteomicsDoc();">

<p id="title">
Waters-Micromass M@LDI-ToF Use

<hr>

<div id="first-content-block">
The major source for this text is


<!-- FEATURES -->


<h1>Features of the Waters-Micromass M@LDI-ToF</h1>
<p>
The instrument takes two kinds of target plates:
<ol>
<li>96 well targets with 3.0 mm dia, pitched at 4.5 mm, in a 12 &times;
8 format, that include 24 near-point lockmass (calibration) wells
<li>384 well targets with 1.5 mm dia, pitched at 2.25 mm, with 24
lockmass wells
</ol>
<p>
The X-Y stepper motor has a 4-micrometer resolution for finding spots
on the target.
<p>
The reflectron optics provides an effective path of 2.3 meter.  In addition
the source used delayed extraction to avoid effects that reduce resolution
and give inaccurate masses.
<p>
<b>Detector</b>.  A one-second span allows for the sampling of 2 billion
different masses within the clock interval of the detector.
Moreover the signal can be gated to reject low mass matrix ions.
<p>
<b>Laser</b>.  A nitrogen UV laser at &lambda;=337 nm.  Possesses a motorized
iris for automated laser energy control
<p>
In <b>Reflectron</b> mode, the MALDI-ToF can serve in automated
high throughput peptide mass fingerprinting to be used in database searching.
<p>
In <b>Linear</b> mode, the MALDI-ToF optimized to identify bacteria,
intact protein, and analysis of synthetic polymers.  The effective path
length is 0.7 meters, with a delayed extraction source.

<h2>Inlet System</h2>
<p>The inlet system is the platform that the target plate sits on and the
vacuum chambers.

<h3>Loading &amp; Unloading of Target Plates</h3>
<p>To <b>load</b> a target plate:
<ol>
<li>Open the cover to the target plate loading platform on the M@LDI.
It should come off without force.  If it does not,
<a href="#opentunepage">open the Tune Page</a> and check the status
bar at the bottom to see if the probe is <b>Unloaded</b>.  (If it is
not unloaded, the status bar will say <b>Loaded</b> and the toggle
button will say <b>Unload Probe</b>.  If so, press the button to unload
the probe.)
<li>Find well <b>A1</b> on the prepared target plate, and place it in the
plate mold on the M@LDI
</ol>

<h3>Sample Spotting On To Target Plates</h3>
<ol>
<li>Plates should be clean and wells chosen for use free of defect for
the best results.  For the wells chosen for use, perform a pre-spotting
cleaning as follows:
 <ol>
 <li>Place 3-4 &micro;l water on the well; the drop will appear nearly
spherical if done right.
 <li>Leave at least 15-30 seconds, so that any material on the surface
is adsorbed into the water.  Continuing to spot other wells with water
usually provides that time.
 <li>Use a P-200 set at high volume to remove the water drops from each
well.
 <li>Dry the well with 1.0-1.5 &micro;l acetontrile.
 <li>Repeat the water adsorption/cleaning cycle if the well seems to dirty
or has a high background.  A high background is immediately evident by
moving the laser around a clean well with laser energy set somewhat high.
Also spot matrix alone on a well to see if the background is coming from
the laser.
 </ol>
<li>Select the correct matrix for the peptide or whole protein analyte.
Generally CHCA is best for peptides (polypeptides &lt; 10 kDa) and
sinapinic acid best for larger molecules (polypeptides &gt; 10 kDa).
However there is no one matrix for all classes of proteins/peptides,
and other matrices may be more suitable.  Select also the correct
stoichiometry for the matrix.  A general rule is that for every amino
acid, there should be at least 100-1000 matrix molecules;  another rule
is 10,000 matrix molecules per peptide molecule.  The final mixing
of the matrix with peptide should be as dilute as possible, since the
peptide(s) may not be soluble in material that matrix is soluble.
<li>Spot the plate with 1.0 &micro;l of matrix/peptide solution.
As the spot dries, it should leave an even coat throughout the well,
and not a single spot on one portion of the well.  A 50% acetonitrile
with 0.1% TFA (in water) usually works well.  Try putting the plate on
a level warmed surface if spotting with some solvents is a problem.
</ol>

</div>

<h3>Selecting Sample Wells</h3>
<p>Wells are selected depending on the signal acquisition mode.  When
the laser is turned on in non-storage acquisition mode (i.e., when the
toolbar button with the laser symbol is pressed and the signal is
seen only on the Tune Page), then the well is selected by entering the
position as a row-column format in the <b>Sample Well</b> text box near
the laser energy controls.  For a typical 96-well plate, there are 12
columns numbered 1-12, and 8 rows designated A-H.  Enter &ldquo;D4&rdquo;
to select row D, column 4.  To select a lockmass well that corresponds
to the set of four wells that surround it, check the lockmass well checkbox.
<p>
For automated storage acquisition mode, sample wells are selected during
creation of the sample list in the Mass Lynx main page.  See this
<a href="#">section</a> for creating sample lists.

<h3>Moving Within One Sample Well</h3>
<p>The laser does not aim at an entire well, but only a portion of it.
In automated data acquisition mode when the run is being controlled
by parameters in the sample list, the operator has no direct manual control
over the laser, but sets parameters by which the laser sets its aim
and the minimum and maximum number of attempts it makes to acquire
useful spectra.
<p>
In manual mode, the user directs the aim of the laser within the well
with the image of a well shown as a yellow circle and moving red colored
cross hairs within the well.
<ol>
<li>&nbsp;
</ol>

<h3>Spatial Calibration (ALPS)</h3>
<p>The M@LDI-ToF has a special system for determing the location of wells
in the source called the Automatic Laser Positiong System (ALPS).  In
older systems, a manual system of positioning was used which required
a camera inside the source (the camera is an option that can be used
for the M@LDI).  With ALPS, the laser is used to locate two reference
holes in the target plate.  A photosensor on the other side of the plate
signals when the laser is over the hole.
<p>
ALPS can be set to work automatically or manually.  The automatic setting
works only when the probe plate is loaded.  To activate the manual mode
for spatial calibration:
<ol>
<li>Select <b>Options-&gt;Plate Spatial Calibration</b> to invoke the
Spatial Calibration dialog.
<li>The various parameters can be set:
 <ul>
 <li>The <b>Search Step (mm)</b> is the distance the probe stage moves
from location to location during a &ldquo;rastering&rdquo; process to
locate the reference holes on the target plate. A good default value is
<tt>0.2</tt>.
 <li>The <b>Delay Per Row (ms)</b> is set to <tt>0</tt>.
 <li>The <b>Search Window (rows x cols)</b> is the number of steps taken
for each axis of the search.  Because there are two axes (up and down,
or <i>x</i> and <i>y</i>, the number of points searched is the square
of the setting.  The usual setting is <tt>10</tt>.  The
Search Window is combined with the Search Step to determine the bounding
box and grid search for locating the reference hole.  For example, with
a Search Window of 10 and Search Step of 0.2, 10<sup>2</sup> or 100 points
will be searched with each step 0.2 mm away from each other.  Thus a
grid of 2 &times; 2 mm will be searched ([0.2 &times; 10] &times; [0.2 &times;
10]).
 </ul>
<li>If <b>Use Initial Calibration Conditions</b> option is checked, the
parameters are cleared (grayed out)
<li>The <b>Start</b> button must be clicked for manual calibration to be
activated.
</ol>
<p>
The automatic spatial calibration activates ALPS itself.  Unless there
is a good reason to use manual spatial calibration, use ALPS instead.  To
activate ALPS:
<ol>
<li><b>Options-&gt;Target Plate</b> to get the Target Plate dialog.
<li>Select the <i>Spatial Calibration</i> tab within the dialog.
There are two groupings of controls.  One is the Spatial Calibration
Initial Conditions and the other is the Current Calibration.  The
Current Calibration has no controls to set, but merely reports the actual
positions of the last target plate loaded.
<li>The option <b>Calibrate on Probe Load</b>, if checked, uses ALPS every
time the plate is loaded on to the probe.  Use of ALPS every time is
not necessary for the standard 96-well target plate, for the settings
for searching are usually sufficient for locating the reference holes.
For higher density plates (384-well or more), it is better to check
this setting.
<li>The X,Y settings refer to the location of the reference holes from the
center of the plate (the center of the plate is thus X=0, Y=0).  When using
different or new plates, use the back of a plate to draw centered lines
and then use a millimeter-measure ruler to locate the reference holes.
<li>The <b>Hysteresis</b> is used to determine a parameter in the searching
pattern that refers to cyclic or backward movements of the motor.  It
should be set to <tt>0.2</tt> mm.
<li>The other parameters (Search Window, Search Step, Search Delay) are
the same as described above for manual calibration.  Their settings are
the defaults mentioned in that section.  Change them if there is a good
reason to do so (such as on the advice of an engineer).
</ol>
<p>
It is sometimes useful to check the values in the Current Calibration
group of controls.  They report the actual specifications of the target
plate as reported by ALPS.  If the target plate is warped (either because
of bending or poor manufacture) this can seriously affect mass measurement.


<h1>Installation</h1>
<p>
The following are some specifics regarding the installation of software
and configuration of the system.

<h2>Networking</h2>
<p>
MassLynx 4.0 is installed normally with its options unlocked using key
files.
<p>
The MALDI "embedded"

<h2>Engineering Parameters</h2>
<p>
The M@LDI-LR at AUBI has certain parameters set by the engineers at
installation.  These parameters can become changed or altered if the
software is re-installed, and the current parameters should be recorded
especially before attempting any software repair or re-installation.
<p>
The following parameters were recorded in the last engineer's visit:
<table>
<tr><td>Linear Field Length  <td>1.13 m
<tr><td>Reflectron Field Length <td> 1.302 m
<tr><td>Reflectron Length <td> 0.322 m
</table>
<p>
These parameters are normally adjusted by the engineer in Reflectron mode
to within 0.5 Da of the known value of ACTH 18-39 (1265.3 Da); and to
within 1 Da of the known value of cytochrome <i>c</i> (12384 Da).
<p>
If these parameters are to be re-set, make sure that spectra are
acquired with NO CALIBRATION SET!! (Select the <b>uncal.cal</b> file
through <b>Calibration-&gt;Calibrate Instrument...</b>, and then
<b>File-&gt;Open</b> to find <b>uncal.cal</b> within the Calibration
dialog.)


<h1>Operation</h1>
<h2>Setting Into Operate</h2>
<p>
The M@LDI has an important on-off switch, with off being Standby, and on
being Operate.  The reason is high voltages are present, and the operator
may have to use the Standby as a panic button to take the instrument out
of the high voltage state if arcing or other problems are present.
<p>
Conversely, high voltages will never be turned on unless the instrument
is put into Operate.  Before putting into Operate, it may be necessary to
set the voltages to zero, and to bring them up gradually to their final
voltages.
<p>
The Operate/Standby button is located in the lower right corner of the
Tune Page.  To the right of it is a square that is colored green when
the instrument is in Operate mode;  colored red if the instrument is in
Standby mode.  In addition, at the bottom edge of the page is a status
bar section that writes &ldquo;Operate&rdquo; if the instrument is in
Operate mode, &ldquo;Standby&rdquo; if in Standby.

<p>

The vacuum for the analyzer Penning should be 2e-6 mbar for operation.
It can reach 5e-7 mbar after several days.

<h2>Start Up</h2>
<p>
The voltages, particularly of the <b>Source</b>, must be
incrementally&mdash;<em>not</em> immediately or suddenly&mdash;increased
from a zero voltage state, especially if the system has been vented
for a long period.

<h3>Procedure for Increasing Voltage from Vented System</h3>
<p>
When the M@LDI is being started up from a vented state, the increases
in the voltages to their final values must be incremental with respect
to voltage and slow with respect to time.  The M@LDI
should already be in Standby mode.

<h4>MCP Conditioning</h4>
<p>
If the system has been vented for a long time, the MCP detectors
should be conditioned.  Before conditioning, make sure that the vacuum
on the system has had at least 24 hours of running within the operational
limits of the system.  That is, the system should have been pumped and
maintained at 3 &times; 10<sup>&minus;6</sup> mbar for overnight.
Attempting to condition the MCPs when there is moisture present can
damage them.
<p>
When ready for conditioning:
<ol>
<li>Go to <b>Options-&gt;Instrument Conditioning</b> to open the
<b>MCP Conditioning</b> dialog
<li>Set the Start to 0, Stop to the final voltage (usually 2000 V).
<li>Set the step at 10 minutes, and the duration for a time that runs
overnight.  If in a hurry, it can be set to 3-4 hours (180-240 mins), although
it depends on how much moisture is in the system.
</ol>

<h4><a name="maldivoltages">Increasing the Voltages</a></h4>

<ol>
<li>Open the voltages dialog:  <b>Options-&gt;Voltages...</b>.
<li>Press the <i>Voltages</i> tab, and put the <b>Source</b>
to zero (press the Enter key to make sure).
If reflectron mode is set, set the <b>Reflectron</b> to zero.  Press the
Save button on the menu toolbar (or select <b>File-&gt;Save As...</b>).
<li>Go into Operate mode.
<li>In positive mode, the increment of voltages is not likely to be
accompanied by arcing.  If in negative mode, <a href="#negmode">follow
the instructions</a> for gradually increasing voltages for the switch
to negative mode.
<li>In general, increase the voltage of the Source and Reflectron together.
The first jump might be to one-third or one-fourth of the final voltage.
This would be followed by a pause.  The length of the pause depends
on the state of the system: if it was vented for a long time, the pauses
between jumps should probably last for the entire day.  If it was out of
operate for only a short time, the pauses can be very short.
<li>In the final one-third or one-fourth jump to the final voltage,
divide the jumps into finer increments.  For example, make the jumps
in five steps (one-fifth of the difference between the final and present
voltage) rather than one step.  As an example, suppose the Source final
voltage is 15,000:  the first jump was to 5000, then either to 7500 or
10,000, and from 10,000 V the jumps were by +1000 V, from 11,000 to 12,000
to 13,000 to 14,000 and maybe to 15,000.  The increment between 13,000
and 15,000 might even be by +500 V.  Experience and good judgment will
help later.
</ol>

<h2 id="maldimode">Mode Switching</h2>
<p>
The M@LDI has essentially four modes, because detection can be either
<b>Linear</b> or <b>Reflectron</b>, and the ions selected for detection
can be either <b>Positive</b> or <b>Negative</b>.
<p>
The Reflectron mode is used to detect peptides (polypeptides &lt;
10 kDa) while the Linear mode is used for larger molecules (polypeptides
&gt; 10 kDa).
<p>
Most M@LDI work uses Positive ion selection from the source, but the
determination whether to use Positive or Negative mode depends on many
factors, such as the chemistry of the matrix and the protein/peptide.
Usually, both positive and negative ions are produced in MALDI-based
ionization of peptides/proteins, and the decision to use positive or
negative mode is one based on whether more positive or negative ions
are produced from the reaction of the matrix with the analyte.
<p>
There is usually no difficulty in mode switching.  However, the
switch from Positive to Negative mode is often accompanied by unwanted
arcing within the flight tube, which might damage components.  Thus
the switch should involve the gradual (slow and incremental) increase
in voltage.
<p>
Switching from Linear to Reflectron or Reflectron to Linear requires
attention to certain parameters.  These are NOT saved within an individual
<b>.ipr</b> (Instrument PaRameter) file.  It might be useful to have
four IPR files that reflect reflectron-positive, reflectron-negative,
linear-positive, and linear-negative.

<h3 id="negmode">Switching From Positive to Negative Mode</h3>
<p>
The switch to negative mode, whether reflectron or linear, should
also involve a slow and incremental increase in the voltage of the
<b>Source, reflectron</b> and the <b>&minus;ve Ion Mode Supply</b>.
Going into Operate immediately after a switch with full voltage values
set can cause arcing in the flight tube, which should be avoided.
<ol>
<li>Put the M@LDI into Standby mode.
<li>Switch to negative mode:  from the Tune Page, <b>Mode-&gt;LDI&minus;</b>.
<li>Open the voltages dialog:  <b>Options-&gt;Voltages...</b>
<li>Press the <i>Voltages</i> tab, and put the <b>Source</b> and
<b>&minus;ve Ion Mode Supply</b> to zero (press the Enter key to make sure).
If reflectron mode is set, set the <b>Reflectron</b> to zero.  Press the
Save button on the menu toolbar (or select <b>File-&gt;Save As...</b>).
<li>Go into Operate mode.
<li>For the next three steps, which increase the voltages to the components
of the M@LDI, be prepared to go <em>immediately into Standby mode</em>
if arcing is observed.  There are two ways to observe arcing:
 <ul>
 <li><b>By ear</b>.  The arcing noise is usually unmistakable, especially
if the &ldquo;cracking&rdquo; noise is heard several times within a second
or two.  Listen for it.
 <li><b>By the Penning vacuum gauge</b>.  Whenever an arc occurs, even
one not heard, the Penning gauge indicator goes upward significantly.
Not enough to put the system into Standby, but it is noticeable, and the
system recovers quickly.
 </ul>
Hit the Standby key immediately when sustained arcing is observed.
<li>Increment the voltages together.  Watch the numbers in red:  these
are the readback voltages and should change when a new voltage is
entered.  Pause for at least a minute before
incrementing the voltages.  Start out by setting the Source at first
to one-fourth or one-third of its final value (3000 to 5000 V, if the final
value is 15,000 V).  Then do the same with the Reflectron and
Ion Mode Supply values.
<li>In the final one-third or one-fourth, make
the jump in voltages finer, such as by fifths or sixths as the increment
in voltage.  For example, if at 10,000 V on the source, make +1000 V jumps
(fifths).  Pause at each step to listen for possible arcing.
<li>If the Standby mode is entered because arcing was heard, reduce the
current voltages to HALF their set value (do not forget to Save),
and begin again (enter Operate mode), this time incrementing
by thirds for the first two-thirds, or by fourths for the first three-fourths,
then making smaller jumps in the voltage to the final value in the final
third or fourth of the difference.
</ol>
<p>
Final typical values for voltages are as follows:
<ul>
<li>Source:  15,000 V
<li>Reflectron:  2000 V
<li>MCP Detector:  1850-2000 (1950 is good) V
</ul>
<p>
The Pulse voltage must be optimized for a particular mode.  Pulse voltage
optimization is described <a href="#PVoptimization">here</a>.
<p>
Note that the readback voltage for <b>&minus;ve (Negative) Ion Mode Supply</b>
does not report the value entered.  The value on the readback is the
sum of the value entered in the <b>&minus;ve Ion Mode Supply</b>
and the Source Voltage.  For example if the Source

<h2 id="PVoptimization">Pulse Voltage Optimization</h2>
<p>
The pulse voltage is set for each system.  Its optimal
setting will improve resolution.  Generally the pulse voltage not be
set day to day, but occasionally it needs adjusting.
<p>
Adjustment is done by acquiring a set of spectra on any sample, although
a standard peptide or set of peptides is usual.  The following steps can
be used as a guide for optimizing the pulse voltage.
<ol>
<li>Decide on the range of pulse voltages to be checked.  Depending
on the magnitude of the range, the pulse voltages will be studied over
five sets of scans.  Divide the range by 5:  for example, if the range
is 500 V (say from 1100 to 1600 V), the steps to be studied are
1100, 1200, 1300, etc.  If the range is 1000 V, then step the voltage
by 200 V.
<li>Start the laser and acquire at least 10 continuous spectra with
a stable ion count.
<li>Combine the spectra and perform a typical background subtraction.
<li>Determine the resolution on the monoisotopic peak:  this is usually
the peak with the lowest <i>m/z</i> in a set of isotopic peaks representing
the species.
<li>Change the pulse voltage to the next voltage and repeat the previous
three steps until all pulse voltages are tested.
<li>Select the pulse voltage with the highest resolution.
<li>Do 5 more acquisitions, setting the pulse voltage at &minus;100,
&minus;50, 0, +50, and +100 V from the pulse voltage giving the highest
resolution.
<li>Combine spectra, background subtract, and then determine the resolution
for each of those acquisitions.  The pulse voltage giving the highest
resolution is now the optimized pulse voltage.
<li>For a finer determination of the pulse voltage, another 5 acquisitions
from &minus;50, &minus;25, 0, +25, and +50 V can be done from the
pulse voltage optimized within 50 V in the previous two steps.  And
again this can be repeated if the pulse voltage is to be optimized within
10 V.
<li><b>NOTE:</b>  The pulse voltage must be determined separately for
the Linear and Reflectron modes.  Save the optimized settings for pulse
voltages and all other settings in their own IPR files (although it
is possible that Waters/Micromass keeps track of the settings within
a single IPR file).  It might also be useful to optimize the pulse
voltage for negative and positive modes in the linear and reflectron
modes (i.e., four pulse voltage optimizations may be necessary).
</ol>
<p>
In reflectron mode, a resolution of 10,000 (FWHM) is expected between
<i>m/z</i> 1000 to 4000.  In the linear mode, a resolution of 1000 (FWHM)
should be possible for <i>m/z</i> &gt; 10,000.

<h2>Target Plate Spatial Calibration:  Automated Laser Positioning System
(ALPS)</h2>
<p>
It has been seen that the laser flash (seen reflected in the well with the
on-board camera in many cases where there is a thick and even layer of matrix),
it not always in the position indicated on the Tune Page aiming guide
(the circle of the well with the red line crosshairs).
<p>
It is good idea to check several wells of the plate to make sure the
laser is aiming correctly.  Otherwise the aim falls outside the moat of the
well and it wastes time by firing at no sample;  what is even worse is that
the aim may be so far off calibration that the aim is on an adjacent well,
and spectra from the adjacent well are being obtained!!!
<p>
Change to several wells when checking laser position.  Do not move the aim
with the mouse.  Each well change is supposed to position the laser at the
dead center of the well.  After you change to a new well, check the
reflection of the laser on the matrix.  You should have a good coating of
matrix on the surface to get a good reflection.
<p>
If it consistently seems like the reflection is off-center, then it
needs to be calibrated.  In MassLynx version 4.0 software, the Plate
Spatial Calibration command under the Options menu could be used to
achieve this.  However, in version 4.1 software, this feature is broken
and will not be patched.  Thus a special method is used to access the
base level of the embedded PC and correct the positioning.  Because it
is possible to create significant problems with incorrect settings to
other menus, you must be trained to do this.  If you have this training,
then you know the password that will reveal the instructions below.
If you don't have this training, contact someone to reveal the instructions
here:

<script>
function checkPassword()
{
	if (document.getElementById("instruction-password").value.toLowerCase() ==
			"alps")
	{
		document.getElementById("positioning-instructions").style.display = "block";
		document.getElementById("password-control").style.display = "none";
	}
	else
		document.getElementById("password-error").style.display = "block";
	return;
}
function clearError()
{
	document.getElementById("password-error").style.display = "none";
}
</script>

<div id="password-control">
Enter password to reveal laser position calibration instructions:
<p style="text-align:center;margin-bottom:0;">
<input id="instruction-password" size="15" maxlength="15">
<button onclick="checkPassword();" onfocus="clearError()"><span
   style="font-size:80%;">Check Password</span></button>
<p id="password-error"
style="display:none;font:bold 90% Arial,Helvetica,sans-serif;color:red;
text-align:center;margin-top:0;">
  entry incorrect
</div>
<div id="positioning-instructions" style="display:none;">
For positioning the laser through the embedded:
<ol>
<li>Open a command (cmd) window in XP using <b>Start&minus;&gt;Run...</b> then
enter <b>cmd</b> in the <b>Open</b> textbox.
<li>From command: <b>telnet epc</b> (or use the IP address)
<li>You know see the <b>&minus;&gt;</b> prompt.
<li>Enter <b>rioMainMenu</b> at the telnet prompt.  Enter it exactly
as written, since the commands may be case-dependent.
<li>The main menu appears.  Enter <b>3</b> for accessing the subsystem.
<li>Enter <b>601</b> to access the vacuum micro subsystem.
<li>Enter <b>1</b> to access Drive Properties.
<li>Enter <b>998</b> several times to find the menu that includes the
numbers 693 and 694.
<li>To change the X-axis index position (left-right), enter <b>693</b>.
<li>You must enter a value.  The lower value causes a shift to the right,
and the higher value a shift to the left.
<li>In order to see the effect of the new index value, you must re-index
the probe:  hit <b>Unload Probe</b>, then hit <b>Load Probe</b> to
initiate the indexing.  Now check the centering of the wells.
<li>If you want to use a new value, you need to hit the <b>998</b> command
to get back to the axis index setting, and then repeat the entry and
evaluation steps.
</ol>

The next step is to make these changes permanent in MassLynx, because this
will disappear if you reset the embedded or MassLynx.
<ol>
<li>Find the <b>MassLynx</b> directory under the <b>C</b> (or installed)
drive.
<li>Find the subdirectory <b>ToFSpecHT</b> and open it.
<li>Find the subdirectory <b>VxWorks</b> and open it.
<li>Open the file <i>inletin.ini</i> in the directory using Notepad or
other text editor.
<li>Use the search tools or otherwise look for the X or Y Axis Index.
<li>Find text that says X_Axis_Index_DAC or something like that.
There are values in a series of commas.  The last value in that series
is the axis index value.  Set or change it.
</ol>
</div>

<h3>X and Y Axis Indices of Target Plate</h3>
<p>
In the MALDI MS the target plate is positioned by stepper motors that
move the plate in the X and Y axes.  The X and Y axis values are the number
of steps the motors must take to position the target plate in the center.
The value 0 for the X axis index indicates that the target plate has
touched a trip switch at the back serving as the X axis back reference,
and the value 0 for the Y axis index means the plate has touched a reference
on the right side.  Loading means the target plate is moved to the right
and back of the system to locate these reference trip switches.
A target plate thus moves to the front and left to position itself.
<p>
The movement of 267 steps equals 1 mm.  Suppose a target is off from its
center.  If a target is 0.5 mm too low and 0.7 mm too much to the left
of the laser spot.  In steps this is 267 &times; 0.5 = 134 steps too low
and 276 &times; 0.7 = 187 too much to left.  Both of these must be subtracted
from the values to move the position back and to the right.  (Note the most
back and most right movement of plate would be 0 and 0 for the values.)

<h3>Using Plate Spatial Calibration under Options Menu (ALPS)</h3>
<p>
A photodiode exists under the target plate to sense laser flashes
during use of the PSC command.  In the standard Micromass target plate
are two 0.5 mm holes drilled near wells A2 and G10, and these holes are
exactly &plusmn; 13.5 mm in the Y and &plusmn; 18 mm in the X axes from the
geometric center of the target plate.

<ol>
<li>If you want to follow the calibration <q>hits</q>, open a telnet
connection to the embedded.
<li>The target plate is loaded into the carrier, its edges pushed in the
carrier to the corner occupied by well H1.  (Note that the corner with
well H1 is notched.)
<li>Set Coarse energy to Medium, and Fine energy to 25% on the laser.
<li>Make sure that link LK4 is removed.
<li>Select Options&mdash;&gt;Plate Spatial Calibration
<li>Set for 0.1 mm search, 0 ms delay per row, and 10 &times; 10 search grid.
Also check the <b>Use Calibration Initial Conditions</b>.
<li>Start the calibration, and wait for it to complete successfully.
If it fails, increase the search grid to 15 and repeat.  If it fails again,
increase laser energy to 50% an repeat.  If it fails again, any of the
following is possible:
 <ul>
 <li>the alignment holes are blocked:  try doing it without a plate loaded
 <li>the glass below the alignment hold in the carriage plate is obscured
 <li>the target plate to laser spot alignment has not been setup correctly
 <li>the target plate is tilted
 <li>the laser setup may be faulty (not completed accurately)
 </ul>
</ol>
<p>
To optimize ALPS, open <b>Options&mdash;&gt;Target Plate</b> and select
the <b>Spatial Calibration</b> tab.  The <i>Spatial Calibration Initial
Conditions</i> is a fieldset in the tab that tells what to search for.
The <i>Current Calibration</i> displays results of the last ALPS search.
The software makes calculations that can determine plate tilt and
offset from center.  The offset values can be used the change the
X and Y axis index values.  One runs the ALPS calibration command
repeatedly after changing the X and Y axis index values until the offsets
are less than 0.1 mm.

<h1 id="maldibeamtuning">Tuning The Beam</h1>
<p>
This is one of the major activities of the M@LDI user.  It involves several
factors or elements:
<ul>
<li>Verifying the voltages are present and their settings optimal
<li>Finding the right laser energy (both course and fine setting)
<li>Finding the positions with the well where the sample amount is sufficient
for getting good spectra
</ul>
<p>
Tuning the beam will take some experience, but there are a few rules to
follow.  It is first important to understand the factors that affect the
ability to tune a beam.  One is the quality of the loaded sample.
Is the matrix-to-peptide or matrix-to-protein ratio correct?  Generally
there should be about 50 molecules of matrix for every amino acid.
Thus for a protein of 300 amino acids, there should be 15,000 matrix molecules
per molecule of protein.  If 50 fmol of peptides are spotted to a well,
then 750 nmol of matrix should be in the well with the peptides.


<h3>Setting Matrix Suppression</h3>
<p>The matrix suppression refers to the period that the detector is turned
off after each laser shot (the period is a small number of microseconds
usually) to prevent signal (or signal overload) from the matrix that
hits the detector.  It can make the detector more responsive to the
analytes of interest.  It is usually set to 80% of the value of the lower
limit (start value) of the mass range being detected.
<ol>
<li>Open <b>Options-&gt;Voltages</b> to get the voltages dialog.
<li>Select the <i>Detector</i> tab.
<li>The <b>Matrix Suppression Delay (Mass Units)</b> is used to set the
time the detector is turned off so as not to have its responsiveness
reduced by matrix ions.  Set it initally at 80% of the lower limit
(start value) of the detected mass range.  For example, if the mass
range being detected is <i>m/z</i> 1000-3000, the lower limit or start
value is 1000.  Set the suppression at 0.8 &times; 1000, or 800.  The
value can be set either by using the slider switch control, or by directly
entering the value in the text box and making sure that the Enter key
is pressed.
</ol>
<p>
If the laser energy is too high, the following can be seen:
<ul>
<li>Spectra too intense, peaks off scale.
<li>Sample consumed too quickly
<li>Excessive matrix ions, sample fragmentation, low signal-to-noise
with possible detector saturation
<li>reduced resolution, especially in linear mode
</ul>


<h3>Adjusting Sample Periods</h3>
<p>
The sample period is the time division within the detector by which
the detector can discriminate the masses of ions hitting it.
The sample period usually needs to be set differently for linear and
reflectron mode.  When in Reflectron mode, the sample period is
set for a shorter period, typically <tt>0.5</tt> nanosecond.
When in Linear mode, set the period for a longer time, usually <tt>1</tt>
nanosecond.


<h2>Laser Energy</h2>
<p>
If the laser is too low, no signal at all or a low S/N ratio is seen.
<p>
Peaks should be on scale and there should be sustained signal for a single
aim for multiple laser shots.

<h2>Uniform Coating of The Well: &ldquo;Sweet Spots&rdquo;</h2>
<p>
It is often found during changing of the aim of the laser
that certain areas of the well give better signals while
other areas give no signal.  The areas that give a good signal are
called &ldquo;sweet spots.&rdquo;  Sweet spots make it difficult to
do automated (unattended) runs, since the algorithm for aiming the
laser has limits built into it the possibility that a sample is
below the level of detection or may even be empty, and that the laser
should stop looking.  A laser aiming algorithm may not run in a way that
looks for sweet spots.
<p>
The usual problem with sweet spots is that the wrong solvent is being
used to spot the plate.  The best solvents seem to be 50% acetonitrile
in water (includes 0.1% TFA).  They leave a uniform coating on the
surface of the well.  Mixtures of ethanol and acetonitrile have
been observed evaporate non-uniformally, or to move matrix around the well
in ways that form mountains on one portion, and which have no matrix
in other portions.

<h2>An Algorithm</h2>
<p>
If uncertain about whether ion counts (a signal) will occur during an automated
run, it is best to evaluate briefly and manually, i.e.
in non-storage acquisition mode, the spectra/signal output
of doubtful wells on the target plate, and to determine the conditions
(laser energy setting) that will make successful scanning likely.
Here is an algorithm to follow in beam tuning and getting a consistent
signal that also avoids dead time.
<ol>
<li>Set the laser in the lowest energy mode that is typical.  If the
laser performance is known already, then the setting is the lowest energy
setting known to barely produce good ion counts on a spectra.  If the
laser performance is unknown, then set the Energy Filter on <b>Low</b>
and the Fine Energy at <b>50%</b>.
<li>Verify the voltage settings under <b>Options-&gt;Voltages</b>.
<b>Save</b> the settings to the open IPR (Instrument PaRameter) file.
<li>Turn on the laser in non-storage acquisition mode.  Be sure of the
mass range settings, the laser firing rate and the number of laser shots
per spectrum.
 <ul>
 <li><b>Mass Range</b>.  Start with a large mass range at first
if there is uncertainty about the <i>m/z</i>.  For peptides in
Reflectron mode, a Start <i>m/z</i> of 500 and End <i>m/z</i> of
5000 is usually too large, but the <i>m/z</i> peaks can be observed,
and the acquisition re-started with a narrow mass
range.
 <li><b>Laser Firing Rate</b>.  The laser firing rate is usually <b>5 Hz</b>.
Larger than 5 Hz may not give time for the spectrometer to recover after
each reading of ions (and it might create deadtime).  The M@LDI however
allows a limited selection of values.
 <li><b>Shots Per Spectrum</b>.  These are the number of shots of the laser
to produce stastically significant count of ions to draw a mass spectrum.
A value of <b>10</b> is usually.  Fewer than 10 may give a stastically
poor sampling, and more than that produces fewer spectra within a certain
time, and itself can lead to statistically invalid spectra.
 </ul>
<li>If there is no signal at the center point of the well (where all
laser shots are aimed at the start of a new well position), then move to
another point in the well without changing energy.
<li>If still no signal at the new position, try moving to another position.
<li>If there is still no signal, and especially after changing positions
in the well 3-4 more times, then increase the energy by 5-10%.
<li>If still no signal, increase again by 5-10%.
<li>If still no signal, change position at least 3-4 times more.  In changing
position, move large distances from one point in the well to the other.
<li>Continue to increase the fine laser energy setting to 80-85%.  Try
changing position a few times too.
<li>If still no signal, reduce the laser energy to 20-25%, and set the
Coarse Filter to the next highest setting, that is, if at <b>Medium</b>,
set to <b>High</b>.
<li>Proceed to change position a few times, and increase the laser energy
in steps of 5-10%, in order to get an ion count.
<li>If the Coarse setting is <b>High</b>, and the Fine setting reaches
70-80% and there is still no meaningful ion count (spectrum), consider
the following in this order:
 <ul>
 <li>Check the voltage readbacks while the laser is on.
Note, that this is NOT voltage settings (the numbers in the boxes entered
by the operator), but the red-colored numbers that show the voltages.
If the voltages are wrong (too low), this must be corrected.  The Pulse
Voltage in particular accelerates the ions into the flight tube.
 <li>The laser is pointed at a well with no sample put on it.  This is
a common mistake.  The operator believes there is sample in the well, but
actually forgot to apply it.
 <li>The laser is pointed at the wrong well.  Check the readback showing
the well position of the laser, and that it matches the well intended as
a target.  When a new well position is input into the text entry box,
do not forget to press <b>Enter</b> or <b>Return</b>.
 <li>There is insufficient sample or the matrix-peptide ratio is incorrect.
The sample on the well may be below the level of detection.
 <li>The laser needs replacing.  This should be the last thing considered.
Laser lifetimes can vary, but they should be good for at least a couple of
years.
 </ul>
</ol>

<h2>Checking for Proper Ion Counts Per Spectrum (Is the Signal Too High?)</h2>
<p>
If ion counts are (a signal is) observed, it is now important to
determine that the ion counts are not too high, or else deadtime is
likely (this causes mass determination inaccuracies).  Generally one
wants about 250 ions per second.
<p>
If a laser is firing at 5 Hz and there
are 10 shots necessary to produce a spectrum, then each spectrum is
produced every 2 seconds.  Thus each spectrum should have about 500 ions.
<ol>
<li>Determine what the optimal ion count per spectrum is as shown in the example
(the example showed 500 counts per spectrum).
<li>When there is a good signal, look at the ion counts for each generated
spectrum as they are updated (make sure that the <b>Real Time Update</b>
option is checked (the stopwatch on the toolbar menu).
<li>The optimal count is controlled by the laser energy.  Adjust the
Fine setting (and the Coarse Filter, if necessary) up or down to modulate
the signal.  Note that the ion count does not respond rapidly to the change
in energy setting (depends on the laser firing rate and shots per spectrum).
The lower the firing rate and the higher the shots per spectrum, the longer
the delay in seeing the response to changes in energy setting.
<li>Do not be overly concerned with getting exactly the value of optimal
counts per spectrum.  In general, the user needs a consecutive set of
spectra (about 10-15) that have acceptable counts that range from 50-200%
of the optimal per spectrum ion count.
<li>An overloaded signal (too high an ion count) is <em>definitely</em>
indicated when the single shot spectrum on the Tune Page (the spectrum
on the right of the two spectral displays) shows a continuous, sustained
high limit of 255 (the maximum).  This indicates the detector is being hit by too
many ions and is probably in dead time.
<li>If changing to a new position in the well, and the ion count is
good or high at first, and then rapidly declines, making it impossible
to get acceptable consecutive spectra, then one of two problems is
possible:
 <ul>
 <li>The laser energy setting is too high.  A laser energy setting too
high essentially &ldquo;burns off&rdquo; (depletes rapidly) the area
of the well aimed at by the laser.   It is possible to reduce the energy
and still get good signal.
 <li>The amount of sample (peptides or matrix) is too little.  Spot the
well again, but at a higher concentration of the sample.
 </ul>
</ol>

<!--  CALIBRATION OF MALDI -->
<h1 id="maldicalib">Calibration</h1>
<p>
Calibration is done from a known reference mixture, usually daily or
before a long automated data acquisition.  In addition, a lock mass
calibration can be done by measuring the mass of a reference after
acquiring the sample.  The &ldquo;near well lock mass&rdquo; is used
to correct the sample result to an accuracy of 50 ppm (0.005%).
<p>
In addition, a known peptide in the sample can perform an <i>internal
lock mass correction</i> to within 15 ppm (0.0015%).

<h2>Reflectron</h2>

<h3 id="createref">Creating Reference Files</h3>
<p>
Reference files are plain ASCII text files that contain information
about the theoretical <i>m/z</i> peaks in a spectrum.  The m/z values
can be generated from any protein database tool in which the correct
input parameters are given.  A good tool to generate theoretical
peptide <i>m/z</i> values from known proteins can be found
<a href="http://www.expasy.org/tools/peptide-mass.html">here</a>
<p>
The generated <i>m/z</i> values can then be cut and pasted into a text
file (Windows' Notepad is an application that reads and makes text files).
Reference files should be stored in the <b>Ref</b> subdirectory under
the MassLynx directory.
<p>
Here is one (a good?) way to generate a reference file:
<ol>
<li>Obtain the list of theoretical masses of peptides from a public server
database like the one linked to above.  When generating the list, keep
in mind the following:
 <ul>
 <li>Be sure to select the right <i>artificial modifications</i>:
 was the protein treated by reductive alkylation (i.e. a sulfhydryl
 like DTT followed by iodoacetamide), which modifies
  the cysteines?
 <li>Also account for any possible <i>biological</i> or
<i>post-translational</i> modifications (modifications not done by
humans, but by nature itself).
 <li>Do not generate a list of <i>all</i> the masses necessarily.
Just the list of masses within the mass range being studied.
 <li>Most importantly, generate a list that accounts for the possibility
of <i>missed cleavages</i>.  Proteolytic enzymes are not perfect, and
some sequences have peculiar
 </ul>
</ol>

<h3>Calibration Parameters</h3>

<h3>Creating An Instrument Calibration</h3>
<p>
This uses tryptically digested ADH to do the calibration.


<h4>Manual Calibration</h4>

<ol>
<li>Load the standard on a target plate well (ADH peptides with
a range of <i>m/z</i> 1000-3000 are suitable).
<li>Set the mass range accordingly, and acquire at least 10 spectra.
<li>Make sure the laser energy is optimal and avoid dead time.
<li>Open the file from MassLynx and combine at least 10 spectra.
<li>From the Spectrum menu bar, select <b>Tools&mdash;&gt;Make Calibration</b>
<li>Select the name of a calibration file of known reference masses from
the protein digest or peptide(s) used (e.g., <b>ADH.ref</b>)
<li>Click the <b>Mass Measure...</b> button and make the following settings:
 <ul>
 <li>Background Subtract: checked
 <li>Polynomial Order: 10
 <li>Below Curve (%): 20
 <li>Smooth: checked
 <li>Peak Width (Da): 3
 <li>Number of Smooths: 1
 <li>Select <b>Savitzky-Golay</b>
 <li>Min Peak Width At Half Height (Channels): 6
 <li>Select <b>Centroid Top (%)</b>: 75
 </ul>
Select <b>OK</b>.

<li>Select the <b>Make New Calibration...</b>.
<b>Calibration parameters</b> must be chosen first.  Set the parameters
as follows:
<ul>
<li>Perform Auto Peak Matching: unchecked
<li>Peak Window (Da): &plusmn;1.00
<li>Initial Error (Da): 2.00
<li>Intensity Threshold: 0.01
<li>Curve Fit Polynomial Order: 2
<li>Intensity Weighting: unchecked
<li>Calibrate Display: unchecked
</ul>
A <b>Calibration Report Window</b> appears with 4 plots.
<p>
The topmost plot contains the processed (background subtracted, smoothed, and
centroided) peaks with labeled masses measured by the MALDI-ToF.
Peaks that are colored red and labeled are those identified as being
<i>matched</i> or the most close to the reference masses (see next plot).
The count of peaks matched in the measured masses with the reference
masses is given in the upper right.
<p>
The next topmost plot is the masses obtained from the reference file
containing the known masses.  All those peaks are labeled and colored
because they represent known masses.  The reference file will be updated
to correspond to the measured masses after calibration is complete.
<p>
The next plot is the difference between the measured (raw) and reference
masses (Raw &minus; Ref), with the difference represent in &ldquo;amu&rdquo;.
1 amu is essentially 1 Da.   Ideally the line should be flat and near
zero.  In reality it is sloped, and should at least cross the zero line.
A line that does not cross the zero line indicates that the effective
flight tube length (<i>Lt</i><sub>eff</sub>) is probably poorly set.
<p>
The bottommost plot measures the residuals, or the square of the
difference between the mass difference and the mean of the differences:
<table class="eqnserif" style="margin:1em auto;">
<tr><td rowspan="2"><span
style="vertical-align:middle;font-size:150%;">&Sigma;</span><span
 style="margin-left:-0.75em;vertical-align:1.5em;font-size:75%;"><i>n</i></span>
 <span style="margin-left:-1em;vertical-align:-1.5em;font-size:75%;"><i>i</i>=1</span>
 <span style="vertical-align:middle;font-size:150%;">(</span><i>x<sub>i</sub></i> &minus;
 <td class="numer" style="padding-left:1em;"><span
 style="vertical-align:middle;font-size:115%;">&Sigma;</span><span
 style="margin-left:-0.75em;vertical-align:1em;font-size:75%;"><i>n</i></span>
  <span style="margin-left:-1.5em;vertical-align:-1.25em;font-size:75%;"><i>i</i>=1</span>
  <i>x<sub>i</sub></i>
 <td rowspan="2"><span
 style="vertical-align:middle;font-size:150%;">)</span><sup>2</sup>
<tr><td><i>n</i>
</table>
<p>
<i>x<sub>i</sub></i> represents each <i>raw - ref</i> mass difference,
and &sum; <i>x<sub>i</sub> / n</i>
represents the means of the differences.	Generally the mean should be
close to zero and so should the &plusmn; error about the mean.  The
mean of the differences and the error about the mean are shown on the
upper right of the bottom plot.  In the calibration process, peptides
from the raw data are matched to the peptides in the reference set,
and some are unmatched if the matching process was set to automatic.
<li>Match and/or unmatch the raw peptides with the reference peptides,
and during the matching/unmatching process, follow the mean of the
differences and the error about the mean.  There should be at least two
considerations when editing the calibration:
<ul>
<li>The number and distribution of peptides that are matching (part of
the calibration) should span a broad range if peptide masses of unknowns
are to be interpolated
<li>The error of the mean of differences (the residuals) should be about
&plusmn; 0.01 Da if the correct peaks were selected in the calibration
data set.
</ul>
<li>When all matching and unmatching is completed, save the calibration
into a .CAL file.  It should be saved to the ACQUDB directory of the
current MassLynx project.
<li>From the Tune Page of the M@LDI, select
<b>Calibrate-&gt;Calibrate Instrument</b> to invoke the Calibration dialog.
<li>Select <b>File-&gt;Save As</b> to get its dialog.
<li>Go to the ACQUDB directory under the current MassLynx project.
<li>Enter or select an existing filename with a .CAL extension.  Select <b>OK</b>.
<li>Close the Calibration window using File&minus;&gt; or the red X:
when asked to accept the calibration, then click <b>Yes</b> if it is
acceptable.
<p>
These next steps are <i>extremely important</i>:  failure to follow it
means that your mass spectra will not be calibrated against the calibration
file just saved.  Although you believe that the instrument is calibrated
by the interface you just encountered, in fact, it is not.
<li>Open the Tune Page of the MALDI (the MS Tune tool from the MassLynx
main window).
<li>Select <b>Calibration&minus;&gt;Calibrate Instrument...</b>.
<li>From the Calibration dialog, select <b>File&minus;&gt;Save As...</b>
<li>Select the .CAL file created/saved above from the Spectrum Calibration
dialog, and then click OK.
</ol>
<p>
The instrument is now calibrated, and the M@LDI acquistion system has been
informed that all data acquired is mass-measured in manual or automatic
mode against this calibration file.

<h4>Automated Calibration</h4>
<p>
The matrix can be spiked with a low concentration of ACTH to serve as
an internal lockmass.
<p>
For automated calibration, select at least one of the lock mass wells
with ADH or a peptide mixture.  This becomes the &ldquo;calibration
well.&rdquo;
<p>
The rest of the lockmass wells are loaded with a single peptide
(e.g., ACTH) in matrix.
<ol>
<li>From the Tune Page interface, select
<b>Calibration&minus;&gt;Calibrate Instrument...</b> to invoke the
Calibration dialog.
<li>From the dialog, select <b>Edit&minus;&gt;Calibration Parameters</b>
to get the Calibration Parameters dialog.
<li>In the Peak Match group, uncheck <b>Perform auto peak matching</b>.
<li>Set the Curve Fit group <b>Polynomial Order</b> option to 2:
especially necessary when acquiring data from <i>m/z</i> 1000-4000
in reflectron mode.
<li>Set Peak Window to &plusmn; 10 Da, Initial Error to 2 Da, and
Intensity Threshold to 5.
<li>Select <b>OK</b> to close the window and accept changes.
<li>Select the Reference file:  the only way to do this is to make
sure the toolbar is being viewed (<b>View&minus;&gt;Toolbar</b>).
A dropdown list of names of reference files then appears.  Select the
appropriate one that corresponds to the masses of the peaks for the
calibrant peptides.
<li>Select <b>Edit&minus;&gt;Tof Mass Measure Parameters...</b>
to get the Mass Measure dialog.  Enter the following parameters:
 <ul>
 <li>Background Subtract:
  <ul>
  <li>Polynomial Order:  10
  <li>Below curve:  20%
  </ul>
 <li>Smooth:
  <ul>
  <li>Smooth window:  &plusmn;3 channels
  <li>Number of smooths:  1
  <li>check Savitzky Golay
  </ul>
 <li>Min Peak Width @ Half Height:  6 channels
 <li>Centroid top:  75%
 <li>check Deisotope (only for reflectron mode)
 </ul>
<li>Select <b>OK</b>
<li>Select <b>Calibrate&minus;&gt;Start Acquisition...</b> which generates
the <b>Automatic Calibration</b> dialog.
<li>In Types group, the <b>Sample</b> is the well or vial position of the
calibrant peptides.  For MALDI target plates, type the row (A-H), a comma,
then the column number (1-12).  So if the calibrant is in well
C4, type &ldquo;C,4&rdquo;.  If it is the lockmass well, make sure to
click lockmass.
<li>In the <b>Process</b> group, select the <b>Acquire &amp; Calibrate</b>
option.
<li>From the Types group, select <b>Acquisition Parameters</b>.
This dialog is the interface to the Experiment Editor, setting Function 1
of the MALDI scan and also setting the details for laser firing.
This is described <a href="WatersTraining.html#exptedit">elsewhere</a>.
Recommended values for the Laser Details:  random laser pattern with
10 aims, RTDS setting with a 10% low and 90% high intensity thresholds,
5 max pass per aim, 3 max fail per aim, and 5 max pass per well.
<li><b>NOTE</b>:  in older versions of MassLynx, the user was asked to
select the instrument parameter (.IPR) file that sets up the tune page
voltages, modes, etc.
<li>Select <b>OK</b> to leave the <b>Acquisition Parameters/Function 1</b>
dialog and then the <b>Automatic Calibration</b> dialog.
</ol>

<h3>Checking Calibration Conditions</h3>

<h3>Using Internal Lockmass</h3>
<p>
Internal multipoint calibrations can improve mass accuracy within 15 ppm.

<h3>Using External Lockmass</h3>
<p>
External lockmass using <i>near well lock mass</i> can correct masses
to an accuracy of 50 ppm.


<h2>Linear</h2>
<p>
Linear mode MALDI-ToF is used for analyzing much larger molecules
(proteins more than 10 kDa).  It might also be used when the <i>m/z</i>
of any ionic species is &gt; 5000 or 10,000.  The reason is that the
Reflectron voltage may be insufficient to bend the ion back to the
reflectron detector.

<h3>Creating Reference Files</h3>
<p>
Look <a href="#createref">here</a> for information about what reference
files are and how to create them.

<h3>Calibration Parameters</h3>
<p>
Parameters used in setting up calibrations are:
<ul>
<li>Auto-peak matching
<li>Polynomial Order and Intensity Weighting
<li>Calibrating the Display
</ul>
<h4>Auto-Peak Matching</h4>
<p>
After the spectra are obtained from the calibration well (the well
containing the calibrant molecules or peptide standards used in a
calibration), the combined spectra are processed (&ldquo;mass measured&rdquo;:
that is, background subtracted, smoothed, centroided, possibly de-isotoped).
The spectrum is then presented in a new window with the centroided (or continuum)
peaks with the noise.  Immediately below the spectrum is another
&ldquo;spectrum&rdquo; which contains the information about the <i>m/z</i>
values within the reference file that has been selected for the calibration.
Note that not all the <i>m/z</i> values from the reference file are presented:
only those specified within the data collected for the low (start) and
high (end) mass range.  The reference peaks represent the accurate (or
calculated) masses of the monoisotopic mass of an ion.
<p>
The reference peaks will all be red in color.
<p>
If auto-peak matching is selected as an option, an attempt will be made
to match a reference peak to a centroided peak from your acquired spectrum.
The centroided peak in the acquired spectrum that is closest to the reference peak
will be matched, but only if it the <i>m/z</i> of the peak is within the
set <b>Peak Window</b> value, and if the relative intensity of the peak
is <em>above or equal to</em> the <b>intensity threshold</b> value
set.  For example, suppose your reference is 1570.6774, and your measured
mass is 1573.58 with an relative intensity of 12%.  If your intensity
threshold value is <b>10</b> (i.e. 10%), then the peak will be matched
because it goes over the intensity threshold.  But if the peak window is
&plusmn; 1 Da, no match occurs because 1570.6774 &minus; 1573.58 =
&minus;2.903, which exceeds the limit.  If the Peak Window is &plusmn; 3 Da,
then a match occurs.
<p>
If you set the intensity threshold too low (say &lt;2%), you more likely
to match noise.
<p>
Auto-peak matching is more valuable or important if you have more than
10 calibrants spanning the spectrum, since manual matching repeatedly
can be tedious.

<h4>Polynomial Order</h4>
<p>
This specifies the type of curve fit that will be used.  An order of
1 does a linear fit (presumably by least squares).  An order of two
attempts a quadratic fit.  Increasing the order does more to force
a line through points, but this is not always useful and could hide
problems with mass spectrometer response.
<p>
<b>Intensity weighting</b> essentially creates a line fitting analysis
that gives greater weight in the line fitting process to mass peaks
that show higher intensity.  Thus the higher the intensity, the higher
the weighting value.  This should not be used when there are very
few calibrants that span the mass range.

<h3>Creating Instrument Calibration</h3>

<h3>Creating Calibration From Spectrum</h3>


<h3>Applying A Calibration</h3>

<h1>Routine Application</h1>
<p>
This is a simple list of steps to take in using the MALDI-ToF on a routine
basis, and typically for sample analysis in an automated fashion
(using the Sample List in MassLynx created in MassLynx or ProteinLynx).

<ol>
<li>Load the target plate with prepared samples in the standard
MALDI wells and lockmass references in the external lockmass wells.
<li>Open the MassLynx software after PC startup
<li>Open the Tune Page for the MALDI-ToF by clicking <b>MS Tune</b>
  on the shortcut bar of MassLynx main page.
<li>In the Tune Page, check the voltages, vacuum and laser settings.
All parameters in the Tune Page can be stored in an IPR (Instrument
PaRameter) file, and whenever a parameter (voltage, laser setting)
is changed, it should be saved in the current IPR file, since all
acquisitions (whether started from the Tune Page or from an automated
MassLynx sample list)
</ol>

<h2>Acquisition</h2>
<p>The acquistion of spectral data can be done in several different modes.
<ul>
<li>Non-storage acquisition:  this mode is usually used to judge the
tuning needs of a sample or set of samples on a plate that is similarly
prepared.  No data is stored.  It is only used to estimate the laser
energy settings and voltages.  Spectra are present on the Tune Page.
Make sure that the Setup Scope is used to set the mass (<i>m/z</i>)
range and the laser firing rate and shots per spectrum.
<li>Storage acquisition:  data can be acquired manually or through
the automated control of a sample list.  The most efficient use of the
MALDI is through the automated control of a sample list.
 <ul>
 <li>Manual storage acquisition:  on the Tune Page, press the <b>Acquire...</b>
button to the lower left of the page.  A dialog presents the following:
  <ul>
  <li><b>File Name</b>:  file naming is a matter of preference.  Although
with the Windows filesystem there is no severe limit on name length, it is
advisable to keep it short.  It might include the name of the analyte being
studied.  It should probably include the date in YYYYMMDD format, with
zeroes filled in where there are single digits.  Probably the best format
is YYYYMMDD-RR (note the hyphen), where RR refers to the number of runs
either for the day.  The alternative is YYYYMMDD-RR-analyte,
where <i>analyte</i> is the name or abbreviated name of the analyte, and
RR is the run for that analyte on that day.  For example, suppose
the analyte is ACTH and the date is 23 January 2005 and the run is the first.
The file name might be 20050123-01, if that is the first run of the day,
or 20050123-01-ACTH if that is the first run of ACTH for that day.
Whatever file naming convention is used, try to be consistent.  Note
that it is not always necessary to include the analyte in the file name,
since MassLynx includes sample descriptions (not part  of the file name)
in the Open dialog box that can be read.
  <li><b>Text</b>:  this is used to include a more extensive sample description
that is usually not a good idea to make part of the file name.  One might
want to include the components and the masses of the sample of well, or make
any other notes that distinguish a sample uniquely from other samples, or
which explain why the data is being acquired.
  <li><b>Mass Range</b>:  this indicates the lower (start) and upper (end)
<i>m/z</i> value limits in creating the mass spectra.  Note <i>m/z</i>
values will be recorded outside the range.
  <li><b>Laser Firing Rate</b> and <b>Shots Per Spectrum</b>.  Set the
firing rate and number of shots necessary to acquire each spectrum.
A firing rate of 5 Hz is typical.  Usually at least 10 shots are acquired
to produce a spectrum representing a useful sampling.
  </ul>
Select <b>OK</b> and then control the data collection through the
Chromatogram and Spectrum windows of the MassLynx main window.  To stop
this acquisition, press the red-colored square on the toolbar of the
Tune page.
 <li>Automated acquistion:  the most effective way to get mass spectra
is by the creation and running of sample lists, described below and
in another <a href="WatersProteomics.html#samplelists">section</a>.
 </ul>
</ul>

<h3>Creating and Using Projects</h3>
<p>MassLynx organizes data for the operator in &ldquo;projects.&rdquo;
When MassLynx is first installed (first run), it creates a project
called the Default project, in which it creates a directory named
<b>Default.pro</b> as a subdirectory of the MassLynx main (installation)
directory.  All projects (as directories) are created under the MassLynx
directory, although it is not strictly required.  The project directories
are usually given the extension to their name <b>.pro</b>.  It is a
good idea not to change the name to remove the <b>.pro</b>, since MassLynx
probably uses that to report the names of projects it creates and manages.
<p>
Within any project subdirectory, several other (sub)directories are
created.  They are:
<ul>
<li><b>Acqudb</b>.  A short form of &ldquo;ACQUistion DataBase.&rdquo;
<li><b>Curvedb</b>.  A short form of &ldquo;CURVE DataBase.&rdquo;
<li><b>Data</b>.  This is the directory in which all acquisition
data files (spectra and chromatograms) are stored.  When a project
data file is to be opened, MassLynx usually opens by the default the
Data directory within the current project.   When a stored acquisition
is started in any mass spectrometer, a subdirectory is created within
the data directory that is the &ldquo;file name&rdquo; given by the operator.
Within this &ldquo;file name&rdquo; directory, several files are created
which are critical to MassLynx being able to draw the Chromatogram
and Spectrum windows.  The various files and their types are of little
importance to the operator, but are explained elsewhere.
<li><b>Finddb</b>.  A short form of &ldquo;FIND DataBase.&rdquo;
<li><b>Methdb</b>.  A short form of &ldquo;METHod DataBase.&rdquo;
<li><b>Peakdb</b>.  A short form of &ldquo;PEAK DataBase.&rdquo;
<li><b>Sampledb</b>.  A short form of &ldquo;SAMPLE DataBase.&rdquo;
This directory holds the information for all sample lists created and
used within MassLynx.
</ul>

<h3>Sample Lists</h3>
<p>Creating sample lists is how to automate the process of acquiring mass
spectra and identifying the proteins based on the information from
the mass spectra (peptide masses).
<p>
Instrument control files are Experiment (.EXP) files.  Files that specify
the processing parameters for the database search are the .MLP files.

<h4>Running Sample Lists</h4>
<p>From the MassLynx menu, select <b>Run-&gt;Start</b> or select the toolbar
button featuring a blue arrow that points to the right.  A <b>Start Sample
List Run</b> dialog opens.
<p>
This dialog feature the following controls:
<ul>
<li><b>Project</b>:  the frame contains the name of the current project.
If another project is wanted, cancel the Run dialog, and open another project
and start the acquisition again.
<li><b>Acquire Sample Data</b>:  all data is acquired from the sample list
<li><b>Auto Process Samples</b>:  causes acquired data to be processed.  The
processing occurs according to the Process column in the sample list.
Note that this can also process existing data too.
<li><b>Auto Quantify Samples</b>:  acquired data is quantified using the
method in the <b>Quantify Samples</b> dialog, which is presented after
the OK button is pressed.
<li><b>Run/From Sample</b>:  the first sample to be acquired
<li><b>Run/To Sample</b>: the last sample to be acquired
<li><b>Run/Priority</b>:  if checked, job is a priority process and placed
at the top of the Queue, to run after any currently running job.
<li><b>Run/Night Time Process</b>:  useful for time consuming acquisitions
that might interrupt work on smaller acquisitions during the day
</ul>
<p>In addition, there are Pre-Run and Post-Run external executables that can
be run to process data
<h3>Sample List Formats</h3>
<p>A basic sample list is created within the MassLynx main page.  This
shows a sample list in a special window with several spreadsheet style
columns.  The columns represent field data, and the rows a record of the
sample.  The presentation of a sample list is its format.  The format
of a sample list, i.e. the particular way that it looks, can be saved
as a .FMT file.

<h3>Setting Acquisition Parameters</h3>

<h3><a name="exptedit"></a>Experiment Editor</h3>
<p>The Experiment Editor is used to control all aspects of the mass spectrometer
of sample analysis during an automated run.  Parameters for the scanning
of the sample wells and the near lockmass wells are set from within the
editor.  When parameters are set, they are called &ldquo;functions&rdquo;
of the experiment.  Thus if the lockmass scanning parameters are set first,
this becomes &ldquo;Function 1&rdquo; of the scan.
<p>
To set either the sample well or lockmass well parameters:
<ol>
<li>Open the Experiment Editor from the MassLynx window by clicking
<b>MS Method</b> from within the <b>Instrument</b> bar.
<li>The window named <b>Experiment Setup</b> should be visible, with
a path name to the last .EXP file previously opened/viewed/edited.
All MS method files are .EXP file types.
<li>Under the <b>Function</b> menu, there are two functions: <b>Maldi</b>
and <b>Maldi Lockmass Correction</b>.  These open up a dialog window
for setting parameters related either to the sample well scanning or
the lockmass well scanning.  These are also accessible from a toolbar
below the menu, with buttons that display &ldquo;Maldi Scan&rdquo; or
&ldquo;LMC&rdquo;.  Select the <tt>Maldi</tt> function.
<li>The window title bar should say something like <b>Function 1: Maldi Scan</b>
if no other functions have been set.  The <b>Function</b> tab in the window
should be displayed.  In this tabbed window are controls that can be
set that are found also in setting the MS from the Tune Page:
 <ul>
 <li>Mass Range:  start (low) and end (high)
 <li>Laser:  firing rate, and shots required to produce one spectrum
 <li>Ionization Mode: positive or negative
 </ul>
Set these parameters accordingly.
<li>Click the <b>Laser Details</b> tab.  This displays two groups of
controls.  One is the <b>Laser Aims</b> and the other is <b>Real Time
Data Selection</b> (RTDS).  The laser aims instruct the automation routine
how to direct the laser fire into each well.  Check <b>Laser Position
Detail</b> to activate the group of controls.  In <b>Laser Pattern</b>
one of three firing pattern algorithms can be selected:
 <ul>
 <li>Circle: the laser will fire in a spiralling circle starting from the
 outside of the well until it reaches the center
 <li>Raster:  the laser aims will move along a horiontal row and then
 move vertically downward, one row at a time, in a left-to-right pattern
 <li>Random:  the laser will move to random locations throughout the
 well
 </ul>
Alternatively, the <b>Laser Coordinates</b> can be entered as (<i>x,y</i>)
coordinates into a text box.  Note that (0,0) is the center of the well,
and that negative <i>x</i> and <i>y</i> values are possible, covering
the appropriate quadrants of a grid system.
<p>
The <b>Number of Aims</b> refers to the total number aims that will be
taken in the well.  Set this group of controls accordingly.
<li>The <b>Real Time Data Selection</b> group of controls is discussed
in the <a href="#rtds">next section</a>.
<li>When both tabbed windows and their groups of controls are set,
click <b>OK</b> to return to the main window of the editor.
<li>The set Functions now have to be saved as an .EXP file.  Select
<b>File-&gt;Save</b> to save the currently named file, or
select <b>File-&gt;Save As...</b> to save the file under a different name.
</ol>
<p>
Note that the Function parameter settings for the Lockmass well do not
differ from the typical MALDI scan (sample) well.

<h3><a name="rtds">Real Time Data Selection (RTDS)</a></h3>
<p>RTDS is used basically to control the kind of mass spectra to be collected
during the automated run.  This is a part of the Experiment Editor.
<ol>
<li>Check the <b>Apply RTDS</b> box to activate the group of controls.
<li>To set the <b>Laser Energies</b> to be used in the automated
run, check the box to activate the input text box.  A brief instruction
is already displayed that has the user enter the laser energies as
percentages, separated by commas.  During the run, the laser energy will
be set to the percentage in the first value of the series, and then
proceed to the next value if the system does not detect signals (get
good spectra).  When writing in a series of laser energies, it is best
to start low and work high.  Note that the laser coarse filter (low,
medium, high) must be set in the Tune Page prior to the starting of the
sample list.  The choice of laser energies will depend on the operator's
experience with the system and knowledge of the capabilities (age) of the
laser.
<li>Set the <b>Low</b> and <b>High Intensity Thresholds</b> to the range
where the major (highest, base) peak intensity in the summed spectra
should fall.  Summed spectra obtained whose major peaks are below
the low threshold and above the high threshold are disregarded (not used
in creating the chromatogram).  Generally spectra below the threshold
are just the noise of analytes not present, and those above show an
overloaded signal that may involve deadtime and inaccurate mass readings.
<li>The Low and High Mass ranges are not the same as the values in the
Function parameters; those refer to the spectral data that will be
recorded.  The Low and High mass ranges here refer to the <i>m/z</i>
values whose high and low intensity thresholds will be determined.
<li><b>Max Pass Per Aim</b> refers to the number of summed spectra
falling within acceptable parameters for making them part of the
chromatographic record&mdash;i.e., that &ldquo;pass&rdquo;&mdash;which
can be collected with a single laser aim.  Set this to a lower value
if peak intensities are likely to vary considerable throughout the area
of the well.
<li><b>Max Fail Per Aim</b>  instructs the laser to change aim if
the indicated number of consecutive summed spectra have been discarded
because they fail to go above the low intensity threshold <em>and</em>
the laser energy has been set at its maximum value if laser energies
are set by the user.
<li><b>Max Pass Per Well</b> is the number of summed spectra to be
collected that pass the criteria for a sample well.  Once the number
has been reached, processing for the well will commence.
<li>The <b>Use Learned Conditions</b> checkbox instructs the RTDS to
start with a laser energy on a sample well that represents the running
average of laser energies that result in passing (qualified) spectra.
</ol>

<h3>Effects of Tuning Parameters and Calibration On Accuracy of Acquired Data</h3>
<p>Data quality in mass spectra is a function of:
<ul>
<li>resolution:  the peak width at half the height of the base (most
intense) peak
<li>mass accuracy:  the use of a well-calibrated instrument, combined
with internal and external standardization (lockmass correction) using
peptides with known masses
</ul>
<p>
Resolution is affected by:
<ul>
<li>Use of optimal laser energies:  energies too high can degrade resolution,
and those too low make the signal indistinguishable from noise
<li>Optimal voltage settings:  the pulse voltage needs to be optimized
between linear and reflectron modes
</ul>
<p>
Mass accuracy is largely determined by the use of calibration peptides,
and matching their proper theoretical masses to the measured mass.
It is important to be aware of the condition of the peptides and matching
those conditions to modifications of the masses, and to distinguish
use of monoisotopic and average masses.

<h3>Data File Sizes</h3>
<p>The data files can become quite large.  File size depends large on the
range of <i>m/z</i> values being surveyed and the number of scans
(chromatographic time) collected for a sample acquisition.  Experienced
operators usually do not need to collect more than 20-30 scans for a
single acquisition, but when performing initial studies in a manual
scan, the single acquisition file can become large as many as 200-300
scans might be obtained.

<h2>High Throughput (Auto loading and Acquisition)</h2>
<p>The M@LDI is not equipped with a robotic arm and target plate loading system,
so this feature is not described.

<h3>MALDI Monitor</h3>
<p>The MALDI Monitor is a feature present under the <b>Options</b> menu
of the Tune Page only when a Robot loader and Barcode reader are installed.

<h3>Creating .OLB Files</h3>
<p>OpenLynx Batch (OLB) files are created with ProteinLynx during the
sample automation run definition process, and imported by MassLynx
for defining and setting up the sample list.  The process involving
the creation of .OLB files is described
<a href="WatersProteomics.html#samplelist">elsewhere</a>.

<h3>Sample Monitor and Autolynx</h3>
<p>The Sample Monitor is software installed with ProteinLynx and used when
the Robot Loader is present.  AutoLynx is installed with MassLynx and
is used to define large batches involving multiple plate loadings.
Since these units are not available, no explanation for their use is
presented.

<h3>Viewing Results</h3>
<p>Sample automation runs created in ProteinLynx produce results viewable
within ProteinLynx.
<ol>
<li>To maximize the viewing area for results within the ProteinLynx Browser,
click the white arrow pointing left located on the blue splitter bar
between the Tools Tray and main ProteinLynx window in order to hide
the Tools Tray.
<li>Select <b>View-&gt;Navigator Tree</b> and uncheck which hides
the Navigator Tree panel.
<li>To see results, click the icon within the <b>View</b> column of the
<b>Results Summary</b> table.
<li>Select a well on the plate, and right-click for a popup menu with
the command <b>View Results</b>.
<li>Double-click the Workflow results node within the Container
Manager navigator tree.
</ol>
<p>
A workflow results browser appears with the results of all database
searches as well as mass spectra.  Presentation of results can include
merged results from a workflow that specified multiple analyses.
<p>
The Results Browser is composed of four sections:  <b>Navigator Tree,
Table of Protein and EST Data, Table of Peptide Data,</b> and
<b>Spectrum Viewer</b>.  When MS/MS spectra are shown, the parent ion
MS spectrum is shown as well.
<p>
Several &ldquo;views&rdquo; are presented depending on the workflow.
If a BLAST query was made, a BLAST View is presented.  In addition,
a <b>Protein View</b> shows proteins and ESTs matched to spectrum data.
<p>
A <b>Peptide View</b> shows the results of

<h2>Post-Accelerator Detector</h2>
<h3>Loading and Unloading PAD</h3>
<h3>PAD Voltages</h3>

<h2>Post-Processing</h2>
<h3>Chromatograms</h3>
<h4>Displaying Chromatograms</h4>
<h4>Displaying Spectra from Chromatograms</h4>
<h3>Spectra</h3>
<h4>Displaying Spectra</h4>

<h4>Background Substract</h4>
<p>Background substraction effectively determines a constant used to substract
the data point so that an offset between the zero intensity and the
baseline is removed.  This is usually done before determining the resolution
of the system.  A good substraction algorithm will not use this
processing step to alter the character of the spectrum.
<p>
This process step may want to make a determination of what constant to
select for subtraction either by looking at the whole or only part of the
spectrum.  It may only want to consider part of the spectrum if there
are abrupt changes in background signal or there are multiple &ldquo;features&rdquo;
of the spectrum.

<h4>Smoothing</h4>

<h4>Centroiding Peaks</h4>


<h4>Changing Spectrum Header</h4>


<h4>Peak Annotation</h4>


<!-- ============================================================= -->


</body>
</html>



