<!DOCTYPE html>
<html lang="en">
<head>
<title>Principles of Mass Spectrometry</title>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8">
<link href="./stdcss/std.css" rel="stylesheet">
<script>
var ionMass, flightTubeLength, ionCharge, acceleratingVoltage;
var noErr = document.createTextNode("");
var massErr = document.createTextNode("Enter a positive real number " +
 "representing the mass of the ion in daltons.");
var lengthErr = document.createTextNode("Enter a positive real number representing " +
	"the length of the mass spectrometer flight tube in meters");
var chargeErr = document.createTextNode("Enter an integer representing " +
				"the charge magnitude of the ion");
var voltageErr = document.createTextNode("Enter a real number representing " +
				"the accelerating voltage of the mass spectrometer flight tube");
function calcToF(InpObj) {
	var protonMass = 1.672621e-27; // units are kg/dalton
	var unitCharge = 1.60217653e-19;  // units are coulombs
	var TOFform = document.getElementById("ToF");
	var objname = InpObj.name.toLowerCase();
	var retVal = true;
	var ErrRep = document.getElementById("ErrorReport");
// showObjProps(ErrRep);
	if (ErrRep.firstChild == null)
		ErrRep.appendChild(noErr);
	else
		ErrRep.replaceChild(noErr, ErrRep.firstChild);

	if (objname == "ionmass") {
		ionMass = parseFloat(InpObj.value);
		if (isNaN(ionMass) == true || ionMass < 0.0) {
			ErrRep.replaceChild(massErr, ErrRep.firstChild);
			retVal = false;
		}
	} else if (objname == "lteff") {
		flightTubeLength = parseFloat(InpObj.value);
		if (isNaN(flightTubeLength) == true || flightTubeLength < 0.0) {
			ErrRep.replaceChild(lengthErr);
			retVal = false;
		}
	} else if (objname == "charge") {
		ionCharge = parseInt(InpObj.value);
		if (isNaN(ionCharge) == true) {
			ErrRep.replaceChild(chargeErr);
			retVal = false;
		}
		if ((ionCharge > 0.0 && acceleratingVoltage &&
				acceleratingVoltage < 0.0) ||
				(ionCharge < 0.0 && acceleratingVoltage && acceleratingVoltage > 0.0)) {
			acceleratingVoltage = (-1.0 * acceleratingVoltage);
			TOFform.elements["voltage"].value = acceleratingVoltage;
		}
	} else if (objname == "voltage") {
		acceleratingVoltage = parseFloat(InpObj.value);
		if (isNaN(acceleratingVoltage) == true) {
			ErrRep.replaceChild(voltageErr);
			retVal = false;
		}
		if ((acceleratingVoltage > 0.0 && ionCharge && ionCharge < 0.0) ||
				(acceleratingVoltage < 0.0 && ionCharge && ionCharge > 0.0)) {
			ionCharge = (-1.0 * ionCharge);
			TOFform.elements["charge"].value = ionCharge;
		}
	}
	if (retVal == false) {
		InpObj.value = "";
		TOFform.elements[objname].focus();
		return retVal;
	}
	if (typeof(ionMass) == "undefined" || ionMass < 0.0 ||
			typeof(flightTubeLength) == "undefined" || flightTubeLength < 0.0 ||
			typeof(ionCharge) == "undefined" ||
			typeof(acceleratingVoltage) == "undefined")
		return retVal;
	var flightTime = Math.sqrt((ionMass * protonMass *
			flightTubeLength * flightTubeLength) / (2 * ionCharge *
			unitCharge * acceleratingVoltage));
	var log10Time = Math.log(flightTime) / Math.LN10; // base 10 log of f time
	TOFform.elements["mantsec"].value =
			Math.round(1e4 * Math.pow(10, log10Time - Math.floor(log10Time)))/1e4;
	TOFform.elements["expsec"].value = Math.floor(log10Time);
	TOFform.elements["microsec"].value = Math.round(1e3 * flightTime * 1.0e6)/1e3;
	calculated = true;
	return retVal;
}

var mzErrSet = 0;
function calcParentMass(section) {
	var HYDROGEN_MASS = 1.0079;
	var z1nodfc, z2nodfc, pmassnodfc;
	var prefix, mz1, mz2;
	if (arguments.length == 1 && typeof(section) == "string")
		prefix = section;
	else
		prefix = "";
	mz1 = Number(document.getElementById(prefix + "mz1").value);
	mz2 = Number(document.getElementById(prefix + "mz2").value);
	if (mz1 < mz2) {
		var mzErr = document.getElementById(prefix + "mzerr");
		mzErr.appendChild(document.createTextNode("m/z-1 is less than m/z-2"));
		mzErrSet = 1;
		return;
	} else if (mzErrSet == 1) {
		var mzErr = document.getElementById(prefix + "mzerr");
		mzErr.removeChild(mzErr.firstChild);
		mzErrSet = 0;
	}
	var z1nod = document.getElementById(prefix + "z1");
	var z2nod = document.getElementById(prefix + "z2");
	var pmassnod = document.getElementById(prefix + "parentMass");
//	alert("mz1 = " + mz1 + "\nmz2 = " + mz2);
	var pmass = (HYDROGEN_MASS * (mz1 + mz2 - HYDROGEN_MASS) - mz1 * mz2) /
			(mz2 - mz1 - HYDROGEN_MASS);
	var z1 = Math.round(pmass / mz1);
	var z2 = Math.round(pmass / mz2);
	if (prefix.length > 0) {
		var adductMass = mz2 * z2 - mz1 * z1;
		var node = document.getElementById("adductMass");
		if (node.hasChildNodes() == true)
			node.replaceChild(document.createTextNode(adductMass), node.firstChild);
		else
			node.appendChild(document.createTextNode(adductMass));
	}
/*	alert("z1 = " + z1 + "\nz2 = " + z2 +
	   "\npmass = " + pmass); */
	if ((z1nodfc = z1nod.firstChild) != null)
		z1nodfc.replaceData(0, z1nodfc.length, z1);
	else
		z1nod.appendChild(document.createTextNode(z1));
	if ((z2nodfc = z2nod.firstChild) != null)
		z2nodfc.replaceData(0, z2nodfc.length, z2);
	else
		z2nod.appendChild(document.createTextNode(z2));
	var pmassString = String(Math.round(pmass * 100) / 100);
	if (pmassString.length > 8)
		pmassString = pmassString.substr(0, pmassString.indexOf(".") + 2);
	if (pmassnodfc = pmassnod.firstChild)
		pmassnodfc.replaceData(0, pmassnodfc.length, pmassString);
	else
		pmassnod.appendChild(document.createTextNode(pmassString));
	return;
}
</script>
<script src="/include/colorContrasting.js"></script>
<script src="/include/dom-xml.js"></script>
<script src="/include/css.js"></script>
<script src="/include/htmlTOCgenerator.js"></script>
<script src="/include/bigimage.js"></script>
<script>
function initPage() {
/*
	var htmlTOCobj = new HtmlToc(document.body, "main-text");
	htmlTOCobj.setTocOptions(USE_HIDE);
	htmlTOCobj.generateTOC();
*/
	document.getElementById('ToF').reset();
}
</script>
<style>
 table.th-left th {text-align:left;}
 table.th-center td {text-align:center;}
 .cite {font:normal 83% Palatino,'Century Schoolbook',serif;}
p.thumbed-image-left {font:normal 70% 'Century Schoolbook',Century,Palatino,serif;
 float:left;margin:0em 1em 1em -5em;width:50%;color:blue;text-align:left;}
p.thumbed-image-right {font:normal 70% 'Century Schoolbook',Century,Palatino,serif;
 float:right;margin:0em -5em 1em 1em;width:50%;color:blue;text-align:right;}
p.thumbed-image-left img, p.thumbed-image-right img {width:100%;cursor:pointer;}
</style>
</head>
<body onload="initPage();">

<p id="title">
Principles of Mass Spectrometry

<div class="outerblock">

<hr>
<!-- putTOChere -->

<!--  INTRODUCTION -->
<h1>Introduction</h1>
<p>
Mass spectrometry is the methodology of determining the mass of molecules or
other substances. This is done by ionizing the molecule in one of dozens of
known ways, placing it in the gas phase, and then detecting the ion. The
accurate mass measurement is done on an instrument calibrated with molecules of
known mass across the range of measurement.
<p>
This document discusses the three essential components of the mass spectrometer:
<ol>
<li>the <b>source</b> which produces <i>gas phase</i> ionized molecules
<li>the <b>mass analyzer</b> which achieves a <i>separation</i>
ionized molecules by which the mass can be determined in an analytical mode
<li>the <b>detector</b> which must sense the ions for the counting circuitry
</ol>
<p>

This document also discusses the data produced by the instrument, <em>the mass
spectrum</em>, and then why the instrument is used in proteomics, chiefly for 
protein identification and quantitation.


<!--  The MS Source  -->

<h1>The Source: Production of Gas Phase Ions</h1>
<p class="cite">
[Images in this section taken from de Hoffmann &amp; Stroobant, <i>Mass
Spectrometry: Principles and Applications</i>, 3rd Ed.]
<p>
The source is responsible for essentially two things:
<ol>
<li>Its most important function clearly is to produced ionized molecules (ions)
<li>It must put those ions in the gas phase so that they can be pulled into the
vacuum of the mass spectrometer as a beam of ions
</ol>
<p>
There are numerous technologies which achieve this goal in mass spectrometry,
and they are classed in various ways.
<ul>
<li>Environment
 <ul>
 <li>Vacuum:  the molecules are volatilized (put into the gas phase) prior to
their ionization.  This is true for ionizations that use electron beams (<q>cathode
rays</q>) directly or indirectly as in electron and chemical ionization
 <li>Atmospheric Pressure:  there are numerous techniques in which ionization
occurs at atmospheric pressure rather than in a vacuum, and then the gas phase
ions (in the atmospheric pressure) are brought into the mass spectrometer
vacuum using both the vacuum (negative) pressure and electrified ion beam guiding
metallic elements
 </ul>
<li>Energy
 <ul>
 <li><b>High</b> energy is used to achieve a <q>hard</q> ionization often by a
  collision with a (charged) particle: for instance, an electron or maybe heavy
  unreactive atom like a noble gas atom xenon or maybe argon).
 <li><b>Low</b> energy is used to achieve a <q>soft</q> ionization involving no
  kinetic energy (collision) at all, but rather the chemical potential of a
  reagent, such as putting in formic acid to protonate biomolecules
 <li>Temperature (thermal energy) might also help to achieve ionization for
 certain molecules whose characteristics are influenced by such parameters
 </ul>
</ul>
<p>
The molecules to be ionized will be presented in different phases: in solution,
complexed as solids, or they may be first put into the gas phase before
ionization. The table below is a partial list of the methods for ionization of
liquid-based (dissolved) molecules:

<p>
<b><q>Beam</q>-based Ionizations</b>.  It should be considered that any means
by which energy can be imparted to the atom, atoms, molecule, or molecules
under investigation can and will be exploited.  For electromagnetic radiation-based
(photon particle-based) ionizations, from gamma to radio waves will be used.
For molecules, which are multiple atoms bonded together in which the molecule
has multiple energy <q>partitions</q> that include translation, rotational,
vibrational, and electronic, and in which the energy spectrum is utilized to
affect one or more particular partitions, the spectrum can be selectively
exploited to achieve a particular energy input in order to selectively fragment
a molecule to obtain structural information.  Particle-based ionizations
would clearly include low and high energy electrons, but possibly other
molecules too (fast atom bombardment, alpha particles, etc.).

<table>
<caption>Liquid-Based Ionizations</caption>
<col style="width:10em;"><col style="text-align:center;">
<col><col><col style="text-align:center;">
<tr><th>Technique <th>Flow Rate
  <th>Information Produced <th>Sample Type  <th>Usual Mass Range
<tr><td>thermospray<br>(TSP, TSI)
 <td>1.0-2.0&nbsp;ml/min
 <td>MH<sup>+</sup>,(M-H)<sup>&minus;</sup>
 <td>polar water soluble, e.g. drugs, metabolites
 <td>&lt;1000
<tr><td>plasmaspray<br>(PSP)
 <td>0.5-2.0&nbsp;ml/min
 <td>MH<sup>+</sup>,(M-H)<sup>&minus;</sup>, some fragmentation
 <td>compounds less polar than for TSP
 <td>&lt;1000
<tr><td>liquid nebulized<br>(LINC)
 <td>0.2-1.0&nbsp;ml/min
 <td>M&bull;<sup>+</sup> + fragmentation<br>EI-type spectra
 <td>non-polar, e.g. fatty acids, pesticides
 <td>&lt;1000
<tr><td>atmospheric pressure chemical ionization<br>(APcI)
 <td>0.2-2.0&nbsp;ml/min
 <td>MH<sup>+</sup>, (M-H)<sup>&minus;</sup>, and some fragmentation
 <td>polar, e.g. pesticides, azo dyes, drugs
 <td>&lt;1000
<tr><td>dynamic fast atom/ion bombardment<br>(FAB/FIB)
 <td>1-10&nbsp;&mu;l/min
 <td>M&bull;<sup>+</sup> + fragmentation<br>EI-type spectra
 <td>non-polar, e.g. fatty acids, pesticides
 <td>&lt;1000
<tr><td>electrospray<br>(ESI)
 <td>1-1000 &micro;l/min
 <td>multiply charge state ions in both positive and negative modes
 <td>polar compounds, particularly large biomolecules
 <td>
</table>

<h2>Electron (Impact) Ionization (EI)</h2>
<p class="cite">
[Source: Chapter A3 <q>Electron Ionization</q> <i>Back to Basics</i>,
Micromass UK Ltd. pp&nbsp;41-54.]
<p>
Electrons are accelerated in an electric field of 70 Volts and acquire a
kinetic energy that is 70 electron-volts (eV). In passing close to or through
the target molecule (M), the electrons (e<sup>&minus;</sup>) cause the ejection
of an electron from the molecule to form a <b>radical cation</b>
(M&bull;<sup>+</sup>):

<p style="text-align:center;" class="eqnserif">
 e<sup>&minus;</sup> + M &rarr; M&bull;<sup>+</sup> + 2e<sup>&minus;</sup>
<p>
The reaction is sometimes called <i>electron impact</i>, although there is no
real impact. However, the energy imparted in 70 eV can be such the molecule
will fragment into ions of smallers species (A<sup>+</sup>, B<sup>+</sup>, etc.).
The fragmentation is characterstic, and is thus a fingerprint. With some
molecules, the energy of the electron and the electon itself is absorbed,
yielding a radical-anion (M&bull;<sup>&minus;</sup>).
<p>
<img src="imagesMassSpectrometryHTML/ElectronIonization.png"
  style="float:right;margin-left:1em;margin-right:-5em;"
  alt="Electron Ionization Source Configuration">
As shown in the figure at the right, the ion source is usually constructed of a
block composed a &lsquo;box&rsquo; more or less gas-tight. A heated filament is
used to produce electrons to ionize sample passing by, and these are
accelerated through a 70 volt potential. The electrons are then directed into a
beam by a magnetic field. This beam then interacts with gas-phase sample
molecules. A set of focusing and extraction plates then takes ionized molecules
from the source into the analyzer.

<h2>Chemical Ionization (CI)</h2>
<p>
With chemical ionization, the molecule (M) to be ionized is first volatilized
(nebulized, evaporated) into a stream with another gas called the <i>reagent
gas</i> (R). If the molecule is already gaseous, it is essentially injected
into the reagent gas. The reagent gas is present in much larger amounts than
the gaseous presence of target molecules.
<p>
The mixture of the reagent gas with gaseous target molecules then enters the
chamber where there is an electron beam just as for electron ionization
described above; in fact, the construction of the source is very much the same.
But the electron beam strikes the reagent gas molecules rather than the target,
largely because the concentration of reagent gas is present in orders of
magnitude far larger than the target molecule gas. This leads to ionization of
the reagent gas molecules. There is a rapid buildup of ionized and somewhat
energetic reagent gas molecules, and these in turn come into contact with
target gas molecules. This contact or interaction typically results in the
ionization of the target molecules. Ionized molecules are brought into the mass
spectrometer beam. This can include ionized reagent gas molecules or their
byproducts, but with special filters or gates at certain voltages, these
unwanted ions can be removed from the beam, basically by pulling them out in
ways that do not affect the target molecules.
<p>
The essential scheme of chemical ionization is:
<p style="text-align:center;" class="eqnserif">
  R (g) + e<sup>&minus;</sup> &rarr; R<sup>&bull;+</sup> (g) + 2 e<sup>&minus;</sup><br>
  R<sup>&bull;+</sup> (g) + R&prime;H (g) &rarr;  R (g) + R&prime;H<sup>+</sup> (g) <br>
  M (g) + R&prime;H<sup>+</sup> &rarr; MH<sup>+</sup> (g) + R&prime; (g)
<p>
<img src="imagesMassSpectrometryHTML/ChemicalIonization.png"
  style="float:right;margin-left:1em;margin-right:-5em;"
  alt="Chemical Ionization Source Configuration with EI mode assembly">
Notice the detail of the ionization by reagent gas.  In the scheme, the reagent
gas becomes the radical cation instead of the target molecule.  This then finds
a hydrogen on another reagent gas molecule (labeled with a prime character);
the radical cation abstracts an electron from R&prime;H to make it a cation.
This finds it way to a target molecule, ionizing it by protonation.
<p>
Note that in this scheme, the reagent gas R is essentially a catalyst:  all
forms of R (ionized or in excited state) are ideally returned to the ground state
with no change in structure.  However, it is possible for the R molecule in the
excited state to break down or react with other excited R molecules and perhaps
form covalently larger molecules.  Since the reagent gas is basically replenished
in a stream mixture with target molecules before entering the cathode ray chamber,
this never becomes a problem.
<p>
The only difference between electron ionization and chemical ionization is
essentially the use of an intermediate, the reagent gas. The scheme shows that 
electrons do not impart their significant energy directly to the target 
molecule M. In fact, it has been shown that this energy is not just sufficient 
to cause ionization (knocking out an electrode), but rather it can actually 
cause sometimes unwanted fragmentation of the target molecule. When this was 
observed, the use of the reagent gas (chemical ionization) was developed as a 
means of <q>softening the blow</q> the prevent fragmentation.  This chemical
ionization is basically <i>indirect</i> electron ionization.  For this reason
an ion source was developed (see figure at right above) that can function either
as EI and CI via a mode switch.
<p>
<img src="imagesMassSpectrometryHTML/EI-vs-CI.png"
  style="float:left;margin-right:2em;margin-left:-5em;"
  alt="chemical versus electron ionization">
If EI can cause target molecule fragmentation instead of mere ionization, then
why discuss it...why does it exist?  Firstly for most molecules, the fragmentation
is not the only resulting pathway:  some of the intact molecules show up on the
spectrum, indicating only ionization (as a radical cation) occurred.  Secondly
and quite importantly, when molecules fragment under EI, they produce
fragment ions (usually called product ions, occasionally child ions) of sizes
(<i>m/z</i> values) and relative intensities that are quite characteristic of
the intact molecule.  In other words, the EI spectra of a molecule, with all the
fragment product ions, is very much like a fingerprint (identifying pattern) for
the molecule, usually unique to it.  The ions produced can have their <i>m/z</i>
and intensity values entered input to a query interface to a spectrum library,
with certain tolerances (error windows) set, and matches can be produced in a
hit list.
<p>
EI is often used in combination with CI (it is easy enough just to switch modes)
in order to get the intact mass at least.  Note that CI too produce fragments of
the target.  Using different reagent gases can help to affect the reduction of
fragmentation.
<p>
Reagent gases typically used are methane [CH<sub>4</sub>], hydrogen
[H<sub>2</sub>], butane [C<sub>4</sub>H<sub>10</sub>], ammonia [NH<sub>3</sub>],
and methanol [CH<sub>3</sub>OH]
<p>
Both EI and CI would be used in a GC-MS instrument configured for doing (small
molecule) metabolite profiling in metabolomics studies.  The GC detector, either
a thermal conductivity device with exit flow interfaced to the EI/CI of the
MS, or a post-GC column split flow to a more sensitive flame ionization detector
(the other split to the MS), would do the quantiation of the metabolite, and the
MS would do the identification.

<h2>Thermospray and Plasmaspray</h2>
<p class="cite">
[Source: Chapter B4 <q>Thermospray and Plasmaspray Interface,</q> <i>Back to
Basics</i>, Micromass UK Ltd.]


<h2>Fast Atom Bombardment/Liquid Secondary Ion Mass Spectrometry (FAB/LSI-MS)</h2>
<p class="cite">
[Source:  Chapter B6 <q>Dynamic FAB LSIMS Interface,</q> <i>Back to Basics</i>,
Micromass UK Ltd.]
<p>
Solutions which are the eluates of a liquid chromatography process can be
interfaced to a mass spectrometer via a technique that accelerates heavy
ionized atoms into the solution, causing a desorption of solvent and solute
molecules into the gas phase, the mixture of which is molecules with a positive,
negative, and zero charge. The desorbed ions can then be accelerated into an
ion beam by setting up the magnitude and polarity of an extraction plate. The
bombardment, and therefore the desorption, is continuous rather than pulsed. If
the solvent (matrix) is of low volatility, desorption often involves positive
and negative ionic forms of the solvent molecules.
<p>
Typically there is little fragmentation, and a solute of mass M within a matrix
of mass S will produce species [M+H]<sup>+</sup> or [M-H]<sup>&minus;</sup> as
well as [S+H]<sup>+</sup> or [S-H]<sup>&minus;</sup>. The solution is usually
set at the ambient (room) temperature; heating would add vibrational and
rotational energy to the ion, and so thermally labile substances (peptides,
proteins, nucleosides, sugars) can be analyzed without structural
disintegration. As liquid chromatography is typically used to separate
involatile, thermally labile substances, this type of ionization is suitable.

<h3>The Fast Atom Gun</h3>
<p>
Fast atoms are produced and shot from a gun aimed at the solution. The
production fast atoms is rather complex. Xenon (Xe) is the usual atom, but fast
heavy ions of cesium (Cs<sup>+</sup>) are also used. To produce the fast atoms
of Xe, the following reactions occur:
<ol>
<li><b>Ionization</b>:  Xe + e<sup>&minus;</sup> &rarr; Xe<sup>+</sup> (slow) + 2 e<sup>&minus;</sup>
<li><b>Acceleration</b>:  Xe<sup>+</sup> (slow) &rarr; Xe<sup>+</sup> (fast)
<li><b>Charge Exchange</b>:  Xe<sup>+</sup> (fast) + Xe (slow)
		&rarr; Xe (fast) + Xe<sup>+</sup> (slow)
</ol>
<p>
The first reaction uses a beam of electrons to ionize Xe, and the Xe ions are
then accelerated within the system. These ions strike slow moving Xe atoms,
accelerating them by a transfer of momentum or a transfer a charge, such that
the fast moving unionized atom is formed. Ions and atoms moving in the
direction of the gun nozzle are are filtered to remove the ions so that only
the unionized atoms emerge from the gun nozzle.

<h3>The Target and the Matrix (Solvent)</h3>
<p>
Usually the solution of interest is flowed past a metal region defining the
target area. If this solution is flowing, this makes the technique <q>
dynamic</q>; if the solution is stagnant, the technique is <q>static.</q>
<p>
Liquid chromatography solvents are usually water, methanol, and acetonitrile.
The problem is that these solvents are too volatile under the high vacuum of a
mass spectrometer interface. Evaporation would lead to solute insolubility and
blocked flow, as well as the inability to desorb using fast atoms. The typical
technique is to create an admixture of a high boiling point solvent is
postcolumn flow with the eluate. The admixture usually creates a matrix whose
composition is 10% of the highly boiling point material. In rare cases, this
high boiling point matrix can be added to the chromatographic solvent without
affecting separation.
<p>
One dynamic FAB probe that has been described has a 50 or 75 &micro;m silica
capillary fitting to a compression fitting on a probe holder with an insulator,
and into a vacuum lock. The capillary continues down the probe shaft and ends
flush with a probe tip composed of a copper surface. Flow rates are set to less
than 5 ml/min: this balances evaporation and production of ions. Solvent and
other matrix elements have masses that are suppressed from the spectral
recording.
<p>
Another dynamic FAB probe makes use of a screen and wick assembly.
A capillary HPLC eluate has a flow of 1-10 &micro;l/min



<h2>Electrospray Ionization (ESI)</h2>
<p>
Electrospray ionization (ESI) is designed to prepare liquid phase
analytes and gas phase ions.  This transition occurs in three stages.
<ol>
<li><b>Formation of charged droplets</b>.  The capillary is charged
to a high voltage as the liquid flows out the end, creating highly
charged droplets that are suspended in blowing gas.  Sample solution enters
the spray chamber through a grounded needle (<i>nebulizer</i>).
<li><b>Solvent evaporation and droplet fission</b>.  Heated nebulizing
gas enters the spray chamber concentrically through a tube that surrounds
the capillary or solution needle.  Solvent
quickly evaporates causing the droplet to decrease in size,
and bringing charged molecules in close proximity to one another.
Because like charges repel, this droplet size decrease accelerates
the break-up of the droplet to even smaller droplets.
<li><b>Formation of gas phase ions.</b>  When a droplet falls to
a radius of &lt; 10 nm (100 &Aring;), the ionized molecule escapes
from the droplet into the gas phase.  The point at which
a droplet size decreases to the point of droplet fission is called the
<i>Rayleigh limit</i> and the fission is called <i>Coulomb explosion</i>.
Ion evaporation occurs when charge density reached approximately
10<sup>8</sup> V/cm<sup>3</sup>.
</ol>
<p>
<b>Cone gas</b>.  A counterflow gas serves to dry the ions that are still
part of the solvent.  This desolvation cone gas is usually nitrogen and
is also heated.  As the droplet disappears, the surface charges are brought
closer together and the ions are focused as a result.
<p>
Studies of the signal response against the concentration of analyte
show that signal is linear over 2-3 orders of magnitude from about
0.1 to 10 &mu;M.  Beyond 10 &mu;M, the increase in signal is minor.
This is possibly related to inhibition of the release of ions from
the surface, either because of a repelling surface layer or because
of formation of a solid residue.  This phenomenon is known as
<i>ion suppression</i>.
<p>
In addition to ion suppression seen as <i>surface competition</i>
in which electrolytes have the same charge as the analyte and compete
with it for gas-phase release, there is also suppression
caused by <i>charge competition</i>, in which electrolyte ions remove
charge from analyte ions such as by proton transfer.
<p>
The relative sensitivity of ions is a reflection of their surface
activity.  The relative sensitivity coefficient <i>k</i> are shown:

<table>
<col style="width:8em;"><col style="width:3em;text-align:center;">
<col style="width:8em;"><col style="width:3em;text-align:center;">
<col style="width:8em;"><col style="width:3em;text-align:center;">
<tr><th>Ion  <th><i>k</i>	<th>Ion  <th><i>k</i>  <th>Ion  <th><i>k</i>
<tr><td>Cs<sup>+</sup>                  <td>1.0
    <td>Ni<sup>2+</sup>(tripyridyl)<sub>2</sub><td>5
    <td>morphine<sup>+</sup>            <td>3

<tr><td>K<sup>+</sup>                   <td>1.0
	 <td>tetraethylammonium              <td>5
	 <td>codeine<sup>+</sup>             <td>5

<tr><td>NH<sub>4</sub><sup>+</sup>      <td>1.3
    <td>tetrapropylammonium             <td>8
	 <td>heroin<sup>+</sup>              <td>6

<tr><td>Li<sup>+</sup>                  <td>1.6
    <td>tetrabutylammonium              <td>2
	 <td>cocaine<sup>+</sup>             <td>10

<tr><td>Na<sup>+</sup>                  <td>1.6
    <td>tetrapentylammonium             <td>14
    <td rowspan="3" colspan="2">
<tr><td rowspan="2" colspan="2">
    <td>triheptylammonium               <td>10
<tr><td>triundecylammonium              <td>10
<tr><td colspan="6" style="font-size:83%;">Ref: P. Kebarle, Y Ho in
<i>Electrospray Ionization Mass Spectrometry:  Fundamentals,
Instrumentation &amp; Applications</i>, R. B. Cole, <i>ed.</i>
</table>
<p>
The table shows that sensitivity increases with the smaller size of the ion,
although ions with increasing surface activity are also more sensitive.

<p>
Several factors affect successful ESI:
<ul>
<li>Analyte concentration:  the higher the analyte concentration, the
better chance of detection
<li>Matrix content:  it is best to avoid the use of a matrix that ionizes
in the same way as the analyte and which competes with the analyte for
detection and analysis
<li>Analyte surface activity: this has to due with the ability of the
analyte to emerge from a droplet to the gas phase
<li>Flow rate:  the optimal flow rate enables detectability without
compromising resolution; in general response falls with increasing flow rate
</ul>
<p>Compounds that are already charged in solution are likely to provide
better sensitivity, but ionization is not necessary.  Hydroen bonding
to salts can provide the charge.
<p>
Molecules with good surface-active properties also provide sensitivity,
but it must be able to associate with a salt to be ionized.
<p>
Molecules that are ionized and have surface-active properties will likely
be highly sensitive.
<p>
The Z-spray source has the following advantages and disadvantages:

<table>
<tr><th>Advantages                     <th>Disadvantages
<tr><td>molecular weight confirmation  <td>limited fragmentation
<tr><td>ionic &amp; easily protonated  <td>limited to relatively low flow rates
<tr><td>good sensitivity               <td>Z-spray rugged enough to handle
                                           non-volatile salts, but highly
														 ionic modifiers reduce sensitivity
<tr><td>sample MW can exceed instrument range if molecule can be multiply charged
 <td>
<tr><td>suitable for capillary LC <td>
</table>
<p>Optimal flow rates tend to be 100-250 &mu;l/min, but may be less if analyte
concentration is high.
<p> Ionization of low MW molecules is usually that they are singly charged,
typically by acquiring a proton.  Biomolecules of low MW that are basic,
particularly amines, will usually gain a proton and should be analyzed in
positive mode.  Extreme care should be taken when using ESI with ammonium,
potassium, or sodium ions present since they form adducts. These adducts
increase the mass by the adduct mass: NH<sub>4</sub><sup>+</sup> (M + 18),
Na<sup>+</sup>(M + 23), and K<sup>+</sup> (M + 39);  this typically causes
problems with interpretation of mass spectra, and analysts should always take
caution with this.
<p>
High MW biomolecules and polymers form multiply charged species. Fragmentation
of these species rarely occurs.
<p>
The sensitivity&mdash;ability to detect an ion&mdash;will invariably depend
on the analyte and its matrix.
<p>
<b>Induced ionization</b>.  Although typically conditions are used that
cause ionization to occur in the mobile phase, before nebulization, ionization
can be induced by droplet evaporation leading to a high charge surface or
volume density.

<h3>In-Source Fragmentation</h3>
<p>Fragmentation usually does not occur in the source.  Increasing the voltage of
the sample cone however can cause fragmentation.  Normally the cone is used to
extract ions from the atmospheric region and then into the first vacuum.  Ions
accelerate to the rear toward the skimmer.  The region between the cone and the
skimmer is under vacuum, although the pressure is high with respect the
analyzing regions.  If the voltage is increased in the cone, the higher
cone-skimmer voltage difference results in a larger acceleration, and with
solvent vapor and nitrogen present, the collisions might cause fragmentation.
The energies are such that only weak sigma bonds between C-N and C-O are likely
to be broken however.

<h3>Starting The Electrospray</h3>
<p>There seems to be more of an art than a science to starting the flow from the
capillary for electrospray.  Available to the user are several independent
variables (i.e. variables the user can control, or inputs), and a couple of
variables to monitor to see how well the setup is, or the outputs.

<h4>The Outputs:  What To Look For in Optimizing Electrospray</h4>
<p>
One of the first and more important variables (outputs) to monitor in
electrospray is a visual one. For the Nanoflow-type sources, which include the
Nano-LC and the Nanovial (glass capillary), the stream must be set up with the
proper distance of the tip of the capillary from the sample cone.
<p>
<img src="imagesMassSpectrometryHTML/ESI-examples.png"
   style="float:right;margin-left:1em;margin-right:-5em;width:70%;"
alt="An example of good electrospray and a couple of examples of bad electrospray">
The figure at right shows a panel of three images. The green-boxed image shows
a typical example of the Nano-LC as seen through the magnifying scope from the
top of the source. The user should absolutely be using the magnifying scope
with the source chamber illuminated according to the setup provided by
Waters/Micromass. Also useful is for the operator to put his hand over the
transparent chamber at the 3 o'clock position (as the operator faces the source)
to prevent stray light especially daylight entering the laboratory from an
unshaded window from preventing viewing of the electrospray. The figure belows
shows a panel boxed in green that shows an electrospray that appeared to be
ideal. It also shows two panels boxed in red that give unsteady or poor ion
beams. Note that the green-boxed image shows a stream that actually looks like
a thin fabric thread slightly wavering as if in the wind, and that it bends
toward the sample cone but passes beyond the sample cone.
<p>
The red-boxed images in contrast represent examples of poor electrospray. In
the middle image red-boxed example, the thread-like stream is actually seen entering
the mouth of the sample cone. While this might seem logical since more of the
analyte enters directly, the problem is that the analyte is trapped in the
liquid phase in aerosolized droplets. Ionized analyte molecules can only become
part of a detectable ion beam if they have been put into the gas phase.
<p>
In the red-boxed image at the far right, two electrospray tips are shown. The
tip on the left shows drops being sprayed as visibly large drops from the tip
of the capillary. The desolvation gas is essentially blowing the drops formed
at the tip off the tip. But the thread-like small stream is not formed because
either the capillary voltage is too low (or non-existent), or if voltage is
sufficiently high based on experience, it could be that the sample is not
prepared correctly; perhaps there is no ion-promoting agent in the solvent,
such as trifluoroacetic acid (TFA) in a sufficient concentration (0.1% v/v). In
the red-boxed example on the right, drops are seen to form on the outer wall of
the capillary, behind the tip, as if emerging from a wall defect itself. This
is probably not the case. The wall was wetted on the surface probably as a
result of flow started in the capillary without desolvation gas or capillary
voltage being on, and so a backward flow from the capillary tip leads to
droplet formation in this way.

<p>
<img src="imagesMassSpectrometryHTML/CapillaryView.png"
style="float:left;margin-left:-5em;margin-right:1em;width:60%;"
 alt="Magnified view of capillary in source block">
In the figure at left a magnified view of a capillary in the source block is
shown, in perspective, and in cross-section. In the cross-section, the liquid
stream is shown with ionized (or perhaps ionizable) particles shown. There
location and motion in the liquid phase shows them to be randomly distributed,
diffusing in this way, and obeying the second law of thermodynamics. The
desolvation gas channels around the capillary are apparent.
<p>
When the capillary voltage is applied in the positive ion mode and liquid flow
is started, the source block is given a high negative voltage (seen in <span
style="border:1px solid black;"><b>A</b></span> in the figure below). The
liquid within the capillary becomes polarized in the sense that positive ions
(or positive portion of a dipole in the case of polarizable molecules) move
toward the wall of the capillary, while negative ions (or the negative part of
a dipole) are crowded within the radial center of the long axis of the
capillary. <p> The stream that emerges from the tip forms a cone, called the <i>
Taylor cone</i> (seen in <span style="border:1px solid black;"><b>B</b></span>).
 The region of the Taylor cone region is magnified in <span style="border:1px
solid black;"><b>C</b></span>). The wall of the cone is merely a continuation
of ordered arrangement of the ions from the source electric field, whose
magnitude diminishes as the stream moves away from the block. Droplets begin to
form from the cone tip, both from the effect of a high temperature, high flow
desolvation gas and from a fission created by charge repulsion of ions at the
surface. Being on the surface of droplets, positive ions are more likely than
negative ions to enter the gas phase, promoted by charge repulsion and by the
typical the typical forces that increase their kinetic energy sufficient to
make the phase transition (hot nitrogen desolvation gas hitting either analyte
or solvent molecules in gas provides the bulk of this kinetic energy.
</div> <!-- class=outerblock -->
<p style="text-align:center;">
<img src="imagesMassSpectrometryHTML/ESI-detail.png"
  alt="Magnified view of Taylor cone formation and droplet fission in the electrospray">
<div class="outerblock">
As the stream (electrospray) gets further from the tip, the electrospray is
seen as a thread blowing in the wind <span style="border:1px solid black;"><b>
D</b></span>). Droplet size gets smaller due to droplet fission promoted by
charge repulsion and due to liquid-to-gas phase transition of solvent molecules
from the desolvation gas. The figures in row <span style="border:1px solid
black;"><b>E</b></span>) show a magnified view of the droplets. In <span style=
"border:1px solid black;"><b>F</b></span>), a more magnfied view of the process,
 droplet formation is shown as a <q>budding</q> process from the surface.
Budding is quite distinct from fission however: the latter is a splitting into
two (equal volumes), while in the former, smaller droplets emerge from a larger
droplet to form <q>daughter</q> droplets.
<p>
Whether ever smaller droplet formation is a fission or budding process is
not entirely clear.  The important point is to promote the transition of the
desired ion (positive or negative) into the gas phase, which electrospray does
well.


<h2>Atmosphere-Pressure Chemical Ionization (APCI)</h2>
<p> APcI is ideal for neutral to polar compounds that are labile.  Few adduct
ions are formed on the analyte molecule, and flow rates from the
<i>uncharged</i> capillary can be set to between 0.2 and 2.0 ml/min without
splitting the flow. APcI can be used to confirm MW and has good sensitivity.
Because it is gentle, there is limited information from fragmentation, and there
is higher noise than signal from chemicals when ranged in the lower masses. APcI
is not appropriate for analytes that are involatile.
<p>
APcI occurs in two stages:
<ol>
<li>flash vaporization:  the capillary and tube are set up to maximize
vaporization of solvent and volatile analyte prior to ionization by the
corona discharge needle
<li>gas-phase chemical ionization:  the capillary is not electrified, and
so ionization does not occur in the liquid stream;  the corona discharge
needle is set at 2000-5000 V to produce a plasma at the sample cone
</ol>
<p>As the sample emerges from the capillary, it encounters an electrified
<i>corona discharge needle</i>.  This typically ionizes the solvent molecules.
These ions then react with the analyte molecules to form ionic clusters. The
ionic charge is then transferred to the analyte molecule.
<p>
With APcI, only singly charged ions form and there is little fragmentation.
Adducts particularly from the solvent can form, and the ion mass will reflect
that.  Relative to ESI, there are fewer matrix effects, e.g., spectra are not
affected by minor changes in most variables such as type and strength of buffer.

<h2>Atmosphere-Pressure Photo-Ionization (APPI)</h2>
<p> The LC eluent is passed through a nebulizing needle to create a fine spray,
and the spray is passed through a heated ceramic tube where the eluent droplets
are fully vaporized.  The resulting gas/vapor mixture then passes through a
photon beam of a krypton (Kr) lamp to ionize the eluent sample molecules. Ions
then pass through a conductive capillary and series of skimmers and lenses
before going to the mass analyzer.
<p>
The major difference between APCI and APPI is that a lamp is subsititued for
the corona needle.
<p>
APPI often uses a <b>dopant</b> or mobile phase modifier to assist with
photo-ionization.


<!--  MALDI    -->

<h2>Matrix-Assisted Laser Desorption Ionization (MALDI)</h2>
<p>MALDI-based MS is important in the analysis of a number of samples
of interest to biological chemists.  In decreasing order of importance:
<ul>
<li>peptides and proteins
<li>synthetic polymers
<li>oligonucleotides
<li>lipids
<li>inorganic molecules
</ul>
<p>Generally MALDI MS is preferred because it is less complicated to prepare
samples, and the spectra are not complicated by multiply charged forms as with
ESI MS.

<h3>Source Ionization</h3>
<p>MALDI works in two stages.  <b>Primary ion formation</b> occurs with ions
generated by the initial laser pulse, which are typically the matrix molecules,
as they absorb the laser light.  <b>Secondary ion formation</b> occurs when the
excited, ionized matrix molecules react with adjacent molecules, in
matrix-matrix and matrix-analyte reactions.
<p> Ionization of the analyte actually involves a great many reactions.
<b>Protonation</b> is generally the most common, but cationization and the
formation of free radicals is usual too.
<p> The nitrogen (N<sub>2</sub>) laser provides a photon have 3.68 eV, and the
ionization potential of the matrix molecule is usually 8-9 eV, the energy
required to raise a &pi;-bonding electron to its anti-bonding orbital. In order
to achieve this ionization, multiple photons must be absorbed by a single matrix
molecule, and the probability of this is quite low. Ionization could occur by a
theory of <b>energy pooling</b>:  multiple excited (but unionized) converge on a
single molecule to cause its ionization.
<p> Another theory is <b>excited-state proton transfer</b> (ESPT).  In this
acidity of a matrix molecule is increased by a single photon putting it into the
excited state.  A typical molecule that works by ESPT is salicylic acid
(<i>o</i>-hydroxybenzoic acid):  a photon would be sufficient to move the proton
from the ortho-hydroxy group to the carbonyl oxygen of the carboxyl group,
thereby producing a zwitter ionic excited state in the molecule.  However, ESPT
is not thought to affect the ionization of carboxylic acids (the proton on the
COOH group itself), which typically make up the MALDI matrix.  Moreover,
substances with strong evidence of ESPT-type reactivity do not make good MALDI
matrices.  There is also no evidence that common MALDI matrices work by ESPT.
<p> In another theory of <b>disproportionation reactions</b>, the following
mechanism is asserted:

<p style="margin-left:5%;">
2 <i>M</i> &nbsp;&nbsp;&mdash;&mdash;&gt;
<span style="font-size:60%;margin-left:-4em;vertical-align:1em;"><i>nh&nu;</i></span>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(<i>MM</i>)* &mdash;&gt;
(<i>M &minus; H</i>)<sup>&minus;</sup> +
(<i>M</i> + <i>H</i>)<sup>+</sup>
<p style="margin-left:5%;margin-top:0.5em;">
2 <i>M</i> &nbsp;&nbsp;&mdash;&mdash;&gt;
<span style="font-size:60%;margin-left:-4em;vertical-align:1em;"><i>nh&nu;</i></span>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(<i>MM</i>)* &mdash;&gt;
<i>M</i> <sup>&minus;</sup> + <i>M</i> <sup>+</sup>
<p> In this mechanism, interaction between excited matrixi molecules is
required, and the interaction can involve proton and/or electron transfer to
produce ionized states.  So far, little experimental evidence has been produced
to indicate this is a credible mechanism.

<h4>Preformed Ions</h4>
<p>Preformed ions occur in compounds that are ionic by nature in their milieu
(many salts), in metal-complexed analytes, and in pre-protonated compounds.
Large molecules with the potential for ion preformation have a lower
electrostatic barrier.  All preformed ions have the ability to desorp (enter the
gas phase from the solid) easily, assuming that ion preformation occurs in the
matrix-analyte sample.  Thermal desorption experiments produce gas phase ions
(thus ions are present without laser-based activation).  But this still poses a
problem in proving that ionization does not occur in the gas phase and is
preformed.  Another problem as well as that preformed negative ions are not
common.

<h4>Other Mechanisms</h4>
<p><b>Thermal ionization</b> is seen with matrices in two phases subjected to
high temperatures.  <b>Pressure pulse ionization</b> has been shown when
mechanical stressed is transduced into electronic excitation. <b>Spallation</b>
is ion and electron emission that occurs from mechanical failure causing
piezoelectric effects.  Finally, thermal excitation resulting from high pressure
shockwaves induced by the laser is another possible mechanism.

<h4>Secondary Ionization</h4>

<h4>Fragmentation/Decay of Metastable Ion</h4>
<p>Ionized species show a significant degree of metastable decay, in which
associated neutral molecules (water, ammonia) and amino acid side chains and
random cleavage at peptide bonds occurs.  Linear mode MALDI cannot resolve
precursor and metastable decay ions since they were already accelerated at the
source and arrive at the same time.  However in reflectron mode there will be
kinetic energy differences in the ion mirror.
<p> Interpretation of spectra takes time and practice.  Fragmentation is usually
residue-specific.  Fragmentation along the peptide backbone is of type
<i>a<sub>n</sub></i>, <i>b<sub>n</sub></i>, and <i>y<sub>n</sub></i>, according
to the Biemann nomenclature. Respectively, this fragmentation is between the
alpha-carbon and carboxyl carbon from the N-terminus, the carboxyl carbon and
alpha-nitrogen (amide bond) from the N-terminus, and the carboxyl carbon and
alpha-nitrogen from the C-terminus.  The systematic approach looks for amino
acid immonium (H<sub>2</sub>N<sup>+</sup>=CHR) and internal acyl ions
(<i>b<sub>n</sub></i>-like ions) and starting N- and/or C-termini chains. Look
for differences of 28u (<i>b<sub>n</sub></i> &minus; <i>a<sub>n</sub></i>) and
doublets with mass difference of 17u (e.g., <i>a<sub>n</sub></i> and
<i>a<sub>n</sub></i> &minus; 17). The information is combined to deduce the
primary sequence.  Derivitization of the analyte may help in providing more
information for sequence deduction.
<p>Alternative to Reflected ToF-MS include Delayed Extraction Linear ToF-MS. DE
Lin ToF-MS means that the ions accelerated in the beam were fragmented in or at
the source (during desportion or ion formation) rather than fragmented after
acceleration.  The peak pattern shows <i>c<sub>n</sub></i>,
<i>y<sub>n</sub></i>, and <i>y<sub>n</sub></i>&minus;15, different from the
post-source decay pattern of <i>a<sub>n</sub></i>, <i>b<sub>n</sub></i>, and
<i>y<sub>n</sub></i> above.  The mass differences of <i>y<sub>n</sub></i> less
15 Da point to <i>z<sub>n</sub></i> + 2 ions, but additional studies are
required to prove that these are <i>z<sub>n</sub></i>+2 ions.

<h3>Sample Preparation</h3>
<p>Sample preparation is critical to good MALDI-ToF.  Factors that affect good
analysis include the following:
<ul>
<li>Matrix and analyte concentration, as well as the matrix-to-analyte
mole ratio
<li>Matrix choice
<li>Analyte <q>history</q>:  exposure to strong ionic detergents,
formic acid
<li>Analyte chemistry:  hydrophobicity and hydrophilicity
</ul>

<h4>Finding the Matrix</h4>
<p>A MALDI matrix right for the analyte being studied should have the
following qualities:
<ul>
<li>should embed and isolate analytes during co-crystallization
<li>be soluble in solvents compatible with the analyte
<li>be vacuum-stable
<li>absorb radiation at the wavelength of the laser being used
<li>must cause co-desorption of analyte during irradiation
<li>must promote analyte ionization
</ul>
<p>Carboxylic acids are generally thought to be the best since their labile
protons can protonate even neutral molecules in a plume with the accelerating
voltage in positive ion mode.  Acidic environments may not be good for some
analytes however, especially if tertiary structure denaturation is to be
avoided.  Those analytes not easily protonated are <q>cationized</q>
instead, usually be adding small amount of salt (alkali metals, Cu, Ag) to the
analyte.  Oligonucleotides are more easily deprotonated and are thus analyzed in
negative ion mode.

<table>
<tr><th>Matrix           <th>Uses/Applications
<tr><td>2,5-dihydroxybenzoic acid (DHB)
    <td>Peptides, proteins, lipids, and oligosaccarides
<tr><td>3,5-Dimethoxy-4-hydroxy-<i>trans</i>-cinnamic acid (sinapinic acid)
    <td>Peptides, proteins, and glycoproteins (usually with
	 mass&nbsp;&gt;&nbsp;10&nbsp;kDa)
<tr><td>&alpha;-cyano-4-hydroxy-<i>trans</i>-cinnamic acid (CHCA)
   <td>Peptides, proteins, lipids, and oligonucleotides (usually with
mass&nbsp;&lt;&nbsp;10&nbsp;kDa)
<tr><td>2-(4-Hydroxyphenylazo)benzoic acid (HABA)
   <td>Peptides, proteins (usually with
mass&nbsp;&lt;&nbsp;10&nbsp;kDa)
<tr><td>3-Hydroxypicolinic Acid (HPA)  <td>
<tr><td>2,4,6-Trihydroxyacetophenone (THAP) <td>
<tr><td>Succinic acid
   <td rowspan="3">Peptides, proteins; used with UV laser
<tr><td>Ferulic acid
<tr><td>Caffeic acid
<tr><td>Glycerol
    <td rowspan="2">liquid matrices
<tr><td>4-Nitroaniline
</table>
<p> MALDI's problems include low shot-to-shot reproducibility, short sample
lifetime, and strong dependence on preparation method.  Hillenkamp and Karas,
who introduced MALDI ToF-MS, used the original simple prep method of the dried
droplet.

<h5><q>Hot</q> and <q>Cold</q> Matrices</h5>

<h4>Solid Matrix</h4>

<h5>Dried Droplet</h5>

<p>Aqueous matrix solution mixed with analyte and dried to form an analyte-doped
matrix co-crystal.  Washing helps remove nonvolatiles that contribute to ion
suppression.  Tolerates presence of salts/buffers well with limits.  Best choice
where samples contain more than one protein/peptide species.  The original
technique when MALDI introduced in 1988 by Hillenkamp &amp; Karas.

<h5>Vacuum Drying</h5>

<p>A variation of the dried-droplet method, sample is applied and rapidly dried
in a vacuum chamber.  This helps to reduce analyte-matrix cocrystal size and
increases crystal homogeneity, with improved spot-to-spot reproducibility and
minimization of need to search for sweet spots.  Smaller crystal formation
enables thinner samples and improved mass accuracy.  Extensive alkali metal
cation adduction is obtained, which can be substantially reduced with cold water
washing of spotted crystals.

<h5>Crushed Crystal</h5>

<p>In this method, crystals can be grown in the presence of high concentrations
of non-volatile solvents (i.e., glycerol, 6 M urea, DMSO, etc.) without the need
to purify.  Films are more uniform than dried-droplet deposits, with respect to
ion production and spot-to-spot reproducibility.  A disadvantage is the effort
(additional steps) necessary to produce sample. <p> Strict control of
particulate formation is necessary to eliminate presence of undissolved matrix
crystals that shift nucleation from metal surface to bulk of droplet.

<h5>Fast Evaporation</h5>
<p>This technique was used to improve resolution and mass accuracy. A drop of
matrix is applied to sample stage and the solvent allowed to evaporate.  The
analyte is then dropped on to the stage and allowed to dry.
<p> <b>Thin Film</b>.  This is probably another name of the fast-evaporation
method.  CHCA is applied in a thin layer by spotting an acetone solution of
CHCA.  Analyte is then layered on top of the dried matrix.
<p> Another variation of this method is of Xiang and Beavis in which matrix
crystals are deposited, then crushed with glass slide or appropriate tool, then
adding analyte-matrix mix.  Spectral improvement is then achieved by washing
with cold water:  put a 0.5-1.0 &mu;l cold water (or 0.1% TFA, or 10% acetic
acid) on the sample spot, let stand for ca. 30 seconds, then remove by
pipetting, shaking, or blowing off.

<h5>Overlayer</h5>
<p>Combines the crushed-crystal and fast-evaporation methods. A first layer of
small crystals is created by fast solvent evaporation, which is followed by
deposition of a mixture of matrix and analyte on top of the base crystal layer.
The difference is the top layer: in fast evaporation, the top layer is analyte
only, whereas in the overlayer, it is an analyte-matrix mix.  This is believed
to improve results, particularly for peptides and proteins.

<h5>Sandwich</h5>
<p>A combination of fast-evaporation and overlayer.  A sample droplet is applied
to the top of a matrix-only bottom layer applied by fast-evaporation.  This is
followed by a matrix-only top layering but in a non-volatile (traditional)
solvent.

<h5>Spin-Coating</h5>
<p>In the original use of this method, large volumes (3-10 &mu;l) of premixed
sample is deposited on 1&quot; diameter stainless steel and quartz plates.
Sample was very homogeneous in response across the sample target.

<h5>Electrospray</h5>
<p>A small amount of matrix-analyte mix is electrosprayed from a high
voltage-biased (3000-5000 V) stainless steel or glass capillary on to a grounded
metal sample plate, mounted 0.5-3.0 cm away from the capillary tip.  The
deposition shows a homogeneous layer of equal-sized matrix microcrystals and
analyte molecules.

<h5>Quick &amp; Dirty</h5>
<p>A drop of matrix is added to the top of a drop of analyte (at 0.1-10 mM). A
pipette tip then mixes the two solutions, and then the mix is dried under air or
nitrogen stream.

<h5>Matrix-Precoated Targets</h5>
<p>A single drop of undiluted sample is applied to a precoated target. An
advantage is that MALDI samples can be directly interfaced to the output of LC
and CE columns.  Thin-layer matrix-precoats membranes are being tried.  Membrane
bases include nylon, PVDF, nitrocellulose, anion- and cation-modified
celluloses, regenerated cellulose or regenerated cellulose dialysis membrane.

<h4>Liquid Matrix</h4>

<h5>Chemical Liquid</h5>
<p>The appropriate molar ratio of analyte is dissolved in liquid matrix, usually
using a solvent that is evaporated before introducing sample into the mass
spectrometer vacuum system.

<h5>Particle-Doped (Two-Phase) Liquid</h5>
<p>There are matrices that exist in two phases.  Some exist as a suspension
of fine particles (fine metal or graphite with a diameter of 1 &mu;m or
less) in solvent and are mixed with analyte and some binder.  The solvent
evaporates and a <q>paste</q> is then applied to the ion source.
Small particulates can sputter from the sample, so consideration should
be taken about contamination of vacuum pumps (perhaps pre-exposure to
another vacuum pump with a trap).
<p>
The fine particles absorb the laser energy and promote desorption.
The liquid molecules provide the ionization mechanism.  There are many
applications of particulates and liquids for proteins, oligosaccharides,
synthetic polymers, and dyes with MW &gt; 10,000.

<h5>Chemical-Doped Liquid</h5>
<p>An organic compound especially absorbent for the laser frequency is
added to the liquid medium.  Traditional MALDI matrix materials are
used very often.

<h4>Special Preparations</h4>

<h5>Solid Supports</h5>
<p>The best known solid support is porous silicon (a desorption/ionization
off silicon, or DIOS).  The concept of a particle-type matrix applies
here.  The concept of an <q>active</q> MALDI support is of interest
here, while limitations of the use of this feature include limited mass
range, poor reproducibility in fabrication, and the serious effect of
surface contamination.

<h5>MALDI on 2D-gels</h5>
<p>Methods are being developed for the analysis of complex samples, including
detecting proteomes in intact cells, MALDI imaging of biological thin
sections, and the direct analysis of 2D gels on the plate (apart from
spot cutting and preparation on standard target plates).

<h5>Insoluble Samples</h5>
<p>For insoluble samples, mix finely ground sample of analyte and sample.
This usually works well for insoluble materials, including high MW
synthetic polymers.  Rather thick samples are produced by this method,
with samples that are long-lived and last thousands of laser shots.


<h3>Matrix Preparation</h3>
<p>Matrix should be at least washed, and at best, recrystallized.
This <a href="ProteomicsReference.html#CHCArecryst">section</a>
of a reference in proteomics describes a procedure for washing or
recrystallizing common matrices such as CHCA.


<h3>Laser Sources Used in MALDI</h3>

<table>
<tr><th rowspan="2">Laser  <th rowspan="2">Wavelength
	<th colspan="2">Photon Energy <th rowspan="2">Pulse Width
<tr><th>(kcal/mol)  <th>(eV)
<tr><td>Nitrogen      <td>337 nm  <td>85   <td>3.68 <td>&lt; 1 ns to few ns
<tr><td>Nd:YAG &mu;3  <td>355 nm  <td>80   <td>3.49 <td>usu. 5 ns
<tr><td>Nd:YAG &mu;4  <td>266 nm  <td>107  <td>4.66 <td>usu. 5 ns
<tr><td>Excimer (XeCl)<td>308 nm  <td>93   <td>4.02 <td>usu. 25 ns
<tr><td>Excimer (KrF) <td>248 nm  <td>115  <td>5.00 <td>usu. 25 ns
<tr><td>Excimer (ArF) <td>193 nm  <td>148  <td>6.42 <td>usu. 15 ns
<tr><td>Er:YAG        <td>2.97 &mu;m <td>9.7 <td>0.42 <td>85 ns
<tr><td>CO<sub>2</sub><td>10.6 &mu;m <td>2.7 <td>0.12 <td>100 ns + 1 &mu;s tail
</table>
<p> Most lasers are nitrogen 337 nm devices.  Alternatives included frequency
tripled Nd:YAG lasers at 355 nm.  Infrared lasers include Er:YAG devices at 2940
nm, but because of cost, IR MALDI is usually an option.  IR MALDI is softer than
UV MALDI, good for analysis of oligonucleotide and noncovalently bound
biomacromolecular complexes. There are a limited choice of IR MALDI matrices,
and also a greater penetration depth and vaporization depth per shot, which
causes a shorter sample lifetime, and a somewhat lower sensitivity.



<!--  =============================================================
   Gas Phase Ion Physics and Chemistry
=================================================================== -->

<h1>Physics and Chemistry of Gas(eous) Phase Ion</h1>
<p>
Some concepts are introduced which are important in understanding how the
gas phase ion can be studied and manipulated.  These include concepts 
with respect to fragmentation of the ion, whether intentional or
spontaneous/natural.  The energetics (study of energy states) of the ion
can be mathematically analyzed for the purpose of predicting how causing
or observing energy states in a precuror/parent can lead to formation of 
(wanted) product/child ions.

<h2 id="frag-gas-ion">Fragmentation of The Gas Phase Ion</h2>
<p>[Source: E. de Hoffmann &amp; V. Stroobant 2007, <i>Mass Spectrometry:
Principles and Applications</i>, 3rd Ed., Ch. 7: <q>Fragmentation Reactions</q>]

<p>
When energy is transferred to isolated molecules, concepts typical in
thermodynamics lose all meaning:  temperature, the statistical distribution
of energy.  Within microseconds, the reaction is over, with fragmentation
either occurring or not.  If there is a purposeful collision, it happens without
subsequent collisions.  Everything is unimolecular.  Rearrangements are observed
only if followed by a fragmentation.
<p>
Many fragmentations are zero- or low-energy reverse reactions, such as 
recombination of radical with cation.  The <i>principle of micro-reversibility</i>
states that the fragmentation activation energy is equal to endothermicity.
In the energy/reaction-coordinate plot, it is seen that 
&Delta;<i>H</i> = <i>E</i><sub>a</sub> &minus; <i>E</i><sub>r</sub>, where
<i>E</i><sub>r</sub> is the activation energy component that refers to the 
energy recovered when recombination of radical and cation occurs, while
<i>E</i><sub>a</sub> is the total activation energy (Arrhenius energy).  But
since <i>E</i><sub>r</sub> &lt;&lt; <i>E</i><sub>a</sub>, thus 
&Delta;<i>H</i> &#8776; <i>E</i><sub>a</sub>.  With thermal reactions, reaction
rate depends on activation-deactivation equlibrium and decomposition rate
of activated complex.  The classic reaction descriptions is:
<div class="eqnserif">
M + N &#8652; M<sup>*</sup> + N
<p>
M<sup>*</sup> &rarr; P<sub>1</sub> + P<sub>2</sub>
</div>
<p>
<i>M</i> is the molecule of interest undergoing fragmentation, and <i>N</i>
supplies the energy as a particle that does not participate in any chemistry
and which does not itself fragment (usually an inert gas).  In the first
reversible reaction, the forward <i>k</i><sub>1</sub> and reverse <i>k</i><sub>&minus;1</sub>
rate constants apply.  In the second reacation, rate constant <i>k</i><sub>2</sub>
is a value difficult to determine for thermal reactions in the condensed phase
since rates measured are a combination of the three reaction directions.
In the gas phase <i>k</i><sub>2</sub> can be measured directly, although this
value cannot be applied to liquid phase reactions as the reacting species is
unique to the gas phase and could not be solvated.

<h3>Time Periods In Stages of Electron Impact Ionization As Example</h3>
<p>
The time periods involved in successful electron ionization have been studied.
Generally the ionization occurs within 10<sup>&minus;16</sup> seconds.  After
10<sup>&minus;14</sup> to 10<sup>&minus;10</sup> seconds, one vibration of the molecule/atom
might have occurred, depending on many factors.  Within 10<sup>&minus;8</sup>
seconds, either fragmentation occurs or ion returns to ground state, giving
off radiation.  This radiation will be within the UV or light for electronic
excitations, and rotational and vibrational excitation radiations are in the 
IR range, occurring within 1 ms to a few seconds.  By 10<sup>&minus;6</sup> seconds, the ion is leaving the source.
Before 10<sup>&minus;4</sup> seconds, further fragmentation might occur for
ions in a metastable state.  Within the period, the ion usually arrives at the
detector and has passed through the analyzer.
<p>
It is important to note that the notion of temperature and statistical
distribution of energy in the unimolecular fragmetnation in a mass spectrometer
loses all meaning: in a few microseconds, the process is over and the reaction
has occurred or not occurred.  Reaction products are <i>kinetically</i>
based.  The principle of micro-reversibility applies:  fragmentation
activation energy is equal to endothermicity.  The energy-reaction coordinate
plot will show a curve with <i>E<sub>a</sub></i> as the energy maximum and
products being a radical and cation with an &Delta;<i>H</i> (difference between
products and reactants) and a recombination activation energy <i>E<sub>r</sub></i>
approximately close to the <i>E<sub>a</sub></i> for the reverse reaction.
With thermal-based reactions, the reaction rate depends on
equilibrium between between activation and deactivation and the decomposition
rate of the activated complex to products:
<p class="eqnserif" style="text-align:left;margin-left:2em;">
M + N &harr; M* + N  [&harr; implies a forward <i>k</i><sub>1</sub> and
reverse <i>k</i><sub>&minus;1</sub> constants] <br>
M* &rarr; P<sub>1</sub> + P<sub>2</sub> [&rarr; implies forward <i>k</i><sub>2</sub>
constant]
<p>
Values of <i>k</i><sub>2</sub> are hard to deduce for thermal reactions in
condensed phase as the rates are combinations of the three directions (reactions).
In the gas phase, <i>k</i><sub>2</sub> is measurable directly (although cannot
be assumed for liquid phase, as reacting species in gas phase not solvated).

<h3 id="rrkm">Quasi-Equilibrium Theory</h3>
<p>
In 1952 two almost identical theories were proposed to explain unimolecular
reactions in gas phase.  Rosenstock et al proposed <i>quasi-equilibrium theory</i> (QET).  This
theory and applies to MS, and another deals with unimolecular dissociation of
neutral molecules as proposed by Rice, Rampsberger, Kassel, and Marcus (RRKM).
<p>
Four suppositions are made with QET or RRKM:
<ol>
<li>Time period for ion dissociation is long compared to the time period for
ion formation and excitation.
<li>Rate of dissociation is much slower (low) compared to the rate of the
redistribution of the excitation energy among all the internal modes of the
ion
<li>The ion achieves an internal equilibrium condition where energy is
distributed with equal probability among all internal modes.  For an ion
with <i>N</i> non-linear atoms, there are 3<i>N</i> &minus; 6 vibrational
degrees of freedom:  thus with large ions, collision yield decrease quickly
(it is inversely proportional to ion mass).
<li>the dissociation products are a result of a series of competitive and
consecutive reactions
</ol>
<p>
These theories assumes that all atomic/molecular energy compartments
(translational, rotational, vibrational, electronic) are not interdependent 
(but independent). It also applies the physics that nuclear energy states can 
be described by classical mechanics and not quantum mechanics, although some 
corrections are applied using QM.
<p>
The first postulate of QET indicates that all microscopic states are equally
probable:  all degrees of freedom participate in energy distribution with
same probability.  The second QET postulate states that movements are on 
a multidimensional surface and a border surface defines the separation between
reactants and products.  The surface is crossed with complete irreversibility
(in one direction): once transition state is crossed, the products form and
remain.
<p>
Suppose an electron is accelerated in a 10 V potential field and acquires 
kinetic energy with <i>v</i> = 1.88 &times; 10<sup>8</sup> cm/s.  In 
10<sup>&minus;16</sup> seconds, it moves 1.88 &times; 10<sup>&minus;8</sup> cm = 
1.88 &Aring;.  The time represents the time of interaction of ionizing electron 
with molecule M.  Ionization must occur within that time (verified 
experimentally).  The ionization is a <i>vertical transition</i> on a <b>energy 
vs. interatomic distance</b> plot.  Since a molecular vibration of any kind 
occurs in a minimum time period of 10<sup>&minus;14</sup> seconds, the 
ionization occurs without any change in interatomic distances.  In the plot are 
three Morse curves, corresponding to M, M<sup>&bull;+</sup> (ground state 
radical or ionized M), and M<sup>&bull;+*</sup> (excited state radical or 
ionized M).  In each plot are the usual energy wells (minimum) for the 
interatomic distance with the lowest energy, and in the wells are quantum states 
that are valid internal energy levels.  A vertical transition (ionization) is 
the movement from one plot to the higher (M &rarr; M<sup>&bull;+</sup>) and it 
is an energy transition that is greater than the transition resulting from an 
adiabatic process.  The energy of the accelerated impact electrons can leave a 
vertical transition in a higher internal energy state.  Molecular ions in an 
excited electronic state (M<sup>&bull;+*</sup>) either fragment or return to 
ground state with photon emission.  Ions leaving the source form stable or 
metastable ions with a lower internal energy in the plot well.
<p>
Fragmentations do not really occur by excited electronic states.  Rather the 
excited electronic states must internally convert to creating excited 
vibrational energy states.
<p>
The internal conversion process is an energy distribution through all degrees of 
freedom and in statistical fashion.  <q>Fast exchange</q> of internal energy is 
done in which all degrees of freedom of the <em>same</em> electronic state is 
described by all degrees of freedom of <em>all</em> electronic states. The 
process of <i>excited</i> &rarr; <i>ground</i> state for the electronic 
component occurs before any fragmentation, since fragmentation relies upon 
excitation of the rotational and vibration (rovibrational) states. The 
redistribution of total energy occurs must faster than the reaction 
(fragmentation).   When an oscillator contains more energy than a certain 
threshold <i>E</i><sub>0</sub>, the oscillator becomes the reaction coordinate 
and M undergoes a fragmentation, and reaction occurs faster if internal energy 
is distributed over the oscillator such that it is &gt; threshold 
<i>E</i><sub>0</sub>. The time period for the redistribution actually occurs 
before a few vibrations occurs (about 10<sup>&minus;10</sup> seconds). Since the 
ion emerges from the MS source at about 10<sup>&minus;7</sup> seconds, 
fragmentation is complete before the ion emerges from source.
<p>
A rate constant can be defined for the reaction:
<p class="eqnserif">
<i>k</i>(<i>E</i>) = (1/<i>h</i>)(<i>Z</i><sup>&Dagger;</sup>/<i>Z</i><sup>*</sup>)
[<i>P</i><sup>&Dagger;</sup>(<i>E &minus; E</i><sub>0</sub>)/<i>&rho;<sub>E</sub></i>]
<p>
<i>h</i> is Planck's constant, <i>Z</i> is the partition function for 
adiabatic degrees of freedom, &Dagger; refers to activated complexes, *
refers to active ionic species, <i>P</i><sup>&Dagger;</sup>(<i>E</i> &minus;
<i>E</i><sub>0</sub>) are total count of states corresponding to activated
complexes between energies of zero and <i>E &minus; E</i><sub>0</sub>, and
<i>&rho;<sub>E</sub></i> represent density of states of ion at energy <i>E</i>.
<p class="eqnserif">
<i>k</i>(<i>E</i>) = <i>&nu;</i> [(<i>E &minus; 
E</i><sub>0</sub>) / <i>E</i>]<sup><i>n</i> &minus; 1</sup>
<p>
where <i>n</i> = number of vibrational degrees of freedom, <i>&nu;</i> =
frequency factor, <i>E</i> is internal energy of ion, and <i>E</i><sub>0</sub>
= transition energy.  Both <i>&nu;</i> and <i>E</i><sub>0</sub> are dependent
upon structural modifications; <i>n</i> will vary from one molecule to another.
The factor <i>&nu;</i> is an inverse function of activated complex steric 
requirements:
<p class="eqnserif">
<i>&nu;</i> = (<i>E &minus; E</i><sub>0</sub>) / <i>h</i> &times; 
<i>e</i><sup>&Delta;<i>S</i>*/<i>R</i></sup>
<p>
&Delta;<i>S</i>* has a lower value when the activated complex is highly ordered.
<p>
From a fragmentation spectrum of <i>n</i>-butanol, we see intensities at
<i>m/z</i> 31 and 56, which suggests the following fragmentations:
<p class="eqnserif">
CH<sub>3</sub>CH<sub>2</sub>CH<sub>2</sub>CH<sub>2</sub>O<sup>&bull;+</sup>H
(<i>m/z</i> 74) &rarr; 
CH<sub>3</sub>CH<sub>2</sub>CH<sub>2</sub><sup>&bull;</sup> +
CH<sub>2</sub>=O<sup>+</sup>H (<i>m/z</i> 31) <br>
CH<sub>3</sub>CH<sub>2</sub>CH<sub>2</sub>CH<sub>2</sub>O<sup>&bull;+</sup>H
(<i>m/z</i> 74) &rarr; 
CH<sub>2</sub><sup>&bull;</sup>CH<sub>2</sub>CH<sub>2</sub>CH<sub>2</sub><sup>+</sup> 
(<i>m/z</i> 56) + H<sub>2</sub>O  
<p>
The first reaction occurs through a transition state that can be from any
conformation (the <q>loose complex</q>).  The second reaction proceeds only 
through a single or definite conformation.  With the lower activation entropy 
of the second reaction, the growth of rate constant wrt energy should be greater for the first 
reaction.  But activation energy of the 2nd reaction is lower than the first
since bond cleavages are partially compensated for by new bond formation is
transition state.  The distribution of energy among itons is not as regular
as it is in thermal reactions.  A Warhaftig diagram plots two ordinates against
an abscissa.  A top half plot is of P vs IE, where P is proportion of ions
with an internal energy plotted by IE; it is thus a histogram with a usually
gaussian distribution.  The lower half plot is log <i>k</i>(<i>E</i>) vs IE.
In this half plot are three dotted lines representing limits:  one limit is
10<sup>6</sup> s<sup>&minus;1</sup> where average lifetime &gt; 10<sup>&minus;6</sup> s.
These ions reach the detector without any fragmentation.  With rate 
constants &gt; 10<sup>7</sup> s<sup>&minus;1</sup> (line 2 in graph), 
these fragment before leaving the source and so only the fragments are detected.
Those between these limits will fragment after leaving the source but before
detection.  Dotted line 3 in the graph is the intersection of the two rate
curves, where metastable ions are found.  To the left (lower IE), 
rearrangement is faster, and on the right, rearrangement is slower.
<p>
To ionize a neutral molecule requires a minimum energy called the <b>ionization
energy</b> (IE) or the <b>ionization potential</b> (IP).  IE can be determined
by MS instruments.  Analytical MS instruments provide an energy disperson too
wide for a precise deetermination.  Fragment ions appearing include and
energy that incorporates ionization energy, which is the activation energy
and the kinetic shift.  The energy of fragmentation is the <b>appearance
energy</b> (AE) or <b>appearance potential</b> (AP) and it = <i>E</i><sub>0</sub>.
Fragmentation should be observed in a constant period to properly fix
the energies involved.  An FTMS provides this ability.

<h3>Positive Ion Fragmentation Reactions</h3>
<p>
McLafferty explored this area quite extensively, and the molecular rearragements
and changes in conformations necessary to bring about fragmentations are
named after him.

<h4>Fragmentation of Odd-Electron Cations or Radical Cations (M<sup>&bull;+</sup>)</h4>
<p>
In the formation of odd-electron or radical (M<sup>&bull;+</sup>) cations
by electron (impact) ionization,  charge delocalizes over the molecule to find 
a stable energy state.  The lost electron in M is assumed to arise from the 
electron with lowest IE.  In order of electron-type ionization, the 
expectation is <i>n-</i> &gt; <i>&pi;-</i> &gt; <i>&sigma;</i>-type electrons.
Heteroatoms with weak ionization energies are expected to carry charge 
preferentially.
<p>
Cleavages are likely initiated by the site where positive charge is 
concentrated with attraction of an electron pair.  Electron pair movement
causes heterolytic clavage and charge site migration.  A radical site initiates
cleavage by its need for pairing.  An odd electron donates to form a new bond
where transfer of single electron induces homolytic cleavage with migration
of unpaired electron.
<h5>Direct Dissociation</h5>
<p>
With electron impact ionization, a <q>&sigma; fragmentation</q> results from 
an electron directly lost which is involved in &sigma; bond pairing:
<p class="eqnserif">
R-R&prime; + e<sup>&minus;</sup> &rarr; [R-R&prime;]<sup>&bull;+</sup> + 
2&nbsp;e<sup>&minus;</sup>
<p>
In the fragmentation, one keeps the cation and the other the radical:
<p class="eqnserif">
1. [R-R&prime;]<sup>&bull;+</sup> &rarr; R<sup>&bull;</sup> +  R&prime;<sup>+</sup>
<br>2. [R-R&prime;]<sup>&bull;+</sup> &rarr; 
R<sup>+</sup> +  R&prime;<sup>&bull;</sup>
<p>
According to Stevenson's rule, the radical having the highest IE will be 
produced when two charged fragments are in competition to produce a neutral
radical by electron attachment.  The other ion has a neutral radical with lower
ionization energy will hold its charge and be the observed fragment.
Look at the product spectrum of 2,2-dimethylheptane, which fragments to
tert-butyl and n-pentyl groups.  n-pentyl radical (<i>m/z</i> 71) 
has IE = 7.85 eV  while tert-butyl radical has IE = 6.7 eV.  The fragment
mass spectrum shows <i>m/z</i> 57 being base peak (100% rel. int), while
<i>m/z</i> 71 is 20% rel. int.  Thus the cation is stabilized better by
tert-butyl while the neutral radical is the <i>n</i>-pentyl group.
1-butene fragments into allyl (1-propenyl) [IE=8.1 eV] and methyl [IE=9.8 eV]
groups.  The spectrum shows that the base peak is at <i>m/z</i> 41, being
the allyl cation, while the peak at <i>m/z</i> 15 is barely detectable, which
would be the methyl cation.  Thus the allyl cation is more stabilized and
the methyl radical more favorable.  Both of these results consistent with
Stevenson rule.  This applies only to competitive formation of fragment ions:
further fragmentations may actually lead to fragments that confuse the rule.
<h5>Cleavage of Bond On Heteroatom</h5>
<p>
<i>Charge site-initiated reactions</i> in which attraction of an electron pair
from such a bond leading to induced cleavage are observed:
<p class="eqnserif">
R-CH<sub>2</sub>-Y<sup>&bull;+</sup>-R&prime; &rarr; R-CH<sub>2</sub><sup>+</sup>
+ <sup>&bull;</sup>Y-R&prime;
<p>
The mechanism seems to be direct dissociation assisted by inductive electron
withdrawal (to Y atom) because of electronegativity, but it occurs after
ionization.  If the IE of <sup>&bull;</sup>YR&prime; &lt; RCH<sub>2</sub><sup>&bull;</sup>
then the Stevenson rule would still apply, then the product <sup>+</sup>Y-R&prime;
could form, a special case of &sigma; bond cleavage.
<h5>Alpha Bond Cleavage</h5>
<p>
In a <i>radical site-initiated reaction</i>, the bond alpha to the radical
cation is broken with transfer of unpaired electron to form new bond on an
alpha atom.
<p class="eqnserif">
R-CH<sub>2</sub>-Y<sup>&bull;+</sup>-R&prime; &rarr; R<sup>&bull;</sup> +
CH<sub>2</sub>=Y<sup>+</sup>-R&prime;
<p>
The bond break is homolytic, with one electron going to one atom and the other
electron to the other atom, which subsequently combines with the lone electron
on the heteroatom to making a <i>&pi;</i> bond.  If Y is oxygen, a positively
charged oxygen is isoelectronic with nitrogen and has three covalent bonds.
<p>
When several competitive cleavages are possible, the loss is of radical with 
highest ionization energy usually.  If several alkyl changes can form product 
radical, the longest chain is favored.  Consider tert-butyl ethyl ether 
[CH<sub>3</sub>&ndash;CH<sub>2</sub>&ndash;O&ndash;C(CH<sub>3</sub>)<sub>3</sub>
]. Potential product cations are tert-butyl<sup>+</sup> (<i>m/z</i> 57) with 
ethanolic radical, isopropenyl ethyl ether cation 
[CH<sub>3</sub>&ndash;CH<sub>2</sub>&ndash;O<sup>+</sup>=C(CH<sub>3</sub>)&ndash;CH<sub>3</sub>] 
(<i>m/z</i> 87) with methyl radical (alpha cleavage).  The isopropenyl ethyl 
ether cation in the second reaction undergoes heteroatom cleavage to produce 
H&ndash;O<sup>+</sup>=C(CH<sub>3</sub>)&ndash;CH<sub>3</sub>] (<i>m/z</i> 59)
with ethene.  The mass spectrum shows a base peak of <i>m/z</i> 59, with
<i>m/z</i> 87 at 50% RI, <i>m/z</i> 57 at 40% RI, and <i>m/z</i> 102 (parent)
at &lt; 5% RI.  Results show that fragmentation is quite thorough, and follows
the alpha cleavage direction with half the product further fragmenting.
The Stevenson rule is observed (IE is higher for methyl radical than 
ethanolic radical).  Because of the high electronegativity of oxygen this
fragment can carry away the electron.  The ion at <i>m/z</i> 45 
(CH<sub>3</sub>CH<sub>2</sub>O<sup>+</sup>) does not appear in the spectrum.
In another example, phenylethyl ethyl ether 
(C<sub>6</sub>H<sub>5</sub>CH<sub>2</sub>CH<sub>2</sub>&ndash;O<sup>&bull;+</sup>&ndash;CH<sub>2</sub>CH<sub>3</sub>)
shows how &alpha; cleavage:  the spectrum shows <i>m/z</i> 91 as the base
peak ([C<sub>6</sub>H<sub>5</sub>CH<sub>2</sub>]<sup>+</sup>) while <i>m/z</i> 59
also appears at 80% relative intensity 
[CH<sub>2</sub>&ndash;O<sup>&bull;+</sup>&ndash;CH<sub>2</sub>CH<sub>3</sub>]<sup>+</sup>
<h5>Competition Between Cleavage of Adjacent and Alpha Bonds</h5>
<p>

When the heteroatom is a large atom, adjacent cleavage is more likely. Where 
there are neighboring atoms, the more electronegative one leads to cleavage of 
adjacent bond.  Alpha cleavage is more likely for electron donors, with the 
order Br, Cl &lt; R<sup>&bull;</sup>, &pi; bond, S, O &lt; N. Halogens therefore 
preferentially cause loss of radical X<sup>&bull;</sup> by cleavage of adjacent 
(rather than &alpha;) bond, whereas amines prefer &alpha; to adjacent bond 
cleavage.  Compare butylamine 
(CH<sub>3</sub>CH<sub>2</sub>CH<sub>2</sub>CH<sub>2</sub>N<sup>&bull;+</sup>H<sub>2</sub>) 
to butanethiol 
(CH<sub>3</sub>CH<sub>2</sub>CH<sub>2</sub>CH<sub>2</sub>S<sup>&bull;+</sup>H) 
radical cations.  The spectrum of butylamine radical cation is dominated by a 
peak at <i>m/z</i> 30, which would be CH<sub>2</sub>=N<sup>+</sup>H<sub>2</sub>, 
meaning the propyl radical had been lost 
([CH<sub>3</sub>CH<sub>2</sub>CH<sub>2</sub>]<sup>&bull;</sup>). In contrast, 
the butanethiol shows a base peak at <i>m/z</i> 57, 
[CH<sub>3</sub>CH<sub>2</sub>CH<sub>2</sub>CH<sub>2</sub>]<sup>+</sup> with 
<sup>&bull;</sup>SH lost, a peak <i>m/z</i> 47 at 90% rel. intensity, which is 
alpha bond-cleaved CH<sub>2</sub>=S<sup>+</sup>H with propyl radical 
[CH<sub>3</sub>CH<sub>2</sub>CH<sub>2</sub>]<sup>&bull;</sup> lost. There is 
also a peak <i>m/z</i> 90 corresponding to uncleaved butanethiol.

<h5>Radical Cation Fragmentation with Rearrangement</h5>
<p>
Numerous rearranements can occur and make spectrum interpretation difficult.
McLafferty rearrangements in contrast are well understood:  an H atom
is moved to a radical cation in which a 6-atom ring forms as intermediate.
This separates the atom bearing the radical from the atom bearing the cation
(a <i>distonic radical cation</i>).  Fragmentation follows, being either
radical site- or charge site-induced; the products are a neutral molecule
and new radial cation.  When there is no N atom, ions have an even mass
and are easily detected.
<p>
Consider 2-hexanone 
CH<sub>3</sub>CH<sub>2</sub>CH<sub>2</sub>CH<sub>2</sub>C(=O)CH<sub>3</sub>
spectrum.  When the radical cation forms on oxygen atom (O<sup>&bull;+</sup>),
the molecule can conform to a 6-atom in which the 5-methylene group (not
end or terminal methyl group) donates one of its H atoms to form a
6-atom ring, as required by McLafferty, into a bridge with oxygen:
CH<sub>3</sub>&ndash;CH&ndash;H---O<sup>&bull;+</sup>=C&ndash;.  The
&mdash;CH&ndash;H bond breaks, transferring the radical to the
methylene group, with the O atom retaining the cation and taking the H
atom to form a stabilized protonated carbonyl:
<p class="eqnserif">
CH<sub>3</sub>&ndash;CH<sup>&bull;</sup> + H---O<sup>+</sup>=C&ndash;
<p>
The cation then migrates to the carbonyl carbon, and from there, fragmentation
ensues, with products being one neutral and a new radical cation:
<p class="eqnserif">
CH<sub>3</sub>CH<sup>&bull;</sup>CH<sub>2</sub>CH<sub>2</sub>C<sup>+</sup>(&ndash;OH)CH<sub>3</sub>
&rarr;
<br>1. CH<sub>3</sub>CH<sup>+</sup>CH<sub>2</sub><sup>&bull;</sup> (<i>m/z</i> 42) + 
CH<sub>2</sub>=C(&ndash;OH)&ndash;CH<sub>3</sub>
<br>2. CH<sub>3</sub>CH=CH<sub>2</sub> + 
<sup>&bull;</sup>CH<sub>2</sub>&ndash;C<sup>+</sup>(&ndash;OH)&ndash;CH<sub>3</sub> (<i>m/z</i> 58)
<p>
The radical cation with the odd number of elections appears at an even 
<i>m/z</i> (58 Th) in agreeement with parity rules.  The most dominant peak
in the spectrum however is <i>m/z</i> 43, which corresponds to an radical 
site-induced alpha cleavage:  n-butyl radical (loss) + 
CH<sub>3</sub>&ndash;C<sup>+</sup>=O (<i>m/z</i> 43).

<h4>Fragmentation of Cation with Even Number of Electrons (M<sup>+</sup>)</h4>
<p>
In electron ionization, radical cations form in the source.  Fragmentation
can lead to one radical fragment and the other carrying the cation (with
even number of electrons), or by rearrangement and multiple steps, a neutral
and smaller radical (odd-electron) cation form.  If the latter, the mass
is even numbered when in absence of N atom.
<p>
Soft ionizations (FAB, TSP, CI, ESI, MALDI, APcI and many others), usually
produce even-number cations usually be adding or abstracting a proton.
The product spectra are more difficult to interpret in determining rearrangements
and fragmentation however, although the molecule is less energetic.
<p>
Consider structural isomers <i>cis</i>- and <i>trans</i>-1,4-cyclohexanediol. 
These have identical EI spectra.  But with chemical ionization (CI), the 
<i>trans</i> isomer yields [M+H-H<sub>2</sub>]<sup>+</sup> peak more intense 
than the <i>cis</i> isomer.  The mechanism appears to involve the proton 
bridging the two O atoms in the <i>cis</i> isomer which resists fragmentation, 
whereas protonation of the <i>trans</i> isomer actually creates an opportunity 
for S<sub>N</sub>1 displacement by the other oxygen atom, leading to a
bridged cation with loss of water.
<p>
In some cases, even-electron cations undergo homolytic cleavage to produce
radicals:
<p class="eqnserif">
M<sup>+</sup> &rarr; N<sup>&bull;+</sup> + <sup>&bull;</sup>R
<p>
It is improbable largely because the reaction is endothermic.  Thus fragmentation
largely happens with with generation of neutral and a smaller even-numbered
cation.  With odd-numbered cations (radical cations), two pathways emerge, 
one being even-numbered cation and odd-numbered neutral, and the other being
specially rearranged intermediates creating another radical cation and a 
neutral.   The same fragmentation rules apply to anion (negative ions), and
the products be as follows:
<ul class="eqnserif" style="text-align:left;margin-left:5em;">
<li>M<sup>&bull;+</sup> or M<sup>&bull;&minus;</sup> &rarr; even mass
<li>M<sup>+</sup> or M<sup>&minus;</sup> &rarr; odd mass
</ul>
<h4>Parity Rule Fragmentation</h4>
<p>
Most often cleavage occurs on bond adjacent to the charged site, with charge
migration to another atom in the product.  Protonated alcohols yield water
and radical cations as typical example:
<p class="eqnserif">
R&ndash;OH + H<sup>+</sup> &rarr; R&ndash;OH<sub>2</sub><sup>+</sup> &rarr;
R<sup>+</sup> + H<sub>2</sub>O
<p>
McLafferty proposed a classification regarding even-electron ions obeying a
parity rule:
<ol>
<li>Cleavage of bond with charge migration:
 <p class="eqnserif">
 CH<sub>3</sub>CH<sub>2</sub>&ndash;O<sup>+</sup>H<sub>2</sub> &rarr;
CH<sub>3</sub>CH<sub>2</sub><sup>+</sup> + H<sub>2</sub>O
<li>Cleavage of bond with cyclization and charge migration:
 <p class="eqnserif">
 CH<sub>3</sub>C(=O)&ndash;NH&ndash;CH<sub>2</sub>CH<sub>2</sub>CH<sub>2</sub>NH<sup>+</sup>=CR<sub>2</sub> 
&rarr; CH<sub>3</sub>C(=O<sup>+</sup>---)&ndash;NH&ndash;CH<sub>2</sub>CH<sub>2</sub>CH<sub>2</sub>---
 + HN=CR<sub>2</sub> 
<li>Cleavage of two bonds in cyclic ion with charge retention:
 <p class="eqnserif">
 benzyl cation (C<sub>6</sub>H<sub>7</sub><sup>+</sup>) &rarr;
   benzylene cation (C<sub>6</sub>H<sub>5</sub><sup>+</sup>) + H<sub>2</sub>
<li>Cleavage of two bonds with rearrangment and charge retention:
 <p class="eqnserif">
 H&ndash;CH<sub>2</sub>CH<sub>2</sub>OH<sup>+</sup>CH<sub>3</sub>
  &rarr;  CH<sub>2</sub>=CH<sub>2</sub> + CH<sub>3</sub>&ndash;O<sup>+</sup>H<sub>2</sub>
</ol>
<p>
The first two reaction types are more likely when there is no collision activation.
First type occurs when protonated site is less basic (see table below):

<table class="th-left td-center">
<caption>Products Ratio of <i>n</i>-Butyl-XR molecules</caption>
<tr><th>&ndash;XR <td>NH<sub>2</sub> <td>&phi; <td>SH <td>OH <td>I <td>Br <td>Cl
<tr><th>[(M+H)&minus;HXR]<sup>+</sup>/(M+H)<sup>+</sup>
 <td> 0.04 <td>0.08 <td>0.15 <td>19.5 <td>240 <td>&gt;250 <td>&gt;250
<tr><th>Proton Affinity (kcal/mol)
  <td>207 <td>183 <td>175 <td>164 <td>145 <td>141 <td>140
</table>
<p>
The ability for an intermediate to stabilize a cation is important too.
For butylamine, the ratio [(M+H)&minus;NH<sub>3</sub>]<sup>+</sup>/(M+H)<sup>+</sup>
is 0.4, 0.07, and 2.3 for <i>n</i>-, <i>sec</i>-, and <i>tert</i>-butyl
cation respectively.
<p>
The second type (type 2) is same reaction favored by possibility of cyclization
if there is heteroatom in position.  Latter provides anchimeric assistance to
expulsion of neutral.  Cyclohexanediols are good example:  &omega;-amino
alcohols H<sub>2</sub>N(CH<sub>2</sub>)<sub><i>n</i></sub>OH will cyclize
when protonated to lose water H<sub>2</sub>O and a protonated secondary
amine.  The [(M+H)&minus;H<sub>2</sub>O]<sup>+</sup>/[M+H]<sup>+</sup> ratio
values from <i>n</i> = 2 to 6 are 0.42, 0.21, 0.20, 0.15., and 0.07.  Alternatively
the NH<sub>3</sub> neutral loss is a possibility, but only when <i>n</i> = 4
and 5 (when <i>n</i> = 5, the product is <a
href="http://en.wikipedia.org/wiki/Tetrahydropyran">tetrahydropyran</a>).
<p>
Type 3 and 4 reactions occur with four-center molecular rearrangements.
For example
<p class="eqnserif">
C<sub>6</sub>H<sub>5</sub>CH<sub>2</sub>NH<sup>+</sup>(CH<sub>3</sub>)<sub>2</sub>
&rarr; C<sub>6</sub>H<sub>5</sub>CH=NH<sup>+</sup>CH<sub>3</sub> + CH<sub>4</sub>
<p>
In another, protonation of ketone leads to dehydration to alkene:
<p class="eqnserif">
R-C(=O<sup>+</sup>H)-CH<sub>2</sub>-R&prime; &rarr;
R-C(-O<sup>+</sup>H<sub>2</sub>)=CH-R&prime; &rarr;
R-C<sup>+</sup>=CH-R&prime; + H<sub>2</sub>O
<p>
with proton migration resulting
simultaneously with alkene bonding.  In chemical ionization a ketone can lose
water but this does not occur with EI (which makes sense).
Esters will yield ion corresponding to protonated acid:
<p class="eqnserif">
R-C(-OH)=O<sup>+</sup>-CH<sub>2</sub>CH<sub>2</sub>CH<sub>3</sub> &rarr;
R-C(-OH)=O<sup>+</sup>H  + CH<sub>2</sub>=CH-CH<sub>3</sub>
<p>
In the spectrum, propyl acetate shows a protonated acetic acid as the base peak,
while methylbutyrarte shows a [RCOOH+H]<sup>+</sup> which is hardly detectable;
the proton derives from the alcohol.  Deuterium labeling shows the proton comes
25% from the carbon closest to ester oxygen, 25% from 2nd carbon away, and 50%
from the methyl group furthest away.
<p>
Iminium ions having short chains will fragment to an ion-neutral complex:
<p class="eqnserif">
H<sub>2</sub>C=NH<sup>+</sup>CH<sub>2</sub>CH<sub>2</sub>CH<sub>3</sub>
&rarr; H<sub>2</sub>C=NH...<sup>+</sup>CH<sub>2</sub>CH<sub>2</sub>CH<sub>3</sub>
&rarr; <br><br>
H<sub>2</sub>C=NH  CH<sub>3</sub>-C<sup>+</sup>H-CH<sub>3</sub>
&rarr; H<sub>2</sub>C=N<sup>+</sup>H<sub>2</sub>  CH<sub>2</sub>=CH-CH<sub>3</sub>
<p>
Note the 1,2-proton shift to stabilize the carbocation in the intermediate, but
eventually to the iminium since that is much more stable.  A longer alkyl chain
leads to McLafferty rearrangement with even-electron ion; they hydrogen will
originate from &gamma; position (3rd carbon from iminium), with dimethylamine
product and the 1,2-alkene.  This occurs with other heteroatoms (oxygen in
oxonium, sulfur in sulfonium).

<h4>Fragmentation Not Obeying Parity Rule</h4>
<p>
Reactions not obeying parity rules with even-numbered electron ions are more
rare and difficult to predict.  Seen in molecules with extended &pi; systems,
and often having complex rearragements.  For example tropylium cation
[C<sub>7</sub>H<sub>7</sub>]<sup>+</sup> is an aromatic species that decays
to the radical cation C<sub>6</sub>H<sub>2</sub><sup>+</sup> although the main
path is a loss of acetylene.  Radical stability will determine products formed:
nitroso compounds ealsly lead to loss of stable radical N<sup>&bull;</sup>=O.

<h3>Negative Ion Fragmentation Reactions</h3>
<p>
Study of negative ions was of a lower priority since EI typically produces
cations and not anions.  Techniques to enhance production of negative ions
are widespread.  Most anions are of even-numbered type [M-H]<sup>&minus;</sup>
with loss of proton off a neutral.

<h4>Even-Electron Anion Mechanisms</h4>
<p>
With this, homolytic radical formation occurs.  Consider the treatment of
acetone with strong base to deprotonate it:
<p class="eqnserif">
C<sup>&minus;</sup>H<sub>2</sub>C(=O)CH<sub>3</sub> &rarr;
C<sup>&bull;</sup>H<sub>2</sub>C(=O)C<sup>&minus;</sup>H<sub>2</sub> + H&bull;
<p class="eqnserif">
C<sub>6</sub>H<sub>5</sub>C<sup>&minus;</sup>HOCH<sub>3</sub> &rarr;
C<sub>6</sub>H<sub>5</sub>CHO<sup>&bull;&minus;</sup>  + CH<sub>3</sub>&bull;
<p>
In the first reaction there is a loss of a hydrogen atom (both its electron and
proton).  In the second an alkyl radical is lost.
<p>
In other reactions, the products are not homolytic radical cleavage, but a smaller
anion with loss of neutral, such at diacetyl (2,3-butadione) forming ketenyl anion
and ketene:
<p class="eqnserif">
C<sup>&minus;</sup>H<sub>2</sub>C(=O)C(=O)CH<sub>3</sub> &rarr;
[CH<sub>3</sub>CO<sup>&minus;</sup>(CH<sub>2</sub>=C=O)] &rarr;
CH<sub>3</sub>C<sup>&minus;</sup>=O + ketene
<p>
In the cases of alkoxides (alcoholates) and enolates, there is a loss
of hydrogen (H<sub>2</sub>), which can be explained by intermediate formation
of hydrides and enolate products:
<p class="eqnserif">
CH<sub>3</sub>CH<sub>2</sub>O<sup>&minus;</sup> &rarr;
[H<sup>&minus;</sup>(CH<sub>3</sub>CHO)] &rarr;
[H<sub>2</sub>(CH<sub>2</sub>=CHO<sup>&minus;</sup>)] &rarr;
H<sub>2</sub> + CH<sub>2</sub>=CHO<sup>&minus;</sup>
<p>
The rearrangements involve various mechanisms:  internal nucleophilic
condensation, intramolecular nucleophilic substitution, ion-molecule complex
formation, and so on.
<p>
Consider the fragmentation of the [M&minus;H]<sup>&minus;</sup> anion form
of ethylene glycol diacetate:
<sup>&minus;</sup>OCH<sub>2</sub>CH<sub>2</sub>-O-CO-CH<sub>2</sub>-CO-CH<sub>3</sub>.
In an MS/MS spectrum it shows at <i>m/z</i> 145 along with seven product ions
(A-G) in decreasing <i>m/z</i> value order, rel. intensities in brackets:
103 [2-3%], 101 [5%], 83 [100%], 61 [3%], 59 [18%], 57 [38%], 41 [10%].
Another ion appears at <i>m/z</i> 39, but is apparently not a product ion.
One mechanism has an ion-molecule complex forming a ketene and an alkoxide,
which dissociates to form ethylene glycol acetate (A) and this further fragments
to form acetate anion (E).  Another path is the transfer of proton from ketene
to alkoxide, and then dissociation to the ynolate ion (G).  On a different path
from ethyleen glycol monoacetoacetate, the acetoacetate anion is produced.
This ion loses CO<sub>2</sub> to form acetone enolate (F).  Another path is
an internal proton shift and then fragmentation to ion-neutral complex being
ethylene glycol anion and acetylketene:  this complex can release as is to
form the ethylene glycol anion (D), or the charge moves to acetylketene to
form an anion (C).

<h4>Radical Anion (Odd-Electron<sup>&bull;&minus;</sup>) Mechanisms</h4>
<p>
Although not often observed, anions as radicals will show single cleavage
reactions. The cleavage occurs &alpha; to the charge site, or &alpha; to an
atom conjugated with the charged site.

<h3>Charge Remote Fragmentation (CRF)</h3>
<p>
Fragmentation often occurs at a point quite distant from the charge center
(charge location).  This appears to occur without influence of the charge in the
mechanism.  Generally the charge is centered in the molecule at its most stable
location (maximum stability for the whole molecule).  CRF occurs because the
molecule is in an excited state in ways that do not reflect the charge presence
itself.  This is true for cations and anions of all kinds.

<h2>Collision-Based Fragmentations</h2>
<p>
According to the Warhaftig diagram, ions leaving the source are of three types.
The first are unfragmented ions in which the high energy for fragmentation has
a lifetime &gt; 10<sup>&minus;6</sup> s.  Another is ions leaving the source
with a lifetime &lt; 10<sup>&minus;7</sup> s and only fragments are detected.
The third category are metastable ions with a lifetime with the range of
100 to 1000 ns and stable enough to be selected (filtered) by the first analyzer
but with enough internal energy to fragment before reaching the second analyzer.
These are usually 1 in 100 molecules (low probability)
<p>
<b>Warhaftig diagram</b>. A double plot of <i>P</i>(<i>E</i>) and
log <i>k</i>(<i>E</i>) versus the internal energy <i>E</i> of ions in
a unimolecular dissociation reaction, where <i>P</i>(<i>E</i>) represents the
proportion (fraction) of ions with energy <i>E</i>, and log <i>k</i>(<i>E</i>)
represents the rate constant plot.  The plot is supposed to reflect a
molecular radical-cation ABCD<sup>&bull;+</sup> (such as might be generated
by electron ionization) which undergoes to competing reactions:  one channel
leads to direct fragmentation AB + CD and the other leads to a rearrangement
prior to fragmentation AD + BC.

<h3>Collision-Induced Dissociation (CID)</h3>
<p>
Hoffmann and Stroobant also refer to CID as collision-activated decomposition
(CAD).
<p>
In CID, the energy of a colliding inert molecule results in a conversion of the
translational energy of the ion into internal energy, bring the ion into the
excited state. The time period is from 10<sup>&minus;16</sup> to 10<sup>&minus;
14</sup> seconds. The next step is the unimolecular decomposition of the
activated ion. <a href="#rrkm">RRKM</a> (or quasi-equilibrium) theory is used
to assess probabilities of the collision yield.
<p>
In some cases the sampled <i>precursor</i> ion will be subjected to the presence
molecules placed in their path to cause fragmentation of the molecule
that results from collision.  Fragmentation usually occurs in a reproducible
pattern, reflecting the fact that the structure of a molecule makes it
susceptible to dissociation along certain moieties.
<p>
Sulfamethazine, with a mass of 278 Da, will typically fragment on either
side of the sulfoxide moiety (O=S=O) to produce child ions of mass 92,
156, and 186 Da, and so the mass spectra reveal that.
<p>
Advantages and disadvantages of CID:

<table>
<tr><th>Advantages                     <th>Disadvantages
<tr><td>sometimes increases structural information for pseudo-molecular
        ions obtained from <q>soft</q> ionization
    <td>lack of mixture analysis capability
<tr><td>suppresses chemical noise from solvent cluster
	 <td>cannot pre-select cursor ion as with MS/MS
<tr><td>often fairly simple spectra to interpret, but does not follow
        established &lsquo;classical&rsquo; rules, as with EI
	 <td rowspan="2">
<tr><td>better ion transmssion than in MS/MS
</table>

<h3>Converting Collision Energy to Internal Energy</h3>
<p>
<img src="imagesMassSpectrometryHTML/CollisionVectors.png"
style="float:right;margin:0 0 0 1.5em;"
 alt="position vectors in the collision description">

A molecular ion of mass <i>m</i><sub>1</sub> collides with an inert
atom (or molecule) of mass <i>m</i><sub>2</sub>.  Consider a point in space
<i>O</i> being the laboratory reference frame for construction position
vectors <i><b>R</b></i>.  Point <i>G</i> is another reference point representing
the center of gravity.  All vectors are shown with respect to the masses
in the figure at right.
<p>
The center of gravity is clearly
(<i>m</i><sub>1</sub> + <i>m</i><sub>2</sub>)<i><b>R</b></i><sub>G</sub> =
<i>m</i><sub>1</sub><i><b>R</b></i><sub>1</sub> +
<i>m</i><sub>2</sub><i><b>R</b></i><sub>2</sub>.  Then the following can be
derived:
<p class="eqnserif">
<i><b>r</b></i><sub>1</sub> = <i>m</i><sub>2</sub>(<i><b>R</b></i><sub>1</sub>
&minus; <i><b>R</b></i><sub>2</sub>)/(<i>m</i><sub>1</sub> + <i>m</i><sub>2</sub>)
<p class="eqnserif">
<i><b>r</b></i><sub>2</sub> = &minus;<i>m</i><sub>1</sub>(<i><b>R</b></i><sub>1</sub>
&minus; <i><b>R</b></i><sub>2</sub>)/(<i>m</i><sub>1</sub> + <i>m</i><sub>2</sub>)
<p>
If position is a function of time, then differentiating with respect to time,
the time-dependent position is shown:
<p class="eqnserif">
<i><b>u</b></i><sub>1</sub> = <i>m</i><sub>2</sub><i><b>g</b></i>/(<i>m</i><sub>1</sub> + <i>m</i><sub>2</sub>)
<p class="eqnserif">
<i><b>u</b></i><sub>2</sub> = &minus;<i>m</i><sub>1</sub><i><b>g</b></i>/(<i>m</i><sub>1</sub> + <i>m</i><sub>2</sub>)
<p>
where <b><i>g</i></b> = <i><b>v</b></i><sub>1</sub> &minus;
<i><b>v</b></i><sub>2</sub> and
<i>m</i><sub>1</sub><i><b>u</b></i><sub>1</sub> +
<i>m</i><sub>2</sub><i><b>u</b></i><sub>2</sub> = 0.
<p>
The conservation of momentum states that the kinetic energy of a rapid
particle colliding with a static target (<i><b>v</b></i> = 0) cannot be
entirely converted to internal energy.  Kinetic energy available for convesion
(<b>E</b><sub>t</sub>, or <q>relative kinetic energy</q>) is the kinetic
energy in the center of mass reference frame (G).  The momentum of the G frame
is zero.  The following equations apply:
<p class="eqnserif">
<b>E</b><sub>t</sub> =
{ <i>m</i><sub>1</sub><i><b>u</b></i><sub>1</sub><sup>2</sup> +
<i>m</i><sub>2</sub><i><b>u</b></i><sub>2</sub><sup>2</sup> } &divide; 2 =
&mu;<i><b>g</b></i><sup>2</sup>/2

<p class="eqnserif">
&mu; = <i>m</i><sub>1</sub><i>m</i><sub>2</sub>/(<i>m</i><sub>1</sub> +
<i>m</i><sub>2</sub>)
<p>
Note that the relative kinetic energy is the product of the kinetic
energy of the molecular ion and the fraction of the mass represented
by the inert collisional target.

<h2>Biomolecular Fragmentation</h2>
<p>
 
<!--  =============================================================
   The MS Discriminator/Analyzer  
=================================================================== -->
<h1>The Mass Discriminator: Separation of the Gas Phase Ions</h1>

<h2>MS versus MS/MS (MS<sup><i>n</i></sup>) Modes</h2>
<p>The mass spectrometer often has several features which allow it to do
more than merely measure the <i>m/z</i> of a gas phase molecular ion.
Many mass spectrometers are equipped with features that allow it to
break apart or add other ions (adducts) to the ion.  These features provide
more information which often includes structural information.


<h2>Time of Flight (ToF or TOF)</h2>
<p>Ions emerge from a source area and then are accelerated in a small entrance
to the flight tube.  In theory, the acceleration and velocity are proportional
to the mass.  Lighter ions will accelerate faster and attain a higher velocity
than high mass ions.  The time they travel a fixed distance through the flight
tube will thus be proportional to their mass (the smaller or shorter the
time value, the smaller the mass value).
<p>
A pulsed <i>pusher</i> voltage ensures that a collection of ions all leave
the source at the same time.
<p>
The following relevant equations apply:
<p class="eqnserif">
<i>E&nbsp;=&nbsp;zeV</i>&nbsp;&nbsp;[1]
<p>
<i>E</i> is the potential energy of an electric field,
<i>e</i> is the coulombic charge of an ion (a universal constant whose
value is 1.60217653&nbsp;&times;&nbsp;10<sup>&minus;19</sup>&nbsp;C),
<i>z</i> the charge magnitude of the ion,
and <i>V</i> the potential difference.  Note that <i>z</i>, the
charge magnitude of an ion, is a non-zero <b>integer</b> that is either negative
or positive (..., &minus;3, &minus;2, &minus;1, +1, +2, +3, ...), depending
on whether the ion is negative or positive.  The voltage <i>V</i>
is effectively the accelerating voltage in the flight tube of the
mass spectrometer.
<p>
The potential energy of ion accelerated in the electric field of
a mass spectrometer (the accelerating voltage) becomes the
kinetic energy of the ion, and so the kinetic energy equation applies:
<p class="eqnserif">
<i>E</i>&nbsp;=&nbsp;&frac12;<i>mv</i><sup>2</sup>&nbsp;&nbsp;[2]
<p>
where <i>m</i> is the mass of the ion and <i>v</i> is its velocity after
being accelerated in the flight tube.
The velocity of the ion can be determined by measuring a known distance,
such as the length of the flight tube, against the <b>time</b> necessary
to traverse that distance.  Thus,
<p class="eqnserif">
<i>E</i> = &frac12;<i>m</i>(<i>d</i>/<i>t</i>)<sup>2</sup>&nbsp;&nbsp;[3]
<p>
where <i>d</i> is the distance traversed, which is the length of the
mass spectrometer flight tube, and <i>t</i> is the <b>time of flight</b>
of the ion.
<p>
The equations of energy are solved for the time of flight of the
ion (<i>t</i>).  (Set equations [1] and [3] equal to one another,
and solve for <i>t</i>.)

<table class="eqnserif">
<tr><td rowspan="2" style="padding:0;vertical-align:middle;"><i>t</i> =
 <span style="font-size:250%;padding:0;margin:0;">&radic;</span>
 <td style="border-top:1px solid black;padding:0;text-align:center;
   border-bottom:1px solid black;"><i>md</i> <sup>2</sup>
 <td rowspan="2" style="vertical-align:middle;padding-left:1em;">[4]
<tr><td>2<i>zeV</i>
</table>
<p>
Equation [4] can be stated in another way:
<p class="eqnserif">
<i>ToF&nbsp;=&nbsp;k</i>&radic;<i><span class="overbar">m/z</span></i>&nbsp;&nbsp;[5]
<p>
That is, the time of flight of any ion in the flight tube
is proportional to the&mdash;or is equal to a
constant times&mdash;the square root of the mass-to-charge ratio.
The <q>constant</q> <i>k</i> is has the following value:

<table class="eqnserif">
<tr><td rowspan="2" style="padding:0;vertical-align:middle;"><i>k</i> =
 <td style="padding:0;border-bottom:1px solid black;"><i>d</i>
 <td rowspan="2" style="vertical-align:middle;padding-left:1em;">[6]
<tr><td>&radic;<span style="border-top:1px solid black;">2<i>eV</i></span>
</table>
<p>
which is algebraically extracted from equation [4].  It is a
<q>constant</q> because all the values are known and essentially
fixed:  the length of the flight tube (<i>d</i>), the accelerating
voltage set by the mass spectrometer operator (<i>V</i>), and the coulombic
charge of any ion (<i>e</i>).

<h3>Applying the Equations</h3>
<p>The MALDI-ToF of Waters/Micromass will be used an example of how these
equations work in practice.
<p>
Waters recommends setting the <q>source</q> voltage at 15,000 volts.
The <q><i>Lt<sub>eff</sub></i></q> (or the <i>effective length
of the flight tube</i>) is about 1.8 meters.  Suppose an analyte which
is a peptide that is ten amino acids long, and has a molecular mass
of 1265.7 Da.  For the purposes of these calculations, we have to
convert the mass to kilograms, and the mass of 1 dalton (1 Da) is the mass
of a proton, which is 1.672621&nbsp;&times;&nbsp;10<sup>&minus;27</sup>&nbsp;kg.
The ionized peptide is generated within
the source&mdash;in the MALDI ToF, it is generated by using a laser on an
analyte embedded in a matrix, but it does not matter for these purposes
how the ion is generated&mdash;and then accelerated within the field.
Suppose that the generation of the ionized peptide involves a proton
(hydrogen ion) from the matrix to the peptide.  The ionized peptide
now has a single positive charge (<i>z</i>&nbsp;=&nbsp;+1).
<p>
The time required to move within the flight tube, assuming the ionized
peptide moves at a constant velocity within the flight tube after
acceleration, is:

<table class="eqnserif">
<tr><td rowspan="2" style="padding:0;vertical-align:middle;"><i>t</i> =
 <span style="font-size:250%;padding:0;margin:0;">&radic;</span>
 <td style="border-top:1px solid black;padding:0;text-align:center;
   border-bottom:1px solid black;">
(1265.7 Da)(1.672621&nbsp;&times;&nbsp;10<sup>&minus;27</sup>&nbsp;kg&nbsp;Da<sup>&minus;1</sup>)(1.8 m)<sup>2</sup>
 <td rowspan="2" style="vertical-align:middle;">= 3.778 &times; 10<sup>&minus;5</sup> s =
  37.78 &mu;s
<tr><td>2(+1)(1.60217653&nbsp;&times;&nbsp;10<sup>&minus;19</sup>&nbsp;C)(15,000&nbsp;V)
</table>
<p>
This is a direct evaluation of equation [4], where the values are
inserted.  Note that 1 volt = 1 J/C (1 Joule per Coulomb), and that
1 Joule is also equal to 1 kg&nbsp;m&nbsp;s<sup>&minus;2</sup>.  The result
will be expressed in seconds.  If the value <i>z</i> is negative, note
that the electric field (voltage) should be a negative number, and so
the equation can be evaluated without giving a result that is not a real
number.
<form id="ToF" action="#">
<p style="color:blue;font:bold 90% Verdana,Arial,Helvetica,sans-serif;">
Do NOT use thousands separators (commas, periods or full-stops) to enter
values.  For example, enter "15000" and NOT "15,000" or "15.000" to indicate
fifteen thousand.

<table class="eqnserif">
<tr><td rowspan="2" style="padding:0;vertical-align:middle;"><i>t</i> =
 <span style="font-size:250%;padding:0;margin:0;">&radic;</span>
 <td style="border-top:1px solid black;padding:0;text-align:center;
   border-bottom:1px solid black;">
 <input type="text" style="text-align:right;" onchange="calcToF(this);"
 name="ionmass" size="8" maxlength="8">&nbsp;Da&nbsp;&times;&nbsp;(1.672621&nbsp;&times;&nbsp;10<sup>&minus;27</sup>&nbsp;kg&nbsp;Da<sup>&minus;1</sup>)&nbsp;&times;&nbsp;(<input
 type="text" name="Lteff" size="6" maxlength="6" onchange="calcToF(this);"
 style="text-align:right;">&nbsp;m)<sup>2</sup>
<td rowspan="2" style="vertical-align:middle;"> =
<input type="text" size="6" maxlength="6" name="mantsec" class="result"
disabled style="text-align:right;"
onchange="calcToF(this);">&nbsp;&times;&nbsp;10<sup><input
type="text" size="3" maxlength="3" name="expsec" disabled class="result"
style="vertical-align:0.5em;" onchange="calcToF(this);"></sup>&nbsp;s&nbsp;=&nbsp;<input
type="text" size="6" maxlength="6" name="microsec" disabled class="result"
onchange="calcToF(this);" style="text-align:right;">&nbsp;&mu;s
<tr><td>2&nbsp;&times;&nbsp;<input type="text" name="charge" size="3" maxlength="3"
onchange="calcToF(this);"
style="text-align:center;">&nbsp;&times;&nbsp;(1.60217653&nbsp;&times;&nbsp;10<sup>&minus;19</sup>&nbsp;C)&nbsp;&times;&nbsp;<input
type="text" size="6" maxlength="6" style="text-align:right;" name="voltage"
onchange="calcToF(this);">V)
</table>
<div id="ErrorReport"></div>
</form>
<p>
The Q-ToF micro MS has a clock generator with a frequency of 3.6 GHz, which
allows it to keep track of a time period of 278 picoseconds
[ = (3.6 GHz)<sup>&minus;1</sup>].  Call this time period of 278 ps
one &lsquo;tick.&rsquo;  With each tick, the ToF system considers
this an increase in mass, assuming charge is the same.
Since the time is proportional to the square
root of the <i>m/z</i>, conversely, the <i>m/z</i> is proportional to
square of the time.  If a singly charged ion of mass <i>m</i><sub>1</sub>
was detected at time <i>t</i><sub>1</sub>, then a singly charged
ion of mass <i>m</i><sub>2</sub> that is 4 &times; <i>m</i><sub>1</sub>
would strike the detector at time <i>t</i><sub>2</sub> which is
2 &times; <i>t</i><sub>1</sub>.
<p>
Recall that the resolution of a mass spectrometer is the mass of ion
(the peak maximum in a mass spectrum), or <i>m</i>
divided by the difference in the mass at 50% of the peak height (also
called <q>full width half maximum</q> or FWHM), or &Delta;<i>m</i>.
Note that <i>m</i> is really <i>m/z</i> and &Delta;<i>m</i> is
&Delta;<i>m/z</i>, but this assumes for simplicity a singly charged ion.
Because <i>m</i> is related to <i>t</i> (mass is related to time)
in a ToF MS, this means that &Delta;<i>m</i> is related to &Delta;<i>t</i>
as well.  One tick represents the minimum time period that the Q-ToF
micro can itself resolve.  The longer the tick period (&Delta;<i>t</i>),
then the larger the mass difference (&Delta;<i>m</i>).
<p>
Consider the difference between singly charged ions of mass
1000 Da and 1001 Da in the Q-ToF micro,
with an accelerating voltage of 800 V and flight tube distance of 1.8 m.
For the 1000 Da ion, time is 145.397 &mu;s.	For the 1001 Da ion, it is
145.47 &mu;s.  That is a difference of 0.046 &mu;s, which represents
262 ticks on the Q-ToF clock.
<p>
Now consider the difference between ions of 1000.0 and 1000.1 Da.
The time is 145.404 &mu;s for the 1000.1 Da ion, a difference of
only 25 ticks now.  If the mass is now 1000.01 Da compared to 1000.00 Da,
the time for 1000.01 Da is now 145.398 &mu;s, only about 3-4 ticks
difference.  Thus the effective limit of the Q-ToF is the ability to
resolve 0.1 Da under these conditions.  The one way to increase the
ability to resolve smaller differences in mass is either to reduce
the accelerating voltage (not practical for orthogonal ToF as used
in the Q-ToF micro) or to increase the flight tube distance (this
is practical in many cases).

<h3>Orthogonal ToF</h3>
<p>The Waters ToF MS is set up to pulse the ion beam into the flight tube
and then to characterize the set of ions in that group.  A new set of
ions cannot be pulsed until the last (heaviest) ion has reached the
flight tube detector.  This usually occurs in nanoseconds.  The pulsing
pusher is set to produce a group of ions at 30 kHz, or once every 33 &mu;s.
Thus, in one second, 30,000 groups of ions can be analyzed and a scan
(1 second spectrum) produced.
<p>
ToF is generally more suited to ionizations that occur rhythmically or
in pulses, such as pulsed laser desorption.  Although there is no upper
limit with respect to mass measure, the practical theoretical limit
is 350,000 Da.

<h3>Resolution in ToF</h3>
<p>
Resolution is affected by many factors.  Generally in ToF, ions of the same 
<i>m/z</i> ratio should theoretically all have the same acceleration, velocity, 
and time in the flight tube.  In reality they do not. These minor differences in 
time show up as a Gaussian distribution which represents the peak in the 
<i>Intensity</i> vs. <i>m/z</i> spectra. The following equation applies to ToF 
resolution:
<p class="eqnserif">
Resolution&nbsp;=&nbsp;time of flight / (2 * time spread)
<p>
The time spread is the difference in the time for the slowest and fastest ions 
to reach the detector.

<h2>Quadrupole Theory</h2>
<p style="font:normal 83% Arial,Helvetica,sans-serif;">
[The theoretical discussion of this section comes from several sources, but the 
figures and much of the discussion come from a web publication by R. E. Pedder 
of ABB Inc. in Pittsburgh PA USA, published by Extrel CMS L.P. in Application 
Note RA 2010A, <q>Practical Quadrupole Theory: Graphical Theory</q>]
<p>
The quadrupole is a set of four rods whose axial length runs in the same 
direction as the ion beam.  The placement of the rods is such that the distance 
between the faces of opposite poles is 1/1.148 times the diameter of the rod. 
This is chosen so that the geometric center approximates a hyperbolic electric 
field.
<p>
The ion beam actually enters into the dead center space that is the same 
distance from all the centers of the rods. The rods are subjected to a 
fluctuating (time-dependent) voltage that varies as a radiofrequency (usually 
fom a few Hz to 700 kHz). Moreover, the voltage of the rods does not have to be 
zero at the start of the wave:  it can be offset from zero. The rods opposite 
one another are of one polarity, and the other pair opposite one another is of 
the other polarity.  An electric field is created by the rods that is hyperbolic 
around the ion beam trajectory. Because the electric field has a component that 
varies with time, the trajectory of the ions is also likely to be affected by 
the fluctuating field.  Ions with a certain mass will begin to resonate in their 
movements through the quadrupole.  Those with a mass either too low or too high 
are not going to resonate, and their path will end at the quadrupole.

<p>
The power supply to the quadrupole is such that the DC pole bias
determines the potential of the centerline.  Two resolving DC supplies
provide equal magnitude but opposite polarities to the quadrupole set.
A high voltage RF transformer has a single primary and two secondaries
which are 180&deg; out of phase
<p>
Resonant ions that pass through quadrupole are usually found by
varying the voltage rather than the RF.


<table class="eqnserif">
<tr><td rowspan="2" style="vertical-align:middle;">&Phi;&nbsp;=&nbsp;<i>V</i><sub>dc</sub>
+ <i>V</i><sub>rf</sub> cos(&omega;<i>t</i>)
 <td style="border-bottom:1px solid black;"><i>x</i><sup>2</sup> &minus;
  <i>y</i><sup>2</sup>
<tr><td><i>r</i><sub>0</sub>
</table>
<p>
where &Phi; is the strength of the electric field at a position of the ion
at <i>x</i> and <i>y</i> in the quadropole;  <i>x</i> and <i>y</i> are
orthogonal to ion movement within the quadrupole long axis (<i>z</i>);
<i>r</i><sub>0</sub> is the radius of the quadrupole rods.
<p>
Solving the equations of motion provides the basis for finding the
stable regions of ion motion.  These are <i>a</i> and <i>q</i>
(often symbolized <i>A</i> and <i>Q</i>, resp.).


<table class="eqnserif">
<tr><td style="border-bottom:1px solid black;"><i>d</i> <sup>2</sup><i>u</i>
<td rowspan="2" style="vertical-align:middle;">+ (<i>a<sub>u</sub></i>
&minus; 2<i>q<sub>u</sub></i> cos 2&xi;)<i>u</i> = 0
<tr><td><i>d</i>&xi;<sup>2</sup>
</table>


<table class="eqnserif">
<col><col> <col style="width:6em;text-align:right;"><col><col>
<col style="width:6em;text-align:right;"><col span="3">
<tr><td rowspan="2" style="vertical-align:middle;"><i>a</i> =
  <td style="border-bottom:1px solid black;">8<i>zU</i>
  <td>
  <td rowspan="2" style="vertical-align:middle;"><i>q</i> =
  <td style="border-bottom:1px solid black;">4<i>zV</i>
  <td>
  <td style="border-bottom:1px solid black;"><i>a</i>
  <td rowspan="2" style="vertical-align:middle;"> =
  <td style="border-bottom:1px solid black;">2<i>U</i>
<tr><td><i>mr</i><sup>2</sup>&omega;<sup>2</sup>
  <td>
  <td><i>mr</i><sup>2</sup>&omega;<sup>2</sup>
  <td>
  <td><i>q</i>
  <td><i>V</i>
</table>
<p>
Here, <i>U</i> represents <i>V</i><sub>dc</sub> and <i>V</i> represents
<i>V</i><sub>rf</sub>.  The charge on the ion is <i>z</i> and its mass
is <i>m</i>.  The applied radiofrequency is &omega; and <i>r</i> represents
the effective radius between electrodes.  For the second order linear
differential equation, &xi; represents &omega;<i>t</i>/2 and <i>u</i>
represents the position along the coordinate axes (<i>x</i> or <i>y</i>).
The solution is:
<p class="eqnserif">
<i>u</i>(&xi;) = &Gamma; <span style="font-size:120%;">&sum;</span>
<sup style="font-size:80%;margin-left:-1.3em;vertical-align:1.5em">&infin;</sup>
<sub style="font-size:65%;margin-left:-1.75em;vertical-align:-1.2em"><i>n</i>=&minus;&infin;</sub>
<i>C</i><sub>2<i>n</i></sub> exp(2<i>n</i> + &beta;)<i>i</i>&xi; +
&Gamma;&prime; <span style="font-size:120%;">&sum;</span>
<sup style="font-size:80%;margin-left:-1.3em;vertical-align:1.5em">&infin;</sup>
<sub style="font-size:65%;margin-left:-1.75em;vertical-align:-1.2em"><i>n</i>=&minus;&infin;</sub>
<i>c</i><sub>2<i>n</i></sub> &minus;exp(2<i>n</i> + &beta;)<i>i</i>&xi;
<p>
This reduces to an infinite sum of sine and cosine functions.  Ion trajectories
are these infinite sums of sine and cosine functions, moving in <i>x</i>
and <i>y</i> directions sinusoidally.  The fundamental frequency is
(&omega;<sub>0</sub>) but harmonic frequencies are added in.
<!--
<p>
<img style="float:left;"
   src="MathieuStability.gif" alt="Mathieu Stability Diagram">
Plots of <i>a</i><sub>u</sub> versus <i>q</i><sub>u</sub> define
stability regions (see diagram at left).	Looking at the first
stability region (diagram below right), one can see how a particular
ion (<i>m/z</i> 219) will resonate at a particular set of rf and dc
voltages.  The stability regions are symmetric around the <i>a<sub>u</sub></i>
axis.  This is expected since the ion path would not be perturbed
by swings in polarity in the rods.

<p>
<img style="float:right;" src="FirstStabilityRegion.gif"
			 alt="First Stability Region">
There are lines of slope <i>a/q</i> (which is related
to a specific <i>V</i><sub>rf</sub>/<i>V</i><sub>dc</sub> ratio) that pass
through the 0 point extend in a way to cut through the stability regions.
<p>
<img style="float:left;" src="MoreStabilityRegions.gif"
			 alt="Stability Diagrams for Specific Ions">
In the diagram below left, the stability regions for three ions of
<i>m/z</i>28, 69, and 219 are shown.  A straight line is drawn through
the origin and below the peaks of stability regions.  This straight line
is the <b>RF/DC line</b>.  Movement along the scan line requires
setting the DC and RF components accordingly.  Ions that have trajectories within a
stability region that fall above the RF/DC line will appear in
the spectrum.  A line that intersects at the highest peak of
a stability region will have a spectrum seen as only a
single peak.  Below the diagram is a plot of what a spectrum would look
like if RF and DC voltages are set according to each point on the line.
When a linear scan line is used, peak widths will increase geometrically
with increasing mass, leading to a constant resolution.
<p>
<img style="float:right;" src="DecreasingScanLine.gif"
			 alt="Effect of Decreasing Scan Line on Mass Spectrum">
On the diagram at right, the scan line slope is lowered.  The resulting
spectrum will have wider mass peaks, since the areas of the stability
regions above the scan line are larger given the points where the scan
line intersects the stability curves.  It should be noted that the
leading each of a stability curve rises about 3 times more slowly
compared to the decrease of the trailing edge.  The effect of such a
curve shape is that resolution is decreased, and it also can shift the
center of the mass peak to an apparent lower mass, making accurate
mass determination problematic.
<p>
The slope of the <i>a/q</i> (RF/DC) line narrows the region of stability and
the count of transmitted ions.  This is seen as a narrowing of the peak
on the mass spectrum.  Thus, increasing RF/DC increases resolution but
decreases sensitivity.  Lowering the slope line increases sensitivity, but
at the expense of resolution.
<p>
The stability line can actually be manipulated in many ways:
<ul>
<li>the slope of the line can be changed
<li>the intercept of line can be changed
<li>the line itself can be made NOT to be linear
</ul>
<p>
One changes the <i>V</i><sub>dc</sub> parameter in order to alter the
slope of the line.  The effect is to control resolution at the higher
mass range.  Why?  Because the way the stability regions appear and the
way that the RF/DC line cuts across them means that the higher masses
have a larger window of transmitted ions.  By altering DC to reduce that
window, the effect is greater for the higher mass ions than the lower mass.
<p>
Adjusting a DC offset (rather than slope) creates better resolution in the
low mass range.  Why?  Again, the window is narrowed faster on the stability
regions of the lower mass ions than on the higher mass ions.
<p>
Finally, there is the ability to manipulate the linearity of the RF/DC
line.  By making it curved, it is possible to increase or decrease
resolution in the middle range as opposed to the lower or higher range masses.
<p>
Resolution and sensitivity are affected by many parameters of the quadrupole.
<ul>
<li>quadrupole rod length
<li>peak amplitude of RF voltage
<li>quadrupole rod separation
<li>rod machining and alignment
<li>ion kinetic energy
</ul>
 -->
<p>
<b>MS (MS1) mode</b>.
When a quadrupole analyzer is scanned in MS mode (full scan), there is
a reduced sensitivity since little time is spent on an <i>m/z</i> species
and thus there are fewer ion counts.

<p>
<b>Single Ion Recording (SIR)</b>.  This particular MS mode sets the
analyzer to let only one <i>m/z</i> species (or a narrow range) pass
to the detector.  This improves sensitivity a great deal, but there
are no mass spectra, essentially a chromatogram (ion count vs. time).


<h2>Hybrid Hexapole-ToF</h2>
<p>[Source:  Chapter C2.2 <q>Hybrid Hexapole-ToF</q> <i>Back to Basics</i>,
Micromass UK Ltd.]
<p>
This type of analyzer combination is often used with an atmospheric
pressure inlet/ion source.  Such a source is ideal for a liquid chromatographic
(LC) eluate flow in the inlet.  Typical mechanisms are APcI or ESI, which
are configured to provide desolvation and ionization.  A Z-path
can be used to direct ions toward the high vacuum analyzer block while
efficiently reducing solvent molecules from entering the ion beam
into the analyzer.
<p>
A quadrupole mass analyzer can be present and set in either a narrow
or wide band pass.  In narrow band pass, two opposed poles have an
applied sinous time-dependent (RF) voltage applied while the other two opposing
poles are fixed at a constant dc voltage.  The combination permits
ions within a range of <i>m/z</i> to pass through the poles whereas ions
outside the range oscillate out of the path.  In wide band pass,
all poles are subject to a sinous RF voltage and there is no DC voltage,
and all ions pass in what is essentially a quadrupole acting as an ion
guide.
<p>
The hexapole is set up with an RF voltage and is used only to help
focus the ion beam, since ions at the source spread because of repulsion
and collision with air and solvent during formation, and they need
beam narrowing.  The hexapole is thus a re-focusing element.  The vacuum
is set at about 10<sup>&minus;4</sup> mbar.  A second hexapole provides
further collimation and is staged to a higher vacuum of 10<sup>&minus;5</sup>
to 10<sup>&minus;6</sup> mbar, necessary for the ToF analyzer.
<p>
Because a continuous stream of ions is produced during ESI or APcI,
and a ToF analyzer can only work with a pulse of ions, the ToF analyzer
entrance is positioned orthogonally (at a right angle) to the
continous ion beam.   Pusher and puller extraction plates provide the
pulse of ions sampled out of the continous stream and into the ToF
analyzer.

<h2>Ion Trap</h2>
<p>The ion trap (IT) collects ions into an assembly composed of two cap electrodes
and a ring electrode that create a magnetic field to contain the ions.
IT does several steps to produce the mass spectra:
<ul>
<li>mass accumulation
<li>selective mass isolation and excitation for collision-induced dissociation
	in MS/MS
<li>sequential mass ejection to produce a mass spectrum
</ul>
<p>
<b>Helium</b>.  Helium gas is injected into the stream because if provides
cooling of ions.  In addition it serves as a collision gas during MS<sup>n</sup>
techniques.
<p>
<b>Electron Transfer Dissociation (ETD)</b>.
Positive ions are collected into the trap and then fluoranthene anions being
held in a container are injected into the focusing stream octopoles into the
trap while the positive ion stream is blocked.

<h2>Fourier-Transform Ion Cyclotron Resonance (FTICR)</h2>
<p>The most accurate mass analyzer presently, ions from the source
are injected into a magnetic ring (cyclotron).  A radiofrequency is
emitted into the ions circling at constant velocity.  This increases
the radius at which they circle, and as the radius is increased, this
brings the ions within the proximity of a loop circuit in which an
oscillating current is induced by cycling ion movement near it.
The signal of this oscillating current can be broken down
Recall that a Fourier transformation converts a time domain to a
frequency domain, that is, the transformation essentially does an
inversion.  Suppose that ions discriminated/analyzed on the basis
of differences in separation by time are instead analyzed on the
basis of a separation by frequency.

<h2>Magnetic Sector</h2>
<p>An ion with value <i>m/z</i> accelerated by voltage <i>V</i> and passing
through a magnet with field <i>B</i> will be deflected in an
arc which will have a radius of curvature <i>r</i>.   Value <i>r</i>
is constant:  the ion coming from the source path must be deflected into
the path leading to the detector so that detection can occur, and this does
not change in the instrument.  To detect a range of ions having different
<i>m/z</i>, one can thus alter the acceleration voltage <i>V</i> or
magnetic field <i>B</i>.
<p>
The relationship of <i>m/z</i> is:
<table class="eqnserif" style="margin-left:2em;">
<tr><td style="border-bottom:1px solid black;"><i>m</i>
 <td rowspan="2" style="vertical-align:middle;"> =
<td style="border-bottom:1px solid black;"><i>r</i> <sup>2</sup>
<td style="border-bottom:1px solid black;"><i>B</i><sup>2</sup>
<tr><td><i>z</i> <td>2 <td> <i>V</i>
</table>
<p>
Note that the constant parameters of the instrument are grouped together.

<h2>Triple Quadrupole</h2>
<p>A triple quadrupole MS features the first quad that becomes an ion
selector within the mass range;  ion (<i>m/z</i> selection is
determined by oscillating <i>V</i><sub>rf</sub> and <i>V</i><sub>dc</sub>.
Each selected ion passes into
the second quad which is a collision cell in which CID can occur
if set in MS/MS mode.  In MS mode it passes to the last quad (MS2)
without CID where there is <i>V</i><sub>rf</sub> only to tune the beam.
<p>
In MS/MS mode, the instrument is capable of <b>multiple reaction
monitoring</b> (MRM).  The first quad selects an <i>m/z</i> species
as the precursor ion, and the third quad (MS2) is set to select
the <i>m/z</i> of the product ion, providing a quantitative analysis.
<p>
MRM in a triple quad can provide data like SIR in a single quad, but
additional advantages are:

<ul>
<li>increase in selectivity
<li>increase in chemical S/N
<li>increase in the sensitivity of the assay
</ul>
<p>
<b>Daughter/Product Ion Scanning</b>.  The MS2 quad can set to full scan mode
rather than MRM during collision activation, which enables scanning
of potential daughter ions to be used in MRM mode.  This is essentially
a qualitative analysis of fragmentation.
<p>
<b>Precursor Ion Scanning</b>.  Alternatively the MS1 quad can be set
in scanning mode while MS2 is set to be static to pass a single
species.  This precursor ion scanning helps in the detection of diagnostic
losses.
<p>
<b>Neutral Loss Scanning</b>.  Useful for detection of functional groups
such as sulfates and glucuronides.

<!-- ==================================================================
    The MS Detector  
======================================================================= -->

<h1>The Counter:  Detection of the Gas Phase Ions</h1>

<h2>Point Ion Collectors</h2>

<h2>Array Collectors</h2>
<p>[Source:  Chapter E2: Array Collectors (Detectors), <i>Back to Basics</i>,
Micromass UK Limited, pp 437-454.]
<p>
Array collectors are actually a collection of point collectors (elements)
which are assembled in a plane.  Ions of a certain <i>m/z</i> are collected
at one particular point detector (element) and ions of larger or smaller
<i>m/z</i> are collected on either side.  When point collectors are lined
up in a plane, recording of the range of spectra is more rapid, since
scanning with a single point collector requires multiple ionization
pulses with prolonged scan times (sequential vs. concurrent scanning).
<p>
Each element in the array is a compact electron multiplier which can
amplify its signal.  The current flow is proportional to the count of
ions striking the element per unit time.  Resolution in the array
depends on:
<ul>
<li>count of elements in the array
<li>the separation of one element from another
<li>the degree of dispersion of ions in the beam
</ul>
<p>
Array detectors make it possible for detecting smaller amounts of substances
(improve sensitivity) or for dealing with intermittent instead of
continuous ionization.
<p>
Quadrupole filters can be set up with simple point collectors since
adjustments to the filter can be made on a continuous stream of ions
over time.  Magnetic sector analyzers disperse ions in space and the
entirety of the ions can be detected simultaneously by an array collector.
Because time-of-flight analyzers separate ions by time, a point detector
is all that is needed.
<p>
The electron multipliers that form the detector element are charged
at a potential of 1000-2000 V, with the cathode (negative) end at the
ion beam entry, and the anode (positive) end drawing electrons that
emerge from the cascade toward it.  These electrons provide the sensing
current in the circuit.  Multiplier tube geometry is important.
A straight tube cylinder (one type of <i>channeltron</i>) would
yield secondary electrons in the cascade with
many different amplification angles (back scatter), and these electrons emerge in the
circuit at different times.  This has the effect of reducing the signal
peak and creating electrical noise.  An alternative geometry in which the
channeltron has a horn shape is designed to minimize backscatter; this type of
multiplier has no separate dynodes.

<h3>Resolution vs. Mass Range Setting</h3>
<p>The area of the array
collector is fixed, so the ability of the detector to resolve ions of
<i>m/z</i> magnitude depends on the how the analyzer is configured
for the mass range in a single scan.  If the mass range is kept narrow
then the analyzer can achieve separations that an array collector can
resolve.  Conversely if the mass range is set for a wide range, the
ability to discriminate <i>m/z</i> values by small differences is
reduced.
<p>
For example, a mass range might be set for <i>m/z</i> from 500 to 1000,
and the resolution in this range is 0.01 Da.  But when the analyzer
is set to analyze <i>m/z</i> between 2000 and 10,000, the limit of
resolution increases to 0.1 Da.

<h3>Dynamic Range</h3>
<p>The job of an element in an array detector (or a point detector) is
primarily to be able to count ions that strike it.  Detectors do not
measure <i>m/z</i> directly:  the system outside the detector determines
the <i>m/z</i> by its position (if it is an element in an array working
in a spatial domain) or by marking time increments (if it a point
detector working in a time domain) in a fully calibrated environment
(i.e., evaluating detection of unknowns against references with known
masses).
<p>
As an ion counter, the <i>dynamic</i> (essentially <i>working</i>) range
of the point ion detector is the ability to count from a few ions
per unit time to tens of thousands of ions per unit time, with a fairly
linear response.  Elements within array detectors become more easily
saturated (loss of linear response) than do point detectors.  This
is basically because electron multipliers larger in size have a larger
dynamic range, and it is impractical to create array detectors whose
elements are as large in size as systems with point detectors.  Usually
the smaller dynamic range of array elements is sufficient for doing work,
but the concept is important to consider.
<p>
An ion's mass also affects the response of the electron multiplier:
the larger the mass, the more the response falls.  This is because
ions of larger mass have a much lower velocity and produce fewer
cascading electrons on impact with the multiplier.

<h3>Uses</h3>
<p>Array detectors are used in preference to point detectors when ions must
be counted for more than one <i>m/z</i> value within a short period of
time.  This is often dictated by the type of ionization being done.
For example, with laser desorption or radionuclides, all ions are produced
in a pulse and then accelerated toward the detector.  The time span in which
they must all be measured is beyond the ability of a point detector
to be responsive to them, even if time-of-flight is the mode of
separation/analysis (ion trap analyzers work differently however).

<h3>Microchannel Plates</h3>
<p>These are array of glass capillaries with a 10-25 &mu;m inner diameter
and coated inside with electron-emissive material.  Biased at a high
voltage, any ion striking the inside wall causes a cascade of
secondary electron emissions, which in turn result in more cascades.
The gain is about 10<sup>3</sup> to 10<sup>4</sup>, and with the dual
MCPs in a chevron configuration, the gain is 10<sup>6</sup> to 10<sup>7</sup>.

<h2>Dead Time</h2>
<p>Dead time is the refractory period when a detector is placed into a state
when it can not detect ions.  When a detector is bombarded with too many
ions, the ions that follow behind are not counted, as they hit the detector
within the refractory period.  This gives a distorted view of the spectrum.
Because the ions which hit the detector first are the ones lighter in mass,
if they overwhelm the detector, the slower ones behind (and the ones with
the heavier mass) will not really be counted in the numbers that they exist.
This weights the count toward the ions that were lighter.  On the spectrum,
this is seen as a shift toward a lighter mass.
<p>
The only way to get an accurate measure of the distribution of the group
is not to overwhelm the detector with a large count of ions.  Thus the
ion beam should be kept to a minimal number of ions.


<!--  The Mass Spectrum -->


<h1 id="mass-spectrum">The Data of Mass Spectrometry:  the Mass Spectrum</h1>
<p>
The mass spectrometer produces essentially a plot of the <b>mass-to-charge</b>
(<b><i>m/z</i></b>) of the ionized molecule with the count of the number of 
ions at that <i>m/z</i> value. The count of ions can be considered the <b>
absolute intensity</b>. However, in practice the count of ions for all ionized 
molecules at an <i>m/z</i> value is presented as a percentage (from 0 to 100%) 
or a ratio (from 0.0 to 1.0) of the <i>m/z</i> value of the ion having the 
greatest number of counts. This is called the <b>relative intensity</b>. The <i>
m/z</i> value of the ion having the greatest number of counted ions is called 
the <b>base peak</b>.
<p class="thumbed-image-left">
Click on image to obtain at a original resolution
<img src="imagesMassSpectrometryHTML/MassSpectrum.png"
 onclick="bigimage(this);"
   style="width:100%;cursor:pointer;" alt="the mass spectrum">
<p>
The figure at left shows how the mass spectrum appears.  This is an unprocessed
mass spectrum showing the ionizable tryptic peptides of the protein yeast
alcohol dehydrogenase of a MALDI-MS analysis.  The mass range measured was from
<i>m/z</i> 1000 to 3000, and the base peak was of a peptide at <i>m/z</i> 1618.722.
The green color-bordered inset zooms in on the mass spectrum to reveal the
detail at the <i>m/z</i> 1618 peak, showing the mass range of about <i>m/z</i>
1617 to 1624.  The detail shows that there are series of peaks separated by
approximately 1 Th (1 amu).  In fact, all of those peaks represent a single
ion, a tryptic peptide.

<table style="float:right;font-size:85%;width:50%;margin-left:1.5em;">
<tr><th>Isotope
 <th style="width:6em;">Mass<br>(amu)
 <th style="width:10em;">Relative Abundance<br>(%)
<tr><td><sup>1</sup>H <td>1.007825037 <td>99.985
<tr><td><sup>2</sup>H <td>2.014101787 <td>0.015
<tr><td><sup>12</sup>C <td>12 <td>98.9
<tr><td><sup>13</sup>C <td>13.003354 <td>1.11
<tr><td><sup>14</sup>N <td>14.003074 <td>99.6
<tr><td><sup>15</sup>N <td>15.00011 <td>0.36
<tr><td><sup>16</sup>O <td>15.99491464 <td>99.8
<tr><td><sup>17</sup>O <td>16.9991306 <td>0.4
<tr><td><sup>18</sup>O <td>17.99915939 <td>0.2
<tr><td><sup>31</sup>P <td>30.9737634 <td>100
<tr><td><sup>32</sup>S <td>31.972074 <td>95
<tr><td><sup>33</sup>S <td>32.9707 <td>0.76
<tr><td><sup>34</sup>S <td>33.96838 <td>4.22
<tr><td><sup>36</sup>S <td>35.96676 <td>0.2
<tr><td colspan="3"
   style="background-color:yellow;font:bold italic 90% Tahoma,Verdana,sans-serif;">Other Isotopes
<tr><td><sup>23</sup>Na <td>22.98976979 <td>100
<tr><td><sup>35</sup>Cl <td>34.968854 <td>75.8
<tr><td><sup>37</sup>Cl <td>36.965896 <td>24.2
</table>
<p>
Before looking at the detail of this mass spectrum, a short discussion of
the atoms that make up molecules in general and of a particular class of
molecules such as the biomolecular class proteins and their component peptides
is necessary.  The table at right shows the isotopes and their relative
natural abundances of atoms that are particularly important to peptides and
proteins.  The amino acids that make up peptides and proteins are primarily made
of carbon, hydrogen, nitrogen, and oxygen, although a few amino acids have sulfur
as well.  Peptides and proteins are also often associated (non-covalently) with
other atoms too, usually as salts or other ions.  It can be seen in the table,
for example, that carbon has two isotopes, mostly (98.9%) the carbon-12
(<sup>12</sup>C) form.  Oxygen has three isotopes, nearly 100% of which is the
familiar <sup>16</sup>O isotope.
<p class="thumbed-image-left">
Click on image to obtain at a original resolution
<img src="imagesMassSpectrometryHTML/MethaneSpectrum.png"
 onclick="bigimage(this);"
alt="schematized spectrum of methane">
<p>
In most chemical or biochemical analyses, the isotopic forms of a molecule do
not affect the chemistry that occurs.  But because mass spectrometry measures
molecular mass with great resolution, easily the difference of a single neutron
or proton of the atomic nucleus, it is able to resolve the isotopic forms.
Look at the schematized spectrum of methane (CH<sub>4</sub>) at the left; note
it is not real spectrum, but plausible schematic form.  If the molecule is made
up of only the <sup>12</sup>C isotopes of carbon and the <sup>1</sup>H isotopes
of hydrogen, it will have a nominal mass of 16 ( = 12 + 4 &times; 1).  But because
<sup>13</sup>C and <sup>2</sup>H (deuterium) also exist in nature, some of the
methane molecules will have an atomic composition of
<sup>13</sup>C<sup>1</sup>H<sub>4</sub> with a nominal mass of 17 (= 13 + 4
&times; 1), and also <sup>12</sup>C<sup>2</sup>H<sup>1</sup>H<sub>3</sub>
(12 + 2 + 3 &times; 1 = 17).  Other possible numeric combinations of isotopes
can increase the mass of methane detected.  The height of the signal peak, that
is the relative intensity, can be predicted by computation based on the known
constants of the relative natural abundance of each isotope.  To compute these
values, it is important to compute the product of the abundances of all the
isotopic atoms that make up the molecule.  Using the table above for
<sup>12</sup>C<sup>1</sup>H<sub>4</sub>, this is
(0.989)(0.99985)<sup>4</sup> = 0.9884067335 (rounded).  This number will be the divisor
in computing relative abundance.  Now compute nominal mass of 17 for methane.
Since there are two forms,
<sup>13</sup>C<sup>1</sup>H<sub>4</sub> and
<sup>12</sup>C<sup>2</sup>H<sup>1</sup>H<sub>3</sub>, the products must be
summed: (0.011)(0.99985)<sup>4</sup> + (0.989)(0.015)(0.99985)<sup>3</sup> =
0.025821727.  Now divide this value by the abundance of the monoisotopic
abundance to get the relative abundance: 0.025821727 &divide; 0.9884067335 =
0.026124596 (2.6%).

<p class="thumbed-image-right">
Click on image to obtain at a original resolution
<img src="imagesMassSpectrometryHTML/MassSpectrumFeatures.png"
 onclick="bigimage(this);"
alt="features of the mass spectrum: isotope cluster, monoisotopic peak">
<p>
The figure at the right looks more closely at the inset from the yeast
alcohol dehydrogenase tryptic petpides from the figure above, zooming into the mass
range <i>m/z</i> 1617-1624, which features the single molecular ionic species
at <i>m/z</i>1618.722.  The series of peaks shown all belong to one molecular
species, which is the peptide with sequence VLGIDGGEGKEELFR.  This peptide is
located in the yeast alcohol dehydrogenase single protein subnit at positions
198 to 212.  The atomic composition of this peptide is
C<sub>71</sub>H<sub>115</sub>N<sub>19</sub>O<sub>24</sub>.  But the series of
peaks shown are the <b>isotopic cluster</b> of that molecule with atomic
composition C<sub>71</sub>H<sub>115</sub>N<sub>19</sub>O<sub>24</sub>.  There
are <i>five</i> observed (detectable) peaks in that series, forming the cluster.
The peak at <i>m/z</i> 1618.722 has the smallest <i>m/z</i> value, and also the
smallest mass of the peaks forming the cluster.  This is called the
<b>monoisotopic</b> peak:  the monoisotopic peak (mass) of a molecular ion
is composed of atoms that make up all the highly abundant atomic isotopes which
have the lightest masses of those atoms.  Thus the monoisotopic peak is
<sup>12</sup>C<sub>71</sub><sup>1</sup>H<sub>115</sub><sup>14</sup>N<sub>19</sub><sup>16</sup>O<sub>24</sub>.
The peak at 1619.713, whose <i>m/z</i> value is ~ 1 Th (1 amu) greater (actually
it is 0.991 amu) is the mass of one neutron greater.  This might be one
<sup>13</sup>C instead of <sup>12</sup>C, or <sup>2</sup>H instead of <sup>1</sup>H,
or <sup>15</sup>N instead of <sup>14</sup>N, or <sup>17</sup>O instead of
<sup>16</sup>O.  Thus, that peak can represent any of the following molecules
having the indicated atomic composition:

<ul>
<li><sup>12</sup>C<sub>70</sub><sup>13</sup>C<sup>1</sup>H<sub>115</sub><sup>14</sup>N<sub>19</sub><sup>16</sup>O<sub>24</sub>
<li style="margin-top:0.5em;"><sup>12</sup>C<sub>71</sub><sup>1</sup>H<sub>114</sub><sup>2</sup>H<sup>14</sup>N<sub>19</sub><sup>16</sup>O<sub>24</sub>
<li style="margin-top:0.5em;"><sup>12</sup>C<sub>71</sub><sup>1</sup>H<sub>115</sub><sup>14</sup>N<sub>18</sub><sup>15</sup>N<sup>16</sup>O<sub>24</sub>
<li style="margin-top:0.5em;"><sup>12</sup>C<sub>71</sub><sup>1</sup>H<sub>115</sub><sup>14</sup>N<sub>19</sub><sup>16</sup>O<sub>23</sub><sup>17</sup>O
</ul>
<p>
[Note: While peptide VLGIDGGEGKEELFR was generated by trypsin digestion of
yeast alcohol dehydrogenase, an enzyme that hydrolyzes peptide bonds on the
C-terminal side of Lys (K) and Arg (R) residues except when the following
residue is Pro (P), there is K residue within the sequence. It has been shown
that trypsin may not hydrolyze particularly after K residues efficiently, and
probably when it is surrounded by negatively charged ionic residues such as the
two Glu (E) residues that follow it and the one that also closely precedes. In
order to improve the efficiency of trypsin digestion for doing proteomic
analysis, there are <b>guanidination</b> kits containing reagent chemicals,
using <i>O</i>-methylisourea, to convert the lysine primary &epsilon;-amino
group into a more strongly basic and more cationic guandinyl function that is
present on the Arg side chain.]

<h2>Charge State (<i>z</i>)</h2>
<p>
The mass spectrum not only contains the two items of information being the
mass-to-charge (<i>m/z</i>) and a measure of its quantity (ion counts as
relative intensity), but also the charge state (<i>z</i>) of the ions.  The
value <i>z</i> is computed from the differences in the <i>m/z</i> values
in the isotope cluster that makes up the ion.  For polyatomic molecules,
in most cases an isotopic cluster is possible.

<p class="thumbed-image-right">
Click on image to obtain at a original resolution
<img src="imagesMassSpectrometryHTML/ChargeState.png"
 onclick="bigimage(this);"
alt="features of the mass spectrum: isotope cluster, monoisotopic peak">
<p>
The computation of the charge state <i>z</i> is quite basic. In a calibrated
instrument, obtain the spectrum. For the ion of interest, find the isotope 
cluster. This is not always easy: two ions of slightly different <i>m/z</i> may 
be very close together, and the start of the isotopic cluster (the monoisotopic
peak) might overlap with the end of the series of peaks of the ion with the
smaller <i>m/z</i> value.  Sometimes clusters can be identified by predicting
the appearance of all the relative intensities that form the cluster.  Once
the cluster is identified, then compute the difference between the first
and second peak (monoisotopic peak and the <i>m</i> + 1 peak).  The difference
in Thomsons or atomic mass units (&Delta;<i>m</i>) = 1/<i>z</i>, or rather
after algebraic rearrangment, <i>z</i> = 1/&Delta;<i>m</i>.
<p>
The figure at right will serve as an example. Substance P is a peptide of
eleven amino acids, RPKPQQFFGLM. In electrospray in fully acidified solution,
it can achieve a <i>z</i> = +3 state and seen in positive ion mode; this is
because protons will bind to the side chains of the one R and K residue, as
well as the N-terminus.. In neutral solution it probably achieves <i>z</i> = +2
state, with the side chains of the one R and K residue being charged. At a
slightly higher pH (perhaps 9 or 10), it might be possible to obtain a peptide
with <i>z</i> = +1. In most cases, a mixture of [M + H]<sup>+</sup> and [M +
2H]<sup>2+</sup> ions are seen. The monoisotopic mass of singly protonated
substance P, the [M + H]<sup>+</sup> form, is 1347.7360 amu. The monoisotopic
mass of the [M + 2H]<sup>2+</sup> form is 674.3719 amu. In the mass spectrum
above, both of these ions appear, and different mass ranges are shown
containing both isotope clusters. Note that for the <i>z</i> = +1 ion with
monoisotopic peak at <i>m/z</i> 1347.7, the cluster peaks are separated by 1
amu (1 Da): &Delta;<i>m</i> = 1. Thus <i>z</i> = 1 / &Delta;<i>m</i> = 1 / 1 =
1. For the <i>z</i> = +2 ion with monoisotopic peak at <i>m/z</i> 674.6, which
is about 0.2 amu off from the calculated 647.4, the peaks are separated by
approximately 0.5 amu (0.5 Th): &Delta;<i>m</i> = 0.5. Thus <i>z</i> = 1 /
&Delta;<i>m</i> = 1 / 0.5 = 2. If there were an ion at <i>z</i> = &plusmn;3,
then the isotope cluster peaks would have &Delta;<i>m</i> = 0.333. For the
<i>z</i> = &plusmn;4, then &Delta;<i>m</i> = 0.25.
<p>
It should be noted that the sign of the charge does not matter, but rather the
magnitude.  Clearly if the ion is detected in negative ion mode, then all ions
detected must be negative in charge.  The converse is true for positive ion mode.
<p>
<b>Instrument resolution</b>. If charge can be computed from the determination
of &Delta;<i>m</i> of the series of isotope cluster peaks, then it is the case
that the peaks of the isotope cluster must be resolved by the instrument. For
most even low resolution mass spectrometers, resolving <i>z</i> = +1 and +2 ion
isotope cluster peaks is not difficult. However, for higher charge states,
higher resolution instruments are absolutely required. In the case of
biomolecules such as proteins and peptides, it might help to perform the
analysis in electrospray positive ion mode at different pH. This would create
the appearance and disappearance of the ionic species at different <i>m/z</i>
values and consistent with the formula <i>m/z</i> = (M + <i>z</i>H) &divide;
<i>z</i> for positive ion mode and
<i>m/z</i>&nbsp;=&nbsp;(M&nbsp;&minus;&nbsp;<i>z</i>H)&nbsp;&divide;&nbsp;<i>z</i>
for negative ion mode.

<h2>Deconvoluting Multiple Mass-To-Charge Signals</h2>
<p>
Many biomolecules, particularly peptides and proteins, have multiple ionizable
functional groups on the molecule.  These groups usually are ionized with a
proton at different pH.  Many forms exist simultaneously at the pH range, and
can be observed in the electrospray mass spectrum.  Bioinformatics analysis using
<i>maximum entropy</i> algorithms can take the numerous <i>m/z</i> values in
an electrospray and compute the parent mass of the molecule that produced them.
The computed mass has a theoretical zero charge state.
<p>
For an ion such as a protein that is multiply charged in positive ion mode
typically by the binding of multiple protons, the <i>m/z</i> is clearly:
<table class="eqnserif">
<tr><td class="numer"><i>m</i>
 <td rowspan="2"> =
 <td class="numer"><i>M</i> + <i>nH</i>
 <td rowspan="2"> &nbsp;&nbsp;[1]
<tr><td><i>z</i>
 <td><i>n</i>
</table>
<p>
The parameter of interest is <i>M</i>, the actual mass of the molecule.
To solve for it, we need to get <i>m/z</i> data on at least two charged
forms of the molecule.  The difference can be a single charge:
<p class="eqnserif">
<i>n</i><sub>2</sub> = <i>n</i><sub>1</sub> + 1 &nbsp;&nbsp; [2]
<p>
Thus the two forms of the ions will be:

<table class="eqnserif">
<col span="2"><col style="width:4em;"><col span="2"><col style="width:2em;"><col>
<tr><td rowspan="2">(<i>m/z</i>)<sub>1</sub> =
 <td class="numer"><i>M</i> + <i>n</i><sub>1</sub><i>H</i>
 <td rowspan="2">
 <td rowspan="2">(<i>m/z</i>)<sub>2</sub>&nbsp;&nbsp;=&nbsp;&nbsp;
 <td class="numer"><i>M</i> + <i>n</i><sub>2</sub><i>H</i>
 <td rowspan="2"> =
 <td class="numer"><i>M</i> + (<i>n</i><sub>1</sub> + 1)<i>H</i>
<tr><td><i>n</i><sub>1</sub>
 <td><i>n</i><sub>2</sub>
 <td><i>n</i><sub>1</sub> + 1
</table>
<p>
Note the substitution for <i>n</i><sub>2</sub> above.  Both equations
are rearranged to solve them for <i>n</i><sub>1</sub>:

<table class="eqnserif">
<col span="2"><col style="width:4em;"><col span="2">
<tr><td rowspan="2"><i>n</i><sub>1</sub> =
 <td class="numer"><i>M</i>
 <td rowspan="2">
 <td rowspan="2"><i>n</i><sub>1</sub>&nbsp;&nbsp;=&nbsp;&nbsp;
 <td class="numer"><i>M</i> + <i>H</i> &minus; (<i>m/z</i>)<sub>2</sub>
<tr><td>(<i>m/z</i>)<sub>1</sub> &minus; <i>H</i>
 <td>(<i>m/z</i>)<sub>2</sub> &minus; <i>H</i>
</table>
<p>
Set the two equation for <i>n</i><sub>1</sub> equal to each other, and
solve for the mass <i>M</i>:

<table class="eqnserif">
<col span="2"><col style="width:2em;">
<tr><td rowspan="2"><i>M</i> =
 <td class="numer"><i>H</i> * [ (<i>m/z</i>)<sub>1</sub> +
   (<i>m/z</i>)<sub>2</sub> &minus; <i>H</i> ] &minus;
   (<i>m/z</i>)<sub>1</sub>(<i>m/z</i>)<sub>2</sub>
 <td rowspan="2"> &nbsp;&nbsp; [3]
<tr><td>(<i>m/z</i>)<sub>2</sub> &minus; (<i>m/z</i>)<sub>1</sub>
  &minus; <i>H</i>
</table>
<p>
Eq. 3 is independent of the charge state (it does not require
knowledge of the charge of any species), and it is solved by knowing
the values of any two <b>adjacent</b> <i>m/z</i> species, which are
easily obtained from the mass spectrum.
<p>
Suppose horse heart myoglobin gives two <em>adjacent</em>
<i>m/z</i> species with values (<i>m/z</i>)<sub>1</sub> = 1060.5 and
(<i>m/z</i>)<sub>2</sub> = 998.1

<table class="eqnserif">
<tr><td rowspan="2"><i>M</i> =
 <td class="numer">1.008 * [1060.5 + 998.1 &minus; 1.008] &minus;
 (1060.5)(998.1)
 <td rowspan="2"> = 16660.56
<tr><td>998.1 &minus; 1060.5 &minus; 1.008
</table>
<p>
For Eq. 3 to work, note that (<i>m/z</i>)<sub>1</sub>
<strong>must</strong> have a larger value than (<i>m/z</i>)<sub>2</sub> because
the molecule with (<i>m/z</i>)<sub>2</sub> has the greater charge
(<i>n</i><sub>2</sub> = <i>n</i><sub>1</sub> + 1).  Moreover, the
two species <strong>must</strong> be adjacent in the spectrum because they
are separated by only one charge.
<p>
For any unfragmented, multiply charged pure ion with two
<em>adjacent</em> <i>m/z</i> species
in the mass spectrum, enter the <i>m/z</i> for the species in the
table (taking care that (<i>m/z</i>)<sub>1</sub> is larger than
(<i>m/z</i>)<sub>2</sub>).  The mass of the parent ion (singly charged)
and charge of the species will be given in the result.

<table id="pMassTbl">
<col span="2">
<col style="text-align:right;width:10em;">
<col style="width:10em;">
<col style="width:8em;">
<tr><td>(<i>m/z</i>)<sub>1</sub>:
 <td><input type="text" size="12" maxlength="12" id="mz1">
 <td><i>z</i><sub>1</sub> =
 <td><span id="z1"></span>
 <td rowspan="3" style="font:bold 90% Arial,Helvetica,sans-serif;color:red;">
   <span id="mzerr"></span>
<tr><td>(<i>m/z</i>)<sub>2</sub>:
 <td><input type="text" size="12" maxlength="12" id="mz2">
 <td><i>z</i><sub>2</sub> =
 <td><span id="z2"></span>
<tr><td colspan="2"><button type="button" onclick="calcParentMass();">Calculate</button>
 <td>Parent mass =
 <td><span id="parentMass"></span>
</table>

<h2>Charged Ratio Analysis Method</h2>
<p>A method for deconvoluting electrospray mass spectra has been described
identifying the charged-species that affect the value of <i>M</i>.
Eq. 4 below is effectively the same as eq 1 in the previous section except
that the ratio (<i>m/z</i>)<sub><i>n</i></sub> is symbolized as
<i>R<sub>z</sub></i>.
<p style="text-align:center;" class="eqnserif">
<i>R<sub>z</sub></i> = (<i>M &plusmn; zm</i>)/<i>z</i> &nbsp;&nbsp; [4]
<p>
The mass-to-charge ratios of two ions of species <i>M</i> are now
themselves expressed as ratios:

<table class="eqnserif">
<tr><td style="border-bottom:1px solid black;">
  (<i>R<sub>z</sub></i>)<sub>a</sub>
 <td rowspan="2"> =
<td style="border-bottom:1px solid black;"><i>z</i><sub>b</sub>(<i>M &plusmn;
	z</i><sub>a</sub><i>m</i>)
<td rowspan="2">&nbsp;&nbsp;&nbsp;[5]
<tr><td>(<i>R<sub>z</sub></i>)<sub>b</sub>
 <td><i>z</i><sub>a</sub>(<i>M &plusmn; z</i><sub>b</sub><i>m</i>)
</table>
<p>
When the mass of the species <i>M</i> is very large, as for proteins,
that is, when <i>M</i> &gt;&gt; <i>z</i><sub>a</sub><i>m</i> or
<i>z</i><sub>b</sub><i>m</i>, then eq 5 simplifies to

<table class="eqnserif">
<tr><td style="border-bottom:1px solid black;">
  (<i>R<sub>z</sub></i>)<sub>a</sub>
 <td rowspan="2"> =
<td style="border-bottom:1px solid black;"><i>z</i><sub>b</sub>
<td rowspan="2">&nbsp;&nbsp;&nbsp;[6]
<tr><td>(<i>R<sub>z</sub></i>)<sub>b</sub>
 <td><i>z</i><sub>a</sub>
</table>
<p>
A comparison of the ratio of <i>m/z</i> values (<i>R<sub>z</sub></i>)
to the unique ratio of any two integers identifies the charge state
values for each of the ions.  Once the charge states are identified,
the <i>R<sub>z</sub></i> values for two ions are multiplied by their
charge state values.
<p style="text-align:center;" class="eqnserif">
(<i>R<sub>z</sub></i>)<sub>a</sub> <i>z</i><sub>a</sub> =
		<i>M &plusmn; z</i><sub>a</sub><i>m</i> &nbsp;&nbsp; [7]<br>
(<i>R<sub>z</sub></i>)<sub>b</sub> <i>z</i><sub>b</sub> =
		<i>M &plusmn; z</i><sub>b</sub><i>m</i> &nbsp;&nbsp; [8]<br>
(<i>R<sub>z</sub></i>)<sub>a</sub><i>z</i> &minus;
   (<i>R<sub>z</sub></i>)<sub>b</sub><i>z</i>  =
		&plusmn;<i>m</i> (<i>z</i><sub>a</sub> &minus;
					<i>z</i><sub>b</sub>) &nbsp;&nbsp; [9]
<p>
These should report the mass of the charge-carrying species <i>m</i>
without assuming that species <i>m</i> is uniformly a single species
such as a proton, sodium, or potassium.
<p>
The example given is of lysozyme (average mass 14305.1438 Da).  In the
electrospray, <i>m/z</i> species of 1301.47452 and 1431.52218 are produced.
The equations identify these as having charges of +11 and +10, respectively.
The (<i>R<sub>z</sub></i>)<sub><i>n</i></sub> values are then multiplied
by the <i>z</i><sub>n</sub>.  The mass of the charge-carrying species
is then determined from the difference (eq. 6):
<p style="text-align:center;">
      (1431.52218 &times; 10) &minus; (1301.47542 &times; 11) = 1.0078
<p>
This is the mass of the proton, and so thus the proton is identified
as the charge carrier accounting for the difference in these species.
<p>
Use the table below to enter in your adjacent <i>m/z</i> species
to determine the mass of the charge carrier and the parent mass.

<table id="pMassTbl2">
<col span="2">
<col style="text-align:right;width:20em;">
<col style="width:10em;">
<col style="width:8em;">
<tr><td>(<i>m/z</i>)<sub>1</sub>:
 <td><input type="text" size="12" maxlength="12" id="CRAMmz1">
 <td><i>z</i><sub>1</sub> =
 <td><span id="CRAMz1"></span>
 <td rowspan="3" style="font:bold 90% Arial,Helvetica,sans-serif;color:red;">
   <span id="CRAMmzerr"></span>
<tr><td>(<i>m/z</i>)<sub>2</sub>:
 <td><input type="text" size="12" maxlength="12" id="CRAMmz2">
 <td><i>z</i><sub>2</sub> =
 <td><span id="CRAMz2"></span>
<tr><td rowspan="2" colspan="2"><button type="button" onclick="calcParentMass('CRAM');">Calculate</button>
 <td>Adduct (Charge-Carrying) mass =
 <td><span id="adductMass"></span>
<tr><td>Parent mass =
 <td><span id="CRAMparentMass"></span>
 <td>
</table>
<p>
See S. D. Maleknia and K. M. Downward (2005) <i>Anal. Chem.</i> <b>77</b>,
111-119.

<h2>The Effect of Isotopes</h2>
<p>Mass spectral peaks are really an <i>envelope</i> of peaks.  That is
there is a main peak (one with the highest relative intensity) and
a set of peaks on its shoulder whose on peak heights (relative intensities)
decline in value on either side.
<p>
The envelope of peaks actually represents one molecular species in all
of its isotopic forms.  Suppose the dipeptide Gly-Lys (GK) is being
analyzed by a mass spectrometer.  The molecular formula of Gly-Lys
is actually
<p style="text-align:center;">
H<sub>2</sub>N-CH<sub>2</sub>-COO-NH-CH(CH<sub>2</sub>CH<sub>2</sub>CH<sub>2</sub>CH<sub>2</sub>NH<sub>2</sub>)-COOH
<p>
or C<sub>8</sub>H<sub>17</sub>N<sub>3</sub>O<sub>4</sub>.  But in a single
molecule of GK, six of the carbon atoms may be isotope 12 (6 protons and
6 neutrons), and two of the carbon atoms may be isotope 14 (6 protons and
8 neutrons).  Nitrogen, oxygen, and certainly hydrogen have isotopic forms
as well.  A mass spectrometer is able to distinguish between all sorts
of isotopic forms not only of an atom, but of a molecule.  In fact
if a molecule has 8 carbon atoms, and carbon has 4 isotopic forms, then
a molecule can potentially have 32 different masses based on carbon alone.
Now when one includes nitrogen and its 4 isotopic forms and the number
of nitrogen atoms, oxygen and its count of atoms and 5 isotopic forms,
and hydrogen and its count of atoms and 3 isotopic forms, then the number
of potential masses climbs into the thousands.
<p>
So why are there not thousands of peaks that form the envelope of a mass
spectral peak?
<h3>Atomic Mass As The Weight Average of Isotopes of An Atom:  Relative
Abundance</h3>
<p>What is the atomic mass of a hydrogen atom?  1.0079 atomic mass units (amu)?
<p>
Not exactly.  The mass of an atom actually represents an average&mdash;that is
the weight or mass average&mdash;of the mass of all the isotopes of the
hydrogen atom.  The mass of <sup>1</sup>H is 1 amu.  The mass of
<sup>2</sup>H (deuterium) is 2 amu.  The mass of <sup>3</sup>H (tritium)
is 3 amu.  The weight average is calculated as:
<p class="eqnserif">
weight average =
&sum;<span style="font-size:60%;margin-left:-1.25em;vertical-align:-1.5em;"><i>i</i>=1</span>
<span style="font-size:60%;margin-left:-1.5em;vertical-align:1.5em;"><i>n</i></span>
 <i>x<sub>i</sub> m<sub>i</sub></i>
<p>
where <i>i</i> represents one of the total number of particular isotopes
of an atom, <i>n</i> represents the total number of isotopes of an atom,
<i>x<sub>i</sub></i> represents the fraction of the atoms that has a
particular isotope (known as the <b>relative abundance</b>), and
<i>m<sub>i</sub></i> is the mass of that isotope in atomic mass units.
<p>
Note that:
<p class="eqnserif">
&sum;<span style="font-size:60%;margin-left:-1.25em;vertical-align:-1.5em;"><i>i</i>=1</span>
<span style="font-size:60%;margin-left:-1.5em;vertical-align:1.5em;"><i>n</i></span>
 <i>x<sub>i</sub></i> = 1
<p>
The relative abundance cannot be determined from knowledge of the weight
average (the atomic weight) and the forms of isotopes an atom has unless
that atom has only two isotopes.  The best way to determine the relative
abundance is to use a mass spectrometer, which can distinguish between
isotopes.
<p>
Once the relative abundance is known (and published in the tables of
the <i>CRC Chemical Handbook of Chemistry &amp; Physics</i>), these values
can be used to predict the intensities of shoulder peaks relative to the
main peak.


<!--  Protein Identification -->

<h1>The Aim:  Identification of Peptides and Proteins</h1>


<!--  Practical Considerations -->
<h1>Practical Considerations</h1>

<h2>Choosing (Buying) A Mass Spectrometer</h2>
<p>[Source:  Chapter G4 <q>Choice of Mass Spectrometer,</q>
<i>Back to Basics</i>, Micromass UK Ltd.]
<p>
Mass spectrometers were necessarily built by the scientists who wanted
to use them for their projects.  Nowadays, commercial mass spectrometers
are available with different types of analyzers and sources, and these
are easily adaptable by scientists doing methods and technology development.
The more basic models are used by those who are not experts in mass
spectrometry theory, but users who want fast answers which the use of an MS
provides.
<p>
In planning the purchase of a mass spectrometer, the user must prepare
as follows:
<ol>
<li>Write down the exact analytical requirements both for the short-term
and the long-term.
<li>Estimate the throughput (demands for time and scheduling) required
for such an instrument, as such systems first start out to be used
infrequently but used more once individuals learn about their capabilities.
<li>Call the commercial sales representative and discuss these
needs on paper.  It is easier for the representative to provide the
list of options that range between the economic system (which may
really be inadeqate) and a more full-featured system (which may
be excessive).
</ol>

<h2>Sequencing Theory</h2>
<p>Structural information from proteins and proteolytic digests/fragments
can be used to sequence peptides.  One method is the protein ladder
sequencing method, and the other takes advantage of the instability of
protein ions in which the molecule decays or fragments.

<h3>Ladder Sequencing</h3>
<p>Reported by Patterson <i>et al.</i>, <i>Anal. Chem.</i> <b>67</b>: 3971, 1995),
the ladder sequencing method involves analysis of the time- or
concentration-dependent degradation from either the N- or C-terminus
using an exopeptidase.  Mass differences will correspond to a single
amino acid residue.
<p>
An example of a typical experiment is the use of carboxypeptidase Y
(CPY) at different concentrations (concentration-dependent) in the digestion
of ACTH 18-39 (mass = 2465.70 Da), done directly on the sample support
with 1 pmol peptide (time was kept constant at 15 min).  The sample
was placed in a delayed-extraction ToF-MS set at 25 kV acceleration,
with a 150 ns pulse delay, and pulse voltage at 1.3 kV, with 50 laser pulses
per spectrum.


<!-- ===================================================================
     ===================================================================
	  =================================================================== -->



<h1>References</h1>
<ul>
<li>Sigma-Aldrich <i>Analytix</i>,
<a href="http://www.sigmaaldrich.com/img/assets/4242/fl_analytix6_2001_new.pdf">June 2001</a>.
</ul>

</div>  <!-- class=outerblock -->

</body>
</html>