<!DOCTYPE html>
<html>
<head>
<title>Reference for Proteomics:  Part 1</title>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8">
<link href="/style/std.css" type="text/css" rel="stylesheet">
<link href="/style/proteomicsRef.css" type="text/css" rel="stylesheet">
<script type="text/javascript" src="/include/colorContrasting.js"></script>
<script type="text/javascript" src="/include/dom-xml.js"></script>
<script type="text/javascript" src="/include/css.js"></script>
<script type="text/javascript" src="/include/htmlTOCgenerator.js"></script>
<style type="text/css">
	table.eqn, table.eqnserif {border:none;}
	#motd {margin:0 auto;width:40em;font-size:80%;}
	button {background:none;border:none;color:blue;cursor:pointer;
	     margin:0;padding:0;font:bold 85% Tahoma,sans-serif;}
</style>
<script type="text/javascript" src="/include/natSciDocs.js"></script>
<script type="text/javascript" src="/include/xmlTextImport.js"></script>
<script>
function newDoc(docname) {
   switch (docname) {
   case "genlabpract":
      window.open("/reference/GeneralLaboratoryPractice.html", "", "");
   }
}
</script>
</head>
<body onload="initNatSciDoc();">


<div id="top-panel">
<p id="index-button-left">
<button type="button" onclick="showIndex('asWindow');">Show index in window</button>
<p id="index-button-right">
<button type="button" onclick="showIndex('embedded');">Show index embedded in document</button>
<p id="contact-author" style="display:none;">
<a href="mailto:mavigozler@yahoo.com?Subject=HtmlToc%20Object%20Not%20Found">Contact
Document Author</a>
<p>
<!-- motd -->
<p id="title">
Reference for Proteomics</p>
<p id="subtitle">
Part 1 (of 2)
</div>

<div id="scrolling-panel">

<div class="contentblock" id="first-content-block">
This reference is divided into multiple parts.  To the left is a clickable
document outline for the current part.  Unfortunately a table of contents
for all parts has not been set up on a single page (yet, but will be).
<p>
Currently there are two parts.  <a href="ProteomicsReference2.html">Part 2</a>
has material mostly on mass spectrometry-related proteomics.

<hr>

<h1 id="setup">Laboratory Setup:  Establishing the Proteomics Laboratory</h1>
<p>Liebler (<q>Essential Elements of the Proteomics Laboratory,</q>
Ch. 1 in <i>Proteomics in Cancer Research</i>, D. C. Liebler, ed., Wiley,
2005) has provided a guide regarding the role and functions of the
proteomics laboratory, described as <q>essential capabilities.</q></p>
<ol>
<li>protein and peptide separations</li>
<li>protein digestion</li>
<li>simple protein identification</li>
<li>identification of protein components of complex mixtures and
multiprotein complexes</li>
<li>mapping protein modifications</li>
<li>quantitative analysis</li>
<li>proteome profiling</li>
</ol>
<p>He then breaks down the methods and techniques in common use by which
these goals are achieved.  His descriptions will not be discussed here, but
only those list of techniques and methods which are justified as essential
to the laboratory:</p>

<ul>
<li>2-D gel electrophoresis (separates intact proteins by two chemistries)</li>
<li>gel imaging and spot excision</li>
<li>1-D (SDS) PAGE</li>
<li>solution phase IEF (available through kit-based apparatus)</li>
<li>Purification/Protein Chromatography:
 <ul>
 <li>preparative gel filtration</li>
 <li>preparative ion-exchange</li>
 <li>immunoaffinity chromatography</li>
 </ul></li>
<li>2-D or multidimensional LC-MS/MS</li>
<li>Enzymatic Digestion in solution and in gel: trypsin, V8 protease,
nonspecific proteases (pepsin, proteinase K)</li>
<li>proteome profiling by MS of <i>intact</i> proteins (MALDI for proteins)</li>
</ul>

<h2>Mass Spectrometry Capabilities</h2>
<p>Ion trap MS systems are the most affordable ($175-300K) and typically
used in ESI-LC-MS, some having MALDI options too.  They can be used in
protein IDs as well as modification analysis.  Quantitation is achieved
by comparing light and heavy isotope-tagged forms. 3-D ion traps can
do one MS/MS spectrum per second, and new linear ion traps do
up to five spectra per second, with unit resolution (error is &plusmn;0.5
<i>m/z</i> units).  Higher order (MS/MS/MS or MS<sup>3</sup>) configurations
are found as well.</p>
<p>MALDI-ToF can analyze up to <i>m/z</i> 250,000 in linear modes.  Sensitivity
decreases with increasing <i>m/z</i>.  Reflectron or delayed extraction
capabilities improves resolution and mass accuracy.  Systems range in
cost from $200-500K.  Tandem MS configurations improve the ability for
MALDI-ionized sambples to be sequenced.  Proteomic profiling of serum
or tissue of intact proteins has caused a resurgence in MALDI-ToF analysis.
SELDI-ToF is part of the proteomic profiling popularity but while its
small linear ToF has high sensitivity, it has limited resolution and
mass accuracy.</p>
<p>
QqToF MS systems have a dc/rf quadruple (Q), followed by an rf-only
quadrupole (q) and reflectron ToF.  This provides high sensitivity,
resolution, and mass accuracy for precursor and product ions.
Resolution in 0.01 <i>m/z</i> unit is possible (necessary for accurate
charge state determinations).  Cost is between $300-600K.</p>
<p>
Triple quads were used in MS/MS of peptides.  Ion traps are better at
fragmentation however and used in that case.</p>
<p>
MALDI-ToF-ToF can be used to select precursors (first ToF) whose
product ion masses are determined in the second ToF.  CID spectra
in TOF-TOF are better than in ion trap or QqTOF.  Resolution and mass
accuracy are comparable to QqTOF.</p>
<p>
FTICR makes use of a trap in a 7-12 T magnetic field generated by superconducting
magnet.  Ionization sources can be MALDI or ESI.  Analysis is <q>top-down</q>
in that whole proteins are excited and fragmented, generating MS/MS of
constituent peptides (as opposed to <q>bottom-up</q> in which
peptides must be generated outside the MS by digestion).</p>
<p>
Additional MS designs continue, usually hybrids of known mass analyzer
configurations.  Qq-trap is quadrupole ion trap.  trap-FT is a linear
ion trap with FTICR.  The linear trap technology brings higher sensitivity
and rapid scanning.  Qq-trap exceeds 3Q performance for quantitation
by selected reaction monitoring (SRM).  By switching from positive to
negative ion modes alternately, Qq-trap detects posttranslational modifications,
and precursor scanning will detect phosphopeptide precursors.
The utility of trap-FT will be in analyzing complex protein or
peptide mixtures (without prior purification by HPLC or gels!).</p>

<h2>Data Management and Processing</h2>
<p>Stored spectra take a great deal of data size for raw and cooked data.
The infrastructure of a laboratory should include the ability to track
samples, store files, and archive data.  An LIMS may be necessary for this
purpose depending on work volume.</p>

<!-- ================================================================
       GENERAL LABORATORY PRACTICE
====================================================================== -->

<h1 id="labprinciples">General Laboratory Practice</h1>
<p>
This section has been placed in another document because it applies to a wide
variety of laboratory practices outside of proteomics, especially general and
organic chemistry, and laboratories doing biochemistry and molecular and cellular
biology.
<p>
This document also contains a link to a separate document for how to set
up quantitative calculations in chemistry and all specialites of chemistry
(biochemistry, molecular and cellular biology, the omics disciplines).
<p>
The document is found <button onclick="newDoc('genlabpract');">here</button> 
(opens up a separate window).

<h1 id="labware">Labware:  Containers, Utensils</h1>
<p>
There are essentially two classes of containers and utensils in the
proteomics laboratory or any other kind of laboratory:  disposable and
non-disposable.
<p>
Disposable containers and utensils certainly have their advantages,
the primary (most significant) being that if the container is used to
hold sample material (as opposed to reagents or stocks), then there
is virtually no concern about sample carryover or cross-contamination.
The other major advantage is that there is nothing to clean up afterward:
the container or utensil is just tossed in the <em>trash/waste</em>.
Which brings up the disadvantage of the disposable product:  for the
concerned environmentalist, the disposable product adds to the
garbage heap or the landfill.  And when that product is not recyclable
or rapidly degradable (or otherwise environmentally friendly), it
might cause real stress for the green scientist.  Maximizing the
use of particularly disposable plastics used in liquid handling/transfer
in sterile cell culture is discussed elsewhere.
<p>
The advantages and disadvantages of nondisposable containers and utensils
are a direct contrast to those of disposable.  They do not clutter up the
environment, but must be cleaned.

<h2>Clean Glassware and The Cleaning of Glassware</h2>
<p>
Clean glassware is a must, and thus proper cleaning must be done.
Even on new glassware.  One might erroneously believe that glassware
or containers directly out of the box is as clean as one can get, but
the scientist never knows how it has been handled.
<p>
A first principle for any scientist is never to trust the beaker or
flask pulled out from the cabinet.  Most glassware has been sitting
too long in the shelf of a cabinet and has gathered dust anyway, so
a rinse with distilled or deionized water is a must, followed by a
drip or air dry.
<p>
If the distilled or deionized water does not leave a fine film along the
glassware when poured out, but it instead beads up into drops, the
glassware is dirty and maybe has a fine oily film on it.  A detergent
wash is needed.
<p>
If a service is used for glassware cleaning, make sure they are using
the correct detergents.  Even when the glassware or utensile come back,
give them a thorough dist. or deion. water rinse and allow to dry,
in case detergent residues remain.
<p>
Not just any detergents and cleaning products can be used.  In particular,
where mass spectrometry is done, use of household dishwashing detergents
or hand soaps is strictly forbidden.  These contain chemicals that often
show up as background contaminants impossible to eliminate.
<p>
The most general purpose laboratory cleaner is something like <b>Alconox</b>.
Prepared as a 1% solution, glassware or other materials can soak in it,
or a brush can be dipped into it and used to scour materials.  Another
popular brand is <b>Decon 90</b> (Decon Labs).
<p>
When using detergents, rinse several times with warm to hot tap water
to get rid of the detergent, then several times with distilled or deionized
water.  Let drip or air dry.
<p>
<b>Glass pipets or burets</b> might also be cleaned with special chromic
acid formulations:  these are nearly satd solutions of
Na<sub>2</sub>Cr<sub>2</sub>O<sub>7</sub> &bull; 2 H<sub>2</sub>O (soluble
to 70 g/l; or the potassium dichromate, soluble to 5 g/l) in conc.
H<sub>2</sub>SO<sub>4</sub>, letting it
sit for several hours (overnight).  After a distilled water rinse, it
is drained dry.  Other acid treatments include mixtures of sulfuric acid
and fuming nitric acid.
<p>
Strong alkaline treatments include 100 g KOH in 50 ml water, followed by
making it up to 1 L with industrial methylated spirit (MeOH).

<h2>Container Materials &amp; Construction</h2>
<p>
In proteomics laboratories, there is not much need for a wide array of
containers as for analytical or organic chemistry laboratories.  Glassware
is typically the Corning Pyrex or Kimax brand and is suitable, which
is borosilicate glass.  Tempered glass is more heat-resistant.
Never apply a burner flame directly on glass, but use a wire gauze
to distribute the heat evenly.
<p>
<b>Porcelain</b> vessels and dishes can withstand high temperatures
and are used in making crucibles for igniting or heating solids.
They are useful for evaporations, and resist attack by alkaline
solutions better than glass.
<p>
<b>Fused silica</b> comes in a translucent and transparent grade.
The translucent is more cost-effective.  It shows a greater resistance
to heat shock and is more durable with acids at high temperature, but
is less resistant to alkaline conditions compared to porcelain.
Corning Vycor (96% silica glass) is a transparent form.

<p>
<b>Metal</b> containers include <b>platinum</b>, which is extremely
resistant to chemical and heat;  platinum containers are hardened
with alloyed metals (rhodium, iridium, or gold in trace amounts).
A better grade is Zirconia Grain Stabilized (ZGS) made using
zirconium (IV) oxide.  Crucibles of platinum should be heated with
Pt or silica triangles; others are unsuitable since they can damage
platinum.  A lumnious flame may cause formation of carbides of platinum,
disintegrating the metal surface or causing it to become brittle.

<h2>Silanizing/Siliconizing Plasticware/Glassware</h2>
<p>
Many working in proteomics laboratories caution against silanizing
plasticware, but instead they recommend buying containers that are
reputed to be low-adsorbing for peptides.  The experience is related
that the silane compound can not be thoroughly washed from the container
such that it does not affect mass spectrometric analysis, and silanes
or their derivative compounds can especially cause fouling of electrospray
ionization sources.
<p>
However, if there is an absolute need to silanize a product, the
procedure is given below.

<p>
<span style="text-decoration:overline;"> (taken from Sambrook,
Fritsch, Maniatis, <i>Molecular Cloning: A Laboratory Manual,</i> 1989)</span>

<p>
Method used by B. Seed:
<ol>
<li>Put items to be silanized/siliconized in a dessicator.
Place in fume hood.
<li>In the fume hood, to a small beaker, add 1 ml of dichlorodimethylsilane
[ (CH<sub>3</sub>)<sub>2</sub>SiCl<sub>2</sub> ].  Place in the dessicator.
<li>Through a trap to a vacuum pump or line, apply suction until
dichlorodimethylsilane starts to boil.
<li>Immediately seal off the vacuum by turning the dessicator top (if
it works that way), or cl&#097;mping the hose line to the vacuum trap.
It is essential that the vacuum be maintained.
<b>Caution:  damage to vacuum pump seals can occur if silane
compound gets to pump, so be prepared to act immediately</b>.
<li>After 1-2 hours, vent the dessicator to the fume hood.
<li>For glassware:  bake for 2 hr at 180&deg;.
For plasticware:  rinse extensively in water of the appropriate quality
and then be allowed to dry, perhaps in an oven at 40-50&deg; if in a hurry.
</ol>
<p>
Another method is to soak or rinse plasticware in a
5% dichlorodimethylsilane in chloroform or heptane.
As solvent evaporates, a coat is deposited on the material.
Glassware should be rinsed in water or baked at 180&deg; for 2 h.

<h2>Shimadzu Electronic Weighing Balances</h2>
<p>
A special <a href="ShimadzuBalanceUse.html">document</a>
has been prepared as a quick reference on the use, care, and maintenance
of the Shimadzu electronic weighing balances available in the Proteomics
Unit laboratory.

<!-- Reagents, Solvents, Chemicals, Compounds Used In Proteomics -->
<h1 id="reagents">Reagents, Solvents, Chemicals, Compounds Used In Proteomics</h1>

<h2 id="water">Water</h2>
<p>
For 99% of all methods and processes in proteomics, especially where the
water makes contact with the substances of interest (peptides and proteins),
the water needs to be of <b>Type I</b> quality.
<p>
This is water that is fully deionized (to &ge;18 M&Omega; cm) and filtered
for volatile and involatile organics (through activated carbon) for which
the metal content is at trace levels.  It might also be filtered for
particles (through a high-flow 0.22 or 0.45 &micro;m filter).  This is
water typically obtained through an on-demand water purification system
(such as Millipore Elix/MilliQ gradient, or Barnsted system).
<p>
The British Standard 3978 (ISO 3696-1987) specifies three different grades
of water (see table below).  Grade 3 is prepared by single distillation of
tap water or deionization or reverse osmosis, and suitable for general
analytical purposes.  Grade 2 is suitable for AAS and determination of
substances in trace quantities; prepared by re-distillation of Grade 3
or RO-treated or deionized water.  Grade 1 is usable for HPLC and ultratrace
amount determinations, and is obtained by RO treatment or deionization of
Grade 2 water (if either not done), followed by filtration through 0.2 &micro;m
filter.
</div> <!-- class=contentblock -->

<div class="widetable">
<table class="centeredTD TD6">
<caption>British Standard Grades of Water</caption>
<tr><th style="background-color:#dad;">Grade
 <th>  Conductance <br> 
     (mS m<sup>&minus;1</sup>) at 25&deg;
 <th>Oxidizable Matter (mg&nbsp;oxygen&nbsp;L<sup>&minus;1</sup>)
 <th>OD<sub>254</sub> 1&nbsp;cm&nbsp;cell
 <th>Residue After Evaporation <br>  
      (mg&nbsp;kg<sup>&minus;1</sup>)
 <th>SiO<sub>2</sub> (mg&nbsp;L<sup>&minus;1</sup>)
 <th>pH at&nbsp;25&deg;
<tr><th>1 <td>0.01 <td>&dagger; <td>0.001   <td>&dagger; <td>0.01    <td>*
<tr><th>2 <td>0.1  <td>0.08     <td>0.01    <td>1        <td>0.02    <td>*
<tr><th>3 <td>0.5  <td>0.4      <td>&Dagger;<td>2        <td>&Dagger;<td>5.0-7.5
<tr><td style="background-color:#eee;text-align:left;" colspan="7">
<pre>
&dagger; Not applicable
&Dagger; Not specified
* pH results difficult in highly purified water; results of doubtful significance
</pre>
</table>
</div> <!-- class=widetable -->

<div class="contentblock">

<h3>Water in Mass Spectrometry</h3>
<p>
The highest quality water is necessary for all mass spectrometry operations,
both MALDI-MS and ESI-MS.
<p>
But it is especially critical for ESI-MS since water is usually the
principal solvent and present at least in at least a 50% proportion by
volume.  In positive mode ESI-MS, a significant count of background
ions can be seen in water of all kinds in the <i>m/z</i>100-800 region,
one important to individuals generating CID MS/MS spectra, since a high
background can make spectra have a low confidence in interpretation.
These background ions are typically plasticizers leaching out into
water or the concentrated acids if they are stored in plastic containers.



<h2 id="denaturants">Denaturing &amp; Chaotropic Reagents</h2>
<p>
Urea and thiourea are commonly used in extraction for proteome analysis
by 2DGE.  They sequester water and specifically breakdown protein
secondary structure (alpha helices and beta sheets), which virtually
form the structure of all globular and fibrous type proteins.
The Walker textbook mentions that thiourea will increase the number
of proteins extracted, but that these proteins are generally cytoplasmic
or water-soluble, and not membrane proteins.
<p>
Use of urea and thiourea is limited to a strict temperature range.
Below 15&deg;, they fall out of solution, so ice-cold solutions are
not possible when at high concentrations.  They should not be kept
above 30&deg; for any length of time as well, since the amide can be
hydrolyzed to form cyanates which are themselves reactive with protein.
This can lead to unpredictable peptide analysis during MS.

<h2 id="redagents">Reducing Agents</h2>
<p>
[some of the information given in this section was obtained from
sales literature published by Pierce Biotechnology, Inc.]
<p>
Reducing agents serve two particular and typically separate purposes
in proteomics.
<p>
<b>Elimination of Oxidizing Compounds</b>.  Dissolved oxygen molecules
have the potential to make trouble.  They can react in particular with
dissolved transition metals, particular iron (Fe<sup>2+</sup>) which
is found everywhere and especially in tissues, to form all sorts of
oxygen-based free radicals (the reactive superoxide anion and the even more
reactive hydroxyl free radical anion).  There are also other compounds
and metabolites that are freed from isolated subcellular compartments
during extraction processes, and they can react with substances that
are normally not mixed.
<p>
The substances that are particularly affected by these oxidizing compounds
are biologically active proteins (the concern here is for enzymes in
particular).  If purification and characterization of biologically active
proteins is the goal of the process, then the elimination or at least
inhibition of oxidizing agents dissolved in solution is essential to
prevent any de-activating modification resulting in loss of function.
<p>
<b>Reduction of Cystine Disulfide Bonds</b>.  Reducing agents are used
to break disulfide bonds

<h3>2-Mercaptoethanol (&beta;-Mercaptoethanol)</h3>
<p>
&beta;-ME (FW 78.13, CAS#&nbsp;60-24-2, density: 1.11 g/ml)
is typically used as a reducing agent when large amounts
of buffer/reagent/solution are employed in a massive purification process
(and used in a pinch when no other reagent is available).  It less
effective than DTT or the triphosphines and thus used to maintain reducing
environments.  It should be used between 1-5 mM for this purpose.
It can also be used to reduce disulfide bonds too, but more time and
a higher concentration should be given to the process.

<h3>Dithiothreitol (DTT)</h3>
<p>
DTT (FW 154.25, CAS#&nbsp;3483-12-3) is used both for maintaining
reducing environments and for reducing disulfide bonds.  Used in
concentrations of 10 mM or greater it is effective at reducing disulfide
bonds.  Use at less than 0.5 mM if reducing environments are wanted without
disulfide bond reduction.
<p>
DTT has a slight acidity at a high pH (pH &gt; 8).  In IEF gels, it thus
ionizes and migrates toward the anode (toward lower pH end).  This can
result in loss of the thiol form of the proteins.

<h3>Trisubstituted phosphines</h3>
<p>
Triphosphines are very reactive reducing agents used by synthetic organic
chemists.  They recently have been been reported to be useful in reducing
cystines (disulfide-linked bonds).  The triphosphines should not be
used as reducing agents to scavenge oxidizing agents in buffers or reagents:
they are generally very unstable in aqueous solutions and rapidly degraded.
<p>
The reaction mechanism is illustrated below.

<h4>Tributylphosphine (TBP)</h4>
<p>
Herbert et al. (<i>Electrophoresis</i> <b>19</b>: 845-851) reported that
replacing DTT with tributylphosphine
[(CH<sub>3</sub>CH<sub>2</sub>CH<sub>2</sub>CH<sub>2</sub>)<sub>3</sub>P,
CAS #998-40-3] in the first dimension of 2D gel electrophoresis
resulted in better protein solubility, and thus shorter run times
and increased resolution.  Less horizontal streaking is seen in the
final gel.  This reducing agent is added to the sample
and also during the IPG strip equilibration procedure.
<p>
TBP is prepared in reagent form as a 100 &times; stock.  Dissolve
TBP in 1-methyl-2-pyrrolidinone (also called <i>N</i>-methylpyrrolidinone,
NMP) to 200 mM and seal under nitrogen gas.  Store at &minus;80&deg;.
The final concentration in use will be 2 mM.
<p>
In handling TBP, one should <em>always</em> wear gloves and lab coat.
It is also best to pipet material from reagent bottles to reaction tubes
within a fume hood, as it not only smells bad, but it can be toxic if
one is pipetting tens of samples over a long period.

<h4>Tris(2-carboxyethyl)phosphine (TCEP)</h4>
<p>
This reagent has the same reaction mechanism as TBP but is odorless.
This reagent (CAS #51805-45-9) has a MW of 250.15; the hydrochloride
salt has a MW 286.65.  Like TBP it is more
effective than DTT and &beta;-mercaptoethanol in reducing disulfides.
This material is stable at room temperature.
<p>
The reaction is as follows:
<p class="eqnserif">
P(CH<sub>2</sub>CH<sub>2</sub>COOH)<sub>3</sub> + R-S-S-R + H<sub>2</sub>O
&rarr; P=O(CH<sub>2</sub>CH<sub>2</sub>COOH)<sub>3</sub> + 2 R-S-H
<p>
A range of concentration from 5-50 mM is usually sufficient, and reaction
is achieved within a few minutes are room temperature.
Ideally TCEP should be used in a neutral pH since there is a minimization
of the possibility of amide bond cleavage during reduction.
<p>
<b>SDS-PAGE</b>.  In reducing proteins for SDS-PAGE, Pierce sells
TCEP dissolved in a neutral pH solution (Prod #77720, Bond-Breaker&reg;
500 mM TCEP solution, neutral pH, 5 ml).  Dissolve this is a 2&times;
Tris-glycine-SDS electrode buffer solution in a 1:10 dilution, then
mix a 1:1 volume with sample.  Heat at 95&deg; for 5 min, cool then load.
<p>
Alternatively the solid can be bought and effective reduction can be done
at room temperature at pH 5 in about 5 min.  This material is resistant
to oxidation in air, it is nonvolatile and does not react with other
functional groups.
<p>
<b>Use of Denaturants and EDTA</b>.  While a denaturant is not really required,
guanidine HCl can help in exposing internal disulfides.  Urea is not
recommended since cyanates formed within react with sulfhydryls.
EDTA probably should be included (Pierce recommends 5-20 mM) since this
removes divalent metals which promote re-oxidation and reduce TCEP activity.

<h5>Pierce Bond-Breaker<sup>&trade;</sup> TCEP Solution</h5>
<p>
This reagent (Cat #77720) is 0.5 M TCEP in a 5 ml neutral pH solution
stored at room temperature.  It can replace 2-mercaptoethanol or DTT
used in SDS-PAGE sample buffers.  Solution neutralization helps to avoid
the effects of protein by strong acid presence.
<p>
For SDS-PAGE sample buffer, the final effective working concentration
is 25 mM (50 mM, a 1:10 dilution, in 2&times; sample buffer).
A typical 2&times; sample buffer will contain 25 mM TrisHCl pH 6.8,
20% glycerol, 4% SDS, 0.005% bromophenol blue, and 50 mM TCEP.
Mix 1 vol sample with 1 vol buffer and heat 95&deg; for 5 min.

<h2 id="proteaseinhib">Protease Inhibition</h2>

<h3 id="serineinhib">Serine Protease Inhibitors</h3>
<p>
The serine proteases are a broad class of proteases that include
trypsin, chymotrypsin, thrombin.

<h4>PMSF</h4>
<p>
Phenylmethylsulfonylfluoride (PMSF) (MW 174.20, CAS#&nbsp;329-98-6)
has been used for many years to
inhibit serine proteases of all kinds usually present particularly in
mammalian tissue extracts.  It is usually used between 0.1 and 1 mM,
or 20-100 &micro;g/ml.  Stock solutions of 10 mg/ml are made in isopropanol.
It is effective against trypsin, chymotrypsin, kallikrein, subtilisin,
and thrombin.  It will also inhibit papain, a cysteine protease.
It is soluble only in methanol and ethanol.

<p>
<b>Stock Solution Preparation</b>.  PMSF is usually prepared fresh from
the solid form when making up extraction buffers.  It hydrolyzes somewhat
rapidly in aqueous solutions, even stored in the refrigerator.  However,
small stock solutions in ethanol can be made, if used within a few weeks.
PMSF is soluble in simple alcohols, and is usually prepared in ethanol.
To prepare a 10 ml solution of 100 mM PMSF:
<ol>
<li>Weigh out 0.174 g of PMSF into a glass screw cap container.
<li>Add 10 ml ethanol and dissolve.
<li>Store at &minus;20&deg;
</ol>

<h4>AEBSF</h4>
<p>
4-(2-aminoethyl)benzenesulfonyl fluoride (AEBSF) (MW 239.70,
CAS#&nbsp;30827-99-7) is a less toxic variant of the serine protease
class irreversible inhibitor, PMSF.  It is used at concentrations
&lt;1 mM, and is effective against trypsin, chymotrypsin,
kallikrein, plasmin, and thrombin.  It is soluble in water and ethanol.

<h4>Aprotinin</h4>
<p>
Aprotinin (CAS#&nbsp;9087-70-1) is a competitive reversible inhibitor of
serine proteases (chymotrypsin, trypsin, kallikrein and plasmin).  The
effective concentration of this water-soluble material is 0.1-0.3 &micro;M,
or 0.1-2.0 &micro;g/ml.  Stock solutions of 10 mg/ml are made in PBS.

<h4 id="leupeptin">Leupeptin</h4>
<p>
Leupeptin (MW 475.6, CAS#&nbsp;103476-89-7, Sigma #L-2884) is
<b>acetyl-Leu-Leu-Arg-al</b>, where the COOH terminus is
an aldehyde instead of a carboxylic acid.  It often is presented
as the hemisulftate salt (MW 493.61) and is soluble in water
at least up to 50 mg/ml (stocks of 10 mg/ml in water should be made),
and should be stored at &lt;0&deg;.
<p>
It is largely targeted to serine and cysteine proteases with positive charge
residues (basic) specificity.  Among the enzymes it inhibits, it includes:
<ul>
<li>trypsin
<li>papain
<li>plasmin
<li>thrombokinase
<li>kallikrein
<li>cathepsin B
</ul>
<p>
The half-maximal inhibitory concentration ranges from 0.5 to 75 &micro;g/ml,
depending on enzyme and substrate.  The working concentration is usually
10-100 &micro;M, or 0.5-2 &micro;g/ml.
<p>
Leupeptin does <em>not</em> inhibit:
<ul>
<li>chymotrypsin
<li>elastase
<li>renin
<li>pepsin
</ul>

<h4>Chymostatin</h4>
<p>
Chymostatin (MW 605.04, CAS#&nbsp;9076-44-2) is a reversible serine protease
inhibitor of &alpha;-, &beta;-, &gamma;-, and &delta;-chymotrypsin, but
will also inhibit cysteine proteases (papain, etc.).  It is soluble in
DMSO and acetic acid, and its effective range is 10-100 &micro;M.


<h4>Benzamidine</h4>
<p>
Benzamidine hydrochloride (MW 156.65, CAS#&nbsp;1670-14-0) is a water-
and ethanol- soluble agent used between 0.5-4 mM against
arginine-specific proteases (trypsin, kallikrein, thrombin).


<h4>Soybean Trypsin Inhibitor</h4>
<p>
Soybean trypsin inhibitor (STI, CAS#&nbsp;9035-81-8) is a water-soluble
substance which inhibits trypsin and trypsin-like serine proteases.

<h3 id="cysteineinhib">Cysteine Protease Inhibitors</h3>
<p>
See <a href="#leupeptin">Leupeptin</a> for an inhibitor that targets
Cys proteases with specificity for basic side chains.  Also many other
inhibitors of serine proteases also inhbit cysteine proteases (see the
serine protease section).

<h4>Antipain</h4>
<p>
Antipain dihydrochloride (MW 677.6, CAS#&nbsp;37691-11-5) is a reversible
inhibitor of cysteine and serine proteases (trypsin, papain,
and cathepsin A and B).  It is soluble in water and methanol and works
between 1-100 &micro;M.


<h3 id="amineinhib">Aminopeptidase Inhibitors</h3>
<p>
Aminopeptidases are <i>exoproteolytic</i> enzymes that hydrolyze
the peptide bond that joins the amino (N-)terminal amino acid
to the second amino acid in the chain.  This is the same type of
hydrolytic function as the exoproteases that are the carboxypeptidases,
except that those enzymes do it from the other (C-terminal) end.
<p>
Many aminopeptidases are
have broad specificity&mdash;that is, they can hydrolyze the peptide
bond of whatever amino acid is on the N-terminus, although there are
limitations as always.
<p>
Inhibitors of aminopeptidases tend to be <q>substrate-type</q>,
that is they mimic the substrate of the enzyme, and then reversibly
or irreversibly inactivate it.

<h4>Bestatin</h4>
<p>
Bestatin (MW 308.4, CAS 65391-42-6) is
<b>(&minus;)-<i>N</i>-3<i>R</i>-amino-2<i>S</i>-hydroxy-4--phenylbutyryl]-L-leucine</b>
HCl.  It was found in <i>Streptomyces olivoreticuli</i> to be a
competitive inhibitor of many aminopeptidases, including
<ul>
<li>aminopeptidase B
<li>leucine aminopeptidase
<li>tripeptide aminopeptidase
</ul>
<p>
The methyl ester form has a slightly stronger inhibition of neutral
aminopeptidases than bestatin, but a weaker activity against basic
aminopeptidases.  Bestatin is usually provided in methanolic solutions
at 1-5 mg/ml.  The recommended working concentration is about 40 &micro;g/ml
(~130 &micro;M).  Wheat germ extracts appear to contain 100-300 times
the levels of aminopeptidases relative to reticulocyte lysates.

<h4>Pepstatin A</h4>
<p>
Pepstatin A (MW 695.9, CAS#&nbsp;26305-03-3) is an aspartic acid protease
inhibitor effective against pepsin, renin, and cathepsin D; it also
inhibits HIV protease .  It is soluble in methanol or DMSO and is
used at 1 &micro;M or 1 &micro;g/ml.  Stock solutions of 1 mg/ml should
be prepared in MeOH.

<h4>E-64</h4>
<p>
E-64 is <i>trans</i>-epoxysuccinyl-L-leucylamido(4-guanidino)butane
(MW 357.4, CAS#&nbsp;66701-25-5) which irreversibly inhibits cysteine proteases
like papain, calpain, and cathepsin B, H, L, and S.  It is soluble in
DMSO and its working range is 1-10 &micro;M.

<h3 id="metalinhib">Metalloproteinase (Metal-Dependent Protease) Inhibitors</h3>
<p>
The metal-chelating inhibitors work because the action of the protease
requires usually a transition metal ion near or in its active site,
or perhaps essential to substrate binding.

<h4>EDTA</h4>
<p>
<i>N,N&prime;</i>-Ethylenediaminetetraacetic acid
(EDTA disodium dihydrate, FW 372.24, CAS#&nbsp;6381-92-6; EDTA tetrasodium,
FW 380.17, CAS#&nbsp;13235-36-4) is an excellent chelator of many
divalent metal ions in both the alkaline earth
(Mg<sup>2+</sup>, Ca<sup>2+</sup>) and transition metal (Cu<sup>2+</sup>,
Zn<sup>2+</sup>, Mn<sup>2+</sup>) classes.  It thus inhibits metal-dependent
enzymes of all types: proteases, protein kinases and phosphatases, and
a great many other enzymes including oxidoreductases, dehydrogenases,
non-protein hydrolases, etc.  The effective concentration is 0.5-2 mM.
<p>
<b>Stock Solution Preparation</b>.  To prepare 100 ml of a 0.5 M sodium
EDTA solution:
<ol>
<li>In a beaker between 100 and 125 ml size, weigh out 18.61 g disodium
EDTA (EDTA&bull;Na<sub>2</sub>) containing 75-80 ml water and a stir
bar, placed on a magnetic stirrer.
<li>Begin stirring (it will not likely dissolve), and add dropwise
6 M sodium hydroxide.  Wait for several seconds and even longer after
each drop (each drop is somewhere between 30 and 100 &micro;l).
<li>As more of the salt begins to dissolve, wait longer between
addition of drops.  Be patient!  Do not overadd the sodium hydroxide.
Although not essentially required, monitor the pH of the dissolution.
It should all be dissolved by the time pH 8.0 is reached, which is the
effective stopping point.  As you get closer to pH 8.0, and assuming the
volume is not close to 100 ml, a switch can be made to 0.5 or 1 M sodium
hydroxide, which gives finer control of the addition of strong base.
</ol>
<p>
This concentrated solution of EDTA is usually the most convenient for
preparing solutions of all kind containing EDTA.

<h4>EGTA</h4>
<p>
Ethylene glycol bis(2-aminoethyl
ether)-<i>N,N,N&prime;&prime;</i>tetraacetic acid (EGTA, MW 380.25,
CAS#&nbsp;67-42-5) is a more limited chelator, acting against calcium-activated
proteases.  It is soluble in alkaline
solution (slight amount of sodium or ammonium hydroxide) and its effective
range is 1-10 mM.

<h4>1,10-Phenanthroline</h4>
<p>
1,10-Phenanthroline (MW 198.2, CAS#&nbsp;5144-89-8) is a better chelator of
divalent transition metal ions such as Fe<sup>2+</sup>, Cu<sup>2+</sup>,
Zn<sup>2+</sup>, Co<sup>2+</sup>, which
are the metals for many classes of metalloproteases.  It is soluble in
alcohol.

<h4>Phosphoramidon</h4>
<p>
Phosphoramidon (MW 543.5, CAS#&nbsp;119942-99-3) is an inhibitor that targets
thermolysin and other metalloproteases;  it also inhibits collagenase.
It is soluble in DMSO or methanol, and is used between 5-500 &micro;M.

<h3 id="inhibcommercialprod">Commercial Products for Protease Inhibition</h3>
</div> <!-- class=contentblock -->

<div class="widetable">
<table id="protease-inhibitors" class="reducedTable td-center" style="margin-left:-10%;">
<tr>
<th rowspan="2">Protein Source
<th colspan="16">Components

<tr>
	<th>AEBSF
	<th>Aprotinin
	<th>Bestatin
	<th>Bromo- tetramisol
	<th>Canthardin
	<th>EDTA
	<th>E-64
	<th>Imidazole
	<th>Leupeptin
	<th>Microcystin LR
	<th>Pepstatin A
	<th>Phosphoramidon
	<th>1,10-Phenanthroline
	<th>Sodium Molybdate
	<th>Sodium Orthovanadate
	<th>Sodium Tartrate
<tr>
<td rowspan="2" class="category">Tissue Culture Media<br>
   (Sigma P1860)
<td> <td>X<td>X<td> <td> <td> <td>X<td> <td>X<td> <td>X<td> <td> <td> <td> <td>
<tr><td colspan="16"  class="reconstitution">Made as solution in DMSO<br> 1 ml 
of cocktail will inhibit dilution of &gt; 1:200 in tissue culture media to 
prevent proteolytic degradation of secreted proteins

<tr>
<td rowspan="2" class="category">Bacterial Cells<br>
 (Sigma P8465)
<td>X<td> <td>X<td> <td> <td>X<td>X<td> <td> <td> <td>X<td> <td> <td> <td> <td>
<tr><td colspan="16" class="reconstitution">Cocktail form:  Lyophilized powder + 
vial of DMSO<br> 1 ml of cocktail will inhibit 20 ml of cell lysate from 4 g 
(wet weight) of <i>E. coli</i> cells

<tr>
<td rowspan="2" class="category">Mammalian Tissue<br>
	(Sigma P8340)
<td>X<td>X<td>X<td> <td> <td> <td>X<td> <td>X<td> <td>X<td> <td> <td> <td> <td>
<tr><td colspan="16" class="reconstitution">Form of cocktail:  Solution in 
DMSO<br> 1 ml of cocktail will inhibit 100 ml of cell lysate from 20 g (wet 
weight) of bovine liver or 10 ml of cell lysate obtained from CHO cells at a 
density of 10<sup>7</sup> cells per ml

<tr>
<td rowspan="2" class="category">Fungal/Yeast Cells<br>
 (Sigma P8215)
<td>X<td> <td> <td> <td> <td> <td>X<td> <td> <td> <td>X<td> <td>X<td> <td> <td>
<tr><td colspan="16" class="reconstitution">Form of cocktail:  Solution in 
DMSO<br> 1 ml of cocktail will inhibit 100 ml of cell lysate from 20 g (wet 
weight) of <i>S. cerevisiae</i> cells

<tr>
<td rowspan="2" class="category">Plant Cells<br>
 (Sigma P9599)
<td>X<td> <td>X<td> <td> <td> <td>X<td> <td>X<td> <td>X<td> <td>X<td> <td> <td>
<tr><td colspan="16" class="reconstitution">Form of cocktail:  Solution in 
DMSO<br> 1 ml of cocktail will inhibit 100 ml of cell lysate from 30 g (wet 
weight) of plant tissue

<tr>
<td rowspan="2" class="category">His-Tagged Proteins<br>
 (Sigma P8849)
<td>X<td> <td>X<td> <td> <td> <td>X<td> <td> <td> <td>X<td>X<td> <td> <td> <td>
<tr><td colspan="16" class="reconstitution">Form of cocktail:  Solution in 
DMSO<br> 1 ml of cocktail will inhibit 100 ml of cell lysate from 20 g (wet 
weight) of <i>E. coli</i> cells or 10 g (wet weight) of baculovirus infected 
cells

<tr>
<td rowspan="2" class="category">General Use<br>
 (Sigma P2714)
<td>X<td>X<td>X<td> <td> <td>X<td>X<td> <td>X<td> <td> <td> <td> <td> <td> <td>
<tr><td colspan="16" class="reconstitution">Form of cocktail:  Lyophilized 
powder (water soluble)<br> 1 ml of cocktail will inhibit 1 mg of pancreatin

<tr>
<td rowspan="2" class="category">Tyr Protein Phosphatases &amp; Alkaline Phosphatases<br>
 (Sigma P5726)
<td> <td> <td> <td> <td> <td> <td> <td>X<td> <td> <td> <td> <td> <td>X<td>X<td>X
<tr><td colspan="16" class="reconstitution">Form of cocktail:  Solution in 
water<br> 1 ml of cocktail will inhibit 100,000 &times; <i>g</i> supernatant 
from human placenta, bovine liver, rabbit muscle, A431, or Jurkat cell extracts 
at a protein concentration of approx. 5 mg/ml

<tr>
<td rowspan="2" class="category">Ser/Thr Protein Phosphatases &amp; L-isozymes of Alk. Phosphatases<br>
 (Sigma P2850)
<td> <td> <td> <td>X<td>X<td> <td> <td> <td> <td>X<td> <td> <td> <td> <td> <td>
<tr><td colspan="16" class="reconstitution">Form of cocktail:  Solution in DMSO
</table>
</div> <!-- class=widetable -->

<div class="contentblock">
<b>Roche Diagnostics Complete</b>.  Produced as a tablet which provides
the <q>complete inhibition of proteases during extractions from
animal and plant tissues or cells, yeast and bacteria.</q>
This product (Cat #1 697 498 for 20 tablets, Cat #1 836 145 for
3 &times; 20 tab) inhibits a broad spectrum of serine, cysteine,
and metalloproteases as well as calpains.  Roche also sells more limited
protease inhibitors such as <b>Pefabloc</b>.
<p>
Tablets are prepared in 2 ml of distilled water or in 0.1 M phosphate pH 7.0,
and become a 25&times; stock.  Thus they will inhibit a total of 50
ml extract solution.  Stored at 2-8&deg;, stock solution is stable for
1-2 weeks, and at least 12 weeks when stored &minus;15 to &minus;25&deg;.
<p>
Animal tissues typically contain serine, cysteine, and metalloproteases.
Serine and cysteine proteases dominate in plants.  Bacterial generally
have serine and metalloproteases.  Aspartate (<q>acid</q>)
proteases occasionally occur in some animal tissues but their activities
are generally observed at low pH;  pepstatin should be added as a precaution
if extractions are done unbuffered or at a low pH.
<p>
EDTA is present at 1 mM.  Extracts containing divalent metals may overwhelm
the presence of EDTA and activate the metalloproteases.  Avoid using
them in extractions.  If proteins are recombinants (His-tagged) to be
purified by metal-chelate affinity chromatography, it is necessary to
remove EDTA (such as by dialysis) prior to chromatography.

<h2>Inhibition of Protein/Peptide-Modifying Processes</h2>
<p>
In the extraction of many cells and tissues of different types, particular
chemicals and enzymes can modify proteins and peptides, and these modifications
must be inhibited or prevented.  This includes protein kinases and
phosphatases, which respectively add or remove phosphate moieties to the
side chains of many residues, and other sorts of enzymes (acetylation
enzymes, glycosyltransferases, etc.).

<h3>Protein Kinases</h3>
<p>
Protein kinases depend on ATP bound to divalent metal ion cofactors
(Mg<sup>2+</sup>, Mn<sup>2+</sup>).  It is their dependence on the divalent
metal ion cofactors that is exploited.  One simply uses a metal chelator
(EDTA, EGTA) to tie up the cofactor and thus slow or stop the kinases.
EDTA concentration can range from 0.1 mM to 10 mM, although 0.5 mM seems
usual.  EGTA is more specific for Ca<sup>2+</sup> than Mg<sup>2+</sup>.

<h3>Protein Phosphatases</h3>
<p>
Protein phosphatases are present in many eukaryotic tissues, where
phosphorylated proteins have their state changed by dephosphorylation
of the protein rather than by degradation of the protein by a protease
(another way of removing phosphorylated proteins).  Some of these
enzymes are metal-dependent (Mg<sup>2+</sup> and Ca<sup>2+</sup>)
and will be inhibited by chelators.  Protein phosphatases in eukaryotes
have many forms: protein phosphatases 1, 2A, 2B, and 2C are well characterized
(see P. Cohen, <i>Meth. Enzymol.</i> <b>201</b>: 389, 1991).
<p>
<b>Okadaic acid</b>.  Produced by halichondric marine sponges (porifera), this
polyether fatty acid is a cause of diarrhetic poisoning and is also
a tumor promoter.  It is very potent at inhibting protein phosphatases 1 and
2A (PP1 and PP2A), weak at inhibiting PP2B, and has no effect on PP2C.
The IC<sub>50</sub> for PP1 and PP2A are 5-10 nM and ~50 pM, respectively.
<p>
<b>Microcystin-LR</b>.
<p>
<b>Trifluoperazine</b>.  Inhibits PP2B.
<p>
Tyrosine protein phosphatases are sensitive to the presence of
sodium orthovanadate, which probably mimics the phosphate group and
competitively inhibits the phosphatase.  <b>Na<sub>3</sub>VO<sub>4</sub></b>
present up to 1 mM is usually sufficient to inhibit Tyr protein
phosphatases.


<h2 id="detergents">Detergents</h2>
<p>
Detergents are surface-active agents that can mean the ability to
solubilize membrane proteins or leaving them insoluble and impervious
to analysis.


<h3 id="zwitterionics">Zwitterionic Detergents</h3>
<p>
CHAPS is 3-[(3-cholamidopropyl)dimethyl-ammonio]-1-propanesulfate (CAS
#75621-03-3, MW 614.89).  It is used as a zwitterionic solubilizer
of proteins for isoelectric focusing (IEF), used anywhere between 1-4%
concentration.  (Sigma sells this at about &euro;50-60 for 1 gram in 2005.)
<p style="text-align:center;">
 <img src="imagesProteomicsReferenceHTML/CHAPS.gif" alt="CHAPS structure">
<p>
CHAPS promotes solubilization and stabilization by shielding hydrophobic
zones from nonspecific aggregration and by stabilizing disordered loops.
<p>
CHAPSO is 3-[(3-cholamidopropyl)dimethyl-ammonio]-2-hydroxy-1-propanesulfate
(CAS#&nbsp;82473-24-3, MW 630.90), and is the 2-hydroxy form of CHAPS.
(Sigma sells anywhere from &euro;70-160 for 1 gram;  the cost difference
is usually due to the extent of analysis for purity, such as presence of
heavy metals, etc.)
<p>
C7BzO is 3-(4-Heptyl)phenyl-3-hydroxypropyl)dimethylammoniopropanesulfonate
(no CAS#&nbsp; yet, MW 399.59) is among a new class of proteins (such as ASB-14)
that is a broadly useful solubilizing detergent that works with bacteria,
mammalian, and plant cells (see structure).
<p style="text-align:center;">
 <img src="imagesProteomicsReferenceHTML/C7BzO.gif" alt="C7BzO structure">
<p>
Sigma-Aldrich claims this detergent is more effective than CHAPS at
solubilizing a total extract at 1% (w/v) concentration.  (Sigma sells
1 gram of this detergent at &euro;100 in 2005.)

<h4>The Sulfobetaines</h4>
<p>
The sulfobetaines listed below are all effective at protein
solubilization, particularly of membrane proteins.
<p>
ASB-14 is 3-[<i>N,N</i>-dimethyl(3-myristoylaminopropyl)ammonio]propanesulfonate
with a formula
CH<sub>3</sub>(CH<sub>2</sub>)<sub>12</sub>C(=O)NH<sub>2</sub>(CH<sub>2</sub>)<sub>3</sub>N<sup>+</sup>(CH<sub>3</sub>)<sub>2</sub>CH<sub>2</sub>CH<sub>2</sub>CH<sub>2</sub>SO<sub>3</sub><sup>&minus;</sup>;
it also goes by the name Amidosulfobetaine-14 (CAS #216667-08-2,
MW 434.68).
<p>
SB3-8 is 3-(octyldimethylammonio)propanesulfate inner salt with a formula of
CH<sub>3</sub>(CH<sub>2</sub>)<sub>7</sub>N<sup>+</sup>(CH<sub>3</sub>)<sub>2</sub>CH<sub>2</sub>CH<sub>2</sub>CH<sub>2</sub>SO<sub>3</sub><sup>&minus;</sup>;
it also known as Zwittergent 3-8, octyl sulfobetaine, and
dimethyloctyl(3-sulfopropyl)ammonium hydroxide inner salt
(CAS #15178-76-4, MW 279.44).
<p>
SB 3-10 is 3-(decyldimethylammonio)propanesulfate inner salt with a formula of
CH<sub>3</sub>(CH<sub>2</sub>)<sub>9</sub>N<sup>+</sup>(CH<sub>3</sub>)<sub>2</sub>CH<sub>2</sub>CH<sub>2</sub>CH<sub>2</sub>SO<sub>3</sub><sup>&minus;</sup>;
it also known as Zwittergent 3-10 and caprylyl sulfobetaine and
sulfobetaine 10 (CAS #15163-36-7, MW 307.49).
<p>
SB 3-12 is 3-(dodecyldimethylammonio)propanesulfate inner salt with a formula of
CH<sub>3</sub>(CH<sub>2</sub>)<sub>11</sub>N<sup>+</sup>(CH<sub>3</sub>)<sub>2</sub>CH<sub>2</sub>CH<sub>2</sub>CH<sub>2</sub>SO<sub>3</sub><sup>&minus;</sup>;
it also known as Zwittergent 3-12 and lauryl sulfobetaine and
sulfobetaine 12 (CAS #14933-08-5, MW 335.55).

<h3 id="anionics">Anionic Detergents</h3>
<p>
Sodium dodecyl sulfate (SDS, also called sodium lauryl sulfate, MW 288.38,
CAS#&nbsp;151-21-3) is about the best known of this class of detergents.
It is a strong denaturant of protein, and binds to protein at a constant
1.4 grams per gram protein.  This imparts a constant negative charge-to-mass
ratio, and because the charge-to-mass ratio is constant, this means
that electrophoretic mobility toward the anode is based on relative
mass differences alone, and so thus mobility is a measure of differences
in mass.  This is the basis of SDS-PAGE.

<h3 id="cationics">Cationic Detergents</h3>
<p>
Cetyltrimethylammonium bromide (CTAB, MW 364.09, CAS#&nbsp;57-09-0) is
about the best known in this class.  Like all ionic detergents with
along nonpolar chain, it is efficient at denaturating proteins.

Cetyldimethylethylammonium bromide (CE2MAB, MW 378.5, CAS#&nbsp;124-03-8)
differs from CTAB by only an ethyl instead of a methyl group on the
ammonium group.

<h3 id="nonionics">Nonionic Detergents</h3>
<p>
Brij<sup>&reg;</sup>-35 is polyoxyethylene lauryl ether, with a structure
(CH<sub>2</sub>CH<sub>2</sub>O)<sub><i>n</i></sub>C<sub>12</sub>H<sub>26</sub>O
(CAS#&nbsp;9002-92-0).
<p>
Brij 30 and Brij 96 are good solubilizers when no denaturing conditions are
present (urea and thiourea are not used).
<p>
Octylphenoxypolyethoxyethanol goes by the name Nonidet P-40 (NP-40)
and is an effective nonionic detergent.
<p>
<i>t</i>-Octylphenoxypolyethoxyethanol is a trimeric form of NP-40
that is called Triton X-100 (MW avg. 647, CAS#&nbsp;9002-93-1).
<p>
Triton X-114 is (CH<sub>2</sub>CH<sub>2</sub>O)<sub>7-8</sub>C<sub>14</sub>H<sub>22</sub>O
<p>
Octyl-&beta;-D-glucose (MW 292.38, CAS#&nbsp;29836-26-8) is excellent at
solubilize membrane proteins without denaturation.  It is the detergent
used in the Beckman PF-2D chromatofocusing buffer.
<p>
Octyl-&beta;-D-thioglucose (MW 308.4, CAS#&nbsp;85618-21-9).
<p>
Tween<sup>&reg;</sup> 20 is polyoxyethylene-20-sorbitan monolaurate (CAS#&nbsp;9005-64-5)
is often used in immunochemistry and in situ hybridization.
<p>
Tween<sup>&reg;</sup> 80 is polyoxyethylene-20-sorbitan monooleate (CAS#&nbsp;9005-65-6).

<h3>The Pierce Protein Extraction Reagents</h3>
<p>
Pierce Chemical developed a <a 
href="http://www.piercenet.com/products/browse.cfm?fldID=060104">set of 
reagents</a> for what it calls is the mild extraction of all kinds of the cell 
types below. Generally the detergent extracts the proteins without complete 
solubilization of membranes that complicate the extraction (such as extracting 
the DNA).  Although the formulation is proprietary, the components were revealed 
in an <a 
href="http://www.ncbi.nlm.nih.gov/pubmed?term=Yang-Boja%5BAll%20Fields%5D%20AND%20DeFilippes%5BAll%20Fields%5D&amp;cmd=DetailsSearch">ESI-MS
study</a>

<ul>
<li><b>B-PER</b>.  The Bacterial Protein Extraction Reagent contains 
<i>n</i>-octyl-<i>&beta;</i>-D-thioglucopyranoside as the major detergent, with
a 20 mM TrisHCl pH 7.5 buffer.
<li><b>Y-PER</b>.  The Yeast Protein Extraction Reagent contains as the major
detergent <i>N</i>-tetradecyl-<i>N,N</i>-dimethyl-3-ammonio-1-propanesulfonate
in a 0.1 M phosphate pH 7.4 buffer.
<li><b>M-PER</b>.  The Mammalian Protein Extraction Reagent contains the zwitterionic
detergent CHAPS in a 25 mM bicine pH 7.6 buffer.
</ul>

<h2>Organic Solvents</h2>
<h3>Methanol</h3>
<p>
Typically used in gel staining, but also in HPLC of peptides and proteins
as well.  It has many uses.  The lowest grade generally used is ACS Reagent
grade, but when used in HPLC, a chromatography grade (higher) should be
used.
<p>
MeOH might be used in combination with chloroform (CHCl<sub>3</sub>) to
dissolve membranes in tissue extractions prior to the use of zwitterionic
or nonionic detergents in membrane protein extraction.

<h3>Chloroform</h3>
<p>
Might be used in combination with MeOH to extract membrane proteins by
first dissolving membranes.


<!--  Cell and Tissue Extraction/Homogenization -->

<h1 id="tissextraction">Cell &amp; Tissue Extraction/Homogenization</h1>
<p>
The thorough extraction of cells and tissues usually involves a combination
of <em>chemically</em> and <em>mechanically</em> techniques.

<h2>Mechanical Extraction Modes</h2>

<h3>Potter-Elvehjem</h3>
<p>
This is a mortar-pestle homogenizer with the mortar being like a
round-bottom glass test tube and the upper expanded mouth.
Homogenization is achieved both by grinding on the rounded tip of the
pestle and the pressing of the tissue through the cylindrical body of the
pestle.  Ground glass tips can also be found and are effective on
connective tissue.  Teflon (PTFE) pestles are used for softer tissues.

<h3>DUALL</h3>
<p>
Provides a conical shape on the lower part of the pestle to provide a
greater grinding surface area compared to the Potter-Elvehjem.  The
ground glass pestle is better for muscle (including heart) and lung
and for tissues which are typically connective.  The PTFE pestles are
most suited to brain and liver.

<h3>Dounce Homogenizer</h3>
<p>
A Dounce homogenizer is a specially constructed variation the mortar
and pestle used to extract suspended cells not associated with tissues,
although it can break up loosely held together tissues too.  The Dounce
homogenizer is ideal for making preparations of intact nuclei from
extracted cells.
<p>
It is typically made of glass such that the extraction is visible to the
user.
<p>
Pestles come in two types: <q>A</q> and <q>B</q>.
<p>
The A-type pestle is larger and might be used for the initial extraction.
A switch to the B-type pestle has a smaller clearance and provides
final homogenization.  The A-type pestle usually has clearance of
70-150 &micro;m in range, which disrupts cultured animal cells.
The B-type pestle has a clearance ranging
from 15-75 &micro;m, which disrupts cells and nuclei.

<p>
To make preparations of intact nuclei, the extraction buffer usually
contains sucrose (0.32 M) to maintain the integrity of the nucleus during
extraction.  Use the A-type pestle and only for at most 10 strokes.
The extraction is then spun down to get a pellet rich in nuclei, which
then are later extracted.

<h3 id="ultrasonic-homogen">Ultrasonic Homogenization</h3>
<p>
Ultrasonic homogenization uses high frequency acoustic energy to shatter cell
membranes of all kinds.  It is best used for individuals cells suspended in 
solution rather than tissues, although it can be used to homogenized minced
or diced solid tissues.  It should be used to homogenize fibrous tissues of
any kind.

<h4 id="vibracell">Sonics Vibra Cell VC130</h4>
<p>
The Sonics Vibra Cell ultrasonic processor features a 130 watt probe.

<h5>Proper Use</h5>
<p>
Take note of these precautions before using the equipment:
<ul>
<li>When cl&#097;mping/mounting probe, NEVER cl&#097;mp the probe itself, but
the upper portion of the converter housing.
<li>The system should be properly electrically grounded.
<li>Never operate the power supply unless it is connected to the converter.
<li>Never touch a vibrating probe.
<li>Never allow a microtip or extender to vibrate in the air for more than
10 seconds.
<li>Never operate a probe with threaded end without a tip, extender, or
microtip.
<li>The converter should be air-cooled when sample temperature exceeds
100&deg; and when working at high intensity for more than 30 min.
<li>It is recommended that a sound-dampening enclosure or ear protection
be used when operating the processor.
<li>When sonicating in low surface tension liquids (total or high proportion
organic solvents), use a <b>solid</b> probe rather than a probe with a
replaceable tip.  Particulates can work their way into the threads
between the tip and the solid portion of the probe, causing loss of
resonance and power supply failure.
</ul>

<h5>Operation</h5>
<p>
The power supply produces a high frequency signal of electrical energy
transmitted to a piezoelectric transducer within the converter.
The probe itself is constructed to intensify or amplify the vibrations
creating pressure waves in a liquid.  Cavitation (bubbling) of the liquid
occurs creating shock waves with elevated pressures and temperatures.
Larger probe tips allow larger volume processing but lower intensities.
Selection of the proper probe tip for the volume to be processed is important.

<table>
<tr><th>Probe tip diameter  <th>Volume scale (ml)
<tr><td>2 mm microtip       <td>0.15 - 5
<tr><td>3 mm microtip       <td>0.25 - 10
<tr><td>6 mm microtip       <td>10-25
<tr><td>13 mm probe         <td>50-150
</table>
<p>
The 3 and 6 mm microtips are standard with some models.  The 13 mm probe
is available with and without replaceable tip.

<p>
The ultrasonic processor has the following controls:
<ul>
<li><b>Amplitude</b>.  This switch move up and down to toggle power on
and off.  It also rotates to set vibrational amplitude (frequency of
vibrations).
<li><b>Pulsing</b> (only with units NOT having thumb-activated pulser).
This is located on the converter and, when pressed,
activates the sonication for purposes of pulsing without timing.
<li><b>Pulse/Continuous</b> (only with units having thumb-activated pulser).
This switch puts the processor into either
a pulsing or continuous mode.
<li><b>Timer</b> (only with units NOT having thumb-activated pulser).
Duration can be set from 1-10 min.  Press it to activate
the ultrasonic in continuous mode for the time set.
<li><b>Pulse</b> (only with units NOT having thumb-activated pulser).
This activates the processer in pulsed mode.  The
duration can be from 1 second off/1 second on, to 1 second off, 10 seconds on.
The off cycle is always 1 second, whereas the on cycle varies from 1-10 sec.
When in the off position, ultrasonication is continuous.
<li><b>Wattmeter</b>.  The power setting to the probe.
</ul>
<p>
If the probe has a pulsing button, it will be connected to the 4-pin
connector on the rear of the unit.  If it does not have one, it will
connect to the BNC connector.  The processor also has a connector for
an optional footswitch cable.

<ol>
<li>Set the AMPLITUDE in OFF position.
<li>Plug in the power line.
<li>Mount probe assembly into lab stand when not in use.
<li>Immerse probe into sample.  Immerse it deeply:  foaming might occur
and inhibit cavitation.
<li>To use pulse operation, set PULSE/CONTINUOUS
switch to Pulse;  for continuous, set switch to Continuous.
<li>Set Amplitude to 50% at first (30% with a microtip).
<li>Set PULSER mode for manual pulsing;  set TIMER for timed pulsing, and
depress TIMER knob.
<li>Adjust AMPLITUDE knob during operation as required to achieve disruption.
<li>It is better to use a lower power setting to prevent foaming than
a higher one that causes it.  Increase the time of processing if lowering
the power setting.
</ol>
<p>
Gram-negative bacteria require 10-15 min of processing,  and gram-positives
like staphylococcus need 20-30 min.  Bacilli (rod-shaped bacteria)
disrupt more easily while cocci (spherical bacteria) are more resistant.
The Mycobacteria (tubercle bacilli) are very resistant to sonication.
Animal cells disrupt more easily than plant cells, and red blood cells
are disrupted more easily than muscle cells.
<p>
Addition of glass beads in the 0.05-0.5 mm size can promote cell disruption,
and is essentially required when trying to extract spores and yeast.
One volume beads to two volumes liquid is useful.
<p>
Freezing at 70&deg; and rapid thawing in water just prior to disruption
is often quite useful.  Dicing tissues into small pieces also aids
disruption.  Generally, work with the smallest volume possible to confine
sample movement, especially of solid tissues.  Glass and stainless steel
containers do better at providing disruption while plastic somewhat dampens
the effects.

<h2 id="bactxtract">Bacterial/Prokaryotic Extraction</h2>
<p>
For mechanical extraction, a sonicator or French press can be used to break open 
cells.  Sonicate on ice at a strong power setting, and long enough to break down 
DNA (if trying to isolate DNA, especially genomic DNA, do not use a sonicator by 
trying the alkaline-SDS lysis method).

<h3>Cytoplasmic Proteins</h3>
<p>
Once bacteria are extracted, cytoplasmic soluble proteins can be obtained after 
a spinning (at 4&deg;) in a microcentrifuge (14,000 <i>g</i>) for at least 30-45 
minutes.  Extraction of the membrane pellet continues with an alkaline solution 
(pH 10.4) of 7 M urea, 2 M thiourea, and a detergent such as 1% C7BzO; after 
centrifugation, the supernatant is combined with the initial supernatant.

<h3>Membrane Proteins</h3>
<p>
Gram-negative bacteria (like <i>E. coli</i>) have membranes composed of lipid 
along with complex protein-sugar molecules (peptidoglycan).  Lysozyme 
specifically attacks peptidoglycans in the bacterial cell wall, and can be used 
to release proteins trapped in it.
<p>
Gram-positive bacteria (the staphylococci and streptococci being good examples)
have a thick peptidoglycan wall lacking lipopolysaccharide structure found
in gram-negative bacteria.  Lysozyme should be very effective.
<p>
Use of lipid solvents might help breakdown the lipid portion of membranes.


<h3>Recombinant Proteins</h3>
<p>
[Notes taken from A McGettrick &amp; DM Worrall <q>Extraction of
Recombinant Protein From Bacteria</q> in <a href="#cutler">Cutler</a>]
<p>
When recombinant proteins are expressed at high levels in typical bacterial 
hosts, a special purification procedure is called for. First determine the rare 
case that the bacteria are actually secreting the protein of interest, which 
will make purification immensely easier. The recombinant might make up 40% of 
total cell protein in ideal cases.
<p>
If inactive protein is being made, inspect the bacteria under a microscope for 
<b>inclusion bodies</b> which are phase bright.  Proteins with strong 
hydrophobic content or regions rich in ionic amino acids are likely to form 
inclusions, which are aggregates of unfolded polypeptide.  Prevention of 
inclusion body and promotion of folding can be effected by reducing growth 
temperature slightly, or by reducing the inducing agent concentration, or by 
increasing aeration.  Construction of the protein to a fusion which promotes 
protein solubility and folding should be considered; fusion proteins like GST 
(glutathione S-transferase), thioredoxin, and maltose-binding protein are 
useful, especially MBP.  Promotion of folding is helped by using co-recombinants 
of GroEL/ES and DnaK-DnaJ-GrpE, or protein disulfide isomerase folding catalyst. 
Some protein folding chaperones are helped by a heat shock induction step.  In 
rare cases, inclusion bodies are actually desirable because they can reduce the 
effects of a toxic characteristic of a recombinant harmful to the host. 
Solubilization of inclusion bodies typically requires denaturants (urea or 
guanidine HCl), or extremes of pH and temperature in some cases.
<p>
If the extraction is on a small scale, use of lysozyme to break down the
cell wall and disruption of the inner cell membrane with detergents,
osmotic lysis, or mechanical extraction should be considered.

<ol>
<li>Preweigh the centrifuge tubes, and then spin out the cells at
1000 <i>g</i> for 15 min at 4&deg;, then weigh the wet pellet.
<li>Add 3 ml of lysis buffer per gram wet cell pellet and resuspend
the cells.  A buffer of 50 mM Tris HCl pH 8, 1 mM EDTA, 50 mM NaCl,
1 mM PMSF can be used in most cases, but it will depend on the particular
situation.  Add lysozyme from a 10 mg/ml chicken lysozyme (Sigma) stock
to 0.3 mg/ml.  Keep stirring the suspension for 30 min at 4&deg;.
<li>Add Triton X-100 to 1% (v/v).  Ultrasonicate in three 30-s bursts with
intermediate cooling.
<li>Leave at room temperature.  Add DNase I from a 1 mg/ml stock to
10 &micro;g/ml and also add MgCl<sub>2</sub> to 10 mM.  Stir for 15 min to
ensure viscosity is reduced.
<li>Spin 10,000 <i>g</i> for 15 min at 4&deg;.  Resuspend pellet in lysis
buffer in a volume equal to the supernatant.  At this point it might
be useful to analyze the supernatant and resuspended pellet using SDS-PAGE.
If most of the protein is in the pellet, expect the recombinant to be
formed as inclusion bodies.
</ol>
<p>
If it discovered that the recombinant forms inclusion bodies which
come out in the pellet, it is wise to wash the pellet prior to solubilizing
the inclusion bodies.  The wash procedure is first determined on a small
scale by resuspending 200 &micro;l aliquots of resuspended cell pellet
in small conicals and spinning 12,000 <i>g</i> for 10 min 4&deg;;  the
resuspension buffer is actually a series of different test solutions:
lysis buffer with 1, 2, 3, and 4 M urea and 0.5% Triton X-100 is suggested.
Mix 10 min at room temperature, spin, and then resuspend in 200 &micro;l
water.  Equal volumes of supernatant and resuspended pellet are added
to SDS-PAGE buffer.  The best washing buffer contains the most proteins
that does <em>not</em> include the protein of interest.  The procedure
is scaled up and inclusion bodies washed twice with the chosen wash buffer.
<p>
The protein is then solubilized in something like 50 mM Tris HCl pH 8, with
8 M urea and 1 M DTT.  Stir 60 min at room temperature.  Spin at
100,000 <i>g</i> for 10 min at 4&deg; to remove insoluble material.
Verify the insoluble material is not the protein.  If so, increase incubation
time or try different reagents to solubilize.  To refold the protein
one method is to add the denatured protein dropwise to a stirring
solution of renaturation buffer (50 mM Tris HCl pH 8, 1 mM EDTA,
100 mM NaCl, 0.5 mM oxidized glutathione, 5 mM reduced glutathione,
0.2 M arginine), with denaturant diluted 25- to 50-fold and final protein
concentration not exceeding 50 &micro;g/ml.  Stir for 2 h at 4&deg;
and dialyze.  Remove aggregates with 0.2 &micro;m filter.
<h2 id="yeastxtract">Yeast Extraction</h2>
<p>
Yeast and other unicellular (<q>free-living</q>) fungi have rather
tough cell walls and require rigorous mechanical force to break
them open.  Those who work with yeast and unicellular fungi typically
use a mill containing glass beads of appropriate size.

<h3>Monitoring Extraction with Microscope</h3>
<p>
As with bacterial extraction, the best way to understand the effectiveness
of the extraction technique (the combination of chemical solubilizers
and mechanical force) is to have a microscope available.  It does not
need to be powerful or expensive.  A 40-100 &times; magnification of
the microscope is sufficient;  this is a scope with 10 &times; oculars and
a 4&times; or 10&times; objective.  If the objective is not phase-contrast,
then it might help to add something to darken the background, such as a dye
like India ink or bromophenol blue, but not something that would
penetrate or damage cells (a vital dye like trypan blue would work).
Also use an appropriate dilution when setting up field:  100-200 cells
in the field is appropriate as a sampling of cells.  Before starting the
extraction, prepare on a slide or hemocytometer the intact cells, since
it will be important to use these as a reference;  otherwise there might
be confusion as to whether non-cell debris is really the remnants of
extracted cells.

<h3>Mini-Bead Beater</h3>
<p>
The Mini-Bead Beater-1 (BioSpec Products, Inc., Cat #3110BXEUR for the
230 V unit) is used for small or micro scale extractions of yeast/fungi
and for tough fibrous tissues.  Extractions volume can be from
100 &micro;l to 1 ml.  Differnet models of the instrument can also
extract 8 and 96 samples at a time.


<h4>Beads</h4>
<p>
Beads come in glass or the more dense zirconia-silica and even more
dense chrome-steel.
<p>
The following are size recommendations when choosing beads:
<ul>
<li><b>Bacteria</b>.  0.1 mm diameter
<li><b>Yeast/Fungi</b>.  0.5 mm dia.
<li><b>Most Tissues</b>. 1.0 mm dia.
<li><b>Skin or Soft Plant Material</b>.  Use 2.5 mm zirconia/silica beads.
<li><b>Spores</b>.  Replace the glass with the same size in zirconia-silica beads.
<li><b>Extremely Tough and/or Fibrous Tissue</b>.  Try chrome-steel beads
such 2.5 and 3.2 mm size.  Liquid nitrogen is used to freeze leaves directly
in vial, then immediately pulverized.
</ul>

<h5>Bead Material</h5>
<table>
<tr><th>Material  <th>Density (g/cm<sup>3</sup>) <th>Use/Application
<tr><td>Glass     <td> 2.5                       <td>general, soft tissues
<tr><td>Zirconia/Silica <td>3.7                  <td>spores, most tissues
<tr><td>Silicon Carbide <td>3.2                  <td>sharp particle as opposed
   to bead (more rapid extraction)
<tr><td>Zirconia    <td>5.5                     <td>tough tissue extraction
<tr><td>Chrome-Steel, stainless or not <td>7.9  <td>grinding leaves
<tr><td>Tungsten Carbide <td>14.9   <td>no recommended for biological samples
<tr><td colspan="3" style="font-size:75%;">info adapted from BioSpec
</table>


<h5>Bead Re-Use/Recycling</h5>
<p>
Laboratory detergent solutions are good enough for washing glass beads.
<p>
Do not acid-wash beads.
<ol>
<li>Soak the dirty/used beads overnight in a laboratory detergent (1%
Alconox?).
<li>Use several rinses with tap water.
<li>Then rinse a few times with RO or distilled water.
<li>Put beads in stainless steel or glass tray and put in 40-70&deg; oven.
</ol>
<p>
The dry beads should roll freely.  If they are caked together, they are
either not cleaned or rinsed well enough.  Repeat the procedure.
<p>
Beads used for nucleic acid isolation can be soaked in 10% bleach for 5
minutes.
<p>
Beads generally can be re-used up to 10 times before they get too small
to use.
<p>
Sterilize beads, if needed, by autoclaving <em>clean</em> beads.

<h4>Containers/Vials</h4>
<p>
The Mini-Bead Beater-1 uses 2.0 ml O-ringed screw-cap microcentrifuge
polypropylene vial.  The vial can be preloaded with beads, and is available
sterile or nonsterile.

<ul>
<li>Cat #<b>10831</b>:  sterile 2 ml vial &amp; cap, 100 ct in foam rack
<li>Cat #<b>10832</b>:  2 ml vial &amp; cap, 500 ct bulk
<li>Cat #<b>522S</b>: sterile 2 ml vial &amp; cap, 500 ct bulk
</ul>

<h5>Loading Beads Into Vials</h5>
<p>
Loading beads is difficult and tedious.  A special loader can be constructed
that works well:
<ol>
<li>Obtain a 1 ml plastic syringe and remove and discard the plunger.
<li>Cut the barrel with scalpel so that the length is at the volume mark
which is to contain the beads.
<li>Find small stainless steel mesh and push the mesh into the barrel
using a nail, drill bit, or thick wire.
<li>Attach tubing to the needle port of the syringe, and connect the other
end to vacuum line.
<li>Dip the modified syringe barrel into the container of beads.
<li>Transfer to the vial/microtube, then break the vacuum to release the
beads.  An alternative is to cut a hole in the vacuum line tubing, cover it
with the finger to create the vacuum, then open to release the vacuum.
</ol>

<h4>Preparation &amp; Operation</h4>
<ol>
<li>The 2 ml screw cap vial should be filled between half and three-quarters
full with beads.
<li>Fill the balance of the tube with buffer and/or extractable material.
It is important to fill to the top as much as possible to eliminate air.
Maximum tissue/cells is 400 mg per milliter buffer.
<li>Screw on the cap.  Invert the vial several times, both to wet the beads
and get trapped air out.
<li>Let the beads and material settle, remove the screw cap, and add more
buffer or material to fill close to top, if needed.
<li>Put vial in MiniBead Beater-1 arm assembly.  Close black safety cover.
<li>Operate for a maximum of three minutes (180 seconds, 18 on the display),
and at a suitable speed setting (4800 rpm, 48 on display).  Consider
cooling the material.
<li>Recover most of the extract easily by letting the beads settle and
pipetting out, or pulse spinning the beads down at a low speed in
a microcentrifuge.
<li>To recover all the extract, recover the supernatant as in the previous
step, then add buffer to about 25-50% of the volume of the beads, to wash
their surface.  Spin down or let settle, and recover the buffer wash on top.
</ol>
<p>
The MiniBead Beater disrupts about 90% of cells in 2-5 min.

<h5>Cooling</h5>
<p>
The beating/milling process generates much heat.
Generally it is best to start with material pre-chilled/cooled.
A 1-minute-ON-1-minute-OFF procedure will work well, with placement of
vials in ice water bath in the OFF cycle.

<h2 id="mammxtract">Mammalian Tissue Extraction</h2>
<p>
[Some notes in this section are taken from JM Skehel <q>Preparation
of Extracts from Animal Tissues</q> in <a href="#cutler">Cutler</a>]
<p>
Tissues are basically structures holding cells together by a combination
of easily disrupted structures.  The specific tissue should be dissected
as carefully as possible to remove other tissues whose extraction contributes
to a wrong idea of the proteome.  Indeed it is better to cut out and sacrifice
more of the desired tissue in order to remove contaminating tissue.
Heart and liver tissues in the tens to
hundreds of grams can be put into a blender which is then pulsed several
times to puree the tissue.  When using the blender it is important to
avoid a sustained vortex which can dissolve air into the extraction,
and possibly oxidize proteins.  Bone is a particular case of tissue
which cannot be blended by usual means.  Bone might be demineralized
using chemicals that dissolve or absorb the mineral element.
<p>
Always think several steps in advance.  In particular colder temperatures
are used in these tissue extractions.  Pre-chill centrifuges and rotors
as well as blender containers (stainless steel containers are preferred
to glass for this reason).
<p>
Buffer is added typically in volumes of 2-2.5 ml per gram tissue.
The container should be approximately the same volume as the tissue
and buffer to minimize mixing air into the extract.  In using any sort
of blending/mincing system (Polytron, blender), keep extractions limited
to 15-30 s with a couple of minutes in between to minimize heating
and foaming.
<p>
<b>Centrifugation</b>.  If large-scale homogenization is being done,
try to spin 600-3000 <i>g</i> for 30 min on the initial spin, and
if up to 3 L homogenate is done, use a 6 &times; 250 ml angled rotor
at 5000 <i>g</i>.  Collect supernatants through muslin or cheesecloth,
which removes floating fatty tissue;  passing through glass wool-plugged
funnels is an alternative.  Re-extraction of the pellet is an option,
especially if yields are the higher priority.

<h3>Native/Active Protein Extraction</h3>
<p>
An isotonic buffer of 0.25 M sucrose with 1 mM EDTA and
buffered in neutral (pH 7.0-7.6) Tris, MOPS,
HEPES, or Tricine is typical in these extractions.
Use of protease inhibitors is especially recommended for liver and
any tissues known to contain them.  0.1-0.5 mM 2-mercaptoethanol or DTT
will prevent oxidation.fs
<p>
<b>Intact Nuclei</b>.  For extraction of cells to prepare intact nuclei,
do not include EDTA but add KCl and MgCl<sub>2</sub> or other divalent
cation (Ca<sup>2+</sup> can activate proteases!).
<p>
<b>Mitochondria</b>. Make sure a nonelectrolyte isotonic component like
sucrose is present, except for mitochondria from skeletal muscle, where
100-150 mM KCl should be substituted for sucrose, since sucrose with
myofibrils is too viscous.

<h3><q>Total</q> Solubilization</h3>
<p>
If the cell or tissue proteome is to be extracted in one go rather than
make use of fractionation procedures, the Walker protocol book describes the
following <q>General Solubilization Cocktail</q> for lysis/extraction/rehydration
of cells and tissues as having these components:
<ul>
<li>5 M urea
<li>2 M thiourea
<li>0.25% (v/v) CHAPS (Sigma)
<li>0.25% (v/v) Tween-20 (Bio-Rad)
<li>0.25% (v/v) sulfobetaine 3-10 (Sigma)
<li>0.25% (w/v) carrier ampholytes:  this is a 1:1:1:1 mix of Bio-Rad
BioLyte 3-10, Serva Servalyte 3-10, Amersham Biosciences Ampholine 3.5-9.5,
and BDH Resolyte 4-8
<li>2 mM TBP
<li>10% isopropanol
<li>12.5% (v/v) water-saturated isobutanol
<li>5% (v/v) glycerol (BioRad)
<li>1 mM sodium vanadate (Sigma)
<li>1 &times; complete protease inhibitor cocktail (Boehringer-Mannheim)
</ul>
<p>
When tightly sealed and stored at &minus;80&deg;, it can be kept for several
weeks.  Alternate components include Triton X-100 (0.25-0.5%) for Tween-20,
100 mM DTT or 5% 2-mercaptoethanol for TBP.  Ampholytes should be
increased when the sample amounts are high (but not more than 1.5%),
and detergents are best kept in the 0.75-2% range.


<h3>Cytoplasmic Proteins</h3>
<p>
Selective extraction of cytosolic proteins of mammalian cells can be achieved 
with low salt buffers containing high amounts of sucrose to maintain the 
integrity of organelles such as nuclei and mitochondria to prevent their 
proteins from becoming part of the mixture.  Combined with gentle mechanical 
cell breakage using a Dounce homogenizer with Pestle type A will disrupt the 
cells without disrupting nuclei (avoid use of Pestle type B in the Dounce for 
this reason). The extraction buffer should control pH (Tris or phosphate will 
do), include a thiol to prevent oxidation (1-10 mM 2-mercaptoethanol or DTT or 
DTE), a chelator such at EDTA in case cytosolic components are bound to nuclear 
or organellar membranes via divalent metal interaction (Ca<sup>2+</sup> or 
Mg<sup>2+</sup>), and inhbitors of proteases for the specific types being 
extracted.  It also might be useful to include protein kinase and phosphatase 
inhibitors of modifications such as phosphorylations or dephosphorylations are 
to be avoided.

<h3>Membrane Proteins</h3>
<p>
Canelle et al. (<i>Electrophoresis</i> <b>27</b>: 1609-1616, 2006) reported that
they found a number of efficiently extracted proteins from MCF-7 breast cancer
cells that could not be found in the whole cell extract.  Cells are extracted
in 25 mM Tris, 10 mM EDTA, 7 M urea, 2 M thiorue, 5% glycerol 0.33% CHAPS,
0.35% TX-100, 0.35% SB 3-10, 10% isopropanol, 12.5% isobutanol, 100 mM DTT,
1 mM orthovanadate, and protease inhibitor cocktail.  After spinning, the pellet
is suspended in 10 mM HEPES, 1.5 mM MgCl<sub>2</sub>, 10 mM KCl, 0.5 mM DTT,
1 mM orthoovandate, protease inhbitors.  Lysate is first spun at 600 &times; g
10 min to remove nuclei, then 100,000 <i>g</i> for 25 min.
<span style="color:blue;font-weight:bold;">The cell pellet is then resuspsended
in 2:1 (v/v) trifluoroethanol:chlorform</span>.
Spinning is 10 min at 10000
<i>g</i> and there are three phases:  the most upper is the TFE phase should
contain the transmembrane proteins of interstes,


<h2 id="plantxtract">Plant Tissue Extraction</h2>
<p>
Different tissues of various plants are among the toughest tissues to
extract.  Animals support their structure usually on a bony skeleton,
which itself is difficult to extract as a tissue.  But plants have no
endoskeleton and they support their tissues by synthesis of polymers
which form into crosslinking fibers which are quite chemically and mechanically
resistant to disintegration.  These products include different kinds
of celluloses and polyphenolic products (lignins).  Modifications to the
polymers provide toughness, or a resistance to dehydration or loss of
stature.
<p>
Typically leaves and roots are the most interest with respect to metabolically,
biochemically, or physiologically active tissue.  The fruit/flower of plants
is also of interest.  Stems may be of interest from time to time, but
the usually develop a thick cambial/phloem layer that makes extraction of
the tissues a lower consideration.
<p>
Generally mechanical methods are more effective in homogenization of
multicellular tissues.  While in the animal tissues, brief pulses in a
Waring blender are sufficient, more sustained blending may be necessary for
plant tissues.  Many plant scientists are using grinding of plant tissues
frozen rapidly by liquid nitrogen.  The rapid freezing causes ice crystal
formation intracellularly, which helps to pierce membranes such as the
cell/plasma membrane, or that of the tonoplast.  The rock-like plant
tissue is then ground with a durable ceramic mortar and pestle until a
relatively very dry powder is formed.  The dry powder is then placed in
extraction buffer to complete the process;  additional mechanical processing
might even be done (blending, ultrasonication).
<p>
For large-scale extractions, liquid nitrogen grinding is impractical,
and so a blender must be used.  If liquid nitrogen is not available
for small-scale fractions, use of acid-washed sand in mortar and pestle
is an alternative.
<p>
Plant proteins have been characterized by Osborne into four classes or
groups:
<ol>
<li>albumins:  proteins extracted into water
<li>globulins:  proteins extracted into dilute salt solutions
<li>prolamins:  proteins extracted into alcohol-water mixtures
<li>glutelins:  proteins extracted into dilute acid or alkali
</ol>
<p>
Osborne sequentially extracted plants in this fashion.

<h3>Grinding of Plant Tissues Frozen By Liquid Nitrogen</h3>
<p>
This is effective for grinding small amounts (no more than 50 grams)
of tissue, and not for large-scale extraction processes (consider other
means).
<p>
For most extractions, it is typical to use 100-200 ml of liquid nitrogen.
It can vary greatly with the size of the mortar used, the type of
tissue used (and its water content), the lab room temperature, and many
other factors.


<ol>
<li>Wear gloves.  Powder-free nitriles are recommended.  They protect
your sample from contamination by your own proteins, and will protect
your hands from contact with liquid nitrogen.
<li>Obtain your plant tissue to be extracted.  Remove, cut, trim away
all tissue that is not be extracted.  Wash your tissue is tap water
to remove all dirt and/or soil.  It may even help to soak the tissue in
a large container of water with changes of water until no more soil is
observed to make the water dirty or settle out.  Be very patient and
give plenty of time to this process.  The more unwanted tissue removed
before extraction by grinding in liquid nitrogen, the more pure the
material will be.
<li>Obtain a ceramic mortar and pestle of the correct size.
The mortar should be fairly
thick and not one of those thin bowls, which might crack with the sudden
temperature changes up and down in liquid nitrogen.  Mortars and pestles
can be cleaned with liberal amounts of tap water, then rinsed possibly with the
extraction buffer to be used (although not really required) or rinsed
with distilled water.  They should be allowed to dry before doing the
procedure.  To assist drying, they can be placed in an oven of low heat
(up to 80&deg;) then allowed to cool to room temperature.  NEVER pour
liquid nitrogen or place plant material in a mortar that is above room
temperature!  Sudden temperature shifts could crack the mortar, or biomolecules
in the plant can be denatured by hot mortars.
<li>Cut or dice your plant material into small pieces if possible.  Large
chunks of tissue are impossible to grind when frozen.  Cubes of tissue no
more than 5 mm cubic are best.  The smaller tissue is, the faster and more
complete the grinding process.  Large flat broad leaves might be shredded
or cut with scissors.  Roots can be cut with scissors or razor blades or
scalpel blades.  The amount of material placed in the mortar should not
be more than 25-33% of its capacity;  repeated extractions might be necessary
if more tissue is to be processed.  Weigh the fresh, pre-ground material
if it is desirable.  However, note that not all the ground material is
recovered or used in extraction.
<li>Carefully and slowly pour the liquid nitrogen over the top of the
tissue.  If the pouring is too fast, a lot spills over the mortar and is
wasted.  The amount of liquid nitrogen used should cover and be at least
twice the volume of the material in the mortar.  Do not fill the mortar;
that is rarely necessary and prolongs the time when grinding can begin.
<li>Allow the tissue to freeze to a rock-solid condition.  This is at least
30-60 seconds.
<li>Hold the pestle firmly, and begin to press down on to the tissue,
gyrating the pestle as it is pressed down.  Do NOT HAMMER the tissue
with the pestle!  It does not effectively grind the tissue, and there
is a big risk of breaking/cracking the mortar.  The way of using a pestle
in the mortar is press down and gyrate the pestle.  The rocks break into
smaller pieces, and eventually the smaller pieces turn into a powder.
In fact, the process will be complete when all tissue has effectively been
turned into a powder that is easily blown up as a dust (but do not that!).
<li>In the course of grinding, all the liquid nitrogen is likely to
evaporate, and quickly.  This is actually desirable, since one does not
want to be grinding in the presence of liquid nitrogen.  If in the course
of grinding you notice that the tissue has become wet and is not grinding
but forming a paste, stop immediately, and pour in another small amount
of liquid nitrogen.  You must be even more careful in pouring this
extra amount of liquid nitrogen, because the powder you have already formed
may be spilled or pushed out by the poured liquid.  You can minimize this by
bringing the mouth of the Dewar closer to the mortar top, and also by
pouring in an empty area of the mortar.
<li>When the dry powder has completely formed in amounts desired, use
a spoon or spatula to scoop the powder into tared (pre-weighed)
storage tubes or a tube to
which extraction buffer may be added.  The selection of the extraction
buffer varies with the application (enzyme or protein purification?
proteomic profiling or 2D-electrophoresis?).  Do not fill an extraction
tube with the powder since you have to allow room for pouring buffer.
If the powder is not to be immediately processed, it can be safely stored
at &minus;20&deg; for a long time (several weeks to a few months or even
longer), sealed tightly (crytotubes, or containers whose tops are
wrapped with parafilm).  It might be better to use &minus;80&deg; or even
4&deg;, depending on the product to be extracted and its known stability.
It can be kept at 4&deg; if it will be extracted within a few days.
Weigh the tubes with the powder, and note the net weight.  Always note the
weight of powder actually extracted.
</ol>

<h3>Extraction Buffer Components</h3>
<p>
Generally a reducing agent, 1 mM DTT or 10 mM 2-mercaptoethanol, should be
present to avoid oxidation.
<p>
A 1 mM EDTA presence will chelate heavy
metals that deactivate some proteins, and to slow protease inhibitors;
use especially with phosphate buffers that tend to carry ferrous ions.
<p>
50 mM NaF inhibits phosphatases.
<p>
Polyvinylpolypyrrolidone (PVPP) is an insoluble form of polyvinylpyrrolidone
(PVP) which added 25 g per kg fresh weight is useful in binding phenolics.
It is removed in centrifugation.
<p>
PMSF at 0.1 mM will inhibit serine proteases.
<p>
Some enzymes have their activity stabilized by presence of glycerol (up to
30%) or ethylene glycol or mannitol.
<p>
Addition of casein or BSA stabilizes enzymes because these proteins serve
as alternative substrates for proteases.

<h3>Sigma-Aldrich Plant Cell Lysis/Extraction Reagent</h3>
<p>
The CellLytic&trade; P product (C-2360) will provide extract of
fresh or frozen leaves for many types of uses of the protein
as it maintains biological activity.  For each milliliter one can
process 500-1000 mg of leaf tissue.
<p>
The procedure involves grinding fresh or frozen leaves in liquid
nitrogen with a mortar and pestle and adding at least 2 ml of the
CellLytic P reagent per gram and grinding again.  As an alternative
it is possible to grind leaves directly in the reagent with a blender
or suitable homogenizer.  The extract is then centrifuged (max speed
in microcentrifuge for 10 min).  The supernatant is then used immediately
or frozen at &minus;70 or &minus;80&deg;.  Glycerol added to 10-25%
final concentration can help as a stabilizing cryopreservative, if it
does not interfere with the application.

<h3>Special Extraction Protocols</h3>
<p>
Notes here are taken from Fido et al <q>Protein Extraction From Plant
Tissues</q> in <a href="#cutler">Cutler</a>.
<p>
<b>Rubisco</b> is an often important enzyme in plants and this CO<sub>2</sub>-fixing
enzyme makes up 50% of chloroplast protein.  The extraction buffer
is important.  Extraction in wheat leaves should be done with a buffer
of 20 mM Tris HCl pH 8.0, 10 mM sodium bicarbonate, 10 mM MgCl<sub>2</sub>,
1 mM EDTA, 5 mM DTT, 0.002% (w/v) Hibitane (antimicrobial), and 1% PVPP.  The extract
should fastidiously be maintained in ice-water bath (2&deg;).  3-week
old leaves are cut into 1 cm lengths and homogenized 6:1 buffer:tissue.
Filter through four layers muslin, then add solid ammonium sulfate to
35% satn.  After the salt is equilibrated (20-30 min), spin 20 min
at 20,000 <i>g</i> 15 min.  Add ammonium sulfate to 55% satn to supernatant
and spin.  The pellet is resuspended in a pH 8 20 mM Tris buffer with
1 mM DTT, 1 mM MgCl<sub>2</sub> and 0.002% Hibitane.  The enzyme can
then be purified using sucrose density centrifugation.
<p>
When the tissue is rich in lipids, use of cold acetone extraction or
use of acetone powders should be considered.
<p>
For purification of nitrate reductase, another common plant protein,
see <a href="#cutler">Cutler</a> chapter named above.



<h2 id="cultcellxtract">Cultured Eukaryotic Cells</h2>
<p>Cultured eukaryotic cells will differ greatly from methods used to extract
prokaryotic cells, which are also cultured in order to grow a large mass.
<p>
<b>Cultured mammalian cells</b>.  Cultured mammalian cells have no cell wall 
which helps the cell to resist injury as well as extraction.  Mere changes in 
extracellular osmolarity (ionic strength) such as adding water to dilute the 
medium will cause cell volume to increase rapidly with lysis of the cells.  This 
is seen with hemolysis with addition of water to blood.  Addition of isotonic 
medium (e.g. physiological saline) would maintain osmolarity and not cause lysis 
by dilution.
<p>
The better approach for extraction mammalian cells is a mechanical extraction
using a <b>Dounce homogenizer</b>.  This is a glass mortar-pestle-like apparatus
in which the mortar has a long cylindrical round-bottom tube with a bowl-like
upper part to contain extract during processing, and the pestle-like part with
a rounded knob end which is a precise fit to the cylindrical tubular part of the
mortar.
<p><b>Dounce homogenizer use</b>.  In general, the material to be extracted is 
filled into the volume of the cylindrical tubular part.  Do not fill into the 
bowl-like portion;  if the volume to be extracted is greater than the amount the 
homogenizer can take, extract the total in aliquots.  Between 12-15 up-and-down 
strokes is often considered sufficient to process most types of cultured cells. 
However, the ultimate determination of how many strokes are necessary to 
complete the extraction process is to sample the extract at each stroke and look 
at a wet mount under a microscope.  If cells are to be completely lysed, the 
apparent count of cells in several optical fields should decrease in a cursory 
survey.  Where cell lysis with maintainence of intact nuclei is desired, a more 
careful examination under the microscope is necessary:  dyes which provide 
positive or negative staining to distinguish cytoplasmic rims from nuclear 
spheres under a brightfield/phase microscope might be helpful in giving 
determinations of the extent of extraction.  The Dounce mortar should be 
assiduously kept immersed in an ice-water bath where extractions are to be done 
at 0-4&deg;.
<p>
When extractions are done, centrifuging should be done appropriately.  Where
extractions were to retain intact nuclei, spin only at an RCF and time necessary
to spin down the nuclei and not cytoplasmic membranes.  While this may need
to be empirically determined, 


<h3>Cytoplasmic Proteins</h3>

<h3>Membrane Proteins</h3>

<h2>Total Protein Extraction</h2>

<h3>University of Missouri - Columbia Proteomics Center</h3>
<p>
These were obtained from a web site published by U. Missouri under
a page titled <q>Protein extraction from whole tissues for
isoelectric focusing.</q>  The text is virtually reproduced below.

<h4>SDS extraction followed by acetone precipitation</h4>
<p>
This is a recommended starting protocol for whole tissue extractions.
<ol>
<li>Grind 1 g of fresh tissue to a powder with liquid nitrogen
in a mortar and pestle.
<li>Add 5 mL of extraction media (0.175 M Tris-HCl, pH 8.8, 5% SDS,
15% glycerol, 0.3 M DTT) directly to mortar and continue grinding
for an additional 30 sec.
<li>Filter the homogenate through two layers of miracloth
into a 50 mL Falcon tube at room temperature.
<li>Immediately add 4 volumes of ice-cold 100% acetone to
the filtered homogenate, and mix by vortexing and place at &minus;20&deg;
for at least one hour to precipitate proteins.
<li>Centrifuge at 5000 <i>g</i> for 15 min to collect precipitated protein,
and discard the supernatant.
<li>Gently blot residual acetone from container with Kimwipe and then
wash pellet in 15-20 mL of cold 80% acetone. Be sure to thoroughly
break up pellet by pipetting, vortexing or sonication.
<li>Repeat the last two steps to wash again.
<li>Collect the final protein precipitate by centrifugation at
5000 <i>g</i> for 15 min.
<li>Dry pellet by inverting on Kimwipe for 15 min at 37 C.
<li>Resuspend the final pellet in 0.5-1 mL of IEF extraction solution
(8 M urea, 2 M thiourea, 2% CHAPS, 2% Triton X-100, 50 mM DTT,
0.5% pH 3-10 ampholytes) by pipetting and vortexing at 25-30&deg;.
Incubate sample for 1 h at room temperature with agitation.
<strong>Do not heat sample under any circumstances as this will
lead to carbamylation of proteins.</strong>
<li>Centrifuge for 10 min at 12000 <i>g</i> and use supernatant to
rehydrate IPG strips.
<li>If protein quantitation is necessary, precipitate protein sample
with TCA or acetone prior to performing Bradford or Lowry assay as
detergents and reducing agents interfere with these assays.
</ol>

    <h4>Phenol extraction followed by methanolic ammonium acetate precipitation</h4>
<p>
This is an effective protocol for sample preparation from protein-poor,
recalcitrant tissues such as plants (see Hurkman and Tanaka, 1986,
<i>Plant Physiol</i> <b>81</b>:802-806).

<ol>
<li>Grind 1 g of fresh tissue to a powder with liquid nitrogen in
a mortar and pestle.
<li>Add 2.5 mL of Tris pH8.8-buffered phenol and 2.5 mL of extraction
media (0.1 M Tris-HCl pH 8.8, 10 mM EDTA, 0.4% 2-mercaptoethanol,
0.9 M sucrose) and continue grinding for an additional 30 sec
in a fume hood.  Alternatively, transfer to a 15 or 50 mL Falcon
tube and homogenize in polytron homogenizer for one minute.
<li>Transfer to Falcon tube and agitate for 30 min at 4&deg;.
<li>Centrifuge 10 min at 5000 <i>g</i>, 4&deg;.
<li>Remove phenol phase (should be top phase) and back-extract top
aqueous phase with 2.5 mL + 2.5 mL of extraction media and phenol
by vortexing. Centrifuge and combine with first extraction.
<li>Precipitate phenol-extracted proteins by adding 5 volumes
of 0.1 M ammonium acetate in 100% methanol (stored at &minus;20&deg;)
to phenol phase.
<li>Vortex and incubate at &minus;20&deg; for at least 1 h or overnight.
<li>Collect the precipitate by centrifugation (20 min, 20,000 <i>g</i>, 4&deg;).
<li>For the following wash series, completely resuspend the
pellet each time with vortexing and if necessary, sonication
(this usually takes longer the first time).
Place the resuspended sample at -20 C for at least 15 min between
each wash. You can store the last suspended pellet in 80% acetone
at -20 C until ready for IEF, or dry the last pellet under
nitrogen (or at 37 C for 15 min) and store at -20 C.  The wash series is:
 <ul>
 <li>Wash the pellet twice with 0.1 M ammonium acetate in methanol.
 <li>Wash pellet twice with ice-cold 80% acetone.
 <li>Wash pellet once with cold 70% ethanol.
 </ul>
<li>Resuspend final pellet in 0.5-1 mL of IEF extraction solution
(8 M urea, 2 M thiourea, 2% CHAPS, 2% Triton X-100, 50 mM DTT,
0.5% pH 3-10 ampholytes) by pipetting and vortexing at 25-30 C.
Incubate sample for 1 h at room temperature with agitation.
<strong>Do not heat sample under any circumstances as this will lead
to carbamylation of proteins.</strong>
<li>If protein quantitation is necessary, precipitate protein sample
with TCA or acetone prior to performing Bradford or Lowry assay as
detergents and reducing agents interfere with these assays.
</ol>

<h2>Serum/Plasma Handling</h2>
<p>
Serum and plasma are very common starting points for proteome analysis,
especially the human proteome.  This is largely because they can be
obtained from living individuals without harm to the life of the individual.
Biomarker analysis is the typical purpose of analyzing the proteome:
it is usually hoped that detection of proteins or peptides or changes
in the proteome are sensitive and specific indicators of disease or disorder.
<p>
When the purpose is to look particularly for low abundance proteins,
it is necessary to deplete the serum or plasma of its high abundance
proteins, or else they can overwhelm and distort separations.  Trying to
detect low abundance proteins requires loading of these proteins in amounts
that would overload the high abundance proteins, and possibly damage
the system as a result.  Hence the use of products to deplete these
high abundance proteins specifically.
<p>
Sigma sells a Blue Albumin kit to deplete albumin specifically.  It also
reduces the gamma globulin fraction.
<p>
Beckman-Coulter is the licensed seller of GenWay Biotech&rsquo;s
Seppro microbead-conjugated avian IgY which can specifically deplete
albumin, fibrinogen (the major component of plasma that distinguishes
it from serum), transferrin, and immunoglobulin types G, A, and M.
It has broad specificity to all mammalian sera or plasma.

<h2 id="xtractcommercialprod">Commercial Products for Extraction</h2>

<h3 id="biorad">Bio-Rad</h3>

<h4 id="RPTotalXtractKit">ReadyPrep Total Protein Extraction Kit</h4>
<p>
Cat #163-2086 provides enough reagents for 20 preps.
<p>
This is a one-step extraction system which contains the following:
<ul>
<li>7 M urea
<li>2 M thiourea
<li>1% ASB-14
<li>40 mM Tris base
<li>2 mM tributylphosphine (TBP) [added from a 100&times; stock dissolved
in 1-methyl-2-pyrrolidine sealed under N<sub>2</sub> gase]
<li>0.001% bromophenol blue
</ul>
<p>
ASB-14 is strongly chaotropic.  Typically cells or tissues should be
50-100 mg.  Extraction is complete in 45 min.  2 ml centrifuge
tubes, microcentrifuges that spin at 12-16,000 &times; <i>g</i>
with controlled temp of 18-20&deg;, a sonicator, and protease
inhibitors (optional), 3-10 carrier ampholytes, and a protein
assay compatible for detergents and reductants should be used
(Bio-Rad RC DC).  For plant extractions, round bottom tubes
(see VWR #60818-725) and ReadyPrep 2-D cleanup should be used;
phenolics from plants can cause severe horizontal streaks.
<p>
The kit is stored at room temperature.  The Rehydration/Sample
buffer can be aliquoted in 1-2 ml volumes and stored at &minus;80&deg;.
Add 5.6 ml proteomics-grade water to the buffer bottle and swirl
until dissolved (never heat this to above 30&deg;, and then only for
a short time).
<p>
Each extraction uses 1.0 ml of the Rehydration/Sample buffer.
Add 10 &micro;l of TBP and the appropriate ampholyte to a 0.2% w/v
concentration.  Add protease inhibitors too at this time.
The final extraction solution is added to 50-100 mg animal tissue
or 50 &micro;l of wet cell pellet (cultured cells, yeast, bacteria).
For plant tissues add 2-3 ml buffer per gram and in a 14 ml round-bottom
tube;  the tissue should have been ground to a powder in liquid nitrogen.
<p>
Put the sample on ice and sonicate sufficiently to fragment the genomic
DNA thoroughly.  This is usually done in 30 second bursts, 3-4 times,
or until lysis is complete.  Keep chilled in ice to prevent heating
of the sample.  But take care that it becomes too cold, urea and thiourea
can come out of solution.  Bacteria will require more sonication than
animal tissues, and yeast are very resistant to sonication.
<p>
The extract is spun at maximum speed in microcentrifuge (~16,000 &times; <i>g</i>)
for 20-30 min at 18-20&deg;.  Transfer the supernatant to new tube.
A cleanup of the supernatant might be necessary to avoid streaking of the
IEF gel.  A Lowry type assay is recommended for protein quantitation
(try the Bio-Rad RC DC assay).
<p>
The following table is a guide to protein and volume loadings
on to IPG strips for this material:

<table>
<caption>IPG Strip Capacities</caption>
<tr><th rowspan="2">IPG Strip Length (cm)
   <th rowspan="2">Rehydration Volume (&micro;l)
   <th colspan="2">Protein Load (&micro;g)
<tr><th>Silver Stain            <th>Coomassie G-250
<tr><td>7   <td>125   <td>5-20   <td>50-100
<tr><td>11  <td>185   <td>20-50  <td>100-200
<tr><td>17  <td>300   <td>50-80  <td>200-400
<tr><td>18  <td>315   <td>50-80  <td>200-400
<tr><td>24  <td>410   <td>80-150 <td>400-800
</table>


<h4 id="RPSeqXtractKit">ReadyPrep Sequential Extraction Kit</h4>
<p>
Cat #163-2100, enough for 15 preps.  This kit makes use of the
differential solubility
in the detergent SB 3-10 based on work of Molloy et al. and
Herbert et al.  There are three reagents:
<ul>
<li>Reagent 1 extracts only the most soluble, typically cytosolic proteins. This 
is 50 ml of 40 mM Tris base.  Should use no more than 3.3 ml per prep if each 
gives 15 preps.
<li>Reagent 2 extracts those of intermediate solubility, and is
three vials with each vial 10 ml of 8 M urea, 4% CHAPS, 40 mM Tris,
and 0.2% (w/v) Bio-Lyte 3/10 ampholytes.  Should use no more than
2 ml per prep if 15 preps are done.
<li>Reagent 3 extracts the least soluble, and is two vials with each
vial 10 mL of 5 M urea, 2 M thiourea, 2% CHAPS, 2% (w/v) SB 3-10,
40 mM Tris, and 0.2% (w/v) Bio-Lyte 3/10.  Should use no more than
1.3 ml per prep if 15 preps are to be done.
</ul>
<p>
A ampule contain 0.6 ml of 200 mM tributyl phosphine (TBP) in
1-methyl-2-pyrrolidinone (NMP) is provided sealed in nitrogen gas.
A storage vial is provided for it as well.
<p>
SB 3-10 is <i>N</i>-decyl-<i>N,N</i>-dimethyl-3-ammonio-1-propanesulfonate,
or caprylyl sulfobetaine, a zwitterionic detergent.
<p>
The kit is used by reconstituting all vials with deionized water.
50 ml for Reagent 1, 6 ml for each vial of Reagent 2, and 6.3 ml
for each vial of Reagent 3.  For storage, aliquots of 10-20% (about 1 ml)
of the total volume should be made to distribute the product, and the
frozen at &minus;20&deg; or less.
<p>
TBP should be handled in a fume hood. Use wet towels to wipe spills.
Transfer the contents to screw cap vial and store at &minus;20&deg;
or lower.  Add 10 &micro;l to each 1 ml of Reagent 2 and/or Reagent 3
but not to Reagent 1.  Use only polypropylene tips (not polystyrene tips!
these dissolve in NMP).
<p>
Take the following steps in doing this serial extraction:
<ol>
<li><b>Extraction</b>.  Extraction is done usually by physical means. As an 
example, if bacteria are the sample, these are harvested from growth medium and 
first washed in Reagent 1.  While suspended in Reagent 1 they are lysed by 
sonication or shearing in a French press.  Animal tissues are homogenized or 
sonicated in Reagent 1 without a real need to wash.  Plant tissues are usually 
grounded and pulverized after first freezing them in liquid nitrogen, and the 
powder made by grinding is then resuspended in Reagent 1.  Cultured cells are 
washed free of growth medium in isotonic solution (PBS or HBSS), and then the 
pellet is resuspended in Reagent 1 (which should cause lysis) and then in order 
to complete or ensure the lysis, the suspension is sonicated or pushed through a 
homogenizer (Dounce).
<p>
Nucleases might be included in the preps.   DNase I at 20-100 &micro;g/ml
and RNase A at 5-25 &micro;g/ml.  There are also nonspecific endonucleases
at 150 U/ml that can be used.
<p>After the physical extraction, the protein solution should be
about 50 mg/ml.  If 0.5 g of <i>E coli</i>, with an approximate dry
weight mass of 100 mg, was suspended in 2 ml of Reagent 1, this would
yield about 50 mg/ml protein.  As far as cell/tissue lysis or
homogenization,
<a href="http://www.expasy.ch/">ExPASy</a> or
<a href="http://biobase.dk/cgi-bin/cells">Biobase</a> can be consulted
on sample prep.
<li>After lysis, spin the extract.  For bacterial lysates, 15 min at top speed 
in a microcentrifuge is sufficient.  For other samples, 10-15 min will be 
enough.
<p>
How do you know how much time and speed to spin an extract? <li>Separate the 
supernatant into one tube (a suggested labeling is <q>E1</q> for <q>Extraction 
1</q>), and take a small aliquot (2-5 &micro;l) into another small (500 &micro;l 
conical) tube for preparing a protein assay.  If the supernatant will not be 
going to IEF separation immediately and will be stored long-term, it is 
recommended that the supernantant be stored in a cryotube (such as an O-ring 
screw cap conical tube designed for long-term freezing) and placed at &minus;70 
to &minus;80&deg;.  Otherwise, store in tightly capped tube at &minus;20&deg;.
<li>The pellet (<q>E1 pellet</q>) The supernatant is then evaluated for protein 
using Bradford.  The pellet is washed twice in Reagent 1 (determine their 
protein content before discarding them!).  Store frozen until use.  Dilute the 
supernatant with Reagent 2 (containing TBP) so that protein is about 1 
&micro;g/&micro;l protein (the typical load for 2D-PAGE).   When doing IEF on 
this extract, start at a low voltage at first:  250 V for the first hour, 500 V 
for the next hour, then ramp up to the final voltage.  The paper wick electrodes 
will help to absorb impurities.
<p>
For next extraction with Reagent 2, add 10 &micro;l TBP for each 1 ml of Reagent 
2 used (final 2 mM TBP).  <em>Half</em> the volume used for Reagent 1 should be 
used for Reagent 2, although the optimal volume will be empirically determined. 
Vortex the pellet from the extraction in Reagent 2 for 5 min.  It might be 
necessary to sonicate or to pass the pellet through a fine-gauge needle to break 
it up.  Spin to give a firm pellet and clear supernatant.  Recover the 
supernatant, determine its protein by assay, and freeze the supernatant.  Wash 
the pellet twice in Reagent 2, and discard if the protein concentrations of the 
washes is insignificant.  Prior to 2D-PAGE, dilute to 1 &micro;g/&micro;l. 
Start out doing IEF in the same way as for the first extraction.
<p>
For the 3rd extraction, add TBP to Reagent 3 as done for Reagent 2.  Use about 
the same volume of Reagent 3 as used for Reagent 2 in solubilizing and 
extracting the pellet.  Vortex and spin as above in extraction #2.  Assay and 
freeze supernatant, and dilute to 1 &micro;g/&micro;l for IEF.
<p>
A final extraction of the pellet in SDS might recover more material. See Molloy 
et al. for more information.
</ol>

<p>
IEF may need to be performed separately on extracts 1, 2, and 3, since
their conductivities could vary.
<p>
Note that the Bio-Rad protein assay may need to be modified slightly to help in 
the assay of the sequential extracts, because of the presence of detergents. 
Instead of a 1:5 dilution, prepare a 1:4 dilution of dye reagent and filter 
throught Whatman 1 or equivalent; this assay assumes the use of 3.5 ml dye 
reagent per tube.  Dilute concentrated HCl to 1/100 (about 0.12 N HCl); 80 
&micro;l will be used per tube.  A standard curve in the range 2-280 &micro;g is 
prepared with gamma globulin:  dilute in extraction solution from 14 to 0.1 
mg/ml (although TBP can be omitted).  Add 80 &micro;l of the 0.12 N HCl. Mix 3.5 
ml of dye and vortex gently.  Measure 595 nm absorbance after 5 min.  The 
standard curve will be nonlinear.  Measure the samples, and dilute within the 
assay range if too concentrated.
<h4 id="RPsolinsolXtractKit">ReadyPrep Soluble/Insoluble Extraction Kit</h4>
<p>
Cat #163-2085 is a kit that divides the extraction basically into a soluble 
hydrophilic protein fraction and that of an insoluble hydrophobic proteins. 
Samples of all kinds (animal and plant cells and tissues, yeast, bacteria) are 
appropriate for the kit. Sample is ultrasonicated in the lysis buffer and 
centrifuged to obtain the soluble protein supernatant and the insoluble protein 
pellet, and a 2nd extraction is done on the pellet to solubilize additional 
soluble proteins.  Finally the insoluble proteins are solubilized.
<p>
The following steps are taken to prepare and use the kit:
<ol>
<li>Add high quality water (5.6 ml)  to the reagent bottle marked
<b>ReadyPrep 2-D Rehydration/Sample Buffer 1</b> which is composed of
the same ingredients as the total extraction buffer kit (urea, thiourea,
ASB-14, Tris base).  Warm it with the hand only; heat above 30&deg; will
cause cyanate formation.
<li>Break open the ampule containing TBP reagent and transfer it
to the screw cap vial.  When storing, screw tightly and store at &minus;20&deg;.
Keep on ice when using at side.
<li>Put the contents of a pouch marked <b>Lysis Buffer</b> into a 50 ml
tube, then add 50 ml of high quality water.  Chill lysis buffer on ice
about 10-15 min before extraction.
<li>With a 2.0 ml conical tube containing 100-200 mg tissue or wet cell
pellet on ice (cell culture, yest, bacteria, animal cells or tissues,
etc), add 1.0 ml of Lysis Buffer.  If working with plants, first
grind the tissue in liquid nitrogen into a powder, then put the powder
into a disposable 14 ml round bottom tube and add 2-3 ml Lysis Buffer.
Adjust the buffer-to-sample volume ratio that works best.  Too little buffer
can result in poor or incomplete solubilization.
<li>Sonicate with ultrasonic probe.  Usually 30 second periods with 3 or 4
periods, cooling on ice in between, will be enough.  Alternatively
for yeast, use of a BioSpec BeadBeater with glass beads may be better.
If a sample is viscous, continue sonication to break up the genomic DNA.
Alternatively to break up the DNA, one can add 150 U benzonase and 2 &micro;l/ml
of 1 M MgCl<sub>2</sub>, and leave 20 min at 4-8&deg; after extraction.
<li>Spin in the microcentrifuge at maximum speed for 20-30 min at 4&deg;.
<li>Transfer supernatant to clean tube and put on ice.
<li>Add to the pellet an equivalent volume of the Lysis Buffer and sonicate
briefly to break up the pellet.  Repeat the centrifugation.
<li>Combine the supernatant with the first supernatant.  This fraction
is the <q>Soluble</q> fraction.
<li>Prepare the next solution (2-D Rehydration/Sample Buffer 1).  Make sure
that TBP is added.  It might also be useful to add ampholytes in the range
in which IEF is to be done (up to 0.2% w/v).  Protease inhibitors might be
useful to add also.
<li>Add 0.5 to 1.0 ml of the prepared 2-D Rehydration/Sample Buffer 1,
and vortex and/or sonicate to suspend the pellet and solubilze.  Keep in
mind when sonicating the sample might become heated and this might hydrolyze
the urea; cool it after short bursts, but only briefly since urea/thiourea
have lower solubility in cold solutions.
<li>Spin maximum speed in microcentrifuge for 10-20 min at room temperature
(18-20&deg;).  Collect the supernatant and discard the pellet.
<li><b>Protein Assay</b>.  Best to use Bio-Rad's RC DC (Lowry type) kit,
which can work with detergents and reducing agents.
</ol>

<h4 id="RPcleanupKit">ReadyPrep 2-D Cleanup Kit</h4>
<p>
Cat #163-2130 does a number of things to prepare samples for 2-D gel 
electrophoresis.  It is essentially a TCA precipitation kit which purifies 50 
samples of 100 &micro;l volume and having up to 500 &micro;g protein (5 mg/ml in 
a 100 &micro;l volume).  The kit contains:

<ul>
<li>Precipitation Agent 1 (TCA) [15 ml]
<li>Precipitation Agent 2 [15 ml]
<li>Wash Reagent 1 [2 ml]
<li>Wash Reagent 2 [50 ml] (likely acetone)
<li>Wash Additive 2 [0.25 ml]
</ul>
<p>
Wash Reagent 2 must be cooled to &minus;20&deg; for 1 h before use. Basically 
the sample up to 100 &micro;l is placed in 1.5 ml conical tube. Up to 500 
&micro;g can be placed.  Fill the rest of the volume with water up to 100 
&micro;l final volume.  300 &micro;l of Precipitating Agent 1 is added, and 
sample is vortexed and incubated on ice for 15 min. Then 300 &micro;l of 
Precipitating Agent 2 is added and vortexed. The material is then spun at 
maximum speed for 5 min to form a pellet. Remove the supernatant and discard. 
Spin again to get out more supernatant. 40 &micro;l of Wash Reagent 1 is added; 
the tube is spun for 5 min max speed, and supernatant discarded.  Now added 25 
&micro;l water, vortex 10-20 seconds.  Add 1 ml of chilled Wash Reagent 2 and 5 
&micro;l of wash 2 additive, vortex 1 min.  Incubate &minus;20&deg; for 30 min, 
and vortex for 30 seconds every 10 minutes.  Spin again for 5 minutes at top 
speed and remove supernatant.


<h3 id="sigma">Sigma-Aldrich</h3>
<p>
<b>ProteoPrep<sup>&reg;</sup> Total Protein Extraction Kit</b>. This kit 
(product <b>PROT-TOT</b>) does a sequential extraction. The kit reagents have 
the following compositions:
<ul>
<li>Extraction Reagent Type 1: urea, CHAPS (product C-0481)
<li>Extraction Reagent Type 2: 7 M urea, 2 M thiourea, CHAPS, SB3-10
  (product C-0606)
<li>Extraction Reagent Type 3: ? (product C-0731)
<li>Extraction Reagent Type 4:  7 M urea, 2 M thiourea, 1% C7BzO,
40 mM Tris base (product C-0356) to a final pH 10.4
</ul>
<p>
The reagents are used in order in a process that involves their separation by 
(micro)centrifugation and separation of pellets and supernatants.  This kit can 
process at least ten 2 ml samples with all four reagents.  For the 2 ml samples 
use between 50-100 mg of a wet cell paste for any species or about 250 mg 
tissue.
<p>
<b>ProteoPrep<sup>&reg;</sup> Universal Extraction Kit</b>. Sigma sells a 
product (<b>PROT-TWO</b>) used for simple soluble and membrane protein 
separation.  Cells are sonicated or extracted using an appropriate method. 
After spinning the supernatant is dried and resuspended in chaotropic buffer, 
and the membranes in the pellet washed once.  Both fractions are reduced and 
alkylated and the salt low enough for IEF.  The kit is contains powder to make 5 
mM Trizma (solulbe cytoplasmic extraction reagent, product S-2688);  a powder 
for making 7 M urea, 2 M thiourea,1% C7BzO detergent (resuspends the dried 
soluble proteins) (product S-3688); Protein Extraction Reagent Type 4 (product 
C-4356) tributylphosphine in 5 &times; 0.5 ml ampules containing 
N-methyl-2-pyrrolidone (the reducing agent, product T-7567); iodoacetamide (the 
alkylator, product A-3221).  Required equipment and materials include high 
purity water, a 37&deg; water bath, a lyophilizer or SpeedVac, sonicator, 
centrifuge and tubes. Cells are extracted using ultrasonic homogenization in the 
soluble cytoplasmic reagent and then centrifuged.  The pellet is re-extracted in 
the reagent and the supernatant pooled with the first.  The supernatant is dried 
down by centrifugal evaporation and the dried pellet dissolved in Soluble 
Protein Resuspension Reagent (S-3688) and kept on ice.  The pellet is treated 
with Extraction Reagent Type 4 by sonication and centrifuged. After 
centfrifuging the pellet is discarded and the supernatants from the soluble and 
membrane extractions treated with TBP (final 5 mM concentration) and then 
iodoacetamide (final 15 mM concentration).  Residual iodoacetamide is removed by 
adding excess TBP (to 5 mM) and protein then assayed by Bradford (better results 
than BCA or Lowry-types).  The kit can process ten or more 2 ml samples.
<p>
<b>ProteoPrep<sup>&reg;</sup> Membrane Extraction Kit</b>.  This kit (product 
<b>PROT-MEM</b>) helps to solubilize membrane proteins. The procedure involves 
use of a reagent (resuspended from powder product S-2813) for soluble 
cytoplasmic and loosely-bound membrane proteins into which cells are suspended 
and then extracted by an ultrasonic homogenizer (sonicated on ice for 1-2 
minutes, which breaks cells and DNA). More solution is added and allowed to be 
stirred on ice for an hour.  This is then ultracentrifuged.  The pellet is 
recovered and washed twice first with water.  Finally the pellet is treated with 
Protein Extraction Reagent Type 4 and sonicated on ice for four 15-sec periods. 
Cell debris is removed by centrifugation (microcentrifuge at max speed for 45 
min at 15&deg;).  The supernatant is treated with 2 mM TBP for an hour and then 
in 15 mM iodoacetamide from 90 min.  Insoluble material is removed by 
centrifugation.  Protein needs to be measured and the material possibly diluted 
with Type 4 reagent to load the right amount for IEF separation.  The kit 
processes six 2 ml samples.
<p>
<b>ProteoPrep&trade; Blue Albumin Depletion Kit</b>.  This kit (product 
<b>PROT-BA</b>) that depletes the albumin and gamma globulin fraction from 
serum.  A proprietary blue matrix (not Cibrachron Blue) is used to remove 
albumin while Protein G agarose is used to deplete gamma globulin.  The kit can 
process up to 25 samples of serum of 25-100 &micro;l volume.

<h3>Pierce</h3>
<p>
<b>Mem-PER Membrane Protein Extraction Kit</b>.  This product focuses on extraction
of <em>integral</em> membrane proteins of cultured mammalian and yeast cells
and from mammalian tissues.  Cells are lysed in one detergent, and then a 
second detergent for solubilizing the membrane proteins.  At 37&deg; a phase
separation is achieved between the detergents to separate hydrophobic and 
hydrophilic proteins.  For membrane proteins with transmembrane crossings
number up to 4, extraction efficiency is up to 90%.  Cytosolic protein 
contamination is &lt; 10%.  Protocols for four different starting materials
(cultured mammalian cells, yeast cells, soft tissue, hard tissue) are detailed.
Cells/tissue are lysed in Reagent A, and then a 2:1 Reagent C:Reagent B mix
is added in the cold.  This supernatant is then subjected to 37&deg; incubration
to achieve a phase separation, in which the bottom phase contains the integral
membrane proteins.  The difference in the protocols are mainly in the first
steps where the mechanical homogenization differs.  The user may also perform
a second extraction of pellets to gain additional material (usually unnecessary).

<h2>Sample Considerations</h2>

<h3>Nucleic Acid</h3>
<p>
DNA and RNA complicate proteomic analysis in several ways.  They cause high 
viscosity when pipetting and they can cause horizontal streaking in 2-D gels by 
acting as ion-exchangers during IEF [Walker].  Their precipitation might also 
cause loss of any associated protein as well. In the case of precipitated DNA, 
it might be useful to extract the precipitate with detergent (see Giavalalisco 
et al. <i>Electrophoresis</i> <b>24</b>: 207-216, 2003).  Use of DNase and/or 
RNase can be done. Shearing in a narrow gauge needle (21-gauge) might be useful. 
QIAShredder (QIAgen) is a product purports to reduce this problem

<h3>Phenolics, Lignins</h3>
<p>
Plant tissues can form polyphenolic substances in the course of their extraction 
which can cause streaking in 2-D gels.  Insoluble polyvinylpyrrilodone (PVPP) 
will inhibit formation and promote removal.



<!--  Protein Quantitation (Protein Assay) -->

<h1 id="proteinquant">Protein Quantitation (Protein Assay)</h1>
<p>
In planning the setup of protein assays, consider the <b>scale</b> of the assay: 
how much volume is necessary for measurement in the spectrophotometer?  For 
example, for the NanoDrop ND-1000, one only needs 2 &micro;l.  Thus at the most 
one can plan a final assay of 10-20 &micro;l if it can be done practically. 
Consider also the number of samples:  if several tens of samples are to be done, 
it is probably best to set up a microtiter plate-type assay, which can be done 
for many of the assays here.
<p>
In general, the first assay of choice is the Bradford assay.  It is rapid and 
convenient, with minimal setup, and can be scaled easily. However, take note 
that the Bradford and each other assay has its interferences.  If the protein 
solution being assayed is known or determined to have interfering substances, an 
alternative assay may have to be used.

<h2 id="protstd">Protein Standard/Reference</h2>
<p>
Bovine serum albumin (BSA) has traditionally been used, and
there is no reason to use another protein.
Try to purchase BSA that is recommended as a protein assay standard.

<h3>Preparation</h3>
<p>
There are two approaches to preparing the standard for the assay. One is to 
prepare a single high concentration and then pipet different progressively 
linear volumes to generate the standard curve, and the other is to prepare 
several different, progressively linear concentrations and then pipet constant 
volumes.  We recommend the latter approach, as it is less subject to error.
<p>
For consistency, a single lot that lasts at least 6 months and possibly a year 
should be prepared.  To avoid changes caused by repeated freeze-thaw cycles, 
materials should be aliquoted in volumes that allow at most four freeze-thaw 
cycles.  For the Bradford assay (the first choice), two scales of standard 
concentrations (low and high) are done.  A 4-point standard curve of evenly 
distributed concentrations is considered sufficient for generating a reliable 
curve and also keeping reagent use to a minimum.


<h2 id="bradford">Bradford (Acidified Coomassie Brilliant Blue G-250) Assay</h2>
<p>
This was described by Bradford (<i>Anal. Biochem.</i> <b>72</b>: 246, 1976).
<p>
It is best to purchase prepared Coomassie dye rather than prepare it. Bio-Rad 
(Cat #500-0006) and Sigma (Cat #B-6916) sell quality reagent in half-liter size 
that lasts for months.

<h3 id="bradford-checklist">Checklist</h3>
<p>
Here is a checklist of consumables and equipment to have ready:
<ul>
<li>lab coat, gloves, and any long hair tied up
<li>0.5 ml to 1.5 ml conical tubes and a rack to hold them.  At least five tubes 
will be used for the standard curve and blank, and one, two or three for each 
unknown if examined in singlicate, duplicate, or triplicate, respectively. 
Suppose 3 unknowns are examined in duplicate:  this means 11 total tubes will be 
used for the assay
<li>Pipettors:
 <ul>
 <li>P-10 or P-20 to pipet 10 &micro;l volumes + yellow tips
 <li>P-200 for pipetting between 20-200 &micro;l volumes + yellow tips
 <li>P-1000 for pipetting between 200-1000 &micro;l volumes + blue tips
 </ul>
<li>highest quality (preferably Type I) water, at least 100-200 ml
<li>Bradford reagent (concentrated stock)
<li>Bovine serum albumin (e.g., Sigma A7030) or bovine plasma gamma globulin 
<li>Properly warmed (stabilized) spectrophotometer set at 595 nm
<li>disposable plastic (polystyrene) cuvette fitting spectrophotometer
<li>computer installed with Excel or curve-fitting program
</ul>

<h3>Stock BSA</h3>
<ol>
<li>Prepare 1.5 ml of 1000 &micro;g/ml of BSA in water in a 1.5 ml conical tube.  
Do not let it foam.
<li>To a 1.5 ml conical tube labeled "750 &micro;g/ml BSA" containing 150 
&micro;l water, add 450 &micro;l of the 1000 &micro;g/ml BSA tube.
<li>To a 1.5 ml conical tube labeled "500 &micro;g/ml BSA" containing 300 
&micro;l water, add 300 &micro;l of the 1000 &micro;g/ml BSA tube.
<li>To a 1.5 ml conical tube labeled "250 &micro;g/ml BSA" containing 450 
&micro;l water, add 150 &micro;l of the 1000 &micro;g/ml BSA tube.  In each of 
the four tubes there will be a 600 &micro;l volume.
<li>To limit the number of freeze/thaw cycles (each time we use 10 &micro;l
for a standard curve), for each standard, label four 0.5 ml conical tubes each 
"250", "500", "750", and "1000 &micro;g/ml BSA", and place 100 &micro;l into
the tubes.  Store/keep all standards at &minus;20&deg;.
</ol>

<h3>The Assay</h3>
<p>
The tubes are arrayed in two parts or sections.  One is for the standard curve 
and the other for unknowns.  The standard curve is a 5-point calibration, 
including the zero/water blank;  no replicates are done, but if extreme accuracy 
is desired, do it in duplicates.  Unknowns should be done in duplicates, even if 
multiple dilutions of a single sample are done. Dilutions should only be done to 
bring the sample absorbance value within interpolatable range. Do not process 
data not interpolatable (do not extrapolate!);  the Bradford assay is not truly 
linear, and standard curve fitting should be polynomial or splined.
<ol>
<li>Thaw samples if necessary.  If they are cloudy or show coagulates, 
spin them down maximum speed in microfuge and transfer supernatant to clean
tube.  Never assay a solution that is cloudy or has coagulates.
<li>To avoid re-assays, try to estimate the protein concentration of the sample.
Use the table below for preparing dilutions so that sample absorbances can be within
the range of the assay.  Since each assay uses 10 &micro;l volumes and unknowns
are done in duplicate, a minimum 20 &micro;l is used.  The dilution preparation
makes 30 &micro;l to be on safe side.
<li>Prepare the dilutions of unknowns, if necessary, as follows:
 <ol>
 <li>Compute 30 &micro;l &divide; <b>dilution</b> value = unknown stock to pipet
<br>If dilution is 1:5, then 30 &micro;l &divide; 5 = 6 &micro;l.  If dilution is
1:20, then 30 &micro;l &divide; by 20 = 1.5 &micro;l.  If the volume is
less than a volume that can be pipetted accurately with the pipettors available,
an intermediate dilution (1:10) is required.  Suppose 1.5 &micro;l is too
small to pipet, and a 1:10 dilution is made.  The adjusted formula is
now 30 &micro;l &times; intermediate dilution &divide; <b>prev calculated dilution</b> value.
Thus 30 &micro;l &times; 10 &divide; 20 = 15 &micro;l
 <li>Add the computed volume (should be &le; 30 &micro;l) of the unknown
or its dilution to a 0.2-0.5 ml conical tube.
 <li>Bring it up to 30 &micro;l with water:  compute 30 &micro;l &minus;
solution added to the tubes in previous step, and add that amount of water.
Each tube should have 30 &micro;l total now
 </ol>
<li>Place five conical tubes (as low as 0.5 to 1.5 ml can be used) in the rack.
Label them "0", "250", "500", "750", "1000".
<li>Thaw the standard tubes.  Place exactly 10 &micro;l in each standard tube.
<li>Label unknownn tubes however you wish.  Best is just to number then 
1, 2, 3, .... and write on paper what is added to the tubes.  For each unknown
and for each dilution of the unknown, add 10 &micro;l each to two tubes.
<li>Compute the volume of Bradford working reagent to prepare (working reagent 
is a 1:5 dilution of concentrated stock).  It is the number of tubes prepared in 
the previous three steps &times; 490 &micro;l &times; 1.15 / 1000 &micro;l/ml. 
Round <em>up</em> to the nearest ml.  Suppose there are 21 tubes in the assay 
(includes the 5 standard curve tubes).  This is 21 &times; 490 &micro;l &times;. 
1.15 / 1000 = 15.435 ml.  Rounding up, the value = 16 ml.
<li>Obtain a container which can contain the computed volume (for &gt; 15 ml
a 50 ml conical tube is likely;  for &lt; 15 ml, a 15 ml polypropylene tube
is perfect).  Add 0.8 &times; computed value of distilled water (or better
quality).  If computed volume is 16 ml, then add 12.8 ml water.
<li>Get the Bradford concentrate:  add 0.2 &times; computed volume to the 
container with the water.  If computed volume is 16 ml, add 3.2 ml of 
the Bradford acidic Coomassie Brilliant Blue G-250 concentrate.  Mix well.
The 1:5 working reagent is prepared.
<li>To each tube, starting with the standard curve, add 490 &micro;l of
the 1:5 working reagent.
<li>Allow the color to develop for at least 15 min.  Check the spectrophotometer
and make sure it is ready.
<li>A single cuvette can be used for the entire assay.  Begin with the blank, a 
zero the spectrophotometer with it.  Remove solution back to its 
tube&mdash;don't discard it!  it might necessary to re-read!&mdash;and add the 
next solution with known increasing absorbance (read the standard curve in 
order).  Always read the lower absorbance (greater dilution) before the next 
assay with higher absorbance.  When switching to new samples, rinse the cuvette 
liberally with much water, and make sure no water droplets are in it before 
adding another sample.  Shake out excess water with snapping wrist movements, or 
use a drying cuvette washer.
<li>If desired, use the spectrophotometer's assay calculator to compute
standard curve and report the unknowns.
<li>Verify the spectrophotometer's report (if there is one) using Excel.
In Excel, set up the standard curve with a polynomial fit, and obtain the
polynomial parameters (if not possibly, use least-squares line fitting by
using SLOPE and INTERCEPT functions, and then use these values to compute
unknowns).  Compute unknowns using Excel's polynomial fit.
</ol>

<table>
<caption>Dilution Preparation For Samples</caption>
<tr><th>Estimated Protein Concentration (mg/ml)
  <th>Dilution*
<tr><td>up to 1 mg/ml <td>1:1 (undiluted)
<tr><td>1-3 mg/ml  <td>1:3 (1:6)
<tr><td>3-5 mg/ml  <td>1:5 (1:10)
<tr><td>5-10 mg/ml <td>1:10 (1:20)
<tr><td>10-20 mg/ml <td>1:20 (1:10, 1:40)
<tr><td>20-50 mg/ml <td>1:30 (1:20, 1:50)
<tr><td colspan="2">* Dilutions indicated in parentheses are optional
but if sample availability is not a problem, it is highly recommended they
be done
</table>

<h2 id="sutherland">Sutherland Assay</h2>
<p>
This is a pre-Lowry assay, probably a source from which Lowry et al. developed
their assay.  See Sutherland, E. W., Cori, C. F., Haynes, R., Olsen, N.S.
(1949)  <i>J. Biol. Chem.</i> <b>180</b>, 825.
<p>
The following stocks are required:
<ul>
<li>Solution A:  30 g anhydrous sodium carbonate (Na<sub>2</sub>CO<sub>3</sub>)
and 4 g sodium hydroxide (NaOH) dissolved in water to 1 liter.  This
can be stored for several months at room temperature.
<li>4% sodium potassium tartrate (Rochelle's salt).  Store in the refrigerator.
<li>2% copper sulfate (CuSO<sub>4</sub> &bull; 5 H<sub>2</sub>O.  Can
be stored at room temperature.
<li>10% sodium desoxycholate in 0.01 N NaOH.
<li>Folin-Ciocalteau (phosphomolybdate solution).  Usually purchased
prepared as a 2 Normal solution.  Store in a brown bottle away from light.  Typically stored
in the refrigerator.
</ul>

<ol>
<li><b>Reagent B</b>.  This solution must be prepared on the day of the assay. 
Multiply the total number of samples (standards, controls, unknowns) by 2.5 ml, 
and add 15% to the total.  Adjust this number to the nearest 5 ml multiple. 
This is the volume of Reagent B to prepare.  To 100 parts of Solution A, add 1 
part of the 4% NaK tartrate, then add 1 part of the 2% copper sulfate.  Suppose 
25 ml is calculated as the volume to prepare:  then take 25 ml of Solution A, 
and add 0.25 ml of the NaK tartrate, then add 0.25 ml of the copper sulfate. 
Always add the tartrate first and mix well, or a cloudiness may occur.
<li><b>Folin Reagent</b>.  Multiply the number of tubes by 0.25 ml
and add 15% for pipetting error.  Take 1 part of 2 N Folin concentrate,
and add 1 part of ultrapure water to it.  For example, suppose 3.0 ml
dilute Folin is wanted:  add 1.5 ml of Folin concentrate to 1.5 ml water.
<li>Obtain tubes that hold at least a 4 ml total volume.  Add 0.4 ml
of the standard/control/unknown to each tube.  The standard curve
reads in the range 10-100 &micro;g.
<li>Add 0.1 ml of 10% desoxycholate in NaOH.
<li>Add 2.5 ml Reagent B.  Vortex, and let the tube sit for 10 min.
<li>Add 0.25 ml Folin reagent, and let sit 30 min after mixing.
<li>Read at 750 nm.
</ol>
<p>
This assay probably scales down to a microtiter plate level.  To get
to a 500 &micro;l total volume, divide all volumes by 7, or to the 1.0
ml level, divide by 3 or 3.5.


<h2>The Pierce Bicinchoninic Acid (BCA) Assay</h2>
<p>

Developed by researchers at Pierce Chemical Co. in 1985 (<a 
href="http://www.ncbi.nlm.nih.gov/pubmed/3843705">Smith et al, <i>Anal. 
Biochem.</i> <b>150:</b> 76-85, 1985</a>), this is a variation on the biuret 
method (alkaline Cu<sup>2+</sup> reaction with amides), upon which the assay by 
Lowry et al is based.  Cupric (Cu<sup>2+</sup>) ion is reduced in the presence 
of the amide of the peptide backbond to cuprous (Cu<sup>+</sup>) ion, and the 
resulting ion chelates with bicinchoninic acid to produce a purple-colored 
complex with fewer color (negative or positive) interferences than the use of 
the Folin-Ciocalteau (phosphomolybdic-tungstic) reagent as used in the Lowry 
reaction.  The F-C reagent is also sensitive to the presence of Trp and Tyr 
residues, and not just the peptide backbone.  An advantage of the BCA assay to 
use of acidic Coomassie Brilliant Blue G-250 (Bradford) assay is that it 
interacts with proteins that do not dissolve well in the Bradford reagent.

<p>
BCA itself is produced acetoin (CH<sub>3</sub>C(=O)CHOHCH<sub>3</sub>) and
isatin (1<i>H</i>-indole-2,3-dione) using NaOH instead of KOH to produce
quinoline-4-carboxylic acid and this is recrystallized three times in minimum
amt of 75&deg; water.  It is dried at 60&deg; to produce amorphous cream-colored
powder.  A 1% salt solution in 2% sodium carbonate, 0.16% sodium
tartrate, and 0.4% NaOH is prepared as stock.  The working reagent is
a mixture of 1 part by weight of 4% copper sulfate pentahydrate with 100 parts
of BCA reagent, and the absorbance of the blank at 562 nm should be &lt; 0.065
or the BCA should be further re-crystallized. 0.1% SDS might be included to
enhance protein solubility.
<p>
<b>Assay method</b>.  2 ml of working reagent is mixed with 100 &micro;l of 
sample which can range from 0-2000 &micro;g total protein.  The reaction is 
accelerated in a 60&deg; water bath for 30 min, then allowed to cool so that 
samples can be read at 562 nm.

<h2 id="biorad-rcdc">The Bio-Rad RC DC&trade; Assay</h2>
<p>
This is basically the method of Lowry et al which contains additives to 
compensate for interferences from reducing agents and detergents.  <q>RC</q> 
means <q>reducing agent-compatible</q> and <q>DC</q> means 
<q>detergent-compatible.</q>  It is important that the protein standards used to 
produce the calibration curve are not made up in water but rather in the buffer 
used to contain and dilute the protein sample when certain reagents which can
produce interference in this assay exceed certain limits as to presence:
100 mM DTT, 2 mM tributylphosphine, 5% &beta;-mercaptoethanol, full-strength
Laemmli buffer (incl. 2-ME), 2% CHAPS, 2% Tween 20, 2% Triton X-100, 100 mM EDTA,
500 mM imidazole, 500 mM Tris pH 8.4, 2.5 M NaOH.
<p>
The RC DC assay is prepared as a kit with the two reagents and one of two standards:
either a bovine &gamma;-globulin (Cat #500-0121) or BSA (Cat #500-0122) standard.
Cat #500-0119 is a kit with only the two reagents (no protein standards), and
Cat #500-0120 contains the RC reagents and reagents for detergent compatiblity.
<p>
<b>Assay method</b>.  Bio-Rad offers two scales:  the <q>standard</q> assay protocol
has a 5 ml tube (500 in number), and the <q>microfuge tube</q> assay protocol is 
in a 1.5 ml tube (2000 in number).  Basically, all volumes used in the microfuge
tube assay are divided by 4.
<p>
Before showing the steps, some notes about working <b>Reagent A</b>.  The 
working Reagent A is a mixture of 1 volume of <b><i>DC</i>&nbsp;Reagent S</b> to 
50 volumes of <b><i>DC</i>&nbsp;Reagent A</b>.  The mixture is stable for 1 
week. Each tube in the standard assay gets 510 &micro;l Reagent A.  If Reagent A 
contains precipitate from storage, warm it and vortex to dissolve the 
precipitate completely.

<p>
The steps of the <b>standard</b> method are:
<ol>
<li><b>Bio-Rad Protein Standards</b>.  Bio-Rad sells and likely provides in the
protein assay kits either a BSA or bovine &gamma;-globulin protein, which when
reconstituted according to Bio-Rad, provides 20 ml of 1.4 mg protein/ml.
Do not re-constitute it according to Bio-Rad, since it does not leave room for
diluting the standard with a different diluent other than water for the standard
curve.  Instead, add 5.0 ml deionized water to give a four-fold higher stock
concentration (around 5-6 mg/ml).  Aliquot this out and freeze it at &minus;20&deg;
and note the concentration.
<li><b>Reagent A</b>.  Count the number of tubes to be assayed.  Multiply this 
by 0.51 ml and then add 10% (for pipetting error); this is computed by 
multiplying by 1.1 and then trying to round this number in some way.  If the 
value is less than 15 ml, use a 15 ml conical tube; if &gt; 15 ml, get a 50 ml 
conical tube. Divide the value by 51 then multiply this value by 1000; this is 
the <b>base value</b> in microliters. Add this volume in &micro;l of 
<b><i>DC</i>&nbsp;Reagent S</b> to the conical tube.  Multiply the base value by 50 
and add this volume in microliters (or divide by 1000 to get the volume in 
milliliters) of <b><i>DC</i>&nbsp;Reagent A</b> to the conical tube. [Note that the 
stock <b><i>DC</i>&nbsp;Reagent A</b> is not the same as working Reagent A, but 
working Reagent A is named so because most of its volume comes from 
<b><i>DC</i>&nbsp;Reagent A</b>.]  Vortex the mixture and label the conical tube 
<q>Reagent&nbsp;A,</q> as its remainder may be stored.
<div style="margin:0.5em 5%;font:normal 75% Arial,Tahoma,Helvetica,sans-serif;">
Consider a 10-tube assay.  The final volume of Reagent A should be 
10 &times; 0.51 ml &times; 1.1 = 5.61 ml.  Better to round this off to 6.0 ml.
Since this is less than the volume of a 15 ml conical, obtain a 15 ml conical 
tube.  The base value is 6.0 ml/51 &times; 1000 = 118 &micro;l.  Therefore add
118 &micro;l of <b><i>DC</i> Reagent S</b> to the 15 ml tube.  Add 118 &micro;l
&times; 50 = 5.9 ml of <b><i>DC</i> Reagent A</b>  to the 15 ml tube, and vortex.
<p>
Consider a 30-tube assay.  This should compute to a 16.8 ml volume.  Dividing by 
51 and multiplying by 1000, this gives a base value of 329 &micro;l (change it 
to 330 &micro;l).  Thus add 330 &micro;l <i>DC</i> Reagent S to a 50 ml tube 
(since 16.8 ml is more than a 15 ml conical), and then add 330 &micro;l &times; 
50 = 16.5 ml of <i>DC</i> Reagant A, and mix.
</div>

<li><b>Standard curve</b>.  The protein standards should vary from 200 to 1500 
&micro;g/ml.  Dilutions can be prepared in water, but this is not recommended. 
It is better to use a diluent that is the solvent for the protein sample. 
Prepare a 4-point curve (0.25, 0.50, 0.75, 1.0 &micro;g/&micro;l) or 5-point 
(4-point + 1.25 &micro;g/&micro;l) in singlicate using either water or the 
buffer containing protein as a diluent.  Label four 5 ml (12 &times; 75 mm 
tubes) conical tubes 2, 3, 4, and 5 (if a 4-point curve) or five 1.5 ml tubes 2 
through 6 (if a 5-point curve). Now add (<i>Z</i>&times;100)/<i>Y</i> &micro;l of 
the standard (at concentration <i>Y</i> &micro;g/&micro;l to tubes 2 through 5 
or 6 for <i>Z</i> = 2.5, 5.0, 7.5, and 10.0 (for 4-point curve), and an extra 
<i>Z</i> = 12.5 (for a 5-point curve).  If <i>Y</i> = 5.5 &micro;g/&micro;l, 
then tube 2 gets (2.5 &times; 100)/5.5 = 4.5 &micro;l.  Tube 3 should get double 
(9.0 &micro;l), tube 4 gets triple (13.5 &micro;l), and so on.  The final volume 
will be 100 &micro;l, so tube 2 will be (100 &minus; 4.5 = ) 95.5 &micro;l of 
water or diluent. Tubes 3 through 5 or 6 will also be adjusted with water or 
diluent to final 100 &micro;l.  Tube 1 will be the blank, and should get the 
full 100 &micro;l volume of the diluent (water or buffer).
<li><b>Samples</b>. Add 100 &micro;l of samples to 5 ml (12 &times; 75 mm) tubes.
Prepare either dilutions or use undiluted.  Unless it is known the protein
concentration is &lt; 1 mg/ml, it is best to prepare one or more dilutions, and
in duplicate in order to be have one of the dilutions to be within the range
of interpolation on the standard curve.  For most extracts of cells (where a
3 volumes of extract buffer was added to 1 vol of solid tissue mass or cell pack),
perhaps a 1/5 and 1/50 dilution set in duplicate is advisable (10-fold difference).
<li>To each tube (standards and samples), add 500 &micro;l <b><i>RC</i> Reagent 
I</b>.  Vortex, and let the tubes sit for 1 minute.
<li>Add 500 &micro;l <b><i>RC</i> Reagent II</b> to each tube, and vortex.
The tubes now need to be centrifuged at 15,000 <i>g</i> for 3-5 minutes.
<li>The supernatant is discarded completely.  The best way to do that is to 
pour gently into a waste container (beaker) and touch the lip of the tube to
container wall or rim, to decant completely.  Then carefully touch the lip to
an absorbent towel to draw out the final amounts of fluid completely by 
capillary action.  Be careful not to dislodge any pellet, which may or may
not be visible.
<li>Add 510 &micro;l of <b>Reagent A</b> to each tube and vortex.  Incubate the tube
for 5 minutes (keep at room temperature), or until the precipitate has completely
dissolved.  Vortex before going to next step.
<li>Now add 4.0 ml of <b><i>DC</i> Reagent B</b> and vortex <em>immediately</em>.
Leave at room temperature for 15 min.
<li>The solution can be read at 750 nm, and the absorbance will be stable for up
to 1 hour.
</ol>
<p>
<b>Microscale <q>Economy</q> Method</b>.  This is basically just my method to 
reduce the scale by half from Bio-Rad's microfuge tube assay, or by 8-fold from 
the standard assay.  It is still done in a 1.5 ml tube as for the microfuge tube 
assay, and can be read in a semi-micro plastic cuvet, but it increases the 
number of assays available to the kit to 4000.  The typical assay will use about 
10 assays if the standard curve in singlicate and one sample of a protein 
extract at two dilutions (in duplicate) are assayed.  Thus at least 400 assays 
of this configuration are afforded.
<ol>
<li><b>Reagent A</b>.  Count all the number of tubes to be assayed, including 
blanks and standards, and then compute: volume required = (# of tubes) &times; 
(64 &micro;l) &times; 1.1. Round to integer value.  Compute: base value =
volume required &divide; 51 (to nearest tenth of microliter).  To a suitable tube
(1.5 ml conical if volume &lt; 1500 &micro;l), add base value in microliters of
<b><i>DC</i>&nbsp;Reagent S</b>.  Now add 50 &times; base value in microliters of
<b><i>DC</i>&nbsp;Reagent A</b>.  Vortex well.
<li><b>Standards</b>.  Prepare working stocks of standards at 250, 500, 750, and 
1000 &micro;g/ml, if a 4-point curve is done, if protein is BSA.  If bovine 
&gamma;-globulin, a 5-point assay including 1250 &micro;g/ml might be desired. 
If the Bio-Rad protein standards were used, and diluted to 5 ml instead of 20 
ml, their concentration should be 5.6 mg/ml, or 5600 &micro;g/ml.  Into four or 
five 0.5 ml tubes labeled <q>250 &micro;g/ml BSA,</q> <q>500 &micro;g/ml 
BSA,</q> and so on (if BSA is the protein), pipet 6.7, 13.4, 20.1, and 26.8 
&micro;l of the 5600 &micro;g/ml stock into the tubes marked 250, 500, 750, and 
1000, respectively.  If a 1250 &micro;g/ml standard will be used, then 33.5 
&micro;l of the 5600 &micro;g/ml stock to its tube.  Compute the difference 
between 150 &micro;l and the amount added, then fill the tubes up to 150 
&micro;l with water or better, a buffer that often serves as an extraction 
buffer.  To 1.5 ml conical tubes numbered 2 through 5 (for 4 point assay) or 2 
through 6 (for 5-point assay), add 12.5 &micro;l from the 250 to 1000 or 1250 
&micro;g/ml stock, respectively. To tube 1, add 12.5 &micro;l of water or the 
diluent for the standards.
<li><b>Samples</b>.  Either add undiluted unknown protein samples or dilutions 
of it to 1.5 ml conical tubes numbered beyond 5 or 6 (depending on standard 
curve) as the unknowns.  The maximum volume is 12.5 &micro;l.  When the 
pipetting of standards, blank, and unknowns is complete, the volume should be 
12.5 &micro;l.  All unknowns should be assayed in duplicate or even triplicate
to be certain.  Multiple dilutions should be done (orders of 10-fold) to ensure
an absorbance value that interpolates within the standard curve (do not 
extrapolate!!)
<li>Pipet exactly 62.5 &micro;l of <b><i>RC</i> Reagent I</b> to each tube,
vortex, and leave for 1 minute at room temperature.
<li>Pipet exactly 62.5 &micro;l of <b><i>RC</i> Reagent II</b> to each tube,
vortex, and spin in microcentrifuge at maximum speed for 3-5 minutes.
<li>Pour out (decant) supernatant into beaker to collect this waste, and touch
rim of tube to absorbent paper to remove as much fluid as possible.
<li>Add 64 &micro;l Reagent A to each tube and vortex.  Leave 5 minutes at
room temp, or until the precipitate completely dissolves.  Vortex again before
going to next step.
<li>Add 500 &micro;l <b><i>DC</i> Reagent B</b> to each tube and vortex
<em>immediately</em>.  Incubate for 15 min.
<li>After the 15 min development period, read absorbance at 750 nm.  Color
is stable for about 1 hour.
</ol>

<h2>Computing &amp; Processing Total Protein Assay Data</h2>
<p>
This can apply to the computation of any assay data.

<h2 id="specs">Spectrophotometers</h2>

<h3>Using Cuvettes</h3>
<p>
A single disposable (polystyrene) semi-micro cuvet can be used with
many spectrophotometers when doing the Bradford assay.  One can use
a plastic (polyethylene or polypropylene) transfer pipet, which works
nicely and avoids scratching the cuvet inner surface, or one can use a
standard glass Pasteur pipet to place in and remove samples.
The use of plastic transfer pipet is <strong>absolutely required</strong>
if the cuvet is glass or quartz so as to prevent any possibility of
scratching the surface of these non-disposable expensive cuvets.
<p>
It should be possible to use cuvets without removing them from the
spectrophotometer.
<p>
When adding sample to the cuvet, place the
solution in from the top slowly.  If the tip is placed at the bottom
of the cuvet, bubbles might be introduced that distort the measurement
of absorbance.
<p>
When removing sample from the cuvet, place the tip of the pipet on one
side in a corner of the cuvet and suck it up.  Then place the pipet tip
on the other side and suck it up, if anything comes out.  There is now
a general assurance that everything has been removed.   One can add the
next sample without generally worrying about sample carryover if the next
sample is contains the higher absorbing concentration.  If the next
sample contains a lower absorbing concentration, then do a full cuvet
wash.
<p>
To do a full cuvet wash, use the blank solution or ultrapure water
and fill the cuvet nearly completely with it using the transfer pipet.
Now suck out the wash completely with the pipet, making sure to use the
solution removal technique used for removing sample.  The cleaning of
the cuvet might be done occasionally after several samples, just to make
sure it is clean.
<p>
When the assay is done, do not neglect to clean the cuvet with several
washes before putting away.  Even disposable cuvets can be re-used if
their surfaces are not scratched or altered by adhering stain.

<h3>Bio-Rad SmartSpec 3000</h3>
<p>
This is function-oriented spectrophotometer measuring from 200-700 nm
(from UV to visible).  One is guided in a stepwise manner through the
assay by inserting blanks, then standards or controls, and then unknown
samples.
<p>
A printout on thermal paper is then done to get the data.  There are
defects in this system however.  One is that the standard curve raw data
is not printed out nor can it be stored in memory.  It is much better
to connect the instrument to a computer and recover data in this way,
assuming there is software for it.

<h3>NanoDrop ND-1000</h3>
<p>
The ND-1000 can read 1 &micro;l samples, although it is customary to pipet
2 &micro;l on to the drop receptacle.  Software is set up to make
all sorts of protein assays easy.
<p>
It is better to avoid the use of the NanoDrop however.  It shows
very low reproducibility of readings of even the same sample, giving
as much or more than a 2-fold difference in final concentration with
repetitive absorbance readings.

<!--  ========================================
        Gel Electrophoresis
  ============================================  -->

<h1 id="gelelec">Gel Electrophoresis</h1>
<p>

Electrophoretic separations of proteins has evolved considerably in 100 years of 
experimental biochemistry.  Proteins, whether intact native multimeric forms as 
multiple polypeptides, or as single functional polypeptides, and their fragments 
as peptides, whether generated chemically or naturally (by enzymatic 
proteolysis) have been separated in electric fields, from high to ultrahigh 
voltages using 10,000-30,000 volts in capillary electrophoresis systems, to much 
lower voltages from maybe 50-100 to 500 volts in liquid, 
semi-solid/semi-liquid/gel media for the purpose of analysis or semi-preparative 
purification for a later analytical step.  As for gels, proteins and peptides 
have been separated in starch, gelatin, cellulose in pure and derivatized forms, 
and in synthetic gels of the acrylamide monomer (polyacrylamide).  Single phase 
and multiphasic buffer systems have been used as the conductive medium.  The 
theoretical concepts and experimental observations of electrophoresis, 
particularly multiphasic buffer systems, have been explained in excellent detail 
by <a 
href="../GeneralScienceVol1%20(to%20and%20incl%201986).html#jovin">Jovin</a> and 
others preceding and following.

<h2 id="one-d-ge">One-Dimensional Gel Electrophoresis</h2>
<p>
One-dimensional (polyacrylamide) gel electrophoresis (1D PAGE) is actually a
reference to the use of sodium dodecyl sulfate (SDS)-based separation
of (denatured) proteins by polyacrylamide gel electrophoresis.  It is the
second dimension of the two-dimensional separation described in the following
section.
<p>

Although there are many forms and variations of using detergent-based (SDS) 
separation of proteins, the classical technique was that reported by U. K. 
Laemmli who used it to separated viral proteins.  He employed a polyacryamide 
gel system with a stacking and resolving gel (these worked out by Ornstein and 
Davis a few years earlier) using a Tris-glycine buffer system.  The stacking gel 
is polymerized with a comb to form lanes at a low concentration (4%) of 
polyacrylamide.  The resolving/separating and stacking gels are prepared with a 
buffer system shown in the lists given below;  in addition an <i>electrode 
buffer</i> or <i>running buffer</i> is also used for the system.  Protein sample 
is prepared in a <i>sample buffer</i> with components whose final concentrations 
are also given below.

<ul>
<li><b>Resolving Gel Composition</b>
 <ul>
 <li>375 mM TrisHCl pH 8.8<br>
  made from 4&times; stock of 1.5 M TrisHCl pH 8.8
 <li>0.1% SDS <br>
  made from a 100&times; stock of 10% SDS or with solid SDS with casts
   of large volumes (&gt; 500 ml)
 <li>7-18% acrylamide/bis (final concentration affects MW separation range)
   <br> prepared from a stock of  a 37.5 g acrylamide + 1.0 g bisacryamide mix with a total
   acrylamide+bis total of 30% or 40% solution usually (Acrylamide is considered
   a neurotoxin that can accumulate in tissues over time!!)
 <li>0.05% ammonium persulfate
  <br>prepared as the minimal volume of 10% ammonium persulfate on the day of 
   casting
 <li>0.05% TEMED
 </ul>
<li><b>Stacking Gel Composition</b>
 <ul>
 <li>125 mM TrisHCl pH 6.8<br>
  made from 4&times; stock of 0.5 M TrisHCl pH 6.8 (note the pH difference)
 <li>0.1% SDS <br>
  made from a 100&times; stock of 10% SDS
   of large volumes (&gt; 500 ml)
 <li>4% acrylamide/bis
   <br> prepared from the 30 or 40$ (T) acrylamide/bis stock as above
 <li>0.05% ammonium persulfate
  <br>prepared as the minimal volume of 10% ammonium persulfate on the day of 
   casting (use same stock used for making resolving gel)
 <li>0.1% TEMED
 </ul>
<li><b>Electrode (<q>Running</q>) Buffer Composition</b>.  For every 1.0 L of
solution, measure out and dissolve the following (there is no need to check pH):
 <ul>
 <li>3.0 g Tris base
 <li>14.4 g glycine
 <li>1.0 g SDS
 </ul>
<li><b>(Protein) Sample Buffer</b>
 <ul>
 <li>62.5 mM Tris HCl, pH 6.8<br>
  This is stacking gel TrisHCl diluted 1/8-fold instead of 1/4-fold for stacking
  gel
 <li>0.5 to 1.0% SDS (prefer 1.0%)
 <li>10% glycerol
 <li><b>Reducing conditions</b>.  2-mercaptoethanol works well and is used at
a 1% (v/v) concentration (this is approximately 0.14 M).  If the odor cannot
be tolerated, use dithiothreitol (DTT) or dithioerythritol (DTE) at 50 mM.
 <li>protein:  final concentration should be at least 0.1 mg/ml, but best results
are at 1 mg/ml; usually depends on purity.  For extracts, 1-5 mg/ml are best,
with 10-50 &micro;g per lane in a mini-gel (also depends on staining to be done).
 <li>It is best to prepare a 4&times; Sample Buffer solution composed as follows:
  <ul>
  <li>250 mM Tris HCl pH 6.8<br>
    1:2 dilution of stacking gel 0.5 M stock
  <li>4% SDS
  <li>40% glycerol
  </ul>
  Prepare in a separate tube a small volume of 10% 2-mercaptoethanol (in water).
 </ul>
</ul>
<p>
For preparing protein sample, suppose the protein is 10 mg/ml and to be analyzed
under reducing conditions.  If 20 &micro;l is sufficient, then 2 &micro;l of
10 mg/ml protein, 5 &micro;l 4&times; sample buffer, 2 &micro;l of 10% 
2-mercaptoethanol, and the balance (11 &micro;l) with water.  The sample is then
placed in a water bath just freshly boiled (95-100&deg;) for at least 5 minutes.
Load on to the stacking gel.
<p>
All reagents <em>must</em> be electrophoresis grade or ACS certified or better.
<p>
The resolving or separating polyacrylamide gel, upon  which the stacking gel is 
poured, is made either as a single concentration gel which will separate 
different ranges of molecular weights of the proteins, or sometimes cast as 
gradient which allows a broader range of separation, although individual band 
resolution is a function of both the amount of total protein loaded and the 
actual physical dimensions (namely height) of the gel.  Its buffer system is 
375&nbsp;mM TrisHCl at pH&nbsp;8.8 containing 0.1% SDS.

<h2 id="two-d-ge">Two-Dimensional Gel Electrophoresis (2DGE)</h2>
<p>2DGE is isoelectric focusing (IEF) in the first dimension,
and SDS-PAGE in the second dimension.

<h3 id="ief">Isoelectric Focusing (IEF)</h3>
<p>
A variety of methods and systems (instruments, materials) are used
achieving isoelectric focusing of proteins.  Many pour their own tube
gels.  This laboratory utilizes strips of dehydrated polyacrylamide gels
which have been prefocused with a pH gradient whose ampholytes are
immobilized to the gel.
<p>
<b>DTT</b>.  DTT is used as a reductant that is typically include
in IEF reagents.  Proteins with all cysteines in thiol form show better
focusing.  A problem with DTT is that it is slightly acidic in the alkaline
pH range and will ionize, migrating to the anode and falling in concentration
in the alkaline range.  To compensate, some include DTT in the wick on
the cathode end.  Other approaches are to reduce the proteins in TBP
and then focus with dithiodiethanol (DTDE), which has a higher pKa.
<p>
<b>Focusing</b>.  Focusing parameters are generally 50,000-75,000 volt-hours.
Narrower range strips require longer focusing times.

<h4>IEF Equipment &amp; Consumable Checklist</h4>
<ul>
<li>IEF strip(s) of desired length and pH range
<li>Tray/container used to rehydrate IEF strips of proper length
<li>Focusing container/tray for IEF strips
<li>Samples properly clarified (no cloudiness, coagulates, precipitatible
material) in 0.5-1.5 ml conical tube
<li>Microfuge (if clarifyying reagents or samples)
<li>Strip rehydration buffer/reagent thawed and clarified with no
component degradation
<li>Pipettors from 2-1000 &micro;l (Gilson P-20, P-200, P-1000)
and clean tips
<li>Electrode filter paper wicks
<li>Mineral oil
<li>IEF electrophoretic cell:  working?  programmed?
<li>Highest quality water (in 15 or 50 ml conical tube or other)
</ul>

<h4>Isoelectric Focusing with Bio-Rad Protean IEF</h4>
<p>
This method is specific to the use of Bio-Rad products, but a few general 
principles regarding isoelectric focusing may be found.
<p>
Bio-Rad has developed several sizes and pH ranges of dehydrated polyacrylamide 
gels containing immobilized (covalently bonded) ampholines (ampholytes). 
Ampholines are essentially alkyl polymers containing different numbers of 
carboxylic acid and amino functional groups on each polymer.  During the 
manufacture, the ratio of carboxylic acid amino functional groups can be 
controlled.  Each polymer thus has a characteristic isoelectric point (neutral 
or zero charge at a certain pH). When a mixture of these polymers is made and 
placed between high voltage positive electrode resting on an paper strip dipped 
in acid (dilute phosphoric acid) and a negative electrode resting on a stripped 
dipped in base (dilute sodium hydroxide).  If the ampholines were not present, a 
pH gradient would develop between the electrodes with a pH of about 0 at anode 
(positive end) and a pH of 14 at the , the ampholines move to certain points 
along the distance of the gel and stop, because the pH matches the pI of the 
ampholines.

<h5 id="protean-ief-cell">Bio-Rad Protean<sup>&reg;</sup> IEF Cell</h5>
<p>
Before processing sample and rehydrating any IPG strips, the user should make 
sure the Bio-Rad Protean IEF Cell (Cat #165-4000) is available and working and 
is familiar with the content of the instrument's instruction manual from Bio-Rad 
(available as PDF online, possibly <a 
href="http://www.bio-rad.com/webroot/web/pdf/lsr/literature/4006164B.pdf">here</a>).
Verify that the instrument initializes and can be programmed.  It may be useful
to collect a monitoring of the voltage and current values during the run into
a computer via a serial cable setup.


<h5 id="rehydration">Rehydration of Bio-Rad
ReadyStrip Immobilized pH Gradient (IPG) Strips</h5>
<p>
This information is taken from Instruction Manual (Cat #163-2099).

<ol>
<li>use of the Rehydration/Equilibration tray for combined buffer/protein
loading (also called passive rehydration)
<li>use of the Protean IEF Focusing Tray directly for rehydration
while a current is applied (active rehydration); used for permeating
high MW proteins when this is a goal
<li>the Cup Loading method, somewhat inconvenient but probably ideal when
the strip is in the alkaline range (pH 7-10) and streaking has been a problem
</ol>

<h6>Getting Sample Ready for Strip Rehydration</h6>
<p>
Protein concentrations will need to be determined for the samples before
the strips can be rehydrated.
<p>
<b>Rehydration Buffer Composition</b>.  Bio-Rad says that IEF strip
rehydration buffers are not uniform, meaning that no single rehydration
buffer can be used for <em>all</em> samples.  The buffer used should be
optimized for each type of sample.  For instance, some samples are not
complex:  they may be partially or significantly purified, and so a
rehydration buffer might be different than for a sample with a whole
cell extract.  For samples of high complexity (i.e., lots of proteins
from a cell or tissue extract), the following rehydration buffer
should be suitable in most cases:

<ul>
<li>7 M urea [7-9.8 M]
<li>2 M thioruea [0-2 M]
<li>4% CHAPS  [1-4%]
<li>50 mM DTT [10-100 mM]
<li>0.5% (w/v) 3-10 ampholytes [0.1-0.5%]
</ul>
<p>
The value in brackets shows the concentration range that should be used
in trying to optimize the composition of strip rehydration buffer.  For
samples of lower complexity (more purified), lower concentrations of
detergents and denaturants might be acceptable, including omission of
thiourea.  For samples that contain polynucleotides (DNA, RNA), use a
higher concentration of ampholytes (do not use ampholytes in the 3-10 range for
narrow range strips!! or if you do, use very low concentrations such as
0.05-0.1%;  special ampholytes should be used for 1 pH unit-wide strips and
for the 7-10 strip).  DTT should generally be kept very high especially for
3-10 and 7-10 strips because it ionizes in the alkaline range and allows
protein streaking due to reformation of disulfide bonds.  It might be
best to treat proteins with DTT/iodoacetamide <em>before</em> IEF to
prevent streaking.
<p>
Note that if the sulfobetaine detergents (SB3-10, ASB-14) are used,
such as a 2% CHAPS/2% SB detergent, keep urea below 5 M since the detergent
is not soluble in higher urea concentrations.  Nonionic detergents might
also be used (Triton X-100, Brij, NP-40, Tween).

<ol>
<li>Put on gloves before doing anything else
<li>Sample should be prepared in a suitable rehydration buffer that
usually includes:
 <ul>
 <li>urea (to &gt; 6-7 M)
 <li>a nonionic or zwitterionic (CHAPS) detergent
 <li>carrier ampholytes
 <li>reducing agents (tributylphosphine or DTT, rarely 2-mercaptoethanol)
 </ul>
The components will depend on the sample proteins themselves.
<li>Obtain a R/E tray which is either new or cleaned, and air-dried.
<li>The final volume of sample/IEF buffer to be used in strip rehydration
is as follows:
  <table id="IPG-rehydration">
  <tr><th>Strip length (cm) <td>7 <td> 11 <td> 17 <td> 18 <td> 24
  <tr><th>Rehydration Volume (&micro;l)
                           <td>125 <td>200 <td>300 <td>315 <td>450
  <tr><th>Recommended Protein Load for Silver Stain (&micro;g)
                          <td>5-20 <td>20-50 <td>50-80 <td>50-80 <td>80-150
  <tr><th>Recommended Protein Load for Coomassie G-250 (&micro;g)
                 <td>50-100 <td>100-200 <td>200-400 <td>200-400 <td>400-800
  <tr><th>Maximum Protein Load
                   <td>500 &micro;g <td>1 mg <td>3 mg <td>3 mg <td>4 mg
  </table>
<li>Pipet the volume as a line along the edge of a channel to which the
strip will be placed.  It should extend along the whole length except
for about 1 cm from each end.  Avoid introducing bubbles, which can
affect an even distribution.
<li>When loading is complete, peel a cover sheet off a strip.
<li>Place the strip upside-down carefully.
<li>Overlay 2-3 ml of mineral oil on the strips:  add slowly by first
dripping on top of plastic backing and move the pipet along the length
of the strip at the same time.  An alternative is to wait for 60 min
before applying the oil.
<li>Cover the R/E tray with plastic lid and leave overnight on a level
bench (11-16 h) for the rehydration to complete.
</ol>

<h4 id="elecsep">Electrical Separation</h4>
<ol>
<li>In the next day (morning), obtain a clean and dry PROTEAN IEF focusing
tray and place it on the lab bench.
<li>Use clean forceps to place a paper wick over each of the ends
of the channels over the wire electrodes.
<li>Pipet 10 &micro;l of ultrapure water on the wick to wet them, and
re-position if necessary.
<li>Use forceps to recover the strips.  Hold vertically for 7-8 seconds
and touch the tip of the strip to a piece of filter paper to allow the
oil to drain and to remove excess oil.
<li><b>Optional</b>:  it has been suggested that strips be dipped in
distilled water for only 1 second (to wash away excess sample volume
outside the strip) and then blotted between moist filter paper as this
removes urea which might crystallize outside and prolong focusing and
cause <q>empty</q> lanes in 2nd dimension development.
<li>Place the strip, gel side down, within the channel of the focusing
tray, taking care to note the polarity.  Make sure the gel is in contact
with the electrode wicks.
<li>When positioned, then re-cover the strip with mineral oil again.
<li>Place the focusing tray with strip in the cell and close its cover.
<li>Program electrical operation for a default 20&deg; cell temperature,
maximum 50 &micro;A current per strip.  Press START when ready.
<li>At the end of the run, remove the strip and wash the focusing tray.
<li>Place the strip, gel side up, into a rehydration/equilibration
tray.  Be sure to remove all excess oil.  Remember not to lose the order
of the strips.  Alternatively, the strip can be stored wrapped in plastic
wrap, at &minus;70&deg; until further development.
</ol>

<p>
Focusing conditions for IEF IPG strips:

<p style="text-align:center;">
<table class="standard">
<tr><th>IPG Strip Length (cm) <th>End Voltage <th>Volt-Hours <th>Ramp <th>Temp
<tr><td>7					   <td>4000        <td>8000-10,000  <td>Rapid <td>20&deg;
<tr><td>11					   <td>8000        <td>20,000-35,000<td>Rapid <td>20&deg;
<tr><td>17 and 18          <td>10,000      <td>40,000-60,000<td>Rapid <td>20&deg;
<tr><td>24					   <td>10,000      <td>60,000-80,000<td>Rapid <td>20&deg;
</table>

<div>
<p id="show-pief-para">
It is possible to collect the run voltage data from the Protean IEF cell by connecting
it to a computer through a serial data cable using a 9-pin D-type connector to the
cell.  To show the instructions, click the button to reveal the section:
<button 
  onclick="document.getElementById('protean-ief-data').style.display='block';document.getElementById('show-pief-para').style.display='none';">Show
Protean IEF Cell data collection instructions</button></p>
<div id="protean-ief-data" 
style="display:none;border-top:2px solid blue;border-bottom:2px solid blue;padding:0.3em 0.5em;background-color:#d0d0ee;">
<p style="font:bold 105% Arial,Helvetica,sans-serif;color:purple;">
Capturing Data From Bio-Rad Protean IEF Cell To Computer
<p>
<b>Note:</b>  Once the PROTEAN IEF is running, the computer can be set up for 
data capture.  The IEF should be paused (bottom right button of IEF control 
panel) and then re-started by pressing run (upper right button).  The data from 
that point forward will be collected and can be displayed to the computer 
screen. 

<p>
The user must obtain/supply the following:

<ol style="list-style-type: lower-alpha;">
<li>DB9 male to DB9 female cable (Bio-Rad 900-0667)
<li>DB9 male to DB9 female null modem adapter (e.g., Radio Shack #26-264B)
<li>Windows 95, 98, 2000, NT or XP PC or compatible system
</ol>

<ol>
<li>Connect male end of the DB9 cable to the RS 232 serial port on the PROTEAN 
IEF cell.
<li>Connect the male end of the null modem adapter to the female end of the DB9 
cable.
<li>Connect the female end of the null modem adapter to the appropriate COM port 
on the computer.  Note which COM port is used (e.g., COM1).
<li>Prepare the data capture program as follows:
 <ul>
 <li>From the Windows 95/98/2000/NT Start menu, select Programs &rarr; Accessories 
&rarr; Communications &rarr; HyperTerminal
 <li>Double click on the Hyper Terminal icon.
 <li>Enter file name for IEF data capture program (i.e., <q>iefcapt</q>) and 
click OK.
 <li>Click "File" and "Properties"
 <li>Select the COM port to which the PROTEAN IEF cell is connected (e.g., Direct to 
COM1). Click OK
 <li>Click "Configure"
 <li>Select 9600 bits per second, "None" for parity, "1" for stop bits, 
"Hardware" for flow control, and then click OK. 
 <li>The program window is displayed and data can be collected from the Protean 
IEF cell.
 </ul>
<li>Program a run on the PROTEAN IEF cell and then start the run.  Immediately 
after starting the run, data will accumulate in the program window. Note that 
the data is in text-only format.  Example data is presented below.

<li>To save the data, select EDIT, select All, select Copy. Open the Notepad 
program (or text editor, and then select Paste. Save the text file by selecting 
File, then Save As,  and enter file name and folder/location.  Click OK.  The 
PROTEAN IEF cell data is readily imported into an Excel spread sheet for 
analysis.
</ol>
<p>
Example Data

<pre style="font-size:83%;">
	Bio-Rad Laboratories
	Protean IEF Cell
	Firmware Version: 1.80
	
	Method "RAPID     "
	Rehydration: Inactive
	Run Temp: 20 deg C.
	Number of Gels: 2   Max uA/Gel: 50
	
	Step 01    250V   Ramp: R    Time: 00:15
	Step 02  10000V   Ramp: R    Time: 03:00
	Step 03  10000V   Ramp: R   Vhour: 60000
	Step 04    500V   HOLD
	End of method list.
	
	Start of run data.
	00:00  1/   50V R   17/   8uA 00000 Vhrs
	00:05  1/  250V R   36/  18uA 00020 Vhrs
	00:10  1/  250V R   24/  12uA 00041 Vhrs
	00:15  2/  853V R   61/  30uA 00062 Vhrs
	00:20  2/ 2054V R  100/  50uA 00197 Vhrs
	00:25  2/ 2818V R  100/  50uA 00400 Vhrs
	00:30  2/ 3430V R  100/  50uA 00657 Vhrs
	00:35  2/ 3764V R  100/  50uA 00951 Vhrs
	00:40  2/ 4098V R  100/  50uA 01273 Vhrs
	00:45  2/ 4372V R  100/  50uA 01621 Vhrs
	00:50  2/ 4864V R  100/  50uA 01997 Vhrs
	00:55  2/ 5069V R  100/  50uA 02407 Vhrs
	01:00  2/ 5555V R  100/  50uA 02844 Vhrs
	01:05  2/ 6146V R  100/  50uA 03309 Vhrs
	01:10  2/ 6461V R  100/  50uA 03824 Vhrs
	01:15  2/10000V R   85/  42uA 04504 Vhrs
	01:20  2/10000V R   24/  12uA 05336 Vhrs
</pre>
</div>
</div>

<h4>IEF Commercial Products</h4>
<p>
Sigma-Aldrich sells ProteoGel IPG strip in many pH ranges and strip sizes:

<table>
<caption>Sigma-Alrdrich ProteoGel IPG Strip Catalog Numbers</caption>
<tr><th rowspan="2">pH range
 <th colspan="3">Sizes
<tr>                <th>7 cm   <th>11 cm     <th>18 cm
<tr><td>pH 3-10     <td>I2531  <td>I3406     <td>I4030
<tr><td>pH 4-7      <td>I2906  <td>I3531     <td>I4156
<tr><td>pH 3-5      <td>I3031  <td>I3656     <td>I4281
<tr><td>pH 5-8      <td>I3156  <td>I3781     <td>I4406
<tr><td>pH 6-11     <td>I7406  <td>I7531     <td>I7656
<tr><td>pH 8-12     <td>I3281  <td>I3906     <td>I4531
</table>

<p>
Amersham-Biosciences (subsidiary of GE Healthcare) sells the
Immobiline&trade; DryStrips in different pH ranges and strip lengths.
These can be run on an Ettan&trade; IPGphor&trade; II IEF system,
with Ettan IPGphor manifold.  They also have a Multiphor&trade; II IEF system.


<h3 id="SDSPAGE">The Second Dimension:  Sodium Dodecyl Sulfate-Polyacrylamide
Gel Electrophoresis (SDS-PAGE)</h3>
<p>
<b>First dimension gel preparation</b>.  The first dimension gel is
made ready for the second dimension by soaking it at room temperature
for 3 minutes in something like the following:
<ul>
<li>62.5 mM Tris HCl, pH 6.8
<li>2.3% SDS
<li>5% 2-mercaptoethanol or 0.1 M DTT (sometimes 2% DTT used)
<li>trace of bromophenol blue for tracking
</ul>
<p>
Gels can be reductively alkylated too.  Soak first for 6 min at room
temperature with the following:
<ul>
<li>5 M urea
<li>50 mM Tris HCl, pH 6.8
<li>2.3% SDS
<li>20-50 mM DTT
<li>5% glycerol
<li>trace of bromophenol blue for tracking
</ul>
<p>
Then place in a solution as above except that DTT is substituted with
40-100 mM iodoacetamide.  Soak for 12 min in the dark.
<p>
Gel strips can then be embedded on top of the 2nd dimension gel in
1% agarose containing 5 M urea, 50 mM Tris pH 6.8, 2.3% SDS, 20-50 mM DTT,
5% glycerol, and trace of bromophenol blue.

<h4>2<sup>nd</sup> SDS-PAGE with Bio-Rad Protean System</h4>
<p>
Before equilibrating the IPG strips, prepare pre-cast gels or cast
the gels you will use for placing the strip.  Polymerization should be
complete and the gels all ready.

<h5 id="stripprep">Preparing IEF IPG Strips for 2nd Dimension
SDS-PAGE</h5>
<p>
The strip must be equilibrated in a two-step buffer.
The first step is in <b>SDS-PAGE Equilibration Buffer I</b>,
which contains the following:
<ul>
<li>6 M urea: maintains protein in random coil (denatured) state
<li>375 mM Tris-HCl, pH 8.8, the buffer (type and concentration)
used in the separating gel
<li>2% SDS:  anionic detergent that denatures proteins and binds to provide
the constant mass-to-charge ratio necessary for molecular size-based
separations (20 times the concentration of that normally
present at 0.1% SDS within the gel itself)
<li>20% glycerol: increases solution density to keep proteins in solution
in a stable zone resting on the gel
<li>2% (w/v) DTT:  a reducing agent to break disulfide bonds (with urea
it helps to destroy the secondary and tertiary structure of a protein
to expose its polypeptide length and make SDS binding consistent)
</ul>
<p>
The second step is with <b>SDS-PAGE Equilibration Buffer II</b> which
contains everything that Buffer I contains except for DTT, and it also
contains iodoacetamide, which alkylates the sulfhydryls of cysteine
that were formed by DTT action during disulfide bond-breaking; the alkylation
prevents reformation of the disulfide bridges.
<ol>
<li>Take out from the freezer and thaw the required volumes of
SDS-PAGE Equilibration Buffers
I and II, and make sure they are thawed or close to being thawed before
you thaw out the IPG strips in the next step.  Avoid too many freeze-thaw
cycles with these products, or leaving especially SDS-PAGE Equilibration
Buffer II unfrozen for too long, as iodoacetamide is sensitive to hydrolysis.
See the <a href="#eqbufs">instructions</a> for preparing these reagents.
<li>If IPG strips were stored at &minus;70&deg;, take them out and let
them come to room temperature for about 15-20 min, but do not leave them
out longer, since diffusion of protein bands may start and result in
reduced sharpness of the spots.  The strips should have had excess
mineral oil removed from them before they were frozen.  The strips
can be thawed out in each lane/channel of the equilibration/rehydration
tray you will use to soak the strips in SDS-PAGE equilibration buffers,
but do not add the buffers.  Place the strip with the <em>gel side UP</em>
in the tray channel.  The correct tray for the strip length
<em>must</em> be used.  Do <b>NOT</b> use a 7 cm in an 11 or 17 cm
tray, or an 11 cm strip in a 17 cm tray.
<li>If the strip is not in the tray channel, put it in <em>gel side UP</em>.
Add Equilibration Buffer I to the strips.  The buffer and the strip
should be at room temperature.  Use the table below for the volume
of Equilibration Buffer I to be added to channel of the tray containing
the strip:

<p style="text-align:center;">
<table class="standard">
<col><col style="width:8em;">
<tr><th>Strip length (cm) <th>Equilibration Buffer volume (ml)
<tr><td>7                 <td>2.5
<tr><td>11                <td>4
<tr><td>17                <td>6
<tr><td>18                <td>6
<tr><td>24                <td>8
</table>

<li>The strip must be incubated at least 10 min on a shaker or rocker.
Adjust the shaker so that it has enough rotation without the solution
spilling out from the tray channel.  Some do it for 15 min.
If the equilibration buffer was not at room temperature, then extend the
time.

<li>Pour out the solution into the sink.  Add SDS-PAGE
Equilibration Buffer II in the same volume to the strip in the tray
channel and incubate it for the same time doing rocking or shaking in
the same way.

</ol>

<h6 id="eqbufs">Preparing Bio-Rad SDS-PAGE Equilibration
Buffers I and II</h6>
<p id="eqbuf1">
To prepare 100 ml of the reagent which Bio-Rad
calls SDS Equilibration Buffer I (Cat #163-2107), do the following:
<ol>
<li>Get a clean 100 ml beaker.
<li><a href="#weighing">Weigh out</a> 2.0 g of
dithiothreitol (DTT) into the beaker (or add it to the beaker).
<li><a href="#weighing">Weigh out</a> 2.0 g of
sodium dodecyl sulfate (SDS) into the beaker (or add it to the beaker).
<li><a href="#weighing">Weigh out</a> 36.0 g of
urea into the beaker (or add it to the beaker).
<li>Measure out 20.0 ml of glycerol into the beaker.  If you use a
graduate cylinder (use a small one) to do it, you can wash the
glycerol that does not come out of the cylinder with the reagent used
in the next step.  (You can alternatively pipet out 5.0 ml of glycerol
four times with a P5000 pipet.)
<li>Measure out 25.0 ml of 1.5 M Tris HCl pH 8.8 (this is resolving
gel buffer for SDS gels).  Add it to the beaker.
<li>Put in a clean teflon (PTFE)-coated stir bar of
the <a href="#stirbar">correct size</a> and
try to begin stirring.
<li>It might help to add water (ultrapure, Type I, or the highest
grade available) to the beaker so that it is at least 80-90% of the final
volume (i.e., 80-90 ml).  Continue stirring.  Urea has a negative
enthalpy of mixing and so the solution will become very cold during
the dissolution of urea-containing solutions.  Do not heat the solution.
Let it sit 15-20 minutes and eventually the urea will dissolve along
with everything else.  A clear solution should be seen.
<li>Get a 100 ml graduated cylinder and transfer the solution to it.
Do not allow the stir bar to be transferred;  this can be avoided by
pouring the beaker near the edge of the cylinder so that the stir
bar is blocked from falling in, and when it is all pour out, the stir
bar can be poured out from the beaker into another container without
even touching it.
<li>Add a little water to the beaker and swirl it to wash in the material
that did not transfer.  Add it to the graduated cylinder, but it should
not be more than the final volume.  The more times you wash in water to the
beaker and add it to the cylinder, the more you successfully transfer
the solution mixture to the cylinder.
<li>Add highest grade water to the 100.0 ml mark.
<li>Transfer the solution to a storage container.  One of the old
Bio-Rad container vials that it uses to sell this buffer is convenient.
<li>Now pour this back into the cylinder.  Pouring the solution from
the cylinder to the container and back again allows you to mix the solution
adequately without using a stir bar.
</ol>
<p>
The equilibration buffer should be divided (aliquotted) into different
containers if the entire volume of it is likely to be subjected to
several freeze-thaw cycles.  At most the solution should undergo
only 3 freeze-thaw cycles, and it is best that it does not do that.
It is often better to divide it into aliquots (e.g., five 20 ml
aliquots) and then just thaw more of the aliquots when bigger volumes
are required in certain experiments.
<p>
Freeze at &minus;20&deg; (or even &minus;80&deg;) for not more than
six months.
<p>
To prepare 100 ml of the reagent which
Bio-Rad calls SDS Equilibration Buffer II (Cat #163-2108),
follow <a href="#eqbuf1">all the steps</a> used to prepare
SDS Equilibration Buffer I except for the third step where you weigh
out 2.0 g of DTT, do not use DTT, but weigh out 2.5 g of iodoacetamide
instead and add it to the beaker.
<p>
It is especially vital to avoid freeze-thaw cycles with SDS-PAGE
Equilibration Buffer II (the one that contains iodoacetamide) or
to store it too long, even though it is in the freezer (&minus;20&deg;
or lower).




<h5>Mounting the IPG Strips On To The 2<sup>nd</sup> Dimension Gel</h5>
<p>
The gel should have a stack on top.  The stacking gel polyacrylamide
concentration is 4% T, and should have an effective stacking height of 5%
of the resolving gel height, but at least a minimum of 3 millimeters
(for mini-gels).  When pouring the stacking gel, use combs that give at least
2 mm each side play on each side of the strip to be mounted, i.e., the
comb edge length should be 4 mm longer than the strip length covered by
the attached IPG gel.  The comb should also include a well for loading
markers.  (Some manufacturers inexplicably do not produce combs with one
tooth for MW markers, and the thinking around here is, why would someone
want to run a 2-D gel without any markers??)
<!--
Weighing Out Solid Reagents

I like to weigh or measure out reagents that will be used to make solution
mixtures directly into container that will be used to prepare or make
the solution mixture.  I also use the lid of the reagent as the
<q>weighing paper</q> if it is convenient and not contaminated
(which it should not be).  That is, for solid reagents, I rarely use
weighing (glassine) paper, but this is just because I think that there
is some loss or possible contamination the more a reagent comes in contact
with material that it is not normally in contact with the reagent.
However, this is just an eccentricity and others might disagree with this
approach.
<p>
The important point is that whatever you use to weigh out the reagent,
it should allow a transfer of unadulterated (clean) reagent to your
mixing or final container used to hold the solution mixture.

-->

<h4>The Ettan DALT<i>twelve</i> System</h4>
<p>
[from the user manual by Amersham Biosciences (now GE HealthCare Life Sciences]
<p>
Components of the system are:
<ul>
<li>12-slot vertical slab separation unit
<li>200 W, 600 V, 1 A power supply/control unit
<li>gel caster<br>
 holds up to fourteen 1.0 mm or thirteen 1.5 mm gel cassettes, using
 separator sheets
<li>gel casting cassettes
<li>gradient maker with peristaltic pump
<li>no-gel cassette inserts
</ul>
<p>
Necessary consumables:
<ul>
<li>separator sheets
</ul>
<p>
Can hold up to 12 slab gels (27 &times; 22 cm size, producing gels of ~25.5 
&times; ~20 cm size) for identical condition separation.  The 1.0 ml gel 
requires ~52 ml volume, and the 1.5 mm gel has ~78 ml volume.  Just as done in 
other systems, the IPG strip is placed on top of gel, sealed with agarose and 
then run.  A live circuit is not allowed when the lid is off and the pump valve 
is not in the circulate position.  The glass gel cassettes are constructed and 
used in exactly the same way as Bio-Rad glass gel cassettes.
<p>
The buffer volume required for the tank is 10 L.

<h3 id="starter">Bio-Rad's 2D Starter Kit</h3>
<p>
Bio-Rad makes kit called the <b>ReadyPrep&trade; 2-D Starter Kit</b>
(Cat #163-2105) that can be used for starting out using Bio-Rad
equipment or for controlling or validating a system.  This
kit supplies a protein sample and enough reagents to do
six 17 cm, ten 11 cm, or sixteen 7 cm 2-D gels.  Not included
are the IPG strips, gels, or stains.
<p>
The kit contains the following components, which can also be ordered
separately (catalog nums given)
<ul>
<li><i>E. coli</i> extract, lyophilized, 2.7 mg (#163-2110)
<li>IPG strip rehydration/sample buffer:  8 M urea, 2% CHAPS, 50 mM DTT
0.2% Bio-Lyte 3/10 ampholyte, 0.001% bromophenol blue (makes 10 ml)
(#163-2106)
<li>Equilibration Buffer I (DTT):  0.375 M TrisHCl, pH 8.8, 6 M urea,
2% SDS, 2% DTT (makes 20 ml) (#163-2107)
</ul>

<h3 id="two-d-trouble">Troubleshooting 2D Gels</h3>
<p>
Manufacturers of 2-D gel systems and products generally offer at their
web sites information for dealing with problems and offering solutions
regarding 2-D gels.
<p>
<b>Electrical (Physical) Properties of The Focusing Strip or Tube Gel</b>.
The electric fields for
focusing on strips or tube gels is somewhere between 400-700 V/cm.
This is a final focusing voltage.  The field allowed in a focusing
system is limited by the current (measured in &micro;A or microamperes).
The reason is the power that is allowed across a strip or tube gel.
Recall from physics that <i>V</i> = <i>I</i> &times; <i>R</i> (known as
Ohm's Law), where <i>V</i> is the voltage or <q>electromotive force</q>
across a system, <i>I</i> is the current or moving charge, and
<i>R</i> is the resistance in the system (which can be a <em>load</em>
on the system as well).  This applies to systems in which there is
direct current (DC), which applies to focusing systems.
There is a certain amount of <q>power</q> generated too, which shows
itself and dissipates as <em>heat</em>, which is why the system is
cooled.  If the heat is not drawn away, it raises the temperature of
the system, and the raised temperature can cause problems, including
evaporation of the water and volatile elements in the system (yes, it
can get that hot).  The power in an electrical circuit is
<i>P = V</i> &times; <i>I</i>, and when <i>V</i> is substituted using
Ohm&rsquo;s Law, <i>P = I</i> <sup>2</sup><i>R</i>.  Note that the units
of <i>P</i> (power) is measured as a magnitude of <b>watts</b>, <i>I</i> (current)
is measured as a magnitude of <b>amperes</b>, and <i>R</i> (resistance) is
measured as a magnitude of <b>ohms</b>.  To limit the power, and thus the
heat generated during focusing, limits are placed on the current <i>I</i>,
since resistance <i>R</i> is a property of the gel and its constituents,
and the user has more direct control over the current <i>I</i> than
over the constituents of the gel.  Manufacturers determine a constant
maximum <i>P</i> value for their strips, and then
use the <i>P = V</i> &times; <i>I</i> equation to adjust <i>V</i> and
measure <i>I</i> so that the product does not exceed the maximum <i>P</i>.
If the maximum power <i>P</i> were to be exceeded, it could cause
regional increases in the temperature of a gel, causing water vaporization
and local drying.  In fact when there is regional drying of the strips,
the resistance <i>R</i> in that part of the gel as well as the resistance
in the strip overall increases, which also increases <i>P</i> according
to the equation, leading to even more heat generation.  The cycle of
heating and drying increases until the gel even burns, showing up as
browning or blackening of the gel.

<p>
<b>Failure to Reach Expected Voltages</b>.  In many cases the focusing
voltage should reach the 400-700 V/cm range.  For its 7 cm strips,
Bio-Rad has set a 4000 V final voltage, about 600 V/cm;  for its
17 cm strips, the final voltage is 10,000 V, also close to 600 V/cm.
Bio-Rad notes that for all strips at all pH ranges, it is possible that
the final target voltage is not reached during the ramp-up to voltage
step.  However, during most of the focusing step (after the ramp-up step),
the voltage should be at least greater than half (50%) of the final voltage.
So for 7 cm strips, it should be &gt; 2000 V during focusing, and for
17 cm strips, it should be &gt; 6000 V.  Bio-Rad states that the combination
of volts and hours (the volt-hours) is sufficient to bring about the
focusing of proteins.
<p>
When voltages do not even come close to that, indeed they are not more
than 200 V during the run, this means that the sample has too much
salt or conductive low MW ions in it.  See the section on salts in the
sample.

<p>
<b>Electroendo-osmosis</b>.  Some samples tend to cause changes in the
strip gel where swelling occurs in the gels that particularly make
contact with the electrodes, and in the middle
of the gel, the depth of the gel has narrowed practically down to the
strip (as if the gel has dried).  This is in fact what has happened.
Substances migrate to different parts of the gel, and often these
substances have a <i>hydration sphere</i> around them which varies as
to the number of water molecules that surround them.  High MW biomolecules
like DNA, RNA and polysaccharides with a net charge (mucins, dextrans)
carry lots of water molecules around them per molecule, and as they move
to one electrode or the other, they carry these water molecules with them,
virtually drying out the strip in the middle, and swelling it at the
electrode ends.
<p>
If this is observed, then the sample preparation technique needs to be
modified to remove high MW ionic components.  If DNA and RNA are thought
to be the problem, the sample should be treated with DNase to break down
DNA and RNase to break down RNA (pH 9 and above buffers will also break
down RNA, if that is an option).  Alternatively, add 3-10 ampholytes (if
an option) since that can precipitate polynucleotides.  To remove
charged polysaccharides, precipitation techniques and organic phase
extractions often work.


<h4>Salts in the Sample</h4>
<p>
The most important thing to be done in preparing any sample for focusing
is to remove the salts (to de-salt).  In many electrophoretic methods,
salts are intentionally added to carry currents or act in phasic ways
to stack or resolve components.  But with isoelectric focusing, the
ideal is for proteins to carry the current and to see the current drop
to an idea zero (which is never reached in practice).  So adding salts
defeats the purpose of focusing.
<p>
Desalting is achieved in standard ways, including dialysis, (centrifugal)
ultrafiltration, acid or acetone precipitation techniques.  See the section
on sample preparation for more information.
<p>
If you have already loaded a sample heavy with salts into the gel, it is
still possible to get good results (good focusing).
<p>
How do you know when salts are the problem?
<p>
Focusing is done in stages.  With the Bio-Rad method, the first 15 minutes
is spent at a low voltage trying to move salts out, for it is assumed that
even the most salt-free of samples has enough low MW electrolytes to limit
the current at the start-up.
<p>
Then there is the ramp-up to the final focusing voltage, which is thought
to take 2-3 hours.  It is at this stage you should see the voltage
ascending, to at least greater than half the final set voltage.
<p>
Suppose the final voltage is 10,000 V.  After 30 minutes into the ramp-up
period (assuming an exponentially rapid ramp-up), you should expect to
see the voltage between 2000-4000 V.  After an hour, you might see it
between 3000-8000 V, perhaps even close to the final.  If you still see
50-200 V after 30-60 minutes, you most likely have significant amounts
of salts.
<ol>
<li>Stop the focusing program.
<li>Setup or run a "de-salting" program to run at 50-100 V (usually)
for a long period (if possible, do not limit it with a time period).
Note the current at the start of the de-salting program.  It might read
&gt;10, 20, or even 50 &micro;A per strip, which is certain proof of a high
amount of salts.  De-salting will usually require a period as long as
overnight.
<li>At the end of the long period, note the current.  It should be reading
0 &micro;A or close to it.  If not, set up another clean and dry
focusing tray with  fresh electrode paper wicks, lift the strips out
of the tray they are in and let the excess oil and fluid drain from
them (touch one end of the strip to a towel to allow capillary siphoning
of the fluid), and then transfer them to the new tray.  Re-start the
de-salting.
<li>When the current drops to zero or near-zero, re-start the focusing
program.
</ol>
<p>
In more than half the cases, you can get decent focusing results.

<p id="rehydbufquality">
<b>Rehydration Buffer Quality</b>.  Recall that salts cannot be added
to the focusing system solution (gel rehydration solution).  That means
that the components of the rehydration buffer are usually either nonionic
or must have a net charge of zero if parts of them are ionized, as
zwitterions are.  CHAPS and other detergents added are zwitterions,
and are molecularly stable over long periods.  Urea (and the same is
true of thiourea) is not entirely stable however.  At temperatures
not much higher than room temperature (above 30&deg;), urea can actually
hydrolyze into ionic constituents.  The hydrolyzed products have lost
their denaturing properties, and worse, they contribute to the unwanted
presence of salts during focusing.
<p>
When two strips with different samples
<p>
To preserve rehydration buffer quality:
<ol>
<li>If rehydration buffer is to be pre-made, always store it frozen
at &minus;20&deg; and limit the time that stock is present thawed
at room temperature.  Use what you need, and quickly re-freeze the stock.
<li>Aliquot the rehydration buffer to limit the number of freeze-thaw
cycles to 3-4 times at the most.  For example, 7 cm strips take 125 &micro;l
of buffer per strip, so it makes sense to aliquot buffer to 1.5 ml tubes
in amounts between 500-1000 &micro;l.  17 cm strips take 300 &micro;l
per strip, and so 1.5 or 2 ml conical tubes can be filled with 1000-1500
&micro;l per tube.  If doing many strips at once, get several aliquots,
combine them together, and use the pool, even if they were aliquotted
from a single pool lot preparation.
<li>When incubating the protein sample with rehydration buffer, put it
in a water bath set at 25-30&deg; and leave it for no more than 10 minutes.
This ensures denaturaton and that the temperature does not exceed one
where incubation is for a long time above a point that could cause urea
to hydrolyze.  If working in warm weather where room temperature is
above 30&deg;, then keep the times for working with rehydration buffer
to a minumum when denaturing sample and getting it on the strip.
Find a cool room in which to do the overnight strip rehydration, or
use the focusing tray which allows for active or passive rehydration
and the setting of the temperature of the cooling plate.
</ol>

<h4 id="horizstreaking">Horizontal Streaking</h4>
<p>
Horizontal streaking is the usual in 2D gels, and a sign of a problem
occurring during IEF.
<p>
When the streaking is seen in the alklaline (pH 8-10) side of broad range
strips (pH 3-10) or in medium-range alkaline strips like pH 7-10, this
is caused by DTT ionizing and failing to keep protein thiols from reforming
disulfide bond with other polypeptides, resulting in aggregates that
precipitate in the gel.  During focusing, portions of aggregates go back into
solution, resulting in a streaking affect as the re-dissolved portions
attempt to migrate to their isoelectric point.
<p>
There are two ways to compensate for this effect.
<p>
<b>Finding reducing agents (thiols) that keep working in the alkaline
range</b>.  A product on the market called <q>Destreak</q> is
2-hydroxyethyl disulfide (HED).  This product apparently becomes a thiol
that does not easily ionize in the
alkaline range and remains reactive to preventing polypeptides from
reforming disulfides that lead to aggregates.
This reducing agent should be used in rehydration solutions that contain
NO other sulfyhydrhyl reducing agents
 should be present, according to
GE Life Sciences, the makers of DeStreak.
<p>
<b>Modifying cysteines in polypeptides so they do not form intermolecular
polypeptide aggregates</b>.  Cysteine thiols are typically modified
<em>after</em> gel electrophoresis prior to MALDI MS work, so why not
do it <em>before</em> gel electrophoresis if this is creating a problem.
And so proteins to be separated by 2DGE are treated with a reducing
agent (DTT, or the very effective phosphines like TBP or TCEP) and then
alkylated, typically with iodoacetamide.
<p>
Reductive alkylation of polypeptides is not routinely done, although it
probably should be.  It is done only when horizontal streaking is seen
to be a problem that is not controllable by other measures.

<p>
<b>Protein Overloading</b>.  With minor overloading of the protein,
streaking along both sides of the spot in the horizontal dimension
is likely to be seen.  This streaking could extend along the entire
length of the gel, but more likely it will be confined to a small length
of the gel on either side of the spot.  This kind of streaking is
unavoidable if the user intentionally overloads the strip in which
there are high abundance proteins and the user wants to see low abundance
proteins;  in some cases the user can by kits or materials that deplete
high abundance proteins.

<p>
<b>Rehydration Buffer Quality</b>.  The issue of rehydration buffer quality
is discussed <a href="#rehydbufquality">elsewhere</a>.

<h4 id="verticstreaking">Vertical Streaking</h4>
<p>
Vertical streaking is not as common as horizontal streaking but is still
seen.  The issues related to it include a rigid adherence to the method
for preparing the strips for the 2nd dimension as well as maintaining
the quality of the solutions used to prepare focused proteins for the
2nd dimension as well as rehydration buffer quality.
<p>
It is not unusual for proteins to reach their isoelectric points and
to want to precipitate.  This may be due being forced into proximity
to each other in the focusing field.  Urea, thiourea, detergents,
and the reducing agents are there to make sure they stay in solution;
if they are not present in the expected amounts, then the likelihood
of precipitation increases.
<p>
Even if precipitates form in the IEF strip, the solutions added to prepare
strips for 2nd dimension can typically re-solubilize them, since one
of them is SDS.  Bio-Rad's method is to incubate strips in a solution
with DTT and then in iodacetamide (although if it was already done
prior to focusing, it really should not be necessary again).  Bio-Rad
specifies 10 minutes, but the minimum time for each step should be 15
min.  It is important to get good shaking or rocking to ensure the strip
diffuses in (this is a weakness seen here).  Something that also might
help is to soak the equilibration solutions (except any solutions
containing iodoacetamide) in a warm water bath of as much as 50&deg;.
The higher temperature can help unfold and solubilize the proteins.
Putting the tray (and the shaker) in at a 50 &deg; (or optimized
temperature) oven during the DTT step may help significantly in preventing
vertical streaking.  It is possible that urea might hydrolyze even for
the limited time, but urea has no importance for the 2nd dimension step,
as SDS assumes the role of denaturing the protein now.

<p>
<b>Freezing/Thawing Strips</b>.  In most, if not all, cases, IEF strips
are not immediately run on the 2nd dimension but frozen at &minus;80&deg;
until a later time when SDS-PAGE is done.  If vertical streaking is
a persistent problem, consider altering the practice of freezing strips
(at least for this kind of sample) and either run the strips immediately
on the 2nd dimension when focusing is done or, if not practical,
maintaining the strips in the focusing equipment with a <q>holding</q>
voltage for at most 12 hours (or overnight) until the strips can be
put on the 2nd dimension.  If you notice that the vertical streaking
problem disappears with all strips, then  it could be related to freezing
of the strips.

<h2 id="staining">Illuminating The Proteins in Gels:  Gel Staining</h2>
<p>
This section deals with general or total (nonspecific classes) protein staining.
<p>
For the specific detection of phosphoproteins and glycoproteins, go to the
<a href="#phosphoproteomics">Phosphoproteomics</a> and
<a href="#glycoproteomics">Glycoproteomics</a> sections, respectively.

<h3>Fixation</h3>
<p>
A variety of acids and alcohol mixtures have been used to fix the proteins
into the gel.
<p>
For gels in which one wants to capture small proteins and peptides that
leak out of the gel, it may be useful to use formaldehyde-alcohol fixation.
Some cautions are warranted with this type of fixation.  Silver diammine-based
staining is not affected, but acidic silver nitrate staining is markedly
affected by HCHO-alcohol fixations.  Generally with small and basic polypeptides
where HCHO-alcohol fixation is required, silver staining should be with a
silver diammine based method.


<h3 id="coomassie">Coomassie Staining and De-Staining</h3>
<p>
The Coomassie dyes were originally developed as dyes for wool, which
is a protein, of course.  It was this fact that was on the minds of
scientists when they were looking for chemicals that might be useful
in the assay/detection of proteins.

<h4 id="colloidalg">Colloidal Coomassie Brilliant Blue G-250</h4>

<h5>Bio-Rad BioSafe Coomassie G Stain</h5>
<p>
Bio-Rad sells a <q>Bio-Safe</q> Coomassie G-250 stain
(Cat #161-0786 for 1 L, Cat #161-0787 for 5 L) that is the colloidal stain
derived from the Neuhoff stain.
This is a fairly rapid that supposedly can be developed within 1 h after
first washing the gels thoroughly with water to remove SDS.
<p>
For mini-gels Bio-Rad recommends washing with shaking each gel three
times for 5 min each with 200 ml deionized distilled water.  Thoroughly
remove all water from the dish and add (about 50 ml) enough stain to cover
the gel.  Gently shake for 1 h, even though bands begin to appear within
20 min.  The gel is then rinsed with 200 ml dd water for at least 30 min
to reduce background, which should not be significant.  Store gels in
water.  Increase the volume for larger gels.

<h5>Neuhoff</h5>
<p>
This is the original formulation from Neuhoff <i>et al.</i> (1988)
<i>Electrophoresis</i> <b>9</b>: 255-262, from which all other
formulations are essentially derived.  This group produced a micellar
(colloidal) formulation prepared in sulfosalicylic acid and then later
in phosphoric acid.  The staining requires at least an overnight incubation.
<p>
Gels are incubated in 2% phosphoric acid, 10% ammonium sulfate, and
20% methanol for at least 1 hour.  The Coomassie G-250 is added to a final
0.1%, and staining is allowed to continue overnight, and even a couple of days.
<p>
The following recipe is a slight modification of the Neuhoff stain.
<ol>
<li>Put a very clean stir bar into a very clean beaker and add 768 ml distilled water.
Begin stirring.  Alternatively mixing can be done an orbital rotary
shaker with gentle setting.
<li>Add 16 ml (ortho)phosphoric acid.
<li>Add 80 g ammonium sulfate and let it completely dissolve.
<li>Add 16 ml of a 5% solution of Coomassie Brilliant Blue G-250 (prepared
in water).
<li>Just before use, add 200 ml methanol.  The final concentrations of
components in this solution are 0.08% CBB G-250, 1.6% phosphoric acid,
8% ammonium sulfate, 20% methanol.
</ol>

<h5><q>Blue-Silver</q></h5>
<p>
This formulation is from Candiano <i>et al.</i> (2004) <i>Electrophoresis</i>
<b>25</b>, 1327-1333, and is yet another modified Neuhoff formulation.
To a water solution representing 10% of the final volume, add phosphoric
acid such that it is 10% at its final volume, then ammonium sulfate as
a powder is added so that its final concentration will be 10%.  After the
ammonium sulfate has dissolved, Coomassie Brilliant Blue G-250 is finally
added as a powder to 0.12%.  When all solids are dissolved, water is
brought to 80% of the final volume and then anhydrous methanol is added.
The stock dye is kept in a brown bottle and stable for &gt;6 months.
<p>
The following recipe is reiterated in absolute amounts (volumes and masses)
to make 1 liter of the colloidal Coomassie G:
<ol>
<li>Prepare a 1 L beaker by rinsing in copious amounts of tap water and
then in a single beakerful of distilled water.  Pour out all water,
then pour in about 30-50 ml of absolute methanol and rinse the inside walls
of the beaker thoroughly.  Again rinse with distilled water, and empty
the beaker.
<li>Place in a clean stir bar (at least one-third, better one-half of the
diameter of the beaker).
<li>Add 100 ml of distilled deionized water, and begin stirring.
<li>Add 100 ml of concentrated phosphoric acid.
<li>Add 100 g of ammonium sulfate.  Stir until it is completely dissolved.
<li>While the beaker is stirring strongly, add 1.2 g of Coomassie
Brilliant Blue G-250 powder a bit at a time.  Make sure
all solids have dissolved before proceeding to the next step.
<li>At this point transfer the solution to a clean 1 L graduate cylinder.
Wash in about 100 ml at a time deionized distilled water to the beaker
to transfer all material to the graduate cylinder, but only to the
800 ml mark on the graduate cylinder.
<li>Pour this back into beaker and begin stirring strongly, but not
too vigorously.
<li>While stirring add 200 ml of anhydrous methanol.
<li>You might want to wash part of the beaker contents into the cylinder
to recover this material.
<li>Get a brown (or foil-covered) bottle and store the dye stock at
room temperature.  It should be good for at least 6 months.
</ol>

<h5>ProteomIQ Blue</h5>


<h5>Other Recipes</h5>
<p>
These other formulations are notes basically found on the Internet
or through other unpublished (not peer-reviewed) sources.  Use them
at your own risk.

<h6>Univ. of Missouri-Columbia Proteomics Center</h6>
<p>
The stock solution is prepared with 1.6% phosphoric acid and 8% ammonium
sulfate with 0.08% Coomassie G-250.  Add ethanol to 20%.

<ol>
<li>Wash the gel three times for 10 min each with ultrapure water.
Longer washes may be necessary for larger, and certainly thicker, gels.
<li>Add enough colloidal Coomassie G stain to cover the gel
<li>Shake for at least 6 hours
<li>Remove stain and wash with ultrapure twice for a minute or so each
<li>Continue de-staining with ultrapure water to eliminate background
(a few hours)
</ol>

<h6>OHSU Proteomics Core Lab</h6>
<p>
The stain can be prepared as follows:
<ol>
<li>Add 170 g ammonium sulfate to 340 ml methanol.
<li>Add water until the volume is 965 ml.
<li>Mix everything until all solids dissolve.
<li>Add 35 ml phosphoric acid.
<li>Add 1 g Coomassie Brilliant Blue G-250 and mix.
</ol>
<p>
Final concentrations are 17% ammonium sulfate, 3.5% phosphoric acid,
0.1% Coomassie G-250, and 34% MeOH.
<p>
The procedure for staining is:
<ol>
<li>Fix the gel in 50% ethanol and 8% phosphoric acid for 3 h to overnight.
<li>Rinse the gel briefly in water.
<li>Stain the gel.  The proteins become visible in several hours, but
may require up to 48 h for sensitivity or quantization.
<li>Rinse the gel briefly in 1% acetic acid before imaging.
</ol>

<h6>Max Planck Institute for Experimental Medicine Proteomics Group</h6>
<p>
From <a href="http://proteomics.em.mpg.de">http://proteomics.em.mpg.de</a>
<p>
The staining stock solution is prepared as follows:
<ol>
<li>Dissolve 100 g ammonium sulfate in 20 g phosphoric acid and ~800 ml water.
<li><em>While stirring</em>, add 1 g Coomassie Brilliant Blue G-250
<li>Let everything dissolve.  Bring to 1 liter with water.
</ol>
<p>
The final staining solution is prepared as an 80% staining stock and
20% methanol solution just before use.  Add 1 part of methanol to
4 parts of the dye stock solution <em>while stirring</em>.

<ol>
<li>Fix the gel in 40% ethanol + 10% acetic acid for about 60 min.
<li>Wash the gel in water twice in water for 10 min each.
<li>Stain the gel in the 80:20 working stain at least for overnight.
<li>Transfer the gel into a new plate in 1% acetic acid.  Keep washing
with acetic acid changes until background dye is removed.
</ol>


<h4 id="coomassier">Coomassie Brilliant Blue R-250</h4>
<p>
Do not use CBB-R250 on gels whose spots are to be excised for mass
spectral analysis; the R-250 dye is not compatible with MS work-up
(although apparently <a href="#mscompat250">not everyone</a> believes this).

<h5>Preparation</h5>
<p>
Prepare only enough to last about 6 months.  This is because the dye
is prepared in high concentrations of methanol and acetic acid, which
esterifies over time.  Sometimes adding methanol to aged dye can
rejuvenate it.
<p>
The dye is prepared as a 0.1% (w/v) solution in 40% methanol and
10% acetic acid.  To prepare 1 liter:
<ol>
<li>Weigh out 1.0 g of the highest grade (a grade intended to stain
protein) Coomassie Brilliant Blue R-250.
<li>Add it to 400 ml of reagent-grade (ACS-grade) methanol.  Make
sure that the dye is dissolved completely before proceeding to the next
step.
<li>Add 100 ml of acetic acid.  Mix well.
<li>Add 500 ml of double distilled (or ultrapure) water, and mix well.
<li><b>Filtration</b>. The need to filter Coomassie dyes depends on
the quality of the reagent.  In the past the dye quality was probably not
what it is today, and so filtration was necessary to deal with materials
that did not dissolve.   Filtration will be necessary if staining of
the gel leaves a high background residue on the gel surface.
</ol>
<p>
The solution is stored at room temperature.  Do <em>not</em> store
at refrigerated temperatures.

<h5>Staining</h5>
<p>
In both staining and de-staining, a shaker (rocker or orbital) should
be used:
<ol>
<li>It speeds up the process.
<li>It ensures even staining.
</ol>
<p>
Separate the glass plates holding the gel.  If the gel sticks to the
upper glass plate compared to the lower one (relative to the opening
process), then flip the plates over and let the gel rest on the plate
to which it adheres most.
<p>
Prepare the fixative.  The fixative is the staining solution solvents
without the stain: 40% methanol + 10% acetic acid fixative.
<p>
Get a clean, clear glass dish that is not smaller than the gel, and should
be as much as 25% larger (in width and height) than the gel ideally.  One
of those Pyrex baking dishes is best, and avoid the discolored dishes.
It should be clean:  first with water, and then maybe with a little bit
of the fixative washed in (if one wants to be on the safe side).  Be
careful of dried stain on the glass dish from previous use,
particularly if the stain is different than the one being used.
A little bit of 100% methanol easily solubilizes all Coomassie stains,
followed by liberal rinsing with water.
<p>
In pouring in the fixative to the glass dish, do not use a large volume
(maybe 5-10 times the gel volume) since this will get pitched.
The fixative does two things:  it essentially fixes the proteins
in the gel and it removes excess SDS from the gel, which can
interfere with staining.
<p>
The time to fix the gel generally depends on the gel thickness.  For
a 1 mm thick gel, 15-20 (some say 30) minutes is enough, as long as the
gel is being properly shaken (on rocker or orbital shaker).  For a gel
1.5 mm thick, add 50% amount of the time since the gel is 50% thicker
(say 30 min?).
<p>
Put the gel in fixative on the shaker and make sure it is covered with
plastic wrap tightly to prevent the methanol evaporation.  Do not use
aluminum foil over acids in solutions unless it is to keep light out
for photosensitive substances, and
then only when there is plastic wrap first covering the dish (this is
a general rule).
<p>
When the fixing is done, pour out the fix.  The best way to do this without
breaking the gel (caused when it folds on itself usually) is to hold the
part of the gel is furthest away from the glass end from which the
fix will be poured out.  Also, follow your local regulations when pouring
out organic solvents:  it may be necessary to pour them into organic
solvent waste and not down the sink.
<p>
Pour in the stain.  It should cover the gel with a volume at least equal
to its thickness (i.e., depth is twice gel thickness).  This allows the
gel to float freely during staining.  Staining is accomplished in about
the same amount of time as fixing (do it 30 min for 1 mm thick gels, to
be on the safe side, longer if really worried about completeness).
Be sure to cover the gel with plastic wrap tightly.

<h5>De-Staining</h5>
<p>
It should be noted that the stain can be recovered and re-used several
times.  Always recover stain in a sink.  Put the bottle in which the
stain will be recovered into the sink, put a funnel in it, and pour
the stain from the dish into the bottle.  A little might be spilled out,
but at least most is recovered.   Put the lid on the bottle with recovered
stain and wash its outside and the funnel liberally with tap water.
<p>
The empty glass dish with the gel is then washed a few times with tap
water.  This removes excess stain before using the de-stain.
<p>
Add the de-stain (usually something like 10% methanol with 5-7%
acetic acid) in the same volume as the stain and fix.  Add something like
small cell foam rubber or folded paper towel to the destain.  The foam
rubber or paper towel absorbs the dye very well and permits de-staining
without having the change the de-stain (and consume expensive solvents).
<p>
Shake the gel.  The amount of time necessary for de-staining depends
largely on how fast it takes for that portion of the gel representing
background (for which there is absolutely no protein) to become
thoroughly transparent, without any shading of dye at all.  Once
the gel background has become thoroughly transparent, remove the de-stain
(either pitch it out, or even recover it, since the dye has largely
been absorbed by the foam or towel) and put the gel in water.
<p>
In photographing/imaging the gel, it is best to put the gel in an
abundant amount of water anyway for 10-15 minutes, perhaps with a change
of water.  This reduces the possibility that acid is present in the gel
which might corrode equipment involved with imaging (although that may
not be a major consideration).
<p>
For long term storage of the gel, some people advise using 1% acetic
acid rather than water.  Put the gel in a cold room or refrigerator
since micro-organisms can sometimes grow if stored several days.
If protein from the bands or spots of the gel are to be recovered, try
to do this in as short a time as possible, since the gel is not the best
way to store this material.



<h4 id="mscompat250">Other Recipes</h4>
<p>
The OHSU proteomics core laboratory believes that Coomassie R-250
stain is compatible with MS work.  The following recipe is offered:

<ol>
<li>Fix the gel for 3 hours in 50% ethanol + 10% acetic acid.  The
fix can also be left overnight.
<li>The gel is placed for 30 minutes in 50% methanol + 10% acetic acid.
<li>Stain the gel in <em>filtered</em> 0.1% Coomassie Brilliant Blue R-250
in 50% methanol + 10% acetic acid.  Leave the gel in the stain for at
least 12 hours to get the greatest sensitivity in minor spots.
<li>De-stain in 50% methanol + 10% acetic acid.  Stop when the background
is clear.
<li>The gel should be stored in 3% acetic acid.  Do imaging and spot cutting
as quickly as possible.
</ol>

<h3 id="silver">Silver Staining and De-Staining</h3>
<p>
Silver staining of proteins is done either by the alkaline silver diammine or
by the acid silver nitrate method.  The silver diammine method is more
sensitive because the background is lower, but the method requires
glutaraldehyde fixation which cross links proteins and thus makes
the method incompatible for MS analysis.  The problem with the high
background of the silver nitrate method is that the sensitizers are
responsible for the background increases, and thus efforts are made
to use different sensitizers.  (These notes by M. Schuchard, R. Mehigh,
and W. Kappel in the ProteoSilver&trade; technical application notes from
Sigma-Aldrich.)
<p>
Silver staining occurs in five phases:
<ol>
<li><b>Fixation</b>.  Typically achieves removal of SDS, Tris, chloride
which have high affinity for silver and which can interfere with staining.
<li><b>Sensitization</b>.  The sensitivity to staining can be enhanced
using several reagents:  formaldehyde, glutaraldehyde, naphthalene disulfonic
acid, sodium thiosulfate, potassium tetrathionate are used between fixation
and silver impregnation.
<li><b>Silver impregnation</b>.  In the acidic approach, gel is soaked
in silver nitrate.  In the basic approach, the gel is soaked in ammonia
silver diamine.
<li><b>Development</b>.  The gel is placed in a reductant, usually dilute
formaldehyde, and the silver is reduced to metallic form which is insoluble
and visible.  Use of thiosulfate will enhance sensitivty by sequestering
weakly bound silver ions and reducing background;  thiosulfate can be
put in the development bath or integrated into the gel.
<li><b>Stop</b>.  An acid (acetic or citric) is added to halt the process.
</ol>
<p>
In general, the acidic silver nitrate methods should be used for MS-compatible
staining [from Poland, Rabilloud, and Sinha in Walker protocols].

<p>
Rabilloud and co-workers over the years have done a significant amount of
study on protein silver staining in gels.  Chevallet et al. (<i>Nat. Protoc.</i>
<b>1</b>: 1852-1858, 2006) have created a list of things to consider in
silver staining.
<p>
Acidic silver nitrate methods which require longer times to stain the gel
should be preferred since they give superior results; the more rapid stains
have issues.  When spots have the hollow appearance, this may be due to any
of the following possibilities
<ul>
<li>use of low silver nitrate concentrations (recipes that use 0.1%
should be doubled to 0.2%)
<li>decrease the time of what should be a brief rinse between the silvering
and the use of developer
<li>add 10 mg/L (1 mg/100 ml) of thiosulfate to the developing solution
</ul>
<p>
The ammoniated (diamine) silver should be done at a temperature above 20&deg;
(typically in the 20-25&deg; range).  The stain is better for basic as opposed
to acidic proteins.  Gels should be run with a glycine or taurine buffer
system, and polymerized with a complex initator (0.7 &micro;l TEMED +
5 &micro;l 10% sodium thiosulfate pentahydrate and 7 &micro;l 10% ammonium
thiosulfate per ml gel, added sequentially!).

<h4 id="rabilloud-Ag">Method of Rabilloud et al.</h4>
<p>
[from Rabilloud, Carpentier, Tarroux (1988) <i>Electrophoresis</i> <b>9</b>:
288-290].
All silver staining methods generally contain the following steps:
fixation, rinsing, sensitization, rinsing (brief to extensive, depending
on agent), silvering, (very) brief rinsing, development, stopping, and then
rinsing.  There are both acidic silver and silver diamine methods, the former
are generally MS-compatible.
<ol>
<li>Ensure that the stacking gel and focusing strip are removed.  They may
	contain substances that can degrade the staining of the gel.
<li>Fix the gel in 40% EtOH / 10% acetic acid for 30 min.  Repeat two more
	times:  this helps to remove ampholytes which contribute to background.
<li>Rinse in 20% EtOH for 10 min
<li>Rinse in water for 10 min
<li>Incubate for only 60 seconds in 30 mg per 100 ml
sodium dithionite (Na<sub>2</sub>S<sub>2</sub>O<sub>4</sub>).  If there is
none, sodium thiosulfate pentahydrate (<q>hypo</q>) will do.
<li>Soak two times in water for 60 seconds each.
<li>Soak the gel for 20-60 min in 200 mg silver nitrate and
	25 &micro;l formaldehyde (37% HCHO) per 100 ml solution
<li>Before doing this step, prepare the developer in the next step fresh
and have it ready.  Remove the silver solution and add plenty of water.
Rinse the gel in the water for only 10-20 seconds, and quickly remove the
water.  It is important not to over-rinse it in water.
<li><b>Developer</b>:  Per 100 ml, 3 g potassium carbonate
(K<sub>2</sub>CO<sub>3</sub>), 25 &micro;l formaldehyde, and 1 mg hypo.
Before adding any developer, make sure that the Tris-acetic acid in the next
step is ready.  Add a small amount of developer:  it may get yellow and
cloudy while rocking it around the gel.  Dump it.  Now add all the developer
and rock/shake the gel.  Watch it constantly as it develops over the next
several minutes.  Watch for the MW standards and make sure they develop.
Do not let it overdevelop unless you are looking for low abundance proteins.
When development has gone to the point where you are satisfied, quickly dump
the developer out of the dish and go immediately to the next step.
<li><b>Stop</b>:  Per 100 ml solution, 5 g Tris and 2 ml glacial acetic
acid.  Pour this solution into gel and leave it incubating at least 30 min.
<li>When development is fully stopped, put the gel in water for at least an
hour.  It is then ready to photograph/image.
</ol>


<h4 id="sigmaAg">Sigma-Aldrich ProteoSilver</h4>
<p>
The <b>ProteoSilver</b>&trade; Silver Stain Kit (product code
<b>PROT-SIL1</b>) is intended for use in SDS-PAGE (1DGE) or two-dimensional
PAGE (2DGE).  It is an acidic silver nitrate-based method.  The silver
ion will bind to proteins under weakly acidic or neutral pH conditions,
which is then reduced with alkaline formaldehyde is added as a developer.
It can detect as little as 0.2 ng BSA on a 1D gel lane.
<p>
This kit will develop 25 mini gels, and consists of five different
components:
<ul>
<li>ProteoSilver silver nitrate solution (P-3739)
<li>ProteoSilver sensitizer (P-3614)
<li>ProteoSilver developer 1 (P-3864)
<li>ProteoSilver developer 2 (P-3989)
<li>ProteoSilver stop (P-4114)
</ul>
<p>
Required but not provided are ethanol (product 27,074-1), acetic acid
(product A-9967), ultrapure water, and a glass or plastic staining tray.
The water quality can especially affect background (and thus sensitivity).
<p>
All steps of the following procedure are done at room temperature,
and all volumes of each step are intended to be 100 ml:
<ol>
<li>The gel is fixed in 50% ethanol and 10% acetic acid for at least
20 minutes, although it can be done overnight.
<li>Wash the gel in 30% ethanol for 10 minutes.
<li>Wash the gel in a large amount of ultrapure water for 10 min.
<li>Incubate the gel for 10 minutes in the sensitizer solution.
The sensitizer product is a 100 &times; concentrate:  add 1 ml to 99 ml
ultrapure water.  It should be used within 2 hours of dilution.  If
a precipitate forms, let it settle and take off the supernatant (the
formation of precipitate does not affect performance).
<li>Wash the gel twice for 10 min each in double the volume of water (200 ml).
<li>Treat the gel for 10 min in the silver solution.  The silver product
is a 100 &times; concentrate:  add 1 ml to 99 ml ultrapure water.
<li>The gel is now wash for only 60-90 seconds in twice the volume
(200 ml) of water.  If washed longer than 90 seconds, sensitivity will
fall.
<li>Prepare the developer by adding 5 ml of the Developer 1 solution to
95 ml of ultrapure water, and then add 0.1 ml (100 &micro;l) of Developer
2 to this solution.  This solution should be prepared immediately
(&lt; 20 minutes) before use.  Add to the gel.  Development should occur
within 3-7 minutes.  Longer times (10-12 min) might be necessary if
low abundance proteins (0.1 ng/mm<sup>2</sup>) are to be detected.
Balance development of spots/bands against the background.
<li>When development is complete, add 5 ml of the stop solution directly
to the developer and leave for 5 minutes.
<li>Wash the gel in ultrapure water and store it in this.  Try to
identify and cut out spots/bands of interest in as short a time as possible
after gel documentation.
</ol>

<h5>ProteoSilver Plus</h5>
<p>
The ProteoSilver Plus kit (product code <b>PROT-SIL2</b>) is intended
for 2DGE gels whose spots must be compatible for MS analysis.  It differs
from the <b>PROT-SIL1</b> kit in that it includes a destaining kit
(ProteoSilver Destainer A, product P-4239;  ProteoSilver Destainer B,
product P-4364) which oxidizes the metallic silver to silver ferricyanide
and then complexes with sodium thiosulfate to form a water-soluble
extractable way of removing the silver.
<p>
The development of the stained protein is the same as for the ProteoSilver
kit (PROT-SIL1).  See instructions for doing the de-staining.

<h4>MS-Compatible Silver Staining</h4>
<p>
[Simpson].  Prepare the following reagents:

<ul>
<li><b>Fix</b>:  30% EtOH + 10% acetic acid
<li><b>Sensitizer</b>: 0.3% potassium tetrathionate + 0.5 M potassium acetate
in 30% EtOH
<li><b>Ag Soln</b>: 0.2% silver nitrate
<li><b>Developer</b>: 3% sodium carbonate, to which is added 30 &micro;l
formaldehyde and 12.5 &micro;l of 10% sodium thiosulfate pentahydrate per
100 ml Developer.  Make fresh.
<li><b>Stop</b>:  4 g Tris base and 2 ml acetic acid per 100 ml.
</ul>
<ol>
<li>Place gel in the Fix solution in a plastic container;  glass might
cause a higher background.  Fix at least 30 min.  Repeat this twice
with fresh fix solution.  Do not re-use fix.
<li>Wash the gel in Sensitizer for 45 min.  Discard Sensitizer.
<li>Wash the gel in water six times, each for 10 min to remove as much
excess sensitizer as possible.
<li>Add the Ag Soln and shake for 1-2 h.  When discarding silver solution
use the appropriate waste container, possibly to re-claim the silver.
<li>Add water to the gel, but wash it only for 5-15 seconds to remove
silver from the surface of the gel, which might precipitate and form
a background.
<li>Add the Developer to the gel, and continuously shake it.
<li>Watch for the development of the bands.  Once they have formed
sufficiently without overdevelopment, remove the developer quickly.
<li>Add the Stop solution, and leave at least 30 min shaking.
<li>The gel can be stored for 6 months in water, or it can be dried.
</ol>


<h4 id="gilchavko">Gilbert &amp; Chavko Method</h4>
<p>
From <i>Anal. Biochem</i> <b>165</b>: 33-37 (1987).
<ul>
<li><b>Fixation</b>
 <ol>
 <li>Fix the gel for 30 min in 5:5:1:0.25 mixture of MeOH:water:acetic acid:glycerol.
 The final concentrations are 44% MeOH, 9% acetic acid, 2% glycerol.  If using an
 agarose gel with DNA to be stained, add Triton X-100
 to 0.25% (this would be 2.5 ml of 10% Triton X-100 for every 100 ml solution).
 <li>If agarose gel, gel is dried on polyethylene foil covered by filter paper soaked
 	in the fix and few layers of dry paper cloth under a 500 g weight load.  Paper
 	cloth is changed with dry one every 20 min, and load increased to 1 kg for 30 min.
 	Gel is dried under stream of cold air for 30 min, washed twice in water with 0.5%
 	Triton X-100 for 15 min and distilled water for 1 min.
 </ol>
<li><b>Staining Preparation</b>
 <ol>
 <li>Prepare 5% sodium carbonate (Solution A).  1 L can be used for about
 	40 mini gels.
 	Good for about 3 months.  Should be stored refrigerated, but bring to
 	room temperature before use.
 <li>Prepare 100 ml of 4% silver nitrate (AgNO<sub>3</sub>) and 4% ammonium
 	nitrate (NH<sub>4</sub>NO<sub>3</sub>).  The Bio-Rad method uses supposedly
 	a 1:2 molar ratio.
 <li>Prepare 100 ml of 20% tungstosilicic acid.
 <li>Prepare 100 ml of 6% formalin.
 </ol>
<li><b>Staining</b>
 <ol>
 <li>Prepare 50 ml of Solution B:  to 35 ml deionized water, add 5 ml
 	of AgNO<sub>3</sub>/NH<sub>4</sub>NO<sub>3</sub>, 5 ml of tungstosilicic acid,
 	and 5 ml of formalin reagents.  Unused Solution B can be stored for about
 	2 weeks refrigerated.
 <li>Put a volume equal to the Solution B to be used (50 ml) of room temperature
 	Solution A in a beaker with stir bar and keep the solution stirring.
 <li>Add solution B to solution A with constant stirring.
 <li>Immediately pour the mix on the gels and shake gently.  Development should
 	take 10-20 minutes.
 <li>The development is stopped by adding placing gels in fixing solution.
 	(Bio-Rad uses 5% acetic acid.)
 <li>For long term storage, gels can be placed in 1% acetic acid.  If spots are
 	to be collected
 </ol>
</ul>


<h4 id="bioradAg">Bio-Rad Silver Stain Plus Kit</h4>
<p>
Bio-Rad's kit is a commercial product of the Gottlieb &amp;
Chavko method (<i>Anal. Biochem.</i> <b>165</b>: 33-37, 1987).
Certain biomolecules, including the sulfhydryl and carboxyl groups of
proteins and bases of DNA will complex with silver ions (Ag<sup>+</sup>).
These complexes react more quickly than silver in solution to be reduced
and form metallic silver (Ag).
<p>
The Bio-Rad Silver Stain Plus kit (Cat #161-0449) includes the following:

<ul>
<li><b>fixative enhancer concentrate</b><br>
  contains methanol and acetic acid as standard fixative components, but
  also contains glycerol to prevent gel shrinkage
<li><b>silver complex solution</b><br>
  contains silver nitrate (AgNO<sub>3</sub>) and ammonium nitrate
  (NH<sub>4</sub>NO<sub>3</sub>) in about a 1:2 molar ratio
<li><b>reduction moderator solution</b><br>
  composed of dodecatungstosilicic acid, a heteropoly acid
  (H<sub>4</sub>O<sub>40</sub>SiW<sub>12</sub>)
<li><b>image development reagent</b><br>
  contains formaldehyde (HCHO)
<li><b>development accelerator reagent</b><br>
  contains sodium carbonate (Na<sub>2</sub>CO<sub>3</sub>)
</ul>
<p>
Ultrapure water rinses should be used with silver staining protocols since
impurities such as acrylamide, glycerol, urea, glycine, Triton X-100,
and agarose can cause a high background.  The tungstosilicic acid
is like an ion-exchange bead which complex Ag<sup>+</sup> ion so that it
is not free in solution.  Under alkaline conditions, formaldehyde
is a good reducing agent;  however, it does not reduce Ag bound wto
tungstosilicic acid compared to Ag<sup>+</sup> in free solution.
When sodium carbonate is added, tungstosilicic acid is deprotonated and
binds Ag<sup>+</sup> ions.  It also promotes ammonia formation, which complexes
with Ag<sup>+</sup>:  otherwise a Ag<sup>+</sup> ion would be precipitated
by CO<sub>3</sub><sup>2&minus;</sup> to form Ag<sub>2</sub>CO<sub>3</sub>,
which also binds to tungstosilicic acid.  The carbonate also activates the
formaldehyde.
<p>
All solutions but the development accelerator reagent (stored at 4&deg;)
can be stored at ambient temperature.

<h5>Preparation</h5>

<ul>
<li>All containers that will contact silver are best cleaned with
50% nitric acid after cleaning with a laboratory detergent.  These
are rinsed with high quality deionized water.  Containers that
themselves become stained are usually because of detergent residues
present that were not removed with acid wash and then water rinse.
<li>Avoid exposing the staining process to direct sunlight or temperatures
above 25&deg;.
<li>Never touch gels with metal objects or bare skin.  Wash gloves with
water, or handle gels with polyethylene or glass rods
</ul>
<p>
This method is described largely for polyacrylamide gels, but
can be used for agarose gels.  Agarose gels must be dried first.
There are three different ways to dry agarose gels:
<ol>
<li><b>Air drying</b>.  The gel is placed on the hydrophobic
side of a gel support film and left in a hood overnight.
Tape may be necessary to hold the gel in place.  The gel
is then removed from the film and processed as for polyacrylamide
gels.
<li><b>Compression drying</b>.  The gel is sandwiched between
to pieces of cellophane, and then this is sandwiched between
three pieces of filter paper.  A 1-2 kg weight is then placed
on top of the filter paper.  This is left overnight for drying.
After drying the cellophane is removed from the gel, and the
gel processed as for polyacrylamide gels.
<li><b>Vacuum drying</b>.  Put a sheet of filter paper on top
of the porous gel support of a vacuum gel dryer.  Place a sheet
of cellophane on the filter paper.  Lay the gel on top of the
cellophane, and cover this with cellophane, plastic wrap, or
gel support film.  Place the sealing gasket and apply the
vacuum. <strong>DO NOT APPLY HEAT!</strong>.  This melts the
gel.  Process the dried gel in the staining method below.
</ol>

<h6>Previously Coomassie-Stained Gels</h6>
<p>
Double-staining of gels with colloidal Coomassie G and silver can
improve sensitivity (detection) by 2- to 4-fold.
<p>
Coomassie-stained gels should be completely destained in
40% methanol/10% acetic acid.  The gel is sufficiently
fixed to skip the fix step of the Silver Stain Plus procedure.
Proceed immediately to the wash.

<h6>Development Accelerator</h6>
<p>
The Development Accelerator solution comes as a powder that must
be made up in 1 liter of deionized distilled water.  Place a teflon stir
bar in a 1 L cylinder with 950 ml water and begin stirring.  Slowly add
the 50 g of the Development Accelerator Reagent and let stir until dissolved.
QS to 1.0 L with water.  Store at 4&deg; but take what is necessary for
use and bring to room temperature.
<p>
The shelf life is 3 months.  Alternatively, weigh out a fraction
of the solution) and adjust the final volume to the fraction
weighed out.
<p>
If the Development Accelerator solution is too dilute or used
when it is cold, the staining
solution may become cloudy or contain a precipitate.  Always
let the solution reach room temperature, and if necessary,
add more solution (2-10 ml) to eliminate the precipitate.  Moreover
always add the solution all at once, never a few ml at a time.
The presence of the precipitate does not stop staining, it only
prolongs it.

<h6>Fixative</h6>
<p>Prepare 400 ml of the following:
<ul>
<li>50% methanol, reagent grade:  200 ml
<li>10% acetic acid, reagent grade:  40 ml
<li>10% fixative enhance concentrate:  40 ml
<li>deionized distilled water:  120 ml
</ul>
<p>This volume is adequate for staining two mini-gels (8 cm &times; 10 cm
&times; 1.0 mm).

<h6>Staining &amp; Developer</h6>
<p>This solution <em>must</em> be prepared within 5 min of use.
<ol>
<li>In a large beaker or Erlenmeyer with a stir bar, place 35 ml
deionized water.  Begin stirring.
<li>Add 5.0 ml Silver Complex Solution
<li>Add 5.0 ml Reduction Moderator Solution
<li>Add 5.0 ml Image Development Reagent
<li>Immediately before use, add 50 ml of room temperature Development
Accelerator Solution; swirl well.
</ol>

<h5>Other Solutions</h5>
<ul>
<li>Stop solution: 5% acetic acid
</ul>

<h5>Hazardous Waste</h5>
<p>The Staining &amp; Developer solution is hazardous and should not
be discarded down the sink.  Place the material in the container
in a fume hood.  Within 1-2 days, the silver present precipitates
and the aqueous solution can be decanted and disposed of.

<h5>Method</h5>
<p>
This method is adequate for staining two mini-gels (8 cm &times; 10 cm
&times; 1.0 mm).
<ol>
<li>Gels are placed in the fixative enhancer solution for 20 min
 after electrophoresis.  Skip this step if working with
 Coomassie-stained gels, which already be fixed.
<li>Rinse the gels in 400 ml deionized distilled water for 10 min.
For larger and thcker gels, rinse in double the water volume for
double the time in all steps.
<li>Replace the rinse with fresh water and rinse for another 10 min.
<li>Add the stain &amp; developer solution (100 ml for two mini
gels, 300 ml for two large gels).  Monitor development
carefully as the bands appear, usually within 15 min.  The whole step
takes about 20 min.
<li>Place the gel in the 5% acetic acid stop solution for at least
15 min.  Use the same volumes as for the fix and washes.
<li>Rinse the gels in water for 5 min.
<li>Photograph and/or dry the gels.
</ol>
<p>
Note that if the bands are taking a longer time to develop, and
a high background is developing with the longer staining time,
possible solutions could include the following:
<ul>
<li><b>longer fixation</b>:  this may be necessary for some proteins
as opposed to others, as not all proteins are identical
<li><b>poorly washed gels</b>:  the fix includes acetic acid,
which is actually the component in the stop, and so a thorough
water washing is essential
<li><b>use of cold development accelerator</b>:  always bring the
development accelerator to room temperature before using
</ul>

<h4>Univ. of Missouri Columbia Proteomics Center</h4>
<p>This method is sensitive to 5-10 ng and is based on Shevchenko et al.
method.
<ol>
<li>Fix gel in 20% ethanol + 1% acetic acid for 1 h.
<li>Wash gel in 20% ethanol for 10 min
<li>Place gel in 0.02% sodium thiosulfate for 1 min.
<li>Decant and rinse gel in ultrapure water for 1 min.
<li>Incubate gel in 0.1% silver nitrate for 15-20 min.
<li>Decant and rinse gel in ultrapure water for 1 min.
<li>Add developing solution and develop until spots visible
(2% sodium carbonate and 40 &micro;l per 100 ml of 37% formaldehyde).
<li>When fully developed, add a stop solution of 5% acetic acid for
at least 5-10 min.
<li>Store gel in ultrapure water
</ol>

<h4>Variations on MS-Compatible Silver Staining</h4>

<h5>Heukeshoven and Dernick</h5>
<p>Gels are fixed in 50% EtOH and 10% acetic acid.  They are sensitized
in 0.5 M sodium acetate, 0.2% (w/v) sodium thiosulfate pentahydrate,
30% EtOH, 0.5% (v/v) glutardialdehyde.  Impregnation is with 0.1%
silver nitrate and 0.01% formaldehyde.  This is then incubated in
2.5% sodium carbonate.  Development is with 2.5% sodium carbonate,
0.01% formaldehyde, 0.05% sodium bicarbonate, and 0.02% (w/v) thimerosal.
The stop is 50 mM EDTA disodium.

<h5>Hochstrasser</h5>
<p>Gels are fixed in 40% EtOH and 10% acetic acid, then in 5% EtOH and 5% acetic
acid.  They are sensitized first in 0.5 M sodium acetate, 1% (v/v) glutardialdehyde.
then in 0.05% (w/v) naphthalene disulfonic acid.  Impregnation is with 0.8%
silver nitrate, 0.34% (v/v) ammonia, 0.2% (v/v) 10 N NaOH.
Development is with 0.01% (w/v) citric acid and 0.1% formaldehyde.
The stop is 5% Tris and 2% acetic acid.

<h4>DNA Silver Staining</h4>

<p><b>Fixing Solution</b>.  Dissolve the following in water up to a 500
ml volume:
<ul>
<li>75 g trichloroacetic acid
<li>25 g zinc sulfite
<li>25 g glycerol
</ul>
<p>The fix is used for at least 10 min, and then several rinses in water are
done.
<p>
<b>Staining Solution B.</b>  Dissolve 25 g sodium carbonate
(Na<sub>2</sub>CO<sub>3</sub>) in water.
<p>
<b>Staining Solution C.</b>  Dissolve in water up to 500 ml
the following in the order given:
<ul>
<li>100 mg ammonium nitrate (NH<sub>4</sub>NO<sub>3</sub>)
<li>100 mg silver nitrate (AgNO<sub>3</sub>)
<li>500 mg tungstosilicic acid
<li>250 &micro;l of 37% formaldehyde
</ul>
<p>
Just before use, mix 68 ml of Solution C with 32 ml of Solution B.
Allow development for 10-20 min.  Make a rapid rinse in water, and then
add a stop solution of 1-5% acetic acid.

<h4 id="Ag-destain">De-Staining</h4>
<p>
<b>Potassium ferricyanide + sodium thiosulfate.</b>  Combine equal volumes
of 30 mM potassium ferricyanide, K<sub>3</sub>Fe(CN)<sub>6</sub>, and
100 mM sodium thiosulfate, Na<sub>2</sub>S<sub>2</sub>O<sub>3</sub> and use
within a half hour of mixing.
<p>
<b>Hydrogen peroxide</b>.
1% Hydrogen peroxide is sufficient for destaining silver-stained
cells.  Keep the gel in the peroxide only for the period required
to destain (2-15 min) to prevent unwanted degradation of the gel
or protein.  Gels can be re-stained by first neutralizing
peroxide with fixative or 10% acetic acid, then going to the
water wash.

<h3 id="fluorescent">Fluorescent Dyes</h3>

<h4 id="syproruby">SYPRO<sup>&reg;</sup> Ruby</h4>
<p>
[from Candiano et al. (2004), <i>Electrophoresis</i> <b>25</b>: 1327]<br> 
SYPRO Ruby is a permanent stain composed of ruthenium as part of an organic 
complex that interacts noncovalently with proteins.  Stained can be excited by 
UV light at 302 nm or with visible light at 470 nm. Fluorescence emission is 
approximately 618 nm.  Detection sensitivity is about 0.5-1.0 ng 
protein/mm<sup>2</sup>.  It is fully compatible with mass spectrometry and Edman 
sequence analysis.
<p>
The Bio-Rad protocol is below.  In all steps involving incubation with solvents 
or stains, use a shaker (orbital or appropriate rocker set at the right speed 
for the volume used).
<ol>
<li>Fix gels for 30-60 min in 10% MeOH and 7% acetic acid.
<li>Take out the fix and add the 1&times; SYRPO Ruby stain.  50 ml for 8 &times; 
10 cm gels, 150 ml for 13.5 &times; 10 cm gels, 330 ml for 16 &times; 20 cm 
gels, and 500 ml for 20 &times; 20 cm gels.  Using too little lowers the 
sensitivity.
<li>Although as little as 30 min can reveal the major proteins, it is best to 
leave for 3 h for maximum sensitivity.  It can alternatively be left overnight 
without overstaining.  Cover the gels with plastic film to prevent dehydration, 
and then cover the gels with aluminum foil to prevent light from getting in.
<li>Remove the stain (you might want to put in used stain bottle).
<li>Wash at least 30 min in 10% MeOH/7% acetic acid (same volume as stain). Do 
not over-destain!  Transfer the gels to water.
<li>Illuminate on either a transilluminating UV box (or in blue light).
</ol>

<h4 id="syprotang">SYPRO<sup>&reg;</sup> Tangerine</h4>
<p>
Can detect as little as 4-8 ng per band.  Stains within an hour in saline 
solution.  No fixation or de-staining step is done:  staining and quick rinsing 
and then visualization.  A standard 300 nm UV transilluminator will suffice. 
The staining is less expensive and quicker than silver staining.  Excitation 
also occurs at 490 nm, and emission at 640 nm.

<h4 id="syproorange">SYPRO<sup>&reg;</sup> Orange</h4>
<p>
This fluorescent reagent for protein gel staining can be used in SDS or native 
polyacrylamide gel electrophoresis.  Standard 302 nm transilluminators can be 
used to image the gels;  also 488 nm laser systems will work. The gel should be 
imaged by film or digital camera.  The excitation spectrum shows two maxima: 
one at 480-500 near full height, and the other maximum at 280 nm having about 
0.15-0.2 of the fluorescence of the other maximum. The emission spectrum shows a 
maximum at about 580-590 nm (hence the orange color).
<p>
SYPRO Orange is compatible for visualizing proteins in gels prior to 
immuno(electro)blotting, electroelution, or protein sequencing.
<p>
The dye emission is not as sensitive to the human eye as film or digital CCD 
chips.  For film, use Polaroid 667 B&amp;W film.  An orange Wratten 23A or 
yellow Wratten 9 should be used for photography. Exposures of 3-30 seconds 
(typically 15 s) with f/5.6 is not unusual for film. For digital cameras, 
exposures 1-5 seconds with 2 sec avg. is likely. Yellow filters will halve the 
exposure time.
<p>
Plastic (polypropylene) dishes and not glass should be used for staining
rather than glass trays.  The dye should also be stored in plastic bottles
that reduce background light.
<p>
SYPRO dyes bind to the protein through the presence of SDS.  Therefore native 
gels must be soaked in SDS prior to staining them.  SYPRO Orange is not as 
sensitive for proteins as SYPRO Ruby is.

<h5>Bio-Rad SYPRO&reg; Orange Concentrate</h5>
<p>
Supplied (Cat #170-3120) as a 5000 &times; concentrated solution in DMSO. The 
solvent is 7.5% acetic acid.  For each 100 ml of 7.5% acetic acid, add 20 
&micro;l  of the 5000 &times; concentrate.
<ol>
<li>Incubate the gel for 20-30 min in 7.5% acetic acid to eliminate
excess SDS.
<li>Replace with the diluted working stain.  Leave only 30 min
shaking gently with aluminum foil covering the gel.
<li>Recover the stain in its container.  Do not discard
until imaging is completed.
<li>Take the gel without adding any solution to it to the imaging system
(transilluminator).  Take with you the necessary amount of fresh 7.5%
acetic acid for a rinse (50-100 ml for a minigel).
<li>At the transilluminator, add the rinse.  Swirl the solution around for
only 30 seconds, then immediately recover the gel and place it on the
transilluminator.  Use distilled water to rinse off any material or to
prepare the gel for imaging.
<li>If you stained the gel for only 30 minutes, the signal-to-noise will
be good enough (the background will be low).  However, if the background is
too high, continue rinsing in 7.5% acetic acid for 30-60 minutes.  Change
the rinse if necessary to reduce the background.
</ol>
<p>
<b>Staining Native Gels.</b>  Fix/incubate native gels in 7.5% acetic acid
containing 0.05% SDS for 30 minutes prior to using the SYPRO Orange stain.
If a simultaneous incubation with SDS and SYPRO Orange is done, a
20-30 min 7.5% acetic acid rinse will be necessary.
<p>
<b>Gels for Electroelution or Electrotransfer to Blots for Westerns</b>. Do NOT 
use the 7.5% acetic acid fix.  Prepare Tris-glycine (no SDS) or the SDS-free 
electrode buffer that was used (with or without 20% MeOH). Add the stain to the 
transfer buffer.  The sensitivity will be reduced compared use of acetic 
acid-based solvents.
<p>
<b>For Protein Sequencing</b>.  The gel is incubated in Towbin buffer and not 
acetic acid.  Prepare the SYPRO Orange dilution on Towbin buffer, then incubate 
for 30 min.  After photographing/imaging the gel, then blot the proteins to PVDF 
using standard techniques.  UV epi-illumination can then be used to find the 
band/spot on the PVDF membrane.  Use pencil to identify location, then cut out 
the membrane and mount on sequencer.
<p>
<b>Peptides</b>.  The SYPRO stain will stain peptides down to 2000 Da separated 
on Tris/Tricine/SDS gels.  Soak gels in 10% acetic acid for 30-45 min, then 
stain as for the standard SDS-PAGE.

<h4 id="syprored">SYPRO<sup>&reg;</sup> Red</h4>
<p>
These are provided as 5000 &times; concentrated solution in DMSO, in 0.5 ml 
vials, which produce a 2.5 L solution, and that is enough to stain ~50 
mini-gels.  Can detect 1-2 ng per band.  In other respects it is like SYPRO 
Tangerine.
<p>
SYPRO Red is not as sensitive as SYRPO Orange is.

<h4>Amersham-Biosciences Deep Purple&trade; Total Protein Stain</h4>
<p>
The active molecule in this reagent is something called <b>epicocconone</b>.
It supposedly has a sensitivity better than the SYPRO dye family, at least
when it comes to staining proteins electroblotted to nitrocellulose
(Amersham's Hybond ECL) or polyvinylidene fluoride (PVDF, Amersham's
Hybond-P).
<p>
For gel staining, gels are fixed overnight in 10% MeOH/7.5% acetic acid. Four 10 
min washes in water are done, and then a 1:200 dilution with water of the Deep 
Purple concentrate is done for 1 h.  The stain is then developed in 1:1000 
concentrated ammonia with three 10 min incubations.  The signal is stabilized by 
washing in 0.75% acetic acid for 10 min before imaging on a Typhoon 9410 
Variable Mode Imager equipped with 532 nm laser and 560LP emission filter.

<h4>Krypton</h4>
<p>
This fluorescent stain is from <a
href="http://www.piercenet.com/Objects/View.cfm?type=ProductFamily&amp;ID=D55C1549-9741-4DD1-92D4-A83631AB37A5">Thermo-Scientific
Pierce</a>.  Excitation/emission is 520/580 nm.  It detects &le; 0.25 ng protein
in a band with a 3-4 orders of magnitude range.

<h4>Zinc Reverse Staining</h4>
<p>
Proteins, whether or not complexed to SDS, have the ability to bind 
Zn<sup>2+</sup> ion.  Imidazole also binds to Zn<sup>2+</sup> and forms an 
insoluble complex.  This deep white material forms on the surface of the gel. 
This reverse staining is more gentle to proteins than other staining methods. 
There is also a high sensitivity, with as little as 10 ng being sensed. 
Development is rapid (15 min, compared to 1 hour for Coomassie blue) and more 
than 2 hours for silver.
<p>
The following reagents are prepared:
<ol>
<li>0.2 M imidazole, 0.1% SDS
<li>0.3 M zinc sulfate
<li>0.5% sodium carbonate
</ol>
<p>
All are reagent- or analytical-grade.  Solutions can be prepared as a
10 &times; stock and prepared as 1 &times; just before use.
<p>
Both SDS-containing and native gels can be developed.  After electrophoresis,
incubate gel for 15 min in imidazole-SDS, then discard.  Then the gel is placed 
in zinc sulfate for about 30-40 s, and no longer than 45 s to avoid 
overdevelopment.
<p>
To photograph a reverse-stained gel, place on glass plate and above
a black underground (by a few centimeters).  Illuminate from the side.
<p>
Zinc ion forms a complex with two imidazolate ions because the pyrrole nitrogen 
loses its proton at high pH values.  But Zn also removes the pyridinium nitrogen 
proton upon complexing, forming the anion and binding to it.

<h3>Dyes for Other Applications</h3>
<p>
Notes on Amido Black and Fast Green FCF taken from C. M. Wilson, <i>Methods
in Enzymology</i> <b>91</b>:  236-247 (1983).
<h4>Amido Black (Acid Black I)</h4>
<p>
Suitable for PAGE-resolved proteins.  Gels are placed in 5-10 volumes of
7% acetic acid containing 0.1% Amido Black, and left minimum 2 h, but usually
overnight.  To remove background, soak in 7% acetic acid.
<h4>Fast Green FCF</h4>
<p>
Gels are stained in 1% Fast Green FCF in 7% acetic acid for 2 h at 5&deg;
or room temperature.  Destaining is with 7% acetic acid on shaker until c
clear (as long as 48-72 h).

<h2>SDS-PAGE (2<sup>nd</sup> Dimension) Results:  What To Expect</h2>
<p>
The figure below was taken from a Bio-Rad catalogue to show the approximate 
positions of where molecular weight markers for gels of different fixed 
polyacrylamide percentage (%T).  Most in-house cast gels will be fixed 
percentage gels since they are easier to pour and control for experienced and 
inexperienced technicians.
<p>
The values shown are the molecular weights of the natural proteins that are 
typically used as markers.  While synthetic markers are available for weights 
divisible modulo 10 or 25, e.g. 25, 50, 75, 100, 150, 200, and so on, projects 
involving protein mass fingerprinting (PMF) by MALDI- or ESI-MS, should really 
use natural proteins, since they offer themselves as controls in every way, 
including the fact that they are processed in nearly the same way as unknown 
samples. Synthetic markers cannot be identified by PMF.  The markers shown in
the figure below are indicated in the table:
<table>
<colgroup></colgroup>
<colgroup span="2"></colgroup>
<tr><th>Protein																<th>MW<br>(kDa)   <th>UniProt Accession Number
<tr><td>lysozyme															<td>14.4   
					<td><a href="http://www.uniprot.org/uniprot/P01012">P01012</a>
<tr><td>soybean trypsin inhibitor							<td>20
					<td><a href="http://www.uniprot.org/uniprot/P01070">P01070</a>
<tr><td>bovine erythrocyte carbonic anhydrase	<td>29 or 31?
					<td><a href="http://www.uniprot.org/uniprot/P00921">P00921</a>
<tr><td>chicken egg ovalbumin									<td>45
					<td><a href="http://www.uniprot.org/uniprot/P01012">P01012</a>
<tr><td>bovine serum albumin (BSA)						<td>66
					<td><a href="http://www.uniprot.org/uniprot/P02769">P02769</a>
<tr><td>rabbit muscle phosphorylase						<td>97.4
					<td><a href="http://www.uniprot.org/uniprot/P00489">P00489</a>
<tr><td><i>E coli</i> &beta;-galactosidase		<td>116
					<td><a href="http://www.uniprot.org/uniprot/P00722">P00722</a>
<tr><td>porcine heart myosin									<td>200
					<td>?
</table>

<p>
Most gels will be done at 7.5% polyacrylamide if the focus is to separate 
high MW proteins (ranging from 50 to 300 kDa), while 12.5% polyacrylamide
would be used to focus more on proteins in a more medium range:  15-125 kDa.
Separation of low MW proteins should be done at 15 and even 18% polyacrylamide.
<p>
<img src="imagesProteomicsReferenceHTML/BioRadMigrationCharts.gif"
  alt="Bio-Rad figure of MW marker band positions for gels of different fixed polyacrylamide percentage">


<h2 id="swissbioinform">Swiss Bioinformatics Institute
Recommendations</h2>
<p>
The Swiss Bioinformatics Institute runs a SWISS-2DSERVICE which attempts to
standardize the use of 2D-PAGE.

<ul>
<li><b>Sample Preparation</b>.  Samples should be solubilized in fresh
8 M urea, 4% CHAPS, 40 mM Tris base, 65 mM dithioerythritol (DTE), and
a trace of bromophenol blue.
<li><b>Staining</b>.
 <ul>
 <li><b>Silver Staining</b>.  If gels are to be silver-stained, then
protein concentration should be 2-10 mg/ml,
or sent lyophilized.  A minimum sample is 100 &micro;g, and should be
de-salted if the salts exceed 0.1 M.
 <li><b>Coomassie Blue staining</b>.  If gels are to be Coomassie-stained,
the protein concentration should be 50-200 mg/ml (or lyophilized) and the
sample minimum is 10 mg.  It should be de-salted if the sample salt exceeds
0.1 M.
 </ul>
</ul>


<h2>Experiments in Staining</h2>
<p>
The Proteomics Unit laboratory has looked at various techniques for staining
and a report is shown on this <a href="extract-staining-expts.html">document</a>.

<h2>Acid-Urea Continuous PAGE</h2>
<p>
[Simpson].  Use this type of gel electrophoresis to separate modified
forms of the same protein (charge isoforms) or minor structural variants.
This type of gel was described first by Panyim and Chalkley.
<p>
Prepare the following reagents:
<ul>
<li><b>50% T: 1.5% C acrylamide:bis</b>.  For every 100 ml, prepare
48.7 g acrylamide and 1.3 g bisacrylamide.  Filter on Whatman, store 4&deg;.
<li>1 mM diphenyliodonium chloride in water, keep at 4&deg; from light;
good for a week
<li>Glacial acetic acid
<li>2 mM methylene blue in water, good for a year at room temperature
if kept from light
<li><b>Resevoir Buffer</b>:  5.4% (0.9 M) acetic acid
<li><b>Sample Buffer</b>: 2.5 M urea and 0.1% pyronin Y in 0.9 M acetic acid.
<li>20 mM sodium <i>p</i>-toluenesulfinate in water, good for 1 week at
4&deg; in dark.
</ul>

<ol>
<li>A 15% polyacrylamide gel at pH 2.7 in urea (2.5 M urea in 0.9 M acetic acid)
is prepared.  The following makes 10 ml:
 <ul>
 <li>3.0 ml 50% T:1.5% C acrylamide:bis
 <li>540 &micro;l glacial acetic acid
 <li>1.5 g urea
 <li>150 &micro;l of 2 mM methylene blue
 <li>250 &micro;l of 20 mM sodium toluenesulfinate
 <li>200 &micro;l of 1 mM diphenyliodonium chloride
 <li>add water to bring to 10.0 ml
 </ul>
<li>Pour the solution into a minigel mold/casting apparatus
<li>The gel is photopolymerized.  An effective method is to position
about 10 cm away a 15 W daylight fluorescent lamp, or two light boxes,
each 12 W neon about 10 cm away on either side.
<li>Mount the gel and place the resevoir (electrode) buffer.
<li>The protein is dissolved in sample buffer, up to 35 mg/ml.
<li>Load the sample.
<li>The electrode placement is the reverse of SDS-PAGE:  electrophoresis
is from anode to cathode (positive to negative).  The voltage used might
vary;  for the Bio-Rad Mini-Protean, a 130 V running voltage is used.
<li>When the dye reaches the gel bottom, stop the electrophoresis.
</ol>

<!--  ==================================================
                  GEL IMAGING & DOCUMENTATION 
======================================================== -->

<h1 id="gelimage">Gel Imaging and Documentation</h1>
<p>
Gel imaging and documentation includes descriptions of handling and methods for 
generic and proprietary systems.  Imaging deals with the elements of photography 
and how to optimize/improve the visualization of cells in a digitized image 
format.  Documentation includes description of software programs that are used 
to analyze and compare images in order to find meaningful data and significant 
differences, as well as to annotate the gel so that the data is presented in a 
reportable manner (for manuscripts, posters, oral presentations).

<h2 id="imaging">Gel (Digital) Photographic Imaging</h2>
<p>
It helps if the biological scientist doing imaging of any kind is familiar with 
the principles of optics and photography, since a camera and an image-sensing 
surface (light-sensitive film or charge-coupled device, or CCD), are used in 
image acquisition.

<h3 id="fundamphoto">Fundamentals of Photography</h3>
<p>
Whether digital or film, a camera is used to acquire gel images in pretty
much every gel documentation system.  Thus it helps to know what the basic
parts of a camera are and what function they serve.

<h4>Exposure</h4>
<p>
The exposure of the image is a function of two factors.  In fact,
it is expressed as the following mathematical equation:
<p class="eqnserif">
<i>E</i> = <i>I</i> &times; <i>t</i>
<p>
where <i>E</i> is <i>exposure</i>, <i>I</i> is illumination, and <i>t</i> is the 
<i>time</i> that a film or light-sensitive CCD chip is exposed to the image. 
This equation says that if the photographer increases the time of the exposure 
by two-fold, the exposure itself will be two-fold.  Or that if the illumination 
(amount of light) is decreased by half, the exposure will be decreased by half. 
There are two basic elements or parts of the camera that directly affect or 
control exposure, and they will be discussed below.

<h4>Basic Camera Elements</h4>
<p>
There are <b>four</b> essential or principal parts found on any camera
that control both the <b>exposure</b>, appearance, and sharpness of the
image.  These are:
<ol>
<li>zoom lens
<li>aperture
<li>focus
<li>shutter
</ol>

<h5>Zoom Lens</h5>
<p>
A camera can have a single, fixed focal length or one with an adjustable
focal length.  A lens system with an adjustable focal length is usually
called a <b>zoom lens</b>, because one can <q>zoom</q> into or
magnify a faraway subject in the image of the camera.  The <b>focal length</b>
of a lens (or lens set or combination) is the point of focus behind a lens
when it is exposed to a subject that is <q>infinitely</q>
(or effectively <q>very</q>) far away, emitting light rays toward
the camera that essentially enter the lens in a parallel fashion.
Cameras having a 35 mm film format with only one single, fixed focal length (no adjustable
focal length or zoom features) will typically be equipped with a lens
having a 50 mm focal length (the <q>normal lens</q>), because this
produces an image with a field of view typical of the human eye.
When the focal length is increased, the lens causes a magnification or
zooming in of the image.  When the focal length of the lens is decreased,
there is a widening of the angle of view or zooming out of the image.
<p>
The difference between experienced and beginner photographers is how
they frame the subject and the background of the subject using the zoom
lens, if the photographer has a zoom lens (photographers who do not have
a zoom lens can simply move closer to or further away from the subject
to achieve the zoom effect).  For a scientist imaging a gel, there is
only one subject and no background of interest (of course, for a photographer
taking a picture of a person with perhaps trees or mountains and blue sky
in the background, the background may be of more interest).  The approach
to imaging the gel will be to zoom out completely so as to see the entirety
of the gel, then center the gel in the view being seen, and then slowly
zoom in while also making minor re-centering adjustments so that the gel
takes up the entire field of view either top-to-bottom or left-to-right,
whichever dimension the gel image fills up first in the field of view.
<p>
The camera on the VersaDoc features a zoom lens adjustment ring (the
biggest ring on the camera) with the numbers of the focal length printed
on the ring.  On less sophisticated camera (typically consumer grade),
the zoom is an electronic control rocker switch with one end marked
<b>W</b> (for <q>wide angle</q>) to decrease the focal length
and widen the angle of view, and on the other end marked <b>T</b>
(for <q>telephoto</q>) to increase the focal length and magnify
the subject (while, of course, narrowing the angle of the field of view).
<p>
For more information on focal lengths, try this <a
href="http://www.dpreview.com/learn/?/Glossary/Optical/Focal_Length_01.htm">page</a>.

<h5>Aperture</h5>
<p>
There is another ring on the camera called the <b>aperture</b> or
<b>field stop</b>.  The field stop or aperture is an iris-type stop
which changes diameter by turning a ring that features a series of number
values on it.  As a photographer turns the number values toward
a <em>higher</em> number, the diameter of the iris <em>decreases</em>.
<p>
This series of numbers on the aperture ring is called
<b>f-stops</b> or <b>f/stops</b>.  Many camera users never learn what
these numbers or rings do, so consequently they are at a disadvantage
when doing photography.  The camera might compensate by automatically
adjusting other settings so that an image can be taken, but not knowing
how the field stop works on a camera severely limits what a photographer
can do.  Note that on many digital cameras these, the
aperture or f-stop values are not on the ring of the lens,
but are set electronically from menus or buttons on the back of the camera.
<p>
The lowest f-stop value on the aperture ring or setting might be something
like 1.<i>x</i> or 2 or 2.<i>x</i> (usually 2.8), where <i>x</i> is any number.  The
rest of the numbers in the series are usually more standard:  4, 5.6,
8, 11, 16, and 22.  Usually 16 or 22 are the highest values.
Before explaining what these numbers mean and how they are calculated,
first look at the relationship between the numbers as a series:  they
all differ by a factor of the square root of 2 (&radic;2).  That is,
2.8 (the typical lowest number) &times; &radic;2 = 4, 4 &times; &radic;2 =
5.6, 5.6 &times; &radic;2 = 8 (rounded from 7.9), 8 &times; &radic;2 =
11 (rounded from 11.3), 11 &times; &radic;2 = 16 (rounded from 15.5),
and 16 &times; &radic;2 = 22 (rounded down from 22.6).
<p>
So how are the numbers calculated?  The f-stop value is the ratio
of the focal length of the lens and the
<p style="text-align:center;">
<table class="eqnserif">
<tr><td rowspan="2">f-stop =
 <td class="numer">focal length
<tr><td>aperture (field stop) iris diameter
</table>
<p>
Thus if the focal length of a lens remains constant, any decrease in the
iris diameter of the aperture means that the f-stop increases.
And f-stop value of 4, typically written <q>f/4,</q> means that
the focal length (the distance between the lens and the film or CCD chip)
is 4 times bigger than the iris diameter.
<p>
What is the importance of this f-stop value?
<p>
The field stop or aperture controls <i>two</i> important functions
of the camera.  One is the amount of light that enters the lens:
after all, the iris field stop is like a black wall that will block light.
The smaller the diameter set on the iris (i.e., the bigger the f-stop
value) the less light that enters the lens.  In fact, it is a well known
fact that when the diameter of the aperture is set at <i>twice</i> the diameter
of another aperture, the light entering the lens is <i>four</i> times
more at the larger than the smaller diameter.  Why?  Because of the
geometry of a circle (which is what an iris stop forms).  Light enters
through the <em>area</em> of a circle.  The area of a circle is
proportional to the <em>square</em> of its radius (circle area =
&pi;<i>r</i> <sup>2</sup>).  If the diameter is increased by 2, that means
the radius is increased by 2 as well.  It is easy to see then that the
area of the circle is increased by 4, and thus the light entering the lens
is increased by 4.   Now if we set the aperture from 4 to 8 on the ring,
this increases the iris diameter of the aperture by 2, and thus 4 times
more light enters the camera.  If we set the aperture from 4 to 5.6,
the aperture diameter is increased by the square root of 2 (&radic;2),
and thus we calculate that the amount of light entering the camera is
increased 2 times.  In fact, for each change in value on the aperture ring,
the amount of light entering the camera is exactly 2 times the amount of
light on the next
lower setting.  This is why cameras are constructed with these apertures:
to produce twice the light.
<p>
Note that it is light that produces illumination of the film or chip,
and so the aperture f-stop value directly affects exposure as in the
equation above.

<h6>Depth of Field</h6>
<p>
There is another important feature of the aperture.
It has to do with the physics of light and optics.  When the aperture
diameter is decreased, often to nearly a pinhole, objects both in front
and in back of the subject become in focus with the subject.  Note that
the subject is the object set in focus purposely using the focusing ring.
By thus increasing the f-stop (which decreases the aperture diameter),
the photographer <i>increases</i> the <b>depth of field</b>.
When the iris diameter of the aperture is increased (the f-stop value
is decreased), the objects in front of and in back of the subject
become blurred (out of focus).  Thus the photographer <i>decreases</i>
the depth of field.
<p>
There are often many good reasons for being able to blur and to sharpen
the focus of objects in front and in back of the subject.  But in capturing
gel images, there is no useful purpose to that, since there is no differences
in the depth of stained bands on a gel, at least any that can be resolved
using the f-stop on a camera.

<h6>Contrast in Images:  Field Stops in Microscopes</h6>
<p>
Field stops (iris diaphragm stop) also are found in microscopes and
are used to a great extent.  When the stop diameter is reduced, it
improves the resolution between two objects magnified on the slide.
But at the expense of contrast.  <b>Contrast</b> is very important in
being able to visualize objects (and the internal structure of transparent
objects) in a microscope.	Contrast is achieved usually by staining of
the objects, but if the objects are living cells, staining is impossible.
Changing the phase of light transmission to achieve interference can
create contrast (phase-contrast), but it can also be improved by
widening the field stop at the condenser or the lamp stop.  But then again
widening the field stop can reduce resolution as well.
<p>
Contrast too can be improved widening the aperture diameter of the camera
(reducing the f-stop).  The loss is <q>resolution</q> is effectively
the reduction in the depth-of-field and the blurring of objects.
<p>
For gel imaging, it is usually best to use the lowest f-stop values
(f/2.7 or f/4).  There will a balance between the time the shutter is open
and the f-stop value chosen.

<h5>Focus</h5>
<p>
Focusing does not do anything except to bring the subject in focus.
It has little effect on the light (exposure) and is not affected by the
zoom.  However, adjustments to the focus are done after setting the zoom
(to frame the field of view) and the aperture (to set the amount of light
and affect contrast, and to set depth of field).

<h5>Shutter</h5>
<p>
The shutter is another iris-type stop that remains fully closed until the
the shutter release is pressed.  It then opens for the amount of time
set in the clockwork of the camera, and then quickly closes.  The
<b>shutter speed</b> (or rather time the shutter is open) thus directly
determines the <i>time</i> of the exposure in the equation above.
<p>
In most consumer-used cameras, the shutter speed has fixed settings of time.
1 second is one such setting.  Above 1 second, the time moves in 2-fold
steps:  2, 4, 8 seconds.  There is also a setting in which the camera
shutter can be forced open and left open until the photographer sets it
to be closed (thus an <q>indefinite</q> setting).  Below 1 second,
the time decreases usually in 2-fold steps:  1/2, 1/4, 1/8, 1/15, 1/30,
1/60, 1/125, 1/250, 1/500, 1/1000, 1/2000.  (In milliseconds, these
are 500, 250, 125, 67, 33, 17, 8, 4, 2, 1, and 0.5.)
Cameras will differ in the number and value of the shutter speed settings.
Usually they go as fast as 1/1000, and more expensive cameras can get down
to 1/2000.  Handheld cameras usually need to set their shutter speeds
to faster than 1/60 or 1/30 seconds, because slower speeds (longer times)
usually show blurs in the subject caused by <q>camera shake</q> or
rather the shaking of the hands (tripods allow for longer shutter times).
<p>
If one has a light meter (cameras are often internally equipped with them),
one can calculate the correct shutter speed given a particular film
sensitivity (ASA or ISO rating) and f-stop setting.  With our gel documentation
system, there is a feature that will take a number of exposures within a
specified range of shutter speed (exposure time) settings.  We do as many
as 7 or 8 exposures.  In the old days, we would have to be conservative about it,
because expensive film was used.   But in these digital days, the <q>film</q>
is relatively free.

<h3 id="imagingguidelines">General Guidelines</h3>
<p>
With any image acquisition, there are a few rules by which to work.
<ul>
<li>The optimal image&mdash;and thus optimization&mdash;should be
done with the device that acquires the image (produces the raw file)
and not by software tricks or adjustments;  thus adjust the lighting
and contrast and the camera settings to get the best image in
the raw file, and don't rely too heavily on brightness/contrast/gamma
correction as a means of correcting poorly done photography.
<li>It is always cut away too much information than it is to try
to add it:  this means you do better by reducing the size of an
image that is too large than it is to increase the size (<q>
zoom in</q>) of an image that is too small
</ul>

<h3 id="versadoc">Bio-Rad's VersaDoc System and
PDQuest Gel Image Analysis Software</h3>

<h4>Start Up of VersaDoc and PDQuest</h4>
<ol>
<li>Find the power switch on the VersaDoc 1000 and turn it on.  The switch
is on the back of the VersaDoc 1000.
<li>Turn on the PC running Windows 2000 that runs the PDQuest software.
Open the PDQuest software.
</ol>
<h4>Place Gel in VersaDoc</h4>
<ol>
<li>If the gel is a Coomassie- or silver-stained gel, take the
white light conversion screen to the sink with the gel.
<li>Squirt liberal amounts of water on the conversion screen.
<li>Recover the gel from the glass dish:  it is possible to handle
it with gloved hands, but it can also be recovered by using plastic
wrap (stretch film) or even a glass plate placed in the dish and pulling
the gel on to it, then slowly lifting the material containing the gel
on to the conversion screen.  While pulling away the glass plate or
plastic wrap, keep the gel held on the conversion screen.  Develop the
best gel transfer method that works for you.
<li>Place the conversion screen with gel on top of the UV illumination
area of the transilluminator within the VersaDoc.  Try to adjust and
center the gel (this will also be done during preview imaging).
<li>Do not close the VersaDoc door yet.
</ol>

<h4>Acquiring the Image</h4>
<p>
Before the image can be acquired, the components of the VersaDoc
camera should be understood.  Read <a href="#fundamphoto">this
general introduction to photography</a> for understanding more.
<ol>
<li>Under the <b>File</b> menu, select <b>VersaDoc...</b>.  The
VersaDoc image acquisition window opens.
<li>The first step is to select one of four channels.
A channel allows for different applications (different types of stains)
to be imaged on the same gel.   Channels can be superimposed on top of
one another.
<li><b>Setting Imaging Type</b>.
Select the type of application:  protein gel usually.  A staining
submenu opens to allow selection of the type of stain.  If <b>Custom</b>
is selected, a custom application is created and then settings selected
by the user.  These include setting the filter, illumination, gain, and
binning.
<li><b>Initializing the Camera Elements</b>.
Set the zoom on the camera to the widest angle (lowest number on
the zoom ring, probably 20 mm).  Also set the f-stop to f/4 or even
lower.  This prepares you for framing the gel in the field of view and
getting the right illumination.
<li><b>Position/Focus</b>.  Click the <b>Position</b> button.  The VersaDoc
positioning lights come on and the camera begins taking images once
every second.  Check on the <b>Show Alignment Grid</b>.  If it is difficult
to see the gel at all or the gel in its entirety, it means you did not
set the zoom ring to the widest angle.  At the widest angle, it should be
able to see the whole tray within the VersaDoc chamber.
<li><b>Centering and Framing in Position</b>.
While the Position function is on, the camera will be taking 1 second
exposures and updating the viewed image.  During this time, try to center
the image left and right, top and bottom.  Use the blue-lined squares
of the alignment grid to help in the centering of the gel.  After the
first attempt at centering, then increase the zoom setting:  magnify
the gel as much as possible to fill the field of view without losing
the data on the gel.  Either the left and right or top and bottom edges
will be the first to fall out of the field of view if the zoom is too
high.  If one edge comes the edge of the field of view and the opposite
edge is not even close, then re-center the gel, and continue to change
the zoom.
<li><b>Focusing</b>.
Click the <b>Stop</b> button to cease the Position function.  Now
click <b>Focus</b>.  The camera will zoom in (digitally, not by adjusting
the zoom ring) to the middle part of the gel to magnify bands.  The
sharpness and resolution of the bands in this centered region of the
gel will be used to determine the focus.  Adjust the focusing ring
back and forth, slowly, and wait for image update.  Purposely adjust the
focus to see it blurred on either extreme to be able to understand what
a good focus is.  Since the lens is about 80 cm away from the gel,
experience has shown that the focus setting is usually 0.8 (80 cm).
<li><b>Using Optimize Exposure</b>.
Click the Stop button to cease the focus function.  There are several
possibilities for the next step, which is to acquire the image by setting
the correct exposure time.  One can set the time in the input box and
then click the Preview button to see a preview.  Alternatively pressing the
Acquire button will take a single image for the set exposure time and
present it in a separate image document window, which can be saved.
Perhaps the best method is to take several exposures over a range of times,
and then select the best exposure from the set.  The best method that
has been found to work is to click Optimize Exposure, set the number of
exposures to be 7 or 8, and then set the exposure time from the lowest
to the highest.  For Coomassie and silver stained gels on the light
conversion tray, the typical times are from 0.1 to 0.8 seconds.  Thus
if 8 exposures are obtained, the first will be 0.1 second, the second
0.2 seconds, up to the 8th exposure at 0.8 seconds.  Flourescent gels
require much longer exposure times, and depending on whether the
SYPRO dyes are being re-used or are fresh.  For fresh dye, usually from
1 to 10 second exposures are necessary.  But for re-used dyes, from 4-30 seconds
and even longer might be required.  For very weakly fluorescent gels,
it might take 60-120 seconds to visualize something:  this usually means
very little protein.  When producing multiple exposures through Optimize
Exposure, do not automatically save the images, since not all of the images
will be wanted anyway, and the unwanted images will take up lots of
precious hard drive disk space and make file management difficult.
<li><b>Selecting and Saving Image Files; File Naming</b>.
It will be easy to eliminate to the images that are at the extreme.
So click those away and don't save them.  Narrow down the choices to 3 or 4
good images and fit them all on the screen for simultaneous viewing.  Most
of the time, only one image is wanted, but sometimes it is good to put
keep a second different exposure (<q>just in case</q>).  When saving
the images to be kept, do not use the silly default name, which puts the
name of the user, the date and time and version of the image as GSC file.
Keep the version (<q>v</q> number) of the gel.  But when naming the
file, use something descriptive, although abbreviate somewhat to keep the
filename short.  For example, if the gel from a project of looking at
cortisone-treated rats and the gel is silver-stained (while another gel
is also run but is SYPRO Ruby-stained), then including the name of the
staining method is proper too: <q>cortisone rat Ag-stain v1.gsc</q>
is a good name in that case.  Include enough keywords to be able
to distinguish gels within a single project folder.
This is the abiding rule.
</ol>

<h4 id="right-exposure">Choosing The Right Exposure</h4>
<p>
Because this text contains experimental data in an attempt to understand the 
conditions for the right exposure, this section was taken out of the flow of the 
Reference because of its length and because its presentation is more than 
typical of a reference.  The document can be read <a 
href="ChoosingRightExposure.html">here</a>.

<h4>File Types Used By PDQuest</h4>
<ul>
<li><b>.gsc</b>: the <q>raw</q> (unprocessed after image acquisition)
2D scans
<li><b>.gim</b>: images which have been filtered as part of processing
<li><b>.gsp</b>: images which have been filtered and converted to
Gaussian spot representations
<li><b>.csc</b>: Calstrip scans
<li><b>.ms</b>:  MatchSet
</ul>
<p>
A <b>MatchSet</b> is actually set of scans or images that have been analyzed
and compared.  The images in a MatchSet are called <i>members</i>.
The members of a MatchSet can be Raw 2-D, Filtered, or Gaussian types.
<p>
The types of analyses that can be done within a MatchSet include:
<ul>
<li>spot comparisons between gels
<li>MW or pI value determinations
<li>spot annotation
<li>spot grouping for analytical sets
<li>replicate spot evaluation
<li>spot excision selection
</ul>
<p>
A <b>ScanSet</b> represents three different files of the same root
image.  These are the Raw 2-D, Filtered, and Gaussian images.
<p>
An <b>Analysis set</b> is a set of spots chosen for study.
Groups of spots can be selected that have statistical and biological
significance.

<h4>Images Are Three-Dimensional</h4>
<p>The images are really three dimensional (<i>xyz</i> coordinated).
The <i>x</i> and <i>y</i> represent the two dimensional area (width
and height, respectively) of the gel, while the <i>z</i> axis represents
an intensity value.  For a black-and-white image, there are shades of gray
in between.  For all images, the intensity is scaled into 4096 levels
or 12-bit intensity (2<sup>12</sup> = 4096).  With older software, one
usually got only 8-bit intensity levels (256 levels).
<p>
Thus a black spot representing an image of a stained protein can assume
one of 4096 intensity levels.  White has a value of 0 and black has a value
of 4095, and the levels of gray are in between to make up the 4096 levels.

<h4>Transforming and Filtering Images</h4>
<p>
The <b>Image&mdash;&gt;Transform</b> command only adjusts the image
brightness and contrast on the screen, and does not actually change
image intensity data.  Nonetheless, it helps the user find the regions
of the gel with real spot data so that only those are manipulated and
considered.
<p>
Generally the <b>auto-scale</b> is the best feature to use in Transform.
The scale is readjusted so that the lowest intensity spot (which might
be a gray level) is set to white, and the highest intensity spot is
set to black.  All intensities in between them are re-calculated so that
they are properly spaced.  The following formula generally describes this
process:
<p style="text-align:center;">
<table class="eqnserif">
<tr><td class="numer">(<i>PI</i><sub>2</sub> &minus; <i>MinPI</i><sub>2</sub>)
 <td rowspan="2" style="padding:0 1em;"> =
 <td class="numer">(<i>PI</i><sub>1</sub> &minus; <i>MinPI</i><sub>1</sub>)
<tr><td>(<i>MaxPI</i><sub>2</sub> &minus; <i>MinPI</i><sub>2</sub>)
 <td>(<i>MaxPI</i><sub>1</sub> &minus; <i>MinPI</i><sub>1</sub>)
</table>
<p>
where <i>PI</i> is the pixel intensity for any pixel in the image,
<i>MinPI</i> is the lowest pixel intensity in the image, and <i>MaxPI</i>
is the highest pixel intensity in the image.  For <i>PI<sub>n</sub></i>,
<i>MaxPI<sub>n</sub></i>, and <i>MaxPI<sub>n</sub></i>, <i>n</i> = 1 refers
to the old intensity and <i>n</i> = 2 to the new (re-scaled) intensity.
<p>
This re-arranges algebraically:
<p style="text-align:center;">
<table class="eqnserif">
<tr><td rowspan="2"><i>PI</i><sub>2</sub> =
 <td class="numer">(<i>PI</i><sub>1</sub> &minus; <i>MinPI</i><sub>1</sub>)
    (<i>MaxPI</i><sub>2</sub> &minus; <i>MinPI</i><sub>2</sub>)
 <td rowspan="2" style="padding-left:1ex;"> + <i>MinPI</i><sub>2</sub>
<tr><td>(<i>MaxPI</i><sub>1</sub> &minus; <i>MinPI</i><sub>1</sub>)
</table>
<p>
as <i>PI</i><sub>2</sub> is the re-scaled (new) intensity.
<p>
As an example, suppose the raw image has a minimum intensity of 650
and a maximum of 3168.  The new re-scaled (or auto-scaled) image will
convert that maximum of 3168 to 4095 (black), and the 650 value will
be made 0 (white).  Note that the midpoint between 650 and 3168 is
(3168 &minus; 650) / 2 + 650 = 1909.  Thus all pixel intensities less
than or equal to 1909 will have a value lower than their original intensity
in the re-scaled image, and those above 1909 will have a value higher
than their original intensity.  In the equations below, the intensity
of 2398 (one above 1909) and of 1103 (one below 1909) are calculated:
<p style="text-align:center;">
<table class="eqnserif">
<tr><td rowspan="2"><i>PI</i><sub>2</sub> =
 <td class="numer">(2398 &minus; 650) (4095 &minus; 0)
 <td rowspan="2" style="padding-left:1ex;"> + 0 = 2843
<tr><td>(3168 &minus; 650)
</table>
<p style="text-align:center;">
<table class="eqnserif">
<tr><td rowspan="2"><i>PI</i><sub>2</sub> =
 <td class="numer">(1103 &minus; 650) (4095 &minus; 0)
 <td rowspan="2" style="padding-left:1ex;"> + 0 = 737
<tr><td>(3168 &minus; 650)
</table>
<p>
One notes that these re-scaling calculations create an image with a far
greater contrast.  The essential defintion of contrast is to enhance the
differences between black and white, which is occurring by this transformation.
<p>
There is plenty on the Internet about the fundamentals of image filtering
and smoothing:

<ul>
<li><a href="http://www.bioss.sari.ac.uk/~chris/ch3.pdf">http://www.bioss.sari.ac.uk/~chris/ch3.pdf</a>
</ul>

<h4>Detecting Spots</h4>
<p>
There are basically two different ways to detect spots in PDQuest.
There are advantages and disadvantages to both.
<ol>
<li>The automatic way:  that is, let the computer software do it.
The advantage is that the computer does a fair job of actually detecting
hundreds, and as much as thousands, of spots that really do represent
stained protein spots.  This saves time in manually selecting spots that
number in the hundreds and thousands.  The disadvantage is that the
spot detection algorithms
<li>The manual way:  that is, you do the detecting.  The disadvantages of
this method are clear.  When there is a gel with hundreds or thousands
of spots, selecting each one is both time-consuming and tiring.
</ol>
<p>
There is a reason that it is called <q>spot detection and editing</q>
however.  The <i>spot detection</i> part essentially refers to letting
the computer the spots automatically, and the <i>spot editing</i> part
refers to the user adding or deleting spots manually after the computer
has been allowed to go first.
<p>
To some degree, even the automatic spot detection system is manually
controlled by the user.  The user can set three important parameters
used by the spot detection algorithms in the software:
<ul>
<li><b>Sensitivity</b>:  this sets the intensity threshold relative to
 maximum spot intensity for a spot  to be detected (a measure of
 signal-to-noise ratio).  Increasing the sensitivity value increases the number
 of spots detected.  If gel noise is being detected, lower the sensitivity
 value or use the <b>Speckles</b> removal control.
<li><b>Minimum Peak</b>:  while sensitivity adjusts signal-to-noise,
 minimum peak just looks at the signal to see if it passes a certain
 threshold.  That signal could also be noise if the threshold is set too low.
 Adjust this parameter after first adjusting sensitivity in the Step 2
 process or selecting new Faint (fainter) spots in the Step 1 process.
<li><b>Size Scale</b>:  this parameter looks at the size (area) of
 <i>objects</i> to understand if they represent signal (spots) as opposed
 to noise (intensity fluctuations not representing spots).  Larger values
 may be used when the noise is rather noticeable, but smaller values might
 be used if the noise has been smoothed out.
</ul>
<p>
Spots on a gel are actually represented three-dimensionally.  The two
dimensions are the location (<i>x,y</i>) of the spot on the gel.  The
third dimension (<i>z</i> axis) is the intensity of the spot, measured
as a three dimensional peak which has an amplitude.  The peak also has
a perimeter (or circumference) at the base, which corresponds to the
Size Scale.
<p>
There are two ways to set these three parameters for automatic spot
detection.  In the <b>Spot Detection Parameter Wizard</b> or in another
spot detection interface, the user can set numeric values either by
entering them directly or using up or down arrows.  These are in the
<b>Step 2 - Test Settings</b> fieldset of the wizard.  Or the user
can go to <b>Step 1 - Give Manual Guidance</b>, which essentially
determines the initial parameters by having the user select faint and/or small
spots, and then box the largest spot.  If doing either Step 1 or updating
the numeric values directly in Step 2.  Note that by changing any of the
three parameters in the Test Settings fieldset, the spot centers are
immediately re-evaluated using the changed parameter.  Keep track of the
<b>Spot Count</b> during every update.   As one raises the Sensitivity
level, the Spot Count should go up since it requires spots to have a
lower threshold to be detected.   If the Size Scale is raised, the
Spot Count should also decrease since this is the minimum size required
<p>
Note that Bio-Rad reports the following in the Step 1 process:
<ul>
<li><b>Faint</b> spot selection:  find the faintest spot you want to detect
sets both the <i>sensitivity</i> and <i>minimum peak</i> parameters.
<li><b>Small</b> spot selection:  this spot should be <em>smaller</em>
than the faintest spot.  This does not mean that it is actually fainter
than the faintest spot.  It must be smaller and more intense.  If the faint
spot is also the smallest spot, then do not select a small spot.
Selecting a small spot helps to set the <i>size scale</i> parameter
<li><b>Largest Spot Cluster</b> selection:  this will set the <i>radius</i>
of the <b>background subtraction rolling ball</b> and of the <b>streak
removal rolling disk</b>.
</ul>

<h4>Streaks</h4>
<p>
Streaks are typically found in areas of high spot concentrations.
Click the checkboxes for the kind of streaking you want to remove.
<p>
A disk radius is used to evaluate whether streaking has occurred, based
on the size of the largest spot cluster selected.  Streaks are removed
by reducing the size of the disk radius.
<p style="color:blue;">
Reduce the radius by half and do spot detection.  Repeat this process
until the streaks are not detected.

<h4>Background</h4>
<p>
Background subtraction is required for spot detection.  A floating ball
is used as the method, its radius calculated from the largest spot cluster
detected.  The ball radius can be changed, but it is
<em>highly recommended</em> that the calculated default be kept.

<h4>Smoothing</h4>
<p>
Smoothing is a <i>noise reduction</i> process, along with background
substraction.  These processes are usually always done when any signal
is significantly accompanied by noise.
<p>
The smoothing of a jagged line in one dimension, or the filtering of a noisy
image in two dimensions, is done by looking that the neighboring adjacents
signals around the signal whose value is to be adjusted.  Consider
a pixel whose intensity is 900.  It is surrounded by 8 pixels (assuming
it is not an edge or corner pixel).  Suppose their intensities are as follows:
<pre>
      882   892   903

      891   900   909

      897   896   902
</pre>
<p>
The mean of all those intensities is 896.8.  Thus the value 900 at the center
can be made the mean of itself and all the values around it.  The value
is then filtered, in a 3 &times; 3 kernel centered at the pixel to be
adjusted.  The kernel itself can be made bigger: 5 &times; 5, 7 &times; 7,
9 &times; 9, etc.  By including more surrounding pixels, the influence
of these pixels on the pixel at the center is greater, and the adjusted
pixel is brought closer to the values of the neighboring pixels.  Thus
a larger kernel achieves better smoothing.  However, a larger kernel can
also reduce signal intensity and thus the signal-to-noise ratio.
<p>
The filter algorithm works by moving in raster-like fashion
along a row of image pixels.  The filtering window describes what values
will be obtained of the centered pixel and surrounding pixels.  These
values are then processed according to the type of filter selected,
and then the pixel at the center of the window (kernel), that is the pixel
being processed, is changed or not changed according to the specification
of the filter.  Processing occurs pixel by pixel within a row, then row
by row within the image.
<p>
Smoothing is achieved in gel image data by applying <b>filters</b>.
The filters available are:
<ul>
<li><b>Median</b>.  This is more effective at dealing with so-called
salt and pepper noise, and does little to smooth out Gaussian noise.
If a small window is selected (3&times;3 vs. 9&times;9), less blurring
is created in the signal.  This filter evaluates the median of all values
in the kernel (window) and replaces the pixel being processed with the
median value.
<li><b>Weighted Mean</b>.  In contrast to the Median filter, this
is more effective at dealing with Gaussian noise and less effective
with salt and pepper type noise.
<li><b>Power Mean</b>.  This filter should be used to control salt
and Gaussian noise;  it will worsen the pepper noise.  The power mean
of all pixel intensities in the kernel is determined, and the value
replaces the value of the processed pixel.
<li><b>ContraMean</b>.	Use this for smoothing pepper and Gaussian noise;
will worsen the effect of salt.  The contra-harmonic mean of all pixel
intensities in the kernel is evaluated, and this value replaces the value
of the processed pixel.
<li><b>Adaptive</b>.  When all the other filters have not worked with
a particularly noisy gel, then try this.  It is best to use this filter
when there is a mix of salt and pepper noise.
<li><b>Out of Range Pixel (Mean)</b>.  Used to suppress salt and pepper noise,
with minimal effects on Gaussian noise.  This is a strictly <i>mean</i> filter:
the mean of all pixels in the kernel is evaluated;  if the difference between
the mean and the pixel being processed rises above a threshold, the
processed pixel is replaced by the mean.
<li><b>Maximum</b>.  This filter can eliminate pepper noise (and will
worsen the effect of salt noise).  It replaces the processed pixel with
the maximum intensity value of the pixels in the kernel (window).
<li><b>Minimum</b>.  This filter is better for eliminating salt noise
and will worsen the effect of pepper-like noise.  It replaces the processed
pixel with the minimum intensity value of any pixel in the kernel (window).
<li><b>Mid-Value</b>.  This filter is best for suppressing uniform noise
within an image and will make pepper and salt noise worse.   The algorithm
finds the minimum and maximum pixel intensities within the kernel, then
finds their midpoint (the mean of these two values), and replaces the
processed pixel with that value.
</ul>

<p>
The <b>Kernel Size</b> (filtering window) is the window matrix
to which all steps of the filtering processing algorithm are applied.

This should be set as follows:
<ol>
<li>Magnify a background region to see the individual pixels and look at
several noise features, and determine the average size.  Move the gel
around if necessary.
<li>Magnify the smallest spots and determine the pixel dimensions.
<li>The Kernel Size should be <em>larger</em> than the average noise
size, but <em>smaller</em> than the pixel area of the smallest spots.
</ol>

<h4>File Type Conversions and Considerations</h4>
<p>
TIFF images are importable but are missing information about the
type of scanner, color, etc. that were used in image acquisition.
There are many types of TIFF files, and PDQuest is able to import
8- and 16-bit grayscale uncompressed formats.
<p>
JPEG is an export format used to prepare reports and presentations,
and is used more than TIFF because of its ability to compress data.
PDQuest cannot import JPEG files.  When exporting, a quality setting
between 50-80 is usually a good compromise between image quality
and file size (the higher the quality, the larger the file size).
<p>
MatchSet data can be exported to a Text file for import of info
into spreadsheets.  Exportable data includes spot position, quantity,
quality, Gaussian size, normalized quantity and peak value.
Data can also be exported in an XML format.

<h4>Spot Editing</h4>
<p>
Adding and removing spots can be done with the raw and filtered images.
Bio-Rad also recommends using the Gaussian image too, especially with
match sets.
<p>
Check the gel image with its spot center crosshairs overlaid on the image.
Carefully inspect the Raw and Filtered images as follows:
<ol style="list-style-type:lower-alpha;">
<li>Look at the spots without the overlaid cross hairs.  Count the number
of spots in the Raw and Filtered image, both zoomed out and zoomed in,
and come to an agreement as to what you see.
<li>Now overlay the spot center crosshairs on to the image.  See if the
count of crosshairs agrees with the count you made.
<li>If not, remove spot center crosshairs that clearly do not correspond
to the center of the spots you have visualized.  If there are spots with
no crosshairs to their centers, add them.
</ol>
<p>
The result of this manual editing will be that the correct number of
spots are made and their centers determined.
<p>
Note that there is a difference betweeng <i>cancelling</i> and
<i>removing</i> spots.  Cancelling spots in scansets or matchsets
removes the spot from being considered in matching or other analyses,
but keeps the information about the spot.  Removing a spot takes it
out of the Gaussian member of the set, indicating that there is no spot
at all entirely.
<p>
Spots that are cancelled can be restored by selecting them or a group of
them and using the Restore command in the Spot tools.
<p>
<b>Combining</b> spots is used to combine a cluster of spots into one.
It is done on Filtered images of a scanset or matchset.  The Gaussian
image shows a single spot although the raw and Filtered images show
several spots.  This combining of spots will be quantitated as a single
spot.

<h4>Quantitation</h4>
<p>
The quantity of the Gaussian image spots is evaluated by the following
equation:

<p class="eqnserif">
spot quantity = <i>spot height</i> &times; &pi; &times; &sigma;<sub><i>x</i></sub> &times;
&sigma;<sub><i>y</i></sub>
<p>
where <i>spot height</i> is essentially the peak intensity.  The value
for the spot height (intensity) has dimensions of either <b>OD</b> (<i>optical density</i>) or
<b>counts per square image units</b> (OD/IU<sup>2</sup> or
counts/IU<sup>2</sup>),
depending on imaging device.  One image unit (IU) is 0.1 mm (100 &micro;m).
&sigma;<sub><i>x</i></sub> is the standard deviation of the spot along
the <i>x</i> axis, and &sigma;<sub><i>y</i></sub> is the standard deviation
along the <i>y</i> axis, which are measured in IU values.
<p>
The <i>spot quantity</i> will have dimensions of ODs or counts when the
equation is evaluated.  This system is preferable to summing the pixel
intensity values.

<h4>Spot Quality</h4>
<p>
The software has the ability to score <i>spot quality</i> based on the
following factors:
<ol>
<li><b>Gaussian fit</b>.  Assigns a value to how well the spot fits the
Gaussian model.
<li><b>X-streaking</b>.  Determines how much of the spot is affected by
gel streaking in <i>x</i>-direction.
<li><b>Y-streaking</b>.  Same determination as for X-streaking, but in
<i>y</i> direction.
<li><b>Overlap</b>.	Determines percentage of spot overlapping with other
spots.
<li><b>Linear Range of Scanner</b>.  Is the peak intensity within the
linear range of the scanner?
</ol>
<p>
The score is from 0 to 100.  A value of 100 is assigned to spots which
fit the Gaussian model perfectly, have no streaking in <i>x</i> or
<i>y</i> direction, no overlap with any spot, and has a peak intensity
within the scanner measuring range.



<h4>Problems with Saturation</h4>
<p>
Spot intensities can have a maximum value of 4095.  There are some cases
where the image analysis has a large spot region where the maximum
value is reached at a contour far from the spot center.  It can be assumed
that center itself is at a greater intensity than those on the contour
several (or several tens) of pixels away from it.  Spot boundaries and
contours help to make determinations about spots who intensity levels are
such that the peak may be saturating.  The boundary tools can help to
instruct the Gaussian image about the true size of the spot for purposes
of quantitation.  Drawing boundaries with the tool or freehand draw tool
can also be used in spot matching procedures with other gels.

<h4>Spot Numbering</h4>
<p>
The system uses an <b>SSP</b> or <b>standard spot</b> numbering assignment
to MatchSet Master gels.  The SSP is a 4 digit number.  The software
divides the Master into rows and columns whose distances are placed
so that there approximately an equal number of spots in each row/column
section.  No section can have more than 100 spots:  this is because the
last two digits correspond to the number of the spot in the section.
The first digit is X- (column) coordinate representing the section
(increasing from left to right edge of image), and
the second digit is the Y- (row) coordinate of that section (increasing
from <i>bottom</i> to <i>top</i> edge of image).  Thus at the upper right
corner of the image, an SSP of 99<i>xx</i> is expected,
where <i>xx</i> represents the number of the spot in the section.
At the lower left, an SSP of 00<i>xx</i> is expected.

<h4>Match Sets</h4>
<p>
Match sets are created using either the <b>Match&minus;&gt;New MatchSet...</b>
command or through <b>Spot&minus;&gt;Automated Detection and Matching</b>
dialog interface.
<p>
Editing of MatchSets is done through the <b>Match&minus;&gt;MatchSet</b>
tool and the <b>Match&minus;&gt;Edit Matches</b> submenu.
<p>
MatchSets are composed of:
<ul>
<li><i>copies</i> of all ScanSet images (Raw, Filtered, and Gaussian images)
<li>a Master image, which is a synthetic image of all the spot data from
all gels in the MatchSet
</ul>
<p>
<i>All comparisons and analyses of spot data from all gels are done through
the Master</i>.
<p>
The Master images from various MatchSets can also be combined/grouped
into higher level MatchSets for other comparisons.
<p>
The following analyses can be done with MatchSets:
<ul>
<li>Qualitative spot comparisons
<li>Quantitative spot comparisons
<li>pI/MW value estimations
<li>Spot annotation
<li>Grouping of spots into analysis sets
<li>Replicate group creation:  combining spots from multiple replicate
(identically loaded) gels
<li>Selection of spots for excision
<li>Creation of <q>Cybergels</q>:  combining gels with different
pH gradients into a longer representative gel
</ul>
<p>
Note that for quantitative comparisons, gels must have intensity values
(spot quantity values) in the same units/dimensions.  Thus spot values
in OD units cannot be compared to spot values in counts.

<h5>Creating MatchSet</h5>
<p>
This command is under the <b>Match</b> menu.  It opens a dialog interface
for creating match sets.
<p>
Specifying a <b>Name</b> for the MatchSet is a required element.  Click
the <b>Auto</b> button to generate an automated name, or create your
own name.  A short description and more extensive comments can be used
to annotate the MatchSet.
<p>
A MatchSet can be a collection of ScanSets (the members of the set are
gels), or it can be a grouping of MatchSets, whose members are Masters
of the MatchSets.  The latter is called a <b>Higher Level MatchSet</b>.
<p>
By default, all original images are copied into a folder created for all
work of the MatchSet.  Alternatively, the MatchSet can refer to the
originals, but if the originals are altered, so will the MatchSet be
affected.
<p>
Finally a list of gels to become the members of the MatchSet are listed.
The list initially consists the names of all gels currently opened.  The
<b>Add</b> button calls up a dialog to allow inclusion of new members to
the list.  The <b>Remove</b> button will remove from the list all
selected gels.  Gel selection is by a mouse click in the list.  A
range of gels can be selected by holding the Shift key and clicking on
the end or beginning of range of listed gels, and all gels in the range
will be selected from a gel that is already selected.  Alternatively
single gels in a multiple selected list can be selected or de-selected
by holding down the Ctrl key and clicking on a gel.
<p>
<b>Selecting A Master</b>.  If the option of letting the software
Auto Select Master is not used, then select your own Master.  A good
Master gel will have the most spots of all gels and the least streaking.

<h5>Automated Detection and Matching</h5>
<p>
This command is selected under the <b>Spot</b> menu.  There are
four steps to completing the command.

<h6>Step 1: Selection of Gels</h6>
<p>
When the command is first selected, all currently open images are
added to the list of gels to be analyzed.  Thus it is useful to open
all the gels of interest before selecting the command.  For selecting
more than one gel in a list, use the mouse click and hold down the
Ctrl key to select single gels, or the Shift key to select a range of
gels extended from a previously selected gel.
<p>
If a gel is present in the list not to be analyzed, select the gel
and click the <b>Remove</b> button.
<p>
If a gel is absent in the list that is to be analyzed, then click the
<b>Add</b> button to get a <b>Select New Member</b> dialog to find the
gel of interest.
<p>
Click the option <b>Auto Select Master</b> if the software is to select
the Master.  Alternatively, select your own Master by clicking the
Master button for the selected gel.  The software selects as the Master
the gel with the best average spot quality.
<h6>Step 2:  Spot Detection Parameters</h6>
<p>
This is the same as the Spot Detection Wizard.  The currently selected
gel is used to set up the parameters.

<h6>Step 3:  Create Match Set</h6>
<p>
Enter a name of the match set or click the <b>Auto</b> button to get an
automatically assigned name (default names are set in the Preferences
dialog). A short description, and a longer set of comments can also be
added.
<p>
Specify the directory for saving the MatchSet.
<p>
The option <b>Copy originals into MatchSet's data folder</b> is selected
by default.  This allows the originals to be used in other operations,
other MatchSet creations.

<h5>Editing MatchSets</h5>
<p>
The editing of MatchSets is done either by <b>Match&minus;&gt;Edit Matches</b>
or <b>Match&minus;&gt;Match Set Tool</b>.  It has the same look as Create
MatchSet dialog except that the Name, MatchSet Level, and Member Sharing
elements are disabled (cannot be changed).
<p>
The only changes that can be made are to the Description, Comments.  More
importantly, the members of the MatchSet can be added or removed, and
the Master gel can be changed.
<p>
<b>Caution</b>: if the Master gel is changed, all annotations, analysis sets,
landmarks and other stuff pertaining to the current Master are lost.

<h5>How Matching Works</h5>
<p>
Matching is done in several ways, but is generally automated and/or manual

<h6>Automated Matching</h6>
<p>
The software does the matching automatically.  The user assists this
operation by helping the software match in two phases.
<p>
<b>Primary Matching</b> is the first phase, making <i>high confidence</i>
matches. A slider from 0-100 is used to set the percent confidence level. The
higher the confidence level, a spot will be expected to have an exact position
based on its neighbors. Lowering the confidence level allows less exact/precise
positioning. The disadvantage to lowered confidence is that matching may occur
that is not true.
<p>
Matches made at the primary level are shown in
<span style="color:green;"><b>green</b></span>.
<p>
Cybergels can be created in this process, but it slows down matching, and
they can be created in other ways as well.
<p>
<b>Extended Matching</b>. Extended matching allows lower confidence matching.
Matches made by extended matching are shown in a different color. <b>Classic</b>
method extended matching uses an algorithm that regards all matches in the
Primary phase as landmarks made by automatic matching. The Classic match finds
more spots at a lower confidence (the equivalent of the <q>liberal</q> setting
in Restricted method); it can increase the likelihood of false matches. <b>
Restricted</b> method extended matching presents to the user a slider that
allows extended matching at a level of confidence at the Primary phase.

<p>
Matches made by Extended Matching are shown in
<span style="color:blue;"><b>blue</b></span>.

<h5>Monitoring Matching</h5>
<p>
The user can do the automated matching by selecting <b>Match All</b>, and
the default Primary and Extended Matching parameters are used.
<p>
Alternatively, the user can alter the settings and monitor what happens
in the matching process.  A selected member will have its count of
primary and extended phase matching spots shown.

<h5>Landmarking Matched Spots</h5>
<p>
This feature can assist in automated matching, and is a form of manual
matching.
<p>
When landmarking the Master and the member gel:
<ol>
<li>Zoom in on the Master for the spot to be landmarked.  It shows also
the software's idea of the relative position of the member gels.
<li>Select as a landmark only well-resolved spots clearly present in all
the member gels.
</ol>

<h4 id="analysis-sets">Analysis Sets</h4>
<p>
Six different analysis sets are available.  Three of the analyses are
classed as <q>A-B</q> (A gel and B gel, or rather two or more gel) analyses.

<p>
The <b>Arbitrary</b> analysis is used to do a lot of manual decisions
regarding spots, especially for only one gel and when cut lists are to
be generated from a single gel.
<p>
The <b>Matching</b> and <b>Boolean</b> analyses are higher order analyses
which are generated from existing analyses sets.

<h5>What If There Are No Comparative Analyses Between Two or More Gels?</h5>
<p>
When only a single gel is available and selected spots are to be cut,
the following steps are taken:
<ol>
<li>Make sure that a matchset has been generated for the gel image.
This is done with Automated Detection and Matching (even though no
gel matching has been done).
<li>Select <b>Analysis Set Manager</b> from the <b>Analysis</b> menu.
<li>A dialog box is shown with the matchset that is currently open.
 Click the <b>Create...</b> button.
<li>Select the <b>Arbitrary</b> type.  This means that spots will be
selected manually.
<li>A new dialog shows.  Make sure the option <b><q>All spots in
member</q></b> are selected.
</ol>

<h5>Qualitative</h5>
<p>
Used to distinguish spots <i>present</i> in one gel and
<i>absent</i> in another.
<p>
The name field is required although the description is not.
<p>
The comparison in this analysis involves one of three possibilites:
<ol>
<li><b>Gels</b><br>
 A simple A-B gel comparison for spot presence/absence.  Select an A and
 B gel from the matchset.
<li><b>Replicate (Repl.) Gels</b><br>
 Comparison of A and B gels loaded with the identical or similar sample proteome.
 One should select what is really an A and A pair of gels from the matchset.
<li><b>Classes</b><br>
</ol>
<p>
The <b>Method</b> indicates exactly how the analysis will be done during
in comparison.  <var>On in A</var> indicates that the spot should be
present in gel A, and absent in B.  Likewise  <var>On in B</var> means
that the spot is present in B, and absent in A.  Presence and absence
have variable meanings, since there is even a signal present in the background.
For this reason, for any spot to qualify as present, it must be said to be
a certain amount about the minimum sensitivity (the default is 10.00).
<p>
In the <b>Options</b> fieldset, when the <var>Estimate missing spots</var>
is <em>not</em> checked, the spot must be detected in <em>all</em>
gels for it to be included in analysis set.  If checked, the spot
must be detected in <em>at least half</em> of the gels to be included
in the analysis.  This is important since the analysis is based on
whether spots are in group or class and not another.
<p>
The <var>Estimate saturated spots</var> option will have the analysis
estimate the total peak value for any saturated spots.
<p>
The options are activated only when the analysis involves groups and
classes.  For instance, there is no need to estimate or guess that a
spot is missing if the gels are not replicate gels or members of a class,
that is, if it known that the gels are different because the samples represent
distinct proteomes.
<p>
Note the <b>Spot Count</b> at the lower right of the interface.  This
will change depending on whether you choose gel A or B and the minimum
fold over background (the higher the value, the less spots).

<h5>Quantitative</h5>
<p>
Used to distinguish spots present in two gels with different samples,
but whose quantity is present with a possibly threshold fold-difference.
<p>
A name is required (a default is provided).  Descriptions are optional.
<p>
<b>Estimate missing spots</b> is available for replicate gels and classes.
Because replicate gels should all have the same spots and intensities,
the software can assume that a spot present in one gel should be present
in replicates.
<p>
<b>Estimate saturated spots</b> instructs the software to guess a quantitation
for a peak despites its saturation.  It can do this by extending the
slopes of the peak sides to find its top, assuming a Guassian shape.
<p>
The comparison of gels involves deciding the limits of the quantitative
change.  Spots will fall into several groups:  those within the limit,
and those outside a limit.   Gel A is the reference.  The spots in gel
A that match in gel B will probably vary in their intensity.  Spots in
B may be below a fraction of the intensity of A, or over a fold intensity
of A.  Spots in B may be selected if they are outside the set limits,
or only above the upper limit, or only below the lower limit, or between
the limits.
<p>
Once spots with quantitative differences are determined, which spot in
which gel is selected for excision?  For example, if a spot in gel B
is 0.1 the intensity of the spot in gel A, it is more sensible to cut
the spot out of gel A, since it is present in gel A in greater amounts,
and would be more likely to be identified.  Hence spot excision selection
should always favor the spot in the gel having the greater intensity.
<p>
<i>Normalization</i>.  Before quantitative analyses are applied, an
adjustment will be made to the spot intensities if a control or standard
protein has been loaded on gels in equal amounts, but which have variable
intensities.  Thus spot intensities will be <q>normalized</q> to account
for the possibility that recovery or electrophoresis was quite variable,
and cause false notions about differences in expression.

<h5>Statistical</h5>
<p>
Spots considered significant based on a selected comparative
statistical test, and apply to replicate gels or classes.  Note that
normalization can be applied to these kinds of analyses.
<p>
The interface requires a name (a default is provided), and can contain
a description.  One only has the option of choosing Classes or Replicate
Gels.
<p>
The statistical tests available are shown in the table below:
</div> <!-- class=contentblock -->

<div class="widetable">
<table>
<tr><th>Test <th>Use/Purpose <th>Replicate Groups?  <th>Classes?  <th>Minimum Number of
Gels Per Group <th> Minimum Number of Sample Pairs  <th>Maximum Number
of Samples Per Group or Class
<tr><td>Student T
  <td>Mean values of two groups are compared, and standard deviations
  determined to make the test.
  <td> Yes <td>Yes  <td>2  <td>NA  <td>NA
<tr><td>Mann-Whitney
	 <td>Values are converted to ordered ranks and the nonparametric
	 test applied
	 <td> Yes <td>Yes <td>4   <td>NA  <td>20
<tr><td>Wilcoxon Paired-Sample Test
    <td>Tests null hypothesis that median of the paired differences
	 is zero
	 <td>No   <td>Yes  <td>6  <td>6   <td>25 (class only)fs
<tr><td>Partial Least Squares
  <td>A multivariate discriminant analysis technique with many options
  <td>Yes <td>Yes <td>2  <td>NA <td>lsk
</table>
</div> <!-- class=widetable -->

<div class="contentblock">
<p>
For the Wilcoxon, samples in a sample database must be created.
These are then assigned one of two attributes (membership in either
of two classes).  An example is <q>treated</q> and <q>untreated</q>.
The same count of samples should be in both classes, and there must be
at least 6 samples in a class.  Now the samples are <i>paired</i>.
In the Analysis Set Manager, select Statistic for type and then choose
to compare Classes to activate the Wilcoxon.

<p>
Partial Least Squares is like the Student <i>t</i> but the latter
averages spots at a single position wheras PLS looks at the spots
together.  PLS allows many options such as prefiltering noisy data
so it is not considered in the analysis.  PLS is iterative with
convergence occurring in a few iterations;  spots not found to be
significant can be removed from evaluation after an iteration.
Significance can also be set as confidence value (90%, 95%, etc.)
or as a threshold.  Spots can be scaled to ignore size differences:
this is done by standard normalization (computation of a <i>Z</i>)
value in which the value is subtracted with the mean then divided by
standard deviation.

<h5>Arbitrary</h5>
<p>
Manually selected spots, those arbitrarily chosen.  After naming and describing 
the set, one has several <b>Method</b> options from which to choose.
<p>
<b>No spots</b>. This is an empty analysis set.  Click on <b>Add spots to 
set</b> and then from the image, click on the spot.  To remove any added spots, 
click on the <b>Remove spots from set</b>.
<p>
<b>All spots in member</b>.  All spots from the member that have matched
to the Master will be added (in the case of the member being the Master,
all of its detected spots are added).
<p>
<b>Saturated spots</b>.  Only saturated spots from <em>all</em> members
of the matchset are added to the analysis set.
<p>
<b>Import list of SSPs from text file</b>.  All matchsets contain
detected spots with SSP numbers assigned to them.  If SSPs are listed
in a plain text file, separated by whitespace of any kind (space, tabs,
line returns), this list will be used to select the spots if they are
numbered in the matchsets.

<h5>Boolean</h5>
<p>
Determined from comparing two or more analysis sets:
for example, the derivative analysis set shows <q>true</q> for a
spot present in analysis sets A and B.

<h5>Matching</h5>
<p>
A spot cut list generated for spots either unique
to one member of a matchset, or present in all members of the set.


<h4>Annotation</h4>

<h4>Spot Cutting</h4>

<h5>Basic Excision Tool</h5>
<p>
The Basic Excision Tool should not be used if the gels will be treated
with digestion before mass spectrometry.  It is better to use the
Integrated Excision Tool, because the system keeps track of the gel
pieces and associated data (especially <i>M</i><sub>r</sub> and
pI information), which is not possible with the Basic Excision Tool.
<h6>Spot Cutter Setup</h6>
<p>
Call the Basic Excision dialog window.
<p>
There is a toolbar for the setup in the Setup fieldset.   This permits
several functions and activities.
<p>
<b>Focusing Cutter Camera</b>.  See if a lens filter is installed or
necessary.  Do all focusing after installing or removing a filter.
For doing focusing, the exposure time can be adjusted using either the
<b>Auto-Expose</b> or setting the <b>Time</b> in milliseconds.
The <b>Ceiling</b> shows the highest pixel value in the focused region,
and the <b>Levels</b> shows the gray scale levels in the image.  These
values can be used to adjust camera aperture and position.
<h5>Integrated Excision Tool</h5>
<p>
Before spot cutting can occur, the images require additional processing
to generate <a href="#analysis-sets"><b>Analysis Sets</b></a>.  An
analysis set is used to generate the <b>cut list</b>.  The cut list
can be used as is, or edited manually

<h5>Preparing the Gel For Cutting</h5>
<p>
Experience has instructed the members of the Proteomics Unit certain
valuable things:

<p>
<b>Remove excess water off the surface of the gel!  (but not necessarily
the sides of it)</b>.
The gel should be hydrated but should <strong>not</strong> have
excess water on its surface.  We have discovered that as the sliced gel
is lifted out, excessive water on the surface can pull the gel slice
out of the cutter hole, leaving the excised cylinder on the gel!  It
is nearly impossible to track the SSP number of the gel slice when
tens to hundreds of them are left like this.  Use the Bio-Rad ruler
and focusing card edge to gently scrape the excess water off the top
of the gel.  There is no need to blot or soak up the water from the side,
for it will help to keep the gel hydrated.  Remove the excess water
just before beginning cutting, but keep it on there if the gel sits
while any pre-cutting analysis or editing of the cut list is going on.


<h3 id="quantone">Bio-Rad's Quantity One</h3>
<p>
This software is used to image and analyze the following:
<ul>
<li>one-dimensional SDS-PAGE gels
<li>dot blots
<li>arrays
<li>colonies
</ul>
<p>
It can examine illuminated regions which are detected using radioactivty,
chemiluminesce, fluorescence, and color-stained samples from densitometers,
phosphor images, flourescent imagers, and gel doc systems.
<p>
Lane-based functions can determined molecular weights, isoelectric points,
VNTRs, presence/absence and up/down regulation of bands.  Software has
algorithms to deal with distortions (curvature/smiling of bands).
<p>
The workflow is as follows:
<ol>
<li>Acquire the image
<li>Optimize the image
<li>Peform one of the following:
 <ul>
 <li>Lane and Band Analysis
 <li>Volume Analysis
 <li>Colony Counting
 </ul>
<li>Report Results
</ol>
<p>
Alternatively the results of the optimized image can also be reported directly.

<h2>2-D Gel Pattern Analysis</h2>
<p>
[from F. Seillier-Moiseiwitsch in Walker protocols].
<p>
This provides theory for the gel image analysis logic used in MELANIE
and HERMeS.  The term I(<i>x, y</i>) will refer to the intensity of
a pixel located at coordinate <i>x, y</i> in a gel image, and the larger
the value is, the darker the pixel is.
<h3>Filtering</h3>
<p>
Convolution of the image in <i>Gaussian smoothing</i> makes use of the
operator:

<table class="eqnserif">
<col><col span="3" style="width:2em;"><col>
<tr><td rowspan="3">1/16<span style="font-size:300%;">[</span>
 <td>1  <td>2  <td>1
 <td rowspan="3"><span style="font-size:300%;">]</span>
<tr><td>2 <td>4 <td>2
<tr><td>1 <td>2 <td>1
</table>

<p>
For <i>diffusion smoothing</i>, the following operations are done:
<div class="eqnserif" style="text-align:left;margin-left:20%;">
 <i>I</i> <sup>(<i>t</i> + 1)</sup>(<i>x,y</i>) = &frac12;
   {<i>I</i> <sup>(<i>t</i> + 1)</sup>(<i>x</i> &minus; 1, <i>y</i>) +
  <i>I</i> <sup>(<i>t</i>)</sup>(<i>x</i> + 1, <i>y</i>)}
<p>
 <i>I</i> <sup>(<i>t</i> + 2)</sup>(<i>x,y</i>) = &frac12;
   {<i>I</i> <sup>(<i>t</i> + 1)</sup>(<i>x</i> &minus; 1, <i>y</i>) +
   <i>I</i> <sup>(<i>t</i> + 2)</sup>(<i>x</i> + 1, <i>y</i>)}
<p>
 <i>I</i> <sup>(<i>t</i> + 3)</sup>(<i>x,y</i>) = &frac12;
   {<i>I</i> <sup>(<i>t</i> + 3)</sup>(<i>x</i>, <i>y</i>  &minus; 1) +
   <i>I</i> <sup>(<i>t</i> + 2)</sup>(<i>x</i>, <i>y</i> + 1)}
<p>
 <i>I</i> <sup>(<i>t</i> + 4)</sup>(<i>x,y</i>) = &frac12;
   {<i>I</i> <sup>(<i>t</i> + 3)</sup>(<i>x</i>, <i>y</i>  &minus; 1) +
   <i>I</i> <sup>(<i>t</i> + 4)</sup>(<i>x</i>, <i>y</i> + 1)}
</div>
<p>
<i>Polynomial smoothing</i> is where intensities in a small area,
say 3 &times; 3 or 7 &times; 7, are approximated into a quadratic
polynomial in <i>x</i> and <i>y</i>.
<p>
<i>Adapative smoothing</i> is controlled by:
<p class="eqnserif">
<i>I</i><sup>(<i>t</i>+1)</sup>(<i>x</i>, <i>y</i>) =
(1/<i>N</i><sup>(<i>t</i>)</sup>)
&sum;<sub style="vertical-align:-1.5em;margin-left:-1.5em;"><i>i</i>=&minus;1</sub>
<sup><span style="vertical-align:4em;margin-left:-4em;">1</span></sup>
&sum;<sub style="vertical-align:-1.5em;margin-left:-1.5em;"><i>j</i>=&minus;1</sub>
<sup><span style="vertical-align:4em;margin-left:-4em;">1</span></sup>
<i>I</i><sup>(<i>t</i>)</sup>(<i>x</i> + <i>i</i>,<i>y</i> + <i>j</i>)
<i>w</i><sup>(<i>t</i>)</sup>(<i>x</i> + <i>i</i>,<i>y</i> + <i>j</i>)
<p>
<i>w</i><sup>(<i>t</i>)</sup>(<i>x,y</i>) =
exp{&minus;[<i>d</i><sup>(<i>t</i>)</sup>(<i>x,y</i>)]<sup>2</sup>/2 <i>K</i><sup>2</sup>}
<p class="eqnserif" style="text-align:left;">
<i>N</i><sup>(<i>t</i>)</sup> =
&sum;<sub style="vertical-align:-1.5em;margin-left:-1.5em;"><i>i</i>=&minus;1</sub>
<sup><span style="vertical-align:4em;margin-left:-4em;">1</span></sup>
&sum;<sub style="vertical-align:-1.5em;margin-left:-1.5em;"><i>j</i>=&minus;1</sub>
<sup><span style="vertical-align:4em;margin-left:-4em;">1</span></sup>
<i>w</i><sup>(<i>t</i>)</sup>(<i>x</i> + <i>i</i>,<i>y</i> + <i>j</i>)
<p class="eqnserif" style="text-align:left;">
<i>d</i><sup>(<i>t</i>)</sup>(<i>x,y</i>) = &radic;<i>G</i><sup>2</sup><sub><i>x</i></sub>
+ <i>G</i><sup>2</sup><sub><i>y</i></sub>, where <i>G<sub>x</sub></i> and
<i>G<sub>y</sub></i> are gradients along the <i>x</i> and <i>y</i> axis.

<h3>Spot Detection</h3>
<p>
Automatic spot detection is done using nonparametric analysis of second
derivatives or mathematical morphology.  For these sets of equations
assume <i>p</i> is the (<i>x,y</i>) coordinate on the gel, that
<i>S<sub>i</sub></i> is a spot, and <i>T</i> is the saturation threshold:
<table class="eqnserif">
<tr><td rowspan="2"><i>T</i> = max(<i>I</i>) &minus;
 <td style="border-bottom:1px solid black;">100 &minus; <i>saturation</i>
 <td rowspan="2">(max(<i>I</i>) &minus; min(<i>I</i>))
<tr><td>100
</table>
<p>
where <i>saturation</i> is a value between 0 and 100 and is 100 when no
pixel is saturated.

<!-- Beckman-Coulter PF-2D -->

<h1 id="pf2d">Protein Fractionation Two-Dimensional (PF-2D or PF2D)</h1>
<p>
The PF-2D system was developed by Beckman-Coulter together with a group
from a Michigan university (Lubman's group) and including a company called
Eprogen, Inc., a manufacturer of non-porous silica HPLC columns.
<p>
The information in this section is too little to describe all that there
is about the system.  The idea is to provide just enough information
to get the system running.  Jim Thorn&rsquo;s very detailed
<a href="http://80.251.43.3:10000/Proteomics/PF2D/AppTip7-PF2D-Checklist.pdf">checklist</a>
should be used (printed out) at all times for setting up a run in general.
Each run costs hundreds of dollars and hours of time (especially on the
2nd dimension), so getting the setup right is extremely crucial.
The checklist might be at
<a href="http://80.251.43.3:10000/Proteomics/PF2D/AppTip7-PF2D-Checklist.pdf">this web link</a>
should you want it.   If you want an editable copy of the checklist, try
this
<a href="http://80.251.43.3:10000/Proteomics/PF2D/PF%202D%20AppTip%237_PF2D%20checklist_v6_Europe.doc">link</a>.
<p>
For an illustrated explanation of the PF2D system, try
<a href="http://80.251.43.3:10000/Proteomics/PF2D/Beckman-CoulterPF2DTraining/">document</a>.

<h2>Sample Preparation</h2>
<p>
The protein sample has to be prepared to a sufficient concentration
and be assayed to determine that concentration.  This determines the final
volume.  There is a maximum limit of 5 milligrams (5 mg) for loading on to the
column.  The minimum limit is 0.1 mg in order to find detectable material.

<h3>Lysis Buffers</h3>
<p>
Beckman-Coulter recommends the following buffer as an extraction or lysis
buffer for cells or tissues:
<ul>
<li>7.5 M urea
<li>2.5 M thiourea
<li>12.5% glycerol
<li>62.5 mM Tris HCl
<li>2.5% <i>n</i>-octylglucoside (&alpha;-octyl &alpha;-D-glucopyranoside)
<li>1.25 mM inhibitors
</ul>
<p>
The chemicals can be added to a 12.5% glycerol solution in the preparation.

<h3>Protein Concentration Assay Using BCA Assay</h3>
<p>
The micro BCA assay (sold only by Pierce Biotechnology, Inc.) is
used to analyze the sample that is injected.
The reason this assay is used is because the sample is diluted within
the Start buffer, and there are a number of materials that can interfere
with other assays (such as the Bradford).  Beckman-Coulter does not
recommend assaying protein before the sample is exchanged in this solvent
system because it makes too many assumptions about the effects of
manipulation.  Nonetheless, if the BCA assay is not available, use the
Bradford and make assumptions if necessary about the solution.
<p>
In general, see the <a href="#bca">BCA section</a> of this reference
for a description of setting up the assay.
<p>
But see the notes below pertaining to the assay in use with the PF2D.

<h4>BSA Standards</h4>
<p>
Prepare all BSA standards in a diluent of 10% Start buffer and 90% PBS or
TBS (this will be called <b>diluent</b> from now on).  It is best to
dissolve assay-grade BSA directly in diluent, but if there is a stock
in 10 or more mg/ml concentration, it can be diluted from that as well.
Prepare or obtain a 10 mg/ml stock of BSA (dissolved in saline, water,
or diluent).  Aliquot it to 100-200 &micro;l for a few aliquots.
Standards can be prepared that will be good for 4-5 freeze/thaw cycles.
After 5 or so freeze-thaw cycles, prepare new standards from 10 mg/ml
stock.
<p>
Prepare 500 &micro;l of the standards in either 0.5 or 1.5 ml conical tubes.
The range for the micro BCA assay is from 2.5 to 40 &micro;g/ml.
An assay from 5 to 7 points is useful.  Suppose a 6-point curve:
in this case, standards of 2.5, 5, 10, 20, 30, and 40 &micro;g/ml
are useful.   Intermediate dilutions will be necessary from the 10 mg/ml
stock.  Since this is equivalent to 10,000 &micro;g/ml, to get to
40 &micro;g/ml requires a 1/250 dilution.  The best approach is to move
in no more than 1/10 or 1/20 dilution steps, if dilutions are to be done
with as minimal volumes as possible.  The approach here is to get down
to a 100 &micro;g/ml as a working dilution, and at least 1 ml will be enough.
First do a 1/10 dilution to get to 1 mg/ml:  add 50 &micro;l of 10 mg/ml BSA
to 450 &micro;l of PBS (best) or ultrapure water.  Take 150 &micro;l
of the 1 mg/ml intermediate dilution and add it to 1.35 ml (2 &times;
675 &micro;l) of diluent.  The diluent should now be 90% PBS and
10% Start buffer, all dilutions will be done with this.
There is now 2000 &micro;l (2 ml) of 0.1 mg BSA/ml (or 100 &micro;g/ml BSA).
<p>
Assume that a spectrophotometer with a semi-micro cuvet is used.  A
0.5 ml volume is usually a comfortable volume to use, but it depends on
the spectrophotometer, and sometimes as little as 0.2 ml (200 &micro;l)
can be used.  Thus the standards will be prepared in an amount to allow
at least 5 uses from the tube (4-5 freeze-thaw cycles).  Since the standard
is mixed with 1 volume of standard with 1 volume of working reagent, a
250 &micro;l volume is necessary for the standard for a 0.5 ml cuvet volume.
Making a 1.5 ml (instead of 5 &times; 250 &micro;l = 1.25 ml) will cover
all possibilities, including pipetting error.
<p>
To make 1.5 ml of each standard:
<ul>
<li>Pipet 600 &micro;l of the 100 &micro;g/ml to a 1.5 ml tube.  Add
900 &micro;l of diluent.  Mark this as the 40 &micro;g/ml BSA standard.
<li>Pipet 450 &micro;l of the 100 &micro;g/ml to a 1.5 ml tube.  Add
1050 &micro;l (2 &times; 525 &micro;l) to the tube and mark as 30 &micro;g/ml
BSA standard.
<li>Pipet 300 &micro;l of the 100 &micro;g/ml to 1.5 ml tube and add
1200 &micro;l (2 &times; 600 &micro;l) to tube and mark as 20 &micro;g/ml
BSA standard.
<li>Pipet 150 &micro;l of the 100 &micro;g/ml to 1.5 ml tube and add
1350 &micro;l (2 &times; 675 &micro;l) to tube and mark as 10 &micro;g/ml
BSA standard.
<li>Pipet  75 &micro;l of the 100 &micro;g/ml to 1.5 ml tube and add
1425 &micro;l (2 &times; 712.5 &micro;l) to tube and mark as 5 &micro;g/ml
BSA standard.
<li>Pipet 37.5 &micro;l of the 100 &micro;g/ml to 1.5 ml tube and add
1464 &micro;l (2 &times; 732 &micro;l) to tube and mark as 2.5 &micro;g/ml
BSA standard.
</ul>
<p>
Use and store at &minus;20&deg;.  Again limit the number of freeze-thaw
cycles.

<h4>Dilutions of Unknowns</h4>
<p>
It is good to have a reasonable estimate about the concentration of the
sample ahead of time.  For body fluids, this is fairly well known for
persons in a healthy (and sometimes diseased) condition.  If not sure
about the concentration, do a 1/5 and 1/50 to be on the safe side.  One
of the dilutions is bound to be within the range of the standard curve.
Duplicates of both dilutions will make the reading more sure.

<h4>Doing The Assay</h4>
<p>
Once the standards are ready and the pre-dilution of unknowns to get in
range, the rest is easy.  It is just a 1:1 v/v mix of the material to
be assayed with the working reagent.  The working reagent is a mix
of three stocks supplied by Pierce.  Follow the direction in making the
working reagent.  If the final cuvet volume is to be 0.5 ml, then the
total volume of the working reagent will have to be 250 &micro;l times;
the total number of tubes.  Add 15% on top of that for customary pipetting
error.  The assay needs cooking (at 60&deg;) for an hour, then a cooling
down period before reading at 562 nm.

<h2>Pre-Run Instrument &amp; Column Setup</h2>
<p>
The columns need to be washed of their storage buffers.  The safest
thing to do is to wash columns in ultrapure (HPLC grade or better)
water.  Never start putting salt- or solute-containing buffers in a
column in which you do not know what it is stored in:  salts or solutes
might precipitate out inside the column, causing blockage or other damage.
Start a standard flow rate (0.2 ml/min on the HPCF column, 0.75 ml/min
on the HPRP column, both with gradual pressure increase or duration = 0.5 min)
using the water, and flush at least 20 column volumes.

<h3>HPCF Column</h3>
<p>
The HPCF column normally operates in a pressure range of 0.5-1.2 kpsi.
The pressure usually goes up with equilibration, and it falls when a
high ionic strength eluent is used.  If the pressure goes above 2.0 kpsi,
the column will require special attention to unplug it.  A backflush
(backflow:  that is, connect the outlet end to the inlet end) with
a solvent of <i>n</i>-propanol at 0.2 ml/min for 1 hour may help.

<h3>HPRP Column</h3>
<p>
The HPRP column normally operates at a pressure of 1.8-2.2 kpsi.
If the pressure goes above 3.5 kpsi, special attention may be required
to recondition or clean the column.
<p>
First try using a solvent of <i>n</i>-propanol containing 0.1% TFA, at
the usual flow of 0.75 ml for 30 min.  Check the pressure in equilibration
buffer again.
<p>
If the pressure remains high, then try the following steps in this order,
for 15-20 min each:
<ol>
<li>100% MeCN
<li>75% MeCN
<li>25% isopropanol
<li>100% isopropanol
<li>100% methylene chloride (CH<sub>2</sub>Cl<sub>2</sub>)
<li>100% isopropanol
<li>100% MeCN
<li>100% tetrahydrofuran (THF)
</ol>

<h2>The Beckman-Coulter Checklist</h2>
<p>
Print out the checklist for performing a run, and follow it strictly.
The Start and Elution buffers are quite expensive and must be handled
properly.  Note that they are light-sensitive (which is why they are stored
in dark bottles).
<p>
When adjusting the pH of the buffers at the start of a run, do not use the
automatic temperature compensation fetaure (or set it at 25&deg;).
<p>
The use of aged or mishandled buffer may cause an improper pH gradient
during the run.  Be prepared for that possibility.


<h2>Post-Run System and Column Handling / Storage</h2>
<p>
After a run, the HPCF column can be kept stored in the water that was
used to wash it after the run.  The HPRP column can sit in the 0.08% TFA
in acetonitrile (that is, the solvent in solvent bottle B).
<p>
<b>Long-Term Storage</b>.  Jim Thorn of Beckman-Coulter has said that
any time a column will go for more than 2 days (and that includes over
the weekend), it needs to be put in long-term storage buffer.
<p>
For the HPCF column, that means you need to wash in 10% <em>HPLC-grade</em>
isopropanol (2-propanol).  Note that isopropanol can and often does
come in less than HPLC-grade, so don't just use any!  In a pinch, you
can use less than HPLC grade isopropanol, but try to filter it with a
0.2 or 0.45 &micro;m filter.
<p>
For the HPRP column, use 100% isopropanol after washing it out with
HPLC grade water first.

<!--  PROTEIN PURIFICATION -->

<h1 id="proteinpure">Protein Purification</h1>

<h2 id="ultracentrifugation">Ultracentrifugation</h2>
<p style="font-size:75%;">
[this material extracted from Voet &amp; Voet, <i>Biochemistry</i>,
2<sup>nd</sup> Ed., (New York: Wiley &amp; Sons), 1995].
<p>
The force <i>F</i> on a particle in a centrifugal field is related to its
mass <i>m</i> and the radial distance <i>r</i>  from the center of
the field and to the angular velocity <i>&omega;</i>, less the force
exerted in resistance against it by the bouyant density of the solution:
<p class="eqnserif" style="text-align:center">
 <i>F = m&omega;<sup>2</sup>r &minus; V<sub>p</sub>&rho;&omega; <sup>2</sup>r</i>
<p>
where <i>V<sub>p</sub></i> is the particle volume and <i>&rho;</i>
is the solution density.  In general the motion of a particle is
resisted by a frictional force, <i>F<sub>friction</sub></i>, and this
force is a function of the particle&rsquo;s velocity:
<p class="eqnserif" style="text-align:center;">
 <i>F<sub>friction</sub> = vf</i>
<p>
where <i>v</i> = <i>dr/dt</i>, the rate of migration of the sedimenting
particle, and <i>f</i> is a <i>frictional coefficient</i>.
The forces which make a particle move in a centrifugal field will balance:
<p class="eqnserif" style="text-align:center">
 <i>m&omega;<sup>2</sup>r &minus; V<sub>p</sub>&rho;&omega; <sup>2</sup>r = fv</i>
<p>
The partial specific volume, <i style="text-decoration:overline;">V</i>,
of a particle is the change in volume when 1 gram of particles is dissolved
in an infinite volume of solvent.  For proteins in pure water, it is
0.73 cm<sup>3</sup>/g at 20&deg;.  The particle volume <i>V<sub>p</sub></i>
is a product of the partial specific volume and the particle mass, and the
particle mass is related to the molar mass.
<p class="eqnserif" style="text-align:center">
 <i>V<sub>p</sub> = <span style="text-decoration:overline;">V</span>m =
	<span style="text-decoration:overline;">V</span>M/N</i>
<p>
where <i>m</i> is the particle mass (dimension: gram/particle),
<i>M</i> is the molar mass (dimension: gram/mol), and <i>N</i> is
Avogadro's number (dimension: particles/mol).
<p>
For proteins, the partial specific volumes are generally close approximations
of the sum of the amino acid <i style="text-decoration:overline;">V</i>
that compose them, and indication of close packing of the atoms.  Substitute
the last two equations into the force equation above:
<p style="text-align:center;">
<table class="eqnserif">
<tr><td rowspan="2"><i>vf =</i>
<td style="border-bottom:1px solid black;"><i>M</i>(1 &minus;
   <i><span style="text-decoration:overline;">V</span>&rho;</i>)<i>&omega;</i><sup>2</sup><i>r</i>
<tr><td><i>N</i>
</table>
<h3>Sedimentation Coefficient</h3>
<p>
The quantity <i>s</i>, the <i>sedimentation coefficient</i>, is
defined as a measure of the mobility of a particle in a centrifugal field
(much as &mu; is a measure of the electrophoretic mobility of a charged particle
in voltage field).
<p style="text-align:center;">
<table class="eqnserif">
<tr><td rowspan="2"><i>s =</i>
 <td style="border-bottom:1px solid black;"><i>v</i>
 <td rowspan="2"> =
 <td style="border-bottom:1px solid black;"> 1
 <td rowspan="2" style="font-size:200%;"> (
 <td style="border-bottom:1px solid black;"><i>d</i> ln <i>r</i>
 <td rowspan="2"> <span style="font-size:200%;">)</span> =
 <td style="border-bottom:1px solid black;"><i>M</i>(1 &minus;
     <i><span style="text-decoration:overline;">V</span>&rho;</i>)
<tr><td><i>&omega;<sup>2</sup>r</i>
 <td><i>&omega;<sup>2</sup></i>
 <td><i>dt</i>
 <td><i>Nf</i>
</table>


<hr>
<p>
This reference is divided into multiple parts.
<p>

<a href="ProteomicsReference2.html">Part 2</a>
has material mostly on mass spectrometry-related proteomics.

</div> <!-- class=contentblock -->


</div> <!-- id=scrolling-panel -->

</body>
</html>
