<!DOCTYPE html>
<html>
<head>
<title>Reference for Proteomics</title>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8">

<link href="/style/std.css" type="text/css" rel="stylesheet">
<link href="/style/proteomicsRef.css" type="text/css" rel="stylesheet">
<style type="text/css">
 	div.email { white-space:pre;font:normal 83% 'Courier New',Courier,monospace; 
			border:1px solid black;background-color:white;padding:0.5em 1em;margin:1em -2em; }
	#motd {margin:0 auto;width:40em;font-size:80%;}
</style>
<script type="text/javascript" src="/include/colorContrasting.js"></script>
<script type="text/javascript" src="/include/dom-xml.js"></script>
<script type="text/javascript" src="/include/css.js"></script>
<script type="text/javascript" src="/include/htmltocgenerator.js"></script>
<script type="text/javascript">
function calcAmmSO4(buttonObj) {
	var i;
	var startConc = document.getElementById("startAmmSO4").value;
	var finalConc = document.getElementById("endAmmSO4").value;
	if (isNaN(startConc) == true || startConc < 0 || startConc > 100)
		return (AmError("Starting concentration must be a number between 0 and 100%"));
	if (isNaN(finalConc) == true || finalConc < 0 || finalConc > 100)
		return (AmError("Final concentration must be a number between 0 and 100%"));
	if (finalConc < startConc)
		return (AmError("The final ammonium sulfate concentration must be" +
			"greater than the starting concentration"));
	var vol = document.getElementById("AmmSO4volume").value;
	if (isNaN(vol) == true || vol <= 0)
		return(AmError("The volume must be a number greater than 0"));
	var inpNodes = document.getElementsByTagName("input");
	for (i = 0; i < inpNodes.length; i++)
		if (inpNodes[i].name && inpNodes[i].name == "AmmSO4temp" &&
				inpNodes[i].checked == true)
			break;
	var temp = inpNodes[i].value;
	var satnAtTemp = [
		{ temp: 0, satn: 505, sv: 0.3 }, { temp: 10, satn: 525, sv: 0.3 },
		{ temp: 20, satn: 533, sv: 0.3 }, {temp: 25, satn: 540, sv: 0.3 }
	];
	for (i = 0; i < satnAtTemp.length; i++)
		if (temp == satnAtTemp[i].temp)
			break;
	var satn = satnAtTemp[i].satn;
	var domObj = document.getElementById("AmmSO4result");
	domObj.replaceChild(document.createTextNode((vol / 1000.0 * satn *
		((finalConc - startConc) /
		(100 - satnAtTemp[i].sv * finalConc))).toFixed(1) + " g"),
		domObj.firstChild);
	document.getElementById("AmmSO4error").style.display = "none";
}
function AmError(msg) {
	var domObj = document.getElementById("AmmSO4text");
	domObj.replaceChild(document.createTextNode(msg), domObj.firstChild);
	document.getElementById("AmmSO4error").style.display = "inline";
}
</script>
<script type="text/javascript" src="/include/natSciDocs.js"></script>
<script type="text/javascript" src="/include/xmlTextImport.js"></script>
</head>
<body onload="initNatSciDoc();">
<div id="top-panel">
<p id="index-button-left">
<button type="button" onclick="showIndex('asWindow');">Show index in window</button>
<p id="index-button-right">
<button type="button" onclick="showIndex('embedded');">Show index embedded in document</button>
<p id="contact-author" style="display:none;">
<a class="button" href="mailto:mavigozler@yahoo.com?Subject=HtmlToc%20Object%20Not%20Found">Contact
Document Author</a>
<p>
<!-- motd -->
<p id="title">
Reference for Proteomics</p>

<p id="subtitle">
Part 2 (of 2)
</div>

<div class="contentblock" id="first-content-block">
This reference is divided into multiple parts.  To the left is a clickable
document outline for the current part.  Unfortunately a table of contents
for all parts has not been set up on a single page (yet, but will be).
<p>
Currently there are two parts.  <a href="ProteomicsReference1.html">Part 1</a>
covers a lot of chemicals used in proteomics as well as one- and two-dimensional
gel electrophoresis.


<hr>
<!-- putTOChere -->



<!--  SAMPLE PREPRATION -->

<h1 id="sampleprep">Sample Preparation</h1>
<p>
Sample preparation is a set of generally useful methods.  They are methods
that generally are used in the preparation of a sample for some later
analytical step.
<p>
Some samples may be too dilute for use.  For MALDI- and ESI-MS work,
concentrations of peptides or proteins should be in the range of 0.1
to 10 pmol per microliter (0.1 - 10 &micro;M).


<h2 id="concprotein">Concentrating/Dialyzing Protein-Containing Solutions</h2>

<h3 id="ammsulfpptn">Ammonium Sulfate Precipitation</h3>
<p>
Ammonium sulfate is an old technique for precipitating proteins (not
peptides) both as a purification step primarily, but also as a concentration
step as well.	It is typically used just after extraction of cells
or tissues, and prior to high resolution separation steps such as
chromatography.  Inclusion of EDTA (from 0.1-0.5 mM) may be desirable
when using ammonium sulfate since it tends to coordinate heavy
metals which may be detrimental to preparative work.
<p>
An ammonium sulfate calculator is provided below:

<table id="ammoniumsulfate-calculator">
<tr><th>Temperature:
 <td><input type="radio" name="AmmSO4temp" value="0" checked> &ensp;0&deg; C<br>
  <input type="radio" name="AmmSO4temp" value="10"> 10&deg; C<br>
  <input type="radio" name="AmmSO4temp" value="20"> 20&deg; C<br>
  <input type="radio" name="AmmSO4temp" value="25"> 25&deg; C<br>
<tr><th>Volume of solution:
  <td><input size="6" id="AmmSO4volume"> milliliters (mL)
<tr><th>Starting ammonium sulfate concentration:
  <td><input size="6" id="startAmmSO4"> % saturation
<tr><th>Final desired ammonium sulfate concentration:
  <td><input size="6" id="endAmmSO4"> % saturation
<tr><th><button onclick="calcAmmSO4(this);">Calculate</button>
 <td style="background-color:#faf;color:green;
   padding:0;font:130% bold Helvetica,Arial,sans-serif;">
 &emsp;<span id="AmmSO4result">&nbsp;</span>
<tr><td colspan="2">
 <span id="AmmSO4error" style="display:none;"><br>
  <span id="AmmSO4text">&nbsp;</span></span>
</table>

<h3>Ultrafiltration</h3>
<p>
Ultrafiltration is the separation of solutes (proteins) from molecules
smaller than them (low MW molecules, salts, solvent) by a process of
filtration through a porous membrane whose pores are sized to approximate
molecules of a particular weight (assuming molecules whose space approximates
a sphere).  The filtration process is achieved usually be applying a force
on top of the membrane to the solution.  The force might be the pressure of
a nonreactive (usually nitrogen) gas, or it might come from the increased
gravities in a centrifugal field.  Ultrafiltration achieves an increased
concentration of all solutes held by the membrane barrier.  Moreover, it can
be used to achieve a solvent/buffer exchange with the addition of an alternative
buffer or solvent, and then its re-concentration.

<h4 id="centultrafilt">Centrifugal Ultrafiltration</h4>
<p>
Centrifugal ultrafiltration is the preferred step for both buffer
exchange (dialysis) and concentration of both proteins and peptides longer
than 15 amino acids.  It works at all scales, but particularly well
for the nano- to microvolumes used in sample preparation for proteomic
analysis of all types, including protein identification and quantitative
proteomics, and particularly for intact protein proteome profiling.
<p>
Millipore owns the Amicon/Centricon name, and supplies well known
products in filtration and ultrafiltration, including centrifugal units.
Sartorius owns the Vivascience line of products, and Sartorius has a
reputed name in filtration;  they offer innovative products in ultrafiltration,
including super-low volume retentate systems and filtration whose force
vector is angled to the membrane, which improves flow properties.

<h3>Centrifugal Evaporation</h3>
<p>
Centrifugal evaporation places a sample in a spinning rotor at a fixed
speed and then applies a vacuum to the chamber.  The application of the
vacuum accelerates the process of evaporation of any solvent, and nonvolatile
solutes remain even after reaching dryness.  Many units will heat up the
chamber containing the spinning rotor to 50-60&deg;; this is particularly
effect to improve the rate of evaporation of solvents that are largely
aqueous (water), although the application of heat to that temperature will
depend also on the sensitivity of the sample to that kind of heat.
<p>
This description is of the classical operation of the Savant SpeedVac
product.  Centrifugal evaporation must be the capability of any proteomics
laboratory, and should also include carrier rotors that hold standard
volume microtubes as well as 96- and 384-multiwell plates.

<h3 id="tcapptn">Trichloroacetic Acid Precipitation</h3>
<p>
The TCA precipitation is okay if the sample will continue on to some
denaturing process.   If the sample contains detergent, TCA precipitation
may not be the best way to proceed however:  as an alternative, use 5-10
volumes of ice-cold acetone to precipitate, especially if the sample is
to be separated by 2DGE.
<p>
This is adapted from Sigma-Aldrich protein precipitation kit protocol.
<p>
This method works well for sample masses of protein down to 30 &micro;g.
Add sodium deoxycholate for sample masses of protein down to 3 &micro;g.
<p>
The following reagents should be prepared:
<ul>
<li>100% (w/v) trichloroacetic acid (TCA)
<li>0.2% sodium deoxycholate (NaDOC)
<li>25% acetone, ice-cold
</ul>
<p>
Store the TCA and NaDOC at room temperature.  Put the 25% acetone at
&minus;20&deg; at least an hour before use.

<h4>Sample containing protein down to mass of 30 &micro;g</h4>
<ol>
<li>Add the sample to a 1.5 ml tube.
<li>Bring the volume up to 1.0 ml using ultrapure water.
<li>Add 100  &micro;l of 100% TCA.
<li>Vortex and leave on ice bath for at least 15 min
<li>Spin at 15,000 <i>g</i> (top speed in microcentrifuge) for 10 min at room temp.
<li>Discard the supernatant.
<li>If interfering substances are to be removed, add 1 ml of ice-cold
25% acetone wash, vortex to disperse pellet, then spin at top speed for
5 min at room temp. Discard supernatant.
<li>Dry the pellet in a SpeedVac for 10-20 min to remove residual solvent.
An alternative is to air-dry the pellet for 20 min or so.
</ol>
<p>
Resuspend the pellet in suitable buffer depending on application.
<table>
<tr><th>Application            <th>Buffer/Reagent
<tr><td>SDS-PAGE               <td>1&times; Laemmli sample buffer
<tr><td>BCA/Lowry Protein Assay <td>40 mM sodium carbonate, pH 11.25
<tr><td>Isoelectric Focusing   <td>IEF sample buffer containing urea, thiourea
 Tris base, zwitterionic detergent (ASB-14, SB3-10, C7BzO, CHAPS)
<tr><td>Solution trypsin digestion <td>100 mM ammonium bicarbonate, pH 8.5
</table>
<p>
A Bradford assay is not recommended.

<h4>Sample containing protein down to mass of 3 &micro;g</h4>
<p>
This method is better if the protein-containing solution has an ionic
strength of at least 10 mM (add NaCl to bring the solution to at least
10 mM if ionic strength is unknown).
<ol>
<li>Put 100 &micro;l sample in 1.5 ml tube.  Add 10 mM NaCl if necessary
to make the volume 100 &micro;l.
<li>Add 900 &micro;l of the 0.2% NaDOC
<li>Vortex and let sit at room temperature for 10 min
<li>Add 100 &micro;l of 100% TCA.
<li>Vortex and let sit on an ice bath for at least 15 min
<li>Spin at 15,000 <i>g</i> (top speed in microcentrifuge) for 10 min at room temp.
For larger samples, increase the time proportionally.
<li>Discard the supernatant.
<li>Add 1 ml of ice-cold 25% acetone wash.  Vortex to disperse pellet.
<li>Spin at top speed for 5 min at room temp.  Remove supernatant
<li>Repeat the acetone wash step.
<li>Dry in centrifugal evaporator for 10-20 min, or alternatively, let
air dry with open tube for a bit longer.
</ol>
<p>
To resuspend the pellet, gentle heating and vortexing might be required.

<h3>Other Precipitation Techniques</h3>
<p>
Many proteins are soluble in the environment in which they are found.
In tissues, there is usually an ionic strength of 0.15-0.2 M.  The salts
present in this environment are important in maintaining solubility.
If the concentration is lowered (such as by dilution), many proteins
will become insoluble as they begin to aggregate with one another.
This is true of serum globulins when serum is diluted.  Proteins in
low ionic strength environment and near their isoelectric point are
no longer repelled by one another, making an aggregation and precipitation
possible.
<p>
Use of organic solvents and polyethylene glycol and many divalent metal
ions (50 mM Mg<sup>2+</sup>) can result in precipitation;  the metal
salt of a protein will have an higher isoelectric point because of the
divalent cation so a higher pH results in aggregation.

<h2 id="trypsininfo">Trypsin in Proteomics</h2>
<p>
In many methods here, a target protein, that is, a protein targeted for
digestion, is digested with the most commonly used proteolytic enzyme,
<b>trypsin</b>.  There are many reasons trypsin is used compared to
all other proteases.  Firstly, it produces a nice distribution of
peptides with different masses, which aids in peptide mass fingerprinting
and calibration of mass spectrometers tremendously.  It is also
one of the most specific of proteases, hydrolyzing the peptide bond
on the carboxyl side (-COOH) of arginines (R) and lysines (K), with
some exceptions.  The exceptions are:
<ul>
<li>Trypsin cannot hydrolyze R or K residues when the carboxyl (COOH)
side (or next) residue is proline (P).  This likely has to do with
chemistry as much as steric hindrance.
<li>Trypsin will not cut R or K residues that are surrounded by
negatively charged residues.  This includes glutamate (E), aspartate (D),
and also phosphorylated residues such as threonines (T-p) and serines
(S-p).  For instance, the partial sequence -DKEE- has been observed
not to be cut by trypsin, probably because of the multiple negative
charges surrounding (masking?) the positive charge on the trypsin.
</ul>
<p>
Trypsin is usually stored in a pH 4 solution of acetate until it is
mixed with a buffer that activates it at pH 8.  If it is not stored
in this way, trypsin will inactivate itself by digesting itself
(auto-digestion).
<p>
Trypsin is used in different weight/mass ratios to its substrate, depending
upon whether a complete and rapid digestion is desired.  For a complete
and rapid digestion, a higher trypsin:substrate ratio is used, but
the drawback is that peptides of auto-digested trypsin themselves will
produce a signal in the mass spectrometer that competes with the peptides
of the target protein itself (that is, peptides from autolytic trypsin
appear in the mass spectrum too).
<p>
In general, a high grade trypsin should be used for proteomics work.
Trypsin is usually treated with
<i>N-p</i>-Tosyl-<span style="font-variant:small-caps;">L</span>-phenylalanine
chloromethyl ketone (TPCK) to inactivate chymotrypsin, a very common
contaminant present in the source extracts from which trypsin is purified
for use as a reagent.
<p>
Sigma-Aldrich sells (Cat #T6567) a &ldquo;proteomics grade&rdquo; of
dimethylated trypsin purified from porcine pancreas.  Its Lys residues
have been reductively methylated to make it resistant to autoproteolysis.
It is also treated with TPCK to inactivate contaminating chymotrypsin.
It is then affinity purified.

<h2 id="solntrypdigest">Solution Digestion with Trypsin</h2>
<p>
First <a href="#labbasics">be familiar</a>
with clean, efficient, and accurate ways of preparing stock solutions.
<p>
If preparing a set of peptides to be used as mass spectrometry calibration
standards, then go to <a href="#tryptcalib">this section</a>.
<p>

The following reagents will be required:

<ul>
<li><b>&ldquo;Proteomics Grade&rdquo; water</b>
 <br>Type I or HPLC grade water filtered to at least 0.45 &micro;m is
  sufficient.	Distilled water will do if it is the only thing available.
  Wherever the word &ldquo;water&rdquo; is used below, it refers to
  &ldquo;proteomics grade&rdquo;.
<li><b>250 mM ammonium bicarbonate (NH<sub>4</sub>HCO<sub>3</sub>)</b>
 <br>Prepare as follows:
 <ol>
 <li>To a 50 ml container (plastic tube),
    add 0.988 g solid NH<sub>4</sub>HCO<sub>3</sub>.
 <li>Add 30-40 ml water and dissolve it by shaking or stirring.
 <li>QS to 50 ml with water if the graduations are true in the
 plastic container; or use a absolutely clean 50 or 100 ml glass
 graduated cylinder  and transfer back to container.
 </ol>
<li><b>100 mM dithiothreitol</b> (DTT, FW 154.25, CAS#&nbsp;3483-12-3)
 <br>Prepare as follows:
 <ol>
 <li>Get a 15 ml container (such as 15 ml plastic conical tube), and
 add 154 mg DTT to it.
 <li>Add to it 10.0 ml of water if the graduations on the tube are
trustworthy, or add 2 &times; 5.0 ml water from a P-5000.
 <li>Dissolve and mix on vortex or by inversion.
 </ol>
<li><b>10 mM DTT in 50 mM ammonium bicarbonate</b>
 <br>Prepare only what is necessary as follows:
 <ol>
 <li>Calculate the number of tubes or wells to which you will add DTT.
Multiply the volume of the tubes/wells and the count of tubes/wells, and
add a small amount (15% is usual) for pipetting error.  Example:  suppose
there are 48 wells to be treated and 20 &micro;l is to be added to each
well.  The calculated volume necessary is
(48 wells &times; 20 &micro;l/well &times; 1.15 = 1104 &micro;l = 1.104 ml.
Make 1.1 or 1.2 ml to make it easy.
 <li>With a 100 mM DTT stock, a 1/10 dilution is necessary to make 10 mM
 DTT.  With a 250 mM ammonium bicarbonate stock, a 1/5 dilution is necessary
 to make 50 mM ammonium bicarbonate.  If 1.2 ml of working solution is
 to be made, then 1.2 ml / 10 = 120 &micro;l and 1.2 ml / 5 is 240 &micro;l.
 <li>Into a proper tube (1.5, 2 ml, etc.), pipet the DTT stock.  In the
 example this is 120 &micro;l of 100 mM DTT stock.
 <li>Into the tube, now pipet the 250 mM ammonium bicarbonate stock.  In
 the example this is 240 &micro;l.
 <li>Calculate the difference between the total volume necessary and
 the volumes of stock pipetted, and pipet this amount as water.
 In the example, and since 1.2 ml = 1200 &micro;l, this is
 1200 &minus; (120 + 240) &micro;l = 840 &micro;l water.
 </ol>
<li><b>55 mM iodoacetamide in 50 mM ammonium bicarbonate</b>
 <br>This <em>must</em> be prepared fresh before use.
 <ol>
 <li>First determine the volume of reagent necessary by calculating the
 volume per well or tube with the count of the wells or tubes, as described
 for 10 mM DTT in ammonium bicarbonate.  In an example, suppose that
 15 &micro;l/well or tube is to be added and there are 50 wells.
 The calculated necessary volume is (15 &micro;l/well &times; 50 wells)
 &times; 1.15 = 863 &micro;l, which can be rounded off to 900 &micro;l.
 <li>Find the right container to hold the volume calculated.  Tare the
 container on a balance.
 <li>Weigh out directly into the container an approximately necessary amount
 of iodoacetamide.  In the example, the required amount of iodoacetamide
 is calculated to be:  900 &micro;L &times; (55 mmol / L) &times;
 (1 L / 10<sup>6</sup> &micro;L) &times; (184.96 mg / mmol) = 9.155 mg.
 Suppose that 11.89 mg is weighed out instead.  As long as this is
 close to 9.155 mg, it is not important to be exact.
 <li>Now re-calculate the total volume to be made of the working solution
 based on the iodoacetamide weighed out:
 <p style="text-align:center;">
<table class="eqnserif">
<tr><td rowspan="2">11.89 mg &times;
 <td class="numer">mmol
 <td rowspan="2">&times;
 <td class="numer">L
 <td rowspan="2">&times;
 <td class="numer">10<sup>6</sup> &micro;L
 <td rowspan="2"> = 1168.8 &micro;l
<tr><td>184.96 mg
 <td>55 mmol
 <td>L
</table>
<p>
Assume the volume is 1170 &micro;l.  As long as the re-computed volume does
not exceed the container volume, this will be acceptable.  Otherwise
it will be necessary to wash the iodoacetamide in the container into
a larger container.
 <li>Since the stock of ammonium bicarbonate is 250 mM and the final concentration
in the working dilution is 50 mM, the dilution must be 1/5.  Divide the
final volume of the working dilution by 5 to get the volume of 250 mM
NH<sub>4</sub>CO<sub>3</sub> to add.  In the example, this is
1170 &micro;l / 5 = 234 &micro;l.  Add 234 &micro;l to the tube.
If it is necessary to transfer to a larger container, wait for the iodoacetamide
to dissolve and then transfer.
 <li>The balance to be added is water.  It is the difference between the
total recomputed volume and the volume of ammonium bicarbonate stock added.
In the example, this is 1170 &minus; 234 = 936 &micro;l water.  Add this
and mix well.
 </ol>
<li><b>100 ng/&micro;l sequencing grade modified trypsin</b>
 <br>Do not make use of regular trypsin.  The following trypsin products
 are discussed:
 <ul>
 <li><b>Promega V5111</b>.  This modified porcine trypsin comes as
a 20 &micro;g lyophilized trypsin in vial.  Promega recommends
using a trypsin:protein ratio of 1:100 to 1:20 (w/w).  Promega
supplies 50 mM acetic acid as <b>Trypsin Resuspension Buffer</b>.
A 100 &micro;g/ml trypsin solution will have 85% of its activity
after 3 hours at 37&deg; in 50 mM ammonium bicarbonate.  It can
resist mildly denaturing conditions such as 0.1% SDS, 1 M urea,
and 10% acetonitrile; it retains half of its activity in 2 M guandine HCl.
 <br>Add 200 &micro;l of Promega's Trypsin Resuspension Buffer to produce
a 100 ng/&micro;l trypsin stock.  For any unused trypsin stock, refreeze
at &minus;20&deg; (and avoid multiple freeze-thaw cycles).
 <li><b>Sigma T6567</b>.  This is supplied as a 20 &micro;g in vial.
 <br><b>Solution Digests</b>.  Trypsin is reconstituted
with 20 &micro;l of 1 mM HCl to produce a 1 &micro;g/&micro;l
solution at pH ~3.  Dissolve protein/peptide subtrate in 0.1 M
ammonium bicarbonate or Tris-HCl pH 8.5 (note Tris-HCl cannot be
used for MALDI).  Prior reductive alkylation of the protein is recommended.
Add trypsin to substrate in recommended amounts, and incubate between
2-18 hours at 37&deg; (the time depends on trypsin:protein ratio).
 <br><b>In-Gel Digests</b>.  If it is to be used in in-gel digests, add
100 &micro;l of 1 mM HCl to a vial.  Mix well, then add 900 &micro;l
40 mM ammonium bicarbonate in 9% acetonitrile.  This is a 20 &micro;g/ml
solution.  (Alternatively, if not all the trypsin is to be used,
then store the 100 &micro;l acidifed trypsin at 2-8 or &minus;20&deg;,
and add 9 volumes of 40 mM ammonium bicarb + 9% acetonitrile to 1 volume
trypsin.)
 </ul>
</ul>
<p>
Prepare a water bath or incubator or heating system at 37&deg;.

<p>
The reductive alkylation and digest is performed as follows:

<ol>
<li>Prepare the protein substrate in a 10 &micro;l volume with a concentration
that is between 0.2 and 5 pmol/&micro;l.
For a 30 kDa protein, that is between 6-150 ng/&micro;l (or
6-150 &micro;g/ml).  The total protein thus ranges 2-50 pmol (or 60-1500 ng).
<li>Add 15 &micro;l of 10 mM DTT in 50 mM ammonium bicarbonate.
<li>Incubate for 30 min.
<li>Add 15 &micro;l of 55 mM iodoacetamide in 50 mM ammonium bicarbonate.
<li>Wait for 20 minutes.
<li>Dilute the 100 ng/&micro;l trypsin to 6 ng/&micro;l with 50 mM
ammonium bicarbonate.  Add 12.5 &micro;l of 6 ng/&micro;l trypsin.
<li>Incubate 3 hours at 37&deg;C.
<li>If the material is to be used on ESI-MS, then add 20 &micro;l
of 1% formic acid.  If the material is to be used for MALDI, then add
20 &micro;l 1% trifluoroacetic acid, and spot on to MALDI plates.
</ol>
<p>
The final volume is about 70 &micro;l.
<p>
For spotting MALDI plates, make up 2 mg/ml CHCA in 50% acetonitrile
containing 0.1% TFA.  Prepare a lockmass solution of ACTH 18-39
by adding 1 mg water to 1 mg ACTH (makes 400 pmol/&micro;l), dilute
25 &micro;l of this in 1 ml water to make 10 pmol/&micro;l,
then add 4 &micro;l of this to 200 &micro;l of the matrix to
give 200 fmol/&micro;l.
<p>
Note that trypsin has a mass of 23.3 kDa.  If 75 ng of trypsin is added
to do the digestion, that is 3.2 pmol of trypsin to digest between 2-50
pmol substrate.

<h3 id="calibpeptideprep">Preparation of A Set of Peptides for
Mass Spectrometer Calibration</h3>
<p>
Calibration peptides obtained from digest of a well-known protein
with a good distribution of peptide masses are obtained by reductive
alkylation and then digestion with trypsin.
<p>
Some preliminary calculations are necessary before performing the digestion.
Two sets of calibration peptides can be prepared.  One is for MALDI-MS
and the other for ESI-MS.  Probably 10 times as much is used for
ESI-MS than MALDI-MS, so the first calculation can be for MALDI-MS.
<p>
Generally the peptides from 100-1000 fmol digested protein are used in
a MALDI well in order to perform a meaningful calibration.  A single
calibration is done for a long run, so perhaps one or two wells might
be used for the calibration.  The 2nd well can test plate flatness, or
be used to confirm the calibration or re-calibrate within a run.
Since this is done once per run, the
period of time that a set of calibration peptides lasts depends on the run.
If enough peptides for 100 MALDI plates is prepared, a total of 200 pmol
of protein must be digested.  If 10 times as many peptides are used for
ESI-MS, that is 2000 pmol of protein to be digested.
<p>
A total of 30 pmol (15 pmol over 2 additions) of trypsin will be added
to perform the digestion.  This is is approximately 700 ng.
If the protein substrate is 2000 pmol and is
BSA, this is 133 &micro;g of BSA, and the weight ratio is 200:1 (w/w).
This is usually small than recommended, but it helps to ensure that
trypsin is not a contaminant.
<p>
The following reagents are required, and the volumes indicated sufficient
for a single tube of protein to be digested:
<ul>
<li>50 mM DTT:  10 &mu;l
<li>250 mM ammonium bicarbonate:  about 100 &mu;l
<li>150 mM iodoacetamide in 50 mM ammonium bicarbonate: place 1.39 mg
iodoacetamide in a tube and add 50 &micro;l of 50 mM ammonium bicarbonate.
<li>100 ng/&micro;l of proteomics-grade trypsin:
add 200 &micro;l of
reconstitution solution (or if it is not provided, use 1 mM HCl or
50 mM acetic acid) to a 20 &micro;g vial of lyophilized trypsin
</ul>
<p>
The following steps are used:
<ol>
<li>Prepare the protein in the correct aqueous solution that makes it
dissolve.  The best is water, since use of sodium salts to raise the
ionic strength slightly can possibly cause formation of sodium adducts.
Make the concentration between 100-300 pmol/&micro;l.  For example,
a 100 pmol/&micro;l bovine serum albumin (BSA) solution is 6.6 mg/ml.
A 100 pmol/&micro;l alcohol dehydrogenase (ADH) solution is 15 mg/ml
(but because ADH is a tetramer of identical subunits, it will produce
400 pmol of peptides).
<li>Pipet 20 &micro;l (containing 2 nmol or 2000 pmol) of
calibration protein to a microtube (0.5 ml), thin-walled tube preferred.
<li>Pipet 20 &micro;l of 50 mM DTT into the tube.
<li>Pipet 10 &micro;l of 0.25 M ammonium bicarbonate into the tube.
<li>Allow to sit for 45-60 min after mixing a few times with finger in the
 37&deg; water bath.
<li>Remove from the water bath and let come to room temperature.
Add 30 &micro;l of 150 mM iodoacetamide in 50 mM ammonium bicarbonate
and mix well.
<li>Incubate for 60 min.
<li><b>First Trypsin Digestion</b>.
 <ol>
 <li>To the reaction tube add 2.5 &micro;l water
 <li>Add 3.5 &micro;l of 100 ng/&micro;l trypsin.
 <li>Add 1.5 &micro;l of 250 mM ammonium bicarbonate.
 </ol>
<li>Mix well by flicking tube with finger.  Incubate at 37&deg; for
2.5-3 hours, and flick the tube with finger slightly about halfway into
incubation.
<li><b>Second Trypsin Digestion</b>.
Repeat the addition of water, trypsin, ammonium bicarbonate in that order
as given for the First Trypsin Digestion.
<li>Let stand another 3 hours or overnight.
<li>Determine the total volume with a pipet.  It should be about 90-100
&micro;l, depending on pipetting accuracy.  Take 80% of the volume
to another tube.  To the tube containing 80% of the volume, add 20 &micro;l
of 1% formic acid and label it &ldquo;&lt;protein&gt; peptides for
ESI-MS,&rdquo; where &lt;protein&gt; is the name of the protein, such
as &ldquo;BSA&rdquo; or &ldquo;ADH.&rdquo; To the other tube,
add 5 &micro;l of 1% TFA and label it &ldquo;&lt;protein&gt; peptides for
MALDI-MS.&rdquo;
<li><b>MALDI-MS peptides</b>.  Prepare 9 small tubes (500 &micro;l size).
To the tube with about 10 &micro;l MALDI peptides, add 90 &micro;l of
0.1% TFA, which produces about 2 pmol/&micro;l of peptides.  Distribute about
10 &micro;l each to the other 4 tubes.  Freeze them at &minus;20&deg;.
When calibration of the MALDI is to be done, remove a tube from the freezer,
and for every 1 &micro;l of peptides, add 1 &micro;l of 100 ng/&micro;l
(= 0.1 mg/ml) of CHCA (this is a 500:1 mol/mol matrix:peptide ratio).
The matrix should be dissolved in 50% MeCN + 0.1% TFA.
<li><b>ESI-MS peptides</b>.  To the volume remaining in the ESI-MS
tube, add about 10 times its volume in 0.1% formic acid (if you need
a larger tube, transfer the amount to larger tube, and then when adding
the formic acid, do it by washing the formic acid into the old tube,
and transferring it to the larger tube).  Distribute 75-100 &micro;l
aliquots of the dilution to each of a set of small (0.5 ml) tubes.
</ol>
<p>
Make sure that the concentration of peptides is written on the label.
It can be approximate.  The peptides we made here are about 2 pmol/&micro;l.


<h2 id="ingeldigest">MassPREP:  In-Gel and Solution Digestion with Trypsin</h2>
<p>
In-gel digestion of proteins in excised gel pieces is done using the
automated liquid handling/pipetting system, the MassPREP Station (A
Robotic Liquid Handling System, modified form of the Perkin-Elmer
MultiProbe II), which is
equipment that is a specialized modification by Waters of the the
Perkin-Elmer MultiPROBE II Robotic Liquid Handling station.
<p>
The software interface is called WinPREP.  Waters has already supplied
programmed modules for doing typical routines involving both in-gel
and solution digestions in multiwell plates.  Generally one only has
to run the module and make sure that the tips and labware is in position.

<h3>Checklist for In-Gel Trypsin Digestion</h3>
<p>
The deck of the MassPREP station should be loaded with the following:

<ul>
<li><b>Labware</b>
 <ul>
 <li>Heater support tile on G16 (for gel plate 1)
 <li>Heater support tile on E16 (for gel plate 2)
 <li>200 &micro;l pipet tips on C16
 <li>20 &micro;l conductive pipet tips on C13 (for gel plate 1)
 <li>20 &micro;l conductive pipet tips on A13 (for gel plate 2)
 <li>Multiwell (96) plate on cooled support at C10 (for peptides from plate 1)
 <li>Multiwell (96) plate on cooled support at A10 (for peptides from plate 2)
 <li>Lid rack on C7 for plates 1
 <li>Lid rack on A7 for plates 2
 <li>4 &times; 8 Reagent Rack on cooled support at E10
 <li>2 med trough + 3 small trough reagent rack at G7
 <li>Two targets on left side (front and back) of target holder at E13
 <li>Used tip bin centered at A17
 <li>Wash station at G13
 </ul>
<li><b>Reagents for Coomassie- or SYPRO-stained gels</b>
 <ul>
 <li><b>Buffer</b>.
    0.1 M ammonium bicarbonate in leftmost, medium-sized (48/57 ml)
    reagent trough.  30 ml enough for one plate.
 <li><b>Reducing Agent</b>.  Put 15.4 mg of DTT in 10 ml of ammonium
 bicarbonate Buffer, enough for one plate of 96 samples.  This is 10 mM
 This should always be made fresh.
 <li><b>Alkylating Agent</b>.  Put 102 mg of iodoacetamide in 10 ml
  of ammonium bicarbonate Buffer; this is 55 mM.  Should also be made
  fresh every time.
 <li><b>Digestion Agent</b>.  To one vial of 20 &micro;g of MS-grade
   trypsin (Promega V5111, for example), add 200 &micro;l of the
	supplied resuspension buffer (50 mM acetic acid).  23 &micro;l is
	used for 48 samples, 45 &micro;l for one plate, and 90 &micro;l for
	two plates of 192 samples.
 <li><b>Extraction Agent</b>.  This is a 1% formic acid solution in
   2% MeCN.  Add 100 &micro;l formic acid and 200 &micro;l MeCN to 10
	ml water, enough for one plate of 96 samples.
 <li><b>Matrix</b>.  If the target plate spotting setup is used, one
 dissovles 2 mg CHCA in 1 ml of 0.2% TFA in 50% MeCN, enough for
 384 samples (four 96-well plates).
 <li>Acetonitrile go in the large wash station container.  80 ml for one
 plate of 96 samples, and 5 mm from the top for two plates.
 <li><b>Lockmass</b>.  If the target plate spotting setup is used:
  <ol>
  <li>Prepare a 400 pmol/&micro;l ACTH solution (1 ml water to 1 mg ACTH)
  <li>Add 25 &micro;l of this to 1 ml water to get a 40 pmol/&micro;l solution.
  <li>Add 10 &micro;l of this to 500 &micro;l of the matrix solution to
    get a 200 fmol/&micro;l solution.  200 fmol is thus spotted.
  </ol>
  This 0.5 ml solution is enough for 4 plates (384 samples).
  </ul>
<li><b>Reagents for silver-stained gels</b>
 <ul>
 <li><b>De-Stain</b>.  Prepare a 20 ml solution (enough for 2 plates:
  192 samples) of a
  1:1 (v/v) mixture of 30 mM potassium ferricyanide (K<sub>3</sub>FeCN<sub>6</sub>)
  and 100 mM sodium thiosulfate (Na<sub>2</sub>S<sub>2</sub>O<sub>3</sub>) as
  follows:
  <ol>
  <li>Dissolve 248 mg sodium thiosulfate in 10 ml water.
  <li>Dissolve 99 mg potassium ferricyanide in water.
  </ol>
  Place in De-Stain trough.  Should be used immediately.
 </ul>
</ul>

<h4>Manually Doing The MassPREP Method</h4>
<p>
The MassPrep in-gel digestion method (Digestion 5.7) can be done by hand
rather than with the MassPREP station.  This might be useful if only doing
a few samples (less than a column of 8, for example), or there is something
wrong with the MassPREP station for some reason.  Try this method:

<ol>
<li>Set an oven on at 37 &deg;
<li><b>Remove excess liquid</b>.  Read this procedure even if you have no
excess liquid to remove.  You will use this procedure to remove all reagents
that have been pipetted into wells and which are to be removed.
 <ol style="list-style-type:lower-roman;">
 <li>Fit a P-200 with a yellow tip and set it at about 100-150 &micro;l.
 <li>Starting with the first well to have excess liquid in  its well removed,
   place the tip off the center of the well and begin pipetting out the liquid.
 <li>When much less liquid is present in the well, carefully move the pipet
  tip toward the center and continue sucking out liquid with pipet.
  Watch for the possibility that gel pieces might be sucked into the tip.
  Use pipet tips with lumenal openings smaller than the gel pieces to prevent
  this.
 <li><b>Using the same tip</b>.  You can limit the number (conserve) the tips
  you use merely by washing the tip between wells.  Fill a large beaker with
  ultrapure or distilled water, and just pipet the water in and out to rinse
  between wells.  The MassPREP does the same thing.
  </ol>
<li><b>Initial Washing &amp; Destaining</b>.  Coomassie- and SYPRO-stained
gels are destained in different ways.  Pre-warm the reagents to 37 &deg;
before using them.
 <ul>
 <li><b>Coomassie/SYPRO-stained gels.</b>  Perform the following ONLY for
wells on the plate(s) containing Coomassie- or SYPRO-stained pieces.
  <ol>
  <li>Pipet 50 &micro;l of 100 mM ammonium bicarbonate.
  <li>Pipet 50 &micro;l of acetonitrile (MeCN).
  <li>Wait 10 min. at 37 &deg;.
  <li>Remove the solution with a P-200/yellow tip, rinsing between wells.
  <li>Repeat the destaining (previous 4 steps).
  </ol>
 <li><b>Silver-stained gels</b>. Perform the following ONLY for
wells on the plate(s) containing silver-stained pieces.  Silver destain
reagent is a 1:1 (v/v) mixture of 30 mM potassium ferricyanide
(K<sub>3</sub>Fe(CN)<sub>6</sub>)
and 100 mM sodium thiosulfate (Na<sub>2</sub>S<sub>2</sub>O<sub>3</sub>).
  <ol>
  <li>Pipet 80 &micro;l of the silver detain reagent into the well.
  <li>Wait 15 minutes at 37&deg;.
  <li>Add 120 &micro;l of the reagent to each well, in a 2nd addition.
  <li>Wait 15 minutes at 37&deg;
  <li>Remove all the reagent with P200/yellow tip.
  <li>Add 150 &micro;l water.
  <li>Wait 10 min. at 37&deg;.
  <li>Remove the wash with P200/yellow tip.
  <li>Add 50 &micro;l of 100 mM ammonium bicarbonate.
  <li>Wait 5 min at 37 &deg;.
  <li>Add 75 &micro;l of MeCN.
  <li>Wait 5 min. at 37 &deg;.
  <li>Remove the liquid in the wells with P200/yellow tip.
  </ol>
 </ul>
<li><b>Reduction &amp; Alkylation</b>.  If a reduction and akylation is
to be done, use the following steps.  This is to be done to ALL wells.
All Wait/incubation steps are to be done at 37&deg;!!
 <ol>
 <li>Add 50 &micro;l MeCN
 <li>Wait 5 min.
 <li>Remove MeCN with P200/yellow tip.
 <li>Put back at 37&deg; and wait another 10 min. for everything to evaporate.
 <li>Add 50 &micro;l of 10 mM DTT in 50 mM ammonium bicarbonate.
 <li>Incubate for 30 min.
 <li>Add 50 &micro;l of 55 mM iodoacetamide in 50 mM ammonium bicarbonate.
 <li>Incubate 20 min.
 <li>Add 100 &micro;l MeCN
 <li>Let sit for 5 min.
 <li>Remove all liquid from wells with P200/yellow tip.
 </ol>
<li><b>Washing &amp; Dehydration</b>.  You are now ready for the preparation
for trypsinization.  Do this to all the wells and all incubation/sitting/waiting
steps are to be done in the 37&deg; oven.
 <ol>
 <li>Add 50 &micro;l of 100 mM ammonium bicarbonate.
 <li>Wait 10 min.
 <li>Add 50 &micro;l of MeCN.
 <li>Wait 5 min.
 <li>Remove the liquid with P200/yellow tip.
 <li>Repeat the 50 &micro;l MeCN, incubation, and liquid removal step two more
 times.
 </ol>
<li>Let the plate sit another 15 minutes at room temperature and wait for
all the acetonitrile to evaporate.
<li><b>Trypsin Addition</b>.  This will be done to all wells.  Follow
carefully which steps require a 37&deg; incubation (not all do!).
 <ol>
 <li>Prepare 6.25 ng/&micro;l of trypsin in 50 mM ammonium bicarbonate.
Prepare enough for 25 &micro;l per well.
 <li>Add 25 &micro;l of the dilute trypsin to each well.
 <li>Let the gel pieces soak up the trypsin for 30 min at room temperature.
 <li>After covering with a secure lid, or wrapping in strech film,
 put the plate into the 37&deg; oven and let it sit at least 4.5 hours
(it can go longer, but it is not particularly necessary).
 <li>Take the plate out and let it cool (15-30 min).
 </ol>
<li><b>Extraction</b>.  Do this for all wells and at room temperature.
The extraction reagent used here is 1% formic acid and 2% acetonitrile.
 <ol>
 <li>To the cooled plate, add 30 &micro;l of extraction reagent.  The wells
still contain the trypsin digestion solution.
 <li>Let sit 30 min, covered with a lid.
 <li>Get a clean PCR plate.  Obtain enough of the cleanest yellow tips
possible (or use diamond tips).  With a P-20 or P-200 or P-10 and the proper
tip on it, transfer the solution in the digest plate to the receiving PCR
plate.  Be careful to match well A1 to well A1, or to keep track of what
well you take from to what well you put into.  Use a new tip for each well
transfer.  Do NOT rinse pipet tips and use for multiple wells!!!  To
conserve tip use, you can place the used tip back into an empty rack
and then re-use if you plan on doing repeated extractions (see next step).
 <li>If you want to repeat the extraction up to two more times, pipet
 12 &micro;l of the extraction reagent and 12 &micro;l of MeCN to each
 well, wait 30 minutes, and then recover with a well-to-well transfer,
 using the same wells as in the first step.  Be very careful.
 </ol>
<li>You can use the extracted peptides in the PCR plate immediately by
spotting on to MALDI.  If you do not get good spectra, consider concentrating
the peptides down in a vacuum centrifuge that can accomodate multi-well plates,
then resuspend in a suitable solvent (50% MeCN with 0.1% TFA or formic acid),
and then do MALDI or ESI.
</ol>

<h2 id="otherproteases">Other Proteases Used To Prepare Peptides</h2>

<h3>Endoproteinase Arg-C</h3>
<p>
Endoproteinase Arg-C (CAS#&nbsp; 82047-85-6, EC 3.4.21.35) specifically
cleaves on the carboxy side of arginine residues.

<h3>Endoproteinase Asp-N</h3>
<p>
Endoproteinase Asp-N (CAS#&nbsp; 9001-92-7, EC 3.4.24.33) specifically
cleaves on the amino side of aspartate residues.

<h3>Endoproteinase Glu-C</h3>
<p>
Endoproteinase Glu-C (CAS#&nbsp; 66676-43-5, EC 3.4.21.19) specifically
cleaves on the carboxy side of glutamate residues.  Because it was
first isolated from <i>Staphylococcus aureus</i> V8, it is also called
the V8 protease.

<h3>Endoproteinase Lys-C</h3>
<p>
Endoproteinase Lys-C (CAS#&nbsp; 72561-05-8, EC 3.4.21.50) specifically
cleaves on the carboxy side of lysine residues.


<!--  Glycoproteomics -->
<h1 id="glycoproteomics">Glycoproteomics</h1>
<p>
This offers strategies for determining the nature of glycans attached to
proteins and their points of attachment in glycoproteins.
<p>
Glycoproteins are generally classed in three ways:
<ol>
<li><b>N-linked</b>
<br>These glycans attach to their proteins usually on asparagine
amide nitrogens at the consensus site -Asn-X-Ser/Thr-, where X is any
amino acid except Pro.  N-linked glycoproteins are found only in eukaryotes,
because the glycosyltransferase (sugar-attachment enzyme) is found
in the membrane of the endoplasmic reticulum (ER), and those organelles
are found only in eukaryotes.
<li><b>O-linked</b>
<br>The glycans attach to the hydroxyl moieties of Ser or Thr residues.
A survey of glycosylation sites in both prokaryotic and eukaryotic
organisms does not really show a consensus sequence for mucin-type
glycoproteins, and the glycosyltransferases (glycosylating enyzmes)
probably use a secondary or tertiary structure to recognize the glycosylation
site.  Many O-linked glycoproteins have special recognition sites, but
there is no consensus.  At least eight different types of cores have
been identified for these proteins.
<li><b>GPI-anchored</b>
<br>Proteins attach to the plasma membrane covalently by linking to
a phosphatidylinositol on the intracellular side of the membrane.  This
is called GPI (glycosylphosphatidylinositol) anchor.  The point of
attachment of the protein is the oxygen on its carboxyl terminus.
</ol>
<p>
The biology of N-linking is well studied.  A mannose-rich sugar composed
of 14 monomers (-GlcNAc-GlcNAc-Man(-Man(-Man-Man)(-Man-Man)(-Man-Man-Man)))
is a triantennary branched chain attached to the Asn amide nitrogen
via a GlcNAc.  Some proteins are then exported retaining the <b>high-mannose</b>
type sugar, or they move to the Golgi for further maturation.
Before moving to the Golgi, the mannose residues are trimmed where
they will go to form <b>hybrid</b> (bi- or triantennary sugars, one antenna
mannose branches, others GlcNAc) or <b>complex</b> type where two, three
or four antenna feature a heteropolymer mix (NeuNAc-Gal-GlcNAc-Man)
and with the GlcNAc near the attachment point fucosylated.  The
glycosylation often depends on what class the organism is (plants, mammals,
yeast, etc.).  Some N-linked glycoproteins go to the Golgi to get
O-linked glycosylations.

<h2 id="enzymaticdeglycos">Enzymatic Deglycosylation</h2>
<p>
Removing the glycan for analysis without the protein attached is typical.
<p>
<b>Peptide N-glycosidase F (PNGase F)</b> is an effective deamidase
in that it hydrolyzes the Asn amide to produce the glycan with an
amine on the C1 of the GlcNAc.  It is not effective when fucose
is attached to the GlcNAc (seen in plants and some annelids).
<b>PNGase A</b> is effective in this case.  Glyko (GKE-5003),
QAbio (E-PNG01 kit), Sigma-Aldrich (P7367), and New England Biolabs
(P0704S) sell purified and/or recombinant forms of the enzyme;
take care that each defines a unit of activity differently.  The
enzyme generally has the best activity at pH 7.5/37&deg;.

<p>
<b>O-glycanase</b>.  Glyko sells an O-glycanase specific for
glycosylated serines and threonines.
<p>
Perhaps the most effective deglycosylation treatment for them is to use
a &beta;-elimination method used for esters or ethers, such as treatment
with hydroxide.
<p>
<b>&alpha;-Fucosidase</b>.  N-linked glycosidases cannot deglycosylate
with a &alpha;-fucose attached.  Use of this enzyme enables the determination
of the presence of fucose if the enzyme is deglycosylated.

<h3 id="PNGaseF">Demodification of N-Linked Glycoproteins Using PNGase F</h3>
<p>
This method is a <em>tentative</em> specific method for N-linked deglycosylation
using Glyko's PNGase F (Prod. #GKE-5003), which is a naturally purified
form from <i>Chrysobacterium (Flavobacterium) meningosepticum</i>.
This method prepares a large batch of protein that is sampled representing
time points of incubation for a next step preparation of 2DGE.
This particular method prepares glycoproteins for loading on to 7-cm IPG
strips (200 &micro;g each for Bio-Rad's product) using active rehydration.

<h4>Substrate Calculations and Preparations</h4>
<p>
The following time points will be
tried:  0, 60 min, 3 hours, overnight (8-12 hours).  This represents
four 2D gels.  Since each 2D gel gets 200 &micro;g protein, a total of
800 &micro;g protein is required.
<p>
A protein determination (Bradford, BCA) must be done.
The protein concentration should be &ge; 5 &micro;g/&micro;l,
and 10-20 &micro;g/&micro;l is best.  A concentration step using centrifugal
ultrafiltration is recommended (include protease inhibitors when doing
concentration procedures).  In the procedure below, it will be assumed that
a protein concentration of 5 and 10 mg/ml (same as 5 and 10 &micro;g/&micro;l)
are used.

<ol style="text-align:justify;">
<li><b>Water bath</b>.  Set a water bath at 37&deg;.
<li><b>Buffer/Reagents</b>.  Make sure the following are ready.  Generally
the volume you need is at most 1 ml in a 1.5 ml conical.
 <ol>
 <li>200 mM sodium phosphate, pH 7.5 (10X)
 <li>9 M urea (3X)
 <li>1 M DTT (20X)
 <li>Roche Complete (25X)
 <li>Glyko GKE-5003; 2 mU/&micro;l (100 mU in 50 &micro;l).
Keep <em>only</em> this product in an ice-water bath.  Generally
10 mU of enzyme is added for every 100 &micro;g denatured protein, and
incubation is from 2 h to overnight at 37&deg;.
 </ol>
<li><b>Protocol Table</b>.  The following table has been prepared to
account for different concentrations of the protein substrate used.
It has been prepared this way to keep the amount of components to a minimum
since these might interfere with the IEF step.  It is therefore highly
recommended to use a substrate whose protein concentration is as high as
possible.  Follow the instructions below the table, but use the table as
a guide.
<table class="centeredTD">
<caption>Pipet Volume Protocol Based on Protein Concentration of
Substrate<br> (except where noted, all volumes in &micro;l)</caption>
<col span="2">
<col span="4" style="width:16%;">
<tr><th rowspan="2" colspan="2" style="background:none;">
  <th colspan="4">If the protein concentration is:
<tr>  <th>5-10 mg/ml  <th>10-15 mg/ml  <th>15-20 mg/ml <th>&ge;20 mg/ml
<tr><th class="left" rowspan="2">protein substrate
  <th style="text-align:right;">volume
		 <td>180 <td>90  <td>67.5  <td>45
 <tr><th style="text-align:right;font-size:70%;">mass added (&micro;g)
		 <td style="font-size:80%;">900-1800
		 <td  style="font-size:80%;">900-1350
		 <td  style="font-size:80%;">1012-1350
		 <td style="font-size:80%;"> &ge; 900
<tr><th class="left" colspan="2">water
   <td>43.3  <td>38  <td>60.5   <td>83
<tr><td colspan="6" style="text-align:left;background-color:#333;color:yellow;">
Do the boiling water denaturation step <b>BEFORE</b>
adding any of the components below!
<tr><th class="left" colspan="2">1 M DTT (20X) <br>
<span style="font-size:75%;">this should be added immediately after removing
from protein from hot water</span>
 <td>25  <td>15  <td>15  <td>15
<tr><td colspan="6" style="text-align:left;background-color:#333;color:yellow;">
Add the components below <b>AFTER</b> cooling the reaction tube in an
ice-water bath, and then bringing the tube to 37&deg;
<tr><th class="left" colspan="2">9 M urea (3X)
  <td>166.7  <td>100  <td>100   <td>100
<tr><th class="left" colspan="2">* Roche Complete (25X)
  <td>20  <td>12  <td>12 <td>12
<tr><th class="left" colspan="2">200 mM sodium phosphate, pH 7.5 (10X)
  <td>50     <td>30   <td>30    <td>30
<tr><th  class="left" rowspan="2" style="vertical-align:top;">
&dagger; PNGase F enzyme (2 mU/&micro;l, GKE-5003)<br>
<span style="font-size:75%;">to be added last</span>
 <th style="text-align:right;">volume
		 <td>15 <td>15  <td>15  <td>15
 <tr><th style="text-align:right;font-size:70%;">milliUnits added
		 <td style="font-size:80%;">30 mU
		 <td  style="font-size:80%;">30 mU
		 <td  style="font-size:80%;">30 mU
		 <td style="font-size:80%;"> 30 mU
<tr><th class="left" colspan="2">Final Volume
  <td>500 <td>300    <td>300   <td>300
<tr><th class="left" colspan="2"> Volume Removed At Time Points
<br><span style="font-size:75%;">one-fourth for four 2D gels</span>
  <td>125    <td>75     <td>75   <td>75
<tr><td colspan="6"
style="text-align:left;background-color:#ddf;font-size:70%;">
* If Roche Complete is already present in the protein substrate sample,
an adjustment to the Roche Complete reagent volume might be considered
when the protein substrate volume is &gt; 25% of the total volume.
For example, in protocol column for 5-10 mg/ml
where 180 &micro;l of the substrate is added representing
&gt; 25% of the 500 &micro;l final
volume.  In this case, 320 &micro;l is the volume in which the reagent is
not present.  It would be necessary to add 12.8 &micro;l instead of 20 &micro;l.
But even if 20 &micro;l was added this is a bit more than a 50% excess,
generally not something to be overly concerned about.
<p>
&dagger; Why 30 mU? <!-- A Glyko protocol specifies the use of 5 mU
for 50-500 &micro;g substrate.  Since double the substrate is being
employed here, at least 10 mU must be added.  But the protocol also
specifies denaturation with 0.1% SDS, which cannot be used here.
To compensate, it is usually recommended to increase the amount of enzyme.
Anywhere from 2-5 &times; the amount of enzyme would seem logical.  Here
we use 3 &times; the recommended amount. -->
</table>
<li><b>Reaction Container Selection</b>.
Select a reaction tube container whose capacity is such that the reaction
mixture final volume represents a range of 20-50% of the container volume.
Thus if a 1.5 ml conical is used, the reaction mixture final volume would
range from 300-750 &micro;l.  If a 0.5 ml conical tube was used, a
final reaction mixture volume would range from 100-250 &micro;l.
The geometry and capacity of the container is such that the ability to
sample for the purpose of removing aliquots without affecting changes
in temperature during continuing incubations are the priorities here.
A higher surface area-to-volume ratio probably means more accurate sampling,
but it also results in greater temperature fluctuations during sampling
(although sampling should be brief and the tube quickly returned to
incubation).  Use this as a guide (not an absolute) in container selection;
always give your own best judgment the highest priority.
<li>To the selected container, add the proper volumes of substrate protein
and water.  Mix gently without foaming (finger-flicking or very,very gentle
vortexing).
<!-- start right box -->
<div style="float:right;width:20em;background-color:#ddf;
			margin:1em 0 1em 1em;padding:1em;text-align:left;">
<b>Notes on Glyko protocol</b>.  Glyko reports that most scientists using
their products vary their protocol.  The following is a protocol Glyko provides
for its GKE-5006 product, a recombinant PNGase F.
<ol style="list-style-type:decimal;">
<li>Prepare 50-500 &micro;g glycoprotein in 45 &micro;l in a reaction buffer
(recommended is 20 mM sodium phosphate pH 7.5 final; if direct to MS, since
phosphate reduces ionization efficiency, a 10 mM TrisHCl pH 8.0 buffer can
be substituted).
<li>2.5 &micro;l of 2% SDS and 1 M 2-mercaptoethanol is added.
<li>The mixture is heated at 100&deg; for 5 min., then allowed to cool
<li>Because SDS can adversely affect PNGase, 2.5 &micro;l of 15% NP-40 is
added.
<li>2 &micro;l of PNGase F (N-glycanase, &ge; 2.5 mU/&micro;l) and allowed
to work for 2 hours to overnight at 37&deg;.
</ol>
The final volume is approximately 50 &micro;l.  The final buffer:
20 mM sodium phosphate (or 10 mM TrisHCl).  0.1% SDS, 50 mM 2-mercaptoethanol,
0.75% NP-40, and 0.1 mU/&micro;l enzyme (total 5 mU).  The substrate
concentration is from 1-10 &micro;g/&micro;l.  If the enzyme purity is
about 10 mU/&micro;g, then 0.5 &micro;g enzyme was added, providing a
ratio of 100-1000 &micro;g substrate per &micro;g enzyme.
</div>
<!-- end right box -->
<li>Prepare a boiling hot water bath in a beaker large enough to accept one
of those urethane foam floats with holes cut in it for floating conical
tubes in water baths.  Just after removing the boiling water bath from
the microwave oven, place the float in the water, and place the tube
in the float (note if the tube was made to fit 1.5 ml conicals, and the
tube is a 0.5 ml conical, place a 1.5 ml conical in the float, push the
float down to fill the 1.5 ml tube with hot water, zap in the oven on high
for another 30 seconds, then remove the beaker and put the 0.5 ml conical
within the water of the 1.5 ml conical).  From the time of placing the
tube into the freshly zapped boiling water, wait at least 5 minutes and
no longer!  If cooling starts, re-naturation is possible, and this is
not desirable.
<li>Remove the tube from the hot water bath
and add the proper volume of DTT now.  Wait for about 1-2 minutes.
<li>Reduce the temperature of the tube in an ice-water bath, perhaps 30-60
seconds.
<li>Place the tube in a 37&deg; water bath for 5 min.
<li>Add 3X urea in the proper volume and mix well but gently.  Wait 30-60 sec.
<li>Add 25X Roche Complete and 10X sodium phosphate. Mix well but gently.
<li>Re-heat the boiling water bath. Keep it ready for the next step.
Place a 1.5 ml conical tube in a float and make sure it is filled the boiling
hot water (just dunk it to immerse it briefly).  Reheat in microwave for
30 seconds to get it boiling.
<li>Prepare a pipettor with the volume set to 25% (one-fourth) of the volume
the final reaction volume (see protocol table above).  Also have ready an
empty 0.5 ml conical tube.
<li>Add the enzyme in the proper volume to the reaction tube.
Addition should be with the pipet tip immersed in the solution. Pipet
up and down to make sure all the enzyme is added.  Very
gently mix by finger flicking or slow rotation on the vortex for at least
30-60 seconds.
<li>Immediately remove one-fourth of the volume with the pipettor whose
volume is already set, place it in the 0.5 ml conical tube, and place that
conical tube within the 1.5 ml conical tube set in the float and containing
boiling/hot water.  Leave this for at least 3-5 min.  Put the rest of
the reaction mixture in a 37&deg; water bath.
<li>At each time point, take the appropriate aliquot, put the tube in
boiling hot water to stop the enzyme reaction, and the dry down the contents
of the aliquot in vacuum centrifuge.
</ol>
<p>
In drying the tube, do not apply external heat to assist drying.  If drying
is slow, you might want to use ethanol or acetonitrile additions in small
amounts to accelerate drying.
<p>
Try to dry it completely.  But if not possible, dry it as much as possible.
Add up to the proper volume (usually 125 &micro;l for 7 cm
strips) of rehydration buffer.   Try to dissolve everything.  Spin it
down and apply the supernatant to the focusing tray.  Add the IPG strip
gel side down and begin active rehydration overnight at 50 V.  Focus and
complete 2D gel work as usual.

<h4>Freeze Method</h4>
<p>
This method was described by H. H. Freeze, "Endoglycosidase and Glycoamidase
Release of N-Linked Oligosaccharides" (Unit 17.13A) in <i>Current Protocols
in Molecular Biology</i>, looseleaf ed., Ausubel F et al, <i>eds</i>.
<p>
The glycoprotein substrate had been immunoprecipitated.  Obtain the following
reagents/environment:
<ul>
<li>0.1 &beta;mercaptoethanol/0.1% SDS
<li>0.5 M TrisHCl, pH 8.6 at 37&deg;
<li>10% (w/v) Triton X-100 or NP-40
<li>200-250 mU/ml PNGase F (Sigma or Glyko)
<li>10&times; SDS sample buffer
<li>30-37&deg; and 90&deg; water baths
</ul>

<ol>
<li>Denature glycoprotein substrate in 20-30 &micro;l of the 2ME/SDS solution
for 3-5 min at 90&deg;.
<li>Cool, then pulse spin to bring solution down
<li>Transfer to two tubes 10 &micro;l each of the protein
<li>Add:
	<ol>
	<li> 3 &micro;l 0.5 M TrisHCl, pH 8.6
	<li> 5 &micro;l water
	<li> 2 &micro;l 10% NP-40 or Triton X-100
	<li> 5 &micro;l 200-250 mU/ml PNGase F or diluted Tris buffer only as
		no-enzyme control
	</ol>
<li>Incubate overnight at 30 or 37&deg;.
<li>Stop reaction with 2.5 &micro;l 10&times; SDS sample buffer for 3-5 min
	at 90&deg; and process for SDS-PAGE, or use other stop mechanism.
</ol>
<p>
Note that electrophoretic mobility is increased with oligosaccharide removal.

<h3 id="neuraminidase">Demodification of Sialylated Glycoproteins
  by Neuraminidase</h3>
<p>
This is a tentative protocol to remove sialic acid residues from
glycoproteins.  This can be used in connection with 2DGE to determine
whether IEF-resolved isoforms are due to sialic acid moieties present
on glycoproteins.  Neuraminidases are freely available from Sigma-Aldrich;
use of a high activity purified form is recommended.
<p>
The treatment described here involves protein substrates treated for
different incubation times to follow the de-sialyation.  Four time
points are used:  a zero (0) time point, and a percentage of expected
time of completion such as 5-10%, 25-50%, and 100%, for a total of four
aliquots for four 2D gel analyses.  The choice of times depends a lot
on what is known about the enzyme reaction and its preparation.

<ol>
<li><b>Buffer/Reagents</b>.  Make sure the following are ready.  Generally
the volume you need is at most 1 ml in a 1.5 ml conical.
 <ol>
 <li>250 mM sodium acetate, pH 5.0 (5X)
 <li>100 mM calcium chloride (50X)
 <li>Sigma Prod N2133, 1 Unit (CAS 9001-67-6, acyl-neuraminyl hydrolase, sialidase),
Type X from <i>Clostridium perfringens</i> present as lyophilized powder,
purified by affinity chromatography from Type VII. 1 Unit removes 1 &micro;mole
of N-acetylneuraminic acid at pH 5.0 and 37&deg; in 1 minute, where the substrate
is NAN-lactose or bovine submaxillary mucin.<br>
The enzyme comes in a lyophilized form and needs to be prepared by adding
100 &micro;l of 100 &micro;g/ml BSA to it (but SEE NOTE BELOW!).  The
100 &micro;l volume water and/or BSA is usually required because Sigma
lyophilizes the enzyme down from large vials, and any volume smaller than
100 &micro;l disappears (you can recover 85-90% back).
If there are 1000 mU (1 U), then
this is 10 mU/&micro;l.  If you re-freeze this, you might want to aliquot
it to 10 &micro;l aliquots.
  <ul>
  <li><b>NOTE</b>.  If you are doing 2-D gels, adding a resuspension solution containing
   BSA could complicate the analysis.  Therefore add ONLY water to the
	lyophilizate such that the concentration of the enzyme never goes below
	1 U/ml (1000 mU/ml), and keeping it above 5 and even 10 U/ml is probably
	even advised.  Aliquot and freeze; you should probably have no more than
	one freeze-thaw cycle; that is, you are allowed one more thaw, so aliquot
	advisedly.  If you put into frost-free freezer, use storage tubes with
	an O-ring to prevent drying.
  </ul>
 </ol>
<li><b>Protocol Table</b>.  The following table has been prepared to
account for different concentrations of the protein substrate used.
It has been prepared this way to keep the amount of components to a minimum
since these might interfere with the IEF step.  It is therefore highly
recommended to use a substrate whose protein concentration is as high as
possible.  Follow the instructions below the table, but use the table as
a guide.
<table class="centeredTD">
<caption>Pipet Volume Protocol Based on Protein Concentration of
Substrate<br> (all volumes in &micro;l)</caption>
<tr><th rowspan="2" style="background:none;width:33%;">
  <th colspan="4">If the protein concentration is:
<tr>  <th>5-10 mg/ml  <th>10-15 mg/ml  <th>15-20 mg/ml <th>&ge;20 mg/ml
<tr><th class="left">protein substrate
 <td>180 <td>90  <td>67.5  <td>45
<tr><th class="left">water
   <td>0    <td>90    <td>112.5 <td>135
<tr><td colspan="5" style="text-align:left;background-color:#333;color:yellow;">
Do the boiling water denaturation step <b>BEFORE</b>
adding any of the components below!
<tr><th class="left">* Roche Complete (25X)
  <td>10 <td>10  <td>10  <td>10
<tr><th class="left">250 mM sodium acetate, pH 5.0 (5X)
  <td>50     <td>50   <td>50    <td>50
<tr><th class="left">100 mM calcium chloride (50X)
  <td>5      <td>5    <td>5     <td>5
<tr><th  class="left">neuraminidase enzyme (10 mU/&micro;l)<br>
<span style="font-size:75%;">50 mU to be added last</span>
  <td>  5     <td>  5    <td> 5  <td> 5
<tr><th class="left">Final Volume
  <td>250    <td>250    <td>250   <td>250
<tr><th class="left"> Volume Removed At Time Points
<br><span style="font-size:75%;">one-fourth for four 2D gels</span>
  <td>62.5   <td>62.5   <td>62.5 <td>62.5
<tr><td colspan="5"
style="text-align:left;background-color:#ddf;font-size:70%;">

* If Roche Complete is already present in the protein substrate sample,
an adjustment to the Roche Complete reagent volume might be considered
when the protein substrate volume is &gt; 25% of the total volume.
For example, in protocol column for 5-10 mg/ml
where 180 &micro;l of the substrate is added representing
&gt; 25% of the 500 &micro;l final
volume.  In this case, 320 &micro;l is the volume in which the reagent is
not present.  It would be necessary to add 12.8 &micro;l instead of 20 &micro;l.
But even if 20 &micro;l was added this is a bit more than a 50% excess,
generally not something to be overly concerned about.
</table>
<li><b>Reaction Container Selection</b>.
Select a reaction tube container whose capacity is such that the reaction
mixture final volume represents a range of 20-50% of the container volume.
Thus if a 1.5 ml conical is used, the reaction mixture final volume would
range from 300-750 &micro;l.  If a 0.5 ml conical tube was used, a
final reaction mixture volume would range from 100-250 &micro;l.
The geometry and capacity of the container is such that the ability to
sample for the purpose of removing aliquots without affecting changes
in temperature during continuing incubations are the priorities here.
A higher surface area-to-volume ratio probably means more accurate sampling,
but it also results in greater temperature fluctuations during sampling
(although sampling should be brief and the tube quickly returned to
incubation).  Use this as a guide (not an absolute) in container selection;
always give your own best judgment the highest priority.
<li>To the selected container, add the proper volumes of substrate protein
and water.  Mix gently without foaming (finger-flicking or very,very gentle
vortexing).
<li>Prepare a boiling hot water bath in a beaker large enough to accept one
of those urethane foam floats with holes cut in it for floating conical
tubes in water baths.  Just after removing the boiling water bath from
the microwave oven, place the float in the water, and place the tube
in the float (note if the tube was made to fit 1.5 ml conicals, and the
tube is a 0.5 ml conical, place a 1.5 ml conical in the float, push the
float down to fill the 1.5 ml tube with hot water, zap in the oven on high
for another 30 seconds, then remove the beaker and put the 0.5 ml conical
within the water of the 1.5 ml conical).  From the time of placing the
tube into the freshly zapped boiling water, wait at least 5 minutes and
no longer!  If cooling starts, re-naturation is possible, and this is
not desirable.
<li>Reduce the temperature of the tube in an ice-water bath, perhaps 30-60
seconds.
<li>Place the tube in a 37&deg; water bath for 5 min.
<li>Add 25X Roche Complete, 5X sodium acetate, and 50X calcium chloride.
Mix well but gently.
<li>For taking a zero time point aliquot, re-heat the boiling water bath.
Keep it ready for the next step.  Place a 1.5 ml conical tube in a float
and make sure it is filled the boiling
hot water (just dunk it to immerse it briefly).  Reheat in microwave for
30 seconds to get it boiling.
<li>Prepare a pipettor with the volume set to 25% (one-fourth) of the volume
the final reaction volume (see protocol table above).  Also have ready an
empty 0.5 ml conical tube.
<li>Add the enzyme in the proper volume to the reaction tube.
Addition should be with the pipet tip immersed in the solution. Pipet
up and down to make sure all the enzyme is added.  Very
gently mix by finger flicking or slow rotation on the vortex for at least
30-60 seconds.
<li>Immediately remove one-fourth of the volume with the pipettor whose
volume is already set, place it in the 0.5 ml conical tube, and place that
conical tube within the 1.5 ml conical tube set in the float and containing
boiling/hot water.  Leave this for at least 3-5 min.  Make sure that the
reaction tube with the rest of its mixture is placed floating in a 37&deg;
water bath as quickly as possible.
<li>At each time point, take the appropriate aliquot, put the tube in
boiling hot water to stop the enzyme reaction, and the dry down the contents
of the aliquot in vacuum centrifuge.
</ol>
<p>
In drying the tube, do not apply external heat to assist drying.  If drying
is slow, you might want to use ethanol or acetonitrile additions in small
amounts to accelerate drying.
<p>
Try to dry it completely.  But if not possible, dry it as much as possible.
Add up to the proper volume (usually 125 &micro;l for 7 cm
strips) of rehydration buffer.   Try to dissolve everything.  Spin it
down and apply the supernatant to the focusing tray.  Add the IPG strip
gel side down and begin active rehydration overnight at 50 V.  Focus and
complete 2D gel work as usual.

<h2>Chemical Deglycosylation</h2>
<p>
<b>Hydrazine</b> treatment will release intact and unreduced forms of both N-
and O-linked glycans.  Both time and temperature will control the cleavage
quantitatively and selectively.
<p>
Strongly alkaline conditions will also result in &beta;-elimination of O-linked
glycans in particular.
<p>
The polypeptide is chemically altered (destroyed or damaged) by hydrazinolysis
and &beta;-elimination.  Use of anhydrous trifluoromethanesulfonic acid (TFMS)
removes N-linked oligosaccharides (except for innermost Asn-lined GlnNAc or
GalNac) and O-linked glycans, as well as glycosaminoglycans in a proteoglycan
core and saccharides on collagen linked to hydroxyproline.  There is no damage
to proteins but the oligosaccharides are destroyed.

<h3 id="TMFS">TFMS Method</h3>
<p>
Trifluoromethanesulfonic acid (CF<sub>3</sub>SO<sub>3</sub>H, MW 150.08,
CAS 1493-13-6) must be handled within a fume hood.  All reactions are
set up with gloved hand within a fume hood and possibly protective barrier.
TFMS is a pungent, light-yellow liquid with a 1.70 g/ml density and a
vapor pressure of 8 mm Hg at 25&deg;.  It is slightly soluble in water.
As for disposal, it is probably best to pass air over it and let it
go out a fume hood.
<p>
To control/test the effectiveness of this method, obtain a model glycoprotein
such as ribonuclease B (other glycoproteins include bovine fetuin, immunoglobulin
G, ovalbumin, and horseradish peroxidase).  Variables are time and temperature
of incubation.  Note that HRP is PNGase F-resistant because of its
fucose(&alpha;1&rarr;3)core composition.  The procedure below is given by
<a href="http://www.sigmaaldrich.com/sigma/bulletin/PP0510bul.pdf">Sigma</a>
<ol>
<li>Dry the RNase B (or substrate glycoprotein) in a glass vial about 2 ml
	with a screw top (any 12 &times; 32 vial with wide opening will do).
	Dryness is extremely important, since water can damage the reactant.
	Cool the vial to 2-8&deg;.  There should be about 1 mg RNase B in
	the vial.
<li>If you are going to include <b>anisole</b> as a free radical scavenger
	in the reaction (although optional, it is said to yield higher amounts of
	protein), then do the following:
 <ol style="list-style-type:lower-roman;">
 <li>Cool the anisole and the TFMS to 2-8&deg; (put on ice-water bath).
 <li>Multiply 15 &micro;l and the number of samples (volume A).  Multiply
 	140 &micro;l and the count of samples (volume T).  In a glass vial,
 	mix volume A of anisole with volume T of TFMS.
 <li>Cool the reaction mixture.  At this point, transfer ununsed TFMS to
 	one or more ampules, flush with inert dry gas, and re-seal with flame.
 <li>Cool the sample and/or control vial(s).
 <li>Add 150 &micro;l of the pre-cooled 10% anisole in TFMS and rapidly seal.
 	A white vapor may form but should dissipate after several minutes.
 <li>Shake the vial until it dissolves completely. Do not vortex.
 <li>Incubate on ice or at 2-8&deg; for 3 hours with occasional shaking.
 </ol>
 If you are <em>not</em> using anisole in the reaction:
 <ol style="list-style-type:lower-roman;">
 <li>Cool the TFMS (in ampule) to 2-8&deg;.  Note that it will not keep after
	opening as it is very hygroscopic.  Add 150 &micro;l to each sample
	(the ampule will generally do four 1 mg samples).
 <li>Mix by shaking until the protein is completely dissolved.  This takes
	2-5 min.  Do NOT vortex, since foam can form and lead to protein degradation.
 <li>The reaction vial is left on ice or at 2-8&deg; for an additional 25
	min with occasional shaking.
 </ol>
<li>Cool a 60% pyridine solution (Sigma P5496) and the sample vial to &minus;15&deg;
in a MeOH/dry ice bath (or place in the freezer for a time).  Do not mix these
solutions!  Just cool them.
<li>Add 4 &micro;l of 0.2% bromophenol blue (Sigma B1560) along inside wall of
vial.  The solution is now a red color.
<li>Immediately after adding indicator, add <em>dropwise</em> the 60% pyridine
to the vial.  Cool the vial between adding drops.  The first drop is highly exothermic
and takes more cooling.  Additional drops are less so.
<li>As more pyridine is added, the solution should turn from red to yellow as
the pH increases.  After adding an equal volume (150 &micro;l) of pyridine, a
fine precipitate may form.  Add 20 &micro;l water and shake and then the precipitate
dissolves.
<li>60% pyridine is added until the color becomes light purple or blue, making a
pH of about 76.  In general, about 300 &micro;l of 60% pyridine is added to a
150 &micro;l TFMS solution to give a final volume less than 500 &micro;l.
<em><strong>Note:</strong> the process of neutralization should be done quickly,
keeping the reaction mixture cold at all stages to minimize protein degradation.</em>
<li>A desalting method should now be used to remove low MW reactants/products
so the protein can be analyzed.
</ol>

<h2>Lectins and Lectin Chips</h2>
<p>
Lectins have affinity for certain configurations/branchings of the glycan
and can be used to make identifications which include the type
of branching (alpha or beta, 1&rarr;6, 1&rarr;4, etc.).

<h2>Glycoprotein Sample Enrichment</h2>
<p>
Glycoproteins can be concentrated/enriched using a variety of methods.

<ul>
<li><b>Lectin Affinity Chromatography/Resin</b>.  Concanavilin A (ConA),
wheat germ agglutinin (WGA), and modified ricin.  These can bind to
intact proteins or their peptides after digestion.
<li><b>Boronate</b>.  Boronates bind cis-diols, and
<i>m</i>-aminophenylboronic acid is usually available on solid media.
<li><b>GPI-specific proteins</b>.  A technique to selectively enrich GPI-anchored proteins and
release them from their membrane binding has been reported (see
Mann and Jensen).
</ul>

<h2>2D and 3D Mapping</h2>
<p>
Analytical separations of glycans can be achieved with a variety of
methods, including HPLC and TLC.

<h2>GLYCOCATCH</h2>
<p>
This technique reported by Hirabayashi and Kim and involves a defined
procedure of separating and analyzing multiple fractions of glycoproteins
in a very systematic approach.

<h2>Detection:  Gel or Membrane Staining for Glycoproteins</h2>

<h3>Periodic Acid/Periodate</h3>
<p>Periodic acid (HIO<sub>4</sub>) oxidizes cis-diols
(which saccharides have) to produce dialdehydes which can then be
reacted with a aldehyde-reactive dye to produce a color reaction,
or alternatively reduced with tritiated borohydride
(NaB[<sup>3</sup>H]<sub>4</sub> to track saccharides with a
radiolabel.  At lower concentrations, HIO<sub>4</sub> can be
used in the sensitive detection of sialic acids.</p>
<ol>
<li>Dissolve 100-1000 &micro;g glycoprotein in 2 ml 100 mM acetate,
pH 5.5 buffer containing the following concentrations of sodium
periodate (NaIO<sub>4</sub>):
 <ul>
 <li>for all monosaccharides: 15 mM</li>
 <li>for alditols: 8 mM</li>
 <li>for sialic acids:  1 mM</li>
 </ul></li>
<li>Keep reaction in dark at room temperature for 60 min for
oligosaccharides, at 4&deg; for 48 h for alditols, and at
0&deg; for 60 min for sialic acids.  Note periodate degrades
under light and should always be freshly prepared, and the reaction
should be kept in dark since nonspecific oxidation can occur</li>
<li>End the reaction by adding 25 &micro;l ethylene glycol and
leave at 4&deg; overnight.</li>
<li>Add 0.1 M sodium hydroxide (NaOH) until pH 7.0 is reached
(about 1.5 ml)</li>
<li>Labeling can be done by reduction with 25 mg
NaB[<sup>3</sup>H]<sub>4</sub> at 4&deg; overnight.  Alternatively
any amine-containing compound can also be used, such as fuchsin
or digoxigenin (used with anti-DIG coupled enhancement).</li>
<li>Acetic acid is added to pH 4.0 and the sample is concentrated
to dryness</li>
<li>Boric acid can be removed by evaporations with MeOH (100 &micro;l)
three times.  Acidic MeOH causes formation of volatile methyl
borate.</li>
<li>Use phenol/sulfuric acid assay to check on levels of mono- or
oligosaccharides with C-2 hydroxyl</li>
</ol>


<h3>Periodic Acid Schiff-Based Staining</h3>
<p>
Perhaps the oldest method for illuminating  glycoproteins.
Periodic acid oxidation produces the sugar dialdehydes which can then be
complexed to an amine dye (Schiff's base).
<p>
Sigma-Aldrich <b>GLYCOPRO</b>.  This kit is senstive for proteins
containing 25-100 ng carbohydrate, and will develop 10 minigels or
5 large gels or membranes.  The kit is composed of assembled reagents
already in the catalog (periodic acid, sodium metabisulfite reduction
reagent, Fuchsin-Sulfite/Schiff's reagent).
<p>
Sigma-Aldrich <b>SIALICQ</b>.  This is a multi-component kit which can
quantitate solutions containing <i>N</i>-acetylneuraminic acid (NANA).
It is a coupled assay in which a neuraminidase removes the
sialic acid groups and the products exposed to a NANA aldolase and
LDH in the presence of NADH and fetuin.
<h4>Moller &amp; Poulsen Method of SDS-PAGE Gel Staining</h4>
<p>
[from <i>Anal. Biochem.</i> <b>226</b>: 371, 1995]
<ol>
<li>The following steps are performed at 30&deg; with reagents
prewarmed:
 <ol>
 <li>Fix the gel in 10% trichloroacetic acid for 10 min
 <li>Wash gel twice for 2 min each in 5% acetic acid
 <li>Incubate gel in 1% periodic acid in 5% acetic acid for
 20 min
 <li>Wash twice for 2 min each in 5% acetic acid
 <li>Wash twice for 2 min each in water
 <li>Incubate in 0.5% potassium metabisulfite for 12 min
 </ol>
<li>The following steps are performed at 50&deg; with reagents
properly prewarmed:
 <ol>
 <li>Wash the gel twice with 25% EtOH + 10% acetic acid
  for 2 min each wash.
 <li>Stain with 0.125% Alcian Blue in 25% EtOH/10% acetic acid
 	 for 15 min
 <li>Wash the gel three times with 25% EtOH/10% acetic acid
 	for 1, 4, and 5 min in that order.
 <li>Continue with the silver staining process, omitting any
 first fixation steps.
 </ol>
</ol>



<h3>Hydrazide-Conjugated Direct and Indirect Illluminators</h3>
<p>
Sugars are also modified by hydrazides (-NH-NH<sub>2</sub>).
Hydrazides will form a hydrazone (-C=N-NH-) bond with aldehyde (-CH=O).
Sugars are briefly/mildly treated with periodic acid to produce the
aldehyde groups, and then treated with the hydrazide.
<p>
The hydrazide is conjugated to either a fluorophore, lumiphore,
or biotin.  Molecular Probes has a fluorescein semicarbazide
for direct fluorescence.  Sigma-Aldrich <b>GlycoProfile III
Fluorescent Glycoprotein Detection Kit</b> uses a dansyl hydrazide
dye; the kit includes a positive control.  Biotinylated hydrazides
are sold by Pierce and others for use with streptavidin-conjugated
peroxidase and detection on blots.

<h3>Fluorescent Labeling/Staining</h3>
<p>
<b>Sugar-specific fluorescent dyes</b> are available from Molecular Probes.
SYPRO dyes used in combination with <b>Pro-Q Emerald</b> can be used
to reveal glycoproteins among the unglycosylated proteins.
<p>
Sigma-Aldrich GlycoProfile 2-AA Labeliing kit (described below) can
be use for labeling glycoproteins in SDS-PAGE.

<h4>Pro-Q&reg; Emerald</h4>
<p>
Pro-Q Emerald is a fluorescent hydrazide that reacts with aldehydes.
To react with glycoproteins, the oligosaccharides are treated with periodic
acid oxidation to generate the aldehydes from the vicinal diols, and then
the dye is added.
<p>
The dye absorbs maximally at 288 nm and so should work with standard UV
transilluminators (302 nm).
<p>
This Invitrogen (acquired from Molecular Probes) product has limited reports
in the literature (perhaps because of cost or technical difficulty).
Kanninen et al (<i>Neurosci Lett</i> <b>367</b>: 235-240, 2004) used the
Pro-Q Emerald 300 (Invitrogen dye by a previous method for 2D gels:
proteins were fixed in 50% MeOH and then treated with 1% periodic acid in
3% acetic acid for 30 min to oxidize glycans.  After wshing with 3% acetic acid,
gels are incubated for 2 h in the PQE-300 dye (room temp, gentle agitation).
The gels are then illuminated.  Gels are then stained with SYPRO Ruby and
illuminated again; note that CandyCane MW markers were used in the 2nd dimension.

<h2>Sialic Acid &amp; De-sialylation</h2>
<p>
Sialic acid groups on a glycoprotein might cause it to form one or
more isoforms separable on an IEF gel because of alterations in the
isoelectric point.  The presence of sialic acid groups might be
confirmed by removal of the neuraminic acid groups by the enzyme
neuraminidase.  Sigma-Aldrich provides neuraminidase from
<i>Clostridium perfringens</i>:  N2876 purified but possibly containing
proteases; N2904 gamma-sterilized for cell culture use; N3001 much more
purified with 2-10-fold specific activity; N5631 with greater purity
and specific activity; N2133 affinity-purified with highest purity.


<h2>Chromatographic/Mass Spectrometric Analysis</h2>
<p>
Glycans from deglycosylated glycoproteins can be complexed with
flurometric substances for improving their detection.  Sigma-Aldrich
<b>GlycoProfile 2-AA Labeling Kit</b> (PP0530) is a kit using
2-anthranilic acid under reducing conditions (sodium cyanoborohydride)
to produce a fluorophoric complex.  The <b>GlycoProfile 2-AB Labeling
Kit</b> (PP0520) is a 2-aminobenzamide version.  Both kits work for
proteins to be analyzed by HPAE, HPLC, and ESI-MS.  The 2-AA kit is
also sensitive for SDS-PAGE proteins.

<h2>Sugar / Monosaccharide / Oligosaccharide / Glycan Chemistry</h2>

<h3>Phenol / Sulfuric Acid Assay</h3>
<p>
Caramelized sugar can be analyzed quantitatively using
spectrophotometry.
<ol>
<li>Prepare a range of hexose (galactose or mannose) standards
from 1 to 10 &micro;g in 10 &micro;l and pipet to a microtiter plate</li>
<li>Unknowns should be in the range for 1 &micro;g in 10 &micro;l.</li>
<li>Add 25 &micro;l 4% aqueous phenol to each well and mix well.
Leave for 5 min.
<li>Add 200 &micro;l <i>concentrated</i> sulfuric acid
(H<sub>2</sub>SO<sub>4</sub>)
<strong>very carefully</strong>.  The acid will <q>spit</q>
particularly in the presence of salt.  The acid will damage the
microplate reader as well if it is spilled.</li>
<li>Read at 492 nm.</li>
</ol>

<h3>Monosaccharide Analysis by HPAEC</h3>
<p>See Hounsell, Davies and Smith reference in Protein Protocols.  This
uses highly specialized equipment that includes biocompatible
chromatography system with pulsed amperometric detection</p>

<h3>Monosaccharide Analysis by Gas Chromatography</h3>
<p>See the Hounsell et al method described in Protein Protocols.</p>
<ol>
<li>Glycoproteins or oligosaccharides with 1-50 &micro;g carbohydrate and
10 &micro;g of internal standard (inositol or pereitol) are dried in
screw-top PTFE septum vials in vacuum over phosphorus pentoxide in dessicator.</li>
<li>With a gentle stream of nitrogen, add 200 &micro;l methanolic HCl.
Methanolic HCl can be obtained commercially, or by bubbling HCl gas into
MeOH until desired pH reached, or be adding molar equivalent of acetyl
chloride to MeOH.  This cleaves glycosidic bond and oligosaccharide-peptide
bonds to produce methyl glycosides and methyl carboxylates.  Should reducing
monosaccharides be required, hydrolysis can be done.</li>
<li>Cap immediately and heat at 80&deg; for 18 h.  Note that &alpha; and
&beta;-methyl glycosides and a mixture of pyranose and furanose rings form.
The four possible isomers form in ratios characterstic of the monosaccharide
and allow unambiguous assignment.</li>
<li>Cool vial, open add about 50 mg silver carbonate.  Mix and test for
neutrality, which is when solid silver carbonate goes from pink to green</li>
<li>Add 50 &micro;l acetic anhydride and leave at room temp for 4 h in dark.
This removes N-acetyl groups with acetyl groups.</li>
<li>Solid residues are spun out, and supernatant transferred to clean vial.</li>
<li>Add 100 &micro;l MeOH to precipitate to wash it, spin, and combine with
first supernatant. Repeat again.</li>
<li>Dry down the supernatant pool with nitrogen stream.  Continue drying over
phosphorus pentoxide.</li>
<li>Add 20 &micro;l of trimethylsilylating (TMS) reagent, such as Supelco's
Sylon HTP kit which is trimethylchlorosilane and pyridine hexamethyldisilazane.</li>
<li>Heat at 60&deg; for 5 min, then evaporate under nitrogen stream.
Add 20 &micro;l dry toluene (toluene stored over 3-&Aring; molecular sieve</li>
<li>Inject on capillary GC column, for example 25 m &times; 0.22 mm i.d.
BP10 (SGE, Austin TX) or 30 m &times; 0.2 mm i.d. ultra-2 (HP, UK).
Injection is at 14 psi He head pressure with a 130 to 230&deg; over 20 min
program, then hled at 230&deg; for 20 min.</li>
<li>Quantitate by calculating total peak area by sum of individual peaks
and divide this by peak area of internal standard.  Compare to the standard
curves for a molar determination.
</ol>

<!--  Phosphoproteomics -->
<h1 id="phosphoproteomics">Phosphoproteomics</h1>
<p>
Phosphorylation is nature&rsquo;s on-and-off switch for regulating
the activity or function of proteins.  It occurs in both prokaryotes
and eukaryotes.  The other approach to regulating activity is
protein synthesis and degradation, and to some extent bacteria make
use of it more than eukaryotes do.  Protein sythesis and degradation
has a heavy energy cost compared to phosphorylation, since putting
together peptdie bonds is huge requirment with respect to energy use.
Phosphorylation is seen in every biological process:  signal transduction,
gene expression, cell divsion, and in aberrant cases, cancer.
<p>
Serine (Ser, S), threonine (Thr, T) and tyrosine (Tyr, Y) are the amino
acids typically phosphorylated in eukaryotes.  Protein kinases exist in two major classes:
Ser/Thr protein kinases and Tyr protein kinases, which phosphorylate these
hydroxyl groups.
<p>
In prokarytoes, aspartic acid (Asp, D), glutamic acid (Glu, E),
and histidine (His, H) are phosphorylated.  Histidine can be phosphorylated
on both nitrogen (1- and 3-phosphohistidine) atoms of the imidazole
side chain.  Arginine phosphorylation involves formation of a
phosphorimide bond [O=P(OH)<sub>2</sub>-NH-C(=NH<sub>2</sub>)-NH-(CH<sub>2</sub>)-CH(NH<sub>2</sub>)-COOH
<p>
Both eukaryotes and prokaryotes will phosphorylate Arg, Lys, and Cys residues.
Phosphocysteine involves a phospho-sulfur bond.

<h2>Immunoaffinity Purification of Phosphoproteins</h2>
<p>
There are commercial preparation of antibodies that claim to be specific
for phosphoserine (PSer), phosphothreonine (PThr), and phosphotyrosine (PTyr).
Some monoclonals are useful in immunoprecipitation (with solid phase
protein A or G Sepharose) for whole extract enrichment.  Anti-PSer/PThr
are less reliable.
<p>
The antibodies work better against large phosphproteins than for smaller
phosphopeptides.

<h2>IMAC</h2>
<p>
<b>I</b>mmobilized <b>m</b>etal-<b>a</b>ffinity <b>c</b>hromatography
takes advantage of the ability of phosphate groups to bind (chelate)
metals, particularly transition metals.  These also work for polyhistidine-tagged
recombinant fusions, as the imidazole groups wrap around to coordinate
to the metal.
<p>
The immobilized metal is either Fe<sup>3+</sup> or Ga<sup>3+</sup>, although
the metal and the particular coordinate group varies.
<p>
It may be necessary to esterify carboxyl groups (C-term, Asp and Glu
side chains) since they have an anion-exchange affinity for the IMAC
columns.

<h2>Other Chromatography</h2>
<p>
Phosphopeptides can be enriched on reversed-phase columns as well
as porous graphitic carbon (PGC) beads.

<h2>Chemistry of Phosphopeptides and Phosphoproteins</h2>
<p>
Phosphates on serine and threonine side chains are good leaving groups
for nucleophilic substitution or elimination.  Adding a strong base (hydroxides)
will lead to &beta;-elimination, with formation of an alkene, which
is often &beta; to the carbonyl.  If ethanedithiol is added, this forms
a thioethylcysteinyl side chain.  Addition of N-biotinyl succinimide
results in the formation of a succinylimidyl biotin form of the peptide,
with the reaction at acidic to neutral pH.  Phosphopeptides can now be
enriched on avidin columns.
<p>
In another reaction, phosphopetides or proteins are fixed to a solid phase
(bead).  First the amino terminus is protected with tert-butyl oxycarbonyl
(tBOC (amine) and then the carboxyl terminal with ethanolamine
in the presence of N,N&prime;-dimethylaminopropyl ethyl carbodiimide HCl
(EDC).  The phosphate also is converted by EDC/ethanolamine
(-OP(=O/OH)-NHCH<sub>2</sub>CH<sub>2</sub>OH, a phosphoamidate).
When treated with acid (TFA), the phosphate is restored by hydrolysis.
The phosphopeptide must be cleaned up on a reversed phase extraction
column and then a second condensation with EDC and cystamine
(NH<sub>2</sub>CH<sub>2</sub>CH<sub>2</sub>-S-S-CH<sub>2</sub>CH<sub>2</sub>NH<sub>2</sub>)
is done.  DTT is added to reduce cystamine-converted phosphate for the
next step.  The phosphopeptide is again cleaned up on reversed phase.
The peptide is added to glass beads that have an iodoacetyl group attached
to the bead.  The thiol then is bound to the glass beads.  After all
other material is washed away, including all nonphosphopeptides,
the phosphopeptide is liberated from the glass bead by treating in
aqueous TFA.  The peptide has a free amino terminus and phosphate
although the carboxyl group is still modified.


<h2>Demodification of Phosphoproteins/Phosphopeptides (Dephosphorylation)</h2>
<p>
It is often useful to remove phosphates from proteins or peptides as a
means of analysis.  Essentially two ways can be tried: enzymatic and
chemical.

<h2>Chemical Dephosphorylation</h2>
<h3>Hydrofluoric Acid (HF)</h3>
<p>
This is from a method described by Kuyama et al. (<i>Rapid Commum Mass
Spectrom.</i> <b>17</b>: 1493, 2003).
<p>
If you intend to do a desulfation reaction on the proteins or peptides,
use 95% TFA first, which does not dephosphorylate proteins/peptides (or
use in a separate reaction).	Then use the chemical dephosphorylation
step, which removes both sulfate and phosphate.

<ol>
<li><b>Reagent</b>.  There is choice with the HF reagent to be used.
The first reagent is typically incubated at 3 hours in an ice-water bath,
but it does not completely dephosphorylate threonine residues.  When used
for 1.5 hours at room temperature, the dephosphorylation of Thr is more
complete.  The second reagent complete desphosphorylation of all serines,
threonines and tyrosine residues on an ice-water bath within 60 min.
 <ol>
 <li><b>50% hydrofluoric acid (HF)</b>.  This obtained commercially usually
at this concentration.  If you have a 40% reagent, that will be acceptable.
Lower than 40% is untested (you will have to make a determination yourself).
 <li><b>70% HF in pyridine</b>.  This is a commercially available preparation.
 </ol>
<li>Approximately 50 &micro;l of HF reagent will be used per 100 &micro;g
protein to be dephosphorylated.  Choose the appropriate size of the reaction
tube.  Generally a standard 1.5 ml conical polypropylene tube with tightly
sealing cap is sufficient.
<li>Pipet the appropriate volume of the protein to the tube.
<li>Dry down the protein solution in a centrifugal evaporator.  If the
volume is large and it is taking a lot of time, addition of an organic
solvent (acetonitrile, methanol, ethanol) from one-third to one-half the
volume present can accelerate the drying process.  There should be no
concern about denaturation.
<li>When the tube is completely dry, add 50 &micro;l per 100 &micro;g protein
present.  Put on an ice-bath or at room temperature, depending on what you
want to achieve, and incubate for the appropriate length of time.
<li>The HF reagent must be removed by vacuum centrifuge.  You are advised
to set up a NaOH or CaO trap prior to the cold trap in order to trap HF.
This will prevent corrosion and the effects of any toxicity.
</ol>
<p>
Note.  You may want or need a zero time point incubation, so set up the
incubation if necessary to take out aliquots at a zero and perhaps other
time points.

<h2>Commercial Products</h2>

<h3 id="PhosphoProfile1">Sigma-Aldrich PhosphoProfile&trade;
Enrichment Kit</h3>
<p>Sigma-Aldrich provides the PhosphoProfile Enrichment Kit (Cat #PP0410)
for the enrichment of phosphopeptides to be used with an LC-MS/MS
method.  The kit provides the ability to process 24 samples
containing up to 25 nmol phosphopeptides.  This is a variation on
IMAC techniques.
<p>
Samples are digested in <b>Proteomics Grade Trypsin</b>.  Digests are
then passed through <b>PHOS-Select Gallium Spin Columns</b>, which
specifically bind phosphate-containing peptides in a Ga<sup>3+</sup>
silicia medium.  The kit includes controls, buffers/reagents, and
even plasticware (collection tubes).  Samples are ready for LC-MS/MS.

<h2 id="phosphoprotein-staining">Staining Phosphoproteins In
	Electrophoretic Gels</h2>

<h3>Cutting &amp; Roth</h3>
<p>
[Cutting JA, Roth TF (1973). Staining of Phospho-Proteins on Acrylamide Gel
Electropherograms.  <i>Anal. Biochem.</i> <b>54</b>: 386-394.

<ol>
<li>Fix gels in 10% sulfosalicylic acid (SSA).  The time of fixation should be
sufficient to allow any free phosphate and low MW phospho compounds to diffuse
out of the gel.  In serum samples, this could take as long as 12 h with several
changes of fixative; the step can be omitted when no phosphate is present.
<li>Gels are transferred to fresh 10% SSA containing 0.5 M calcium chloride
	for 1 h or longer.  (Note several concentrations of calcium chloride were tested
	and 0.5 M was optimal.)
<li>Gels are rapidly rinsed in deionized water to remove surface presence of
	calcium chloride, which could cause irregular staining.
<li>The gel is placed in 0.5 N NaOH at 60&deg; for 30 min.  Note the gel shrinks
as hydrolysis ensues, a function of time and temperature of incubation and
of hydroxide concentration.  Different concentrations (0.1, 0.5, 1.0, and
2.0) of hydroxide, times (30, 60, 120 min), and temperatures (37, 60, 80&deg;)
were all tried.
<li>Rinse twice for 10 min each with 1% ammonium molybdate, letting the ions
penetrate gel before decreasing pH, which forms the phospho-molybdate complex.
If acidification occurs prior to molybdate complexation, the calcium phosphate
solubilization occurs with the result of broader bands and missing fine bands.
The second molybdate rinse is done to reduce the level of NaOH.
<li>Gels are transferred to 1% ammonium molybdate in 1 N nitric acid for 30 min.
<li>Transfer to a 0.5% methyl green in 7% acetic acid solution for 30 min.
<li>Destain with several changes of 10% SSA.
<li>Gels are best kept in 7% acetic acid.
</ol>

<h3>Pierce GelCode Phosphoprotein Staining Kit</h3>
<p>
This kit (Prod Num 24550 for the stain reagent + control proteins, 24551
for stain reagent kit only, 24552 for control proteins only) provides for
the staining of ten 8&times;8 cm mini-gels and one, maybe two, large
(20&times;20 cm) 2D gels.  This kit is based entirely on the Cutting &amp;
Roth method, so there is nothing proprietary about it.

<ol>
<li>Preheat an incubation oven at 65&deg;.  A water bath produces poor results.
<li>Wash the gel for 10 min in ultrapure water.  In all steps the gel must be
submerged.
<li>Discard the water and add 15 ml Reagent 1 (Salicylic Acid, 1 L bottle) for
	15 min agitating gently.
<li>Discard Reagent 1, and incubate in 25 ml Reagent 2 (Salicylic Acid + CaCl<sub>2</sub>,
	250 ml) for 30 min, agitating gently.
<li>Discard Reagent 2 and briefly rings gel with ultrapure water to eliminate any
	CaCl<sub>2</sub> on the gel surface.
<li>Discard rinse water, and add 25 ml Reagent 3 (0.5 N NaOH, 250 ml), cover the
tray and incubate at 65&deg; in oven for 20 min.  There is no gel agitation needed.
(<b>Note</b>:  the gel swells with an increase in 10% of its volume, so use
a tray large enough to handle the increase in size.  Formation of a white precipitate
is normal in this step.)
<li>Discard Reagent 3 and add 25 ml Reagent 4 (500 ml ammonium molybdate solution)
	and agitate for 10 min.
<li>Repeat this step with another 25 ml Reagent 4.
<li>Discard the reagent, and add 25 ml Reagent 5 (ammonium molybdate + nitric
	acid, 250 ml) and agitate for 20 min.
<li>Discard Reagent 5 and add 25 ml Reagent 6 (methyl green soln, 250 ml)
	and incubate for 20 min.
<li>Discard reagent 6 and add 25 ml Reagent 1 and agitate for 15 min.
	Repeat the washing with Reagent 1.  Green phosphoprotein bands should be
	visible against the faint green background.
<li>Continue destaining in 7% acetic acid overnight.
</ol>
<p>
To do total protein staining, wash the gel three times with 100 ml ultrapure water
for 10 min each.

<h1 id="quantitative-proteomics">Quantitative Proteomics</h1>

<h2>Label-Free Quantitation</h2>


<!--  =========================================================
                      Structural Proteomics
       ========================================================  -->
<h1 id="structprottools">Structural Proteomics Tools</h1>

<h2 id="chemmod">Protein/Peptide Modifying Chemistry</h2>
<p>
Several chemical methods are available to cause changes typically in the
side chains of proteins.  These methods are analytical and preparative.

<h3>Amino Acid Analysis</h3>
<p>
The amino acid content of a pure protein or peptide can be determined
quantitatively for most of the 20 residues by acid hydrolysis followed by
HPLC separation using optimized analytical methods.
<ol>
<li>Prepare at least 100 pmol (typically tens of nanograms of typical
intact proteins) of the protein or peptide.  Ideally it is dried down in
a solution that contains little to no salt or detergent.  Add no more than
5 nmol (about a few micrograms).  It is best to add it to a glass ampule
that will be flame-sealed later.
<li>Add 50-200 &micro;l of 6 N high grade hydrochloric acid (HCl).  You
should use a reagent specifically designed for use in amino acid analysis
(Pierce Chemical makes such HCl).
<li>Using a glass wool- or suitably-filtered nitrogen gas line, stream in
nitrogen gas into the ampule for at least 60-120 seconds to fully flush
out the atmosphere.  Have a Bunsen burner flame ready with hot blue
inner cone.
<li>Remove the ampule from the nitrogen gas flush line, and immediately
flame-seal the ampule.
<li>Place the ampule either in an oven or on a heat block that is set
to 100 &plusmn; 2&deg;.
<li>Leave the ampule for 24 h.
<li>Remove the ampule from heat and allow to cool to room temperature
before opening.
<li>Break open the cooled ampule.
<li>If the sample is to analyzed on HPLC, transfer all or part of the
sample required to a container like an autosampler vial (or glass conical
insert for one) that will fit into centrifugal evaporator (vacuum centrifuge).
Dry down the sample in the centrifugal evaporator.  Add a water/acetonitrile
mix in small volume (25 &micro;l) and re-dry it down to make sure all the
hydrochloric acid is removed.
<li>The amino acid hydrosylate can be resuspended in a column equilibration
mobile phase/solvent and used in whatever manner intended.
</ol>

<h4>Other Methods</h4>
<p>Other methods use hydrolysis without acid catalysis.  See <q>Subnanomole
Range Amino Acid Analysis</q> by P. E. Hare (1977)
<i>Meth. Enzymol.</i> <b>47</b>: 3-18 for such methods.

<h3>N-Terminal Residue Chemistry</h3>

<h4>Dinitrofluorobenzene (DNFB)</h4>
<p>
DNFB is called Sanger&rsquo;s reagent because it was Sanger who found it.


<h4>Edman Chemistry</h4>
<p>
Edman degradation is usually run in an automated fashion, combined with
immediate HPLC analysis.  Applied Biosystems, Inc. was probably the first
company to make automated Edman cyclers and its instruments are the best
on the market.  Its reagents are without equal.
<p>
Edman chemistry can be done manually, although it is tedious.  Depending
on sample amounts, at least a few cycles can be done by hand and information
can be obtained by it.
<ol>
<li>You will need at least 100 pmol for manual work.  The automated instrument
can use as little as 5 pmol.
<li>The peptide/protein must be as clean and free of salts and detergents
as the material used for amino acid analysis (above).  In fact, more so.
<li><b>Bind to membrane</b>.  The peptide/protein is bound to
solid phase material such as nitrocellulose
or hydrophobic PVDF.
  <ol>
  <li>Wet the membrane in appropriate material (for PVDF, use methanol).
  <li>Wash the membrane in ultrapure water.
  <li>Use a scalpel or Xacto knife to cut a 2-3 mm diameter circle or square
      from the membrane.
  <li>Place in the reaction tube where all Edman chemistry will be done.
    (might be a glass tube or conical plastic tube).
  <li>Remove all fluid from tube and leave membrane.
  <li>Add protein/peptide solution to membrane.  Let sit 10 minutes.
  <li>Wash membrane with several changes of
  </ol>
</ol>

<h3>Cys-S-S-Cys Disulfide Bond Chemistry</h3>

<h4>Reduction</h4>
<p>
This reaction is described all over this reference.  The most effective
reagents are the intramolecular dithiols (that is, the dithiols are
on one molecule, creating a concerted reaction).  These are dithiothreitol
(DTT) and its diastereomer dithioerythritol (DTE).  The monothiol
2-mercaptoethanol (&beta;-mercaptoethanol) is also effective and
relatively inexpensive and probably the first ever to be used.

<h4>Oxidation</h4>
<p>
Peformic acid, H(C=O)-O-O-H, will thoroughly oxidize the disulfide
link to form -Cys-CH<sub>2</sub>-SO<sub>3</sub><sup>&minus;</sup> anion.

<h3>Methionine</h3>
<h4>Cleavage</h4>
<p>
Cyanogen bromide (CNBr) in anhydrous formic acid will result in the
carboxyl side cleavage of Met.  Inglis &amp; Edman (1970) reported
that methyl thiocyanate (CH<sub>3</sub>-S-CN) is formed.  They identified
a cyclic imidate.  This occurs when the methyl thiocyanate leaves with
a very reactive carbonium bonding the the carbonyl oxygen to form
the cyclic imidate.
<p style="text-align:center;">
  -NH-CH...-(C=NH-R)-O-CH<sub>2</sub>-CH<sub>2</sub>-....
<p>
This method  from Crimmins DL, Mische SM, Nancy D. Denslow ND (2000) <i>
Curr. Prot. Protein Sci.</i>, Unit 11.4.1-11.4.11
<ol>
<li>To a lyophilized 1-50 &micro;g protein sample in 1.5 ml conical tube, add
80 &micro;l 88% HCOOH (formic acid), 15 &micro;l water, and 5 &micro;l of
5 M CNBr freshly made in MeCN.  This assumes that Met residues make up 2.3%
of residues of proteins on average and a molar ratio of 200:1 CNBr:Met in the
reaction.  Where Met is higher in content, use more in adjustment.
<li>Vortex until there is no solid material present.  Wrap in aluminum foil or
place in light-tight opaque container.  Leave 24 h in fume hood. {Unused CNBr
should be disposed of by slow addition of concentrated [10 M] NaOH, about 5-10
vol; the reaction is quite exothermic.)
<li>Add 10 volumes of Type I water and lyophilize overnight or use centrifugal
evaporator to dry the sample.
</ol>
<p>
The sample might be analyzed using RPLC or used in electrophoresis if the reaction
was preparative.  Use warm water and/or sonication to help in resolubilization.
Sometimes a chaotropic agent or detergent might be required, as cleavage products
acquire different chemical properties.

<h4>Carboxymethylation</h4>
<p>
Methionine is reactive to alkylation by iodoacetate, forming a sulfonium
zwitterion:  Met-CH<sub>2</sub>-CH<sub>2</sub>-S-CH<sub>3</sub> +
ICH<sub>2</sub>-COO<sup>&minus;</sup> &minus;&gt;
Met-CH<sub>2</sub>-CH<sub>2</sub>-S<sup>+</sup>(CH<sub>3</sub>)-CH<sub>2</sub>-COO<sup>&minus;</sup>
<h4>Oxidation</h4>
<p>
Performic acid, H(C=O)-O-O-H, treatment results in the formation of
methionine sulfone:  Met-CH<sub>2</sub>-CH<sub>2</sub>-S(=O)(=O)-CH<sub>3</sub>


<h3>Carboxylic Acids Aspartic and Glutamic Acid</h3>

<h4>Methyl Esterification by Diazomethane</h4>
<p>
Diazomethane CH<sub>2</sub>=N<sup>+</sup>=N<sup>&minus;</sup> is a
well known very reactive methylator of carboxylic acids, forming the
methyl ester and nitrogen gas as products.

<h4>Amidation by Carbodiimides</h4>
<p>
EDAC is 1-ethyl-3-(3-dimethyl)aminopropylcarbodiimide,
CH<sub>3</sub>-CH<sub>2</sub>-N=C=N-(CH<sub>2</sub>)<sub>3</sub>-N(CH<sub>2</sub>)<sub>3</sub>.
In the presence of glycine methyl ester, CH<sub>3</sub>-O-C(=O)-CH<sub>2</sub>-NH<sub>2</sub>,
it will result in the amidation of carboxylic acids:
CH<sub>3</sub>-O-C(=O)-CH<sub>2</sub>-NH-C(=O)-CH<sub>2</sub>-Asp/Glu-

<h3>Histidine</h3>

<h4>Alkylation</h4>
<p>
Iodoacetate reacts with many amino acids (Asp, Cys, Met, Glu, Lys).
The secondary amine of the imidiazole ring will also displace the halogen
in a substitution reaction:
His-CH<sub>2</sub>-...CH-NH-CH=N-CH=... + ICH<sub>2</sub>COO<sup>&minus;</sup>
&minus;&gt; His-CH<sub>2</sub>-...CH-N(CH<sub>2</sub>-COO<sup>&minus;</sup>)-CH=N-CH=... + I<sup>&minus;</sup>
<p>
Note that ...A-B-C-D-E... is a circle where the three dots means that
A and E bond to each other in the circle.

<h4>Ethylcarboxamidation</h4>
<p>
Diethylpyrocarbonate (DEPC) modifies the imidazole group on the secondary
amine as well.
His-CH<sub>2</sub>-...CH-NH-CH=N-CH=... +
CH<sub>3</sub>-CH<sub>2</sub>-O-C(=O)-O-C(=O)-O-CH<sub>2</sub>-CH<sub>3</sub>
&minus;&gt; His-CH<sub>2</sub>-...CH-N[-C(=O)-O-CH<sub>2</sub>-CH<sub>3</sub>]-CH=N-CH=...
<p>
Carbon dioxide and ethanol would be the other products.


<h3>Arginine</h3>

<h4>1,2-Cyclohexanedione</h4>
<p>
Described by Toi et al (1965) <i>J. Biol. Chem.</i> <b>240</b>: PC3455-PC3457, 
arginine in the presence of 1,2-cyclohexanedione in aqueous 0.2 NaOH has its 
guanidinyl function changes into a bicyclic 
cyclotetramethylene-4-imidazolidinone.  This compound was synthesized also by a 
confirmatory reaction between 
<i>S</i>-methyl-5,5-cyclotetramethylene-2-thiohydantion and 
&alpha;-formyl-L-ornithine followed by acid hydrolysis.  A melting point 
determination showed decomposition at 240&deg; and optical activity of 
[&alpha;]<sub>D</sub><sup>20</sup> = +18.2 &plusmn; 0.5&deg; (<i>c</i>, 1, in 5 
N HCl).  It is stable in 6 N HCl at 110&deg; for 20 h.
<p>
<img src="imagesProteomicsReferenceHTML/Arg-Cyclohexanedione.gif"
	style="width:auto;display:block;margin:0.5em auto;"
  alt="reaction of arginine with 1,2-cyclohexanedione to form imidazolidone">
<p>
Protein treatments are possible and show similar results.  Up to 10 mg/ml in 
0.2 NaOH with 10-fold excess of the cyclohexanedione at 25&deg; was tested by
Toi et al.  Reactions can be dialyzed or diafiltered against water, then lyophilized,
with about 75% recovery.

<h3>Lysine</h3>

<h4>Succinylation</h4>

<h3>Tryptophan</h3>
<p>
<b>Cleavage</b>.  The method below describes how to cleave peptides containing
unoxidized tryptophan using BNPS-skatole (from Crimmins et al, 2000).
<ol>
<li>1-50 &micro;g protein sample is lyophilized 500 &micro;l conical tube, making
sure the final solvent is Type I water or dilute acid (0.1% TFA, or 1% formic or
acetic acid)
<li>Add 60 &micro;l of 1 mg/ml BNPS-skatole (Pierce) prepared freshly in aldehyde-free
glacial acetic acid (Mallinckrodt).  Wrap in alumimum foil or place in light-tight
container.  Leave 30 min at room temperature.  Cleavage may be improved by
adding chaotropic agents (8 M urea, 6 M guanidine HCl), which are compatible
in the reaction.  In addition increases in incubation time and temperature may
also produce a more optimal reaction.  If enough sample is available, consider
performing experiments in which incubation temperature spans the range of room
temperature to 47&deg;, and then from that result, check incubation time from
30 min to 24 h.
<li>80 &micro;l of Type I water is added at end of incubation to precipitate
the BNPS-skatole.  (Organic extraction is an alternative, using ethyl acetate
or ethyl ether: see Rahali and Gueguen 1999.)  Spin at maximum speed in microfuge
at room temperature.
<li>Transfer supernatant to new 500 &micro;l tube and lyophilize or use
centrifugal evaporator.  Resolubilization can be done appropriate to the next
step.  Mild heat and sonication might be done first, and use of chemical
solubilizers considered later.
</ol>
<p>
2,4-Dinitrophenylsulfenyl chloride, Cl-S-C<sub>6</sub>H<sub>3</sub>(NO<sub>2</sub>)<sub>2</sub>
reacts with the side chain.
<p>
<img src="imagesProteomicsReferenceHTML/Trp-DPSC.gif" alt="reaction of 2,4-dinitrophenylsulfenyl chloride with tryptophan">

<p>
N-Bromosuccinimide oxidizes many amino acids.
<img src="imagesProteomicsReferenceHTML/Trp+NBS.gif" alt="reaction of N-bromosuccinimide with tryptophan">

<h3>Tyrosine</h3>
<p>
Tetranitromethane, C(NO<sub>3</sub>)<sub>4</sub>, will nitrate the
hydroxyphenyl ring in a classic reaction of organic chemistry.  Because
the hydroxyl adduct is an ortho,para activator, the nitration will be
ortho to the hydroxy group, forming a 3-nitrotyrosine.

<h3>Asp-Pro Bonds</h3>
<p>
Formic acid will enable cleavage on the C-terminal side of Asp in an Asp-Pro
combination.  The mechanism likely combines the protonation of the &alpha;-nitrogen
of Pro with a nucleophilic attack by the aspartyl carboxyl side chain to form
favorably 6-member ring intermediate, with hydrolysis then ensuing.
<ol>
<li>Prepare 1-50 &micro;g of lyophilized protein in a 500 &micro;l conical tube.
Add 50 &micro;l of 70% formic acid of highest grade, prepared fresh from 88%
HCOOH stock (Fluka).  Vortex to dissolve.  Incubate 37&deg; for 48 h.
<li>To the hydrosylate add 150 &micro;l Type I water.  Lyophilize or dry in
centrifugal evaporator.
</ol>
<p>
Sample can be resolubilized with solvents or reagents appropriate to next step.
Use mild heat and/or sonication at first before trying to add any chaotropic agents
or detergents if they are not called for.  See Crimmins et al <i>Curr. Prot.
Protein Sci.</i>

<h3>Asn-Gly Bonds</h3>
<p>
<ol>
<li>To 1-50 &micro;g of lyophilized protein in a 500 &micro;l conical tube,
add 200 &micro;l of 1.8 M hydroxylamine.   Vortex to dissolve.  Incubate 45&deg;
for 3 h.
 <ul>
 <li>To prepare the hydroxylamine, put 5 g hydroxylamine in 50 ml grad
  cylinder, add 20 ml of 5 M NaOH and continue adding until the solid dissolves.
 <li>Add  1 g sodium carbonate.
 <li>Adjust to pH 9 with concentrated HCl.
 <li>Add water to 40 ml and readjust to pH 9 if ncessary.  Should be used
same day.
 </ul>
<li>Let cool to room temperature.  Slowly add 10 &micro;l of 10% TFA.
<li>Either reversed phase or size-exlcusion chromatography can be used to desalt
the sample.  This might be necessary for SDS-PAGE analysis as the salt is quite
high.
</ol>

<h3>Cys</h3>
<p>
The Cys side chain can be modified with 2-nitro-5-thiocyanobenzoate (NTCB), and
the cyanylate product will cleave on the N-terminal side.  This creates
an iminothiazolidine carboxyl ring on the N-terminal of the Cys residue, and
so this product cannot be sequenced by Edman chemistry.
<ol>
<li>Prepare fresh 20 mM NTCB:  9 mg dissolved in 2 ml of 200 mM Tris acetate pH 8.
<li>Dissolve 25-50 pmol of protein in 200 mM Tris acetate pH 8 containing 1
mM EDTA and 1% SDS.
<li>Fully reduce the Cys thiol using Ekathiol resin (Ekagen), using 10-20&times;
molar excess, and keep tubes under nitrogen and shake 2 h at room temperature.
Ekagen has 0.3-0.7 mmol SH per gram resin.  If resin is not available, 1 mM DTT
can be used, but more NTCB must be used to compensate for the effect of reacting
with DTT present.  This reduction step should not be done if there are cystine
bridges that are to be mapped.
<li>Spin out the resin and transfer supernatant to new tube.  Add 20 mM NTCB
stock (usually about 5 &micro;l) to the solution so that the final concentration
is 4 mM NTCB, a 5-fold dilution.  Seal under nitrogen and incubate 40&deg; for
20 min.  Note that the NTCB should be 10-fold in excess of thiol groups present.
<li>The protein can be purified either by ultramicrospin gel filtration column
or by acidified acetone precipitation.
 <p><b>Gel filtration spin column</b>:
 <ol>
 <li>Add 100 &micro;l distilled or Type I water and mix.
 <li>Transfer mix to top of the spin column according to direction.
 <li>Spin according to directions:  usually 5 min at 1000 <i>g</i>.
 <li>Put the filtrate on a centrifugal evaporator to concentrate to a volume
reduced by about 20 &micro;l.
 <li>Add equal volume of 100 mM sodium borate pH 9.  Incubate 50&deg; for 60 min.
 </ol>
 <p><b>Acidified acetone precipitation</b>:  the protein should be at least
1 &micro;g/ml.
 <ol>
 <li>Add 9 volumes of ice-cold acidified acidified acetone.
 <li>Leave overnight at &minus;20&deg;, or put 30 min in dry ice/EtOH bath (or
put at &minus;80&deg;).
 <li>Spin 5 min in microfuge at max speed.  Carefully remove supernatant without
disturbing the pellet.  Add ice-cold acetone to pellet, mix, and re-spin, and
again remove the supernatant.
 <li>Let the pellet air dry.
 <li>Re-dissolve in 20 &micro;l 100 mM sodium borate, and add 20 &micro;l of
water.  Incubate 50&deg; for 60 min.
 </ol>           --
<li>Take 1 &micro;l aliquot and make sure pH is 9 using pH paper.  Use 2 mM NaOH
to adjust pH if necessary.  Vortex to make sure samples are dissovlved.
Incubate 1 h at 50&deg;.
<li><b>MALDI ToF MS</b>.  If MALDI MS is possible, do the following:
 <ol>
 <li>Prepare a ZipTip with 10 &micro;l MeCN, then 10 &micro;l 50% MeCN in 0.1% TFA,
and then twice with 10 &micro;l 0.1% TFA.  Keep the tip wetted until just before
use.
 <li>After alkaline treatment to cleave is complete, add 10 &micro;l of 0.5% TFA,
or so final solution is 0.1% TFA.
 <li>Draw the sample back and forth through the ZipTip, and only do it 10 &micro;l
at a time.  Save the material in another tube, in case the ZipTip is oversaturated.
 <li>Wash ZipTip two times in 10 &micro;l 0.1% TFA
 <li>Elute with 10 &micro;l 60% MeCN, or alternatively a solution containing
the matrix and spotting directly to target plate.  A 10 mg CHCA recrystallized
can be dissovled in 50% MeCN + 0.1% TFA (it is saturating) and the supernatant
used.
 </ol>
</ol>

<h2 id="bifuncxlink">Bifunctional Crosslinking Reagents</h2>

</div> <!-- class=contentblock -->

<div class="widetable">
<table>
<tr><th>Reagent  <th>MW/CAS#&nbsp; <th>Structure  <th>Other Name(s)  <th>Use

<tr><td>Glutaraldehyde
 <td>
 <td>
 <td>
 <td>probably the oldest crosslinker, amine-reactive primarily

<tr><td>1,4-Bis[3-(2-pyridyldithio)propionamido]butane
 <td>MW 482.71<br>CAS#&nbsp;141647-62-3
 <td><img src="imagesProteomicsReferenceHTML/dpdpb.gif"  alt="chemical structure of crosslinker DPDPB">
 <td><i>N,N'</i>-Tetramethylenebis[3-(2-pyridyldithio)propionamide]<br>
1,16-Di(2-pyridyl)-1,2,15,16-tetrathia-6,11-diazahexadecane-5,12-dione<br>
 	DPDPB
<td>cleavable homobifunctional SH-reactive crosslinker (via cysteine)
 <br>19.9 E spacer

<tr><td>Bis[2-(N-succinimidyl-oxycarbonyloxy)ethyl] sulfone
 <td>MW 436.35<br>CAS#&nbsp;57683-72-4
 <td>
 <td>BSOCOES
 <td>homobifunctional crosslinker (&ldquo;molecular ruler&rdquo;)<br>
  water soluble, cleavable in pH 11.6, 2 h, 37&deg;, amino-reactive

<tr><td>Bis[2-(4-azidosalicylamido)ethyl] disulfide
 <td>MW 474.52<br>CAS#&nbsp;199804-21-2
 <td><img src="imagesProteomicsReferenceHTML/BASED.gif" alt="chem structure of BASED">
 <td><br>N, N'-(Dithiobis-ethylene)bis(4-azido-2-hydroxy-benzamide)<br>
  N,N'-Bis(4-azidosalicoyl)cystamine<br>
	BASED
 <td>cleavable homobifunctional photo- and amino-reactive crosslinker

<tr><td>Dimethyl 3,3'-dithiopropionimidate dihydrochloride
 <td>MW 309.28<br>CAS#&nbsp;38285-78-8
 <td><img src="imagesProteomicsReferenceHTML/DTBP.gif" alt="chem structure of DTBP">
 <td>Dimethyl 3,3'-dithio-bis(propionimidate) dihydrochloride<br>
  Wang/Richards'reagent<br>DTBP
 <td>cleavable, homobifunctional cross-linker; reactive with primary amines
   at pH 7-10 (forms amidine bonds); can activate solid supports;
	cleaved by presence of mild reducing agents; linker spans 8 atoms

<tr><td>Ethylene glycol-bis(succinic acid N-hydroxysuccinimide ester)
 <td>MW 456.36 <br>CAS#&nbsp;70539-42-3
 <td><img src="imagesProteomicsReferenceHTML/EGS.gif" alt="chem structure of EGS">

 <td>Ethylene glycol bis(succinimidyl succinate)<br>
  Di(N-succinimidyl) ethylene glycol disuccinate<br> EGS<br>
  Ethylene glycol disuccinate di(N-succinimidyl) ester
 <td>cleavable crosslinker for protein linking and reversible immobilization

<tr><td>Sebacic acid bis(N-succinimidyl) ester
 <td>MW 396.39 <br> CAS#&nbsp;23024-29-5
 <td><img src="imagesProteomicsReferenceHTML/DSSeb.gif" alt="chem structure of DSSeb">
 <td>Di(N-succinimidyl) sebacate <br>
 	DSSeb <td>
</table>
</div> <!-- class=widetable -->

<div class="contentblock">

<h2>Quantitation of Specific Amino Acids, Derivatives, Side Chains</h2>

<h3>Thiol Quantitation</h3>

<p><b>Ellman's Reagent</b>.  5,5'-dithio-bis(2-nitrobenzoic acid) (DNTB) [CAS
69-78-3] is a disulfide that is cleaved by many thiols.  The reaction is as
follows: <p class="eqnserif" style="text-align:center;"> R-SH + DTNB &rarr;
R-S-S-C<sub>6</sub>H<sub>4</sub>(NO<sub>2</sub>)(COOH) + 2-nitro-5-thio-benzoic
acid (TNB)
<p>TNB in neutral or alkaline pH becomes di-anionic and produces a yellow color
that at 412 nm has an extinction coefficient of 14,150 M<sup>&minus;1</sup>
cm<sup>&minus;1</sup> in low ionic strength and 13,700 M<sup>&minus;1</sup>
cm<sup>&minus;1</sup> at high ionic strength (6 M guanidine HCl, 8 M urea).
Ellman's reagent might require recrystallization for purity (according to 1983
Meth. Enzymol. article).

<h3>Methionine Sulfoxide Quantitation</h3>
<p>
A <a href="http://www.ncbi.nlm.nih.gov/pubmed/6137483">method by Levine</a>
for quantiation of methionine sulfoxide is reported using acid hydrolyzed
CNBr-digested samples.

<h3>Aromatic Amino Acids</h3>
<p>A multicomponent analysis of the near ultraviolet spectrophotometric scan
of a purified protein can often be used to assess the number of aromatic
residues, as reported by
<a href="http://www.ncbi.nlm.nih.gov/pubmed/7093207">Levine and Federici</a>.

<h3>Quantifying Oxidation of Proteins</h3>
<p>Specific oxidants are often used on proteins.  The oxidation of proteins can
be measured themselves, or the amount of oxidant consumed can also be
done.

<h4>Measuring Consumption of Oxidant</h4>
<p><b>Iodometric analysis</b>.  Oxidation of I<sup>&minus;</sup> to
I<sub>2</sub> and then to periodide, I<sub>3</sub><sup>&minus;</sup>.
Periodide absorbs at 353 and 287.5 nm with extinction 22,900 / M-cm
with excess I<sup>&minus;</sup> present.

<p><b>TNBA</b>.  5-thio-2-nitrobenzoic acid (TNBA) [CAS #]
(not to be confused with Ellman's reagent, which is TNBA's dimolecular
disulfide form!) is quite effective at being oxidized by a broad array of
oxidants.  It is produced using as a precursor Ellman's reagent, reducing
1 mol reagent to produce  2 mol of TNBA.  TNBA in dianionic form in neutral
or alkaline solutions can be read spectrophotometrically in low ionic strength
solutions at 412 nm with extinction coefficient of 14,500 / M-cm, and with
coefficient of 13,600 / M-cm at higher ionic strength solutions, especially
in denaturing salts.

<h2>Peptide Separation &amp; Analysis</h2>
<p>
Peptides created by chemical and/or enzymatic digestion or cleavage of a 
polypeptide can be separated (or purified) from each other and analyzed by 
numerous means.  Perhaps the classic means is reversed-phase HPLC with an 
octadecylsilane (ODS), namely linear C<sub>18</sub> bonded to silica particle or 
to a monolith, using aqueous solvent containing an ion-pairing agent 
(trifluoroacetic acid, to 0.1% v/v) and with an increasing gradient of 
acetonitrile (MeCN) over time.  Peptides are detected on the UV absorbance 
maximum of the peptide bond (around 214 nm), but if a detector has 
multiwavelength monitoring ability, it should include wavelength monitoring at 
254 and 280 nm (and perhaps 190 or 203 nm, but only with the highest grades
of MeCN).  Detection can be improved using pre- or post-column derivatized
peptides which are tagged with a fluorophore and detected by an on-column
fluorimeter.
<p>
Peptides containing radiolabel or which can be uniformly radiolabeled 
can be detected with if the HPLC uses a radiometric detector, or fractions can
be collected and measured in gamma or scintillation counter.  Alternatively,
some peptides can be resolved using high percentage polyacrylamide gels,
dried down and autoradiographed and possibly stained with ninhydrin.
Or they can be applied to a corner of thin-layer silica gel plate and
developed by chromatograhy (descending) in the first dimension, then
high voltage electrophoresis in the second dimension.  After drying them
down, they can be stained with ninhydrin or other special stains.

<h3>Multidimensional Peptide Mapping Methods</h3>
<p>
In the 1950s and 1960s, several developments were made in the use of 
chromatography and electrophoresis on (filter) paper, thin-layer silica, and 
gels of starch, agar, and polyacrylamide, and thin-layer films of silica 
particles and cellulose acetate.  These methods were called 
<q>fingerprinting</q> or <q>peptide mapping</q> because the patterns of spots or 
bands could be used to compare or identify proteins.
<p>
These methods are somewhat obsolete because the proteins had to be purified in 
large amounts at the start in order to detect peptides in these separation and 
analytical systems.  Nonetheless they still may prove useful in limited
circumstances.

<h4>Hanada <q>Fingerprinting</q></h4>
<p>

Hanada &amp; Rucknagel (1964) <i>Blood</i> <b>24</b>: 624-635 described a method 
for separating the peptides of heat-denatured hemoglobin treated with trypsin. 
It is two-dimensional method in which 1-2 mg peptide sample, applied as a 
concentrate of 50 mg/ml in water, is spotted on one corner of Whatman No. 3 or 
3MM filter paper (46 &times; 57 cm sheet), with drying by non-heated air.  Spot 
diameter should be limited to &lt; 1 cm, if possible, but if the spot diameter 
is too tight with up to 2 mg material, insoluble forms can occur and lead to 
streaking in chromatography; in the case of large peptides which show no 
mobility if allowed to dry, apply the spot just as the solvent front reaches the 
spotting origin.  Spot origin should be about 5 cm below glass rod over which 
paper is hung. Descending chromatography using 4:1:5 n-butanol:acetic 
acid:water, 90:18:60:72 n-butanol:acetic acid:pyridine:water, or 35:35:30 
isopropanol:acetic acid:water.  Preparation of solvents should always be fresh
(mixed only within last 24 hours).  For two-phase solvents, the aqueous (bottom)
phase is usually discarded (or placed at bottom of a Chromatocab.  Development
is usually overnight (16 h).
<p>
Electrophoresis in the second dimension requires high voltage power supplies
(2000 V or more) and cooling systems, since the amperage is from 100-150 mA and 
can produce 200-300 W heat (power).  A coolant of Varsol (a mixture of hexane, 
heptane, and aromatic 8-carbon compounds) is usually satisfactory, nontoxic and 
not easily flammable, with stainless steel (or copper) cooling coils to 
circulate cold water.  The paper must be moistened prior to electrophoresis,
and it must be done in a way not to cause convective flow of the chromatographed
peptides from the origin line.  Electrophoresis at 2000 V with currents between
75-150 mA will allow complete migration within 60-90 min.  In the Harada study,
it was done at 2500 V (50 V/cm) at 125 mA.  Electrophoresis buffers of
pyridine:acetic acid:water are used:  1:10:289 for pH 3.45 and 10:0.4:100 for
pH 6.5.  Staining with 0.2% ninhydrin in water-satd butanol will develop all.


<!-- =====================================================================
                   Interaction Proteomics Tools 
========================================================================== -->
<h1>Interaction Proteomics Tools</h1>

<h2>Immunoblotting (Western Blotting)</h2>
<p>
The transfer of proteins from polyacrylamide gels to nitrocellulose was
first described by Towbin, Staehelin, and Gordon.  Protein also binds to
PVDF and many other materials.  The protein is generally immunoreactive
(binds to antibodies specific to it) and can be analyzed by other interactions
as well.
<h3>Gel Electrophoresis</h3>
<p>
Both native and denaturing polyacrylamide gel electrophoresis (SDS-PAGE)
can be done for immunoblotting purposes.  Not all antibodies are able to
recognize unfolded proteins.  When buying commercial antibodies, make
sure the manufacturer claims they work for immunoblotting (westerns)
from denaturing gels.

<h3>Electrotransfer Blotting</h3>
<p>
Selection of the membrane will be a consideration depending on the
application.  If it is vital to visualize the proteins with a reversible
stain prior to immunoblotting, use of the Instaview Nitrocellulose
membrane (BDH Laboratory Supplies, Poole UK) might be considered;
it requires a special staining and enhancer (background remover) and
destain as part of a kit, and gives better sensitivity than use of
Ponceau S.

<h3>Antibody Detection</h3>

<h3>Staining &amp; Destaining Blots</h3>
<p>
The proteins bound to blotting media can usually be reversibly stained,
i.e. stained and destained.  The reversibility of
staining means that subsequent procedures on the blot should not
be affected, such as immunodetection.
<p>
The type of staining procedure to be used for bound proteins depends
upon the blotting medium used.
<p>
When destaining gels or membranes, it is usually more effective to
use several small volumes, each replacing the previous, of
fresh destain solution rather than one large volume.
For example, if there is 100 ml of destaining solution,
and one only needs 20 ml to immerse a membrane very well, one might possibly
achieve complete destaining with 3 &times; 20 ml (total 60 ml) than if
one used 100 ml all at once, which might not thorougly destain the membrane
at all.  In addition, Coomassie dyes adsorb to polyurethane foam and
Kimwipes (laboratory-grade tissues), and these might be included in the
destain.
<p>
When staining or destaining, motion is useful.  Use a rocker or an orbital
shaker set at the correct period (frequency).
<p>
Note that in handling the membrane, especially PVDF, it is best never to
let the membrane dry out at any time.  Always keep it wetted unless for
some reason you have to dry it out.

<h4>Proteins Blotted To Nitrocellulose</h4>
<p>
After electrotransfer or pipet spotting of proteins to nitrocellulose
membrane, make sure that the membrane is washed a couple of times in
highest quality water to remove salts and interfering substances.
The washing process can include a recommended use of PBS or TBS,
done prior to the water washing.

<p>
<b>Staining</b>.  Treat the membrane in 0.1% Ponceau S in 1% acetic
acid.  Rock or shake orbitally for at least 10 minutes, and perhaps longer.
Rinse the membrane with the highest quality water to remove unbound dye
use several changes of water if necessary, and rock or shake.
If the background is too high, try rinsing in the destaining solution
(described below) very briefly to remove the background without causing
destaining of the spots/bands.  Immediately place in water to prevent
further destaining.  Image the wet membrane if that is intended (keep moist
or wet as much as possible without affecting imaging).

<p>
<b>Destaining</b>. Use 1% acetic acid to destain, rocking
or shaking orbitally for a short period of time.  (Others have
also used PBS and/or TBS to destain Ponceau S-stained blots, and that
is usually effective;  high salt/ionic strength is effective.)
Read the destaining recommendations above as well.

<h4>Proteins Blotted To Polyvinylidene fluoride (PVDF)</h4>
<p>
Follow the washing steps used for nitrocellulose prior to any staining.
<p>
<b>Staining</b>.  Treat the membrane in 0.1% Coomassie Brilliant Blue
R-250 in 40% MeOH for at least at least 5 minutes,
rocking or shaking orbitally.  Rinse the membrane well with highest quality
water to remove unbound dye;  use several changes of water, and rock
or shake.  If the background is too high, try rinsing in 10% MeOH, and
if it does not come off faster, increase to 25-30% MeOH.  Do not let it
sit in MeOH or the bands might become unstained;  rinse in water to prevent
band de-staining.  Image the wet membrane if that is intended (keep moist
or wet as much as possible without affecting imaging).

<p>
<b>Destaining</b>. Soak the stained membrane in 50% methanol, rocking
or shaking orbitally.  Use a minimal volume, perhaps with a tissue or
sponge to soak up dye.  Read the destaining recommendations above as well.

<h4>Proteolytic Digestion of Blotted Proteins</h4>
<p>
It is not unusual in proteomics to have need to digest proteins
blotted to PVDF or nitrocellulose.  There are a couple of published
papers on the subject.  The most effective method tested for
nitrocellulose-bound proteins is described below.
<p>
Cut/excise the membrane with protein to a minimal surface.  This is
essential, since trypsin is a protein and will be bound itself to the
membrane.  There is no effective means to block the binding surface of the
membrane prior to trypsin addition (yes, it&rsquo;s been tried).
<ol>
<li>For each 1 square millimeter (1 mm<sup>2</sup>) of membrane, prepare
1 &micro;l of a solution containing 5 nanograms (ng) of trypsin
and 1% Zwittergent 3-16 in 100 mM ammonium bicarbonate.
<li>Add the solution to the membrane in the smallest container (tube, well
of a PCR plate compatible for low protein-binding)
<li>Incubate for 2 hours at 37&deg;.
<li>Sonicate the solution for 5 minutes.
<li>Transfer the supernatant to new container.
<li>To extract more, wash the membrane with digestion buffer alone
(1% Zwittergent 3-16 + 100 mM ammonium bicarbonate), using no more than
1-2 &micro;l per mm<sup>2</sup> membrane.  Sonicate, then transfer
supernatant.
</ol>
<p>
It is possible that MALDI spectral acquisition will not be possible since
detergents significantly interfere with sample ionization.




<!-- =================================================================
                 CHROMATOGRAPHY  
====================================================================== -->
<h1 id="chromatography">Chromatography</h1>
<p>
See the <a href="allbiochem.html">document</a> about all sorts of
chromatography dealing with HPLC, protein/enzyme purifcation, etc.

<h2>Dionex/LC Packings Ultimate 3000 System</h2>
<p>
The Ultimate 3000 is a multi-module system for doing micro-, capillary-
and nano-scale liquid chromatography for protein and peptide separations.
The package of interest to us is the two-dimensional liquid chromatography
for separation of tryptic peptides with a MALDI spotting option using the
Probot fraction collector;  required but not supplied with this is a
MALDI MS/MS system.  Nonetheless there are many other useful things that
can done right now:

<ul>
<li>Offline 2DLC with collection to multiwell plates or standard
autosampler vials, which can then be sampled directly into ESI-MS/MS
<li>Direct MALDI target plate spotting from multiwell plates containing
extracted peptides from in-gel digestion of proteins
<li>Other direct MALDI target plate spotting of pure proteins or peptides,
possibly used for calibration or lockmass
</ul>

<h3>Connections, Fittings, Tubing, Plumbing</h3>
<p>
There are many tips and facts when making connections with fittings
and tubing.  Tubing connections between each part of the Ultimate system
actually are specialized and have a part number.  Order the correct
part number to connect one component to another.
<p>
To fit capillary tubing or columns to the Valco valves on the Flow Manager:
<ol>
<li>Insert the sleeve into the fitting (screw) and tighten the fitting
slightly on the sleeve <em>without</em> the capillary inside the sleeve.
<li>Now pass the fused capillary column or tubing into the sleeve in a
manner that cleans out the sleeve.
<li>Now tighten the capillary to the sleeve.  Just barely tighten at first.
If it leaks during flow, then tighten slightly more.  It is better to
undertighten than overtighten.  Overtightening might crush the capillary.
</ol>
<p>
LCPackings/Dionex makes flow splitters in three scales:
<ul>
<li><b>nanoflow</b>:  flow split is 1 to 1000.  This means that if you set
the Micropump to flow at 0.3 &micro;l/min, the pump will actually pump
at 0.3 &times; 1000 = 300 &micro;l/min in order to achieve the flow in
the splitter.  If you have 1 liter of solvent, at 300 &micro;l/min, the
flow will last a total of 10<sup>6</sup> &micro;l / 300 &micro;l/min =
3333 min = 55 hours.
<li><b>capillary</b>:  flow split is 1 to 100.  Use this for columns
have between 100-500 &micro;m diameter, with 75 &micro;m diameter tubing
connections.
<li><b>micro</b>:  flow split is 1 to 4.  This is used for 0.5-2.0 mm
diameter columns.
</ul>
<p>
The Load Pump in the Ultimate 3000 pumps directly to the WPS well plate
autosampler.  Its flow usually loads the sample on to a trap column that
takes a high flow rate (such as a 300 &micro;m diameter trap column
which has binding stationary phase).
<p>
The MicroPump in the Ultimate system takes care of isocratic or gradient
elution from the trap column into a resolving column of low diameter.
The flow rate is 10-20 less than the loading rate.
<p>
The solvent delivery system has a vacuum degassing system built-in to
the low pressure side of the pump.  Also, any gradient mixing is done
on the low pressure side before it enters the Micro Pump.

<p>
The Ultimate 3000 in a MALDI spotting-only  configuration uses special parts/components
in setting up the chromatography.  Take very special care in setting up
this configuration.
<ol>
<li>Disconnect the Y-port of pump A and connect its tubing to the
port 3 of the left (L) valve.
<li>Connect 20 &micro;m &times; 50 cm PEEKSil tubing to port 4 of the L
valve and this to port 3 of the autosampler valve.
<li>Replace the 20 &micro;l loop with the 1 &micro;l loop on the autosampler.
<li>Get the 25 &micro;m capillary and connect it on autosampler valve port
4 to the Probot input tubing.
<li>From the Chromeleon interface start the flow control valve in pressure
mode to 130 bar, and build segments.
</ol>

<h3>Chromeleon User Interface</h3>
<p>
This software controls all the Ultimate hardware.
<p>
Chromeleon works as client-server software, as on a network.  It is
designed to work on a network of hosts, in fact.

<h4>Browser (Client) Interface</h4>
<p>
The browser window is divided into several panes.
<p>
The pane on the left is a tree of nodes.  These nodes represent objects.
The green-colored animal node refer to <b>Datasources</b> which contain
data and results.  Thesec contain yellow folders which themselves
contain blue-colored folders with column icons that represent <b>Sequences</b>.
<p>
When installed a single directory is created and the directory name
is the name of the <b>Timebase</b>.  This timebase directory contains
a sequence of manually acquired data (called <b>manual</b>).
Directories have subdirectories, while sequences show a list of files
and samples corresponding to the files.
<p>
On the right side of the browser window are an upper and lower pane.
The upper pane displays files associated with the sequence.
These files are:

<ul>
<li><b>PGM:</b> Control or program files
<li><b>QNT:</b> The method of quantification
<li><b>RDF:</b> Report template or definition
<li><b>PAN:</b> Control panel
</ul>
<p>
In the lower pane is a table of the individual samples of the sequence.
Several icons are used to describe the samples:  unknown (cyan vial with
question mark), blank (gray vial with horizontal line), validation
(green vial with checkmark), standard (yellow vial with exclamation
point), matrix (diagonally hatched black-and-green vial), spiked
(pink vial with plus/cross symbol), unspiked (cyan and pink sample
with '1' on it).
<p>
By double-clicking on PGM, QNT, PAN and RDF files, one opens an editor
that corresponds to these files.  Note that subdirectories, sequences
or files can be moved or copied using mouse clicks and drags.

<h4>Control Panel</h4>
<p>
The control panel is an interface to the control of the hardware.
A control panel can be configured to show any part or whole of the
hardware at all.  The parts are then put into communication with
the hardware through the <b>Control</b> menu by using
<b>Connect to Timebase</b>.  One then selects the host or computer which
is the Chromeleon server and which runs the timebase.  Click
the <b>My Computer</b> icon to find servers running on the local host.
<p>
The <b>Dionex Templates</b> directory is a hierarchy of control panels
for different configurations.  These are not editable since the Dionex
Templates directory is locked (read-only).  One can build a control
panel using controls (button, textboxes, sliders, labels) and then
connect controls to the known components or modules.

<h4>Control File (PGM File)</h4>
<p>
Do not confuse the control panel with the control (program) file.
The first is a way to directly control the hardware, while the latter
is used to describe a chromatographic method and all the operations
during a chromatographic run.

<h4>Sample Analysis</h4>
<p>
Generally you want to create a <b>sequence</b> to analyze one or more samples.
You can use an existing sequence (do File-&gt;Save As..., then start
editing it), or create a new one with a wizard.  In creating with the
wizard, you can decide to use existing component files (PGM for method,
QNT for quantification, RDF).  Each of these components has its own
wizard in the creation of the file.  The component files are then
added to the sequence table.


<h3>Probot Fraction Collector</h3>
<p>
The Probot (P/N 161403) is used to collect eluted peptides or proteins.
It can be configured to spot MALDI target plates with an inflow of matrix
solution.
<p>
Flow to the Probot is normally up to a maximum of 30 &micro;l/min using
a 30 &micro;m i.d. fused silica capillary (280 &micro;m o.d.).
Fitted with a 100 &micro;m i.d., flow can be increased to 300 &micro;l/min
maximum.
<p>
The spotting is done with a capillary that emerges 2-3 mm from a stainless
steel holder.

<h3>MicroCarrier (&micro;Carrier) User Interface Software</h3>
<p>
The Probot is controlled by the &micro;Carrier software.  There are two
basic components of the interface.  The <b>Basic Fractionation</b> involves
providing only the required information while making a large use of default
values.  <b>Advanced Fractionation</b> allows more customized used.
<p>
Three objects are programmed by the application designer software:
<ol>
<li><b>Sample Carrier</b><br>
This is the container used in collection, the target plate of the MALDI,
or a multiwell plate.
<li><b>Tray</b><br>
This is the carrier of the container(s).  For the MALDI target plates,
six target plates can be put on the table or tray.  Every spot on the tray
must be defined, and the sample carriers aligned in the X, Y, and Z positions,
relative to the needle of the Probot.
<li><b>Application</b><br>
These are the rules/tasks/commands for spotting (positioning of the
liquid collection needle).
</ol>
<p>
Many carriers, trays, and applications are already predefined with the
system.  These can be loaded into the interface.  Generally the creation
of carriers and trays through the wizards is straightforward.  When creating
a new carrier, save any unsaved work before proceeding.  You will be
asked for the spot target geometry (circular, rectangular) and dimensions
(diameter, x/y lengths), and the height above the plate or regular surface
(generally zero with MALDI targets).  You are then asked about the
number of targets (rows and columns) and the interspot/well distance.
<p>
For trays, one creates positions for multiple carriers.  The positions
are and alignments are done.
<p>
Note that alignments of the system must be re-done whenever any hardware
component is replaced as a result of maintenance or repair.

<h4>Applications and Editor</h4>
<p>
Sample carrier alignment is done by a wizard, with selection of one
or more carriers in the tray.  Alignment is then done using the manual
controls on the software or a manual control unit attached to the Probot.
Step movements should be done in 0.1 mm steps.  Select either the software
or control unit in which to move manually.  The Z-axis adjustment should
be such that the tip just touches the surface of the target;  this
Z-axis setting will be the <b>spotting height</b>.
<p>
A run or cycle is defined by a collection of samples in spots/wells as
<b>groups</b>.
<p>
An application is defined by a set of different <b>tasks</b> or activities
that the Probot is capable of.  Tasks are grouped into certain classes
(Fraction collection:  basic, advanced, planar; Liquid handling:  aliquot,
duplicate, pipet; Multipurpose:  aspirate, dispense, moveTo, switch,
wait, wash; Sequential/Batch mode: sequence, sequenceEnd).  Task lists
are created, and consist of a number, a name for the task (type of
fraction collection), and a description of the task/activity.
Under the name (actually type) of task, parameters are set according to
the type of fractionation.

<h5>Basic Fractionation Task</h5>
<p>
Three sections of parameters are specified.
<h6>How</h6>
<ol>
<li>the flow rate of the chromatography in &micro;l/min
(for nanoflow, typically 0.30)
<li>is matrix added to the flow?
<li>if matrix is added, what is the fold multiplier (1, 2, 3, 4, etc.)
with respect to the flow rate of the chromatography system?
<li>should 10 &micro;l of matrix be dispensed prior to fractionation?
This should be specified if the needle rests in the wash vial between
runs, or is washed as a result.
</ol>
<h6>When</h6>
<ol>
<li>collection time for the group in minutes:  this is the maximum spotting
time for the corresponding task; aborts task if elapsed time after start
is reached, even if there are available wells and more spotting to occur.
If all spots have been processed, the run also stops, even before the
collection time is reached.
<li>time spent for each spot in seconds: the actual spotting time, which is
the sum of the time on the spot and the time spent in movement.
</ol>
<h6>Where</h6>
<ol>
<li>What group of positions is collection to take place?  The groups of
wells to be spotted.
<li>What group of positions represents the standby group?  The place or
location of the capillary tip when it is not spotting, such as the waste
vial.
<li>What is the Z-coordinate for in-task movement in millimeters?
The height above the carrier/target plate considered to be safe travel
between spots and the collect/standby groups.  Note that the in-task
movement Z coordinate must be lower than the spotting height, or the
target plate will be scratched or the capillary damaged.
</ol>

<h5>Fraction Control Mode</h5>
<p>
Specifies whether the collection is controlled by time, by the user,
or by a control signal.
<ul>
<li><b>Time</b><br>
A table with times is set up to control the tasks or activities of the
application.  Intervals between actions can be as short as 4 seconds.
<li><b>Control Signal</b><br>
A contact closure is actuated for collection based on peak starts and
stops.  Signals from the Micropump or UVD can be channeled through the
Chromeleon software.
<li><b>Manual</b><br>
This is when the user wants to control the start and stop of collection,
requiring someone monitor the run.
</ul>

<h4>Checklist for Run</h4>
<ol>
<li>Check for air bubbles in fluid lines, and degas any wash solvent
<li>Make sure there are no leaks in the lines
<li>Check that no salts are deposited on the fluidics joints or capillary
end
<li>Make sure that the system lines are flushed with solvents which do
not contain buffers or salts to prevent deposit formation
</ol>
<p>
Every 3 months check for cracks, nicks, cuts, or clogging of tubing.
<p>
Every 6 months check the rotary valve and syringe.
<p>
Annually replace syringe/syringe tip and inspect, clean, and lubricate
the spindles.

<h4>Purging The Dosage Module</h4>
<p>
The dosage module delivering matrix should be free of air in the lines.
A 4-port rotary valve directs flow to different areas.  The <b>FILL</b>
line goes to a vial holding the matrix.  The <b>PURGE</b> line goes to a
waste vial.  The <b>AUX</b> port goes to a vial for auxiliary solvent,
typically a wash solvent.  The <b>PUMP</b> port is the line to the
micro-tee.  On the top of the valve is a port to connect the syringe.
<p>
For purging/priming the module:
<ol>
<li>Switch the valve into <b>PUMP</b> position.
<li>Remove the syringe.
<li>Push the syringe piston all the way in, to remove air or solvent
in the barrel.
<li>Switch the valve to <b>FILL</b> position.
<li>Draw matrix/FILL out of the FILL vial by pulling out the syringe piston.
<li>Switch again to <b>PUMP</b>, and push the syringe piston back in to
purge reagent all the way to the tip of the spotting needle.
<li>Again switch to <b>Fill</b> and draw in reagent to the syringe.
<li>Continue pulling back the piston of the syringe till it come out, and let
the reagent drip from the syringe by gravity.  When this happens, put
in the plunger again.
<li>Reinstall the syringe into the pump drive.
<li>Switch to <b>PURGE</b> and do a purge.
<li>Re-fill the syringe by setting to FILL.
</ol>

<!--  MASS SPECTROMETRY -->

<h1 id="massspec">Mass Spectrometry</h1>

<h2 id="usefulpeptides">Peptides Useful In Mass Spectrometry</h2>
<p>
The table below lists peptides and proteins that are useful in mass
spectrometry as lockmass or calibrant peptides.

<table>
<caption>Sigma Calibration Peptide Set</caption>
<col><col span="2" style="text-align:right;">
<tr><th rowspan="2">Peptide/Protein    <th colspan="2">[M + H]<sup>+</sup>
<tr>                                     <th>monoisotopic  <th>average
<tr><td>Bradykinin Fragment 1-7          <td>  757.3997    <td>
<tr><td>Angiotensin II (human)           <td>1 046.5423    <td>
<tr><td>P14R (synthetic peptide)         <td>1 533.8582    <td>
<tr><td>ACTH Fragment 18-39 (human)      <td>2 465.1989    <td> 2 466.72
<tr><td>Insulin oxidized B chain (bovine)<td>3 494.6513    <td>
<tr><td>Insulin (bovine)                 <td>5 730.6087    <td> 5 734.51
<tr><td>Cytochrome C (equine)            <td>              <td>12 361.96
<tr><td>Apomyoglobin (equine)            <td>              <td>16 952.27
<tr><td>Aldolase (rabbit muscle)         <td>              <td>39 212.28
<tr><td>Albumin (bovine serum) - BSA     <td>              <td>66 430.09
</table>
</div>

<div class="widetable">
<table>
<caption>Peptides Used By Micromass</caption>
<tr><th rowspan="2">Peptide/Protein
  <th rowspan="2">Sequence
  <th colspan="2">[M + H]<sup>+</sup>
  <th rowspan="2">CAS #
  <th rowspan="2">Vendor/Cat #
<tr>                                     <th>monoisotopic  <th>average
<tr><td>CHCA matrix
 <td>                                    <td> 190.0504     <td> 190.18
 <td>            <td>
<tr><td>CHCA matrix (dimer)
 <td>                                    <td> 379.0930     <td> 379.35
 <td>            <td>
<tr><td>des-Arg<sup>1</sup>-Bradykinin
 <td>                                    <td> 904.4681     <td> 905.05
 <td>            <td>
<tr><td>Angiotensin I human
 <td>DRVYIHPFHL                          <td>1296.6853     <td>1297.51
 <td>70937-97-2  <td>Sigma A9650
<tr><td>Substance P acetate hydrate
 <td>RPKPQQFFGLM-nh<sub>2</sub>          <td>1347.7360     <td>1348.66
 <td>33507-63-0  <td>Sigma S6883 (&ge;95% HPLC)
<tr><td>Glu<sup>1</sup>-Fibrinopeptide B
 <td>EGVNDNEEGFFSAR                     <td>1570.6774     <td>1571.61
 <td>103213-49-6 <td>Sigma F3261 (&ge;90% HPLC)
<tr><td>Renin Substrate Tetradecapeptide (porcine) [Angiotensinogen 1-14]
 <td>DRVYIHPFHLLVYS                     <td>1758.9326     <td>1760.05
 <td>20845-02-7  <td>Sigma R8129 (&ge;97% HPLC)
<tr><td>ACTH Fragment 18-39 (human) (CLIP)
 <td>RPVKVYPNGAEDESAEAFPLEF             <td>2465.1989     <td>2466.72
 <td>53917-42-3  <td>
<tr><td colspan="6">Substance P Met very susceptible to oxidation
</table>
</div>

<div class="contentblock">
<table>
<caption>Trypsin Internal Calibration Masses</caption>
<tr><th>Porcine    <th>Bovine
<tr><th>m/z        <th>m/z
<tr><td>842.5100	 <td>805.4167
<tr><td>1045.5642  <td>906.5050
<tr><td>2211.1045  <td>1153.5741
<tr><td>2239.1359  <td>2163.0570
<tr><td>2283.1807  <td>2273.1600
<tr><td>2299.1756  <td>2289.1649
<tr><td>2807.3000  <td>2530.3339
</table>

<p>
Note that Sigma-Aldrich sells ProteoMass&trade; kit (<b>MS-CAL1</b>)
that calibrates MALDI-MS between 700-66,000 Da, covering a broad range.
Another kit (<b>MS-CAL2</b>) covers the lower mass range (700-3500 Da)
better for tryptic peptides and PSD.  The high mass range (5-66 kDa)
is done with another kit (<b>MS-CAL3</b>).

<p>
Sigma also sells an HPLC peptide standard mixture (H2016)
composed of about 0.5 mg each of the following:
<ul>
<li>Gly-Tyr:  mono=239.1026  avg=239.2506
<li>Val-Tyr-Val:  mono=380.2180  avg=
<li>Leu-enkephalin (human):  mono=556.277  avg=556.6 (Sigma L9133)
<li>Met-enkephalin (human):  mono=574.234  avg=574.7 (Sigma M6638)
<li>Angiotensin II (human):  mono=1046.542 avg=1047.2 (Sigma A9525)
</ul>
<p>
The mono and average are the [M + H]<sup>+</sup> form.

<h2 id="maldi">MALDI-MS</h2>

<h3 id="containers">Containers</h3>

<h4>Glass versus Plastic</h4>
<p>
There seems to be a wide amount of disagreement about what containers
to use for preparing either MALDI- or ESI-MS samples.  Organic
solvents (acetonitrile, methanol, ethanol) are used heavily in both
preparing all kinds of peptide/protein samples for whatever MS is to be
used.  These solvents tend to extract materials that are present in the
structure (walls) of the containers that hold them.  For example,
acetonitrile will extract plasticizers (phthalates) from plastic
containers in which it sits for a long time, and these show up in
the spectra as interfering peaks.  Thus, it is always recommended that
it be stored long term in glass, and used only in plastic containers
for the day.  And not all plastic containers are alike:  some of the
low-budget plastic containers are said to have rapidly extractable
components and cannot be used at all in mass spec work.
<p>
But having now said that, there is this from Sigma-Aldrich, in an
Application Note titled &ldquo;ProteoMass&trade; Peptide and Protein
MALDI-MS Calibration Kit&rdquo; written by L. Barbacci and J. Boland:
<blockquote>
<p>
Solvents included in the kit, acetonitrile (ACN) and trifluoroacetic
acid (TFA; 0.1 and 1% (v/v) solutions), are tested
to ensure low alkali metal content and high purity. The
solvents are packaged in high-density polyethylene (HDPE)
which eliminates the leaching of sodium and potassium
into the solvent, a common problem with glass bottles
leading to lower resolution and cation adducts.
</blockquote>
<p>
Thus there is a seemingly contradictory statement that plastics
are better containers than glass for storing solvents.
Can it be suggested that perhaps the ideal container must be made of a
low- or no-sodium/potassium glass, or of plastic without organic
extractables?


<h3 id="mschems">Chemical Reagents</h3>

<h4>Ion-Pairing Reagents:  Trifluoroacetic Acid (TFA) and Formic Acid</h4>
<p>
An ion-pairing agent is used in reversed-phase chromatography because
it makes enhances resolution (makes peaks sharper).  TFA is usually used
in all reversed-phase HPLC because it ionizes strongly even as the
solvent polarity (dielectric) falls;  normally, a protonic (acidic)
molecule is less likely to ionize as solvent polarity decreases (such
as with an increasing methanol or acetonitrile proportion in a gradient
flow, solvents with lower dielectrics than water).
<p>
But TFA causes problems with ESI-MS, largely because it overwhelms the
the signal in the

<h4>MALDI Matrices</h4>

<h5>Trans-cinnamic acid-Based Matrices</h5>
<p>
<b>&alpha;-Cyano-4-hydroxycinnamic acid</b> (4-HCCA, CHCA, CAS 28166-41-8,
Sigma 70990 puriss p.a. &ge;99.0% Fluka, also 55272 Fluka 25 mg ampules,
and 39468 &ge;99.5% Ultrapure Fluka) is the most commonly used for peptides
and any polypeptides &lt; 10 kDa.
<p>
<b>Sinapinic acid</b>, also called sinapic acid (CAS 530-59-6, Sigma 85429 puriss.
p.a. &ge;99.0%, also Sigma 78867 Fluka) is used for larger proteins.


<h5>6-Aza-2-thiothymine (ATT)</h5>
<p>
ATT (CAS 615-76-9, Sigma 82393  puriss. p.a. &ge;99.9% Fluka)
has been used for acidic glycans analyzed in negative ion mode.

<h5>1-Isoquinolinol (1-HIQ)</h5>
<p>
Also known as 1-hydroxyisoquinoline and isocarbostyril (CAS 491-30-5, Sigma
55433 puriss p.a. &ge;99.0% Fluka), it is has been used with DNB as comatrix
in oligosaccharide analysis.  Useful when sample contains significant buffer
or other salts and SDS. More sensitive than DHB in analyzing high mannose N-linked
glycans.

<h5>Polyhydroxyacetophenone-Based Matrix</h5>
<p>
2&prime;,6&prime;-Dihydroxyacetophenone, also called 2-acetylresorcinol
(CAS 699-83-2, Sigma 37468 puriss. p.a. &ge;99.5% Fluka) has been used
with diammonium hydrogen citrate for analyzing PMP-derivatized acidic and
neutral glycans.
<p>
2&prime;,4&prime;,6&prime;-Trihydroxyacetophenone, also known by \
2-acetylphloroglucinol and THAP (CAS 480-66-0, Sigma 91928
puriss. p.a. &ge;99.5% Fluka) this is useful for MALDI analysis of acidic glycans
and glycopeptides in negative ion mode and has lower detection limit than
AAT.

<h5>3-Aminoquinoline</h5>
<p>
This will react with any reducing terminus on glycans and make them fluorescent.
Moreover, it can form a liquid composite matrix with CHCA and glycerol to analyze
neutral and acidic glycans.


<h3 id="matrixprep">Matrix Preparation</h3>
<p>
[from a Sigma-Aldrich application note]
<p>
Recrystallized matrix is best prepared (and typically purchased in this
way) in 10 mg amounts.  &alpha;-cyano-4-hydroxycinnamic acid (CHCA)
is dissolved in 1 ml of 50% acetonitrile containing 0.05% trifluoroacetic
acid.  Sinapinic acid is dissolved in 70% MeCN and 0.03% TFA.
Matrices can be stored in the dark for a week before they have to be
discarded.  These solutions are nearly saturated at room temperature.
It is best to spin the solution before use to eliminate crystals.
Other mixtures include use of a 70:30 MeCN:0.1% TFA.

<h4 id="CHCArecryst">Washing &amp; Recrystallization of
&alpha;-Cyano-4-Hydroxycinnamic Acid</h4>
<p>
[Source:  Waters/Micromass M@LDI-ToF Operator's Manual]
<p>
 (CHCA) and 3,5-Dimethoxy-4-Hydroxycinnamic Acid (Sinapinic Acid)
Both of these are cinnamic acid derivatives.  Waters recommends
&alpha;-cyano-4-hydroxycinnamic acid (CHCA, Sigma C-2020), washed
and re-crystallized before use.  Just wash if short of time.

<h5>Washing</h5>
<ol>
<li>Place 100 mg CHCA in a 2 ml tube.
<li>Add 1 ml acetonitrile and vortex.
<li>Centrifuge, then remove the supernatant with a pipette.
</ol>

<h5>Re-crystallization</h5>
<ol>
<li>Place the amount of matrix (CHCA or sinapinic acid) in a tube
of the appropriate size (a 50 ml plastic conical tube is quite convenient
and useful).  For every 100 mg of matrix you weigh out and want to prepare,
add in 10 ml of high-grade (HPLC or Type I) water.
<li>To assist in the next step (dissolution in ammonium hydroxide),
swirl the matrix in the water while holding 30-60 seconds in an ultrasonic
water bath.  This helps to break up matrix into smaller, more easily
dissolvable particles.
<li>Get a P-100 or P-200 pipet with yellow tip, and add 30-50 &mu;l at a time
of <b> concentrated (25-26%) ammonium hydroxide (NH<sub>4</sub>OH)</b>.
You can add more based on your experience.  The matrix will dissolve
in solution and will not settle out.  If it helps, sonicate the solution
again to help dissolve it.  The matrix is an organic acid in its stock
form, and organic acids which are derivatives of cinnamic acid (as MALDI
matrix is) are not soluble in water;  by adding an alkaline reagent like
NH<sub>4</sub>OH, a salt is formed:
<p style="text-align:center;">
R-COOH (s) + NH<sub>4</sub>OH &mdash;&mdash;&gt;
 R-COO<sup>&minus;</sup> (aq) + NH<sub>4</sub><sup>+</sup> +
 H<sub>2</sub>O
<p>
All ions tend to be soluble in water (aqueous solutions).  The
&ldquo;(s)&rdquo; means that the substance is a &ldquo;solid&rdquo; or
insoluble in the reaction, while the &ldquo;(aq)&rdquo; means it is
is &ldquo;aqueous&rdquo; or soluble in the reaction.  Not everything
may solubilize by the addition of NH<sub>4</sub>OH, so try not to over-add.
<li>Spin the tube at 500-1000 &times; <i>g</i> for 10-15 min.
<li>Look for a pellet of material that did not solubilize.  Transfer as much
supernatant as possible to a new tube (50 ml plastic).  Save the tube
with the pellet by washing it out, and re-using it for the next time you
want to re-crystallize matrix.
<li>To the tube with supernatant, get a fresh tip for the P-100 or P-200
pipet and add 30 &mu;l (about 1 drop) of <b>concentrated (36-37%,
12 M) hydrochloric acid</b>.  Swirl the tube with each addition.
A mist may form above the top of the liquid (this is a mist of ammonium
chloride, or NH<sub>4</sub>Cl, formed when the ammonia vapor reacts with
the HCl).  Add enough to get the matrix to settle out.  By adding an
acid (like HCl), the matrix is being re-protonated to make the acid form
of it, which is insoluble in aqueous solution:
<p style="text-align:center;">
R-COO<sup>&minus;</sup> (aq) + HCl &mdash;&mdash;&gt; R-COOH (s) + Cl<sup>&minus;</sup>
<p>
The chloride can react with NH<sub>4</sub><sup>+</sup> ion to produce
ammonium chloride, which should be soluble.
<li>Spin the matrix precipitate down again.  Remove the supernatant (either with a pipet,
or if you gave it a good, hard spin, you can often pour it off).  Discard
the superatant.  If you wonder if more matrix is precipitated with acid
addition, then add small drops of concentrated HCl to the supernatant to
see if any matrix precipitates out.  If so, combine the pellet with the other
pellet after washing and re-dissolving (explained in the next steps).
<li>Add the same volume of 0.1 N HCl to the matrix pellet as you did of
water in the first step (for every 100 mg of your original sample weight,
add 10 ml). Vortex/mix the matrix
in the wash.  Spin down again to get a pellet.  This is the first 0.1 N
HCl wash.  Repeat this a 2nd time.  In general you are washing away any
ammonium chloride.
<li>Dissolve the matrix pellet in acetonitrile (ACN, or MeCN).  For every
100 mg of your original sample weight, add 3-5 ml MeCN.  Just add enough
to make it soluble.  Swirl the solution around.  It might be a bit slow
in dissolving.  You can put it in hot water, or try sonicating it to help
the dissolution.
<li>Get 1.5 or 2.0 ml microcentrifuge or plastic reaction tubes (10 tubes
for every 100 mg of matrix you have).  Use a marker to put a number
(1, 2, 3, ...) for each of the tubes that you can wipe away later on (using
solvent and a Kimwipe, for example).  Record the weight of each of the
empty tubes.
<li>Transfer the correct volume of matrix dissolved in MeCN to each of the
tubes.  If you used 3 ml of MeCN per 100 mg, then transfer 0.3 ml to each
tube.
<li>Put the tubes in a rack that will not melt at 60-80&deg; C.
<b>The caps of the tubes should not be closed</b>.  Turn on
an oven that is set at 60-80&deg; C.  Put the tubes in the oven.  The
oven should vent somewhat so as not to accumulate vapors inside.  Check
on the tubes every 30 min or so to see if all the solvent has evaporated
away.  The residue should leave crystals in the tube.  Do not leave the
tubes in the oven longer than it is necessary to evaporate the acetonitrile.
<li>Let the tubes cool.  Record the weight of each tube according to the
number you wrote on it when pre-weighing the empty tubes for a tare.
Take the difference between the weight now and the tared tube.  Record
this weight on the tube with the name of the matrix.  For example, if
working with CHCA and the weight is 10.3 mg, then write &ldquo;10.3 mg
CHCA&rdquo; on the tube, or if it is 9.8 mg of sinapinic acid, write
&ldquo;9.8 mg SinA&rdquo; on the tube.
<li>Store all tubes at room temperature in an appropriate container marked
for matrix stock.
</ol>
<p>
It is customary to work with 10 mg at a time when using MALDI matrix.
I do not know its shelf life after being reconstituted in matrix preparations.

<h4>2,5-Dihydroxybenzoic acid (DHB, gentisic acid)</h4>
<p>
Weigh out 20-50 mg DHB (FW 154.03) into a glass vial.  Add 1.0 ml of
whichever organic solvent is soluble for this matrix and the sample.
Consider in this order:  MeOH, ACN, acetone.  Vortex 60 sec until dissolved.

<h4>Dithranol</h4>
<p>
Prepare 10 mg in glass vial, and add 1.0 ml tetrahydrofuran (THF).
Vortex until fully dissolved.

<h3 id="matrixanalyte">Matrix and Analyte</h3>
<p>
The following technique is the <b>dried droplet</b> technique, which
is the most common and should be the first to be tried with most, if not
all, samples.  Other techniques are mentioned below.
<p>
The peptide or protein concentration is usually at about 10 &mu;M
while the matrix is done at 10 mM (a 1000-fold molar excess of
matrix-to-analyte is best).  No precipitate must form when matrix
and protein/peptide are mixed.  Sample is spotted on the target plate
and allowed to evaporate <em>without heating of the target plate</em>.
Dried sample is stable and can be stored in the dark for several days
(storing under a vacuum is even better).
<p>
&alpha;-Cyano-4-hydroxycinnamic acid (CHCA) can be dissolved 5-10 mg/ml
in acetonitrile:0.1% aqueous TFA mix, usually at 70:30 or 50:50.
Another solvent is 60:36:4 EtOH:MeCN:water, which evaporates
faster and gives more uniform smaller crystals.  2,5-Dihydroxybenzoic acid
(DHB), or gentisic acid, is used for peptides, alone or as part of a
co-matrix.  Super DHB is a 9:1 mix of DHB with 2-hydroxy-5-methoxybenzoic
acid.  DHB with fucose and 5-methoxybenzoic acid has been used.
<p>
DHB and Super DHB are useful especially for glycopeptides.  If carbohydrate
is not high, matrices that work for normal peptides also work for these
glycopeptides.  DHB is usually made in 10% aqueous EtOH at 5-10 mg/ml.
Solvents that work with CHCA also work with DHB.
<p>
The best samples also make use of fresh matrix.  The crystal-growing
mother liquor should be placed on the sample support (and below pH 4)
so that the matrix does not ionize.  A 0.1% TFA will eliminate adverse
effects.  Avoid nonvolatile solvents since they interfere with crystal
growth.  Non-proteinaceous material present in too high amounts can suppress
ion signal.
<p>
In most cases, the simplest method is to prepare a sample in 0.1% TFA
at 5-10 pmol per microliter, mix this in a proportion between 1:4 to
1:9 with matrix.  A 1 &mu;l aliquot applied will proved 0.5-2.0 pmol
of peptide at the target.  A 1:1 solution may have to be used if the
results show too little analyte.

<h4>Special Situations</h4>
<p>
This section is supposed to deal with special situation of the analytes
(peptides or proteins) and matrix.  Sometimes the analytes are in solution
already, and sometimes they are dried.

<p>
<b>Tryptic Peptides in Solution in 96-well PCR Plate</b>.  This situation
is normal for in-gel tryptic peptides just extracted from the gel using
methods in the reference.  The volume will be approximately 50-60 &micro;l:
25 &micro;l for the tryptic digest + 30 &micro;l for the first extraction +
10-15 &micro;l for the second extraction.  If a gel spot represents a 50 kDa
protein with about 200 ng protein and assuming 50% extraction efficiency,
this will produce 2 pmol or 2000 fmol of peptides.  That is usually well above
the sensitivity of many MALDI MS instruments.  Comfortable spectra can be
obtained with 200-500 fmol peptides in many cases.  The problem is that with
the small volumes allowed for MALDI plate spotting (1 &micro;l usually) and
with the need to mix with matrix (at least a 1:1 volume), the ability to detect
peptides will only be with high abundance protein and probably with many gel
pieces.  If an immediate assessment is desired, mix 1 &micro;l of the peptide
solution directly on the plate well, then add 1 &micro;l of saturated matrix
(CHCA for trypic peptides) in 50% MeCN containing 0.2% TFA.  The TFA is higher
because the peptide solution may contain significant ammonium bicarbonate.
<p>
<b>Tryptic Peptides Dried in 96-well PCR Plate</b>.  This situation
will occur if it was necessary to dry down the peptides in a 96-well PCR
plate by centrifugal evaporation or lyophilization.  This allows them to
be concentrated.
<ol>
<li><b>Concentrating peptides at well bottom</b>.  If there is some concern that 
the peptides have not concentrated to the bottom of the plate wells, it might be 
useful to add 15-20 &micro;l of 80% acetonitrile, swirl it around, and then let 
it evaporate, maybe in a 37&deg; oven.  This could take time.
<li>Prepare CHCA matrix at saturated concentrations (usually about 20-30 mg/ml)
in 50% MeCN + 0.1% TFA, using the formula:  # wells &times; 2 &micro;l / well
&times; 1.15 (15% pipetting error).  Call this volume TV (total volume).
 <ol style="list-style-type:lower-alpha;">
 <li>Take a tube with <i>x</i> mg recrystallized CHCA and add 25 &times; 
<i>x</i> &micro;l of MeCN to it.
 <li>Try to dissolve the CHCA as much as possible by flicking or vortexing the
tube, and putting it in the ultrasonicator.  It might form an insoluble powder
in the MeCN
 <li>Add the same volume of 0.2% TFA (in HPLC water) to the tube.  Again mix
well, including flicking, vortexing, and sonicating.  Try to get as much dissolution
as possible.
 <li>If TV &gt; the volume of re-dissolved CHCA (50 &times; <i>x</i>), then get
more tubes of re-crystallized CHCA and re-dissolve them according to the above
steps.  Pool all re-dissolved CHCA, mix well, vortex and repeat the sonication.
 <li>When the final amount of re-dissolved CHCA is prepared, spin the tube 
at maximum microcentrifuge speed for 10 min.  There is no need to transfer the
supernatant, so long as you don't disturb the pellet layer.
 </ol>
<li>Place a <i>pre-cleaned</i> MALDI target plate in the laminar flow hood 
next to your PCR plate containing your peptides.  Both must be at room
temperature.
<li>Add <i><b>to the bottom of the well</b></i> 2-3 &micro;l of the matrix 
solution using a P-2 or P-20 (whichever works better) to a selected PCR well.
Move the tip in a circular motion around the bottom of the well in order to get
the matrix solution wet around the well bottom and its sides.  Work quicky:
matrix may start to crystallize aroundPipet up and down <i><b>very quickly</b></i>
</ol>

<p>
<b>Dried-down solution-digested protein</b>.  This situation is normal for the
solution digest of a control protein, such as BSA.  Generally the peptides of
control proteins can be spotted at three different levels on the MALDI plates:
500, 1000, and 2000 fmol (0.5, 1.0, and 2.0 pmol).  The solution for spotting 
will be 500, 1000, or 2000 fmol in 1 &micro;l of saturated CHCA in 50% MeCN
+ 0.1% TFA.
<ol>
<li>Prepare the peptides of the protein digest at 10 pmol/&micro;l in water.
If the peptides are dried down, then the microliters of water to add =
(nanograms of protein digested) &times; (1 / molecular weight of protein) 
&times; (100).  Thus if 5 &micro;g BSA (MW = 66 kDa) was 
digested, then the volume = (5000 ng) &times; (1/66,000) &times; 100 = 7.5 &micro;l

</ol>

<h4>Multiple/Co-Matrices</h4>
<p>
Laugesen &amp; Roepstorff (<i>J. Am. Soc. Mass Spectrom.</i> <b>14</b>:
992-1002, 2003) report that matrix mixtures may be superior to use of pure
matrices.  For larger proteins, matrix mixtures may especially improve
the spectra of glycoproteins.

<h3 id="sampquality">Sample Quality and Poor Spectra</h3>
<p>
The spotted sample may produce poor spectra if any of the following is
observed:
<ul>
<li>A ring effect around the crystallized matrix
<li>Clumping of matrix in the well
<li>Failure of the matrix to crystallize
<li>A droplet spreading out over a wide area
</ul>
<p>
These can indicate contamination of the matrix or maybe something in the
sample.  Materials which can interfere with poor crystallization include
any buffer salts, urea, sodium azide, guanidine, EDTA, glycerol, DMSO,
NaCl, detergents.
Generally urea and guanidine up to 2 M can be tolerated.
HEPES, MOPS, Tris, ammonium acetate, and octyl &beta;-glucoside can be
used to 50 mM (as a rule, use the minimum concentration necessary to control
pH, and not more).  TFA, formic acid, 2-mercaptoethanol, DTT, acetic
acid, and any other volatiles (organics, HCl, ammonium hydroxide) do not
interfere.
<p>
Among the techniques that can be used for sample clean-up include:
<ul>
<li><b>Dilution</b><br>
Sometimes improves in signal-to-noise can be done by reducing the noise
without loss of sensitivity of the signal
<li><b>Washing</b><br>
Washing the dried sample after spotting has proven an effective way to
eliminate a lot of low MW salts:  1-2 &micro;l cold 0.1% TFA left on the
sample for 5-10 seconds improves the signal a great deal.  If detergents
are a problem, try cold 5% isopropanol.  If there is contaminants from
cell extracts, try cold 100% isopropanol.
<li><b>Drop Dialysis</b><br>
In the sample class as washing.  From a method reported by
H. Goerisch (<i>Anal. Biochem.</i> <b>173</b>: 393-398, 1998):
 <ol>
 <li>Obtain an appropriate membrane, such as a Type VS Millipore
 25 &micro;m pore, 25 mm dia. membrane.
 <li>Fill a beaker with 250-400 ml deionized water.
 <li>Float the membrane on the water with shiny side up.
 <li>Place 10 &micro;l sample on the membrane.
 <li>Add 1 &micro;l MeCN to the sample spot to increase the surface area.
 <li>Leave for 45 min
 <li>Remove an aliquot with pipet, add matrix, then spot plate
 </ol>
<li><b>Cation exchange</b><br>
In TFA-acidified samples, positively charged peptides more likely to
ionize in the MALDI are actually enriched while other peptides are
removed, along with the low MW materials.  The method is:
 <ol>
 <li>Stir Dowex 50Wx8-200 in H<sup>+</sup> form with a 2&times; vol of
1 M ammonium acetate overnight, and wash thoroughly with deonized water,
then acetone, then hexane; let dry and store for use
 <li>Put 0.1 mg beads on clean Parafilm
 <li>Add 50 &micro;l sample and same volume of matrix to make a slurry.
 <li>Mix with pipet 10-15 times up and down
 <li>Let beads settle (about 15-30 s)
 <li>Pipet the supernatant on to the target plate (the tip might be changed
 to avoid carrying beads over to target plate)
 </ol>
This can help improve mass determinations where multiple positive (sodium,
potassium) are present.
<li><b>Pipette-tip column chromatography</b><br>
A fancy name for nano-scale gel filtration, which should be effective
at eliminating low MW substances
<li><b>ZipTips</b><br>
A brand name for pipet tips containing a small amount of C<sub>18</sub>
reversed-phase silica in the tip, which retain pmol amounts of peptides
and nothing else.  The pmol amounts of peptides are more than enough
for MALDI work.  The standard ZipTip has a 0.6 &micro;l bed, and the
micro has a 0.2 &micro;l bed (smaller is better for automation since there
is less resistance to flow).  A C<sub>4</sub> is good for intact proteins,
and a metal-chelating will retain phosphopeptides.  Tips are first wetted
with MeCN (10 &micro;l), then 0.1% TFA in 50% MeCN, then twice with 0.1%
TFA.  Sample is loaded with 5-10 &micro;l with up-and-down pipetting repeated
several times.  The sample is then washed three times with 10 &micro;l
of 0.1% TFA.  Elution is with 30-70% MeCN, or directly into matrix
(CHCA in 50% MeCN + 0.1% TFA), with a minimal 3 &micro;l volume.
</ul>
<p>
ZipTips can also be used concentrate peptides/proteins 10-50 fold.
<p>
When spectra are poor and there is the belief that sample is of good
MALDI quality and the machine in tune, it is possible to improve the spectrum
by washing the dried crystalline sample with 5-10 &mu;l of distilled
deionized water (or 0.1% TFA), if the matrix is not very highly water
soluble.  Repeated washing can improve spectral quality.  Cinnamic
acid derivative matrices respond well with wash cycling;  DHB do not
because it is highly water soluble.
<p>
Try using different matrices with sample as well.  Each matrix interacts
with different proteins and peptides differently.
<p>
Another way to improve results is with a <b>fast-evaporation sample
preparation</b> method, which seems to work well with very large proteins
(&gt; 100 kDa) (see Vorm <i>et al.</i>, <i>Anal. Chem.</i> <b>66</b>: 3281,
1994).
<p>
Try to remove as many contaminants from the analyte as possible.
Poor spectra are obtained when the contaminants listed below exceed the
maximum allowable:
<p style="text-align:center;">
<table>
<caption>MALDI-ToF Sample Contaminants</caption>
<col><col style="width:8em;">
<tr><th>        <th>Maximum allowable Contaminant concentration (approx.)
<tr><td>Urea                 <td>0.5 M
<tr><td>Guanidine-HCl		  <td>0.5 M
<tr><td>Dithiothreitol       <td>0.5 M
<tr><td>Glycerol             <td>1%
<tr><td>Alkali metal salts   <td>&lt; 0.5 M
<tr><td>Tris buffer          <td>0.05 M
<tr><td>NH<sub>4</sub>HCO<sub>3</sub> <td>0.05 M
<tr><td>Phosphate buffer     <td>0.01 M
<tr><td>Detergents (not SDS) <td>0.1%
<tr><td>SDS                  <td>0.01%
</table>

<h4>Slow Crystallization Spotting</h4>
<p>
When the sample contains either involative solvents (glycerol) or
high amounts of salt, the standard dried droplet technique might produce
unsatisfactory results.  Some substances interfere with crystal growth
and the embedding of proteins.  This technique should be considered
if the protein concentration is too low (&lt; 1 &micro;M) or if
involatiles are high (&gt;30% glycerol) or salts are too much (4 M NaCl).
<p>
Prepare <i>saturated</i> matrix in the analyte with solvent at
room temperature.  Usually, place 20-100 &micro;l of analyte in solvent
in the matrix crystals.  Mix well for at least 5 minutes.  Spin down
to down and transfer the supernatant to a 1.5 ml tube.  Heat the liquor
in the <em>capped</em> tube to 40&deg; to dissolve any small crystals
present.  Remove the tube and let it come to room temp (do not open it
before it does).
<p>
The tube is placed on to a holder over a vortex mixer which is set to
rotate continuously at the mixer's lowest speed.  This is left over
the next several hours to until evaporation is achieved.  If the matrix
is sinapinic acid, this results in formation of crystals with a 0.5 to
1 mm long axis that are protein-doped.
<p>
After crystal growth, the liquor is removed.  The crystals are then
washed with room temperature water while swirling the tube with the vortex
mixer.  Spin  the tube to pellet the crystals, and then remove the
supernatant.  Do the washing several times.  The crystals are then transferred
directly to the sample stage as a slurry in water (these are not dissolved
in a solvent mixture).
<p>
Variations on this technique include preparing saturated matrix without
the protein, but making sure that matrix is not diluted.  Also crystallization
can be hastened by placing in a cold chamber or refrigerator.



<h3 id="maldicalib">Calibration of the Instrument</h3>
<p>
The Water/Micromass M@LDI-ToF MS calibration is described
<a href="/Reference/Waters/M@LDI-ToFUse.html#maldicalib">elsewhere</a>.

<h3 id="plateclean">MALDI Target Plate Cleaning</h3>
<p>
There are three methods given.  One is an adaptation of a Micromass
method by Mitch.  The next two are from John, an engineer at Micromass.
<p>
<strong>Do not use any method that immerses or otherwise treats the
target plates with concentrated mineral acids (sulfuric, nitric, hydrochloric,
etc.)!  Never!!</strong>
<p>
<strong>CAUTION</strong>. Never stack target plates on each other or put
anything on the surface of the plates.  The plates can become scratched
easily, and this ruins the wells that become scratched.  Scratched plates
can also give incorrect mass readings.

<h4>Mitch&rsquo;s Method</h4>
<p>
This is a tentative method for cleaning the stainless steel target plates.
This method makes use of the fact that matrix easily dissolves in
ammonium hydroxide (indeed this is a step in matrix recrystallization).
<p>
If desired&mdash;this may be added as a step in the future&mdash;it
may be useful to treat the plate with a detergent (Decon 90, as detailed
in a Micromass method below) after the matrix dissolution/water rinse
step.  Treat with the detergent either by scrubbing with soft brush
or perhaps by sonicating for 10 min (unless it foams), and then
liberally rinse in water before degreasing and drying in organic solvents
(methanol and acetonitrile).

<ol>
<li>Wear gloves at all times when handling target plates that are clean
or about to be cleaned.  They should be powder-free.  Nitrile is best.
<li>Prepare four clean beakers in the 250 ml size.  This is big enough to
take target plates, although they will be at an angle (they will not rest
flat).  Pour into each of them <em>the minimum volume necessary and no more
than the minimum</em> of each of the following reagents/solvents:
 <ul>
 <li>at 15 ml of 25-26% ammonium hydroxide for every 100 ml of volume
necessary (this is approximately 3-4% ammonium hydroxide)
 <li>water that is distilled or of higher quality;  more than the minimum
volume is required and it is probably better to fill the beaker with water
 <li>methanol
 <li>acetonitrile
 </ul>
The methanol and acetonitrile can be re-used a few times, devoted only to
target plate washing, and so previously used MeOH and MeCN are okay.
Put all beakers in a fume hood for the time being.
<li>The target plates will be processed in serial order for the number to
be cleaned.  If too many are to be cleaned in a single washing series,
it may be necessary to replace solutions that become dirty.
<li>Place the first target plate in the 3-4% ammonium hydroxide solution.
It must cover the entire plate.  The top (wells) side of the plate should
be up so that it is not scratched from contacting the beaker glass.
Never put pressure on the plate that might bend the plate:  bent plates
are useless and cannot measure masses accurately across all wells.
<li>Swirl the ammonium hydroxide solution in the plate for a while.  Leave
it for at least 5 min.
<li>Transfer the plate to the water beaker.  Put a 2nd plate into the
ammonium hydroxide beaker to keep the series going.  Take a toothbrush with
soft nylon bristles, and if needed, very gentle brush across the surface
of the plate, especially to remove visible debris.  Leave in water for 5 min.
<li>After 5 min is completed, move the plate in the water beaker to the
MeOH beaker.  Place this beaker in an ultrasonic bath and start the ultrasonics
for 5 min.  At the same time, move the plate in the ammonium hydroxide
beaker to the water beaker, and process that plate.  Add another plate to
be cleaned to the ammonium hydroxide beaker.
<li>After 5 min for all plates to be processed in their beakers, move the
first plate in the MeOH beaker to the MeCN beaker.  Again also move the other
plates along in the chain of beakers.  Put the MeCN beaker also into the
ultrasonic bath and set for 5 min.
<li>Take the first plate out of the MeCN beaker and put on a clean rack
for drying.  This should take only a few minutes and the plate is ready.
Put all clean plates into a storage area where minimal dust or dirt
can contaminate them.
</ol>
<p>
When a clean target plate is to be used, take the additional step necessary
of doing well cleaning before applying sample to the wells.  This
involves spotting high quality water on the well (3-4 &micro;l), leaving it
for 30-60 seconds at least so that it absorb into the drop materials on the
well surface, then sucking back up the water drop, and drying if necessary
with 1-1.5 &micro;l of MeCN.


<h4>Micromass Method #1</h4>
<p>
This is from an email by John at Micromass, who reports this is one printed
in the MALDI micro manual:

<ol>
<li>Scrub plate with soft nylon brush using 2% Decon 90, a common laboratory
detergent.
<li>Rinse with distilled water.
<li>Sonicate in 1:1:1 dichloromethane:acetone:methanol for 10 minutes.
An alternative degreasing agent can also be used.
<li>Sonicate in methanol for 10 minutes.
<li>Dry under a stream of dry nitrogen.
</ol>

<h4>Micromass Method #2</h4>
<p>
John reports using this &ldquo;in the field&rdquo;:
<ol>
<li>Scrub plate with wet tissue or a soft nylon brush (toothbrush) to
remove larger deposits.
<li>Sonicate in 10% formic acid or acetic acid
<li>Sonicate in water.
<li>Sonicate in methanol or acetone to degrease and remove surface water.
</ol>

<h3>MALDI-MS of Noncovalently Associated Protein Subunits (Multimers)</h3>
<p>
There are several papers on the mass determination of intact quaternary
structures of multimeric (multisubunit) proteins.  Generally the matrix
solution should be acid- and organic solvent-free, with the best conditions
being pH neutral elements.  Trans-cinnamic acid matrices are effective
in some cases (includes sinapinic acid and ferulic acid).
Saturating 2,6-dihydroxyacetophenone (2,6-DHAP) in a neutral
20 mM diammonium hydrogen citrate (DAHC) is a favorite of one laboratory
group.

<h2>ESI-MS</h2>

<h3>Peptide Derivatization</h3>
<p>
ESI-MS/MS fragmentation spectra of precursor peptide ions can be improved
by derivatizing peptides prior to MS/MS (see Sonsmann et al. <i>J. Am.
Soc. Mass Spectrom.</i> <b>13</b>: 47, 2002 for notes on theory).

<h4>Acylation with Chlorosulfonylacetyl Chloride (CASC)</h4>
<p>
This modifies primary amines (namely the N-terminus), not only decreasing
basicity, but adding a negative charge (net +2 loss), from the sulfonyl
anion.  For each modification, the added mass is 122 Da.

<ol>
<li>To 1 nmol peptides (dry), add 2 &micro;l 20 mM of the sulfonic acid
(prepare as 2 &micro;l stock in 500 &micro;l water).
<li>Let evaporate, then dissolve in 20 &micro;l THF/ethyl-diisopropylamine
(4:1).
<li>To this add 2 &micro;l of 100 mM CSAC dissolved in dry THF.
<li>Let react 1-2 min.
<li>Vortex for 30 seconds.
<li>Vacuum dry, then dissolve in ESI solvent (50% MeCN + 0.1% formic acid).
</ol>
<p>
Note that this has also been done with protein in gel, with a limit of
300 fmol protein in the gel.

<h4>Acetylation</h4>
<p>
This has the effect of decreasing basicity (positive charge) by modifiying
primary amines (namely the N-terminus).  For each acetylation, the added
mass is 42 Da.

<ol>
<li>Place 0.5 to 2.0 nmol peptide(s) in 45 &micro;l acetic acid.
<li>Add 5 &micro;l of acetic anhydride (final 10%).
<li>Leave 2 h at room temp, then vacuum dry.
<li>Resuspend in 1 M NH<sub>4</sub>OH for 20 min.  This removes O-acetylated
and leaves N-acetylated modifications.
<li>Evaporate and store at &minus;20&deg; until ready to analyze.  For
ESI-MS/MS, setup in 50% MeCN with 0.1% formic acid.
</ol>

<h4>Aklyation with Bromoethyltriphenylphosphine</h4>
<p>
This derivatizes the N-terminus and leaves a charge on the phosphate
atom of the phosphine.  For each modification, there is an added 289 Da.

<ol>
<li>About 0.5-2 nmol of peptide is placed in 50 &micro;l Tris buffer, pH 9.
<li>10 nmol Br-TPP per &micro;l MeCN is added in a 20-fold molar excess
to peptide.  So if there is 1 nmol peptide in 50 &micro;l buffer, adding
2 &micro;l of the Br-TPP solution would give a 20-fold molar excess.
<li>After vortexing, the reaction is put at 37&deg; for 3 h.
<li>To remove small molecular products, pass the reaction over a properly
scaled BondElut CS or SepPak cartridge, and step elute with increasing MeCN.
<li>Vacuum dry the elution fractions, and analyze.
</ol>


<h3>Background Contamination</h3>
<p>
Many peaks in the useful low <i>m/z</i> range of 100-500  can appear
in the mass spectrum when no sample is actually being analyzed.
They can arise from:
<ul>
<li>solvent itself:  one or more of its components and additives
<li>any reagents mixed in with the sample prior to MS work although not
used in MS work
<li>failure to use clean glassware
<li>leachable components in plastic containers or tubing
<li>presence of detergents (probably not rinsed out from glassware cleaning)
<li>the HPLC system
</ul>
<p>
Many contaminants will accumulate on HPLC columns, particularly reversed
phase C<sub>18</sub>, and then elute from those columns within a gradient.
The table below shows possible contaminants and where they come from.
<table>
<caption>Background Contaminants in Spectra (Typically ESI-MS)</caption>
<tr><th>Contaminant
 <th>How It Shows
 <th>Its Source(s)
<tr><td>Polyethylene Glycol (PEG) and PEG-like Materials
 <td>a polymer whose peaks appear as a series of different monomer
 (mass = 44) lengths (&Delta; <i>m/z</i> = 44).  Ion forms are:
 [M+H]<sup>+</sup>, [M+Na]<sup>+</sup>, [M+K]<sup>+</sup>,
 [M+NH<sub>4</sub>]<sup>+</sup>
 <td><ul><li>(organic) solvents:  MeOH, isopropanol, MeCN, water
  <li>used as a MS calibrant
  <li>used in hand creams
  <li>contaminants of Triton X-100, X-114
  <li>component of glassware detergents
  <li>in cutting solutions used for machining
  <li>used in manufacture of columns
  </ul>
<tr><td>Iron (Fe<sup>3+</sup>)
 <td>ESI spectra showing 539.9 and 555.8 are indicative;
    is often an adduct of analyte ions, but also complexes with acetate
    when used as ion-pair agent;
 <td><ul><li>can be present in water or acetonitrile solvents
  <li>contaminant of acetic acid (less so in formic acid)
  <li>non-passivated stainless steel contact
  </ul>
<tr><td>Diisooctylphthalate
  <td>This is a plasticizer for plastic container.  Look for these in
     the spectra:
   <ul>
   <li>[M+H]<sup>+</sup>:  <i>m/z</i> 391
   <li>[M+Na]<sup>+</sup>:  <i>m/z</i> 413
   <li>[M+K]<sup>+</sup>:  <i>m/z</i> 429
   <li>[2M+NH<sub>4</sub>]<sup>+</sup>:  <i>m/z</i> 798
   <li>[2M+Na]<sup>+</sup>:  <i>m/z</i> 803
   </ul>
  <td>usually extracted from plastics including tubing and water storage
  containers that are plastic, but also present in solvents
</table>
<p>
Example spectra are shown below to indicate the appearance of the contaminants.
<p>
<img src="imagesProteomicsReferenceHTML/PEG-Contamination-ESI.gif"
alt="mass spectrum showing PEG-caused background contamination">
<img src="imagesProteomicsReferenceHTML/Fe-Acetate-Contamination-ESI.gif"
alt="mass spectrum showing Fe-acetate-caused background contamination">
<img src="imagesProteomicsReferenceHTML/MeCN-FA-background-ESI.gif"
alt="mass spectrum showing typical background on an ESI probe with MeCN-formic acid solvent">
<img src="imagesProteomicsReferenceHTML/MeCN-FA-background-ESI-nanoprobe.gif"
alt="mass spectrum showing typical background on an ESI nanoprobe with MeCN-formic acid solvent">

<h3>ESI-MS/MS (QToF micro) Start Up/Testing</h3>
<p>
Standard startup of the CapLC and Q-ToF micro is as follows:
<ol>
<li><b>CapLC Pump Priming</b>.  Begin a wet prime of all pumps if they
were used recently, otherwise do a dry prime.
<li><b>Autosampler</b>.  If necessary begin a wash of the autosampler
syringe.  Begin with MeOH first if there are air bubbles, and these
cannot be removed with standard wash (the standard wash should be
identical to solvent for Pump C usually).  Set the temperature of the
autosampler plate bed if necessary through the Inlet Method (done by
loading the method to the CapLC from MassLynx).
<li><b>Gases</b>.  Turn on the nitrogen generator.  Make sure the valve
to the CapLC system is set to 35-45 psi.  The nitrogen pressure to the
Q-ToF should be 7 psi or 0.5 bar (this is the valve labeled <b>Nanoflow</b>).
Turn on the argon gas cylinder at the main head.
The pressure on the low pressure valve should be 3-4 psi.
The argon pressure on the QToF is adjusted with the valve labeled
<b>Gas Cell</b> and should be adjusted (usually somewhat continuously at
first) until the Analyzer Penning monitor on MassLynx is about 6e-5.
<li><b>Control Reagents/Solutions</b>.  You may want or need to get
control peptides for testing chromatography, and for the lockmass spray.
<li><b>Equilibration</b>.  Depending on the chromatography setup (type of
column and diameter of column) begin passing solvents to equilibrate
the column.  For example, for 300-320 &micro;m C<sub>18</sub> reversed-phase
system, it is best to wash at first in the solvent (usually Pump B) with
organic solvent at 100% B, check the pressure, then in the highest %B
used in the chromatographic program to be used, and then in the lowest %B
in the program.  Check the pressure in all cases to see that the column
(or system of columns) has not changed since the last time it was used.
If it is too close to the pressure limit for the column (or the set limit),
considering re-conditioning or cleaning the column, or replacing it with
a new setup.  For pressures too low, consider the possibility of a leak
somewhere (these are much harder to detect for systems pumping &le;
20 &micro;l/min).
</ol>
<p>
Additional information is found in the <a href="QToFmicroUse.html">Q-ToF
<i>micro</i> web page</a>.

<h3>ESI-MS/MS (QToF micro) End of Day Shutdown</h3>
<ol>
<li><b>Argon</b>.  Turn off the argon gas at the main head, not the flow
valve or the low pressure valve.
<li><b>Nitrogen generator</b>.  Unless you are making sample runs overnight,
shut down the generator.
<li><b>Tune Page Overnight or Long Term No Use</b>.  Open the appropriate
IPR file for either overnight shutdown settings, or for a long term
no use.  Generally this file should set to zero the capillary and cone
voltages, reduce the MCP detector voltage to 1500 V, set the source block
and desolvation temperatures to room temperature, and turn off the gases.
<li><b>NanoLock Spray Controller</b>.  The controller not only operates
the oscillating baffle, but also the light.  Turn it off to keep lamp
life from being reduced.
<li><b>Samples Requiring Refrigeration</b>.  Any samples or materials
requiring freezing or refrigeration should be put in the proper place.
<li><b>CapLC flow</b>.  We want to attach a nitrogen gas tube to the
CapLC to allow overnight flow (the syringe pumps require gas for flow).
Make sure the overnight flow is set if it is desired.  This is only when
there is a period of intense use.  For long-term no use of the system,
fill the chromatography columns in the appropriate storage solution,
remove them from the system, and cap and store them.
</ol>


<h4>Precolumn Trapping on Stream Select Module</h4>
<p>
This tests if the peptide mixture is being trapped on the precolumn and
can be detected with a short gradient.

<h5>Stream Select Module Configuration</h5>
<p>
The following ports should be connected:
<ul>
<li><b>Port 5</b>: to 60 &micro;m (beige) PEEK to the frit, coming in
from the autosampler
<li><b>Ports 4 and 8</b>:  both to the waste solvent vial, with port 8
being 25 &micro;m capillary and port 4 being 75 &micro;m capillary
<li><b>Port 2</b>: plugged
<li><b>Port 3</b>:75 micro;m capillary going to the side port of the NanoTee
<li><b>Port 6</b>:  50 &micro;m capillary to the trap column cartridge
in its holder, continuing with a 50 &micro;m capillary to one of the end
ports of the NanoTee.  Off the other end port of the NanoTee,
the plumbing continues with a 50 &micro;m capillary to the NanoProbe.
Off the side port of the NanoTee, a 75 &micro;m capillary goes to Port 3.
<li><b>Port 7</b>:  60 &micro;m (beige) PEEK tubing to Valco Mixing Tee.
The other two ports of the Valco Tee should be connected to red PEEK
tubing going to Pumps A and B of the CapLC.
<li><b>Port 8</b>:  a 25 &micro;m capillary goes to the waste vial
</ul>
<p>
The image below shows the configuration (if not present, set up the
configuration as described above).
<p>
<img src="imagesProteomicsReferenceHTML/TrapColumnTesting.gif"
alt="set up for testing a trap column with peptides">

<h5>Inlet File</h5>

<ul>
<li><b>Initial Conditions</b>:  5% Solvent B with a 1 &micro;l/min flow
and Run Time = 12.0 minutes.  The Auxiliary flow will be 30 &micro;l/min.
<li><b>Gradient</b>.  All flow is 1.0 &micro;l/min and Curves are 6.
The following are time/%B pairs:  0.1/5, 3.0/5, 5.5/40, 6.0/60, 7.5/60,
7.7/5, 9.0/5, 12.0/5
<li><b>Auxiliary Pump</b>:  set the auxiliary pump to run as follows...the
numbers are  time(min)/flow(ul/min)/Curve:  0.1/30/6, 3.1/30/60,
3.3/1/6, 11.4/1/6, 11.5/30/6, 12.0/30/6
<li><b>Initial Events</b>:  switches 1 and 2: off, Steam Select: Pos 1,
Vent Valve: System
<li><b>Timed Events</b>:  3.0: SSV to 2, 11.8 SSV to 1
<li><b>Method Type</b>:  normal, and select Run Event Table
</ul>

<h5>MS</h5>
<p>
Scan in the <i>m/z</i> 400-1000 range with 1 seconds acquisitions, and
do it in MS mode.
<p>
The sample should be standard peptides.  If Sigma #H2016 is used, add
2.0 ml water, then dilute 500-fold in 0.1% formic acid.  Inject 1 &micro;l
using the Microliter Pickup mode in the


<h4>Setting Up Splits on CapLC with a 75 &micro;m column</h4>
<p>
The following configuration sets up a load of the front end of a trap
column and its elution from the front end.
<p>

Set up the ports and components to the Stream Select Valve as follows:
<ul>
<li><b>Port 7</b>:  a 25 &micro;m capillary is plumbed into the end port
of the Nano Tee.  Out the other end port, a 50 &micro;m capillary
leads to the outlet of a Valco Mixing Tee, whose other ports connect to
the red PEEK tubing lines leading to Pumps A and B of the CapLC.
The side port of the Nano Tee connects to a short 50 &micro;m capillary
which connects to a Microtight union which connects to a 25 &micro;m
capillary leading to waste.  The distance of the 50 &micro;m capillary
between Mixing and Nano Tees, and the distance of the 25 &micro;m capillary
from Nano Tee to port 7 must be as short as possible.
<li><b>Port 8</b>:  Connect a 25 &micro;m capillary to port 1
<li><b>Port 6</b>:  Connect 50 &micro;m capillary to the trap column
cartridge, and out the other end of the cartridge, continue with a
50 &micro;m capillary to port 3
<li><b>Port 5</b>:  Connect a 60 &micro;m PEEK to the frit which leads
to the same tubing to the autosampler valve where injection is done
<li><b>Port 4</b>:  a 75 &micro;m capillary to waste vial
<li><b>Port 3</b>:  a 50 &micro;m capillary leads to the back end of
the trap column cartridge
<li><b>Port 2</b>:  a 25 &micro;m capillary leads to the 75 &micro;m
analytical column, which continues with a 25 &micro;m capillary to the
ESI probe
<li><b>Port 1</b>:  connected with a 25 &micro;m capillary coming from
Port 8.
</ul>
<p>
<img src="imagesProteomicsReferenceHTML/UsingSplitsOnCapLC.gif"
alt="schematic for setting up splits on a CapLC with 75 micron column">

<p>
A flow of 200-300 nl/min will be delivered when the flow is 2-4 &micro;l/min.
Sample is trapped onto the short precolumn and then eluted to the
analytical column, then switching SSV after a wash/desalt period.
<p>
Start flow at 0.5 &micro;l/min with 50% B in SSV position 1.  Let
backpressure stabilize, then increase to 1 &micro;l/min.  Keep increasing
flow until the pressure stabilizes and desired flow is operating.

<h5>Inlet File</h5>
<ul>
<li><b>Initial Conditions</b>:  Set up solvent B to run at 5% with a flow
of 1 &micro;l/min.  Run time is 60 min.  The auxiliary pump (C) is also
run at 1 &micro;l/min.
<li><b>Gradient</b>: 0.1/5, 3.0/5, 30.0/40, 32.0/80, 40.0/80, 45.0/5,
60.0/5.  All flow is 1.0 &micro;l/min and Curves are 6.
<li><b>Initial Events</b>:  Switches 1 and 2 off, Vent Valve System, and
Stream Select Pos 1.
<li><b>Timed Events</b>:  after sample loading, stream select is switched
to Position 1 where the sample is backflushed from the trap column to the
head of the analytical.  Events are:  time 3.0: Position 2, time 59.7: Pos 1.
<li><b>Auxiliary Pump</b>:  Pump C is used for loading the sample and
washing the trap free of salts, and high flow rate is used for this purpose.
The following times, flow, and curves:  0.1/30/6, 3.10/30/6, 3.3/1/6,
60.0/1/6.
<li><b>Autosampler</b>:  set the draw height to 5 mm, <i>slow</i> draw
speed, <b>Partial Loop</b> injection type, flush vol = 3 &micro;l,
wash volume = 50 &micro;l, and sample temperature = 10&deg; with 20&deg; limit.
<li><b>Mix Methods</b>:  leave as defaults
</ul>

<h3>[Glu<sup>1</sup>]-Fibrinopeptide B Fragmentation</h3>
<p>
To check the MS/MS function of the Q-ToF, do the following:

<ol>
<li>Prepare 100 &micro;g/ml (0.1 mg/ml) of [Glu<sup>1</sup>]-Fibrinopeptide
B (Sigma F3261) in high purity water (no formic acid).
This is a 63.7 pmol/&micro;l solution.
<li>Add 1 volume of the 0.1 mg/ml stock to 5.37 volumes of 0.1% formic acid
This prepares a 10 pmol/&micro;l solution.
<li>Add 1 volume of the dilution to 49 volumes of 0.1% formic acid for
a 200 fmol/&micro;l lockmass working solution.
</ol>
<p>
Observe the following fragments:

<table>
<tr><th>Fragment                    <th>Mass     <th>Fragment      <th>Mass
<tr><td>y&prime;&prime;<sub>1</sub> <td> 175.1195<td>b<sub>1</sub> <td> 130.0504
<tr><td>y&prime;&prime;<sub>2</sub> <td> 246.1566<td>b<sub>2</sub> <td> 187.0719
<tr><td>y&prime;&prime;<sub>3</sub> <td> 333.1886<td>b<sub>3</sub> <td> 286.1403
<tr><td>y&prime;&prime;<sub>4</sub> <td> 480.2570<td>b<sub>4</sub> <td> 400.1832
<tr><td>y&prime;&prime;<sub>5</sub> <td> 627.3254<td>b<sub>5</sub> <td> 515.2101
<tr><td>y&prime;&prime;<sub>6</sub> <td> 684.3469<td>b<sub>6</sub> <td> 629.2531
<tr><td>y&prime;&prime;<sub>7</sub> <td> 813.3895<td>b<sub>7</sub> <td> 758.2957
<tr><td>y&prime;&prime;<sub>8</sub> <td> 942.4321<td>b<sub>8</sub> <td> 887.3383
<tr><td>y&prime;&prime;<sub>9</sub> <td> 942.4321<td>b<sub>9</sub> <td> 944.3597
<tr><td>y&prime;&prime;<sub>10</sub><td>1171.5020<td>b<sub>10</sub><td>1091.4281
<tr><td>y&prime;&prime;<sub>11</sub><td>1285.5448<td>b<sub>11</sub><td>1238.4966
<tr><td>y&prime;&prime;<sub>12</sub><td>1384.6133<td>b<sub>12</sub><td>1325.5286
<tr><td>y&prime;&prime;<sub>13</sub><td>1441.6348<td>b<sub>13</sub><td>1396.5657
</table>


<h4>Basic Gradient Using 320 &micro; &times; 150 mm Column</h4>
<p>
Use this is a modifiable template to get started.  This assumes the use
of a Waters Symmetry micro column.  <b>NOTE</b>:  there is no trap column
in this configuration.  Flow is 5-10 &micro;l/min, and run time is
the gradient time and equilibration time;  an equilibration time of
&ge; 15 min for a 150 mm column is recommended.  Where a column heater
is used, 40&deg; is recommended.

<h5>Inlet File</h5>
<ul>
<li><b>Initial Conditions</b>: Solvent A1 is 2% MeCN + 0.1% formic acid.
Solvent B is 98% MeCN + 0.1% formic acid, and is started at 3%.  Flow
is 5 &micro;l/min.  Run Time=50.0 min.  High pressure = 3000 psi.
<li><b>Gradient</b>:  the following are time/%B pairs.  Flow is always
5 &micro;l/min, and Curve is 6:  0.0/3, 25.0/40, 30/80, 31/3.
<li>All other tabs are not programmed:  no initial events, no timed events,
no auxiliary pump, no chart out.  Method type is default:  normal with
Run Event Table checked.
<li><b>Autosampler</b>:  set the draw height to 5 mm, <i>slow</i> draw
speed, <b>Partial Loop</b> injection type, flush vol = 3 &micro;l,
wash volume = 50 &micro;l, and sample temperature = 10&deg; with 20&deg; limit.
<li><b>Mix Methods</b>:  leave as defaults
</ul>

<h4>Basic Gradient Using 75 &micro; &times; 150 mm Nano Column With
Trapping</h4>
<p>
In this method a CapLC is run at 3.5-5 &micro;l/min with a splitter used
upstream of the capillary column providing 200-300 nl/min.  Sample loading
and washing is done at 20 &micro;l/min.

<h5>Inlet File</h5>
<ul>
<li><b>Initial Conditions</b>: The only change from the basic method above
is that the run time is 53, not 50, minutes, and the auxiliary pump
is set at 2.0 &micro;l/min with the same solvent as A1.
<li><b>Gradient</b>: 0.0/3, 3.0/3, 28.0/40, 33.0/80, 34/3.0.  All flow is
5.0 &micro;l/min and Curves are 6.
<li><b>Initial Events</b>:  Switches 1 and 2 off, Vent Valve System, and
Stream Select Pos 2.
<li><b>Timed Events</b>:  after sample loading, stream select is switched
to Position 1 where the sample is backflushed from the trap column to the
head of the analytical.  Events are:  time 0.0: Position 2, time 3.0: Pos 1.
<li><b>Auxiliary Pump</b>:  Pump C is used for loading the sample and
washing the trap free of salts, and high flow rate is used for this purpose.
The following times, flow, and curves:  0.0/2/6, 0.10/20/6, 3.0/2/11
<li><b>Autosampler</b>:  set the draw height to 5 mm, <i>slow</i> draw
speed, <b>Partial Loop</b> injection type, flush vol = 3 &micro;l,
wash volume = 50 &micro;l, and sample temperature = 10&deg; with 20&deg; limit.
<li><b>Mix Methods</b>:  leave as defaults
</ul>




<h2>Detectors: Multichannel Plate (MCP) Handling</h2>
<p>
[This information has been adapted from BURLE, a manufacturer of MCPs.]
<p>
Long-term storage of MCPs should be in an oil-free vacuum.  Any continuously
purged dry box with argon or nitrogen as a dry, inert gase is suitable.
It is not possible to use dessicators in which silica gel or solid
dessicants are the drying agents.
<p>
MCPs should be opened in a laminar flow clean room or hood and handled
with powder-free vinyl gloves.  The should be touched only on the glass
border and never on the active surface of the wafer.  It is vital that
particles not be allowed to contact the wafer (use a single-hair brsuh or
ionized dry nitrogen gun to remove any particles).  Bring the mount for
the MCP into hood, and assemble it there.
<p>
The MCP should never have its voltage applied to it unless the vacuum is
10<sup>&minus;</sup>5 millitorr or lower!!  Pressures of 10<sup>&minus;6</sup>
or lower are preferred.  A dry-pumped or well-trapped/diffusion-pumped
operating environment is best.
<p>
MCPs can be vacuum-baked (to dry them) to a temperature of 380&deg; or
better (no voltage).
<p>
When bringing MCPs to operating voltage, a slow increase in potential
is advised.  Voltages should be increased in steps of 100 volts.

<h2 id="chromprocess">Chromatogram Processing</h2>
<p>
<b>Note</b>:  this is <strong>not</strong> spectrum processing, but
(mass) chromatogram processing.  As any signal with peaks and baselines,
its background can be subtracted, and its plot smoothed.  Other analyses
can also be applied.
<p>
Most of the viewing and peak annotation options will not be described here.
<h3>Background Subtraction</h3>
<p>The chromatogram can be subtracted for background with the following
parameters:

<ul>
<li><b>Polynomial order</b>.  <i>Degrees of freedom</i> allowed to the fitted
curve.  Can be set from 0-99.  Value 0 fits a horizontal straight line.
Value 1 will use a sloping line to fit.  When the baseline is far from being
a straight line, a higher polynomial order is used.  The higher the order,
the more likely the fitted curve follows peak shape.  Typical orders are
from 3-20.
<li><b>Below Curve (%)</b>.  Moves background curve up and down in the noise.
The set value will force a certain percentage of data points making up
baseline to be <i>below</i> the curve that is fitted.  This value will be
typically between 5-30%, reflecting that 10-60% of a chromatogram trace is
made up of baseline.  Increase this value when there are fewer or narrower
peaks.  Max value = 99%.
<li><b>Tolerance</b>.  Permitted values are from 0.001 to 0.200, with the
default being 0.010.  This affects the precision of internal arithmetic.
<li><b>Flatten Edges</b>.  Causes the polynomial that is applied to be flat
at the beginning and end of the trace.
<li><b>Make graph of fitted polynomial</b>.  When checked, the fitted polynomial
itself is output to the plot rather than the background-subtracted chromatogram.
</ul>
<p>
When higher order polynomials are used, convergence might be a problem with
each iteration; a maximum of 300 iterations is imposed.  If the background
subtraction does not seem to make progress, cancel the operation and enter a
lower order polynomial.

<h3>Smoothing</h3>
<p>All types of smoothing available for chromatograms use a sliding window
that is defined by the number of scans as the full width of the window.
<p>
Two types of smoothing are available for chromatogram plots
<ol>
<li><b>Moving Mean</b>.  Evaluates the intensities as the arithmetic
mean of the signal data in the window.  Its disadvantage is that it
tends to flatten peaks, but it does not produce artifacts on either side of
peaks.
<li><b>Savitzky-Golay</b>.  The average of the signal is weighted by
a quadratic curve when smoothing, which has the effects of enhancing
peak and valley shapes.  Its advantage preserves the height of peaks better than the
<b>Moving Mean</b> method; its disadvantage is that it produces small artifacts
on either side of real peaks.
</ol>
<p>
The window size is the half-width of the smoothing window.  This is best
set by holding the right mouse button down along the half-height of a
chromatogram peak, which will enter the number of scans into the value box.
<p>
The default number of smooths is <b>2</b>.  A higher value will produce
a heavier smooth.

<h3>Integration</h3>
<p>Peak integration occurs in the chromatogam in a multi-stage process:
<ol>
<li><b>Smoothing</b>.  If the option is checked, it is performed.
<li><b>Differentiation</b>.  Signal is time-differentiated, and local
maxima and minima determined.
<li><b>Peak-to-peak noise</b> is determined wherein the difference in signal
from successive minima to maxima is determined.  Ratio of signal at the maximum
and of the two neighboring minima are calculated.  The maximum ratio represents
a peak if both ratios are small.  Baselines are set by joining neighboring
minima.  Shoulder peaks that are unreaolved are detected by searching
local gradient minima on sides of peaks.
<li><b>Baselines are finalized</b>.  Component peaks of unresolved multiplet
can be caused to share a common baseline with the setting of a parameter.
Two more parameters compensate for <i>peak tailing</i> effects (peak asymmetry).
Peak baseline can be adjusted to reduce asymmetry to specified maximum.
<li><b>Peak statistics</b> for all peaks are evaluated:  peak area is assessed
by trapezium rule, and vertical height of peak top above baseline is
determined.
<li><b>Peak thresholding and rejection</b> is done with setting of rejection
peak heights or areas being less than an absolute value, of if a specified
fraction of the height or area of largest peak.
</ol>

<h3>Other Chromatogram Determinants</h3>
<p>
<b>Peak Purity</b>.  MassLynx has the ability to assess peak purity.
One of two methods can be used:
<ol>
<li>The <b>simple method</b> works by selecting five spectra across the
peak and correlates each spectrum with the others.  A mean correlation
from 0 to 100% is determined, with 100% being total purity and 0% being
total impurity.  A value of 60% does <em>not</em> imply that there are
two components, for example the component of interest at 60% and another at 40%.
<li>The <b>Bayesian method</b> works be characterizing each mass channel
as a set of its first four moments:  peak position, peak width, peak
asymmetry are the first three moments.  Note that the max number of moments
used to evaluate can be set, which decreases runtime.  The number of mass
peaks used in comparison can also be reduced.  A rigorous probabilistic
analysis is done and the output value is the natural logarithm of the probabiliyt
that the peak is pure.  The purity value is <i>x</i>, and its probability
of purity is <b>exp(<i>x</i>)</b>.
</ol>
<p>
<b>Signal-To-Noise Ratio</b>.  These values can be presented in a
Peak-to-Peak fashion, or as RMS values.  A noise determination is made
by calculating its variance and its RMS deviation.  In assessing the noise,
the mean should first be calculated with or without zeroes (as normal).
Then optionally, 5% of the scans in the noise signal having the greatest
deviation from the mean are discarded OR scans whose deviation is greater
than 1 SD are discarded, OR scans in noise signal whose deviation is greater
than 2 SD are discarded.  With the first and third options, the results are
roughly equivalent, and with the 2nd option, the RMS value will be double that
of the other two options.  When an option to discard scans is done, the
mean and RMS values are determined from the remaining scans.  In completing
the Signal-To-Noise dialog, select scans/time representing signal (peaks)
and those representing noise (baseline), set whether zeroes are to be ignored
and then select the discard option.  Finally the option of showing
Peak-to-Peak or RMS values for S to N are given.

<h4>MS/MS and MS Mode Chromatograms</h4>
<p>
When a plot of TIC vs time or scan in MS mode is presented, a continuous
signal is shown for intensity.
<p>
But when the plot of TIC vs time/scan is shown when the instrument is
operating in MS/MS mode, it will show most a plot where the TIC is
zero, because most of the instrument's time is <em>not</em> spent in
MS/MS mode.  The time or scan in which the instrument is switched into
MS/MS mode will be evident.

<h4>Retention Index</h4>
<p>Used to compare results from different HPLC system and different
columns.  The parameter <b>LogP</b> is a measure of hydrophobicity.
A table of value pairs of Retention Index and a LogP is created
for a set of standard compounds.  Once the table is created,
a <b>calibration is made</b> in which the calibration parameters are
the starting index of the table and the end index is selected, and then
a peak difference, which is normally 1 to measure all peaks.  A higher number
can be used if secondary peaks are not to be used in the calibration.

<h2 id="spectrum">Spectrum Processing</h2>
<p>
The features of the Waters/Micromass MassLynx software will be described
as an example of the processing of mass spectra, which are probably common
to all applications that process mass spectra.

<h3 id="rawspec">The Raw Mass Spectrum</h3>
<p>
The typical mass spectrometer produces a continuous signal output in the
mass range selected.  Although the signal itself is not an analog output
but quantized as an integer, counts of ions may be registered that are
not true counts but reflect the noise invariably present in any instrument
detecting a signal and transducing it to another form of output.
There are really no discrete intervals between <i>m/z</i> values
(i.e., forbidden <i>m/z</i> values) at the level of the sensitivity of
a mass spectrometer, and so a mass spectrum is drawn as a continuum in
the signal output, with a baseline having Gaussian peaks.
<p>
With pulsed source mass spectrometers (all ToF mass spectrometers are
pulsed), a single pulse of ions is insufficient to present a spectrum
which is representative of the sample.  Several
pulses are obtained, and the ions at a certain <i>m/z</i> value are
accumulated (the sum of the counts of each pulse) to produce a raw
spectrum with high confidence (statisical significance) as representative
of the sample.  This is often called the &ldquo;summed&rdquo; spectrum
(the accumulation of many single pulsed spectra).  <b>Note very well</b>
that this spectrum is <strong>not</strong> an <i>average</i> of the
pulsed spectra, but the accumulation (sum of all ion counts at each
<i>m/z</i> value) of the pulsed spectra.
<p>
This represents the <b>raw</b> mass spectrum submitted for processing.

<h3>Combining Spectra</h3>
<p>
The MassLynx software combines spectra from the <b>Chromatogram</b>
window.  The menu command <b>Process&rarr;Combine Spectra...</b>
calls up the <b>Combine Spectrum</b> interface.  In the <b>Average</b>
textbox control, one specifies a <i>range</i> of spectra by indicating
the starting and ending scan numbers separated by a colon character;
for example <q>1:12</q> specifies the range of scan 1 to 12.  While
this interface is open, one can drag the mouse across the chromatogram
while holding the right mouse button down, and the range will immediately
appear in the textbox.  For
<p>
If the option <b>Use Automatic Combine Parameters</b> is checked
<em>and</em> centroided data was acquired, peak combine values will
be set using the instrument resolution and peaks
with possible mass measurment errors will be excluded.  Select this option
if it is available (it is recommended).  For continuum data, use the
<b>Automatic Peak Detection</b> option once the combine is complete.
<p>
<b>Peak separation</b> (in Da) specifies the maximum width of peaks in amu,
and is used to determine which peaks are to be regarded as the same peak
from scan to scan.  A value of 0.05 Da (50 mDa) is recommended.
<p>
<b>Subtract</b>

<h3 id="bkgdsub">Background Subtraction</h3>
<p>
Background subtraction is the first phase of what is called
<b>de-noising</b> or <b>noise reduction</b> or <b>noise filtering</b>.
Its purpose is essentially is to drop the spectral baseline down to the
zero intensity.
<p>
<b>Polynomial Order</b>.
In doing a background subtract, a polynomial is fit to create the line.
The <b>polynomial order</b> should be set to the line shape.  A polynomial
order of <b>0</b> fits the background to a straight line.  If the polynomial
order is <b>1</b>, a sloping straight line is fitted.  If a very high value
polynomial order is set, the curve that is fit begins to follow the peak
shapes.  Although the maximum value is <b>99</b>, typical values range
from 3-20.
<p>
<b>Percent Below Curve</b>.  This determines which data points at
a certain relative intensity will be placed beneath the fitted background
curve.  The typical value is between 5-30% for chromatograms and
40% for ESI spectra, and is dependent on the
abundance and width of peaks in the chromatogram.  Because in ESI spectra,
80% of the data in the spectrum is noise and 20% is signal;  because
half of the noise is <i>above</i> the zero line and half is <i>below</i>
the zero line, then half of 80% is 40%.
Increase the value when peaks are fewer or narrower.
<p>
<b>Tolerance</b>.  Controls the precision in which internal arithmetic
is done; this value is essentially at which the limit of convergence is
defined and stops iterations.  The minimum is 0.001 and maximum 0.200, but the default of
0.010 is suitable.  Increasing the parameter makes the algorithm terminate
sooner, but it may not give a satisfactory result.
<p>
<b>Flatten Edges</b>.  If indicated, will cause applied polynomial fit
to be flat at the beginning and end of trace.
<p>
<b>Make graph of fitted polynomial</b>.  At the end of the subtract
process the background curve substracted will be shown instead of the
chromatogram.
<p>
Note that the Transform and MaxEnt processes rely on having background
subtraction done.  Electrospray data essentially have a curved background
and are helped by background subtraction.

<h3 id="smooth">Smoothing</h3>
<p>
Smoothing is the second phase of de-noising of signal data with a time
domain.  Some algorithms work best on spectral data. Note that
the smoothing process can be
re-iterated:  the more times iteration is done, the final process
will show a flatter signal.
<p>
<b>With the exception of data to be analyzed by MaxEnt</b>, all spectral
data should be smoothed or the centering process is likely to produce
output which includes noise.


<h4>Window Size</h4>
<p>All algorithms use a window that is determined to be the quotient of
the peak-width at half-height (50% intensity of the base peak) divided by
three times spacing between the adjacent points on the mass axis, which
is 62.5 mDa for raw continuum data, or it is equal to the <b>Resolution</b>
parameter used for MaxEnt or Transform data.

<h4>Moving Mean</h4>
<p>
This smoothing is rather simple.  It essentially average all the data
points in the window.  Its chief disadvantage is that the output will
result in broader peaks, while SG results in a less artifactual broadening.

<h4>Savitzky-Golay</h4>
<p>
This is a moving window averaging of the data in which the points in the
window are fit with a 2nd order polynomial (quadratic) curve.  While
peaks and valleys are enhanced and small artifacts are seen on either
side of real peaks, this algorithm does a better job of avoiding excessive
flattening of signficant signal data.

<!--
<p>
The typical moving window average is described by
<p class="eqnserif">
<i>g<sub>i</sub></i> = <span style="font-size:150%;">&sum;</span>
 <sub style="margin-left:-2.5em;vertical-align:-1.5em;"><i>n</i> =
  &minus;<i>n<sub>L</sub></i></sub> <sup
 style="margin-left:-2.5em;vertical-align:2em;"><i>n<sub>R</sub></i></sup>
 <i>c<sub>n</sub> f<sub>i+n</sub></i>
<p>
where <i>g<sub>i</sub></i> is the average calculated at data point <i>i</i>
(in the time domain), <i>n<sub>L</sub></i> and <i>n<sub>R</sub></i>
are a number of points to the left and right of <i>i</i> and forming the
window, and the coefficient <i>c<sub>n</sub></i> = 1/(<i>n<sub>L</sub></i> +
<i>n<sub>R</sub></i> + 1), or the number of points in the window
forming the average.  A polynomial of degree <i>M</i> will be fit
to values <i>f<sub>&minus;n<sub>L</sub></sub>,...,f<sub>n<sub>R</sub></sub></i>
with the equation <i>a</i><sub>0</sub> + <i>a</i><sub>1</sub><i>i</i> + ... +
<i>a<sub>M</sub>i<sup>M</sup></i>
-->

<h4>Moving Median</h4>
<p>
If there are a lot of spikes in the signal data which are not real peaks,
such impulses in electronics, try this smoothing algorithm.

<h4>Calculating the Window</h4>
<p>
The window used in all algorithms is usually governed by the following
equation:

<table class="eqnserif">
<tr><td rowspan="2">Half width of window =
 <td class="numer">Full peak width at 50% intensity
<tr><td>3 <i>&delta;m</i>
</table>

<p>
where <i>m</i> is the spacing between data points in the <i>m/z</i>
axis (16 data points per dalton, or 62.5 mDa for raw continuum data).
<p>
As an example, suppose the full peak width at 50% intensity is 13 channels.
Since &delta;<i>m</i> is 1 channel, then the half width of the smoothing 
window will be 13/3 ~ 4.  Thus the value should be 4.

<h4>Iterations</h4>
<p>A default number (maybe <b>2</b>) might be set.  The number of iterations
will apply only to the Moving Mean and S-G algorithms, and not the
Moving Median, which continues to iterate until there is no change
in the spectrum window result between the input and output data.

<h3 id="centroiding">Centroiding</h3>
<p>When done on continuum or MCA data, a peak center is determined and a
vertical bar from the centroid of that peak to the horizontal axis
is presented, providing both the mass and preservation of its intensity
as the height to the centroid.
<p>
There are actually <b>three</b> methods of centering:
<ol>
<li><b>Peak top</b>.
One is merely to choose the most intense point of the peak&mdash;the
peak <b>top</b>&mdash;and construct the vertical bar from it to the axis.
This method is best for <i>electrospray</i> data which has unresolved
adducts.
<li><b>Peak centroid</b>.
Another is to calculate the <b>centroid</b> of the peak, which is a
method that provides the most accurate mass so long as there are not
unresolved adducts
<li><b>Peak median</b>.  This takes the median of the peak area, which
is equivalent to drawing the line so that half of the peak area lies on
either side.
</ol>
<p>One important thing to consider:  Masses determined from different
experiments centered by different methods should <strong>not</strong> be
compared.
<p>Generally the median and centroid methods produce little practical difference
for symmetrical peaks, but will be different for asymmetrical peaks.
The median is more robust for quite asymmetrical peaks.
<p>
The parameters required for centering include:

<ul>
<li><b>Minimum Peak Width At Half-Height</b>.  This is the number of channels.
Each channel point is the time it takes to collect data:  as there are
16 channels per dalton, the channel width is 62.5 mDa.  A safe choice for
continuum or MCA data is <b>4</b> which reprsents 4 * 62.5 mDa = 0.25 Da.
In using MaxEnt analysis, the peaks can be narrow, so a safer choice is
2 channels.  If the setting is too low, centering will include bars that
come from high frequency noise.  Too high a value will produce centered
spectra in which multiple peaks are represented by a single bar.
<li><b>Method of Centering</b>:  one of three are chosen: <b>top, centroid top
as a percentage,</b> and <b>median</b>.  Percentage reflects the fraction
of the resolved portion of the peak that is used to calculate the centroid.
A default of 80% is given, but recommended values can vary from 60-95%.
<li><b>Intensity as Height or Area</b>:  The option can be to choose the
bar height as a representation of the <b>intensity of the continuum trace</b>
or the <b>height</b>, or to use the <b>sum of the intensities of the points
acrorss the peak in the continuum trace</b> or the <b>area</b>.
</ul>

<h3>Special Spectrum Processing</h3>

<h4>TOF Accurate Mass</h4>
<p>
ToF data can be adjusted using several parameters used in accurate mass
determination.
<ul>
<li><b>Resolution</b>.  Enter this as the approximate resolution of
electrospray data.  For the Q-ToF, this is usually about 5000, but should
be assessed from time to time.
<li><b>Np Multiplier</b>.  This is a correction factor for the <b>number
of pushes</b>, usually a value of 0.7.
<li><b>Lock Mass</b>.  The value for a reference lock mass, if used
<li><b>Lock Mass Window</b>.  The width of the window (in daltons) used to
locate the lockmass.  The limits of the window will be the set value for
the lockmass &plusmn; the window value.  Thus if the window value is
1.000 and the lockmass set value is 785.5, the window searched is that
from <i>m/z</i> 784.5 to 786.5, and the most intense peak is found in that
window, processed, and its mass used in correction.
</ul>

<h4>Integration</h4>
<p>Like all continuum signal data, a mass spectrum has a baseline and peaks which
can be identified by an integration process.  The integration process is
less complex than are the options provided for the mass chromatogram.

<h4>TOF Transform</h4>
<p><b>Note</b>:  this is distinct from the <b>Transform</b> process which
is an option of another kind.
<p>Normally this works on centroided Q-ToF data.  It results in de-isotoping
and then produces a +1 or &minus;1 charge state spectrum, which shows the
actual masses by presenting a virtual <i>m/z</i> spectrum.
<p>
There are three parameters that must be provided in producing the transformed
spectrum:
<ol>
<li><b>Final or virtual spectrum mass range</b>.  The <b>Min Molecular mass</b>
and <b>Max Molecular Mass</b> apply to what the final output mass specrum
mass range will be.  Suppose that the largest mass of interest in the centroided
spectrum is 800 and the charge state is 3:  thus the transform spectrum will
put the ion at 2400 for a charge state of 1, and so the max range should be
2400 or more.
<li><b>Max Charge State</b>.  This value will determine which ions are to
be transformed.  Those found to have a higher charge state will not be
transformed and be presented in the final spectrum.  A max charge state of
4 is allowed.
</ol>

<h4>MaxEnt</h4>
<p>MaxEnt uses <b>maximum entropy</b> method to produce single charged
state spectra from multiple-charged state (electrospray) data.  It differs
from the <b>Transform</b> process in that it:
<ul>
<li>Automatically finds molecular masses of ions without any knowledge
except that they lie within a specified mass range, which can be from
5000 to 100,000 Da.  A preliminary survey run produces a coarse output
and then finds approximate masses present, in order to reduce processing
time.
<li>The MaxEnt output spectrum has enhanced resolution and signal-to-noise
ratio.
<li>Probalistic methods can be used to assess result reliability, and thus
a probable error range calculated for each mass.
<li>Areas of the peaks in profile spectra are representative of the summed
intensities of each components multiply-charge series, and thus are quantitative.
</ul>
<p>The output spectrum is the simplest with respect to being of maximum
entropy.  After an initial analysis of the spectrum, knowledge of chemistry
and MS physics (the <b>damage model</b>) is used to produce the mock spectrum.
The mock spectrum is compared to the real data, and this difference used to
improve the final spectrum.  The iteration terminates when the mock spectrum
and real spectrum show little difference.
<p>
The shape and width of spectral peaks are a composite of two effects: one
chemical.  Another is that the distribution of isotopic peaks has a characteristic
shape that is a function of molecular mass.  Another effect is physical being
the diffraction effects in the mass spectrometer.  The latter effect can be
seen by looking at cesium iodide monoisotopic sample.  A single damage model
is provided:  it is a guassian curve of constant width, and composite model
of the effects above.  This requires knowing the half-height peak width in the
observed spectral data.

<p>The <b>MaxEnt</b> algorithm requires the use of <b>RAW</b> unprocessed
spectra, because it has all the functions necessary to subtract and
smooth data.
<p>
The following user-specified parameters are required:
<ul>
<li><b>Output Ranges</b>.  For a single range, specify the <i>m/z</i> range
as low mass and high mass separated by a colon (<b>x:y</b>).  For multiple
range outputs, separate ranges using a comma (<b>w:x,y:z,a:b</b>).
<li>Specify the <b>Resolution</b> (in Da/channel) of the output.  Default might
be 1, but could be higher.
<li><b>Damage Model</b>.  The <b>Uniform Gaussian</b> model is typical.
Specify in Da the width at half-height (50% intensity).  Alternatively, the
<b>Simulated Isotope Pattern</b> can be selected, and the <b>Spectrometer
Blur Width</b> (in Da) provided.
<li><b>Minimum Intensity Ratios in Frame</b>   Parameters in this frame
put limits on relative heights of adjacent peaks in the same series.
One is set for the <b>Left</b> or low mass range and <b>Right</b> for high
mass range.  If Left is set 30%, then if the most intense peak is 15+, the
16+ peak must be at least 30% as intense as the 15+, the 17+ peak 30% as intense
as 16+ and so on.  From the Right side, if set 40%, the 14+ peak must be
40% intense as the 15+ peak, the 13+ peaks at least as 40% as the 14+, and so
on.
</ul>

<h2>Other MassLynx-Specific Modules, Features, Functions</h2>

<h3>Strip and Combine Functions</h3>

<h4>Strip</h4>
<p>This application removes unwanted background and noise.  The input
file are files from the RAW directory and the output is a RAW directory with
new data files processed.  The processing options are:

<ol>
<li><b>Enhance</b>.  Data points in spectrum are evaluated in relation to
their neighbors to determine whether it is noise.  Noise data points are
removed in the output file.  This can significantly reduce data file size.
<li><b>Subtract</b>.  Can be used either to subtract a background spectrum
or one data file from another.  Both centroid and continuum data can be
subtracted from the same type.
<li><b>Cluster</b>.  Pairs or triplets of peaks can be detected by a specified
mass difference.  Requires specification of mass differences and the
expected intensity ratio (both with tolerances).  A time window and global
threshold are also required.
<li><b>COmponent Detection Algorithm (CODA)</b>.  Removes mass chromatograms
representing background noise from the dataset.  The raw mass chromatogram
is compared to smoothed standardized mass chromatogram:  masses in which
background noise is high or where spikes are present are rejected.
</ol>

<h4>Combine</h4>
<p>This combines all functions in a data file to make a new data file that
contains a single function, and is the sum of the multiple functions.
Functions must be acquired in the same scan range and rate or contain SIR
or MRM channels.  This function is useful for combining function acquired
with different cone voltage or collision energy values.

<h3>Accurate Mass Measure</h3>

<h3>Molecular Mass Calculator</h3>

<h3>Quantify</h3>

<h3>Databridge</h3>

<h3>Library</h3>

<h3>Map</h3>
<p>Map provides a 3D representation of the entire data file.  In this way
it provides a complete data file overview rather quickly, and is very useful
for LC-MS data.  It can be searched for particular masses and provide
simultaneous display of mass chromatograms and spectra.  The display is
in three parts:
<ol>
<li>The top trace shows mass chromatograms of the currently selected mass.
<li>The bottom trace shows spectrum for currently selected retention time.
<li>The middle trace shows a map display of mass in <i>m/z</i> units (Da/e)
against the retention time (in minutes) in a 2D bottom plane, while the
3rd axis is the intensity, and its appearance is affected by a mapping mode
and color scheme.
</ol>



<h1>Databases</h1>
<p>
Databases contain the useful information that enab.  For proteomics,
the databases of in

<h2>Database Structure</h2>
<p>
Databases are essentially, or can be constructed as <b>tables</b>
(data in a tabular form), and like all tables, are composed of <b>rows</b>
and <b>columns</b>.  Database engineers and designers call the rows
of these database tables <b>records</b>, and they call the columns
<b>fields</b>.  A field holds some characteristic or property that
is descriptive of the record.  Not all records will contain a value
for a field (especially when the value is either unknown or impossible to
assign).
<p>
The records of a database are usually <b>unique</b>:  there is no
real purpose to having multiple records in which all fields are the same.
Database managment systems often feature routines that find replicate
records and then resolve the issue by eliminating the repeat occurrences.
Multiple records can occur because databases might be distributed as copies
among several sites, and when the updates between the copies occurs, the
replicate records are found.
<p>
One or more fields might be used that make the record unique, and which
are used to search databases to find records matching a certain criterion.
These fields are often called <b>keys</b>.  There is usually one
<b>primary</b> key, the key that is used to give the record uniqueness,
and it can be made up of one or more fields (a combination of fields
can create uniqueness).  Other keys might be secondary keys.
<p>
The theory of the management and distribution of databases, including
updating of these databases, is a complex part of computer engineering
(a specialty in electrical engineering usually).  Databases can be
created from databases in fact.  A database might actually be created
that is an index for searching a larger database.   Consult the many
books there are on the theory of the design and managment of databases.

<h3>SWISS-PROT DB Structure</h3>
<p>
The SWISS-PROT database is the go-to source of information for the
expression proteomist.

<h2>Annotation</h2>
<p>
Some databases contain
only raw data, and other usually more useful databases contain
<em>annotated</em> data.  The annotation of data is basically the act
of making sense of data.  For instance, it is one thing

<h2>Useful Proteomics Databases</h2>

<ul>
<li><a href="http://www.mpiib-berlin.mpg.de/2D-PAGE">Proteome Database
System for Microbial Research at Max Planck Institute for
Infection Biology</a>

<li><a href="http://expasy.ch/chad/2d-index.html">Swiss-Prot 2D Gel
Electrophoresis Map Database</a><br>
Contains images of 2D gels for a variety of tissues.
</ul>

<h1 id="protident">Protein Identification</h1>
<p>
The purpose of using mass spectrometric data is to generate either
information about a unique set of masses from the peptides obtained
by digestion of the protein (when MALDI MS is used to get the
<q>peptide mass fingerprint</q>) or to get one peptide mass with
partial or even near-complete sequencing of the peptide discerned from
controlled fragmentation spectra (MS/MS) of the peptide.  This information
is then used to search the proteomic databases (typically translated
genomic databases) with the hope of identifying the protein, or at least
its gene if no function is found.

<h2>Scoring Hits</h2>
<p>
The quality of a protein identification with MS data (the quality of
the database search) is scored on a number of factors.  For the
Waters/Micromass PLGS system, the score for a protein is:

<p style="text-align:center;">
<table class="eqnserif">
<tr><td rowspan="2">Score = <b>ln</b> <span style="font-size:200%;">[</span>
 <td style="border-bottom:1px solid black;">
 probability of protein given the data and initial assumptions
 <td rowspan="2"><span style="font-size:200%;">]</span>
<tr><td>probability of protein given initial assumptions
</table>

<ul>
<li>For a database with <i>N</i> proteins, each protein has an equal
probability of being the protein matched to a spectrum, and so the
probability of a protein being identified is 1/<i>N</i>.
<li>The maximum possible score for a protein with prior probability of
1/<i>N</i> will be ln(N).  If the data provide no information then the
minimum possible top score is zero (0).
<li>As part of the report of the scoring, the posterior probability
is given, i.e. the probability of the protein identified <em>given</em>
both the data and initial assumptions (presented as a percentage).
<li>Peak intensity is factored as <i>R</i> &times; (peak area &divide;
area standard deviation)<sup>2</sup>, where <i>R</i> is a constant
showing reliability of detected counts.  This computation relates
directly to signal-to-noise.
<li>The count of matched and unmatched peptides is a factor.
</ul>

<p>
Each peptide is scored if it is sequenced, given a likelihood value
whose natural log is input into the score.

<h2>Evaluating The Hits</h2>
<p>
The PLGS user guide presents the question <q>How do I know if a hit
is real?</q>
<p>
The answer is in the answers to the following questions:
<ul>
<li>Is the score of the top-scoring protein differ by 5 or more
(a factor of ~150, since ln[150] ~ 5)?  If yes, it will probably have
a confidence level near 100% and also comprise nearly all of the probability.
<li>Does the top-scoring protein share the probability of being identifed
with those proteins ranked immediately below it?  If yes, the top-ranked
protein will likely have a lower confidence level approximately the same
as the proteins scoring below it.
<li>Are the topmost two, three, four, five proteins actually homologous,
i.e., just species variations of the same protein with the same function?
If proteins with two identical sequences were located, they share the
probability/confidence (=0.5) and so the topmost score is ln(<i>N</i> &times;
0.5).  The database query will be presented as a <q>collapsed</q> hit
in most cases, but not all.
</ul>
<p>
Confidence level is determined directly from the score:
<p class="eqnserif">
Confidence = exp(score &minus; ln(<i>N</i>))
<p>
where <i>N</i> is the number of proteins in the database.  A protein
with score of 11.46 and 100,000 proteins in the database will have a
95% confidence.  The highest possible score is 11.51 (= ln(<i>N</i> &times;
0.95) = ln(<i>N</i> + ln(0.95))
+ ln(100%)).

<h2>Mixtures of Proteins in A Sample Spectrum</h2>
<p>
In the cases where spectra may contain more than one protein, re-submit
the search but exclude the topmost hit.
<!--  APPENDICES  -->

<h1 id="appendix">Appendix</h1>

<h2>Masses and Abundances of Biologically Relevant Isotopes</h2>

<table style="white-space:pre;">
<tr><th>Isotope           <th> Mass (amu)     <th> Relative Abundance
<tr><td><sup>1</sup>H     <td> 1.007825037    <td>100.0
<tr><td><sup>2</sup>H     <td> 2.014101787    <td>  0.015
<tr><td><sup>12</sup>C    <td>12.00000        <td> 98.9
<tr><td><sup>13</sup>C    <td>13.003354       <td>  1.11
<tr><td><sup>14</sup>N    <td>14.003074       <td> 99.6
<tr><td><sup>15</sup>N    <td>15.00011        <td>  0.36
<tr><td><sup>16</sup>O    <td>15.99491464     <td> 99.8
<tr><td><sup>17</sup>O    <td>16.99915939     <td>  0.40
<tr><td><sup>18</sup>O    <td>17.99915939     <td>  0.200
<tr><td><sup>31</sup>P    <td>30.9737634      <td>100
<tr><td><sup>32</sup>S    <td>31.972074       <td> 95
<tr><td><sup>33</sup>S    <td>32.9707         <td>  0.76
<tr><td><sup>34</sup>S    <td>33.96838        <td>  4.22
<tr><td><sup>36</sup>S    <td>35.96676        <td>  0.20
<tr><td><sup>23</sup>Na   <td>22.9897679      <td>100
<tr><td><sup>35</sup>Cl   <td>34.968854       <td> 75.8
<tr><td><sup>37</sup>Cl   <td>36.965896       <td> 24.2
</table>


<h2>Mass Excess versus Mass</h2>
<table>
<tr><th>Mass <th>Mass Excess
<tr><td>1    <td>0.06
<tr><td>2    <td>0.11
<tr><td>3    <td>0.17
<tr><td>4    <td>0.23
<tr><td>5    <td>0.29
<tr><td>6    <td>0.34
<tr><td>7    <td>0.40
<tr><td>8    <td>0.46
<tr><td>9    <td>0.51
<tr><td>10   <td>0.57
<tr><td>11   <td>0.62
<tr><td>12   <td>0.68
<tr><td>13   <td>0.73
<tr><td>14   <td>0.79
<tr><td>15   <td>0.85
<tr><td>16   <td>0.91
<tr><td>17   <td>0.97
<tr><td>18   <td>0.02
<tr><td>19   <td>0.08
<tr><td>20   <td>0.13
<tr><td>21   <td>0.19
<tr><td>22   <td>0.25
<tr><td>23   <td>0.31
<tr><td>24   <td>0.37
<tr><td>25   <td>0.42
</table>


<h2 id="commontrypticpeptides">Tryptic Peptides of Common Protein Standards</h2>

<h3 id="adhpeptides">Yeast <i>Saccharomyces cerevisiae</i> Alcohol Dehydrogenase</h3>

</div> <!-- class=contentblock -->

<div class="widetable">
<table class="pepmasses" style="margin-left:-10%;">
<caption>ADH Peptides (Monoisotopic) in Mass Range 1000-3000</caption>
<tr><th>Mass
 <th>Position
 <th>Count Missed Cleavages
 <th>Artificial (Fixed) Modification
 <th>Natural (Variable) Modification
 <th>Peptide Sequence
<tr><td>8419.830
 <td>84-160
 <td>1
 <td>Cys_CAM: 97, 100, 103, 111, 153
 <td>
 <td>IGDYAGIKWLNGS<b>C</b>MA<b>C</b>EY<b>C</b>ELGNESN<b>C</b>PHADLSGYTHDGSFQQYATADAVQAAHIPQGTDLAQVAPIL<b>C</b>AGITVYK
<tr><td>8134.722
 <td>84-160
 <td>1
 <td>
 <td>
 <td>IGDYAGIKWLNGSCMACEYCELGNESNCPHADLSGYTHDGSFQQYATADAVQAAHIPQGTDLAQVAPILCAGITVYK
<tr><td>7914.612
 <td>92-163
 <td>1
 <td>Cys_CAM: 97, 100, 103, 111, 153
 <td>
 <td>WLNGS<b>C</b>MA<b>C</b>EY<b>C</b>ELGNESN<b>C</b>PHADLSGYTHDGSFQQYATADAVQAAHIPQGTDLAQVAPIL<b>C</b>AGITVYKALK
<tr><td>7629.505
 <td>92-163
 <td>1
 <td>
 <td>
 <td>WLNGSCMACEYCELGNESNCPHADLSGYTHDGSFQQYATADAVQAAHIPQGTDLAQVAPILCAGITVYKALK
<tr><td>7602.396
 <td>92-160
 <td>0
 <td>Cys_CAM: 97, 100, 103, 111, 153
 <td>
 <td>WLNGS<b>C</b>MA<b>C</b>EY<b>C</b>ELGNESN<b>C</b>PHADLSGYTHDGSFQQYATADAVQAAHIPQGTDLAQVAPIL<b>C</b>AGITVYK
<tr><td>7317.289
 <td>92-160
 <td>0
 <td>
 <td>
 <td>WLNGSCMACEYCELGNESNCPHADLSGYTHDGSFQQYATADAVQAAHIPQGTDLAQVAPILCAGITVYK
<tr><td>4475.218
 <td>39-80
 <td>1
 <td>Cys_CAM: 43
 <td>
 <td>YSGV<b>C</b>HTDLHAWHGDWPLPVKLPLVGGHEGAGVVVGMGENVK
<tr><td>4418.196
 <td>39-80
 <td>1
 <td>
 <td>
 <td>YSGVCHTDLHAWHGDWPLPVKLPLVGGHEGAGVVVGMGENVK
<tr><td>3469.747
 <td>30-59
 <td>1
 <td>Cys_CAM: 43
 <td>
 <td>ANELLINVKYSGV<b>C</b>HTDLHAWHGDWPLPVK
<tr><td>3412.726
 <td>30-59
 <td>1
 <td>
 <td>
 <td>ANELLINVKYSGVCHTDLHAWHGDWPLPVK
<tr><td>3278.656
 <td>164-196
 <td>1
 <td>
 <td>
 <td>SANLMAGHWVAISGAAGGLGSLAVQYAKAMGYR
<tr>
 <td>3107.633
 <td>226-257
 <td>1
 <td>
 <td>
 <td>DIVGAVLKATDGGAHGVINVSVSEAAIEASTR
<tr><td>3012.609
 <td>161-191
 <td>1
 <td>
 <td>
 <td>ALKSANLMAGHWVAISGAAGGLGSLAVQYAK
<tr><td>2730.381
 <td>234-260
 <td>1
 <td>
 <td>
 <td>ATDGGAHGVINVSVSEAAIEASTRYVR
<tr><td>2723.331
 <td>261-286
 <td>1
 <td>Cys_CAM: 276, 277
 <td>
 <td>ANGTTVLVGMPAGAK<b>CC</b>SDVFNQVVK
<tr><td>2700.393
 <td>164-191
 <td>0
 <td>
 <td>
 <td>SANLMAGHWVAISGAAGGLGSLAVQYAK
<tr><td>2609.288
 <td>261-286
 <td>1
 <td>
 <td>
 <td>ANGTTVLVGMPAGAKCCSDVFNQVVK
<tr><td>2588.260
 <td>276-298
 <td>1
 <td>Cys_CAM: 276, 277
 <td>
 <td><b>CC</b>SDVFNQVVKSISIVGSYVGNR
<tr><td>2475.166
 <td>39-59
 <td>0
 <td>Cys_CAM: 43
 <td>
 <td>YSGV<b>C</b>HTDLHAWHGDWPLPVK
<tr><td>2474.217
 <td>276-298
 <td>1
 <td>
 <td>
 <td>CCSDVFNQVVKSISIVGSYVGNR
<tr><td>2418.145
 <td>39-59
 <td>0
 <td>
 <td>
 <td>YSGVCHTDLHAWHGDWPLPVK
<tr><td>2390.265
 <td>60-83
 <td>1
 <td>
 <td>
 <td>LPLVGGHEGAGVVVGMGENVKGWK
<tr><td>2312.148
 <td>234-257
 <td>0
 <td>
 <td>
 <td>ATDGGAHGVINVSVSEAAIEASTR
<tr><td>2019.069
 <td>60-80
 <td>0
 <td>
 <td>
 <td>LPLVGGHEGAGVVVGMGENVK
<tr><td>1987.017
 <td>207-223
 <td>1
 <td>
 <td>
 <td>EELFRSIGGEVFIDFTK
<tr><td>1961.997
 <td>1-17
 <td>1
 <td>
 <td>acetylation: 1
 <td><b>S</b>IPETQKGVIFYESHGK
<tr><td>1919.986
 <td>1-17
 <td>1
 <td>
 <td>
 <td>SIPETQKGVIFYESHGK
<tr><td>1888.127
 <td>22-38
 <td>1
 <td>
 <td>
 <td>DIPVPKPKANELLINVK
<tr><td>1873.068
 <td>315-331
 <td>1
 <td>
 <td>
 <td>SPIKVVGLSTLPEIYEK
<tr><td>1835.982
 <td>319-334
 <td>1
 <td>
 <td>
 <td>VVGLSTLPEIYEKMEK
<tr><td>1804.974
 <td>258-275
 <td>1
 <td>
 <td>
 <td>YVRANGTTVLVGMPAGAK
<tr><td>1694.882
 <td>287-302
 <td>1
 <td>
 <td>
 <td>SISIVGSYVGNRADTR
<tr><td>1643.854
 <td>1-17
 <td>1
 <td>
 <td>
 <td>GVIFYESHGKLEHK
<tr><td>1618.843
 <td>197-211
 <td>1
 <td>
 <td>
 <td>VLGIDGGEGKEELFR
<tr><td>1569.816
 <td>212-225
 <td>1
 <td>
 <td>
 <td>SIGGEVFIDFTKEK
<tr><td>1522.768
 <td>192-206
 <td>1
 <td>
 <td>
 <td>AMGYRVLGIDGGEGK
<tr><td>1447.804
 <td>319-331
 <td>0
 <td>
 <td>
 <td>VVGLSTLPEIYEK
<tr><td>1421.775
 <td>335-347
 <td>1
 <td>
 <td>
 <td>GQIVGRYVVDTSK
<tr><td>1411.696
 <td>299-310
 <td>1
 <td>
 <td>
 <td>ADTREALDFFAR
<tr><td>1400.826
 <td>18-29
 <td>1
 <td>
 <td>
 <td>LEHKDIPVPKPK
<tr><td>1386.741
 <td>261-275
 <td>0
 <td>
 <td>
 <td>ANGTTVLVGMPAGAK
<tr><td>1365.753
 <td>303-314
 <td>1
 <td>
 <td>
 <td>EALDFFARGLVK
<tr><td>1355.608
 <td>276-286
 <td>0
 <td>Cys_CAM: 276, 277
 <td>
 <td><b>CC</b>SDVFNQVVK
<tr><td>1312.678
 <td>212-223
 <td>0
 <td>
 <td>
 <td>SIGGEVFIDFTK
<tr><td>1251.669
 <td>287-298
 <td>0
 <td>
 <td>
 <td>SISIVGSYVGNR
<tr><td>1241.565
 <td>276-286
 <td>0
 <td>
 <td>
 <td>CCSDVFNQVVK
<tr><td>1207.647
 <td>81-91
 <td>1
 <td>
 <td>
 <td>GWKIGDYAGIK
<tr><td>1136.573
 <td>38946
 <td>0
 <td>
 <td>
 <td>GVIFYESHGK
<tr><td>1071.641
 <td>224-233
 <td>1
 <td>
 <td>
 <td>EKDIVGAVLK
<tr><td>1017.551
 <td>332-340
 <td>1
 <td>
 <td>
 <td>MEKGQIVGR
<tr><td>1013.599
 <td>30-38
 <td>0
 <td>
 <td>
 <td>ANELLINVK
<tr><td>968.484
 <td>303-310
 <td>0
 <td>
 <td>
 <td>EALDFFAR
<tr><td>944.505
 <td>197-206
 <td>0
 <td>
 <td>
 <td>VLGIDGGEGK
<tr><td>893.545
 <td>22-29
 <td>0
 <td>
 <td>
 <td>DIPVPKPK
<tr><td>844.441
 <td>1-7
 <td>0
 <td>
 <td>acetylation: 1
 <td><b>S</b>IPETQK
<tr><td>841.551
 <td>311-318
 <td>1
 <td>
 <td>
 <td>GLVKSPIK
<tr><td>836.451
 <td>84-91
 <td>0
 <td>
 <td>
 <td>IGDYAGIK
<tr><td>814.503
 <td>226-233
 <td>0
 <td>
 <td>
 <td>DIVGAVLK
<tr><td>811.420
 <td>341-347
 <td>0
 <td>
 <td>
 <td>YVVDTSK
<tr><td>802.431
 <td>1-7
 <td>0
 <td>
 <td>
 <td>SIPETQK
<tr><td colspan="6" class="left">
<a href="http://au.expasy.org/cgi-bin/sprot-ft-details.pl?ADH1_YEAST@CHAIN=@1@347">Chain
  Alcohol dehydrogenase 1 at positions 1 - 347 [Theoretical pI: 6.26 / Mw
  (average mass): 36691.94 / Mw (monoisotopic mass): 36668.63]</a>
</table>
</div> <!-- class=contentblock -->

<div class="contentblock">
<h3 id="bsapeptides">Bovine Serum Albumin</h3>

<table class="pepmasses">
<caption>BSA Peptides (Monoisotopic)</caption>
<tr><th>mass <th>position<th>#MC<th>artif. modification(s)<th>peptide sequence
<tr><td>500.2463<td>25-28 <td>0<td><td class="left">DTHK
<tr><td>508.2514<td>229-232<td>0<td><td class="left">FGER
<tr><td>509.3194<td>558-561<td>0<td><td class="left">HKPK
<tr><td>516.3504<td>545-548<td>1<td><td class="left">QIKK
<tr><td>517.298 <td>281-285<td>0<td><td class="left">ADLAK
<tr><td>537.282 <td>157-160<td>0<td><td class="left">FWGK
<tr><td>545.3405<td>101-105<td>0<td><td class="left">VASLR
<tr><td>567.3249<td>434-437<td>1<td><td class="left">YTRK
<tr><td>609.2878<td>524-528<td>0<td><td class="left">AFDEK
<tr><td>649.3338<td>205-209<td>0<td><td class="left">IETMR
<tr><td>649.3338<td>223-228<td>0<td><td class="left">CASIQK
<tr><td>658.3155<td>118-122<td>0<td><td class="left">QEPER
<tr><td>660.3563<td>490-495<td>0<td><td class="left">TPVSEK
<tr><td>689.3729<td>236-241<td>0<td><td class="left">AWSVAR
<tr><td>701.4014<td>198-204<td>0<td><td class="left">GACLLPK
<tr><td>703.4097<td>212-218<td>0<td><td class="left">VLASSAR
<tr><td>706.3552<td>223-228<td>0<td>Cys_CAM: 223<td class="left">CASIQK
<tr><td>712.3736<td>  29-34<td>0<td><td class="left">SEIAHR
<tr><td>725.2593<td>581-587<td>0<td><td class="left">CCAADDK
<tr><td>752.3573<td>341-346<td>0<td><td class="left">NYQEAK
<tr><td>758.4229<td>198-204<td>0<td>Cys_CAM: 200<td class="left">GACLLPK
<tr><td>789.4716<td>257-263<td>0<td><td class="left">LVTDLTK
<tr><td>818.4254<td>562-568<td>0<td><td class="left">ATEEQLK
<tr><td>839.3022<td>581-587<td>0<td>Cys_CAM: 581, 582<td class="left">CCAADDK
<tr><td>841.46 <td>483-489<td>0<td><td class="left">LCVLHEK
<tr><td>886.4152<td>131-138<td>0<td><td class="left">DDSPDLPK
<tr><td>898.4815<td>483-489<td>0<td>Cys_CAM: 484<td class="left">LCVLHEK
<tr><td>922.488 <td>249-256<td>0<td><td class="left">AEFVEVTK
<tr><td>927.4934<td>161-167<td>0<td><td class="left">YLYEIAR
<tr><td>974.4577<td> 37-44 <td>0<td><td class="left">DLGEEHFK
<tr><td>977.4509<td>123-130<td>0<td><td class="left">NECFLSHK
<tr><td>1002.583<td>598-607<td>0<td><td class="left">LVVSTQTALA
<tr><td>1011.42 <td>413-420<td>0<td><td class="left">QNCDQFEK
<tr><td>1014.6193<td>549-557<td>0<td><td class="left">QTALVELLK
<tr><td>1015.4877<td>310-318<td>0<td><td class="left">SHCIAEVEK
<tr><td>1024.455<td>499-507<td>0<td><td class="left">CCTESLVNR
<tr><td>1034.4724<td>123-130<td>0<td>Cys_CAM: 125<td class="left">NECFLSHK
<tr><td>1050.4924<td>588-597<td>0<td><td class="left">EACFAVEGPK
<tr><td>1052.4499<td>460-468<td>0<td><td class="left">CCTKPESER
<tr><td>1068.4415<td>413-420<td>0<td>Cys_CAM: 415<td class="left">QNCDQFEK
<tr><td>1072.5091<td>310-318<td>0<td>Cys_CAM: 312<td class="left">SHCIAEVEK
<tr><td>1107.5139<td>588-597<td>0<td>Cys_CAM: 590<td class="left">EACFAVEGPK
<tr><td>1138.4979<td>499-507<td>0<td>Cys_CAM: 499, 500<td class="left">CCTESLVNR
<tr><td>1142.7143<td>548-557<td>1<td><td class="left">KQTALVELLK
<tr><td>1145.6425<td>236-245<td>1<td><td class="left">AWSVARLSQK
<tr><td>1153.6939<td>257-266<td>1<td><td class="left">LVTDLTKVHK
<tr><td>1163.6306<td>66-75  <td>0<td><td class="left">LVNELTEFAK
<tr><td>1166.4928<td>460-468<td>0<td>Cys_CAM: 460, 461<td class="left">CCTKPESER
<tr><td>1177.5591<td>300-309<td>0<td><td class="left">ECCDKPLLEK
<tr><td>1193.6021<td>25-34vv<td>1<td><td class="left">DTHKSEIAHR
<tr><td>1197.5568<td>337-346<td>1<td><td class="left">DVCKNYQEAK
<tr><td>1249.6211<td>35-44  <td>1<td><td class="left">FKDLGEEHFK
<tr><td>1254.5783<td>337-346<td>1<td>Cys_CAM: 339<td class="left">DVCKNYQEAK
<tr><td>1283.7106<td>361-371<td>0<td><td class="left">HPEYAVSVLLR
<tr><td>1291.602 <td>300-309<td>0<td>Cys_CAM: 301, 302<td class="left">ECCDKPLLEK
<tr><td>1305.7161<td>402-412<td>0<td><td class="left">HLVDEPQNLIK
<tr><td>1349.546 <td>76-88  <td>0<td><td class="left">TCVADESHAGCEK
<tr><td>1352.6661<td>496-507<td>1<td><td class="left">VTKCCTESLVNR
<tr><td>1362.6722<td>89-100 <td>0<td><td class="left">SLHTLFGDELCK
<tr><td>1364.4803<td>106-117<td>0<td><td class="left">ETYGDMADCCEK
<tr><td>1386.6206<td>286-297<td>0<td><td class="left">YICDNQDTISSK
<tr><td>1388.5708<td>375-386<td>0<td><td class="left">EYEATLEECCAK
<tr><td>1399.6926<td>569-580<td>0<td><td class="left">TVMENFVAFVDK
<tr><td>1418.7381<td>298-309<td>1<td><td class="left">LKECCDKPLLEK
<tr><td>1419.6936<td>89-100 <td>0<td>Cys_CAM: 99<td class="left">SLHTLFGDELCK
<tr><td>1439.8117<td>360-371<td>1<td><td class="left">RHPEYAVSVLLR
<tr><td>1443.642<td>286-297 <td>0<td>Cys_CAM: 288<td class="left">YICDNQDTISSK
<tr><td>1463.5889<td>76-88  <td>0<td>Cys_CAM: 77, 86<td class="left">TCVADESHAGCEK
<tr><td>1478.5232<td>106-117<td>0<td>Cys_CAM: 114, 115<td class="left">ETYGDMADCCEK
<tr><td>1479.7954<td>421-433<td>0<td><td class="left">LGEYGFQNALIVR
<tr><td>1497.6314<td>387-399<td>0<td><td class="left">DDPHACYSTVFDK
<tr><td>1502.6137<td>375-386<td>0<td>Cys_CAM: 383, 384<td class="left">EYEATLEECCAK
<tr><td>1511.8427<td>438-451<td>0<td><td class="left">VPQVSTPTLVEVSR
<tr><td>1519.7461<td>139-151<td>0<td><td class="left">LKPDPNTLCDEFK
<tr><td>1532.7811<td>298-309<td>1<td>Cys_CAM: 301, 302<td class="left">LKECCDKPLLEK
<tr><td>1539.8199<td>483-495<td>1<td>Cys_CAM: 484<td class="left">LCVLHEKTPVSEK
<tr><td>1554.6529<td>387-399<td>0<td>Cys_CAM: 392<td class="left">DDPHACYSTVFDK
<tr><td>1567.7427<td>347-359<td>0<td><td class="left">DAFLGSFLYEYSR
<tr><td>1576.7675<td>139-151<td>0<td>Cys_CAM: 147<td class="left">LKPDPNTLCDEFK
<tr><td>1578.5981<td>267-280<td>0<td><td class="left">ECCHGDLLECADDR
<tr><td>1633.6621<td>184-197<td>0<td><td class="left">YNGVFQECCQAEDK
<tr><td>1667.8131<td>469-482<td>0<td><td class="left">MPCTEDYLSLILNR
<tr><td>1700.7869<td>372-386<td>1<td><td class="left">LAKEYEATLEECCAK
<tr><td>1723.8438<td>347-360<td>1<td><td class="left">DAFLGSFLYEYSRR
<tr><td>1724.8346<td>469-482<td>0<td>Cys_CAM: 471<td class="left">MPCTEDYLSLILNR
<tr><td>1738.8105<td>387-401<td>1<td><td class="left">DDPHACYSTVFDKLK
<tr><td>1747.705 <td>184-197<td>0<td>Cys_CAM: 191, 192<td class="left">YNGVFQECCQAEDK
<tr><td>1749.6625<td>267-280<td>0<td>Cys_CAM: 268, 269, 276<td class="left">ECCHGDLLECADDR
<tr><td>1756.7339<td>581-597<td>1<td><td class="left">CCAADDKEACFAVEGPK
<tr><td>1823.8996<td>508-523<td>0<td><td class="left">RPCFSALTPDETYVPK
<tr><td>1850.9107<td>529-544<td>0<td><td class="left">LFTFHADICTLPDTEK
<tr><td>1880.9211<td>508-523<td>0<td>Cys_CAM: 510<td class="left">RPCFSALTPDETYVPK
<tr><td>1888.9268<td>169-183<td>0<td><td class="left">HPYFYAPELLYYANK
<tr><td>1901.8698<td>123-138<td>1<td>Cys_CAM: 125<td class="left">NECFLSHKDDSPDLPK
<tr><td>1907.9207<td>529-544<td>0<td>Cys_CAM: 537<td class="left">LFTFHADICTLPDTEK
<tr><td>1927.7982<td>581-597<td>1<td>Cys_CAM: 581, 582, 590<td class="left">CCAADDKEACFAVEGPK
<tr><td>1942.8204<td>264-280<td>1<td><td class="left">VHKECCHGDLLECADDR
<tr><td>1955.9596<td>319-336<td>0<td><td class="left">DAIPENLPPLTADFAEDK
<tr><td>2045.0279<td>168-183<td>1<td><td class="left">RHPYFYAPELLYYANK
<tr><td>2076.8783<td>267-285<td>1<td><td class="left">ECCHGDLLECADDRADLAK
<tr><td>2105.934 <td>569-587<td>1<td><td class="left">TVMENFVAFVDKCCAADDK
<tr><td>2247.9427<td>267-285<td>1<td>Cys_CAM: 268, 269, 276<td class="left">ECCHGDLLECADDRADLAK
<tr><td>2435.2542<td>45-65  <td>0<td><td class="left">GLVLIAFSQYLQQCPFDEHV K
<tr><td>2458.1806<td>319-340<td>1<td>Cys_CAM: 339<td class="left">DAIPENLPPLTADFAEDKDV CK
<tr><td>2492.2642<td>45-65  <td>0<td>Cys_CAM: 58<td class="left">GLVLIAFSQYLQQCPFDEHV K
</table>

<h3 id="keratins">Keratins</h3>

<table class="pepmasses">
<caption>Human Keratin P04264</caption>
<tr><th>Peptide Mass <th>[M+H]<sup>+</sup> <th>Sequence        <th>Position
<tr><td>1178.593     <td>1179.601          <td>YEELQITAGR      <td>376-385
<tr><td>1276.703     <td>1277.711          <td>LALDLEIATYR     <td>472-482
<tr><td>1382.683     <td>1383.691          <td>SLNNQFASFIDK    <td>185-196
<tr><td>1392.725     <td>1393.733          <td>TNAENEFVTIKK    <td>277-288
<tr><td>1474.778     <td>1475.786          <td>FLEQQNQVLQTK    <td>199-210
<tr><td>1522.781     <td>1523.789          <td>LLRDYQELMNTK    <td>460-471
<tr><td>1715.844     <td>1716.852          <td>QISNLQQSISDAEQR <td>417-431
<tr><td>1880.963     <td>1881.971          <td>EQIKSLNNQFASFIDK<td>181-196
<tr><td>1992.969     <td>1993.977          <td>THNLEPYFESFINNLR<td>223-238
<tr><td>2149.071     <td>2150.078          <td>THNLEPYFESFINNLRR<td>223-239
<tr><td>2226.844     <td>2227.851  <td>GGGGGYGSGGSSYGSGGGSYGSGGGGGG <td>518-548
</table>

<table class="pepmasses">
<caption>Keratin Contaminants (Including Tryptic Peptides)</caption>
<tr><td> 897.414  <td>1184.5911 <td>1383.6909 <td>2312.1482
<tr><td> 973.5318 <td>1193.6166 <td>1434.7705 <td>2383.9524
<tr><td>1037.5267 <td>1234.6796 <td>1474.7494 <td>2510.1323
<tr><td>1060.5639 <td>1307.6782 <td>1474.7858 <td>2705.1617
<tr><td>1066.4992 <td>1320.5834 <td>1699.8251 <td>2831.1947
<tr><td>1066.5169 <td>1357.7188 <td>1707.7727 <td>3312.3087
<tr><td>1140.5649 <td>1357.6963 <td>1716.8517 <td>
<tr><td>1165.5853 <td>1365.6399 <td>1838.9149 <td>
<tr><td>1179.601  <td>1373.6549 <td>1993.9772 <td>
</table>

<table class="pepmasses">
<caption>Keratins (Monoisotopic)</caption>
<tr><th>Peptide Masses<th>Name <th>Accession Number <th>Source
<tr><td colspan="4" style="background-color:black;
  color:magenta;font:bold 100% Helvetica,Arial,sans-serif;">Do not
  forget to adjust for soft ionization mass changes (H<sup>+</sup>
  addition or loss, Na<sup>+</sup> adduct)
<tr><td>1034<td><td>Q8NEY9<td>hair
<tr><td>1138<td><td>Q14715<td>epidermis
<tr><td>1351<td>UKA2<td>P31144<td>keratina cytes
<tr><td>2087<td>UKA1<td>P31940<td>keratina cytes
<tr><td>3420<td><td>Q9BYS0<td>scalp
<tr><td>6284<td><td>Q8NFU0<td>hair
<tr><td>6826<td>KRA81<td>Q8IUC2<td>scalp
<tr><td>8098<td><td>Q8N120<td>hair
<tr><td>8789<td>KR191<td>Q8IUB9<td>scalp
<tr><td>9069<td><td>Q9BYT5<td>scalp
<tr><td>9109<td><td>Q96S60<td>hair
<tr><td>9288<td>KRA71<td>Q8IUC3<td>scalp
<tr><td>9314<td><td>Q16275<td>hair
<tr><td>9504<td><td>Q9BYP8<td>scalp
<tr><td>9695<td><td>Q9BYR1<td>scalp
<tr><td>9737<td>KR121<td>P59990<td>scalp,root hair
<tr><td>9966<td>KR123<td>P60328<td>root hair
<tr><td>10255<td><td>Q9BYU5<td>scalp
<tr><td>10365<td>KRA33<td>Q9BYR6<td>scalp
<tr><td>10395<td>KRA43<td>Q9BYR4<td>scalp,hair
<tr><td>10407<td>KRA32<td>Q9BYR7<td>scalp
<tr><td>10425<td><td>Q6NTD4<td>hair
<tr><td>10539<td>KRA31<td>Q9BYR8<td>hair,scalp
<tr><td>11433<td>KR124<td>P60329<td>root hair
<tr><td>11488<td>KRA95<td>Q9BYQ1<td>scalp
<tr><td>11571<td>KR413<td>Q9BYU7<td>scalp
<tr><td>11784<td>KRA56<td>Q6L8G9<td>root hair
<tr><td>12027<td>KRA48<td>Q9BYQ9<td>scalp
<tr><td>12207<td>KRA48<td>Q9BYQ9<td>scalp
<tr><td>12860<td><td>Q9NSB0<td>hair
<tr><td>12897<td><td>Q9NSB5<td>hair
<tr><td>13291<td>KR410<td>Q9BUQ7<td>scalp
<tr><td>13480<td>KRA24<td>Q9BYR9<td>scalp
<tr><td>13605<td><td>Q96S67<td>hair
<tr><td>13780<td>KRA24<td>Q9BYR9<td>scalp
<tr><td>14402<td>KRA42<td>Q9BYR5<td>scalp
<tr><td>14516<td><td>Q8IUG0<td>hair
<tr><td>14610<td>KR511<td>Q6L8G4<td>root hair
<tr><td>14689<td>KR122<td>P59991<td>scalp,root hair
<tr><td>15150<td>KRA57<td>Q6L8G8<td>root hair,scalp
<tr><td>16266<td>KRA99<td>Q9BYP9<td>scalp
<tr><td>16271<td>KRA52<td>Q701N4<td>root hair,scalp
<tr><td>16276<td>KRA59<td>P26371<td>root hair
<tr><td>16454<td>KRA94<td>Q9BYQ2<td>scalp
<tr><td>16772<td>KRA98<td>Q9BYQ0<td>scalp
<tr><td>16854<td>KRA93<td>Q9BY03<td>scalp
<tr><td>17085<td>KR111<td>Q8IUCI<td>scalp
<tr><td>17141<td><td>Q9BYS2<td>scalp
<tr><td>17519<td>KRA58<td>75690<td>root hair
<tr><td>17637<td><td>Q6PG45<td>pooled
<tr><td>17732<td><td>Q13092<td>epitel keratin
<tr><td>17984<td>KR510<td>Q6L8G5<td>root hair
<tr><td>18010<td>KRA15<td>Q9BYS1<td>scalp
<tr><td>18023<td>KRA44<td>Q9BYR3<td>scalp
<tr><td>18183<td><td>Q8IUG1<td>scalp
<tr><td>18218<td><td>Q07628<td>hair
<tr><td>18234<td><td>Q07627<td>scalp
<tr><td>18261<td>KRA92<td>Q9BY04<td>scalp
<tr><td>18320<td>KR131<td>Q8IUB9<td>scalp
<tr><td>19828<td>KRA55<td>Q701N2<td>root hair,scalp
<tr><td>19917<td>KRA45<td>Q9BYR2<td>scalp
<tr><td>20572<td>KRA49<td>Q9BYQ8<td>scalp
<tr><td>20666<td>KR415<td>Q9BYQ5<td>scalp
<tr><td>20800<td>KR414<td>Q9BYQ6<td>scalp
<tr><td>21407<td>KR412<td>Q9BQ66<td>scalp,muscle
<tr><td>22106<td>KRA53<td>Q6L8H2<td>root hair,scalp
<tr><td>22404<td>KR103<td>P60369<td>root hair
<tr><td>22491<td>KRA47<td>Q9BYR0<td>scalp
<tr><td>24194<td>KRA51<td>Q6L8H4<td>root hair
<tr><td>25107<td>KR10C<td>P60413<td>root hair
<tr><td>25249<td>KRA54<td>Q6L8H1<td>root hair
<tr><td>25571<td>KR10A<td>P60014<td>root hair, scalp
<tr><td>25616<td>KR102<td>P60368<td>root hair
<tr><td>26318<td>KR108<td>P60410<td>root hair
<tr><td>27567<td>KR105<td>R60370<td>root hair
<tr><td>28664<td>KR101<td>P60331<td>root hair,scalp
<tr><td>29977<td>KR109<td>P60411<td>root hair
<tr><td>30266<td>KR10B<td>P60412<td>root hair
<tr><td>30837<td><td>Q14660<td>epidermis
<tr><td>36781<td>KR106<td>P60371<td>root hair
<tr><td>37372<td>KR107<td>P60409<td>root hair
<tr><td>40429<td>KR104<td>P60372<td>root hair
<tr><td>42468<td>K2C6D<td>P48667<td>skin
<tr><td>43138<td><td>Q6IFU5<td>hair keratin
<tr><td>44689<td>K1H4<td>76011<td>hair
<tr><td>45935<td>K1HA<td>76009<td>hair
<tr><td>45968<td><td>Q6NTB9<td>hair
<tr><td>46214<td>K1HB<td>Q14525<td>hair
<tr><td>47232<td>K1H1<td>Q15323<td>hair
<tr><td>47237<td><td>Q9UE12<td>hair
<tr><td>47597<td>K1H5<td>Q92764<td>hair, scalp
<tr><td>49318<td><td>Q7Z3Z0<td>scalp
<tr><td>49411<td><td>Q8IUT8<td>skin, hair
<tr><td>49747<td>K1M7<td>76014<td>hair
<tr><td>49823<td><td>Q7Z3Y8<td>scalp
<tr><td>50319<td>K1H2<td>Q145532<td>hair
<tr><td>50490<td>K1H8<td>76015<td>hair
<tr><td>50510<td><td>Q6KB66<td>scalp
<tr><td>50595<td><td>Q723Y7<td>scalp
<tr><td>51137<td>K1CP<td>P08779<td>keratinocytes
<tr><td>51490<td>K1CN<td>P02533<td>skin,pancreas,skin
<tr><td>51868<td><td>Q723Y9<td>scalp
<tr><td>52247<td>K1H6<td>76013<td>hair
<tr><td>53501<td>KRHB6<td>43790<td>hair
<tr><td>54195<td><td>R6NT21<td>hair,tissue:pcr rescued clones
<tr><td>54214<td>KRHB3<td>P78385<td>hair
<tr><td>54971<td>KRHB1<td>Q14533<td>scalp
<tr><td>55651<td><td>Q6IFU6<td>hair
<tr><td>55802<td>KRHB5<td>P78386<td>hair
<tr><td>56648<td><td>Q701L7<td>scalp
<tr><td>56683<td>KRHB2<td>Q9NSB4<td>hair
<tr><td>57247<td><td>Q14664<td>foreskin epidermis
<tr><td>57291<td><td>Q96DU2<td>scalp
<tr><td>57295<td>K2C4<td>P19013<td>epitel
<tr><td>57775<td><td>QGPG65<td>skin
<tr><td>57809<td><td>05XKES<td>skin
<tr><td>57835<td><td>Q7Z793<td>skin
<tr><td>57851<td><td>Q86Y45<td>scal
<tr><td>58827<td><td>Q8N175<td>skin
<tr><td>58923<td><td>Q86Y46<td>scalp
<tr><td>59504<td><td>95678<td>hair follicle
<tr><td>59519<td>K1CJ<td>P13645<td>keratino cytes
<tr><td>59868<td>K2C6B<td>P0259<td>skin
<tr><td>59894<td>K2C6E<td>P48668<td>skin
<tr><td>59914<td>K2C6A<td>P02538<td>skin,brain
<tr><td>60069<td>K2C6C<td>P48666<td>skin
<tr><td>61801<td><td>Q72794<td>skin
<tr><td>62447<td>K2C5<td>P13647<td>skin,epidermal cells
<tr><td>64511<td>K2C3<td>P12035<td>cytokeratin
<tr><td>64610<td><td>Q701L6<td>scalp
<tr><td>64849<td>KRHB4<td>Q9NSB2<td>hair
<tr><td>65865<td>K22E<td>P355908<td>thihepidermis
<tr><td>65871<td>K220<td>Q01546<td>epidermal
<tr><td>65886<td>K2C1<td>P04264<td>epidermal
<tr><td>     <td>K1CL<td>Q99456<td>cornea
</table>

<h4>Other Sources/Tables</h4>
<p>These tables from
</div>

<div class="widetable">
<table>
<caption>K2C1_HUMAN: Keratin, Type II Cytoskeletal 1 (K1 skin)</caption>
<tr><th>Position in sequence
 <th>Ionized mass [M+H]<sub>+</sub>
 <th>Theoretical mass
 <th>Sequence
<tr><td>185-196 <td> 1382.68 <td> 1383.69 <td> SLNNQFASFIDK
<tr><td>211-222 <td> 1474.75 <td> 1473.74 <td> WELLQQVDTSTR
<tr><td>417-431 <td> 1716.85 <td> 1715.84 <td> QISNLQQSISDAEQR
<tr><td>223-238 <td> 1993.98 <td> 1992.97 <td> THNLEPYFESFINNLR
<tr><td>518-548 <td> 2383.95 <td> 2382.94 <td> GGGGGGYGSGGSSYGSGGGSYGSGGGGGGGR
<tr><td>549-587 <td> 3312.31 <td> 3311.30 <td> GSYGSGGSSYGSGGGSYGSGGGGGGHGSYGSGSSSGGYR
</table>

<table>
<caption>K2E keratin (Dandruff), 67K type II epidermal</caption>
<tr><th>Position in sequence
 <th>Ionized mass [M+H]<sub>+</sub>
 <th>Theoretical mass
 <th>Sequence
<tr><td>245-253<td>1037.50 <td>1036.49<td>YLDGLTAER
<tr><td>381-390<td>1193.60 <td>1192.59<td>YEELQVTVGR
<tr><td>46-61  <td>1320.60 <td>1319.59<td>HGGGGGGFGGGGFGSR
<tr><td>71-92  <td>1838.90 <td>1837.89<td>GGGFGGGSGFGGGSGFGGGSGFSGGGFGGGGFGGGR
<tr><td>93-128 <td>2831.20 <td>2830.19<td>SISISVAGGGGGFGAAGGFGGR
</table>

<table>
<caption>K.9. keratin 9 (Skin), type I, cytoskeletal - human</caption>
<tr><th>Position in sequence
 <th>Ionized mass [M+H]<sub>+</sub>
 <th>Theoretical mass
 <th>Sequence
<tr><td>233-239 <td>897.40  <td>896.39  <td>MTLDDFR
<tr><td>224-232 <td>1060.60 <td>1059.59 <td>TLLDIDNTR
<tr><td>242-249 <td>1066.50 <td>1065.49 <td>FEMEQNLR
<tr><td>449-471 <td>2510.10 <td>2509.09 <td>EIETYHNLLEGGQEDFESSGAGK
<tr><td>63-94   <td>2705.20 <td>2704.19 <td>GGGSFGYSYGGGSGGGFSASSLGGGFGGGSR
</table>

<table>
<caption>K10 (dandruff) keratin 10, type I, cytoskeletal - human</caption>
<tr><th>Position in sequence
 <th>Ionized mass [M+H]<sub>+</sub>
 <th>Theoretical mass
 <th>Sequence
<tr><td>442-450<td>1165.60<td>1164.59<td>LENEIQTYR
<tr><td>323-333<td>1365.60<td>1364.59<td>SQYEQLAEQNR
<tr><td>166-177<td>1381.64<td>1380.63<td>ALEESNYELEGK
<tr><td>41-59  <td>1707.80<td>1706.79<td>GSLGGGFSSGGFSGGSFSR
<tr><td>423-439<td>2024.94<td>2023.93<td>AETECQNTEYQQLLDIK
<tr><td>208-228<td>2366.26<td>2365.25<td>NQILNLTTDNANILLQIDNAR
</table>
</div>

<div class="contentblock">

<h1 id="services">Proteomics Methods As A Service</h1>

<p>
Below are some laboratories that are university- or privately-based that offer
services related to proteomics, with their fees [some links might no longer work
because they are dated and moved].
<ul>
<li> University of Albany Center for Functional Genomics <a 
href="http://www.albany.edu/genomics/proteomics-fees.html">Proteomics and Mass 
Spectrometry Services</a>
<li>Huck Institutes of Life Science <a 
href="http://www.huck.psu.edu/facilities/proteomics-mass-spectrometry-up">Proteomics 
and Mass Spectrometry Core Facility</a>
<li>Iowa State University Office of Biotechnology <a 
href="http://www.biotech.iastate.edu/service_facilities/proteinrates.pdf">Protein 
Facility</a>
</ul>


<h2>Ankara University Biotechnology Institute</h2>
<p>
Here is an email message I wrote to the Ankara University Biotechnology Institute
office on 20 May 2005 about a list of services that the Institute could offer at the 
time.

<div class="email">
To: Fatih Yesil &lt;yesil@ankara.edu.tr&gt;
Subject: Re: Proteomics unit service list
Date: Fri, 20 May 2005 12:00:33 +0300

Hello Fatih,

We can offer as services following:

1. Sample (Cell/Tissue) Extraction
   currently with respect to device use, we can only do
   ultrasonic extraction, which is suitable for animal tissues
   and bacteria, possibly yeast, and cultured cells of any type

   We will later acquire the ability to do plant tissue extraction
   (by pulverization in liquid nitrogen principally), extensive
   tough/fibrous tissue extraction (with a rotary Polytron or
   Turax homogenizer)

2. Isoelectric Focusing (IEF) using Bio-Rad Proteome Works system

   A very self-contained system, although alternatives to Bio-Rad
   consumables are possible if they are considered overpriced

3. SDS-PAGE in 2nd Dimension

   Shockingly, we have no real ability to do 1D-SDS PAGE at the 
   moment unless pre-cast gels are ordered/used.

4. Gel Imaging and Documentation using Bio-Rad's Proteome Works
   VersaDoc Gel Image Acquisition and the PDQuest Gel Analysis
   System

   This requires extensive expert consultation time at the moment
   because we are less than efficient.  Alternatively the user
   can attempt to use it himself without expert help with possible
   little risk to damage to the system.

5. Gel Sample Excision Using Bio-Rad's Proteome Works Plus Spot
   Cutting System

   Fairly simple to operate, but self-sufficient users will have 
   to be tested to ensure they know what they are doing and do not
   damage the system

6. In-Gel and Solution-Based Sample Trypsin/Protease Digestion Using
   Micromass' MassPREP Liquid Handling System

   Up to 384 samples can be accomodated in a single automated run.

7. MALDI Mass Spectrometry with Peptide Mass Fingerprinting in MS mode 
   analysis

   Up to 96 samples can be accomodated in a single automated run

8. ESI Mass Spectrometry with Peptide Mass Fingerprinting in MS/MS or 
   De Novo Sequencing by MS/MS mode combined with LC separation

   Up to 48 samples can be accomodated in a run

9. Bio-Compatible Chromatography for Protein Purification

   Users should provide their own chromatography columns, but perhaps the
   Institute can provide columns at minimal rent (estimated depreciation
   for column life) with no guarantees with respect to performance.

Each service is separate and independent of the other, and the users may 
wish to choose several services that move samples through a serial 
(chain) of processing in the proteomics analysis workflow.  

Other services will be included in the future hopefully.

Services (hizmetler) will include the following:

 a. consultation (time given for expert advice or information)
 b. equipment use (rental?)
 c. technician/expert time
 d. combinations of (a), (b), and (c)
</div> <!-- class=email -->



<h1 id="literature">Literature</h1>
<p>
None of this information would be possible without the vast abundance of
information out there provided by hardworking researchers and scientists.
<p>
Where the text &ldquo;[Simpson]&rdquo; is seen, this refers to methods
or information taken from:

<p>
Richard J. Simpson<br>
<i>Proteins and Proteomics:  A Laboratory Manual</i><br>
Cold Springs, New York:  Cold Springs Harbor Laboratory Press<br>
c. 2003, <a href="http://www.proteinsandproteomics.org">www.proteinsandproteomics.org</a>

<h2>Reviews</h2>
<p>
<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=pubmed&amp;dopt=Abstract&amp;list_uids=12563305&amp;query_hl=1">Glish
GL</a>,
Vachet RW (2003) <i>Nat. Rev. Drug Disc.</i> <b>2</b>, 141-150.
<span style="font-variant:small-caps;">The Basics of Mass Spectrometry
in The Twenty-First Century</span>.
<p>
Patterson SD, Aebersold R (2003) <i>Nat. Genet. Suppl.</i> <b>33</b>,
311-323.  <span style="font-variant:small-caps;">Proteomics: The First
Decade and Beyond.</span>
<p>
Adam GC, Sorensen EJ, Cravatt BF (2002) <i>Molec. Cell. Proteomics</i>
<b>1</b>, 781-790.   <span style="font-variant:small-caps;">Chemical
Strategies for Functional Proteomics.</span>
<p>
Lilley KS, Razzaq A, Dupree P (2001) <i>Curr. Opin. Chem. Biol.</i>
<b>6</b>, 46-50.  <span style="font-variant:small-caps;">Two-Dimensional
Gel Electrophoresis:  Recent Advances in Sample Preparation, Detection,
and Quantitation.</span>
<p>
Kebarle P (2000) <i>J. Mass Spectrom.</i> <b>35</b>, 804-817.
<span style="font-variant:small-caps;">A brief overview of the present
status of the mechanisms involved in electrospray mass spectrometry
identification technology.</span>
<p>
Corthals GL, Wasinger VC, Hochstrasser DF, Sanchez J-C (2000)
<i>Electrophoresis</i> <b>21</b>, 1104-1115.  <span
style="font-variant:small-caps;">The Dynamic Range
of Protein Expression: The Challenge for Proteomic Research.</span>
<p>
G&ouml;rg A, Weiss W, Dunn MJ (2004) <i>Proteomics</i> <b>4</b>, 3665-3685.
<span style="font-variant:small-caps;">Current two-dimensional
electrophoresis technology for proteomics.</span>
<p>
Causier B (2004) <i>Mass Spec. Rev.</i> <b>23</b>, 350-367.
<span style="font-variant:small-caps;">Studying The
Interactome with The Yeast Two-Hybrid System and Mass Spectrometry.</span>

<h2>Research Reports</h2>
<p>
Jiang L, He L, Fountoulakis M (2004) <i>J. Chromatog. A</i> <b>1023</b>, 317-320.
<span style="font-variant:small-caps;">Comparison of protein precipitation
methods for sample preparation prior to proteomic analysis.</span>
<p>
Gygi SP, Corthals GL, Zhang Y, Rochon Y, Aebersold R (2000)
<i>Proc. Natl. Acad. Sci. USA</i> <b>97</b>, 9390-9395.
<span style="font-variant:small-caps;">Evaluation of Two-Dimensional
Gel Electrophoresis-Based Proteome Analysis Technology</span>
<p>
Wu CC, MacCoss MJ, Howell KE, Yates 3rd JR (2003)
<i>Nat. Biotechnol.</i> <b>21</b>, 532-538.
<span style="font-variant:small-caps;">A Method for the
Comprehensive Proteomic Analysis of Membrane Proteins</span>
<p>
Bodnar WM, Blackburn RK, Krise JM, Moseley MA (2003)
<i>J. Am. Soc. Mass Spectrom.</i> <b>14</b>, 971-979.
<span style="font-variant:small-caps;">Exploiting
the Complementary Nature of LC/MALDI/MS/MS and LC/ESI/MS/MS
for Increased Proteome Coverage.</span>
<p>
Mitulovi&#263;, Stingl C, Smoluch M, Swart R, Chervet J-P, Steinmacher I,
Gerner C, Mechtler K (2004) <i>Proteomics</i> <b>4</b>, 2545-2557.
<span style="font-variant:small-caps;">Automated, On-Line
Two-Dimensional Nano Liquid Chromatrograph Tandem Mass Spectrometry
for Rapid Analysis of Complex Protein Digests.</span>
<p>
Rejtar T, Chen H-S, Andreev V, Moskovets E, Karger BL (2004)
<i>Anal. Chem.</i> <b>76</b>, 6017-6028.
<span style="font-variant:small-caps;">Increased Identification
of Peptides by Enhanced Data Processing of High-Resolution
MALDI TOF/TOF Mass Spectra Prior to Database Searching.</span>
<p>
Molloy MP, Brzezinski EE, Hang J, McDowell MT, VanBogelen RA (2003)
<i>Proteomics</i> <b>3</b>, 1912-1919.
<span style="font-variant:small-caps;">Overcoming
technical variation and biological variation in quantitative proteomics.</span>
<p>
Lubman DM, Kachman MT, Wang H, Gong S, Yan F, Hamler RL, O'Neil KA,
Zhu K, Buchanan NS, Barder TJ (2002) <i>J. Chromatog. B</i> <b>782</b>, 183-196.
<span style="font-variant:small-caps;">Two-Dimensional Liquid Separations
Mass Mapping of Proteins from Human Cancer Cell Lysates</span>
<p>
Wolters D, Washburn MP, Yates 3rd JR (2001) <i>Anal. Chem.</i> <b>73</b>,
5683-5690. <span style="font-variant:small-caps;">An Automated
Multidimensional Protein Identification Technology for Shotgun Proteomics.</span>
<p>
<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=pubmed&amp;dopt=Abstract&amp;list_uids=10404161&amp;query_hl=7">Link
AJ</a>,
Eng J, Schieltz DM, Carmack E, Mize GJ, Morris DR,
Garvik BM, Yates JR 3rd (1999) <i>Nat. Biotechnol.</i> <b>17</b>, 676-682.
<span style="font-variant:small-caps;">Direct Analysis of
Protein Complexes Using Mass Spectrometry.</span>
<p>
Puig O, Caspary F, Rigaut G, Rutz B, Bouveret E, Bragado-Nilsson E,
Wilm M, Sraphin B (2001) <i>Methods</i> <b>24</b>, 218-229.
<span style="font-variant:small-caps;">The Tandem Affinity
Purification (TAP) Method: A General Procedure of Protein
Complex Purification.</span>
<p>
Washburn MP, Wolters D, Yates 3rd JR (2001)
<i>Nat. Biotechnol.</i> <b>19</b>, 242-247.
<span style="font-variant:small-caps;">Large-scale analysis of
the yeast proteome by multidimensional protein identification technology.</span>
<p>
M&ouml;ller S, Croning MDR, Apweiler R (2001) <i>Bioinformatics</i>
<b>17</b>, 646-653. <span style="font-variant:small-caps;">Evaluation
of Methods for the Prediction of Membrane-Spanning Regions.</span>
<p>
Wilkins MR, Gasteiger E, Gooley AA, Herbert BR, Molloy MP, Binz P-A,
Ou K, Sanchez J-C, Bairoch A, Williams KL, Hochstrasser DF (1999)
<i>J. Mol. Biol.</i> <b>289</b>, 645-657.
<span style="font-variant:small-caps;">High-Throughput Mass
Spectrometric Discovery of Protein Post-Translational Modifications.</span>
<p>
Laugesen S, Roepstorff P (2003) <i>J. Am. Soc. Mass Spectrom.</i> <b>14</b>,
992-1002.  <span style="font-variant:small-caps;">Combination of Two
Matrices Results in Improved Performance of MALDI MS for Peptide Mass
Mapping and Protein Analysis.</span>

<h2>Monographs</h2>
<p>
Twyman RM.  <i>Principles of Proteomics</i>,
Garland Science/BIOS Scientific Pub., 2004.
<p id="cutler">
Cutler P, <i>ed.</i>, <i>Protein Purification Protocols</i>, Vol 244
in Methods in Molecular Biology&trade; series, Humana Press, 2004.
[<a href="javascript:history.back();">Go back</a>]

</div> <!-- class=contentblock -->


<div id="caminfo" style="display:none;">
<p class="title">
The VersaDoc Camera
<div class="essaybox">
This is a quick guide to the VersaDoc camera.  It might be useful
to look at the &ldquo;<a onclick="ShowPhotoHelp();">Introduction
to Photography</a>&rdquo; to understand the concepts behind the camera
components.
<p>
The camera is a standard digital type with CCD chip as the image collector.
It has the following standard camera components:
<ul>
<li><b>Shutter</b>.  The shutter speed is set through the software.
There is a gel-positioning mode in which the VersaDoc illuminates
reflecting lamps and the shutter is opened once every second to allow
the operator to position, focus, and zoom the gel.
<li><b>Aperture</b>.  The aperture has the usual <i>f</i>-stop settings
for controlling the amount of light.  Depth-of-field is not an issue
for gel imaging.  The <i>f</i>-stop might be set to one or two stops
lower from the <i>f</i>-stop used for gel positioning, particularly for
fluorescence.	See what works best.
<li><b>Zoom</b>.  The focal lengths are marked on the zoom ring, but
the zoom is used during gel positioning to set the proper magnification.
Use the preview mode to fill the image window with as much of the gel
or interesting region of a gel as possible.  Do not rely on image
enlargement routines in software to zoom in on interesting regions if
only they are of interest.
<li><b>Focus</b>.  Once the gel is centered and properly zoomed, adjust
the focus.  Move the focus from one end to the other, and carefully
observe either lines, bands, or spots for the correct sharpness.
</ul>
<h1>Tips</h1>
<ul>
<li>Set the <i>f-stop</i> to the highest value before previewing.
Then lower it to increase the brightness one stop at a time during preview.
<li>Move the zoom to the widest angle (lowest focal length) before
previewing.  A gel with few spots or bands will look completely white
at a low f-stop and with the zoom set in the telephoto range;  this
can be confusing.
<li>Use a gloved hand and place it just above the gel during preview
positioning.  This can often help in setting the proper aperture
(<i>f</i>-stop), zoom, and focus.
<li>The gel should remain hydrated, but avoid putting excess water on
top of the gel.  This can cause reflections that are too bright during
positioning, and it can change the transilluminance of the gel during
the image acquisition.  Pour off excess water, and then work quickly
if there is a concern about gel dehydration/drying.
<li>Image acquisition and processing (editing, spot detection,
normalization, comparisons) are two different phases:  <em>first</em>
get all the images that are wanted, <em>then</em> remove the gel and
place it back in its dish before processing and analyzing the images.
</ul>
</div> <!-- class=contentblock -->
</div> <!-- display=none id=cambox -->


</body>
</html>
