<!DOCTYPE html>
<html lang="en">
<head>
<title>Human Immunodeficiency Virus (HIV) Information</title>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8">
<base href="/proteomics/access/">
<link href="../Abiyotek.css" rel="stylesheet">
<script src="../include/bigimage.js"></script>
<script src="../include/css.js"></script>
<script src="../include/dom-xml.js"></script>
<script src="../include/colorContrasting.js"></script>
<script src="../include/htmlTOCgenerator.js"></script>
<script src="../include/logos.js"></script>
<script src="AbiyotekDocs.js"></script>
<style>
 div.figure {font:normal 70% Century,Bookman,serif;color:navy;
   text-align:justify;}
 img {width:100%;}
</style>
</head>
<body onload="initAbiyotekDoc();">
<p style="float:left;width:10%;margin:0 auto auto 0;">
<a href="."><img src="images/AbiyotekLogo.png"
 alt="company logo"></a>

<!-- hostsite top logo here -->

<!-- logo here -->


<p id="title">
Information on Human&nbsp;Immunodeficiency&nbsp;Viruses&nbsp;(HIV)
<br>and<br>
Acquired&nbsp;Immunodeficiency&nbsp;Syndrome&nbsp;(AIDS)


<!-- putTOChere -->
<div class="outerblock">

<div class="special-note">
These notes are developed by <b>S. M. Halloran</b> for use by Abiyotek biotechnology 
company <img class="no-border" src="images/Abiyoteklogo-inline.png" alt="Abiyotek inline 
logo">.
<p>
Unless a note is made in this textbox that the content for this document is 
complete and in final form, please continually check back and refresh this 
document (press key <b>F5</b>) to look for any updated/added/edited content.
</div>

<hr>
<p>
<b>References</b>
<ul>
<li>RA Weiss, AG Dalgleish, C Loveday, D Pillay (2004) <q>Human Immunodeficiency 
Viruses</q> (Chap 25) in AJ Zuckerman, JE Banatvala, JR Pattison, PD Griffiths, 
BD Schoub, eds., <i>Principles and Practice of Clinical Virology</i>, 
5<sup>th</sup> Ed., (UK:  J. Wiley &amp; Sons Ltd).
</ul>

<hr>

<p>
Retroviruses are single group of RNA viruses making RNA-dependent DNA polymerase
(reverse transcriptase, RT).  The enzyme catalyzes a double-stranded DNA genomic form.
This genome (provirus) can integrate into host chromosomal DNA.  From within the 
chromosome it can replicate more viral genome.  The host's transcriptional system
also can make viral mRNA, which is then translated by the host system to form viral
proteins.  Some of these proteins are then used to transcribe the integrated viral
DNA to segments of RNA that will be packaged into virions (viral capsids) formed
by the other translated proteins.
<p>
Additional features of biology are:
<ul>
<li>The retroviruses can form a diploid RNA genome.
<li>They have the ability to do intermolecular recombination with related viruses.
<li>They can acquire host genes encoding functions able to cause neoplastic transformation
(oncogenes).
<li>Endogenous proviruses are also found in normal cellular DNA:  these are <q>fossil</q>
infections of the germline, indicating prior infection by retroviruses.
</ul>

<p>
Retroviruses have in common at least three genes, and in this order from 
5&prime; to 3&prime;.
<ul>
<li><i><b>gag</b></i>:  encodes precursor polypeptide later processed (cleaved) 
to produce 3 or 4 structural core and matrix proteins
<li><i><b>pol</b></i>:  codes for the reverse transcriptase, a protease, and an integrase
<li><i><b>env</b></i>:  two envelope proteins are formed, a surface (SU) protein and a
transmembrane (TM) protein
</ul>
<p>
Note that a precursor of <i>gag-pol</i> is synthesized, and this is cleaved.
<p>
The mature proteins themselves are typically named by their molecular size:  p24 
and p17 in HIV.  The SU protein in HIV is gp120, or gp70 in the 
&gamma;-retroviruses. The anchored TM protein, which binds to the SU protein, is 
glycosylated in some retroviruses (e.g., gp41 of HIV) and not in others (e.g., 
p15E of &gamma;-retroviruses).
<p>
Containing the viral genome on both its ends are long terminal repeats (LTR 
segments). In addition, the RNA form is polyadenylated as for any eukaryotic 
mRNA.  The LTR segments are used in host chromosome integration of the proviral 
DNA form.  They also contain promoter and enhancer sequences binding both viral 
and host cell proteins and regulating gene expression of the provirus.

<h1>Retrovirus Classification</h1>
<p>
Retroviruses are taxonomically composed of three subfamilies:
<ol>
<li><i>Oncovirinae</i>:  these all have oncogenic potential
<li><i>Lentivirinae</i>:  known as the <q>slow viruses</q> and including HIV and the
Maedi-Visna virus (MVV), which is a prototype, and infects sheep, causing progressive
wasting disease, pneumonia, and central nervous system degeneration
<li><i>Spumavirinae</i>:  known as the <q>foamy viruses</q> and not yet observed to
be pathogenic
</ol>

<p>
<img src="images/hiv/retroviral-phylogenetic-tree.gif" 
    alt="genome maps of HIV-1 and HIV-2">
The oncoviruses are further subdivided into five genera, for a total of seven distinct
generat for all retroviruses.  Classification is based on both genome sequence and
morphology.  The phylogenetic tree is shown in the figure above, and the virus
families shown in the table below.
<p>
Those families with <q>simple</q> genomes possess the <i>gag, pol</i> and <i>env</i>
and perhaps another gene.  The <q>complex</q> genomes possess additional genes
with regulatory function, such as <i>tat</i> and <i>rev</i> in HIV, <i>tax</i> in
HTLV, and accessory genes, such as <i>nef, vif</i> and <i>vpr</i> in HIV.
</div> <!-- class=outerblock -->

<table class="wide-table" style="font-size:95%;">
<caption>Classification of retroviruses of vertebrates</caption>
<tr><th>Genus <th>Example <th>Virion morphology<sup>&dagger;</sup> 	<th>Genome 	 
<tr><td>1. Alpha-retroviruses <td>Rous sarcoma virus<br>Avian leukosis virus
  <td>Central spherical core; C-type 	<td>Simple
<tr><td rowspan="2">2. Beta-retroviruses <td>Murine mammary tumour virus 	
  <td>Eccentric spherical core; B-type <td rowspan="2">Simple
<tr><td>Simian retrovirus type I 	<td>Central spherical core; D-type 		 
<tr><td style="width:12em;">3. Gamma-retroviruses  <td>Murine leukaemia virus
 <td>Central spherical core; C-type <td>Simple 	 
<tr><td>4. Delta-retroviruses  <td style="width:12em;">Human T cell leukaemia virus
  <td>Central spherical core  <td>Complex 	 
<tr><td>5. Epsilon-retroviruses  <td>Fish dermal sarcoma virus
  <td>Central spherical core; C-type <td>Simple 	 
<tr><td>6. Lentiviruses  <td>Human immunodeficiency virus
  <td>Cone-shaped core <td>Complex 	 
<tr><td>7. Spumaviruses  <td>Primate foamy virus
  <td>Central spherical core; pronounced envelope spikes <td>Complex
<tr><td colspan="4" style="font:normal 100% 'Times New Roman','Times Roman',Times,serif;">
<sup>&dagger;</sup> 
Beta-retroviruses and spumaviruses have condensed cores visible in the cytoplasm 
of infected cells, whereas in the other retroviruses the cores condense as 
crescent-shaped bodies during maturation and budding at the cell membrane (see 
Figure 25.5). 	 
</table>

<div class="outerblock">

<p>
HIV types 1 and 2 (HIV-1 and HIV-2) causes human disease known as AIDS.  Prior to its
single naming as HIV in 1986, it was variously called lymphadenopathy virus (LAV),
HTLV-III, or ARV.
<p>
Human T lympotrophic virus types 1 and 2 (HTLV-1 and HTLV-2) are deltaviruses also
causing disease in humans.  HTLV-1 causes neurological disease and adult T cell 
leukemia.
<p>
Primate foamy virus (PFV) is a spumavirus originally detected in cultured 
nasopharyngeal carcinoma of Kenyan patient.
<p>
The human endogenous retroviruses (HERVs) are essentially Mendelian loci which are
<q>fossil</q> infections of the germline, and derive from &gamma; and &beta;
(HERV-K) retroviruses.  They have no origin in lentiviruses or spumaviruses.
The loci represent defective genomes and they cannot be rescued, unlike with BaEV
in baboons and PERV in pigs.  The HERV genomes however express envelope and other
proteins (ERV-3, HERV-W in human placenta; HERV-K in testicular tumors and type I
diabetes; HERV-W  (MSRV) in multiple sclerosis), but none have been identified as
disease-causing.  It is like that 8% of the human DNA genome is composed of fossil
retrovirus genomes.

<h1>Virology</h1>

<h2>Culture &amp; Isolation</h2>
<p>
HIV can be propagated in short-term CD4<sup>+</sup> peripheral blood mononuclear
cells (PMBCs) stimulated by phytohemaglutinin and IL-2.
<p>
SI strains adapt to immortalized CD4<sup>+</sup> T lymphocytes and can become 
chronic producers.  NSI strains need to be propagated in PBMCs, causing cytopathy,
and also in macrophage cultures where they are non-cytopathic.
<p>
Successful viral isolation is detectable by several ways:
<ol>
<li>cytopathic effect (CPE), which includes the observation of syncytia in SI strains
<li>detection of viral Ags in cells using immunofluorescence, enzyme-linked
immunocytology, p24 ELISA either in cells or supernatant of infected cultures
<li>reverse transcriptase assay, either using ELISA or enzyme activity
<li>detection of the viral genome using PCR or Southern blots
</ol>

<h2>Molecular Biology:  Genome and Proteome</h2>
<p>
<img src="images/hiv/HIV-1-and-HIV-2-genome-maps.gif" 
   alt="genome maps of HIV-1 and HIV-2">
<p>
The proviral genomes of HIV-1 and HIV-2 are shown schematically in the figure 
above. The similarities between the maps shows significant identity, except that 
HIV-2 lacks <i>vpu</i> but carries <i>vpx</i>.
<p>
Core and matrix proteins are coded by <i>gag</i>.  A precursor protein (p55)
is translated and then processed by the viral protease (coded by <i>pol</i>)
to produce from the N-terminus to C-terminus as:
<ul>
<li>p17: a matrix protein localized to inner layer of viral envelope and requires
myristoylation, allowing it to be tightly bonded to inner envelope; it has a vital 
function in budding
<li>p24:  a capsid antigen producing the core shell;  it figures heavily in 
immunochemical-based diagnostics
<li>p7
<li>p6
</ul>
<p>
The <i>pol</i> gene expresses three functions:
<ul>
<li>the protease:  processes  the <i>gag</i> product significantly
<li>reverse transcriptase: produces the proviral ds DNA form
<li>integrase:  required to insert the proviral ds DNA genome into host genome
</ul>
<p>
The <i>env</i> gene produces a gp160 precursor, which is then cleaved by host
enzymes (furins) to form envelope glycoproteins gp41 and gp120.
<p>
The <i>tat</i> gene encodes a protein which binds to TAR RNA sequences in the 
5' LTR, and this upregulates viral RNA transcription by complexes with host
cell transcription facts and by cyclin T.
<p>
The <i>rev</i> gene product helps in the export of long HIV transcripts from 
nucleus to cytoplasm by the recognition of Rev-response elements (RREs) in
unspliced or singly-spliced viral mRNA.
<p>
The <i>nef</i> gene provides multiple function in signal transduction and downmodulation
of CD4 cell surface expression
<p>
The <i>vpr</i> provides for transport of newly infected pre-integration complexes 
of HIV proviral DNA into the nucleus for integration (mediated by the integrase)
into the host chromosome.  This protein also arrests cells in G<sub>2</sub> phase
of mitosis, which serves to increase virion production.
<p>
The <i>vif</i> gene product is incorporated into virions and functions to assist
the infection of new target cells by abrogating restriction exerted by host APOBECG3.
SI strains of HIV-1 can be propagated in T cell lines with deletions or non-functional
mutations in <i>nef, vpu</i> or <i>vif</i>, which can adversely affect HIV replication
in PBMCs and macrophages, so that attenuated infection or no replication occurs.
<p>
The functions of <i>vpu</i> are poorly understood.

<h2>Serology/Immunochemical Assays</h2>
<p>
Several commercial ELISA-based assays are now available which are highly specific
and sensitive.  Most assays are configured to detect anti-HIV antibodies using
HIV antigen-coated plates, with detection of enyzme-linked antiglobulin (anti-IgG
and anti-IgM).  Assays now use a mix of recombinant antigens:  gp41, p24, and
synthetic peptides based on immunodominant epitopes.
<p>
Some ELISAs use competitive format:  anti-HIV antibody displaces an enyzme-labeled
standard antibody in test kit to score positive.
<p>
Microparticle agglutination assays with crude inactivated HIV antigens are also used in
assays.
<p>
Blood bank mass screening must combine HIV-1 and -2 tests, and sometimes other viruses.
Differential tests can distinguish the two types of HIV.
<p>
Neutralizing antibodies can be measured by reduction of infectious titer of HIV in
cell culture assays.

<h2>Molecular Diagnostics</h2>
<p>
RT-PCR kits can be used to detect viremia.  In addition, branched DNA (bDNA)  and nucleic
acid sequence-based amplification (NASBA) are also available.
<p>
Alternatively, serial dilution of a biofluid sample containing virus can be used against
a known standard in competitive PCR, or using real-time kinetic methods.
<p>
<img src="images/hiv/HIV-AIDS-progression-time-plots.gif"
  alt="Clinical, virological, and immunological phases of HIV/AIDS">

<h1>Pathogenesis</h1>
<p>
HIV targets CD4<sup>+</sup> cells, since this surface recepter is how it gains 
entry to the cell.  CD4<sup>+</sup> cells include T lymphocytes, monocytes, 
macrophages, dendritic (Langerhans) cells, some brain cells, microglia.  CD4 
receptor interacts with the gp120 envelope protein with high affinity. 
Thereafter, chemokine receptors must gather to complete infection.  NSI strains 
utilize CCR5, a receptor for chemiokins MIP-1&alpha;, MIP-1&beta;, and RANTES. 
SI strains utilize CXCR4 (X4) strains, receptor for stromal-derived factor 1 
(SDF-1).  This explains why some chemokines inhibit HIV-1 infection in vitro. 
Some individuals have mutations in CCR5:  a deletion occurs frequently in 
white populations, and so individuals with homozygous <q>delta 32</q> mutations
are resistant to infection by HIV-1 R5 strains.

<h1>AIDS</h1>
<p>
First described in 1981 in the USA when physicians observed <i>Pneumocystis 
carinii</i> pneumonia and Kaposi's sarcoma in homosexual men, and was thought to
be caused by immunodeficiency, resulting from depletion of CD4<sup>+</sup> T helper
lymphocytes.  It was expanded to include conditions of generalized and extended
lymphadenopathy and also called gay-related immune deficiency (GRID).  CDC noticed
it among injection drug users, sex workers, and blood transfusion and blood
product recipients (particularly hemophiliacs).  Later serological studies indicate
that the virus had spread to American gay men as early as 1977.  Earliest known
sample was found in blood sample from Zaire in 1959.  Molecular clock analyses 
suggest a common origin of HIV-1 (Group M) dating at 1931, and for HIV-2 around
1940.
<p>
Within two years (1983) the cause of AIDS was identified as the virus that would
be called HIV-1; eventually confirmed by other laboratory groups.  HIV-2 was isolated
in 1986.  HIV-1 likely originated as a zoonosis from chimpanzee which has a related
lentivirus SIVcpz.  HIV-2 probably is from the sooty mangabey monkeys of West Africa.
<p>
HIV-1 is divided into 3 groups: main (M), new (N), and outlier (O), and it is possible
they represent three zoonotic transfers from chimpanzee.  Groups N and O are confined
to West Central Africa (Gabon, Cameroon).  Group M has the largest global widespread
incidence causing the AIDS pandemic.  HIV-1 subtypes and clades A-K all belong to 
group M.
<p>
HIV-1 and -2 can be classified by phenotype not relating to any genotypic 
classification. Some HIV-1 subtypes induce in vitro in CD4 cells the formation 
of a syncytium (syncytium-inducing, SI) or they may not do so 
(non-syncytium-inducing, NSI).  Most primary HIV-1 transmitting strains have NSI
phenotype, and the SI phenotype appears as AIDS progresses.  This phenotype classification
is related to cellular tropism for macrophages or T cells to which kind of chemokine
co-receptor is used by the virus to gain cell entry.  NSI strains use the CCR5 
co-receptor (R5 viruses), and SI strains use CXCR4 co-receptor (X4 viruses).
<h2>Epidemiology</h2>
<p>
AIDS high risk groups in western nations in the early years of the epidemic were:
<ul>
<li>gay men actively involved in anal intercourse
<li>men and boys with hemophilia exposed to contaminated clotting factors
<li>blood transfusion recipients
<li>IV drug users
</ul>
<p>
In African nations, virus transmission was largely by heterosexual sex.  Blood
screening tests became available in 1985 and reduced transmission by contamination
blood products.  Safe sex practices have reduced transmission by sexual means.
Antiviral drug treatment has reduced perinatal transmission.  Three in 10 new 
infections in developed countries, and half in developing countries affect women.
An estimated 63 million cases globally have been identified at start of 2004:
of these there have been 21 million deaths assigned to HIV-related AIDS, and 42
million living with HIV since 1981.
<p>
About 8 in 10 new HIV infections among new adult caes are by heterosexual transmission,
and 6% involve homosexual sex, 7% by IV drug use, and another 7% through contaminated
transfused blood or hypodermic needles.  About 10% of HIV infection is in children who
are born with it from infected mothers.  Breast feeding is a route of HIV transmission
and particularly during high viremia in the mother during primary infection.  
<p>
HIV-1 clades A, C, D, and F are frequent in Africa, while subtype b is most common in the
west.  In Thailand, it is recombinant subtype A/E.  HIV-2 has a lower transmission rate
than HIV-1.  It is endemic in west Africa, but HIV-1 has surpassed it in incidence.

</div> <!-- id="outerblock" -->

</body>
</html>
