<!DOCTYPE html>
<html>
<head>
<title>Genomics Methods Reference</title>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8">
<link href="/style/std.css" type="text/css" rel="stylesheet">
<link href="/style/proteomicsRef.css" type="text/css" rel="stylesheet">
<style type="text/css">
  h1 {font-size:150%;margin-left:-1em;}
  h2 {font:bold 125% Tahoma,Arial,sans-serif;margin-left:0;color:green;}
  h3 {font-size:110%;color:purple;margin-left:0.5em;}
  h4 {font-size:100%;margin-left:1em;color:brown;}
  h5 {font-size:90%;margin-left:2em;color:orange;margin-bottom:0;}
  .header-cell {background-color:black;color:yellow;
		font:bold italic 90% Verdana,sans-serif;}
  .description  {background-color:#fdfdfd;text-align:justify;
		font:normal 83% 'Times New Roman',Times,serif;padding:0 2em 0 3em; }
  .enzyme-name {font:bold 100% Tahoma,Arial,sans-serif;color:navy;min-width:12em;}
  .enzyme-description {font:normal 100% Palatino,'Times New Roman',Times,serif;
         padding:0.25em 0.5em;}
  .enzyme-application {background-color:white;color:#050;min-width:12em;
			font:normal 90% 'Arial Narrow',Tahoma,Arial,sans-serif;}
</style>
<script type="text/javascript" src="/include/css.js"></script>
<script type="text/javascript" src="/include/dom-xml.js"></script>
<script type="text/javascript" src="/include/colorContrasting.js"></script>
<script type="text/javascript" src="/include/htmlTOCgenerator.js"></script>
<script type="text/javascript" src="/proteomics/proteomicsDocs.js"></script>
</head>
<body>
<div id="top-panel">
<p id="index-button-left">
<button type="button" onclick="showIndex('asWindow');">Show index in window</button>
<p id="index-button-right">
<button type="button" onclick="showIndex('embedded');">Show index embedded in document</button>
<p id="contact-author" style="display:none;">
<a type="button" href="mailto:mavigozler@yahoo.com?Subject=HtmlToc%20Object%20Not%20Found">Contact
Document Author</a>

<p id="title">
   Genomics Methods Reference
</div>

<div id="scrolling-panel">
<div class="outerblock" id="first-content-block">
Reference is currently organized as follows
<ul>
<li><b><a href="#culture-and-growth">Sources of DNA and RNA:  Culture and
Growth</a></b>.  Discusses the sources of DNA and RNA, and the culture and
growth of model organisms that are bacterial or yeast.
<li><b><a href="#extract-and-purify">Extraction and Purification</a></b>.  It is first necessary to <q>get at</q>, 
that is, to extract DNA and RNA in order to manipulate and use these 
biomolecules. It especially includes notes on how to properly take cautions for 
isolating and purifying, both from cells and from media used in electrophoresis
and chromatography.
<li><b><a href="#characterize-and-purify">Characterizing Properties and 
Quantitating</a></b>.  It includes all methods used to quantitate mass and 
concentration of the polynucleotides.  Discusses the separation of DNA and RNA
using electrophoresis and chromatography, and the use of blots to analyze
fragments, including Southern and Northern analysis.
<li><b><a href="#cloning">Cloning Genes and Making Recombinants</a></b>.
<li><b><a href="#in-vitro">In Vitro Methods.</a></b>.  Includes preparing
and using DNA for in situ hybridization, or performing in vitro transscription.
Methods for perform in vitro translation.
<li><b><a href="#solutions-reagents">Solutions and Reagents</a></b>.  While it seems more logical to actually
put these at the start of a reference, most texts/manuals/textbooks actually
put notes about the making (i.e. recipes) for preparing, using, and storing
solutions in an appendix at the back of manual rather than at its front.
Links (jumps) to a solution or reagent used in a method will hopefully be
provided.
</ul>
<p>
Within each section, details on using viral, prokaryotic, and eukaryotic
(yeast, mammalian) DNA and RNA are discussed.
<p>
This reference is also working on a <a href="Glossaries/OmicsGlossary.html">
glossary</a>, so look here if there is an issue related to terms and their uses.

<!-- ==============================================================
     Sources of DNA and RNA, Cell Culturing
=================================================================== -->

<h1 id="culture-and-growth">Sources of DNA and RNA / The Culture of Model 
Organisms</h1>
<p>
Molecular and cellular biologists analyze the DNA and RNA from many 
biological sources:
<table>
<tr><th>Source   <th>Model organisms
<tr><td>tissues of animals
 <td><i>Homo sapiens</i> (humans), <i>Mus musculus</i> (mouse)
  <i>Rattus norgeicus</i> (rat)
<tr><td>tissue of plants
 <td><i>Arabidopsis thaliana</i>, <i>Lycopersicum esculentum</i> (tomato),
  <i>Zea mays</i> (corn).
<tr><td>bacteria
 <td><i>Escherichia coli</i>
<tr><td>yeast  <td> <i>Saccharomyces cerevisiae</i>, <i>Pichia pastoris</i>
</table>
<p>
The harvest of genomic DNA from an organism is not generally complicated,
because genomic DNA is constant in each and every cell in most cases,
particularly for micro-organisms.  But this is not true in all cases of
all organisms.  For example, it would not be correct to harvest red blood
cells for DNA, since their maturation/development is programmed to eliminate
the DNA.  Likewise, getting DNA from lymphocytes would not be good, since they
undergo DNA rearrangements in order to generate the antibodies they do, and
this involves the loss of the <q>germ line</q> DNA.
<p>
The harvest of RNA requires more thought.  Usually the RNA represents the
<em>phenotype</em>, not the genotype of the cell or tissue or organism (if
that organism exists as a cell).  Thus the RNA content (transcriptome) is
an instant snapshot of what the cell requires in order to function in its
current state.  Thus there is a time component.

<h2>Culture of <i>Escherichia coli</i></h2>
<p>
The growth phases, or kinetics, represented by a curve or plot of the
cell number (or a correlate such as absorbance measurement of turbidity
related to cell number through light scattering) will be factors in determining
the best point for harvesting <i>E coli</i> for DNA or RNA.
<p>

<i>E coli</i> grows best in a rich medium with no growth inhibitory chemicals 
such as antiibiotics.  This is true even when the bacteria have resistance to 
antibiotics.  A good guideline for growing <i>E coli</i> properly can be found 
in <a href="http://www.qiagen.com/literature/qiagennews/0199/991theq.pdf">The 
QIAGEN Guide to Good Microbiological Practice</a>

<h2>Yeast Culture</h2>
<p>
Many yeast species are used either for the study the effects of cloned genes 
(inserts) on their biology or just as a host to maintain or to express the
cloned genes as a recombinant host.  Culture conditions always depend on the
application, although a base (common) medium with essential nutrients is
usual.

<h3>Culture of <i>Saccharomyces cerevisiae</i></h3>
[to be done]

<h3>Culture of <i>Pichia pastoris</i></h3>
<p>
A strain of <i>Pichia pastoris</i> was described in which its entire Golgi
subcellular structure was modified such that it could produce glycoproteins
that had not been glycosylated the way yeast (i.e., <i>Pichia</i>) would
modify the proteins, but rather in a way that humans would glycosylate the
proteins.  Thus the <i>Pichia</i> was <q>humanized.</q>

<h3>Culture of <i>Kluyveromyces lactis</i></h3>

<h2>Mammalian Cell Culture</h2>
<p>
Mammalian cells are ultimately derived from tissue blocks of cells. To extract 
these cells, usually small blocks of tissue, 1 to 5 mm<sup>3</sup>, are cut and 
placed in a rich (not selective, minimal or nutrient-limiting) medium, such as 
RPMI-1640.  Antibiotics are present to prevent the growth of bacteria; 
penicillin-streptomycin are used in combination.  Some times an antifungal 
(amphotericin B, Fungizone) is included if the growth of yeast has been 
observed.
<p>
<b>Primary cell extraction</b>.  
<p>
Information about mammalian cell culture can be found <a 
href="CellCulture.html">here</a>.

<!-- ==============================================================
     DNA and RNA Extraction and Purification
=================================================================== -->
<h1 id="extract-and-purify">Methods to Extract and Purify DNA and RNA</h1>
<p>
In this section, the concern is the extraction and isolation of DNA and RNA,
whether double- or single-stranded from a biological source (tissue, or cells
from a unicellular or multicellular organism) from all other biomolecular classes
that can contaminate/interfere with later methods that will be used to 
<p>
The extraction methods differ considerably depending on what DNA or RNA is
of interest.
<p>
Because extraction is usually strongly connected to purification, methods to
purify DNA and RNA are included in this section.  In quite a few cases, 
purification of DNA will follow a method that used purified DNA but added
materials that require <q>cleaning up</q> of the DNA for a subsequent method.
Thus, this kind of purification will be considered <q>clean up.</q>
<p>
<b>Prokaroytes</b>.  For prokaryotes (bacteria), the DNA and RNA are generally 
mixed together within a cell that has no nucleus.  This is because the biology 
of bacteria is such that transcription and translation are tightly linked, as 
the major regulatory control is at the level of transcription: once the RNA 
polymerase starts making the messenger RNA, the ribosome binds to the RNA and 
finds the translational start, immediately making protein.  This is not always 
true (e.g., regulation can be at the translational level too).  The manner in 
which DNA might be extracted is different.  Extraction is complicated by the 
presence of a tough cell wall that is essential for a single-celled free-living 
organism to survive.  Cell walls may have a thick usually polysaccharide capsule 
in addition to having cross-linked proteinaceous glycans (peptidoglycan). 
Extraction can be highly energetic, in a mechanical way using ultrasonication or 
a French pressure cell with high shearing forces, or it might be more gentle, 
done by chemical means, using a combination of reagents (detergents, and maybe 
pH extremes) and/or enzymes specific to breaking down the cell wall and 
membrane, releasing the intracellular contents.  If genomic DNA (the bacteria's 
usually single circular superhelical chromosome) is desired intact, more gentle
methods must be used since mechanical methods can break the chromosome.  If plasmid
DNA is required intact, mechanical methods might be employed.  Usually a combination
of mechanical and chemical/enzymatic methods are used.
<p>
<b>Unicelluar Eukaryotes</b>.  These are basically yeast.  Like prokaryotes, they
are unicelluar, free-living forms (except for bacterial that are strictly host-requiring
like some pathogens), but they have a nucleus.  Thus the DNA is compartmentalized
within the nucleus.  It contains some RNA within the nucleus too, but this is
usually exported to the cytoplasm.  It is possible to perform a multi-step
extraction in which first the cell wall and plasma membrane of the yeast is
broken, and the cytoplasm is removed while the nucleus remains intact.  Then the
nucleus is separated from the cytoplasmic extract, washed, and then itself 
extracted to obtain the DNA.  But often this is not necessary.
<p>
<b>Multicellular Eukaryotes:  Tissues and Cells</b>.  Selective dissection of 
tissues and organs of an multicellular organism (animal, plant) might be done.
Since the DNA is constant in most tissues that contain DNA, the tissue that 
conveniently provides access to the most abundant and easily extractable genomic
DNA is used (the white blood cell <q>buffy coat</q> in a centrifuged blood sample
from a human, for example).  What is not consistent in all tissues of a multicellular
organism is the transcriptome and proteome:  so that tissue must be carefully 
dissected free of other tissues, the mRNA extracted and purified, and then the
enzymes that generate and amplify cDNA employed for cloning and expression.


<h2 style="text-align:left;">Large- and Small-Scale Phenol Extraction: 
De-Proteinization of DNA and RNA</h2>
<p>
Notes in this section are abstracted from
D. M. Wallace (1987) <span style="font-variant:small-caps">Large- and
Small-Scale Phenol Extractions</span>, <i>Meth. Enzymol.</i> <b>152</b>:
33-41.
<p>
Phenol extraction accomplishes a de-proteinization of the nucleic acid
polymer.  Under the right conditions, phenol is mixed and the phases
separated by centrifugation, with the protein precipitating as a
white flocculent mass at the interface between the bottom organic
phenol layer, and nucleic acid intact in the top aqueous layer.
Optimal conditions for the use of phenol extraction depend upon the
ultimate application of the nucleic acid, and on the nature of the nucleic
acids themselves.
<p>
Add a saturating amount of extraction buffer (30 ml buffer per 100 ml
phenol) to frozen phenol and thaw in a 65&deg; water bath, when the
phenol is to be used.  Phenol oxidizes more rapidly at higher pH, so
with pH values &gt; 8, use fresh phenol always.  Equilibrated phenols
can be stored for 1 month at 4&deg;.  For RNA extraction, always use
phenol made fresh from frozen stock.  8-Hydroxyquinoline is useful
in that it prevents oxidation during storage, its yellow color can be
used to show where the phenol phase is, it can partially inhibit
ribonucleases, and when extracting RNA in vanadyl-ribonucleoside complexes,
it is used as an inhibitor and the 8-hydroxyquinoline reacts with the
VRC to form a black substance; phenol remaining yellow indicates
that all VRC have been extracted.
<p>
Chloroform (CHCl<sub>3</sub>) improves extraction efficiency largely through its 
ability to denature protein, and its high density promotes phase separation. 
Chloroform also removes lipid from a sample. Isoamyl alcohol is added to prevent 
foaming.
<p>
A single extraction with a 24:24:1 (v/v/v) solution of TE-saturated 
phenol:chloroform:isoamyl alcohol equal in volume to the sample is sufficient. 
The DNA in the aqueous phase is then precipitated with ethanol.
<p>
Phenol-chloroform also improves extraction of poly(A)<sup>+</sup> RNA (mRNA) 
compared to methods in which phenol is used alone. Greater yields are possible 
with extraction by buffer-saturated phenol followed by re-extraction of the 
aqueous phase and phase interface with an equal volume of phenol mixed 1:1 with 
chloroform:isoamyl alcohol (96:4).  This reduces losses of RNA at the interface 
occurring because of RNA-protein aggregation.
<p>
The buffer determines what nucleic acid is extracted.  pH 5-6 buffer causes DNA 
to partition toward the organic phase and phase interface, most RNA in the 
aqueous.  Ribonucleases are also less active at the lower pH (an added 
advantage).  A buffer of pH 8 or higher is used to extract DNA.  Both DNA and 
RNA partition to the aqueous phase.  The higher pH also inhibits depurination. 
In using phenol alone for extraction, a pH value below 7.6 can cause 
poly(A)<sup>+</sup> RNA to partition to the organic phase.  In a pH 9 buffer, or 
with pH 5-9 buffer in combination with chloroform or SDS, the 
poly(A)<sup>+</sup> RNA partitions to the aqueous phase.
<p>
RNA is extracted with the presence of ribonuclease inhibitors (in the
past, a high salt concentration of 0.2-0.5 M NaCl was used to reduce
nuclease activity).  Optimal RNA yields are obtained when salt concentrations
are &lt;0.2 M, not a problem because of effective nuclease inhibitors;
the maximum solubility of RNA is about 0.15 M Na<sup>+</sup>.  Use
of K<sup>+</sup> salts should be avoided since SDS is a common additive
and potassium precipitates the detergent.
<p>
Salt presence does not affect DNA extraction efficiency as it affects
that of RNA.  Low salt concentrations (below 50 mM) in the aqueous phase
can reduce DNA yield however, since oligomers will partition to the
phenol phase.  Phenol saturated in TE containing either 50 mM NaCl or
KCl will prevent this.
<p>
Detergents promote dissociation of proteins that bind to nucleic acids,
and SDS is the most commonly used.  Concentrations between 0.1-2.0% are
the most effective.  Temperatures of the extraction should be 10&deg;
or higher to prevent precipitation.  SDS also mildly inhibits nucleases
(an added benefit).  Sodium deoxycholate is also used, particularly
in tissues with more lipid due to rough endoplasmic reticulum (rough ER).
<p>
Because Mg<sup>2+</sup> mediates aggregation of nucleic acids to each other
and to proteins, EDTA is used.  EDTA also prevents Mg(OH)<sub>2</sub>
from precipitating, and so an excess is added when alkaline conditions
for trapping and precipitating DNA are required.
<p>
Proteinase K with a level between 50-200 &micro;g/ml at 37&deg; for 30 min
in the presence of SDS is often used to improve dissocation of proteins.
Pronase is also used.  It also reduces the amount of protein formed at
the phenol-aqueous interphase.
<p>
Nuclease inhibition is paramount in successful extractions of large sized 
nucleic acid.  EDTA is sufficient to inhibit the many metal-dependent 
deoxyribonucleases that would degrade DNA extracted from simple solutions. 
Ribonuclease inhibition requires stronger measures, since many are still active 
despite metal chelation (some are not metal-dependent apparently).  Intact mRNA 
from ribonuclease-rich sources has been obtained using vanadyl-ribonucleoside 
complexes (VRC) and RNasin (placental ribonuclease inhibitor).  Added before 
lysis to a level of 10 mM, VRC is present just before phenol extraction.  VRC 
does not co-purify with RNA and can be used in cell-free translation systems. 
RNasin at 250-1000 U/ml precipitates with other proteins during phenol 
extraction.  Heparin is another inhibitor but can co-purify with RNA.
<p>
Ice-cold buffers and extractions on ice are the best way to minimize actions of 
nucleases.  SDS presence requires temperatures &gt;10&deg;. Phenol extractions 
at 50-60&deg; are used in mRNA preparations where ribonuclease activity is 
already known to be low or non-existent. Quick extractions at room temperature 
provide good recovery of DNA. Timing and vigor of phenol extractions for DNA 
involve vortexing equal volumes of solution with phenol or 
phenol-chloroform-isoamyl alcohol for 1 min in polypropylene (micro)tube, then 
separating the phases in a centrifuge (microfuge) for 1 min.  The aqueous phase 
is carefully removed from the organic with careful use of a micropipettor.  DNA 
with a size &gt; 10,000 bp requires more gentle handling and extraction: 
inversion by hand or use of a rotating wheel for 10-20 min @ 4&deg;. Removal of 
the aqueous phase is with a wide-mouthed pipet.
<p>
Large-scale extractions of RNA are done by shaking the two phases in a 
glass-stoppered bottle for 5 min, stopping occasionally to vent pressure buildup 
(particularly with CHCl<sub>3</sub>), then spinning in 15 or 30 ml Corex tubes, 
or in 250 ml Maxiforce polypropylene bottles, at 4000 rpm for 5 min in 
fixed-angle rotor at 4&deg;.  Obvious white interphases present at the 
<q>final</q> extraction require another phenol-chloroform-isoamyl alcohol 
re-extraction.
<p>
<strong>It is essential that all steps be carried out in a single 
session</strong> as quickly as possible, certainly within one day. Volumes of 
RNA extractions are dependent upon cell/tissue extract volumes and also on the 
type of extract (polysome pellet, whole tissue extract, cytoplasmic fraction). 
The buffer added is usually 10-20 times the volume of biological material.
<p>
When there is undesirable carryover of trace phenol in the aqueous phase, 
water-saturated diethyl ether (saturated by good shaking in sterile water), done 
for 1 min.  The volume of phases is 1:1, with vortexing for microliter volumes, 
and then phases separate with gravity for 5 min in ice.  The upper phase is 
discarded with careful pipetting.  If the sample is not high MW DNA, the whole 
thing can be frozen and ether remains a volatile liquid, which can be removed 
more completely.

<!-- ===========================================================
   Nucleic Acid Precipitation
 =============================================================== -->


<h2 id="pptdna"> Precipitation of Nucleic Acids</h2>
<p style="font-size:83%;">
Many of the methods here are from Sambrook &amp; Russell, <i>Molecular Cloning:
A Laboratory Manual</i> (3rd Ed).
<p>
Before discussion of the extraction methods of DNA and RNA, it is purposeful
to present an important aspect of the chemistry of polynucleotides, which is
purposefully to make them insoluble, that is, precipitating them, for the 
purpose of both purifying and concentrating them.

<h3>Precipitation in Alcohols</h3>
<p>
Standard ethanol (EtOH) precipitation follows these general steps:
<ol>
<li>Estimate the volume of the DNA solution.
<li>The concentration of monovalent cations is adjusted, either by dilution with 
TE pH 8.0 to decrease monovalent cations, or with stock salt solutions to 
increase monovalent cations. See below to see which salt solution is best.
<li>After adjustment of monovalent cation concentrations, add <em>exactly</em> 
two volumes ice-cold ethanol and mix well.  Place the mix on ice or place it in 
a &minus;20&deg; or refrigerator to allow DNA to precipitate.  Usually takes 
15-30 min. If the size of the DNA is small (&lt; 100 nt) or present in low 
concentrations (&lt;0.1 &micro;g/ml), then allow for longer periods:  leave 1 
hour and add Mg<sup>2+</sup> to 10 mM.
<li>Spin out the DNA at 0&deg;.  Maximum speed for 10 min is usually sufficient.
But when concentrations are low and/or fragments are small, spin longer.
<li>Use an aspiration system to remove the supernatant.  If there is concern 
about precipitation, recover the supernatant to new tube using a pipet, in case 
it is needed later.
<li>Fill tube halfway with 70% ethanol, and re-centrifuge at maximum speed
for 2 min at 4&deg;.
<li>Repeat the aspiration process.
<li>Place tube on bench at room temp until all fluid has evaporated.
<li>Dissolve pellet in desired buffer (TE pH 7.6 or pH 8.0) and volume.
</ol>

<p>
In selecting the appropriate salt solution for monovalent cation adjustment, 
consider the following:

<table>
<tr><th>Salt  <th>Stock Solution (M) <th>Final Concentration (M)
 <th>Specific Uses
<tr><td>ammonium acetate <td>10.0  <td>2.0-2.5
 <td>Used to reduce coprecipitation with certain contaminants,
such as dNTPs or sugars.  Best to use when recovering DNA from digestion
of agarose gel with agarase.
<tr><td>lithium chloride <td> 8.0  <td>0.8
    <td>Usually used when high concentrations of ethanol are
to be used in the precipitation of nucleic acid (namely RNA).  Large
RNAs are less soluble than smaller (tRNAs, 5S RNAs) in high ionic strength
LiCl (0.8 M), and can be selectively precipitated.
<tr><td>sodium chloride  <td> 5.0  <td>0.2
    <td>If the sample contains SDS, this salt is used, as the detergent
remains soluble in 70% ethanol.
<tr><td>sodium acetate   <td> 3.0  <td>0.3
    <td>Used pretty much in all other cases which do not specify use of
another salt.
</table>
<p>
The best technique is to re-dissolve the precipitated DNA pellet while damp, as 
they are slow to dissolve if allowed to dry, and smaller fragments can denature. 
If ethanol presence is a problem, incubate the re-dissolved DNA for 2-3 min. in 
a 45&deg; heating block.
<p>
Carriers might need to be used to improve recovery of the desired DNA. These are 
insoluble in ethanol and form precipitates that trap target nucleic acids. Yeast 
tRNA at 10-20 &micro;g/ml is quite often used and is inexpensive, but later 
reactions involving polynucleotide kinase or terminal transferase means the tRNA 
cannot be used since it competes as a substrate for these enzymes.  Glycogen at 
50 &micro;g/ml is useful when combined with 0.5 M ammonium acetate and 
isopropanol and is not a nucleic acid;  one drawback is that it can interfere 
with DNA-protein interactions. Linear polyacrylamide at 10-20 &micro;g/ml is an 
efficient neutral carrier for precipitating picogram amounts of DNA or RNA with 
ethanol, and of proteins with acetone.

<h3>Previously Specified Method</h3>
<p>
<small>
This procedure from D. M. Wallace (1987) <i>Meth. Enzymol.</i> <b>152</b>: 
41-48, which covers general techniques for concentrating nucleic acids because 
it is either too dilute or a solvent change is desired. Included in this section 
are notes from Sambrook et al. (1989), which either update or challenge certain 
method steps.</small>
<p>
Precipitates of DNA or RNA can be effected by mixing a salt of the sample with 
ethanol at a very low temperature (&minus;20&deg;).  The salt to be formed prior 
to precipitation depends on the nature of the sample and intended use.  Samples 
with phosphate or 10 mM EDTA will also precipitate with the sample;  these 
should be dialyzed out before precipitation. For the sole desire of changing 
solvents, consider using drop dialysis, using a Millipore 25 nm porosity filter 
disk (13 mm dia).
<p>
The general steps (1 &micro;g DNA in 20 &micro;l) are:
<ol>
<li>Add 2 &micro;l of 3 M sodium acetate, pH 5.5, for every 20 &micro;l aqueous
DNA sample
<li>If the sample contains SDS that is to be removed, it is optional to
add 0.04 volumes of 5 M NaCl (to make 0.2 M NaCl final) so that the SDS
will remain soluble
<li>Add 40 &micro;l ethanol (two volumes)
<li>Vortex, and immerse in MeOH in dry ice (&minus;70&deg; bath) for 15 min
(powdered dry ice can be used instead of the bath).  The mixture should
freeze or form a slurry
<li>Spin the DNA precipitate in a microfuge at max speed 10 min (in cold
room).  A whitish pellet of DNA should be at bottom.  (Pellets of 10 &micro;g
or more are visible.)
<li>Remove the ethanol supernatant with micropipettor.  Do not disturb
pellet
<li>Add 100 &micro;l of 70% ethanol (kept at &minus;20&deg;) to sample
and vortex.  This should remove trapped solute in precipitate.
<li>Re-spin for 2 min, and remove supernatant.
<li>Dry the DNA 1-2 min in lyophilizer or centrifugal evaporator
<li>Resuspend in TE (10 mM TrisHCl, pH 8; 1 mM disodium EDTA)
</ol>
<p>
Certain modifications have been used for this method.  Instead of 0.3 M sodium 
acetate, the sample can be made 2.5 M in ammonium acetate before ethanol 
addition.  (0.5 vol 7.5 M ammonium acetate, followed by 2.5-3 vol ethanol)  This 
is placed in dry ice 10 min (or in a &minus;70&deg; freezer).  This substitutes 
for sodium salt (possibly useful) and prevents precipitation of dNTPs (removing 
unreacted triphosphates from an RT reaction or DNA polymerase or terminal 
transferase).  Ammonium salts inhibit kinases and terminal transferase, and this 
alternative should not be used if the DNA will be treated by these enzymes.
<p>
Sambrook et al. indicate that ammonium acetate (10.0 M stock to 2.0-2.5 M 
final), lithium chloride (8.0 M stock to 0.8 M final), sodium chloride (5.0 M 
stock to 0.2 M final), and sodium acetate (3.0 M at pH 5.2 to 0.3 M final) can 
be used.  Ammonium acetate is used to reduce co-precipitation of dNTPs, but 
should not be used if the nucleic acid is to be phosphorylated, since ammonium 
inhibits T4 polynucleotide kinase.  When higher concentrations of ethanol are to 
be used in the precipitation, lithium chloride is least likely to precipitate 
with the nucleic acid;  do not use when cell-free translation or reverse 
transcription is to be done, since chloride inhibits protein synthesis 
initiation and RTases.  If the sample contains SDS, sodium chloride is best 
used, since SDS remains soluble in 70% ethanol.
<p>
Li<sup>+</sup> is also used in place of Na<sup>+</sup>.  Add 0.1 vol 
sterile-filtered 8 M LiCl and then 2-3 vol EtOH.  LiCl is soluble in EtOH at 
&minus;70&deg;.
<p>
SDS will precipitate unless sodium chloride is added to 0.2 M before EtOH 
addition.  This can be done by adding 0.04 vol 5 M NaCl followed by 2 vol EtOH.
<p>
A 15 min precipitation at &minus;70&deg; for small (1 ml or less) volumes with 
10 &micro;g/ml DNA will give 80% recovery.  Larger volumes are precipitated in 15 
or 30 ml glass Corex tubes at &minus;70&deg; for a longer period for the larger 
scale (at least 30 min for a 30 ml sample).  For very low DNA concentrations (10 
ng/ml), overnight incubation at &minus;20&deg; is effective.
<p>
Centrifugation to pellet DNA will depend upon the concentration and scale 
volume.  Nanogram quantities in 1 ml or less in absence of carrier require 
ultracentrifugation.  Tubes are completely filled (to prevent cracking) and 
placed in bucket of swing-out rotor (0.4 ml tubes in SW41; 1.4 ml tubes in 
SW27).  These are half-filled with ice-cold 20% EtOH to prevent freezing.  Spin 
at 41,000 rpm for 30 min in the SW41 or 27,000 rpm for 60 min in the SW27, at 
&minus;2&deg;.  Pelleted DNA in the microfuge tube is treated as stated, except 
that spinning after the 70% EtOH wash is done in refrigerated system with a 
swing-out (e.g., HB-4 rotor in RC-5B centrifuge, at 11,500 rpm for 15 min at 
&minus;10&deg;).  A co-precipitant, such as 50 &micro;g/ml tRNA, can be 
used/added to help in the recovery of nanogram amounts.  Ultracentrifugation 
must be used when a co-precipitant cannot be (such as with DNA to be 
phosphorylated by polynucleotide kinase). With picogram amounts of DNA, 
coprecipitants (tRNA or purified glycerol) become essential.
<p>
All these methods are useful for DNA which is at least 200 bp long. DNA of 
smaller size can be precipitated by adding magnesium chloride (MgCl<sub>2</sub>) 
to 10 mM before the EtOH pptn.
<p>
The nucleic acid should be stored at 4&deg; in pH 8 TE buffer.  The EDTA 
chelates ions essential for DNase activity and a pH 8 minimizes deamidation. 
Storage for a longer term can be at &minus;70&deg; without any freeze-thaw 
cycling, or in buoyant CsCl at 4&deg;.

<h3>Precipitating RNA with Ethanol</h3>
<p>
RNA contained in solutions with the following high salt concentrations:
<ul>
<li>0.8 M lithium chloride (LiCl)
<li>5 M ammonium acetate (NH<sub>4</sub>OOCCH<sub>3</sub>)
<li>0.3 M sodium acetate (NaOOCH<sub>3</sub>)
</ul>
<p>
can be precipitated with 2.5 to 3.0 volumes of EtOH.  Use of potassium acetate 
(KOOCCH<sub>3</sub>) is not recommended if the RNA will be treated with SDS 
(potassium makes it insoluble):  for example, SDS is used in chromatography of 
oligo(dT)-columns.  If RNA is to be used in in vitro (cell-free) translation or 
in reverse transcription, avoid LiCl, as it inhibits protein synthesis in many 
cases, and suppresses RNA-dependent DNA polymerase.
<p>
Ensure that all solutions in RNA precipitation are free of RNases.

<h3>Isopropanol Precipitation of DNA</h3>
<p>
Mix one volume of isopropanol (room temperature) with one volume of the DNA
solution.
<p>
This is about half the volume used for EtOH precipitations, and it should be
considered when large volumes of DNA in solution are to be precipitated.
Being less volatile, isopropanol requires a more extensive period for 
evaporation.  Salts are less soluble in isopropanol and will be precipitated
if this is a factor.  More washing is needed to remove these salts.

<h3>Spermine in the Precipitation of DNA</h3>
<p>
Purification of large DNA fragments (&gt; 20 kb) can be done by adding 
polyamines like spermine and spermidine.  Processes like phenol extraction,
vortexing and EtOH precipitation can cause shearing, so the addition of 
spermine to 30 &micro;M and spermidine to 70 &micro;M can enhance recovery
of large fragments without shearing.

<h3>Concentrating DNA with Butanol</h3>
<p>
Several cycles of partitioning with <i>sec</i>-butyl alcohol (isobutanol) or 
<i>n</i>-butyl alcohol (<i>n</i>-butanol) draw off water molecules in the 
organic phase, essentially dehydrating the aqueous phase, and producing a 
concentration of polynucleotides, which remain that phase.
<p>
This is important because if polynucleotides are dilute, they will not 
effectively precipitate in EtOH.
<ol>
<li>Add an equal volume of isobutanol.  Vortex well.  Avoid adding too much 
isobutanol, which can remove all water and cause precipitation of 
polynucleotide; if it occurs, add water to cause its phase to re-appear.
<li>Spin in microcentrifuge for 20 s at room temp (or 1600 <i>g</i> for 60 s for 
a benchtop centrifuge).  Remove and discard upper organic phase.
<li>Repeat above steps until the aqueous phase is reduced to desired volume. 
Note that salt concentration is likely to increase in the aqueous phase, so to 
separate polynucleotides from these increased salts, spun column chromatography 
or EtOH precipitation is certainly necessary.
</ol>

<h3>Concentrating and Diafiltering by Ultrafiltration</h3>
<p>
Ultrafilters can be used both to concentration and diafilter (de-salt) 
polynucleotides. Make sure the ultrafilter cut-off is smaller than the 
polynucleotide size to be retained.

<h3>Precipitating RNA with High Salts</h3>
<p>
Large RNA molecules, such as rRNA and mRNA but not tRNA and 5S RNA (which
are considered small), are not soluble in high ionic strength solutions.
To precipitate them:
<ol>
<li>Add to the sample one-fifth (0.2) of its volume of 8 M LiCl (RNase-free),
mix, and then put on ice for about 2 hours
<li>Spin at 15,000 <i>g</i> for 20 min at 0&deg;.
<li>Resuspend the precipitate in 0.2 volume of water.
<li>Repeat the LiCl incubation and spin steps above one more time.
<li>Precipitate the RNA aqueous solution with 2 vol EtOH.
</ol>

<h3 id="tcadna">Precipitation of Radiolabeled Nucleic Acids
on Glass Fiber Filters with Trichloroacetic Acid (TCA)</h3>
<p>
The method below is from Sambrook, J, Fritsch, EF, Maniatis, T (1989)
   <i>Molecular Cloning, A Laboratory Manual</i>, 2nd Ed., p. E.18.
<p>
This method will precipitate any DNA and RNA molecules more than 50 bases long.
<ol>
<li>Label 2.4 cm-diameter glass fiber (Whatman GF/C) filters with a soft lead 
pencil, and impale the filters on a pin which is itself stuck on a polystyrene 
support.
<li>Spot within the center of each of two filters a known volume (up to 5 
&micro;l) of one sample.  One filter will measure total radioactivity (both 
acid-soluble and acid-precipitable), while the other measures only the 
acid-precipitable.
<li>Let the filters sit at ambient temperature until dry (all solution
has evaporated).  Putting them under a heat lamp might help but is usually
not necessary.
<li>With blunt-end forceps (e.g., Millipore), transfer one of each pair of 
filters to a beaker with 200-300 ml of ice-cold 5% TCA and 20 mM sodium 
pyrophosphate.  Swirl filters for 2 minutes, then transfer to another beaker 
containing fresh 5% TCA/pyrophosphate solution. Repeat washing two more times.
<li>Place washed filters into beaker with 70% EtOH and leave briefly.
Then dry at room temp.
<li>Place each filter into scintillation vial and count.  For <sup>32</sup>P
Cerenkov counting in a <sup>3</sup>H channel is sufficient.  More efficiency
occurs with a toluene-based cocktail, which is required nonetheless
for <sup>3</sup>H, <sup>14</sup>C, <sup>35</sup>S, and so on.
</ol>

<p>
Calculations:
<p class="eqnserif">
incorporated proportion = cpm in washed filter / cpm in unwashed filter

<p class="eqnserif">
total amount of product = incorporated proportion &times; total weight

<p class="eqnserif">
specific activity = incorporated cpm / total amount of product

<p>
See also the <a href="#de81">Adsorption to DE-81 Filter</a> method.

<h3>Non-Precipitation To Remove Organics (Phenol, etc): By Ether Extraction</h3>
<p>
Particularly with high MW (genomic) DNA, it is desirable to avoid methods
that shear it, such as ethanol precipitation.  To extract organic solvent,
ether can be used (followed by concentration in <i>sec</i>-butanol).
<ol>
<li>First water- or buffer-saturated diethyl ether by adding one volume to
one volume of water or TE pH 8.0 in polypropylene tube.  Vortex vigorously
for 10 s.  Use the top phase for extraction.
<li>Add hydrated ether to DNA solution, 1:1.  If shearing is not a concern,
mix by vortexing; else use gentle inversion.
<li>Microfuge 5 s at max speed, or let phases separate by setting tube upright
in rack.
<li>Recover/remove the top ether phase and discard.  Repeat the extraction
one more time.
<li>Allow sample to sit in fume hood for 15 min (for small volumes) or
under put under vacuum for larger volumes.
</ol>

<!-- ===========================================================
   Methods to Extract and Precipitate DNA and RNA
 =============================================================== -->


<h2>Generalized / Standard Methods of Extraction &amp; Precipitation of DNA
and RNA</h2>
<p>
The phenol extraction and EtOH precipitation is most commonly used to isolate
and clean DNA free from lysates in particular, where there are many contaminating
classes of biochemicals (lipids, polysaccharides, proteins, and so on).
The quality of the DNA should be good enough for any method for the DNA to 
be used.  The DNA is (or should be) &le; 1 mg/ml at the start.

<h3 id="std-phenol-xtract">Phenol Extraction and Ethanol (EtOH) Precipitation</h3>
<p>
Describes manipulation for a small volume.
<ol>
<li>To the solution containing DNA in a 1.5 ml conical tube, add an equal
volume of 25:24:1 (v/v/v) buffered phenol/chloroform/isoamyl alcohol.  Use
the appropriate buffer-saturated phenol. The monovalent cation concentration
should be &le; 0.5 M strength.  If the volume &le; 100 &micro;l, dilute with
water or buffer to make the volume easier to work with.  Note that if the 
ionic strength (salts) of aqueous phase too high, it causes inversion of the
layers (i.e., aqueous phase more dense; organic phase usually has yellow color).
<li>Vortex vigorously 10 s, then spin at max speed in microcentrifuge for 15 s
(at room temperature).  If the DNA is quite viscous or there is large amount of
protein, spin another 1-2 min to separate.
<li>Carefully transfer top aqueous layer with P-200 to new tube.  If a white
precipitate was observed at the interface, repeat the organic extraction and
spin.  If the DNA amount is quite small, a back-extraction of the organic phase
with 100 &micro;l TE 8.0, which is pooled with first extract.
<li>Add 0.1 vol of 3 M sodium acetate pH 5.2, mix briefly by vortexing or by
finger-flicking tube several times.  If solution already has high (0.3 to 0.5 M)
sodium chloride or acetate, then this step not necessary, although it is
advisable to try to dilute before phenol extraction.  If DNA concentration is
high (50-100 &micro;g/ml), precipitation occurs at room temperature.  When
EtOH precipitiation is not done to remove traces of phenol or concentrate the
DNA, organic components can be removed with ether; if so, do not add salt.
<li>Add 2 to 2.5 vol (which is the volume after addition of salt!) of ice-cold
absolute ethanol (EtOH).  Mix by vortexing and put in crushed ice for at least
5 min.  Alternatively, the solution can be placed in &minus;70&deg; freezer
for 15 min or longer, or in &minus;20&deg; freezer for at least 30 min.
<li>Spin in microcentrifuge for 5 min at max speed.  If pellet is large, 
usually supernatant can be poured off.  But if small (&lt; 1 &micro;g)
aspirate carefully with Pasteur or standard pipettor.
<li>Add 1 ml of 70% EtOH (room temp).  Invert several times and then spin again 
as in previous step.  However, if the DNA length is short (&lt; 200 bases), use 
95% EtOH for this rinsing step.
<li>Remove the supernatant, taking care to understand that the pellet does not 
adhere to the container as well in aqueous EtOH.  Dry the DNA thoroughly 
(centrifugal evaporator, or dessicator under vacuum).
<li>Rehydrate the DNA in TE pH 8.0, if to be stored indefinitely, otherwise
dissolve in water if to be used immediately.  Add a volume of rehydration 
solution that puts the DNA about 1 mg/ml.
</ol>
<p>
If resuspension is difficult is slow, consider heating briefly (5 min) at 
65&deg;.  For high MW DNA that is not to be sheared (by vortexing), resuspension 
might take one to several days. Gentle dissolution on rocking platform is 
recommended.


<!-- ===========================================================
   gDNA isolation
 =============================================================== -->

<h2>Isolation of Genomic DNA</h2>
<p>
These notes have their source in B. G. Herrmann &amp; A.-M. Frischauf,
<i>Meth. Enzymol.</i> <b>152</b>: 180-183.
<p>
Methods to obtain high MW DNA from a variety of mammalian sources (tissues) are 
widely reported.  Cultured cells are the easiest to work with, and it is no 
longer necessary to isolate nuclei first, since mitochondrial DNA does not 
interfere, and RNA does not inhibit restriction enzyme activity.  DNA should at 
a minimum be 100-200 kb if genomic &lambda; libraries or Southern blots are to 
be done.  Construction of cosmid libraries requires larger DNA.  The following 
is a modified version of the technique of Blin &amp; Stafford (1976).  DNA size 
can be determined in 0.3% agarose at low voltage; pulsed field electrophoresis 
should be used to look at size distribution.

<h3 id="wholeorgan">Whole Organ DNA</h3>
<p>f
This method is appropriate for liver, spleen, or kidney.  Good yields
of pure DNA come from the spleen.  Starve the animal for 24 h to deplete
glycogen prior to sacrifice.  Tissue should be frozen in liq nitrogen
and stored &minus;80&deg; if to be stored for long periods.
<ol>
<li>Get dry ice, precooled mortar and pestle, liquid nitrogen, metal
spoon, 400 ml beaker with TEN9, DNase-free RNase A (0.1 mg/ml)  (prepare
by dissolving DNase-free RNase in water or 0.01 TrisHCl pH 7.5 and
heat to 90&deg; for 10 min, then store frozen and thaw when needed).
Sterilize all tools and rinsed liq nitrogen container with water to
avoid any contamination of DNA.  (Note that TEN9 is a Tris-EDTA-NaCl
buffer at pH 9.0.  The concentrations of Tris vary from lab to lab,
but are typically in the range of 10-50 mM.  EDTA ranges from 1-100 mM,
but many use 10 mM.  NaCl is typically 150 minus the Tris concentration
in millimolar, but is typically 100 mM.)
<li>Pour liq nitrogen into mortar and add frozen tissue pieces.  With pestle
break pieces into smaller bits, then grind to powder each piece (do not add more 
nitrogen).
<li>Chill one end of spoon in nitrogen, and distribute powder on surface of 
buffer in a small beaker (container).  Allow spread on surface, then swirl to 
submerge.
<li>Transfer solution to 50 ml plastic tube and shake to get homogeneous 
solution.  Let tube rock (30 time/min) at RT for 10 min.
<li>Add 1 ml 20% SDS, invert tube a few times, continue rocking 10 min.
<li>Add 1 ml proteinase K (10 mg/ml in water).  Invert and shake tube gently, 
and incubate 37 or 55&deg; ON on rocker.  Solution should be clear after 
digestion.
<li>Solution is transferred to flat-sided glass bottle with tight cap
and 20 ml phenol is added
<li>Let this rock on a platform for 1-3 hr at room temperature
<li>Spin the mixture in 50 ml plastic tube on table top centrifuge for 10 min at 
3000 rpm.
<li>Remove the phenol phase by using a pasteur pipet connected to an aspirator: 
clamp the vacuum tube and dip the pipet so the tip goes to the bottom of the 
tube, wait until viscous thread on end of tip has detached, then slowly apply 
suction to remove phenol.  Close clamp and remove pipet.  The phenol is 
discarded.
<li>Repeat the preceding phenol extraction if not all particles have 
precipitated to the interphase.
<li>With a clear aqueous phase, transfer this to a Corex tube and spin in an 
SS-34 at 9000 for 20 min at 25&deg; (the warmer temperature avoids SDS 
precipitation).  Unwanted suspended particles and protein stick to the glass.
<li>Carefully pour the supernatant to a 50 ml tube, dialyze against TE (1000 
volumes), first at room temperature to pull out SDS, then in the cold room.
<li>Bring the dialyzed solution to 0.3 M with sodium acetate pH 6.5, then add 
0.8 volumes isopropanol.  Invert gently and repeatedly. The DNA is spooled out 
with a glass pipet (its end sealed), and placed into 3 ml TE.
<li>Redissolve the DNA by rotating on a wheel (20 rpm) overnight at room 
temperature.
</ol>

<h3 id="tissuedna">Another Method for Tissue DNA Extraction</h3>
<p>
This is from L. G. Davis, M. D. Dibner, and J. F. Battey,
<i>Basic Methods in Molecular Biology</i>, (Elsevier: New York), 1986,
pp. 47-50.
<p>
This is a two day procedure, the first day taking 6-7 h, the second
taking 1-2 h.  Yields are 1-5 mg DNA for 5 g tissue (for most tissues).
Up to 5 g tissue can be done with this method.  Connective tissue should
be cut out.

<ol>
<li><b>Preparing Tissue</b>.  If it is desirable to prepare nuclei, skip this 
and go to next step.  Tissue should be pulverized in liquid nitrogen with mortar 
and pestle.  The powder is transferred to 50 ml tube after being finely ground. 
For each gram of tissue power processed, add 2 ml of RSB buffer (10 mM TrisHCl 
pH 7.4, 10 mM NaCl, 25 mM EDTA).
<li><b>Preparing Nuclei</b>.  Use this step if the extraction process
is to be for nuclei first.
 <ol>
 <li>The tissue is homogenized in a Dounce appartus in 5% citric acid,
   starting first with a loose-fitting pestle, then a tighter-fitting
   pestle (the pestle should move freely up and down 8-10 times).
 <li>Prewet 10 layers of sterile gauze with the citric acid, and pour
   the homogenate through.
 <li>Spin for 5 min at 2500 &times; <i>g</i>.
 <li>Resuspend pellet in 5% citric acid (about 2 ml/g tissue), doing
   very gently with glass pipet.
 <li>Place about 3 ml/g tissue of 0.88 M sucrose (30%) in 5% citric acid
   in acid-washed 30 ml Corex tube, and overlay the homogenate on it.
 <li>Spin 5000 &times; <i>g</i> for 5 min win swinging bucket at 3500
   rpm (HB-4).
 <li>Remove upper layers by aspiration.
 <li>Resuspend the nuclear pellet in RSB buffer (2 ml per g tissue).
 <li>Spin 2500 &times; <i>g</i> for 2 min.
 <li>Repeat pellet wash in RSB until the pH of suspension is about
   pH 7.4 (test with drop on pH indicator paper).
 </ol>
<li>Add SDS to extract until it is 1%.  The sample becomes viscous at this
  point if the nuclei prep was done.
<li>Proteinase K digestion is done (recommended concentration is 1 mg/ml).
Incubate 1-2 h at 37&deg;.  This is repeated with incubations until
solution is clear.
<li>NaCl (from 5 M stock) is added to bring the solution to 0.4-0.5 M.
<li>An equal volume of buffer-saturated phenol:chloroform:isoamyl alcohol
(10:10:1) is added.  This is mixed gently by inversion (rotating on wheel),
then spun to separate the phases.  Transfer the upper aqueous phase to
a new tube
<li>In a fume hood, extract the aqueous phase with two volumes of
diethyl ether, then aspirate off the ether.
<li>Add 2.5 volumes (making 70%) of absolute EtOH, which precipitates
the DNA.  The DNA can be spooled on a glass rod then transferred to a
new tube.  Excess ethanol can be removed with a sterile cotton swab.
<li><b>Removing RNA</b>.  Perform this step when desirable.
 <ol>
 <li>Dissolve the DNA overnight at 4&deg; in 0.1 &times; SSC
   (2 ml per gram tissue originally used)
 <li>Add DNase-free RNase A to make 20 &micro;g/ml (use 10 mg/ml
    stock, heat at 70&deg; for 10 min to de-activate any DNases).
    Incubate 30 min at 37&deg;.
 <li>Add 5 M NaCl to make 0.4-0.5 M.
 <li>Add buffer-saturated phenol:CHCl<sub>3</sub>:isoamyl alcohol
  (24:24:1) and mix gently by inversion.
 <li>Spin 12,000 &times; <i>g</i> for 10 min
 <li>Transfer upper aqueous phase to a fresh tube.  Add prechilled
  (&minus;20&deg;) ethanol to make 70%.  The DNA is removed with a glass
   pipette, rolling the filament on it.
 <li>???
 </ol>
<li>Dissolve the DNA in TE buffer overnight at 4&deg;.  If the sample
is still too viscous, add more TE buffer. The sample should
be quite stable.  For long term storage, a drop of chloroform is helpful.
</ol>

<h3 id="wholeblood">Fast Method for High Quality Genomic
DNA Extraction from Whole Human Blood</h3>
<p>
A method published by Gustincich et al., <i>Biotechniques</i> <b>11</b>: 
298-302.
<p>
Genomic DNA purification protocols typically require a crude leukocyte 
enrichment, and once postnuclear preparations are obtained, there is long 
proteinase digestions followed by phenol/chloroform extraction and dialysis.
<ol>
<li>0.3 ml EDTA-whole blood is placed in a 2 ml Eppendorf and mixed with
0.6 ml of a <q>blood lysis buffer</q> made of 8% DTAB, 1.5 M NaCl,
100 mM TrisHCl pH 8.6, 50 mM EDTA.
<li>Incubate 5 min at 68&deg;.
<li>0.9 ml chlorform is added, mixed by inversion 2-3 times.
<li>Microfuge at 10,000 rpm for 2 min.  Transfer (pour) upper aqueous phase
to new 2 ml tube.
<li>Add 0.9 ml deionized distilled water.
<li>Add 100 &micro;l of 5% CTAB made in 0.4 M NaCl (final CTAB = 0.3%).
<li>Mix by inversion gently at room temperature.
<li>Spin at 10,000 rpm in microfuge for 2 min.
<li>Resuspend the CTAB pellet in 0.2 ml of 1.2 M NaCl
<li>Precipitate the DNA by adding 0.75 ml ethanol, mix by inversion.
<li>Spin the samples in a microfuge for 10 min at 10,000 rpm at room 
temperature.
<li>Rinse the pellet with 70% ethanol.
<li>Finally resuspend in 0.3 mL in 10 mM TrisHCl pH 7.5 and 1 mM EDTA
within 10 min at 55&deg; (or using lower volume).
</ol>
<p>
Dodecyltrimethylammonium bromide (DTAB) is a strong denaturing cationic
detergent which in high salt (1 M NaCl) in the presence cetyltrimethylammonium
bromide (CTAB) will precipitate DNA.

<h3 id="cultureDNA">Cultured Cell DNA</h3>
<p>
Cells in a confluent monolayer can be extracted directly in the flask, if it 
resists phenol.  Otherwise, detach washed cells, put them in TEN9 buffer, add 
proteinase K (to 0.5 mg/ml) and SDS (to 1%).  Incubate at 37 or 55&deg; 
overnight.  Add phenol (1 volume) and let rock for 1 h at room temperature. 
Transfer to 50 ml tube, and continue as in the whole organ method above.

<h3 id="hmwplantDNA">Rapid Isolation of High Molecular Weight Plant DNA</h3>
<p>
This is title of report of M. G. Murray and W. F. Thompson in
<i>Nucleic Acids Research</i> <b>8</b>: 4321-4325, 1980.
<p>
Plant DNA cloning is complicated by methods that require cell and tissue 
shearing forces that also shred DNA to small size, as well as the presence of 
polysaccharides causing anomalous reassocation kinetics and contaminants 
interfering with restriction endonucleases. DNA and RNA are soluble in CTAB when 
NaCl is 0.7 M, but precipitate when salt is reduced to 0.4 M.  Polysaccharides 
are insoluble in this salt range too.  Polyphenol oxidase activity appears to be 
inhibited.
<p>
Tissues were lyophilized and ground to a fine powder with mortar and pestle and 
small (&lt;1 mm) glass beads.  Liquid nitrogen can be useful for fibrous 
tissues, and mechanical milling can be used for leguminous embryos and other 
hard tissues.  The powder is dispersed in an extraction buffer of 0.7 M NaCl, 1% 
CTAB, 50 mM Tris-HCl (pH 8.0), 10 mM EDTA and 1% 2-mercaptoethanol.  The 
proportion of tissue to buffer depends on species and tissue:  the addition of 
buffer should be sufficient to disperse clumps but keep the solution somewhat 
viscous (1 ml/30-100 mg dry weight of powder).  The mixture is incubated 20-30 
min at 50-60 &deg; with occasional gentle mixing;  CTAB solutions should be kept 
&gt;15&deg; to prevent precipitation.  Chloroform/octanol extraction removes the 
glass beads, cell wall debris, denatured protein, and most polysaccharides. An 
equal volume of chlorform:octanol (24:1) is added to emulsify, spun at 13,100 
&times; <i>g</i> for 10 min, and aqueous phase taken with large bore pipet. 
One-tenth vol of 10% CTAB, 0.7 M NaCl is added, and chloroform/octanol 
extraction repeated.  The final phase may be colored but is clear.
<p>
A CTAB-nucleic acid precipitate forms when NaCl is reduced from 0.70 to 0.35 M 
by adding 1 vol of 1% CTAB, 50 mM Tris-HCl (pH 8.0), 10 mM EDTA. This is left 30 
min at RT.  The pellet is obtained by centrifuging at low speed (2000 <i>g</i> 
for 5 min).  Note that high speed spins may form a pellet difficult to 
re-dissolve.  Re-dissolution can vary with CTAB-nucleic acid precipitates, and 
older CTAB reagents may be mixtures of alkylammonium bromides which form 
complexes requiring homogenization or heating to dissolve.
<p>
The precipitate is dissolved in 2 ml of 1.0 M CsCl (1.12 g/ml), 50 mM Tris-HCl 
(pH 8.0), 10 mM EDTA, and 200 &micro;g/ml ethidium bromide. Heating to 60&deg; 
briefly can speed dissolution.  The solution is moved to an ultracentrifuge 
tube, then 2.4 ml of 6.6 M CsCl (1.82 g/ml) 50 mM Tris-HCl (pH 8.0), 10 mM EDTA 
and 0.1% sarkosyl are added in a way that allows considerable mixing to form a 
crude density gradient. Centrifugation is done (2-3 h at 58,000 rpm in TV-865 
rotor) and DNA is visualized by long wave UV.  The band is removedwith large 
bore syringe needle and moved to fresh centrifuge tube.  The tube is filled with 
4.5 M CsCl (1.55 g/ml), 50 mM Tris-HCl, 10 mM EDTA, 0.1% sarkosyl, and 100 
&micro;g/ml EtBr.  A 6 h to overnight centrifugation at the same speed is done, the 
DNA removed again.
<p>
NaCl-saturated isopropanol is used to partition EtBr from the solution. CsCl 
itself is removed by dialysis against 10 mM sodium acetate. Alternatively DNA 
can be pptd by adding 2 vol water and 7.5 vol ethanol.
<p>
Yields of 20-70 &micro;g DNA per 100 mg dry weight of powder can be obtained, 
varying with species and tissue.  DNA from wheat, peas, oats, carrots, mung 
beans, tobacco, <i>Mimulus</i>, and various species of <i>Atriplex</i> has been 
isolated with this method.  The A<sub>260/230</sub> ratio is 2.2-2.5, and 
A<sub>260/280</sub> is 1.8-1.9.  RNase can be used to remove residual RNA that 
is often present; in general restriction analysis or cloning work is not 
affected by contaminating RNA.  DNA preps co-migrate with intact lambda DNA, 
indicating lengths of 50 kbp or greater.
<p>
Lyophilization of tissue provides an advantage of disrupting tissue and making 
DNA less susceptible to shear.  Hydration in EDTA and detergent minimizes 
nucleolytic degradation.  Also the best storage is of dry tissue over several 
years with little loss of DNA quality. Note that this method might be used to 
isolate total RNA as well, since analysis of RNA on formaldehyde-agarose gels 
shows undegraded RNA.

<h3 id="dneasy">QIAGEN DNeasy Tissue Kit</h3>
<p>
The QIAGEN DNeasy Tissue Kit (50-pak Cat #69504) is a complete kit for 
extracting DNA of sufficient purity to perform many manipulations, from PCR to 
RFLP to Southern blotting.  It is <q>designed for rapid isolation of total 
DNA (e.g., genomic, mitochondrial, viral) from a variety of sample sources 
including fresh or frozen animal tissues and cells, yeasts, or bacteria.</q>

<h4>Principle</h4>
<p>
A silica gel membrane is used to obtain the DNA and requires no organic 
extraction or ethanol precipitation step.  The buffer serves to lyse cells and 
tissues and at the same time prepare the DNA for selective adsorption to the 
membrane.  It takes 20 min to perform the separation once lysis and digestion is 
complete.  The lysate/digestate is passed through the column by centrifugation, 
the membrane is washed twice, and the DNA can then be eluted from the membrane 
in water or buffer. Fragments as small as 100 bp are recovered.  Purity using 
the OD<sub>260</sub>/OD<sub>280</sub> ratio is between 1.7 and 1.9.
<p>
Samples should be fresh or frozen immediately and stored &minus;20&deg; to 
&minus;70&deg; for the best results and yields.  Samples can also be stored 
after Proteinase K digestion in Buffer ATL for up to 6 months at ambient 
temperature without degradation of DNA quality. When isolating DNA from bacteria 
or yeast which accumulate large amounts of metabolites or which have dense cell 
walls, it is best to obtain cells from early log phase of growth.
<p>
Yields are proportional to sample size, from 100 cells to 5 &times; 
10<sup>6</sup> cells.  A maximum loading should be observed.  Spleen and 
polyploid cells have large amounts of DNA.  For animal tissues, 25 mg is a 
general maximum (use half of that if unsure).  In general 2 mm<sup>3</sup> 
tissue weighs about 10-15 mg.  For mouse tails, 6-12 mm tail is recommended. 
For rat tail, 6 mm.  For cultured cells, 5 &times; 10<sup>6</sup> cells.
<p>
The ready-to-use Proteinase K is supplied at 20 mg/ml (600 mAU/ml soln or 40 
mAU/mg protein).  Buffer ATL is for tissue lysis.  Although no mechanical 
homogenization is used, digestion and lysis will proceed more quickly if the 
tissue is minced as much as possible.  Lysis time can be reduced by 
pulverization using liquid nitrogen prior to addition of Buffer ATL and 
proteinase K.
<p>
RNA copurifies with the DNA and usually does not interfere with reactions 
intended for use with purified DNA.  If an RNA-free sample is desired, add RNase 
A to sample prior to adding Buffer AL.  The RNase should be DNase-free.  Use 20 
&micro;l of a 100 mg/ml solution.
<p>
DNA elutes from the DNeasy mini column using either Buffer AE or water. Two 
successive 200 &micro;l aliquots of Buffer AE are used to elute the nucleic acid. 
The number of 200 &micro;l aliquots is actually dependent on the amount of DNA 
bound.  One 200 &micro;l aliquot is good for up to 10 &micro;g DNA bound.  For 
greater, a second aliquot is useful.  60-80% of DNA comes out in first elution. 
When > 30 &micro;g is bound, at least three 200 &micro;l elutions are necessary.  A 
rough guide for yield is 0.2 to 1.2 &micro;g DNA per mg tissue for animal tissue. 
For mammalian cells, it is 6 &micro;g DNA per 10<sup>6</sup> cells.

<h3 id="nokit">No-Kit Method</h3>
<p>
We can also do this without resorting to use of commercial kits at all.
The method [9] described below is actually intended for the purification
from tissue pieces of high molecular weight DNA to be used in
the making of genomic libraries and cosmids, which require high MW DNA.
It is probable that this method is the basis for the more refined methods
used in the commercial kits.
<p>
There is some sacrifice (loss) of <q>clotted</q> DNA during the process:  the 
importance of the procedure is to get quality DNA absent of protein or other 
interfering substances that is representative of the genomic DNA, not all the 
DNA in the tissue. This method would necessarily have to be modified if 
quantitative recovery of all DNA were desirable, since the quality of the DNA as 
representative of the genome is unimportant here. One useful modification would 
be to prevent DNA <q>clots</q> from forming, perhaps as result of insolubility 
of particularly large fragments of DNA.  Large fragments might also account for 
the very significant viscosity of the solution.  Hence a DNA endonculease 
(restriction enzyme?) that produces random cuts/nicks in the strands and reduces 
their length might help, since this probably makes the DNA more soluble.

<ol>
<li>Block of tissue, if large enough, is immersed in liquid nitrogen in
   a stainless steel cup of Waring Blendor, run at top speed until
   pulverized (made into fine powder).  After allowing nitrogen to evaporate
   the powder is taken up in 10 vol extraction buffer:
   <br><b>Extraction Buffer</b>
  <ul>
  <li>10 mM TrisHCl, pH 8.0
  <li>0.1 M EDTA
  <li>20 &micro;g/mL pancreatic RNase
  <li>0.5% SDS
  </ul>
  After making certain powder is in solution, this is put into 50 ml
  spin tube and set at 37&deg; for 60 min.
<li>Proteinase K is added to final 100 &micro;g/mL concentration.  Mix
the viscous solution using glass rod.  Proteinase K usually available
as 20 mG/mL in water stock.
<li>Put in 50&deg; water bath for 3 h, and swirl viscous soln periodically
<li>Cool to room temp; transfer to another spin tube, if necessary. Add
   equal volume of 0.5 M TrisHCl, pH 8.0-saturated (equilibrated) phenol.
   <em>Gently</em> mix two phases by <em>slow</em> turning of tube end
   over end for 10 min.
  <br>The two phases should form an emulsion.  If not, place on roller
     apparatus for 1 h.
<li>Separate two phases at 5000 &times; g for 15 min at room temp.
  <br><b>Note</b> DNA can become trapped at aqueous-organic interface
  if pH of buffered phenol is not 8.0.
<li>Use wide-bore (3 mm orifice) pipet to recover aqueous phase
    to clean tube.  It is essential that the material at the interface
    not be disturbed or transferred.  The aqueous solution may be very
    viscous (the reason for using a wide-bore pipet):  if this is
    the case, it may be better to use a water-suction vacuum to draw
    away the organic phase.  This can be done by hooking a pipet tip
    with small orifice to a trap, the trap then hooked to the vacuum hose,
    which is connected to a stopcock connected to a hose leading to
    the vacuum fitting.  Immerse the tip into the phenol and ensure
    that any thread of aqueous material detaches, then carefully
    open the stopcock to draw out the phenol and close it immediately
    before the interface is drawn in.  Spin the tube again (5000 &times;
    g for 20 min RT) to sediment clots, protein.

<li>Repeat the phenol extraction two more times.
<li>Pool aqueous phases (from back extractions?) and add 0.2 vol of
   10 M ammonium acetate.  Add 2 vol ethanol at room temp.
   Swirl tube until thoroughly mixed.  The DNA forms a precipitate
   that can be removed with hooked pipet, or spun at 5000 &times; g for
   5 min at room temp.
<li>Wash ppt twice with 70% EtOH.  Remove ethanol as much as possible.
   Allow as much of the ethanol to evaporate as possible without letting
   DNA go completely dry.
<li>Take up DNA in pH 8 Tris/EDTA (1 ml per 5 million cells).  Let
    tube sit on rocking platform to allow complete dissolution (as long
    as 12-24 h).
<li>If you want to check purity, measure at 260 and 280 nm.  The 260/280
    ratio should be &gt;1.75.  An OD<sub>260</sub> of 1.0 is about
    50 &micro;g DNA per ml.
</ol>

<h3 id="de81">Adsorption of Nucleic Acid to DE-81</h3>
<p>
This method is described by Sambrook, J, Fritsch, EF, Maniatis, T (1989)
   <i>Molecular Cloning, A Laboratory Manual</i>, 2nd Ed., p. E.19,
<p>
DE-81 filters are prepared and washed very much like the glass fiber filters
in the <a href="#tcadna">TCA precipitation</a> method, except that the
solution for washing is 0.5 M sodium phosphate, pH 7.0, and not
TCA/sodium pyrophosphate.
<p>
A big difference between DE-81 filters and the TCA/glass fiber filter
method is that the DE-81 filters recover even the small oligonucleotides
not recoverable with TCA precipitation.

<!-- ===========================================================
   plasmid DNA isolation from bacteria
 =============================================================== -->

<h2 id="plasmidisoln">Isolating Plasmid DNA from Bacteria</h2>
<p>
Plasmids can be isolated at different scales, usually small 
(<q>minipreps</q>), medium (<q>midipreps</q>) and large (<q>maxipreps</q>).

<h3 id="miniprep">Miniature/Small-Scale Plasmid Isolation/Preparation</h3>
<p>
Mini-preps are good for getting DNA for electrophoretic analysis or restriction 
digests.  If precipitated with <a href="#peg">PEG</a>, the DNA is good enough as 
a template in sequencing.

<h4 id="alkaline-conical">Alkaline Lysis of Single Colonies in 1.5 Milliliter Conical 
Tubes</h4>
<p>
The alkaline lysis method disrupts double-stranded DNA, but plasmids can 
renature so long as the concentration and duration of exposure to hydroxide is 
kept to a minimum.  Alkaline lysis works well at all scales.  Much of the SDS
will be precipitated because it is insoluble with potassium.

<p>
Prepare the following solutions:
<ul>
<li>Solution 1: 25 mM TrisHCl, pH 8.0, 50 mM glucose, 10 mM EDTA<br>
 Filter-sterilize and store at 2-8&deg;
<li>Solution 2: 0.2 M NaOH, 1% SDS <br>
 Make this fresh.
<li>Solution 3:  3 M potassium + 5 M acetate, pH 5.0-5.5
<br>
 Prepare a 5 M potassium acetate solution, and add 60 ml to 100 ml graduate
 cylinder.  Add 11.5 ml of glacial acetic acid.  Bring it up to 100.0 ml
 with water.  Store at 4&deg; and keep in ice-water just before use.
<li>Solution 4: chloroform : isoamyl alcohol (24:1, v/v)
<li>Solution 5: phenol buffered to equilibrium against 10 mM TrisHCl, pH 8.0
<li>Solution 6: 5.0 M ammonium acetate, pH 5.
</ul>
<p>
Also make sure that the following are available:
<ul>
<li>70% and 95% EtOH
<li>TE (Tris-EDTA and the proper pH)
<li><em>optional</em>: 10 mg/ml DNase-free RNase
<li>LB medium and the selection antibiotic
</ul>
<p>
If refrigerated microcentrifuge is available, set at 4&deg;.

<h5>Protocol</h5>
<ol>
<li>Inoculate bacteria into a tube containing sterile 5 ml LB broth with 
a single colony and containing antibiotic selection.
<li>Grow overnight at the appropriate temperature (usually 37&deg;) shaking 
200-300 rpm until end of log growth phase (until saturation).
<li>Fill up a 1.5 ml conical tube and spin 20 seconds in a microfuge at 4&deg; 
(room temperature is okay too).  Avoid spinning so long that the pellet is too 
tight. Place the rest of culture into refrigerator.
<li>Aspirate the supernatant away (a needle connected to trapped vacuum
line with recovery bottle provides the best control).
<li>Add 100 &micro;l of Solution 1 and resuspend pellet with a vortex and/or
pipettor.  Do not let the pellet sit too long or cell death may occur.
After this step, mixing will <em>not</em> be done by vortexing or shaking
the tube with like force, since shearing of the bacterial chromosome
will contaminate the plasmid.  After resuspending cells, let sit 5 min
at room temperature.
<li>Add 200 &micro;l of alkaline lysis solution (Solution 2).  Mix very gently
3-5 times using inversion.  Place on ice for 5 min.
<li>Add 160 &micro;l Solution 3 (KOAc) and mix gently by inversion.  Place on ice
for 10 min.
<li>Spin in cold microfuge at 4&deg; for 10 min.
<li>Remove 400 &micro;l of supernatant to a new tube and discard the pellet.
<li>Add 200 &micro;l chloroform:isoamyl alcohol (Solution 4) to 200 &micro;l
phenol (Solution 5) and mix well.
<li>Add the phenol:chloroform:isoamyl alcohol solution to the supernatant,
vortex very gently for 30 seconds.
<li>Pulse-spin (about 10 sec).
<li>Transfer the upper aqueous phase to a new tube (do not recover any
white flocculent at the interface between the phases!)
<li>Add 1.0 ml ice-cold (&minus;20&deg;) absolute ethanol, and let set
for 5 min at room temperature.
<li>Spin 4&deg; for 10 min in microfuge.
<li>Aspirate supernatant.  Dissolve pellet in 50 &micro;l autoclaved water,
and vortex to dissolve.
<li>Add 20 &micro;l of Solution 6.  Then add 140 &micro;l ice cold absolute
ethanol, and set at &minus;20&deg; for 10 min.
<li>Spin 10 min at room temperature, aspirate supernatant, and dry the
pellet under a vacuum.
<li>Dissolve the pellet in autoclaved water or Tris-EDTA, pH 7.6
(depending on use).
</ol>
<p>
DNA is good enough for restriction digestion.

<h4 id="alkaline-96well">Alkaline Lysis in 96-Well Plates</h4>
<p>
This procedure allows for far more preps.  Obtain the following materials:
<ul>
<li>TYGPN medium
<li>ice-cold 70% EtOH
<li>isopropanol
<li>96-well round-bottom plates (similar to Dynatech PS plates)
<li>Multichannel pipettor (8-well Costar, or 12-well TiterTek) or 
repetitive pipettor (perhaps autoclavable or sterilizable)
<li>Multitube vortexer
<li>Centrifuge with microplate carrier (Sorvall RT-6000 low speed, H-1000B rotor)
</ul>

<h5>Protocol</h5>
<ol>
<li>Add 300 &micro;l TYGPN medium to each well using multichannel or 
repetitive pipettor.
<li>Inoculate each well with single colony.  Don't overdo it.
<li>Place in 37&deg;incubator.  Takes 48 h, but might be reduced if
placed in shaker incubator.
<li>Spin the plates in centrifuge with microplate carrier for 10 min at 600 
<i>g</i> set at 4&deg;.  To remove supernatant, decant with brief flick.
<li>Resuspend well bottoms by clamping to multitube vortexer and running
20 s at setting 4.
<li>Add 50 &micro;l of the glucose/Tris/EDTA solution for the 1.5 ml conical
tube protocol.
<li>Add 100 &micro;l NaOH/SDS solution from the 1.5 ml protocol.
<li>Add 50 &micro;l KOAc from the 1.5 ml protocol.
<li>Cover plate with tape or parafilm, and agitate strongly in the vortexer
20 s at setting 4.  Spin 5 min at 600 <i>g</i>
<li>Get 96-well plate.  Immerse a pipet tip at the edge of where the
round part of the well starts (not all the way to bottom).  Slowly draw
up 200 &micro;l, transfer to corresponding well of new plate.
<li>To each well of recovered supernatants in new plate, add 150 &micro;l
isopropanol. Cover with plate tape, agitate, and place in &minus;20&deg;
for 30 min.
<li>Spin 25 min at 600<i>g</i> 4&deg;, then decant supernatant.
<li>Wash each pallet with ice-cold 70% EtOH, gently decant supernatant.
It is important to decant alcohol solutions as much as possible without
losing pellets, since NaOH and SDS interfere with restriction digestions.
If pellets become loose, re-spin at maximum speed for 5 min.
<li>Let air dry (about 30 min), then resuspend in 50 &micro;l TE.  
</ol>
<p>
10 &micro;l can be used for digests.

<h4 id="boillys">Boiling Lysis</h4>
<p>
This method will denature the DNA in the process of extracting it from the cell. 
Cells are treated with Triton X-100 and lysozyme to weaken the cell wall, then 
placed in boiling bath.  Denaturing the DNA is not a problem when the plasmid is 
small (&lt; 15 kb), covalently closed circular DNA.  The amount of bacterial 
culture that can be handled ranges from 1 ml minipreps to 250 ml volumes.
<p>
This method is not recommended under the following conditions:
<ul>
<li>for <i>E. coli</i> strains that release large amounts of carbohydrates when 
exposed to detergent, lysozyme, or heat; these can inhibit DNA cutting or 
polymerizing enzymes (strain HB101 is a well known example of a high carb form).
<li>when the bacteria express <i>endonuclease A</i> (<i>endA</i><sup>+</sup>), 
since this is not inactivated completely during boiling.  Use the alkaline lysis 
method.
</ul>
<p>
The following small-scale boiling lysis method produces DNA good enough for 
electrophoresis and restriction digests.  Further purification (PEG) will 
produce DNA of sequencing quality.
<p>
The plasmid DNA is processed as follows:

<ol>
<li>Prepare bacterial cultures at 37&deg; with vigorous shaking
in 2 ml of rich media (LB, YT, Terrific) with appropriate selection.
<li>Transfer 1.5 ml of the culture to microfuge tube and spin 20-30 seconds at 
maximum speed in cold (4&deg;) microfuge.  Store rest of culture at 4&deg;.
<li>Use careful aspiration to remove the supernatant.  Remove as much as 
possible, including wicking all the medium away.  Materials in the medium can 
inhibit restriction endonucleases.
<li>Resuspend <em>quickly</em> (by vortexing briefly) the pellet
in 350 &micro;l <a href="#stet">STET</a>.
<li>Add 25 &micro;l of 10 mg/ml lysozyme.  Vortex briefly (3 sec).
<li>Put tube on ice for 30 s to 10 min.  The variation in time depends on
observations about how much the cell wall weakens.
<li>Place in boiling water bath for <em>exactly</em> 40 seconds (others have 
specified 1-2 min). (If superhelical DNA is heated for too long, it undergoes 
irreversible denaturation.)
<li>Spin at room temperature in a microfuge at maximum speed for 15-30 min.
Supernatant will contain RNA and plasmid DNA, while pellet has all usual
debris and chromosomal DNA
<li>Transfer supernatant carefully&mdash;do not take or stir up pellet&mdash;to
new tube.
<li>Add 40 &micro;l of 2.5 M sodium acetate, pH 5.2, and 420 &micro;l isopropanol.
Vortex to mix, then let stand 5 min at room temp.
<li>Spin at maximum speed for 10 min in a cold microfuge.
<li>If preparing an <i>endA</i><sup>+</sup> strain, at this point it is wise to
extract with phenol:chloroform by resuspending the isopropanol-pptd
pellet in TE pH 8 (100 &micro;l), adding equal volume of 1:1 phenol:chloroform
and vortexing 15-30 s, spinning 2 min to separate phases, and then
adding 0.1 vol 3 M sodium acetate pH 5.2 + 2.5 vol ethanol to the aqueous
phase.  Otherwise go to next step directly.
<li>Remove the alcohol precipitating solution by aspiration and inverting
on to towel to wick dry.
<li>Rinse pellet again with 1 ml cold (4&deg;) 70% ethanol.  Aspirate
away.  Spin tube if it looks like the pellet was dislodged (might be
useful to do anyway).  Allow the tube to sit to make sure all the alcohol
has evaporated away.
<li>Dissolve in 50 &micro;l TE pH 8.0 containing DNase-free RNase A, vortex
briefly, then store at &minus;20&deg;.
</ol>
<p>
About 5 &micro;l can be used for restriction digest.

<h4>Lithium Miniprep</h4>
<p>
Plasmid can be obtained directly from colonies on plates or from liquid
culture. This preparation can be scaled up.  This method is more rapid
than others, and DNA has a quality suitable for most applications.
<p>
Obtain the following:
<ul>
<li><a href="#telt">TELT solution</a>
<li>1:1 (w/v) phenol-chloroform
<li>absolute EtOH prechilled to &minus;20&deg;
<li>TE buffer
</ul>
<p>
Also LB medium with appropriate antibiotic. 
<h5>Protocol</h5>
<p>All steps at room temperature.
<ol>
<li><b>If taking from colonies on a plate</b>:  with microspatula, scoop entire 
colony from the agar plate which has grown to 2- to 5-mm diameter.  Transfer to 
1.5 ml conical tube containing 100 &micro;l TELT.  Vortex to suspend cells.
Skip to step after next.
<li><b>If taking from liquid culture</b>:  inoculate colony to 1.8 ml sterile LB 
medium with selection, then grow while shaking to saturation about 18-24 h at 
30&deg; in a glass tube with plastic cap, tube at an angle to maximize
agitation.  Transfer culture to 1.5 ml conical tube, spin 10,000 <i>g</i>
for 20 s, aspirate the supernatant, then add
100 &micro;l TELT, making sure cells are suspended.
<li>Add 100 &micro;l 1:1 phenol:chloroform, vortex 5 s.  Better to leave this
sit for 15 min (but not more), so plasmid can increase.
<li>Spin 15,000 <i>g</i> (or max speed) for 60 s.
<li>Carefully pipet from the top about 75 &micro;l of upper aqueous phase
and transfer to new tube.  Do not collect debris at interface!
<li>Add 150 &micro;l cold absolute EtOH and mix to precipitate plasmid.
Spin max speed for 5 min.
<li>Invert tube to decant supernatant.  Use paper tissue to draw off all
EtOH with capillary action.
<li>Add 1 ml cold EtOH again, and repeat spin and removal of supernatant as 
above.
<li>Close tube cap, perforate top with needle or thumbtack, and dry completely 
in a empty (no dessicant) vacuum dessicator.
<li>Dissolve in 30 &micro;l TE buffer.  Vortex/mix well to get DNA around
inner surface of tube.
</ol>
<p>
2-5 &micro;l can be used in a 20 &micro;l digest reaction.  Add 1 &micro;l
of 10 mg/ml RNase (DNase-free) to the digest reaction to remove RNA that
may obscure detection of DNA fragments.
<h4 id="toothpick">Toothpick Preps</h4>
<p>
Quite convenient in that you don't have to grow up minipreps if all you are 
interested in doing is analyzing whether new transformed bacteria contain the 
desired plasmids.  The prep is too dirty for restriction analysis, but these can 
be run on gels and used on Southerns, or alternatively analyzed for inserts 
using PCR.  This requires large single colonies on plates (2-3 mm dia).
<p>
Prepare NSS solution for this method: 0.2 N NaOH, 0.5% SDS, 20% sucrose.
<ol>
<li>Use a sterile toothpick or loop to transfer a small segment of the
colony to a streak or patch on a master agar plate with selection.
<li>Transfer remainder of colony to numbered microfuge tube containing
50 &micro;l sterile 10 mM EDTA (pH 8.0).  Do this for all colonies desired.
<li>Incubate that master plate for several hours at 37&deg; and then store
at 4&deg;
<li>Continue processing the suspensions in the tubes.  Add 50 &micro;l
of <em>freshly made</em> NSS.  Vortex for 30 seconds.
<li>Incubate tubes in 70&deg; water bath for 5 min, then allow to cool
to room temp.
<li>Add 1.5 &micro;l of 4 M KCl to each tube, and vortex 30 s.
<li>Place 5 min on ice.
<li>Spin 3 min at 4&deg; a maximum speed in the microfuge
<li>Transfer supernatant to fresh tubes containing 0.5 &micro;l of 0.4%
bromophenol blue if they are only to be analyzed by agarose gel electrophoresis,
<em>or</em> containing 2 &micro;l of 10 mM cresol red if samples are to be
analyzed by both PCR and gels.
<li>For gel electrophoresis, prepare a 0.7% agarose gel at least 5 mm thick,
and with a comb that makes wells 5 mm long and 2.5 mm wide.  Do
<strong>not</strong> make the gel up with ethidium bromide;  the intercalating
dye alters the migration of superhelical DNA according to its molecular weight.
<li>Load at least 50 &micro;l of material in the gel.  Run the gel so that
the bromophenol blue migrates between two-thirds and three-fourths the
length of the gel, or if cresol red is used, about half the length.
<li>Soak gel for 30-45 min in 0.5 &micro;g/ml ethidium bromide (in water)
(alternatively use Molecular Probes' SYBR Gold dye if working with low
copy number plasmids)
<li>Visualize with transilluminator.  Colonies containing vector without
insert should show faster migrating bands.
<li>If cresol red dye was used, additional analysis can be done with
PCR.  Cresol red, unlike bromophenol blue or xylene cyanol, does not
inhibit the Taq polymerase.
</ol>
<p>
Once a promising culture has been identified, prepare a plasmid miniprep
and do additional confirmation of the presence of the plasmid with
restriction endonuclease digestion to find the expected bands.


<h3 id="midi-prep">Midi-Preps</h3>
<p>
Use this approx. 10-fold greater scale to prepare DNA for gels and digests. This 
DNA can be use in chromatography kits to prepare it for mammalian cell 
transfection.   The reagents and solutions are the same for the mini-prep.

<ol>
<li>Grow bacteria in 10 ml of a rich medium (LB, YT, Terrific) with the 
appropriate selection, inoculating using a single colony.  The container should 
allow for adequate shaking to aerate the sample (and should be 4 times larger 
than the culture volume itself).
<li>Transfer the culture to sterile 15 ml tube and spin 2000 &times; <i>g</i>
for 10 min at 4&deg;.
<li>Aspirate the medium away and get pellet as dry as possible.  Failing to 
remove medium usually leaves contaminates that inhibit retriction 
endonucleases.
<li>Resuspend pellet in 200 &micro;l of the TE+glucose (Soln 1) in the miniprep
and transfer to microfuge tube.
<li>Add 400 &micro;l of fresh NaOH/SDS (Soln 2) to each tube.  Invert tube rapidly 
about 5 times.  <em>Absolutely do <strong>NOT</strong> vortex</em>! Keep on ice.
<li>Add 300 &micro;l of the 5 M potassium acetate (Soln 3).  Mix it very well
by inversion several times.  Leave on ice 3-5 min.
<li>Spin at maximum speed for 5 min at 4&deg;.  Transfer 600 &micro;l of
supernatant to fresh tube.
<li>Add an equal volume of phenol:chloroform and mix by vortexing. Spin at 
maximum speed for 2 min at 4&deg;, and then move upper aqueous phase to a fresh 
sterile tube.
<li>Add 600 &micro;l isopropanol at room temperature to precipitate the nucleic 
acids.  Mix by vortexing, then let sit 2 min at room temp.
<li>Spin 5 min at room temp maximum speed.
<li>Use very careful aspiration to remove supernatant, and turn tube upside down 
to ensure alcohol wicks away.  Also remove drops adhering to walls.
<li>Add 1 ml of 70% ethanol to the pellet and spin at room temp for 2 min. 
Remove all the supernatant carefully again, ensuring all the alcohol is gone. 
Sometimes the pellet can dislodge, be careful!  Let all the alcohol evaporate. 
Avoid using a vacuum drying centrifuge since the DNA is difficult to dissolve or 
may denature.
<li>Dissolve in 100 &micro;l TE pH 8.0 containing 20 &micro;g/ml DNase-free RNase A 
(pancreatic RNase).  Vortex a few seconds, then store at &minus;20&deg;.
</ol>

<h3>Maxi-Preps</h3>
<p>
Large scale cultures (&ge; 500 ml) are used to make DNA that may be purified by 
CsCl-ethidium bromide gradients.

<ol>
<li> [list is missing here]
</ol>

<h4>Large-Scale Prep</h4>
Use for larger cultures (500 ml).  DNA is good for everything that the
small-scale prep is for, as well as purification on CsCl/EtBr gradients.
<ol>
<li>Inoculate 30 ml of rich medium with selection using either a single
colony from a loop, or with 0.1-1.0 ml of a smaller scale broth culture.
<li>Incubate at correct temperature shaking at 250 cycles/min with good
aeration.  Bacteria should reach late log phase (about an OD<sub>600</sub>
of 0.6).
<li>Prepare 500 ml of the rich medium with antibiotic in a 2 liter flask,
and add 25 ml of the late log phase culture.  Incubate 2.5 hours at
37&deg; shaking at 250 cycles/min.  The OD<sub>600</sub> should be
about 0.4.
<li>Add 2.5 ml of 34 mg/ml chloramphenicol (final concentration about
170 &micro;g/ml).  Continue incubating with shaking for about 12-16 hours.
<li>Recover the culture and store 1-2 ml aliquot at 4&deg;.  Spin the
culture down at 2700 &times; <i>g</i> for 15 min at 4&deg;.  Discard the
supernatant, and invert bottles to let medium wick away.
<li>Resuspend pellet in 200 ml <a href="#ste">STE</a>.  Spin again as in
the previous step, and store the pellet at &minus;20&deg; to freeze it.
<li>While the pellet is freezing, recover the 1-2 ml aliquot and prepare
plasmid from this using the miniprep protocol.  Analyze plasmid by
digestion with restriction enzymes to verify you have it.
</ol>

<!-- ===========================================================
   Purifying Plasmid DNA
 =============================================================== -->

<h2>Purifying Plasmid DNA</h2>

<h3>Polyethylene Glycol</h3>
<p>
Plasmids obtained by alkaline lysis are usually treated with materials
to remove RNA and then high ionic strength PEG can be used to precipitate
plasmid DNA of larger sizes.  Final cleanup is with phenol:chloroform and
ethanol precipitation.
<ol>
<li>A large-scale crude plasmid prepared from the final step of alkaline
or boiling lsysi is suitable.  Use a 3 ml volume, and place into 15 ml
Corex tube.  Chill to 0&deg; in ice-water bath.
<li>Add 3 ml of 5 M lithium chloride (molecular biology grade).
This helps the precipitation of high molecular weight RNA.  Mix well
<li>Spin the mix at 12,000 &times; <i>g</i> for 10 min at 4&deg;.
<li>Transfer the supernatant to 30 ml Corex tubes.
<li>Add an equal volume of isopropanol at room temperature, and mix well.
<li>Spin at 12,000 &times; <i>g</i> for 10 min at room temperature.
<li>Decant the supernatant away carefully and wick it from the tube.
<li>Carefully rinse the walls of the tube and the pellet with 70% ethanol
at room temperature.  Discard the bulk of the ethanol, then use aspiration
to carefully remove the rest.  Invert on to towels to wick away.  Pellet
should be damp at the end.
<li>Dissolve the damp pellet in 500 &micro;l TE pH 8 containing RNase A,
then transfer to microfuge tube and let sit for 30 min at room temp.
<li>Extract the mixture with phenol:chloroform and then chloroform,
once each.
<li>Use ethanol precipitation to recover the DNA.
<li>Dissolve pellet in 1 ml sterile water, then add 0.5 ml of
<a href="#pegmgcl2">PEG-MgCl<sub>2</sub></a>.
<li>Let sit for &ge; 10 min at room temperature.
<li>Spin for 20 min at maximum speed at room temperature.
<li>Aspirate the supernatant carefully.  Wash out remaining traces of PEG
with 0.5 ml of 70% ethanol; resuspend the pellet and spin 5 min at
maximum speed.
<li>Aspirate the ethanol, and repeat the ethanol wash one more time.
Allow the ethanol to evaporate (10-20 min) with the tube open on the bench.
<li>Dissolve in 500 &micro;l TE pH 8.0.  Measure a 1:100 dilution in the
spectrophotometer at 260 nm (50 &micro;g DNA/ml is about 1 OD<sub>260</sub>).
<li>Aliquot and store at &minus;20&deg;.
</ol>

<h3>Chromatography (Kits)</h3>
<p>
The use of proprietary kits to clean up plasmid DNA is widespread. However, the 
old standby methods work just as well and are much less expensive.
<p>
Kits come in two basic types.  Those that yield plasmid DNA suitable for enzyme 
work (PCR, restrictions, ligations) and for transformation into bacteria, and 
higher quality DNA required for eukaryotic transfections (as well as all other 
purposes).
<p>
Resins used generally are of two chemistries:  hydrophobic and mixed 
ion-exchange.  Higher quality DNA usually is prepared from the mixed 
ion-exchange kits.
<p>
Since the 1950s it has been known that DNA binds to silica with chaotropic salts 
present.  The salts effectively reduce hydration around the phosphate backbone 
and make it favorable for them to bind to silica.  Once bound certain solvents 
(50% ethanol) that do not affect DNA binding can then be used to rinse away RNA 
and polysaccharides.  Washing with Tris-EDTA or water will then cause the DNA to 
elute from the column.
<p>
These kits are usually ideal for circular plasmids and long linear DNA. DNA &lt; 
100-200 bp long absorb poorly.
<p>
Kits are limited to the amount of the bacteria used, but are scaled for small, 
medium, and large sizes.

<h3>Cesium Chloride Equilibrium Ultracentrifugation</h3>
<p>
It was established years ago that ethidium bromide molecules bind in different 
numbers to linear and to closed circular DNA, and that the final density of the 
complexes will differ.  Linear DNA is usually the large pieces of the bacterial 
chromosome, while the closed circular form is the plasmid.  One can also purify 
open circular (or nicked circular) DNA this way.
<p>
Because of the different buoyant densities of ethidium bromide-complexed linear 
and circular DNA, these can be separated in the heavy salt CsCl in a centrifugal 
field.  A disadvantage of CsCl purification is that it is expensive and 
time-consuming, although traditionalists claim it is the most pure form of DNA.
<p>
In picking the centrifuge, realize that equilibrium will occur faster in the 
vertical tube rotors compared to slanted tubes, and with rotors capable of 
higher <i>g</i> forces.

<h4>Continuous Gradient</h4>
<ol>
<li>Determine the mass of the crude plasmid DNA preparation.  For each 1.0 gram 
of solution, add 1.01 g of solid CsCl (molecular biology grade). Heat the 
solution in 30&deg; water bath with gentle mixing until the salt has dissolved. 
Normally the prep from a 50 ml culture can be used per gradient:  a single 
Beckman VTi65.2 tube can hold 5 ml of CsCl/EtBr preparation, and so the 50 ml 
prep should be suspended in about 3 ml of TE prior to CsCl addition.
<li>For each 5.0 g of DNA/CsCl solution, add 100 &micro;l of 10 mg/ml
ethidium bromide.  The final density of the solution will be about
1.55 g/ml, and ethidium bromide is about 200 &micro;g/ml.  Saturating
(higher) concentrations were used in the past, but that made UV
illumination necessary.  Today, bands can be made seen under visible light.
<li>If the DNA solution is in Corex tubes, spin at 7700 &times; <i>g</i>
for 5 min at room temp.  If disposable polypropylene is used, spin
at 1100 &times; <i>g</i> for 10 min.
<li>Remove material floating at the top (try a Pasteur pipet); these are 
complexes of EtBr and bacterial proteins.  The pellet at the bottom is other 
bacterial material.  Use either a pasteur pipet or a large gauge needle/syringe, 
transfer the material to a centrifuge tube appropriate for the ultracentrifuge 
rotor.  Make sure to top off any space on top either with <a 
href="#liteparaff">light paraffin oil</a> or <a href="#rebandsoln">rebanding 
solution</a>.
<li>Centrifugation is at 20&deg;:
<table>
<caption>Beckman Rotors</caption>
<col>
<col span="3" style="text-align:center;">
<tr><th>Rotor     <th>RCF (&times; <i>g</i>)  <th>RPM      <th>Time (hours)
<tr><td>NVT 65      <td>  366,000             <td> 62,000  <td>  6
<tr><td>VTi65       <td>  194,000             <td> 45,000  <td> 16
<tr><td>Type 50Ti   <td>  180,000             <td> 45,000  <td> 48
<tr><td>Type 65Ti   <td>  314,000             <td> 60,000  <td> 24
<tr><td>Type 70.1Ti <td>  331,000             <td> 60,000  <td> 24
</table>
<li>Remove tubes from the rotor and place on rack covered with foil. With the 
room dimly lit (overhead lights off), mount a single tube held by a clamp 
attached to a ring stand.  Within the tube, certain bands should be visible.  At 
the bottom is probably a deeply red colored pellet which is RNA complexed with 
EtBr.  At the top between the paraffin oil and salt solution is protein.  Two 
bands should be prominent in the middle of the tube.  A thicker, higher density 
(lower) band represents the closed circular plasmid DNA, the band of interest. 
Above it should be a thinner, lower density (upper) band representing both 
linear and nicked circular DNA.
<li>To collect the lower, closed circular plamid fraction, insert a 21-gauge 
needle at the top of the tube to vent the tube with air as it is tapped from the 
bottom.
<li>Wipe the side of the tube carefully with ethanol to remove grease or
oil.  Then attach a piece of Scotch tape to the side (this will prevent
leaking from outside the tap to be formed).
<li>Attach a 5-10 cc syringe to a 18-gauge needle and insert the needle
to the side of the tube, beveled side up, and just below the closed
circular plasmid band.
<li>Slowly withdraw the plunger, trying to recover the lower band.
Do not try to recover all of it, since some of the upper chromosomal band
can contaminate it.  An alternative is to try to recover the upper band
first, but sometimes it drags the lower band with it.
<li>If very pure plasmid DNA is critical, repeat the centrifugation
step again with rebanding solution.
</ol>
<p>
<a href="#removeEtBr">Methods</a> to remove ethidium bromide and cesium chloride 
are described elsewhere.

<h4>Discontinuous Gradient</h4>
<p>
This is a more rapid method of purification but the product is not
as pure because the sample does not move to its true equilibrium point
as in a continuous gradient.  The choice is between a two-step and three-step
gradient.  Three-step gradients give better results in just as short a time
and should be preferred.  The method described here is a three-step
gradient from or based on Dorin &amp; Bornecque, 1995.
<ol>
<li><b>Top layer</b>.  Use molecular biology grade CsCl.
Dissolve 4.720 g of CsCl in 8 ml of TE pH 8.0.  Adjust the
volume to 10.0 ml.  Add 120 &micro;l of 10 mg/ml
ethidium bromide. This is the <i>top</i> layer of the gradient; it is
35% (w/w) with a molarity of 2.806 and refractive index of 1.3670.
<br><b>Middle layer</b>.  In 1.0 ml of the crude DNA preparation to be
purified, dissolve 0.8 g CsCl, and add 30 &micro;l of 10 mg/ml EtBr
(44% w/w, 3.870 M CsCl, RI=1.3792).  The crude DNA prep is generally from
a mini, midi, or maxi prep in which much of the protein and small molecules
have been removed.
<br><b>Bottom layer</b>.  10.4 g CsCl in 7 ml TE pH 8.0, and the volume
is adjusted to 10.0 ml.  120 &micro;l 10 mg/ml EtBr is added.  The final
CsCl concentration is 59% (w/w), 6.180 M, RI=1.4052.
<li>A 3-cc (hypodermic) syringe with a 10 cm 18-gauge bone marrow needle is used to
transfer 1.5 ml of the <b>top</b> layer to a 5-ml polyallomer ultracentrifuge
tube (e.g., Beckman Quick-Seal).
<li>A 1-cc (tuberculin) syringe with a 10 cm 18-gauge needle to transfer
0.5 ml of the <b>middle</b> layer with the DNA into the tube:  the material
is <em>underlayed</em> below the top layer.
<li>With a 5-cc hypodermic fitted to a long needle, the balance of the tube
is filled with the <b>bottom</b> layer.
<li>The sealed tubes are centrifuged at 330,000 <i>g</i> (60,000 rpm in
Beckman Type 70.1Ti rotor) for 5 hours.  Note that if a higher speed
centrifuge is used, the spin time is reduced considerably.  For instance
in a VTi90 rotor in a Beckman XI-90, the spin time is only 20 min.
<li>The collection of the closed circular DNA is somewhat similar to
that for the continuous gradient.
 <ol>
 <li>A 21-gauge needle is used to make a hole at the top to vent the tube.
 <li>The outside of the tube where collection begins is wiped with ethanol
 to remove oil, and then tape is place over it.
 <li>An 18-gauge needle is fit to a 5 or 10 cc syringe.  Insert needle
 at the sampling point, through the tape, <em>beveled side up</em> and
 <strong>just below</strong> the lower DNA band shown by UV illumination
 with a 302 nm lamp or seen in dim light.  This is the closed circular
 plasmid band.
 <li>Remove the plasmid without disturbing the upper band of chromosomal
 DNA.  It may be prudent NOT to remove every visible trace of the lower
 band, since contaminatin of chromosomal DNA is more problematic.
 </ol>
<li>The ethidium bromide is then <a href="#removeEtBr">removed</a>.
</ol>

<!-- ===========================================================
   Removing EtBr from DNA
 =============================================================== -->

<h2 id="removeEtBr">Removing Ethidium Bromide from DNA</h2>
<p>
One method described is by extraction with organic solvents, and another
is by ion-exchange chromatography.
<h3>Extraction by Organic Solvents</h3>
<ol>
<li>The solution of DNA should be in a glass or polypropylene (chemically
resistant) tube.  Add an equal volume of either water-saturated
<i>n</i>-butanol or isoamyl alcohol.  Close the cap of the tube.
<li>Mix by vortexing.
<li>Spin at 450 <i>g</i> (1500 rpm in Sorvall RT-6000 with HL-4 rotor)
for 3 min at room temp, or let tube stand until phases separate.
<li>Transfer the upper (organic) phase to appropriate waste container
(it should have a beautiful deep pink color).
<li>Repeat extraction 5-6 times until the pink color disappears from both
phases.
</ol>
Removal of CsCl from the solution can be done by ethanol precipitation,
spin dialysis on a microconcentrator, or overnight dialysis.  If the
precipitation step is used:
<ol>
<li>Measure the volume of the CsCl-DNA solution and call it <i>x</i>.
Add 3<i>x</i> volumes of water to the CsCl-DNA solution (this prevents
precipitation of the salt itself by ethanol).
<li>Add 8<i>x</i> volumes of ethanol to the solution and mix (final volume
is 66% ethanol).  Leave for 15 min at 4&deg;; greater recoveries are possible
if left overnight at 4&deg;.  Do not freeze at &minus;20&deg; or even
on ice, since there is a risk of the salt precipitating out.
<li>Spin the tubes at 20,000 <i>g</i> (13,000 rpm in SS-34) for 15 min
at 4&deg;.
<li>The supernatant is decanted to a fresh centrifuge tube.  Add an
equal vol. of absolute ethanol and let stand 15 min at 4&deg;.  Spin this
under the same conditions.
<li>Wash all pellets with 70% ethanol.  Remove as much as possible and let
as much fluid evaporate as possible at room temperature.
<li>Redissolve in 2 ml water (if to be sequenced) or TE pH 8.0 (if to be
stored long term).
<li>Use phenol:chloroform, the chloroform, followed by EtOH pptn, if there
is still significant EtBr present in the DNA.
</ol>
<h3>Ion-Exchange Chromatography</h3>
<ol>
<li>Prepare ~20 g Dowex AG50W-X8 (100-200 dry mesh size) in ca. 100 ml
1 M NaCl for 5 min.  Let resin settle and remove by aspiration.
<li>Add ~100 ml of 1 N HCl and stir slurry for about 5 min.  Allow resin
to settle, and aspirate.
<li>Wash the resin twice with about 100 ml water each, and then follow
it with one wash of TEN buffer.  The equilibrated resin can be stored
at 4&deg; in TEN containing 0.2% sodium azide.
<li>Take a Pasteur pipet and pack a small amount of sterile glass wool
at its narrowing.  Pack about 1 ml of Dowex AG50 within it.
<li>Rinse two column volumes with TE pH 8.0.
<li>Apply the DNA to the column.
<li>After it has entered the column, wash with 1.2 column vol TE pH 8.0,
but collect the eluate into a 30 ml Corex tube.
<li>When the column has run dry, dilute the eluate with 2.5 col vols water.
<li>Precipitate DNA by adding 8 (not column) volumes of ethanol and
let sit 15 min at 4&deg;.  Spin out the ppt at 17,000 <i>g</i>
(12,000 rpm in SS-34) for 15 min @ 4&deg;.
<li>Take the supernatant to a new centrifuge tube, add an equal vol of
absolute ethanol, let stand 15 min @ 4&deg;, and spin at 20,000 <i>g</i>
for 15 min at 4&deg;.
<li>Wash all ppts in 70% ethanol.  Remove as much as possible and let
evaporate at room temp.
<li>Dissolve in water if for sequencing, and TE pH 8.0 for long term storage.
<li>Use phenol:chloroform, the chloroform, followed by EtOH pptn, if there
is still significant EtBr present in the DNA.
</ol>
<p>
DNA concentration can be determined by standard methods (OD<sub>260</sub>).

<h3 id="etbrdecon">Decontamininating Ethidium Bromide Solutions</h3>
<p>
Source:  Sambrook, Fritsch, Maniatis <i>Molecular Cloning:  A Laboratory
Manual</i>, 2nd Ed., Cold Springs Harbor Laboratory Press, 1989, pp. E.8-9.

<h4>Concentrated Solutions (&gt;0.5 mg/ml)</h4>
<p>
Two methods are presented.

<h5>Lunn and Sansone</h5>
<ol>
<li>Water is added to bring the ethidium bromide in the solution to
&le; 0.5 mg/ml.
<li>Prepare fresh 5% hypophosphorous acid (supplied usu. as 50%, which
is very corrosive).  Add 0.2 vol of the acid.
<li>Prepare fresh 0.5 M sodium nitrite (add 34.5 g sodium nitrite
to water for each 500 ml).  Add 0.12 vol (of the amount before
the acid was added) of the nitrite.
<li>The pH should be below 3.0...check it
<li>Leave 24 h at room temperature
<li>Add a large excess of 1 M sodium bicarbonate.
<li>The solution can be discarded.
</ol>

<h5>Quillardet and Hoffnung</h5>
<ol>
<li>Add water to bring concentration of ethidium bromide down to
&lt; 0.5 mg/ml
<li>Add 1 vol of 0.5 M KMnO<sub>4</sub>.  Mix carefully.
<li>Add 1 vol 2.5 N HCl.  Again mix carefully and let solution stand
several hours at room temperature.
<li>Neutralize with 1 vol 2.5 N NaOH, mix, then discard.
</ol>

<h4>Dilute Solutions</h4>
<p>
This technique is ideal for dealing with electrophoresis buffer solutions
containing 0.5 &micro;g/ml EtBr.
<p>
Two methods are presented.

<h5>Lunn and Sansone</h5>
<ol>
<li>For each 100 ml solution, add 2.9 g Amberlite XAD-16 (Rohm &amp; Haas)
<li>Let stand for 12 h at room temperature, shaking intermittently
<li>Filter through Whatman No. 1, and discard filtrate
<li>Seal filter and Amberlite resin in plastic bag, then dispose of
in hazardous waste.
</ol>

<h5>Bensaude</h5>
<ol>
<li>For each 100 ml solution, add 100 mg powdered activated charcoal
<li>Store for 1 hour, shaking intermittently
<li>Filter through Whatman No. 1, then discard filtrate
<li>Seal filter and charcoal in plastic bag, then dispose of in hazardous
waste.
</ol>
<p>
Ethidium bromide decomposes at 262&deg;C and can be rendered harmless by 
incineration.

<!-- ===========================================================
   Purifying DNA in general
 =============================================================== -->

<h2 id="DNApurification">Purification of DNA</h2>
<p>
DNA prepared from PCR or restriction digestion, or recovered from
gels, is resistant to further enzymatic manipulation.  The usual
belief is that this is due to the presence of <q>inhibitors.</q>
DNA is a polyanion, and so the DNA can be bound to an anion-exchange
resin such as DEAE-Sephacel or -Sephadex.  This should only be used for
double-stranded DNA, not for single-stranded or plasmid DNA.
<p>
DNA can also be purified using alternatives.  Reversed-phase resins
such as Elutip-d (Schleicher &amp; Schuell) and NACS (Life Technologies),
silica such as Wizard PCR Preps (Promega) and StrataClean (Stratagene),
or glass powder such as Sephaglas (Pharmacia-LKB) or GENECLEAN (Q-Biogene)
with different binding and eluting chemistries.

<h3 id="deae-dna"> Anion-Exchange Chromatography Purification of
DNA Recovered from Agarose and Polyacrylamide Gels</h3>
Most any DEAE-cellulose (Sigma, Pharmacia-LKB, Whatman DE-52) can be
used.  They must be washed prior to use since they usually include
preservatives (sodium azide).
<ol>
<li>Add 20 vol. of TE pH 7.6 containing 0.6 M NaCl.  Allow resin to settle,
then aspirate the supernatant.  Add another 20 vol. of TE with NaCl
and resuspend.  Allow to settle, remove the excess, and then store
at 4&deg;.
<li>0.6 ml of DEAE resin will bind 20 &micro;g DNA.  Pack a small column
(a 2 ml syringe barrel works) with the resin.
<li>Wash with 3 ml TH pH 7.6 + 0.6 M NaCl, then 3 ml of TE pH 7.6, then
3 ml of TE pH 7.6 + 0.1 M NaCl.
<li>The DNA in TE pH 7.6 is mixed with an equal volume of TE pH 7.6 +
0.2 M NaCl.  This is loaded to the column.  Collect the effluent, and
pass it through the column again.
<li>The column is now washed with 2 &times; 1.5 ml TE pH 7.6 + 0.3 M NaCl.
<li>The DNA is now eluted with 3 &times; 0.5 ml washes of TE pH 7.6 +
0.6 M NaCl.
<li>Extract the elute with phenol:chloroform.
<li>Divide the eluate in two.  In one tube, mix with an equal volume
of isopropanol.  Leave for 15 min at room temperature, then recover
by centrifugation at maximum speed for 10 min at 4&deg;.
<li>Wash pellet carefully with 70% ethanol, and allow to evaporate
for a few minutes.  Redissolve DNA in small amount of TE pH 7.6
(3-5 &micro;l).
<li>To check the amount and quality of the product:
 <ol>
 <li>Put ~20 ng DNA (a small aliquot) into 10 &micro;l TE pH 8.0 and add
6&times; gel loading buffer.
 <li>Prepare a polyacrylamide or agarose gel, as appropriate, and analyze
the DNA on the gel.
 <li>Look carefully for faint fluorescent bands that might indicate
presence of contaminating DNA.
 </ol>
</ol>

<h3>Removal of Small Fragments of Nucleic Acid from Plasmid DNA</h3>
<p>
Small fragments of DNA can accumulate, and it can be necessary
to remove then before the next reaction.  They can possibly interfere
with reactions that depend on the number of 5&prime; and 3&prime;
ends.

<h4>By Centrifugation Through Sodium Chloride</h4>
<p>
This method removes small RNA fragments.
<ol>
<li>To the volume of the plasmid preparation, add 0.1 vol of 3 M
sodium acetate (pH 5.2) and 2 vols of ethanol.  Store 30 min at 4&deg;.
<li>Spin at &gt;10,000 <i>g</i> for 15 min at 4&deg;.  Remove as much of
the supernatant as possible, and let the tube remain open a few minutes to
permit ethanol to evaporate.
<li>Dissolve the pellet in 0.5-1.0 ml TE pH 8.0.  (Plasmid conc. is
&ge;100 &micro;g/ml.)
<li>Add DNase-free RNase to final 10 &micro;g/ml; incubate 1 h room temp.
<li>Add 4 ml of 1 M NaCl in TE pH 8.0 to Beckman SW50.1 centrifuge tube
or equivalent.
<li>To the top of the NaCl solution, carefully layer the plasmid prep
above it.  Use a disposable tip pipettor.  Fill the tube as necessary with
TE pH 8.0.
<li>Spin 150,000 <i>g</i> (40,000 rpm in SW50.1) for 6 h at 20&deg;.
<li>Remove carefully and discard the supernatant.
<li>Dissolve pellet in 0.5 ml TE pH 8.0.  Add 50 &micro;l 3 M sodium acetate
pH 5.2.  Transfer to microfuge tube.
<li>Add 2 vols ethanol, and keep at 4&deg; for 10 min.  Spin out for 15 min
at max speed in a microfuge at 4&deg;.  Allow ethanol to evaporate a few
minutes after removing as much supernatant as possible.
<li>Dissolve damp pellet in TE pH 8.0.
</ol>

<h4>By Chromatography Through Sepharcryl S-1000</h4>
<p>
This method is usually chosen for all fragment separations.
<ol>
<li>Sephacryl S-1000 is prepared in a 1 &times; 10 cm column, and equilibrate
it in Sephacryl equilibration buffer.
<li>To the volume of the plasmid preparation, add 0.1 vol 3 M sodium acetate
(pH 5.2) and 2 vols ethanol.  Let sit 30 min at 4&deg;.
<li>Recover the precipitate after spinning at 10,000 <i>g</i> for 15 min
at 4&deg;.  After removing the supernatant, leave open tube on bench so
that ethanol evaporates for a few minutes.
<li>Dissolve damp pellet in TE pH 8.0 (about 0.4 ml) containing 20 &micro;g/ml
RNase A.  The final DNA concentration should be not go below 100 &micro;g/ml.
<li>Leave for 1 h at room temp.
<li>Extract once with TE pH 8.0-saturated phenol (equal volume).
<li>Recover aqueous phase, and add 100 &micro;l bromphenol dye-sucrose solution
and mix.  Load this on to the Sephacryl column.
<li>Begin collecting 0.5 ml fractions off the column, after the load has
entered, and elution begins.
<li>After about 15 fractions, the dye should have traveled about halfway.
Sample 10 &micro;l from each fraction and run a 0.7% agarose gel or
use ethidium bromide fluorescence to find the fractions which have the
plasmid DNA.
<li>Pool the fractions with the DNA, add 2 vols ethanol for 10 min at 4&deg;,
and spin 10,000 <i>g</i> for 15 min at 4&deg;.  Decant the supernatant,
let ethanol evaporate a few minutes from open tube, and dissolve damp
pellet in TE pH 8.0.
</ol>

<h4>By Precipitation With Lithium Chloride</h4>
<p>
LiCl is able to precipitate RNA by removing water molecules that can hydrate
it.  It also removes proteins from chromatin.
<ol>
<li>To the volume of the plasmid preparation, add 0.1 vol 3 M sodium
acetate, pH 5.2, and then 2 vols ethanol.  Wait 30 min at 4&deg;.
<li>Spin out at 10,000 <i>g</i> for 15 min at 4&deg;.  Remove supnt. and
let ethanol evaporate a few minutes.
<li>Dissolve damp pellet in 1 ml TE pH 8.0 containing 20 &micro;g;/ml RNase A.
DNA concentration should be &ge;100 &micro;g/ml.
<li>Add 3 ml of 4 M LiCl, and leave on ice 30 min.
<li>Centrifuge at 12,000 <i>g</i> for 15 min at 4&deg;.
<li>Transfer supernatant to fresh centrifuge tube, add 6 ml isopropanol.
Let sit for 30 min at room temperature.
<li>Spin tube at 12,000 <i>g</i> for 15 min at 4&deg;.
<li>Remove supernatant, add 5-10 ml of 70% ethanol by vortexing, and
respin at 12,000 <i>g</i> for 10 min at 4&deg;.  Remove supernatant,
store open tube to let ethanol evaporate a few min, and dissolve damp
pellet in TE pH 8.0.
</ol>

<!-- ===========================================================
   Extracting, purifying, analyzing eukaryotic mRNA
 =============================================================== -->

<h2>Eukaryotic Messenger RNA (mRNA):  Extraction, Purification, Analysis</h2>
<p style="font-size:83%;">
Much of notes and method steps of this section are from Sambrook &amp;
Russell, <i>Molecular Cloning:  A Laboratory Manual</i>, 3rd Ed.
<p>
Each typical mammalian cell has about 10 picograms of total RNA, with 80-85% of 
it ribosomal RNA (28S, 18S, 5.8S, and 5S).  The balance is tRNAs and small 
nuclear RNAs (low MW species).  Gel electrophoresis, density gradient 
centrifugation, and anion-exchange chromatography or HPLC can be used to purify 
these.  mRNA is only 1-5% and is heterogeneous in size (few hundred to tens of 
kilobases).

<h3>Inhibiting Ribonucleases (RNases)</h3>
<p>
The major source of contamination from RNases comes from
<ul>
<li>Contaminated Buffers
<li>Use of Automatic Pipettors
</ul>
<p>
Generally the equipment and supplies used that are not consumable or disposable, 
namely glassware, pipettors, re-usable plasticware, should be devoted to the use 
of RNA isolation and purification, and should be used for no other purpose.
<p>
Try to set aside a part of the laboratory whose space is used strictly for the 
purpose of RNA isolation.  Place on a nearby shelves of a cabinet the glassware, 
buffers, etc. that are used only for RNA isolation, and mark the area with a 
prominent notice that the materials are to be used only for RNA isolation.
<p>
If possible, an electrophoresis apparatus should be devoted to the use of RNA 
analysis.  To prepare the apparatus for use, clean it with a detergent solution, 
rinse in water, dry with EtOH, then fill with 3% H<sub>2</sub>O<sub>2</sub>. 
After 10 minutes, rinse the tank with DEPC-treated water.
<p>
In dispensing chemicals, use disposable spatulas or tap out the chemical without 
using spatulas.  Prepare these in RNase-free glassware and DEPC-treated water. 
Where possible, solutions should be treated with 0.1% DEPC for 60 min at 
37&deg;, then autoclaved for 15 min (liq cycle).
<p>
Glassware is best prepared by baking for 4 hours at 300&deg;  Plasticware will 
have to be treated either with DEPC or special products that inactivate RNase on 
contact (RNaseZap from Ambion).  Use tubes and pipets tips certified as 
RNase-free.
<p>
Make use of the known chemical inhibitors of RNases as well (see section below 
on RNase Inhibitors).

<h4>Diethylpyrocarbonate (DEPC)</h4>
<p>
This was first identified in 1971 as able to inactivate RNases.  It does it by 
the ethxoyformylation of histidine groups in the active site. Glassware and 
plasticware are treated by filling water containing 0.1% DEPC, allowing to stand 
60 min at 37&deg; or overnight at room temp. It is then rinsed several times 
with DEPC-treated water, then autoclaved where possible.
<p>
DEPC decays into CO<sub>2</sub> and ethanol, having a half-life in phosphate 
buffer of 20 min at pH 6.0, and 10 min at pH 7.0.  Tris and other amines 
accelerate the process (and are consumed).  Because DEPC breaks down to carbon 
dioxide and ethanol, it can increase ionic strength and lower pH of unbuffered 
solutions.  DEPC that remains undegraded can modify mRNAs and reduce their 
efficiency in <i>in vitro</i> translation systems.
<p>
DEPC is also used to form alkali-labile adducts with the imidazole ring N7 of 
unpaired purines.  This causes cleavage of the base from its glycosidic bond. 
DEPC is thus a chemical probe for secondary structure.  DEPC is highly reactive 
with unpaired adenines, as well as guanine residues in Z-DNA.

<h5>RNase Inhibitors</h5>
<p>
DEPC has already been mentioned as an RNase inhibitor.  Its use is restricted to 
inactivating RNase on equipment and supplies, and in buffers with which it is 
compatible.
<p>
Vanadylribonucleoside complexes are transition state analogs and also inhibit 
RNase.  They do not inactivate the enzyme, and so must be present during the 
extraction and purification.  They must be removed by several washes with phenol 
containing 0.1% hydroquinolone, since they also inhibit RNA polymerases and in 
vitro translation.
<p>
The placenta produces a protein ~50 kDa that binds to and inhibits RNases. 
Their affinities are extremely high.

<h5>Guanidinium Salts</h5>
<p>
These are chaotropic salts that denature proteins.  The isothiocyanate and 
chloride forms of guanidium denature proteins to random coil state. Cox (1968) 
made the first use of the salt to deproteinize RNA for its isolation.  Certain 
tissues are rich in RNase (pancreas) and must be extracted with the strongest 
denaturant, typically guanidinium isothiocyanate. Use of guanidinium 
isothiocynate also include sulfhydryls to break disulfide bonds as well as a 
detergent (Sarkosyl) to disrupt hydrophobic bonds.

<h4>Acid Phenol-Guanidine Thiocyanate-Chloroform Extraction of Cells and 
Tissue</h4>
<p>
Use of guanidinium salts was first mentioned by Ullrich et al (1977). Han et al. 
(1987) developed a method that is quite labor intensive and involves slow 
solubilization of RNA pellets into 5 M guanidine thiocyanate. Chirgwin et al. 
(1979) extracts cells/tissues in 4 M guanidinium isothiocyanate and then is 
layered over CsCl:  this was the best way to produce high purity, high MW RNA. 
The problem is that this cannot process multiple samples.  For a large number of 
samples, the Chomczynski &amp; Sacchi (1987) method is used, and described here. 
The advantage is that this method makes no use of ultracentrifugation. 
Limitations of this method include the ability to extract RNA from adipose 
tissue rich in triglycerides.  Guanidine lysis produces RNA rich in 
polysaccharides and proteoglycans, which can decrease solubilization after 
ethanol precipitation.  RT-PCR reactions can also be inhibited.  Yields are 4-7 
&micro;g/mg tissue or 5-10 &micro;g/million cells, and the 
A<sub>260</sub>/A<sub>280</sub> is 1.8-2.0.

<ol>
<li>RNA is extracted according to the kind of source it has.
 <p>If working with tissues:
 <ol>
 <li>If the tissue is rich in hydrolytic enzymes (pancreas), the
   tissue is best dissected into small pieces and then snap-frozen in
   liquid nitrogen and then stored at &minus;70&deg; or used immediately.
 <li>The tissue is pulverized but kept frozen by addition of liquid nitrogen
   during the processing.
 <li>The powder is transferred to solution (3 ml/100 mg tissue pulverized) of 
 <q>Solution D</q> (denaturing solution):  4 M guanidinium thiocyanate, 25 mM 
 sodium citrate &bull; 2 H<sub>2</sub>O, 0.5% sodium lauryl sarcosinate, and 100 
 mM 2-mercaptoethanol (this can be prepared with heat without adding &beta;-ME 
 until after everything is dissolved; stored several months at room temperature 
 protected from light). This should be in a snap-cap polypropylene tube.
 <li>The solution is homogenized 15-30 s in a polytron.
 </ol>
 <br>If the source is mammalian cells in suspension:
 <ol>
 <li>Spin cells out at 200-1900 <i>g</i> for 5-10 minutes at room
  temp in benchtop centrifuge.
 <li>Remove medium and resuspend pellets in 1-2 ml sterile PBS.
 <li>Spin the cells again, remove the PBS, and add 2 ml of Solution D
 for each 1 million cells.
 <li>Polytron at room temp for 15-30 s.
 </ol>
 For the cells in monolayers, remove the medium and add 5-10 ml sterile
 PBS and rinse.  Then after removing add 2 ml Solution D per 90 mm culture
 dish (or 1 ml per 60 mm dish).  Transfer to a polypropylene tube and
 polytron as usual.
<li>Transfer the homogenate to fresh tube and add, in order:
 <ol>
 <li>0.1 ml of 2 M sodium acetate, pH 4.0
 <li>1 ml of phenol
 <li>0.2 ml chloroform:isoamyl alcohol per milliter Solution D
 Mix after each addition before adding then next.
 </ol>
<li>Vigorously vortex for 10 seconds.  Place on ice for 15 min as
the protein separates from the RNA.
<li>???
</ol>

<h4 id="largephenol">Phenol Extraction of RNA in Large Scale (for mRNA)</h4>
<p>
Used successfully for mammalian cells.  Cells might be extracted in a lysis 
buffer of 1.25% sucrose, 0.3% NP-40 or Triton N-101, 10 mM NaCl, 3 mM 
MgCl<sub>2</sub>, 20 mM TrisHCl pH 7.4, and 10 mM vanadyl-riboside complexes; 
this produces a postnuclear supernatant (8 ml of buffer for 10<sup>9</sup> 
cells).  Prepare in 250 ml glass-stoppered bottle.

<ol>
<li>Add 2 ml 10% SDS, 30 ml cold of ACE extraction buffer (10 mM
sodium acetate pH 5.1, 50 mM NaCl, 3 mM Na<sub>2</sub>EDTA),
40 ml warmed ACE-saturated phenol.
<li>Mix vigorously by hand for 5 min at room temperature and
transfer to 30 ml Corex tubes.
<li>Spin at 8000 rpm for 10 min at 4&deg;.  The rotor can be a SS-34
or Sorvall RC-5B.
<li>Carefully pipet upper aqueous phase to clean cold 250-ml glass-stoppered
bottle (use a Pasteur pipet when near the interphase).
<li>Add to the aqueous phase 20 ml chloroform-isoamyl alcohol (94:6, v/v)
and 20 ml ACE-saturated phenol.
<li>Repeat the extraction procedure.
<li>Re-extract aqueous phase(s) if necessary until no white flocculent
interphase is present.
</ol>
<p>
The RNA is then precipitated overnight with ethanol (see method below).
<p>
To obtain whole-cell RNA extractions, a washed cell pellet can be lysed with ACE 
extraction buffer which contains ribonuclease inhibitor (10 mM VRC or 0.6% 
bentonite or both).  10 ml of this buffer with 50-200 million cells is mixed 
with 0.5 mL of 10% SDS.  This is inverted several times to mix well.  10 mL 
ACE-saturated phenol at 50&deg; is added and extraction carried out for 5 min at 
50&deg;.  From step 3 above centrifugation is done, while steps 4-7 are done 
with the scale down for 10 ml aqueous phase.
<p>
All operations are done on ice and buffers made sterile and ribonuclease-free by 
autoclaving and/or treating with diethyl pyrocarbonate, storing at 4&deg;. 
Solutions of 10% SDS and chloroform-isoamyl alcohol are stored at room 
temperature.  Phenol is stored at &minus;20&deg; and contains a 0.1% 
8-hydroxyquinoline stabilizer.  Phenol is thawed by adding 30 ml ACE extraction 
buffer per 100 ml phenol.  Glassware should be sterilized by overnight baking at 
160&deg;.  Gloves must be worn at all times.
<p>
A yield of 7.5 mg RNA in 10<sup>9</sup> fibroblasts and 2.8 mg in 10<sup>9</sup> 
leukocytes is typical.

<h3 id="anotherRNA">Another Method of RNA Isolation</h3>
<p>
Bake glassware at 108&deg; overnight, including the reagent bottle
for the phenol prep.  Plasticware should be soaked in 1% SDS and baked
overnight at 65&deg;.
<p>
The following reagents are required:
<ul>
<li>Diethylpyrocarbonate (DEPC)-treated distilled water
<li>one 500 ml bottle of phenol equilibrated in buffer
<li>0.1 M TrisHCl, pH 8.0
<li>1 M TrisHCl, pH 8.0
<li>Chloroform/isoamyl alcohol (24:1), 500 ml
<li>Solubilization buffer, 100 ml
 <ul>
 <li>10 mM TrisHCl, pH 7.5
 <li>1 mM EDTA
 <li>0.1% SDS
 </ul>
<li>Grinding buffer, 500 ml or more (make up on day of use)
 <ul>
 <li>5 M guanidine thiocyanate
 <li>10 mM EDTA
 <li>0.3 M TrisHCl, pH 7.6
 <li>8% (v/v) 2-mercaptoethanol
 </ul>
<li>4 M LiCl, &ge; 2 liters
<li>3 M LiCl, &ge; 1 liter
<li>ice-cold 100% ethanol
<li>ice-cold 80% ethanol (make using DEPC-treated water and the absolute
  ethanol)
<li>5 M ammonium acetate, 250 ml
<li>0.5 M EDTA, 100 ml
<li>Oligo-dT column loading buffer, 100 ml
 <ul>
 <li>20 mM TrisHCl, pH 7.6
 <li>0.5 M NaCl
 <li>1 mM EDTA
 <li>0.1% SDS
 </ul>
<li>Oligo-dT column elution buffer, 100 ml
 <ul>
 <li>10 mM TrisHCl, pH 7.5
 <li>1 mM EDTA
 <li>0.05% SDS
 </ul>
<li>Wash buffer, 100 ml
 <ul>
 <li>1 M NaOH
 <li>5 mM EDTA
 </ul>
<li>5 M NaCl, 100 ml
</ul>

<h4>Day 1</h4>
<ol>
<li>Place tissue in sealable bag or mortar &amp; pestle.  Freeze
with liquid nitrogen.  Grind appropriately.
<li>Pass ground tissue to beaker and add grinding buffer.
Continue homogenization with tissue-mizer (the rotor/blades should have
been baked overnight to destroy RNases).  If using ground tomato,
use 100 ml buffer for every 10 g;  if avocado, 150 ml for every 50 g.
<li>Place homogenate in Oakridge tubes (prepare as for plasticware),
and tubes in centrifuge.  Spin 20,000 rpm (in JA-20 rotor) for 20 min.
<li>Filter the supernatant through a single layer of miracloth into
a beaker of the same size.
<li>28 ml aliquots are distributed to 250 ml bottles.  Add 196 ml
(7 volumes) of 4 M LiCl.  Shake the bottle.  Leave overnight at 4&deg;.
</ol>

<h4>Day 2</h4>
<ol>
<li>The RNA is pelleted for 90 min at 8500 rpm (JA-14).
<li>Pour off the supernatant, then resuspend in a minimum volume of 3 M LiCl.
Add 20 ml to the first pellet, resuspend that by forcing a stream over the
pellet with a 10 ml pipet.  Then transfer to 2nd pellet, resuspend, etc.
Rinse a 2nd time with 3 M LiCl
<li>Transfer to Oakridge tubes, and fill the tube completely at the end.
Spin for 60 min at 8500 rpm (JS 13.1).
<li>Pour off supernatant.  Add 15 ml solubilization buffer.  Place
in a &minus;70&deg; freezer for about 10-15 min until just about frozen.
<li>Vortex to resuspend pellet as the solution thaws.  This takes about
10 min.
<li>Transfer to sterile 50 ml conical tubes, rinsing the old tube with
a little solubilization buffer.  Keep volume in each tube to 20 ml, but
distribute to pairs of tubes and balance them.
<li>Add 1 volume of chloroform/isoamyl alcohol to 1 volume buffered
(or water-saturated) phenol (about 100 ml of one to the other, or the
volume necessary for the extraction below).
<li>Add an equal volume of the phenol/chloroform/isoamyl alcohol reagent
to each volume in the 50 ml tubes.  Vortex to mix, then spin 10 ml
on a bench top centrifuge.
<li>The upper phase&mdash;not the white flocculant!&mdash;is transferred to
fresh 50 ml tubes using a sterile (10 ml) pipet.  Repeat the
phenol/CHCl<sub>3</sub>/isoamyl alcohol extraction.  Do as often as necessary.
<li>Once extraction is satisfactory and the upper phase is in a fresh
50 ml tube, add an equal volume of chloroform/isoamyl alcohol (containing
no phenol).  Vortex, spin briefly, and transfer the upper phase.  Repeat
the process.
<li>Pool all aqueous phases to find the total volume.  Distribute no more
than 6 ml of aqueuos phases to 30 ml Corex tubes.  Add 0.4 volumes (2.4
ml if 6 ml) of 5 M ammonium acetate to each tube, and mix well.
<li>Add 2.5 volumes of the sample + ammonium acetate volume
(21 ml if 6 ml used) of ice-cold absolute ethanol.  Place in the
&minus;70&deg; freezer to precipitate the RNA.
</ol>

<h4>Day 3</h4>
<ol>
<li>Take the samples from the &minus;70&deg; freezer and permit to thaw
slightly.  Transfer to 30 ml Corex tubes.  Rinse out tubes with cold
80% ethanol.
<li>Centrifuge the tubes at 8500 rpm for 20 min (JS 13.1 swinging bucket).
<li>Pour off the supernatant;  use a baked beaker in case the pellet comes
loose and it is necessary to recover it.  Invert the Corex tube on to
a clean Kimwipe; it should not be touched by human hands.  In removing
traces of ethanol, use a long clean pasteur pipet to aspirate off the
drops.
<li>Cover the tubes with parafilm and vent the cover with needle sticks.
Place in a vacuum dessicator and dry for 5 min.
</ol>
<p>
The pellets are then to be re-suspended, depending upon the use or next step in 
enrichment.  If the poly(A)<sup>+</sup> selection is to be done immediately, 
resuspend pellets in the oligo-dT buffer recommended for the protocol used. 
Always resuspend in a minimum volume, and adjust volume as necessary.
<p>
It is important to characterize the yield of RNA.  At the very least, pool the 
resuspended pellets and scan a dilution (about 1:100) of the RNA sample in the 
range 225-300 nm.  Note particularly the OD values for 230, 260, and 280 nm. 
The OD<sub>260</sub>/OD<sub>280</sub> should be &ge; 2.  If the ratio is 
significantly less than 2 (&le;1.5) then protein or phenol is in the sample and 
the RNA should be re-extracted with phenol/chloroform/isoamyl alcohol and then 
chloroform/isoamyl alcohol. An OD<sub>260</sub>/OD<sub>230</sub> ratio &ge; 2.5 
indicates a sample virtually free of polysaccharides, but it is not uncommon to 
find the ratio &lt; 1.0, which is not troublesome.
<p>
A 40 &micro;g/ml RNA solution has an OD<sub>260</sub> of 1.0.
<p>
Always store RNA long term in precipitated form in the &minus;70&deg; freezer. 
Precipitate with ammonium acetate and ice-cold ethanol using the procedure 
above.

<h3>Poly(A)<sup>+</sup> Enrichment</h3>
<p>
Prepare a 65&deg; water bath.
<ol>
<li>Pellet and dry RNA
<li>Prepare a cellulose column:
 <ol>
 <li>Resuspend 0.5-1.0 g cellulose in DEPC-treated water.  Remove any fines.
 <li>Pour the column into a disposable (Bio-Rad) column.
 <li>Equilibrate with loading buffer
 </ol>
<li>Obtain an oligo dT column from its proper storage (&minus;20&deg; freezer):
 <ol>
 <li>Drain off the ethanol.
 <li>Wash with DEPC-treated water.
 <li>Wash with 0.1 M NaOH, 5 mM EDTA
 <li>Wash with DEPC water until pH is &lt; 8.
 <li>Equilibrate with loading buffer.
 </ol>
<li>Stack the cellulose column first in a tandem flow with the oligo-dT column.
Place a clean collection tube under the oligo-dT column.
<li>Resuspend RNA to 2-5 mg/ml in loading buffer.
<li>Heat to 65&deg; for 10 minutes to denature any intramolecular or
intermolecular pairs.
<li>Rapidly cool in ice just to room temperature (only takes a few quick
swirls in ice-water bath).
<li>Add 0.11 volumes of 5 M NaCl (final [NaCl] = 0.5 M)
<li>Load the sample on to the cellulose column.  Wash the cellulose column
with enough loading buffer to get unbound material into oligo-dT column.
<li>Remove the cellulose column from the oligo-dT column.  Recover the
effluent from the oligo-dT column and heat to 65&deg;, quick-cool, and
reapply the material to the oligo-dT column (NOT to the cellulose column).
<li>Wash through with enough loading buffer to remove unbound material.
Save the poly(A)<sup>&minus;</sup> RNA in the effluent for possible later
use.
<li>Apply the elution buffer after setting up a new collection tube.
Apply enough elution buffer to remove all material.
</ol>
<p>
Measure the OD at 230, 260 and 280 nm and characterize the quantity and purity 
of the RNA.  If storing the RNA, precipitate with acetate and ice-cold ethanol, 
aliquot to storage tubes, spin down and remove supernatant.  Regenerate and 
store the oligo dT columns with DEPC-water, NaOH/EDTA, and DEPC-water washes, 
storing in ethanol.


<!-- ===========================================================
   small scale DNA extraction
 =============================================================== -->

<h2 id="smalldnaxtr">Extraction of DNA at Small Scale</h2>
<p>
50 &micro;l extract in a 1.5 ml tube is worked up as follows:
<ol>
<li>50 &micro;l TE-saturated phenol:chloroform:isoamyl alcohol (24:24:1, v/v/v)
is added.  (TE is 10 mM TrisHCl pH 8.0, 1 mM sodium EDTA, and TE-saturated
phenol is 3 ml TE per 10 ml phenol.)
<li>Vortex for 1 min and spin in a microfuge for 1 min.
<li>Transfer carefully the aqueous phase to a clean tube with a
micropipettor.
</ol>
<p>
Generally the extraction need only be done once, and the DNA can be
precipitated with ethanol.  TE-saturated phenol-chloroform-isoamyl alcohol
is stable for 1 month at room temperature or &minus;20&deg;.  However
basic conditions are essential to prevent acid depurination of DNA,
and so the aqueous phase of stored phenol should be checked for its pH.

<!-- ===========================================================
   Screening Colonies by In situ Lysis and Hyb
 =============================================================== -->

<h2 id="colonyscreen">Special Method:  <i>In Situ</i> Lysis and Hybridization
for Screening Bacterial Colonies for Clones</h2>
<p>
This is a rather special method of DNA extraction in that the DNA of bacteria in 
a plate colony can be extracted and the DNA simultaneously analyzed or evaluated 
right on the colonies on the growth plate.
<p>
After transformation and growth of colonies on several plates, the colonies
that grow must be screened to verify the presence of the desired plasmid.
(Methods describing transformation of bacteria are described in major sections
below.)
<p>
The technique follows these steps:
<ul>
<li>Colonies are consolidated (gridded) to a master agar plate and onto
nitrocellulose or nylon filter on tope of a second agar plate
<li>After the colonies grow on replating, the master plate is stored cold
and the filter plate is given a lysing solution in situ and then
hybridized with probe
</ul>
Filters used are nitrocellulose, nylon or Whatman 451.  For most
routine screening, nitrocellulose and nylon are preferred.  Nylon
is the most durable and can take several rounds of hybridization and washing
at high temperatures; when screening involves multiple sequential probes,
use nylon.  A drawback is that nylon is probably more expensive and
may require activation or pretreatment.  Whatman 451 is used to screen
arrayed libraries stored as cultures of transformed colonies in separate
well of microtiter plates.  W451 is usually more durable than nitrocellulose
and less prone to distortion or cracking.  But signal can be weaker if
care is not taken during denaturation.
<p>
In addition to cells transformed with recombinant plasmid, you should use
cells also transformed with nonrecombinant plasmid controls, such as the
vector without the insert (pUC might be used too).
<h3>Transferring Colonies to Filter and Master Plates</h3>
The screening is done in the following steps:
<ol>
<li>Using gloved hands, place a nitrocellulose or nylon filter on
an agar plate containing the selection antibiotic.
<li>Place a grid behind a second (master) plate, with 1 cm squares
marked off for maximum effect.
<li>Using sterile toothpicks or inoculating loops, transfer colonies
one by one on to the filter test plate and then on to the master plate.
Streak 2-3 mm or make dots within a single cell of the grid according to
the grid pattern.  Up to 100 colonies can be placed on a 90 mm plate.
<li>Prepare a similar grid with bacteria transformed with nonrecombinant
plasmid (pUC).
<li>Grow the plates at 37&deg; inverted until the streaks have growth
to a width of 0.5-1.0 mm (takes about 6-16 h).
<li>If the plasmid copy number is expected to be low, at this time, the filter
is transferred to a plate with chloramphenicol.  In normal circumstances
amplification is usually not required.
<li>Use a needle (18 ga.) to stab the filter into the agar in at least
3 asymmetric locations (use India ink, if necessary).
<li>Seal the master plate with parafilm and store at 4&deg; inverted.
<li>Proceed to the in situ lysis and hybridization steps with the filter(s).
</ol>

<h3>In Situ Lysis of Colonies on the Filter</h3>
This method was developed by Grunstein and Hogness in 1975.
<p>
The following reagents and solutions will be required:
<ul>
<li>Denaturation solution
 <ul>
 <li>1.5 M NaCl
 <li>0.5 M NaOH
 </ul>
<li>Neutralizing solution
 <ul>
 <li>0.5 M Tris HCl, pH 7.2
 <li>1.5 M NaCl
 </ul>
<li>10% SDS (not always required)
<li>2 &times; <a href="#SSPE20x">SSPE</a>
</ul>
Other equipment/materials necessary include:
<ul>
<li>Something to hold filters for processing.  Cafeteria trays are quite
suitable and usually hold 25 filters.
<li>A means of cross-linking DNA to the filter:  usually a microwave
oven, a vacuum baking oven set at 80&deg;, or UV light.
<li>Whatman 3MM paper
</ul>
The following steps are taken:
<ol>
<li>Cut 4 pieces of Whatmann 3MM to the correct size of the filter and
place each in one of four glass or plastic trays.  Each is then saturated
with one of the four solutions above:  10% SDS (if used), denaturing
solution, neutralizing solution, 2&times; SSPE
<li>Excess liquid is poured off, and the paper is rolled back on to
the tray with a 10 ml pipet to remove air bubbles.  The 3MM paper should
not be too wet, or the colonies will diffuse and blur the signal.
<li>Peel off the filter with blunt end forceps from the plates, and place
the filter, colony side up, on paper impregnated with 10% SDS for 3 minutes.
The SDS will limit diffusion of the plasmid and result in sharper signal.
<li>Transfer filter to paper impregnated with denaturing solution for
5 min.  When doing the transfer, attempt to remove excess solution by
dragging the underside of the filter against the edge of the tray of the
paper containing the previous solution.  It is also possible to remove
excess by dropping briefly on dry paper towel.
<li>Move the filter now to the neutralizing solution for 5 min.  An option
is to repeat this step, placing on a second piece of paper with neutralizing
solution.
<li>Place the filter on the paper saturated with 2&times; SSPE for 5 min.
An alternative is to fill a tray with the 2&times; SSPE and float the filter
on top, agitating to sink it.  And leaving it there until the last filter
is processed.
<li>The filters are then dried, colony side up, on a sheet of 3MM paper,
for about 30 min at room temp, if DNA is to be baked.  However, if the
filters are to be UV-treated, lay on 2&times; SSPE-impregnated paper
or dry paper, according to directions.
<li>DNA is linked to the filter either by baking, in which filters are
sandwiched between 3MM and then baked 1-2 hours at 80&deg; or by
following the manufacturer's directions for fixing the DNA using
a specific device.  Note that overbaking will cause filter brittleness.
<li>Filters can now be stored or go to the hybridization step.  To store,
sandwich between 3MM (you can interleave for multiple filters) and
cover in aluminum foil at room temp.
</ol>



<!-- ==============================================================
     Analysis of, Characterizing Properties of and Quantitating DNA and RNA
=================================================================== -->
<h1 id="characterize-and-purify">
Analysis of, Characterizing Properties of and Quantitating DNA and RNA</h1>
<p>
After the DNA or RNA is extracted and purified, it should be characterized
in its pure state.  The characterization and analysis methods described here
are to determine the following properties of the DNA and RNA:
<ul>
<li>its amount or quantity
<li>its purity
<li>its structure
 <ul>
 <li>primary structure:  its sequence
 <li>secondary structure:  its bonding or molecular interactions, including
intermolecular and intramolecular interactions: single- or double-strandedness,
formation of stem-loops and hairpins
 </ul>
</ul>
<p>
Separation methods described here can be analytical or preparative.  In general
analytical methods and not preparative (purification) methods are described.


<h2 id="DNAbyOD">Spectrophotometric Determination of DNA and RNA 
Concentration</h2>
<p>
The method below is from Sambrook, J, Fritsch, EF, Maniatis, T (1989)
<i>Molecular Cloning, A Laboratory Manual</i>, 2nd Ed., p. E.5.
<p>
Most likely the DNA/RNA solution is too concentrated to measure directly in a 
spectrophotometer.  A dilution is necessary usually. The best dilution is one 
that will measure the optical density at 260 nm (OD<sub>260</sub>, also given as 
A<sub>260</sub>) in a range between 0.1 and 1.0.  The table below shows the type 
of solution and the concentration of the solution which would produce an 
OD<sub>260</sub>=1:

<table>
<caption>Solutions producing OD<sub>260</sub>=1</caption>
<col>
<col style="text-align:center;">
<tr>
 <th>Type of solution
 <th>Concentration
<tr> <td>ds DNA <td>50 &micro;g/ml
<tr> <td>ss DNA <td>40 &micro;g/ml
<tr> <td>RNA    <td>20 &micro;g/ml
</table>
<p>
Therefore, if a dsDNA solution measures an OD<sub>260</sub> of 0.78,
the concentration of DNA is:
<table class="eqn" style="margin:2em auto;">
<col><col style="text-align:center;">
<tr><td rowspan="2">0.78 AU &times;
 <td style="border-bottom:1px solid black;">50 &micro;g dsDNA/ml
 <td rowspan="2">= 39 &micro;g dsDNA/ml
<tr>
 <td>1.0 AU
<tr><td colspan="2" style="font-size:75%;">AU = <q>absorbance units</q>
</table>
<p>
Nucleic acid purity can also be determined.  The OD<sub>280</sub> of the
dilute DNA or RNA solution is measured, and for DNA, the ratio of
OD<sub>260</sub>/OD<sub>280</sub> should be 1.8.  For RNA, it is 2.0.
Generally if protein or phenol contaminate the mixture, the 260/280 ratio
is much less.

<h2 id="DNAbyEtBr">Determination of Double-Stranded DNA by
Ethidium Bromide Fluorescence</h2>
<p>
The method below is from Sambrook, J, Fritsch, EF, Maniatis, T (1989) 
<i>Molecular Cloning, A Laboratory Manual</i>, 2nd Ed., p. E.5. For dsDNA 
solution too dilute (&lt;250 ng/ml) or containing interfering UV-absorbing 
agents which make spectrophotometric determination impossible, ethidium bromide 
fluorescence can be used to indicate concentrations.  Three variations of this 
are possible:

<h3>Transparent film method</h3>
<ol>
<li>Transparent film is stretched over a UV transilluminator or over a sheet
of black paper.
<li>1-5 &micro;l DNA solution is spotted on to the film.
<li>Equal volumes of a series of DNA standards (0, 1, 2.5, 5, 10, 20 &micro;g/ml)
in an array on the film.  Such a standard should have a size approximately
the size of the DNA unknown whose concentration is to be determined.
<li>To each spot and equal volume of TE (pH 7.6) containing 2 &micro;g/ml
ethidium bromide is added.  Mix using the pipet.
<li>The spots are photographed under short UV.  The concentration can now
be estimated.
</ol>

<h3>Agarose plate method</h3>
<p>
DNA is spotted on an agarose gel slab (1% gel) which contains 0.5 &micro;g/ml 
ethidium bromide.  The gel is allowed to stand at room temperature for a few 
hours so that small molecules can diffuse away.

<h3>Minigel method</h3>
<ol>
<li>2 &micro;l of the DNA sample is mixed with 0.4 &micro;l of 6&times; gel-loading
buffer, and this is loaded into a slot of a 0.8% agarose minigel with
0.5 &micro;g/ml ethidium bromide.
<li>Mix 2 &micro;l DNA standards of 0, 2.5, 5, 10, 20, 30, 40, and 50 &micro;g/ml
with 0.4 &micro;l gel loading buffer.
<li>Electrophoresis is done until the dye moves 1-2 cm.
<li>Immerse the gel for 5 min in electrophoresis buffer containing 10 mM
MgCl<sub>2</sub> to destain.
<li>Photograph and compare intensities in the image.
</ol>

<h2 id="picogreen">Molecular Probes PicoGreen<sup>&reg;</sup>
ds DNA Fluorescence Assay</h2>
<p>
Limit of detection (LOD) for the 2 ml assay format is 25 pG/mL (50 pG in 2 ml). 
The limit of detection for the microplate reader is 250 pG/mL (50 pG in 0.2 ml). 
Fluorescence is linear up to 1000 nG/mL, 4000- to 40,000-fold greater than the 
LOD.

<p>
The P-7581 kit contains:
<ul>
<li>1 ml PicoGreen reagent in DMSO
<li>25 ml of 20 &times; Tris-EDTA reagent (200 mM Tris, pH 7.5; 20 mM EDTA)
<li>1 ml of 100 &micro;g ds DNA standard (&lambda; phage)
</ul>
<p>
This can produce 200 assays of 2 ml total volume, or 2000 assays of
the microplate format 0.2 ml total volume.
<p>
The PicoGreen reagent should warm to room temp before the vial is opened.
<strong>Wear gloves:  it is unknown whether the reagent is mutagenic,
but it is safe bet that it could be.  When disposing of nucleic acid
reagents, it is advised they be poured over activated charcoal, and then
the charcoal incinerated.</strong>
<p>
Whatever Tris-EDTA buffer is used, it should be both nucleic acid-free
(to keep background low) and DNase-free.
<ol>
<li>Prepare 1 &times; Tris-EDTA by dilution of the necessary amount of
20 &times; stock.
<li>Prepare a 200-fold dilution of the PicoGreen reagent using the
1 &times; Tris-EDTA solution.  Use plastic rather than glass since the
reagent may adsorb to glass.  Protect the container from light (cover
in foil or place in dark).  Use this within a few hours of preparation.
<li>Standard Curve
 <br>For nonreplicated assays:
 <ol>
 <li>Take 2.5 &micro;l of 100 &micro;g/mL &lambda; ds DNA standard
     dilute 50-fold with 122.5 &micro;l of Tris-EDTA.  This
   is 2 &micro;g/mL working solution.
 <li>Pipet 100 &micro;l to the well marked as 1000 nG/mL.
 <li>Pipet 10 &micro;l to the well marked 100 nG/mL and
   add 90 &micro;l Tris-EDTA to that.
 <li>Pipet 10 &micro;l to a new tube and then add 990 &micro;l Tris-EDTA
    to make a 1:100 dilution, and now the solution is 20 nG/mL
    working solution.
 <li>Pipet 100 &micro;l of this to the well marked 10 nG/mL.
 <li>Pipet 10 &micro;l of this to the well marked 1 nG/mL, and
     add 90 &micro;l Tris-EDTA.
 <li>Pipet 25 &micro;l of the 20 nG/mL solution to a new tube,
     and add 975 &micro;l Tris-EDTA to obtain another 1:40 dilution,
     or 500 pG/mL.
 <li>Add 100 &micro;l to the well marked 250 pG/mL.
 </ol>
 For duplicated assays, take 5 &micro;l of the 100 &micro;g/mL &lambda; standard
 and dilute with 245 &micro;l Tris-EDTA.  Steps b through h should be okay
 from then on.
<li>Include a blank which has only 100 &micro;l Tris-EDTA reagent.
<li>Pipet 0 to 100 &micro;l of unknown solutions.  They should within the
best guess be in the range of the standard curve.
<li>Pipet 100 &micro;l of the working PicoGreen reagent to each well.
Mix well, then let sit between 2-5 min protected from light.
<li>Set the microplate reader for excitation and emission typical for
fluorscein (excitation ~ 480 nm, emission ~ 520 nm).  Set the gain
of the instrument so that the highest standard is represents the
maximum fluoroscence intensity of the instrument.
</ol>

<h3>Interferences</h3>
<p>
Taken from Table 1 in the PicoGreen kit instructions

<table>
<col>
<col span="2" style="text-align:center;width:5em;">
<tr>
 <th>Compound
 <th>Concentration
 <th>Percent Signal Change
<tr>
 <td>sodium chloride
 <td>200 mM
 <td>&minus;30%
<tr>
 <td>urea
 <td>2 M
 <td>&minus;9%
<tr>
 <td>sodium dodecyl sulfate
 <td>0.01%
 <td>&minus;1%
<tr>
 <td>Triton X-100
 <td>0.1%
 <td>+7%
<tr>
 <td>bovine serum albumin
 <td>2%
 <td>&minus;16%
</table>

<!-- ===========================================================
   separation and analysis of polynucleotides
 =============================================================== -->


<h2>Analysis/Separation of Nucleic Acids</h2>

<h3 id="agagel">Agarose Gel Electrophoresis in DNA and RNA Analysis and
Preparation/Purification</h3>
<p>
Agarose gel electrophoresis (AGE) can be used in both the analysis and
preparation (purification) of DNA and RNA.  But since it is often used as
an analytical tool, it appears in this section, along with its use in 
purifiying/preparing DNA and RNA and the recovery of those biomolecules
from gels.
<p>
Agarose is copolymeric structure of [<span 
class="smallcap">D-</span>galactose-&alpha;1&rarr;3-<span 
class="smallcap">L-</span>galactose-&beta;1&rarr;4] repeat units, in which the 3 
and 6 carbons of the L-galactose have an anhydro (ether) bridge. 
Ultrastructurally agarose forms helical fibers aggregating into supercoiled 
forms with a 20-30 nm radius.  When it gels a 3-D mesh of channels forms with 
diameters ranging from 50 nm to &gt; 200 nm.  A single chain of the polymer 
generally has ~800 galactose monomers, although the average length varies from 
one preparation to another.  Lower grades of agarose can be contaminated with 
salts and proteins.
<p>
Migration of DNA through agarose in an electric field (agarose gel
electrophoresis) is affected by several factors.
<ul>
<li>Size of DNA polymer.  The migration rate is inversely proportional
to the logarithm of the number of bases (or the molecular weight/mass).
<li>Concentration of agarose.  The mobility (&mu;) of DNA through the
gel can be related through the following equation:
 <p  style="margin:2em auto;" class="eqnserif">
   log &mu; = log &mu;<sub>0</sub> &minus; <i>K</i><sub>r</sub>[gel]
 <p>
where &mu;<sub>0</sub> is the mobility in buffer without agarose
and <i>K</i><sub>r</sub> is a retardation coefficient, and the gel
concentration is a percentage.
<li>DNA conformation.  DNA comes in different forms (linear, nicked
circular, superhelical closed circular).  The forms which move fastest
depend on the environment, including ionic strength, strength of applied
current, etc.  If there is some confusion about results, it is best to
try to include controls that provide a superhelical form and one linearized
by digestion with a one-site enzyme.
<li>Presence of intercalating dyes.  Binding reduces DNA negative charge
and increase the stiffness and length.
<li>At low voltages, the migration rate of linear DNA fragments is
proportional to applied voltage.  At higher voltages, high MW fragments
move much faster than expected.  The best resolution for fragments
&gt; 2 kb means that the voltage should be 5-8 V/cm.
<li>Agarose type.  A standard and low-melting type of agarose are
available.  Intermediate types with respect to melting and gelling
temperatures are also known.  Gel strength is important for low percentage
gels (as low as 0.3%, which can separate bands as high as 60 kb.
Hydroxyethylation of agarose alters is melting and gelling temperature,
and low-melting agaroses are important for recovery of ds DNA.  Low
electroendo-osmosis (EEO) gels may be important when trying to separate
high MW fragments.
<li>Buffer.  High ionic strength causes higher mobility, but heats
up the gel and melts it and denatures the DNA.  Pure water does not really
let the DNA migrate at all.
</ul>
<p>
Although gels of 0.1 and 0.2% can be cast and run, they are very fragile and
take days to run if trying to separate DNA in the hundreds of thousands of base
pairs.  The practical upper limit is 750 kb.  Some genes are in fact several
thousand kilobases (megabases) long and cannot be contained in a gel.
Chromosomes in lower eukaryotes can be 7000 kb long.  To separate large segments
of DNA, see <a href="#pulsedfield"><b>pulsed field electrophoresis</b></a>.

<h4>Buffers</h4>
<p>
Three different Tris buffers are normally used in electrophoresis:
<ul>
<li><a href="#tae-buffer">TAE</a>
<li><a href="#tbe-buffer">TBE</a>
<li><a href="#tpe-buffer">TPE</a>
</ul>
<p id="comparing-tris-age-buffers">
All work equally well when used at the appropriate concentration. Because of its 
lower buffering capacity, TAE will become exhausted with longer electrophoresis; 
this is seen by the bromophenol blue dye changing to a yellow color in the gel 
towards the anode, which become acidic.  TBE and TPE are more expensive buffers, 
but have higher buffering capacity.  TAE provides higher mobilities for ds 
linear DNA, and the resolution of higher MW fragments is better with TAE, but 
worse for lower MW fragments.  Prefer TAE when planning to do Southerns on 
genomes.  Also supercoiled DNA resolves better in TAE.


<h4>Gel-Loading Buffers</h4>
<p>
Buffers containing dyes and density-producing agents are a matter of personal 
preference.  Xylene cyanole FF and bromophenol blue are used at 0.25% in 
6&times; buffers.  Sucrose (40%), Ficoll (15%, Type 400), or glycerol (30%) are 
components that produce the greater density in the 6&times; stock.  If using 
glycerol or sucrose, store the buffer at 4&deg;.
<p>
The dyes also serve as markers of a sort.  Bromophenol blue moves with a 300 bp 
ds DNA in 0.5&times; TBE, whereas xylene cyanol FF moves much slower with the 
4000 bp linear ds DNA marker.

<h4>Pouring and Running Gels</h4>
<p>
A power supply capable of 500 V and 200 mA should be used.  Have a
55&deg; water bath available.
<ol>
<li>Tape off either a clean, dry glass plate or the open ends of a tray
mold supplied with an electrophoresis apparatus.
<li>Have enough buffer for both gel and tank, since different preparations
of buffers can affect electrophoresis.
<li>Add powdered agarose to the correct amount of buffer to produce the
desired concentration.  See table for best resolving range:

<table>
<col span="5" style="text-align:center">
<tr><th rowspan="2">% Agarose <th colspan="4">Base Pair Resolution by Gel Type (bp)
<tr>      <th>Standard Type <th>High Strength <th>Low Gelling/Melting
         <th>Low Gelling/Melting<br>Low Viscosity

<tr><td>0.3  <td>              <td>              <td>              <td>
<tr><td>0.5  <td> 700 - 25,000 <td>              <td>              <td>
<tr><td>0.8  <td> 500 - 15,000 <td> 800 - 10,000 <td> 800 - 10,000 <td>
<tr><td>1.0  <td> 250 - 12,000 <td> 400 - 8,000  <td> 400 - 8,000  <td>
<tr><td>1.2  <td> 150 - 6,000  <td> 300 - 7,000  <td> 300 - 7,000  <td>
<tr><td>1.5  <td>  80 - 4,000  <td> 200 - 4,000  <td> 200 - 4,000  <td>
<tr><td>2.0  <td>              <td> 100 - 3,000  <td> 100 - 3,000  <td>
<tr><td>3.0  <td>              <td>              <td> 500 - 1,000  <td> 500 - 1,000
<tr><td>4.0  <td>              <td>              <td>              <td> 100 - 500
<tr><td>6.0  <td>              <td>              <td>              <td> 10 - 100
</table>
<li>Heat the stock solution for the minimal time in water bath or microwave.
<li>Transfer to 55&deg; water bath.  Let cool to that temperature, then
add 0.5 &micro;g/ml ethidium bromide, and mix by swirling.
<li>While the gel is cooling, prepare the casting tray with a comb such
that the comb is 0.5-1.0 mm above the bottom to allow complete well
formation.
<li>If the gel is a low-percentage (&lt;0.5%) gel, first pour a 1% support
base gel without wells, and let harden;  then pour the correct gel on
top of the base gel.  The final gel with or without a higher percentage
base should be about 3-5 mm thick.   Do not pour hot gel on
plastic molds, since the plastic will warp or craze.  Eliminate air
bubbles by poking with the corner of a Kimwipe.
<li>Permit the gel to set (about 30-45 min).  Pour a small amount of tank
buffer on top of the gel, and remove the comb.  Pour the rest in the tank
and remove the tape.  Add enough to cover the gel about 1 mm deep
above the gel.
<li>DNA samples should be mixed with 0.2 volumes of 6&times; loading buffer.
Load the samples and avoid overloading.  The pipet tip can be rinsed rather
than a new tip for each, but if doing Southerns or gel purifications, it
is probably wise to use new tips with each sample.
<li>The gel is run 1-5 V/cm (the distance is actually the distance between
electrodes, not ends of the gel).  Ethidium bromide actually migrates
towards the cathode.  Over long periods of time, it may not light up the
lower ends.  Re-soak the gel in dye (0.5 &micro;g/ml in buffer or water)
in that case.
<li>At the end of the run, stain the gel if it does not contain ethidium
bromide (in 0.5 &micro;g/ml of buffer or water) for 30-45 min.
Alternatively 1 to 10,000 dilution of SYBR-Gold can be used.
</ol>
<p>
The same gel can effectively be re-used so long as it maintains integrity.
The only thing necessary is to run the gel overnight to run all the
bands off.  Over time however, a well may become contaminated with
precipitates and affect the run.

<h4>Visualization of Bands in Gels</h4>
<p>
Ethidium bromide is the most commonly used intercalating dye to illuminate 
double-stranded DNA.  Saturated in solution at high ionic strength, one molecule 
binds for about every 2.5 bp.  The dye is naturally fluorescent in solution, but 
its yield is enhanced when stacked in between the bases. 254 nm light is 
absorbed by the bases, which fluoresces at 302 and 366 nm, within the excitation 
spectrum of EtBr.  EtBr emits at 590 nm, within a red-orange region.
<p>
EtBr can bind to single-stranded DNA and RNA, but the number of molecules bound 
is much less and so is the fluorescence.  Indeed it is believed that EtBr 
association with ss DNA or RNA is because of intrastrand duplex formation, and 
EtBr associates with the duplex.  The fluorescence yield is about 20-30 times 
greater when the dye is bound to ds DNA as when it is not bound, and as little 
as 10 ng can be detected in 0.5 &micro;g/ml EtBr.
<p>
EtBr stock is prepared 10 mg/ml in water and then stored at room temp
in a dark bottle or wrapped in foil.  Dye is incorporated in gels and/or
buffers at 0.5 &micro;g/ml.  EtBr inhibits polymerization of acrylamide,
so cannot be included in casting reactions;  gels must be soaked after
running.
<p>
ds DNA mobility is decreased by about 15% with the dye bound to it.
Although there is an advantage to being able to monitor gels during the
run, bands are sharper without the dye present.  If size is important
to know, such as doing digest maps, then the gel should be stained until
after its run.  Sometimes the appearance of a band that is present
in small amounts (&lt;10 ng) can actually be enhanced by destaining
the gel to remove background fluorescence:  use water or 1 mM MgSO<sub>4</sub>
for about 20 min.
<p>
SYBR Gold is an ultrasensitive dye whose fluorescence yield is more than
3 orders of magnitude and can detect &lt; 20 pg ds DNA, 100 pg ss DNA,
and 300 pg RNA.  Its excitation peak is 495 nm with a secondary peak at
300 nm.  Emission occurs at 537 nm.  The molecule is an unsymmetrical
cyanine dye.  It comes as a 10,000&times; stock.

<h5>Photography</h5>
<p>
Transilluminators provide light that generally emits at 302 nm.  This is
a compromise for illumination at 254 nm, where the fluorescence is
greater but nicking of DNA is also greater.  Photographic systems making
use of film use Polaroid Type 57 or 667 (ASA 3000), with a Wratten 22A
(red/orange) filter, with good lens (<i>f</i> = 135 mm).

<h3 id="pulsedfield">Pulsed Field Gel Electrophoresis</h3>
<p>
DNA molecules below a critical length show electrophoretic mobility based on 
size (bp length).  Above it, mobility is a function of the electric field 
strength:  a constant field strength will not achieve separation. Pulsed field 
GE allows separation by varying the electric field direction and magnitude over 
time (periods from 0.1 to 1000 seconds).  Shorter DNA molecules can change 
direction faster than larger, so they separate/fractionate from each other. 
Another name for PFGE is AFE, alternating field electrophoresis.
<p>
<b>Vertical pulsed field electrophoresis</b> or <b>transverse alternating
field electrophoresis</b> (TAFE) is a 3-D variation of OFAGE (<b>orthogonal
field agarose gel electrophoresis</b>, the original name), in which the gel
is vertical and supported by thin plastic strips and the buoyancy of buffer.
Electrodes (cathodes) at the top are placed closed to the gel while the
anodes at the bottom are far removed from the gel.  Field alternation is
110&deg; at the wells (top of gel) but the angle is much greater at the other
end of the gel.  Sharp bands are obtained because of the field gradient
produced by this geometry.
<p>
<b>Field inversion gel electrophoresis</b> (FIGE) is a standard horizontal
bed with one set of electrodes whose field is merely reversed.  To move in
a net direction along the gel, either time or strength of forward field must
be greater than the inverse field strength or time.
<p>
One system makes use of a <b>rotating gel</b> platform, in which the gel
is rotated at an angle to electrodes rather than using multiple electrode sets.
<p>
<b>Counter-clamped homogeneous electric field</b> (CHEF) systems make use of
a hexagonal array of point electrodes in a voltage divider circuit which
encircle the gel.  This produces fields at 120&deg; to the gel.
<p>
Of the factors that affect PFGE, a plot of the size of the DNA fragment against
its relative mobility relative to a marker show that mobility is linear
with changes in size and <em>not</em> the <em>logarithm</em> of the size.
Longer pulse times allow the resolution of larger fragments whereas shorter
pulse times are better for smaller.
<p>
Field strengths can also be varied:  the higher the field strength the more
likely one will resolve larger fragments.  The effect of field strength
has a lower weight than does pulse time.  With longer pulse times, field
strength should be lowered to reduce the effect of trapping.
<p>
Angle between fields can also be varied.  Generally a 110-120&deg; geometry
is employed.  Another parameter is temperature.  Although DNA is more mobile
with higher temperature, the smaller fragments can become trapped and this
leads to band broadening.  Better results are seen with temperatures of
4-15&deg;.

<h4>DNA Sample Preparation</h4>
<p>
DNA cannot be prepared in the usual way since it will be sheared to lengths
too small to require PFGE.  Instead, living cells are embedded in agarose, and
in numbers that keep the DNA load (1-2 &micro;g) much smaller than is typically
loaded (3-20 &micro;g).
<p>
In addition, bp size markers must be prepared especially.  &lambda; DNA (about 
50 kb long) can be ligated to produce concatamers (monomer, dimer, etc). 
Dissolve 0.1 g LMT agarose in 10 ml of 0.1 M TrisHCl pH 7.6 + 20 mM 
MgCl<sub>2</sub> in boiling water bath (or in microwave) and cool to 37&deg;. 
Dissolve 10 &micro;g purified &lambda; DNA in 172.5 &micro;l TE pH 7.6 and heat 
to 56&deg; for 5 min to denature phage cohesive termini.  Cool this to 37&deg; 
and add in this order:  62.5 &micro;l 8% PEG 8000, 5 &micro;l 20 mM 
MgCl<sub>2</sub>, 5 &micro;l 100 mM ATP, 5 &micro;l 1 M DTT, 0.5 Weiss units of 
T4 DNA ligase, and 250 &micro;l of 1% LTM agarose produced earlier.  Draw the 
mix into short length of Tygon tubing and chill until agarose is set.  Blow the 
agarose into sterile disposable plastic tube with 3 vols 1 &times; of the 
ligation buffer containing PEG.  Incubate the plug in buffer for 24 h at RT, 
then transfer to tube with 10 vol 20 mM EDTA pH 8.0. Incubate 1 h, transfer to 
another tube with 10 vol 20 mM EDTA and store 4&deg; until ready to use.
<p>
Other markes include <i>S. cerevisiae</i> with 16 chromosomes ranging from 200 
kb to &gt; 2000 kb.  Prepare cells in agarose rather than usual. In addition 
<i>S. pombe</i> has useful chromosomes at 3.5, 4.6 and 5.7 Mb.
<p>
For preparing cells and tissues in agarose plugs, wash thoroughly in PBS
(generally ice-cold).  Tissue should be diced to 1-2 mm<sup>3</sup> cubes
and then homogenized in Dounce, filtered through cheesecloth to remove
connective tissue.  Frozen tissue can be ground to fine powder in liquid
nitrogen, suspended in PBS, filtered if necessary.
<ol>
<li>Prepare a 1% low-melting agarose in L buffer:  10 mM TrisHCl pH 7.6, 20 mM 
NaCl, 100 mM EDTA (normally stored at 4&deg;).  Cool to 42&deg;
<li>Warm the cells/tissue to the same temperature.  Mix with an equal vol
of the 1% LMT agarose in L buffer.
<li>Stir with a flame-sealed Pasteur pipette to mix cells and agarose.
<li>Pipette or pour molten mixture into preformed Plexiglas molds of
	50-100&micro;l volume (see BioRad or Pharmacia), or draw into Tygon
	tubing of 1/8-inch (3.2 mm) diameter, or into 1 ml plastic syringe barrel.
	Hold for 15 min at RT, then place at 4&deg; for 15-30 min.
<li>When set, collect plugs from Plexiglas molds or blow out from tubing
	or syringe barrel into Petri dish.  Cut plugs into 1 cm sections.
	Each of these is about 45 &micro;l agarose with 500,000 cells or
	2.5 &micro;g DNA.
<li>Transfer plugs to 3 vol L buffer which contains 0.1 mg/ml proteinase
	K and 1% Sarkosyl.  Incubate 3 h at 50&deg;.  Replace the digestion
	mixture with fresh mixture and hold another 12-16 h at 50&deg;.
<li>Incubate plugs in 50 vol TE pH 7.6 room temp with 3-5 changes of buffer
	over 3 h.
<li>Now replace with 2 vol TE pH 7.6 containing 40 &micro;g/ml PMSF and
	incubate 30 min at 50&deg;.
<li>Again incubate with 50 vol TE pH 7.6 for 3 h with 3-5 buffer changes.
	Store at 4&deg;.  For shipping or long-term storage, store in 0.5 M EDTA
	pH 8.0 at 4&deg;.
</ol>
<p>
Yeast DNA requires an alternate method that includes use of zymolyase 5000
or lyticase to digest the cell wall.  Also agarose plugs can be treated
with restriction enzymes prior to PFGE:  the plugs should be incubated
in the correct buffer for enzyme activity, then incubated with 20-30 U enzyme
at optimal temperature.
<p>
If recovery of the DNA after PFGE is desired, perform the PGFE but create two 
lanes: one analytical and one preparative for the same sample. The analytical 
lane is cut out and stained with EtBr or SYBR Gold (30 min RT) and destain if 
necessary (30 min water).  Make notches or slices in the gel to indicate DNA of 
interest.  Re-assemble the gel, and then cut out slices of the preparative lane 
corresponding the DNA seen in the analytical lane.  Transfer each to conical 
tube.  This can be stored at 4&deg; so long as it does not go dry.  Incubate the 
slice in the buffer used for agarase digestion; prepare buffer according to 
manufacturer directions or use what is supplied (although supplementing with 100 
mM NaCl, 30 &micro;M spermine, and 70 &micro;M spermidine appears to enhance 
recovery). This usually takes 1 h with some agitation.  Repeat this twice to 
ensure buffer exchange.  Melt the agarose at 65-68&deg; and swirl while raising 
the temperature.  Add the agarase to melted gel and digest according to 
manufacturer instructions; digestion is done usually from 37-45&deg; from 1 h to 
overnight. The agarase is inactivated by heat or alternatively by using 
phenol:CHCl<sub>3</sub> (1:1) extraction; prefer heat inactivation since DNA is 
less likely to be sheared.  Ligation or restriction enzyme can be done in 
presence of agarose monomers, but if clean up is desired, add pH 5.2 sodium 
acetate to 0.3 M and then 2 vols isopropanol to ppt DNA.
<p>
If DNA is to be injected or transfected into cells (fertilized mouse eggs or 
mouse embryonic stem cells), it might be useful to transfer from low-melting 
agarose slice into a high percentage agarose gel by electrophoresis, then obtain 
by agarase digestion.

<h3>Recovery from Gels</h3>
<p>
Methods are constantly published which discuss the recovery of DNA from gels. 
DNA recovered from gels is often difficult to do further enzyme (ligation, 
digestion, labeling) work because of charged polysaccharide inhibitors present. 
Yield can be a problem, although bands &lt; 5 kb can be obtained with &gt; 50% 
yield.  Yields are also a problem when the DNA species is present in small 
amounts;  if a band is estimated to be &lt; 500 ng, it's probably not worth 
attempting to recover it all.  There is also a limitation on the number of 
fragments that can be recovered, since the effort to recover one alone is 
enormous.

<h4>Recovery onto DEAE-Cellulose Membranes</h4>
<p>
A well is cut into the gel just ahead (on the anodic side) of the band. 
DEAE-cellulose membrane cut to the width of the well is placed within. If DNA 
were eluted from polyacrylamide gel, the band would be sliced from the gel, 
placed within a well cut in agarose, and the membrane placed on the anodic side. 
Electrophoresis is continued until the band is eluted.  The membrane is removed 
and washed in low ionic strength buffer, then the DNA desorbed using high ionic 
strength buffer.  The DNA is pure enough for the most difficult techniques: 
e.g., transforming mammalian cells.
<p>
The first attempt to elute into cut wells involved dialysis membrane,
and then Dretzen et al. (1981) used DEAE paper.  However, very large
fragments (&gt; ~10 kb) pass through the membrane and the method is
not suitable for recovery of large bands.  ss DNA will bind nearly
irreversibly to the membrane and this method cannot be used to recover it.
<ol>
<li>In planning a digestion, make sure the band of interest is
generated in a sufficiently worthwhile amount (&gt; 100 ng).
Include 0.5 &micro;g/ml ethidium bromide in the gel.   Illuminate the gel
with a hand-held UV lamp having a 302 nm emission.
<li>Make an incision at the leading edge using scalpel or razor blade.
The incision should be ~2 mm wider than the band.  If all the DNA is
to be eluted, make a long incision <i>parallel</i>
to the lane.
<div class="inset">
Note: this is not the same as cutting a well!  It is merely an incision.
</div>
<li>Handling with gloves, cut DEAE-cellulose membrane as wide as the
well and about 1 mm deeper.  The membrane is soaked in 10 mM EDTA pH 8
for 5 min, then <q>activated</q> in 0.5 N NaOH for 5 min.  The membrane
is washed six times in water.  Activated strips can be stored at 4&deg;
in sterile water for several weeks.
<li>Blunt-end forceps or tweezers hold apart the slit in the gel well
while the membrane is inserted.  Do not trap air bubbles.
<div class="inset">
Various strategies have been used to ensure that the DNA captured on
the membrane is only the DNA of interest.  One is to cut out the band
and place it within another region of the gel with no bands and perform
the membrane capture.  Another is to make an incision at the band's
trailing edge and also insert a membrane to trap DNA that might also
elute on to the membrane containing the DNA wanted.
</div>
<li>Start the electrophoresis (5 V/cm) and monitor the elution with
a hand-held lamp.  Perform it for only the minimal time to prevent
contamination with other bands or of components of the agarose.
<li>Recover the membrane with the forceps and rinse in 5-10 ml of
low ionic strength wash (50 mM TrisHCl pH 8.0, 0.15 M NaCl, 10 mM EDTA).
Never let the membrane dry or the DNA may become irreversibly bound.
<li>Place the membrane in a microfuge tube.  Add enough high salt
elution buffer (50 mM TrisHCl pH 8.0, 1 M NaCl, 10 mM EDTA).  Incubate
the membrane in buffer for 30 min at 65&deg;.
<li>Photograph the gel again to indicate the bands isolated, while the
DEAE membrane is cooking.
<li>Transfer the fluid from the tube to another microfuge tube, and add
a second wash to the tube with the membrane.  Incubate another 15 min
at 65&deg;  Note that the hand-held lamp can be used to inspect the
membrane for the band.
<li>Add an equal volume of phenol:chloroform to extract.  Transfer
aqueous phase to new tube, add 0.2 vol 10 M ammonium acetate and then
2 vol of 4&deg; ethanol.  Keep at room temp 10 min, then spin max speed
10 min at room temp.  Rinse pellet with 70% ethanol, then let the tube
dry for a few minutes.  Redissolved in 3-5 &micro;l TE pH 8.
<div class="inset">
It might be useful, even necessary to improve the recovery of DNA by
adding carrier RNA (10 &micro;g).  However any addition must not interfere
or complicate the use of the DNA in subsequent enzymatic reactions.
To prepare carrier, obtain yeast tRNA and dissovle 10 mg/ml in TE pH 7.6
with 0.1 M NaCl.  Extract twice with phenol (equilibrated in TrisHCl pH 7.6),
and then twice in chloroform.  Precipitate in 2.5 vol of room temp ethanol,
and spin at 5000 <i>g</i> for 15 min at 4&deg;  Dissolve to 10 mg/ml
in pH 7.6 TE and store in aliquots at &minus;20&deg;.
</div>
<li>If the DNA is required for some reactions (microinjection into
fertilized mouse eggs or electroporation into cultured cells), stricter
purification may be necessary:
 <ol>
 <li>Suspend the DNA in 200 &micro;l TE pH 8.0.  Add 25 &micro;l of 3 M sodium
 acetate, pH 5.2.  Precipitate with 2 vol refrigerated ethanol.
 <li>Spin at maximum speed for 5-15 min at 4&deg;
 <li>Rinse pellet with 70% ethanol.  Let ethanol evaporate from tube,
 then resuspend in 3-5 &micro;l TE pH 8.0
 </ol>
<li>Inspect the quality of the DNA on gel.  Suspend 10-50 ng of material
in 10 &micro;l TE pH 8.0 and add 2 &micro;l gel loading buffer.
</ol>

<h4>Recovery by Electroelution into Dialysis Bags of Bands from Agarose
and Polyacrylamide</h4>
<p>
When other methods fail or prove difficult, this should be the next
one tried.  It is not efficient if a large number DNA bands are to be
recovered.
<ol>
<li>The digest should produce bands of interest that amount to 100 ng
or more.  Run the gel and stain the bands as usual.  Photograph the gel.
<li>The gel containing the band is sliced out with scalpel or razor
blade, and placed on Parafilm wetted with 0.25&times; TBE.  The smallest
slice possible should be cut.
<li>Photograph the gel with slice cut out to establish the band that
was removed.
<li>Prepare dialysis tubing that is molecular biology grade; soak in
0.25&times; TBD.  Tie one end with secure knot.  Fill the tubing to
overflowing with buffer, then pass the slice into the tubing.
<li>Let the gel slice settle, and squeeze out the excess buffer to
a minimum, but not so much that the slice is not in constant contact with
buffer.
<li>Close with dialysis tubing clip without trapping air.
<li>Place the bag on the shallow middle of a horizontal gel
electrophoresis tank.  To prevent the tubing from floating, place
a pipet or glass rod on it to weigh it down.  Turn the power on
to 7.5 V/cm for 45-60 min.  A UV (302 nm) lamp can be used to monitor
movement out of the gel slice.  It is important to run it long enough,
but running it too long risk irreversible binding of the DNA to the
tubing.  The factors that affect the time of elution include the buffer,
size of DNA, and agarose concentration.
<li>At the end, the polarity is reversed for about 20 seconds to release
DNA that may be on the wall.  Recover the bag and massage gently to mix
the DNA.
<li>Carefully recover the buffer to a collection tube after opening the
clip.  You might want to recover it using a Pasteur pipet. Do not damage
the gel slice within.  Add a small amount of 0.25&times;
TBE to the bag as a wash, and use it to rinse the tubing within, then add
it to the.  Keep the tubing in the buffer and wet for the time being.
<li>Recover the gel slice and stain it for 30-45 min in 0.5 &micro;g/ml
ethidium bromide in 0.25&times; TBE.  Look at the slice by illumination
to verify recovery.  If adequately removed, then discard the tubing.
<li>The DNA can now be purified using <a href="#deae-dna">DEAE-Sephacel</a>,
chromatography, or by phenol:chloroform with EtOH pptn.
</ol>

<h4>Recovery of DNA from Low Melting-Temperature Agarose Gels:
Organic Extraction</h4>
<p>
Hydroxyethylated agarose reduces the melting temperature of agarose.
LMT agarose at 37&deg; can maintain a liquid state (and melt) so that
it becomes part of the reaction.  LMT agaroses are designed in several
ways:  not to inhibit restriction enzymes or ligases, to be free of DNases
and RNases, and to give minimal fluorescence with ethidium bromide.
<ol>
<li>Prepare the LMT agarose at the appropriate concentration in
1&times; TAE.  TAE is a preferred buffer to TBE because borate
can inhibit ligations and also interfere with DNA purification on glass
beads.  It is optional to use 0.5 &micro;g/ml ethidium bromide cast in the
gel or to stain after.  To ensure gel setting, the gel can be placed at
4&deg; as well.
<li>Prepare the DNA samples, load and run at 3-6 V/cm.  The voltage
is slightly lower than for standard agarose gels because DNA migrates
a bit faster in LMT agarose.
<li>After imaging the gel and then locating the band by UV illumination
(nothing less than 302 nm), use scalpel or razor to slice out the gel
and transfer to plastic tube.
<li>Re-image the gel to document the band excised.  Add about 5 vols
of LMT elution buffer to the tube with the slice.  LMT elution buffer
is 20 mM TrisHCl pH 8.0 and 1 mM EDTA.  Close the top of the tube
and then place tube in 65&deg; water bath for about 5 min.
<li>Cool the solution to room temp, then add an equal volume of
equilibrated phenol.  Vortex 20 sec, then spin out the aqueous phase
at 4000 <i>g</i> for 10 min (20&deg;).
<li>The aqueous phase is then extracted with phenol:chloroform then
chloroform.
<li>Add 0.2 vol of 10 M ammonium acetate and then 2 vol. absolute
ethanol (chilled to 4&deg;).  Hold at room temperature for 10 min,
then spin out (5000 <i>g</i> for 20 min at 4&deg;).
<li>Wash in 70% ethanol and dissolve in TE pH 8.0 after it evaporates.
</ol>
<p>
The DNA is suitable for most reactions in cloning.
<p>
As an alternative the LMT agarose gel slice with DNA can be melted in the 
presence of glass beads or powder in the presence of high ionic strength buffer. 
After washing, the DNA is removed in a low salt buffer.
<ol>
<li>To the gel slice in the tube, add ~3-5 vols of 6 M sodium iodide
solution.  (Add 0.75 g Na<sub>2</sub>SO<sub>3</sub> to 40 ml water,
then 45 g sodium iodide and stir to dissovle.  Filter through Whatman
or nitrocellulose and store out of light.  Should be good for 2-3 months;
discard if preciptitate found).
<li>Place at 55&deg; for the minimal amount of time to melt the agarose,
and no longer.
<li>Add 5 &micro;l acid-washed glass beads (Sigma) if DNA &le; 5 &micro;g.
Add 2 &micro;l/&micro;g washed glass beads for DNA &gt; 5 &micro;g.  Incubate
5 min at room temp shaking occasionally.
<li>Spin the mixture at max speed for 5 seconds and then remove and
discard the supernatant.
<li>Wash the pellet with 3 &times; 0.5 ml of a wash solution of
20 mM Tris pH 7.4, 1 mM EDTA, 100 mM NaCl containing an equal volume
of ethanol (store 3-4 months at 0&deg;).
<li>Resuspend in TE pH 8.0 at about 0.5 &micro;g/&micro;l.
<li>Incubate for 3 min at 45&deg; to elute the DNA.
<li>Spin for 1 min, then transfer supernatant to new tube, and store 4 &deg;.
</ol>

<h4>Recovery of DNA from Low Melting-Temperature Agarose Gels:
Digestion of the Gel</h4>
<p>
The gel slice is placed in a digestion mixture of agarase and then 
phenol-extracted and ethanol-precipitated.  This method might be preferable for 
high MW DNAs being obtained after pulsed-field electrophoresis as it is so 
gentle.  Nonetheless the method is suitable for all kinds of DNA.  Several 
manufacturers make agarase (e.g., GELase from Epicenter Technologies, and 
&beta;-Agarase I forms from NEB and Calbiochem).
<ol>
<li>Perform the electrophoresis as usual <em>using a low-melting agarose</em>
at the appropriate concentration.  Illuminate and get an image of the gel.
<li>Cut out the gel portion containing the band and incubate in a tube
for 30 min at room temp in 20 vols of 10 mM Bis-Tris HCl, pH 6.5,
5 mM EDTA, 100 mM NaCl (equilibration buffer).
<li>While the slice(s) are incubating, get a new image of the gel to
record the portions excised.
<li>Remove the equilibration buffer, and add a volume of fresh equilibration
buffer that is about equal in volume to the gel slice.
<li>Melt the gel slice by placing in 65&deg; water bath for 10 min.
Cool to 40&deg;.  Then add DNase-free agarase, about 1-2 U per gel slice
of 200 &micro;l.  Leave for 1 hour.
</ol>
<p>
After the digestion the DNA is of suitable quality to do ligations,
restriction enzyme digests, and transformations.  Further purification
and concentration can be done as follows.

<h5>Purifying Small (&lt; 20 kb) DNA Fragments</h5>
<ol style="list-style-type:lower-alpha;">
<li>Extract twice with equilibrated phenol
<li>In fresh tube, add 2 vol TE pH 8.0 to aqueous phase
<li>Add 0.05 vol of 5 M NaCl then 2 vol ethanol.  Hold for 15 min at
0&deg;, and then spin for 15 min in a 4&deg; microfuge.
<li>Wash pellet with room temp 70% ethanol (0.5 ml).  Vortex, then spin
down 15 min at 4&deg;.
<li>Remove the supernatant, and allow to evaporate for a few minutes at
room temp.  Dissolve in minimal volume of TE pH 8.0.
</ol>
<h5>Purifying Large (&gt; 20 kb) DNA Fragments</h5>
The sample is essentially dialyzed.  Move it to a bag of boiled dialysis
tubing of molecular biology grade.  Dialyze against 100 ml TE pH 8.0
for several hours at 4&deg;.
<p>
Alternatively use a microdialysis (drop-dialysis) system
(e.g., Life Technologies).
<p>
Do not use phenol extraction, vortexing, or ethanol precipitation since
this shears the DNA.
<p>
Addition of spermine to 30 &mu;M and spermidine to 70 &mu;M to the dialysis
buffer can enhance recovery of large DNA fragments (&gt; 15 kb).

<h4>Gel Extraction Kits</h4>
<p>
Several manufacturers make kits for extracting DNA and RNA from agarose gels,
and some even have several different methods.  The principles and yields and
applications are shown in the table below.
<table>
<tr><th>Product    <th>Principle    <th>Uses      <th>Application
<tr><td>Qiagen QIAEX II Gel Extraction Kit
 <td>
 <td><ul><li>40 bp to 50,000 bp
 <li>agarose or polyacrylamide (TBE or TAE)
 <li>yield: 10 ng to 10 &micro;g
 <li>final volume: 20 &micro;l </ul>
 <td>
<tr><td>QIAquick Gel Extraction Kit
 <td>spin columns
 <td><ul><li>70 bp to 10,000 bp
 <li>agarose or polyacrylamide (TBE or TAE)
 <li>yield: 10 ng to 10 &micro;g
 <li>final volume: 20 &micro;l </ul>
 <td>
<tr><td>MinElute Gel Extraction Kit
 <td>spin columns
 <td><ul><li>70 bp to 4000 bp
 <li>agarose or polyacrylamide (TBE or TAE)
 <li>yield: 10 ng to 10 &micro;g
 <li>final volume: 20 &micro;l </ul>
 <td>
</table>

<h3>Alkaline Agarose Gel Electrophoresis</h3>
<p>
At high pH thymidine and guanine lose a proton and base pairing in disrupted.
Single-stranded DNA migrates through gels on the basis of size just as
does other DNA.  Urea and formamide tend to make agarose rubbery and
are not used as denaturants.
<p>
These gels would be useful for assessing the size of first and second
strands of synthesized cDNAs during reverse transcription.   They are
also useful for looking at sizes of strands after digestion of DNA-RNA
hybrids with nuclease S1.
<p>
In the past, alkaline gels were used when commercial enzyme quality
was questionable:  it could be used to see if enzymes were nicking the
enzyme and to evaluate reagents involved in nick translation.
<p>
Bromocresol green and xylene cyanole are used for the tracking dyes.
Prepare 6&times; loading buffer it as follows:
<ul>
<li>300 mM NaOH
<li>6 mM EDTA
<li>18% (w/v) Ficoll Type 400
<li>0.15% (w/v) bromocresol green
<li>0.25% (w/v) xylene cyanole
</ul>
<p>
Prepare 10&times; alkaline gel buffer as 0.5 M NaOH and 10 mM EDTA.
It should be made fresh.  Prepare just enough to make the gel and the
run buffer.
<ol>
<li>Agarose is dissolved in water in water in a flask or glass bottle.
The volume of water used should be in 0.91 vol (91%) of the final volume.
Microwave or boil the agarose as usual, but only for the minimum time.
<li>The solution is cooled to 55&deg;.  Add 0.1 vol (or 0.09 or 9% of
the final volume) of 10 &times; alkaline gel buffer concentrate.
It is essential to wait for the gel to cool since the hydroxide degrades
agarose and will do it rapidly at higher temperature.  Do not add
ethidium bromide since its binding is impaired at high pH.
<li>Precipitate the DNA in ethanol using standard methods.  Resuspend
the precipitates in 10-20 &micro;l 1 &times; gel buffer (not alkaline);
if the original samples were small volumes, add 0.5 M EDTA until the
concentration is about 10 mM.  Add 0.2 vol of 6&times; alkaline loading
buffer.  The EDTA is essential for chelating Mg<sup>2+</sup>, which can
precipitate under high pH conditions.
<li>Load and run the gels.  The voltage used should be much lower than
for standard gels since the current runs higher.  Use &lt; 3.5 V/cm.
Note that it is not strictly necessary to alkalinize the DNA in the load
tube, since it will denature in the gel too.
<li>After the tracking dye has entered the gel ~0.5-1 cm, turn off
the system and place a glass plate on top of the gel.  Continue running
until the dye moves about two-thirds of the distance of the gel.
The reason the glass plate is used is because the gel can detach from
the casting surface and will float, and the bromocresol green dye can
migrate out of the gel.
</ol>
<p>
If the gel is to be used for Southern hybridization, soak the gel
in neutralization buffer for 45 min at room temperature, and then
transfer to nitrocellulose or nylon membrane  (if a charged nylon
membrane is to be used, the neutralization step can be omitted). Then
detect target sequences of immobilized DNA with labeled probe.
<p>
If the gel is to be stained, soak the gel 45 min in neutralization
buffer, then stain with 0.5 &micro;g/ml ethidium bromide in 1&times; TAE.
<p>
If the gel is to be autoradiographed because <sup>32</sup>P-labeled DNA
was separated, then the exposure can be done on wet or dry gels.
Drying the gel should be done if there is an issue with the sharpness
of the band or the DNA is low in quantity (drying improves sharpness and
low DNA may not be detected).
<ol>
<li>For wet gels:
 <ol>
 <li>Place gel on glass plate
 <li>Insert into heat-sealable (or sealable) plastic bag.
   Make sure all the excess air is removed from the bag, then heat-seal.
   Alternatively wrap well in Saran Wrap, but note that fluids
   can leak out of it.
 <li>Mark adhesive labels either with radioactive ink or luminescent
 markers, then apply labels to the plastic bag/wrap.  Use cellophane tape
 to then cover the ink/marker since it can contaminate film or intensifying
 screens.
 <li>In the darkroom, tape the gel to the x-ray film cut to the size of
 the glass plate.  Wrap light-tight in aluminum foil.  Metal casettes cannot
 be used since it crushes the gel. and or break the glass plate.
 <li>Expose the film for the appropriate amount of time.
 </ol>
</ol>

<!-- ===========================================================
   southerns
 =============================================================== -->

<h2 id="southern">Southern Hybridization</h2>
<p>
The method below is from Sambrook, J, Fritsch, EF, Maniatis, T (1989)
<i>Molecular Cloning, A Laboratory Manual</i>, 2nd Ed., pp. 9.31ff.
<p>
Genomic DNA is usually digested with restriction enzymes (one or more) and then 
the fragments separated on agarose gel electrophoresis. DNA is then denatured, 
then transferred to a membrane filter of nitrocellulose or derivatized nylon 
(Zeta-Probe is one trademark of cationized nylon).  Single-stranded nucleic acid 
binds to the filter at the position in which it migrated in the gel.  This is 
then incubated with a solution containing a <b>labeled probe</b> of 
polynucleotides (usually an oligonucleotide) under conditions in which the probe 
hybridizes to the single-stranded polynucleotide bound to the membrane.  The 
labeling is often a radioisotope of phosphate (<sup>32</sup>P), and so when the 
hybridization is done, the membrane is allowed to dry and then it can be placed 
on x-ray film for autoradiography.  Alternatively, newer labeling techniques 
attach a fluorescent marker, and so the membrane might be illuminated with a 
shortwave UV lamp.  The use of digitoxigenin (Dg)-labeled probes allows the 
development of the band using antibodies to digitoxigenin, with illumination by 
peroxidase coupled to antibodies or ligands that can bind to antibodies.  As 
little as 0.1 pg DNA can be detected for autoradiography done over several days 
with a probe whose specific radioactivity of <sup>32</sup>P is 
&gt;10<sup>9</sup> cpm/&micro;g.
<p>
The transfer of polynucleotide from gel to membrane can occur in one of three 
ways.  Southern originally described a <b>capillary</b> transfer in which 
Whatmann 3MM paper is placed on a support and its ends allowed to be wetted into 
a solution of transfer buffer.  The gel is then placed on the filter paper.  The 
wetted membrane is then placed on top of the gel. Another sheet of Whatmann 3MM 
is then placed on top of the membrane. Then several paper towels are overlayed 
atop the Whatmann paper.  A glass plate is then placed on top of the towels, and 
then a weight (about 500 g) is placed on top of the glass.  The transfer rate is 
dependent upon many factors:  concentration of agarose in gel, the size of the 
DNA being transferred.  DNA &lt; 1 kb is transferred quantitatively from a 0.7% 
gel within an hour.  Transfers of polymers &gt;15 kb require at least 18 hours. 
Partial DNA breakage can assist the transfer. Weak acid followed by strong base 
treatment can break the DNA at purine sites.
<p>
Another means is by electrophoresis, but since binding of nucleic acid to 
membrane requires high ionic strengths, use of electrophoresis can overheat a 
transfer process.  Some membranes such as diazobenzyloxymethyl (DBM) or 
<i>o</i>-aminophenylthioether (APT)-cellulose will bind DNA at lower ionic 
strengths, but these are not really used.  Charged nylon will also work.

<ol>
<li>DNA is digested with one or more restriction enzymes.  About 10 &micro;g
is loaded into each gel lane for detection of single-copy sequences
using standard size probes (&gt;500 bp) labeled to high specific
radioactivity (&gt;10<sup>9</sup> cpm/&micro;g).  30-50 &micro;g should be used
when oligonucleotides are used as probes.  Note that during the digestion,
it is useful to stir the reaction mixture because of solution inhomogeneities.
<li>Concentrated gel-loading buffer is added, and this is loaded to a
the gel.  For genomic DNA, a 0.7% gel in 0.5 &times; TBE and containing
0.5 &micro;g/ml ethidium bromide is typical.  Electrophoresis is done
slowly (&lt;1 V/cm) so that smearing is not a problem.
<li>The gel is transferred to a baking dish and trimmed with razor to
remove unused areas.  Nick the bottom of the left-hand corner for
purposes of orientation.
<li>Soak the gel for 45 min in several volumes of 1.5 M NaCl, 0.5 N NaOH,
keeping the gel rotating slowly on a platform.
</ol>

<!-- ===========================================================
   repetitive DNA analysis
 =============================================================== -->

<h2 id="cot1dna">Preparing Highly Repetitive <i>C</i><sub>0</sub><i>t</i> = 1 
DNA</h2>
<p>
In situ hybridizations with DNA probes typically require nonspecific
as well as competitor DNA to guard against high backgrounds (false positives).
The nonspecific DNA is typically DNA from another species;  DNA from
calf thymus, salmon or herring sperm, human placenta is used.
<p>
Competitor DNA is the highly repetitive DNA fragments from the same species 
however. This is called <i>C</i><sub>0</sub><i>t</i> = 1 (or <q>Cot-1</q>) 
because it refers to a technique of molecular geneticists in which DNA at a 
definite concentration (which is <i>C</i><sub>0</sub>) is denatured and then 
allowed to re-anneal over a particular time (which is <i>t</i>).  Thus Cot-1 
refers to the product of the initial DNA concentration and the time allowed for 
re-annealing, the product value being 1. The re-annealing process can be 
monitored spectrophotometrically at &lambda;=260 nm.  As re-annealing occurs, a 
plot of A<sub>260</sub> vs. time shows a series of increases in A<sub>260</sub> 
as a series of S-shaped curves with plateaus.  Once a plateau has been reached, 
a specific class of DNA has been re-annealed.  Generally highly repetitive 
fragments anneal first, then less repetitive, and then unique DNA is the 
slowest.
<p>
At some point, one can remove the single-stranded, unannealed DNA from the 
double-stranded, annealed DNA.  One can do this enzymatically: the S1 nuclease 
is an enzyme that digests single-, but not double-stranded DNA (at high enzyme 
and low salt concentration, it can introduce breaks into ds DNA, RNA or DNA/RNA 
hybrids).

<h3>Method</h3>
<ol>
<li>Genomic DNA should be sheared to fragments of 400 bp on average.
A smear on the agarose gel should be visible between 300-600 bp.
<li>Denature in polypropylene tube in boiling water bath.
<li>Place DNA in 65&deg; water bath and leave 3-4 minutes to equilibrate.
<li>Add NaCl pre-warmed to 65&deg; to bring to a final 0.3 M concentration.
<li>The DNA should be annealed to a <i>C</i><sub>0</sub><i>t</i> of 1:
the <i>C</i><sub>0</sub><i>t</i> happens to be 5.92 min for 1 mg/ml.
That is, <i>C</i><sub>0</sub><i>t</i> = 5.92 mg-min/ml.  If the DNA is
1 mg/ml, anneal for 5.9 min.  If 0.5 mg/ml, then 11.8 min.
<li>Add an equal volume of 2&times; S1 nuclease buffer (0.5 M NaCl,
 2 mM ZnSO<sub>4</sub>, 60 mM sodium acetate pH 4.2, 10% glycerol).
<li>Add S1 nuclease to 1 U/&micro;g DNA.  1 unit of enzyme at pH 4.6
will convert 1 &micro;g heat-denatured DNA to acid-soluble form
in 1 minute at 37&deg;.
<li>Incubate 37&deg; for 30 min.
<li>Precipitate the ds DNA using standard method : phenol,
  chloroform/isoamyl alcohol, ethanol precipitate in ammonium acetate,
  wash with 70% EtOH, dry, and resuspend in TE.
<li>Determine concentration, aliquot and store.
</ol>

<!-- ==============================================================
     In Vitro Methods
=================================================================== -->
<h1 id="in-vitro"><i>In Vitro</i> Methods to Modify (Copy, Cut, Join) DNA and
RNA</h1>

<h2 id="cloningenzymes">Enzymes Commonly Used To Manipulate DNA 
&amp; RNA</h2>
<p>
These enzymes are commonly used for all <i>in vitro</i> work in the construction 
of plasmids and artificial chromosomes for the purpose of cloning or expression.
But there may be roles for <i>in vitro</i> transcription and translation 
as well.

</div> <!-- class=outerblock -->


<table class="widetable" style="max-width:50em;">
<col style="width:10em;">
<tr><th>Enzyme <th>Activity  <th>Application
<tr><td class="header-cell" colspan="3">Polymerases &amp; Reverse Transcriptases
<tr><td class="description" colspan="3">These enzymes build DNA or RNA
polymers (polynucleotides) naturally and have been exploited and in some
case modified for in vitro work.  These enzymes can fill in single-stranded
portions of double-stranded DNA, thereby making cohesive ends blunt.  They
can also use RNA or DNA template strands and make RNA or DNA, which include the
reverse transcriptases.
<tr>
<td class="enzyme-name"><i>E coli</i> DNA Polymerase&nbsp;I
<td class="enzyme-description">This enzyme is used chiefly to replicate the the 
host chromosome and plasmids of <i>E coli</i> in vivo as part of its biological 
function.  It also has three kinds of exonuclease activity, as part of its 
ability to <q>edit</q> (repair) mismatches or defects in strands:  it is an 
exonuclease in both 5&prime; &rarr; 3&prime; and 3&prime; &rarr; 5&prime; 
directions, and it also is a single-stranded exonuclease in the 3&prime; &rarr; 
5&prime; direction, and so should not be present when cutting with restriction 
enzymes that produce 3&prime; ss ends.  Additionally it has RNase H activity and 
can degrade hybrid RNA-DNA duplex where the RNA existed to prime replicative 
synthesis.  While it was widely used in the first days of molecular cloning 
work, its application is limited to performing nick translation to label DNA and 
second strand synthesis during cDNA generation.
<td class="enzyme-application">
 <ul>
 <li>second strand synthesis during cDNA generation
 <li>nick translation
 </ul>

<tr>
<td class="enzyme-name">Klenow Fragment
<td class="enzyme-description">This form of the <i>E coli</i> DNA polymerase I
enzyme is generated by digestion with the subtilisin protease.  It loses the 
5&prime;&rarr;3&prime; exonuclease activity which usually generates nicks
in ds DNA, but it retains the 3&prime;&rarr;5&prime; exonuclease function
(some modified Klenow fragment products have also eliminated the 3&prime;&rarr;5&prime;
exonclease function too).
<td class="enzyme-application">
 <ul>
 <li>dideoxy sequencing
 <li>second strand synthesis
 <li>filling in (blunting) 5&prime; overhangs
 <li>removing 3&prime; overhangs
 <li>random priming labeling
 <li>second strand synthesis in mutagenesis protocols
 </ul>

<tr>
<td class="enzyme-name"><i>Taq</i> Polymerase
<td class="enzyme-description">The DNA polymerase of <i>Thermus
aquaticus</i> (<i>Taq</i>) which is the basis of the polymerase chain reaction 
(PCR); there are many forms of the <i>Taq</i> polymerase, some of which can be 
used to polymerase very long target sequences but which may have low fidelity in 
base pairing, and others that have high fidelity in base pairing but which are 
slow or can polymerize targets up to a certain size.  Because of the importance 
of this enzyme, <a href="#taq-polymerase">more information</a> about it is given 
below this table.
<td class="enzyme-application">
 <ul>
 <li>polymerase chain reaction (PCR)
 </ul>

<tr>
<td class="enzyme-name">Moloney murine leukemia virus (M-MuLV, MMLV, M-MLV) 
reverse transcriptase
<td class="enzyme-description">This is a RNA/DNA-dependent DNA polymerase (reverse
transcriptase) which synthesizes DNA from a single-stranded RNA or DNA, although
it is generally used as just a reverse transcriptase rather than polymerase.
It also has a RNase H activity in which it can degrade RNA that is in a DNA
hybrid, thereby making room for the complement synthesis.  Modifications of
the M-MuLV enzyme have been done to increase make it function at increased 
temperatures to eliminate mRNA secondary structure which can inhbit the 
ability of enzyme to copy perfectly.
<td class="enzyme-application">
 <ul>
 <li>first strand synthesis for cDNA production or RT-PCR
 <li>second strand synthesis from ss DNA
 </ul>

<tr>
<td class="enzyme-name">Avian myeloblastosis virus (AMV) reverse transcriptase
<td class="enzyme-description">This is a RNA/DNA-dependent DNA polymerase with 
very much the same function and properties as the M-MuLV enzyme.  The AMV
enzyme appears to have a stronger RNase H activity and would be a more suitable
choice for cDNA construction when the mRNA is quite long.
<td class="enzyme-application">
 <ul>
 <li>first strand synthesis for cDNA production or RT-PCR
 <li>RNA sequencing
 </ul>

<tr>
<td class="enzyme-name">T4 DNA Polymerase
<td class="enzyme-description">This polymerase has a strong 3&prime;&rarr;5&prime;
exonclease activity and is the one chosen when high fidelity is required
and gap filling is to be done.  It has no strand displacement activity, and
so cannot replicate ds DNA like <i>E coli</i> polymerase; it only works off
single-stranded templates that are primed.
<td class="enzyme-application">
 <ul>
 <li>gap filling and creating blunt ends
 <li>high fidelity polymerase
 </ul>

<tr>
<td class="enzyme-name">T7 DNA Polymerase
<td class="enzyme-description">This enzyme 
<td class="enzyme-application">
 <ul>
 <li>Dideoxy sequencing
 </ul>

<tr>
<td class="enzyme-name">Terminal Deoxynucleotidyl Transferase (TdT)
<td class="enzyme-description">Usually just called <b>terminal transferase</b>,
it will create a DNA polymer without using any template strand at all, adding
a homopolymeric tail to the 3&prime; end.  It works better with an 3&prime;
overhang, but will also work from a 3&prime; blunt end.  It requires a 
Co<sup>2+</sup> cofactor.  It can work with double or single-stranded DNA.
<td class="enzyme-application">
 <ul>
 <li>Can add modified nucleotides (ddNTP, DIG-dUTP) to label 3&prime; ends
 <li>TUNEL assay (<i>in situ</i> localization of apoptosis
 </ul>


<tr><td class="header-cell" colspan="3">Kinases and Phosphatases
<tr><td class="description" colspan="3">The 5&prime; and 3&prime; ends 
DNA and RNA polynucleotides can have phosphate groups or not (normally the
5&prime; end is phosphorylated, the 3&prime; end is not).  The presence or
absence of a phosphate can enable directional cloning and other manipulations

<tr>
<td class="enzyme-name">(T4) Polynucleotide Kinase
<td class="enzyme-description">A workhorse enzyme that is necessary in every
organism, it is even coded for by bacteriophages (who might just depending on
the enzyme).  It catalyzes an <i>equilibrium</i> reaction, namely
<p style="font-size:83%;text-align:center;color:green;">
<b>molecule-with-5&prime;-OH + ATP &larr;&ndash;&rarr; molelcule-with-5&prime;-PO<sub>4</sub>
+ ADP</b>
<p>
It can serve two functions.  The first is just to use ATP to phosphorylate
the 5&prime; end of double- or single-stranded DNA or RNA, which is necessary if
it is to be annealed and ligated.  The second is to use it to label DNA with
a labeled ATP with a molecule that already has a 5&prime;-phosphate:  in this
case, add unlabeled ADP into the mix with labeled ATP, so that the ADP drives the 
phosphatase (<q>reverse</q>) reaction, and the labeled ATP will then drive the
kinase (<q>forward</q>) reaction too.  This enzyme works on ds and ss DNA and RNA,
and also on dNMP and NMP.  It has phosphatase activity on dNMP/NMP and dNDP/NDP
(just add ADP and no ATP).
<td class="enzyme-application">
 <ul>
 <li>Phosphorylates 5&prime;-hydroxyl ends of DNA or RNA (oligo)nucleotides 
   for ligation
 <li>Can be used to <i>exchange</i> unlabeled phosphates with labeled 
			phosphates in presence of ADP
 <li>At low pH, can dephosphorylate 3&prime;-phosphates from DNA or RNA
 </ul>

<tr>
<td class="enzyme-name">Alkaline Phosphatase (Calf Intestinal)
<td class="enzyme-description">Dephophorylates 5&prime; end of DNA and RNA,
and the phosphates from dNTPs and rNTPs.  Also has protein phosphatase 
activity.
<td class="enzyme-application">
 <ul>
 <li>General enzyme to remove 5&prime; phosphate from DNA and RNA
 <li>Prepares oligos / templates for 5&prime; labeling
 <li>prevents recirculization of cloning vectors and transfections
containing insertionless plasmids
 </ul>

<tr><td class="header-cell" colspan="3">Ligases, Helicases,
Topoisomerases
<tr><td class="description" colspan="3">This is sort of collection of enzymes
with special properties that join nicks in DNA, create nicks and twists 
to make DNA superhelical, and so on.

<tr>
<td class="enzyme-name">T4 DNA Ligase
<td class="enzyme-description">This enzyme will create a phosphodiester bond
between a 5&prime;-phosphate and 3&prime;-hydroxyl for two parts of a strand.
and so it repairs single-stranded nicks.  It can be used to join blunt
and cohesive ends of two double-stranded molecules.  It works only on
double-stranded molecules:  DNA, RNA, and DNA/RNA hybrids.  To limit the
number of unwanted products in blunt-end ligations, one of the substrate
molecules should not have a 5&prime;-phosphate (an end not to be joined).
Do not use the ligase in circumstances where a DNA polymerase serves a
better purpose.
<td class="enzyme-application">
 <ul>
 <li>Joins linkers and adaptors to blunt-ended DNA
 <li>Often used in cloning operations
 </ul>

<tr>
<td class="enzyme-name"><i>Taq</i> DNA Ligase
<td class="enzyme-description">A thermostable form of a DNA ligase.
Will join two oligonucleotides which have annealed to a long complement
but only if the oligonucleotides are adjacent and are perfectly matched
to the complement.  It should not be used where T4 DNA ligase is more
suitable and flexible.
<td class="enzyme-application">
 <ul>
 <li>allele-specific gene detection in the Ligase Detection Reaction (LDR)
and Ligase Chain Reaction (LCR)
 <li>mutagenesis done by incorporation of phosphorylated oligonucleotide
during primer extension amplification
 </ul>

<tr>
<td class="enzyme-name"><i>E. coli</i> DNA Ligase
<td class="enzyme-description">Join single-stranded breaks in double-stranded
DNA molecules; and joins two molecules of dsDNA when they have cohesive 
ends.
<td class="enzyme-application">
 <ul>
 <li>Okayama and Berg method cDNA cloning
 </ul>

<tr><td class="header-cell" colspan="3">Nonspecific Exo- and Endonucleases
<tr><td class="description" colspan="3">These enzymes usually degrade (<q>chew 
up</q>) DNA or RNA in a way which is not sequence-specific as for <i>restriction 
endonucleases</i>.  These enzymes usually have specificity for DNA or RNA, either
in the single- or double-stranded form (but not always) and can have applications
for removing DNA or RNA when it is not wanted, or by making cohesive (ss ends)
blunt, but not by filling in, as polymerases do.

<tr>
<td class="enzyme-name">DNase I
<td class="enzyme-description">Specifically hydrolyzes single- and double-stranded
DNA with products being di-, tri- and oligonucletides (not to dNMPs).
The products have 5&prime;-phosphate and 3&prime;-hydroxyl ends.  It also
degrades the DNA strand of RNA-DNA hybrids.
<td class="enzyme-application">
 <ul>
 <li>Removes genomic DNA contaminating RNA isolates
 <li>For in vitro transcription reactions, removes DNA template
 <li>Used in footprinting to examine binding regions of proteins interacting
with DNA
 <li>nick translation (labeling)
 </ul>

<tr>
<td class="enzyme-name">RNase
<td class="enzyme-description">Specifically hydrolyzes single- and double-stranded
DNA with products being di-, tri- and oligonucletides (not to dNMPs).
The products have 5&prime;-phosphate and 3&prime;-hydroxyl ends.  It also
degrades the DNA strand of RNA-DNA hybrids.
<td class="enzyme-application">
 <ul>
 <li>Removes genomic DNA contaminating RNA isolates
 <li>For in vitro transcription reactions, removes DNA template
 <li>Used in footprinting to examine binding regions of proteins interacting
with DNA
 <li>nick translation (labeling)
 </ul>

<tr>
<td class="enzyme-name">BAL-31 exonuclease
<td class="enzyme-description">found in <i>Pseudomonas</i> culture medium; an 
 endonuclease highly specific for ssDNA and single-stranded, weakly 
 hydrogen-bonded regions of supercoiled closed circular DNA.  Contains 
 exonuclease activity for nicked dsDNA.  Activity persists at high (4 M) ionic 
 strength and in denaturing detergents (up to 5% SDS).  Requires both calcium 
 and magnesium for optimal activity (loss of magnesium reduces, loss of calcium 
 eliminates).
<td class="enzyme-application">
 <ul>
 <li>Can progressively shorten both termini of dsDNA
 <li>useful in restriction site mapping
 </ul>

<tr> 
<td class="enzyme-name">S1 Nuclease
<td class="enzyme-description">This is a 29 kDa glycosylated enzyme that is 
 secreted by <i>Aspergillus oryzae</i> and whose structure includes two 
 disulfide bonds and three Zn<sup>2+</sup> ions clustered in the base of the 
 cleft of the active site. Its primary structure was determined in 1991, its 
 tertiary structure in 1993, and the <i>nucO</i> gene encoding it cloned in 
 1995. 
 <p> 
 The enzyme is heat-stable. In a solution of 0.1-0.4 M NaCl and low pH (4.2) in 
 1 mM Zn<sup>2+</sup>, it degrades both single-stranded DNA and RNA by both exo- 
 and endonucleolysis. When added in greater amounts, it will cleave at 
 single-stranded nicks in ds DNA, but does not hit single base-pair mismatches. 
 In more harsh environments, including 30% formamide, 25 mM glyoxal, 30% DMSO, 
 the enzyme is still active.
<td class="enzyme-application">
 <ul>
 <li>any procedure to degrade single-stranded overhangs from DNA
 <li>nuclease protection assays
 </ul>

<tr>
  <td class="enzyme-name">Mung Bean Nuclease
  <td class="enzyme-description">First isolated in 196 from mung bean sprouts, it is a glycoprotein
  of 35-39 kDa, composed of two subunits with intersubunit disulfide bonds.
  It is Zn<sup>2+</sup>-dependent with an optimum low ionic strength
  (best is 25 mM ammonium acetate, pH 5.0).
  <p>
  The enzyme is used to make blunt-ended ds DNA, which may have protruding
  single-stranded termini, particularly after restrictions.  Mung bean
  is generally preferred to S1 nuclease for this purpose since it is
  less likely to cleave nicked DNA.  A problem is that mung bean nuclease
  prefers 5&prime; ends and is less efficient than S1 nuclease at the 3&prime; end.
  Mung bean nuclease is also more likely to leave frayed ends when the
  terminal base pair is AT, and is better when the terminal base pair
  is GC.  Care should also be taken not to add too much enzyme, since
  it can nick or degrade particularly AT-rich ds DNA  (it is 30,000-fold
  more active on ss DNA than ds DNA).
<td class="enzyme-application">
 <ul>
 <li>Removes any 3&prime; or 5&prime; overhangs to make blunt-end dsDNA or dsRNA
 <li>transcriptional mapping
 <li>cleaves hairpin loops
 <li>new restriction site generation
 <li>excises the gene coding sequences from genomic DNA
 </ul>


<tr>
  <td class="enzyme-name">Exonuclease VII
  <td class="enzyme-description">This is active on all forms of ss DNA, including hanging termini
   on ds DNA, denatured DNA, and on displaced single-stranded regions of
   DNA.  It will also remove pyrimidine dimers from UV-irradiated, nicked
   ds DNA.  It works from both the 3&prime; and 5&prime; ends, with oligonucleotides
   of 2-25 nt long.  It does not produce blunt-ended ds DNA however, since
   its exonucleolytic activity does not produce mononucleotides.
   Exo VII is compsed of two subunits of 10.5 and 54 kDa coded by the
   <i>xseA</i> and <i>xseB</i> genes of <i>E. coli</i>.
   <p>
   The enzyme is generally useful for removing primers after PCR, especially
   in multi-step PCR.  It is also used in DNA-RNA hybrid analysis as a
   substitute for S1 or mung bean nuclease;  the DNA remains and its length
   is the sum of exons and introns complementary to the probe.  It has
   been used also to remove inserts attached to plasmid vectors by
   poly(dA-dT) tracts.
</table>

<div class="outerblock">

<h3>The DNA Polymerase of <i>Thermus aquaticus</i></h3>
<p>
<i>Taq</i> polymerases vary considerably as to their performance.  Some are
very rapid in polymerization, but at the expense of fidelity in synthesis.
<p>
Some polymerases add a single nucleotide to the 3&prime; end in a 
template-independent manner (as terminal transferase does).  If the 3&prime;
base of the template is C, then an A is added.  The presence of T4 or <i>Pfu</i>
DNA polymerases provides a 3&prime;&rarr;5&prime; exonuclease activity which
removes this added base.  Then the amplicon can be phosphorylated using T4
polynucleotide kinase and ligated to blunt-ended dephosphorylated vector.
The <i>Pwo</i> and <i>Pfu</i> polymerases lack this extendase activity.

<!-- ===========================================================
   general ligation
 =============================================================== -->

<h2 id="dna-ligation">Ligation of the Phosphodiester Bond in Double-Stranded
DNA</h2>
<p>
The DNA ligase enzyme is present in the cells of living species to <q>repair</q> 
double-stranded DNA in which the phosphodiester covalent bond within one strand 
of the double-stranded molecule becomes broken.
<p>
The DNA ligase requires that the single strand be annealed (base-paired) to its 
complementary strand, and that the 3&prime; end of the break be a hydroxyl 
group, and the 5&prime; end of the break have an &alpha;-phosphate (single 
phosphate).  If the 5&prime; end is missing the phosphate, as it sometimes does 
purposely, then it can be phosphorylated using <a href="#pnk"><b>polynucleotide 
kinase</b></a>.
<p>
Ligation reactions are most typically done when it is time to insert a linear 
segment of ds DNA of interest (to be cloned) into a linearized vector DNA to 
construct a new plasmid.  The result will be that the insert becomes part of
a re-circularized molecule, which is then mixed with bacterial cells made 
porous (<q>competent</q>) to absorb and incorporate the circular ds DNA into 
their genetics.
<p>
The most efficient form of DNA ligase found for doing molecular manipulations
of DNA comes from the <i>E coli</i> bacteriophage T4.  The T4 DNA ligase has
its activity measured in <b>Weiss units</b> and has simple requirements of
its enzymatic activity:
<ul>
<li>ATP in concentration ranging from 0.5 to 1.0 mM
<li>magnesium (in the form of MgCl<sub>2</sub>) at concentration of 10 mM
<li>environment against against oxidizing agents:  10 mM DTT
<li>Tris buffer to maintain pH at about 7.5-8.0 at its required temperature,
present from 30-50 mM
of 16&deg;
<li>Enzyme added to 0.1 Weiss unit.
</ul>
<p>
1 Weiss unit is enzyme amount that converts 1 nmol of <sup>32</sup>P-labeled 
PP<sub>i</sub> into a product which is adsorbed into Norit in 20 min at 37&deg;. 
A new definition may be Cohesive End Unit, which is the enzyme amount that 
yields ligation of half of the <i>Hin</i>dIII fragments of &lambda; DNA in 30 
min at 16&deg;; there are 120 nM of 5&prime; DNA termini in these fragments in a 
20 &micro;l reaction tube.  One Weiss unit is 67 cohesive end units.
<p>
Mixtures are incubated anywhere from 10 min when the ends are cohesive to 
2 h when blunt-end ligation is done.  It may be useful to do it overnight as well.
To stop the ligase reaction, incubate the mixture at 65&deg; for 10 min.
<p>

<b>Linker ligation to linear DNA</b>.  When ligating linkers to target DNA, 
higher linker DNA concentrations can be used, with high linker:target ratios
(5-10:1?) used.  Excessive linkers will be removed with restriction digestion, which
is the purpose of adding linkers.
<p>
<b>Insert:vector ligation</b>.  When attempting to create circularized 
molecules, lower DNA concentrations overall must be used:  1-10 &micro;g/ml 
overall.  The insert:vector ratio should be between 2:1 and 6:1 (Sambrook and 
Russell indicate a 1:1 ratio for directional cloning).  When &lt; 2:1, there is 
a lower ligation efficiency, and &gt; 6:1, the circularized plasmid is likely to 
contain multiple inserts. If possible, conduct the ligation at different ratios. 
<p>
<b>Compatibility of Ligation with Other Enzymatic Handling</b>.  Since ligation 
usually follows restriction digestion or phosphorylation by PNK, can the ligation
be done without cleaning up the products or interference from those reagents?
The answer is yes.  The only essential requirement is that ATP should be present,
up to 1 mM.  In some cases, the restriction enzyme might re-cut the ds DNA if it
is still active from the previous reaction (not fully inactivated).  In this case
the products may need to be cleaned with phenol-chloroform and EtOH pptn.

<!-- ===========================================================
   reverse transcription
 =============================================================== -->

<h2>Reverse Transcription:  First Strand Synthesis</h2>
<p>
Reverse transcription is slow compared to DNA polymerase activity,
roughly 5 nt/s, 100 times slower than T7 DNA polymerase.

<h3>Method 1</h3>
<ul>
<li>Reverse Transcriptase Buffer (5&times;)
 <ul>
 <li>250 mM TrisHCl, pH 8.3
 <li>375 mM KCl
 <li>15 mM MgCl<sub>2</sub>
 <li>50 mM DTT
 </ul>
<li>Actinomycin D (1 mg/ml in ethanol)
<li>10 mM dNTP mix (P-L ultrapure)
<li>5 mg/ml BSA (BRL, nuclease-free)
<li>[&alpha;-<sup>32</sup>P]dCTP (400 Ci/mmol)
<li>M-MLV reverse transcriptase (200 U/&micro;l)
</ul>
<p>
Prepare 2 &micro;g of a T-tailed vector (e.g., pCGN1703, pARC7) by ethanol 
precipitation (wash with 80% ethanol if necessary) and dry in a vacuum, then 
resuspend in 5 &micro;l water (warm briefly in 37&deg; water bath, if necessary) 
and then hold on ice.
<p>
The mRNA samples are prepared by pelleting 5 &micro;g of poly(A)<sup>+</sup>
RNA for 15 min at 4&deg;, and washing in 70% ethanol (made with DEPC-water
and sterile).  Dry this briefly under vacuum, then resuspend in 10 &micro;l
and place on ice.
<p>
Include a control mRNA such as purified globin mRNA but use only 0.5 &micro;g.
<ol>
<li>Prepare actinomycin D/Reverse Transcriptase Buffer mix.  Count the
number of samples and pipet out 2.5 &micro;l times the number of samples
of 1 mg/ml actinomycin D (in ethanol) into a tube.  Dry this down in
a vacuum (proceed with further steps while this is drying down).
To the dry tube add the number of samples &times; 10 &micro;l of
5&times; Reverse Transcriptase Buffer and place on ice.
<li>Transfer the mRNA samples to tubes containing the T-tailed vector.
Add 16.5 &micro;l sterile water.  Total volume in all tubes is 31.5 &micro;l.
<li>Heat the tubes to 65&deg; for 3 min, then place on ice.
<li>Add 1 &micro;l of RNAguard to each sample.
<li>Dry down in a vacuum.
<li>Add in this order:
 <ol>
 <li>10 &micro;l 5&times; Reverse Transcriptase Buffer/actinomycin D mix
 <li>2.5 &micro;l of 10 mM dNTP mix
 <li>1.0 &micro;l 5 mg/ml nuclease-free BSA
 <li>1.0 &micro;l (10 &mu;Ci) of [&alpha;-<sup>32</sup>P]dCTP
 <li>32.5 &micro;l sterile DEPC-water
 </ol>
<li>Mix well, then add 3.0 &micro;l (600 U) of M-MLV reverse transcriptase,
mixing gently (but no vortexing).  Final volume should be 50.0 &micro;l
<li>Place in 37&deg; water bath for 60 min.
<li>Add an additional amount of enzyme (1 &micro;l, 200 U) to re-charge
the reaction (if the manufacturer believes in necessary), mix,
and leave additional 30 min at 37&deg;.
<li>Take out 1 &micro;l to a tube and add 99 &micro;l TE, pH 8.0.  This is
the 1:100 sample.
<li>Stop the reaction with 2.0 &micro;l of 0.5 M EDTA pH 8.0.  Place on ice.
</ol>
<p>
The total volume is about 53 &micro;l, with all additions.
<p>
Take the 1:100 sample and spot 5 &micro;l in duplicate on a DE-81 filter
square (2 &times; 2 cm).  Wash the filter squares:
<ul>
<li>6 &times; with 0.5 M disodium phosphate (Na<sub>2</sub>HPO<sub>4</sub>),
for 5 min ea wash
<li>2 &times; with 95% ethanol
</ul>
<p>
After the filters dry, count them in scintillant.  Total cpm incorporated
will be 20 &times; that to get the cpm per microliter multiplied by
53, since 53 &micro;l were in the reaction.


<h4>First Strand Product Purification</h4>
<ol>
<li>Take 53.0 &micro;l of the first strand synthetic reaction into a 0.5 ml tube.
<li>Add an equal volume of 1:1 phenol:chloroform.  Mix, and spin down
for 60 s.  Transfer upper phase to new tube.
<li>To the lower phase of the extraction tube, add 53 &micro;l TE pH 8.0 for
a back-extraction.  Spin down for 1 min after mixing well, then combine
the aqueous phases.
<li>Take 100.0 &micro;l of combined aqueous phase and add equal volume of
chloroform.  Vortex and spin down 1 min.  Transfer aqueous phase to 1.5 ml
tube.
<li>To the aqueous phase, add an equal volume of 4 M ammonium acetate.
<li>Add &minus;20&deg; cold absolute ethanol (500 &micro;l).
<li>Precipitate at room temp for 30 min
<li>Spin down the products for 30 min at room temperature.  Use a drawn-out
pipet to carefully remove the supernatant to another tube;  save until
sure that the products have been recovered.  Wash the pellet with
80% ethanol.  Determine Cerenkov counts in pellet.
<li>Resuspend pellet in 20 &micro;l pH 8 TE.  Add 20 &mu; of 4 M ammonium
acetate, then 100 &micro;l ice-cold ethanol and precipitate for 30 min
at room temperature.  Spin down, wash with 80% ethanol, dry under vacuum
and count the pellet again.
<li>Re-dissolve in 10 &mu; of sterile pure water and store at 4&deg;.
</ol>

<h4>First Strand Quality Assessment</h4>
<ol>
<li>Obtain cDNA products suspended in 10.0 &micro;l pH 8 TE.
<li>Prepare a <i>Bam</i>HI mix in this order:
 <ol>
 <li>9 &micro;l 10 &times; salts (pH 8, 100 mM)
 <li>78 &micro;l water
 <li>3 &micro;l (51 U) of 17 U/&micro;l <i>Bam</i>HI enzyme
 </ol>
<li>Pipet out 1 &micro;l (200 ng) cDNA to tube.  Add 9 &mu; of <i>Bam</i>HI
mix.
<li>After digestion, transfer reaction to a 1% agarose/TAE(1&times;) gel
lane.  As a control, include 100-200 ng linearized vector, along with
the markers.  Run gel at 5 V/cm.
<li>Soak gel in a denaturation buffer (2 times, 15 min ea) of
1.5 M NaCl and 0.5 M NaOH.
<li>Neutralize gel in 1 M TrisHCl, pH 7.4 with 1.5 M NaCl.  Twice for
15 min ea.
<li>Wash briefly in 20&times; SSC.
<li>Set up a Southern transfer in 20&times; SSC to either zetaprobe or
nitrocellulose.  Run it at least 5 hours or overnight.
<li>Remove but do not discard the gel.  Dry the blot,
then expose 4-6 hours on film.
</ol>

<h3>Method 2</h3>
<p>
[Klickstein LB, Neve RL, Golemis EA, Gyuris J (1995) <i>Curr. Prot. Molec. 
Biol.</i> 5.5.1-5.5.14)
<ul>
<li>5 mM dNTPs
<li>5&times; reverse transcriptase (RT) buffer
<li>0.2 M DTT
<li>0.5 mg/ml oligo(dT)<sub>12-18</sub>  or 15- to 40-mer code/gene-specific
antisense primer or random hexamer primers
<li>RNasin ribonuclease inhibitor (Promega)
<li>AMV (avian myeloblastosis virus) reverse transcriptase
<li>10 &micro;Ci/&micro;l [&alpha;-<sup>32</sup>P]dCTP (10000 Ci/mmol)
<li>0.5 M EDTA pH 8.0
</ul>

<!-- ===========================================================
   nick translation
 =============================================================== -->

<h2 id="nickxlat">Nick Translation</h2>
<p>
Source for notes in this section is J. Meinkoth &amp; G. M. Wahl,
<i>Meth. Enzymol.</i> <b>152</b>: 91-94, 1987.
<p>
<i>E. coli</i> DNA polymerase I is used in the repair of <q>nicked</q> chains of 
DNA, where a phosphodiester bond might be broken as a result of repair 
activities or other events.  It possesses a 5&prime;&rarr;3&prime; exonuclease 
activity and 5&prime;&rarr;3&prime; polymerase activity, in which it can advance 
in a 5&prime;&rarr;3&prime; direction with the nick on the 3&prime; side, and 
remove bases with its exonuclease in advance of its movement, while pairing 
correct bases with its polymerase in its wake.  <i>E coli</i> DNA pol I can work 
with linear, supercoiled, nicked, or gapped circular double-strands. Labeling 
can be as high as 10<sup>8</sup> cpm/&micro;g using deoxynucleotide 
5&prime;-[&alpha;-<sup>32</sup>P]triphosphates.
<p>
Nicks can be introduced at random sites using (pancreatic) deoxyribonuclease 
(DNase I).  The size of radioactive fragments which can be generated depends on 
the concentration of DNase I.  Generally fragments that are 500-1500 nt long 
provide optimal signal-to-noise ratios for hybridization to immobilized DNA or 
RNA.  Long probes with significant specific radioactivity can produce higher 
backgrounds.

<ol>
<li>Prepare a 10 &times; nick translation (NT) buffer composed of
50 mM Tris-HCl pH 7.4, 50 mM MgCl<sub>2</sub>, and 0.20-0.25 mM of
unlabeled deoxyribonucleoside triphosphates (dNTPs).  In preparing the
buffer, omit the dNTP to be used as label.
<li>Stock DNase is diluted in a buffer of 50 mM TrisHCl pH 7.4 containing
10 mM MgCl<sub>2</sub> and 1 mg/ml DNase-free BSA just before its
use.  The concentration is 40-80 pg/&micro;l.
<li>Add 2 &micro;l 10 &times; NT buffer to 100-400 ng DNA, and mix with
50-300 &mu;Ci of the 5&prime;-[&alpha;-<sup>32</sup>P]dNTP (3000 Ci/mmol,
10 mCi/ml)
<li>Add 1 &micro;l DNase and 19 &micro;l water.  Incubate 2 min at room temp.
<li>Add 1 &micro;l of <i>E. coli</i> DNA polymerase I (Pol I, 0.8 U/&micro;l).
Incubate 15&deg; for 30-150 min.  The reaction can be monitored using
TCA precipitation.
<li>To purify the finished product, add 0.1 vol of 2% SDS., 0.25 M EDTA,
or an equal volume of phenol-saturated water to stop reaction.
<li>Unincorporated nucleotides are removed using either one of the
following:
 <ul>
 <li>passing through a Bio-Rad P60 (100-200 mesh) column poured
  into a Pasteur pipet
 <li>spinning through a Sephadex G-50 fine grade media
  <ol>
  <li>pack a 1 ml syringe with a small amount of glass wool
  <li>add an aliquots of resin (enough to fill syringe)
  <li>spin about 1000 <i>g</i> for 5 min
  <li>repeat until packed to 0.9-1 ml
  <li>place syringe 15 ml conical plastic tube:  the lip of the
    syringe hangs on to the edge of the tube, and effluent does
    not contact the column
  <li>Add 100 &micro;l water to column
  <li>spin 1000 <i>g</i> 5 min
  <li>xfer to fresh disposable tube
  <li>bring sample to 100 &micro;l with water and add to column
  <li>spin 1000 <i>g</i> 5 min
  <li>collect the eluate
  </ol>
 </ul>
<li>If further purification beyond removing nucleotides is desired,
place DNA in microfuge tube and precipitate it with 5-25 &micro;g tRNA,
0.5 vol 7.5 M ammoniumm acetate, and 2.5 vol ethanol, incubating 10 min
in dry ice.  Microfuge (12,000 <i>g</i>) for 15 min.
</ol>
<p>
To prepare the nick-translated DNA for hybridization, dissolve the DNA in ~0.1 
ml water, denature in boiling water for 5 min or heat at 37&deg; in 0.1-0.3 M 
NaOH, quench in ice 2 min.
<p>
DNase I can be stored at 1 mg/ml in 50% glycerol, 50 mM Tris HCl pH 7.4, 10 mM 
MgCl<sub>2</sub>, 1 mM DTT, placing at &minus;70&deg; in small aliquots.  It is 
recommnended that new stocks of DNase be titrated to optimize producing probes 
500-1500 bp long:  a fixed concentration of DNase is used over a time course, 
about 200,000 cpm are removed, NaOH is added to 0.1 M and incubated 37&deg; 
10-15 min, 0.2 vol of 50% glycerol and 0.5% bromphenol blue, then the samples 
are run on 2% agarose with size markers in range of 100-2000 bp added.  The gel 
is run at 50 V until dye moves 8-10 cm, then stained and visualized. Good probes 
are seen as a smear in the range of 250-1500 bp.  Probes smearing from the gel 
origin give high backgrounds.
<p>
Nick translation failures may be attributable to DNA quality. Eliminate the 
possibility that enzymes are inactivated by treating control DNA.  To improve 
sample quality try phenol extraction or pass of a Elutip (Schleicher &amp; 
Schuell) or NACS (BRL) column. Alternatively, increase the DNase I concentration 
or increase the time before polymerase is added.  Sometimes scaling up enzymes 
and buffer volumes, but not sample DNA, helps.


<h3>Cultured Canine Cells</h3>
<ol>
<li>Cultured canine cells are fixed in a methanol:acetic acid (3:1) solution. 
These should be stored in the solution (not kept air dried) at &minus;20&deg;.
<li>Cells are placed on a slide by spotting MeOH:AcAc on to a clean glass slide, 
then dropping fixed cells several centimeters above the slide with a pipet so 
that they spread well.
<li>Cells should be <q>aged</q> overnight in absolute ethanol on the slide, but 
if the need to perform the reaction is immediate, aging can be achieved by 
heating at 94&deg; for 30 seconds in the ethanol with a large coverslip to 
prevent drying.  The heating block in a PCR thermal cycler can be adapted to the 
purpose:  an aluminum plate (foil?) can be cut to fit the block and at least be 
greater than the slide so that heat transmission is uniform, and then the cycler 
programmed to provide the heat for the appropriate time.  The coverslip is 
removed and the slide allowed to air dry.
<li>If desired, the cells are digested with pepsin at this time.
<li>Cells are washed once in PBS for a few minutes.
<li>Dehydration is achieved through graded increases in ethanol:  50%,
70%, 90%, and then 100% for 5 min each.
<li>Drying is on Hybaid for 5 min at 50&deg;.
<li>Prepare the probe such that 10 &micro;l is pipetted on the marked area
of the slide.  Place the coverslip over, and seal with rubber cement.
<li>The block is then heated to 65&deg;, the slide placed on the block, and a 
program commenced to heat the slide at 75&deg; for 1 min to denature the DNA.
<li>Hybridization is then allowed to proceed overnight at the hybridization
temperature.
<li>Washing is done at the designated temperatue.  Prepare the following washes: 
(i) Wash A: 50% formamide with 2 &times; SSC at pH 7.4 (ii) Wash B:  2&times; 
SSC pH 7.4 (iii) Wash C:  2&times; SSC with 0.1% Tween 20 pH 7.4.  Heat all 
solutions to 46&deg;.  Place the slide in Wash A for 5 min, and repeat this 
twice in separate baths. Place in Wash B for 5 min.
</ol>

<!-- ===========================================================
   hybridization to filters
 =============================================================== -->

<h2>Hybridization of DNA Probes on to DNA Fixed On Filters</h2>
<p>
The method described here are for probes larger than 100 bases. Probe purity is 
essential.  If sequences from the vector itself contaminate the probe, a false 
positive results and it is likely one will pick a colony not truly containing 
recombinant plasmid. One excellent way to ensure probe purity is to <a 
href="#gelpurify">gel-purify</a> the probe twice.
<p>
The following reagents will be required (except where noted):
<ul>
<li><b>optional</b> high quality <a href="#formamide">formamide</a>
<li><a href="#blotto">BLOTTO</a> (1 &times;) for when target DNA is
immobilized to nitrocellulose, or <a href="#denhardt">Denhardt's</a>
for when fixed to nylon
<li><a href="#prehybsolns">Prehybridization</a> and
 <a href="#hybsolns">hybridization</a> solutions
<li>Pre-Washing solution: 6 &times; <a href="SSC20x">SSC</a> or
  6 &times; <a href="#SSPE20x">SSPE</a>
<li>Wash solution 1
 <ul>
 <li>2 &times; SSC
 <li>0.1% SDS
 </ul>
<li>Wash solution 2
 <ul>
 <li>1 &times; SSC
 <li>0.1% SDS
 </ul>
<li>Wash solution 3
 <ul>
 <li>0.1 &times; SSC
 <li>0.1% SDS
 </ul>
<li>????
</ul>
<p>
The following equipment/materials will be used:

<ul>
<li>Boiling water bath
<li>glass baking dish or other hybridization chamber
<li>incubators set at 50&deg; and 68&deg;; also one at 42&deg; if
prehybridization in formamide is done
<li>radioactive ink or chemiluminescent marker for marking autorads
<li>water-soluble glue stick (UHU Stic)
<li>Whatman 3MM or equiv.
<li>wooden toothpicks or inoculating needle
</ul>

<h3>Prewashing and Prehybridization</h3>
<ol>
<li>The baked or cross-linked filters are floated on to a tray containing
2 &times; SSC until wetted completely from beneath.  Then submerge filters
for 5 minutes.  It might help to autoclave filters between 3MM paper
for 10 min at 10 psi/liquid cycle to prevent distortion that occurs between
filter and autorad.
<li>Move filters to baking dish with prewashing solution.  Use at least
enough to cover the filter in the dish, and use at least 200 ml if
doing thirty 82-mm filters.
Stack filters on top of one another.  Cover with plastic wrap and transfer
to rotating platform in incubator set at 50&deg; and incubate for 30 min.
<li>Soak Kimwipes in prewashing solution and gently scrape bacterial
debris from surface of filters.
<li>Transfer filters to prehybridization solution in glass dish or other
hybridization chamber (Seal-a-Meal bag, whatever).
Use at least enough solution to cover filters in the chamber used.
In a glass dish for thirty 82-mm filters, 150 ml is usu. sufficient.
Incubate 1-2 hours with agitation.  Set at 68&deg; when the hybridization
is in aqueous solution, 42&deg; when in 50% formamide.
</ol>

<h3>Probe Preparation, Hybridization, and Posthybridization Wash</h3>
<ol>
<li>Denature the <sup>32</sup>P-labeled ds DNA probe for 5 min in boiling
water bath.  Chill rapidly in ice-water.  (An alternative denaturation
is by adding 0.1 vol of 3 N NaOH at room temp. and leaving 5 min.
The probe is placed on ice and 0.05 vol of 1 M Tris HCl, pH 7.2, and
0.1 vol 3 N HCl added.  If the probe is already single-stranded, denaturation
is unnecessary.)
<li>The probe is now added to the prehyb solution and filters.  Incubation
is until 1-3 <i>C</i><sub>0</sub><i>t</i><sub>&frac12;</sub>.  The amount
of probe used depends on various factors.  More probe increases the
background (nonspecific hybridization) and less probe requires longer
hybridization period.  About 2-10 &times; 10<sup>5</sup> cpm should be
used (with a specific activity of 5 &times; 10<sup>7</sup> cpm/&micro;g).
<li>After hybridization, immerse filters in Wash solution 1 at room
temperature.  Use at least 10-20 ml per filter.  Agitate for 5 min.
Repeat this wash two more times.  Never let the filters dry.
<li>Wash the filters twice for 30-90 min in Wash solution 2 at 68&deg;.
Use at least 10-20 ml per filter.  Background is sufficiently low at
the end of this point to skip use of Wash solution 3, but if background
is high or high stringency is demanded go to next step, else skip.
<li>(Optional).  Remove solution 2 and add Wash solution 3 at 68&deg;
and leave for 1 h.  10-20 ml per filter.
<li>Air-dry filters on 3MM paper at room temperature.  Prepare the filters
for autoradiography by streaking underside with glue stick and arranging
numbered side-up on clean, dry 3MM paper; press to make sure they adhere.
<li>To adhesive dot labels, apply the radioactive ink or chemilumiscent marker
and then stick these asymmetrically on the 3 MM paper.  Cover everything
with plastic wrap, and use tape to secure the wrap to paper, and smooth
out wrinkles.
</ol>
<p>
Note that hybridization solutions with radiolabeled probe can be stored
at 4&deg; for several days and be re-used.

<h3>Autoradiography and Analysis</h3>
<ol>
<li>Set the filters in X-ray film (Kodak XAR-2, XAR-5, or equivalents)
for about 12-16 h at &minus;70&deg; with intensifying screen.
<li>After developing film, align with filters, and use a nonblack nonradioactive
fiber-tip pen to mark the file with positions of the dots.
<li>Tape clear mylar or firm transparent sheet to film and mark the positions
of the positive signals.  Using a different color mark positions of
asymmetrically located dots.  Remove the clear sheet from the film and
identify positive colonies by aligning dots on clear sheet with those
on agar plate.  As an alternative, one can place the film on a light
box and align the agar plate with master colonies or filter with stab
marks in the agar/master filter.
<li>A sterile toothpick or inoculating needle can be use to transfer positives 
to 1-2 ml of a rich medium (LB, YT, or Terrific Broth) containing selection 
antibiotic.  A plasmid miniprep is then obtained and analyzed by restriction 
digest or PCR.
</ol>
<p>
To remove filters glued to 3MM paper, soak in 2&times; SSC.


<!-- ===========================================================
   chemical synthesis of oligo probes
 =============================================================== -->

<h2 id="oligos">Synthetic Oligonucleotide Probes</h2>
[Much of the content of this section is from Sambrook &amp; Russell, 3rd Ed.]
<p>
Several temperatures for the melting or hybridization of oligonucleotide probes
can be computed by algorithms.
<p>
The Wallace Rule is used for calculation of <i>T</i><sub>m</sub> of duplexes
of 15-20 nt in length in high ionic strength (1 M NaCl or 6&times; SSC):
<p class="eqn">
<i>T</i><sub>m</sub> = 2(A + T) + 4(G + C)
<p>
Bolton &amp; McCarthy provide an estimate where the length is &lt; 100 nt
and the cation concentration is &le; 0.5 M and GC content is 30-70%,
also compensating for some mismatch:
<p class="eqn">
<i>T</i><sub>m</sub> = 81.5&deg;C + 16.6(log [Na<sup>+</sup>]) +
 0.41(%[G + C]) &minus; 675/<i>n</i> &minus; 1.0<i>m</i>
<p>
where <i>n</i> is number of bases and <i>m</i> is the percentage of base-pair
mismatching.
<p>
Sequence dependency is not a factor of the above equations.  The following
equation takes into account nearest neighbor thermodynamic data:
<p class="eqn">
<i>T</i><sub>m</sub> = (<i>T</i><sup>o</sup> &Delta;<i>H</i><sup>o</sup>) /
(&Delta;<i>H</i><sup>o</sup> &minus; &Delta;<i>G</i><sup>o</sup> +
<i>RT</i><sup>o</sup> ln[<i>c</i>]) + 16.6 log([Na<sup>+</sup>]/{1.0 +
0.7[Na<sup>+</sup>]}) &minus; 269.3
<p>
where &Delta;<i>H</i><sup>o</sup> = &sum;<sub>nn</sub>(<i>N</i><sub>nn</sub>
&Delta;<i>H</i><sup>o</sup><sub>nn</sub>) + &Delta;<i>H</i><sup>o</sup><sub>p</sub>
+ &Delta;<i>H</i><sup>o</sup><sub>e</sub>, &Delta;<i>G</i><sup>o</sup> =
&sum;<sub>nn</sub>(<i>N</i><sub>nn</sub>
&Delta;<i>G</i><sup>o</sup><sub>nn</sub>) + &Delta;<i>G</i><sup>o</sup><sub>i</sub>
+ &Delta;<i>G</i><sup>o</sup><sub>e</sub>, <i>N</i><sub>nn</sub> = number
of nearest neighbors (e.g., 13 for a 14-mer), <i>R</i> = 1.99 cal/mol-&deg;K,
<i>T</i><sup>o</sup> = 298.2&deg;K, <i>c</i> = molar strand concentration,
[Na<sup>+</sup>] &le; 1 M.
<p>
The approximations for the nearest neighbor enthalpies and free energies
are as follows:
<ul>
<li>&Delta;<i>H</i><sup>o</sup><sub>nn</sub> (average nearest neighbor
	enthalpies) = &minus;8.0 kcal/mole
<li>&Delta;<i>G</i><sup>o</sup><sub>nn</sub> (average nearest neighbor
	free energies) = &minus;1.6 kcal/mole
<li>&Delta;<i>G</i><sup>o</sup><sub>i</sub> (initiation term) = +2.2 kcal/mole
<li>&Delta;<i>H</i><sup>o</sup><sub>e</sub> (average dangling end enthalpy) =
	&minus;8.0 kcal/mole/end
<li>&Delta;<i>G</i><sup>o</sup><sub>e</sub> (average dangling end free energy) =
	&minus;1 kcal/mole/end
<li>&Delta;<i>H</i><sup>o</sup><sub>p</sub> (average mismatch/loop enthalpy) =
	&minus;8.0 kcal/mole/mismatch
</ul>

<h3>Chemical Synthesis of Primers and Oligonucleotides</h3>
<p>
This is a very special category that in most cases will not involve the 
average life science researcher.   So a discussion of the details of 
chemical synthesis are in a <a href="oligosynthesis.html">separate document</a>.

<!-- ===========================================================
   PCR
 =============================================================== -->

<h2 id="pcr">Polymerase Chain Reaction (PCR)</h2>

<h3>Thermal Cyclers</h3>
<p>
According to <a 
href="http://www.articleclick.com/Article/PCR-Thermal-Cycler-Buying-Tips/1404445">one
article</a>, thermal cyclers for PCR must have several important features, among 
these being fast rate of temperature change to shorten overall amplification 
time for long cycle programs, short <q>settling</q> times to permit a step
to proceed once temperature stability is established, uniformity of temperature
in the block so results are reliable, interchangeable blocks, gradient and
non-gradient temperature modes, instrument with small footprint, and a large
memory for storing multiple programs.

<h3>Reaction Containers</h3>
<p>
Containers for PCR reactions can occur in individual tubes, strips for tubes,
strips detached from multiwell plates, and multiwell plates usually with 
formats of 96 and 384 wells.  Multiwell plates are skirted, semi-skirted,
and unskirted.  Skirting makes plates stackable and usable in robotic
processes, while unskirted plates provide a more certain chance that the 
PCR wells make better contact with the thermal cycler.
<p>
<b>Domed vs. Flat Caps</b>. 

<h3>The Reaction Components and Conditions</h3>
<p>
Every chemical component in the standard PCR reaction is considered in this
section.

<h4>Template DNA</h4>
<p>
The template can be single- or double-stranded.  Linear DNA is more efficiently
amplified than closed circular DNA.  When the template is embedded within
high MW DNA (&gt; 10 kb), it is better to digest this DNA with a restriction 
enzyme with no site within the target to improve amplification.  In theory
although a single molecule can be amplified, a mass of 1.0 &micro;g of
mammalian genomic DNA might be used, which has 300,000 copies of a single copy
gene.  As for other DNA types, 10 ng yeast, 1 ng bacterial, and 1 pg plasmid 
DNA are useful for added templates.

<h4>Primers</h4>
<p>
Good PCR starts with good primers.  Not only purity, but primer design. The 
probability of a sequence occurring randomly that is complementary to a primer 
is given by:
<p style="margin:1em auto;" class="eqnserif">
<i>K</i> = [<i>g</i>/2]<sup>G+C</sup> &times; [(1 &minus; <i>g</i>/2]<sup>A+T</sup>
<p>
where <i>K</i> is the expected frequency of occurrence in sequence space,
<i>g</i> is the relative G+C content, and G, C, A, and T are the
number of specific nucleotides in the oligo.  For a ds genome with size
<i>N</i>, the expected number <i>n</i> of sites complementary to the
oligo is <i>n</i> = 2<i>NK</i>.  A 15 nt oligo would only be represented
once in a mammalian genome where <i>N</i> is ~3 &times; 10<sup>9</sup>,
according to these equations.
<p>
Factors that dictate selection of a good primer sequence:
<ul>
<li>Is the G+C content between 40-60%, and are bases distributed evenly, with no 
polypurine or polypyrimidine tracts and no dinucleotide repeats?
<li>The length of primer that is complementary should be 18-25 nt long, and 
length of pairs should not differ by &gt; 3 nt.
<li>Inverted repeats or self-complementary sequences should not be present, 
since the hairpins that form can prevent proper interstrand annealing.
<li>Is there primer-pair complementarity?  Especially when multiple primers are 
part of a batch, the 3&prime; ends of primers will form hybrids and the reaction 
will end up forming primer dimers.  Design should consider and account for the 
possibility of 3&prime; end complementarity. Also use hot-start or touchdown PCR 
to compensate.
<li>The calculated melting temperature <i>T</i><sub>m</sub> between primers 
should not differ by &gt;5&deg;, and the <i>T</i><sub>m</sub> of the amplified 
product should not differ by &gt;10&deg;.  This is important for denaturing in 
each cycle.
<li>The 3&prime;-terminus:  certain 3&prime; ends create difficulty for PCR.  If
possible, the base should be G or C at the 3&prime; end.  Avoid NNCG or
NNGC at the 3&prime; end since hairpins can form and create primer dimers.
<li>The 5&prime;-terminus:  it is not uncommon to add sequences for restriction 
enzymes, promoters, and other special nts to the 5&prime; end.  These may not be 
complementary to the target
</ul>
<p>
Degenerate primers are often constructed with codon usage in mind. These are 
obtained from a protein sequence whose nucleotide sequence is unknown.

<p>
<b>Primer purity</b>.  Generally the desalted primers obtained from commercial
synthesis have a quality sufficient for immediate use in standard PCR.  HPLC-
or gel-purified primers may be necessary in some instances however, for example
when amplification is done from only a single copy (single molecule) of
a genomic template.

<h5>Melting Temperatures</h5>
There are several equations that estimate the melting temperature between
an oligonucleotide primer and its target complementary sequence.
It is a matter of personal preference which one to prefer.
<p>
The <q>Wallace Rule</q> describes the melting temperature for
a perfect duplex of 15-20 nt where there is high ionic strength (1 M NaCl):
<p style="margin:2em auto;" class="eqnserif">
 <i>T</i><sub>m</sub> (&deg;C) = 2(A + T) + 4(G + C)
<p>
Each letter refers to the count of each residue.  Thus if a 17-mer has
3 adenines, 5 cytosines, 3 guanines, and 6 thymines, the melting temperature
is:
<p style="margin:1em auto;" class="eqnserif">
 <i>T</i><sub>m</sub> = 2(3 + 6) + 4(3 + 5) = 50&deg;
<p>
If the oligo were a 17-mer of only G and/or C residues, this would be
a Wallace rule maximum, or 68&deg;.  If it were only A and/or T residues,
it would be a Wallace rule minimum, or 34&deg;.
<p>
The Baldino et al. modification of the Bolton and McCarthy equation is
is a good estimator for the melting temperature of oligonucleotides ranging
from 14-70 nt, where cation concentrations are &le; 0.4 M:
<p style="margin:1em auto;" class="eqnserif">
 <i>T</i><sub>m</sub> = 81.5&deg;C + 16.6(log [K<sup>+</sup>]) +
0.41(%[G+C]) &minus; (675/<i>n</i>)
<p>
where <i>n</i> is the number of bases in the oligo.  There are other equations
which incorporate sequence information as much as base composition
using nearest-neighbor thermodynamic data.
<p>
Contamination of unwanted (exogenous) DNA is a common problem in PCR.
The DNA is introduced into one of the components added to the PCR reaction,
and amplified with the rest, discovered by use of a negative control
for a PCR rxn that omits template DNA.  The simplest solution is to
discard all solutions, reagents, and other disposables within reason, and
to decontaminate all instruments.
<p>
Ideal PCR setups demand that all equipment be devoted to the use of PCR.
Reactions are set up in a laminar flow hood, so that hair and cells from
the body are not introduced.  Change gloves frequently.

<h4>Thermostable DNA Polymerases</h4>
[this information is taken from content in Sambrook &amp; Russell, 3rd Ed.,
Vol. 2].
<p>
There are many thermostable polymerases from sources other than 
<i><b>T</b>hermus <b>aq</b>uaticus</i> (<i>Taq</i>).   The choice of which to 
use might depend on features desired other than a polymerase that works at 
72&deg;.  Some have 3&prime;&rarr;5&prime; (<i>UlTma</i>, <i>Pwo</i>, 
<i>Tli</i>, <i>DeepVent</i>, <i>Pfu</i>) or 5&prime;&rarr;3&prime; (<i>Taq</i>, 
<i>rTth</i>, <i>Tbr</i>, <i>rBst</i>) exonuclease activity.  Others might be 
chosen because of a high fidelity in replication:  <i>UlTma</i>, <i>Bst</i> 
isotherm large fragment, <i>Pwo</i>, <i>DeepVent</i>, <i>Pfu</i>.  Some 
polymerases have been characterized which show very good stability at 
95-100&deg; for a much longer time:  <i>DeepVent</i> 480 min at 100&deg;, 
<i>Pfu</i> 240 min at 95&deg;, <i>Tbr</i> 150 min at 96&deg;, <i>Tfl</i> 120 min 
at 70&deg;.  Compare to <i>Taq</i> which has only a stability of 9 min at 
97.5&deg;.  An advantage of <i>Taq</i> over other polymerases is that it has a 
lower <i>K</i><sub>m</sub> for dNTPs. <i>Tfl</i> might be preferred when 
amplifying large segments of DNA.
<p>
The <i>Bst</i> Isotherm large fragment might be used when sequencing difficult
segments of DNA.
<p>
<i>Pwo</i> produces blunt-end fragments and so might be preferred for using
amplified products in cloning.

<h4>Cofactors, Small Molecules, Ions</h4>
<p>
<b>dNTPs</b>.  Usually supplied in equimolar mixes of dATP, dTTP, dCTP,
and dGTP, final concentratios in reactions are 200-250 &micro;M.  Concentrations
&gt; 4 mM are inhibitory, but on the low end, as little as 20 &micro;M can
be used.  Pyrophosphates which inhibit PCR can form with repeated freeze-thaw
cycles of stocks, so it is wise to aliquot stocks whether homemade or purchased.
Ideal pH for stocks is 8.1, when NaOH is used to adjust free acid.  Always
centrifuge thawed stocks and vortex briefly before using.
<p>
<b>Mg<sup>2+</sup></b>.  DNA polymerases require divalent cations, with 
Mg<sup>2+</sup> being usual, but Mn<sup>2+</sup> has also been found to work. 
Ca<sup>2+</sup> does not work.  The molar concentration of cation should exceed 
the concentration of phosphates on dNTPs and primers.  Usually the concentration 
is 1.5 mM and as high as 2.5 mM Mg<sup>2+</sup>, but reports show that 
concenrations of 4.5 and 6 mM can decrease nonspecific mispriming, while a 
contradictory report indicates that it increases mispriming.  Kits to optimize 
Mg<sup>2+</sup> are sold; alternatively, a 10-sample PCR set up can be done with 
Mg<sup>2+</sup> from 0.5 to 5.0 mM, followed by 2nd round where 10-sample PCR is 
done in 0.2 mM increments centered around the result of the 1st round.  It 
may be necessary to clean up the DNA for PCR if EDTA or phosphate ions are present
significantly.
<p>
<b>pH and Monovalent Ions</b>.  The polymerase and cycling work best in a pH 
range from 8.3 to 8.8 (determined at room temperature), using 10 mM TrisHCl.  At 
72&deg;, the pH drops to the optimal 7.2.  KCl at 50 mM helps in the amplification
of long fragments (&gt; 500 bp), and increasing it to 70-100 mM may improve 
amplification of shorter fragments.

<h3>PCR Method</h3>
<p>
Unless a PCR program is being used based on results in the literature, 
a <q>default</q> program that should be entered in the thermal cycler
could be as follows:
<table style="margin:1em auto;">
<tr><th>           <th>Denaturation   <th>Annealing     <th>Polymerization
<tr><td>30 cycles  <td>30 s @ 94&deg; <td>30 s @ 55&deg;<td>60 s @ 72&deg;
<tr><td>Last cycle <td>60 s @ 94&deg; <td>30 s @ 55&deg;<td>60 s @ 72&deg;
</table>
<p>
For the initial denaturation step, 94-95&deg; for 45 s should be sufficient
when the DNA is linear and has 55% G+C content.  If GC-content is higher,
consider using a thermostable polymerase designed for this type of template.
<p>
For setting an annealing temperature, it should be about 3-5&deg; lower than
the computed <i>T</i><sub>m</sub> of the primer.  Optimization of annealing
conditions can be done by doing trial PCR runs at temperatures which are
2-10&deg; below computed primer <i>T</i><sub>m</sub> values.  If the PCR can
be programmed for <a href="#touchdown-pcr">touchdown PCR</a> consider that
possibility although the yields might be more modest.
<p>
The program above is typical for 50 &micro;l reactions in thin-walled tubes.
Times and temps may vary with the equipment and reaction volumes (larger
volumes may take more time to reach temp).  The usual rule of thumb
is that for each 1000 bp of the target DNA length, 1 min of polymerization
is required.  Cyclers may lower the temperature to 4&deg; after the last
cycle to store the products until taken from the machine.  Freeze amplified
products at &minus;20&deg;.
<p>
When software is available for computing all steps (initial, cycling, and
termination), use it.  It may take into account the ionic strength and
concentration of salts as well as the GC content of the primers and 
template.

<ol>
<li>Obtain a 0.5 ml sterile microfuge or amplification tube, or the well
of a sterile microtiter plate.  Add to it the following in this order:
 <table style="margin:1em auto;" >
 <tr><td>10&times; amplification buffer            <td>5 &micro;l
 <tr><td>20 mM dNTPs (pH 8.0)                      <td>1 &micro;l each
 <tr><td>20 &mu;M forward primer                   <td>2.5 &micro;l
 <tr><td>20 &mu;M reverse primer                   <td>2.5 &micro;l
 <tr><td>Thermostable DNA polymerase (1-5 U/&micro;l) <td>1-2 U
 <tr><td>water                                     <td>28-33 &micro;l
 <tr><td>template DNA                              <td>5-10 &micro;l
 </table>
 <p>
The total volume will be 50 &micro;l.  A typical PCR reaction should contain
1.5 mM Mg<sup>2+</sup>, 50 mM KCl, 200 &mu;M dNTPs, 1 &mu;M primers, 1-5 U
polymerase, and 1 pg to 1 &micro;g template DNA.  The wide variation on
template DNA is because of sequence complexity:  if mammalian DNA is
used, up to 1 &micro;g is tried, but if yeast, bacterial, or plasmid DNA
is used, 10 ng, 1 ng, and 10 pg, are used respectively.  Controls
include two positives and two negatives.  One positive contains
bystander DNA + the target-not-template DNA, plus the primers.
Target-not-template DNA is any DNA that contains the site for primer
complementary binding but is not the DNA of interest, and bystander DNA
is DNA that contains no target but it resembles the template DNA in terms
of complexity, size, and concentration.  The second positive control is
the primers with the target DNA, but no bystander or template DNA.
There are also two negative controls:  one contains only primers, and
the other contains the primers with only bystander DNA (no target or
template DNA).
<li>If the thermal cycler has no heated lid, overlay this with 1 drop
(~50 &micro;l) light mineral oil (prevents evaporation), or place a bead
of wax in the tube if doing a hot start protocol.  Place tubes in cycler.
<li>Run the program for PCR as entered above.
<li>Obtain 5-10 &micro;l of the sample after amplification and the 4 control
reactions and run on gels (agarose or polyacrylamide).  Bands of expected
size should be found.  Sequencing or Southern blots will confirm the
identity of the product.
<li>The mineral oil (if used) can be removed with extraction in 150 &micro;l
chlorform.
</ol>

<h3>PCR Optimization &amp; Enhancement</h3>
<p>
Studies show that certain additives appear to improve PCR, such as preventing 
mispriming.  Formamide (1.25-10%), DMSO (to 15%), glycerol (1-10%), TMAC, 
potassium glutamate (10-200 mM), ammonium sulfate, nonionc and cationic 
detergents (betaine, TX-100, Tween 20, NP-40), BSA, and <q>specificity 
enhancers</q> sold commercially have been claimed to help.  However, it is more 
useful to optimize the basic components, such as Mg<sup>2+</sup> and 
K<sup>+</sup> concentrations before attempting the use of these enhancements.

<h3>Hot Start PCR</h3>
<p>
First described in 1991, this is done to suppress nonspecific amplification.
Either the polymerase or Mg<sup>2+</sup> or some other component required
is withheld until the reaction is heated that inhibits nonspecific priming
or primer oligomerization.  Template DNA and primers are mixed and held
at a temperature above that which stops nonspecific binding of primer
to template.  All components except the polymerase are added.
<p>
Primers, Mg<sup>2+</sup>, dNTPs and buffer are mixed and then covered with
wax  (ampliwax PCR Gems from P-E or any wax melting at 53-55&deg;.  The
other components are layered atop the wax barrier.  When the denaturation
phase starts, the wax melts and the components merge.  Melted wax floats
to top and acts as an anti-evaporative barrier.
<p>
Other variants of hot start include using antibodies that inhibit the
polymerase.  When things heat up, the antibody dissociates and the
polymerase is active (Clontech's TaqStart and tTHstart and Life Technologies'
Platinum <i>Taq</i> Polymerase are examples).  Perkin-Elmer's AmpliTaq Gold
requires heating to ~93&deg; for 10 min during a special first cycle.
Another variation embeds Mg<sup>2+</sup> into wax beads which is released
upon melting.
<p>
Hot start PCR generally is not required when the primers are well-designed
and generate high yields of product.  When the template is low in
concentration or highly complex (mammalian genomic DNA) or when multiplex
PCR is used, hot start should be used with touchdown PCR.

<h3 id="touchdown-pcr">Touchdown PCR</h3>
<p>
Sometimes the melting temperature of primers is not known with certainty
such as when primers are designed from amino acid sequences (in this
case, degenerate primers must be used), or when amplifying from a
multigene family, or the template is complex and takes up a large
sequence space, or the DNA to be amplified is from another species than
that for which the primers were designed, or the primers contain
<q>universal</q> bases such as inosine.
<p>
That is, there is a likelihood of mismatch for the primer, and this
creates a high probability of nonspecific amplification.  Optimum annealing
temperature must be determined empirically and not from formulae.
<p>
In this case a set of reactions are done with different annealing temperatures
set for the cycle.  Some thermal cyclers are individually programmable
to handle different annealing temperatures, but the method is inefficient
otherwise.

<h3>Troubleshooting Amplifications</h3>
<p>
If the bands of the DNA of interest and the positive controls are sharp but 
faint, or smear into the lower MW DNA region, this is due to inefficient priming 
or extension.  There are several remedies to try. One is to set up reactions 
containing different concentrations of the two primers and then find the optimal 
concentration, and then setting up touchdown PCR reactions to find the optimal 
Mg<sup>2</sup> concentration as well.  Another is to include Clontech's GC-Melt 
in the reaction mixture. A third solution is to use the minimum possible 
annealing temperature. And finally:  consider adding adjuvants such as 0.2-0.6 
mg BSA/ml, 5% DMSO, or 5% glycerol.
<p>
If the bands are very faint in the test sample and in the positive control 
containing only target-not-template DNA, but the positive control containing 
bystander and target-not-template DNA is strong, this indicates a dirty template 
DNA.  Clean it up by phenol:chloroform and ethanol precipitation twice, and 
resuspend the DNA in water alone with no EDTA.
<p>
If there are bands in the negative controls (there should be none!), the 
solutions or plasticware are contaminated with template.  Make up new reagents, 
and use new plasticware.
<p>
If there are bands of the wrong size (MW), this is because of nonspecific 
priming by one or both primers.  Either decrease annealing time, or increase 
annealing temperature, or do both.
<p>
If there is a smear that obscures the product, this is amplification of primer 
dimers, or there is too much template DNA.  Primer dimers are when primers 
anneal at two or more sites, perhaps because the primer has homology with highly 
repetitive sequences in the template;  each primer concentration may need to be 
optimized independently.  An alternative is to use hot start or touchdown PCR, 
or both.  Another possibility would be to reduce the template DNA used by 2.5-, 
or by 10-fold.

<h3>Troubleshooting PCRs</h3>
<p>
Is amplification weak or not detectable?  Many possibilities here. Many 
components can be defective:  reagents, the thermal cycler.  Perhaps a 
programming error.  One can compare old and fresh reagents in two different 
thermal cyclers.
<p>
The quality of the template DNA may be a problem.  Its preparation may contain 
substances known to inhibit PCR: proteinase K, phenol, EDTA, ionic detergents,
heparin, polyanions (spermidine), hemoglobin, even gel-loading dyes such
as bromophenol blue and xylene cyanole FF.  Cleaning up the template by
dialysis, EtOH precipitation, chloroform extraction, and chromatography on
resin (or use of a cleanup kit), can solve the problem.
<p>
Suboptimal annealing conditions could explain this.  Verify the 
<i>T</i><sub>m</sub> of the primers.  Use touchdown in combination with hot 
start PCR.  Verify primer concentration (may need to optimize). It may be 
necessary to design and synthesize new primers.
<p>
Suboptimal extension of annealed primers may account for weak or undetectable 
amplification.  It may be necessary to optimize concentrations of 
MgCl<sub>2</sub>, template DNA, and dNTPs.  The pH and buffer used may be a 
factor too; consider replacing Tris with tricine, bicine, or EPPs with pKas that 
are less or non-temperature-dependent.  Another problem is use of template DNA 
that contains inhibitors of the PCR: repurify the DNA.  Consider also increasing 
the number of cycles. Try adding enhancers such as 10% DMSO, 5% PEG-6000, or 10% 
glycerol if problems persist.  If they still persist, reamplify a 1:100 dilution 
of the PCR products in fresh buffer and primers, or do nested PCR. GC-Melt from 
Clontech may also help.
<p>
Weak or undetectable amplification may also result from ineffective
denaturation.  Increase the time or temperature of denaturation.
<p>
Finally, weak or undetectable denaturation may result from the distance
between primers being very large.  In this case, use the special polymerases
that are optimized for amplifying long segments of DNA.
<p>
If multiple amplification products are the problem, try touchdown PCR in 
combination with hot start or booster PCR.  Or optimize the concentrations of 
magnesium, template DNA, polymerase, or dNTPs.  Consider also testing different 
pH values for the reaction.  Primers may be at the wrong concentration, or may 
need to be newly designed and synthesized. One might also reamplify a 1:100 
dilution with new primers and fresh buffer (30 cycles), or do nested PCR.
<p>
If primer dimers are in excessive amounts, try touchdown PCR with hot start or 
booster PCR.  Where problems continue, redesign new primers and be careful of 
the 3&prime; ends.

<h3>Preparing Amplified Products for Cloning</h3>
<p>
The presence of <i>Taq</i> polymerases after PCRs can interfere with many
subsequent reactions such as restriction digests (<i>Taq</i> can fill in
recessed ends).  Moreover, unreacted dNTPs can interfere with other
enzymatic processes.  Thus a clean-up is necessary:
<ol>
<li>Up to 8 PCR reactions (1 &micro;g DNA in about 400 &micro;l) can be pooled 
if pooling is necessary.  The reactions should have been analyzed on gels first: 
if significant levels of nonspecific products are present, it may be better to 
gel-purify the products before proceeding.  If mineral oil was used, spin down 
the tubes and take only the aqueous phase. <li>Add 0.2 vol of 5&times; 
proteinase K buffer and then add proteinase K to 50 &micro;g/ml (usually it is 
present as a 20 mg/ml stock).  Incubate at 37&deg; for 60 min.
<li>Stop the digestion by heating in 75&deg; for 20 min
<li>Extract 1&times; in phenol:chloroform, then 1&times; in chloroform.
<li>Add 0.2 vol 10 M NH<sub>4</sub>OAc, 2.5 vol EtOH, mix well, and let sit
30 min at 4&deg;.
<li>Spin the DNA ppt at max speed for 5 min at 4&deg; in a &mu;fuge.
Discard the supernatant.  Wash pellet with 70% EtOH, spin, and remove
supernatant.  Allow to air dry.
<p style="margin:0 5%;font-size:83%;">At this point, it might be
useful to use one of the chromatography resins to do further cleanup.
This is especially recommended if the primers were used to add restriction
sites.
<li>Dissolve the precipitate in TE pH 8.0.  Assume a 50-80% recovery,
and dissolve to 25 ng/&micro;l.
<li>Analyze by agarose gel electrophoresis with EtBr.  Verify that the
intensity of the band is consistent with a 25 ng band.
</ol>

<h3>Removing Oligonucleotides and Excess dNTPs from Amplified DNA by
Ultrafiltration</h3>
<p>
Removal of primers, primer dimers and excess dNTPs after PCR reactions prevents 
problems that will follow with doing other work.  One can use 
phenol:CHCl<sub>3</sub> extractions and chromatography on anionic silica 
gels/glassmilk/chaotropic silica (typically in kits).  The organic extractions 
are good when there are numerous samples.  Spin dialysis is popular however.  An 
anisotropic, hydrophilic, nonabsorbent, porous membrane retains the products, 
while primers and dNTPs are passed through the membrane into the filtrate. 
Centricon-100 or Microcon microconcentrators let oligos (primers) up to 48 bp 
long pass through, retain ds DNA as small as 125 bp.  Longer oligo will have to 
be removed by gel purification.
<ol>
<li>Prepare a 1-2 ml centrifugal ultrafiltration device (Centricon or Microcon
product with 100,000 MWCO or equivalent, with membrane optimized for ds DNA)
by washing with water and then TE pH 8.0.  Put 2 ml of TE in the reservoir
above the membrane.
<li>Recover the PCR reaction mixture.  If it contains mineral oil, extract with 
chloroform first.  Add it to the TE solution in the ultrafilter device.
<li>Spin micro-ultrafilter according the manufacturer directions: usually 
1000 <i>g</i> for 30 min at temperatures between 4-25&deg;
<li>When the solution is close to the limit of concentration, it can be
recovered by inverted spinning at low rpm for a couple of minutes.
<li>It might be necessary to precipitate the DNA:  0.1 vol of 3 M sodium acetate
and 2-3 vol EtOH.  The sample should have quality for sequencing or ligation.
</ol>

<h3>Quantitative PCR</h3>
<p>
Quantitative PCR is any PCR-based method that can provide in its result an
estimation of the amount of DNA in the (starting/original) sample, usually as
an integer value representing the number of copies of the genome (genomic
equivalents).
<p>
Not all quantitative PCR has to be real-time PCR.

<h3>Real-Time PCR (qPCR)</h3>
<p>
Real-time PCR is a PCR reaction that can be followed as it is happening.
Generally the term <q>real-time</q> applies to any process that can be tracked
as it is happening. This tracking means that if there is a signal to be
measured, the signal rises during the course of the process of amplification of
double-stranded DNA, which is what happens in PCR, and it can be monitored.
<p>
When quantitative, a process of this kind can be described as an <dfn>assay</dfn>.
The ability to track assays in the course of their reaction has been around for
a long time.  There are basically two kinds of assays.
<ol>
<li><b>End-point assays</b>: these assays measure the reaction only at the end
of the reaction, usually when something has been done to stop the reaction
(change in temperature, perhaps the addition of a <q>stopping</q> reagent).
<li><b>Kinetic assays</b>: these assays are designed to measure the extent of
the reaction <q>continuously</q> (while it is happening), or at least at
meaningful time period intervals (every 15 or 30 or 60 or 120 seconds). A good
signal vs time plot can be obtained from the kinetic assay.
</ol>
<p>
When PCR was first developed, it was neither quantitative nor could the
amplification be followed in <em>real-time,</em> that is kinetically. To do so
required the setup of several reaction tubes all having the same ingredients
(primers and template) and then stopping the amplification of each tube at
various cycles. The signal could then be plotted vs. the cycle number. How to
measure the signal might vary, whether looking at the liberation of a
photo-bearing label whose signal increased or running the amplification on an
agarose gel and using linear densitometric scanning of fluorescent bands or
perhaps using dot-blots with fluorophoric, radioactive or chemiluminescent
systems would give better quantitation.
<p>
The process in the current technological generation of real time PCR (qPCR) is
like this today. The real time-capable thermal cycler is able to measure a
signal at the end of each amplification round and then plot signal vs. cycle
number. In most cases the signal is not observed until a <q>threshold</q> cycle
number (C<sub>T</sub>) and then there is a rapid exponential rise up to an
asymptote.  And so real-time PCR is tracked as it is happening.  Because that
tracking feature (or signal) can be measured or quantitated, it is thus
quantitative PCR.  As the convention is not to use RT-PCR to refer to
reverse-transcription PCR, it creates confusion to abbreviate real-time PCR
at RT-PCR, although it has been proposed that the lowercase letter be used, as
in rt-PCR.  The convention for real-time PCR is now to abbreviate it as qPCR.

<h4>Types of qPCR</h4>
<p>
The TaqMan method of qPCR (developed by Applied Biosystems) uses
oligonucleotide probes with a fluor attached to the 5&prime;-end and a quench
molecule attached to the 3&prime;-end: when not hybridized, the fluor and 
quench are in close proximity and no fluorescence signal is detected, but when 
hybridized to the internal region of the PCR product, the polymerase cleaves
the TaqMan probe and separates from the fluor from the quench portion, and 
F&ouml;rster Resonance Energy Transfer (FRET) does not occur. The fluorescence 
signal increases, directly with the level of expression. A drawback is the 
expense for preparing TaqMan probes for each mRNA target.

<p>
Another approach by Molecular Beacons also uses FRET with 5&prime; fluor and 
3&prime; quench but these must remain connected with each amplification cycle. 
The probes form a stem loop and this close proximity quenches fluorescence.  But 
when hybridized, the two components are separated and fluorescence increases. 
Again the drawback is to synthesize probes with the fluor and quench ends for 
each mRNA.

<h4>Design and Optimization of Molecular Beacon Real-Time PCR Assays</h4>
<p>
These notes are from J. A. M. Vet and S. A. E. Marras in <q>Oligonucleotide
Synthesis:  Methods and Applications</q>, P Herdewijn, ed., <i>Methods in
Molecular Biology</i>, Vol 288, pp 273-
<p>
Many real-time PCR methods make use of a dye like SYBR Green that binds only to 
double-stranded molecules and obtains special fluorescence properties. As each 
cycle of amplification continues, more dye binds and the fluorescence signal 
increases.  A major disadvantage is that dye binds even to double-stranded 
formations that are not specific for the target: for exmaple primer-dimers and 
other products not related to the target.  A hybridization probe that is 
self-complimentary such that a stem-loop provides less background signal: if one 
end of the strand has a fluorophore and the other end a quencher, it is only 
when these are separated that fluorescence signal is increased.  This occurs 
when the self-pairing probe is denatured, but also when the probe anneals to a 
target strand rather than re-pairing within itself, as the fluorophore remains 
distant from quencher.  Thus a molecular beacon by definition is a properly 
designed probe that self-pairs a quencher and fluorophore but which becomes 
separated when annealed to the target, thus indicating its presence by enhaned 
fluorescence.
<p>
Other techniques that make use of fluorescence include 5&prime;-nuclease probes, 
adjacent probes, and duplex scorpion primers.  Applications for molecular 
beacon-type assays include general detection of genomic sequences, SNPs, mRNA 
expression levels, and pathogen identification.  The following must be done in 
appropriate use of molecular beacons.
<p>
<b>Target design</b>.  Targets should be selected that do not have significant
secondary structure, since these can inhibit annealing to beacon or prevent
product extension by Taq polymerase.  Target length to between 75 and 250 bp to
reduce the likelihood of self-pairing.  Shorter amplification products generally
produce higher fluorescence signals since beacon-labeled probes are able to
compete with complementary strands.  Use a DNA folding program to determine
if the target has secondary structure and what melting temperature the pairing
might form.  It should be lower than the probe/primer annealing temperatures.
<p>
<b>Primer design</b>.  Primers should have a <i>T</i><sub>m</sub> higher than
predicted template secondary structures.  The following characteristics can be
used to help ensure this.
<ul>
<li>Primers should be designed with 50-60% GC content.
<li>A <i>T</i><sub>m</sub> of 50-65&deg; is best.
<li>Eliminate secondary structures:  no more than two consecutive basepairs
should create self-pairing possibilities.
<li>Avoid G or C repeats of three consecutive bases.
<li>Make sure there are no 3&prime;-complementarities, which lead to primer-dimer
formation.
<li>G and C bases should be put on the ends of primers.
<li>Verify the specificity of primers with BLAST to avoid false priming.
<li>Prepare a few (more than one) primer sets for the target, in case any one
fails and to find the better of the sets.
<li>Test these sets also using SYBR Green.
</ul>
<p>
<b>Optimizing Before Beacon-Labeling</b>.  Before preparing beacon-labeled 
probes, test the designed probes using SYBR Green.
<ol>
<li>50 &micro;l reactions containing 1X PCR buffer and 2.5 U AmpliTaq Gold
DNA polymerase, 1X SYBR Green, and 250 &micro;M dNTPs should be tested with
the following variables:  3-6 mM, in 0.5 mM steps, MgCl<sub>2</sub> and
100, 200, 500, and 1000 nM of the primers.
<li>A positive and negative template control are required.  Use 10<sup>5</sup>
copies of template for the positive, and MilliQ water for the negative.
<li>The PCR program is initially 95&deg; for 10 min (required for activation
of the AmpliTaq Gold polymerase, but use the correct one for the polymerase
used), then for 40 cycles a 95&deg; 30 s denaturation, 30 s for the correct
annealing temp, then 30 s at 72&deg; for extension.
<li>Monitor fluorescence for SYBR Green.  If the instrument does not have
specific settings for this dye, use the FAM (fluorescein ester) since its
excitation/emission are similar to SYBR Green.
</ol>
<p>
In interpreting the data, use the proper method for background and baseline
subtraction.  Find the Ct (threshold cycle) for the positive and negative
controls.  Generally 100,000 copies of template will have a Ct between 21
and 25.  If the negative control shows a Ct &le; 30, a new primer set should
be chosen (re-designed if necessary) since nonspecific amplification interferes
with analysis.  If Ct is between 30 and 40, a new primer pair must be chosen
if the analysis is likely required to detect target as low as 100 copies.
If analysis is done as low as 1 copy, then the negative control should not
show a Ct below 40 (and actually not show a Ct at all).
<p>
<b>PCR efficiency</b> is determined by creating a standard curve of a log 
dilution (10<sup>5</sup>, 10<sup>4</sup>, 10<sup>3</sup>, 10<sup>2</sup>, 
10<sup>1</sup> copies) and plotting the Ct against log copy number.  A slope of 
&minus;3.322 would indicate 100% efficiency, with a correlation coefficient of 
at least <i>r</i> = 0.995.  After these SYBR Green studies, the molecular beacon 
is ready for design and testing.

<h5>Molecular Beacon Design</h5>
<p>
A good beacon must not interfere with the ability of the probe to pair to the 
target at the annealing temperature, while free beacons must self-pair properly 
and remain nonfluorescent at this temperature. Probe and arm lengths are 
factors.  <i>T</i><sub>m</sub> values can be predicted using <q>percent GC</q> 
rule.  Predictions for the probe and stem arm sequences should be obtained. 
Probe length is usually 15-30 nt and the probe-target <i>T</i><sub>m</sub> 
should not exceed 72&deg; to avoid interference with the DNA polymerase.
<p>
In <b>standard target detection</b> probe lengths of 22-30 nt can be used and 
the <i>T</i><sub>m</sub> of probe-target hybrid should be 7-10&deg; above PCR 
annealing temperature.  Thermal denaturation profiles can be used to confirm 
predicted <i>T</i><sub>m</sub>.  Longer loop lengths ensure that mismatches 
remain stable at annealing temp;  mismatch possibility can occur when trying to 
detect retroviral subtypes with one or two base substitutions.
<p>
When beacons are to be used to look at SNPs and discriminating alleles, probes 
must be perfectly complementary and beacons must form a stable probe-target 
hybrid.  In the presence of mismatched targets, beacons must remain closed 
(self-pair).  Thus:
<ol>
<li>Probe must be selected that dissocitate from target at temperatures 5-8&deg;
higher than the annealing temp.  Probe length will be between 15-21 nt.
<li>Plot the fluorescence signal of solutions of beacons in presence of target
as a function of temperature.  One temperature range will be a <q>window
of discrimination</q> which is the range between perfect complementarity
and one where mismatched probe-target hybrid cannot form.
</ol>
<p>
After determining probe sequence, the <b>stem arms are designed</b>.  Percent-GC
rules and nearest-neighbor methods for Tm prediction cannot be use for
self-pairing as they can be for primer and probe hybridization predictions;
instead use DNA folding programs that also provide Tm estimations
such as the <a href="http://www.bioinfo.rpi.edu/applications/mfold">mfold
server</a>.  Stems will usually be 5-7 nt long.  GC-rich arms that are 5 nt
long have predicted Tm of 55-60&deg;; with 6 nt, it is 60-65&deg;; and with
7 nt, is it 65-70&deg;.
<p>
A wide range of <b>fluorophores</b> (FAM, TET, TMR, Texas Red, Cy3, Cy5, Rox) 
are available.  As for quenchers, DABCYL, BHQ-1 and BHQ-2 have broad specificity 
for the fluorophores.  Fluorophore choice is often instrument-specific. Many 
instruments allow multiplex detection, which requires that fluorophore emission 
does not overlap.

<h5>Beacon Synthesis</h5>
<p>
There is a <a href="http://www.molecular-beacons.org/">list of commercial oligo 
synthesis companies</a>.  They also purify and characterize as part of the 
services.  In addition to the beacon, the complementary strand representing the 
target should be ordered for synthesis.

<h3>Quantitative PCR (Q-PCR)</h3>
<p>
[to be written]

<!-- ==============================================================
     Cells and Viruses/Phages: Specific and General Methods
=================================================================== -->

<h1>Cells and Viruses/Phages:  Specific and General Methods</h1>
<p>
In this section are notes on working generally with certain types of cells
(bacterial, yeast, mammalian, other eukaroytic types) and with viruses
of eukaryotes and prokaryotes (phages).  Information might also be found
in other section regarding facts about the biology of different cells
and viruses.

<!-- ===========================================================
   making competent E coli and transforming them
 =============================================================== -->

<h2 id="xformecoli">Preparation &amp; Transformation of Competent <i>E. coli</i> 
Cells</h2>

<h3>Chemically Competent <i>E coli</i> and Their Transformation</h3>
<p>
Source for notes in this section is Sambrook &amp; Russell,
<i>Molecular Cloning</i>: A Laboratory Manual, 3<sup>rd</sup> Ed., Vol. 1,
p. 1.116-1.118, &copy; 2001.
<p>
Prepare the following reagents:
<ul>
<li>1 M CaCl<sub>2</sub> 2 H<sub>2</sub>O
 <p style="margin-top:0;margin-left:5%;font-size:83%;">
  Thaw 10 ml aliquot and dilute to 0.1 M by adding 90 ml pure water.
  Sterilize with 0.45 &mu;m pre-rinsed filter, and let chill at 0&deg;.
<li>Alternatively used <a href="#TFB"><b>Standard Transformation Buffer</b>
  (TFB)</a>.
<li>MgCl<sub>2</sub>-CaCl<sub>2</sub> Solution
 <ul style="margin-bottom:0;">
 <li>80 mM MgCl<sub>2</sub>
 <li>20 mM CaCl<sub>2</sub>
 </ul>
 <p style="margin-top:0;margin-left:5%;font-size:83%;">
 It should be ice-cold.
<li><a href="#LBmedium">LB</a> or <a href="#SOBmedium">SOB medium</a>
 <p style="margin-top:0;margin-left:5%;font-size:83%;">
  This is for the initial growth of culture
<li>SOB agar selection plates
 <p style="margin-top:0;margin-left:5%;font-size:83%;">
 Prepare plates containing 20 mM MgSO<sub>4</sub> instead of
 10 mM MgCl<sub>2</sub> normally used.
 Include antibiotic used in selection at the appropriate concentration.
<li><a href="#SOCmedium">SOC medium</a>
</ul>
<p>
Other materials required include:
<ul>
<li>Centrifuge with GSA rotor or equivalent
<li>50 ml polypropylene tubes, on ice
<li>17 &times; 100 mm polypropylene tubes (Falcon 2059), on ice
<li>42&deg; water bath
</ul>

<h4>Preparing Cells</h4>
<p>
All steps performed below should be done aseptically.
<ol>
<li>Streak bacteria on a plate and incubate for 16-20 h at 37&deg;.
<li>A single colony (2-3 mm dia) is picked and transferred to 100 ml LB
or SOB medium in 1 liter flask.
<li>Shake vigorously for 3 hours at 37&deg;, and monitor growth at 600 nm
(1 OD<sub>600</sub> unit ~ 10<sup>9</sup> bacterial/ml).
 <p style="margin-top:0;margin-left:5%;font-size:83%;">
 The number of viable cells should not exceed 10<sup>8</sup>/ml, which
 is about OD<sub>600</sub> ~ 0.4.  Sample every 15-20 min, record OD,
 and try to predict when the OD will be 0.4.  Begin harvesting at
 OD of 0.35.  Viable cell numbers can be calculated for each strain
 by doing serial dilutions on LB agar plates w/o antibiotics.
<li>Transfer to sterile 50 ml tubes on ice and leave on the ice for 10 min
 to reduce solution temperature to 0&deg;.
<li>Spin at 2700 &times; <i>g</i> for 10 min at 4&deg;.
<li>Remove supernatant.  Stand tubes upside-down on towels to wick away
   as much medium as possible (about 1 min).
<li>Resuspend cells by swirling or vortexing <em>gently</em> in 30 ml
of ice-cold MgCl<sub>2</sub>-CaCl<sub>2</sub> solution.
<li>Spin the cells again at the same time and force.
<li>Remove the solution and wick on to paper towels as above.
<li>Resuspend by the same gentle technique with 2 ml of ice-cold 0.1 M
CaCl<sub>2</sub>.
</ol>
<p>
It is reported that these cells can be left for 24-48 h at 4&deg; in the calcium 
chloride solution.  Efficiency of transformation increases 4-6 times during the 
first 12-24 h, but starts to fall after this. At this point, cells can be stored 
at &minus;70&deg; after aliquoting them, or transformation can proceed directly.
<p>
To freeze competent cells, do the following:
<ol>
<li>To each 4 ml of cell suspension, add 140 &micro;l DMSO.  Mix gently and
leave on ice 15 min.
<li>Dispense aliquots into chilled, sterile, microfuge tubes or
tissue culture vials.  These should be snap-frozen in liquid nitrogen.
Then stored at &minus;70&deg;
</ol>
<p>
Cells are recovered by taking a vial and thawing in your hand.  As they thaw, 
transfer to ice and leave 10 min.  Transfer cells to the tube where the 
transformation will take place.

<h4>Transformation</h4>
<p>
Use of appropriate positive and negative controls should be part of these
steps.
<ol>
<li>Transfer 200 &micro;l of the competent cells suspended in calcium chloride
to sterile, chilled 17 &times; 100 mm polypropylene tubes with chilled
micropipet tip.
<li>The DNA is then added to each tube.  Use no more than 50 ng in a volume
of 10 &micro;l or less.  Mixing is by gentle swirling.
<li>Let sit on ice 30 min.
<li>Transfer tubes to 42&deg; water bath.  Tubes are left for exactly
90 seconds without shaking.  (Heat shock is important.)
<li>Place tubes on ice-water bath for 1-2 min.
<li>Add 800 &micro;l SOC medium to each tube.
<li>Hold tubes in a 37&deg; water bath for 45 min to permit recovery of
bacterial and expression of antibiotic resistance.  The cells can be
agitated at 50 cycles/min or less during recovery.
<li>Up to 200 &micro;l per 90 mm plate is then plated on SOB agar medium
with 20 mM MgSO<sub>4</sub> and antibiotic selection.  Use a bent glass
rod spreader for plating.
<li>Store plates at room temp until all moisture is absorbed.
<li>Invert plates at 37&deg; for about 12-16 h.  Cells selected for
ampicillin should be plated at low density (&lt;10<sup>4</sup>/90 mm plate)
and not incubated more than 20 h at 37&deg;.
</ol>
<p>
When doing transformations, include a negative control of bacteria <em>not</em> 
containing plasmid DNA.  Plate these bacteria on their own plate.  If bacteria 
grow, consider the following possibilities:
<ul>
<li>An antibiotic-resistant strain has developed or contaminated the
competent cells.  It could even be one of the solutions/reagents.
<li>The antibiotic in the selection plates has degraded or is missing.
<li>The selection plates are contaminated with resistant bacteria.  Bacteria
would be seen in the agar as well as on its surface.
</ul>
<p>
Positive controls are useful but not always available.  A controlled known 
amount of superhelical DNA whose transformation efficiency is known can be used.

<h3>Methods for Producing Chemically Competent <i>Eschericia coli</i></h3>
<p>
The methods below are for the production of chemically competent cells.  The
Inoue et al method is considered better than Hanahan in many ways.

<h4 id="inoue">Inoue et al. Method</h4>
<p>
Efficiencies of 1-3 &times; 10<sup>8</sup> colonies per microgram are
achievable. It is less inconsistent (more reproducible) than Hanahan. Growth is 
at 18&deg; which means it cannot be done in a standard shaker incubator whose 
ambient temperature is above. The alternative is to perform the incubation is a 
cold room with the thermostat set. If not possible, then bacteria can be grown 
at 20-23&deg; (ambient) without changes in efficiency. Because cultures grow 
slowly at this temperature (doubling time is 2.5-4 h), time the start of 
culture in the evening and the growth should reach OD<sub>600</sub> of 0.6 by morning.
<p>
<b>Materials/Buffers/Solutions</b>
<ul>
<li>DMSO of highest quality/recent purchase:  oxidation products can inhibit
transformation
<li>0.5 M PIPES-KOH, pH 6.7<br>
  Dissolve 15.1 g PIPES in 80 ml water.  Adjust to pH 6.7 with 5 M KOH. Bring
  to 100 ml. Sterile-filter.  Aliquot and store at &minus;20&deg;.
<li>Inoue et al. Chemical Competence Reagent.  For each 1.0 liter, to 800 ml
water add the following:
 <ul>
 <li>10.88 g manganese chloride tetrahydrate (MnCl<sub>2</sub> 4 &bull;
 H<sub>2</sub>O, final 55 mM)
 <li>2.20 g calcium chloride dihydrate (CaCl<sub>2</sub> 2 &bull;
 H<sub>2</sub>O, final 15 mM)
 <li>18.65 g potassium chloride  (KCl, final 250 mM)
 <li>20 ml 0.5 M PIPES-KOH pH 6.7  (final 10 mM)
 </ul>
Add water to 1.0 L, then sterile-filter.  Aliquot and store &minus;20&deg;
<li>LB medium
<li>SOB medium
<li>SOB agar plates with 20 mM MgSO<sub>4</sub> + selection antibiotic
<li>SOC medium
</ul>
<p>
<b>Method</b>.  All steps are done aseptically.
<ol>
<li>Ionue et al chemical competence reagent is thawed and kept on ice (0&deg;).
Note water quality should be the highest grade.  If using distilled water with
organic contaminants, treat with activated charcoal before using.
<li>Pick single colony (about 2-3 mm dia) from plate incubated for 16-20 h
at 37&deg;.  Do not use aged plates; always re-plate old colonies for this.
Inoculate 25 ml LB or SOB medium in 250 ml conical flask.  Incubate 6-8 h at
37&deg; at 250-300 rpm.
<li>Late in the day (1800 hours), inoculate three 1-L conical flasks with
250 ml SOB medium, but use different volumes:  flask 1 is 10 ml, flask 2 is 4 ml,
and flask 3 is 2 ml.  Use moderate shaking at a temperature of 18-22&deg;.
<li>Check the OD<sub>600</sub> the next morning.  Monitor OD every 45 min.
<li>When OD<sub>600</sub> is ~0.55, place culture on ice-water bath 10 min.
Discard other two cultures.  (In the future, you could avoid this if results
are consistent.)
<li>Spin cells down at 2500 <i>g</i> for 10 min at 4&deg;.  Discard supernatant
and invert bottle on towels to soak up remaining supernatant.  Use a vacuum
aspirator to get any medium from within container.
<li>Resuspend cells in 80 ml ice-cold Inoue et al buffer.  Use swirling rather than
pipetting or vortexing.
<li>Centrifuge again at 2500 <i>g</i> for 10 min 4&deg;.  Repeat the process of
removing the supernatant as much as possible.
<li>Now resuspend cells in 20 ml ice-cold Inoue et al buffer.  Add 1.5 ml
DMSO.  Mix by swirling and let sit on ice for 10 min.
<li>Quickly aliquot into chilled, sterile micro-conical tubes. 50 &micro;l
aliquots are generally adequate (although it might be larger when trying to
construct cDNA libraries). Then snap-freeze in liquid nitrogen, immersing the
tightly closed tube completely.
<li>Store in &minus;70&deg; freezer and use when necessary.
<li>For the transformation, thaw cells in the palm of hand, then put the cells
in ice water bath and let sit for 10 minutes.
<li>Use a chilled sterile pipet tip to transfer competent cells to a chilled
17&times; 100 mm polypropylene tube.  Keep on ice.  Do not use glass, since
efficiencies are reduced 10-fold.
<li>When transforming do not forget both positive and negative controls.
For each 50 &micro;l of competent cells add up to 25 ng of the transforming
DNA.  The volume added must not be &gt;5% of competent cell volume.
<li>Swirl tubes gently several times to mix.  Control tubes should include
bacteria transformed with known preparation of superhelical DNA, and a tube that
gets no DNA at all.  Set tubes on ice for 30 min.
<li>Transfer tubes to rack in water bath that is set at 42&deg;.  Leave for
exactly 90 s.  Do not shake tubes.  Note that the rate of heat shock can depend
on the tube used.
<li>Return tubes quickly to ice-water bath, and leave 1-2 min
<li>Add 800 &micro;l SOC medium to each tube, and place immediately in 37&deg;
water bath.  Transfer to shaker-incubator at 37&deg;.   Incubate 45 min agitating
at &lt; 225 rpm.
<li>Spread approximate 200-250 &micro;l each of the cell suspension to 100 mm
plates with SOB + 20 mM MgSO<sub>4</sub> + antibiotic.
</ol>

<h3>Hanahan <q>Liquid Gold</q> Method</h3>
<p>
This is protocol 23 (the first listed for preparing competent <i>E coli</i>) in 
the Sambrook &amp; Russell, <i>Molecular Cloning</i>: A Laboratory Manual, 
3<sup>rd</sup> Ed., Vol. 1, p. 1.105-1.111, &copy; 2001.
<p>
Transformation of 5 &times; 10<sup>8</sup> colonies per &micro;g superhelical
plasmid DNA are possible.  This method requires very high purity of reagents;
media and reagents should be recently purchased since they degrade over time.
In addition, stocks of <i>E coli</i> should have been stored in cryogenic medium
at &minus;70&deg;, not at 4&deg; or room temperature.  Glassware should not be
used for any other purpose than growing competent cells.  It should only be washed
with Type I water and autoclaved, and the sterilized water left in the flask
until just before use.  Detergents must not be present.  For most <i>E coli</i>
strains used in transformation, the method works well (DH1, DH5, MM294, JM108 and
JM109, DH5&alpha;, etc).
<ol>
<li>Prepare immediate-use transformation buffer (the buffer when cells are to be
made competent and used immediately):
 <ol>
 <li>dissolve 19.52 g MES in 80 ml Type I water, adjust pH to 6.3 with 5 M KOH, 
  then bring to final volume of 100 ml.  Use 0.45 &micro;m (or smaller) sterile 
  filter to sterilize and divide into 10-15 ml portions and freeze at 
  &minus;20&deg;.
 <li>For each 1 L of transformation buffer, dissolve in this order of addition
   in water 8.91 g MnCl<sub>2</sub> &bull; 4 H<sub>2</sub>O (final 45 mM), 
   1.47 g CaCl<sub>2</sub> &bull; 2 H<sub>2</sub>O (final 10 mM), 7.46 g
   KCl (final 100 mM), and 0.80 g Co(NH<sub>3</sub>)<sub>6</sub> Cl<sub>2</sub>
   (final 3 mM).  Make sure all salts are dissolved in about half of final volume
   of water, then add 10 ml of 1 M MES pH 6.3 (final 10 mM).
 <li>Filter sterilize.  Then aliquot into 40 ml volumes in tissue culture flasks
  and store 4&deg;.
 </ol>
<li><b>Frozen storage buffer</b> (FSB).  This is used when cells that are competent
are to be stored frozen.  The buffer is not MES but intead potassium acetate.
 <ol>
 <li>Dissolve 9.82 g KOOCCH<sub>3</sub> in 90 ml Type I water and adjust pH to 7.5
with 2 M acetic acid.  Final volume is brought up to 100 ml.  Aliquote in 10-15 ml
portions and freeze as for MES.
 <li>Prepare the FSB now by mixing the Mn, Ca, K, and Co salts in the same amounts
 per liter.  Then add 100 ml (per liter) (final 10%) of glycerol.  Add 10 ml
 of potassium acetate (instead of MES) to final 10 mM concentration.  The pH 
should be adjusted down to 6.4 with 0.1 N HCl <em>carefully</em>.  If too much 
HCl is added, base cannot be added to adjust it up...it must be discarded and 
made again.
 <li>Sterile filter this solution and aliquot as for the standard transformation
buffer.
 </ol>
<li>Recover <i>E coli</i> from frozen stock.  Use inoculating loop to streak on an 
SOB agar plate and incubate 16 h (overnight) at 37&deg;.  Note that there is no
need to thaw the tube:  enough bacteria are yielded by scrathing the surface.
<li>Transfer 4 or 5 single colonies (not overgrown, i.e. &le; 2-3 mm diameter) 
to 1 ml SOB medium containing 20 mM MgSO<sub>4</sub>. Vortex at moderate speed 
to disperse bacteria.  Transfer the 1 ml dispersion to a 1 L flask containing
30-100 ml SOB medium + 20 mM MgSO<sub>4</sub>.
<li>Incubate 37&deg; for 2.5-3.0 h.  It is important to harvest cells when the 
OD<sub>600</sub> reaches ~0.4 AU (&#8815; 10<sup>8</sup> cells/ml);  monitor the
OD<sub>600</sub> every 15-20 min in order to predict when OD reaches 0.4.  Best
competence is obtained with cells in early or mid log phase.  To get an accurate
growth phase determination, it may be necessary to sample media and grow dilutions 
on LB plates to count vialbe cells.
<li>Transfer cells to sterile ice-cold 50 ml conical tubes and put on ice for 10 
min.
<li>Spin the tubes (2700 <i>g</i>) for 10 min at 4&deg;.
<li>Decant the medium carefully (maintaiin the pellet), then invert the tubes for
about 1 min to recover traces of medium
<li>Resuspend pellets now in either the standard transformation buffer or the
frozen transofmation buffer, depending on whether the cells will be made competentn
and used immediately or stored.  Use ~20 ml buffer for each tube.
<li>Spin the cells again as above and invert 1 min to drain away traces of buffer.
<li>Now add about 4 ml of <em>ice-cold</em> buffer per tube, and swirl or gently
vortex to resuspend pellets.
<li>Prepare DMSO/DTT (DnD) solution:  To final 10 ml, prepare 1.53 g DTT 
and 9 ml DMSO, add 100 &micro;l of 1 M potassium acetate pH 7.5, and bring to final
volume with water.  Sterile filter using Millipore Millex SR (or equivalent) 
which can withstand DMSO.  Dispense to 160 &micro;l aliquots in sterile 500 &micro;l
microfuge tubes.  Close tightly and store &minus;20&deg;.  DTT is 10 mM, DMSO is
90%, and potassium acetate is 10 mM.
<li><b>Immediate-use competent cells</b>.
 <ol>
 <li>Add 140 &micro;l DnD into the center of each cell suspension and immediately
mix by swirling, then put on ice for 15 min
 <li>Add another 140 &micro;l to each suspension, swirl gently and leave another
15 min on ice.
 <li>To chilled sterile 17 &times; 100 mm polypropylene (not glass!) add aliquots
of the bacterial suspensions and put on ice.  50 &micro;l aliquots should be sufficient,
but use a larger volume when large counts of transformed colonies are required.
 </ol>
<li><b>Deferred-use competent cells (freezing)</b>.
 <ol>
 <li>Add 140 &micro;l of DMSO to 4 ml of resuspended cells, and mix by swirling
and store 15 min on ice.
 <li>Repeat the DMSO addition and place on ice bath.
 <li>Dispense aliquots of the cell suspension insot chilled sterile micro-conical
tubes or into tissue culture vials.  Snap freeze in liquid nitrogen and then store
tube at &minus;70&deg; until needed.  50 &micro;l aliquots are useful in most
purposes, but larger volumes should be used where higher counts of transformed
colones are required (e.g., cDNA libraries).
 <li>When frozen cells are needed, remove tubes from the freezer, thaw by holding
in palm of hand.  Place on an ice bath after thawing and leave 10 min.
 <li>Use chilled sterile pipet tips to transfer the cells to chilled, sterile
17 &times; 100 mm polypropylene (not glass!) tubes.  Keep the tubes on ice.
 </ol>
<li>Add the DNA to be transformed to the competent cells.  No more than 25 ng
DNA should be added per 50 &micro;l of competent cells.  Do not add this in
a volume &gt; 5% of the competent cell volume.  Swirl tubes several times gently
to mix.  Also include controls:  one in which a standard preparation of 
superhelical plasmid (no insert) and another with no plasmid at all.  Put
tubes on ice for 30 min.
<li>Place tubes in 42&deg; water bath.  Keep exactly 90 seconds without any
shaking.  Note this heat shock rate works for Falcon 2059 tubes (17 &times; 100
mm polypropylene tubes).
<li>Transfer tubes quickly to an ice bath.  And allow cells to cool for 1-2 min.
<li>Add 800 &micro;l of SOC medium to each tube.  Warm to 37&deg; in water
bath.
</ol>

<h3 id="blue-white-xform">Plating Transformed Blue-White <i>E coli</i></h3>
<p>
This method is from Sambrook and Russell (<i>Molecular Cloning</i>, 3rd Ed).
<p>
You must use a strain of <i>E coli</i> that has &alpha;-complementation for
expression of &beta;-galactosidase and you must use as a vector one that
encodes the <i>lacZ</i> gene, and which features a cloning site that inactivates
the <i>lacZ</i> when the DNA of interest becomes inserted (cloned).

<ol>
<li>Note: transformation is not explained here:  the <i>E coli</i> described 
here must already have been transformed with the recombinant plasmid.  As controls,
use vector that has no insert (will produce blue colonies), parent strain of
bacteria with no vector/plasmid (will produce white colonies), and any other 
controls.  Concentrations of the bacteria must be adjusted to 3000 bacteria per
100 &micro;l for 90 mm plates, and 10,000 bacteria per 100 &micro;l for 150 mm
plates.
<li>Aliquot melted top agar&mdash;3 ml for 90 mm plates, 7 ml for 150 mm 
plates&mdash;into 17 &times; 100 mm tubes and place the tubes 45&deg; heating 
block or water bath.  Use agar mixed with LB or YT medium.
<li>To each tube, remove from the heating block or water bath, add 100 &micro;l
bacterial suspension with &lt; 3000 viable bacteria for 90 mm place and &lt; 10,000
for 150 mm plate.  Disperse the bacteria by closing tube top and inverting several
times.
<li>For the 90 mm plates with 3 ml agar, add 40 &micro;l of 2% X-Gal and 7 &micro;l
of 20% IPTG to the solution.  For 150 mm plates with 7 ml agar, add 100 &micro;l
of the X-Gal, and 20 &micro;l IPTG.  
<li>Quickly pour out the agar-bacteria mixture on to the center of a plate with
hardened bottom agar which contains medium with antibiotic selection.  Swirl the
top agar around to distribute it.  Then continue with the other tubes and transformed
bacterial suspensions.
<li>After the top agar hardens at room temperature, remove condensation from the lid
and incubate the plates inverted overnight (12-16 h) at 37&deg;.
<li>Remove from the 37&deg; incubator and place at 4&deg; for several hours, allowing
blue color to develop.
<li>Colonies pale blue at center and deep blue at periphery are wild-type plasmids
without the insert.
<li>Colonies creamy-white or eggshell blue with faint blue spot at center sometimes,
are recombinant.
<li>To obtain the desired colony, stab into the thin layer of soft agar with
sterile inoculating needle or toothpick, and transfer to medium with selecting
antibiotic.
</ol>

<!-- ===========================================================
   titering bacteriophages
 =============================================================== -->


<h2 id="phage-titer">Titering Bacteriophages</h2>
<p>
This method should apply to a wide variety of bacteriophages that produce
visible or detectable plaques on a lawn of bacterial host.  The method
below is how to determine the titer for &lambda; phage, but should be similar
for any phage.
<p>
Any materials used to make contact must be sterilized.  The following steps
are performed
<ol>
<li>Grow the host (<i>E coli</i>) to saturation in medium.  For &lambda; phage, 
grow in &lambda; broth (10 g tryptone + 2.5 g NaCl per L) containing 0.2% 
maltose and 10 mM MgSO<sub>4</sub>; induces <i>lamB</i> protein, the &lambda; 
receptor being the maltose transporter, and the Mg<sup>2+</sup> helps adsorption 
of phage.
<li>Prepare top agar (&lambda; broth in 0.7% agar) in microwave using the 
defrost setting (or in boiling water bath for 15 min).  Let top agar cool to
room temp for 5 min, then put the bottle in 45-50&deg; water bath.
<li>Put 300 &micro;l of <i>E coli</i> culture into five 8 &times; 80 mm tubes.
<li>Prepare serial dilutions of phage in a suspension medium (LB medium is
usefuly).  Dilutions can be done by two orders of magnitude with each serial
step.  From each dilution 100 &micro;l of suspension will be used for inoculation.
In preparing dilutions, sterile 16- or 18-mm tubes can be used, and 3-5 ml total
volume, transferring 30-50 &micro;l tube-to-tube.
<li>From each step dilution series, add 100 &micro;l to each of the five tubes
containing <i>E coli</i>, noting which tube for which dilution.  Leave at room
temp for 20 min.
<li>Heat tubes to 37&deg; in water bath or heating block for 10 min.  During
this incubation, label five prewarmed &lambda; plates according to tubes.
&lambda; plates are &lambda; broth in 1% agar, should not contain moisture 
(condensate), but should be fresh.
<li>To each tube with phage-infected <i>E coli</i> broth, add 2.5 ml top 
agar, vortex gently to mix, and pour contents out to a plate.  Make sure agar
is spread uniformly over all the plate.
<li>Incubate 37&deg;.  After 6-8 h plaques should appear, but after 12 h will
be easier to score and count.  Compute the titer.
<li>In an optional step, find a plate with lowest plaque count (single plaques
whose edge does not overlap with another plaque), and pick plaque with sterile
capillary tube or toothpick (a capillary tube yields a plug of agar with plaque
and this can be blown into tube with 1 ml suspension medium (or tip of toothpick
agitated into medium.
</ol>

<!-- ===========================================================
   isolating M13 phage DNA
 =============================================================== -->

<h2>M13 Phage DNA Isolation</h2>

<h3>Isolation of RF DNA form of M13 Phage</h3>
<p>
<cite>[Method used in laboratory of Bruce Roe at Univ. Oklahoma]</cite>
<p>The RF form is a ds DNA form of the M13 phage, which normally is a ssDNA
genome in the phage body.
<ol>
<li>In 250 or 500 ml sterile Erlenmeyer with 50 ml 2&times; TY medium add
frozen (glycerol) stock of <i>E coli</i> JM101 and pre-incubate 1 h at 37&deg;
(no agitation), letting it proceed to early log phase.  Distribute 1.5 ml
aliquots to four tubes.
<li>Pick a plaque of the desired M13 clone to each of the four tubes (thus four 
single plaques).  Incubate 8 h at 37&deg;. (call these Tubes Z).
Proceed to next steps immediately.
<li>Take 1.5 ml aliquot from early log phase <i>E coli</i> (uninfected) and put 
one aliquot in four flasks containing fresh 50 ml 2&times; TY and grow 8 h or 
overnight at 37&deg; (flasks A).  Put another 1.5 ml aliquot in four flasks 
containing 50 ml of fresh 2&times; TY medium, and grow this for 2 h at 37&deg;, 
and put in cold room (flasks B).
<li>Take out flasks B from cold room after Tubes Z have finished their 
incubation, add the 1.5 ml of Tubes Z to these flasks.  Incubate 8 hours
at 37&deg;.  Call these flasks C.
<li>After flasks C have incubated 6 hours, flasks A should have completed 
their 8 h incubation.  To each of four flasks (2, 3 or 4 L size?) containing 1.0 L fresh
2&times; TY, add the 51.5 ml contents of flasks A.  These contain again just
<i>E coli</i> cells.  Call these big flasks "flasks D."  Incubate these 
for 2 h at 37&deg;.
<li>The 2 h flask D and 8 h flask C incubation should have completed 
at the same time now.  Pour the ~50 ml of flask C (with M13 phage) into
flask D.  Perform an additional 8 h (or overnight) incubation at 37&deg;
<li>With all coordinated incubations complete, cells are harvested from the
big flasks:  spinning in large volume capacity bottles (500 ml) at 6000-8000
<i>g</i> for 20 min.
<li>Resuspend cell pellets in fresh 2&times; TY and transfer to two bottles
and spin.  Decant media (remove extracellular phage with ssDNA).  At this
point cell pellets can be frozen &minus;70&deg; or proceed.
<li>Resuspend cell pellets in total of 120 ml 1&times; STB buffer (30 ml for 
each centrifuge bottle); 1&times; STB is 25% sucrose in 50 mM TrisHCl pH 8.0, 
filter-sterilized and stored 4&deg;.  Pellets are teased with a spatula.  Add 
total 24 ml (6 ml for ea. bottle) of lysozyme solution (5 mg/ml lysozyme in 50 
mM TrisHCl pH 8.0 + 10 mM EDTA, made in sterile water), gently mix, incubate 5 
min in ice-water bath
<li>48 ml of 50:2:10 TTE buffer (12 for each bottle) and 2 ml of 10 mg/ml RNase A
(0.5 ml for each bottle), which are mixed and incubated ice-water bath for 5 min.
<li>Pour lysate through 2 layers of cheesecloth to catch precipitated SDS, proteins,
membranes, chromosomal DNA.  Lysate is transferred to 250 ml centrifuge bottle, spun
in centrifuge at 15,000 <i>g</i> for 30 min.  [50:2:10 TTE buffer is 50 mM 
TrisHCl pH 8.0, 2% Triton X-100, and 10 mM EDTA]
<li>Add 6 ml of 5 mg/ml ethidium bromide.  Add cesium chloride so that final
[CsCl] = 1 g per ml.
<li>Transfer to 35 ml polyallomer tube.  Top off with solution of 1 vol of
100 mM TrisHCl pH 8.0 + 10 mM EDTA mixed with 1 vol of 1 g/ml cesium chloride.
Make sure air bubbles removed.  Seal with rubber stoppers, and crimp.
<li>Spin in ultracentrifuge (like Sorvall OTD-75B in T-865 rotor at 60,000 rpm, 
which is about 350,000-400,000 <i>g</i>) overnight (16-20 h) at 15-20&deg;.
<li>Detect the bands in the tubes using long-wave UV light.  Get the lower band
with a 5 ml syringe and 25 gauge needle.  It might be better to first remove the
solution and upper band above with just a 18-20 gauge needle.
<li>Ethidium bromide is removed on a prepared Dowex AG (BioRad) column, 1.5 
volume.  Collect 500 &micro;l fractions.  The Dowex AG resin is prepared with 1 
M NaOH, water, the 1 M TrisHCl pH 7.6, making sure eluate has a pH of 7.6.
<li>Decide which fractions have A<sub>260</sub> &ge; 1.00 and pool them.  Put
into 35 ml Corex glass tubes, add 1 vol water, then 2.5 vols ice-cold 95% EtOH.
Leave 2 h at &minus;20&deg;, spin 45 min at 10,000 <i>g</i> (4&deg;?).
<li>Remove supernatant gently, add 80% EtOH, centrifuge, remove supernatant,
the dry in vacuum oven.
<li>Resuspend the RF DNA in 10 mM TrisHCl pH 7.6, 0.1 mM EDTA.
</ol>

<h3>Isolation of M13 Single-Stranded DNA (Genome)</h3>
<p>
This has many applications, including use of M13 for DNA sequencing.
Perform all the steps below using sterile technique.
<ol>
<li>Prepare an early log phase <i>E coli</i> JM101 culture (see the RF M13
isolation for how to do this).  Transfer 1.5 ml aliquots to 12 &times; 75 tubes.
<li>From an M13 plaque plate, pick single plaques using sterile toothpicks
and place into the 12 &times; 75 mm tubes.  Shake at 250 rpm in a 37&deg;
incubator for 4-6 h.
<li>Transfer all cultures to 1.5 ml conical tubes and spin in 4&deg; 
microcentrifuge at 12,000 rpm 15 min.
<li>Transfer 1 ml of supernatant from the top to new 1.5 ml tube which contains
200 &micro;l 20% PEG-8000 + 2.5 M NaCl.  Invert several times to mix.
Let sit 15-30 min (room temp).
<li>Spin in microcentrifuge at max speed for 15 min at 4&deg;.  Carefully remove
as much supernatant as possible.
<li>Resuspend pellet in 100 &micro;l 10 mM TrisHCl pH 7.6 (use vortex).
Add 50 &micro;l TE-saturated phenol.
<li>Perform a standard extraction with the phenol.  Then extract the phenol
from the aqueuos phase with hydrated ether twice (use 1 vol ether with 1 vol
aqueous phase).  Finally precipitate with cold ethanol and let dry.
<li>Resuspend dried DNA in 6 &micro;l 10 mM TrisHCl pH 7.6 + 0.1 mM EDTA
</ol>
<p>
Can be used immediately for sequencing.


<!-- ==============================================================
	  Methods to Maintain DNA of Interest In Vivo
=================================================================== -->

<h1 id="cloning">Methods to Maintain DNA of Interest In Vivo</h1>
<p>
In general, ds DNA of interest will be inserted into either a <b>cloning
vector</b> or an <b>expression vector</b>.
<p>
The <b>cloning vector</b> should be used to protect against loss of the ds
DNA and for its easy recovery.  An appropriate host that maintains the plasmid
which has its recombination genes inactivated so as not to integrate plasmids
into the host chromosome should be used.
<p>
If the inserted DNA codes for a protein, then its primary usefulness will be 
to express that protein within an <b>expression vector</b>.  Other vectors may be
created which analyze the DNA of interest if it codes for transcriptional elements
such as a promoter, in which case it may be inserted into a plasmid at a site
5&prime; to a reporter gene.

<!-- ===========================================================
   inventory maintenance
 =============================================================== -->


<h2 id="controlling-inventories">Maintaining Inventories of Vectors, Plasmids, and 
Host Strains</h2>
<p>
When purchasing a kit or receiving any material which is replicable&mdash;whether
DNA, RNA, or a living microbial form&mdash;the first thing to do is to safeguard
that material by replicating it in minimally sufficient amounts for the inventory,
especially if the DNA or bacterial strain is difficult to obtain or costly to
purchase repeatedly.
<p>
For any vector or plasmid DNA supplied in <i>in vitro</i>, purified or not, the 
first step is to transfect an appropriate maintenance host, generally a host 
recommended by the manufacturer, or indicated in the literature.
<p>
If the material is an RNA who sequence is not known, then the first step is to 
reverse transcribe it into a stable DNA form, and then to follow the steps above
for dealing with (plasmid or vector) DNA.
<p>
If the material is a living microbial host or strain, follow the directions
for growing and storing it long-term at the appropriate temperature.  If there
is no information or guidance, then growing it in a rich medium and diluting
20-50% of its volume with a compatible cryoprotectant (such as glycerol) and
storing it at &minus;80&deg; or colder (on liquid nitrogen) is usually sufficient.

<p>
If the plasmid/vector is supplied in the kit purified in vitro, perform
the following steps to secure its vivo and in vitro supply:
<ol>
<li>Using the appropriate host (usually some bacterial host that supports it,
especially if it is a shuttle vector), transfect the plasmid/vector into that
host using competent cells and grow the cells in small (2 ml) cultures in rich
medium overnight or for a period suitable to the strain after it has been
transfected.
<li>Prepare the setup for 3-10 minipreps (at least 3).  Transfer an aliquot
of the transfected cells in the first culture to new cultures that are designed
for a miniprep.  Use rich medium (or minimal, if suitable) and include the 
selective reagent or process (antibiotic, restrictive temperature, whatever)
that selects for hosts that can only grow if they were transformed with the DNA
<li>Verify/confirm the presence of the DNA within the host using at least two
identifying methods, of which the usual are listed below:
 <ul>
 <li>PCR using appropriate primers, resulting in bands of expected size
 <li>restriction digest analysis using at least two enzymes, with predicted
   sizes observed
 <li>Use of labeled probes 
 <li>Test of a biological function as a feature of the cloned DNA
 </ul>
<li>Store the DNA (plasmid/vector) in <i>in vivo</i> in conditions suitable for 
the long-term storage of the microbial host.  For hosts such as <i>E coli</i> it 
might be a rich medium containing glycerol.  For other host types, check the 
literature or the web for long-term storage (e.g., ATCC has a <a 
href="http://atcc.custhelp.com/app/answers/detail/a_id/976/~/long-term-storage-of-cultures">guide</a> 
on long-term storage
<li>Finally prepare the DNA (plasmid or vector) in <i>in vitro</i> form and
purified for quick use.  As a general target, 1 nmol of the DNA should be 
prepared and purified, and this should be stored in 20 tubes of 50 pmol/tube.
It is probably a good idea not to store these aliquots in the same storage
place (room, refrigerator, freezer), but to divide them in half and put them
in two different places under identical storage conditions (this is the 
not-all-cookies-in-one-jar principle). [Note that to calculate the 1 nmol
total in micrograms required, multiply the base pair count by 0.66 for
double-stranded DNA or 0.33 for single-stranded DNA.  For example if
the plasmid is 3000 bp ds DNA, then the number of micrograms required is
3000 &times; 0.66 ~ 2000 &micro;g = 2 mg.]

</ol>
<p>
<b>How to store purified DNA in vitro?</b>  This question is not perfectly 
settled according to many.  Some say that if it to be stored for a very long
term, that it is best to store it after it has been precipitated and 
washed in alcohol, then allowed to dry, with all residue of alcohol removed.
Then it can be stored at &minus;20&deg;.  However, it has been noted that
DNA dried this way takes a long time to go into solution once it is rehydrated.
Still it is a good idea if it will be long term.  In this case, when rehydrating
DNA stored lyophilized or in dry condition for a long period, allow the rehydration
to take place over a 24 h period, and you can be sure it will have resolubilized.
<p>
An alternative is store the purified DNA in a 10:1 TE solution at pH 8.0 in 
minimal volume and then freeze it at &minus;80&deg; or colder.  This has been 
recommended by some if storage is not long-term (&lt; 1 year).
<p>
<b>Other Guides</b>.  Read page 1.29 of the 3<sup>rd</sup> edition of 
<i>Molecular Cloning:  A Laboratory Manual</i> for how to handle bacteria and 
DNA (plasmids, vectors).  It gives specific instructions in good detail.
They include a plating step between the first culture and the cultures for
preparing the DNA from a mini- or midi-prep, in order to control colony appearance.


<!-- ===========================================================
   directional cloning
 =============================================================== -->

<h2>Directional Cloning into Plasmid Vectors</h2>
<p>
If a vector (plasmid) is cut with a single restriction enzyme, and the
insert DNA of interest is cut with the enzyme or linkers are added to
the 3&prime; and 5&prime; ends of the insert, the chances are high that many unwanted
products may form when the insert and the vector are mixed with a
DNA ligation reaction system.  First of all, the vector and the insert
could ligate their own ends in an intramolecular reaction.  Then there
are the reactions in which the insert and vector ligate alternately
or in multiple copies, before they might form a circular re-sealing.
One can alter the stoichiometries to prevent multiple copies, such as
putting 1 insert for every 100 copies of vector.  In that case, one can
select for the rare case an insert makes it with a vector.
<p>
Something interesting about restriction enzymes is that while they may
recognize different sequences, the protruding termini they generate
can be identical.  <i>Bam</i>HI and <i>Bgl</i>II both recognize 6-base
sequences that are different, but the 3&prime; protruding ends are the same.
Suppose that the vector and the insert were digested with both enzymes.
When mixed with each other, insert will orient in both directions.
However, when digested with one or the other enzymes, one of the inserts
ends will cut and the other will not, and the direction will be known.
The enzyme might be included in the ligation mix, or if not compatible,
the digestion can be done after ligation.  The reaction exploits the
major difference in transformation between closed circular and linear DNA.
<p>
Other strategies include ligating synthetic linkers/adaptors to linearized
insert or vector, amplifying insert with primers that include restriction
sites, and filling 3&prime; recessed ends of either the insert or the vector
to become blunt-ended and this control ligation this way.  The latter
strategy especially prevents oligomerization and circularization during
ligation.

<ol>
<li>The insert and the vector are digested with the two appropriate restriction 
enzymes.  Use 10 &micro;g vector.  Avoid enzymes that will cleave within 12 bp 
of one another if the digestion is of a multiple cloning site in a vector, since 
cleavage by the 2nd enzyme after the 1st enzyme cuts is less effienct.  When the 
sites are close by necessity, cut first with the enzyme that has a lower 
catalytic efficiency, and then second with the enzyme having the higher 
efficiency.  NEB keeps a table on the efficiency of restriction catalysis for 
the different enzymes.  Simultaneous digestion may be possible if the enzymes 
are compatible in the same buffer system;  if not, digest first with the enzyme 
that requires the lower salt concentration, verify the cut by agarose gel 
electrophoresis, and when verified increase the salt concentration and do the 
second digestion.
<li>The digested insert is extracted with phenol:chloroform and precipitated 
with ethanol.  Alternatively, gel purify on neutral agarose or polyacrylamide 
the insert DNA.  This eliminates elements present after the digestion of the 
insert DNA that will compete for ligation and determination of which products 
are desired elements.
<li>Clean up the vector by spin-column chromatography and then ethanol 
precipitation.  Generally a small fragment is removed from the double digest 
that passes through a pore in a membrane.
<li>The insert and the vector are resuspended in TE pH 8.0 to a concentration of 
~100 &micro;g/ml.  Convert the concentration to nM (pmol/ml using the fact that 
1 bp has a mass of 660 Da.  Confirmation by gel analysis might be advisable.
<li>The table belows shows tube labels and their contents.  Prepare the tubes 
according to the table.
 <table>
 <tr><th>Tube        <th>DNA
 <tr><td>A &amp; D	<td>vector, 30 fmol (~100 ng)
 <tr><td>B           <td>insert,  30 fmol (~10 ng)
 <tr><td>C &amp; E   <td>vector + insert, 30 fmol
 <tr><td>F           <td>superhelical vector, 3 fmol (~10 ng)
 </table>
<li>Add to tubes A, B, and C the following:
 <table class="none" style="margin:0 0 0 10%;">
 <col style="width:20em;">
 <tr><td>10&times; Ligation buffer   <td>1.0 &micro;l
 <tr><td>Bacteriophage T4 DNA ligase <td>0.1 Weiss unit
 <tr><td>10 mM ATP                   <td>1.0 &micro;l
 <tr><td>H<sub>2</sub>O              <td>to 10 &micro;l
 </table>
To tubes D and E, add the following:
 <table class="none" style="margin:0 0 0 10%;">
 <col style="width:20em;">
 <tr><td>10&times; Ligation buffer   <td>1.0 &micro;l
 <tr><td>10 mM ATP                   <td>1.0 &micro;l
 <tr><td>H<sub>2</sub>O              <td>to 10 &micro;l
 </table>
Add all components but the ligase first. Before adding ligase, warm the tubes up 
to 45&deg; for 5 min (melts cohesive termini), then chill to 0&deg;.
<li>The ligase reaction is incubated at 16&deg; overnight, or at 20&deg; for 4 
hours.
<li>Competent <i>E. coli</i> are then transformed with dilutions of the ligase 
reactions.  Expected numbers of transformed colonies are zero or near-zero for 
tubes A, B, and D.  A and D will at least have 10,000-fold fewer colonies than 
Tube F.  Tube C should have ~10-fold more than Tubes A or D.	Tube E should have 
some, but fewer than Tube C. Tube F will probably produce &gt;2 &times; 
10<sup>5</sup>.
</ol>


<!-- ===========================================================
   dephosphorylation of DNA using phosphatases
 =============================================================== -->

<h2>Dephosphorylating Plasmid DNA</h2>
<p>
Removing 5&prime;-phosphate groups helps to prevent self-ligation or 
circularization of plasmid.  DNA ligase requires a 5&prime;-phosphate group on 
one nucleotide and a 3&prime;-hydroxyl on the other.  It is usual to ligate 
5&prime;-phosphorylated foreign DNA to dephosphorylated vector, and then 
transform bacteria with the open circular material rather than linear fragments.
<p>
There is not wide agreement that dephosphorylation works however. 
Recircularization is suppressed, but there is also a decline in the count of 
colonies with the desired recombinant.  There is also a wide belief that 
dephosphorylation of the 5&prime; end leads to increased frequency of rearranged 
or deleted clones.
<p>
Dephosphorylation of the vector is only recommended under the following 
conditions:
<ul>
<li><b>The insert is available only in small amounts</b>.  A 10-fold
molar excess of dephosphorylated vector over the insert ensures that
all insert is ligated to vector.
<li><b>When transformants are to be screened by restriction enzyme digestion
of minipreps of plasmid DNA</b>.  As the preparation of more than a dozen
or two small-scale bacterial cultures is tedious, use of dephosphorylated
vector ensures a high frequency of desired recombinants in small sample
of transformants.
<li><b>When cloning blunt-ended fragments of DNA</b>.
<li><b>If vector prepared by cleavage with two different enzymes generates
a large number of transformed colonies</b>.  Such would be the case if
either one of the two enzymes failed to cut the vector completely or
the small fragment from the multiple cloning site has not beem removed
from the vector preparation but is instead ligated into vector.  
Dephosphorylation of vector should prevent its recircularization of the
vector alone.
</ul>
<p>
There is no need for dephosphorylation for inserts with complementary protruding
ends to be screened by &alpha; complementation or colony hybridization, which are
easily visible even among high number of background colonies.

<ol>
<li>Digest a good amount (10 &micro;g) of the closed circular plasmid DNA with
2- to 3-fold excess of restriction enzyme for 1 h
<li>Examine the digest (0.1 &micro;g aliquot) on 0.7% agarose gel in TBE with 
0.5 &micro;g/ml EtBr along with undigested plasmid.  Continue treatment with 
more enzyme if digest is not complete.
<li>Extract with phenol:chloroform and add EtOH for standard precipitation:
keep ethanolic solution of ice for 15 min.  Spit max speed at 4&deg; for 10 min
and dissolve in 110 &micro;l  of 10 mM TrisHCl pH 8.3.  Set aside 20 &micro;l of
this as a control.
<li>To the 90&micro;l linearized plasmid, add 10 &micro;l of 10&times; buffer 
for other calf intestinal alkaline phosphate (CIAP) or shrimp alkaline 
phosphatase (SAP), depending on what is used.  Then add the enzyme CIAP or SAP
according to the table below, depending on how the restriction enzyme cuts the
DNA:
  <table>
  <tr><th>Terminus      
  <th style="width:12em;">Amount of Enzyme per Mole DNA Ends  <th>Incubation Temp/Time
  <tr><td rowspan="2">5&prime; protruding 
      <td>0.01 U CIP  <td>37&deg;, 30 min (twice)
	<tr><td>0.1 U SAP   <td>37&deg;, 60 min
  <tr><td rowspan="2">3&prime; protruding 
      <td>0.1-0.5 U CIP  <td>37&deg;, 15 min<br>2nd aliquout: 55&deg;, 45 min
	<tr><td>0.5 U SAP   <td>37&deg;, 60 min
  <tr><td rowspan="2">blunt
      <td>0.1-0.5 U CIP  <td>37&deg;, 15 min<br>2nd aliquout: 55&deg;, 45 min
	<tr><td>0.2 U SAP   <td>37&deg;, 60 min
  </table>
<li>If using CIAP:  Add SDS to 0.5% and EDTA pH 8.0 to 5 mM (from 100&times; 
stock).  Add proteinase K to 100 &micro;g/ml (from 100&times; stock) then 
incubate 30 min at 55&deg;. If using SAP:  incubate at 65&deg; for 15 min. 
Extract protein once with phenol and then once with 1:1 phenol-chloroform and 
precipitate with EtOH keeping it on ice for 15 min. Wash with cold 70% EtOH, 
allow to dry and resuspend in TE pH 8.0 at about 100 &micro;g/ml.  Store 
aliquotted at &minus;20&deg;.
</ol>


<!-- ===========================================================
   ligating adapters
 =============================================================== -->

<h2>Attaching Adaptors to Protruding Termini</h2>
<p>
Adaptors are attached to protruding termini using DNA ligase.  The ligase
enzyme requires phosphorylated 5&prime;-ends.  If adaptors are purchased, they
can be purchased to include phosphorylation or not.  If they are not
phosphorylated, they must be treated with polynucleotide kinase to
phosphorylate them first.
<ol>
<li><b>Phosphorylation of Unphosphorylated Adaptors</b>.  Add the following
to a sterile microfuge tube:
 <table class="none" style="margin:0 0 0 10%;">
 <col style="width:20em;">
 <tr><td>synthetic oligonucleotide or adaptor   <td>0.5-2.0 &micro;g, dissolved in TE pH 8.0
 <tr><td>10&times; linker kinase buffer         <td>1.0 &micro;l
 <tr><td>10 mM ATP                              <td>1.0 &micro;l
 <tr><td>water                                  <td>to 10.0 &micro;l
 </table>
Finally add 1.0 unit of phage T4 polynucleotide kinase.  Incubate at 37&deg; for 
1 hour.  Purification of the products is unnecessary, and the reaction mixture 
can be used directly in the ligase reaction.
<li>The ligation reaction is set up on ice-water as follows:
 <table class="none" style="margin:0 0 0 10%;">
 <col style="width:20em;">
 <tr><td>DNA fragment                  <td>100-200 ng
 <tr><td>phosphorylated adaptor        <td>10-20-fold molar excess
 <tr><td>10&times; ligation buffer     <td>1.0 &micro;l
 <tr><td>10 mM ATP                     <td>1.0 &micro;l
 <tr><td>water (H<sub>2</sub>O)        <td>enough to a final 10 &micro;l volume
 <tr><td>phage T4 DNA ligase           <td>0.1 Weiss unit
 </table>
The reaction is incubated at 4&deg; for 6-16 h.  Ligation is more efficient when 
the volume is as small as possible (5-10 &micro;l).  Mixing the ATP with the 
buffer is often advisable to keep the volume small.
<li>The ligase is inactivated by placing at 65&deg; for 15 min.
<li>Add 10 &micro;l of the appropriate 10&times; restriction enzyme buffer. 
Calculate the amount of enzyme necessary for a final volume of 100 &micro;l. Add 
the amount of water necessary to bring the final volume to 100 &micro;l, less 
the volume of the enzyme.  Then add 50-100 U restriction enzyme. A large amount 
of enzyme is added because of high concentration of adaptors.
<li>Incubate 37&deg; for 1-3 h.
<li>The digestion reaction is extracted with phenol:chloroform and the DNA
recovered by ethanol precipitation.
<li>Spin out the precipitated DNA at maximum speed for 15 min at 4&deg;.
Resuspend in 50 &micro;l TE pH 8.0.
<li>The resuspended DNA is passed through a spin column to remove excess
adaptors and the cleavage products.
<li>The modified DNA fragment can now be ligated to the vector with
protruding ends that are complementary to the cleaved adaptor.
</ol>

<!-- ===========================================================
   attaching adapters/linkers to blunt DNA
 =============================================================== -->

<h2>Adding Synthetic Linkers to Blunt-Ended DNA</h2>
<p>
Linkers contain desired restriction sites plus 2 or more bases on both ends
which on the blunt end will be ligated to insert DNA.  Linkers may or may
not be phosphorylated on the 5&prime; end.  Before a DNA ligase can be used, the
5&prime; end must be phosphorylated using polynucleotide kinase.  Phosphorylated
linkers are ligated in 75- to 100-fold excess of blunt-end insert DNA.  This will
create tandem repeats (polymers) of linker to the insert, but these are cleaved
with restriction enzyme.  Purification is by gel filtration or electrophoresis.
<p>
When putting linkers on DNA inserts which may harbor the restriction site, 
then the insert DNA may be methylated with appropriate enzyme that protects
against restriction prior to ligating the linkers.
<p>
If the <i>Xho</i>I cloning site on vector has its 3&prime; recessed termini 
partially filled and then ligated to the insert DNA which has 5&prime; 
protruding (3 bases) termini complemetary to <i>Xho</i>I site, then the vector 
and insert DNA cannot ligate to themselves since they cannot anneal to 
themselves. As <i>Xho</i>I is regenerated, the insert DNA can be recovered by 
digesting with enzyme.  This eliminates need to methylate insert DNA or digest 
before inserting into vector.

<ol>
<li>If the linkers are not phosphorylated, then they must be phosphorylated:
to a sterile 500 &micro;l conical tube, add 1.0 &micro;l of 10&times; linker
kinase buffer (600 mM TrisHCl pH 7.6, 100 mM MgCl<sub>2</sub>, 100 mM DTT,
2 mg/ml BSA);  1.0 &micro;l 10 mM ATP, 2.0 &micro;g synthetic linker in TE pH 8,
water up to 9 &micro;l, 10 U of T4 polynucleotide kinase.  Incubate 37&deg; for
1 h.
<li>If a methylation reaction must be done, perform at this step.
<li>First calculate the number of moles of termini.  Note that for a 1 kb
fragment, 50 &micro;g is 78.7 nmol of termini.  To sterile 500 &micro;l tube, 
add the insert DNA so that 2 pmol of termini are present.  Then add 150-200
pmol of phosphorylated linkers.  Then add water to 7.5 &micro;l volume.
Add 1.0 &micro;l 10&times; ligation buffer (), then 1.0 &micro;l of 5 mM 
ATP (the free acid), and 1.0 Weiss unit of T4 DNA ligase.  Incubate 12-16 h
at 4&deg;.  Note ligation is best if all materials are as concentrated as possible
(limited to 5-10 &micro;l volume); if the ATP is included in ligation stock,
this is good.
<li>After ligation, incubate at 65&deg; for 15 min.
<li>Let the mixture cool and then add 10 &micro;l 10&times; restriction enzyme
buffer, 50 U restriction enzyme, then sterile water to final 100 &micro;l volume
and incubate 1-3 h at 37&deg;.
<li>Clean the DNA with phenol-chloroform, and add 2 vol EtOH with 2 M ammonium
acetate present.  Spin 4&deg; for 15 min and dissolve in 50 &micro;l TE pH 8.0.
<li>Use a spin column cleanup to remove unattached linkers.
<li>Methods to monitor whether linking occurred can be done. One is to analyze
all the linkers that formed as 1-, 2-, 3-, 4-mers, etc during ligation: 
1 &micro;l of reaction on 10% polyacrylamide may show a ladder with EtBr or SYBR
Gold staining.  If linkers are radiolabeled, then the 10% gel can be run
and autoradiographed to show both the ladder of linkers, and agarose gel
with EtBr and cutting out to count radioactivity at the band.  As for the
restriction digest:  in the gel detecting the ladder, run the digest and
it should show elimination of ladder and just the monomer or perhaps a light
dimer band.
<li>Precipitate recovered DNA with EtOH and suspend in 10-20 &micro;l TE pH 8.0.
<li>The adapted insert DNA is not ready for ligating into vector.
</ol>


<h2>Ligating Plasmid and Target DNAs in Low-Melting-Temperature Agarose</h2>
[to be done]


<h2>Blunt-Ended Cloning Plasmid Vectors</h2>
<p>
If it is possible, and certainly where it is necessary, to ligate insert DNA
directionally into vector (or to other DNA), then attempts should be made to
add cohesive ends to both vector and insert DNA, and with two different 
cohesive ends (two restriction enzymes) on the vector and insert.  Ligation
of blunt ends is 10- to 100-fold less efficient than ligation of DNA molecules
with complementary protruding termini.
<p>
Maximizing the desired number products of cloned blunt-end fragments involves 
adjusting the stoichiometry appropriately.  If plasmid vector to target DNA is 
too high, more products are empty vectors both monomeric and polymeric.  If the 
ratio is too low, then an excess of linear and circular homo- and heteropolymers 
of various sizes, orientations, and compositions is likely.  Confirmation 
requires restriction mapping, but the most efficient way to generate products is 
using equimolar amounts of vector and fragment, and keep the total DNA 
concentrations &lt; 100 &micro;g/ml.
<p>
This protocol describes cloning blunt-end fragments into linearized vector.
<ol>
<li>1-10 &micro;g plasmid and insert (foreign) DNA.
<li>The digested plasmid and insert fragments are purified using 
phenol:chloroform and ethanol precipitation.  It might also be necessary or 
desirable to purify the insert fragments on neutral agarose or polyacrylamide 
gel electrophoresis if there are large number of ligatable materil.
<li>Resuspend in TE pH 8.0 to a concentration of ~100 &micro;g/ml.  It might
be useful to confirm concentration by AGE.
<li>Dephosphorylate the plasmid vector DNA using the dephosphorylation
protocol.
<li>The following steps set up the reaction tubes:
 <ol>
 <li>Prepare sterile 0.5 ml microfuge tubes according to the table.
   <table>
   <tr><th> Tube  <th> DNA
   <tr><td>A and E <td>vector, 60 fmol (~100 ng)
   <tr><td>B       <td>foreign, 60 fmol (~10 ng)
   <tr><td>C and F <td>vector + foreign, 60 fmol each
   <tr><td>D       <td>linearized vector, 5&prime;-phosphorylated, 60 fmol
   <tr><td>G       <td>superhelical vector, 6 fmol (~10 ng)
   </table>
  <li>To tubes A, B, and C, add 1.0 &micro;l 10&times; ligation buffer,
0.5 Weiss unit T4 DNA ligase, 1.0 &micro;l of 5 mM ATP, add water to
8.5 &micro;l, then add 1-1.5 &micro;l of 30% PEG-8000.
  <li>To tubes D, E, and F, add everything added to tubes A-C, but
   <em>not</em> the ligase.
  </ol>
<li>Incubate 1t 16&deg; overnight, or at 20&deg; for 4 hours.
<li>Dilutions of the ligation products are then used to transform competent 
<i>E. coli</i>.  The table provides an indication of what to expect:
 <table>
 <col span="2">
 <col style="text-align:center;">
 <tr><th>Tube <th>DNA                 <th>Ligase  <th>Expected number of transformants
 <tr><td>A    <td>vector (D)          <td>+       <td>~0
 <tr><td>B    <td>vector              <td>+       <td>0
 <tr><td>C    <td>vector (D) + insert <td>+       <td>~5-fold more than F
 <tr><td>D    <td>vector (D)          <td>&minus; <td>~0
 <tr><td>E    <td>vector              <td>&minus; <td>~50-fold more than D
 <tr><td>F    <td>vector (D) + insert <td>&minus; <td>~50-fold more than D
 <tr><td>G    <td>superhelical vector <td>&minus; <td>2 &times; 10<sup>5</sup>
 <tr><td colspan="4">vector (D) = dephosphorylated vector
 </table>
</ol>
<p>
The greatest efficiency in ligation is achieved when the volume is as small as 
possible (5-10 &micro;l).  Adding ATP to 10&times; ligation buffer itself often 
helps, and commercial buffers usually include it. The water might be added to 
the DNA and the fragments warmed to 45&deg; for 5 min: this helps to dissociate 
any DNA clumps; chill to 0&deg; before adding rest of components.  The PEG stock 
(30%) should be warmed to room temperature before adding; this ingredient should 
also be added LAST!  DNA might precipitate at cold temperatures in presence of 
PEG-8000.

<h2>Blunt-End Ligation of PCR Amplicons into Plasmid Vectors</h2>
<p>
[This method from <i>Molecular Cloning: A Laboratory Manual</i> 3rd Ed.]
<p>
The vector chosen should have restriction site that produces blunt ends, and
this site should not be contained in the insert amplicon.  Moreover, ligation
should not regenerate the restriction site.  The plasmid vector and host
should have blue-white screening.  The amplified insert must contain only one
molecule:  otherwise gel-purify on low melting/gelling agarose.  The amplified
insert must not contain PCR reaction materials: perform phenol-chloroform and
ultrafiltration, or use an amplicon cleanup kit.
<ol>
<li>Add the following to a small conical tube in this order:
 <ul>
 <li>1 &micro;l  50 &micro;g/ml closed circular plasmid vector
 <li>8 &micro;l  25 &micro;g/ml amplified insert DNA 
 <li>2 &micro;l  10&times; Universal KGB buffer [1 M potassium acetate,
   250 mM Tris acetate pH 7.6, 100 mM magnesium acetate tetrahydrate
   5 mM 2-mercaptoethanol, 100 &micro;g/ml bovine serum albumin]
 <li>water to 20 &micro;l final volume (compute this based on following amounts
  of enzyme to be pipetted below)
 <li>1 &micro;l 10 mM ATP
 <li>1 &micro;l 2 mM dNTPs
 <li>2 U restriction enzyme
 <li>1 U T4 DNA polymerase
 <li>3 U T4 DNA ligase
 </ul>
As a control, set up all of the above except substitute insert DNA with water.
<li>Incubate 22&deg; for 4 h.
<li>Transfer 5 &micro;l of the ligation mixtures to another tube and add
10 &micro;l water.  Transform chemically or electro-competent <i>E coli</i> 
and then plate them to media with IPTG+Xgal and selection antibiotic.
<li>After colonies grow, count them.  Pick several white colonies and 
control by isolating the plasmid and doing restriction digest or use colony
PCR protocol (or both).
<li>Confirm the cloning using DNA sequencing, restriction mapping, or 
Southern hybridization.
</ol>

<h2>Amplicon Ligation to T Vector</h2>
<p>
[This method from <i>Molecular Cloning: A Laboratory Manual</i> 3rd Ed.]
<p>
A T vector is any vector with a 3&prime; protruding terminus with single T base
to match the A base on the 3&prime; end of amplicons which gets added by 
<i>Taq</i> polymerases.  Vectors of this type can be made by digesting
with restriction enzymes that leave a single T base as 3&prime; protruding
termini, using terminal transferase (TdT) with ddTTP to add a single base to
the 3&prime; end, using the <i>Taq</i> polymerase itself to add the T base
as it has terminal transferase activity.  Vendors also supply ready made T
vectors as well.
<ol>
<li>Set up a conical tube with the following in this order:
 <ul>
 <li>1 &micro;l  25 &micro;g/ml amplified insert DNA 
 <li>20 ng  T vector (should be 1:5 vector:insert ratio)
 <li>1 &micro;l  10&times; T4 DNA ligase buffer
 <li>water to 10 &micro;l final volume
 <li>3 U T4 DNA ligase
 </ul>
A control should have everything but the target.
<li>Incubate at 14&deg; for 4 h in temp controlled water bath
<li>Mix 5 &micro;l each of both ligation reaction with 10 &micro;l and
transform competent <i>E coli</i> using antibiotic resistance.  Plate
the transformed bacteria on media with IPTG, Xgal, and with antibiotic
<li>Do a colony count and then isolate plasmid and confirm its identity
with restriction analysis or colony PCR.
<li>Fractionate the restriction digest using agarose electrophoresis
and determine size.  Then use sequencing, restriction mapping or Southern
hybridization.
</ol>

<h2 id="adding-seq-2-pcr">Adding Restriction Sites to PCR Amplicon Termini</h2>
<p>
[This method from <i>Molecular Cloning: A Laboratory Manual</i> 3rd Ed.]
<p>
When using this very useful approach, it is important to ensure:
<ul>
<li>the added bases are long enough for restriction enzymes to digest the
ends of the amplicon
<li>that the amplicon has been cleaned up so that residual polymerase and
dNTPs do not remain to fill in restriction digests
</ul>
<ol>
<li>Design the forward and reverse primers to include restriction sites. Use two 
different restriction sites if the amplicon will be cloned directionally. The 
3&prime;-end should be an exact complement of ~15 nt at a selected site in the 
insert DNA.  The 5&prime;-end will serve as a clamp to hold together the termini 
of amplified DNA and provide substrate for restriction enzyme: it should be 3-10 
nt long.  Each primer will then be 24-31 nt long and have equal numbers of four 
bases with balanced distribution of G and C residues and checked for ability to 
form secondary structure.  Oligo synthesized on automated synthesizers are of 
adequate quality usually.
<li>Perform the amplification in quadruplicate.  Follow usual PCR protocol.
<li>Analyze on agarose gel electrophoresis at the appropriate percentage. If 
more than two bands are found, purify the target using low melting/gelling 
agarose.  Otherwise, take the amplification and extract in phenol-chloroform and 
purify on ultrafilter or use PCR cleanup kit.  Put the amplicon in TE pH 7.5 
with 25 &micro;g/ml concentration.
<li>Set up 20 &micro;l reaction volumes with 1.0 to 2.0 U of the restriction
enzymes and ~ 100 ng purified amplicon to prepare the ends.  Digest for 1 h
at optimum temperature.
<li>Adjust volume of reaction mixture to 100 &micro;l with water, and add 0.5 M 
EDTA to final 5 mM concentration.  Extract reaction mixture with 
phenol-chloroform and then once with chloroform.
<li>Transfer aqueous phase to another tube and add 0.1 vol 3 M sodium acetate
pH 5.2 and 2 vol EtOH.  Leave at 0&deg; for 30 min.
<li>Spin max speed for 5 min at 4&deg;.  Carefully remove supernatant and
wash pellet with 70% EtOH.  Spin down again and remove supernatatnt and allow
to dry for a few minutes.
<li>Resuspend in 10 &micro;l water.
<li>Set up the ligation reaction in a conical tube:
 <ul>
 <li>1.0 &micro;l of 25 &micro;g/ml amplicon insert
 <li>20 ng plasmid vector
 <li>1.0 &micro;l 10&times; ligation buffer
 <li>1 U T4 DNA ligase
 <li>water to 10 &micro;l
 <li>it might be necessary to add ATP to 1 mM
 </ul>
When directional cloning is to be done, a 1:1 of insert:vector is done in
ligation.  As a control, set up reaction without the amplicon insert.
<li>Ligate 4 h at 16&deg;.
<li>Take 5 &micro;l of each ligation mixture and add 10 &micro;l water.
Transform competent <i>E coli</i> using appropriate protocol and antibiotic.
Then plate the transformed cultures on media that includes IPTG and Xgal
with antibiotic.
<li>Count colonies, then select several white colonies from the transformation.
Grow plasmid in miniprep and isolate and digest with restriction enzyme
flanking insert or do colony PCR.  Fractionate the restricted DNA on agarose
with markers.  Continue to confirm clone identity with sequencing, restriction
mapping, or Southern.
</ol>
<p>
Note:  if there are no white colonies, it may be necessary to perform the 
ligation/transformation with some kind of positive control.  The ligations
may be analyzed by agarose gel before transformation.  If the nonrecombinant
colony count is proprotionally high, this might be due to incomplete cleavage
of vector by restriction enzyme.

<!-- ===========================================================
   directed mutagenesis
 =============================================================== -->

<h2>Directed Mutagenesis Methods</h2>
<p>
Directed mutagenesis methods are about purposely altering the sequence of
DNA either at a point (single base) or of short stretches 
of DNA for doing the following:
<ul>
<li>creating polymorphisms (natural variations) or mutations (where 
changes are unknown or done to discover structure-function properties
of gene or coding products); these may or may not result in phenotype
changes
<li>creating alleles: changes that result in a definitely in a change
in phenotype
<li>enabling utility or convenience of recombinant manipulation:
inserting (unique) restriction sites, removing or adding expression control
elements
</ul>

<h3>Oligonucleotide-Directed Mutagenesis without Phenotypic Selection</h3>
<p>
<cite>[Kunkel TA (1987) <i>Curr. Prot. Molec. Biol.</i> 8.1.1-8.1.6]</cite>
<p>
An M13 phage single-stranded phage vector produced in an <i>E coli</i> with 
genotype <i>dut<sup>&minus;</sup> ung<sup>&minus;</sup></i> is used.  This <i>E 
coli</i> genotype does not have a dUTPase enzyme (<i>dut<sup>&minus;</sup></i>) which 
causes elevated levels of dUTP and this competes for TTP on DNA polymerase, 
incorporting into phage genome.  The bacterium also lacks uracil N-glycosylate
(<i>ung<sup>&minus;</sup></i>, an enzyme used to edit DNA and remove uracil from it.
Presence of uracil does not create biological consequences such as problems with
pairing nor inhibits DNA synthesis.
<p>
Mutagenesis occurs by transforming wild-type <i>E coli</i> host.  The U-containing
strand is degraded <i>in vivo</i>: the host uracil N-glycosylate transforms uracil
into apyrimidinic sites which are lethal lesions as they block DNA synthesis
and are targeted by endonucleases, producing strand breaks.  The template strand
become replicationally inactive and it is the complementary strand which is 
replicated, allowing for production of mutants.
<p>
Success of this process depends on having a good template and successful DNA 
polymerase reaction.  It should show strong difference in survival between 
<i>ung<sup>&minus;</sup></i> and <i>ung<sup>+</sup></i> hosts.  It should not 
contain endogenous low MW DNA which might prime in vitro DNA synthesis in 
absence of added oligonucleotide, yielding nonmutant clones.  The polymerase 
should complete the synthesis of the circular template containing desired oligo. 
The oligo itself should be highly pure (eliminating contaminants from incomplete 
synthesis). Oligos larger than 40 nt might need to be purified on polyacrylamide 
gels.
<p>
Problems that might arise include inefficient hybridization of the oligo as a 
primer for DNA polymerase; inactive or excess polymerase; contaminants of all 
kinds in DNA, polymerase, and reagents; or template DNA with structures blocking 
polymerization (hairpin/stem-loop structures).  Other problems arise from low 
quality dNTPs, in which dUTP is present (by dCTP deamination) and is 
incorporated into complementary strand, making it possible to target that strand 
with lethal apyrimindinic sites.
<p>
Expected results:  50-80% of plaques should contain desirable mutants.  It takes 
1 day to prepare ssDNA vector with U-containing residues.  All enzyme reactions 
can be complete in 1 day, and reaction products used to transform E coli at end 
of day.  Next day, plaques can be purified (1 day) and ssDNA ready for 
sequenence analysis (1-2 added days).  Sequencing is done in 1 day, results next 
day.  Thus it takes 5-7 days.
<p>
Perform the following steps:
<ol>
<li>Using sterile Pasteur pipet, transfer a plaque of M13 (or ssDNA phage) 
containing insert of DNA of interest to 1.5 ml conical tube containing sterile
TY medium (per L: 8 g tryptone + 5 g yeast extract + 2.5 g NaCl)
per liter).
<li>Place in 60&deg; incubator or water bath 5 min to kill cells, and vortex
to release phage from agar.  Pellet cells and agar with 2 min max speed spin.
<li>To a 1 liter flask containing 100 ml TY + 250 ng/ml uridine, add:
 <ul>
 <li>5 ml midlog <i>E coli</i> CJ236, which should have genotype 
<i>dut<sup>&minus;</sup> ung<sup>&minus;</sup></i> F&prime;
 <li>100 &micro;l supernatant of phage isolate above
 </ul>
The moi is &lt; 1 phage per cell, which means that phage will be <q>passaged</q>
through many hosts, and few non-U-containing phage are present.
<li>Incubation with vigorous shaking at 37&deg; for 6-18 h (phage titers require
good aeration).
<li>Spin 5000 <i>g</i> for 30 min to pellet cells.  The supernatant will have
a phage titer of ~10<sup>10</sup> to 10<sup>11</sup> pfu/ml.
<li>Phage should be titered on <i>ung<sup>&minus;</sup></i> (CJ236) and
<i>ung<sup>+</sup></i> (JM105, JM107, JM109) <i>E coli</i>.  Phage with 
uracil in DNA will have expected numbers but will be &gt; 10<sup>5</sup>-fold
lower numbers in <i>ung<sup>+</sup></i> hosts.
<li>Precipitate the phage with 1 vol 5&times; PEG/NaCl (15% w/v PEG-8000 +
2.5 M NaCl) to 4 vol supernatant.
Mix and incubate 1 h @ 0&deg;.
<li>Spin 5000 <i>g</i> for 15 min to collect precipitated phage.  After decanting
very well, resuspend in 5 ml TE buffer and place in 15 ml Corex tube.  Vortex
vigorously.
<li>Keep the suspension on ice for 1 h, then centrifuge as above.  This should
remove cell debris and also endogenous low MW DNA (which might cause nonspecific
priming).
<li>Use <a href="std-phenol-xtract">phenol extraction and EtOH precipitation</a> to purify phage ssDNA,
and then determine DNA concentration (1 OD<sub>260</sub> = 36 &micro;g/ml).
</ol>
<p>
<b>Phoshphorylating Oligonucleotide and Extending Primers</b>.  Perform 
following steps:
<ol>
<li><b>Skip this and next step if the oligonucleotide is phosphorylated on 5&prime; end</b>.
Add the following to 1.5 ml conical tube:
 <ul>
 <li>2 &micro;l 10&times; T4 polynucleotide kinase buffer
 <li>2 &micro;l 10 mM ATP
 <li>oligonucleotide used to create mutagenesis (15-50 nt long)<br>
Amount should be so that the oligo:ssDNA phage template ranges from 2:1
to 10:1 (in rare cases, it will be higher).
 <li>water to 20 &micro;l
 </ul>
<li>Add 2 U T4 polynucleotide kinase.  Incubate 37&deg; for 60 min.  Add
3 &micro;l 0.1 M EDTA to stop reaction, heat to 70&deg; to denature enzyme.
<li>Add to the tube the U-containing phage ssDNA (should be 1 &micro;g in 
1 &micro;l).  Add 1.25 &micro;l of 20&times; SSC.
<li>Mix well, spin down to collect drops to bottom of tube, and put in 500
ml beaker water at 70&deg;.  Let this cool to room temperature.  Re-spin
to collect condensate, and put on ice.  The high temp opens up template 
from any hairpin formations and to hybridize with oligo as a heteroduplex.
<li>Now add
 <ul>
 <li>20 &micro;l 5&times; polymerase mix (100 mM TrisHCl pH 8.0, 10 mM DTT,
50 mM MgCl<sub>2</sub>, 2.5 mM each of dATP|dTTP|dGTP|dCTP, 5 mM ATP
 <li>2.5 U T7 or T4 DNA polymerase<br>
 <li>2 U T4 DNA ligase
 <li>water up to 100 &micro;l
 </ul>
Might be necessary to add more T4 DNA polymerase units depending on age and 
source of enzyme; do not use Klenow fragment since it can displace mutagenic 
strand; T7 DNA polymerase (that is not Sequenase) is good as it is better at 
completing synthesis; in some cases, these polymerases do not work well, so 
Klenow fragment (1 U) can be used.
<li>Mix well, incubate 5 min 0&deg;, then 5 min room temperature, then 2 h at
37&deg;.  Note that this gradual increase is important for function of T4
DNA polymerase.
<li>Add 3 &micro;l 0.5 M EDTA to stop.
<li>Run the product in 0.8% agarose gel electrophoresis by using 20 &micro;l
reaction material.  Use the following controls: phage circular ssDNA;  
ds, closed circular DNA; nicked ds circular DNA.  All the ssDNA template should
look like closed circular dsDNA or nicked circular dsDNA.
<li>Use the gel analysis result to transfect <i>ung</i><sup>+</sup> <i>E coli</i>
with 1 to 100 ng of the ds DNA.
<li>Obtain clones (either as plaques or as cell colonies) as pure genetic
stocks.  If phenotype is not known, choose randomly the plaques or colonies.
Sequence the obtained clones.
</ol>

<h3>Mutagenesis with Degenerate Oligonucleotides: Creating Numerous 
Mutations in a Small DNA Sequence</h3>
<p>
<cite>[Hill DE (1989) <i>Curr. Prot. Molec. Biol.</i> 8.2.1-8.2.7]</cite>
<p>
In this procedure there are four stages:
<ol>
<li>designing the oligonucleotide for mutagenesis, which involves determining
mutation frequency, nucleotides to be mutated, and the 5&prime; and 3&prime; end
sequences
<li>mutually primed synthesis to convert ss oligonucleotide to ds homoduplex
<li>restriction cutting of the terminal ends (part of the design) of the homoduplex
and the product purification
<li>cloning of the ds oligonucleotide into vectors and identifcation of 
mutagenic sequence
</ol>
<p><b>Oligonucleotide design</b>.  The following must be considered in 
designing the oligo:
<ul>
<li>3&prime; end must have an 8 nt palindromic sequence that includes 
restriction endonuclease cleavage site.  The restriction cleavage can produce
cohesive or blunt ends.  The palindromic sequence is required for hybridization
of two oligonucleotides for producing substrates for DNA polymerase
<li>The 5&prime; end&mdash;if possible&mdash;should have sequences creating
a restriction endonuclease site.  It is not essential because the DNA
polymerase produces blunt ends for cloning.
<li>The central part of the oligo contains a mutagenized sequence of 
interest.  These will be a mix (degenerate sequence) of oligos ultimately.
</ul>
<p>Here is an example of one designed oligo:
<p style="font-size:110%;font-weight:bold;text-align:center;">
5&prime;-<span style="color:green;">CG</span><span 
style="color:purple">CTAAG</span><span 
style="color:blue;">AAAAAAAAAGAGTCAT<span 
style="text-decoration:underline;">C</span></span><span 
style="color:red;text-decoration:underline;">GAATTC</span><span 
style="text-decoration:underline;">G</span>-3&prime;
<p>
The <span style="text-decoration:underline;">underlined</span> sequence is
an 8-base palindrome.  The <span style="color:green;">green</span>-colored 
bases are added for enabling efficient cleavage at the 5&prime; end.
At the 5&prime; end a <i>Dde</i>I restriction site was inserted (the
<span style="color:purple">purple</span>-colored bases).  The <span
style="color:blue;">blue</span>-colored bases are the 17 base region where
mutagenesis occurs.  Each position actually has a 10% mutation frequency,
so the base shown will be made 90% of degenerate oligonucleotides, while
the other three bases are 3.3% of occurrence.  The <span style="color:red;">red</span>-colored
bases are an <i>Eco</i>RI restriction site.
<p>
The oligos are synthesized, using pure precursors (where no degeneracy is
required at a cycle) or a mix (where it is desired).  The degeneracy depends
on the mutation rate at a position.  Suppose it is established that mutation
rate is 10% at a position.  Then the synthesis will use a mixture in which
the nonmutant position is 90% of the precursor concentration, and the balance
is divided by three for the other possible bases (that is, 10% / 3 = 3.3%).
If the instrument cannot be programmed for this mix, then a precursor mix
must be manually prepared for an available reagent bottle position.
<p>
The oligonucleotides should be HPLC- or electrophoretically-purifed (if using
gels, use denaturing polyacrylamide with 7 M urea).  The purified oligos
should be dissolved finally in water at 1 mg/ml (50-100 &micro;M).
<p><b>Mutually primed synthesis</b>. 
<ol>
<li>In a 500 &micro;l conical tube, place 200 pmol (~1-2 &micro;g) 
oligonucleotide. The final volume should be 7 &micro;l (adjusted with water).
<li>If the 5&prime; end does not have a restriction site, then the 
oligonucleotide mix should be incubated with T4 polynucleotide kinase to 
phosphorylate the 5&prime; end.  Otherwise, skip this step.
<li>Place the oligonucleotides in 70&deg; for 5 min.  Add 1 &micro;l of 
10&times; DNA polymerase I buffer, then let cool to room temperature.  (This 
allows the 3&prime; ends palindromic ends of any two single-stranded 
oligonucleotides to hybridize.)  Leave for 60 min.  The hybridization temp does 
not have to be room temp: any temperature that allows hybridization can be used.
<li>Add 2 &micro;l of the dNTP mix (i.e., should be 2.5 mM of each dNTP), 5 U 
Klenow fragment, and 10 &micro;Ci of any [&alpha;-<sup>32</sup>P] dNTP. The 
radiolabel will facilitate DNA purification.  Incubate 60 min at 23&deg;. Add 5 
U more of polymerase after the 60 min and leave at least 2 h or overnight.
<li>Add 1 &micro;l of 500 mM EDTA to stop reaction.  Adjust to 50 &micro;l
with TE buffer, then add sodium acetate to 0.3 M.  Extract once with buffered
phenol, precipitate with EtOH.  Resuspend the DNA with 20 &micro;l TE buffer.
<li>Take 2 &micro;l of this product for analysis on denaturing polyacrylamide 
gels at a later step.
<li>If there is a restriction site on the original 5&prime; end of the 
synthesized oligo, digest it under standard conditions in a final 30 &micro;l
volume reaction.  For each &micro;g of oligo, it might require 10-40 U
of enzyme for at least 2 h.  If there is no 5&prime; restriction site, instead
digest the oligonucleotides with the 3&prime; end restriction site.
<li>Take a 2 &micro;l aliquot for later analysis on denaturing polyacrylamide 
gels. Purify the DNA:  buffered phenol extraction, EtOH precipitation and wash.
<li>Purify the DNA using non-denaturing polyacrylamide gels.  (It is important
that DNA remain double-stranded.)  This removes oligos that reduce ligation
efficiency.  After eluting the ds DNA, resuspend in 20 &micro;l TE and store
at &minus;20&deg;.
<li>If not yet done, digest the oligo with the restriction enzyme which formed 
the original 3&prime; end.  Again, take an aliquot for analyzing on denaturing 
polyacrylamide (2 &micro;l).  Extract with phenol, EtOH precipitate, and suspend 
in 20 &micro;l TE buffer.
<li>Now analyze all the 2 &micro;l aliquots from the three previous steps.
Gels of 6-12% concentration (similar for sequencing) can be used, depending
on oligo size.  As size standards, use <sup>32</sup>P end-labeled oligonucleotides
of known size.
</ol>
<p><b>Cloning the oligonucleotides</b>.
<ol>
<li>The oligonucleotides (in TE buffer) are diluted 10-fold until a 10,000-fold
dilution is obtained.
</ol>

<h3 id="colony-pcr">Colony/Plaque PCR</h3>
<p>
Bacteria, yeast, and bacteriophages can be analyzed quite rapidly by using
a pipet tip to scrape some of the cells growing in colones or some phages
in plaques and washing the scraping directly into a PCR tube containing all
the normal PCR components.  The DNA is released into solution during a 
sustained heating step which normally be part of the denaturation of the DNA.
<p>
The primers used in the reaction are sequences from the vector immediately 
flanking the 5&prime; and 3&prime; ends of the cloning site  (often called 
<q>universal primers</q>, perhaps because the vector is commonly used or known). 
In this way, the PCR should always produce a band, but one that would be smaller 
in size on an agarose gel than if the insert were successful.
<ol>
<li>Prepare PCR master mix based on a count of the number of colonies or 
&lambda; phage plaques.  Each tube takes 25 &micro;l.  1 ml of master mix
can be prepared as follows:
 <ul>
 <li>100 &micro;l 10&times; amplification buffer [500 mM KCl, 
100 mM TrisHCl pH 8.3 (room temp adjustment), 15 mM MgCl<sub>2</sub>]
 <li>50 &micro;l 20 mM dNTPs
 <li>1 nmol forward primer
 <li>1 nmol reverse primer
 <li>water to 1.0 ml
 </ul>
<li>To each PCR tube distribute 25 &micro;l aliquots.
<li>With sterile 20 &micro;l pipet tip (not toothpick), touch each bacterial
colony or &lambda; phage plaque.  Use the master mix in the tube to wash the 
material acquired at the tip, with the pipettor going up and down gently
and set at 20 &micro;l, so as not to introduce air into solution.
<li>Close caps and incubate closed tubes in boiling water bath for 10 min
(if bacterial colony) and 2 min (plaque).
<li>While tubes incubate, dilute <i>Taq</i> polymerase to 1 U &micro;l<sup>&minus;1</sup>
using 10 mM TrisHCl pH 7.6.  Keep on ice.
<li>Let the incubated tubes cool and then spin briefly to bring solution down
to bottom of tube.
<li>Pipet 1 &micro;l diluted polymerase to each tube.
<li>For controls, put in everything except a colony or plaque (negative 
control). In another reaction, include a previous reaction that was known to be 
successful as a positive control.  A very useful control is also to check a 
colony of bacteria where only the vector is present and no insert.  That will 
help show the predicted band length when there is no insert.  Overlay tubes with 
drop (~50 &micro;l) mineral oil if there is no heated lid.
<li>Amplify over 30 cycles with 1 min at 94&deg;, 2 min at 50&deg;, 
2 min at 72&deg;.  If experience suggests another program, use that.
<li>Run 5 &micro;l of amplified mixture on agarose gel.  Stain with EtBr or SYBR 
Gold.  
</ol>
<p>
The band can be further processed (additional confirmation) using restriction
mapping or Southern hybridization.
<p>
Note: this method is so popular for screening colonies now, that the PCR can
be set up in 96-well plates and kits are available strictly for doing colony
PCR.
<p>
<b>Troubleshooting</b>.  When this method fails, it is uaually because too
much of the colony was placed in the master mix and this created a bacterial
lysate that altered the PCR solution environment.  Another cause of failure
is agar fragments from the plate, which inhibit the polymerase.  It is not
necessary to see a mass of cells when picking colonies.  Another reason may
be that the insert is just too large for regular <i>Taq</i> polymerase:
in this case, use a polymerase engineered or altered to be able to extend
longer lengths.  Also primers that cover the ends of the insert can be tried,
rather than the vector.  In addition, the orientation of the insert can be
evaluated by using primers that span both vector and insert boundary..

<!-- ===========================================================
   viral transduction of mammalian cells
 =============================================================== -->


<h2>Viral Transduction of Cultured Mammalian Cells</h2>

<h3>Adenovirus</h3>
<p>
At least 49 different adenovirus types have been identified serologically. The 
Ad5 is the basis of the current vector however.  It is a linear ds DNA virus 
with a 35.9 kb genome, and protein covalently attached to the 5&prime; end of 
each strand.  It will package up to 105% of the size of its genome, allowing 1.8 
kb insertion of foreign DNA.  However, because the E1 and E3 early transcribed 
regions are deleted, it can accomodate a total of 7.5 kb of foreign DNA.  E1 
deletion mutants are replication-defective and produce smaller titers (by 5-6 
logs) than wild-type in nonpermissive cells.  E3 deletion help the virus fight 
off immune cells, and so are dispensable except for applications in which gene 
therapy is important.

<pre>
GENOMIC ORGANIZATION OF Ad5


                            major late transcription unit
                  ---------------------------------------------------->
   E1A E1B                                                 E3
   --> -->                                              --------->
  |=========================================================================|
  0                                                                        100
           <-----------------------------------------                   <----
                           E2                                             E4

The genome is 100 map units long.  This means each map unit is 365 bp
since the genome is 36.5 kb long.
</pre>
<p>
E1 consists of E1A and E1B and cause transformation in a large number of cell 
types.

<!-- ===========================================================
   cDNA integrity
 =============================================================== -->


<h2 id="cdnainteg">cDNA Integrity</h2>
<p>
KPL offers a kit (Cat #60-06-00, 15 reaxns) to assist in the determination of 
whether cDNAs are full-length or not (useful in library construction, RT-PCR, 
microarrays, RAGE, differential display). Sample can be ss or ds cDNA from 
human, mouse, or rat cells/tissue. Kit contains 3&prime; and 5&prime; primer 
sets of commonly expressed genes GAPDH, clathrin, L3, and S6, plus PCR buffer, 
control DNA, and DEPC water.  PCR reactions are done on cDNtA products, and run 
on polyacrylamide and stained using fluorescent dye (SYBR Green).  Both 3&prime; 
and 5&prime; sets will show bands on gels if cDNA is full-length, but only the 
3&prime; sets will show bands if the cDNA are <em>not</em> full-length.


<!-- ===========================================================
    Recombinant Gene Expression Systems
 =============================================================== -->

<h1 style="text-align:left;">The <i>In Vivo</i> Expression of (Cloned) 
Nucleotide Sequence As a Functional Protein for Biotechnology</h1>
<p>
In this section, the recombinant genetic material most likely has an
open reading frame (translatable coding sequence) and it is desired that
the cloned insert be placed in an in vivo system designed to produce the
protein, which will be extracted/isolated and purified and employed <i>in
vitro</i> in other applications.
<p>
This section does not describe the case where the protein is to be expressed in 
vivo in order to examine an effect on the biology of the host organism 
expressing the protein (e.g., placed in a transgenic animal or a protein 
normally part of the recombinant host but its regulation altered).  The host is 
merely used as a means of producing large quantities of the protein so that
the protein can be extracted/isolated, purified, and then used, often in
medical diagnostics or therapeutics in biotechnology (biomolecular 
pharmaceuticals).
<p>
The host systems that are typically employed in protein expression of 
recombinant coding genes (open reading frames) are <i>E coli</i>.  It can 
produce it in abundant amounts in most cases.  When the protein should be 
glycosylated or modified in a way that eukaryotes modify the protein, a yeast 
system such as <i>Saccharomyces cerevisiae</i>, <i>Kluyveromyces lactis</i> or 
<i>Pichia pastoris</i> might be considered.  A strain of <i>Pichia pastoris</i> 
has had its entire Golgi organelle modified to glycosylated human glycoproteins 
in the way that a human would glycosylate them (humanized <i>Pichia</i>). 
Mammalian cell expression systems (Chinese hamster ovary cells, and COS cells) 
have been developed as 
<p>
<b>Intracellular expression versus secretion</b>.  Strains of <i>E coli</i>
and the yeasts are often able to secrete a protein rather than express it 
intracellularly.  The difference is considerable, since intracellular expression
requires extracting the cells and mixing the protein with a great deal of other
biomolecules that require multiple purification steps, each with a less than 100%
yield.  Combined with a reduced yield, there is also more time and cost.  With
secretion, the protein is already relatively pure, and there are many fewer
steps in purification.  The problem is that secretion may lead to lower quantities
of protein produced in a fixed period of time;  moreover, not all proteins are
stable to secretion.  Expression constructs for both secretion and intracellular
expression should be compared for which gives the greater advantage.
<p>
<b>Stable versus transient expression (transfection).</b>  This is an issue
for eukaryotes generally.  If the DNA coding for the translatable gene is
not integrated into one of the chromosomes, it might be degraded after a short
period and then the expression of protein is only transient.  If integrated,
expression is usually stable, so long as the biological regulatory mechanisms
to induce or enable constitutive expression are manipulated also.
<p>
<b>Single vs. multicopy genes</b>.  An issue for all kinds of hosts.  It is 
often the case that when multiple copies of a gene are present, the expression
level is increased significantly without the level being toxic to the host biology.
Ways of increasing copy number of a gene are possible.
<p>
<b>Tagging</b>.  When possible, a small tag usually on the N- or C-terminus is
inserted in-frame into the translation sequence.  This may enable easy identification
and/or purification of the cloned material.  Sometimes the tag may include 
a site that allows its easy removal (cleavage by chemical or enzymatic means)
from the natural wild-type sequence, after the tag has been exploited and is not
wanted in the final product.

<!-- ===========================================================
   E coli as host
 =============================================================== -->


<h2>Using <i>Eschericia coli</i> As Host for Recombinant Genes</h2>
<p>
[Notes from Sambrook &amp; Russell, <i>Molecular Cloning: A Laboratory Manual</i>
3rd Ed.
<p>
Factors that should be considered when using <i>E. coli</i> as a host for 
recombinant proteins.

<ul>
<li><b>Size</b>.  Cytosolic proteins of small size (&lt; 100 residues)
are better expressed as fusions, or they can be degraded.  But even with large
polypeptides they can be degraded and form insoluble inclusion bodies.
<li><b>Desired expression level</b>.  If recombinant protein is to produced
at high quantities, it is better to try several host-vector systems and purification
schemes before going to a large scale.
<li>If a sensitive biological or catalytic activity is required, vectors
that are optimal for producing soluble active proteins should be used.
If protein is used only to produce antibody, then expression systems facilitating
an easy purification should be used (tagging); formation of inclusion bodies
can be useful too.
<li>If protein is to be used for structural work, the best expression is
as a soluble protein.  Several <i>E coli</i> strains may need to be tested
to minimize misfolding.
</ul>

<h3>Promoters and Vectors</h3>
<p>
Inducible expression provides a fine control as to when is the right time
to induce.
<p>
<b><i>trp-lac</i> (<i>tac</i> and <i>trc</i>) promoters:</b>  <i>tac</i> is a 
hybrid of two promoters, with &minus;35 region of <i>trp</i> fused to &minus;10 
region of <i>lac</i>UV5.  It is regulated by <i>lac</i> repressor using IPTG, 
and has no dependence on cAMP regulation (<i>crp</i> product).  Plasmid pKK223-3 
is one commercial form.  <i>trc</i> is another hybrid promoter built just like 
<i>tac</i>, but the <i>trp</i> and <i>lac</i> separation is 17 bp instead of 16 
bp.  There is little difference in expression however,  Plasmid p<i>Trc</i>99A 
is one form; Invitrogen also markets pSE280, pSE380, and pSE420.
<p>
<b><i>lac</i></b>:  this promoter is not as strong as the <i>trc</i> and
<i>tac</i> promoters, and it requires a presence of cAMP activator protein
(CAP, <i>crp</i> product, in which <i>E coli</i> are grown in a medium whose
carbon source does not contain glucose.  However, this promoter is used for
high copy number plasmids, so expression is not entirely a problem.  It is 
used with fusions of recombinants with <i>lacZ</i> on the amino terminus, 
particular in vectors that help show inserts as blue/white colonies.
<p>
<b>Phage T7 Transcripton</b>:  Studier et al developed the pET 
vectors which allow expression under the T7 RNA polymerase.  The pBR322
colicin E1 (colE1) replicon is included, as well as resistance to Amp and
Kan.  A multisite sequence for clone insertion is downstream of the &phi;10
promoter which binds the T7 RNA polymerase with high efficiency transcription.
A variation includes a T7<i>lac</i> promoter which features the <i>lacO</i>
sequence to which the <i>lacI</i> repressor product binds and makes it 
inducible.
<p>
<b>&lambda; phage <i>p</i><sub>L</sub></b>.  This promoter binds the 
<i>c</i>I<i>ts</i>857 repressor, which is temperature-sensitive.
At low temperatures (30&deg;) there is no transcription as the repressor
binds well; at high (permissive) temperature of 42&deg;, expression proceeds.
These systems are popular if the recombinant protein might be toxic to the
host.  The plasmids pHUB, pPLc, pAS1, pRM1/pRM9, and pTrxFus feature the
use of this promoter.  The host itself codes for the <i>ts</i> repressor,
so a strain having this mutated repressor must be used. 

<h3>Fusions</h3>
<p>
Creating hybrids of two proteins (fusions) serve several purposes.  One may
be to tag the recombinant so that a purification technology exploiting the
tag makes purification easy.  Another may be to help direct the recombinant
to a specific subcellular compartment (signal tag or peptide).  Another
might be to stop degradation of the recombinant, or its solubility.

<table>
<caption>Vectors Used In Fusions</caption>
<tr><th>Vector System <th>Fusion Partner  <th>Comments <th>Reference/Source
<tr><td>pUC, pSK, pBluescript, pGEM
 <td>&beta;-galactosidase 
 <td>under control of <i>p</i><sub>lac</sub><i>lacO</i>
 <td><a href="http://www.stratagene.com/">Stratagene</a><br>
  <a href="http://www.lifetech.com/">LifeTechnologies</a>
<tr><td>pTA1529, pBAce
  <td>alkaline phosphatase
  <td>PhoA signal sequence for secretion to periplasm
  <td>Oka et al, 1985
<tr><td>pGEX series
  <td>glutatione S-transferase
  <td><i>lacO/lacI</i> control (IPTG), cleavable fusion
  <td>Smith and Johson 1988
<tr><td>pMAL
  <td>maltose-binding protein (43 kDa)
  <td><i>lacO/lacI</i> control, exportable to periplasm
  <td>di Guan et al 1988, Maina et al 1988<br> <a href="http://www.neb.com/">NEB</a>
<tr><td>pTrx, pTrxFus
  <td>thioredoxin (12 kDa)
  <td><i>lacO/lacI</i> control, enterokinase cleavage site
  <td>LaVallie et al 1993, <br> <a href="http://www.invitrogen.com/">Invitrogen</a>
<tr><td>pET series
  <td>His6x tag, also tags for GST, Trx, DbsA, DbsC, CBD
  <td>T7 transcription (IPTG control), chemical and enzyme cleavage possible
  <td>Studier et al 1990,<br> <a href="http://www.novagen.com/">Novagen</a>
   <a href="http://www.promega.com">Promega</a>
</table>
<p>
MBP has a signal peptide that allows it to be secreted to periplasm.  If not
used, fusion is expressed in cytoplasm.  Thioredoxin associated with cytoplasmic
membrane, and its fusions can often be released by osmotic shock or freeze-thaw
cycles.


<table>
<tr><th>Tag/Fusion  <th>Column Ligand   <th>Elution   <th>References
<tr><td>&beta;-galactosidase
  <td><i>p</i>-aminophenyl-&beta;-D-thiogalactoside (APTG, TPEG)
  <td>sodium borate pH 10
  <td>Germino et al 1983, Ullmann 1984
<tr><td>Protein A
  <td>IgG
  <td>0.5 M acetic acid
  <td>Nilsson et al 1985, Moks et al 1987, Nilsson and Abrahmsen 1990
<tr><td>glutathione S-transferase (26 kDa cytoplasmic from <i>Schistosoma japonicum</i>
  <td>glutathione
  <td>reduced glutathione, 5 mM
  <td>Smith and Johnson 1988
<tr><td>Maltose-binding protein
  <td>cross-linked amylose
  <td>maltose, 10 mM
  <td>di Guan et al 1988, Maina et al 1988
<tr><td>chloramphenicol acetyltransferase
  <td><i>p</i>-chloramphenicol-Sepharose
  <td>chloramphenicol, 5 mM
  <td>Knott et al 1988
<tr><td>carbonic anhydrase II
  <td>sulfonamide affinity
  <td>Tris sulfate pH 6.8
  <td>van Heeke et al 1993
<tr><td>cellulose binding domain
  <td>cellulose
  <td>water
  <td>Ong et al 1989
<tr><td>polyhistidine
  <td>immobilized Zn<sup>2+</sup> or Ni<sup>2+</sup>
  <td>acid gradient (pH 6.0 &rarr; 4.0)
     <br>imidazole (up to 0.5 M)
  <td>Smith et al 1988
<tr><td>FLAG
  <td>anti-FLAG Ab
  <td>EDTA/neutral pH, glycine buffer/pH 3.0
  <td>Hoppe et al 1988
<tr><td>polyArg
  <td>S-Sepharose
  <td>NaCl gradient
  <td>Brewer and Sassenfeld 1983
<tr><td>polyCys
  <td>thiopropyl-Sepharose
  <td>DTT or 2-mercaptoethanol
  <td>Persson et al 1988
<tr><td>polyPhe
  <td>phenyl-Superose
  <td>ethylene glycol
  <td>Persson et al 1988
</table>
<p>
Fusion removal can be done with chemical cleavage (CNBr at Met, and 
HCOOH at Asp-Pro), but they must not be present in the desired recombinant.
When recombinants are very short in length and need the fusion anyway for
expression, this makes sense.  The <i>Bam</i>HI site also codes for Asp-Pro,
and if inserted in frame, it can code for this.  Other complications include
denaturing conditions are used.
<p>
For enzymatic based cleavages, Factor Xa protease can be used if it sees the 
-I(D/E)GR- substrate site (cleavage is after R), usually upstream of 
recombinant.  Contained on the plasmid is the <i>E coli</i> MalE protein for 
pre-cleavage purification on amylose/maltose column. Many cleavage sites work 
only half the time.  it would require denaturation with 6 M guanidine HCl or 8 M 
urea to unfold the protein for better cleavage or to ensure a spacer sequence. 
Also the proteases are not often pure, and the other proteases can damage
recombinant.
<p>
<b>Inclusion Bodies</b>.  When inclusion bodies definitely occur as they
often do (and there is not much known about how the form since they form
with proteins having all kinds of structures), there are standard approaches
to working with them.  Bacterial extraction methods include sonication
(Saito et al 1987, Belder et al 1988), French press (Goeddel et al 1979,
Schumacher et al 1986), freeze-thaw cycle (Schein and Noteborn 1988),
and lysozyme (Goliger and Roberts 1987).  Choice of method is based on 
what is available, not particularly the best method for processing.
Two important extraction procedure points:
<ul>
<li>Work rapidly.  Do not dawdle.
<li>Keep the preparation on ice-water bath at all times (do not let temperature 
exceed 4&deg;).
</ul>
<p>
Being insoluble, inclusion bodies can be centrifuged out of extracts: 
10,000-20,000 <i>g</i> for 20-30 min at 4&deg;.  Pellets can be resuspended in 
EDTA, detergents, and DNase to degrade or remove unwanted material. After 
washing and degrading unwanted material, resuspend in refolding buffer without 
solubilizing material:  a buffer of TrisHCl pH 7.2, 50 mM NaCl, 1-5 mM EDTA, and 
protease inhibitors.  Include DTT (2 mM) if the target protein does not have 
disulfides.  Other washing components might include deoxycholate or Triton X-100 
(Promega booklet on <i>Proteins</i>). The following references might be useful: 
Marston 1986, Belder et al 1988.
<p>
Inclusion bodies are often re-solubiized with strong chaotropic agents:
urea (5-9 M), guanidinium hydrochloride (4-8 M), potassium thiocyanate 
(3-6 M).  Other approaches include strong detergents (SDS) and pH extremes.
After solutbilization, refolding is done.

<h3>Translation Efficiency</h3>
<p>
Prokaryotic mRNA usually features a ribosome-binding site which is the 
Shine-Dalgarno consequence sequence, where a 5-9 nt long strand complements
the 3&prime; end of 16S rRNA.  In constructing transcribed sequences to be
vectors, there is a particular distance between ATG translational start and
the SD sequence (5-7 nt between).  Also it is important to consider that
if there is 2&deg; structure in the mRNA, this could affect efficiency.
In several instances, reducing 2&deg; structure improves translational
efficiency.  (See Shine and Dalgarno 1974, Steitz and Jakes 1975, Steitz 1979,
Rinquist et al 1992, de Smit and van Duin 1994ab, Chen et al 1994, Makoff and
Smallwood 1990, Rangwala et al 1992)
<p>
<b>Codon Usage</b>.  61 codons determine 20 amino acids, with Met and Trp only 
determined by a single codon.  Grantham et al (1980ab, 1981) showed that codon 
usage for single amino acids is not equally probable.  Replacing nucleotides 
(using mutagensis, for example) might help expression (Rangwala et al 1992). 
Some heterologous genes have sequences that contain SD sequences or those not 
compatible with the host (Ivanov et al 1992).  Both PCR and site-directed 
mutagenesis in particular can help improving the coding region at the N-terminal 
portion (changing 1st 7-8 codons of recombinant gene. It is recommended that G+C 
content in 5&prime; end be reduced to &lt;40% by these procedures.

<h3>Growth Condition Optimization</h3>
<p>
Different media should be tested for finding optimal expression.  A standard
medium such as LB establishes general parameters, but minimal media should be
tested (M9), defined salts for induction, rich media such as Terrific, YT,
NZCYM.

<h3>Inducible Expression Example</h3>
<p>
Expression level under these system usually about one-third of total cell 
protein.  Not practical for large-scale expression because of IPTG cost. pGEM-3Z 
is 2743 bp vector with a LacZ &alpha; peptide under <i>lac</i> operon, an mcs 
featuring a dozen restriction sites, and promoters for T7 and SP6 RNA 
polymerase (reversed directions), and Amp<sup>r</sup> (&beta;-lactamase) resistance
Equipment and media and reagents required include 1 M IPTG (stock), 
materials for 10% SDS-PAGE (including stain), LB liquid medium and agar with
50 &micro;g/ml ampicillin, high speed centrifuge, boiling water bath, shaker
incubator, PCR reagents (10&times; amplification buffer, 20 mM dNTPs, Taq
polymerase, agarose or polyacrylamide gel, primers, positive control DNA,
template DNA), directional and blunt-ended cloning reagents (DNA ligases
[T4], restriction endonucleases, DNA purification chemicals), preparation of
competent <i>E coli</i> to high efficiency,  and sequencing reactions.
<ul>
<li><b>Strains</b>. Use <i>E coli</i> strains suitable for transformation and 
having <i>lacI</i><sup>q</sup> or <i>lacI</i><sup>q<sup>1</sup></sup> coding. 
Sometimes plasmids code for this <i>lacI</i> system (pMAL, pGEX) and strains 
such as JM101, DH5F&prime; and TG1 are used.
<li><b>Vector with <i>lacO</i> control</b>.  pGEM-3Z, pGEX-1, pKK223-3, pMEX,
p<i>Trc</i>A, pMAL are examples
<li><b>Positive controls</b>.  If availalble, get a IPTG-inducible plasmid
carrying a LacZ fusion of known size.
</ul>
<p>
<b>Construction and Transformation</b>.
<ol>
<li>Obtain the DNA fragment with 5&prime; and 3&prime; ends comptabile with the 
IPTG-inducible expression vector.  Either use simple restriction digestion to 
produce this, or use PCR modification protcols to produce this 
fragment.
<li>Ligate insert into vector.
<li>Transform competent <i>E coli</i> containing <i>lacI</i><sup>q</sup> either
in plasmid or host chromosome, and then plate transformants on to LB agar
with 50 &micro;g/ml ampicillin.  Incubate overnight with 37&deg;  Also transform
cells with vector only (no insert) and perhaps with positive control (pGEX-1).
<li>Screen transformants from colonies using any of the following:
 <ul>
 <li>Colony hybridization and/or restriction enzyme analysis
 <li>Oligonucleotide hybridization
 <li>direct DNA sequencing of plasmid minipreps
 </ul>
</ol>
<p>
<b>Optimizing Induced Expression</b>.
<ol>
<li>During these studies, make note of the count of bacteria inoculated into
fresh growth medium, time of growth <em>before</em> induction, and density
of growth at the end of induction.  Overgrowth or too rapid a growth can
affect sythesis and/or cause formation of inclusion bodies.
<li>Inoculate 1 ml cultures (LB medium + 50 &micro;g/ml ampicillin) with one
or two colones of vector-only plasmid, positive control (pGEX-1), and recombinant.
Incubate overnight between 20-37&deg;:  growth rates are very slow (four-fold
less at 20&deg; than 37&deg; but this could inhibit inclusion bodies).
<li>Transfer 50 &micro;l of each overnight culture to 5 ml tubes with LB
medium only at temperature used for first incubation.  Grow for &gt; 2 h 
in shaking incubator until cells are A<sub>550</sub> 0.5-1.0.
<li>Transfer 1 ml to conical tube and proceed to step following the next one
to process uninduced cells.
<li><b>Induction</b>.  Add IPTG stock to 1 mM and continue incubating with
vigorous shaking.  It may be necessary to do a study of optimal IPTG
concentration, generally from 10 &micro;M to 5.0 mM.  It may also be necessary
to do a temperature optimum study from between 15-42&deg;: in fact optima
can span a 2-4&deg; difference.
<li><b>Time of induction</b>.  With 3-4 possible time points (1, 2, 4, 6 h),
transfer 1 ml aliquot to conical tube, measure A<sub>550</sub>, and then
spin microfuge 1 min max speed.  Aspirate all supernatants.
<li>Resuspened in 100 &micro;l SDS gel loading buffer and heat 100&deg; for
3 min.  Spin 1 min again in microfuge at max speed, and place on ice until
ready to load on gel.
<li>Warm sample when read and load either 0.15 A<sub>550</sub> units or 
40 &micro;g on to 10% SDS-PAGE.  Electrophorese at 8-15 V/cm until fully
developed.  Stain with Coomassie or silver, or perform immunoblot.
</ol>
<p>
For doing large-scale induction:
<ol>
<li>Inoculate 50 ml LB + ampicillin in 250 ml flask with a colony of E coli
construct and incubate at optimial temperature.
<li>Inoculate a 450-500 ml LB + ampicillin in 2 L flash with 5-50 ml of 
overnight culture and incubate with shaking at optimal tempertaure until
mid-log phase (A<sub>550</sub> = 0.5-1.0) is reached.
<li>Induce with IPTG using IPTG concentration, induction time, and induction
temperature from optimization studies.
<li>Harvest cells at 5000 <i>g</i> for 15 min at 4&deg; and proceed to
purification.
</ol>
<p>
Other factors to consider or when troubleshooting inducible expression:
<ul>
<li>Sequences in the 5&prime; and 3&prime;-UTR of the transcript (mRNA)
that promote translation efficiency or mRNA stability or transcription
termination.
<li>The host must overexpress (or express in large amounts) the 
<i>lac</i> repressor.  This is because expression plasmids may be present in
high copy numbers (30-600 per cell) and this can lead to leaky expression, which
can slow growth because the recombinant is toxic to the host.  In this
case, it might be necessary for expression vector to contain a <i>lac</i>
repressor gene for tight coupling of plasmid copy number to repressor.
<li>Transform a broad range of hosts since one or the other may be necessary
for high level expression, avoiding degradation, or maintaining solubility.
<li>Other reasons for undesirably low expression include RNA instability, 
premature termination, translation initiation problems, protein degradation. To 
test protein instability: grow cells to midlog phase in minimal (M9) medium with 
20 &micro;g/ml for all amino acids but either Met or Cys, then add 15 
&micro;Ci/ml for 10 min of [<sup>35</sup>S]Met or [<sup>35</sup>S]Cys, then
add 1 mM Met or Cys.  Collect samples every 10 min from just before induction
to 1 h after, run on gels, autoradiograph and look for signs of instability of
protein.  <i>E coli</i> Y1089 and Y1090 are &Delta;<i>Lon</i> deficient strains
that remove an important protease.
<li>Use of stains lacking <i>rho</i> transcription terminator or in which
RNA metabolism is altered (<i>pnp, ma</i>) can increase transcription.
</ul>

<h3>T7 Promoter/T7 RNA Polymerase-Controlled Expression</h3>
<p>
The T7 promoter/T7 RNA polymerase is particularly a good choice for high level 
expression.  Rifampicin inhibits the <i>E coli</i> RNA polymerase, and so if 
needed, can be used to shut down host promoter gene expression. The T7 RNA 
polymerase only transcribes from the T7 promoter.  The T7 RNA polymerase is 
provided on a &lambda; vector, on a plasmid, or integrated into the host 
chromosome.  To prevent constitutive expression of a potentially toxic 
recombinant, the &lambda; lysogen BL21(DE3) contains phage T7 gene <i>1</i> 
under <i>lac</i>UV5 promoter which binds <i>lacI</i> repressor and is thus 
IPTG-inducible.  T7 gene <i>1</i> is also put under control of &lambda; 
<i>p</i><sub>L</sub> promoter which comes under the control of the 
temperature-sensitive repressor.  The gene itself is inserted downstream of T7 
promoter:  pET-3 derives from pBR322 carrying T7 gene <i>10</i> 
(Pf<sub>10</sub>), <i>Bam</i>HI for gene insertion, and T7 transcription 
terminator (Tf).  pET-3a includes a translation start (S<sub>10</sub>) up to 
codon 11.  There are three translation frames for expressing fusion which is 
part of gene <i>10</i>, for which antibody is available for detecting this 
fusion protein.  Optional vector constructs include fusions with longer 
N-terminal sequence for inserts that are short (pET-3xa-c), or fusion to a 
periplasmic localization signal (pET-12a-c), or a vector with His6x tag 
(pET-14b).  In particular, an inducible system may be better since constitutive 
expression for a highly transcribing/translating system as T7 could overload or 
be toxic to host.
<p>
The general steps of using this system are as for <i>lac</i>-inducible systems:
<ol>
<li>Use PCR to modify, or obtain by specific restriction enzyme digestion,
the fragment with 5&prime;- and 3&prime;-compatible ends for ligation to
plasmid with T7 promoter.  Perform sequencing to verify the construct, and
if necessary, place a ribosome-binding site upstream of the ATG of the gene
or cDNA.
<li>Perform the ligation.
</ol>

<h3>Secretion Signals for <i>E coli</i></h3>
<p>
Vectors might be obtained which include signal peptides for protein secretion
using the <i>ompA</i> (Takahara et al 1988, Cheah et al 1994), <i>pelB</i>
(Power et al 1992), <i>phoA</i> (Oka et al 1985), and <i>hisJ</i> (Vasquez et al 1983).
Secreted protein expression is however often low.  Inclusion bodies have been known
to form even in the periplasm as well.

<h3>Temperature-Sensitive Expression</h3>
<p>
For gene under &lambda; <i>p</i><sub>L</sub> promoter control with the
<i>c</i>I857 repressor cells are grown at non-permissive temperature of 30&deg;,
then induced at permissive temperature of 42&deg;.  Use a strain that expresses
the cI857(<i>ts</i>) repressor.
<ol>
<li>A single colony of <i>E coli</i> on LB agar with antibiotic having been 
transformed with recombinant gene under <i>p</i><sub>L</sub> promoter is 
inoculated to 2 ml LB medium in 20 ml tube containing selection medium.  Incubate
overnight at 30&deg; 200 rpm to stationary phase.
<li>Check OD<sub>600</sub> against water to verify stationary phase is reached,
usually between 1-2 OD units in 1 cm path: ~0.5 to 1 billion cells per ml; 
use 1:10 aliquot in PBS to do reading.
<li>Prepare 50 ml LB + antibiotic in 150 ml shaker flask in 42&deg; shaking 
water bath to prewarm medium (as control, check temperature with thermometer of 
same volume of water in same flask type that is prewarmed similarly).
<li>Transfer the 2 ml culture to the 50 ml scale-up.
<li>After mixing briefly, immediately take a 1 ml aliquot and process quickly
to next step.  Continue incubating at 42&deg; for 3 hr at 200 rpm and at end
take another 1 ml sample.
<li>For each of the two 1 ml samples (times: 0 and 3 h), spin down 2 min at max
speed in room temp microfuge.  Remove supernatant and examine on SDS-PAGE or
store at &minus;20&deg; until SDS-PAGE is done.
<li>With the rest of culture, assess plasmid stability, recombinant solubility,
and prepare periplasmic or extracellular medium samples.  Store leftover culture.
</ol>

<h3><i>trp</i> Auxotrophy with the <i>trp</i> Promoter</h3>
<p>
[to be written]



<!-- ===========================================================
   Expression of Recombinants in Commercial Kits
 =============================================================== -->

<h2>Commercial Kits for Recombinant Protein Expression</h2>

<h3 id="KlacKit"><i>K. lactis</i> Protein Expression Kit</h3>
<p>
New England Biolabs Product Number E1000S, $619 (2011).   The <a 
href="http://www.neb.com/nebecomm/manualfiles/manuale1000.pdf">instruction
manual</a> is summarized in the text below.
<p>
<b>Principle</b>.  [NEB Application Note.] In case where multiple expression of 
polypeptides which are subunits forming a functional multi-subunit protein, it 
is desirable to express all polypeptides in a single host.  Strategies include 
constructing a single vector with multiple promoters that control the different 
genes, or successive transformation and selection of different expression 
vectors for each host. Read et al. have observed that pKLAC vectors in <i>K. 
lactis</i> grown on YCB agar + 5 mM acetamide showed presence of multiple tandem 
copies of linear vector integrated into host genome.  If two to four pKLAC 
vectors are included in a transformation solution with cells, these can be 
successfully integrated and be made to secrete (as reported by the group).

<p>
<b>Method</b>.  The single gene of interest is incorporated into the pKLAC 
series vector, and 2 &micro;g should be purified for transformation for each 
vector plus insert.  The construct is digested with <i>Sac</i>II (20 U in 50 
&micro;l in NEB buffer 4; 2 h; 37&deg;).  Note that pKLAC, pKLMF, and pKLCF 
vectors can be used.  All digests of the different gene-containing vectors are 
pooled and desalted (use Qiagen QIAquick PCR purification kit), eluting with 30 
&micro;l water. Linearized vector can be stored at &minus;20&deg; for up to a 
month. The <i>K. lactis</i> GG799 strain is transformed with 15 &micro;l volume 
of linearized vectors (1 &micro;g each vector).  Not all transformants will 
express the total number of protein targets.  Selection is helped by acetamide 
as it enriches for multicopy integrants.  70-93% of cells transformed with two 
vectors showed both target proteins, while 63% of cells transformed with three 
genes expressed all three.
<p>
<b>pKLAC2</b>. This is 9107 bp circular vector featuring the <i>LAC4</i> 
promoter (P<sub>LAC4-PBI</sub>).  The promoter is not function until 
<i>SacII</i> or <i>Bst</i>XI digestion joins the 5&prime; and 3&prime; ends, 
removing ampicillin resistance (Ap<sup>R</sup>) and the <i>E coli</i> pMB1 ori 
site.  On the 3&prime; end is the &alpha;-mating factor (&alpha;-MF), separated 
from P<sub>LACA-PBI</sub> by a <i>Hind</i>III site.  The 3&prime; of &alpha;-MF 
joins with an MCS [XhoI, NdeI, NcoI, NotI, EcoRV, SalI, BamHI, EcoRI, SbfI, 
StuI] for inserting the gene.  After the MCS is a transcription terminator 
sequence for LACA. The vector also includes an acetamidase (<i>amdS</i>) under 
control of the yeast <i>ADH1</i> promoter (P<sub>ADH1</sub>).  The <i>Hind</i>III
GAACTT represents position 1 in the vector.
<p>
<b>&alpha;-MF</b>.  In the pKLAC2 vector, its DNA and AA sequence is
<pre style="font-size:83%;">

$ = base 9107 = end of plasmid
x = restriction enzyme cohesive end cut 5' to the base marked 
     on sense strand
|| = restriction enzyme blunt end cut between the two ||
@@@@@@### = yeast Kex protease KR recognition cleavage site 
   (cut is between @#)

------------------------------------------------------------------
         1         2         3         4         5         6
123456789012345678901234567890123456789012345678901234567890

GAATTGTGAGCGGATAACAAGCTCAACACTTGAAATTTAGGAAAGAGCAGAATTTGGCAA 9068

                                     $.x....
AAAAAATAAAAAAAAAATAAACACACATACTCATCGAGAAGCTTGAAAAAAATGAAATTC   22
                                     $....x.        M  K  F     3
                                      HindIII      &lt;-- aMF--

TCTACTATATTAGCCGCATCTACTGCTTTAATTTCCGTTGTTATGGCTGCTCCAGTTTCT   82
 S  T  I  L  A  A  S  T  A  L  I  S  V  V  M  A  A  P  V  S    23
----------------- alpha mating factor (aMF) ----------------

ACCGAAACTGACATCGACGATCTTCCAATATCGGTTCCAGAAGAAGCCTTGATTGGATTC  142
 T  E  T  D  I  D  D  L  P  I  S  V  P  E  E  A  L  I  G  F    43
----------------- alpha mating factor (aMF) ----------------


ATTGACTTAACCGGGGATGAAGTTTCCTTGTTGCCTGTTAATAACGGAACCCACACTGGT  202
 I  D  L  T  G  D  E  V  S  L  L  P  V  N  N  G  T  H  T  G    63
----------------- alpha mating factor (aMF) ----------------

                                                      .x....
ATTCTATTCTTAAACACCACCATCGCTGAAGCTGCTTTCGCTGACAAGGATGATCTCGAG  262
 I  L  F  L  N  T  T  I  A  E  A  A  F  A  D  K  D  D  L  E    83
                                                      ....x.
                                                       XhoI
----------------- alpha mating factor (aMF) ----------------

                           ..x... .x......x.......||...x....
AAAAGAGAGGCTGAAGCTAGAAGAGCTCATATGTCCATGGGCGGCCGCGATATCGTCGAC  322

 K  R  E  A  E  A  R  R  A  H  M  S  M  G  G  R  D  I  V  D   103
@@@@@@###                  ...x.. ....x......x....||......x.
                            NdeI  &lt;NcoI&gt;&lt; NotI &gt;&lt;EcoRV&lt;SalI&gt;
----- aMF ----------------&gt;

.x.....x..........x.           ..||..
GGATCCGAATTCCCTGCAGGTAATTAAATAAAGGCCTTGAATCGAGAATTTATACTTAGA  382
 G  S  E  F  P  A  G  N  *     ..||..                         111
....x.....x..x......            StuI
BamHI&gt;EcoRI&gt;&lt; SbfI &gt;

TAAGTATGTACTTACAGGTATATTTCTATGAGATACTGATGTATACATGCATGATAATAT  442

TTAAACGGTTATTAGTGCCGATTGTCTTGTGCGATAATGACGTTCCTATCAAAGCAATAC  502
-------------------------------------------------------------------
</pre>

<p>
This &alpha;-mating factor sequence represents only the first 92 amino acids of 
sequence that is fully 129 amino acids long.  The full 129 amino acids include 
the 23-amino acid sequence KREAEARRARSPRGTVDGAPAAA is repeated twice at the 
C-terminal end (starting at positions 84 and 107).  These first 92 amino acids 
at least feature a Kex protease cleavage site, which must exist between the 
&alpha;-MF leader and the N-terminal start of the protein of interest, in order 
for it to be cleaved during processing for secretion;  the recognition sequence
are the two KR amino acids where the cut is after Arg.
<p>
The pKLAC2 vector provides a couple of cloning strategies:
<ol>
<li><b>Recombinant protein N-terminus amino acids are only from insert 
(preferred)</b>.  The insert is constructed using a forward primer that will
include on the 5&prime; end in this order an <i>Xho</i>I site followed by
the Kex protease dipeptide recognition site followed by the start of the
insert with translation in the frame of the Kex site.  The vector pKLAC2 and
the amplicon are then digested with <i>Xho</i>I and then ligated using 
directional procedures.  To use this strategy, the insert must not contain an
<i>Xho</i>I site.
<p>

In an example, suppose the first 20 5&prime; nt of the sense-strand insert are 
5&prime;-GCACCCATGGCAGAAGGAGG-3&prime; and the last 17 3&prime; nt of the anti-sense 
strand are 5&prime;-AAATGTGACAAGCCGAGGCG-3&prime;.  The forward primer 
should then 

<li><b>Recombinant protein N-terminus includes amino acids coded by vector</b>. 
This would have to be used if the insert sequence contains an <i>Xho</i>I site. 
Such fusion have additional amino acids coded by vector DNA between Kex site and 
the multiple cloning site.
</ol>
<p>
<b>Intracellular Expression</b>.  Insert the gene at the <i>Hin</i>dIII site 
located upstream of the &alpha;-MF translational start, and also cut the 
multiple cloning site 3&prime; to &alpha;-MF with any of the unique restriction 
enzymes to remove the &alpha;-MF leader.  The gene of interest itself must have 
a Met intiation codon, either naturally or added by PCR or site-directed 
mutagenesis. During identification of integrated strains using PCR, a reverse 
primer should be designed to anneal to 5&prime; end of cloned insert.  Use a 
primer inside the LAC4mt promoter region when making amplicons for sequencing
from the 5&prime; end of the cloned insert.
<p>
<b>Epitope Tags</b>.  NEB recommends adding C-terminal tags.  They advise against
using His tags on the C-terminus because <i>K lactis</i> has active carboxypeptidases.
If used, at least 10 His residues should be added.
<p>
<b>Stop codons</b>.  The insert is assumed to have its own stop codon.  If not
true, it should be engineered into the cloning strategy.  Stop codons can be
incorporated into reverse primers for PCR.
<p>
<b><i>Sac</i>II and <i>Bst</i>XI sites in the cloned insert</b>.  Since 
pKLAC2 will be linearized to ensure that the vector + insert is integrated
into genome (stable transformation) at the <i>LAC4</i> locus, the insert
must not contain either <i>Sac</i>II or <i>Bst</i>II.  If it contains both,
use site-directed mutagenesis must be use to engineer any cutting sites at least
for one of the enzymes, which can then be used during linearization.
<p>
<b>Sequencing</b>.  NEB has specified which primers to use for sequencing, 
depending upon whether the &alpha;-mating factor leader was used in cloning or 
not:  this determines which of either two forward primers to use.   The reverse 
primer (pKLAC1 sequencing reverse primer) is an 18-mer with sequence 
5&prime;-TTATCGCACAAGACAATC-3&prime; (computed <i>T</i><sub>m</sub> = 46&deg;).
If the &alpha;-MF is used, then the forward primer should be 
5&prime;-GAAGAAGCCTTGATTGGA-3&prime; (computed <i>T</i><sub>m</sub> = 50&deg;).
If the &alpha;-MF is not used in the cloning strategy, the Lac4mt forward 
primer should be used 5&prime;-GCGGATAACAAGCTCAAC-3&prime;
(computed <i>T</i><sub>m</sub> = 51&deg;)


<h4>Preparing Insert in pKLAC2 Vector and Transfecting into Maintenance Host</h4>
<p>
The following reagents, media, and solutions should be prepared:
<ul>
<li><b>40% D-glucose</b>.  Per 100 ml of solution to be made, dissolve
40 g glucose in 50 ml deionized water, stirring to dissolve, and perhaps
putting a warm water bath (50&deg;).  Adjust to final volume with water.
Filter sterilize.  Or it can be autoclaved at 121&deg; for 20 min, but 
do not use higher temperature or longer time.
<li><b>40% D-galactose</b>.  Prepare as for glucose above.
<li><b>YPGlu &amp; YPGal Medium</b>.  Per 100 ml of medium to be prepared,
dissolve 1 g yeast extract and 2 g Bacto-Peptone (Becton-Dickinson #211820)
in 95 ml deionized water.  Autoclave (20 min at 121&deg;), then let cool.
Aseptically add 5 ml of sterile 40% glucose for YPGlu or 40% galactose
for YPGal.  For solid medium, add 2 g Bacto-agar (BD #21050) to the 
yeast extract and bacto-peptone before autoclaving.  In the liquid medium,
the final concentration is 1% yeast extract, 2% Bacto-Peptone, and the solid
medium is 2% agar.  Note for pouring 60 and 100 mm dishes to a 2-3 mm depth,
a volume of 5.5-8.5 and 15.7-23.6 ml media, respectively, is required.
<li><b>Sterile 1.0 M TrisHCl, pH 7.0</b>.  Per liter, 121.14 g in 800 ml
water, then adjusting to pH 7.0 with conc. HCl.  Autoclave.
<li><b>PBS</b>.  Per liter, dissolve 8 g NaCl, 0.2 g KCl, 1.4 g disodium
hydrogen phosphate, 0.24 g potassium dihydrogen phosphate.  Adjust to pH 7.4
with either phosphoric acid or sodium hydroxide.  Filter sterilize, although
if autoclaving causes salt formation, try stirring vigorously to re-dissolve.
<li><b>Yeast Carbon Base (YCB) with 5 mM Acetamide solid medium</b>.
To a 500 ml bottle, add 15 ml 1 M TrisHCl pH 7.0, 5.85 g YCB medium (supplied
with kit), 10 g Bacto-Agar (BD #214050).  Bring volume up to 495 ml.
Autoclave 20 min at 121&deg;.  Place in a 60&deg; water bath to let it cool
to 60&deg;.  In an aseptic hood, 5 ml of 100 &times; (0.5 M) acetamide stock.
Dispense to disposable Petri dishes, close plates and allow to cool to
room temperature.  Invert and let set 12-18 before using.
</ul>
<p>
For the cloning of the PCR amplicon into vector, follow these steps:
<ol>
<li>Prepare the amplicon with 5&prime; <i>Xho</i>I site and 3&prime; end with
stop codon followed by restriction site from mcs.
<li>Digest 0.5 &micro;g pKLAC2 vector with 10 U <i>Xho</i>I and 10 U of the 
mcs restriction enzyme at 37&deg; for 2 h; buffer should include BSA to 100 
&micro;g/ml.  All the enzymes used here use same buffer and components so a double
digest should be possible.
<li>Digest 0.5 &micro;g of the PCR amplicon with 10 U <i>Xho</i>I and 10 U of
the mcs restriction enzyme, using the same buffer, BSA, time and temperature.
<li>Clean up the reaction products with phenol-chloroform and EtOH precipitation
using either a clean-up kit or the procedure below (if a kit is used skip these
next steps):
 <ol>
 <li>Add 1:1 vol/vol (equal volume) of phenol-chloroform to the digests, mix well,
then spin and carefully remove top aqueous layer to new tube
 <li>Repeat the cleaning, using only chloroform
 <li>Add 10 &micro;g glycogen or tRNA as a carrier to the digests, then add
0.1 vol 3 M sodium acetate pH 5.2, mix, and add equal volume of 100% 2-propanol.
Leave at room temperature for 10 min.
 <li>Microcentrifuge at 12,000 <i>g</i> for 15 min.  Decant supernatant and
gently rinse with 70% EtOH.  Let pellet air-dry (~10 min).
 <li>Resuspend in TE pH 8.0 (10 mM TrisHCl + 1 mM EDTA).
 </ol>
<li>Mix together:
 <ul>
 <li>2 &micro;l pKLAC2 digest (~40 ng)
 <li>10 &micro;l PCR amplicon digest  (~200 ng)
 <li>2 &micro;l of 10&times; T4 DNA ligase buffer (assuming NEB)
 <li>14 &micro;l deionized water
 <li>1 &micro;l T4 DNA ligase (NEB #M0202) (~400 U)
 </ul>
 Incubate at 16&deg; at least 2 h, or overnight.  After the reaction, if the 
 products are not used immediately in transformation, store at &minus;20&deg;
<li>Transform competent <i>E coli</i> (e.g. 5-&alpha; F&prime; I<sup>q</sup>
<i>E coli</i>, NEB C2992) with 2-4 &micro;l of the ligation reaction.
<li>Prepare miniprep DNA of several transformants and digest with appropriate
enzymes to determine cloned insert presence.
</ol>

<h4>Transformation of <i>K lactis</i> Expression Host with Vector-Insert</h4>
<p>
The first step is to linearize the pKLAC2 containing the insert:
<ol>
<li>Cut 2 &micro;g pKLAC2 with cloned insert with 20 U of <i>Sac</i>II
(preferred) or <i>Bst</i>XI if the insert had a <i>Sac</i>II site in it.
Use appropriate enzyme buffer and process 2 h at 37&deg;.  If using the 
pKLAC1-malE control, cut with only the <i>Sac</i>II enzyme
<li>Desalt the digest:  use a DNA fragment purification kit (e.g.
Qiagen QIAquick PCR Purification kit).
</ol>
<p>
Transformation of the <i>K. lactis</i> GG799 cells is done with the following 
steps.  There are about 10,000 transformants per microgram DNA.  Selection is 
done with a Yeast Carbon Base (YCB) agar medium containing 5 mM acetamide. The 
selection is based on nitrogen utilization, and the only source of nitrogen is 
acetamide.  If the yeast has acetamidase (coded for on pKLAC2 by the 
<i>amdS</i>, present only in transformants), then it will grow as ammonium is 
realized in hydrolysis.
<ol>
<li>All pipet tips, tubes, solution, and water should be sterilized before use. 
Use aseptic technique.  Have an incubator oven ready at 30&deg;, and 
incubator/shaker ready at 30&deg; and a water bath ready at 37&deg;.
<li>Thaw one tube of <i>K. lactis</i> GG799 competent cells on ice.  Add 
620 &micro;l of the NEB Yeast Transformation Reagent to the cells.  
<em>Without vortexing</em> shake or invert tube briefly to mix the solution
until homogeneous.
<li>Add 1 &micro;g (but not more than 15 &micro;l!) of the linearized pKLAC2
DNA.  Again shake or invert tube to mix but without vortexing.
<li>Let sit 30 minutes at 30&deg;.
<li>Incubate cells in 37&deg; water bath for 1 h (this causes a heat shock).
<li>Spin cells in microcentrifuge at ~7000 rpm for 2 min.  Discard supernatant.
<li>Resuspend pellet in 1 ml sterile YPGlu medium
<li>Pellet cells at 7000 rpm for 2 min, and remove supernatant.
<li>Resuspend in 1 ml of YPGlu medium, then transfer to sterile culture tube, 
and then put the tube in a incubator shaker set at 30&deg; and shaking at 
250-300 rpm.  Allow to incubate 3-4 hours (not less than 3 hours).
<li>Transfer the culture to sterile 1.5 ml conical spin tube, and spin 7000
rpm for 2 min and discard supernatant.  Resuspend in sterile PBS.
<li>Prepare three sterile 1.5 ml conical spin tubes with 50 &micro;l sterile
deionized water.  To the three tubes, add 10, 50, and 100 &micro;l of the cell
suspension.  Mix briefly, then spread entire volume from each tube to separate
YCB agar medium plates with 5 mM acetamide.  Incubate plates 3-4 days at 30&deg;
until colonies form.  (Plates can be stored up to two weeks at 4&deg;.)
<li>Subculture by streaking or patching 10-20 distinct colonies on to fresh
YCB agar medium + 5 mM acetamide plates.  Incubate 1-2 d at 30&deg;.  Patches
of 1 mm<sup>2</sup> are recommended.  Plates with patched cells can be stored 
up to 3 days at 4&deg; before whole cell PCR is done.
<li>Although this is optional, test the transformants for stable integration 
into the <i>K lactis</i> genome.  Also test to see if there are multiple copies
of integration.  Both these test methods are <a href="#stable-copies">described
below</a>
</ol>

<h4>Expressing the Recombinant Protein in <i>K. lactis</i></h4>
<ol>
<li>Take cells from a 2 mm<sup>2</sup> patch by scraping with a sterile pipet
tip and then resuspend a sterile culture tube containing 2 ml YPGal medium.
<li>Incubate at 30&deg;, shaking at 250 rpm.
<li>Growth will depend on the protein secreted.  Generally a 2 day minimum 
growth will produce saturated culture (~ 30 OD<sub>600</sub> units per ml). 
Determine the growth kinetics in a useful manner, perhaps sampling at different 
times, looking at total (Bradford/BCA assay) and specific protein (running SDS 
gel + western?).  When scaling up, generally it is best to inoculate with a 
1:100 vol/vol (e.g., 10 ml in 1 L) with starter cultures grown overnight at 
30&deg;.
<li>Microcentrifuge 1 ml of each culture for 30 s.  Transfer supernatant to 
fresh tube and store on ice.
<li>Analyze the culture supernatant.  Expression could be as high as 10 mg/ml or 
even higher.  SDS-PAGE followed by westerns using antibodies to protein or 
fusion tags can be done.  Make sure to grow controls which include cells with no 
pKLAC2 transformation as well as cells transformed with pKLAC2 without insert 
(the best control).  Bioactivity assays might be performed too (if protein is 
enzyme, try an activity assay).
<li><b>Storage of <i>K lactis</i> expressing the protein of interest</b>.
Cells can be placed in a deep freeze (&minus;70 to &minus;80&deg;) by adding
glycerol to 20% in cells in medium (mix 500 &micro;l freshly grown cells in
culture medium with 500 &micro;l sterile 40% glycerol).  Alternatively, cells
from a streak or colony can be scraped and resuspended in 20% sterile glycerol.
<li><b>Reviving frozen <i>K lactis</i> cells</b>.  Streak the small aliquot of
cells on YCB agar medium with 5 mM acetamide.  Thereafter, cells can be grown
without use of selection on a rich medium, such as what they are grown in for
protein expression.
</ol>

<h4>Verifying Stable Integration of Transformants and Determining Copy 
Number</h4>
<p>
<b>Identification of Properly Integrated Cells</b>.
<ol>
<li>Transformants grown on selective medium (YCB agar with 5 mM acetamide) are 
used.  Use a sterile pipet tip to scrape cells on solid medium from a 1 
mm<sup>2</sup> area and resuspend in 25 &micro;l of 1 M sorbitol with 2 mg/ml 
lyticase (Sigma L2524).  Vortex.  Incubate 1 h at 30&deg;.
<li>Place tubes of lyticase-treated cells in a thermocycler at 98&deg; for
10 min.
<li>To each tube, add the following
 <ul>
 <li>10 &micro;l of 10&times; Integration Primer 1 
(5&prime;-ACACACGTAAACGCGCTCGGT-3&prime;)
 <li>10 &micro;l of 10&times; Integration Primer 2
(5&prime;-ATCATCCTTGTCAGCGAAAGC-3&prime;)
 <li>10 &micro;l 2 mM dNTPs (NEB #N0447, 10 mM stock)
 <li>10 &micro;l 10&times; ThermoPol Buffer (NEB #B9004)
 <li>1 &micro;l <i>Taq</i> DNA Polymerase (NEB #M0267 or #M0273)
 <li>34 &micro;l deonized water
 </ul>
<li>The cycling program is 94&deg; for 30 s, 50&deg; for 30 s, 72&deg; for
2 min, for 30 cycles, with a final 72&deg; incubation for 10 min.  It will 
take about 1 h 40 min.
<li>Analyze 10 &micro;l of products by 1% agarose gel electrophoresis.  A 2.4 kb 
product should be observable if integration occurred.
</ol>
<p>
<b>Identifying Multicopy Integrants</b>.  As much as 10 copies may be tandemly 
inserted, leading to high levels of secreted protein. To analyze, every step of 
the cell lysis and PCR amplification to the detection of amplicon on gel is the 
same as for checking for integration, except the primers are Integration Primer 
2 and Integration Primer 3 (5&prime;-ACCTGAAGATAGAGCTTCTAA-3&prime;).  An 
amplicon of 2.3 kb is observed.

<h3 id="pichia"><i>Pichia pastoris</i></h3>
<p>
This yeast has had its N-glycan biosynthetic processing humanized in one 
particular straing.  Protein expression in <i>Pichia</i> is under the inducible 
control of a promoter derived from alcohol oxidase I (<i>AOX1</i>) promoter. 
Techniques used to construct in <i>Pichia</i> are similar for those in <i>S. 
cerevisiae</i>.  Higher cell densities can be reached since <i>Pichia</i> 
prefers oxygen (respiratory growth) as opposed to using fermentation.  More has 
been learned about the system since Phillips Petroleum Co. released the 
expression system to research laboratories.  When a protein is expressed in 
bacterial and becomes an inactive inclusion body, it should be cloned into <i>P 
pastoris</i> where it might be biologically active.  <i>Pichia</i> is part of a 
group of four genera (the others are <i>Candida</i>, <i>Hansenula</i>, and 
<i>Torulopsis</i>) that metabolize methanol:  products are formaldehyde and 
hydrogen peroxide, and the enzyme is alcohol oxidase (AOX).  This oxidation 
occurs in the peroxisome to ensure H<sub>2</sub>O<sub>2</sub> does not damage 
other parts of the intracellular contents.  The AOX enzyme is a homo-octomer and 
each subunit has an FAD coenzyme.  As O<sub>2</sub> is required, and the enzyme 
has a poor affinity for the molecule, it synthesizes AOX in large quantities. 
Two AOX genes exist, <i>AOX1</i> and <i>AOX2</i>, although <i>AOX1</i> is the 
primary product.  Expression is tightly regulated to MeOH presence, is usually 
5% of total souble protein but can reach 30% under certain growth conditions.
Gene regulation is similar to <i>GAL1</i> regulation in <i>S cerevisiae</i>: 
repression/de-repression with an induction mechanism.  This generally means
that if another carbon source is present despite MeOH being present for induction,
repression will still be enabled.
<p>
<b>Expression of secreted inserts</b>.  Generally the use of the secretion 
mechanism should only be employed if the proteins to be inserted are themselves
normally secreted in their native environments.  The yeast's own secretion 
signal is usually made part of the construct although it is possible that the
protein's own secretion signal might work as well.  Secreted inserts might 
be broken down in the culture medium by the presence of proteases; vacuolar
proteases in fermetative cultures appear to be the major factor here.
Strains SMD1163 (<i>his4 pep4 prb1</i>), SMD1165 (<i>his4 prb1</i>) and
SMD1168 (<i>his4 pep4</i>) are protease-deficient mutants should be considered
when this is a problem.  <i>PEP4</i> encodes vacuolar aspartyl proteinase A,
and this activates other vacuolar proteases such as carboxypeptidase Y and 
proteinase B.  <i>PRB1</i> encodes proteinase B, which has about half the
activity of the proteinase B that has been processed by proteinase A.
<p>
All strains used in expression are derived from Northern Regional Research
Laboratories of Phillips (Peoria, IL) strain NRRL-Y 11430.  Strains such as
GS115 (<i>his4</i>) show auxotrophy with one or more mutant genes requiring
a complement active gene, allowing them to be selected on miminal medium with
the required nutrient, and all growing on undefined media.  Mutants of both
<i>AOX</i> genes can express heterologous proteins better than the wild-type,
and a lower concentration of methanol will induce expression better than the
wild type.  The GS115 strain contains active <i>AOX1</i> and <i>AOX2</i> in 
the main chromosomes (they are the wild type genes) and will grown on MeOH
at the wild type rate (MeOH utilization plus, the Mut<sup>+</sup> phenotype).
Strain KM71 (<i>his4 arg4 aox1&Delta;::ARG4</i>) has its <i>AOX1</i> gene
deleted and replaced with <i>S cerevisiae ARG4</i>.  With MeOH present, 
AOX2 is expressed and utilized at a slower rate of groth (called methanol
utilization slow, Mut<sup>S</sup> phenotype).  Strain MC100-3 (<i>his4
arg4 aox1&Delta;::SARG4 aox2&Delta;::Phis4</i>) has been altered to remove
both <i>AOX1</i> and <i>AOX2</i> and cannot grow on MeOH (methanol utilization
minus or Mut<sup>&minus;</sup>) unless transformed stably with an <i>AOX</i>
gene.
<p>
The most common plasmid expression vector contains the <i>AOX1</i> promoter with
a multiple cloning site (mcs) with unique restriction sites immediately 3&prime; to
the promoter.  3&prime; to the mcs is a transcriptional termination sequence
also from <i>AOX1</i> gene.  Additional 3&prime; flanking sequence from the 
original <i>AOX1</i> gene can be used to direct the integration of the plasmid
into the main chromosome for stable transformation.  For secretion, the 
<i>S cerevisiae</i> &alpha; factor prepro signal sequence is usually included.
Alternatively the acid phosphatase (<i>PHO1</i>) signal sequence might be used.
Commercial vector pPIC3K and pPIC9K contain drug resistance markers with the
bacterial kanamycin-resistance gene 
<!-- ==============================================================
     Common Solutions and Reagents
=================================================================== -->
<h1 id="solutions-reagents">Common Solutions and Reagents</h1>


<h2>Stock Chemicals in the Molecular Biology Laboratory</h2>
<table style="margin:1em auto;">
<col><col span="4" style="text-align:center;">
<tr><th>Reagent  <th>Concentration  <th>How Much <th>How Sterilized <th>How Stored
<tr><td>Calcium chloride (CaCl<sub>2</sub>) <td>2.5 M <td>20 ml
   <td>filter sterilize <td>1 ml aliquots @ 4&deg;
<tr><td>Dimethyl sulfoxide (DMSO) <td colspan="3">highest quality  <td>1 ml aliquots @ &minus;20&deg;
<tr><td>EDTA, pH 8.0            <td>0.5 M <td>1 L <td>aliquot, then autoclave <td>
<tr><td>Ethidium bromide        <td>10 mg/ml <td>100 ml <td>     <td>RT
<tr><td>Glycerol                <td>10%      <td>1 L in sterile water
             <td>filter sterilize     <td>200 ml aliquots at 4&deg;
<tr><td>Lithium chloride (LiCl)  <td>5 M  <td>100 ml <td>autoclave <td>4&deg;
<tr><td>Magnesium chloride (MgCl<sub>2</sub> &bull; 6 H<sub>2</sub>O)
   <td>1 M <td>1 L <td>autoclave <td>RT
<tr><td>Magnesium sulfate (MgSO<sub>4</sub>) <td>1 M <td>100 ml <td>autoclave <td>RT
<tr><td>Polyethylene Glycol (PEG) 8000<td colspan="2">13-40% in sterile water
     <td>filter sterilize <td>RT
<tr><td>Potassium chloride (KCl) <td>4 M  <td>100 ml <td>autoclave <td>RT
<tr><td>Sodium acetate (@ pH 5.2 and 7.0)<td>3 M <td>1 L <td>aliquot, then autoclave <td>
<tr><td>Sodium chloride (NaCl)  <td>5 M   <td>1 L <td>autoclave <td>RT
<tr><td>SDS                     <td>20%
    <td colspan="2">1 L (stir @ 68&deg; to dissovle) <td>RT
<tr><td>TrisHCl @ pH 7.4, 7.6, and 8.0<td>1 M<td>1 L<td>autoclave<td>
</table>

<h2>TE</h2>
<p>
Tris-EDTA is prepared as three different kinds of 10 &times; TE,
one for each pH at 7.4, 7.6, and 8.0.

<h2>Phosphate-Buffered Saline (PBS)</h2>
<p>
This from Sambrook &amp; Russell, <i>Molecular Cloning:  A Laboratory
Manual</i>, 3<sup>rd</sup> Ed., Vol. 3., p. A1.7.
<ul>
<li>137 mM NaCl
<li>2.7 mM KCl
<li>10 mM Na<sub>2</sub>HPO<sub>4</sub>
<li>2 mM KH<sub>2</sub>PO<sub>4</sub>
</ul>
<b>Note:  the numbers in brackets are for a 20&times; solution</b>.
<p>
Dissolve 8 [160] g NaCl, 0.2 [4] g KCl,
1.44 [28.8] g Na<sub>2</sub>HPO<sub>4</sub>, and 0.24 [4.8] g
KH<sub>2</sub>PO<sub>4</sub> in 800 ml distilled water.  Adjust pH
to 7.4 with HCl.  Bring to one liter.  Dispense into aliquots and
sterilize by autoclaving 20 min at 15 psi on liquid cycle, or
filter-sterilize.  Store at room temperature.
<p>
PBS commonly contains divalent ions:  1 mM CaCl<sub>2</sub> and
0.5 mM MgCl<sub>2</sub>.

<h2>Tris-Buffered Saline (TBS)</h2>
<p>
Dissolve 8 g NaCl, 0.2 g KCl, 3 g Tris base in 800 ml distilled water.
Add 15 mg phenol red (if used).  Adjust to pH 7.4 with HCl.  QS to 1 liter
with water.  Dispense into aliquots and sterilize by autoclaving 20 min
at 15 psi on liquid cycle.  Store at room temperature.

<h2>Other Balanced Salts Solutions</h2>
<table>
<caption>Balanced Salt Solutions</caption>
<col style="text-align:left;">
<col span="7" style="text-align:center;width:10ex;">
<thead>
<tr>
 <th>
 <th>Ringer
 <th>Tyrode
 <th>Gey
 <th>Earle
 <th>Puck
 <th>Hanks
 <th>Dulbecco (PBS)
</thead>
<tbody>
<tr>
<th>NaCl<td>9.00<td>8.00<td>7.00<td>6.80<td>8.00<td>8.00<td>8.00
<tr>
<th> KCl<td>0.42<td>0.20<td>0.37<td>0.40<td>0.40<td>0.40<td>0.20
<tr>
<th>CaCl<sub>2</sub>
        <td>0.25<td>0.20<td>0.17<td>0.20<td>0.012<td>0.14<td>0.10
<tr>
<th>MgCl<sub>2</sub> 6 H<sub>2</sub>O
        <td>    <td>0.10<td>0.21<td>    <td>     <td>0.10<td>0.10
<tr>
<th>MgSO<sub>4</sub> 7 H<sub>2</sub>O
        <td>    <td>    <td>0.07<td>0.10<td>0.154<td>0.10<td>
<tr>
<th>Na<sub>2</sub>HPO<sub>4</sub> 12 H<sub>2</sub>O
        <td>    <td>    <td>0.30<td>    <td>0.39<td>0.12<td>2.31
<tr>
<th>NaH<sub>2</sub>PO<sub>4</sub> H<sub>2</sub>O
        <td>    <td>0.05<td>    <td>0.125<td>   <td>    <td>
<tr>
<th>KH<sub>2</sub>PO<sub>4</sub>
        <td>    <td>    <td>0.03<td>    <td>0.15<td>0.06<td>0.20
<tr>
<th>NaHCO<sub>3</sub>
        <td>    <td>1.00<td>2.27<td>2.20<td>    <td>0.35<td>
<tr>
<th>Glucose
        <td>    <td>1.00<td>1.00<td>1.00<td>1.10<td>1.00<td>
<tr>
<th>Phenol Red
        <td>    <td>    <td>    <td>0.05<td>0.005<td>0.02<td>
<tr>
<th>Atmosphere
        <td>air <td>air <td>5% CO<sub>2</sub><td>5% CO<sub>2</sub>
        <td>air<td>air<td>air
<tr><td colspan="8" style="background-color:#666;color:yellow;font:bold 90% Verdana,Helvetica,sans-serif;">
  All values are expressed as <span style="color:lime;">grams per liter</span>
  of solution
<tr><td colspan="7" style="background:none;">Note that<ul><li>
  if using CaCl<sub>2</sub> 2 &bull; H<sub>2</sub> (the dihydrate),
  use 1.325 g for every 1.00 g of CaCl<sub>2</sub><br>
 </ul>
</tbody>
</table>
<p>
Sources:
<ul>
<li>S. Ringer, <i>J. Physiol. (London)</i> <b>18</b>, 425 (1895).
<li>M. V. Tyrode, <i>Arch. Int. Pharmacodyn. Ther.</i>, <b>20</b>, 205 (1910).
<li>R. C. Parker, <q>Methods of Tissue Culture,</q> 3rd Ed., p. 57,
  Harper (Hoeber), New York, 1961.
<li>G. O. Gey, M. K. Gey, <i>Am. J. Cancer</i> <b>27</b>, 55 (1936).
<li>W. R. Earle, <i>J. Natl. Cancer Inst.</i> <b>4</b>, 165 (1943).
<li>T. T. Puck, S. J. Cieciura, A. Robinson, <i>J. Exp. Med.</i> <b>108</b>,
  945 (1958).
<li>J. H. Hanks, R. E. Wallace, <i>Proc. Soc. Exp. Biol. Med.</i> <b>71</b>,
  196 (1949).
<li>R. Dulbecco, M. Vogt, <i>J. Exp. Med.</i> <b>99</b>, 167 (1954).
</ul>

<h2>Agarose Gel Electrophoresis Buffers</h2>
<p>
The Tris-acetate-EDTA, Tris-borate-EDTA, and Tris-phosphate-EDTA all differ 
somewhat as to their properties, particularly with AGE.  See <a 
href="#comparing-tris-age-buffers">notes</a> on their particular use.

<table>
<col style="width:5em;">
<col style="width:20em;">
<col style="width:20em;">
<tr><th>Buffer  <th>Working        <th>Stock (per liter)
<tr><td>TAE
    <td>1&times;<ul><li>40 mM Tris-acetate<li>1 mM EDTA</ul>
    <td>50&times;
      <ul><li>242 g Tris<li>57.1 ml glacial acetic acid
        <li>100 ml of 0.5 M EDTA (at pH 8.0)</ul>
<tr><td>TPE
    <td>1&times;<ul><li>90 mM Tris-phosphate<li>2 mM EDTA</ul>
    <td>10&times;<ul><li>108 g Tris<li>15.5 ml phosphoric acid
        <li>40 ml of 0.5 M EDTA (pH 8.0)</ul>
<tr><td>TBE
    <td>0.5&times;<ul><li>45 mM Tris-borate   <li>1 mM EDTA</ul>
    <td>5&times;<ul><li>54 g Tris<li>27.5 g boric acid
        <li>20 ml of 0.5 M EDTA (pH 8.0)</ul>
</table>


<h2>Borate Buffer</h2>
<p>
These buffers are useful in priming or preparing glass (such as microscope
coverslips) for coating with polymers (poly-D/L-lysine or -ornithine) for
cell culture.  A pH 8.5 solution is not uncommon.

<ul>
<li>12.4 g/L boric acid (H<sub>3</sub>BO<sub>3</sub>, MW 61.8)
<li>19.1 g/L sodium tetraborate (borax)
(Na<sub>2</sub>B<sub>4</sub>O<sub>7</sub> &bull; 10 H<sub>2</sub>O, MW 381.37)
</ul>
Adjust pH with sodium hydroxide.

<h2>SSC:  Standard Saline-Citrate 20&times;</h2>
<p>
20&times; SSC is 3 M NaCl, 0.3 M citrate at pH 7.0.
<p>
Add the following mass of components (in grams per liter) to nuclease-free
water that is 80% of the final volume:
<ul>
<li>175.3 g NaCl
<li>88.2 g trisodium citrate  2 H<sub>2</sub>
</ul>
<p>
Adjust to pH 7.0 with 1 M HCl.

<h2 id="SSPE20x">SSPE: Standard Saline, Phosphate, EDTA 20 &times;</h2>
<p>
20 &times; SSPE is 3 M NaCl, 0.2 M NaH<sub>2</sub>PO<sub>4</sub>, and 20 mM 
EDTA.
<p>
In 800 ml pure water, dissolve:
<ul>
<li>175.3 g NaCl
<li>27.6 g NaH<sub>2</sub>PO<sub>4</sub> &bull; H<sub>2</sub>O (monosodium
  phosphate monohydrate)
<li>7.4 g EDTA disodium
</ul>
Adjust to pH 7.4 with NaOH.  QS to 1 liter.  Dispense into aliquots.
Sterilize by autoclaving.

<h2 id="ste">STE</h2>
<ul>
<li>10 mM TrisHCl, pH 8.0
<li>0.1 M NaCl
<li>1 mM EDTA
</ul>
<p>
Prepare from the stock solutions, autoclave, and store at 4&deg;

<h2 id="stet">STET</h2>
<p>
This is STE + 5% (v/v) Triton X-100.  Verify the pH.

<h2>Lysozyme solution</h2>
<p>
Make up as a 10 mg/ml fresh from solid lysozyme in 10 mM TrisHCl at pH 8.0 
immediately before use.  Make sure pH is not below 8, since enzyme works better 
slightly alkaline.

<h2>RNase A (Pancreatic RNase)</h2>
<p>
Make up as a 1 mg/ml solution, weighing out 2 mg in 2 ml TE, pH 7.6

<h2 id="blotto">BLOTTO 1 &times;</h2>
<p>
This is 5% nonfat dried milk dissolved in water with 0.02% sodium azide. BLOTTO 
is stored at 4&deg;.  It is usually diluted 10- to 25-fold into the 
prehybridization before use.  Add NP-40 to 1% (v/v) if background hybridization 
is a problem.  Used in Grunstein-Hogness, Benton-Davis, and all Southerns other 
than single-copy dot blots and slot blots.
<p>
When used in northerns or when RNA is used as a probe, BLOTTO must treated to 
inactivate RNases present in high levels.  Add DEPC or heat overnight at 
72&deg;.
<p>
BLOTTO is less effective than Denhardt's when the target DNA is immobilized on 
nylon filters.

<h2>Denhardt's Solution</h2>
<p>
This is 0.2 g/l polyvinylpyrrolidone (PVP), 0.2 g/l bovine serum albumin (BSA), 
and 0.2 g/l Ficoll 400.  A 100 &times; solution is made by dissolving 10 g PVP, 
10 g BSA, and 10 g Ficoll 400 in 500 ml water. This is sterile-filtered and 
stored at 4&deg;.   It should be mixed before using.

<h2>Phenol</h2>
<p>
Phenol is used extensively in purifying DNA and RNA.  It is used to remove 
proteins and enzymes that can interfere with subsequent reactions.  In some 
cases, the nucleic acids are present in very complex environment, such as 
extracted cells and tissues, and it may be necessary to digest the proteins 
present with enzymes before doing extractions with phenol and other organic 
solvents.
<p>
The typical process involves using phenol:chloroform present in
equal mixtures as the first extraction (0.1% hydroxyquinolone is
an oxidant indicator optionally present) and then chloroform in
a 2nd extraction to remove phenol dissolved in the aqueous phase.
<p>
A <q>back</q> extraction would include adding an equal volume
of aqueous buffer (Tris-EDTA) to the phenol:chloroform phase to
recover nucleic acid that partitioned in the organic phase or which
is present in the interface.  It is combined with the original aqueous
phase prior to the precipitation step.
<p>
Phenol is very efficient at denaturing proteins, and this is why it
is used.  Poly(A)<sup>+</sup> RNA is relatively soluble in phenol, and
this should be considered.  Moreover RNase activity is not inhibited
by phenol.  The poly(A)<sup>+</sup> RNA and RNase problem can be
resolved by extracting instead in 25:24:1 (v/v/v)
phenol:chloroform:isoamyl alcohol.
<p>
It is important to make sure the phenol has been equilibrated in TE buffer
at an alkaline pH (pH should be 7.8-8.0), or else nucleic acid will
partition into the organic phase.
<p>
It may be necessary to perform a phenol:chloroform extraction multiple
times.  How many times?  Basically until precipitated/coagulated
material (protein) is no longer seen at the interface.
<p>
The final step should be the use of chloroform to extract the phenol.
Ethanol is then used to precipitate the protein.
<p>
Liquified phenol is often commercially available for direct use.  Pure phenol is 
colorless and clear.  Do not use phenol that is pink or yellow in color.
<p>
Do not obtain crystalline phenol. Crystalline phenol or reagent which is pink or 
yellowish should be purified by distillation.  The coloration indicates 
oxidation products (quinones, diacids, etc.) which can damage DNA (P-O-P 
cleavage, strand cross-linking) and must be removed.  To re-distill phenol, 
<strong>prepare in a fume hood with protective screen and while wearing 
protective clothing</strong> a round-bottom distillation flask with size 2 
liters or larger and place liquified phenol (volume no more than 75% of flask 
volume) in the flask with a few boiling chips.  If using crystalline phenol, 
melt at 65&deg; in water bath after adding some water at no more than 10% of 
phenol volume; <strong>do not use phenol containing hypophosphorous acid 
(H<sub>3</sub>PO<sub>2</sub>).</strong>  Bring the mixture to boiling with a 
heating mantle.  The condenser should be wrapped in electrical heating tape or 
hot water at 55-60&deg;, since the phenol is likely to condense to a solid 
within the condenser otherwise, and a blocked passage could lead to pressure 
explosion.  The initial distillate formed at about 120&deg; is cloudy and should 
be discarded. Pure phenol emerges as a clear liquid at 180&deg; in a single 
phase. Collect in 100 ml screw-capped glass bottles.  8-Hydroxyquinoline at 100 
mg/100 ml  phenol can be added to the bottle.  Wrap bottles in foil and store 
&minus;20&deg;.  The distilland in the flask should never fall below 100 ml, 
since a concentrated residue is explosive.  Wash the flask out with hot water, 
then rinse with ethanol.
<p>
Phenol needs to be equilibrated in buffer at a pH &gt; 7.8, or else the DNA 
partitions into the organic phase.  Take the following steps:

<ol>
<li>The liquefied phenol is stored at &minus;20&deg;.
<li>When used, remove it, heat it to 68&deg; to melt it.
<li>Add hydroxyquinolone to 0.1%.  It will make the phenol a yellow color.
<li>Add an equal volume of 0.5 M TrisHCl, pH 8.0, at room temperature
<li>Stir 15 min on magnetic stirrer.
<li>Let phases separate.  Aspirate as much of the top (Tris buffer)
phase as possible.
<li>Add equal volume of 0.1 M TrisHCl, pH 8.0
<li>Stir 15 min.
<li>Let phases separate.
<li>Use pH paper to test pH of phenol phase.  If not &gt; 7.8, repeat
with Tris buffer treatments.
<li>Once pH is obtained, add 0.1 volume of 0.1 M TrisHCl, pH 8.0, containing
0.2% 2-mercaptoethanol.  If kept out of light at 4&deg;, it can be kept
1 month this way.
</ol>
<p><b>Another method</b>.
<ol>
<li>Add 0.5 g 8-hydroxyquinoline to 2 L glass beaker with stir bar.
<li>Pour in 500 ml liquified phenol, or of crystals of redistilled 
phenol melted in 65&deg; water bath.  Pnenol may take on yellow color because
of the 8-hydroxyquinoline.
<li>Add 500 ml of 50 mM Tris base.
<li>Cover with aluminum foil, stir low speed at room temperature for 10 min.
<li>After the phases separate at room temperature, decant the top aqueous 
phase into proper waste container.  Use pipet to remove what cannot be decanted.
<li>Add 500 ml of 50 mM TrisHCl pH 8.0.  Repeat the previous two steps (stirring
covered beaker and decanting).  Control the pH of phenol (8.0) with pH indicator
paper.  If not at pH 8.0 continue equilibrating in pH 8.0 Tris.
<li><b>Add</b> 250 ml of 50 mM TrisHCl pH 8.0 or pH 8.0 TE buffer and 
<b>store</b> in 4&deg; either in a dark-colored (brown glass) bottle or a bottle 
covered with foil.
</ol>

<h3>The Mechanics of Mixing</h3>
<p>
When mixing the organic:aqueous phases, one has to be very careful about
the shaking process.  If the DNA is rather small (&lt; 10 kb), vortexing
is permitted.  If the DNA is moderately large (10-30 kb), only gentle
shaking is allowed.  If the DNA is very large (&gt; 30 kb), one must
rotate the tube slowly (20 rpm) on a wheel <em>and</em> use large-bore
pipets to avoid shearing of the DNA.

<h2>Phenol:Chloroform:Isoamyl Alcohol (25:24:1)</h2>
<p>
Mix 25 vol of phenol (prepared above) with 24 vol chloroform and 1 vol
isoamyl alcohol and store up to 2 months at 4&deg; in foil-wrapped or dark
bottle.

<h2 id="formamide">Formamide Deionization</h2>
<p>
Over time, formamide hydrolyzes to form ionic decay products that interfere with 
hybridizations.  Formamide stocks that look yellowed should be deionized. It is 
useful to have a mixed-bed resin column through which to pour formamide to 
de-ionize it. Alternatively, batch removal can be done followed filtering 
through Whatman No. 1.  Formamide should be aliquoted and stored at &minus;20.

<h3>Bio-Rad AG 501-X8</h3>
<p>
A useful mixed-bed ion-exchange resin includes Bio-Rad AG 501-X8, which is mixed 
with AG 50W-X8 strong cation exchange resin (H<sup>+</sup> form and AG 1-X8 
strong anion exchange resin (OH<sup>&minus;</sup> form). There are different 
grades of Bio-Rad's resin:  Molecular Biology Grade is certified endo- and 
exonuclease-free with no ligase inhibitors, and Biotechnology Grade contains 
&lt;100 micro-organisms per gram. AG 501-X8(D) and Bio-Rex MSZ 501(D) are "dye" 
forms with a blue dye irreversibly bound to the anion-exchange form and turning 
from blue to gold when the exchange capacity is exhausted.
<p>
The minimum capacity of all mixed bed resins listed above is 1.5 mEq/ml. The 
mesh size ranges from 20-50 (particle diameters 0.3-1.18 mm). The matrix is 
styrene divinyl benzene.
<p>
The exchange in a charged column is illustrated for saline:
<p style="margin:1em auto;">
<b>Resin-H + Resin-OH + NaCl &rarr; Resin-Na + Resin-Cl + H<sub>2</sub>O</b>
<p>
Batch deionization is done by weighing out 5 g fresh resin for 100 ml sample and 
adding resin to sample.  Stir/shake for 1 hour.  Then filter or decant the 
sample from the resin.  (<b>Note</b>:  samples with sugars or polyhydric 
alcohols should use the minimum quantity of resin and remove the resin from the 
sample in the minimum period to minimize adsorption. Samples with hydrophilic 
protein should be placed in dialysis tubing separated from resin to avoid direct 
contact.  Increase resin used if dye changes from blue to gold.)
<p>
Use column deionization in preference to batch when larger volumes of a mobile 
phase (e.g., water) are to be deionized.  Column geometry should be such at 
length/diameter &ge; 5.  Weigh 0.6 grams resin for each milliliter of column 
volume.  The column is poured nearly dry. First scoop it out in sections.  Add 
water to barely above the bed. Alternate resin and water addition.  Column will 
have tiger's tail appearance if it is highly efficient.  When packed, wash resin 
with 3 bed volumes deionized water.  Pour the solution to be deionized at the 
bed, then begin eluting.  Discard first 1-2 bed volumes.
<p>
The following table shows the capacity of various columns to deionize a 0.1 M 
NaCl solution:

<table style="margin:1em auto;">
<col span="3" style="text-align:center;">
<tr><th>Column Size<br>(diameter &times; length<br>in cm)
  <th>weight of resin<br>(g)
  <th>Volume in Liters of Water<br>Obtained Deionized<br>(from 0.1 M NaCl load)
<tr><td>  0.5 &times;  20 <td>    2.6    <td>   1.5
<tr><td>  1.0 &times;  50 <td>   27.0    <td>   6.7
<tr><td>  1.5 &times; 100 <td>  118.0    <td>  29.0
<tr><td>  2.5 &times; 100 <td>  325.0    <td>  80.0
<tr><td>  5.0 &times; 100 <td> 1300.0    <td> 325.0
<tr><td> 15.0 &times; 120 <td>   13.6 kg <td>3500.0
</table>
<p>
Columns sitting for more than a week should be washed with 3 bed volumes
of water before using.

<h4>Regeneration</h4>
<p>
Lab-scale regeneration is not cost-effective because of the large volumes
of regenerants for the anion resin.  Regeneration on a large-scale requires
first separating the anion and cation resin by back-washing in column using
a low flow rate at first to expand the bed and the anion-exchanger floats
to top and heavier cation-exchanger to bottom.  The flow is increased to
decant off the anion resin into a separate container.  Alternatively,
the reason can be shaken to separate the two.
<p>
The cation resin is regenerated with 3 bed vols of 3 N HCl, then rinsed
with water until pH &gt; 5.  The anion resin is regenerated with at least
10 bed volumes of 3 N NaOH and rinsed until pH &lt;9.

<h2 id="TFB">Standard Transformation Buffer</h2>
<p>
Also called TFB.
<ul>
<li>10 mM MES, pH 6.3
<li>45 mM MnCl<sub>2</sub> 4 H<sub>2</sub>O
<li>10 mM CaCl<sub>2</sub> 2 H<sub>2</sub>O
<li>100 mM KCl
<li>3 mM hexamminecobalt chloride
</ul>
<p>
To prepare 1 M MES at pH 6.3:
<ol>
<li>Add 19.52 g MES in 80 ml water
<li>Adjust pH to 6.3 with (5 M) KOH
<li>Bring to 100 ml with water
<li>Sterilize with pre-rinsed 0.45 &mu;m filter
<li>Divide into 10 ml aliquots and store &minus;20&deg;
</ol>
<p>
To prepare TFB, add the following to about 500 ml water:
<ul>
<li>10 ml of 1 M MES pH 6.3
<li>8.91 g manganese chloride tetrahydrate
<li>1.47 g calcium chloride dihydrate
<li>7.46 g potassium chloride,
<li>0.80 g hexamminecobalt chloride
</ul>
<p>
Bring to 1 liter with water. Sterilize with a 0.45 &mu;m filter.  Store at 
4&deg;.

<h2 id="prehybsolns">Prehybridization/Hybridization Solutions</h2>
<p>
The solution depends on whether formamide is present, or if used in a 
phosphate-SDS buffer.

<h3>Aqueous Buffer Hybridizations (No Formamide)</h3>
<ul>
<li>6 &times; SSC or SSPE
<li>5 &times; Denhardt's
<li>0.5% SDS
<li>1 &micro;g/ml poly(A)
<li>100 &micro;g/ml salmon sperm DNA
</ul>

<h3>Formamide-Containing Buffer</h3>
<p>
All components of the aqueous buffer above plus 50% formamide. Filter the 
solution through 0.45 &mu;m disposable cellulose acetate. The highest quality 
formamide is used especially if one wants to decrease background hybridization 
under reduced stringency (20-30% formamide).

<h3>Phosphate-SDS-Containing Buffer</h3>
<ul>
<li>0.5 M phosphate, pH 7.2
<li>1 mM EDTA
<li>7% SDS
<li>1% BSA, electrophoresis grade at least
</ul>
<p>
It is not necessary to use blocking agents or hybridization rate enhancers
when using this solution for prehyb/hyb.

<hr>

<h1 id="media">Media</h1>
<p>
For solid medium made with agar or agarose, standard plates are made with 1.5% 
concentration (1.5 g/100 ml).  When a top agar or top agarose is used, it is 0.7 
or 0.8% (0.7 or 0.8 g/100 ml).  Add it just before autoclaving.  Sterilize for
20 min on liquid cycle, and swirl it just before pouring to make sure it is mixed
well.
<p>
Always allow the medium to cool to 50-60&deg; before adding heat-sensitive 
substances like antibiotics.
<p>
When making agar plates for bacterial culture, pour hot agar to a depth of about 
5-8 mm.  For 90 mm plates that's about 30-35 ml per plate.
<p>
Keep a Bunsen burner on to remove air bubbles by flame.  Use a color code for
plates, such as one black mark for LB, maybe two red marks for LB+ampicillin
(note, mark the bottom plate, not the cover!).  Store plates inverted at 4&deg;
after they harden and until use.
<p>
Do not allow condensed droplets on the lid to fall on the agarose.  There are two
ways to deal with this:  wipe condensation off the lid and incubate 37&deg; for
several hours inverted before use, or remove liquid by shaking lid with single
quick motion.  Always invert plate before removing lid to do this, to minimize
contamination.


<h2 id="LBmedium">Luria-Bertani (LB) medium (broth) and agar</h2>
<p>
Add:
<ul>
<li>10 g bacto-tryptone
<li>5 g bacto-yeast extract
<li>10 g NaCl
</ul>
<p>
to 950 ml water, adjust pH to 7.4 with NaOH., bring to 1 liter. Sterilize by 
autoclaving.
<p>
To make the agar, add the three components to water, adjust pH,
then add 15 g agar (8 g for top agar), then bring to volume and autoclave.
If adding antibiotics, wait till the agar cools to 50&deg;.
<p>
Note that TB agar is LB agar without the yeast extract.  A top agar
has reduced amount of agar added.

<h2>M9 medium</h2>
<p>
Per liter:
<ul>
<li>6 g disodium phosphate (Na<sub>2</sub>HPO<sub>4</sub>)
<li>3 g monopotassium phosphate (KH<sub>2</sub>PO<sub>4</sub>)
<li>0.5 g sodium chloride (NaCl)
<li>1 g ammonium chloride (NH<sub>4</sub>Cl)
</ul>
<p>
Adjust to pH 7.4.  Autoclave, then add 2 ml of 1 M MgSO<sub>4</sub>
that has been autoclaved or filter-sterilized.
<p>
M9 can be supplemented with glucose:  10 ml of filter-sterilized or
autoclaved 20% glucose.
<p>
M9 can also be supplemented with casamino acids:  10 ml of 20% casamino acids 
(autoclaved)

<h2>YT medium</h2>
<p>
Per liter:
<ul>
<li>8 g bacto-tryptone (casein hydrolyzate)
<li>5 g bacto-yeast extract
<li>5 g NaCl
<li>For agar:  10 g bacto-agar.  For top agar:  7 g bacto-agar.
</ul>
Autoclave.

<h2>NZM / NZYM / NZCYM</h2>
<p>
For <b>NZM</b> medium, add to 950 ml deionized water the following:
<ul>
<li>10 g NZ amine
<li>5 g NaCl
<li>2 g magnesium sulfate heptahydrate (MgSO<sub>4</sub> &bull; 7 H<sub>2</sub>O)
</ul>
<p>
Vigorously shake to dissolve solids.  Adjust pH to 7.0 with concentrated NaOH
(about 200 &micro;l of 5 N).  Adjust to 1.0 L with deionized water.  Sterilize
in the autoclaves (20 min, 15 psi, liquid cycle).
<p>
To make <b>NZYM</b>, add 1 g casamino acids to the mix of NZM, and sterilize.
<p>
To make <b>NZCYM</b>, add 5 g yeast extract to the NZYM formula, and sterilize.

<h2 id="SOBmedium">SOB Medium</h2>
<p>
Per liter:
<ul>
<li>20 g tryptone
<li>5 g yeast extract
<li>0.5 g NaCl
</ul>
<p>
These should be dissolved in 950 ml deionized water, shaking until
dissolution is complete.  Add 10 ml of 0.25 M KCl.  Adjust pH to 7.0
with NaOH, then qs up to 1.0 liter.  Sterilize for 20 min at 15 psi
on a liquid cycle.  Just before use, add 5 ml of sterile 2 M MgCl<sub>2</sub>.

<h2 id="SOCmedium">SOC Medium</h2>
<p>
Identical to SOB medium, except it contains 20 mM glucose.  After
autoclaving SOB medium, allow to cool to 60&deg; or less.  Add
20 ml of sterile 1 M glucose.  (Prepare sterile 1 M glucose
by dissolving 18 g in 90 ml deionized water, dissolve, adjust to 100 ml,
and filter through sterile 0.22 &mu;m filter.)

<h2>MacConkey Agar</h2>
<p>
Per liter, 40 g of Difco MacConkey base and 10 g specific sugar.
Autoclave and pour.

<h2>Stab Agar</h2>
<p>
Per liter, 10 g bacto-tryptone (1% w/v) with 7 g (0.7% w/v) agar.  Heat to 
dissolve, distribute to vials, then autoclave vials.
<p>
In another formulation (Curr Protocol Prot Sci), 10 g nutrient, 10 mg cysteine,
5 g NaCl, 10 mg thymine, and 6 g agar per liter is used.

<h2>Antibiotics</h2>
<p>
These should be made up at 100&times; stocks of the concentration at
which they are used.  Many, if not all, are heat-sensitive, so do not
add to solutions/agars until cooling has reached 50&deg;.  Some are
light-sensitive and should be stored in the dark.  The following table
shows the typical concentration and solvent used.
<table>
<col><col span="3" style="text-align:center;">
<tr><th>Antibiotic<th>Stock Concentration<br>(mg/ml)<th>Solvent
   <th>Useful working concentration*
     <br>(&micro;g/ml)<th>Other remarks
<tr><td>Tetracycline<td>5<td>ethanol<td>10-50<td>light-sensitive
<tr><td>Streptomycin<td>10<td>water <td>10-50<td>
<tr><td>Kanamycin   <td>10<td>water<td>10-50<td>
<tr><td>Chloramphenicol<td>34<td>ethanol<td>25-170<td>
<tr><td>Carbenicillin<td>50<td>water  <td>20-60<td>
<tr><td>Ampicillin<td>50<td>water<td>20-60<td>not stable; quickly inactivated
   by resistant bacteria
</table>
<p>
* The lower concentration in the range is for stringent plasmids,
while the higher value in the range is for relaxed plasmids.<br>
All antibiotics should be stored in the dark at &minus;20&deg;.
Filter-sterilize using 0.22 &mu; filter antibiotics made up in water.
Note that magnesium ions interfere with tetracycline action, and media
free of added magnesium should be used.


<h1>Useful Information</h1>

<h2>Genetic Code</h2>

<table style="margin:1em auto;font-family:'Courier New',Courier,monospace;">
<caption>The Genetic Code</caption>
<tr><th style="background:none;" colspan="2"><th colspan="8">Second Nucleotide
<tr><th style="background:none;" colspan="2"><th colspan="2">U
    <th colspan="2">C <th colspan="2">A <th colspan="2">G
<tr><th rowspan="16">F<br>i<br>r<br>s<br>t<br><br>N<br>u<br>c<br>l<br>e
     <br>o<br>t<br>i<br>d<br>e
    <th rowspan="4" style="vertical-align:middle;">U
    <td>UUU <td>Phe <td>UCU <td>Ser <td>UAU <td>Tyr   <td>UGU <td>Cys
<tr><td>UUC <td>Phe <td>UCC <td>Ser <td>UAC <td>Tyr   <td>UGC <td>Cys
<tr><td>UUA <td>Leu <td>UCA <td>Ser <td>UAA <td>ochre <td>UGA <td>opal
<tr><td>UUG <td>Leu <td>UCG <td>Ser <td>UAG <td>amber <td>UGG <td>Trp
<tr><th rowspan="4" style="vertical-align:middle;">C
    <td>CUU <td>Leu <td>CCU <td>Pro <td>CAU <td>His   <td>CGU <td>Arg
<tr><td>CUC <td>Leu <td>CCC <td>Pro <td>CAC <td>His   <td>CGC <td>Arg
<tr><td>CUA <td>Leu <td>CCA <td>Pro <td>CAA <td>Gln   <td>CGA <td>Arg
<tr><td>CUG <td>Leu <td>CCG <td>Pro <td>CAG <td>Gln   <td>CGG <td>Arg
<tr><th rowspan="4" style="vertical-align:middle;">A
    <td>AUU <td>Ile <td>ACU <td>Thr <td>AAU <td>Asn   <td>AGU <td>Ser
<tr><td>AUC <td>Ile <td>ACC <td>Thr <td>AAC <td>Asn   <td>AGC <td>Ser
<tr><td>AUA <td>Ile <td>ACA <td>Thr <td>AAA <td>Lys   <td>AGA <td>Arg
<tr><td><b>AUG</b> <td>Met <td>ACG <td>Thr <td>AAG <td>Lys   <td>AGG <td>Arg
<tr><th rowspan="4" style="vertical-align:middle;">G
    <td>GUU <td>Val <td>GCU <td>Ala <td>GAU <td>Asp   <td>GGU <td>Gly
<tr><td>GUC <td>Val <td>GCC <td>Ala <td>GAC <td>Asp   <td>GGC <td>Gly
<tr><td>GUA <td>Val <td>GCA <td>Ala <td>GAA <td>Glu   <td>GGA <td>Gly
<tr><td>GUG <td>Val <td>GCG <td>Ala <td>GAG <td>Glu   <td>GGG <td>Gly
</table>

<h2>Named or Branded or Popular/Well-Known Plasmids, Vectors, and Artificial
Chromosomes</h2>
<p> 
Some constructions of double-stranded DNA are reported in the literature and 
often become very popular and even commercialized.  An attempt is made to give 
as much useful information about the application/use and origin and status 
(commercial, patent restrictions) on the DNA constructions as vectors or 
plasmids or artificial chromosomes named below.

<h3>Vectors</h3>
<p>
A <b>vector</b> in molecular biology is a plasmid specialized for the purpose of 
inserting into a designated position another construction of double-stranded DNA 
for the purpose of studying its biology in some way, or to express a gene 
product (RNA or protein, the essential characterisitic of <b>expression 
vectors</b>).  A vector or plasmid that is a <b>reporter</b> usually has a gene 
as part of the vector which has a well-known <q>signalling</q> or 
<em>reporting</em> function, and the inserted DNA is studied for its ability to 
initiate the expression of the reporter gene:  thus there is a search for the 
DNA as a promoter or in some way involved in transcription.  Vectors usually 
contain genes that confer properties on the transfected hosts such that only 
hosts that contain the vector will grow or survive:  thus antibiotics or poisons 
for the host are 

<h3>Plasmids</h3>
<p>
<b>Compatibility of Replication Origins</b>. It should be kept in mind that 
plasmids may have replication origins which are incompatible with each other. 
pMB1 oris and colE1 oris are incompatible.  IncFII and pT181 are incompatible. 
And P1, F, R6k pSC101 and p15A are incompatible.

<h4>pBR322</h4>
<p>
<img src="imagesGenomicsReferenceHTML/pBR322.gif"
  style="float:right;width:60%;"
  alt="restriction map of pBR322">
<ul>
<li><b>Size/length</b>: 4363 bp
<li><b>Insertion</b>: any restriction site; with two antibiotic genes, insertion
might inactivate resistance gene but leave the other active
<li><b>Unique restriction sites</b>
<li><b>replication</b>: <i>E coli</i> <i>ori</i> (derived from pMB1)
<li><b>Transfected host selection</b>: ampicillin resistance (Ap<sup><i>r</i></sup>)
and tetracycline resistance (Tet<sup><i>r</i></sup>)
<li><b>Copy numbers</b>: 1000-3000 per genome
<li><b>Use/Application</b>:  
<li><b>Other links/resources</b>:
</ul>

<h4>pUC Plasmids</h4>
<p>
This is a family of plasmids derived from the laboratory of Messing. pUC18 and 
pUC19 are particularly popular.  A polylinker in the &alpha; region of the 
<i>lacZ</i> gene is present (opposite orientations in pUC18 and pUC19): this 
means that inserted DNA inactivates the gene and produces white colonies rather 
than blue colonies.  This plasmid family represented an evolution in cloning 
vectors by introducing a short sequence representing the <q>multiple cloning 
site</q> because it combined sequences digestible by several restriction 
endoucleases, and there is only one site on these plasmids that is cut tby the 
enzyme (unique restriction sites).  It also provides only one means of selection 
(a single Ap<sup>r</sup> site) rather than two in which one is inactivated by 
insertion of the target sequence.

<ul>
<li><b>Size/length</b>: 
<li><b>Unique restriction sites</b>:
<li><b>replication</b>: <i>E coli</i> <i>ori</i> (derived from pMB1)
<li><b>Transfected host selection</b>: ampicillin resistance 
(Ap<sup><i>r</i></sup>)
<li><b>Copy numbers</b>: 1000-3000 per genome; use chloramphenicol to amplify
<li><b>Use/Application</b>:  wild-type plasmids should have LacZ<sup>+</sup>
phenotypes; the &beta;-galactosidase gene will be inactivated with recombinants
</ul>


<h4>M13 Phage</h4>
<ul>
<li><b>Size/length</b>: 
<li><b>Unique restriction sites</b>:
<li><b>replication</b>:
<li><b>Transfected host selection</b>:
<li><b>Copy numbers</b>:
<li><b>Use/Application</b>:
</ul>

<h4>M13K</h4>
<ul>
<li><b>Size/length</b>: 
<li><b>Unique restriction sites</b>:
<li><b>replication</b>:
<li><b>Transfected host selection</b>:
<li><b>Copy numbers</b>:
<li><b>Use/Application</b>:
</ul>

<h4>pEFG-N1</h4>
<ul>
<li><b>Size/length</b>: 
<li><b>Unique restriction sites</b>:
<li><b>replication</b>:
<li><b>Transfected host selection</b>:
<li><b>Copy numbers</b>:
<li><b>Use/Application</b>:
</ul>

<h4>pGEM series</h4>
<ul>
<li><b>Size/length</b>: 
<li><b>Unique restriction sites</b>:
<li><b>replication</b>:
<li><b>Transfected host selection</b>:
<li><b>Copy numbers</b>:
<li><b>Use/Application</b>:
</ul>

<h4>pGEM-T</h4>
<ul>
<li><b>Size/length</b>: 
<li><b>Unique restriction sites</b>:
<li><b>replication</b>:
<li><b>Transfected host selection</b>:
<li><b>Copy numbers</b>:
<li><b>Use/Application</b>:
</ul>

<h4>pGEM-3Z</h4>
<ul>
<li><b>Size/length</b>: 
<li><b>Unique restriction sites</b>:
<li><b>replication</b>:
<li><b>Transfected host selection</b>:
<li><b>Copy numbers</b>:
<li><b>Use/Application</b>:
</ul>

<h4>pGEM-ZF</h4>
<ul>
<li><b>Size/length</b>: 
<li><b>Unique restriction sites</b>:
<li><b>replication</b>:
<li><b>Transfected host selection</b>:
<li><b>Copy numbers</b>:
<li><b>Use/Application</b>:
</ul>

<h4>pBluescript</h4>
<p>
<img src="imagesGenomicsReferenceHTML/pBluescript.gif"
  style="float:right;width:60%;"
  alt="restriction map and features of pBluescript">
<p>
A phagemid cloning vector with polylinker in &alpha; region of <i>LacZ</i>
(derived from pUC plasmids).  The cloning site is flanked by T3 and T7 promoter,
so the T3 or T7 RNA polymerases can be used in vitro.
<p>
<b>pBluescript(SK+)</b>.  There is an f1 filamentous phage <i>ori</i> site in 
the pBluescript(SK+) form of the vector:  this allows recovery of <i>lacZ</i> 
sense strand as ssDNA.
<p>
<b>pBluescript(SK&minus;)</b>.  The f1 filamentous phage <i>ori</i> site is
in the opposite orientation in the vector, allowing recovery of other strand
as ssDNA.

<h4>pWE15</h4>
<p>
A cosmid which can take inserts of 35-45 kbp in length.  Cloning site is exposed
by <i>Bam</i>HI, flanked by T3 and T7 promoters.
<ul>
<li><b>Size/length</b>: 
<li><b>Unique restriction sites</b>: <i>Bam</i>HI for cloning
<li><b>replication</b>: 
<li><b>Transfected host selection</b>: ampicillin resistance 
(Ap<sup><i>r</i></sup>)
<li><b>Copy numbers</b>: 1000-3000 per genome; use chloramphenicol to amplify
<li><b>Use/Application</b>:  production of labeled RNA probes from each end of 
the DNA insert: can then be used to identify overlapping cosmids for chromosome 
walking and cosmid contig construction
</ul>

<h4>Baculovirus</h4>
[to be done]

<h4>pMD</h4>

<h4>&lambda;gt</h4>
<p>
Expression vector coding MMLV gag-pol under control of CMV IE



<h2><i>E coli</i> Strains and Genotypes</h2>

</div>

<table class="widetable">
<tr><th>Strain <th> Genotype <th>Source <th>Comment   
<tr><td>B  
  <td>Wild type
  <td>ATCC 23227/nonK12 lon&nbsp;deficient 
  <td>Commonly used
<tr><td>B834  
  <td>F<sup>&minus;</sup> <i>ompT gal met r</i><sub>B</sub><i>m</i><sub>B</sub> 
     (DE3)<sup><i>a</i></sup>  
  <td>Novagen
  <td>Host for selenomethionine labeling with T7 expression 
<tr><td>BL21  
  <td>F<sup>&minus;</sup> <i>ompT r</i><sub>B</sub><i>m</i><sub>B</sub> 
    (DE3)<sup><i>a</i></sup>  
  <td>Novagen
  <td>Host for T7 expression 
<tr><td>DH1   
  <td>F<sup>&minus;</sup> <i>gyrA96 recA1 relA1 endA1 thi-1 hsdR17 supE44 
    &lambda;<sup>&minus;</sup>mcrA mcrB nalR</i>
  <td>ATCC 33849
  <td>General host
<tr><td>DH5&alpha;  
  <td>F<sup>&minus;</sup> <i>gyrA96 recA1 relA1 endA1 thi-1 hsdR17 supE44 
     &delta;lacU169</i> (<i>&phi;80lacZ&delta;M15</i>) &lambda;<sup>&minus;</sup>
     <i>mcrA mcrB nalR</i>
  <td>GIBCO/BRL
  <td>General host
<tr><td>HMS174
  <td>F<sup>&minus;</sup><i>recA r<sup>&minus;</sup><sub>K12</sub> 
     m<sup>+</sup><sub>K12</sub> Rif</i><sup>R</sup>
  <td>Novagen
  <td>Host for T7 expression
<tr><td>HB101 
  <td>F<sup>&minus;</sup> <i>leuB6 thi-1 hsdS20 lacY1 proA2 ara-14 galK2 xyl-5 
  mtl-1 recA13 rpsL20 supE44 &lambda;<sup>&minus;</sup>mcrA strR</i>
  <td>ATCC 33694 
  <td> Inhibited at 42&deg;C
<tr><td>JM101
  <td><i>supE thi-1</i> &delta;(<i>lac-proAB</i>) 
      F&prime;[<i>traD36 proAB<sup>+</sup> lacI<sup>q</sup> lacZ&delta;M15</i>] 
  <td>ATCC 33876
  <td>Host for lac and tac expression
<tr><td>MM294 
  <td>F<sup>&minus;</sup> <i>endA1 thi-1 hsdR17 supE44 
     &lambda;<sup>&minus;</sup>mcrA mcrB</i> 
  <td>ATCC 33625 
  <td>General host
<tr><td>RV308 
  <td><i>su<sup>&minus;</sup></i> &Delta;<i>lacX74 gal</i>ISII::OP<i>308 strA</i>
  <td>ATCC 31608
  <td>Host for secretion
<tr><td>SG936 
  <td>F<sup>&minus;</sup> <i>lac</i>(<i>am</i>) <i>trp</i>(<i>amM</i>) 
    <i>pho</i>(<i>am</i>) <i>supC</i>(<i>ts</i>) <i>rpsL</i> mal(am) 
    <i>htpR</i>(<i>am</i>) <i>tsx</i>::Tn<i>10</i> <i>lonR9</i>
  <td>ATCC 39624
  <td>Defective in heat-shock response
<tr><td>W3110 
  <td>F<sup>&minus;</sup> <i>&lambda;<sup>&minus;</sup>mcrA mcrB</i>
  <td>ATCC 27325
  <td>General host
<tr><td colspan="4">
<small><sup><i>a</i></sup> &lambda;DE3 lysogen</small>
</table>

<div class="outerblock">



For a very comprehensive list, see <a 
href="http://openwetware.org/wiki/E._coli_genotypes">OpenWetWare's list</a> of 
<i>E coli</i> genotypes.

</div> <!-- id=outerblock -->

</div> <!-- id=scrolling-panel -->


</body>
</html>