<!DOCTYPE html>
<html>
<head>
<title>Cloning and Expression of Hepatitis B Virus Surface Antigen 
(HBsAg)</title>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8">
<base href="/proteomics/access/">
<link href="../../style/std-numbered-headings.css" type="text/css" rel="stylesheet">
<link href="../../style/proteomicsRef.css" type="text/css" rel="stylesheet">
<style type="text/css">
	.outerblock {text-align:justify;width:40em!important;}
	#title {width:70%;margin:1em auto;}
  .turkish {font:normal 75% Tahoma,Arial,sans-serif;color:navy;margin:0.5em 5em 0.5em 2em;}
  .boxed {border:1px solid black;padding:0.5em 1em;margin:1em -1em 0 -1em;}
  .green {color:green;font-weight:bold;text-decoration:underline;}
  .red  {color:red;font-weight:bold;text-decoration:underline;}
  .brown {color:brown;font-weight:bold;text-decoration:underline;}
  .blue {color:blue;font-weight:bold;text-decoration:underline;}
  .purple {color:purple;font-weight:bold;text-decoration:underline;}
  .boxed-instructions {padding:0.5em 1em;background-color:#f8f8ff;
    border:2px solid purple;margin:1em -0.5em;}
  .header {padding-left:2em;color:yellow;background-color:black;
     font:italic 110% "Times New Roman", Times, serif;  }
  .xout {text-decoration:line-through;color:red;}
   img[alt] {color:white;}
</style>
</head>
<body>
<p style="float:left;width:10%;margin:0 auto auto 0;">
<a href="."><img src="images/AbiyotekLogo.png"
 alt="company logo"></a>

<p id="title">
Cloning and Expression of Hepatitis B Virus Surface Antigen (HBsAg)

<p id="subtitle">
S, M, and L proteins and Their Serological Subtypes

<hr>

<div class="outerblock">
[Whenever you open this page, hit the refresh page key (F5), since its content
has probably changed and you may be looking at an older, cached copy.]

<p>
<b>To Be Done List</b>.
<ul>
<li>Backup Maintenance Cloning method details
<li>B subtilis Expression plasmid construction details
<li>other details
</ul>

<!-- =======================================================================
       Background
 ========================================================================== -->


<h1>Published Information on Proteins and Their Genes Studied in This
Project</h1>
<p>
This section is intended to contain published (verified/confirmed) information
about the biological structure and function of proteins and their genes 
connected to this project.


<h2>Database Registration</h2>  
<p>
<b>UniProtKB/SwissPROT</b>.  The HBsAg is expressed for assembly into the 
envelope mature virion of HBV in three different lengths which vary in the 
deletion of their N-terminus;  all three forms have 226 amino acids in 
common, which is the same as the S form (SHBs).  The M form (MHBs) has
55 extra N-terminal amino acids added to the S form for a total of 
281 amino acids, and the L form (LHBs) has another 108 or 119 amino acids
added to the N-terminus, depending on 
<ul>
<li>SHBs protein:  226 amino acids. <a 
href="http://www.uniprot.org/uniprot/P30019">P30019</a> 
<li>MHBs protein:  281 amino acids. <a 
href="http://www.uniprot.org/uniprot/Q9QAC8">Q9QAC8</a>
<li>LHBs protein:
 <ul>
 <li>400 amino acids. <a href="http://www.uniprot.org/uniprot/P03140">P03140</a>
<a href="http://www.uniprot.org/uniprot/P03141">P03141</a>
 <li>389 amino acids. <a href="http://www.uniprot.org/uniprot/P03142">P03142</a>
This isoform is missing the first 11 amino acids of the 400 aa form (MGGWSSKPRQG)
 </ul>
</ul>
<p>
<b>NCBI</b>.  There are countless entries in the NCBI database.  See below which
are considered reference standards for the various HBV genotypes.

<h2>Biology</h2>
<p>
[ ]

<h2>Bibliography</h2>
<ul>
<li>[ ]
</ul>

<!-- =======================================================================
       Experimental Plan
 ========================================================================== -->


<h1>Experimental Plan:  Coding Sequence Construction, Cloning, and Expression</h1>
<p>
The experimental plan will include a plan and backup plan (Plans A and B).
The goal is to obtain the coding sequence for the S, M, and L forms
<p>
The roadmap will involve these steps:
<ol>
<li>Using PCR assembly with oligos optimally chosen by the Gene2Oligo bioinformatics
software, a total chemical synthesis of:
 <ol> 
 <li>a consensus coding sequence for the S protein
 <li>a consensus coding sequence for the M protein N-terminus including
  the N-terminus of the S protein up to a restriction site in S sequence
 <li>a consenus coding sequence for the L protein N-terminus including
  the N-teminus of the M protein
 </ol>
<li>Ligations and transformations into blue-white E coli using pUC19 vector:
 <ol>
 <li>pUC19 + synthetic S coding sequence
 <li>pUC19 + synthetic M-Nterm/S-Nterm + S Cterm digest product
 <li>pUC19 + synthetic L-Nterm/M-Nterm + M Cterm digest product 
 </ol>
<li>Characterization of cloned products
<li>Plasmid extraction, digestion, cloning into expression vector and
expression host
</ol>

<p>
<b>IMPORTANT!!!</b>  Perhaps the first thing to do is to make sure that all 
important vectors, plasmids, and host strains used in the project and used in 
general&mdash;especially vectors and DNA and host microbes obtained in kits that 
are either very costly or take too much time to replenish by purchase and 
acquisitions&mdash;are prepared in sufficient stocks/inventories. A guide is 
given in a <a 
href="/proteomics/reference/GenomicsReference.html#controlling-inventories">genomics 
reference</a>.

<h2>Materials To Prepare And Have Ready/Available</h2>
<p>
<b>>Notes</b>.
<ul>
<li>All glassware and utensils for microbial culture is to be used
<em>only</em> for microbial culture.  General purpose glassware and utensils
are to be purchased and segregated from dedicated glassware and utensils.
</ul>

<h2>In Vitro Gene Synthesis</h2>
<p>
Oligonucleotides and primers have been chemically synthesized and should be 
available.  The coding sequence represents the consensus of multiple alignment 
analysis of all accession numbers specified by Bartholomeusz &amp; Schaefer as 
reference sequences for all HBV genotypes 

<h3>LHBs Message PCR Assembly</h3>

<!--  PCR ASSEMBLY TEXT BOX start -->
<div style="float:right;margin-left:1em;width:25em;background-color:#fff8f8;border:1px solid red;padding:0.5em 1em;font:normal 85% Tahoma,Helvetica,sans-serif;">
<p style="font:bold 110% Verdana,Tahoma,sans-serif;color:blue;">
The Two-Step PCR Assembly Method

<ol>
<li>The volume of the reaction is 50 &micro;l. Into a PCR tube, place the following:
 <ul>
 <li>10 nmol of each dNTP (final&nbsp;200&nbsp;&micro;M)
 <li>5 pmol of each oligo and primer to be used (final&nbsp;100&nbsp;nM)
 <li>125 nmol of MgCl<sub>2</sub> (final&nbsp;2.5&nbsp;mM)
 <li>Add the <i>Taq</i> polymerase buffer in the appropriate volume.
 <li><i>Taq</i> polymerase in appropriate amount<br>
   Either 0.5 U Promega polymerase or 1 U of NEB <i>Vent</i> DNA polymerase are
   recommended
 <li>Add Type I water to a final volume of 50 &micro;l
 </ul>
<li>Set the thermal cycler program as follows:
 <ul>
 <li>Initial step: 95&deg; for 2 min
 <li>Denaturation: 95&deg; for 30 s
 <li>Anneal:  52&deg; for 30 s
 <li>Elongation: 72&deg; for 1 min
 <li>Termination: 72&deg; for 5 min
 <li>Number of cycles:  30
 </ul>
<li><b>Second PCR</b>.  In a PCR tube, in a final 50 &micro;l volume, 3 &micro;l 
of the PCR assembly mix is added to a mix with a final 200 &micro;M each dNTP, 1 
&micro;M primers, the appropriate amount of <i>Taq</i> polymerase and its buffer 
and 2.5 mM MgCl<sub>2</sub>.  The thermal cycler program is as for the first 
PCR, except that the annealing temperature is 5&deg; less than the lowest primer 
<i>T</i><sub>m</sub>.  Note that primers as long as 40 nt have been used:  30-35
nt of template complement, 6 nt of restriction site, and a 6 nt extension for
restriction enzyme foothold.
</ol>
</div>
<!--  PCR ASSEMBLY TEXT BOX end -->
<p>
The LHBs translated sequence is exactly 400 amino acids, or 1200 bp long in 
oligo form.  The consensus sequence was optimized for codon usage in bacteria 
(<i>E coli</i>, <i>B subtilis</i>).  It might need to be optimized to prevent 
possible tRNA exhaustion.  The 1200 bp sequence was divided into three fragments 
(LHBs F1, LHBs F2, and LHBs F3) and each fragment was submitted to processing by 
Gene2Oligo.  Oligos and primers for gene synthesis are to be chemically 
synthesized and purified using at least gels or HPLC (we don't want to risk just 
using desalted oligos, which may contain shortened fail synthesis oligos).

<p>
The following steps will be performed to synthesize the LHBs message:
<ol>
<li><b>Premxing synthesized oligos and primers</b>.  Perform the first step PCR 
assembly with reactions for the three LHBs fragments.  The oligos and primers 
for each of the LHBs three message fragments assemblies are in the <a 
href="#g2o">Gene2Oligo</a> section:  <a href="#LHBsG2OFrag1">LHBs message 
fragment 1</a>, <a href="#LHBsG2OFrag2">LHBs message fragment 2</a>, <a 
href="#LHBsG2OFrag3">LHBs message fragment 3</a>.  It is vital to mix 
oligos and primers properly.  It might be necessary to resuspend oligos and 
primers in distilled water and then mix them in a large tube, then lyophilize 
the tube down, and resuspend in smaller volume of water for the reaction.  For 
instructions on how to do that, see the box at left.
<li><b>First Step Assembly of three LHBs fragments</b>.  Now do the first step 
of the PCR assembly for 30 cycles according to the protocol at the top.  The 
second PCR is done normally with just primers to eliminate the bases added for 
doing PCR assembly which are not part of the sequence.  This step is omitted, 
because now we have to assemble the three LHBs fragments using overlapping 
oligos and the LHBs 5&prime; sense/forward and 3&prime; anti-sense/reverse 
complement ends only.  This is also PCR assembly but with fewer fragments.

<li><b>Joining three LHBs fragments By Another PCR reaction</b>.  Mix into
a PCR tube the following oligonucleotides and primers (final 100 nM each):
 <ul>
 <li>L1L2fo
 <li>L1L2ro
 <li>L2L3fo
 <li>L2L3ro
 <li>LHBsfp
 <li>LHbsrp
 </ul>
It might be necessary to form a premix, and dry it down using lyophilizer
or centrifugal evaporator, then resuspend in water and place in the reaction.
<li>To the synthetic sequences, add 3 &micro;l of each of amplification reactions
for Fragments 1, 2, and 3 of LHBs reactions.  Add the PCR components (master mix,
or dNTPs, buffer, magnesium, and enzyme) and run the program.  The annealing 
temperature should be set at 60&deg;.  Times and temperatures for other steps
and cycle steps should be as for the PCR assembly.  Use 30 cycles.
<li>Examine using agarose gel electrophoresis about 10 &micro;l of this 
amplification with 10 &micro;l each of the PCR assembly reactions of Fragment 1, 
2, and 3 amplifications.  You are looking for 1200 bp exactly for the fragment
joining reaction, and bands of 420, 420, and 360 bp for Fragments 1, 2, and 3
respectively.
<li>If the 1200 bp band is not detected, perform a standard PCR (not assembly) 
reaction using the forward and reverse primers of LHBs (LHBsfp and LHBsrp).
Verify that a 1200 bp band is present.  It might be a good idea to perform this
reaction anyway.
</ol>


<div style="background-color:#f8f8ff;border:1px solid blue;padding:0.5em 1em;font:normal 80% Helvetica,Palatinosans-serif;">
Commercially obtained primers will often be purchased in <i>x</i> nmol amounts, 
where <i>x</i> is usually 25 or 50. Since the primer and oligos per reaction are 
about 5 pmol, which is one ten-thousandth of the amount you have, it will be a 
good idea to premix the PCR assembly reaction oligos and primers, dry them down, 
and resuspend in the volumes needed.  Since there are about 30-40 oligos and 
primers per mix, and we can comfortably pipet 5 &micro;l, giving a total of 
150-200 &micro;l, we will put the mix in a 500 &micro;l tube.
<p>
In addition, for each 5 &micro;l per oligo or primer, let's use 50 pmol, enough 
for 10 such reactions.  Therefore, you want to resuspend the commercial tube of 
<i>x</i> nmol to a concentration of 50 pmol/5 &micro;l = 10 pmol/&micro;l. If 
you have <i>x</i> nmol = 1000<i>x</i> pmol, then this is 1000<i>x</i> pmol 
&times; 1 &micro;l/10 pmol = 100<i>x</i> &micro;l = (<i>x</i>/10)ml.  
<p>
The problem is that most vials delivered are about 1 ml maximum in volume.  If 
the value <i>x</i>/10 is &gt; 1, then it is not possible to resuspend that 
amount.  So instead, do the following: 
<ol>
<li>Add 500 &micro;l water to the tube containing 1000<i>x</i> pmol of oligo or 
primer, giving a concentration of 2<i>x</i> pmol/&micro;l.
<li>Now take 500/<i>x</i> &micro;l of 2<i>x</i> pmol/&micro;l, and add to (100 
&minus; 500/<i>x</i>) &micro;l of water, 100 &micro;l of 10 pmol/&micro;l.
<li>To the 500 &micro;l tube, add 5 &micro;l (total 50 pmol each) of the oligos
and primers to mix them.
<li>Lyophilize the tube down.
<li>In resuspending, you can resuspend in water so that each reaction is 
in a 5 or 10 &micro;l volume, as you wish:  this means the total volume in the
resuspension will be 50 or 100 &micro;l, since there are 10 reactions.
</ol>
</div>

<h4>Confirming the LHBs Message Amplicon</h4>
<p>
For the time being, we will verify the amplification reaction using restriction
digestions instead of sequencing.  Perform three digestions with
<ul>
<li>1-cutter:  <i>Not</i>I or <i>Sph</i>I
<li>2-cutter:  <i>Pvu</i>II or <i>Pst</i>I
<li>3-cutter:  <i>Bam</i>HI or <i>Bgl</i>I
</ul> 
<p>
Check the expected band sizes in the tables below.  Run the digests in separated 
lanes by 2.0% agarose gel electrophoresis in 0.5 &times; TBE

<h3>SHBs and MHBs Messages</h3>
<p>
Simply use 1-3 &micro;l of the LHBs amplification reaction in a new PCR with 
the primers below to get the SHBs and MHBs.  Primers (at 250 nM) will be:
<ul>
<li><b>SHBs</b>
 <ul>
 <li>sense/forward: SHBsfp
 <li>anti-sense/reverse-complement:  LHBsrp
 </ul>
<li><b>MHBs</b>
 <ul>
 <li>sense/forward: MHBsfp
 <li>anti-sense/reverse-complement:  LHBsrp
 </ul>
</ul>
<p>
Use an annealing temperature of 60&deg;.  Other temperature settings and times
are as for the second PCR after PCR assembly.

<h3>Nomenclature</h3>
<p>
For purposes of distinguishing DNA sequences created here for expression cloning,
it is suggested that the sequences have the format
<b>[L|M|S]HBs[Cns|Nat][0-9][0-9]</b>.  With the [ ] brackets and separated by
the <q>|</q> character will be one character of L, M, or S depending on the
message.  In the case of a sequence generated as a consensus and not spliced
segments from sequences found in nature, the Cns (for sequence) designation is
used.  If the sequence could basically generated from joining segments of DNA
that could be cut from natural sequences, the Nat designation is used.  Finally
two digits are used ordinally for each sequence created.  It is vital to
keep a catalogue with the proper name of the sequence connected with the sequence.
<p>
For the consensus LHBs message created for this project, the name shall be
<span class="tt"><b>LHBsCns01</b></span> and it should be included in the
naming of other products including/inserting it.d

<h2>Cloning Into <i>E coli</i> Maintenance Host</h2>
<p>
For the purposes of maintenance cloning, only the LHBs translated code and not
the MHBs or SHBs message regions will be inserted into maintenance vector pUC19
and transformed into DH5&alpha; for blue-white screening.  Successfully
transformed hosts will be stored in a deep freezer with a standard cryoprotectant
medium.  In addition, plasmid preps will be obtained from transformed hosts
and stored in the freezer as well.
<p>
Chemically competent cells might be purchased rather than generated (?).  If
purchased, transformation will be according to manufacturer's directions.
Subsequent to treatment of DNA with competent cells, 100 &micro;l of cell
suspension will be used for each standard 100 mm LB agar plate with the
appropriate selection medium at the appropriate concentration.
<p>
The purpose of this section will be to take the 1200 bp LHBsCns01 sequence and
insert it into 2686 bp pUC19 vector to create pLHBsCns01-UC19.  If the OE-PCR
method is used, the final product will be a relaxed circular plasmid which has
removed bases 402 to 434 of the multiple cloning site, between <i>Eco</i>RI and
<i>Pst</i>I, at total of 33 bp, and in its place will the 1200 bp LHBsCns01
insert.  The resulting product should be 2686 - 33 + 1200 = 3853 bp,
pLHBsCns01-UC19.  If that product is or can not be created, and the standard
cut-and-ligate methods are used, then the LHBs message will be extended by
PCR with <i>Bam</i>HI and <i>Pst</i>I segments and then it and the pUC19 vector
are digested with those enzymes.  Positions 423 to 434 are removed from the pUC19
vector and the 1200 bp is inserted between, to produce a product that is
2686 - 12 + 1200 = 3874 bp long.

<h3>Overlap-Extension PCR Cloning</h3>
<p>
We will try this cloning technique. This is an alternative traditional use of
restriction enzyme cutting and ligating inserts into vectors. As a backup, we
will have materials and methods for performing the traditional cut-and-ligate
technique (see the next section).
<p>
OE-PCR saves steps that can affect yields and time of processing. It requires
special high fidelity and processive thermostable DNA polymerases. Those
polymerases that work as well as NEB Phusion DNA polymerase are used. The
general principles for this technique were described by <a
href="/proteomics/reference/GeneralScienceVol3%20%282001-present%29.html#bryskin">Bryskin
&amp; Matsumura (2010).</a>
<p>
<b>PCR Amplification of Insert With Vector-Bridging (Overlap) Sequence</b>
<ol>
<li>To 200 &micro;l PCR tubes or tray wells, add 0.1 fmol to one tube and
1.0 fmol of 1200 bp LHBs coding sequence to another tube.  Use in calculations
a molecule mass of 79.2 pg/fmol for the LHBs, and that 1 AU<sub>260</sub> =
50 &micro;g/ml purified ds DNA.  The DNA should have been cleaned using PCR
clean-up kit or standard phenol-chloroform-isoamyl alcohol with EtOH precipitation.
<li>Add 10 pmol of OE-UC19-LHBs-fp and OE-UC19-LHBs-rp primer pairs to the reaction.
(This is 2 ul of a 5 pmol/&micro;l primer stock dilution.)
<li>Add ice-cold PCR components except, preferably as a master mix.  Keep on
ice-water bath.  Use a final 1.5 mM MgCl<sub>2</sub> concentration, and 0.2 mM each
of the dNTPs.  Use a standard <i>Taq</i> DNA polymerase in amounts specified
by the product vendor (usually 0.5 to 2 U).
<li>Add distilled or Type I water to a final 50 &micro;l
<li>The PCR program starts with 98&deg; for 2 min.  Then 30-32 cycles of:
 <ul>
 <li>denaturation: 94&deg; for 30 s
 <li>annealing:  60&deg; for 30 s
 <li>polymerization (extension):  68&deg; C for 120 s (2 min)
 </ul>
The final extension is 68&deg; for 10 min.
<li>Pour a small 0.6% agarose gel and load the amplification reaction.  When the
bromophenol blue marker has reached to the middle of the gel, examine the gel in
the transilluminator:  prepare a photo and turn on the transilluminator only
long enough to get a photo, then turn it off.  Rapidly inspect the photo to look
at the separation.  When you identify a 1246 bp fragment, that is mosly likely
your amplicon.
<li>Gel-purify the fragment after you cut it out of the gel. Use a gel
extraction kit to recover the product and estimate recovery and concentation
using A<sub>260</sub>. Suspend the fragment in a minimal amount of TE, and
store remainder that is not used in the next process below at &minus;80&deg; in
an O-ring screw cap tube.  This PCR product will have the name <b>LHBs-pUC19flanked</b>
for the purposes of this document, and you can label it so.
</ol>

<p>
<b>pLHBs-UC19: PCR-Based Insertion To Vector Using the Overlap Extension</b>.
You will perform at least five reactions: four are with different ratios of
vector-insert primers-to-vector ratios, and one is with a control template DNA
and its 1.3 kb primers. The tubes can be numbered 1 to 5 (or if tray wells are
used, determine their label according to number 1 to 5).  The target molecule created
in this section is a circular relaxed double-stranded DNA which is 3853 bp in
length, with sequences derived from LHBs translation sequence and pUC19 vector.
It shall be named <b>pLHBs::UC19</b>

<ol>
<li>Prepare in a 200 &micro;l PCR tube or well a volume of 25 &micro;l of a 2X
master mix for the NEB Phusion polymerase as follows:
 <ul>
 <li>1.0 &micro;l of 10 mM dNTP mix =&gt; 0.4 mM
 <li>10 &micro;l of 5X buffer =&gt; 2X
 <li>1.5 &micro;l DMSO =&gt; 6%
 <li>1.5 &micro;l 50 mM MgCl<sub>2</sub> =&gt; 3 mM
 <li>1.0 &micro;l of the Phusion DNA polymerase =&gt; 0.04 U/&micro;l
 <li>10.0 &micro;l distilled (or PCR-grade) water
 </ul>
Do not vortex, but mix using gentle finger flicking 20-25 times. Place on
ice-water bath to cool it down. It must be prepared within the same day of use.
Do not use a master mix from a previous day.
<li>To tube #1, transfer 0.5 &micro;l of the 1.3 kb primers (4 &micro;M) for
final 0.2 &micro;M.
<li>With a new tip, transfer 0.4 &micro;l of the control template DNA (0.5
ng/&micro;l) to tube #1 (final 0.02 ng/&micro;l).
<li>To tubes #2 to #5, add 3 ng (1.7 fmol) of purified pUC19 plasmid. Use in
calculations a molecule mass of 1.78 ng/fmol for pUC19 if necessary. You should
adjust an aliquot of the pUC19 plasmid stock to a concentration to 1.5
ng/&micro;l, then add 2.0 &micro;l.
<li>To tubes #2 through #5, add the add respectively 69, 139, 347, and 693 ng
of LHBs-pUC19-flanked amplicon to these tubes.  The are 84, 169, 421, and 843
fmol of the amplicon, and represent a LHBs-pUC19-flanked insert:pUC19 vector
ratio of 50:1, 100:1, 250:1, and 500:1.  Prepare four tubes which are
(#&nbsp;ng amt needed)/3 nanograms per microliter for each of the four tubes to
be filled.  Thus prepare four tubes with at least 3 &micro;l each of
LHBs-pUC19-flanked at 69/3 = 23, 139/3 = 45, 347/3 = 139, and 693 = 231 ng/&micro;l.
(If you worry about pipetting error, make 4 &micro;l of each.)  Now pipet the
3.0 &micro;l to tubes #2 through #5.
<li>Program the thermal cycler to initialize at 98&deg; for 2 min.  Then for
18 cycles, program the following:
 <ul>
 <li>Denaturation: 98&deg; for 30 s
 <li>Annealing:  60&deg; for 30 s
 <li>Extension:  68&deg; for 360 s (6 min)
 </ul>
The final extension time at 68&deg; should be set for 10 min.
<li>Set the temperature at 98&deg; until the program starts. Directly from the
ice-water bath to the thermal cycler, transfer the tubes or tray, then close
the lid quickly. Start the program immediately (don't let it sit)!
<li>After the PCR is done, digest each reaction with <i>Dpn</i>I enzyme using
specified conditions. If the digest is not compatible with PCR contents, clean
the PCR reaction with PCR clean-up kit first, then digest.
</ol>

<h4>Backup Method In Case of OE-PCR Failure</h4>
<p>
A classic cut-and-ligate will be done on the insert and vector in the event of
failure of OE-PCR. First the LHBs message will be extended on the 5&prime; and
3&prime; ends using PCR to include restriction sites for enzymes in pUC19 and
already ordered for this project for other uses; the sites will be <i>Eco</i>RI
and <i>Xba</i>I.  Then the amplicon and pUC19 plasmid will be digested with the
enzymes.  This will be followed by mixing the digests with high insert:vector
ratios in the presence of T4 DNA ligase.  The ligation reaction is then
immediately used to transform competent <i>E coli</i> cells, and transformants
detected using blue-white screening.

<p>
Before starting the procedure, make sure the following have been prepared:
<ul>

<li><a href="/proteomics/reference/GenomicsReference.html#blue-white-xform">100 mm
petri dishes</a> with X-gal + IPTG (be economical): make no more than 5 plates
beyond what you estimate that you will need.  Store unused plates in proper place
(add wet towels wrapped in plastic bag to keep hydrated).
<li>Chemically competent <i>E coli</i> using <a
href="/proteomics/reference/GenomicsReference.html#inoue">Inoue method</a>
<li>Pure plasmid prep of pUC19: see genomics methods for multiple methods of making
pure plasmid prep
</ul>

<p>
Perform the following steps:
<ul>
<li>Placing Vector Restriction Sites on Inserts Using PCR
 <ol>
 <li>In a 200 &micro;l PCR tube, add 50 pmol each of primers cutlig-UC19-LHBs-fp and
OE-UC19-LMSHBs (final 1 &micro;M in a 50 &micro;l total volume reaction).  Place
0.01 fmol (10 amol) of LHBs message.  Perform the PCR using a standard method.
Include an amplification in which there is no LHBs message.
 <li>Run the reaction and negative control on 0.6% agarose gel to verify
amplification.  Use markers if possible separated by 100 or 50 bp in the range
between 1100 and 1400 bp.
 <li>Get the OD<sub>260</sub> to estimate the amount of amplicon available, or
try to estimate it from the gel band the amount present.
 </ol>
<li>Double Digestion of Insert and Vector
 <ol>
 <li>To a small tube (0.2 or 0.5 ml conical), add 3 &micro;l 10&times; NEB Buffer
4, 1.5 &micro;l of 20&times; BSA (from a 5-fold diluted 100&times; BSA stock),
and  22.5 &micro;l of the amplification reaction.  Pre-warm tubes in 37&deg; water
bath.
 <li>To another small tube, add 200 ng of pUC19.  Now add 2 &micro;l of 10&times;
NEB Buffer 4, 1.0 &micro;l 20 &times; BSA, and bring to total up to 18.0 &micro;l
with distilled water. Pre-warm to 37&deg;.
<li>Add 1.0 &micro;l each of 1 U/&micro;l <i>Eco</i>RI and <i>Xba</i>I and incubate at
least 30 min at 37&deg; (or follow manufacturer's directions).
 <li>After the time of incubation is complete, place the tube in a 65&deg;
water bath for at least 20 min.  This will allow ligation of the digest
products without the need for purification; otherwise active endonucleases
might cause digestion of the newly ligated product.
 </ol>
<li>Ligation
 <ol>
 <li>[ ]
 <li>[ ]
 </ol>
</ul>

<h3>Transformation and Culture</h3>

<h3>Plasmid Prep &amp; Characterization</h3>

<h2>Constructing Expression Systems for HBsAg Forms</h2>
<p>
Expression of the recombinant proteins is the most difficult part of the project.
It is made more complicated by the fact that attempts to produce HBsAg in the
various have proved difficult because HBsAg is likely toxic to the <i>E coli</i>
host or it forms inclusion bodies of insoluble protein product that is difficult
to solubilize, or the processes involved in solubilization denature the protein
and make it difficult or impossible to re-fold the protein in the proper form for
restoring immunoreactivity, necessary to make an ELISA product with the protein
product.  We are trying to maximize the yield of protein, so even after we find
one system that expresses product enough to get started, we must optimize the 
system, which means continue testing expression systems that produce even more
protein
<p>
We will initially use a bacterial host.  This is generally <i>Eschericia 
coli</i>, but there is a <i>Bacillus subtilis</i> system to be tried too. 
Expression in bacterial hosts  is usually sufficient for protein to be used for 
medical diagnostic kit (ELISA, etc) production.  The advantages are that 
bacteria can be grown rapidly at very low cost compared to other expression 
systems.  The disadvantage is that they lack the post-translational modification 
processes such as glycosylation, particularly for many secreted or membrane 
proteins that are to be cloned.
<p>
<b>Intracellular Expression vs. Secretion</b>.  Bacteria can secrete proteins
as well as express them intracellularly.  When possible, secretion should be
preferred since it means that cell extraction is not required, which could
eliminate the need for multi-step purification as well as make it possible to
express the protein with the use of fusion or chimeric tags, which even of short
length might affect the function of the recombinant if the tag is present.
In cases where intracellular expression is done, it will likely be tried with
a tag for an affinity purification step.

<h3>Expression in a <i>Bacillus subtilis</i> Secretory System</h3>
<p>
Perform the following steps:
<ol>
<li>Order and obtain the Takara <i>B subtilis</i> expression kit and all of the
other required components, including but not limited to:
 <ul>
 <li><i>Mlu</i>I and <i>Eco</i>52I restriction enzymes for MHBs and SHBs cloning
 <li>Plasmid purification kit,  e.g. Clontech NucleoSpin Plasmid (Cat 740588.10)
 <li>chemically competent <i>E coli</i>, e.g. HST08 Premium Competent Cell (Cat 9128)
 <li>PCR-based cloning, e.g. In-Fusion HD PCR Cloning kit (Clontech 639633-639650)
 </ul>
<li><b>Preserving the pBE-S vector DNA</b>.  Perform this step if you are have
used up nearly all the pBE-S vector DNA, for instance if there is only 4 &micro;g
of vector, leaving enough for only two reactions.  You should do this after the
first reaction in order to gain practice.
 <ol>
 <li>Transform the pBE-S vector DNA into a strain of <i>E. coli</i> that can
replicate plasmids with ColE1 origin.  Culture the transformant using kanamycin
or ampicillin selection.
 <li>Isolate the plasmid from a small amount of the bacteria
 <li>Perform PCR with the following primers
 <li>On a portion of the amplification reaction, perform restriction digests
 with the following enzymes:
  <ul>
  <li>Enzyme
  </ul>
 <li>Run the undigested and digested PCR products on agarose, and look
  for the following band sizes:
  <ul>
  <li>Undigested
  <li>Enzyme 1 digest
  <li>Enzyme 2 digest
  </ul>
 </ol>
<li>Store bacteria in deep-freeze in cryoprotected medium
</ol>
<!-- ======================================================================
                   START OF 
        TaKaRa Bio Company Bacillus subtilis expression
======================================================================= -->
<div class="boxed-instructions">
The Takara Bio Company (a Clontech subsidiary) has developed a <i>B. subtilis</i>
strain in which protein product is secreted by attaching a signal peptide to the
sequence.  Their system can attach 173 different signal sequences to the cloned
insert and test all of them simultaneously to find out which clone provides
the best secretion.
<p>
Most of the cloning activities to build the final plasmid will be done in <i>E 
coli</i> because <i>B. subtilis</i> strain RIK1285 has a low copy number for 
producing plasmids necessary in construction.  The shuttle vector pBE-S will be 
used as the base in building the plasmid.  A mixture of SP (secretory peptides)
DNA is provided in the kit for insertion and creating clones.
<p>
The pBE-S vector is constructed with an origin of replication from pUB110 and
a kanamycin-resistance gene (Kan<sup>r</sup>) for selection and plasmid copying
in <i>B. subtilis</i>.  The vector also has a ColE1 origin of replication and
an Amp<sup>r</sup> gene for selection and growth in <i>E coli</i>.  The promoter
is <i>aprE</i> and the <i>aprE</i> SP (signal peptide).
<p>
Target gene is constructed and inserted into the MCS.  Note that it cannot 
contain <i>Mlu</i>I and <i>Eco</i>52I cutting sites, since the plasmid is 
linearized using those enzymes for use with the Clontech In-Fusion system to 
introduce the SP DNA mixture in place of <i>aprE</i> SP, which is the subsequent
step.  The constructed plasmid with 173 different SP and the insert is now
transformed into <i>E coli</i> HST08.   From these a plasmid library is now
prepared, and these are transformed into <i>B subtilis</i> RIK1285.  Screening
is then done to look for the highest secretor.
<p>
The kit (Cat #3380) contains enough for 10 reactions and has the following
three components:
<ol>
<li>45 &micro;l (4.5 &micro;l per reaction) of 32 fmol/&micro;l SP DNA mixture
<li>20 &micro;g (2 &micro;g per reaction) of 500 ng/&micro;l pBE-S DNA
<li>200 &micro;l (as 2 &times; 100 &micro;l) <i>B subtilis</i> RIK1285 in
glycerol stock
</ol>
<p>
The bacteria must be stored at &minus;80&deg; and the DNA at &minus;20&deg;.
Note the pBE-S is in 40 &micro;l of TE pH 8.0 buffer
<p>
Additional components <b>required</b> to use the system not in the kit are:
<ul>
<li>In-Fusion HD PCR Cloning kit (Clontech Cat #639633-639650)
<li>competent <i>E coli</i> HST08 Premium competent cells (Cat 9128)
<li><i>Mlu</i> I (Cat #1071A)
<li><i>Eco</i>51I (Cat #1039A)
<li>Plasmid purification kit (NucleoSpin Plasmid: Clontech Cat #740588.10)
<li>[ ]
</ul>
<p>
The Takara Bio Systems document describes the step-by-step protocol to use.
The first and long set of steps are the construction of the plasmid library.
<ol>
<li>The target insert is constructed, with restriction sites on the 5&prime;
and 3&prime; ends for insertion into the pBE-S multiple cloning site, and so
that it matches the translation frame.
<li>The constructed plasmid is then digested with <i>Mlu</i>I and <i>Eco</i>52I,
although not simultaneously.  The fragment is then purified by agarose gel
electrophoresis.  Note that the expression plasmid may need to be <q>linearized</q>
using PCR if the target gene contains <i>Mlu</i>I or <i>Eco</i>52I sites.
The LHBs, but not MHBs and SHBs forms of HBsAg contains an <i>Eco</i>52I (CGGCCG)
site at nt 303, so it will have to be cloned using PCR.
<li>The SP DNA mixture contains fragments already digested using <i>Mlu</i>I and 
<i>Eco</i>52I.  Mix the SP fragments at twice the molar ratio as the vector plus 
insert.  Use 100 ng of the plasmid.  Add 2 &micro;l of the 5&times; In-Fusion HD 
Enzyme Premix, then bring up the final volume to 10 &micro;l with sterile 
distilled water.  As a control, perform the reaction without addition of any the 
SP DNA mixture.  If the volume of plasmid and the volume of SP DNA &gt; 7 
&micro;l, then double the 5&times; Premix volume, then add water up to 20 
&micro;l.  If the insert were about 1.0 kb, and the vector is 5.8 kb, 100 ng of 
a 6.8 kb plasmid is 22 fmol.
<li>The mixture is incubated 50&deg; for 15 min then placed on ice.  At this
time, start thawing the <i>E coli</i> HST08 competent cells.
<li>Gently mix the thawed cells and transfer 100 &micro;l to 14 ml round bottom
test tube but do not vortex.
<li>Add 2 &micro;l of the In-Fusion reaction with SP+plasmid to the bacteria.
<li>Leave on ice for 30 min.
<li>Shock in a 42&deg; water bath for 45 seconds
<li>Incubate on ice for 1-2 min.
<li>Add 37&deg; pre-warmed SOC medium to final volume of 1 ml
<li>Shake 160-225 rpm for 1 h 37&deg;.
<li>Spread to LB + ampicillin (100 &micro;g/ml):  you might want to spread 
100-200 &micro;l to several plates.  Leave overnight at 37&deg;.
<li>Count the colonies obtained.  A count of 2000 or more would be necessary to
make a sufficient plasmid library.  (It has been found that 960 colonies 
incorporated 163 types.)  If the colony count is not obtained, transformation 
scale may need to be increased.
<li>Colonies are resuspended from the plate into LB medium.  If collecting colonies
from a square plate (140 &times; 104 &times; 16 mm) collect all the colonies into
5 ml LB medium, and wash twice in 3-5 ml of LB medium.  Spreaders can be used for
harvesting.
<li>Now purify plasmid DNA after pelleting the suspension.  About 20 &micro;g 
plasmid can be obtained from harvested colonies.  Adjust the DNA concentration to
0.1 to 1 &micro;g/&micro;l
</ol>
<p>
With the plasmid library constructed, the plasmids are transformed into 
<i>B. subtilis</i>, which must be made competent freshly (immediately used).
<ol>
<li>Prepare the following reagents, either autoclaving or filter-sterilizing:
 <ul>
 <li><b>SP I salts</b>.  Per liter:
  <ul>
  <li>2 g ammonium sulfate (NH<sub>4</sub>SO<sub>4</sub>)
  <li>14 g dipotassium hydrogen phosphate (K<sub>2</sub>HPO<sub>4</sub>)
  <li>6 g monopotassium dihydrogen phosphate (KH<sub>2</sub>PO<sub>4</sub>)
  <li>1 g sodium citrate dihydrate
  <li>0.2 magnesium sulfate heptahydrate (MgSO<sub>4</sub> &bull; H<sub>2</sub>O)
  </ul>
 <li><b>Casamino Acids/Yeast Extract</b> Per 100 ml:
  <ul>
  <li>2 g casmino acids
  <li>10 g yeast extract
  </ul>
 <li>50% D-glucose
 <li>50 mM CaCl<sub>2</sub>
 <li>250 mM MgCl<sub>2</sub>
 <li>100 mM EGTA: adjust to pH 7.0 with NaOH
 </ul>
<li>Prepare <b>SP I medium</b> by mixing the following using sterile technique:
 <ul>
 <li>10 ml SP I salts
 <li>0.1 ml 50% glucose
 <li>0.1 ml casamino acids/yeast extract
 </ul>
<li>Prepare <b>SP II medium</b> by mixing the following using sterile technique:
 <ul>
 <li>5 ml SP Medium
 <li>50 &micro;l 50 mM calcium chloride
 <li>50 &micro;l 250 mM magnesium chloride
 </ul>
<li>Spread glycerol stock of the <i>B subtilis</i> (RIK1285) strain on LB
plate and incubate overnight (~16 h) at 37&deg;.
<li>Take a loop full of colonies and inoculate a 2 ml tube of LB medium.  Grow
overnight (~16 h) at 28&deg;, shaking 150-180 rpm.
<li>Add 50 &micro;l of the broth to 5 ml of <i>fresh</i> <b>SP I medium</b>.
<li>Incubate 37&deg; 170-180 rpm.  Every 30 min after the first hour, measure
the OD<sub>660</sub> and stop culturing after growth has started to become 
stationary.  Plot the log OD<sub>660</sub> vs time.  This typically takes 5 h.
<li>Add 500 &micro;l of broth to 4.5 ml <b>SP II</b> medium, incubate 37&deg; at
90-100 rpm for 90 min.
<li>Add 50 &micro;l of 0.1 M EGTA, and shake 90-100 rpm at 37&deg; for 5-10 min.
<li>The culture can be divided to 14 ml test tubes, dispensing 300 &micro;l volume
to each tube.
<li>To each 300 &micro;l, be prepared to add immediately 1 &micro;g DNA.
<li>Incubate all tubes at 30&deg;, 90-100 rpm, for 90 min.
<li>Spread each culture on LB plates with 10 &micro;g/ml kanamycin.  Incubate
overnight at 37&deg;.  Note that transformation efficiency depends on the target
genes inserted.  Transformation is normally 300-800 colonies per microgram.
<li>Colonies are picked and incubated in 5 ml medium containing kanamycin at 
37&deg; between 24-48 hours.  Check the culture supernatant for the target 
protein.  It might be useful to acid precipitate the medium.  Measure the total
protein level as well.
</ol>
<p>
The NucleoSpin Plasmid kit can be used to purify plasmid.  Inoculate with
a single colony and shake/incubate overnight at 37&deg;.  Spin culture down
for 30 s at 11,000 <i>g</i>.  Add 250 &micro;l Buffer A1 with lysozyme added
to 4-10 mg/ml, vortex or pipet to resuspend pellet and transfer to 1.5 ml microtube
(lysozyme may need to be purchased separately).  Incubate 30 min 37&deg;, then 
continue purifying plasmid with the kit, starting at step where Buffer A2 is added.
Once the plasmid is purified, PCR might be used to amplify the signal peptide
coding segment and then to sequence it.
<p>
<b>Insert contains <i>Mlu</i>I or <i>Eco</i>52I</b>.  In the event the insert
contains the restriction sites, the plasmid is linearized using inverse PCR.
The primers are both 34 nt long with the sequences:
<pre>
     5&prime;-CGCGTCCCTCTCCTTTTGCTTAAGTTCAGAGTAG-3&prime;
     5&prime;-GGCCGGTGCACATATGXXXXXXXXXXXXXXXXXX-3&prime;
</pre>
</div>
<!-- ======================================================================
                   END OF 
        TaKaRa Bio Company Bacillus subtilis expression
======================================================================= -->


<h3>Cold Shock Expression:  pCOLD</h3>
<p>
Incubating <i>E coli</i> at lower temperatures causes the organism to express a set
of genes regulated by a cold shock system.  This may be useful for recombinant 
(heterologous) proteins that cause toxicity or which may show unwanted insolubility
within the intracellular space (cytoplasm) of the cell.  In general, protein expression
rates are reduced at cold temperatures except for one system, the cold system.  One
gene expressed in particular is <i>cspA</i> (cold shock protein A).  The DNA elements
necessary for transcription and translation of the <i>cspA</i> gene have been isolated
and created in a series of vectors called pCOLD and marketed by Takara Bio/Clontech.
These elements are in 5&prime;&rarr;3&prime; order:
<ul>
<li><i>cspA</i> promoter
<li><i>lac</i> operator
<li>the 5&prime; untranslated region of the mRNA
<li>the translation-enhancing element (TEE) (depends on vector)
<li>a His-tag sequence (depends on vector)
<li>a Factor Xa cleavage site  (depends on vector)
</ul>
<p>
Immediately following are the MCS and then the 3&prime; untranslated region containing
transcription termination elements.
<p>
In addition the vectors contain:
<ul>
<li><i>lacI</i> gene to code for the repressor which binds to the <i>lac</i> operator
<li>the ColE1 origin of replication
<li>beta-lactamase gene for ampicillin resistance (selection)
<li>M13 IG
</ul>
<p>
All of these are 5&prime; to a multiple cloning site featuring unique 
restriction sites of common enzymes.  The features of the series of vectors
are as follows:
<ul>
<li>pCOLD I (4407 bp, GenBank AB186388) contains the TEE, His-tag, and Factor Xa 
protease site in 5&prime;&rarr;3&prime; order just 5&prime; to the MCS.
<li>pCOLD II (4392 bp, GenBank AB186389) omits the Factor Xa protease site in 
vector pCOLD I.
<li>pCOLD III (4377 bp, GenBank AB186390) omits the Factor Xa and His tag 
elmenents (just the TEE).
<li>pCOLD IV (4359 bp, GenBank AB186391) which omits all three of the elements
</ul>
<p>
All four vectors can be purchased (Cat #3360) at 5 &micro;g each, or each vector 
can be purchased separately at 25 &micro;g each (Cat #3361 through #3364 for 
vectors pCOLD I-IV).
<p>
Factors to control in cultivation and induction are the medium and temperature of
culture, aeration, timing of the induction, inducer concentration, and induction
time period.
<p>
The general protocol for creating expression-competent recombinants is
<ol>
<li>Construct the expression plasmid using selected pCOLD vector using standard 
tools:  restriction enzyme, ligases, polymerase, etc.  The MCS sequence is 60 nt 
long containing 10 restriction enzyme sites (6 nt each): 
5&prime;-CATATGGAGCTCGGTACCCTCGAGGGATCCGAATTCAAGCTTGTCGACCTGCAGTCTAGA-3&prime;. 
These are <i>Nde</i>I, <i>Sac</i>I, <i>Kpn</i>I, <i>Xho</i>I, <i>Bam</i>HI, 
<i>Eco</i>RI, <i>Hin</i>dIII, <i>Sal</i>I, <i>Pst</i>I, <i>Xba</i>I
<li>Transform into proper strain (BL21), plate on ampicillin selection medium
<li>Select a transformant and inoculate into amp-containing medium (50-100 &micro;g/ml),
and culture at 37&deg; with shaking.
<li>When OD<sub>600</sub> ~ 0.4-0.5, refrigerate at 15&deg; quickly.  Let stand about
30 min.
<li>Add IPTG from 0.1 to 1.0 mM and continue shaking at 15&deg; for 24 h
<li>Get the cell pellet and extract.  Check for expression on SDS-PAGE minigels.
</ol>

<h3>Intracellular Expression in <i>E coli</i> using pET</h3>
<p>
[to be completed]

<h3>Secretion Expression in <i>E coli</i> using pET</h3>
<p>
[to be completed]


<!-- =======================================================================
       Nucleotide and Amino Acid Sequences
 ========================================================================== -->


<h1 id="sequences">Sequences &amp; Other Detailed Bioinformatics</h1>


<h2>Amino Acid Sequences</h2>
<p>
Genotype A (X02763) is renumbered to match other genotype numberings.  
Positions 1410-3221 are renumbered 1-1812, and positions 1-1409 are renumbered
1813-3221.


<!--
 S protein consensus 1
   MENITSGFLGPLLVLQAGFFLLTRILTIPQSLDSWWTSLNFLGGTPVCLG
   QNSQSPTSNHSPTSCPPICPGYRWMCLRRFIIFLFILLLCLIFLLVLLDY
   QGMLPVCPLIPGSSTTSTGPCKTCTTPAQGTSMFPSCCCTKPSDGNCTCI
   PIPSSWAFAKYLWEWASVRFSWLSLLVPFVQWFVGLSPTVWLSVIWMMWY
   WGPSLYNILSPFIPLLPIFFCLWVYI

 M protein consensus 1
   MQWNSTTFHQALQDPRVRGLYFPAGGSSSGTVNPVPTIASLISSIFSRTG
   DPAPNMENITSGFLGPLLVLQAGFFLLTRILTIPQSLDSWWTSLNFLGGT
   PVCLGQNSQSPTSNHSPTSCPPICPGYRWMCLRRFIIFLFILLLCLIFLL
   VLLDYQGMLPVCPLIPGSSTTSTGPCKTCTTPAQGTSMFPSCCCTKPSDG
   NCTCIPIPSSWAFAKYLWEWASVRFSWLSLLVPFVQWFVGLSPTVWLSVI
   WMMWYWGPSLYNILSPFIPLLPIFFCLWVYI

 L protein consensus 1
   MGGWSSKPRKGMGTNLSVPNPLGFFPDHQLDPAFRANSNNPDWDFNPNKD
   HWPDANKVGVGAFGPGFTPPHGGLLGWSPQAQGILTTLPADPPPASTNRQ
   SGRQPTPISPPLRDTHPQAMQWNSTTFHQALQDPRVRGLYFPAGGSSSGT
   VNPVPTIASLISSIFSRTGDPAPNMENITSGFLGPLLVLQAGFFLLTRIL
   TIPQSLDSWWTSLNFLGGTPVCLGQNSQSPTSNHSPTSCPPICPGYRWMC
   LRRFIIFLFILLLCLIFLLVLLDYQGMLPVCPLIPGSSTTSTGPCKTCTT
   PAQGTSMFPSCCCTKPSDGNCTCIPIPSSWAFAKYLWEWASVRFSWLSLL
   VPFVQWFVGLSPTVWLSVIWMMWYWGPSLYNILSPFIPLLPIFFCLWVYI
-->

<pre style="font-size:75%;">

+--- Legend ----------------------------------------------------------+
| Numbering:                                                          |
|   xxx-xxx:  numbering of all amino acids in sequence                |
|  [xxx-xxx]:  numbering of amino acids in region/domain of S protein |
| 1. *  S protein position 122 d|y determinant Lys&rarr;Arg variation      |
| 2. #  S protein position 160 r|w determinant Lys&rarr;Arg variation      |
| 3. % ayw1&rarr;ayw2 variation,ayw1=adw2 at these positions               |
| 4. + adw2&rarr;ayw1 variation                                            |
| 5. R122 required for w1                                             |
| 6. T127 required for w3                                             |
| 7. L127 and S140 required for w4                                    |
+---------------------------------------------------------------------+

   1- 50:
[  1- 50]:
         1         2         3         4         5
12345678901234567890123456789012345678901234567890
&lt;------------- PreS1 (L - Nterm) -----------------
MGGWSSKPRKGMGTNLSVPNPLGFFPDHQLDPAFGANSNNPDWDFNPVKD  X02763   Genotype A (adw2)
-----------.......................................  AF297621 Genotype A (african)
..................................K...E.....L..N..  D00330   Genotype B-a (adw)
..................................K...E.....L..H..  AB073858 Genotype B-j (adw)
.........Q........................................  AB033556 Genotype C (adw)
.......H..........................................  AB048704 Genotype C-aus (ayw)
-----------..Q...TS...............R..TA........N..  X02496   Genotype D
-MGL.WTVPLEW.K.I.TT...............R..TR.....H..N..  X75657   Genotype E
..APL.TT.R...Q..................L.R...SS......TN..  X69798   Genotype F (adw4)
-MGL.WTVPLEW.K...AS.....L.........R..T.........K..  AF160501 Genotype G
..APL.TA.R...Q..................L.R...SS......TN..  AY090454 Genotype H
.........Q.....................................N..  P31868 Jap (adr)
.........Q............................H........N..  P12934 Jap (adr)
.........Q.....................................N..  P03140 Jap (adr)
.........Q.....................................N..  Q67867 Kor (adr)
-----------..Q...TS...............R..TA........N..  Bozdayi consensus (D)
MGGWSSKPRKGMGTNLSVPNPLGFFPDHQLDPAFRANSNNPDWDFNPNKD  consensus 1
                                               V    consensus 2


  51-100:
[ 51-100]:
         1         2         3         4         5
12345678901234567890123456789012345678901234567890
----------------- PreS1 (L - Nterm) --------------
DWPAANQVGVGAFGPRLTPPHGGILGWSPQAQGILTTVSTIPPPASTNRQ  X02763   Genotype A (adw2)
H..Q.H.........GF......V...........A.LPAV.........  AF297621 Genotype A (african)
N..D..K........GF......L.........L....PAA.........  D00330   Genotype B-a (adw)
N..D.HK.......SGF......L............S.PAA.........  AB073858 Genotype B-j (adw)
H..E.....A.....GF......L..............PAA.........  AB033556 Genotype C (adw)
H..E.....A.....GF......F..............PAA.........  AB048704 Genotype C-aus (ayw)
T..D..K..A....LGF......L...........E.LPAN.........  X02496   Genotype D
H.TE..K........GF......L.........M.K.LPAD.........  X75657   Genotype E
S..M..K....GY..GF......L.........V...LPAD........R  X69798   Genotype F (adw4)
P..E..K.....Y..GF......L......S..T...LPAD.........  AF160501 Genotype G
N..M..K....G...GF......L.............SPPD........R  AY090454 Genotype H
H..E.......T...GF......L..............PAA.........  P31868 Jap (adr)
...E...........GF......L.........V....PVA.........  P12934 Jap (adr)
Q..E.....A.....GF......L..............PAA.........  P03140 Jap (adr)
R..E.....A.....GYP.....L..............PAA.........  Q67867 Kor (adr)
T..D..K..A....LGF......L...........Q.LPAN.........  Bozdayi consensus (D)
HWPDANKVGVGAFGPGFTPPHGGLLGWSPQAQGILTTLPADPPPASTNRQ  consensus 1
N  E  Q                              V  A           consensus 2


 101-119:
[101-119]:
         1         2         3         4         5
12345678901234567890123456789012345678901234567890
--- PreS1 --------&gt;
SGRQPTPISPPLRDSHPQA                                 X02763   Genotype A (adw2)
...................                                 AF297621 Genotype A (african)
.......L......T....                                 D00330   Genotype B-a (adw)
.......L......T....                                 AB073858 Genotype B-j (adw)
...................                                 AB033556 Genotype C (adw)
..............T....                                 AB048704 Genotype C-aus (ayw)
.......L.....NT....                                 X02496   Genotype D
........T.....T....                                 X75657   Genotype E
...K...V......T....                                 X69798   Genotype F (adw4)
...................                                 AF160501 Genotype G
...K...V......T....                                 AY090454 Genotype H
...................                                 P31868 Jap (adr)
...................                                 P12934 Jap (adr)
...................                                 P03140 Jap (adr)
...................                                 Q67867 Kor (adr)
.......L.....NT....                                 Bozdayi consensus (D)
SGRQPTPISPPLRDTHPQA                                 consensus 1
       L      S                                     consensus 2

 120-169:
[  1- 50]:
         1         2         3         4         5
12345678901234567890123456789012345678901234567890
&lt;---------------- PreS2 (M - Nterm) --------------
MQWNSTAFHQTLQDPRVRGLYLPAGGSSSGTVNPAPNIASHISSISARTG  X02763   Genotype A (adw2)
..........A..........F.........L..V.....L.....ST..  AF297621 Genotype A (african)
......T........G..A..F..........S..Q.TV.A....LSK..  D00330   Genotype B-a (adw)
......T...........A.............S..Q.TV.A....LSK..  AB073858 Genotype B-j (adw)
......T...A.L........F............V.TT..P...TFS...  AB033556 Genotype C (adw)
..........A..........F............V..TV......FTK..  AB048704 Genotype C-aus (ayw)
......T..............F............V.TTV.P....FS.I.  X02496   Genotype D
......T...A..........F............V.TT..L....FS.I.  X75657   Genotype E
......Q...A.L.....A..F.........Q....T...LT...FSK..  X69798   Genotype F (adw4)
..........A..N.K.....F............V.T........FS.I.  AF160501 Genotype G
......Q...A.L........F.......E.Q..V.T...LT...FSK..  AY090454 Genotype H
......T...A.L........F............V.TT..P....FS...  P31868 Jap (adr)
......T...A.L........F............V.TT..P....FS...  P12934 Jap (adr)
......T...A.L........F............V.TT..P....FS...  P03140 Jap (adr)
......T...V.L........F.P..........V.TT..P.....S...  Q67867 Kor (adr)
......T..............F............V.TTV.P....FS.I.  Bozdayi consensus (D)
MQWNSTTFHQALQDPRVRGLYFPAGGSSSGTVNPVPTIASLISSIFSRTG  consensus 1
      A                              T  H      K    consensus 2


 170-174:
[ 51- 55]:
         1         2         3         4         5
12345678901234567890123456789012345678901234567890
PreS2&gt;
DPVTN                                               X02763   Genotype A (adw2)
...P.                                               AF297621 Genotype A (african)
...P.                                               D00330   Genotype B-a (adw)
...P.                                               AB073858 Genotype B-j (adw)f
..AP.                                               AB033556 Genotype C (adw)
..AS.                                               AB048704 Genotype C-aus (ayw)
..AL.                                               X02496   Genotype D
..AP.                                               X75657   Genotype E
G.AM.                                               X69798   Genotype F (adw4)
..AP.                                               AF160501 Genotype G
..AM.                                               AY090454 Genotype H
..AP.                                               P31868 Jap (adr)
..AP.                                               P12934 Jap (adr)
..AP.                                               P03140 Jap (adr)
..AP.                                               Q67867 Kor (adr)
..AL.                                               Bozdayi consensus (D)
DPAPN                                               consensus 1
  VS                                                consensus 2


 175-224:
[  1- 50]:
         1         2         3         4         5
12345678901234567890123456789012345678901234567890
&lt;-------------  	S protein ----------------------
MENITSGFLGPLLVLQAGFFLLTRILTIPQSLDSWWTSLNFLGGSPVCLG  X02763   Genotype A (adw2)
..................................................  AF297621 Genotype A (african)
....A..L...............K....................T.....  D00330   Genotype B-a (adw)
....A..L...............K....................T.....  AB073858 Genotype B-j (adw)
...T........................................A.T.P.  AB033556 Genotype C (adw)
..ST........................................A.G.I.  AB048704 Genotype C-aus (ayw)
............................................TT....  X02496   Genotype D
..S....................K....................A.....  X75657   Genotype E
.D.....L.........VC....K....................L.G.P.  X69798   Genotype F (adw4)
............................................V...P.  AF160501 Genotype G
.......L.........VC....K...................VP.G.P.  AY090454 Genotype H
....................W.......................A.T.P.  Q69598 (ayr)
...T........................................A.T.P.  P31868 Jap (adr)
...T........................................A.T.P.  P12934 Jap (adr)
...T........................................A.T.P.  P03140 Jap (adr)
..ST........................................A.T.P.  Q67867 Jap (adr)
............................................TT....  Bozdayi consensus (D)
MENITSGFLGPLLVLQAGFFLLTRILTIPQSLDSWWTSLNFLGGTPVCLG  consensus 1
       L               K                        P   consensus 2


 225-274:
[ 51-100]:
         1         2         3         4         5
12345678901234567890123456789012345678901234567890
-------------- S protein -------------------------
QNSQSPTSNHSPTSCPPICPGYRWMCLRRFIIFLFILLLCLIFLLVLLDY  X02763   Genotype A (adw2)
..................................................  AF297621 Genotype A (african)
.....QI.S....C....................C...............  D00330   Genotype B-a (adw)
.....QI.S....C...........Y........C...............  AB073858 Genotype B-j (adw)
..................................................  AB033556 Genotype C (adw)
.....Q...........T................................  AB048704 Genotype C-aus (ayw)
.................T................................  X02496   Genotype D
........S.........................................  X75657   Genotype E
..........L......T................................  X69798   Genotype F (adw4)
L...........I....T................................  AF160501 Genotype G
..........L......T................................  AY090454 Genotype H
..L...............................................  Q69598 (ayr)
..................................................  P31868 Jap (adr)
..................................................  P12934 Jap (adr)
..................................................  P03140 Jap (adr)
..................................................  Q67867 Jap (adr)
.................T................................  Bozdayi consensus (D) (ayw)
QNSQSPTSNHSPTSCPPICPGYRWMCLRRFIIFLFILLLCLIFLLVLLDY  consensus 1
                 T                                  consensus 2


 275-324:
[101-150]:
         1         2         3         4         5
12345678901234567890123456789012345678901234567890
-------------- S protein -------------------------
                     *           %        %
             +       +        +
                       @            @ #       #     Cys-Cys disulfides
&lt;----------- determinant a -----------------------
QGMLPVCPLIPGSTTTSTGPCKTCTTPAQGNSMFPSCCCTKPTDGNCTCI  X02763   Genotype A (adw2)
..................................................  AF297621 Genotype A (african)
.............S................T...................  D00330   Genotype B-a (adw)
.............S................T.L.................  AB073858 Genotype B-j (adw)
.........L..TS...........I....T...........S.......  AB033556 Genotype C (adw)
.........L...........R...IT...T...........S.......  AB048704 Genotype C-aus (ayw)
.............S.......R........T..Y........S.......  X02496   Genotype D
.............S.......R....L...T........S..S.......  X75657   Genotype E
.........L................L...T........S..S.......  X69798   Genotype F (adw4)
.............S...................Y........S.......  AF160501 Genotype G
.........L................L...T...........S.......  AY090454 Genotype H
.........L..TS.......R........T...........S.......  Q69598 (ayr)
.........L..TS................T...........S.......  P31868 Jap (adr)
.........L..TS...........I....T...........S.......  P12934 Jap (adr)
.........L..TS...........I....T...........S.......  P03140 Jap (adr)
.........L..TS...........I....T...........S.......  Q67867 Jap (adr)
.............S.......R........T..Y........S.......  Bozdayi consensus (D) (ayw)
QGMLPVCPLIPGSSTTSTGPCKTCTTPAQGTSMFPSCCCTKPSDGNCTCI  consensus 1
             T       R                              consensus 2


 325-374:
[151-200]:
         1         2         3         4         5
12345678901234567890123456789012345678901234567890
-------------- S protein -------------------------
        %#%      %                                    @ W182S or W182C, G185E important
-det a-->                  !   @! @                   ! = P178Q, F183C in adw4
PIPSSWAFAKYLWEWASVRFSWLSLLVPFVQWFVGLSPTVWLSAIWMMWY  X02763   Genotype A (adw2)
..................................................  AF297621 Genotype A (african)
...........................................V...I.F  D00330   Genotype B-a (adw)
...........................................V......  AB073858 Genotype B-j (adw)
..........F......................A.........V......  AB033556 Genotype C (adw)
......G...F......................A.........V...I..  AB048704 Genotype C-aus (ayw)
........G.F......A.........................V......  X02496   Genotype D
........G.F......A...............A.........V......  X75657   Genotype E
.......LG........A.........Q....C.........LV...I..  X69798   Genotype F (adw4)
..................................................  AF160501 Genotype G
........G........A.........Q....C.........LV...I..  AY090454 Genotype H
.........RF......A.........................V......  Q69598 (ayr)
.........RF......................A.........V......  P31868 Jap (adr)
.........RF...G............................V......  P12934 Jap (adr)
.........RF................................V......  P03140 Jap (adr)
.........RF...G............................V......  Q67867 Jap (adr)
........G.F......A.........................V......  Bozdayi consensus (D) (ayw)
PIPSSWAFAKYLWEWASVRFSWLSLLVPFVQWFVGLSPTVWLSVIWMMWY  consensus 1
        GRF      A                                  consensus 2


 375-400:
[201-226]:
         1         2         3         4         5
12345678901234567890123456789012345678901234567890
------- S protein -------&gt;
WGPSLYSIVSPFIPLLPIFFCLWVYI                          X02763   Genotype A (adw2)
......N.L.................                          AF297621 Genotype A (african)
......N.L...M.............                          D00330   Genotype B-a (adw)
......N.L.................                          AB073858 Genotype B-j (adw)
......N.L...L.......Y.....                          AB033556 Genotype C (adw)
......N.L...L......L......                          AB048704 Genotype C-aus (ayw)
........L...L.............                          X02496   Genotype D
......N.L.................                          X75657   Genotype E 
...N.C..L..........CY...S.                          X69798   Genotype F (adw4)
...N..N.L.................                          AF160501 Genotype G
...N.C..L..........CY..AS.                          AY090454 Genotype H
.....HN.LN..L.............                          Q69598 (ayr)
......N.L...L.............                          P31868 Jap (adr)
......N.L...L.............                          P12934 Jap (adr)
......N.L...L.............                          P03140 Jap (adr)
......N.L...L.............                          Q67867 Jap (adr)
........L...L.............                          Bozdayi consensus (D)
WGPSLYNILSPFIPLLPIFFCLWVYI                          consensus 1
   N  S     L       Y                               consensus 2

=============================================================================

CLUSTAL O(1.0.3) multiple sequence alignment

*  all positions have fully conserved (only one) residue
:  conservation between group with strongly similar properties
     score &gt; 0.5 on Gonnet PAM 250 matrxi
.  conservation between group with weakly similar properties
     score &lt; 0.5 on Gonnet PAM 250 matrxi
[blank] position has residues with no similar properties

MENITSGFLGPLLVLQAGFFLLTRILTIPQSLDSWWTSLNFLGGSPVCLGQNSQSPTSNH Q69589 ayw1
MENTTSGFLGPLLVLQAGFFWLTRILTIPQSLDSWWTSLNFLGGAPTCPGQNSQSPTSNH Q69599 adrq+
MENITSGFLGPLLVLQAGFFLLTRILTIPQSLDSWWTSLNFLGGSPVCLGQNSQSPTSNH Q773S4 adw2
MDNITSGLLGPLLVLQAVCFLLTKILTIPQSLDSWWTSLNFLGGLPGCPGQNSQSPTSNH Q69612 adw4q-
MENIASGLLGPLLVLQAGFFLLTKILTIPQSLDSWWTSLNFLGGTPVCLGQNSQSQISSH Q69593 ayw1
MENITSGFLGPLLVLQAGFFWLTRILTIPQSLDSWWTSLNFLGGTTVCLGQNSQSPTSNH Q69601 ayw2
*:* :**:*********  * **:********************   * ******  *.*       
                                                                   
SPTSCPPICPGYRWMCLRRFIIFLFILLLCLIFLLVLLDCQGMLPVCPLIPGSTTTSTGP Q69589 ayw1  
SPTSCPPICPGYRWMCLRRFIIFLFILLLCLIFLLVLLDYQGMLPVCPLLPGTSTTSTGP Q69599 adrq+ 
SPTSCPPICPGYRWMCLRRFIIFLFILLLCLIFLLVLLDYQGMLPVCPLIPGSTTTSTGP Q773S4 adw2  
LPTSCPPTCPGYRWMCLRRFIIFLFILLLCLIFLLVLLDYQGMLPVCPLIPGSTTTSTGP Q69612 adw4q-
SPTCCPPICPGYRWMCLRRFIIFLCILLLCLIFLLVLLDYQGMLPVCPLIPGSSTTSTGP Q69593 ayw1  
SPTSCPPTCPGYRWMCLRRFIIFLFILLLCLIFLLVLLDYQGMLPVCPLIPGSSTTSTGP Q69601 ayw2  
 **.*** **************** ************** *********:**::******       
                                                                   
CRTCTTPAQGNSMFPSCCCTKPTDGNCTCIPIPSSWAFAKYLWEWASVRFSWLSLLVPFV Q69589 ayw1  
CKTCTIPAQGTSMFPSCCCTKPSDGNCTCIPIPSSWAFARFLWEWASARFSWLSLLVPFV Q69599 adrq+ 
CKTCTTPAQGNSMFPSCCCTKPTDGNCTCIPIPSSWAFAKYLWEWASVRFSWLSLLVPFV Q773S4 adw2  
CKTCTTLAQGTSMFPSCCCSKPSDGNCTCIPIPSSWALGKYLWEWASARFSWLSLLVQFV Q69612 adw4q-
CRTCTTPAQGTSMFPSCCCTKPTDGNCTCIPIPSSWAFAKYLWEWASVRFSWLSLLVPFV Q69593 ayw1  
CRTCTTPAQGTSMYPSCCCTKPSDGNCTCIPIPSSWAFGKFLWEWASARFSWLSLLVPFV Q69601 ayw2  
*:***  ***.**:*****:**:**************:.::******.********* **       
                                                                   
QWFVGLSPTVWLSVIWMMWYWGPSLYNILSPFIPLLPIFFCLWVYI               Q69589 ayw1  
QWFAGLSPTVWLSVIWMMWYWGPSLYNILSPFLPLLPIFFCLWVYI               Q69599 adrq+ 
QWFVGLSPTVWLSAIWMMWYWGPSLYSIVSPFIPLLPIFFCLWVYI               Q773S4 adw2  
QWCVGLSPTVWLLVIWMIWYWGPNLCSILSPFIPLLPIFCCLWVSI               Q69612 adw4q-
QWFVGLSPTVWLSVIWMIWYWGPSLYNILSPFMPLLPIFFCLWVYI               Q69593 ayw1  
QWFVGLSPTVWLSVIWMMWYWGPSLYSILSPFLPLLPIFFCLWVYI               Q69601 ayw2  
** .******** .***:*****.* .*:***:****** **** *                                     

</pre>
<p>
Four <i>adr</i> isolates of the L (PreS1-PreS2-S) sequence with accession
numbers 
<a href="http://www.uniprot.org/uniprot/P31868">P31868</a>
<a href="http://www.uniprot.org/uniprot/P12934">P12934</a>
<a href="http://www.uniprot.org/uniprot/P03140">P03140</a>, and
<a href="http://www.uniprot.org/uniprot/Q67867">Q67867</a> have been entered.
The question first is what differences in sequence exist that rule
out amino acids that account for the epitopes.  In the N-terminal part that
creates the L sequence, amino acid position 51 is quite variable (H, D, Q,
and R); pos. 60 can be V or A.  In the N-terminal part of M, there is no
real heterogeneity of sequence.  In the S core sequence, position 165 clearly
can be G or W.  These isolates strongly represent Genotype C reference sequence.
<p>
Consider Genotype D, which is <i>ayw</i>.  How does it radically differ from
<i>adr</i>?  Examining only the S core sequence alone, postion 4 is I in every
genotype but is T in the <i>adr</i>.   Between position 45-49, Genotype D (ayw)
is TTVCL but for Genotype C it is APTCP (only C is present at the same position).
In the determinat <i>a</i> region between 101-159, position 122 is R in <i>ayw</i>
but K in <i>adr</i>.  Between 110-113, the sequence is IPGS in <i>ayw</i>, but
LPGT in <i>adr</i>.  There is a F134Y from <i>ayw</i> to <i>adr</i>.  The two
amino acids from 159-160 are GK in <i>ayw</i>, and AR in <i>adr</i>.  Position
165 is quite variable in <i>adr</i> as noted:  G or W, but in <i>ayw</i> it is
W.  At position 168, <i>ayw</i> is A, but <i>adr</i> is clearly V.  Position
207 is clearly N in <i>adr</i> but S in <i>ayw</i>.  There is significant decrease
in reactivity to 4-7B if W182S or W182C and G185E changes occur.


<h2>Nucleotide Sequences</h2>
<p>
Below is a record of all nucleotide sequences to be used in the project, either
to be synthesized or as part of the process of plasmid construction for 
recombinants.

<h3 id="reverse-translate">Reverse Translator-Generated Sequences for S, M, and L Proteins</h3>
<p>
Oligonucleotides are to be chemically synthesized that will produce coding 
(translated) sequence of the S, M, and L proteins that are the three forms of 
the HBsAg.  First the S coding sequence will be entirely chemically synthesized. 
The S coding sequence used here is not known to be a <i>naturally occurring</i> 
sequence in any of the various the eight genotypes of HBV, but it is instead a 
<i>consensus</i> sequence created by the multiple alignment analysis of the 
reference amino acids of the S coding sequences considered in one paper to be 
the genotypes of HBV.  Why does this make sense?  Because if there are multiple 
sequences available, the alignment would probably show an idealized structure 
that represents the evolution of the virus with respect to the protein.  It is
hope that the consensus sequence product is broadly reactive to/against all genotypes,
although it is likely that we may need to create recombinants coding exactly for 
naturally occurring S forms in particular genotypes of interest.
<p>
After the consensus sequence was determined, several <q>reverse translations</q> 
sequences for the SHBs, MHBs, and LHBs products of HBV were generated using the 
<a href="http://www.bioinformatics.org/sms2/rev_trans.html">Reverse 
Translate</a> online tool.  The several sequences were necessary because 
expression will be tested in different host organisms.  Codon usage tables 
provided through a link on the Reverse Translate tool were selected in order to 
perform the reverse translation.  All of the amino acid sequences used for 
reverse translation of the S, M, and L proteins are determined as a consenus 
based on the multiple alignment analysis of sequences considered by <a 
href="../PublicationOzetlemeler.html#bartholomeusz">Bartholomeusz &amp; 
Schaefer</a> to be reference sequences for the eight HBV genotypes.

<table class="td-center th-center" style="font-size:75%;float:right;width:40%;margin-left:1.5em;">
<tr><th rowspan="2">Amino Acid
  <th rowspan="2">Codon  <th colspan="2">Usage
<tr>                                      <th><i>E coli</i>  <th><i>B subtilis</i>
<tr><td rowspan="6" class="left">Arg (R)    <td>AGG    <td>7%     <td>10%
<tr>                           <td>AGA    <td>13%    <td>25%*
<tr>                           <td>CGG    <td>15%    <td>17%
<tr>                           <td>CGA    <td>9%     <td>10%
<tr>                           <td>CGT    <td>30%    <td>18%
<tr>                           <td>CGC    <td>26%*   <td>20%
<tr><td rowspan="2" class="left">Asn (N)    <td>AAT    <td>59%    <td>56%*
<tr>                           <td>AAC    <td>41%*   <td>44%
<tr><td rowspan="2" class="left">Gln (Q)    <td>CAG    <td>65%*   <td>48%
<tr>                           <td>CAA    <td>35%    <td>52%*
<tr><td rowspan="4" class="left">Thr (T)    <td>ACG    <td>22%    <td>27%
<tr>                           <td>ACA    <td>25%    <td>40%*
<tr>                           <td>ACT    <td>22%    <td>16%
<tr>                           <td>ACC    <td>31%*   <td>17%
<tr><td rowspan="6" class="left">Ser (S)    <td>AGT    <td>18%    <td>11%
<tr>                           <td>AGC    <td>20%*   <td>23%
<tr>                           <td>TCG    <td>11%    <td>10%
<tr>                           <td>TCA    <td>18%    <td>23%*
<tr>                           <td>TCT    <td>18%    <td>20%
<tr>                           <td>TCC    <td>14%    <td>13%
<tr><td rowspan="4" class="left">Val (V)    <td>GTG    <td>29%*   <td>26%
<tr>                           <td>GTA    <td>19%    <td>20%
<tr>                           <td>GTT    <td>32%    <td>28%*
<tr>                           <td>GTC    <td>19%    <td>26%
<tr><td colspan="4"
 style="font:normal 83% 'Times New Roman','Times Roman',Times,serif;">
* indicates the codon that was preferred by the reverse translator
</table>
<p>
To minimize the work necessary for this first stage of bacterial expression in 
<i>E coli</i> and <i>B subtilis</i>, differences in codon usage between these
two species were resolved so that the best single codon for both <i>E coli</i>
and <i>B subtilis</i> were selected, rather than trying to create sequences optimized
for each, which would require PCR-mediated site-directed mutagenesis and considerable
work in sequencing.  Comparing the <i>E coli</i> and <i>B subtilis</i> sequences,
the following amino acids and their preferred and alternate codons were determined from
usage tables for <a href="http://www.kazusa.or.jp/codon/cgi-bin/showcodon.cgi?species=37762&amp;aa=1&amp;style=GCG"><i>E coli</i></a>
and <a href="http://www.kazusa.or.jp/codon/cgi-bin/showcodon.cgi?species=1423&amp;aa=11&amp;style=GCG">B subtilis</a>

<table style="font-size:75%;float:left;margin-right:1.5em;width:10em;">
<tr><th>Amino Acid  <th>Selected Codon
<tr><td>Arg (R)    <td>CGC
<tr><td>Asn (N)    <td>AAT
<tr><td>Gln (Q)    <td>CAG
<tr><td>Thr (T)    <td>ACA
<tr><td>Ser (S)    <td>AGC
<tr><td>Val (V)    <td>GTT
</table>

<p>
The logic of choosing the final codons for bacteria will not be detailed. Rather,
the codons to be chosen are listed in the table at the left.
<p>
The final editions of the S, M, and L sequences for bacterial expression in 
<i>E coli</i> and <i>B subtilis</i> are given below

<pre style="font-size:85%;clear:both;">


Consensus SHBs protein,  681 nt,  226 aa

         1         2         3         4         5         6
123456789012345678901234567890123456789012345678901234567890

Codon usage for E coli
ATGGAAAACATTACCAGCGGCTTTCTGGGCCCGCTGCTGGTGCTGCAGGCGGGCTTTTTT
CTGCTGACCCGCATTCTGACCATTCCGCAGAGCCTGGATAGCTGGTGGACCAGCCTGAAC
TTTCTGGGCGGCACCCCGGTGTGCCTGGGCCAGAACAGCCAGAGCCCGACCAGCAACCAT
AGCCCGACCAGCTGCCCGCCGATTTGCCCGGGCTATCGCTGGATGTGCCTGCGCCGCTTT
ATTATTTTTCTGTTTATTCTGCTGCTGTGCCTGATTTTTCTGCTGGTGCTGCTGGATTAT
CAGGGCATGCTGCCGGTGTGCCCGCTGATTCCGGGCAGCAGCACCACCAGCACCGGCCCG
TGCAAAACCTGCACCACCCCGGCGCAGGGCACCAGCATGTTTCCGAGCTGCTGCTGCACC
AAACCGAGCGATGGCAACTGCACCTGCATTCCGATTCCGAGCAGCTGGGCGTTTGCGAAA
TATCTGTGGGAATGGGCGAGCGTGCGCTTTAGCTGGCTGAGCCTGCTGGTGCCGTTTGTG
CAGTGGTTTGTGGGCCTGAGCCCGACCGTGTGGCTGAGCGTGATTTGGATGATGTGGTAT
TGGGGCCCGAGCCTGTATAACATTCTGAGCCCGTTTATTCCGCTGCTGCCGATTTTTTTT
TGCCTGTGGGTGTATATTTAA

Codon usage for B subtilis
ATGGAAAATATTACATCAGGCTTTCTGGGCCCGCTGCTGGTTCTGCAAGCAGGCTTTTTT
CTGCTGACAAGAATTCTGACAATTCCGCAATCACTGGATTCATGGTGGACATCACTGAAT
TTTCTGGGCGGCACACCGGTTTGCCTGGGCCAAAATTCACAATCACCGACATCAAATCAT
TCACCGACATCATGCCCGCCGATTTGCCCGGGCTATAGATGGATGTGCCTGAGAAGATTT
ATTATTTTTCTGTTTATTCTGCTGCTGTGCCTGATTTTTCTGCTGGTTCTGCTGGATTAT
CAAGGCATGCTGCCGGTTTGCCCGCTGATTCCGGGCTCATCAACAACATCAACAGGCCCG
TGCAAAACATGCACAACACCGGCACAAGGCACATCAATGTTTCCGTCATGCTGCTGCACA
AAACCGTCAGATGGCAATTGCACATGCATTCCGATTCCGTCATCATGGGCATTTGCAAAA
TATCTGTGGGAATGGGCATCAGTTAGATTTTCATGGCTGTCACTGCTGGTTCCGTTTGTT
CAATGGTTTGTTGGCCTGTCACCGACAGTTTGGCTGTCAGTTATTTGGATGATGTGGTAT
TGGGGCCCGTCACTGTATAATATTCTGTCACCGTTTATTCCGCTGCTGCCGATTTTTTTT
TGCCTGTGGGTTTATATTTAA

Codon usage for K lactis
ATGGAAAATATTACTTCTGGTTTCTTGGGTCCATTGTTGGTTTTGCAAGCTGGTTTCTTC
TTGTTGACTAGAATTTTGACTATTCCACAATCTTTGGATTCTTGGTGGACTTCTTTGAAT
TTCTTGGGTGGTACTCCAGTTTGTTTGGGTCAAAATTCTCAATCTCCAACTTCTAATCAT
TCTCCAACTTCTTGTCCACCAATTTGTCCAGGTTATAGATGGATGTGTTTGAGAAGATTC
ATTATTTTCTTGTTCATTTTGTTGTTGTGTTTGATTTTCTTGTTGGTTTTGTTGGATTAT
CAAGGTATGTTGCCAGTTTGTCCATTGATTCCAGGTTCTTCTACTACTTCTACTGGTCCA
TGTAAAACTTGTACTACTCCAGCTCAAGGTACTTCTATGTTCCCATCTTGTTGTTGTACT
AAACCATCTGATGGTAATTGTACTTGTATTCCAATTCCATCTTCTTGGGCTTTCGCTAAA
TATTTGTGGGAATGGGCTTCTGTTAGATTCTCTTGGTTGTCTTTGTTGGTTCCATTCGTT
CAATGGTTCGTTGGTTTGTCTCCAACTGTTTGGTTGTCTGTTATTTGGATGATGTGGTAT
TGGGGTCCATCTTTGTATAATATTTTGTCTCCATTCATTCCATTGTTGCCAATTTTCTTC
TGTTTGTGGGTTTATATTTAA

Codon usage for P pastoris
ATGGAAAACATTACTTCTGGTTTTTTGGGTCCATTGTTGGTTTTGCAAGCTGGTTTTTTT
TTGTTGACTAGAATTTTGACTATTCCACAATCTTTGGATTCTTGGTGGACTTCTTTGAAC
TTTTTGGGTGGTACTCCAGTTTGTTTGGGTCAAAACTCTCAATCTCCAACTTCTAACCAT
TCTCCAACTTCTTGTCCACCAATTTGTCCAGGTTACAGATGGATGTGTTTGAGAAGATTT
ATTATTTTTTTGTTTATTTTGTTGTTGTGTTTGATTTTTTTGTTGGTTTTGTTGGATTAC
CAAGGTATGTTGCCAGTTTGTCCATTGATTCCAGGTTCTTCTACTACTTCTACTGGTCCA
TGTAAGACTTGTACTACTCCAGCTCAAGGTACTTCTATGTTTCCATCTTGTTGTTGTACT
AAGCCATCTGATGGTAACTGTACTTGTATTCCAATTCCATCTTCTTGGGCTTTTGCTAAG
TACTTGTGGGAATGGGCTTCTGTTAGATTTTCTTGGTTGTCTTTGTTGGTTCCATTTGTT
CAATGGTTTGTTGGTTTGTCTCCAACTGTTTGGTTGTCTGTTATTTGGATGATGTGGTAC
TGGGGTCCATCTTTGTACAACATTTTGTCTCCATTTATTCCATTGTTGCCAATTTTTTTT
TGTTTGTGGGTTTACATTTAA

=================================================================

Consensus MHBs protein,  846 nt,  281 aa

         1         2         3         4         5         6
123456789012345678901234567890123456789012345678901234567890

Codon usage for E coli
ATGCAGTGGAACAGCACCACCTTTCATCAGGCGCTGCAGGATCCGCGCGTGCGCGGCCTG
TATTTTCCGGCGGGCGGCAGCAGCAGCGGCACCGTGAACCCGGTGCCGACCATTGCGAGC
CTGATTAGCAGCATTTTTAGCCGCACCGGCGATCCGGCGCCGAACATGGAAAACATTACC
AGCGGCTTTCTGGGCCCGCTGCTGGTGCTGCAGGCGGGCTTTTTTCTGCTGACCCGCATT
CTGACCATTCCGCAGAGCCTGGATAGCTGGTGGACCAGCCTGAACTTTCTGGGCGGCACC
CCGGTGTGCCTGGGCCAGAACAGCCAGAGCCCGACCAGCAACCATAGCCCGACCAGCTGC
CCGCCGATTTGCCCGGGCTATCGCTGGATGTGCCTGCGCCGCTTTATTATTTTTCTGTTT
ATTCTGCTGCTGTGCCTGATTTTTCTGCTGGTGCTGCTGGATTATCAGGGCATGCTGCCG
GTGTGCCCGCTGATTCCGGGCAGCAGCACCACCAGCACCGGCCCGTGCAAAACCTGCACC
ACCCCGGCGCAGGGCACCAGCATGTTTCCGAGCTGCTGCTGCACCAAACCGAGCGATGGC
AACTGCACCTGCATTCCGATTCCGAGCAGCTGGGCGTTTGCGAAATATCTGTGGGAATGG
GCGAGCGTGCGCTTTAGCTGGCTGAGCCTGCTGGTGCCGTTTGTGCAGTGGTTTGTGGGC
CTGAGCCCGACCGTGTGGCTGAGCGTGATTTGGATGATGTGGTATTGGGGCCCGAGCCTG
TATAACATTCTGAGCCCGTTTATTCCGCTGCTGCCGATTTTTTTTTGCCTGTGGGTGTAT
ATTTAA


Codon usage for K lactis
ATGCAATGGAATTCTACTACTTTCCATCAAGCTTTGCAAGATCCAAGAGTTAGAGGTTTG
TATTTCCCAGCTGGTGGTTCTTCTTCTGGTACTGTTAATCCAGTTCCAACTATTGCTTCT
TTGATTTCTTCTATTTTCTCTAGAACTGGTGATCCAGCTCCAAATATGGAAAATATTACT
TCTGGTTTCTTGGGTCCATTGTTGGTTTTGCAAGCTGGTTTCTTCTTGTTGACTAGAATT
TTGACTATTCCACAATCTTTGGATTCTTGGTGGACTTCTTTGAATTTCTTGGGTGGTACT
CCAGTTTGTTTGGGTCAAAATTCTCAATCTCCAACTTCTAATCATTCTCCAACTTCTTGT
CCACCAATTTGTCCAGGTTATAGATGGATGTGTTTGAGAAGATTCATTATTTTCTTGTTC
ATTTTGTTGTTGTGTTTGATTTTCTTGTTGGTTTTGTTGGATTATCAAGGTATGTTGCCA
GTTTGTCCATTGATTCCAGGTTCTTCTACTACTTCTACTGGTCCATGTAAAACTTGTACT
ACTCCAGCTCAAGGTACTTCTATGTTCCCATCTTGTTGTTGTACTAAACCATCTGATGGT
AATTGTACTTGTATTCCAATTCCATCTTCTTGGGCTTTCGCTAAATATTTGTGGGAATGG
GCTTCTGTTAGATTCTCTTGGTTGTCTTTGTTGGTTCCATTCGTTCAATGGTTCGTTGGT
TTGTCTCCAACTGTTTGGTTGTCTGTTATTTGGATGATGTGGTATTGGGGTCCATCTTTG
TATAATATTTTGTCTCCATTCATTCCATTGTTGCCAATTTTCTTCTGTTTGTGGGTTTAT
ATTTAA


Codon usage for P pastoris
ATGCAATGGAACTCTACTACTTTTCATCAAGCTTTGCAAGATCCAAGAGTTAGAGGTTTG
TACTTTCCAGCTGGTGGTTCTTCTTCTGGTACTGTTAACCCAGTTCCAACTATTGCTTCT
TTGATTTCTTCTATTTTTTCTAGAACTGGTGATCCAGCTCCAAACATGGAAAACATTACT
TCTGGTTTTTTGGGTCCATTGTTGGTTTTGCAAGCTGGTTTTTTTTTGTTGACTAGAATT
TTGACTATTCCACAATCTTTGGATTCTTGGTGGACTTCTTTGAACTTTTTGGGTGGTACT
CCAGTTTGTTTGGGTCAAAACTCTCAATCTCCAACTTCTAACCATTCTCCAACTTCTTGT
CCACCAATTTGTCCAGGTTACAGATGGATGTGTTTGAGAAGATTTATTATTTTTTTGTTT
ATTTTGTTGTTGTGTTTGATTTTTTTGTTGGTTTTGTTGGATTACCAAGGTATGTTGCCA
GTTTGTCCATTGATTCCAGGTTCTTCTACTACTTCTACTGGTCCATGTAAGACTTGTACT
ACTCCAGCTCAAGGTACTTCTATGTTTCCATCTTGTTGTTGTACTAAGCCATCTGATGGT
AACTGTACTTGTATTCCAATTCCATCTTCTTGGGCTTTTGCTAAGTACTTGTGGGAATGG
GCTTCTGTTAGATTTTCTTGGTTGTCTTTGTTGGTTCCATTTGTTCAATGGTTTGTTGGT
TTGTCTCCAACTGTTTGGTTGTCTGTTATTTGGATGATGTGGTACTGGGGTCCATCTTTG
TACAACATTTTGTCTCCATTTATTCCATTGTTGCCAATTTTTTTTTGTTTGTGGGTTTAC
ATTTAA


Consensus LHBs protein,  1203 nt,  400 aa

         1         2         3         4         5         6
123456789012345678901234567890123456789012345678901234567890

Codon usage for E coli
ATGGGCGGCTGGAGCAGCAAACCGCGCAAAGGCATGGGCACCAACCTGAGCGTGCCGAAC
CCGCTGGGCTTTTTTCCGGATCATCAGCTGGATCCGGCGTTTCGCGCGAACAGCAACAAC
CCGGATTGGGATTTTAACCCGAACAAAGATCATTGGCCGGATGCGAACAAAGTGGGCGTG
GGCGCGTTTGGCCCGGGCTTTACCCCGCCGCATGGCGGCCTGCTGGGCTGGAGCCCGCAG
GCGCAGGGCATTCTGACCACCCTGCCGGCGGATCCGCCGCCGGCGAGCACCAACCGCCAG
AGCGGCCGCCAGCCGACCCCGATTAGCCCGCCGCTGCGCGATACCCATCCGCAGGCGATG
CAGTGGAACAGCACCACCTTTCATCAGGCGCTGCAGGATCCGCGCGTGCGCGGCCTGTAT
TTTCCGGCGGGCGGCAGCAGCAGCGGCACCGTGAACCCGGTGCCGACCATTGCGAGCCTG
ATTAGCAGCATTTTTAGCCGCACCGGCGATCCGGCGCCGAACATGGAAAACATTACCAGC
GGCTTTCTGGGCCCGCTGCTGGTGCTGCAGGCGGGCTTTTTTCTGCTGACCCGCATTCTG
ACCATTCCGCAGAGCCTGGATAGCTGGTGGACCAGCCTGAACTTTCTGGGCGGCACCCCG
GTGTGCCTGGGCCAGAACAGCCAGAGCCCGACCAGCAACCATAGCCCGACCAGCTGCCCG
CCGATTTGCCCGGGCTATCGCTGGATGTGCCTGCGCCGCTTTATTATTTTTCTGTTTATT
CTGCTGCTGTGCCTGATTTTTCTGCTGGTGCTGCTGGATTATCAGGGCATGCTGCCGGTG
TGCCCGCTGATTCCGGGCAGCAGCACCACCAGCACCGGCCCGTGCAAAACCTGCACCACC
CCGGCGCAGGGCACCAGCATGTTTCCGAGCTGCTGCTGCACCAAACCGAGCGATGGCAAC
TGCACCTGCATTCCGATTCCGAGCAGCTGGGCGTTTGCGAAATATCTGTGGGAATGGGCG
AGCGTGCGCTTTAGCTGGCTGAGCCTGCTGGTGCCGTTTGTGCAGTGGTTTGTGGGCCTG
AGCCCGACCGTGTGGCTGAGCGTGATTTGGATGATGTGGTATTGGGGCCCGAGCCTGTAT
AACATTCTGAGCCCGTTTATTCCGCTGCTGCCGATTTTTTTTTGCCTGTGGGTGTATATT
TAA


Codon usage for K lactis
ATGGGTGGTTGGTCTTCTAAACCAAGAAAAGGTATGGGTACTAATTTGTCTGTTCCAAAT
CCATTGGGTTTCTTCCCAGATCATCAATTGGATCCAGCTTTCAGAGCTAATTCTAATAAT
CCAGATTGGGATTTCAATCCAAATAAAGATCATTGGCCAGATGCTAATAAAGTTGGTGTT
GGTGCTTTCGGTCCAGGTTTCACTCCACCACATGGTGGTTTGTTGGGTTGGTCTCCACAA
GCTCAAGGTATTTTGACTACTTTGCCAGCTGATCCACCACCAGCTTCTACTAATAGACAA
TCTGGTAGACAACCAACTCCAATTTCTCCACCATTGAGAGATACTCATCCACAAGCTATG
CAATGGAATTCTACTACTTTCCATCAAGCTTTGCAAGATCCAAGAGTTAGAGGTTTGTAT
TTCCCAGCTGGTGGTTCTTCTTCTGGTACTGTTAATCCAGTTCCAACTATTGCTTCTTTG
ATTTCTTCTATTTTCTCTAGAACTGGTGATCCAGCTCCAAATATGGAAAATATTACTTCT
GGTTTCTTGGGTCCATTGTTGGTTTTGCAAGCTGGTTTCTTCTTGTTGACTAGAATTTTG
ACTATTCCACAATCTTTGGATTCTTGGTGGACTTCTTTGAATTTCTTGGGTGGTACTCCA
GTTTGTTTGGGTCAAAATTCTCAATCTCCAACTTCTAATCATTCTCCAACTTCTTGTCCA
CCAATTTGTCCAGGTTATAGATGGATGTGTTTGAGAAGATTCATTATTTTCTTGTTCATT
TTGTTGTTGTGTTTGATTTTCTTGTTGGTTTTGTTGGATTATCAAGGTATGTTGCCAGTT
TGTCCATTGATTCCAGGTTCTTCTACTACTTCTACTGGTCCATGTAAAACTTGTACTACT
CCAGCTCAAGGTACTTCTATGTTCCCATCTTGTTGTTGTACTAAACCATCTGATGGTAAT
TGTACTTGTATTCCAATTCCATCTTCTTGGGCTTTCGCTAAATATTTGTGGGAATGGGCT
TCTGTTAGATTCTCTTGGTTGTCTTTGTTGGTTCCATTCGTTCAATGGTTCGTTGGTTTG
TCTCCAACTGTTTGGTTGTCTGTTATTTGGATGATGTGGTATTGGGGTCCATCTTTGTAT
AATATTTTGTCTCCATTCATTCCATTGTTGCCAATTTTCTTCTGTTTGTGGGTTTATATT
TAA


Codon usage for P pastoris
ATGGGTGGTTGGTCTTCTAAGCCAAGAAAGGGTATGGGTACTAACTTGTCTGTTCCAAAC
CCATTGGGTTTTTTTCCAGATCATCAATTGGATCCAGCTTTTAGAGCTAACTCTAACAAC
CCAGATTGGGATTTTAACCCAAACAAGGATCATTGGCCAGATGCTAACAAGGTTGGTGTT
GGTGCTTTTGGTCCAGGTTTTACTCCACCACATGGTGGTTTGTTGGGTTGGTCTCCACAA
GCTCAAGGTATTTTGACTACTTTGCCAGCTGATCCACCACCAGCTTCTACTAACAGACAA
TCTGGTAGACAACCAACTCCAATTTCTCCACCATTGAGAGATACTCATCCACAAGCTATG
CAATGGAACTCTACTACTTTTCATCAAGCTTTGCAAGATCCAAGAGTTAGAGGTTTGTAC
TTTCCAGCTGGTGGTTCTTCTTCTGGTACTGTTAACCCAGTTCCAACTATTGCTTCTTTG
ATTTCTTCTATTTTTTCTAGAACTGGTGATCCAGCTCCAAACATGGAAAACATTACTTCT
GGTTTTTTGGGTCCATTGTTGGTTTTGCAAGCTGGTTTTTTTTTGTTGACTAGAATTTTG
ACTATTCCACAATCTTTGGATTCTTGGTGGACTTCTTTGAACTTTTTGGGTGGTACTCCA
GTTTGTTTGGGTCAAAACTCTCAATCTCCAACTTCTAACCATTCTCCAACTTCTTGTCCA
CCAATTTGTCCAGGTTACAGATGGATGTGTTTGAGAAGATTTATTATTTTTTTGTTTATT
TTGTTGTTGTGTTTGATTTTTTTGTTGGTTTTGTTGGATTACCAAGGTATGTTGCCAGTT
TGTCCATTGATTCCAGGTTCTTCTACTACTTCTACTGGTCCATGTAAGACTTGTACTACT
CCAGCTCAAGGTACTTCTATGTTTCCATCTTGTTGTTGTACTAAGCCATCTGATGGTAAC
TGTACTTGTATTCCAATTCCATCTTCTTGGGCTTTTGCTAAGTACTTGTGGGAATGGGCT
TCTGTTAGATTTTCTTGGTTGTCTTTGTTGGTTCCATTTGTTCAATGGTTTGTTGGTTTG
TCTCCAACTGTTTGGTTGTCTGTTATTTGGATGATGTGGTACTGGGGTCCATCTTTGTAC
AACATTTTGTCTCCATTTATTCCATTGTTGCCAATTTTTTTTTGTTTGTGGGTTTACATT
TAA
</pre>

<h3>Final Edition Sequences For Cloning</h3>
<p>
The following sequences for SHBs, MHBs, and LHBs forms of HBsAg are the final
for use in expression for bacterial hosts <i>E coli</i> and <i>B subtilis</i>.
The HBsAg sequences are the consensus sequences based on multiple alignment
of the reference sequences for the HBV genotypes.

<pre style="font-size:75%;">

Nucleotides for cloning of SHBs in E coli, B subtilis

         1         2         3         4         5         6
123456789012345678901234567890123456789012345678901234567890

ATGGAAAATATTACAAGCGGCTTTCTGGGCCCGCTGCTGGTTCTGCAGGCGGGCTTTTTT
 M  E  N  I  T  S  G  F  L  G  P  L  L  V  L  Q  A  G  F  F
CTGCTGACACGCATTCTGACAATTCCGCAGAGCCTGGATAGCTGGTGGACAAGCCTGAAT
 L  L  T  R  I  L  T  I  P  Q  S  L  D  S  W  W  T  S  L  N
TTTCTGGGCGGCACACCGGTTTGCCTGGGCCAGAATAGCCAGAGCCCGACAAGCAATCAT
 F  L  G  G  T  P  V  C  L  G  Q  N  S  Q  S  P  T  S  N  H
AGCCCGACAAGCTGCCCGCCGATTTGCCCGGGCTATCGCTGGATGTGCCTGCGCCGCTTT
 S  P  T  S  C  P  P  I  C  P  G  Y  R  W  M  C  L  R  R  F
ATTATTTTTCTGTTTATTCTGCTGCTGTGCCTGATTTTTCTGCTGGTTCTGCTGGATTAT
 I  I  F  L  F  I  L  L  L  C  L  I  F  L  L  V  L  L  D  Y
CAGGGCATGCTGCCGGTTTGCCCGCTGATTCCGGGCAGCAGCACAACAAGCACAGGCCCG
 Q  G  M  L  P  V  C  P  L  I  P  G  S  S  T  T  S  T  G  P
TGCAAAACATGCACAACACCGGCGCAGGGCACAAGCATGTTTCCGAGCTGCTGCTGCACA
 C  K  T  C  T  T  P  A  Q  G  T  S  M  F  P  S  C  C  C  T
AAACCGAGCGATGGCAATTGCACATGCATTCCGATTCCGAGCAGCTGGGCGTTTGCGAAA
 K  P  S  D  G  N  C  T  C  I  P  I  P  S  S  W  A  F  A  K
TATCTGTGGGAATGGGCGAGCGTTCGCTTTAGCTGGCTGAGCCTGCTGGTTCCGTTTGTT
 Y  L  W  E  W  A  S  V  R  F  S  W  L  S  L  L  V  P  F  V
CAGTGGTTTGTTGGCCTGAGCCCGACAGTTTGGCTGAGCGTTATTTGGATGATGTGGTAT
 Q  W  F  V  G  L  S  P  T  V  W  L  S  V  I  W  M  M  W  Y
TGGGGCCCGAGCCTGTATAATATTCTGAGCCCGTTTATTCCGCTGCTGCCGATTTTTTTT
 W  G  P  S  L  Y  N  I  L  S  P  F  I  P  L  L  P  I  F  F
TGCCTGTGGGTTTATATTTAA
 C  L  W  V  Y  I  *

MW = 420690 g/mol, GC = 54%
1.0 AU 260 nm = 0.115 &micro;M

Nucleotides for cloning of PreS2 of MHBs in E coli, B subtilis

         1         2         3         4         5         6
123456789012345678901234567890123456789012345678901234567890

ATGCAGTGGAATAGCACAACATTTCATCAGGCGCTGCAGGATCCGCGCGTTCGCGGCCTG
 M  Q  W  N  S  T  T  F  H  Q  A  L  Q  D  P  R  V  R  G  L
TATTTTCCGGCGGGCGGCAGCAGCAGCGGCACAGTTAATCCGGTTCCGACAATTGCGAGC
 Y  F  P  A  G  G  S  S  S  G  T  V  N  P  V  P  T  I  A  S
CTGATTAGCAGCATTTTTAGCCGCACAGGCGATCCGGCGCCGAAT
 L  I  S  S  I  F  S  R  T  G  D  P  A  P  N

MW = 101843 g/mol, GC = 58%
1.0 AU 260 nm = 0.473 &micro;M

Nucleotides for cloning of PreS1 of LHBs in E coli, B subtilis

         1         2         3         4         5         6
123456789012345678901234567890123456789012345678901234567890

ATGGGCGGCTGGAGCAGCAAACCGCGCAAAGGCATGGGCACAAATCTGAGCGTTCCGAAC
 M  G  G  W  S  S  K  P  R  K  G  M  G  T  N  L  S  V  P  N
CCGCTGGGCTTTTTTCCGGATCATCAGCTGGATCCGGCGTTTCGCGCGAATAGCAACAAT
 P  L  G  F  F  P  D  H  Q  L  D  P  A  F  R  A  N  S  N  N
CCGGATTGGGATTTTAACCCGAATAAAGATCATTGGCCGGATGCGAATAAAGTTGGCGTT
 P  D  W  D  F  N  P  N  K  D  H  W  P  D  A  N  K  V  G  V
GGCGCGTTTGGCCCGGGCTTTACACCGCCGCATGGCGGCCTGCTGGGCTGGAGCCCGCAG
 G  A  F  G  P  G  F  T  P  P  H  G  G  L  L  G  W  S  P  Q
GCGCAGGGCATTCTGACAACACTGCCGGCGGATCCGCCGCCGGCGAGCACAAATCGCCAG
 A  Q  G  I  L  T  T  L  P  A  D  P  P  P  A  S  T  N  R  Q
AGCGGCCGCCAGCCGACACCGATTAGCCCGCCGCTGCGCGATACACATCCGCAGGCG
 S  G  R  Q  P  T  P  I  S  P  P  L  R  D  T  H  P  Q  A

MW = 220510 g/mol, GC = 62%
1.0 AU 260 nm = 0.219 &micro;M

</pre>

<!--
ATGGGCGGCTGGAGCAGCAAACCGCGCAAAGGCATGGGCACAAATCTGAGCGTTCCGAAC
CCGCTGGGCTTTTTTCCGGATCATCAGCTGGATCCGGCGTTTCGCGCGAATAGCAACAAT
CCGGATTGGGATTTTAACCCGAATAAAGATCATTGGCCGGATGCGAATAAAGTTGGCGTT
GGCGCGTTTGGCCCGGGCTTTACACCGCCGCATGGCGGCCTGCTGGGCTGGAGCCCGCAG
GCGCAGGGCATTCTGACAACACTGCCGGCGGATCCGCCGCCGGCGAGCACAAATCGCCAG
AGCGGCCGCCAGCCGACACCGATTAGCCCGCCGCTGCGCGATACACATCCGCAGGCGATG
CAGTGGAATAGCACAACATTTCATCAGGCGCTGCAGGATCCGCGCGTTCGCGGCCTGTAT
TTTCCGGCGGGCGGCAGCAGCAGCGGCACAGTTAATCCGGTTCCGACAATTGCGAGCCTG
ATTAGCAGCATTTTTAGCCGCACAGGCGATCCGGCGCCGAATATGGAAAATATTACAAGC
GGCTTTCTGGGCCCGCTGCTGGTTCTGCAGGCGGGCTTTTTTCTGCTGACACGCATTCTG
ACAATTCCGCAGAGCCTGGATAGCTGGTGGACAAGCCTGAATTTTCTGGGCGGCACACCG
GTTTGCCTGGGCCAGAATAGCCAGAGCCCGACAAGCAATCATAGCCCGACAAGCTGCCCG
CCGATTTGCCCGGGCTATCGCTGGATGTGCCTGCGCCGCTTTATTATTTTTCTGTTTATT
CTGCTGCTGTGCCTGATTTTTCTGCTGGTTCTGCTGGATTATCAGGGCATGCTGCCGGTT
TGCCCGCTGATTCCGGGCAGCAGCACAACAAGCACAGGCCCGTGCAAAACATGCACAACA
CCGGCGCAGGGCACAAGCATGTTTCCGAGCTGCTGCTGCACAAAACCGAGCGATGGCAAT
TGCACATGCATTCCGATTCCGAGCAGCTGGGCGTTTGCGAAATATCTGTGGGAATGGGCG
AGCGTTCGCTTTAGCTGGCTGAGCCTGCTGGTTCCGTTTGTTCAGTGGTTTGTTGGCCTG
AGCCCGACAGTTTGGCTGAGCGTTATTTGGATGATGTGGTATTGGGGCCCGAGCCTGTAT
AATATTCTGAGCCCGTTTATTCCGCTGCTGCCGATTTTTTTTTGCCTGTGGGTTTATATT

-->


<h3 id="g2o">Gene2Oligo Analysis</h3>

<pre style="font-size:85%;" id="LHBsG2OFrag1">

input sequence 1:    LHBs fragment 1

         1         2         3         4         5         6
123456789012345678901234567890123456789012345678901234567890
ATGGGCGGCTGGAGCAGCAAACCGCGCAAAGGCATGGGCACAAATCTGAGCGTTCCGAAC
CCGCTGGGCTTTTTTCCGGATCATCAGCTGGATCCGGCGTTTCGCGCGAATAGCAACAAT
CCGGATTGGGATTTTAACCCGAATAAAGATCATTGGCCGGATGCGAATAAAGTTGGCGTT
GGCGCGTTTGGCCCGGGCTTTACACCGCCGCATGGCGGCCTGCTGGGCTGGAGCCCGCAG
GCGCAGGGCATTCTGACAACACTGCCGGCGGATCCGCCGCCGGCGAGCACAAATCGCCAG
AGCGGCCGCCAGCCGACACCGATTAGCCCGCCGCTGCGCGATACACATCCGCAGGCGATG
CAGTGGAATAGCACAACATTTCATCAGGCGCTGCAGGATCCGCGCGTTCGCGGCCTGTAT
</pre><pre style="font-size:75%;">

======  start  ========  Gene2Oligo Output (partially edited) ==============

The average Tm used for this computation is 67.2

Hybridization Map:
* Upper (+) strand is given 5' - 3'
* lower (-) strand is 3' - 5'
* Numbers are hybridization Tm
* [DNA] = 1 microM
* [Na+] = 1.0 M)

cgcgttcgtttgtcAT-GGGCGGCTGGA GCAGCAAACCGCG-CAAAGGCATGGGCA CAAATCTGAGCGTTCCG-AACC
gcgcaagcaaacagTA CCCGCCGACCT-CGTCGTTTGGCGC GTTTCCGTACCCGT-GTTTAGACTCGCAAGGC TTGG
64.08            63.49       66.17         64.41          66.87             68.42

CGCTGGGCT TTTTTCCGGATCATCAGCT-GGATCCGGCGTTTC GCGCGAATAGCAACAA-TCCGGATTGGGATTTTA
GCGACCCGA-AAAAAGGCCTAGTAGTCGA CCTAGGCCGCAAAG-CGCGCTTATCGTTGTT AGGCCTAACCCTAAAAT-
2         68.44               64.5           66.65            63.6              

ACCCGAATAAAGATCATTG-GCCGGATGCGAAT AAAGTTGGCGTTGGC-GCGTTTGGCCCG GGCTTTACACCGCC-GC
TGGGCTTATTTCTAGTAAC CGGCCTACGCTTA-TTTCAACCGCAACCG CGCAAACCGGGC-CCGAAATGTGGCGG CG
63.80               63.26         66.2            64.45        64.88          66

ATGGCGGCCT GCTGGGCTGGAGC-CCGCAGGCGCA GGGCATTCTGACAACA-CTGCCGGCGGA TCCGCCGCCGG-CG
TACCGCCGGA-CGACCCGACCTCG GGCGTCCGCGT-CCCGTAAGACTGTTGT GACGGCCGCCT-AGGCGGCGGCC GC
.21        66.96         65.62       65.0             64.05       68.46       66

AGCACAAATCGCC AGAGCGGCCGC-CAGCCGACACCGA TTAGCCCGCCGC-TGCGCGATACACATC CGCAGGCGATG
TCGTGTTTAGCGG-TCTCGCCGGCG GTCGGCTGTGGCT-AATCGGGCGGCG ACGCGCTATGTGTAG-GCGTCCGCTAC
.43           64.65       65.34         65.63        64.53           64.1       

C-AGTGGAATAGCACAACA TTTCATCAGGCGCTG-CAGGATCCGCGCG TTCGCGGCCTGT-ATcactggtgcccc
G TCACCTTATCGTGTTGT-AAAGTAGTCCGCGAC GTCCTAGGCGCGC-AAGCGCCGGACA TAgtgaccacgggg
  64.26             64.8            67.38         64.40        65.88         
</pre><pre style="font-style=85%;">

Statistics:

    Min Tm is 63.26&deg; C
    Avg Tm is 65.40&deg; C
    Max Tm is 68.46&deg; C
    Min hybridization unit size is 11 nucleotides
    Avg hybridization unit size is 13.9 nucleotides
    Max hybridization unit size is 19 nucleotides

Oligonucleotides to be synthesized:
* All sequence are 5'- 3'
* R stand for reverse or (-) strand
* F for forward or (+) strand
* Sequence in lower case is extra sequence 
   used for termination  and does not belong 
   to the input sequence
                       1         2         3         4
 #        nt  1234567890123456789012345678901234567890
==       ==== ========================================
 1 R0     16  ATgacaaacgaacgcg
 2 F0     27  cgcgttcgtttgtcATGGGCGGCTGGA
 3 R16    24  CGCGGTTTGCTGCTCCAGCCGCCC
 4 F27    27  GCAGCAAACCGCGCAAAGGCATGGGCA
 5 R40    31  CGGAACGCTCAGATTTGTGCCCATGCCTTTG
 6 F54    30  CAAATCTGAGCGTTCCGAACCCGCTGGGCT
 7 R71    32  AGCTGATGATCCGGAAAAAAGCCCAGCGGGTT
 8 F84    33  TTTTTCCGGATCATCAGCTGGATCCGGCGTTTC
 9 R103   30  TTGTTGCTATTCGCGCGAAACGCCGGATCC
10 F117   33  GCGCGAATAGCAACAATCCGGATTGGGATTTTA
11 R133   36  CAATGATCTTTATTCGGGTTAAAATCCCAATCCGGA
12 F150   32  ACCCGAATAAAGATCATTGGCCGGATGCGAAT
13 R169   28  GCCAACGCCAACTTTATTCGCATCCGGC
14 F182   27  AAAGTTGGCGTTGGCGCGTTTGGCCCG
15 R197   26  GGCGGTGTAAAGCCCGGGCCAAACGC
16 F209   26  GGCTTTACACCGCCGCATGGCGGCCT
17 R223   25  GCTCCAGCCCAGCAGGCCGCCATGC
18 F235   24  GCTGGGCTGGAGCCCGCAGGCGCA
19 R248   27  TGTTGTCAGAATGCCCTGCGCCTGCGG
20 F259   27  GGGCATTCTGACAACACTGCCGGCGGA
21 R275   22  CCGGCGGCGGATCCGCCGGCAG
22 F286   26  TCCGCCGCCGGCGAGCACAAATCGCC
23 R297   26  GCGGCCGCTCTGGCGATTTGTGCTCG
24 F312   24  AGAGCGGCCGCCAGCCGACACCGA
25 R323   25  GCGGCGGGCTAATCGGTGTCGGCTG
26 F336   27  TTAGCCCGCCGCTGCGCGATACACATC
27 R348   27  GCATCGCCTGCGGATGTGTATCGCGCA
28 F363   29  CGCAGGCGATGCAGTGGAATAGCACAACA
29 R375   32  CAGCGCCTGATGAAATGTTGTGCTATTCCACT
30 F392   28  TTTCATCAGGCGCTGCAGGATCCGCGCG
31 R407   25  ACAGGCCGCGAACGCGCGGATCCTG
32 F420   26  TTCGCGGCCTGTATcactggtgcccc
33 F432   14  ggggcaccagtgAT
        =====
         892

</pre><pre style="font-size:75%;">
======  end  ========  Gene2Oligo Output (partially edited) ==============
</pre><pre style="font-size:85%;"  id="LHBsG2OFrag2">

input sequence 2:    LHBs fragment 2

         1         2         3         4         5         6
123456789012345678901234567890123456789012345678901234567890
TTTCCGGCGGGCGGCAGCAGCAGCGGCACAGTTAATCCGGTTCCGACAATTGCGAGCCTG
ATTAGCAGCATTTTTAGCCGCACAGGCGATCCGGCGCCGAATATGGAAAATATTACAAGC
GGCTTTCTGGGCCCGCTGCTGGTTCTGCAGGCGGGCTTTTTTCTGCTGACACGCATTCTG
ACAATTCCGCAGAGCCTGGATAGCTGGTGGACAAGCCTGAATTTTCTGGGCGGCACACCG
GTTTGCCTGGGCCAGAATAGCCAGAGCCCGACAAGCAATCATAGCCCGACAAGCTGCCCG
CCGATTTGCCCGGGCTATCGCTGGATGTGCCTGCGCCGCTTTATTATTTTTCTGTTTATT
CTGCTGCTGTGCCTGATTTTTCTGCTGGTTCTGCTGGATTATCAGGGCATGCTGCCGGTT

</pre><pre style="font-size:75%;">
======  start  ========  Gene2Oligo Output (partially edited) ==============

The average Tm used for this computation is 64.4

Hybridization Map:
* Upper (+) strand is given 5' - 3'
* lower (-) strand is 3' - 5'
* Numbers are hybridization Tm
* [DNA] = 1 microM
* [Na+] = 1.0 M)

TTTCCGGCGGGC-GGCAGCAGCAGC GGCACAGTTAATCCG-GTTCCGACAATTGCG AGCCTGATTAGCAGC-ATTTTT
AAAGGCCGCCCG CCGTCGTCGTCG-CCGTGTCAATTAGGC CAAGGCTGTTAACGC-TCGGACTAATCGTCG TAAAAA
62.30        64.04        62.001          63.54           63.7            61.36 

AGCCGCACA GGCGATCCGGC-GCCGAATATGGAAAATAT TACAAGCGGCTTTCT-GGGCCCGCTGC TGGTTCTGCAG
TCGGCGTGT-CCGCTAGGCCG CGGCTTATACCTTTTATA-ATGTTCGCCGAAAGA CCCGGGCGACG-ACCAAGACGTC
          61.56       60.75              63.05           66.3        62.43      

GC-GGGCTTTTTTCTGCT GACACGCATTCTGAC-AATTCCGCAGAGCC TGGATAGCTGGTGG-ACAAGCCTGAATTTT
CG CCCGAAAAAAGACGA-CTGTGCGTAAGACTG TTAAGGCGTCTCGG-ACCTATCGACCACC TGTTCGGACTTAAAA
   62.156          62.34           63.62          61.3           62.320         

CT GGGCGGCACAC-CGGTTTGCCTGGG CCAGAATAGCCAGAGC-CCGACAAGCAATCAT AGCCCGACAAGC-TGCCC
GA-CCCGCCGTGTG GCCAAACGGACCC-GGTCTTATCGGTCTCG GGCTGTTCGTTAGTA-TCGGGCTGTTCG ACGGG
   62.33       63.94         64.59            61.541          61.379       64.57

GCCGATT TGCCCGGGCTAT-CGCTGGATGTGCCT GCGCCGCTTTATTA-TTTTTCTGTTTATTCTGCT GCTGTGCCT
CGGCTAA-ACGGGCCCGATA GCGACCTACACGGA-CGCGGCGAAATAAT AAAAAGACAAATAAGACGA-CGACACGGA
        62.57        66.74          61.09          61.44               60.80    

GATTT-TTCTGCTGGTTCTGC TGGATTATCAGGGCA-TGCTGCCGGTTc gtagtgcgccgt
CTAAA AAGACGACCAAGACG-ACCTAATAGTCCCGT ACGACGGCCAAg-catcacgcggca
      64.29           61.51           61.851       61.46       
</pre><pre style="font-style=85%;">

Statistics:

    Min Tm is 60.75&deg; C
    Avg Tm is 62.67&deg; C
    Max Tm is 66.74&deg; C
    Min hybridization unit size is 11 nucleotides
    Avg hybridization unit size is 13.9 nucleotides
    Max hybridization unit size is 19 nucleotides

Oligonucleotides to be synthesized:
* All sequence are 5'- 3'
* R stand for reverse or (-) strand
* F for forward or (+) strand
* Sequence in lower case is extra sequence 
   used for termination  and does not belong 
   to the input sequence
                       1         2         3         4
 #        nt  1234567890123456789012345678901234567890
==       ==== ========================================
 1 R0     12  GCCCGCCGGAAA
 2 F0     24  TTTCCGGCGGGCGGCAGCAGCAGC
 3 R12    27  CGGATTAACTGTGCCGCTGCTGCTGCC
 4 F24    30  GGCACAGTTAATCCGGTTCCGACAATTGCG
 5 R39    30  GCTGCTAATCAGGCTCGCAATTGTCGGAAC
 6 F54    30  AGCCTGATTAGCAGCATTTTTAGCCGCACA
 7 R69    26  GCCGGATCGCCTGTGCGGCTAAAAAT
 8 F84    29  GGCGATCCGGCGCCGAATATGGAAAATAT
 9 R95    33  AGAAAGCCGCTTGTAATATTTTCCATATTCGGC
10 F113   26  TACAAGCGGCTTTCTGGGCCCGCTGC
11 R128   24  GCCTGCAGAACCAGCAGCGGGCCC
12 F139   28  TGGTTCTGCAGGCGGGCTTTTTTCTGCT
13 R152   30  GTCAGAATGCGTGTCAGCAGAAAAAAGCCC
14 F167   29  GACACGCATTCTGACAATTCCGCAGAGCC
15 R182   28  CCACCAGCTATCCAGGCTCTGCGGAATT
16 F196   31  TGGATAGCTGGTGGACAAGCCTGAATTTTCT
17 R210   28  GTGTGCCGCCCAGAAAATTCAGGCTTGT
18 F227   24  GGGCGGCACACCGGTTTGCCTGGG
19 R238   29  GCTCTGGCTATTCTGGCCCAGGCAAACCG
20 F251   31  CCAGAATAGCCAGAGCCCGACAAGCAATCAT
21 R267   27  GCTTGTCGGGCTATGATTGCTTGTCGG
22 F282   24  AGCCCGACAAGCTGCCCGCCGATT
23 R294   24  ATAGCCCGGGCAAATCGGCGGGCA
24 F306   26  TGCCCGGGCTATCGCTGGATGTGCCT
25 R318   28  TAATAAAGCGGCGCAGGCACATCCAGCG
26 F332   33  GCGCCGCTTTATTATTTTTCTGTTTATTCTGCT
27 R346   33  AAATCAGGCACAGCAGCAGAATAAACAGAAAAA
28 F365   29  GCTGTGCCTGATTTTTCTGCTGGTTCTGC
29 R379   30  TGCCCTGATAATCCAGCAGAACCAGCAGAA
30 F394   27  TGGATTATCAGGGCATGCTGCCGGTTc
31 R409   24  acggcgcactacgAACCGGCAGCA
32 R421   12  gtagtgcgccgt
        =====
         866

</pre><pre style="font-size:75%;">

======  end  ========  Gene2Oligo Output (partially edited) ==============

</pre><pre style="font-size:85%;"  id="LHBsG2OFrag3">

input sequence 3:    LHBs fragment 3

         1         2         3         4         5         6
123456789012345678901234567890123456789012345678901234567890
TGCCCGCTGATTCCGGGCAGCAGCACAACAAGCACAGGCCCGTGCAAAACATGCACAACA
CCGGCGCAGGGCACAAGCATGTTTCCGAGCTGCTGCTGCACAAAACCGAGCGATGGCAAT
TGCACATGCATTCCGATTCCGAGCAGCTGGGCGTTTGCGAAATATCTGTGGGAATGGGCG
AGCGTTCGCTTTAGCTGGCTGAGCCTGCTGGTTCCGTTTGTTCAGTGGTTTGTTGGCCTG
AGCCCGACAGTTTGGCTGAGCGTTATTTGGATGATGTGGTATTGGGGCCCGAGCCTGTAT
AATATTCTGAGCCCGTTTATTCCGCTGCTGCCGATTTTTTTTTGCCTGTGGGTTTATATT

</pre><pre style="font-size:75%;">

======  start  ========  Gene2Oligo Output (partially edited) ==============

The average Tm used for this computation is 63.9

Hybridization Map:
* Upper (+) strand is given 5' - 3'
* lower (-) strand is 3' - 5'
* Numbers are hybridization Tm
* [DNA] = 1 microM
* [Na+] = 1.0 M)

aTGCCCGCTGATTC-CGGGCAGCAGC ACAACAAGCACAGG-CCCGTGCAAAACA TGCACAACACCGG-CGCAGGGCAC
tACGGGCGACTAAG GCCCGTCGTCG-TGTTGTTCGTGTCC GGGCACGTTTTGT-ACGTGTTGTGGCC GCGTCCCGTG
60.809         62.4        60.59          60.39         63.27         60.56     

A AGCATGTTTCCGAG-CTGCTGCTGCACA AAACCGAGCGATGG-CAATTGCACATGCAT TCCGATTCCGAGC-AGCT
T-TCGTACAAAGGCTC GACGACGACGTGT-TTTGGCTCGCTACC GTTAACGTGTACGTA-AGGCTAAGGCTCG TCGA
  59.96          63.3          63.58          60.27           61.28         60.6

GGGCGTTT GCGAAATATCTGTGGG-AATGGGCGAGCG TTCGCTTTAGCTGG-CTGAGCCTGCTGG TTCCGTTTGTTC
CCCGCAAA-CGCTTTATAGACACCC TTACCCGCTCGC-AAGCGAAATCGACC GACTCGGACGACC-AAGGCAAACAAG
         62.62            62.07        59.96          62.62         60.27       

AGT-GGTTTGTTGGCCTG AGCCCGACAGTTT-GGCTGAGCGTTATT TGGATGATGTGGTATTG-GGGCCCGAGCC TG
TCA CCAAACAACCGGAC-TCGGGCTGTCAAA CCGACTCGCAATAA-ACCTACTACACCATAAC CCCGGGCTCGG-AC
    62.04          60.86         60.06          61.85             64.75       62

TATAATATTCTGAGCCC-GTTTATTCCGCTGCT GCCGATTTTTTTTTGC-CTGTGGGTTTATATTcct gcagccggta
ATATTATAAGACTCGGG CAAATAAGGCGACGA-CGGCTAAAAAAAAACG GACACCCAAATATAAgga-cgtcggccat
.255              61.537          60.798           61.95              61.60     

gg
cc
</pre><pre style="font-style=85%;">

Statistics:

    Min Tm is 59.96&deg; C
    Avg Tm is 61.56&deg; C
    Max Tm is 64.75&deg; C
    Min hybridization unit size is 11 nucleotides
    Avg hybridization unit size is 13.9 nucleotides
    Max hybridization unit size is 19 nucleotides

Oligonucleotides to be synthesized:
* All sequence are 5'- 3'
* R stand for reverse or (-) strand
* F for forward or (+) strand
* Sequence in lower case is extra sequence 
   used for termination  and does not belong 
   to the input sequence
                       1         2         3         4
 #        nt  1234567890123456789012345678901234567890
==       ==== ========================================
 1 R0     14  GAATCAGCGGGCAt
 2 F0     25  aTGCCCGCTGATTCCGGGCAGCAGC
 3 R14    25  CCTGTGCTTGTTGTGCTGCTGCCCG
 4 F25    27  ACAACAAGCACAGGCCCGTGCAAAACA
 5 R39    26  CCGGTGTTGTGCATGTTTTGCACGGG
 6 F52    24  TGCACAACACCGGCGCAGGGCACA
 7 R65    25  CTCGGAAACATGCTTGTGCCCTGCG
 8 F76    27  AGCATGTTTCCGAGCTGCTGCTGCACA
 9 R90    27  CCATCGCTCGGTTTTGTGCAGCAGCAG
10 F103   29  AAACCGAGCGATGGCAATTGCACATGCAT
11 R117   28  GCTCGGAATCGGAATGCATGTGCAATTG
12 F132   25  TCCGATTCCGAGCAGCTGGGCGTTT
13 R145   28  CCCACAGATATTTCGCAAACGCCCAGCT
14 F157   28  GCGAAATATCTGTGGGAATGGGCGAGCG
15 R173   26  CCAGCTAAAGCGAACGCTCGCCCATT
16 F185   27  TTCGCTTTAGCTGGCTGAGCCTGCTGG
17 R199   28  ACTGAACAAACGGAACCAGCAGGCTCAG
18 F212   29  TTCCGTTTGTTCAGTGGTTTGTTGGCCTG
19 R227   27  AAACTGTCGGGCTCAGGCCAACAAACC
20 F241   27  AGCCCGACAGTTTGGCTGAGCGTTATT
21 R254   31  CAATACCACATCATCCAAATAACGCTCAGCC
22 F268   28  TGGATGATGTGGTATTGGGGCCCGAGCC
23 R285   30  GGGCTCAGAATATTATACAGGCTCGGGCCC
24 F296   34  TGTATAATATTCTGAGCCCGTTTATTCCGCTGCT
25 R315   31  GCAAAAAAAAATCGGCAGCAGCGGAATAAAC
26 F330   34  GCCGATTTTTTTTTGCCTGTGGGTTTATATTcct
27 R346   30  cctaccggctgcaggAATATAAACCCACAG
28 R364   12  gcagccggtagg                           
        =====
         752

</pre><pre style="font-size:75%;">
======  end  ========  Gene2Oligo Output (partially edited) ==============
</pre>


<h3>Sequences of Oligos and Primers for In Vitro Gene Synthesis</h3>
<p>
This section should contain the information about all oligonucleotides and
primers to be used for gene synthesis by PCR assembly.  Other primers
and oligos for standard cloning techniques should be found in another 
section.

<h4>Oligos for LHBs Segment-Joining By Overlap Amplifcation</h4>
<p>
Melting temperature calculations for oligos and primers were done with an <a 
href="http://eu.idtdna.com/analyzer/Applications/OligoAnalyzer/">online 
calculator</a> using as parameters [oligo] = 0.1 &micro;M,
[Na<sup>+</sup>] = 50 mM, [Mg<sup>2+</sup>] = 1.5 mM, and [dNTPs] = 0.2 mM.

<pre style="font-size:85%;">

[name: L1L2fo]
LHBsF1-LHBsF2 overlap sense oligo (complements anti-sense)
    5&prime;-GCGCGTTCGCGGCCTGTATTTTCCGGCGGGCGGCAG-3&prime;  (19 + 17 = 36 nt)
             67-68&deg;       |     67-68&deg;

[name: L1L2ro]
LHBsF1-LHBsF2 anti-sense oligo (complements sense)
    5&prime;-CTGCCGCCCGCCGGAAAATACAGGCCGCGAACGCGC-3&prime; (17 + 19 = 36 nt)
            67-68&deg;      |     67-68&deg;

[name: L2L3fo]
LHBsF2-LHBsF3 overlap sense oligo (complements anti-sense)
    5&prime;-GATTATCAGGGCATGCTGCCGGTTTGCCCGCTGATTCCGGGC-3&prime; (24 + 18 = 42 nt)
               67-68&deg;          |     67-68&deg;

[name: L2L3ro]
LHBsF2-LHBsF3 anti-sense oligo (complements sense)
    5&prime;-GCCCGGAATCAGCGGGCAAACCGGCAGCATGCCCTGATAATC-3&prime; (18 + 24 = 42 nt)
            67-68&deg;      |     67-68&deg;
</pre>

<h4>Primers For Boundaries of LHBs, MHBs, and SHBs Message Sequences</h4>
<p>
The reverse-complement (anti-sense) primers for SHBs, MHBs, and LHBs are all
shared (identical), since their 3&prime; ends are in common.

<pre style="font-size:85%;">
[name: LHBsfp]
LHBs 5&prime;-sense (forward) primer (complements anti-sense)
    5&prime;-ATGGGCGGCTGGAGCAGC-3&prime;  (18 nt) 
                67-68&deg;

[name: LHBsrp]
LHBs 3&prime;-anti-sense (reverse complement) primer (complements sense)
    5&prime;-AATATAAACCCACAGGCAAAAAAAAATCGGCAG-3&prime;  (33 nt) 
                67-68&deg;

[name: MHBsfp]
MHBs 5&prime;-sense (forward) primer (complements anti-sense)
    5&prime;-ATGCAGTGGAATAGCACAACATTTCATCAGG-3&prime;  (31 nt) 
                67-68&deg;

[name: SHBsfp]
SHBs 5&prime;-sense (forward) primer (complements anti-sense)
    5&prime;-ATGGAAAATATTACAAGCGGCTTTCTGGGC-3&prime;  (30 nt) 
                67-68&deg;
</pre>

<h4>Primers For OE-PCR Cloning into pUC19</h4>
<p>
These are the primers for doing maintenance cloning into pUC19.  The forward
primer includes the pUC19 vector portion 5&prime; to the
<p>
The reverse complement primer on its 5&prime; end includes
sequence from the pUC19 vector just 3&prime; to the <i>Xba</i>I site in the mcs.
The forward primers between the <q>|</q> character and the 3&prime; end contain
5&prime; sequence from the inserts, and the reverse primer between the 5&prime;
end and the <q>|</q> character is sequence from the 3&prime; end of the insert.

<pre style="font-size:85%;">
[name: OE-UC19-LHBs-fp]
OE-PCR-pUC19-LHBs sense (forward) primer (complements anti-sense) 58 nt
                           $$$$$$
    5&prime;-GTTGTAAAACGACGGCCAGTGAATTC|ATGGGCGGCTGGAGCAG-3&prime;

[name: OE-UC19-LMSHBs-rp]
OE-PCR-pUC19-LHBs antisense (reverse comp) primer (complements sense) 42 nt
       %%%%%%++++++
    5&prime;-GTCGACTCTAGA|AATATAAACCCACAGGCAAAAAAAAATCGG-3&prime;

[name: cutlig-UC19-LHBs-fp]
cutlig-PCR-pUC19-LHBs sense (forward) primer (complements anti-sense) 27 nt
           $$$$$$
    5&prime;-GTACGAATTC|ATGGGCGGCTGGAGCAG-3&prime;


  $$$$$$ = EcoRI
  %%%%%% = HindIII (GTYRAC) | SalI (GTCGAC) | AccI (GTMKAC)
  ++++++ = XbaI
</pre>

<h3>Restriction Sites for Optimized LHBs Message</h3>
<p>
Below is the optimized LHBs sequence used for input for finding 
restriction sites

<pre style="font-size:85%;">
         1         2         3         4         5         6
123456789012345678901234567890123456789012345678901234567890

ATGGGCGGCTGGAGCAGCAAACCGCGCAAAGGCATGGGCACAAATCTGAGCGTTCCGAAC
CCGCTGGGCTTTTTTCCGGATCATCAGCTGGATCCGGCGTTTCGCGCGAATAGCAACAAT
CCGGATTGGGATTTTAACCCGAATAAAGATCATTGGCCGGATGCGAATAAAGTTGGCGTT
GGCGCGTTTGGCCCGGGCTTTACACCGCCGCATGGCGGCCTGCTGGGCTGGAGCCCGCAG
GCGCAGGGCATTCTGACAACACTGCCGGCGGATCCGCCGCCGGCGAGCACAAATCGCCAG
AGCGGCCGCCAGCCGACACCGATTAGCCCGCCGCTGCGCGATACACATCCGCAGGCGATG
CAGTGGAATAGCACAACATTTCATCAGGCGCTGCAGGATCCGCGCGTTCGCGGCCTGTAT
TTTCCGGCGGGCGGCAGCAGCAGCGGCACAGTTAATCCGGTTCCGACAATTGCGAGCCTG
ATTAGCAGCATTTTTAGCCGCACAGGCGATCCGGCGCCGAATATGGAAAATATTACAAGC
GGCTTTCTGGGCCCGCTGCTGGTTCTGCAGGCGGGCTTTTTTCTGCTGACACGCATTCTG
ACAATTCCGCAGAGCCTGGATAGCTGGTGGACAAGCCTGAATTTTCTGGGCGGCACACCG
GTTTGCCTGGGCCAGAATAGCCAGAGCCCGACAAGCAATCATAGCCCGACAAGCTGCCCG
CCGATTTGCCCGGGCTATCGCTGGATGTGCCTGCGCCGCTTTATTATTTTTCTGTTTATT
CTGCTGCTGTGCCTGATTTTTCTGCTGGTTCTGCTGGATTATCAGGGCATGCTGCCGGTT
TGCCCGCTGATTCCGGGCAGCAGCACAACAAGCACAGGCCCGTGCAAAACATGCACAACA
CCGGCGCAGGGCACAAGCATGTTTCCGAGCTGCTGCTGCACAAAACCGAGCGATGGCAAT
TGCACATGCATTCCGATTCCGAGCAGCTGGGCGTTTGCGAAATATCTGTGGGAATGGGCG
AGCGTTCGCTTTAGCTGGCTGAGCCTGCTGGTTCCGTTTGTTCAGTGGTTTGTTGGCCTG
AGCCCGACAGTTTGGCTGAGCGTTATTTGGATGATGTGGTATTGGGGCCCGAGCCTGTAT
AATATTCTGAGCCCGTTTATTCCGCTGCTGCCGATTTTTTTTTGCCTGTGGGTTTATATT
</pre>
<p>
The table below shows selected enzymes found to be able to cut the
sequence.  This is not an exhaustive list of enzymes that cut, so it
is essential to check whether a desired enzyme for cloning will 
affect the sequence.
<table class="td-left th-center" style="width:100%;font-size:90%;">
<caption>(Some) Restriction Enzymes Cutting Within Optimized LHBs Message</caption>
<tr><th>Enzyme <th style="width:3em;">Number of Cuts <th>Fragment Start/End Positions  <th>Fragment Sizes
<tr><td><i>Not</i>I <td class="center">1  <td>1/303, 304/1200               <td>303, 897
<tr><td><i>Msl</i>I <td class="center">1  <td>1/916, 917/1200               <td>916, 284
<tr><td><i>Sph</i>I <td class="center">1  <td>1/831, 832/1200               <td>831, 396
<tr><td><i>Sma</i>I <td class="center">2  <td>1/194, 195/731, 732/1200      <td>194, 537, 469
<tr><td><i>Ssp</i>I <td class="center">2  <td>1/531, 532/1143, 1144/1200    <td>531, 612, 57
<tr><td><i>Pvu</i>II<td class="center">2  <td>1/87, 88/986, 987/1200        <td>87, 899, 214
<tr><td><i>Mfe</i>I <td class="center">2  <td>1/467, 468/957, 958/1200      <td>467, 490, 243
<tr><td><i>Pst</i>I <td class="center">2  <td>1/395, 396/569, 570/1200      <td>395, 174, 631
<tr><td><i>Alw</i>NI <td class="center">2  <td>1/390, 391/615, 616/1200     <td>390, 225, 585
<tr><td><i>Bam</i>HI<td class="center">3  <td>1/90, 91/270, 271/396, 397/1200<td>90, 180, 126, 804
<tr><td><i>Bgl</i>I <td class="center">3  <td>1/224, 225/302, 303/503, 504/1200<td>224, 78, 201, 697
</table>
<p>
Based on these, a good selection of enzymes would be either <i>Not</i>I or 
<i>Sph</i>I (perhaps <i>Sph</i>I is slightly better, <i>Pvu</i>II and 
<i>Pst</i>I as two-cutters, and <i>Bam</i>HI as a 3-cutter, although <i>Bgl</i>I 
is good too.

<h1>Communications from Doruk</h1>
<p>
<b>31 August PDF Materials and Methods for project</b>.  Goal is to obtain 
recombinant hepatitis B surface antigen expressed in E coli for ELISA detection. 
Our project HBV surface antigen pre-S1/pre-S2/S region coding HBVgp2 product 
from a PLC/PRF/5 cell line (ATCC CRL-8024 being integrated into cells with adw2 
type) and Hep3B (ATCC HB-8064) cell line.  Cell lines are grown in RPMI-1640 
with 10% FBS.  RNA isolation is done when cells reach 70% confluence in culture 
flasks.  A lysis solution containing guandinium is added to the culture flasks
of cells for doing RNA isolation.  The lysate is then extracted with acid phenol
chloroform and the aqueous phase is washed consecutively with 99% and 70% ethanol
in that order.  The pellet is collected and resuspended in RNase-free water.
Purified RNA samples are stored at &minus;80&deg;.
<p>
The MuMLV is used in the reverse transcription of the total RNA prep using
an oligo(dT)<sub>24</sub> primer.  Second strand synthesis and HBVgp2 
amplification by PCR are then done.   
<p>
<b>Email from 6 Oct 2011</b>:  For the 8 genotypes of the S, M, and L
segments of HBV gp2 the following primers have been determined:
<ul style="font:normal 90% 'Courier New',Courier,monotype;">
<li> gp2SF: 5'-ATGGCTAGCATGGAAAGCATCACATCAGGA-3'
<li> gp2MF: 5'-ATGGCTAGCCCTCAGGCMATGCAGTGGAAC-3'      
<li> gp2LF: 5'-ATGGCTAGCGGGAACAAGAKCTACAGCATG-3'
<li> gp2R:  5'-ATGGAATTCTGTTGGGATTTAAATGTATACC-3'
</ul>

<p>
<b>Email from 5 Nov 2011</b>:  First happy holidays. We have realized a big 
step today in spite of our not officially being in the project.  I started a
Hep3B cells that has an integrated HBV genome, with cells adhering a little
before under my control.  Now the cells have DNA and RNA isolated we will 
amplify with pro=imers that have been ordered and done according to design.
The products will be cloned into bacterial expression vector.  Before the panel
our first necessary activity will be to learn the genotype using DNA sequence
analysis. of the HBVgp2 segment.  I wish happy holidays to all of the project
group.
<p>
<b>Email from 20 Nov 2011</b>:  Didem and I have advanced our work to getting
a HBsAg prototype.  We succeeded in copy and reverse transcribing isolated RNA
from Hep 3B cells  At this moment in our hands it appears that S and M segments
have been copies (in the gel image they are after the the 1st and 3rd molecular
weight standards) and the L segment will come out this week.  We will be able 
to close into an expression vector.  We are nearing the project presentation date.
We have strengthened our hand being able to copy the HBV genome within Hep3B
cells.  We are collecting for coordination.
<p style="text-align:center;">
<img src="DorukSMclone.jpg" class="orig-size"
  alt="clone of S and M HBsAg segments">

<h2>Proposal Notes</h2>
<p>
The source for the HBV genome is to be infected cell line cultures PLC/PRF/5
(ATCC CRL-8024) containing <i>adw2</i> type and Hep3B cells (ATCC HB-8064)
with the HBV genome integrated.  Growth is in RPMI-1640 with 10% FBS in culture
bottles where the cells adhere to the surface.
<p>
RNA is isolated using the guanidinium method and then the lysate extracted with
acid phenol chloroform.  The aqueous phase is washed with 99% then 70% EtOH
in series.  After the pellet is dry it is resuspended in RNase-free water and 
stored at &minus;80&deg; if necessary.
<p>
The total RNA is mixed with oligo(dT)<sub>24</sub> primer with MuMLV reverse
transcriptase.  2nd strand synthesis is done using PCR with primer containing
restriction sites.  Amplified product is cleaned using PCR kit.  The cleaned
product is then run on 2% agarose gel (in TBE with EtBr).  The target will be
about 1200 bp long.  Purified product is treated with <i>Eco</i>RI and
<i>Hin</i>dIII along with pGEM-3Zf(+) vector, which are then both purfied
on gels.  Material is quantitated for the ligase reaction using T4 DNA ligase.
<p>
Ligated product is then transfected into <i>E coli</i> JM109 using PEG-DMSO
and heat shock (method of Chung, Niemela, Miller PNAS 1989).  Bacteria are plated
to SOC agar with 100 &micro;g/ml ampicillin.   Transformants containing inserts
are selected with  SOB top agar medium containing X-gal and IPTG with an overnight
culture at 35&deg;.  White colonies are amplified with M13-20 and M13-26 primers
to reveal a 1450 bp product to be sequenced.
<p>
For sequencing, the 1203 bp HBVgp2 region must be amplified in segments of 450-500
bp using PCR.  
<p>
<b>Pilot expression</b>.  Several different systems will be used to control 
expression:  T7-lac, Trp-lac, lacUV5 and cspA cold shock with arabinose will be 
promoters that are studied.  These systems to be tested are pET28a, pCold, 
pBAD/His, and pET-SUMO.  Cultures will be induced with either IPTG or arabinose
for 6-18 hour periods.  For pilot expression microwell plates and small volume
Erlenmeyer flasks are used.  Bacterial pellets are extracted in heat and sonicated
in SDS sample buffer.  Samples are run on 12-15% SDS gels, then stained in 
Coomassie Blue.  Bands of the predicted sizes (25.5, 31.2, and 43.7 kDa for S,
M, and L forms) are sought.  Higher sizes will be sought if a tag is present.
Western blots are done on PVDF, with chemiluminescent detection.
<p>
For purification, factox Xa protease with its tag is contemplated.


<h1 id="consumables">Consumables &amp; Equipment Required</h1>
<p>
The <a href="AbiyotekEquipment.html#HBsAgProj">table of equipment and
instruments</a> required for the project is located on <a
href="AbiyotekEquipment.html#HBsAgProj">another page</a>.

</div> <!-- outerblock -->

<!-- CONSUMABLES TABLE -->

<table class="widetable" style="font-size:75%;margin:0 3em;">
<caption>Consumables &amp; Service</caption>
<tr><th>Item/Product
  <th>Use(s)
  <th>Make/ Vendor
  <th>Product Name
  <th>Catalog or Prod Num
  <th>List or Estimated Price
  <th>Comment
  <th>Quantity Needed
<tr><td class="header" colspan="8">Chemical Competence for E. coli (Inoue)

<tr><td>17 &times; 100 mm polypropylene tubes <td>Bacterial transformation reactions
  <td>Fisher <td>Translucent polypropylene no cap <td>14-956-7E <td>$125/1000ct
  <td>example  <td> 100 ct
<tr><td>manganese chloride tetrahydrate, MnCl<sub>2</sub> 4 &bull; H<sub>2</sub>O [CAS 13446-34-9]
     <td>Inoue chemical competence reagent component
  <td>Fisher <td>Fisherbrand mang chlor crystalline ACS <td>M87-100 <td>$61/100 g
  <td>example  <td> 50 g 
<tr><td>calcium chloride dihydrate, CaCl<sub>2</sub> 2 &bull; H<sub>2</sub>O [CAS 10035-04-8]
     <td>Inoue chemical competence reagent component
  <td>Fisher <td>FisherBioReagent calc chlor white cryst <td>BP510-100 <td>$32/100 g
  <td>example <td> 25 g
<tr><td>PIPES [CAS 10035-04-8]
     <td>Inoue chemical competence reagent component
  <td>Fisher <td>JustPURE PIPES <td>BP2924-50 <td>$132/50 g
  <td>example <td>25 g
<tr><td>potassium hydroxide, KOH [CAS 1310-58-3]
     <td>Inoue chemical competence reagent component
  <td>Fisher <td>KOH pellets, cert ACS >= 85% <td>P250-1 <td>1 kg
  <td>example <td>250 g
<tr><td>potassium chloride, KCl [CAS 7447-40-7]
     <td>Inoue chemical competence reagent component
  <td>Fisher <td>KCl ACS cert 99-100%<td>P3911-1 <td> $76/1 kg
  <td>example  <td> 500 g 
<tr><td>magnesium sulfate, MgSO<sub>4</sub> [CAS ]
     <td>Inoue chemical competence reagent component
  <td>Fisher <td>AcrosOrganics magnesium sulfate 7H<sub>2</sub>O ACS <td>AC42390-0250<td>$32/25 g
  <td>example <td>10 g
<tr><td>dimethyl sulfoxide, DMSO [CAS 67-68-5]
     <td>bacterial transformation, PCR
  <td>Fisher <td>Fisher BioReagents DMSO<td>BP231-100 <td> $33/100 ml
  <td>example  <td> 20 ml

<tr><td class="header" colspan="8">For Transforming <i>B. subtilis</i>
<tr><td>magnesium chloride, MgCl<sub>2</sub> [CAS]
     <td>B. subtilis transformation component
  <td>Sigma-Aldrich <td>magnesium chloride hexahydrate <td>M9272-100G <td> $23/100 g
  <td>example  <td> 25 g
<tr><td>dipotassium hydrogen phosphate, K<sub>2</sub>HPO<sub>4</sub> [CAS 7758-11-4]
     <td>B. subtilis transformation component
  <td>Sigma-Aldrich <td>K<sub>2</sub>HPO<sub>4</sub> ACS Reagent &ge;98% <td>P3786-100G <td> &euro;33/100 g
  <td>example  <td> 25 g
<tr><td>potassium dihydrogen phosphate, KH<sub>2</sub>PO<sub>4</sub> [CAS 77778-77-0]
     <td>B. subtilis transformation component
  <td>Sigma-Aldrich <td>K<sub>2</sub>HPO<sub>4</sub> for mol biol &ge;98%  <td>P9791-100G <td> &euro;44/100 g
  <td>example  <td> 100 g
<tr><td>trisodium sodium citrate dihydrate [CAS 6132-04-3]
     <td>B. subtilis transformation component
  <td>Fisher <td>AcrosOrganics citric acid trisodium salt dihydrate Pure 99%
  <td>AC22173-5000 <td> $23/500 g <td>example  <td> 100 g
<tr><td>ammonium sulfate  [CAS 7783-20-2] <td>B. subtilis transformation component
  <td>Fisher <td>ammonium sulfate (Gran/Cert ACS) <td>A702-500 <td> $58/500 g
  <td>example  <td> 250 g
<tr><td>casamino acids
     <td>B. subtilis transformation component
  <td>Fisher <td>casamino acids  <td>BP1424-100 <td> $64/100 g
  <td>example  <td> 100 g
<tr><td>D-glucose monohydrate [CAS 14431-43-7]
     <td>B. subtilis transformation component; media supplement
  <td>Sigma-Aldrich <td>D-(+)-glucose monohydrate <td>49159-1KG <td> &euro;25/1 kg
  <td>example  <td> 500 g
<tr><td>Ethylene glycol-bis(2-aminoethylether)-N,N,N',N'-tetraacetic acid, EGTA [CAS 67-42-5]
     <td>B. subtilis transformation component
  <td>Fisher <td>AcrosOrganics EGTA, 99%  <td>AC40991-0250 <td> $98/25 g
  <td>example  <td>10 g


<tr><td class="header" colspan="8">DNA/Plasmid Preparation
<tr><td>phenol [CAS 108-95-2]
     <td>general DNA de-proteination
  <td>Fisher <td>Acros Organics ACS phenol<td>AC41717-0250 <td> $37/25 g
  <td>example  <td>30 g
<tr><td>chloroform [CAS 67-66-3]
     <td>DNA cleanup
  <td>Fisher <td>Fisher BioReagents mol biol chloroform<td>BP1145-1 <td> $64/1 L
  <td>example <td> 100 ml
<tr><td>isoamyl alcohol [CAS 123-51-3]
     <td>DNA cleanup
  <td>Fisher <td>Fisher BioReagents DMSO<td>ICN19405980 <td> $34/100 ml
  <td>example <td>20 ml
<tr><td>ethanol [CAS 64-17-5]
     <td>DNA precipitation, other needs
  <td>Fisher <td>absolute (200 proof) <td>BP2818-4 <td> $247/4 L
  <td>example  <td> 2.5 L
<tr><td>2-propanol (isopropanol) [CAS 67-63-0]
     <td>precipitating DNA
  <td>Fisher <td>molec biol grade 2-propanol<td>BP2618-500 <td> $45/500 ml
  <td>example  <td>later
<tr><td>sodium hydroxide [CAS 1310-73-2]
     <td>precipitating DNA, general purpose
  <td>Fisher <td>sodium hydroxide ACS <td>S318-1 <td> $135/1 kg
  <td>example <td>500 g

<tr><td class="header" colspan="8">Protein Chemistry Work
<tr><td>trichloroacetic acid [CAS 76-03-9] <td>protein precipitation
  <td>Sigma-Aldrich <td>ACS reagent &ge;99.0% <td>33731-100G <td> &euro;15/100 g
  <td>example  <td>100 g

<tr><td class="header" colspan="8">Protein Electrophoresis / SDS-PAGE
<tr><td>sodium dodecyl sulfate [CAS 151-21-3]
     <td>SDS-PAGE component, plasmid prep
  <td>Bio-Rad <td>Sodium Dodecyl Sulfate <td>161-0301 <td> $32/100 g
  <td>example  <td>100 g
<tr><td>acrylamide/bis-acrylamide solid 37.5:1 pre-mix
     <td>protein gel electrophoresis
  <td>Bio-Rad <td>37.5:1 acrylamide/bis premix <td>161-0125<td> $75/150 g
  <td>example <td>150 g
<tr><td>ammonium persulfate [CAS 7727-54-0]
     <td>protein gel electrophoresis
  <td>Bio-Rad <td> Ammonium Persulfate (APS) <td>161-0700 <td> $14/10
  <td>example <td>10 g
<tr><td>TEMED [CAS 110-18-9]
     <td>protein gel electrophoresis
  <td>Bio-Rad <td>TEMED <td>161-0800 <td> $17/10 ml
  <td>example  <td>10 ml
<tr><td>Coomassie Brilliant Blue R-250 [CAS 6104-59-2]  <td>protein staining
  <td>Bio-Rad <td>Coomassie BB-R250 <td>161-0400 <td> $?/10 g
  <td>example  <td>5 g
<tr><td>methanol [CAS 67-56-1]<td>protein staining, general purpose
  <td>Fisher <td>ACS Certified <td>A412-4 <td> 95 YTL/4L glass
  <td>example <td>4 L
<tr><td>glycine [CAS ]
     <td>SDS-PAGE electrode buffer component
  <td>Bio-Rad <td>glycine <td>161-0718 <td> $49/1 kg
  <td>example <td>1 kg
<tr><td>hydrochloric acid, HCl [CAS 7647-01-0]  <td>SDS-PAGE gel/electrode buffer component
  <td>Sigma-Aldrich <td>ACS Reagent Grade <td>320331-500ML <td> &euro;48/500 ml
  <td>example <td>500 ml
<tr><td>broad range 10-250 kDa MW markers
     <td>protein MW determination
  <td>Bio-Rad <td>Precision Plus Protein Unstained Standards <td>161-0363<td> $96/100 uses
  <td>example <td>50 uses


<tr><th>Item/Product
  <th>Use(s)
  <th>Make/ Vendor
  <th>Product Name
  <th>Catalog or Prod Num
  <th>List or Estimated Price
  <th>Comment
  <th>Quantity Needed
<tr><td class="header" colspan="8">Bacterial Culture

<tr><td>100 mm polystyrene culture dish (sterile)
     <td>solid culture medium container
  <td>Fisher <td>100 &times; 15 mm raised ridge PS petri dish<td>08-757-12 <td>$225/500ct
  <td>example  <td>200 ct
<tr><td>50-60 mm polystyrene culture dish (sterile)
     <td>solid culture medium container (smaller work)
  <td>Fisher <td>60 &times; 15 mm raised ridge PS petri dish<td>08-757-13A <td>$176/500ct
  <td>example <td>50 ct
<tr><td>sodium chloride, NaCl [CAS 7647-14-5]
     <td>Luria-Bertani culture medium component, multiple uses
  <td>Fisher <td>Fisher BioReagents sodium chloride &gt; 99.5% <td>BP358-212 <td>$64/2.5 kg
  <td>example <td>2.5-3 kg
<tr><td>tryptone <td>Luria-Bertani culture medium component
  <td>Fisher <td>Fisher BioReagent tryptone (gran.) <td>BP9726-500  <td>$126/500 g
  <td>example <td>500 g
<tr><td>yeast extract  <td>Luria-Bertani culture medium component
  <td>Fisher <td>Fisner BioReagent yeast extract (gran.)  <td>BP9726-500 <td>$85/500 g
  <td>example <td>250 g
<tr><td>agar <td>solid microbial culture medium component
  <td>Fisher <td>Fisher BioReagents agar (gran.)<td>BP1423-500 <td>$140/500 g
  <td>example <td>500 g
<!-- 500 g agar will make the following:

1 plate/15 ml * 100 ml/1.5 g agar * 500 g agar

   Conc  Vol/plate  Count              Conc  Vol/plate  Count
   ====  =========  =====              ====  =========  =====
    1%      15 ml    3333              1.5%     15 ml   2222
            30 ml    1666                       30 ml   1111
-->
<tr><td>ampicillin sodium [CAS 69-52-3]<td>antibiotic used in selection
  <td>Fisher <td>Fisher BioReagents ampicillin sodium<td>BP1760-25 <td>$85/25 g
  <td>example <td>25 g
<tr><td>kanamycin sulfate [CAS 25389-94-0]<td>antibiotic used in selection
  <td>Fisher <td>Fisher BioReagents kanamycin sulfate<td>BP906-5 <td>$63/5 g
  <td>example <td>5 g
<tr><td>autoclave tape
     <td>culture supplies sterility
  <td>Fisher <td>Strate-Line Autoclave Tape, 13 mm x 55 m<td>11-889-1 <td>$2
  <td>example <td>50-100 m

<tr><td class="header" colspan="8">Agarose Gel Electrophoresis
<tr><td>agarose, for electrophoresis  [CAS 9012-36-6]
     <td>agarose gel electrophoresis component
  <td>Fisher <td>Fisher BioReagent Low EEO Multipurpose<td>BP160-500 <td>$97/500 g
  <td>example  <td>500 g
<tr><td>ethidium bromide (EtBr) [CAS 1239-45-8]
     <td>agarose gel electrophoresis component
  <td>Fisher <td>Fisher BioReagents ethidium bromide <td>BP102-1 <td>$45/1 g
  <td>example <td>500-1000 mg
<tr><td>Tris (base) [CAS 77-86-1]
     <td>buffer salt for many needs
  <td>Fisher <td>Tris base, white cryst powder <td>BP152-1 <td>$134/1 kg
  <td>example <td>1 kg
<tr><td>boric acid [CAS 10043-35-3] <td>agarose gel electrophoresis buffer component
  <td>Fisher <td>Fisher BioReagents boric acid crystalline <td>BP168-500 <td>$51/500 g
  <td>example <td>500 g
<tr><td>EDTA [CAS 60-00-4]   <td>agarose gel electrophoresis buffer component
  <td>Fisher <td>Fisher BioReagents EDTA white cryst <td>BP118-500 <td>$87/500 g
  <td>example <td>100 g
<tr><td>acetic acid, glacial [CAS 64-19-7]<td>agarose gel electrophoresis buffer component
  <td>Fisher <td>Acetic Acid, Certified ACS Plus <td>A38-212 <td>105 YTL/2.5 L glass
  <td>example <td>4 L
<tr><td>glycerol [CAS 56-81-5]   <td>multiple uses
  <td>Fisher <td>Fisher BioReagents mol biol glycerol  <td>BP229-1  <td>$99/1 L
  <td>example  <td>500-1000 ml
<tr><td>xylene cyanole FF <td>agarose gel electrophoresis sample prep component
  <td>Fisher <td>AcrosOrganics xylene cyanole FF <td>AC61192-0100 <td>$45/10 g
  <td>example <td>5 g
<tr><td>bromophenol blue
     <td>agarose gel electrophoresis sample prep component
  <td>Fisher <td>AcrosOrganics bromophenol blue ACS <td>AC40314-0050 <td>$16/5 g
  <td>example  <td>5 g
<tr><td>agarose gel extraction ki <td>extracting DNA from agarose gels (can be combined with PCR cleanup kit)
  <td>Axygen Biosciences <td>AxyPrep DNA Gel Extraction Kit <td>AP-GX-50 <td>$66/50 preps
  <td>example <td>20 preps

<tr><td class="header" colspan="8">Blue-White (LacZ) Screening/Maintenance Cloning
<tr><td>pUC19 plasmid
     <td>vector for maintenance of cloned inserts
  <td>NEB <td>pUC19  <td>N3041S  <td>$61/50 &micro;g
  <td>example <td>COLLEAGUE
<tr><td><i>E coli</i> DH5&alpha;
     <td>vector for maintenance of cloned inserts
  <td>NEB <td>NEB 5-alpha competent  <td>C2988J  <td>$62/6 &times; 400 &micro;l/tube
  <td>example <td>COLLEAGUE
<tr><td>5-Bromo-4-chloro-3-indolyl-b-D-galactoside (X-gal, BCIG) [CAS 7240-90-6]
     <td>screening for positive clones
  <td>Fisher <td>AcrosOrg 99% BCIP <td>AC22777-1000 <td>$78/100 mg
  <td>example <td>25 mg
<tr><td>isopropyl-1-thiogalactopyranoside (IPTG) [CAS  367-93-1]
     <td>&beta;-galactosidase induction
  <td>Fisher <td>IPTG, dioxane-free <td>BP1755-1 <td>$56/1 g
  <td>example  <td>500 mg

<tr><td class="header" colspan="8">PCR, including PCR-Based Cloning
<tr><td>high fidelity and processive DNA polymerase<td>OE-PCR-based cloning
  <td>NEB <td>Phusion&reg; High-Fidelity PCR kit <td>E0553S (50 rxn) <td>$65
  <td>required <td>1
<tr><td><i>Taq</i> DNA polymerase + components<td>General amplification
  <td>NEB <td><i>Taq</i> DNA Polymerase with Standard Mg-Free <i>Taq</i> Buffer
  <td>M0320S <td>$58/400 U <td>required <td>1
<tr><td>Magnesium Chloride 10-20&times; stock  <td>supplement to PCR master mix
  <td>NEB <td>Magnesium Chloride (MgCl<sub>2</sub>) Solution
  <td>B9021S <td>$16/4 &times; 1.5 ml <td>required <td>1
<tr><td>PCR Clean-Up Kit  <td>required for downstream reactions (restriction cutting, etc)
  <td>GenScript  <td>QuickClean II PCR Extraction Kit <td>L00419-50
  <td>$45/kit  <td>required <td>1

<tr><td class="header" colspan="8">DNA Modifying Enzymes (Cutting, Ligating, etc)
<tr><td>T4 DNA ligase
  <td>joining DNA fragments/vectors
  <td>NEB <td>T4 DNA Ligase, 400&nbsp;U/&micro;l <td>M0202S  <td>$63 <td>required
   <td style="width:6em;">1 (20,000&nbsp;U)
<tr><td><i>Bam</i>HI restriction enzyme
  <td>LHBs restriction analysis
  <td>NEB <td><i>Bam</i>HI-HF, 20&nbsp;U/&micro;l <td>R3136S  <td>$54 <td>required  <td>1 (10,000 U)
<tr><td><i>Bgl</i>I restriction enzyme
  <td>LHBs restriction analysis
  <td>NEB <td><i>Bgl</i>I, 10&nbsp;U/&micro;l <td>R0143S  <td>$54 <td>required  <td>1 (2000 U)
<tr><td><i>Mlu</i>I restriction enzyme
  <td>B. subtilis cloning
  <td>NEB <td><i>Mlu</i>I, 10&nbsp;U/&micro;l <td>R0198S  <td>$59 <td>required  <td>1 (1000 U)
<tr><td><i>Eco</i>52I restriction enzyme (isoschizomer: <i>Eag</i>I)
  <td>B. subtilis cloning
  <td>NEB <td><i>Eag</i>I-HF, 20&nbsp;U/&micro;l <td>R3505S  <td>$59 <td>required  <td>1 (500 U)
<tr><td><i>Not</i>I restriction enzyme
  <td>LHBs restriction analysis
  <td>NEB <td><i>Not</i>I-HF, 20&nbsp;U/&micro;l <td>R3189S  <td>$67 <td>required  <td>1 (500&nbsp;U)
<tr><td><i>Pvu</i>II restriction enzyme
  <td>LHBs restriction analysis
  <td>NEB <td><i>Pvu</i>II-HF, 20&nbsp;U/&micro;l <td>R3151S  <td>$54 <td>required  <td>1 (5000&nbsp;U)
<tr><td><i>Pst</i>I restriction enzyme
  <td>LHBs restriction analysis
  <td>NEB <td><i>Pst</i>I-HF, 20&nbsp;U/&micro;l <td>R3140S  <td>$59 <td>required  <td>1 (10,000&nbsp;U)
<tr><td><i>Sph</i>I restriction enzyme
  <td>LHBs restriction analysis
  <td>NEB <td><i>Sph</i>I-HF, 20&nbsp;U/&micro;l <td>R3182S  <td>$62 <td>required  <td>1 (500&nbsp;U)
<tr><td><i>Alw</i>NI restriction enzyme
  <td>characterizing pLHBs-UC19 plasmid construct
  <td>NEB <td><i>Alw</i>NI, 10&nbsp;U/&micro;l <td>R0514S  <td>$62 <td>required  <td>1 (500&nbsp;U)
<tr><td><i>Ssp</i>I restriction enzyme
  <td>characterizing pLHBs-UC19 plasmid construct
  <td>NEB <td><i>Ssp</i>I-HF, 20&nbsp;U/&micro;l <td>R3132S  <td>$64 <td>required  <td>1 (1000&nbsp;U)
<tr><td><i>Dpn</i>I restriction enzyme
  <td>digestion of methylated (naturally derived) plasmids (does not digest PCR products)
  <td>NEB <td><i>Dpn</i>I, 20&nbsp;U/&micro;l <td>R0176S  <td>$59 <td>required  <td>1 (1000&nbsp;U)
<tr><td><i>Eco</i>RI restriction enzyme
  <td>Backup for Cut &amp; Ligate to pUC19
  <td>NEB <td><i>Eco</i>RI-HF, 20&nbsp;U/&micro;l <td>R3101S  <td>$54 <td>required  <td>1 (10,000&nbsp;U)
<tr><td><i>Xba</i>I restriction enzyme
  <td>Backup for Cut &amp; Ligate to pUC19
  <td>NEB <td><i>Xba</i>I, 20&nbsp;U/&micro;l <td>R0145S  <td>$64 <td>required  <td>1 (3000&nbsp;U)

<tr><td class="header" colspan="8">Recombinant Protein Expression--B. subtilis
<tr><td><i>B. subtilis</i> expression system
  <td>recombinant protein host and vector system
  <td>Takara/Clontech <td><i>B. subtilis</i> Secretory Protein Expression System
  <td>3380  <td> <td>required  <td>1 (10 reactions)


<tr><td class="header" colspan="8">Synthetic Oligonucleotides/Primers
<tr><td rowspan="2">LHBs Three Fragment PCR Assembly oligos + primers <td>in vitro gene synthesis of LHBs
  <td>Sigma <td>Oligo Custom Service <td>14-487-65 <td>1  <td>  <td>required
<tr><td colspan="7" style="white-space:pre;font-family:monospace;">
   LHBs Fragment 1 oligos + primers  (33 oligos, 892 nt total)
      1 R0     16  ATgacaaacgaacgcg
      2 F0     27  cgcgttcgtttgtcATGGGCGGCTGGA
      3 R16    24  CGCGGTTTGCTGCTCCAGCCGCCC
      4 F27    27  GCAGCAAACCGCGCAAAGGCATGGGCA
      5 R40    31  CGGAACGCTCAGATTTGTGCCCATGCCTTTG
      6 F54    30  CAAATCTGAGCGTTCCGAACCCGCTGGGCT
      7 R71    32  AGCTGATGATCCGGAAAAAAGCCCAGCGGGTT
      8 F84    33  TTTTTCCGGATCATCAGCTGGATCCGGCGTTTC
      9 R103   30  TTGTTGCTATTCGCGCGAAACGCCGGATCC
     10 F117   33  GCGCGAATAGCAACAATCCGGATTGGGATTTTA
     11 R133   36  CAATGATCTTTATTCGGGTTAAAATCCCAATCCGGA
     12 F150   32  ACCCGAATAAAGATCATTGGCCGGATGCGAAT
     13 R169   28  GCCAACGCCAACTTTATTCGCATCCGGC
     14 F182   27  AAAGTTGGCGTTGGCGCGTTTGGCCCG
     15 R197   26  GGCGGTGTAAAGCCCGGGCCAAACGC
     16 F209   26  GGCTTTACACCGCCGCATGGCGGCCT
     17 R223   25  GCTCCAGCCCAGCAGGCCGCCATGC
     18 F235   24  GCTGGGCTGGAGCCCGCAGGCGCA
     19 R248   27  TGTTGTCAGAATGCCCTGCGCCTGCGG
     20 F259   27  GGGCATTCTGACAACACTGCCGGCGGA
     21 R275   22  CCGGCGGCGGATCCGCCGGCAG
     22 F286   26  TCCGCCGCCGGCGAGCACAAATCGCC
     23 R297   26  GCGGCCGCTCTGGCGATTTGTGCTCG
     24 F312   24  AGAGCGGCCGCCAGCCGACACCGA
     25 R323   25  GCGGCGGGCTAATCGGTGTCGGCTG
     26 F336   27  TTAGCCCGCCGCTGCGCGATACACATC
     27 R348   27  GCATCGCCTGCGGATGTGTATCGCGCA
     28 F363   29  CGCAGGCGATGCAGTGGAATAGCACAACA
     29 R375   32  CAGCGCCTGATGAAATGTTGTGCTATTCCACT
     30 F392   28  TTTCATCAGGCGCTGCAGGATCCGCGCG
     31 R407   25  ACAGGCCGCGAACGCGCGGATCCTG
     32 F420   26  TTCGCGGCCTGTATcactggtgcccc
     33 F432   14  ggggcaccagtgAT

   LHBs Fragment 2 oligos + primers  (32 oligos, 866 nt total)
      1 R0     12  GCCCGCCGGAAA
      2 F0     24  TTTCCGGCGGGCGGCAGCAGCAGC
      3 R12    27  CGGATTAACTGTGCCGCTGCTGCTGCC
      4 F24    30  GGCACAGTTAATCCGGTTCCGACAATTGCG
      5 R39    30  GCTGCTAATCAGGCTCGCAATTGTCGGAAC
      6 F54    30  AGCCTGATTAGCAGCATTTTTAGCCGCACA
      7 R69    26  GCCGGATCGCCTGTGCGGCTAAAAAT
      8 F84    29  GGCGATCCGGCGCCGAATATGGAAAATAT
      9 R95    33  AGAAAGCCGCTTGTAATATTTTCCATATTCGGC
     10 F113   26  TACAAGCGGCTTTCTGGGCCCGCTGC
     11 R128   24  GCCTGCAGAACCAGCAGCGGGCCC
     12 F139   28  TGGTTCTGCAGGCGGGCTTTTTTCTGCT
     13 R152   30  GTCAGAATGCGTGTCAGCAGAAAAAAGCCC
     14 F167   29  GACACGCATTCTGACAATTCCGCAGAGCC
     15 R182   28  CCACCAGCTATCCAGGCTCTGCGGAATT
     16 F196   31  TGGATAGCTGGTGGACAAGCCTGAATTTTCT
     17 R210   28  GTGTGCCGCCCAGAAAATTCAGGCTTGT
     18 F227   24  GGGCGGCACACCGGTTTGCCTGGG
     19 R238   29  GCTCTGGCTATTCTGGCCCAGGCAAACCG
     20 F251   31  CCAGAATAGCCAGAGCCCGACAAGCAATCAT
     21 R267   27  GCTTGTCGGGCTATGATTGCTTGTCGG
     22 F282   24  AGCCCGACAAGCTGCCCGCCGATT
     23 R294   24  ATAGCCCGGGCAAATCGGCGGGCA
     24 F306   26  TGCCCGGGCTATCGCTGGATGTGCCT
     25 R318   28  TAATAAAGCGGCGCAGGCACATCCAGCG
     26 F332   33  GCGCCGCTTTATTATTTTTCTGTTTATTCTGCT
     27 R346   33  AAATCAGGCACAGCAGCAGAATAAACAGAAAAA
     28 F365   29  GCTGTGCCTGATTTTTCTGCTGGTTCTGC
     29 R379   30  TGCCCTGATAATCCAGCAGAACCAGCAGAA
     30 F394   27  TGGATTATCAGGGCATGCTGCCGGTTc
     31 R409   24  acggcgcactacgAACCGGCAGCA
     32 R421   12  gtagtgcgccgt

   LHBs Fragment 3 oligos + primers  (28 oligos, 752 nt total)
      1 R0     14  GAATCAGCGGGCAt
      2 F0     25  aTGCCCGCTGATTCCGGGCAGCAGC
      3 R14    25  CCTGTGCTTGTTGTGCTGCTGCCCG
      4 F25    27  ACAACAAGCACAGGCCCGTGCAAAACA
      5 R39    26  CCGGTGTTGTGCATGTTTTGCACGGG
      6 F52    24  TGCACAACACCGGCGCAGGGCACA
      7 R65    25  CTCGGAAACATGCTTGTGCCCTGCG
      8 F76    27  AGCATGTTTCCGAGCTGCTGCTGCACA
      9 R90    27  CCATCGCTCGGTTTTGTGCAGCAGCAG
     10 F103   29  AAACCGAGCGATGGCAATTGCACATGCAT
     11 R117   28  GCTCGGAATCGGAATGCATGTGCAATTG
     12 F132   25  TCCGATTCCGAGCAGCTGGGCGTTT
     13 R145   28  CCCACAGATATTTCGCAAACGCCCAGCT
     14 F157   28  GCGAAATATCTGTGGGAATGGGCGAGCG
     15 R173   26  CCAGCTAAAGCGAACGCTCGCCCATT
     16 F185   27  TTCGCTTTAGCTGGCTGAGCCTGCTGG
     17 R199   28  ACTGAACAAACGGAACCAGCAGGCTCAG
     18 F212   29  TTCCGTTTGTTCAGTGGTTTGTTGGCCTG
     19 R227   27  AAACTGTCGGGCTCAGGCCAACAAACC
     20 F241   27  AGCCCGACAGTTTGGCTGAGCGTTATT
     21 R254   31  CAATACCACATCATCCAAATAACGCTCAGCC
     22 F268   28  TGGATGATGTGGTATTGGGGCCCGAGCC
     23 R285   30  GGGCTCAGAATATTATACAGGCTCGGGCCC
     24 F296   34  TGTATAATATTCTGAGCCCGTTTATTCCGCTGCT
     25 R315   31  GCAAAAAAAAATCGGCAGCAGCGGAATAAAC
     26 F330   34  GCCGATTTTTTTTTGCCTGTGGGTTTATATTcct
     27 R346   30  cctaccggctgcaggAATATAAACCCACAG
     28 R364   12  gcagccggtagg

Total  93 oligos, 2510 nt
<tr><td rowspan="2">LHBs Completion Overlap Oligos <td>in vitro gene synthesis of LHBs
  <td>Sigma <td>Oligo Custom Service <td>14-487-65 <td>1  <td>   <td>required
<tr><td colspan="7" style="white-space:pre;font-family:monospace;">
    L1L2fo:  5&prime;-GCGCGTTCGCGGCCTGTATTTTCCGGCGGGCGGCAG-3&prime;
    L1L2ro:  5&prime;-CTGCCGCCCGCCGGAAAATACAGGCCGCGAACGCGC-3&prime;
    L2L3fo:  5&prime;-GATTATCAGGGCATGCTGCCGGTTTGCCCGCTGATTCCGGGC-3&prime;
    L2L3ro:  5&prime;-GCCCGGAATCAGCGGGCAAACCGGCAGCATGCCCTGATAATC-3&prime;
Total 4 oligos, 156 nt
<tr><td rowspan="2">Message Sense/Anti-Sense Primers <td>PCR amplification of SHBs, MHBs, LHBs
  <td>Sigma <td>Oligo Custom Service <td>14-487-65 <td>1  <td>  <td>required
<tr><td colspan="7" style="white-space:pre;font-family:monospace;">
    LHBsfp:  5&prime;-ATGGGCGGCTGGAGCAGC-3&prime;
    LHBsrp:  5&prime;-AATATAAACCCACAGGCAAAAAAAAATCGGCAG-3&prime;
    MHBsfp:  5&prime;-ATGCAGTGGAATAGCACAACATTTCATCAGG-3&prime;
    SHBsfp:  5&prime;-ATGGAAAATATTACAAGCGGCTTTCTGGGC-3&prime;
Total 4 oligos,  112 nt
<tr><td rowspan="2">pUC19 OE-PCR-cloning primers <td>PCR-based cloning into pUC19
  <td>Sigma <td>Oligo Custom Service <td>14-487-65 <td>1  <td> <td>required
<tr><td colspan="7" style="white-space:pre;font-family:monospace;">


    5&prime;-TGTAAAACGACGGCCAGTGAATTCGAGCTCGGTACCCGGGATGGGCGGCTGGAGCAGC-3&prime;
                                               |


[name: OE-UC19-MHBs-fp]
OE-PCR-pUC19-LHBs sense (forward) primer (complements anti-sense)
    5&prime;-TGTAAAACGACGGCCAGTGAATTCGAGCTCGGTACCCGGGATGCAGTGGAATAGC-
                                               |
      -ACAACATTTCATCAGG-3&prime;


</table>



<!-- hostsite bottom logo here -->


</body>
</html>