<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01//EN"
   "http://www.w3.org/TR/html4/strict.dtd">
<html>
<head>
<title>Cytogenetics and Population Genetics:  Genetics 100B</title>
<meta http-equiv="Content-Type" content="text/html; charset=iso-8859-1">
<link href="../style/std.css" type="text/css" rel="stylesheet">
<link href="ucdlectures.css" type="text/css" rel="stylesheet">
<style type="text/css">
  th.empty {background:none;border:none}
  span.punnett-fraction {font:normal 80%  "Times New Roman","Times Roman",Times,serif;}
  table.punnett {border:1px solid black;margin:0 auto 1em auto;}
  table.punnett caption {font:normal 75% Bookman,Palatino,Century,serif;
    color:white;background-color:#444;}
  table.punnett th, table.punnett td {text-align:center;}
  th.punnett-sex {border:none;background-color:#e8e8ff;
			font:bold 75% monospace!important; text-align:center;
      vertical-align:middle;}

  .no-spacing li {margin-top:0;}
</style>
</head>
<body>

<p id="subtitle">
Genetics 100B

<p id="title">
Cytogenetics
<br>&amp;
<br>Population Genetics


<p>
The following is a transcription of notes taken by S. M. Halloran
when he was an undergraduate student at the University of California
at Davis.
<p>
The lectures were given by Professors Prout and Turelli.  The dates of the
lecture are noted.

<p class="date">
8 Jan 1980

<p>
Course is graded on two midterm examination worth 30% each and a final
examination counting for 40%.

<h1>Cytogenetics</h1>
<p>
<b>Cytogenetics</b> is a study of the variation of chromosome number and 
structure.  In the figure below is a representation of three paired
chromosomes with their centromeres shown.
<img src="imagesMolPopGeneticsHTML/PairedChromosomes.gif" class="img-center"
  alt="representation of paired chromosomes showing centromeres">
<p>
The <b>diploid</b> (left or top) and <b>gametic</b> (right or below) 
representations would be as in the figures below.  Note that gametic might be 
called <b>haploid</b>, if the gametic and haploid are the same.
<p>
<img src="imagesMolPopGeneticsHTML/DiploidSymbols.gif" 
   style="width:49%;vertical-align:middle;"
  alt="diploid representation of paired chromosomes in figure above">
<img src="imagesMolPopGeneticsHTML/HaploidSymbols.gif"
   style="width:49%;vertical-align:middle;"
  alt="haploid representation of paired chromosomes in figure above">
<p>
Variation in chromosome number is represented in the figure below, where
the set of chromosome numbers can be a single set of unique chromsomes, but
then duplicated or triplicated.  Or some of the chromosomes can be triplicated
and other duplicated.
<img src="imagesMolPopGeneticsHTML/ChromosomeNumberVariation.gif"
   class="img-center"
  alt="variations in chromosome number">
<p>
Variation in chromosome structure is represented in the figure below.
Segments or parts of chromosomes might be re-arranged in various ways.
Segments of one chromosome might be moved to another chromosome.
<img src="imagesMolPopGeneticsHTML/ChromosomeStructureVariation.gif"
   class="img-center"
  alt="variations in chromosome structure">
<p>
Variations in chromosome structure can and often arise in the processes of
mitotis and meiosis, and cytogenetics is concerned with analyzing these
variations on a chromosome level or scale.

<h2>Number Variation</h2>
<p>
<img src="imagesMolPopGeneticsHTML/BananaSeed.gif"
   class="img-left"
  alt="banana seed with its chromosomes">
<b>Polyploidy</b> is a term given to what the number is of a basic set of 
chromosomes, and important the ability to create multiple integer numbers of the 
basic set naturally or by engineering of some type (through plant breeding or by 
biochemical or molecular manipulation).
<p>
Consider the banana, which has a haploid set of chromosomes totaling nine, 
(<i>x</i>) = 9.  These chromosomes provide the minimum number of traits 
necessary to express a banana genotypically.  The value (<i>x</i>) is the 
<i>basic number</i> of chromosomes, which are 9.  The value (2<i>x</i>) would be 
18 chromosomes.  The banana seed actually has a value of (3<i>x</i>) or 27 
chromosomes.
<p>
The commercial strawberry has (8<i>x</i>) = 56 chromosomes, while its wild 
counterpart actually has (2<i>x</i>) = 14 chromosomes, thanks to the 
engineering of geneticists.  Polyploid forms of plants comes naturally 
and can be observed in about 1 in 3 higher plants.  This clearly has some
evolutionary advantage.
<p>
Two mechanisms may be responsible for polyploidy: <b>unreduced gametes</b> and
<b>somatic cell doubling</b>.

<h3>Unreduced Gametes</h3>
<p>
<img src="imagesMolPopGeneticsHTML/GameticReduction.gif"
   class="img-center"
  alt="mechanism of gametic reduction by an abnormal 1st meiotic division">
Using the banana as an example, its reproductive organs contain somatic cells 
that undergo gametogenesis.  Normally gametogenesis in such cells produce cells 
having a haploid character, but if there is no assortment (see <q>abnormal</q> 
in the figure), then the gamete formed from the cell still retains diploid 
character.  Note that cell division occurs, but the other cell is without 
chromosomes and will die quickly. If the diploid gamete fuses with a gamete that 
formed normally (i.e., is haploid), the zygote is triploid.  If the triploid 
zygote is viable, it produces an organism retaining that character.  Note that
cells that are 2<i>n</i> for all chromosomes are diploid, and those with 3<i>n</i> for all
chromosomes are triploid.  Diploid and triploid cells or organims can be 
described as <b>euploid</b> (in which all chromosome numbers are an exact multiple
of the monoploid state <i>n</i>).
<p>
When the triploid organism matures and produces its own gametes, the chromosomes 
can assort abnormally, with a mixture of diploidy in some chromosomes and 
haploidy in for other chromosomes.  This condition is <b>aneuploidy</b> since 
some of the chromosomes in a gamete are 2<i>n</i> and others <i>n</i>.  The 
gametes are referred to as <b>aneuploids</b>.  It might even be possible that 
one gamete has none of one type of chromosome, and its sister gamete has all 
three chromosomes.
<p>
<img src="imagesMolPopGeneticsHTML/Aneuploidy.gif"
   class="img-left"
  alt="a gamete with diploidy in two chromosomes and haploidy in the third">
The figure above shows three chromsomes forming the genome.  The haploid state 
is just one copy of a chromosome, while the diploid is two copies, and the 
triploid is three copies.  In forming gametes from a triploid organism, suppose 
that the only possible assortment is one copy of a chromosome to one gamete and 
two copies to the sister gamete.  Thus of the two gametes that form, the probability
is &frac12; that a gamete gets one or two copies of a chromosome.  For a 
three-chromosome genome, the probability of a gamete have diploidy for two 
of the chromosomes and haploidy for the third chromosome would be 
&frac12; &times; &frac12; &times; &frac12;  = 1/8.  For a banana having
a 9-chromosome genome, the probability of having any particular karyotype will
be (&frac12;)<sup><i>m</i></sup>, where <i>m</i> is the number of chromosomes.
Thus the probability that a gamete has two of chromosomes #1, #4, #5, #6 and 
one of chromosomes #2, #3, #7, #8, and #9 is (&frac12;)<sup>9</sup> =
1/512.
<p>
<img src="imagesMolPopGeneticsHTML/AneuploidyXEuploidy.gif"
   class="img-center"
  alt="crossing an aneuploid gamete with euploid gamete">
What is the significance of aneuploidy?  What happens when aneuploidy gametes
fuse with other gametes?  Is the organism viable?  Consider the zygote
of an aneuploid gamete and euploid (haploid) gamete (see figure immediate above).
The observation in nature is that metabolic or structural disorders occur.
In plants formation of aneuploid gametes does not continue to the formation of
mature pollen grains.  And it is odd-numbered euploids (triploids, pentaploids,
heptaploids) that would yield aneuploid gametes by necessity.

<h3>Somatic Cell Doubling</h3>
<p>
In gametic reduction, the process of <b>meiosis</b> leads to the division in which 
ploidy is reduced by half, and gametes form which then join to result in
either the euploid or aneuploid state.
<p>
<img src="imagesMolPopGeneticsHTML/SomaticCellDoubling.gif"
   class="img-right"
  alt="somatic cell doubling as a defect of mitosis">
With somatic cell doubling, changes in ploidy no longer involve defects in 
meiosis, but rather a defect in <b>mitosis</b>.  In the figure, a 3-chromosome 
genome with each chromosome pair named <i>a</i>, <i>b</i> and <i>c</i> lines up 
on the mitotic axis (metaphase plate).  Normally the sisters of the pairs move 
one to each daughter cell. But in the figure both pairs of chromosomes <i>a</i> 
and <i>b</i> move to a daughter, while chromosome <i>c</i> separated normally to 
both daughter cells.  Both gametes are technically aneuploid, with one gamete 
showing tetraploidy for chromosomes <i>a</i> and <i>b</i>.
<p>
<b>Colchicine</b> is a plant alkaloid which interacts with <b>microtubules</b>. 
Microtubules are filamentous structures that form inside eukaryotic cells and 
are responsible for many functions of the cells, such as motility.  Microtubules 
are a major part of the cytoskeleton.  In addition, microtubules are required 
for mitotis, which is the process by which a cell properly replicates its genome 
(DNA) and then distributes one of the two copies to daughter cells during cell 
division as part of the organism's survival, or growth and development. 
Microtubules extend from the asters on either side of the chromosomes aligned on 
the metaphase plate, and the attach to the centromeres of each chromosome.  If 
anything affects proper microtubule function, mitosis might proceed abnormally. 
<img src="imagesMolPopGeneticsHTML/Tetraploidy.gif"
   class="img-left"
  alt="a tetraploid genome respresented">
An abnormal process in particular could lead to both of the homologous 
chromosomes going to one daughter cell (the other getting none, if another 
daughter cell even forms). Colchicine stops microtubules from functioning 
properly, and so a diploid cell forms a tetraploid cell (genome replication 
without formation of daughter cells).  Colchicine applied to meristematic 
tissues (root cells) produce cells showing <b>eupolyploidy</b>.
<p>
<img src="imagesMolPopGeneticsHTML/SomaticCellDoublingMechanisms.gif"
   class="img-left"
  alt="somatic cell doubling mechanism">
Given a tetraploid having chromosomes <i>a</i>, <i>b</i>, and <i>c</i>, suppose 
its cell undergoes meiosis.  In the first prophase, recombination occurs for 
each of the four copies of the homologous chromosomes.  Now a 
<i>quadrivalent</i>, in which the chromosomes will go into one daugher cell, and 
not the other.  Yet another possiblity is a trivalent and a univalent in the
homologous chromosomes.  Or yet another is a bivalent formed, with each daughter
cell to get equal content of that homologous chromosome.  What is often observed
in tetraploids is an alignment that leads to diploid gametes for all chromosomes.

<p class="date">
10 Jan 1980

<p>
Suppose in looking at the fruit of certain dicotyledons (bananas and 
strawberries), the question arises why large seeds have not been discovered. 
These would be polyploids.  If large seeds do not form, it is likely because it 
forms abnormal fruit and such fruit would be culled out.

<table class="td-center" style="float:right;margin:0 0 1em 2em;">
<caption>Euploidy</caption>
<tr><td colspan="2"> <th colspan="2">Gametes
<tr><td colspan="2" class="left">1. Monoploidy  <td>X <td> 0
<tr><td colspan="2" class="left">2. Diploidy    <td>X <td> X
<tr><td colspan="2" class="left">3. Polyploidy  <td colspan="2">
<tr><td><td class="left"><i>a.</i> odd          <td colspan="2">
<tr><td><td class="left"><i>b.</i> even         <td colspan="2">&frac12;<i>p</i>X
</table>
<p>
Consider the watermelon, which is reproduced sexually, and not asexually
like the banana.  If a 4X watermelon, which is stable (viable), produces gametes
of 2X, and a 2X watermelon, which produces 1X gametes, are crossed, a 3X
triploid is expected.  Indeed it forms, but the 3X watermelon produces no
seeds.  Since seedless watermelons are desired by consumers, it makes sense
for plant breeders than to cross tetraploids and diploids to produce the 
triploids, which are sterile but delicious.
<p>
Recall that in the strawberry, a commercial type is 8X and the wild type
is 2X.  Their gametes should thus be 4X and X for the commercial and wild
type respectively.  If they are crossed, 5X progeny should be the result.
This odd ploidy should again produce gametes that will form no seeds.
<p>
Some important new terms should be learned.  <b>Autopolyploidy</b> is polyploidy 
that occurs from within a species by a process such as gametic reduction in 
meiosis or by somatic cell doubling in mitosis.  Thus a 2X cell or organism
forms a 4X form of itself by these processes, or a 4X forms an 8X form of
itself.
<p>
In contrast, <b>allopolyploidy</b> results from an interspecies crossing,
such as an 6X species with 3X gametes joining the 1X gametes from a 2X species
to form a 4X (tetraploid) that is a new species but with homologous chromosomes
of the two others.
<p>
In the figure below, allopolyploidy is demonstrated.  Species A and B formed 
from an ancestor as their genomes diverged.  They retain the sets of homologous
chromosomes (<i>a</i> belonging to A, <i>b</i> belonging to B).  And so a 
diploid gamete of A is crossed with diploid gamete of B to form a tetraploid
progeny which can be described as AABB.  This is effectively a new species.
<img src="imagesMolPopGeneticsHTML/NewSpeciesGeneration.gif"
   class="img-center"
  alt="how a new species is generated">
<p>
<img src="imagesMolPopGeneticsHTML/HaploidNewSpecies.gif"
   class="img-right"
  alt="generating a species that is haploid for all chromosomes">
Suppose the new species resulted from an A &times; B crossing, which are haploid
gametes.  The new species would be symbolized cytogenetically as the figure
on the right.  Gametogenesis by this progeny will almost certainly result in
aneuploidy.  If the progeny, composed of haploid chromsomes, underwent something
like somatic cell doubling, the species would then be diploid, but with a larger
genome.  Such a diploid species would then produce euploid gametes and 
the successive progeny are euploid as well.
<p>
Four polyploidy experiments are described achieved by allopolyploidy.
<p>
<img src="imagesMolPopGeneticsHTML/ColchicineAllopolyploidy.gif"
   class="img-left"
  alt="how to induce polyploidy from interspecies crossing">
The first is a cross between a cabbage (<i>Brassica olaracea</i>) and radish
(<i>Raphanus sativa</i>).  Both are from the same family but different genera.
<p>
The figure at left shows the scheme.  The cabbage and raddish both have 9 
chromosomes (2X = 18).  They generate hapoid gametes (R and C) which are then
crossed.  But the chromosomes are in the haploid state in the progeny.  So 
<img src="imagesMolPopGeneticsHTML/rabbage-or-caddish.gif"
   class="img-right"
  alt="conceptual phenotype of a radish and cabbage cross">
a colchicine-induced somatic cell doubling is done to produce polyploidy 
and achieve the diploid state, which will produce gametes that provide viable
progeny:  <i>Raphanobrassica</i>, or the <q>rabbage.</q>
<p>
In the second experiment, tobacco (<i>Nicotiana</i>) has three species. The 
species <i>N. tomentosa</i> has a diploid 2X = 24, and its genome is symbolized 
TT.  <i>N. sylvestrus</i> also has diploid 2X = 24 and is symbolized SS.   The 
species <i>N. tobaccum</i> is actually the cultivated variety, and it shows 48 
chromosomes.  Could its representation be TTSS, that is, a diploid cross of the 
other two species?
<p>

To test this, a TT &times; TTSS cross should produce 12 bivalent and 12 
univalents in the progeny karyotype, representing a TTS genome, as shown in the 
figure at left below.  
<img src="imagesMolPopGeneticsHTML/TobaccoAneuploidProgeny.gif"
   class="img-left"
  alt="test tobacco cross of TT x TTSS to make TTS">
<p>
With a SS &times; TTSS cross, a TSS genome.  And a 
TT &times; SS should produce the diploid TS genome, with no bivalents and
24 univalents.  All progeny should be sterile.  With the TS genome, a somatic
cell doubling should produce TTSS.  If its phenotype is like <i>N. tobaccum</i>,
then it suggests that <i>N tobaccum</i> arose in this way, as an allopolyploid
cross between <i>N tomentosa</i> and <i>N. sylvestrus</i>.
<p>
In a third experiment, the genus <i>Tragopogan</i> (goat's beard) includes
three species:  <i>T. dubius, T. porrifolius,</i> and <i>T. pratensis</i>.
These are diploid types whose flower colors are pale yellow, purple, and 
chrome yellow, respectively.  If <i>T. dubius</i> and <i>T. pratensis</i> 
have their diploid gametes crossed, the 4X progeny can then be used to create
2X progeny which have phenotypes of <i>T. dubius</i> and <i>T. porrifolius</i>,
indicating how much of the genome is shared.
<p>
Finally in a fourth experiment, cord grass is composed of an American type
(2X = 70) and a European type (2X = 56).  The alloploid from this is XY = 126.
It does not possess the same mulitple sets of chromosomes (homologues) and 
so these species of cord grass are <b>amphiploids</b>.

<p class="date">
15 Jan 1980

<p>
In the previous lecture there was a discussion of two species with genomes
AA and BB.
<img src="imagesMolPopGeneticsHTML/AllotetraploidyReview.gif"
   class="img-left"
  alt="allopolyploidy review">
<p>
The classic case, shown in figure at left, is to cross the haploid gametes of 
two species, and then take the progeny with its karyotype of univalents and 
induce diploid in the somatic cell (using antimitotics).
<p>
This method can be repeated to add more chromosomes to the genome from different 
species (see figure right).
<img src="imagesMolPopGeneticsHTML/AddingMoreAllopolyploidy.gif"
   class="img-right"
  alt="adding more allopolyploidy">
Genomes A and B were combined and formed into a diploid state (AABB), and their 
gametes combined with a CC diploid gamete to form again a haploid species (ABC) 
which was then made diploid (AABBCC) by somatic cell doubling.
<p>
An allohexaploid in bread wheat was generated in this way.  Suppose the
original wheat is genome AA (X = 7, 2X = 14).  This is combiined with other
forms of wheat (genomes BB and CC) with X = 7, to produce AABBCC (2X = 42).
<p>
Polyploidy is generated quite naturally by a species under evolutionary
control or stress to adapt to a different environment, although the result is
that chromosome structure changes very little in order to make the adaptation.
In the figure below, ancestral genome C evolves divergently to form genomes
X and Y, which retain the same number and generally the structure of the
chromosomes of genome C.  X and Y gametes cross.  One of the chromosomes 
retain homology, so these line up on metaphase in mitosis, while the others
do not.  The XY will show four univalents and one bivalent.
<img src="imagesMolPopGeneticsHTML/EvolutionaryPolyploidy.gif"
   class="img-center"
  alt="evolutionary polyploidy">

<h2>Genetics of the Polyploid</h2>
<p>
<img src="imagesMolPopGeneticsHTML/PolyploidMeiosis.gif"
   class="img-center"
  alt="meiosis in the polyploid">
Suppose a self cross is performed with heterozygous diploids.  This means that 
for locus <b>A</b> on a chromosome, the <b>A</b><i>a</i> &times; 
<b>A</b><i>a</i> cross is done.  The progeny have an expected 3:1 phenotypic
ratio for diploids.  But for tetraploids, their haploid gametes produce six
possible fertile diploid types (seen in figure above: note the subscript
refers to the particular distinguishing chromosome, even though the allele
is identical).  All gametes compositions are equally probable, thus a probability
of 1/6.

<table class="punnett">
<caption>Punnett Square <br> Tetraploid Selfcross</caption>
<tr><th colspan="2" class="empty"><th colspan="3" class="punnett-sex">male
<tr><th colspan="2" class="empty">
  <th>AA <br><span class="punnett-fraction">1/6</span>
  <th>Aa <br><span class="punnett-fraction">4/6</span>
  <th>aa <br><span class="punnett-fraction">1/6</span>
<tr><th rowspan="3" class="punnett-sex" style="line-height:0.9em;padding:auto 0;">
   f<br>e<br>m<br>a<br>l<br>e
  <th>AA <br><span class="punnett-fraction">1/6</span>
  <td>AAAA <br><span class="punnett-fraction">1/36</span>
  <td>AAAa <br><span class="punnett-fraction">4/36</span>
  <td>AAaa <br><span class="punnett-fraction">1/36</span>
<tr><th>Aa <br><span class="punnett-fraction">4/6</span>
  <td>AaAA <br><span class="punnett-fraction">4/36</span>
  <td>AaAa <br><span class="punnett-fraction">16/36</span>
  <td>Aaaa <br><span class="punnett-fraction">4/36</span>
<tr><th>aa <br><span class="punnett-fraction">1/6</span>
  <td>aaAA <br><span class="punnett-fraction">1/36</span>
  <td>aaAa <br><span class="punnett-fraction">4/36</span>
  <td>aaaa <br><span class="punnett-fraction">1/36</span>
</table>
<p>
If allele <b>A</b> is dominant to <b><i>a</i></b>, the phenotype ratio will be
35:1 (or 35 in 36) for tetraploid heterozygote crosses compared to the 3:1 (or 
27 in 36) for the crossing of diploids.
<p>
The above cross is an autotetraploid.  For an allotetraploid, species X and Y
have two different genomes but have a similar gene A.  But other genes are
not similarly ordered or sequenced, and so no pairing occurs.
<img src="imagesMolPopGeneticsHTML/PolyploidSegregation.gif"
   class="img-right"
  alt="allopolyploidy without pairing">
<p>
When alloploids are formed, the homologous chromosomes must group in order to 
get the alleles of gene loci designated in a single group symbolically. For 
example, in the figure, which has two <b>A</b> alleles and two <b><i>a</i></b> 
at the gene locus, this would be symbolized <b>AAaa</b> in the genotype if it 
were a tetraploid, and the chromosomes grouped as quadrivalents.
<p>
But since the chromosomes group as bivalents despite a four alleles of one gene 
locus, the designation must be different.  And so the designation for the locus 
is <b>A<i>a</i>,A<i>a</i></b>.  Note the comma.
<p>
If a self cross is done on these alloploids, the gametes assort as follows:
<ol class="no-spacing">
<li><b>A<sub>1</sub>, A<sub>2</sub></b>
<li><b>A<sub>1</sub>, <i>a</i><sub>2</sub></b>
<li><b><i>a</i><sub>1</sub>, A<sub>2</sub></b>
<li><b><i>a</i><sub>1</sub>, <i>a</i><sub>2</sub></b>
</ol>
<p>
The progeny genotypes are:

<table class="punnett">
<caption>Allopolyploid Selfcross</caption>
<tr><th colspan="2" class="empty"><th colspan="4" class="punnett-sex">male

<tr><th colspan="2" class="empty">
  <th>A<sub>1</sub>, A<sub>2</sub> <br><span class="punnett-fraction">1/4</span>
  <th>A<sub>1</sub>, a<sub>2</sub> <br><span class="punnett-fraction">1/4</span>
  <th>a<sub>1</sub>, A<sub>2</sub> <br><span class="punnett-fraction">1/4</span>
  <th>a<sub>1</sub>, a<sub>2</sub> <br><span class="punnett-fraction">1/4</span>

<tr><th rowspan="4" class="punnett-sex" style="line-height:0.9em;padding:auto 0;">
   f<br>e<br>m<br>a<br>l<br>e

  <th>A<sub>1</sub>, A<sub>2</sub> <br><span class="punnett-fraction">1/4</span>
  <td>A<sub>1</sub>A<sub>1</sub>, A<sub>2</sub>A<sub>2</sub>
  <td>A<sub>1</sub>A<sub>1</sub>, a<sub>2</sub>A<sub>2</sub>
  <td>a<sub>1</sub>A<sub>1</sub>, A<sub>2</sub>A<sub>2</sub>
  <td>a<sub>1</sub>A<sub>1</sub>, a<sub>2</sub>A<sub>2</sub>

<tr><th>A<sub>1</sub>, a<sub>2</sub> 
        <br><span class="punnett-fraction">1/4</span>
  <td>A<sub>1</sub>A<sub>1</sub>, A<sub>2</sub>a<sub>2</sub> 
  <td>A<sub>1</sub>A<sub>1</sub>, a<sub>2</sub>a<sub>2</sub> 
  <td>a<sub>1</sub>A<sub>1</sub>, A<sub>2</sub>a<sub>2</sub> 
  <td>a<sub>1</sub>A<sub>1</sub>, a<sub>2</sub>a<sub>2</sub> 

<tr><th>a<sub>1</sub>, A<sub>2</sub> 
      <br><span class="punnett-fraction">1/4</span>
  <td>A<sub>1</sub>a<sub>1</sub>, A<sub>2</sub>A<sub>2</sub> 
  <td>A<sub>1</sub>a<sub>1</sub>, a<sub>2</sub>A<sub>2</sub> 
  <td>a<sub>1</sub>a<sub>1</sub>, A<sub>2</sub>A<sub>2</sub>
  <td>a<sub>1</sub>a<sub>1</sub>, a<sub>2</sub>A<sub>2</sub> 

<tr><th>a<sub>1</sub>, a<sub>2</sub> 
     <br><span class="punnett-fraction">1/4</span>
  <td>A<sub>1</sub>a<sub>1</sub>, A<sub>2</sub>a<sub>2</sub>
  <td>A<sub>1</sub>a<sub>1</sub>, a<sub>2</sub>a<sub>2</sub>
  <td>a<sub>1</sub>a<sub>1</sub>, A<sub>2</sub>a<sub>2</sub>
  <td>a<sub>1</sub>a<sub>1</sub>, a<sub>2</sub>a<sub>2</sub>
</table>
<p>
Since there are 16 possible genotypes, each has a 1/16 probability of being
in the progeny.  Since allelic dominance is a factor, there is a 15:1
phenotypic ratio for <b>A:<i>a</i></b>.

<h2>Aneuploidy</h2>
<p>
Aneuploidy refers to variations in chromosome number, which is other
than each homologous chromosome being equal in numbers as multiples.
<img src="imagesMolPopGeneticsHTML/AneuploidyTypes.gif"
   class="img-center"
  alt="types of aneuploidy: nullisomic, monosomic, etc.">
<p>
It will be recalled that an odd-numbered euploid organism will generate
aneuploid gametes that then will produce non-viable zygotes.  Of course
aneuploid zygotes can even occur when there is even-numbered ploidy
in the parent.  In diploids, if non-disjunction of one or more paired
homologues occurs, aneuploidy results.  This should not be confused with
gametic nonreduction:  in that event, all chromosomes are separated to
one daughter cell, and none to another (if it is even generated);
that daughter becomes a gamete.  In non-disjunction, only some and not
all of the chromosomes are affected.
<p>
Aneuploidy can be seen in three example species.
<h3><i>Datura stramonium</i> (jimson weed)</h3>
<p>
The diploid plant has 24 chromosomes (X=12, <i>n</i>=12, 2X=24,
2<i>n</i>=24).  All 12 chromosomes are designated by letters A to L.
Trisomic groups have been observed for each chromosome, although triploidy
has not been seen.  In trisomy for a particular chromosome, there is an
effect on the seed capsule.  This capsule is called the thorn aapple, and on
its body there are mulitple <b>echinus</b> (thorns).  Changes in the thorn
appearance or structure vary by the different trisomics possible.  These
are not allelic differences, but rather the effects of trisomy.  Trisomy-B
(that is the B chromosome which is trisomic) produces a small glossy
thorn apple with small echinuses.

<h3><i>Homo sapiens</i> (humans)</h3>
<p>
With 46 diploid chromosomes, numbered (rather than lettered) by cytogeneticists,
the most significant and well-known effect of aneuploidy is Down's syndrome,
or G-trisomy #21.  The Down's syndrome individual suffers from mental 
retardation usually but is of good-natured character.  Some physical 
deformities are evident, and life expectancy is only 35-40 years.
<p>
Trisomy 13 occurs in 1 in 10,000 births.  The individual is afflicted with
Patau's syndrome.  In most cases the fetus does not thrive and miscarriage
occurs.  Some do survive beyond birth and there are multiple disorders and
developmental abnormalities affecting all the body systems.  Trisomics are
observed in chromosome 18, which causes Edwards syndrome.  Most other trisomic
and all monosomic conditions result in prenatal abortion.

<h3><i>Drosophila melanogaster</i> (fruit or vinegar fly)</h3>
<p>
<img src="imagesMolPopGeneticsHTML/DrosophilaChromosomes.gif"
   class="img-right"
  alt="Drosophila genome/chromosomes">
Geneticists designate its four chromosomes using roman numerals.  The figure
shows the approximate position of the centromeres for the four chromosomes.
Trisomics for IV have been observed, but they appear normal.  Trisomy in 
chromosomes II and III are inviable.  In general, where trisomy involves the
smallest chromooomes of the genome, viability is not a problem.

<p class="date">
17 Jan 1980
<p>
The discussion of aneuploidy in three species was the stopping point in
the previous lecture.  In looking at <i>Drosophila</i> it is observed that
only some trisomics in this species survives.  The triploid species survives
as well.
<table class="punnett" style="float:right;margin:1em 0 1em 1.5em;">
<tr><th class="empty">
  <th><i>C</i><br><span class="punnett-fraction">&frac12;</span>
  <th><i>C</i><br><span class="punnett-fraction">&frac12;</span>
<tr>
  <th><i>CC</i> <br><span class="punnett-fraction">&frac12;</span>
  <td><i>CCC</i> <br><span class="punnett-fraction">&frac14;</span>
  <td><i>CCC</i> <br><span class="punnett-fraction">&frac14;</span>
<tr><th><i>C</i>  <br><span class="punnett-fraction">&frac14;</span>
  <td><i>CC</i> <br><span class="punnett-fraction">&frac14;</span>
  <td><i>CC</i> <br><span class="punnett-fraction">&frac14;</span>
</table>
<p>
<img src="imagesMolPopGeneticsHTML/trisomic-aneuploid.gif"
 class="img-left"
 alt="a trisomic set forming an aneuploid for the C gene locus">
Suppose that there is gene <i>C</i> whose locus is found on a trisomic 
chromosome set in an otherwise diploid species (see figure).  Examine the cross 
for this aneuploid and a normal diploid:  <i>CCC</i> &times; <i>CC</i>. The 
gametes and probabilities and the resulting cross shown in the table at the 
right shows that half the phenotype is trisomic and half is normal. With Down's 
syndrome, this sort of segregation is observed based upon the expectations of 
this genetics, but the disorder symptoms may arise or be suppressed depending on 
the environmental influences.  Clearly the trisomic condition results from a 
non-disjunction during formation of either the maternal or paternal gamete.
<p>
Consider the glossy variety of the <i>Datura</i> jimson weed which shows
trisomy for <i>B</i> (24 + <i>B</i>).  Gametes will be produced with either 
<i>BB</i> or <i>B</i> along with normal haplosomes.  Gametes are normally
degenerate in the abnormal aneuploids, which is an effect of an autosome.
In this case, the resulting zygote will almost always be nonviable or abnormal.
<table style="float:left;margin:1em 1.5em 1em 0;">
<tr><th>Genotype <th><i>Drosophila</i> <th>Mammals
<tr><td>XX       <td>female       <td>female
<tr><td>XY       <td>male         <td>male
<tr><td>XO       <td>sterile male  <td>sterile female<br>Turner's
   <br><span style="font-size:75%;">(4 in 10,000)</span>
<tr><td>XXY      <td>fertile female<td>sterile male<br>Klinefelter's
   <br><span style="font-size:75%;">(1 in 1000)</span>
</table>
<p>
Abnormalities with sex chromosomes are not as potentially lethal as autosomal
abnormalities.  Sex determination is shown for the normal.  The Y chromosome
absolutely determines maleness in mammals. Two X chromosomes and only two 
X chromosomes determine femaleness.  Consider the crossing in <i>Drosophila</i>
of a normal male (XY) and fertile female with karyotype XXY.  The latter
can produce four possible gametes as shown in the square (table below).
<table class="punnett" style="float:right;margin:1em 0 1em 1.5em;">
<tr><th class="empty">
  <th>XX <br><span class="punnett-fraction">1/6</span>
  <th>Y  <br><span class="punnett-fraction">1/6</span>
  <th>XY <br><span class="punnett-fraction">2/6</span>
  <th>X  <br><span class="punnett-fraction">2/6</span>
<tr>
  <th>X   <br><span class="punnett-fraction">&frac12;</span>
  <td>XXX <br><span class="punnett-fraction">1/12</span>
  <td>XY  <br><span class="punnett-fraction">1/12</span>
  <td>XXY <br><span class="punnett-fraction">1/6</span>
  <td>XX  <br><span class="punnett-fraction">1/6</span>
<tr><th>Y <br><span class="punnett-fraction">&frac12;</span>
  <td>XXY <br><span class="punnett-fraction">1/12</span>
  <td>YY  <br><span class="punnett-fraction">1/12</span>
  <td>XYY <br><span class="punnett-fraction">1/6</span>
  <td>XY  <br><span class="punnett-fraction">1/6</span>
</table>
<p>
Note the probability of gamete formation.  In determining the scheme of gametes, 
there are two X, so technically the karyotype is X<sub>1</sub>X<sub>2</sub>Y, 
and so the gametes form X<sub>1</sub>X<sub>2</sub>&nbsp;|&nbsp;Y, 
X<sub>1</sub>Y&nbsp;|&nbsp;X<sub>2</sub>, and 
X<sub>2</sub>Y&nbsp;|&nbsp;X<sub>1</sub>.  Thus of the six possible gametes, 
there are two XY and two X gametes, and one of the other two kinds, which 
explains the probabilities in the tables.
<p>
The YY genotype will not be viable, since an X chromosome is required.  The
XX, XY, XXY, and XYY genotypes will survive.  The XXX phenotype lives but develops
no sex organs despite being female in character.  Thus there are four
viable phenotypes in which the ratio will be 3:3:2:1 for  XY:XXY:XYY:XXX.
<p>
<img src="imagesMolPopGeneticsHTML/nondisjunction.gif"
  class="img-left"  alt="classic nondisjunction">
Chromosomal non-disjunction can occur either in the first or second division of 
meiosis or mitosis.  An XYY karyotype for example might result when there is an 
NDJ in the second division of spermatogenesis.  Clearly the X and Y bivalents 
assorted properly in the first division, but sister chromatids failed to 
separate from the centromere in the second.
<p>
What about dominance?  Suppose the following monogenic phenotype in which seed
color is green with <i>W</i>_ and white with <i>ww</i>. Suppose a homozygous
trisomc green parent crosses with a recessive disomic.  The <i>F</i><sub>1</sub>
generation is a heterozyote which is either disomic or trisomic:
<br> <i>P</i>:  <i>WWW</i> &times; <i>ww</i>
<br> <i>F</i><sub>1</sub>:  <i>WWw</i> and <i>Ww</i>
<p>
In a test cross of the <i>F</i><sub>1</sub> progeny which are trisomics 
(<i>W</i><sub>1</sub><i>W</i><sub>2</sub><i>w</i> &times;
<i>W</i><sub>1</sub><i>W</i><sub>2</sub><i>w</i>), each 
will produce six possible gametes in the trisomic.  The resulting genotypes are
<i>WWw</i>, <i>Www</i>, <i>Ww</i>, and <i>ww</i>.  In fact genes are mapped thusly
by crossing and then counting the number of phenotypes using recessives and dominants.
<p>
An experiment by Bridges using <i>Drosophila</i> was done crossing to look at 
genes on the sex chromosomes.  A locus on the <i>X</i> chromosome confers a red 
eye color if the allele is <i>X</i><sup>W</sup>, which shows dominance, and a 
white eye color if allele is <i>X</i><sup>w</sup> which is the recessive. Now 
consider a white-colored female (<i>X</i><sup>w</sup><i>X</i><sup>w</sup>) is 
crossed with red-colored male (<i>X</i><sup>W</sup><i>Y</i>).  The phenotypes of
the next generation will be nothing but red-colored females 
(<i>X</i><sup>W</sup><i>X</i><sup>W</sup> and <i>X</i><sup>W</sup><i>X</i><sup>w</sup>)
and half the males red (<i>X</i><sup>W</sup><i>Y</i>) and half white
(<i>X</i><sup>w</sup><i>Y</i>).  These kinds of ratios, if observed, thus
show sex chromosome-linked inheritance.  In some cases Bridges observed a
white female and this must be explained by a nondisjunction event.  To test this
the white female would have to be crossed with a white male to observe what might
be happening.

<p class="date">
22 Jan 1980

<h1>Human Gene Mapping and Hybrid Cells</h1>
<p>
A special technique used in genetics is the culturing of human cells and the
fusion of their nuclei with the nuclei of mouse cells.  These hybrid cells
are capable of dividing and growing, and the mouse (2<i>n</i> = 40) and human
(2<i>n</i> = 46) chromosomes all line up at metaphase.  But when mitosis
completes, the human chromosomes are not seen in the interphase nucleus
and their DNA strands are destroyed.  Curiously on the mouse genome is retained
and replicates.

<h1>Part II of Cytogenetics:  Chromosome Structure</h1>
<p>
In the first part there was a concern about chromosome number and the effects
of changing it.  In this part the concern is with the structure of a chromosome
itself and the effects of its abnormalities.
<p>
A <b>deficiency</b> of a chromosome, apart from the concept of deletions, is
when a piece of a chromosome is lost because of any of the following.
<ul>
<li>unequal crossing over: <img src="imagesMolPopGeneticsHTML/UnequalCrossing.gif"
 class="img-right"  alt="unequal crossing of paired chromosomes">
<li>chemical or physical damage
<li>chromosomal structural changes: translocations, inversions, duplications
</ul>
<p>

Deficiencies can be examined as nullisomic (homozygous) or monosomic. For 
instance in the figure the chromosome pair labeled <b>1</b>, some genes are 
missing in one chromosome, but the other homologue possesses alleles that still 
allow for survival of a deformed phenotype.  In chromosome pair <b>2</b>, the
individual would die but might survive if the deficient region codes for less than
1/40 (2.5%) of the genome.  For example, the notched wing phenotype in 
<i>Drosophila</i> is the result of a complete deficiency of a gene locus.
<p>
In humans, there are two cases of heterozygous deficiency.  One in chromosome
#13 results in retinoblastoma, a cancer induced in retinoblasts.  In 
chromosome 5, complete deficiency of one of its parts results in cri-du-chat
syndrome, causing both physical and mental abnormalities in children.
<p>
Looking at deficiency in the genetics of sex transmission in <i>Drosophila</i>,
supposed at loci <i>N</i>, the allele <i>N</i> is dominant, and resulting in 
a notched wing phenotype discussed earlier.  The recessive genotype <i>nn</i>
is the normal or wild-type phenotype.  Also described above, the transmission of
a red eye is because of a dominant genotype, <i>R_</i>, and the white genotype
is recessive, <i>rr</i>, and these loci are on the X chromosome.
<p>
Thus if a notched-wing female crosses to a male (X<sup>N</sup>X<sup>n</sup>
&times; X<sup>n</sup>Y), the possible genotypes of the progeny are 
notched-wing females (X<sup>N</sup>X<sup>n</sup>), normal females 
(X<sup>n</sup>X<sup>n</sup>), and normal males (X<sup>n</sup>Y).
Notched-wing males (X<sup>N</sup>Y) are also possible, but they are not viable
and so die.  A homozygous red-eyed female crosses to a white-eyed male 
(X<sup>R</sup>X<sup>R</sup> &times; X<sup>r</sup>Y) and only heterozygous
red-eyed females (X<sup>R</sup>X<sup>r</sup>) and red-eyed males (X<sup>R</sup>Y)
are possible.
<p>
Bridges found that a notched-wing, red-eyed female crossed to a white
normal male (X<sup>R,N</sup>X<sup>R,n</sup> &times; X<sup>r,n</sup>Y)
would produces females half of which are white-eyed with normal wings,
the other half red-eyed with normal wings.  This attempt at crossing with
sex chromosome-linked traits tries to map the R locus situated in the  N 
deficiency.  Deficiencies can be used to map loci, and this process is known
as <b>deficiency mapping</b>.  Thus crossing certain genotypes leads to getting
the locus wanted.  Of course this mapping is not possible for unlinked genes.

<h2>Duplication</h2>
<p>
Whereas deficiences can be considered to be <q>hyposomic</q> expressions, 
duplications can be considered to be <q>hypersomic.</q>  Duplications here
are of genes within the structure of the chromosome and do not really refer
to creation of one more in the number of a chromosome.  Phenotypically 
duplications could very well possibly cause abnormalities, such as developmental
or physical deformities, an early end (death) in the life cycle, or 
lethality in embryogenesis (nonviable individuals).
<p>
<img src="imagesMolPopGeneticsHTML/Duplication.gif"
 class="img-center"  alt="the phenotypic result of duplication">
Since eye pupil shape and color are loci on the X chromosome of 
<i>Drosophila</i>, how do duplications of these genes affect the fly?  The 
figure above shows the result.  On the left is the normal wild-type, with 
round-shaped eye as a result of the monogenic locus <i>B</i>.  If there is a 
duplication on one of the chromosomes to give digenic <i>B</i>, eye shape is 
changed in the phenotype (center drawing above); the phenotype is heterozygous 
for the duplication.  If the duplication occurs on the other chromosome on the 
diploid, which easily occurs in the next generations of flies, then the eye 
shape becomes elliptical.
<img src="imagesMolPopGeneticsHTML/Mispairing.gif"
 class="img-right"  alt="mispairing of duplicated genes during recombination">
<p>
Once a duplication arises, there is a higher frequency of additional duplication 
because of mispairing that occurs during homologous recombination (figure at 
right).  Notice that the recombined result will produce triple gene (ultrabar)
in one gamete, and just the one <i>R</i> on the other gamete, which is basically
a <b>revertant</b> to the normal.  The former is likely to die, but the revertant
should be viable.
<p>
<img src="imagesMolPopGeneticsHTML/HemoglobinMap.gif" style="width:70%;margin-bottom:0;"
 class="img-right"  alt="map of hemoglobin locus">
Adult human hemoglobin (Hb) is composed of two types of single chain proteins, 
&alpha; and &beta;, and two of each come together to form a heterotetreamter 
which is the functional hemoglobin protein, and each of the four are capable of 
carrying a single oxygen molecule.  The interesting thing is that although the 
&alpha; and &beta; chain provide the same biological function, they have a 
different structure (amino acid sequence). To have a different amino acid 
sequence suggests separate codings, and separate genes (an alternative would be 
spliced genetic elements during posttranscriptional processing).
<p>
<img src="imagesMolPopGeneticsHTML/HbExpressionOverTime.gif"
 class="img-left"  alt="Forms of Hb expression during embryonic development and after birth">
Indeed, when looling at the map of the chromosome that holds the Hb locus, the 
genes for the &alpha; and &beta; strands can be found and identified (see figure 
above right).  What is also identified are genes in between them, with highly 
similar sequences.  These genes are actually forms of hemoglobin that are 
expressed during embryonic and fetal development, with other forms just 
postnatally (figure at left). Thus the Hb composition is different at different 
developmental stages, and adult hemoglobin differs from that at earlier ages.
<p>
How could these juxtaposed genes making up Hb (and its function) have arisen? 
Quite possibly from duplication events.

<p class="date">
24 Jan 1980

<p>
<img src="imagesMolPopGeneticsHTML/DupByMissRecomb.gif" 
 class="img-left"  alt="duplication by misaligned recombination">
The duplication of genes occurs quite easily when there is a mispairing or
misalignment of sequences that have high similarity or near identity, and they are
adjacent to each other.  But once a duplication occurs, further duplication
becomes more likely since duplicated genes have virtual sequence identity
with one another and misalignment is quite probable.
<p>
In the previous lecture, there was a map of the Hb genes.  In the map, 
the &beta;, &delta;, and &epsilon; forms of Hb were shown drawn a single
chromosome segment.  The &alpha; gene was shown on another segment.  And
in fact, the &alpha; gene is not linked to the other three.  90% of adult Hb is
composed of an &alpha;<sub>2</sub>&beta;<sub>2</sub> heterotetramer, and
10% is &alpha;<sub>2</sub>&delta;<sub>2</sub>.  
<p>
<img src="imagesMolPopGeneticsHTML/HbGeneFineStructures.gif" 
 class="img-right"  alt="finer structure of Hb genes">
So how would these duplications arise?  Consider that the structure of the
gene for the &beta; and the &delta; forms of Hb are as shown the figure
at right.  Although subregions A, B, C, D are not idential to subregions
W, X, Y, Z, they are quite similar molecularly.
<p>
<img src="imagesMolPopGeneticsHTML/Lepore-AntileporeHb.gif" 
 class="img-left"  alt="finer structure of Hb genes">
It is observed in some mRNA transcripts that the translated products
result in abnormal hemoglobins.  For example, there are hemoglobins such
as Lepore's and anti-Lepore's Hb as shown in the figure.  Note that segments
of the &beta; gene seem to be present together with segments of the &delta;
gene in the Lepore Hb, where the N-terminal part of &delta; is fused with
the C-terminal part of &beta;.  For the anti-Lepore Hb, it is switched:
N-terminal of &beta; is fused to C-terminal of &delta;.
<img src="imagesMolPopGeneticsHTML/LeporeFormation.gif"
 class="img-center"  alt="formation of Lepore/anti-Lepore Hb">
Because the subgene regions are similar, a misalignment of the &beta; gene
on one chromosome occurs with the &delta; gene on other chromosome during
homologous recombination (see figure above).  Inheritance of the Lepore 
form will result in a nonviable individual (it is a lethal mutation), while
those with the anti-Lepore form of hemoglobin are viable but the phenotype
shows anemia because of the production of the chimeric &beta;-&delta; Hb
form, which disrupts structure.  These chimeric or hybrid forms are revealed
during electrophoresis in hematological analysis.
<p>
Generally <i>inter</i>genic crossovers which are mispaired while not show
an abnormal phenotype, because the gene sequence is not disturbed and proteins
are thus normal when expressed. But <i>intra</i>genic crossovers as illustrated
above create significant problems.  Evolutionary biologists have examined the
&beta; and &delta; sequences and believe that the &delta; gene duplicated
from the &beta; gene, probably resulting from when the &beta; and &gamma;
genes mispaired, and a third gene developed.
<p>
While a duplicated gene may do no damage to the individual, it will not retain
its function as a gene if it is not truly useful.  The fact that the &delta;
gene retains an active presence suggests it provided a selective advantage
of some kind.  It may have been allowed to mutate freely since a destructive
mutation would not result in lethality.

<h2>Inversion</h2>
<p>
<img src="imagesMolPopGeneticsHTML/Inversions.gif"
 class="img-center"  alt="inversions and how they cause a change in gene order">
Chromosomal inversions are just a re-sequencing on the chromosome that are 
caused by the looping of chromosomes.  Inversions generally arise as a result of 
looping occurring during meiotic or mitotic prophase.  In the image above, the 
order of genes A through F are changed as a result of inversion.  The figure 
shows that when the genes all on one arm of the chromosome (namely, with no 
centromere separating them), any inversion of their order is 
<b><i>paracentric</i></b>, where as genes ordered on two arms (across the 
centromere) any inversion of their order is <b><i>pericentric</i></b>.
<p>
<img src="imagesMolPopGeneticsHTML/InversionDetail.gif"
 class="img-center"  alt="detail of inversions">
The interesting thing is when an individual possesses homologous chromosomes in
which one of the chromosomes is inherited from one parent with the normal order, 
and the other chromosome is from a parent in which a paracentric inversion 
occurred.  During meoisis (anaphase I) to produce gametes from that individual, 
recombination with crossover will occur, in which complimentary sequences should 
pair. Because of the altered gene order, the homologous strands are more than 
likely to produce a loop. If a crossover starts within the loop, chromatids will 
be formed in which some genes are clearly missing (deficient) and others are 
duplicated (see figure above).
<p>
Note that one chromosome forms without a centromere (the one lacking gene A), 
which is required for chromosome duplication and ordering during mitosis.  Thus 
this chromosome cannot make copies of itself and will be lost.  The other 
chromsosome, with duplicate A genes, has two centromeres (one will probably 
disappear with time) and will carry on;  the effect of missing E and F genes
may not be lethal if genes are supplied by the other parent, or the complete
absence of the genes is not itself lethal.
<p>
<img src="imagesMolPopGeneticsHTML/OneCrossoverBivalentInversion.gif"
 class="img-center"  alt="single crossover bivalent inversion">
Looking the bivalent formation in metaphase/anaphase chromosomes with a 
recombination and crossover of two strands (arms) on the homologous chromosomes,
a single arm without a centromere emerges which is severely devoid of all genes,
and a complex looped chromosome structure emerges with two centromeres (figure
immediately above).
<p>
<img src="imagesMolPopGeneticsHTML/TwoCrossoverBivalentInversion.gif"
 class="img-center"  alt="double crossver bivalent inversion">
With a double crossover event on the same arms, two centromeric chromosomes 
result, but with arMs containing a segment with an inverted gene order. When 
multiple arms are involved in one, two or multiple crossover events, obviously 
the resulting chromosomes and chromatids can become more complicated.
<p>
<img src="imagesMolPopGeneticsHTML/PericentricInversion.gif"
 class="img-center"  alt="pericentric inversion">
Now consider the pericentric inversion.  In meiosis, a loop during homologous
pairing forms, but with the centromeres involved in the looping.  The gene order
inversion occurs across the centromere (see figure above).  During recombination 
only one crossover event occurs with two of the arms (note it occurs between loci
D and E in the figure).  The result will be chromatids, both with centromeres, but
with definite deficiencies and duplication of some genes.
<img src="imagesMolPopGeneticsHTML/MeioticParacentricInversion.gif"
 class="img-right"  alt="meiotic paracentric inversion, one and two crossover">
<p>
Thus, given any two marker loci which show a paracentric inversion (as shown 
above), any single crossover event will result gametes in which one gamete
is bicentromeric and the other acentromeric, and with both allelic forms
of one gene on the same chromosome, but on different syntenic loci!  If
a double crossover event occurs <i>across</i> <b>one</b> of the loci, then both
genes will be represented (a rescue of the single crossover event occurs), and
both chromosomes are centromeric.
<img src="imagesMolPopGeneticsHTML/DoubleCrossoverBetweenInvertedLoci.gif"
 class="img-center"  alt="double crossover event between inverted loci">
<p>
The same is true of a double crossover event <i>between</i> <b>both</b> of
the loci, it is as if no crossover occurred at all between the two loci, and
so gamete chromosomes will form as if no crossover occurred.  Pericentric
inversions with double crossoves should yield identical results.

<p class="date">
29 Jan 1980
<p>
Inversions are observed quite readily in smaller animals.  Instances can be
demonstrated where inversions have occurred frequently and the evolution can
be traced.
<img src="imagesMolPopGeneticsHTML/DeducingInversionFromOrder.gif"
 class="img-center"  alt="deducing how inversion occurs from gene order">
In the figure above, the gene order between two individuals is observed with the 
sequence of one individual on the far left (gene order: 
1&nbsp;2&nbsp;3&nbsp;4&nbsp;5&nbsp;6&nbsp;7&nbsp;), while that of the other 
individual is on the far right (gene order: 
1&nbsp;2&nbsp;3&nbsp;6&nbsp;4&nbsp;5&nbsp;7&nbsp;).  The change in gene order
can be deduced as having occurred with a double inversion event.  The first
event is shown in the middle strand of the figure, and then the 2nd event is
seen as the strand on the right.
<p>
If an inversion is selectively advantageous, then these kinds of events will 
occur at greater frequency  and the chromosome will contain inverted elements.
<img src="imagesMolPopGeneticsHTML/InversionDescendancy.gif"
 class="img-left"  alt="inversion descendancy">
<p>
In the figure on the left, chromosome <b>X</b> contains an inverted 
(paracentric) sequence, arising from chromosome <b>S</b> spontaneously. 
Chromosome <b>Y</b> also arose from <b>S</b> by the same biological processes, 
but the inversion is on a different part of the arm.  Chromosome <b>Z</b> formed 
from <b>Y</b>, when two segments around the inverted sequence in Y switched 
positions with each other.
<p>
All of these inversions and inversions-on-inversions (pivots) are shown in 
the same below the visualization with genes numbeed 1 through 9.  And the resulting
order is shown for each inversion.
<p>
So how is it assumed that <b>Z</b> arose from <b>Y</b>?  It is easy enough to
see that <b>X</b> and <b>Y</b> arose from <b>S</b>.  But in any way that <b>X</b>
can be inverted, it cannot yield <b>Z</b>, whereas a single inversion of <b>Y</b>
can produce <b>Z</b>.  Thus deducing the order of a number of inversion events
can be tricky.
<p>
<img src="imagesMolPopGeneticsHTML/TexasInversion.gif"
 class="img-center"  alt="a two-loop inversion:  the Texas inversion">
The interesting thing is how these homologous chromosomes might pair during 
meiotic recombination.  Chromosome <b>Y</b> (<span 
style="color:blue;">blue</span> strand) would pair with chromosome <b>Z</b> 
(<span style="color:red;">red</span> strand) to form a single loop as shown in 
the left cartoon of the figure above. If chromosome <b>S</b> (<span 
style="color:blue;">blue</span> strand) should pair with chromosome <b>Z</b> 
(<span style="color:red;">red</span> strand), expect the two-loop pairing as 
shown in the right cartoon of the figure above.  This inversion-on-inversion is 
in fact called the <b>Texas inversion</b>, probably because of the knot-like 
looping above.  The presence of multiple loops in fact suggests multiple 
inversion events in a segment.  Multiple inversion events are observed within
the third chromosome of <i>Datura pseudoobscura</i> and <i>Datura persimilis</i>.
Comparisons of the ninth chromosome between humans and chimpanzees suggests 
the occurrence of a pericentric inversion.
<p style="font:bold 100% Helvetica,Arial,sans-serif;">
All the lecture content up to the end of this lecture will be on the first 
midterm examination.

<h2>Translocations</h2>
<p>
Translocations are chromosomal abnormalities which involve transfers of the
genome from one chromosome to another chromosome.  In usual cases, the
transfers are two-way:  they involve an exchange of one part of chromosome
1 to chromosome 2, and from chromosome 2 to chromosome 1.
<p>
<img src="imagesMolPopGeneticsHTML/ReciprocalTranslocation.gif"
 class="img-center"  alt="reciprocal translocation">
In the figure above, consider two nonhomologous chromosomes.  On one end of one 
chromosome (telomere) are three genes (&nbsp;1&nbsp;2&nbsp;3&nbsp;) and on 
another end of the other chromosome are three genes 
(&nbsp;6&nbsp;5&nbsp;4&nbsp;).  Call this genotype normal (<b>N</b>).
<p>
A <b>reciprocal translocation</b> occurs, as in the figure shown, and the gene
order is preserved with respect to direction of telomere to centromere, except
that a segment of one nonhomologous chromosome exchanges with a segment of the 
other.  But the other of each of the homologous pairs is normal (<b>N</b>), existing
alongside the reciprocally translocated chromosomes (<b>T</b>).  Consider this
genotype <b>N/T</b>.
<p>
The reciprocal translocation that occurred on the one set of nonhomologous 
chromosomes can in fact be repeated with the other pair (or perhaps eventually
emerge in matings with gametes from individuals who contribute forms with 
<b>T</b>.  Translocations can certainly result in deficiencies when chromosomes
assort independently in ways in which, for example, the chromosome with a deletion
because of singly translocated genes becomes a gamete and then a zygote 
in the next generation.  The figure below shows how once combination of 
assortments can lead to gametes that are abnormal (that is, deficient in 
gene loci), whereas another set is normal by the chance of one particular 
assortment.
<img src="imagesMolPopGeneticsHTML/ReciprocalTranslocationResult.gif"
 class="img-center"  alt="gametic results of reciprocal translocation">
<p>
With reciprocal translocations, it is not uncommon to see metaphase chromosomes
pairing with a plus symbol as shown in the figure below.
<img src="imagesMolPopGeneticsHTML/ReciprocalTranslocationPairing.gif"
 class="img-center"  alt="non-homologous chromosome pairing because of reciprocal translocation">
When chromosomes separate in anaphase, it is possible for two types of disjuction
to occur.  <b>Adjacent disjunction</b> occurs when two nonhomologous chromosomes
remain connected.  With <b>alternate disjunction</b>


<p class="date">
12 Feb 1980

<p class="date">
14 Feb 1980

<p class="date">
19 Feb 1980

<p class="date">
21 Feb 1980

<p class="date">
26 Feb 1980

<p class="date">
4 Mar 1980

<p class="date">
6 Mar 1980

<p class="date">
12 Mar 1980

<p class="date">
13 Mar 1980



</body>
</html>

