<!DOCTYPE html>
<html>
<head>
<title>Biology for First Year Undergraduates</title>
<meta http-equiv="Content-Type" content="text/html; charset=iso-8859-1">
<link href="style/std.css" type="text/css" rel="stylesheet">
</head>

<body>
<p id="title">
Biology for First Year Undergraduates
<hr>

<h1>Cell Theory</h1>
<p>

In 1665, Robert Hooke uses a crude microscope with about 30&times; magnifying
power to look at cork (bark tissue) from an oak tree and describes small pore-like
compartments invisible to naked eye.  These structures are called <b>cells</b>.
<p>
Anton von Leeuwenhoek later builds a more powerful microscope (300&times;
magnifying power and observed <i>Paramecium</i> in pond water, and cells in
human blood and semen (spermatozoa).
<p>
In 1670s a researcher looking at plant leaves and stems with a scope concluded
these structures are composed of many indiviudal cells.  By 1800s, the accumulated
data led to conclusion that all organisms consist of cells.
<p>
The smallest organism today are bacteria that are 200 nm long.  Today the
conclusion still holds:  all living organisms are cells, which are structures
that can utilize energy and produce waste to maintain a functional viability and
even reproduce (grow).
<p>
<b>Origins of Cells, Spontaneous Generation</b>. Scientific theories have two
parts: the first part is describing a pattern in the natural world, and second
is to describe a mechanism or process responsible for creating the pattern.
Hooke and others had detailed the pattern. Rudolf Virchow (1858) described the
2nd part by stating all cells come from pre-existing cells. This challenged a
<b>spontaneous generation</b> concept in which life arises from nonliving
components, which is a <b>hypothesis</b>, a proposed explanation.
<p>
Louis Pasteur decided to test the spontaneous generation idea by experiment.
The <b>prediction</b> of this type is that it is something that can be measured
or observed if the hypothesis is valid. In one experiment, Pasteur has a flask
with a straight neck. He boils the broth to kill any living things pre-existing
in the broth, then lets it cool. Organisms from the air will enter the flask
neck and inoculate the broth and grow. This experiment see growth whether the
spontaneous generation or all-cells-from-cells hypotheses were true, but it
would be a useful control. Then Pasteur created a flask with a long neck tube
like a swan: it creates a trap such that when the broth is boiled, water vapor
is condensed in the trap when cooled. The airborne cells then enter the mouth
of the swan neck of the flask, but the broth shows no growth. If spontaneous
generation were true, we should still see growth. But if only all
-cells-from-cells is true, the absence of growth explains this hypothesis.

<h2>Theory of Evolution By Natural Selection</h2>
<p>
In the same year (1858) that Pasteur established the all-cells-from-cells
explanation for life, Charles Darwin and Alfred Russel Wallace published their
short papers (authored separately) to the Linnean Society of London their ideas
on speciation and the common ancestry of all life.  Darwin later published a
book <i>The Origin of Species</i> which brought all these papers into clarity.
The theory of evolution observed a pattern and then explained a mechanism or
process, as a biological theory.  The two claims made by Darwin and Wallace were
that the species had a common ancestry (contrasting with the Biblical view in
Genesis) and that the species do not retain their characteristics constantly
through time, but that modification occurs generation after generation, and that
later generations are evidence of <q>descent with modification.</q>  Others
had already indicated the pattern of relatedness, but what was new was that Darwin
explained evolution by a process of <b>natural selection</b>.
<p>
Natural selection requires two conditions:
<ol>
<li>Traits in individuals forming a population must be heritable:  they must be
passed on to offspring.
<li>Certain traits provide an advantage depending on the environment or conditions
in which the population exists:  certain individuals will be more fit than others
to adapt to the environment
</ol>
<p>
Natural selection affects individuals in populations.  Some are less fit to
survive to create offspring for the next generation, and so their heritable
traits do not appear.
<p>
From generation to generation, a population of individuals may evolve from another
to the point that the two populations have traits that create species, in a
process called speciation.  Examples of species being formed can be found today,
in support of the theory of evolution that forms a tree of life.
<p>
<b>Linnean taxonomy</b> is a system to name and classify organisms.  In 1735
botanist Carolus Linnaeus set out to create a classification to organize the
concepts of nature.  Taxonomic levels were created in this organization:
<ul>
<li>kingdom
<li>phylum
<li>class
<li>order
<li>family
<li>genus
<li>species
</ul>
<p>
Humans are the species <i>Homo sapiens</i>. They are coincidentally the only
member of genus <i>Homo</i>. They belong to the family Hominidae, which include
orangutan, gorilla, and bonobo. Hominidae are grouped under an order called
Primates, and this encompasses gibbons, monkeys, and lemurs. Primates are
grouped in the class Mammalia, which includes rodents, bison, and species that
have fur and produce milk. Mammals then join with other animals that possess
notochords in the phylum Chordata. Finally this phylum joins all animals within
the kingdom Animalia. Each of these levels of grouping is called a <b>taxon</b>.
 Linnaeus originally had two kingdoms, Plantae and Animalia. He classified
sessile species able to produce their own food as plants, and organisms that
were motile and eating organisms for food as animals. Some organizms do not
categorize so easily however: Molds, mushrooms, and other fungi absorb
nutrients from dead and living plants and animals, but were classed as plants
because they do not move. With the discovery that bacteria do not have nuclei
in their cells while cells of unicellular and multicellular species did have
nuclei, the classification system became very complicated.
<p>
Since these findings, there have been many proposals to reform the Linneaen system.
One was to propose five kingdoms in the 1960s:
<ul>
<li>Monera:  includes all prokaryotes
<li>Protista:  several groups of unicellular eukaryotes
<li>Fungi:  molds, fungi
<li>Plantae
<li>Animalia
</ul>
<p>
Carl Woese looked at the idea of classification differently.  The goal was to
understand a true phylogeny to trace relationships.  The basis of that was
to study ribosomal RNA (rRNA), namely the sequence of RNA in the small subunit
of the ribosome.  The sequence dictates the biology of the organism, but it is
also a trait or marker of the species.  The small subunit rRNA has about 2000
bases:  the sequence from all species can be obtained, and computer algorithms
used to study relatedness of the sequences.  This rRNA analysis has given rise
to a new tree of life.  Divisions are not between plants and animals, nor
even eukaryotes and eukaryotes.  Instead, three <b>domains</b> are used to describe
life so not to confuse them with the traditional kingdoms:  Eubacteria (Bacteria)
Archaebacteria (Archaea), and Eukaryota (Eukarya).  Both figures below are
dendrograms to show the relationships, and distance between nodes is a measure
of the degree of difference (not necessarily in time).  The figure at the bottom
provides a bit more detail for groups like the proteobacteria.
<p>
<img src="imagesBiologyFirstYearHTML/Woese-Phylogenetic-Tree-rRNA.png"
alt="Phyllogenetic tree according rRNA sequences (by Woese)"
style="width:100%;">
<img src="imagesBiologyFirstYearHTML/Another-Phylogenetic-Tree.png"
alt="another phylogenetic tree"
style="width:120%;margin:1em -10%;">
<p>
One thing to notice that this representation of molecular evolution indicates
that if plants and animals are to be distinguished, then the number of kingdoms
among prokaryotes will be dozens and dozens.

<h2>Biological Research</h2>
<p>
Research of most any type begins with a question about a natural phenomenon.
For example, in biology, why do giraffes have long necks?  Someone will
speculate for an answer:  because the function of the long neck enables to
get food in their environment, and to compete with other species for
access to food that is not available.  This speculation is a <b>hypothesis</b>.
It is an explanation, and a speculative one, because there needs to be prove
that this food competition hypothesis is true.  A <b>test</b> must be done to
support the hypothesis.  The first step in formulating the hypothesis is to
produce a list of predictions that would prove it.  The next step is to design
a study to observe or experiment and see if the predictions hold true or are
false.
<p>
If food competition hypothesis is true, then it should be possible to observe
a population of giraffes with varying neck lengths, the ones with longer necks
might thrive better under conditions where short-necked giraffes cannot reach
food. Robert Simmons and Luc Scheepers however found that most giraffes feed
by bending down, even in tall tree growth (they plotted feeding height vs.
percentage of bites taken in feeding in plant growth available at many heights).
There is still support of the food competition hypothesis, but an alternative
was offered by Simmons and Scheepers:  they proposed that long necks result
from competition for sex.  Giraffe mating involves males nuzzling the rumps
of females who urinate into the mouths of males, by which the components in the
urine may indicate to the male that the female is in estrus.  Males, which usually
have much longer necks, then come together in a contest involving the swinging
of necks to exert blows against each other.  The force of the neck swing is
enough to cause the loser to lose consciousness or even die, but in most cases
the male with the longer neck can generate more force (longer moment arm) than
a shorter-necked male.  The additional question is why males fight in this way
at all, with their necks.  The issue of the long necks is not fully setlled
however.
<p>
<b>Chili peppers</b>. Chili peppers contain the chemical capsaicin which will
bind to receptors in the epithelial tissue of mammals (especially humans) to
give the sensation of something hot or burning. The peppers are the
seed-containing fruit of the plant. Fruit is often consumed by the seeds are
not destroyed in passing through the digestive system. When the p
epper-consuming animal excretes the digestive waste, the seeds are present and
ready for germination, and also have a ready supply of fertilizer. Since the
animal moves considerably, the seed is dispersed. The <b>directed dispersal
hypothesis</b> states that the plant produces fruit in a way that enables the
seed to be consumed and not destroyed and to be excreted in a new location to
germinate.   To test this hypothesis, the cactus mouse and curve-billed thrasher
bird are the most important fruit- and seed-eating animals where chilies grow.
Earlier observations suggest that the mouse destroy seeds while the birds
disperse them.
<p>
Biologists set up an experiment with three plants:
<ol>
<li>chilies with capsaicin-producing very pungent fruit
<li>chilies of the same color, size and nutritional value that do not produce
capsaicin
<li>hackberries that look like chilies, not quite as red and with no capsaicin
</ol>
Five individuals from both species were observed as to what they ate, the
percentage distribution of what three types of food were eaten.  The mice did
not each the pungent chilies at all, and 90% of the time ate hackberries, and
10% of the time ate the nonpungent berries.  The birds on the other hand ate
nearly all the chilies of both types, and quite a bit of the hackberries too.
In a follow-up experiment, chilies without capsaicin were fed to the mice and
birds and the animal excreta collected.  These were planted to soil along with
14 seeds that had never been eaten.  About 50% of the control uneaten seeds
germinated, and 60% of the thrasher-eaten seeds did.  None of the cactus mice
eaten seeds germinated.

<h1>Water and Carbon</h1>
<p>
Hydrogen, carbon, nitrogen, and oxygen make up 96% of the mass of all biological
organisms.  The chemistry of these elements (atoms) is thus fundamental to
understanding biology.  The fundamentals of chemistry must also be understood:
<ul>
<li>atomic number:  count of protons in an element, establishing its identity
<li>isotopes:  elements in which the count of neutrons in atomic nucleus varies
<li>mass number: integer representing the sum of protons and neutrons in a nucleus
<li>atomic mass unit (amu):  the mass of a proton which is actually one-twelfth
of the mass of the <sup>12</sup>C isotope, and expressible as a <b>dalton</b>
(Da) or in grams per mole
<li>orbitals: the space that is occupied by an electron in its orbit around the
nucleus; often a geometric shape representing the probability of finding the
electron in the space
</ul>
<p>
Chemistry involves understanding how bonds are formed, and the types of bonds.
Electronegativity refers to how an atom wants or is willing to give up
electrons, and this is related to the desire to complete an electron shell of
the valence (bond-forming) orbitals. Electronegativity will determine the bond
types in chemical bonds: covalent, polar covalent, ionic. The number of
electrons in the valence shell will also determine the number of bonds each
element can form, and whether there will be pairs of non-bonded electrons that
also affect the geometry of molecular formation. As to geometry, consider
CH<sub>4</sub>, NH<sub>3</sub> and H<sub>2</sub>O.  The four protons of methane
will form a tetrahedron from the positions of the vertices of the H atoms.
With ammonia, the three protons form the base and the nitrogen atom the point
of a pyramid, although the nonbonding pair could extend to be tetrahedral.
With water, the three atoms form a simple 2D plane.
<p>
Water is a vital molecule for all living things.  Cells are 75% water by volume.
70% of body mass is water.  A loss of 2% of body water can lead to weakness,
dizziness, irritability, headache, difficulty urinating.  Water is an important
solvent because its polarity and it can form hydrogen bonds.  The abundance
of oxygen and nitrogen atoms in biomolecules make interaction with water likely.
<p>
Among the useful properties of water is <b>cohesion</b>:  this is a feature
of H-bonding which allows the molecule to strongly interact with itself.
<b>Adhesion</b> is a property that allows molecules to interact with molecules
different from the molecule.  Water as a liquid will adhere to solid surfaces
that have a polarity or charged component.  The meniscus formed by water in a
container is an example of adhesion of water molecules on the inner surface
of the container.  Cohesion of water with itself causes a downward <q>pull</q>,
and so both adhesion and cohesion contribute to the formation of meniscus.
Cohesion accounts for the high <b>surface tension</b> of water also.  Surface
tension refers to the resistance of a substance to increase its surface area
in interaction with an interface (surface) of substance of unlike molecules.
All liquids have a surface tension, but that of water is due to its property
of H-bonding.
<p>
<b>Why Ice Floats</b>.  Density is mass divided by volume, and the density of
pure water at slightly above 0&deg; (freezing) is 1.0 grams per cubic centimeter.
Things that are denser than water will sink in it.  But things that are less
dense than water should float.  Frozen water&mdash;ice&mdash;will float, meaning
it is less dense than water.  The property of water as being dense means it can
sustain life as we know it.  If ice were more dense, it would stay frozen in
the cold depths and the oceans would have remained frozen solid, making life
impossible.
<p>
Another useful property is its high <b>specific heat</b> (heat capacity). The
specific heat is actually the amount of energy required to increase by 1&deg; C
1 gram of a substance. By convention, amount of heat at 25&deg; C is used.
Water requires 4.18 J to heat. Ammonia is a strong hydrogen bonding molecule
too and requires 4.70 J. Ethanol requires 2.44 J to heat 1 g to 1&deg; C.
Ethylene glycol requires 2.22 J. Of those molecules with little or no
hydrogen-bonding, benzene, 1.80 J; xylene, 1.72 J; and sulfuric acid, 1.40 J.
The <b>heat of vaporization</b> is the amount of energy required to change water from
liquid to a gas.  This evaporation is the basis of cooling during sweating.
<p>
Additional principles of chemistry in biology include acid-base reactions and
the concept of buffering.  Additionally chemical reactions, their mechanisms,
equilibria, potential and kinetic energy, entropy, the laws of thermodynamics
(first: energy is conserved; second: entropy always increases in an isolated
system); reactions are heat-absorbing (endothermic) or heat-releasing (exothermic).
If change in Gibbs energy (&Delta;<i>G</i>) &lt; 0, then reactions are spontaneous
(exergonic); endergonic reactions are those where &Delta;<i>G</i> &gt; 0.
<p>
Question: what happens to chemical reaction rates when temperature and
concentrations of reactants are increased?  The rate increase hypothesis is that
reaction rates increase with either temperature or reactant concentration.
The null hypothesis is tath there is no effect on rate.  An experiment is
set up with bisulfite and iodate:  3 HSO<sub>3</sub><sup>&minus;</sup> (aq) +
IO<sub>3</sub><sup>&minus;</sup> (aq) &hArr;
3 HSO<sub>4</sub><sup>&minus;</sup> (aq) + I<sup>&minus;</sup> (aq).  The
experiment tests temperatures from &minus;1 to 50&deg; C, with numerous points.
For the reaction concentration, three different conditions are tried at a
constant temperature (T = 23&deg;:  bisulfite at 83, 83, and 170 mM, and iodate
at 5, 10, and 10 mM respectively.  Then multiple replicates are done with those
three trials.  Then the reaction rate (the inverse of time to completion) is
measured vs. temperature or concentration trial conditions.  The results did
show that temperature and reactant concentration increases increased the
reaction rate.
<p>
Biochemistry is about the chemistry of biomolecules, and biochemistry is
a subset of organic chemistry.  Organic chemistry is about the chemistry of
carbon.  Life begins with understanding the chemistry of carbon.  There is a
hypothesis fundamental to the basis of chemical evolution.  In primordial earth,
there is liquid water and the gaseous atmosphere.  In the atmosphere there would
be CO, N<sub>2</sub>, CO<sub>2</sub>, H<sub>2</sub>, NH<sub>3</sub>, and
H<sub>2</sub>O (as vapor).  In water, there would be few dissolved molecules
(just pure H<sub>2</sub>O).   As light energy comes to the Earth, the following
reaction occurs:  CO<sub>2</sub> (g) + H<sub>2</sub> (g) &rarr; HCHO (g) +
H<sub>2</sub>O (g).   NH<sub>3</sub> + CO &rarr; HCN + H<sub>2</sub>O is another
possible reaction, with formation of cyanide and formaldehyde.  By heating
hydrogen cyanide and formaldehyde in a water solution, we can form even more
compounds:  glycine, ribose, acetaldehyde.

<h1>Protein Structure and Function</h1>
<p>
Alexander Oparin and J.B.S. Haldane proposed a chemical evolution hypothesis in
1923.  In their hypothesis, they assumed that HCHO and HCN are formed from
H<sub>2</sub>, CO<sub>2</sub>, CH<sub>4</sub>, and NH<sub>3</sub>.  HCHO and
HCN react to form amino acids, sugars, and nitrogenous bases, and they call this
the <b>prebiotic soup</b>.  These link to form proteins, carbohydrates, and
nucleic acids.  Life becomes possible when one of these molecules acquires
the function of self-replication.
<p>
An important molecular class are proteins.  Proteins provide many functions
to cells:  defense (antibodies and complement), movement (contractile proteins
such as actin and myosin), catalysis (all enzymes speed up reactions that have
high activation energies), signaling (peptide hormones) structure (polymeric
proteins forming fibrils, or membranes).
<p>
<b>Miller's Origin-of-Life Experiment</b>.  In 1953, Stanley Miller tested the
theory of chemical evolution by connecting a large glass flask with methane,
ammonia, and hydrogen gases to a smaller flask with 200 ml of water, representing
primordial ocean and atmosphere.  Miller boiled the water in the smaller flask,
whose vapor went to the larger flask, and this fluxed continuously.  But none
of the chemical components changed:  water, hydrogen, methane, and ammonia
remained in the same quantities, even at 100&deg; boiling input of heat.
Miller added electrodes to the larger flask to create discharges, creating the
effect of lightning.  The solution in the boiling flask changed color, becoming
pink.  After a week, it was deep red and cloudy.  He found large amounts of
formaldehyde and hydrogen cyanide.
<p>
Proteins are composed of amino acid polymers.  Each of the 21 amino acids has
a side chain which has one or more functional groups that confers a kind of
chemistry to it.  Proteins have levels of structure: primary, secondary, tertiary,
and quaternary.  While the sequence of proteins typically dictates how it will
fold and form these structures, folding may also be accomplished by special
proteins called <b>molecular chaperones</b>.  Many chaperones are heat-shock
proteins:  after a rise in temperature, they may be necessary to speed refolding
any denatured proteins.  In particular are the case of prions (protein
infectious particles) which are abnormally folded proteins that induce abnormal
folding in other proteins.
<p>
As enzymes, proteins lower the activation energy of biochemical reactions.
Turnover in enzymes can be as slow as a few reactions per minute to as many
as 10<sup>6</sup> per second (carbonic anhydrase).  E. Fischer proposed a
lock-and-key mechanims of enzymes in 1894.  Enzymes were proteins that bound
substrates and had an active site that catalyzed the reaction.  Later enzymes
were predicted to be flexible in conformation and substrate binding was believed
to cause an induced fit prior to catalysis.  Enzymes can have <b>cofactors</b>,
which can be metals like Zn<sup>2+</sup> or Mg<sup>2+</sup>, or small organic
molecules which are <b>coenzymes</b>.  Enzymes can be inhibited and regulated
as well (sometimes indistinguishable).   The rate (velocity) of enzyme reactions
is often a function of substrate concentration.  When substrate levels are low,
increases in concentration show an increase in the rate of reaction.  When
substrates reach a high level, the rate does not increase but plateaus out,
showing saturation kinetics.

<h1>Nucleic Acids</h1>
<p>
The discussion of nucleic acids is vital because of the belief that life started
as a nucleic acid, specifically RNA.  One important aspect of life is that
<b>reproduction</b> is vital to living things.  Thus a molecule needed to make a copy
of itself.  Another attribute of life is <b>metabolism</b>, which are the series
of chemical reactions used to harness the energy for reproduction.  These
must occur in a membrane-bound space too.
<p>
Nucleic acids are composed of ribo- and deoxyribonucleotides.  The nucleotide
is composed of phosphate, ribose or deoxyribose sugar, and nitrogenous base.
In the theory of chemical evolution however, while amino acids are thought to
have been abundant, no experiment has shown the formation of a nucleotide.
In the prebiotic soup experiments, the heating of HCHO in solution will form
all of the 5- and 6-carbon sugars (pentoses, hexoses).  These are all produced
in equal quantities.  So how would ribose come to be the dominant sugar in
DNA or RNA?
<p>
One chemical evolution problem is how the pyrimidines (C, T, U) came to form.
(Purines are readily synthesized from HCN, found in the soup, and both A and G
have been found in the spark experiments.) Since RNA is considered the first
polymer to exist as a biomolecule, a hypothesis must be put forward to account
for how it could occur spontaneously. Nucleotides may have existed in an
activated form (as di- or triphosphates). These may have polymerized on the
surface of very small grained mineral particles (those found in clay soil).
Experiments with these clay particles and radiolabeled activated nucleotides
were done in which a reaction was allowed to occur for one day, then a fresh
batch of activated nucleotides is added. This was done over 14 days, and
samples were analyzed by gel electrophoresis and autoradiography.  RNA molecules
up to 40 nt were observed.
<p>
DNA has a secondary structure that was determined by Watson and Crick in 1953.
It was already known that DNA was a polymer formed by phosphodiester linkages.
E. Chargraff had determined that the total number of purines and pyrimidines
is identical (A + G = T + C), and that T = A and C = G also.  Additionally,
R. Franklin and M Wilkins had found the difference between groups of atoms in
DNA from X-ray crystallography:  data showed periodicities of 0.34 nm, 2.0 nm,
and 3.4 nm, and there was an indication that DNA had a helical or spiral symmetry.
W and C had determined that the helical width was 2 nm (20 &Aring;), and that
0.34 nm was the distance between the stacked bases in the polymer.  The 3.4 nm
distance they determined to be the length of one spiral turn of the helix,
approximately 10 stacked bases.  Combining with Chargraff's findings, they
deduced that DNA had a double strand in which opposing strands ran in an
anti-parallel fashion.  They developed a base-pairing scheme with hydrogen
bonding between bases of the strands.  And they showed the helical winding
created a major and minor <q>groove</q>.
<p>
DNA is an ideal molecule for a stable form of genetic information.  RNA can
degrade quite easily because of the possibility of self-cleavage of the polymer
by the 2&prime;-hydroxyl.  Can DNA by a catalyst?  Likely not.  DNA can assume
only a small number of 3D structures, and those could not be good at folding in
a way to provide a great many reactions.
<p>
RNA was once thought to have a function only in the transcription  of DNA and
translation into protein until Altman and Cech had led research showing that
RNA had catalytic properties, as molecules of <b>ribozymes</b>.  Studying
Tetrahymena, they found that phosphodiester bonds could be hydrolyzed or
formed.  They could also be used to form peptide bonds between amino acids.
But in order to show that RNA is the molecule that catalyze its own copying
or replication.  Proving this in the laboratory would be useful.
<p>
Wendy Johnston and others in D. Bartel's laboratory wanted to find a ribozyme
that catalyzes the addition of rNTPs to an existing strand by complementary
base pairing. This would constitute and <b>RNA replicase</b>.  If a RNA replicase
existed, it would have to have a certain sequence.  But what is it?  Perhaps
the best approach is to randomly generate billions of large RNA molecules with
a unique base sequence, and then incubate them with smaller RNA templates (21 nt)
with short (7 nt) primer RNA molecules that bind to the 3&prime; end of the
21 nt RNA templates.  The next step is to incubate the large RNA molecules with
the RNA template + RNA primer, and then examine which large RNA molecule batches
best extend the complement from the primer.  These are the candidate ribozymes.
Now make many copies of the candidate ribozymes so that the sequence is altered,
and repeat the experiment with template+primer extension another 17 times.
This refines the  ribozyme sequence to make it the most efficient.   In round 18,
the scientists analyzed the ribozyme batch with template and could detect that
14 bases were added by 24 hours.

<h1>Carbohydrates</h1>
<p>
At this point the student is introduced to carbohydrates as having the
formula (CH<sub>2</sub>O)<sub><i>n</i></sub> (<q>carbon-hydrate</q>).  Sugars
can be monosaccharides, oligosaccharides, and polysaccharides.  Trioses are
3-carbon sugars, tetroses 4-carbon, pentoses 5-carbon, and hexoses 6-carbon.
Galactose is an isomer of glucose (4-epimer).  Linear forms of the sugar can
open and close to form a hemiacetal ring, in which carbon-1 hydroxyl can be
on the oppoosite side as the carbon-6 hydroxymethyl of the ring plane, the
&alpha;-anomeric configuration, or on the same side, the &beta;-anomeric
configuration.  In equilibrium, the &beta; form is present in slightly higher
levels because it is more stable.
<p>
Carbohydrates are easily formed in chemical evolution.  The heating of formaldehyde
will form all of the pentoses and hexoses and some of the heptoses as well.
The ketotriose dihydroxyacetone was recently found on the Murchison meteorite.
<p>
These sugars will form rings, and these rings will polymerize by condensation
to form glycosidic linkages and large polymers.  Because their are two possible
anomers, &alpha; and &beta;, the linkage must specify which.  In addition, the
linkage can be with any of the hydroxyls on carbons in the ring.  In nature,
the polymers formed by organisms are special:
<ul>
<li>Starch:  synthesized by plants as their storage form for metabolism, these
are Glc&alpha;1&rarr;4Glc polymers.  When unbranched, the glucose polymer is
called <b>amylose</b>.  When Glc&alpha;1&rarr;6Glc branching occurs, the polymer
is <b>amylopectin</b>.  Branching occurs with a periodcity of one in 30 monomers.
<li>Glycogen:  the form of glucose storage in animals, mostly stored in liver and
skeletal muscle.  Glycogen is highly branched, with a periodcity of one branch
every 10 monomers, and the linkage of the linear and branching part are the same
as starch.
<li>Cellulose:  the foundational polymer for plants and algae, this is a polymer
of Glc&beta;1&rarr;4&beta;Glc monomers.  This enables the polymer to form
more hydrogen bonds with other polymers, making the packing arrangement quite
sturdy and ideal material to produce a cell wall.
<li>Chitin:  this can stiffen fungal cell walls, and is the basis of the
exoskeleton of insects and crustaceans.  This is a polymer of N-acetylglucosamine
monomers forming &beta;1&rarr;4 linkages (GlcNAc&beta;1&rarr;4GlcNAc).  The
interpolymeric bonding like cellulose (with its &beta; linkages) is even stronger.
</ul>
<p>
<b>Peptidoglycan</b>. Also called <i>murein</i>, while this structure is not
exclusively carbohydrate in nature, this is the basis of the cell wall in 
bacteria, which cannot produce cellulose. The carbohydrate portion is a 1:1 
copolymer of N-acetylmuramic acid (MurNAc) and N-acetylglucosamine linked &beta;
1&rarr;4. On carbon-3 hydroxyl of MurNac is attached
[hydroxyl]-CH(CH<sub>3</sub>)-C(=O)-4|5peptide.  4|5peptide is either a tetra-
or pentapeptide, depending on the bacterial species.  In E coli, it is the
sequence (L-Ala)-(D-Glu)-(<i>meso</i>-diaminopimelic acid)-(D-Ala).  Then two
4|5peptide branches are connected by an bridging peptide of 5 glycine residues.
This composes the peptidoglycan wall.
<p>
Carbohydrates were very unlikely in the origin of life models.  They would not
have formed polymers under the conditions of the prebiotic soup.  Glycosidic
linkages of all the carbohydrate polymers only form by enzymatic action.
Additionally, ribozymes that form in the prebiotic soup do not catalyze
polysaccharide formation.  Monosaccharides also lack functional group diversity
as opposed to amino acids.  Polysaccharides form simple secondary structures
moreover, and no catalytic activity have been attributed to them unlike with RNA.
Polysaccharides show no form of pairing or replication which demonstrates that
they could have provided information.
<p>
Carbohydrates can be structural molecules, but also provide energy for metabolism
by their oxidation.   Carbohydrates also have structures that can indicate cell
identity or type, as glycoproteins directing cell movements, cell-to-cell adhesions,
and identification self to immune cells.
<p>
As sources of energy, the C-C and C-H bonds in carbohydrates get oxidized down
to carbon dioxide.

<h1>Lipids and Membranes:  First Cells</h1>
<p>
Beside a catalytic self-replicating molecule that could code for life, another
feature would be the enclosure of a system within a membranous sac.  Polymerization
may have first occurred on clay surfaces, but later on, it could have occurred
on surfaces of lipids forming a membrane.  Within a membrane enclosure reactants
are brought together in higher concentrations.
<p>
It is generally accepted that lipids existed in the prebiotic soup.  Miller's
experiment was found to produce two lipids in fact.  A. Bangham in the late 1950s
looked at lipid-water mixtures with TEM to understand the ultrastructural
features, and he observed the formation of vesicles, which would have easily
produced the primordial cell.  The formation of phospholipids would have been
ideal:  one part forms a (negatively) charged region to interact with water,
while the other creates a hydrophobic portion that interacts with other
lipid molecules and excludes water.  Liposomes can be created by the sonication
of phospholipids of varying types to form vesicles with different properties.
<p>
Membranes show a selective permeability.  Permeability can be measured using
numeric value (coefficient) that measures the rate by which an ion or molecule
can pass through a lipid membrane.  For lipid bilayers, permeability can be
seen by the class of molecules or ions:
<ul>
<li>Small, nonpolar molecules like O<sub>2</sub> and CO<sub>2</sub> pass with
the most rapid rate, at 10<sup>&minus;1</sup> cm/s.
<li>Small, uncharged polar molecules like H<sub>2</sub>O at 10<sup>&minus;3</sup>
to 10<sup>&minus;2</sup> cm/s
<li>Large, uncharged polar molecules like glycerol and urea at 10<sup>&minus;6</sup>
to 10<sup>&minus;5</sup> cm/s, and like glucose at 10<sup>&minus;7</sup> cm/s
<li>Ions, with very low permeability, like Cl&minus; at 10&minus;10 cm/s and
like K<sup>+</sup> at 10<sup>&minus;11</sup> cm/s, and the lowest permeability
of Na<sup>+</sup> at 10&minus;12 cm/s
</ul>
<p>
The small uncharged with no to little polarity will cross the membrane.  But the
large uncharged and ions do not really cross.
<p>
Permeability is also affected by the degree of unsaturation of a lipid forming
the bilayer.  Experiments with liposomes where degree of unsaturation is
controlled in the composition reveal this property.  As unsaturation in a pure
lipid is increased, it can go from being a solid fat to a liquid oil.
<p>
Cholesterol is a fairly rigid molecule without a lot of saturation.  The effect
of including it in membranes increases the density of the hydrophobic portion
and it also reduces membrane permeability.  Phospholipids were mixed with varying
concentrations of cholesterol (0, 20 and 50% by molar ratio), and the permeability
of the membrane to glycerol after liposome formation was tested at different
temperatures.  As the proportion of cholesterol increased, the permeability
was reduced, although in all cases of increasing temperature, permeability is
increased.
<p>
<b>Temperature and Membrane Fluidity and Permeability</b>.  Temperature is
also important.  At 25&deg;, the phospholipids in plasma membranes are effectively
oils.  As the temperature drops, the fluidity decreases, and the membranes can
actually solidify.  Following labeled individual molecules shows that lateral
movements within the 2D plane of the membrane are 2 &micro;m/s, which means a
molecule can travel the length of a bacterial cell within one second.
An interesting thing is that phospholipids do not cross (flip) from one monolayer
to the other.
<p>
In understanding the importance of membrane formed by lipids as a barrier between
two compartments, it is important to understand concepts of diffusion,
concentration gradients (and the relationship to entropy), and osmosis.
It is also important to understand hypertonic, hypotonic, and isotonic.

<h2>Membrane Proteins</h2>
<p>
Knowing that plasma membranes had as much protein by mass as lipid bilayer,
in 1935 H. Davson and J. Danielli proposed a model for the plasma membrane
that had the bilayer sandwiched between intra- and extracellular layers of
protein bounding the membrane.  Electron micrographs appeared to support that
model, although they were primitive.  S. J. Singer and G. Nicolson in 1972
proposed another theory, knowing that proteins had amphipathic character to them.
They proposed a <b>fluid mosaic model</b>, the mosaic being proteins spanning
the membrane interspersed within (integrated to) the membrane.  Evidence
for this model was provided by the development of freeze-fracture scanning electron
microscopy (in the 1970s).  The nature of the specimen preparation allows
for the splitting of one monolayer of the membrane from the other, and to
observe the abutment of the monolayer split off.
<p>
We soon understand that some proteins pass through the membrane, and these
are <b>integrated membrane proteins</b>, or <b>transmembrane proteins</b>.
Other proteins are just found on one side of the monolayer, but clearly belong
to the monolayer, and are peripheral membrane proteins.  To study membrane
proteins, scientists in the laboratory use special detergents to make them
soluble and hopefully without destroying the structure or function of the
protein.
<p>
Three broad classes of membrane proteins involved with the passage of ions and
molecules have been identified.
<p>
<b>Channel proteins</b> are those that basically form a channel for the specific
movement of ions where there is an existing concentration gradient.  One such
channel protein was identifed by the antibiotic <b>gramicidin</b>, itself a
peptide produced by <i>Bacillus brevis</i>.  The bacterium uses gramicidin as
poison for other bacterial species to prevent their growth;  it forms a resistant
coating around its own membrane to prevent gramicidin from poisoning the host
producing it.  What gramicidin does it kill the other bacteria by causing them
to leak ions.  Experiments done with planar lipid membranes with electrodes on
two sides, and the addition of gramcidin to measure electrical currents (flow
of ions) shows that when gramicidin is added, ions flow;  when not present, there
is no flow.  Thus gramicidin formed an ion channel.   Additional study showed
it was selective, allowing only cations to pass:  H<sup>+</sup> is most permeable,
followed by K<sup>+</sup> and Na<sup>+</sup>.  3D structure studies of gramicidin
shows that the 15 amino acid-peptide forms an &alpha; helix through the bacterial
membrane with a large pore down the long axis.
<p>
<b>Aquaporins</b> happen to be channel proteins that allow H<sub>2</sub>O to
pass at a rate 10 times faster than through the membrane itself. No other ion
or molecule is allowed. Investigations of specific K<sup>+</sup> channel
proteins show the mechanism by how particular ions are selected. The channel
protein maintains a closed configuration normally. But when the correct ion is
bound at the entrance of the channel pore, it triggers a change in conformation
that puts it into an open configuration, and the ion is shunted through. In all
casas, channel proteins provide passive transport, requiring that the driving
force of the molecular movement is an electrochemical gradient, and that Gibbs
free energy is less than zero. Channel proteins enable <b>facilitated
diffusion</b>.
<p>
<b>Transporters</b>. Also called <b>carrier proteins</b>, and like channel
proteins in that they use as the driving force electrochemical gradients, they
are involved in the specific movement of molecules across the membrane. A
classic example is the GLUT-1 glucose transporter. Scientists prepared
membranes from erythrocyte ghosts (ghosts are made by bursting cells in
hypotonic solution and recovering the membrane.  When gluose solution was added
on one side, the RBC membranes were more permeable to glucose than pure bilayers.
Proteins were purified, and when tested by constructing liposomes with these
proteins present, one was found that transported Glc at the same rate as living
cells.  It was specific for D-glucose, not the L-form.  The mechanism is likely
a Glc molecule occupying a position at a gate on GLUT-1, and then it becomes a
key that triggers the opening of a lock, allowing the molecule to pass through.
<p>
<b>Pumps</b>. Certain proteins harness energy to move specific ions and
molecules along, usually moving them across the membrane against the
electrochemical gradient. A classic example is the <b>Na<sup>+</sup>/K<sup>
+</sup>-ATPase</b>. Step 1 involves binding three Na<sup>+</sup> to high
affinity sites on the intracellular side of the protein. The protein is then
phosphorylated by ATP at a position on the intracellular side. This induces a
conformational (shape) change that opens the pump on the extracellular side and
closes it to the intracellular side. This conformational change reduces the
affinity for Na<sup>+</sup> ions as well, which diffuse into the extracellular
fluid. It also opens two high affinity sites for K<sup>+</sup> ion, which bind
from the extracellular space. When they bind, the phosphate group is hydrolyzed,
and this induces a conformational change back to the shape that opens the pump
entrance to the intracellular side. The affinity for K<sup>+</sup> is low now,
and so the two K<sup>+</sup> ions diffuse into the intracellular fluid. The net
effect is as follows: ATP undergoes hydrolysis to ADP and P<sub>i</sub>, with
the release of its energy harnessed in causing a two-state conformational
change in a pump protein. Three Na<sup>+</sup> ions are moved out, and two
K<sup>+</sup> ions are moved in. This will create a chemical gradient for
Na<sup>+</sup> and K<sup>+</sup>. It will also create an electrical gradient,
with more positive charge on the extracellular side of the membrane, and more
negative charge on the intracellular side.  Note that other such pumps exist
for other ions (e.g., H<sup>+</sup> and Ca<sup>2+</sup>) and for molecules too.

<h1>Cells</h1>
<p>
Features of the cell are now introduced.  The concept of a plasma membrane has
been discussed.  In classifying life according to morphology, a distinction is
made between prokaryotes and eukaryotes.  According to molecular phylogeny and
to evolutionary biologists (based on RNA analysis), life is divided into
three domains: Bacteria, Archaea and Eukarya.  Still two of the domains are
prokaryotic, and the third defines eukaryotes.

<h2>Prokaryotes</h2>
<p>
The first thing to take notice of is that these cells bound a fluid called the
<b>cytoplasm</b> (<q>cell-formed</q>).  Its contents of molecules and ions
is hypertonic relative to the extracellular environment.  Thus water can rush
in from the osmotic pressure and literally explode the cell.  This is prevented
however because prokaryotes have a rigid cell wall that counteracts the
force (osmotic pressure).  This cell wall also features glycolipids, which are
used in other ways to protect the cell.
<p>
Prokaryotes have a genome composed of a single circular chromosome that is not
bounded by a membraneous organelle called the nucleus:  this is what distinguishes
the prokaryote from eukaryote.  Nonetheless the intracellular space in which
the chromosome is found is not entirely like the rest of the cytoplasm, and its
space is called the <b>nucleoid</b>.  Ultrastructurally, the nucleoid occurs
centrically in the cell and takes up 20% of the total cell volume.  If the
circular chromosome of E coli were cut once to make it linear, its length would
be 1 mm, 500 times the length of the cell itself.  The DNA is thus supercoiled
to make it compact.
<p>
<b>Plasmids</b>.  In addition to the main circular chromosome, bacteria
contain very small circular supercoiled molecules of DNA, numbering from
one to 100, called plasmids.   The genes they contain are usually not vital
to the life of the bacterium, but can be if it gives an advantage in environments
where bacterial growth is prevented or when the bacterium is threatened.
<p>
Also visible ultrastructurally are ribosomes, with as many as 10,000, always
active in providing protein synthesis. The assembled ribosome with its large
and small subunit can span 200 &Aring; &times; 300 &Aring;, enabling its
visibility.
<p>
Flagella are clearly visible for bacteria with motility, usually aquatic
species.  They are few in number when present.  40 different proteins are
involved in constructing the flagellum.  Their rotation (spinning) to move the
bacterium can make it move 60 lengths of the organism per second. (Note that the
cheetah can only sprint 25 of its body lengths per second.)
<p>
<b>Internal membranous structures</b>.  While the distinguishing feature between
eukaryotes and prokaryotes is usually organelles having cell-like membranous
bound structures, photosynthetic bacteria require such structures to perform
their function.  These are derived by infoldings of the plasma membrane with
an accumulation of pigments and proteins involved with photosynthesis, and the
infolded membranes stack upon one another.  Technically, these structures
qualify as <b>organelles</b> just they do in eukaryotes.   Other organelles
include calcium-storage vesicles, and others include crystals of magnetite:
these actually allow the bacteria to find their location in a magnetic field
and swim in directed way.
<p>
<b>Fibrillar proteins</b>.  Polypeptides can interact in a self-like way to
form fibers (fibrils) and thus form a <q>rope</q> or cable.  These can perform
several functions, in particular forming in internal framework as a
cytoskeleton.  These are required for some functions, in particular to enable
cell division.

<h2>Eukaryotes</h2>
<p>
There are vast differences in the generic eukaryote compared to the prokaryote.
The first noticeable difference is size.  Most eukaryotic cells range from
5-100 &micro;m in diameter, while prokaryotes are 1-10 &micro;m.
Another difference is that eukaryotes are composed of many membranous structures
which are clearly organelles.  Some of these organelles appear to resemble
the bacteria and archaea, and it is believed that the first eukaryote acquired
the ability to survive by ingesting bacteria and archaea.  The fact that
eukaryotic cells actively engulf and have structures to breakdown and consume
prokaryotes (as a defense) suggests that this is the case.
<p>
Because of the larger size of the cells, eukaryotes acquired a problem in that
chemical diffusion is too slow to compensate for the need to make a chemical
available to processes.  For example, ATP produced on one side of the cell
cannot traverse quickly enough to point in the cell where it is needed.  That
would be true if the eukaryote was nothing but cytoplasm.  But eukaryotes fixed
that by the use of organelles.  It compartmentalizes biochemical reactions
for one thing, increasing efficiency and enabling the making of local gradients
both chemical and electrical.  Organelles are evenly distributed usually around
the cell so that no one side is left without.
<p>
<b>Nucleus</b>. The largest organelle in the eukaryote (typically), its
function is to contain the genome and to make sure part of it is transcribed to
provide the genetic information for coding protein and metabolism and function
of the cell.  The membrane is called a nuclear envelope, studded with pore-like
openings.  On the inner surface, are fibrous proteins forming a lattice called
the <b>nuclear lamina</b>.  Chromosomes then attach to the lamina in one or
more locations.  Many cells also contain a nucleolus:  this is a space within
the nucleus where rRNA molecules are actively transcribed, and the proteins
for the ribosome are imported from the cytoplasm into the nucleus for assembly.
<p>
<b>Cytosol</b>.  This part of the eukaryotic cytoplasm is everything within the
cell not bounded by a membranous organelle.  Within this space are the ribosomes,
usually numbering in the millions.  The eukaryotic ribosome is larger than the
prokaryotic, in molecules and assembled dimension.
<p>
<b>Endoplasmic Reticulum, Rough and Smooth</b>.  The endoplasmic reticulum
(<q>inside-formed network</q>) is a series of membrane-bound vesicles and tubules
that has continuity with with the outer membrane of the nuclear envelope.
When observed ultrastructurally, ribosomes can associate with the ER membrane
and give it a studded appearance (<b>rough ER</b>).  Ribosomes on the rough ER
are structured to synthesize proteins that will either be secreted from the cell
or which will be integrated into the plasma membrane or the membranes of certain
organelles, such as the lysosome.   Other parts of the ER in the cell have no
ribosomes attached:  this is the smooth ER, and it is involved in lipid
biosynthesis or in breaking down lipids that are foreign (and potentially
poisonous) to the cell.  The smooth ER also acts as a local storage pool of
Ca<sup>2+</sup> ion, which is stored in high concentrations of the lumen, and
which enters the cytosol as a signalling mechanism when mtetabolic functioning
of the cell is to be changed.
<p>
<b>Golgi Complex</b>.  This is a part of the cell featuring a series of
flattened, membranous sacs/vesicles called <b>cisternae</b>, stacked against
each other.  The arrangement has a clear polarity, the poles being <i>cis</i>
and <i>trans</i> in designation.  The cis pole membranes joins with rough ER
vesicles, and the trans pole membranes form new vesicles that leave the Golgi
for the plasma membrane.  During the transition from cis to trans poles, the
proteins undergo biochemical modifications, namely glycosylations, that turn
them into glycoproteins which are typically integrated into the cell surface
or secreted from the cell into the extracellular space.
<p>
<b>Peroxisomes</b>.   These are found in nearly all eukaryotic cells and their
presence is not a specialization unique to a cell type.  They are bound by
a single membrane.  Their purpose is to conduct biochemical reactions related
to oxidation.  This oxidation often generates hydrogen peroxide
(H<sub>2</sub>O<sub>2</sub>), which is quite destructive to the phospholipids
of all membranes (organellar and plasma membrane).  Within the peroxisome
is the enzyme catalase, which rapidly breaks down hydrogen peroxide to water
and oxygen.  The membrane tries to contain the generated peroxide so that
it can be converted without diffusing away.  Peroxisomes differ in their
oxidative enzyme composition based on cell type.  Liver cells use oxidation
to detoxify foreign chemicals, including beneficial ones (drugs) and less than
beneficial ones (toxins, poisons).  Other peroxisomes specifically exist to
oxidize lipids, a normal catabolic activity, to generate high energy compounds
like acetyl-CoA used in synthesis of other metabolites.  <b>Glyoxisomes</b>
are special peroxisomes in plant cells that convert metabolites of photosynthesis
into sugars whose energy can be stored.  Seeds do not have glyoxisomes because
they are not photosynthesizing, but have peroxisomes to convert fatty acids
into glucose.  Peroxisomes appear spherical ultrastructurally with diameters
of 300-500 nm (0.3-0.5 &micro;m).
<p>
<b>Lysosomes</b>. These can vary quite widely in size and function, depending
on the cell. In plants, fungi, and certain other kingdom-level organisms, the
lysosome is actually called a <b>vacuole</b>. In animal cells, they are used to
chemically digest the luminal (interior) contents of the lysosome. The membrane
has proton pumps that increase luminal H<sup>+</sup> concentration to a pH 5.0
or lower. Other enzymes (acid hydrolases with optimal pH at a lower value) are
present that break down a variety of biomolecules: proteins, polynucleotides,
lipids, and carbohydrates.  The smaller molecules (monomers) generated are then
brought into the cytosol.  The single membrane lysosome is about can reach
diameters larger than 500 nm (0.5 &micro;m).  The way lysosomes work is by
fusing with membrane-bound organelles.  When something touches the plasma
membrane of the eukaryotic cell, its response will be to invaginate and attempt
to enclose the plasma membrane around it.  This becomes a phagosome, and the
process is called phagocytosis.  As the phagosome is formed, a lysosome
works towards it and fuses with it.  In the fusion, which is the formation
of a phagolysosome, the degradative enzymes and the acidic environment starts
to chemically degrade the material in the phagosome.   In a process called
<b>autophagy</b>, the lysosome is recruited to degrade organellar structures that may
be old or damaged.
<p>
<b>Receptor-mediated endocytosis</b> is a specialized process in which macromolecules
bind to receptors on the plasma membrane, initiating invagination of the
membrane to form a vesicle called an <b>endosome</b>.  The receptor proteins are
then recycled back to the surface of the plasma membrane for another cycle.
The endosome acquires proton pumps that acidify the contents.  In addition,
it fuses with vesicles coming from the Golgi complex, transitioning from an
<b>early endosome</b> to a <b>late endosome</b>.  This late endosome then
goes on to form into a functioning lysosome.
<p>
<b>Endocytosis</b>.  Note that all processes that stimulate invaginations for
plasma membrane to form an internalized vesicle are called endocytosis.  Endocytosis
has three forms:  receptor-mediated endocytosis discussed above, phagocytosis
already discussed, and pinocytosis, in which the cell basically envelopes
merely the extracellular fluid.
<p>
<b>Vacuoles</b>.  Vacuoles in plants and fungal cells are very large in these
cells, taking up as much as 80% of cell volume.  Rather than being used to
catabolize macromolecules, they are used to store water or metabolites.
Sometimes, vacuoles are used to encase toxins that are consumed by predators
to stop predation.
<p>
<b>Mitochondria</b>.  This is a double-membraned organelle, with an outer
membrane that bounds the organelle simply.  The inner membrane is more complex,
with series of invaginations that are called <b>cristae</b>.  The fluid encased
by the inner membrane is called the <b>matrix</b>.  Most of the soluble enzymes
doing the functional work of catabolism are in the matrix.  Membrane-bound
enzymes are in the cristae.  In addition, mitochondria contain small circular
and supercoiled DNA quite similar to bacterial chromsomes, and mitochondria
replicate their numbers without synchrony to nuclear and cell division.
<p>
<b>Chloroplasts</b>.  Algal and plant cells have the chloroplast in which to
perform photosynthesis.  Like mitochondria, it is a double membraned organelle.
But within the double membrane are a series of flattened stacks of membrane-bound
vesicles.  These are not continuous with the inner membrane.  These are called
<b>thylakoids</b>.  Certain parts of the membranes are thicker than others, and
these thicker parts are also stacked one on another:  these are called <b>grana</b>,
and look like stacks of coins.  The fluid outside of the thylakoid vesicles
but contained within the inner membrane is called the <b>stroma</b>.  Chloroplasts
also have small circular DNA, their own chromosome.  Like mitochondria, chlorosplasts
can replicate themselves.
<p>
<b>Cytoskeleton</b>.  With the cell are fiber-like elements often composed of
polymers of materials that are just protein, or often proteins modified with
sugars and other molecules.  The fibers have names:  actin, microtubules,
intermediate filaments, and so on.  They may be involved in cell motility,
cell shape and structure stabilization, or even metabolite transport.  Many
specialized (differentiated) cell types, show cell polarity, so the movement
of materials from pole to pole is achieved by the skeleton.
<p>
<b>Cell Wall</b>.  The cell wall is a structure that is outside and envelopes
the plasma membrane.  It is not found in multicellular animals, but rather in
fungi, algae, and plants.  It is typically composed of polysaccharides
interacting to form a fibrillar (fibrous) structure.  The primary cell wall
of plant cells is generally cellulose.  Plants may secrete additional chemicals
to form a secondary wall composed of lignin.
<p>
<b>Cell Structure Is An Reflection of Function</b>.   The structure of a cell
will often reflect its function.  Cell in the pancreas will have a lot of
rough ER with secretory vesicles  with enzymes to be activated in digestion.
Testicular cells will have vesicles with a lot of lipid-soluble chemicals as
signals.  The cells in the leaves of plants will have the organelles set up
for photosynthesis and the formation and transport of sugars.  Root cells
will show the storage of a lot of starch.

<h2>The Nuclear Envelope</h2>
<p>
As indicated, the nuclear envelope is a double-membraned structure that has
continuity with the endoplasmic reticulum.  SEM micrographs of freeze-fracture
preps also show nuclear pores, which are composed of at least 50 different
proteins.  TEM studies involving injection of gold particles into cytoplasm
eventually showed that the gold was in the nucleus too, indicating that pores
are doors for molecules.  Later studies showed that passage is quite selective.
Clearly DNA does not pass through the pore.  Ribosomal RNA is synthesized in
the nucleus, and the proteins to make ribosomes are brought into the nucleus,
where ribosomes are assembled in the nucleolus.  The completed ribosome is
then exported to the cytoplasm.  Many proteins of all kinds related with some
function in the nucleus are imported to the nucleus from the cytoplasm.
The traffic is heavy:  about 500 molecules pass through each of the 3000-4000
pores every second.
<p>
<b>Nuclear Localization Signal (NLS).</b>  The first studies of proteins that
move into the nucleus were of viral proteins.  It was observed that changing
certain amino acids removed the ability of the protein to move to the nucleus,
and so nuclear-bound proteins had an NLS on them.  The protein <b>nucleoplasmin</b>
is important in chromosome assembly in the nucleus;  it has a globular core
and has a series of tails extending from it.  Injections into cytoplasm caused
the protein to be quickly localized in the nucleus.   Proteases were used to
digest the tails from the core, and these were purified into to fractions.
These were injected into cytoplasm again, and the protein cores stayed in the
cytoplasm and the tails ended up in the nucleus.  A sequence of 17 amino acids
was eventually found to contain the contain the NLS in the tail.
<p>
Passage through the nuclear pore complex is regulated by <b>importins</b> and
<b>exportins</b>.  Passage is also an ATP-dependent process.

<h2>Endomembranes</h2>
<p>
Proteins targeted to other organelles in the cell (mitochondria, chloroplasts)
also have signals.  Small molecules and ions diffuse randomly but larger ones
like proteins have their relocalization tightly regulated.  G. Palade and
coworkers studied how proteins traffic using pulse-chase experiments.  He
<q>pulsed</q> cells in a radiolabeled amino acid for a short time to label
proteins.  He would then replace the medium with the unlabeled amino acid, which
has the effect of producing unlabeled proteins.   Thus a population of molecules
are labeled and their fate can be tracked over time.  In the experiments,
radiolabeled leucine was pulsed 3 min, then the medium changed with unlabeled
Leu.  Cells can be prepared for electron microscopy and autoradiography to show
the movement of the label and track the proteins.
<p>
G. Blobel proposed that proteins bound for the ER and Golgi had a signal,
probably on the first few amino acids.  It was known that proteins that locate
in the ER had 20 extra amino acids present when there was only the ribosome and
no ER.  If there ER is present, these 20 amino acids are clipped off.  Recent
work showed that as the ribosome synthesizes a protein directed to the ER lumen,
the nascent protein binds to a <b>signal recognition particle (SRP)</b> in the
cytosol.  The SRP is a ribonucleoprotein (RNA + protein), and it has a receptor
on the ER membrane.  The synthesis of the protein continues, but the polypeptide
is directed into the ER lumen.  The entire complex of SRP and SRP receptor
snips off the signal sequence, and leaves the protein in the lumen.  Note that
ER lumenal proteins are also folded by molecular chaperones.
<p>
Palade then discovered how proteins in the ER are moved to the Golgi.  Using
labeling experiments and differential centrifugation, it was shown that vesicles
bud off the ER and then fuse into the cis face of the Golgi.  The proteins
undergo several modifications by glycosylating and deglycoslyating and other
enzymes.  The oligosaccharide modifications often dictate the transport/export
fate of the protein and the vesicle.
<p>
For example, mannose-6-phosphate bind to protein in membranes of certain vesicles.
The vesicles have proteins that interact with proteins in lysosomal membranes,
and thus Man-6P will target proteins that are destined for lysosomes.

<h2>Cytoskeleton</h2>
<p>
Ultrastructural analysis of cells shows fibrillar elements of various types:
<ul>
<li>actin filaments (microfilaments)
<li>intermediate filaments
<li>microtubules
</ul>
<p>
<b>Actin filaments</b> are the smallest in diameter.  They are formed by the
fibrillar polymerization of actin, a globular protein.  In animal cells,
actin is so abundant in its presence in the cytoskeleton that it is 5-10%
of the total protein by mass.  The filamentous actin is composed of a single
monoomer that forms two long strands that form a helical coil around one
another that is 7 nm in diameter and has helical periodicity of 37 nm.
The strands have a polarity, a plus and minus end.  Microfilament growth
occurs at the plus end and dissolves at the minus end in a process called
treadmilling.  Actin filaments are quite abundant under the plasma membrane.
Groups of filaments organize into long bundle or dense networks and are
linked to one another by other proteins.  These can help stiiffen cells and
give it shape by resisting tension (pulling).  Actin also interacts with
myosin in muscle contraction:  ATP binds to the myosin head, and is hydrolyzed
to ADP, and this causes the myosin head to interact with actin in a way that
causes sliding movement of the actin.  The movement of cells (cell motility)
is the result of actin-myosin interactions to push cytoplasm in one direction,
while actin polymerization is used to extend pseudopodia.  In cell division,
actin-myosin interactions cause a pinching of the membrane in a ring that
complete cell division, a process called cytokinesis.  In plant cells,
actin-myosin interactions move cytoplasm around the cell, in a process called
cytoplasmic streaming, moviing organelles and cytosolic fluid.  This is because
plant cells are formed with a cell wall that keeps them rigid.
<p>
<b>Intermediate filaments</b> refer to the size of the fibrils relative to
other structures; they have a diameter of 10 nm usually. Intermediate filaments
refer to a class of fibrils with different compositions, unlike microfilaments
and microtubules. There is no polarity, and their synthesis and breakdown does
not occur by treadmilling. The filaments have only a structural role. Filaments
belong to a family of proteins that are the keratins. They provide ability to
resist pressure and abrasion. 10 different types of keratins are produced and
they can form fingernails, toenails, and hair. The nuclear lamins that make up
the nuclear lamina are also intermediate filaments. They play a role in the
breakup and reassembly of the nuclear envelope. Some filaments project from the
nucleus through the cytoplasm to the plasma membrane, holding a shape to the
cell and keeping the nucleus at a relative position.
<p>
<b>Microtubes</b> are composed of two types of globular protein subunits,
&alpha;- and &beta;-tubulin.  These two join to form a dimer.  The dimers then
polymerize in a spiral manner to form a hollow tube that shows a plus-minus
polarity just like actin.  The tubule has a diameter of 25 nm.  Microtubules
are initiated from a microtubule organizing center, and grow outward.  Plant
cells have hundreds of these, but animal cells have only one, the
<b>centrosome</b>.  Centrosomes are actually two bundles of microtubules
called <b>centrioles</b>, in which the long axis of one orients at the midpoint
of the long axis of another centriole.  Interestingly microtubules do not
require centrioles from which to extend, and the real function of centrioles
is not known.  Microtubules function in providing stability and assist in
cell movement, actings like girders in a skyscraper.  They are required for
organellar movement, and the ER will not form a network without them.  In cell
division, microtubules are required for separating chromosomes to the daughter
cells.   A study of the squid giant axon showed the vesilces are transported
from the soma (where the main cell body and nucleus is) to the axon terminals.
In vitro experiments with purified microtubules, vesicles transported by them,
and ATP did not produce movement.  Further addition of materials showed that
a protein <b>kinesin</b> was required.  Kinesin forms a superfamily of proteins
of which kinesin-1 is a heterotetramer  with two heavy chains and two light chains.
Heavy chains have globular head forming a motor domain at the N-terminus and
this forms a stalk &alpha;-helical coiled coil ending in C-terminal tail domain
that associates with light chains.  The stalks intertwine to enable the dimerization.
The ATP-dependent movement likely has the head motor domain moving along the
microtubule alternately stepping forward, rather than an inchworm movement with
one head leading and the other catching up trailing.
<p>

<b>Flagella and Cilia</b>. These are present in both prokaryotes and eukaryotes,
 but the structures are radically different. Flagellin is the protein that 
composes the bacterial structure, while microtubules form the eukaryotic one 
with tubulin. The flagellum in bacteria rotate like a ships propeller, while 
eukaryotic structures undulate with a whipping back and forth. Additionally, 
flagella are enclosed in a plasma membrane in eukaryotes. These eukaryotic 
flagella are similar to <b>ciia</b> too, which also undulate. Unicellular 
eukaryotes either have cilia or flagellar, while multicellular eukaryotes have 
both: respiratory tract epithelia feature cilia while sperm have flagella in 
humans. The basic structure for both is the same though:  a 9 + 2 arrangement of
microtubules.  There are 9 microtubule doublets that form radially along the
circumference of the flagellum/cilium, and two central microtubules centrically
located.  The length of this structure is an <b>axoneme</b>, and the attachment
point is a <b>basal body</b>, having a structure identical to a centriole.
The 9 + 2 arrangement has a framework formed by proteins.  The protein
<b>dynein</b> forms between each microtubule doublet in the circumference to
link them, while there are spokes that connect each microtubule doublet to the
center where the two microtubules are.  Gibbons used an inhibitor of axoneme
action and found that no bending or utilization of ATP could occur.  EM
had shown the loss of arms had occurred and he showed the protein required was
dynein.   Flagella undulate when ATP binds dynein:  the phosphorylated dynein
changes shape and moves it along the microtubule similar to kinesin, but one of
the members of the doublet slides along the other, and this transmits force to
each of the links and the spokes.  This results in the bending of the flagellum.

<h1>Cell-Cell Interactions</h1>
<p>
The focus is now on the plasma membrane and how its surface is configured to
have the cell be part of community that forms a population among unicellar
species or tissues in multicellular species.  On the surface will be
oligosaccharides that modify proteins (glycoproteins) or which are connected to
phospholipids to form glycolipids.
<p>
<b>Cell Wall in Plants</b>.  The <b>primary cell wall</b> of the newly formed plant
cell is composed of <b>microfibrils</b> of cellulose that form multiple
criss-crossing layers.  Other oligosaccharides, <b>pectins</b>, become sandwiched
in the layers and they are considerably hydrophilic, keeping the cell wall moist
and fluid.  These pectins and gelatinous polysaccharides are produced in the rough
ER and Golgi, while the basic cellulose microfibril is polymerized on the
plasma membrane by an enzyme complex.  Plant cells acquire a high hyperotonicity
which increase the <b>turgor pressure</b> of the cell against the wall.  To
enable growth and elongation at times, plant cells secrete <b>expansins</b> which
cause the microfibrils to slide past one another for elongation and expansion.
When plant cells mature and no longer grow, they secreate material to form a
secondary cell wall between the plasma membrane and primary wall.  Leaf cells
on the upper surface might secrete a wax to prevent transpiration.  Cells that
form the stem may create a wall with significant lignin content, which involves
cross-linking the wall to stiffen it.
<p>
<b>Animal Cell Extracellular Matrix</b>.  Animal cells will secrete the materials
necessary for form fibers or basement membranes.  These are usually made of
protein, but can be a polymer of protein and sugars (proteoglycans).  The function
of the ECM is to be more elastic and flexible than is the case for cellulose and
lignin.  Synthesis of these materials is done in the rough ER with Golgi processing
and then exocytosis.  Some protein-bound polysaccharides are made by membrane
protein complexes.  The ECM varies with tissue type.  Bone and cartilage are
rich in ECM that make it rigid.  The ECM in lung tissue contains elastin to
enable tissue shape changes with breathing movements.  The ECM provides a
foundation upon which cells may move.  Actin filaments within the cell
can connect with transmembrane proteins called <b>integrins</b>, and integrins
being to the <b>fibronectin</b> component in the ECM, and fibronectin is
connected to collagen.  <b>Desmosomes</b> are structures that transduce
cytoskeletal connections.  Between cells are a lock-key interaction, and within
the cells, interemdiate filaments that connect to the desmosome anchor proteins.

<h2>Cell-Cell Connections and Communication</h2>
<p>
The plaque that forms on teeth is actually a biofilm that bacteria use to
colonize themselves in the oral cavity.  <i>P. aeruginosa</i> will secrete
a signal to draw other individuals to form a biofilm and protect against loss
of attachment.
<p>
Plant cells form a primary wall, but between each other they have a central
layber, the middle lamella composed of primarily gelatinous pectins.  Degradation
of the middle lameella occurs enzymatically, when flower petals or leaves
detach during the seasons.
<p>
Animals cells join in tissues with an intervening ECM and connect via integrins.
 In some tissues, particularly epithelial that form a layer forming a barrier
between internal and external tissues, they need a stronger cell-cell
connection. <b>Tight junctions</b> are where specialized proteins appear on the
plasma membrane near the surface of the epithelial cell that is on the mucosal
(external) side and bonds it tightly with other epithelial cells. Tight
junctions provide the tightest barrier to the diffusion of extracellular fluid
around cells, making the cells form a very interlocked barrier. The <b>
desmosome</b> is another structure that connects cell-to-cell by a small patch.
Proteins on the membrane between cells create an interlock, while connections
to the transmembrane proteins inside the cell are to intermediate filaments.
Thus desmosomes may represent the ability to produce coordinated connections
of cytoskeleton intercellularly.
<p>
<b>Cadherins</b>.  In experiments in the early 1900s, two different marine
sponge species had their cells dissociated and then the cells were observed
to see if they would reform into the same tissue.  In the 1970s it was supposed
that special proteins were involved.  Scientists isolated the membrane proteins
from certain cell types and then purified each protein.  They then injected the
purified protein into rabbit to produce <b>antibodies</b> to those proteins.
From the blood, they purified those antibodies to each protein using an
affinity purification procedure to the membrane protein.  They would then add
the purified antibody to a mixture of dissociated cells and observe whether
the cells re-associate (a no-antibody control should always show re-association).
Through this method, they hoped to identify the protein involved in cell-cell
adhesion.  In fact, the desmosome interlocking proteins were identified as a
class known as cadherins.
<p>
Intercellular tunnels that connnect the cytoplasm between two cells were found
for plants and animals.  In plants plasmodesmata are formation where the plasma
membrane between two cells join to become continuous, and thus opens a connection
between the cytoplasm.  Smooth ER even connects between these cells.  The opening
is not entirely barrier-free:  only specific proteins can pass, so the
plasmodesmata may be like a nuclear pore complex.  Proteins that pass through
are involved in intercellular coordination.  In animal cells <b>gap junctions</b>
effectively act as channel proteins do for a cell to the extracellular fluid:
they allow the passage of small molecules and ions, especially when cells form
a tissue that requires rapid coordinateed responses.

<h2>Cell Communication At A Distance</h2>
<p>
Multicellular organisms have cells and their tissues often separated from other
cells in other tissues separated at a distance and must often communicate
that a metabolic state exists or signal to other tissues to change their metabolic
state.  In these organisms, they use <b>hormones</b> to achieve this communication.
Typically hormones are small molecules that can move or diffuse rapidly, and
have processes that reduce their presence or existence when signalling or
stimulation is not to be done.  Examples of hormones and their molecular
nature and their function are shown in the table below.
<table>
<tr><th>Hormone  <th>Molecular Structure  <th>Function
<tr><td>ethylene  <td>H<sub>2</sub>C=CH<sub>2</sub> (gas)
  <td>stimulate fruit ripening, regulate aging
<tr><td>insulin  <td>51 amino acid peptide <td>stimulate glucose uptake
   by regulated tissues (liver, adipose tissue, muscle)
<tr><td>systemin  <td>18 amino acid peptide  <td>stimulate defenses against
  herbivores in the plant
<tr><td>estrogen  <td>steroid  <td>stimulate sexual characteristics in female
<tr><td>brassinosteroids  <td>steroid  <td>stimulate plant cell elongation
<tr><td>prostaglandins <td>modified fatty acid <td>numerous functions
<tr><td>thyroxine (T4) <td>modified amino acid <td>regulation of metabolism
<tr><td>FSH     <td>glycoprotein <td>production of ovum in females, sperm in males
<tr><td>auxin  <td>small organic molecule <td>elongation of stem in plants
</table>
<p>
Hormones bind to receptors whose numbers and affinity can be changed.  Drugs
acting as inhibitors can bind and block receptors to prevent hormone binding
and action.
<p>
The signals come to the surface of the cell.  If steroids, they might pass
through the membrane, then bind to a protein in the cytosol, which would then
translocate to the nucleus where the protein interacts with DNA to start
transcribing.  In other cases, they might bind protein receptors which undergo
a change at the surace, which then create a <q>second messenger</q> intracellularly
which causes changes in the signal.
<p>
<b>G proteins</b> are an example of such a class. A hormone binds to its
receptor at the surface. Consider epinephrine. It binds to a specific receptor
that is bound to a G protein of a special type called a stimulatory G protein
(G<sub>s</sub>) and composed of three subunits &alpha;, &beta;, and &gamma;.
GTP binds to and activates the G<sub>s&alpha;</sub>, which then binds to
another membrane protein, adenylate cyclase. Already bound to the G<sub>s</sub>
multimer was a GDP compound, and it was displaced by GTP. Note that the
G<sub>s&alpha;</sub>-GTP complex is held to the membrane by a C<sub>16</sub>
phospholipid. But G<sub>s&alpha;</sub> has an intrinsic GTPase activity: this
will hydrolyze the &gamma;-phosphate from the bound GTP and convert it to
still-bound GDP. G<sub>s&alpha;</sub>-GDP is inactivated, and it will retunrn
to a receptor-G<sub>s&beta;&gamma;</sub> complex and bind it, and await for
another hormone binding event to occur. The second messenger made is cyclic AMP
(cAMP) from ATP. A feature of a second messenger is that it must be small and
rapidly diffusible and made in large quantities, and it must have a system for
removing it quickly. An important feature of 2nd messengers is that they will
have different signalling effects in different cell types. Additionally more
than one 2nd messenger system may be active at one time.
<p>
<b>Receptor tyrosine kinases (RTKs)</b> are another class of signal tranducer.
Hormones bind to the receptor which requires a 2nd hormone-bound receptor to be
functional, forming a dimer. The tyrosine kinase domain is phosphorylated by
ATP, activating the catalytic domain. The RTK binds to the membrane protein <b>
Ras</b> through another briding protein. This results in GTP binding of Ras,
which then triggers phosphorylation of another protein, activating it. A
phosphorylation cascade begins.
<p>
These signals exist in plants too.  In wheat seeds, the embryo produces
gibberellic acid 1 (GA<sub>1</sub>) when growth starts.  GA<sub>1</sub> binds
to cells in the starch storage section and the intracellular cGMP levels rises,
stimulating transcription of the &alpha;-amylases.  GA<sub>1</sub> also
stimulates increases in intracellular Ca<sup>2+</sup>, which promotes membrane
fusion of vesicles in the endomembrane system through which the &alpha;-amylases
will pass as they are readied for secretion by the ER/Golgi axis.
<p>
<b>Signal termination</b>.  All second messengers need to be deactivated or
removed to attenuate the signal.  GTP is hydrolyzed to GDP in G proteins.
cAMP is hydrolyzed to AMP by phosphodiesterases.  Ca<sup>2+</sup> is pumped back
into its storage vesicles.  When signal termination fails, such as when Ras
protein becomes defective and does not convert GTP to GDP to shutdown, cells
lose control and their proliferation is not stopped, possibly causing cancer.
25-30% of all cancers may involve a defective Ras protein.

<h1>Respiration and Fermentation</h1>
<p>
The storage of potential energy in a catabolic process is into ATP.  ATP
uses that energy to cause phosphorylation, which can induce conformation
changes in proteins, and cause the protein to perform in a desired way.
ATP has considerable exergonic release energy at 7.3 kcal/mol.  Considerable
discussion must be paid to how ATP couples energy to reactions and how it
uses phosphorylation to achieve an aleration in a metabolic process.
<p>
<b>Reduction-oxidation reactions</b>.  These important class of reactions typically
drive the formation of ATP.   Redox reactions are all about transferring electrons
from one atom to another, and harnessing the energy in ATP.  A long explanation
is given in which the bond energy is altered when electrons are moved from one
atom to another.  Electron movements also accompany proton movements, so that
a hydride is basically transferred.  The oxidation of glucose to carbon dioxide
and water using oxygen will end up transferring 686 kcal/mole of energy.
This energy is coupled ultimately to the formation of ATP.

<h2>Glycolysis</h2>
<p>
An accidental discovery by Hans and Edward Buchner in late 1890s who were
trying examining ways to produce yeast extracts for baker's yeast for
commercial and medicinal use, they added sucrose to an extract intending it to
be a preservative from food decay. But instead it was broken down and turned
into alcohol. This revealed that fermentation could occur in vitro, until then
thought only to happen in vivo. They learned also that the reaction proceeded
to longer degree if phosphate was added. Fructose bisphosphate was found. In
fact all of the compounds were phosphorylated except pyruvate and glucose. When
the mixture was boiled, no fermentation occurred (1905). It was already known
that enzymes were heat sensitive, so the process was believed to depend on
enzymes. Most of the steps and reactions were eventually determined by Gustav
Embden and Otto Meyerhoff, after whom the pathway is named.
<p>
Ten chemical reactions were revealed to occur in glycolysis. Some energy forms
ATP from the coupled phosphorylation of ADP. Another reaction reduces NAD<sup>
+</sup>, producing NADH by transfer of a hydride (two electrons and a proton).
It is an electron carrier with reducing power. At the end of glycolysis,
pyruvate is formed with the ATP and NADH.
<p>
In the first three steps, two ATP molecules are actually consumed rather
than produced.  Hexokinase is a Mg<sup>2+</sup>-dependent enzyme phosphorylating
the inwardly transported glucose into glucose-6-phosphate.  Glucose-6-phosphate
isomerase then converts the aldohexose phosphate to the ketohexose phosphate
fructose-6-phosphate.  Phosphofructokinase (PFK) is like hexokinase, utilizing
ATP (depending on Mg<sup>2+</sup> again) to produce fructose-1,6-bisphosphate.
<p>
The enzyme aldolase then divides the 6-carbon-bisphosphate into two
3-carbon phosphates, one a ketotriose, dihydroxyacetone phosphate (DHAP) and the
other an aldotriose, glyceraldehyde 3-phosphate (GA3P).  The rest of the
steps require GA3P as an entry point, so DHAP must be converted to GA3P to
continue, which is done by triosephosphate isomerase.
<p>
In the next steps, the energy-yield reactions now occur. The -CH=O aldehyde in
GA3P is simply oxidized with a coupled phosphophorylation, producing 1,
3-bisphosphoglycerate (1,3-PGA), using an NAD<sup>+</sup>-dependent enzyme,
glyceraldehyde 3-phosphate dehydrogenase. That NADH can go on to produce 3 ATP.
The next step is by 3-phosphoglycerate kinase: from its name, this enzyme would
seem to catalyze the formation of 1,3-PGA from 3-PGA using ATP and producing
ADP, but in fact, the enzyme works in both directions, and in glycolysis, it
actually drives the formation of ATP rather than use it.  Note that 1,3-PGA
is actually an acid phosphate and the &Delta;<i>G</i>&prime;&deg; =
&minus;18.5 kcal/mol indicates that the formation of ATP is more favored.
<p>
Phosphoglycerate mutase is the next enzyme reaction, and enzyme with a forward
and reverse reaction to convert 3-PGA to 2-phosphoglycerate in a
Mg<sup>2+</sup>-dependent reaction.  An intermediate 2,3-bisphosphoglycerate is
actually formed, since the enzyme has a phosphate in its active site and
swaps between the 2-hydroxyl and 3-hydroxyl.  2,3-BPGA is actually vital in
regulating affinity of hemogloboin for oxygen.
<p>
In the next reaction, a very high energy intermediate is generated by the
dehydration of the 3-hydroxyl and a 2-hydride on 2-PGA.  This is carrried out by
enolase and forms phosphoenol pyruvate (PEP).  This high energy phospho compound
is has even more energy than ATP.  The hydrolysis of phosphate from 2-PGA has
a &Delta;<i>G</i>&prime;&deg; = &minus;17.6 kcal/mol;  for PEP, it is
&minus;61.9 kcal/mol.  The energy in PEP phosphate is recovered in the next
reaction by pyruvate kinase.  This is actually the reverse reaction of the
enzyme, with &Delta;<i>G</i>&prime;&deg; = &minus;31.4 kcal/mol.  Unfortunately
only one ATP is made, but the driving force of the reaction is to move 3-PGA
towards pyruvate formation as an entry point to Krebs cycle.
<p>
The net reaction of glucose &rarr; pyruvate is to produce 2 ATP + 2 NADH.
Glycolysis is tightly regulated.  Pasteur discovered more glucose being consumed
under anaerobic rather than aerobic conditions (the <q>Pasteur effect</q>).
This was because much less ATP is produced by fermentation, so to sustain the
required level of ATP, the organism must break down more glucose.  Points of
regulation of glycolysis are at the enzymes hexokinase, phosphofructokinase
isoform 1, and pyruvate kinase.  High levels of ATP inhibit PFK.  The strange
thing is that ATP is also a substrate.  The 3D structure revealed however that
PFK has two binding sites.  The high affinity site is for the substrate portion
near the active site.  This means it only takes low concentrations of ATP to
occupy it and drive the reaction.  The low affinity site however is on a
separate part of PFK:  it requires high concentrations to bind it, and when it
does reach high levels, the enzyme is inhibited.
<p>
<b>Pathological glycolysis</b>.  Cells of tumor tissues undergo glycolysis at
rates far above normal tissues, because cells in solid tumors are generally
hypoxic:  in general, a cell more than 100-200 &micro;m away from a capillary
will show greater rates of glycolysis.  Additionally cancer cells show lower
presence of mitochondria, contributing to the need for glycolysis.

<h2>Krebs Cycle</h2>
<p>
Once pyruvate is formed, what happens next depends on whether the organism is
aerobic or metabolizing in an anaerobic way.  Eukaryotes have mitochondria,
and these are the centers of aerobic metabolism.
<p>
Pyruvate crosses the mitochondrial membranes into the matrix.  Contiguous with
the inner membrane and created by tubular invaginations are cristae.  These
effectively create a large surface area between matrix and the space within the
cristae.  The crossing through the outer membrane is via pores that permit
molecular exchange.  The inner membrane crossing requires a carrier protein that
consumes ATP to move pyruvate.  Once inside, pyruvate is bound to pyruvate
dehydrogenase in a reaction that involves Coenzyme A, which binds an acetyl
group to form acetyl-CoA and CO<sub>2</sub>.  The reaction involves multiple
steps with the formation of NADH, although the protons do not come from
pyruvate.  The reaction is highly regulated.  When phosphorylated by regulatory
protein kinases, the enzyme complex is inhibited; this occurs with high ATP
levels.  In contrast, when NAD<sup>+</sup> or AMP or even CoA levels are high,
the reaction is accelerated.
<p>
In elucidating the rest of the reactions that would produce CO<sub>2</sub> from
glucose, it was known that there were several carboxylic acids involved and
none of them appeared to accumulate, although the order of their production
could be determined.  It was Hans Krebs who made the determination that the
reactions did not occur in a linear fashion but rather likely occurred in a
cycle, explaining how they would no accumulate.  It was known that oxaloacetate
was the endpoint of the pathway were it a linear one (it was the most oxidized
of the carboxylic acids).  The proof was found when oxaloacetate was mixed
with pyruvate alone, and formed citrate.  Since citrate was thought to be
an intermediate in the linear pathway, if it was formed with pyruvate, it must
occur cyclically.  Krebs hypothesis was further confimed with the use of
radiolabeled carbon atoms in the early 1940s.  It also confirmed that two
CO<sub>2</sub> molecules were produced in the cycle, and which carbons they
came from.  Later work showed that it was acetyl-CoA that was all that was needed
to start the pathway.  The eight raactions that occur along with pyruvate
dehydrognase are:
<ol>
<li><b>citrate synthase</b>: acetyl-CoA condenses with keto group of
oxaloacetate (OAA) to yield citrate and CoA-SH.
<li><b>aconitase</b>:  this induces a dehydration-rehydration reaction
in which the hydroxyl (-OH) group is moved from carbon-3 to carbon 2 to form
isocitrate.
<li><b>isocitrate dehydrogenase</b>:  this NAD<sup>+</sup>-dependent enzyme
oxidizes isocitrate to &alpha;-ketoglutarate, causing spontaneous decarboxylation
and formation of NADH.  Note that the decarxylation is of the CO<sub>2</sub>
that is &beta; to the formed &alpha;-keto group.
<li><b>&alpha;-ketoglutarate dehydrogenase</b>:  a reaction very much like that
of pyruvate dehyrogenase, reducing NAD<sup>+</sup> with the release of
CO<sup>2</sup> and formation of a CoA product, succinyl-CoA.
<li><b>succinyl-CoA synthetase</b>:  the reverse of this reaction actually
occurs in the direction of Krebs cycle, with the thioester energy harvested
to produce an ATP but via formation of a GTP intermediate from GDP, then the
transfer of the phosphate.  Succinate and CoA-SH are produced along with the
ATP.
<li><b>succinate dehydrogenase</b>:  this FAD-dependent enzyme performs a
true dehydrogenation, forming FADH<sub>2</sub> which goes on to the electron
transport chain.  The desaturated dicarboxylate product is fumarate.
<li><b>fumarase</b>:  fumarate is hydrated on the C=C bond to produce the
chiral compound malate
<li><b>malate dehydrogenase</b>:  in an NAD<sup>+</sup>-dependent reaction
similar to isocitrate denydrogenase in which an &alpha;-hydroxyl is
oxidized to an &alpha;-keto, oxaloacetate is formed.
</ol>
<p>
The net production is of three NADH, one FADH<sub>2</sub>, and one ATP (via
GTP).  The TCA cycle is regulated based on ATP levels.  Citrate synthase
is inhibited by high levels of ATP.  With the dehydrogenases, as NADH levels
increase, they occupy the active site an compete with NAD<sup>+</sup>, preventing
oxidation.  In the complete conversion of glucose to six CO<sub>2</sub> molecules, a
total 10 NADH, 2 FADH<sup>2</sup>, and 4 ATP are produced.  Ultimately, all of
the energy is produced as ATP.

<h2>Electron Transport Chain and Oxidative Phosphorylation</h2>
<p>
Proteins in the inner mitochondrial membrane will
process NADH and FADH<sub>2</sub> in a series of redox reactions that will
generate ATP, the ultimate energy carrier.  In bacteria, this same set of reactions
occurs in their plasma membrane.  As electrons pass through from membrane protein
to another, the conformation changes that is coupled to protons being obtained
from the matrix, and then released but to the other side of the membrane.  This
has the effect of creating a proton gradient.  An electrochemical gradient builds
up:  the protons are then passed to an ATP synthase whose conformation is again
changed, but this is coupled to the formation of ATP from ADP and phosphate.
In aerobic organisms, the final electron pair acceptor in the respiratory chain
is oxygen, forming water as the end product.
<p>
These membrane proteins in the electron transport system have flavin coenzymes,
iron-sulfur complexes, or iron-containing heme groups. The ETC has these
components in which electrons are passed in this order:
<ul>
<li>NADH dehdyrognase:  a flavin mononucleotide (FMN)-containing enzyme
<li>Fe-S protein (NADH side)
<li>Ubq (Q)*
<li>cytochrome <i>b</i>
<li>Fe-S protein
<li>cytochrome <i>c</i><sub>1</sub>
<li>cytochrome <i>c</i>
<li>cytochrome <i>a</i>
<li>cytochrome <i>a</i><sub>3</sub>
</ul>
<p>
<b>Ubiquinone</b>, or <b>coenyzme Q</b>, or just Q, is a lipid-soluble
non-proteinaceious molecule that can traverse both sides of the inner
mitochondrial membrane without being bound to protein. It is a quinone ring
with long tail of isoprene monomers. Quinones play central roles in all
electron transport systems in mitochondria (ubiquinone), chloroplast
(plastoquinone), and bacterial (menaquinone) membranes. They are involved in
one-electron transfers, forming the semiquinone radical QH<sup>&bull;</sup>, or
two-electron transfers to form the quinol (QH<sub>2</sub>). It carries both the
electrons and protons.
<p>
The <b>cytochromes</b> are all heme proteins binding Fe atom which is 2+ in the
reduced state. There are three classes, <i>a, b</i>, and <i>c</i>, named 
because of their absorption spectra. The cytochrome <i>a</i> class absorbs near 
600 nm, type <i>b</i> near 560 nm, and type <i>c</i> near 550 nm. Usually they 
are named for their absorption maxima (e.g., cytochrome <i>b</i><sub>562</sub>).
The absorption maxima differ largely because of the side groups on the 
tetrapyrrole function of the heme ring: <i>b</i> cytochromes usually just have 
methyl, acetyl, and allyl groups (iron protoporphyrin IX), but <i>a</i> 
cytochromes can have one formyl and a trimer of isoprenes (Heme A). The heme
groups in types <i>a</i> and <i>b</i> are not covalently bound to their
apopproteins, but type <i>c</i> is by Cys sulfhydryls. While types a and b and
some types of c are integrated in the membrane, mitochondrial cytochrome <i>
c</i> is actually a soluble protein that binds on the outer surface of the
inner membrane electrostatically.
<p>
The <b>iron-sulfur</b> proteins effectively have a cubic arrangement of
Fe and S atoms, in which the Fe atoms can number 1, 2 or 4, called Rieske
iron-sulfur proteins.  Fe can be coordinated with either Cys S atoms or His
N atoms.  There are eight Fe-S proteins in mitochondrial ETC.  Each Fe atom
participates in one-electron transfer:  Fe<sup>3+</sup> &rarr; Fe<sup>2+</sup>
&rarr; Fe<sup>3+</sup>&nbsp;...  Reduction potential varies from &minus;0.65 V
to +0.45 V.

<p>
NADH and FADH<sub>2</sub> differ in their reducing
power, with FADH<sub>2</sub> having less reducing power.

<p>
Note that at Ubq, it is reduced by a separate entry point for FADH<sub>2</sub>
produced by succinate dehydrogenase, which delivers its electron pair by a
different Fe-S protein.
<p>
The chain order was basically determined by using inhibitors (poisons) known
the bind to certain sites.  When antimycin A was included in the reaction,
it could be determined that levels of reduced Ubq and cytochrome <i>b</i>
accumulated.  The change in free energy from NADH to oxygen is 53 kcal/mol.
<p>
Study of the structure of these ETC components in the membrane reveals an
organization into <i>four</i> large complexes, along with Coenzyme Q (Q) and
cytochrome <i>c</i>.  In looking at the structure, the outer membrane is
dissolved with digitonin.  The inner membrane vesicle is purified and then
osmotically ruptured.  A detergent is added and the mixture separated by IEC
into the four complexes.
<ol>
<li>Complex I: the <b>NADH dehydrogenase complex</b> (also <b>NADH:ubiquinone
oxidoreductase</b>) actually oxidizes NADH to NAD<sup>+</sup>. pumping four
protons vectorially across the membrane. It then reduces Q in a way that takes
two more protons from the matrix side, effectively taking four protons from the
matrix by its action. The mitochondrial complex is composed of 43 different
subunits (14 in prokaryotes) with a total mass of 850 kDa. The proteins contain
FMN-type flavoprotein and six Fe-S centers. High-resolution EM shows an
L-shaped protein with one arm in the membrane and the other in the matrix.
Useful inhibitors of Complex I are amytal, rotenone, and piercidin A.
<li>Complex II: this is the <b>succinate dehydrogenase complex</b> which
provides the reduction of FADH<sub>2</sub>. It does not pump protons, but does 
interact with Q on the matrix side and reduces it, taking protons from the 
matrix. It is composed of 4 subunit proteins with a total 140 kDa mass: one is 
a heme <i>b</i> protein, and the other a Ubq-binding protein. Two other 
subunits extend into the matrix (cytosol in prokaryotes) and have three 2Fe-2S 
centers, bound FAD, and succinate binding site. The path of electrons ges from 
succinate &rarr; FAD &rarr; Fe-S centers &rarr; Q-binding site, a distance of 
40 &Aring;, but the component distances are not more than 11 &Aring; in this 
small chain. The 3D structure of the <i>E coli</i> Complex II has been nicely 
solved. The function of heme <i>b</i> may not be in pathway transfer but in 
moving electrons from succinate to O2, generating reactive oxygen species 
(H<sub>2</sub>O<sub>2</sub>, O<sub>2</sub><sup>&bull;&minus;</sup>).
<li>Complex III: this is the <b>ubiquinone:cytochrome <i>c</i> oxidoreductase
complex</b> interacts with Q, but on the intermembrane space (cristae space) 
side, cause Q to become oxidized and release its protons. It passes the 
electrons over to cytochrome <i>c</i> (two molecules of them). It is composed 
of 11 subunits with a total of 250 kDa, and the proteins contain heme and Fe-S 
proteins. It actually moves four protons: those released by oxidation of QH<sub>
 2</sub> and to protons vectorially moved by the protein complex itself. There 
is a cytochrome <i>b</i> with two hemes (<i>b</i><sub>L</sub> and
<i>b</i><sub>H</sub>). There is also a cytochrome <i>c</i><sub>1</sub> with
one heme (not the same as the soluble cytochrome <i>c</i> product).  The complex
is actually a dimer of these proteins and cytochromes.  The complex binds Q
in two places, on the matrix and the intermembrane space side.  Q moves between
the matrix and intermembrane sides formed by a pocket in the dimer.
<li>Complex IV: this is the <b>cytochrome (<i>c</i>) oxidase complex</b>, which
oxidizes cytochrome <i>c</i> and transfers the electrons to the O<sub>2</sub> 
molecule. The mitochondrial complex is composed of 13 subunits with total 204 
kDa mass and composed of heme proteins as well as Cu<sub>A</sub> and Cu<sub> 
B</sub> proteins coordinated by Cys sulfhydryls; the prokaryotic form has 3-4 
subunits. Functionally, it is only three of the subunits that move the 
electrons while the other ten proteins envelope those three subunits. The path 
of electron movement starts with two reduced cyt <i>c</i> molecules passing 
electrons Subunit II (not I) containing the binuclear Cu<sub>A</sub>, which 
takes the electrons. The electrons are then passed to heme <i>a</i> in Subunit 
I, and then to Fe-Cu center with cyto <i>a</i><sub>3</sub> and Cu<sub>B</sub> 
present, all of these also in Subunit I. The O<sub>2</sub> molecule is bound to 
cyto <i>a</i><sub>3</sub> and reduced to O<sub>2</sub><sup>2&minus;</sup>,
a peroxy ion, from the Fe-Cu center.  Two more electrons are brought in from
cyt <i>c</i> oxidation, and then four protons are pulled in from the matrix
side to form two H<sub>2</sub>O molecules.
</ol>

<h2>Making ATP:  ATP Synthase</h2>
<p>
In studying the electron transport chain, it was not shown that ATP could be
formed by the addition of ADP. Mitchell in 1961 proposed instead that the
electron train transported protons across a membrane, from the matrix to the
intermembrane space or interior of cristae. A large electrochemical gradient
would form, and this would drive the activity of the ATP synthase. This is the
<b>chemiosmotic hypothesis</b>. Experimentally, it could be established as true
by creating artificial vesicles containing the ATP synthase with and without
bacteriorhopsin, which is also a membrane protein that functions as a proton
pump in the presence of light. In creating the vesicles, ADP and P<sub>i</sub>
would be included because ATP synthesis might possibly occur inside the vesicle.
 The vesicles would be illuminated: in theory protons would be pumped out of
the vesicle. The gradient would then drive the synthesis of ATP inside the
vesicle, and the formation of ATP could be assayed.  The studies of the
mitochondrial and bacterial protein complexes that pump protons described above
verified those studies, although they came decades later.
<p>
In 1960 Racker had experimented with vesicles formed from mitochondrial
membranes, with some vesicles formed inside out.  Membranes of the vesicles
under EM showed a studded appearance.  The bodies had a lollipop shape, with a
stalk in the membrane and the knob sticking out.  Vibration or use of urea
resulted in the knobs and stalks falling off.  Experiments with the dissembled
stalks and knobs in the presence of ATP caused its hydrolysis to ADP and
P<sub>i</sub>.  Vesicles with just the base had no effect on ATP, but the base
did move protons across the membrane.  The entire complex was thus an
ATP synthase, with two components:  F<sub>1</sub> featured the knob with ATP
hydrolyzing ability, and the membrane-bound, proton transporter was F<sub>0</sub>.
The model was that protons flowing through F<sub>0</sub> cause the stalk to
spin.  The attachment of long actin filaments to the stalk using a videomicroscope
reveals that the knobs can rotate at 350 rev/second.
<p>
Research shows that about 30 molecules of ATP are produced for each molecule
of glucose fully oxidized.  26 molecules are produced by the mitochondrial
ATP synthase. (In the past, the yield was calculated to be 36 ATP which was
not determined experimentally, but later <a
href="http://www.ncbi.nlm.nih.gov/pubmed/2012815">Hinkle</a> showed that the
yield was far less and actually varies with conditions in the cell.)
<p>
The final electron acceptor in all eukaryotes is oxygen, as all mitochondria
carry out oxidative phosphorylation.  Some bacteria and archaea will use
different electron acceptors from oxygen and electron donors not glucose.
H<sub>2</sub>, H<sub>2</sub>S, and CH<sub>4</sub> and ohter inorganic
compounds will serve as electron donors.  NO<sub>3</sub><sup>2&minus;</sup> and
SO<sub>4</sub><sup>2&minus;</sup> can act as acceptors.  However all such
reactions create proton gradients that are used to drive ATP synthesis.
Oxygen molecule has a high electronegativity and is therefore a more effective
acceptor.  Because the potential energy is larger with molecular oxygen, more
ATP can be generated than when the acceptor is another molecule.

<h2>Fermentation</h2>
<p>
This is the pathway undertaken when oxygen is not available as an acceptor.
Glycolysis produces a net amount of ATP and also NADH.  But NADH cannot
accumulate, as there is no unlimited supply, and it needs to regenerate
NAD<sup>+</sup> if glycolysis is to proceed at all.
<p>
All eukaryotes make use of generating ATP when oxygen is low or unavailable.
Skeletal muscle used in a sprint will have enough glucose but not enough
oxygen.  In muscle. the pyruvate will accept the NADH generated in glycolysis
by being reduced to lactate.  This type is lactic acid fermentation.  In yeast
however, the 3-carbon pyruvate is converted to acetaldehyde with loss of
CO<sub>2</sub> and then the acetaldehyde is reduced with the NADH using alcohol
dehydrogenase in the reverse direction to form ethanol.

<h2>Cellular Metabolism in Context</h2>
<p>
Glycolysis, Krebs cycle, and the oxidative phosphorylation system all have
points where other metabolic pathways enter and exit.  These can be
anabolic, in which smaller components are used to build larger components,
often using ATP or energy in other forms, or they can be catabolic, with
products that become waste and often yielding ATP.
<p>
Apart from carbohydrates such as glucose, other biomolecular classes include
proteins and fats, which can be catabolized.  With fats, they are catabolized
primarily to glycerol, which enters glycolysis as GA3P and as acetyl-CoA, which
can enter Krebs cycle.  With proteins, they are broken down to constituent
amino acids, and then the amino acids catabolized usually into pyruvate and
acetyl-CoA.
<p>
In anabolic pathways, biosynthesis of DNA, RNA, phospholipids, amino acids
and fats will occur.  Humans can synthesize about half of the 20 amino acids
they require, the rest must come from the diet.  In synthesizing fats, it all
starts from acetyl-CoA.  To synthesize the ribose and deoxyribose for RNA and
DNA, it starts with glycolytic intermediate G6P being oxidized to
6-phosphogluconate by glucose-6-phosphate dehydrogenase.

<h1>Photosynthesis</h1>
<p>
About 3 billion years ago a bacterial cell capable of transducing light energy
to chemical energy likely formed.  Organisms that can make biomolecules from
simpler non-biomolecules and ions are called <b>autotrophs</b>.  In contrast
organisms that derive their energy from the breakdown of the biomolecules
produced by autotrophs are classed <b>heterotrophs</b>.  Autotrophs produce
what they can via photonsynthesis.  Since heterotrophs cannot survive without
autotrophs, it is a safe bet that photosynthesis emerged first before glycolysis.
<p>
Photosynthesis in the green part of plants was known in the 1770s.  Sunlight,
carbon dioxide, and water were required.  Oxygen was produced as a byproduct.
By 1840s biologists figured out that sunlight was converted to the chemical
energy of carbohydrates.
<p>
In the 1930s van Niel was studying photosynthesis in purple sulfur bacteria
and determined that growth could occur without the presence of sugars, and
therefore growth was autotrophic.  He noticed that sunlight and H<sub>2</sub>S
were required.  Oxygen was not produced, but rather elemental sulfur (S) was
produced:
<p class="eqnserif">
CO<sub>2</sub> + 2 H<sub>2</sub>S + light &rarr;
(CH<sub>2</sub>O)<sub><i>n</i></sub> + H<sub>2</sub>O + 2 S
<p>
This work revealed that CO<sub>2</sub> and H<sub>2</sub>O were in fact not
linked in the production of carbohydrate.  It showed that H<sub>2</sub>O was
actually a product and also that CO<sub>2</sub> did not become O<sub>2</sub>.
From this work, it was hypothesized that in photosynthesis producing O<sub>2</sub>,
the O atoms must come from H<sub>2</sub>O.
<p>
Experiments with isolated chloroplasts in light and no CO<sub>2</sub> did show the
evolution of O<sub>2</sub>.  Algae and plants were treated with
H<sub>2</sub><sub>18</sub>O labeled water, and the oxygen molecule was labeled,
and it was light-dependent.  Between 1945-55, Calvin and coworkers used
<sup>14</sup>CO<sub>2</sub> in studies with algae to identify labeled compounds,
leading them the metabolic sequence in which carbox was fixed into sugars.
<p>
<b>Chloroplasts</b>.  In photosynthetic eukaryotes, chloroplasts serve to
harness the light energy in green-colored pigments.  Most leaf cells have about
40-50 of these organelles, and 1 mm<sup>2</sup> has 500,000 of them.  Isolated
chloroplasts produce O<sub>2</sub> in sunlight.  In the 1950s EM showed the
chloroplasts to be membrane rich.  Like mitochondria, it had a double membrane,
one inner and outer, but the inner did not evaginate like cristae.  Inside are
vesicles called <b>thylakois</b>.  At certain point the thylakoids are thick
and membranes abutt in <b>grana</b>.  The fluid outside the thykoid vesicles
is called the <b>stroma</b>; within the thylakoids, the space is the <b>lumen</b>.
<p>
Pigments absorb the light and transmit the energy of absorption to a process
that converts to chemical energy.  These pigments can be isolated by grinding
leaves in a solvent like methanol with perhaps hexane or petroleum ether.
The extract is spotted on a thin layer of porous material and developed in a
solvent mixture of various polarities.  A typical separation will have
carotene, pheophytin, chlorophyll <i>a</i>, chlorophyll <i>b</i>, and carotenoids,
in order of highest to lowest mobility.  If a pigment appears looks green, the
it is reflecting the green light and likely absorbing blue and red light (both
ends of the spectrum).  Carotenoids absorb in the blue and green parts of the
spectrum and thus appear yellow, orange or red.  In contrast the chlorophylls
absorb in the blue and red end and look green.
<p>
In assessing which part of the spectrum drove photosynthesis, Engelmann placed
filamentous alga on glass slide illuminated with spectrum of colors.  He then
placed bacterial cells that would be attracted to oxygen, and found them
gathered in the blue and red parts of the slide, and thus concluded this was
the <b>action spectrum</b> of photosynthesis.  This also indicated that the
chlorophylls were likely responsible for photosynthesis.
<p>
However, carotenoids serve a function as <b>accessory pigments</b> as they
transmit energy to chlorophyll. They are composed of carotenes and xanthophylls.
&beta;-carotene gives carrots their orange color; and zeaxathin is a
xanthophyll giving corn kernels their color. These appear in chloroplasts, and
the color emerges after the chlorophylls degrade. An important function of
carotenoids may be in protecting chlorophylls from rapid degradation, since
herbicides that prevent carotenoid synthesis cause plants to turn white and die.
 Carotenoids are particularly good at detoxifying free radicals that form in
light reactions. Another interesting finding is the study of <i>Arabidopsis
thaliana</i> unable to synthesize flavonoids, normally found in leaf cell
vacuoles. These molecules absorb UV light, preventing damaging effects. When
not present, the plant also withers and dies. Thus with carotenoids, their
flavonoides have a protective function.
<p>
The chlorophylls have a molecular structure designed to absorb a photon of
light and to transmit its energy for excitation of an electron.  The chlorophylls
work well in absorbing light wavelengths in the red and blue end of the spectrum.
Chlorophyll can absorb photons and release the energy in several ways.  It might
deal with it as heat, or it might release a photon of a slighly longer wavlength
as fluorescence.  In fact 2% of photon absorptions come out as fluorescence.
The rest is dealt with another way.

<h1>Photosystems I and II</h1>
<p>
<b>Antenna Complex</b>.  These are a set of chlorophylls that absorb energy
in the blue and red region and transmit it by resonance along the bonds of the molecule
to a special chlorophyll molecule.  These antenna chlorophylls constitute part
of a <b>photosystem</b> and in effect vibrate to direct the energy to a
chlorophyll constituting the <b>reaction center</b>.  At the reaction center,
the electron in chlorophyll is excited to the point that it is transferred to
an electron acceptor.  The acceptor catches the energy which will be converted
to chemical energy.
<p>
It was discovered in the 1950s that light at 680 and 700 nm was critical
for photosynthesis.  Emerson illuminated one or the other and found the rate
of photosynthesis moderate.  But when both wavelengths were used, the rate
increased dramatically.  As the increase was multiplicative rather than additive,
it was a mystery why this was so.  Then Hill and Bendall looked at the data
and speculated that there were two reaction centers, later named photosystem
I and II (numerals are in order of discovery).
<p>
The photosystems were studied in bacteria, which is useful because they exist
in only one form rather than having two photosystems.  The photosystem in purple
sulfur and nonsulfur bacteria was studied, which has many components similar
to photosystem II in cyanobacteria, algae and plants.  In PSII, <b>pheophytin</b>
has a central role, which is a chlorophyll without a coordinate Mg<sup>2+</sup> ion.
Pheophytin plays the role of electron acceptor, whereas a reaction center
chlorophyll is the donor and oxidized in light.
<p>
From pheophytin, electron transport to plastoquinione in one-electron transfers
near the stromal side of the thylakoid vescicle. The PQ is then shuttled the
lumenal side where it is reoxidized by a cytochrome complex, which contains an
array of iron- and copper proteins. The oxidation leads to the release of
protons to the lumenal side. Again these protons form an electrochemical
gradient whose energy drives ATP synthesis.  Experimental determinations show
that lumenal pH can drop to as much as 5 while stromal pH is 8, representing
a 1000-fold difference in concentration.  The synthesis of ATP driven by
photosynthesis is called photophosphorylation.
<p>
The bacterial photosystem chlorophyll that was oxidized is again reduced by
the electrons as they pass through the cytochrome complex.  Thus the electrons
move in a cyclical fashion and just generate ATP.
<p>
But electrons can be used to drive the synthesis of organic molecules with
higher free energy from reduction.  These electrons cannot return in a cyclical
fashion but require an electron donor to replace electrons lost by a reaction
center chlorophyll.  Photosystem II in plants produces a reaction center
oxidized chlorophyll, but it obtains electrons from water:
<p class="eqn">
2 H<sub>2</sub>O &rarr; 4 H<sup>+</sup> + 4 e<sup>&minus;</sup> + O<sub>2</sub>
<p>
Those electrons are reclaimed by the reaction center chlorophyll.  Only photosystem
II has the capability of oxidizing water.  When it evolved, it enabled the
formation of aerobic heterotrophs that would dominate the planet.  In fact, if
the chemistry involved in photosystem II action can be fully understood, it
can generate both O<sub>2</sub> and H<sub>2</sub> for clean fuel cars.
<p>
<b>Photosystem I</b> analogue was studied in heliobacteria (<q>sun bacteria</q>). These
bacteria take the electrons from their photosystem and use them to reduce
NAD<sup>+</sup> rather than pumping protons. In cyanobacter, algae and plants,
the coenzyme is actually NADP<sup>+</sup>. With photosystem I, the reaction
center chlorophyll is oxidized and the electron accepted by a series of
iron-sulfur proteins. The molecule <b>ferredoxin</b> accepts the electrons, and
then interacts with a protein ferredoxin/NADP<sup>+</sup> oxidoreductase, also
called NADP<sup>+</sup> reductase.
<p>
Since photosystem II oxidizes water and generates proton gradients to drive
ATP, and photosystem I takes electrons and reduces NADP<sup>+</sup> to make NADPH,
how do they interact.  The Hill and Bendall Z scheme describes it.  In it
P680 (the chlorophyll reaction center in PSII) is oxidized and is accepted by
pheophytin.  The quinones and cytochromes move the electrons through to an
acceptor called <b>plastocyanin</b> (PC), which receive electrons from the
cytochrome complex.  Plastocyanin diffuses in the thylakoid lumen and then
reduces P700 chlorophyll reaction centers.  These are the chlorophylls of
PSI.  In fact PC can shuttle 1000 electrons per second between the photosystems.
This Z scheme represents a noncyclic flow of electrons that is important in
producing NADPH that will be used in the biosynthesis of carbohydrates from
carbon dioxide.
<p>
If only 680 nm light is used, only PSII can run maximally.  PSI runs but its
efficiency is reduced without the far red 700 nm light.
<p>
<b>Cyclic photophosphorylation</b>.  The PSI can also be used similar to the
bacterial system that just generates ATP through proton pumping.  In a cyclic
fashion P700 is oxidized to its acceptor and the acceptor passes electrons to
ferredoxin.  But the ferredoxin does not react with the NADP reductase: instead
it passes electrons on to PQ, which then passes them to the cytochrome complex
to PC, and PC returns the electrons to P700 again.  The energy is converted into
the pumping of protons.  This will generate ATP for reducing CO<sub>2</sub>.
<p>
The spatial organization of the complexes in the thylakoids shows PSII is more
abundant in the grana, while PSI and ATP synthase are in non-granal portions of
the membrane.   The reason for this organization is not understood entirely.

<h2>Carbon Fixation: Calvin Cycle</h2>
<p>
The nature of how carbon was fixed (from CO<sub>2</sub> into sugar) was studied
using pulse-chase labeling of green algal cultures with <sup>14</sup>CO<sub>2</sub>.
After various periods of time (5-60 s), cells were extracted and the organic compounds
resolved by 2-dimensional chromatography and the 2D separation exposed to
X-ray film.  The spots are identified by comparing resolution of known standards.
The sequence of labeled intermediates that appear over time could be used to
determine the pathway.
<p>
The study showed that 3-phosphoglycerate was the first element to be dominantly
labeled.  In fact it is an intermediate in glycolysis.  At first Calvin
considered that CO<sup>2</sup> was adding to a 2-carbon metabolite, but that
could not be found.  Then Calvin considered the possibility that a 5-carbon
metabolite might be carboxylated to form a 6-carbon intermediate that would
then be divided into two 3-PGA molecules.
<p>
This 5-carbon metabolite is <b>ribulose-1,5-bisphosphate</b> (RuBP).  The enzyme
that catalyzes the carboxylation of RuBP was found by Weissbach and coworkers
using ground spinach leaves.  Proteins were purified, and each purified fraction
tested to see which could fix <sup>14</sup>CO<sub>2</sub> into RuBP and form
3-PGA.  They found the enzyme constituted 10% of the total protein mass of the
leaf.  it is <b>ribulose-1,5-bisphosphate carboxylase/oxygenase</b> but is called
<b>rubisco</b> for short.  The 3D structure shows a cubic enzyme with a total
of eight active sites.  The enzyme is quite slow however, fixing only three
CO<sub>2</sub> molecules per second.  Additionally, rubisco will cause RuBP
to react with O<sub>2</sub>.  It was thought that rubisco retained this
chemistry from an enzyme that existed before there was oxygenic photosynthesis.
However, it may play a role in <b>photorespiration</b>:  the reaction forms
one 3-PGA and one 2-phosphoglycolate.  The latter will generate CO<sub>2</sub> again and
consume ATP in processing it.  Photorespiration will be favored when O<sub>2</sub>
is at high levels and CO<sub>2</sub> levels are low.
<p>
<b>Carbon Dioxide In The Leaf</b>.   The underside of the leaf surface usually
has a set of cells that control the entry of gases into the mesophyll.
Specialized structures called stoma (plural stomata) are composed of guard
cells that will open to form a pore.  If CO<sub>2</sub> levels are low inside
the leaf, proton pumps in the guard cells will establish a charge gradient,
and this results in K<sup>+</sup> entry into the guard cells.  Osmosis causes
the guard cells to swell, causing pore opening.
<p>
<b>C<sub>4</sub> Plants</b>.  Kortschak and Karpilov repeated the work of the
Calvin group to establish the metabolites and the cycle in leaves of sugarcane
and corn.  However, they found that the carbon was incorporated into malate
and aspartate.  It was understood that phosphoenolpyruvate (PEP) could be
carboxylated, using <b>PEP carboxylase</b>.  This enzyme is abundant in leaf
mesphyll, while rubisco was still present but in the bundle sheath cells
that are closer to the vascular interior.  Hatch and Slack proposed a model
in which fixed carbon is transported as an C<sub>4</sub> intermediate, and then
released as CO<sub>2</sub> into rubisco.  C<sub>4</sub> plants occur in hot and
dry habitats.  During the day, the stomata close and oxygen builds up while
CO<sub>2</sub> levels fall.  This leads to photorespiration ordinarily. PEP
has a higher affinity for CO<sub>2</sub> than rubisco, so it is able to fix
CO<sub>2</sub> for rubisco and keep O<sub>2</sub> away.
<p>
<b>Crassulacean Acid Metabolism (CAM)</b>.  Just as C<sub>4</sub> plants
emerge as a way of avoiding the ill effects of photorespiration and accumulating
CO<sub>2</sub>, another form of a CO<sub>2</sub> shuttle was identified.  CAM
plants include the cacti and plants in arid climates.  At night, stomata open
to accumulate CO<sub>2</sub>, and these are fixed into organic acids and stored
in the vacuoles of photosynthetic cells.  Then in the day, the CO<sub>2</sub>
molecules are released for the Calvin cycle.  In one case, C<sub>4</sub> plants
spatially separate the fixation process, while in CAM plants, it is done by
temporally.
<p>
Carbohydrate synthesis can now occur by a set of reactions in which a net
accumulation of 3-carbon intermediates can be siphoned off to produce 6-carbon
glucose and fructose.  This can be stored within the chloroplast stroma as
starch (glucose polymers connected &alpha;1&rarr;4) or they can be exported
to the cytosol, where invertase can condense glucose and fructose to make
sucrose.  The disaccharide sucrose is the form exportable to other plant tissues
needing energy to maintain metabolism: stem cells, root cells, etc.

<h1>The Cell Cycle:  Mitosis and Cell Division</h1>
<p>
Rudolf Virchow proposed that cells replicated themselves from pre-existing
cells, countering the hypothesis that cells formed as minerals grew crystals,
but in vitro assembly of cells.  Microscopic study of single-celled embryos
verified Virchow's belief, and as microscopes got better, this was revealed.
<p>
In animals it became apparent that specialized cells, <b>gametes</b>, with sperm from
males and eggs from females, combined to start this process.  The process of
meiosis that produces gametic cells is described in the section later.
The process of <b>mitosis</b> deals with the replication of the genome and its
separation into progeny cells;  this refers to the formation of two nuclei
with replicate genomes.  <b>Cytokinesis</b> is the process in which the
cytoplasm is divided to complete true progeny cell formation.  The process
of cell divison (mitosis/cytokinesis) occurs for all types of functions:
growth in size and development of the organism; repair of wound or damage to
tissues; and reproduction in organisms that are asexual, such as yeast.
<p>
In 1879 it was Flemming who first observed chromosomes in the staining of
salamander embryos and first used the term <q>mitosis.</q>  He used chemical
stains to identify <q>threads</q> in the nuclei that separate from one another.
Similar observations were seen for the <i>Ascaris</i> roundworm, with threads
forming and remaining constant in cell division.  Waldeyer in 1888 called these
threadlike structures <b>chromosomes</b>.  It is now understood that mitosis
involves the formation of very compact forms of the DNA of the genome into
a small space for the purpose of the efficient creation of daughter cells.
<p>
<b>Interphase</b>.  Biologists realized that the threadlike chromosomes were
not always visible when the cells were not actively dividing.  Cells which were
mitotic existed in the <b>M phase</b>, while much of the time, cells were in
<b>interphase</b> when they were not mitotic.  Visible chromosomes are the
representation useful only in mitosis.
<p>
<b>Cell Cycle</b>.  Eventually biologists established a cyclical nature to the
various phases of cell reproduction.  To determine when DNA was being synthesized,
radiolabeled phosphorus or thymidine was used (note that thymidine is not in
RNA, so it avoids testing transcription).  When Howard and Pelc did these experiments
in interphase, they found black dots on autoradiographs, but not so when
labeling was done in M phase, which means no synthesis was occurring in M phase.
Howard and Pelc used the term <b>cell cycle</b> and distinguished the <b>S phase</b>
of DNA synthesis from the M phase.
<p>
The time period of the phases was established using pulse-chase labeleing with
radiolabeled thymidine.  Cells were harvested 2 h, 4 h, 6 h, and so after
the addition of the chase, then plotted percent of cells undergoing mitosis
that are labeled against time.  They found a 4 hour lag between the appearnce
of the first labeled cells.  This labeling went from 4 to 12 hours, thus the
S phase was about 8 hours long.  It took about 24 hours to complete one cell
cycle.  One thing noticed as well was that the cells were not in synchrony.
Also importantly, there was a gap between the end of S phase and the beginning
of M phase.   This gap would later be called G<sub>2</sub> phase, since there
is another gap between the end of M and start of S, G<sub>1</sub>.  It has been
found that G<sub>1</sub> takes about twice as long as G<sub>2</sub> in many
cases (about 7-9 hours).  The gaps exist to allow for cells to prepare for what
happens in M or S phase, such as growth in size, organellar copying.
<p>
The events of mitosis have been described in detail.  Chromosomes can be counted:
humans and potatos have 46, fruit flies 8, corn has 20, and dogs have 66.
Normally the DNA is bound by <b>histones</b> at a base level, and this combination of
DNA and protein is called <b>chromatin</b>.  In the linear chromosome of
eukaryotes, each long thread of DNA is a <b>chromatid</b> in the visible
chromosome.  Chromatids are joined by a structure called the <b>centromere</b>.
Chromosomes from the same chromosome are called <b>sister chromatids</b>, which
are exact copies of the two chromatids that make up a chromosome.
<p>
M phase is cytologically divided into subphases:  prophase, prometaphase,
metaphase, anaphase, and telophase.
<ul>
<li><b>Prophase</b>.  Chromosomes begin to condense into compact form, becoming
visible in the light microscope.  The <b>mitotic spindle</b> appears, composed
of microtubule array, attaching to the centromere via spindle fibers, the fibers
forming from a microtubule organizing center (in animal cells, the centrosome,
a pairing of centrioles).  Spindles either move to opposite sides, or form
from there.
<li><b>Prometaphase</b>.  The nucleolus disappears and the nuclear envelope
fragments.  Spindle fibers from each pole attach to one of the sister chromatids
of each chromosome via a <b>kinetochore</b>.  Kinetochores are at the centromre
of the chromosome.  Centrosomes continue migration to opposite poles.  Chromosomes
are moved by the microtubules to the middle of the cell.
<li><b>Metaphase</b>.  Centrosomes have reached the opposite poles.  Kinetochore
microtubules have moved the chromosomes to the equatorial alignment called the
<b>metaphase plate</b>.  Each chromosome is held by kinetochore spindle fibers
and exerting same amount of tension or pull.
<li><b>Anaphase</b>.  The chromosomes are separating now and force is applied
to pull the chromosomes to the opposit poles.  Kinetochore spindle fibers begin
to shorten.  The chromatids going to each pole represent the unreplicated
whole of the genome.
<li><b>Telophase</b>.  The nuclear envelopes form in the daughter cells around
the chromosomes.  The spindle disassembles and the chromosomes de-condense.
</ul>
<p>
<b>Cytokinesis</b>.  In this part of the process of cell division, the cell
must ensure that organelles are present in the daughter cells.  It organizes
in such a way to ensure that it happens on both sides.  This phase then
starts the formation of a <b>cleavage furrow</b>.   A ring of actin filaments
forms inside the plasma membrane, and myosin binds to the actin.  As myosin
binds ATP, it begins a movement to slide the actin filaments, which causes the
cleavage furrow to close into a smaller ring, pulling the membrane.  This
is the process that occurs in animals, fungi, and slime molds.  In plants
the process differs:  microtubules and other proteins organize where plasma
membranes and cell walls will form.  Vesicles from the Golgi move toward the
middle of the cell in which chromosomal separation has already occurred in
anaphase, and a <b>cell plate</b> is formed.  The cell plate is then formed
from the vesicles transported to the equator.
<p>
The study of cell division at the molecular level has asked questions about
how sister chromatids separate and how chromosomes move to daughter cells.
Spindle fibers form from the organizing center as a plus end that adds tubulin
dimers and these grow to attach to the kinetochore.  The question is how the
chromosome movement is generated in anaphase.  Is it because microtubules
are made shorter by tubulin disassembly or is it because microtubules
slide past each other?  To determine what happens, fluorescently labeled
tubulin was added to prophase or metaphase cells, labeling the whole of all
spindles.  A laser light was then illuminated on one part of the spindle to
bleach it.  What followed was that the photobleached region remained stationary
and the spindle fibers got shorter between photobleached region and kinetochore.
Results indicated that microtubule sliding must occur rather than polymerization
and depolymerization.  Studies of the kinetochore show it consists of an
inner and outer set of plates as well as motor proteins.  The kinetochore base
attaches to the centromere and forms a crown of fibrous proteins projecting
outward.  Associated proteins include dynein and other proteins know to be
motor proteins.  These proteins walk down the microtubules from the plus end
to the minus.  The mechanism is similar to vesicle movement with kinesin or
dynein interaction to cause flagellar whiplike movement.  The hypothesis is that
as motor proteins cause the chromosome to move toward the spindle pole, the
microtubule is disassembled from the plus end.
<p>
<b>Prokaryotic Fission</b>.  Bacteria reproduce by a process called <b>binary
fission</b>.  As they have only one circular chromosome and no nucleus, the
process is perhaps less complicated, and the cell cycle is much more rapid:
as fast as 20 minutes to complete replication and formation of daughter cells
for <i>E coli</i> at 37&deg;.  Binary fission involves the formation of
FtsZ fibers that divide the replicated chromosomes; these fibers are similar
to microtubules.

<h2>Regulating The Cell Cycle</h2>
<p>
In multicellular eukaryotes, there is tremendous variation of the cell cycle
events with respect to timing. Intestinal epithelial cells divide twice a day
to renew tissue lost in digestion. In contrast, nerve and muscle cells do not
divide at all. Thus the G<sub>1</sub> period varies, from near zero for rapid
proliferation to a permanently long period in nondividing cells. Cells arrested
in G<sub>1</sub> are said to be in G<sub>0</sub> phase: these are post-mitotic
with no cell cycle. Liver cells normally divide once a year, but if damaged,
cells can divide every one or two days until repair is complete. Unicellular
eukaryotes, bacteria, and archaea divide in nutrient-rich environments, but
become quiescent when the medium is poor.
<p>
The regulation of the cell cycle was studied in mammalian cells using materials
to fuse cells in different stages of the cell cycle.  Consider a cell in M
phase:  when fused with cell in interphase, the nucleus of the interphase cell
was initiated to M phase:  this M phase cells contain components to regulate
its appearance.  Experiments with <i>X leavis</i> oocytes arrested in
G<sub>2</sub>-like phase, and then progressing to a mature egg entering M phase.
Cytoplasm from the M phase eggs was injected into frog oocytes in the G<sub>2</sub>
phase, with the immediate effect of immature oocytes entering M phase.  Interphase
cytoplasm had no effect on cells in G<sub>2</sub> however.  The factor clearly
was present in M phase, and it was purified as <b>mitosis-promoting factor
(MPF)</b>.  This factor is found in all eukaryotes.  In fact, mammalian
MPF injected into immature frog eggs induces egg maturation.  Human MPF triggers
yeast cell mitosis.
<p>
<b>MPF</b> is composed of two polypeptide subunits.  One is a protein kinase
whose phosphorylation is the switch that starts the process of mitosis.
This kinase is present throughout the cell cycle.   Its activity is regulated
by the second subunit, a member of the <b>cyclin</b> protein family.  The presence
of cyclins fluctuates in the cell cycle.  From interphase the cyclin level
increases and peaks in M phase.  The protein kinase is only active when
bound to the kinase (it is a CDK, or cyclin-dependent kinase).  The CDK itself
is phosphorylated at two sites after binding the cyclin, and the phosphorylation
inactivates the protein.  In late G<sub>2</sub> however, phosphatases remove
the phosphate from CDK, and this results in MPF activation.  When activated
the kinase has several targets:  chromosomal proteins involved with M phase
initiation and promoting condensation, nuclear lamins that hold the nuclear
envelope together; microtubule-associated proteins that cause formation of the
mitotic spindle, and most importantly, phosporylating a protease that degrade cyclin.
Thus cyclin levels degrade until and then oscillate back upward.
<p>
<b>Other cyclin-CDK complexes</b>.  Other proteins also regulate the cell cycle
besides MPF, and they usually trigger the movement of one phase to another
(e.g., from S &rarr; G<sub>2</sub>, or G<sub>1</sub> to S).  Other regulatory
proteins maintain G<sub>0</sub>.  These molecules either hold cells in phases
or induce their transition.  Hartwell and Weinert introduced the <b>cell cycle
checkpoint</b> concept after identifying defects in the cell cycle of yeast.
Defective cells would continue to divide when culture conditions would lead
normal cells to stop growing.  Defects in multicellular organisms could lead
to a tumor.  Three checkpoints were identified.
<ol>
<li>The <b>G<sub>1</sub> checkpoint</b> prevents entry to the S phase for DNA
synthesis: it would make sure that nutrients are sufficient, that growth
factors are present as signals from other cells, and that cell size is adequate.
<li>The <b>G<sub>2</sub> checkpoint</b> controls the entry into M phase:  only
the chromosomes have been successfully replicated, there is no DNA damage and
active MPF is present will the G<sub>2</sub> to M transition occur.
<li>The Metaphase checkpoint is controlled to make sure all chromosomes are
attached to the mitotic spindle.
</ol>
<p>
Particularly the late G<sub>1</sub> checkpoint is vital, because it decides
whether to commit to S and M phases.  If the DNA is damaged, the p53 protein
will activate genes that stop the cell cycle until damage is repaired  or
leads to the cell inducing its own destruction by <b>apoptosis</b>.  In fact
a defective p53 protein can lead to uncontrolled growth (cancer).  Proteins
such as p53 are in fact called tumor suppressors.
<p>
The G<sub>2</sub> checkpoint similarly controls that chromosome duplication has
occurred and is not riddled with errors. Since MPF triggers M phase, MPF
activity is also vital to the checkpoint.  If DNA is damaged or chromosomes not
replicated, then MPF dephosphorylation and thus activation is blocked.

<h2>Cancer: Unregulated Cell Division</h2>
<p>
Half of American men and one-third of women will develop cancer in their lifetime.
One in four deaths is due to cancer.  At least 200 types of cancer are known.
Certain proteins like Ras, which are mitotic stimulators, become defective and
are no longer turned off.  If division is limited to the area, a tumor develops
which can be surgically removed.  However, cancers represent more than just
uncontrolled cell division.  Cells can become motile and spread and invade
other tissues.   The tumor becomes malignant.
<p>
While unicellular organisms use a G<sub>1</sub> checkpoint of cell size
and nutrient availability, multicellular organisms depend on one more factor:
hormonal signals such as growth factors to inform them when to undergo mitosis.
Growth factors are particularly high in fetal serum.  Platelet-derived
growth factor (PDGF) was identified in 1974 as a natural growth factor released
from platelets, which makes sense it is involved in clotting of wounds, and
could stimulate repair.  PDGF is actually produced by many cell types other than
platelets.
<p>
Consider one instance of growth factor control.  Growth factors stimulate the
synthesis of cyclins, which will bind to the CDKs.  Growth factor also stimulate
production of the E2F protein, which is a protein similar to MPF.  E2F then
triggers S phase gene expression.  Expressed E2F will bind the tumor suppressor
Rb (discovered because its defect causes retinoblastoma), which modulates the
G<sub>1</sub> checkpoint.  Rb binding of E2F prevents the action of E2F in
activating S phase proteins, keeping a cell in G<sub>0</sub>.  As cyclins for
G<sub>1</sub> increase in level, they bind their CDK and this targets
phosphorylation of Rb.  Phospho-Rb has less affinity for E2F, and so now E2F
can induce S phase.
<p>
In cancer, the G<sub>1</sub> cyclins can be overproduced, effectively continuallly
stimulating the CDK that phosphorylates Rb and keep it from binding to E2F.
Ras is upstream of the cyclin production process, and its defect can cause
cyclin overproduction.
<p>
Another cause of cancer is a defective Rb protein.  This would be entirely
unable to bind E2F at any point.

<h1>Meiosis</h1>
<p>
In studying meiosis, the question <q>why?</q> comes up:  why sex?  why not
asexual reproduction?  Most organisms actually reproduce asexually.  In
eukaryotes, mitosis is actually a form of asexual reproduction.  With sexual
reproduction, a new individual is produced.  The genetic variation is good for
the species.  A different explanation (a <q>proximate</q> vs. <q>ultimate</q> one)
explains how rather than why something happens.  In sexual reproduction, a
male gamete (sperm) and female gamete (egg) join or fuse in a process called
fertilization.  This process was described in sea urchins in 1876.  The fertilization
led to development of an embryo which grew by mitotic divisions.
<p>
In a 1883 study of <i>Ascaris</i> roundworms with its four chromosomes, sperm
and egg nuclei were observed to have only two chromosomes.  Weismann in 1887
proposed that gametes would form with a cell division leading to a reduction
in chromosome number.  This number would be restored with sperm fuses with
egg.  Study of gametes in animals and plants confirmed this finding.
<p>
In 1902 Sutton observed the formation of sperm in the lubber grasshopper.
There were 23 chromosomes and these could be arrayed into types based on the
size of the chromosome and the lengths of arms.  He notice two chromosomes that
did not look alike.  11 of them could be designated <i>a</i> through <i>k</i>,
and the twelfth was called X.  Later N. Stevens determined the X chromosome was
related with defining the sex of the individual.   Chromosome <i>a-k</i> were not
related and called <b>autosomes</b>.  For this species, females have two
sex chromosomes (XX), while males have only one (XO, where O means missing).
Sutton also introduced the terms <b>homologous</b> chromosomes to indicate
chromosomes of the same type that pair in meiosis.  Homologous chromosomes
actually carry the same genes in the same position of the chromosome (called
a locus).  What differs in the gene are sequences, and this variation indicates
an allele of the gene.  Sutton had thus characterized the <b>karyotype</b>, which
is the number of types of paired arrangements.  The normal cell in eukaryotes
are diploid cells:  this means they ahve two versions of each type of chromosome.
Bacteria, archaea, and many algae are haploid however.  The ploidy of a cell
indicates the number of versions of the types of chromosomes, and is designated
<i>n</i> for haploid, 2<i>n</i> for diploid, and so on.  In studying cytological
characteristics related to meiosis, it is important to understand all these
terms such as chromatid, sister chromatid, and what is a replicated chromosome.
<p>
Meiosis occurs in two specific cell divisions, called <b>meiosis I</b> and
<b>meiosis II</b>.  In meiosis I, the homologous chromosomes separate, and it
is this division that makes the cell go from diploid (2<i>n</i>) to haploid
(<i>n</i>).  In meiosis II, the sister chromatids separate and go to daughter
cells.  In effect they go from the replicated to unreplicated state.  In fact
meiosis II is equivalent to mitosis, except that half the chromosome numbers
are separated.  Once the daughter cells are formed, they undergo changes that
make it possible for the gametes to fuse later on in the fertilization event.
<p>
<b>Phases of Meiosis I</b>.  Once S phase is completed to replicate all
chromosomes with sister chromatids, meiosis I can beging.  It occurs in
phases just as mitosis does.  Early prophase I involves chromosome condensation,
spindle apparatus formation, and nuclear envelope disintegration.  A special
event also occurs in which the homologous chromosomes tightly join each other
in a process called <b>synapsis</b>.  This forms a structure called a
<b>tetrad</b> in which the two homologous chromosomes each with their two
sister chromatids show an array of four chromatids.  The chromatids actually
intertwine to form <b>chiasmata</b> (singular <b>chiasma</b>) forming a Greek
letter chi.  T. H. Morgan proposed that there was an exchange of genetic material
between non-sister chromatids, a process he called <b>crossing over</b>.
This process allows paternal alleles on one gene to become part of the larger
body of the maternal chromosome.  The chromosomes then proceed to metaphase I:
they line up on the metaphase plate by the direction of spindle fibers.  Each
tetrad moves independently of other tetrads and the alignment of chromosomes is
random.  In anaphase I, homologous chromosomes separate, and in telophase I,
the homologous pairs have been separated.  Cytokinesis ensues to form separate
cells.
<p>
<b>Phases of Meiosis II</b>.  The only thing distinct between mitosis and
meiosis II is that there is no S phase.  The separated homologous pairs already
have sister chromatids, and so prophase II can begin with spindle fibers attaching
to each side of the chromosome, one fiber from each pole for each sister
chromatid.  With metaphase II chromosomes line up at the metaphase plate.
Sister chromatids are separated in anaphase II and movement to poles complete
in telophase II.
<p>
While all these events must occur in eukaryotes, the timing of these vary
considerably.  In the females of some animal species, meiosis begins in oocytes
and stops and restarts as the oocyte matures into an egg.  In mammals, female
gamete meiosis does not actually complete until after fertilization occurs.
In plants, the meiosis products do not form gametes but haploid cells become
spores.
<p>
<b>Karyotyping</b>.  This is the science of numbering and typing the chromosomes
of a species.  Since chromosomes are only visible during mitosis, it is necessary
to select cells that are undergoing division.  Cancer researchers might want to
look at tumor cells and cytogeneticists might want to understand a genetic
defect by karyotyping.  Cells must then be grown in culture.  Colchicine can
be added:  this is a poison that will bind microtubules such as those in the
spindle and arrest their action.  Chromosomes can be stained to illuminate them:
stains for DNA are very well known.  Some stains can form a banding pattern in
the chromatids (arms) of the chromosome.  <b>Spectral karyotyping</b> (chromosome
painting) can be done with fluorescent dyes attached to short DNA molecules, and
the DNA binds to particular regions.  Karyotyping has shown that chronic
myelogenous leukemia (CML) has been caused by a translocation of chromosome 22
to chromosome 9.
<p>
Recall that in prophase I, the chromosomes underwent synapsis, in which
nonsister chromatids intertwined with each other.  This is achieved by a set
of proteins called the <b>synaptonemal complex</b> (synaptonemal means "joined
thread").

<h2>Sexual Reproduction</h2>
<p>
Meiosis serves to provide sexual reproduction in eukaryotes.  In the early 1900s
biologists began using the term <i>gene</i> to refer to the coding for a
trait.  Chromosomes are composed of genes, and identical copies of chromosomes
are in daughter cells in mitosis.  Individuals that are exact copies of a parent
are clones.  The value of sexual reproduction is the genetic variation it does.
It achieves it in three ways.
<p>
The first way is by the chromatid recombination process that causes exchange of
DNA. Each cell of the body has half of its chromosomes from the mother, and
half from the father. When meiosis occurs in the production of gametes,
prophase of meiosis I has an important process involving non-sister chromatids
forming synaptonemal complexes and exchanging DNA from one strand to another.
This actually provides genes on a maternal chromosome to become part of the
larger body on a paternal chromsome, or vice versa. Therefore, the alleles on
the paternal chromatid get mixed with the maternal chromatid.
<p>
The second way is by the random or independent assortment of chromosomes to be
segregated in gametes. This occurs in metaphase of meiosis I, the homologous
chromosomes line up on the equatorial plate in a random fasion: the assortment
is independent. It is not the case that all the maternally derived chromosomes
line up on one side, and paternal on the other. In fact, the possible number of
alignments for the 23 human chromosomes is 2<sup>23</sup>, or 2<sup><i>n</i>
</sup> for any species with <i>n</i> chromosomes. That is about 8.4 million
combinations. If there are about 50 chiasmata formed per cell undergoing
meiosis I prophase, that is even more variation.
<p>
The third way genetic variation is achieved is in the process of fertilization.
Two gametes from the source individual might possibly join in an act of
self-fertilization and it would be thought that this is basically the same
genome from the individual.  <q>Selfing</q> is possible in plant species in fact,
and occurs in animal species where individuals have both male and female sex
organs.  But in most cases, the gametes of different individuals in the species
combine for offspring in an <b>outcrossing</b>.  This is how fertilization
causes genetic variation:  by mixing the half-genomes of different individuals.
When an individual (male) mates with another (female), and suppose each can
produce 8.4 million different chromosomal arrangements in gametes (assuming no
synaptonemal recombination), then that is 8.4 million &times; 8.4 million =
70.6 &times; 10<sup>12</sup> possible offspring.
<p>
Genetic variation in bacteria are also discussed in how they use conjugation
for the one way transfer of small amounts of DNA (plasmids, or parts of the
whole chromosome) in order to make individuals with a mix of genes.

<h2>The Basis for Meiosis and Sexual Reproduction</h2>
<p>
The following can be noted:
<ul>
<li>Bacteria and archaea can only reproduce by asexual production, although
genetic variation to a small extent occurs by conjugal transfer of small
amounts of DNA as a plasmid or parts of chromosome; bacterial viruses
(bacteriophages) can also achieve transfer
<li>most algae, fungi, and some animals and plants can reproduce both
asexually and sexually
<li>mitosis is a verifiable form of the asexual reproduction of cells, whether
in uni- or multicellular species
</ul>
<p>
Quaking aspens will form new trees by simply extending roots near the soil
surface and new stems and shoots can form a tree.  In vertebates the
guppy in <i>Poeciliopsis</i> can reproduce just by mitosis.  The question is
why sex is better, or why genetic variation is better.
<p>
J. Maynard Smith (1978) said that sexual reproduction is a paradox.  He
developed a mathematical model to show asexually reproducing individuals do
not have to produce male offspring, and that progeny can produce twice as many
offspring in the next generation (second) as those reproducing sexually.  First
he showed that in asexual reproduction, the number of individuals (cloned)
is larger in count than for sexual reproduction, because males and females
have to combine their gametes to produce single individuals.  This has in large
part to do with the fact that only females actually produce the individual, and
not the males.  Maynard Smith calls this the <q>two-fold cost of males.</q>
Asexual reproduction is more efficient on this matter because there is no need
for two sexes.  Maynard Smith contends, <q>all other things being equal,</q>
asexually reproducing species can easily compete with sexually reproducing
species.
<p>
<b>Purifying Selection Hypothesis</b>. If a gene is damaged in a way that
causes it to function poorly, then <em>all</em> individuals in the species will 
inherit that flawed gene where there is asexual reproduction. If the damaged 
gene is in generation 1, it might result in four individuals in generation 2 
producing fewer offspring, which may die young. And generation 3 counts are low 
to since the loss of function compared to when function is normal. Now suppose 
the deleterious allele arises in sexual reproduction in the female of 
generation 1; note that the other allele is normal. If she mates with a male 
whose both alleles are normal, than only half of her offspring are affected on 
average. Note that the parent has the defect, but the offspring do not have to 
have it; this is not true in asexual reproduction. Natural selection against 
deleterious alleles is called <q>purifying selection.</q> Daphnia is a pond and 
lake aquatic species that can reproduce sexually and asexually, and they found 
that more deleterious alleles emerge in asexual vs. sexual reproduction.
<p>
<b>Changing Environment Hypothesis</b>.  This is basically the idea that
genetic variation resulting from sexual reproduction makes it possible to produce
individuals that can survive to perpetuate the species when the environment is
severely unfavorable to certain individuals.  Offspring might persist even
though the parents might not survive.  In humans, there is not much concern about
things like temperature, moisture, predators, competitors and so on, but it more
important the ability to resist pathogens and disease, such as SARS virus,
new strains of HIV, malarial parasites, and tuberculosis bacterium.   Thus in
a pandemic of diseases, millions might die, but thousands might remain, resistant
to infection or coming out immune to it.
<p>
Does the changing environment hypothesis work?  C. Lively and coworkers studied
a New Zealand snail that thrives in freshwater habitats and is also susceptible
to infection by over a dozen species of parasitic trematode worms.  The worms
eat the reproductive organs of the snails.  Some individuals of the snail species
reproduce asexually, others sexually.  If the changing environment hypothesis
is correct, then frequency of sexually reproducing individuals should be higher
in habitats where parasites are common than in habitats were parasites are rare.
Asexually reproducing individuals should have higher fitness in environments
where parasites are rare logically:  they produce so many offspring in
parasite-free environments, alleles associated with asexual reproduction should
increase in frequency.   In contrast, where parasites are common, the advantage
is with sexual reproduction to create individuals that better resist the
parasites.  So large sample sizes were collected of snails from habitats where
parasites are rare and common.  Males and females are counted:  where there are
lots of males, there is a high amount of sexual reproduction.  A plot
was made of the parasitism rate (percentage of individuals infected) vs.
two categories:  sexually and asexually reproducing populations.  Results showed
that in fact sexual reproduction is more common where parasitism is high, and
asexual reproduction where parasitism is rare.

<h2>Defective Meiosis</h2>
<p>
While the biology of meiosis has been described as it should happen, nature
does not always work the way it should.  How does nature deal with the mistakes?
L. Down (1866) described a syndrome in humans characterized by mental
retardation, a high risk of heart problems and leukemia, and neurodegenerative
disorder similar to Alzheimer's.  Down syndrome is seen in 0.15% of live births
(1 in 666).  In the 1950s a report identified that it was caused by cells having
three copies instead of just two of chromosome 21 (trisomy-21).
<p>
This could only be explained by an understanding of the biology of cell in
meiosis. In anaphase I, homologous chromosomes did not separate as they should 
have, and there is <b>meiotic nondisjunction</b>. One gamete had a ploidy of <i>
n</i> + 1: the <i>n</i> chromosomes it should have, plus 1 that did not disjoin.
 This could be called a partially diploid state. The other gamete did not have 
chromosome 21 (its state is <i>n</i> &minus; 1). A <i>n</i> + 1 gamete fusing 
with an <i>n</i> gamete produces a 2<i>n</i> + 1 trisomic individual. A <i>n</i>
 &minus; 1 gamete fusing with an <i>n</i> gamete produces a 2<i>n</i> &minus; 1 
monosomic individual. Both will be <b>aneuploid</b>.
<p>
About 10% of meiotic divisions involve nondisjunction.   One study showed that
38% of naturally aborted pregnancies involved atypical chromosome complements.
Trisomy occurred in 36% of the abnormal karyotypes.  In some cases, triploidy
(3<i>n</i>) was found.  Less common was abnormally sized or shaped chromosomes
and monosomy.
<p>
Mistakes in meiosis do occur and there is no suggestion that there is actually
a genetic predisposition to disjoint meiotic events.  While errors appear to be
random, some patterns are observed.
<ul>
<li>most trisomies and monosomies in humans involve the sex chromosomes:
Klinefelter XXY occurs in 1 in 1000 live births as well as Trisomy X (XXX).
Turner syndrome (XO) occurs in 1 in 5000.
<li>If aneuploidy, usually trisomy, occurs, it is more common in smaller
chromosomes (13-22) than in the larger chromosomes; trisomy-21 is the most
common and are related to maternal errors in division.  Maternal age is a factor
in trisomy as well.
</ul>

<h1>Mendelian Genetics</h1>
<p>
Cell theory and the theory of evolution are great developments in biology
explaining what organisms are made of where they come from.  In this section
one learns why offspring resemble their ancestors in traits and characteristics.
Mendel was an Austrian monk who had received education in the natural sciences
at U. Vienna and studied physics and math with Doppler.  Heredity was
the province of animal breeders and horticulturists.  At the time, one
hypothesis was that offspring are a blend of the traits of mother and father,
being intermediate in degree.  Thus the mating of a black and white sheep
would be expected to produce a gray offspring.  Another hypothesis was that
acquired characteristics could be inherited:  Lamarck was a prominent proponent
of this idea that parents who acquired characteristics in their liftimes would
pass them to their offspring.
<p>
Mendel's success in genetics arises from the fact he chose a <b>model organism</b>
for his observations.  A model organism would have a rapid generation cycle,
inexpensive to maintain, and can produce large numbers of offspring.  His was
the pea plant <i>Pisum sativum</i>.  Important also for his experiments was
that Mendel could choose which parents were involved in mating, and mating
between individuals differing in recognizable traits could be done.
<p>
The garden pea flower has male and female reproductive organs.  The pollen
grains are small sacs maturing in male reproductive structure of the plant.
Pollen grains extend tubelike projections toward egg cells where fertilization
takes place.  In normal conditions pea plants self-pollinate rather than cross
with other plants because the flower structure does not promote pollen travel
to other plants.  The flower also tends to keep out pollinating insects like
bees and other insects.  Mendel circumvented this process by removing the male
part before pollen formed.  He could then use a brush to transfer pollen from
one plant to another.  Thus Mendel performed a <b>cross-pollination</b>.
<p>
Mendel identified seven traits in peas that he could test for two phenotypes:
<ol>
<li>seed shape: round and wrinkled
<li>seed color: yellow and green
<li>pod shape: inflated and constricted
<li>pod color:  green and yellow
<li>flower color:  purple and white
<li>flower and pod position:  axial (on stem) and terminal (at stem tip)
<li>stem length:  tall and dwarf
</ol>
<p>
Mendel first started with obtaining pure or true-breeding lines:  these are
plants producing offspring identical to themselves when self-pollinated or
crossed to another member of the pure line.  These pure lines were necessary
as a starting point to understanding the mating of different lines.
<p>
With the pure lines, he then crossed plants that differed in only one trait,
keeping the other six traits identical during mating. Thus he took round peas
as one parent and wrinkled peas in the other and crossed them in the <b>
parental generation</b>. The progeny will be the <b>F<sub>1</sub> (for <q>first
filial</q>) generation.</b>  He found all the F<sub>1</sub> progeny to be
round.  This was counter to the hypothesis that offspring were a blend of the
traits of the parental generation.  The wrinkled genetic determinant appeared
to have disappeared.
<p>
Mendel then wondered if the disappearance of wrinkled seed phenotype was
related to pollen and egg:  did it matter if the pollen from the round phenotype
brushed with wrinkled phenotype egg or if pollen came from wrinkled phenotype
was brushed with round phenotype egg.  Mendel did these experiments and found
that it did not matter: all progeny were still round, and it was not a factor
of pollen and egg.  This is what is called a <b>reciprocal cross</b>.  In
autosomally based traits, the results shown above for pea seed shape will apply,
but in sex chromosome-linked traits, the reciprocal cross will have a different
result, and indicate a sex chromosome-linked trait.
<p>
Mendel then allowed the F<sub>1</sub> individuals to self-pollinate, and observed
the progeny (the F<sub>2</sub> generation) in which 5474 individuals were round
and 1850 were wrinkled.  The surprise is that the wrinkled phenotype re-appeared
whereas it had disappeared in the previous generation.  To account for the
observation, Mendel described traits seen as <b>dominant</b> or <b>recessive</b>.
Round peas were dominant, and wrinkled peas were recessive.  Mendel understood
that the F<sub>2</sub> progeny was present in a 2.96:1 ratio, essentially 3:1,
or 3 round:1 wrinkled.  He did the same for the six other traits, and found
a similar ratio.
<p>
Mendel then countered the hypothesis of blending inheritance with a hypothesis
of particulate inheritance.  Mendel proposed that traits were described by
<b>genes</b>, and that different versions of genes were caused by <b>alleles</b>.
Thus for the gene of pea shape, there is an allele that indicates a shape
being round, and one being wrinkled.  All the alleles found in an individual
make up a <b>genotype</b>.  Alleles existing in pairs (of chromosomes) was
allowed for the concept of dominance and recessiveness.  He reasoned that
when one or more alleles for round shape were present, the dominant one, then
the shape would be round.  When both alleles were present for the wrinkled
shape, then it was wrinkled.
<p>
<b>Segregation</b>. The F<sub>2</sub> phenotype 3:1 ratio could be explained by
understanding that each gamete would produce four gametes with one allele only; 
this is what segregation is. Mendel symbolized allelic forms by upper- and 
lowercase letters. Uppercase were to represent dominant alleles, while 
lowercase was recessive. Thus the <i>R</i> allele indicates pea shape roundness,
 while <i>r</i> represents pea shape wrinkledness. The <i>RR</i> genotype would 
be homozygous roundedness, while <i>rr</i> would represent wrinkled shape. 
Homozygous genotypes would effectively represent the parental (P) generation. 
When pure lines are crossed, the F<sub>1</sub> progeny would all be <i>Rr</i> 
genotypes, the <b>heterozygous</b> genotye, which are all round shaped 
phenotypes. When the F<sub>1</sub> progeny are crossed with each other (or 
self-crossed, also called the <b>monohybrid cross</b>), and following the 
principle of segregation of gametes, the progeny will be
1&nbsp;<i>RR</i>&nbsp;:&nbsp;2&nbsp;<i>Rr</i>&nbsp;:&nbsp;1&nbsp;<i>rr</i>
as to genotye. However, the phenotype is 3:1 because <i>RR</i> and <i>Rr</i>
have the same phenotype.
<p>
Years after Mendel finished his work, Punnett devised a system to each understand
the segregation of gametes and to show the resulting genotypes of offspring.
The <b>Punnett square</b> can be constructed as follows:
<ol>
<li>The haploid genotype of each possible gamete from a parent (father/mother) is
written on a single row.
<li>The haploid genotype of each possible gamete from the other parent is written on
a column.
<li>Extend the lines of rows and columns to created boxes as in a table.
<li>Fill in the boxes with the resulting diploid genotype of the progeny, for
each gene locus.
<li>Proportions of the specific diploid genotypes that are replicated in the
different squares are determined by adding them up in their ratios.  Additionally
the proportions of the phenotypes are determined after determining the genotypes.
</ol>
<p>
For a gene with two alleles, a 2 &times; 2 Punnett square can be constructed.
For a gene with three alleles, a 3 &times; 3 Punnett square is possible.
In fact, when a gene has <i>h</i> alleles, the Punnett square will be
<i>h</i> &times; <i>h</i>.
<p>
Mendel proposed a genetic model as follows:
<ul>
<li>Peas have two versions (alleles) of each gene:  in many cases this is
true
<li>Inheritance of alleles involves no blending:  hereditary determinants
maintain integrity from generation to generation
<li>Each gamete contains one allele of a gene:  the principle of segregation
of alleles into gametes is consistent with the data
<li>Males and females contribute equally to genotype of offspring:  gametes
do join to make the diploid individual
<li>Some alleles are dominant to other alleles:  in fact a dominant allele
does result in an expression of the dominant phenotype
</ul>
<p>
<b>Proving Mendel's Model</b>.  With the model Mendel should be able to make
predictions and see them in the results.  Having identified dominance and
recessiveness in seed shape, crosses producing <i>rr</i> should be wrinkled and
those which are <i>RR</i> or <i>Rr</i> are round.   So Mendel planted the
F<sub>2</sub> seeds and allowed self-pollination of the different phenotypes.
The F<sup>3</sup> for wrinkled parents always produced wrinkled progeny.
When Mendel let 565 round-seeded plants self-pollinate, 193 plants produced
round-seeded offspring:  he inferred these are <i>RR</i> genotype.
372 round-seeded parents either round or wrinkled however.  He determined
these parents must have the <i>Rr</i> genotype.  A ratio of 372:193 is
1.93:1, close to the predicted 2:1.

<h2>Two Trait Genetics</h2>
<p>
Additional tests of the model were necessary, such as whether segregation works
for two different traits.  Mendel used a pure line of round, yellow and another
of wrinkled, green seeds.  The F<sub>1</sub> of the cross would be heterozygous
for both genes.  The mating would be a <b>dihybrid cross</b>.  Mendel had already
established that yellow was dominant to green, and so the allele <i>Y</i> would
designate the dominant yellow and <i>y</i> for the recessive green.  The model
believes that traits (genes) would be transmitted independently of each other:
the law of <b>independent assortment</b>.  As an alternative it was considered
that certain traits also segregated with each other, a dependent
assortment.  Mendel predicted the ratios and then did the experiment.
<p>
If independent assortment of the traits were true, then the
<i>RRYY</i> &times; <i>rryy</i> mating will yield F<sub>1</sub> heterozygous
progeny, <i>RrYy</i>, which when self-crossed, will produce a 4 &times; 4
Punnett square of all the possible gametes.  The genotype proportions of the
F<sub>2</sub> generation are
9&nbsp;<i>R_Y_</i>&nbsp;:&nbsp;3&nbsp;<i>R_yy</i>&nbsp;:&nbsp;3&nbsp;<i>rrY_</i>&nbsp;:&nbsp;1&nbsp;<i>rryy</i>.
If in fact dependent assortment occurred, only two gametes would result, with
a 2 &times; 2 Punnett square, and the proportions are
1 <i>RRYY</i> : 2 <i>RrYy</i> : 1 <i>rryy</i>, with a 3 round-yellow :
1 green-wrinkled phenotype.
<p>
The results showed 9:3:3:1 proportions and supported the law of independent
assortment. Mendel performed a <b>test cross</b>: such a cross requires one
parent have the recessive genotype in order to determine the genotype of the
other parent. The parent with unknown genotype may show the dominant phenotype
and the results allow determining whether the genotype is homozygous or
heterozygous. Mendel's testcross with the F<sub>1</sub> <i>RrYy</i> &times;
<i>rryy</i> showed a 1:1:1:1 phenotype of round-yellow, round-green,
wrinkled-yellow, and wrinkled-green (absolute counts were 31:26:27:26,
respectively). Since the F<sub>1</sub> was clearly heterozygous for seed shape
and color, the genotypes would have to be <i>RrYy, Rryy, rrYy,</i> and
<i>rryy</i>. Note the Punnett square for the testcross is 1 &times; 4.

<h2>The Chromosome Theory of Inheritance</h2>
<p>
Mendel's work went ignored for a whole generation (30 years).  A factor may have
due to how he used probability theory and his quantitative treatment of data.
Moreover, the belief in blending inheritance was still quite strong.  In 1900
three biologists confirmed Mendel's findings, but connected his studies of the
gene to experiments with chromosomes.
<p>
Sutton and Boveri connected meiosis to how Mendelian progeny formed from parents
in 1903.  The process of gamete formation could be linked to the principles
of segregation and independent assortment, and the link to chromosomes as
carriers of the genes was made.  A gene holds a position on the chromosome,
called the <b>locus</b> (physical location).  In meiosis I anaphase, the principle
of Mendelian segregation has its physical explanation.  As genes occurs on
different chromosomes which themselves align randomly at metaphase, the
principle of independent assortment is observed.
<p>
A model organism emerged as a test of the chromosome theory of inheritance:
an insect the fruit fly <i>Drosophila melanogaster</i>.  Its attributes as a
model include a small size, easy laboratory breeding, a short (10 d) reproductive
cycle, and abundatn offsprring (a couple of hundred per mating).  The fly has
many easily identifiable and interesting anatomical features also.  For example
it can have a red or white eye color.  T. H. Morgan first had to work characterizing
phenotypes of the fly.  The individuals in the population with the most
common phenotype were referred to as the <b>wild type</b>.  The red color eye
type was the wild type, while the white eye type was considered a <b>mutation</b>,
and flies having it were <b>mutants</b>.  A red-eyed female was then crossed
to a white-eyed male, and all the F<sub>1</sub> had red eyes.  Morgan then
did a <b>reciprocal cross</b>:  white-eyed females were mated to red-eyed
males, and all the F<sub>1</sub> females had red eyes and the F<sub>1</sub>
males had white eyes.  Mendel's reciprocal crosses had given similiar results
but Morgan's did not.  Thus eye color in flies was clearly linked to sex.
When the female parent is white-eyed, the white-eyed phenotypes show in
F<sub>1</sub> males.
<p>
At the time that Morgan was doing his work, N. Stevens was studying the
<i>Tenebrio molitor</i> beetles.  Females of the species had 20 chromosomes
in the diploid cells, and males had 19 large chromosomes and 1 small one.
Stevens called the small one the Y chromosome.  In meiosis I it was found that
the Y chromosome was found with a large chromosome that had been called the
X chromosome.  Stevens noted that the female cells had two X chromosome, and
the males had X and Y, and proposed that this determined sex.
<p>
Morgan took his observations and those of Stevens' and concluded that the flies
have X and Y and how gametes segregated with the sex chromosomes.  He reasoned
that eye color could be carried on the X chromosome with no corresponding
gene locus on the Y chromosome, and this would account for the results.  This
altered genetic theory to include sex-linked inheritance.  The Punnett squares
were as in the tables below, which shows the cross and the reciprocal cross.
<table>
<tr><th rowspan="2" colspan="2"><th colspan="2">Male
<tr>       <th><i>w</i> <th><i>Y</i>
<tr><th>Female
   <th><i>w</i><sup>+</sup>  <td><i>w</i><sup>+</sup><i>w</i>
   <td><i>w</i><sup>+</sup><i>Y</i>
</table>

<table>
<tr><th rowspan="2" colspan="2"><th colspan="2">Male
<tr>       <th><i>w</i><sup>+</sup> <th><i>Y</i>
<tr><th>Female
   <th><i>w</i>  <td><i>w</i><sup>+</sup><i>w</i>
   <td><i>w</i><i>Y</i>
</table>
<p>
It should be noted that the allele <i>w</i><sup>+</sup> is for the red eye
color, and that the symbol <q>+</q> is used often to designate the wild type.
Other genes found on the X chromosome includes body color, where the gray color
is the wild type, and yellow body is the rare mutant.  Linked genes would
violate the principle of independent assortment, as genes need to be on
separate chromosomes.  Linked genes on the X chromosome of a female progeny with
one parent being white eyed with gray body and the other parent being red-eyed
with yellow body would be <i>wy</i><sup>+</sup>/<i>w</i><sup>+</sup><i>y</i>.
A small percentage of males had new phenotypes and thus the genotypes
<i>wy</i> and <i>w</i><sup>+</sup><i>y</i><sup>+</sup>.  Morgan understand that
these individuals were <b>recombinant</b>, and proposed that crossing over
occurred in the X chromosome.  While linked genes could violate the principle
of independent assortment, they might seem to assort independently if
crossing over occurs.  Morgan found that 1.4% of offspring showed the
recombinant phenotype.  In testing other X-linked genes however, such as
eye color and bristles, males with a recombinant phenotype occurred at a
higher frequency, 19.6%.  Morgan's explanation was that genes were on a linear
array in the chromosome, and that the distance in locus between bristles
and eye color was greater than for eye color and body color.
<p>
Morgan's student Sturtevant thought that a genetic map indicating the relative
positions could be made to explain the observations.  He gave the map physical
units that represented the percentage frequency of recombination.  The whole
length of the chromosome was 1 Morgan, and 1 centiMorgan (cM) was 1% of the
recombination frequency.  The eye color and bristle shape would be 19.6 cM
or map units apart, and that between body color and eye color only 1.4 map units
apart.  What is the relationship between body color and bristle shape?  It can
be computed if the distances are known between loci distances known in relative
degree.
<p>
What is interesting is that when genes are on the same chromosome but more than
50 map units apart, they recombine in a way that makes them seem like they are
not linked:  they appear to follow the principle of independent assortment.
<p>
The work of Mendel covered basic fundamentals of genetics and heredity:
dominance and recessiveness, segregation, independent assortment, and linked
genes that crossed over as unlinked genes.  In most cases, genes do not have
only two alleles but multiple alleles.  However, the &beta;-globin has over
500 different sequence variations and thus alleles:  while there is a reduced
oxygen capacity with some variations, this cost is because it confers resistance
to malarial parasites, or adaptation to high altitude.  A <b>polymorphic</b> trait
is one where the gene shows <b>multiple allelism</b>.
<p>
<b>Codominance and Incomplete Dominance</b>.  Mendel studied seven traits whose
genes showed alleles with clear dominance.  But in flowers called four o'clocks,
a pure line producing purple flowers and a pure line producing white flowers
were crossed, the offspring were lavender, which is a hue between purple and white.
Biologists self-crossed the lavender F<sub>1</sub> generation and looked at the
F<sub>2</sub> progeny.  1/4 were purple, another 1/4 were white, and 1/2 were
lavender.
<p>
The inheritance of alleles from gametes is the same for both peas and four
o'clocks. The difference is that peas show complete dominance while the four 
o'clocks show <b>incomplete dominance</b>. Thus dominance is not all-or-none. 
Many alleles show <b>co-dominance</b>. This is the case when both alleles (in 
the diploid) express the phenotype. Consider for example the <i>L</i> gene that 
codes for a human RBC membrane glycoprotein: it has two allelic forms, <i>L<sup>
M</sup></i> and <i>L<sup>N</sup></i>. The possible genotypes are then <i>L<sup>
M</sup>L<sup>M</sup></i>, <i>L<sup>M</sup>L<sup>N</sup></i>, and
<i>L<sup>N</sup>L<sup>N</sup></i>.  With the heterozygote, both the
<i>L<sup>M</sup></i> and <i>L<sup>N</sup></i> type membrane glycoproteins are
present:  co-dominance.
<p>
Dominance and recessiveness must be understood on the molecular level.  It is
often the case that complete or incomplete dominance results for alleles that
code for enzymes.  Plant height-related alleles studied by Mendel are related
to enzymes that code for a growth hormone.  With recessive alleles, the enzyme
is usually dysfunctional:  for the plants, the homozygous recessive appears
as the dwarf because it lacks the hormone for elongation.  Dominant alleles
generally represent the normally functioning enzyme.  With heterozygotes, the
one normal allele is sufficient to produce the dominant phenotype, just as
much as the homozygote.
<p>
However with the four o'clock, the presence of the alleles had a quantitative
effect.  With homozygous genotype <i>RR</i>, twice as much pigment is produced
for flower color as the heterozygote <i>Rr</i>.
<p>
<b>Pleiotropy</b>.  Genes may not just have an effect on a single trait, in
which the gene is the direct cause of the phenotype.  A single gene may in
fact influence many traits.  For example, <i>FBN1</i> is the gene that is
responsible for Marfan syndrome, seen as increased height, long limbs and
fingers, abnormally shaped chest, and potentially severe heart problems, along
with spinal problems.  This gene is quite pleiotropic.

<p>
<b>Environmental Effects on Genetic Expression</b>.  Genes do not express
themselves willy nilly the presence of the environment.  The physical conditions
clearly regulate gene expression.  Mendel's height experiments depended on
uniform treatment with sunlight and soil type.  The phenotype is often a factor
of the physical environment as much as genotype.  PKU occurs because the genotype
fails to code for an enzyme converting Phe to Tyr, but by modifying the diet
(the environment), the disease can be prevented.
<p>
<b>Gene Interaction Affecting Phenotype</b>.  With Mendel's pea plants, the
genes with respect to seed shape were affected by one gene and one gene only.
There was no apparent interaction caused by other genes.  Researchers
W. Bateson and R.C. Punnett however found in 1905 that the combs of chickens
had four phenotypes:  rose, pea, walnut, and single combs.  When the
rose &times; pea was done in the P generation, the F<sub>1</sub> had all walnut,
which was not one of the parental types.  When the F<sub>1</sub> was selfed,
the F2 showed a 9:3:3:1 walnut:rose:pea:single comb ratio.  Whenever such a
ratio is observed, it must be a two locus set of genes showing typical
dominance-recessive phenotype.  To explain this mode, assume the rose comb
genotype is <i>RRpp</i> and the pea comb is <i>rrPP</i>.  The F<sub>1</sub>
cross is <i>RrPp</i>.  The means the dominance of <i>R</i> and <i>P</i> interact
in a way to produce walnut comb.  In the self cross the genotype will be
<i>R_P_, R_pp, rrP_</i>, and <i>rrpp</i>, which should be walnut, rose,
pea, and single comb genotypes.  In classic Mendelian observations, <i>R</i> and
<i>P</i> would never interact.  When a gene like <i>P</i> masks or alters the
expression of R dominance, this is <b>epistasis</b> (stopping, diminishing).
<p>
<b>Quantitative Traits</b>.  <b>Discrete traits</b> are actually qualitative:
in this sense seed color is yellow or green, and not a mix or blend.  In many
cases there is no on-off, yes-no phenotype.  There is a blending.  Height,
weight, skin color will fall on a continuous scale of measurement.  Such
traits are quantitative.  H Nilsson-Ehle (1909) examined kernel color of wheat
using pure lines of white and dark-red wheat.  After crossing to get the
F<sub>1</sub>, he selfed the F<sub>1</sub> and the F<sub>2</sub> showed a
continuous variation in color: white to pink to red.  He identified
seven categories in a ratio of 1:6:15:20:15:6:1.  To explain this ratio,
three loci were assumed with standard dominance-recessiveness features.
The pure white phenotype was assumed to be <i>aabbcc</i>, and the pure red was
<i>AABBCC</i>.  The F<sub>1</sub> all showed a medium red (pink?) and so these
would be <i>AaBbCc</i>. These were selfed.  This should give 64 possible
genotypes in the Punnett square.  The interesting thing is that each
dominant allele A, B, C contributes to a color intensity.  Thus the pure
recessive is white.  The next reddish level up will have one dominant allele
A, B, or C out of the six possible alleles:  <i>Aabbcc, aaBbcc, aabbCc</i>.
Note there are six total genotypes in the Punnett square.
The next level up will have six possible genotypes that contain two and only
two dominant alleles:  <i>AAbbcc, AaBbcc, aaBBcc, AabbCc, aabbCC, aaBbCc</i>, and
the Punnett square will reveal 15 possible genotype permutations.  None of
the recessive alleles contribute to pigmentation, but all the dominant forms
do.  Quantitative traits thus reflect <b>polygenic inheritance</b>.

<h2>Human Mendelian Inheritance</h2>
<p>
Genetic counseling typically involves a pedigree analysis, constructing the
family tree to trace a trait or disorder.  In starting the analyze, some
assumptions are made first:  the trait belongs to a single autosomal gene
(not sex-linked), and the alleles show a dominant-recessive relationship.
Then establish the conditions that <i>must</i> be met with a particular pattern of
inheritance compared to conditions <i>likely</i> to be met.  If the phenotype
of interest is autosomal recessive, then the individuals must be homozygous
for the gene.  If parents don't have the trait, but the affected person does,
they must be heterozygous (they are thus <b>carriers</b>).  If carriers mate,
the occurrence must be 1 in 4 in issue (25% of time).
<p>
As an example, the sickle cell disease is observed.  It is found in both boys
and girls have it, even though parents do not.  This is precisely the pattern
for an autosomal recessive with heterozygous parents.  If the affected person
mates and has not children with the trait, then the mate is probably homozygous
for the wild-type, true if the recessive is rare in the population.
<p>
Considering patterns of autosomal dominant traits, if parents are heterozygous
and the other recessive, half the children showing the phenotype indicates
autosomal dominance.  One human disease showing this pattern is Huntington's
disease.  If a child shows the trait, one parent will.  The trait also shows
up in every generation where there are large numbers of children in the
generation.  It also shows up in males and females alike.
<p>
As for sex-linked traits:  if a trait shows up with near-equal frequency in
males and females, it is likely autosomal.  If it occurs in males with much
greater frequency or exclusively, it is likely X-linked (Y-linked inheritance
would be extremely rare).  Pedigree analysis for <b>hemophilia</b> is a classic
demonstration of sex-linked analysis.  Prince Albert and Queen Victoria had
14 children.  One boy was a hemophiliac (the X chromosome carries an important
blood clotting factor).  Three grandchildren were boys also affected with
hemophilia, but they were children from two daughters, not the affected son.
Later, a first cousin marriage of one grandaughter from the Queen's daughter
produced two boy with hemophilia (one boy was not affected!).  The queen's
affected son had a normal boy, but one of the grandsons was affected, so the
trait clearly passed through the daughter.  Thus X-linked traits tend to
skip a generation:  this is because daughters will be carriers, as they get
an X chromosome from an affected father, and sons only get the Y chromosome
from the diseased father.

<h1>Synthesis and Repair of DNA</h1>
<p>
The examination of the molecular nature of the gene is now pursued.  In
1928 F. Griffith was trying to develop vaccine against <i>Streptococcus
pneumoniae</i> with only murine-sensitive strains.  A <b>strain</b> is a
population of genetically identical individuals.  Colonies on blood agar
plates showed two properties:  the smooth (S) strain had colonies that where
more whitish and had shiny appearance, where the rough (R) colonies were not
so shiny.  The former were more virulent, and the reason was that the shiny
appearance was a protective polysaccharide capsule around the bacterium.
In his experiments he injected mice with the R and S strains, and the S strain
killed mice, while the R did not.  He then heat-treated the S strain, and injected
mice lived.  He then injected in heat-treated S strain with live R strain, and
he noticed the mice died from infection.  Thus, some nonliving property in
S strain transferred into R strain:  R strain was <b>transformed</b>.
<p>
Scientists were already aware of proteins and DNA.  Many believed proteins were
the transforming agent.   O. Avery, C. M. MacLeod, and M. McCarty in the 1940s
then grew virulent S cells in culture.  They then purified to remove lipids
and carbohydrates, leaving protein, RNA, and DNA.  One sample was treated with
proteases to destroy the proteins.  The other was treated with ribonuclease
to destroy RNA.  Then another was treated with DNase to destroy the DNA.  Small
quantities were then mixed with nonvirulent R cells in culture.  They found that
S cells appeared on plates in the samples treated with proteases and RNases, but
not in the DNase-treated sample.  Thus they identified clearly that DNA was
conferring the transformative property.
<p>
Scientists supporting the protein hypothesis argued that the treatments did
not sufficiently degrade the protein.  A. Hershey and M. Chase were studying
T2 phage with E coli.  The phage is composed of only protein and DNA.  Hershey
and Chase knew that proteins have sulfur but no phosphorus, and that the DNA
had phosphorus but not sulfur.  So they radiolabeled growing T2 with
<sup>35</sup>S or <sup>32</sup>P.  They then infected E coli cells with either
the sulfur- or phosphorus-labeled phage.  The reasoning goes to determine the
location of the protein and DNA:  the one that is inside the cell is most likely
the material containing genes, while those outside the cell would not likely
be involved.  The viral ghosts were sheared off the cells using blenders, and
then the samples were centrifuged.  Cells are heavier and would pellet, while
viral ghosts remain in the supernatant.  They then showed that <sup>35</sup>S
was present on the ghosts, while the <sup>32</sup>P was within the cells:
the DNA was the material containing genes.  DNA structure was already discussed
above, with its molecular and atomic components.  Each deoxyribonucleotide
contains a 2&prime;-deoxyribose, with a 5&prime; phosphate group, and a 3&prime;
hydroxyl.  Important is the attachment of one of four bases.  The DNA polymer
also has a directionality:  biologically DNA is synthesized and processed
from the 5&prime; to the 3&prime; ends.  Two strands of DNA pair with each
other with anti-parallel directionality.
<p>
Watson and Crick had shown that DNA is probably replicated in a s
emiconservative manner, with strands separating and daughter strands
synthesized with the old strands as templates. But it remained to be proved
experimentally. M. Meselson and F. Stahl wanted to settle the question of
conservative, semiconservative and dispersive replication by growing E coli
cells for multiple generations in <sup>15</sup>N to make sure all the DNA was
<q>heavy.</q>. They then let cells have two more cycles (generations), and
isolated cells after generations 1 and 2. DNA was extracted and centrifuged in
a density medium, where the low and high density forms of the DNA will separate.
 If replication is conservative, the bands in generation 2 will be 3/4 of the
DNA will be low density, while 1/4 will be high density. If it is dispersive,
there will be neither a high or low-density band, but a larger band of
intermediate densities. If semiconservative replication occurs, then after two
generation the high density band is gone and an equal intensity of low density
and intermediate are observed.  The latter was observed, and so replication
is semiconservative.

<h2>Comprehensive Synthesis Model</h2>
<p>
A breakthrough occurred when the enzyme <b>DNA polymerase</b> was discovered
and found to synthesize DNA.  In fact organisms contain different types of
DNA polymerase.  DNA polymerase III is responsible for copying the DNA before
cell division.  All polymerases add nucleotides in the 5&prime;&rarr;3&prime;
direction.  Polymerases also use the triphosphate forms (dNTPs), which explains
just how much energy DNA synthesis needs.
<p>
Another question is how DNA synthesis is started.  Electron micrographs
nicely show <q>bubbles</q> which begin with a sequence that is an <b>origin
of replication</b>.  In bacteria, there is actually only one site where
synthesis begins.  But in eukaryotes there are multiple sites on the linear
chromosomes.
<p>
Replication starts with proteins that bind to the origin of replication
and they direct a <b>helicase</b> to being strand separation in a process
requiring ATP.  To prevent reformation of the duplex bonding,
<b>single-strand DNA-binding proteins (SSBPs)</b> stabilize the open form.
As the helix is unwound, this creates more tension further down the helix,
the tension similar to trying to pull apart twisted strands of rope.  To relax
this, <b>topoisomerases</b> move one strand on the other side of the other.
<p>
<b>Leading Strand Synthesis</b>.  DNA polymerase III requires a template and
it must initiate from a 3&prime;-OH end.  The 3&prime;-end is provided by
a few short ribonucleotides placed by a <b>primase</b>.  As the DNA polymerase
III moves continuously, an assembly called a <b>sliding clamp</b> with a toric
shape falls in behind the polymerase to hold it in place.
<p>
<b>Lagging Strand Synthesis</b>.  The question occurred as to how the lagging
strand does synthesis if the direction must be 5&prime;&rarr;3&prime; and
the replication fork was moving away from the direction of lagging strand
synthesis.  R. Okazaki proposed that a discontinuous replication occurs.  He
suggested that the primase synthesizes the short RNA that primes DNA pol III
and that the discontinuous strands are linked to make them continuous.  To
test the theory, radiolabeled dNTPs were pulsed and then chased with unlabeled.
If the discontinuous hypothesis is true, some short radiolabeled single
stranded fragments should appear.  Cells were extracted and then centrifuged,
and small-sized fragments of size 1000 bp were found.  These are called
Okazaki fragments.  In eukaryotes, the fragments are about 100-200 bp long.
The continuity which involves removing the RNA primers and replacing them with
deoxy form is done by an enzyme different from DNA polymerase III in bacteria:
it is done by <b>DNA polymerase I</b>, which has 5&prime;&rarr;3&prime;
<q>editing</q> ability.  DNA pol I cannot join the succeeding lagging strands
however: this is done by <b>DNA ligase</b>.
<p>
<b>One Multienzyme Complex</b>.  The synthesis of the lagging strand is not
done by enzymes moving down the strand.  Rather, the DNA strand is looped
around a multienzyme and the strand is synthesized on the complex, and the
complex can move together down the replication fork.  While much of the
elements of synthesis have been worked out in E coli, the process is nearly
identical in eukaryotes.  Of the differences, the enzymes are larger in
eukaryotes.  Also lagging strand synthesis is worked by two different DNA
polymerases whereas there is on DNA pol III in bacteria.

<h2>Telomere Replication</h2>
<p>
Bacteria have a circular chromosome and so the circle is always completed.
However in eukaryotes with linear chromosomes, replication is different.
While the leading strand is replicated to the end of linear strand, the lagging
strand is not.  A problem arises when DNA polymerase III cannot add DNA near
the chromosome tip because there is not enough room for the pimase to add
a new RNA primer.  The tip leaves a single strand.  The mechanism degrades
the strand and leaves the chromosome shortened.  Each time a cycle occurs,
the chromosome is shortened 50-100 nt.
<p>
Fortunately telomeres contain no coding sequences but rather thousands of
repeats of the sequence TTAGGG, for a total 10 kb length. The enzyme <b>
telomerase</b> can replicate these 6-base repeats on to the end of the
chromosome, using and RNA template that is integrated into the enzyme. The
single strand overhang is the 3&prime; end of the lagging strand. The enzyme
binds to the overhang and catalyzes the 5&prime;&rarr;3&prime; synthesis to
further extend the overhang, using the built-in RNA template in the telomerase.
Later a primase, DNA polymerase, and then a ligase will fill in the overhang as
much as possible. There will be an overhang, but the double strand is
slightly longer.  Telomerase is not found active in all cell types.  In humans
it is present in cells undergoing meiosis.  Somatic cells generally lack it.
<p>
Hypotheses have been suggested that telomere shortening can lead to cell cycle
arrest and lead to G<sup>0</sup> phase entry.  Another hypothesis is that
failure to regulate telomerase can lead to cells having uncontrolled growth
and cancer.  Experiments were done to add active telomerase to cultured cells
and cells were found to continually divide past the point where they would
normally become quiescent.  The link to cancer is not so certain:  mutant
mice incapable of making telomerase still developed tumors.  Cancerous cells
have active telomerase or at least a mechanism to maintain telomere length.
Thus a drug to target telomerase might be useful in stopping cancer.

<h2>Repair of DNA</h2>
<p>
DNA synthesis occurs quickly, at rates of 50 bp/s in yeast.  The replication
is generally accurate though, with an error rate of less then 1 error per
billion bases.  Each cell in human DNA has about 6 billion bases.  If errors
occurred in replication cycles, cells would be riddled with errors.
<p>
Generally the best way to deal with errors is to avoid them.  The DNA polymerase
is so well built it has only one bad mismatch per 100,000 bases in replication.
E coli cells with high mutation rates were analyzed.  They found that one
strand of the DNA polymerase III, forming the &epsilon; subunit was mutated.
This subunit normally acts as an exonuclease, which functions whenever
the polymerase bonds a mismatched base.  The exonuclease works in the
3&prime;&rarr;5&prime; direction, and only on single-stranded form, meaning
that base-pairing must not have occurred.  The workhorse of bacterial synthesis,
the DNA polymerase III, thus has proofreading ability.  The proofreading
decreases error rate to 1 in 10 million, about 100-fold better than no proofreading.
<p>
But this error rate is also unacceptable.  So the cell has a <b>mismatch repair</b>
system.  This repair system was discovered in a manner similar to how the
polymerase proofreading system was discovered:  by observing E coli mutants.
While DNA polymerase III was normal, cells still had high mutation rates.
The first discovered (in 1960s) was <i>mutS</i> (<i>mut</i> means
<q>mutator</q>).  After 20 years, 10 proteins were identified in E coli to be
involved in mismatch repair.
<p>
<b>Damage Repair</b>.  DNA is under constant stress by its environment:  from
chemicals such as hydroxyl radicals and aflatoxin B1 (in peanuts and corn),
and benzo[&alpha;]pyrene from cigarette smoke.  Radiation more energetic than
UV can also cause damage, such as thymine-thymine (TT) dimerization.
The <b>nucleotide excision repair</b> system detects chemical changes and removes
the damaged bases.  Polymerase and ligase then follow to fill in and seal the
polymer.
<p>
<b>Xeroderma pigmentosum</b> is an autosomal recessive disorder where the
individual is sensitive to UV light.  Even slight exposure to sunlight causes
lesions in the skin.  J. Cleaver (1968) proposed it was related to a defect
in the DNA excision repair system.  E coli with mutations in these systems
showed sensitivity to radiation, and he suspected this was so for humans.
To show this, he collected tissues from XP patients and showed how their
cells were more sensitive to UV radiation in culture, dying off.  To support the
finding, Cleaver then gave radiolabeled dNTPs to the cells to see how they
compared in the incorporation into the cell DNA, and he found vast differences
in incorporation, far less in the XP cells.  The latest studies show that
the mutations can occur in one of seven genes.
<p>
Defects in DNA repair genes have also been associated with cancer.  XP individuals
are 1000-2000 times more likely to get skin cancer.

<h1>Genes and The Genetic Code</h1>
<p>
Genes were studied with respect to inheritance by Mendelian rules in the early
20th century, but not until 1941 was attention given to how they work. G. Beadle
and E. Tatum working with bread mold <i>Neurospora crassa</i> came upon the idea
that gene function can be learned by trying to make gene defective, and then
inferring the function from the phenotype of the mutant.  Alleles that have no
function because of the effort to make them defective are called
<b>knock-out mutants, null mutants,</b> or <b>loss-of-function mutants</b>.
They treated <i>N crassa</i> with radiation, known to damage DNA and make it
nonfunctional.  They were able to isolate one mutant which could not produce
pyridoxine (vitamin B6);  they found it could be traced to a single gene.
They proposed a <b>one-gene, one-enzyme hypothesis</b>:  thus proteins were
made from corresponding genes.
<p>
In 1944 A. Srb and N. Horowitz reported on the hypothesis looking at Arg
biosynthesis in <i>N crassa</i>.  Previous work showed that Arg is synthesized
by a <b>metabolic pathway</b> in three steps, from a precursor &rarr; ornithine
&rarr; citrulline &rarr; arginine, using three enzymes in those steps.
<p>
After generating the mutants by radiation, they conducted a <b>genetic screen</b>
to select for the mutants they were interested in.  The screen started with
supplying arginine to the medium.  Cells in the arginine-supplied medium were
than added to medium with no arginine:  if they grew, they were normal in the
sense they could make their own arginine; if not, they were arginine-dependent
mutants.  In the next experiments, they grew arginine-dependent mutants on
slanted agar medium containing ornithine, citrulline, arginine, or nothing,
representing the metabolic pathway.  They found that they could identify
mutants at each step, and they called them <i>arg1</i>, <i>arg2</i> and <i>arg3</i>.
These represent the metabolic pathway enzymes necessary for biosynthesis.
These results convinced scientists that the one-gene, one-enzyme hypothesis
was correct.  Genes were therefore the instructions for making proteins.

<h2>Central Dogma of Molecular Biology</h2>
<p>
When the one-gene, one-enzyme hypothesis had been shown, scientists already
knew the chemical composition of DNA, four bases, ribose, phosphate.  DNA
was not thought to produce protein directly.  Crick said that DNA was a code,
with different combinations specifying the 20 amino acids.
<p>
Evidence that DNA did not produce proteins directly was found within eukaryotic
cells.  The DNA is bound in a nucleus, while ribosomes which made proteins were
not found there.  F. Jacob and J. Monod proposed that RNA was the link between
DNA and protein.  Their model predicted that <b>messenger RNA</b> were the molecules
with a short life that carry the information.  Proof of this came later when
<b>RNA polymerase</b> was discovered.    Additionally, poly(dT), or DNA composed
only of thymidine, was added to an solution with ribonucleotides and RNA
polymerase.  Fragments of RNA which were poly(dA) were then found, indicating
that RNA is produced as the complement of the DNA, and proving RNA is coded
by the code of DNA.
<p>
These experiments thus proved the central dogma of molecular biology:
<p class="center">
DNA &rarr; RNA &rarr; proteins
<p>
From this, biologists developed a vocabulary to describe this fundamental
process. <b>Transcription</b> is the process of creating a copy from DNA of the
gene into mRNA. <b>Translation</b> is the step of synthesizing the protein from
the mRNA intermediate.
<p>
The dogma also connects what is known about Mendelian inheritance principles
involving genotypes and phenotypes to the molecular underpinning.  DNA clearly
represents the genotype, while proteins ultimately show the phenotype.  The
connection between the phenotypic trait of plant height to an enzyme that
synthesizes a hormone that affects plant height had been shown.
<p>
Additionally, because the diploid organism has two copies of the same gene,
namely alleles, it was discovered that the DNA sequences of the alleles differed
and so therefore the protein sequence coded must differ.  The function would
therefore differ.  Would the function of one work less well&mdash;or not at
all&mdash;compared to the other?  Alternatively, the function of the other
allele, might be better, creating a phenotype with an advantage.  The main point
however is the connection between genetics as it is studied with respect to
inheritance and the molecular basis for genetics.
<p>
The central dogma has variations though.  Not all RNA codes for and is used
in the coding for protein.  RNA alone has important functions.  Among these are
<ol>
<li>it helps to affect the temporality of transcription:  regulates which genes
are transcribed at various times
<li>it is itself involved in mRNA processing from the pre-mRNA transcript
<li>it carries energetic forms of the amino acids used in protein synthesis
<li>it is involved in peptide bond formation in protein synthesis
</ol>
<p>
The central dogma is even defied with respect to the finding that RNA &rarr; DNA:
that is, the information in RNA is used to produce DNA.

<h2>Genetic Code</h2>
<p>
What remained unknown was the <b>genetic code</b>.  The first question was
to determine the <q>word</q> size.  G. Gamow suggested a word size of three
bases with quite simple reasoning.  If the word size was only one DNA base,
that would be for only four amino acids for the four bases.  If the code was
two bases, that could only code for 16 amino acids (4<sup>2</sup>).  With a
three-base (or <b>triplet</b>) code, 64 different amino acids are possible.
<p>
The quick realization was that if it was triplet <b>codon</b>, it would be
redundant for amino acids:  that is one amino acid would have many codons.
Experiments by Crick and S. Brenner to used chemical modification to delete
one, two and three bases at a time confirmed that the <b>reading frame</b> of
the sequence was three bases long.  They did these by examining the amino acids
that remained after the alterations, to see if they changed.
<p>
With the word length established, the time came to crack the code. In 1961 M.
Nirenberg and H Matthaei found a way to create RNAs with known sequence. <b>
Polynucleotide phosphorylase</b> is an enzyme that will add ribonucleotides in
a 5&prime;&rarr;3&prime; direction to the end of a 3&prime; tail. (It also has
3&prime;&rarr;5&prime; exonuclease activity (essentially the reverse of the
direction) in which is hydrolyzes polymer to ribonucleotides. If a large
concentration of NTPs are added, it will perform a synthesis of random
polymer.  If only UTP is present, then poly(U) is formed.  They added this
poly(U) to an in vitro protein synthesis system, and the peptide poly-Phe was
formed.  This could only mean that the UUU codon codes for Phe.  Note that the
DNA sense complement would be AAA.  In using the same process to form
poly(A) and poly(C), they found that poly-Lys and poly-Pro were formed, respectively.
<p>
Also discovered was one codon that is used to signal the start of translation,
and three codons to signal the end of translation. It tooks 5 years and several 
teams to ultimately establish the genetic code. Two amino acids are coded for 
by only one codon. Additionally, no codons coded for more than one amino acid, 
which might complicate the code. Perhaps even the most important point is that 
the genetic code does not change for species, genera, phyla, or life domains: 
it is virtually universal, although there are exceptions. The protists <i>
Tetrahymena</i> and <i>Paramecium</i> actually use the stop codons UAA and UAG 
for coding for Gln.  <i>Candida cylindracea</i> uses CUG to code for Ser instead of
Leu.  Also, the AUG codon is not the only initiator codon in some species.
The common ancestor had one code, and the variations that occur are the rare
moments where a mutation occurs that does not alter the biology of the organism.

<h1>Transcription and Translation</h1>
<p>
The count of proteins coded for in the human genome are not precisely known
although guessed to be 100,000.  The study of how DNA gets made into protein
has been elucidated in great detail, and it differs in the various domains
(bacteria vs. eukaryotes).  The examination will look at the data and see
what is similar and what is different.

<h2>Bacterial Transcription</h2>
<p>
RNA polymerases are enzymes already mentioned above that make the RNA from
the DNA code, without using a primer.  The polymerase reads the template the
way a DNA polymerase would read it:  forming a bond between the complement base
coming in as an NTP to the sense template of the DNA, and performing the synthesis
in a 5&prime;&rarr;3&prime; direction.  A major difference in the pairing is
that uracil (U) is matched for adenines.
<p>
<b>RNA Polymerase</b>.  An important study was to purify the polymerase and
crystallize it to look at is X-ray crystal structure.  They found a core enzyme
of multiple subunits in addition to a subunit that could bind the DNA (sigma
subunit).  The core enzyme catalyzes the polymer.  It is the presence of the
sigma subunit that allows initiation of the polymerase.  In experiments in which
the core polymerase, sigma subunit, and ds DNA were added, the enzyme bound
to specific sites of DNA.  These sites are the <b>promoters</b>, because
transcription is initiated from those points.
<p>
<b>Promoter Sequence Structure</b>.  D. Pribnow looked at the sequences
of many bacteria and also of phages that bind the RNA polymerase.  Promoters
were always on the non-template strand, and about 40-50 bp long.  The
consensus sequence TATAAT was also observed about the window 10 bases
upstream of transcriptional start (the <b>&minus;10 box</b>).  Additional study
also found the sequence TTGACA which was called the <b>&minus;35 box</b>.
In fact, it is the sigma subunit that binds to both these boxes, and it directs
the RNA polymerase core to bind to it.  The details of polymerization are also
understood in how the ds DNA is opened for the template to be read, and how
the NTP adds to the growing transcribed RNA.
<p>
Multiple sigma subunits have been identified. In <i>E coli</i> there are seven,
while in <i>Streptomyces coelicolor</i>, there are more than 60.  The
&minus;35 and &minus;10 boxes exist to bind these subunits, but their sequences
vary for specific binding.
<p>
<b>Elongation and Termination</b>.  Within the RNA polymerase interior
certain domains have been identified that show the function of the enzyme.
The <q>zipper</q> helps to open the ds DNA at the upstream end, while a
<q>rudder</q> directs the separation of template and non-template strand into
channels.  In particular the template strand is directed to the active site
of NTP binding and polymerization.   Termination of polymerization is achieved
by the nascent RNA itself:  it forms a stem-loop structure (hairpin) that
causes the RNA polymer to be released from the enzyme, and this probably pulls
the enzyme off the ds DNA too.

<h2>Transcription and Posttranscriptional Processing in Eukaryotes</h2>
<p>
In bacteria, the sigma subunit, integral to the RNA polymerase, directs
the enzyme (core + sigma) to the promoter, and it initiates the transcription.
<p>
In eukaryotes, nothing in the RNA polymerase actually finds the DNA sequences
that are the promoters.  Rather, distinct proteins called <b>transcription
factors</b>.  For routine expression, there are <b>basal transcription factors</b>.
These bind to the ds DNA independently from the RNA polymerase, and direct the
RNA polymerase to bind to them.
<p>
Early in vitro system experiments with purified RNA polymerase and ds DNA
template revealed this difference. RNA polymerase would transcribe the DNA, but
at random locations and not at designated gene sites. Moreover, both the
template and nontemplate strands were transcribed. Thus sigma-like proteins
must be involved that direct transcription. This was proved by adding fractions
of proteins to the in vitro system, and systematically identifying the proteins
involved.
<p>
<b>Multiple RNA polymerases</b>.  These experiments also revealed that
eukaryotes have three distinct types:
<ol>
<li>RNA polymerase II is the expected polymerase that produces RNA that encode
proteins (that is, it makes mRNA products)
<li>RNA polymerase I specifically produces large RNA molecules that form
the ribosome (though not all the RNA molecules)
<li>RNA polymerase III produces smaller RNA molecules, some in the ribosomes,
and those that form transfer RNA (tRNA)
</ol>
<p>
Both RNA pol II and III also produce the RNA molecules that form snRNPs used
in posttranscritpional processing.
<p>
Of interest is how these types were discovered by research on how <i>Amanita
phalloides</i> mushroom causes poisoning.  Cultured cells treated with the
toxin &alpha;-amanitin caused a cessation of mRNA production, but not so for
tRNA and rRNA.  Higher concentrations of toxin stopped tRNA, but not rRNA.  This
suggested three types of RNA polymerases.
<p>
Additional differences in eukaryotes from bacteria are that promoter diversity
is far greater in eukaryotes.  The RNA polymerase II also includes a TATA box
centered about &minus;30 nt from start.  RNA pol I and III enzymes also interact
with different DNA sequences directly.
<p>
<b>Discovery of Exons and Introns</b>.  Perhaps the most astounding finding
in the eukaryotic genome was that the DNA coding for polypeptide sequences
was itself not a continuous segment, but had several segments of non-coding
DNA.  This was shown by P. Sharp and coworkers in an experiment where
ds DNA was denatured and allowed to hybridize with mRNA, and the hybrid
duplexes observed under an electron microscope.  The images showed that there
were loops of ss DNA with the mRNA pairing with other parts of the DNA.
They then hypothesized that eukaryotic genes had intervening DNA that had to
be processed out.  W. Gilbert suggest the exon (for the coding or <i>ex</i>pressed
parts) and intron (for <i>in</i>tervening sequences) nomenclature.
<p>
<b>RNA splicing</b>. The question arose as to how introns are removed and the
exons joined to create the mRNA. The first finding was a group of RNA-protein
complexes (ribonucleoproteins) that functioned to catalyze the breaking of the
two ends of the intron and the joining (<q>splicing</q>) of the ends of a
preceding exon and a following exon. The ribonucleoproteins are called
<b>small nuclear ribonucleoproteins</b> (<b>snRNPs</b>, <q>snurps</q>).
Importantly, they are located in the nucleus where the processing occurs.  They
are small compared to many RNA-protein complexes (e.g. the ribosome).
Separate snRNPs  come together.  Some bind the RNA:  one snRNP binds at the
3&prime; end of the preceding exon and intron border, another binds on the
RNA just upstream of the intron-following exon border, at a site with an
A base.  Other snRNPs bring together the bound snRNPs to form the
<b>spliceosome</b>, which will cause the intron to loop and to bring in
the A base close to the preceding exon boder.  The 2&prime;-OH on the A base
acts as a nucleophile on the 5&prime; end of the intron, causing a transesterification
forming, and a lariat-like loop.  The free 3&prime; end of the preceding exon
is then brought close to the 5&prime; end of the following exon, with a
transesterification joining the two exons, and the lariat-looped intron set
free.
<p>
About 145 different protein and RNA species have been identified as making
up a variety of spliceosomes with varying specificity.
<p>
<b>Final Transcript Processing: Caps and Tails</b>.  During pre-mRNA
transcription (before intron processing), the 5&prime; end of the RNA molecule
is given a <b>cap</b>, consisting of 7-methylguanylate and a triphosphate
sequence.  Additionally one enzyme cleaves the 3&prime; end while another
directs the polyadenylation of that end to form a poly(A) tail of about 100-250
nt.  The final molecule has a 5&prime; and 3&prime; untranslated segments
which will be crucial to regulation of translation.  The cap and tail to the
mRNA is vital to both its stability and to translational regulation, since
in vitro studies show that the absence of a cap or tail or both cause
ribonucleases to degrade the mRNA or they affect translation efficiency or
initiation.

<h2>Translation</h2>
<p>
The first question about translation was which cell structure and in what
location it occurred. An initial observation was that ribosomes, visible in the
electron microscope, were found in great abundance in cells with very active
protein synthesis.
<p>
R. Britten and coworkers treated E coli with a pulse of [<sup>35</sup>
S]-sulfate to metabolically label Met and Cys for 15 seconds and then chased 
with unlabeled sulfate. At various times thereafter, they stopped the reaction 
and traced the fate of the radiolabel.  They found most of it in free amino
acids.  They also isolated the ribosomes to see if it accumulated there:
it did.  Finally they traced it to being part of proteins/polypeptides.  They
then inferred that ribosomes are involved in protein synthesis.
<p>
Later studies with bacteria show in electron micrographs where ribosomes are
bound to and translating mRNA that is still being actively transcribed.  As there
is no nuclear envelope, coupled transcription and translation is common.
<p>
The details of translation answered many questions as to how it occurs.
One proposal was that the mRNA triplet codon interacted directly with amino
acids in a way that allowed them to be polymerized.  Crick proposed that an
<q>adapter molecule</q> carried the amino acids into place, and that the adapter
interacted with the mRNA codon.  Crick was correct.
<p>
In experiments with cell-free protein synthesis, scientists discovered that
ribosomes, mRNA, amino acids, ATP and GTP were required for translation.
They also learned that another fraction of RNA was required.  This RNA was
eventually identified as <b>transfer RNA (tRNA)</b>.  How it functioned was
learned by accident when researchers working with radioactive Leu was found
to be covalently bonded to the tRNA, and that the binding was ATP-dependent.
The product is an <b>aminoacyl-tRNA</b>, and it is produced by an aminoacyl-tRNA
synthetase.  Later experiments with radiolabeled Leu-tRNA showed that the
radiolabel was associated with proteins, indicating their role in protein
synthesis.
<p>
tRNA molecules are generally 75-85 nt long, and they have extensive secondary
structure, forming at least three hairpins and also pairing on both ends.
The 3&prime; end has an overhang, and the last three bases are
5&prime;-CCA-3&prime;, which is what couples to the amino acid carboxylate.
The 3D structure does not resemble the 2D presentation of the molecule however.
Instead it has a kind of L-shape, precisely positioning the anticodon and the
charged amino acid.
<p>
While 61 codons of the 64 codons specify coding for an amino acid, it should be
expected that there are 61 tRNA species in the genome.  However, the genome
only codes for about 40.  So how can a codon be paired with a tRNA that does
not exist to match it?  The answer appears to be in what is called the
<b>3rd base codon wobble hypothesis</b>.  Many of the multiple codons that
specify just one amino acid have identical 1st and 2nd bases of the triplet;
it is the 3rd base that veries.  This is called a wobble.  It is generally the
case that even if the 3rd base does not perfectly match, the first two will
match sufficiently to pair the tRNA to the codon.

<h2>Details of Translation</h2>
<p>
That ribosomes are composed of RNA and protein have been understood since the
1930s.  High speed centrifuges enabled the separation of the large and small
subunits.  The bacterial large subunit is composed of two large RNA molecules
and 34 different proteins.  The small subunit is used to hold the mRNA into
position while the large subunit does the peptide bond formation.  Within the
ribosome are three cavities holding tRNA molecules.
<p>
<b>Initiation</b>.  The cycle was shown to have an initiation above, but
translational initiation involves many other steps.  The most important part
of initiation is the correct placement of the ribosome.  In the 5&prime;-UTR
of the mRNA of bacteria, there is a sequence called the Shine-Dalgarno sequence
located about six nt upstream from the AUG start, and 7 nts long:
5&prime;-AGGAGGU-3&prime;, and this appears to bind a complementary sequence
of RNA within the small subunit (3&prime;-UCCUCCA-5&prime;).  Initiation factors
then bind to the mRNA-small subunit complex and bring the fMet-aminoacyl-tRNA
in contact with the AUG codon.  The large subunit is then brought into contact
with the complex, placing the fMet-aa-tRNA in the P site (fMet = N-formylMet).
Note that in eukaryotes initiation factors bind to the 7Gppp-mRNA cap, and
direct the mRNA to the ribosome, and the first amino acid Met is not formylated.
<p>
With this initiation process, the elongation proceeds according to the steps
below:
 <ol>
 <li>The next aminoacyl-tRNA enters the ribosome A site, again matching the codon.
 <li>The amino acid or amino acids (however long) at the P site transfers to its
 chain to the amino acid at the A site:  the NH<sub>2</sub> function of the
aa-tRNA is directed to the acyl-phosphate of the peptidyl-tRNA in the P site.
The peptide bond is formed, and the peptide is transferred to the A site.
 <li>The ribosome moves forward three bases:  the P site tRNA moves to the E site
 (for <q>exit</q>), the peptidyl-tRNA moves to the P site.  Any tRNA that slides
out of the E site will be returned to the cytoplasm.
 <li>The cycle continues by looping back to the step where the next aa-tRNA
 enters the A site
 </ol>
<p>
The cycle depends on the peptide molecule being transferred from P to A site.
Protein synthesis proceeds at a rate of 20 amino acids per second in bacteria
and only 2 per second in eukaryotes.
<p>
How does peptide bond formation occur?  The answer was not forthcoming until
3D models could be constructed to show the active site.  Before then there
was guessing about whether RNA mediated peptide bond catalysis or protein.
What was shown is that the active site is surrounded almost entirely by
ribosomal RNA, and so the ribosome is a ribozyme.  This supports the view that
the life may have started with catalysis mediated entirely by RNA molecules.
<p>
<b>Translocation</b>.  The elongation process described above has been shown
to depend on protein <b>elongation factors</b> and on GTP.  Modeling reveals
that the ribosome undergoes shape changes with the elongation cycle.
<p>
Electron micrographs reveal that mRNA strands can hold more than a single
translation assembly (<q>polyribosomes</q>).
<p>
<b>Termination</b>.  When one of the stop codons is encountered, a protein
called a <b>release factor</b> occupies the A site, with size, shape and
electrical charge very much like a tRNA molecule.  Its presence induces the
the hydrolysis of the polypeptide from the CCA tail of the tRNA in the P site.
The two subunits of the ribosome also separate.
<p>
<b>Posttranslational Modifications</b>.  As the nascent polypeptide emerges
from the ribosome, changes to its structure are already occurring.
Molecular chaperones bind with the protein as it emerges from the ribosome
and accelerate its folding process, even it occurs spontaneously.  Modifications
of eukaryotic proteins include glycosylation and lipidation.  Additionally
make proteins are activated or deactivated by on-and-off phosphorylation.

<h2>Molecular Basis of Mutation</h2>
<p>
This section concerns how changes occurs in the DNA.  Any <b>mutation</b> is a
permanent change in the organism's DNA.  In previous sections, the wrong
insertion of a base that is not corrected by the proofreading of DNA polymerase
or the mismatch repair system could become a mutation.  These single base
errors can become a <b>point mutation</b>.  Perhaps the first point mutation
ever described was in the &beta;-globin subunit that makes hemoglobin, causing
sickle cell disease.  The mutation is an A&rarr;T transversion in the 2nd position
of the 6th codon of the &beta; subunit gene.  This single base change produces
a change in the amino acid from Glu&rarr;Val.  Missense (or replacement) mutations
are often deleterious.  In some cases, a point mutation does not alter the
amino acid, so these are <b>silent mutations</b>.
<p>
Note that the sickle cell mutation confers a selection advantage in the
heterozygote (Hb S/Hb A).  When the parasite infects a red cell, sickling
is promoted, and the phagocytes that destroy badly shaped cells also destroy
the parasite.  In uninfected cells, they can retain their normal shape.
<p>
<b>Chromosome Level Mutations</b>.  Previously mutations were discussed that
dealt with changes observable in chromosomes in the microscope.  Polyploidy,
aneuploidy, translocations, and inversions were discussed.

<h1>Regulation of Bacterial Gene Expression</h1>
<p>
<i>E coli</i> has 4300 genes but not all of those are expressed simultaneously.
Which get expressed typically depends on the bacterial environment.  In particular
E coli can get its food from a wide variety of substrates.  When these substrates
are present in the environment, triggers are set to cause genes that utilize
these molecules to be transcribed and translated.  The regulation of expression
follows the central dogma with an extension perhaps:
<p class="center">
DNA &rarr; RNA &rarr; protein &rarr; activated protein
<p>
Some proteins may be active as soon as they are translated.
<p>
At each point from transcription to translation to posttranslational modification,
the activity of the protein can be regulated.  Bacterial gene expression is
regulated at each step.  Transcriptional control is perhaps the most important
because it does not expend wasteful energy making RNA and protein that will
not be immediately used.  With translational control changes in the proteins
expressed can be done.  With posttranslational control, there is no time lag
in which the cell requires an immediate response with a protein that becomes
activated.
<p>
<b>Lactose Metabolism: A Model</b>.  J. Monod, F. Jacob and others discovered
how <i>E coli</i> metabolized lactose in the 1950s and 1960s.  The bacterium
metabolizes a variety of sugars.  Glucose is preferred, but when not available
other carbohydrates can be used.  Lactose utilization first requires the
expression of a transporter to bring the sugar.  Then the enzyme &beta;-galactosidae
can hydrolyze the disaccharide to glucose and galactose.  But the enzyme
is only produced in the presence of lactose, which acts as an <b>inducer</b>.
But is lactose the only factor in inducing the enzyme?  Monod asked that
question and conducted the following experiment.  In one culture there was
only glucose, in another only lactose, and in a third both glucose and lactose.
In the glucose and glucose + lactose dish, there was no enzyme.  In the lactose
only dish, there was enzyme.  Thus glucose is the preferred carbohydrate in
every case, and others are only utilized when it is not present.

<h2>The Genes in Lactose Metabolism</h2>
<p>
Monod and Jacob then set about identifying the genes involved.  Their approach
was to generate mutants that could not metabolize lactose.  These mutants would
lack either the &beta;-galactosidase or transporter protein.  E coli were treated
with UV, x-radiation, and mutagens.  To identify mutants, colonies must be grown
on glucose medium, and then replica plated to a medium that has lactose only.
If a colony appears on the lactose plate, it retains the wild type.  But if a
colony is missing on the lactose only plate, the corresponding colony on the
glucose plate is the lactose-deficient mutant.  Replica plating is done with
sterilized velvet.  Velvet has a rough surface that picks up bacteria in colonies
and confines them to a space.
<p>
Monod used another strategy to identify lactose metabolizers however.  He
added a compound with a galactose moiety that when hydrolyzed, would produce
a colored (yellow) product.  Since this reactant could only be metabolized
by &beta;-galactosidase, if the enzyme was present and active, it made the colony
turn yellow.  If it was a mutant, the colony remained colorless (white).
<p>
Eventually three types of mutants were identified.  The important one was the
<b>&beta;-galactosidase</b>-deficient mutant.  The gene responsible for making
the enzyme was designed with the genotype <i>lacZ</i>.
<p>
In another type, cells could not accumulate lactose within the cell, and so this
was likely a transporter problem.   This enzyme is <b>galactoside permease</b>
and its genotype is <i>lacY</i>.
<p>
In a third type the indicator substrate was hydrolyzed even if not lactose
is present to induce the enzyme.  This shows constitutive expression of both
the <i>lacZ</i> and <i>lacY</i> genes.  This gene was designated <i>lacI</i>
(for inducer).  It was concluded that this gene prevents the expression of the
<i>lacY</i> and <i>lacZ</i> genes.  It is only when lactose, or actually a
metabolized form of lactose called <i>allolactose</i> (the &beta;1&rarr;6 instead
of the &beta;1&rarr;4 linkage), binds to the inducer that the genes are expressed.
<p>
They followed up by mapping the genes on the circular chromosome and found them
close together.

<h2>Negative Control: Repression</h2>
<p>
Transcription can be reduced or augmented. If it is shut off, and then turned
on, it is induced. If it is on and then shut off, it is repressed. In fact, <i>
lacZ</i> and <i>lacY</i> expression can be positively and negatively controlled.
In the late 1950s, L. Szilard proposed to Monod that <i>lacI</i> codes for a
transcriptional repressor.  The repressor binds the DNA.  When interacting with
the DNA, it loses its affinity.  The genes can be freely transcribed.
<p>
In another experiment, they added the repressor protein exogenously to mutants
that did not express it and which were constitutively making the
&beta;-galactosidase and permease.  They found transcription to halt.
<p>
In 1961 Jacob and Monod published a model for how the system worked.
Important in the model was the fact that both lacZ and lacY were controlled by
a single transcriptional unit, which they called the <b>operon</b>.  An
operon is where multiple genes are under single transcriptional control, and
the transcription is one long mRNA product.  Within the <i>lac</i> operon

<h1>Regulation of Eukaryotic Gene Expression</h1>

<h1>Engineering and Recombining Genes:  Biotechnology</h1>

<h1>Genomics</h1>

<h1>Developmental Biology:  First Principles</h1>

<h1>Development in Animals</h1>

<h1>Development in Plants</h1>

<h1>Details of Natural Selection</h1>

<h1>Population Genetics and Evolutionary Processes</h1>

<h1>Speciation</h1>

<h1>The Science of Phylogeny and Paleobiology</h1>

<h1>Taxonomy:  The Prokaryotes Bacteria and Archaea</h1>

<h1>Taxonomy:  Protists</h1>

<h1>Taxonomy:  Deuterostomes</h1>

<h1>Viruses</h1>

<h1>Ecology:  Principles</h1>

<h1>Ecology:  Behavior</h1>

<h1>Ecology:  Populations</h1>

<h1>Ecology:  Communities</h1>

<h1>Ecology:  Ecosystems</h1>

<h1>Biodiversity</h1>


<!-- =================================================================== -->
<hr>
<h1>Sources</h1>
<ul>
<li>Freeman S, <i>Biological Science</i>, 3rd Ed., (San Francisco: Pearson
Education, Inc.), 2008
</ul>

</body>
</html>

