
<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0 Transitional//EN">

<html lang="en">
<head>
<title>Molecular Biology</title>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8">
<link href="file://c:./stdcss/std.css" rel="stylesheet">
<link href="./stdcss/std.css" rel="stylesheet">
<style>
   .baretable, .baretable TD { background:none; }
</style>
</head>

<body>
<p id="title">  Molecular Biology
<hr>
Lectures by Martin Privalsky in the year 1987.
<p>
Suppose a plasmid found in a bacterium called strain A is extracted
and isolated.  It is then used to transform two strains, A and B.
It will be found that strain A produces better (more numerous, greater
quality) transformants.
<p>
Or suppose a phage that emerges from lysis of strain A is now used
to try to infect strain B.  The result: no infection.  Closer examination
shows that the DNA degraded when the capsid attached.
<p>
Meselson &amp; Jahn isolated such an endonuclease from the <i>E coli</i>
K12 strain.  This was the first report of something called a
<dfn>restriction endonuclease</dfn>.
<p>
Restriction enzymes recognize specific DNA sequences, and they compete
for enzyme activities that involve either cleavage of the DNA or its
modfication.
<p>
Type I endonucleases are isolated from strains like K12.  Cofactors
for activity include ATP and S-adenosylmethionine (SAM).

<hr>
<table class="baretable"
  style="width:100%;font:bold 160% Helvetica;background-color:black;color:yellow;">
<tr>
 <td>Biochemistry 201C
 <td style="text-align:center;">Molecular Biology
 <td style="text-align:right;">Spring Quarter 1987
</table>

<h1>Bacterial chromosome structure</h1>
4/6/87
<p>
Processes such as replication, recombination, packaging into
viral capsids, and transcription require the rearrangement of
topological forms of nucleic acid (chromosomes).
DNA in bacteria are packaged into the cells only if <i>supercoiling</i>
is possible.  Enzymes called <b>topoisomerases</b> are responsible for
achieving this coiling.
<p align="center">
 <img src="./molbiol/circDNA.gif"
   alt="superhelical DNA nicked to form closed circular DNA">
<p>
A structural feature of the E. coli chromosome is the presence of
small coiled elements which resemble beads on a chain.
A protein called <b>DNA-binding protein II</b> is present in E. coli
and has a total MW of 20,000, consisting of two identical subunits of 9500
Da.  Experiments with SV40 DNA show this protein is responsible for
forming the beaded structures that look like nucleosomes in eukaryotic DNA.
Unlike nucleosomes, these structures show some instability.
<p>
DNA domains--DNA that is in a restrained state and which is irradiated
to nick it will form a DNA that is partially relaxed.
DNA completely unrestrained and irradiated for nicking will
form thoroughly relaxed DNA.  If coumeromycin is added to these irradiated
DNA solutions, the partially relaxed DNA becomes partially supercoiled,
while the relaxed DNA becomes or stays completely relaxed.
The DNA studied in these particular experiments was F plasmid DNA.
It is known that coumeromycin inhibits a protein called DNA gyrase
which is responsible for supercoiling DNA in bacteria.
Bacterial DNA is never completely relaxed even when multiply nicked.

<p>
E coli which has been carefully
lysed and studied under EM shows 50-100 regions which are 20 ï¿½m long each,
each region representing a <i>domain</i> of supercoiling.  It has been found
that supercoiling of this DNA is likely to have a large RNA component,
as digestion with RNase lowers the sedimentation rate of the DNA
(the rate would be an index of a supercoiled state).
<p align="center">
  <img src="./molbiol/DNAdigestion.gif"
  alt="digestion of DNA from 1500S to 1200S to 850S by DNase and to 155 by RNase">
<p>
Trimethylpsoralen (TMP) is a mutagenic substance which photobinds to DNA
(it is excited by 360 nm light).  A relationship between DNA
supercoiling and the binding number and binding rate of TMP has been
demonstrated.  Nicked DNA shows a lower slope on a curve of TMP binding
vs. light exposure at 360 nm.  Addition of coumeromycin also reduces TMP
binding, the reason apparently being that DNA gyrase is necessary
to maintain a steady state of supercoiling, as events that lead
to loss of supercoiling are always occurring.
<p>
The RNA component in supercoiling is not well known.  Addition of the RNA
polymerase inhibitor rifampicin shows inconclusive results.
RNase is clearly seen to lower the supercoiled state however.

<h2>Topoisomerase</h2>
Supercoiling is described by DNA having certain properties.
<b>Linking number</b> (L) is said to be the number of times
which DNA is intertwined.  It is essentially equal to the sum of the
number of double helical turns (T) plus number of supercoils (W).
With relaxed DNA, W = 0 and so T = L.  For linking number to change,
either T or W or both must change. The topoisomerases
of E coli come in two types.

<table align="center">
<tr>
 <th>Type I
 <th>Type II
<tr>
 <td>DNA topoisomerase I (MW 110,000)
 <td><i>E coli</i> DNA gyrase (also called topoisomerase II)
<tr>
 <td>causes single-stranded breaks (changes L by 1)
 <td>causes double-stranded breaks (changes L by 2)
<tr>
 <td>relaxes negatively supercoiled DNA, with rate of relaxation
   becoming progressively slower with DNA in a more relaxed state
 <td>requires MgATP for activity:  converts + to &minus; twists
  (producing supercoiling) in presence of ATP; in absence of ATP, DNA is
  relaxed
<tr>
 <td>a Tyr residue forms the important catalytic site
 <td>DNA-dependent ATP hydrolysis is observed
<tr>
 <td>maps to <i>topA</i> at 28 min on chromosome;
   mutations to <i>topA</i> produces mutations in DNA gyrase
  <p>
   E coli topoisomerase III is an ATP-binding protein which changes
  L by 1 and binds to novobiocin
 <td>DNA gyrase composed of two subunits:  subunit A is present as
  100 to 105 kDa enzyme resistant to antibiotics nalidixic acid and
  oxolinic acid and responsible for breaking and reforming
  DNA;  subnunit B is 90-95 kDa and is responsible for binding
  to novobiocin and coumeromycin and appears responbile
  for energy coupling
</table>

<p>
The gyrase seems to show a substrate specificity (see below).
Cleavage of the DNA by the gyrase A subunit does not require ATP itself,
only reforming the bonds.  Isolation of DNA mixed with gyrase in ATP
absence produces a complex of DNA covalently bound to a Tyr residue
on the A subunit.

<p style="text-align:center;font:bold 100% 'Courier New',Courier,monospace;">
--AT/AGTC CT--<br>
--TA TCAG/GA--

<p>
Supercoiling and gene expression may be mutually exclusive in bacteria.
DNA relaxed by treatment with coumeromycin shows increased expression of
mRNA species by 10-fold.  It is also known that temperature-sensitive
mutants of the gyrase show increased expression at the nonpermissive
temperature.

<h1>Transposable Elements</h1>
4/8/87
<p>


These are genetically and physically relocatable elements hopping
from one genome to another.


</body>
</html>
