<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01//EN"
   "http://www.w3.org/TR/html4/strict.dtd">
<html>
<head>
<title>Oligonucleotide Synthesis and Purification</title>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8">
<meta name="keywords" content="oligonucleotide synthesis">
<meta name="author" content="S. M. Halloran">
<link href="style/std.css" type="text/css" rel="stylesheet">
<style type="text/css">
 @media print {
		h1 {page-break-before:always;}
	}
 h1 {border-top:10px groove #70a;padding-top:0.3em;margin:auto -2em;
    border-right:10px groove #70a;border-left:10px groove #70a;
   border-bottom:10px groove #70a;padding:0.25em 1em;}
 img.origsize {width:auto;margin:1em auto;display:block;}
 .white-on-blue {background-color:blue;color:white;}
 .magenta-on-blue {background-color:blue;color:magenta;}
 .cyan-on-blue {background-color:blue;color:cyan;}
 .red-on-blue {background-color:blue;color:red;}
 .lime-on-blue {background-color:blue;color:lime;}
 .yellow-on-blue {background-color:blue;color:yellow;}
 .boxed-key {border:solid 2px black;background-color:#ddd;padding:0.01em 0.05em;
       font:bold 100% Arial,Helvetica,sans-serif;text-transform:uppercase;
       display:inline-block;min-width:1.5em;text-align:center;vertical-align:middle;}
  .smaller {font-size:75%;line-height:100%;}
  .header {background-color:#ffffa0;color:navy;
     font:bold oblique 90% 'Times New Roman','Times Roman',Times,serif;}
  th {text-align:center;}
  #errmsg-table {margin:1em -3em;}
  #errmsg-table td {text-align:left;font:normal 83% Helvetica,Arial,sans-serif;
    color:navy;padding:0.1em 0.3em;}
  td.errmsg {color:red!important;font:bolder 90% Verdana,sans-serif!important;}
  table * ul {margin-top:0;}
  .units {margin:0.25em 0 0 0;font-size:83%;}
  td.table-inline-header {background-color:#ffa;
    font:bold italic 100% 'Times New Roman','Times Roman',Times,serif;}
</style>
<script type="text/javascript">
function showHitachi(mode) {
	if (mode == true) { // show it
		document.getElementById("hitachi-hidden").style.display = "";
		document.getElementById("revealHitachi").style.display = "none";
	} else if (mode == false) { // hide it
		document.getElementById("hitachi-hidden").style.display = "none";
		document.getElementById("revealHitachi").style.display = "";
	}
}
</script>
</head>
<body>
<!-- Make use of STRUCTURAL tags, all tags and attributes LOWERCASE!
   p, h1-6.
  underused:  dl/dt/dd, blockquote, q, label, th, thead/tfoot, fieldset, button, cite
   samp (output of programs/scripts), kbd (text entered by user),
   abbr (abbreviated forms of words, phrases), acronym (use for acronyms)
  Avoid OVERUSE of div, span, and class attributes
-->

<p id="title">
Oligonucleotide Synthesis

<p id="subtitle">
with notes on purification

<div class="outerblock">
[<b>NOTE</b>.  This document is not fully complete.  Use the refresh key F5
until this notice is removed.]

<hr>
<p>
This document has been prepared by Mitch Halloran.
<p>
It currently has three sections:
<ul>
<li><a href="#principles">Principles of Chemical Synthesis of Oligonucleotides
and Product Purification</a>
<li><a href="#polygen-10col">PolyGen 10-Column Oligonucleotide Synthesizer System</a><br>
  <a href="#polyeasy">PolyEasy Software User Manual</a>
<li><a href="#transgenomic">Transgenomic WAVE HPLC System:  Use in Oligonucleotide
Purification</a>
</ul>
<p>
Notes on the PolyGen 10 synthesizer were mostly copied from the PolyGen user 
manual, but a lot of the text has been re-worded and generally summarized. 
However, there is a lot of detail in the document as well.
<p>
Notes on the Transgenomic WAVE System are taken from the Transgenomic user
manual and the Hitachi module manuals

<h1 id="principles">Principles of Chemical Synthesis of Oligonucleotides and 
Product Purification</h1>
<p>
This section explains the chemistry principles involved in standard 
oligonucleotide synthesis, as well as what methods can be used to purify
the synthesized products.  Purification can be done in several ways, 
including liquid chromatography, and especially high performance liquid
chromatography (HPLC).

<h2>Synthesis Setup</h2>
<p>
The DNA bases are mixed in liquid phase with a solid phase, to which the base
will be added in a cyclic fashion and attached to the solid phase by a chemical
reaction with functional groups first on the solid phase itself, and later with
addition of a nucleotide monomer.  Attachment to the solid phase allows the washing
away of reagents and byproducts at each step of the cycle extending the polymer.
<p>
The solid phase used for oligonucleotide synthesis in automated synthesizers is 
typically  <b>controlled pore glass</b> (CPG).  Note that CPG is a registered 
trademark of Millipore.
<p>
CPG is made by melting borosilicate glass such that the borates and the 
silicates are separated. The silicate phase is then extracted into a liquid 
medium that involves a sol-gel transformation used to control pore size.  Pore 
sizes can vary from 75 to 3000 &Aring; (7.5 to 300 nm).
<p>
For oligo synthesis, CPG with pore sizes of 500 and 1000 &Aring; is typically 
used.  The size of the CPG particle itself is usually about 100 &micro;m or more 
(the finest grain of sand).  These particles are practical for analytical and 
batch work.  The glass surface is then chemically altered to add functional 
groups such as aminoethyl chains (or perhaps longer alkyl chain when necessary). 
It is the amino group on the end that is then reactive for adding the first 
oligo base.  Generally, the CPG particles for oligo synthesis already have the 
3&prime; base added to it, because the chemical reactions for linking the base 
to the CPG are not done by the typical automated synthesizer (or it requires 
special programming) and the demand for this modified CPG particle is high, so 
it is present as a convenience for rapid product synthesis.  Therefore it is 
possible to purchase CPG-A, CPG-C, CPG-G, and CPG-T either in bulk or packed 
into a column based on the quantity to be synthesized, usually from 25 to 500 
nmol, and even into the micromole range.

<p>
<b>Polystyrene</b> with high cross-linking or as copolymers with polyethylene 
glycol is another type of solid support that might be used as well.  It may be
used particularly with the synthesis of very long oligonucleotides.

<h2>The Four-Step Cycle of Synthesis</h2>
<p>
Chemical <i>de novo</i> DNA synthesis involves the repetition of four chemical
reactions, only one of which actually adds a base to the chain.  The other steps
effectively prepare for the addition of a base, remove chains from the synthetic

<p>
<b>Synthesis is 3&prime;&rarr;5&prime;</b>.  The addition of bases in the 
biological synthesis of DNA and RNA oligonucleotides is well known to occur in 
the 5&prime;&rarr;3&prime; direction, that is, in the 3&prime;&rarr;5&prime; 
direction.  But the chemical synthesis occurs in the opposite direction.  This 
is because of the convenience of how the phosphate diester bonds that form the 
DNA or RNA polymer are modified at each step.
<p>
The steps of synthesis are:
<ol>
<li>Deblocking or deprotection of the 5&prime; hydroxyl
<li>Activation and addition of the base to the 5&prime; hydroxyl
<li>Capping of unreacted 5&prime; hydroxyl
<li>Oxidation of the phosphotriester group of the added base to stabilize it
</ol>
<p>
All of these steps are detailed and illustrated in the sections below.
<p>
The four bases used in the chemical synthesis are shown below.  The atoms and 
bonds that will be part of the finished nucleotide are colored <span 
class="white-on-blue">white</span>.  The important chemically reactive groups 
are bonded with <b><i>protecting</i></b> groups that prevent reaction until the 
step when they are deprotected, either by a reaction that removes a single or 
multiple protecting groups.  The 5&prime;-OH group is protected with a <span 
class="cyan-on-blue">cyan-colored trityl group</span>.  The 3&prime;- phosphorus 
ester is protected by a <span class="magenta-on-blue">magenta-colored 
2-cyanoethyl group</span> and a <span class="lime-on-blue">lime-colored 
diisopropyl amine group</span>.  These protecting groups, one on the 5&prime; 
hydroxyl, and two on the phosphate ester, are common to all the bases used in 
these chemical reactions.  Note that <span 
class="cyan-on-blue">R<sub>1</sub></span>, <span 
class="magenta-on-blue">R<sub>2</sub></span>, and <span 
class="lime-on-blue">R<sub>3</sub></span> on bases G, C, and T correspond to the 
groups on base A shown in the figure below.  Bases A and C have an additional 
protecting group on the base moiety:  the free primary amine must be protected 
as a <span class="red-on-blue">red-colored benzoic acid group</span>.  Base G 
primary amine must be protected as an amide, but it is a <span 
class="red-on-blue">red-colored isobutyric acid group</span> rather than benzoic 
acid group.  The rings of base T have no reactive primary amine, so it requires 
no protecting group.
<p>
At least two or three decades have been required for organic chemists to develop
and revise (improve) what the protecting groups are and how they would synthesized
and removed.
</div>

<p>
<img class="origsize" src="imagesGenomicsReferenceHTML/4-DMT-bases.gif"
  alt="the four dimethoxyltrityl (DMT) bases">

<div class="outerblock">
<h3>Obtaining the Base-Modified Solid Phase Particles</h3>
<p>

The chemical synthesis starts with acquiring (purchasing) the solid phase 
support (controlled pore glass particles) with the 3&prime; base already bonded 
(covalently) to the glass particle.   Before these were commercially available, 
several organic reactions were required to activate the glass particle, attach 
linking molecules to them, and then attach the first (3&prime;) base.  The glass 
particle is composed of silica being a particle with a particular pore size.  It 
is chemically activated to take a <span class="magenta-on-blue">linear alkyl 
amine</span>.  The amine group will then be bonded to the ligand of interest. In 
this case, a <span class="yellow-on-blue">succinyl ester</span> of the base is 
formed.  It will have the <span class="cyan-on-blue">trityl protecting group</span>
on the 5&prime; hydroxyl.  If the base needs a <span class="red-on-blue">protecting
group</span>, it will also be present.  It should be noted that there is no
need for a 3&prime; phosphotriester in this first base, as is present with the
bases that are added.
</div>

<p>
<img class="origsize" src="imagesGenomicsReferenceHTML/oligo-syn-initial.png"
  alt="the first base (3'-end) attached to controlled pore glass particle">

<div class="outerblock">

<h3>Step 1: Deblocking or Deprotecting the 5&prime; End</h3>
<p>
The first step in the 4-step cycle is to remove the <span class="cyan-on-blue">trityl
protecting group</span> to create a 5&prime; hydroxyl on the deoxyribonucleotide.
This is done with a small amount of a trichloroacetic acid catalyst (3%) in
dichloromethane.  The dimethoxyltrityl leaving group will form a stable carbocation
on the methyl group because the three aromatic rings contribute significant
electron density via pi electron resonance and other ways of delocalizing the
positive charge.  The dimethoxytrityl cation has a very deep orange color in fact,
and the completion of the reaction can be monitored. 

</div>
<p>
<img class="origsize" src="imagesGenomicsReferenceHTML/oligo-syn-step-1.png"
  alt="Cycle first step:  deblocking the 5' end, removing the dimethoyxtrityl group">

<div class="outerblock">
<h3>Step 2: Coupling or Condensing An Activated Base</h3>
<p>
It is step 2 that actually covalently extends the oligonucleotide with a new
base.  Step 1 made available the hydroxyl group on the 5&prime;-carbon of the 
deoxyribose, and now step 2 brings the new base in contact with the forming oligo
on the solid phase support.
<p>
The base cannot just be added however to the solution.  It is not in an activated
form, so an activator must be mixed in (co-addition) which targets the diisopropylamine
group bound to the phosphorus atom on the 3&prime;-carbon.
</div>

<p>
<img class="origsize" src="imagesGenomicsReferenceHTML/oligo-syn-step-2.png"
  alt="coupling a new base activated in solution">

<div class="outerblock">
<h3>Step 3: Capping Unreacted 5&prime; Hydroxyls</h3>
<p>
The newly added base has been activated and then coupled, in 3&prime;&rarr;5&prime;
fashion.  The problem is that this reaction is not 100% efficient.  There will be
some molecules that still have a 5&prime;-hydroxyl end that did not get coupled
to an activated base.  Perhaps as little as 1-2% of molecules may be in this
state.  It will be necessary to <q>cap</q> these molecules so that they do not
create problems in the next steps of the cycles.  <b>Capping</b> involves doing
two things:
<ol>
<li>Removing reactive groups (like a hydroxyl group) that can interfere with
the chemistry of the next step or steps
<li>Removing the subset of molecules from the synthesis chain that can 
complicate recovering a homogeneous set of molecules that represent the desired
nucleotide.
</ol>
<p>
For example, suppose you are creating a 15-mer oligo with the sequence 
5&prime;-ATTCAAGCGATTCTG-3&prime;.   If there is no capping, and if there is
failure on each cycle, it is possible to end up with a 14-mer that is a mixture
of molecules where each base was missing at each of the 14 cycles used in making
the 15-mer:

<ul style="list-style-type:none;font-family:'Courier New',Courier,monospace;">
<li>5&prime;-ATTCAAGCGATTCG-3&prime;
<li>5&prime;-ATTCAAGCGATTTG-3&prime;
<li>5&prime;-ATTCAAGCGATCTG-3&prime;
<li>5&prime;-ATTCAAGCGATCTG-3&prime;
<li>5&prime;-ATTCAAGCGTTCTG-3&prime;
<li>5&prime;-ATTCAAGCATTCTG-3&prime;
<li>5&prime;-ATTCAAGGATTCTG-3&prime;
<li>5&prime;-ATTCAACGATTCTG-3&prime;
<li>5&prime;-ATTCAGCGATTCTG-3&prime;
<li>5&prime;-ATTCAGCGATTCTG-3&prime;
<li>5&prime;-ATTAAGCGATTCTG-3&prime;
<li>5&prime;-ATCAAGCGATTCTG-3&prime;
<li>5&prime;-ATCAAGCGATTCTG-3&prime;
<li>5&prime;-TTCAAGCGATTCTG-3&prime;
</ul>
<p>
The first oligo is the molecule that results from a failed couple at cycle 1.  
The second oligo is the molecule that results from a failed couple at cycle 2.
The <i>n</i>th oligo is the molecule that results from a failed couple at
cycle <i>n</i>.
<p>
However if capping is done, those molecules above never elongate and the 
molecules generated will be:  
<ul style="list-style-type:none;font-family:'Courier New',Courier,monospace;">
<li>5&prime;-G-3&prime;
<li>5&prime;-TG-3&prime;
<li>5&prime;-CTG-3&prime;
<li>5&prime;-TCTG-3&prime;
<li>5&prime;-TTCTG-3&prime;
<li>5&prime;-ATTCTG-3&prime;
<li>5&prime;-GATTCTG-3&prime;
<li>5&prime;-CGATTCTG-3&prime;
<li>5&prime;-GCGATTCTG-3&prime;
<li>5&prime;-AGCGATTCTG-3&prime;
<li>5&prime;-AAGCGATTCTG-3&prime;
<li>5&prime;-CAAGCGATTCTG-3&prime;
<li>5&prime;-TCAAGCGATTCTG-3&prime;
<li>5&prime;-TTCAAGCGATTCTG-3&prime;
</ul>
This makes the final product easier to purify.  Keep in mind that these molecules
represent ideally only 1-2% of all the molecules present.  The full length
oligo is probably 98-99% of the molecules present.
<p>
Capping is done (as shown below) by using an anhydride (4-<i>tert</i>-butylphenoxyacetic 
anhydride) to esterify the 5&prime;-hydroxyl in the molecules that failed
to couple, taking them out of the elongation cycle.
</div>
<p>
<img class="origsize" src="imagesGenomicsReferenceHTML/oligo-syn-step-3.png"
  alt="capping off any 5-prime unreacted hydroxyls">
<div class="outerblock">
<h3>Step 4: Stabilization of The Phosphate Group Through Oxidation</h3>
<p>
The phosphate atom on the newly added base (extended to the 5&prime; end), which 
was a phosphoramidite and is now a phosphotriester, is not particularly stable
to the chemical steps that will follow in the cycle, in a repeat of the cycle.
To stabilize it against these chemical steps, the phosphate must be oxidized,
in this case using elemental iodine (I<sub>2</sub>), which is a weak oxidant.
A strong oxidant may affect other functional groups, especially the functional
groups on the purine and pyrimidine bases.
<p>
The oxidation state of phosphorus(V) is much more stable than phosphorus(III), 
which might be subject to cleavage or other modifications of the phosphate
group.
</div>
<p>
<img class="origsize" src="imagesGenomicsReferenceHTML/oligo-syn-step-4.png"
  alt="oxidizing the phosphotriester to stabilize it">

<div class="outerblock">
<h2>Cleaving Synthesized Oligo From Solid Phase Support</h2>
<p>
Once the oligonucleotide synthesis is complete, it is necessary to cleave
the molecule off the solid phase (CPG particle).  Moreover, all the protecting
groups on the purine and pyrimidine bases (protecting the primary amine groups
as amides) should be removed.  This can be done using pH extremes.  First the last
dimethyltrityl (DMT) group on the 5&prime; base is removed using mild acid
hydrolysis in 3% trichloroacetic acid in dichloromethane.  Then the amide 
form protection of the purine and pyrimidine bases (A, G, C, not T) are 
deprotected using ammonium hydroxide.  The ammonium hydroxide also cleaves the
carboxylic acid ester that connects the 3&prime; base to the CPG particle, 
liberating the oligonucleotide into solution.
<p>
The only thing that remains is further purification of the oligonucleotide.
</div>
<p>
<img class="origsize" src="imagesGenomicsReferenceHTML/oligo-deprotection.png"
  alt="detritylation and then deprotection and cleavage from support">

<div class="outerblock">
<h2>Purification of Synthesized Oligonucleotides</h2>

<h3>Desalting</h3>
<p>
Dealting encompasses a variety of methods in which the purpose is to separate 
the liberated (cleaved) oligo from unreacted materials and byproducts present in 
the reaction mixture.  What is not separated are the truncated <i>n</i> &minus; 
1, <i>n</i> &minus; 2, <i>n</i> &minus; 3, ... oligonucleotides also present as
a result of the fact that organic reactions are never 100% completed.  Anyone
who has done organic chemistry in the laboratory knows that yields are never
100% because reactions are not always complete for various reasons.  Desalting
only removes the small molecule substances that are substantially different in
chemical character from a single-stranded DNA molecule of short length.

<h4>Solid Phase Extraction</h4>
<p>
A classic SepPak cartridge packed with C<sub>18</sub> silica is quite efficient
at purifying single-stranded DNA from the ions and the small molecule protecting
groups that were removed in deprotection.  Because single-stranded DNA of 
a length of 15 bases or more (the typical oligo/primer) is not a typical 
hydrophobic organic molecule that binds with pure water with a small amount
of trifluoroacetic acid (TFA) ion-pairing agent, a different solvent is used to
get the DNA to bind to the solid phase other than 0.1% TFA solution.  The problem
here is that charged groups on molecules need to be masked in order to bind well
to the phase of hydrophobic C<sub>18</sub> linear chains.  The phosphodiester
backbone is nothing but a high charge density of anions whose negative charge
needs to be masked well in order to make the DNA strand more hydrophobic.  
The role of TFA is that it is a very strong acid and it protonates the silanol
groups on the silica support (which are negatively charged and cause ionic
interactions), and it also protonates any carboxylates found on many organic
molecules.  In addition, the trifluoroacetate anion associates with any protonated
functions on the organic molecules being separated (amine groups get protonated)
and these are masked.
<p>
The ss DNA in 0.1% TFA however might break down the oligo, so a more mild 
pairing agent is used.  In this case, triethylamine acetate, 
[CH<sub>3</sub>CH<sub>2</sub>NH(CH<sub>2</sub>CH<sub>3</sub>)<sub>2</sub>]<sup>+
</sup> <sup>&minus;</sup>OOCCH<sub>3</sub>, is the most effective.  The cationic 
part associates with negatively charged phosphate and its triethyl groups 
increase its hydrophobocity for binding to C<sub>18</sub>, and the acetate group 
can associate with any positive charge that might exist on the purine or 
pyrimidine moieties.

<h4>Alcoholic Precipitation</h4>
<p>
That DNA is insoluble (precipitated) in alcohol such as ice-cold ethanol (EtOH)
or perhaps room temperature isopropanol or butanol is well known.  But the DNA
precipitated is both double-stranded and much longer (usually &gt; 1000 bp)
than the DNA for oligos here.  Oligos that are too short are still soluble in
alcohols typically.

<h3>Reversed-Phase Chromatography (RP-HPLC)</h3>
<p>
When multiple samples are to be purified, HPLC is a must.

<h3>Ion-Exchange Chromatography (IEX-HPLC)</h3>
<p>
GE HealthCare Life Sciences published and application note titled 
<i>Oligonucleotide purification by ion exchange chromatography: process 
development, optimization and scale-up</i> in which they did a very large scale
(150 grams) purification of phosphorthiolate-based antisense oligonucleotides 
with a yield of about 80% and purity of 95%.  They used a quaternary ammonium
column.
<p>
Antisense oligonucleotides as potential therapeutics are synthesized as 
phosphorthiolates rather than as natural phosphodiesters.  The reason is because
the sulfur-based nonbridging atom has a longer half life compared to the natural
form.
<p>
Using ion-exchange chromatography to purify oligonucleotides is quite useful in 
that it works well in aqueous conditions and that elution order is based on 
oligonucleotides length, the desired feature of cleaning up after 
oligonucleotide synthesis.  Use of this however requires an additional clean-up 
step to remove salts using either ultrafiltration or gel filtration with 
desalting column (Sephadex G-25).
<p>
The goal this purification is to remove oligos having lengths other than the 
planned target.  Usually the impurities have a smaller number of bases.  Other
impurities may be related to unwanted oxidation of phosphorthiolates, that is
(P=O)<sub>x</sub> impurities, where <i>x</i> is the number of linkage where 
nonbridging sulfur is replaced by an oxygen atom.  The (P=O)<sub>1</sub> 
is the most abundant impurity.
<p>
At the laboratory scale an AKTAexplorer 100 automated LC with XK 26 (2.6 cm dia) 
column was used.  The buffer is 20 mM NaOH and elution is with increasing NaCl. 
Use of NaOH ensures that no secondary structure is present and that oligos do no 
aggregate.  The 20-mer ISIS 2302 oligo (5&prime;-GCCCAAGCTGGCATCCGTCA-3&prime;) 
was tested.  To determine purity based on length, capillary gel electrophoresis
with 40 cm gel-filled capillary is run at 300 V/cm.  Phosphodiester content is
determined by analytical anion exchange: 1 ml RESOURCE Q column run with a 
20 mM NaOH equilibration and 20 mM NaOH + 2.5 M NaCl (solvent B) in a gradient
with <i>T</i> = 70&deg;.
<p>
Three different media were tested: Q Sepharose Fast Flow, Q Sepharose High
Performance, and SOURCE 30Q.

<h3>Urea Polyacrylamide Gel Electrophoresis</h3>

<h3>Thin-Layer Chromatography (TLC)</h3>
<p>
TLC is a less expensive alternative to HPLC for small-length (primer-sized)
oligos, able to separate <i>n</i> (full-length) from <i>n</i> &minus; 1 oligos.

<!-- =====================================================================
     PolyGen 10-Column Synthesizer
========================================================================== -->

<h1 id="polygen-10col">PolyGen 10-Column Synthesizer</h1>
<p>
The PolyGen 10-Column Synthesizer is a very flexible instrument.  Bulk CPG
particles (CPG not prepacked into and purchased as columns) is used and poured
into cylindrical holes (<q>columns</q>) in a sliding bar (<q>slider</q>).
The instrument has <a href="http://www.polygen.de/polygen_slider_e.html">four 
<i>sliders</i></a>.

<p>
<b>Fast and High Quality (HQ) Programs</b>.  The Fast program would probably be
the choice for routine service.  The oligos synthesized are adequate for 
commercial production.  Total solvent use is 600 &micro;l.  The High Quality
program uses more solvent (1.1 ml) and more time in which to improve the 
efficiency of synthesis to a greater level.

<p>
The instrument has special sensors within it to measure volumes of reagents
transferred from stock/working containers to columns.

<p>
<b>Capacities</b>.  The standard slider allows the synthesis of oligos in the
range of 10 to 1000 nmol in each column.  A special large capacity slider 
provides columns in the 1000 - 5000 nmol (1 to 5 &micro;mol).

<p>
<b>Turnaround</b>.  The chemistry of the fast program is such that 10 couplings
(base additions) can occur in 6 minutes.  A typical primer of 18-25 nt can thus
be complete in 15 minutes.

<h2>Safety and Risks</h2>
<p>
<b>Always wear nonpowder laboratory gloves</b>.  There are two purposes here:

<ul>
<li><b>Worker protection</b>.  The larger purpose is to protect the instrument 
user here.  Many of the chemicals, such as reagents and solvents, used here are 
caustic, toxic, and possibly mutagenic.  Dropping chemicals on the skin might 
cause irritation and hyperinflammatory reactions, as with many chemicals in the 
laboratory.  In addition, the synthesized oligo is a small DNA molecule that 
might have some kind of unwanted biological effect that is unknown or uncertain.
Many small lengths of DNA might exist in nature that have a particular biological
effect that is harmful.
<li><b>Prevention of sample contamination</b>.  Small molecules of DNA (oligos)
are present in the sloughed dead skin of the body, and these too can act as
oligos that complicate PCR-based studies if they become primers in an amplification.
</ul>
<p>
<b>Wear a laboratory coat</b>.  These chemicals can damage ordinary street clothing,
and materials from street clothing c
<p>
<b>Work in a properly rated fume hood</b>.  The first thing you should do is to
turn on the fume hood if it requires a warm-up period for proper operation (some
do require it).  Then continue preparing for the synthesis in other ways.  
Pipetting and mixing reagents that give off caustic fumes is dangerous outside
of a fume hood.
<p>
<b>Safety goggles/eye protection and Eye Wash</b>.  You should be wearing some
kind of eyewear approved for use in the laboratory in acting as a barrier between
your eyes and any chemicals you are working with.  If you are working in a fume
hood, at least that also provides an eye barrier.  But not all operations where
splashing of chemicals in the eyes occurs inside a fume hood.  <b>You are risking
permanent eye damage and even blindness by manipulating chemical containers and
samples without eye protection of any kind!  DON'T DO IT!!</b>.

<h3>Waste and Its Disposal</h3>
<p>
Although small volumes of waste are generated, you should be aware that it is
caustic, toxic, and mostly likely flammable.
<p>
If there is no proper waste disposal service for the facility, then perhaps the 
following can be done, in this order:
<ol>
<li>volatile solvents can be allowed to evaporate in a large 
surface-area-to-volume container.  Waste can be poured into a glass baking dish 
with large area, and then put under the fume hood from one to a few days to 
allow evaporation.  Alternatively, the glass dish can be placed outside under 
the sun in dry weather, where there is no human traffic, and it will evaporate 
quickly.
<li>The dried residue might be a toxic mix of nonvolatile chemicals, and it may
not be safe to wash them down a drain.  Perhaps the best detoxifier is the most
economical yet powerfully strong oxidizer:  bleach.  With the glass dish in the
fume hood, mix bleach with water in a 1:1 to 1:3 volume of bleach:water.  Just
enough volume to cover the bottom of the glass dish.  Allow to sit for 24 h.
<li>Pour the oxidized mix down the sink with lots of water, being careful not to 
spill any material on the skin.
</ol>

<h3>Electrical Supply</h3>
<p>
Always switch the power off and remove the plug from the electrical socket
if any kind of service involving removal of the instrument cover panel is to be
done.
<p>
The PolyGen-10 has a universal power supply that operates the instrument
with A/C voltages ranging from 80 to 240 VAC.



<h2>User Interface &amp; Programming</h2>
<p>
There are different software interfaces available to the user.  The instrument 
features its own LCD display that operates the instrument and allows programming 
in a blunt (less user-friendly) interaction with the user.  The LCD display
and softkeys panel is shown in the image below.

</div>
<p>
<img class="origsize" src="imagesGenomicsReferenceHTML/PolyGen10Col-UserInterface.png"
  alt="LCD display and softkeys interface to PolyGen-10Col synthesizer">
<div class="outerblock">
<p>
Alternatively, separately purchased software that operates on a Windows system 
and which controls the instrument can be used, providing a much more friendly 
interface. The Windows software would be recommended for instruments being used 
for commercial production, as it would reduce the risk of making errors, as well 
as be used in records keeping (database interaction), which the LCD interface 
does not offer.

<p>
The software interface for the PolyGen 10-Column has many parameters configured
by the user.  Flow rates and step times are all settable to values other than
default values, providing significant flexibility and not forcing the user into
using only preset programs.

<h3>The Edit Menu</h3>
<p>
Press the <span class="boxed-key">edit</span> key to go into the <b>Edit</b> 
menu.
<p>
The edit menu has four selections on start-up.  These selections present submenus
and the purpose is given below.
<ul>
<li><b>Sequence</b><br>
The sequence to be synthesized is entered
<li><b>Seq2Col</b><br>
Chooses the column number (from the ten available) on which the sequence is to be
synthesized
<li><b>Main-Prg</b><br>
The details of the steps to be executed for the entire synthesis
<li><b>Cpl-Prg</b><br>
The details of the single step involving the addition of base monomer which
is to be activated and coupled
</ul>
<p>
Use the <span class="boxed-key">&lt;</span> and <span class="boxed-key">&gt;</span>
keys for selecting the submenu choice, and the <span class="boxed-key">ret</span>
key to enter the submenu.

<h3 id="edit-seq">Editing the Sequence</h3>
<p>
By selecting <b>Sequence</b> submenu, a series of displays will be presented
for entering a sequencing and assigning the sequence to a column.  Each of these
selections has an entry or additional submenu set of options
<ul>
<li><b>Sequence Number</b> submenu<br>
Up to 10 (for each column position) can be stored in the PolyGen-10Col memory.
Choose a number 1 to 10 using either the numeric keys <span class="boxed-key">1</span>,
<span class="boxed-key">2</span>, ... <span class="boxed-key">0</span> or the
<span class="boxed-key">&Lambda;</span> or <span class="boxed-key">V</span> (down
or up) keys to select a sequence number.  The <span class="boxed-key">ret</span>
enters that submenu:
 <ul>
 <li>Sequence Number submenu choice: <b>New Base</b><br>

  Use the <span class="boxed-key">&lt;</span> and <span 
  class="boxed-key">&gt;</span> keys and select <span 
  class="boxed-key">ret</span> in order to delete any existing sequence before 
  adding a new base, if desired. For adding a new base, use the keys  <span 
  class="boxed-key">A</span> <span class="boxed-key">C</span> <span 
  class="boxed-key">G</span> <span class="boxed-key">T</span>  <span 
  class="boxed-key">X</span> <span class="boxed-key">Y</span> and <span 
  class="boxed-key smaller">dmt<br>off</span>.  During editing, the <span 
  class="boxed-key">&lt;</span> and <span class="boxed-key">&gt;</span> keys can 
  be used to position the cursor and the <span class="boxed-key">ins</span> and 
  <span class="boxed-key">del</span> keys used to insert or delete bases, 
  respectively.  The display will show <b>Sequenz</b> number being edited,
  <b>Total</b> number of bases in sequence, and the <b>Position</b> of the 
  cursor in the editing field.  When editing is complete, press the 
  <span class="boxed-key">esc</span> key to return to the parent menu;
  the updated/edited sequence is stored.

 <li>Sequence Number submenu choice: <b>Edit Base</b><br>

  If there is an existing sequence, the editing of it can be done with this
  option.  Selecting <b>New Base</b> can cause a deletion of the existing
  sequence with input of a new one.  All the keys of the New Base option are
  to be used.

 <li>Sequence Number submenu choice: <b>Edit Comment</b><br>
  Use this selection to enter perhaps a customer order, tracking, or lot number
  if useful.  For writing alphabetic characters, they are selected by the 
  <span class="boxed-key">&Lambda;</span> or <span class="boxed-key">V</span> keys.
  To move from character position from left to right and back, use the
  <span class="boxed-key">&lt;</span> and <span class="boxed-key">&gt;</span>
  keys.
 </ul>
</ul>

<h3>Assigning A Sequence To A Column</h3>
<p>
In this selection, the ten stored sequences can be assigned to one of the ten
columns in the synthesizer.  Look at the image of the LCD display below:
<p>
<img class="origsize" src="imagesGenomicsReferenceHTML/PolyGen10Col-Seq2Col-Display.png"
  alt="LCD display of Sequence-To-Column configuration">
<p>
There are two rows of values.  The top row corresponds to the position 
(numbering) of the ten columns of the synthesizer.  The bottow row corresponds 
to the number of the defined sequence explained in the <a 
href="#edit-seq">Editing the Sequence</a> section.
<p>
In the example given in the LCD display, Sequence Number (<b>Seq-Nr</b>) 1 is 
assigned to column 1.  Both columns 2 and 3 have the Sequence Number 3 assigned 
to them;  this might the case if double the amount of oligo is to be 
synthesized, or if column 2 is to be made at 50 nmol and column 3 is made at 500 
nmol, but both with the same sequence.  Column 4 is assigned to make the defined 
sequence of Sequence Number 7.  Columns 8, 9, and 10 are not assigned.
<p>
<b>IMPORTANT</b>:  if a column is NOT to be used in a synthesis run, use the 
<span class="boxed-key">0</span> key for that column assignment, or else
chemicals will be passed through the column anyway.
<p>
The <span class="boxed-key">ret</span> key is pressed when the assignments are 
complete and the changes are to be saved.  Do not press the <span 
class="boxed-key">ESC</span> key <em>unless</em> the changes are <em>not</em> to 
be saved.

<h3>Creating/Editing A Main Program</h3>
<p>
Up to <b>five</b> different main programs can be defined, and each is numbered
<b>1</b> to <b>5</b>.
<p>
Press the key <span class="boxed-key">edit</span>, select the <b>Main-Prg</b> 
option for its submenu and use the numeric key pad or the <span 
class="boxed-key">&Lambda;</span> or <span class="boxed-key">V</span> to select
a program number, and then press <span class="boxed-key">ret</span>.
<p>
In the next display, a <b>Sub-Prg</b> or <b>Comment</b> choice is offered.
The Comment can be used to create a name for the main program.  The
<b>Sub-Prg</b> selection (press <span class="boxed-key">ret</span>) presents
another submenu of the following choices:
<ul>
<li><b>Main-Drv</b><br>
Takes user to the main program for editing it: adding, deleting, changing steps
<li><b>Remove DMT</b><br>
Specifies whether to remove the dimethoxytrityl (DMT) group present on the last 
coupled (5&prime; end) base.  This program step works only if <span 
class="boxed-key smaller">dmt<br>off</span> was specified in sequence input.
<li><b>Finish</b><br>
Specifies a shutdown-run program at the end of the main program run.  This
can be used, for example, to pass a drying gas over the column to dry the 
CPG material.
</ul>

<h4>Main-Drv (Main Program)</h4>
<p>
Each main program can have up to 40 steps.
<p>
Each main program must have one coupling program.
<p>
There are 12 possible steps that can be entered in the main program.
The way to insert/add a step to the program is to select a step with
the arrow keys, then press the soft key under the <b>ENTER</b> command
(this will be the key with the <span class="boxed-key smaller">dmt<br>
off</span> label; see the image of the display below).
<p>
<img class="origsize" src="imagesGenomicsReferenceHTML/PolyGen10Col-MainProgCoding.png"
  alt="interface for main program coding">
<p>
Fluids such as reagents and waste and washing fluids are moved using the pump
in the syntheizer.  The volume of fluid moved is a function of the speed
of the pump (specified in <b>RPM</b>) and the <b>Time</b> that the pump is
operated.  The soft keys for these two parameters are shown in the interface,
and correspond to the <span class="boxed-key">X</span> and 
<span class="boxed-key">Y</span> keys for pumping time and speed (rpm), 
respectively.  Time values can range from 0.1 seconds to 999.0 seconds.
The RPM values can range from 0 to 990, but usually values over 600 rpm
are not needed.  A separate section discusses how to calibrate the pump rpm
to its flow rate.
<p>
<b>Saving A Program</b>.  All additions, deletions and edits to the steps
of the main program are saved by when the <span class="boxed-key">esc</span>
is pressed to return to the parent menu.

<table style="float:left;margin-right:2.5em;">
<caption>Standard Synthesis Program</caption>
<tr><th rowspan="2">Step # <th rowspan="2">Step Name <th colspan="2">Pump
<tr> <th>duration (s) <th> rpm
<tr><td colspan="4" class="header">Main
<tr><td>01  <td>Wash<td>3.5<td>300
<tr><td>02  <td>Debl<td>2.5<td>250
<tr><td>03  <td>Wait<td>55 <td>
<tr><td>04  <td>Wash<td>8.0<td>300
<tr><td>05  <td>Act <td>2.5<td>250
<tr><td>06  <td>CPL  <td>  <td>
<tr><td>07  <td>Wait<td>55 <td>
<tr><td>08  <td>Wash<td>1.0<td>300
<tr><td>09  <td>Cap A  <td>1.5<td>150
<tr><td>10  <td>Cap B  <td>1.5<td>150
<tr><td>11  <td>Cap A  <td>1.5<td>150
<tr><td>12  <td>Cap B  <td>1.5<td>150
<tr><td>13  <td>Wash<td>1.0<td>300
<tr><td>14  <td>Oxid.  <td>2.5<td>250
<tr><td>15  <td>Wash<td>3.5<td>350
<tr><td colspan="4" class="header">Remove DMT
<tr><td>01  <td>Wash  <td>3.0 <td>300
<tr><td>02  <td>Debl  <td>2.5 <td>250
<tr><td>03  <td>Wait  <td>55 <td>
<tr><td>04  <td>Wash  <td>8.0 <td>350
<tr><td>05  <td>Gas<td>4.5 <td>400
<tr><td colspan="4" class="header">Finish
<tr><td>01  <td>Gas<td>10  <td>400
</table>

<table>
<caption>Program High Quality HQ</caption>
<tr><th rowspan="2">Step # <th rowspan="2">Step Name <th colspan="2">Pump
<tr> <th>duration (s) <th> rpm
<tr><td colspan="4" class="header">Main
<tr><td>01 <td>   Wash <td>   3.5 <td>  300
<tr><td>02 <td>   Gas  <td>   4.0 <td>  400
<tr><td>03 <td>   Debl <td>   2.0 <td>  200
<tr><td>04 <td>   Debl <td>   2.0 <td>  200
<tr><td>05 <td>   Wait <td>   30  <td>
<tr><td>06 <td>   Wash <td>   10  <td>  350
<tr><td>07 <td>   Act  <td>   3.0 <td>  250
<tr><td>08 <td>   CPL  <td>       <td>
<tr><td>09 <td>   Wait <td>   60  <td>
<tr><td>10 <td>   Wash <td>   2.0 <td>  300
<tr><td>11 <td>   Cap A <td>  3.0 <td>  200
<tr><td>12 <td>   Cap B <td>  1.5 <td>  150
<tr><td>13 <td>   Cap A <td>  1.5 <td>  150
<tr><td>14 <td>   Cap B <td>  1.5 <td>  150
<tr><td>15 <td>   Wash <td>   2.0 <td>  300
<tr><td>16 <td>   Oxid <td>   2.5 <td>  250
<tr><td>17 <td>   Wash <td>   4.0 <td>  350
<tr><td colspan="4" class="header">Remove DMT
<tr><td>01 <td>  Wash <td>  3.0 <td>  300
<tr><td>02 <td>  Gas  <td>  4.0 <td>  400
<tr><td>03 <td>  Debl <td>  2.0 <td>  200
<tr><td>04 <td>  Debl <td>  2.0 <td>  200
<tr><td>05 <td>  Wait <td>  30  <td>
<tr><td>06 <td>  Wash <td>  10  <td>   50
<tr><td>07 <td>  Gas  <td>  4.0 <td>  400
<tr><td colspan="4" class="header">Finish
<tr><td>01 <td>  Gas  <td>  10  <td>  400
</table>

<table style="float:left;margin-right:2.5em;">
<caption>Program Single Column HQ</caption>
<tr><th rowspan="2">Step # <th rowspan="2">Step Name <th colspan="2">Pump
<tr> <th>duration (s) <th> rpm
<tr><td colspan="4" class="header">Main
<tr><td>01<td> Wash <td> 3.0 <td>  300
<tr><td>02<td> Gas  <td> 4.0 <td>  400
<tr><td>03<td> Debl <td> 2.0 <td>  200
<tr><td>04<td> Wait <td> 20 <td>
<tr><td>05<td> Debl <td> 2.0 <td>  200
<tr><td>06<td> Wait <td> 20 <td>
<tr><td>07<td> Wash <td> 10 <td>  350
<tr><td>08<td> Act  <td> 3.0<td>  250
<tr><td>09<td> CPL  <td>    <td>
<tr><td>10<td> Wait <td> 120 <td>
<tr><td>11<td> Wash <td> 2.0 <td>  300
<tr><td>12<td> Cap A<td> 3.0 <td>  200
<tr><td>13<td> Cap B<td> 1.5 <td>  150
<tr><td>14<td> Wait <td> 15  <td>
<tr><td>15<td> Cap A<td> 1.5 <td>  150
<tr><td>16<td> Cap B<td> 1.5 <td>  150
<tr><td>17<td> Wait <td> 20 <td>
<tr><td>18<td> Wash <td> 2.0 <td>  300
<tr><td>19<td> Oxid <td> 2.5 <td>  250
<tr><td>20<td> Wait <td> 20 <td>
<tr><td>21<td> Wash <td> 4.0 <td>  350
<tr><td colspan="4" class="header">Remove DMT
<tr><td>01<td> Wash <td> 3.0 <td>  300
<tr><td>02<td> Gas  <td> 4.0 <td>  400
<tr><td>03<td> Debl.<td> 2.0 <td>  200
<tr><td>04<td> Wait <td> 20  <td>
<tr><td>05<td> Debl <td> 2.0 <td>  200
<tr><td>06<td> Wait <td> 20  <td>
<tr><td>07<td> Wash <td> 10 <td>  350
<tr><td>08<td> Gas  <td> 4.0<td>  400
<tr><td colspan="4" class="header">Finish
<tr><td>01<td> Gas <td>10<td> 400
</table>

<table>
<caption>Program Biotin or another&nbsp;modifier</caption>
<tr><th rowspan="2">Step # <th rowspan="2">Step Name <th colspan="2">Pump
<tr> <th>duration (s) <th> rpm
<tr><td colspan="4" class="header">Main
<tr><td>01 <td>Wash <td>3.5 <td>300
<tr><td>02 <td>Gas  <td>4.0 <td>400
<tr><td>03 <td>Debl <td>2.0 <td>200
<tr><td>04 <td>Debl <td>2.0 <td>200
<tr><td>05 <td>Wait <td>30  <td>
<tr><td>06 <td>Wash <td>10  <td>350
<tr><td>07 <td>Act  <td>3.0 <td>250
<tr><td>08 <td>CPL  <td>    <td>
<tr><td>09 <td>Wait <td>600 <td>
<tr><td>10 <td>Wash <td>2.0 <td>300
<tr><td>16 <td>Oxid <td>2.5 <td>250
<tr><td>17 <td>Wash <td>10  <td>350
</table>

<table style="float:left;margin-right:2.5em;">
<caption>Program S&nbsp;–&nbsp;Oligos (only&nbsp;with&nbsp;AKC&nbsp;Software)</caption>
<tr><th rowspan="2">Step # <th rowspan="2">Step Name <th colspan="2">Pump
<tr> <th>duration (s) <th> rpm
<tr><td colspan="4" class="header">Main
<tr><td>01<td>Wash <td>3.5 <td>300
<tr><td>02<td>Gas  <td>4.0 <td>400
<tr><td>03<td>Debl <td>2.0 <td>200
<tr><td>04<td>Debl <td>2.0 <td>200
<tr><td>05<td>Wait <td>30  <td>
<tr><td>06<td>Wash <td>10  <td>350
<tr><td>07<td>Act  <td>3.0 <td>250
<tr><td>08<td>CPL  <td>    <td>
<tr><td>09<td>Wait <td>60  <td>
<tr><td>10<td>Wash <td>2.0 <td>300
<tr><td>11<td>Cap A<td>3.0<td>200
<tr><td>12<td>Cap B<td>1.5<td>150
<tr><td>13<td>Cap A<td>1.5<td>150
<tr><td>14<td>Cap B<td>1.5<td>150
<tr><td>15<td>Wash <td>2.0<td>300
<tr><td>16<td>Oxid <td>2.5<td>250
<tr><td>17<td>Sulphur<td>2.5<td>300
<tr><td>19<td>Wait <td>90 <td>
<tr><td>20<td>Wash <td>4.0<td>350
<tr><td colspan="4" class="header">Remove DMT
<tr><td>01<td>Wash <td>3.0<td>300
<tr><td>02<td>Gas  <td>4.0<td>400
<tr><td>03<td>Debl <td>2.0<td>200
<tr><td>04<td>Debl <td>2.0<td>200
<tr><td>05<td>Wait <td>30 <td>
<tr><td>06<td>Wash <td>10<td>350
<tr><td>07<td>Gas  <td>4.0<td>400
<tr><td colspan="4" class="header">Finish
<tr><td>01<td>Gas<td>10<td>400
</table>
   
<table>
<caption>RNA*</caption>
<tr><th rowspan="2">Step # <th rowspan="2">Step Name <th colspan="2">Pump
<tr> <th>duration (s) <th> rpm
<tr><td colspan="4" class="header">Main
<tr><td>01 <td>Wash <td>3.5 <td>300
<tr><td>02 <td>Gas <td>4.0 <td>400
<tr><td>03 <td>Debl <td>2.0 <td>200
<tr><td>04 <td>Debl <td>2.0 <td>200
<tr><td>05 <td>Wait <td>30  <td>
<tr><td>06 <td>Wash <td>10 <td>350
<tr><td>07 <td>Act <td>3.0 <td>250
<tr><td>08 <td>CPL <td>  <td>
<tr><td>09 <td>Wait <td>800 <td>
<tr><td>10 <td>Wash <td>2.0 <td>300
<tr><td>11 <td>Cap A <td>3.0 <td>200
<tr><td>12 <td>Cap B <td>1.5 <td>150
<tr><td>13 <td>Cap A <td>1.5 <td>150
<tr><td>14 <td>Cap B <td>1.5 <td>150
<tr><td>15 <td>Wash <td>2.0 <td>300
<tr><td>16 <td>Oxid <td>2.5 <td>250
<tr><td>17 <td>Wash <td>4.0 <td>350
<tr><td colspan="4" class="header">Remove DMT
<tr><td>01 <td>Wash <td>3.0 <td>300
<tr><td>02 <td>Gas <td>4.0 <td>400
<tr><td>03 <td>Debl <td>2.0 <td>200
<tr><td>04 <td>Debl <td>2.0 <td>200
<tr><td>05 <td>Wait <td>30 <td>
<tr><td>06 <td>Wash <td>10 <td>350
<tr><td>07 <td>Gas <td>4.0 <td>400
<tr><td colspan="4" class="header">Finish
<tr><td>01 <td>Gas <td>10 <td>400
<tr><td colspan="4" style="font-size:85%;background:none;">* different reagents!
</table>


<table style="float:left;margin-right:2.5em;">
<caption>Program Fast</caption>
<tr><th rowspan="2">Step # <th rowspan="2">Step Name <th colspan="2">Pump
<tr> <th>duration (s) <th> rpm
<tr><td colspan="4" class="header">Main
<tr><td>01 <td>Wash <td>2,5 <td>300
<tr><td>02 <td>Debl <td>2,0 <td>200
<tr><td>03 <td>Debl <td>2,0 <td>200
<tr><td>04 <td>Wait <td>25
<tr><td>05 <td>Wash <td>6,0 <td>300
<tr><td>06 <td>Act  <td>2,5 <td>250
<tr><td>07 <td>CPL  <td>    <td>
<tr><td>08 <td>Wait <td>25  <td>
<tr><td>09 <td>Cap A<td>1,5 <td>150
<tr><td>10 <td>Cap B<td>1,5 <td>150
<tr><td>11 <td>Cap A<td>1,5 <td>150
<tr><td>12 <td>Oxid.<td>2,5 <td>250
<tr><td>13 <td>Wash <td>3,0 <td>350
<tr><td colspan="4" class="header">Remove DMT
<tr><td>01 <td>Wash <td>3.0 <td>300
<tr><td>02 <td>Debl <td>2.5 <td>250
<tr><td>03 <td>Wait <td>30  <td>
<tr><td>04 <td>Wash <td>8.0 <td>350
<tr><td>05 <td>Gas  <td>4.5 <td>400
<tr><td colspan="4" class="header">Finish
<tr><td>01 <td>Gas  <td>10  <td>400
</table>

<table>
<caption>Program Coupling</caption>
<tr><th rowspan="2">Step # <th rowspan="2">Step Name <th colspan="2">Pump
<tr> <th>duration (s) <th> rpm
<tr><td colspan="4" class="header">Normal
<tr><td>01 <td>BASE <td>1.3 <td>270
<tr><td>02 <td>Flsh <td>4.0 <td>400
<tr><td colspan="4" class="header">Double cpl.
<tr><td>01 <td>BASE <td>1.3 <td>270
<tr><td>02 <td>Wait <td>20  <td>
<tr><td>03 <td>Act  <td>1.3 <td>270
<tr><td>04 <td>BASE <td>1.3 <td>270
<tr><td>05 <td>Flsh <td>4.5 <td>400
</table>

<table>
<caption>5&nbsp;&micro;mol&nbsp;Program (Special&nbsp;Slider)</caption>
<tr><th rowspan="2">Step # <th rowspan="2">Step Name <th colspan="2">Pump
<tr> <th>duration (s) <th> rpm
<tr><td>01 <td>Wash <td>4.0 <td>400
<tr><td>02 <td>Gas  <td>4.0 <td>400
<tr><td>03 <td>Debl <td>2.5 <td>270
<tr><td>04 <td>Debl <td>2.5 <td>270
<tr><td>05 <td>Wait <td>55  <td>
<tr><td>06 <td>Wash <td>16  <td>350
<tr><td>07 <td>Act  <td>5.0 <td>270
<tr><td>08 <td>CPL  <td>5 or 6 <td>
<tr><td>09 <td>Wait <td>60  <td>
<tr><td>10 <td>Wash <td>2.0 <td>300
<tr><td>11 <td>Cap A<td>3.5 <td>250
<tr><td>12 <td>Cap B<td>3.0 <td>170
<tr><td>13 <td>Cap A<td>3.0 <td>170
<tr><td>14 <td>Wash <td>2.0 <td>350
<tr><td>15 <td>Oxi  <td>5.0 <td>220
<tr><td>16 <td>Wash <td>10.0 <td>300
</table>

<table>
<caption>CPL program</caption>
<tr><th rowspan="2">Step # <th rowspan="2">Step Name <th colspan="2">Pump
<tr> <th>duration (s) <th> rpm
<tr><td colspan="4" class="header">No. 5
<tr><td>01 <td>BASE <td>5.0 <td>270
<tr><td>02 <td>Flsh <td>4.0 <td>400
<tr><td colspan="4" class="header">No. 6
<tr><td>01 <td>BASE <td>5.0 <td>270
<tr><td>02 <td>Wait <td>20  <td>
<tr><td>03 <td>Act  <td>3.5 <td>270
<tr><td>04 <td>BASE <td>3.5 <td>270
<tr><td>06 <td>Flsh <td>4.0 <td>400
</table>

<h3 style="clear:both;">Creating/Editing A Coupling Program</h3>
<p>
This specifies the steps that define the addition (coupling) of a new 
base monomer to the growing oligo on the solid phase.  Although the different
columns may have different bases coupled, this program is run to determine 
how each of the base monomers is to be coupled depending on the base.
<p>
From the main edit interface, select <b>Coupl-Prg</b> and press <span 
class="boxed-key">ret</span>.  The operator then selects a Coupling Program
Number (<b>CouplPrg-No</b>) to create or edit.
<p>
After pressing <span class="boxed-key">ret</span>, a submenu display for
selecting a step in the coupling program <b>Coupl-Step</b> or for editing the
coupling program comment (<b>Comment</b>) is available.
<p>
In the <q>Base Step</q> (Coupling Step) menu, the first step is presented
automatically.  If a different step is available, the user can enter a number
to choose a step or scroll through with the <span 
class="boxed-key">&Lambda;</span> or <span class="boxed-key">V</span> keys.
Each step shows a synthesizer operation that is available (Wash, Fl[u]sh,
Wait, etc.).  It shows also the type of operation currently performed, the 
time it is perfored, and the <b>Rpm</b> value.  <b>By default</b> the
base step time is 1.3 seconds at 270 rpm.  This value is correct and should
not be altered in nearly every case.  A time value too long could cause 
activator to be washed out of the reaction chamber, and a time value too
short will mean the base does not reach the CPG material in slider.

<h3>Preparing For Synthesis:  Filling the Columns</h3>
<p>
The oligonucleotides are synthesized on a solid substrate of <i>controlled pore 
glass</i> (CPG) beads or particles.  The CPG has already been modified with one 
of the four bases, A, G, C, or T.  Because chemical synthesis, unlike natural 
enzymatic synthesis, is from 3&prime;&rarr;5&prime; rather than 
5&prime;&rarr;3&prime;, if the oligo to be synthesized is specified as 
5&prime;-ATTCAGGATTCGTCCGAT-3&prime;, then in order to synthesize this oligo,
the column borehole must be filled with the thymidine (T)-modified CPG (i.e.
CPG-T), since it is the last (3&prime;) base.  As synthesis occurs, base A,
G, C, C, T, ... and so on will be added, in backwards (3&prime;&rarr;5&prime;)
order.
<p>
Each column borehole in the ten-column (ten-borehole) column slider contains two 
<b>frits</b> within the column.  This bottom frit holds the CPG particles in the 
column as reagents are pumped/flushed through the borehole (cylinder) of the 
column. Thus fluid moves through, but not the particles/beads.
<p>
In order to manipulate the bottom frit, a pair of tweezers (forceps) must be 
used. The bottom frit fits snugly into the bottom of a column borehole 
(cylinder). There is a <b>stamp</b> tool (a special cylindrical tool) which is 
used to position the frit down the bottom of the borehole (see image).
<img class="origsize" src="imagesGenomicsReferenceHTML/ColumnSlider-Stamp.png"
  alt="column slider with stamp tool">
It is critical not to use too much pressure on the frit with the stamp tool, 
since it can crush the frit and make it porous, and defeat its purpose.
<img class="origsize" src="imagesGenomicsReferenceHTML/FritPositioningWithStamp.png"
  alt="positioning a frit with the stamp tool">
<p>
Once the bottom frit is positioned, then the CPG particles are spooned into the 
column. The <b>spoon spatula</b> provided with the instrument has two different 
sizes, 0 and 1.  Size 0 will scoop about 3 mg of CPG particles, while size 1 
scoops about 5 mg of particles.  There can be variation on the mass, depending 
on the particle (grain) size.  Four rounded size 1 spoonfuls of CPG can fill a 
column borehole. The range of synthesis is from 10 nmol to 5 &micro;mol (5000 
nmol), a 500-fold range.
<img class="origsize" src="imagesGenomicsReferenceHTML/Spooning-in-CPG.png"
  alt="spooning in the CPG particles into the column borehole">
<p>
After the CPG particles are spooned in, the <b>top frit</b> is then placed on top
of the column.  It is pressed into the borehole with the stamp until it falls 
within the borehole and level with the upper edge of the borehole.
<img class="origsize" src="imagesGenomicsReferenceHTML/Filled-Column.png"
  alt="top frit placement">
<p>
The quantity of CPG to use will depending on experience, namely running tests
to determine proper levels.


<h3>Executing A Synthesis Run</h3>
<p>
In order to perform a synthesis run, a main program must be selected.
<p>
The main program selection interface for executing a run is brought to the 
display using the <span class="boxed-key">SLCT</span> key.  Enter the number 
using a number pad key or using <span class="boxed-key">&Lambda;</span> or <span 
class="boxed-key">V</span> keys to set the value after the prompt 
<b>MainPrg-No:</b>.  Press the <span class="boxed-key">ret</span> to complete 
the selection.
<p>
Press the <span class="boxed-key">start</span> key.  The display is shown as 
below:
<p>
<img class="origsize" src="imagesGenomicsReferenceHTML/PolyGen10Col-StartKeyPressed.png"
  alt="After the START key is pressed: Loading Column Mode">
<p>
The top line <q>Load columns with bases</q> instructs the operator to load the
holes in the Column Slider with CPG particles whose bases are indicated in the
row of A, T, C, G, X, or Y letters shown below the numbers 1 through 10 indicating
the column positions.  The message <b>Calc-time</b> indicates the total time
for the completion of the main program, and <b>MainPrg-No</b> indicates the 
main program selected.
<p>
At this point the operation of the instrument in the run is halted so that the
operator can fill the columns with the appropriate base-modified CPG particles.
<p>
Once the loading of the Column Slider is complete and everything is in place,
the operator presses <span class="boxed-key">start</span> again.
The instrument enters into the <b>RUNNING</b> mode, and the LCD display is shown
as follows:
<p>
<img class="origsize" src="imagesGenomicsReferenceHTML/PolyGen10Col-RunningMode.png"
  alt="After the START key is pressed a 2nd time:  running the syntheses">
<p>
In the top line, <b>Prog</b> indicates the main program number that is running, 
<b>Coupling</b> indicates the number of bases that have been added or coupled to 
the sequence for all columns so far in the execution of the program, and 
<b>Remaining</b> indicates the time in hours and minutes until the entire 
synthesis run is complete for all columns which are programmed for a synthetic
run.
<p>
In the second line, the <b>Step</b> value indicates the step in the main program
currently being executed.  The value <b>Col</b> indicates the column that is 
currently being operated upon for the step currently running.  A time values in 
seconds to the right indicates the time remaining for the performance of the 
step.
<p>
The last two lines shows the column positions and the number of nucleotides that
will eventually be produced by the running program.
<p>
<b>DMT Monitoring</b>.  During the running of the main program, pressing <span 
class="boxed-key">slct</span> changes the display to include the <b>DMT 
Monitoring</b> mode.  The display includes a new field <b>actual Trityl</b> with 
a number value between 0-100 (as a percentage of absorption).  For this number 
to be accurate, the DMT monitor must be calibrated.
<p>
<b>Pausing or Aborting A Run</b>.  There may be a need to <em>pause</em> the 
operation of a synthesis run in order to service the instrument or perform some 
other manual operation. Press the <span class="boxed-key">stop</span> key 
<strong>just once</strong> in order to pause the run.  During a paused run, the 
<span class="boxed-key">spcl</span> (for <q>special</q>) key can be pressed to 
enter an interface that allows the performance of some operations that can be 
done during a paused run, such as washing a column or a tubing line.  If the
<span class="boxed-key">stop</span> key is pressed a second time, the execution
of the main program (and the entire synthesis run) is aborted.
<p>
Whenever a synthesis run is to be interrupted for a possibly longer time, 
make sure it is done during a washing step.  A synthesis can then usually be
resumed successfully from that point.

<h3>Removal of Final DMT Functional Group: Remove DMT</h3>
<p>
Go to the Edit menu <span class="boxed-key">edit</span>.  Select the choice
<b>Remove DMT</b>.
<p>
<img class="origsize" src="imagesGenomicsReferenceHTML/Edit-remove-DMT.png"
  alt="Remove DMT command under Edit menu">
<p>
The interface below is presented.  Note <b>MainPrg: 1 RDMT  Choose 
step...(1..40)</b> in the display.  Up to 40 steps can be programmed using the 
six functions available in the interface.  At the bottom of the display, the 
keys immediately below.  Note that the <span class="boxed-key smaller">dmt<br>
off</span> key functions as the <b>ENTER</b> key.  After the
program has been entered, use <span class="boxed-key">esc</span> to quit
the interface and save changes.
<p>
<img class="origsize" src="imagesGenomicsReferenceHTML/RDMT-program.png"
  alt="RMDT program interface">
<p>
Note that many oligonucleotide purifications that are performed after the oligo
synthesis is complete and after the oligo is separated from the CGP particle
exploit the presence of a DMT group on the 5&prime; end base to achieve the
purification, and so final DMT removal is logically optional. The oligo is then
acidified after purification to remove the DMT group. Determine what your
purification method is before determining whether to use DMT removal or not.

<h3>The Finish Program</h3>
<p>
The purpose of a Finish program may be to perform additional processing on the
synthesized oligos, for example, drying.
<p>
The Finish program is also accessed from the Edit menu (<span 
class="boxed-key">edit</span>) as shown in the display below.  Use the
<span class="boxed-key">&lt;</span>, <span class="boxed-key">&gt;</span>,
and confirm with <span class="boxed-key">ret</span>.
<p>
<img class="origsize" src="imagesGenomicsReferenceHTML/Finish-program.png"
  alt="Finish program select">
<p>
The display will read <b>MainPrg: 1 Fnsh  Choose step...(1..40)</b> as indicated
in the figure below.  
<p>
The programmming of Finish can take up to 40 steps, with the functions
shown below.
<p>
<img class="origsize" src="imagesGenomicsReferenceHTML/Edit-Finish.png"
  alt="Finish program interface">
<p>
When adding or editing a step, using the ENTER soft key.  On the bottom line
of the display just above the soft key line, the current <b>Step</b> number
is shown, the type of step, the duration (<b>Time</b>) of the step, and the
<b>Rpm</b> of the pump if it is pumping operation.  To control and edit 
steps, use the <b>Step+</b> and <b>Step&minus;</b> soft keys.

<h3>Post-Run Recovery of Samples for Cleavage and Deprotection</h3>
<p>
After synthesis is completed, the column slider must be removed from the 
instrument.  One end of the <b>stamp</b> tool contains a pick for removing the
frits and the CPG packing, used as shown in the image below.
<img class="origsize" src="imagesGenomicsReferenceHTML/RemoveSolidPhase.png"
  alt="picking out the solid phase from the column borehole">
<p>
Within the instrument tool set is a <b>holding block</b> that allows the 
CPG particles in column slider to be collected in sample vessels so that they
can be processed in a further manner (cleavage, deprotection).
<img class="origsize" src="imagesGenomicsReferenceHTML/DeliveryBlockUse.png"
  alt="delivery block used to recover CPG-bound oligo">

<h3>Cleavage/Deprotection</h3>
<p>
The oligonucleotide must be liberated from its covalent attachment to the 
solid phase support, that is, the CPG particle.  A 32% ammonium hydroxide
(NH<sub>4</sub>OH) is used and requires at least 7.5 h and might require
25 h dependent on sequence composition and procedure.  CPG in sample containers
of 300 &micro;l up to 1000 nmol and a 1 ml container for 1000-5000 nmol
are used.
<p>
For the smaller container, add 200 &micro;l of concentrated NH<sub>4</sub>OH.
For the larger container, 500-1000 &micro;l.
<p>
Let stand 90 min at room temperature.  Decant the centrifuged material into
another container.  Add 200 &micro;l, shake/vortex, then spin and combine the
supernatant with the first.
<p>
Now incubate the glass container in heating block at 55&deg; for 6-8 h, but
if the proporition of G monomer is &gt; 25%, incubate overnight at 55&deg;.
At as alternative, allow sample to sit at room temperature for 24 h.
<p>
<b>Fast deprotection</b>.  Heat the CPG with 1 ml concentrated ammonium hydroxide,
then add 50 &micro;l triethylamine for 60 min at 80&deg;, then purify.
<p>
<b>Caution</b>.  Always allow sealed containers with ammonium hydroxide to cool
to room temperature before opening!
<p>
The samples can be lyophilized prior to any purification step.

<h3>Special Functions &amp; Service</h3>
<p>
Press the <span class="boxed-key">spcl</span> on the left of the display.
The following interface is presented:
<p>
<img class="origsize" src="imagesGenomicsReferenceHTML/Special-interface.png"
  alt="Finish program interface">
<p>
The meanings of the soft keys are:
<ul>
<li><b>Clck</b>.  Set <b>Cl</b>o<b>ck</b>:  time and date.  A choice between 
European and American format is possible.
<li><b>DMT</b>.  Displays the DMT values for the previous synthesis operations
for all ten columns.
<li><b>Prim</b>.  Priming (flushing) of the reagent lines past the columns via
the flash position in the column slider.
<li><b>Service</b>.  Access to the pump and DMT monitor calibration, manual
operation, setting Ethernet address, the RS485 address, and automatic prime
function.
</ul>

<h4>Setting Date and Time</h4>
<p>
<img class="origsize" src="imagesGenomicsReferenceHTML/Special-Clock-Display.png"
  alt="setting date and time">
<p>
Press key <span class="boxed-key">a</span> to open display for setting date and 
time.  Use the <span class="boxed-key">V</span> and <span 
class="boxed-key">&Lambda;</span> for chaning the month, then 
<span class="boxed-key">ret</span> to confirm.  Then use the number keys 
<span class="boxed-key">1</span>, <span class="boxed-key">2</span>, 
<span class="boxed-key">3</span>, ... to enter the day and year.  Always use
<span class="boxed-key">ret</span> for each input parameter.  Enter the time
also using the numeric keys.  Continue to press <span class="boxed-key">ret</span>
which causes display cursor to jump to format field.  Choose the desired
European or USA format.  [Note that the date/time format display does not 
occur in the Special Clock display, but instead on the start screen.]
<p>
To leave the interface, press <span class="boxed-key">esc</span>

<h4>Display DMT Monitor</h4>
<p>
<img class="origsize" src="imagesGenomicsReferenceHTML/Special-DMT-Display.png"
  alt="showing DMT monitor">
<p>
Press the <span class="boxed-key">C</span> key to the DMT monitor.  Ten values
showing the percentage absorbance of each of the ten columsns are given.
The <span class="boxed-key">V</span> and <span class="boxed-key">&Lambda;</span>
can be used to select the columns, and the <span class="boxed-key">&lt;</span> and <span 
class="boxed-key">&gt;</span> are used to change position within the sequence.
<span class="boxed-key">esc</span> leaves the submenu.

<h4>Priming and Flushing Reagent Tubing</h4>
<p>
In the SPECIAL interface, click the soft key <b>Prim</b> (<span 
class="boxed-key">g</span>) to enter the SPECIAL PRIMING interface.  The display
(below) shows reagent lines, pump speed, and two standard flushing times,
a <b>Time short</b> and <b>Time long</b> values in seconds.  The display row
with blue background represent the reagent lines, and the hard keys that select
them are shown.
<p>
<img class="origsize" src="imagesGenomicsReferenceHTML/Special-Priming-Display.png"
  alt="Special Priming display">
<p>
In order to flush (empty) the line from the oxidizer solution to the waste reservoir,
press and hold the <span class="boxed-key">t</span> key.  The slider will move to the
desired position and the pump runs for as long as the key is held down.
<p>
To select a different line (tubing) to flush, use the <span class="boxed-key">V</span>
and <span class="boxed-key">&Lambda;</span>.  Two additional functions in this
menu are <b>ALLs</b> and <b>ALLl</b>.  These respectively do a flush of all
reagent lines for a short or long duration, the duration being the seconds 
entered into the Time short or Time long setting.
<p>
To set the short and duration times, the Time short and Time long settings 
toggle using the <span class="boxed-key">ret</span>.  This also toggles to the
pump speed (Rpm) key.  Use the numeric keys to set values.  To quit the SPECIAL
PRIMING interface, use the <span class="boxed-key">esc</span> or
<span class="boxed-key">spcl</span> keys.
<!--
<img class="origsize" src="imagesGenomicsReferenceHTML/Special-Priming-Interface.png"
  alt="">
-->
<h4>Bottle Changing for Modifiers</h4>
<p>
When more than four bases are used in a synthetic run, bottles have to be 
changed.  When synthesis reaches the point of requiring a modifying base and
then a bottle change, the instrument gives both a display and audio alert.
The display shows the bottle to be changed.
<p>
Change the bottle, and then press the <span class="boxed-key 
smaller">dmt<br>off</span> hard key for the <b>Prime</b> soft key.  Note in
the example below, with a coupling program running in progress, the bottle
X1 position needs changing.  After changing, and pressing <b>Prime</b>, the
reagent line for X1 will be primed.
<p>
<img class="origsize" src="imagesGenomicsReferenceHTML/Priming-2.png"
  alt="X1 bottle position priming example">
<p>
Once the priming step is done, press <span class="boxed-key">start</span>
to resume the running of the program.
<p>
<img class="origsize" src="imagesGenomicsReferenceHTML/Priming-3.png"
  alt="another example of the stopping of the program for priming">

<h3>Service of the PolyGen 10</h3>
<p>
As for the special functions, press the <span class="boxed-key">spcl</span>
at the left of the display.  Press the hard key (<span class="boxed-key">t</span>)
for the soft key <b>Service</b>.  The Service interface is shown below:
<p>
<img class="origsize" src="imagesGenomicsReferenceHTML/Service-Menu.png"
  alt="Service Interface">
<p>
The service interface has the following selections/functions:
<ul>
<li><b>Manl</b><br>
Enters values for setting offset, rapport, and basic positions of column and
reservoir sliders, and for the calibration and manual operation of the pump.
<li><b>Clear</b><br>
Deletes all memory of entered programs and sequences
<li><b>Monitor</b><br>
Used to calibrate the DMT monitor
<li><b>ID</b><br>
Used to set address values for making network connections, setting LCD brightness,
and duration of the background brightness
<li><b>Aprim</b><br>
Open the autoprime submenu, with its autoprime function providing automatic
flushing of reagent select slider and slider lock (BOLT)
<li><b>EXIT</b><br>
Quits the service menu
</ul>
<h4>MANL:  Manual Movement of Slider</h4>
<p>
<b>NOTE: only experienced users, probably at the direction and under remote
supervision PolyGen service personnel, should use this function, since it makes
critical settings affecting the positions of the sliders.</b>

<p>
The display looks like this after entering this service function:
<p>
<img class="origsize" src="imagesGenomicsReferenceHTML/Column-Slider-Service.png"
  alt="MANL display">
<p>
This state of the instrument now allows movement of the column slider by hand,
within the calibrated intervals.  The default interval is 9.0 mm as shown
in <b>Slider Rapport</b>.  This is because of the spacing of the column boreholes.
<p>
The <span class="boxed-key">V</span> and <span class="boxed-key">&Lambda;</span> 
keys are used to move the slider in the desired direction.  Use of the <span 
class="boxed-key">&lt;</span> and <span class="boxed-key">&gt;</span> causes the 
column slider to be moved in increments of 0.1 mm.
<p>
The slider rapport setting should remain at 9.0 mm, although if there is any 
reason to change it, use the <b>Rap</b> soft key (<span 
class="boxed-key">t</span> hard key) to alter the settings.  A number key enters 
the new value.
<p>
The value <b>Offset</b> is distance from the zero position of the slider to the 
first column in millimeters.
<p>
The <b>Slider Position</b> in the display refers to the current distance or 
position of the slider from its zero point (in millimeters).  
<p>
The soft key <b>Ref</b> (<span class="boxed-key">g</span> hard key) is used for
<i>referencing</i>:  it returns both sliders to the zero point, and then the
sliders move to the starting position.  
<p>
<b>Moving the Reagent Select (Reservoir) Slider</b>.  Press the <b>Res</b> soft 
key (<span class="boxed-key">a</span> hard key) for moving the reservoir slider. 
(In this interface, note the soft key for the <span class="boxed-key">a</span> 
hard key has changed from <b>Res</b> to <b>Col</b>, to indicate manual control 
is no longer on the column slider.) The <span class="boxed-key">V</span> and 
<span class="boxed-key">&Lambda;</span> keys move the slider in calibrated 
intervals in order to flush tubing manually. By pressing <b>Ref</b> (<span 
class="boxed-key">g</span> hard key), reagent select slider is moved to its 
reference position, which is determined using an initialization sensor.  Note 
also that this slider rapport value is different than the column slider (4.0 
mm), which is the distance between individual reservoir supply lines.  This
value should only ever be changed at the direction of PolyGen service engineers.
<p>
<img class="origsize" src="imagesGenomicsReferenceHTML/Reservoir-Slider-Service-2.png"
  alt="Reservoir Slider Service">
<p>
<b>Calibrating Reagent Select Slider</b>.  Follow these directions:
<ol>
<li>Remove the front cover of the PolyGen synthesizer using a screwdriver.
This exposes the column and the PEEK sliders, which are then removed.
<li>Set the reagent select slider to the position next to the initialization 
sense (Res position) using the <span class="boxed-key">V</span> and <span 
class="boxed-key">&Lambda;</span>.  The gauge msut come to a stop at least 
2 mm away from sensor to prevent it from making contact (see the image below).
<img class="origsize" src="imagesGenomicsReferenceHTML/Initialization-Setup.png"
  alt="spacing of gauges">
<li>Use the portable illuminating light (included in the PolyGen service kit) to 
illuminate the supply line tubing of the RES position below.  The display <b>C:0 
R:0</b> appears when the initialization sensors are not making contact, but 
appear as <b>C:1</b> or <b>R:1</b> when contact occurs.  When looking down 
through the borehole marked <b>Res</b>, light should be shining through (see 
image below).
<img class="origsize" src="imagesGenomicsReferenceHTML/Borehole-Inspection.png"
  alt="Checking the reservoir line position using a flash light">
<li>Use the <span class="boxed-key">&lt;</span> and <span 
class="boxed-key">&gt;</span> keys to move the slider to achieved maximum 
brightness (light transmission) is a <q>full moon</q> effect.  This means the
boreholes of the reagent select slider and reservoir supply line line up 
concentrically.
<li>When aligned fully, then register this position with the <b>Set</b> soft
key (<span class="boxed-key">x</span> hard key).
<li>The column slider is then restored to its place.
</ol>
<p>
<b>Calibrating the Column Slider</b>.  After calibrating the reagent select
slider, then the column slider is calibrated.
<ol>
<li>Using the <span class="boxed-key">V</span> and <span 
class="boxed-key">&Lambda;</span> keys, move the slider to the position closest 
to the initialization sensor, which is near column position 10.  The sensor may 
be moved only to within a maximum 2 mm to the mark plate.
<img class="origsize" src="imagesGenomicsReferenceHTML/Calibrating-Column-Slider.png"
  alt="display for calibrating column slider">
<li>Use the illuminating portable light as above to the supply line tubing of the
RES position from below (see image below).
<img class="origsize" src="imagesGenomicsReferenceHTML/Calibrating-Column-Slider-view.png"
  alt="illuminating the column slider borehole for calibrating it">
<li>As for the reagent select slider calibration, use the <span 
class="boxed-key">&lt;</span> and <span class="boxed-key">&gt;</span> keys to 
move the slider to achieved maximum brightness (light transmission) is a <q>full 
moon</q> effect.  This means the boreholes of the column slider align with both 
the reagent select slider and the reservoir supply line line up concentrically.
<li>When aligned fully, then register this position with the <b>Set</b> soft
key (<span class="boxed-key">x</span> hard key).
<li>The PEEK slider is then restored to its place.
</ol>
<p>
<b>Calibrating the Pump</b>.  Pump flow is controlled via its speed in 
revolutions per minute (rpm).  Click the <b>Pump</b> soft key (<span 
class="boxed-key">c</span> hard key) within the MANL interface.   The 
display interface will now appear as follows:
<img class="origsize" src="imagesGenomicsReferenceHTML/PumpSpeedInterface.png"
  alt="pump speed interface">
The pump calibration software interface is as follows:
<ul>
<li><b>Col</b><br>Press this soft key in order to move the <i>column</i> slider 
using <span class="boxed-key">V</span> and <span 
class="boxed-key">&Lambda;</span> keys to the position that makes flow go to the 
waste container.
<li><b>Res</b><br>Press this soft key in order to move the <i>reservoir</i> slider 
using <span class="boxed-key">V</span> and <span 
class="boxed-key">&Lambda;</span> keys to the position that makes flow go to the 
waste container.
<li><b>Cal</b><br>Moves the reagent select slider (reservoir slider) to the Wash 
position and the column slider to the Flush position.  Pump speed it set to 270 
rpm and the duration of pumping to 1 second.  Pumping can be stopped using 
either the <b>Off</b> soft key (<span class="boxed-key">y</span> hard key) or 
the <span class="boxed-key">stop</span> hard key directly.
<li><b>Back</b><br>
Pump motor reverses its direction, causing backward (reversed) flow.  Used to
backwash blocked tubing.
<li><b>Forw</b><br>
Pump motor is set to its normal (forward) direction;  needed when it was set
for backward flow.
<li><b>Off</b><br>
As mentioned above, stops any pump motor operation (stops flow).
<li><b>Exit</b><br>
Leaves the pump calibration interface.
</ul>
<p>
In order to calibrate the pump, do the following:
<ol>
<li>Make sure the tubing from the pump has no air (bubbles) in it.  The tubing
from the pump to the waste container should be filled with acetonitrile (MeCN).
<li>Position a 5 ml graduated cylinder (or pre-calibrated container) at the 
waste end for collecting pumped fluid.
<li>Press the <b>Cal</b> soft key (<span class="boxed-key">g</span> hard key)
in the (<span class="boxed-key">spcl</span> hard key) menu.  This sets speed to
270 rpm, duration for 60 seconds and the sliders move to their position for 
calibration.
<li><b>NOTE:</b> as an alternative to the previous step, the user can the
instrument into MANL, move column slider to Flush and reagent select slider
to Wash, and then use the <b>Forw</b> function to fill the container.
<li>As the pump is operating, the display reports actual values for the speed
(RPM), electric current used (I), voltage of the motor (U), and the time 
remaining (T).  See the image below:
<img class="origsize" src="imagesGenomicsReferenceHTML/PumpCalibration.png"
  alt="position of flow control adjustment ring on the instrument pump">
Electric current consumption is limited to 1.7 ampere maximum (anything higher creates
an error message).  Allow the pump to operate for the full 60 seconds to collect
flow.
<li>At 270 rpm, the pump should be pumping 2 milliters per minute (2 ml/min).
If this is not the case, the the flow adjustment control ring on the pump at the
back of the instrument must be turned and the flow tested with each turn or 
adjustment (see the image for finding the ring).
<img class="origsize" alt="pump calibration output"
src="imagesGenomicsReferenceHTML/FlowRateMechanicalAdjustments.png" >
Pumping occurs by a reciprocating motion through the rotation of a ceramic piston
within the ceramic cylinder.  A full revolution of the piston performs one
cycle of fill-and-empty of the cylinder (suction-discharge cycle).  The piston
travels to the lowest point of the cylinder to transport greatest possible volume
of fluid and removes or clears any air bubbles.  Turning the flow rate control 
ring changes the pump head position and it alters piston stroke length, which
causes changes in flow rate.  Note that the ring can be turned whether the pump
operates or is motionless.
</ol>
<h4>Clearing Memory</h4>
<p>
Within the Service menu, select the <b>Clear</b> soft key (<span 
class="boxed-key">c</span> hard key).  Note that <b><i>all</i></b> stored 
programs&mdash;main, coupling, additional&mdash;will be deleted.  In addition, 
DMT values and sequences done are also deleted.  To leave the function, press 
<span class="boxed-key">esc</span>.

<h4>Calibrating the DMT Monitor</h4>
<p>
A photometric detector tuned to 470 nm is used to measure the absorbance of
dimethyltrityl cation released from the 5&prime; group.  Monitoring can be
done on a linear (value 0) or logarithmic (value 1) scale.  When synthesis is
done at scales of 5 &micro;mol or more, the use of logarithmic mode is 
recommended.  The value for <b>Monitor Mode</b> in the display is set.
<img class="origsize" src="imagesGenomicsReferenceHTML/DMTMonitorCalibration.png"
  alt="calibration of DMT monitor">
<p>
In order to calibrate:

<ol>
<li>Fill the supply line with any fluid not absorbing at 470 nm;  MeCN is 
generally used.  The <b>Monitor-value</b> setting in the display will change.
<li>To zero against this reference fluid (no 470 nm absorbance), press the 
<b>Zero</b> soft key (<span class="boxed-key">a</span> hard key).
<li>Use MANL mode now to fill the tubing with orange-colored calibration fluid
or Oxidizer solution:  
 <ol>
 <li>this requires quitting the DMT Monitor menu (<b>EXIT</b> soft key)
 <li>in MANL, move the column slider to flush (use its command)
 <li>move reservoir slider to Oxidizer position
 <li>press Pump
 <li>press <b>Forw</b> to fill tubing
 <li>stop pump with <b>Off</b>
 <li>quit MANL with <b>EXIT</b>
 <li>in service main menu, press <b>Monitor</b> soft key to resume
 </ol>
<li>Enter <q>75</q> as the <b>Calibration-value</b> if it is not already set.
<li>Press the <b>Set</b> soft key (<span class="boxed-key">c</span> hard key)
to set the calibrant solution.  Note that the Monitor-value shows the transmission
as it is currently measured;  this changes to the calibration-value with the 
calibration fluid still in it.
</ol>
<p>
Note that other displayed parameters are <b>Z</b> (zero), <b>H</b> (set),
<b>K</b> (calibration factor), and <b>AD</b> (current measurement signal of the
transducer).

<h4>Identities</h4>
<p>
In the main Service menu, press the <b>ID</b> soft key (<span 
class="boxed-key">t</span> hard key).  The display below is presented:
<img class="origsize" src="imagesGenomicsReferenceHTML/IdentitiesInterface.png"
  alt="display of the identity service menu">
<p>
The display has the following indicating parameters:
<ul>
<li><b>ID</b><br>
MAC (Media Access Card/network card) address of the instrument.  This is a 
standard of six 8-bit values in hexadecimal format (0-9 and A-F to represent the 
values 0-15 in hexadecimal) normally separated by colon characters.  For 
example, 00:00:00:8A:5A:AD might be the MAC address, although this might be 
presented as 000008A5AAD in the display.  To understand the proper MAC address, 
read it <i>from right to left</i>, putting a colon between values for every two 
digits.  The instrument communicates with the computer it might be connected to
using network addressing, and it uses the TCP/IP protocol (see below).
<li><b>LCD</b><br>
This adjusts the contrast on the LCD display.  A value representing the contrast 
level is given.  Use the <span class="boxed-key">V</span> and <span 
class="boxed-key">&Lambda;</span> hard keys to adjust level.
<li><b>On-Time</b><br>
Indicates the countdown time for the display to switch to stand-by mode after the
last hard key is pressed.  The value can be adjusted by pressing the <b>On</b>
soft key (<span class="boxed-key">g</span> hard key) and then using numeric keys
to set the countdown timer in minutes.
<li><b>IP</b><br>
This reports the Internet Protocol (IP) address of the instrument when 
communicating with a computer running external controlling software. 
Communications use the TCP/IP standard of network protocols.  Make sure the 
computer running external controlling software has the IP address of the 
instrument in order to communicate with and to control it.  To change the IP 
address if necessary, use the <b>IP</b> soft key (<span 
class="boxed-key">a</span> hard key). The subnet 192.168.x.y is assigned by 
standards authorities for use as a private LAN addressing system, so this should 
be used.  The values <i>x</i> and <i>y</i> can be between 0 and 255.  The 
computer running the controlling software should also be on the subnet.  For 
example, the PolyGen 10 can be set to 192.168.0.0 and the computer controlling 
it can be set to 192.168.0.1. On the computer, the subnet mask can be set to 
255.255.255.0.  There should be no need to set the gateway or the DNS addresses.
<li><b>RS485</b><br>
If the PolyGen is communicating through a serial connection following the RS485
standard, then set its value using the <b>RS485</b> soft key (<span 
class="boxed-key">c</span> hard key) to a non-zero value.  The other device should
have its RS485 address set properly as well.  If it is not used, set the 
address value to zero;  this turns off the addressing, and avoids communication
problems.
</ul>

<h4>Autoprime and Bolt</h4>
<p>
Use this service menu to control autopriming and bolting together of sliders.
The <b>APprim</b> soft key (<span class="boxed-key">x</span> hard key) is
pressed, and the following display is shown:
<img class="origsize" src="imagesGenomicsReferenceHTML/Autoprime-Bolt.png"
  alt="display of autoprime and bolt settting">
<p>
<b>Enabling autoprime</b>.  Autopriming causes the automatic flushing of 
the reagent select slider (Flsh position).  If autoprime is enabled, it
shows a value of <q>1</q> and <q>0</q> if disabled.  Enabling and disabling
is toggled using the <span class="boxed-key">a</span> hard key.  In general
autopriming should always be enabled to avoid problems associated with 
residual volumes of reagents in the supply lines/tubing.
<p>
The priming is controlled by the pump settings, which are duration (time, in 
seconds) and speed (rpm) of the pump.   Duration will be controlled by the
<b>Time</b> soft key (<span class="boxed-key">c</span> hard key);  typically
this is a value from 1.0 to 2.5 seconds.   The pump speed is controlled by the
<b>RPM</b> soft key (<span class="boxed-key">g</span> hard key); typically this
is a value between 200-300.
<p>
<b>Bolting</b>.  When sliders are bolted together, a cross flow is prevented.
When a slider moves, it will be between two positions, as shown in the image below:
<img class="origsize" src="imagesGenomicsReferenceHTML/BoltingShown.png"
  alt="slider moving between positions">
<p>
When sliding movement is complete, then boreholes in the system should align,
as indicated in the image below.  Column position 8 of the column slider is
now aligned with Wash position of the reagent select slider, as shown in the
image.
<img class="origsize" src="imagesGenomicsReferenceHTML/SliderAlignment.png"
  alt="slider movements completed, and boreholes lined up">
<p>
The bolting function has three settable values, numeric as 0, 1, and 2:
<ol style="list-style-type:none;">
<li>0: bolt locking is switched off
<li>1: bolt locking is active only during <b>Wait</b> steps
<li>2: bolt locking is active for all sliding movements (default)
</ol>

<h3>Error Messages</h3>

<table id="errmsg-table">
<tr><th>Message  <th>Problem    <th>Corrective Action

<tr><td class="errmsg">Basemotor overcurrent
 <td>The Reagent Select Slider motor is using &gt; 1.7 ampere current.
A protective sensor circuit stops the power to the motor.
 <td>Sliding movement is being prevented by a mechanical problem, typically
caused by buildup of solid material that fouls the slider operation.
<p>
The slider needs to be removed, washed with <b>acetone</b>, mounted after
drying.  Press START; if it remains unresponsive, power the instrument off
and then on.

<tr><td class="errmsg">Column motor overcurrent
 <td>The Column Slider motors are using &gt; 1.7 ampere current.
A protective sensor circuit stops the power to the motors.
 <td>Sliding movement is being prevented by a mechanical problem, typically
by an object within the column that impedes sliding movement.  It might also
be caused by a buildup of solid material that prevents sliding.
<p>
The slider is removed and inspected for objects that should not be in it. Clean 
with <b>acetone</b> and re-mount the slider.  Press START; if it remains 
unresponsive, power the instrument off and then on.

<tr><td class="errmsg">Pump motor overcurrent
 <td>The Pump motor is using &gt; 1.7 ampere current.
A protective sensor circuit stops the power to the motor.
 <td>The Pump head is fouled with a buildup of solid or viscous material.
If the pump can still be operated in the MANL mode (to avoid dismantling
the pump head), backflush the line and the pump head with ethanol or acetone. 
Remove the PEEK slider above the Reagent Select Slider, hold a paper towel
below the borehole, and hang the waste tube in a glass beaker with ethanol
or acetone, then use the reverse pump setting to flush out material.  
If the pump cannot be operated at all, remove the pump head and clean
it preferably in a sonicating bath with water and then ethanol.
The Pump head is tested.  If it fails to operate, the head is replaced.

<tr><td class="errmsg">Check locking!!!
 <td>Two interlocking bolts necessary to slider movement are not correctly
positioned in order to perform proper sliding movement and to create sealing.
 <td>Position the bolts to their correct placements, at the far left.  Close
both of the bolts and restart.
<tr><td colspan="3" style="background:none;padding:0;margin:0;text-align:center;">
<img class="origsize" src="imagesGenomicsReferenceHTML/CheckLockingError.png"
  alt="check locking error" style="margin:0 auto;padding:0;">

<tr><td class="errmsg">Slider Communication error
 <td>During synthesis, an error in activating or selecting a slider occurred.
This might be due to a lost electrical connection (loose cable or wire)
or a defect in the slider control board on the left side of the instrument.
 <td>Examine the electrical connections between the main board and the slider
control board.  Re-connect any loose connection.  If all connections seem 
proper, the slider board may have a defect, requiring replacement of the 
slider control board.  Contact PolyGen service.

<tr><td class="errmsg">Slider Communication error
 <td>During synthesis, an error in activating or selecting a slider occurred.
This might be due to a lost electrical connection (loose cable or wire)
or a defect in the slider control board on the left side of the instrument.
 <td>Examine the electrical connections between the main board and the slider
control board.  Re-connect any loose connection.  If all connections seem 
proper, the slider control board may have a defect, requiring replacement of 
the slider control board.  Contact PolyGen service.

<tr><td class="errmsg">No Pump Communication
 <td>During synthesis, the pump may no longer be responding or it runs too fast
or there is a high-pitched humming noise emanating from it.
 <td>This results from a commuication problem between the main board and the
pump motor control board.  Examine the electrical connections between the main 
board and the pump motor control board.  Re-connect any loose connection.  
If all connections seem proper, the pump motor control board may have a defect, 
requiring replacement of the slider control board.  Contact PolyGen service.
</table>

<h3>Troubleshooting</h3>
<p>
In addition to the error messages reported by the user interface, the following
issues may arise:

<table>
<tr><th>Problem  <th>Possible Cause(s)   <th>Corrective Action
<tr><td rowspan="3">No Power
 <td>Power cord not connected to outlet
 <td>Control the power cord connection to the outlet and the main instrument
switch being turned on after connection
<tr><td>Circuit break tripped
 <td>Disconnect the instrument power cord and reset the breaker.  If it
trips during reset, look for possible causes of an electrical short.  Call
electrical service if necessary
<tr><td>Blown fuse
  <td>The instrument is protected by a fuse, near the power cord entrance. 
Check that the fuse is not burned out, and replace it with correctly 
rated fuse
<tr><td rowspan="3">Fluid under the instrument
 <td>One or more of the sliders is producing leakage
 <td>Check the sliders and ensure that they are fitting tightly without
leaking.  Replace worn or defective sliders, if necessary.
<tr><td>A tubing screw joint is not connected or is loose fitting
 <td>Tighten (without overtightening) the joints that are screwed in
<tr><td>A reagent bottle or flask is loose or not properly secured or is
broken or cracked
 <td>control all the bottles and flasks, to see if they are tight or might
be damaged or broken.  Replace as necessary.
<tr><td>Bubbles are in the waste tubing, and not in monitor tube
 <td>A leak is present on the inlet port side of the pump:  check the 
tubing fit on the pump inlet port and replace the tubing if necessary.
Check the tubing connector and replace it if necessary.
</table>

<p>
The table below describes the terms/abbreviations used for reagents and the
path they take as they go from the reagent bottle to waste.

<table>
<caption>Reagents And Paths Taken In Instrument</caption>
<tr><th>Keyword  <th>Description
<tr><td class="keyword">Wash
  <td>Dry acetonitrile (max water content 30 ppm or 0.003%)
<tr><td class="keyword">Oxid
  <td>iodine in THF / pyridine / water (oxidizer)
    <br> from the reservoir via the column to waste
<tr><td class="keyword">Deblock
  <td>3% trichloroacetic acid (TCA) in dichloromethane (CH<sub>2</sub>Cl<sub>2</sub>)
    <br> from the reservoir via the column to waste
<tr><td class="keyword">CAP A
  <td> acetic anhydride / lutidine / THF
    <br> from the reservoir via the column to waste
<tr><td class="keyword">CAP B
  <td> 1-methylimidazole / THF
    <br> from the reservoir via the column to waste
<tr><td class="keyword">Act
  <td>activator composed of tetrazole / acetonitrile
    <br> from the reservoir via the column to waste
<tr><td class="keyword">Rsrv
  <td>optional reservoir
  <br> might be filled with sulfur for phosphorthioate synthesis
<tr><td class="keyword">Flsh
  <td>flush is open/working, and the tubing line is not in use;
   the column is in bypass and is being flushed/washed in flush
   position (at far left of the column slider)
<tr><td class="keyword">Gas
  <td>the tubing from the supply line is flowing with argon gas
<tr><td class="keyword">Wait
  <td>this program step stops any activity of instrument (pumping,
 reagent transfer) while a chemical reaction incubation occurs
<tr><td class="keyword">DWait
  <td>a special (dynamic) wait step that includes the time period
of the previous step in multiple column operations.  Suppose a deblock
step of 2.5 s duration occurs on 10 columns and DWait = 30 s.  This
means that when deblock is complete on 10 columns, there will be
5 s remaining on the DWait step (column 10 will only have a 5 s 
deblock).
<tr><td class="keyword">CPL
  <td>switching on the coupling program
<tr><td class="keyword">AWash
  <td>wash step special to cleaning out the flush line after
amidite coupling:  reagent select slider moves to MeCN reagent, and
column slider moves to Flsh position; no column flushing is done;
use in combination to Flsh step or as alternative to it
</table>


<h2>Reagent Setup</h2>
<p>
<b>Amidite Preparation</b>.  DNA synthesis grade acetonitrile (MeCN) which is 
specially dried (&lt; 30 ppm water) is added to each amidite bottle to that 
concentration is 50 mg/ml.  To a 1 g reagent bottle, add 20 ml MeCN, and to a 2 
g bottle, add 40 ml.  This results in a concentration of about 58 mM for the 
adenine amidite to 67 mM for the thymidine amidite.
<p>
To draw up the MeCN, first fill a second syringe with argon gas, and perforate
the septum with the gas.  Now perforate another part of the septum with empty
syringe and draw up the MeCN, allowing the other syringe to compensate for pressure
changes by filling with argon.
<p>
Remove needle and allow solution to flow into inner wall of amidite flask. 
Screw back on its lid and dissolve with careful shaking action.  It must fully
dissolve.  Mount the reagent bottle on to the instrument.

<table>
<tr><th>Amidite Name <th>CAS <th>MW  <th>Product Number
<tr><td>DMT-dA(bz) Phosphoramidite  <td>98796-53-3 <td>857.93 
<td>A111081-12, 12 &times; 1 g<br>A111082-12, 12 &times; 2 g
<tr><td>DMT-dC(bz) Phosphoramidite  <td>102212-98-6 <td>833.91 
<td>C111081-12, 12 &times; 1 g<br>C111082-12, 12 &times; 2 g
<tr><td>DMT-dG(ib) Phosphoramidite  <td>93183-15-4 <td>839.92 
<td>G111081-12, 12 &times; 1 g<br>G111082-12, 12 &times; 2 g
<tr><td>DMT-dT Phosphoramidite  <td>98796-51-1 <td>744.81 
<td>T111081-12, 12 &times; 1 g<br>T111082-12, 12 &times; 2 g
</table>

<p>
Amidite reagent flasks should be in a 20 to 40 ml size, with a flask neck
threading of 20-400.
<p>
Other reagents are in flask of 200 to 900 ml size, with flask neck threading
of 28-400.

<p>
<img class="origsize" src="imagesGenomicsReferenceHTML/PolyGenReagentPositions.gif"
  style="width:100%!important;"
  alt="reagent positions for the PolyGen 10-column instrument">

<h2>Operation</h2>

<h3>Synthesis Scale</h3>
<p>
The synthesis scale is critical to an estimation of how 
much oligo will be synthesized.  Most primer synthesis services offer to make 
oligos at 25, 50, 100, 200, 500, 1000, and 5000 nmol.  When primers are used in 
life science research perhaps in cloning procedures (reverse transcription and 
amplification of reversed transcribed DNA, cloning into maintenance and 
expression vectors), then 100 nmol or less of oligo is needed.  If instead 
hundreds of PCR amplifications are to be done as part of a large-scale clinical 
study or as routine diagnostics, then from 200 nmol and even more of oligo is 
desirable.  For CPG-based synthesis as used by the PolyGen, the amount of the 
starting nucleoside attached to the glass support is a function of the surface 
area of the particle and required for determining scale.  
The table below shows the CPG type and the amount of base per mass of particles:
<table>
<tr><th>CPG Type (Pore Size) <th>nmol base/mg CPG
<tr><td>500 &Aring; CPG  <td>30-40
<tr><td>1000 &Aring; CPG  <td>25-35
<tr><td>UHL CPG   <td>50-70, 90-110
</table>
<p>
The scale of synthesis is computed:
<p class="eqnserif">
nmol oligo = CPG Type &times; Amount of CPG
<p>
For 25 nmol scale oligo, if using the 500 &Aring; CPG, this computes to 
25 nmol &times; 1 mg CPG / 30-40 nmol = 0.63-83 mg.  For 50, 100, 200 nmol scale,
just multiply that range by 2, 4, and 8, respectively.  If the 1000 &Aring;
CPG is being used, the range for 25 nmol is 0.71 to 1.0 mg.  This is basically
a fraction of the spoon size 0 (3 mg) and size 1 (5 mg) provided by PolyGen.
Therefore, it might be better to show the spoon sizes and the assumes CPG types,
as given in the table below:

<table class="td-center">
<caption>Predicted Scale (nmol Oligo Synthesized) From Spoonfuls CPG</caption>
<col span="2" style="width:4em;">
<col span="4" style="width:7em;">
<tr><th rowspan="3">Spoon size <th rowspan="3">Spoon Amount<br><small>(mg&nbsp;CPG)</small>
   <th colspan="4">CPG Types
<tr>                     <th>1000 &Aring; <th>500 &Aring; <th colspan="2">UHL
<tr>   <th><big>25-35</big> <p class="units">nmol&nbsp;base / mg&nbsp;CPG
       <th><big>30-40</big> <p class="units">nmol&nbsp;base / mg&nbsp;CPG
       <th><big>50-70</big> <p class="units">nmol&nbsp;base / mg&nbsp;CPG
       <th><big>90-110</big> <p class="units">nmol&nbsp;base / mg&nbsp;CPG
<tr><td>0  <td>3    <td>75-105  <td>90-120  <td>150-210 <td>270-330 
<tr><td>1  <td>5    <td>125-175  <td>150-200  <td>250-350  <td>450-550
</table>
<p>
Note this is a very rough guide and will likely not be the final yield, since
one must account for losses (reduced amounts) that will be obtained because of
failed syntheses (could be from 2-10% depending on sequence length, quality of
reagents and solvents, and non-optimal programming) as well as the necessarily
less than 100% yields due to purification (HPLC, gel electrophoresis, other
systems). You should do a &lambda; = 260 nm reading on the final product to
make sure you know the mass obtained, which can be critical for downstream <i>
analytical</i> applications with a low tolerance for out-of-range oligo
concentrations.

<h3>Instrument Preparation</h3>
<p>
<b>Slider cleaning and oiling</b>.  If necessary (such as not using for
a long period):  
<ol>
<li>Clean the column slider with alcohol (ethanol or isopropanol) using a 
lint-free cloth.
<li>Oil the slider with its specific oil.
</ol>


<h3>Pre-Synthesis Run Checklist</h3>
<p>
Room temperature should be between 18 and 23&deg;.
<p>
<b>Synthesizer reagent volumes</b>. Control the volume of CapA, CapB, TCA 
Deblock, Oxidizer, DCL Activator, and acetonitrile.  Estimate the volumes to be 
used and the amounts in reagent containers.
<p>
<b>Amidite reagent preparation, freshness/quality</b>.  Amidite bottles should 
be consumed within two (2) weeks after being opened.  Thus the time of synthesis 
and the opening of reagent bottles should be well-coordinated.  The Proligo 
brand of phosphoramidites are solubilized with 20 ml of approved solvent; mix 
well.  Dissolved amidites should be flushed with argon gas, which serves as a 
protectant.  Argon gas is heavier than air, so it should not escape the bottle 
headspace so easily.  A few particles of a proper molecular sieve are then added 
to the dissolved amidite bottle; this acts as a dessicant to remove water as it 
might form in solution from atmospheric vapor.  The presence of water 
particularly in the coupling phase when amidites are mixed with activator can 
reduce oligo synthesis efficiency. If heat is produced during molecular sieve 
addition, this usually indicates that there will be no problem in activation 
(base coupling).  Sieve addition should be about 1 cm high from the bottom of 
the bottle.  After 1 week, more molecular sievw should be added to the amidite 
solution to compensate for additional humidity.
<p>
<b>Activator dryness</b>.  Since the DCI Activator is also quite sensitive to 
water, add molecular sieve (to 1 cm high from bottle bottom) to its bottle too.
<p>
<b>Cleaning sliders</b>. Remove and clean the column, reagent select, and PEEK 
sliders with ethanol (EtOH).  Never use water.
<p>
<b>Software setup</b>.  Open the PolyEasy software on the Windows system. 
Select menu item Oligo&rarr;New.  Complete the name of the oligo to be made, 
enter its 5&prime;&rarr;3&prime; sequence (note that the last base will be the 
CPG base to be used), and turn DMT removal off if isopropanol precipitation is 
to be done.  If cartridge purification is to be done, keep DMT on.
<p>
On the main window of PolyEasy, select the Clear Slider button. Now use the 
Assign control to determine which oligos created in the database will be 
selected for synthesis in the 10 columns of the synthesizer.  Use sort which 
will order the assignments according the oligo length so that slider movements 
are minimized.  Select the type of Main program, Coupling program, Remove DMT, 
and Finish program.  The Start Info panel shows which column should be loaded 
with the CPG-base (where base is A, C, T, G), whether DMT removal is on or off, 
what the oligo sequence is and how it is named.  The names of the oligos in the 
instrument columns are recorded in an experimental record.
<p>
<b>Filling Columns with CPG</b>. On the computer and the machine displays, look 
for the column number and which CPG to load.  Place the bottom frit to the column
first, then fill with CPG.  Use the spoon provided by PolyGen and select the size
and spoon out the amount desired.
<p>
<b>Oiling the Sliders</b>.  Two drops of mineral oil approved by PolyGen should
be used on all sliding surfaces between column, reagent select, and PEEK sliders.
<p>
<b>Priming all tubing lines</b>.  Power on the PolyGen and put it into 
communication with the PolyEasy application on the computer.  Press the 
instrument's <span class="boxed-key">spcl</span> key to select the priming
function.  Except for the Gas, Res, and Flsh lines, priming is done on all
lines.  Priming must be done until fluid is seen emerging from the transparent
plastic tubing that is waste line coming out of the topmost slider.
<p>
During synthesis, press the <b>Get Trityl Data</b> button to open the DMT 
monitor.  Values at or above 70% should be observed.  If values are below this 
water contamination of the activator or amidite bottles should be suspected, and 
synthesis stoppted.  If synthesis is stopped (by pressing <b>Stop</b> key) 
during a <b>Wash</b> step the reagents can be changed or molecular sieve added 
and synthesis can be re-started:  press <b>Start</b> to re-start.  Pressing 
<b>Stop</b> twice will cancel the ongoing synthesis.
<p>
Place ten 0.2 ml (PCR) tubes in the column emptying apparatus after synthesis
completes.  Columns are emptied using the column needle apparatus.  Wash columns
with water, then ethanol.  Wipe PEEK and Reagent Select sliders with ethanol.  
Place sliders on the instrument oiling the sliding parts.
<p>
<b>Ammonium hydroxide</b>.  Keep stored at &minus;20&deg;.  Prepare 15-20 ml bottles
in the fume hood.  Transfer to small bottles in one go using funnel and Erlenmeyer
flask.  Label the bottle.  Every bottle can be used at most six times.
<p>
<b>Decoupling from CPG and Deprotecting Oligos</b>.  With four spoons you can get
1 &micro;mol scale.  Add 200 &micro;l of ammonium hydroxide.  If 2-5 &micro;mol of 
oligo was synthesized, use 500 &micro;l of the 32% ammonium hydroxide.  Leave at
room temperature for 90 min, then spin down for 1 min at 6000 rpm.  Cool down the
tubes at &minus;20&deg; for 3 min before opening.  The oligo solution, frit and CPG
is then transferred to a new 1.5 ml conical.  Add 200 &micro;l ammonium hydroxide to
tube with frit and CPG, and vortex.  Spin down 6000 rpm for 60 s.  Again cool down
the ammonium hydroxide, then open.  Leave for 24 hours at room temperature.
A more rapid protocol incubates at 55&deg; for 6-8 hours;  however, if &gt; 25% of 
bases are G, then incubate at this temp for 24 h.  Another alternative is to decouple
and deprotect in 1000 &micro;l ammonium hydroxide, 50 &micro;l triethylamine at 80&deg;
for 1 hour.
<p>
<b>About pausing (interrupting) and re-starting a syntheis run</b>.  If any 
solution runs low and a reagent must be added while in the course of synthesis, 
the run can be paused using the <b>Stop</b> key but only when the synthesizer is 
in the <b>Wash</b> step.  At that time, the reagent(s) can be changed and the 
run resumed using <b>Start</b>.


<h3>Post-Synthesis Tasks</h3>
<ol>
<li>Perform a slider cleaning with alcohol and oiling as specified in the
Pre-Synthesis Tasks list.
</ol>

<h2>Maintenance, Care, Repair</h2>
<p>
<b>Before and after every synthesis run</b>.  Clean column slider with alcohol
and a no-nap cloth, and oil the slider.
<p>
The intervals for maintenance given below are guidelines.  The maintenance
required will depend on the frequency or constancy of use.
<p>
<b>Weekly Maintenance (or when shutdown for a long period)</b>. The instrument 
has several sliding parts (<q>sliders</q>):  a column slider, a PEEK slider, and 
a Reagent Select slider.  If the instrument is being constantly used, at least 
once per week, clean it with acetone.   Also clean it with acetone if there has 
been a long period where it has not been operated.
<p>
After acetone cleaning, oil the sliders with approved oil (one provided by
PolyGen).
<p>
<b>Every 3 Months</b>.  
The Reagent Select and Column sliders move on spindles that needed to have
the used (black) grease removed and then thinly regreased with approved 
lubricant.

<p>
<b>Every 6 Months</b>.
<ol>
<li>Change or turn by 90&deg; the PEEK slider.
<li>Replace the PTFE (Teflon) 1.8 mm dia tube between the PEEK slider and the
pump inlet port (through the DMT monitor).
</ol>
<p>
<b>Consumables</b>.  It is important to maintain available stocks fo the following
for the PolyGen-10:

<ul>
<li>column top and bottom frits
<li>lubricating oil for the sliders
<li>caps for screw-on bottles
<li>grease for lead screws
<li>2.5 L gas bags
<li>the PEEK slider
<li>the column slider
<li>the reagent select slider
<li>O-rings for bottles
<li>PTFE tubing
<li>acetone (used for slider cleaning)
<li>isopropanol or ethanol (used for slider cleaning)
</ul>


<h2>Instrument Shutdown</h2>
<p>
If the instrument is not to be used for a long period (&gt; 1 week), all 
reagent tubing/lines must be flushed with acetonitrile.  Failure to do so
will cause a gumming up of the sliders and pump head, causing stuck or 
blocked movement and failure to slide.

<h2>Accessories and Included Components</h2>

<table>
<tr><th>Item                             <th>Quantity
<tr><td>forceps (tweezers), straight     <td>1
<tr><td>spoon spatula, size 0 and 1 ends <td>1
<tr><td>solid phase stamp tool           <td>1
<tr><td>ejector                          <td>1
<tr><td>holding block/decanting station  <td>1
<tr><td>power cable                      <td>1
<tr><td>RS232 serial cable with connectors <td>1
<tr><td>illuminating pen lamp            <td>1
<tr><td>lead screw grease                <td>20 g tube
<tr><td>slider oil                       <td>50 ml bottle
<tr><td>frits                            <td>1 pack of 1000 in bottle
<tr><td>active moleculear sieve          <td>1 bottle
<tr><td>2.5 L gas bag                    <td>1
<tr><td>GL45, 2-port screw-cap &frac14;-inch 28-thread bottle <td>1
<tr><td>screw-cap 28-400 bottle          <td>1
<tr><td>screw-cap 20-200 bottle          <td>1
<tr><td>O-rings for screw-cap bottles    <td>1 set (qty?)
<tr><td>fitting/ferrules                 <td>5
<tr><td>PTFE tubing, o.d. 1.6 mm         <td>5 m
<tr><td>PTFE tubing, o.d. 1.8 mm         <td>5 m
</table>

<h2 id="polyeasy">PolyEasy User Manual</h2>
<p>
[This is condensed or summarized version of the PolyGen GmbH user manual for this
Windows software.  Figures/illustrations from the manual are copied and presented
in this web page.]
<p>
When first started, the software presents a main window as shown below:
</div>

<p>
<img class="origsize" src="imagesGenomicsReferenceHTML/PolyEasyMainWindow.png"
  alt="PolyEasy main window interface">

<div class="outerblock">
The main window is organized with a client (sub)window, the following window 
controls are found: <b>Menu</b>.  The usual Windows menu interface, with a menu 
header containing menu commands that appear when the pointer clicks on the menu 
headers.  The menu headers are:
<ul>
<li><b>File</b>
<li><b>Oligos</b>
<li><b>Synthesizer</b>
<li><b>Modifier</b>
<li><b>Configuration</b>
</ul>
<p>
<b>Toolbar</b>.  Below the menu is a <i>toolbar</i> with a set of iconized buttons representing
commonly used commands which are also accessible from the menu interface.
<p>
<b>Oligo Database</b>.  This is a panel (subwindow) organized as a table with
six fields.  The fields are:
<ul>
<li><b>Status</b>
<li><b>Code</b>
<li><b>Sequence</b>
<li><b>DMT</b>
<li><b>Start</b>
<li><b>Length</b>
</ul>

<p>
<b>PolyGen-10 Column Setup</b>.  Another panel (subwindow) organized in a tabular
design with three fields:
<ul>
<li>Col
<li>Code
<li>Sequence
</ul>
<p>
<b>Database-Synthesizer Linking</b>.  A set of four button controls at the 
bottom of the window.

<p>
<b>Synthesizer Setup</b>.  A fieldset with a popup select list of defined 
synthesizers and their defined programs (Main, RemoveDMT, Finish, Coupling).


<h1 id="transgenomic">Transgenomic HPLC Oligonucleotide Purification System</h1>
<p>

This section is intended as a quick guide and summary of 
<a 
href="/UserManuals/Transgenomic%20WAVE%203500%20Operator%20Guide%20(May%202010).pdf">Transgenomic 
WAVE&reg; System 3500 Operator's Manual</a> content.
<p>
Although there are many ways to purify chemically synthesized oligonucleotides 
from a synthesizer, perhaps among the best is an HPLC system for purification. 
Equivalent methods includes gel electrophoresis, but that method likely involves 
more loss during recovery as well as a greater insensitivity.  Capillary 
electrophoresis might be a better system, although it is not well developed or 
routine.

<h2 id="transg-startup">Starting Up The Hardware</h2>
<p>
This must be done as a routine.
<ol>
<li>The HPLC system will probably appear as two stacks. On the right hand stack
where the solvent bottles are on top of the detector are four modules, which
from top to bottom, are the degasser (next to bottles with tubing going in and
out from bottles to pump), the L-7400 UV detector, the L-7100 pump, and the
D-7000 system interface module. On the left hand stack are three modules: the
L-7300 column oven (where the column is), the L-7200 autosampler, and the
L7200ACC chiller. Power up the Hitachi modules on the Transgenomic system in
this order:
 <ol style="list-style-type:lower-roman;">
 <li>D-7000 System Interface Module (located under pump usually)
 <li>L-7100 Pump
 <li>L-7200ACC Chiller
 <li>L-7200 Autosampler
 <li>L-7300 Oven
 <li>L-7400 UV Detector
 <li>L-761 Degasser
 <li>FCW-180 Fragment Collector
 </ol>
<li>After turning on all power switches, wait for the message <b>D-LINE SYSTEM
IS NOT READY</b> on all modules except the L-7300 oven;  for the L-7300, you
should see <b>E5</b>.
<li>Turn on the computer with the WaveMaker software, if it is not already on.
<li>Start the WaveMaker 4.1 software using Start&rarr;Programs or its shortcut
icon on the desktop.
<li>Once the main window appears, press the <b>Initialize</b> button on the
toolbar. It is just above the Results tab on the large subwindow and it looks
like a yellow-colored lightning bolt. After pressing the button, at the lower
left of the main window, the message HSM INITIALIZED appears. The window will
probably switch to the Monitor tab showing an Abort button, and you may see at
the lower left a status message of <b>Downloading</b>. When information from
all the modules is downloaded, WaveMaker shows the Sample Table and the status
message <b>Idle</b>. In addition the Hitachi HPLC modules will no longer say <b>
D-LINE SYSTEM IS NOT READY</b> but show the normal interface in operational
mode.
<li><b>Preheating the column oven</b>.
 <ol style="list-style-type:lower-roman;">
 <li>Press the <b>SET MONIT</b> green-colored button on the L-7300 oven. The
    display should show a small blinking line at the upper left of the button,
    next to SET.
 <li>Press the up or down arrow to alter the number on the display so that it is
    above 40&deg;.  Put it on the set temperature actually.  Press
    <b>SET MONIT</b> again and the line is blinking on <b>BUSY</b>.  The means
    that the module is adjusting the temperature of the oven.  Make sure
    the oven door is closed so that the temperature can adjust and hold.
 <li><b>IMPORTANT</b>. Before starting any mobile phase flow in the system, you
  must set the column oven to 40&deg; or greater to warm the mobile phase.
 </ol>
<li><b>Preparing Mobile Phase Solvents</b>.  Swirl gently the solvent bottles.
If they have white precipitate or flocculent, it may be necessary to discard
the contents and wash the bottle very thoroughly.  Triethyleammonium acetate
(TEAA) at 0.1 M in water (solvent A) can be a growth substrate for bacteria,
and at room temperature has shelf life of 5-7 days.  If prepared solvent will
not be used for several days, it is useful to store the solvent in refrigerated
(2-4&deg;) then to let it come to room temperature before using it.  <b>NOTE</b>:
Whenever any salts or chemicals are added to and dissolved in HPLC-grade
solvents, those salt-containing solvents must be filtered through a 0.45 &micro;m
filtration membrane mounted on a solvent filtration flask; re-pass it through
the filter 2 to 3 times so that any solvent contacting glass containers
(e.g. graduated cylinders) that might have dust are removed in the filter.
If unfiltered mobile phase is used, it can clog the pre-column filter and possibly
damage the very expensive column with permanent effect.
<li><b>Inline solvent filters</b>.  They should be replaced if they have a slimy
film or are discolored.  Make sure the filter is placed at the bottom of the
bottle, since any filter above the fluid level will draw air, make the run
useless, and possibly damage the pump.
<li>Estimate the volume of mobile phase to be used.  Use the time of the method,
the flow rate (usually 0.9 ml/min), and then assume one-half to two-thirds will
be used by solvent A and the balance by solvent B.
<li><b>Priming degasser</b>. When air is in the solvent inlet lines and
whenever solvent is changed, it is necessary to prime the lines with new
solvent and remove air. Priming must be done up to the point of bringing the
solvent in the bottle to the proportioning valves and the pump. Normally you
must take out the solvent line that goes from the degasser (left-hand side) to
the pump, attach a syringe with fitting, and then draw in 2-5 syringe fills,
which replaces the old solvent on the inlet line with new solvent. However, if
the fitting and syringe are not available (were not purchased from
Transgenomics), then you can try priming using the pump and lifting the solvent
bottle to a high level (see next step). You then re-attach the degasser-to-pump
line. Note that the left-hand side ports are outlet ports of the degasser, and
the right-hand side ports are inlet.
<li><b>Pump priming</b>.  Transgenomics says the pump should be purged at the
start of every day.  However, it is probably sufficient to make sure that no
air bubbles (cavitation) has occurred; if so, then perform a purging.
 <ol style="list-style-type:lower-roman;">
 <li>Open the drain valve at the right of the dual pump head.  Turn the valve
 counterclockwise, the handle going from left to right.  Put the end of the long
 waste line into a small flask or beaker.
 <li>Press the <b>MANUAL SET</b> button on the pump keypad.
 <li> All lines can be purged/primed simultaneously.  Since there is no C line,
 and the D bottle goes to the autosampler wash, only the A and B lines need
 priming.  Enter 50% for the %B value (key 5, key 0, then ENTER).  Hit ENTER
 twice if %C and %D are already zero
 <li>The flow rate is 0.9 ml/min.  Set FLOW at 0.9 (key ., key 9, ENTER).
 <li>The pressure has a maximum of 3600 psi and a minimum of 50 psi.  Press
 the appropriate keys to set values and then press ENTER to store the values.
 <li>If you make a mistake, when you press ENTER to the end, you are returned
 out of the manual set state.  Hit MANUAL SET button again, and cycle through
 the values, correcting any problems.
 <li>Now that the manual settings are entered, press PUMP ON/OFF button.  The
 pump starts flowing at 0.9 ml/min.
 <li>Since we want to prime/purge the pump inlet lines, we can increase the flow
 rate to the maximum 9.999 ml/min.  Press PURGE button while the pump is working.
 Watch the inlet lines to the pump to make sure that all the air is removed from
 the lines.  Air bubbles that get into the pump could cause it to lose the ability
 to pump solvent, and it could introduce air into the column, an unwanted
 condition.
 <li>If you continue to see air bubbles coming into the line after a few minutes,
 then the air may be entering the lines through fittings that are loose and
 need tightening, or the solvent inlet filter is up above the solvent liquid
 level.  Re-tighten the fittings (firm hand-tightening should be enough) on the
 degasser and at the proportioning valves.
 <li>Press PURGE button again to toggle off the purge.  Press it again if you
 correct a condition that requires re-purging.
 <li>Turn off the pump by toggling PUMP ON/OFF.  Turn it on again as needed.
 during purging/priming after correcting any conditions.
 <li>Turn the drain valve back to the left (clockwise) to close it.  Stow the
 waste line back in the pump head casing.
 </ol>
<li>Disconnect the tubing at the fitting pre-column frit connected at the front
of the oven.  It is vital to do this if there is an OligoPREP column connected
and the oven has not reached 40&deg;.
<li>Press the PUMP ON/OFF button to start the flow at the set value of 0.9 ml/min.
Check to make sure there is no leaking from the pump mixing outlet next to the
proportioning valves, and at the autosampler injection valve from pump inlet
to the outlet going to the column oven.  After flow from the tubing end is
seen, reconnect the fitting to the precolumn frit while the flow is going.
<li>Turn the pump off using PUMP ON/OFF.  Use the MANUAL SET function on the
pump to set %B to 50%.  Press the PUMP ON/OFF to restore flow.  If a project
is being started and method used for the first time, run two blank (no) injection
methods with the gradient, using detection.  Otherwise, if the method is fully
test, allow the system to equilibrate from 10 to 60 minutes.  A longer equilibration
time would be necessary for any columns coming out of long-term storage, after
any ActiveClean flish, or if solvent bottles were run in the dry state.
<li>Open the column oven and look for any leaks.  If the column oven leak detector
is not calibrated or functional, make it functional.
<li><b>Autosampler priming</b>.  While the column equilibration or test blank
runs are continuing, the autosampler syringe can be primed.  Press the WASH
button and let the wash cycle complete.  Observe any air bubbles.  It may be
necessary to repeat the wash for a total of 3-15 times until all air is removed.
<li><b>Waste Containers</b>.  It is possible that liquid waste will come firstly
from the flow of the column, which goes through the UV detector, which then goes
to the fragment collector valve.  Make sure the waste line goes to its container,
and that its container is not more than half full.  Because it contains MeCN,
dispose of the waste in the proper manner.  Also make sure that a waste container
is located where the autosampler dumps its waste on the lower left side of the
autosampler.
</ol>
<p>
Read the section on preparing the software (projects) in order to create new
methods and handle data.  After completing the steps above, perform the following
in this order:
<ol>
<li>Review instrument configuration and project options.
<li>Set up the methods
<li>Set up the Sample Table
<li>Run the samples
<li>View the results
</ol>

<h2>Shutting Down The System</h2>
<p>
If the system will again be used within 3-4 days:
<ol>
<li>Keep the column flow working at 0.05 to 0.1 ml/min with a 50% solvent
B setting.  At this setting, each bottle will use 40-75 ml per 24 hour period,
so make sure there is enough.
<li>Turn off the UV detector lamp!  Make sure you turn it on when you starting
using the system the next time.
</ol>
<p>
If the system will not be used for a longer period, follow these steps for
long-term storage:
<ol>
<li>If the pump is flowing, press the PUMP ON/OFF button.
<li>Power off all the modules, in reverse order.  Note that if you shut off the
UV detector, you must wait for the UV lamp to cool down completely (at least
1 hour) before re-powering it on.
<li>Turn off the power strips (or unplug them).
<li>Shut down the computer in the normal manner that it must be shut down.
</ol>

<h2>Preparing the Software: Projects</h2>
<p>
When you want to use the Transgenomics HPLC system, it is usually because of
a project you have.  It might be oligo purification in general, or if you are
in the business of doing oligo purification, it might be for one person or
customer.  Organize your work into projects.
<ol>
<li>Go to the menu command <b>File&rarr;New Project</b>.  A window might appear
asking you to save the currently opened project.
<li>Write a descriptive name for the project&mdash;for example, maybe
<q>SerdarOligos</q>&mdash;for a project in which the oligos of Serdar will be
purified.
<li>Save the project under the <b>D:\WaveMaker\</b> path that was created
in order to save all projects.
</ol>

<h3>Methods</h3>
<p>
WaveMaker does not create <b>project methods</b>, which would be methods that
are of general use for a project.  Instead it creates <b>sample methods,</b>
which it calls <q>virtual methods.</q>  That is a method is created for each
sample, and the method is named using a prefix (<b>Method Prefix</b>) whose
name is under the control of the user, and a three-digit number with leading
zeroes and starting with number 1.  Methods are saved not as separate method
files but with the current project.
<p>
A new <q>virtual</q> method is created whenever an Application Type under
the Sample tab is chosen (one of Mutation Detection, ds Single Fragment,
ds Multiple Fragment, Universal Linear, Quantitative RT-PCR, or Oligo
Purification).
<p>
To open an existing method, from <b>File&rarr;Open Method...</b>, navigate
to the folder where the project file (type *.wmp) is located, and the method
file (type *.emt) will be present.   Methods can only be opened if they were
previously saved using the <b>File&rarr;Save Method</b> command.  Methods are
saved by clicking on the sample row for the method to be saved.
<p>
Methods defined/created in the project can be viewed by selecting
<b>View&rarr;Methods</b>. A View Method dialog box opens.  Several parameters
are defined in a method.

<h4>Oligo Purification</h4>
<ol>
<li>On the Sample tab, with the <b>App. Type</b> control selected
for application type, select the <b>Oligo Purification</b> option
<li>Additional options appear with respect to base length range and
the presence of a fluorescence tag or not.  These select the basic gradient
method to be created.
<li>If the gradient is to be modified, click the Gradient control button
to show the gradient steps and view of the gradient.
</ol>

<h3>Sample Table</h3>
<p>
The sample table is a grid that looks much like a spreadsheet.  Each row
is a sample, and the key for the sample is the row number, which cannot
be changed by the user.  The columns (fields) of the table are as follows:
<ul>
<li><b>Vial</b>.  This number is the vial position in the sample tray
that is placed in the autosampler.  The numbering depends on how the
tray itself is defined.  Autosampler position can be changed if the movement
of the needle of the sampler is re-defined.
<li><b>Volume</b>.  The numeric value is the expression in microliters, and
it is set by default when the autosampler was configured.  To change the
default injection volume, go to
<b>Instrument&rarr;Instrument Configuration&rarr;Injector</b> tab and change the
Default Injection Volume field.
<li><b>Injections</b>.  The number of injections to be done with the sample.
Usually this is more than one if analytical rather than preparative methods
are done, to check for reproducibility.
<li><b>Application Type</b>.  Selects one of the standard WaveMaker application
types.  Selection can be used to alter parameters.
<li><b>Sample Name</b>.  Defaults to <q>Sample &lt;row number&gt;</q> unless
the user defines it.
<li><b>Sequence File</b>.  The sequence can be required for some application
types, and in other types, it is not allowed to be used at all, even if
the sequence is known.
<li><b>Oven Temp</b>.  This is often set automatically based on application
type.  However, it can be reset by the user.
<li><b>Base Pairs</b>.  Setting this field can be used to alter the %B necessary
to obtain an elution.  Setting it can alter the gradient profile.
<li><b>Meth. Name</b>.  Generally just an identifer unless the method is
saved to disk; then it is name of stored method.
<li><b>%B</b>.  The initial value of %B in the method.  Check the gradient
to alter any points in the method.
<li><b>Time Shift</b>.  A value in minutes by which a fragment of interest
is to be eluted earlier or later in the gradient.  It offsets gradient according
to (time shift value) &times; gradient slope.  Thus if there is a 2%/min
gradient slope, and entry of 1 min decreases all values in %B column by
2%, which increases retention time by 1 minute.
<li><b>Comment</b>.  Any comments in the Comment field on Run Information
under the Sample tab are shown here.  Comments can also be entered directly
into the field.
<li><b>FCTable</b>.  The fragment collection table assigned to the sample.
Set this in the Fragment Collector tab.
<li><b>Method Summary</b>.  Should be a condensed description of the method.
For example, <tt>BP 200, Mutation, @ 58, SL 2, Normal, DL 5, LD 0.5, GD 4.5,
CL 0.1, EQ 1, TS 0, FL 0.9, 0.0/35, 0.5/35, 2.0/35, 2.1/100, 5.1/100,
5.25/35, 8.2/35</tt>.  The two letter definitions and symbols are:
AU = automatic method, BP = base pairs, @ = temperature, SL = slope,
DL = drop for loading, LD = loading duration, GD = gradient duration,
CL = clean duration, EQ = equalibration duration, TS = time shift, and
FL = flow rate.  Each x.x/yy number pair is a gradient table entry for
time/%B.
</ul>
<p>
Certain button controls are found on the Sample Table tab.  These are as follows:
<ul>
<li><b>Initialize</b>.  Initializes the instrument so that WaveMaker communicates
with the HPLC system.
<li><b>Pumps On</b>.  Turns on the pump.
<li><b>Run Samples</b>  Start analysis of samples in the Sample Table.
<li><b>Pause Table</b>.  Will pause the analysis after the end of the currently
running sample.  Clicking the button again resumes analysis.
<li><b>Abort</b>.  Completely stop (abort) all analyses, including the
currently running sample.  Data for aborted samples is not saved, and sample
is read-only after termination of run.
</ul>
<p>
Injections continue until a blank row is encountered.  After the last method,
pumps remaining running unless the <i>Enable Stanndby</i> option has been
selected.  (Transgenomics does not indicate, but a blank sample with a lower
flow rate can be run to reduce flow rate.)

<p><b>Calculating Maximum Injection Volume for 100 &micro;L Syringe</b>.
When the syringe is 100 &micro;l volume, some calculations must be done.
Note the following constants:
<ul>
<li>Injector Air Samples:  5 &micro;l before + after the sample = 10 &micro;l
<li>Injector Dead Volume:  for L-7200 = 28 &micro;l
</ul>
These are the calculations for maximum allowable injection depending on
injection type:
<ul>
<li><b>ALL</b>:  100 &micro;l &minus; (Air Sample) &minus; (Dead Volume)
&minus; (Feed Volume).
<li><b>CUT</b>:  100 &micro;l &minus; (Air Sample) &minus; (Dead Volume)
&minus; (Lead Volume) &minus; (Rear Volume).
<li><b>LOOP</b>:  100 &micro;l &minus; (Air Sample) &minus; (Dead Volume)
&minus; (Waste Volume)
</ul>

<h3>Acquiring Data</h3>
<p>
It is necessary to set up project options using the <b>Project&rarr;Project
Options</b>.  The dialog box has several tabs, which are described below:
<ul>
<li><b>Instrument</b>
 <ul>
 <li><b>Detector Options</b>:  enable one or both of the Ultraviolet or
 Fluorescence.
 <li><b>Sleep Options</b>: enable the options Pump Off, Lamp Off, and
 Oven Off at the end of a run
 <li><b>Enable Standby</b>:  with a Standby Time setting, this is the amount
 of time system waits after completion before going into standby mode, which
 is 0.05 ml/min at 50% B.  Default standby time is 15 min.
 <li><b>Clean Options</b>:  Can be set to one of three:  Normal, Fast
 Clean, or Active Clean.  The normal option just sets %B at 100 during the
 run.  The Fast Clean uses the Wave Accelerator module (this can bypass)
 the normal route to accelerate cleaning.
 <li>If a line is hooked up to Buffer D, it is run at 100% D, and it
 contains 75% MeCN
 <li>
 </ul>
<li><b>Samples</b>.
 <ul>
 <li><b>Equilibrate Column</b>.  There are several options.  One is to
 equilibrate before 1st injection, before each sample, change temperature,
 and change gradient (less %B), with a time setting (default = 3 min).
 For the 1st injection, and any changes in temperature and %B, enable
 the option, and for 3 min.  The before each sample option should be used
 only if there is difficulty in equilibration.
 <li><b>Run In Temperature Order</b>.  Samples will be run in order of
 increasing temperature despite their order in the Sample Table.  If this
 option is selected, and Run Samples is pressed, all samples are grayed-out
 and cannot be edited.
 </ul>
<li><b>Printing</b>.  These allow options for what should be printed upon
completion.  Options are chromatogram, peak list, confidence report, and
within the confidence report, the system suitability table and module
report.  Also there is the option to print at the end of the sample table
completion or at the end of each run.
<li><b>Gradient</b>.  In the Default Setting control group:
 <ul>
 <li> a Caution Level (Normal, Cautious, Fast) can be set for automatic
 duration settings when WaveMaker does gradient setups. Cautious means
 that durations of the gradient will be increased slightly, while Fast
 means they will be decreased.
 <li><b>Drop %B for Loading</b> is an option to lower the %B by a certain
 value just before the %B level where the gradient begins.  This is to
 allow nonretained components to run ahead of the DNA so they don't
 interfere with chromatogram.  A 5% drop in B is recommended.  Note that
 if a nonspecific peak is found in chromatogram at 2.5-3.5 min, reduce
 the drop for loading to between 0-3% B.
 <li><b>Loading Duration</b>.  This is the time over which Drop for Loading
 increase the %B to the start of linear gradient, and is a value between
 0.1 and 0.5 min.
 <li><b>Gradient Duration</b>.  This is a %B gradient change per unit time
 and is 2% per minute as a default.
 <li><b>Clean Duration</b>.  
</ul>

<!--
<h2>Quick Setup</h2>
<p>
This section is written at the beginning to give experienced operators of the
Transgenomic system a quick guide to setup the system.
<p>
Four system solutions are used in the WAVE HPLC system and should be at least
half full if starting up the system:
<ul>
<li>Buffer A
<li>Buffer B
<li>Syringe Wash
<li>Active Clean Solution
</ul>
-->

<h2>WAVE Nucleic Acid Fragment Analysis System</h2>
<p>
This HPLC system is designed to resolve oligonucleotides at multiple levels.
Its principal goal is to be a <q>nucleic acid fragment analysis system.</q>
In this respect, it can resolve oligonucleotides that differ because they
are polymorphisms, which can be nucleotides with equal length of bases, but
which differ chemically as to base composition and position of that base.
<p>
Transgenomic produces the separation (HPLC) columns that are used for different 
applications.  The stationary phase is a nonporous matrix of 
polystyrene-divinylbenzene copolymer beads with linear C<sub>18</sub> alkyl 
chains attached to the matrix to provide a reversed phase;  the OLIGO SepPrep HC 
cartridge is an exception in that it has a porous beaded matrix.  The matrix
is supposed to not interact with the highly charged (anionic) polynucleotides.
<p>
In reversed phase chromatography, analytes interact more properly with an 
<b>ion-pairing agent</b> which in many cases is trifluoroacetic acid (TFA). But 
this depends on the nature of the analytes.  For consistently and highly anionic 
molecules, a cationic ion-pairing agent is better.  This is why triethylammonium 
acetate [TEAA:  (CH<sub>3</sub>CH<sub>2</sub>)<sub>3</sub>N<sup>+</sup>H 
<sup>&minus;</sup>OOCCH<sub>3</sub>].  The ion-pairing agent helps to reduce
the surface charge density on analytes, creating a more hydrophobic character
to the analyte, and allowing it to interact with the C<sub>18</sub> alkyl chains.
<p>
The mobile phase for this chromatography is binary.  The equilibration system
is 0.1 M TEAA.  To elute the polynucleotides (oligos), a solvent of 0.1 M TEAA
containing 25% acetonitrile (MeCN) is increased in a linear gradient.

<h2>Three Modes of Operation</h2>
<p>
The table below shows in general the three modes of operation and their 
applications.  This table is taken directly from the manual.

<table style="font-size:83%;" class="left">
<tr><th>Operation Mode <th>Temperature  <th>Application <th>Principle of Separation
<tr><td>Non-denaturing <td>~50&deg;   
  <td><ul><li>sizing ds DNA up to 2000 bp
		<li>checking PCR quality; purification of amplicon
   <li>quantitative analysis (qRT-PCR)</ul>
  <td><ul><li>dependence on size <li>sequence independent </ul>
<tr><td>Partially denaturing
 <td>52-75&deg; 
 <td><ul><li>detecting mutation <li>SNP discovery </ul>
 <td>both size and sequence dependence
<tr><td>Fully denaturing
 <td>75-80&deg;
 <td><ul><li>sizing ss DNA (up to 2000 bp)
  <li>RNA analysis (requires RNA Sep cartridge)
  <li>Determining oligo quality:  DNASep and OLIGOSep
  <li>Determining purification analysis: Fragment Collector</ul>
 <td>both size and sequence dependence
</table>
<p>
<b>Non-denaturing</b>.  The column is run at elevated temperature but usually 
below a temperature where double-stranded oligonucleotides (15 bp or more) will 
denature to single-stranded DNA.  The ability to separate on size is based 1% of 
the product size:  thus a 100 bp oligo can be separated from a 101 bp oligo, and 
a 300 bp oligo can be separated from a 303 bp oligo. In the figure below is a 
chromatogram showing the size separation of oligos.  The separation on agarose 
gel shows that oligos of 257 and 267 bp could not be resolved, but can be on 
HPLC.
</div>

<p>
<img class="origsize" src="imagesGenomicsReferenceHTML/WaveOligoSeparationChromatogram.png"
 alt="chromatogram of oligos separated on size">

<div class="outerblock">
This mode of operation can be used to verify PCR reactions.  In a separate 
HPLC run, analyze a ladder of known fragments.
<p>
<b>Partially Denaturing</b>.  This mode of operation is useful when doing a PCR 
analysis in which the amplicons might contain single nucleotide polymorphisms 
but are of equal size.  If the sample is from a heterozygote, the amplicons
should be two different molecules, one wild-type, the other a mutant, and
present quantitatively in equal (1:1) ratios.  If the mixture is denatured
(95&deg;) and then cooled, the amplicons will form homo- and heteroduplexes.
In the case of analyzing for mutants that are homozygous, the DNA from a 
known wild-type must be mixed with the sample from the mutant in order to 
create a mix of hetero- and homoduplexes.  Note in the figure below, the
heating and cooling forms four types of double-stranded molecules, two 
heteroduplex and two homoduplex (the homoduplex molecules represent the
original forms before heat denaturation, which can be re-formed on annealing). 
<p>
<img class="origsize" src="imagesGenomicsReferenceHTML/hetero-and-homoduplex-mixing.png"
  alt="hetero- and homoduplex formation from heating and cooling oligo mixtures">
<p>
Heteroduplex DNA represents base mismatching, and such an oligonucleotide is
less thermally stable compared to homoduplex DNA, where all bases of one strand
properly match the complement strand.  At the proper temperature, a heteroduplex
will be partially denatured while the homoduplex is still fully annealed.
This kind of difference enables the separation of these types of duplexes
on a chromatography system.  In the Transgenomic product line, the DNASep
cartridge is the column of choice.
<p>
In the figure below are overlapped chromatograms with temperature increasing
from 52&deg;, which non-denaturing, up to 59&deg;, which is partially denaturing.
At nondenaturing conditions, all four duplex types have the same retention
time, not being of different size.  At partially denaturing conditions, all
four forms are distinguished, ideally at 56&deg;, which provides the greatest
separation of the four forms.  Note that at 55&deg;, there are three forms:
the first eluting two are the heteroduplexes, whereas the homoduplexes both
remain undenatured and have the same retention time.
<p>
<img class="origsize" src="imagesGenomicsReferenceHTML/MutationAnalysis.png"
 alt="overlapped chromatograms showing mutation analysis">
<p>
Other things to note:
<ul>
<li>Increasing temperature generally leads to faster retention times for any
analyte
<li>Single-stranded DNA elutes earlier than double-stranded DNA, and that is
for partial duplexes too.  In general heteroduplexes, with partial single-strandedness,
elute earlier than homoduplexes.
<li>Duplexes with higher GC content will still be more double-stranded at
higher temperatures, and so will elute later than duplexes with higher AT content.
</ul>

<p>
<b>Fully Denaturing</b>.  These conditions are used to check DNA and RNA of
oligonucleotide size both for differences in sequence and in size.  All strands
are single-stranded (because conditions are fully denaturing).  Applications
include verifying analytically how well oligonucleotides were purified 
preparatively (possibly using Transgenomic system).

<h2>Guidelines for Mutation Detection</h2>
<p>
For creating a research or diagnostic test to find mutations or polymorphisms
in a region of DNA that might be a gene or exons or translated sequence, 
there are a number of considerations to be made in designing the optimal
detection system using HPLC conditions.
<p>
<b>Amplicon Design</b>.  The Transgenomic Navigator software has a utility that 
can analyze a sequence in order to find the optimal amplicon that can be 
separated under HPLC conditions.  The best amplicons will have a single melt 
domain:  the helical fraction will go from 100% to 0% within a 5&deg; interval.
Fragments that multiple melt domains will generally take more than 5&deg; 
to denature.  They might require redesign with GC clamps, or the amplicon might
need to be of smaller size.
<p>
<b>Primer Design and Purity</b>.  After amplicon design is complete, then the 
correct primer pair is designed.  Primers should not be closer than 30-50 bp 
from the sequence portion analyzed for mutations (e.g., an intron/exon 
boundary).  They should be 18-30 bp long.  GC content should be between 45-60%. 
The <i>T</i><sub>m</sub> difference between primer pairs should be &lt; 2&deg;. 
Primers should avoid self-complimentarity (hairpin loops).  Complementarity 
between primers should be avoided (primer dimers).  Primers with runs of 3 or 
more nucleotides of the same base.  Avoid primers with non-templated tails, such 
as universal sequencing primers, M13 or T7 primers.  In addition, primers should 
be pure, with no failed full-length shorter nucleotides contaminating the 
preparation.
<p>
<b>Template Source and Purity</b>.  In general, the template DNA should be 
free of other classes of biomolecules.  It should not contain high amounts of
salts or other low MW materials.  The best preparations are phenol-chloroform
with back extractions using chloroform-isoamyl alcohol.  If spin column with
chaotropic salts are used, good ethanol precipitation should be the last step.
<p>
<b>PCR Conditions</b>. Transgenomic offers its own polymerase (Optimase) which
is high fidelity; in any case, a high fidelity polymerase should be used.
Other factors such as Mg<sup>2+</sup>, presence of inhibitors, annealing temperature,
cycle times and temps, template and primer concentrations require optimization.
<p>
<b>Factors Affecting Separation Cartridge Performance</b>.  The following 
materials or reagents or substances can cause loss of resolution or degradation
of separation cartridges:
<ul>
<li>any unidentified proprietary ingredients that come under the name stabilizers,
enhancers or additives
<li>BSA
<li>autoclaved water
<li>mineral oil
<li>formamide
<li>template DNA not purified using accepted protocols
<li>proteinase K
</ul>
<p>
Also remove from the injection material high molecular weight stabilizers
such as polyethylene glycol, or detergents present at levels &gt; 1% such
as Triton X100, NP40, Tween 20, SDS, and others.  Injection of glycerol &gt; 2%,
DMSO &gt; 10%, or betaine &gt; 1.25 M should not be done.

<h2>WAVE System Modules</h2>
<p>
In using the WAVE system, it is necessary to check that
<ul>
<li>Buffer A
<li>Buffer B
<li>Syringe Wash
<li>Active Clean Solution
</ul>
<p>
are at least half full, capped and that solvent filter are placed at bottom of 
bottle.  Prepare more of each of the solutions.  
<p>
Check the waste bottles.  Dispose of any material with full bottles appropriately.
If you do not want to pour organic waste down the sink drain (a good idea), consider
laying it out in a flat glass dish and letting it evaporate outside the laboratory
under the hot sun and perhaps in a good wind.  This is effective.
<p>
<b>Degasser</b>.  There is a four-port degaasser used for degassing Buffers A and B,
the Syringe Wash Solution, and the Active Clean Solution.
<p>
<b>Interface</b>.  The D-7000 interface communicates between software on computer 
and with the system hardware modules.
<b>Pump</b>.  The WAVE system makes use of three of the four inlet lines (A, B, and
D), with A and B lines being the binary HPLC gradient components.  Make sure that
line C is disconnected from the four-way valve and plug its port.
<p>
<b>Chiller</b>.  Connected to the Autosampler, it maintains temperature between
4&deg; and 60&deg; C.  Note that WAVE OLIGO Systems do not include a chiller.
<p>
<b>Autosampler (with Injection Valve)</b>.  Downstream of the pump mixed solvent
is the Autosampler with inline injection valve.  Samples are placed in 200 &micro;l
PCR conical tubes in a 2 &times; 96 or 2 &times; 384 well plate format.  A PCR
plate can be transferred from thermal cycler to WAVE System constant temperature
rack for analysis.
<p>
<b>Separation Cartridge + Oven</b>.  An in-line filter precedes the column.  This
travels to a pre-heat coil on the oven.  Sample analysis is not done at &lt; 40&deg;
usually.

<h2>Maintenance Procedures</h2>

<h3>Daily</h3>
<p>
The following are <i><b>daily</b></i> maintenance for the HPLC system:
<ul>
<li><b>Oven Temperature must be &ge; 40&deg;</b>.  The buffer must be fully warmed
to at least 40&deg; or the components of the buffer can precipitate and clog the
column.  On the oven, press <b>Set/Monitor</b> button (green colored) and use the
<b>Up</b> arrow key to increase temperature to at least 40&deg;.
<li><b>Buffer</b>.  Make sure solvent filter inlets are submerged (tape down to bottom
of bottle).  Prepare volume of buffer necessary.
<li><b>Degasser Priming</b>.  Generally when a buffer is changed, the degasser should
be primed using syringe connected by a fitting to the outlet part of the degasser.
This is necessary to remove any air in the lines of the solvent inlets as well.
<li><b>Autosampler Priming</b>.  The <b>Wash</b> function key is pressed to cause
the priming of the autosampler.  Press 3-5 times, and observe the autosampler wash
syringe for any air bubbles.  If air bubbles persist, manually move syringe plunger
from starting position then down about 1 cm, then all the way back up to the top.
Repeat this, and use the Wash button three additional times.  Make sure that the
syringe valve fittings are finger tight.
<li><b>Pump Purging and Priming</b>.  The <b>Manual Set</b> function key must be
pressed to access purging.  Set %B and %D to 33%, %C to 0.  Set flow to 0.9 ml/min.
Open the drain valve, turn on the pump and press <b>Purge</b>.  Purge for 1 to 5
minutes until there is no air in the lines.  Press the <b>Purge</b> toggle to stop
pumping, then close the drain valve.  Proceed to equilibrate the cartridge.
<li><b>Equilibrating WAVE System/Separation Cartridge</b>.  Go to <b>Manual Set</b>
(if not already in manual mode).  Set %B to 50% and %D and %C to 0.  Pump the column
at 0.900 ml/min for 15 min.  Run through one or two gradients without sample injection.
<li><b>Waste Container</b>.  Empty the waste container if it is appropriate.
</ul>

<h3>Weekly</h3>
<p>
The following are <b><i>weekly</i></b> maintenance for the HPLC system:
<ul>
<li><b>Oven</b>.  This is tested using two injections of a WAVE Mutation Standard
or Control Sample.  The run should be as expected.
<li><b>Degasser and Buffers</b>.  Generally new buffers A and B should be prepared if
the current ones are more than a week old.
<li><b>Pump and Autosampler</b>.  There is no need for pump or autosampler 
maintenance.
<li><b>Detector</b>.  The lamp energy and wavelength accuracy should be checked.
Press <b>Confidence</b> on the detector keypad, then <b>2</b> and <b>ENTER</b>.
Record the energy level in a maintenance log book.
</ul>

<h3>Monthly and Yearly</h3>
<p>
<b>Every month</b>:
<ul>
<li>The inline filter should be changed, unless 1000 injections occurs within a month.
<li>Solvent inlet filters should be checked for contamination
<li>No pump, detector, or autosampler checks are necessary except the daily.
</ul>
<p>
<b>Every THREE (3) months</b>:
<ul>
<li>Check the oven leak sensor
<li>Change solvent inlet filters
<li>Flush the WAVE system with 50-100% HPLC isopropanol <em>after</em> removing
the inline filter and column and putting PEEK unions in place.
<li>No maintenance on pump, autosampler or detector
</ul>
<p>
<b>Every SIX (6) months</b>:
<ul>
<li><b>Pump</b>.
 <ol>
 <li>Change pump seals.
 <li>Check drain valve seal.
 <li>Check plungers.
 <li>Check inlet and outlet check valves.
 <li>Check the four-way valve packing
 <li>Check, and if necessary change, the flow path filters.
 </ol>
<li><b>Autosampler</b>.
 <ol>
 <li>Check injection valve (rotor) seal and the isolation seal.
 <li>Check injection valve stator.
 <li>Change the injection port seal.
 <li>Check the syringe valve seal.
 <li>Check, and if necessary change, the spring support bearing.
 </ol>
<li>Replace the PEEK tubing with new tubing.
<li>No special maintenance on detector, column oven or degasser or buffers
other than the one and three month needs.
</ul>

<p>
<b>Every year (TWELVE [12] months)</b>:
<ul>
<li><b>Pump</b>.  Perform the usual six month maintenance and in addition:
 <ol>
 <li>Change drain valve seal and packing.
 <li>Check plungers.
 <li>Check inlet and outlet check valves.
 <li>Change four-way valve packing
 <li>Change the flow path filters.
 <li>Check the solenoids on the proportioning valves.
 </ol>
<li><b>Autosampler</b>.  Perform the six-month maintenance and in addition:
 <ol>
 <li>Change injection valve (rotor) seal and the isolation seal.
 <li>Check injection valve stator.
 <li>Change the injection port seal.
 <li>Change the syringe valve seal.
 <li>Check, and if necessary change, the spring support bearing.
 <li>Check diluter valve stator.
 <li>Replace syringe if necessary.
 <li>Check the needle and replace if needed.
 </ol>
<li>Replace the PEEK tubing with new tubing between mixer to autosampler and
from autosampler to oven.  Perform any runs using controls or standards.
<li><b>Detector</b>.
 <ol>
 <li>Check the quartz window.
 <li>Check the mirror.
 <li>Check the flow cells
 <li>Check the lamp.
 </ol>
</ul>
<p>
<b>Quarterly Isopropanol Flush</b>.  Prepare a 25% solution of 2-propanol 
that is HPLC quality. Then do the following:
<ol>
<li>Perform a long-term column storage procedure first.
<li>Remove the column and place its end plugs on.  Replace column with a 
PEEK union.
<li>Combine all solvent inlets into the isopropanol solvent.  Start a 
flow = 0.9 ml/min and make sure flow is divided evenly for all solvent
lines (for A and B bottles, 50% each;  for A, B, and C, 33% each).  Let
run for 30 min. 
<li>While pump is flowing, wash autosampler syringe pressing it 15-20 times
(alternatively, program a multiple wash cycle:
<pre style="margin:0 auto;font-size:83%;">
 UTILITY
   &rarr;  3 (APL PARAM) &rarr; ENTER 
   &rarr;  1 (WASH) &rarr; ENTER
   &rarr; 20 (NEEDLE WASH STROKES) &rarr; ENTER
   &rarr;  5 (NEEDLE WASH SPEED) &rarr; ENTER
   &rarr; 20 (INJ. PORT WASH STROKES) &rarr; ENTER
   &rarr;  5 (INJ. PORT WASH SPEED) &rarr; ENTER
   &rarr;  0 (PUMP PLUNGER WASH) &rarr; ENTER
      To start, press WASH button.
</pre>
<li>Now place all solvent lines in HPLC water, and use syringe to prime
through the degasser.
<li>Purge the system to remove isopropanol from pump lines (3-5 min).
<li>Now program for a flow of 0.9 ml/min and equal flow through all lines
for 30 min.
<li>Place the solvent lines in their standard buffers, purge the solvent
lines and prime the pump as above.  Flush with the desired solvents for
15 min at 0.9 ml/min.
<li>Change the composition to 50% A/50% B for 15 min.
<li>Remove PEEK union and restore column with inlet filter.
</ol>

<p>
<b>Column maintenance</b>.  It is best to leave columns that are not used
for only a few days in a 50 &micro;l/min flow in 50% B in two-solvent system.
If not to be used for more than four days, long term storage is to run
in 75% MeCN at 0.9 ml/min at 50&deg; for 15 min, the remove the column
and end plug it.

<h3>Part Numbers</h3>


<table class="td-left" style="font-size:85%;">

<tr><th style="width:12em;">Part  
   <th style="width:4em;">Number   <th>Comment

<tr><td colspan="3" class="table-inline-header">HPLC Parts
<tr><td>universal PM kit
  <td class="center">SP2018
  <td>contains diluter spring support bearing, UMWHPE filter, drain valve seal,
diluter valve seal, drain valve seal, 4-way valve packing, drain valve filter
packing, rheodyne build kit, injection port seals
<tr><td>solvent inlet filters
  <td class="center">530010
  <td>filter, clear fitting, yellow ferrule (smaller end faces clear fitting,
  larger end faces filter)
  <br>generally changed when slimy in feel or discolored; replaced every 3 months
<tr><td>PEEK fittings + ferrule
 <td class="center">280229
 <td>
<tr><td>PEEK ferruled fitting (10-32)
 <td class="center">280248
 <td>fingertight
<tr><td>PEEK ferrules
 <td class="center">282035
 <td>fingertight
<tr><td>PEEK union
 <td class="center">280230
 <td>black color
<tr><td>PEEK tubing, 5 ft (1.5 m)
 <td class="center">SP5906
 <td>
<tr><td>Polymide PEEK tubing, 7.5 ft (2.3 m)
 <td class="center">SP5917
 <td>
<tr><td>tubing, 0.007-inch i.d. (1.65 mm), PFA
 <td class="center">300190
 <td>yellow color: connects inline mixer outlet to autosampler
<tr><td>tubing, 0.065-inch i.d. (1.65 mm), PFA
 <td class="center">280230
 <td>tubing for detector outlet extension or the degasser

<tr><td colspan="3" class="table-inline-header">Pump
<tr><td>four-way valve packing
  <td class="center">500072
  <td>checked every 6 months, replaced if necessary
<tr><td>plunger
  <td class="center">540086
  <td>watch video for how to dis-assemble pump head,
   checked every 6 months along with pump seal check
<tr><td>plunger guide
  <td class="center">540090
 <td>same as for plunger
<tr><td>pump seal
  <td class="center">542001
  <td>UHMW PE piston seal (order two for complete set)
    <br>check every 6 months after taking apart pump head
<tr><td>drain valve seal
  <td class="center">500067
  <td>Disassemble and check every 6 months
<tr><td>drain valve packing
  <td class="center">540089
  <td>same as for drain valve seal
<tr><td>inlet and outlet check valves
  <td class="center">500096 <br>(outlet) <br> 500098 <br>(inlet)
  <td>check every 12 months
<tr><td>flow path filter nuts and frits
  <td class="center">
  <td>check every 6 months
<tr><td>single solenoid valve
  <td class="center">500064
  <td>if needed
<tr><td>priming syringe
  <td class="center">540091
  <td>plastic syringe and fitting to prime pump

<tr><td colspan="3" class="table-inline-header">Autosampler
<tr><td>injection valve (rotor) seal
  <td class="center">540118
  <td>checked every 6 months and replaced every 12 months
<tr><td>injection valve isolation seal
  <td class="center">540117
  <td>same as for rotor seal
<tr><td>injection port seal
  <td class="center">500068
  <td>changed every six months
<tr><td>syringe valve seal
  <td class="center">500070
  <td>changed every six months
<tr><td>spring support bearing
  <td class="center">450251
  <td>checked every six months
<tr><td>syringe
  <td class="center">540088
  <td>this is for the 0.5 ml kit, its transfer tubing: 300194
 <br>changed (usu. every 12 months) if leaking, corrosion or persistent air
  bubbles are observed; get transfer tubing only if changing syringe size
<tr><td>needle
  <td class="center">450205
  <td><div></div>
<tr><td>needle sleeve
  <td class="center">450207
  <td>

<tr><td colspan="3" class="table-inline-header">Column Oven
<tr><td>inline filter
  <td class="center">530011
  <td>0.5 &micro;m, PEEK, this is the whole assembly (not usually necessary)
<tr><td>filter end fitting
  <td class="center">530016
  <td>this part contains the filter, and is usually the only thing replaced

<tr><td colspan="3" class="table-inline-header">UV Detector
<tr><td>deuterium lamp
  <td class="center">070024
  <td>usually replaced when energy falls below 150  or half its value when
  newly installed

</table>

<h2>Application Notes</h2>
<p>
Single-stranded oligo separation is dependent on ion-pairing agent.  With 
correct pairing agent, stationary phase becomes a dynamic anion exchanger.
The DNASep&reg; column hydrophobic properties are affected by count of 
-CH<sub>2</sub>- groups in alkyl chains of ion-pairing agent.  Short alkyl
chain pairing agents will emphasize the reversed-phase or hydrophobic nature
of the column.  Longer chains de-emphasize the column hydrophobicity and 
the mechanism of separation is more like anion-exchange.  A triethylammonium
pairing agent will reduce hydrophobicity partially and thus separation will
be a function of length (size) and thus anionicity, but also base composition
(bases provide hydrophobicity).  A tetrabutylammonium pairing agent will more
likely reduce the effect of base composition (base hydrophobicity) and so 
oligo separation is based only on anionic character, and thus oligo length
(size).
<p>
Transgenomic has constructed HPLC system and methods for the analysis and 
purification of oligonucleotides, such as those produced synthetically by the 
PolyGen. <a href="http://www.transgenomic.com/lib/an/612003.pdf">Application 
Note 103</a> provides details of the chromatogrpahy for analysis and 
purification. For example, one method shows the size/length-separated oligos 
which are 14, 15, and 16 bases long.  The LC program has total time of 20 min, 
although retention times are 16.36, 17.13, and 17.78 min for the 14-, 15-, and 
16-mers respectively. Note that the 14, 15, and 16 mers have the same base 
sequence except for the 5&prime; bases.  Thus the 14- and 15-mers represent 
<i>n</i> &minus; 2 and <i>n</i> &minus; 1, respectively, failures in synthesis 
and would be capped from extension.  The LC chemistry uses a DNASep&reg; column 
with a binary gradient with A = 25 mM tetrabutylammonium bromide (TBAB) and B = 
25 mM TBAB + 50% MeCN.  The program is from 10&rarr;60% B in 17 min, 
60&rarr;100% B in 1 min, and 100-10% B in 1 min, finishing out. Detection is set 
at 260 nm and column temperature is 60&deg;.  Sample is injected in 5 &micro;l.
<p>
<b>Quality Verification of Commercially Obtained Oligo</b>.  This can be used to 
verify whether an oligo is purified as claimed.  A 15-mer was analyzed using 
DNASep&reg; column with temperature set at 80&deg;, mobile phase A 0.1 M TEAA pH 
7.3 and mobile phase B of A + 25% MeCN with intial 10% B and a gradient of 
10&rarr;60% B over 17 min, 60&rarr;100% B over 1 min, and returning to 10% B 
over 1 min.  Injection is 5 &micro;l and detection at 260 nm.  A retention time
of 6.46 min is observed for the 15 mer under these conditions, and oligo was only
70% pure, with likely shorter length oligos as impurities.

<div id="revealHitachi">
Clicking the button below will show detailed information
about some of the Hitachi components of the Transgenomic WAVE system.   This
information is for advanced troubleshooting and is not really necessary to understanding
the WAVE system.
<p>
<button onclick="showHitachi(true);">Show Detailed Info</button>
</div> <!-- id=revealHitachi -->

<div style="display:none;" id="hitachi-hidden">
<p>
<button onclick="showHitachi(false);">Hide Detailed Info</button>

<h2>System Hitachi Components</h2>
<p>
The HPLC system consists of mostly Hitachi components, which are described 
below.


<h3>Hitachi L-7100 Pump</h3>
<p>
For isocratic systems, this is the Single Liquid Deliver for which the L-7100 
can be set up in two modes:
<ol>
<li><b>Constant flow rate mode</b>.  The pump works to deliver a constant flow rate
of the incompressible liquid.
<li><b>Constant pressure mode</b>.  The pump monitors the back pressure of the 
delivery and adjusts the flow rate up or down to make sure the pressure is not
altered.  This is usually used to protect fragile column packing.
</ol>
<p>
When more the one solvent is involved (multi-nary gradients and mixing), the following
modes are available:
<ol>
<li><b>Low pressure gradient mode</b>.  In this mode, a single pump is operating 
in which multiple solvents are mixed using a proportioning valve system on the 
low pressure end.
<li><b>High pressure gradient mode</b>.  In this mode, two or more pumps each take
in a single solvent and mixing is done on the high pressure side.
</ol>

<h4>Configuration for Different Modes</h4>

<h5>Single Liquid Delivery (Constant flow rate or pressure mode) Configuration</h5>
<p>
<b>Logic cabling</b>. On the back of the L-7100 are <b>D-Line</b> connectors 
that interconnect the pump with other Hitachi component modules, such as the 
D-7500 integrator or the L-7400 UV detector.  The L-7400 is usually stacked on top
of the L-7100, and the D-7500 integrator has its own footprint.
<p>
<b>Plumbing/tubing interconnections</b>.  The pump has a dual head body.  Teflon
tubing enters on the low pressure side from within a solvent bottle with solvent
filter.  Stainless steel tubing connects one pump head to another (0.8 mm dia).
Tubing from the second pump head goes to a shunt valve (which has a drain point
for pump priming) and tubing then goes to the 6-point injection value (into port
2).  Port 3 from the injector connects to 0.25 mm steel tubing to the column.
Ports 1 and 4 connect to the sample loop, and ports 5 and 6 connect to a sample
drain bottle.
<p>
<b>Postcolumn interconnections</b>.  Teflon tubing (0.25 mm dia) connects to the
L-7400 UV detector flow cell.  From the flow cell exit, connect 0.33 mm dia 
Teflon tubing.


<h5>Low Pressure Gradient Mode Configuration</h5>
<p>
<b>Logic Cabling</b>.  D-Line cabling is same as for the Single Liquid delivery modes.
<p>
<b>Proportioning Valve</b>.  Open the right side cover for the proporitioning valve
by loosening four set screws.  The proportioning system has four solenoid valves labeled
<b>A, B, C, D</b> each with logic cables that make connections to the pump system.
Teflon tubing from the outlet side port of the valve then joins to an 4-way joint
that screws into the inlet side of the pump head.
<p>
Beyond the pump head, tubing connections are as for the single-liquid delivery modes.

<h5>High Pressure Gradient Mode Configuration</h5>
<p>
<b>Instead of using proportioning valves to obtain solvent on the low pressure 
end, the high pressure configuration requires multiple pump modules.  The 
Transgenomic system is configured for the low pressure gradient mode, as it only 
comes with one pump module containing the proportioning valve board.</b>

<h5>Connecting Tubing</h5>

<ol>
<li><b>Inlet Tube</b>.  Tighten the solvent inlet tubing connectors only by 
hand.  Place tube with filter at bottom of solvent bottle.
<li><b>Sample loop injection valve</b>.  If there is one (and an autosampler is 
not used), there is a place to mount it on the right front of the pump module.
Two screws are used to mount it.
<li>Connect tubing according to the configuration.  If necessary, prime or
use the pump to fill tubing before completing downstream connections by
final tightening.  In general, after hand tightening a fitting as much as
possible, another full turn of the nut should be sufficient and also avoid
overtightening.  For placing new ferrule, tighten just a quarter turn after
resistance to tightening is observed.  Place system under high pressure to
determine if any ferrules remain untightened.  Never use a ferrule fitting
tightened for one connection on another connection.
<li>When mounting a column, it should always be done while there is a flow in 
the upstream tubing from the pump.  Use a low flow of 0.1-0.25 ml/min of the 
compatible lowest viscosity solvent (in a reversed-phase column, it would be
HPLC-grade MeOH) and while it is flowing, mount the upstream connector to the
column.  If appropriate, ensure there is an upstream guard frit or pre-column!
The pump module is fitted with a column holder (optionally).
<li><b>Detector outlet tubing</b>.  Note that the tubing is set up for the
inlet and outlet of the flow cell.  Ensure that the tubing is never cut:
it is this length to ensure that the transition to low pressure on the outlet
does not lead to cavitation and the entrace of air into the flow cell.
<li><b>Waste bottle</b>.  A waste bottle of appropriate volume should be placed.
At a minimum, it should be 500 ml.
</ol>
<p>
<b>Priming The Pumps</b>. When the pump heads are dry, they may require priming 
using a syringe. The priming is done from the discharge side of the pump head 
(at the top of the pump head).  Draw in the mobile phase by pulling on the 
syringe and creating negative pressure.  If flow continues not to be aspiratable
during priming, open the drain valve to the right of the two pump heads,
connect the syringe (P/N 810-1351) with an adapter (P/N 810-1378), and push
methanol into the pump head.

<h4>Operation &amp; Functions</h4>
<p>
The L-7100 front has a multi-touch key panel with 2-line, 80-char/line
LCD.  The pump head (with suction valve and discharge value) and drain/shunt 
valve are visible when the swing front plastic panel is opened.  Turn on the 
power switch to initialize pump logic control.
<p>
There are 19 keys on the function key panel.  These are:
<ul>
<li><b>Pump On/Off</b>.  A toggle to start pumping at the set flow rate or 
pressure.  If flow rate = 0, pump is put into Ready state.
<li><b>Purge</b>.  Activated only when the drain valve is open for the purpose
of priming the pump with new solvent when wet, or solvent when dry.  A
max pressure of 427 psi (2.9 MPa).
<li><b>Start/Stop</b>.  Starts or stops a pumping program.  The pump must be
in a state where the initial conditions exist for start to occur.  If a program
is started, pressing it again stops the program.
<li><b>Manual Set</b>.  Sets up the pump for manual control, particularly
to monitor parameters such as flow rate, max and min pressure, and any mixing
of the mobile phase.
<li><b>Set Prog</b>.  Used to select, create, edit, or delete a pump program.
If pump is in manual mode, the <b>Init</b> key must be pressed before using this
key.
<li><b>Confidence</b>.  Used to select confidence data.  This indicates as well as
sets the date when pump seal replacement occurs and designates a key lock too.
The key lock can be reset using the <b>Escape</b> key.
<li><b>Utility (Set Up)</b>.   The <b>Utilty</b> part of this dual mode key is
used for several functions:
 <ul>
 <li>compensating for the pressure zero point
 <li>setting the motor control method
 <li>setting the purge flow rate
 <li>assigning the pressure unit:  MPa, psi
 <li>obtaining a program printout
 <li>gradient curve printout
 <li>setting the column flush
 <li>inputting an event code
 <li>setting a link program
 </ul>
In the <b>Set Up</b> part, it is used to select one of three operational modes:
single-liquid delivery, low pressure gradient, high pressure gradient.  Note that
when pressing the power key and <b>Utility (Set Up)</b> keys simultaneously at power
on shows the mode in the display.
<li><b>Escape (Lock Off)</b>.  A dual mode key.  It can be used to return the 
display to monitor mode when numerical input is finished.  In addition it can be
used to reset the key lock function.
<li><b>A, B, C, D</b> keys.  In manual mode, pressing any of the keys sets the
pumping of %A, %B, %C, or %D to 100%.  All keys are available in low pressure gradient
mode, since it engages proportioning valve operation.  With 2- and 3-liquid high
pressure gradient modes, only A and B or A, B, and C are available, respectively.
<li><b>&Delta;</b> and <b>&nabla;</b>.  Keys used to reverse or advance, respectively,
the pump program by one step in edit mode.
<li><b>Delete</b>.  Removes the currently displayed pump program step in edit mode.
<li><b>Enter</b>.  Used to register numeric input during pump operation or programming.
<li><b>CL</b>.  Used to CLear entry of numeric input or an error status (error
message).
<li>(<b>.</b>).  A decimal point indicator.  Note for values &lt; 1, it does not 
require preceding a <q>0</q> for numeric input.
<li><b>Init</b>.  After using <b>Set Prog</b> to select a pump program, this key
puts the program into the initial state, including setting flow/pressure rate
and proportions of mobile phase in order to equilibrate the system.
</ul>
<p>
The pump operates in one of three states
<ol>
<li><b>Initial</b>.  Pressing the <b>Start/Stop</b> toggles between <b>Run</b>
mode and <b>Initial</b>.  Pressing <b>Init</b> takes pump out of <b>Manual</b> state
and put its inot <b>Initial</b>.
<li><b>Manual</b>.  Pressing <b>Manual Set</b> from either Intial or Run state
puts it into Manual state.
<li><b>Run</b>.  Pressing <b>Start/Stop</b> in Initial state puts it into Run
state.  It is not possible to go to Run state directly from Manual state.
</ol>
<p>
When the pump is powered on, it goes through initialization to check for
self-diagnostic actions, as well as D-line cabling requirements to other modules.
Then it shows a monitor display with Time, Percent A/B/C/D, flow rate, and pressure
reading, as well as pump status and numerical display.
<p>
If pump status is <b>[--]</b>, it is stopped in programming mode.  Other statuses
are:
<ul>
<li><b>[GR]</b>. Executing gradient program
<li><b>[IN]</b>. Flowing under initial conditions.
<li><b>[MN]</b>. Manual mode
<li><b>[PG]</b>. Purging in progress.
<li><b>[LK]</b>. Control panel keyboard is locked.
<li><b>[PR]</b>. A program, gradient curve, or logbook is being printed.
<li><b>[FL]</b>. Column flushing occurring.
<li><b>[LC]</b>. Link program conditioning is in progress.
</ul>
<p>
A separate display is shown for pump pressure units and minimum and 
maximum pressure readings when pump is in run mode.
<p>
Another display appears for the status of event contact closures.  If 
showing ON, the contact is closed.  If OFF, contact is open.  Also 
indicated is the pump station number (A, B, etc.).  The value
FLUSH indicates whether a column flush is occurring.
<p>
Another display shows the status of a LINK program.  If one is present, then
ON is shown.  STEP indicates which step in the link program is being executed.
CYCLE indicates which count of a cycle is being executed.  
<p>
Use the <b>&Delta;</b> and <b>&nabla;</b> keys to switch between the different 
displays.

<h4>Single-Liquid Delivery Configuration</h4>

<h5>Programming and Setup</h5>
<p>
The key sequence for setting up single-liquid delivery mode:
<ol>
<li>Power On + <b>Utility (Set Up)</b>.  Shows the Set Up screen with the
single pump designation to cause flow, the Mode, and the D-Line status.
<li><b>Enter</b>.  Allows selection of the pump station number in order to 
set the mode and D-line state of that pump.  Screen shows PUMP with A, B, C
set to numerals.
<li><b>1/2/3</b> + <b>Enter</b>.  If 1 + Enter, sets the state for pump
A, 2 + Enter for Pump B.   Screen shows GR MODE (gradient mode) with 
LOW P. = 1, HIGH P.=2, FLOW=3 for low pressure gradient, high pressure
gradient, or single liquid delivery.
<li><b>3 + Enter</b>.  Pump is now set for single liquid delivery. Screen
now shows D-LINE:  OFF=0, ON=1.
<li><b>1 + Enter</b>.  D-Line is now turned on for coordinated pump control.
Screen now shows SET UP:  PUMP [A/B/C]  MODE L.P.GR  D-LINE ON.  Note that
system think pump is in previous mode.
<li>Power Off
<li>Power On.  After initialization, screen should show TIME 0.0, 
FLOW 0.000   PRESS 0.0 as well as status.
</ol>
<p>
This means system now believes it is in single-liquid delivery mode.
<p>
The next key sequence is for programming the pump in single-liquid delivery mode.
At time = 0, pressure max = 41.2 MPa and pressure mix = 0 MPa, flow rate = 1.0 ml/min.
From 0 to 10 min, the flow rate is increased linearly to 2.0 ml/min.
<ol>
<li>Put the pump into initial state if necessary <b>Init</b>.
<li>Enter the programming state:  <b>Set Prog</b>.  The screen shows 
SET PROGRAM &lt;1-9&gt;  No. _.  The current program number is displayed.
Up to nine programs can be set.
<li><b>1 + Enter</b>.  Sets program creation/editing for program #1.
Screen shown is MODIFY=0, NEW=1, with 0 set by default.  If current program 
is to be edited, selected modify.  If current program is to be deleted for 
a new definition, select new.  In this case, a new program is created.
<li><b>1 + Enter</b>.  Screen nows PRESSURE MAX 41.2  MIN 0.0 if pressure is
in MPa mode.  To set values, use numeric keys with <b>Enter</b> key to set
the pump pressure limits.
<li>Numeric keys + Enter for pressures.  Screen now displays TIME 0.0,
FLOW 0.000  EVENT.  Again numeric keys with <b>Enter</b> key set all the
values.  Time indicates the time in the program that the flow or
event occurs.  Note that the event entry is a two-digit code:  the first
digit is the contact (1 through 4), and the second digit is what will
happen (0: contact open, 1: contact closed, 2: contact closes for a 1
second period).  Using the <b>Enter</b> key enters the numeric or 
current value and moves to the next parameter in a cycle.
<li><b>0 ENTER 1 ENTER 1 2 ENTER 2 2 ENTER 3 2 ENTER 4 2 ENTER</b>. 
This sets time 0 for flow = 1 ml/min and all four event contact to be
closed for 1 second only.  The screen will show TIME  FLOW  EVENT again
for a new step.
<li><b>1 0 ENTER 2 ENTER ENTER ENTER ENTER ENTER</b>.   At time = 10 min,
the flow rate will be 2 ml/min.
<li><b>ESCAPE</b>.  New program saved. Returns to monitor screen.
</ol>
<p>
The pump has a memory for 99 steps for all nine programs.

<h5>Manual Mode</h5>
<p>
This mode can be used to maintain a constant flow of one solvent without the
use of a program.  It is only in this mode can the pump be operated under
constant flow or constant pressure.  The following key sequence is observed:
<ol>
<li><b>Manual Set</b>.  The screen shows MANUAL: FLOW=1, PRESS=2.  Pressing
1 puts in constant flow (by default) or 2 puts in constant pressure.
<li><b>1 + Enter</b>.  Screen shows CONSTANT FLOW:  FLOW MAX MIN.  In this
case, the flow rate must be set along with pump pressure maximum and minimum.
<li>Flow, pressures, + Enter.  Screen returns to pump status monitor mode
with CONSTANT FLOW and the flow and current pressure.
</ol>
<p>
If constant pressure had been selected, only a pressure entry is allowed
from 2-39.2 MPa or 29-5685 psi.
<p>
The pump is not immediately started.  When the monitor screen re-appears, verify
the pump pressure is 0.  Turn on the pump using <b>Pump On/Off</b> key.  Progressively
fill the tubing with solvent before connecting downstream elements.


<h4>Low Pressure Gradient Configuration</h4>

<h5>Pump Mode Setup and Programming</h5>
<ol>
<li>Hold down the <b>Power</b> and <b>Utility (Set Up)</b> keys simultaneously.
In powering on, the pump is placed into set up mode.  The display will 
read <b>SET UP: PUMP A  MODE L.P.R.  D-LINE OFF</b>
<li>Press the <b>Enter</b> key.  The display now shows a pump selection
choice:  <b>PUMP &lt;1-3&gt; PUMP A=1  PUMP B=2 PUMP C=3</b>.  Selection 1
(Pump A) might be already selected.
<li>After selecting a pump, press <b>Enter</b>.  The display now shows
a gradient mode:  <b>GR MODE: &lt;1-3&gt;  LOW P.=1  HIGH P.=2  FLOW=3</b>.
<li>Select option 1 and press Enter. Display now reads <b>D LINE: OFF=0
ON=1</b>.  It might already be selected for being on.
<li>Make a selection and press Enter.  The display now summarizes all the
choices made in the setup.  <b>SET UP:  PUMP A  MODE L.P.GR  D-LINE ON</b>
<li>Press the <b>Power Off</b> key, the press Power On key.  Turning off and
on is necessary to save and exit setup mode.
</ol>
<p>
<b>SLOW and FAST Low Pressure Gradient Types</b>.  Two types of flow rate
ranges are possible, depending on mixer types and configurations, and also
depending upon if a degasser is used.  The table below reports the ranges:
<table>
<tr><th rowspan="2">Configuration <th colspan="2">Flow Rate Range (ml/min)
<tr><th>SLOW <th>FAST
<tr><td colspan="3" 
style="font:bold 80% Arial,sans-serif;background-color:#ffffe0;">With Degasser
<tr><td>no mixer     <td>not recommended  <td>0.1 - 0.3
<tr><td>static mixer <td>0.1 - 2.0  <td>0.3 - 4.0
<tr><td>dynamic mixer<td>1.0 - 4.0 <td>4.0 to (7.0)*
<tr><td>static + dynamic mixer <td>1.0 - 4.0 <td>4.0 - (9.999)*
<tr><td colspan="3" 
style="font:bold 80% Arial,sans-serif;background-color:#ffffe0;">Without Degasser
<tr><td>no mixer     <td>not recommended  <td>not recommended
<tr><td>static mixer <td>0.1 - 2.0  <td>not recommended
<tr><td>dynamic mixer<td>1.0 - 4.0 <td>not recommended
<tr><td>static + dynamic mixer <td>1.0 - 4.0 <td>4.0 - 9.999
<tr><td colspan="3" 
style="font:italic 80% Palatino,sans-serif;background-color:#ffffe0;"> * 
When L-7610 degasser is used, flow can be up to 5.0 ml/min
</table>
<p>
The static mixer is made available in a single pump with low pressure
gradient proportioning valve system (Transgenomic configuration).
<p>
To set up the pump mode correctly, go to <b>Utility (Set Up)</b>.  
From <b>Utility</b> menu, select <b>Cond=1</b> (Condition Set), 
from <b>Condition Set</b> menu, then select <b>Type=3</b>, and from
<b>L. P. Gradient Type</b> choose either <b>SLOW=1</b> or <b>FAST=2</b>
depending on the range and mixer configuration and presence of degasser.
<p>
<b>Programming</b>.  For manual programming, press <b>Set Prog</b> button
(if in manual mode, must press <b>Init</b> first).  Select a program number
(1-9).  Indicate whether it is <b>Modify=0</b> or <b>New=1</b>:  if new,
any current steps are deleted.  Enter the max and min pressure for the run,
using the Enter key for each value.  All subsequent entries are based on 
what happens at a time point in the run.  Enter time values, the values
of solvent percents (ratios), the flow rate, and any event values.   When 
done, use Escape key to quit programming mode.
<p>
<b>Manual (Non Gradient) Mode</b>.  Use this for isocratic conditions and
no change in flow rate.  Press the <b>Manual Set</b> key and then set the
percent solvents, flow rate, pressure max and min.  To leave manual mode,
power off the pump and then turn on again.

<h4>Utility</h4>
<p>
This should be used to perform various functions important for pump operation,
as necessary.
<p>
<b>Zero Adjustment Reading of Pump</b>.  The zero point might need to be set
after some period of time.  Before doing this, open the pump drain valve. 
Enter <b>Utility (Set Up)</b>, select <b>Cond=1</b>, then select 
<b>Z-Adj=1</b> from Condition Set, and then select <b>Yes=1</b> to set the
zero value.
<p>
<b>Pump Control Method</b>.  Pump flow rate can be variable (STANDARD) or
set at a constant rate (FIX).  As for the zero-adjustment above, enter 
Utilty and Condition Set menus, select <b>M.Control=2</b> from Condition
Set.  The next menu allows setting of <b>Standard=1</b> or <b>Fix=2</b>.
If Fix is selected, then either Fix Step or Fix Auto can be chosen.
If Fix Step is chosen, then the user can select values of 0 or between
30-300;  this is the Fix Step Area which becomes invariable regardless of
pressure variation.  For Fix Auto, both a Slope value (between 0 and 200)
and a Origin Step (between 0 and 100) are chosen.

<h4>Performance Checking</h4>
<p>
Obtain the following for checking the pump operation/performance:
<ul>
<li>500 ml distilled water
<li>500 ml HPLC MeOH
<li>Syringe with 5.1 cm blunt-end needle and 0.71 mm thickness (supplied?)
<li>50-100 ml beaker for waste collection
<li>~500 ml waste bottle
<li>C<sub>18</sub> columm, 4 &times; 150 mm
<li>500-1000 ml bottle for HPLC mobile phase solvent
</ul>
</div> <!-- id=hitachi-hidden -->


<h1>Bibliography</h1>
<ul>
<li>Buck GA, Wenner RJ, Reynolds TR (1992) Strategies for Purification of 
Synthetic Oligonucleotides in DNA Core Laboratories. ABRF News, June 1992.

<li><a href="http://www.purebiotechllc.com/cpgp/index.html">About Controlled 
Pore Glass</a>.  Pure Biotech LLC content.

</ul>


</div> <!-- class=outerblock -->

</body>
</html>
