<!DOCTYPE html>
<html lang="en">
<head>
<title>Real-Time PCR and Its Equipment</title>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8">
<link href="./stdcss/std.css" rel="stylesheet">
<style>
  img {border:none;}
</style>
</head>
<body>

<!-- hostsite top logo here -->

<!-- logo here -->

<p id="title">
Real-Time PCR and Its Equipment


<div class="outerblock">
This is information about real-time PCR technology:  principles, applications,
technology, industry equipment (instruments / systems).
<p>
This is an Abiyotek internal document.  Do not share with others outside of
Abiyotek.

<!-- putTOChere -->

<hr>

<h1>Goals</h1>
<p>
Supply all consumables necessary in a kit to conduct qPCR for the following
targets:
<ul>
<li>HBsAg
<li>HBV
<li>HBC
<li>HIV
<li>Tuberculosis
<li>CMV
<li>HPV
</ul>

<h1>Principles</h1>
<p>
Important principles, terms, and definitions are addressed here. 

<h2>Standard PCR</h2>
<p>
The purpose of PCR is to take a small amount of DNA and make a large number of
copies of that DNA (usually a fragment of it).  PCR requires the following
components in the reaction tube:

<ul>
<li>template DNA:  as little as one copy of the DNA strand can be in the tube
<li>a set (two) of sense (forward) and anti-sense (reverse) primers:  usually
present at a concentration of 0.1 to 1.0 &micro;M (&micro;mol/L)
<li>the deoxyribose nucleotide triphosphases (dNTPs), namely dATP, dCTP, dGTP,
and dTTP:  usually present at 200 &micro;M (0.2 mM)
<li>magnesium chloride (MgCl<sub>2</sub>):  present between 1.0 to 2.5 mM, and
should be optimized, but most use 1.5 mM if they are in a hurry
<li>a thermostable DNA polymerase, usually from <i>Thermophilus aquaticus</i>
(<i>Taq</i>) but can be from other thermostable organisms
<li>a buffer for the enzyme, optimized for it, usually a Tris-saline solution
at pH 7.0-7.5:  supplied by the enzyme maker as  10U&times; concentrate generally
</ul>
<p>
Most PCR tubes are of a conical shape, with a standard conical shape
<h2>Reverse Transcription Polymerase Chain Reaction (RT-PCR)</h2>
<p>


<h2>Quantitative PCR</h2>
<p>
Quantitative PCR is any PCR-based method that can provide in its result an
estimation of the amount of DNA in the (starting/original) sample, usually as
an integer value representing the number of copies of the genome (genomic
equivalents).

<h2>Real Time PCR (qPCR, Q-PCR)</h2>
<p>
[A word about <dfn>RT-PCR</dfn>:  the abbreviation <dfn>RT-PCR</dfn> has a
confusing usage, because it can be used as shorthand for <b>reverse-transcription
polymerase chain reaction</b> as well as <b>real-time polymerase chain reaction</b>,
and those two terms refer to different things.  In fact, you can combine
reverse transcription with a real-time PCR system....what do you call it then?
(RT)<sup>2</sup>-PCR?  The convention is to shorthand real-time PCR as qPCR
or Q-PCR, but you should understand that not all quantitative PCR needs to be
real-time.]
<p>
The term <q>real time PCR</q> really means a PCR reaction that you can follow
as it is happening.  It can be quantitative and usually is.  Many assays in which
a reaction can be followed over time come in two types:
<ol>
<li>End-point assays:  these assays measure the reaction only at the end of the
reaction, usually when something has been done to stop the reaction (change
in temperature, perhaps the addition of a <q>stopping</q> reagent).
<li>Kinetic assays: these assays are designed to measure the extent of the
reaction <q>continuously</q> (while it is happening), or at least at meaningful
time period intervals (every 15 or 30 or 60 or 120 seconds).  A good signal vs
time plot
</ol>
<p>
Note that the term <dfn>assay</dfn> is any kind of test or reaction which
produces a value representing quantity or amount of something.  So both
qPCR and real-time PCR.
<p>
Clearly kinetic assays would be preferable, since they include the concept of
an end-point signal value in them.  Moreover, kinetic assays can reveal if the
reaction is occurring properly or according to expectations, and continuous
measurement can help to troubleshoot problems with sample complexity, poor
method configuration, improvements in instrument or container design and
engineering.  Instruments capable of kinetic measurements would likely be much
more difficult to design and configure, and thus the hardware and software is
probably more expensive.

<p>
<b>TaqMan</b>.  The TaqMan PCR was developed by Applied Biosystems in which
a fluorescent probe that cannot be extended by DNA polymerases hybridizes to
its complement if it is present and a 5&prime;&rarr;3&prime; exonuclease
degrades the double-stranded formation, releasing a fluorophore.  When the
fluorophore is still part of oligo, its fluorescence is quenched by another
label on the oligo which stops the fluorescence.  This sort of assay can be
multiplexed with several targets and use of filters that have excitation
and emission differences.  Phosphorylation of the 3&prime; end can inhibit
extension.

<p>
<b>Light Upon eXtension (LUX)</b>.  This system by Invitrogen makes use of a
hairpinned primer in which the 3&prime; end has the fluorophore and the 5&prime;
end has the quencher.  As hybridization occurs with unfolding during amplification
cycle

<h2>Multiplex PCR</h2>
<p>
The term <q>multiplex</q> refers to the ability to examine/determine/assay
more than one analyte in a single (the same) (PCR) reaction tube.  This has
numerous benefits, including rapid time to multiple target/analyte determination,
lower consumable cost, maximum information from minimal sample, and most
important, reduces technical variation that might require more controls to
test for differences in reaction conditions.
<p>
<b>GMO Testing</b>. An example is given by Bio-Rad to demonstrate its iCycler
iQ real-time PCR system. Roundup-Reday soya (RRS) in GMO soya was tested using
a soya lectin as endogenous control and the CP4SPSPS gene specific for RRS. An
oligo probe with FAM and Black Hole Quencher 1 was constructed along with
specific primers. Lectin was also to be detected with a HEX/Black Hole Quencher
2 dual-labeled probe. DNA prepared from soya bean power standard (Fluka) with
2% GMO (2% RRS in 100% soya) was used to make dilutions for standard curve: 0.
02 to 2.0 ng RRS and 1 to 100 ng soya. Also DNA was extracted from 0.1% and 1.
0% RRS soya bean powder standards (Fluka), and used as control (references).
Percentages were calculated based on the threshold cycle values (C<sub>T</sub>),
 and RRS DNA normalized to amount of lectin DNA to determine % RRS/soya in
unknowns and references. Standard curves were determined with four replicates
per dilution to make the curve. Amplification efficiences can be determined
from the linear fit of the standard curve. Unknowns were amplified along with
references in triplicate.
<p>
<b>Gene Expression Analysis</b>.  Human prostate and thymus tissue were tested
for expression of two enzymes ornithine decarboxylase (ODC) and S-adenosylmethionine
decarboxylase (AdoMetDC), and compared to a reference gene &beta;-actin.  These
enzymes are rate-limiting in polyamine biosynthesis.  Generally the prostate
produces higher levels of polyamines compared to thymus.  RNA was extracted from
these tissues and 1 mM oligo dT<sub>12-18</sub> was used with 1.0 &micro; RNA
in a 20 &micro;l 1st strand synthesis with SuperScript II reverse transcriptase
(Invitrogen).  A 5-fold dilution series was generated from the cDNA synthesis,
and relative quantitation was based on &Delta;&Delta;C<sub>T</sub> method with
equivalent of 0.08 &micro;l of the cDNA synthesis product.  PCR was in 25 or 50
&micro;l:  20 mM TrisHCl pH 8.4, 50 mM KCl, 0.2 mM dNTPs 0.5 &micro;M primers,
0.2 &micro;M oligo probes, 3.5 mM MgCl<sub>2</sub>.  PCR efficiency validation
was done with dilution series duplicate reactions.  Four replicates were used
for the relative quantitation (&Delta;&Delta;C<sub>T</sub>) method.  Polymerase
was 0.625 U for singleplex, and 1.875 for triplex reactions.  PCR is 3 min 95&deg;,
45 cycles fo 57&deg; for 30 sec (anneal+extension), and 95&deg; 10s for denaturation.
Use of &Delta;&Delta;C<sub>t</sub> method requires multiplexed genes to be
amplified with approx same efficiency, and the efficiency should not be significantly
different between singleplex and triplex systems.  At first each target is
checked as a singleplex so that starting amounts of cDNA, slope of the curve,
PCR efficiency and correlation coefficient are determined.  In most cases
efficiencies are &gt; 90% with well-designed PCR.

<h2>Optimization</h2>
<p>
Optimization has many goals. One might be to reduce time of assay. Another
might be material cost. Another might be to improve sensitivity or dynamic
range of assay. Steps to combine annealing and extension steps with no
temperature change can reduce total time. Improving design of primers and probe
can affect sensitivity and time of assay. Better master mixes reduce variation.
Thin-wall containers with high heat coefficients improve time and accuracy.
Optimal targets will have amplicon &lt; 150 bp while maintaining specificity
(range might be from 75-300 bp). Polymerases operating at lower temperature
(down to 60&deg;) can improve work. Primers should anneal at a target <i>T<sub>
m</sub></i> = 60&deg;; modified base use in primer can be used to fine-tune the
<i>T</i><sub>m</sub>.  For short amplicon size, a 15 s 95&deg; is actually
excessive:  half that is possible.


<h2>Fluorophores and Quenchers</h2>
<p>

<table>
<caption>Fluorophores</caption>
<tr><th>
<tr><td>SYBR Green
<tr><td>FAM
<tr><td>VIC
<tr><td>JOE
<tr><td>NED
<tr><td>TAMRA
<tr><td>TET
<tr><td>Texas Red
<tr><td>ROX
 <br>5- or 6-carboxy-X-rhodamine, succinimidyl ester, FW = 631.6836
<tr><td>Cy3
<tr><td>Cy5
</table>
<p>
Structure for the fluorophore TAMRA family is shown below.  Typical members are
5-TAMRA [CAS 91809-66-4] and 6-TAMRA [CAS 150322-06-8].
<p style="text-align:center;">
 <img src="imagesRealTimePcrHTML/TAMRAstructure.png" 
   alt="general structure of TAMRA molecules">
<br><img src="imagesRealTimePcrHTML/TAMRA-table.png" 
    alt="table of substituents for TAMRA general structure">

<p>
N-(1-Naphthyl)ethylenediamine dihydrochloride [CAS 1465-25-4]
<p>
The ideal quencher is an molecule that can absorb light without emitting or
producing any fluorescence itself.
<p>
The BHQ dyes are true dark quenchers in that they produce not native emission
contributing to a high background signal.  They have a polyaromatic-azo backbone.
Their general structure is below:
<p style="text-align:center;">
 <img src="imagesRealTimePcrHTML/BHQstructure.png" style="width:80%;"
   alt="general structure of BHQ molecules">
<br><img src="imagesRealTimePcrHTML/BHQ-table.png" style="width:35%;"
    alt="table of substituents for BHQ general structure">
</div> <!-- class=outerblock -->

<table style="margin:1em -20%;">
<caption>Quenchers</caption>
<tr><th>Name  <th>Absorption Max <th>Absorption Range <th>Comments
<tr><td>DABCYL                        <td> <td>           <td>signal:noise reduced
          <img src="imagesRealTimePcrHTML/DABCYL.png" 
            alt="structural formula of DABCYL">
<tr><td>TAMRA                         <td> <td>           <td>has residual background signal
<tr><td>Black Hole Quencher 0 (BHQ-0) <td>493 <td>430-520 nm <td>
<tr><td>Black Hole Quencher 1 (BHQ-1) <td>534 <td>480-580 nm <td>
<tr><td>Black Hole Quencher 2 (BHQ-2) <td>579 <td>560-670 nm <td>
<tr><td>Black Hole Quencher 3 (BHQ-3) <td>672 <td>620-730 nm <td><img 
      src="imagesRealTimePcrHTML/BHQ3.png" alt="structural formula of BHQ3"
      style="width:100%;">
</table>

<div class="outerblock">

<!-- ===================================================================
                 Real-Time PCR Kits
======================================================================== -->

<h1>Real-Time PCR Kits</h1>

<h2>AmpliSense&reg; Real-Time PCR Kits</h2>
<p>
Information from <a
href="http://www.pcrdiagnostics.eu/en/Products/TORCH-infections.alej">the
TORCH products</a> at the <a href="http://www.pcrdiagnostics.eu/en/Ecoli.alej">PCR
diagnostics.eu web site</a>.  The entire catalog (2012 products) is also in
a <a href="http://www.pcrdiagnostics.eu/files/documents/catalogue.pdf">PDF</a>.

<p>
Produced by Russian Central Research Institute for Epidemiology.
Kits have the CE-IVD marking.
<p>
Kits have two styles of packaging:
<ul>
<li>Pre-mixed PCR tube with wax cover:  just add sample and diluent
<li>Vials of stock reagents (Taq-F, primers, probes, buffers)
</ul>
<p>
Kits are compatible with following fluorescent real-time detection technology:
<ul>
<li>Bio-Rad iCycler IQ/IQ 5
<li>Stratagene Mx3000P, 3005P
<li>QiaGen Rotor-Gene Q
<li>Corbett Research Rotor-Gene 3000/6000
</ul>
<p>
Kit systems are:
<ul>
<li><i>Toxoplasma gondii</i>, 400 copies/ml sensitivity
<li>Parvovirus B19, 800 to 10<sup>7</sup> copies/ml sensitivity
<li>Rubella, 400 copies/ml sensitivity
<li>Cytomegalovirus, 1000 copies/ml (urogenital swabs), 2000 copies/ml (urine)
<li>EBV, 500 to 10<sup>7</sup> copies/ml
<li>Multiplex:  EBV/CMV/HHV6 screen, 400 copies/ml, 5 copies/10,000 cells
<li>Multiplex: HSV/CMV multiprime, 1000 copies/ml swabs, 2000 copies/ml urine
<li>HIV provirus RT-PCR + extraction
<li>HIV Monitor quantitative
<li>HIV RNA
<li>HBV
<li>HBV Monitor
<li>Genoscreen HLA B*5701 MHC allele
<li>HCV
<li>HCV genotypes 1, 2, 3
<li>HCV Monitor quantitative, + extraction kit form
<li>Pneumocystis jirovecii (carinii)
<li>HDV Monitor quantitative
<li>HDV
<li>HBV/HCV/HIV multiplex
<li>HAV
<li>HBV/HDV multiplex
<li>High risk HPV (types 16, 18, 31, 33, 35, 39, 45, 51, 52, 56, 58, and 59)
screen titer (quantitative), 2 color (FAM + HEX/JOE) and 4-color
</ul>

<!-- ===================================================================
                 qPCR Instruments/Systems
======================================================================== -->


<h1>Real-Time PCR Systems / Instruments / Thermal Cyclers</h1>

<h2>(Life Technologies) Applied Biosystems</h2>
<a
href="http://www.invitrogen.com/site/us/en/home/Products-and-Services/Applications/PCR/real-time-pcr/real-time-pcr-instruments.html"><img
src="./LifeTech-ABi-logo.png" style="float:right;" alt="Applied Biosystems logo"></a>
<p>
The list below was taken from <a
href="http://www.invitrogen.com/site/us/en/home/Products-and-Services/Applications/PCR/real-time-pcr/real-time-pcr-instruments.html">Life
Tech/Invitrogen web site</a>.  There is also a <a
href="http://tools.invitrogen.com/content/sfs/brochures/Real-Time-PCR-Instruments-Brochure.pdf">PDF
brochure</a>.
<p>
All systems have sensitivity down to 1 copy of the template DNA.
<ul>

<li><b>StepOne&trade; / StepOnePlus&trade;</b>: 48 or 96 reaxns per run, 3-4
color combinations: filters are FAM/SYBR Green, VIC/JOE, NED/TAMRA, ROX
(photodiode detection). Fast as 40 min runs times. 1-40&micro;l programmable
(10-40 &micro;l validated) volumes. 9 orders of magnitude dynamic range.
Applications: gene expression, genotyping, copy number variation, HRM, protein
thermal shift, protein detection, microRNA (not for IVD)

<li><b>7500</b>: up to 96 reaxns per run, 5 color combinations: filters are
FAM/SYBR Green, VIC/JOE, NED/TAMRA, Cy3/ROX/Texas Red, Cy5 (CCD detection). 2 h 
run times. 1-100&micro;l programmable (20-100 &micro;l validated) volumes. 9 
orders of magnitude dynamic range. Applications: gene expression, genotyping, 
copy number variation, protein thermal shift, protein detection, microRNA (not 
for IVD)

<li><b>7500 Fast</b>: up to 96 reaxns (96 fast format) per run, 5 color
combinations: filters are FAM/SYBR Green, VIC/JOE, NED/TAMRA, Cy3/ROX/Texas Red,
 Cy5 (CCD detection). As fast as 30 min run times. 1-30&micro;l programmable 
(5-30 &micro; l validated) volumes. 9 orders of magnitude dynamic range. 
Applications: gene expression, genotyping, copy number variation, HRM, protein 
thermal shift, protein detection, microRNA (IVD available for certain countries)

<li><b>ViiA 7</b>: up to 384 reaxns (96, 96 fast, 384, TaqMan array formats)
per run, 21 color combinations: 6 excitation filters from 450-670 nm, 6 
emission filters from 500-720 nm (CCD detection). As fast as 30 min run times.
1-200&micro;l programmable (1-100 &micro; l validated) volumes. 9 orders of
magnitude dynamic range. Applications: gene expression, genotyping, copy number
variation, HRM, protein thermal shift, protein detection, microRNA (IVD
available for certain countries), OptiFlex with halogen

<li><b>QuantStudio 12K Flex</b>: up to 12,000 reaxns (96 fast format) per run,
21 color combinations: 6 excitation filters from 450-670 nm, 6 emission filters
from 500-720 nm (CCD detection). As fast as 30 min run times. 1-200&micro;l
programmable (33 nl-100 &micro; l validated) volumes. 9 orders of magnitude
dynamic range. Applications: gene expression, genotyping, copy number variation,
 HRM, protein thermal shift, protein detection, microRNA, digital PCR (not for
IVD), OptiFlex with White LED excitation

</ul>


<h2>Bio-Rad</h2>
<a
href="http://www.bio-rad.com/evportal/en/US/LSR/Category/059db09c-88a4-44ad-99f8-78635d8d54db/Real-Time-PCR-Detection-Systems"><img
src="./BioRad-logo.png" style="float:right;" alt="Bio-Rad logo"></a>
<p>
The list below was taken from <a
href="http://www.bio-rad.com/evportal/en/US/LSR/Category/059db09c-88a4-44ad-99f8-78635d8d54db/Real-Time-PCR-Detection-Systems">Bio-Rad
web site</a>.
<p>
Bio-Rad features the following real-time PCR systems:

<ul>

<li><b>MiniOpticon</b>: 48 reactions, with 10-100 &micro;l sample volume (20
&micro;l usual), detects up to 2 targets, excitation: 470-500 nm, emission
523-543 nm (channel 1) and 540-700 nm (channel 2), 35-99&deg; gradient range
</ul>

<p>
<b>iCycler iQ</b>.  System capable of 4-color multiplex PCR


<h2>ESCO</h2>
<img src="./Esco-logo.png" style="float:right;"
 alt="ESCO logo">
<p>
An Asian manufacturer who makes the <b>Swift&trade; Spectrum 48 and 96 Real
Time Thermal Cyclers</b>.
<p>
The <a href="http://www.escoglobal.com/product.php?id=SPT-RT">48-well thermal
cycler</a> has 4 color reading at excitation = 470, 523, 543, and 571, and
emission at 525, 564, 584, and 612 for FAM, SYBR Green 1, HEX, VIC, TAMRA,
Texas Red, and ROX. Block temperature uniformity is &plusmn; 0.3&deg; C. Sample
volumes are 10-100 &micro;l for 200 &micro;l capacity, and sensitivity is from
10 to 10<sup>10</sup> copies.
<p>
The <a href="http://www.escoglobal.com/product.php?id=SPT-RT-96">96-well
thermal cycler</a> has 6-8 channles color reading at excitation = 470, 523, 543,
628, and 678 nm, and the emission at 525, 564, 584, 612, 692, and 718 nm for
FAM, SYBR Green 1, Cy3, 5-TAMRA, NED, Cy5, HEX, VIC, TAMRA, TET, JOE, Texas Red,
ROX, Light Cycler Red Quasar 705. Block temperature uniformity is &plusmn;
0.3&deg; C. Sample volumes are 5-100 &micro;l for 200 &micro;l capacity, and
sensitivity is from 10 to 10<sup>10</sup> copies.

<!-- ===================================================================
                 DNA Purification
======================================================================== -->

<h1>Fully Automated DNA Purifiers</h1>
<p>
Fully automated DNA purifiers have several advantages to the clinical (medical
technology / diagnostics) industry:  hospitals and private laboratories.

<ul>
<li>They are high throughput, processing thousands of samples rapidly
<li>They avoid the mistakes
<li>They can process (have methods for processing) most biofluid samples: blood,
 urine, CSF, saliva

</ul>

<h2><i>fluor</i>ion Mag 16</h2>

<ul>
<li>uses an extraction system based on cellulose-coated magnetic beads.
<li>processes between 1-16 samples
<li>accepts sample volumes of 200, 400, or 1200 &micro;l
<li>final volumes of 60, 100, 150, or 200 &micro;l
<li>in 200 &micro;l volume will have 6 &micro;g DNA
<li>has a heating block and UV sterilization
</ul>

<h2>iPrep Instrument / Consumables</h2>
<p>
A Life Technologies product.  Purification is an ion-exchange system that does
not use guanidine, EtOH or silica systems.  All buffers are aqueous and do
not inhibit PCR, qPCR, real time PCR, STR, sequencing, and WGA.  Processes
12 samples and 1 control per run, with the ability to load 16 runs (total
192 samples) in a conveyor, complete in 8-hour workday.  Reagent cartridges
and tips are loaded first, then the samples, and a protocol selected.


Consumable products come in the form of the following kits:
<ul>
<li>iPrep PureLink Virus kit:  for viral RNA and DNA and bacterial gDNA.
Uses 200-400 &micro;l serum or plasma with final product in 20 &micro;l volume.
<li>iPrep ChargeSwitch gDNA Tissue kit:  uses special ionizable nucleic acid-binding
ligand whose charge state changes with pH.  Needs 10 mg of powder of tissue ground in
liquid nitrogen, or can use 10 mg of frozen tissue to prepare lysate.
<li>iPrep&trade; PureLink&trade; gDNA Blood kit: uses uncentrifuged blood sample
and extracts from fresh or frozen or buffy coat cells using Dynabeads MyOne
SILANE magnetic particles (monodisperse 1 &micro;m particles, 2.4 mg per
reaction).  350 &micro;l whole blood or 300 &micro;l buffy coat purified to
final volume of 15 &micro;g (whole blood) or 48 &micro;g (buffy coat) in final
100, 150, or 200 &micro;l volume, and DNA at least 40 kb in size.
<li>iPrep&trade; PureLink&reg; Total RNA and TRIzol Plus RNA kits:  for the
Total RNA kit, processes animal (up to 10 mg), mammalian cells 
(10<sup>6</sup>), fresh whole blood (1.0 ml) and bacterial cells (10<sup>9</sup>).  
For TRIzol+ RNA kit, animal (up to 50 mg), mammalian cells (10 million), 
plant tissue (100 mg),  and bacterial cells (up to 5 &times; 10<sup>9</sup>).  
Total RNA only kit to include DNase I.  Plant tissues should be liquid nitrogen powder.
<li>iPrep&trade; ChargeSwitch&reg; Forensic and Buccal Cell kits: the Forensic
kit will process dried blood on paper or clothing 1-5 paper punch circles, root 
end hair follicles, cigarette butts (8 mm from butt end), semen, and <q>touch 
DNA</q> on tools, mobile phones, etc. The Buccal Cell kit will get 1-3 &micro;g 
gDNA from swabs. Isolated using magnetic particles having a ChargeSwitch 
ionizing ligand.
</ul>



</div> <!-- class="outerblock" -->

<!-- hostsite bottom logo here -->
</body>
</html>
