<!DOCTYPE html>
<html lang="en">
<head>
<META HTTP-EQUIV="Content-Type" CONTENT="text/html; charset=utf-8">
<title>Tyrosine Hydroxylase</title>
<link href="../stdcss/std.css" rel="stylesheet">
<link href="../stdcss/biblio.css" rel="stylesheet">
<style>
  td { font:normal 83% Helvetica,Arial,sans-serif;}
  th { font:bold 83% Helvetica,Arial,sans-serif;}
  .refs {font-size:75%;}
   #headernotes { text-indent:0;font:bold 90% Helvetica,Arial,sans-serif; }
   #headernotes p { margin:1em 0; }
</style>
</head>
<body>

<p id="title">
Facts and Data on
<br>Tyrosine Hydroxylase

<h1>Structure</h1>

<table class="bibliofacts">
<col>
<col width="30%">
<tr><th>    <th>Ref.
<tr><td>Tyrosine hydroxylase reported as initial step in norepinephrine
     synthesis
  <td class="refs">
     Nagatsu, T., Levitt, M., Udenfriend, S. (1964) <i>J. Biol. Chem.</i>
      <b>239</b>, 2910-2917.
</table>
<p style="text-align:center;">
   <img src="THstruct.gif"
    alt="Schematic of tyrosine hydroxylase holoenzyme with ligand binding sites and four serines in the N-terminal regulatory domain">
<p>
A crystal structure<sup>1</sup> at 2.3 &Aring; has been characterized which 
reveals that the Fe atom is surrounded by a basket of alpha helical strands. 
Also a 40 &Aring; long anti-parallel coiled coil forms the core of the enzyme. 
The catalytic Fe atom is held 10 &Aring; below the surface in a 17 &Aring; deep 
well of the active site, held there by two His and one Glu (His-331, His-336, 
and Glu-376).  More interesting is the effect that point mutations in TH and 
phenylalanine hydroxylase would have on the function of these enzymes and how 
they might appear in diseased individuals.

<p>
1.Goodwill KE, Sabatier C, Marks C, Raag R, Fitzpatrick PF, Stevens RC.
 (1997) <i>Nat. Struct. Biol.</i> <b>4</b>,578-85.
 <span style="font-variant:small-caps;">
 Crystal structure of tyrosine hydroxylase at 2.3 A and its implications for
 inherited neurodegenerative diseases.
 </span>


<h2>Sequences</h2>

<pre>
Tyrosine hydroxylase [L-tyrosine,tetrahydrobiopterin:oxygen
oxidoreductase (3-hydroxylating), EC 1.14.16.2]
Rat brain
(from Grima et al.)

498 amino acids
        1|0       2|0       3|0       4|0       5|0
         |         |         |         |         |
MPTPSAP<b>S</b>PQPKGFRRAV<b>S</b>EQDAKQAEAVT<b>S</b>PRFIGRRQ<b>S</b>LIEDARKERE   50
AAAAAAAAAVASSEPGNPLEAVVFEERDGNAVLNLLFSLRGTKPSSLSRA   100
VKVFETFEAKIHHLETRPAQRPLAGSPHLEYFVRFEVPSGDLAALLSSVR   150
RVSDDVRSAREDKVPWFPRKVSELDKCHHLVTKFDPKLDLDHPGFSDQVY   200
RQRRKLIAEIAFQYKHGEPIPHVEYTAEEIATWKEVYVTLKGLYATHACR   250
EHLEGFQLLERYCGYREDSIPQLEDVSRFLKERTGFQLRPVAGLLSARDF   300
LASLAFRVFQCTQYIRHASSPMHSPEPDCCHELLGHVPMLADRTFAQFSQ   350
DIGLASLGASDEEIEKLSTVYWFTVEFGLCKQNGELKAYGAGLLSSYGEL   400
LHSLSEEPEVRAFDPDTAAVQPYQDQTYQPVYFVSESFNDAKDKLRNYAS   450
RIQRPFSVKFDPYTLAIDVLDSPHTIQRSLEGVQDELHTLAHALSAIS     498
         |         |         |         |         |
        1|0       2|0       3|0       4|0       5|0


Tyrosine hydroxylase [L-tyrosine,tetrahydrobiopterin:oxygen
oxidoreductase (3-hydroxylating), EC 1.14.16.2]
Bovine adrenal medulla (rat probe used)
(from Saadat, S., Stehle, A. D., Lamouroux, A., Mallet, J.,
Thoenen, H. (1988)   J. Neurochem. 51, 572-578)

491 amino acids
        1|0       2|0       3|0       4|0       5|0
         |         |         |         |         |
MPTPNAA<b>S</b>PQAKGFRRAV<b>S</b>ELDAKQAEAIM<b>S</b>PRFVGRRQ<b>S</b>LIQDARKERE    50
KAEAAASSSESAEAgslvERDGkAVLTLLFALrPTKPPALTRAIKVFETF   100
EAHLHHLETRPAQPLRAGSPPLE<b>C</b>FVR<b>C</b>EVPGPVVPALLSALRRVAEDVR   150
AAGESKVLWFPRKVSELDK<b>C</b>HHLVTKFDPDLDLDHPGFSDQAYRQRRKLI   200
AEIAFQYKQGDPIPHVEYTAEETATWKEVYSTLRGLYPTHA<b>C</b>REHLEAFE   250
LLERF<b>C</b>GYREDRIPQLEDVSRFLKERTGFQLRPvAGLLSARDFLASLAFR   300
VFQ<b>C</b>TQYIRHASSPMHSPEPd<b>CC</b>HELLaHgPMLADRTFAQFSQDIGLASL   350
GVSDEEIEKLSTLYWFTVEFGL<b>C</b>KQNGEVnAYGAGLLSSYGELLHSLSEE   400
PEIRAFDPDAAAVQPYQDQTYQPVYFVSESFSDAKDKLRSYASRIQRPFS   450
VKFDPYTLAIDVLDSPHAIRrALDGVQDEMQALAHALNAIS            491
         |         |         |         |         |
        1|0       2|0       3|0       4|0       5|0

(Compare lowercase of this sequence to D&acute;Mello et al. sequence)

Tyrosine hydroxylase [L-tyrosine,tetrahydrobiopterin:oxygen
oxidoreductase (3-hydroxylating), EC 1.14.16.2]
Bovine adrenal medulla (rat probe used)
(from D&acute;Mello, S.R., Weisberg, E.P., Stachowiak, M.K., Turzai, L.M.,
Gioio, A.E., Kaplan, B.B. (1988)  J. Neurosci. Res. 19, 440-449

491 amino acids
        1|0       2|0       3|0       4|0       5|0
         |         |         |         |         |
MPTPNAA<b>S</b>PQAKGFRRAV<b>S</b>ELDAKQAEAIM<b>S</b>PRFVGRRQ<b>S</b>LIQDARKERE    50
KAEAAASSSESAEAaawlERDGeAVLTLLFALpPTKPPALTRAIKVFETF   100
EAHLHHLETRPAQPLRAGSPPLE<b>C</b>FVR<b>C</b>EVPGPVVPALLSALRRVAEDVR   150
AAGESKVLWFPRKVSELDKGHHLVTKFDPDLDLDHPGFSDQAYRQRRKLI   200
AEIAFQYKQGDPIPHVEYTAEETATWKEVYSTLRGLYPTHA<b>C</b>REHLEAFE   250
LLERF<b>C</b>GYREDRIPQLEDVSRFLKERTGFQLRPaAGLLSARDFLASLAFR   300
VFQ<b>C</b>TQYLRHASSPMHSPEPe<b>CC</b>HELLgHvPMLADRTFAQFSQDIGLASL   350
GVSDEEIELLSTLYWFTVEFGL<b>C</b>KQNGEVkAYGAGLLSSYGELLHSLSEE   400
PEIRAFDPDAAAVQPYQDQTYQPVYFVSESFSDAKDKLRSYASRIQRPFS   450
VKFDPYTLAIDVLDSPHAIRhALDGVQDEMQALAHALNAIS            491
         |         |         |         |         |
        1|0       2|0       3|0       4|0       5|0
</pre>

<h1>Rat and Bovine TH Amino Acid Sequences Aligned</h1>
<p>
(not necessarily optimally or by computational algorithm)

<p>Rat is from Grima <i>et al</i>. and bovine from Saadat <i>et al</i>.
<br>Lowercase letters in sequence indicate differences reported
between Saadat et al. and D&#146;Mello et al. bovine sequences.

<pre>
MPTPSAP<b>S</b>PQ PKGFRRAV<b>S</b>E QDAKQAEAVT <b>S</b>PRFIGRRQ<b>S</b> LIEDARKERE AAAAAAAAAV   <b> 60</b>
MPTPNAA<b>S</b>PQ AKGFRRAV<b>S</b>E LDAKQAEAIM <b>S</b>PRFVGRRQ<b>S</b> LIQDARKERE KAEAAA----

ASSEPGNPLE AVVFEERDGN AVLNLLFSLR GTKPSSLSRA VKVFETFEAK IHHLETRPAQ   <b>120</b>
SSSESA---E AglsvERDGk AVLTLLFALr PTKPPALTRA IKVFETFEAH LHHLETRPAQ

RPLAGSPHLE YFVRFEVPSG DLAALLSSVR RVSDDVRSAR EDKVPWFPRK VSELDKCHHL   <b>180</b>
PLRAGSPPLE CFVRCEVPGP VVPALLSALR RVAEDVRAAG ESKVLWFPRK VSELDKGHHL

VTKFDPKLDL DHPGFSDQVY RQRRKLIAEI AFQYKHGEPI PHVEYTAEEI ATWKEVYVTL   <b>240</b>
VTKFDPDLPL PHPGFSDQAY RQRRKLIAEI AFQYKQGDPI PHVEYTAEET ATWKEVYSTL

KGLYATHACR EHLEGFQLLE RYCGYREDSI PQLEDVSRFL KERTGFQLRP VAGLLSARDF   <b>300</b>
RGLYPTHACR EHLEAFELLE RFCGYREDRI PQLEDVSRFL KERTGFQLRP vAGLLSARDF

LASLAFRVFQ CTQYIRHASS PMHSPEPDCC HELLGHVPML ADRTFAQFSQ DIGLASLGAS   <b>360</b>
LASLAFRVFQ CTQYLRHASS PMHSPEPdCC HELLaHgPML ADRTFAQFSQ DIGLASLGVS

DEEIEKLSTV YWFTVEFGLC KQNGELKAYG AGLLSSYGEL LHSLSEEPEV RAFDPDTAAV   <b>420</b>
DEEIELLSTL YWFTVEFGLC KQNGEVnAYG AGLLSSYGEL LHSLSEEPEI RAFDPDAAAV

QPYQDQTYQP VYFVSESFND AKDKLRNYAS RIQRPFSVKF DPYTLAIDVL DSPHTIQRSL   <b>480</b>
QPYQDQTYQP VYFVSESFSD AKDKLRSYAS RIQRPFSVKF DPYTLAIDVL DSPHAIRrAL

EGVQDELHTL AHALSAIS
DGVQDEMQAL AHALNAIS
</pre>

<p style="text-align:center;">
   <img SRC="hTHseq.gif"
alt="comparison of selected regulatory sequences of
human tyrosine hydroxylase types 1, 2, and 4">

<hr>
<h1>Activity Determination</h1>

<h2>Vulliet, Langan, Weiner <i>PNAS</i> (1980)</h2>
<br>Based on method of Waymire et al. (1971) <i>Anal. Biochem.</i> <b>43,</b>
588.
<p>Final assay mixture
<ul>
<li>200 mM Na acetate, pH 6.2
<li>250 &mu;g partially pure aromatic acid
<li>40 mM 2-mercaptoethanol decarboxylase
<li>1 mM FeSO<sub>4</sub>
<li>1 mM 6-methyltetrahydrobiopterin
<li>100 &mu;M L-[1-<sup>14</sup>C]tyrosine
</ul>
Total vol=0.2 mL.  Assay time=15 min.  Assay temp=37 &deg;C
<br>1 U=1 nmol DOPA/min at 37 &deg;C

<h2>Greene, Seeley, Rukenstein, DiPiazza, Howard (1984) <i>J. Neurochem.</i></h2>

<ul>
<li>100 &mu;M L-[3,5-<sup>3</sup>H]tyrosine (Sigma) (0.2 &mu;Ci total, 53 Ci/mmol, NEN)
<li>200 mM Tris-Acetate, pH 6.1
<li>1 mM D,L-6-methyl-5,6,7,8-tetrahydropteridine
<li>800 U catalase
</ul>
Labeled tyrosine is purified by ion-exchange chromatography to reduce blank
(see Nagatsu et al. (1964) <i>Anal. Biochem.</i> <b>9</b>, 122 and
Markey et al.  (1980) <i>Mol. Pharmacol. </i><b>17</b>, 79.)

<p>
50 &mu;L of this substrate solution is mixed with 50 &mu;L of enzyme
on ice, incubated 10 min at 37 C, stopped with 200 &mu;L 0.1 M NaOH.
Loaded onto mixed-bed ion-exchange column, washed with water.
Tritiated water formed is collected in eluate, counted with
hydrofluor (Natl Diagnostics).

<h2>Vrana, Roskoski Jr. (1983) <i>J. Neurochem.</i></h2>
Based on Vrana et al. (1981) <i>J. Neurochem.</i> <b>36</b>, 92.
<p>
TH measured in 125 &mu;M 6-MPH4 and 100 &mu;M [<sup>14</sup>C]tyrosine
in presence of aromatic acid decarboxylase and sheep liver
dihydropteridine reductase along with co-factors. 37 &deg;C for 15 min.
Dopamine product is separated on paper electrophoresis, then counted.

<h2>Kuhn, Arthur Jr., Yoon, Sankaran (1990) <i>JBC</i></h2>
Based on method of Lerner et al. (1978) <i>Neurochem. Res.</i> <b>3,</b> 641
using tetrahydrobiopterin as the pterin co-factor. Sucrose is added to
TH reaction mixture to 0.3 M for intact chromaffin granule assay.

<h2>Vulliet, Hall, Mitchell, Hardie <i>JBC</i> (1989)</h2>

CO<sub>2</sub>-trapping procedure used with coupled de-carboxylation
of DOPA using L-[1-<sup>14</sup>C]tyrosine
(see Waymire et al. (1971) <i>Anal. Biochem.</i> <b>43</b>, 588.)
<p>
Primary hydroxylation conditions:

<ul>
<li>0.1 mM L-[1-<sup>14</sup>C]tyrosine
<li>1.0 mM 6-methyl-5,6,7,8-tetrahydropterin
<li>2 mg/mL catalase
<li>60% (v/v) sample in 120 mM Tris-acetate, pH 6.2
</ul>
37 C for 10 min

<p>Secondary decarboxylation conditions:
<ul>
<li>0.6 mg/ml 3-iodo-L-tyrosine
<li>10.3 &mu;g/ml PLP (pyridoxal 5-phosphate)
<li>10% (v/v) partially purified DOPA decarboxylase
<li>150 mM Tris-acetate, pH 8.0
</ul>
37 &deg;C for 30 min
<p>
Stopped by acidification with 10% TCA; incubate 1 hr at 37 &deg;C
<br>Radiolabeled CO<sub>2</sub> collected on Protosol-soaked filter paper.

<h2>Meligeni, Haycock, Bennett, Waymire <i>JBC</i> (1982)</h2>
Assay performed using the Zivokovic et al. (1974) <i>Mol. Pharmacol.</i>
<b>25</b> 91 modification of the method of Waymire et al. (1971)
<i>Anal. Biochem.</i> <b>43</b>, 588.
<br>
Total vol=100 &mu;L, containing 10 nmol of L-[1-<sup>14</sup>C]tyrosine and
100 nmol of 6MPH<sub>4</sub> and final pH of 6.2 at 37 C.

<h2>Vulliet, Woodgett, Cohen <i>JBC</i> (1984)</h2>
Modification of Waymire et al. (1971) assay performed.
<ul>
<li>200 mM sodium acetate, pH 6.2
<li>1 mM ferrous sulfate
<li>40 mM 2-mercaptoethanol
<li>0.2 mM MPH<sub>4</sub>
<li>100 &mu;M L-[1-<sup>14</sup>C]tyrosine (58.5 Ci/mol)
</ul>

final volume=200 &mu;L. Assay done for 5 min at 37 &deg;C, terminated by
addition of 0.2 &mu;mol of 3-iodo-L-tyrosine in 10 &mu;L of
1 M NaP<sub>i</sub>, pH 8.0. Porcine kidney decarboxylase (50 &mu;g)
was added, then incubation for 30 min at 37 C.  Addition of 50 &mu;L
of 10% (w/v) TCA to release CO2 was done, which was trapped in
30 &mu;L.  Protosol soaked in 1 &times; 3 cm filter paper suspended
above the mixture.

<h2>Reinhard Jr., J.F., Smith, G.K., Nichol, C.A. (1986)
<i>Life Sci.</i> <b>39</b>, 2185-2189.</h2>

A working solution of 500 &mu;M [3,5-<sup>3</sup>H]-L-tyrosine
(0.4 Ci/mmol) is prepared from the commercial stock product
(usu. 30-50 Ci/mmol).  A small volume is taken from the commercial
solution and dried under a stream of nitrogen or argon and then
reconstituted using unlabeled L-tyrosine.  This working solution
is discarded.  Commercial solutions are discarded after 4-6
months or re-purified using established procedures
(see Petrack et al. (1968) <i>J. Biol. Chem.</i> <b>243</b>, 743).

<p>
In a 12 &times; 75 mm test tube is placed:
<ul>
<li>5 &mu;mol MES, pH 6.10
<li>2.5 nmol tyrosine HCl
<li>180 &mu;G catalase (Sigma C100)
<li>0.5 &mu;mol DTT
<li>50 nmol 6(<i>R</i>)-L-erythro-5,6,7,8-tetrahydropterin (BH<sub>4</sub>)
</ul>
BH<sub>4</sub> is added last to start the reaction.
Total volume of the reaction should be 100 &mu;L.
Blanks are obtained by omitting the addition of BH<sub>4</sub>.
Reactions proceed for 20 min at 37&deg;.

<p>
The following concentrations and stocks might be used:

<table style="margin:1em auto;">
<col span="2">
<col style="text-align:center;">
<tr>
 <th>Final concentration<br>
      (components listed in order of addition)
 <th>Concentrated Stock
 <th>Volume of conc'd stock to use<sup>*</sup>
<tr>
 <td>50   mM  MES <br>
      5   mM  DTT
 <td> 0.5  M  MES <br>
     50   mM  DTT
 <td>10 &mu;l
<tr>
 <td>50 &mu;M [3,5-<sup>3</sup>H]L-tyrosine<sup>*</sup>
 <td> 1   mM  Tyr
 <td> 5 &mu;L
<tr>
 <td>enzyme and/or water
 <td>
 <td>0-65 &mu;L
<tr>
 <td>10 &mu;M ferrous ammonium sulfate
 <td>0.2   mM FeAmSO<sub>4</sub>
 <td>5 &mu;L
<tr>
 <td>500 &mu;M 6-MPH<sub>4</sub>
 <td>  5 mM cofactor
 <td>10  &mu;l
<tr>
 <td>1.8 &mu;g/&mu;l catalase
 <td>36 &mu;g/&mu;l
 <td>5 &mu;l
<tr>
 <td>
 <td align="right">final volume =
 <td>100 &mu;l
</table>

<sup>*</sup>order of addition of components should be MES/DTT, water,
tyrosine.  Just prior to beginning reaction (within 2 min), subsequent
addition of catalase, enzyme, FeAmSO<sub>4</sub>, and then cofactor
added last, if at all.

<p>
<sup>**</sup>count the number of tubes to receive tyrosine.
Commercial stock is ~1 &mu;Ci/&mu;l, and since each tube to receive
1 &mu;Ci, # &mu;l of commercial stock=# tubes.  Blow off liquid with
N<sub>2</sub> till dry.  To dry tube, now add unlabeled
1 mM tyrosine in a volume = 5 &mu;l * # tubes.  You are ready.

<p>
Reactions are terminated by adding 1.0 mL of stirred
7.5% (w/v) Darco G-60 charcoal (J.T. Baker) in 1.0 M HCl.
After rapid vortexing for 1-2 seconds and centrifugation
at 300-500 <i>x g</i> for 10 min, aliquots of clear supernatant
are placed in 10 mL Aquasol II (NEN).  Additionally two tubes
containing 100 &mu;L of 3H<sub>2</sub>O (100,000 dpm in each) are treated
with charcoal and worked up in the same manner.  The fraction
of tritiated water recovered is assessed (usually 94%) and
is used to adjust the final count.  The CPM can be converted
to PMOL/MIN by using the following equation:

<table style="margin:1em auto;" style="text-align:center;">
<tr><td style="border-bottom:1px solid black;">
   CPM<sub>sample</sub> &minus; CPM<sub>blank</sub>
<tr><td>(CPM added/2 &times; pmol tyrosine added) (time)
  (fraction <sup>3</sup>H<sub>2</sub>O)
</table>

The optimum of amount of charcoal used for the assay was
assessed at 1, 2.5, 5, 7.5, and 10% (w/v). According
to the results, a 7.5% value gave the greatest % of triated
water recovered while keep the triated tyrosine recovery low.
The volume of charcoal used was also varied and 1.0 mL deemed
the best. Different charcoal grades were examined: alkaline
Norit A was not satisfactory, but G-60 charcoal from Fisher
and RIA grade Norit A from Becton-Dickinson were adequate.
Assay linearity (r<sup>2</sup>=0.9929) was compared
to the method using Dowex cation column recovery, with
results being linear up to 30 min. The blanks as a % of added
substrate were compared to the other methods and found to be
0.1% of the total tyrosine added. The separation method
is comparable.

<hr width="50%" align="left">
Bostwick, J. R., Le, W.-D. (1991) <i>Anal. Biochem.</i> <b>192,</b>
125-130. <span style="font-variant:small-caps;">
A tyrosine hydroxylase assay in microwells using coupled 
nonenzymatic decarboxylation of DOPA</span>
<p>
Winder, A. J., Harris, H. (1991) <i>Eur. J. Biochem.</i> <b>198,</b> 
317-326. <span style="font-variant:small-caps;">
New assays for the tyrosine hydroxylase and dopa oxidase
activities of tyrosinase</span>

<h1>Preparation of TH Ser-31 specific kinases from chromaffin cells</h1>
Before beginning the procedure, prepare the following:
<ol>
<li>KRH buffer, warmed to 37&deg;
<li>KRH-KCl buffer, also warmed
<li>Chromaffin cell extraction buffer
<li>DEAE-cellulose equilibration and elution buffers
<li>Mono Q buffers A and B (loading and high salt eluting)
</ol>

Cultured chromaffin cells are shaken loose from the flask and the medium is 
removed by centrifugation. Cells are resuspended in KRH buffer warmed to 
37&deg;.

<table style="margin:1em auto;">
<caption>KRH buffer</caption>
<tr>
 <th>Component
 <th>To make 100 mL
<tr>
 <td>125 mM NaCl
 <td>6.25 mL of 2 M or <b>0.7305</b> G of NaCl
<tr>
 <td>4.8 mM KCl
 <td>0.48 mL of 1 M or <b>0.0358</b> g of KCl
<tr>
 <td>2.6 mM CaCl<sub>2</sub>
 <td>1.3 mL of 0.2 M CaCl<sub>2</sub> or <b>0.0263</b> g of CaCl<sub>2</sub>
   2 H<sub>2</sub>O
<tr>
 <td>1.2 mM MgSO<sub>4</sub>
 <td>0.24 mL of 0.5 M MgSO<sub>4</sub> or <b>0.0296</b> g MgSO<sub>4</sub> 7H<sub>2</sub>O
<tr>
 <td>25 mM HEPES NaOH, pH 7.4
 <td>2.5 mL of 1.0 M HEPES/NaOH or <b>0.5958</b> G HEPES
<tr>
 <td>5.6 mM D-glucose
 <td>0.56 mL of 1.0 M or <b>0.1009</b> g glucose
</table>

<p>
Cells are incubated at 37&deg; for at least 1 hr,
but 1.5 to 2 hr may be better insofar as removing
the cells from the effects of "growth" conditions.
(This technique was done by J. Mitchell for his PC12 work,
so that the cells were in a basal state free from the
effects of growth factors present in the serum supplements
of the media.)  The cells are rotated gently in the KRH
using a roller apparatus for roller bottles.

<p>
Cells are then centrifuged and then resuspended in
KRH-KCl warmed to 37&deg;.  After rapid and complete
resuspension, the cells are incubated for a period of
between 2.5 to 3 min, as determined from previous work.

<table style="margin:1em auto;">
<caption>KRH KCl</caption>
<tr>
 <th>Component
 <th>To make 100 mL:
<tr>
 <td>74 mM NaCl
 <td>3.7 mL of 2 M or <b>0.4313</b> g NaCl
<tr>
 <td>56 mM KCl
 <td>5.6 mL of 1 M or <b>0.4175</b> g KCl
<tr>
 <td>2.6 mM CaCl<sub>2</sub>
 <td>1.3 mL of 0.2 M CaCl<sub>2</sub> or <b>0.0263</b> g of CaCl<sub>2</sub> 2H<sub>2</sub>O
<tr>
 <td>1.2 mM MgSO<sub>4</sub>
 <td>0.24 mL of 0.5 M MgSO<sub>4</sub> or <b>0.0296</b> g MgSO<sub>4</sub> 7H<sub>2</sub>O
<tr>
 <td>25 mM HEPES NaOH, pH 7.4
 <td>2.5 mL of 1.0 M HEPES/NaOH or <b>0.5958</b> g HEPES
<tr>
 <td>5.6 mM D-glucose
 <td>0.56 mL of 1.0 M or <b>0.1009</b> g glucose
</table>

<p>
Cells are then immediately transferred into 2 volumes of
ice-cold 1.5-strength extraction buffer and placed on ice.
The cells are processed in a Dounce homogenizer (a dozen
strokes) and centrifuged for 30-45 min at 40000 &times; <i>g</i>
(18,000 rpm in SS-34 rotor).

<table>
<caption>Chromaffin cell extraction buffer</caption>
<tr>
 <th>Component at final strength
 <th>1.5-strength solution
 <td>To make 500 mL:
<tr>
 <td>50 mM Tris acetate pH 8.0
 <td>75 mM
 <td>37.5 mL of Tris base<br>
      titrate with glacial acetic acid
<tr>
 <td>0.27 M sucrose
 <td>0.405 M
 <td>69.31 g sucrose
<tr>
 <td>25 mM sodium fluoride
 <td>37.5 mM
 <td>25 mL of 500 mM NaF
<tr>
 <td>1.0 mM EDTA
 <td>1.5 mM
 <td>1.5 mL of 0.5 M EDTA
<tr>
 <td>0.05% 2-mercaptoethanol
 <td>0.075%
 <td>375 &mu;L of &beta;-mercaptoethanol
<tr>
 <td>0.2% Triton X-100
 <td>0.3%
 <td>15 mL of 10% TX-100
<tr>
 <td>1 mM sodium vanadate (Na<sub>3</sub>VO<sub>4</sub>)
 <td>1.5 mM
 <td>7.5 mL of 100 mM Na<sub>3</sub>VO<sub>4</sub>
<tr>
 <td>5 mM &szlig;-glycerophosphate
 <td>7.5 mM
 <td>7.5 mL 500 mM &szlig;-glycerophosphate
<tr>
 <td>[40 mM p-nitrophenyl phosphate]
 <td>60 mM
 <td>&nbsp;
<tr>
 <td colspan="3">bring this to 492 mL with water for later addition of components below
<tr>
 <th colspan="2" style="text-align:left">These reagents are added freshly
 <th style="text-align:left">per 10 mL of 1.5-strength
<tr>
 <td>1.0 mM PMSF
 <td>1.5 mM
 <td>0.15 mL of 0.1 M PMSF
<tr>
 <td>5.0 &mu;G/mL leupeptin
 <td>7.5 &mu;G/mL
 <td>20 &mu;L of 5 mg/ml leupeptin
<tr>
 <td>10 nM microcystin
 <td>15 nM
 <td>0.6 &mu;L of 250 &mu;M microcystin
</table>

<p>
Supernatants are recovered and are added to an equal volume of
cold DEAE-cellulose equilibration buffer is added.  An approximately
10 mL equilibrated DEAE-cellulose column is loaded either by gravity
or pump with the supernatant.

<p>
The column is washed until OD<sub>280</sub> reaches baseline.
Elution is by step with DEAE elution buffer again until
OD<sub>280</sub> is baseline.  Fractions of 3-4 mL are
collected from this step to minimize pool volume.  Pooling
is done by looking at the OD<sub>280</sub> (protein-containing)
fractions.  An aliquot of the pool should be saved for peptide
activity assay.

<table style="margin:1em auto;">
<caption>DEAE equilibration buffer</caption>
<tr>
 <th>Component
 <th>To make 1 L:
<tr>
 <td>20 mM Tris HCl pH 7.5
 <td>20 mL of 1 M Tris base  titrate with HCl
<tr>
 <td>0.5 mM EDTA
 <td>1.0 mL of 0.5 M EDTA
<tr>
 <td>0.05% 2-mercaptoethanol
 <td>0.5 mL of &beta;-mercaptoethanol
<tr>
 <td>5 nM microcystin
 <td>20 &mu;L of 250 &mu;M microcystin
<tr>
 <td>0.1 mM PMSF
 <td>1 mL of 0.1 M PMSF
<tr>
 <td>1 &mu;G/mL leupeptin
 <td>0.2 mL of 5 mG/mL leupeptin
<tr>
 <td>10% glycerol
 <td>200 mL of 50% glycerol or 100 mL neat
<tr>
 <td>[0.5 mM sodium vanadate]
 <td>[5 mL of 100 mM Na<sub>3</sub>VO<sub>4</sub>]
</table>

<table style="margin:1em auto;">
<caption>DEAE elution buffer</caption>
<tr>
 <th>Component
 <th>To make 100 mL:
<tr>
 <td>0.4 M NaCl
 <td>add 2.34 g NaCl to equilibration buffer
       and re-titrate to pH 7.5 if necessary
</table>

<p>
The DEAE pool is filtered with a low protein binding
0.2 &mu;m cellulose acetate filter cartridge, and the
filtrate is diluted with 3 volumes of Mono Q buffer A
just prior to Mono Q chromatography.  Using a Superloop,
the diluted pool is loaded on to Mono Q at room temperature,
and then developed using a standard program of
0 to 400 mM NaCl over 25 column volumes (25 mL).
The activity consistently elutes at the desired step.

<table style="margin:1em auto;">
<caption>Mono Q buffer A</caption>
<tr>
 <th>Component
 <th>To make 1 L:
<tr>
 <td>20 mM Tris HCl pH 7.5
 <td>20 mL of 1 M Tris base  titrate with HCl
<tr>
 <td>0.1 mM EDTA
 <td>0.2 mL of 0.5 M EDTA
<tr>
 <td>0.5 mM DTT
 <td>1.0 mL of 0.5 M DTT
<tr>
 <td>filter both Mono Q buffers with 0.2 &mu; filter
 <td>&nbsp;
</table>

<table style="margin:1em auto;">
<caption>Mono Q buffer B</caption>
<tr>
 <th>Component
 <th>To make 500 mL:
<tr>
 <td>1.0 M NaCl
 <td>add 29.22 g NaCl to Mono Q buffer A
     and re-titrate to pH 7.5 if necessary
</table>

<p>
The kinase preparations may then be additionally worked up.

<h1>Physiology</h1>

<p style="text-align:center;">
   <img src="TH-nervterm.gif"
   alt="multiple protein kinases that target tyrosine hydroxylase">

<p style="text-align:center;">
   <img src="THactivate.gif"
alt="pathway showing how nicotinic acetylcholine receptor binding leads to activation of protein kinases leading to activation of tyrosine hydroxylase">
</body>
</html>
