<!DOCTYPE html>
<html>
<head>
<meta http-equiv="Content-Type" content="text/html; charset=iso-8859-1">
<title>Notes on the Literature from General Science (Vol. 1)</title>
<link href="/style/std.css" type="text/css" rel="stylesheet">
<link href="/style/biblio.css" type="text/css" rel="stylesheet">
<style type="text/css">
   #headernotes { text-ident:0;font:bold 90% Helvetica,Arial,sans-serif; }
   #headernotes P { margin:1em 0; }
 #two-d-struct TD {text-align:center;padding:0;
     font:normal 90% 'Courier New',Courier,monospace;}
 .underline {text-decoration:underline;}
</style>
</head>
<body>

<p id="title">
General Science Literature (Vol 1)
<p id="subtitle">
	Notes on scientific articles specifically unclassified to year 1987
<p>
This document has been divided into four parts.  This is part 1.
The other parts are:
<p><a href="GeneralScienceVol2%20(1987-2000).html">General Science Volume 2</a>
  (from 1987 to 2000)
<p><a href="GeneralScienceVol3%20(2001-2012).html">General Science Volume 3</a>
  (from 2001 to 2012)
<p><a href="GeneralScienceVol4%20(2013-present).html">General Science Volume 4</a>
  (from 2013 to present)
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Hill, A. V.
 </span>
 <span class="year">
(1913)
 </span>
 <span class="jname">
Biochem. J.
 </span>
 <span class="jvol">
7,
 </span>
 <span class="pages">
471.
 </span>
<p class="reftitl">
[this paper refers to the famous Hill plot used in calculation of binding sites]
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Krebs, H. A., Henseleit, K.
 </span>
 <span class="year">
(1932)
 </span>
 <span class="jname">
Hoppe-Seyler's Z. Physiol. Chem.
 </span>
 <span class="jvol">
210,
 </span>
 <span class="pages">
33-66.
 </span>
<p class="reftitl">
Untersuchungen &Uuml;ber die Harnstoffbildung im Tier-K&ouml;rper
<!-- ================================================================ -->
<p class="tags">

<p class="ref">
 <span class="authors">
A. C. Chibnall &amp; R. G. Westall
 </span>
 <span class="year">
(1932)
 </span>
 <span class="jname">
Biochemistry
 </span>
 <span class="jvol">
26,
 </span>
 <span class="pages">
398-411.
 </span>
<p class="reftitl">
The Estimation of Glutamine in the Presence of Asparagine
<p class="Para1">
Only one-half of the total nitrogen of asparagine reacts with Van Slyke
nitrous acid, whereas all nitrogen reacts with glutamine, as shown
by Schulze &amp; Bosshard (1883) and noted by Thierfelder &amp; von Cramm
(1919).  Bosshard (1883) had shown that glutamine boiled in magnesia
quickly hydrolyzes the amide group.
<p>
Ammonia is determined at 100&deg; using a micro-Kjehldahl apparatus.
The procedure involves adding 3 ml 10% magnesia suspension to 10 ml
plant extract.  Ammonia is distilled under vacuum for about 15 min
with flask in 40&deg; water bath.  The liquid was concentrated by
passing a small air stream (as opposed to the use of steam).  Nearly
all (at least 98% in control experiments) of the ammonia passes from
the distilland.  Acetic acid is added to dissolve excess magnesia.
The concentrate is then transferred to a 10 ml graduate cylinder,
from which 2 ml aliquots are processed through the Van Slyke
micro-apparatus to determine amino-N.
<p>
When Gln is processed through Van Slyke without heat or magnesia,
Van Slyke determination after 10 min shows that 92% of total nitrogen
is measured as amino-N.  Increased times in Van Slyke micro-apparatus
only in improve the measured yield slightly:  94% after 30 min,
97.5% after 150 min.  The theoretical amino-N for Gln is 50% of
total N.
<p>
If Gln is boiled (100&deg;) for 5 min, 78% of total N is found as amino-N,
with 8.9% being amide-N.  If heated at 40&deg; for 15 min, 92% is
found as amino-N, 1% as amide-N.  Now when magnesia is included
at 100&deg; for 10 min, 74% is amino-N, 13.5% amide-N.  At 40&deg;
for 15 min in magnesia, 1% is amide-N, and 86% as amino-N.
Asparagine yields amino-N identical with the theoretical (50% of
total N);  this is true no matter what the conditions.  Urea,
which has no theoretical amino-N, shows 10% of total N as amino-N
when heated 40&deg; for 15 min in magnesia.  With allantoin, also
having no amino-N, only 1-2% of total N is amino-N (theoretically
none).
<p>
Amides such as acetamide and propionamide are stable in
nitrous acid/acetic acid solution but not in strong HCl (fully
hydrolyzed in 24 h).  Plimmer (1925) has suggested that in
aqueous or acetic acid solution, amides tautomerize
to a R-C(OH)=NH and resist hydrolysis, while in mineral acid
they are sensitive to attack being in the form R-C(=O)-NH<sub>2</sub>.
<p>
pH stability was tested using 30 mg of either Gln, Asn, or Glu dissolved
in 10 ml of buffer at the experimental pH, then heated in a boiling
water bath over a time course.  Amide-N and amino-N were determined.
Eight pH values from 2 to 9 were tested, with heating for 2 hours,
while five different times (from 1-5 h) were tested at pH 4.  Asn
is quite stable from pH 4-8, while only 7% of amide-N is hydrolyzed
at pH 2.  Gln is quite unstable at the slightly acidic (pH 3, 4, 5)
and alkaline (8, 9, 10) pH values.  Glu forms the lactam ring.
But what is interesting is that while Gln amide-N declines, so
does amino-N.  What seems clear is that the Gln combines with another
Gln to form a six-membered anhydride, 2,5-diketopiperazine-3:6-dipropionamide.
<p>
Boiling Gln at pH 4 for 3 h provides amide-N of 44% of total N, 5% for
amino-N.  Controls (40&deg; for 15 min in magnesia) give 1% amide-N
and 86% amino-N.  With Asn, the difference between conditions shows
essentially the same result for amide- and amino-N, being  0.5% and
49%, resp.
<p>
Solutions of Gln and Asn mixed in various proportions were tested in
control conditions, under conditions that hydrolyze Gln, and under
conditions (2 N sulfuric acid) that hydrolyze both Gln and Asn.
Results show that the expected hydrolysis based on what is known
about the chemistry of Gln and Asn match the observed hydrolysis,
which is an 81% decrease in amino-N, and an 43% in amide-N values,
which can be expressed as a ratio of 1.9 (81/43).   Determinations
of this ratio of decrease in amino-N/increase in amide-N were consistent.
<p>
Urea has also been shown to be unstable at alkaline and acidic pH values,
forming ammonium carbonate.  Results here show that 34% of urea is
converted to ammonia at pH 4, only 5% at pH 8.
<p>
In agreement with previous reports, Gln appearing in dipeptide (Leu-Gln)
shows the amide is much more stable at pH 8 than is Gln itself.  Certain
errors will occur in estimating Gln in plant extracts under hydrolytic
condtions.  The following is recommended in making these estimations:
<ol>
<li>Heat in 1 N sulfuric acid to determine the amide-N, which represents
the sum (total) of Gln and Asn amide-N hydrolysis.
<li>From control conditions, compare the increase in amide-N
yield under pH 8 hydrolyzing conditions, then multiply by 1.4,
which is the total Gln amide-N.
<li>The Asn amide-N is the difference between the total amide-N of
step 1 and the Gln amide-N determined in step 2.
<li>Under pH 8 conditions, the ratio of the percent decrease in amino-N
to the percent increase in amide-N should be about 2.3.
<li>The ratio of Gln amide-N as percent of total N to the <i>decrease</i>
in amino-N under 1 N sulfuric acid conditions should be about 1.  This
will be increased if peptide hydrolysis occurs.
<li>Urea presence can be detected by any increase in the step 2 value
if analyzed under pH 4 conditons.
<li>A small error due to the presence of allantoin will affect the value
of Asn amide-N.
</ol>
<p>
This technique was applied to plant extracts obtained by the method
of Chibnall (<i>Biochem J.</i> <b>16</b>: 144, 1922) modified such
that a mincing machine was used instead of an end-runner mill, and
cell wall was not extracted with boiling water.  Protein was coagulated
by steam heating, then filtered with filtrate rapidly cooled to 40&deg;
and concentrated.  Sodium hydroxide was added to titrate to pH 8 (both
determined with a pH meter and also to a faint pink of phenolphthalein
indicator).  Etiolated castor oil plant (<i>Ricinus communis</i>) contains
considerable Gln relative to seedlings exposed to light.  Young
seedlings of Brussels sprouts (<i>Brassica oleracea</i> var. <i>bullata</i>)
grown in light contain more Asn than Gln:  although the ratio
of the decrease in amino-N to the increase in amide-N at pH 8 is 2,
there is an increase rather than decrease in amino-N in sulfuric acid
hydrolysis, making it necessary to reject the determined Gln amide-N
small value of 1%.  A small Gln value should be confirmed by the
decrease in amino-N by sulfuric acid, since it is affected by peptide
hydrolysis.
<!-- ================================================================ -->
<p class="tags">

<p class="ref">
 <span class="authors">
J. B. Gilbert, V. E. Price, J. P. Greenstein
 </span>
 <span class="year">
(1937)
 </span>
 <span class="jname">
J. Biol. Chem.
 </span>
 <span class="jvol">
119,
 </span>
 <span class="pages">
309-331.
 </span>
<p class="reftitl">
The Glutamic Acid-Pyrrolidonecarboxylic Acid System
<p class="Para1">
Glutamine or pyrrolidonecarboxylic acid were placed in reaction
mixtures of HCl or NaOH, and boiled in alcohol (78&deg;), water (100&deg;),
or acetic acid (118&deg;).  The manometric van Slyke assay was used
to measure amino nitrogen.  With multiple determinations, the
conditions for equilibrium could be established and the pH measured.
<p>
The equation:
<table class="eqn">
<tr>
 <td rowspan="2"><i>t</i>(<i>k'</i> + <i>k''</i>) = ln
 <td><i>Ka</i>
<tr>
 <td style="border-top:1px solid black;">
           <i>Ka</i> &minus; (<i>K</i> + 1)<i>x</i>
</table>
<p>
is used to calculate rates of reactions (velocities). The values
<i>a</i> and <i>x</i> have their &ldquo;usual meaning&rdquo; (taken from
text), and the velocity of dehydration = <i>k</i>'[glutamic acid] and
hydration = <i>k</i>''[anhydride].  Results show that at neutral
pH range, reaction rates were on the order of days.  A figure of
the log <i>K</i> against pH at reveals at least 4 concepts:
<ol>
<li>At neutral pH the equilibrium is shifted to greater dehydration
of glutamic acid, although the reaction rate is quite slow at
1% conversion in 2-3 hours at 100&deg;.
<li>At extremes of pH (&gt; 2 M HCl or &gt; 0.5 M NaOH), the ring
opening of pyrrolidonecarboxylic acid to Glu is rapid
<li>These reactions were considered as interesting in isolating and
determining the levels of Glu.  Hence conditions should be set to
favor ring closure.  At pH 4 or 10 at 100&deg;, the reaction is 98%
complete in 50 h.  At a lower pH (2-3), the reaction rate is twice
as fast but reaction is 90-94%.  Increasing the temperature to 120&deg;
at pH 3 gives a reaction reaching equilibrium in 3 hours, and with
a more favorable conversion.
<li>Hydrolysis of the ring is best at 100&deg; in not less than 2 M HCl
or 0.5 M NaOH and done in 1-2 h.
</ol>
<!-- ================================================================ -->
<p class="tags">

<p class="ref">
 <span class="authors">
J. B. Gilbert, V. E. Price, J. P. Greenstein
 </span>
 <span class="year">
(1949)
 </span>
 <span class="jname">
J. Biol. Chem.
 </span>
 <span class="jvol">
???
 </span>
 <span class="pages">
209-218.
 </span>
<p class="reftitl">
Effect of Anions on the Non-Enzymatic Desamidation of Glutamine
<p class="Para1">
Hydrolysis was tested non-enzymatically and using a fresh rat liver
extract.  The effect of various anions (their sodium salts) was tested.
The anions are phosphate, phosphite, arsenate, methyl arsenate, bicarbonate,
sulfate, borate, pyruvate, acetate, chloride, and nitrate.
<p>
A typical time course for the &mu;moles of amide nitrogen evolved shows
a significant effect of phosphate (a veronal acetate, pH 8.0, buffer and
47&deg; temperature is part of the reaction conditions).  After 48 h,
desamidation is nearly 100% complete with anion present, only 30%
in the absence of it.  A dose curve for several anions is shown
in a figure (with reaction time 8 hours at 37&deg;).  Phosphate had the
greatest effect at equimolar concentrations, followed by arsenate, and
methyl arsenate, although the differences in effect were not that signficant.
Phosphite had a weak effect.  Borate had a slight effect, although it
was tested over a limited range because of insolubility.  Chloride, sulfate,
nitrate, acetate, and pyruvate up to 0.2 M had no ability to accelerate
desamidation.  There was also no synergy:  combinations of phosphate and
sulfate had the same effect as phosphate alone.  Bicarbonate was quite
active, but the pH could not be controlled and its effect was not controlled.
The effect of pH was actually tested: at the slightly higher pH 8.4
(&plusmn; 0.1), it was clear that there was greater desamidation among
all anions tested that had any effect at pH 8.0.
<p>
The effect of varying Gln was tested using phosphate.  Generally there
was no effect.  Double the Gln (from 25 to 50 mM) will yield double the
product at all concentrations of phosphate.  The pH curve shows that
acidification below pH 4 results in glutamine desamidation whether or
not anions are present.  This decreases as the pH gets higher, but the
effect of the anion is seen at the higher pH.  Gln is most stable in the
range pH 4-7.  Temperature effects were also determined, with the
variable of phosphate presence or absence also assessed:  for each
10&deg; increase, the rate of desamidation increases by about 2.
<p>
It is known that glutamine undergoes intramolecular transamidation
to form the closed ring, a reaction accelerated by phosphate, bicarbonate,
and other anions.  The presence of pyrrolidonecarboxylic acid (product)
or Glu added 400:1 to Gln did not affect this reaction.
<p>
Chloroacetylglutamine was tested in water or phosphate at 47&deg; for
4 and 8 hours.  Significant desamidation occurs with glutamine alone, but
with this compound there is no reaction.  In rat liver extracts treated
with pyruvate, it has been noted that significant ammonia production
occurs with chloroacetylglutamine addition:  this is likely because of
hydrolysis of the amide to produce glutamine, which is then cyclized.
<p>
As for enzymatic desamidation, bicarbonate and borate did not help or
alter the action of the enzyme compared to their absence.  It had
been shown elsewhere that enzymatic action is accelerated by the presence
of phosphate, arsenate, and sulfate between 0.02 and 0.20 M.
<p>
The effect of pH particularly on sulfate-accelerated desamidation was
studied.  The product vs. pH curve for the presence of sulfate shows
a sharp peak maximal at pH 8, dropping off sharply at pH 10 and 6 (these
are about 15% of maximal).  In the absence of sulfate the peak is still
at pH 8, but the production is 5-10% of the maximum of sulfate presence.
Sulfate concentration was varied, and it was found that when the molar
ratio of 8 and above was reached, the extent of desamidation reaches a
maximum; up to that point, the change in accelerated desamidation is
linear.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Lowry, O. H., Rosebrough, N. J., Farr, A., Randall, R. J.
 </span>
 <span class="year">
(1951)
 </span>
 <span class="jname">
J. Biol. Chem.
 </span>
 <span class="jvol">
193,
 </span>
 <span class="pages">
265-275.
 </span>
<p class="reftitl">
Protein measurement with the Folin phenol reagent
<!-- ============================================================= -->

<p class="ref">
 <span class="authors">
Bornstein, M. B.
 </span>
 <span class="year">
(1958)
 </span>
 <span class="jname">
Lab. Invest.
 </span>
 <span class="jvol">
7,
 </span>
 <span class="pages">
134-137.
 </span>
<p class="reftitl">
Reconstituted rat-tail collagen used as substrate for tissue cultures on 
coverslips in Maximow slides and roller tubes
<p class="Para1">
Ehrmann and Gey [(1956) <i>J. Natl. Cancer Inst.</i> <b>16</b>, 1375] have 
described a method for preparing walls of roller tubes with a slice or coating 
of reconstituted rat-tail collagen as a substrate for growing CNS cells from 
chicken embryo and newborn feline in culture.  Its use in other cell types is 
described.
<p>
Either fresh or &minus;l80&deg; stored tails are soaked in 95% EtOH for 15 min, 
and the tail is fractured into small pieces beginning at the tip.  The long 
silvery tendons are cut free and allowed to drop into a Petri dish containing 
sterile distilled water.  Fine forceps are used to tease apart the finer 
filaments and put en masse into a 250 mL-centrifuge bottle containing 150 mL of 
1:1000 acetic acid solution which is capped and stored 48 hrs in the 
refrigerator. Centrifugation for 2 hr at 2300 rpm obtains a pellet...
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Bray, G.
 </span>
 <span class="year">
(1960)
 </span>
 <span class="jname">
Anal. Biochem.
 </span>
 <span class="jvol">
1,
 </span>
 <span class="pages">
279-285.
 </span>
<p class="reftitl">
A simple efficient liquid scintillator for counting aqueous solutions
in a liquid scintillation counter
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Douglas, W. W., Poisner, A. M.
 </span>
 <span class="year">
(1961)
 </span>
 <span class="jname">
Nature (London)
 </span>
 <span class="jvol">
192,
 </span>
 <span class="pages">
1299.
 </span>
<p class="reftitl">
[something about acetylcholine stimulating Ca<sup>2+</sup>to enter cells]
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Anfinsen, C. B., Haber, E., Sela, M., White, F. H., Jr.
 </span>
 <span class="year">
(1961)
 </span>
 <span class="jname">
Proc. Natl. Acad. Sci. USA
 </span>
 <span class="jvol">
47,
 </span>
 <span class="pages">
1309.
 </span>
<p class="reftitl">
[reports direct evidence that conformation of proteins is
determined by amino acid sequence, i. e., primary structure dictates
secondary structure]
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Monod, J., Changeux, J.-P., Jacob, F.
 </span>
 <span class="year">
(1963)
 </span>
 <span class="jname">
J. Mol. Biol.
 </span>
 <span class="jvol">
6,
 </span>
 <span class="pages">
306 (1963).
 </span>
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Vogt, M., Dulbecco, R.
 </span>
 <span class="year">
(1963)
 </span>
 <span class="jname">
Proc. Natl. Acad. Sci. USA
 </span>
 <span class="jvol">
49,
 </span>
 <span class="pages">
171-179.
 </span>
<p class="reftitl">
Steps in the neoplastic transformation of hamster embryo cells by polyoma virus
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Bohak, Z.
 </span>
 <span class="year">
(1964)
 </span>
 <span class="jname">
J. Biol. Chem.
 </span>
 <span class="jvol">
239,
 </span>
 <span class="pages">
2878-2887.
 </span>
<p class="reftitl">
<i>N</i><sup>&epsilon;</sup>-(<span 
style="font-variant:small-caps;">dl</span>-2-Amino-2-carboxyethyl)-<span 
style="font-variant:small-caps;">l</span>-lysine, a new amino acid formed on 
alkaline treatment of proteins
<p class="Para1">
Ribonuclease A is exposed to alkali and hydrolyzed. Some of the lysines from the 
protein form a new amino acid.  The extent of the recovery was the same over a 
range of ribonuclease concentrations, suggesting an intramolecular reaction. 
When free L-lysine and poly-L-lysine were treated similarly, neither loss of 
lysine nor recovery of the new amino acid was observed, indicating the 
involvement of other amino acids in the reaction.
<p>
Cystine was also affected by the alkali: after treatment, performic acid was 
added and cysteic acid analysis revealed less oxidized acid.  Dehydroalanine 
residues were recovered to the same extent.  Performic acid oxidation prior to 
alkali treatment prevented the formation of the new amino acid.  However, 
reduced and carboxymethylated protein resulted in the formation of the new amino 
acid as much as the protein itself, although at slower rate. The rate of 
formation was both pH and temp. dependent, with increases in pH being especially 
important.
<p> 
Other proteins were tested and not all containing both cystine and lysine 
produced the new amino acid.  Phosvitin had 80% of its lysines transformed, and 
it has no cystinyl or cysteinyl residues, but is rich in phosphoserine, which 
are known to undergo rapid &beta;-elimination to dehydroalanine. Lysinoalanine 
is the trivial name given to this amino acid, although it is identical with 
arginosuccinic acid.  Periodate oxidation of the new amino acid yields 1 mol 
lysine, ammonia, and HCHO, which is what is expected of a molecule containing 
two vicinal amino groups.
<p>
The synthesis of a standard is reported: 
<i>N</i><i><sup>a</sup></i>-carbobenzyloxy-<span 
style="font-variant:small-caps;">l</span>-lysine (2.8 g, 10 mmol), 
<i>N</i>-phenylacetyldehydroalanine (2.05 g) and Ba(OH)<sub>2</sub> (6.3 g, 20 
mmol) were mixed mixed and 5 ml of freshly boiled dH<sub>2</sub>O was added with 
stirring.  The hazy soln. is left 7 days in 40&deg; water bath, being shaken 
occasionally. The susp. is poured into 10 ml water and titrated to pH 6.0 with 2 
M H<sub>2</sub>SO<sub>4</sub> with rapid stirring.  The pH should remain stable 
for 15 min.  The mixture is brought to pH 1.5 with 3 N HCl and left 30 min at 
room temp.  The ppt. is spun for 20 min @ 15000 &times; <i>g</i>.  and extracted 
3 times with 15 ml 0.1&nbsp;N HCl. The supnt. and washings are evap'd. and the 
residue suspnded in anhydrous CHCl<sub>3</sub> and cooled to 0&deg;;  the suspn 
was sat'd with anhydrous HBr and stood at room temp for 2 hr.  The solvent was 
removed under reduced pressure, the residue refluxed for 3 hr with 30 ml 3 N 
HCl, and the soln evap'd to dryness.  Phenylacetic acid was removed with three 
20 ml ether extractions, and the residue taken up in 5 ml water and filtered. 
The filtrate contains 1.6 mmol derivative and 4.6 mmol lysine.  Purification is 
done on 2 &times; 30 cm BioRad AG50-X8 (-400 mesh) with a 3 ml load of the 
filtrate.  The fractions chosen are pooled, desalted (Dr&egrave;ze, Moore, 
Bigwood (1954) <i>Anal. Chim. Acta</i> <b>11</b>, 554), and evap'd.  The 
resulting solid was dissolved 3 ml ice-cold anhydrous EtOH, filtered, ppt'd. 
with 10 ml ether.  The ppt. was taken up in 2 ml water in a 18 &times; 250 mm 
tube and cooled to 0&deg;. Ice-cold 2 M pyridine in EtOH (1.5 ml) was added and 
ice-cold EtOH was layered on the top.  Crystallization starts in the boundary 
layer and clusters of well-formed needles are seen after a day in the fridge. 
Crystals are increased when the tube is swirled gently, causing gradual mixing 
of the two layers.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Mans, R. J., Novelli, G. D.
 </span>
 <span class="year">
(1964)
 </span>
 <span class="jname">
Arch. Biochem. Biophys.
 </span>
 <span class="jvol">
94,
 </span>
 <span class="pages">
48-53.
 </span>
<p class="reftitl">
Measurement of the incorporation of radioactive amino acids into
proteins by a filter paper disk method
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Ferdinand W
 </span>
 <span class="year">
(1964)
 </span>
 <span class="jname">
Biochem. J.
 </span>
 <span class="jvol">
92,
 </span>
 <span class="pages">
578-585.
 </span>
<p class="reftitl">
The isolation and specific activity of rabbit-muscle glyceraldehyde phosphate
dehydrogenase
<p class="Para1">
This technique exploits the high solubility of this protein in ammonium sulfate.
This protein is about 10% of the total rabbit muscle protein by weight, with a
yield of 1.2 g/kg muscle.
<p> Minced muscle is extracted in a solution of 50 mM
EDTA-NH<sub>4</sub><sup>+</sup>.  The extract is centrifuged for 20 min at
10,000 g.  The supernatant pH is brought to 7.4, and then solid AmSO<sub>4</sub>
is added to 50% saturation.  After an appropriate stirring period, the solution
is centrifuged.
<p> The supernatant is then brought to 70% saturation with AmSO<sub>4</sub>,
stirred, then centrifuged.  Now the supernatant is brought to 72% saturation
with AmSO<sub>4</sub>, then centrifuged.  The enzyme is now in the pellet.
<p> To enhance the purification, the enzyme can be dissolved in buffer and taken
through AmSO<sub>4</sub> once again. The enzyme should crystallize in the end.
<p> Another useful method for purifying the enzyme takes advantage of an
NAD+-Sepharose affinity column, which can work directly on crude extracts.  (JD
Hocking, JJ Harris, FEBS Lett. 34, 280, 1973)
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Guzzo, A. V.
 </span>
 <span class="year">
(1965)
 </span>
 <span class="jname">
Biophys. J.
 </span>
 <span class="jvol">
5,
 </span>
 <span class="pages">
809.
 </span>
<p class="reftitl">
[concerns protein secondary structure prediction model which analyzes
&ldquo;helix breakers&rdquo;]
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Prothero, J. W.
 </span>
 <span class="year">
(1966)
 </span>
 <span class="jname">
Biophys. J.
 </span>
 <span class="jvol">
6,
 </span>
 <span class="pages">
367.
 </span>
<p class="reftitl">
[concerns report of protein secondary structure
prediction model which analyzes in terms of helix formation]
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Grollman, A. P.
 </span>
 <span class="year">
(1966)
 </span>
 <span class="jname">
Proc. Natl. Acad. Sci. USA
 </span>
 <span class="jvol">
56,
 </span>
 <span class="pages">
1867-1874.
 </span>
<p class="reftitl">
Structural basis for inhibition of protein synthesis by emetine and
cycloheximide based on an analogy between ipecac alkaloids and
glutarimide antibiotics
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Burton, K.
 </span>
 <span class="year">
(1966?)
 </span>
 <span class="jname">
Meth. Enzymol.
 </span>
 <span class="jvol">
??,
 </span>
 <span class="pages">
163-166.
 </span>
<p class="reftitl">
Determination of DNA concentration with diphenylamine
<p class="Para1">
Deoxyribose reacts with diphenylamine to produce a characteristic
color reaction indicating presence of DNA.  Dische
(<i>Mikrochemie</i> <b>8</b>: 4, 1930) first described it:
heat the sample to 100&deg; for a few min in a mixture of acetic acid,
sulfuric acid and diphenylamine.  A much better sensitivity is
obtained by also adding perchloric acid and acetaldehyde and
developing color for 17 h at 30&deg;.
Burton (<i>Biochem J.</i> <b>62</b>: 315, 1956) modifications.
<ol>
<li>1.6% aqueous acetaldehyde is prepared.  The acetaldehyde is cooled
and 1 ml is transferred in a cooled pipette in 50 ml water.  Commercial
acetaldehyde is satisfactory if fresh;  otherwise mix with dilute
sulfuric acid and re-distill.  The aqueous solution is stable for
several months at 4&deg; in a well-stoppered bottle.
<li>Diphenylamine reagent is prepared by dissolving 1.5 g diphenylamine
in 100 ml glacial acetic acid, followed by addition of 1.5 ml conc.
sulfuric acid.  Just before use, 0.1 ml of 1.6% acetaldehyde is
added to 20 ml of the reagent.
<li>DNA standards of 0.3&nbsp;mg/ml in 5 mM NaOH are prepared.  Working
standards can be prepared from this by mixing a measured volume of
the stock with equal volume of 1&nbsp;N perchloric acid, heat to 70&deg;
for 15 min.  Both stock and working standard are stored at 4&deg;.
Other standards can include deoxyribose, deoxyadenosine, deoxyguanosine,
or the purine mononucleotides.  Different compounds show different rates
of color development.
</ol>
<p>
If the sample to be determined has protein, heat with 0.5 to 1.0 N
perchloric acid or with 5% trichloroacetic acid to extract the DNA.
(If TCA is used, bring the final extract to 0.5 to 1.0&nbsp;N with respect
to perchloric acid.)  Conditions such as 20 min with 0.5&nbsp;N HClO<sub>4</sub>
heated to 70&deg; or 10 min with 5% TCA heated to 90&deg; are sufficient,
destroying &lt;5% of the deoxyribose.  Others have suggested that animal
tissues be treated with either 2% HClO<sub>4</sub> (0.3&nbsp;N) or 5% TCA
at 90&deg; for 30 min.  The presence of protein can seriously affect
the modified reaction.
<p>
The DNA extract is diluted with 0.5&nbsp;N perchloric acid to bring the
DNA-phosphate to 20-250 nmol/ml (20-250&nbsp;&mu;M).  Varying portions
are mixed with 2 vols of the diphenylamine reagent.  Standards and
blanks are prepared;  all tubes should contain the same amt of
perchloric acid.  Incubate 15-17 h at 25-30&deg; and measure OD
at 600 nm.  With the optimum acetaldehyde amt., a 0.37 AU/100 nmol DNA
(1 cm path length) is typical.  Color development rate does increase
with temperature, but sensitivity can be lost.
<p>
Among seriously interfering substances is included ascorbic acid
(0.4 mg) and cysteine HCl (2 mg) causing a >70% depression in color.
Other substances had small effects.
<p>
<span style="text-decoration:underline;">Croft and Lubran (<i>Biochem.
J.</i> <b>95</b>: 612, 1965) modification</span>.  These authors were measuring DNA in saline
washes of human stomachs.  This material has high amt of sialic
acid which is an agent interfering with the reaction.  A substance
with absorption maximun at 550 nm is observed.  The Burton method
was largely used except that the color was developed for 48 hr at
6-13&deg; (reading at both wavelengths helps) and 2% diphenylamine
was used instead of 1.5%.
<p>
<span style="text-decoration:underline;">Giles and Myers (<i>Nature</i>
<b>206</b>: 93, 1965) modification.</span>
Found that blank readings could be reduced by omitting sulfuric acid
and by adding acetaldehyde to individual tubes.  An increase in
sensitivity was also obtained by increasing diphenylamine concentration
and by altering the relative volumes of sample and reagent.  Samples
and stds are prepared by heating in 1&nbsp;N perchloric acid.  Suitable
volumes are mixed with an equal volume of freshly prepared
4% diphenylamine in glacial acetic acid.  1.6% acetaldehyde
is added (5% of the volume of the sample or reagent).  Incubation
is ON at 30&deg;.  Absorbance is measured at 595 nm.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Periti, P. F., Quagliarotti, G., Liquori, A. M.
 </span>
 <span class="year">
(1967)
 </span>
 <span class="jname">
J. Mol. Biol.
 </span>
 <span class="jvol">
24,
 </span>
 <span class="pages">
313.
 </span>
<p class="reftitl">
[concerns use of helical and antihelical pair analysis in
predicting polypeptide secondary structure]
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Schiffer, M., Edmundson, A. B.
 </span>
 <span class="year">
(1967)
 </span>
 <span class="jname">
Biophys. J.
 </span>
 <span class="jvol">
7,
 </span>
 <span class="pages">
121.
 </span>
<p class="reftitl">
[this report concerns analyze of protein secondary structure and
introduces the concept of a &ldquo;helical wheel&rdquo; to locate
hydrophobic arcs]
<!-- ============================================================= -->
<p class="tags">
IgM
<p class="ref">
 <span class="authors">
Mihaesco C, Seligmann M
 </span>
 <span class="year">
(1968)
 </span>
 <span class="jname">
J. Exp. Med.
 </span>
 <span class="jvol">
127,
 </span>
 <span class="pages">
431-453
 </span>
<p class="reftitl">
Papain Digestion Fragments of Human IgM Globulins
<p class="Para1">
In a preliminary report on digestion of IgM, product Fab, Fab&prime;, and Fc
could be obtained similar to IgG analysis.  With papain digestion a high
yield of small peptides and of Fab&mu; having similar features to Fab of
IgG were found.  There is a large Fc&mu; with small &mu;-chains linked with
disulfide bonds.
<p>
<b>M &amp; M</b>.  The IgM source is from patients with Waldenstr&ouml;m
macroglobulinemia.  One patient had K type light chains, the other L type.
Citrate-dextrose plasma was defibrinated, brought to 45% ammonium sulfate
saturation to get the globulin, then NaCl added to 1.061 g/ml density to
delipidate; centrifugation was 30 krpm for 3 h 4&deg; (Spinco Model L).  IgM
was purified using Pevikon block preparative electrophoresis then Sephadex G-200.
IgM elutes in the void volume:  the peak was fractionated into two, its ascending
and descending peak signal.  Both were analyzed by ultracentrifugation,
Ouchterlony agar diffusion, and microimmunoelectrophoresis (at 20-30 mg/ml)
using anti-human rabbit antisera.  The ascending fraction had no impurities,
but traces of IgG and IgA were seen in the descending fraction, which was
repurified on G-200.  In starch gel electrophoresis with 8 M urea (10 mg/ml),
one IgM prep was fully excluded in the gel; the other prep showed single faint
band migrating to cathode.  This sample was passed through Sephadex G-100 in
1 M acetic acid and hade <i>V</i><sub>e</sub>/<i>V</i><sub>0</sub> = 2 and
was 12% of the sample.  Purified material was also treated with 0.1 M 2-mercaptoethanol
in 0.5 M Tris pH 8.0 at room temp for 90 min, alkylated with 10% molar excess
of iodoacetamide in ice bath, then passed over G-200.  5% of material eluted
in void, and the main peak was  <i>V</i><sub>e</sub>/<i>V</i><sub>0</sub> = 1.4.
Preparation of light and heavy chains was the same reduction and alkylation,
after which dialysis is done in saline for 2 h then against 1 m acetic acid
overnight in the cold; then gel filtration on G-100 in 1 M acetic acid.  For the
heavy chain, the mid-third of the first peak is used.  Immunochemical analysis
shows them to be pure.  Papain digestion using procedures similar to Porter,
with 10 mg/ml protein in 0.1 M phosphate pH 7.0, 1 mM cysteine HCl, and 1 mM
EDTA.  Twice crystallized papain is added 50:1 (w/w) substrate:enzyme, and
done in 37&deg; for 24 h, with 1-2 gtt toluene present.  50% molar excess of
iodoacetamide is added to stop reaction.  NaCl is added to 1 M to digests and
run on G-200 and pooled fractions concentrated.  For Fc&mu; Pevikon preparative
electrophoresis is used to purify, and for Fab&mu; starch gel electrophoresis
is used.  In cases where Fc&mu; yield is low, sodium sulfate is added to 1 g
of IgM at 16% w/v to salt out (16 h at room temp).  After centrifugation, it
is dialyzed in saline.
<p>
<b>Results</b>.  Immunoelectrophoretic analysis of 24 h papain digests were
done on samples from two patients with Waldenstrom's macroglobulinemia.  Two
peaks from G-200 chromatography as well as unfractionated load were analyzed.
One immunoreactive component moved toward the positive electrode a great
distance, and this is peak I in the gel-filtered material.  The other component
was a lower mobility molecule moving toward the negative electrode, and is
Peak II in the gel-filtered material.  Peak I is Fc&mu;, while peak II is Fab&mu;.
In the G-200 chromatography (monitored at OD280), peak I coeluted with
intact IgG as an impurity (Ve/V0 = 1.25).  Peak II had Ve/V0 = 2.0 and traces
of Fab&gamma; as well, probably from digest of IgG present.  Peak III was
a very large peak with Ve/V0 =3 and was composed of small peptides.  All results
were reproduced 15 times.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Blomb&auml;ck, B.
 </span>
 <span class="year">
(1967)
 </span>
 <span class="jname">
Meth. Enzymol.
 </span>
 <span class="jvol">
11,
 </span>
 <span class="pages">
398-411.
 </span>
<p class="reftitl">
Derivatives of Glutamine in Peptides
<p class="Para1">
Glutamine was first discovered in beet root juice in 1883 by Schulze and
Bosshard.  It more easily de-amidates than asparagine, at neutral pH
under mild conditions.  The product of de-amidation is the lactam
<b>pyrrolid-2-onecarboxylic acid</b>, also called <b>pyroglutamic acid</b>
(<b>I</b>).
This also forms by heating glutamic acid powder close to its melting point,
and it is found in hydrolysates of protein.  At constant pH the dehydration
of glutamate is a first-order process, and the rate is faster when
the solution is either significantly alkaline or acidic (otherwise deviated
from neutral);  in strongly acidic (not less than 2 M HCl) or
alkaline (not less than 0.5 M NaOH) conditions, hydrolysis
is favored instead.  Ammonia is produced in glutamine solutions
in either weakly alkaline or acidic solutions.  The amide group reacts
with nitrous acid.  The van Slyke nitrous acid procedure for &alpha;-amino
acids gives larger values than the theoretical.
<p>
Chibnall and Westall demonstrated in 1932 that pyroglutamate is directly
formed during Gln hydrolysis:  the evidence was that one mole of ammonia
was liberated while the &alpha;-amino nitrogen (van Slyke) decreased.
Another explanation is that glutamate condenses with another glutamate
to form a six-membered ring compound
<b>2,5-diketopiperazine-3,6-dipropiondiamide</b> (<b>II</b>) is formed.
<p style="text-align:center;">
<img src="GlnDerivs.gif"
alt="Structure of pyroglutamate and 2,5-diketopiperazine-3,6-dipropiondiamide">
<p>
Deamidation of Gln can also occur enzymatically via <b>glutaminase</b>.
Certain anions accelerate both the enzymatic process, which yields
only glutamate with ammonia, and the nonenzymatic, which yields
the lactam pyroglutamate with ammonia.  At more alkaline pH, when
anions would be expected to increase their charge valence, reaction
rates are more rapid.  Phosphate produces the best results, but
citrate, acetate, and lactate also have effects.  Under nonenzymatic
conditions, ammonium glutamate does not produce appreciable pyroglutamate,
and so the cyclication occurs via the favorability of acyl substitution
of amides by a primary amine.
<p>
Work by Melville in the 1930s has shown that synthetic peptides with
N-terminal Gln very readily undergo cyclization to the lactam under
conditions in which the peptide or proteins would be handled.
<p>
Pyroglutamate is readily extracted from aqueous solutions with organic
solvents (ether) or with glacial acetic acid, or with ethyl acetate
in acidified solutions.  It partitions mostly to phenol in phenol-water.
The Rydon and Smith chlorine-starch-iodine reaction system can be
used for detection.  Paper chromatography in <i>n</i>-butanol-water is
reported.  The pK of 3.32 makes it more mobile than most amino acids
towards the anode.
<p>
The first demonstration (by Ohira) of naturally occurring N-terminal
pyroglutamate was a tripeptide <b>eisenine</b>
<span class="sansfontbold">(pyrGlu-Gln-Ala)</span>
in the brown alga <i>Eisenia bicyclis</i> .  Another alga,
<i>Pelvetia fastigiata</i>, produces <b>fastigiatine</b>
<span class="sansfontbold">(pyrGlu-Gln-Gln)</span>
with a similar N-terminal pyroglutamate.  Other natural products
with N-terminal pyroglutamate include the undecapeptide
eledoisin <span class="sansfontbold">
(pyrGlu-Pro-Ser-Lys-Asp-Ala-Phe-Ile-Gly-Leu-Met-NH<sub>2</sub>)</span>,
a hypotensive agent in the salivary glands of <i>Eledone</i>;
and human fibrinopeptide B <span class="sansfontbold">
(pyrGlu-Gly-Val-Asn-Asp-Asn-Glu-Glu-Gly-Phe-Phe-Ser-Ala-Arg).</span>
<p>
The sorts of tests that can be done to reveal sequence information include:
<ul>
<li>partial and complete mineral acid hydrolysis of the peptide
<li>ninhydrin identification of amino acid at N-terminus (can
also react with lysine)
<li>reaction with Edman (phenylisothiocyanate) or Sanger
(fluorodinitrobenzene) reagent
<li>enzymatic digestion with specific endo- and exopeptidases, including
carboxypeptidase and elastase
</ul>
<p>
Human fibrinopeptide B is a good example of how to analyze a pyr-Glu type 
protein.  It is ninhydrin-negative but reacts with chlorination or with 
Sakaguchi &alpha;-naphthol reagent.  Papain digestion yields a tetrapeptide with 
a reactive amino terminal, and acid hydrolysis shows its composition is Asp, 
Glu, Gly, and Val.  Carboxypeptidase and elastase digestion of this small 
peptide produces Val and Asn and a N-terminal dipeptide that is Glu and Gly on 
acid hydrolysis.  Hydrazinolysis to reveal the carboxyl terminus shows Gly. 
Methanolysis of the whole hF-B peptide gives no volatile methyl ester, and 
because there was no acetyl hydrazide in the hydrazinolysis of the dipeptide 
previously mentioned, these results strongly indicate an N-terminal 
pyroglutamate. Further evidence of this was shown by the synthesis of the 
dipeptide <span style="font-variant:small-caps;">l</span>-pyroglutamylglycine, 
which was indistinguishable from the proteolyzed dipeptide under electrophoretic 
(0.1 M pyridine-acetate, pH 4.1) and 2-D TLC (I: <i>n</i>-propanol:water, 70:30; 
II: <i>n</i>-butanol:acetic acid:pyridine:water, 60:12:40:48).
<p>
Fibrinopeptide B from bovine fibrinogen (<span 
class="sansfontbold">pyrGlu-Phe-Pro-Thr-Asp-Tyr[SO<sub>3</sub>H]-Asp-Glu-Gly-Gln
-Asp-Asp-Arg-Pro-Lys-Val-Gly-Leu-Gly-Ala-Arg</span>) was also analyzed. 
Subtilisin and partial acid hydrolysis with 1% acetic acid yield a 
ninhydrin-negative tetrapeptide that is Glu, Phe, Pro, and Thr in equimolar 
amounts upon acid hydrolysis.  Hydrazinolysis of this fragment indicates Thr, 
and like the human whole B-peptide, methanolysis produces no volatile methyl 
esters.  The peptide is not reactive with PITC, but is reactive after mild 
alkaline hydrolysis;  Edman degradation shows Glu, Phe, and Pro in cycles 1-3. 
Taken together, a pyroglutamyl N-terminus is indicated, although one group 
suggests that N-acetylthreonine is a strong possibility.
<p>
Gastrin II (<span 
class="sansfontbold">pyrGlu-Gly-Pro-Trp-Met-Glu-Glu-Glu-Glu-Glu-Ala-Tyr[SO<sub>3
</sub>H]-Gly-Trp-Met-Asp-Phe-NH<sub>2</sub></span>) is a hormone in this class. 
Papain digestion yields three peptides 3, 4, and 5 residues long, all of which 
are ninhydrin-negative, monoanionic at pH 6.1 and neutral at pH 2.1. 
Carboxypeptidase on the 5-peptide releases Met then Trp.  Carboxypeptidase on 
the 3-peptide produces nothing, indicating a C-terminal Pro:  its sequence is 
inferred to be pyrGlu-Gly-Pro.  CNBr digestion of the whole B-peptide produces a 
5-peptide neutral at both pH 2.1 and 6.1 in electrophoresis:  this means there 
are no amino and carboxyl terminal groups and no side chain Glu.
<p>
Snake venom from several species (<i>Agkistrodon halys blomhoffii</i>, 
<i>Crotalus adamanteus</i>, <i>Bothrups jararaca</i>, <i>Trimeresurus 
flavoviridis</i>) contains two pyroglutamyl peptides, <span 
class="sansfontbold">pyrGlu-Gln-Trp</span> and <span 
class="sansfontbold">pyrGlu-Asn-Trp</span>.  Only the latter is found in 
<i>Vipera russelli</i>, and none of these in <i>Naja naja atra</i>.
<p>
Although the cyclization reaction generally is spontaneous, an enzyme from 
papaya latex has been reported to catalyze the reaction.  This <b>glutamine 
cyclotransferase</b> is not like glutaminase I, which only de-amidates 
glutamine; it actually cyclizes glutamine.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
DeLange, R. J., Kemp, R. G., Riley, W. D., Cooper, R. A., Krebs, E. G.
 </span>
 <span class="year">
(1968)
 </span>
 <span class="jname">
J. Biol. Chem.
 </span>
 <span class="jvol">
243,
 </span>
 <span class="pages">
2200-2208.
 </span>
<p class="reftitl">
Activation of skeletal muscle phosphorylase kinase by
adenosine trisphosphate and adenosine-3':5'-monophosphate
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Low, B. W., Lovell, F. M., Rudko, A. D.
 </span>
 <span class="year">
(1968)
 </span>
 <span class="jname">
Proc. Natl. Acad. Sci. USA
 </span>
 <span class="jvol">
60,
 </span>
 <span class="pages">
1519.
 </span>
<p class="reftitl">
[concerns subject of matching helical polypeptide fragments of known
conformation to making predictions of protein conformation or secondary
structure]
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Cox, R. A.
 </span>
 <span class="year">
(1968)
 </span>
 <span class="jname">
Meth. Enzymol.
 </span>
 <span class="jvol">
12B,
 </span>
 <span class="pages">
120-129.
 </span>
<p class="reftitl">
The use of guanidinium chloride in the isolation of nucleic acids
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Levi-Montalcini, R., Angeletti, P. U.
 </span>
 <span class="year">
(1968)
 </span>
 <span class="jname">
Physiol. Rev.
 </span>
 <span class="jvol">
48,
 </span>
 <span class="pages">
534-569.
 </span>
<p class="reftitl">
Nerve growth factor
<!-- ============================================================= -->
<p class="tags">
gel electrophoresis, sodium dodecyl sulfate (SDS)
<p class="ref">
 <span class="authors">
Laemmli, U. K.
 </span>
 <span class="year">
(1970)
 </span>
 <span class="jname">
Nature (Lond.)
 </span>
 <span class="jvol">
227,
 </span>
 <span class="pages">
680-685.
 </span>
<p class="reftitl">
Cleavage of structural proteins during the assembly of the head of bacteriophage 
T4
<p class="Para1">
T-even bacteriophages contain many proteins coded by many genes.  An improved
electrophoretic technique revealed 28 components, 11 forming the phage head.
Four head proteins were traced to genes 22, 23, 24 and a protein called IP
which is cleaved during assembly.  Gene 23 product is the principal component
of the head, while two minor components are also in capsids.  The products of
genes 20-22, 24, 32, 40, and 66 also set the size and shape of the head shell.
The functions determined includ the diameter determination (an initiation protein)
in head formation for gene 22, gene 66 involved in particle elongation, and
gene 20 and 40 involved in forming the hemispherical cap.  Ten more proteins
(products of genes 2, 4, 13, 14, 16, 17, 49, 50, 64, and 65) probably control
later steps in head formation.
<p>
Many phage proteins are separated with disk electrophoresis using sodium dodecyl
sulfate.  Proteins are separated according to molecular weight as reported
for a continuous system.  At least 28 bands are distinguished in an autoradiogram
of labeled purified phage particles.  These proteins are more than 15 kDa and
do not include smaller proteins;  the smaller proteins do not sieve in gels of
10% acrylamide and migrate with marker dye.  At higher acrylamide concentration,
at least three more proteins are resolved.  The largest protein has a MW of
at least 120,000.  No radiolabel is found as unable to migrate into the gel.
11 of the 28 protein are part of purified head preparation.  The other 17 are 
found in whole phage and presumed to be tail or tail fiber proteins.
<p>
[Much more text about the phage proteins is presented].
<p>
The gel electrophoresis method is explained in a small paragraph of the legend 
of Figure 1.  A tube gel with a discontinous phase with 3% stacking gel and 8 or 
10% separating gel is prepared from a stock of 30% acrylamide and 0.8% 
N,N&prime;-<i>bis</i>-methylene acrylamide (37.5% acrylamide:1% bis).  The 
separating gel is made of 0.375 M Tris HCl pH 8.8 and 0.1% SDS.  Polymerization 
is effected by 0.025% (v/v) TEMED and 0.025% (w/v) ammonium persulfate. The 
stacking gel is 3% acrylamide with a buffer system of 0.125 M TrisHCl pH 6.8 and 
0.1% SDS and polymerized in the same manner.  The electrode buffer is 0.025 M 
Tris, 0.192 M glycine, and 0.1% SDS (pH 8.3).  A sample buffer has final 
concentrations of 0.0625 M Tris HCl pH 6.8, 2% SDS, 10% glycerol, and 5% 
2-mercaptoethanol, and 0.001% bromophenol blue.  Dissociation occurs with 1.5 
min in boiling water.  
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Schotz MC, Garfinkel AS, Huebotter RJ, Stewart JE
 </span>
 <span class="year">
(1970)
 </span>
 <span class="jname">
J. Lipid Res.
 </span>
 <span class="jvol">
11,
 </span>
 <span class="pages">
68-69.
 </span>
<p class="reftitl">
A Rapid Assay for Lipoprotein Lipase
<p class="Para1">
Assays of lipoprotein lipase activity have included use of naturally
occurring chylomicrons or commercial fat emulsions (Ediol, Intralipid,
Lipomul).  Standardization of the preparation is difficult for substrate
preparation.  Greten et al. recently described used of a radioactive
triglyceride and this paper reports a modification of it.
<p>
[<sup>12</sup>C]-triolein (purity &gt; 99%) and [<sup>14</sup>C]triolein
(84 mCi/mmol) were obtained from commercial suppliers.  TLC was used to
verify radioactive purity; lipid spots are visualized by iodine and rhodamine
G.  The scintillant used here is Instagel with nonionic surfactant.
Enzyme is prepared as a 25 mM ammonium hydroxide extract of rat adipose
tissue.
<p>
Blood from dog fasted overnight was allowed to clot for 1 h at room temp
and the serum obtained.  1 &micro;Ci of [<sup>14</sup>C]triolein and
133 mg of [<sup>12</sup>C]triolein were mixed and dried under nitrogen.
To this was added 3 ml serum, 0.9 ml 1% BSA (which had been adjusted to
pH 8.0), 0.9 ml 1% Triton X-100, and 7.2 ml TrisHCl pH 8.0.  The mixture
was sonicated on an ice-water bath for 4 min max power (125 W).  This
makes for 15 assay samples.  0.2 ml of enzyme extract is added to 0.8 ml
of the emulsion and incubated 20 min at 37&deg;.  At end of reaction
4 ml of isopropanol:3 N sulfuric acid (40:1) is added.  2 ml of water and
5 ml hexane is then added and tubes shaken end-to-end for 1 min.  After
20 min, 5 ml of the hexane phase is added to 1 ml of 0.1 N KOH.  This
is now shaken 10 min.  As the volume of the lower phase increases, its
measure is determined.  1 ml of the KOH phase is dissolved into 5 ml
Instagel and radioactivity counted.
<!-- ================================================================ -->
<p class="tags">

<p class="ref">
 <span class="authors">
P. van der Werf &amp; A. Meister
 </span>
 <span class="year">
(197?)
 </span>
 <span class="jname">
Advances in Enzymology
 </span>
 <span class="jvol">
??,
 </span>
 <span class="pages">
519-556.
 </span>
<p class="reftitl">
The Metabolic Formation and Utilization of 5-Oxo-L-Proline (L-Pyroglutamate,
L-Pyrrolidone Carboxylate)
<p class="Para1">
Haitinger in 1882 showed that glutamic acid heated to 180-190&deg;
dehydrates to pyroglutamic acid.  Mennozzi &amp; Appiani in 1894
later reported its molecular structure as the cyclic lactam of glutamate,
5-oxoproline (5-OP).  It has no primary amino group and so is colorless in
ninhydrin.
<p>
Woodward and Reinhart (1942) reported what they considered an enzymatically
controlled breakdown of glutathione by rat kidney extracts, one of the
products being 5-OP.  Other products includes Gly, Cys, and Glu, and the
proportion of Glu and 5-OP formed depended on the pH (greater Glu at the
lower pH).  Others found that GSH breaks down in two steps, first to
the dipeptide Cys-Gly and a &gamma;-glutamyl amino acid as a result of
transpeptidyl reaction involving the &gamma;-glutamyl targeted to an
acceptor amino acid.  Connell &amp; Hanes (1956) found a pig liver
enzyme, &gamma;-glutamyl lactamase (&gamma;-glutamylcyclotransferase),
which uses substrates &gamma;-Glu-Glu, &gamma;-Glu-Phe, and GSH to
form a 5-OP product, and the corresponding amino acids.  This enzyme
is present in a variety of tissues.
<p>
The first step in GSH catabolism is transpeptidation of GSH to an
L-amino acid by &gamma;-glutamyltranspeptidase to form
L-&gamma;-glutamyl-L-amino acid and cysteinyl-glycine, then
the action of &gamma;-glutamylcyclotransferase on the first product
of the first reaction to produce 5-OP and the amino acid.  Note that
the amino acid is not really consumed in the reaction, but just an
intermediate acceptor providing the appropriate reactivity.
<p>
The kidney contains considerable amounts of both enzymes, although
this tissue has high levels of GSH and also the enzymes
(&gamma;-glutamylcysteine and glutathione synthetases) that synthesize
GSH.
<p>
Mice and rats fed 5-oxo-L-[<sup>14</sup>C]-proline rapidly produce
respiratory <sup>14</sup>CO<sub>2</sub>.  This was eventually traced
to an ATP-mediated hydrolysis controlled by the enzyme 5-oxoprolinase.
The lactam ring is opened on 5-OP with the requirement of ATP to produce
glutamate, with decarboxylation of &alpha;-carboxyl group.  A
<b>&gamma;-glutamyl cycle</b> was discovered and is thought to
be involved in an amino acid transport mechanism between the kidney and
other tissues.
<p>
The enzyme-mediated cyclization of &gamma;-glutamylamino acids is
essentially the assisted reaction of the non-enzymatic transamidation
of glutamine, but the substrate of the cyclotransferase is a substituted
amide of glutamine.  Tandem &gamma;-glutamyl (i.e.,
&gamma;-glutamyl-&gamma;-glutamylamino acids) are synthesized from
two &gamma;-glutamylamino acids by the &gamma;-glutamyltranspeptidase.
The cyclotransferase then produces the 5-OP and &gamma;-glutamylamino acid.
The cyclotransferase is found in many tissues:  testes, brain, heart,
liver, spleen, skeletal muscle, intestine, lung, thymus, thyroid, adrenal,
ciliary body, lens, erythrocytes, and is abundant in skin and kidney.
The transpeptidase is a membrane-bound glycoprotein while the cyclotransferase
is soluble without carbohydrate.  The 27 kDa rat liver cyclotransferase
prefers &gamma;-glutamyl derivatives of Gln, Cys, Ala, Gly,
&alpha;-aminobutyrate, and Met to other amino acids.  It works well
on model substrates &gamma;-Glu-&gamma;-Glu-<i>p</i>-nitroanilide
and &gamma;-Glu-&gamma;-Glu-&alpha;-naphthylamide.  The preparation of
the enzyme yields what appear to be two isoforms, one with broader
substrate specificity;  this has now been attributed to limited
proteolysis of the enzyme, with the result that the proteolyzed enzyme
shows less stringent specificity.  It remains possible that cleaved
enzyme is also physiologically signficant.
<p>
The formation of 5-oxoprolyl residues at the N-terminus of peptides/proteins
is well known.  Since the number of proteins present is unlikely to
contribute quantitatively to the metabolism of the &gamma;-glutamyl cycle
described above, there is nonetheless a role that N-terminal pyroglutamyl
proteins could play.  5-OP cannot incorporate into tRNA directly,
and it is possible that pyroglutamyl forms arise from N-terminal
glutamyl or glutaminyl or the aminoacyl tRNA forms.  N-terminal 5-OP
can arise from Glu and not just Gln;  this is easily achieved by well
known phosphotransferase (kinase) reactions to activate the &gamma;-carboxyl
side chain, and then cyclization is energetically favorable.  This
is especially significant in view of the finding that
a &gamma;-glutamylphosphate-tRNA<sup>GLN</sup> has been found as an
intermediate in the synthesis of &gamma;-glutamyl-tRNA<sup>GLN</sup>
to &gamma;-glutaminyl-tRNA<sup>GLN</sup>.
<p>
Good quantitative determinations of tissue levels of 5-OP help
in developing hypotheses regarding metabolism of the &gamma;-glutamyl
cycle.  Samples are best acidified and extracted with ethyl acetate.
Alternatively homogenized and deproteinized samples can be passed
through Dowex 50 (H<sup>+</sup>), and the effluent treated with HCl
and heat to convert 5-OP to Glu for its determination.  One report
treats deproteinized tissue homogenates with picric acid which is
then passed sequentially through Dowex 2 (chloride) to remove
picric acid, then Dowex 50 (H<sup>+</sup>), and then the effluent
is concentrated, treated with 5-oxoprolinase, and the Glu quantitated.
Mouse kidney, liver, and brain were found to range in 5-OP levels
from 20-50 &mu;mol per kg tissue, although this range seems too
high and may be due to nonenzymatic contributions.  Serum and CSF
levels have been found to be 20 and 60 &mu;M, resp.  Excretion of
5-OP is quite variable (0.5-5 mg/day).  Very high levels have been
found in the skin, with 2-2.5% of the water-soluble material
(lactate is 3% by comparison).  The skin of those with psoriasis
appears to contain only 10% of the level found in those with healthy
skin.  Those with icthyosis vulgaris and pustulosis palmoplantarsis
show similar findings.  Those with burns and certain allergies appear
to excrete greater than normal amounts of urinary 5-OP.
<p>
Subcutaneous injections of racemic 5-OP into rabbit were used to
study how 5-OP was utilized.  All the 5-OP found in the urine
was 5-oxo-D-proline, and this suggested that only the L-form was
utilized.  Pigs fed racemic 5-OP excreted 5-OP that did not differ
in optical rotation with the starting material.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Davies, G. E., Stark, G. R.
 </span>
 <span class="year">
(1970)
 </span>
 <span class="jname">
Proc. Natl. Acad. Sci. USA
 </span>
 <span class="jvol">
66,
 </span>
 <span class="pages">
651-656.
 </span>
<p class="reftitl">
Use of dimethyl suberimidate, a cross-linking reagent, in studying
the subunit structure of oligomeric proteins
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Walsh, D. A., Ashby, C. D., Gonzalez, C., Calkins, D., Fischer, E., Krebs, E. G.
 </span>
 <span class="year">
(1971)
 </span>
 <span class="jname">
J. Biol. Chem.
 </span>
 <span class="jvol">
246,
 </span>
 <span class="pages">
1977-1985.
 </span>
<p class="reftitl">
Purification and characterization of a protein inhibitor of
adenosine-3',5'-monophosphate dependent protein kinase
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Aviv, H., Leder, P.
 </span>
 <span class="year">
(1972)
 </span>
 <span class="jname">
Proc. Natl. Acad. Sci. USA
 </span>
 <span class="jvol">
69,
 </span>
 <span class="pages">
1408-1412.
 </span>
<p class="reftitl">
Purification of biologically active globin messenger RNA by
chromatography on oligothymidylic acid-cellulose
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Palacios, R., Palmiter, R. D., Schimke, R. T.
 </span>
 <span class="year">
(1972)
 </span>
 <span class="jname">
J. Biol. Chem.
 </span>
 <span class="jvol">
247,
 </span>
 <span class="pages">
2316-2321.
 </span>
<p class="reftitl">
Identification and isolation of ovalbumin-synthesizing polysomes
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Aviv, H., Leder, P.
 </span>
 <span class="year">
(1972)
 </span>
 <span class="jname">
Proc. Natl. Acad. Sci. USA
 </span>
 <span class="jvol">
69,
 </span>
 <span class="pages">
1408-1412.
 </span>
<p class="reftitl">
Purification of biologically active globin messenger RNA by
chromatography on oligothymidilic acid-cellulose
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Schotz MC, Garfinkel AS
 </span>
 <span class="year">
(1972)
 </span>
 <span class="jname">
J. Lipid Res.
 </span>
 <span class="jvol">
13,
 </span>
 <span class="pages">
824-826.
 </span>
<p class="reftitl">
A Simple Lipase Assay Using Trichloroacetic Acid
<p class="Para1">
In developing this assay, glyceryl trioleate at &gt; 99% purity was used.
[2-<sup>3</sup>H]glyceryl trioleate (266 mCi/mmol) from Amersham/Searle,
fatty acid-poor BSA, Triton X-100, egg lecithin (Schwarz/Mann) were used.
The egg lecithin was purified in a water wash with CHCl<sub>3</sub>:MeOH
2:1 extraction.  Pasteur pipets are plugged with small amt of glass wool,
and assay done in a 13 x 100 mm glass tube.
<p>
Substrate is made by sonicating 270 mg glyceryl trioleate (containing 20
&micro;Ci of [2-<sup>3</sup>H]glyceryl trioleate), 16.2 mg egg lecithin,
3.0 ml of 10% fatty-acid poor BSA adjusted to pH 8.0, 1.6 ml 2 M TrisHCl
pH 8.2, and 1.4 ml water.  Sonication was done in a 18 &times; 100 mm
plastic tube immersed in ice-water bath.  The microprobe of the Branson
Model LS75 Sonifier is maximally tuned to no. 2 setting, with 3 min
duration, with 1 min on and 1 min off.  Separate batches of the 6 ml
mixture are prepared for multiple batches.  The emulsion is activated by
incubation at 37&deg; for 30 min with 12 ml serum taken from dogs fasted
overnight.  600 &micro;l aliquot of activated substrate incubated at 37&deg;
with enzyme preparations, with 1 ml total volume.  Samples are routinely
incubated for 60 min.
<p>
Reactions are terminated with 1 ml of 10% (w/v) trichloracetic acid.
The precipitate is sedimented by spinning at 1000 <i>g</i> for 5 min.
The supernatant contains glycerol product, and 1 ml is filtered through
a glass-wool plugged pasteur pipette (14.5 cm) is fitted to one hole of
a two-hole No. 2 stopper into a counting vial, which contains 50 &micro;l
of 5 N NaOH.  The solution is pushed through with a stream of nitrogen
for about 5 s.  An 8 ml mixture of 3 ml of Triton X-100 and 5 ml of 1%
PPO in toluene is added, then 5 ml of 0.4% PPO and 0.01% POPOP in toluene.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Verwoerd DW, Els HJ, De Villiers E-M, Huismans H
 </span>
 <span class="year">
(1972)
 </span>
 <span class="jname">
J. Virol.
 </span>
 <span class="jvol">
10,
 </span>
 <span class="pages">
783-794.
 </span>
<p class="reftitl">
Structure of the Bluetongue Virus Capsid
<p class="Para1">
The bluetongue virus (BTV) belongs to a group of viruses known to have
ds RNA genomes, a groups that includes a broad ranges of hosts
(mammals, insects, and plants).  With exception of the reovirus
all ds RNA viruses in this family are transmitted by arthropod vectors
or have insects as a primary host.  Reovirus as an example of the member
of this group with its 750 &Aring; diameter and double-layered capsid.
The outer layer has 92 capsomeres, but is likely to have a different
capsid structure with a trimeric or dimeric arrangement of subnits.
BTV has a reported 550 &Aring; diameter with a capsid of single layer
of 32 capsomeres.
<p>
There is no serological relationship between reovirus and BTV, and while
hemagglutination is seen in reovirus, there is none in BTV.  Reovirus
is stable to high temp and acidity but BTV is sensitive to both.
Two groups studying reovirus have shown that the virus has 7 proteins,
three forming the core particle.  Two of the four non-core proteins
is actually a proteolytic product of a primary translation product that
is processed later.  Two additional noncapsid polypeptides are found in
infected cells, and there appears to be strong correlation between the
count of genome segments (8) and these primary gene products.
<p>
Type 10 BTV and type 3 (Abney strain) reovirus virions were produced
using BHK-21 monolayers by methods described
previously.  The cells were washed with MEM without serum and then
treated with medium containing virus at 10-20 PFU/cell.  Infection was
allowed to proceed at 37&deg; for 36-48 h.  Cells were detached by
shaking and formed into a pellet, and stored at 4&deg; until purification.
<p>
For the viral purification, cells in roller bottles (100 count) were
washed in 72 ml of 2 mM TrisHCl pH 8.8 and then suspended in 150 ml
of the buffer by an Ultra-Turrax homogenizer.  A 1% Sephadex G200 suspension
was added at one-tenth of the volume, and then the mixture extracted
with 50 ml Freon 113 in cold room at 4&deg;.  Cell debris and Freon phase
were washed once with 70 ml Tris, and combined aqueous phases extracted
twice with Freon.  BSA was added to extract to final 0.5%, and then
10% Tween-80 added at one-tenth of the volume.  Now 0.5 volume of freshly
distilled ether was shaken in at cold temperature.  The aqueous phase
was adjusted to bring it to 0.1 M with NaCl.  Virus was now pelleted
through 40% sucrose in a Spinco 60Ti rotor at 40,000 rpm for 75 min.
Use of higher velocities tended to disrupt BTV.  Pellet was resuspended
in 2 mM TrisHCl buffer and layered into 10-30% sucrose gradient made
up in buffer.  Gradients were centrifuged 50 min at 24,000 rpm in Spinco
SW27 rotor.  The viral bands were collected and pelleted again at
40,000 rpm.
<p>
Isopycnic CsCl density gradient centrifugation was done for final purification
and selective degradation.   Preformed 40-55% gradients in 2 mM Tris buffer
were centrifuged at 47,000 rpm in Spinco SW50.1 rotor for 16 h.  Virus
bands were recovered and dialyzed against cold 2 mM Tris to remove CsCl.
<p>
Virus was also pulse-labeled by growth in BHK cell monolayers.  8 h after
infection in Eagle MEM, medium is changed to Earle salt containing
<sup>14</sup>C-protein hydrosylate at 0.1 &micro;Ci/ml.  After 2 h
medium replaced with MEM and cultures allowed to proceed (for 36 h).
Virus was purified, dissociated, and electrophoretically separated.
Bands were excised rom gel and dissolved in 0.2 ml 10% piperide then counted
in 8 ml Kinard's scintillation cocktail.  To dissociate virus, 10 mg/ml
protein in Tris was made 0.4% in SDS and 1% in 2-mercaptoethanol.  These
were heated 15 min at 37&deg;, then 1 min at 70&deg;, then 1 min at 96&deg;.
<p>
Type 10 BTV was purified, dissociated, and analyzed by PAGE.  Gels were
stained with Coomassie Brilliant Blue and scans done with a flying spot
densitometer.  The trace lines differ from reovirus, also analyzed.
Four major components make up both viruses but their MWs are different.
Virus was also purified on sucrose gradients; faint bands seen were variable
from run to run, indicating they are probably cellular contaminants.
<p>
BTV is sensitive to high salt concentrations, with a &ge;90% loss of
infectivity even after only 2 h in a CsCl gradient.  Gradients were done
both in unbuffered solutions and in pH-controlled solutions (pH 6, 7, 8).
At pH 8 a main band with density 1.38 g/ml was found, same as for infective
virus.  This band was seen to contain all seven peptides on PAGE, with
loss of the variable components.  Thus the faint bands are probably of
cellular origin.
<p>
Virus was visualized in EM and found to have two shells or layers which
were disrupted or altered at different pH.  The size of the RNA genome
segment was correlated with the size expected and size found the protein.
Genome segments are numbered from 1 to 10, 1 being largest in RNA size
and 10 being smallest.  Only 7 proteins were found for each virus.
BTV had major component proteins of 110, 101, 61, and 29.5 kDa
(corresponding to segments 2, 3, 6, and 8).  Minor components include
140, 82, and 42 kDa (corresponding to segments 1, 4, 7; kDa of last two are
uncertain).  Reovirus had as its major components 138, 132, 81, and
34.5 kDa (segments 1, 3, 5, 10; the 81 kDa assignment is uncertain), and
the minor components 89, 47.5, 40.5 kDa (segments 4, 7, and 8).
<p>
An attempt was made to estimate the count of proteins per virion using
radiolabeled bands cut out of PAGE gels.  The estimated count of proteins
per virion is: 140 kDa: 7, 110 kDa: 98, 101 kDa: 76, 82 kDa: 6,
61 kDa: 156, 42 kDa: 32, and 29 kDa: 570.
<p>
If one removes the outer capsid of reovirus with chymotrypsin or heat
treatment, an RNA polymerase activity is induced.  This outer layer
is composed of polypeptides 5 plus 4, 7, and 10 (or &mu;1, &mu;2,
&delta;1 and &delta;3 in Joklik's notation).  The inner core is
made of polypeptides 1, 2, and 8 (&lambda;1, &lambda;2, and &delta;2),
and the transcriptase is probably associated with either one or all
three of the inner core proteins.  Others have shown that chymotrypsin
and heat has no effect on BTV, nor do other proteases or detergents in
activating the enzyme.  To isolate activity, viral bands were separated
on CsCl gradients and incubated with RNA polyermase activity assay components.
In the pH 7 CsCl tube the greatest amount of activity was seen with the
1.42 g/ml band, and in the pH 6 CsCl tube, in the 1.39 g/ml band.
No activity was seen in control sucrose, nor in pH 8 CsCl 1.38 g/ml band
or in pH 7 CsCl 1.29 g/ml band.  This activity corresponds to loss of
polypeptides 2 and 6 or 2 only.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Knowles AF, Penefsky HS
 </span>
 <span class="year">
(1972)
 </span>
 <span class="jname">
J. Biol. Chem.
 </span>
 <span class="jvol">
247,
 </span>
 <span class="pages">
6617-6623.
 </span>
<p class="reftitl">
The subunit structure of beef heart mitochondrial adenosine triphosphatase.
Isolation procedures.
<p class="Para1">
The highlight is the use of techniques in organellar purification.
<p>
Minced muscle is extracted with 0.25 M sucrose, pH 7.3.
(Note sucrose maintains osmolarity to preserve particularly
mitochondrial integrity.)  The homogenate is centrifuged
for 10 min at 1600 tg.  The supernatant is removed and then
spun 10 min at 10,000 g.  This isolates mitochondria in the pellet.
<p> The pellet is suspended in sucrose and then subjected to about a total 4 min
of sonication (ultrasonic oscillations). The material is then spun for 90 min at
40,000 g. The submitochondrial pellet is not subjected to 10 min of sonication
in a solution of 4 mM ATP.  This is centrifuged another 90 min at 40,000 g.
<p> The supernatant is adjusted to 5.4, and then centrifuged after stirring.
The supernatant is adjusted to pH 8.0, then loaded on to DEAE-Sephadex.  After
washing, elution is done with a gradient of sodium sulfate
(Na<sub>2</sub>SO<sub>4</sub>). Fractions are pooled.  Protein is precipitated
with AmSO<sub>4</sub>, then refractionated using AmSO<sub>4</sub>.  Pure enzyme
is thus obtained.  Ultracentrifugation and native gel electrophoresis are the
best determinants of purity for this multimeric protein; SDS-PAGE is not likely
to indicate its purity.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Knowles AF, Penefsky HS
 </span>
 <span class="year">
(1972)
 </span>
 <span class="jname">
J. Biol. Chem.
 </span>
 <span class="jvol">
247,
 </span>
 <span class="pages">
6624-6630.
 </span>
<p class="reftitl">
The subunit structure of beef heart mitochondrial adenosine triphosphatase.
Physical and chemical properties of isolated subunits
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Warren, S., and Chute, R. N.
 </span>
 <span class="year">
(1972)
 </span>
 <span class="jname">
Cancer 
 </span>
 <span class="jvol">
29,
 </span>
 <span class="pages">
327-331.
 </span>
 <p class="reftitl">
Pheochromocytoma
<p class="Para1">
Gillman et al. (<i>Cancer</i> <b>6</b>: 494, 1953) have reported the occurrence 
of adrenal medullary tumors in old Wistar rats and indicated that high protein 
CHO-free diets reduced the frequency of tumors in the GG strain.  We now show a 
transplantable tumor which can kill the animal in 30-60 days which reaches 1.5 
cm dia (1 g weight).  A high incidence of these tumors occurs in elderly 
irradiated parabiotic New England Deaconess Hospital rats.  This tumor appeared 
as one of multiple neoplasms in the irradiated partner of a male parabiotic pair 
killed at 611 days postirradiation at 723 days of age.  There were no metasteses 
and the cells were pleomorphic, predominantly large with considerable amounts of 
cytoplasm.
<p>
The tumor takes readily when transplanted in untreated male as well as female 
young adult NEDH rats.  A tumor mass of about 1 &times; 1 &times; 3 mm is 
transplanted by trocar subcutaneously, usually interscapularly.  Many rats lose 
ability to grow and to accept being handled in a short period.  Blood pressure 
determinations in animals with decimated body weights and under ether anesthesia 
indicate a very elevated systolic pressure average of 167 mm Hg, well above a 
control range of 85-120 mm Hg.  Changes in blood sugar, urine volume, urine 
glucose, and urine pH were not significant in animals with well-grown tumors. 
Urine protein was increased 3- to 30-fold however, with hematuria.  Urinary 
vanilmandelic acid was well above normal levels, as were urinary metanephrines.
<!-- ============================================================= -->
<p class="ref" id="jovin">
 <span class="authors">
Jovin, T.
 </span>
 <span class="year">
(1973)
 </span>
 <span class="jname">
Biochemistry
 </span>
 <span class="jvol">
12,
 </span>
 <span class="pages">
871-878.
 </span>
 <p class="reftitl">
Multiphasic zone electrophoresis.  I.
Steady-state moving-boundary systems formed by different electrolyte
combinations
<p class="Para1">
This is a 3-part series of excellent articles.
<p>
Smithies (1955) introduced starch gel electrophoresis and
Raymond and Weintraub (1959) and Ornstein &amp; Davis (1962) introduced
polyacrylamide gels.  With the introduction of disc electrophoresis,
planned buffer discontinuities were exploited in an initial period which
served to concentrate sample and then resolve the proteins in a second
period/phase.  The theoretical explanation which follows can work in
rigid support media (such as starch or polyacrylamide) or in a free
system.  4269 systems are available on computer output.
<p>
The following variables/quantities are defined:
<p>
<i>C</i><i><sub>j</sub></i> =  concentration of constituent <i>j</i>.
It is a signed quantity, with the sign of the ionic subspecies of the
constituent.
<p>
<i>C</i><i><sub>j(i)</sub></i> = concentration of subspecies of
constituent <i>j</i> with valence <i>i. </i> It is signed.
<p>
<i>&mu;</i><i><sub>j</sub></i> = ion mobility of the ionic subspecies
of constituent j.  It is signed according to valence and has units of
cm<sup>2</sup> s<sup>-1</sup> V<sup>-1</sup>.
<p>
<i>r</i><i><sub>j</sub></i>= relative ion
mobility of the ion subspecies of constituent j, or the ratio of
&mu;<sub>j</sub> to the mobility of the Na<sup>+</sup> ion
determined at the same ionic strength and temperature.
It is signed and dimensionless.
<p>
<i>M</i><i><sub>j</sub></i> = constituent mobility of
constituent <i>j</i>, is signed and represents the average
mobility for the entire constituent.  Dimension as for &mu;<sub>j</sub>.
<p>
<i>R</i><i><sub>j</sub></i> = relative mobility of all species
of constituent <i>j</i>.
<p>
<i>K</i> = specific conductance (mhos cm<sup>-1</sup>).
<p>
<i>S</i> = relative conductance (C cm<sup>-3</sup>).
<p>
<i>N</i> = boundary displacement:  the volume swept by a moving
boundary during passage of one coulomb.  A signed
quantity with is + if the boundary moves in the same direction as the
current, i.e. toward the cathode.  A double superscript is used to
indicate the adjoining phases forming the moving boundary.
(cm<sup>3</sup> C<sup>-1</sup>).
<p>
<i>G/2</i> = ionic strength
<p>
<i>BV</i> = buffer value (Edsall and Wyman, 1958) (equiv. L<sup>-1</sup>).
<p>
<i>E</i> = electric field strength = -potential gradient (V cm<sup>-1</sup>).
<p>
<i>I</i> = current density = current/cross-sectional area (A cm<sup>-2</sup>).
<p>
<i>f</i><i><sub>j</sub></i> = fraction of constituent <i>j</i>
dissociated into ionic subspecies of greatest |valence|.
<p class="Para1">
The following formulae correspond to the definitions given above:
<p>
<i>C</i><i><sub>j</sub></i> = &Sigma;<sub><i>i</i>=1</sub><sup><i>n</i></sup>
<i>c</i><i><sub>j(i)</sub></i> where <i>n</i> = number of subspecies of
constituent <i>j</i>.

<table class="eqn" style="margin:0 3ex;padding:0;">
<tr>
 <td rowspan="2"><i>M<sub>j</sub></i> =
 <td>&Sigma;<sub><i>i</i>=1</sub><sup>n</sup> &mu;<sub><i>j</i>|<i>i</i>|</sub><i>c</i><sub><i>j</i>(<i>i</i>)</sub>
<tr>
 <td style="border-top:1px solid black;"><i>C<sub>j</sub></i>
</table>
<table class="eqn" style="margin:0 3ex;padding:0">
<tr>
 <td rowspan="2"><i>R<sub>j</sub></i> =
 <td>&Sigma;<sub><i>i</i>=1</sub><sup>n</sup> <i>r</i><sub><i>j</i>|<i>i</i>|</sub><i>c</i><sub><i>j</i>(<i>i</i>)</sub>
<tr>
 <td style="border-top:1px solid black;"><i>C<sub>j</sub></i>
</table>
<table class="eqn" style="margin:0 3ex;padding:0">
<tr>
 <td rowspan="2"><i>K</i> =
 <td><i>F</i>
 <td rowspan="2">&Sigma;<sub><i>i</i>=1</sub><sup><i>k</i></sup>
    &Sigma;<sub><i>i</i>=1</sub><sup>n</sup>
   &mu;<sub><i>j</i>|<i>i</i>|</sub><i>c</i><sub><i>j</i>(<i>i</i>)</sub>|<i>i</i>|
<tr>
 <td style="border-top:1px solid black;">1000
</table>
<table class="eqn" style="margin:0 3ex;padding:0">
<tr>
 <td><i>F</i>
 <td rowspan="2"> = 96.5
<tr>
 <td style="border-top:1px solid black;">1000
</table>
<p>
and <i>k</i> = number of constituents present in a phase
<p>
<i>S</i> = 96.5 &Sigma;<sub><i>i</i>=1</sub><sup><i>k</i></sup>
&Sigma;<sub><i>i</i>=1</sub><sup><i>n</i></sup>
<i>r</i><i><sub>j|i|</sub>c</i><i><sub>j(i)</sub></i>|<i>i</i>|
<p>
Therefore <i>K</i> ~ <i>S</i> &times; <i>&mu;</i><sub>Na</sub>+
<p>
<i>G/2</i> = 0.5 &times; &Sigma;<sub><i>i</i>=1</sub><sup><i>k</i></sup>
&Sigma;<sub><i>i</i>=1</sub><sup><i>n</i></sup>
|<i>c</i><i><sub>j(i)</sub></i>|<i>i</i><sup>
2</sup>
<p>
<i>BV</i> = ln(10) &times; &Sigma;<sub><i>i</i>=1</sub><sup><i>k</i></sup>
<i>f</i><i><sub>j</sub></i>(1-<i>f</i><i><sub>j</sub></i>)|<i>c</i><i><sub>
j</sub></i>|
<p>
<i>E</i> = <i>I / K</i>
<p>
Certain assumptions are made:
<ol>
<li>relative ion mobilities are constant
<li>uncharged species have zero mobility
<li>ion mobilities are additive
<li>activity coefficients for all species are = 1.
<li>there is a negligible contribution of H<sup>+</sup> and
   OH<sup>-</sup> to conductance for the pH range of 3 to 11.
<li>there is a constancy of apparent acid dissociation constants
  although values representative of the ionic strengths encountered
  can be selected
<li>the moving boundaries are ideal in steady state
<li>there is a negligible effect of the support medium on buffer
ion mobilities
<li>cross-sectional area is constant
<li>there is an absence of bulk flow of solvent
</ol>
<p class="Para1">
The Alberty equation for weak electrolytes applies:
<table class="eqn" style="margin:0 3ex;padding:0">
<tr>
 <td><i>M<sub>j</sub><sup>&alpha;</sup> C<sub>j</sub><sup>&alpha;</sup></i>
 <td rowspan="2">&minus;
 <td><i>M<sub>j</sub><sup>&beta;</sup> C<sub>j</sub><sup>&beta;</sup></i>
 <td rowspan="2"> = <i>N<sup>&alpha;&beta;</sup></i>
 	(<i>C<sub>j</sub><sup>&alpha;</sup></i>
     &minus; <i>C<sub>j</sub><sup>&beta;</sup></i>)
<tr>
 <td style="border-top:1px solid black;"><i>K<sup>&alpha;</sup></i>
 <td style="border-top:1px solid black;"><i>K<sup>&beta;</sup></i>
</table>
<table class="eqn" style="margin:0 3ex;padding:0">
<tr>
 <td><i>R<sub>j</sub><sup>&alpha;</sup> C<sub>j</sub><sup>&alpha;</sup></i>
 <td rowspan="2">&minus;
 <td><i>R<sub>j</sub><sup>&beta;</sup> C<sub>j</sub><sup>&beta;</sup></i>
<td rowspan="2"> = <i>N<sup>&alpha;&beta;</sup></i>(<i>C<sub>j</sub><sup>&alpha;</sup></i>
     &minus; <i>C<sub>j</sub><sup>&beta;</sup></i>) similarly applies
<tr>
 <td style="border-top:1px solid black;"><i>S<sup>&alpha;</sup></i>
 <td style="border-top:1px solid black;"><i>S<sup>&beta;</sup></i>
</table>
<p>
These apply to single constituent <i>j</i> in phases
<i>&alpha;</i> and <i>&beta;</i> separated by a moving boundary
with displacement <i>N</i><i><sup>&alpha;&beta;</sup></i>.
Conservation of mass is reflected in these statements.
<p>
In monovalent acid/base systems, Dismukes and
Alberty derived the function <i>Z</i> (the Kohlrausch regulating
function) for weak electrolytes:
<p>
<i>Z</i> = &Sigma;<sub><i>i</i>=1</sub><sup><i>k</i></sup>
  <i>C</i><i><sub>j</sub> / r</i><i><sub>j</sub></i>
<p>
This function has the same value in two phases
separated by a moving boundary and can be used to calculate the
composition of an unknown phase.  From the first 3 formulae above,
for monovalent acids and bases:
<p style="text-align:center;">
<i>M<sub>j</sub>C<sub>j</sub></i> =
&mu;<i><sub>j</sub>c<sub>j</sub></i><sub>(&plusmn;1)</sub> and
<i>R<sub>j</sub>C<sub>j</sub></i> =
<i>r<sub>j</sub>c<sub>j</sub></i><sub>(&plusmn;1)</sub>
<p>
We can restate the moving boundary equation above in new terms then:
<p>
<i>r<sub>j</sub>c<sub>j</sub></i><sub>(&plusmn;1)</sub><sup>&alpha;</sup> /
<i>S</i><sup>&alpha;</sup> &minus;
<i>r<sub>j</sub>c<sub>j</sub></i><sub>(&plusmn;1)</sub><sup>&beta;</sup> /
<i>S</i><sup>&beta;</sup> = <i>N</i><sup>&alpha;&beta;</sup>
(<i>C<sub>j</sub></i><sup>&alpha;</sup> &minus;
<i>C<sub>j</sub></i><sup>&beta;</sup>)
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Chou, P. Y., and Fasman, G. T.
 </span>
 <span class="year">
(1974)
 </span>
 <span class="jname">
Biochemistry
 </span>
 <span class="jvol">
13,
 </span>
 <span class="pages">
222-245.
 </span>
<p class="reftitl">
Prediction of protein conformation
<p class="Para1">
A model for the prediction of the secondary structure of globular proteins is
presented.  This model makes use of the <i>P</i><sub>&alpha;</sub> and
<i>P</i><sub>&beta;</sub> conformational parameters computed in the preceding
paper (Chou &amp; Fasman, Biochemistry 13:  211). This model makes use of
empirical rules for predicting the initiation and termination of helical and
&beta; sheet regions: when four helix formers out of six residues or three
&beta;-sheet formers out of five residues are clustered together, nucleation
commences and propagates in both directions until terminated by a sequence of
tetrapeptides designated as breakers.  This model was successful in locating 88%
of helical and 95% of &beta; sheet regions as well as correctly predicting 80%
of the helical and 86% of the &beta; sheet residues in 19 proteins evaluated.
<p>
Methods.  Analysis of helix-coil boundary residues and the central helix
residues in proteins indicate residues with the highest helical potential,
<i>P</i><sub>&alpha;</sub>, reside mostly at the helix center while the strong
helix breakers cluster just beyond the helix ends.  Helix nucleation starts at
these centers, propagates in both directions until the breakers terminate the
helical growth.  Frequent occurrence of negatively charged residues, proline
itself, at the amino terminal of the helix and positively charged residues at
the carboxyl terminal are used in locating helix boundaries.  Six classes of
potentials for both helix and &beta; sheet, P<sub>&alpha;</sub> and
P<sub>&beta;</sub>, with each potential group are designated as, for example, in
the helix formation group as H<sub>&alpha;</sub>, h<sub>&alpha;</sub>,
I<sub>&alpha;</sub>, i<sub>&alpha;</sub>, b<sub>&alpha;</sub>,
B<sub>&alpha;</sub>.  These are <i>H</i> = strong former, <i>h</i> = former,
<i>I</i> = weak former, <i>i</i>=indifferent, <i>b</i>=breaker, and
<i>B</i>=strong breaker, and these 6 classes also pertain to &beta; sheet
formation as well.  The table lists the potentials of each amino acid and the
class to which it belongs.
<table>
<caption>
Helix predictive potentials: (P<sub>&alpha;</sub>)
</caption>
<tr>
 <th colspan="2"><i>H</i><sub>&alpha;</sub>
 <th colspan="2"><i>h</i><sub>&alpha;</sub>
 <th colspan="2"><i>I</i><sub>&alpha;</sub>
 <th colspan="2"><i>i</i><sub>&alpha;</sub>
 <th colspan="2"><i>b</i><sub>&alpha;</sub>
 <th colspan="2"><i>B</i><sub>&alpha;</sub>
<tr>
 <td>Glu<sup>(-)</sup>
 <td>1.53
 <td>His<sup>(+)</sup>
 <td>1.24
 <td>Lys<sup>(+)</sup>
 <td>1.07
 <td>Asp
 <td>0.98
 <td>Asn
 <td>0.73
 <td>Pro
 <td> 0.59
<tr>
 <td>Ala
 <td>1.45
 <td>Met
 <td>1.20
 <td>Ile
 <td>1.00
 <td>Thr
 <td>0.82
 <td>Tyr
 <td>0.61
 <td>Gly
 <td>0.53
<tr>
 <td>Leu
 <td>1.34
 <td>Gln
 <td>1.17
 <td colspan="2">
 <td>Ser
 <td>0.79
 <td colspan="4">
<tr>
 <td colspan="2">
 <td>Trp
 <td>1.14
 <td colspan="2">
 <td>Arg<sup>(+)</sup>
 <td>0.79
 <td colspan="4">
<tr>
 <td colspan="2">
 <td>Val
 <td>1.14
 <td colspan="2">
 <td>Cys
 <td>0.77
 <td colspan="4">
<tr>
 <td colspan="2">
 <td>Phe
 <td>1.12
 <td colspan="8">
</table>
<table>
<caption>
&beta; sheet predictive potentials: (P<sub>&beta;</sub>)
</caption>
<tr>
 <th colspan="2"><i>H</i><sub>&beta;</sub>
 <th colspan="2"><i>h</i><sub>&beta;</sub>
 <th colspan="2"><i>I</i><sub>&beta;</sub>
 <th colspan="2"><i>i</i><sub>&beta;</sub>
 <th colspan="2"><i>b</i><sub>&beta;</sub>
 <th colspan="2"><i>B</i><sub>&beta;</sub>
<tr>
 <td>Met
 <td>1.67
 <td>Cys
 <td>1.30
 <td>Ala
 <td>0.97
 <td>Arg<sup>(+)</sup>
 <td>0.90
 <td>Lys<sup>(+)</sup>
 <td>0.74
 <td>Glu<sup>(-)</sup>
 <td>0.26
<tr>
 <td>Val
 <td>1.65
 <td>Tyr
 <td>1.29
 <td colspan="2">
 <td>Gly
 <td>0.81
 <td>Ser
 <td>0.72
 <td colspan="2">
<tr>
 <td>Ile
 <td>1.60
 <td>Phe
 <td>1.28
 <td colspan="2">
 <td>Asp<sup>(-)</sup>
 <td>0.80
 <td>His<sup>(+)</sup>
 <td>0.71
 <td colspan="2">
<tr>
 <td colspan="2">
 <td>Gln
 <td>1.23
 <td colspan="4">
 <td>Asn
 <td>0.65
 <td colspan="2">
<tr>
 <td colspan="2">
 <td>Leu
 <td>1.22
 <td colspan="4">
 <td>Pro
 <td>0.62
 <td colspan="2">
<tr>
 <td colspan="2">
 <td>Thr
 <td>1.20
 <td colspan="8">
<tr>
 <td colspan="2">
 <td>Trp
 <td>1.19
 <td colspan="8">
</table>
<p>
Secondary structure analysis can begin using the following procedures,
keeping in mind the limitations:
<ol style="list-style-type:upper-alpha;">
<li>Helical Region Search
 <ol>
 <li>Helix nucleation.  Locate clusters of 4 helical residues (class
 H<sub>&alpha;</sub> or h<sub>&alpha;</sub>) within a region of 6 in the chain.
 Weak residues (<i>I</i><sub>&alpha;</sub>) count as 0.5 h<sub>&alpha;</sub> or
 H<sub>&alpha;</sub>, meaning three helicals plus two I<sub>&alpha;</sub>
 residues within a 6-residue region meets the criterion.  Helix formation is
 unfavorable if the segment contains 1/3 or more breakers (b<sub>&alpha;</sub>
 or B<sub>&alpha;</sub>) or if there is less than 1/2 helix formers.
 <li> Helix termination.  The helical segment is then extended in both
 directions until terminated by any sequence of tetrapeptides where the
 &lang;<i>P</i><sub>&alpha;</sub>&rang; is &lt;1.00 (note that
 &lang;<i>x</i>&rang; is the average of <i>x</i>).  The following class sequence
 combinations automatically stop helix propagation: b<sub>4</sub>,
 b<sub>3</sub>h, b<sub>2</sub>i<sub>2</sub>, b<sub>2</sub>ih,
 b<sub>2</sub>h<sub>2</sub>, bi<sub>3</sub>, bi<sub>2</sub>h, bih<sub>2</sub>,
 and i<sub>4</sub>. Note that the lowercase notations also apply to I, B, and H
 as well.  Adjacent &beta; sheet regions can terminate the &alpha; helical
 regions.
 <li>Pro residues cannot occur in the inner helix or at the C-terminal helical
 end.
 <li>Helical boundaries.  Pro, Asp, Glu prefer the N-terminal helix end.  His,
 Lys, and Arg prefer the C-terminal end. I<sub>a</sub> assignments are given to
 Pro and Asp (near the N-terminal helix) and Arg (near the C-terminal of the
 helix if necessary to satisfy condition A-1. <br><b>Rule 1</b>: <i>any segment
 of six residues or longer in a native protein with
 </i>&lang;P<sub>&alpha;</sub>&rang; &gt;= 1.03 <i>as well as an average
 </i>&lang;P<sub>&alpha;</sub>&rang;&nbsp;&gt;&nbsp;&lang;P<sub>&beta;</sub>&rang;
<i>over the same region and satisfying conditions A-1 through A-4 is
 predicted as helical</i>.
 </ol>
<li>&beta; Sheet Region Search
 <ol>
 <li>Nucleation.  Locate clusters of 3 &beta; residues (H<sub>&beta;</sub> or
 h<sub>&beta;</sub>) out of five along the chain.  Formation is unfavorable if
 the segment contains &gt;= 1/3 breakers (b<sub>b</sub> or B<sub>b</sub>) or
 less than 1/2 formers.
 <li>Termination.  Apply condition A-2 for the termination of &beta; sheet
 propagation, substituting &alpha; where &beta; is and vice versa.
 <li>Glu occurs rarely in the &beta; region.  Pro occurs rarely in the inner
 &beta; region.
 <li>Boundaries.  Charged residues occur rarely at the N-terminal &beta;-sheet
 end, and infrequently in the inner &beta;-sheet region and C-terminal end.  Trp
 occurs mostly at the N-terminal &beta;-sheet end and rarely at the C-terminal
 end. <br><b>Rule 2</b>: any segment of five residues or longer in a native
 protein with average P<i><sub>&beta;</sub> &gt;= 1.05 </i>as well as an average
 P<i><sub>&beta;</sub>&nbsp;&gt;&nbsp;</i>
 &lang;<i>P</i><sub>&alpha;</sub>&rang; <i>over the same region and satisfying
 conditions B-1 through A-4 is predicted as &beta;-sheet.</i>
 </ol>
</ol>
<p> RESULTS.  Using this predictive algorithm, 15 proteins whose X-ray crystal
structures have been determined and secondary structures assigned are tabulated
in comparison with the model's prediction of the residue range for either
helical or &beta;-sheet content, and the &lang;<i>P</i><sub>&alpha;</sub>&rang;
and &lang;<i>P</i><sub>&beta;</sub>&rang; are given for the range. 70 of 81
&alpha; regions (86%) as well as 64 of 66 &beta; regions (97%) are correctly
localized. Of the 70 helices predicted, 112 of the 140 helical boundaries are
defined with an accuracy of &plusmn;2 residues. 10 helical ends are predicted
incorrectly with more than &plusmn;4 residue difference.
<p> In analysis the correctness of the prediction, certain factors are used. The
percentage of correctly predicted residues in any conformational state <i>k</i>
is given by the equation %<sub><i>k</i></sub> = 100&nbsp;(<i>n<sub>k</sub></i>
&minus; count incorrect) / <i>n<sub>k</sub></i>, where <i>k</i> represents the
&alpha;, &beta;, or coil regions in the native structure as shown by X-ray
analysis.  The percentage of <em>total</em> residues would be for all correctly
predicted residues for all possible conformational states: %<sub><i>N</i></sub>
= 100&nbsp;(<i>N</i> &minus; total incorrect) / <i>N</i>.
<p> The analysis works for small proteins as well as large ones.  Bovine
pancreatic trypsin inhibitor has only 58 residues.  Under the amino acid
sequence, the &alpha; or &beta; potential class value is recorded, which
provides a quick glimpse of where the potential nucleation sites are.  This also
shows how the rules work:
<table id="two-d-struct">
<tr>
 <td>
 <td>1
 <td>2
 <td>3
 <td>4
 <td>5
 <td>6
 <td>7
 <td>8
 <td>9
 <td>10
 <td>11
 <td>12
 <td>13
 <td>14
 <td>15
<tr>
 <td>
 <td>Arg
 <td>Pro
 <td>Asp
 <td>(<span class="underline">Phe</span>)
 <td><span class="underline">Cys</span>
 <td>(<span class="underline">Leu</span>)
 <td>(Glu)
 <td>Pro
 <td>Pro
 <td><span class="underline">Tyr</span>
 <td><span class="underline">Thr</span>
 <td>Gly
 <td>Pro
 <td><span class="underline">Cys</span>
 <td>Lys
<tr>
 <td>&alpha;
 <td>i
 <td>[ I&nbsp;&nbsp;
 <td>I
 <td>h
 <td>i
 <td>H
 <td>&nbsp;&nbsp;H ]
 <td>B
 <td>B
 <td>b
 <td>i
 <td>B
 <td>B
 <td>i
 <td>I
<tr>
 <td>&beta;
 <td>i
 <td>b
 <td>i
 <td>h
 <td>h
 <td>h
 <td>B
 <td>b
 <td>b
 <td>h
 <td>h
 <td>i
 <td>b
 <td>h
 <td>b
<tr>
 <td colspan="16">&nbsp;
<tr>
 <td>
 <td>16
 <td>17
 <td>18
 <td>19
 <td>20
 <td>21
 <td>22
 <td>23
 <td>24
 <td>25
 <td>26
 <td>27
 <td>28
 <td>29
 <td>30
<tr>
 <td>
 <td>(Ala)
 <td>Arg
 <td><span class="underline">Ile</span>
 <td><span class="underline">Ile</span>
 <td>Arg
 <td><span class="underline">Tyr</span>
 <td>(<span class="underline">Phe</span>)
 <td><span class="underline">Tyr</span>
 <td>Asn
 <td>(Ala)
 <td>Lys
 <td>(Ala)
 <td>Gly
 <td>(<span class="underline">Leu</span>)
 <td><span class="underline">Cys</span>
<tr>
 <td>&alpha;
 <td>H
 <td>i
 <td>I
 <td>I
 <td>i
 <td>b
 <td>h
 <td>b
 <td>b
 <td>H
 <td>I
 <td>H
 <td>B
 <td>H
 <td>i
<tr>
 <td>&beta;
 <td>[ I&nbsp;&nbsp;
 <td>i
 <td>H
 <td>H
 <td>i
 <td>h
 <td>h
 <td>&nbsp;&nbsp;h ]
 <td>b
 <td>I
 <td>b
 <td>[ I&nbsp;&nbsp;
 <td>i
 <td>h
 <td>h
<tr>
 <td colspan="16">&nbsp;
<tr>
 <td>
 <td>31
 <td>32
 <td>33
 <td>34
 <td>35
 <td>36
 <td>37
 <td>38
 <td>39
 <td>40
 <td>41
 <td>42
 <td>43
 <td>44
 <td>45
<tr>
 <td>
 <td>(<span class="underline">Gln</span>)
 <td><span class="underline">Thr</span>
 <td>(<span class="underline">Phe</span>)
 <td>(<span class="underline">Val</span>)
 <td><span class="underline">Tyr</span>
 <td>Gly
 <td>Gly
 <td><span class="underline">Cys</span>
 <td>Arg
 <td>(Ala)
 <td>Lys
 <td>Arg
 <td>Asn
 <td>Asn
 <td>(<span class="underline">Phe</span>)
<tr>
 <td>&alpha;
 <td>h
 <td>i
 <td>h
 <td>h
 <td>b
 <td>B
 <td>B
 <td>i
 <td>i
 <td>H
 <td>I
 <td>i
 <td>b
 <td>b
 <td>[ h&nbsp;&nbsp;
<tr>
 <td>&beta;
 <td>h
 <td>h
 <td>h
 <td>H
 <td>h
 <td>i
 <td>i
 <td>&nbsp;&nbsp;h ]
 <td>i
 <td>i
 <td>b
 <td>i
 <td>b
 <td>b
 <td>h
<tr>
 <td colspan="16">&nbsp;
<tr>
 <td>
 <td>46
 <td>47
 <td>48
 <td>49
 <td>50
 <td>51
 <td>52
 <td>53
 <td>54
 <td>55
 <td>56
 <td>57
 <td>58
 <td colspan="2">
<tr>
 <td>
 <td>Lys
 <td>Ser
 <td>(Ala)
 <td>(Glu)
 <td>Asp
 <td><span class="underline">Cys</span>
 <td>(<span class="underline">Met</span>)
 <td>Arg
 <td><span class="underline">Thr</span>
 <td><span class="underline">Cys</span>
 <td>Gly
 <td>Gly
 <td>(Ala)
 <td colspan="2">
<tr>
 <td>&alpha;
 <td>I
 <td>i
 <td>H
 <td>H
 <td>i
 <td>i
 <td>h
 <td>I
 <td>&nbsp;&nbsp;i ]
 <td>i
 <td>B
 <td>B
 <td>H
 <td colspan="2">
<tr>
 <td>&beta;
 <td>b
 <td>b
 <td>I
 <td>h
 <td>i
 <td>h
 <td>H
 <td>i
 <td>h
 <td>h
 <td>i
 <td>i
 <td>I
 <td colspan="2">
</table>
<p> Predicted secondary structure is defined by brackets.  Helical residues are
enclosed in parentheses, while &beta;-sheet residues are underlined.
<p> Note that residues 4, 6, and 7 are assigned h<sub>a</sub> according to the
table, but that residues 2 and 3 are assigned I<sub>a</sub> according to
condition A-4.  This way helix initiation is possible (A-1).  Arg-1 is not
included in the helix since it prefers the C-terminal part of the helix.  Pro-8
is not &alpha; since it cannot occur at the C-terminal helix (A-3). Furthermore
the tetrapeptide 8-11 is a helix-breaker (BBbi)<sub>&alpha;</sub> under A-2.
Since 3 b-formers are present, the P<sub>a</sub> and P<sub>b</sub> average over
the bracketed helical region must be calculated and compared.  Since the average
P<sub>a</sub>&nbsp;&gt;&nbsp;1.03 and P<sub>a</sub>&nbsp;&gt;&nbsp;P<sub>b</sub>
over residues 2-7, rule 1 is obeyed and the region is predicted to be a helical.
It is also noted that the grouping of &alpha; and &beta; assignments shows the
2-7 region has two strong &alpha; formers and no &alpha; breakers
(H<sub>2</sub>hI<sub>2</sub>i)<sub>&alpha;</sub>, while no strong &beta;
breakers and two &beta; breakers are present
(h<sub>3</sub>ibB)<sub>&beta;</sub>.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Heil A, M&uuml;ller G, Noda L, Pinder T, Schirmer H, Schirmer I, von Zabern I
 </span>
 <span class="year">
(1974)
 </span>
 <span class="jname">
 Eur. J. Biochem.
 </span>
 <span class="jvol">
43,
 </span>
 <span class="pages">
131-44.
 </span>
<p class="reftitl">
The amino-acid sequence of sarcine adenylate kinase from skeletal muscle
<p class="Para1">
Protein of M<sub>r</sub> = 21,000, catalyzing Mg<sup>2+</sup>-dependent
reaction:
<p style="text-align:center;">
                ATP + AMP &larr;&rarr; 2 ADP
<p> Muscle is minced, extracted in 0.01 M KCl using standard devices. Filtered
through cheesecloth to remove chunks, the pH of extract is adjusted to 3.5 and
stirred for 5 min.  The pH is then adjust to 7.0, and the solution is
centrifuged.
<p> The supernatant is passed over phosphocellulose and washed free of unbound
material.  Elution is by pulsing with a buffer containing 5 mM AMP.  Fractions
containing activity are pooled. The pooled fractions are precipitated with
AmSO<sub>4</sub>. The recovered pellet is resuspended in a minimal volume and
loaded on to a Sephadex G-75 column (fractionation range between 3000-70000).
This column is particularly valuable in that it will remove a 60 kDa
contaminant.
<p> Fractions from the G-75 column are pooled.  Ammonium sulfate is slowly
brought to 62% saturation.  After a time, the adenylate kinase crystallizes.
The purity of enzyme produced by this method can be evaluated by
ultracentrifugation or gel electrophoresis (see Schirmer et al., FEBS Lett. 10:
333, 1970).
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Krauss RM, Levy RI, Fredrickson DS
 </span>
 <span class="year">
(1974)
 </span>
 <span class="jname">
J. Clin. Invest.
 </span>
 <span class="jvol">
54,
 </span>
 <span class="pages">
1107-1124.
 </span>
<p class="reftitl">
Selective Measurement of Two Lipase Activities in Postheparin Plasma
from Normal Subjects and Patients with Hyperlipoproteinemia
<p class="Para1">
<b>NOTE</b>. The only interest in this article is its described method
of doing lipoprotein lipase assays.
<p>
Blood is collected in tubes with 2 U heparin/ml whole blood, kept at
4&deg;.  Samples are obtained before and 10 min after rapid i.v.
injection of Na heparin at 10 U/kg for standard test and 100 U/kg
in special test.  Plasma was collected in refrigerated IEC centrifuge,
frozen in dry ice in 0.5 ml aliquots, stored at &minus;20&deg;.
<p>
10-100 &micro;l of plasma or tissue extract in incubated in a
100 &micro;l total volume of 194 mM TrisHCl pH 8.6 with 0.1 M NaCl
with 1.5 U heparin for 10 min at 27&deg;.  In another set of tubes,
3 mg protamine sulfate in the 100&micro;l is used.  900 &micro;l
of substrate emulsion is then added.  The substrate emulsion is
5 &micro;Ci glyceryl-tri-[1-<sup>14</sup>C]oleate at 39-41 mCi/mmol,
113 &micro;mol (100 mg) of triolein, 200 mg fatty acid-free BSA,
600 &micro;l of 1% (v/v) Triton X-100 in 12 ml Tris-saline final volume,
and sonicated not less than 30 min before use.  In sonicating, insert the
tip about 9-10 mm below surface, put in 25 ml scintillation vial and
suspended in ice-water bath, 1 min at setting 5 (60 W), 15 sec pause,
another 30 s, increasing to setting 7 at the last 5 s.  Note that
12 ml will produce 13 assays, and that 375 nCi (about 833,000 dpm)
is added.  Incubations are at 27&deg; for 60 min.  The Schotz et al
method of extraction is used.  The [<sup>14</sup>C]FFA is counted in
Packard Instagel LSC.  Release of FFA is determined after correcting
for efficiency of extraction and quenching.
<!-- ============================================================= -->
<p class="ref">
 <span class="authors">
Pardee, A. 
 </span>
 <span class="year">
(1974)
 </span>
 <span class="jname">
Proc. Natl. Acad. Sci. USA
 </span>
 <span class="jvol">
71,
 </span>
 <span class="pages">
1286-1290.
 </span>

<p class="reftitl">
A restriction point for control of normal animal cell proliferation
<p class="Para1">
The cell cycle occurs in four major phases (G<sub>1</sub>, S, G<sub>2</sub>, and 
M).<span class="c9">&nbsp;</span> There is some disagreement as to whether there 
is a G<sub>0</sub>, or quiescent phase, since some people would rather argue 
that cell proliferation is just stopped in G<sub>1</sub>. Results of this study 
are intended to determine populations of cells in G<sub>1</sub> and how close 
they may be to the end of M or the beginning of S phases.

<p>
Cells in an exponentially growing culture are
stopped by starvation of essential nutrients, then complete medium added
and the kinetics of [<sup>3</sup>H]thymidine followed.
Starvation of isoleucine, glutamine, or serum was then
followed by 8 hr interval in which cells returned to complete medium and
the resumption of an active thymidine incorporation (DNA synthesis).
A three-paneled figure showing how serum, Ile,
and Gln starvation of BHK (Syrian hamster BHK21/C3) cells shows the
resumption of DNA synthesis after 8 hr and a steady increase that does not
change.  Control cells (cells continued in
deficient-medium) never showed an enhanced rate.
This 8 hr interval has been similarly observed when
there was prestarvation for phosphate, or when cAMP levels were increased
as a result of treatment with (1) 0.05 mM papaverine, (2) 1.0 mM dibutyryl
cAMP + 2.0 mM theophylline, or (3) 30 &mu;g/ml prostaglandin E1 + 1 mM
theophylline.  Times of starvation really did
not affect the consistent time interval seen between restoration of growth
conditions and the initiation of DNA synthesis.
<p>
One question not answered to satisfaction is how
the individual cells react.  Variability in
the length of G<sub>1</sub> is still possible, since such results only
measure the subpopulation of cells which are involved in the earliest
initiation of synthesis.  Are not all cells
really in the same state? 
Cytoautoradiographic evidence indicates that many cells are active in
synthesis whereas others show no activity.
<p>
It is possible to use cell proliferation blocks
which act at different points in G<sub>1</sub>.  Imposing a block at a late point should block all
cells, then shifting to a block at an early point will continue to block
some cells, but if cells are truly variable as to their state, will allow
some cells to proceed into S phase.  In this
experiment growing BHK cells are exposed to the first block for ~two days
then transferred to the second block or into complete medium
(control).  Seven first block conditions (no
Ile, no Gln, low serum, Bt<sub>2</sub>cAMP+theophylline, papaverine,
hydroxyurea, colchicine) were tried and eight second block conditions (no
Ile, no Gln, low serum, papaverine, PGE<sub>1</sub>+theophylline,
hydroxyurea, colchicine) followed for each of the first block except in
some cases.  For thymidine incorporation
rates &lt;20% of controls, the second block is judged successful in
preventing DNA synthesis.  Values of ~60%
would be expected if cells passed through a single round of DNA synthesis
and then stopped since in the control, cells have initiated a second cycle
of DNA synthesis.  Results show that
hydroxyurea as first block does not prevent reinitiation of DNA synthesis
for any second blocking condition.  Similar
results are seen for colchicine.  Hence these
agents block the cell cycle at a later point than the other blocking
agents.
<p>
The effect of high cell density on growth
inhibition ("contact inhibition") was similarly analyzed using Nil 8 cells
since they are more sensitive to density-dependent growth arrest.
Nil 8 cells which had stopped growing because of
confluence were replated at low density in complete medium and thymidine
incorp. measured after 20 hr.  Cells were
also replated in medium absent of Ile or Gln or in medium having low
serum.  Results show incorp. was active in
beginning between 20-30 hr, peaking at 60 hr.
Cells in deficient medium never got started.<span
Hence the latter inhibitory conditions block
cells at the same point or later than does contact inhibition.
<p>
To show that quiescent cells (contact-inhibited)
were not weakened sufficiently to resume any proliferative cycle, cells
were transferred into complete medium containing 5 &mu;g/ml colchicine or 1
mM hydroxyurea for 9 hr (both are toxic) and BHK cells were seen to
rapidly synthesize DNA.
<p>
Other experiments measure the restriction point in
a more refined manner and require three sequential treatments.
A block is first applied to force cells into
quiescence, then a second block is applied for about 9 hr and allows for
escape from first block, if any.  Cells are
finally placed in complete medium and thymidine incorporation
measured.  A control set of cells does not
use the second block and allow escape.  In
the experiment, the first block used was low serum and the second block
was any of 11 different blocking conditions. 
Only the hydroxyurea and colchicine blocks allowed escape of first-blocked
cells as expected from the double block studies above.
<p>
In another experiment, cells are blocked first with
either no Ile or low serum, a variety of second block conditions applied
for ~9 hr, then cells returned to either low serum or no Ile and thymidine
incorp measured.  If escape occurs in the
middle period prior to restoring the first block conditions, DNA synthesis
will be observed.  The data shows very little
DNA is made, but there is a difference between the middle block using
hydroxyurea and those of the other agents, where hydroxyurea allows some
synthesis.  In general, a greater DNA
synthesis is seen with low serum/X/low serum, since it has an incomplete
blocking effect.
<p>
Cells were exposed to a block, then a variable time
interval in complete medium, then to block again, although conditions for
the second block may not be the same.  Cells
in low serum followed by complete medium and then low serum are able to
synthesize DNA.  Cells were made quiescent in
no-Ile medium, given complete medium for various time intervals, then
given [<sup>3</sup>H]thymidine-containing medium which had either no Ile
or no Gln.  Results show that cells able to
escape their Ile requirement and undergo a round of DNA replication
similarly escaped their Gln requirement.  The
kinetics of escape parallels also the autoradiographic evidence in which
the percentage of individual cells undergoing mitoses relative to the
whole population was also assessed.
<p>
Note that all of the experiments above show that
the cells exist in a quiescient state which are at the same point between
start of S and end of M phase, i.e. the R-point (restriction point) is the
same is the same.
<p>
But are there other points that exist?
Colchicine and hydroxyurea are nonphysiological
blockers stopping cells in M and S phases, respectively.
Cells die after 24 hr of treatment and do not
incorporate thymidine.  DNA synthesis has
been seen to proceed in the presence of colchicine and complete absence of
Ile.  Hydroxyurea as a block is more
complicated since thymidine incorporation is the measure here and it is a
synthetic block:  nonetheless cells in
induced quiescence were transferred to 1 mM hydroxyurea for 9 hr in
complete medium, then transferred into complete or deficient medium w/o
hydroxyurea and immediately began thymidine incorporation, showing they
had escaped G<sub>1</sub> block.  Cells moved
from low hydroxyurea to nutrient-deprived or high cAMP conditions
incorporate thymidine rapidly as well, indicating they are past the block
points induced by those conditions as well.
<p>
Transformed cells were similarly investigated.
BHK cells transformed by SV40 virus and made
quiescent by Ile or Gln starvation.  After 40
hr of starvation, cells could not recover their ability to incorporate
thymidine when restored to complete medium. 
Hence transformed cells show a fundamentally different response to
nutritional deficiency than do normal cells.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Yavin, E., Yavin, Z.
 </span>
 <span class="year">
(1974)
 </span>
 <span class="jname">
J. Cell Biol.
 </span>
 <span class="jvol">
62,
 </span>
 <span class="pages">
540-546.
 </span>
<p class="reftitl">
Attachment and culture of dissociated cells from rat embryo cerebral hemispheres 
on polylysine-coated [glass] surface
<p class="Para1">
Rat brain cells obtained from the dissociation by 5 min. trypsinization of 
cerebral hemispheres from 16 to 18-day old rat embryos showed nearly 85-90% 
adhesion after only 20 min incubation to poly-L-lysine coated surfaces (avg. 
mol. wgt. 400,000).  Cells not attaching within this time did not attach even 
after 24 hr.  There was no detachment of attached cells when treated with medium 
containing 5 mM EDTA, which was observed in attached cells plated to uncoated 
Petri dishes.  Detachment was only achieved with limited trypsination.
<p>
The effect of temperature was assessed:  at 4&deg; only 20% of cells attached 
after 20 min, whereas at ambient temperature or at 37&deg; more than 70% 
attached.  A plot of nonadhered cells vs. temperature reveals a transition point 
at which adhesion rate has two rates.  An Arrhenius plot (nonadhered cells vs. 
inverse temperature) clearly reveals the breakpoint of rate difference.
<p>
Respiration inhibitors and uncouplers (2,4-DNP, CN-, NaF), thiol binding agents 
(PCMB, NEM), and Ca<sup>2+</sup> chelators (EDTA) were assessed for effects on 
attachment. KCN, 2,4-DNP, and NaF had only a slight inhibitory effect on 
attachment, although pre-incubation of suspended cells in KCN and NEM caused 
nearly complete attachment inhibition.  Thiol binders (NEM and PCMB) markedly 
affected adhesion. Neither CaCl<sub>2</sub> nor EDTA affected cell adhesion, 
suggesting that attachment to poly-LYS is not mediated by divalent cation. 
Polymers of other amino acids were tested and not suitable for substrata 
(poly-ASP, -GLY, and -CYS).  D or L isomers of poly-LYS of various avg. mol. 
weights were very good as adhesive substrata.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Grunstein, M., Hogness, D. S.
 </span>
 <span class="year">
(1975)
 </span>
 <span class="jname">
Proc. Natl. Acad. Sci. USA
 </span>
 <span class="jvol">
72,
 </span>
 <span class="pages">
3961-3965.
 </span>
<p class="reftitl">
Colony hybridization:  a method for the isolation of cloned DNAs that
contain a specific gene
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Maniatis, T., Jeffrey, A., van deSande, H.
 </span>
 <span class="year">
(1975)
 </span>
 <span class="jname">
Biochemistry
 </span>
 <span class="jvol">
14,
 </span>
 <span class="pages">
3787-3794.
 </span>
<p class="reftitl">
Chain length determination of small double- and
single-stranded DNA molecules by polyacrylamide gel electrophoresis
<p class="Para1">
Restriction enzymes <i>Hin</i>II and <i>Hin</i>III, <i>Hpa</i>II, and 
<i>Hae</i>III were prepared.  T4 polynucleotide kinase was prepared.  A slab gel 
similar to the one prepared by Dewachter and Fiers (1971) was constructed as 20 
&times; 20 &times; 0.3 cm in a buffer of 90 mM Tris-borate, pH 8.3, and 5 mM 
MgCl<sub>2</sub> (TBM) or in 90 mM Tris-borate, pH 8.3, and 2.5 mM EDTA (TBE) 
(used by Peacock &amp; Dingman).  These were also the reservoir buffers.  The 
gel preparation formula is generally 3 ml of 1.6% ammonium persulfate + 7.5 ml 
of 10X buffer, and an appropriate volume of 30% acrylamide (29 g monomer and 1 g 
bis in 100 mL), with polymerization started with 25&nbsp;&mu;l in 75 ml total 
gel bed volume. EtOH-precipitated (to remove salts) samples are prepared in 0.1 
vols of 20% Na acetate, 20&nbsp;&mu;g yeast tRNA carrier, and then 2 vols 95% 
EtOH is added which is placed on a dry ice-EtOH bath for 15 min followed by 5 
min microfugation.  Ppts are rinsed 2 times with 70% EtOH, resuspended in sample 
solution (5% glycerol, 0.025% xylene cyanol FF, and 0.025% bromophenol blue). 
Presence of salts affects mobility and leads to inaccurate MW estimations. 
Electrophoresis is at RT at 200 V constant (10 V/cm).  Denaturing gels contain 7 
M urea.  Larger DNA fragments were analysed on 5% gels containing 98% formamide 
(deionized).
<p>
Results are plotted as log #base pairs of duplex DNA vs. relative mobility on 
the gels in relation to xylene cyanol FF migration.  Gels of 7, 10, 12 and 15% 
were analyzed using a range of DNA fragments from 20-100 bp.  Repeated gels show 
the same characteristic mobilities for the various fragments.  For the size 
range selected, 12% gels offer the best resolution.
<p>
For a size range of 80-400 bp duplex DNA, 3.5-5% gels resolved the best.  The 
DNA was largely generated by Hin digestion of the phi-chi174 RFI phage DNA.  The 
mobility plots were virtually the same as to relative mobility of the DNA with 
respect to dye for TBM and TBE buffered gels.  However, the absolute mobility 
was greater on TBE buffered gels.  The chain lengths known for the phage 
restriction fragments are in good agreement with the mobility of the synthetic 
DNA oligonucleotides.
<p>
Lambda phage DNA was similarly Hin-digested and compared to the phi-chi174 phage 
Hin-digested fragments.  All mobilities were as expected for the predicted 
digest patterns.  RNA and denatured DNA also ran similarly in TBE gels 
containing 7 M urea.  The predicted length of the single-stranded DNA correlated 
with predictions about length of duplex DNA sized by nondenaturing gels.
<p>
Small length chains (10-100 nt) have been shown to pile up near the dye front in 
formamide gels in phenobarbital buffers.  However, 98% formamide gels in 
phosphate buffer do not show this tendency.  98% formamide-12% polyacrylamide 
gels showed a good linear relationship between log(size) vs relative mobility, 
but 7 M urea gives slightly better resolution and is easier to prepare.  For DNA 
&gt;200 nts however, urea gels do not work since they do not maintain consistent 
denaturation like the formamide gels.  It is interesting to note that DNA and 
RNA having the same size do not have the same mobilities on formamide gels:  an 
RNA molecule of 168 bases migrates to a 155 b of the DNA.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Sanger, F., Coulson, A. R.
 </span>
 <span class="year">
(1975)
 </span>
 <span class="jname">
J. Mol. Biol.
 </span>
 <span class="jvol">
94,
 </span>
 <span class="pages">
441-448.
 </span>
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
O'Farrell, P. H.
 </span>
 <span class="year">
(1975)
 </span>
 <span class="jname">
J. Biol. Chem.
 </span>
 <span class="jvol">
250,
 </span>
 <span class="pages">
4007-4021.
 </span>
<p class="reftitl">
Two-dimensional electrophoresis of proteins
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Kohler, B., Milstein, C.
 </span>
 <span class="year">
(1975)
 </span>
 <span class="jname">
Nature (London)
 </span>
 <span class="jvol">
256,
 </span>
 <span class="pages">
495-497.
 </span>
<p class="reftitl">
Continuous cultures of fused cells secreting antibody of predefined
specificity
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Kohler, B., Milstein, C.
 </span>
 <span class="year">
(1975)
 </span>
 <span class="jname">
Nature (London)
 </span>
 <span class="jvol">
256,
 </span>
 <span class="pages">
495-497.
 </span>
<p class="reftitl">
Continuous cultures of fused cells secreting antibody of predefined
specificity
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Tischler, A. S., and Greene, L. A.
 </span>
 <span class="year">
(1975)
 </span>
 <span class="jname">
Nature 
 </span>
 <span class="jvol">
258,
 </span>
 <span class="pages">
341-342.
 </span>
<p class="reftitl">
Nerve growth factor-induced process formation by cultured rat pheochromocytoma 
cells
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Letourneau, P. C.
 </span>
 <span class="year">
(1975)
 </span>
 <span class="jname">
Dev. Biol.
 </span>
 <span class="jvol">
44,
 </span>
 <span class="pages">
77-91.
 </span>
<p class="reftitl">
Possible roles for cell-to-substratum adhesion in neuronal morphogenesis
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Laskey, R. A., Mills, A. D.
 </span>
 <span class="year">
(1976)
 </span>
 <span class="jname">
Eur. J. Biochem.
 </span>
 <span class="jvol">
56,
 </span>
 <span class="pages">
335-341.
 </span>
<p class="reftitl">
Quantitative film detection of <sup>3</sup>H and <sup>14</sup>C in
polyacrylamide gels by fluorography
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Kohler, G., Milstein, C.
 </span>
 <span class="year">
(1976)
 </span>
 <span class="jname">
Eur. J. Immunol.
 </span>
 <span class="jvol">
6,
 </span>
 <span class="pages">
511-519.
 </span>
<p class="reftitl">
Derivation of specific antibody producing tissue culture and
tumor lines by cell fusion
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Wilson, S. P., Kirshner, N.
 </span>
 <span class="year">
(1976)
 </span>
 <span class="jname">
J. Neurochem.
 </span>
 <span class="jvol">
27,
 </span>
 <span class="pages">
1289-1298.
 </span>
<p class="reftitl">
Isolation and characterization of plasma membranes from the adrenal medulla
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Bradford, M. M.
 </span>
 <span class="year">
(1976)
 </span>
 <span class="jname">
Anal. Biochem.
 </span>
 <span class="jvol">
72,
 </span>
 <span class="pages">
248-254.
 </span>
<p class="reftitl">
A rapid and sensitive method for the quantitation of microgram
quantities of protein utilizing the principle of protein-dye binding
<!-- ============================================================= -->
<p class="ref">
 <span class="authors">
Nurse, P., Thuriaux, P., Nasmyth, K. 
 </span>
 <span class="year">
(1976)
 </span>
 <span class="jname">
Molec. Gen. Genet.
 </span>
 <span class="jvol">
146,
 </span>
 <span class="pages">
167-178.
 </span>

<p class="reftitl">
Genetic control of the cell division cycle in the fission
yeast <i>Nature</i>
<b>292,</b> 558-560.
<p class="reftitl">
Gene required in G<sub>1</sub>
for commitment to cell cycle and in G<sub>2</sub>
for control of mitosis in fission yeast
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Greene, L. A., and Tischler, A. S. 
 </span>
 <span class="year">
(1976)
 </span>
 <span class="jname">
Proc. Natl. Acad. Sci. USA
 </span>
 <span class="jvol">
73,
 </span>
 <span class="pages">
2424-2428.
 </span>
<p class="reftitl">
Establishment of a noradrenergic clonal line of rat adrenal pheochromocytoma 
cells which respond to nerve growth factor
<p class="Para1">
Spinal ganglion neurons form axonal processes in vitro, but when trypsinized, 
the processes are no longer seen.  They re-appear after several hours after 
re-plating.
<p>
<b>Effects of substrata and Ca<sup>2+</sup></b>.  Cell adhesion (tested by 
"blasting" cells with an air stream from 2.4 cm distance with a 3 L/min airflow, 
which produces shearing forces in the liquid medium) studies of these cells 
shows that tissue culture plastic alone produces a 44-60% distraction while 
dishes coated with polylysine or polyornithine or in medium with 10 mM 
Ca<sup>2+</sup> (30-fold higher than normal) show only 5-17% distraction. 
Adhesion to collagen is a slightly better than to plastic alone, while adhesion 
to BSA or polyglutamate-BSA coated dishes is slightly less than that of plastic.
<p>
<b>Morphology</b>.  It had been reported previously that axons grown in poly-LYS 
and poly-ORN coated dishes, in addition to high Ca<sup>2+</sup> medium, had a 
crooked and twisted appearance, much the same as axons extending on the upper 
surface of co-cultured glial cells (see Luduena, <i>Dev. Biol.</i> <b>33</b>, 
470).  Axons extending only on plastic were straight.  Increases of 
Ca<sup>2+</sup> were found to cause spreading of the growth cone over the 
surface, i.e., the growth cone increased its surface area.
<p>
<b>Poly-ORN as a substratum</b>.  Poly-ORN was added to culture medium for cells 
in untreated dishes to see if it had a specific effect on cell adhesion.  In 
dishes with high amounts of poly-ORN (50 and 25 &micro;g/ml), none of the cells 
attached; many small granular precipitates were observed.  At the lower 
concentrations (2.5 and 0.25 &micro;g/ml), glial cells flattened out and neurons 
had straight axons, indicative of low adhesion.
<p>
<b>Axon initiation</b>.  Poly-ORN and -LYS coated dishes seem to have an effect 
on the number of axons created in neurons, with virtually double the amount of 
processes extended compared to plastic alone.  Moreover, the processes on 
uncoated dishes preferred to extend along the upper surface of glial cells, 
while most of the processes in poly-ORN or poly-LYS treated dishes extended 
along the nonliving substratum.
<p>
<b>Axon elongation rate</b>.  Axon length/neuron determinations after 24 hr in 
vitro culture showed a growth rate in poly-ORN surfaces 4-fold (at 600 &micro;m 
of axon/neuron) over untreated plastic.  The average length of an axon was 
nearly three-fold (275 &micro;m/axon) greater for poly-ORN surfaces as well. 
Neurons of over 2800 &micro;m were measured in poly-ORN dishes after 24 hr.
<p>
<b>Axon branching</b>.  Few neurons with &gt;2 axons cultured in untreated plastic 
could be found, whereas many neurons in poly-ORN plates have 3 or 4 axons.  The 
number of growth cones per neuron in poly-ORN treated dishes is over twice (avg. 
4.7 cones/neuron) that of neurons in plastic (2.2).
<p>
It is likely that the positive charges of the polyamines result in masking of 
the sulfonated plastic (manufacturer's treatment of plastic with sulfuric acid) 
to permit the negatively charged surface of the cell to adhere.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Martial, J. A., Baxter, J. D., Goodman, H. M., Seeburg, P. H.
 </span>
 <span class="year">
(1977)
 </span>
 <span class="jname">
Proc. Natl. Acad. Sci. USA
 </span>
 <span class="jvol">
74,
 </span>
 <span class="pages">
1816-1820.
 </span>
<p class="reftitl">
Regulation of growth hormone messenger RNA by thyroid and
glucocorticoid hormones
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Cleveland, D. W., Fischer, S. G., Kirschner, M. W., Laemmli, U. K.
 </span>
 <span class="year">
(1977)
 </span>
 <span class="jname">
J. Biol. Chem.
 </span>
 <span class="jvol">
252,
 </span>
 <span class="pages">
1102-1106.
 </span>
<p class="reftitl">
Peptide mapping by limited proteolysis in sodium dodecyl sulfate and analysis by 
gel electrophoresis
<p class="Para1">
While SDS-PAGE is a good tool for separating polypeptides in complex (impure)
samples, identification cannot be made based on electrophoretic mobility 
(which yields molecular mass data) alone.  The bands thought to be the
protein of interest must be further analyzed.
<p>
In a first experiment, alkaline phosphatase from <i>E coli</i> was digested with 
by V8 protease from <i>Staphylococcus aureus</i>.  The protease cleaves the 
carboxyl side of Glu and Asp residues.  Variables were incubation time and 
protease concentration.  All peptide analyses were done in 15% gels (30%:0.8% 
monomer:bis) and staining with 0.1% Coomassie Brilliant Blue in 50% MeOH, 10% 
acetic acid, and destained in 5% MeOH, 10% acetic acid.  For the incubation time 
study, 0.33 mg/ml substrate was incubated 37&deg; with 25 &micro;g/ml protease 
from 0 to 150 min.  The substrate has a 50 kDa mass and can be seen undegraded
at time = 0 min.  Note that enzyme is barely visible too (as a doublet just
below the substrate).  Up to 30 min there is progressive disappearance of 
intact substrate band.  With progressing time, about 10-15 fragments of 
faster mobility are observed, their masses being from 6-30 kDa.  Some of these
bands at slower mobility (higher mass) also disappear with time, as they are
targets of digestion too.  After 60 min, all but two of the fragments have
disappeared, with 2-3 new fragmetns appearing.
<p>
In the second experiments, protease concentration was varied while incubation
time kept constant at 150 min.  This map will look different in that the protease
itself will appear in the gel strongly (this can be used to understand which
bands are associated with protease).  In this study, protease concentration was
varied from 0.04 to 200 &micro;g/ml.
<p>
The effect of substrate concentration was tested using albumin with the protease
chymotrypsin, from between 0.16 to 2.56 mg/ml (16-fold change).  There was
no difference on peptide pattern, meaning that there was no dependend on 
substrate concentration.
<p>
Additional studies were done with BSA, tubuline and alkaline phosphatase using
papain.  More than a dozen bands are seen, each distinctive or unique to the
protein.  These proteins were also digested with chymotrypsin and V8 protease.
The band patterns can be used as a fingerprint for the protein.
<p>
&alpha;- and &beta;-Tubulin have very similar amino acid compositions and
N-terminal sequences, and exist as dimers in the native state.  They both
produce bands on SDS-PAGE at 53 and 55 kDa.  Preparations were cut from gels
and electroeluted (dialysis bags to recover protein).  Eluted proteins were
then digested with chymotrypsin.  In this way, they were able to be distinguished.
<p>
<b>In situ digestion of protein in gel slice</b>.  Gel slices are placed in 
sample wells of the SDS gel used for proteolytic peptide analysis.  Protease
solution is then overlayed and digestion occurs in the sample well of the
stacking gel.  1.5 mm thick gels are used and inclusion of 1 mm EDTA gives
best results.  The procedure is to cut the band out of the gel to a width
just smaller than the sample wells of the second gel.  Bands are incubated
in 10 ml 0.125 mM TrisHCl pH 6.8 with 0.1% SDS and 1 mM EDTA for 30 min
(slices can be stored at &minus;20&deg; if necessary).  The gel slice
is put in the well with 20% glycerol in buffer, and the proteas overlayed
with 10% glycerol in buffer.  Electrophoresis is done until the bromophenol
blue dye reaches to nearly the bottom of the stacking gel, then turned off
for 30 min.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Rigby, H., Diekmann, M., Rhodes, C., Berg, P.
 </span>
 <span class="year">
(1977)
 </span>
 <span class="jname">
J. Mol. Biol.
 </span>
 <span class="jvol">
113,
 </span>
 <span class="pages">
237-251.
 </span>
<p class="reftitl">
Labeling deoxyribonucleic acid to high specific activity
<i>in vitro</i> by nick-translation with DNA polymerase I
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
  <span class="authors">
   Grouse LD, Schreier BK
  </span>
  <span class="year">
   (1977)
  </span>
  <span class="jname">
   Anal. Biochem.
  </span>
  <span class="jvol">
   79,
  </span>
  <span class="pages">
   95-103.
  </span>
<p class="reftitl">
  Radioactive Labeling of DNA in Cells in Culture
<p class="Para1">
  <b>Methods</b>.
  Chang liver cells were cultured (about 5000 cells/cm<sup>2</sup>
  in 75 cm<sup>2</sup> flasks with 1 mL MEM/10% FBS) and treated
  with varying levels of [<sup>3</sup>H]thymidine (50.8 Ci/mmol),
  and uptake, incorporation, total DNA, and specific radioactivity
  were measured at 1, 2, and 3 days; the varying levels were
  1, 3, 6, and 10 &mu;Ci/ml medium.  In some cultures, the
  thymidine kinase inhibitor fluorodeoxyuridine (FdU)
  was added to 20 &mu;g/ml with 50 &mu;g/ml uridine.  Each determination
  is the average of two or three flasks.  Cells were lysed with 5 ml 1% SDS.
  Incorporation was determined by precipitating 25 &mu;l lysate with
  10% TCA containing 0.5 mg/ml BSA as carrier and filtering on glass
  fiber (GF/C) filters, and counting the filter in toluene-Liquifluor.
  Uptake was evaluated by counting 25 &mu;l lysate in scintillant (10 ml
  Aquasol); note that a proportion will represent the incorporated amount.
<p>
  A fluorometric assay was used to quantitate DNA:  the detergent lysate
  containing ca. 5-10 &mu;g DNA is treated with 0.1 vol 50% TCA at room
  temp for 30 min.  This is centrifuged at ca. 7000 &times; <i>g</i>
  and resuspended in 1 ml 5% TCA, then respun.  The second pellet is
  resuspended in 0.1 M potassium acetate in ethanol at 4&deg;C for 30 min,
  dissolved in 100 &mu;l 1 <i>N</i> NH<sub>4</sub>OH, and this
  distributed in portions containing 0.5-2 &mu;g DNA, frozen and lyophilized.
  Dried material is treated with 30 &mu;l of 2 M 3,5-diaminobenzoic acid 2 HCl
  in water (the DABA was decolorized with Norit A).  This was incubated
  tightly capped for 30 min at 60&deg;.  1 ml of 0.6 N perchloric acid
  is added and mixed immediately before reading (&lambda;<sub>ex</sub> 416 nm,
  &lambda;<sub>em</sub> 504 nm).
<p>
 <b>Results</b>.  Uptake of radiothymidine in Chang liver cells was
    seen to increase and reach a maximum after 2 days.  The level of
    radioactivity always increased with higher levels radiolabel in
    the culture medium, although about a 1.5- to 1.8-fold increase in
    cells was seen for a 2-fold greater medium level.  The presence
    of FdU had a dramatic effect:  uptake in 10 &mu;Ci/ml medium was
    about 30% less than uptake in 3 &mu;Ci/ml medium, and the FdU-treated
    culture had not reached a maximum until day 3 instead of day 2.
 <p>
    Total DNA was seen to rise to a maximum after 1 day (a burst of synthesis),
    then decline  to near start of culture levels on day 2
    and then a little more decreased by day 3.  The 1 &mu;Ci/ml
    culture was measured on day 6 and DNA levels were twice those on day 3.
 <p>
   Incorporation did not reach maximal until at least day 3 of culture
   (and was not really maximal measured at mid-level intensities of
   3 and 6&nbsp;&mu;Ci/ml medium).  Incorporation was always greater with
   increasing &mu;Ci/ml value, but a 2-fold increase in the medium was
   again less than a 2-fold increase in incorporation.  Again the
   3&nbsp;&mu;Ci/ml FdU-treated had the greatest incorporation over any
   untreated culture, but the effect of FdU seems to be more in enhancing
   uptake than incorporation (which is probably expected).
 <p>
   All culture medium radiolabel levels showed increased in specific
   radioactivity for each radiolabel concentration, and the plots
   show these increases never reached maximal level, except for
   the level at 1 &mu;Ci/ml.  Moreover, there was little to no
   difference in the change in specific radioactivity, indicating
   that the thymidine pool contributing to DNA biosynthesis had its
   source from available thymidine in the medium (the FdU-treated
   culture showed no enhancement in specific radioactivity).
 <p>
   It was estimated that by day 1 that only 5% of available thymidines
   were labeled in the 3 &mu;Ci/ml (no FdU) culture, and was 9% of
   total labeled precursor in medium.  From this it was assumed that
   some <i>de novo</i> synthesis had taken place.  Fresh radiothymidine
   was added to cultures after 3 days with no further uptake of precursor.
   Addition of unlabeled thymidine competed with uptake of
   [<sup>3</sup>H]thymidine and cell division did not occur.
 <p>
   Results generally show that moderate (3 &mu;Ci/ml) concentrations
   of radiothymidine with inclusion of FdU may be optimal for labeling
   purposes.  A theoretical maximum of 10<sup>6</sup> cpm/&mu;g DNA
   is extrapolated with the use of 10 &mu;Ci/ml medium (calculated
   against machine counting efficiency).
 <p>
   The effect of cell density was assessed.  Chang cells were plated
   about 10,000 cells/cm<sup>2</sup> and 1 d later incubated with
   3 &mu;Ci/ml.  Within 12 h a noticeable uptake was observed, very
   high at 1 d, maximal at 2 d, and then slightly decreasing before leveling
   off at days 3 and 4.  By day 4, the cell density (count) had slightly
   more than tripled, and 80% of label was TCA-precipitable, with an
   average labeling of 2.6 pCi/cell.
 <p>
   Cells grown in the same way but which had not been labeled until
   day 6 in culture, nearly reaching confluency.  Uptake was rapid,
   reaching maximum by day 8 (also within 2 days), and showing a
   slight decline before leveling as in the low density cultures.
   However, the maximal percent uptake was higher than in the low
   density cultures.  Up to day 10 (4 day post-labeling), cell count
   had also slightly more than tripled.  But only 40% of total counts
   was acid-precipitable, meaning an average 0.87 pCi/cell.  If each
   cell at lower density contains 8 pg DNA, it would have ca. 8 dpm
   tritium with specific radioactivity of 280,000 cpm/&mu;g DNA.
 <p>
   Different cell lines (nine, from seven different species) were
   also evaluated.   Cultured in 3 &mu;Ci/ml (no FdU) medium (DMEM or
   MEM with 10% FBS) for 3-5 d after plating at
   6700-13,400 cells/cm<sup>2</sup>, the specific radioactivities are
   given in the table:
<table>
<col span="2">
<col style="text-align:right;padding-right:1em;width:8em;">
<tr><th>Cell Line <th>Species <th>Specific Radioactivity (cpm/&mu;g DNA)
<tr><td>L Cells (TK<sup>&minus;</sup> <td>Mouse                <td>    950
<tr><td>Neuroblastoma NS20            <td>Mouse                <td> 51,800
<tr><td>LLC-MK<sub>2</sub>            <td>Rhesus Monkey        <td> 53,600
<tr><td>Primary fibroblast            <td>Chick                <td> 71,100
<tr><td>CRFK                          <td>Cat                  <td> 76,600
<tr><td>WI-38                         <td>Human                <td>119,000
<tr><td>A-6 Kidney                    <td><i>Xenopus laevis</i><td>152,000
<tr><td>Chang liver                   <td>Human                <td>203,000
<tr><td>L8 Muscle                     <td>Rat                  <td>214,000
</table>
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Sanger, F., Nicklen, S., Coulson, A. R.
 </span>
 <span class="year">
(1977)
 </span>
 <span class="jname">
Proc. Natl. Acad. Sci. USA
 </span>
 <span class="jvol">
74,
 </span>
 <span class="pages">
5463-5467.
 </span>
<p class="reftitl">
DNA sequencing with chain-terminating inhibitors
<p class="Para1">
<span class="underline">Method</span>.  DideoxyTTP preparation has been 
described and materials are now commmercially available.  ddA was prepared by 
one procedure with modifications and converted to the triphosphate.  ddGTP and 
ddCTP preparation have not been described but the method used to prepare ddATP 
was used.  Yields were low.
<p>
ddGTP was prepared starting from 
N-isobutyryl-5'-O-monomethoxytrityldeoxyguanosine.  The 3'-OH group was 
tosylated and then converted to the 2',3'-didehydro derivative with NaOMe.  The 
isobutyryl groups is removed as well, but completion of removal is done with 
ammonia (sp gr 0.88) overnight at 45&deg;.  The didehydro derivative is reduced 
to the dideoxy derivative and converted to the triphosphate as for ddATP.  The 
monophosphate was purified by fractionation on DEAE-Sephadex using a 
triethylamine carbonate gradient (25 to 300 mM).
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Kemp, B. E., Graves, D. J., Benjamini, E., Krebs, E. G.
 </span>
 <span class="year">
(1977)
 </span>
 <span class="jname">
J. Biol. Chem.
 </span>
 <span class="jvol">
252,
 </span>
 <span class="pages">
4888-4894.
 </span>
<p class="reftitl">
Role of multiple basic residues in determining the
substrate specificity of cyclic AMP-dependent protein kinase
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Mort, A. J., Lamport, D. T. A.
 </span>
 <span class="year">
(1977)
 </span>
 <span class="jname">
Anal. Biochem.
 </span>
 <span class="jvol">
82,
 </span>
 <span class="pages">
289-309.
 </span>
<p class="reftitl">
Anhydrous hydrogen fluoride deglycosylates glycoproteins
<p class="Para1">
Some polysaccharides such as cellulose are notoriously difficult to hydrolyze 
without involving considerable destruction of sugars.  Anhydrous HF does this 
easily however.
<p>
Results for the efficient solubilization of tomato cell wall to yield analyzable 
sugar monomers of neutral and acidic sugars is presented.  Also presented are 
results showing HF-mediated deglycosylation of tomato cell wall glycopeptides. 
These proteins were found to contain mainly proline and hydroxyproline. 
Deglycosylation of specific hydroxyproline-rich plant cell wall proteins, as 
well as N- and O-linked fetuin without degrading the protein was presented. 
Treatment of BSA with HF showed it was not degraded as well.  This paper has 
been cited by the Binder group working with tau protein dephosphorylation by HF.

<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Hietanen E, Greenwood MRC
 </span>
 <span class="year">
(1977)
 </span>
 <span class="jname">
J. Lipid Res.
 </span>
 <span class="jvol">
18,
 </span>
 <span class="pages">
480-490.
 </span>
<p class="reftitl">
A Comparison of Lipoprotein Lipase Activity and Adipocyte Differentiation
in Growing Male Rats
<p class="Para1">
[Note:  only the method of the LPL activity assay was of interest at the time in
reading this.]
<p>
The [<sup>14</sup>C]triolein assay of Schotz et al. was performed.  Substrate
emulsion is prepared by adding 1.8 &micro;Ci [<sup>14</sup>C]triolein to
36 mg cold triolein in benzene into plastic scintillation vial together with
1.5 mg lysolecithin (L-&alpha;-phosphatidylcholine, egg yolk, Type I, 98% pure)
in 3 ml of heptane.  These are evaporated in water bath
with a nitrogen flow.  2.55 ml of 0.2 M TrisHCl pH 8.0, 0.45 ml of 1% BSA
(fatty acid-free fraction V), and 3.0 ml thawed human serum taken after a 12 h
fast.  The mixture is sonicated in ice-water bath at 60 W for 4 min at 30 s
intervals using microtip.  As a control, a no-serum preparation is also made:
only lipolysis occurring under serum activation is considered to have LPL
activity.  A 100 &micro;l substrate emulsion containing 700 nmol triolein
is preincubated at 37&deg; for 30 min.  100 &micro;l of LPL source is added and
reaction occurs for 30 min.
<p class="tags">
ELISA, HBsAg
<p class="ref">
 <span class="authors">
Vandervelde EM, Cohen BJ, Cossart YE
 </span>
 <span class="year">
(1977)
 </span>
 <span class="jname">
J. Clin. Path.
 </span>
 <span class="jvol">
30,
 </span>
 <span class="pages">
714-716.
 </span>
<p class="reftitl">
An Enzyme-Linked Immunosorbent-Assay Test For Hepatitis B Surface Antigen
<p class="Para1">
Passive hemagglutination (pHA) and radioimmunoassay (RIA) for HBsAg has been 
well established.  With the many disadvantages to RIA other than its 
sensitivity, the use of enzyme-linked immunosorbent assays were developed by 
Engvall and Perlmann (1971).  Wolters et al (1976) reported HBsAg by an assay of 
this type.  The Organon Scientific Development Group Hepanostika test system was 
used with results read visually.  Two HBsAg positive controls (weak and strong) 
and a negative control were used.  The two positive sera were of <i>ad</i> and 
<i>ay</i> subtypes were used to control specificity and sensitivity.  276 
samples from high risk specimens were tested: these patients are chronic liver 
disease, recovering individuals with confirmed HepB infection, hemophiliacs, 
institutionalized patients, and others. 497 samples as part of routine testing 
are screened.  All but seven specimens are sera:  six of the 7 were commercial 
factor VIII concentrates, and one urine sample from patient with 
glomerulonephritis and seropositive for HBsAg.
<p>
Results show that the ELISE was the most sensitive of the different assays 
measuring HBsAg comparing both subtype <i>ad</i> and <i>ay</i> (immunoelectrophoresis,
hemagglutination tests Hepanosticon, Hepatest, and Auscell, and the Ausria II RIA
test).
<p>
773 specimens were tested and 38 were positive by Hepanosticon and ELISA, with
25 of those also positive by electrophoresis.  39 samples were negative using
Hepanosticon but ELISA-positive.  28 of the 39 were convalescent specimens which
are proven HBV cases collected 3-13 weeks after onset.  In a urine sample from
a seropositive patient with glomerulonephritis was positive by ELISA and RIA
but negative by Hepanosticon.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Markwell, M. A., Haas, S. M., Bieber, L. L., Tolbert, N. E.
 </span>
 <span class="year">L
(1978)
 </span>
 <span class="jname">
Anal. Biochem.
 </span>
 <span class="jvol">
87,
 </span>
 <span class="pages">
206-210.
 </span>
<p class="reftitl">
A modification of the Lowry procedure to simplify protein
determination in membrane and lipoprotein samples
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
B&ouml;hme, E., Graf, H., Schultz, G.
 </span>
 <span class="year">
(1978)
 </span>
 <span class="jname">
Adv. Cyclic Nucleotide Res.
 </span>
 <span class="jvol">
9,
 </span>
 <span class="pages">
131-143.
 </span>
<p class="reftitl">
Effects of sodium nitroprusside and other smooth muscle
relaxants on cyclic GMP formation in smooth muscle and platelets
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Garnier, J., Osguthorpe, D. J., and Robson, B.
 </span>
 <span class="year">
(1978)
 </span>
 <span class="jname">
J. Mol. Biol.
 </span>
 <span class="jvol">
120,
 </span>
 <span class="pages">
97-120.
 </span>
<p class="reftitl">
Analysis of the accuracy and implications of simple
methods for predicting the secondary structure of globular proteins
<p class="Para1">
THEORY.  The predictive theory applied here differs from Chou &amp; Fasman 
(1974) in making simpler assumptions. Prediction of secondary structure here is 
based on a prediction of the conformational state of each residue in the 
sequence.  If S<i><sub>j</sub></i> is the conformational state (<span 
class="c6">a</span>-helix, <span class="c16">b</span>-sheet) of the <i>j</i>th 
residue of the sequence, and if R<i><sub>k</sub></i> is the type of residue 
(Ala, Gln, etc) at the <i>k</i>th position, then the most general statement 
about the information of the conformation of the jth residue is:
<p style="text-align:center;">
   <i>I</i> (S<i><sub>j</sub></i>; R<sub>1</sub>,
   R<sub>2</sub>, ..., R<sub>last</sub>)
<p class="Para1">
where this statement reads as &ldquo;information which the residues at the 
first, second, and so on up to the last position carry information about the 
conformation of the jth residue.&rdquo;  Suppose that four possible 
conformational states A, B, C, and D exist for the residue.  So a predictive 
procedure would find four values for the function <i>I</i> associated with each 
residue:
<p class="Para1" style="text-align:center">
  <i>I</i> (A; R<sub>1</sub>, R<sub>2</sub>, ..., R<sub>last</sub>) <br>
  <i>I</i> (B; R<sub>1</sub>, R<sub>2</sub>, ..., R<sub>last</sub>) <br>
  <i>I</i> (C; R<sub>1</sub>, R<sub>2</sub>, ..., R<sub>last</sub>) <br>
  <i>I</i> (D; R<sub>1</sub>, R<sub>2</sub>, ..., R<sub>last</sub>)
<p>
Whichever value is highest among the possibilities would be considered the 
conformational state predicted.  If the most information was for state B, then 
that state is predicted.  The unit of information has been previously defined as 
the <i>nat</i>.
<p>
The basis for the inputs would be the observed values for known conformations of 
proteins whose structural relationships have been established.  A statistical 
analysis will reveal parameters for only one, two, or three residues (maybe) of 
these known proteins to give useful information.  Including more terms in which 
residues far from the residue under consideration are included in the terms will 
produce information with a zero value or close to it.  Previous studies have 
shown that the effect of one residue type on the conformation of residues up to 
eight residues distant plays a predominating role, while choosing shorter 
separation distances neglects significant information.  A particularly simple 
approximation is thus:
<table class="eqn">
<tr>
 <td rowspan="3">
   <i>I</i>(S<sub><i>j</i></sub>; R<sub>1</sub>,
   R<sub>2</sub>, ..., R<sub>last</sub>) &#8776;
 <td style="font-size:60%;padding:0;"><i>m</i>=+8
 <td rowspan="3">
  <i>I</i>(S<sub><i>j</i></sub>; R<sub><i>j + m</i></sub>)&nbsp;&nbsp;(1)
<tr>
 <td style="padding:0;margin:0;font:normal 250% 'Times New Roman','Times Roman',serif;">&Sigma;
<tr>
 <td style="font-size:60%;padding:0;"><i>m</i>=&minus;8
</table>
<p>
Here terms in the expansion containing &gt;1 R<i><sub>j</sub></i> parameter are 
neglected and interactions between residues separated by more than 8 residues 
are similarly neglected.
<p>
The <i>I</i>(S<i><sub>j</sub></i>; R<i><sub>j+m</sub></i>) measure is the 
information which the type of residue at <i>j + m</i> carries about the 
conformational state of the <i>j</i>th residue. With 20 types of residue R and 
four possible conformational states, then there must be 20 &times; 4 different 
parameters for each separation <i>m</i>.  If the window for m is -8 &le; 
<i>m</i> &le; +8, then including <i>m</i>=0, there must be 17 possible 
separations. Hence a total of 20 &times; 4 &times; 17 parameters must be 
provided, the conformational states defined in Methods.
<p>
METHODS.  Tables 1-4 provide the parameters for the four conformational states 
considered: S<i><sub>j</sub></i>= H (&alpha;-helical state); 
S<i><sub>j</sub></i>=E (extended-chain state or &beta;-sheet); 
S<i><sub>j</sub></i>=T (reverse-turn); and S<i><sub>j</sub></i>=C (coil or 
aperiodic state).  The last state is used to define the state which is not H, E, 
or T.  Details of the states have been defined previously, and T has also been 
defined as T<i><sub>m</sub></i>, because the two middle residues of a 
four-residue reverse turn are the only residues with conformations relevant to 
the definitions of a reverse turn.  A prediction that residues 26-29 constitute 
a reverse turn makes no statement concerning the conformation of residues 26 and 
29, only for residues 27 and 28.  Actually information concerning T (turn of 4 
residues) and T<sub>m</sub> (turn of 2 residues) is not significantly different 
except in the cases of Trp and Pro amino acids.
<p>
Parameters obtained from the directional information plots of 
<i>I</i>(S<i><sub>j</sub></i>; R<i><sub>j+m</sub></i>) versus <i>m</i> given 
previously (Robson &amp; Suzuki, <i>J. Mol. Biol.</i> <b>107</b>: 327, 1976). 
Values are based on statistical (information theory) analysis of 25 proteins of 
known sequence and conformation.  Sampling noise due to the finite size of the 
sample has been eliminated in value determinations using accepted smoothing 
procedures in curve fitting.  Directional values for the coil state were not 
previously reported.
<p>
The function:
<table class="eqn"
   style="font:normal 100% 'Times New Roman','Times Roman',Times,serif;">
<tr>
 <td rowspan="2"> <i>I</i><sub>clus</sub>(R<sub><i>j</i></sub>) = ln
 <td><i>P</i>(S<sub><i>j</i></sub> = H; R<sub><i>j</i></sub>|R<sub><i>j &minus; m'</i></sub>
   = &Phi;, R<sub><i>j + m'</i></sub> = &Phi;)
 <td rowspan="2">&nbsp;&nbsp;&nbsp;(2)
<tr>
 <td style="border-top:1px solid black;">
   <i>P</i>(S<sub><i>j</i></sub> = H; R<sub><i>j</i></sub>)
</table>
<p class="Para1">
takes into account the clustering of hydrophobic
residues, particularly on the surface of an a-helix.  The numerator
probability is the probability that R<i><sub>j</sub></i>
will be a-helical, when R<i><sub>j-m'</sub></i> and
R<i><sub>j+m'</sub></i> is hydrophobic (&Phi;) as
determined from the database of Robson and Suzuki.  The definition of
a hydrophobic side chain is detailed below.  This clustering was only
taken into accounted in a limited series of studies.
<p>
<i>The basic method</i> (<i>directional method</i>).
Prediction of the conformational state involves
evaluating equation (1) for each conformational state, then selecting
information with the highest information content.  From the
information content and for each conformational state, a constant value or
decision constant (<i>DC</i><sub>S</sub>) is subtracted before comparing
the information contents.  <i>DC</i><sub>S</sub> is an adjustable
parameter chosen in order to produce optimal predictions, and is a
function only of S<sub>j</sub>.  If there are <i>N</i> different
conformational states S, the prediction made is a function of eqn (1) and
of <i>N</i>-1 values of <i>DC</i><sub>S</sub>.  The N-1 term arises
because only relative values of DC<sub>S</sub> affect the prediction so
that one of the DC<sub>S</sub>, usually for the coil state, can be fixed
at an arbitrary value of zero.
<p>
<i>The single-residue information method</i>.

<!-- ================================================================ -->
<p class="tags">

<p class="ref">
 <span class="authors">
D. N. Podell &amp; G. N. Abraham
 </span>
 <span class="year">
(1978)
 </span>
 <span class="jname">
Biochemical and Biophysical Research Communications
 </span>
 <span class="jvol">
81,
 </span>
 <span class="pages">
176-185.
 </span>
<p class="reftitl">
A Technique for the Removal of Pyroglutamic Acid from the Amino Terminus of 
Proteins Using Calf Liver Pyroglutamate Amino Peptidase
<p class="Para1">
Structural analysis of proteins and peptides like gastrin, fibrinopeptides, 
collagen, bradykinin-potentiating peptides, and immunoglobulins, has been 
complicated by the presence of an N-terminal glutamine residue that cyclizes 
naturally in mild aqueous (physiological) conditions to form pyroglutamate, a 
stable structure that effectively blocks the primary &alpha;-amino group that is 
often a target of the analytical chemistry required for structural elucidation.
<p>
Takahashi &amp; Cohen reported in 1969 that diborane in either tetrahydrofuran 
or tetramethylurea will reduce the pyrrolidone ring to a pyrrolidine. 
Unfortunately side reactions also cause reduction of carboxyl and peptide bonds 
under conditions that only reduce the pyrrolidone even to 35-46% yield. 
Doolittle and Armentrout found in <i>Bacillus subtilis</i> and Szewczuk and 
Mulczyk isolated from <i>Pseudomonas fluorscens</i> an enzyme that liberated 
pyroglutamate from N-terminal pyroglutamyl proteins and peptides. The bacillus 
preparation was 100-fold purified but still quite unstable when stored in 
ammonium sulfate or when frozen.  The need to purify the bacterial enzymes to a 
point where they are stable is impractical.
<p>
Cryoglobulins were purified from human serum by centrifuging at 4&deg;, washing 
three times with ice-cold PBS (pH 7.4), and following with dissolution in 15 mM 
phosphate buffer, pH 7.3, at 37&deg; and passing over DEAE-cellulose.  Purified 
globulins were reductively alkylated in 0.2 M Tris with 4 M urea at pH 8.6 (see 
Fleishmann et al, <i>Arch. Biochem. Biophys</i> <b>Suppl 1</b>:  174-180, 1962), 
and the heavy and light chains separated in Sephadex G-100 in 1 M acetic acid.
<p>
Calf liver pyroglutamate amino peptidase (L-pyroglutamyl peptide hydrolase, EC 
3.4.11.8) was obtained from Boehringer Mannheim Biochemicals and stored 
lyophilized at &minus;20&deg;.  1 mg/ml reductively alkylated globulin is 
dialyzed in a (de-blocking) buffer of 100 mM sodium phosphate, pH 8.0, 5 mM DTT, 
10 mM disodium EDTA, 5% (v/v> glycerol.  0.5 mg crude calf liver protein 
(containing 25 &mu;g enzyme) is added in lyophilized form directly to 10 ml of 
the 1 mg/ml dialysate in a screw-cap tube, and tube is flushed with nitrogen, 
capped, mixed.  The prep is kept for 9 h at 4&deg;, then brought to room 
temperature.  Another 0.5 mg of the crude enzyme prep is added and the vial 
gassed and mixed, with incubation for 14 h at room temp.  Afterward, the mixture 
is dialyzed against 50 mM acetic acid, lyophilized, and then analyzed by 
sequencing.
<p>
The N-terminus was analyzed by the dansylation method of Groos and Labouesse. 
Chromatography was on 5 &times; 5 cm Cheng Chin polyamide thin layer in two 
dimensions:  the first is 1.5% formic acid, and the second is benzene:acetic 
acid (9:1) for the full run followed by drying and then continued development in 
ethyl&nbsp;acetate:methanol:acetic&nbsp;acid (20:1:1) until its front reached 
the end.
<p>
Two proteins identified as WEB and LON were dansylated, and both revealed Asp 
and Tyr, resp.  Both proteins were then subjected to 6 cycles of Edman 
degradation, no PTH-amino acid derivatives were identified using GLC, TLC or 
HPLC;  this indicated effective blocking.
<p>
Heavy chains were then deblocked with enzyme, and the resulting product 
subjected again to Edman cycling, and PTH derivatives identified by HPLC and 
TLC.  The first cycle from HPLC of the degradation of LON showed a percent yield 
of 83-85% of a residue matched as valine (Val). This shows that substantial 
de-blocking had occurred.  The 2nd cycle showed the presence of Glu and Gln on 
HPLC, and a substantial Gln spot on TLC confirmed it was Gln.
<p>
The bacterial enzymes are sensitive to temperature, redox conditions, divalent 
metal ions, alkylating agents, and results also depend on time of incubation. 
Optimal conditions were determined for real proteins (25-50 kDa).  Iodoacetamide 
inactivates the enzyme, indicating that it works through a sulfhydryl group. 
Both 2-mercaptoethanol and DTT stabilize the enzyme better, but DTT is better 
able to resist oxidation. Because metals like Hg<sup>2+</sup> bind to &mdash;SH 
groups effectively, inclusion of EDTA or other metal chelators also prevents 
enzyme inactivation.  The enzyme is also sensitive to dissolved oxygen, with a 
59% reduction in activity in room air; hence it is necessary to flush with 
nitrogen.  The enzyme also did not work at 37&deg;
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Lieberman M
 </span>
 <span class="year">
(1978)
 </span>
 <span class="jname">
Ann. Rev. Plant Physiol.
 </span>
 <span class="jvol">
30,
 </span>
 <span class="pages">
533-591.
 </span>
<p class="Para1">
Two developments make C<sub>2</sub>H<sub>4</sub> studies
possible:  (i) the use of gas chromatography
(ii) the rediscovery of the relationship between C<sub>2</sub>H<sub>4</sub>
and auxin C<sub>2</sub>H<sub>4</sub> influences many aspects of growth and
development:  seed germination, seedling growth, root growth, leaf
growth, stress phenomena, and regulation of ripening.
<p>
MICROORGANISMAL PRODUCTION
<p>
<i>Penicillium digitatum</i>&mdash;&mdash;origin of C<sub>2</sub>H<sub>4</sub>
thought to be from fumarate in TCA cycle; one concluded that methylene Cs
of citrate responsible.  U-<sup>14</sup>C Met does not yield lableled
ethylene.  Source of C<sub>2</sub>H<sub>4</sub> then considered
to be a metabolic event from isocitrate.
<p>
Chou &amp; Yang showed that C<sub>2</sub>H<sub>4</sub> comes C3 and C4
of 2-oxoglutarate or glutamic acid.  Note structural similarity
between Glu and Met.  C<sub>2</sub>H<sub>4</sub> production was
possible only with mycelial mat (no shaking).  One group showed that
Met added to shaking cultures induced and was the substrate for
C<sub>2</sub>H<sub>4</sub>;  induction was inhibited by
transcriptional and translational inhibitors (act D and
cyclohexiimide).  Both a physiological and nonphysiological lag for
C<sub>2</sub>H<sub>4</sub> production were seen.
Small P<sub>i</sub> (0.01 mM vs. 100 mM) concentrations
contributed to heavy synthesis.
<p>
Primrose has shown prodn in E. coli (222).  Others have shown that
these organisms require light and Fe<sup>2+</sup>, however,
which causes nonenzymatic conversion of Met to
C<sub>2</sub>H<sub>4</sub>.  The role of C<sub>2</sub>H<sub>4</sub>
in bacteria remains to be elucidated.
<p>
HIGHER PLANTS
<p>
Apples and tomatoes, which produce very high in vivo levels
of C<sub>2</sub>H<sub>4</sub>, produce none in vitro.
The positions of EFE has been more carefully defined.
Several model systems studied:
<p>
Peroxidized linoleate systems in Cu<sup>+</sup> or Fe<sup>2+</sup>,
a family of hydrocarbon gases are produced.  Accelerated peroxidation
and decomposition was the mechanism.  Propanal was the immediate
precursor of C<sub>2</sub>H<sub>4</sub> in this model.  However,
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Unsicker, K., Krisch, B., Otten, U., Thoenen, H.
 </span>
 <span class="year">
(1978)
 </span>
 <span class="jname">
Proc. Natl. Acad. Sci. USA
 </span>
 <span class="jvol">
75,
 </span>
 <span class="pages">
3498-3502.
 </span>
<p class="reftitl">
Nerve growth factor-induced fiber outgrowth from isolated rat adrenal chromaffin 
cells: impairment by glucocorticoids
<p class="abstract">
<b>Abstract</b>  Addition of nerve growth factor to cultures of dissociated rat 
adrenal medullary cells caused fiber outgrowth from chromaffin cells. These 
fibers exhibited all the characteristics of neurites, particularly the formation 
of typical growth cones exhibiting intense catecholamine-specific fluorescence. 
Because this nerve growth factor-mediated neurite outgrowth could be abolished 
by physiological concentrations of glucocorticoids, it is concluded that the 
high glucocorticoid concentrations normally present in the adrenal medulla 
prevent the fiber outgrowth from medullary chromaffin cells in vivo.  In 
dissociated sympathetic neurons the same concentrations of glucocorticoids 
markedly reduce but do not completely abolish neuronal fiber outgrowth.

<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Chirgwin, J. M., Przybyla, A. E., MacDonald, R. J., Rutter, W. J.
 </span>
 <span class="year">
(1979)
 </span>
 <span class="jname">
J. Biochem. (Tokyo)
 </span>
 <span class="jvol">
18,
 </span>
 <span class="pages">
5294-5299.
 </span>
<p class="reftitl">
Isolation of biologically active ribonucleic acid from sources
enriched in ribonuclease
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Towbin, H., Staehelin, T., Gordon, J.
 </span>
 <span class="year">
(1979)
 </span>
 <span class="jname">
Proc. Natl. Acad. Sci. USA
 </span>
 <span class="jvol">
76,
 </span>
 <span class="pages">
4350-4354.
 </span>
<p class="reftitl">
Electrophoretic transfer of proteins from polyacrylamide
gels to nitrocellulose sheets:  procedure and some applications
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Carpenter, G., King, L., Cohen, S.
 </span>
 <span class="year">
(1979)
 </span>
 <span class="jname">
J. Biol. Chem.
 </span>
 <span class="jvol">
254,
 </span>
 <span class="pages">
4884-4891.
 </span>
<p class="reftitl">
Rapid enhancement of protein phosphorylation in A431 cell
membrane preparations by epidermal growth factor
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Renart, J., Reiser, J., Stark, G. R.
 </span>
 <span class="year">
(1979)
 </span>
 <span class="jname">
Proc. Natl. Acad. Sci. USA
 </span>
 <span class="jvol">
76,
 </span>
 <span class="pages">
3116-3120.
 </span>
<p class="reftitl">
Transfer of proteins from gels to diazobenzyloxymethyl-paper
and detection with antisera:  a method for studying antibody
specificity and antigen structure
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Thayer, R.
 </span>
 <span class="year">
(1979)
 </span>
 <span class="jname">
Anal. Biochem.
 </span>
 <span class="jvol">
98,
 </span>
 <span class="pages">
60-63.
 </span>
<p class="reftitl">
An improved method for detecting foreign DNA in plasmids of
<i>Escherichia coli</i>
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Galfre, G., Milstein, C., Wright, B.
 </span>
 <span class="year">
(1979)
 </span>
 <span class="jname">
Nature (London)
 </span>
 <span class="jvol">
277,
 </span>
 <span class="pages">
131-133.
 </span>
<p class="reftitl">
Rat rat hybrid myelomas and a monoclonal anti-Fd portion of mouse IgG
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Nelson T, Brutlag D
 </span>
 <span class="year">
(1979)
 </span>
 <span class="jname">
Meth. Enzymol.
 </span>
 <span class="jvol">
68,
 </span>
 <span class="pages">
41-50.
 </span>
<p class="reftitl">
Addition of homopolymers to the 3'-ends of duplex DNA with terminal
transferase
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Kim, H., Binder, L. I., Rosenbaum, J. L.
 </span>
 <span class="year">
(1979)
 </span>
 <span class="jname">
J. Cell Biol.
 </span>
 <span class="jvol">
80,
 </span>
 <span class="pages">
266-276.
 </span>
<p class="reftitl">
The periodic association of MAP2 with brain microtubules in vitro
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Chamberlain, J. P.
 </span>
 <span class="year">
(1979)
 </span>
 <span class="jname">
Anal. Biochem.
 </span>
 <span class="jvol">
98,
 </span>
 <span class="pages">
132-135.
 </span>
<p class="reftitl">
Fluorographic detection of radioactivity in polyacrylamide gels
with water-soluble fluor sodium salicylate
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Baginsky ML, Brown WV
 </span>
 <span class="year">
(1979)
 </span>
 <span class="jname">
J. Lipid Res.
 </span>
 <span class="jvol">
20,
 </span>
 <span class="pages">
548-556.
 </span>
<p class="reftitl">
A New Method for The Measurement of Lipoprotein Lipase in Postheparin Plasma
Using Sodium Dodecy Sulfate for The Inactivation of Hepatic Triglyceride Lipase
<p class="Para1">
It was found that SDS could remove the activity in hepatic triglyceride
lipase (H-TGL) when up to 1 mM concentration was used, producing a modest
increase in LPL activity.  LPL activity was lost rapidly if dilution with
buffer alone or if SDS concentration fell &lt; 0.2 mM.
<p>
Purified enzymes of LPL and H-TGL were then mixed with normal serum which
had been preincubated with SDS.  In this system, H-TGL was inhibited but
not LPL;  however a greater SDS concentration is required (&gt; 25 mM).
This assay system required presence of serum as an activator and that NaCl
could not exceed 0.75 M.  It was necessary to minimize the time of SDS
treatment and the concentration of SDS in postheparin plasma, since
beyond that necessary for H-TGL inactivation, LPL inactivation could be
observed.  Also SDS preincubation times had to be longer for patients with
hyperlipoproteinemia.  Generally all lipase activity&mdash;both H-TGL and
LPL&mdash;was lost if SDS at 20 mM for 2 h was done, true for both normal
and hypertriglyceridemia patients.
<p>
[more results not presented here, as they don't seem relevant]
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Nei M, Li W-H
 </span>
 <span class="year">
(1979)
 </span>
 <span class="jname">
Proc. Natl. Acad. Sci. USA
 </span>
 <span class="jvol">
76,
 </span>
 <span class="pages">
5269-5273.
 </span>
<p class="reftitl">
Mathematical Model for Studying Genetic Variation in Terms of Restriction
Endonucleases
<p class="Para1">
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Schneeweiss, F., Naquira, D., Rosenheck, K., Schneider, A. S.
 </span>
 <span class="year">
(1979)
 </span>
 <span class="jname">
Biochim. Biophys. Acta
 </span>
 <span class="jvol">
555,
 </span>
 <span class="pages">
460-471.
 </span>
<p class="reftitl">
Cholinergic stimulants and excess potassium ion increase the fluidity of plasma 
membranes isolated from adrenal chromaffin cells
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Meyer, D. I., Burger, M. M.
 </span>
 <span class="year">
(1979)
 </span>
 <span class="jname">
J. Biol. Chem.
 </span>
 <span class="jvol">
254,
 </span>
 <span class="pages">
9854-9859.
 </span>
<p class="reftitl">
Isolation of a protein from the plasma membrane of adrenal medulla which binds 
to secretory vesicles
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Yanagihara, N., Isosaki, M., Ohuchi, T., Oka, M.
 </span>
 <span class="year">
(1979)
 </span>
 <span class="jname">
FEBS Lett.
 </span>
 <span class="jvol">
105,
 </span>
 <span class="pages">
296-298.
 </span>
<p class="reftitl">
Muscarinic receptor-mediated increase in cyclic GMP level in isolated bovine adrenal medullary cells
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Garrels, J. I. and Schubert, D.
 </span>
 <span class="year">
(1979)
 </span>
 <span class="jname">
J. Biol. Chem.
 </span>
 <span class="jvol">
254,
 </span>
 <span class="pages">
7978-7985.
 </span>
<p class="reftitl">
Modulation of protein synthesis by nerve growth factor
<!-- ============================================================= -->
<p class="tags">
HBV, gene expression, E coli host
<p class="ref">
 <span class="authors">
Pasek M, Goto T, Gilbert W, Zink B, Schaller H, MacKay P, Leadbetter G, Murray K
 </span>
 <span class="year">
(1979)
 </span>
 <span class="jname">
Nature
 </span>
 <span class="jvol">
282,
 </span>
 <span class="pages">
575-579.
 </span>
<p class="reftitl">
Hepatitis B virus genes and their expression in E. coli
<p class="Para1">
ABSTRACT only.
A composite DNA sequence of regions of hepatitis B virus, determined from a 
series of recombinant plasmids, reveals the genes for the surface antigen and 
the core antigen of the virus. The sequence of the core antigen shows it to be a 
DNA binding protein. The core antigen gene is expressed in Escherichia coli and 
when injected into rabbits the bacterial product induces antibodies which react 
with core antigen isolated from human sources.
<!-- ============================================================= -->
<p class="tags">

<p class="ref" id="swenson">
 <span class="authors">
Swenson RP, Howard JB  
 </span>
 <span class="year">
(1980)
 </span>
 <span class="jname">
J. Biol. Chem. 
 </span>
 <span class="jvol">
254,
 </span>
 <span class="pages">
44524456.
 </span>
<p class="reftitl">
Structural Characterization of Human &alpha;<sub>2</sub>-Macroglobulin Subunits
<p class="Para1">

<b>Isolation of a2M</b>.  All steps except chromatography step are done at 
0-4&deg; and using polyethylene containers.  500 ml of outdated human plasma is 
diluted in 2 vol. of 20 mM sodium phosphate pH 7.4 with 0.1 M NaCl.  This is 
adjusted with PEG 6000 to 4% (w/v) using a 50% (w/v) stock solution.  After 30 
min stirring, centrifugation is done (speed? time?).  The supernatant is 
adjusted to 12% with PEG 6000, then centrifuged after 30 min stirring. 
Precipitate is is dissolved in 100 ml of 10 mM sodium phosphate pH 7.6 with
5 mM EDTA, then dialyzed against 10 L deionized water overnight.  The dialysate
is centrifuged to clarify.  A 4.0 M ammoniumm sulfate is added to supernatant
(stirring) to a final 1.6 M concentration, and after 30 min, centrifuged.
Supernatant is brought to 1.92 M with ammonium sulfate.  The pellet is obtained
in the centrifuge, washed two times with 1.92 M ammonium sulfate, and finally
resuspended in 25 ml in the phosphate/EDTA pH 7.6 buffer.  This is dialyzed
against 10 L of 10 mM sodium acetate pH 5.2 for 24 h, clarified by centrifuge
and finally dializd by 2 L of 25 mM TrisHCl pH 8.0.  Whatman DE52 cellulose
chromatography is done with 4 &times; 20 cm column in 25 mM TrisHCl pH 8.0.
A final yield of 200 mg protein is obtained from 500 ml plasma.


<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Efstratiadis, A., Posakony, J. W., Maniatis, T., Lawn, R. M.,
O'Connell, C., Schritz, R. A., Deriel, J. K., Forget, B. G.,
Weissman, S. M., Slightom, J. L., Bechl, A. E., Smithies, O., Baralle, F.
E., Shoulders, C. C., Proudfoot, M.
 </span>
 <span class="year">
(1980)
 </span>
 <span class="jname">
Cell
 </span>
 <span class="jvol">
21,
 </span>
 <span class="pages">
653-668.
 </span>
<p class="reftitl">
[report concerning the presence of CCAAT consensus sequence within promoters]
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Barnes, D., Sato, G.
 </span>
 <span class="year">
(1980)
 </span>
 <span class="jname">
Cell
 </span>
 <span class="jvol">
22,
 </span>
 <span class="pages">
649-655.
 </span>
<p class="reftitl">
Serum-free cell culture:  a unifying approach
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Marcus, M., Lavi, U., Nattenberg, A., Rottem, S., Markowitz, O.
 </span>
 <span class="year">
(1980)
 </span>
 <span class="jname">
Nature
 </span>
 <span class="jvol">
285,
 </span>
 <span class="pages">
659-661.
 </span>
<p class="reftitl">
Selective killing of mycoplasmas from contaminated mammalian cells
in cell cultures
<p class="Para1">
Mycoplasma are common contaminants of animal cells in
cultures.  25 Mycoplasma and Acholeplasma spp have been identified by
the most frequent are <i>Mycoplasma hyorhinis, Mycoplasma orale,
Mycoplasma arginini,</i> and <i>Acholeplasma laidlawii.</i>  These
are responsible for 85% of the contaminations.  The differences in
the way mycoplasma and mammalian cells use nucleic acid precursors can be
exploited:  mycoplasma have a nutritional requirement for nucleic
acid precursors which can be met by purines or pyrimidine bases or by
nucleosides.  Mammalian cells incorporate very little free
pyrimidines.  Selective incorporation of 5-bromouracil (5-BU) into
mycoplasma is advantageous since visible light induces breaks in DNA
containing 5-BU and this photosensitivity is increased by binding of the
fluorochrome Hoechst-33258 to DNA.  Mycoplasma has an unusually high
A+T content, and as H-33258 has a high affinity for A-T pairs and an even
higher affinity for A-5BU base pairs, the capacity for breakage is
enormous.
<p>
2 &times; 10<sup>6</sup> E-36 cells from Chinese hamster in a
small flask were inoculated with 10<sup>6</sup> colony-forming
units of <i>M. hyorhinis</i> or <i>A. laidlawii.</i>
After several weeks, a stable contamination was present.
Using the method of Chen (<i>Exp. Cell Res.</i> <b>104</b>: 255, 1977),
100 mycoplasma cells could be found for
every cultured cell.  BHK-21 from Syrian hamster and RAG cells from
mouse were also used and known to be contaminated with mycoplasma.
Both contaminated and uncontaminated RAG cells established in were
incubated in DME + 10%FCS containing 1&nbsp;&mu;Ci/ml (20 Ci/mmol) of
<sup>3</sup>H-uracil, <sup>3</sup>H-thymine, or <sup>3</sup>H-5BU
for 1 h and cells were pptd in 5% cold TCA.  Ppt. collected on glass
fiber GF/C filters was counted and results indicated that uracil and, to a
much lesser degree but still significant, 5-BU were incorporated into the
material.  Thymine was always low.
<p>
Cells were treated with 5-BU (30&nbsp;&mu;g/ml) for 24 h
and then H-33258 (1&nbsp;&mu;g/ml) was added for 1 h before the cells were
illuminated under a 400-W cold bright fluorescence lamp from 5 cm for 30
min.  Fresh medium with 5-BU is added and the Hoechst treatment done
at 24 h intervals.  After series of 2-5 such treatments, cells are
trypsinized, washed, and diluted in fresh medium containing 10&nbsp;&mu;M
thymidine (to prevent incorporation of any 5-Br-2-deoxyuridine (5-BUdR)
released from dead mycoplasma into cellular DNA) and 10&nbsp;&mu;M 2-deoxycytidine
(to prevent possible toxic effects of 5-BUdR on growth of cells).
Cells had been taken as single cells or in groups of 100.  Results
show that single cells had been cured after 2-3 d of these periodic
treatments.  Clones of groups of 100 cells took longer (5 d).
<p>
Remaining viable mycoplasma contaminants in growing
clones can be detected by selective [<sup>3</sup>H]uracil
incorporation (see Kenny in <i>Microbiology</i>, ed. Schlesinger, D.,
32-36, 1975), by cytological observation (see Chen), and by viability
counting of CFU (see Stanbridge et al., <i>Nature New Biol</i>.
<b>232</b>: 242, 1971).
<p>
These killing techniques proved efficient in curing
RAG and BHK-21 cells.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Blobel, G.
 </span>
 <span class="year">
(1980)
 </span>
 <span class="jname">
Proc. Natl. Acad. Sci. USA
 </span>
 <span class="jvol">
77,
 </span>
 <span class="pages">
1496-1500.
 </span>
<p class="reftitl">
Intracellular membrane topogenesis
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
West, R. W., Jr., Rodriguez, R. L.
 </span>
 <span class="year">
(1980)
 </span>
 <span class="jname">
Gene
 </span>
 <span class="jvol">
9,
 </span>
 <span class="pages">
175-193.
 </span>
<p class="reftitl">
Construction and characterization of <i>E. coli</i>
promoter-probe plasmid vectors.  II. RNA polymerase binding studies
of antibiotic-resistance promoters
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Oakley, B. R., Kirsch, D. R., Morris, N. R.
 </span>
 <span class="year">
(1980)
 </span>
 <span class="jname">
Anal. Biochem.
 </span>
 <span class="jvol">
105,
 </span>
 <span class="pages">
361-363.
 </span>
<p class="reftitl">
A simplified ultrasensitive silver stain for detecting protein
in polyacrylamide gels
<!-- ============================================================= -->
<p class="tags">
HBsAg ELISA, photometer
<p class="ref">
 <span class="authors">
Tanaka F, Nishi K, Kazutaka A, Hayashi C, Miyai K
 </span>
 <span class="year">
(1980)
 </span>
 <span class="jname">
Clin. Chem.
 </span>
 <span class="jvol">
26,
 </span>
 <span class="pages">
1674-1651.
 </span>
<p class="reftitl">
Enzyme Immunoassay of Hepatitis B Surface Antigen with Use of the 
<q>Hepanostika</q> Test Kit and a New Photometer, <q>Photo-Elisa 1</q>
<p class="Para1">
HBsAg tests can now be done by RIA and passive and <q>reversed passive</q>
hemagglutination sold as test kits.  RIA is more sensitive than agglutination
but the latter is used in screening because there is no equipment and not
radioisotope.  In addition only 25 &micro;l sample is required and it is
easy to configure for high throughput.  Several EIA-based kits have made it
to commerce and are as sensitive as RIA: Auszyme (Abbott), Cordia H (Cordis),
Reverseia (Megro Research Institute).  All use the sandwich assay but differ
in solid phase preparation and enzyme used.  Each manufacturer supplies a
microliter-scale photometer which is very expensive.  This report describes
a new photometer <q>Photo-Elisa 1</q> which is an adaptation of a photometer
created for assaying bilirubin (<q>Photo B-H meter</q>) and optimized for 
measuring HBsAg with the Hepanostika test kit.  Serum was used from in- and outpatients
with acute and chronic hepatitis (posttransfusion or not), cirrhosis, and
pre-operative patients as well as control from 200 paramedical and medical
personnel (chosen w/o bias from 1400 persons to be screened).  
The Photo-Elisa 1 instrument has a 490 nm filter with a correction filter
at 575 nm (for subtracting absorbance due to Hb): it produces a value
A<sub>0</sub> = A<sub>490</sub> &minus; A<sub>575</sub> using a computational
circuit in the instrument.  A special amplifying circuit provides 10-fold gain
compared in this derivative from Photo B-H meter.  The electronics showing
how light is generated and adjusted and amplified and converted to a digital
readout is given in a figure.  7.5 cm long capillary tubes with 1.1-1.2 i.d.
 and a 0.2 mm thick wall (Cat #NHT 100) are made by Propper.  Hepanostika
kits are from Nippon Organon (Tokyo).  In the method, 100 &micro;l of serum
is placed in the Micro-Elisa plate well, incubated 37&deg; for 2 h or at RT
overnight, washed four times with 300 &micro;l wash fluid, then labeled Ab
is added (100 &micro;l), left 2 h at 37&deg;, again washed four times, and
then a 100 &micro;l substrate is added and left at RT for 50 min to an endpoint.
50 &micro;l 2 M sulfuric acid is used to stop and stabilize color development
and the solution is read in the Photo-Elisa 1 instrument.  The result is a 
ratio of background-corrected sample absorbance and background-corrected
negative control [ (A<sub>sample</sub> &minus; A<sub>reagent blank</sub>) &divide;
(A<sub>negative control</sub> &minus; A<sub>reagent blank</sub>) ].  Positive
values would be &ge; 2.1, borderline in the range of 1.5 to 2.1, and negative
values &lt; 1.5.
<p>
In testing the photometer, potassium dichromate, cobalt chloride, and Congo
Red absorb at 492 nm.  Congo Red shows an absorbance close to test sample
so it is used in standardizing the instrument.  A stock 1 mg in 30 mg water
was diluted to show linear response of instrument to be identical to the
linear response with another isntrument (Hitachi 181 spectrophotometer).
Then double dilutions from a 10 mg in 30 ml stock showed a linear photometric
response down to 1:64 from 1:1.  Three dye concentrations were prepared as
ten replicates and read in the instrument and CV values were from 3-5%.
<p>
Three patients A, B, C which are HBsAg-postive had their serum serially double-diluted
and assayed with Hepanostika assay (diluent was a HBsAg-negative control serum).
Patient titers had previously been assessed using reversed passive hemagglutination
with A = 256, B = 4096, and C = 64.  The curves showed excellent linear response
below absorbance values of 8.0.  Assay precision was evaluated by 10 repeated assays
of diluted samples (from 1:64 to 1:2048), and CV values were generally &lt; 3%
at lower dilutions and about 5% at higher dilutions, for values inside the linear
range of the instrument (Abs. 4-12).
<p>
Naked eye reading was done with 100 samples and two individuals.  They disagreed
on 7 of the 100 samples, but generally agreed on reading with the Photo-Elisa 1,
which also resolved the disagreement on the seven samples.
<p>
HBsAg subtype <i>ay</i> (5.6 &micro;g/L) and subtype <i>ad</i> (8.0 &micro;g/L) 
are provided and diluted 1:1 to 1:5.  The smallest detectable concentration was
2.8 &micro;g/L for <i>ay</i> and 4.0 &micro;g/L for <i>ad</i>.
<p>
Of the 200 putatively negative control samples provided (as assed by reversed
passive hemagglutination (Yamanouchi Pharmaceutical Co), four tested positive
on the inhibition test with Hepanostika kit.
<p>
30 samples were assayed using an RIA and the Hepanostika kit.  There was a perfect
correlation with respect to positive/negative result and quantitatively (cpm values
for RIA and absorbance readings for the Photo-Elisa 1).
<p>
The Hepanostika test with result solution read in capillary tube with Photo-Elisa
1 proves as sensitive as an RIA, with both much more sensitive than the reversed
passive agglutination test.  Sensitivity levels were assessed with different
HBsAg subtypes.  Absorbance readings may be affected if capillary tube tolerances
are not met, since pathlength of 1.2 mm dia is quite short.  Instrument was designed
with a two-point wavelength spectrophotometry to account for reflections and 
absorptions created by the capillary tube itself.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Duggleby, R. G.
 </span>
 <span class="year">
(1981)
 </span>
 <span class="jname">
Anal. Biochem.
 </span>
 <span class="jvol">
110,
 </span>
 <span class="pages">
9-18.
 </span>
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Brinster, R. L., Chen, H. Y., Trumbauer, M., Senear, A. W.,
Warren, R., Palmiter, R. D.
 </span>
 <span class="year">
(1981)
 </span>
 <span class="jname">
Cell
 </span>
 <span class="jvol">
27,
 </span>
 <span class="pages">
223-231.
 </span>
<p class="reftitl">
Somatic expression of herpes thymidine kinase in mice following
injection of a fusion gene into eggs
<p class="Para1">
The introduction of genes into animals is more than mere
transfection or injection of DNA/nucleic acid into cultured cells.
The use of cultured cells for the analysis of gene expression and
regulation is not always suitable to all genes.  Animal genes may
instead be introduced into living systems by introducing the genes first
into embryos, then analyzing the result in differentiated adult
cells.  Foreign cells, e.g., teratocarcinoma cells are placed within
mouse preimplantation blastocysts and these are implanted into
pseudopregnant mice.  The genes express themselves in the adult
tissue and in the germ line cells as well.
<p>
RESULTS.  Preliminary work with mouse L
cultured cells and mouse eggs show they can be injected and will express
the herpes simplex virus thymidine kinase structural gene under the
control of the mouse metallothionein (MT-I) promoter;  the plasmid
containing this fusion of MT-I and HSV<i>tk</i> is pMK.  Induction of
TK expression is possible with the presence of heavy metals.  200
copies of pMK are injected into the male pronucleus of fertilized one-cell
eggs.  16 eggs are transferred to the oviducts of each of 15 pseudo
pregnant mice.  Six of the 15 had litters, with a total 12 male and 7
female offspring.   At 4 weeks all males were mated to normal
females.  Mice are injected with 2&nbsp;mg/kg CdSO<sub>4</sub>,
a dose known to induce MT-I mRNA synthesis in liver
and kidney.  After 18 h, mice are euthanized, liver tissue samples
are prepared for TK assay (tritiated TMP product is adsorbed to DE-81
paper--while triated thymidine does not adsorb--to measure
phosphorylation), and the rest of the animal is analyzed for nucleic acid
presence.
<p>
One animal initially showed 40-fold greater liver
TK activity relative to the others; the activity was outside the linear
range of the assay, and when diluted, indicated at least 200-fold greater
activity than the others.  Since TK activity is endogenous in the
liver, the source of it was determined by adding antibody specific for
HSV<i>tk</i> to liver extracts, then measuring activity:  TK activity
was inhibited 97% while the activity in litter mates was unaffected.
Moreover, whereas endogenous mouse TK cannot phosphorylate
iododeoxycytidine (IdC) and HSV TK can, the presence of IdC should show a
marked inhibition (competitive) of tritiated thymidine if the TK is viral,
which it did.  <sup>125</sup>I-IdC was added
to liver extracts from normal mice and was converted to phosphorylated
derivatives, as deaminases can convert <sup>125</sup>I-IdC
to iododeoxyuridine which can be phosphorylated
by endogenous TK.  But when tetrahydrouridine (an inhibitor of
deaminase for IdC) is added, no conversion is seen except in the
HSV-TK-expressing mouse liver, which can phosphorylate IdC directly.
<p>
DNA from kidney was obtained, digested with
<i>Bst</i>EII, and then probed according to Southern with oligos which
detect endogenous MT-I gene and the fusion.  Endogenous MT-I is
located on a 6 kb fragment, while the MK fusion should be found on a 2.3
kb fragment.  The HSV-TK expressing mouse and three of its litter
mates showed positive for a 2.3 kb band.  Densitometric analysis
indicates the MK gene is half intense as MT-I in all mice except one
(19-2), which is different from the high liver TK producer (23-2);&nbsp;
19-2 showed an MK band six times more intense.  Gene copy number was
estimated by first hybridizing probes to equimolar amounts of MT-I and MK
genes as controls.  The genes were electrophoretically run with
increasing DNA mass from 40-160 pg of plasmid DNA and then
autoradiographed.  The MT-I gene was consistently 4-fold more intense
than the MK gene, and since the MT-I gene band was twice as intense as the
MK gene band,  there must be twice as many MK genes than MT-I genes
per cell.  Knowing that there are two MT-I genes per cell, then
inference shows 4 MK genes are present in each cell.  Continuing this
inference, mouse 19-2 must have 48 copies of MK in each cell.
<p>
As mice 19-2, 21-3, and 21-3 showed MK gene
presence but no TK activity, these mice were checked for MT-I mRNA
induction by Cd by solution hybridization with
<sup>32</sup>P-MT-I cDNA.  All of the mice expressed between
600 and 2700 molecules MT-I mRNA per liver cell.  Since a previous
report shows the basal level, although variable, averages 150 molecules
per liver cell, while optimal induction is 2300 per cell, the failure to
observe TK activity was not a result of Cd failing to induce the
metallothionein promoter.  Solution hybridization of liver extract
with <sup>32</sup>P-HSV-TK cDNA showed that mouse
23-2 (which expressed excellent TK activity) had 28 molecules per
cell.  Mouse 19-2, which had nearly 50 copies of the gene, showed a
low amount of HSV-TK mRNA as well, and all other mice had &lt;2 TK mRNA
per cell.
<p>
Tissue distribution analysis indicated the presence
of the gene in liver, kidney, brain, muscle, and testis, with similar
intensities in each tissue.  Analysis of HSV-TK activity and mRNA
levels were lower in the kidney than in the liver, and were undetectable
in the brain.  This indicates the MK gene expression parallels MT-I
gene expression in the tissues.
<p>
DNA was taken from the four mice expressing MK gene
and digested with several enzymes that would cut twice, once, or not at
all with the pMK plasmid.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Hager, L. J., Palmiter, R. D.
 </span>
 <span class="year">
(1981)
 </span>
 <span class="jname">
Nature (Lond.)
 </span>
 <span class="jvol">
291,
 </span>
 <span class="pages">
340-342.
 </span>
<p class="reftitl">
Transcriptional regulation of mouse liver metallothionein-I gene
by glucocorticoids
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Jaken, S., Tashjian, A. H., Blumberg, P. M.
 </span>
 <span class="year">
(1981)
 </span>
 <span class="jname">
Cancer Res.
 </span>
 <span class="jvol">
41,
 </span>
 <span class="pages">
2175-2181.
 </span>
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Seamon, K. B., Daly, J. W.
 </span>
 <span class="year">
(1981)
 </span>
 <span class="jname">
J. Cyclic Nucleotide Res.
 </span>
 <span class="jvol">
7,
 </span>
 <span class="pages">
201-224.
 </span>
<p class="reftitl">
Forskolin:  a unique diterpene activator of cyclic AMP-generating systems
<!-- ============================================================= -->
<p class="tags">
radioallergosorbent test, RAST
<p class="ref">
 <span class="authors">
Gleich GJ, Yunginger JW
 </span>
 <span class="year">
(1981)
 </span>
 <span class="jname">
Bull. N. Y. Acad. Med.
 </span>
 <span class="jvol">
57,
 </span>
 <span class="pages">
559-567.
 </span>
<p class="reftitl">
The Radioallergosorbent Test: A Method To Measure IgE Antibodies, IgG Blocking
Antibodies, And The Potency Of Allergy Extracts
<p class="Para1">
First described in 1967 by Wide, Bennich and Johansson as in vitro diagnostic
for allergen antibodies.  Allergens are covalently bound to solid phase
polysaccharides (e.g., Sephadex) using CNBr.  The solid phase allergen is
incubatd with serum and IgE antibodies react with solid phase.  Unbound
material is washed away.  After washing radioiodinated affinity-purified
anti-IgE antibody is incubated.  The higher the IgE from standard or sample
in the firt incubation, the higher the anti-IgE bonding and therefore the
higher the counts in the gamma counter. The RAST interference assay is used
to measure IgG blocking antibody.
<p>
To verify the power of RAST, it was compared to provocation and skin tests
for danders and pollens.  RAST agreement with skin test 68%, and with the
provocation test 98%.  Studies in children attempted to determine its
sensitivity and specificity for bronchial, conjuctival, and nasal provocation;
if provocation was positive, it was evidence of allergy.  Agreement overall
was 73%.  10-16% of cases were false negative.  Another study found the same
73% value between positive RAST and allergy diagnosis.  Fish allergies were
correlated at 93% and dust was 59%.  An <i>r</i> = 0.89 correlation constant
was found between Prausnitz-Kustner passive transfer  test (IgE) and RAST.
Dose-response in RAST relating total bound label to serum volume added with
log-log shows good constants on linear regression.
<p>
RAST inhibition is a competitive assay in which solid phase Ag is mixed with
soluble Ag in the presence of IgE.  The more soluble Ag present, the more it
competes for binding with IgE compared to solid phase Ag.  In the first step
wash IgE is washed away with soluble Ag, and so less radiolabel with anti-IgE
is bound.  This allows comparison of claims of the effectiveness of allergens
(RAST potencies).
<p>
With use of RAST came interfering factors.  These are antibodies that compete
with IgE for binding to the solid phase allergen.  IgG has been implicated
as an interfering substance.  In using immnotherapy for short ragweed extracts,
there were dramatic drops of IgE levels as shown by RAST.  Sera from these
patients had to have the endogenous IgE destroyed without destroying IgG
present to determine if IgG is invovled in RAST interference.  It was clear that
heating the serum destroyed IgE while it preserved IgG.  Tests showed that
nonimmunized serum and normal serum did not interfere, but heat-treated
allergic serum had an interference from 63-69%.  Another source of interference
was allergen available:  by using Sepharose instead of microcrystalline
cellulose, interference related to the higher binding capacity of Sepharose
was reduced.
<p>
Another finding is that heat-treated IgE denatured antigenic determinants on
IgE that were recognized by anti-IgE labeled Ab.  To determine if IgG was
the blocking antibody reacting with allergen, solid phase short ragweed is mixed
with heat-treated serum in absence of allergic serum pool.  After overnight
incubation and wash, then the allergic serum pool is added.  After a second
overnight incubation, the anti-IgE is incubated.  67% interference is detected
when the heated serum is reacted first in the absence of IgE.  The interference
must occur at antigenic site and not by any reaction with IgE antibody.  When
competitive soluble Ags  were used, one ragweed and one rye grass, it was the
ragweed allergen that inhbited, not rye extract.  To show evidence that IgG
binds allergen, it was purified on DEAE and Sephadex G-200 and fractions containing
and not containing IgG were tested to prove that the interference was IgG.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Breathnach, R., Chambon, P.
 </span>
 <span class="year">
(1981)
 </span>
 <span class="jname">
Annu. Rev. Biochem.
 </span>
 <span class="jvol">
50,
 </span>
 <span class="pages">
349-383.
 </span>
<p class="reftitl">
Organization and expression of eukaryotic split genes coding for proteins
<!-- ============================================================= -->
<p class="tags">
anti-delta Ag, ELISA
<p class="ref">
 <span class="authors">
Crivelli O, Rizzetto LM, Lavarini C, Smedile A, Gerin JL
 </span>
 <span class="year">
(1981)
 </span>
 <span class="jname">
J. Clin. Microbiol.
 </span>
 <span class="jvol">
14,
 </span>
 <span class="pages">
173-177.
 </span>
<p class="reftitl">
Enzyme-Linked Immunosorbent Assay for Detection of Antibody to the Hepatitis B 
Surface Antigen-Associated Delta Antigen
<p class="Para1">
The &delta; antigen is associated with the hepatitis B surface antigen (HBsAg). 
This antigen is produced by a defective agent that is transmissible and whose 
expression requires also infection by HBV. This antigen appears to be produced 
by the liver cell in quantities too small for direct detection, but the 
anti-&delta; antibody in blood can be detected. The anti-&delta; antibody is 
associated with HBsAg carriers with chronic hepatitis.  A solid-phase radioimmunoassay
and immunofluorescence blocking test (IFL) with low sensitivity have been reported.
<p>
To develop an ELISA, a crude liver homogenate from 50 g tissue from an 
HBsAg<sup>+</sup> patient with chronic hepatitis was obtained at autopsy who 
also was seropositive for &delta; antigen.  Tissue was minced and washed in cold 
saline until supernatant fluid was clear.  Tissue was suspended in 9 vols 0.5 M 
sucrose + 5 mM TrisHCl pH 7.4 + 25 mM KCl + 2 mM MgCl<sub>2</sub> (TKM buffer + 
sucrose).  Homogenization was with Ultra Turrax followed by Dounce homogenizer. 
Solid guanidine HCl was added to 6 M, and stirred 4 h at RT; this was then 
diluted 5-fold with PBS (10 mM phosphate pH 7.4 with 0.85% NaCl). The 1.2 M guan 
HCl solution is centrifuged 20 min @ 4000 rpm.  The supernatant becomes the 
stock &delta; reagent.  This same purification was also done on the livers of a 
patient who is seronegative for HBsAg (died of myocardial infarction) and from 
one with cirrhosis and positive for HBcAg. &delta; Ag was also partially 
purified (using another cited method) from the same liver (its nuclei) and used 
as a reference agent.  In addition a 22 nm form of HBsAg was purified from 
plasma of a carrier.  An anti-&delta; reagent was also used (serum was positive
also for HBsAg) who had chronic persistent hepatitis with high titer for &delta;
(1:10<sup>4</sup> to 1:10<sup>6</sup> by RIA), with also low titer (1:500)
for anti-HBc.  IgG was isolated using ammonium sulfate and DEAE ion-exchange, 
then labeled with HRP.  A reference antiserum to HBsAg subtype <i>ad</i>
was used.  Anti-HBc and anti-HBe were previously described and used; both
were negative for &delta; Ag and anti-&delta; Ab.  ELISA development used
polyvinyl microwell plates coated with 1:1000 anti-&delta; Ab with 150 &micro;l
volume and incubated 4 h @ RT.  <q>Repeated</q> PBS washings then led to
blocking with 200 &micro;l PBS + 1% BSA and incubated 4&deg; overnight.  After
washing and drying, plates are inoculated with 25 &micro;l optimal dilution of
standard &delta; reagent (previously established by ELISA titration) and
incubated overnight 4&deg;  After washing, duplicate wells are treated with 
100 &micro;l test sera dilutions, incubated 2 h at 37&deg;, plates washed, then
filled with standard HRP-conjugated anti-&delta; IgG, incubated another 2 h
at 37&deg;, washed, and HRP developed.  Positive controls include 1:20 anti-&delta;
reagent, 1:20 nega+tive control; and blank (PBS+1%BSA).  Diluent for all sera
and reagents (other than HRP-anti-&delta;) is PBS+1%BSA. HRP-anti-&delta; is
diluted in 50% FBS in PBS. Freshly made 400 &micro;g/ml <i>o</i>-phenylenediamine
+ 0.0006% hydrogen peroxide is development reagent.  Reaction is stopped with
50 &micro;l 2 M sulfuric acid.  The ELISA is scored semi-quantitatively by
eye, with various brown coloring graded as +, ++, +++, and so on, with the more
brown color corresponding to less positive result.
<p>
[More on this necessary?]

<!-- ============================================================= -->
<p class="tags">
Ab engineering
<p class="ref">
 <span class="authors">
Sch&ouml;nherr OT, Roelofs H
 </span>
 <span class="year">
(1981)
 </span>
 <span class="jname">
Dev. Biol. Stand.
 </span>
 <span class="jvol">
50,
 </span>
 <span class="pages">
235-242
 </span>
<p class="reftitl">
Monoclonal antibodies for diagnostic tests and affinity chromatography: a first 
step to antibody engineering
<p class="Para1 abstract">
ABSTRACT. The hybridoma technology for obtaining murine monoclonal antibodies 
has been developed for the production of antibodies to human gonadotrophins and 
viral antigens. Various aspects of the hybridomna selection strategy and the 
cloning to continuous and stable antibody producing cell lines are reported. 
This leads to production of antibodies of predefined characteristics (antibody 
engineering). The application of monoclonal antibodies for diagnostics and for 
isolation of products in industrial processes are indicated.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
B&uuml;rki, K., Ullrich, A.
 </span>
 <span class="year">
(1982)
 </span>
 <span class="jname">
EMBO J.
 </span>
 <span class="jvol">
1,
 </span>
 <span class="pages">
127-131.
 </span>
<p class="reftitl">
Transplantation of the human insulin gene into fertilized mouse eggs
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Lamers, W. H., Hanson, R. W., Meisner, H. M.
 </span>
 <span class="year">
(1982)
 </span>
 <span class="jname">
Proc. Natl. Acad. Sci. USA
 </span>
 <span class="jvol">
79,
 </span>
 <span class="pages">
5137-5141.
 </span>
<p class="reftitl">
cAMP stimulates the transcription of the gene for
cytosolic phospho<i>enol</i>pyruvate carboxykinase in rat liver nuclei
<p class="Para1">
<b>ABSTRACT</b>. The effects of starvation, glucose refeeding, dibutyryl cAMP,
and dexamethasone on expression of the gene for phosphoenolpyruvate
carboxykinase (GTP) [GTP:oxaloacetate carboxy-lyase (transphosphorylating), EC
4.1.1.32] from rat liver cytosol was studied by using a cloned cDNA probe. The
rate of transcription of the gene for phosphoenolpyruvate carboxykinase in
hepatic nuclei isolated from starved rats decreased rapidly after refeeding with
glucose. Administration of dibutyryl cAMP to glucose-refed animals increased the
rate of phosphoenolpyruvate carboxykinase gene transcription seven-fold within
20 min. Phosphoenolpyruvate carboxykinase mRNA in the cytosol is 2.8 kilobases
long whereas liver nuclei contain four precursor RNA species that are up to 6.5
kilobases long. Feeding glucose to starved rats rapidly decreased the sequence
abundance of enzyme mRNA in both nuclei and cytosol. However, the decrease in
cytosolic phosphoenolpyruvate carboxykinase mRNA was preceded by a transient
increase in enzyme mRNA over the first 20 min after glucose refeeding.
Administration of dibutyryl cAMP to glucose-refed starved animals increased the
concentration of the nuclear RNA precursors of phosphoenolpyruvate carboxykinase
five- to eight-fold within 30 min and induced the mRNA for the cytosolic enzyme
over a period of 60 min. We conclude that cAMP induces phosphoenolpyruvate
carboxykinase mRNA by increasing the rate of gene transcription.
<p>
<b>Methods</b>. Nuclei from liver or normal or adrenalectomized rats were isolated.
Cytosolic RNA was obtained by lysing nuclei in 0.25 M sucrose-citric acid pH 2.3 with
0.5% Triton X-100 and centrifuging 2000 <i>g</i> 30 min at 0&deg; on 1.8 M sucrose
with 1.0 M sucrose cushion on it; 0.25 M and 1.0 M sucrose cushions combined
and spun 30 min 20,000 <i>g</i> for cytosolic RNA.  The nuclear pellet  is suspended
in 0.25 M sucrose and citric acid with 0.1% TX-100 and layered on to 1.0 M sucrose
with 20 mM KOAc pH 4.5 and TX0100 and spun 1000 <i>g</i> 20 min; its pellet resuspended
and spun 10 min 600 <i>g</i>.  The nuclear pellet is dissolved in 4 M GuanSCN
with 0.5% Na N-lauroylsarcosine with 20 mM EDTA, 0.2 M 2-mercaptoEtOH, 50 mM PIPES pH 7
and CsCl added to 1.4 M.  This is layered to 5.7 M CsCl/0.1 M EDTA pH 7.0 and
centrifuged 25&deg; 16-18 h @ 150,000 <i>g</i>.  RNA in pellet is dissolved
in 10 mM HEPES pH 7.5, 1 mM EDTA, 0.1% SDS and extracted chloroform/isoamyl alcohol
(24:1).  RNA yield is 50-100 &micro;g/g liver.  Cytosolic RNA is purfied as well.
For <b>isolation of nuclei used in determination of RNA synthesis</b>, livers are homogenized 4&deg; in 5-10
vol 0.3 M sucrose, 5 mM DTT, 5 mM magn chloride, 10 mM TrisHCl pH 7.5 with 0.1%
Triton X-405 (Medium A), filtered through 4 layers cheesecloth, spun at 800 <i>g</i> 5 min.
Nuclei are homogenized in Medium A with no TX-405, sucrose added to 1.65 M,
mixture centrifuged to 25,000 <i>g</i> for 60 min 4&deg; with 2 M sucrose
2 mM magn chloride, 10 mM TrisHCl pH 7.5 cushion.  Pellet homogenized in 50%
glycerol, 5 mM magn chloride, 0.1 mM EDTA, 50 mM HEPES pH 7.5.  Nuclei counted
using phase contrast microscope then frozen at &minus;80&deg; at 5-10 &minus;10<sup>&minus;8</sup>/ml.
For <b>measurement of RNA transcription</b>, the frozen nuclei were thawed
and 25 &micro;Ci of &alpha;-[<sup>32</sup>P]-UTP in 25% glycerol, 2.5 mM magn
chloride, 50 &micro;M EDTA, 75 mM HEPES pH 7.5, 0.1 M KCl, 4 mM DTT, 0.5 mM CTP
and GTP, 1.0 mM ATP, 40 &micro;g/ml creatine kinase, 8.8 mM creatine phosphate
were added to 20-40 million nuclei in a 0.2 ml volume.  Reactions are terminated
by adding 15 &micro;g/mol RNase-free DNase with 5 mM magn chloride for 10 min
@ 37&deg;.  Digestion with proteinase K (90 &micro;g/ml with 0.5% SDS, 5 mM EDTA)
for 20 min 37&deg;.  Extaction with phenol:chloroform 1:1 several times, then
addition of NaOAc to 0.2 M and 2 vol EtOH at &minus;20&deg; for several hours
to precipitate.  Another round of DNase digestion and cleaning up RNA is done.
Nitrocellulose is boiled, soaked in 3.0 M NaCl/0.3 M sodium citrate and baked
at 75&deg;.  A cDNA of the gene of interest is boiled 5 min in 0.1 M NaOH and
then cooled and 2.0 &micro;g in 20 &micro;l applied to 1-cm<sup>2</sup> of
prepared nitrocellulose filter.  The dried spot is basked 2 h at 75&deg;.
To determine how much cDNA is bound, perchloric acid is added to release the
DNA from filter and measured at OD260.  Prehybridization is done and the
phosphate-labeled transcribed RNA added, along with a tritium-labeled cRNA.
Filters are washed with a decreasing levels of SSC containing EDTA.
<p>
<b>Results</b>.  Transcription rate was adjusted to give linear rate of UTP
uptake into total RNA.  Preliminary expts shows that 2.5 mM MgCl<sub>2</sub>
gives maximal rate while adding Ca<sup>2+</sup> or Mn<sup>2+</sup> was not
stimulatory. Relatively high 25% glycerol also critical for maximal synthesis.
[<sup>3</sup>H]UTP into TCA-pptble RNA is linear for 60 min at 22&deg;.
&alpha;-amanitin gives 50% inhibition at 0.5 &micro;g/ml.  With this medium
9-14% of [<sup>32</sup>P]UTP (5-7 pmol) into total RNA per 20-40 million nuclei
per hour.
<p>
The absolute rate of RNA synthesis required including [<sup>3</sup>H]cRNA used to determine
hybridization efficiency since synthesis rate uses hybridization as the means
of quantitation.  The rate of synthesis is computed as the cpm of incorporation
into pPCK2 (plasmid containing the 600 bp insert of P-enolpyruvate carboxykinase
into the PstI site of pBR322) less the cpm of incorporation into the pBR322
vector itself, and the difference divided by the cpm of the total RNA; this
quotient is multiplied to 100% divided by the efficiency of hybridization.
This finally adjusted by multiplying by 2800/600:  where 2800 is length of a
processed PEP-carboxykinase mRNA and 600 is the length of the pPCK2 in base pairs.
The basal synthesis rate of P-enolpyruvate carboxykinase RNA
was 0.05% or 500 ppm in adrenalectomized and control glucose re-fed rates.  this
indicates glucocorticoids are not necessary to maintain basal enzyme RNA synthesis.
In glucose-refed adrenalectomized rats injected with Bt<sub>2</sub>cAMP and theophylline
the rate of synthesis was increased 7-fold after 20 min to 3415 ppm (0.34%) and
by 60 min it decreased to 1779 ppm.  For control rates the increase was 4-fold
after 20 min.  With dexamethasone-treated rats the transcription rate increase was
only 35% above control at 60 min.  &alpha;-amanitin at 0.5 &micro;g/ml reduced
the basal synthesis of both total RNA and PEP carboxykinase RNA to about 260 ppm.
Glucose re-feeding of starved rats reversed the induction of PEP-CK mRNA:  in
starved rats the level of RNA synthesis is 3676 ppm, close to the rate of nuclei
from animals injected with Bt<sub>2</sub>cAMP.  Addition of &alpha;-amanitin
was sufficient to block RNA polymerase II of PEP-CK RNA synthesis by 92%.  Glucose
refeeding produced rapid decrease, reaching basal levels after 2 h.
Bt2cAMP/theophylline adminstration stimulted synthesis by 6-fold within 20 min.
<p>
Liver nuclei have several RNA species hybridizing to PEP-CK cDNA probe.  These
RNA species range in size from 1.0 to 6.5 kb, and four species are larger than
than the 2.8 kb mature PEP-CK mRNA.  Three smaller persistent species might be
breakdown products.  These speciesa are polyadenylated, and none are observed
in the rat liver cytosolic RNA.  When glucose is given to the starved rats, the
levels of these nuclear PEP-CK mRNA precursors to bare detectability within 2 h.
Bt<sub>2</sub>cAMP administration brought these species levels up 5- to 8-fold
inside 20 min.  Adrenalectomized rats injected with dexamethasone showed
an increase in sequence abundance of precursors but the induction was not as
marked as for Bt<sub>2</sub>cAMP.  In the cytosol PEP-CK mRNA is present as
a 2.8 kb species.  Feeding glucose to starved rat results in transient increase
in the species with a rapid decrease in sequence abundance (half-life 20 min).

<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Gorman, C. M., Moffat, L. F., Howard, B. H.
 </span>
 <span class="year">
(1982)
 </span>
 <span class="jname">
Mol. Cell Biol.
 </span>
 <span class="jvol">
2,
 </span>
 <span class="pages">
1044-1051.
 </span>
<p class="reftitl">
Recombinant genomes which express chloramphenicol acetyltransferase
in mammalian cells
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Dickerson RE, Drew HR, Conner BN, Wing RM, Fratini AV, Kopka ML
 </span>
 <span class="year">
(1982)
 </span>
 <span class="jname">
Science
 </span>
 <span class="jvol">
216,
 </span>
 <span class="pages">
475-485.
 </span>
<p class="reftitl">
The Anatomy of A-, B-, and Z-DNA
<p class="Para1">
Structural analysis of drawn DNA fibers produced by varying salt, alcohol,
and humidity and using circular dichroism, IR spectroscopy revealed
DNA assuming A and B forms.  These differ by a sequence-dependent base
stacking and sugar-phosphate backbone helical twisting.
<p>
The ability to synthesize DNA using triester methods made possible preparation
of tens of milligrams of pure DNA with pre-determined sequence, which
can be crystallized for x-ray analysis.  The tetramer pApTpApT did not
form a double helix, but assumed A-, B-, and Z-DNA forms.
<p>
Diiodo-CpCpGpG (CCGG) and octamer GGTATACC have been crystallized
to show A-DNA.  The dodecamer CGCGAATTCGCG and its isomer which is the
5-bromocytosine at the 9th position form B-DNA.  Daunomycin also contributes
to B-DNA by intercalating between the CpG in CGTACG.
<p>
Z-DNA is a left-handed form seen with the tetramer CGCG in low- and high-salt.
It is also formed in CGCGCG hexamer with cationic variants.  Besides the
left-handed turn of the helix, a second feature that gives Z-DNA its name
is the use of two consecutve nonequivalent base pairs as the helical
repeating unit which give a zigzag conformation to the sugar-phosphate
backbone.
<p>
The characteristics of these DNA forms is described in the table:
<table>
<col>
<col span="3" style="width:12em;">
<tr><th>Feature     <th>A             <th>B              <th>Z
<tr><td>general <td>wide, stubby <td>slimmer, elongated
  <td>thinner, very elongated, &ldquo;inverse of A&rdquo;
<tr><td>bases   <td>canted sharply to helical axis
   <td>perpendicular to helical axis <td>perpendicular stacking
<tr><td>major groove<td>cavernous <td> wide
   <td>completely flattened out on surface of molecule
<tr><td>minor groove<td>too shallow to be a groove <td>cavernously deep
  <td>narrow
</table>
<p>
B-DNA has 10 bases per helical turn, for a base rotation of 35.9&deg;.
The standard deviation is 4.2&deg;, indicating a wide variability, with a
range between 27.4 and 41.9&deg; (from 13.1 to 8.6 bases per turn).
<p>
Sequence-dependence on the twist of the bases has been revealed through
DNase I, whose rate constant is correlated with the twist angle on the
phosphate.  Generally only B-DNA appears to rely much on sequence-dependent
base rotation.
<p>
The protein backbone features planar amide bonds that means that torsion
angles need only be described by &phi; and &psi; with permitted conformations
seen in Ramachandran plots.  DNA must be described by six torsion angles,
and &delta; is a primary one because it describes sugar puckering, or
the deoxyribose angle.  One atom of the furanose will be either to one
side or the other, and the conformation is described as Xn&prime;-<i>endo</i>
or Xn&prime;-<i>exo</i>, where X is the atom and n is the number of the atom,
as in C5&prime;-<i>endo</i>.  Crystal structure analysis can usually resolve
the &delta; angle to within 5 or 10&deg;.  C3&prime;-<i>endo</i> and
C2&prime;-<i>exo</i> have been assumed for A-DNA, C2&prime;-<i>endo</i>
and C3&prime;-<i>exo</i> for B-DNA, but these are oversimplifications.
<p>
The glycosyl angle &chi; is that between the C1&prime; atom of the sugar
and N of the attached base.  When the ring is turned so that it bent toward
the minor groove, this is called a <i>syn</i> conformation (&chi; has a value
around +70&deg;).  The <i>syn</i> conformation is restricted for purines
because of close contacts between the sugar ring and purine N3 atom.  This
conformation is disfavored for pyrimidines since the O2 of the six-membered
ring is brought too close to the sugar.  The <i>anti</i> conformation is
far less restrictive and bends the sugar away from the minor groove.
&chi; can have a large range of values from &minus;80 to &minus;180&deg;.
Thus in A- and B-DNA, the <i>anti</i> conformation is found.  Z-DNA
has the zigzag path because of a <i>syn</i> conformation at guanines
and <i>anti</i> conformation at cytosines.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Perferoen, M., Huybrechts, R., De Loof, A.
 </span>
 <span class="year">
(1982)
 </span>
 <span class="jname">
FEBS Lett.
 </span>
 <span class="jvol">
145,
 </span>
 <span class="pages">
369-373.
 </span>
<p class="reftitl">
Vacuum-blotting:  a new simple and efficient transfer of proteins
from sodium dodecyl sulfate-polyacrylamide gels to nitrocellulose
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Weiss SJ, Klein R, Slivka A, Wei M
 </span>
 <span class="year">
(1982)
 </span>
 <span class="jname">
J. Clin. Invest.
 </span>
 <span class="jvol">
70,
 </span>
 <span class="pages">
598-607.
 </span>
<p class="reftitl">
Chlorination of Taurine by Human Neutrophils
<p class="Para1">
Myeloperoxidase in the neutrophil is able to employ hydrogen peroxide to
oxidize a variety of anions, namely halides such as Cl<sup>&minus;</sup>,
Br<sup>&minus;</sup>, and I<sup>&minus;</sup>, but also thiocyanate
(SCN<sup>&minus;</sup>).  Since Cl<sup>&minus;</sup> is present abundantly,
it is thought that this acid-catalyzed reaction occurs:
<p class="eqnserif" style="text-align:center;">
H<sub>2</sub>O<sub>2</sub> + Cl<sup>&minus;</sup> &rarr;
HOCl + H<sub>2</sub>O
<p>
Hypochlorous acid is an extremly powerful oxidant, shown to occur in a model
system and the enzyme produces this in vitro, thus it is believed to be
a physiological occurrence.  Phorbol myristate acetate (PMA)-stimulated
human neutrophils were shown to destroy cell targets in a myleoperoxidase
dependent process inhibited using HOCl scavengers.  Attempts to measure
HOCl are complicated by its high reactivity and short half-life.  Taurine
is a &beta;-amino acid that will react with HOCl to form a stable chloramine
derivative, which can be assessed.  To quantitate taurine chloramine,
formation of taurine chloramine was tested first with NaOCl as a limiting
reagent to taurine.  OCl<sup>&minus;</sup> absorbs at 292 nm at pH 12.0
with assumed extinction coefficient of 350 / M-cm (see Morris (1966)
J. Phys. Chem. 70: 3798, the acid ionization constant of HOCl from 5-35&deg;).
The absorbance spectrum from 360 to 220 nm was measured for (a) taurine,
(b) OCl<sup>&minus;</sup>/HOCl, (c) taurine + OCl<sup>&minus;</sup>/HOCl
with 5-fold more taurine, (d) OCl<sup>&minus;</sup>/HOCl + H<sub>2</sub>O<sub>2</sub>
and (e) taurine chloramine with peroxide present.  Taurine does not absorb at
all in this range.  OCl<sup>&minus;</sup>/HOCl shows absorption at 292 nm
as reported, which is lost when peroxide is added.  Taurine + hypochlorite
a strong absorption around 250 nm typical of monochloramines (dichloramines
absorb with peak about 300 nm and were not detected), and isolated taurine
chloramine with peroxide present also shows a similar spectrum as taurine
+ hypochlorite.  Spectrophotometric determination however is insensitive,
so the oxidizing potential of taurine chloramine was assessed using
5-thio-2-nitrobenzoic acid (TNBA).  For every mole of taurine chloramine,
2 mol TNBA (or 1 mol of its precursor Ellman's reagent) is consumed.  Formation
of I<sub>2</sub> from I<sup>&minus;</sup> is done by measuring periodide
formation I<sub>3</sub><sup>&minus;</sup> in presence of excess I<sup>&minus;</sup>.
Taurine chloramine is assayed by adding TNB (to 0.45 mM) or potassium iodide
(to 20 mM), centrifuging (500 g/5 min) and reading the supernatant at 412
or 350 nm, respectively.
<p>
<b>RESULTS</b>.  H<sub>2</sub>O<sub>2</sub> was added to purified myeloperoxidase
(16 mU) in Dulbecco's pH 7.4 with 5 mM taurine in 1 ml reaction and kept 37&deg;
for 60 min then assayed by TNB method.  Plot of taurine chloramine (nmol)
vs. H<sub>2</sub>O<sub>2</sub> present linear from 0-100 nmol H<sub>2</sub>O<sub>2</sub>.
with 1 nmol taurine chloramine per mol H<sub>2</sub>O<sub>2</sub> used.
This is a 100% conversion.  If 0.1 mM (100 nmol in the 1 ml reaction) instead
of 5 mM taurine were used, it was nearly always consumed as well.
If taurine were not present in the reaction, then added after 60 min,
&lt; 5% of chloramine was formed relative to adding taurine at the start.
This is likely because myeloperoxidase generates the HOCl and is inactivated
by this product when no reactant is present to scavenge it; moreover,
residual HOCl is reduced by H<sub>2</sub>O<sub>2</sub> itself.
<p>
Table 1 shows what each component does.  With taurine in the presence of
peroxide and chloride, no chloramine is formed; thus enzyme is required.
The same is true for  taurine, peroxide, and enzyme in a chloride-free
buffer (Dulbecco's phosphate); thus chloride is required.
If catalase (50 &micro;g/ml) is present in the complete system (taurine,
enzyme, chloride as Dulbecco's buffer, peroxide), only 4-5% of chloramine is
formed, since catalase degrades peroxide.  When heat-inactivated catalase is
added, the reaction is like the complete system.  With superoxide dismutase
present (10 &micro;g/ml) in complete system, there is no effect;  SOD actually
produces peroxide from superoxide, and does not consume peroxide. Azide (1 mM),
cyanide (1 mM), and aminotriazole (10 mM) are established myeloperoxidase
inhibitors and they all effectively inhibited chloramine formation.
Ethanol (40 mM) and mannitol (40 mM) are well known hydroxyl radical scavengers
and had no effect:  thus hydroxyl radical is not a necessary intermediate.
<p>
Human neutrophils were obtained on Ficoll-Hypaque and then sedimented in dextran
(&gt; 96% purity, with 2-3% eosinophils), suspended in Dulbecco's PBS with
1 mg/ml glucose and then incubated with 100 ng/ml phorbol myristate acetate (PMA).
2.5 million PMA-treated neutrophils were then mixed with 15 mM taurine for 2 h.
Catalase was added and then TNB.  To the supernatant of the reaction cysteine
was added to reduced DTNB to TNB in some samples.  Spectra were obtained
(280-550 nm) DNTB (peak max 325 nm), for TNB (peak max 412 nm), TNB present
in the cell mixture (peak max 412 nm), and TNB+cell mixture with cysteine added
(peak max 325 nm).  These results show that something oxidized the TNB to form
DTNB, which was then reduced again to TNB by Cys.  Cells not treated with PMA
did not show this property.  Spectral scans were also obtained (with untreated
cells containing no taurine as a zero reference) for 20 million cells treated
or not with PMA and with or without taurine.  Only activated cells with taurine
caused chloramine formation (seen spectrophotometrically in the range 230-280 nm).
 Similar results were obtained with opsonized zymosan-activated neutrophils.
<p>
Cell extracts were placed on paper and high voltage electrophoresis (Whatman
3MM for 30 min in pH 9 borate buffer at 3000 V (150 mA current).  Ninhydrin
used to visualize taurine; 2% KI in acetone for taurine chloramine. (Standards
of taurine and taurine chloramine move 5.5 and 17.0 cm to anode.)  All these
data taken together show that neutrophils generate taurine chloramine.
<p>
PMA- (30 ng/ml) and opsonized zymosan (1.25 mg/ml)-activated cells were
incubated in increasing levels of taurine (0, 0.1, 1, 10, 15, 20 mM) and
taurine chloramine formation monitored.  There is a clear concentration
dependence with a plateau at about 15 mM taurine.  The respiratory burst
was also monitored and taurine did not affect that.  Peroxide release
(used by myeloperoxidase) was also not affected by taurine presence:
1 mM azide prompted the release of peroxide at levels similar when taurine
was present or absent.  Time dependence was also determined: there was
a clear time dependence, and taurine chloramine formation was maximal at
2 h under these conditions using either PMA- or opsonized zymosan-treated
cells.  In general PMA-treated cells alway produced more chloramine in all
test conditions.  The dependence on cell number in the assay was tested
for cell counts of 125,000, 250,000, 500,000, 1 and 2 million:  there was
a direct linear correlation between cell number and chloramine formed, which
makes sense since product formation is a function of the presence of the
product-making system.  A final experiment was to determine whether the
assay system of taurine chloramine product was reliable:  if the product is
unstable, then the assay system is flawed.  Thus the cell-based reactions
were spiked with 50 nmol taurine chloramine and the amount of exogenous
material recovered determined:  in all assays &gt; 95% was recovered.
<p>
The in vitro requirements for taurine chloramine production included
enzyme, chloride-containing buffer, and peroxide.  The tests for the in vitro
purified components were then tested.  2 million PMA- or zymosan-activated
cells were mixed with taurine and various additives.  With catalase present
90% and 65% reduction in chloramine was seen with PMA- and zymosan-treated
cells, which was abolished with use of heat-denatured catalase.  SOD had
no effect as before.  Azide at four levels (0.1, 1 10, and 100 &micro;M) had
increasing inhibitory effect, virtually complete at 10 &micro;M.  Cyanide
at 10 &micro;M was barely inhibitory, 70-80% inhibitory at 100 &micro;M,
and complete at 0.5 mM.  10 mM aminotriazole was thoroughly inhibitory.
Hydroxyl scavengers EtOH and mannitol (40 mM) did not reduce chloramine
production, as expected.  When chloride was not present, there was a marked
reduction in chloramine formation.  Exogenously added peroxide, myeloperoxidase,
and both peroxide and myeloperoxidase only partially alleviated this effect.

<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Furth, M. E., Davis, L. J., Fleurdelys, B., Scolnick, E. M.
 </span>
 <span class="year">
(1982)
 </span>
 <span class="jname">
J. Virol.
 </span>
 <span class="jvol">
43,
 </span>
 <span class="pages">
294-304.
 </span>
<p class="reftitl">
Monoclonal antibodies to the p21 products of the transforming gene
of Harvey murine sarcoma virus and of the cellular <i>ras</i> gene family
<p class="Para1">
Cell fusion was done using lymphocytes from individual rats and 8 hybrid myeloma 
lines were found to produce MAbs to p21 encoded by Harvey murine sarcoma virus 
(HMSV).  Antisera from tumor-bearing rats (injected with HMSV-transformed NRK 
cell syngeners) have been reported to show that p21 has a GDP-binding ability 
associated with it.  Moreover, these antisera show p21 to localize to the inner 
surface of the membrane of HMSV-transformed cells.  However, there is doubt as 
to whether these antibodies were to other proteins (antigens) normally expressed 
in untransformed cells.  Therefore, as done in previous reports, we incubated 
rat antisera obtained bearing HMSV-induced tumors with labeled GDP, partially 
pure p21, and protein A-Sepharose.  The same was done with the MAb to p21 
(Y13-238). Plots of pmol [<sup>3</sup>H]GDP bound vs.&nbsp;&mu;g p21 extract 
show that the curves for the MAb and the antisera show the same binding curves, 
while the control serum is flat (no increase in bound GDP).  The curves are 
directly dependent upon p21-containing extract used;  extracts from 3T3 cells 
show nothing of this profile.
<p>
HMSV-transformed cells were fixed with MeOH or acetone and treated with the 
antisera and then fluorescent anti-rat IgG.  The MDCK cell line demonstrates 
fluorescence in rings at the borders between cells, especially in the short 
microvilli interdigitating with other cells.  Immunofluorescence using the 
hybridoma supernatants for antibodies instead produced the same fluorescence 
pattern, whether the hybridoma produced good p21-immunoprecipitating abilities 
or whether it gave the highest level of p21-dependent GDP binding.
<p>
The Kirsten-murine sarcoma virus (KMSV) is also known to encode another form of 
p21 and was tested for its reaction with the MAb panel and antiserum.  Normal 
NRK fibroblasts and fibroblasts transformed by HMSV were labeled with [<sup> 
35</sup>S]Met were lysed and incubated with four anti-p21 MAbs.  Each 
immunoprecipitated a characteristic doublet present in the HMSV-transformed 
cells, but only two precipitated a single protein in the KMSV-transformed cells. 
These same two also precipitated a small amount (barely detectable) of 21 kDa 
protein in the uninfected NRK cells.  p21 from HMSV and KMSV were metabolically 
labeled with <sup>32</sup>P and found to bind to the MAbs, with the 
phosphorylated HMSV-encoded p21 running more slowly on SDS polyacrylamide gels. 
Only two of the four crossed-reacted with KMSV-encoded p21, and there was no 
phosphorylated p21 in the NIH 3T3 controls.  The phosphorylated residue in these 
cases was threonine.
<p>
The v-Ha-<i>ras</i> and v-Ki-<i>ras</i> genes (carried by HMSV and KMSV, resp.) 
are two members of a conserved gene family related to cellular components in 
murine sarcoma virus (BALB-MuSV) and rat sarcoma virus (Ra-SV).  Our panel of 8 
MAbs to the HMSV p21 also cross react with the BALB-MuSV and Ra-SV p21 products. 
The normal rat genome also contains two genes closely related to v-Ha-ras 
(c-Ha-<i>ras1</i> and -<i>ras2</i>), in which the <i>ras1</i> product can 
transform normal 3T3 cells when ligated to the MLV LTR region and express high 
levels of p21 which can be immunoprecipitated to the anti-p21 serum.  We show 
that that all 8 (one somewhat weakly) react with the cellular counterparts of 
p21 as well for both ras1 and <i>ras2.</i>&nbsp; The murine hemopoietic cell 
line 416B also expresses p21 in high levels and reacted with 3 of the 8 
strongly, weakly with a fourth;&nbsp; hybridization reactions using 
v-Ha-<i>ras</i> and v-Ki-<i>ras</i> p21 cDNA probes show this p21 more related 
to the v-Ki-<i>ras</i> gene.
<p>
We tested other cell types against the panel as well.  We consistently found a 
21 kDa product immunoprecipitated with 3 of the 8 MAbs, and these 3 (Y13-4, 
Y13-128, Y13-259) were the same found to precipitate the KMSV-produced p21 and 
p21 found in 416B cells. This strongly suggests that rat cells normally express 
a p21 ras product which may be c-Ki-ras or a yet unidentified member with shared 
epitopes.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Greenberg, A., Zinder, O.
 </span>
 <span class="year">
(1982)
 </span>
 <span class="jname">
Cell Tiss. Res.
 </span>
 <span class="jvol">
226,
 </span>
 <span class="pages">
655-665.
 </span>
<p class="reftitl">
Alpha- and beta-receptor control of catecholamine secretion from isolated 
medullary cells
<!-- ============================================================= -->
<p class="ref">
 <span class="authors">
Beach, D., Durkacz, B., Nurse, P. 
 </span>
 <span class="year">
(1982)
 </span>
 <span class="jname">
Nature
 </span>
 <span class="jvol">
300,
 </span>
 <span class="pages">
706-709.
 </span>
<p class="reftitl">
Functionally homologous cell cycle control genes in
budding and fission yeast
<!-- ============================================================= -->
<p class="tags">
alpha2 macroglobulin, purification
<p class="ref" id="arnaud-1982">
 <span class="authors">
Arnaud P, Gianazza E
 </span>
 <span class="year">
(1982)
 </span>
 <span class="jname">
FEBS Lett.
 </span>
 <span class="jvol">
137,
 </span>
 <span class="pages">
157-161.
 </span>
<p class="reftitl">
A Two-Step Purification Procedure For &alpha;<sub>2</sub>-Macroglobulin Based On 
Pseudo-Ligand Affinity Chromatography
<p class="Para1">
&alpha;<sub>2</sub>-Macroglobulin (a2M) is a protein with <i>M</i><sub>r</sub>
725,000 and a major protease inhibitor in human plasma.  Current procedures use
ammonium sulfate and polyethylene glycol (PEG) precipitation then gel filtration
or ion-exchange chromatography.  After several sequential steps, yields are about
20%.  Using metal-chelate chromatography on an ammonium sulfate-precipitated load
recovery was 22%.  a2M over Cibachron blue followed by gel filtration gave a
75% yield; the problem is that this works for plasma from people who are homozygotes
of haptoglobin phenotype 1, who are one-fifth of Caucasians.
<p>
Human plasma from healthy individuals with phenotypes 1, 1-2, and 2 for 
haptoglobin was collected in citrate-soybean trypsin inhibitor (see Harpel, <i>J 
Exp Med</i> <b>138</b>: 508, 1973):  blood is collected in plastic (to minimize 
activation of Hageman factor) and mixed 9 parts with 1 part 0.5 mg/ml STI in 
3.8% sodium citrate.  Cells are pelleted at 500 &times; g for 15 min 4&deg;.  20 
ml plasma dialyzed overnight at 4&deg; with 800 ml 50 mM sodium acetate pH 5.0. 
Affigel blue (4.5 &micro;mol dye per ml gel) is equilibrated with acetate in 1.6 
&times; 70 cm column at room temperature.  Loading was 20 ml/h and fractios of 
1.25 ml collected at 4&deg;.  Column was washed with 75 ml (0.5 col vol) loading 
buffer, then 0-2.0 M NaCl gradient total vol=225 ml=1.6 col vol.  Column washed 
with 60.0 ml 2.0 M NaCl in buffer.  a2M-containing fractions pooled and 
concentrated to 5.0 ml under nitrogen, dialyzed for gel filtration.  AcA-22 gel 
filtration done on 2.5 &times; 100 cm column in 100 mM potassiium phosphate pH 
6.5 at 10 ml/h with 2 ml fractions.  Anti-a2M sera (DAKO) used for 
immunoelectrophoresis, fused rocked immunoelectrophoresis, and 
electroimmunoassay.  PAGE used for haptoglobin phenotype analysis.  Trypsin 
binding was done using soybean trypsin inhibitor and 
&alpha;-N-benzoyl-D,L-arginine-p-nitroanilide assay (BAPNA), with trypsin type 
XI bovine pancreas used from Sigma; p-nitrophenyl-p'-guanidobenzoate HCl assay 
shows this trypsin is 72% active.
<p>
Virca et al (1979) proposed use of Cibacron blue F3GA and gel filtration for
purifying a2M.  They had separated plasma proteins at pH 8.0 on Sepharose
matrix with bound Cibracron blue, and a2M was the first protein to elute.
With a higher capacity affinity column, a2M binds but elutes only with a salt
gradient, and it elutes with IgM and haptoglobin.  At pH 5, most plasma proteins
bind to the column, but haptoglobin and IgM are eluted in the 1.25-1.5 M NaCl
range while a2M is eluted at a higher 1.6-2.0 M range.  Haptoglobins with 
phenotype 1, 1-2, and 2 are eluted at same molarity.  High <i>M</i><sub>r</sub>
components such as C4, C1q, and fibrinogen are absent from a2M elution zone.
Immunoelectrophoresis and fused rocket immunoelectrophoresis of affinity fractions
show presence of pre-albumin, albumin,C3, ceruloplasmin, group-specific
component, IgG, and monomeric IgA.  All contaminants are removed by AcA-22
gel filtration, largely because a2M is much larger than these proteins.
This purification yields about 60% of a2M from the initial step with no 
loss of biological activity.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Young, R. A., Davis, R. W.
 </span>
 <span class="year">
(1983)
 </span>
 <span class="jname">
Proc. Natl. Acad. Sci. USA
 </span>
 <span class="jvol">
80,
 </span>
 <span class="pages">
1194-1198.
 </span>
<p class="reftitl">
Efficient isolation of genes using antibody probes
<p class="Para1">
The &lambda;gt11 vector described here
was constructed is a derivative of &lambda;gt7-<i>lac</i>5
(<i>b522</i> <i>nin</i>5) and &lambda;gt4 (<i>c</i>I857 S100 <i>nin5</i>).
&lambda;gt7-<i>lac</i>5 and &lambda;540 (&Delta;B <i>imm</i>21
<i>nin</i>5) were cut with <i>Hin</i>dIII, fragments
pooled and ligated with T4 DNA ligase.  The desired recombinant
produces turbid (imm21) blue (lac5) plaques when naked DNA is transfected
to E coli BNN93 and cells are plated on X-Gal (5-bromo-4-chloro-3-indolyl
&beta;-<span style="font-variant:small-caps;">d</span>-galactoside).
The &lambda;gt7-<i>lac</i>5-&lambda;540 hybrid is then crossed
with &lambda;gt4 (<i>c</i>I857 S100 <i>nin5</i>) and the
recombinants grown at 42&deg; and scored for clear
(cI857) blue plaques.  Presence of the amber S100 was confirmed by
examining plating efficiency on hosts containing or lacking amber
suppressor <i>supF</i> (BNN45 or BNN93, resp.) The lac5 cI857 S100 phage
was mapped for <i>Eco</i>RI (one found) and for other enzymes.
<p>
The single <i>Eco</i>RI site is used for insertion of foreign DNA
and is located within <i>lacZ</i> and 53 bp upstream of
&beta;-galactosidase termination codon.  Phage with
inserts should not have an active &beta;-galactosidase
and recombinants and nonrecombinants are distinguished
by blue plaques in a <i>lacZ</i><sup>-</sup> host.
The vector is 43.7 kb long, and can take up to 8.3 kb of
foreign DNA (assuming a maximum 52 kb packageable genome).
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Frackelton, A. R. Jr., Ross, A. H., Eisen, H. N.
 </span>
 <span class="year">
(1983)
 </span>
 <span class="jname">
Mol. Cell. Biol.
 </span>
 <span class="jvol">
3,
 </span>
 <span class="pages">
1343-1352.
 </span>
<p class="reftitl">
Characterization and use of monoclonal antibodies for isolation of
phosphotyrosyl proteins from retrovirus-transformed cells and growth
factor-stimulated cells
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Jungman, R. A., Kelley, D. C., Miles, M. F., Milkowski, D. M.
 </span>
 <span class="year">
(1983)
 </span>
 <span class="jname">
J. Biol. Chem.
 </span>
 <span class="jvol">
258,
 </span>
 <span class="pages">
5312-5318.
 </span>
 <p class="reftitl">
Cyclic AMP regulation of lactate dehydrogenase.
<p class="Para1">
Isoproterenol and N<sup>6</sup>,O<sup>2</sup>'-dibutyryl cyclic AMP
increase the rate of transcription and change the stability of lactate
dehydrogenase A subunit messenger RNA in rat C6 glioma cells
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Wilbur, W. J., Lipman, D. J.
 </span>
 <span class="year">
(1983)
 </span>
 <span class="jname">
Proc. Natl. Acad. Sci. USA
 </span>
 <span class="jvol">
80,
 </span>
 <span class="pages">
726-730.
 </span>
<p class="reftitl">
Rapid similarity searches of nucleic acid and protein data banks
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Dignam, J. D., Lebovitz, R. M., Roeder, R. G.
 </span>
 <span class="year">
(1983)
 </span>
 <span class="jname">
Nuc. Acids Res.
 </span>
 <span class="jvol">
11,
 </span>
 <span class="pages">
1475-1489.
 </span>
<p class="reftitl">
Accurate transcription initiation by RNA polymerase II in
a soluble extract from isolated mammalian nuclei
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Jacobs, A., Kull, F. C., Earp, H. S., Svoboda, M., Van Wyk, J. J.,
Cuatrecasas, P.
 </span>
 <span class="year">
(1983)
 </span>
 <span class="jname">
J. Biol. Chem.
 </span>
 <span class="jvol">
258,
 </span>
 <span class="pages">
9581-9584.
 </span>
<p class="reftitl">
Somatomedin-C stimulates the phosphorylation of the &beta;-subunits
of its own receptor
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Anderson, D. J., Blobel, G.
 </span>
 <span class="year">
(1983)
 </span>
 <span class="jname">
Meth. Enzymol.
 </span>
 <span class="jvol">
96J,
 </span>
 <span class="pages">
111-120.
 </span>
<p class="reftitl">
Immunoprecipitation of proteins from cell-free translations
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Connolly, M. L.
 </span>
 <span class="year">
(1983)
 </span>
 <span class="jname">
J. Appl. Cryst.
 </span>
 <span class="jvol">
16,
 </span>
 <span class="pages">
548-558.
 </span>
<p class="reftitl">
Analytical molecular surface calculations
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Gubler, U., Hoffman, B. J.
 </span>
 <span class="year">
(1983)
 </span>
 <span class="jname">
Gene
 </span>
 <span class="jvol">
25,
 </span>
 <span class="pages">
263-269.
 </span>
<p class="reftitl">
A simple and very efficient method for generating cDNA libraries
<p class="Para1">
With the first reports of the molecular cloning of cDNAs by Rougen &amp; Mach
and Efstratiadis et al, the technology has evolved into powerful tool in molecular
analysis of eukaryotic and prokaryotic genes.  Techniques to obtain low
abundance mRNA molecules are wanted.  One technique is to eliminate the S1
nuclease-mediated cleavage of the hairpin loop (Land et al 1981, Okayama and
Berg 1982):  it causes considerable loss resulting in low clone numbers
generated by microgram mRN, also removing important 5&prime;-terminal sequences
from mRNA.  Okayama and Berg have described using a spcial plasmid-vector primer:
<ol>
<li>a cloning vector with a T-tail is used to prime the reverse transcription of the
mRNA into cDNA
<li>circularization results when a linker fragment is annealed and ligated to 
the recombinant construction
<li>second strand synthesis ocurs when RNA in hybrid is replaced with combiantion 
of RNase H, DNA polymerase I and DNA ligase before transfecting into competent
cells.
</ol>
<p>
This report is about the modification of the Okayama and Berg protocol.  It combines
the oligo(dT) primed 1st strand synthesis and the RNase H-DNA polymerase I-mediated
2nd strand synthesis.  The vector is not part of the priming and the S1 nuclease
hairpin loop cleavage.
<p>
Rabbit reticulocyte poly(A)<sup>+</sup> RNA was isolated and reverse transcribed
without use of labeled triphosphates.  Yields were determined using parallel 
reaction which included [&alpha;-<sup>32</sup>P]dCTP.  After 1st strand synthesis,
the RNA-DNA hybrid.  To determine if the RNA had been replaced, label was incorporated
into 2nd strand.  The count data show that incorporation of label into full length
globin DNA was dependent on addition of RNase H and DNA polymerase I.  Conditions
for producing nick translation with DNA polymerase I are used.  The AMV reverse
transcriptase does not have sufficient RNase activity to generate its own primers,
and so RNase H must be added.  It should be noted that addition of DNA ligase is
completely unnecessary.  The reverse transcriptase can also be used to synthesize
the 2nd strand, but are not full length.
<p>
S1 nuclease digestion of the globin ds DNA product shows 70% of radioactivity is
nuclease-resistant.  Some of products are twice size of original mRNA of about
600 nt; the hairpin loop of 1st strand might have been used as a priming structure
for 2nd strand.  Variation ranged to a few percent with respect to this oversizing
of molecules.
<p>
After 2nd strand synthesis, the globin ds DNA is tailed with dGTP, then annealed
to a <i>Eco</i>RV-cut and C-tailed pBR322 vector.  About 600,000 clones per microgram
of mRNA are generated.  When expressed this way, the conversion from hybrid to
ds DNA becomes irrelevant.  Ten randomly selected globin close showed the following:
(1) 5 in 10 clone are full length according to restriction analysis. (2) 2 in 10 
clones have a <i>Bam</i>HI site in insert (a characteristic of rabbit &beta;-globin
translated sequence), (3) one full-length &alpha;-gene clone and one &beta; clone
show recombinants missing 14 and 15 nt, respectivevely of the 5&prime;-UTR between
mRNA cap and start of sequence.
<p>
Vector pBR322 has a <i>Eco</i>RV site closer to the <i>Eco</i>RI which is 
converted to <i>Bam</i>HI.  The <i>Eco</i>RV generates blunt ends (not 3&prime; 
overhangs). The creation of <i>Bam</i>HI (GGATCC) is the result of C-tailing the 
blunt ends.  Thus the cDNA inserts can be sized by cutting
<pre>
       180       190       200
         |         |         |
      ---TGCGG<b>GATATC</b>GTCCATTCCG----
      ---ACGCC<b>CTATAG</b>CAGGTAAGGC----
                    |
                    |     EcoRV digest
                    V
      ---TGCGGGAT         ATCGTCCATTCCG----
      ---ACGCCCTA         TAGCAGGTAAGGC----
                    |
                    |    dCTP homopolyermic tailing
                    V
      ---TGCG<b><i>GGATCC</i></b>CCCC   ATCGTCCATTCCG----   BamHI site
      ---ACGCCCTA   CCCCCCTAGCAGGTAAGGC----
</pre>
<p>
The insert contains 188 bp DNA from pBR322, from the <i>Eco</i>RV site 
(pos 187) to the <i>Bam</i>HI site (pos 375).
<p>
To test less abundant mRNA, poly(A)<sup>+</sup> RNA from bovine adrenal meduall
was isolated.  The bovine pre-proenkephalin is about 1300 nt long and is only 
1 in 1000 molecules of the mRNA.  Its sequence has been reported by us and Noda
et al (1982).  The poly(A)<sup>+</sup> RNA was first sized in the 1000-3000 kb
range using sucrose gradient, then the cDNA library according to the protocol in
this report.  Different ratios of C-tailed pBR322 vector (kept constant at 15 ng)
to insert cDNA (none, 0.3, 0.6, 1.2, 2.4, 4.8, 9.6 ng) were tested such that
vector:insert ratios were 50, 25, 12.5, 6, 3, and 1.5.  Colony counts were 
18 (for no insert), 337, 230, 605, 448, 684, and 157, respectively, giving efficiencies
of none, 1123, 383, 504, 187, 143, and 16 colonies per ng cDNA.  Transformation 
with supercoiled pBR322 alone was 2 million colonies per microgram.  Taking the
highest efficiency point (50:1 vector:insert), it was scaled up by 100-fold using
30 ng ds G-tailed cDNA (from 200 ng mRNA), and mixed with 1.5 &micro;g pBR322 
vector, to generate 200,000 clones, for an efficiency of 10<sup>6</sup> clones
per &micro;g mRNA.
<p>
Inserts were analyzed by <i>Bam</i>HI digestion, and 8 in 12 clones had inserts
1 kb or larger.  As the published length of preproenkephaline mRNA is 1223 nt (not
including poly(A) tail), full-length clones are about 1500 bp:  <i>Bam</i>HI insert
of 1223 bp + 190 bp (portion of pBR322 vector) + 100 bp clone part of poly(A) tail
and GC-tails.  Two clones fullfilled the criteria.   Recombinants were digested
with <i>Nco</i>I as this is part of initiator AUG codon, then labeled by filling
with Klenow fragment polymerase or using polynucleotide kinase.  Cutting with 
<i>Bam</i>HI should yield molecules with only one end labeled.  One of the two 
clones was identical in size to the published cDNA.  Primer extension indicates
that the mRNA is 65 nt longer than the cDNA insert contained in the clone from
Noda et al.  Thus this method shown shows better complete full-length cloning of
genes than other methods.
<p>
In other generations of cDNA libraries, a consistent production of 300,000 clones
per microgram mRNA has been shown, which is a 10- to 50-fold improvement compared
to the S1 nuclease method.  Yields are also improved using sizing on sucrose gradients
of poly(A)<sup>+</sup> RNA.  Unfractionated RNA often gives smaller sizes.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Reimann EM, Beham RA
 </span>
 <span class="year">
(1983)
 </span>
 <span class="jname">
Meth. Enzymol.
 </span>
 <span class="jvol">
99,
 </span>
 <span class="pages">
51-55.
 </span>
<p class="reftitl">
Catalytic Subunit of cAMP-Dependent Protein Kinase
<p class="Para1"><b>Materials</b>.
<ul>
<li>DEAE-cellulose (Whatman DE-23)
<li>hydroxylapatite (BioRad BioGel HTP)
<li>Histone type IIA  (Sigma or Aldrich) made to 30 mg/ml then
  heated 100&deg; for 10 min.
<li>[<sup>32</sup>P]ATP:  in the procedure, the use of
ATP prepared according to Walseth &amp; Johnson,
<I>Biochem. Biophys. Acta</I> <B>562</B>, 11 (1979) was given.
Commercial ATP is fine however.
</ul>
<p>
Test the biochemical purity of ATP in a reaction which contains
0.01 kinase units/ml and 10 &mu;M labeled ATP.  The phosphate is
transferred quantitatively to histone in &lt; 30 min.
<p><b>Assay</b>.
This is the method given in the book; recommend using assay
method of Walsh instead, which uses Kemptide as substrate:
found in Glass et al., <I>J. Biol. Chem.</I> <B>264</B>: 14579, 1989
and elsewhere!)
<p>
This reaction has been modified from previous methods:
<ul>
<li>50 mM morpholinoethane sulfonic acid (MES)
<li>6 mg histone/ml
<li>10 mM magnesium acetate
<li>200 &mu;M [gamma-<sup>32</sup>P]ATP
<li>0.01 to 0.5 mU/ml kinase in 50 mM P<sub>i</sub> containing 1 mM EDTA
and 1.5 mg gelatin/ml
</ul>
<table>
<tr><th>Stock solution
 <th>Volume of stock solution to use in assay
 <th>final assay concentration
<tr><td>0.5 M MES (pH is not specified but pH 7.0 is used)
 <td>5 &mu;l
 <td>50 mM
<tr><td>30 mg/ml histone
 <td>10 &mu;l
 <td>6 mg/ml
<tr><td>0.1 M MgOAc + 2 mM [<sup>32</sup>P]ATP
 <td>5 &mu;l
 <td>10 mM Mg + 200 &mu;M ATP
<tr><td>enzyme or water
 <td>combination adds up to 30 &mu;l
 <td>
<tr><td>
 <td>final assay volume = 50 &mu;l
 <td>
</table>
<p> Incubate 10 min, take 40 &mu;L aliquots on to 2 &times; 2 pieces of P81
cellulose, drop into beaker containing 50 mM NaCl.  Wash four times with 50 mM
NaCl for 5 min ea., then wash in acetone 5 min.  Let dry. Count.  1 unit = 1
&mu;mol phosphate to histone in 1 min under these conditions.
<p><b>Preparation</b>The whole procedure for purification can be accomplished in
two days. All inorganic phosphate buffers contain equal parts of mono- and
dibasic KP<sub>i</sub>, 0.1 mM DTT, and other additives as indicated.
<ol><li>All steps at 4&deg;.  Two beef hearts (~800 g ea) are homogenized in 10 mM
P<sub>i</sub> containing 1 mM EDTA.
<li>Spin the homogenate 10,000 <I>g</I> for 30 min;  filter the supernatant
through cheesecloth and glass wool.
<li>Apply supernatant to DEAE column (7.5 x 20 cm);  wash ON with 5-10 L of 55
mM P<sub>i</sub> + 1 mM EDTA.
<li>Wash the column with 400 mL of 45 mM P<sub>i</sub> containing 10-100 &mu;M
cAMP, followed by 2 L of 45 mM P<sub>i</sub> buffer. Approx. ONE BED VOLUME
after beginning the cAMP wash, the A<sub>260</sub> (or A<sub>280</sub>) will
drop by a factor of 2 due to the DECREASING IONIC STRENGTH.  This change in the
absorbance signals the start of the elution of the C subunit.
<li>At this point, the column outlet is DIRECTLY CONNECTED to a hydroxylapatite
column (2.6 x 4 cm) which has been equilibrated in 45 mM P<sub>i</sub>.  After
~1000 mL have been collected, the A260 off the DEAE will increase because cAMP
concentration in the eluate becomes significant. This marks the end of the C
subunit elution peak. (The increase in A280, if that wavelength is used, to
measure the end of C subunit elution off DEAE, is not apparent if a low--10
&mu;M--cAMP concentration is used.)
<li>Disconnect the hydroxylapatite column from the DEAE column. Develop the
column with a linear gradient using 60 mL of 45 mM P<sub>i</sub> and 60 mL of
500 mM P<sub>i</sub>.  At about 200 mM P<sub>i</sub> into the gradient, the
kinase elutes, which you should verify with an activity assay of the fractions.
</ol>
<p> Store the kinase for months at 4&deg;.  If quick frozen, add sucrose to a
20% (w/w) concentration.  At -70&deg;, the enzyme can be stored indefinitely,
but repeated thawing and freezing will inactivate the enzyme slowly.  10-15 mg
of electrophoretically pure kinase with a specific activity of 3-4 U/mg
(~1000-fold purified with ~30% yield) is obtained.
<p> With slight modifications, this method has been applied to beef liver, pig
gastric mucosa, and rabbit kidney.  The beef heart contains the Type II kinase.
If only small amounts of the C subunit are needed, scaling down by a factor of
40 will be useful.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Blache D, Bouthiller D, Davignon J
 </span>
 <span class="year">
(1983)
 </span>
 <span class="jname">
Clin. Chem.
 </span>
 <span class="jvol">
29,
 </span>
 <span class="pages">
154-158.
 </span>
<p class="reftitl">
Simple, Reproducible Procedure for Selective Measurement of Lipoprotein Lipase
and Hepatic Lipase
<p class="Para1">
In the use of this radiolabeled assay to measure lipase activity, labeled
oleic acid must be extracted/separated from excess triolein and the products
of lipolysis (labeled di- and mono-olein).  Anion-exchange resins can be used
for this purpose, and after eluting/washing, these eluates are then counted
in scintillation counter.  Addition of resin to counting vial increases the
count of [<sup>14</sup>C]oleic acid by 20%, and the change is not linear with
increasing amount of resin.  Use of Triton X-100 (found in some methods) causes
quenching.  If oleate is extracted with an alkaline mixture and radioactivity
directly counted in scintillant, there are marked increases, up to 400% for
100 mmol, in counts obtained with constant amount of labeled oleic acid.
Use of a MeOH/chloroform/heptane extraction procedure recovers only 72.2% of
label and even benzene/chloroform/MeOH with 0.1 mM oleic acid present followed
by hexane gets only 86.6% recovery.
<p>
Substrate emulsion stability was then tested by pipetting eight 50 &micro;l
aliquots of freshly prepared substrate emulsion at 0, 1, 3, and 5 h and counting
in scintillant after sonication.  Emulsion was stable after all periods of time
tested.  Within-assay variation was 0.65%, and between-assay was 0.86%.  The
emulsion retained these properties for at least one day at 4&deg; if resonicated
before use.  In general, the freshly prepared emulsion has lowest background
counts.
<p>
Maximal activity conditions were determined.  Substrate concentrations of
&gt; 15 mM are not rate limiting.  Activity is linear for at least 60 min;
it is also linear to increasing amounts of postheparin plasma.  Lipase activity
was maximally stimulated with about 12% of added normal plasma.  The same
activation is seen with powdered acetone extracts of adipose tissue from
fed rats.  Addition of NaCl to higher molarities was inhibiting.  For six
replicates, within-assay variation is 3.2% for frozen milk LPL.  Day-to-day
precision using means of duplicate milk LPL standard reconstituted from
lyophilized aliquots was 12.3%, a CV for 24 duplicate determinations within
a month.
<p>
Postheparin plasma was incubated with different periods of time with antiserum
against purified human LPL.  Incubation was at 0&deg; to prevent denaturation
of enzyme.  About 70% of the initial activity is lost if preincubation time
exceeds 30 min, although it varies with the sample.  If NaCl is increased to
1.0 M, after prior incubation, LPL activity is completely inhibited and less
than 2% of initial activity remains.  Hence residual lipase activity after
inactivation with hepatic lipase antiserum could be ascribed to LPL alone.
<p>
Patients were given 10 IU/kg heparin i.v. and after 10 min plasma was collected.
The mean &plusmn; SD for LPL was 8.02 &plusmn; 1.69 U/ml for LPL and 5.09 &plusmn;
1.25 U/ml for hepatic lipase.  Mean plasma triglycerides was 790 &plusmn; 400
mg/l.
<p>
<b>Assay Procedure</b>.  Hepatic lipase must be inactivated with an antibody
to HL;  an activator is added to LPL as well.  Anti-HL is first checked for
the titer that offers inhibition without change.  LPL assays are done in
duplicate.  25 &micro;l post-heparin plasma in placed on ice for 1 h with
25 &micro;l of anti-HL antiserum.  During the incubation, 300 &micro;l of
0.2 M TrisHCl, 0.15 M NaCl pH 8.4, 50 mg/ml BSA and 12% pooled normal plasma
(as activator) is kept on ice.  20 &micro;l of the preincubation
plasma-antiserum mixture is then added to the tubes.  100 &micro;l of triolein
emulsion is then added, the tubes placed at 37&deg; for 30 min with gentle
shaking.  At the end, 3.0 ml benzene:chloroform:MeOH (10:5:2, v/v/v) is added
to stop the reaction, which contains 0.1 mM oleic acid.  200 &micro;l of 0.2 M
NaOH is then added, mixed 30 s, then tubes spun for 20 min at 2000 rpm.
1 ml of the 1.95 ml upper phase is then transferred to a tube with 1 ml
of 0.25 M HCl and 4.5 ml hexane.  The hexane phase is recovered by
by centrifugation and transferred to counting vials.  Extraction is repeated
twice, and hexane phases combined.  This phase is evaporated with air stream
at 50&deg; and dissolved in scintillant.  Use of this procedure avoids
quenching of the system (strong alkali, interfering substances).
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Gusella JF, Wexler NS, Conneally PM, Susan L. Naylor SL, Anderson MA, Tanzi RE,
Watkins PC, Ottina K, Wallace MR, Sakaguchi AY, Young AB, Shoulson I, Bonilla E,
Martin JB
 </span>
 <span class="year">
(1983)
 </span>
 <span class="jname">
Nature
 </span>
 <span class="jvol">
305,
 </span>
 <span class="pages">
234-238.
 </span>
<p class="reftitl">
Polymorphic DNA marker genetically linked to Huntington's disease
<p class="Para1">
Huntington's disease (chorea) is an autosomal dominant neurodegenerative disorder
showing signs in 3rd to 5th decade of life with 100% penetrance.  The chorea is
a motor abnormality with intellectual deterioration with psychiatric symptoms
including severe depression. Histology shows premature neuronal death in basal
ganglia.  Prevalence is 5-10 persons/100,000.  Use of polymorphic antigen and
enzyme markers do not show positivity but can exclude the locus from 20% of the
genome.  Use of DNA markers called restriction fragment length polymorphisms
(RFLPs) permits the ability to locate the disease locus.  Restriction fragments
of different sizes are electrophoretically separated and blotted to membrane to
be analyzed with radioactive probes using the transfer and hybridization of
Southern.  RFLPs are present in all regions of the genome, and this makes it
possible to construct human genetic linkage maps to locate the gene(s) involved.
An anonymous fragment from chromosome 4 detects two different RFLPs in a
<i>Hin</i>dIII digest of human genomic DNA.
<p>
Crucial to the study was identifying a family with a good pedigree showing the
disease.  Blood samples were taken from the members of the pedigree who had been
tested by a neurologist.  EBV-transformed lymphoblastoid cell lines were established
for the genotypes of each member of the American family.  A larger pedigree was
also identified in Venenzuela:  this 3000-member pedigree shows the defective
gene traced to a common ancestor in the early 1800s.  The functional stage of the
disease was classed according to Shoulson and Fahn.  All samples were analyzed
for 20 RBC and plasma markers to establish paternity as well.  Gene linkage
was also analyzed using the program LIPED with correction for the late age of
onset.
<p>
The DNA probes used have been reported previously.  Some contain known gene
loci while others are anonymous DNA segments chosen because they do not have
repetitive sequences.  In an initial screen for the HD gene, 12 markers were
tested and probe G8 results showed the possible of linkage to the HD gene.
G8 was taken from a human genomic library (made by Maniatis and coworkers).
It encompasses a 17.6 kb repetitive-free zone.  In a Southern blotting
using G8 DNA probe for the pedigree DNA, two fragments were invariant (present
in all members) and there were several variable <i>Hin</i>dIII fragments.
The invariants have size 8.4 and 2.3 kb.  The variable fragments had sizes
(decreasing order) of 17.5, 15, 4.9, 3.7, and 1.2 kb.  The <i>Eco</i>RI-digested
probe itself produces fragments of 6.0, 5.5, 2.4, 2.2, and 1.5 kb; each was
subcloned to pBR322 or pUC9 (in order to produce sufficient DNA).  Then products
were digested with <i>Hin</i>dIII, and the order of the <i>Eco</i>RI fragments
can be established using each subclone as a probe against single- or double
digests of G8 DNA using <i>Hin</i>dIII and <i>Eco</i>RI.  The variation
(presence or absence) of one <i>Hin</i>dIII site in the HD members was actually
outside the G8 probe region.  If the <i>Hin</i>dIII site is present in this
<b>Site 1</b>, there is a 15.0 kb fragment, when absent, a 17.5 kb fragment
occurs.  Another site, <b>Site 2</b> is within the G8 probe region.  Two
fragments are observed, 1.2 and 3.7 kb, when present, and a 4.9 kb frgament is
seen when absent.
<p>
Studies of 23 unrelated N. American individuals show that Site 1 presence has
an allelic frequency of 0.26.   Site 2 presence has a frequency of 0.81.  The
presence or absence of sites shows classic Mendelian inheritance.  Recombination
frequency for these restriction sites should be negligible given their
closeness (18.7 kb) to other.  A lod score of 4.53 was obtained with a recombination
fracton (&theta;) of 0.0; this supports their strong linkage as a single
haploptype.  If there is no linkage disequilibrium between two polymorphic
<i>Hin</i>dIII sites then the frequency of the four possible haplotypes
(combinations of Site 1 and 2 presence or absence) are 0.61, 0.14, 0.20,
and 0.05 for Haplotypes A (-/+), B (-/-), C(+/+) and D (+/-), respectively,
where the parentheses are (Site 1/Site 2) presence (+) or absence (-).  These
are the predicted frequencies:  the observed are 0.64, 0.11, 0.22, and 0.04
for haplotypes A through D (respectively) so there is not a high degree of
disequilibrium.  A larger sample would be necessary to precisely assess
disequilibrium.  G8 locus predicted level of heterozygosity if two sites in
equilibrium is 57% (making it a good marker).
<p>
Human-mouse somatic cell hybrids were employed in Southern blot analysis to
show which chromosome the gene is located upon.  DNA was isolated from several
hybrid cell lines (the lines featured all human chromosomes).  Some cell
lines were digested with <i>Hin</i>dIII, others with <i>Eco</i>RI.  A hybrid was
considered positive if a signal was seen in &gt;10% of cells.  Control tests
included looking for marker enzymes known to be on certain chromosomes to verify
experiments.  The common finding in all cases was that G8 was located in chromosome
4.
<p>
After analyzing the N. American family, the larger Venezuelan pedigree was
analyzed.  Linkage was analyzed using the LIPED program.  The predictability
of HD in those where no onset had occurred was determined assuming full
equilibrium of the affected sites.  The maxium lod score for HD against G8
was 8.53 at a &theta; = 0.0, indicating that the two restriction sites as loci
are very closely linked.  The HD gene segregated with the A haplotype in the
N. American family, but the C haplotype in the Venezuelan.  To resolve the
problem of which was so, a 99% confidence interval (one-sided) for the distance
between HD locus and G8 is computed as the &theta; at which likelihood is
100 times less than maximum value, or 2 log units less than 8.53.  The lod
score at 10 centimorgans is 6.52.  And the 99% interval is 0-10 centimorgabs,
Lod scores for HD and G8 against MNS (located between 4q28 and 4q31 and for
GC (located between 4q11 and 4q13) indicate these marker are not close to HD
or G8 locus.
<p>
That HD is caused by a single locus is probable.  If it is genetically
heterogeneous (caused by mutations at numerous loci) it would have a higher
mutation rate with loss of HD.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Young, A. P., Gunning, P. W., Landreth, G. E., Ignatius, M., and Shooter, E. M.
 </span>
 <span class="year">
(1983)
 </span>
 <span class="jname">
Birth Defects
 </span>
 <span class="jvol">
19,
 </span>
 <span class="pages">
33-46.
 </span>
<p class="reftitl">
The nerve growth factor induced differentiation of a pheochromocytoma PC12 cell
line
<!-- ============================================================= -->
<p class="tags">
HBsAg, E coli expression
<p class="ref">
 <span class="authors">
Fujisawa Y, Ito Y, Sasada R, Ono Y, Igarashi K, Ryuji Marumoto R, Kikuchi M, 
Sugino Y
 </span>
 <span class="year">
(1983)
 </span>
 <span class="jname">
Nuc. Acids Res.
 </span>
 <span class="jvol">
11,
 </span>
 <span class="pages">
3581-3591.
 </span>
<p class="reftitl">
Direct expression of hepatitis B surface antigen gene in <i>E. coli</i>
<p class="Para1">
Recombinant methods can be used for producing antigens for a potential vaccine.
Several groups have cloned HBsAg and HBcAg in bacteria and their genes 
identified.  HBsAg has also been cloned in yeast.  Attempts to clone HBsAg
in E coli have succeeded as fusions:  &beta;-galactosidase and &beta;-lactamase.
<p>
The HBsAg/adw gene was prepared from pHBV933 by cleaving 400 &micro;g pHBV933 
with <i>Bam</i>HI and separating on 1% agarose gel (110 V/2 h) in TBE (0.1 M 
Tris pH 8.3 + 0.1 M boric acid + 2 mM EDTA).  The 1.4 kb fragment was recovered 
from gel using McDonell et al (1977).  200 &micro;g pHBV933 is now digested with 
<i>Bam</i>HI and <i>Hpa</i>I to give a 931 bp fragment, which encompasses the 
gene, then is partially digested with <i>Sau</i>3A to produce a 809 bp fragment 
still containing complete gene;  since there are three <i>Sau</i>3A sites inside
the gene, complete digestion would cut into the gene.  20 &micro;g of 931 bp
fragment were digested with <i>Hin</i>cII and <i>Xba</i>I for a 744 and 712 bp
fragments, respectively.  To construct expression plasmid, 2 &micro;g 809 bp
DNA filled with DNA poymerase I, and 1 &micro;g blunt end and 50 ng BamHI
linker d(CCGGATCCGG) is mixed with T4 ligase and <i>Hpa</i>I added to produce
cohesive ends.  This now purified on Sepharose 4B (3 &times; 22 mm) in PBS
pH 7.4.  200 ng pure fragment is ligated to 500 ng Cla<i>I</i>-digested and
alkaline phosphatase-treated pTR801, and <i>E coli</i> C600 is transformed.
After selection of 12.5 &micro;g/ml tetracycline in L agar, the expression
plasmid pTRP SS-6 is obtained, with HBsAg controlled by <i>trp</i> promoter.
Now 1 &micro;g of the 744 bp fragment and 50 ng of double-stranded phosphorylated adapter 
d(5&prime;-CGATGCGAT-3&prime;||3&prime;--TACGCTA-5&prime;), where the hyphens
indicate an overhang on the opposite strand are ligated with T4 DNA ligase.
This effectively add 7 bp to both ends to make a 758 bp fragment.  This is cleaned
on Sepharose 4B and then inserted into the <i>Cla</i>I site of pTRP801 in
an orientation to give pTRP SS-39.  2 &micro;g of 712 bp fragment are filled
in with DNA polymerase I and then ligated to the phosphorylated <i>Cla</i>I
linker, d(CATCGATB).  After treating with <i>Cla</i>I and cleaning on Seph 4B,
and in order to adjust space between Shine-Dalgarno sequence and start codon,
500 ng of fragment was filled with DNA polymerase I, ligated to <i>Cla</i>I
linker, cleaned on Seph4B, and inserted into <i>Cla</i>I site of pTRP801 to
give expression plasmid pTRP SS-50.
<p>
Plasmid with complete HBsAg is transfected into <i>E coli</i> K-12 derivatives.
Two kinds of transformants are seen.  One is normal, the other whose growth
is inhibited.  Restriction analysis showed that the growing strain had a plasmid
pTRP SS-3 where HBsAg gene is inserted in opposite, and the inhbited one had
the plasmid pTRP SS-6 in the correct orientation.  Growth of pTRP SS-39 or
pTRP SS-50 where the sequence has the amino terminal hydrophobic domain deleted
was not inhbited in growth.  It is possible that protein expression in <i>E coli</i>
host attaches the N-terminal to the bacterial membrane and inhbits growth, and
if absent, no inhbition.
<p>
Colonies contain the pTRP SS-6 full length growth-inhibited were grown on 
BHI agar, and then transferred to IAA-containing modified M-9 plate to induce
expression.  These colonies were positive on an immunoassay. If the plasmid
was in E coli strain C600, there was less inhibtion of growth.  Grown on M-9
agar, colonies with pTRP SS-39 or SS-50 were highly positive in a direct 
immunoassay.  No colonies with just pTRP801 or pTRP SS-3 were positive in the
assay.
<p>
[<sup>35</sup>S]L-Met was added to minicells of <i>E coli</i> P678-54 containing
plasmids pTRP SS-6, pTRP SS-39, or pTRP SS-50.  The minicells with pTRP SS-6 
show an additional protein at 24 kDa not seen in control cells (with pTRP771).
This is the size of the unglycosylated polypeptide in HBsAg particles.  Minicells
with pTRP SS-39 and SS-50 form polypeptides of 22-23 kDa, which is predicted
for the peptides representing positions 23-226 and 32-226.  These reacted with
a mAb on a western.
<p>
The growth kinetics was then studied in terms of bacterial numbers and also
protein expression (plots of Klett units and HBsAg ng/ml verus culture time).
pTRP SS-6 in C600 host shows detectable synthesis after 3 h and reaching maximal
at 7-8 h.  Bacterial with control plasmid clearly grew more rapidly. HBsAg
levels were 20 ng/ml, roughly 1000-1500 molecules per cell (this assumes that
affinity to HBsAg and of the particle form are same for antibody.  Recombiants
with the deleted N-termini pTRP SS-39 and SS-50 grew normally and expression
was 30-45 ng/ml, and the protein was about 0.001% of total protein.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Gidoni, D., Dynan, W. S., Tjian, R.
 </span>
 <span class="year">
(1984)
 </span>
 <span class="jname">
Nature
 </span>
 <span class="jvol">
312,
 </span>
 <span class="pages">
409-413.
 </span>
<p class="reftitl">
Multiple specific contacts between a mammalian transcription factor
and its cognate promoters
<!-- ============================================================= -->
<p class="ref">
 <span class="authors">
Hindley, J., Phear, G. A. 
 </span>
 <span class="year">
(1984)
 </span>
 <span class="jname">
Gene
 </span>
 <span class="jvol">
31,
 </span>
 <span class="pages">
129-134.
 </span>

<p class="reftitl">
Sequence of the cell division gene <i>Schizosaccharomyces pombe</i>; patterns
of splicing and homology to protein kinases
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Parslow, T. G., Blair, D. L., Murphy, W. J., Granner, D. K.
 </span>
 <span class="year">
(1984)
 </span>
 <span class="jname">
Proc. Natl. Acad. Sci. USA
 </span>
 <span class="jvol">
81,
 </span>
 <span class="pages">
2650-2654.
 </span>
<p class="reftitl">
Structure of the 5' ends of immunoglobulin genes:  a novel conserved sequence
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Stanley KK, Luzio JP
 </span>
 <span class="year">
(1984)
 </span>
 <span class="jname">
EMBO J.
 </span>
 <span class="jvol">
3,
 </span>
 <span class="pages">
1429-1434.
 </span>
<p class="reftitl">
Construction of a new family of high efficiency bacterial expression
vectors:  identification of cDNA clones coding for human liver proteins
<p class="Para1">
Screening cDNA clones using hybrid-selected translation difficult if mRNA
is expressed at low level (being rare) or mRNA is isolated from tissue culture
or if in vitro translation efficiency is low (for large mRNA usually).
Grunstein and Hogness (1975) provide alternative to use oligonucleotide 
obtained from short amino acid sequence and using colony hybridization
(requires N-terminal sequencing and then degenerate probes to cover the possibilities).
Another solution is to clone the cDNA into bacterial expression vector
and screen for antigenic determinants in situ, in which antibody is required.
The solution offered here is to use a <i>cro-&beta;-galactosidase</i> hybrid
which precipitates within the host cell (partially protecting it from proteolysis).
The "colony blot" using immunochemistry for insoluble hybrid has been decribed
before.
<p>
The <b>pEX</b> vectors 1 through 3 are constructed as follows.  Sugimoto et al
(1977) produced the pLK91 plasmid with tandem 103 bp terminator from phage fd
and an oligonucleotide called STOP  with stop codons in all three reading frames,
placed between HindIII and PvuII sites in pBR322.  The XmnI-MstI fragment in
pBR322 was replaced by its similar fragment from pUC8 to remove a PstI site in
&beta;-lactamase gene., and the polylinker in pUC8 was inserted into EcoRI site.
HindIII in pLK91 and the EcoRI and BamHI sites were removed to maximum unique
sites in polylinker.  Two modified halves were recombined to produce pLK915.
pCL19 is <i>cro-lacZ</i> fusion (from Zabeau and Stanley 1982) with both in
different reading frames.  The BamHI site is filled in at the junction of 
<i>cro</i> and <i>lacZ</i> to generate pCL6191 which makes a 117 kDa hybrid
of cro and &beta;-gal (can make up to 30% of cell protein).  An EcoRI contains
the P<sub>R</sub> promoter:  all but last 53 bp of cro-lacZ fusion is was put
into EcoRI site of pLK915 upstream of polylinker, stop oligonucleotide and
transcription terminators..  The EcoR1 site is removed at position 0 to make
pEX2. pEX1 and pEX3 are &minus;1 and +1 frameshifts at EcoRI site of polylinker.
<p>
Constructs are transformed into hosts with <i>c</i>I<i>ts</i>857 repressor.
Plasmids are amplified at 30&deg; and then transient expression done at 42&deg;
After 2 h at 42&deg; pEX express the cro-&beta;-galactosidase hybrid (~25% of
SDS-extractable protein).  Lysozyme-TritonX-100 extractions produce less than 
4% of this protein (thus it is insoluble in this system).  The hybrids are
easily purified and antibodies can be raised against expressed regions of DNA;
moreover insolubility protects them from degradation (seen Cheng et al 1981).
DNA up to 400 bp long can be inserted into cloning site without reducing
hybrid expression level.  Even if inserts are long and some degradation occurs,
it does not affect ability to check expression of the hybrid protein with
antibodies.  Translational reading frame can be randomized either by adding
homopolymer tails to fragments or using mixture of all three vectors.
<p>
As a tests, PstI fragments between 400-1600 bp long from human liver cDNA
were inserted into pEX1.  This library was screened by hybridization with
nick-translated full length cDNA probe for human albumin and screening by 
colony blot.  The number of colonies that could be stained with the colony
blot was &gt; 35% of number of colonies detected by colony hybridization.
Less than 10% of the antibody-reacting clones were false positives.  Screening
this cDNA library in pEX1 with antisera against &alpha;<sub>1</sub>-antitrypsin,
&alpha;<sub>1</sub> acid-glycoprotein, and &gamma;-fibrinogen gives positives
clones which were found to have cDNA identical to known DNA sequences of the
genes.  There is not a correlation between strong positives of the colony
blot with strong positives of colony hybridization, because hybridization
siganl increases with cDNA insert size, while small pieces of cDNA are 
expressed more efficiently and degrade less.
<p>
[Additional text on cloning epitopes of complement C9]
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Zick, Y., Sasaki, N., Rees-Jones, R. W., Grunberger, G., Nissley, S. P., Rechler, M. M.
 </span>
 <span class="year">
(1984)
 </span>
 <span class="jname">
Biochem. Biophys. Res. Comm.
 </span>
 <span class="jvol">
119,
 </span>
 <span class="pages">
6-13.
 </span>
<p class="reftitl">
Insulin-like growth factor-I (IGF-I) stimulates tyrosine
kinase activity in purified receptors from a rat liver cell line
<!-- ============================================================= -->
<p class="tags">
Phage T7, RNA polymerase
<p class="ref">
 <span class="authors">
Davanloo P, Rosenberg AH, Dunn JJ, Studier FW
 </span>
 <span class="year">
(1984)
 </span>
 <span class="jname">
Proc. Natl. Acad. Sci. USA
 </span>
 <span class="jvol">
81,
 </span>
 <span class="pages">
2035-2039.
 </span>
<p class="reftitl">
Cloning and Expression of the Gene for Bacteriophage T7 RNA polymerase
<p class="Para1">
The RNA polymerase (EC 2.7.7.6) is produced early in T7 infection.  It is a 
single chain enzyme with ~100 kDa mass and is very specific for its own 
promoters. The promoter is 23 consecutive conserved base pair including 
transcription start site.  There are 17 promoters in T7 genome, but not in the 
target host. As T7 RNA polymerase makes its own gene products, one of them 
inactivates host RNA polymerase and only phage transcripts are made.  The RNA 
polymerase is very efficient and can produce large amounts of RNA.  The RNA 
targeted for production can be used for hybridization probes, mRNAs for in vitro 
translation, studies of RNA processing and splicing, and any other purpose 
requiring RNA. The promoter itself might be targeted to a genome and then direct 
transcription of selected genes in the cell.  The amount of phage RNA polymerase 
is not considerable: it is synthesizeed in life cycle only a few minutes.  So 
the cloning of the phase enzyme is necessary to direct the making of large 
quantities of it.  Stahl and Zinn obtained a clone whose sequence had all but
the last nucleotide of the stop codon, but even so the clone had no activity.
With the whole of the T7 phase genome sequenced by this laboratory, it was
a simple effort to produce an active clone in an expression vector.
<p>
Gene <i>1</i> of the T7 phage genome is its RNA polymerase.  It was natural
to use as a cloning vector the <i>Bam</i>HI site of pBR322, used to clone other
T7 genes.  For genes whose high level expression might be lethal, cloning
in the <q>silent</q> orientation.  The inserted DNA would have to contain the
promoter.  
<p>
The problem in cloning is that we want to control the transcription of the T7 
RNA polymerase.  We need to clone the gene in a way that does not include its 
own promoter, which is the C promoter (a minor promoter) for the <i>E coli</i> 
RNA polymerase, its natural promoter.   On the immediate 3&prime; end of the 
gene <i>1</i> (T7 RNA polymerase) coding sequence is the sequence that binds the 
gene <i>1</i> product and begins transcription of a T7 gene under the control of 
its own RNA polymerase.  This promoter, if cloned into pBR322 vector, could 
possibly direct the complete transcription of the insert with its vector, 
creating much more of the polymerase which then continues to produce itself 
exponentially, causing lethality.  
<p>
No restriction sites could be found which isolate the two promoters from the 
gene <i>1</i> coding sequence.  It was then necessary to use heteroduplexes 
formed from mixing the DNA of double-deletion mutants, and then using the S1 
nuclease to eliminate the nonhybridized single-stranded portions. 
Double-deletion mutants D159 and C74  and LG4 and LG256 would ideally produce 
fragments of gene <i>1</i> from nt 3128 to 5845.  The T7 RNA polymerase AUG 
codon is at nt 3171, and UAA stop codon is at 5822.  The fragment also starts 11 
bp upstream of the 5&prime; end of the mRNA formed, so contains translation 
initiation signals, but should not contain all of the promoter for the <i>E 
coli</i> RNA polymerase which naturally transcribes gene <i>1</i> during 
infection:  the promoter starts at nt 3113. Moreover, the promoter on the 
3&prime; end is between nt 5831 and 5853, so the only the first 15 of the 23
conserved base pairs are present.
<p>
In addition to using the S1 nuclease, the BAL-31 exonuclease was also used to
digest a few nucleotides from the ends of the fragments in a very short incubation,
in case the 15 nucleotides was also enough to generate RNA polymerase initiation.
Fragments were purified by gel electrophoresis, both the S1-only digest as well
as S1 + BAL-31 treatments.  <i>Bam</i>HI linkers were then ligated


<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Lowe, P. A., Marsten, F. A. O., Angai, S., Schoemaker, J. A.
of Celltech, Ltd.
 </span>
 <span class="pages">
European Patent Office application 84301995.1 (filed 23.03.84)
 </span>
<p class="reftitl">
A process for the production of a protein
<p class="Para1">
This patent application is concerned with a process of
recovering a protein in an insoluble form expressed as a recombinant in
large amounts within a heterologous host.  Beyond the background
introduction is the idea that the insoluble protein of interest is exposed
to an alkaline aqueous solution at a pH which would undoubtedly denature
the protein but also make it soluble.  The pH is then reduced to a
point which permits renaturation.  This approach apparently works
with chymosin produced as a recombinant in <i>E. coli.</i>  Native
prochymosin in monomeric and contains 3 intramolecular disulfide
bonds;  the insoluble aggregate (probably?) results from
intermolecular disulfide bond formation during synthesis.  An
alkaline solution at pH 10.7 (&plusmn; 0.5) can break these bonds
(Cys is known to have a pKa value of 10.46) and the pH subsequently
reduced to permit
proper renaturation.  Any alkali can be used in the process (NaOH,
KOH, aqueous buffered solution, or an aqueous solution with organic base
such as triethylamine).
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Derynck, R., Roberts, A. B., Winkler, M. E., Chen, E. Y., Goeddel, D. V.
 </span>
 <span class="year">
(1984)
 </span>
 <span class="jname">
Cell
 </span>
 <span class="jvol">
38,
 </span>
 <span class="pages">
287-297.
 </span>
<p class="reftitl">
Human transforming growth factor-&alpha;:  precursor structure
and expression in <i>E. coli</i>
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Derynck, R., Roberts, A. B., Winkler, M. E., Chen, E. Y., Goeddel, D. V.
 </span>
 <span class="year">
(1984)
 </span>
 <span class="jname">
Cell
 </span>
 <span class="jvol">
38,
 </span>
 <span class="pages">
287-297.
 </span>
<p class="reftitl">
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Herbomel, P., Bourachot, B., Yaniv, M.
 </span>
 <span class="year">
(1984)
 </span>
 <span class="jname">
Cell
 </span>
 <span class="jvol">
39,
 </span>
 <span class="pages">
653-662.
 </span>
<p class="reftitl">
Two distinct enhancers with different cell specificities coexist
in the regulatory region of polyoma
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Wynshaw-Boris, A., Lugo, T. G., Short, J. M., Fournier, R. E. K., Hanson, R. W.
 </span>
 <span class="year">
(1984)
 </span>
 <span class="jname">
J. Biol. Chem.
 </span>
 <span class="jvol">
259,
 </span>
 <span class="pages">
12161-12169.
 </span>
<p class="reftitl">
Identification of a cAMP regulatory region in the gene
for rat cytosolic phospho<i>enol</i>pyruvate carboxykinase (GTP)
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Anderrson, T., Hagel, L.
 </span>
 <span class="year">
(1984)
 </span>
 <span class="jname">
Anal. Biochem.
 </span>
 <span class="jvol">
141,
 </span>
 <span class="pages">
461-465.
 </span>
<p class="reftitl">
Some properties and applications of Superose 6B
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Brake, A. J., Merryweather, J. P., Coit, D. G.,
Heberlein, U. A., Masiarz, F. R., Mullenbach, G. T., Urdea, M. S.,
Valenzuela, P., Barr, P. J.
 </span>
 <span class="year">
(1984)
 </span>
 <span class="jname">
Proc. Natl. Acad. Sci. USA
 </span>
 <span class="jvol">
81,
 </span>
 <span class="pages">
4642-4646.
 </span>
<p class="reftitl">
&alpha;-Factor-directed synthesis and secretion of mature
foreign proteins in <i>Saccharomyces cerevisiae</i>
<p class="Para1">
Vectors containing the a-factor leader were constructed
with fusions to genes coding for heterologous proteins.  The a-factor
contains presumptive processing signals Lys-Arg-(Glu-Ala)<sub>3</sub>
where the leader is removed.  pAB112 is a plasmid
containing an <i>Eco</i>RI insert which encompasses the entire 495 bp-coding
sequence of the alpha mating factor gene plus 5' and 3' flanking region
(total 1.72 kbp).  Digestion of pAB112 with <i>Hin</i>dIII and
<i>Sal</i>I removes a fragment of the <i>Eco</i>RI cassette which codes for the
mature alpha mating factor.
<p>
[Note entry for this article was not completed.]
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Woollett LA, Beitz DC, Hood RL, Aprahamian S
 </span>
 <span class="year">
(1984)
 </span>
 <span class="jname">
Anal. Biochem.
 </span>
 <span class="jvol">
143,
 </span>
 <span class="pages">
25-29.
 </span>
<p class="reftitl">
An Enzymatic Assay for Activity of Lipoprotein Lipase
<p class="Para1">
Shimizu et al (<i>Anal. Biochem.</i> <b>98:</b> 341, 1979) have previously
described an enzymatic assay using spectrophotometry for determining plasma
free fatty acid (FFA) levels.  A fluorometric method has been described by
Miles et al (<i>J. Lipid Res.</i> <b>24:</b> 96, 1983).  This study attempts
to find out if an adaptation can be used to assess lipoprotein lipase
activity.
<p>
The assay is done in two phases.  The first phase is the
production of free fatty acid (FFA) product by the LPL enzyme using an added
triolein substrate emulsion.  The reaction is then stopped and heated to
inactivate enzymes that alter or affect substrates in the 2nd phase.  In the
2nd phase, an
aliquot (25 &micro;l) of this reaction mixture is then added to a mixture
of enzymes and substrates in a series of coupled reactions in which FFA
reacts with CoA and ATP to produce acyl-CoA + AMP + PPi via acyl CoA synthetase.
The AMP product then reacts with ATP to produce 2 ADP via myokinase control.
The 3rd coupled reaction involves ADP reacting with PEP through pyruvate kinase
to form ATP + pyruvate.  And the 4th and final coupled reaction takes 1 mol
pyruvate product from the 3rd reaction which is reduced by NADH under LDH
to form lactate and NAD<sup>+</sup>.  The NAD<sup>+</sup> formation can be
monitored spectrophotometrically.  If the equations are balanced, for every
molecule of FFA entering the coupled reactions, 2 molecules of NAD<sup>+</sup>
are formed.  This should be taken into account.  Also a standard curve of
oleic acid can be used in the 2nd phase to monitor the reaction.  When doing
the standard curve, add oleic acid to the LPL sample, and also run a control
in the 2nd phase without the presence of LPL sample (may need to
use an emulsion).
<p>
Adult male Sprague-Dawley rats were sacrificed by decapitation after head blow.
Epididymal fat pads were homogenized with Polytron in 0.25 M sucrose, 1 mM EDTA
adjusted to pH 7.4 at 37&deg;, using 4-5 ml buffer per gram tissue (a 20% homogenate
w/v).  After
spinning 15 min at 12,000 <i>g</i>, the lipid layer is removed and infranate
recovered.
<p>
A triolein emulsion as described by Schotz et al (<i>J. Lipid Res.</i> <b>11</b>:
110, 1968) and modified by Hietanen &amp; Greenwood (<i>J. Lipid Res.</i>
<b>18:</b> 480, 1977) is used. [<sup>14</sup>C]Triolein is added at 1.8
&micro;Ci for 36 mg triolein total.  The emulsion is prepared by sonication
with phosphatidylcholine, BSA, and serum in a pH 8 Tris buffer.  Heat-inactivated
calf serum in the LPL activator.  0.25 ml emulsion is mixed with 0.25 ml of
tissue homogenate and run for 60 min.  The assay is stopped by three different
methods.
<p>
<b>Method.</b>
The number of assays (tubes) should be determined.  This assay requires for
each sample/unknown a 0 and 60 min time point with and without the presence
of 1 M NaCl.  Without 1 M NaCl, total lipolysis is determined; with 1 M NaCl,
lipolysis not due to LPL is determined.  Thus for a single sample to be done
in duplicate, there are 2 replicates &times; 2 time points &times; 2 NaCl
concentrations for a total of 8 tubes; if the assay is to be done in triplicate
then it is 12 tubes per unknown.  As an example, computations
of amounts needed are done for 20 unknowns, or 160 tubes.
<ol>
<li><b>1st Phase Substrate Emulsion</b>.  This preparation is from the Hitanen
and Greenwood paper.  Essentially 700 nmol triolein per assay is used.  The
final substrate emulsion should 6 mg/ml triolein, 0.4 mg/ml
L-&alpha;-phosphatidylcholine (lysolecithin), 85 mM TrisHCl pH 8.0,
0.075% BSA, and 50% serum obtained from a 12 h fast.  250 &micro;l emulsion
will be used.  If emulsion for 50 assays is to be prepared, then total
volume of emulsion including 15% pipetting error is 50 &times; 1.15 &times;
0.25 ml = 14.375 ml.  Round this to 14.5 ml.  To prepare 14.5 ml emulsion:
 <ol>
 <li>Into a glass container that is double the volume necessary (in this
 	example, a 30 ml vial or possibly beaker, for instance) and into
 which you can fit a sonicator probe, place 870 &micro;l of 100 mg/ml
 triolein in chloroform.
 <li>Add to this tube 58 &micro;l of 100 mg/ml phosphatidylcholine.
 <li>Dry this down under a stream of nitrogen.
 <li>Add 1.23 ml of 1.0 M Tris pH 8.0
 <li>Add 1.09 ml of 1% fatty acid-free BSA
 <li>Add 7.25 ml of the fasting serum which has been treated first at 50&deg;
 for 30 min.
 <li>Add 4.93 ml of high quality water
 <li>Sonicate this mixture on ice-water bath:  use 60 W input power for
 	30 second on-off bursts for a total of 4 minutes (8 cycles).
 </ol>
This emulsion should used fresh for the day, and any not used discarded.
<li><b>1st Phase Incubation</b>.  Mix 250 &micro;l of emulsion with
250 &micro;l of the LPL source (plasma, tissue extract, etc.)  Immediately
remove duplicate tubes as a 0 (zero) time point by heating at 50&deg; for
2 h, and then starting the 2nd phase.  60 minutes later
<li><b>2nd Phase Coupled Enzymes/Substrates Mix</b>.  Prepare this while the
1st phase is incubating.
Multiply the tube count by 0.46 ml:  this is for a 0.4 ml volume of the
coupled enzymes/substrates mix and a 15% excess for pipetting error.  Prepare
the computed volume of enzyme mix as follows:  78 mM TrisHCl pH 8.0, 7.5 mM
magnesium chloride (MgCl<sub>2</sub>), 0.1% (v/v) Triton X-100, 0.5 mM EDTA,
4 mM ATP, 9.25 mM phosphoenolpyruvate (PEP), 0.35 mM NADH, 27 U/ml myokinase,
15 U/ml pyruvate kinase, and 15 U/ml lactate dehydrogenase.  Suppose you have
50 assays to run.  The computed volume is 50 assays &times; 0.46 ml/assay =
23 ml.  You can prepare this mix from the following stock reagents:
 <ul>
 <li>1.79 ml of 1.0 M TrisHCl pH 8.0
 <li>1.725 ml of 100 mM MgCl<sub>2</sub>
 <li>230 &micro;l of 10% (v/v) Triton X-100
 <li>115 &micro;l of 100 mM EDTA disodium, pH 8.0
 <li>920 &micro;l of 100 mM ATP
 <li>2.125 ml of 100 mM PEP
 <li>805 &micro;l of 10 mM &beta;-NADH
 <li>621 &micro;l of 1000 U/ml myokinase
 <li>345 &micro;l of 1000 U/ml pyruvate kinase
 <li>345 &micro;l of 1000 U/ml lactate dehydrogenase
 <li>14.0 ml of highest grade water
 </ul>
<li>Prepare 400 mU/ml acyl coenzyme A synthetase.  You will need a volume which
is 5.75 &micro;l &times; number of assay tubes.
<li>Prepare 21 mmol/L of coenzyme A.  You will need 11.5 &micro;l &times; the
number of assay tubes.
<li>Heat the LPL homogenate/source  at 50&deg; for 2 h to inactivate enzymes
present in the LPL tissue source that would bind coenzymes or consume substrates.
<li>Mix 400 &micro;l of the enzymes/substrates
mix with 2 mU of acyl coenzymeA synthetase (5 ul of a 400 mU/ml acyl-CoA synthase
stock) + 25 &micro;l of the heat-activated LPL source above.
<li>Add 10 &micro;l of 21 mM coenzyme A to each solution.  For blank tubes, omit
the coenzyme A addition and use its vehicle.
<li><b>STANDARD CURVE</b>.  For the standard curve, use oleic acid.  Results show
linearity from 0-25 nmol.
<li><b>NON LPL Lipase Activity</b>.  LPL is inhibited by 1 M NaCl.  Include tubes
which have at least 1 M NaCl to assess the presence of lipase activity not related
to LPL.
<li>Leave for 2 h at 23&deg; (room temperature).
<li>After incubation time, measure the 340 nm.  You can measure against water
blank, or use the tube without coenzyme A as blank.
</ol>
<p>The radiolabeled methods were done for comparison too.  These are 60 min assays.
We also modified the radiolabeled triolein method as follows:
<ol>
<li>200 &micro;l aliquots from the reaction were added to 15 ml
chloroform:MeOH:0.1 N HCl (10:20:8, v/v/v), which is vortexed.
<li>Chloroform and 0.1 N HCl are added so that the final composition is now
20:20:18, v/v/v.
<li>The lower layer is recovered and evaporated to dryness.
<li>The residue is dissolved in diethyl ether:hexane (1:3).
<li>Pass this through a 0.5 &times; 7.5 cm column with 0.5 g of 7% hydrated
Florisil (60-100 mesh) which has been saturated in hexane.
<li>10 ml of the ether-hexane is added to elute triglycerides from the column.
<li>Remove the Florisil from the column, and allow it to dry.
<li>Transfer to a scintillation vial.  Add 10 ml of scintillation cocktail
(Beckman Ready-Solv EP) and vortex vigorously.  Count.
</ol>
<p><b>RESULTS</b>.
With the 20% homogenate, a specific activity of 1.37 &micro;mol / (h &times; g).
Roughly the same value was found with 5, 10, and 30% homogenates as well.  This
value was determined after finding a lipase activity not related to LPL of
0.34 &micro;mol FFA.

<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Kaczmarek, L., Hyland, J. K., Watt, R., Rosenberg, M., Baserga, R.
 </span>
 <span class="year">
(1985)
 </span>
 <span class="jname">
Science
 </span>
 <span class="jvol">
228,
 </span>
 <span class="pages">
1313-1315.
 </span>
<p class="reftitl">
Microinjected c-<i>myc</i> as a competence factor
<p class="Para1">
Previous reports indicate that microinjection of cloned
Harvey <i>ras</i> gene or p21<sup>v-ras</sup>
protein induces cellular DNA synthesis in quiescent mammalian cells while
injection of the v-<i>myc</i> gene or the c-<i>myc</i> protein has no such
effect.  However, because (i) c-<i>myc</i> expression is induced by
growth factors and mitogens; (ii) c-<i>myc</i> expression regulation is
altered in chemically transformed cells; (iii) DNA synthesis occurs when a
c-<i>myc</i> gene under the control of the MMTV promoter is introduced
into 3T3 cells in the presence of glucocorticoids; and (iv) c-<i>myc</i>
expression is altered in certain forms of neoplasia, c-myc is therefore
thought to play a role in control of cell proliferation.  The current
model of cell cycle induction is the &ldquo;competence-progression&rdquo; model,
wherein cells are made competent by a factor such as PDGF, and then
progress into S phase by a factor such as &ldquo;platelet-poor plasma&rdquo;
(PPP).  Neither PPP nor PDGF alone is able to stimulate induction of
the cell cycle, but in the appropriate sequence, they can.
<p>
Purified recombinant c-myc protein, which in a
previous report is shown to migrate to the nucleus upon microinjection and
upon tracking with antibodies to specific c-myc peptides, is injected
directly into the nucleus using a manual technique.  A control
protein (albumin) was also injected;  Ca- and Mg-free PBS is used as
the carrier since these ions are potential competence factors.  Swiss
3T3 cells were cultured in 60 mm petri dishes to 50,000/dish in 10%
FCS.  Serum was reduced to 1% and continued for 3-4 days, at which
stage the cells are quiescent.
<p>
Selected cells were injected with either protein,
while other cells were not injected.  Injected cells were treated
thereafter with PPP or not.  Incorporation of <sup>
3</sup>H-thymidine was used as the measure of DNA
synthesis.  In cells injected with c-myc protein without following
with PPP, <sup>3</sup>H-thymidine incorporation
(3.9%) was not significantly greater than incorporation in uninjected
controls (1.4%).  When treated with PPP after microinjection with
c-myc, <sup>3</sup>H-thymidine incorporation was
very significantly increased 19.7% (with a range of 10.5 to 30%).
That microinjection itself was the cause was ruled out because BSA
injection produced incorporation at the level of controls.  Other
proteins, except for c-<i>ras</i> and the SV40 large T antigen which
stimulated DNA synthesis even in the absence of PPP, had no effect on
incorporation as well.  These results alone support the view that
c-myc is a competence factor in a class of others identified:  FGF,
PDGF, Ca<sub>3</sub>(PO<sub>4</sub>)<sub>2</sub>, human serum,
SV40 and local wounding of cultured quiescent fibroblast monolayers.
<p>
Microinjection of PDGF was also found to stimulate
DNA synthesis, but only if it was followed with PPP incubation, and only
when anti-PDGF antibodies were not added to the culture medium.
Stimulation by microinjected c-myc was not affected by addition of
anti-c-myc to the culture medium.  It was assumed that the action of
PDGF resulted from its leaking out to stimulate its receptor.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Livett, B. G.
 </span>
 <span class="year">
(1984)
 </span>
 <span class="jname">
Physiol. Rev. 
 </span>
 <span class="jvol">
64,
 </span>
 <span class="pages">
1103-1161.
 </span>
<p class="reftitl">
Adrenal medullary chromaffin cells in vitro
<p class="Para1">
<b>REVIEW</b>.  Excellent review on the isolation and culture of chromaffin 
cells as well as general biochemical characteristics of the cells.
<p>
<b>Isolation methods</b>.  Most popular isolation method is use of enzymatic 
digestion with collagenase, alone or in combination with hyaluronidase, 
elastase, trypsin, and other proteases.  Use of DNase type I is sometimes 
included by some who have found that filamentous material resulting from 
liberation of DNA from damaged cells causes aggregation and clumping of cells 
resulting in cell loss.  Passage of the material through 50- to 70-&micro;m 
nylon mesh is usually necessary to avoid cell loss by this fibrous material in 
later steps.  Best yields are obtained with use of collagenase digestion using 
retrograde perfusion of intact medulla in a Ca<sup>2+</sup>-free medium buffered 
at pH 7.0-7.2 followed by a 2nd incubation with fresh collagenase.  Maintenance 
of physiological pH during digestion/isolation is stressed, so use of 5 mM HEPES 
at pH 7.4 in Locke's PBS shows isolation of cells with predictably consistent 
results.
<p>
For bovine adrenal glands two methods for cell isolation have emerged.  The 
first method involves blunt dissection of the medulla from the cortex followed 
by its cannulation via the central lobular vein;  a perfusion apparatus for up 
to 8 glands at a time yields about 50 million cells/gland.  The advantage is 
that the extent of medulla digestion can be monitored continuously to allow for 
differences in the effectiveness of the collagenase digestion mix; disadvantages 
include manipulation (sterilization) of much equipment before and between use as 
well as it being time-consuming.  A second method (largely the idea of Greenberg 
and Zinder, <i>Cell &amp; Tiss. Res.</i> <b>226</b>: 655, 1982) is multiple 
infusions of a digestion mix into the whole gland via the vein:  after a brief 
period, the gland is sliced in half and the medulla virtually scooped out as a 
semi-liquid.  This method presumably yields about 100 million cells per gland 
and is shorter in time.
<p>
There appear to be some difference in yield of viable chromaffin cells based on 
breed, age, and nutritional state of the animals.  The Holstein cattle in the 
U.S. and Canada have larger glands (and medullae) than Pol Hereford and Friesian 
cattle common in the U.K. and Australia.  The Hereford crosses have less 
connective tissue and fat however and generate higher cell yields than North 
American breeds.  Other important considerations: (1) tissue obtained should be 
as fresh (warm) as possible (within 5-10 min of death of the animal is good); 
(2) age is not too important although younger animals have less connective 
tissue and the glands are therefore easier to digest; (3) in colder climates in 
North America where feed composition changes during the seasons, yields can be 
affected (generally yields were lowest toward the end of winter).  Mycoplasma 
and gram-negative rods must be handled from time to time:  development of 
bacterial contamination probably results from resistant strains present in 
animals that are given large amounts of antibiotics in the feed;  mycoplasma can 
be tested for and killed (see Marcus et al., <i>Nature</i> <b>285</b>: 659, 1980).
<p>
Chromaffin cells are sensitive to continual and forceful centrifugations. 
Resuspension should be as gentle as possible.
<p>
<b>Cell purification</b>.  Cells can be purified by isopycnic centrifugation or 
differential plating.  As for isopycnic centrifugation, cell separation with 
Percoll, metrizamide (Renografin), and BSA have been used without effect on the 
functional properties of the cells (comparisons of EC50 to agonists are the same 
before and after).  Considerations include availability of a high-speed 
centrifuge (for self-generating Percoll gradients), cost (Percoll and 
metrizamide are more expensive than BSA), and final use of cells (metrizamide 
must be thoroughly washed whereas Percoll can be used in the culture).  Percoll 
may be preferable because it is nontoxic and has a lower viscosity and 
osmolarity that the equivalent densities of metrizamide and BSA.  Cortical cells 
contain lipid and are less dense on Percoll and metrizamide gradients. 
Chromaffin cells produced by collagenase demonstrate pseudopodia and occasional 
cilia and this may account for an observed mobility of the cells on collagen- or 
poly-DL-ornithine-coated dishes (shown by time-lapse videography).  Cells may be 
stored long-term in a Ca<sup>2+</sup>/BSA-containing buffer at 4&deg; and later 
equilibrated to 37&deg; or RT on the day of use.  Isolated cells stored for 3 
days in Locke's buffer supplemented with 2.2 mM Ca<sup>2+</sup> + 0.5% BSA at pH 
7.0 retain their release characteristics for CAs and ATP.  Note that the 
nicotinic response is temperature-dependent.  Freezing of cells in DMSO has been 
attempted but on rethawing after 2 mon of storage in liq N<sub>2</sub>, cells 
had only 30% of original secretory activity even though morphological appearance 
was not changed.
<p>
The differential plating technique is also useful for separating the 
nonchromaffin from chromaffin cells.  Nonchromaffin cells generally adhere more 
strongly to untreated glass or tissue culture plastic.  Waymire et al. reported 
plating 20 million cells on to a 130 cm2 surface area (150,000/cm2) for 6 h at 
37&deg; and most of the nonchromaffin cells adhered during the first 3 h.  The 
"phase-bright"-appearing cells were plated on to collagen and stained positively 
with FIF;  they were 90-95% pure.
<p>
Perlman and coworkers have found that neutral red staining is a quick and 
convenient method for identifying CA-producing cells in culture.  There is a 
very positive correlation between this staining and FIF.  Staining time and 
incubation conditions are important to be discerning of chromaffin cells in the 
culture.  Many investigators obtain a yield of 100-200 million cells per gland 
with 95% purity.
<p>
<b>Other treatments for selection of chromaffin cells</b>.  Other treatments to 
select for chromaffin cells include the use of antimitotics in the medium, such 
as 5-fluorodeoxyuridine (FDU) and cytosine arabinoside (AraC; Cytosar), gamma 
irradiation, and selective drug treatment.  Gamma irradiation is done after a 24 
h initial plating using a cobalt source (60 Gy for 30 min) which reduces 
nonchromaffin contamination from &gt;25% to &lt;5%.  Chromaffin cell function 
and morphology does not appear to be affected since cells 1) exclude trypan 
blue, 2) retain catecholamines, and 3) neurites appear after NGF addition.  As 
for drug treatment, Unsicker and Ziegler have reported that treatment with 
Bt<sub>2</sub>cAMP and theophylline prevented nonchromaffin cell proliferation, 
even causing death of fibroblast contaminants.
<p>
<b>Leaky and skinned chromaffin cells</b>.  Baker and Knight used chromaffin 
cells isolated by collagenase and some were purified by Percoll.  The two groups 
of cells were placed in a 1 cm Perspex cuvette and subjected to discharges from 
a 1 &micro;F capacitor across stainless steel electrodes 1 cm apart (electric 
field was 2 kV/cm exponential decay with time constant 100 &micro;s).  These 
cells are now permeable to ions but not proteins, since only micromolar 
Ca<sup>2+</sup> additions induce CA release (it takes about 1 h for resealing). 
Medium composition in this exocytotic release also appears important in this 
phenomenon, since chloride must be replaced with glutamate and MgATP is 
specifically required.  Brooks and Treml created "skinned" chromaffin cells 
using saponin for its ability to make membranes ion-leaky.  0.01% saponin could 
be used for 160 s to produce a CA release before cell viability was affected (by 
trypan blue).
<p>
<b>Maintenance of cultures</b>.  Previous studies of culture maintenance used 
cells plated in 1.7 cm wells (250,000 per well).  As for substrata, wells were 
coated with rat-tail collagen,as well as with gelatin and nonbiological 
substrates such as poly-D- and poly-L-lysine at 5-100 &micro;g/ml, 
poly-DL-lysine at 100 &micro;g/ml, or poly-DL-ornithine at 100 &micro;g/ml. 
Collagen proved to be the best, with ornithine nearly as effective, and the rest 
improving adherence to some degree in the short-term.  Some difference in cells 
from different species were observed however:  while human and bovine cells 
survive on poly-DL-ornithine, purified rat cells did not last for more than 3 d 
without collagen.  Both rat and bovine chromaffin cells prefer to grow on feeder 
adrenal nonchromaffin cells.
<p>
Culture medium used in studies includes DME and MEM with Earle's salts.  They 
are essentially supplemented with heat-inactivated FCS or heat-inactivated human 
placental serum (HPS).  Commercial or freshly prepared (from 9-day old 
fertilized eggs) chick embryo extract (CEE) is also a supplement to MEM-HPS 
medium.  A formulation routinely used is 88% MEM, 10% HPS, 2% CEE (50% in 
Earle's BSS), 600 mg% glucose, 10 &micro;M AraC, 10 &micro;M FDU, 10 &micro;M 
uridine, 200 &micro;g/ml penicillin, and 100 &micro;g/ml streptomycin. Sometimes 
mycostatin is used in the summer months since fungi are a problem at that time. 
Cultures are maintained at 36&deg; (not 37&deg;, since survival is better at the 
slightly lower temp) in a 5% CO2 atmosphere and thoroughly humidified.  Note 
that DME was originally formulated to work with 10% CO2, but since chromaffin 
cells work well with DME and the 5% gas, this suggests that the cells produce 
sufficient CO2 by themselves via a high glycolytic activity. Serum-free culture 
is being tried since FCS or HPS could become unavailable for a time.  FCS is 
generally required to effect cell attachment.  DME/F-12 medium was a good 
serum-free choice subsequent to attachment.  The 10% FCS is required for only 2 
d for attachment after which the 1:1 medium supplemented with ascorbate and 
insulin are sufficient.  High levels of catecholamines are found in these cells.
<p>
The use of suspension cultures as opposed to monolayers has been reported in 
which petri dishes of "bacteriological" plastic (non tissue culture plastic) 
were used.  Nonchromaffin cells do not attach to this substratum and most cell 
types will not therefore divide, so the culture medium need not include 
antimitotic drugs.  Rat-tail collagen coating of these plates restores the 
proliferative ability of nonchromaffin cells and AraC is a necessary component. 
These cells were grown only in Ham's F-12 medium containing 10% heat-inactivated 
newborn calf serum:  F-12 is a very minimal medium used as a cloning medium. 
These suspension cultures have been used in studying the regulation of TH and CA 
synthesis and the modulation of the nicotinic response of cells to opioid 
peptides and opiates.  Chromaffin cells have been suspended in Bellco 
microcarrier spinner flasks as well.
<p>
<b>Cell morphology in primary cultures</b>.  Cell morphology and appearance has 
been reported for phase, Nomarski, and Hoffman modulation contrast optics.  CA 
staining with "Faglu" and glyoxylic acid, immunofluorescence microscopy, 
time-lapse videography and cinematography, and ultrastructural examination by 
TEM, SEM, and platinum surface replicas have all been done to characterize 
chromaffin cells.
<p>
It is not surprising that the growth characteristics of chromaffin cells differ 
with respect to species and developmental age within a species.  For example, 
neuritic processes in embryonic, neonatal, and early postnatal cells are longer 
and thinner than from adult cells;  they closely resemble processes in 
monolayers of sympathetic neurons and PC12 cells.  Rates of process extension 
and average length have been compared as well.  Bi- and tripolarity of these 
processes are common as well.
<p>
Chromaffin cells are derived from neural crest cells and have the same origin as 
sympathetic and sensory neurons.  NGF and corticosteroids have significant 
effects on neurite and process outgrowth in cultured sympathetic neurons and 
PC12 cells.  NGF also has significant effects on process outgrowth in embryonic 
and neonatal rat and bovine chromaffin cells (its presence may be critical to 
cell survival in fact).  It has no effect on rat or bovine adult cells, not 
being required for survival of the culture.  Medullary "neurons" are only 
observed with NGF present.  In addition increases in TH specific activity 
(between 2- and 3-fold at up to 1 &micro;g/ml mouse NGF) in both mouse and 
bovine cells are observed.  Inhibition of process extension was found with added 
dexamethasone.  Dexamethasone by itself has no effect on process growth or cell 
survival.  Plating density may have additional effects:  PC12 cells at high 
densities will increase CA synthesis and storage 2- to 3-fold with dexamethasone 
addition, and cholera toxin (presumably through cAMP increase) increases TH 
activity.  This is additive with glucocorticoid addition, which may act via 
transcription since TH induction is blocked by actinomycin D.
<p>
<b>Biochemical characteristics of cultured cells</b>.  Several groups have 
verified that cells show an initial drop in CA levels within the first 3 d of 
culture. Specific drops in epinephrine are more pronounced than for norepi. 
Indeed epinephrine levels are better in relatively impure cultures than in pure 
cultures.  It appeared that cells from bovine adults also retained CA better 
than in 10-day old rats, which lost the ability to store epi and were 75% norepi 
and 25% dopamine.  High K<sup>+</sup> (55 mM) also reduced the amounts of stored 
CAs. Inclusion of ascorbate may be helpful as well:  in PC12 cells dopamine is 
largely present but with ascorbate added the principal CA is norepiphrine. Other 
effects on the synthesis of enkephalins and opioid peptides in both principally 
norepinephrine- and epinephrine-producing cells have been described.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Zaks, A., Klibanov, A. M.
 </span>
 <span class="year">
(1985)
 </span>
 <span class="jname">
Proc. Natl. Acad. Sci. USA
 </span>
 <span class="jvol">
82,
 </span>
 <span class="pages">
3192-3196.
 </span>
<p class="reftitl">
Enzyme-catalyzed processes in organic solvents
<p class="Para1">
Ideal model for enzymatic reactions in organic solvents requires that (1) enzyme 
should be easily available and inexpensive, (2) enzyme should be 
cofactor-independent since common cofactors are insoluble in organic solvents, 
(3) substrates should be soluble in organic media, (4) water should be a 
nonreactant in the process.  Lipases (EC 3.1.1.3) meet all of these 
requirements:  although hydrolysis is the natural reaction, transesterification 
can be performed in the absence of water.  Porcine pancreatic, yeast (Candida 
sp.), and mold (Mucor sp,.) lipases are examined here.
<p>
Tributyrin and heptanol are dissolved in 2-pentanone, dehydrated and a known 
amount of water is added.  A lipase powder is added and the mixture is shaken at 
20&deg; to maintain a suspension since all proteins are insoluble in organic 
solvents (see Singer, S. J., Adv. Prot. Chem. 17: 1, 1962).  The rate of 
lipolysis is measured with respect to changing water content:  The porcine and 
mold lipases are roughly equally active without water, while the yeast lipase 
shows no activity.  As the water content is increased to 0.5%, there is little 
change in the porcine and mold lipase, but the yeast lipase shows rapid 
dependence on water.  Between 0.5 and 1.0% water, the porcine lipase increases 
dramatically while increases in yeast lipase taper off;  there is no change in 
mold lipase activity.  It should be stressed that all three lipases have robust 
activity in as little as 0.02% water, sufficient to complete the reaction in the 
mixture.  Transesterification was truly enzyme-dependent over the days the 
reaction was occurring because inactivation by the center-specific reagent 
diethyl <i>p</i>-nitrophenylphosphate completely abolished catalytic activity in 
the organic medium.
<p>
pH dependence of these enzymes in <span class="underline">aqueous</span> 
solutions is very strong usually.  Porcine lipase was dissolved in aqueous 
buffer, adjusted to a particular pH, the enzyme precipitated in cold acetone, 
recovered and dried in a vacuum.  Activity was assayed in water at pH 8.4 and in 
solvent containing 0.02% water.  Plots of activity vs. incubated pH shows that 
the enzyme has no &ldquo;pH history&rdquo; when it comes to the aqueous medium. 
However, the organic medium clearly showed a prior memory of the pH in which it 
was originally precipitated.
<p>
Different solvents were investigated to assess whether lipase activity depended 
on solvent nature.  All solvents tested (17) showed some activity except for 
DMSO and formamide, which showed none.  Curiously these are the only two 
solvents on the list able to dissolve proteins, it can be assumed that these 
enzymes changed their conformation and became inactivated.  One thing about the 
yeast lipase is that it possessed the highest activity for water-immiscible 
solvents (parrafins, butyl ether, toluene, and carbon tetrachloride) and the 
lowest in the water-miscible (acetonitrile, tetrahydrofuran, dioxane, pyridine, 
acetone).  Data from this investigation can be explained in terms of the model: 
the porcine lipase has some water molecules bound to it which hydrophilic 
solvents can not strip.  The water is bound more loosely in the yeast lipase, 
partitioned into water-miscible rather than water-immiscible solvents.  The main 
effect of the solvents seems to be with the enzyme-bound water and not the 
enzyme itself.
<p>
When initial velocity was plotted against inverse substrate at various 
concentrations of reactant alcohol, straight-line MM-type kinetics were 
observed.  A plot of the V<sub>max</sub> against the inverse of the reacting 
alcohol concentration produces a straight line.  K<sub>m</sub> and 
V<sub>max</sub> determined from this work can be used to examine 
K<sub>m</sub>(ester)/V for a given ester and the K<sub>m</sub>(alcohol)/V for a 
given alcohol and different esters.  This demonstrates a &ldquo;ping-pong&rdquo; 
mechanism of reaction. A comparison of activity in organic solvent as opposed to 
water, in the presence and absence of the enzyme, shows that the acceleration 
effect of the reaction of the enzyme is much greater in the presence of the 
enzyme than in its absence, but the rate in neither organic nor aqueous solvents 
is different.  Among the other reactions which can be catalyzed other than 
transesterification is esterification, aminolysis, acyl exchange, 
thiotransesterification, oximolysis.
<!-- ============================================================= -->
, IRL Press, Oxford.<p class="ref">
 <span class="authors">
Huynh, T. V., Young, R. A., Davis, R. W.
 </span>
 <span class="year">
(1985)
 </span>
"Constructing and screening cDNA libraries in lgt10 and lgt11"
in <i>DNA Cloning.  A Practical Approach</i>. (Glower, D. M., ed.),
 <span class="pages">
pp. 49-78
 </span>
<p class="reftitl">
<!-- ============================================================= -->
<p class="tags">
alpha2 macroglobulin, cloning
<p class="ref" id="first-a2M-cloning">
 <span class="authors">
Kan C-C,  Solomon E, Belt KT, Chain AC, Lynne R. Hiorns LR, Fey G
 </span>
 <span class="year">
(1985)
 </span>
 <span class="jname">
Proc. Natl. Acad. Sci. USA
 </span>
 <span class="jvol">
82,
 </span>
 <span class="pages">
2282-2286.
 </span>
<p class="reftitl">
Nucleotide sequence of cDNA encoding human &alpha;<sub>2</sub>-macroglobulin and 
assignment of the chromosomal locus
<p class="Para1">
A cDNA library of human liver enriched for mRNA species &gt; 2000 nt long was 
maide in pAT 153 Pvu II-8.  Met residues at 1378 (QMAIVD) and 1385 (DVKMV) and 
their surrounding sequences were chosen for probes since they encode triplets 
with minimal degeneracy; the probes synthesized were 17-mer 
5&prime;-TCNACDATNGCCAT[A/G]TT-3&prime; and the 14-mer 
5&prime;-ACCAT[T/C]TTNAC[A/G]TC-3&prime;.  The library is about 15,000 recombinant
clones.  30,000 colonies were plated to nitrocellulose disks, with replica 
filters made from master filters.  Colonies on replica filters are grown,
transferred to chloramphenicol to amplify plasmid.  Colonies are processed and
hybridized with <sup>32</sup>P-labeled 14- and 17-mer probes.  Positive colonies
are rescreened until genetic homogeneity is attained.  cDNA insert length is
assessed by <i>Hind</i>III and <i>Sal</i>I restriction analysis.  For DNA
sequence analysis, the inserts are digested with <i>Hin</i>dIII and <i>Sal</i>I
and fragments electrophoresed in low melting agarosse and recovered using
phenol extraction.  200-400 bp subfragments are shotgun cloned into <i>Sma</i>I
site of M13 mp8 and single stranded DNA from phage obtained.  The dideoxy
terminator method is used [&alpha;-<sup>35</sup>S]-ATP and buffer gradient
gels.  Staden's computer analysis was done to organize sequence.  For
<b>chromsome assignment</b> of the a2M locus, human/rodent somatic cell hybrids
are grown.  High MW DNA from human, rodent, and hybrids are digested with 
<i>Eco</i>RI, <i>Pst</i>I or <i>Taq</i>I, and separated on 0.7% agarose.
DNA is denatured and transferred to nitrocellulose and filters hybridized
with nick-translated a2M cDNA in the following solution:
<ul>
<li>2.5&times; SSC (SSC = 0.15 M NaCl + 15 mM trisodium citrate)
<li>10&times; Denhardt's solution (2% Ficoll, 2% BSA, 2% polyvinylpyrrolidone)
<li>0.1% SDS
<li>6% PEG 600 or 10% dextran sulfate
<li>50 &micro;g/ml denatured salmon sperm DNA
</ul>
<p>
This was done at 65&deg; overnight.  Filters were then washed at 65&deg; 
successively with 2&times;, 1&times;, 0.3&times; and 0.1&times; SSC, three
times for 20 min each.
<p>

Of the 30,000 colonies screened, 34 were found to hybridize both the 17- and 
14-mer probes.  Six recombinants were selected after restriction analysis. One 
was p&alpha;<sub>2</sub>M1 with 4586 bp insert which was sequenced, and it 
contained a 5&prime;-UTR and poly(A) tail.  Both strands were sequenced and each 
character was determined 3.75 times on average.  At nt 44-46 is the Met 
translation initiation site.  22 more amino acids not part of the mature peptide 
but constituting the signal peptide follow.  10 of the 23 N-terminal residues 
show homology with mouse complement C3 signal peptide, indicating a common 
origin.  Comparison with other studies in which amino acid sequence shows only 3 
variant residues: Cys at 563 vs Glu, Gln at 975 vs Glu, and Val at 1000 vs Ile. 
Bases 4558-4563 are a 6-base sequence (AATAAA) for the poly(A) addition. The 
poly(A) sequence is nine bases long (4578-4586), 14 nt downstream of the signal. 
The translational stop codon (UGA) is 89 nt upstream of the poly(A) signal
<p>
This group also used rat-human somatic cell hybrids to show that the locus is in 
chromsome 12.  DNA from human, mouse, and hybrid cells was digested with 
<i>Pst</i>I, separated in agar electrophoresis and blotted by Southern, and then
probed with nick-translated 1600 bp long fragment from 3&prime; end of the
a2M mRNA.  The characterization of the hybrids as to chromosomal segments 
had already been done, so it was possible to locate all hybridizing bands 
as having chromosome 12 in common.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Kadowaki, T., Fujita-Yamaguchi, Y., Nishida, E., Takuku, F.,
Akiyama, T., Kathuria, S., Akanuma, Y., Kasuga, M.
 </span>
 <span class="year">
(1985)
 </span>
 <span class="jname">
J. Biol. Chem.
 </span>
 <span class="jvol">
260,
 </span>
 <span class="pages">
4016-4020.
 </span>
<p class="reftitl">
Phosphorylation of tubulin and microtubule-associated proteins by the purified 
insulin receptor
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Hunter, T., Cooper, J. A.
 </span>
 <span class="year">
(1985)
 </span>
 <span class="jname">
Annu. Rev. Biochem.
 </span>
 <span class="jvol">
54,
 </span>
 <span class="pages">
897-930.
 </span>
<p class="reftitl">
Protein-tyrosine kinases
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Lehrman, M. A., Goldstein, J. L., Brown, M. S., Russell, D. W., Schneider, W. J.
 </span>
 <span class="year">
(1985)
 </span>
 <span class="jname">
Cell
 </span>
 <span class="jvol">
41,
 </span>
 <span class="pages">
735-743.
 </span>
<p class="reftitl">
Internalization-defective LDL receptors produced by genes with nonsense
and frameshift mutations that truncate the cytoplasmic domain
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Brinster, R. L., Chen, H. Y., Trumbauer, M. E., Yagle, M. K., Palmiter, R. D.
 </span>
 <span class="year">
(1985)
 </span>
 <span class="jname">
Proc. Natl. Acad. Sci. USA
 </span>
 <span class="jvol">
82,
 </span>
 <span class="pages">
4438-4442.
 </span>
<p class="reftitl">
Factors affecting the efficiency of introducing foreign
DNA into mice by microinjecting eggs
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Hammer, R. E., Pursel, V. G., Rexroad, C. E., Jr., Wall, R. J.,
Bolt, D. J., Ebert, K. M., Palmiter, R. D., Brinster, R. L.
 </span>
 <span class="year">
(1985)
 </span>
 <span class="jname">
Nature
 </span>
 <span class="jvol">
315,
 </span>
 <span class="pages">
680-683.
 </span>
<p class="reftitl">
Production of transgenic rabbits, sheep, and pigs by microinjection
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Jones, K. A., Yamamoto, K. R., Tjian, R.
 </span>
 <span class="year">
(1985)
 </span>
 <span class="jname">
Cell
 </span>
 <span class="jvol">
42,
 </span>
 <span class="pages">
559-572.
 </span>
<p class="reftitl">
[report concerning Sp1 binding protein or sites in promoters]
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Folks, T., Benn, S., Rabson, A., Theodore, T., Hoggan, M. D.,
Martin, M., Lightfoote, M., Sell, K.
 </span>
 <span class="year">
(1985)
 </span>
 <span class="jname">
Proc. Natl. Acad. Sci. USA
 </span>
 <span class="jvol">
82,
 </span>
 <span class="pages">
4539-4543.
 </span>
<p class="reftitl">
Characterization of a continuous T-cell line susceptible
to the cytopathic effects of the acquired immunodeficiency syndrome
(AIDS)-associated retrovirus
<p class="Para1">
The AIDS retrovirus, a/k/a lymphadenopathy-associated
virus and human T-lymphotropic virus type III, replicates and produces
cytopathic effects in activated human Leu-3<sup>+</sup> cells,
the same T cell type reduced in the clinical
disease. &nbsp;The virus can be propagated in normal human peripheral
blood lymphocytes, susceptibility to infection depends upon the donor and
usually requires phytohemagglutinin or interleukin 2 for
stimulation.  Certain lines which continually produce the virus,
T-lymphoid leukemia cells from Gallo and co-workers and Epstein-Barr
virus-transformed B cells from Montagnier and coworkers, are not killed or
suffer CPE from the virus.
<p>
A3.01 is a hypoxanthine-aminopterin-thymidine(HAT)-sensitive
cell line derived from
the CEM line, originally isolated in 1965 from a child with acute
lymphocytic leukemia.  The clone was produced by selection in
8-azaguanine, and is suitable for human T-lymphocyte fusions.
Markers on A3.01 show them to be &gt;95% Leu-3<sup>+</sup>,
Leu-8<sup>+</sup>, Leu-1<sup>+</sup>, Tac<sup>-</sup>,
and transferrin-receptor positive.  Additional HLA typing indicated
that the parent CEM (obtained as CCRF-CEM from ATCC)
was of type A1,19 B40 Cw3,7 DR<sup>-</sup>.   Because of complement
sensitivity, the A3.01 typing could not be established.
<p>
[Further results remain to be entered here but are not done at this time.]
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Davie, J. R.
 </span>
 <span class="year">
(1985)
 </span>
 <span class="jname">
Anal. Biochem.
 </span>
 <span class="jvol">
144,
 </span>
 <span class="pages">
522-526.
 </span>
<p class="reftitl">
Peptide mapping of basic proteins by proteolysis in
acetic acid/urea minislab polyacrylamide gels
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Damodaran, S.
 </span>
 <span class="year">
(1985)
 </span>
 <span class="jname">
Anal. Biochem.
 </span>
 <span class="jvol">
145,
 </span>
 <span class="pages">
200-204.
 </span>
<p class="reftitl">
Estimation of disulfide bonds using 2-nitro-5-thiosulfobenzoic acid:
limitations
<p class="Para1">
The method of Thannhauser et al. (Anal. Biochem. 138:
181, 1984) uses 2-nitro-5-thiosulfobenzoic acid (NTSB) in the presence of
excess sodium sulfite to quantify disulfide bonds.  The resulting
product is 2-nitro-5-thiobenzoate (NTB), the formation of which can be
measured at 412 nm (molar extinction is 13,600 M<sup>-1</sup> cm<sup>-1</sup>).  1 mol
of NTB is formed for every 1 mol of -S-S- bond.  The reaction
contains 0.2 M Tris HCl, pH 9.5, 2 M guanidine thiocyanate, 3 mM EDTA, and
0.1 M Na<sub>2</sub>SO<sub>3</sub>.  NTSB stock is made at 50 mM and
stored at -20&deg; in aliquots, and is diluted 100 times with the
reaction buffer to a final 0.5 mM on the day of use.  Lysozyme was
used in these particular studies (its cocentration can be determined
spectrophotometrically at 281 nm with E<sub>281</sub> &nbsp;of a
1% solution being 26.3.
<p>
Results show that while absorbance at 412 nm
increases within the first 5 min, there are differences in absorbance vs
reaction time depending upon whether the sample is incubated in the dark,
in the light, or transferred to the dark after being in light for 15
min.  When in the dark, the absorbance reaches a stable asymptote
after 15 min and remains level even after 30 min.  In the light the
reaction never reaches its maximum at the asymptote and the absorbance
decays with a half-time of 15 min.
<p>
The effect of the sulfite concentration was
determined by following absorbance changes over time.  In the
presence of light, NTB reacts presumably with the anion of sulfite to form
a new nonchromophoric substance.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Nagamine, Y., Reich, E.
 </span>
 <span class="year">
(1985)
 </span>
 <span class="jname">
Proc. Natl. Acad. Sci. USA
 </span>
 <span class="jvol">
82,
 </span>
 <span class="pages">
4606-4610.
 </span>
<p class="reftitl">
Gene expression and cAMP
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Ringold, G. M.
 </span>
 <span class="year">
(1985)
 </span>
 <span class="jname">
Annu. Rev. Pharmacol. Toxicol.
 </span>
 <span class="jvol">
25,
 </span>
 <span class="pages">
529-566.
 </span>
<p class="reftitl">
Steroid hormone regulation of gene expression
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Smith, P. K., Krohn, R. I., Hermanson, G. T., Mallia, A. K.,
Gartner, F. H., Provenzano, M. D., Fujimoto, E. K., Goeke, N. M.,
Olson, B. J., Klenk, D. C.
 </span>
 <span class="year">
(1985)
 </span>
 <span class="jname">
Anal. Biochem.
 </span>
 <span class="jvol">
150,
 </span>
 <span class="pages">
76-85.
 </span>
<p class="reftitl">
Measurement of protein using bicinchoninic acid
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Legler, G., M&uuml;ller-Platz, C. M., Mentges-Hentkamp, M., Pflieger, G.,
J&uuml;lich, E.
 </span>
 <span class="year">
(1985)
 </span>
 <span class="jname">
Anal. Biochem.
 </span>
 <span class="jvol">
150,
 </span>
 <span class="pages">
278-287.
 </span>
<p class="reftitl">
On the chemical basis of the Lowry protein determination
<p class="Para1">
Stock solutions for the alkaline copper reagent of Lowry
et al. are prepared:  solution A is
2% Na<sub>2</sub>CO<sub>3</sub> in 0.1&nbsp;N NaOH;  solution B
is 1% CuSO<sub>4</sub> 5 H<sub>2</sub>O and 2.3% NaK tartrate in 0.1&nbsp;N NaOH
(prepare by mixing the copper with the tartrate salt <span class="underline">before</span> adding
the base or the copper will precipitate).  Protein solutions up to
0.1 ml are placed in a tube.  1.0 mL solution A followed with 50&nbsp;&mu;l
of solution B is added.  After 15 min, 0.1 mL of 1&nbsp;N Folin reagent is
then added followed by <span class="underline">immediate</span> mixing.  The OD at 720 nm is
measured 1 hr later.
<p>
Proteins, peptides, and other reducing agents were
used in the assay to examine effect and interference.  With
nonpeptide reductants (Fe<sup>2+</sup>,
Sn<sup>2+</sup>, [Fe(CN)<sub>6</sub>]<sup>4-</sup>, ascorbate,
phenols, indoles) plots of A<sub>720</sub> were
linear up to ~1.5 while the small peptides (tetraalanine) were strongly
curved.  With proteins and polyalanine the curves were
intermediate.  Deviation from linearity might be explained by rapid
decomposition of the heteropolyacids of Folin reagent under alkaline
conditions (pH 9.7) which causes a relatively more rapid depletion of
oxidant when the reaction is slow.  This is more plausible than the
explanation of Ohnishi and Barr who suggested an equilibrium between
oxidant and reduced species, which is impossible under alkaline conditions
because the heteropolyacid anions are completely dissociated into
monomeric ions.
<p>
Oxidation of Fe<sup>2+</sup>, Sn<sup>2+</sup>, and
ascorbate was done under acidic conditions prior to adding excess alkali
afterwards.  This caused a 1.7-fold increase in A<sub>720</sub>
for the pH shift (from 1 to 10) for the substances examined.
This increase in A<sub>720</sub> (representing likewise a change
in the molar absorptivity) was reversed
upon re-acidification.  This is increase in absorbance may be
attributed to a structural change in the tungsten/molybdenum blue complex
caused by deprotonation.  Reactions done with thiols and ascorbate
were done in the absence of Cu to prevent copper-catalyzed
autooxidation.  From the data an average absorbance corresponding to
a change in molar absorptivity at 720 nm is
3200 M<sup>-1</sup> cm<sup>-1</sup> per reducing equivalent.
<p>
Electron transfer from peptides appears to be
determined mainly from formation of a tetradentate complex with
Cu<sup>2+</sup>.  A low color yield is
apparent in dipeptides and for peptides containing proline, which would
work against such a complex.  There is a transfer of two electrons
per tetradentate complex from the stoichiometric data except for glycine
peptides and polyglutamic acid.  The lower color yield for glycine
compared to alanine indicates that there is a facilitation of electron
transfer by the +I effect of the methyl group in alanine or some different
mode of dehydrogenation.  The greater color yield from trialanine
compared to tetraalanine is due to participation of the carboxylate group
as a ligand with the latter:  this negative charge provides greater
stability of the complex and promotes electron transfer to the
oxidant.  Homopolypeptides containing side chains with complex
forming properties are likely to cause copper to associate with side
chains instead of the peptide backbone, producing less of a color
yield.  An the oxidation of polyglutamic acid is likely due to
electrostatic repulsion of the highly negatively charged heteropolyacid
anion.  As for proteins, a higher-than-expected color yield for BSA
and for the ANGTAV hexapeptide might be explained by the presence of
asparagine or histidine residues, found by Chou and Goldstein to
contribute to color formation when present as dipeptides while other amino
acids except tryptophan and tyrosine do not.
<p>
Other increases in color at 720 nm may be the
result of oxidation of additional copper complexes after the first
oxidation event.  Dehydrogenation of the biuret-type copper complex
to an imino peptide converts the anionic copper-binding site to a neutral
one, comparable to the protonation of an anionic peptide nitrogen, leading
to rapid complex decomposition.  Note that the low color yield from
gelatin is due to high proline and hydroxyproline content to prevent the
normal biuret-type copper complex formation.
<p>
Hence the oxidation of the protein copper complexes
by Folin reagent under optimal conditions occurs as one two-electron
dehydrogenation event per tetrapeptide unit.  Tyrosine and tryptophan
contribute other reducing equivalents of four electrons.
<p>
The oxidation products were then examined:
with no explanation for its strong interaction, the blue color product
adheres strongly to the polacrylamide Bio-Gel gel filtration medium, and
so it was used to remove the interfering Folin reagent.  Reaction of
the oxidized tetra- and polyalanine with DNP-hydrazine gave hydrazones
with absorption spectra identical in neutral and alkaline solution with
those of pyruvic amide DNP-hydrazone.
<p>
[There is much more on the analysis of oxidation
products and discussion about the time course of the reaction.]
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Larson, E., Howlett, B., Jagendorf, A.
 </span>
 <span class="year">
(1985)
 </span>
 <span class="jname">
Anal. Biochem.
 </span>
 <span class="jvol">
155,
 </span>
 <span class="pages">
243-248.
 </span>
<p class="reftitl">
Artificial reductant enhancement of the Lowry method for protein determination
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Landreth, G. E., Rieser, G. D. 
 </span>
 <span class="year">
(1985)
 </span>
 <span class="jname">
J. Cell Biol. 
 </span>
 <span class="jvol">
100,
 </span>
 <span class="pages">
677-683.
 </span>
<p class="reftitl">
Nerve growth factor- and epidermal growth factor-stimulated phosphorylation of a 
PC12 cytoskeletally associated protein in situ
<!-- ============================================================= -->
<p class="tags">
formalin fixation, formaldehyde chemistry
<p class="ref">
 <span class="authors">
Fox CH, Johnson FB, Whiting J
 </span>
 <span class="year">
(1985)
 </span>
 <span class="jname">
J. Histochem. Cytochem.
 </span>
 <span class="jvol">
33,
 </span>
 <span class="pages">
845-853.
 </span>
<p class="reftitl">
Formaldehyde Fixation
<p class="Para1">
The value of HCHO as a fixative in histochemistry was described by Bulterov
in 1859, its synthesis was made possible later in 1868 when
Van Hoffman described a synthesis from MeOH.  Trillat patented its industrial
production on a larger scale in 1889.  The firm Meister, Lucius and Brunig (MLB)
located at Hoechst am Main (near Frankfurt am Main) would license this production
in 1891.  Aqueous HCHO had been reported as an antiseptic to treat or prevent
wound infections.  MLB approached Frankfurt native Ferdinand Blum to test this
property.  HCHO gas is bubbled through water until it no longer dissolves, which
is at 40% concentration.  Blum first chose a dilution, typically 1 part in 9 parts
water (4%) and looked at bacteriocidal properties against common organisms
(<i>B anthracis, B typhi, S. aureus, Proteus</i> spp.) and found it slow but
effective.  In a later report Blum noted that chemical had contacted his fingers
and caused them to harden (as with alcohol).  He noted that anthrax-infected
mouse also showed hardening as much as alcohol.  The staining of tissues with
hematoxylin and aniline dyes was excellent.  HCHO caused only marginal shrinkage
and distortion of tissues, unlike alcohol.
<p>
Nomenclature has changed over the years.  British companies called it formic
aldehyde while MLB used the name Formol.  The Chemischen Fabrik auf Actien
(later Schering) called it as <i>formalin</i>, for saturated aqueous formaldehyde
solution.  However, American producers called formalin the 40% aqueous solution,
and so call the diluted solution 10% formalin.  Concentrated HCHO solutions
will undergo condensation reactions, and these can be inhibited by including
alcohols as a preservative (hence 10% MeOH presence).
<p>
One problem with HCHO as a fixative is the penetration-fixation paradox.
HCHO has a MW of 30 and diffusion theory predicts that it is not dependent on
HCHO concentration.  Using coagulated chicken plasma, Medawar wanted to assess
just how long diffusion took.  Between a 5-fold concentation range (40% to 8%),
penetration was at a constant rate (he used a chemical method to assess
penetration).  He could not assess whether fixation of the fibrin clot occurred
however.  Previous work indicated that fixation was however slow: thus penentration
is rapid, but fixation slow.
<p>
An artifact of using formalin fixation is that formic acid can or might
accumulate.  This will result in formation of birefringent finely divided
pigment precipitates in blood-rich specimens:  results probably from hematin
derivative when pH &lt; 6.0 in fixative.  To prevent oxidation to HCOOH or
to reduce its presence, dilute HCHO solutions have been stored with
excess CaCO<sub>3</sub> (<q>marble chips</q>) which neutralizes any acidity
created by formation of HCOOH.  Phosaphate-buffered formalin serves the same
purpose.  Passing over a mixed-bed ion-exchange resin will also remove protons
and formate ions.
<p>
When HCHO dissolves in water, it forms methylene glycol
[H<sub>2</sub>C(OH)<sub>2</sub>].  Aqueous HCHO solutions show a reactivity
that is an example of a <q>clock</q> reaction:  the equilibrium favors
methylene glycol so much that formation of HCHO is a real-time clock measurable
in hours.  Both molecules penetrate immersed tissues but HCHO is the reactive
form, and it forms quickly in equilibrium.  Generally the shift back to HCHO
is favor by low pH, high HCHO concentrations, and elevated temperatures in
leathering (tanning).  Functional groups believed to be reactive with the
carbonyl HCHO are primary amines, thiols, and less reactive groups like
primary amides, guanidine groups, and tyrosine ring carbons.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Hosang, M., and Shooter, E. M.
 </span>
 <span class="year">
(1985)
 </span>
 <span class="jname">
J. Biol. Chem. 
 </span>
 <span class="jvol">
260,
 </span>
 <span class="pages">
655-662.
 </span>
<p class="reftitl">
Molecular characteristics of nerve growth factor receptors on PC12 cells
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Dockham, P. A., Steinfeld, R. C., Stryker, C. J., Jones, S. W., Vidaver, G. A.
 </span>
 <span class="year">
(1986)
 </span>
 <span class="jname">
Anal. Biochem.
 </span>
 <span class="jvol">
153,
 </span>
 <span class="pages">
102-115.
 </span>
<p class="reftitl">
An isoelectric focusing procedure for erythrocyte
membrane proteins and its use for two-dimensional electrophoresis
<p class="Para1">
INTRO.  In doing a two-dimensional IEF/SDS gel
analysis of proteins as in O'Farrell's technique, membrane proteins must
necessarily be solubilized usually by dissolution in SDS.  In the IEF
part however, SDS seriously affects focusing resolution and works poorly
with membrane proteins from microsomes and those which are highly
hydrophobic.  Others have reported that dissolving membranes in
zwitterionic detergents such as NP-40 produced better results.  But
the patterns of some gels showed streaking from the origin.  This
report shows two-dimensional analysis free of SDS use in the first
dimension using membranes which have been treated by reductive methylation
(using formaldehyde and tritiated borohydride,
B[<sup>3</sup>H]<sub>4</sub><sup>-</sup>) and for selective labeling
of the anionic
porter components of RBC protein band 3 and other proteins using pyridoxal
5-phosphate (PLP) and tritiated borohydride.
<p>
METHODS.  An RBC lysis buffer of 9 M urea, 6.4% (w/v) Triton X-100,
0.205 M L-lysine, 5 mM DTT, and 15 mM Na thioglycolate is prepared.
IEF tube gel solution of 9 M urea, 2% (w/v) TX-100, 0.8% (w/v)
pH 5-8 Ampholines (LKB), 1.2% pH 3.5-10 ampholines, 4.85% acrylamide,
0.15% bisacrylamide, and 3.1 mM ammonium persulfate is prepared.
An IEF gel overlay is made from the IEF solution except that urea
is 8.5 M and there are no acrylamides.  An
IEF sample overlay is made with 0.5 the strength of lysis buffer and urea
at 8 M.  At the electrodes, solutions will be 0.01 M phosphoric acid
and 0.02 M NaOH.
<p>
For SDS equilibration buffer:  2.5% SDS, 62.4 mM Tris HCl, pH 6.8,
10% glycerol (w/v), 5 mM DTT, and 5 mM sodium thioglycolate.
<p>
Membranes which were worked up to a stage of being
in 10 mM KCl were made 9 M in urea and mixed 1:1 with lysis buffer and
sonicated for 30 min at 25&deg;.  95 mm long gels in 3 &times; 120-140 mm glass
tubes were polymerized under gel overlay solution.  After
polymerization, the overlay was removed and sample overlay placed on the
gel.  The sample was introduced and focusing commenced at 25&deg; for 12
h at 400 V, then 2 h at 1500 V.  Gels can be stained (extrude into a
50% MeOH fix) or prepared for the 2nd dimension.
<p>
Focused gels are equilibrated in SDS equilibration
buffer (end-over-end mixing) for 15 min at RT.  Gels are then placed
in fresh buffer prewarmed to 40&deg;, kept 5 min at this temp, then quickly
rinsed 2 times with SDS gel overlay solution.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Jarrett, H. W., Cooksy, K. D., Ellis, B., Anderson, J. M.
 </span>
 <span class="year">
(1986)
 </span>
 <span class="jname">
Anal. Biochem.
 </span>
 <span class="jvol">
153,
 </span>
 <span class="pages">
189-198.
 </span>
<p class="reftitl">
The separation of o-phthalaldehyde derivatives of amino
acids by reversed-phase chromatography on octylsilica columns
<p class="Para1">
Separation and quantitation of amino acids to this point
is classically done by ion-exchange chromatography followed by post-column
derivatization.  Disadvantages to this system include necessity of
specialized equipment, long analysis times (of 1-3 h), and a postcolumn
chemistry in which the limits of detection are susceptible to elution
buffer contaminants.  Precolumn derivitization with o-pthalaldehyde
and other reagents followed by HPLC can be done at room temp and produces
few if any side reactions.  Limits of detection are 50 fmol and
analysis can be done using conventional HPLC.  However precolumn
derivitization with OPA does not work with proline or other 2&deg; amino
acids;  4-chloro-7-nitrobenzofurazan can be used as an alternative
reagent.  Cysteine produces a low fluorscence but can be assessed as
cysteic acid or carboxymethylcysteine in a prior conversion.  Lysine
also produces a lower fluorescence but the use of detergents in the
precolumn chemistry provides a truer value.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Redinbaugh, M. G., Turley, R. B.
 </span>
 <span class="year">
(1986)
 </span>
 <span class="jname">
Anal. Biochem.
 </span>
 <span class="jvol">
153,
 </span>
 <span class="pages">
267-271.
 </span>
<p class="reftitl">
Adaptation of the bicinchoninic acid protein assay for
use with microtiter plates and sucrose gradient fractions
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Golumbeski, G. S., Jr., Dimond, R. L.
 </span>
 <span class="year">
(1986)
 </span>
 <span class="jname">
Anal. Biochem.
 </span>
 <span class="jvol">
154,
 </span>
 <span class="pages">
373-381.
 </span>
<p class="reftitl">
The use of tolerization in the production of monoclonal
antibodies against minor antigenic determinants
<p class="Para1">
Attempts to prepare MAbs against beta-glucosidase (a
lysosomal protein) from <i>Dictyostelium discoideum</i> did not work
because a fusion on the protein was highly immunogenic and recognized all
lysosomal protein of Dictyostelium.  To compensate for this, a BALB/c
mouse neonate was immunized to the common epitope within 40 h after birth
by injection of the lysosomal protein, N-acetylglucosaminidase.  The
animal was therefore tolerized to the common lysosomal epitope.  When
the mouse reached adult stage, it was then put on immunization schedule
with beta-glucosidase.  Nine hybridomas specific for the enzyme were
recovered.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Burzio, L. O., Zarraga, A. M., Siddiqui, M. A. Q.
 </span>
 <span class="year">
(1986)
 </span>
 <span class="jname">
Gene Anal. Tech.
 </span>
 <span class="jvol">
3,
 </span>
 <span class="pages">
93-95.
 </span>
<p class="reftitl">
[report including method for performing luciferase
activity assay]
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Caprioli, R. M., Fan, T.
 </span>
 <span class="year">
(1986)
 </span>
 <span class="jname">
Anal. Biochem.
 </span>
 <span class="jvol">
154,
 </span>
 <span class="pages">
596-603.
 </span>
<p class="reftitl">
Peptide sequence analysis using exopeptidases with
molecular analysis of the truncated polypeptides by mass spectrometry
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Kubo, K., Takagi, T.
 </span>
 <span class="year">
(1986)
 </span>
 <span class="jname">
Anal. Biochem.
 </span>
 <span class="jvol">
156,
 </span>
 <span class="pages">
11-16.
 </span>
<p class="reftitl">
Binding of lithium dodecyl sulfate to polyacrylamide gel at 4&deg;
perturbs electrophoresis of proteins
<p class="Para1">
Polyacrylamide gels have no affinity for lithium dodecyl
sulfate (LDS) at room temp., but gels maximally bind 17 mg LDS per gram
dry weight at 4&deg;.  LDS gel electrophoresis might be necessary for
samples to be run at 4&deg; since SDS is much less soluble.
Electrophoresis in LDS gels at 4&deg; using a continuous buffer system caused
a migration of proteins with lower MWs to be accelerated:  this was
presumably because of the deficiency of LDS at the front of
electrophoretic migration.  However, when LDS is used in saturating
amounts instead of SDS at 4&deg;, the resolution is higher than that of
regular SDS gels.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Choi, H.-Y., Hwang, K. J.
 </span>
 <span class="year">
(1986)
 </span>
 <span class="jname">
Anal. Biochem.
 </span>
 <span class="jvol">
156,
 </span>
 <span class="pages">
176-181.
 </span>
<p class="reftitl">
Application of anion-exchange resin to remove lipophilic chelates
from liposomes
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Keler, T., Nowotny, A.
 </span>
 <span class="year">
(1986)
 </span>
 <span class="jname">
Anal. Biochem.
 </span>
 <span class="jvol">
156,
 </span>
 <span class="pages">
189-193.
 </span>
<p class="reftitl">
Metachromatic assay for the quantitative determination of bacterial endotoxins
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Bachas, L. G., Meyerhoff, M. E.
 </span>
 <span class="year">
(1986)
 </span>
 <span class="jname">
Anal. Biochem.
 </span>
 <span class="jvol">
156,
 </span>
 <span class="pages">
223-238.
 </span>
<p class="reftitl">
Theoretical models for predicting the effect of bridging
group recognition and conjugate substitution on hapten enzyme immunoassay
dose-response curves
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Saul, A., Yeganeh, F.
 </span>
 <span class="year">
(1986)
 </span>
 <span class="jname">
Anal. Biochem.
 </span>
 <span class="jvol">
156,
 </span>
 <span class="pages">
354-356.
 </span>
<p class="reftitl">
Electrophoretic identification of fusion proteins expressed in
single recombinant lambda bacteriophage plaques
<!-- ============================================================= -->
<p class="ref">
 <span class="authors">
Russell, P., Nurse, P. 
 </span>
 <span class="year">
(1986)
 </span>
 <span class="jname">
Cell
 </span>
 <span class="jvol">
45,
 </span>
 <span class="pages">
145-153.
 </span>

<p class="reftitl">
<i></i> functions as an
inducer in the mitotic control of fission yeast
<p class="Para1">
The <i>cdc25</i><sup>+</sup> gene was cloned
using yeast transformation and complementation using the 
cdc25</i>-<i>22</i> strain. 
One transformant with a cdc25<sup>+</sup>-containing plasmid (pCDC25-1)
was identified.  A <i>32</i> h<sup>+</sup> strain was
transformed with an <i>S. cerevisiae</i> <i class=
'c7'>LEU2</i> gene complementing <i>32</i>.  Gene
bank constructed with HindIII partial digest of S. pombe DNA.<span class=
Selection for leucine prototrophy at 25&deg; on mimimal
medium, then replica plating to rich medium at 35&deg; for cdc25<sup>+</sup>
activity screening.  Plasmid DNA is isolated
from unstable transformants having leu<sup>+</sup> cdc<sup>+</sup>
phenotype.  Genomic DNA isolation and
Southern analysis show four stable transformants had pCDC25-1 integrated
at the <i>cdc25</i>-<i class=
'c7'>22</i> <i>32</i> h<sup>-</sup> strains.<span
Spore viablity was &gt; 90% and random spore
analysis shows the leu<sup>+</sup> plasmid had integrated within 1 cM of
the <i>cdc25</i><sup>+</sup> locus.
<p>
Subcloning of pCDC25-1 was done to check for
complementation and for sequencing purposes. 
Restriction analysis of the plasmid using eleven different enzymes was
done and a map presented.  Three subclones
(pCDC25-S1, -S2, and S3) complemented cdc25-22 mutants, although the S3
clone (a <i>Pvu</i>I fragment) does so
weakly.  The S4 subclone only complements if
integrated at cdc25<sup>+</sup> locus, indicating that the lesion causing
cdc25-22 is contained in this fragment (internal <i>Xba</i>I site).<span class=
The necessary elements of cdc25<sup>+</sup> expression
are fully contained between the SstI and XbaI sites in the 7.5 kbp long
plasmid, and the sequence of this 2745 bp fragment was therefore
determined.  Both the complementation studies
and analysis of DNA sequence for open reading frames indicate that a 1.74
kbp open reading frame starting ~200 upstream of the BamHI site and
terminating ~300 bp upstream the XbaI site are involved.<span class=
One reading frame codes for a polypeptide 580 residues
long with a MW of 67,000.  Another subclone
pSX179 contains on 60 bp of 5' flanking and 280 bp of 3' flanking sequence
and supports complementation, further confirming that the protein is
contained in this sequence.  Other RNA
mapping studies have indicated that coding passes through internal
restriction sites (cDNAs are cut, that is). 
The Wilbur and Lipman method were used for doing Doolittle, NBRF, and EMBL
data base searches.
<p>
Temperature-sensitive cdc25 mutants can be
suppressed by <i>wee1</i><sup>-</sup> mutants, which inactivate
the dosage-dependent mitotic inhibitor <i class=
'c11'>wee1</i><sup>+</sup>, and it is thought that <i>cdc25</i><sup>+</sup> might be a dosage-dependent mitotic
inducer.  If <i>cdc25</i><sup>+</sup> product works as expected, abnormal increases
in its expression should result in premature (early initiated) mitosis
which is characterized by small cell size at division.<span class=
Increasing expression can be achieved by getting
multicopy plasmids to express their product, but most plasmids are
unstable in S. pombe.  An alternative is to
integrate multiple copies of the gene at the chromosomal <i>cdc25</i><sup>+</sup> locus.  Cells
dividing with a small size were isolated for stable integration and for
presence of multiple copies of pCDC25-1.  A
<i>22</i> mutant strain was
transformed with linearized plasmid cut at the SstI site near the <i
class='c7'>cdc25</i><sup>+</sup> open reading frame, since linearized
plasmids are better integrators.  Five
particular clones with normal division times but reduced size were
identified by microscopic screening, with a size (as measured by cell
length/diameter) reduced to between 18-24% relative to wild-type.<span
This phenotype is "semiwee."<span class=
Southern hybridization and RNA mapping was checked to
see for changes in <i>cdc25</i><sup>+</sup> expression.<span
Southerns revealed integrants with copy numbers
of 3 to 5 (controls had only one).  S1
nuclease protection and quantitative analysis shows a higher level of
transcription of <i>cdc25</i><sup>+</sup> in semiwee as well,
with five times more message than in wild-type.
<p>
In another method transcription could be increased
by fusing the region immediately upstream the <i>cdc25</i><sup>+</sup> gene to the strong alcohol dehydrogenase
promoter of <i>S. pombe</i> in a plasmid pACSX.<span class=
Three different inserts were constructed and inserted
into the vector containing <i>LEU2</i> and 2 &mu; ori.<span class=
All three plasmids complement cdc25-22 mutants.<span
The plasmid pSX-179 with the shortest distance
between adh promoter and gene was integrated in the cdc25<sup>+</sup>
locus, and Northern of four such integrants show cdc25<sup>+</sup> RNA
levels far exceeding controls.  Cell division
occurred at just about half the size of wild-type cells, and wee phenotype
of these pSX integrants is similar to the <i>wee1</i><sup>-</sup> mutants.
<p>
Since increased cdc25 expression reduces size at
cell division, then the cause must be due to a shortened G<sub>2</sub>
period.
<p>
How do <i>wee1</i><sup>-</sup> mutations
lead to suppression of cdc25<sup>t.s</sup> (temp-sensitive mutants)?<span
wee1<sup>+</sup> gene product may act by
repressing cdc25<sup>+</sup> gene expression or by reducing product
activity.  If either were the case and if
cdc25-ts is partially active at the restrictive temperature, then
wee1<sup>-</sup> suppression of cdc25-ts alleles may occur by derepression
of cdc25-ts gene product activity.  Three
alleles are known for cdc25-ts and two retain activity at 35&deg;.<span class=
First cdc25<sup>+</sup> was deleted from the
chromosome, done by a known method called the one-step replacement method
and shown for S. cerevisiae.  [Then a 900 bp
region inside cdc25<sup>+</sup> open reading frame was replaced with
ura4<sup>+</sup> and the deletion mutant used to disrupt a
cdc25<sup>+</sup> gene in a diploid strain. 
This requires a double crossover event. 
Southern analysis revealed that transformants were obtained in this
way.]  Then this mutant was used to construct
a diploid strain having genotype:  <i>ura4</i><sup>+</sup>/<i class=
'c7'>cdc25</i><sup>+</sup> <i>ura4</i>-<i class=
'c7'>294</i>/<i>294</i>
h<sup>+</sup>/h<sup>-</sup>.  wee1-50 is
temperature sensitive, producing the wee phenotype at 35&deg;.<span class=
&gt; 90% of spores of this strain germinated at
35&deg;.  26% of these germinated spores then
arrested as very elongated cells without division at 35&deg;, indicating that
cdc25<sup>+</sup> was disrupted.  About 21%
(a nearly equal fraction) germinated to produce viable cells at 35&deg;, but
arrested with a cdc<sup>-</sup> phenotype at 25&deg;.<span class=
These cells were ura<sup>+</sup> and hence contained
the deletion allele.  Tetrad analysis shows
that had the wee1-50 allele as well.
<p>
Hence wee1<sup>-</sup> suppression of
cdc25<sup>t.s</sup> alleles does not occur by derepression of
cdc25<sup>t.s</sup> activity which is residual at restrictive
temperature.  The mechanism is independent of
cdc25<sup>+</sup> activity.  Moreover, cdc25
function is not essential for mitotic induction, but only necessary if the
wee1<sup>+</sup> inhibitor is active. 
cdc25<sup>-</sup> mutants arrest in G<sub>2</sub> because they fail to
overcome the inhibition produced by wee1<sup>+</sup> and so
cdc25<sup>+</sup> is designed to be a check on wee1<sup>+</sup>.
<p>
These studies also demonstrated that premature
mitosis caused by wee1<sup>-</sup> and by increased cdc25<sup>+</sup>
expression were additive.  If
cdc25<sup>+</sup> works by altering wee1<sup>+</sup> activity (whether at
the transcriptional or posttranslational level) then cdc25<sup>+</sup>
increased activity should have no effect in a wee1<sup>-</sup>
mutant.  A strain with temp-sens wee1-50
mutant and an integrated copy of the adh promoter/cdc25<sup>+</sup> gene
fusion plasmid pSX-179 was prepared.  It is
viable at 25&deg; and has a wee phenotype similar to wee1<sup>+</sup> strains
so transformed.  With a temperature shift,
cell division stops and cell viability falls to &lt;1%.<span class=
Nuclear and mitotic abnormalities are evident (nuclei
partitioned to the same half of a dividing cell with anucleate and
binucleate cells; division without mitosis; bisection of the early mitotic
spindle by the cell plate; unequal nuclear division with different nuclear
sizes; asymmetrical location of cell plates).<span class=
These abnormalities and dramatic decrease in viability
occur about 2 h after the temperature shift. 
Cell plates were seen when cells are only 5 &mu;m long, even smaller than wee
mutant cells.  Hence a temporal synchrony
between mitosis and cell division is disturbed.<span class=
What is seen here would be translated as an additive
effect of the increased <i>wee1</i><sup>-</sup> mutation.
<p>
A possibility remains that <i>wee1</i>-<i
class='c7'>50</i> activity to induce the lethal phenotype seen since no
evidence has been obtained showing that no residual <i>wee1</i><sup>+</sup> activity is present in ts mutants at the
restrictive temperature.  But this is
unlikely inasmuch as <i>50</i> cells
grown 35&deg; are no less wee than over 100 independently isolated
nonconditional <i>wee1</i><sup>-</sup> allele.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Fanger, B. O., Sporn, M. B.
 </span>
 <span class="year">
(1986)
 </span>
 <span class="jname">
Anal. Biochem.
 </span>
 <span class="jvol">
156,
 </span>
 <span class="pages">
444-453.
 </span>
<p class="reftitl">
A binding assay for the solubilized receptors of type beta
transforming growth factor:  adsorption and removal of free ligand by
dextran-coated charcoal
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Frmont, P., Dionne, F. T., Rogers, P. A.
 </span>
 <span class="year">
(1986)
 </span>
 <span class="jname">
Anal. Biochem.
 </span>
 <span class="jvol">
156,
 </span>
 <span class="pages">
508-514.
 </span>
<p class="reftitl">
Recovery of biologically functional messenger RNA from agarose gels
by passive elution
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Arathoon, W. R., Birch, J. R.
 </span>
 <span class="year">
(1986)
 </span>
 <span class="jname">
Science
 </span>
 <span class="jvol">
232,
 </span>
 <span class="pages">
1390-1395.
 </span>
<p class="reftitl">
Large-scale cell culture in biotechnology
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Moolenaar, W. H., Aerts, R. J., Tertoolen, L. G. J., de Laat, S. W.
 </span>
 <span class="year">
(1986)
 </span>
 <span class="jname">
J. Biol. Chem.
 </span>
 <span class="jvol">
261,
 </span>
 <span class="pages">
279-284.
 </span>
<p class="reftitl">
The epidermal growth factor-induced calcium signal in A431 cells
<p class="Para1">
EGF added to A431 cells causes a 2- to 4-fold increase in
free cytoplasmic Ca<sup>2+</sup>, as determined by quin-2 fluorescence.
The effect in rapid but transient, being
maximal with 30-60 s and returning to its basal value of 182 nM after 5-8
min.  The induced effect is entirely dependent upon extracellular
Ca<sup>2+</sup>, is abolished by La<sup>3+</sup> and Mn<sup>2+</sup>,
and does not affect membrane potentials.  Serum-induced increases in
calcium are observed as well, but this does not depend upon extracellular
Ca.  The phorbol ester TPA completely inhibits the EGF- and
serum-induced increase in Ca without changing basal Ca itself.  Hence
EGF acts by triggering voltage-independent Ca channels while serum affects
Ca increases through intracellular stores.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Darnell, J. E., and Doolittle, W. F.
 </span>
 <span class="year">
(1986)
 </span>
 <span class="jname">
Proc. Natl. Acad. Sci. USA
 </span>
 <span class="jvol">
83,
 </span>
 <span class="pages">
1271-1275.
 </span>
<p class="reftitl">
Speculations on the early course of evolution
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Aguayo, J. B., Blackband, S. J., Schoeniger, J., Mattingly, M. A., Hintermann, M.
 </span>
 <span class="year">
(1986)
 </span>
 <span class="jname">
Nature
 </span>
 <span class="jvol">
322,
 </span>
 <span class="pages">
190-191.
 </span>
<p class="reftitl">
Nuclear magnetic resonance imaging of a single cell
<p class="Para1">
NMR is now established in clinical imaging and competes
favorably with conventional X-ray computerized tomography.  Fields of
1.5 T (60 MHz) are normally used in whole body imaging.  Images of
sub-millimeter resolution are possible.  Imaging at higher fields is
desirable since signal increases with the square of the magnetic
field.  A modified 9.5 T 89-mm bore high resolution NMR spectometer
has allowed the first imaging of a single cell.  Gradients for
spatial delineation in excess of 20 G cm<sup>-1</sup> were obtained.
A radiofrequency (r.f.) modulator, linear amplifier, waveform memories
for gradient and r.f. control, and image display was interfaced.
The spin-echo imaging technique was used, and the delay between
excitation and acquisition of the spin-echo (TE) and the time
between excitation pulses (TR) is given for each image.
<p>
Xenopus ova were obtained by standard surgical
extraction techniques and soaked in Barth's solution.  Ova in
different stages of oogenesis were placed in a 1.1 mm (i.d.) tube and
imaged.  The nucleus is imaged easily, while a variation in image
intensity spreads from one pole (animal) to the other (vegetal).
Differences in image intensity can be attributed to proton density
variations or differences in relaxation times, or both.  Water in the
cell cytoplasm is distinct from the free water surrounding the cell, and
the signal in the nucleus is different from the cytoplasmic signal but
indistinguishable from the extracellular free water.  This suggests
that nuclear water is less strongly bound.  The signal-to-noise ratio
of these images indicates that a greater spatial resolution can be
obtained in similar times.  A 10 &times; 13&nbsp;&mu;m resolution
with a 250&nbsp;&mu;m slice width has been achieved.  A greater
signal is observed with the animal pole compared to the vegetal pole.
<p>
A chemical change was also detected during imaging.  A signal probably
related to lipid metabolism was observed to increase and then disappear
during the study.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Woo, D. D. L., Fay, S. P., Griest, R., Coty, W., Goldfine, I., Fox, C. F.
 </span>
 <span class="year">
(1986)
 </span>
 <span class="jname">
J. Biol. Chem.
 </span>
 <span class="jvol">
261,
 </span>
 <span class="pages">
460-467.
 </span>
<p class="reftitl">
Differential phosphorylation of the progesterone receptor
by insulin, epidermal growth factor, and platelet-derived growth factor
receptor tyrosine protein kinases
<p class="Para1">
PDGF and insulin receptors were purified from murine NR6 cells and
human placental membranes, resp.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Jacobs, S., Cuatrecasas, P.
 </span>
 <span class="year">
(1986)
 </span>
 <span class="jname">
J. Biol. Chem.
 </span>
 <span class="jvol">
261,
 </span>
 <span class="pages">
934-939.
 </span>
<p class="reftitl">
Phosphorylation of receptors for insulin and insulin-like growth factor I:
effects of hormones and phorbol esters
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Schreiber, A. B., Winkler, M. E., Derynck, R.
 </span>
 <span class="year">
(1986)
 </span>
 <span class="jname">
Science
 </span>
 <span class="jvol">
232,
 </span>
 <span class="pages">
1250-1253.
 </span>
<p class="reftitl">
Transforming growth factor-&alpha;:  a more potent angiogenic mediator
than epidermal growth factor
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Montminy, M. R., Sevarino, K. A., Wagner, J. A., Mandel, G., Goodman, R. H.
 </span>
 <span class="year">
(1986)
 </span>
 <span class="jname">
Proc. Natl. Acad. Sci. USA
 </span>
 <span class="jvol">
83,
 </span>
 <span class="pages">
6682-6686.
 </span>
<p class="reftitl">
Identification of a cyclic AMP responsive element within the rat somatostatin gene
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Theisen, N., Lohoff, E. S., von Figura, K., Hasilik, A.
 </span>
 <span class="year">
(1986)
 </span>
 <span class="jname">
Anal. Biochem.
 </span>
 <span class="jvol">
152,
 </span>
 <span class="pages">
211-214.
 </span>
<p class="reftitl">
Sequential detection of antigens in Western blots with differently colored products
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Serwer, P., Hayes, S. J., Griess, G. A.
 </span>
 <span class="year">
(1986)
 </span>
 <span class="jname">
Anal. Biochem.
 </span>
 <span class="jvol">
152,
 </span>
 <span class="pages">
339-345.
 </span>

<p class="reftitl">
Determination of a particle's radius by two-dimensional
agarose gel electrophoresis
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Plikaytis, B. D., Carlone, G. M., Edmonds, P., Mayer, L. W.
 </span>
 <span class="year">
(1986)
 </span>
 <span class="jname">
Anal. Biochem.
 </span>
 <span class="jvol">
152,
 </span>
 <span class="pages">
346-364.
 </span>
<p class="reftitl">
Robust estimation of standard curves for protein
molecular weight and linear-duplex DNA base-pair number after gel
electrophoresis
<p class="Para1">
Ferguson (Metabolism 13: 985, 1964) has presented the
concepts leading to directly analyzing gel electrophoretic migration to
determination of apparent molecular weights of proteins.  These
followed from the theoretical discussions of Ogston (Trans. Faraday Soc.
54: 1754, 1958) whose studies indicate that such determinations can follow
from an analysis of the components at different gel concentrations.
There are many reports however that the analyses show nonlinearity outside
a limited range of weights.  Relative mobilities are often used to
compensate for the irregularities that occur with these systems.
<p>
Logarithmic transformations used to produce linear
plots may not be the best method for the entire range of
observations.  The Pearson correlation coefficient is a measure of
fit that may indicate that such transformations work well, but comparisons
of known MWs of proteins or #bp of DNA with predicted values may show a
large percentage error.
<p>
The linear equation of the form <i>y<sub>i</sub></i> = <i>a</i> +
<i>b(x<sub>i</sub>)</i> + <i>e<sub>i</sub></i> for i=1,...,n (the
number of data pairs) applies.  <i>y<sub>i</sub></i> is the
log(MW or #bp DNA), <i>x<sub>i</sub></i> is the migration distance, and
<i>e<sub>i</sub></i> is the error associated with the fit.
The following equation however is a nonlinear function which
more accurately describes the inherent curvature
found in data observations:
<p>
<i>f(x<sub>i</sub>,<b>O</b>)</i> = <i>y<sub>i</sub></i> =
<i>a + b/(1+c(x<sub>i</sub>)<sup>d</sup>) + e<sub>i</sub></i>
<p>
The meanings of <i>y<sub>i</sub></i>, <i>x<sub>i</sub></i> and
<i>e<sub>i</sub></i> are the same but the vector <b><i>O</i></b> is a
vector of the parameters <i>a, b, c,</i> and <i>d.</i>  The first
coefficient a refers to an estimate of the horizontal asymptote of the
curve or that point on the y axis (MW or #bp DNA) where the hyperbola
would become horizontal if the curve were extended indefinitely.  The
coefficient b is the point on the y axis where the curve becomes vertical
again.  Coefficients c and d are measures of curvature, determining
its sharpness.
<p>
First the x<sub>i</sub> values are normalized to a reference point
such that they fall in a numerical range of between 0 and 1.  The
shortest distance (that is
the distance of the band with the highest MW or bp #) is substracted from
all values to correct for the problem of band-distance offset.  A
correction factor of 0.01 was added to each of the readings to avoid a 0
migration distance associated with the largest fragment.  Now each
value is multiplied by 100 to give a new range of value from 1.00 to
101.0:
<p>
x<sub>normalized</sub> = [(migration dist/ref dist - minimum mig dist)
+ 0.01] &times; 100.
<p>
Given the function y<sub>i</sub> = f(x<sub>i</sub>,<b>O</b>),
the vector <b>O</b> may be estimated using the Taylor series expansion or
linearization method.  It can be shown that the parameters defining
the vector may be expressed in the form of a pseudo-weighted least squares
estimate:
<p>
<b><i>O</i></b><i><sub>j</sub></i><sub>+1</sub> =
  <b><i>O</i></b><i><sub>j</sub></i> +
  <i>(<b>A</b><sub>j</sub>'<b>W</b><sub>j</sub><b>A</b><sub>j</sub>)</i><sup>-1</sup><b><i>A</i></b><i><sub>j</sub><b>W</b><sub>j</sub><b>R</b></i><i><sub>j</sub></i>
<p class="Para1">
where <b><i>O</i></b><i><sub>j</sub></i> is the former estimate and
<b><i>O</i></b><i><sub>j+1</sub></i> is the revised estimate.
<b><i>A</i></b> is the matrix of partial derivatives of the
function with respect to the parameters, evaluated at
<b><i>O</i></b><i><sub>j</sub></i>.
<p>
Setting A<sub>ip</sub> equal to the partial derivative of f(x,O)
with respect to O<sub>p</sub>, where p=1,..,4 for a, b, c, and d
evaluated at the normalized migration distance x<sub>i</sub>,
then the matrix A is:
<p>
A<sub>11</sub>&nbsp;A<sub>12</sub>&nbsp;A<sub>13</sub>&nbsp;A<sub>14</sub>
<p>
<sup>.</sup>&nbsp;&nbsp;<sup>.</sup>&nbsp;&nbsp;<sup>.</sup>&nbsp;&nbsp;<sup>.</sup>
<pre>
BASIC program provided in Appendix (this adapted to MS QuickBASIC)
' Iterative robust and approximate least squares solutions for the
' modified hyperbola.
' For input the data should be entered in the following order:
' The graph length or total length of the gel;  the number of readings
' for the standard curve; the individual data pairs--the migration
' distance followed by the known MW or DNA fragment length.  The
' observation  corresponding to the heaviest or longest band MUST be
' entered first.  This is followed by the number of unknown migration
' distances from which MWs or DNA fragment lengths will be calculated.
' Use 0 if there are none.  Finally enter the migration distance for the
' unknowns.  # after a variable indicates double precision.
OPTION BASE 1
DIM X#(50), X1#(50), Y#(50) B#(4), B1#(4), BB#(4), W#(50), XTWX#(4,4)
DIM XTWXINV#(4,4), XTWR#(4), YHAT#(50), RESIDUAL#(50), XX#(50,4), RES#(50)
DIM RESIDUAL1#(50), A#(16), L(4), M(4)
DIM PROD(50), DWT(50), DDIST(50), DPROD(50), C(50), D(50)
INPUT "Enter proportion for B update -- usually 1.0";PROP#
AGAIN:
INPUT "Enter 1 for Robust fit;  2 for approximate least squares fit";FIT
IF FIT &lt;&gt; 1 AND FIT &lt;&gt; 2 THEN GOTO AGAIN
READ GRAPH# :  READ N1  ' Compute starting values for the iterative process
SWT=0 : SDIST=0 : SPROD=0
FOR I=1 TO N1 : READ X1#(I), Y#(I)
X#(I)=X1#(I)/GRAPH#
IF I=1 THEN TEMP1#=X#(1)
X#(I)=(((X#(I)-TEMP1#)+0.01#)*100#)) : SWT = SWT + Y#(I) : SDIST = SDIST + X#(I)
PROD(I) = Y#(I) * X#(I) : SPROD = SPROD + PROD(I)
' Initialize the weight vector W
W#(I) = 1#  : NEXT I
MWT = SWT/N1 :  MPROD = SPROD/N1
FOR I=1 TO N1 : DWT(I) = Y#(I) - MWT
DDIST(I) = X#(I) - MDIST : DPROD(I) = PROD(I) - MPROD : NEXT
CSSL =0 : CSSM =0 : CSCPML=0 : CSPMLM=0
FOR I=1 TO N1 : CSSL =CSSL + DWT(I)^2
CSSM=CSSM + DDIST(I)^2 : CSCPML =CSCPML + DDIST(I) * DWT(I)
CSPMLL = CSPMLL + DPROD(I) * DWT(I) : CSPMLM=CSPMLM + DPROD(I)*DDIST(I)
NEXT
DET = CSSL*CSSM-CSCPML^2 : MO=(CSSM*CSPMLL-CSCPML*CSPMLM)/DET
LO=(-CSCPML*CSPMLL + CSSL*CSPMLM)/DET
SC =0 : FOR I=1 TO N1 : C(I)=(Y#(I)-LO)*(X#(I)-MO) : SC=SC+(C(I)-C(1)) :NEXT
CBAR = SC/N1 + C(1)
' Compute starting values
B#(1)=LO : B#(2)=-CBAR/MO : B#(3)=-1/MO : B#(4)=1#
' Start the iteration process
I1=0
DO UNTIL SUMB# &lt;= .00001# OR I1&gt;100
I1=I1+1 : ? USING " ###";I1;
? USING "#########.#####";B#(1),B#(2),B#(3),B#(4)
' Calculate the matrix of partial derivatives evaluated at X and
' the current estimate of B.
SSQ1#=0# :  SSQ2#=0#
' Initialize summation variable used for checking convergence of solution
SUMB# = 0# : FOR II=1 TO N1
' Calculate predicted Ys (YHATs) and RESIDUALS
YHAT#(II) = B#(1) + (B#(2)/(1#+(B#(3)*(X#(II)^B#(4)))))
RESIDUAL#(II)=Y#(II)-YHAT#(II) :  RESIDUAL1#(II)=ABS(RESIDUAL#(II))
' Calculate the sums of squares of the residuals
SSQ1# = SSQ1# + RESIDUAL#(II)^2 * W#(II)
XX#(II,1)=1# :  XX#(II,2) = 1#/(1#+(B#(3)*X#(II)^B#(4)))
XX#(II,3)=(-1#)*B#(2)*X#(II)^B#(4)*XX#(II,2)^2
XX#(II,4)=XX#(II,3)*B#(3)*LOG(X#(II)) : NEXT
IF FIT=1 THEN Weight
' Initialize X'WX and X'W(Y-f(X)) and update BB
FOR J=1 TO 4 :  FOR K=J TO 4 : XTWX#(J,K)=0#
' Take advantage of the symmetry of the X'WX matrix
XTWX#(K,J)=0# : NEXT
' Initialize X'W[Y-f(X)]
XTWR#(J)=0#
' Update BB
BB#(J) = B#(J) : NEXT
FOR II=1 TO N1
' Calculate X'WX
FOR J=1 TO 4 : FOR K=J TO 4
XTWX#(J,K) = XTWX#(J,K) + W#(II) * XX(II,J) * XX#(II,K)
' Take advantage of the symmetry of the X'WX matrix
XTWX#(K,J)=XTWX#(J,K) : NEXT
' Calculate X'W[Y-f(X)]
XTWR#(J) = XTWR#(J) +
W#(II)*XX#(II,J)*RESIDUAL#(II) : NEXT : NEXT
' Compute INV[X'WX]
Invert
' Compute new Beta vector (B)
FOR I=1 TO 4
' Initialize the temporary B vector
B1#(I)=0# : FOR J=1 TO 4 : B1#(I)=B1#(I) + XTWXINV#(I,J) * XTWR#(J)
NEXT
' Finally update the last estimate of B with this new calculation.
' Make surethat the proportion of change, inputted by the user, is
' incorporated.
B#(I) = B#(I) + PROP# * B1#(I)
' Now compute the difference between the old B vector (BB) and the
' new B vector (B)
B1#(I)=B#(I)-BB#(I)
' Add up these differences to check for convergence later on
SUMB#=SUMB# + ABS(B1#(I)/BB#(I)) : NEXT
' Now compute a second sums of squares of residuals.  This
' statistic is what the least squares procedure is really
' minimizing: Y-f(X)-[B1(1)XX(1)+B1(2)XX(2)+B1(3)XX(3)+B1(4)XX(4)]
' where B1=B-BB (or the new estimate - the old estimate)
FOR I=1 TO N1
' Initialize the temporary variable RES.
RES#(I)=0# : FOR J=1 TO 4
' now compute [B1(1)XX(1)+B1(2)XX(2)+B1(3)XX(3)+B1(4)XX(4)]
RES#(I)=RES#(I)+B1#(J) * XX#(I,J) : NEXT
' Now compute the second modified sum of squares.
SSQ2#=SSQ2# + ((RESIDUAL#(I)-RES#(I))^2*W#(I) : NEXT
? "Regular and adjusted sums of squares = ";
? USING "##.#######^^^^"; SSQ1#,SSQ2# :  ? " "
' Check for convergence of former and present B estimate
LOOP
? " " : IF FIT=1 THEN ? "                         Results of Robust fit"; : ?
IF FIT=2 THEN ? "                       Results of approximate least squares fit"; : ?
? : ? : ? "Migr.      Rel.     Known       Pred."
?         "Dist.      Positn    M.W.        M.W.        Residual      % Error    Weight"
? : FOR I=1 TO N1 : ? USING "##.###"; X1#(I);
? USING "########.##"; X#(I); : ? USING "###########"; Y#(I);
? USING "###########.##"; YHAT#(I);
? USING "###########.##"; RESIDUAL#(I);
? USING "#######.##"; (ABS(RESIDUAL#(I))/Y#(I))*100#;
? USING "######.####"; W#(I) : NEXT
? : ? " The final coefficients are:" : ?
? USING
"###########.#####";BB#(1),BB#(2),BB#(3),BB#(4) : ? : ?
? "Regular and adjusted sums of squares = "; : ? USING "##.#######^^^^"; SSQ1#, SSQ2# : ?
' Compute molecular weights or DNA fragment lengths for unknowns.
READ N1
IF N1&lt;1 THEN END
? "Calculation of molecular weights or DNA fragment lengths"  : ?
? "Migr.       Rel.       Pred."
? "Dist.      Positn       M.W." : ?
FOR I=1 TO N1 : READ XUNKN# : XUNKN1# = XUNKN#/GRAPH#
XUNKN1# = ((XUNKN1# - TEMP1#) + .01#) * 100#
YUNKN# = BB#(1) + BB#(2) / (1# + (BB#(3) * XUNKN1#^BB#(4)))
? USING "###.###"; XUNKN#; : ? USING "#######.##";
XUNKN1#; : ? USING "##########.##"; YUNKN# : NEXT
END
DATA 10,13,1.204,12216,1.335,11198,1.497,10180,1.660,9162,1.888,8144
DATA 2.116,7126,2.454,6108,2.897,5090,3.483,4072,4.310,3054,5.664,2036
DATA 6.380,1635,7.891,1018,3,1.497,1.888,6.380

SUB Invert
' Subroutine transcribed from the IBM FORTRAN SSB routine MINV
'
' Method
'   The standard Gauss-Jordan method is used.  The determinant is also
'   calculated.  A determinant of zero indicates that the matrix is
'   singular.
'
N=4 : FOR I=1 TO N : FOR J=1 TO N : IJ=N*(J-1) + I
: A#(IJ)=XTWX#(I,J) : NEXT : NEXT
' Search for the largest element
D#=1# : NK=-N : FOR K=1 TO N : NK=NK+N : L(K)=K : M(K)=K : KK=NK+K
BIGA#=A#(KK) : FOR J=K TO N : IZ=N*(J-1) : FOR I=K TO N : IJ=IZ+I
IF ABS(BIGA#)&lt;ABS(A#(IJ)) THEN BIGA#=A#(IJ) : L(K)=I : M(K)=J
NEXT : NEXT
' Interchange rows
J =L(K) : IF J &lt;= K THEN CHANGECOL
KI=K-N : FOR I=1 TO N : KI = KI+N : HOLD# = -A#(KI) : JI=KI-K+J
A#(KI)=A#(JI) : A#(JI)=HOLD# : NEXT
CHANGECOL:
' Interchange columns
I=M(K) : IF I&lt;=K THEN WARN
JP=N*(I-1) : FOR J=1 TO N : JK=NK+J : JI=JP+J : HOLD#=-A#(JK)
A#(JK)=A#(JI) : A#(JI)=HOLD# : NEXT

WARN:
' Divide column by minus pivot (value of pivot element is contained in BIGA)
IF ABS(BIGA#) &lt;= .000000000000001# THEN D#=0# : ? "Determinant=0 -- the matrix is singular" : END
FOR I=1 TO N : IF I=K THEN NEXT
IK=NK+1 : A#(IK)=A#(IK)/(-BIGA#) : NEXT
' Reduce matrix
FOR I=1 TO N : IK=NK+I : HOLD#=A#(IK) : IJ=I-N
FOR J=1 TO N : IJ=IJ+N : IF I=K THEN NEXT J
IF J=K THEN NEXT I
KJ=IJ-I+K : A#(IJ)=HOLD#*A#(KJ)+A#(IJ) : NEXT : NEXT
' Divide row by pivot
KJ=K-N : FOR J=1 TO N : KJ=KJ+N : IF J=K THEN NEXT
A#(KJ)=A#(KJ)/BIGA# : NEXT
' Product of pivots
D#=D#*BIGA#
' Replace pivot by reciprocal
A#(KK)=1#/BIGA# : NEXT
' Final row and column interchange
K=N

CUT6:
K=K-1 : IF K&lt;=0 THEN GOTO VECTORTOMATRIX
I = L(K) : IF I &lt;= K THEN GOTO CUT5
JQ=N*(K-1) : JR=N*(I-1) : FOR J=1 TO N : JK=JQ+J : HOLD#=A#(JK)
JI=JR+J : A#(JK)=-A#(JI) : A#(JI)=HOLD# : NEXT

CUT5:
J=M(K) : IF J&lt;=K THEN GOTO CUT6
KI=K-N : FOR I=1 TO N : KI=KI+N : HOLD#=A#(KI) : JI=KI-K+J
A#(KI)=-A#(JI) : A#(JI)=HOLD# : NEXT : GOTO CUT6

VECTORTOMATRIX:
' Convert vector back to matrix array
FOR I=1 TO N : FOR J=1 TO N : IJ=N*(J-1) + I : XTWXINV#(I,J)=A#(IJ)
NEXT : NEXT
END SUB

SUB Weight
' This subroutine computes the weight vector for the robust fit
IE=N1 : J1=1 : FOR I5=1 TO N1
' Set the lower relative index of the array to be sorted.
IB=I5
Orders
NEXT
' Compute SD for those times N1 is even
IND =3 + INT((N1-2/2) : SD#=RESIDUAL1#(IND)
' If N1 is odd then use the average of the two middle values of the
' last N1-3 elements of RESIDUAL1
IF (N1 MOD 2) &lt;&gt; 0 THEN
SD#=(RESIDUAL1#(IND)+RESIDUAL1#(IND+1))/2#
' Now set up the weight vector
FOR I=1 TO N1 : W#(I)=0# : TEMP#=RESIDUAL#(I)/(SD#*2.1#)
IF (-3.141592654# &lt;= TEMP#) AND (TEMP# &lt;= 3.141592654#) THEN W#(I)=SD#*2.1#*SIN(TEMP#)/RESIDUAL#(I)
NEXT : RETURN
SUB Orders
' This subroutine finds the i'th order statistic from an array of
' numbers.  RESIDUAL1 is the array;  J1 is the order statistic
' desired;  IB is the lower index of the array to be examined;
' IE is the upper index of the array to be examined.
JO = IB + J1-1 :  NN = IE +1 : IJ = IB - 1 : K = NN : I = IJ

CUT0:
T# = RESIDUAL1#(JO)

CUT1:
K =K-1 : W1# = RESIDUAL1#(K)
IF W1#&gt;T# THEN GOTO CUT1
IF W1#&lt;T# THEN GOTO CUT2
IF K &lt;&gt; JO THEN GOTO CUT1
IF I=K THEN RETURN
CUT2:
I=I+1 : Z# = RESIDUAL1#(I)
IF Z#&lt;T# THEN CUT2
IF Z#&gt;T# THEN CUT3
IF I &lt;&gt; JO THEN CUT2
IF I=K THEN RETURN
RESIDUAL1#(I) = W1# : RESIDUAL1# = Z# : I =IJ : NN=K : GOTO CUT0
CUT3:
IF I=K THEN RETURN
RESIDUAL1#(I)=W1# : RESIDUAL1#(K)=Z# : IF K &lt;&gt; JO THEN GOTO CUT1
K = NN : IJ =I : GOTO CUT0
</pre>
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Domen, J., van Leen, R. W., Lubsen, N. H., Shoenmakers, J. G. G.
 </span>
 <span class="year">
(1986)
 </span>
 <span class="jname">
Anal. Biochem.
 </span>
 <span class="jvol">
155,
 </span>
 <span class="pages">
379-384.
 </span>
<p class="reftitl">
A vital staining method for measuring the efficiency of transfection of eukaryotic cells
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Carney, S. L., Bayliss, M. T., Collier, J. M., Muir, H.
 </span>
 <span class="year">
(1986)
 </span>
 <span class="jname">
Anal. Biochem.
 </span>
 <span class="jvol">
156,
 </span>
 <span class="pages">
38-44.
 </span>
<p class="reftitl">
Electrophoresis of <sup>
35</sup>S-labeled proteoglycans on polyacrylamide-agarose
composite gels and their visualization by fluorography
<p class="Para1">
Methods.  Gels are cast in a 140 &times; 140 &times; 1.5 mm
apparatus.  A 10% polyacrylamide plug about 5 cm long must be cast
first since the composite gel will not hold within the apparatus.
Gel plates are pre-heated in a 40-50&deg; oven prior to pouring the gel.
The 1.2:0.6% polyacrylamide:agarose gel is prepared by the method of
McDevitt and Muir (1971):  0.24 g agarose is prepared in 40 mM
Tris-acetate, pH 6.8, 1 mM Na<sub>2</sub>SO<sub>4</sub> with final 22.5
ml and heated with constant stirring to 90&deg;.  During cooling, 0.456 g
acrylamide and 0.024 g bisacrylamide is dissolved in 9.7 ml gel buffer,
and 4.8 ml of 6.4% (v/v) -dimethylaminopropionitrile is added.  This
second solution is warmed on the heater next to the agarose.  When
the agarose solution is cooled to 52&deg;, 3 ml of 0.3% ammonium persulfate is
added to the acrylamide.  The agarose and acrylamide solutions are
rapidly mixed and poured into the heated gel apparatus.  An 8-place
comb is added to a depth of 5 mm.  Water-saturated n-butanol is added
to the top to provide exclusion of air.  The gel slab is left at 4&deg;
for 1 h.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Rivera-Bustamante, R. F., Gin, R., Semanick, J. S.
 </span>
 <span class="year">
(1986)
 </span>
 <span class="jname">
Anal. Biochem.
 </span>
 <span class="jvol">
156,
 </span>
 <span class="pages">
91-95.
 </span>
<p class="reftitl">
Enhanced resolution of circular and linear molecular forms of viroid
and viroid-like RNA by electrophoresis in a discontinuous-pH system
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Schecter, Y.
 </span>
 <span class="year">
(1986)
 </span>
 <span class="jname">
J. Biol. Chem.
 </span>
 <span class="jvol">
261,
 </span>
 <span class="pages">
66-70.
 </span>
<p class="reftitl">
Selective oxidation and reduction of methionine residues in peptides
and proteins by oxygen exchange between sulfoxide and sulfide
<p class="Para1">
<b>INTRO.</b>  Chloramine-T and N-chlorosuccinimide (NCS) at neutral
or slightly alkaline pH have been shown to cause the oxidation of
methionine to its sulfoxide in proteins with no -SH groups, and
they will only oxidize methionine.  Periodate, hydrazoic acid,
iodination, photooxidation, mild brominating agents, and hydrogen
peroxide were less selective, modifying other oxidizable groups.
Chloramine-T is 4- to 5-times more reactive to cysteinyl -SH than
to methionine, and in acidic mixtures, will also oxidize tryptophanyl
residues.  This study explores the chemical conversions of Met as
an amino acid and component of protein.
<p>
<b>RESULTS</b>.  Methionine conversion to its sulfoxide by Me<b>2</b>SO
(DMSO) under various conditions was examined.  0.2 mM Met in the presence
of 0.2 M DMSO for 2 h at room temp with different catalytic environments.
The tested environments are water, sodium bicarbonate at 0.1 M, pH 8.5,
HCl at 0.01, 0.1, and 1 M, and acetic acid at 1 and 5 M.  There was no
conversion in water, alkaline sodium bicarbonate, or in 1 M acetic acid.
There was only slight conversion with 5 M acetic acid (20%) and
0.01 M HCl (8%), but even more dramatic conversion with increased
acidity: 70% with 0.1 M HCl and complete (100%) with 1 M HCl.
Hence strong acidity coupled with a mechanism mediated by
chloride-anion oxidation is the catalyst for conversion.
This is suggested by successful results with other chlorinating
compounds, Chloramine-T and NCS.
<p>
A mixture of amino acids (D,T,S,E,P,G,A,half-cystine,V,M,I,L,Y,F,K,H,R,
cysteic acid,M:  0.5 mM) was made 2 M and 1 M with respect to HCl and
DMSO, resp.  Reaction was done for 3 hr at 25&deg; and recovery of each
amino acid measured.  There was virtual 100% recovery for all amino
acids intact, even cysteine, except for cysteic acid and methionine,
where its sulfoxide was recovered 100% (0% sulfone form).  Monitoring
the UV spectrum of N-acetyltryptophan during the reaction helped
determine whether Trp is truly modified:  after 2 h under the same
conditions, there is no change in spectral character.  When
chloramine-T is added to the quartz cell, the peak at 280 nm falls
rapidly indicating conversion of the tryptophanyl side chain to
2-oxyindolealanine.  As to temperature effects, others have shown
the DMSO/HCl conditions are selective for Met even at elevated
temperatures (at 50&deg; and 100&deg;).
<p>
In a further measure as to what effect there is on the free -SH
groups, 3.6 mM reduced glutathione was incubated several hours
in water, 10% N,N-dimethylformamide, 1.2 M HCl or at two different
concentrations of DMSO/HCl.  Only after 2 h does oxidation of
glutathione by the DMSO/HCl medium occur (already 100% complete
with Met).  There was no observed oxidation in the other media.
<p>
DMSO was added in large excess (25-250 M) to methionine in 1 M HCl.
The oxidation was pseudo-first order with a rate of 0.23 &plusmn; 0.01
M<sup>&minus;1</sup> s<sup>&minus;1</sup> at 22&deg;.  Quantitative
conversion occurred within 30 min.  Even at 10 M excess was
quantitative conversion observed.  At 4 M excess, conversion was 50%.
<p>
Oxidation of Met in di- and tripeptides and in proteins by
DMSO/HCl was examined.  In all cases (Met, Met-Val, Met-Asp,
Met-Phe, Met-Phe-Gly, the proteins ACTH, glucagon, &alpha;-lactalbumin)
the Met residues were 100% converted to the sulfoxide.  Only in bovine
pancreatic ribonuclease was conversion &lt;100% (25%), but since only
one Met in four is oxidized by other oxidizing agents, it only
demonstrates that the DMSO/HCl can not penetrate buried Met
residues any better than the other agents.  The &alpha;-lactalbumin
protein was reduced to break its four disulfide bonds:  in the
DMSO/HCl oxidizing system, only Met was converted and the free
-SH groups did not change.
<p>
Reduction of Met sulfoxide by Me2S (DMS) was examined as well:
the reaction appeared to require high concentrations of HCl
in order to occur.  With 0.5 M DMS and 89 &micro;M MetSO present
in all reactions, with 1.0 M HCl, only 10% conversion to Met
is seen after 40 min.  With 4.4 M HCl, 50% conversion occurs
after 10 min and appears complete after 30 min.  With 10.7 M
HCl, reaction is very rapid:  50% by 5 min and 100% done in 16 min.
The same pattern of reduction of MetSO in alpha-lactalbumin is
observed as it was for free MetSO.
<p>
When the Met of insulin was oxidized to MetSO, there was no
irreversible denaturation of the hormone and no change in
its biological activity.  On the other hand oxidation of
Met in ACTH causes it to retain only 4% of the native activity.
When the oxidized derivative is exposed to 10.3 M HCl/0.3 M DMS
for 15 min at 37&deg;, nearly full reactivation of biological
properties is observed.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Freier SM, Kierzek R, Jaeger JA, Sugimoto N, Caruthers MH, Neilson T,
Turner DH
 </span>
 <span class="year">
(1986)
 </span>
 <span class="jname">
Proc. Natl. Acad. Sci. USA
 </span>
 <span class="jvol">
83,
 </span>
 <span class="pages">
9373-9377
 </span>
<p class="reftitl">
Improved Free-Energy Parameters for Predictions of RNA Duplex Stability
<p class="Para1">
Stability of RNA duplexes and secondary structure is predicted often by
free-energy parameters employing nearest-neighbor model, but predictions
are inconsistent with experimental data.  To close the gap on this
45 complementary model oligonucleotides were studied for their melting
temperatures.  Oligos were synthesized by phosphoramidite chemistry and
purified by HPLC.  Absorbance/melting temp curves were done in 1 M NaCl
in 5 mM sodium phosphate at pH 7 with 0.5 mM EDTA present.  To arrive
thermodynamic parameters, the melting curves were fit to a two-state
model with sloping baselines and enthalpy and entropy changes which were
derived from the fits were averaged.  Also the reciprocal melting temp,
<i>t</i><sub>m</sub><sup>&minus;1</sup>, was plotted against the
log (<i>C</i><sub>T</sub>) to calculate enthalpy and entropy changes.
The enthalpy and free energy changes for helix initiaton and propagation
were found by multiple linear regression for assessing nearest-neighbor
interactions.
<p>
Table 1 shows the thermodynamic parameters determined for several
RNA oligonucleotides both from this work and previous studies.  The
measured and predicted <i>t</i><sub>m</sub> for the oligos is given
to show how closely the model works.  The free energy is a calculated
parameter:  &Delta;<i>G</i>&deg;<sub>37</sub> = &Delta;<i>H</i>&deg;
&minus; <i>t</i>&Delta;<i>S</i>&deg; = &Delta;<i>H</i>&deg; &minus;
310.15 &times; &Delta;<i>S</i>&deg;.  From the nearest neighbor
model, helix formation standard-state free energy is the sum of three
terms:  (i) a free-energy change for helix initiation associated with
forming the first base pair in the duplex, (ii) the sum of the free
energies in the propagation of the helix, (iii) a correction for symmetry
if the sequence is self-complementary.
<p>
[continued]
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Iverius P-H, &Ouml;stlund-Lindqvist A-M
 </span>
 <span class="year">
(1986)
 </span>
 <span class="jname">
Meth. Enzymol.
 </span>
 <span class="jvol">
129,
 </span>
 <span class="pages">
691-695.
</span>
<p class="reftitl">
Preparation, Characterization, and Measurement of Lipoprotein Lipase
<p class="Para1">
Lipoprotein lipase (LPL) requires a protein cofactor, apolipoprotein C-II,
to hydrolyze a triglyceride emulsion at an optimal rate.  ApoC-II is a normal
constituent of chylomicrons and VLDL.  Assays should include serum as a
source of apoC-II and albumin, which binds to the free fatty acids that
form in the product.  The assay Intralipid (Vitrum AB) are quite reproducible
but lack the sensitivity of radioassays, although the latter has reproducibility
problems when new emulsions are prepared in each assay.
<p>
The assay uses:
<ol>
<li><b>phosphatidylcholine</b> from egg, 50 mg/ml, prepared as chloroform
solution (Serdary Res. Lab., Ontario, Cat #A-31)
<li><i>triolein</i>, 200 mg/ml in benzene (Sigma T7502)
<li><b>glycerol-tri[1-<sup>14</sup>C]oleate</b> (radiolabeled triolein),
100 &micro;Ci/ml (30-60 mCi/mmol) in toluene.  125 &micro;Ci aliquots are
evaporated under nitrogen, redissolved in 3 ml heptane, further purified
by extraction with 7 ml of 50 mM NaOH in 50% (v/v) EtOH.  Purified material
evapd under nitrogen, dissolved in 100 ml heptane, stored at &minus;20&deg;.
<li><b>[1-<sup>14</sup>C]oleic acid</b> (radiolabeled oleic acid), 100 &micro;Ci/ml
(&gt; 50 mCi/mmol) in toluene.  50 &micro;Ci is evapd under nitrogen, mixed
with 50 &micro;mol (14.124 mg) of unlabeled oleic acid, diluted with 100 ml
heptane and stored &minus;20&deg;.
<li><b>0.223 M TrisHCl, pH 8.5</b>: place 29.289 g of Trizma-8.5 (Sigma) in
1.0 liter water and store 4&deg;.
<li><b>0.78 M NaCl in Tris</b>:  4.558 g in 100 ml of the above Tris buffer;
used for salt correction.
<li><b>methanol-chloroform-heptane</b>, 1.41:1.25:1 (v/v/v).  141 ml MeOH with
125 ml chloroform, then add 100 ml heptane.
<li><b>50 mM carbonate-borate, pH 10.5</b>.  Dissolve 6.91 g anhydrous
K<sub>2</sub>CO<sub>3</sub> and 3.092 g boric acid, adjusted to pH 10.5 with
5 M KOH, then brought to 1.0 L.  Should be made weekly, stored 4&deg;.
<li><b>albumin, 183.3 mg/ml in Tris</b>:  45.825 g (Sigma A4503) dissolved
in the Tris buffer made above, diluted to 250 ml, stored in 10 ml aliquots
at &minus;20&deg;.
<li><b>heparin, 1 mg/ml in Tris or Krebs-Ringers phosphate (KRP)</b>:  50 mg
sodium heparin (Sigma H3125) dissolved in 50 ml Tris or KRP, stored in 2 ml
aliquotes at &minus;20&deg;.
<li><b>pooled serum</b>:  whole blood is collected in EDTA disodium (20 mg/ml),
plasma collected and re-centrifuged prior to pooling.  Solid calcium chloride
is added to 0.1% and then 1 U/ml thrombin and plasma incubated at 37&deg; for
30 min.  Serum is recovered in a spin, dialyzed against 0.15 M NaCl and
heat-inactivated at 56&deg; for 30 min.  Serum is dialyzed agaain againtst
KRP and stored &minus;20&deg; in 10 ml aliquots.
<li><b>2 mg/ml deoxycholate + 80 &micro;g/ml Nonidet P-40 + 50 &micro;g/ml
heparin + 10 mg/ml BSA + 0.25 M sucrose in Tris buffer</b>.  Mix 500 mg
sodium deoxycholate and 21.394 g sucrose with 2 ml of 1% Nonidet P40, 12.5 ml
of 1 mg/ml heparin, and 13.7 ml albumin (?? mg/ml).  Use Tris buffer (above)
for all component solutions and for dilution to 250 ml after warming to 37&deg;.
Store in 5 ml aliquots at &minus;20&deg;.
<li><b>skim milk standard</b>.  The cream of unpasteurized milk fresh from
dairy farm is removed by centrifugation at 4&deg;.  500 ml of the skim milk
is stirred 30 min 4&deg; with 14.7 g trisodium citrate dihydrate (makes 0.1 M),
dialyzed against 0.15 M NaCl-5 mM sodium phosphate pH 7.4 for 6 h, three times,
in buffer which is 10 volumes more.  Glycerol is added to 30% (v/v) after
dialysis and standard frozen in 4 ml aliquots with EtOH-dry ice, stored at
70&deg;.
<li><b>Krebs-Ringer phosphate (KRP)</b>.  Prepare fresh:  100 ml of 45 g/l
NaCl, 5 ml of 57.4 g/l KCl, 1.5 ml of 81.4 g/l CaCl<sub>2</sub>&bull;2
H<sub>2</sub>O, 1.0 ml of 191.0 g/l MgSO<sub>4</sub>&bull;7 H<sub>2</sub>O,
432 ml distilled water, 15 ml of 13.80 g/l NaH<sub>2</sub>PO<sub>4</sub>
&bull; H<sub>2</sub>O, and 85 ml of 14.196 g/l anhydrous
Na<sub>2</sub>HPO<sub>4</sub>.
<li><b>Elution buffer</b>.  Prepare fresh:  5 vol serum, 1 vol of 1 mg/ml
heparin in KRP buffer, 14 vol of KRP buffer.  This is 25% (v/v) serum and
50 &micro;g/ml heparin.
<li><b>Diluted serum</b>.  Prepare fresh: 1 vol serum, 3 vol KRP buffer.
This is 25% serum.
<li><b>Triglyceride emulsion</b>. Prepare fresh:  Mix in a 2.2 &times; 7 cm
flat-bottom glass vial:  120 &micro;l of containing 6 mg phosphatidylcholine,
250 &micro;l containing 50 mg triolein, and 2 ml containing 2.5 &micro;Ci
radiolabeled triolein.  Evaporate with stream of nitrogen.  Add 1.95 ml
Tris buffer, 1 ml of salt correction solution, and 3 ml albumin.  Cool on
ice, sonicate at 50W for 10 sec bursts 10 times with 10 sec pauses using
0.5 inch probe with flat tip.  Keep emulsion at 0&deg; and use within 2 h.
The 6 ml batch will do 10 assays; if more are required, make several batches
and pool.
</ol>
<p>
Prepare adipose tissue (~50 mg) through needle aspirates, cut into 5-8 mg
pieces, transfer to 12 &times; 75 mm glass tube with 300 &micro;l elution
buffer, cover with parafilm, incubate in 37&deg; shaking water bath for 30
min at 80 cycles/min.  Take an aliquot of 200 &micro;l at end and assay
for activity.
<p>
Thaw out stock detergent solution at 37&deg; and homogenize
~50 mg adipose tissue (weighed) at room temp with 200 &micro;l of
detergent solution in all-glass Kontes Duall size 20 tissue grinder.
Add another 300 &micro;l solution and mix briefly, transfer all to centrifuge
tubes and spin 12,000 <i>g</i> 15 min at 4&deg;.  Assay 0.2 ml of infranatant
for enzyme activity.
<p>
With postheparin (collected in lithium heparinate vacutainers) recover
plasma, freeze in EtOH-dry ice and store &minus;70&deg;.  On day of analysis
thaw rapidly in cold water, get 50 &micro;l and dilute in 200 &micro;l
pooled serum, 50 &micro;l of 1 mg/ml heparin in KRP buffer, and
200 &micro;l KRP buffer.  Take 150 &micro;l of the mixture and add equal
volume of any inhibiting antiserum (anti-LPL antibody), already diluted with
KRP buffer.  A control sample is prepared with dilute nonimmune serum or IgG.
Mixtures are incubated 2 h at 4&deg; before an aliquot of 200 &micro;l is
assayed for lipase activity.
<p>
Lipase activity in cell culture medium is assayed without dilution.
As for cells, rinse a layer in a 35 mm dish twice with 1 ml saline,
carefully remove 2nd rinse, lyse by scraping with PTFE policeman.
Continue extraction as for adipose tissue above.
<p>
Thaw frozen skim milk standard in cold water and aliquot 100 &micro;l
to test tube with 2 ml ice-cold Tris buffer.
<p>
<b>PROCEDURE</b>.  The assay mixture is 0.178 M TrisHCl, 0.11 M NaCl,
55 mg/ml albumin, 10 &micro;g/ml heparin, 5% serum, and 5 mg/ml
glycerol-tri[1-<sup>14</sup>C]oleate (50 nCi/mg) emulsified in
0.6 mg/ml lecithin.  The solution has an ionic strength of 0.16,
pH 8.2, and is incubated at 37&deg;.  The blank is 0.6 ml triglyceride
emulsion, 0.2 ml Tris buffer, and 0.2 ml elution buffer.
<p>
Add 0.2 ml aliquots of tissue eluates or postheparin plasma with
0.6 ml triglyceride emulsion and 0.2 ml Tris buffer.  Aliquots of the
samples in the Tris buffer (detergent extracts, skim milk standard,
miscellaneous samples) are mixed with 0.6 ml triglyceride emulsion
and 0.2 ml diluted serum.
<p>
The test tubes now contain 0.8 ml of the incomplete assay mixture
on ice. 0.2 ml of the enzyme is added, covered in parafilm, incubated
shaking in 37&deg; water bath at 80 cycles/min for 1 h.
<p>
While the incubation is occurring, prepare 3.25 ml of MeOH-chloroform-heptane
in 13 &times; 100 tubes for each tube in the shaker.  Take four 0.2 ml
aliquots from each incubation mixture to extraction tubes and agitate
vigorously.  Add 1.05 ml carbonate-borate buffer, stopper the tubes again
and agitate.  Centrifuge the tubes to obtain phase separation for 20 min
at room temp, using swingout rotor.  Take a 2 ml aliquot from upper phase
and count in 4 ml scintillation fluid (Aquasol) for 10 min.
<p>
For determining the specfic radioactivity of triolein, count 20 &micro;l
in duplicate in empty scintillation vials after drying under nitrogen
stream.  Before counting, add 2 ml of a blank extraction to 4 ml
scintillation fluid.
<p>
To determine extraction recovery, add 200 &micro;l radioactive oleic acid
is extracted in test tubes and also added directly to scintillation vials.
<p>
In general blank extractions are run in triplicate and one replicate is
used for triolein specific radioactivity determination and extraction
recovery.  Hydrolysis products of radioactive triolein should be removed
during extraction to avoid low background radioactivity of the blank.
<p>
<b>Calculations</b>.  Compute the means of all replicates.  Now compute
the net radioactivity by subtracting the mean of the blanks from the
mean of samples/standards/controls.  The recovery of extraction <i>R</i>
is:  <i>R</i> = 5<i>E/F</i> where <i>R</i> is fraction of oleic acid
distributing to 2 ml of aqueous phase, <i>F</i> is the total oleic acid
radioactivity (cpm) added to separate scintillation vial or to blank incubation
(1 ml) and <i>E</i> is the radioactivity extracted from a 0.2 ml aliquot into
2 ml of aqueous phase.  The specific radioactivity, <i>S</i>, is:
<i>S</i> = (10 &times; 885.43 &times; <i>T</i>) / (5 &times; 10<sup>&minus;3</sup>)
(3 &times; 10<sup>&minus;9</sup>) = 5.903 &times; 10<sup>&minus;4</sup> <i>T</i>,
and <i>T</i> is the radioactivity of triolein added to scintillation vial,
and <i>T</i> &times; 10 is the amount in a 1 ml complete assay.  The other
factors convert the value to fatty acid equivalents in nanomoles given a 5 mg/ml
solution.  The enzyme activity <i>A</i> is now calculated as:
<p>
<i>A</i> = <i>C</i> / (<i>R</i> &times; <i>S</i> &times; 0.2 &times; 0.2 &times;
60 = <i>C</i> / <i>R</i> &times; <i>S</i> &times; 2.4
<p>
expressed in nanomoles of fatty acid released per minute and per milliliter sample.
To deal with interassay variation multiply by an apparent enzyme activity
correction factor (<i>f</i>) = A<sub>0</sub>/A<sub>1</sub>, where A<sub>1</sub>
is apparent activity of skim milk standard in an assay and A<sub>0</sub> is
arbitrarily chosen reference value for the standard.  Multiply by 20 to correct
for the postheparin plasma lipolytic activity.  Any activity remaining after
incubation with anti-LPL antibody is hepatic lipase.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Rosenheck, K., Plattner, H.
 </span>
 <span class="year">
(1986)
 </span>
 <span class="jname">
Biochim. Biophys. Acta 
 </span>
 <span class="jvol">
856,
 </span>
 <span class="pages">
373-382.
</span>
<p class="reftitl">
Ultrastructural and cytochemical characterization of adrenal medullary plasma 
membrane vesicles and their interaction with chromaffin granules
<p class="Para1">
Medullae were removed from fresh adrenal glands, finely cut and homogenized in 
0.3 M sucrose (pH 7.0 Tris-buffered) in a Duall glass-to-glass tissue grinder. 
First a low- then a high-speed spin was done to obtain membrane-rich pellets. 
Resuspension in buffer then layering over a 1.4 M sucrose buffer was followed by 
an ultra-speed spin in an SW 28 rotor.  Marker proteins verified the specific 
content of the plasma membrane vesicle enrichment.  In this preparation 
fragments of mitochondria, secretory granules and other structures could not be 
seen using EM (ultrathin sections or negatively stained preps);  the 
acetylcholinesterase assay which is viewed under EM was shown reactive; and the 
appearance of freeze fracture micrographs indicated a homogeneity distinct from 
that of chromaffin granules or membranes from such granules.  The size 
distribution of the vesicles was also estimated based upon the analysis of 
freeze-fracture replicas:  over half were vesicles of less than 0.15 &micro;m in 
diameter, and a third were between 0.15 and 0.25 &micro;m.  The number of layers 
(lipid layers) was examined by all techniques (freeze-fracture, ultra-thin 
sectioning, and negative staining) and this showed that vesicles had only one 
shell for the most part;  multiple membrane layers occurred and some plasma 
membrane vesicles showed smaller vesicles inside.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Hagag, N., Halegoua, S., Viola, M.
 </span>
 <span class="year">
(1986)
 </span>
 <span class="jname">
Nature 
 </span>
 <span class="jvol">
319,
 </span>
 <span class="pages">
680-682.
</span>
<p class="reftitl">
Inhibition of growth factor-induced differentiation of PC12 cells by 
microinjection of antibody to <i>ras</i> p21
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Blenis, J., Erikson, R. L.
 </span>
 <span class="year">
(1986)
 </span>
 <span class="jname">
EMBO J.
 </span>
 <span class="jvol">
5,
 </span>
 <span class="pages">
3441-3447.
</span>
<p class="reftitl">
Regulation of protein kinase activities in PC12 pheochromocytoma cells
<!-- ================================================================ -->
<p class="tags">

<p class="ref">
  <span class="authors">
   Puzas JE, Brand JS
  </span>
  <span class="year">
   (1986)
  </span>
  <span class="jname">
   Calcif. Tissue Int.
  </span>
  <span class="jvol">
   39,
  </span>
  <span class="pages">
   104-108,
  </span>
<p class="reftitl">
  The Effect of Bone Cell Stimulatory Factors Can Be Measured with
  Thymidine Incorporatin Only Under Specific Conditions
<p class="Para1">
Pool of dTTP precursor within cultured cells depends on endogenous
and exogenous source and proportionate contribution of each.
Various levels of thymidine were tested.  [<sup>3</sup>H]thymidine
was added to culture vessels to 2.0 &mu;Ci/ml; specific radioactivity
would be adjusted with unlabeled thymidine.  The first experiment
increased exogenous thymidine until apparent rate of incorporation
was saturated (maxed).  Eadie-Hofstee (V vs. V/S) plots were done
in which V was pmol thymidine incorporated per culture per total
time period (2 h); S is the exogenous substrate concentration in
&mu;M.  A control set of bone cells and those treated with a
heterotopic bone extract were tested.  Both control and treated
cells showed the same K<sub>m</sub>, indicating that the nature
of uptake and incorporation is the same.  The V<sub>max</sub> was
higher in treated cells by 7-fold:  this means that a larger number
of cells of total cells were participating in DNA synthesis, a result
of the treatment.  A substrate concentration at 0.9 V<sub>max</sub>
was selected for later experiments;  this meant using 5.0 &mu;M
thymidine.
<p>
At 1 day (24 h) intervals up to 4 d (96 h), thymidine incorporation
was measured with cultures containing carrier-free (40 nM) thymidine
and 5 &mu;M thymidine.  Total DNA in each culture (measured by
a diaminobenzoic acid fluorometric assay) and pmol dTTP were
measured, including the net changes over 24 h from each time point.
Rat DNA is 29.5% thymine (dTMP, MW=304 pg/pmol), and this can be
used to calculate total DNA against the measured.  Student's t test
can then be used to see if the calculated and measured values are
different.  For the 5.0 &mu;M thymidine, calculations correlate
with measurements.  With carrier-free thymidine however, there
was a clear difference (P &le; 0.025).
<p>
Surprisingly few cells participate in DNA synthesis during an
experiment.  The following equation determines this:
<p class="eqnserif">
Fraction of cells in DNA synthesis = (EOP &divide; SOP)<sup>(1/<i>n</i> &minus; 1)</sup>
<!-- ================================================================ -->
<p class="tags">

<p class="ref" id="sottrup-jensen-1986">
 <span class="authors">
Sottrup-Jensen L, Gliemann J, Van Leuven F
 </span>
 <span class="year">
(1986)
 </span>
 <span class="jname">
FEBS Lett.
 </span>
 <span class="jvol">
39,
 </span>
 <span class="pages">
104-108,
 </span>
<p class="reftitl">
Domain structure of human &alpha;<sub>2</sub>-macroglobulin:  Characterization 
of a receptor-binding domain obtained by digestion with papain
<p class="Para1">
Complex of proteinase with &alpha;<sub>2</sub> macroglobulin (a2M) is initiated
by proteolysis in the cleavage region (residues 681-686) which traps proteinase 
and causes formation of a2M dimers from its tetramer.  This is followed by 
exposure of recognition sites that clear a2M from serum via receptor-mediated 
endocytosis. Methylamine-treated a2M is also cleared.  Digestion with 
<i>Lysobacter enzymogenes</i> at pH 4.5 localizes a 138-residue C-terminal 
domain which has epitopes of neoantigens appearing after thiol ester cleavage, 
which can be obtained with pH 4.5 papain digestion.  Human a2M was prepared by 
Sottrup-Jensen et al (1983) and treated with methylamine or iodoacetamide 
(Sottrup-Jensen et al 1980).
<p>
MA-treated a2M (MA-a2M) normally resists proteinase attack, but a 2-4 fold 
excess (mol/mol) of trypsin, chymotrypsin, elastase and papain will digest a2M. 
Papain-digested a2M was analyzed on 5 &times; 107 cm  Sepharcyl S-200 
equilibrated in 100 mM ammonium bicarbonate pH 8.2 at 100 ml/h flow and 10 ml 
fractions collected.  OD 280 measurements show to major peaks: a significantly 
absorbing peak eluting early between fractions 82 and 120 (between 0.39 and 0.57 
column volumes) and a small peak (pool 2) between fractions 150 and 162 (between 
0.71 and 0.77 col vols).  These were analyzed on SDS-PAGE and pool 2 was shown 
to consist of a 17 and 20 kDa fragment, while pool 1 were several fragments all 
interlinked by disulfides.  Digestion with papain (1:80 w/w) at pH 4.5 for 4-6 h 
releases these fragments completely.
<p>
Sequencing of the 95% pure small fragment produced the sequence EEFPFALGVNTLP. 
This indicates that specific cleavage at Lys1313-Glu occurs.  Amino acid 
analysis results [Asx 13.2 (14); Thr 7.4 (9); Ser 13.6 (15); Glx 13.7 (13); Pro 
8.4 (8); Gly 3.9 (4); Ala 9.5 (9); Val 12.6 (15); Cys 1.4 (2); Met 2.3 (3); Ile 
5.1 (6); Leu 12.7 (12); Tyr 5.9 (6); Phe 7.4 (7); Lys 8.6 (8); His 2.6 (3); Arg 
4.1 (4)] are consistent with the code between residues 1314-1451, which has 
computed mass of 15331; this is about 18 kDa with carbohydrate at Asn1401.  In 
SDS-PAGE, this shows an apparent size of 20 kDa.
<p>
Large and small fragments sets were incubated with rat hepatocytes at 4&deg; in 
competition with radioiodinated a2M-trypsin complex. (400,000 cells in 250 
&micro;l are incubated with 30 pM <sup>125</sup>I-a2M-trypsin [5000 cpm/tube] or 
unlabeled ligand for about 20 h; 4 expts were done.) In plots of bound (as 
percent of control vs. [a2M-trypsin] or [fragments], four plots of a2M-trypsin, 
18 kDa fragment, large fragment, and 18 kDa fragment + 6 M guandinium HCl.  The 
18 kDa fragment clearly competed with radiolabel complex, with an affinity 0.1% 
of that a2M-trypsin (<i>K</i><sub>d</sub> ~ 0.3 &micro;M vs 0.2 nM).  Different 
pH values (9.0, 3.5, 2.5) for incubation had no effect.  Digestion with pepsin 
at pH 2.5 (100:1 mol/mol) for 1 h at RT also had no effect; sequence analysis 
showed fragment was still intact.  Incubation with 6 M Guan HCl for 20 h or at 
pH 12.0 for 15 min showed the 18 kDa fragment retained 10% of its activity.  If 
Lys residues were succinylated or carboxylates modified by EDC, there was &lt;1% 
retained activity.  1,2-cyclohexanedione modification of Arg did not affect 
binding.
<p>
The weak binding of the 18 kDa fragment compared to a2M-trypsin complex may be 
the result that the dimer of C-terminal domain is better than a monomer. The a2M 
receptor-binding domain is composed of alternating &beta;-sheets and reverse 
turns, probably to form a &beta;-barrel.  This is stable in a pH range from 
2.5-9.0 and quite tight (inaccessible to protease digestion).  Disulfide 
reduction and 50% formic acid causes loss of binding ability: hence folding is 
necessary for receptor interaction.  Certain residues are key however, such as 
Lys, Glu and Asp.
<p>
<b>Methods</b>.  770 mg MA-a2M in 100 ml of 100 mM sodium acetate pH 4.5 was 
mixed with 12 mg papain dissolved in 6 ml acetate containing 5 mM cysteine pH 
5.0 and left room temperature for 20 h (this is a 64:1 w/w or 1:0.3 mol/mol). 
Tris base is added at end of digest to raise pH to 8.0 and 1 ml 0.5 M 
iodoacetamide to inactivate papain.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Biek DP, Cohen SN
 </span>
 <span class="year">
(1986)
 </span>
 <span class="jname">
J. Bacteriol.
 </span>
 <span class="jvol">
167,
 </span>
 <span class="pages">
594-603.
 </span>
<p class="reftitl">
Identification and Characterization of <i>recD</i>, a Gene Affecting Plasmid 
Maintenance and Recombination in <i>Escherichia coli</i>
<p class="Para1">
To maintain extrachromosomal elements (e.g. plasmids) the genetic material must 
have a means to replicate itself.  Plasmid pSC101 requires the <i>E coli 
dnaA</i> gene product, and mutations in other genes involving replication 
(<i>dnaB, dnaC,</i> and <i>dnaG</i>) also affect replication.  A locus on the 
plasmid <i>par</i> is necessary to partition the plasmid for distributing these 
to daughter cells. <i>par</i> does not encode protein [my note: but probably 
binds to proteins on the plate that forms to create daughter cells and which may 
bind two plasmids, putting one on either side].  In this paper the pSC101
was mutated and characterized for its stability:  that is, to be able to 
propagate into all daughter cells.  This is called the Pma (plasmid maintenance)
phenotype.
<p>
The transposable element Tn<i>10</i> was used as mutagen.  A defective &lambda; 
phase containing Tn<i>10</i> (&lambda;::Tn<i>10</i>) was used to infect <i>E 
coli</i> strain DPB6, which holds pZC1, derived from pSC101 and containing 
Km<sup>r</sup> and the <i>lacI</i> repressor gene.  The host chromosome has no 
<i>lacI</I>, so all repressor must come from plasmid.  If the plasmid is 
present, colonies are white on Xgal plates, since the <i>lacZYA</i> operon 
transcription is inhbitied.  If the plasmid becomes lost, then colonies resume
being white where they were blue before.
<p>
1 in 200 colonies with Tn<i>10</i> transposition show blue sectors.  If DBP6
was plated on Xgal without Tn<i>10</i> transposition, blue-sector containing
colonies occurred with 1 in 10,000 frequency.  Most of <i>Lac</i><sup>+</sup>
cells from blue sectors after transposition mutagenesis result from spontaneous
loss of plasmid or deletion removing <i>lacI</i> and <i>neo</i>, other sectors
of Tn<i>10</i> insertions affected pZC1 stability.  To identify, P1 phage is
is grown on the mutants with sectored colonies, and this
transduces Tn<i>10</i>) into DPB6 (not mutagenized) and plated on Xgal with 
tetracycline.  Only strains where Tn<i>10</i> affects genes in plasmid maintenance
can produce Tc<sup>r</sup> transductant colonies with blue sectors.
<p>
About 50,000 transposition events were screened to produce 70 indepedent Lac<sup>+</sup>
clones that are Km<sup>s</sup> and Tc<sup>r</sup>.  Nine insertion mutants show
show pZC1 instability (Pma<sup>&minus;</sup> phenotype) when transduced into
DPB6.  These were in several classes.  One class was represented by 4 of the 9
Tn<i>10</i> insertion mutants, affecting a locus at first named <i>pmaA</i>
but was renamed <i>recD</i>.
<p>
The <i>recD</i>::Tn<i>10</i> insertion mutants show moderate rate of plasmid 
loss as shown by number of sectors per colony grown on minimal agar plates 
containing X-Gal.  Cells in blue sectors of colony (Lac<sup>+</sup> segragants) 
were Km<sup>s</sup> and lacked pZC1 plasmid.  With nonselective LB medium, 
<i>recD</i> mutants show only limited instability:  after 100 generations, 10% 
of cells in culture lost the plasmid.  The low rate of loss is not unexpected 
given degree of sector formation seen when DPB28 (&Delta;<i>lacI5 
recD1901</i>::Tn<i>10</i>/pZC1) was plated on casamino acids-supplemeented plates
with X-gal.  Growth in DPB28 in supplemented liq minimal media shows higher
frequence of plasmid-free cells than growth in liq LB, which explains degree
of sector formation  seen on minimal plates and slow rate of loss in LB-grown
cultures.
<p>
Does <i>recD</i> affect plasmid instability other than in pSC101(pZC1)? Plasmids 
pBR322, pACYC184, pSC166 (Km<sup>r</sup> derivative of ColE1), F<i>kan</i>, or 
R1 was transfected to DPB115 (&Delta;<i>lacI5 recD1901</i>). It was shown that 
ColE1-like plasmids pBR322 and pACYC184 were lost faster than pSC101 derivative 
pZC1.  Plasmid pSC166 (ColE1::<i>kan</i>) was lost at a rate between that of 
pBR322 and pZC1.  Low copy number plasmids F<i>kan</i> and R1 are stable in the 
<i>recD1901</i> mutant.
<p>
The <i>recD</i> mutations were transduced to strain MG1655 because the DPB6
(&Delta;<i>lacI5</i>) strain is slow-growing, likely by a deletion removing
<i>tonB</i>.  Plasmid stability was tested in resulting isolates: DPB267
(<i>recD1901</i>) and DPB271 (<i>recD1903</i>):  in general plasmids found
unstable in <i>recD1901</i>::Tn<i>10</i> &Delta;<i>lacI5</i> were also unstable
in MG1655-derived <i>recD</i> strains.
<p>
Covalently closed circular DNA was isolated from <i>recD</i><sup>+</sup> and the 
Tn<i>10</i> insertional mutants (<i>recD1901 recD1903</i>).  Agarose gel 
electrophoresis for three different plasmids (pBR322, pSC166(ColE1::<i>kan</i>), 
and pZC1 were put into three different strains MG1655, DPB267 and DPB271, the 
first a <i>recD</i> wild type and the last two mutants.  Plasmids in mutants 
clearly showed multimeric forms, and these were pronounced for ColE1-like 
plasmids which are unstable in <i>recD</i> mutants.  When cleaved with 
restriction enzymes, the plasmid DNA from <i>recD1901</i> and 
<i>recD</i><sup>+</sup> looked identical on gel;  this means the slower mobility 
species do not result form sequence rearrangement.  (Note that the pZC1 gel 
analysis need to be transferred to nitrocellulose and probed with radiolabeled 
nick-translated pZC9 to see the multimeric forms, because it was faint on 
EtBr-stained gels.)
<p>
The Sundin and Varshavsky high resolution gel electrophoresis method was used
to resolved DNase I-treated plasmid from <i>recD1901</i>::Tn<i>10</i> or the
<i>recD</i><sup>+</sup>.  Plasmid pA (derived from pBR322) comigrated with 
concatameric pA species and not with catenanes of pA generated with DNA gyrase.
DNA gyrase in absence of ATP and other conditions can decatenate supercoiled DNA.
Treatment of multimers favoring decatenation did not alswer pattern of <i>recD</i>-promoted
multimers but converts gyrase-generate catenanes to monomers.  Thus the multimer
plasmids in <i>recD</i> mutants are concatenated molecules.
<p>
Plasmid stability was assessed in <i>recA</i> mutants and wild types.  pZC1 and
pBR322 were transfected to isogenic <i>recA</i> strains of the <i>recD</i> wildtype
and mutant strains.  With a <i>recA</i> (<i>recA56</i> mutant caused loss of 
stability that had been found in <i>recD1901</i>::Tn<i>10</i> mutant.  These
<i>recA recD</i> double mutants had few concatemeric molecules.  The <i>recD</i>
concatamer formation requires <i>recA</i> homologous recombination.
<p>
<i>recD</i> wildtype and mutants were transformed with pRDK41, a dimeric form
of pBR322, and ccc forms of DNA analyzed by agarose gel.  Some dimer&rarr;monomer
conversion was seen for the wildtype in small amount, and at least as much was
seen for dimer&rarr;monomer was seen in two <i>recD</i> mutants;  these conversions
depended on <i>recA</i> gene.  A <i>recA</i> control shows that the plasmid
DNA monomers seen in <i>recD</i> mutants does not result from transformation by
rare monomeric plasmids present in the transforming DNA.
<p>
If <i>recD</i> mutants have mechanism to convert multimers to monomers, then is
the concatamer excess associated with increased rates of plasmid recombination?
The pRDK41 dimeric derivative of pBR322  has two inactive copies of <i>tet</i>,
each copy having <i>Xho</i>I linker at a different site.  This can be used to
monitor recombination events.  The tested recombination can be used to see if
active <i>tet</i> gene can be formed.  Mutants cannot be tested that contain
Tn<i>10</i> sinc it features Tc<sup>r</sup>;  to circumvent the problem, 
recombination of <i>recD1901</i>::Tn<i>10</i> and <i>recD<sup>+</sup></i>:
a plasmid for carbenicillin resistance [Cb<sup>r</sup>] was introduced and then
scored for Tc<sup>r</sup> recombinants by isolating plasmid DN from transformants
and retransforming an <i>recA56</i> Tc<sup>s</sup> recipient and selecting 
for Cb<sup>r</sup> and Cb<sup>r</sup> Tc<sup>r</sup>.  Tc<sup>r</sup> recombinants
were found at 6-fold higher frequency in <i>recD1901</i>::Tn<i>10</i> compared to
<i>recD</i><sup>+</sup>, and this required a <i>recA</i> wildtype.  A double
mutant of <i>recA</i> and <i>recD</i> plasmid recombination was higher than 
<i>recD</i> wild type and <i>recA</i> mutant, although difference in absolute
count of recominbants was small.  The relative recombination between <i>recD</i>
mutant and wild type with a <i>recA</i> wild type differed by about 2-fold.
<p>
For testing recombination with chromosomal genes, chromosome markers <i>his</i>
or <i>trp</i> were tested for replacement using P1 transduction;  replacement 
studies were also do by conjugational crossing as a measurement of recombinational
events.  The <i>recD</i> mutants showed a recombination  frequencies 2- to 3-fold
greater than isogenic <i>recD</i> wild type.
<p>
What happens with plasmid stability in srains that are hyper-recombinogenic?
Plasmid concatamers and recombination is high in <i>recBC sbcA</i> mutants
were <i>recE</i> pathway is also active.  Isogenic derivatives of a strain
(AB1157-derived) differing in <i>recBC sbcA</i> genotypes were transfected with
pBR322 and pZC1 and plasmid stability measured.  Stability was maintained for
at least 100 generations in nonselective medium for wild type <i>recBC sbcA</i>
and in <i>recBC</i> mutants that were still wild type in <i>sbcA</i>.  Plasmids
were not stably maintained in double mutants however:  pZC1 instability was
slight, but pBR322 loss was rapid.  Analysis for concatameric species showed
increased quantities of multimeric pBR322 in the double mutant relative to 
<i>sbcA</i> wild type.  Correlated with the only slight instability of pZC1,
the isolated plasmid pZC1 had a small percentage of multimers.
<p>
Tn<i>3</i> encodes site-specific recombining system able to resolve Tn<i>3</i>
cointegrate plasmid species into monomers.  Concatameric dimer pA DNA was
isolated from DPB267 (<i>recD1901</i>::Tn<i>10</i>), the pBR322 derivative plasmid
has a <i>res</i> (resolution site) for Tn<i>3</i>, and it was transformed into
isogenic <i>recD</i><sup>+</sup> and <i>recD1901</i>::Tn<i>10</i> strains containing
either pTU4 or pZC8 which contains a Tn<i>3</i> resolvase (pTU4::Tn<i>3 bla</i><sup>s</sup>).
With the Tn<i>3</i> element present in complementation the pA dimers transformed
into DPB395 (<i>recD1901</i>::Tn<i>10</i>/pZC8) stabilized with &lt; 1% loss
after 80 generations of nonselective growth.  In contrast the loss was 97% 
after 80 generations  when no Tn<i>3</i> complementation was present.  Dimer pA
is stably inherited in <i>recD</i> mutant or wild type where <i>recA</i> is
not functional.  The isolated pA is found to be in monomeric form when
Tn<i>3</i> is present;  it is dimeric when Tn<i>3</i> is absent.  With the 
<i>recD</i> mutant and <i>recA</i> wild type and no Tn<i>3</i>, pA is found
in multimers and dimers:  thus the Tn<i>3</i> resolvase can work in <i>trans</i>
on <i>recD</i>-promoted multimers which contain the <i>res</i> site to resolve and
stabilize plasmid in <i>recD</i> mutants.
<p>
<i>Hfr</i> mapping experiments show that <i>recD1901</i>::Tn<i>10</i> is about 
61 min in the <i>E coli</i> chromosome.  Phage P1 cotransduction  with markers 
put it close to <i>argA</i>.  Located in this region are the <i>recBC</i> genese 
encoding subunits of ExoV.  The Tn<i>10</i> insertions altered recombination 
levels, but the <i>recD</i> strains did not have <i>recB</i> or <i>recC</i> null 
mutation properties, namely sensitivity to UV ligh or propensity for segregating 
inviable cells.  The <i>recBC</i> region is well defined genetically and 
physically.  This region was created as a probe for Southern hybridization, 
chromosomal DNA from wild type and mutant <i>recD</i> was restriction 
enzyme-digested and the Tn<i>10</i> insertion sites that created the 
Pma<sup>&minus;</sup> (<i>RecD</i><sup>&minus;</sup>) phenotype were located.  The 
pCDK2 containing 3 kb <i>Sal</i>I segment which contains region between 
<i>argA</i> and <i>recB</i> and the <i>argA</i> gene itself.  A <i>Pst</i>I 
digest allowed location of the Tn<i>10</i> insertion to 3.5 kb <i>Pst</i>I 
fragment between 3&prime; end of <i>recB</i> to beginning of <i>argA</i>. 
Additional digests allowed a more precise placement.
<p>
Further complementation studies were done as part of the mapping.  Plasmid pCDK2
contains that 3.0 kb <i>Sal</i>I fragment and includes site on which 
<i>recD1901</i>::Tn<i>10</i>, and it does not complement <i>recD1901</i>::Tn<i>10</i>
for stabilizing pZC1.  pCDK3 has an 18 kb <i>Bam</i>HI fragment that partially
overlaps the <i>Sal</i>I, and this contains the <i>thyA</i> to <i>argA</i> segment:
the pZC1 could be maintained stably in <i>recD</i> inactivated insertions.
This narrowed down <i>recD</i> location and shows that the mutation is recessive
to the wild type.  A 3.5 kb <i>Pst</i>I on which all the <i>recD</i> Tn<i>10</i>
insertions mapp was then inserted into the <i>Pst</i>I site of pBR325.  pZC1 in
insertionally inaactivated <i>recD</i> mutants was maintained with either orientation
of fragment.
<p>
With the localization of <i>recD</i> near <i>recBC</i>, it was probably the case
they are related functionally.  <i>recBC</i> codes for exonuclease V (ExoV).
To assess a relationship, gene 2 muntants of phage T4, which protects the ends
of T4 from being digested by ExoV, in which T4 cannot be grown for plaques on
<i>recBC</i><sup>+</sup> strains.  The gene 2 mutants can form plaques on
<i>recBC</i> strains.  Phage &lambda; mutants defective in <i>red</i>-mediated
recombination and lacking <i>gam</i> (which blocks RecBC exonuclease) make tiny
plaques on <i>recBC</i><sup>+</sup> strain as rolling circle replication does 
not occur.  On <i>recA recBC</i><sup>+</sup> strains, the &lambda; <i>red<sup>-</sup>
gam<sup>-</sup></i> phage fail to make plaques, as packageable dimer is not formed
and rolling circle is blocked, so &lambda; concatemers are not formed for packaging.
All five <i>recD</i>::Tn<i>10</i> mutants (from <i>recD1901-1905</i>) have same
plating of phage T4<i>2<sup>&minus;</sup></i> as isogenic <i>recD</i> wildtype, 
indicating ExoV is not active.
<p>
This study isolated and characterized <i>E coli</i> mutations affecting plasmid
stability during cell proliferation.  The defect created by <i>recD</i> mutation
is not by resolution (partitioning between daughter cells) but rather because
plasmid concatamers form.  Multimers placed in <i>recD</i> strains will resolve
just as well as for wild type;  it is plasmid recombination taht is increased.
<i>recD</i> probably functions in an interplasmidic rather than intraplasmidic
way given the formation of concatamers.  Plasmids show varying rates of stability
may be related to the partitioning system.  Plasmids F, R1, P1 and pSC101 have 
specific sites to ensure copies are resolved to daughter cells.
<!-- ============================================================= -->
<p class="ref">
 <span class="authors">
Swenson, K. I., Farrell, K. M., Ruderman, J. V. 
 </span>
 <span class="year">
(1986)
 </span>
 <span class="jname">
Cell
 </span>
 <span class="jvol">
47,
 </span>
 <span class="pages">
861-870.
 </span>

<p class="reftitl">
The clam embryo protein cyclin A induces entry into M
phase and the resumption of meiosis in Xenopus oocytes

<!-- ============================================================= -->
<hr>
<p>
This document continues to
  <a href="GeneralScienceVol2%20(1987-2000).html">General Science Volume 2</a>
     (from 1987 to the year 2000)
<p style="text-indent:0;">
The last part is <a href="GeneralScienceVol3%20(2001-present).html">General
Science Volume 3</a> (from the year 2001 on)
</body>
</html>
