<!DOCTYPE html>
<html lang="en">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8">
<title>Notes on the Literature from General Science (Vol. 3)</title>
<link href="./../stdcss/std.css" rel="stylesheet">
<link href="./../stdcss/biblio.css" rel="stylesheet">
<style>
   #headernotes { text-indent:0;font:bold 90% Helvetica,Arial,sans-serif; }
   #headernotes p { margin:1em 0; }
 #two-d-struct td {text-align:center;padding:0;
     font:normal 90% 'Courier New',Courier,monospace;}
  .uline {text-decoration:underline;}
</style>
</head>
<body>
<p id="title">
General Science Literature (Vol 4)
<p id="subtitle">
	Notes on scientific articles specifically unclassified from year 2013
	to present

<div class="container">
<p>
This document has been divided into four parts.  This is part 4.
The first parts can be found here:
<p><a href="GeneralScienceVol1%20(to%20and%20incl%201986).html">General Science Volume 1</a>
  (up to year 1986)
<p><a href="GeneralScienceVol2%20(1987-2000).html">General Science Volume 2</a>
  (years between 1987 and 2000)
<p><a href="GeneralScienceVol3%20(2001-2012).html">General Science Volume 3</a>
  (years between 2001 and 2012)

<!-- ============================================================= -->
<p class="tags">
microRNA, miRNA, functional genomics, high-throughput screen, qPCR, tumor suppression
<p class="ref">
 <span class="authors">
Cheng W-C, Kingsbury TJ, Sarah J. Wheelan SJ, Civin CI
 </span>
 <span class="year">
(2013)
 </span>
 <span class="jname">
BioTechniques
 </span>
 <span class="jvol">
54,
 </span>
 <span class="pages">
77-86
 </span>
<p class="reftitl">
A simple high-throughput technology enables gain-of-function screening of
human microRNAs
<p class="Para1">
MicroRNA molecules serve the biological purpose of regulating gen exression
at multiple levels, serving in cell survival, proliferation, and differentiation.
Disturbances in expression can lead to disease.  For example the <i>miR-34</i>
family of miRNAs is down-regulated in multiple cancers; these are transactivated
by p53-dependent tumor suppressor mechanisms, includding apoptosis, cell cycle
arrest, and senescence.  In mouse models having low <i>miR-34a</i> expression,
mimics were found to inhibit lung cancer growth.  Preclinical and clinical studies
show that miR-based therapeutics show no toxicity to normal cells, which shows
a high therapeutic index.  Transcriptome work shows differences in miRNA expression
between diseased and normal cells, as well as among different cell types in
stages of development (e.g. hematopoietic cells).  However difference in
expression cannot be indicative of a physiological or pathophysiological state,
nor can they be concluded to be a cause of a disease state.  There might be
miRNAs that <q>drive</q> the cell state, and those which are merely <q>passengers</q>
of it.
<p>
Large- (genomic-wide) scale approaches to doing miRNA functional screens have been
reviewed (Serva et al, J Nucleic Acids 2011: 870903).  One approach is
transfection of miRNA mimic/inhibitor libraries where synthetic miRNAs are arrayed
in 96- or 384-well plates.  Wang et al (Nat. Genet. 39: 380, 2007) used this to
identify miRNAs involved in G<sub>1</sub> arrest of DGCR8 knockout embryonic
stem cells.  But for this to work, (i) cells must be transfected efficiently
and (ii) screening is limited by transient arrays in which the mimic/or inhbitor
is diluted with every cell division.   These two limitations can be overcome
using an miRNA expression lentivirus (lenti-miR) libraries, using limiting
dilution to get one virus particle per plate.  The lenti-miR approach was used
to identify miRNAs that suppress growth or metasstais of cancer cell lines.
The disadvantage of all approaches (mimic, inhibitor, virus) per well is that
one miR can increase or decrease a function.  Screens done in an array format
require assays adaptable to microplates and require robotics.
<p>
An alternative is to use pool of multple species of lenti-miR or retroviral
miR libraries where each virus encodes a unique miRNA or an miRNA cluster.
The assay must detect a phenotype that results from over- or under-representation
of cells that over-express the miRNAs.  The identification of the miRNA causing
the effect is done by noting that the miRNA integrates into the host genome,
and computing the number of cells that have this property.  Properties such
as miRNAs that suppress cancer or that promote its invasiveness have been used
to identify the miRNAs in question.  This study reports the use of a <q>barcode</q>
within the pooled-miRNAs approach and which is found by qPCR methods.  This
system is called <b>miR-HTS</b>.  In the test case a pooled lenti-human miR
library and custom qPCR program is used to screen IRM90 human lung fibroblasts
for miRNAs regulating cell growth.
<p>
IMR90 cells were obtained from ATCC and cultured in MEM + 10% FBS, 1 mM sodium
pyruvate, 1&times; GlutaMAX (LifeTech), 1&times; pen-strep (Mediatech).
The genomic DNA representation qPCR (GRE-qPCR) assay design is as follows:
System Biosciences (Mountain View, CA) provided the miRNA (plus flanking)
insert sequences for each lenti-miR precursor.  The GRE-qPCR is designed
using Primer Express.  Internal miR-specific forward primers for each assay
are custom-plated by ABI at 18 pmol/well in 384-well plates.  Internal
common reverse primer, GFP primers, and external command forward and reverse
primers are synthesized by IDT.  GFP and internal common probes are synthesized
by ABI.  Sequences of all primers and TaqMan probes are given in supplementary
Table 1, which contains nearly 600 sequences of the varying miRNAs.
<pre>
           *F      &   ++@    # % # *R
====[LTR]--[CMV&gt;-[miR]--[EF1&alpha;&gt;[GFP]---[LTR]=====
</pre>
The GRE-qPCR is a nested PCR assay.  First the lenti-miR construct is shown in
the diagram above.  Between the viral LTR elements, the cassette contains a
CMV promoter controlling the miR cassette, and the EF1&alpha; promoter
controlling the GFP product.  The outer PCR amplicon is amplified using
the external common forward (F*) and reverse (R*) primers outside the
two expression systems.  Within the GFP coding sequence, forward and reverse
primers (both '#') and a probe oligo ('%') are used.  Another real-time
PCR amplification makes use of the inner part of the nest:  the forward primer
is miR-specific ('&amp;') and the reverse primer is
<p>
Experiments start with 2.7 million IMR90 cells infected with MOI of 0.2 with
the human lenti-miR pooled virus libary (Cat #PMIRHPLVA-1, Systems Bioscience inc).
8 &micro;g/ul polybrene is the infection vehicle (see ref).  On days 3, 11, 19,
and 27 (which are times <i>t</i><sub>0</sub> to <i>t</i><sub>3</sub>) about
2 million cells (a fraction)

<!-- ============================================================= -->
<p class="tags">
qPCR guidelines
<p class="ref">
 <span class="authors">
Ware LC
 </span>
 <span class="year">
(2013)
 </span>
 <span class="jname">
BioTechniques
 </span>
 <span class="jvol">

 </span>
 <span class="pages">

 </span>
<p class="reftitl">
<a href="http://www.biotechniques.com/news/MIQE-qPCR-Guidelines-Three-Years-Later/biotechniques-342718.html">
MIQE qPCR Guidelines, Three Years Later</a>
<p class="Para1">
Reverse transcription qPCR is a gold standard for quantifying gene expression
in a cell, but even using identical techniques two researchers can end up with
different results.  S. Ruskin led a group of researchers in 2009 to specify
guidelines in <i>Clinical Chemistry</i> which are called the Minimum Information
for Publication of Quantitative Real-Time PCR Experiments (MIQE).  These address
several variables in qPCR:  sample quality control, assay design, PCR efficiency
and normalization.  One problem is the selection of reference genes and the
reporting of the efficiency of amplification.
<!-- ========================================================================

=========================================================================== -->

<hr>

<p>This document has been divided into the three parts.  This is part 3.
The first parts can be found here:
<p><a href="GeneralScienceVol1%20(to%20and%20incl%201986).html">General Science Volume 1</a>
  (up to year 1986)
<p><a href="GeneralScienceVol2%20(1987-2000).html">General Science Volume 2</a>
  (years between 1987 and 1999)
</div> <!-- class=container -->
</body>
</html>

