<!DOCTYPE html>
<html lang="en">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8">
<title>Notes on the Literature from General Science (Vol. 2)</title>
<link href="./../stdcss/std.css" rel="stylesheet">
<link href="./../stdcss/biblio.css" rel="stylesheet">
<style>
   #headernotes { text-indent:0;font:bold 90% Helvetica,Arial,sans-serif; }
   #headernotes p { margin:1em 0; }
     .top td {vertical-align:top}
     .nobkgd, .nobkgd td {background:none;}
     .right td {text-align:right;}
     .rborder {border-right:2px solid blue;}
</style>
</head>
<body>
<p id="title">
General Science Literature (Vol 2)
<p id="subtitle">
	Notes on scientific articles (specifically unclassified) from years 1987
   to 2000
<div class="container">
<p>
This document has been divided into four parts.  This is part 2.
The other parts are:
<p><a href="GeneralScienceVol1%20(to%20and%20incl%201986).html">General Science
Volume 1</a> (all years up to 1986)
<p><a href="GeneralScienceVol3%20(2001-2012).html">General Science Volume 3</a>
  (between years 2001 to 2012)
<p><a href="GeneralScienceVol4%20(2013-present).html">General Science Volume 4</a>
  (between years 2013 to present)

<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Chou, C. K., Dull, T. J., Russell, D. S., Gherzi, R., Lebwohl, D.,
Ullrich, A., Rosen, O. M.
 </span>
 <span class="year">
(1987)
 </span>
 <span class="jname">
J. Biol. Chem.
 </span>
 <span class="jvol">
262,
 </span>
 <span class="pages">
1842-1847.
 </span>
<p class="reftitl">
Human insulin receptors mutated at the ATP-binding site lack protein
tyrosine kinase activity and fail to mediate postreceptor effects of insulin
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Lee, W., Mitchell, P., Tjian, R.
 </span>
 <span class="year">
(1987)
 </span>
 <span class="jname">
Cell
 </span>
 <span class="jvol">
49,
 </span>
 <span class="pages">
741-752.
 </span>
<p class="reftitl">
Purified transcription factor AP-1 interacts with TPA-inducible
enhancer elements
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Hunter, T.
 </span>
 <span class="year">
(1987)
 </span>
 <span class="jname">
Cell
 </span>
 <span class="jvol">
50,
 </span>
 <span class="pages">
823-829.
 </span>
<p class="reftitl">
A thousand and one protein kinases
<p class="Para1">
<b>Review</b>.  The sequence similarity between PK-A and pp60<sup>v-src</sup>
over a region of 300 amino acids and the general relatedness of the catalytic
domain of protein serine/threonine and protein tyrosine kinases suggests that
kinases come from a single archetypal gene.
Another useful discovery is that oncogenes have been identified as
metabolism-controlling kinases.  Finding
kinases is easy when one uses cloning strategies such as taking advantage of
short sequence motifs.  For example,
Gly-X-Gly-X-X-Gly followed by Lys 15-20 downstream which form the ATP binding
site.  The conserved Arg-Asp-Leu,
Asp-Phe-Gly, and Ala-Pro-Glu are found in a 60 amino acid-region 80-100
residues farther down.  Moreover, Tyr
surrounded by acidic amino acids is a perfect predictor of potential tyrosine
kinases.
<p>
Low stringency hybridization of genomic libraries with
these conserved sequences has allowed identification of a family of these
kinases.  PSK-J3, PSK-H1, and PSK-C3
have been cloned from a Hela cell cDNA library using degenerate probes
corresponding to two short sequences which are highly conserved in protein-serine
kinases, and their functions have yet to be determined.
In some cases, kinases are encoded by more
than one gene:  protein kinase C is
coded for by four genes, one of which undergoes alternate splicing to give five
isoenzymes.  PK-A is known to have
several R subunits, but only recently has been found to have more than one C
subunit.  In addition there are
protein-histidine and protein-lysine kinases about which very little is known.
<p>
<b>Cell cycle kinases.</b>  CDC28 in budding yeast and cdc2+ in fission
yeast are necessary for the moving past <q>start</q> in G1 and cdc2+
is required to pass a point in G2.  Russell and Nurse have
shown that a putative kinase, wee1+, regulates cdc2+.
Wee1+, in turn, is regulated by another kinase, nim1+.
Hence a cascade is demonstrated.
Patterson et al. (1986) have shown CDC7
encodes a kinase and SNF1 (Celenza and Carlson) and STE7 (Teague et al.) are
also kinases.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Booher, R., Beach, D. 
 </span>
 <span class="year">
(1987)
 </span>
 <span class="jname">
EMBO J.
 </span>
 <span class="jvol">
6,
 </span>
 <span class="pages">
3441-3447.
 </span>

<p class="reftitl">
Interaction between <i>cdc2</i><sup>+</sup> in the control of mitosis in
fission yeast;  dissociation of the
G<sub>1</sub> and G<sub>
2</sub> roles of the <i>cdc2</i><sup>+</sup>
protein kinase
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Russell, P., Nurse, P. 
 </span>
 <span class="year">
(1987)
 </span>
 <span class="jname">
Cell
 </span>
 <span class="jvol">
49,
 </span>
 <span class="pages">
559-567.
 </span>

<p class="reftitl">
Negative regulation of mitosis by wee1<sup>+</sup>, a
gene encoding a protein kinase homolog
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Russell, P., Nurse, P. 
 </span>
 <span class="year">
(1987)
 </span>
 <span class="jname">
Cell
 </span>
 <span class="jvol">
49,
 </span>
 <span class="pages">
569-576.
 </span>

<p class="reftitl">
The mitotic inducer 
functions in a regulatory network of protein kinase homologs controlling
the initiation of mitosis
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Das, S. K. 
 </span>
 <span class="year">
(1987)
 </span>
 <span class="jname">
Biochim. Biophys.
Acta
 </span>
 <span class="jvol">
928,
 </span>
 <span class="pages">
29-35.
 </span>

<p class="reftitl">
Abrogation of the effects of aphidicolin on NIH 3T3 and
V79 cells by caffeine
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Lee, M. G., Nurse, P. 
 </span>
 <span class="year">
(1987)
 </span>
 <span class="jname">
Nature
 </span>
 <span class="jvol">
327,
 </span>
 <span class="pages">
31-35.
 </span>

<p class="reftitl">
Complementation used to clone a human homologue of the
fission yeast cell cycle control gene <i>cdc2</i>
<p class="author">Draetta, G., Brizuela, L., Potashkin, J., Beach, D.
(1987) <i>Cell</i> <b>50,</b> 319-325.
<p class="reftitl">
Identification of p34 and p13, human homologs of the cell
cycle regulators of fission yeast encoded by <i class=
'c7'>cdc2</i><sup>+</sup> and <i>suc1</i><sup>+</sup>
<p class="author">Lee, M. G., Norbury, C. J., Spurr, N. K., Nurse, P.
(1988) <i>Nature</i> <b>333</b>, 676-679.
<p class="reftitl">
Regulated expression and phosphorylation of a possible
mammalian cell cycle control protein
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Macdonald, R. J., Swift, G. H., Przybyla, A. E., Chirgwin, J. M.
 </span>
 <span class="year">
(1987)
 </span>
 <span class="jname">
Meth. Enzymol.
 </span>
 <span class="jvol">
152,
 </span>
 <span class="pages">
219-.
 </span>
<p class="Para1">
[method on the guanidinium chloride isolation of RNA]
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Livingstone, R. E.
 </span>
 <span class="year">
(1987)
 </span>
 <span class="jname">
in Current Protocols in Molecular
Biology, F. M. Artabel et al., eds., (Wiley: NY),
 </span>
 <span class="pages">
p. 4.8.2
 </span>
<p class="Para1">
[method on primer extension analysis]
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Imagawa, M., Chiu, R., Karin, M.
 </span>
 <span class="year">
(1987)
 </span>
 <span class="jname">
Cell
 </span>
 <span class="jvol">
51,
 </span>
 <span class="pages">
251-260.
 </span>
<p class="reftitl">
Transcription factor AP-2 mediates induction by two different
signal transduction pathways:  protein kinase C and cAMP
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
De Wet, J. R., Wood, K. V., DeLuca, M., Helinski, D. R., Subramani, S.
 </span>
 <span class="year">
(1987)
 </span>
 <span class="jname">
Mol. Cell. Biol.
 </span>
 <span class="jvol">
7,
 </span>
 <span class="pages">
725-737.
 </span>
<p class="Para1">
[report on technique for performing chloramphenicol
acetyltransferase (CAT) activity assay]

<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Gilman, A. G.
 </span>
 <span class="year">
(1987)
 </span>
 <span class="jname">
Annu. Rev. Biochem.
 </span>
 <span class="jvol">
56,
 </span>
 <span class="pages">
615-649.
 </span>
<p class="reftitl">
G proteins:  transducers of receptor-generated signals
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Ebina, Y., Araki, E., Taira, M., Shimada, F., Mori, M., Craik, C. S.,
Siddle, K., Roth, R. A.
 </span>
 <span class="year">
(1987)
 </span>
 <span class="jname">
Proc.&nbsp;Natl.&nbsp;Acad.&nbsp;Sci.&nbsp;USA
 </span>
 <span class="jvol">
84,
 </span>
 <span class="pages">
704-708.
 </span>
<p class="reftitl">
Replacement of lysine residue 1030 in the putative ATP-binding
region of the insulin receptor abolishes insulin- and antibody-stimulated
glucose uptake and receptor kinase activity
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Zoon, R. A.
 </span>
 <span class="year">
(1987)
 </span>
 <span class="jname">
Meth. Enzymol.
 </span>
 <span class="jvol">
152,
 </span>
 <span class="pages">
25-29.
 </span>
<p class="reftitl">
Safety with <sup>32</sup>P- and <sup>35</sup>S-labeled compounds
<p class="Para1">
Both of these isotopes are <q>pure</q> beta emitters, that
is, they produce no photons.  <sup>35</sup>S has a maximum beta
energy of 167 keV with a 87.4 d half-life.  <sup>32</sup>P produces
a maximum energy of 1710 keV with a 14.3 d half-life.  <sup>32</sup>P
is far more dangerous as an external radiation hazard:  with hundreds
of &micro;Ci or to tens of mCi, one can receive the maximum permissible
occupational dose in a matter of seconds to minutes.  The
maximum skin does a U.S. worker is permitted annually is 30,000 mRem/annum,
distributed at a rate of 7500 mRem/quarter.
The dose equivalent to the lens of the eye (the organ of concern) is
5000 mRem/annum or 1250 mRem/quarter.
At the surface of a glass <q>Combi-v-vial</q> usually used to ship
the isotopic compound is a dose equivalent of 470 mRem/hr-mCi.
The exterior surface of a plastic syringe
containing a <sup>32</sup>P-compound solution can be 170,000 mRem/hr-mCi.
With <sup>35</sup>S, only internal exposure from inhalation or ingestion
presents a hazard.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Casal HL, K&ouml;hler U, Mantsch HH
 </span>
 <span class="year">
(1987)
 </span>
 <span class="jname">
Z. Naturforsch.
 </span>
 <span class="jvol">
42c,
 </span>
 <span class="pages">
1339-1342.
 </span>
<p class="reftitl">
Conformational Changes in Proteins Induced by Low Temperatures:
An Infrared Study
<p class="Para1">
Hydro-alcoholic mixtures (phosphate buffer/glycerol, 1:1) and glycerol-free
buffers were used that had no spectral interference in the 1800-1600
cm<sup>&minus;1</sup> range.  Solutions had a pD 7 at room temperature,
which increases to pD 8.2 at &minus;100&deg;.
<p>
The spectral region of 1750-1600 cm<sup>&minus;1</sup> contains the
amide&nbsp;I bands.  These amides are involved in secondary structure,
so the bands provide information about hydrogen-bonding.  The spectra
requires two phases of computational processing to obtain the required
components frequencies.  The first is a deconvolution and the second
is a derivatization.  The process is similar to a Fourier analysis
and adding the constraints about what is understood about all levels
of molecular structure for the molecule studied.  Component peaks are
teased out from the process.
<p>
1.5 mM hemoglobin in phosphate/glycerol at 25&deg; shows a major band at
1655 cm<sup>&minus;1</sup> with three weaker bands at 1680, 1640,
and 1621 cm<sup>&minus;1</sup>.  In glycerol-free buffer, the bands
are 1680, 1653, 1638, and 1622 cm<sup>&minus;1</sup>.  This is compatible
with a hemoglobin structure composed primarily of &alpha;-helices.
Glycerol has only a negligible effect.  The shift in 2 cm<sup>&minus;1</sup>
of the lower wavenumbers when glycerol is absent is likely due to
reduced polarity of a hydro-alcohol solution.
<p>
At &minus;100&deg; in phosphate/glycerol buffer,
the spectrum is quite different.  Bandwidths are
wider in general, and the deconvolved and derivative spectra reveal
a new band along with shifts in bands present at room temperature.
Peaks at 1680, 1665 (new band), 1654, 1634, and 1620 cm<sup>&minus;1</sup>
are present.  The changes result from the change in temperature and
not the increase in pD to 8.2, since spectra of pD 8.2 and 7.0 are the
same at room temperature.
<p>
All these spectral changes are reversible.  Protein warmed from &minus;100&deg;
to 25&deg; show identical spectra to samples before freezing.  Spectra
taken at several transition temperatures starting from 25&deg; to
&minus;100&deg; show the emergence of a band at 1665 cm<sup>&minus;1</sup>,
a band that disappears upon warming at a transition between &minus;60&deg;
and &minus;50&deg;.  In glycerol-free buffer at &minus;100&deg;
the major band is 1652 cm<sup>&minus;1</sup> and there are three
weaker bands at 1675, 1635, 1618 cm<sup>&minus;1</sup>.
This differs from the hydro-alcoholic (phosphate/glycerol) solution
in that no new band occurs.  Proteins in D<sub>2</sub>O solutions
contain bands in the 1668 to 1662 cm<sup>&minus;1</sup> region which
report turn-type structures.  The 1665 cm<sup>&minus;1</sup> band
that appears in phosphate/glycerol indicates a formation of turns
at the expense of &alpha;-helix.  But since the band appearance is
reversible, it seems unlikely that these are turns.  In the re-folding
process, the residues involved would have to undergo H-D exchange
in the D<sub>2</sub>O solution and so the spectrum with re-warming
should be different (characteristic of &alpha;-helix containing
deuterium-exchanged NH groups).
<p>
Instead the 1665 cm<sup>&minus;1</sup> band probably corresponds to
an &alpha;-helix assuming a different H-bonding pattern, namely the
formation of a 3<sub>10</sub> helix.  A 3<sub>10</sub> helix was
calculated for poly-(&alpha;-aminoisobutyric) acid.  A 1662 cm<sup>&minus;1</sup>
band in bacteriorhodopsin has been attributed to distorted &alpha;-helix
rods.
<p>
To rule out that the band may have been attributable to changes
in quaternary structure in hemoglobin, myoglobin was analyzed in
phosphate/glycerol at 25&deg; and &minus;100&deg;.  The 1665
cm<sup>&minus;1</sup> band is not present in the cold temperature
range, where it is in the case of hemoglobin.  Hence the 1665
cm<sup>&minus;1</sup> band may reflect changes in the quaternary
structure.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Khoo JC, Miller E, Goldberg DI
 </span>
 <span class="year">
(1987)
 </span>
 <span class="jname">
J. Lipid Res.
 </span>
 <span class="jvol">
28,
 </span>
 <span class="pages">
873-877.
 </span>
<p class="reftitl">
A Rapid and Sensitive Enzymatic-Radiochemical Assay for the Determination
of Triglycerides
<p class="Para1">
Triglyceride determinations to date fall into two main chemistries: (1)
measurement of glycerol liberated from triglycerides by lipolysis or
chemical hydrolysis or (2) separation of lipid classes by TLC which are
then assayed by charring of triglycerides using sulfuric acid or by
using gas chromatography.  Glycerol determinations are easy to use and
specific but are not sensitive enough for quantifying triglycerides in
small samples of cultured cells or tissues.  Recently an enzyme-coupled
assay with fluorimetric sensitivity was available but required fluorimeter.
The GLC method requires special equipment and is a bit time-consuming,
although sensitive.
<p>
In order to measure triglycerides on assays based on measurement of free
fatty acids released by hydrolysis, there are two fundamental requirements:
<ol>
<li>  all the FFA must come from the triglycerides
<li> triglycerides must be totally hydrolyzed
</ol>
<p>
It is important to make sure that enzymatic hydrolysis does not releases
FFAs esterified to cholesterol, only to glycerol.  Some commercial preps
cause this.  Generally microbial and pancreatic lipases do not cause this.
A combination of both microbial and pancreatic lipases is necessary to
achieve <em>total</em> hydrolysis of the triolein substrate.  To test
these, 10 nmol of [<sup>14</sup>C]triolein was incubated for various times
with lipases, individually or in combination, at 37&deg;, with the first
stops at 15 min and going as long as 90 min.  It was determined that 60
min was all that was necessary.  Adding more enzyme did not achieve a
faster result.  In general, combined lipases achieved total hydrolysis
and neither alone did.  In general the pancreatic lipase was found to
be more active against triolein, but the microbial lipase was better
on diolein or mono-olein as a substrates (partial glycerides).
<p>
A FFA-<sup>63</sup>Ni complex can be formed once hydrolysis is complete,
and this be used to measure the amount of triglycerides.  The assay is
linear from 0.25 to 25 nmol in the assay used.  Tripalmitin gave similar
results but has a low solubility.  This method was used to measure
triglycerides in mouse peritoneal macrophages.  It was sensitive enough
to detect 300 pmol triglyceride with 45 &micro;g cellular protein.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Mahoney CW, Azzi A
 </span>
 <span class="year">
(1987)
 </span>
 <span class="jname">
Biochem. J.
 </span>
 <span class="jvol">
243,
 </span>
 <span class="pages">
569-574.
 </span>
<p class="reftitl">
The Synthesis of Fluorescent Chlorotriazinylfluorescein-concanavalin A and
Its Use As a Glycoprotein Stain on Sodium Dodecyl Sulphate/Polyacrylamide Gels
<p class="Para1">
FITC-labeled lectins have been useful in specific probing of glycoproteins.
Conjugation of FITC to concanvalin A (con A) requires alkaline pH (pH 9.5) for
long times (20 h) and in many cases the con A lectin is inactivated and forms
a precipitate.  Even if a product is obtained, it does not penetrate
polyacrylamide gels.  Dichlorotriazinylaminofluorescein (DTAF) is an analogue of
fluorescein that reacts with the free amino group under less alkaline conditions
and more rapidly.
<p>
DTAF is synthesized according to method of Blakeslee, with organic solvents dried
over molecular sieves and reactants with P<sub>2</sub>O<sub>5</sub> <i>in vacuo</i>.
50 mg (144 &micro;mol) of 4-aminofluorescein (Fluka) is placed in 1.5 ml anhydrous
MeOH, then added dropwise to a stirring solution of 35 mg (190 &micro;mol) of
cyanuric chloride in 250 &micro;l of anhydrous chloroform (at 4&deg;).  The reaction
is stirred at 4&deg; for 2 h, quenched by dropwise addition of 50 &micro;l of
10.3 M HCl, then stirred another 1 h.  The hydrochloride salt is collected by
vacuum filtration, washed with 14% (v/v) anhydrous chloroform in dry MeOH, and
then dried.  DTAF is obtained in 85-90% yield, with purity checked by TLC on silica
gel with MeOH/CHCl<sub>3</sub> (9:1, v/v) in which it has <i>R<sub>f</sub></i> 0.95
or a 1:9 (v/v) solution with <i>R<sub>f</sub></i> 0.51.  Store desiccated at 4&deg;
and it is stable for 15 months. (Note that Sigma already prepares this reagent now.)
<p>
To conjugate DTAF to lectin:  a final reaction mixture of 2.1 mg/ml (78 &micro;M)
con A, 0.15 M HEPES-sodium pH 7.0 or 8.0, and a 1- to 3-fold molar excess of DTAF
over con A is prepared, conducted in presence or absence of 0.2 M &alpha;-methyl
D-mannoside.  At various times, 100 &micro;l is sampled and passed over a
1 ml Sephadex G-25 column in HEPES-sodium buffer containing (or not) the methyl
mannoside.  Absorbance at 492 nm is read (DTAF extinction is 63400 M<sup>&minus;1</sup>
cm<sup>&minus;1</sup> at 492 nm), and protein assays are done.  Optimal conjugation
was found to be 3-fold molar excess with reaction done for 6 h at room temp.
Adding 1 M ethanolamine pH 8.0 stock to 10 mM quenches the reaction, leaving
another 6 h.  Dialysis/diafiltration against 50 mM TrisHCl pH 8 with 0.1 M NaCl,
1 mM MnCl<sub>2</sub>, and 1 mM CaCl<sub>2</sub> and 4 mM 2-mercaptoethanol.
Solution is diluted to 0.5 mg/ml con A, and 3-fold molar excess BSA added.
pH is adjusted to 5.5 and storage is at 4&deg;.  The presence or absence of
the methyl D-mannoside had no effect on efficiency or kinetic of the DTAF conjugation,
indicating that DTAF does not attach to the carbohydrate part of the lectin.
The more alkaline the pH, the better the labeling, being 2-fold greater at pH 8
than at pH 7.  Increasing the molar excess from 1 to 3 also results in a 2-fold
better yield.  With optimal conditions, 0.35 mol of CTAF incorporates per mol
protein.  With more optimization attempts, this can reach as high as 0.5 mol
CTAF per mol lectin.
<p>
Con A tends to aggregate under such conditions, forming dimers or even tetramers
and precipitating aggregates.  The longer the time of treatment, the greater the
chance of precipitation.  DTAF reactions are complete in 6 h, whereas with FITC
it is 30 h.
<p>
SDS-PAGE under reducing conditions using quenched CTAF-con A showed a single
fluorescent band corresponding to con A monomer.  No aggegates were seen in the
stacking/separating gel interface.  A problem is that on storage, aggregate formation
is seen, because the DTAF has three chloro substituents of high, medium, and low
reactivity, and the final chloro group reacts slowly in storage to aggregate with
con A.  Thus, BSA presence is essential.  Gels are fixed in 40% MeOH/10% acetic
acid for 1 h and then equilibrated in four 30-min washes of 50 mM sodium acetate
pH 5.5 with 0.1 M NaCl and 4 mM 2-mercaptoethanol.  Gels are then incubated
(40 ml/gel) with CTAF-con A solution and agitated overnight or for 2-3 days.
Gels are destained in four 30-min washes of the acetate-saline buffer and then
dehyrated in 94% EtOH.  Imaging is done with transilluminator using camera with
UV cutoff and orange filter.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Lubas WA, Spiro RG
 </span>
 <span class="year">
(1987)
 </span>
 <span class="jname">
J. Biol. Chem.
 </span>
 <span class="jvol">
262,
 </span>
 <span class="pages">
3775-3783.
 </span>
<p class="reftitl">
Golgi Endo-&alpha;-D-mannosidase from Rat Liver, a Novel <i>N</i>-Linked
Carbohydrate Unit Processing Enzyme
<p class="Para1">
Rough ER from livers from decaptitated male rats (250-400 g, Charles River CD
strain) fasted 18-20 h using method of Leelavathi et al as modified by Tulsiani
et al (for liver excision) and also Depierre and Dallner (for RER).  The RER
procedure was also modified at initial stage with spin of 600 <i>g</i> for 10
min, then spinning this supernatant at 10000 <i>g</i> for 20 min, and this
supernatant layered on 0.6 and 1.3 M sucrose, both containing 15 mM calcium
chloride.  Homogenzing buffer was 4 vol 0.5 M sucrose, 50 mM sodium maleate pH
6.5 with 5 mM MgCl<sub>2</sub> by centrifuging the 20 min/10000 <i>g</i> supnt
at 100,000 <i>g</i> for 1 h. Membranes isolated are stored @ &minus;70&deg;
until enzyme assay. &mdash; <b>[<sup>14</sup>C]Oligosaccharides</b>.
Glc<sub>1</sub>Man<sub>9</sub>GlcNAc and the polymannose Man<sub>5</sub>GlcNAc
to Man<sub>9</sub>GlcNAc were prepared in calf thyroid slices with
D-[U-<sup>14</sup>C]glucose (325 mCi/mmol) for 3 h/37&deg; as described
previously. This produces the reduced forms which are still substrates but
more traceable. The postnuclear supernatant is delipidated
(CHCl<sub>3</sub>:MeOH/H<sub>2</sub>O, in 3:2:1 then in 10:10:3), then digested
with pronase followed by endo-&beta;-N-acetylglucosaminidase H.  The
oligosaccharide released is reduced witn NaBH<sub>4</sub> (Turco) and
fractionated on 4 mm &times; 25 cm LiChrosorb-NH<sub>2</sub> (5 &micro;m)
protected by Vydac amino precolumn.  Flow is 0.5 ml/min of isocratic 60% MeCN
with 1 ml fractionation and samping for scintillation.  Fractions are pooled
and solvent evaporated.  Each peak resolved is dissolved in water and passed
over Dowex 50-X2 (H<sup>+</sup> form, 200-400 mesh) and Dowex 1-X2 (acetate
form, 200-400 mesh) in series.
<p>
<b>Results</b>.  Triton X-100-treated Golgi membranes were incubated with
GlcMan<sub>9</sub>GlcNAchH<sub>2</sub> results in conversion to
Man<sub>5</sub>GlcNAcH<sub>2</sub> and Man<sub>6</sub>GlcNAcH<sub>2</sub>.
The addition of deoxynojiimycin (DNJ) at 1 mM had no effect;  DNJ is a glucosidase
II inhibitor and so the removal of Glc must be achieved by another means.
Golgi &alpha;-mannosidase I is an EDTA-dependent enzyme, and so EDTA + DNJ,
and another oligosaccharide (GlcMan<sub>8</sub>GlcNAcH<sub>2</sub>) was
seen on the TLC plate.  From these findings a determination was made about
released monosaccharides:  the TLC results show Man and a small amount of Glc
and another component probably a disaccharide.   Presence of DNJ alone had no
effect but with DNJ + EDTA, the Man and Glc could not be observed, although
the disaccharide is.  The Golgi fraction likely contains an endoglycosidase to
remove a Glc-Man disaccharide.  EDTA inhibits the &alpha;-mannosidase I but
DNJ must be included must be added to preclude cleavage of the
GlcMan<sub>9</sub>GlcNAcH<sub>2</sub> or disaccharide by the glucosidase.
This is demonstrated by incubating large amounts of enzyme, which results in
substantianal presence of Glc and small amts of the disaccharide.  Man is quite
evident.  With DNJ, only Man and disaccharide are present, no Glc.  With EDTA +
DNJ, no Man or Glc, just disaccharide.  A plot of change in radiactivity vs time
for G1cMan<sub>9</sub>GlcNAcH<sub>2</sub> showed its gradual decrease while there
is a increase in an oligosaccharide (OL) and a smaller increase in disaccharide (Di)
The sum of the increases in OL + Di matched the decrease in the GM<sub>9</sub> oligo.
<p>
Di was purified in two chromatographic systems.  Acid hydrolysis revealed Glc and
Man in equivalent amts.  There was a discrepancy in radioactivity totals in the
monosaccharides because the glucose in GlcMan<sub>9</sub>GlcNAcH<sub>2</sub> has a higher
specific activity (1.4-fold) than mannose.  When Di is NaBH<sub>4</sub>-treated
and then hydrolyzed, a mannitol and glucose are obtained, indicating Man is at
the reducing end.  This supports the finding that jack bean &alpha;-mannosidase
does not break down Di, but that it did break down to Glc and Man with and ER or
baker's yeast &alpha;-glucosidase.  Smith degradation (periodate oxidation,
sodium borohydride reduction, then mild acid hydrolysis, produces glycerol and
arabinitol.  If a pentitol product is formed, this suggests C1-C2 cleavage by
periodate, and so a 1,3-glycosidic bond
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Creutz, C. E., Zaks, W. J., Hamman, H. C., Crane, S., Martin, W. H., Gould, K. 
L., Oddie, K. M., Parsons, S. J.
 </span>
 <span class="year">
(1987)
 </span>
 <span class="jname">
J. Biol. Chem.
 </span>
 <span class="jvol">
262,
 </span>
 <span class="pages">
1860-1868.
 </span>
<p class="reftitl">
Identification of chromaffin granule-binding proteins.  Relationship of the 
chromobindins to calectrin, synhibin, and the tyrosine kinase substrates p35 and 
p36
<p class="Para1">
Calelectrins were isolated from bovine liver and adrenal medulla.  First the 
pellet from a liver homogenate was washed in Ca<sup>2+</sup>-containing buffers 
and then the Ca<sup>2+</sup>-dependent membrane-binding proteins extracted in a buffer 
containing EGTA (this procedure used to prepare calelectrins from Torpedo).  The 
extracted protein is prepared on phenyl Sepharose in presence of EGTA to remove 
both lipid and proteins with Ca<sup>2+</sup>-independent hydrophobic character. 
Mono Q chromatography is then used to separate 32 and 67 kDa membrane-binding 
proteins, both of which elute at 140 and 240 mM NaCl, resp.  The 67 kDa protein 
is contaminated with small amounts of 32.5 and 35 kDa proteins identified on 2-D 
gels as chromobindins 5 and 7, resp.  Amino acid analyses were done on the 
purified 32 and 67 kDa protein and the composition is similar to mammalian 
calelectrins.
<p>
These calelectrins are known to cause aggregation of chromaffin granules in 
vitro in the presence of Ca<sup>2+</sup>.  While the two proteins possess such 
activity, the 67 kDa promotes only a small degree of granule aggregation.  The 
32 kDa is much more efficient, according to a protein dose curve (granule 
aggregation is measured as an increase in granule solution turbidity by 
absorbance at 540 nm after 6 min).  Combined addition of the 32 and 67 kDa 
protein actually appears to inhibit granule aggregation, as the turbidity 
(absorbance) is shown to decrease.  Synexin-induced chromaffin granule 
aggregation was also inhibited by the 67 kDa protein.  Previously isolated 
protein having a 67 kDa weight and paradoxically an aggregating agent by itself 
but an aggregate-inhibiting agent with other proteins was called synhibin.
<p>
A previous report describing the use of antisera to 32 kDa Torpedo calelectrin 
or to the bovine liver protein indicated a reaction with both 32 and 67 kDa 
protein from mammalian sources.  The proteins purified in this study were 
prepared on SDS gels and injected into rabbits, but the obtained antisera showed 
no immunocrossreactivity on blots.  The anti-32 kDa protein also detected a 
protein in human blood platelet extracts, thought to lack calelectrins. 
Immunoblotting of several bovine tissues shows the presence of the 32 kDa 
protein with the following order indicating greater relative abundance:  adrenal 
medulla ~ liver &gt; adrenal cortex &gt; brain gray matter &gt; human platelets 
&gt; brain white matter.  It should be noted that gray matter is composed mostly 
of synaptic endings while white matter is largely myelinated axons.
<p>
[this paper is continued].
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Zimlichman, R., Goldstein, D. S., Zimlichman, S., Stull, R., Keiser, H. R.
 </span>
 <span class="year">
(1987)
 </span>
 <span class="jname">
Cell Calcium
 </span>
 <span class="jvol">
8,
 </span>
 <span class="pages">
315-325.
 </span>
<p class="reftitl">
Angiotensin II increases cytosolic calcium and stimulates catecholamine release 
in cultured bovine adrenomedullary cells
<p class="Para1">
Glands are perfused with collagenase for digestion to obtain cells.  Angiotensin 
II (A-II) induces dose-dependent CA release from the cultured cells.  CA release 
is expressed as % release after 15 min;  NorEpi, Epi, and dopamine are each 
measured.  Maximal release obtained with 1 &micro;M A-II, and half maximal is 
somewhere between 1 nM and 10 nM (although table shows log dose measurements). 
The % release by 1 &micro;M A-II was compared to 50 mM KCl-induced release 
(shown to be comparable), and to 62 &micro;M nicotine, shown to be 40% of the 
level of nicotine-stimulated release.  Control releases show A-II was at least 
2.5 times greater, indicating significant effect.  Intracellular calcium changes 
were measured, in which time-dependent changes were monitored using Quin-2 
fluorescence as a measure.  A-II, nicotine, and KCl (at the doses above) all 
showed nearly the same times to reach maximal Ca concentration (A-II showed the 
slowest rate but not by much) and they showed that they reached the same maximal 
concentrations of calcium.  The return of Ca<sup>2+</sup> to baseline of the 
A-II stimulated cells required several minutes; it was slower than the 
nicotine-stimulated response, but faster than the KCl-stimulated cells.  It is 
noted by the authors that the various intracellular Ca concentration responses 
varied between cells cultured from different batches of collagenase.
<p>
The log dose-dependency of A-II on intracellular Ca<sup>2+</sup> was also 
assessed. Minimum dose necessary to produce a response was 1 nM A-II, with 
maximal response produced by 0.5 &micro;M, consistent with the CA release 
dose-dependency curve.  It should be noted that 1 nM A-II affected intracellular 
Ca levels, but dose not elicit CA release, indicating that intracellular 
Ca<sup>2+</sup> levels must exceed a certain threshold.
<p>
Combination treatment studies during a time course measuring changes in 
intracellular Ca were also done.  Nicotine added at maximal dose followed 2 min 
later by nicotine gave no additional stimulation.  A-II at maximal dose followed 
2 min later by A-II also showed no additional increases.  Nicotine followed by 
A-II did stimulate a greater intracellular Ca level.  This suggests that A-II 
and nicotine work through different Ca mobilization pathways.
<p>
Cells suspended in Ca-free medium abolished any A-II-induced intracellular 
calcium increases, suggesting that A-II stimulates Ca increases via Ca-channel 
proteins on the plasma membrane.  Addition of the Ca-channel blocker verapamil 
blocks all A-II, nicotine, and KCl-induced intracellular Ca increases.  At 0.1 
mM, verapamil attenuates A-II-induced increases, at 1 mM it abolishes it. 
Nifedipine showed similar responses.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Wilson, S. P.
 </span>
 <span class="year">
(1987)
 </span>
 <span class="jname">
J. Neurosci. Meth.
 </span>
 <span class="jvol">
19,
 </span>
 <span class="pages">
163-171.
 </span>
<p class="reftitl">
Purification of adrenal chromaffin cells on Renografin gradients
<p class="Para1">
Chromaffin cells were prepared by collagenase digestion (see Wilson and Viveros, 
1983; and Wilson and Kirshner, 1983) with modifications.  Briefly, glands from 
the slaughterhouse are brought to the lab and injected 4 times over a 15-30 min 
period at 37&deg; by syringe into the adrenal vein with 3-5 ml warmed buffer 
(the buffer is similar to Krebs-Ringers sans calcium with HEPES).  Glands are 
perfused 4 times at 15 min intervals with 3-4 ml of buffer containing 1.5 to 2 
&micro;g/ml collagenase (Type I from Cooper Biomedical), 30 &micro;g/ml DNase 
type I (Sigma), and 100 &micro;M CaCl<sub>2</sub>.  The cortices are removed and 
medullae pooled and minced, and further digested by stirring with 10 ml of 
collagenase solution per gland for 30 min at 37&deg;.  The solution is filtered 
through a sandwich of nylon mesh (250, 125, 53 &micro;m) and filtrate twice 
centrifuged (100 <i>g</i> for 5 min) in buffer.
<p>
Crude cells are resuspended in 45% Percoll made in an isotonic solution. 
Centrifugation for 20 min at 15-20&deg; in SS-34 at 48,000 <i>g</i> is done. 
Two gradients of 13.5 ml each are done for each adrenal gland digested.  Cells 
between densities of 1.045 and 1.075 g/ml are collected.
<p>
In another preparation cells are resuspended in 14% (327 mOsm, 1.068 g/ml) or 
15% (345 mOsm, 1.073 g/ml) Renografin which was prepared by dilution of the 60% 
commercial stock with water.  A 7.5% solution (1.038 g/ml) is also prepared. 
This solution is layered over 20 ml of the resuspended cells for a volume of 15 
ml or so to form a discontinuous gradient.  Centrifugation at 15-20&deg; for 20 
min at 7700 g in an SS-34 rotor is performed.  The diffuse band of cells forming 
at the interface is collected.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Houchi, H., Oka, M., Misbahuddin, M., Morita, K., and Nakanishi, A.
 </span>
 <span class="year">
(1987)
 </span>
 <span class="jname">
Biochem. Pharmacol.
 </span>
 <span class="jvol">
36,
 </span>
 <span class="pages">
1551-1554.
 </span>
<p class="reftitl">
Stimulation by vasoactive intestinal polypeptide of catecholamine synthesis in 
isolated bovine adrenal chromaffin cells
<p class="Para1">
Cells are isolated by sequential digestion of adrenal medullary slices.  The 
freshly isolated cells are then treated.  <sup>14</sup>C-catecholamine synthesis 
was assessed over a dose course of vasoactive intestinal polypeptide (VIP) in 
which cells had been preincubated with <sup>14</sup>C-tyrosine for 15 min.  Four 
points (0, 0.3, 1.5 and 7.5 &micro;M) were tested.  At 1.5 &micro;M considerable 
increases in synthesis have take place (25% increase).  This stimulation was 
sustained for at least 45 min. While it may stimulate synthesis, it does not 
stimulate CA release.  Other bioactive peptides such as secretin and substance P 
at 10 &micro;M did not affect CA synthesis.  With <sup>14</sup>C-DOPA as a 
substrate instead of <sup>14</sup>C-tyrosine, CA synthesis was not increased: 
therefore the VIP must have stimulated tyrosine hydroxylation.
<p>
Since stimulation of CA synthesis by nicotinic agonists and depolarizing agents 
is known to depend upon extracellular Ca<sup>2+</sup>, the dependence of 
Ca<sup>2+</sup> mediated by VIP was tested.  The results show that whereas 
carbamylcholine did depend upon Ca<sup>2+</sup>, the action of VIP did not.  The 
effect of carbamylcholine was however over 3-fold greater than that of VIP.
<p>
The effect on cAMP level was assessed (by separating on ion-exchange column then 
using a protein binding method) to see if VIP mediated its effect through this 
pathway.  VIP did not increase cAMP.  Forskolin (10 &micro;M) did raise levels 
10-fold.  With a combination of forskolin and VIP, the effect was the same as 
with forskolin alone.
<p>
It is known that 4?-phorbol 12,13-didecanoate (PDD) does not stimulate protein 
kinase C in cells where it is active.  0.1 to 10 &micro;M PDD had no effect with 
these cells as well (data not shown).  Treatment of the cells with VIP and 
carbamylcholine was found to be additive, indicating independent pathways at the 
very least.  TPA, which alone stimulated CA synthesis as much as carbamylcholine 
alone, was found not to be additive with VIP treatment.  H-7, an inhibitor of 
PK-C, was found to prevent the increase caused by TPA and VIP, whether singly or 
in combination.  It seems therefore that VIP and TPA act through a common 
mechanism, strongly indicating that VIP acts through PK-C.  The hydrolysis of 
phosphoinositides often accompanies PK-C activation and is being investigated.
<!-- ============================================================= -->
<p class="tags">
immunoselection, expression library, surface antigen, cloning strategy, T11 antigen
<p class="ref">
 <span class="authors">
Seed B, Aruffo A
 </span>
 <span class="year">
(1987)
 </span>
 <span class="jname">
Proc. Natl. Acad. Sci. USA
 </span>
 <span class="jvol">
84,
 </span>
 <span class="pages">
3365-3369.
 </span>
<p class="reftitl">
Molecular cloning of the CD2 antigen, the T-cell erythrocyte receptor, by a 
rapid immunoselection procedure
<p class="Para1">
Human thyocytes and peripheral T lymphocytes form rosettes with sheep erythrocytes.
Specific anti-T cell mAbs will block this formation, and some of the mAbs inhibit
T cell activation and cytolysis.  All antibodies recognize 50 kDa surface antigen
on thymocytes and T lymphocytes called CD2.  Pairwise combinations of anti-CD2 mAbs will 
cause activation of T cells.  A recent finding shows phytohemagluttinin will cause
T cell or leukemia cells to have a calcium influx if they have CD2, but this does
not occur in T cells having no CD2 or pretreated with anti-CD2 mAbs.  To clone CD2,
a technique to express it transiently in COS cells and select expressing cells using
antibody-coated tissues was done.  The technique permits a large number of surface
antigen cDNAs to be cloned in short time period.
<p>
The expression vector &pi;SV is constructed for COS cells.  HPB-ALL are 
a T-cell acute lymphocytic leukemia cell line, and their RNA was prepared
using guanidinium thiocyanate and cesium chloride.  cDNA was prepared according 
to the Gubler and Hoffman method (RNase H + <i>E coli</i> DNA polymerase I
+ T4 DNA polymerase, and probably using methylation and linker adapters).  
Insertion into vector was through <i>Bst</i>XI non-selfcomplmentary.  Ligations
are transfected into <i>E coli</i> MC1061/p3 using a special protocol to make
them competent.  Falcon 1007 dishes were used to culture bacteria for doing
panning, except unreacted sites are blocked using PBS (8.1 mM disodium phosphate,
1.5 mM monopotassium phosphate, 2.7 mM potassium chloride, 137 mM sodium chloride,
pH 7.5) with 1 mg/ml BSA.  Round 1 screening has 24 6-cm dishes with 50% confluent
COS cells that are transfected by spheroplast fusion.  3 d after fusion cells
are detached and incubated in PBS + 1 mM EDTA + 0.02% sodium azide at 37&deg;
for 30 min.  Cells were pooled, spun, and resuspended in cold PBS/10 mM EDTA/5% FBS
containing mAbs usually as 1:1000 ascites.  After 1 h on ice, cells are diluted
1:1 in PBS/EDTA/NaN<sub>3</sub> (PEN) and layered on to 10 ml PEN containing
2% Ficoll 400.  This is spun at 400 <i>g</i> for 5 min, the supernatant aspirated,
and pellet resuspended in small amt of PEN + 5% FBS, then cells distributed to
12 panning plates, each with 3 ml PEN + 5% FBS.  Cells take 2 h to attach
at RT, then plates washed with PEN + FBS.  400 &micro;l of 10 mM EDTA + 0.6% SDS
are added to each dish, and after 5 min lysates collected and treated as 
described by Hirt (J Mol Biol 26: 365, 1967).
<p>
DNA from Round 1 panning is transformed into MC1061/p3.  The colonies that grow
are pooled and amplified by growth in liq culture and subjected to spheroplast
fusion with COS cells in six 6-cm dishes (Round 2).  The panning with mAbs 
and the rescue are carried out as before.  In Round 3, transfected cells are
divided into portions and each portion treated with Ab or Abs specific to 
particular surface antigen.  Panning and transformation of different portions
produce colonies from which plasmid DNA is isolated by alkaline miniprep method.
The DNA from 300 &micro;l of culture is then used to transfect each 6-cm COS
cell culture for immunofluorescence analysis.
<p>
The HBP-ALL cell RNA was inserted into expression vector &pi;H3 and transfected
to COS cells by spheroplast fusion and produced about 300,000 recombinants.
After 3 d cells are detached (EDTA) and treated with pool of mAbs OKT-11,
anti-Leu-5b, and Coulter anti-T11 which are all anti-CD2 mAbs.  The antibody-treated
cells are distributed to dishes coated with affinity-purified sheep anti-mouse
IgG.  Attached cells presumably express CD2.  Gentle washing should remove 
nonadherents.  The Hirt procedure is then used to remove episomal DNA, which
is transfected into <i>E coli</i>  Colonies of <i>E coli</i> are pooled, and 
fused into COS cells.  These are subjected to Round 2 panning with pool of mAbs
as indicated above.  After all the panning with divided portions in Round 3, 
isolation of the plasmid shows a 1.5 kb insert in all six isolates found
expressing the protein.  This is subcloned into M13 mp19 in both directions
and dideoxy method used for sequencing.  A predicted 351 residue proteins is 
determined that appears to be a typical integral membrane protein with large
intracytoplasmic domain.  The matrix of signal sequence residue frequencies 
constructed by von Heine indicates that the cleavage site would be between
residues 19 (Ser) and 20 (Lys).  There are 26 uncharged residues forming the
membrane-spanning protion.   During review of this manuscript, Crumpton and
coworkers reported a CD2 clone where the sequence differs in three locations:
codon 247 shows A instead of C in 3rd position, which would code for Gln instead
of His;  there is an extra C in codon 319, causing a frameshit, and there is
a T instead of C at position 1341 in the 3&prime; UTR.
<p>
The COS cells were than transfected with CD2 expression plasmid and surface-labeled
with <sup>125</sup>I using lactoperoxidase 60 h after transfection.  After cell
extraction, immunoprecipitation using anti-CD2, then electrophoresis, there is
a prominent band in the transfect, but not in controls.  The calculated mean
of the molecular mass was 51 kDa, compared to the mean molecular mass of 54 kDa for
T lymphoblast and T cell line material.  Antigen from HPB-ALL cells were found to have
a mass of ~61 kDa.   Differences may be relted to different patterns of glycosylation
in different cell types.  COS cell extract also had a minor 38 kDa not present
in T cell or HPB-ALL cells.  The size agrees with predicted mass of the mature
but unglycosylated protein with mass 37.3 kDa.  Also a faint band of 28 kDa in both
COS and T cell immunoprecipitates is seen in controls and may not be CD2-related.
<p>
The CD2-expressing COS cells are treated for 60 min with purified MT910 (an
IgG<sub>1</sub> with &kappa; light chain) at 1 &micro;g/ml, or with purified
MB40.5 (an IgG<sub>1</sub> with &kappa; light chain) at same concentration.
The latter antibody recognizes a monomorphic HLA-ABC determinant and crossreacts
with African green monkey histocompatibility antigens.  It is an isotype-matched
antibody recognizing surface antigen of approximately the same abundance as the
CD2 antigne expressed in transfected cells.  Human and sheep RBC rosettes are 
observed in MB40.5, but not MT910.  Inhbition of rosettes was found iwth OKT11
Ab, but not with control antibodies of various types.
<p>
Human T cells also form rosettes with horse, pig, dog, goat, and rabbit RBCs
in addition to those of sheep.  They do not with rat or mouse.  Human erythrocytes
form auto-rosettes with human thymocytes. CD2-expressing COS cells were treated
with either MT910 or control MB40.5 and exposed to RBCs from all the animals given
above: rosettes formed expect with mouse and rat.  Rosette formation is blocked
if pre-treated with MT910 Ab, but not MB40.5.  Mouse RBCs show weak spontaneous 
binding to culture dish to MT910- and MB40.5-treated cdll by rat RBCs show no
detectable binding.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Cheek, T. R., Burgoyne, R. D.
 </span>
 <span class="year">
(1987)
 </span>
 <span class="jname">
J. Biol. Chem.
 </span>
 <span class="jvol">
262,
 </span>
 <span class="pages">
11663-11666.
 </span>
<p class="reftitl">
Cyclic AMP inhibits both nicotine-induced actin disassembly and catecholamine 
secretion from bovine adrenal chromaffin cells
<p class="Para1">
Chromaffin cells were isolated immediately and preincubated 1 h in a medium of 
Krebs-Ringer containing Ca and 0.5% BSA before use.  Cells were treated to log 
increases of nicotine and 3 responses were measured.  The first response was the 
change in intracellular calcium (using quin-2):  Maximal [intracellular Ca] was 
found to be 482 nM occurring at 0.5 mM nicotine (the highest dose used although 
the maximum Ca response appeared not to be reached).  F-actin disassembly and 
catecholamine release were also measured, but the maximal (13% maximal release 
of CAs of the total CA present and maximum reduction of F-actin from 47% to 34% 
of total cellular actin) responses were found to occur at 10 &micro;M nicotine. 
Indeed the responses of F-actin disassembly and CA release were reduced 30% when 
nicotine dose was increased to 50 &micro;M while the Ca increase response was 
still on the rise (cells were treated at room temp for 6 min).
<p>
A previous report shows the forskolin treatment of chromaffin cells leads to 
increases in cAMP, which in turn inhibits the CA release elicited by nicotine 
but not that elicited by high K<sup>+</sup>.  This inhibition must be by a step 
distal to Ca<sup>2+</sup> entry.  Chromaffin cells were incubated with 30 
&micro;M forskolin for 6 min, 500 &micro;g/ml cholera toxin for 3 hr (acts on 
the Gs protein), or 2.3 mM 8-Br-cAMP for 4 h prior to secretagogue treatment 
(either nicotine or K<sup>+</sup>).  The forskolin, cholera toxin, and 8-Br-cAMP 
all raised the cAMP levels to different degrees, with forskolin more effective 
than cholera toxin for example, and measurement of nicotine-induced CA release 
was 10%, 40% and 45% of the control release for forskolin, cholera toxin, and 
the cAMP analogue, resp.  This difference is reflected in the intracellular cAMP 
levels.  K<sup>+</sup> caused a CA release near control levels, showing that 
cAMP levels did not effect the effect of K<sup>+</sup> to elicit CA release.  In 
the event adenosine receptors were also affected by 8-Br-cAMP, a study of 
nicotine- and K<sup>+</sup>-induced CA release with prior treatment with 100 
&micro;M adenosine for 4 h indicates that CA release for both secretagogues was 
actually enhanced above controls.
<p>
To examine the possibility that forskolin was working by inhibiting cortical 
actin disassembly in the nicotine-induced response, cells were challenged 10 
&micro;M nicotine after pretreatment with log increases in forskolin dose.  The 
three responses were again measured and % inhibition of response plotted against 
the log[forskolin].  It is noted that nicotine-induced F-actin disassembly is 
inhibited by forskolin and that the curve overlaps with that for the inhibition 
of secretion by forskolin, showing a close link between these events.  The 
dose-response curve for inhibition of rise in [intracellular Ca], although 
close, is less sensitive to forskolin-induced inhibition that secretion or actin 
disassembly.
<p>
<b>Discussion</b>.  A notion has previously been put forth that cortical actin 
filaments serve as a barrier to secretory vesicle fusion to the membrane to 
effect CA release.  This results from a nicotine-induced increase in 
intracellular Ca concentration.  This model is consistent with results that show 
that nicotine induces F-actin disassembly, and actin-destablizing agents enhance 
while actin-stabilizing agents reduce both basal and Ca<sup>2+</sup>-induced CA 
secretion from permeabilized chromaffin cells.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Witters, L. A., Blackshear, P. J.
 </span>
 <span class="year">
(1987)
 </span>
 <span class="jname">
Meth. Enzymol.
 </span>
 <span class="jvol">
141,
 </span>
 <span class="pages">
412-428.
 </span>
<p class="reftitl">
Protein kinase C-mediated phosphorylation in intact cells
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Greene, L. A., Aletta, J. M., Rukenstein, A., Green, S. H.
 </span>
 <span class="year">
(1987)
 </span>
 <span class="jname">
Meth. Enzymol.
 </span>
 <span class="jvol">
147,
 </span>
 <span class="pages">
207-216.
 </span>
<p class="reftitl">
PC12 pheochromocytoma cells:  culture, nerve growth factor treatment, and 
experimental exploitation
<p class="Para1">
<b>REVIEW</b>.  PC12 clonal cell line has been used as a model for studies on 
chromaffin cells and neurons.  Established in 1975 from a transplantable rat 
adrenal medullary pheochromocytoma.  When treated with the neuronotrophic NGF 
they shift from a proliferating chromaffin/pheochromocytoma phenotype to a 
nonproliferating, neurite-bearing sympathetic-like neuron.
<p>
PC12 culture medium as originally described is 85% RPMI 1640, 10% 
heat-inactivated horse serum, 5% fetal calf serum, 25 U/ml penicillin, and 25 
&micro;g/ml streptomycin.  The nature of the horse serum is highly important: 
donor grade serum from some suppliers is good but not all batches perform 
adequately and pre-testing is recommended.
<p>
Some PC12 cells have been adapted to DME-based medium, but with a lower growth 
rate.  Such adaptations should be made gradually, but the integrity of the 
original cell line in the RPMI 1640 medium is important to minimize variation of 
cell phenotype from one lab to the next.  Much like neurons and chromaffin 
cells, PC12 cells attach poorly to tissue culture plastic or glass and prefer to 
autoassociate.  Collagen- or polyLys- or polyOrn-coated dishes will permit 
formation of monolayers, but the polyamino acid substrates will deteriorate with 
time.  Collagen is therefore the substrate of choice for cells to be plated for 
&gt; 3 d.  The Bornstein method of coating dishes with collagen is given.  For 
certain short-term experiments, polyLys- and polyOrn-coated coverslips have been 
used in fluorescence or other optical studies fo PC12 cells.  Coverslips are 
exposed to 50 &micro;g/ml polyLys HBr (membrane-sterilized) for 60 min, washed 
extensively (4 times) with sterile water and allowed to air dry.
<p>
Cultures are maintained normally by changing the complete growth medium three 
times weekly (MWF). 2/3 of the medium is exchanged since the presence of 
conditioned medium facilitates growth.  A 36&deg; incubator (chromaffin and PC12 
cells prefer the cooler temperature) is used with 7.5% CO<sub>2</sub> atmosphere.  At 
near-confluence, subculturing is done 1:3 or 1:4 by trituration of the small 
clumps and single cells.  Doubling occurs every 2.5 to 4 d, meaning cells are 
subcultured once a week.
<p>
For studies examining neurite growth of cultures, ethanol-applied collagen is a 
substrate used with the most satisfactory results.  Various dilutions of the 
substrate should be done to achieve optimal results.  Cell density for looking 
at neurite growth is also important:  too high and neurite outgrowth is 
suppressed;  too low and cell survival is a problem.  Good results are found 
with 200,000 to 800,000 cells per 35 mm dish.  For lower cell densities, used of 
a medium conditioned by high density PC12 cell cultures is helpful.
<p>
Since PC12 cells adhere strongly to one another, they form clumps which are 
difficult to dissociate.  Since certain experiments depend upon a large number 
of single cells present and clumps absent, a procedure has been devised to 
produce suspensions in which over half of cells are present as single forms with 
very few clumps of greater than 4 cells:  
<ol style="list-style-type:lower-roman;">
<li> EGTA from a 0.1 M stock (made up in normal, not Ca<sup>2+</sup>-free 
medium) is added to the medium to a final 1 mM and left for at least 6 h or 
overnight without a serious loss of viability
<li> the cells are washed three times with medium diluted 1:1 with sterile 
water, but not containing serum
<li> the cells are now triturated vigorously (with Pasteur pipet)
</ol>
<p>
When plated at a low density (&lt;100,000 in a 100 mm dish), plating efficiency 
will be low unless a medium by dense PC12 cultures is used.  Trypsin (0.1 to 
0.5%) may be used for 10 min at 37&deg; if single cells plated at low density 
are not desired to have a high plating efficiency.
<p>
PC12 cells may be grown in suspension in large numbers with spinner bottles 
containing complete medium.  Cells should be inoculated to a high density of 
&gt;106 cells/ml.  Fresh medium is added by degrees whenever the pH indicator 
changes color.
<p>
To avoid phenotypic and genotypic variants that will likely arise from continued 
cell divisions of the line, homogeneity of the line is recommended to be 
preserved as follows.
<ol>
<li> After a given number of generations, cells should be replaced with frozen 
cell stocks from earlier passages.  This lab currently employs cells between 
passages 20 and 40.  
<li> Cell growth conditions should be maintained as uniform as possible; 
otherwise extremes in the environment will introduce selection pressure leading 
to the emergence of a major phenotypic variant. 
<li>All maintenance cultures should be grown on collagen or other suitable 
substrate:  this avoids selection of variants with a higher adhesivity than the 
parent cells.  
</ol>
<p>
Cells can be frozen in complete medium containing 10% DMSO, with slow-freezing 
at -70&deg;, then long-term storage in liquid nitrogen.
<p>
[Other subjects addressed:  use of PC12 mutants and mutagenesis techniues; use 
of neurite-enriched cultures and a method for separating neurites from the cell 
bodies.]
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Suganama, M., Fujiki, H., Suguri, H., Yoshizawa, S., Hirota,
M., Nakayasu, M., Ojika, M., Wakamatsu, K., Yamada, K., Sugimura, T.
 </span>
 <span class="year">
(1988)
 </span>
 <span class="jname">
Proc.&nbsp;Natl.&nbsp;Acad.&nbsp;Sci.&nbsp;USA
 </span>
 <span class="jvol">
85,
 </span>
 <span class="pages">
1768-1771.
 </span>
<p class="reftitl">
Okadaic acid:  an additional non-phorbol-12-tetradecanoate-13-acetate-type
tumor promoter
<p class="Para1">
Non-TPA-type tumor promoters do not bind phorbol ester
receptors.  Palytoxin and thapsigargin are of such type,
and they do not activate PK-C in vitro.  Okadaic acid is
a toxic polyether isolated from black sponge, <i>Halichondria okadai</i>,
and from a sponge, <i>Halichondria melanodocia.</i>  Okadaic acid and
dinophysistoxin-1 (35-methylokadaic acid) are implicated as the causative
agents of diarrhetic shellfish poisoning resulting from ingestion of mussels
and scallops contaminated by presence of dinoflagellates which synthesize these
compounds.
<p>
Tumor promoting activity was assessed in mice divided into
four group for 30 week treatment:  the
first group was treated with 7,12-dimethylbenz(a)anthracene
(DMBA) and okadaic acid, the second with DMBA and teleocidin,
a third with DMBA alone, and the last with okadaic acid alone.
The DMBA+okadaic acid group showed 5 tumors in 4/15 mice by week 5,
and 93% tumor incidence by week 16.  This % fell slightly after
week 17 and was 80% by week 30.  The DMBA+teleociding group all
had tumors by week 15.  The avg number of tumors per mouse was 2.6
for the first group and 6.8 for the second.  In the DMBA and okadaic
acid alone groups, tumor incidence was 6.7% at the end of the study
for both groups.
<p>
The type of tumors present in the first two groups (39 in
the DMBA+okadaic acid and 93 in the DMBA+teleocidin) were histologically
diagnosed.  Nearly all the tumors in both groups were papillomas (36/39
in group 1 and 89/95 in group 2).  2 of the remaining 3 in group 1 were
squamous cell carcinoma and the remaining 4 of group 2 were also of that
type.  One of the tumors in the group 1 study was a sarcoma.  Hence
teleocidin and okadaic acid appear to show the same potencies for
production of the same tumor.
<p>
Competitive binding studies using [<sup>3</sup>H]TPA were done
using increasing concentrations of okadaic acid or teleocidin in mouse particulate
fractions.  Specific binding by TPA was
not inhibited by okadaic acid while teleocidin clearly inhibited this binding.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Foley JP
 </span>
 <span class="year">
(1987)
 </span>
 <span class="jname">
Anal. Chem.
 </span>
 <span class="jvol">
59,
 </span>
 <span class="pages">
1984-1987.
 </span>
<p class="reftitl">
Equations for Chromatographic Peak Modeling and Calculation of Peak Area
<p class="Para1">
Two approaches generally take for chromatographic peak characterization.
<ol>
<li>Gaussian peak shape assumption
<li>Calculations of statistical moments
</ol>
<p>
The first is grossly inaccurate and the second is frequently imprecise and
moderately inaccurate due to noise, uncertainties in peak start/stop assignments
and baseline drift.  Higher moments are sensitive to peak tailing.  Recently the
<b>exponentially modified Gaussian</b> (EMG) peak shape model was introduced
and helps with Gaussian and peak tailing fits.  The adequacy of using EMG model
has to be verified however.
<p>
The EMG function results from a convolution of a Gaussian function and an
exponential decay function and can be expressed in many ways.  The one used here
is:
<p class="eqnserif">
EMG(<i>t</i>) = <i>A/&tau;</i> exp [&frac12; &times; 
(<i>&sigma;</i><sub>G</sub>/<i>&tau;</i>)<sup>2</sup>
&minus; (<i>t</i> &minus; <i>t</i><sub>G</sub>) / &tau;] 
&nbsp;&nbsp;&int;<sub>&minus;&infin;</sub><sup><i>Z</i></sup>
exp (&minus;<i>y</i><sup>2</sup> / 2) / (2<i>&pi;</i>)<sup>&frac12;</sup> <i>dy</i>
<p>
where <i>A</i> is peak area, t<sub>G</sub> and &sigma;<sub>G</sub> are retention 
time and standard deviation of Gaussian function, &tau; is time constant of 
exponential decay function, and <i>z</i> = (<i>t</i> &minus; 
<i>t</i><sub>G</sub>)/<i>&sigma;</i><sub>G</sub> &minus; 
&sigma;<sub>G</sub>/&tau;.  Accurate polynomial approximation is used to evaluate
the integral.  Ratio &tau;/&sigma;<sub>G</sub> is fundamental measure of asymmetry;
when <i>&tau;/&sigma;</i><sub>G</sub> increases, tailing increases and when the
ratio approaches 0, the approaches that of a Gaussian.  Values of <i>A</i> = 1, 
<i>t</i><sub>G</sub> = 100, and <i>&sigma;</i><sub>G</sub> = 5 were used in
peak generation.  The <i>&tau;/&sigma;</i><sub>G</sub> ratio was varied from
0.3 to 4.2 in 0.1 increments which produces data equivalent to 40 peaks.
<p>
Equations to calculate accurately the peak area were obtained by least-squares
fitting of <i>A</i><sub>true</sub>/<i>A</i><sub>G</sub> vs. <i>b/a</i> from the
ratio going from 0.3 to 4.2 where <i>A</i><sub>true</sub> is the area from the
above equation, <i>b/a</i> is empirical assymetry factor, <i>A</i><sub>G</sub>
is area calculated using Gaussian equation:
<p class="eqnserif">
<i>A</i><sub>G</sub> = 
<i>C</i><sub>r</sub><i>h</i><sub>p</sub><i>W</i><sub>r</sub>&nbsp;&nbsp;[2]
<p>
where <i>A</i><sub>G</sub> is the peak area, <i>r</i> is the peak height fraction,
<i>C</i><sub>r</sub> = &frac12;[<i>&pi;</i> / ln(1/<i>r</i>)]<sup>&frac12;</sup>, 
<i>h</i><sub>p</sub> = peak height, and <i>W</i><sub>r</sub> = peak width at
specified peak height fraction.  General form of fitted equations is:
<p class="eqnserif">
<i>A</i><sub>true</sub> / <i>A</i><sub>G</sub> = 
<i>C</i><sub>1</sub>(<i>b/a</i>)<sup><i>C&prime;</i></sup>&nbsp;&nbsp;[3]
<p>
where <i>C</i><sub>1</sub> and <i>C&prime;</i> are fitting constants, and 
<i>b/a</i> is peak asymmetry at the same peak height fraction as <i>W</i><sub>r</sub>.
Substituting Eq 2 into Eq 3 and dropping the <q>true</q> subscript from 
<i>A</i><sub>true</sub> provides the equation:
<p class="eqnserif">
<i>A</i> = 
<i>Ch</i><sub>p</sub><i>W</i><sub>r</sub>(<i>b/a</i>)<sup><i>C&prime;</i></sup>&nbsp;&nbsp;[4]
<p>
where <i>C = C</i><sub>r</sub><i>C</i><sub>1</sub>.  Experimental peaks were
obtained on RPLC using acetophenone, anisole, and toluene from different column
manufacturers and <i>k&prime;</i> values ranging from 2-6.  A Varian Model 5000
LC and UV detection (254 nm) was used with 1.1 s time constant.  Relative error
for Gaussian area equations (Eq 2) is plotted as %relative error vs &tau;/&sigma;
for <i>W</i><sub>r</sub> = 0.10, 0.30, 0.50, and 0.75 for the range of &tau;/&sigma;
0.0 to 4.0.  Gaussian equations thus have an inaccuracy since they do not account
for peak asymmetry.  While the plot shows area is overestimated for <i>W</i><sub>0.1</sub>,
they are underestimated for <i>W</i><sub>0.3</sub>, <i>W</i><sub>0.5</sub>,
and <i>W</i><sub>0.75</sub>.  When <i>r</i> = 0.25, the Gaussian equation is
accurate within 1% for the entire range of EMG peaks.  The least-squares fitting
was used to obtain empirical area equations (Eq 4) at peak height fractions
<i>r</i> = 0.1, 0.3, 0.5, and 0.75.  The following equations can be used to 
assess peak fit:
<table>
<tr><th>Equation <th>% Relative Error  <th>% Relative SD
<tr><td><i>A</i> = 0.586 <i>h</i><sub>p</sub> <i>W</i><sub>0.1</sub> 
(<i>b/a</i>)<sup>&minus;0.133</sup>
  <td> &plusmn;0.50  <td>1.44
<tr><td><i>A</i> = 0.753 <i>h</i><sub>p</sub> <i>W</i><sub>0.25</sub> 
(<i>b/a</i>)<sup>&minus;0.133</sup>
  <td> &minus;1.0, +0.6  <td>1.41
<tr><td><i>A</i> = 1.07 <i>h</i><sub>p</sub> <i>W</i><sub>0.5</sub> 
(<i>b/a</i>)<sup>+0.235</sup>
  <td> &minus;1.2, +1.0  <td>1.49
<tr><td><i>A</i> = 1.64 <i>h</i><sub>p</sub> <i>W</i><sub>0.75</sub> 
(<i>b/a</i>)<sup>+0.717</sup>
  <td> &minus;1.1, +0.6  <td>2.01
</table>
<p>
Thus, once a fit of unknowns to the Gaussian or EMG models has been 
confirmed by the area test, then equations by Foley and Dorsey (1983)
can be used to measure column efficiency, peak variance, and statistical
moments.  Alternatively, statistical moment calculations by Anderson and
Walters (1984) can be used.  Note that for <i>r</i> = 0.25 there is no
asymmetry factor to modify the equstion, accurate for Gaussian and EMG-type
peaks.  Other Gaussian area equations will not be accurate for non-Gaussian
peaks.  Note that non-chromatographic signal peaks such as in FIA and graphite
furnace AAS show strong resemblance to EMG peaks
<!-- ============================================================= -->
<p class="tags">
CD20, lymphocyte differentiation antigen
<p class="ref">
 <span class="authors">
Tedder TF, Streubli M, Schlossman SF, Saito H
 </span>
 <span class="year">
(1988)
 </span>
 <span class="jname">
Proc. Natl. Acad. Sci. USA
 </span>
 <span class="jvol">
85,
 </span>
 <span class="pages">
208-212.
 </span>
<p class="reftitl">
Isolation and structure of a cDNA encoding the B1 (CD20) cell-surface antigen of 
human B lymphocytes
<p class="Para1">
Lymphocyte antigen B1 (CD20) is expressed in B lymphocytes in pre-B cell stages
and when they become plasma cells.  Anti-B1 mAbs with B cells show they lose the
ability to proliferate in response to known mitogens, by exposure to EBV,
or to activated T cells;  the progression through S/G2 and M stages is stopped.
The mAbs also prevent differentiation  The mAb 1F7 is reported to augment B
lymphocyte function:  thus B1 (CD20) is likely involved in cell cycle progression
and differentiation.  It is a highly hydrophobic phosphoprotein with three
isoforms:  a major 33 kDa form and less prevalent 34.5 and 36 kDa, which might
be phosphorylated forms.
<p>
In this paper, poly(A)<sup>+</sup> RNA from microsomes of B1-positive 
lymphoblastoid cell line Raji was size-fractionated on sucrose density gradient 
Fractions were translated in vitro in presence of [<sup>35</sup>S]Met and 
immunoprecipitation done with anti-B1a mAb (at 50 &micro;g/ml) and using protein 
A-Sepharose.  The peak fraction where the most radioactivity was found was with
mRNA having a length of 2.2-3.0 kb.  This indicates that the mRNA is 2- to 3-fold
longer than that required to encode a protein of mass 33 kDa.  The mRNA required
to encode HLA class I molecules is about 45 kDa, and its mRNA is about 2000 bases
long.
<p>
<sup>32</sup>P-labeled cDNA probes were made from B1 mRNA and size-fractionated
mRNA from T cell line HSB-2.  These probes were used in screening a human 
tonsillar lymphocytes cDNA library in &lambda; phage.  40,000 plaques were screened
and 261 hybridized with Raji cDNA but not HSB-2 cDNA.  58 of these hybridized
with DNA encoding known B-lymphocyte proteins.  Insert were isolated from 60 of
remaining 203 clones, nick-translated, and used to make groups of cross-hybridzing
clones.  Nine groups containing more than one clone were identified (groups A through
I).  Cytoplasmic RNA was isolated from cell lines either positive or negative
for cell surface expression of B1 and used for dot-blot hybridization.  Group D
clones hybridized to RNA from B1-positive cell lines Raji, Bjab, and Daudi, but
not to RNA from B1-negative cell lines Nalm-6, PB697, HSB-2, HuT-78, CEM, Jurkat,
K-562, and U937.  Hybridization patterns of remaining groups did not correlate
with B1 expression.  Group D was further characterized as the pattern of expression
is unique among known B lymphocyte surface antigens.  Longest cDNA (2.8 kb)
from group D clones was subcloned into pSP64, creating the plasmid pB1-21.
<p>
A restriction map (using triple digests) was made from pB1-21 and Maxam and 
Gilbert sequencing done.  1153 bp at the 5&prime; end were sequenced; also the
extreme 3&prime; end was also sequenced and poly(A) signal (AATAAA) also found,
ending with short strech of A residues.  An open reading frame of 891 nt was
found, and it shows a translation intiation consensus of ANNATG.  297 amino acids
are deduced.  Kyte and Doolittle hydropathy shows three stretches of strongly
hydrophobic regions.  The cDNA does not show a typical signal peptide.  There are
three potential N-glycosylation sites at positions 9, 171, 293.  A search of PIR
shows no significant sequence homology.
<p>
With the pSP64 expression vector, the cDNA was in vitro translated and compared
with the metabolically labeled B1 protein from Raji cells.  Both molecules migrated
with <i>M</i><sub>r</sub> 33,000 on SDS-PAGE.  The anti-B1 mAb did not immunoprecipitate
the pB1-21 product 

<!-- ============================================================= -->
<p class="tags">
lipoprotein lipase receptor (LDL-R)
<p class="ref">
 <span class="authors">
Herz J, Hamann U, Rogne S, Myklebost O, Gausepohl H, Stanely KK
 </span>
 <span class="year">
(1988)
 </span>
 <span class="jname">
EMBO J.
 </span>
 <span class="jvol">
7,
 </span>
 <span class="pages">
4119-4127.
 </span>
<p class="reftitl">
Surface location and high affinity for calcium of a 500-kd liver membrane 
protein closely related to the LDL-receptor suggest a physiological role as 
lipoprotein receptor
<p class="Para1">
Studies of LDL receptor (LDLR) have contributed a great deal to understanding of 
cholesterol homeostasis in normal and diseased states.  LDLR cannot be only 
receptor for specificity to apolipoprotein E (apoE) as chylomicron remnants, 
HDLc and &beta;-VLDL particles (also containing apoE) are not elevated in 
animals or patients with defective LDLR.  Two low MW apoE-binding proteins 
expressed in liver have been found using affinity chromatography and ligand 
blooting, but were not apoE receptors.  Different receptors may be involved in 
apo E binding.
<p>
Both molecular cloning and site-directed mutagenesis of LDLR and one of its
two ligands, apoE, show that ligand and receptor involve basic region of ligand,
and acidic region on each of seven Cys-rich domains on receptor N-terminus.
Homologous seqeunce with same pattern of Cys residues (class A Cys-rich motifs)
are in terminal complement components, and in complement factor I.  Initially
a mouse lymphocyte cDNA library was probed and a partial cDNA clone with eight
of the motifs arranged in similar way to apoB,E-binding doman of LDLR was used
for cDNA walking.  Eight clones coding for 15 kb cDNA were found from human
liver cDNA and constructed in plasmid and &lambda; vectors.  Overlapping cDNA
segments from the clones eventually gave rise to an 11 kb total length.
One of the 3&prime; end clones had an unspliced 359 bp intron.  Another rescreening
produced clones of which none had the intron.  The assembled sequence is
4544 amino acids long with 3&prime;-UTR which is 798 bp and a AAATAAAA polyadenylation
signal 4 bp up from the oligo(dT) priming site.  The total cDNA is 14,896 bp.
Predicted mass of the polypeptide is 503 kDa for an unglycosylated product.
If the 52 kDa extracellular sites are N-glycosylated, the mass increases 150 kDa.
High amino acid sequence identity (40% in class A regions) is shown by the LDLR
and EGF precursor but there is little homology at nucleotide level: they must
arise from different genes.
<p>
Kyte and Doolittle hydropathy predicts two hydrophobic regions at the N- and
C-termini.  One shows cleavable signal peptide of 19 aa, and the other is a 
membrane spanning segment:  there is a His residue on the N-side of the segment,
and 3 positive residues on the C-side.  On the N-side the domain is large and
filled with Cys residues.  Immunoprecipitated protein reduced with thiols for
SDS-PAGE shows a mobility shift compared with protein not treated with thiols, 
indicating that Cys residues are disulfides and likely on extracellular side of
membrane;  a signal peptide usually used for membrane and secreted proteins would
indicate this property too.  The C-terminal side of the transmembrane segment has
about 100 amino acids, which is twice the length of LDLR cytoplasmic tail and it
has a duplication of Tyr residue (position 807), providing the function in LDLR
for clustering into coated pits.  Presence of NPVY with Asn and Pro usually form
a surface turn structure in protein and might be involved in coat component 
interaction.  High conservation of this region of cytoplasmic tail indicates
this protein might be clustered into pits and endocytosed like LDLR.
<p>
In the predicted extracellular portion of the structure there are three types
of repeats also found in LDLR and EGF precursor.  Six Cys residues in a 40-residue
span only seen in LDLR and in complement proteins.  The class A motif is 
a pattern of Cys residues spaced with other conserved amino acids.


<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Arrondo JLR, Young NM, Mantsch HH
 </span>
 <span class="year">
(1988)
 </span>
 <span class="jname">
Biochim. Biophys. Acta
 </span>
 <span class="jvol">
952,
 </span>
 <span class="pages">
261-268.
 </span>
<p class="reftitl">
The solution structure of concanavalin A probed FT-IR spectroscopy
<p class="Para1">
Infrared spectroscopy of concanavalin A in the amide I region
was done for a 0.68 mM solution in D<sub>2</sub>O and H<sub>2</sub>O,
pD=5.8.  The amide I region covers 1750-1600 cm<sup>&minus;1</sup>.
The maximum in both solvents was the same, but the peak in
H<sub>2</sub>O was broader.  This difference is not in one of
conformation, but maybe due to D/H exchange in amide groups.
Amide I bands consist of several individual bands assignable to
different types of secondary structure.  Fourier self-deconvolution
followed by Fourier derivation will narrow bands to reveals component
elements (this process is sometimes called &ldquo;resolution enhancement&rdquo;,
but this not to be confused with instrumental resolution).
Subsequent to the deconvolution (but not derivation) of the broad
peaks, the following bands are revealed: a main peak at 1636
cm<sup>&minus;1</sup> in both solvents, and bands at 1624,
1649, 1658, 1671, 1682, and 1696 cm<sup>&minus;1</sup>.
Derivation did not reveal any new peaks.
<p>
Concanavalin A binds sugars with D-arabino configuration at C3, C4,
and C5 and on hydroxyls at C3, C4, C6.  It was incubated
with excess methyl &alpha;-D-mannoside and only minor changes were
seen in the amide I band, with a shift of the main peak from
1636 to 1635 cm<sup>&minus;1</sup> and a relative increase in
intensities at component bands between 1645 and 1685 cm<sup>&minus;1</sup>
(these are amide vibrations from proteins having no repetitive structure).
These spectral changes do not show secondary structure changes
by minor local conformation changes in protein outer part.
<p>
Con A was de-metallized of its Mn- and Ca-binding and there was
an increase in the amide I band, displacement to a lower wavenumber.
Derivative spectra show that there was no shift in the 1624 and
1694 cm<sup>&minus;1</sup> bands.  The 1636 cm<sup>&minus;1</sup> band
shift to 1634 cm<sup>&minus;1</sup>.  There are also small changes
in the 1650-1690 cm<sup>&minus;1</sup> region, where loops and
&beta;-turns are reported.  All data indicate little change in
secondary structure and that the metal cations influence tertiary
structure predominantly.  A thermal denaturation study
(plots of frequency in cm<sup>&minus;1</sup> vs. temperature from
40-90&deg;) of ConA shows a transition point 85&deg; when the metal
is present, and 63&deg; for the cation-depleted apoprotein, consistent
with the amide I band shift rom 1635 cm<sup>&minus;1</sup> to 1623
cm<sup>&minus;1</sup>.
<p>
ConA dimers form tetramers with the pH &gt; 6, and when pH &gt; 7,
these begin to aggregate.  Spectra were taken at pD 7.6 after 30&nbsp;min
from change in pD, and there were no differences with spectra at pD 5.8
At pH 6 after 6 h, spectra were changed, and even more so at 30 h.
Spectral deconvolution or derivation show that the changes are not
due to appearance/disappearance of (new) bands, but rather to changes
in the intensity of existing bands.  There is a decrease in the
1636 cm<sup>&minus;1</sup> band as the aggregated tetramer forms
from the dimer.
<p>
Peptide side chain vibrations are measured in the spectra too.
The C-C stretch in the tyrosine aromatic ring (there are 7 tyrosines
in ConA) is visible in resolution-enhanced spectra.  This band
changes when ConA is de-metallized or when ConA aggregates.
In D<sub>2</sub> solvent native ConA shows a Tyr band at
1515.8 cm<sup>&minus;1</sup> while cation-depleted protein
is at 1516.8 cm<sup>&minus;1</sup>, and aggregated protein
the band shifts to 1512.2 cm<sup>&minus;1</sup>.  There is no
change in the peak with addition of excess methyl &alpha;-D-mannoside.
<p>
ConA consists of two anti-parallel &beta;-sheets
making up 60% of the residues.  IR spectra of ConA in aqueous solution
reports a 1636 cm<sup>&minus;1</sup> typical of &beta;-structure.

<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Czech, M. P., Klarlund, J. K., Yagaloff, K. A., Bradford, A. P., Lewis, R. E.
 </span>
 <span class="year">
(1988)
 </span>
 <span class="jname">
J. Biol. Chem.
 </span>
 <span class="jvol">
263,
 </span>
 <span class="pages">
11017-11020.
 </span>
<p class="reftitl">
Insulin receptor signalling.  Activation of multiple serine kinases
<!-- ============================================================= -->
<p class="tags">
polymerase chain reaction (PCR), 5' and 3' cDNA ends, cDNA cloning, 
low-abundance mRNAs, <i>int</i>-2 gene
<p class="ref">
 <span class="authors">
Frohman MA, Dush MK, Martin GR
 </span>
 <span class="year">
(1988)
 </span>
 <span class="jname">
Proc. Natl. Acad. Sci. USA
 </span>
 <span class="jvol">
85,
 </span>
 <span class="pages">
8998-9002.
 </span>
<p class="reftitl">
Rapid production of full-length cDNAs from rare transcripts: Amplification using 
a single gene-specific oligonucleotide primer
<p>
Getting full-length copies of low abundance mRNAs is not a trivial matter.
A technique is used here called RACE, for rapid amplification of cDNA ends
and its first demonstration is to clone transcripts of <i>int</i>-2 gene,
a low abundance gene in early mouse embryo.
<p>
The 3&prime; end of a cDNA is obtained from a population of RNA molecules by 
reverse transcription to create the negative strand.  The primer is a 35-base 
oligo with 17 dT bases and an adaptor featuring three endonuclease recognition 
sites which are not frequent in genomic DNA;  this is called a 
(dT)<sub>17</sub>-adapter.  A gene-specific amplification primer (3&prime;amp) 
which is part of the sense (mRNA) strand will anneal to a  small segment of the 
first (&minus;) strand product (generally &lt; 1 ng) and extend to generate 
complementary second (+) strand.  PCR iteration with the 3&prime;amp and adapter 
primer (this is a primer which omits the dT bases since they do not pair well
at temperatures designed to prevent mismatch) will amplify the region of the 
cDNA at the mRNA 3&prime; end (3&prime; end RACE).
<p>
For 5&prime; end RACE, a gene-specific primer (5RT) is used in reverse ttranscription.
Primer-extended products are separated from excess primer and a poly(A) tail
added using dATP and TdT enzyme.  Then a (dT)<sub>17</sub>-adapter primer
is used to perform second (+) strand synthesis.  Products are amplified using
only the adapter as primer and a 5&prime;amp primer, which anneals to a
sequence which is 3&prime; to the extension primer sequence:  the internal primer
increases specificity and efficiency of amplification.  This would bind only 
to cDNAs of interest while 5RT binds to all (+) strand cDNAs.
<p>
To test the method, cDNA copies of the <i>int</i>-2 mRNA were tested, which 
makes only 2 copies per cell, and whose complete sequence is known.  It 
generates multiple transcripts with alternate start sites.  The 3&prime;amp-D 
primer is specific for long <i>int</i>-2 mRNA species (there are four total). 
The 3&prime; end cDNA amplification would be about 1050 bp long, and DNA of that 
size is seen in gels and identity established using Southern blot with 
<i>int</i>-2 cDNA probe.  Other unexpected sizes were found, but they were 
single-stranded molecules.  Other products seen in EtBr staining did not 
hybridize with <i>int</i>-2 probe however:  they likely resulted from mispriming 
and temperature optimization minimizes their appearance.   In the control, bands 
are also apparent but do not hybridize with <i>int</i>-2 probe.  Another 
3&prime;amp primer, 3&prime;amp-U which is <q>upstream</q> (5&prime; to) 
3&prime;amp-D overlap with long and short <i>int</i>-2 transcripts and its 
extension would produce products of 480 and 1580 bp for the short and long 
transcripts types.  But only the 480 bp product could be found on EtBr stained 
gels, although Southern analysis detects both 480 and 1580 bp!  This is because 
the 480 bp product is 10-fold more abundant than the 1580 bp, although Northern 
blots show that the short transcripts are only 2-fold more abundant than long 
transcripts.  Thus the relative abundance of mRNA transcripts may not be shown 
in PCR because of differences in the amplification efficiency as the length of 
extension increases.
<p>
<i>int</i>-2 transcripts also have two different 5&prime;-UTR types:  either
the full length of exon 1 (1A + 1B) or the 5&prime; part of exon 1 missing,
that is the transcript has only exon 1B.  The full exon 1 is a class A transcript
and the truncated is class B.  In doing 5&prime; RACE, the expected result is
~690 and ~930 bp (the truncation is 240 bp).  Note that Northerns show that
class A (longer) is less abundant than class B.  Both of these 690 and 930 bp
fragments are visible on Southerns but they are not always detectable using 
EtBr staning.
<p>
Specific products of 3&prime; and 5&prime;-end RACE were isolated from agarose
gels and cloned.  Separation of different classes of cDNAs by gel electrophoresis
before cloning is useful in obtaining cDNA representing class A mRNAs.  The clones
represented all four transcripts expressed in the <i>int</i>-2 gene, longest
being 2.9 kb.
<p>
The following should be noted concerning the use of RACE protocol:
<ol style="list-style-type:lower-roman;">
<li>In some mRNA species not all 3&prime;-end cDNA species are full length.
This may be due to the (dT)<sub>17</sub>-adapter primer binding to A-rich region
within the coding sequence (for example, in one gene, there is AAAGAAGAAAAAAGAA
upstream of poly(A) tail.  By decreasing concentration of this (dT)-adapter
primer to 15 ng/&micro;g mRNA, this can minimize this occurrence.
<li>Some reaction products are single-stranded cDNA species (not amplified),
and thus will not be digested with restriction enzymes.  These can be eliminated
with mung bean nuclease digestion.  These single stranded products can be reduced
by altering buffer conditions as well, indicating they are derived from a
double-stranded state.  While ss DNA is not common in PCR, use of a gene-specific
primer + primer binding to all cDNA could results in more ss molecules, or 
to mismatched pairing of some strands in final cycle of PCR.
<li>Relative amount of longer cDNA products can be increased by increasing 
extension time of second (+) strand synthesis.  Small (2&deg; changes in annealing
temperature cause significant changes in ratio of specific to nonspecific
cDNA amplification.  An initial conservative annealing temperature of 50&deg; is 
recommended for the early studies, and optimum adjusted to obtain cDNA of interest
according to Southern analysis.
</ol>
<!-- ============================================================= -->
<p class="tags">
CD20
<p class="ref">
 <span class="authors">
Einfeld DA, Brown JP, Valentine MA, Clark EA, Ledbetter JA
 </span>
 <span class="year">
(1988)
 </span>
 <span class="jname">
EMBO J.
 </span>
 <span class="jvol">
7,
 </span>
 <span class="pages">
711-717
 </span>
<p class="reftitl">
Molecular cloning of the human B cell CD20 receptor predicts a hydrophobic 
protein with multiple transmembrane domains
<p class="Para1">
[Note that Stamenkovic &amp; Seed also reported CD20 cloning and sequencing
independently.]
<p>
CD20 is a hydrophobic phosphoprotein about 35 kDa size and expressed in mature
B lymphocytes and also found in lymphomas and chronic lymphocytic leukemias.
In quiescent B cells it is present at about 150,000 molecules per cells circulating
in blood and in lymphoid organs, and 4-fold higher in activated B cells in germinal
centers.  CD20 likely functions in B cell activation  since its stimulation 
results in G<sub>0</sub> &rarr; G<sub>1</sub> transition  when activated by
the 1F5 anti-CD20 mAb.  1F5 with B cell growth factor (a 12 kDa protein) causes
B cells to enter the cell cycle for active proliferation.  It induces c-<i>myc</i>
transcription, increased expression of class II histocompatibility antigens.
1F5 effects similar to anti-Ig on solid phase, but effects are additive.
The greatest response is when anti-Ig is added to 1F5-preincubated B cells.
Changes in intracellular Ca<sup>+</sup> are not involved in anti-CD20 mAb
activattion.  The B1 anti-CD20 mAb in contrast blocks B cell activation, 
whether exposed to mitogenic stimulation or EBV infection.  Effects of 1F5 are
early in activation while those of B1 occur at later stage.  CD20 gets phosphorylated
at primarily serines, some threonines, but not tyrosines.  PMA treatment of
dense tonsillar B cells leads to CD20 phosphorylation.
<p>
450,000 recombinant plaques from &lambda;gt11 library of B cell (Daudi) cDNA 
were screened with polyclonal anti-CD20.  One clone, gt11.13a hybridizes with 
Northern blots of mRNA from Daudi B cells, but not from Jurkat T cells or LMCF-7 
breast carcinoma cells.  The clone is 337 bp long, and had multiple stop codons 
in all reading froms from both orientations.  Clone orientation was determined 
by using two oligonucleotides from opposite strands to probe the Northerns: 
these had same size as message.  Additional cDNA clones were found in 
&lambda;gt10 library using gt11.13a or fragments of other clones.  Clones 35.7 
and 35.10 were found with this screening.  A 342 bp fragment of 35.10 (named 
10S) lead to isolation of clone S7 from library.  Clones 10.19 and 10.20 came 
from &lambda;gt10 of cDNA primed with oligo P6.  The overlap of sequences from 
&lambda; clones were aligned to produce single long sequence except near the 
5&prime; terminus fo the cDNA.  Using a combination of additional clones to find 
the 5&prime; end, a 963 bp open reading frame (321 amino acids) is found that 
has the same 3&prime; position in many clones.  Northern blot analysis of 
lymphocyte cell lines shows a correspondence between CD20 antigen presence and 
the presence of an mRNA detected by cDNA probes.  The Ramos and Daudi Burkitt 
lymphoma lines, EBV-transformed lines T-51 and 616, and DHL10 from B cell 
lymphoma all are western and cDNA probe positive.  Two major mRNA species are 
found at 2.6 and 3.3 kb.  (Negative controls are IgG meyeloma 8226, T-lineage 
ALL lines CEM, 8402, HPB-ALL, HSB-2, Jurkat, and Molt-4.)  Oligo P7 is 
complementary to bases &minus;77 to &minus;46 of sequence S7 and oligo P8 is 
complementary to bases &minus;123 to &minus;79 of sequence 10.20.  P7 binds to 
the 2.6 and 3.3 kb species, and P8 binds to 3.3 kb message only.  Additional 
probes to other clones indicate that the 2.6 and 3.3 kb share homologous, if not 
identical, sequence of about 1500 bp.  The failure of P8 oligo to bind to 2.6 bp 
may be because the 3.3 kb band contains one or more introns that are spliced out 
to form the 2.6 kb species.  The P7 oligo likely binds to a sequence which is 
5&prime; of the deleted region.  Sequences 10.20 contains consensus for the 
3&prime; end of an intron for 11 of 14 bases:  the 10.20 sequence is 
ATTTGTTTTATTTTTAG|GAGT, where --YYYYYYYYYYNCAG|G-- is the consensus and a 
near-match for 3&prime; intron end, and the '}' character represents the splice 
site (see Kozak 1984 and Padgett et al 1986). On the S7 sequence, the sequence 
ACTCAG|GAGTT is similar to the consensus --AG:GUAAGU for the 5&prime; splice 
site.  The AG dinucleotide 5&prime; to the splice of a 5&prime; splicing site
is critical.
<p>
Of the two cDNA sequences stitched together, the difference is in the 5&prime;
untranslated region, so they both code for the same protein.  With the first
ATG codon (having a consensus A in &minus;3 position), this would produce
translation of a 297 amino acid polypeptide with computed mass of 33,093 Da, 
which is quite close to the SDS-PAGE bands showing a doublet at 33 and 35 kDa.
Two constructs of CD20 cDNA were done:  a <i>Eco</i>RI-<i>Hin</i>cII digest
of plasmid SP65 was ligated with a <i>Hin</i>dII-<i>Hin</i>cII from clone 35.10,
extending from the <i>Hin</i>dIII site at 126 to a <i>Hin</i>cII site in
&lambda;gt10 ~650 bp byond 3&prime; end of 35.10 insert.  Additionally a
<i>Eco</i>RI-<i>Hin</i>dIII fragmetn from either 35.10 or S7 (extending from 
5&prime; <i>Eco</i>RI end to <i>Hin</i>dIII site  at 126.  In vitro translation
was then done off the plasmid construct, producing a 33 kDa band.  It is not
present in steps that omit the foreign RNA.  The in vitro translation product
could be precipitated by 2H7 and 1F5 mAbs as anti-CD20. It should be noted that
it appears as single band not doublet.
<p>
Predicted CD20 sequence has no homolgoy to sequences in Genbank50 or 
PIR12 databases.  Kyte-Doolittle plots show three hydrophobic segments of
53, 25, and 20 amino acids, called A, B and C.  Segments B and C are consistant
with transmembrane spanning lengths, whereas segment A could span twice.
The C-terminal doman is highly charged, whereas the N-terminus has no obvious
signal sequence.  The 85 residues at C-terminus have 25 negatively charged
residues, with 22 Glu and 3 Asp; there are 8 positively charged residues, 6 Lys
and 2 Arg.  A model of the protein proposes that both N- and C-terminal are
oriented cytoplasmically.  If the solution-TM models is segmentized, it is
C1-TM2-C3-TM4-E5-TM6-C7, where C stands for cytoplasmic segment/orientation,
TM is transmembrane, E is extracellular, and the numbers just refer as an ordinal
to all possible segments (7 in this case).  Only one loop is extracellular.  Segment
C3 contains a single Cys residue, as does C7, and these are shown as disulfides.
With TM2 (transmembrane region A), there is no expected Ex extracellular segment
that follows.  Transmembrane domain A spans the membrane twice and the protein
bends back on itself in the segmenting containing Pro-73, Gly-75, Pro-79; there
is also a Pro-87, though not in middle of the predicted bend-back.  There is
a good case for the C-terminal being cytoplasmic:  anti-CD20 pAb serum reacts
with the 28 C-terminal amino acids in a &lambda;gt11 clone during immunoprecipitation,
but does not bind intact cells that express CD20.   The whole CD20 sequence
has 5 Cys residues, and there is no evidence of intermolecular disulfide bond
formation from SDS-PAGE.  Both Cys-111 and Cys-220 lie in cytoplasm and could
form a disulfide.  Cys-167 and Cys-183 are extracellular and might form a disulfide.
Valentine and Clark (1988) reported differences in SDS-PAGE mobility between 
reduced and non-reduced samples.
<p>
Anti-CD20 and anti-CDw40 mAbs in combination can cause B cell activation and
proliferation.  Anti-CDw40 mAb G28-5 was used to assess CD20 phosphorylation:
when mAb was incubated with tonsillar B cells preincubated with <sup>32</sup>P,
CD20 phosphorylation increases 11-fold.
<p>
Multiple membrane spanning proteins fall into two families.  Rhodopsin, &beta;-2 
adrenergic receptor and muscarinic ACh receptor are all multi-spanning proteins 
that signal through GTP-binding proteins.  These are glycoproteins around
70 kDa and have seven spanning segments.  Each membrane spanning segment has
about 20-24 uncharged or generally nonpolar amino acids, they lack amino terminal
signal sequence and have glycosylated N-terminal segments.  The second and more
heterogeneous group are ion channel proteins: (Ca<sup>2+</sup> + Mg<sup>2+</sup>)
ATPase, (Na<sup>+</sup> + K<sup>+</sup>) ATPase catalytic subunit, sodium
channels, erythrocyte band 3, and nicotinic ACh receptor.  Band 3 is 100 kDa
and has 10 hydrophobic regions thought to span membrane 12 times.  The ATPases
have about the same mass and cross between 7-9 times.  Hydrophobic region B
has a single charged Asp-137 and six polar residues, possibly providing an
amphipathic helix for cation conduction.  The C-terminal has features consistent
with ion channel function.   Proteins with multiple membrane spanning also
include glucose transporter, lactose carrier protein, and a EBV-encoded membrane
proteins expressed in infected cells.  The monosaccharide binders cross the membrane
12 times, while the EBV protein is similar to CD20.
<p>
In cells that do not express CD20, no 2.6 or 3.3 kb mRNA is found, and it is
always found in cells that express CD20.  Thus regulation is by transcription.
The difference between 2.6 and 3.3 kb mRNA seems to be related to splicing, but
as they are both poly(A) RNA, the function is not understood.
<p>
Immunoprecipitation of CD20 in B cells shows a SDS-PAGE doublet of 33 and 35 kDa.
The in vitro translation produces only the 33 kDa product.  While there are other
ATG start codons, the doublet is not explained by alternate translational start
sites.  There is no evidence that CD20 is glycosylated, and treatment with 
alkaline phosphatase does not eliminate doublet.
<p>
<b>Methods</b>.  
cDNA from Daudi B cells was made with 10 &micro;g poly(A)+ RNA using oligo(dT)
primer and 250 U AMV reverse transcriptase in a Tris-KCl-MgCl<sub>2</sub>-DTT
buffer with 400 U RNA and 500 &micro;M dNTPs.  2nd strand synthesis was done
with HEPES-MgCl<sub>2</sub>-KCl-DTT-dNTPs and 20 U Klenow fragment overnight at
15&deg;.  After S1 nuclease, cDNA was dG-tailed, then ligated in &lambda;gt10
digested with <i>Eco</i>RI using 5&prime;-AATTCCCCCCCCCCCC-3&prime; and packaged
in vitro.  <i>E coli</i> C600 Hfl<sup>+</sup> was infected with phage and grown
on 150 mm plates (30 ct).  Phage was pelleted in SW28 rotor at 23000 rpm at 
90 min with SM buffer overlay (see Maniatis et al 1982).  To isolate &lambda;
DNA, phage is incubated 70&deg; 5 min in 0.2% SDS, 200 mM Tris HCl pH 8.0, 
20 mM EDTA, 0.2% diethyl pyrocarbonate;  KOAc is added to 0.45 M to precipitate
protein.  DNA is then cleaned by EtOH precipitation.  DNA is digested with
<i>Eco</i>RI, precipitated again, applied to 10-40% sucrose sucrose step gradient
(13 steps), spun in SW28 at 26000 rpm for 6.5 h at 15&deg;.  Three topmost fractions
were analyzed on 0.7% agarose and free of &lambda; arms.  These were pooled 
and precipitated, ligated into <i>Eco</i>RI-digested &lambda;gt11 and packaged
in vitro.  Plaque lifts of phage-infected E coli Y1090 were made on nitrocellulose
filters, exposed to chloroform vapors 15 min, rinsed in Tris buffered saline
with 0.05% Tween 20 (TBST), blocked with TBST + 5% milk powder 60 min at RT (or
overnight 4&deg;).  Filters were incubated 90 min @ RT in TBST/5% milk powder
with anti-CD20 antiserum at 1:1000.  Filters were rinsed in TBST, incubated
in TBST with alkaline phosphatase-conjugated goat anti-mouse Ig at 1:7500 for
30 min, rinsed in TBST with BCIP/NBT. Positive plaques were replated twice or
until all plaques were positive.
<p>
CD20 was purified from Daudi B cells after they were solubilized in 0.5% NP-40, 
and then passed on immunoaffinity with 1F5 bound to CNBr-activated Sepharose 4B. 
Polyclonal Ab from mouse to CD20 was prepared by suspending Ag in Freund's 
complete adjuvant, injected into mice i.p. 100 &micro;l sepharose per mouse.
Boosts were also given 25 and 32 days thereafter.
<p>
Northern were made using total RNA extracted with GuanSCN using CsCl ultracentrifugation,
and isolating poly(A)<sup>+</sup> RNA with oligo(dT) columns, running the mRNA
on agarose-formaldehyde gels, transferring to nylon, irradiating with UV.
Prehybridization was 4 h at 42&deg; in 5 &times; SSC, 5 &times; Denhardt's,
0.5% SDS, 50% formamide, and tRNA for nick-translated fragments;  or 
45&deg; in 5&times;SSC, 10 &times; Denhardt's, 50 mM NaPi pH 7.2, and 7% SDS
for oligos.
<!-- ============================================================= -->
<p class="tags">
CD20
<p class="ref">
 <span class="authors">
Stamenkovic I, Seed B
 </span>
 <span class="year">
(1988)
 </span>
 <span class="jname">
J. Exp. Med.
 </span>
 <span class="jvol">
167,
 </span>
 <span class="pages">
1975-1980.
 </span>
<p class="reftitl">
Analysis Of Two cDNA Clones Encoding The B Lymphocyte Antigen CD20 (B1, Bp35), A 
Type III Integral Membrane Protein
<p class="Para1">
[This is an independent report for the cloning of CD20 and its sequence 
determination as reported by Einfeld et al elsehwere.]
<p>
Poly(A)<sup>+</sup> RNA was prepared from Burkitt cell line Daudi from total 
RNA and using oligo(dT) column.  cDNA was prepared as an expression library.
mAbs 1F5, 2H7, B1, L27, G28-2, 93-1B3, B-C1, and NU-B2 are all anti-CD20 and
obtained from Intl Leukocyte Typing Workshop.  Purified mAbs were used at
1 &micro;g/ml and ascites at 1:1000.  B lymphocytes can be teased from tonsillar
tissue in sterile PBS and T cells depleted from it using anti-CD3/complement
lysis (anti-CD3 mAb in PBS at 4&deg; for 45 min, wash with PBS twice, incubate
with commercial rabbit complement 37&deg; for 45 min; monocytes can be removed
in two 30 min rounds of panning on tissue culture dishes at 37&deg;.
<p>
<b>Results</b>.  This work yielded to two cDNA clones, CD20.4 with 1.5 kb insert
and CD20.6 with 1.0 kb.  Panning with anti-CD20 mAbs was used to purify the 
clones, which are used to detect for expression on the surface of transiently
transfected COS cells.  The immunoprecipitated proteins have a 33 kDa band size.
The insert in clone CD20.6 has a short poly(A) tail and is missing a consensus
polyadenylation signal, while clone CD20.4 insert has not poly(A) tail and
it extends 431 bp beyond 3&prime; terminus in CD20.6.  Einfeld et al and 
Tedder et al have reported isolating the cDNA of CD20 in phage and plasmid
libraries, respectively.
<p>
B lymphocytes show the presence of 3 transcripts of sizes 1.5, 3.0, and 3.8 kb
using RNA blot analysis but are absent in numerous other cell types, consistent
with antibody studies.  The CD20.6 clone may be derived from the 1.5 kb 
transcript or a shorter undetectable species.  The CD20.4 clone source cannot
be inferred with no poly(A)<sup>+</sup> tail.  DNA blot (Southern) analysis
indciates that CD20 genomic sequences are not altered during development
nor are amplified in cell lines where it is expressed.  RFLP analysis reveals
a polymorphism in placenta.
<p>
The translated portion is expected to be 297 amino acids with calculated mass
of 33,097 Da.  Three hydrophobic segments at 51-103, 117-141, and 183-203
are identified.  The N-terminus shows there is no signal peptide, and the 
C-terminal domain is highly ionized.  Polyclonal anti-CD20 to the last 18 
residues of the C-terminus reacts with lysates but not intact cells, indicating
the C-terminal is cytoplasmically located.  With no signal peptide on the N-terminus
it too may be located in cytoplasm.  The first hydrophobic region is 53 amino
acids long and might span the membrane twice if as an amphipathic helix.
There are two potential N-glycosylation sites (N-9 and N-293) but will likely
not be modified if these segmetns are intracellular.  With all the identified
transmembrane crossings and location of termini, the only extracellular domain is
between 142 and 182, rich in Ser and Thr and thus potential O-glycosylation sites.
When anti-CD20 mAb B1 is used in immunoprecipitation of transfected COS cells,
there is a 33 kDa protein.  In other B cell subsets, 35 and 37 kDa species are
identified and this might be O-glycosylation.
<p>
CD20 cDNA has two ATG codons separated by 16 residues, although neither is within
a Kozak consensus initiation site.  The protein for the first site is predicted
to be a 33.1 kDa while for the second would be 30.8 kDa.  Immunopptn indicates
that there may be only one species, but with another protocol, a major 33 kDa
and minor 31 kDa band are seen.
<p>
Sequence comparisons of peptide with the NBRF database showed no significant
homology with other proteins using the FASTP rapid sequence alignment algorithm.
It is possible that since much of the protein is involved in the interior of 
cell membrane it plays a role in transducing signals.  The relatively hydrophilic
nature of the hydrophobic regions could be such that there are hydrogen bond
interactions with transmembrane portions of other membrane proteins.

<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Shenolikar S, Langston J, Schworer CM, Kelly PT 
 </span>
 <span class="year">
(1988)
 </span>
 <span class="jname">
Biochem. Biophys. Res. Comm.
 </span>
 <span class="jvol">
151,
 </span>
 <span class="pages">
1332-1338.
 </span>
<p class="reftitl">
Substrate specificity of Ca<sup>10.0pt'>2+</sup>/CaM-dependent
multifunctional protein kinases:
comparison of isoenzymes from brain, liver, and skeletal muscle
<p class="Para1">
Purified rabbit skel musc. PK-A catalytic subunit was used to
phosphorylate a variety of substrates:
glycogen synthase, synapsin I, pyruvate kinase, phenylalanine
hydroxylase (PAH), histone H1, and syntide 2.
In all assays, a majority of <sup>32</sup>P
label was incorporated into single protein bands on gels which represented
substrate proteins.  In absence of PK-A,
only glycogen synthase and pyruvate kinase preps revealed an endogenous kinase
activity not removed.  (Commercial
preparations of glycogen synthase (Sigma) were poor substrates for PK-A, having
only 5-10% of incorporation rates compared to purified synthase.)
<p>
CaM-K was obtained from liver, brain, and skeletal muscle
and used to phosphorylate synapsin I at rates paralleling autophosphorylation
of the CaM-K subunits. 2 mol P/mol synapsin I was incorporated in 5 min
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Yamamoto, K. K., Gonzalez, G. A., Biggs, W. H. III, Montminy, M. R.
 </span>
 <span class="year">
(1988)
 </span>
 <span class="jname">
Nature
 </span>
 <span class="jvol">
334,
 </span>
 <span class="pages">
494-498.
 </span>
<p class="reftitl">
Phosphorylation-induced binding and transcriptional efficacy of
nuclear factor CREB
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
G. E. Tarr
 </span>
 <span class="year">
(1988)
 </span>
 <span class="jname">
in A. S. Bhown, ed. CRC Press.
 </span>
 <span class="pages">
35-47.
 </span>
<p class="reftitl">
Manual methods for protein/peptide sequence analysis
<p class="Para1">
With manual sequencing there are basically two methods,
the <i>partition</i> and the <i>film</i> method.
<p>
The partition method is good for small peptides up to a dozen residues,
or peptides which would wash out using the film method.  The partition
method is not recommended for large peptides or proteins.  Partitioning
in general gives better recoveries for Ser and Thr, lower extractive
yields of Arg, is more work and time-consuming.  While the film method
is preferable, the materials best for filming are also best for
automatic sequencing.  A sequencing standard such as insulin B
should always be included with each batch of samples.
<p>
The film method is essentially the same as used in spinning-cup
and gas-phase automated sequencers.  Rather than a lower aqueous
phase, a layer of peptide and Polybrene (a hydrated carrier) is
found.  Since polarity is the discriminating property of extraction,
the organic solvent should remove hydrophobic products without
removing peptide.  Samples to be filmed are ideally salt- and
detergent-free;  it is good idea to prepare such samples by
RP-HPLC in a volatile buffer system, exhaustively dialyzing
in 50% EtOH (even if the sample precipitates), or precipitating
sample from concentrated aqueous solution with acetone or MeCN
or from a small volume of formic acid (HFo) with ethyl acetate.
HFo and hexafluoroacetone trihydrate (HFA) are good general
solvents for sample transfer.  (Note while HFA is more toxic
and expensive, it is gentler and compatible with long-term storage.)
<p>
<b>Partition Method</b>.  Dissolve 50 pmol to 5 nmol sample in
5-50&nbsp;&micro;l&nbsp;dilute acid in water or aqueous alcohol.
While HFo is an effective solvent, the peptide should not
be stored in it.  Redry the sample from 5-10 &micro;l
EtOH:TEA:water (2:2:1) to remove ammonia.

<ol>
<li><b>Thiocarbamylation (coupling).</b>  Add 10 &micro;l
freshly prepared EtOH:TEA:water:PITC (7:1:1:1) to the dry
peptide and mix, but do not splash anything high in the tube.
Reagents may be added to tubes with the 25&nbsp;ml&nbsp;incubation vial
and mixed by cautious lateral shaking and tilting.
Incubate 8-10 at 50&deg;, under N2 if preferred, although
it seems to make no difference.  With DMF as the solvent
(3 min, 50&deg;), reaction is faster but more by-products
are observed.  Trimethylamine (TMA) as an amine catalyst
is not as good as TEA.  Vacuum dry until all PITC is gone
(it will <i>look</i> dry given no salt accumulation), which
takes 5-10 min at room temp and 2-4 min at 50&deg;, which is
arranged by locating part of the vacuum system above the
heating block.
<li><b>Washing.</b>  Add 2.5&nbsp;&micro;l&nbsp;water low into the tube
and ~200&nbsp;&micro;l&nbsp;heptane:EA (15:1) containing 0.5% TMA
(serves to exchange TEA, creating a more hydrophilic,
less extractable, peptide salt).  Vortex and centrifuge
briefly.  Decant upper phase, being careful not to tip
tube such that lower phase decants.  Wash 2-3 times in same
manner with heptane:EA (7:1) without amine.  Now is
a good time to add internal standard to normalize
analytical results.  Vacuum until quite dry.
<li><b>Cleavage.</b>  Add low into tube 5 &micro;l
concentrated HCl (11-12 M) with a glass capillary
or plastic needle syringe.  (One can add acid high
to tube then centrifuge:  this may reduce chances
of cross-contamination.)  Mix (avoid splashing),
let stand 5 min at 22&deg; or 1.5-2 min at 50&deg;.
Although not essential, an inert atmosphere may improve
efficiency and or background.  While almost all residues
cleave well at room temperature, Pro-X bonds require
50&deg; treatment.  Using a vacuum port different from
the one used to dry the coupling medium, pull vacuum
carefully to avoid bumping and leave on a full vacuum
3-5 min at 22&deg;.
<li><b>Stabilization.</b>  Add 5-10&nbsp;&micro;l&nbsp;MeOH and
incubate 3 min at 50&deg;.  This opens the thiazolinone
product of cleavage and prevents it from hydrolyzing in
Step 5, which would prevent the extraction of His and Arg.
This step is not absolutely necessary if Arg or His are not
in the peptide.  Vacuum dry.
<li><b>Extraction.</b>  Add 4&nbsp;&micro;l&nbsp;water and 40 &micro;l
benzene/MeCN, 2:1, mix well, and centrifuge.  Heptane:EA (1:5)
can be used instead if one worries more about the toxic benzene
than the peroxide-prone EA.  Decant the upper phase carefully in
6 &times; 50 mm tubes containing 50 nmol thiourea in
2-5&nbsp;&micro;l&nbsp;MeOH.  Extract now with 40&nbsp;&micro;l&nbsp;0.02&nbsp;M HFA,
0.016&nbsp;M TMA in benzene:MeCN (2:1), combining the extracts.
When processing a batch of samples, be compulsively
organized to prevent errors:  line up the extraction tubes
in order in the microtiter plate and line up the sequencing
tubes in a parallel row;  always use the same hand to grasp
each kind of tube, in the same order, and proceed
systematically along the rows.  If a peptide contains
no very polar residues, a decrease in the strength of
the extraction solvent may give less washout and, therefore,
higher recoveries toward the end of the peptide (e.g.,
heptane:EA 1:2).  Dry lower phase and return <i>N</i>-1
peptide to Step 1.
<li><b>Conversion.</b>  Extract is dried carefully with
weak vacuum or aspirator, then with full vacuum.
Add 15-20&nbsp;&micro;l&nbsp;1 <i>N</i> HCl/MeOH and incubate
65&deg; for 10 min, or the same volume of 2 <i>N</i> HCl/MeOH
and 10 min at 50&deg;.  Note if the analytical system does
not accomodate Asp or Glu methyl esters, use 25% aqueous
TFA at 65&deg; instead.
</ol>
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Maher, P.
 </span>
 <span class="year">
(1988)
 </span>
 <span class="jname">
Proc.&nbsp;Natl.&nbsp;Acad.&nbsp;Sci.&nbsp;USA
 </span>
 <span class="jvol">
85,
 </span>
 <span class="pages">
6788-6791.
 </span>
<p class="reftitl">
Nerve growth factor induces protein-tyrosine phosphorylation
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Hanley, R. M., Means, A. R., Kemp, B. E., Shenolikar, S.
 </span>
 <span class="year">
(1988)
 </span>
 <span class="jname">
Biochem. Biophys. Res. Comm.
 </span>
 <span class="jvol">
152,
 </span>
 <span class="pages">
122-128.
 </span>
<p class="reftitl">
Mapping of calmodulin-binding domain of
Ca<sup>2+</sup>/calmodulin-dependent protein kinase II from rat brain
<p class="Para1">
Purified rat brain CaM-K II and purified rabbit skel. musc. CaM
were used in these studies.  In the
presence of Ca<sup>2+</sup>, CaM, and ATP,
CaM-K II demonstrated autophosphorylation of both its alpha (50 kDa) and beta
(60 kDa) subunits.  Ca<sup>2+</sup>-dependence was shown by lack of
incorporation with EGTA present.  The affinity constant (K<sub>a</sub>)
of CaM for this autophosphorylation for either subunit is ca. 100 nM
under these conditions.
<p>
Synthetic peptides corresponding to select regions/domains
of the CaM-K II alpha subunit were used to study the CaM-binding domain (a
deduced sequence by Lin et al., <i>PNAS</i> <b>84</b>: 9497, 1987,
from the molecular clone of Hanley et al, <i>Science</i>
<b>237</b>: 293, 1987, makes it possible):  peptide 1-20 corresponds to
residues 290-309 and shown to contain the CaM binding domain.
Other peptides 1-13, 6-15, and 7-20 are modeled around this region.
Both 1-20 and 7-20 at 1 &micro;M completely inhibited autophosphorylation
of both subunits to the same extent as 10 mM EGTA.  Peptides
1-13 and 6-15 at the same concentration were without effect,
and still had no effect up to 10 &micro;M.  <i>K</i><sub>i</sub> for 1-20 and
7-20 were between 50 and 70 nM, with curves of inhibition being the
same for both alpha and beta subunits.  This inhibition could be
overcome if Ca<sup>2+</sup> was brought to 5 &micro;M.  The synthetic
peptides were not phosphorylated in any manner, and since inhibition
was competitive with respect to CaM concentration changes, these
peptides were probably antagonists.
<p>
The binding of these peptides to CaM can be examined
looking at mobility shifts in CaM on 7.5% non-denaturing gels in the presence
of 1 mM CaCl<sub>2</sub> or 10 mM
EGTA.  Mobility of CaM in the presence
of Ca<sup>2+</sup> was retarded when both
peptide 1-20 and 7-20 were bound to CaM at a molar ratio of 2:1 peptide:CaM; a
tight association is shown.  In EGTA no
mobility change is observed.  Peptides
1-13 and 6-15 did not bind and caused no mobility change likewise.
With the ratio of 1-20 and 7-20 peptides at
1:1 with respect to CaM, and with the gels containing Ca<sup>2+</sup>
and 8&nbsp;M urea, there was still an observed shift, which is consistent
with the previous report that the whole subunit binds to CaM in a
Ca-dependent manner with 8&nbsp;M urea present.  In urea-containing 10% gels,
the peptide-bound CaM actually shows an increased mobility:
about 50% of the CaM showed the increased mobility while a less
mobile band was free CaM.  Increasing peptide to 2:1, 5:1, and 10:1
eventually caused all CaM to migrate faster.
<p>
The peptide 7-20 was the minimally necessary sequence
capable of binding CaM, demonstrated by inhibition of
Ca<sup>2+</sup>/CaM-induced autophosphorylation and
altering CaM mobility on gels.  (Peptide 1-20 is
LKKFNA<b>RRKLKGAILTTMLA</b>, with peptide 7-20 covering all underlined
amino acids.)  Peptide 1-13 did not
contain any residues which would inhibit the reaction, nor did 6-15.
This suggests amino acids 16-20 were
minimally necessary for Ca-dependent association with CaM.
<p>
Other CaM-binding peptides have been reported
previously:  mellitin (GIGAVILKVL<b>TTGLP</b>ALISWIKRKRQQ),
M-13 (KRRWKKN<b>FIAVS</b>AANRFKKISSSGALM), and
RS-20 (RRKWQK<b>TGHAV</b>RAIGRLSSS),
where the underlined pentapeptide sequences are the most hydrophobic amino
acids in these peptides.  This compares with
CaM-K II 7-20 (RRKLKGAIL<b>TTMLA</b>).
For M-13, it has been shown that a peptide composed of the first 11
N-terminal amino acids retains CaM-antagonism, and 4 of the 5 indicated amino
acids are present.  The amphiphilic domain is necessary however,
since elimination of 2 of the 4 arginines in RS-20 reduces the
K<sub>d</sub> for CaM binding by 8-fold.  The homology between
mellitin's and peptide 7-20's <q>hydrophobic</q> region is striking,
predicted to conform to a beta sheet structure.  Helix wheels show that
the residues conform poorly to an amphipathic helix.
<p>
Previous work by Seeholzer et al. (PNAS 83: 3634, 1986) in
which mellitin binding to CaM was studied using NMR and deuterated CaM shows
that mellitin induced perturbations in both NH<sub>2</sub>-
and COOH-terminal halves of CaM, although it was expected to bind only the COOH
half.  Additional studies eventually
showed that the first 13 residues of mellitin bound to the amino half of CaM
while the next 13--containing some of the <q>hydrophobic</q> region plus
some basic residues, bound to the carboxyl half.
The structural comparison of 7-20 peptides with mellitin suggest
7-20 binds to the COOH half of CaM, and while the alpha helix in 7-20 peptide
contributes to CaM antagonism, it is the hydrophobic residues responsible for
high affinity binding to CaM.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Dahmer, M. K., Perlman, R. L.
 </span>
 <span class="year">
(1988)
 </span>
 <span class="jname">
J. Neurochem.
 </span>
 <span class="jvol">
51,
 </span>
 <span class="pages">
321-323.
 </span>
<p class="reftitl">
Bovine chromaffin cells have insulin-like growth factor-I (IGF-I) receptors: 
IGF-I enhances catecholamine secretion
<p class="Para1">
Chromaffin cells were cultured at a 400,000 to 1,000,000/cm<sup>2</sup> density 
on collagen-coated dishes.  Isolation values were &gt;95% viability; 88-93% 
neutral red-stained cells.  Cells maintained in DME/F-12 medium containing 5-FDU 
and Ara-C + 10% FCS.  Cells were refreshed in serum-free medium without DNA 
synthesis inhibitors 2 d prior to experimentation.
<p>
Competitive binding experiments with <sup>125</sup>I-IGF-I were performed with 
insulin and IGF-I.  Specific binding vs log [hormone] revealed a magnitude of 
two orders difference between insulin and IGF-I, with IGF-I being more potent. 
Scatchard analysis indicated 111,000 &plusmn; 40,000 IGF-I sites/cell with an 
apparent <i>K</i><sub>d</sub> of 1.1 &plusmn; 0.3 nM.
<p>
Chemical cross-linking reagents were added to the <sup>125</sup>I-IGF/cell 
mixture and SDS gels were run.  In the absence of reducing conditions, a labeled 
band of &gt;200 kDa was apparent.  In the presence of disulfide reducing 
conditions, this labeled band had an <i>M</i><sub>r</sub> ~125,000.  Unlabeled 
IGF-I added to 30 nM inhibited labeling of this protein strongly, but 30 nM 
insulin had little effect.  These results show that IGF binds to the &alpha; subunit 
of the receptor. Since the cell preparations here show about 90% pure chromaffin 
cells, it is likely that these cells contain the IGF-I receptors and not the 
contaminating cells.  If the contaminating cells contained the receptors, it is 
predicted that binding sites per cell would be uncharacteristically high 
(10<sup>6</sup>/cell). Differential plating studies, designed to enrich 
chromaffin cells, would also be expected to alter the sites/cell number:  these 
methods did not.
<p>
Cell catecholamine secretion was measured in response to long-term (4 day) 
incubation with insulin or IGF-I at various concentrations.  Cells were 
stimulated with high KCl for 10 min prior to measure the release.  Compared to 
controls not treated with hormones, the IGF-I-treated cells released a larger 
percentage of CAs with stimulation.  At 10 nM IGF-I, the increased release was 
2-fold.  Insulin was much less effective in this regard, not differing from the 
controls.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Lohka, M. J., Hayes, M. K., Maller, J. L. 
 </span>
 <span class="year">
(1988)
 </span>
 <span class="jname">
Proc. Natl. Acad. Sci. USA
 </span>
 <span class="jvol">
85,
 </span>
 <span class="pages">
3009-3013.
 </span>

<p class="reftitl">
Purification of maturation-promoting factor, an
intracellular regulator of early mitotic events
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Draetta, G., Brizuela, L., Moran, B., Beach, D. 
 </span>
 <span class="year">
(1988)
 </span>
 <span class="jname">
Cold Springs Harbor Symp. Quant. Biol.
 </span>
 <span class="jvol">
53,
 </span>
 <span class="pages">
195-201.
 </span>

<p class="reftitl">
Regulation of the vertebrate cell cycle by the <i>cdc2</i> protein kinase
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Draetta, G., Piwnica-Worms, H., Morrison, D., Druker,
B., Roberts, T., Beach, D. 
 </span>
 <span class="year">
(1988)
 </span>
 <span class="jname">
Nature
 </span>
 <span class="jvol">
336,
 </span>
 <span class="pages">
738-744.
 </span>

<p class="reftitl">
Human cdc2 protein kinase is a major cell-cycle regulated
tyrosine kinase substrate
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Draetta, G., Beach, D. 
 </span>
 <span class="year">
(1988)
 </span>
 <span class="jname">
Cell
 </span>
 <span class="jvol">
54,
 </span>
 <span class="pages">
17-26.
 </span>

<p class="reftitl">
Activation of cdc2 protein kinase during mitosis in human
cells:  cell cycle-dependent phosphorylation
and subunit rearrangement
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Draetta, G., Beach, D., Moran, E. 
 </span>
 <span class="year">
(1988)
 </span>
 <span class="jname">
Oncogene
 </span>
 <span class="jvol">
2,
 </span>
 <span class="pages">
553-557.
 </span>
<p class="reftitl">
Synthesis of p34, the mammalian homolog of the yeast <i>CDC28</i> protein kinase,
is stimulated during adenovirus-induced proliferation of primary baby rat
kidney cells
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Niwa, O., Yanagida, M. 
 </span>
 <span class="year">
(1988)
 </span>
 <span class="jname">
Nature
 </span>
 <span class="jvol">
336,
 </span>
 <span class="pages">
430.
 </span>

<p class="reftitl">
Universal and essential role of MPF/<i>cdc2</i><sup>+</sup>
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Labbe, J. C., Lee, M. G., Nurse, P., Picard, A., Doree,
M. 
 </span>
 <span class="year">
(1988)
 </span>
 <span class="jname">
Nature
 </span>
 <span class="jvol">
335,
 </span>
 <span class="pages">
251-254.
 </span>

<p class="reftitl">
Activation at M-phase of a protein kinase encoded by a
starfish homologue of the cell cycle control gene 
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Blatt, J., Taylor, S. R., Stitely, S. 
 </span>
 <span class="year">
(1988)
 </span>
 <span class="jname">
J. Lab. Clin. Med.
 </span>
 <span class="jvol">
112,
 </span>
 <span class="pages">
433-436.
 </span>

<p class="reftitl">
Mechanism of antineuroblastoma activity of deferoxamine
in vitro
<p class="Para1">
Deferoxamine is an iron chelator with antineuroblastoma
activity.  With 60 &micro;M of the inhibitor, a 35%
increase of CHP 126 neuroblastoma cells in nonproliferating and
prereplicative phases is observed.  A 60%
decrease in thymidine incorporation shows reduced DNA synthesis.
Effects on RNA and protein synthesis were
minimal.  Effects of other chelators were
tested as well, and they showed similar actions on these cells.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Elliot, G. T., Kelly, K. F., Bonna, R. L., Wardlaw, T.
R., Burns, E. R. 
 </span>
 <span class="year">
(1988)
 </span>
 <span class="jname">
Cancer Chemother. Pharmacol.
 </span>
 <span class="jvol">
21,
 </span>
 <span class="pages">
233-236.
 </span>

<p class="reftitl">
In vitro antiproliferative activity of
2'-(2-hydroxyphenyl)-2'-thiazoline-4'-carboxylic acid and its methyl ester
on L1210 and P388 murine neoplasms
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Maurer-Schultze, B., Siebert, M., Bassukas, I. D. 
 </span>
 <span class="year">
(1988)
 </span>
 <span class="jname">
Exp. Cell Res.
 </span>
 <span class="jvol">
174,
 </span>
 <span class="pages">
230-243.
 </span>

<p class="reftitl">
An <i>in vivo</i> study on the synchronizing
effect of hydroxyurea
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Yoshida, M., Beppu, T. 
 </span>
 <span class="year">
(1988)
 </span>
 <span class="jname">
Exp. Cell
Res.
 </span>
 <span class="jvol">
177,
 </span>
 <span class="pages">
122-131.
 </span>

<p class="reftitl">
Reversible arrest of proliferation of rat 3Y1 fibroblasts
in both the G1 and G2 phases of trichostatin A
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Sokoloski, J. A., Beardsley, G. P., Sartorelli, A. C.

 </span>
 <span class="year">
(1988)
 </span>
 <span class="jname">
Cancer Res.
 </span>
 <span class="jvol">
49,
 </span>
 <span class="pages">
4824-4828.
 </span>

<p class="reftitl">
Induction of HL-60 leukemia cell differentiation by the
novel antifolate 5,10-dideazatetrahydrofolic acid
<p class="author">Erikson, E., Maller, J. L. (1988) <i>Xenopus</i> eggs
<p class="author">Tsuda, H., Sakaguchi, M., Kawakita, M., Nakazawa, S.,
Mori, T., Takatsuki, K. (1988) <i>Int. J. Cell Cloning</i> <b>6</b>, 209-220.
<p class="reftitl">
Alteration of cell cycle progression in human leukemia
cell line (KOPM-28) induced by 12-O-tetradecanoylphorbol 13-acetate
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Gautier, J., Norbury, C., Lohka, M., Nurse, P., Maller,
J. 
 </span>
 <span class="year">
(1988)
 </span>
 <span class="jname">
Cell
 </span>
 <span class="jvol">
54,
 </span>
 <span class="pages">
433-439.
 </span>

<p class="reftitl">
Purified maturation-promoting factor contains the product
of a Xenopus homolog of the fission yeast cell cycle control gene 
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Sitney, K. C., Budd, M. E., Campbell, J. L. 
 </span>
 <span class="year">
(1989)
 </span>
 <span class="jname">
Cell
 </span>
 <span class="jvol">
56,
 </span>
 <span class="pages">
599-605.
 </span>

<p class="reftitl">
DNA polymerase III, a second essential DNA polymerase, is
encoded by the S. cerevisiae <i>CDC2</i> gene
<p class="author">Rudkin, B. B., Lazarovici, P., Levi, B.-Z., Abe, Y.,
Fujita, K., Guroff, G. (1989) <i>EMBO J.</i> <b>8,</b> 3319-3325.
<p class="reftitl">
Cell cycle-specific action of nerve growth factor in PC12
cells:  differentiation without proliferation
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Burgoyne, R. D.
 </span>
 <span class="year">
(1988)
 </span>
 <span class="jname">
Nature 
 </span>
 <span class="jvol">
331,
 </span>
 <span class="pages">
20.
 </span>
<p class="reftitl">
Calpactin in exocytosis?
<p class="Para1">
<b>REPORT</b>.  Describes a study by Drust and Creutz on p. 88 of issue in which 
calpactin may play a role in calcium-dependent membrane fusion which leads to 
release of secretory vesicle contents.  Previously proteins from adrenal 
medullary extracts were isolated for their ability to bind reversibly to 
secretory vesicle membranes in the presence of micromolar calcium.  Some 
proteins isolated included CaM, PK-C, caldesmon and a series of closely related 
Ca<sup>2+</sup>- and lipid-binding proteins having the generic name annexins. 
Protein and cDNA sequence analysis of the annexins reveals significant 
similarity between them with five separate members (calpactin, lipocortin I, 
endonexin, endonexin II, and p70) have conserved domains.  Calpactin is a 
heterotetramer with a (p36)<sub>2</sub>(p10)<sub>2</sub> structure.  The heavy 
chain (p36) is well known to be a substrate for retroviral tyrosine kinases.
<p>
These annexin proteins are tested for their ability to elicit 
Ca<sup>2+</sup>-dependent binding of membranes or liposomes to each other at 
physiologically relevant calcium concentrations.  Among the members of this 
group, only calpactin is able to aggregate secretory vesicles at low micromolar 
levels of Ca, levels low enough to cause Ca-activated secretion in permeabilized 
chromaffin cells. Truncated p36, which cannot bind p10, requires higher Ca 
concentrations to induce granule aggregation.  Intact calpactin was much better 
than isolated heavy chain in aggregating liposomes at lower [Ca<sup>2+</sup>]. 
Hence p10 is a necessary element in making calpactin more efficient at membrane 
fusion (vesicle aggregation) than the other annexins;  it may increase calcium 
and/or lipid affinity for the heavy chain, or its participation in creating the 
heterotetrameric structure may enable membrane "crosslinking."
<p>
One experiment that can be done to examine whether calpactin is responsible for 
the observed processes is to let proteins leak from digitonin-permeabilized 
chromaffin cells.  Calcium added to these cells evokes the secretion response. 
However, with continued leaking of the proteins which may possibly be necessary 
and sufficient for secretion, calpactin can be added back to see if it restores 
the lost function.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Higgins, L. S., Berg, D. K.
 </span>
 <span class="year">
(1988)
 </span>
 <span class="jname">
J. Neurosci.
 </span>
 <span class="jvol">
8,
 </span>
 <span class="pages">
1436-1446.
 </span>
<p class="reftitl">
A desensitized form of neuronal acetylcholine receptor detected by 
<sup>3</sup>H-nicotine binding on bovine adrenal chromaffin cells
<p class="Para1">
Chromaffin cells were prepared in DME + 10% FCS with pen/strep.  10 &micro;M 
Ara-C was added after the first day of culture.  Cultures were used for 
experiment only between 3-8 d after plating.
<p>
Membranes were prepared for tritiated nicotine binding studies be scraping 
plates in HEPES buffered KRH-like medium containing 2 mg/ml BSA.  Studies were 
also done with intact cells.  Competitive bindings studies were done and a plot 
of [<sup>3</sup>H]nicotine bound vs. [alpha bungarotoxin] indicates that two 
classes of nicotine receptors are in cells.  At &lt;1 nM &alpha;-BGT, 100% of 
the nicotine is bound. When the &alpha;-BGT is between 0.1 and 1 &micro;M, only 
50% of the labeled nicotine is bound, indicating that half the receptors binding 
nicotine are low affinity. With even 10 &micro;M &alpha;-BGT, there was no 
decrease in the labeling of the high affinity class.
<p>
Looking at this high affinity class, the binding kinetics of the receptors are 
established using [<sup>3</sup>H]nicotine and 1 &micro;M &alpha;-BGT.  Plots of 
[<sup>3</sup>H]nicotine bound vs. incubation time in min (for association 
kinetics) or vs. wash time in s (prelabeling of cells with labeled nicotine and 
placement in 1000-fold dilute preincubation buffer for dissociation kinetics) 
are shown.  The association curve shows equilibrium to occur in 60 min.  If 
ln[Be/(Be-B)] vs. time is plotted, an association rate constant 
(<i>k</i><sub>1</sub>) of 1.1 x 10<sup>6</sup> min<sup>&minus;1</sup> is found. 
Dissociation curves show a half-time dissociation of 2 min.  A dissociation rate 
constant (<i>k</i><sub>&minus;1</sub>) of 0.343 min<sup>&minus;1</sup> 
mol<sup>&minus;1</sup> is found.  An equilibrium constant of 391 nM can be derived, 
which 20-fold higher than the<i>K</i><sub>D</sub>shown in the Scatchard analysis 
(to follow).  The discrepancy indicates that nicotine binding cannot be 
described as a simple reversible bimolecular reaction.
<p>
Plots of [<sup>3</sup>H]nicotine bound vs. the [<sup>3</sup>H]nicotine solution 
concentration in presence of 1 &micro;M &alpha;-BGT were done to examine the 
saturability of the binding curve.  Near-saturable binding was found to occur at 
45 nM nicotine.  A Scatchard plot (B/F vs. B) immediately yielded a plottable 
single line indicating a single class of receptor.  The<i>K</i><sub>D</sub>is 
found to be 20 &plusmn; 2 nM (<i>n</i>=4) and falls within range of values 
reported for rat and chicken brain membrane components.  The B<sub>max</sub> was 
found to be 104 fmol receptor/mg protein.  With various amounts of membranes 
added, the binding was linear.
<p>
Competition with other ligands (agonists and antagonists) was done with 20 nM 
[<sup>3</sup>H]nicotine and 1 &micro;M &alpha;-BGT present.  Competiting ligand 
is actually added 10 min prior to [<sup>3</sup>H]nicotine addition and then 
binding carried out at 70 min (22&deg;). Increasing concentrations of all agents 
eventually displaced the specifically-bound radionicotine.  Hill coefficients in 
all cases were near unity.  IC<sub>50</sub> are calculated in addition to Ki for 
the agonists (ACh, carbachol, and nicotine used) and for the antagonists 
(atropine, decamethonium, hexamethonium, mecamylamine, trimethaphan, 
<i>d</i>-tubocurarine used).  We have shown previously that the neuronal 
bungarotoxin (n-BGT) blocks nicotine-induced CA release from cultured chromaffin 
cells. Plots of [<sup>3</sup>H]nicotine bound vs [n-BGT] were done with 
membranes preincubated 40 min with the toxin in presence of 1 &micro;M 
&alpha;-BGT and then [<sup>3</sup>H]nicotine added for the standard reaction 
time:  At 10 &micro;M n-BGT, ~75% of the binding sites are blocked, again 
indicating further heterogeneity of nAChR with respect to n-BGT binding.
<p>
In another study, cultured cells are preloaded with <sup>3</sup>H-norepinephrine 
for CA release studies.  Washed cells are then stimulated for a 3 min period. 
Plots of <sup>3</sup>H-NE released as a % of total radioactivity in and out of 
the cell vs. [agonist] for nicotine alone, ACh in presence of 10 &micro;M 
eserine (an AChE antagonist) and 1 &micro;M atropine (a muscarinic antagonist), 
and carbachol in presence of 1 &micro;M atropine all show the same % release 
maxima (13% at their particular concentrations).  Nicotine was the most potent 
with maximal release at 10 &micro;M, ACh was next at 100 &micro;M, and carbachol 
had not quite reached its maximum release at 10 mM.  It is noteworthy that past 
their maximal release concentrations, the % release fell with higher [nicotine] 
and [ACh], indicating desensitization of the receptor.  10 &micro;M pilocarpine, 
oxotremorine, decamethonium, and hexamethonium were tested as well but did not 
cause CA release.  Pilocarpine and oxotremorine are muscarininc agonists, which 
have been shown to effect no CA release in chromaffin cells, consistent with 
previous reports.  Decamethonium is a partial muscle nicotinic agonist.  True nH 
(Hill coefficients) and EC<sub>50</sub> could not be calculated for nicotine, 
ACh and carbachol listed in the table because of the desensitization phenomena, 
however, electrophysiological measurements of EC<sub>50</sub> values for ACh 
(10-15 &micro;M) for rat adrenal chromaffin cells are in close agreement with 
the apparent EC<sub>50</sub> for ACh (15 &micro;M) given here.
<p>
Cells pre-loaded with <sup>3</sup>H-NE were then incubated with an antagonist 
(mecamylamine, <i>d</i>-tubocurarine, trimethaphan, hexamethonium, 
decamethonium,atropine, and &alpha;-BGT) for 1 h prior to incubation with 1 
&micro;M nicotine for a 3 min period.  CA release was plotted against 
[antagonist].  A table lists the calculated Hill coefficients (unity for all), 
the IC<sub>50</sub>, and the apparent Ki for all except &alpha;-BGT, which did 
not at all compete with nicotine up to the practical concentrations used, as 
expected. What the results show is that while antagonists demonstrate a 
classical pattern of inhibiting NE release for the agonist used, the agonists on 
the other hand show nonclassical stimulation that is attenuated at higher 
agonist concentrations.
<p>
Clapham &amp; Neher (1984) were the first to show desensitization of AChR using cell 
current measurements of chromaffin cells.  Cells preloaded with <sup>3</sup>H-NE 
were incubated with low [nicotine] and 10 &micro;M substance P for 1 h, with 
[nicotine] substimulatory for CA release.  A CA-releasing nicotine concentration 
was added for the 3 min period.  Substance P has been shown to enhance the rate 
but not extent of AChR desensitization.  Plots of CA release vs. stimulatory 
[nicotine] with three curves of substimulatory preincubation nicotine (at 0, 20, 
and 100 nM nicotine) indicates that all curves show the typical desensitization: 
the stimulating [nicotine] producing maximal release is at 10 &micro;M, and the 
release declines immediately with higher stimulating [nicotine].  What is 
different is that the preincubating desensitzing nicotine reduces the maximal 
release % of NE without changing the apparent EC<sub>50</sub>.  The effects were 
reversible after a 1 hr wash, which restored control levels of NE release.  If 
the % inhibition of NE release is plotted against the preincubation [nicotine], 
and this data is plotted as an Eadie-Hofstee plot of % inhibition vs. % 
inhibition/[nicotine] to produce a linear plot, the EC<sub>50</sub> for nicotine 
necessary to desensitize the AChR is found to be 20 nM, in agreement with the 
binding affinity for the nAChR site, showing them to be the same.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Grant, N. J., Demeneix, B., Aunis, D., Langley, O. K.
 </span>
 <span class="year">
(1988)
 </span>
 <span class="jname">
Neurosci.
 </span>
 <span class="jvol">
27,
 </span>
 <span class="pages">
717-726.
 </span>
<p class="reftitl">
Induction of neurofilament phosphorylation in cultured chromaffin cells
<p class="Para1">
Cells are grown in DME/F-12 (1:1) with 5% hiFCS, P/S/K modified for glucose, 
glutamine, and bicarb.  Cells were isolated on Percoll gradients.  Cells retain 
their in situ phenotype general under these conditions.  The monoclonals 
developed by Sternberger and Sternberger were used: they work against 
phosphorylated and nonphosphorylated 200 kDa neurofilament (NF200) epitopes.
<p>
Initial immunofluorescence studies show that label specific for 
nonophosphorylated NF200 is present.  No other possibilities exist.  It was in 
the perinuclear spot or filament strands.  There was very little to virtually no 
fluorescent staining for the phosphorylated antibodies.
<p>
Does neurite growth in chromaffin cells correlate with neurofilament 
phosphorylation?  Cells from the intermediate lobe were cultured and used to 
condition the medium that was then used to treat chromaffin cells:  neurite 
processes are observed to extend.  They develop in a week, and by 10 d about 40% 
of the cells had &ge; 1 neurites compared with 15% in control cultures (cultures 
not exposed to conditioned medium).  The immunostaining shows no phosphorylated 
neurofilaments in the processes;  what little fluorescence there was was in the 
perinuclear region.  Anti-tubulin antibodies did show that microtubule were 
present in the neurite-like extensions.
<p>
Day 17 of culture indicated the presence of &gt;50% of cells having neurites. 
Anti-nonphosphorylated NF indicated staining in the cell body and one neurite 
was always intensely stained in 55% of cells having neurites (this was true for 
a second MAb to anti-nonphospho NF).  Control culutres (i.e., those cells not 
receiving conditioned medium) showed little staining for NF-containing neurites. 
In the 17-day culture 40% of chromaffin cells with neurites stained with 
anti-phospho NF, but only the extensions and not the perikarya were stained. 
The immunofluorescen indicated that the staining was in the broadened neurite 
tip, but the edges of the tip were not stained.
<p>
Ultrastructural study of the 17-day old conditioned cells showed that the cell 
extensions contained numerous microtubules with thinner filaments discerned in 
some areas as well.  Immunoperoxidase staining with anti-phospho NF was 
associated with these linear arrays of tubulues and filaments;  the cell body 
and edge of neurite tip were unstained, further extending the immunofluorescence 
results.  In contrast, anti-nonphospho-NF antibody consistently stained both 
cell bodies and neurites.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
O'Sullivan, A. J., Burgoyne, R. D.
 </span>
 <span class="year">
(1988)
 </span>
 <span class="jname">
Bioscience Reports
 </span>
 <span class="jvol">
9,
 </span>
 <span class="pages">
243-252.
 </span>
<p class="reftitl">
A comparison of bradykinin, angiotensin II and muscarinic stimulation of culture 
bovine adrenal chromaffin cells
<p class="Para1">
Catecholamine (CA) release from cultured cells was examined for dose response to 
bradykinin (BK) and angiotensin II (A-II).  At the maximum dose for each, A-II 
elicited 1-2% release above basal, while BK elicited between 4-6% above basal. 
The very flat appearance of the response curves is typical of cells with 
multiple classes of receptors for these ligands. 
Des-Arg<sup>9</sup>-Leu<sup>8</sup>-bradykinin is a competitive B<sub>1</sub> antagonist 
and no effect on the bradykinin response up to 10 &micro;M, indicating all 
receptors are of the B<sub>2</sub> class.  The half-maximal doses for BK and A-II were 30 
nM and 3 nM, resp.
<p>
Time course of CA release was done for BK and A-II at 300 nM.  CA release for 
both was virtually identical, with 90% of the total release occurring in the 
first 2 min and complete by 4 min.  When both agonists were used in combination, 
the total release was slightly greater than for BK alone;  the level of 
secretion is therefore additive.
<p>
Ca<sup>2+</sup>-dependence was assessed by removing extracellular 
Ca<sup>2+</sup> (normally 3 mM CaCl<sub>2</sub> but instead 1 mM EGTA).  Even 
the basal level of secretion was much lower.  With stimulation by BK, A-II, and 
by 0.3 mM methacholine and 10 &micro;M nicotine, while there was increased 
release over basal containing no Ca<sup>2+</sup>, % release was not as high as 
with basal containing Ca<sup>2+</sup>.  It is possible that inhibition of 
muscarinic-, BK-, and A-II-induced release may result from depletion of internal 
Ca<sup>2+</sup> store by EGTA or by a process dependent upon entry of external 
Ca<sup>2+</sup>.
<p>
Cells were incubated 20 min in 2 &micro;M fura-2 ester for intracellular 
Ca<sup>2+</sup> measurements.  Cells removed from culture dishes have been shown 
to change their [Ca<sup>2+</sup>]<sub>i</sub> in a manner identical to cells 
freshly isolated and loaded with fura-2 and challenged with nicotine.  Basal 
[Ca<sup>2+</sup>]<sub>i</sub> is 80.2 &plusmn; 4.1 nM.  Fura-2-loaded cells in 
the presence or absence of 3 mM Ca are challenged with 10 &micro;M nicotine and 
the fluorometric response indicates a 200 nM rise in the 
Ca<sup>2+</sup>-containing medium which is 90% inhibited in Ca<sup>2+</sup>-free 
medium.  300 nM BK causes a rise about half that of the nicotine response, and 
300 nM A-II elicited an even smaller response.  300 &micro;M methacholin 
produced an ever smaller rise than A-II. Note in all of these stimulations, the 
[Ca<sup>2+</sup>]<sub>i</sub> returns to basal logarithmically depending upon 
the initial rise, with a half-life of 30 s to 1 min.  CA release measured 
correlated directly with the peak rise in [Ca<sup>2+</sup>]<sub>i</sub> produced 
by the agonists.  Even in Ca<sup>2+</sup>-free extracellular medium (EGTA added 
about 1 min prior to stimulation), all agonists but nicotine produced sharp 
rises in intracellular Ca<sup>2+</sup>, although the magnitude of rise was about 
50-75% of that produced in Ca<sup>2+</sup>-containing medium.
<p>
A prolonged elevation of intracellular Ca<sup>2+</sup> is observed with BK and 
nicotine stimulation, while methacholine and A-II induction is followed by a 
more rapid return to basal concentrations.  This prolonged phase is not observed 
in Ca<sup>2+</sup>-free medium.  Methoxyverapamil (MeOVP) is a Ca<sup>2+</sup> 
channel blocker and has no effect on the initial rise seen with BK stimulation. 
However, the prolongation of high intracellular Ca<sup>2+</sup> is no longer 
observed.  The time necessary for [Ca<sup>2+</sup>]<sub>i</sub> to decay by 90% 
was 5.0 min. in cells containing no channel blocker, but only 1.8 min in the 
presence of 100 &micro;M MeOVP. Therefore blocking voltage-dependent calcium 
channels has no effect on the initial rise of Ca<sup>2+</sup>, but blocks a 
secondary phase of entry.
<p>
BK, A-II, and methacholine would be expected to act through phospholipase C via 
a G protein signal transduction mechanism.  Pertussis toxin sensitive G-proteins 
in the presence of the toxin would inhibit this response.  BK, A-II, and 
methacholine stimulation in the toxin's presence, rather than inhibiting, 
instead enhanced the CA release observed to greater levels than would be 
observed with BK, A-II, or methacholine alone.  This enhancement was not the 
result of changes in the rise of [Ca<sup>2+</sup>]<sub>i</sub>, since the peak 
rises were not distinguishable in the presence or absence of toxin.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Bourke, J. E., Bunn, S. J., Marley, P. D., Livett, B. G.
 </span>
 <span class="year">
(1988)
 </span>
 <span class="jname">
Br. J. Pharmacol.
 </span>
 <span class="jvol">
93,
 </span>
 <span class="pages">
275-280.
 </span>
<p class="reftitl">
The effects of neosurugatoxin on evoked catecholamine secretion from bovine 
adrenal chromaffin cells
<p class="Para1">
Neosurugatoxin (NSTX) is a toxin found in Japanese ivory mollusc, <i>Babylonia 
japonica</i>.  Early studies showed it was selective in its effect on ganglionic 
nicotinic receptors relative to its effect on smooth muscle muscarinic 
receptors.  NSTX inhibited rat and guinea pig ileum contraction response by 
acting at the nicotinic ganglionic receptors facilitating postganglionic 
transmitter release.  NSTX did not inhibit contraction when ACh was applied 
exogenously.  In rat superior cervical ganglia, mollusc extracts depressed 
orthodromic transmission and antagonized the depolarizing action of carbachol 
with affecting the action of muscarine.  NSTX competes with 
[<sup>3</sup>H]nicotine binding to rat forebrain membranes but does not alter 
binding equilibria of [<sup>3</sup>H]quinuclidinyl benzilate.
<p>
Chromaffin cells were cultured in the typical fashion with 25,000 to 40,000 
cells per well (~2 cm<sup>2</sup>/well) in a 24-well collagen-coated dish. 
Cells were treated with increasing doses of neosurugatoxin and hexamethonium and 
CA release was measured by HPLC using electrochemical detection 5 min after 5 
&micro;M nicotine addition.  At 
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Yamamoto, T., Tsuda, T., Hamamori, Y., Nishimura, H., and Takai, Y.
 </span>
 <span class="year">
(1988)
 </span>
 <span class="jname">
J. Biochem.
 </span>
 <span class="jvol">
104,
 </span>
 <span class="pages">
53-56.
 </span>
<p class="reftitl">
Inhibition of DNA synthesis by protein kinase C-activating phorbol esters in 
NIH/3T3 cells
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Hoshi, M., Akiyama, T., Shinohara, Y., Miyata, Y., Ogawara, H., Nishida, E., and 
Sakai, H.
 </span>
 <span class="year">
(1988)
 </span>
 <span class="jname">
Eur. J. Biochem.
 </span>
 <span class="jvol">
174,
 </span>
 <span class="pages">
225-230.
 </span>
<p class="reftitl">
Protein kinase C-catalyzed phosphorylation of the microtubule-binding domain of 
microtubule-associated protein 2 inhibits its ability to induce tubulin 
polymerization
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Goldberg, Y., Glineur, C., Gesquiï¿½re, J.-C., Ricouart, A., Sap, J., Vennstrï¿½m, 
B., Ghysdael, J. 
 </span>
 <span class="year">
(1988)
 </span>
 <span class="jname">
EMBO J. 
 </span>
 <span class="jvol">
7,
 </span>
 <span class="pages">
2425-2433.
 </span>
<p class="reftitl">
Activation of protein kinase C or cAMP-dependent protein kinase increases 
phosphorylation of the c-erbA-encoded thyroid hormone receptor and of the 
v-erbA-encoded protein
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Wollheim, C. B., Dunne, M. J., Peter-Riesch, B., Bruzzone, R., Pozzan, T., and 
Petersen, O. H.
 </span>
 <span class="year">
(1988)
 </span>
 <span class="jname">
EMBO J. 
 </span>
 <span class="jvol">
7,
 </span>
 <span class="pages">
2443-2449.
 </span>
<p class="reftitl">
Activators of protein kinase C depolarize insulin-secreting cells by closing 
K<sup>+</sup> channels

<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Solomon, M., Booher, R., Kirschner, M., Beach, D. 
 </span>
 <span class="year">
(1988)
 </span>
 <span class="jname">
Cell
 </span>
 <span class="jvol">
54,
 </span>
 <span class="pages">
738-740.
 </span>

<p class="reftitl">
Cyclin in fission yeast [letter]
<!-- ============================================================= -->
<p class="tags">
HBsAg subtyping, nucleotide level
<p class="ref">
 <span class="authors">
Okamoto H, Tsuda F, Sakugawa H, Sastrosoewignjo RI, Imai M, Miyakawa Y, Mayumi M
 </span>
 <span class="year">
(1988)
 </span>
 <span class="jname">
J. Gen. Virol.
 </span>
 <span class="jvol">
69,
 </span>
 <span class="pages">
2575-2583.
 </span>
<p class="reftitl">
Typing hepatitis B virus by homology in nucleotide sequence: Comparison of 
surface antigen subtypes
<p class="Para1">
HBV nucleocapsid is 27 nm sphere containing core antigen, e antigen, and viral DNA
and the polymerase.  An envelope of surface antigen covers it with common group-specific
determinant <i>a</i>.  It also carries mutually exclusive subtype determinants
either <i>d</i> or <i>y</i> and <i>w</i> or <i>r</i> (see Le Bouvier 1971 and
Bancroft et al 1972).  Thus four major subtypes are <i>adw, adr, ayw, ayr</i>.  
These show a geographical distribution which reveals historical migration patterns.
There are numerous report of genomic DNA sequences of the subtypes, allowing
analysis at nucleotide level.  Divergence between two genomes of same subtypes
is smaller than between genomes between subtype, ranging from 1.1-2.7% for intrasubtypic
divergence and 8.5-10% for intersubtypic divergence.  However, subtype <i>ayr</i>
differs from <i>adr</i> only by 2.2%.
<p>
The 3215 bp genome of HBV <i>adw</i> for three sequences is show (one from Japan,
one from Okinawa, and another from Indonesia.  Four open-reading frames are 
identified:  pre-S region with S gene (1200 bp), pre-C region with C gene (636 bp),
X gene (462 bp), and P gene (2529 bp).  Pairwise sequence comparisons show 
125 b (3.9%) differences between mainland Japan and Okinwawa Pref., 144 b (4.5%) between
Okinawa and Indonesia, and 170 b (5.6%) differences between mainland Japan and Indonesia.
These were compared to 15 other HBV genomes previously reported or which are unpublished:
six <i>adw</i>, four <i>adr</i>, four <i>ayw</i> and one <i>ayr</i>.  Intergroup
difference with more than 8% mismatch.  Group A contains two subtyped <i>adw</i>,
group B has four <i>adw</i> subtypes, and group D contained four subtyped <i>ayw</i>
genomes.  Group C was heterogeneous with respect to subtypes:  three <i>adw</i>,
four <i>adr</i> and one <i>ayr</i>.    The <i>ayw</i>, <i>adr</i>, and <i>ayr</i>
genomes were heterogeneous insofar as genome sequence, but <i>adw</i> is not.
Nine <i>adw</i> genomes are distributed into groups A, B and C.  AT the amino acid
level the four ORFs are compared.
<p>
In general differences of the <i>d|y</i>, <i>w|r</i> subtypes arises at amino acid 
level of the 226 aa-long translation product of S gene.  Sequence divergence in 
genomes may not translate to epitope differences however.  The nucleotide substitution
rate is estimated to be 1.4 to 3.2 &times; 10<sup>&minus;5</sup> sites per year,
or 31-70,000 years per site.  This rate of substitution is 100-fold less than for
HIV <i>env</i> gene, and 10,000-fold less than for most DNA-based genomes, but it
is similar to retroviral genome rates.  HBV uses an RNA intermediate, and many RNA
viruses and retroviruses are susceptible to mutations, as they do not have proof-reading
enzymes to correct errors in duplication.
<p>
Some important notes:  two HBV <i>adw</i> clones in group A have 6 bp insertions
in the C gene (+2355 to 2356 relative to <i>Eco</i>RI site) that is not shared
in the 16 other clones.  All four <i>ayw</i> clones in group D have length 3182
which is shorter than 14 clones:  this results from 33 bp deletion between 2850
and 2882 in these clones.
<p>
The heterogeneity in <i>adw</i> gives at lest four subgroups:  <i>adw</i><sub>1</sub>,
<i>adw</i><sub>2</sub>, <i>adw</i><sub>3</sub> and <i>adw</i><sub>4</sub>.
<!-- ============================================================= -->
<p class="tags">
rHBsAg structure, mass spectrometry
<p class="ref">
 <span class="authors">
Hemling ME, Carr SA, Capiau C, Petre J
 </span>
 <span class="year">
(1988)
 </span>
 <span class="jname">
Biochemistry
 </span>
 <span class="jvol">
27,
 </span>
 <span class="pages">
699-705.
 </span>
<p class="reftitl">
Structural characterization of recombinant hepatitis B surface antigen protein 
by mass spectrometry
<p class="Para1">
ABSTRACT only.
The primary structure of recombinant hepatitis B surface antigen protein 
produced in yeast has been confirmed by mass spectrometric peptide mapping. 
These studies corroborate more than 85% of the amino acid sequence derived by 
sequencing of the gene and identified the presence of an acetyl moiety on 
approximately 70% of the NH2-terminal methionine residues. Prior to the present 
work, direct structural analysis was largely prevented by the insolubility of 
this integral membrane protein and its primary degradation fragments in aqueous 
buffers and by partial blockage of the NH2 terminus. These difficulties were 
overcome by preparative isolation using electroelution of the monomeric 226 
amino acid protein from a polyacrylamide electrophoretic gel in the presence of 
sodium dodecyl sulfate. Chymotryptic digestion of the reduced and 
carboxymethylated monomer produced a large number of small, predominantly 
hydrophobic peptides ideally suited for peptide mapping by fast atom bombardment 
mass spectrometry. The percentage of NH2-terminal methionine blocked by acetyl 
was determined by a new strategy involving cyanogen bromide cleavage, 
permethylation, and gas chromatography/mass spectrometry identification and 
quantitation of the N-methyl-N-acetylhomoserine produced.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Cadenas, E.
 </span>
 <span class="year">
(1989)
 </span>
 <span class="jname">
Annu. Rev. Biochem.
 </span>
 <span class="jvol">
58,
 </span>
 <span class="pages">
79-110.
 </span>
<p class="reftitl">
Biochemistry of oxygen toxicity
<p class="Para1">
<b>REVIEW</b>.  Reactive and
undesirable O<sub>2</sub> intermediates are a fact of life in aerobic biological
metabolism.  Some physiological
mechanisms have been developed to defend against the formation of these
intermediates.  The O<sub>2</sub> molecule is inert
kinetically because of its electronic structure.
Despite its even number of electrons, there are two unpaired
electrons in its molecular orbitals (MOs), and the molecule
is said to be in a <i>triplet ground state</i>, which means that
these unpaired electrons have the same spin quantum number.
O<sub>2</sub> cannot oxidize another atom by accepting
a pair of electrons from it, since it necessitates either
pairing the unpaired electrons and also changing the spin of one of them.
If O<sub>2</sub> were a singlet molecule (all electrons paired and with
opposite spins), then O<sub>2</sub> would react with everything in sight.
For this reason, O<sub>2</sub> generally oxidizes other
atoms in a series of one-electron transfer reactions, which explains why
superoxide and peroxide radicals are formed.
<p>
The formation of superoxide radical (&bull;O<sub>2</sub><sup>&minus;</sup>)
by one electron transfer to O<sub>2</sub> is thermodynamically unfavorable
(E<sub>0</sub> = &minus;0.33 V) because of the spin-forbidden
reaction.  Usually O<sub>2</sub> must react with
another paramagnetic center such as a transition metal (Fe or Cu) which have
unpaired electrons available for exchange coupling.
For example, Fe<sup>2+</sup>&nbsp;+&nbsp;O<sub>2</sub> &rarr;
Fe<sup>3+</sup>&nbsp;+&nbsp;&bull;O<sub>2</sub><sup>&minus;</sup>.
The superoxide might also be formed when certain electrophiles (quinoid
compounds) are reduced to semiquinones by flavoproteins in electron-transfer
chains, and O<sub>2</sub> may be reduced only one electron in this
<i>redox cycling</i> process.
Superoxide can disproportionate in aqueous neutral pH solutions as well
to form H<sub>2</sub>O<sub>2</sub>:

<p style="text-align:center;">
2 &bull;O<sub>2</sub><sup>&minus;</sup> + 2 H<sup>+</sup> &rarr; H<sub>2</sub>O<sub>2</sub>&nbsp;+&nbsp;O<sub>2</sub>

<p>
H<sub>2</sub>O<sub>2</sub> can decompose to form very reactive HO&bull;
radicals, which diffuses about 5-10 molecule diameters before reacting,
and is responsible for the greatest biological damage.
The generation of these hydroxyl radicals has been illustrated as
Fenton-type chemistry in which the appropriate state of a transition metal
(such as Fe<sup>II</sup> or Cu<sup>I</sup>) is generated by the superoxide
radical, and then  serves as catalyst:

<p style="text-align:center">
Fe<sup>III</sup>&nbsp;+&nbsp;&bull;O<sub>2</sub><sup>&minus;</sup>
&rarr; Fe<sup>II</sup>&nbsp;+&nbsp;O<sub>2</sub>

<p style="text-align:center">
Fe<sup>II</sup> + H<sub>2</sub>O<sub>2</sub> &rarr; Fe<sup>III</sup> + &bull;OH + OH<sup>&minus;</sup>

<p>
An important consideration is the form of the metal catalyst
which might be available to H<sub>2</sub>O<sub>2</sub>.
Iron complexed in the cell may be unavailable, while certainly the iron
released from hemoglobin can participate in such reaction.
In generating reactive hydroxyls, other oxidizing species
should also be considered, e.g., ferryl radical ([FeOH]<sup>3+</sup> or
[Fe=O]<sup>2+</sup>) and the perferryl radical ([Fe=O]<sup>3+</sup> or
[Fe<sup>3+</sup>-&bull;O<sub>2</sub><sup>&minus;</sup>]).
<p>
Certain transition metal complexes scavenge superoxide
radicals for the purpose of removing their damaging potential.
The disproportionation reaction above is mediated by ping-pong
binding of superoxide substrate in copper-bound proteins such
as superoxide dismutase and catalyzed by several ligands of the metal (L-Cu).

<p style="text-align:center">
L-Cu<sup>II</sup>&nbsp;+&nbsp;&bull;O<sub>2</sub><sup>&minus;</sup>
&rarr; L-Cu<sup>I</sup>&nbsp;+&nbsp;O<sub>2</sub>

<p style="text-align:center">
L-Cu<sup>I</sup>&nbsp;+&nbsp;&bull;O<sub>2</sub><sup>&minus;</sup>&nbsp;+&nbsp;2 H<sup>+</sup>
&rarr; L-Cu<sup>II</sup> + H<sub>2</sub>O<sub>2</sub>

<p>
These same enzymes can also catalyze the formation of the
HO radical as shown above in the Fenton-type production of HO radical, and
damage is prospect.  The HO radical is unique among oxy-radicals in that
it not only has a high electrophilicity and thermochemical reactivity,
but its production can also occur near DNA and can modify DNA by adding
to bases or abstracting H atoms.
<p>
The abundance of dihydroquinones and semiquinones in one electron transfer
reactions allows them to participate in metal-independent formation of
HO radical from H<sub>2</sub>O<sub>2</sub>:

<p style="text-align:center">
semiquinone radical&nbsp;+&nbsp;H<sub>2</sub>O<sub>2</sub> &rarr;
 quinone&nbsp;+&nbsp;OH<sup>&minus;</sup>&nbsp;+&nbsp;&bull;OH

<p>
These reactions have actually been demonstrated for menadione and
several ubiquinones.
<p>
In other biological reactions, carbon-centered free
radicals are sometimes formed in which O<sub>2</sub>
can quench their presence by reaction, forming peroxyl radicals.
Hydroperoxides can subsequently form from these products in a chain in
which another carbon radical is formed by abstraction of a proton in a reaction
with the peroxyl radical.

<p style="margin:2em auto;">
<img src="./imagesGeneralScienceHTML/PeroxyRadical.gif"
  alt="Peroxidation of a carbon radical">

<p>
One way to increase the reactivity of O<sub>2</sub> is by removing
the spin restriction and forming singlet oxygen
(<sup>1</sup>&delta;<i>g</i>).  (Another form of singlet oxygen,
(&Sigma;<i>g</i><sup>+</sup>), decays to the (<sup>1</sup>&Delta;<i>g</i>)
state before reacting with anything.)
Singlet oxygen is both electrophilic and nucleophilic.
It has been detected as being formed in
biological systems using specific singlet O<sub>2</sub> scavengers,
observing D<sub>2</sub>O solvent effects, and the fact that
singlet O<sub>2</sub> (unlike OH radical) oxidizes cholesterol
to form different products (5-&alpha;-hydroperoxide adducts).
Chemical trapping using water-soluble 9,10-bis(ethylene) anthracene
disulfate produces an endoperoxide which produces
low-level chemiluminescence, a sensitive method for determination of rate
constants.  Two different photons are
produced, in which <i>monomeric </i>singlet <sup>1</sup>O<sub>2</sub>
decays to the triplet state, <sup>3</sup>O<sub>2</sub>, emitting
a 1268 nm photon.  <i>Dimeric</i> singlets
<sup>1</sup>O<sub>2</sub>&mdash;<sup>1</sup>O<sub>2</sub> also generate
a 634 nm photon (one half the 1268 nm photon) to form the triplet state.
Detection of the infrared 1268 nm photon during enzymic reactions
clearly shows the singlet oxygen presence since no other molecular electronic
states emit in this region.
<p>
Is superoxide radical formation very commonplace in biochemical
metabolism?  Actually only 5% of the O<sub>2</sub> used in reactions appears to form the superoxide radical,
indicating that it is a minor pathway and not obligate in these reactions.
The question of whether cytochrome oxidase
produces &bull;O<sub>2</sub><sup>&minus;</sup> is not yet known;
SOD does not inhibit the oxidation of
cytochrome c by cytochrome oxidase, indicating that either reduction of bound
O<sub>2</sub> is a two-electron transfer, or that any
&bull;O<sub>2</sub><sup>&minus;</sup> formed during
1-electron transfer is not released.  (Cytochrome oxidase has also
been implicated in &bull;O<sub>2</sub><sup>&minus;</sup> scavenging
activities.)   Still, the mitochondrial electron-transfer
chain has been shown to be an excellent source of generating
&bull;O<sub>2</sub><sup>&minus;</sup> radicals.  A novel and active
&bull;O<sub>2</sub><sup>&minus;</sup> generator in heart mitochondria
has been described which has high affinity for
O<sub>2</sub>, is not involved in ATP-coupled
respiratory chain, requires NADH to initiate generation,
and releases the &bull;O<sub>2</sub><sup>&minus;</sup>
product into the extramitochondrial space.  It is noteworthy
that heart mitochondria contain an oxidoreductase which can dehydrogenate
<i>exogenous</i> NADH (NADH approaching from
within the cytosol rather than the matrix), and generate these
&bull;O<sub>2</sub><sup>&minus;</sup> radicals;  liver mitochondria
have no such exogenous NADH oxidizing ability.
<p>
Antioxidant defenses employed by biological systems
include the <b>primary</b> and <b>secondary</b> defenses,
the former being the biochemical and enzymatic systems
specifically employed to deal with damaging free-radical
intermediates that arise or may arise; these include &beta;-carotene,
uric acid, glutathione, ascorbic acid, and &alpha;-tocopherol.
The latter include certain proteo- and lipolytic enzymes, as
well as the DNA repair systems which work under any conditions
whether damage was oxidizing or not.
<p>
<i>Vitamin E.</i> Also called &alpha;-tocopherol, it reacts with
many free radicals rapidly.  In particular it reacts with
peroxy radicals which abstract an H atom from the antioxidant.

<p style="text-align:center;">
VitE-OH + ROO&bull; &rarr; VitE-O&bull; + ROOH
<p>
In a nonpolar environment is the possibility that electron
transfer occurs alone, with the antioxidant later being deprotonated in a
reaction with water immersed in teh lipid phase.

<p style="text-align:center;">
VitE-OH + ROO&bull; &rarr; VitE-OH&bull;<sup>+</sup> + ROO<sup>&minus;</sup>

<p style="text-align:center;">
VitE-OH&bull;<sup>+</sup> +  H<sub>2</sub>O &rarr;
VitE-O&bull; + H<sub>3</sub>O<sup>+</sup>

<p>
What is not explained is why hydroperoxides do not accumulate
for example in the &alpha;-tocopherol quenched peroxidation of lipids,
as expected in the reaction sequences shown above.
It is thought that &alpha;-tocopherol actually may
serve as a molecular &ldquo;channel.&rdquo;  In that
case, the antioxidant scavenges the <i>carbon-centered</i>
(R&bull;) radical prior to its reaction with O<sub>2</sub>,
providing it a proton for abstraction, with the antioxidant
radical undergoing intramolecular rearrangement to bring the
odd electron to a stable configuration.  This radical is
actually long-lived (becaused of its apparent stability),
and can react with another VitE radical (dimerize) or perhaps
with other extramembrane antioxidants (ascorbate) to form
ready VitE again (the odd electron is passed to the other antioxidants).
Ascorbate, uric acid, and serotonin have been shown to react
with the vitamin E-chromanoxyl radical, but glutathionyl
radical cannot react with one of its analogues (Trolox).
<p>
<i>Ascorbic acid.</i> As indicated above, it appears to be involved
in H transfer-mediated recovery of vitamin E radical.
It also directly reacts with peroxy radicals and with singlet O<sub>2</sub>.
Other excited states, such as triplet carbonyls, can elicit H
abstraction from the enediol groups of ascorbate.

<p style="text-align:center;">
ascorbateH<sup>&minus;</sup> + ROO&bull; &rarr;
ascorbateH&bull;<sup>&minus;</sup> + ROOH

<p style="text-align:center;">
ascorbateH<sup>&minus;</sup> + <sup>3</sup>[&gt;C=O]*
&rarr; ascorbate&bull;<sup>&minus;</sup> + &gt;C&bull;&minus;OH

<p>
<i>Glutathione</i>.  Generally reacts quickly with free radicals,
and particularly those produced during radiolysis of H<sub>2</sub>O.
Peroxy radicals are also quenched by abstraction of electrons from the
sulfur atom:

<p style="text-align:center;">
GS<sup>&minus;</sup> + ROO&bull; &rarr; GS&bull; + ROO<sup>&minus;</sup>

<p>
The thiyl radical subsequently formed can be quenched by O<sub>2</sub>,
undergo diradical annihiliation, be oxidized by GSH, and subsequently
react again with its product:

<p style="text-align:center;">
GS&bull; + O<sub>2</sub> &rarr; GSOO&bull; <br>
GS&bull; + GS&bull; &rarr; GSSG <br>
GS&bull; + GS<sup>&minus;</sup> &rarr; [GSSG&bull;]<sup>&minus;</sup> <br>
GS&bull; + [GSSG&bull;]<sup>&minus;</sup> &rarr; GS<sup>&minus;</sup> + GSSG

<p>
The reaction of O<sub>2</sub> with GS&bull; to produce the sulfenyl-peroxo
radical is not necessary a molecule of low reactivity and has been
implicated in inactivating lysozyme and trypsin.  The thiol reacts
with excited states, including singlet O<sub>2</sub> and triplet carbonyls.
Other thiols (cysteine, N-acetylcysteine, methionine) react similarly.

<p>
<i>Uric acid</i>. May be more important than ascorbic acid
insofar as trapping peroxy radicals in the aqueous phase.
Urate has been found to inactivate alcohol dehydrogenase however,
which may occur after it has trapped radicals and forms a toxic compound.

<p>
<b>Metabolism of hydroperoxides</b>.  Once hydroperoxides form in organic
molecules as a result of oxygen reaction with carbon-centered radicals, the
cells must have a way of dealing with it.  Several hemoproteins and
halide-dependent enzymic reactions take place to deal with these products.
One reaction is a disproportionation reaction forming singlet O<sub>2</sub>,
conducted by halide-mediated lacto-, myelo-, and chloroperoxidases;
the prostaglandin-endoperoxided synthase reaction; and hydroperoxide
metabolism by cytochrome P<sub>450</sub>.
The peroxidases proceed via the well-known chemical disproportionation
of H<sub>2</sub>O<sub>2</sub> by hypohalous acid (OCl<sup>&minus;</sup>)
formed by the enzymes to form reactive singlet O<sub>2</sub>:

<p style='text-align:center;text-indent:0in'>
X<sup>&minus;</sup>&nbsp;+&nbsp;H<sup>+</sup> + H<sub>2</sub>O<sub>2</sub>
&rarr; HO&minus;X  + H<sub>2</sub>O

<p style="text-align:center;">
HO&minus;X + H<sub>2</sub>O<sub>2</sub> &rarr;
H<sup>+</sup>&nbsp;+&nbsp;X<sup>&minus;</sup>&nbsp;+&nbsp;H<sub>2</sub>O&nbsp;+&nbsp;<sup>1</sup>O<sub>2</sub>

<p>
Note that two molecules of H<sub>2</sub>O<sub>2</sub> are used in the reaction.
<p>[continued]

<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Suzuki, M.
 </span>
 <span class="year">
(1989)
 </span>
 <span class="jname">
J. Mol. Biol.
 </span>
 <span class="jvol">
207,
 </span>
 <span class="pages">
61-84
 </span>
<p class="reftitl">
SPXX, a frequent sequence motif in gene regulatory proteins
<p class="Para1">
DNA-binding proteins have a variety of known secondary
structures which include the &alpha;-helix, the &beta;-sheet, and Zn fingers.
A new structure is proposed which has a &beta;-turn I.  This is stabilized by two
hydrogen bonds and composed of an SPKK sequence.  The lysines are
important for DNA-binding but the SP has a structure unimportant to
that function.  The hydroxyl hydrogen of Ser is involved the stabilization
of the bend caused by Pro.
<p>
The primary structures of 135 gene regulatory proteins
were studied:  38 had Zn fingers, 41 helix-turn-helix and 56 miscellaneous.
Klug and Rhodes (<i>TIBS</i> <b>12</b>: 464, 1987) have shown the
importance of the Zn finger, with its 4 Cys or His residues coordinated
around a Zn atom;  TF-IIIA is a paradigmatic protein in this regard.
Amino acid between the 2nd and 3rd coordinating Cys or His are thought
to be involved in DNA binding.  Other proteins contain a
homeobox, with is thought to form the helix-turn-helix motif.
<p>
Steroid hormone receptors have 4 regions:
the N-terminal &ldquo;hypervariable&rdquo; region,
called  Symbol'>t 1 or immunogenic; the Zn
finger region in which 2 Zn fingers are in the center, immediately followed by
a hinge region; and a C-terminal steroid hormone-binding region.  The
length of the hypervariable region differs between steroid-binding
proteins, but the length of the other three is conserved.
<p>
Of the 4028 proteins surveyed in the database, 2748 have
SPXX and 2932 have TPXX sequences  The total and effective
number of residues are 963,031 and 950,947, resp.;  the effective
number is the total minus the SPX or TPX residues that form the
C-terminus, which cannot make the SPXX or TPXX motif.  SPXX and
TPXX occur in these proteins with a frequency of 0.289% and 0.308% resp.
The probability of SP combinations occurring randomly in proteins is
actually 0.376%, so it is not likely that formation of
an SP is important compared with other combinations having Ser and Pro.
The frequency of these motifs in gene regulatory proteins, such as the
<i>krï¿½ppel</i> gene product is much higher than the total of the proteins.
In all proteins, SPXX and TPXX occur with
about the same frequency, whereas in the gene regulatory proteins SPXX is
higher than that of TPXX.  Since total
Ser and Pro residues occur in these class of proteins at a higher frequency as
well compared to total proteins, it seems likely that occurrence of SPXX
sequences have some functional value.
<p>
The location of (S/T)PXX sequences within gene regulatory
proteins is interesting.  In 38 Zn
finger proteins studied are 298 S(T)PXX sequences, none of them in the Zn
finger regions nor in their linker sequences connecting the fingers.
However they tend to be on either side.
In krï¿½ppel gene product N-terminus is:
<b>SPPASPHSPLSTPLG</b>SGKHPLN<b>SPN</b><br>
<b>STPQH</b> where the SPXX and TPXX motifs are underlined.
A 5 times repeat of Zn fingers is in the
center, containing no such sequences and is the sandwiched by two short
repeats:  <b>SPPS</b>SGS<b>SPNS</b> and <b>SPPTPALSPAM</b>.
In steroid-binding proteins, these sequences
occur not in the Zn finger or hormone-binding regions, but mostly in the
N-terminal hypervariable region and in the C-terminus in some family members
with longer C-termini (C-<i>erb</i>A-T).
In helix-turn-helix proteins there are no
S(T)PXX these regions, not even the longer region called the homeobox composed
of 60 amino acids.  There are none in
the Cro protein.

<p>
In some proteins known to interact with gene control, a
functional motif called a leucine zipper has been identified,
in which proteins with an &alpha;-helix have leucines aligned
on one side of the helix which interact with leucines on another monomer to
form a dimer (C/EBP; <i>myc, fos</i>, and <i>jun</i> gene products; GCN4).
These proteins also have basic regions, and the occurrence of the
S(T)PXX sequences is neither in the zipper nor the basic regions.
Hence, these sites occur in regions outside
the functional areas known to interact with DNA or some other ligand.

<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Futai, M., Noumi, T., Maeda, M.
 </span>
 <span class="year">
(1989)
 </span>
 <span class="jname">
Annu. Rev. Biochem.
 </span>
 <span class="jvol">
58,
 </span>
 <span class="pages">
111-136.
 </span>
<p class="reftitl">
ATP synthase (H<sup>+</sup>-ATPase):
results by combined biochemical and molecular biological approaches
<p class="Para1">
<b>REVIEW</b>.  ATP synthase is
found in the membranes of mitochondria, chloroplasts, and bacteria.
One large subunit complex, F<sub>1</sub>, is an ATPase and the other large
complex, F<sub>0</sub>, is a H<sup>+</sup> ion channel.
Together they couple the reversible synthesis of ATP from H<sup>+</sup>
translocation across membranes with electrochemical gradients.  In
mitochondria they make up the ATP synthase called complex V in oxidative
phosphorylation chain.  The proteins in eukaryotes generally have similar
structures but are more complex than those in bacteria.
All F<sub>1</sub> complexes are composed of five subunits
(&alpha;, &beta;, &gamma;, &delta;, &epsilon;) which can be
detached from membranes as an active ATPase.  F<sub>0</sub> is an
integral membrane protein which becomes a passive proton channel
if the F<sub>1</sub> complex is removed.  In <i>E coli</i> F<sub>0</sub>
complexes are made of only three subunits (<i>a</i>, <i>b</i>,
<i>c</i>) but the eukaryotic F<sub>0</sub> assemblies are more
complex (see table).

<table style="margin:1em auto;">
<tr>
 <th><i>E coli</i>
 <th>chloroplasts
 <th>mitochondria
<tr>
 <td>&alpha;
 <td>&alpha;
 <td>&alpha;
<tr>
 <td>&beta;
 <td>&beta;
 <td>&beta;
<tr>
 <td>&gamma;
 <td>&gamma;
 <td>&gamma;
<tr>
 <td>&delta;
 <td>&delta;
 <td>OSCP
<tr>
 <td>&epsilon;
 <td>&epsilon;
 <td>&delta;
<tr>
 <td colspan="2">
 <td>&epsilon;
<tr>
 <td>
 <td>subunit II
 <td>
<tr>
 <td><i>a</i>
 <td>subunit IV
 <td>subunit 6
<tr>
 <td><i>b</i>
 <td>subunit I
 <td>subunit 4 (subunit b)
<tr>
 <td><i>c</i>
 <td>subunit III
 <td>subunit 9
<tr>
 <td>
 <td>
 <td>subunit 8 (A6L, <i>aap1</i>)
<tr>
 <td>
 <td>
 <td>ATPase inhibitor protein
<tr>
 <td>
 <td>
 <td>9 kDa protein, 15 kDa protein
<tr>
 <td>
 <td>
 <td>Factor B, F6, subunit <i>d</i>
</table>

<p>
The <i>unc</i>operon of the E. coli genome codes for all of this bacterium&rsquo;s
subunits of F<sub>0</sub>F<sub>1</sub>.  The entire operon has been cloned
and sequenced, with the gene order <i>uncB</i> (<i>a</i>), <i>uncE</i>
(<i>c</i>), <i>uncF</i> (<i>b</i>), <i>uncH</i> (&delta;),
<i>uncA</i> (&alpha;), <i>uncG</i> (&gamma;), <i>uncD</i> (&beta;),
and <i>uncC</i> (&epsilon;).  Another open reading frame <i>uncI</i>
is upstream of <i>uncB</i> and its role is unknown.  A promoter element
has been identified upstream of <i>uncI</i>, but two weak
promoters are also to be found within the <i>uncI</i>
gene are found.  A terminator sequence is located downstream of
<i>uncC</i>.  Operons in other bacteria (<i>Rhodospirillium
rubrum</i> and <i>Rhodopseudomonas blastica</i>) with the same gene order
have been identified in other bacteria, with minor variations.
In the cyanobacteria <i>Synechococcus</i> and <i>Anabaena</i>,
two gene clusters code for the F<sub>0</sub>F<sub>1</sub> set:
<i>atp1</i> encodes gene I (similar to <i>uncI</i>),
<i>a</i>, <i>c</i>, <i>b</i>, and <i>b</i>&rsquo; (diverged form of <i>b</i>),
and three of the F<sub>1</sub> genes (&alpha;, &gamma;, &delta;).
The other two F<sub>1</sub> subunits are encoded on <i>atp2</i>.
<p>
Organellar genomes in eukaryotes code for many of the subunits of
F<sub>0</sub>F<sub>1</sub>.  As in the blue greens, the &beta; and
&epsilon; subunits in the chloroplast genome form their own single
transcriptional unit.  Genes IV, III, and I (<i>a</i>, <i>c</i>, and
<i>b</i> in <i>E coli</i>) and the <i>a</i> subunit in the same order
as in <i>E coli</i>, but gene I appears to contain an intron.
Yeast mitochondrial DNA encodes the subunits 9 (<i>oli1</i>) (c),
8 (aap1), and 6, but the mammalian mtDNA encodes for
only subunits 6 and 8 (A6L).
<p>
cDNAs have been cloned and sequenced for the F<sub>1</sub> &beta;
subunits (human, rat, bovine, yeast), &alpha; subunit (yeast),
&gamma; subunit (in chloroplast), <i>c</i> subunit (N. crassa,
bovine, human), <i>b</i> subunit (bovine), <i>d</i> subunit (bovine)
and subunit 4 (yeast).  Interesting are the mechanisms
which coordinate expression of the nuclear genes and the
organellar genomes.
<p>
F<sub>1</sub> subunits show primary structures typical of soluble,
globular proteins, which many hydrophilic residues.  &beta; subunits
show the greatest conservation between species, with 42% identity
between 20 compared subunits across the phylogenetic spectrum.
&alpha; subunits show some, although less homology
in species compared;  in particular,
three regions show conservation (residues 148-186, 257-303, and
322-378 in one particular F<sub>1</sub> &alpha; subunit), and they
may be involved in catalysis of subunit-subunit interaction.
In the &gamma; subunit, only COOH terminal regions are conserved
in about 8 species and may be essential for
catalysis or assembly; the chloroplast g subunit unit shows 55% (the highest)
homology with cyanobacteria, and only 33% homology with E coli.
The chloroplast/blue-green bacterial g
subunit also show an extra 40 amino acid domain not present in other
groups.  &delta; and &epsilon; subunits show limited homology
between species in the carboxyl and amino terminal regions, resp.
<p>
ATPase inhibitor proteins exist in these assemblies by
binding both the native F<sub>0</sub>F<sub>1</sub> and to soluble
F<sub>1</sub>, an interaction stabilized by 9 and 15
kDa proteins (as shown in yeast).  These two proteins
show homology to the ATPase inhibitors found elsewhere.
The &epsilon; subunit also shows limited homology with
the inhibitor protein as well, consistent with an intrinsic
inhibitory function identified with it and with
its ability to alter &beta; subunit conformation.
<p>
The F<sub>0</sub> <i>a</i>subunit shows little homology between
species despite similar hydropathy profiles and some conservation of polar
residues in the COOH terminus.  The E coli subunit <i>b</i> is
hydrophilic with a 48% polar residue content, and a stretch of
23 hydrophobic residues is at the amino terminus.  This bacterial subunit
also demonstrates some homology with the subunit I in the plant and
blue-green F<sub>0</sub>F<sub>1</sub>, and corresponds to the mammalian
subunit <i>b</i> and the yeast subunit 4.  Subunit <i>c</i> (subunit 9 or III)
is a proteolipid or DCCD-binding protein and is very hydrophobic; DCCD binds to
a Glu or Asp in the middle of the second hydrophobic stretch in the
protein.  It also has two alpha helices which traverse the membrane
in an antiparallel fashion with a central polar axis.
<p>
The E coli subunit stoichiometry for F<sub>0</sub>F<sub>1</sub>
has been shown to be
&alpha;<sub>3</sub>&beta;<sub>3</sub>&gamma;&delta;&epsilon;<i>ab</i><sub>2</sub><i>c</i><sub>6-10</sub>.
Low resolution F<sub>1</sub> crystal structures show a
pseudohexagonal arrangement (no crystals have been obtained
for F<sub>0</sub>F<sub>1</sub>.)
Reconstitution experiments suggest that F<sub>0</sub> interacts with
the F<sub>1</sub> assembly through the &delta; and &epsilon; subunits.
EM of negatively stained F<sub>0</sub>F<sub>1</sub> from E coli shows
a sphere of 9 nm dia. attached to the bilayer by a 4.5 nm stalk.
A phospholipid photoaffinity analog was found to attach to the
&alpha; and &beta; subunits very poorly, while the <i>a</i> and
<i>b</i> subunits were very well labeled.  Disulfide crosslinking between
a and d subunits indicates a close interaction. Both hydrophobic
and water-soluble aldehydes react with the &delta; subunit from
solubilized F<sub>1</sub>, but not with F<sub>0</sub>F<sub>1</sub>
complex, indicating that the residues are protected by elements of
F<sub>0</sub>.  The &delta; subunit may not be part of
the stalk, since F<sub>1</sub> can bind to submitochondrial
particles depleted of F<sub>1</sub> without the presence of
OSCP (the &delta; protein in mitochondria).  OSCP shows it
may be between the interfaces of &alpha; and &beta; subunit
based on reactions with zero-length crosslinkers.  Based on
a combination of chemical crosslinking studies and on EM
of immunostained preparations, it is likely that <i>a</i> and <i>b</i>
subunit are arranged alternately to form a hexganol array
which is somewhat staggered such that subunit interactions
&alpha;-&beta;, &alpha;-&alpha;, and &beta;-&beta; occur.
<p>
Reconstitution of &alpha;, &beta;, and &gamma;
alone are sufficient to produce an active F<sub>1</sub>
ATPase;  the need for &gamma; subunit is not known since the catalytic
site is either on the &beta; subunit or at the interface between &alpha;
and &beta;.  Mixing subunits from different bacterial species (E coli and
S. typhimurium) also produced active ATPases.  It is likely that the
one-half of the &gamma; subunit which shows homology between species
is involved in assembly of the &alpha; and &beta; subunits.  There
are also many cases of complementation between <i>a</i> and <i>b</i> subunits
between bacterial species as shown in
reconstitution experiments producing active ATPases.
Study of mutant organisms has give a more precise definition of
the regions of the g subunit necessary for F<sub>1</sub>
assembly:  these are the region between
Gln-269 and the COOH terminus and Lys-21 to Ala-27 in the NH<sub>2</sub>
terminus.  It is noteworthy that deletion of the <b>A</b>A<b>A</b>V
sequence at the C-terminus still permits formation of active
F<sub>0</sub>F<sub>1</sub> complex although the two A residues
underlined are conserved in many
species.  Single substitutions in the &alpha; and &beta;
subunit were often sufficient disrupting normal
F<sub>0</sub>F<sub>1</sub> complexes.  Several such mutants were
cloned and sequenced and included &beta;Glu-40&rarr;Lys,
&beta;Glu-185&rarr;Lys, &beta;Gly-223&rarr;Asp, &beta;Ser-292-Phe,
&beta;Gln-361&rarr;end, and &beta;Gln-397&rarr;end.
The missense mutations generally resulted in poor subunit
assembly.  The importance of bGlu-185 was also demonstrated
since complexes did not form well.  A table is given which
shows which mutations result in either assembly or catalytic
dysfunctions or losses.  This shows that a subunit residues are
clearly necessary for F<sub>1</sub> binding to the membrane.
<p>
Alterations in the expression of the &delta; and &epsilon;
subunit were shown to affect &alpha;&beta;&gamma;
complex binding to F<sub>0</sub>.  The &epsilon; subunit amino
terminal portion seemed sufficient for allowing active F<sub>1</sub>
to bind to F<sub>0</sub>, based on used of truncated uncC gene.
A particular mutation (&epsilon;Gly-48&rarr;Asp) was sufficient to
eliminate and ATP-dependent H<sup>+</sup>-translocation.
A pseudorevertant (ePro-47&rarr;Ser or Thr) indicated that the
peptide oxygen between ePro-47 and eGly-48 is somehow involved in hydrogen
bonding or in H<sup>+</sup> movement.
<p>
[continued]
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Otvos, Jr., L., Elekes, I., Lee, V. M.-Y.
 </span>
 <span class="year">
(1989)
 </span>
 <span class="jname">
Int. J. Peptide Protein Res.
 </span>
 <span class="jvol">
34,
 </span>
 <span class="pages">
129-133.
 </span>
<p class="reftitl">
Solid-phase synthesis of phosphopeptides
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Glass, D. B., Cheng, H.-C., Mende-Mueller, L., Reed, J., Walsh, D. A.
 </span>
 <span class="year">
(1989)
 </span>
 <span class="jname">
J. Biol. Chem.
 </span>
 <span class="jvol">
264,
 </span>
 <span class="pages">
8802-8810.
 </span>
<p class="reftitl">
Primary structural determinants essential for potent inhibition
of cAMP-dependent protein kinase by inhibitory peptides corresponding to the
active portion of the heat-stable inhibitor protein
<p class="Para1">
<b>INTRO</b>.  Although protein kinase inhibitor (PKI) is 75 amino acids
long, the portion of the molecule primarily responsible for potent
inhibition of PK-A has been narrowed to the N-terminal region from
residues 5-24 (TTYADFIASGRTGRRNAIHD), determined using peptides
obtained from V8 protease digestion.  Using peptide or protein
substrates for PK-A, inhibition with PKI is always competitive
(<i>K</i><sub>i</sub>=2.3 nM).  The RTGRRNAI sequence makes up the
pseudo-substrate region, but the residues on the N-terminal
side of this site are necessary for potent inhibition of mammalian kinase.
<p>
<b>RESULTS</b>.  In all PK-A assays, Kemptide is the substrate
(10 min/30&deg;/30 mM MES, pH 6.8&nbsp;+&nbsp;5 mM MgOAc&nbsp;+&nbsp;125 &micro;M ATP [200-300
cpm/pmol]/22.5 &micro;M kemptide/3.75 mM 2-mercaptoethanol/0.4 mg/ml BSA
and 0.1 &micro;g/ml or 2.5 nM catalytic subunit).  Plots of PK-A activity
against log[inhibitor peptide] produce slopes have parallel linear
response, indicating that inhibitor interacts in a similar fashion.
Hill coefficients of between 0.85 an 1.15 are obtained.
Double-reciprocal and Henderson
([total inhibitor peptide]/{1&minus;<i>v</i><sub>i</sub>/<i>v</i><sub>0</sub>} vs.
<i>v</i><sub>0</sub>/<i>v</i><sub>i</sub>) plots are linear for all inhibitor
peptide concentrations.  Both the double
reciprocal data and the linearity obtained with plotting the Henderson
slope vs. [Kemptide] curves indicate competitive inhibition in with
all inhibitors.
<p>
Peptide length was altered in PKI(5-24)amide.  Previous reports
indicated that His-23 and Asp-24 were not required for inhibition,
as PKI(5-22)amide had sufficiently potent inhibition.  PKI(11-24)amide
caused a 50-fold increase in the <i>K</i><sub>i</sub> value:  Removal
of residues 5-9 results in 30-fold decrease in inhibition and
further deleting Phe-10 caused a 50% additional loss of potency.
Further deletion of Ile-11 results in no change, but removal of
Ala-12 and Ser-13 allowed a regain of inhibitory action.
Removal of Gly-14 and especially of Arg-15 produced very weak inhibitors
(the significance of Arg-15 has been shown in previous work).
<p>
Substitution of PKI(5-22)amide was tested because of the
apparent high affinity for the N-terminal to the PK-A catalytic subunit.
A singly-substituted peptide was synthesized
for each position 5-11 in the N-terminal portion, with Ala being the
substituting residue.  Substitution of
Thr-5 by sarcosine was without effect, whereas substitution of Thr-6, Asp-9,
Ile-11, and Ala-8 (using Leu), resulted in a 3-fold
<i>K</i><sub>i</sub> increase in these peptides.  The Tyr-7 substitution
produced a greater loss of inhibition.  When Phe-10 was
substituted, the loss was two orders of magnitude, suggesting that
Phe-10 plays a critical role in the inhibitory activity of this peptide!
<p>
Since substitution studies of the COOH terminal pseudo-substrate
site have been done insofar as substitution of the three Arg
residues and Ala-21, other residues were substituted and tested,
in which the parent was either PKI(5-22)amide or PKI(14-22)amide.
Substitution of Gly-17 with Leu or Asn-20 with Ala had significant
effects on the inhibitory constant, resulting in between 7- and
15-fold increases in <i>K</i><sub>i</sub> values.  Doubly substituted Gly-17 and
Asn-20 peptide showed that the decrease in inhibitory activity was
synergistic.  Thr-16 was replaced with
its analog, 2-aminobutyric acid, and this had no effect in
PKI(5-22)amide; replacement with Ala in PKI(14-22)amide
caused a 4-fold decrease in potency.  The most dramatic
effect was to substitute Ile-22 with Gly in PKI(5-22)amide:
maximum inhibitory activity required the presence of this
large hydrophobic residue.  If Ile-22 was substituted by Leu, modest
increases in <i>K</i><sub>i</sub> are observed, but substitution with Gly
results in 150- and 40-fold increases in <i>K</i><sub>i</sub> values of
PKI(5-22)- and PKI(14-22)amide, resp.  The inhibitory
potency of this substitution is lost more with the Gly substitution than with
PKI(14-22)amide, which represents the loss of the important N-terminal region.
<p>
CD spectral studies and Chou &amp; Fasman analyses
indicate that the secondary structure of PKI(5-22)amide may contain a
&beta;-turn, in the middle of the peptide between Ala-12 and Arg-15
(or Ser-13 and Thr-16), as well as in the pseudosubstrate
region between Gly-17 and Asn-20.  The detrimental effects
seen with substitution of Gly-17 with Leu and Asn-20 for Ala
suggests that perturbation of the structure may be occurring.
A doubly substituted PKI(5-22)amide peptide (Ser-13 --&gt; Leu
and Gly-14 --&gt; Ile), since it apparently reduces the ability
of the peptide to form a &beta;-turn structure:  the results
show it to be a poor inhibitor, having a <i>K</i><sub>i</sub> which
is 42-fold higher than the parent.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Glass, D. B., Lundquist, L. J., Katz, B. M., Walsh, D. A.
 </span>
 <span class="year">
(1989)
 </span>
 <span class="jname">
J. Biol. Chem.
 </span>
 <span class="jvol">
264,
 </span>
 <span class="pages">
14579-14584.
 </span>
<p class="reftitl">
Protein kinase inhibitor-(6-22)-amide peptide analogs with
standard and nonstandard amino acid substitutions for phenylalanine 10:
inhibition of cAMP-dependent protein kinase
<p class="Para1">
PKA catalytic subunit was titrated with selected analogs of
PKI(6-22)amide, the peptide corresponding to the minimally
necessary inhibitory portion of PKI.  17 peptides
corresponding to 17 substitutions of the amino acid Phe at
position 10 were shown to produce IC<sub>50</sub> values
over a 3.5-order of magnitude range.  At sufficiently high
concentrations, inhibition could be maximal.  The Hill plots
(log{<i>V</i><sub>i</sub>/(<i>V</i><sub>0</sub>-<i>V</i><sub>i</sub>)}
vs log[Inhibitor]) show Hill coefficients of 0.90.
Henderson and double-reciprocal plots both indicated that
PKI(6-22)amide analogs were competitive with respect to Kemptide.
Replots of either slope vs. [Kemptide] (Henderson plots) or
slope vs. [inhibitor] (double-reciprocal show linearity and produce
<i>K</i><sub>i</sub> values.  For the parent peptide, <i>K</i><sub>i</sub>
was 1.6 nM, in agreement with previous data.
<p>
The effect of substitution is analyzed for each:
when Phe-10 is replaced with Ala, there is
over a 260-fold increase in <i>K</i><sub>i</sub>.
Substitution with Leu results in a 33-fold
increase, an 8-fold greater potency.  If
 <span style="font-variant:small-caps">d</span>-Phe is used to replace  <span style="font-variant:small-caps">l</span>-Phe-10, this is
less potent than even the Leu analog by half (double the
<i>K</i><sub>i</sub> for Leu);  hence location of the phenyl ring
must be important.  Use of homophenylalanine instead was
even less potent than the <span style="font-variant:small-caps">d</span>-Phe analog (<i>K</i><sub>i</sub>=115 nM)
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Reed, J., De Ropp, J. S., Trewhella, J., Glass, D. B., Liddle, W. K.,
Bradbury, E. M., Walsh, D. A.
 </span>
 <span class="year">
(1989)
 </span>
 <span class="jname">
Biochem. J.
 </span>
 <span class="jvol">
264,
 </span>
 <span class="pages">
371-380.
 </span>
<p class="reftitl">
Conformational analysis of PKI(5-22)amide, the active inhibitory
fragment of the inhibitor protein of the cyclic AMP-dependent protein kinase
<p class="Para1">
PKI(5-22)amide is synthesized (TTYADFIASGRTGRRNAI-nh<sub>2</sub>)
in addition to some of its derivatives shown in the table.

<table>
<tr>
 <th>Peptide
 <th>Sequence
 <th><i>K</i><sub>i</sub> (nM)
<tr>
 <td>PKI(5-22)amide
 <td>TTYADFIASGRTGRRNAI-nh<sub>2</sub>
 <td>3.1
<tr>
 <td colspan="3"><b>Mono-substituted derivatives</b>
<tr>
 <td>[Ala<sup>6</sup>]PKI(5-22)amide
 <td>T<b>A</b>YADFIASGRTGRRNAI-nh<sub>2</sub>
 <td>7.1
<tr>
 <td>[Ala<sup>7</sup>PKI(5-22)amide
 <td>TT<b>A</b>ADFIASGRTGRRNAI-nh<sub>>2</sub>
 <td>14
<tr>
 <td>[Leu<sup>8</sup>]PKI(5-22)amide
 <td>TTY<b>L</b>DFIASGRTGRRNAI-nh<sub>2</sub>
 <td>8.9
<tr>
 <td>[Ala<sup>9</sup>]PKI(5-22)amide
 <td>TTYA<b>A</b>FIASGRTGRRNAI-nh<sub>2</sub>
 <td>8.2
<tr>
 <td>[Ala<sup>10</sup>]PKI(5-22)amide
 <td>TTYAD<b>A</b>IASGRTGRRNAI-nh<sub>2</sub>
 <td>270
<tr>
 <td>[Ala<sup>11</sup>]PKI(5-22)amide
 <td>TTYADF<b>A</b>ASGRTGRRNAI-nh<sub>2</sub>
 <td>8.1
<tr>
 <td colspan="3"><b>Truncated peptide</b>
<tr>
 <td>PKI(14-22)amide
 <td>GRTGRRNAI-nh<sub>2</sub>
 <td>36
</table>

<p>
IR spectra were taken of the parent peptide, the
Ala-10-substituted peptide, and the truncated peptide (subtracted from
buffer).  Fourier-deconvolved spectrum
of the parent indicated 5 partially resolvable peaks assigned frequencies of
1585, 1610, 1639, 1653, and 1674 cm<sup>&minus;1</sup>.
Resolution of 2nd derivative spectra
indicated similar assignments, except the band at 1639 was resolved into two
bands of 1634 and 1645 cm<sup>&minus;1</sup>.
The 1620 to 1720 cm<sup>&minus;1</sup> range corresponds to carbonyl group
stretching within the peptide backbone for conformationally sensitive
carbonyls.  The band at 1653 cm<sup>&minus;1</sup> suggests alpha helix while
those at 1674 and 1639 cm<sup>&minus;1</sup> would
indicate extended chain structures.
>From other reports that 1663 &plusmn; 4, 1670 &plusmn; 2, 1688 &plusmn; 2,
and 1694 &plusmn; 2 cm<sup>&minus;1</sup> correspond to turns in the molecule, it
is possible that the 1674 cm<sup>&minus;1</sup>
band results from contributions of extended chain and turn structures.
>From the IR spectra, a 26% alpha helical
content is predicted, in close agreement with previous CD spectral analysis.
<p>
IR spectral analysis of Ala-10-substituted peptide showed
it to vary &lt;3% from the maximum amplitude (spectral intensities) at
significant bands.  A similar alpha helical content is predicted.  Its
structure is not likely to be different.  On the other hand,
the truncated peptide displayed quite different spectra:
the region between 1640 and 1660 cm<sup>&minus;1</sup> was quite
different.  In brief, the truncated peptide showed &lt;6% helical content.
This is true even when this truncated peptide is monosubstituted (at
Leu-17 and Ala-20) and disubstituted (at Leu-17, Ala-20).  This lack
of alpha helical content in the truncated peptides shows that the
alpha helical structure is in the N-terminal region of the parent peptide.
<p>
500 MHz NMR spectra equilibrated in D<sub>2</sub>O at neutral pH were
analyzed.  The nondegenerate resonances for the three
alanine and three threonine methyl protons the complexity of the alpha-CH
region indicate a random coil peptide structure, compatible with
Fourier-transform IR spectal data.  Resonance assignments of the
parent peptide were made based on chemical-shift
and spin-spin coupling patterns, assisted by analysis of specific residue
substitution peptides.  The method of sequential assignments (see Wï¿½thrich,
<i>NMR of Proteins and Nucleic Acids</i>, New York: Wiley &amp; Sons, 1986)
could not be used since 1- and 2-dimensional nuclear Overhauser experiments
failed to detect inter-residue nuclear Overhauser effects.
It is probable because of their size that peptides have correlation
times on the order of the reciprocal of the Larmor frequency, in
which case the nuclear Overhauser effect approaches zero.
Chou &amp; Fasman analysis indicates that
substitutions with alanine or leucine would tend to have no effect on the
secondary structure of the parent peptide, and the NMR-analyzed solution
structures of the substituted peptides indicate no overt structural changes.
<p>
Under denaturing conditions (100% trifluoroacetic acid and similarly
in DMSO at 120&deg; or 4&nbsp;M LiCl at 90&deg;) resonances in Ala, Thr, and Ile
indicate a random coil structure and nondegeneracy is decreased.
The spectral changes (noted in detail in the article) show that
the parent native peptide is well-structured.  Titration of the
pH shows the effect of chemical shifts on the nonexchangeable protons:
deflections at pH 3-5 are typical of titration of the beta-carboxylate
group of the Asp residue.
<p>
NMR analysis was used to examine the kinetics of proton
exchange, which can be used to ascertain structures potentially involved in
hydrogen bonding.  Half-times for amide exchange rates at pH 4.5 indicate
a number of slow-exchanging protons are present.  The amide protons of
Phe and Ile (probably the Ile at position 11) show a half-time of 20 min,
being the most highly stabilized of the protons, and locating them within
the core of any structure present.  If the alpha helical
structure suggested by the IR data is present in the N-terminal region, it is
probable that Phe-10 and Ile-11 are hydrogen-bonded to alpha carbonyl groups
of Thr-6 and Tyr-7, resp.  Asp-9 also showed a slower than normal half-time
(~5 min), compatible with its hydrogen-bonding to Thr-5. A previous
report indicates that the <sup>3</sup><i>J</i><sub>NH</sub><sub>&alpha;</sub>
and the dihedral angle &phi; can be calibrated and it is
determined that segments with 3 or more spin-spin coupling constants
<sup>3</sup><i>J</i><sub>NH</sub><sub>&alpha;</sub> &lt;6 Hz serve
as an independent criterion for the presence of helical structure.
Spectra were recorded for parent and substituted peptides in a
H<sub>2</sub>O/D<sub>2</sub>O mixture (9:1, v/v) to obtain the
<sup>3</sup><i>J</i><sub>NH</sub><sub>&alpha;</sub> values:
results show all amide protons for Thr-6, Tyr-7, Asp-9, and
Ile-11 to have spin-spin constants &lt;6 Hz.  Ala-8 and Ala-12
have a 2/3 probability of having constants &lt;6 Hz, since two
of the three constants are &lt;6 Hz but cannot be assigned
unambiguously to those two of the three Ala residues present
in the parent peptide.  The Phe-10 amide protons coupling
constants could not be determined because of overlapping spectra.
<p>
X-ray scattering data was collected to give other physical
features such as molecular geometry/shape.  The radius of
gyration (R<sub>g</sub>) was measured as a function of changes
in peptide concentration.  A pair-distance distribution
[<i>P(r)</i>] function is first determined.  With infinite
dilution (zero peptide concentration) values of <i>P(r)</i>
determined by extrapolation of the scattering intensity data.
Plots of <i>I</i><sub>0</sub>/<i>c</i> vs. <i>c</i>, where
<i>c</i> is peptide concentration and <i>I</i><sub>0</sub>
is scattering intensity at zero angle are linear and show no
evidence of aggregation.  The calculation of the <i>M</i><sub>r</sub>
from extrapolation of that plot to <i>c</i>=0, with troponin C
as a standard, is in good agreement with the theoretical.
The <i>P(r)</i> vs length <i>R</i> function is symmetrical about a
maximum where <i>R</i>=11.5 &Aring;.  For a prolate ellipsoid with
dimensions of 25 &Aring; x 17 &Aring;, the distribution of scattering
intensity indicate a peptide which is spherical.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Engelman, D. M., Steitz, T. A., Goldman, A.
 </span>
 <span class="year">
(1989)
 </span>
 <span class="jname">
Annu. Rev. Biochem. Biophys.
 </span>
 <span class="jvol">
15,
 </span>
 <span class="pages">
321-353.
 </span>
<p class="reftitl">
Identifying nonpolar transbilayer helices in amino acid sequences of membrane proteins
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Benore-Parsons, M., Seidah, N. G., Wennogle, L. P.
 </span>
 <span class="year">
(1989)
 </span>
 <span class="jname">
Arch. Biochem. Biophys.
 </span>
 <span class="jvol">
272,
 </span>
 <span class="pages">
274-280.
 </span>
<p class="reftitl">
Substrate phosphorylation can inhibit proteolysis by trypsin-like enzymes
<p class="Para1">
Kemptide was phosphorylated by PK-A and analyzed on reversed-phase
HPLC.  Amino acid analysis indicated phosphoserine.  Fast-atom bombardment
mass spec analysis (using a Finigan TSQ 70) showed a prominent 852 amu
(MH<sup>+</sup>) peak scanned in the positive ion mode.  Sensitivity to
trypsin was assessed at a fixed substrate concentration and adding trypsin.
1.43 &micro;g/ml trypsin was necessary to effect 50% hydrolysis under
our conditions.  In contrast, 48.0 &micro;g/ml trypsin was required
to effect 50% hydrolysis of phosphokemptide (33.6-fold difference).
At a fixed trypsin concentration of 3 &micro;g/ml, 50% of kemptide was
digested in 64 s, while half of the phosphokemptide took 3016 s to digest
(rate shows 47-fold difference).  To eliminate the possibility that
phosphokemptide acts as an inhibitor, both substrates were
combined in the same tube and it was found that kemptide
was rapidly digested while phosphokemptide was not.
Additionally, HPLC-purified peptides confirmed the observed results.
<p>
As kemptide has two Args, amino acid analysis of the
products indicated that cleavage was at Arg-3 (products were LRR and
ASLG).  Analysis of phosphokemptide
products indicated RASLG and LR and one other set of minor peaks:
the latter products represented &lt;5% of
those formed.  Rat plasma kallikrein was
also used and two peptides generated from kemptide were ASLG and LRR.
With phosphokemptide, this enzyme cleaved
between Arg-2 and -3.  The cleavage of
kemptide was 13-fold more rapid than phosphokemptide.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Helander, I. M., Kitunen, V.
 </span>
 <span class="year">
(1989)
 </span>
 <span class="jname">
FEBS Lett.
 </span>
 <span class="jvol">
250,
 </span>
 <span class="pages">
565-569.
 </span>
<p class="reftitl">
Cleavage of the O antigen 4,5,12 of <i>Salmonella typhimurium</i>
by hydrofluoric acid
<p class="Para1">
Lipopolysaccharide (LPS) of the <i>S. typhimurium</i> SH2183
contains glucosamine, phosphate, neutral sugars, and a thiobarbituric
acid-positive material called 3-deoxy-6-<i>manno</i>-octulosonic acid.
It is an O-specific side chain.  When the LPS is treated for 48&nbsp;h&nbsp;at
4&deg; in 48% HF and the reaction material dialyzed, analysis of the
dialysate and the modified LPS (HF-LPS) shows that the HF-LPS was
(i) virtually devoid of phosphate; (ii) it was positive for an
assay for glucosamine, and (iii) it contained only minor amounts
if any of O-specific sugars.  The dialysate contained identifiable
phosphate and abeqose (by its characteristic spectrum).
<!-- ============================================================= -->
<p class="tags">
alpha2 macroglobulin
<p class="ref" id="reddy">
 <span class="authors">
Reddy VY, Pizzo SV, Weiss SJ
 </span>
 <span class="year">
(1989)
 </span>
 <span class="jname">
J. Biol. Chem.
 </span>
 <span class="jvol">
264,
 </span>
 <span class="pages">
13801-13809.
 </span>
<p class="reftitl">
Functional Inactivation and Structural Disruption of Human
&alpha;<sub>2</sub>-Macroglobulin by Neutrophils and Eosinophils
<p class="Para1">
Many anti-proteinases that protect healthy normal tissues are likely inactivated
by the release of oxidants and proteases in the inflammatory process.  These
antiproteinases contain critical Met residues at or near their active sites
which are sensitive to oxidation or have proteinase-sensitive sequences where
cleavage results in loss of function.  It contains a sequence of 30 amino acids
corresponding to a <q>bait region</q> in which the proteases cleave at sites
specific for their substrate.  With cleavage, the thiol estor bond that springs
the trap is broken with a conformation change entrapping the protein.
<p>
10<sup>6</sup> neutrophils are treated with 25 ng PMA to trigger activation of 
neutrophil elastase.  The method involves incubating 60 min at 37&deg;, removing 
cell-free supernatant, and measuring amidase activity with 
CH<sub>3</sub>O-succinyl-Ala-Ala-Pro-Val p-nitroanilide.  &alpha;<sub>2</sub>M 
was present in increasing concentrations.  With no &alpha;<sub>2</sub>M present 
during incubation, elastase activity ws recovered that could be complexed 
completely by 100 &micro;g/ml &alpha;<sub>2</sub>M addition.  With 100 
&micro;g/ml (140 nM) &alpha;<sub>2</sub>M present during triggering, large 
amounts of elastase were uncomplexed, and higher concentrations of 
&alpha;<sub>2</sub>M did not complex all of the released elastase.  Seven 
experiments showed that the percent uncomplexed elastase in presence of 100, 
200, and 400 &micro;g/ml &alpha;<sub>2</sub>M (= % uncomplexed elastase activity 
relative to amt of elastase activity released from cells triggered in absence of 
&alpha;<sub>2</sub>M) was 142 &plusmn; 16%, 121 &plusmn; 23%, and 46% &plusmn; 
1%.  Because elastase bound by functional &alpha;<sub>2</sub>M remains active 
against small MW substrates and would be shielded from inhibition by high MW 
antiproteinases, if released elastase activity is sensitive to 
&alpha;<sub>1</sub>PI, then this means the antiproteolytic activity of 
&alpha;<sub>2</sub>M had signif. decreased.
<p>
Larger counts of neutrophils were incubated with 125 &micro;g/ml (170 nM) 
&alpha;<sub>2</sub>M and pancreatic elastase activity was quantified.  As few as 
10<sup>5</sup> PMA-triggered neutrophiles could inactivate &alpha;<sub>2</sub>M 
by the amidase assay. Opsonized zymosan also triggers neutrophil elastase, and 
its use required 200,000 neutrophils to inactivate 74.5% of 
&alpha;<sub>2</sub>M.  In a 60 min incubation, almost all the anti-proteinase 
activity of &alpha;<sub>2</sub>M was lost.
<p>
Since the antiproteinase activity of &alpha;<sub>2</sub>M against elastase is 
not to stop small substrate hydrolysis, the dysfunctional &alpha;<sub>2</sub>M 
was tested in the presence of [<sup>3</sup>H]elastin and quantified.  Both the 
amidase and elastinolytic activity assays show that &alpha;<sub>2</sub>M 
function was lost 81.8% and 78.5%, respectively.
<p>
&alpha;<sub>2</sub>M that had been inactivated by neutrophil exposure was 
analyzed by native PAGE. 25 &micro;g &alpha;<sub>2</sub>M was mixed with 
10<sup>5</sup> PMA-triggered neutrophils in 0.2 ml for 90 min 37&deg; and an 
aliquot of ~20 &micro;g and further incubated alone or with 2-fold molar excess 
pancreatic elastase (~2 &micro;g) for 30 min 37&deg;. Besides native PAGE, 
SDS-PAGE under reducing and non-reducing conditions was done. 4 lanes are run: 
&alpha;<sub>2</sub>M alone (control), &alpha;<sub>2</sub>M with pancreatic 
elastase, &alpha;<sub>2</sub>M with neutrophils, &alpha;<sub>2</sub>M + 
neutrophils then elastase.  &alpha;<sub>2</sub>M + elastase results in an 
electrophoretically faster form, probably the result of 
&alpha;<sub>2</sub>M-complexed/trapped elastase.  An even faster electrophoretic 
form was seen with neutrophil exposure, with loss of native structure.  SDS-PAGE 
under reducing conditions shows multiple bands, indicating the presence of 
neutrophils have major structural effects on &alpha;<sub>2</sub>M.
<p>
Neutrophils were then triggered in the absence of any &alpha;<sub>2</sub>M. 
Cells were then spun and the cell pellet and cell-free supernatant used with 25 
&micro;g of &alpha;<sub>2</sub>M.  In both cases &alpha;<sub>2</sub>M retained 
all antiproteolytic activity, migrated as native &alpha;<sub>2</sub>M, and in 
the faster form when elastase was added.  Whatever was generated by triggered 
neutrophils was not stable or long-lived.  When &alpha;<sub>2</sub>M was 
incubated with neutrophils but in presence of catalase, it did not lose any of 
its activity. Gel analysis also showed that the electrophoretically faster form 
did not appear as well.  When heat-denatured catalase was used the protective 
effect was lost.  Thus peroxide must play a role in &alpha;<sub>2</sub>M 
structural alteration. Since H<sub>2</sub>O<sub>2</sub> is a precursor in the 
formation of the more oxidzing HOCl by myeloperoxidase in the presence of 
chloride, both the enzyme inhibitor azide and the HOCl scavenger Met were 
incubated.  These agents prevented the destruction of &alpha;<sub>2</sub>M 
although they do not affect generation of peroxide and superoxide.  When 
elastase was added to &alpha;<sub>2</sub>M incubated with neutrophils in 
presence of Met, the electrophoretic fast form had been seen as with 
&alpha;<sub>2</sub>M + elastase alone. The same was true for 
&alpha;<sub>2</sub>M + neutrophils + catalase or azide + elastase. Exogenous 
myeloperoxidase was also added (because PMA is s weak stimulus for peroxidase 
release) and this increased the quantity of &alpha;<sub>2</sub>M inactivated by 
3-fold (from ~20 &micro;g to 60 &micro;g).  All these data point to the role of 
HOCl in &alpha;<sub>2</sub>M inactivation.  Addition of superoxide dismutase 
also partially protected against &alpha;<sub>2</sub>M inactivation:  while it 
produces peroxide and it might indirectly regulate HOCl generation, 
heat-denatured and peroxide-inactivated SOD paradoxically protected 
&alpha;<sub>2</sub>M more effectively than the native enzyme.
<p>
If neutrophils work through HOCl, then these substances should also affect 
&alpha;<sub>2</sub>M in a cell-free system.  10<sup>5</sup> neutrophils 
activated by PMA will generate ~5 nmol HOCl based on the average determined in 
our studies.  Thus &alpha;<sub>2</sub>M was treated with ~5 nmol of either 
peroxide (with myeloperoxidase and chloride present) or to reagent HOCl for a 60 
min 37&deg; condition.  There was a 100% inactivation. Note that neither 
peroxide nor myeloperoxidase alone had an effect.  Reagent HOCl inactivated 
&alpha;<sub>2</sub>M about 89% within 5 min of start of incubation.  Assay of 
functional status of &alpha;<sub>2</sub>M did not matter if the protease is 
neutrophil elastase or trypsin instead of pancreatic elastase.  Presence of 500 
nmol Met in the reaction completely blocked &alpha;<sub>2</sub>M inactivation.
<p>
Thus the chlorinated oxidants modify &alpha;<sub>2</sub>M in a fashion similar 
to the PMA-treated neutrophils.  After denaturation and reduction, monomeric 
&alpha;<sub>2</sub>M band intensity decreased and high MW complexes appear 
similar to the neutrophil system. In contrast to results with the intact cell 
treatments, low MW fragments were not observed until after an exogenous 
proteinase was added.  The low MW fragments generated by intact cell system 
probably was the susceptibility of &alpha;<sub>2</sub>M to attack by unregulated 
neutrophil proteinases.
<p>
Besides HOCl affecting &alpha;<sub>2</sub>M, HOCl itself can generate derivative 
oxidants, such as N-chloramines (RNCl).  Chloramine itself (NH<sub>2</sub>Cl) 
and N-chlorotaurine are major lipophilic and hydrophilic chloramines coming from 
neutrophils. Reactions were set up with HOCl, NH<sub>2</sub>Cl, and 
N-chlorotaurine at 5 nmol and HOCl inactivated 98% of &alpha;<sub>2</sub>M in 60 
min, while NH<sub>2</sub>Cl inactivated 35% and N-chlorotaurine did 8%.  Cells 
were then activated in the presence or absence of 100 or 500 &micro;M 
NH<sub>4</sub>Cl or 5 mM taurine.  Under such conditions &alpha;<sub>2</sub>M 
inactivation by neutrophils was reduced from 83% to 48% and 24% with 100 and 500 
&micro;M ammonium chloride, respectively and to 17% in presence of taurine. The 
preferred oxidant thus appears to be HOCl and not the amine derivatives.
<p>
Studies appear to indicate that in contrast to neutrophiles, triggered 
eosinophils will used bromide (Br<sup>&minus;</sup>) to make HOBr. 200,000 
eosinophils were triggered with PMS and then incubated in Dulbecco's with 25 
&micro;g a2M in the presence or absence of 100 &micro;M bromide.  When no 
bromide is present, HOCl is made in small quantities:  small quantitites of a2M 
inactivated.  With bromide presence (physiological levels) eosinophil peroxidase 
makes HOBr:  a2M inactivation is increased from ~20% to 65%.  Native catalase 
(not heat-treated catalase), azide, and Met all prevented a2M inactivation with 
or without bromide.  The presence of native SOD dismutase <q>dramatically</q> 
increases a2M inactivation, but denatured enzyme has an inhibitory effect.  Thus 
SOD has the ability to enhance HOX acid generation in native state, or depress 
it in the inactive state.
<p>
Finally HOBr was itself added in a cell-free system instead of generated by 
triggered cells.  10 nmol of HOBr activated ~75% of 25 &micro;g of 
antiproteinase (60 min in 200 &micro;l at 37&deg;; reaction stopped with Met). 
Both PAGE and SDS-PAGE show that the tetrameric and monomeric structure are 
altered.  HOBr is less effective than HOCl however.  It does seem however that 
eosinophils preferred use of HOBr to destroy a2M compared to HOCl.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Dahmer, M. K., Ji, L., Perlman, R. L. 
 </span>
 <span class="year">
(1989)
 </span>
 <span class="jname">
J. Neurochem.
 </span>
 <span class="jvol">
53,
 </span>
 <span class="pages">
1036-1042.
 </span>
<p class="reftitl">
Characterization of insulin-like growth factor-I receptors in PC12 
pheochromocytoma cells and bovine adrenal medulla
<p class="Para1">
IGF-I receptors were found in cultured bovine chromaffin cells.  Cells which had 
been incubated with various concentrations of ligands in the presence of trace 
<sup>125</sup>I-IGF-I.  IGF-I competed most strongly with the label 
(IC<sub>50</sub> = 8 nM) with IGF-II and insulin also binding with respectively 
decreasing affinity. Scatchard analysis reveals 111,000 receptors/cell in 
chromaffin cells compared to 11,100 receptors in PC12 cells.  Nonlinear least 
squares analysis described by Yamaoka et al. (<i>J. Pharmacobiodyn.</i> 
<b>4</b>: 879, 1981) revealed no multiple classes of receptors.
<p>
When <sup>125</sup>I-insulin was used as tracer on both PC12 and chromaffin cells, 
unlabeled insulin competed with IGF-I for binding to both cell types, thus 
indicating receptors for each ligand existed.  Insulin receptors were present in 
equal numbers in PC12 and chromaffin cells (about 10,000/cell).  The affinity 
for the ligands overall was much greater in chromaffin than in PC12 cells.
<p>
Affinity labeling of the PC12 cell receptors using labeled IGF revealed by 
SDS-PAGE a labeled band at <i>M</i><sub>r</sub>&gt;200,000 in nonreducing gels, 
while in 2-mercaptoethanol, a labeled band of Mr 125,000-130,000 shows it to be 
comparable to the &beta;-subunit of the IGF-I receptor.  Addition of unlabeled 
IGF inhibited labeling and addition of unlabeled insulin did not.
<p>
Receptors from PC12 and adrenal medulla were solubilized and purified on 
WGA-Sepharose.  Scatchard analysis of competitive binding to these purified 
receptors revealed decreased <i>K</i><sub>D</sub> by 10-fold for chromaffin 
cells and nearly 40-fold for PC12 cells, each near <i>K</i><sub>D</sub>=0.15 nM. 
When medullary receptors were incubated with labeled ATP, several labeled bands 
are seen.  In presence of 10 nM IGF-I, phosphorylation of a band with high 
<i>M</i><sub>r</sub> in nonreducing gels but <i>M</i><sub>r</sub> of 97,000 in 
reducing gels was enhanced.  Insulin had little effect. Autophosphorylation 
studies in PC12 cells using both IGF-I and insulin treatments revealed the same 
effect:  presence of a labeled 97 kDa band.
<p>
Tyrosine kinase activity of the medullary and PC12 receptors was examined using 
the synthetic substrate poly(Glu:Tyr) 4:1.  IGF stimulation of the medullary 
receptors was concentration dependent, with half-maximal activation at 0.2 nM 
IGF-I.  Insulin is much less potent.  Activation toward the substrate was 
comparable for both IGF-I and insulin in PC12 cells.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Oddie, K. M., Litz, J. S., Balserak, J. C., Payne, D. M., Creutz, C. E., Parsons, S. J.
 </span>
 <span class="year">
(1989)
 </span>
 <span class="jname">
J. Neurosci. Res.
 </span>
 <span class="jvol">
24,
 </span>
 <span class="pages">
38-48.
 </span>
<p class="reftitl">
Modulation of pp60<sup><i>c-src</i></sup> tyrosine kinase activity during 
secretion in stimulated bovine adrenal chromaffin cells
<p class="Para1">
It was previously shown that pp60<sup><i>c-src</i></sup> activity was 
partitioned in bovine chromaffin cells to the plasma membrane and to the 
membranes of the cytoplasmic secretory vesicles.  This technique employed 
subcellular fractionation techniques and pp60<sup><i>c-src</i></sup>-specific in 
vitro immune complex assay.  Later it was found that the amount of 
pp60<sup><i>c-src</i></sup> in chromaffin cells is approx. equal the amount of 
<i>v-src</i> in Rous sarcoma virus-transformed chick fibroblasts, i.e., 
0.01-0.02% of total cell protein.  At this level, visualization is possible with 
immunofluorescence.  Cells were cultured on glass cover slips and immunostained 
with GD11, a pp60<sup><i>c-src</i></sup>-specific MAb.  Results showed plasma 
membrane fluorescence with abundant cytoplasmic staining, but no nuclear 
staining was observed.  EB8, a second pp60<sup><i>src</i></sup>-specific 
antibody, showed an identical pattern of fluorescence, while a control MAb of 
the same subclass but to human pancreatic amylase was without effect.  This 
shows that pp60src is localized not only to the plasma membrane but to 
chromaffin granules in the cytoplasm as well.
<p>
Chromaffin cells were then treated with 20 &micro;M nicotine, 300 &micro;M 
nicotine, or mock solution to investigate whether pp60 activity is altered 
during exocytosis. After 5 min incubation, detergent extracts are 
immunoprecipitated with GD11 MAb and pp60 activity assayed using the immune 
complex assay.  Autoradiographs of SDS gels of immunoprecipitates from either 
nicotine- or carbachol-stimulated cells indicate a decrease in 
autophosphorylating ability of pp60 compared to untreated cells.  Cells were 
then treated or not treated with 20 &micro;M nicotine for 3 or 10 min and 
immunoprecipitates of extracts obtained therefrom were divided with one portion 
being subjected to in vitro immune complex autophosphorylation assay and the 
other portion analyzed by Western with Q10 rabbit anti-peptide serum raised 
against the COOH terminal 12 amino acids of avian pp60<sup><i>c-src</i></sup>, 
also conserved in mouse and human.  Results show the greatest decrease in 
autophosphorylating ability for the longest time point measured (10 min). 
Although immune complexes at 10 min treatment contained less <i>c-src</i> protein 
(50&plusmn;24%) than that of mock-stimulated, autophosphorylating activity in 10 
min treated immune complexes was 22&plusmn;10% of mock-stimulated, and the 
transphosphorylating activity was 38&plusmn;10% of the control.  Specific 
activity calculations indicate that auto- and transphoshorylating ability of 
treated was 44% and 86% of control cells, resp.
<p>
<sup>3</sup>H-Norepinephrine release and pp60<sup><i>c-src</i></sup> 
inactivation were measured in cultures treated with nicotine.  The results show 
that both occur very rapidly within 5 minutes, and this may suggest a role of 
pp60<sup><i>c-src</i></sup> in exocytosis.  Different preparations of chromaffin 
cells indicate that the time course and magnitude of pp60 activity varied with 
the preparations.  In most cases, activity in treated cells followed the 
secretory activity measured by <sup>3</sup>H-NE.  Long-term studies measuring 
pp60<sup><i>c-src</i></sup> activity over a treatment period of 6 h indicate 
that activity returns to normal 2-6 hr after secretagogue addition.
<!-- ============================================================= -->
<p class="tags">
VEGF
<p class="ref" id="Leung-VEGF">
 <span class="authors">
Leung DW, Cachianes G, Kuang W-J, Goeddel DV, Ferrara N
 </span>
 <span class="year">
(1989)
 </span>
 <span class="jname">
Science
 </span>
 <span class="jvol">
246,
 </span>
 <span class="pages">
1306-1309
 </span>
<p class="reftitl">
Vascular Endothelial Growth Factor is a Secreted Angiogenic Mitogen
<p class="Para1">
Elucidation of factors involved in angiogenesis is critical for understanding
organ devleopment, wound healing, tissue regeneration, and abnormalities such
as neoplastic proliferations, retinopathies and rheumatoid arthritis.  EGF,
TGF-&alpha; and -&beta;, TNF, angiogenin, and prostaglandin E<sub>2</sub> can
induce angiogenesis in vivo, but they are not mitogens of endothelial cells
directly.  bFGF and aFGF can induce endothelial cell proliferation in vitro
and angiogenesis in vivo.  The problem is that they are not secreted and lack
a signal peptide.  As angiogenesis requires a diffusible factor, it becomes
necessary to find a secreted factor.
<p>
A heparin-binding vascular endothelial growth factor (VEGF) was purified from
conditioned medium of bovine pituitary folliculostellate cells.  It is a dimer
with mass of 45,000 composed of two subunits with mass 23,000.  It is a potent
mitogen of cells in small and large vessels.  It has no effect on BHK-21
fibroblasts, lens epithelial cells, corneal endothelial cells, keratinocytes,
or adrenal cortical cells.  A cDNA library was prepared from the folliculostellate
cells and probed with 59-base oligo based on partial amino terminal sequence
(positions 2-21).  20 clones were identified in 1.5 million clone library.
Two of these were sequenced for protein coding and noncoding regions, with
identical result seqeunces.  The clone contains open reading frame of 190 amino
acids.  Comparison of deduced amino acid and N-terminal sequence shows 26
amino acids beginning with initiator Met codon.  This is preceded by 
CCGAAACC which agrees with initiation consensus (GCC)GCCA/GCCATG for vertebrates.
The 26 residues contain hydrophobic core of 16 amino acids flanked by polar
or charged residues characteristic of secretory signal sequence.  A Ser-Gln-Ala
at positions &minus;3 to &minus;1 represent a known signal cleavage site.
The mature sequence would have 164 amino acisds with computed mass of 19162.
An Asn<sup>74</sup> glycosylation site indicates it might be a glycoprotein.
Basic amino acids at position 110, 123, and 163 could indicate heparin binding,
analogous to bFGF and aFGF.  A 421 nt 3&prime;-UTR with poly(A) signal sequence 
AATAAA is 20 bases upstream of poly(A) tail.
<p>
Poly(A) RNA was prepared on Northern blot and a single band at 3.7 kb was detected.
The 5&prime; UTR will be 2.5 kb as the ATG codon and poly(A) site is &lt; 1 kb.
The 545 nt long 5&prime; noncoding sequence is &gt; 75% GC content.
<p>
Using the bovine cDNA as a probe,  cDNA libraries of phorbol ester-activated
HL60 promyelocytic leukemia cells were examined.  Sequencing of cDNA sequence
reveals human and bovine VEGF have 95% identity in sequence.  The 165 amino acid
form has an inserted Gly at position 6 compared to the 164 aa bovine form.
Additional clones were found with a deletion of 44 amino acids between positions
116-159, and insertion of 24 aa (KSVRGKGKGQKRKRKKSRYKSWSV) at position 116.
The mature amino acids would be 121 and 189 aa long.  The Asn<sup>115</sup> is
replaced by Lys.  Alternative RNA splicing explains these different mRNAs.
<p>
Significant homology between VEGF and PDFG A and B chaines is seen.  8 Cys residues
are conserved within the 50 C-terminal residues.  All clones were put into an
expression vector under CMV promoter control and human 293 cells transfected.
Secreted product was able to promote proliferation of endothelial cells, as
plots of 100,000s cells / well vs. fraction conditioned medium with bovine
and human VEGF show.
<p>

<b>Methods</b>.  Bovine pituitary follicular cell mRNA was prepared using 
guanidine thiocyanate-LiCl (Cathala et al <i>DNA</i> <b>2</b>: 329, 1983) then 
by oligo(dT) cellulose (Aviv &amp; Leder <i>J Mol Biol</i> <b>134</b>; 743, 
1972).  Amersham kit used with oligo(dT) and random primers to make cDNA. 
Hemikinased <i>Eco</i>RI adaptors were ligated to cDNA.  EcoRI-digested cDNA 
were purified by spin dialysis with Sephacryl S500HR and cloned to &lambda;gt10 
to make libraries.  5&prime;-<sup>32</sup>P-labeled 59 bp probe (for 20 amino 
acids)  with hybridization done at low stringency.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Keshmirian, J., Bray, G., Carbonetto, S.
 </span>
 <span class="year">
(1989)
 </span>
 <span class="jname">
J. Neurocyt.
 </span>
 <span class="jvol">
18,
 </span>
 <span class="pages">
491-504.
 </span>
<p class="reftitl">
The extracellular matrix modulates the response of PC12 cells to nerve growth 
factor:  cell aggregation versus neurite outgrowth on 3-dimensional laminin 
substrata
<p class="Para1">
Previous reports show that certain substrata coated with laminin, collagens I 
and IV, or polyLys in the presence of NGF cause PC12 cells to spread and extend 
neurites.  With reconstituted basement membrane (RBM), we show essentially the 
same effects.  In a 3-D substrata of RBM gels, cells are seen to attach but not 
spread, instead forming cell aggregates after 6-8 h.  At high cell densities 
these form networks interconnected by extended cell processes.  RBM gels 
overlayed on cells previously attached to 2-D substrata still behave as if on 
2-D substrata, failing to aggregate.  When cultured within the gel however, they 
do aggregate.
<p>
After 2-4 d the networks of small aggregates coalesce inot larger aggregates, 
extending a halo of neurites similar to dorsal root ganglia in media 
supplemented with NGF.  Coomassie staining shows that the gel has actually been 
disrupted (it is not uniformly blue); the aggregates have pulled the gel into 
pits on their underside.
<p>
PC12 cells grown on collagen either in 3-D on 2-D substrata will attach, spread, 
and extend short neurites with little to no aggregation.  On reconstituted 
laminin gels, however, they aggregate with the same pattern as on RBM gels. 
This suggests that the cells respond to both composition and configuration (2-D 
vs. 3-D) of the support substrata.
<p>
Video-time lapse shows a "blebbing" phenomenon typically seen in active cells. 
Cell bodies are then pulled forward.  The blebbing is enhanced with NGF included 
in the medium.  If they contact other cells, they show a preference for cells 
over the substrata, forming small aggregates.  As the aggregate grows, some 
cells extend processes along the substrata and bridge to other aggreagates.  On 
collagen gels, the cells bleb extensively and similar to RBM gels, but 
apparently attach to collagen better, becoming less motile.  On 2-D substrata, 
the cells aggregate not as well with any medium, and this is presumably due to a 
decreased strength of the fibrillar matrix in 3-D gels in general.  On collagen 
the fibrillar matrix is even stronger (breakdown is difficult) and makes 
aggregation less likely.
<p>
[There is more on this interesting paper.]
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Norbury, C. J., Nurse, P. 
 </span>
 <span class="year">
(1989)
 </span>
 <span class="jname">
Biochim.
Biophys. Acta
 </span>
 <span class="jvol">
989,
 </span>
 <span class="pages">
85-89.
 </span>

<p class="reftitl">
Control of the higher eukaryotic cell cycle by p34 homologues
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
North, G. 
 </span>
 <span class="year">
(1989)
 </span>
 <span class="jname">
Nature
 </span>
 <span class="jvol">
339,
 </span>
 <span class="pages">
97-98.
 </span>

<p class="reftitl">
Regulating the cell cycle
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Cisek, L. J., Corden, J. L. 
 </span>
 <span class="year">
(1989)
 </span>
 <span class="jname">
Nature
 </span>
 <span class="jvol">
339,
 </span>
 <span class="pages">
679-684.
 </span>

<p class="reftitl">
Phosphorylation of RNA polymerase by the murine homologue
of the cell-cycle control protein cdc2
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Labbe, J. C., Picard, A., Peaucellier, G., Cavadore, J.
C., Nurse, P., Doree, M. 
 </span>
 <span class="year">
(1989)
 </span>
 <span class="jname">
Cell
 </span>
 <span class="jvol">
57,
 </span>
 <span class="pages">
253-263.
 </span>

<p class="reftitl">
Purification of MPF from starfish: identification as the H1 histone kinase p34 
and a possible mechanism for its periodic activation
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Russell, P., Moreno, S., Reed, S. I. 
 </span>
 <span class="year">
(1989)
 </span>
 <span class="jname">
Cell
 </span>
 <span class="jvol">
57,
 </span>
 <span class="pages">
295-303.
 </span>

<p class="reftitl">
Conservation of mitotic controls in fission and budding
yeasts
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Boulet, A., Simon, M., Faye, G., Bauer, G. A., Burgers,
P. M. J. 
 </span>
 <span class="year">
(1989)
 </span>
 <span class="jname">
EMBO J.
 </span>
 <span class="jvol">
8,
 </span>
 <span class="pages">
1849-1854.
 </span>

<p class="reftitl">
Structure and function of the <i>Saccharromyces cerevisiae CDC2</i> gene encoding the large subunit
of DNA polymerase III
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Cyert, M. S., Thorner, J. 
 </span>
 <span class="year">
(1989)
 </span>
 <span class="jname">
Cell
 </span>
 <span class="jvol">
57,
 </span>
 <span class="pages">
891-893.
 </span>

<p class="reftitl">
Putting it on and taking it off: phosphoprotein phosphatase involvement
in cell cycle regulation
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
McVey, D., Brizuela, L., Mohr, I., Marshak, D. R.,
Gluzman, Y., Beach, D. 
 </span>
 <span class="year">
(1989)
 </span>
 <span class="jname">
Nature
 </span>
 <span class="jvol">
341,
 </span>
 <span class="pages">
503-507.
 </span>

<p class="reftitl">
Phosphorylation of large tumor antigen by <i>cdc2</i> stimulates SV40 DNA replication
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Erikson, E., Maller, J. L. 
 </span>
 <span class="year">
(1989)
 </span>
 <span class="jname">
J.
Biol. Chem.
 </span>
 <span class="jvol">
264,
 </span>
 <span class="pages">
19577-19582.
 </span>

<p class="reftitl">
Biochemical characterization of the p34 protein kinase component of purified
maturation-promoting factor for <i>J. Biol. Chem.</i>
<b>264,</b> 13697-13700.
<p class="reftitl">
Regulation of the product of a possible human cell cycle
control gene CDC2Hs in B-cells by &alpha;-interferon
and phorbol ester
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Warren, G. 
 </span>
 <span class="year">
(1989)
 </span>
 <span class="jname">
Nature
 </span>
 <span class="jvol">
342,
 </span>
 <span class="pages">
857-858.
 </span>

<p class="reftitl">
Mitosis and membranes
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Tuomikoski, T., Felix, M.-A., Doree, M., Gruenberg, J.

 </span>
 <span class="year">
(1989)
 </span>
 <span class="jname">
Nature
 </span>
 <span class="jvol">
342,
 </span>
 <span class="pages">
942-945.
 </span>

<p class="reftitl">
Inhibition of endocytic vesicle fusion in vitro by the
cell-cycle control protein kinase cdc2
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Shenoy, S., Choi, J.-K., Bagrodia, S., Copeland, T. D.,
Maller, J. L., Shalloway, D. 
 </span>
 <span class="year">
(1989)
 </span>
 <span class="jname">
Cell
 </span>
 <span class="jvol">
57,
 </span>
 <span class="pages">
763-774.
 </span>

<p class="reftitl">
Purified maturation promoting factor phosphorylates
pp60 at the sites phosphorylated during
fibroblast mitosis
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Morgan, D. O., Kaplan, J. M., Bishop, J. M., Varmus, H.
E. 
 </span>
 <span class="year">
(1989)
 </span>
 <span class="jname">
Cell
 </span>
 <span class="jvol">
57,
 </span>
 <span class="pages">
775-786.
 </span>

<p class="reftitl">
Mitosis-specific phosphorylation of p60 by p34<i class=
'c7'><sup>cdc2</sup></i>-associated protein kinase
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Pardee, A. 
 </span>
 <span class="year">
(1989)
 </span>
 <span class="jname">
Science
 </span>
 <span class="jvol">
246,
 </span>
 <span class="pages">
603-608.
 </span>

<p class="reftitl">
G1 events and regulation of cell proliferation
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
McIntosh, J. R., Koonce, M. P. 
 </span>
 <span class="year">
(1989)
 </span>
 <span class="jname">
Science
 </span>
 <span class="jvol">
246,
 </span>
 <span class="pages">
622-628.
 </span>

<p class="reftitl">
Mitosis
<!-- ============================================================= -->
<p class="tags">
<p class="ref">
 <span class="authors">
Hartwell, L., Weinert, T. A. 
 </span>
 <span class="year">
(1989)
 </span>
 <span class="jname">
Science
 </span>
 <span class="jvol">
246,
 </span>
 <span class="pages">
629-634.
 </span>

<p class="reftitl">
Checkpoints:  controls that
ensure the order of cell cycle events
<!-- ============================================================= -->
<p class="tags">
<p class="ref">
 <span class="authors">
Laskey, R. A., Fairman, M. P., Blow, J. J. 
 </span>
 <span class="year">
(1989)
 </span>
 <span class="jname">
Science
 </span>
 <span class="jvol">
246,
 </span>
 <span class="pages">
609-613.
 </span>

<p class="reftitl">
S phase of the cell cycle
<!-- ============================================================= -->
<p class="tags">
<p class="ref">
 <span class="authors">
Murray, A. W., Kirschner, M. W. 
 </span>
 <span class="year">
(1989)
 </span>
 <span class="jname">
Science
 </span>
 <span class="jvol">
246,
 </span>
 <span class="pages">
614-621.
 </span>

<p class="reftitl">
Dominoes and clocks:  the
union of two views of the cell cycle
<!-- ============================================================= -->
<p class="tags">
<p class="ref">
 <span class="authors">
O'Farrell, P. H., Edgar, B. A., Lakich, D., Lehner, C.
F. 
 </span>
 <span class="year">
(1989)
 </span>
 <span class="jname">
Science
 </span>
 <span class="jvol">
246,
 </span>
 <span class="pages">
635-640.
 </span>

<p class="reftitl">
Directing cell division during development
<!-- ============================================================= -->
<p class="tags">
<p class="ref">
 <span class="authors">
Moreno, S., Hayles, J., Nurse, P. 
 </span>
 <span class="year">
(1989)
 </span>
 <span class="jname">
Cell
 </span>
 <span class="jvol">
58,
 </span>
 <span class="pages">
361-372.
 </span>

<p class="reftitl">
Regulation of p34
protein kinase during mitosis
<p class="Para1">
A better assay is developed for the <i>cdc2</i> kinase since previous assays used relatively insensitive
immunoprecipitates.  The assay uses H1
histone as a substrate with the important note of eliminating cAMP and
using EGTA to inhibit cAMP and Ca<sup>
2+</sup>-dependent kinases. 
Extracts from wild-type fission yeast show high activity at 25&deg;, 37&deg;, or
40&deg;, while <i>cdc2</i> temperature-sensitive strains show low
activity at 37&deg; and barely detectable activity at 40&deg; using this assay.
<p>
p13<sup>suc1</sup> will rescue <i>cdc2</i> temperature-sensitive mutants at the 
restrictive temperature of 36&deg; if enough copies are used. So when the gene 
product is included in extracts of wild-type and <i>cdc2</i>-33 cells to a 
concentration of 0.25 &micro;M, it is detectable in both while at 1.0 &micro;M (50-
fold higher than physiological), it is restored in the mutant extracts.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Booher, R. N., Alfa, C. E., Hyams, J. S., Beach, D. H.
 </span>
 <span class="year">
(1989)
 </span>
 <span class="jname">
Cell
 </span>
 <span class="jvol">
58,
 </span>
 <span class="pages">
485-497.
 </span>
<p class="reftitl">
The fission yeast cdc2/cdc13/suc1 protein kinase: regulation of catalytic
activity and nuclear localization
<!-- ============================================================= -->
<p class="tags">
<p class="ref">
 <span class="authors">
Zhao, J.-Y., Kuang, J., Adlakha, R. C., Rao, P. N.
 </span>
 <span class="year">
(1989)
 </span>
 <span class="jname">
FEBS Lett.
 </span>
 <span class="jvol">
249,
 </span>
 <span class="pages">
389-395.
 </span>
<p class="reftitl">
Threonine phosphorylation is associated with mitosis in HeLa cells
<!-- ============================================================= -->
<p class="tags">
<p class="ref">
 <span class="authors">
Giordano, A., Whyte, P., Harlow, E., Franza, Jr., B. R.,
Beach, D., Draetta, G. 
 </span>
 <span class="year">
(1989)
 </span>
 <span class="jname">
Cell
 </span>
 <span class="jvol">
58,
 </span>
 <span class="pages">
981-990.
 </span>
<p class="reftitl">
A 60 kd cdc2-associated polypeptide complexes with the
E1A proteins in adenovirus-infected cells
<p class="author">Brooks, R., Fantes, P., Hunt, T., Wheatley, D., eds.,
The Cell Cycle, Proceedings of the British Society for Cell Biology, a
Journal of Cell Science symposium at St. Andrews in April 1989; a
supplement to the Journal of Cell Science (Suppl. 12).
<p class="Para1">
Contains lots of material cyclins, cell biology studies looking at cycling.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Branda, R. F., Moore, A. L., McCormack, J. J. 
 </span>
 <span class="year">
(1989)
 </span>
 <span class="jname">
Biochem. Pharmacol.
 </span>
 <span class="jvol">
38,
 </span>
 <span class="pages">
3521-3526.
 </span>
<p class="reftitl">
Immunosuppressive properties of chloroquinoxaline sulfonamide
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Matherly, L. H., Schuetz, J. D., Westin, E., Goldman, I. D. 
 </span>
 <span class="year">
(1989)
 </span>
 <span class="jname">
Anal. Biochem.
 </span>
 <span class="jvol">
182,
 </span>
 <span class="pages">
338-345.
 </span>
<p class="reftitl">
A method for the synchronization of cultured cells with aphidicolin: 
application to the large-scale synchronization of L1210 cells and
the study of the cell cycle regulation of thymidylate synthase and
dihydrofolate reductase
<!-- ============================================================= -->
<p class="tags">
<p class="ref">
 <span class="authors">
Elliot, G. T., Nagle, W. A., Kelly, K. F., McCollough, D., Bona, R. L., 
Burns, E. R. 
 </span>
 <span class="year">
(1989)
 </span>
 <span class="jname">
J. Med. Cheml.
 </span>
 <span class="jvol">
32,
 </span>
 <span class="pages">
1039-1043.
 </span>
<p class="reftitl">
In vitro antiproliferative activity of 4-substituted 
2-(2-hydroxyphenyl)thiazolines on murine leukemia cells
<!-- ============================================================= -->
<p class="tags">
<p class="ref">
 <span class="authors">
Smith, P. J., Watson, J. V., Shall, S. 
 </span>
 <span class="year">
(1989)
 </span>
 <span class="jname">
Exp. Cell Res.
 </span>
 <span class="jvol">
185,
 </span>
 <span class="pages">
353-362.
 </span>
<p class="reftitl">
Effects of 3-aminobenzamide on cellular ribosomal RNA content and
cell cycle progression in inhibitor resistant and sensitive L1210 cells
<!-- ============================================================= -->
<p class="tags">
<p class="ref">
 <span class="authors">
Messina, J. P., Lawrence, D. A. 
 </span>
 <span class="year">
(1989)
 </span>
 <span class="jname">
J. Immunol.
 </span>
 <span class="jvol">
143,
 </span>
 <span class="pages">
1974-1981.
 </span>
<p class="reftitl">
Cell cycle progression of gluathione-depleted human peripheral blood
mononuclear cells is inhibited at S phase
<!-- ============================================================= -->
<p class="tags">
<p class="ref">
 <span class="authors">
Conradt, P., Dittmar, K. E. J., Schleiphacke, H., Trowitzsch-Kienast, W. 
 </span>
 <span class="year">
(1989)
 </span>
 <span class="jname">
J. Antibiotics
 </span>
 <span class="jvol">
42,
 </span>
 <span class="pages">
1158-1162.
 </span>

<p class="reftitl">
Myxothiazol:  a reversible blocker of the cell cycle
<!-- ============================================================= -->
<p class="tags">
<p class="ref">
 <span class="authors">
Castellot, J. J., Jr., Pukac, L. A., Caleb, B. J.,
Wright, T. C., Jr., Karnovsky, M. J. 
 </span>
 <span class="year">
(1989)
 </span>
 <span class="jname">
J. Cell Biol.
 </span>
 <span class="jvol">
109,
 </span>
 <span class="pages">
3147-3155.
 </span>

<p class="reftitl">
Heparin selectively inhibits a protein kinase C-dependent
mechanism of cell cycle progression in calf aortic smooth muscle cells
<!-- ============================================================= -->
<p class="tags">
<p class="ref">
 <span class="authors">
Hadwiger, J. A., Wittenberg, C., Richardson, H. E., de
Barros Lopes, M., Reed, S. I. 
 </span>
 <span class="year">
(1989)
 </span>
 <span class="jname">
Proc. Natl. Acad. Sci.
USA
 </span>
 <span class="jvol">
86,
 </span>
 <span class="pages">
6255-6259.
 </span>
<p class="reftitl">
A family of cyclin homologs that control the G<sub>1</sub> phase in yeast
<!-- ============================================================= -->
<p class="tags">
<p class="ref">
 <span class="authors">
Murray, A. W., Kirschner, M. W. 
 </span>
 <span class="year">
(1989)
 </span>
 <span class="jname">
Nature
 </span>
 <span class="jvol">
339,
 </span>
 <span class="pages">
275-280.
 </span>
<p class="reftitl">
Cyclin synthesis drives the early embryonic cell cycle
<!-- ============================================================= -->
<p class="tags">
<p class="ref">
 <span class="authors">
Murray, A. W., Solomon, M. J., Kirschner, M. W. 
 </span>
 <span class="year">
(1989)
 </span>
 <span class="jname">
Nature
 </span>
 <span class="jvol">
339,
 </span>
 <span class="pages">
280-286.
 </span>
<p class="reftitl">
The role of cyclin synthesis and degradation in the control of maturation
promoting factor activity
<!-- ============================================================= -->
<p class="tags">
<p class="ref">
 <span class="authors">
Minshull, J., Blow, J. J., Hunt, T. 
 </span>
 <span class="year">
(1989)
 </span>
 <span class="jname">
Cell
 </span>
 <span class="jvol">
56,
 </span>
 <span class="pages">
947-956.
 </span>

<p class="reftitl">
Translation of cyclin mRNA is necessary for extracts of
activated Xenopus eggs to enter mitosis
<!-- ============================================================= -->
<p class="tags">
<p class="ref">
 <span class="authors">
Lehner, C. F., O'Farrell, P. H. 
 </span>
 <span class="year">
(1989)
 </span>
 <span class="jname">
Cell
 </span>
 <span class="jvol">
56,
 </span>
 <span class="pages">
957-968.
 </span>

<p class="reftitl">
Expression and function of Drosophila cyclin A during
embryonic cell cycle progression
<!-- ============================================================= -->
<p class="tags">
<p class="ref">
 <span class="authors">
Westendorf, J. M., Swenson, K. I., Ruderman, J. V.
 </span>
 <span class="year">
(1989)
 </span>
 <span class="jname">
J. Cell Biol.
 </span>
 <span class="jvol">
108,
 </span>
 <span class="pages">
1431-1444.
 </span>
<p class="reftitl">
The role of cyclin B in meiosis I
<!-- ============================================================= -->
<p class="tags">
<p class="ref">
 <span class="authors">
Luca, F. C., Ruderman, J. V. 
 </span>
 <span class="year">
(1989)
 </span>
 <span class="jname">
J.
Cell Biol.
 </span>
 <span class="jvol">
109,
 </span>
 <span class="pages">
1895-1909.
 </span>
<p class="reftitl">
Control of programmed cyclin destruction in a cell-free system
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Montminy, M. R., Gonzalez, G. A., Yamamoto, K. K.
 </span>
 <span class="year">
(1990)
 </span>
 <span class="jname">
Trends Neurosci.
 </span>
 <span class="jvol">
13,
 </span>
 <span class="pages">
184-188.
 </span>
<p class="reftitl">
Regulation of cAMP-inducible genes by CREB
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Rudd, C. E.
 </span>
 <span class="year">
(1990)
 </span>
 <span class="jname">
Immunol. Today
 </span>
 <span class="jvol">
11,
 </span>
 <span class="pages">
400-406.
 </span>
<p class="reftitl">
CD4, CD8 and the TCR-CD3 complex:<br>
a novel class of protein-tyrosine kinase receptor
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Ichinose, M., Endo, S., Critz, S. D., Shenolikar, S., Byrne, J. H.
 </span>
 <span class="year">
(1990)
 </span>
 <span class="jname">
Brain Res.
 </span>
 <span class="jvol">
533,
 </span>
 <span class="pages">
137-140.
 </span>
<p class="reftitl">
Microcystin-LR, a potent protein phosphatase inhibitor, prolongs
the serotonin- and cAMP-induced currents in sensory neurons of
<i>Aplysia californica</i>
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Haystead, T. A., Weiel, J. E., Litchfield, D. W., Tsukitani, Y.,
Fischer, E. H., Krebs, E. G.
 </span>
 <span class="year">
(1990)
 </span>
 <span class="jname">
J. Biol. Chem.
 </span>
 <span class="jvol">
265,
 </span>
 <span class="pages">
16571-16580.
 </span>
<p class="reftitl">
Okadaic acid mimics the action of insulin in stimulating protein
kinase activity in isolated adipocytes:  the role of protein
phosphatase 2A in attenuation of the signal
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Jonat, C., Rahmsdorf, H. J., Park, K.-K., Cato, A. C. B., Gebel, S.,
Ponta, H., Herrlich, P.
 </span>
 <span class="year">
(1990)
 </span>
 <span class="jname">
Cell
 </span>
 <span class="jvol">
62,
 </span>
 <span class="pages">
1189-1204.
 </span>
<p class="reftitl">
Antitumor promotion and antiinflammation: down-modulation of AP-1
(fos/jun) activity by glucocorticoid hormone
<p class="Para1">
Cultured HeLa tk<sup>-</sup> cells were grown to semi-confluence
then treated with 0.1 &micro;M dexamethasone or 0.1 &micro;g/ml TPA or
both.  After 12 hr, <sup>35</sup>S-Met was added to the medium for 2.5 h
and cell-free supernatants (secreted proteins) collected, TCA-ppt'd,
run on SDS gels then the gels autorad.  The phorbol ester induced
4 new proteins of apparent MW 55, 52, 42, and 30 kDa.  The 52 kDa
band was identified as collagenase (the band was excised and
one-dimensional peptide mapping with V8 protease showed identical
pattern with in vitro translation product of collagenase cDNA;
also immunoblotting with antiserum to highly pure collagenase
demonstrated the same thing).  Untreated controls and dexamethasone-treated
cells produced no 52 kDa collagenase, but two other proteins of 250 and
46 kDa were enhanced.  DM&nbsp;+&nbsp;TPA reduced the expression of
the 55, 52, and 42 kDa protein but the 30 kDa protein increase was
unaltered and the enhancement of the 250 kDa protein seen in DM alone was
enhanced as well.
<p>
Cells were treated then poly(A)<sup>+</sup> RNA was isolated and 5 &micro;g
used to determine specific collagenase transcript by RNA protection:
end-labeled 656 bp DNA fragment (KpnI-XbaI) covering the collagenase
transcription start site was incubated with the RNA, then digested
with S1 nuclease.  Correctly protected fragments are 139-144 bp long.
In control cells, a low level of transcription is barely visible,
but in TPA-treated for 6 h, 50-fold induction is observed.  With
DM added, both the basal and TPA-induced levels are reduced to
virtually no RNA detectable.
<p>
Transfection experiments with chimeric constructs of collagenase
promoter-chloramphenicol acetyltransferase (CAT) reporter gene were
done and treatment of the transfectants with DM performed to analyzed
effect on specific transcription induction.  So far all collagenase
gene expression control has been found in the immediate 5' flanking
DNA of the gene.  First the different lengths of DNA corresponding
to the collagenase promoter were tested:  The -1200 to +63 and -517 to
+63 constructs gave the highest relative activity, indicating the minimal
sequence necessary delimiting the promoter.
Measurement of CAT activity in the basal and TPA-treated was high, but
DM addition to either system severely reduced CAT activity.
One construct used in the study was &#8209;517/+63
mTRE, in which the major phorbol ester-responsive enhancer element (has been
identified) is inserted into the collagenase promoter and is localized to
position -72 and -65, which reduces basal CAT expression to 2-fold the vector
background level.  Phorbol ester induction is below detection in
the experiment.  And since there is no further reduction of CAT
activity with DM addition, the basal and induced transcription
appear to be mediated by the same factor, AP-1, and DM acts through TRE.
Deletion of other sequences other than AP-1 binding site did not prevent
inhibition by DM.  The most extreme 5' deletion mutant, -73/+63, which
still retains the AP-1 binding site is still repressible by DM.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Otvos, Jr., L., Tangoren, I. A., Krzysztof, W., Hollosi, M., Lee, V. M.-Y.
 </span>
 <span class="year">
(1990)
 </span>
 <span class="jname">
J. Chromatog.
 </span>
 <span class="jvol">
512,
 </span>
 <span class="pages">
265-272.
 </span>
<p class="reftitl">
Reversed-phase high-performance liquid chromatographic separation
of synthetic phosphopeptide isomers
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Martin, B. L., Wu, D., Jakes, S., Graves, D. J.
 </span>
 <span class="year">
(1990)
 </span>
 <span class="jname">
J. Biol. Chem.
 </span>
 <span class="jvol">
265,
 </span>
 <span class="pages">
7108-7111.
 </span>
<p class="reftitl">
Chemical influences on the specificity of tyrosine phosphorylation
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Rubenstein JLR, Brice AEJ, Ciaranello RD, Denney D, Porteus MH,  Usdin TB
 </span>
 <span class="year">
(1990)
 </span>
 <span class="jname">
Nuc. Acids Res.
 </span>
 <span class="jvol">
62,
 </span>
<span class="pages">
 4833-4842.
</span>
<p class="reftitl">
Subtractive hybridization system using single-stranded phagemids with
directional inserts
<p class="Para1">
Subtractive hybridziation can used to isolate coding sequences of differentially
expressed genes such as those in development or disease.  Tens of micrograms
of poly(A)<sup>+</sup> RNA is the <i>driver</i> which is present at least 10-fold
more than cDNA, which is the target, prepared from another source.  After
forming RNA-DNA hybrids, ss cDNA is collected by hydroxylapatide
chromatography.  The remaining is either cloned or used as probe in differential
screening.  A drawback is the requirement for large amounts of driver mRNA,
difficult to do if the source is an embryo or other difficult to obtain tissue.
Even after subtraction, on a few nanograms of target sequences are left and so
clones in large numbers are difficult.  This method produces single-stranded
phagemid vectors as driver and target.  Although this method has been described
by Duguid and co-workers, it differs in providing directional insertion of
cDNA or other DNA fragment into ss phagemids, quantitation of efficiency of
subtraction and protocol employed to purify ss DNA.
<p>
Phagemid vector series were made based on pBluescript SK(&minus;) and KS(&minus;)
vectors from Stratagene.  Standard cloning methods (Sambrook et al <i>Molecular
Cloning</i> 1989, Davis-Dibner-Battey <i>Basic Methods in Mol Biol</i> 1986,
Berger-Kimmel Guide to Mol Cloning Techniques 1987) were to modify vectors.
Four plasmids (E61, E145, E10, and E11) were constructed:  E61 derived from
pBluescript SK(&minus;) and with polylinker domain (restriction sites are
shown in fig 2).  Between the EcoRI and NotI sites is DQ&beta;2, a 950 bp
cDNA fragment encoding &beta; chian of DQ allele of HLA-D.  E145 is derived
from pBluescript KS(&minus;) also with polylinker in Fig 2.  It has the DQ&beta;2
cDNA inserted in polylinker but with opposite orientation to E61.  E10 has
relation to E61 except that TN5 neomycin phosphotransferase gene (<i>neo</i>)
replaces teh DQ&beta;2 gene.  E11 is related to E145 but the <i>neo</i> gene
replaces it and in opposite orientation to E10.  Plasmid E4 is similar to E61
but has no insert in the polylinker domain.
<p>
Single-stranded phagemid DNA was isolated from XL-1 bacteria (Bullock
et al, Biotech 5: 376, 1987) harboring ds form of phagemid using M13 rescue
technique (Messing, Meth Enzymol 101: 20, 1983; Yanisch-Perron et al, Gene 33:
103, 1985; Viera and Messing, Meth. Enzymol. 153: 3, 1987).  Overnight cultures
 are diluted to OD600 = 0.05 (~1:100 dilution) into 450 ml <q>superbroth</q>
(35 g tryptone, 20 g yeast extract, 5 g NaCl per liter, pH 7.5), grown at
37&deg; to OD600 = 0.1-0.2 with good aeration.  The M13 R408 strain is then
added to multiplicity of infection of 10-20 for 6-9 h.  Two rounds of
centrifuging at 6000 rpm/20 min 4&deg; (Sorvall GSA) and then vacuum filtration
through Whatman #2 to remove bacteria.  Phage are precipitated adjusting
PEG-8000 to 4% and NaCl to 0.5 M.  This is left 4&deg; for 2-4 h, then centrifuged
6000 rpm 40 min 4&deg;.  Pellet is dissolved in 0.5% SDS, 10 mM TrisHCl pH 7.4,
1 mM NaEDTA, 5 mM sodium sulfite, 0.92 g potassium iodide /ml, and 80 &micro;g
ethidium bromide per ml solution.  The lysate is centrifuged in Beckman Ti50
at 47,000 rpm 20&deg; 48 h.  The ss DNA fraction forms fluorescent band below
the ds DNA and above RNA.  Remove ss DNA using 18-gauge needle and syringe
and spin this smple in a Ti100 rotor at 70,000 rpm 18 h 20 &deg;.  Collect ss
DNA as before then extract ethidium bromide with 1-butanol.  Sample is diluted
with 2 vol 10 mM TrisHCl pH 7.4 and 1 mM EDTA, pptd with 6 vol EtOH. Residual
ds DNA is removed using restriction endonucleases and Mg-phenol extraction.
A marker plasmid (1 &micro;g marker per 100 &micro;g ss DNA) was added;
it expresses neomycin resistance or tetracycline resistance, but not ampicillin.
100 &micro;g ss DNA is digested with BglI and PvuII (1-2 U each per 1 &micro;g
ss DNA) for 2-4 h in 0.3 ml 100 mM NaCl, 10 mM TrisHCl pH 7.4, 10 mM MgCl<sub>2</sub>,
1 mM DTT in typical reaction.  These enzymes cut the ds vector at two sites
and do not affect ss DNA according to agarose gel electrophoresis or DNA
transformation assay.  For all procedures with ss DNA, siliconized microtubes
are used (PGC Scientific).  Reaction stopped with SDS and EDTA pH 8.0 to final
0.1% and 12.5 mM, resp.  150 &micro;l Tris-buffered phenol pH 7.5 is used
to extract proteins; the phenol phase is extracted with 300 &micro;l TE and
both aqueous phases combined.  MgCl<sub>2</sub> added to aqueous phase (~600 &micro;l)
so that it is final 100 mM, and 300 &micro;l Tris-buffered phenol added, and
solution vortexed.  ss DNA should precipitate; RNA and ds DNA remain in solution.
Spin 16000 <i>g</i> 10 min to pellet: it is often at bottom of tube but can
something be at phenol-water interface.  Remove aqueous phase, and extract
phenol phase two times with 0.3 ml TE containing 100 mM MgCl<sub>2</sub>.
Dissolve the ss DNA in 0.1 mL 50 mM sodium EDTA pH 8.0 to phenol phase, and
again vortex and spin.  Collect aqueous phase, and phenol phase and residual
precipitate extracted twice with 150 &micro;l TE.  pH 6.5 sodium acetate
(45 &micro;l of 3 M solution) and 1 ml EtOH added to pooled aqueous phases
to ppt DNA.  Subject to another round of BglI and PvuII digestion and Mg-phenol
extraction.  The marker DNA was reduced by 10<sup>6</sup>-fold and a &gt;75%
yield of ss DNA obtained.  Store ss DNA in TE at 4&deg;.
<p>
ss DNA is biotinylated by mixing 50 &micro;l 1 &micro;g/&micro;l ss DNA in
water with 50 &micro;l 1 &micro;g/&micro;l long arm Photoprobe biotin
and irradiating in open tube on ice bath with 275 W sun lamp from 10 cm
distance for 15 min.  Another 25 &micro;l added  of photobiotin is added
and another 15 min.  To wash out unconjugated biotin, add 12.5 &micro;l
1 M TrisHCl pH 9.0 and extract with water-saturated 2-butanol for 5 times.
Add pH 6.5 sodium acetate to 0.3 M, and ppt with 2 vol EtOH.
<p>
For subtractive hybridization, prepare control sample of only target ss DNA
and the experimental sample is target and biotinylated driver ss DNA.
The R408 ss DNA (10% of target concentration) is present in all reactions.
In  one set of hybridizations 800 ng  biotinylated E145 (E145B) is the
driver and either E61 or E4 (100 ng) or E10 (50 ng) is the target in 20 &micro;l
consisting of 0.2 M NaCl, 50 mM HEPES pH 7.6, and 2 mM sodium EDTA.  Sample
is placed in 650 &micro;l siliconized tube and covered with light mineral oil.
Heated to 95&deg; for 2 min, then incubated 65&deg; for 11 h.  In another
set of reactions, 1800 ng biotinylated E11 (E11B) is driver and
200 ng E10 is target.  Added to these are 2.5 ng of ds pACYC184 containing
Tet<sup>R</sup> as control for DNA which cannot be subtracted.  DNA is
dissovled in 11 &micro;l 0.5 M NaCl, 50 mM sodium HEPES pH 7.6, 2 mM EDTA
and 40% deionized formamide, then heated to 70&deg; for 2 min and incubated
52&deg; 21 h.
<p>
The protcol of Sive and St John is used in modified form here.  After
hybridization, most of mineral oil removed and sample diluted to 300 &micro;l
with 0.5 M NaCl, 50 mM HEPES pH 7.6, 2 mM sodium EDTA, and 15 &micro;g/ml
strepavidin pre-warmed to 37&deg;, incubated 2 min.  The strepavidin-bound
biotin-DNA is extracted with 200 &micro;l phenol:chloroform (1:1) pH 7.5,
prewarmed to 37&deg;.  vortexed 5 s, and centrifuged 5 min at RT in 16000<i>g</i>.
Phenol:chloroform extraction is done 3 more times and organics back-extracted
with TE and thoe combined.  Aqueous phase is pptd with EtOH and dissolved in
TE.
<p>
Horizontal electrophoresis is done in 0.8-1.05 agarose (SeaKem, FMC) using
either TBE or TAE with 0.3 &micro;g/ml EtBr at 3 V/cm of ss DNA after
subtractive hybridization.
<p>
The ss DNA is converted to ds form: 100 ng (80 fmol/&micro;l) or less of target DNA
is dissolved in 30 &micro;l of 50 mM KCl, 10 mM TrisHCl pH 8.3, 1.5 mM MgCl<sub>2</sub>,
and 0.01% gelatin (Kodak) [Taq buffer].  30 ng (4 amol/&micro;l)
of 5'-GAGCTCCTCGAGGTCTAGAGGCCT-3' primer complementary to polylinker sequence
is added.  After denaturing 95&deg; 1 min, then annealing for 30 min at 55&deg;,
dNTPs (200 &micro;M each of dATP, dTTP, dCTP, and dGTP) and 0.5 U Taq polymerase
in buffer (pre-heated to 55&deg;) is added for 2 min, heated to 70&deg; for
10 min.  0.1% SDS and 12.5 mM EDTA pH 8.0 are added, then extraction in 40 &micro;l
1:1 phenol:CHCl<sub>3</sub>
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Bittner, M. A., Holz, R. W.
 </span>
 <span class="year">
(1990)
 </span>
 <span class="jname">
J. Neurochem.
 </span>
 <span class="jvol">
54,
 </span>
<span class="pages">
 205-210.
</span>
<p class="reftitl">
Phorbol esters enhance exocytosis from chromaffin cells by two mechanisms
<p class="Para1">
Cultured cells were plated to 500,000/cm<sup>2</sup> and 12 h before any 
experiment, media without Fungizone was used.  4-8 d old cultures were used. 
Cells pretreated with 30 nM TPA for 30 min caused increased 
Ca<sup>2+</sup>-dependent [<sup>3</sup>H]norepinephrine secretion.  The relative 
enhancement was very high at low Ca<sup>2+</sup> concentrations (0.3-1.0 
&micro;M).  With a 2 hr phorbol ester pretreatment, the secretion enhancement at 
10 and 30 &micro;M Ca<sup>2+</sup> was greatly increased while no significant 
change in enhancement was observed at &le; 1 &micro;M Ca<sup>2+</sup>.  (The same 
was true for an 8 hr preincubation but data not shown.  Phorbol dibutyrate 
(PDBu) also produced the same effect as TPA while the inactive phorbol ester 
(4&beta;-phorbol 13-acetate) produced no effect.
<p>
The subcellular distribution of PK-C was studied with 18-24 hr incubation with 
30 nM TPA.  A previous report shows that rolonged incubations with higher (&ge; 
1 &micro;M) concentrations of TPA decrease PK-C activity by 85%, but at 30 nM, 
TPA had little effect on the total amount of PK-C activity which was measurable. 
When TPA at 30 nM was incubated for 18 h and then chromaffin cells placed in 
TPA-free medium for 6 hr, there was a marked reduction in membrane-bound 
activity, returning to the level observed in untreated cells.  The enhancement 
of <sup>3</sup>H-norepinephrine secretion at 0.6 &micro;M Ca<sup>2+</sup> was 
reversed.  Note that the enhancement of secretion by a higher Ca<sup>2+</sup> 
(10 &micro;M) was maintained even after TPA had been removed for hours (30 min 
exposure to 30 nM TPA followed by TPA-free medium for 15.5 h).  Hence the 
enhancement at 0.6-1 &micro;M Ca<sup>2+</sup> is related to membrane-bound 
enzyme.  When the more water-soluble PDBu (100 nM) was used, the loss of 
enhanced secretion at low Ca<sup>2+</sup> and the loss of membrane-bound PK-C 
were more rapid when PDBu was removed than when TPA was used.
<p>
1 &micro;g/ml cycloheximide (CHX) was added to cells for 30 min, then 30 nM TPA 
added for 30 min in the continuing presence or absence of CHX.  With 1 &micro;M 
Ca stimulation, TPA treatment increased secretion of NE whether or not CHX was 
present.  With 10 &micro;M Ca stimulation, secretion was increased with TPA 
treatment the same with or without CHX.  The enhancement of secretion observed 
with longer TPA incubations was however inhibited by CHX.
<!-- ============================================================= -->
<p class="tags">
DNA purification, silica, guanidine thiocyanate
<p class="ref">
 <span class="authors">
Boom R, Sol CJA, Salimans MMM, Jansen CL, Wertheim-Van Dillen PME, Van Der 
Noordaa J
 </span>
 <span class="year">
(1990)
 </span>
 <span class="jname">
J. Clin. Microbiol.
 </span>
 <span class="jvol">
28, 
 </span>
 <span class="pages">
495-503
 </span>
<p class="reftitl">
Rapid and Simple Method for Purification of Nucleic Acids
<p class="Para1">
DNA will bing to silica or glass in presence of sodium iodide (NaI) or
sodium perchlorate (NaClO<sub>4</sub>).  Guanidinium thiocyanate (GuSCN)
also has been show to be useful in purification and detection of DNA and
RNA: it lyses cells and inactivates nucleases.  Guandiunium HCl has been
used in purification by is less effective at RNase inhibition.
<p>
In this procedure a small sample of human serum or urine, or bacteria scrape
from colony on plate is pipetted to vessel with size-fractionated silica particle
or diaotoms in a lysis buffer containing GuSCN.  Mammalian cells, viruses, 
and gram-negative bacteria were checked with this procedure.  DNA/RNA is bound
to the carrier silica/diatoms, and washed twice with GuSCN-containing buffer,
then twice with 70% EtOH, and once with acetone.  Complexes are dried and 
DNA eluted with low salt buffer.  This uses $0.50 of materials for the 
purification.  The silica particles can be silicon dioxide from Sigma; this
is acid-washed (add HCl to pH 2);  60 g of material will do ~1500 purifications.
After acid washing, small portions can be put in tightly closed glass bottles
and autoclaved.  As for diatoms, analytical grade Celite (10 g) can be suspended
in 50 ml water and acidified (0.5 ml 32% HCl); these can be aliquotted for 
autoclaving.
<p>
The lysis buffer L6 is made with 120 g GuSCN in 100 ml 0.1 M Tris HCl pH 6.4.
22 ml of 0.2 M EDTA is adjusted to pH 8.0 and 2.6 g Triton X-100 is added to the
GuSCN.  The washing buffer L2 is made by dissolving GuSCN with Tris as above,
and it contains no EDTA or TX-100.  Note that GuSCN dissolves with difficulty,
so shakingin a 60-65&deg; water bath helps.  Both L6 and L2 are stable for at least
3 w at room temp in dark.  The elution buffer is standard TE (10 mM TrisHCl +
1 mM EDTA pH 8.0).  GuSCN should be prepared in fume hood since contact with
acid can generate HCN;  moreover, GuSCN waste is collected in a strongly alkaline
solution, starting at 10 N NaOH, and not allowed to fall below 0.3 N.
<p>
For reaction 900 &micro;l L6 buffer and either 40 &micro;l silica or 
40 &micro;l diatom suspension is added to 1.5 ml conical tube.  Strong vortexing
is done.  These reaction vessels are effective for up to 1 week.
<p>
When using serum or urine, a 50 &micro;l sample is added and then vortexed about
5 s.  Left 10 min at room temp, it is vortexed again then centrifuged max
speed for 15 s.  The supernatant is aspirated.  The pellet is washed with L2
wash twice, then twice with 70% EtOH and then once with acetone.  Removing the
acetone, vessels are dried at 56&deg; in heating block for 10 min.  Finally
a TE elution is done (with or without RNase inhibitor):  first vortexing, then
let sit 10 min at 56&deg;, then vortexing and spinning in microfuge at max speed.
Collect the supernatant.  For diatoms, follow the same procedure; it should
be noted that DNA yields might be greater.
<p>
This system was tested with plasmid pHC624 mixed with serum and urine and added
in doubled amounts (5, 10, 20, 40 &micro;g) in constant volume and used with
the silica particle protocol.  Recoveries were near 100%, as band intensities
were similar to the control being the 100% marker. The superhelical plasmid also
show relaxed circular and linear forms, probably resulting during washing.
<p>
In another experiement 10 randomly chosen urine samples are mixed with DNA ranging
in size from 60 bp to 23,000 bp, and purified using silica method.  Some of the
eluate was treated with T4 DNA ligase.  No differences were found with respect
to recovery or action of ligase.  When DNA fragment size is &lt; 4 bp, yields
are at least 50%.  At higher than 4 kb, there is shearing and a smear can be
apparent.  Another experiment:  <i>E coli</i> containing a 2.4 kb plasmid is 
mixed with the ten random urine samples and purified on the silica system and
eluted with RNase inhibitor.  Some of the eluate is treated with <i>Eco</i>RI
in presence of RNase A.  The gel analysis shows quite clearly excellent presence
of rRNA forms (23S and 16S) and the superhelical plasmid.  In the samples with
restriction + RNase, the RNA has disappeared and the plasmid is in a band with
slower mobility showing the linearized plasmid.
<p>
Serum samples were then randomly chosen and mixed with high and low MW DNA,
then extracted using silica method.  Some of the eluate is treated with
T4 DNA ligase or treated with <i>Hin</i>dIII.  The low MW DNA was recovered
in yields &gt; 50% and the ligase effective on these.  The high MW was a good
target for restriction enzyme and recovery ~50%; this high MW DNA is a 48 kb,
and the lower recovery is likely because of shearing, as shown by the use
of radiolabeled 48 kb DNA.
<p>
Hepatitis B virus is a 3.2 kb circular DNA, partly in single-stranded form.
So the extraction system was tested for ss DNA and relaxed circular.  
24 serum specimes with a 2.9 kb plasmid enriched for relaxed circular ds DNA
mixed in were run on silica system.  Both relaxed circular and ss DNA forms
show yields to be 50%, and was similar in all 24 specimens.
<p>
Smaller amount recoveries were tested.  These are nanogram and picogram amounts. 
<sup>32</sup>P-labeled DNA is mixed with human serum and extracted and the 
eluate counted in LSC.  At the picogram level, recovery is about 40%.  When 400 
ng carrier DNA is present, the yield increases to 60%.  High MW DNA was tested 
alone:  from 100 pg to 100 ng, mixed with serum and extracted in silica, the 
yields were about 50% over broad DNA concentration range.
<p>
When silica is used for very large amounts of genomic DNA purification (&gt; 5 
&micro;g), the complex betweeen silica and DNA is so tight that the complexes 
cannot be redispersed (washed).  This can be circumvented by using diatoms 
instead of silica. Diatoms are much larger than silica particles (although they 
too are silicates): their vortexing during elution likely allows them to be 
sheared off the commplex. The recovered genomic DNA has a lenghth of at least 30 
kb on average.  When the high copy plasmid pHC624 (used for recombinants) is 
purified from a 1 L overnight culture of <i>E coli</i> K-12 HB101, the yields 
are 10-20 mg plasmid. The diatoms method was used to purify plasmid from the 
equivalent of 400 &micro;l culture medium added to each agarose lane, and it 
shows up well.  This plasmid and genomic DNA were good substrates for 
restriction enzymes.
<p>
With this method, recovery rates are generally lower for oligos in the 40-60 bp
region.  Recoveries are not dependent on loading amount:  pico- and nanogram
loads are recovered in the same percentage.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Wilson, S. P. 
 </span>
 <span class="year">
(1990)
 </span>
 <span class="jname">
J. Biol. Chem. 
 </span>
 <span class="jvol">
265,
 </span>
<span class="pages">
648-651.
</span>
<p class="reftitl">
Regulation of chromaffin cell secretion and protein kinase C activity by chronic 
phorbol ester treatment
<p class="Para1">
Bovine chromaffin cells were exposed to high concentrations of PMA, PDBu, and 
PDD (4&beta;-phorbol 12,13-didecanoate) for 1 h to 4 d.  All treatments 
depressed the lipid-dependent protein kinase (protein kinase C) activity, and 
the higher doses (500 nM PDD or 1 &micro;M PMA) depressed activity maximally 
(1/10 to 1/5 of the untreated cells after 16-24 h). 
<!-- ============================================================= -->
<p class="tags">
<p class="ref">
 <span class="authors">
Elion, E. A., Grisafi, P. L., Fink, G. R. 
 </span>
 <span class="year">
(1990)
 </span>
 <span class="jname">
Cell
 </span>
 <span class="jvol">
60,
 </span>
 <span class="pages">
649-664.
 </span>

<p class="reftitl">
<i>FUS3</i> encodes a
cdc2<sup>+</sup>/CDC28-related kinase required for the transition from
mitosis into conjugation
<!-- ============================================================= -->
<p class="tags">
<p class="ref">
 <span class="authors">
Peter, M., Nakagawa, J., Dorée, M., Labbé, J. C., Nigg,
E. A. 
 </span>
 <span class="year">
(1990)
 </span>
 <span class="jname">
Cell
 </span>
 <span class="jvol">
60,
 </span>
 <span class="pages">
791-801.
 </span>

<p class="reftitl">
Identification of major nucleolar proteins as candidate
mitotic substrates of cdc2 kinase
<!-- ============================================================= -->
<p class="tags">
<p class="ref">
 <span class="authors">
Kipreos, E. T., Wang, J. Y. J. 
 </span>
 <span class="year">
(1990)
 </span>
 <span class="jname">
Science
 </span>
 <span class="jvol">
248,
 </span>
 <span class="pages">
217-220.
 </span>
<p class="reftitl">
Differential phosphorylation of c-abl in cell cycle
determined by <i>cdc2</i> kinase and phosphatase activity
<p class="author">Bischoff, J. R., Friedman, P. N., Marshak, D. R.,
Prives, C. R., Beach, D. (1990) <i>Proc. Natl. Acad. Sci.
USA</i> <b>??</b> 4766-4770.
<p class="reftitl">
Human p53 is phosphorylated by p60-cdc2 and cyclin B-cdc2
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Pines J, Hunter T 
 </span>
 <span class="year">
(1990)
 </span>
 <span class="jname">
The New Biologist
 </span>
 <span class="jvol">
2,
 </span>
 <span class="pages">
389-401.
 </span>
<p class="reftitl">
p34<sup>cdc2</sup>:  the S and M kinase?
<p class="Para1">
REVIEW. <i>S. cerevisiae CDC28.</i> p34<sup>cdc2</sup> required for entry into 
mitosis. Homologs of p34 one component of the vertebrate MPF. CDC28 must be 
present for entry into S phase, while cdc2<sup>+</sup> gene in S. pombe needed 
for "start." Lee and Nurse (1987) showed that human homolog of p34 can be used 
in defective yeast for G1 to S transition.
<p>
Mitosis-specific form of p34<sup> cdc2</sup> best exemplified as MPF. MPF 
purified and shown to have p34 component; activity is up at mitosis and 
disappears thereafter. Growth-associated histone H1 kinase activity also 
characterized (activity peaks at mitosis) as having p34 component.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Feiler HS, Jacobs TW 
 </span>
 <span class="year">
(1990)
 </span>
 <span class="jname">
Proc. Natl. Acad. Sci. USA
 </span>
 <span class="jvol">
87,
 </span>
 <span class="pages">
5397-5401.
 </span>
<p class="reftitl">
Cell division in higher plants:  a cdc2 gene, its p34-kDa product, and 
histone H1 kinase activity in pea
<!-- ============================================================= -->
<p class="tags">
<p class="ref">
 <span class="authors">
Félix, M.-A., Cohen, P., Karsenti, E. 
 </span>
 <span class="year">
(1990)
 </span>
 <span class="jname">
EMBO J.
 </span>
 <span class="jvol">
9,
 </span>
 <span class="pages">
675-683.
 </span>
<p class="reftitl">
Cdc2 H1 kinase is negatively regulated by a type 2A phosphatase in the 
<i>Xenopus</i> early embryonic cell cycle: evidence from the effects of okadaic acid
<!-- ============================================================= -->
<p class="tags">
<p class="ref">
 <span class="authors">
Moreno, S., Nurse, P. 
 </span>
 <span class="year">
(1990)
 </span>
 <span class="jname">
Cell
 </span>
 <span class="jvol">
61,
 </span>
 <span class="pages">
549-551.
 </span>
<p class="reftitl">
Substrates for p34: in vivo veritas?
<!-- ============================================================= -->
<p class="tags">
<p class="ref">
 <span class="authors">
Lewin, B. 
 </span>
 <span class="year">
(1990)
 </span>
 <span class="jname">
Cell
 </span>
 <span class="jvol">
61,
 </span>
 <span class="pages">
743-752.
 </span>
<p class="reftitl">
Driving the cell cycle:  M phase kinase, its partners, and substrates
<p class="Para1">
A MEETING REVIEW. An M phase kinase shown to be phosphorylated and 
dephosphorylated indicates that many events other proteins are involved in 
control of the cell cycle. Unravelling what has been known has come from genetic 
studies using <i>Schizosaccharomyces</i> <i>pombe</i>, and in studies of meiotic 
events in Xenopus. A paradigm is presented schematized in which the crucial 
points are at the G<sub>2</sub>/M phase boundary and in G<sub>1</sub>. A kinase 
of many names (MPF in meiotic Xenopus oocytes; histone H1 kinase) become 
activated in M phase and is responsible for releasing the block on meiosis in 
Xenopus oocytes. Histone H1 is phosphorylated by this kinase. It was initially 
purified by Lohka et al (1988) and had two components of MW ~34,000 and ~45,000. 
Reports by Dunphy et al (1988) and Gautier et al (1988) showed that the gene <i>
S pombe</i> was a homolog of the Xenopus p34 subunit. Sequence analysis of cdc2 
showed that it was protein kinase of serine/threonine residues. It is also a 
phosphoprotein requiring phosphates on some residues, dephosphorylation of 
others. The 45 kDa subunit has been identified as cyclin (Draetta et al, 1989), 
regulatory and needed for kinase activation and directing substrate targeting. 
This protein like all cyclins is destroyed and the kinase inactivated; its 
destruction is necessary to stop mitosis.
<p>
The M phase kinase also consists of other subunits in other species. In S pombe, 
it is p34<sup>cdc2</sup> associated with p60. In S cerevisiae it is p34<sup>
CDC28</sup> and its regulatory subunit is unknown. In X leavis it is a p34 and 
cyclins identified as A, B1, and B2. In humans, a p34 and p62 and p60<i><sup>
cyclinA</sup></i>. In starfish, a p34 and p60, and in clams, a p34 and cyclins A, 
B1, and B2.
<p>
Regulation of mitosis and entry into other phases by this kinase may occur with 
several models possible. The M phase kinase may act as a master regulator, the 
linchpin, by phosphorylating target proteins which act on other targets to carry 
out the work leading to an event change (the cascade approach) or it may 
directly work to phosphorylate (activate or de-activate) the proteins whose 
function is directly involved with the details of phase transition (the 
workhorse approach).
<p>

<!--[if gte vml 1]><v:shapetype
 id="_x0000_t75" coordsize="21600,21600" o:spt="75" o:preferrelative="t"
 path="m@4@5l@4@11@9@11@9@5xe" filled="f" stroked="f">
 <v:stroke joinstyle="miter"/>
 <v:formulas>
  <v:f eqn="if lineDrawn pixelLineWidth 0"/>
  <v:f eqn="sum @0 1 0"/>
  <v:f eqn="sum 0 0 @1"/>
  <v:f eqn="prod @2 1 2"/>
  <v:f eqn="prod @3 21600 pixelWidth"/>
  <v:f eqn="prod @3 21600 pixelHeight"/>
  <v:f eqn="sum @0 0 1"/>
  <v:f eqn="prod @6 1 2"/>
  <v:f eqn="prod @7 21600 pixelWidth"/>
  <v:f eqn="sum @8 21600 0"/>
  <v:f eqn="prod @7 21600 pixelHeight"/>
  <v:f eqn="sum @10 21600 0"/>
 </v:formulas>
 <v:path o:extrusionok="f" gradientshapeok="t" o:connecttype="rect"/>
 <o:lock v:ext="edit" aspectratio="t"/>
</v:shapetype><v:shape id="_x0000_i1025" type="#_x0000_t75" style='width:197.25pt;
 height:235.5pt' o:ole="">
 <v:imagedata src="./CellCycleLitNotes_files/image001.wmz" o:title=""/>
</v:shape><![endif]-->

<img src="./CellCycleLitNotes_files/image002.gif" 
alt="">

<!--[if gte mso 9]><xml>
 <o:OLEObject Type="Embed" ProgID="MSDraw" ShapeID="_x0000_i1025"
  DrawAspect="Content" ObjectID="_1084085794">
 </o:OLEObject>
</xml><![endif]-->

<p>

The M phase kinase specificity is reported to
depend on S/T-P-R/K or R/K-S/T-P-R/K type sequences.
Work to identify substrates will likely consist of in
vitro techniques for which a question will remain as whether it is an in
vivo target.  Some criteria for establishing
targets include answering the following questions:
Are the in vitro phosphorylated sites the same as in
vivo sites cyclically phosphorylated when p34 is known to be active?
Does mutation of p34 block or alter
phosphorylation in vivo.  These studies are
more feasible in yeast where mutants for p34<sup>cdc2</sup> and
p34<sup>CDC28</sup> are available and genetic analysis is convenient and
rapid.  Unfortunately histone H1 and lamins,
potential substrates for obvious roles in mitotic reorganization are
unavailable.  Proof that phosphorylation of a
target has a physiological role can be seen if a mutant protein with no
phosphorylation site has an altering effect (something possible in other
than yeasts as well).
<p>
Histone H1 is a case in point.
It is phosphorylated (at two sites) during DNA
replication and further phosphorylated (at four sites) during
mitosis.  This was initially reported by
Bradbury et al (1974) who first made the connection to cell cycle events,
but was only elaborated upon 15 years later when Arion et al (1988) showed
that an H1 kinase activity had a p34 component.
What effect does H1 phosphorylation play, since no
chromatin structural changes have been observed?
It may play a role in chromosome condensation at M
phase.
<p>
Cyclins were first reported in 1983 by T. Hunt's
group and are proteins destroyed with each cell division.
Cyclins are either type A or B, with a stretch of ~150
amino acids making the distinction and only weakly related.
B cyclins are phosphorylated in mitosis, although
preventing phosphorylation does not affect the role they play in mitosis
nor affect destruction.  Little is known
other than that, and the A cyclins are not as well understood.
Cyclins are only one part of the requirement for p34
kinase activation.  A cyclins appear to
precede B cyclins in the timing of destruction.
Data show that cyclins may nonetheless be destroyed by
the same protease since the time separations are only on the order of a
few minutes.  p34 is usually produced in
excess to cyclins, with intact dimer available in active form.
p34 remains constant throughout the cycle.
A critical point results in activation to enter M
phase, inactivation to leave M phase:  indeed
mutant cyclin B resistant to proteolysis causes an arrest in metaphase.
<p>
The p60 found associated with either adenovirus E1A
or with HeLa cells having unknown function was shown to by a type A
cyclin.  Association of cyclin B with p34
occurs in G<sub>2</sub> (with activation at the G<sub>2</sub>/M boundary),
and association with cyclin A during S phase.
Cyclin B can associate with either phospho- or
dephosphop34, while cyclin A associates only with dephosphop34.
Cyclin B accumulates in the perinuclear region of
HeLa cytoplasm during G<sub>2</sub>, enters the prophase nucleus, migrates
to spindle caps, and then degrades at the metaphase/anaphase
boundary.  Such movements of cyclin may be
necessary to bring p34 in contact with its substrates.
<p>


<!--[if gte vml 1]><v:shape id="_x0000_i1026" type="#_x0000_t75"
 style='width:223.5pt;height:267pt' o:ole="">
 <v:imagedata src="./CellCycleLitNotes_files/image003.wmz" o:title=""/>
</v:shape><![endif]-->

<img src="./CellCycleLitNotes_files/image004.gif"
alt="">

<!--[if gte mso 9]><xml>
 <o:OLEObject Type="Embed" ProgID="MSDraw" ShapeID="_x0000_i1026"
  DrawAspect="Content" ObjectID="_1084085798">
 </o:OLEObject>
</xml><![endif]-->

<!-- ============================================================= -->
<p class="tags">
<p class="ref">
 <span class="authors">
Dasso, M., Newport, J. W. 
 </span>
 <span class="year">
(1990)
 </span>
 <span class="jname">
Cell
 </span>
 <span class="jvol">
61,
 </span>
 <span class="pages">
811-823.
 </span>

<p class="reftitl">
Completion of DNA replication is monitored by a feedback
system that controls the initiation of mitosis in vitro:
studies in Xenopus
<!-- ============================================================= -->
<p class="tags">
<p class="ref">
 <span class="authors">
Richardson, H. E., Stueland, C. S., Thomas, J., Russell,
P., Reed, S. I. 
 </span>
 <span class="year">
(1990)
 </span>
 <span class="jname">
Genes &amp; Development
 </span>
 <span class="jvol">
4,
 </span>
 <span class="pages">
1332-1344.
 </span>

<p class="reftitl">
Human cDNAs encoding homologs of the small
p34<sup>CDC28/cdc2</sup>-associated protein of <i>Schizosaccharomyces pombe</i>
<p class="author">Michalovitz, D., Halevy, O., Oren, M. (1990) <i>Cell</i> 
<b>62,</b> 855-862.
<p class="reftitl">
A cdc2-like protein is involved in the initiation of DNA replication in 
Xenopus egg extracts
<!-- ============================================================= -->
<p class="tags">
<p class="ref">
 <span class="authors">
Th'ng, J. P. H., Wright, P. S., Hamaguchi, J., Lee, M.
G., Norbury, C. J., Nurse, P., Bradbury, E. M. 
 </span>
 <span class="year">
(1990)
 </span>
 <span class="jname">
Cell
 </span>
 <span class="jvol">
63,
 </span>
 <span class="pages">
313-324.
 </span>

<p class="reftitl">
The FT210 cell line is a mouse G2 phase mutant with a
temperature-sensitive CDC2 gene product
<!-- ============================================================= -->
<p class="tags">
<p class="ref">
 <span class="authors">
Gould, K. L., Moreno, S., Tonks, N., Nurse, P. 
 </span>
 <span class="year">
(1990)
 </span>
 <span class="jname">
Science
 </span>
 <span class="jvol">
250,
 </span>
 <span class="pages">
1573-1576.
 </span>

<p class="reftitl">
Complementation of the mitotic activator, p80 by a human protein-tyrosine phosphatase
<!-- ============================================================= -->
<p class="tags">
<p class="ref">
 <span class="authors">
Kinoshita, N., Ohkura, H., Yanagida, M. 
 </span>
 <span class="year">
(1990)
 </span>
 <span class="jname">
Cell
 </span>
 <span class="jvol">
63,
 </span>
 <span class="pages">
405-415.
 </span>

<p class="reftitl">
Distinct, essential roles of type 1 and 2A protein
phosphatases in the control of the fission yeast cell division cycle
<!-- ============================================================= -->
<p class="tags">
<p class="ref">
 <span class="authors">
Chang, F., Herskowitz, I. 
 </span>
 <span class="year">
(1990)
 </span>
 <span class="jname">
Cell
 </span>
 <span class="jvol">
63,
 </span>
 <span class="pages">
999-1011.
 </span>

<p class="reftitl">
Identification of a gene necessary for cell cycle arrest
by a negative growth factor of yeast:  FAR1
is an inhibitor of a G1 cyclin, CLN2
<!-- ============================================================= -->
<p class="tags">
<p class="ref">
 <span class="authors">
Solomon, M. J., Glotzer, M., Lee, T. H., Philippe, M.,
Kirschner, M. W. 
 </span>
 <span class="year">
(1990)
 </span>
 <span class="jname">
Cell
 </span>
 <span class="jvol">
63,
 </span>
 <span class="pages">
1013-1024.
 </span>

<p class="reftitl">
Cyclin activation of p34
<p class="author">Furukawa, Y., Piwnica-Worms, H., Ernst, T., Kanakura,
Y., Griffin, J. D., (1990) <i>cdc2</i> gene expression at the G<sub>
1</sub> to S transition in human T lymphocytes
<!-- ============================================================= -->
<p class="tags">
<p class="ref">
 <span class="authors">
Belenguer, P., Caizergues-Ferrer, M., Labbè, J.-C.,
Dorèe, M., Amalric, F. 
 </span>
 <span class="year">
(1990)
 </span>
 <span class="jname">
Mol. Cell. Biol.
 </span>
 <span class="jvol">
10,
 </span>
 <span class="pages">
3607-3618.
 </span>

<p class="reftitl">
Mitosis-specific phosphorylation of nucleolin by p34 protein kinase
<!-- ============================================================= -->
<p class="tags">
<p class="ref">
 <span class="authors">
Chambers, T. C. &amp; Langan, T. A. 
 </span>
 <span class="year">
(1990)
 </span>
 <span class="jname">
J. Biol. Chem.
 </span>
 <span class="jvol">
265,
 </span>
 <span class="pages">
16940-16947.
 </span>
<p class="reftitl">
Purification and characterization of growth-associated
histone H1 kinase from Novikoff hepatoma cells
<p class="Para1">
A procedure for extraction and purification of the growth-associated histone H1 
kinase is given. Of interest is the preparation of chromatin pellets. 
Exponentially growing Novikoff hepatoma cells are used by the authors because no 
detectable cAMP-dependent PKase or free catalytic subunit, histone kinase 2, or 
protein kinase C is present. The chromatin-bound nature of H1 kinase activity 
was demonstrated by isolating oligonucleosomes on sucrose gradients which had 
been generated by micrococcal nuclease digestion of washed nuclei.
<p>
Purification of the H1 kinase involved salt extraction (0.4 M NaCl), ammonium 
sulfate precipitation, and chromatography on Sephadex G-200 (elutes at 125,000), 
14 column volume gradient elution from DEAE-cellulose (elutes between 20 and 50 
mM (NH<sub>4</sub>)<sub> 2</sub>SO<sub>4</sub>), and same volume gradient 
elution from phosphocellulose (elutes in a broad range between 0.05 and 0.2 M 
KCl, with the most activity at the end--placed into pool B). SDS-PAGE of 
phosphocellulose pools A and B show 8-10 stained bands. A preparative sucrose 
gradient was done on pool B, producing a single uniform peak. SDS-PAGE across 
the active fractions in the gradient reveal a 60 KDa and 33 KDa bands which co-
sediment. Other bands (a doublet at 50 KDa and single bands at 64 KDa) are ruled 
out as being necessary because of immunological evidence showing H1 kinase as 
being composed of 60 Kd and 33 Kd subunits.
<p>
The purification scheme for 7 steps is given. 20 g wet wgt cells (10 billion 
cells, 9.6 L culture) produced a 0.4 M NaCl extract of chromatin containing 68 
mg total protein with 14.7 U total activity (1 U=1 &micro;Mol phospho-H1 histone 
produced per hour). The phosphocellulose A pool contained 112 &micro;g total 
protein with 6.1 U total activity (37% yield, 777-fold purified) while the B 
pool had 17 &micro;g total protein with 2.7 U activity (17% of the initial activity 
and 2300-fold purified). The purification of B pool over the sucrose gradient is 
estimated to be nearly 10,000-fold althought the protein was too little to 
measure. Cells exposed to 0.27 &micro;M (100 ng/ml) colcemid for 7 hr prior to 
harvest showed the same activity initially, but activity declined more rapidly 
during the early stages of purification than with interphase cells.
<p>
The activity pH optimum is broad between 9.0 to 7.0, but falls to 70% of maximum 
at pH 6.5. Mg<sup>2+</sup> at 5 mM gives maximal activity. Heat-stable inhibitor 
of PK-A does not inhibit. A K<sub> m</sub> of 16.6 &micro;g/ml (0.8 &micro;M) is 
shown for histone H1. The chymotryptic amino terminal of H1 is phosphorylated 
with K<sub>m</sub> of 6 &micro;M. Both ATP and GTP work with a 110 &micro;M 
K<sub> m</sub> for ATP but M-M kinetics not applying for GTP. After DEAE-
cellulose, no detectable proteolytic or phosphatase activity is shown, and this 
pool can be stored at 4 C in excess of 6 months without an appreciable loss of 
activity. The multiple forms eluted from ion-binding columns is probably due to 
post-translational modification heterogeneity (multiply phosphorylated forms).
<p>
Substrate specificity of the purified kinase was tested against histones and 
other chromosomal and non-chromosomal proteins. Stoichiometric phosphorylation 
of 3-6 mol P/molecule is observed whereas protamine, histone H3, HMG-1 and 2 are 
not phosphorylated at all.
<p>
Immunochemical studies of the 33 kDa component show that it is cdc2<sup>+</sup>
/CDC28. The 60 kDa subunit cross-reacts with anti-sea urchin cyclin A. The 
holoenzyme composed of two 33 kDa subunits and one 60 kDa subunit would explain 
the 125,000 estimated MW from gel filtration, but the true oligomeric nature was 
not determined. Characterization of substrate specifity previously showed a 
sequence recognition of Lys-Ser/Thr-Pro-Lys and Lys-Ser/Thr-Pro-Xxx-Lys. H1 
kinase activity preparations from other sources (CHO cells, several murine cell 
lines, plasmacytomas, and Chinese hamster fibroblasts) were of a lower specific 
activity, especially as phosvitin, casein, histone H2B and protamine 
phosphorylation was reported. Direct comparison can be made with cdc2<sup> 
+</sup>/CDC28 protein kinase preparations: <i>Xenopus</i> MPF purified &gt;3000-
fold, composed of two subunits of 45 kDa cyclin and 32 kDa kinase. A mitotic 
CDC2-encoded kinase has been purified from HeLa cells 6700-fold.

<!-- ============================================================= -->
<p class="tags">
<p class="ref">
 <span class="authors">
D'Urso, G., Marraccino, R. L., Marshak, D. R., Roberts,
J. M. 
 </span>
 <span class="year">
(1990)
 </span>
 <span class="jname">
Science
 </span>
 <span class="jvol">
250,
 </span>
 <span class="pages">
786-791.
 </span>
<p class="reftitl">
Cell cycle control of DNA replication by a homologue from
human cells of the p34 protein kinase
<p class="author">Hosokawa, N., Hosokawa, Y., Sakai, T., Yoshida, M.,
Marui, N., Nishino, H., Kawai, K., Aoike, A. (1990) <i>Int. J.
Cancer</i> <b></b>, 1119-1124.
<p class="reftitl">
Inhibitory effect of quercetin on the synthesis of a
possibly cell cycle-related 17 kDa protein, in human colon cancer cells
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Tischler, A. S., Ruzicka, L. A., Perlman, R. L.
 </span>
 <span class="year">
(1990)
 </span>
 <span class="jname">
J. Neurochem.
 </span>
 <span class="jvol">
55,
 </span>
<span class="pages">
1159-1165.
</span>
<p class="reftitl">
Mimicry and inhibition of nerve growth factor effects:  interactions of 
staurosporine, forskolin, and K252a in PC12 cells and normal rat chromaffin 
cells in vitro
<p class="Para1">
Chromaffin cells from rats were obtained by collagenase digestion of entire 
neonatal adrenal glands and dissected medulla from adults.  Cells were plated on 
laminin-coated 8-well Lab-Tek chamber slides.
<p>
Light microscopy figures show that 10 nM staurosporine during culture promote 
short processes in PC12 cells, similar to NGF induction showed by Greene and 
Tischler.  Cells incubated with 50 nM staurosporine in presence of 1 &micro;M 
camptothecin did not show process formation.  Unlike NGF-induced processes, the 
staurosporine-induced processes attained maximal length after 2-5 d and 
maintained formation.  The concentration dependence of staurosporine study 
(#cells with processes vs. [staurosporine] shows processes are induced at 10 nM 
and are maximal at 100 nM.  If 100 ng/ml is present during dose response, nearly 
half of the cells without staurosporine added already have processes;  as 
staurosporine is added, the number of cells increases to maximal.  If 1 &micro;M 
forskolin is present during dose response, it has no effect with staurosporine 
absent, but the number of cells is nearly maximal with only 10 nM staurosporine, 
indicating an additive effect.  With both NGF and forskolin present, nearly all 
cells show maximal process formation, even without staurosporine, throughout the 
dose course.
<p>
Dose response curves for the NGF inhibitor, K252a, (cells with processes vs. 
[K252a]), were done in presence of either 50 nM staurosporine, 100 ng/ml NGF, or 
NGF + 1 &micro;M forskolin.  Increasing K252a had no effect on 
staurosporine-mediated process formation.  Inhibition of NGF-mediated process 
formation was rather dramatic, however.  This was alleviated somewhat with 
forskolin present, with a noticeable additive shift upward.
<p>
While NGF present in serum-free cultures allows PC12 cells to survive, 
staurosporine did not substitute for NGF in this manner.  In fact, fewer cells 
survived than in control serum-free cultures, suggesting a toxic effect.
<p>
Greene &amp; Tischler had showed that NGF caused small decreases in DOPA 
synthesis and CA storage in PC12 cells.  Staurosporine reverses those decreases. 
When NGF was mixed with dexamethasone (at 1 &micro;M), those reversal of 
decreases was dramatically apparent (Dex has a dramatic effect on DOPA synthesis 
and measurement of dopamine content).  With increases in [staurosporine] alone, 
a trend to increased DOPA synthesis and dopamine content was observed in some 
experiments.
<p>
In neonatal rat chromaffin cells, staurosporine also induces neurite outgrowth, 
but to a lesser extent than NGF.  This trend was also observed in adult 
chromaffin cells, but was not statistically significant.  A dramatic flattening 
is observed in both types of cells, accompanied by formation of numerous 
filopodia from the cell surface.  K252a did not prevent this flattening.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Lehner CF, O'Farrell PH
 </span>
 <span class="year">
(1990)
 </span>
 <span class="jname">
Cell
 </span>
 <span class="jvol">
61,
 </span>
 <span class="pages">
535-547.
 </span>
<p class="reftitl">
The roles of Drosophila cyclins A and B in mitotic
control
<p class="author">Meijer, L., Arion, D., Golsteyn, R., Pines, J.,
Brizuela, L., Hunt, T., Beach, D. (1989) <i>J</i>. <b>8</b>, 2275-2282.
<p class="reftitl">
Cyclin is a component of the sea urchin egg M phase
specific histone H1 kinase
<!-- ============================================================= -->
<p class="tags">
<p class="ref">
 <span class="authors">
Wittenberg, C. Sugimoto, K., Reed, S. I. 
 </span>
 <span class="year">
(1990)
 </span>
 <span class="jname">
Cell
 </span>
 <span class="jvol">
62,
 </span>
 <span class="pages">
225-237.
 </span>
<p class="reftitl">
G1-specific cyclins of S. cerevisiae: cell cycle peroidicity, regulation by 
mating pheromone, and association with the p34 protein kinase
<!-- ============================================================= -->
<p class="tags">
<p class="ref">
 <span class="authors">
Wang, J., Chenivesse, X., Henglein, B., Bréchot, C.
 </span>
 <span class="year">
(1990)
 </span>
 <span class="jname">
Nature
 </span>
 <span class="jvol">
343,
 </span>
 <span class="pages">
555-557.
 </span>
<p class="reftitl">
Hepatitis B virus integration in a cyclin A gene in a hepatocellular carcinoma
<!-- ============================================================= -->
<p class="tags">
<p class="ref">
 <span class="authors">
Roy, L. M., Singh, B., Gautier, J., Arlinghaus, R. B.,
Nordeen, S. K., Maller, J. L. 
 </span>
 <span class="year">
(1990)
 </span>
 <span class="jname">
Cell
 </span>
 <span class="jvol">
61,
 </span>
 <span class="pages">
825-831.
 </span>
<p class="reftitl">
The cyclin B2 component of MPF is a substrate for the
c-<i>Nature</i>
<b>346,</b> 760-763.
<p class="reftitl">
Human cyclin A is adenovirus E1A-associated protein p60
and behaves differently from cyclin B
<h1 id="InhbitorsRegulators">Cell Cycle Inhibitors/Regulators</h1>

<table>
<tr>
 <th>Compound/Inhibitor/Poison
 <th>Main Effect
 <th>Side Effects
 <th>Cell Type Studied
 <th>Useful Concentration
 <th>Cost for Large-Scale (16L)
 <th>References
<tr>
 <td>2'-(2-hydroxyphenyl)-2'-thiazoline-4'-carboxylic acid (TFAL)
  <br>its methyl ester
 <td>Fe chelator
  <br>G<sub>1</sub>/S block
 <td>
 <td>L1210
  <br>P388
 <td>
 <td>d
 <td>Elliot et al. (1988) Cancer Chemother Pharmacol 21: 233-236
  <br>Elliot et al. (1989) J. Med. Chem. 32: 1039-1043
<tr>
 <td>5,10-dideazatetrahydrofolic acid (DDATHF)
   <br>(affects purine biosynthesis)
   <br>glycinamide ribonucleotide transformylase inhibitor
 <td>folate-dependent enzyme inhibitor
   <br>minimal cytotoxicity
 <td><br><br>
    ATP and GTP levels depleted
 <td>HL-60
 <td>50 nM to 10 &micro;M
 <td>
 <td>Sokoloski et al. (1989) Cancer Res. 49: 4824-4828
<tr>
 <td>chloroquinoxaline sulfonamide (CQS)
 <td>antiproliferative
 <td>
 <td>hPBMC
 <td>10 &micro;M
 <td>
 <td>Branda et al. (1989) Biochem. Pharmacol. 38: 3521-3526
<tr>
 <td>
 <td>mechanism unk.
 <td>
 <td>mCTLL
 <td>
 <td>
 <td>
<tr>
 <td colspan="7" style="height:4pt;background-color:silver;">
<tr>
 <td>quercetin (3,3',4',5,7-pentahydroxyflavone)
 <td>antiproliferative;
 <td>
 <td>COLO320 DM
 <td>60 to
 <td>ca. $0.28
 <td>Hosokawa et al. (1990) Int. J. Cancer 45: 1119-1124
<tr>
 <td>FW = 338.3
 <td>suppresses 17 kDa protein
 <td>
 <td>(HeLa, L1210,
 <td>100 &micro;M
 <td>(Sigma or
 <td>
<tr>
 <td>
 <td>G1/S block evident
 <td> 
 <td>HGC-27 also)
 <td>
 <td>Aldrich)
 <td>
<tr>
 <td colspan="7" style="height:4pt;background-color:silver;">
<tr>
 <td>deferoxamine
 <td>Fe chelator
 <td>
 <td>CHP 126 (human
 <td>60 &micro;M
 <td>$10.50
 <td>Blatt et al. (1988) J Lab Clin Med 112: 433-436
<tr>
 <td>FW=656.8
 <td>
 <td>
 <td>neuroblastoma)
 <td>
 <td>(Sigma)
 <td>
<tr>
 <td colspan="7" style="height:4pt;background-color:silver;">
<tr>
 <td>aphidicolin
 <td>a-DNA polymerase
 <td>
 <td>L1210 (murine
 <td>1.5 &micro;M to
 <td>$133.00
 <td>Matherly et al. (1989) Anal. Biochem. 182: 338-345
<tr>
 <td>FW=383.5
 <td>inhibitor
 <td>
 <td>leukemia cells)
 <td>5 &micro;M
 <td>to
 <td>
<tr>
 <td>
 <td>
 <td>
 <td>3T3 and V79
 <td>0.5 to
 <td>$513.00
 <td>Das (1987) Biochim Biophys Acta 928: 29-35
<tr>
 <td>
 <td>
 <td>
 <td>
 <td>1.0 &micro;g/ml
 <td>(Sigma)
 <td>
<tr>
 <td colspan="7" style="height:4pt;background-color:silver;">
<tr>
 <td>12-o-tetradecanoylphorbol-13-acetate (TPA)
 <td>PK-C activator;
 <td>
 <td>KOPM-28
 <td>10 ng/ml
 <td>$3.95
 <td>Tsuda et al. (1988) Int. J. Cell Cloning 6: 209-220
<tr>
 <td>
 <td>cytodifferentiation
 <td>
 <td>human leuk.
 <td>
 <td>(Sigma)
 <td>
<tr>
 <td colspan="7" style="height:4pt;background-color:silver;">
<tr>
 <td>buthionine-S,R-sulfoximine (BSO)
 <td>glutathione
 <td>
 <td>hPBMC
 <td>0.5 to
 <td>$53.67 to
 <td>Messina &amp; Lawrence (1989) J. Immunol. 143: 1974-1981
<tr>
 <td>FW=222.3
 <td>depletion
 <td>
 <td>
 <td>5 mM
 <td>$536.72
 <td>
<tr>
 <td>
 <td>
 <td>
 <td>
 <td>
 <td>(Sigma)
 <td>
<tr>
 <td>trichostatin A (TSA)
 <td>G1 block and
 <td>
 <td>rat 3Y1
 <td>
 <td>
 <td>Yoshida &amp; Beppu (1988) Exp. Cell Res. 177: 122-131
<tr>
 <td>trichostatin C (TSC)
 <td>G2 block
 <td>
 <td>
 <td>
 <td>
 <td>
<tr>
 <td colspan="7" style="height:4pt;background-color:silver;">
<tr>
 <td>hydroxyurea (HU)
 <td>early S block
 <td>
 <td>L1210
 <td>
 <td>$9/g or
 <td>Maurer-Schultze et al (1988) Exp. Cell Res. 174: 230-243
<tr>
 <td>
 <td>
 <td>
 <td>
 <td>
 <td>$50/10 g
 <td>
<tr>
 <td>heparin
 <td>delays S entry &amp;
 <td>
 <td>smooth musc
 <td>
 <td>$20 per
 <td>Castellot et al. (1989) J Cell Biol 109: 3147-3155
<tr>
 <td>
 <td>reduces entry from G0
 <td>
 <td>cells (SMC)
 <td>
 <td>50 mg
 <td>
<tr>
 <td>
 <td>
 <td>
 <td>
 <td>
 <td>(Sigma)
 <td>
<tr>
 <td>3-aminobenzamidine (3AB)
 <td>G1 or S block;
 <td>irreversible
 <td>L1210
 <td>10 to
 <td>$460 to
 <td>Smith et al. (1989) Exp. Cell Res.
<tr>
 <td>
 <td>G0 promoter
 <td>G2 block
 <td>L25A
 <td>25 mM
 <td>$1,150
 <td>
<tr>
 <td>
 <td>
 <td>
 <td>
 <td>
 <td>(Sig or Ald)
 <td>
<tr>
 <td>myxothiazol
 <td>cytochrome bc(1)
 <td>
 <td>Jurkat 886
 <td>0.5 &micro;g/ml
 <td>$62.72
 <td>Conradt et al. (1989) J. Antiobiotics 42: 1158
<tr>
 <td>FW=487.7
 <td>reductase inhibitor
 <td>
 <td>T lymphoblast
 <td>
 <td>(Sigma)
 <td>
<tr>
 <td colspan="7" style="height:4pt;background-color:silver;">
<tr>
 <td>tamoxifen
 <td>steroid hormone
 <td>
 <td>HIV-infected
 <td>
 <td>$10.30
 <td>Laurence et al. (1990) Blood 75: 696-703
<tr>
 <td>(Z)-1-[p-dimethylaminoethoxyphenyl]-1,2-diphenyl-1-butene
 <td>inhibition
 <td>
 <td>U1.A1
 <td>
 <td>/100 mg or
 <td>
<tr>
 <td>FW=371.5
 <td>
 <td>
 <td>
 <td>
 <td>$57/g (Sig)
 <td>
<tr>
 <td>6-methylmercaptopurine ribonucleoside
 <td>purine synthesis inhibition
 <td>
 <td>
 <td>
 <td>
 <td>
<tr>
 <td>
 <td>HL-60 cytodifferentiation
 <td>
 <td>
 <td>
 <td>
 <td>
<tr>
 <td>
 <td>
 <td>GTP and ATP depletion
 <td>
 <td>
 <td>
 <td>
<tr>
 <td>Dihydrofolate reductase (DHFR) inhibitors
 <td>
 <td>
 <td>
 <td>
 <td>
 <td>
<tr>
 <td>aminopterin
 <td>
 <td>
 <td>
 <td>
 <td>
 <td>
<tr>
 <td>methotrexate
 <td>
 <td>
 <td>
 <td>
 <td>
 <td>
<tr>
 <td colspan="7" style="height:4pt;background-color:silver;">
<tr>
 <td>Ribonucleotide reductase inhibitors
 <td>
 <td>
 <td>
 <td>
 <td>
 <td>
<tr>
 <td>hydroxyurea
 <td>
 <td>
 <td>
 <td>
 <td>
 <td>
<tr>
 <td>deferoxamine
 <td>
 <td>
 <td>
 <td>
 <td>
 <td>
<tr>
 <td>guanazole
 <td>
 <td>
 <td>
 <td>
 <td>
 <td>
<tr>
 <td>1-formylisoquinoline thiosemicarbazone
 <td>
 <td>
 <td>
 <td>
 <td>
 <td>
<tr>
 <td>parabactin
 <td>
 <td>
 <td>
 <td>
 <td>
 <td>
<tr>
 <td>compound II
 <td>
 <td>
 <td>
 <td>
 <td>
 <td>
</table>

<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Barinaga, M.
 </span>
 <span class="year">
(1991)
 </span>
 <span class="jname">
Science
 </span>
 <span class="jvol">
252,
 </span>
 <span class="pages">
505-506.
 </span>
<p class="reftitl">
On the right track to the NGF receptor
<p class="Para1">
Trk, a protein encoded by the <i>trk</i> gene, may be the long
lost component of the NGF receptor.  Initially cloned in 1985
from a human colon tumor, Trk was shown to resemble a structure
of members of the tyrosine kinase receptor family.  Later, the mouse
<i>trk </i>gene was cloned in 1987.  Northern analysis of mouse embryos
showed its expression in neural crest cells, terminally differentiated
and not at all mitotic.
<p>
Another lab at Stanford had shown in 1985 that cells
respond to two types of NGF receptors--one 80 kDa and another 140 kDa
receptor.  Both were required for NGF
response.  It was assumed that a 60 kDa
combined to the low MW 80 kDa to form a dimeric receptor.
But Trk was shown itself to be 140 kDa, and
subsequent studies with NGF revealed an autophosphorylated pp140 protein, which
has been identified as Trk.  Chemical
cross-linking has shown an assocation between Trk and the ligand NGF.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Kaplan, D. R., Hempstead, B. L., Martin-Zanca, D., Chao, M. V., Parada, L. F.
 </span>
 <span class="year">
(1991)
 </span>
 <span class="jname">
Science
 </span>
 <span class="jvol">
252,
 </span>
 <span class="pages">
554-558.
 </span>
<p class="reftitl">
The <i>trk</i> proto-oncogene product:  a signal transducing receptor
for nerve growth factor
<p class="Para1">
It had been shown earlier that NGF elicits a response in PC12
cells, which included autophosphorylation of tyrosines in
p140<sup>prototrk</sup>.  It is rapid and specific and occurs
in physiological amounts of NGF.  The neuroblastoma cell lines SY5Y
and LA-N-5 were used as hosts of the receptor and treated but produced lower
amounts of pp140<sup>prototrk</sup>; however,
the amount of trk mRNA is four-fold less as well, so expression may have been a
factor.  NGF-unresponsive NR18, derived
from PC12, have lost p75<sup>NGFR</sup>, the high affinity
LMW receptor:  no phosphorylated p140 was found in these cells
when exposed to NGF.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Nebreda, A. R., Martin-Zanca, D., Kaplan, D. R., Parada, L. F., Santos, E.
 </span>
 <span class="year">
(1991)
 </span>
 <span class="jname">
Science
 </span>
 <span class="jvol">
252,
 </span>
 <span class="pages">
558-561.
 </span>
<p class="reftitl">
Induction by NGF of meiotic maturation of Xenopus oocytes expressing
the trk proto&#8209;oncogene product
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Yan, H., Schlessinger, J., Chao, M. V.
 </span>
 <span class="year">
(1991)
 </span>
 <span class="jname">
Science
 </span>
 <span class="jvol">
252,
 </span>
 <span class="pages">
561-563.
 </span>
<p class="reftitl">
Chimeric NGF-EGF receptors define domains responsible for neuronal
differentiation
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Wright, B. D., Henson, J. H., Wedaman, K. P., Willy, P. J., Morand, J. N.,
and Scholey, J. M.
 </span>
 <span class="year">
(1991)
 </span>
 <span class="jname">
J. Cell Biol.
 </span>
 <span class="jvol">
113,
 </span>
 <span class="pages">
817-833.
 </span>
<p class="reftitl">
Subcellular localization and sequence of sea urchin kinesin
heavy chain:  evidence for its association with membranes
in the mitotic apparatus and interphase cytoplasm
<p class="Para1">
<b>Intro</b>.  Kinesin is a heterotetrameric, two-headed motor protein which 
uses MgATP to move particles toward the plus end of microtubule tracks in vitro. 
Mutations in kinesin have been observed to cause neuromuscular malfunctions 
which are consistent with defects in axonal vesicle transport.  Kinesin has been 
purified from sea urchin eggs and early embryos, in cells likely to form a 
mitotic apparatus. However, the role of kinesin in mitosis is unclear since 
antikinesins have been used in immunocytochemical staining and do not show up in 
mitotic spindles in other cell types.
<p>
<b>Results</b>.  Both Northern blotting and cDNA sequencing show that a 3093 
nucleotide open reading frame in 10 kb maternal mRNA encodes heavy chain of sea 
urchin kinesin. Deduced amino acid sequence of a protein from a cloned 4.8 kb 
cDNA, pKHC5, was obtained using overlapping cDNAs amplified by PCR using 
pKHC5-specific primers, and the protein is 1031 residues long (est. MW 117,390 
and pI=6.62). Comparison to previously published sequences of Drosophila and 
squid kinesin indicates organization into three structural domains:
<ol>
<li> head domain residues 1-393 with pI~8.08 containing ATP-binding consensus 
sequences (residues 84-91:  YGQTSSGKT);
<li>the stalk residues 394-857 with pI=4.83;
<li> the COOH terminal domain residues 858-1031 with pI=11.53.  A great deal of 
sequence similarity is present in the sea urchin sequence with the fly and squid 
sequences.
</ol>
<p>
SUK4, 6, and 7 are monoclonals known to bind to kinesin
heads.  Polyclonals to stalk and tail domains have been
affinity purified and epitopes to other mAbs mapped, all of
which involved preparation of bacterially expressed subfragments
of the heavy chain whose apparent mobility on SDS gels matched
the MW of the predicted sequence.  Head, stalk, and tail
fragments were prepared on NC blots after separation on SDS
gels and probed with the various mAbs.  SUK4 bound neither
head or tail, SUK2 bound both, and SUK5 binds only to the tail, as
shown in the tail+stalk fragment, but did not bind to stalk alone.
<p>
Homogenates of terminally differentiated coelomocytes and
cleavage-stage embryos (synchronously developed) were run on gels and
immunoblotted with antikinesin.  An M<sub>r</sub> 130,000 band was
found to react (heavy chain is est. to be 117 kDa from sequence analysis).
None was significantly detected in embryos at a stage where
flagella or cilia are present.  When methanol-fixed unlysed
and detergent-lysed first metaphase embryos were probed
with domain-specific anti-kinesins, anti-tubulin, and a nonspecific control
antibody, the peroxidase reaction indicated that kinesin is more concentrated
in the mitotic apparatus than in the surrounding cytoplasm.
1% NP-40-lysed preparations reduced anti-kinesin staining
in the cytoplasm was decreased with the MA staining became
more <q>fibrous,</q> which indicates that while kinesin-bound membranes
are stripped away, the kinesin remains associated with the MA
microtubules.  Other detergents (0.02% saponin, 0.2% Brij-58,
1% Triton X-100) produced the same effect as NP-40. Only 1% SDS
effectively solubulized the kinesin from MTs.  The fixative used (aldehyde
or MeOH) did not affect these results.
<p>
Kinesin, tubulin, and calsequestrin (a luminal ER protein)
distributions were analyzed in mitotic cells treated to disrupt MA membranes
and MTs using antibodies to the three proteins.  Peroxidase staining
shows that calsequestrin is concentrated in MAs, but with a less
fibrous distribution than that indicated of tubulin.  There was
diffuse cytoplasmic staining of calsequestrin as well.  10 &micro;G/mL
nocodazole-treated embryos caused a complete loss of antitubulin staining in
MAs, resulting in a <q>disorganization but not loss</q> of kinesin and
calsequestrin staining as well.  Detergent lysis before fixation
completely removed calsequestrin and caused appearance of the
fibrous appearance in anti-kinesin staining.  MT depolymerization
prior to detergent lysis followed by fixation completely eliminated
kinesin from MA.  Detergent lysis followed then by MT depolymerization
and then fixation proved that no concentration of nocodazole
could completely depolymerize the MTs within the time period studied.
These experiments suggest that kinesin
cross-links the membranes to MTs in the MA, in which kinesin remains associated
with the MA even when MTs depolymerize.
Likewise it remains associated with MTs even when membranes are disrupted.
<p>
Depletion of endogenous nucleotides with apyrase stabilizes
kinesin-MT interactions and it enhances anti-kinesin staining of the
MA.  Addition of MgATP dissociates kinesin from MTs and this completely
eliminated kinesin staining in the MA. MgADP, which also causes
dissociation in vitro, also depleted kinesin staining.
The results indicate that kinesin association is through
nucleotide-sensitive bonds, presumably through the motor domains.
<p>
Kinesin is constant in the embryo

<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Zheng, B., Woo, C. F., Kuo, J. F.
 </span>
 <span class="year">
(1991)
 </span>
 <span class="jname">
J. Biol. Chem.
 </span>
 <span class="jvol">
266,
 </span>
 <span class="pages">
10031-10034.
 </span>
<p class="reftitl">
Mitotic arrest and enhanced nuclear protein phosphorylation in human
leukemia K562 cells by okadaic acid, a potent protein phosphatase
inhibitor and tumor promoter
<p class="Para1">
Okadaic acid is a polyether fatty acid and diarrhetic shellfish poisoning factor 
found in marine sponges.  Growth of K562 cells is inhibited by the toxin at 
10-15 nM, with no effect at 1 or 5 nM. <sup>3</sup>H-Thymidine uptake and 
concomitant loss of cell viability are decreased when various doses are plotted 
versus treatment time.  Cytological analysis revealed that okadaic acid probably 
impairs a point in mitotic anaphase:  chromosomal condensation is seen but 
scattering of the condensed forms is noticed. Colchicine controls at a typical 
50 to 100 mM concentration show metaphase block.  Action of the poison added for 
24&nbsp;h&nbsp;is shown to be reversible, as incubation of poisoned cells in 
toxin-free medium restored normal thymidine uptake after 4 days.
<p>
Comparison with the tumor promoter TPA using saponin-permeabilized K562 cells 
showed that both were as effect in promoting overall total protein 
phosphorylation, and that in combination, a geometric increase is observed. 
Subcellular localization revealed that with TPA or okadaic acid more nuclear 
proteins were phosphorylated than post-nuclear proteins.  The same effect on 
nuclear proteins was observed in native (non-saponin treated) cells, except the 
post-nuclear protein exhibited a higher (quantitative) and different 
(qualitative) pattern of phosphoproteins.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Cohen, P.
 </span>
 <span class="year">
(1991)
 </span>
 <span class="jname">
Meth. Enzymol.
 </span>
 <span class="jvol">
201,
 </span>
 <span class="pages">
389-398.
 </span>
<p class="reftitl">
Classification of protein-serine/threonine phosphatases:
identification and quantitation in cell extracts
<p class="Para1">
Phosphatases can be classified as follows (see table)

<table>
<tr>
 <th>Property
 <th>PP1
 <th>PP2A
 <th>PP2B
 <th>PP2C
<tr>
 <td>Phosphorylase kinase subunit preference
 <td>beta subunit
 <td>alpha
 <td>alpha
 <td>alpha
<tr>
 <td>Inhibition by Inhibitor-1 and Inhibitor-2
 <td>yes
 <td>no
 <td>no
 <td>no
<tr>
 <td>Absolute requirement for divalent cation
 <td>no
 <td>no
 <td>yes (Ca<sup>2+</sup>)
 <td>yes (Mg<sup>2+</sup>)
<tr>
 <td>Calmodulin  stimulation
 <td>no
 <td>no
 <td>yes
 <td>no
<tr>
 <td>Inhibition by trifluoperazine
 <td>no
 <td>no
 <td>yes
 <td>no
<tr>
 <td>Okadaic acid inhibition
 <td>potent
 <td>very potent
 <td>weak
 <td>no
<tr>
 <td>Phosphorylase phosphatase activity
 <td>high
 <td>high
 <td>very low
 <td>very low
<tr>
 <td>Heparin-Sepharose binding at 0.1&nbsp;M NaCl
 <td>retained
 <td>excluded
 <td colspan="2">
</table>

<p>
These phosphatases are found in virtually every mammalian tissue
and are likely the highest in brain. Okadaic acid is a potent tumor promoter.
Assays for PP1, PP2A, PP2B, and PP2C have been already described
in this series (<i>Meth. Enzymol.</i> <b>159</b>: 390, 409, 416, and 426).
Activity can be measured against logarithmic increases in [okadaic acid],
with PP2A and PP1 showing markedly different IC<sub>50</sub> of ~50 pM and
5-10 nM, resp, which can be used to distinguish these proteins.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Cohen, P. T. W., Berndt, N.
 </span>
 <span class="year">
(1991)
 </span>
 <span class="jname">
Meth. Enzymol.
 </span>
 <span class="jvol">
201,
 </span>
 <span class="pages">
408-414.
 </span>
<p class="reftitl">
Reactivation of protein phosphatase 1 expressed at high levels
from recombinant baculovirus
<p class="Para1">
The baculovirus/insect cell system is superior to bacterial, yeast, and 
mammalian expression systems since active nonfusion proteins can undergo 
posttranslational processing, can target to the correct organelle, and can be 
expressed in significant amouunts despite toxicity to the cell. Protein 
phosphatase 1 (PP1) cDNA with its 5' noncoding region truncated was ligated into 
pAcYM1 which is under the control of polyhedrin promoter, with only a few bases 
between the promoter and the ATG start.  [Bluescript plasmid (Stratagene) 
containing PP1 cDNA from rabbit skeletal muscle is digested with HindIII to 
linearize, partially digested with NarI, and fragments isolated by agarose gel 
electrophoresis.  The pertinent fragment is electroeluted, its overhanging ends 
removed with mung bean nuclease, and ligated with BamHI linkers prior to 
ligation to BamHI-digested pAcYM1.] Recombinant plasmid is cotransfected with 
the wild-type virus, and recombination events help yield an active virus which 
expresses PP1 instead of polyhedrin. After techniques are done to purify the 
virus, a time course for maximal expression of protein is conducted with a 1 
pfu/cell moi. Maximal expression was observed after 42 h.  Harvested cells are 
washed with 50 mM Tris, pH 7.0, 1 mM DTT, and 0.25&nbsp;M sucrose. Cells are 
extracted in the same buffer also containing 0.5&nbsp;M NaCl. Lysing is with 4 
freeze-thaw cycles in a dry ice-ethanol bath. Lysate can be stored for 2 months 
at -20&deg;.
<p>
PP1 is subsequently purified by making the lysate 1% (v/v) with Triton X-100, 
letting sit for 60 min at RT, then spinning for 10 min at 15000 g; the 
supernatant is assayed. The pellet is washed with buffer (Tris, DTT, NaCl) 
containing 1% TX-100.  This is centrifuged. Now the pellet contains &gt;80% pure 
PP1 which can be solubilized with Tris, DTT, and 6&nbsp;M guanidinium HCl. After 
letting the mixture sit 1 hr, the solution is passed through a 0.22 &micro; 
filter. The solution is now diluted with rapid mixing with 1000 volumes of 50 mM 
Tris, 1 mM MnCl<sub>2</sub>, 50 mM 2-mercaptoethanol, 0.5&nbsp;M NaCl, 0.02% 
Tween-20. Reactivation of PP1 occurs in 60 min and is stable for &gt;3 hr at 
room temp.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Christanson, T. W.
 </span>
 <span class="year">
(1991)
 </span>
 <span class="jname">
Biotechniques
 </span>
 <span class="jvol">
10,
 </span>
 <span class="pages">
608-610.
 </span>
<p class="reftitl">
Several methods of quick freezing in an ultracold freezer
<p class="Para1">
Liquid nitrogen and dry ice are typically used to quick freeze samples which 
preserves cell viability and enzyme activity. For laboratories which do not have 
easy access to liquid nitrogen or dry ice. As coolants, an isopropanol slush, or 
granular aluminum (10-30 mesh) both dry and slurried in ethanol, are helpful. 
Aluminum, because of its high heat conductivity and low cost, is ideal.  Tubes 
can be placed in it much like a sand bath.
<p>
To compare, 10&nbsp;ml&nbsp;95% EtOH was placed in a 20 x 150 mm Pyrex tube, and 
a -20&deg; to 100&deg; thermometer inserted via a stopper until it was in the 
middle of the EtOH.  The tube was warmed to 35&deg;, then placed in the test 
coolant up to the EtOH level. The time required to reach -20&deg; was recorded. 
<q>Wet</q> coolants were more effective than dry coolants. Liquid 
N<sub>2</sub> was the best, followed by dry ice-EtOH bath.  Granular Al in EtOH 
pre-cooled to -90&deg; was next, followed by that same slurry pre-cooled only to 
-70&deg;.  The isopropanol syrup/slush pre-cooled to -70&deg; was the least 
effective, but superior to the dry coolants, which were granular Al at -90&deg;, 
crushed dry ice, and granular Al at -70&deg;, in order of effectiveness.  The 
freezing time of 1&nbsp;ml&nbsp;water in a 1.5&nbsp;ml&nbsp;polypropylene 
microfuge tube was also tested, but the ranking of the order of effectiveness 
did not change from the ethanol cooling study.

<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Tyler-Cross, R., Schirch, V.
 </span>
 <span class="year">
(1991)
 </span>
 <span class="jname">
J. Biol. Chem.
 </span>
 <span class="jvol">
266,
 </span>
 <span class="pages">
22549-22556.
 </span>
<p class="reftitl">
Effects of amino acid sequence, buffers, and ionic strength on
the rate and mechanism of deamidation of asparagine residues in small peptides
<!-- ============================================================= -->
<p class="tags">
VEGF, sequencing, vascular permeability factor
<p class="ref" id="Tischler-VEGF">
 <span class="authors">
Tischer E, Mitchell R, Hartman T, Silva M, Gospodarowicz D, Fiddes JC, Abraham 
JA
 </span>
 <span class="year">
(1991)
 </span>
 <span class="jname">
J. Biol. Chem.
 </span>
 <span class="jvol">
266,
 </span>
 <span class="pages">
11947-11954.
 </span>
<p class="reftitl">
The human gene for vascular endothelial growth factor. Multiple protein forms 
are encoded through alternative exon splicing.
<p class="Para1">
Polypeptide-based mitogens such as basic fibroblast growth factor (bFGF),
platelet-derived growth factor (PDGF) act on many cell types.  VEGF in contrast
targets vascular endothelial cells:  absence of activity has been verified
on fibroblasts, keratinocytes, vascular smooth muscle, lens epithelium, corneal
endothelium, adrenal cortical cells, and granulosa cells.  It has been purified
by Gospodarowicz et al (1989) and Ferrara and Henzel (1989) from conditioned
medium of bovine pituitary folliculo stellate cells which required use of 
endothelial cell proliferation assay.  Another protein called <b>vascular 
permiability factor</b> had its protein sequenced and cDNA cloned and shown to
be identical to VEGF.  Several tumor-derived cell lines express and secrete VEGF:
human histiocytic lymphoma cell line U-937, guinea pig line 10 hepatocarcinoma,
rat glioma cell line GS-9L, mouse neuroblastoma cell line NB41, mouse pituitary
cell line AtT-20.  It is a homodimeric 46 kDa protein.  It corresponds to 
several cDNA clones however, which express residues that are 189-, 165- and
121-residues long.  (The bovine subunit shows two forms:  164- and 120-residues
long.)  A 164-residue long form has been cloned in the rat.  It has one single
N-glycosylation site for which the 23 kDa size corresponds to the 164 residue
chain.  Sequence comparisons show distant relatedness to PDGF A and B chains;
it has 8 conserved Cys sites forming inter- and intradisulfide chains.  This 
paper identifies VEGF mRNA in vascular smooth muscle (VSM) cells and multiple forms of
protein and mRNA using PCR and cDNA cloning.
<p>
VSM cells are in close contact with endothelial cells.  Fetal human VSM cells 
were isolated and grown to confluence in DMEM + 10% FBS with Gln/pen-strep 
supplements, along with every other day addition of 1 ng/ml bFGF.  Also U-937 
cells were cultured in RPMI 1640 + 10% FBS and supplement to more than 4 
million/ml; these were diluted and supplemented with 20 ng/ml phorbol ester for 
3 d, then RNA isolated. Total RNA is isolated using GuanSCN using Chirgwin et al 
method. Poly(A) RNA is fractionated using oligo(dT)-cellulose and both total and 
poly(A) RNA are examined using agarose-HCHO gels (Lehrach et al 1977) and 
transferred to either nitrocellulose or Hybond-N nylon, then hybridized to 
nick-translated probes. A 43-base anti-sense primer 
5&prime;-GCCAAGCTTGCTCCTGCCCGGCTCACCGCCTCGGCTTGTCACA-3&prime; was used with
BoehringerMannheim cDNA synthesis kit with 5 &micro;g fetal human VSM poly(A)
RNA.  After 1st strand synthesis and cleaning up with phenol-chloroform and
EtOH pptn, amplification for 30 cycles was done with antisense and a sense
primer (5&prime;-GCCAAGCTTGAGTGTGTGCCCACTGAGGAGTCCAACATCACCATGCAG-3&prime;)
Primers have <i>Hin</i>dIII site near 5&prime; ends, so product is digested and
analyzed on 3% agarose.  After eluting from gel they are cloned into M13mp18
sequencing vector (Yanisch-Perron et al) for dideoxy sequencing (Sanger et al 1980).
<p>
Northern blot analysis of the human VSM showed transcripts at 5.5, 4.4 and
3.7 kb.  Others have shown a single 3.7 kb mRNA in bovine pituitary folliculo
stellate cells (Leung et al) and 4.2, 3.7, and 3.4 kb mRNAs  (Tischler et al),
and one 3.8 kb transcript has been shown in U-937 cells.
<p>
Conditioned medium from bovine VSM cells purification done on heparin-Sepharose
and C<sub>4</sub> RP HPLC.  Eluates were examined on 12.5% SDS-PAGE.
These proteins were transferred to PVDF.  Any protein migrating to bovine 
VEGF was excised for sequencing. An N-terminal sequence of APMAExGQ (where
x is unknown) was found:  this agrees with bovine VEGF sequence of APMAEGGQ
where x is identifed as G.
<p>
Previously a 189-amino acid form of hVEGF was obtained from U-937 cells (Keck et 
al 1989) and 121- and 165-aa forms from phorbol ester-activated HL60 cells 
(Leung et al 1989).  The 165-aa form has 44 residues inserted at a position 6 
amino acids from C-terminus. The 189-aa form has its 24 extra residues 
immediately upstream of 44-residue insert.  PCR analysis has confirmed the 
coding region heterogeneity.  Oligos flanking the variable C-terminal part
of coding region were using to prime PCR synthesis with human VSM ss cDNA
as template:  products were excised from agarose gel, subcloned into M13mp18
and sequenced.  Results show the three different forms made in human VSM
as for the HL60 cells.
<p>
Two &lambda; clones spanning the human VEGF gene representative of a human
lung fibroblast genomic library, were isolated by probes constructured from
bovine VEGF cDNA frag or a synthetic oligo which is the coding region for
portion of secretion signal of bovine VEGF.  Restriction enzyme maps were 
constructed for single and double digests of <i>Hin</i>dIII, <i>Eco</i>RI,
and <i>Bam</i>HI.  Southern analysis of human genomic DNA of these digests
showed band sizes predicted for the digestions.  These clones were overlapping
(both have exon 2) for a total length of ~28 kb, with a coding domain of
about 14 kb.  The sequencing of these was done.  Comparison of human cDNA
from the two reports and the gene sequence shows 8 exon/7 intron gene.
Intron/exon boundaries show proper consensus sequences splicing signals
(Breathnach &amp; Chambon).
<p>
<b>Alternative splicing</b>.  If all 8 exons are spliced into the transcript, 
the VEGF<sub>189</sub> form is generated.  If exon 6 is removed, the 
VEGF<sub>165</sub> form is generated, and this causes a K115N change.  If both 
exons 6 and 7 are left out the VEGF<sub>121</sub> form occurs, again with the 
K115N change.  Note that exon 7 contains 7 Cys residues, not present in 121-form 
but present in larger forms.  This 44-residue segment has 9 Arg and Lys 
residues, making it quite basic.  The 24-residue segment in VEGF<sub>189</sub> 
is also quite basic with 12 Arg + Lys residues.
<p>
The 5&prime; end of hVEGF mRNA was mapped using Northern and by primer extension
with synthetic oligos.  U-937 total RNA was probed by fragments of genomic
DNA cut using restriction enzyme to show the approximate location of 
transcriptional start.  Probe P2 (nt &minus;1063 to nt &minus;848) is the most
5&prime; still hybridizing to the mRNA.  Probe P1 (nt &minus;1388 to nt &minus;1064)
does not hybridize on Northern.  Then primer extension was done with two
antisense synthetic oligos whose 5&prime; ends are 35 bases apart.  This 
revealed a single major transcriptional start although other minor sites might
be seen:  this is 1038 bp upstream of the ATG initiation codon.
<p>
The fetal human VSM cells were also treated with phorbol ester (20 ng/ml
TPA) for various times to assess effect on expression.  Poly(A) RNA was
isolated and results show that VEGF mRNA does increase to TPA reaching a
maximum of 2-4 h.
<p>
Exon 1 of VEGF contains a majority of the secretion signal for VEGF.
Despite finding the three VEGF poly(A) RNA forms, there is no evidence that
all three proteins are actually translated into proteins.
The gene structure of VEGF is very similar to PDGF A and B chains with respect
to number of exons/introns, but at the protein sequence level, there is only
20-25% similarity.  Exon 1 codes for a long 5&prime;-UTR (ranging from 
840-1030 bp for VEGF and both PDGFs) and the majority of hydrophobic signal
domain.  Exon 2 in VEGF and exons 2 and 3 in PDGFs have no sequence similarity;
VEGF exon 3 and PDGF exon 4 are comparable (6 of 8 Cys residues conserved),
but intron/exon boundaries are no conserved.  VEGF exons 4 and 5 correspond
to PDGF exon 5, with two conserved Cys but little other similarity.  Exon 6
is not required for VEGF activtity since the 165-form has biological activity.
Exon 7 in VEGF has no counterpart in PDFG; it features 7 Cys residues.
<p>
There is a very GC-rich region just 5&prime; to the translational start,
both in VEGF and PDGF, and shown also for other growth factor genes.
What is also shown is an ATG codon in the 5&prime;-UTR that is not the
true start codon (at nt 185-187 upstream).  The two PDGF transcripts have three
ununsed ATG codons upstream, whose role is not understood.
<p>
Around the transcriptional start site (at &minus;1038 bp upstream ATG start 
codon) are a cluster of four GC boxes starting at 50 bp 5&prime; to transcript 
start.  These are 6 nt sequences in form 5&prime;-GGGCGG-3&prime; or 
5&prime;-CCGCCC-3&prime; at &minus;1096 to &minus;1091, &minus;1112 to 
&minus;1107, &minus;1123 to &minus;1118, and &minus;1133 to &minus;1128, sites 
for Sp1 transcription factor binding.  Three of the four sites also correspond 
to what are considered strong consensus for Sp1 TF: 
5&prime;-G/TGGGCGGG/AG/AC/T-3&prime; or complement 
5&prime;-G/AC/TC/TCCGCCCC/A-3&prime;.  No TATA box-like promoter close the start 
site is seen;  both PDGF gene promoters have TATA boxes.  Four potential 
AP-1 binding sites (5&prime;-TGAC/GTCA-3&prime;) at positions &minus;620 to
&minus;614, &minus;1528 to &minus;1522, &minus;2265 to &minus;2258, and 
&minus;2930 to &minus;2924 are found:  this shows how VEGF could be under
control of phorbol ester TPA or serum.  The presence of AP-2 sites 
(5&prime;-CCCCAGGC-3&prime; or 5&prime;-T/CCCCCA/CNG/CC/GG/C-3&prime;) at
&minus;135 to &minus;128 and &minus;1875 to &minus;1868 indicates both TPA-
and cAMP-inducibility.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Okuhara, D. Y., Macara, I. G.
 </span>
 <span class="year">
(1991)
 </span>
 <span class="jname">
Biotechniques
 </span>
 <span class="jvol">
10,
 </span>
 <span class="pages">
611.
 </span>
<p class="reftitl">
An efficient method to dialyze small volumes
<p class="Para1">
Samples between 0.1 and 2&nbsp;ml&nbsp;can be dialyzed using this
technique, which avoid loss of sample due to high surface area.
Commercial dialysis can be avoided.
<p>
Cut a Spectra/Por 1 dialysis membrane, 6.4&nbsp;ml&nbsp;diameter and 6-8 kDa 
cut-off, and tie a double knot 1 cm from the end. Cut 2 mm off the bottom of a 
microfuge tube.  Using a flame, form a lip around the severed end of the tube. 
Slip the dialysis membrane over the bottom of the tube. Twist a 5 x 2 cm 
Parafilm strip into a cord and wrap it tightly around the area where membrane 
and tube overlap. Wrap another 5 x 2 cm Parafilm strip flat against the first 
strip.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Miller, R. J.
 </span>
 <span class="year">
(1992)
 </span>
 <span class="jname">
J. Biol. Chem.
 </span>
 <span class="jvol">
267,
 </span>
 <span class="pages">
1403-1407.
 </span>
<p class="reftitl">
Voltage-sensitive Ca<sup>2+</sup> channels
<p class="Para1">
<b>REVIEW</b>.  Nowycky et al. first tried to categorize Ca channels.
They identified a low threshold current name T-type and two high threshold 
currents called L- and N-type. Dihydropyridine (DHP) drugs are known to bind to 
the L-channels, while omega-conotoxin (from the piscivorous marine mollusk 
<i>Conus geographus</i>) binds the N-type channels. Some channels are 
insensitive to DHPs and w-CgTx and are also not T-type.  They are called P-type 
since they occur in the cerebellar Purkinje neurons in high concentrations.
<p>
The skeletal muscle L-channels have offered the most about the structure of the 
ion channels.  Four distinct subunits of &alpha;1 (170 kDa), &alpha;2/&delta; 
(175 kDa), &beta; (52 kDa), and &gamma; (32 kDa) are found. The &delta; 
<q>subunit</q> can be separated from &alpha;2 by thiol reagents.  The cDNA 
sequence for the rabbit &alpha;1 predicts a protein of ~210 kDa, and its 
sequence shows considerable homology with subunits of members of the 
voltage-dependent ion channel superfamily including Na<sup>+</sup> and 
K<sup>+</sup> channels.  There is 30% homology between the sk musc &alpha;1 
subunit and Na<sup>+</sup> channel. Four quasi-homologous internal repeats are 
found with a predicted six membrane-spanning helices.  The 4th of these helices 
contain a region of positively charged amino acids occurring every 3 or 4 
residues (this is believed to be the voltage-sensing region). Affinity labeling 
studies indicate that &alpha;1 binds to L-channel drugs (DHPs and 
phenylalkylamines), specifically to the &alpha;1 subunit.
<p>
Various observations suggest that the &alpha;1 subunit is itself capable of 
forming a functional channel.  There have been attempts to express the subunit 
in Xenopus oocytes, but these have not worked.  Expression in mouse L cells has 
met with success as there are DHP-sensitive Ca<sup>2+</sup> channels.  Since 
these cells lack the &alpha;2/&delta;, &beta;, and &gamma; subunits, the belief 
that &alpha;1 can be used to form a channel by itself may be correct.
<p>
The physiological aspects of &alpha;1 subunit function differ in certain muscle 
tissues.  Cardiac muscle depends upon the proper function of the L channels for 
triggering muscle contraction; incubation in Ca-free medium arrests this 
phenomenon. Electrically-stimulated skeletal muscle, however, is able to 
contract in Ca-free medium;  this has been explained as the skeletal muscle 
depending upon an intracellular Ca source (the SR) which depends still upon a 
depolarization event throughout the transverse tubules. These intracellular 
channels are also DHP-sensitive, indicating the role of the &alpha;1 subunit as 
well, but the &alpha;1 subunit acts here as a sensing component not part of the 
channel formation but physically linked to SR channel for the purpose of 
signalling its opening (the ryanodine receptor) by a conformational change.
<p>
Much of the solubilized and immunoprecipitable &alpha;1 protein appears to be 
165-175 kDa on gels, but one group has reported that &lt;10% of the sk muscle 
tissue contains an M<sub>r</sub> ~210,000 form which is close to the protein's 
predicted size.  It is speculated that the larger form may be the form necessary 
in the assembly of the L channel, whereas the smaller form can be adapted by 
other channel proteins as a voltage-sensing associate. However, reconstitution 
studies in phospholipid bilayers indicates the formation of a functional L 
channel with only the low MW form of the &alpha;1 subunit present.
<p>
Molecular biology has distinguished the cardiac and sk musc forms of &alpha;1 
subunit. Mouse skeletal muscle myotubes possessing a muscular dysgenesis 
mutation have been critical to this study as the normal alpha1 subunit is absent 
and they are deficient in electrically stimulated excitation-contraction 
coupling and L-channels. Expression of sk musc alpha1 subunit produces myotube 
showing Ca<sup>2+</sup> currents with slow activation kinetics and contraction 
independent of extracellular Ca<sup>2+</sup> (as in sk musc.). Expression of 
cardiac alpha1 subunit produces myotubes showing Ca<sup>2+</sup> currents with 
fast activation kinetics and with contraction dependent upon extracellular 
Ca<sup>2+</sup>. Tanabe et al. (Nature 346: 567, 1990) constructed alpha1 
subunit chimeras by using the cardiac subunit and introducing sequences 
corresponding to intracellular loops from the sk muscle subunit. The resultant 
excitation-coupling was characteristic more of sk muscle, yet the Ca current was 
more like that of cardiac muscle, i.e. rapidly activating.  Conversely, the four 
homologous repeating domains of cardiac were placed into the sk muscle alpha1 
subunit sequence:  replacing the first one was sufficient to alter the Ca 
current to the cardiac type while replacing the other three had no effect.
<p>
Since determination of the sk musc alpha1 subunit, oligo probes under high and 
low stringency hyb conditions have been used to identify members of a 
superfamily in other tissues with unknown function. It is possible that one gene 
encodes and is alternatively spliced to produce an L-channel specific for its 
particular tissue.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Ellis, L., Levitan, A., Cobb, M. H., Ramos, P.
 </span>
 <span class="year">
(1988)
 </span>
 <span class="jname">
J. Virol.
 </span>
 <span class="jvol">
62,
 </span>
 <span class="pages">
1634-1639.
 </span>
<p class="reftitl">
Efficient expression in insect cells of a soluble, active human insulin
receptor protein-tyrosine kinase domain by use of a baculovirus vector
<p class="Para1">
Sf9 cells were either mock-infected or infected with wild-type AcMNPV or the 
virus AchIRPTK 75.5 which contains the recombinant human insulin receptor 
protein-tyrosine kinase domain (hIR PTK).  The insert corresponding to the hIR 
PTK starts from Gly-959 and continues some 400 amino acids to the end of the 
coding region for the IR beta subunit, which comprises the cytoplasmic domain of 
the receptor containing the PTK active site.  24 to 48&nbsp;h&nbsp;after 
infection, cells were metabolically labeled for 4-6&nbsp;h&nbsp;with 
L-[<sup>35</sup>S]Met and L-[<sup>35</sup>S]Cys. Cell lysates were run on SDS 
PAGE and fluorgraphed:  a 48 kDa protein was revealed in the hIR PTK-recombinant 
virus-infected cells but not in the mock- or wild type-infected cells.  Similar 
results were found with other viral recombinants containing the PTK domain.
<p>
The labeled lysates were also incubated with three MAbs specific to the IR 
cytoplasmic domain.  The viral recombinant lysates were found to precipitate a 
band of ~48 kDa not found in control mouse IgG immunoprecipitates.  It is known 
that all three MAbs require the native conformation of the protein for epitope 
recognition and that each binds distinct epitopes of the domain, suggesting that 
the 48 kDa protein represents the entire length of the domain.  The solubility 
(subcellular localization) of the ~48 kDa protein in the Sf9 cells was examined 
for this fragment of a normally membrane protein:  cells at 
48&nbsp;h&nbsp;postinfection and swollen in a hypotonic buffer (10 mM Tris HCl, 
pH 7.3, 10 mM NaCl, 1.5 mM MgCl<sub>2</sub>) for 40 min at 4&deg; were 
homogenized (Teflon-glass) and the ~48 kDa protein was found to distribute in 
both the particulate and soluble fractions of the lysate after 
ultracentrifugation (100,000 <i>g</i>), although the preference was for the 
soluble fraction.
<p>
Sf9 cells labeled for 3&nbsp;h&nbsp;with <sup>32</sup>P<sub>i</sub> for 
48&nbsp;h&nbsp;were lysed and fractionated by centrifugation, followed by 
analysis with SDS PAGE and autoradiography.  A phosphoprotein at ~48 kDa 
represents the major labeled soluble protein in infected cells not present in 
the mock- or wild type-infected cells.  This 48 kDa protein was 
immunoprecipitated with all 3 MAbs and not by control IgG.  One of the MAbs, 
17A3, did recognize the [<sup>35</sup>S]Met- or [<sup>35</sup>S]Cys-labeled 
protein but did recognize this autophosphorylated form.  This result is 
consistent with reports that MAb 17A3 blocks autophosphorylation.  This result 
shows that there is a mixture of auto- and unphosphorylated forms of PTK. 
Phosphoamino acid analysis of the band indicates that both serine and tyrosine 
are phosphorylated in these proteins.  The nature of the endogenous serine 
kinase is unknown.  Tyrosine phosphorylation presumably is by 
autophosphorylation.
<p>
The protein kinase activity of the expressed recombinant was determined by 
harvesting Sf9 lysates 48&nbsp;h&nbsp;postinfection and immunoprecipitating with 
MAb 2G7.  Immune complexes were then incubated with labeled ATP and divalent 
cations and a single ~48 kDa band was found to be labeled upon SDS 
PAGE/autoradiography. This was not visible in control immune complexes.  The 
excised band was found to have only P-Tyr upon PAA analysis.  Incubation of 
exogenous substrates poly(Glu-Tyr) and histone H2B indicated that <sup>32</sup>P 
was incorporated.  [The stoichiometry of the assays is not really mentioned.] 
The estimated protein content of the infected cells is ~5% specific to total 
cell protein, with an overall yield of ~50-200 &micro;g/10 million cells.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Aizawa H, Kamijo M, Ohba Y, Mori A, Okuhara K, Kawasaki H, Murofushi H,
Suzuki K, Yasuda H
 </span>
 <span class="year">
(1991)
 </span>
 <span class="jname">
Biochem. Biophys. Res. Comm.
 </span>
 <span class="jvol">
179,
 </span>
 <span class="pages">
1620-1626.
 </span>
<p class="reftitl">
Microtubule Destabilization by <i>cdc2</i>/H1 Histone Kinase:
Phosphorylation of a "Pro-rich Region" in the Microtubule-Binding
Domain of MAP-4
<p class="Para1">
Bovine MAP-4 was added to <i>Xenopus</i> egg crude extract which had been 
activated to proceed to the M phase.  Phosphorylation of MAP was 
phase-dependent:  phosphorylation about 2-fold greater was found for H1, MAP-4, 
and the P fragment relative to their phosphorylation by extracts from interphase 
eggs.  (P fragment is a part of the MAP-4 sequence that is Pro-rich.)  The 
strong suggestion is that MAP-4 is phosphorylated by cell cycle-dependent H1 
kinase, and would be consistent with a report that MAP-4 is phosphorylated 
during M phase in CHO cells.
<p>
Histone H1 kinase purified from FM3A cells was incubated with MAP-4. 
Phosphorylation had a <i>K</i><sub>m</sub> of 94 nM, indicating a very strong 
interaction, even stronger than for histone H1 itself (<i>K</i><sub>m</sub> = 
1900 nM).  Prolonged incubation showed 0.8 mol P per mol MAP-4 with the kinase. 
A microtubule assembly assay was performed, and the results show that 
phosphorylated MAP-4 was inhibited in its promotion of assembly.  (The assembly 
assay involves mixing MAP-4 with tubulin and GTP in appropriate buffer, warming 
it, and following turbidity at 350 nm.)  It seems that phosphorylation of MAP-4 
by H1 kinase may be the cause of microtubule disruption occurring at the end of 
the G<sub>2</sub> phase.
<p>
The region phosphorylated was identified using fragments made in bacteria. The 
fragment studies revealed that MAP-4 was phosphorylated in a Pro-rich region. 
Fragments were treated with <sup>32</sup>P-labeled ATP and kinase and then run 
on SDS gels, followed by autoradiograms (controls omitted the kinase).  The P 
fragment was treated with lysylendopeptidase (LEP) and the peptide with the 
highest level of incorporated radiolabeled phopshate sequenced on a gas-phase 
sequencer for several cycles. This peptide was MAP-4 757-779 (<b>APATATLA<span 
style="text-decoration:underline;">S</span>P</b>GSTSRNLS<b>T</b>PLPK); a serine 
and threonine with carboxyl proline are found in this fragment, shown 
underlined. The Lys-779 end of the fragment is inferred.
<p>
Crude extracts of <i>Xenopus</i> eggs will mimic cell cycle events such as DNA 
replication, chromosome condensation, nuclear envelope breakdown, and 
dissolution of the microtubular network in the interphase. It also involves and 
oscillating histone H1 kinase activity after in vitro activation.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Lyznik LA, Peng J, Hodges TK
 </span>
 <span class="year">
(1991)
 </span>
 <span class="jname">
Biotechniques
 </span>
 <span class="jvol">
10,
 </span>
 <span class="pages">
294-300.
 </span>
<p class="reftitl">
Simplified procedure for transient transformation of plant
protoplasts using polyethylene glycol treatment
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Cahill, A. L., Perlman, R. L. 
 </span>
 <span class="year">
(1991)
 </span>
 <span class="jname">
J. Neurochem.
 </span>
 <span class="jvol">
57,
 </span>
 <span class="pages">
1832-1839.
 </span>
<p class="reftitl">
Activation of a microtubule-associated protein-2 kinase by insulin-like growth 
factor-I in bovine chromaffin cells
<p class="Para1">
Bovine chromaffin cells were isolated by collagenase digestion and treated with 
IGF-I for 10 min at 37&deg;.  Treatment of cells was either in EBSS, EBSS 
containing 0.1 mM Na<sub>3</sub>VO<sub>4</sub>, 10 nM IGF-I, or IGF-I + 
Na<sub>3</sub>VO<sub>4</sub>.  The cells were extracted and the crude homogenate 
used in an incubation of MAP-2 and hot ATP.  MAP-2 was analyzed for 
phosphorylation on 6% SDS gels and the band was visualized and then Cerenkov 
counted for incorporation.  The results indicate that when the vanadate 
inhibitor was present in the IGF-treated sample, the effects of the kinase are 
potentiated.  The inhibitor added to untreated cells causes only a small 
increase in phosphorylation.  With IGF-I alone, the phosphorylation was greater 
than with the vanadate inhibitor alone.  The control (untreated and no vanadate 
inhibitor) had the least phosphorylation of all.
<p>
A time course of MAP-2 kinase activity with 10 nM IGF-I treatment indicates 
maximal activation is achieved between 10-15 min.  The 30 min time point does 
indicate a decline and that the activation is not stable.  Note that the time 
course is done in the presence of 0.1 mM Na<sub>3</sub>VO<sub>4</sub>.  An 
increase in activation is observed even after 2 min.  The concentration 
dependence of MAP-2 kinase activity on [IGF-I] shows that 10 nM IGF-I is the 
dose necessary to achieve maximal activation, with higher concentrations having 
no effect.  This curve also done in presence of vanadate inhibitor.
<p>
Phosphotyrosine-containing proteins were isolated by passing extracts of 
IGF-I-treated cells over a BSA-Sepharose to remove proteins tending to bind 
nonspecifically and then over a 1G2 monoclonal antibody affinity column 
generated from activated Sepharose 4B.  This monoclonal was grown in the ascites 
of BALB/c mice, 50% ammonium sulfate precipitated and purified over a 
phosphotyrosine-Sepharose 4B column.  Eluates from 1G2-Sepharose beads (elution 
is with phenyl phosphate) were found to have MAP-2 kinase activity in both 
control and IGF-I-treated cells.  The activity was increased with both IGF-I and 
Na<sub>3</sub>VO<sub>4</sub>.  Extract loads spiked with phenyl phosphate did 
not elute a MAP-2 kinase activity from the beads.  An SDS gel on MAP-2 incubated 
in presence of labeled ATP and extracts shows MAP-2 to incorporate the 
phosphate.
<p>
The pattern of phosphotyrosine proteins was examined on SDS gels in which cells 
were prelabeled with <sup>32</sup>P<sub>i</sub>, stimulated with IGF-I with 
vanadate present, adsorbed to 1G2-Sepharose beads, and then specific eluates. 
The unstimulated cells showed prominent PTyr protein bands at 116, 92, and 50 
kDa.  IGF-I enhanced phosphorylation into all of these bands and caused the 
appearance of a 42 kDa protein.  The 92 kDa band may be the ï¿½-subunit of IGF-I 
receptor.
<p>
The divalent cation specificity of the MAP-2 kinase activity in extracts and 
1G2-Sepharose eluates was examined and 10 mM Mg<sup>2+</sup> was necessary and 
sufficient for activity. 2 mM Mn<sup>2+</sup> does not support MAP-2 kinase 
activity and appears to be inhibitory since activity was not increased/normally 
present in unstimulated cells with 10 mM Mg<sup>2+</sup> present as well.  This 
was assumed since the activation was much greater/more apparent with IGF-I 
treatment.
<p>
The MAP-2 kinase was distinguished from other kinases in that the assays of 
crude extracts carried out in the presence of EDTA (absence of Ca<sup>2+</sup>) 
and found to be active, which necessarily eliminates protein kinase C or 
Ca<sup>2+</sup>/calmodulin as being involved in the phosphorylation.  Addition 
of the heat-stable inhibitor of PK-A did not affect activity.  The 
insulin-stimulated MAP-2 kinase of 3T3 fibroblasts is inhibited by 80 mM NaF; so 
is the IGF-I stimulated kinase in crude extracts or 1G2-Sepharose eluates.  The 
MAP-2 was analyzed for phosphoamino acids and shown to contain PSer, some PThr, 
but no PTyr.  The MAP-2 is not therefore directly phosphorylated by receptor.
<p>
Alternative substrate analysis using Lys-rich histones, mixed histones, and 
ribosomal S6 protein indicate that these were all poor substrates for IGF-I 
activated protein kinase.  MBP was a good substrate however.  The kinase 
activity for MAP-2 and MBP co-chromatographed on DEAE-cellulose.  The activity 
was nearly all in the cytosol, with 10% found in the particulate fraction.
<p>
Commercial alkaline phosphatase was used to pre-treat samples of kinase from 
crude extracts and 1G2-Sepharose eluates of control and stimulated cells.  These 
were in some cases prepared in buffers lacking vanadate.  MBP activity was 
reduced 75% in crude extracts from control and stimulated cells treated with 
APase, and the effect of APase was blocked by inhibitors used:  vanadate, EDTA, 
Na<sub>4</sub>P<sub>2</sub>O<sub>7</sub>, and phenyl phosphate.  This indicates 
that dephosphorylation by phosphatases is the cause of loss of activation (and 
not to proteolysis or other undefinable activation mechanism).  The same was 
true for APase and inhibitor effects in 1G2-Sepharose eluates.
<p>
Other agonists working on chromaffin cells were tested.  Insulin also increased 
MAP-2 kinase activity but was much less effective at 10 nM than IGF-I.  EGF, 
phorbol esters (PDBu), and nicotinic agonists (DMPP) increase the PTyr content 
of the pp42<sup><i>mapk</i></sup> (see Ely et al., 1990), and in this case 
increased MAP-2 kinase activity.  FGF was reported to increase MAP-2 kinase 
activity in PC12 cells and did so in chromaffin cells.
<!-- ============================================================= -->
<p class="tags">
HBsAg preS domain expression
<p class="ref">
 <span class="authors">
Delos S, Villar MT, Hu P, Peterson DL
 </span>
 <span class="year">
(1991)
 </span>
 <span class="jname">
Biochem. J.
 </span>
 <span class="jvol">
276,
 </span>
 <span class="pages">
411-416
 </span>
<p class="reftitl">
Cloning, expression, isolation and characterization of the pre-S domains of 
hepatitis B surface antigen, devoid of the S protein
<p class="Para1">
HBsAg is synthesize as nested set of three protein.  The S (short form) is
226 amino acids and the two addtional proteins are N-terminal extensions,
M (middle-length) is 55 amino acids (pre-S2) and L (large form) has 109 or 118
additional amino acids depending on antigen subtype (pre-S1).  Only the S
protein contains regions closely associated with lipid and the pre-S regions
are hydrophilic, entirely outside the particle.  HBsAg particle structure is
not affected by absence of pre-S domains, and do the pre-S and S domains are
likely independent.  Pre-S1 and -S2 contain glycosylation sites.  What is known
is that L protein N-terminus is myristylated.  Pre-S domain function include
antigenic sites reactive to B and T lymphocytes.   These sequences bind to 
hepatocytes as well as to polymerized human serum albumin, and so preS domains
may be involved in infections.
<!-- ============================================================= -->
<p class="tags">
<p class="ref">
 <span class="authors">
Roy, L. M., Swenson, K. I., Walker, D. H., Gabrielli, B.
G., Li, R.-S., Piwnica-Worms, H., Maller, J. L. 
 </span>
 <span class="year">
(1991)
 </span>
 <span class="jname">
J.
Cell Biol.
 </span>
 <span class="jvol">
113,
 </span>
 <span class="pages">
507-514.
 </span>

<p class="reftitl">
Activation of p34/<i>cdc2</i> kinase by cyclin
A
<p class="Para1">
<u>Cyclins stimulate meiosis I and II in absence of
protein synthesis.</u>  Active cyclins A and
B were overexpressed in a baculovirus expression system using Sf9 host
cells:  immunoblots with anti-cyclin A or B
verified expression with clam egg cyclin as a positive control.
<p>
Cyclin A was microinjected into oocytes in presence
of cycloheximide (CHI) and germinal vesicle breakdown (GVBD) was induced
in 1-2 hr with or without CHI;  this very
rapid process indicates entry into meiosis I.
CHI is added to inhibit kinase levels to below that functional for GVBD, but
cyclin A is sufficient for this process.
<p>
The concentration dependence of cyclins A and B
microinjected into CHI-treated oocytes for a 2-hr period, after which GVBD
was determined, showed that both cyclins induced GVBD, but cyclin B was
only 20% as potent, with 50 nM as half-maximal for A and 250 nM for B.
<p>
Progesterone-induced oocyte maturation has shown
that MPF activity as measured by oocyte injection <u>transiently</u>
declines 50% 1-2 hr after GVBD in conjunction with metaphase/anaphase
transition of meiosis I, returning to high levels in metaphase II-arrested
eggs.  With CHI injection, the MPF activity
does not rise, and declines if injection is after GVBD (Gerhart et al.,
1984).  This is now correlated with a change
in H1 kinase activity of p34/<i>cdc2</i> instead of GVBD by
MPF.
<p>
Cyclin A or B lysates were microinjected into
oocytes which had been previously injected with CHI immediately after
undergoing progesterone-induced GVBD.  Both
cyclins immediately increased the level of H1 kinase activity in the
presence of CHI, reaching the same activity level as in metaphase
II-arrested oocytes, but unlike controls which exhibit as stable activity,
the level rapidly declining.  Mos<sup>
xe</sup>, which has a very short half-life and is
essential for metaphase II arrest of the egg, has a kinase activity which
is inhibited by protein synthesis inhibition.
It probably phosphorylates cyclin B2, a subunit of
pre-MPF, and it is possible that meiosis II arrest can be overcome by
addition of the cyclins.  This rescue of
CHI-inhibited cell cycle occurred at concentrations of cyclin similar to
those that cause GVBD and meiosis I.
<p class="Para1">
<u>Cyclin A activation of p34</u><u><sup>
cdc2</sup> for mitotic cell cycles.</u>
Baculovirus-produced clam cyclin A protein was added to
RNase-treated Murray oocyte extracts in a sensitive and quantitative assay
for the activation of <i>cdc2</i> kinase.
(Murray extracts are the Murray and Kirschner
modifications to the extraction method of Lohka et al., in which mitotic
extracts oscillate spontaneously between DNA replication and mitosis as
opposed to Lohka extracts which require exogenous addition of components
to enable cell cycle transition.)  In
extracts receiving no cyclin A, two consecutive cycles of H1 kinase
activity were observed when no RNase was added, with no significant
cycling of activity in RNase-treated extracts.
These oscillations are verified by direct kinase
activity of extracts or after p13<sup>
suc1</sup>-Sepharose immobilization.
<p>
When as little as 25 nM cyclin A is added to RNased extract, one cycle of p34<i> 
</i> activation is restored, after a 10 min lag, to a level of a control cycle 
at metaphase. Increasing amounts of cyclin A did not change the lag time but did 
elevate the maximal activity, and initial rates of activation were higher. At 
&ge;250 nM, p34 activity became "hyperactivated"--8-fold at the metaphase point 
(45 min) in the cycle--maintaining activity for several hours and behaving 
essentially like a CSF-arrested extract. Nuclear morphology could also be used 
to show that kinase activity could consistently correlate with cell cycle stage. 
A linear response over a 10-fold range between added cyclin A and maximal p34<i>
</i> kinase activation suggests a direct interaction between the two proteins. 
We also see that <sup>35</sup>S-labelled cyclin A from Sf9 cells added to <i>
cdc2</i> kinase on p13<sup> suc1</sup>-Sepharose beads.
<p>
Addition of unlabeled cyclin to Murray extracts containing <sup>35</sup>S-Met 
caused a delay in the degradation of both the A- and B-type cylins, supporting 
the idea that cyclin A levels set the rate of exit from M phase without normally 
controlling the rate of entry into M phase. Also observed was a decline in the 
rate of <i>cdc2</i> kinase deactivation with progressively higher cyclin 
concentrations.
<p>
Emetine was added to oscillating Murray extracts after completion of one cycle 
resulting in the no further activation of <i>cdc2</i> kinase. Addition of cyclin 
A however restored activation to the metaphase levels. Note that control studies 
using immobilization on p13<sup>suc1</sup>-Sepharose show degradation of all 
newly synthesized cyclins at exit from M phase. From Murray and Kirschner's work 
with cyclin B and from our work with cyclin A, extracts with no protein 
synthetic ability can use cyclin A or B alone to stimulate high levels of <i>
cdc2</i> kinase, although in the Murrary extracts, eggs are activated 30 min 
before crushing, allowing possibly some new synthesis of cyclin or other protein. 
Cytostatic factor (CSF) extracts which had been emetine-treated and which had 
completed meiosis II received cyclin A and this caused a marked and stable 
hyperactivation of the <i>cdc2</i> kinase. Ca<sup> 2+</sup>-treated CSF extracts 
arrest again at the first mitotic interphase (typical after fertilized cells 
begin to divide), but a second addition of calcium to those arrested cells also 
causes release from arrest. Cyclin A was able to activate <i>cdc2</i> kinase 
after a second M phase in vitro, showing it requires no other protein to 
activate the kinase after exit from either meiosis or mitosis.
<!-- ============================================================= -->
<p class="tags">
<p class="ref">
 <span class="authors">
Marx, J. 
 </span>
 <span class="year">
(1991)
 </span>
 <span class="jname">
Science
 </span>
 <span class="jvol">
252,
 </span>
 <span class="pages">
1490-1492.
 </span>

<p class="reftitl">
The cell cycle:  spinning
farther afield
<p class="Para1">
Reviews the current knowledge of cell cycle
regulation.  The <i>cdc2</i>
kinase was originally thought to control start of mitosis along with its
associated mitotic cyclins.  In budding
yeast, G1 cyclins are shown to control a "Start" point for the
kinase.  For yeast, certain growth factors
are required to get the cell to the point past Start, and the cell can
continue without these factors through mitosis back to G1.
Two groups (Nasmyth at Res. Inst. of Molec. Path. in
Vienna along with Futcher at Cold Spring Harbor and S. Reed and
collaborators at Res. Inst. of Scripps Clinic in La Jolla) reported a new
cyclin from budding yeast which appeared in G1.
Other groups are now reporting the mammalian equivalent
of yeast Start cyclins (Beach, Sherr, Arnold).
The Beach group had looked for their cyclins screening
for human genes which could rescue Start activity in yeast, while the
other groups found theirs working with oncogenes and growth factors.
Arnold's group found the site of a defect causing
benign growth of a parathyroid tumor:  the
cloned and sequenced gene bears familial resemblence to cyclins.
It was the same as Beach's cyclin.
<!-- ============================================================= -->
<p class="tags">
<p class="ref">
 <span class="authors">
Tyson, J. J. 
 </span>
 <span class="year">
(1991)
 </span>
 <span class="jname">
Proc. Natl. Acad. Sci.
USA
 </span>
 <span class="jvol">
88,
 </span>
 <span class="pages">
7328-7332.
 </span>

<p class="reftitl">
Modeling the cell division cycle - <i>cdc2</i> and cyclin interactions
<!-- ============================================================= -->
<p class="tags">
<p class="ref">
 <span class="authors">
Ferris, D. K.; White, G. A.; Kelvin, D. J.; Copeland T.
D.; and others. 
 </span>
 <span class="year">
(1991)
 </span>
 <span class="jname">
Cell Growth &amp; Differentiation
 </span>
 <span class="jvol">
2,
 </span>
 <span class="pages">
343-349.
 </span>

<p class="reftitl">
p34<i>cdc2</i>-specific
<p class="reftitl">
tyrosine kinase
<!-- ============================================================= -->
<p class="tags">
<p class="ref">
 <span class="authors">
Crissman, H. A., Gadbois, D. M., Tobey, R. A., and
Bradbury, E. M. 
 </span>
 <span class="year">
(1991)
 </span>
 <span class="jname">
Proc. Natl. Acad. Sci. USA
 </span>
 <span class="jvol">
88,
 </span>
 <span class="pages">
7580-7584.
 </span>

<p class="reftitl">
Transformed mammalian cells are deficient in
kinase-mediated control of progression through the G<sub>
1</sub> phase of the cell cycle
<p class="Para1">
Cells which are early-passage, nonimmortalized and
nontransformed were treated with staurosporine at various
concentrations.  Flow cytometry was employed
to analyze DNA (with mithramycin staining at 457 nm) and cells in cycles
as opposed to cells not in cycles (using the 5-Br-deoxyuridine
technique).  Two types of responses were
observed in the normal cell types (Nude mouse lung, CHO, HFL-1 human lung,
and HSF-43 human dermal cells).  For
concentrations of staurosporine in the range 1-10 ng/ml (2.2-22 nM) for
periods about the culture doubling time, cells in S, G<sub>
2</sub>, and M phases dropped in population, causing an
accumulation of cells with a 2C DNA content in G<sub>
0</sub>/G<sub>1</sub>.
Cultures were maintained at &lt;3 x 10<sup>
6</sup>/75 cm<sup>2</sup> flask and
arrest/inhibition of growth is unlikely to be due to confluent
cultures.  Continued exposure resulted in no
further changes in DNA distribution.  When
staurosporine was increased to &gt;=50 ng/ml staurosporine, accumulation
was in both G<sub>0</sub>/G<sub>
1</sub> and G<sub>2</sub> and there
was no increase in cell number.  Microscopic
examination showed that cells arrested in late interphase were nonmitotic.
<p>
In transformed cells (FT210 mouse mammary
carcinoma, Chinese hamster WCHE/5 tumor, HL-60 human leukemia, and
CT10-2C-T1 transformed derivatives of HSF-43), staurosporine at 10 ng/ml
did not affect DNA distribution nor did it arrest at G<sub>
0</sub>/G<sub>1</sub>, with cell
number increases the same as controls.  At
&gt;=50 ng/ml, the cells arrested but only at G<sub>
2</sub>.  No growth conditions
attempted could arrest cells at G<sub>
1</sub>.  From all of the these
results we conclude:
<p>
(i) staurosporine-sensitive kinase-mediated
controls are essential for progression through both G<sub>
1</sub> and G<sub>2</sub>/M phases
in nontransformed cells.
<p>
(ii) staurosporine-sensitive kinase-mediated
controls necessary for G<sub>1</sub> progression
in normal cells are lost in transformed cells.
<p>
Serum-deprived HSF-43 cells in G<sub>
0</sub> were placed in complete medium to restore
cycling.  The kinetics of restored cycling
were then compared to cells which had been exposed to low concentrations
of staurosporine.  Cells recovering from the
drug began DNA synthesis 9 h after drug removal, whereas 12 h elapsed
before the initially G<sub>0</sub> cells began to
replicate their DNA.
<!-- ============================================================= -->
<p class="tags">
<p class="ref">
 <span class="authors">
Dunphy, W. G., Kumagai, A. 
 </span>
 <span class="year">
(1991)
 </span>
 <span class="jname">
Cell
 </span>
 <span class="jvol">
67,
 </span>
 <span class="pages">
189-196.
 </span>

<p class="reftitl">
The cdc25 protein contains an intrinsic phosphatase
activity
<p class="Para1">
It was previously reported that an E coli strain which
overexpresses biologically active cdc25 had been created.
A fragment from the conserved COOH-terminal domain of
Drosophila cdc25 (p35<sup>cdc25</sup>)
efficiently induced tyrosine dephosphorylation and activation of cdc2
protein kinase in Xenopus oocyte extracts. 
It also dephosphorylated cdc2 bound to p13-agarose beads.
<p>
In this report we show that purified cdc25
dephosphorylates certain model substrates. 
p-Nitrophenyl phosphate hydrolysis is the first substrate tested, and the
A<sub>410</sub> is shown to increase
proportionally to the protein added. 
Addition of recombinant sea urchin cyclin as a control was
ineffective.  Purified recombinant
p13<sup>suc1</sup> was also added by had no
effect (at up to 1 mg/ml).  Gel filtration of
enzyme on Sephacryl S-200 eluted produced activity that eluted in the
middle of the column;  since it was the only
protein, the peak of protein also coincided with the activity
observed.  This peak off gel filtration
successfully adsorbed to an immunoaffinity column in which the antibody
ligand had been directed against an epitope known to exist on cdc25.
Adsorption of other proteins did not occur, and
adsorption of activity was blocked when the epitope sequence itself was
included in the column buffer.
<!-- ============================================================= -->
<p class="tags">
<p class="ref">
 <span class="authors">
Nishitani, H., Ohtsubo, M., Yamashita, K., Iida, H.,
Pines, J., Yasudo, H., Shibata, Y., Hunter, T., Nishimoto, T. 
 </span>
 <span class="year">
(1991)
 </span>
 <span class="jname">
EMBO J.
 </span>
 <span class="jvol">
10,
 </span>
 <span class="pages">
1555-1564.
 </span>

<p class="reftitl">
Loss of RCC1, a nuclear DNA-binding protein, uncouples
the completion of DNA replication from the activation of cdc2 protein
kinase and mitosis
<p class="Para1">
INTRO.  DNA oscillates
between condensed and decondensed state in cell cycle.
Prescott (1976) showed that postmitotic decondensed
chromosomes express mRNAs required to initiate G<sub>
1</sub> phase of the cell cycle.
MPF is necessary in the initiation of this
process.  In G<sub>
1</sub> there is very little cyclin B, p34<sup>
cdc2</sup> is not phosphorylated, and there is no MPF
activity.  At the end of G<sub>
1</sub>, phospho-p34<sup>cdc2</sup>
is apparent and its histone H1 kinase activity is high at the end of
G<sub>2</sub>.  With
increase in H1 histone kinase activity, the p34<sup>
cdc2</sup> is dephosphorylated, cyclin B is degraded, and the
kinase activity falls.
<p>
Mitosis should only begin (with chromosomal
condensation) after DNA synthesis is completed.
Drugs and mutations have been found which cause
premature initiation of mitosis before replication is complete.
Caffeine has been shown to induce condensation in
BHK21 cells and in <i>Xenopus</i> egg cell-free systems with
DNA replication blocked by hydroxyurea or aphidicolin.
Mutants in <i>wee</i> mutants in
fission yeast and tsBN2 cells have also shown uncoupling of replication
and mitosis.  Except for tsBN2 cells, the
other mutants intiate mitosis prematurely at the restrictive temp only
when DNA synthesis is prevented.  In the
tsBN2 line however, mitosis can occur with use of a replication
inhibitor.  The tsBN2 mutant was isolated as
a DNA<sup>-</sup> mutant from
nitrosoguanidine-treated BHK21 cells by FdUR selection.
Other biochemical features of premature condensation
have already been shown:  appearance of MPM-2
antigens and phosphorylation of histone H1 and H3.
<p>
Complementation of the wild-type with the tsBN2
cells has permitted the cloning of the <i>RCC1</i> gene
responsible for the mutation.  A protein of
45 kDa is encoded, has DNA binding activity and is located in the nucleus.
<p>
RESULTS.  Cultures of
tsBN2 cells are synchronized to the G<sub>
1</sub>/S boundary with hydroxyurea.
Cells grown at the restrictive 39.5&deg; temperature after
release from block and after 2 h "rounded-up" cells with condensed
chromosomes appear.  After 4 h these cells
disappeared and cells with micronuclei and flat morphology appear.
Anti-beta-tubulin staining of these cells
demonstrates the appearance of a normal spindle in these cells.
As the spindle was bipolar, centrosomes are
assumed to have duplicated normally despite DNA replication block and to
have assumed their proper positions at mitotic onset.
After 6 h, interphase MT arrays appear;
micronuclei show the appearance of a membrane, pores
and lamina, as shown with anti-lamin staining.
<p>
<i>RCC1</i> gene structure indicates a
single missense mutation, and therefore at the restrictive temperature,
the protein's conformation must be profoundly altered.
Anti-Xenopus RCC1 has been previously prepared and
recognizes homologs in frogs, hamster, and human cells.
Exponentially growing tsBN2 was shifted to the
restrictive 39.5&deg; and cells were prepared for either immunofluorescence or
immunoblotting.  RCC1 was not detectable by
either.  Both the 45 and 46 kDa forms of RCC1
became undetectable at the restrictive temp after 1 h, whereas in the
wild-type BHK21 cells these bands consistently appeared even after 8 h
(the qualitative pattern of total protein did not vary that much according
to stained gels).  When exponentially cells
were incubated with [<sup>35</sup>S]Met for 3 h
at 33.5&deg;, then incubated in chased medium at 33.5&deg; and 39.5&deg; and harvested
over a time course, RCC1 expression (as measured by immunoprecipitation of
RCC1 with anti-RCC1 antibody) did not change (fall) in the wild-type BHK21
cells over the next 3 h period.  On the other
hand, the 45 and 46 kDa forms of RCC1 in tsBN2 fell in the 1st h
significantly (according to the autorads) and were undetectable after 3
h.  There was no change in RCC1 expression in
either cell lines at the permissive temperature (33.5&deg;).
Restoring tsBN2 cells to 33.5&deg; after disappearance of
RCC1 at 39.5&deg; did not permit RCC1 to reappear if cycloheximide were
present.  Therefore RCC1 is degraded and not
merely inactivated.
<p>
Cultures of tsBN2 were synchronized in G<sub>
1</sub> with isoleucine starvation, released from
block, then arrested at G<sub>1</sub>/S with
hydroxyurea.  In the presence of inhibitor,
cells were shifted to 39.5&deg; and lysed at 1 h intervals for RCC1 expression
and p34<sup>cdc2</sup> kinase activity
analyses.  After 1 h at the higher temp, RCC1
had decreased by 80% while histone H1 kinase activity of p34<sup>
cdc2</sup> had increased. 
SDS gel analysis revealed that the faster migrating dephosphorylated band
of p34 was beginning to appear and was the only form present 3 h after
shift to the nonpermissive temp.  Histone H1
kinase activity was also maximal.  Monitoring
of cyclin B expression with immunoprecipitates using anti-cyclin B serum
also verified these correlative changes.
<p>
In the presence or absence of cycloheximide, RCC1
degrades rapidly when the tsBN2 cells are shifted to 39.5&deg;.
The cdc2/H1 kinase activity does not increase nor does
p34<sup>cdc2</sup> become dephosphorylated during
this shift, indicating a dependence upon protein synthesis for this
process.  With use of anti-cyclin B,
phosphorylated p34<sup>cdc2</sup>
co-immunoprecipitates with cyclin B. 
Anti-cdc2 antibody demonstrates the presence of cyclin B as well;
suggesting that cyclin B is already complexed in tsBN2 cells at the
G<sub>1</sub>/S boundary.
This cyclin B amount did not change as the histone H1
kinase activity increased.  Therefore
synthesis of other proteins other than cyclin B is necessary for
p34<sup>cdc2</sup> kinase activation upon loss of
RCC1.
<h1 id="Cyclins">Cyclins</h1>
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Pearson, M., Bjornson, R., Pearson, G., Rohrmann, G.
 </span>
 <span class="year">
(1992)
 </span>
 <span class="jname">
Science
 </span>
 <span class="jvol">
257,
 </span>
 <span class="pages">
1382-1384.
 </span>
<p class="reftitl">
The <i>Autographa californica</i> baculovirus genome:  evidence for
multiple replication origins
<p class="Para1">
The AcMNPV genome has a series of six homologous regions (<i>hr</i>1
through <i>hr</i>5, with <i>hr</i>4 being subdivided into an <i>hr</i>4L
and <i>hr</i>4R separated by 4 kb).  The five major <i>hr</i> regions
vary in size from 0.4 to 1.0 kb and are separated by 25, 42, 19, 15,
and 15 kb, resp.  The viral genome is 128 kb.  Each <i>hr</i> region
contains from 2-8 (for a total of 33) of 30 bp imperfect palindromes
with naturally occurring <i>Eco</i>RI sites in the core.  Besides the regular
structure of the palindromes having 10- to 12-bp core sequence with
two areas of mismatch at 4 or 5 bp and 10 or 11 bp from each <i>Eco</i>RI
site, being separated by about 83 +/- 34 bp, these homologous regions
serve as transcriptional enhancers.
<p>
To test the possibility that these <i>hr</i> regions served as origins
of replication, the <i>hr</i>5 region (6 palindromes distributed over
600 bp) was placed in a plasmid and transfected into AcMNPV-infected
Sf9 cells.  DNA extracted from infected cells was digested with
<i>Dpn</i>I endonuclease, which cleaves GmeATC recognition (it
can only cut the methylated site).  Because <i>Dpn</i>I sites in
bacterial plasmids are methylated, they are cut, while eukaryotic
DNA, being unmethylated, is not cut, allowing differentiation of
replicated plasmid and input plasmid.  It was found that plasmid
containing <i>hr</i>5 was amplified and became <i>Dpn</i>I-resistant,
showing it is replicated.  The vector by itself (pBluescript) is not
replicated under these conditions.
<p>
Deletion mutants were constructed so that resulting plasmids had
6, 4, 2, 1, and then an incomplete palindrome from <i>hr</i>5.
After transfection into coinfected Sf9 cells, DNA was recovered
after 65&nbsp;h&nbsp;and digested with <i>Dpn</i>I and then <i>Eco</i>RI to linearize
remaining plasmid.  In addition, total DNA was digested with
<i>Eco</i>RI alone for relative determination of plasmid DNA replicated.
Afer electrophoresis and blotting, the vector pBKS- is used as
a probe.  If probe hybridizes to <i>Dpn</i>I-resistant bands of identical
size to <i>Eco</i>RI-digested input plasmid DNA, then replication of the
plasmids is shown.  From the autoradiographs, the relative level of
replication is seen with the loss of each palindrome:  with 6
palindromes, it is the highest, gradually falling off.  The presence
of only 1 palindrome showed quite low levels, while the truncated
palindrome demonstrated no replication.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Lynn, D. E.
 </span>
 <span class="year">
(1992)
 </span>
 <span class="jname">
Biotechniques
 </span>
 <span class="jvol">
13,
 </span>
 <span class="pages">
282-285.
 </span>
<p class="reftitl">
Improved efficiency in determining the titer of the
<i>Autographa californica</i> baculovirus nonoccluded virus
<p class="Para1">
Determination of extracellular virus production for baculovirus
has relied on two methods:  (1) plaque assays and (2) endpoint
titering.  Plaque assays are technically more difficult.  As
for endpoint titering, Leighton tubes, 60-well microplates
and 24- and 96-well TC plates have been used.
<p>
Three cell lines used in the experiments:  TN-368 and IPLB-TN-R
are <i>Trichoplusia ni</i> continuous cell lines grown in
TNM-FH medium and the IPLB-SF21AE cell line from
<i>Spodoptera frugiperda</i> were lepidopteran strains grown
in ExCell 400.  Cells in log growth were diluted in TNM-FH
medium to 10,000/ml, with gentamicin sulfate added to 50 ï¿½g/ml
in the final dilution.  0.2&nbsp;ml&nbsp;(2000 cells) were inoculated
into each well of a 96-well plate (Falcon), with 9 wells of
one row left empty.  Log dilutions were made of the virus up
to 10<sup>-8</sup> with up to 200 ï¿½l per well.  Now for each dilution
from 10<sup>-2</sup> to 10<sup>-8</sup>, 10 ï¿½l was removed from each
diltion and placed in the 12 empty wells for a new row;  that is,
twelve 10 ï¿½l aliquots from one dilution were placed in the 12 wells
of a row.  Plates should be sealed with 1 cm wide masking tape
as they will dry unless placed in a humidified chamber.  Wells
are scored for the presence of polyhedra after 4 days and again
at 1-3 day intervals using an inverted microscope with 10&times; phase
objective.  The 60-well endpoint procedure of Summers and Smith
was used for comparison.
<p>
Results to distinguish the effectiveness of the 60- vs. the
96-well procedure were analyzed using ANOVA and Duncan&rsquo;s
multiple comparison of means test, with the 96-well procedure
indicated as a significantly better measurement (<i>p</i> = 0.05) than
the 60-well test.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Skoufias, D. A., Wilson, L.
 </span>
 <span class="year">
(1992)
 </span>
 <span class="jname">
Biochemistry
 </span>
 <span class="jvol">
31,
 </span>
 <span class="pages">
738-746.
 </span>
<p class="reftitl">
Mechanim of inhibition of microtubule polymerization by
colchicine:  inhibitory potencies of
unliganded colchicine and tubulin-colchicine complexes
<p class="Para1">
Colchicine binding to tubulin appears to inhibit MT formation by
interaction with the polar ends of the tubulin monomer.
Kinetics show this to be a two-step process:  1) rapid formation of a
transient, reversible, low-affinity pre-equilibrium complex followed by
2) one or more slow steps in which conformational changes in tubulin
and in colchicine lead to formation of a very poorly reversible final
state tubulin-colchicine (TC) complex.  Various physical methods
used to characterize observations:  CD spectra of tubulin,
fluorescence changes, changes in tubulin GTPase activity.
<p>
For this study, a bovine brain MT protein preparation consisting of
70% tubulin and 30% MAPs was isolated by 3 cycles of the method
of Asnes and Wilson as modified by Farrell and Wilson without use of
glycerol.  MTs were purified in centrifuged sucrose cushions.
A necessary first experiment involved the addition of unliganded
colchicine (CCh) or TC complex to a stable MAP-rich brain MT
preparation: addition of CCh or TC in increasing amounts
did not significantly change the nature of the MT preparation insofar as
polymer mass--as determined by sedimentation (pellet measured by total protein)
and turbidimetry at 350 nm--mean length--as determined by EM--and direct
observation of MT number--also by EM.
<p>
MT dynamics generally show that tubulin assembles and
disassembles in a steady state flux with a treadmilling mechanism.
Addition is to the A ends and loss at the D ends at a rate of 1.1
tubulin dimers/s.  MTs in reassembly buffer in the steady state
were pulsed with [<sup>3</sup>H]GTP and the TC complex or unliganded
colchicine added.  Plots of [<sup>14</sup>C]GXP-tubulin uptake at
the MT A ends agaist time indicate that TC complex arrest monomer
addition after 10-15&nbsp;min with 0.2 &micro;M TC complex added.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Volontï¿½, C., Nichols, R. A., Greene, L. A.
 </span>
 <span class="year">
(1992)
 </span>
 <span class="jname">
Biotechniques
 </span>
 <span class="jvol">
12,
 </span>
 <span class="pages">
854-863.
 </span>
<p class="reftitl">
Rapid measurement of protein kinase and phosphatase activities
by slot-filtration
<p class="Para1">
The nitrocellulose slot-filtration assay has been used to
quantify activity of a novel NGF-activated protein kinase designated protein
kinase N (PK-N): after starting the reaction with labeled ATP, the reaction is
stopped with SDS-containing buffer.
Aliquots are blotted onto NC within a slot-filter apparatus with
unincorporated label removed with washing.
Autorad/densitometry or direct scintillation counting is used to
measure.  Data presented shows very high
specific-to-non-specific ratios, indicating ATP has very little binding to NC.
<p>
To examine methods for reducing nonspecific binding, NC
paper was pre-wetted with ATP-containing solution and this reduces nonspecific
binding, while water has no effect.  For
the washing solutions themselves, SDS-PAGE running buffer, 5% TCA, and PBS did
not differ from water as washing solutions of choice.
A 20 mM Tris (pH 8)/0.9% NaCl/0.05% NP-40 caused detachment of
the substrate (histone HF1 phosphorylated by PK-N and PK-A) from the NC paper.
<p>
A linear response to the slot filter assay from 10 to 2000
ng phosphorylated protein was observed.
When the labeled histone was diluted with a solution of unlabeled
histone substrate, it was found that the slot could handle up to 10 &micro;g of
protein before competition for binding was observed.
<p>
Other substrates including tyrosine hydroxylase, ribosomal
S6 protein, myelin basic protein, and the <i>fos</i>
and <i>jun</i> proteins were tested as
substrates for PK-N and worked well.
Casein is not known to be a substrate and its reaction was near
background as expected.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Benkovic, S. J.
 </span>
 <span class="year">
(1992)
 </span>
 <span class="jname">
Annu. Rev. Biochem.
 </span>
 <span class="jvol">
61,
 </span>
 <span class="pages">
29-54.
 </span>
<p class="reftitl">
Catalytic antibodies
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Fanning, E., Knippers, R.
 </span>
 <span class="year">
(1992)
 </span>
 <span class="jname">
Annu. Rev. Biochem.
 </span>
 <span class="jvol">
61,
 </span>
 <span class="pages">
55-85.
 </span>
<p class="reftitl">
Structure and function of simian virus 40 large tumor antigen
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Law, J. H., Ribeiro, J. M. C., Wells, M. A.
 </span>
 <span class="year">
(1992)
 </span>
 <span class="jname">
Annu. Rev. Biochem.
 </span>
 <span class="jvol">
61,
 </span>
 <span class="pages">
87-111.
 </span>
<p class="reftitl">
Biochemical insights derived from insect diversity
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Blackburn, E. H.
 </span>
 <span class="year">
(1992)
 </span>
 <span class="jname">
Annu. Rev. Biochem.
 </span>
 <span class="jvol">
61,
 </span>
 <span class="pages">
113-129.
 </span>
<p class="reftitl">
Telomerases
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Arnheim, N., Erlich, H.
 </span>
 <span class="year">
(1992)
 </span>
 <span class="jname">
Annu. Rev. Biochem.
 </span>
 <span class="jvol">
61,
 </span>
 <span class="pages">
131-156.
 </span>
<p class="reftitl">
Polymerase chain reaction strategy
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
van den Bosch, H., Schutgens, R. B. H., Wanders, R. J. A., Tager, J. M.
 </span>
 <span class="year">
(1992)
 </span>
 <span class="jname">
Annu. Rev. Biochem.
 </span>
 <span class="jvol">
61,
 </span>
 <span class="pages">
157-197.
 </span>
<p class="reftitl">
Biochemistry of peroxisomes
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Schleif, R.
 </span>
 <span class="year">
(1992)
 </span>
 <span class="jname">
Annu. Rev. Biochem.
 </span>
 <span class="jvol">
61,
 </span>
 <span class="pages">
199-223.
 </span>
<p class="reftitl">
DNA looping
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Majerus, P. W.
 </span>
 <span class="year">
(1992)
 </span>
 <span class="jname">
Annu. Rev. Biochem.
 </span>
 <span class="jvol">
61,
 </span>
 <span class="pages">
225-250.
 </span>
<p class="reftitl">
Inositol phosphate biochemistry
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Lindah, T., Barnes, D. E.
 </span>
 <span class="year">
(1992)
 </span>
 <span class="jname">
Annu. Rev. Biochem.
 </span>
 <span class="jvol">
61,
 </span>
 <span class="pages">
251-281.
 </span>
<p class="reftitl">
Mammalian DNA ligases
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Hiraga, S.
 </span>
 <span class="year">
(1992)
 </span>
 <span class="jname">
Annu. Rev. Biochem.
 </span>
 <span class="jvol">
61,
 </span>
 <span class="pages">
283-306.
 </span>
<p class="reftitl">
Chromosome and plasmid partition in <i>Escherichia coli</i>
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Kornfeld, S.
 </span>
 <span class="year">
(1992)
 </span>
 <span class="jname">
Annu. Rev. Biochem.
 </span>
 <span class="jvol">
61,
 </span>
 <span class="pages">
307-330.
 </span>
<p class="reftitl">
Structure and function of the mannose 6-phosphate/insulin-like growth
factor II receptors
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Pastan, I., Chaudhary, V., FitzGerald, D. J.
 </span>
 <span class="year">
(1992)
 </span>
 <span class="jname">
Annu. Rev. Biochem.
 </span>
 <span class="jvol">
61,
 </span>
 <span class="pages">
331-354.
 </span>
<p class="reftitl">
Recombinant toxins as novel therapeutic agents
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Clarke, S.
 </span>
 <span class="year">
(1992)
 </span>
 <span class="jname">
Annu. Rev. Biochem.
 </span>
 <span class="jvol">
61,
 </span>
 <span class="pages">
355-386.
 </span>
<p class="reftitl">
Protein isoprenylation and methylation at carboxyl-terminal cysteine residues
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Rizo, J., Gierasch, L. M.
 </span>
 <span class="year">
(1992)
 </span>
 <span class="jname">
Annu. Rev. Biochem.
 </span>
 <span class="jvol">
61,
 </span>
 <span class="pages">
387-418.
 </span>
<p class="reftitl">
Constrained peptides: models of bioactive peptides and protein substructures
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Wahle, E., Keller, W.
 </span>
 <span class="year">
(1992)
 </span>
 <span class="jname">
Annu. Rev. Biochem.
 </span>
 <span class="jvol">
61,
 </span>
 <span class="pages">
419-440.
 </span>
<p class="reftitl">
The biochemistry of 3&rsquo;-end cleavage and polyadenylation of
messenger RNA precursors
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Norbury, C., Nurse, P.
 </span>
 <span class="year">
(1992)
 </span>
 <span class="jname">
Annu. Rev. Biochem.
 </span>
 <span class="jvol">
61,
 </span>
 <span class="pages">
441-470.
 </span>
<p class="reftitl">
Animal cell cycles and their control
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Pryer, N. K., Wuestehube, L. J., Schekman, R.
 </span>
 <span class="year">
(1992)
 </span>
 <span class="jname">
Annu. Rev. Biochem.
 </span>
 <span class="jvol">
61,
 </span>
 <span class="pages">
471-516.
 </span>
<p class="reftitl">
Vesicle-mediated protein sorting
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Thompson, J., Donkersloot, J. A.
 </span>
 <span class="year">
(1992)
 </span>
 <span class="jname">
Annu. Rev. Biochem.
 </span>
 <span class="jvol">
61,
 </span>
 <span class="pages">
517-557.
 </span>
<p class="reftitl">
N-(carboxyalkyl)amino acids: occurrence, synthesis, and functions
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Hanson, P. I., Schulman, H.
 </span>
 <span class="year">
(1992)
 </span>
 <span class="jname">
Annu. Rev. Biochem.
 </span>
 <span class="jvol">
61,
 </span>
 <span class="pages">
559-601.
 </span>
<p class="reftitl">
Neuronal Ca<sup>2+</sup>/calmodulin-dependent protein kinases
<p class="Para1">
<b>REVIEW</b>.  Hormonal signaling is often intended to stimulate
2nd messengers: cAMP, DAG, Ca<sup>2+</sup>.  PK-C and PK-A
mediate the action of DAG and cAMP, resp.  Intracellular [Ca<sup>2+</sup>]
generally is at 0.1 &micro;M but will rise to 1 &micro;M or greater.  Many
proteins respond to these increases in Ca<sup>2+</sup>, whereas
few respond to the other messengers.  CaM is one, which
activates enzymes as a Ca-binding protein.
MLCK, phosphorylase kinase, and CaM kinase III (which phosphorylates
EF-2) are all dedicated to the phosphorylation of only one substrate
protein.  Recent studies have identified
a multifunctional CaM kinase (CaM kinase II or Type II CaM kinase) in neuronal
tissues.  It is multifunctional in that
it can have several protein substrates, distinguishing it from CaM kinases
which phosphorylate only a single substrate.
The CaM kinase has been shown to regulate a broad array of physiological
activities:  carbohydrate, lipid, and
amino acid metabolism;  neurotransmitter
release and synthesis; ion channel function and Ca<sup>2+</sup>
homeostasis, cytoskeletal function; and maybe gene expresssion.
The CaM kinase is activated by any pathway increasing intracellular
Ca<sup>2+</sup> (Ca channel proteins, IP<sub>3</sub>).
It autophosphorylates to become Ca<sup>2+</sup>-independent.
Of particular interest is that (1) CaM
kinase is concentrated in neuronal tissues (in particular, the cortex and
hippocampus), and is found in pre- and postsynaptic elements regulating events
related to transmission; (2) its substrates are typically involved with
neuronal signaling and the kinase is involved with long-term potentiation, a
model for long-term memory; (3) brief Ca<sup>2+</sup>
signals activate and stimulate autophosphorylation, the latter of which permits
a continued activated state, a fact which can affect synaptic plasticity.
<p>
Subsequent to finding that cAMP activated a particular
kinase central to its function as a cellular physiological activating source,
the involvement of Ca<sup>2+</sup> as an
cellular signal implicated a central protein kinase which could also function
in mediating its effects.  In 1978, a
multifunctional CaM kinase was described in synaptosomal membranes and also
found in a variety of tissues.  It has a
number of substrates:  myosin light
chains, synapsin I, Trp hydroxylase, tubulin, MAP-2, casein.
Skeletal muscle and liver glycogen synthases
have been shown to be nonneuronal isoforms of CaM MPK.
Some variations in tissue expression include
differences in the size and number of subunits in the holoenzymes.
Rat forebrain CaM kinase has a 3-4:1 subunit
ratio of 50-54 kDa &alpha; subunits and a 58-60 kDa &beta; subunit.
Both subunits are catalytically active and both show autophosphorylation
on SDS-PAGE gels.  cDNA sequencing of several subunits called
&beta;&rsquo;, &gamma;, and &delta; show that they are only isozymes
of the &beta; subunit.
<p>
CaM kinase holoenzyme total MW is between 460,000 and
650,000 (hydrodynamic analysis) and is likely a decamer or dodecamer.
A cerebellar CaM kinase (unlike the cortical) is also composed of &alpha;
and &beta; subunits, but the ratio is the opposite:  1 &alpha;
for 3-4 &beta; subunits.  Both holoenzymes are similar in size,
chromatographic properties, autophosphorylation, and substrate specificity.
<p>
Recombinants of the a subunit have been transfected into
COS-7 and CHO cells and into a baculovirus vector in Sf9 cells.
The recombinant composed only of &alpha; subunit forms a functioning
holoenzyme.  A recombinant &beta;-CaM kinase has also been produced in CHO cells.
The &alpha;-CaM kinase recombinant has also been expressed in <i>E. coli</i>,
but solubility and proteolysis are problems, and the enzyme is apparently
monomeric rather than oligomeric.
<p>
A finding of different a:b subunit association ratios has
led to the belief that holoenzyme assembly does not involve specific numbers of
subunits and is a random assembly of whatever subunits are available in the
ratios that they are expressed.  Moreover, there is evidence that the
number of subunits in a holoenzyme may depend on the particular isozymic
subunit: &alpha; subunits tend to form decamers, while &beta; subunits
arrange themselves in octamers, according to EM and immunoprecipitation
studies in which antibodies specific for &alpha; and &beta; subunits
were used.
<p>
cDNA sequences corresponding to coding regions for 4
distince CaM kinase isoforms have been determined using rat brain
libraries.  The &alpha; subunit is 478 amino acids (M<sub>r</sub>
54,111) on a 5.0 kb message, and the b subunit is 542 aa (M<sub>r</sub>
60,333) on a 4.8 kb message.  91% of the residues between the subunits are
identical in the amino terminal half, 76% identity in the COOH half.
Cloning has been done as well for &alpha; subunit from mouse brain.
Alternative mRNA processing also appears to produce a &beta;&rsquo;
isoform.  Cloned as from rat brain are &gamma; and &delta;
subunits which show 85% identity with the a subunit;
Northern analysis shows a wide tissue distribution for these
isoforms.  The &gamma; subunit is 527 aa (M<sub>r</sub> 59,038) and
the &delta; 533 aa (M<sub>r</sub> 60,080).
Differences between isoforms are generally found in a variable
region at the end of the CaM binding domain.  Identification of
other polypeptides with CaM kinase activity and having
a similar polypeptide size to the cloned isoforms may show that differences
are due to alternatively spliced messages, accounting for variability.
CaM kinases have also been cloned in D. melanogaster (localized mainly
from the fly&rsquo;s head), yeast, and <i>Aspergillus</i>.
<p>
The catalytic domain is in the N-terminal half (residues 1
to 263-274 in the isoforms).  Partial proteolysis can generate
a catalytic active fragment with only this domain.  Homology of
this domain with other CaM regulated kinases is particulary
significant with MLCK and phosphorylase kinase.  The region
between residues 275-314 is regulatory, with an autoinhibitory
and CaM-binding region.  The balance of the protein&rsquo;s
structure is an association/oligermization domain for assembly of the
holoenzyme.  Deletions of these sequences produce a monomeric
Ca<sup>2+</sup>- and CaM-dependent kinase.

<p style="margin:1em auto;">
   <img src="./imagesGeneralScienceHTML/CaMStructure.gif"
   alt="General Structural Domains of CaM Subunit Isozymes">

<p>
In the basal state CaM MPK is inactive because of an autoinhibitory
domain.  With Ca/CaM binding, the kinase
phosphorylates itself and other substrates.
Autophosphorylation slows the dissociation of CaM from the subunit, even
when Ca<sup>2+</sup> levels fall. Even when CaM dissociates, the phosphoCaM
MPK is still Ca<sup>2+</sup>-independent until dephosphorylated.
<p>
In two studies in which casein and MAP-2 were used as substrates,
the rat brain CaM MPK employed a rapid equilibrium random bi-bi
mechanism (ATP, protein are substrates;  ADP, phosphoprotein products).
Using the catalytic fragment and syntide-2 peptide substrate,
another group demonstrated a rapid equilibrium <i>ordered</i>
bi-bi mechanism.
<p>
The purified kinase is inactive in the absence of Ca<sup>2+</sup>
and CaM.  With saturating CaM, Ca<sup>2+</sup> dose-dependence is
in the physiological range for predictable Ca-activable protein:
half maximal activation requires 0.5 to 1 &micro;M.  (CaM has four
Ca<sup>2+</sup>-binding sites.)  With saturating Ca<sup>2+</sup>,
half-maximal activation requires 25-100 nM CaM.  This affinity is lower
than for other CaM-binding proteins (MLCK, calcineurin) which are
activated at [CaM] &le; 1 nM.  This affinity improves with higher ATP
concentrations.  The autophosphorylation and substrate phosphorylation
reactions show Hill coefficients of 1.6, showing
that CaM exerts positive cooperativity with respect to activation.
Full activation requires 1:1 CaM:subunit
stoichiometric binding of CaM, a
finding verified by fluorescence emission anisotropy of CaM binding to
CaM kinase in solution.  Model peptide
studies have shown that CaM binds to a short hydrophobic region (residues
296-309 in the &alpha; subunit form the core,
and the last 5 very hydrophobic are very critical) with a K<sub>d</sub>
of 0.1-0.3 nM, a binding affinity actually greater than that of CaM for
the CaM MPK.  It is therefore thought that CaM access in the native
protein is hindered, and that with autophosphorylation, the protein
conformation is altered to allow CaM full access (explaining why it
remains bound even as Ca<sup>2+</sup> levels fall).
In studies with CaM mutations, mutations in the CaM central helix
known to affect CaM affinity for other CaM-binding proteins appears not to
affect CaM affinity for CaM MPK; rather mutations in the COOH terminal
of CaM have more significance for CaM MPK.  An
interesting study in which the MLCK CaM-binding domain was replaced with a
subunit residues 296-314 corresponding to the CaM MPK CaM-binding region
produced a recombinant MLCK which had the same characteristically lower
affinity and interaction with mutant CaMs as for the native CaM MPK.
<p>
Study of the autoinhibitory domain has been done using
isolated synthetic peptides with native enzyme and catalytic fragments and
using site-directed mutagenesis.
Peptide 281-309 of the a subunit binds both CaM and inhibits CaM MPK
activity.  Peptide 273-302 inhibits CaM
MPK whether CaM is present or not (it is missing the critical CaM-binding
residues).  Replacement of His-282 and
Arg-283 with Asp produces an &alpha;-CaM MPK with nearly Ca<sup>2+</sup>-independent
activity (67%  independent).  If only R-283 was replaced, there was little
effect.  Deletions of 291-315 or 304-315
in the CaM-binding domain also disrupt the inhibitory region, with
Ca<sup>2+</sup>-independent activity.  Inhibtion of CaM MPK by the
bifunctional 218-309 peptide shows competitive inhibtion with ATP,
noncompetitive with peptide substrate.  With Ca<sup>2+</sup> and
CaM present, inhibtion is reduced 10-fold, evident from enhanced
substrate phosphorylation and the fact that
protein modification reagents phenylglyoxal and 5&rsquo;-p-fluorosulfonylbenzoyl
adenosine (FSBA) show improved access to the ATP-binding site in the
protein.  Shorter peptides (209-309) also have bifunctional ability
to bind CaM and inhibit activity, but inhibiton is competitive with
peptide substrates and noncompetitive with ATP.  Taken together
the N-terminal half of the inhibitory domain probably binds to the
ATP site, while the C-terminal half of
the domain binds to the protein substrate site.
<p>
Limited chymotryptic digestion of solubilizes a 30 kDa
fragment from the postsynaptic density fractions which is a
Ca<sup>2+</sup>-independent kinase.  The proteolytic products
appears to depend upon the phosphorylation state of the CaM kinase,
with dephosphoprotein producing a 31 kDa inactive fragment, and the
autophosphorylated protein the 30 kDa active fragment.  The active
chymotryptic fragment shows a carboxyl terminus corresponding to Ile-271,
while the inactive fragment can generate Phe-293 or Leu-299.
The inactive fragment clearly contains the  autoinhibitory sequences
while the active fragment does not;  also the CaM-binding residues are missing,
which means the inactive fragment is not activatable by Ca/CaM.
The fact that calpain and trypsin also produce active catalytic
fragments suggest a subunit tertiary structure which
presents a hinge between the regulatory and catalytic domains.
<p>
Enzyme autophosphorylation can incorporate as much as 42 mol P/mol
holoenzyme.  Autophosphorylation is intramolecular since the reaction
is independent of enzyme concentration.
Autophosphorylation produces Ca<sup>2+</sup>-independence,
in which the Ca-independence is accompanied by a 2- to 5-fold increase in
substrate K<sub>m</sub> (depends on substrate; others report a decreased
K<sub>m</sub> for certain substrates) and a reduced if not unchanged
V<sub>max</sub>.   Autophosphorylation appears to inhibit maximal
stimulation, pronounced at subsaturating ATP concentrations.
Autophosphorylation may make the CaM MPK susceptible to irreversible
thermal inactivation, and ATP may have a protective effect.
<p>
Substrate specificity may change with the autophosphorylated enzyme.
The Ca<sup>2+</sup>-independent CaM MPK may phosphorylate
CaM-binding proteins within the CaM-binding regions of these proteins.
These sites would be blocked otherwise in the presence of high
Ca<sup>2+</sup> or bound CaM.  Such proteins in this class
include calcineurin, Ca/CaM-dependent phosphodiesterase, and smooth muscle
MLCK.  Histone and myelin basic protein
phosphorylation increase with autophosphorylated CaM MPK as well.
<p>
Studies with nonhydrolyzable ATP show that activity cannot
occur without ATP in the autophosphorylated CaM MPK.
Treatment of the autophosphorylated CaM MPK with protein
phosphatase 1 or 2A restores Ca<sup>2+</sup>-dependence.
ATP analogues with the &gamma;-S moeity (resistant to phosphatases)
produce Ca<sup>2+</sup>-independence which
is not lost with phosphatase treatment.  And extent of
autophosphorylation varies directly with the graded activation
of the enzyme.  Controlling the rate of autophosphorylation shows
a particular P-Thr phosphopeptide increases with activity.  This is
Thr-286 in the &alpha; subunit, Thr-287 in
the &beta; subunit.  A consensus Arg-Gln-Glu-Thr has been
identified for all of the known isoforms in this region.
Phosphorylation of Thr-382 in the &beta; subunit has no
corresponding site in &alpha; or &beta;&rsquo;
subunits and is not required for Ca<sup>2+</sup>-independence.
Site-directed mutagenesis demonstrates that
Thr-286 is necessary and sufficient for the Ca<sup>2+</sup>-independent
activity:  replacement with Leu, Pro, or Ala eliminated the potential
for autonomous activity, with mutants having normal activity which
phosphorylates sites in the subunit other than Thr-286.  Replacement
of Thr-253, Thr-261, Thr-276, and Ser-279 as other residues within
this region did not affect autophosphorylation/Ca-independence.  When
Thr-286 was replaced with Asp and expressed in COS cells and by in
vitro translation, the kinase was also Ca-independent as for the
wild-type autophosphorylated, indicating a negative charge at this
site is all that is necessary.
<p>
The autophosphorylation has been shown to be intramolecular with
respect to the holoenzyme.  This autophosphorylation can be inter-
or intrasubunit, a finding based on use of &alpha; subunit truncated to
prevent oligmeric association since dissociation of wild-type expressed
polypeptide into monomers is difficult.
Autophosphorylation of the truncated monomers was largely intersubunit.
Holoenzymes containing a mixture of inactive subunits (Lys-42 is replaced,
necessary for activity) and active subunits lacking Thr-286 were able
to phosphorylate the Thr-286 of the inactive subunits.
<p>
While studies have been done to determine the necessity of
autophosphorylation as a prerequisite for substrate phosphorylation, in which
high concentrations of substrate inhibit both autophosphorylation and their own
phosphorylation, or studies showing that the phosphorylation of substrates
appears to lag autophosphorylation under limiting concentrations of ATP, the
genetic studies in which Thr-286 is replaced by Ala, Leu, or Pro and kinase
activity remains present would indicate that autophosphorylation plays no role
in the activation of the kinase.
<p>
Autophosphorylation of CaM MPK requires Ca/CaM only during
an initial period in which some of the enzyme is phosphorylated.
Removing Ca or CaM does not slow the autophosphorylation of kinase.
After the initial autophosphorylation, phosphorylation continues
which reduces CaM affinity for CaM MPK.  This phosphorylation
is reversed with the typical phosphatases.
Site analysis has shown that phosphopeptides containing Thr-305 and
Ser-314 in the CaM binding domain, in which phosphatases dephosphorylated
Thr-305 and restored CaM binding activation.
A Thr-306 is also present and its phosphorylation has the same effect as
Thr-306.  Combined mutagenesis and phosphorylation studies indicate
phosphorylation of a Thr-310 or Ser-314 has little effect on CaM binding.
<p>
Why is the CaM MPK a multisubunit enzyme of such large size?
While cooperativity with CaM-stimulated activation and intersubunit
autophosphorylation are observed effects of the
oligomeric organization, another effect is that only a few subunits need be
autophosphorylated to induce Ca<sup>2+</sup>-independence
in all subunits.  As little as 3-4 moles
P/mol holoenzyme produced maximal Ca-independent activation.
It has been proposed that the CaM kinase
structure presents a molecular switch which may influence memory by encoding
memory at the neuronal protein level.
The model proposes that a minimal level of subunits by
autophosphorylated and have kinase activity, and by an presumed allosterism,
they force the other (albeit unphosphorylated) subunits into the active state
and independent of Ca<sup>2+</sup> level.  The kinase keeps up the level of
activity by phosphorylation of subunits within the holoenzyme and on other
holoenzymes in competition with phosphatase activity.
For the model to work, (1) autophosphorylation must convert
kinase subunits to a Ca<sup>2+</sup>-independent
state; (2) there should be cooperativity in the effect of autophosphorylation;
(3) autophosphorylation should continue in the absence of Ca<sup>2+</sup>,
including the site which turns on the kinase; (4) nascent subunits should
be insertable into pre-existing oligomers since autophosphorylation an
intraholoenzyme event.
<p>
Autophosphorylation also causes the trapping of CaM to the
protein.  This may be useful in the neuronal cells in particular
undergo frequent spikes in Ca<sup>2+</sup> levels, and this may
be a way of keeping the protein prepared for such fluctuations.
Measurements of dissociation rates of dansylated CaM (F-CaM) and
CaM MPK were done using fluorescence emission anisotropy (possible since
rotational mobility in free F-CaM differs from the F-CaM bound to kinase).
In the presence of unlabeled CaM and high Ca<sup>2+</sup>, dissociation
time is ~400 ms.  Autophosphorylated kinase increases that
time by 1000-fold (F-CaM is bound for hundreds of seconds).
The affinity is shown to change from 15 nM to &lt;10 pM, from
one of the weakest affinities for CaM to one of the highest.
At low Ca<sup>2+</sup> (100 nM), the dissociation time increases
from 0.2 s to 10-20 s.  Thr-286 is found to be involved since mutant
(Ala-286) enzyme does not trap CaM; only when Thr-286 is phosphorylated
does trapping occur.  Trapping has been shown for the &beta; and &alpha;
subunits, and it is presumed other subunits are equally capable.
<p>
It has also been shown that autophosphorylation and
trapping are cooperatively stimulated by CaM.
The autophosphorylation and trapping may involve two neighboring
subunits in the oligomer each bound to CaM.

<p>
The current model of CaM MPK regulation can be considered
for two neighboring subunits within an oligomer.
Prior to Ca<sup>2+</sup>
increases and CaM binding, both are inactive as a result of the autoinhibitory
domain in the polypeptide binding to the catalytic domain.
The binding of Ca<sup>2+</sup>/CaM opens the catalytic domain by
displacing the inhibitory domain, providing access to protein substrate and
ATP.  Intersubunit phosphorylation can
occur within the holoenzyme, but only when Thr-286 is presented by the binding
of CaM to the subunit.
Nonautophosphorylated subunits rapidly deactivate when the Ca<sup>2+</sup>
level falls.  Autophosphorylated subunits remain active for many seconds
as CaM remains trapped.  With dissociation of CaM,
a Ca<sup>2+</sup>-independent phosphorylation of either Thr-305 or
-306 leads to blocking of further CaM binding.
In high frequency neuronal stimulation, each Ca<sup>2+</sup> spike
may be subsaturating and only a few molecules of CaM may bind;
although these particular subunits are activated transiently,
no autophosphorylation can occur by one neighboring subunit to another.
When CaM does bind to adjacent subunits,
Thr-286 is phosphorylated and CaM will stay bound to this subunit
for a time longer than others.  If Ca<sup>2+</sup> again rises
within 1-10 seconds, CaM again binds, and with the autophosphorylated
subunits already bound with CaM, the probability is greater now that
CaM will bind to other neighboring subunits, leading to more
autophosphorylation and continued greater levels of trapping.
<p>
Comparing the regulation of CaM MPK to other 2nd messenger-activated
kinases PK-A and -C shows some similarities.
(1) Each of these kinases has an autoinhibitory domain, with PK-A being
regulated by a peptide region on a separate subunit, while PK-C is like CaM MPK
in that it has a regulatory domain with a single polypeptide.
(2) The second messengers interact at a site
either next to or overlapping the inhibitory region.
(3) PK-A is autophosphorylated just as is CaM MPK, except that
autophosphosphorylation (of the type II regulatory subunit) can occur without
prior activation of the catalytic subunit.
The effect of regulatory subunit phosphorylation appears to both enhance
dissociation rate of the R and C subunits when cAMP levels rise and to slow the
re-association of the R and C subunits.
But in this case, the activity continues beyond a change in the cellular
signal levels for both CaM MPK and PKA.
<p>
The autoregulatory behavior of the CaM MPK has been shown
to occur in situ in a number of cells and tissues using in vitro assays of
freshly lysed cells.  K<sup>+</sup>-depolarized tissues show rapid
Ca<sup>2+</sup> influx and later the Ca<sup>2+</sup>-independent activity.
CaM MPK activity is elevated in response to IP<sub>3</sub>-mediated
increases in Ca in PC12 and GH3 cells.  Bradykinin is a
hormone that works through this system.  The CaM MPK in neurons
appears to have a high basal level of Ca<sup>2+</sup>-independent activity.
<p>
Using MAbs and PAbs for immunosubcellular localization
studies, high concentrations of CaM MPK could be found in neuronal cell bodies
and dendritic processes, with moderate levels in nerve terminals and
spines.  Lower levels are in the axons of the main fiber tracts.  CaM MPK is
generally found all through the cell, and in the areas on both sides of the
synapse.  Biochemical separation shows cytosolic
and particulate associations, including the plasma membrane, postsynaptic
density, cytoskeleton, synaptic vesicles, and nucleus.
<sup>125</sup>I-CaM binding studies show that 35% of the &alpha;
subunit is soluble and 65% particulate.  In situ hybridization using &alpha;
and &beta; subunit probes indicates that the mRNAs themselves appear
to translocate into the various processes of the neuron
where they are expressed and probably regulated translationally.
<p>
Some reports indicate that subcellular location changes
with physiological activation and autophosphorylation of CaM MPK
in <i>Aplysia</i>, in which the kinase moves from the cytoskeleton
to the cytosol.  Blue light adaption studies with Drosophila
(in which treatment alters light/dark behavior) was correlated
with an increase in membrane-bound CaM MPK in the fly head/brain.
Rat retinal CaM MPK shows a similar shift in location.
<p>
Developmental changes also are accompanied by change in
subunit expression.  The &beta; subunit predominates at birth in rat
(&alpha;:&beta;&lt;1), but declines with age (&alpha;:&beta; &gt;2).
The increase in &alpha; subunit is coincidental with synaptogenesis
and synaptic maturation, and these may be related.  The cerebellum
always has CaM MPK in which &beta; significantly outnumbers
&alpha; in the holoenzyme. The kinase also is predominantly cytosolic
postnatally, shifting to a particulate/membrane fraction with age.
<p>
Studies in macaque monkeys show that neuronal stimulation
affects CaM MPK expression.  With monocular visual deprivation,
analysis of the columns in the visual cortex
deprived of visual input show increased CaM MPK in those tissues,
related with an increase in mRNA levels (hence regulation is
transcriptionally controlled).  In contrast, neuronal
hyperexcitability is associated with reduction of CaM MPK levels.
<p>
Nearly 40 proteins are listed in a table as substrates for
the CaM MPK.  These proteins include enzymes involved in vital
biochemical pathways, cytoskeletal proteins, ion channel proteins,
and transcription factors.  In some cases the in vitro phosphorylation
must be demonstrated in situ.  The consensus phosphorylation site
requires a basic residue three residues upstream;  The RXY(S/T)
is the ideal.  In PK-A, X is a basic amino acid; CaM MPK phosphorylates
that much less than if X is not basic.  Some deviations in the consensus
are found (for example, R is K in some cases, or their are multiple
basic redidues.
<p>
CaM kinase inhibitors selective and nonselective have been
found.  Since it is a CaM-dependent enzyme, any inhibitor
affecting CaM will do the job.  CaM antagonists include
neuroleptics (trifluoperazine), miconazoles (calmidazolium),
naphthalenesulfonameds (W-7), &amp;c.  Some inhibitors affect
the ATP binding site, but would affect most other kinases:
these include staurosporine family members (staurosporine, K-252a,
UCN-01) and isoquinolinesulfonamides (H-7).
Sphingosine and 12-hydroperoxyeicosatetraenoic acid are identified as
inhibitors:  sphingosine is known to block PK-C and CaM-dependent
enzymes and therefore not very selective. The other agent is a potent
CaM MPK inhibitor.  KN-62, or
1-[N,O-bis(5-isoquinolinesulfonyl)-N-methyl-<span style="font-variant:small-caps">l</span>-tyrosyl]-4-phenylpiperazine,
binds to CaM MPK and blocks activation by CaM; it is not very soluble however,
and does not inhibit autophosphorylated enzyme.
<p>
The substrates phosphorylated in situ by CaM MPK have been
identified by labeling studies, then correlated by in vitro experiments.
Substrates implicated include synapsin I, pyruvate kinase, phenylalanine
hydroxylase, tyrosine hydroxylase, phospholamban, MLCK, and MAP-2.  Use of
peptidergic inhibitors (resembling the autoinhibitory domain) injected into
postsynaptic cells shows a role for CaM MPK in nuclear envelope breakdown,
release of Glu and NorEpi, controlling a Ca-activated chloride channel protein,
and regulating the ileal brush border Na<sup>+</sup>/H<sup>+</sup> exchange.

<p>
Other CaM kinases have also been identified.
CaM kinase I is found in neuronal tissues
and phosphorylates a number of proteins, including synapsin I, synapsin II,
smooth muscle myosin light chain, and the transcriptional activator CREB.
Its specificity is closer to that of PK-A
than the CaM MPK II as it phosphorylates the &ldquo;PK-A site&rdquo; of synapsin I, which
is a Ser in the globular domain.
Serines in the vesicle-associating domain are phosphorylated by the CaM
MPK.  CaM kinase I is monomeric with M<sub>r</sub> 42,000, while three
species (37, 39, and 42 kDa) have been shown on SDS-PAGE.  Its cDNA
sequence remains unknown, and so does any work on its in situ function.
<p>
CaM kinase IV (CaM kinase-Gr) was isolated initially in a
screen of expression libraries looking for novel CaM kinases.
This kinase codes for a 474 aa polypeptide,
with the kinase domain showing many conserved features of kinase catalytic
domains.  The COOH terminal of this kinase in rat has been found to be
identical to calspermin, and genomic structure analysis indicates that
one gene with several exons encodes both CaM kinase IV and calspermin.

<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
West, S. C.
 </span>
 <span class="year">
(1992)
 </span>
 <span class="jname">
Annu. Rev. Biochem.
 </span>
 <span class="jvol">
61,
 </span>
 <span class="pages">
603-640.
 </span>
<p class="reftitl">
Enzymes and molecular mechanisms of genetic recombination
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Symons, R. H.
 </span>
 <span class="year">
(1992)
 </span>
 <span class="jname">
Annu. Rev. Biochem.
 </span>
 <span class="jvol">
61,
 </span>
 <span class="pages">
641-671.
 </span>
<p class="reftitl">
Small catalytic RNAs
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Marians, K. J.
 </span>
 <span class="year">
(1992)
 </span>
 <span class="jname">
Annu. Rev. Biochem.
 </span>
 <span class="jvol">
61,
 </span>
 <span class="pages">
673-719.
 </span>
<p class="reftitl">
Prokaryotic DNA replication
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Tan, J. L., Ravid, S., Spudich, J. A.
 </span>
 <span class="year">
(1992)
 </span>
 <span class="jname">
Annu. Rev. Biochem.
 </span>
 <span class="jvol">
61,
 </span>
 <span class="pages">
721-759.
 </span>
<p class="reftitl">
Control of nonmuscle myosins by phosphorylation
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Hershko, A., Ciechanover, A.
 </span>
 <span class="year">
(1992)
 </span>
 <span class="jname">
Annu. Rev. Biochem.
 </span>
 <span class="jvol">
61,
 </span>
 <span class="pages">
761-807.
 </span>
<p class="reftitl">
The ubiquitin system for protein degradation
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Marcu, K. B., Bossone, S. A., Patel, A. J.
 </span>
 <span class="year">
(1992)
 </span>
 <span class="jname">
Annu. Rev. Biochem.
 </span>
 <span class="jvol">
61,
 </span>
 <span class="pages">
809-860.
 </span>
<p class="reftitl">
<i>myc</i> Function and regulation
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Biemann, K.
 </span>
 <span class="year">
(1992)
 </span>
 <span class="jname">
Annu. Rev. Biochem.
 </span>
 <span class="jvol">
61,
 </span>
 <span class="pages">
977-1010.
 </span>
<p class="reftitl">
Mass spectrometry of peptides and proteins
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Wallace, D. C.
 </span>
 <span class="year">
(1992)
 </span>
 <span class="jname">
Annu. Rev. Biochem.
 </span>
 <span class="jvol">
61,
 </span>
 <span class="pages">
1175-1212.
 </span>
<p class="reftitl">
Diseases of the mitochondrial DNA
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Blau, H. M.
 </span>
 <span class="year">
(1992)
 </span>
 <span class="jname">
Annu. Rev. Biochem.
 </span>
 <span class="jvol">
61,
 </span>
 <span class="pages">
1213-1230.
 </span>
<p class="reftitl">
Differentiation requires continuous active control
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Glass, D. B., Feller, M. J., Levin, L. R., Walsh, D. A.
 </span>
 <span class="year">
(1992)
 </span>
 <span class="jname">
Biochemistry
 </span>
 <span class="jvol">
31,
 </span>
 <span class="pages">
1728-1734.
 </span>
<p class="reftitl">
Structural basis for the low affinities of yeast cAMP-dependent
and mammalian cGMP-dependent protein kinases for the protein kinase inhibitor
peptides
<p class="Para1">
In <i>Saccharomyces</i> yeast are three genes coding for subunits
which for catalytic subunits for PK-A.  The genes are denoted as
<i>TPK1, TPK2, TPK3</i> and code for the C<sub>1</sub>, C<sub>2</sub>,
and C<sub>3</sub> subunits, resp.  The enzymes have protein structures with
75-88% identity between them, with 50% identity with the mammaliam C
&alpha; and C&beta; isoforms.  The inhibitor peptide, PKI(5-24) is
however 30-fold less potent an inhibitor of the C<sub>1</sub>
kinase (the best studied of the three) than the mammalian kinase.
<p>
Yeast C<sub>1</sub> was titrated with PKI(6-22)amide and other
analogues and curves obtained were transformed to linear by
log-logit transformation to derive slopes, IC<sub>50</sub>, and
<i>K</i><i><sub>i</sub></i> values
[<i>K</i><sub>i</sub>= IC<sub>50</sub>/(1+[S]/<i>K</i><sub>m</sub>)].
The titration curves of the analogues were all parallel to one another.
In experiments with respect to the Kemptide substrate, all inhibition was
competitive.  Certain analogues were
more potent inhibitors of yeast PK-A than the parent compound, all with
<i>K</i><sub>i</sub> values being &gt;100 nM. Complete inhibition is obtained with
peptides in excess of 10 &micro;M.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Lim P-O, Sears BB
 </span>
 <span class="year">
(1992)
 </span>
 <span class="jname">
J. Bacteriol.
 </span>
 <span class="jvol">
174,
 </span>
 <span class="pages">
2606-2611.
 </span>
<p class="reftitl">
Evolutionary Relationships of a Plant-Pathogenic Mycoplasmalike
Organism and <i>Acholeplasma laidlawii</i> Deduced from
Two Ribosomal Protein Gene Sequences
<p class="Para1">
Certain mycoplasma-like organisms (MLOs) that are plant pathogens were
identified in 1969, but their taxonomy is uncertain.  They are members
of the class <i>Mollicutes</i> because they have no cell wall,
but their biochemical and nutritional
requirements are unknown because they have not been cultured.
A segment of the ribosomal protein gene cluster from the MLOs and
from <i>Acholeplasma laidlawii</i> (a mycoplamsa with cell wall that
has not sterol) was cloned and sequenced.  rRNA genes are not useful
for study since base variations rarely occur since critical pairing
is affected.  The codon usage of the protein genes was studied also,
since this distinguishes acholeplasmas (does not require sterol)
from mycoplasmas (requires sterol).  UGA is a terminator codon in
eukaryotes and prokaryotes, but animal mycoplasmas and spiroplasmas
use it as a tryptophan codon (UGG is also a Trp codon).
<p>
A 2.7 kb <i>Hin</i>dIII fragment from an MLO pathogenic to <i>Oenothera</i>
was cloned using a heterologous probe holding a segment of the
ribosomal gene operon from <i>M. capricolum</i> (GenBank M74770).
From this clone the entirety of the genes <i>rpl22</i> and
<i>rps3</i> were sequenced.  A 1.2 kb fragment from <i>A.
laidlawii</i> using PCR and using conserved sites bracketing <i>rpl22</i>
and <i>rps3</i> as primers was also obtained and sequenced
(GenBank M74771).

<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Akins, R. E., Levin, P. M., Tuan, R. S.
 </span>
 <span class="year">
(1992)
 </span>
 <span class="jname">
Anal. Biochem.
 </span>
 <span class="jvol">
202,
 </span>
 <span class="pages">
172-178.
 </span>
<p class="reftitl">
Cetyltrimethylammonium bromide discontinuous gel electrophoresis:
M<sub>r</sub>-based separation of proteins with retention of enzyme activity
<p class="Para1">
<b>INTRO</b>.  The method of Laemmli is the most common M<sub>r</sub>-based 
separation of proteins used in biochemistry.  This method exploits observations 
made by others (Shapiro et al., <i>Biochem. Biophys. Res. Comm.</i> <b>28</b>: 
815, 1967; Weber and Osborn, <i>J. Biol. Chem.</i> <b>244</b>: 4406, 1969) 
showing anionic detergents such as SDS could be used to separate proteins on 
that basis.  When use of detergent combined with the Ornstein (<i>Ann. N.Y. 
Acad. Sci.</i> <b>121</b>: 321, 1964) and Davis (<i>Ann. N.Y. Acad. Sci.</i> 
<b>121</b>: 404) description of polyacrylamide gels, then separation is better 
than in free solution.  SDS/protein complex formation indicates a complex shaped 
like a prolate ellipsoid with dimensions related to the M<sub>r</sub> of the 
protein.  Problems with SDS include the fact that it is an often irreversible 
denaturant. Use of the nonionic detergent Triton X-100 does not separate 
proteins on the basis of size, and M<sub>r</sub> assignments require multiple 
runs at different gel concentrations.
<p>
Recently the cationic detergent cetyltrimethylammonium bromide (CTAB) has been 
found to combine the useful properties of SDS and TX-100.  Proteins are 
separated in gels based on a logarithmic function of their M<sub>r</sub>. 
Moreover, no loss of enzymatic activity is observed.  Samples are prepared 
without boiling or reducing agents.  Resolution is poor however, because a 
stacking gel is not used. A new system is described which uses <i>C</i>TAB as a 
solubilizing agent, <i>a</i>rginine as a stacking agent, and <i>t</i>ricine as a 
counterion and buffer:  it is therefore called CAT gel electrophoresis.
<p>
<b>METHODS</b>.  CTAB and many other chemicals and biochemicals obtained from 
Sigma.  Protein samples are dissolved in CTAB sample buffer (10 mM Tricine NaOH, 
pH 8.8, 1% CTAB, 10% glycerol). In some cases 10 &micro;l saturated aqueous 
crystal violet is added per&nbsp;ml&nbsp;CTAB sample buffer to visualize the 
salt front during electrophoresis.
<p>
Monomeric acrylamide/bis solutions are prepared as 40% T (w/v) stocks with 2.67% 
C (w/w).  Stocks are diluted to 6% T as a final concentration with Tricine-NaOH, 
pH 8.0, at 0.375 M.  Polymerization is initiated with 0.1% ammonium persulfate 
and 0.01% TEMED. After 1&nbsp;h&nbsp;to ensure complete polymerization, a 
stacking gel of 0.7% agarose, 0.1% CTAB, and 0.125&nbsp;M Tricine-NaOH, pH 10.0, 
was overlayed and the comb not removed until thoroughly cooled.  Alternatively, 
low percentage T acrylamide stacks (polymerized with 0.1% ammonium persulfate 
and 0.025% TEMED) were also cast.  Running buffer is made as 5X stock (0.5% 
CTAB, 125 mM Tricine adjusted to pH 8.2 with 1&nbsp;M arginine free base).
<p>
Gels are run at various temperatures until the salt/dye front approaches the end 
of the gel (~45 min for mini-gels; 4&nbsp;h for large gels).  The run is stacked 
at 100 V, separated at 150 V.  Note that the <i>anode</i> is at the top of the 
gel, the <i>cathode</i> at the bottom (opposite the normal Laemmli system). 
Gels when stained were fixed in 10% acetic acid/25% MeOH, then stained in 0.025% 
Coomassie Brilliant Blue in 25% isopropanol, 10% acetic acid for 60 min. 
Destaining was commenced with 0.001% CBB in 10% isopropanol/10% acetic acid, 
completed with 10% acetic acid.
<p>
Catalytic activities were determined.  Peroxidase was assayed at room temp using 
0.5 mg/ml diaminobenzidine in 10 mM NaP<sub>i</sub>-buffered 0.9% saline, pH 
7.2, with 0.1% hydrogen peroxide.  After 5 min, reactions are stopped with 
addition of 10 mM Na<sub>2</sub>S to a final 100 &micro;M. The OD at 480 nm is 
used to assess product formation. Ca<sup>2+</sup>-ATPase activity, alkaline 
phosphatase, lactate dehydrogenase, and isocitrate dehydrogenase activities are 
measured as well.  Gel in situ activity measurements are determined for 
peroxidase and Ca<sup>2+</sup>-ATPase.
<p>
<b>RESULTS</b>. CAT gels of 8% T acrylamide with a 4% T stack were used to 
separate marker proteins, and E coli extract, chick CAM microsomal membrane 
proteins, and a preparation of horseradish peroxidase.  The stained bands 
appeared to have excellent resolution.  Separation plotted with log 
M<sub>r</sub> against R<sub>f</sub> of the protein indicated linearity, as with 
SDS gel systems.  In the test of the linearity, samples were boiled and reduced, 
then separated in a 6% CAT gel with 0.7% agarose stack.
<p>
Various enzymes were dissolved in the CTAB, SDS, or Triton X-100 detergents and 
recoverable activities assessed. Results show that activities were all 
significantly affecte by all detergents, but that CTAB was the least 
deleterious. With alkaline phosphatase, 0.5% CTAB was 75% of the Triton 
X-100-dissolved activity, while 0.5% SDS was 25% of that value.  With isocitrate 
dehydrogenase, the CTAB-treated enzyme was not significantly different from the 
TX-100-treated enzyme, while the SDS-treated enzyme was virtually inactivated. 
With HRP, the CTAB-treated enzyme was 50% as active as the TX-100-treated 
protein, while the SDS-treated protein was 15% as active.  But CTAB was not kind 
to Ca<sup>2+</sup>-ATPase (5% of TX-100 value) while SDS virtually inactivated 
the membrane protein. LDH was inactivated by both CTAB and SDS equally, relative 
to the TX-100 value.  All enzymes in either CTAB or SDS, when heated to 100&deg; 
in presence of 2-mercaptoethanol, lost all activity.
<p>
Histochemical staining of enzymes fractionated on CAT gels showed that the stain 
of Ca<sup>2+</sup>-ATPase migrated to a single band near its M<sub>r</sub> of 
170,000.  This shows that some detectable activity is present despite the use of 
CTAB. HRP activity was separated into its multimeric forms:  peroxidase activity 
count not be found in the monomeric forms, but all activity detected was found 
in the tri- and tetrameric forms migrating to their slower positions.
<p>
<b>DISCUSSION</b>.  Choice of arginine as the primary stacking ion was based on 
the fact that a zwitterion with a basic pI value is a low mobility ion in the 
stack. Lysine was considered but has a higher mobility because its pI is to 
close to the stack buffer pH.  Tricine was selected since it has a negative 
charge to act as a counter ion for arginine and its pK is near the pH of the 
separating gel and running buffer.  Unfortunately acrylamide polymerization is 
prolonged with this buffer at pH 8.0.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Peterson J, Fujimoto WY, Brunzell JD
 </span>
 <span class="year">
(1992)
 </span>
 <span class="jname">
J. Lipid Res.
 </span>
 <span class="jvol">
33:
 </span>
 <span class="pages">
1165-1170.
 </span>
<p class="reftitl">
Human Lipoprotein Lipase:  Relationship of Activity, Heparin Affinity, and
Conformation As Studied with Monoclonal Antibodies
<p class="Para1">
Adult human males from 30-55 y age and on no meds submitted pre- and postheparin
plasma.
<p>
Monoclonal to whole length lipoprotein lipase (LPL) obtained from bovine milk (see
Iverius et al., <i>Methods Enzymol.</i> <b>129:</b> 691, 1986) was prepared
according to a previous method (Campbell, <i>Monoclonal Antibody Technology: The
Product and Characterization of Rodent and Human Hybridomas</i>, Elsevier: Amsterdam,
1984).  Clone 5D2 was screened positive for bovine LPL and then examined on western.
Plasma samples were first immunoprecipitated and the precipitates run on SDS-PAGE,
followed by western analysis.  Preheparin plasma showed three bands with the
anti-5D2 western.  Postheparin plasma shows the same profile, but with a much
more intense uppermost band.  Another lane shows the postheparin plasma purified
on heparin-Sepharose affinity, with only a single band co-migrating with the uppermost
band.  Purified bovine LPL was also run on the gel and had the same M<sub>r</sub>
as the uppermost human plamsa band.  The two bands with lower mass (faster mobility)
are likely degradation bands.
<p>
In the heparin-Sepharose chromatography, two lipase activity peaks were identified,
one eluting at 0.70 M and another at 1.1 M.  The first peak was hepatic lipase
and the second LPL.  Because they have significant homology, it was important to
see how specific 5D2 is for LPL.  The (pooled?) activity from each peak was mixed
with one of three things:  (1) no addition, (2) addition of Sepharose
(3) an immunoaffinity solid phase of 5D2-Actigel (with a Sepharose matrix), then
activity of supernatant measured.  Supernatant lipase activity of the first peak
was identical (34.2-34.8 nmol/ml*min).  Supernatant lipase activity of the no addition
and Sepharose controls were high (86.0 and 88.8 nmol/ml*min) and barely detectable
in the 5D2-Actigel treated sample (0.3 nmol/ml*min).  This result shows that 5D2
is specific for human LPL.
<p>
An total 5D2 ELISA sandwich assay was done because it requires the same epitope
in duplication to be detected.  In addition, a sandwich using 5F9 to coat the plate
and 5D2 for antigen detection was done.  These mAbs recognize different epitopes
based on the appearance of multiple degradation fragments (&lt; 30 kDa) on
westerns that were not identical.  mAb 5F9 also requires partial denaturation
of LPL to detect.  Human LPL was heat-treated at 45&deg; and tested on ELISA.
The 5F9 ELISA measured low amounts with only limited time treatment, but higher
amounts as the time of heat treatment progressed and was negatively correlated
with LPL activity.  Immunoreactivity decreased with time for the 5D2 ELISA and
correlated with loss of activity;  however, 5D2 reactivity did not disappear
entirely.  When the denaturing treatment was 1 M guanidine HCl as opposed to heat,
the ELISA results were similar.
<p>
The activity peak eluting at 1.1 M from the heparin column was treated with 1 M
Guan HCl and rechromatographed.  No LPL activity was found at all in the column
load nor in eluted fractions.  Fractions were tested in the 5F9 ELISA.  Most (75%)
of the Guan HCl-treated LPL eluted with a peak from 0.75 M NaCl, and some LPL
was seen in the unbound fraction, and no LPL mass was seen in fractions where active
LPL would elute.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Marks JD, Hoogenboom HR, Griffiths AD, Winter G
 </span>
 <span class="year">
(1992)
 </span>
 <span class="jname">
J. Biol. Chem.
 </span>
 <span class="jvol">
267,
 </span>
 <span class="pages">
16007-16010
 </span>
<p class="reftitl">
Molecular Evolution of Proteins on Filamentous Phage:  Mimicking The
Strategy Of The Immune System
<p class="Para1">
<b>Minireview</b>.  Nature engineers proteins through a mutation/selection
process.  Enzymes evolve to have Km values similar to the concentration of
substrates in vivo, and this changes as other enzyme activities alter to produce
different substrate concentrations.  The immune system produces a diverse
repertoire of antibody molecules by gene arrangements designed to produce
randomness in short sequence segments.  This repertoire is expressed on the
B cell surface, each cell committed to the production of particular species
of a randomized gene rearrangment.  When bound to antigen in a productive
manner, B cells are activated:  proliferation and a subset of cells differentiating
to short-lived plasma cells; another subset becomes memory cells.  In another
process called <b>affinity maturation</b>, random (somatic) point mutations
occur and further antigen selection.
<p>
B cells display a form of the antibody so that Ag can bind and activate the cells.
Viruses can also be used as a genetic display package in which Ab genes are
inserted into a gene of the viral coat protein.  The virus displays the Ab
on  the surface and also contains the gene.  Filamentous phage have been used
to display small peptides fused to the minor coat protein (pIII) or to the
major coat protein (pVIII).  Three (possibly five) copies of pIII are at the
phage tip and about 2500 copies of pVIII per phage.  pIII is required for
attachment to bacterial F-pilus and for infection; pVIII coats the ss DNA
genome.  Protein pIII has two <q>domains.</q>  Fusions are made to N-terminus
of pIII or N-terminal domain can be removed and fusions made to 2nd domain.  Phage
lacking N-terminal domain are not infective.
<p>
A gene encoding a single chain Fv (scFv) Ab fragment inserted into gene III
fused to N-terminus of pIII, the phage can bind to Ag.  Heterodimeric Fab
fragments can be displayed in which H or L chains are linked to coat protein
and the other chain is secreted to periplasmic space where the chains associate.
Fab fragments can be displayed at pIII N-teriminus, pIII second domain
or pVIII N-terminus.
<p>
Inserting an amber codon between the Ab gene and gene III allows the possibility
of growing phage in an E coli supE suppressor strain, in which Gln codes for
the amber codon and Ab-pIII fusions are then expressed.  When grown in
non-suppressors, then only the Ab protein is expressed and secreted in soluble
form from bacteria.
<p>
Phage displaying peptide Ags have been selected by adsorbed/bound Ab and those
displaying Ab fragments are selected to adsorbed/bound Ag.  Other formats
include binding phage-Ab to biotin-Ag or phage-Ag to biotin-Ab, and then
binding these to Strepavidin-coated beads.  Alternatives inlcude Ag-coated
wells or tubes or Ag bound to chromatography matrix.  Acid- or alkali-eluted
bound phage are enriched 20- to 1000-fold (in one case 1,000,000-fold in a
single round).
<p>
Selection of phage can be done using affinity (no, low, high), avidity,
or non-equilibrium kinetics of binding.  In one example, phage displaying
Ab bind monovalent biotinylated Ag in solution and then captured on strepavidin
paramagnetic beads.  If biotin-Ag concentration is lower than <i>K</i><sub>d</sub>
value, but in excess of phage concentration, then phage is selected by affinity
of binding.  With up to 12 rounds of selection, antibodies can be cultivated
with small differences in affinity (2-4-fold).  By diluting phage in excess
unlabeled Ag prior to capture on magnetic beads selection can be on the basis
of kinetics of association rather than equilibrium.  Binding of phage to solid
phase followed by washing also selects fro phages with slower dissociation rates.
<p>
Selection by avidity to Ag-coated solid phase depends on affinity of each
molecule of displayed Ab and on number of Ab fragments per phage.  Number of
Ab fragments per phages is determined by choice of either pIII or pVIII for
fusion, the use of either phage or phagemid vectors, and used fo amber
mutations between Ab and coat protein.  Other factors affect avidity, such as
Ab fragment association into dimers or proteolysis of fusion protein.  Avidity
of binding to solid phase also depends on Ag ocat density.
<p>
Fusion of Ab to pIII will produce 3 copies of fusion protein on each phage
particle allowing multivalent Ag binding.  With pVIII fusion more copies
are displayed (up to 24 Ab molecules/phage).  With multivalent display
the increased avidity will help lower affinity phage bind to solid phase Ag
during washing, especially for phage with fase dissociation rates, but it
hinders discrimination between phage of different affinities.
<p>
In phagemid vectors, intact pIII coat protein from helper phage competes with
pIII fusion for incorporation into phage particle:  there is less than 3 copies
of Ab-pIII fusion protein per phage.  Fusions to pIII second domain are such
the infective fusion phage must have at least one pIII from helper phage, thus
maximum 2 fusion proteins per phage.
<p>
<b>Mimicking Affinity Maturation</b>.  Increases in binding affinity of Ab
occur by somatic mutation of V-gene pairings that occur after initial
immunization, and also from appearance of Ab that use V genes not seen in
the primary response (repertoire shift).
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Ahn, N. G., Robbins, D. J., Haycock, J. W., Seger, R., Cobb, M. H., Krebs, E. G. 
 </span>
 <span class="year">
(1992)
 </span>
 <span class="jname">
J. Neurochem.
 </span>
 <span class="jvol">
59,
 </span>
 <span class="pages">
147-156.
 </span>
<p class="reftitl">
Identification of an activator of the microtubule-associated protein 2 kinases 
ERK1 and ERK2 in PC12 cells stimulated with nerve growth factor or bradykinin
<!-- ================================================================ -->
<p class="tags">
<p class="ref">
 <span class="authors">
Burgeson RE, Nimni ME
 </span>
 <span class="year">
(1992)
 </span>
 <span class="jname">
Clin. Orthop. Rel. Res.
 </span>
 <span class="jvol">
282,
 </span>
 <span class="pages">
250-272.
 </span>
<p class="reftitl">
Collagen Types:  Molecular Structure and Tissue Distribution
<p class="Para1">
<b>Review</b>.  Single most abundant (30% of all) proteins in mammals. Molecule 
is synthesized and secreted from cell, then assembled into characteristic fibers 
depending on tissue (bone, skin, cartilage, tendon).
<p>
13 different types identified.  Organized into three basic groups or classes. 
Class 1 is the banded fiber-forming collagens composed of Types I-III, V, and 
XI.  Long collagenous domains are seen in biosynthetic precursors, and all types 
derive from a single ancestral gene based on genomic structure analysis. 
&alpha;1 and &alpha;2 chains of Type I collagen are coded for on chromosome 17.
<p>
Class 2 are the banded, fibril-associated collagens, represented by Types IX and 
XII.  These do not form supramolecular aggregates, but will form part of a 
fibril structure with other collagen types. Type IX will cross-link with Type II 
in cartilage.  Type XII homology with Type IX suggests it may associate with 
surfaces of banded collagen in noncartilage tissues.
<p>
Class 3 collagens form fibrous structures different from the banded collagens. 
These collagens form supramolecular assemblies.  Includes Type IV (basement 
membranes), Type VI (beaded microfilaments), Type VII (anchoring fibril major 
component).  Type X forms network surrounding hypertrophic chondrocytes.  Type 
VIII and XIII have only been partially characterized.
<p>
<b>Molecular Structure</b>.  <var>Triple-helical</var> and <var>globular</var> 
domains form from the collagen molecule.  Size and number and distribution of 
the domains varies in the different types. <p>Within the triple helical domain, 
every third amino acid is Gly, and then either Pro or hydroxyproline (HOPro) 
follow;  10% of collagen follows a Gly-Pro-HOPro motif.  Each alpha carbon chain 
forms a left-handed helix with a periodicity of 8.7 &Aring; between successive 
Gly positions (3.27 residues per turn).  Each left-handed helix in turn twists 
around two others to form a supercoil with pitch of about 86 &Aring;.  There is 
extensive hydrogen bonding between chains to stabilize the triple helix.
<p>
The procollagen molecule is secreted into the extracellular milieu and its 
nonhelical ends removed by enzymes, producing a fragment that spontaneously 
assembles into a fiber.  Propeptides cleaved from the ends have been sequenced 
and characterized.  Type I collagen N-terminal propeptide has one 
N-acetylglucosamine and intrachain disulfides. Type III procollagen features 
interchain disulfides in the N-terminus. The C-terminus of pro-&alpha;1 and 
pro-&alpha;2 are segments of 30-35 kDa forming a globular domain.  Asn-linked 
oligosaccharides of N-acetylglucosamine and mannose are present in the latter 
(C-terminal side) half of the C-terminal propeptide;  the sugar is attached to 
sequences recognized as <b>-Asn-X-Thr-</b>.
<p>
<b>Gene Expression</b>.  Fibrillar collagen genes span a length about 10 times 
the length of the trimmed message.  The &alpha;(I) gene in chicken is 38 kb 
long with 52 exons, each exon about 54 bp long and separated by introns varying 
from 80-2000 bp.  The consistent 54 bp length in nearly every exon strongly 
indicates that an ancestral sequence gave rise to multiple instances ideal for 
making a fiber required to have structural periodicity.  (Note that a 
Gly-Pro-HOPro tripeptide can be repeated 6 times in a fragment coded by 54 bp.) 
This 54 bp sequence is common to all collagen types forming banded fibers.
<p>
The transcribed message is 3 kb and is exported to the cytoplasm for translation 
on membrane-bound ribosomes (rough ER).  The translated message features a 15-30 
residue long <i>signal sequence</i> to indicate export of the nascent 
polypeptide to the cell exterior.  During message translation, <i>lysyl</i> and 
3- and 4-<i>prolyl hydroxylases</i> are present and hydroxylate Lys and Pro, 
resp.;  these iron-bound enzymes also depend upon ascorbate and 
&alpha;-ketoglutarate for activity. Glycosylation of hydroxylysines occurs by 
the action of a glucsyltransferase and galactosyltransferase.  The signal 
sequence is then cleaved, and interchain disulfides formed between two 
pro-&alpha;1 and one pro-&alpha;2 chain (for Type I) as the first step before 
the triple-helix is wound.  This all occurs in the endoplasmic reticulum.
<p>
From the cisternae of the ER, the triple helix assembly moves toward the Golgi. 
This normally requires further modification of mannose-rich carbohydrate side 
chains, such as the one present in the C-terminal globular domain.  Once 
secreted from the cell, peptidases trim off the globular ends;  for Type I the 
amino acts first, then the carboxyl.
<p>
<b>Extracellular processing</b>.  Newly formed microfibrils are targets of the 
enzyme <i>lysyl oxidase</i>, which further modifies Lys and HOLys groups by 
oxidative deamination to form aldehydes. This enzyme is 30 kDa (probably forms 
homomultimers), requires Cu<sup>2+</sup>, O<sub>2</sub>, and probably pyridoxal. 
The lathyrogen &beta;-aminopropionitrile is an effective inhibitor. The enzyme 
prefers microfibrillar aggregates to monomeric collagen.
<p>
The structure of the triple helix is such that interhelical association to form 
microfibrils is spontaneous.  Electron microscopy and X-ray diffraction reveal 
periodicity.  The aggregation is thought to precede with a 15 &Aring; diameter 
collagen trihelix associating with up to 4 other such molecules in which 
association is staggered by one-fourth of the length.  The 5-helix microfibril 
aggregate is 40 &Aring; in diameter.  The 5-molecule microfibrils then associate 
with each other in <i>lateral aggregation</i>.  Then the lateral aggregates meet 
end-to-end to form <i>end-to-end aggregation</i> to form the mature fibril. 
Well cross-linked fibers span 80 nm or more in diameter.
<p>
In vitro experiments with monomeric collagen have confirmed the ability of 
collagen to aggregate in vivo.  Heated to 37&deg; polymerization occurs in 
solutions that become turbid.  The quarter-length staggering is the result of 
segments of the helix that are charged (and likely to by hydrated) and 
relatively hydrophobic.  As chains associate with the stagger, hydrophobic 
residues create the initial interstrand association that becomes stronger as 
more residues line up.  Water molecules in the intersubunit crease organized 
around charged residues then become free (more likely at a warmer temperature) 
and intersubunit attractions between charge residues of opposite polarity 
strengthen strand association even more.
<h1 style="font-variant:small-caps">Class 1 Collagens</h1>
<p>
<b>Cartilage</b>.  The native form of collagen which composes collagen
is best isolated by prior removal of proteoglycans and the nonhelical,
protease-sensitive crosslinks, or by administering &beta;-aminopropionitrile
to animals during tissue development.
<p>
Type II is the most abundent form in collagen composing cartilage.
It is made of three identical &alpha; chains with chromatographic
and electrophoretic characteristics similar to Type I &alpha;-1 chains.
The Type II chains are designated &alpha;-1(II).
<p>
Collagen in cartilage is rich in hydroxylysine and oligosaccharide content
(N-linked sugars);  sugar content is 5-fold greater than in skin?
Greater glycosylation is correlated with a smaller fibril diameter, and
this may limit the length of the fibril in turn.
<p>
Type II collagen also makes up the central gel-like region of the intervertebral 
disk called the <i>nucleus pulposus</i>,  which itself is surrounded by 
concentric rings of highly ordered, dense collagen fibers making up the annulus 
fibrosus.  This highly hydrated gel is close in structure to hyaline cartilage. 
Type II collagen is elaborated during chondrogenesis from mesoderm.
<p>
Mutations in Type II collagen genes result in well characterized
pathological conditions all forms of chondrodysplasia, such as
spondyloepiphyseal dysplasia congenita, achondrogenesis-hypochondrogenesis,
and Kniest dysplasia.
<p>
Type XI collagen was first isolated from differential salt fractionation of the 
pepsin digests of hyaline cartilage:  SDS-PAGE reveals it to be composed of 
three distinct &alpha; chains which migrate slightly slower (have higher MW) 
than &alpha;-1(II).  This collagen resembles Type V in many properties.  The 
chains have been provisionally called 1-, 2-, and 3-&alpha;.  CNBr digests of 
3-&alpha; show an almost identical peptide map with &alpha;-1(II):  the 
distinction is probably not in the polypeptide sequence, but the degree of 
modification (hydroxylation of Lys and glycosylation).  The function of Type XI 
collagen may be in varying the diameter or growth of Type II collagen, where it 
forms more of the interior than is at the surface of fibrils, and where the 
N-propeptide may not even be cleaved during maturation.  No genetic variants 
causing disease have been available to suggest a function.
<p>
<b>Non-cartilaginous collagen</b>.  Type I collagen was the first of all 
collagens discovered and characterized.  Composed of two &alpha;-1 and one 
&alpha;-2 chain, it is the most abundant in skin, tendon, ligament, bone, and 
the cornea, forming 80-99% of the total collagen.  Bone matrix is entirely Type 
I collagen.
<p>
Its function has been established in part by mutations causing diseases. 
Insertions or deletions or even substitutions of single amino acids can cause 
osteogenesis imperfecta (OI).  These changes can be caused by failure of the 
fibril to form during aggregation, or by cellular mechanisms to eliminate 
proteins recognized as defective.  Substitution of glycine in the tripeptide 
motif -Gly-X-Y- accounts for different degrees of severity in OI.
<p>
An &alpha;-1(I) trimer has been described in viral-caused tumors (polyoma) of 
different animals, and in human skin.  The amino acid composition is similar to 
&alpha;-1(I) in Type I collagen, but the lysine is more heavily hydroxylated.  A 
diagnostic test for a Type I trimer might be useful to assess (pre-)pathological 
states, since it is known this kind of collagen becomes present in chondrocytes 
de-differentiating.
<p>
The residues which remain after neutral salt and dilute acetic acid extraction 
can be treated with CNBr to produce peptides not related to Type I or Type II 
peptides.  Two of the peptides show a similarity to well-known regions in 
&alpha;-1(I) and &alpha;-1(II), and have been called &alpha;-1(III).  These type 
III collagen molecules migrate similarly to type I and II &alpha;-1, with 
reports saying sometimes slower.  The collagen has a high degree of Pro 
hydroxylation, a higher (&gt;33%) Gly content, and the presence of 
intramolecular disulfides of the two Cys residues in the C-terminal.  In the 
tissue these may be involved in interchain disulfide bonding.
<p>
Type III is ofen associated with Type I found in lung, heart muscle and valves, 
uterus, nerves, liver, placenta, umbilical cord vessels, spleen, gingiva, 
kidney, lymph nodes, slcera and other eye structure, and recently in normal 
bone.  Blood vessels are particularly rich.
<p>
Type III collagen may be involved with extensibility of tissue.  It is found in 
greater abundance in fetal tissue, and the ratio of Type III to Type I decreases 
with advancing age.  Mutations in Type III (Ehlers-Danlos Type IV syndrome)

<!-- ============================================================= -->
<p id="amthauer-palp" class="tags">
placental alkaline phosphatase (PALP)
<p class="ref">
 <span class="authors">
Amthauer R, Kodukula K, Udenfriend S
 </span>
 <span class="year">
(1992)
 </span>
 <span class="jname">
Clin. Chem.
 </span>
 <span class="jvol">
38,
 </span>
 <span class="pages">
2510-2516.
 </span>
<p class="reftitl">
Placental alkaline phosphatase: a model for studying COOH-terminal processing
of phosphatidylinositol-glycan-anchored membrane proteins
<p class="Para1">
ALP is known to be membrane-bound in cells expressing it through a
phosphatidylinositol-glycan (PI-G) linkage, and it was the first enzyme
reported to be attached in this way. Components of the PI-G have been
identified by radiotracer analysis: inositol, ethanolamine, fatty acids,
mannose. Now more than 100 proteins have been identified as PI-G-linked to
membranes. Studies by Low &amp; Finean and Ikezawa et al. had previously shown
that placental alkaline phosphatase (PLAP) could be liberated from cells
treated with bacterial phosphoinositol-specific phospholipase C (PI-PLC). When
PLAP cDNA is transfected to COS cells, PLAP is expressed on the cell surface as
observed with immunofluorescence using polyclonal anti-PLAP. Radiolabeled
precursors of PI-G indicate that the PLAP is attached to the plasma membranes
of these cells, and both enzyme activity and radioactivity are released when
cells are treated with PI-PLC. Micanovic et al and Ogata et al both
demonstrated that PI-G attaches to Asp-484 in the protein. This involves
cleavage of a 29-residue peptide from the 513 amino acid-long translation
product. The COOH-terminus of these proteins are relatively hydrophobic; the
NH2-terminus is likewise, since they are secreted proteins and the amino
terminus must code for a signal peptide to direct the translocation across the
membrane. Deletion of the code for the last six residues permits PI-G
attachment processing, whereas further deletions disrupt the process.
Substitution of the hydrophobic residues (Thr503-Leu-Leu-Leu-Leu) with
Ser-Gln-Gln-Gln-Gln shows that both cleavage and PI-G tailing do not occur, but
with Ser-Ile-Ile-Ile-Ile, function is restored, indicating an absolute
requirment for hydrophobicity. It has also been shown that replacing the
hydrophobic sequence with a segment known to be a transmembrane sequence in a
typical membrane-bound protein causes the product not to be releasable by
PI-PLC and does not incorporate PI-G.
<p>
The site of PI-G attachment, called the &omega;-site because it is the last
residue in the fully mature polypeptide, is limited to a few particular amino
acids. Asp-484 is the wild-type, by Gly, Ala, Cys, Ser, and Asn all appear to
be &omega;-sites in other PI-G-tailed proteins. By changing the site for all
amino acids and following PLAP cleavage and PI-G tailing (by immu
nocytochemistry, enzyme activity release after PI-PLC, etc.), it was shown that
amino acids with small side chains are typically preferred. All six different
&omega; sites were generally reactive to release of activity by PI-PLC. There
were large differences in formation of the PI-G tail however; there was more
tailing with the Ser and Asn mutants than with the wild-type Asp; the activity
of Leu and Val was small but detectable. Cells transfected with Asp-484 and
certain mutants were subjected to immunofluorescent staining both before and
after permeabilization; intact cells showed intense fluorescence with the
wild-type and with the Ser mutant, whereas the Trp- and Val-484 mutants only
showed intense fluorescence in permeabilized cells. This shows that these
mutants do not make it to the surface, as they remain unprocessed and in
association with membrane ER.
<p>
Studies of cell-free PLAP polypeptide processing define the steps necessary to
produce mature protein. For these studies, site-specific antibodies are
generated to synthetic peptides of certain regions but encompassing the entire
length of the PLAP translation product. One set of antibodies is generated to
an epitope which only becomes reactive once the NH<sub>2</sub>-signal peptide
is removed. Another reacts with the COOH-terminus of the mature protein, and a
third reacts with the COOH-terminus signal peptide. PLAP mRNA is translated in
rabbit reticulocyte lysates. Addition of rough microsomal membranes then starts
processing: cleavage of amino terminal signal peptide to form the
propolypeptide form, the removal of COOH fragment and tailing of PI-G. These
can be studied using immunoblots with the various antibodies described above.
<p>
A better system for monitoring these changes is to create a mutant where
changes in mass due to signal peptide cleavages or tailing of a glycan
represent significant percentage of the mass of the processed translation
product. ~60% of the internal sequence of prepro-PLAP513 was therefore deleted
(208 amino acids in final length), yet retaining epitopes for antibodies, but
no glycosylation site, catalytic site, or its Cys residues. The translation
product has relative mass of ~28 kDa. When placed in rough ER from CHO cells,
two more products formed, separable on SDS-PAGE and identifiable as the
prepromini, promini, and mini forms. Prolactin shows comparable transitions.
<p>
Another significant finding was the important of context amino acids at the
&omega; site. The two amino acids carboxyl to the site are small in size, but 
smallness is not a factor, at least for site &omega; + 1 since any amino acid 
substitution by Pro had no effect on processing. However, at site &omega; + 2, 
only Gly permitted processing to the mini-PLAP form (tested by translating the 
mRNA in [<sup>35</sup>S]Met with HeLa rough ER present, immunoprecipitating 
with anti-PLAP, then running on a gel to see processing changes). There is a 
similarity between COOH-terminal signal peptide processing and that at the 
NH2-terminus signal peptide: the amino acid at the &minus;1 position relative 
the amino terminal at the mature protein and that at &minus;3 are critical to 
proper cleavage, while the position at &minus;2 can be any amino acid.
<p>
Evidence that PI-G incorporation was occurring in the miniPLAP deletion mutant
comes from several experiments. One is an experiment in which the 29-residue
COOH terminus was removed from the coding. This actually would produce a
translation product with a &omega; site still the same relative to the &omega;
site of the product had the 29-residue fragment not been deleted. After
processing, weight differences among the various prepromini, promini, mini, and
the 3 forms of the even smaller product suggest that PI-G tailing is occurring.
Triton X-114 partition experiments in which the cells are treated with PI-PLC
or not treated also give direct evidence of the presence of a PI-G group. And
another set of experiments with T cell mutants defective in creating the PI-G
anchor site processing shows that transfected cells can not process the
mini-PLAP; cells rescued by transfection have their ability restored.
<p>
As for the reaction mechanism of PI-G tailing, the reaction is clearly a
transamidation, wherein the fully matured PI-G is brought to the &omega; site
by the appropriate transamidase, and the amine (of the ethanolamine moiety) is
brought to react. The reaction has been shown to depend on the presence of high
energy phosphate (ATP, GTP) and Mg2+. It is proposed but not yet shown that a
nucleophilic oxygen in the enzymeÃ¢â,¬â"¢s active site probably starts the cleavage
process.
<!-- ============================================================= -->
<p class="tags">
HBV serotyping
<p class="ref">
 <span class="authors">
Norder H, Courouce A-M, Magnius LO
 </span>
 <span class="year">
(1992)
 </span>
 <span class="jname">
J. Gen. Virol.
 </span>
 <span class="jvol">
1,
 </span>
 <span class="pages">
e72-e77.
 </span>
<p class="reftitl">
Molecular basis of hepatitis B virus serotype variations within the four
major subtypes
<p class="Para1">
HBsAg is predominant protein, forming 20 nm small spherical particles when
the protein is circulating in excess.  It has 226 amino acid and has one
hydrophobic (presumbaly transmembrane) region and two hydrophilic:
a surface exposed region between positions 99 to 168.  The surface exposed
domain is rich in Cys and Pro residues, can be reductively alkylated (indicating
disulfide formation).  The <b>a</b> determinant is in this region and 
is common in all serotypes and two mutually exclusive determinants:
<b>d|y</b> and <b>w|r</b>, with four major subtypes: <i>adw, adr, ayw,</i>
and <i>ayr</i>.  In addition, the <b>w</b> determinant has four subdeterminants
(<b>w1</b> to <b>w4</b>).  There are four serotypes of <b>ayw</b> which include
all four <b>w</b> determinants (<b>ayw1, ayw2, ayw3, ayw4</b>) and two serotypes
of <b>adw</b> (<b>adw2, adw4</b>).  HBV genomes have been classed into six
groups (A-F) based on sequence similarity.  Strains with <b>adw</b> are found
in A, B, C, and F;  with <b>ayw</b> in A, B, D, and E.  Those with <b>r</b>
are only in Group C.  Point-type polymorphisms are associated with subtype
variation.  The <b>d|y</b> determinants are because of allelic variation at
position 122, and <b>r|w</b> at position 160, both involving Lys&rarr;Arg
shifts.  Molecular basis for subdeterminants of <b>w</b> and of other determinants
(<b>g, q, x</b>) is not known.
<p>
HBsAg-positive sera containing HBV DNA from 46 persons was obtained and subtyped
using immunodiffusion.  Whole genomes were sequenced where subtyping gave
aberrant results (codes Tar and Man).  Tar subtyped to both <b>adw4</b> and
<b>adr</b>, and also partial identity with <b>adw2</b>.  Primers were created
to amplify the S gene sequence from amino acids &minus;8 to 198:  hep3 at
positions 738-750 (same as MD03 of Larzul et al 1988) and hep33 at positions
131-146, which is 5&prime;-AGGACTGGGGACCCTG-3&prime;.  Products from three 
tubes were pooled and extracted in chloroform; Magic PCR Preps (Promega) were
used to clean the products  Sequencing was done according the Norder et al 1992
using hep3 as the primer.
<p>
The amino acid sequences from positions 101 to 180 were deduced.  The following 
<i>ayw1 &rarr; ayw2</i> variations were found:  F134Y, T143S, A159G, Y161F, 
V168A.  The <i>ayw1</i> and <i>adw2</i> types do not differ in these five
positions, but do differ here:  <i>adw2</i> &rarr; <i>ayw1</i>: T114S,
K122R, and N131T.  This was true in 7 of the 9 <i>ayw1</i> types found;
in the other two, only the K122R polymorphism was found.  At the very least
the <i>w1</i> subtype requires a Arg-122 polymorphism.  The <i>w3</i> subtype
appears to require a Thr-127 position.  For <i>w4</i> a Leu127 and Ser-140
are required.  In fact the amino acid changes forming <i>w3</i> and <i>w4</i>
involve the change from a Pro to Thr or Leu residue, effectively removing
a beta turn, a significant structural change.
<!-- ============================================================= -->
<p class="tags">
patent, HBV recombinant vaccine
<p class="ref">
 <span class="authors">
Pent&oacute;n E, Herrera L, Muzio V, Ramirez V, Garcia A, Duarte C, Ruiz C, 
Izquierdo M, Perez L, Fontirrochi G, Gonzales M, Nazabal M, Beldarrain A, Padron 
G, Garcia J, Riva G, Santiago A, Ayan F, Paez R, Agraz A, Diaz R, 
Qui&ntilde;ones Y
</span>
 <span class="year">
(1992)
 </span>
<br> <span class="pages">
European Patent Office 480252
 </span>
<p class="reftitl">
Method for obtaining recombinant surface antigen of hepatitis B virus (Hep B) of 
higher immunogenic capacity and use there of in a vaccine preparation
<p class="Para1">
<a href="https://register.epo.org/espacenet/application?number=EP91202615&amp;tab=main">EPO patent title</a>
Recombinant HBsAg is hosted in Pichia pastoris C226.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Littlechild, J.A., Watson, H.C.
 </span>
 <span class="year">
(1993)
 </span>
 <span class="jname">
TIBS
 </span>
 <span class="jvol">
18,
 </span>
 <span class="pages">
36-39
 </span>
<p class="reftitl">
A data-based reaction mechanism for type I fructose bisphosphate aldolase
<p class="Para1">
Type I aldolase is found in most cells; it cleaves FBP into two
C3 sugars, dihydroxyacetone acetone (DHAP) and glyceraldehyde-3-phosphate
(G3P).  The type II enzyme catalyzes the
same reaction but has a metal ion requirment.
In mammalian tissues, three type I isoenzymes exist:
A found in muscle, B found in liver, and C
found in the brain.  They differ in
their reactivities toward FBP and fructose-1-phosphate, with A and C
preferring FBP and B showing little or no preference.
All 3 isoenzymes have four identical subunits, each with a polypeptide
chain ~360 aa.  Each subunit has an active
site Lys residue which forms a Schiff base during the catalytic cycle.
<p>
Work done earlier indicated the involvement of a Cys, His,
a second Lys, and a COOH-terminal Tyr in the catalytic events.
Horecker et al. (
<i>The Enzymes</i>, Vol. 7, P.D. Boyer, ed., pp. 213-258) had proposed a
model whereby the Cys mediates transfer of a proton from the C-4 carbonyl
hydroxyl to the side-chain nitrogen of the 2nd Lys.
His is thought to provide a proton to release the enamine intermediate,
and Tyr is supposed to provide the conformation required for interaction
between the Cys and Lys.  The problem is
that Cys is not at all located near the active site cavities, according to
x-ray diffraction data (Sygusch et al.,
<i>PNAS USA</i> <b>84</b>: 7846-7850, 1987).
Hori and coworkers have extended the x-ray
data by showing that mutagenesis of aldolase to substitute with Ala the two
proposed active site Cys residues at 72 and 338 in the human A enzyme did not
affect the enzyme activity when compared to wild-type.
<p>
The three type I aldolase structures which have been determined (rabbit:
Sygusch et al.; human: Gamblin et al., <i>FEBS Lett</i>. <b>262</b>: 282-286,
1990; <i>D. melanogaster</i>: Hester et al., <i>FEBS Lett</i>. <b>292</b>:
237-242, 1991) have marked homology.  All are tetramers with near, or exact
(human enzyme), 3-fold axis symmetry. The intersubunit contact areas show the
same structure in all three enzymes. Subunit structure is the beta-barrel type
surrounded by alpha-helical and random coil interconnecting loops. The consensus
structure suggests this is true for subunits for all species.  Residues assumed
to be involved in substrate action are directly or indirectly conserved, and
this includes residues thought to be particularly involved in substrate binding.
The side-chain of Schiff base-forming Lys (Lys-229 in human type I) is oriented
toward the center of the 8-stranded beta-barrel. The alpha-helix on the amino
terminus prevents access to the active site from the amino end of the
beta-barrel. The COOH-terminus residues traverse the rim of the open-end of the
barrel.
<p>
Residues involved in substrate binding belong to four categories: (a) those
involved in shuffling protons in either abstraction or donation, (b) those
interacting with C-1 phosphate, (c) those interacting with C-6 phosphate, and
(d) acidic residues which serve to produce a neutral electrostatic potential
around the active-site cavity.  The Schiff base-forming Lys has its
epsilon-amine side chain running parallel to the beta-barrel axis, pointing to
the floor of the active site cavity. Its environment is suitable for the
uncharged, more basic amine group.  A proton is expected to transfer from this
Lys in the Schiff base formation to the C-2 carbonyl oxygen to form the
carbinolamine.  A 2nd Lys is positioned to collect the remaining proton from the
side-chain nitrogen of the Schiff-base forming Lys. (see figure 2).
<p>
The catalytic cycle can be envisioned as starting with binding of either FBP or
DHAP (in the case of gluconeogenesis). The cycle will be described in terms of
neogenesis using the human A enzyme numbering. DHAP binds in the active-site
cleft, in which the uncharged amine of Lys-229 adds to the carbonyl; the
temporary carbinolamine is formed by movement of an amine proton to the carbonyl
oxygen. The more stable imine form is made when Asp-33 donates a proton to the
C-2 -OH group, making it a good leaving group. In the formation of the enamine
from the imine, a second Lys (Lys-146) must be in position to donate a proton to
the Schiff base nitrogen of Lys-229.  At the same time, a proton on C-1 must be
removed.  In some isozymes this C-1 proton is removed by a C-terminal Tyr
brought into position after the substrates bind.  (In less catalytically active
enzymes, water seems to serve as the proton acceptor.) The fully protonated Tyr
phenolic oxygen shows one of its nonbonding pairs to the C-1 proton. As it
accepts the C-1 proton, the proton already bound to it is positioned such that
it is transferred to the C-1 carbonyl oxyen on G3P, thereby activating G3P
toward a condensation.
<p>
The necessity of carboxyl terminal residues in the catalytic activity of rabbit
muscle aldolase was shown long ago when carboxypeptidase was found to inactivate
the aldolase. It was a surprise when Hori and coworkers showed not a loss, but
decrease in activity when Tyr was replaced with Ser.  The B enzyme, moreover,
does not require Tyr at all, losing no activity when its C-terminal Tyr is
replaced.  His-361 in the human A isozyme is observed to interact with both the
1-phosphate and the -COOH group of the C-terminal residue.  These interactions
serve to orient the Tyr residue relative to the substrate. The phenolic hydroxyl
is positioned to abstract a proton from DHAP and provide the one it already has
simulateously to G3P.  For the B enzyme, there is no His group to assist the
Tyr-mediated proton channeling; for mutant A isozyme in which Ser replaces Tyr,
the effect is the same. The reason there is no loss of activity is that water
serves as the acceptor and donator, albeit much less efficiently.
<p>
Even before their x-ray structures showed the position of both Lys residues in
the active site, chemical modification studies had already shown their
importance.  Aldolase which had been incubated with [<sup>14</sup>C]DHAP was
treated with NaBH<sub>4</sub>. Alkylation of Lys-146 with
N-bromoacetylethanolamine phosphate completely abolished the activities of both
FBP aldolase and transaldolase, and it was surmised that it was involved in
binding the phosphate group of the substrate (it is).  Used at pH 6.5, this
affinity label modifies His residues;  it modified a His which was two
residues away from the C-terminal Tyr.  It was found that when His was
alkylated, the Lys could not be, and that when Lys was alkylated, His could not
be;  this supports the mechanism proposed for catalysis.
<p>In this entire mechanism, there is no role for any Cys
residues, as originally proposed.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Goodsell, D. S., Olson, A. J.
 </span>
 <span class="year">
(1993)
 </span>
 <span class="jname">
TIBS
 </span>
 <span class="jvol">
18,
 </span>
 <span class="pages">
65-68.
 </span>
<p class="reftitl">
Soluble proteins:  shape, size, and function
<p class="Para1">
X-ray and NMR-resolved structures now known for &gt;300 proteins which are
unique (the actual number of structures is 850). Shown in this paper are the van
der Waal representations of some of the globular proteins at a consistent
4,000,000 &times; magnification.  They range from the small hydrolases (125 aa)
to the dodecameric Gln synthetase (5500 aa). Proteins in other classes include
very small hormone peptides (tens of aa) to the large dehydrogenase complexes
(60,000 aa) Most enzymes range between 150-450 aa in length.
<p>
Monod et al have given two explantatons for a minimum protein size.  One is the
need to have enough residues to produce the shape of the protein which orients
the active site residues, with small chains being too flexible. This is
supported by findings that disulfide linkages and metal proteins are often found
in the smallest proteins which are shown to be necessary to maintain the folded
state. A second premise (attributed to Pauling) is that proteins can not
associate with other proteins at random in vivo, forming nonfunctional
aggregates.  Protein aggregation is reduced as protein size increases as the
surface area-to-volume ratio falls.  The surface area residues must then make a
decision whether to function in regulation or enzyme localization rather than
subunit association. Another theory is that a larger surface area for the larger
proteins serves as a funnel to the active site, an affordable structural
arrangment since many of the substrates for the largest proteins are very small
and rapidly diffusible.
<p>
Protein shape is of interest. The large oligomeric proteins form beautifully
symmetric assemblies.  Individual proteins fold into distinct globular domains.
Shape is clearly an indication of function. The Y-shape of immunoglobulins
serves their ligand binding function to large forms and the need for flexibility
in the molecule. DNA-binding proteins show long arms with which they can wrap
around the DNA.

<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Murzin, A. G.
 </span>
 <span class="year">
(1993)
 </span>
 <span class="jname">
TIBS
 </span>
 <span class="jvol">
18,
 </span>
 <span class="pages">
403-405
 </span>
<p class="reftitl">
Can homologous proteins evolve different enzymatic activities?
<p class="Para1">
New catalytic functions can emerge in two ways in enzymes: (i) a
new active site may form on the polypeptide framework or (ii) an old active
site can acquire a new function.  The
first relationship is very difficult to prove since similar protein folds with
very different functions are many but the connection is not clear.
Some examples:
mandelate racemase and muconate lactonizing enzyme; nuclease P1
and phospholipase C; fructose 1,6-bisphosphatase and inositol monophosphatase;
and acid phosphatase and phosphoglycerate mutase;
all show distant relationships to each other.
Their principal catalytic residues are
actually conserved, showing how common their chemistry is.
<p>
The enzymes methionyl aminopeptidase (MAP) and creatine
amidinohydrolase (creatinase) are now shown to be different in catalytic
reaction and mechanism.  It should be
noted that the former is a metal ion-dependent enzyme.
<p>
MAP is well known in both pro- and eukaryotes in its
removal of the Met at the start of nascent polypeptides, with MAP structure in
pro- and eukaryotes highly conserved.  The <i>E. coli </i>crystal structure
has been shown:  263 residues with two Co<sup>2+</sup> ions at the active
site.  Although MAP catalytic mechanism is not defined, it is
likely to be similar to other metalloproteases.  One of the two
metal ions surely interacts to stabilize the carbonyl oxygen
to prepare the peptide bond as a tetrahedral intermediate.
<p>
Creatinase hydrolyzes creatine to sarcosine and urea.
Found also in both prokaryotes and eukaryotes, the sequence
and crystal structure has been determined only for
<i>Pseudomonas putida</i>.  It assembles into homodimeric
subunits each 403 residues with two domains.  The catalytic
function is in the COOH domain (last 240 residues), while
substrate recognition and allosteric regulation functions
are found in the NH<sub>2</sub> domain.
<p>
Comparisons of the 3-D structures show the same folding
pattern.  This fold is described as
having internal pseudo twofold symmetry which structurally relates the first
and second halves of the polypeptide chain, a result most likely from an ancestral
gene duplication/fusion event.  Each
half represents a simple &alpha; + &beta; folding motif, and the
&beta;-sheets of the two &alpha;&nbsp;+&nbsp;&beta; motifs associate
in an unusual manner.  The directions of
the strands adjacent to the pseudo-dyad make a positive angle, opposite to that
in &beta;-sheets with the normal twist.  This means that only a
few hydrogen bonds are made between the two strands, shaping the
&beta;-sheet structure into a &ldquo;trough&rdquo; that accomodates the
active site in both enzymes.
<p>
This unique structure in both enzymes is unlikely to have
independently evolved.  The evidence
that they evolved from the same ancestral gene comes from structural alignment
of the pertinent sequences in the proteins.
The common core of the enzymes consists of ~180 structurally equivalent
sites in both the &alpha;-helices and
&beta;-strands; 39 sites (&gt;20%) of these sites
have identical residues, a significant match with a probability of
&lt;10<sup>-10</sup> that it occurs randomly.

<p style="margin:1em auto;">
       <img src="./imagesGeneralScienceHTML/CreatinaseStructure.gif"
   alt="Creatinase Structure and Catalysis">

<p>
The creatinase reaction is somewhat similar to that
catalyzed by MAP and other proteases, and the mechanism is analogous to known
mechanisms of proteases too.  A
significant difference in the mechanisms is that the carbonyl oxygen in the
peptide amide bond can act as a hydrogen or metal bond acceptor, but the
amidino nitrogen cannot.  Rather
creatinase stabilizes the tetrahedral intermediate by the concerted interaction
of several substrate and protein groups.
<p>
The active sites are found in the same site of the protein
framework.  Three common active site
positions are found in MAP and creatinase (CRE in the diagram) and the E. coli
X-Pro aminopeptidase (PEP).  They are all in the second &alpha;+
&beta; motif (Motif II). Two of them are identical in the aligned sequences:
His-171 in MAP and His-324 in CRE, and Glu-204 (MAP) and Glu-358 (CRE).  A
third site is Glu-235 (MAP) and His-376 (CRE).  In MAP these two
residues bind the two Co<sup>2+</sup> ions; in CRE these residues
bind a positively charged group as well:  the guanidinium group in
creatine.  Glu-358 binds the group directly, while His-324 and -376
bind to it through the same water molecule.  Glu-262 also recognizes the
substrate in CRE by direct interaction with the guanidinium, occupying a
position adjacent to the Co<sup>2+</sup>-linked Asp-108 found in MAP.
The similarities in structure and chemistry are consistent with
suggestions they are related, but it is evident that the enzymes
differ greatly as to the way they recognize their substrates.
<p>
The structural similarities also suggest catalytic
mechanism similarities.  His-232 in CRE
plays the main role, corresponding to a conserved His-79 of MAP in the
alignment.  Significant homology is to be found in the &beta;5 region which
contains the Glu204/358, and the consensus in this region was used in the
pattern search of the protein database which discovered other related
proteins:  <i>E coli</i> X-Pro aminopeptidase, and human and
<i>E coli</i> X-Pro dipeptidases.  These enzymes are prolidases,
are very homologous within their family, and are metal-dependent like MAP.
A global (but weak) homology is evident in MAP and the creatinase catalytic
domain.  What is distinct about the prolidases is that alignment without
forcing insertions or deletions presents 25% identity between the
prolidases and MAP (higher that between MAP and creatinase), with
metal-binding residues well-conserved.  Yet prolidases are very
much like creatinase in size, being larger than prokarytic MAP.
Also the X-Pro bond is actually an imide bond, different from the true
amide peptide bond, and sterically similar to the amidine bond in
creatinase.  Hence the prolidases may represent a transition between
MAP and creatinase.  Combining these families of enzymes together
makes up a superfamily consisting of members with highly diverse
functions as amido-, imido-, and amidinohydrolytic enzymes.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Delidow BC, Lynch JP, Peluso JJ, White BA
 </span>
 <span class="year">
(1993)
 </span>
in B. A. White, <i>ed.</i>,
 <span class="jname">
PCR Protocols: Current Methods and Applications
 </span>
Ch. 1,
 <span class="pages">
1-29.
 </span>
<p class="reftitl">
Polymerase Chain Reaction:  Basic Protocols
<p class="Para1">
PCR is described by three steps in the synthesis of DNA: (i) <i>denaturation</i>
into single strands; (ii) <i>annealing</i> of primers for new strand synthesis;
(iii) <i>extension</i> of new DNA from the primers.  This was tried initially
with DNA polymerases that were temperature-sensitive, requiring addition
of fresh enzyme.  With isolation of a heat-stable polymerase from
<i>Thermus aquaticus</i> (Taq), this method became more feasible.
<p>
The only specialized equipment required is a thermal cycler.  In addition,
it is useful to dedicate a set of pipettors to the method.
<p>
Preparing for PCR involves several steps.
<p>
<b>Obtaining primers</b>. Primer selection/determination is important here.
Sequence and orientation should be double-checked. [the method for
removing primers covalently linked to a CPG support and deprotecting with
ammonium hydroxide is omitted, since the form that primers are given
can vary.]
<p>
<b>Isolation of DNA</b>.
For isolating DNA from tissues/cells, use a Dounce homogenizer with a
saline-Tris-EDTA-SDS buffer (0.1 M NaCl, 10 mM TrisHCl pH 8.0, 25 mM EDTA,
0.5% SDS).  Also obtain: 20 mg/ml proteinase K, Tris-buffered phenol
[to recrystallized molec biol grade
phenol add equal vol 1 M Tris base, put in 65&deg; water bath for 1 h,
allow to cool in fume hood, shake to mix phases, open bottle and let vent,
transfer to 50 ml screw cap tubes, spin 2000 rpm 5-10 min, remove upper
phase (aspirate), add eq vol 10&times; TE pH 8 (0.1 M Tris, 10 mM EDTA),
cap and shake and spin and aspirate upper phase, repeat 2-3 times with
eq vols 1&times; TE pH 8 (10 mM Tris, 1 mM EDTA) until upper phase pH
reads 7-8 with pH paper, aliquot and cover with layer of 1&times; TE,
and store at &minus;20&deg;], chloroform, absolute and 70% EtOH,
PBS (20 &times; stock = 2.74 M NaCl, 53.6 mM KCl, 166 mM disodium phosphate,
29.4 mM monopotassium phosphate at pH 7.4 which is sterile-filtered), 7.5 M
ammonium acetate, 10 mg/ml RNase in 10 mM TrisHCl pH 7.5 + 15 mM NaCl heated
at 100&deg; for 15 min then cooled then stored at &minus;20&deg;, and
20%SDS.
<ol>
<li>After weighing the tissue, put it in cold PBS and mince with razor
blade or scissors.  Cultured cells are collected by centrifugation, washed
once in ice-cold PBS, and resuspend in 1 vol PBS.
<li>Transfer to Dounce homogenizer so there is 12 ml buffer per gram tissue
or per ml packed cells.
<li>Use 20 strokes with the B pestle and keep extract on ice.
<li>Sample is transferred to a test tube, and proteinase K added to
100 &micro;g/ml and incubate 50&deg; overnight.
<li>Extract sample twice with eq vol of 1:1 (v/v) phenol:chloroform.
<li>Extract twice with eq vol of chloroform.
<li>Add 0.5 vol 7.5 M ammonium acetate and 2 vol 100% EtOH.  Mix gently.
A stringy precipitate is seen.
<li>Spin 12,000 <i>g</i> 15 min at 4&deg;, and rinse pellet with 70% EtOH,
decant and let dry.
<li>Resuspend in TE pH 8.0, about 7-10 ml/g tissue used.  Putting in a
65&deg; water bath can facilitate the resuspension.
<li>Add SDS to 0.1%, and RNase A to 1 &micro;g/ml and incubate 1 h 37&deg;.
<li>Re-extract with phenol/chloroform, precipitate and resuspend the DNA
as above.  For long term storage, keep DNA in ethanol at 4&deg;.
</ol>
<p>
<b>Isolation of RNA</b>.  Two methods are presented: CsCl centrifugation and
Guanidinium HCl extraction.  Materials required are PBS (as for DNA isolation),
2 ml Wheaton glass homogenizer, guandinium isothiocyanate/&beta;-mercaptoethanol
solution [4.2 M guanidinium isothiocyanate, 25 mM sodium citrate pH 7.0,
0.5% <i>N</i>-laurylsarcosine (Sarkosyl), 0.1 M 2-mercaptoethanol;
prepared stock in dd water except for 2-mercaptoethanol and filter-sterilize
and store in 50 ml aliquots at &minus;20&deg;; when ready to use, add
7 &micro;l/ml 2-mercaptoethanol], 1 ml tuberculin syringes with 21-g needles,
11 &times; 34 mm (Beckman 347356) Ultraclear ultracentrifuge tubes,
97% diethylpyrocarbonate (DEPC) stored at 4&deg;, DEPC-treated water
[fill baked glass autoclavable bottle to 2/3 capac. with dd water, add
DEPC to 0.1%, cap and shake and vent with loose cap and incubate 37&deg;
for 12 h or overnight then autoclave on liq cycle for 15 min and store
at room temp], 0.2 M EDTA pH 8.0 filtered with 0.2 &micro;m filter and
treated with DEPC as for water, molec biol grade CsCl [20 g solid CsCl
in 50 ml tube, add 10 ml 0.2 M EDTA pH 8 DEPC-treated, bring to 20 ml
with DEPC water, mix and filter and store 4&deg;], 1 &times; TE pH 7.4
made in DEPC water which is filtered and autoclaved 15 min on liq cycle,
freshly made TE-SDS [add 10% SDS (in DEPC water) to 0.2% to an aliquot
of TE pH 7.4], buffered phenol (as described in DNA isolation), chloroform,
4 M NaCl [make in dd water and DEPC treat and aliquot], 95% EtOH stored
at &minus;20&deg;, 1.5 ml polyallomer microcentrifuge tubes for
ultracentrifugation (Beckman 357448), 40 U/&micro;l RNasin RNase inhibitor
(Promega) stored at &minus;20&deg;, and an ultracentrifuge.
<p>
In doing RNA work, always wear gloves, use the highest molecular biology
grade reagents, bake all glassware, and use sterile, disposable plasticware.
Make sure the filters used for guanidine are compatible:  Corning filters
have been found to dissolve whereas this is not true for Nalgene.
Note that DEPC is broken down by Tris and cannot be used to treat them.
<p>
The CsCl procedure rigorously removes DNA better than the other:
<ol>
<li>The tissue should be taken within minutes of animal sacrifice.
Place in ice-cold PBS and trim off excess fat.  Try to cut tissue into
2-3 mm cubes.
<li>Place tissue in 2 ml Wheaton glass homogenizer with 1 ml GITC/&beta;ME
buffer. Homogenize until no visible clumps.fs
<li>Transfer to 5 or 15 ml conical tube.  Without causing foaming, draw
the solution in and out with a 1 ml syringe fitted with 18 gauge needle
until it is not viscous and individual drops are seen; this helps to shear
DNA.
<li>Rinse an ultracentrifuge (Beckman Ultraclear) with 300 &micro;l
of GITC/bME and allow to dry inverted.  In the dry tubes, add 875 &micro;l
of CsCl solution into bottom of each tube.
<li>Add 300 &micro;l of GITC/bME to each tissue sample and mix.  Layer
the entire (1.3 ml) sample on top of a CsCl cushion, taking care not to
disrupt the boundary.
<li>Fill each tube with sample and/or GITC/bME to up to 2 mm of the top,
and balance the mass within 0.01 g with GITC/bME.
<li>Load into the buckets of the rotor (Beckman TLS-55) and spin 40,000 rpm
for 3 h at 16&deg;.  This pellets RNA and not DNA.
<li>Decant the tubes by a rapid inversion and place them upside down in
rack or on clean towel to drain-dry 15 min.  Keep inverted till dry.
Use Kimwipes to remove traces of liquid from tube sides but do not touch
the bottom.  The RNA pellet is not visible.
<li>Add 400 &micro;l of TE-SDS to the bottom, but do not add it down the
side.  Cover the tubes and put on rotary shaker.  Solubilize by gentle
shaking for 20 min.
<li>Use a pipettor set at 200 &micro;l and transfer each sample to 1.5 ml
conical tube;  it requires two transfers.  During each transfer, pipet
sample up and down in the ultracentrifuge tube and scrape the tip to the
bottom to ensure RNA pellet solubilization and avoid foaming of the SDS.
<li>To the RNA sample in the 1.5 ml tube add 200 &micro;l buffered phenol
and 200 &micro;l chloroform.  Mix well.
<li>Spin at max speed in microcentrifuge for 2 min.  Move the upper aqueous
phase to clean conical tube, add 400 &micro;l chloroform, mix well, and
spin again.  Do the chloroform extraction one more time.
<li>Transfer to a Beckman ultramicrocentrifuge tube.  Add 25 &micro;l
of 4 M NaCl and mix.  Add 1 ml cold 95% EtOH and mix again.  Leave overnight
at &minus;20&deg;.
<li>Collect by spinning in ultramicrocentrifuge (Beckman TLA-45) at 15,000 rpm
for 30 min at 4&deg;.  Decant and leave inverted on clean tissue for air
drying.  The RNA should be visible as a translucent white pellet.
<li>Resuspend in 25-100 &micro;l TE pH 7.4;  volume will be determined by
pellet size.  Add 1 U/&micro;l RNasin inhibitor and mix to prevent
degradation.
<li>Measure at 260 and 280 nm of about 3-5 &micro;l in 400-1000 &micro;l
sterile water.  The 260/280 ratio should be close to 2.0.  If the
ratio is &lt; 1.7 the sample might have residual phenol or proteins and
needs re-extraction and precipitation.  The mg/ml RNA concentration
is the OD<sub>260</sub> &times; 40 &times; total volume (ml)/RNA volume used
(&micro;l) for the sample.
<li>If storing (several weeks) put in an aqueous solution at &minus;20&deg;.
For longer term storage, put in EtOH.  Add NaCl to 0.25 M in aqueous
solution, add 2.5 vol of 95% EtOH, mix and store at &minus;20&deg;.
</ol>
<p>
In isolating and purifying RNA by the RNAzol method from tissue samples
&lt; 500 mg:
<ol>
<li>for each 100 mg tissue, add 2 ml RNAzol and homogenize with several
strokes in glass-PTFE homogenizer.  When tissue &lt; 50 mg, place tissue
directly in 1.5 ml conical and use a propylene pestle (Kontes).  Very
brief (without heating!) sonication can be done to break up clumps.
If using cells from suspension culture, spin down 50-200 <i>g</i> for 5 min
and remove supntnt; add 200 &micro;l RNAzol per million cells and lyse
by pipetting and vortexing.  For cells grown in a dish, remove medium
then add RNAzol until dish is covered (e.g. 1.5 ml to a 3.5 mm dish/well);
scrape or pipet vigorously, or collect and repipet and vortex.
<li>For each 1 ml homogenate add 100 &micro;l chloroform.  Vortex rapidly
15 s until frothy white, then hold on ice for 15 min.  Vortex again then
spin down 15 min 10,000 <i>g</i> at 4&deg;.
<li>To a preweighed tube, carefully transfer the upper aqueous phase
without collecting any of
the interface.  This is about half the volume of the homogenate.  Precipitate
with equal volume of ice-cold 2-propanol and let sit 45 min at &minus;20&deg;.
Spin 12,000 <i>g</i> 4 &deg; 15 min or at 10,000 <i>g</i> for 30 min.
The white RNA pellet might be visible.
<li>Decant the supernatant and wash pellet in 80% EtOH, adding 800 &micro;l
for each 100 &micro;g RNA (weigh the tube to estimate, if possible).  Vortex
to loosen pellet, then spin 10 min at 12,000 <i>g</i> 4&deg;.  Decant and
repeat wash again.  The pellet may not be attached well, so decant carefully.
Allow pellet to air dry until damp.
<li>Resuspend in ca. 50 &micro;l TE pH 7.4 per 100 &micro;g using vortexing
and repipetting.  Let sit 15-30 min to ensure dissolution.  Use a 60&deg;
water bath 10-15 min if dissolution seems impossible (probably from letting
pellet dry too much).
<li>Measure 260/280, which should be 2.0-2.1.  If &lt; 1.7, re-extract with
chloroform and precipitate.
</ol>
<p>
<b>First strand synthesis</b>.  The primers used can be oligo dT, random,
or even highly specific if a target gene is sought.
<ol>
<li>Put up to 20 &micro;g purified RNA in conical tube with 4 &micro;g of
oligo dT (for poly(A)<sup>+</sup> RNA) or 200 pmol of specific primer.
The oligo dT should be 18-20 bases long, made up as 5 OD units in 180 &micro;l
sterile water giving 1.6 &micro;g/&micro;l.
The volume of RNA + primer can be up to 36.5 &micro;l.
<li>Add 5 &micro;l of 10&times; buffer usually provided by the manufacturer
for the reverse transcriptase.
<li>Incubate 65&deg; for 3 min, then cool on ice.
<li>Add 5 &micro;l of 0.1 M DTT, 1 &micro;l (40 U) of RNasin or suitable
inhibitor, and 5 &micro;l of 1.25 mM dNTPs (final conc. 125 &micro;M).
dNTP stock solution can be made up as 10 mM, pH adjusted to
neutrality with sterile NaOH and using paper; stored as 50-100 &micro;l
aliquots at &minus;20&deg;; working stock of 1.25 mM can be made
up for each dNTP and stored up to 2 wk.
<li>Add 2.5 &micro;l (500 U) of MMLV reverse transcriptase and mix gently.
Final vol should be 50 &micro;l.
<li>Incubate 37&deg; for 60 min.
</ol>
<p>
<b>PCR</b>.  Most all the equipment and area used for PCR should be dedicated
to the performance of PCR when and where possible.  The worker should take
care NOT to store concentrated stocks of target sequences such as recombinant
plasmids in the PCR area or equipment.  PCR is done in three-step cycles:
<ol>
<li><b>denaturation</b>:  usually at 94&deg; for 1-2 min.  Initially a long
denaturation is done for up to 5 min when amplifying targets in genomic DNA.
<li><b>primer annealing</b>:	this is dictated by the melting temperature
of the primers, which can be computed.  Typical annealing temps are 55&deg;
for 1-2 min,  but sometimes greater stringency is required and temps can
be 60-65&deg;.  In cases where mismatching (reduced specificity) is wanted,
temps can be 37-45&deg;.
<li><b>extension/synthesis</b>:  this is the optimal temperature of the
<i>Taq</i> polymerase, typically 72&deg;.  Time of extension will depend
on how long the target sequence, but the polymerase synthesis rate varies
from 2000-4000 bp/min.  To be cautious, allow 60 seconds per kilobase pair
length, and extra time when the length exceeds 3 kbp.
</ol>
<p>
Usually 20-30 cycles is sufficient.  The synthesis is less efficient as
primers and dNTP concentrations falls.  It is for this reason that the
last step of the last cycle is allowed a longer extension to complete
any partial synthesis, from 5-15 min at 72&deg;.  Thermocyclers can be
programmed to store the products at a reduced temperature (4-15&deg;) for
unattended operation.
<p>
All PCR conditions require optimization for a set of primers and templates.
This is fundamental when an initial set of default conditions does not
give good results.  There are six variables to consider in optimization:
<ol>
<li>primer annealing temperature
<li>primer concentration
<li>template concentration
<li>MgCl<sub>2</sub> concentration
<li>extension/synthesis time
<li>cycle number
</ol>
<ol>
<li>Assuming a final reaction volume of 100 &micro;l (50 &micro;l is also
not uncommon), a master mix of 10 &micro;l of 10&times; PCR buffer,
100 pmol primers (upstream and downstream each), and 16 &micro;l of
1.25 mM dNTPs (final 200 &micro;M) is typical.  Account for the MgCl<sub>2</sub>
to be added. Sterile UV-irradiated water
is used to fill up any volumes.  To guard against falling short as a result
of pipetting error, make up a volume that is 1-2 tubes worth
(for perhaps 10 tubes or less) or a 5-10% extra volume for more than
10 tubes worth.
<li>Transfer to the desired volume of sample to be amplified to the
PCR reaction tube, usually a thin-walled tube of 0.2 ml capacity.
If amplifying cDNA first strand, 10% of the amount produced (5 of a 50
&micro;l product) is not unusual, which is produced from about 10 &micro;g RNA.
Amplfication of genomic DNA between 0.1-1.0 &micro;g is common.  Adjust
volumes of the sample so that they are equal in the tubes.
<li>Now add the <i>Taq</i> polymerase (5 U/&micro;l) to the master mix,
2.5 U in 0.5 &micro;l for each reaction.  Mix thoroughly and pulse spin down.
<li>Distribute the correct aliquot of master mix to each sample, taking
care to avoid cross-contamination.  Cap and vortex, and pulse spin down the
tubes.
<li>If lid is not heated, it is necessary to add mineral oil to the top of
each tube.  Re-open the tubes and add a very small amount of light mineral
oil to prevent evaporation.
<li><b>Optional</b>.  It might be useful to add a drop of mineral oil to
each well of the thermal cycler block which will hold a tube.
<li>Run the amplification program according to the principles outlined.
For 1 kb of cDNA, 30 cycles of 94&deg;/2 min, 55&deg;/2 min, and 72&deg;/1 min
with the last step 72&deg;/5 min is typical.  For 2 kb of genomic DNA,
a 94&deg;/5 min initial step, then 30 cycles of 94&deg;/2 min, 60&deg;/2 min,
72&deg;/2 min, then a final 72&deg;/10 min is usual.
<li>Add 200 &micro;l chloroform, which mixes with the mineral oil and drops
into the bottom phase.  DO NOT MIX OR VORTEX!!!
<li>Spin the tubes 30 s at max speed.
<li>Transfer the upper phase to a clean tube.  Add 50 &micro;l of 7.5 M
ammonium acetate and mix.  Precipitate with 375 &micro;l 95% EtOH.  Leave
at room temp 10 min for concentrated samples, and longer (30 min on ice or
in the freezer for less concentrated).
<li>Spin 15 min at max speed in microfuge.
<li>Decant and allow to air dry to dampness.  Resuspend in 20 &micro;l sterile
water.
</ol>
<p>
<b>Analysis of Amplified Products</b>.  Agarose or acrylamide gels are used
to look at products.  Agarose gels are rapid and easy and DNA is recovered
with better yields.  Acrylamide gels are used when better resolution and
size determination is essential.  When using asymmetric PCR, denaturing
(urea-containing) acrylamide gels are used. <b>NOTE:  never use pipettors
in PCR to aliquot concentrated amplified sequences unless a filtered pipet
tip is being used in transfer.  The product could contaminate the pipettor
and affect all reactions!</b>
<p>
<b>Agarose Gel Electrophoresis</b>.  Typically used to check amplification
prior to cloning or labeling work. It can also be used to resolve fragments
with contaminating sequences.
<ol>
<li>A typical minigel is 5 &times; 7.5 cm in size.  The percentage of
agarose to use will be depend on the fragment sizes to be resolved.
A higher percentage is used to resolve fragments of progressively smaller
size.  A 1% gel can be used to resolve fragments from 1000-4000 bp.
A 1.5% gel will resolve well in the 400-1500 bp range.  A 2% gel might be
used from 100-1000 bp.  To make 25 ml [ = 25 cm<sup>3</sup>, which will make
a gel on a plate the size above 25 cm<sup>3</sup> / (5 cm &times; 7.5 cm) =
0.66 cm = 6 mm deep] of a 1% gel, place 0.25 g electrophoresis-grade agarose
in a 50 ml flask or bottle.  Add 25 ml of the proper buffer.  To start
dissolution, heat in a beaker of boiling water, or place in a microwave
at less than full power, swirl every 30 s.  It takes 3-5 min to dissolve.
<li>Let the gel cool a minute.  With some casting setups, it is essential that
the gel not be poured too hot, since it can craze the plastic supports or
components.  In this case have a 55-60&deg; water bath to place the gel
in, and leave it for 5-10 min to equilibrate, then pour to the ready casting
setup.  The comb can be in place:  it should not form any air bubbles on
it if it has been cleaned (extensively with hot water, then some distilled
water, and possibly a quick rinse with methanol, although let it dry).
Check for any air bubbles in the gel, especially with high concentration
gels:  the corner of a Kimwipe/tissue can be used to pop or remove them
usually.
<li>Allow the gel to form and cool for 20 min.  Without removing the comb,
add gel buffer to submerge the gel at least 3-4 mm.  Leave it for a few
minutes, especially if the gel has been outside a buffer for some time, since
the comb teeth may adhere more firmly to the gel and tear.  Now lift the comb
gently from the gel.  Note that 10&times; TBE buffer (TBE is usual) is
0.89 M Tris-borate pH 8.0, with 2 mM EDTA.
<li>For analyzing the PCR product, load about 10% of the volume of the PCR
reaction.  For example, if you resuspended in 20 &micro;l sterile water,
take 2 &micro;l, mix with 8 &micro;l water, and add 2 &micro;l of 6&times;
loading dye [0.125% bromophenol blue, 0.125% xylene cyanol FF, 25% (v/v)
glycerol, 2.5% SDS, 12.5 mM EDTA:  to make 10 ml, mix 1.25 ml 1% bromophenol
blue, 1.25 ml 1% xylene cyanol, 2.5 ml glycerol, 2.5 ml 10% SDS, and 2.5 ml
of 100 mM EDTA].  The mixing of the sample, water and dye can be done
a sheet of Parafilm, but is also done in a tube.  Transfer the mix to the
well carefully.
<li>Run at a constant voltage that is between 1-5 V per cm separation of
the electrode wires.  This for a mini gel whose wires are separated 15 cm,
the voltage might range from 15-75 V.  Typically a voltage in the middle
is chosen, so 45-50 V would be logical.  The faster dye will move to one-half
to two-thirds down the gel in an hour or two (depends on gel percentage).
<li>The gel is now placed in a solution of water containing 0.1 &micro;g
ethidium bromide per ml solution.  However, it is possible to cast the gel
with ethidium bromide made into it at 0.5 &micro;g/ml.  Shake the gel for
10 min.  View on a transilluminator (UV lightbox).  If the background is
high (overall pink background), it will be useful to destain with 100 ml
plain water for 10-30 min...this improves signal.  Put the gel back on
the electrophoresis stand if the bands are not well-resolved; there is not
usually a need to re-stain if electrophoresis is re-started.
<li>If using a film camera, a Polaroid with yellow filter and Polaplan 52
film is used, with a 1 second shutter at f/4.5 (or open all the way).
</ol>
<p>
<b>Southern Blot Hybridization</b>.  Sometimes PCR products cannot be
detected in the gel with ethidium bromide.  Or there is nonspecific
amplification, and another probe would like to be used on amplified products.
This proves the amplified product is related to a known sequence.  The
Southern blot is ideal.
<ol>
<li>Do the agarose electrophoresis as normal, but use a 1% gel with a larger
format.  Load half to all of the PCR product.  Make sure a ladder is used
for calibrating.  Run the gel at 4 V/cm or much lower for an overnight run.
The faster dye should be two-thirds migrated.
<li>Place the gel in distilled water with ethidium bromide and image it.
it is useful to include a ruler in the image since it might be used for
autoradiographed bands.
<li>Place the gel in a denaturing buffer of 1.5 M NaCl, 0.5 M NaOH and shake
gently 30 min.
<li>Remove the denaturing buffer and soak with gentle shaking in neutralizing
buffer of 1 M TrisHCl pH 8, 1.5 M NaCl.
<li>While the shaking is going on, prepare the blotting setup.  Lay down
a glass plate or gel box cover from edge to edge, sideways, into a pan or
used gel box.  Into the pan, or into each side of the gel box,
pour 10&times; SSC (dilute 20&times; SSC by two: 20&times; SSC is 3 M NaCl,
0.3 M sodium citrate pH 7.0, sterile-filtered and stored room temp)
to a 4 cm depth.  Cut 3MM paper large enough to cover the platform created
by the plate, and long to dip well into the SSC on either side.  Wet the
squares one at a time and lay them across the platform, one on top of the
other, with no air bubbles.
<li>Use a clean sharp blade to cut the gels across the wells.  1-2 mm from
the sizes and the bottom of the gel can be cut, where the agarose slopes
upwards.  Also the markers lane can be cut.  Place the neutralized gel
<em>face down</em> on the blotting paper on top, with no air bubbles between
gel and blot.
<li>Cut nitrocellulose to the exact dimension of the gel.  Wet it in
distilled deionized water, then soak for a few minutes in 2&times; SSC.
Place it to fit on the gel, with no air bubbles in between.
<li>Cut two pieces of 3MM to fit the dimensions of the gel and soak them
in 2&times; SSC, and lay them on top of the nitrocellulose, with no bubbles.
Placing them one at a time is useful.
<li>Cut a stack of paper towels to fit the size of the gel.  Pile them on top
of the filter papers to a 6-8 cm height.  Place a glass or plastic plate to
weigh them down.
<li>Add more 10&times; SSC to the pan if necessary to maintain upward
capillary flow.  Let it continue overnight if necessary.
<li>Remove plate, paper towels, and the filter paper.  Lift off the
nitrocellulose carefully peeling from one corner.  Lay down on clean
filter paper with the side facing the gel now face up.
<li>To fix the DNA to the nitrocellulose, place blot on the filter paper
in Stratagene Stratalinker UV crosslinker and cross link with automatic
program (or place between two 3MM layers and bake at 80&deg; for 2 h in
vacuum oven).
<li><b>Hybridization</b>.  Put blot in a sealable hybridization bag
and seal on 3 sides.  Add 10 ml prehybridization buffer, and squeeze out
bubbles before sealing.  Prehybridization buffer is 50% formamide,
5&times; Denhardt's, 5&times; SSPE, 100 &micro;g/ml denatured salmon sperm
DNA, and 0.1% SDS.  50&times; Denhardt's is 1% Ficoll,
1% polyvinylpyrollidine, 1% BSA, filtered in 0.2 &micro;m filter,
stored at &minus;20&deg;.  20&times; SSPE is 3.6 M NaCl,
200 mM monosodium phosphate pH 7.4, 20 mM EDTA, filtered and stored at RT.
Denatured salmon sperm DNA is made to 10 mg/ml in water and stirred RT for
several hours, sheared through 18 gauge needle, then incubated 10 min in
boiling water bath, aliquotted and stored &minus;20&deg;, and each aliquot
sonicated 30 s before using first time.  Formamide should be deionized and
molecular biology grade:  For each 100 ml formamide mix with 10 g a mixed-bed
resin like Bio-Rad AG501X8 and stir 30 min room temp, filter through Whatman
#1 and store at less than &minus;70&deg; in aliquots.
<li>Leave the bag in 42&deg; water bath for 6 h, with gentle shaking.
<li>Using 200 ng plasmid DNA, prepare a <sup>32</sup>P-labeled probe;
make sure unincorporated nucleotides are removed from the probe.  Count
3 &micro;l to get the cpm/&micro;l.
<li>When prehybridization is complete, take 10<sup>7</sup> cpm probe and
denature in boiling water bath for 5 min.  Chill on ice.
<li>Cut open a corner of the hybridization bag, add the probe, and re-seal
the bag.
<li>Place back in the 42&deg; bath and leave for 2 days.
<li>Open the bag.  Recover the probe solution:  it can be re-used once more
without denaturation/reboiling, stored at &minus;20&deg;.  Wash twice for
15 min RT in high-salt wash buffer with gentle shaking.  Then wash once
in low-salt wash buffer at 55&deg; for 1 h, with shaking.  High-salt wash
is 2&times; SSC + 0.1% SDS.  Low-salt wash is 0.1&times; SSC + 0.1% SDS.
<li>Wrap washed blot in plastic wrap.  Expose to X-ray film for 16 h to 1 week.
</ol>
<p>
In making prehybridization and blotting solutions for use in RNA blots, make
up in RNA-grade solution, with SSC and SSPE treated with DEPC.  Denhardt's
should be made up on DEPC-treated water.
<p>
<b>Nested PCR Analysis</b>.  This re-amplifies an internal portion of the
amplified product.
<ol>
<li>The amplified products should be put on electrophoresis using a 0.7-1.0%
low-melting agarose.  Run the gel at a lower voltage (0.5-2 V/cm) at
4&deg;.
<li>Stain with ethidium bromide and locate the band of interest.  Excise
and transfer to conical tube.
<li>Melt at 68&deg; for 5-10 min.
<li>Transfer 10 &micro;l directly to a tube with a 2nd PCR mix with 100 pmol
of the nested primers and reamplify.
<li>Examine the products of the second PCR with agarose gel.
</ol>
<p>
<b>Acrylamide Gels</b>.
<ol>
<li>Set up glass plates with 1.5 mm spacer.
<li>Prepare 5% acrylamide in 1&times; TBE.  Use acrylamide stock, 10&times;
TBE to make up the solution.  Use a 6&times; gel loading buffer to prepare
samples.
<li>Run the gel at 100-125 V constant for 3-4 h (depends on gel length).
The dye should be 2-3 cm from the bottom.
<li>Line a shallow dish larger than the gel in a staining tank with plastic
wrap.  Add 300 ml dd water into plastic wrap.  Place the gel in the water.
<li>Add 10 &micro;l per 100 ml water of a 10 mg/ml ethidium bromide solution
and shake 10 min.  Remove the staining solution.
<li>Lift the gel onto a UV transilluminator with the plastic wrap.  Smooth
out folds and image the gel.
</ol>
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Haycock, J. W.
 </span>
 <span class="year">
(1993)
 </span>
 <span class="jname">
Anal. Biochem.
 </span>
 <span class="jvol">
208,
 </span>
 <span class="pages">
397-399.
 </span>
<p class="reftitl">
Polyvinylpyrrolidone as a blocking agent in immunochemical studies
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
R. Grabowski, A. E. M. Hofmeister, W. Buckel
 </span>
 <span class="year">
(1993)
 </span>
 <span class="jname">
TIBS
 </span>
 <span class="jvol">
18,
 </span>
 <span class="pages">
297-301.
 </span>
<p class="reftitl">
Bacterial <span style="font-variant:small-caps">l</span>-serine
dehydratases: a new family of enzymes containing iron-sulfur clusters
<p class="Para1">
Proteins with iron-sulfur clusters are widespread and include
the ferredoxins and enzymes participating in redox chemistry.
But Fe-S proteins not limited to electron carriers.  The simple dehydratases
(hydro-lyases, EC 4.2.1) and the flavin-containing hydroxyacyl-CoA dehydratases,
and three other enzymes contain Fe-S.  Many of these enzymes have counterparts
with either no prosthetic group or a different one catalyzing identical or
related reactions.  The function of Fe-S in aconitase remains unknown.
<p>
The l-serine dehydratase from the strict anaerobe
<i>Peptostreptococcus asaccharolyticus</i> was recently found
to have an Fe-S.  Nearly all serine and threonine dehydratases
identified up now were all pyridoxal phosphate (PLP)-dependent.
The dehydratases catalyze the irreversible
deaminations of serine and threonine to pyruvate and &alpha;-ketoglutarate,
resp. (ammonium ion is the other product).
The &alpha;-ketoacids can participate in other metabolic pathways.
The deamination of threonine by dehydratase
is an important step in isoleucine synthesis.
The dehydratase name came from the initial removal of water, which is
used later to hydrolyze the imine to produce the &alpha;-ketoacid.
The bacterial serine dehydratase is quite
specific for Ser (most do not touch Thr), while other dehydratases are less
specific about whether it is Ser or Thr.
<p>
The PLP-dependent mechanism is well-established.
A Schiff base forms with a lysine residue of
the apoenzyme having the consensus (G)S(F)<b>K</b>(I)RG.
During the catalytic cycle, PLP forms a
Schiff base with the hydroxyamino acid, the proton on the
&alpha;-carbon is lost (carbanion stabilized by PLP).  The -OH
group leaves; the resultant enamine is liberated by transamination,
tautomerizes to the ketimine which hydrolyzes to the &alpha;-ketoacid and ammonia.
<p>
Stereospecific protonation at carbon 3 with retention of
configuration is believed to occur while bound to PLP.
The required conversion of electrophilic C3
into a nucleophile is helped by including the phenolate
ion of PLP in the &pi;-system.
<p>
PLP-dependent Ser and Thr dehydratases have all been shown
to be monomers, homodimers, or homotetramers.
Enzyme inactivation is achieved by using carbonyl reagents known to
interfere with PLP (NaBH<sub>4</sub> to reduce the Schiff base;
phenylhydrazine attack of S. base to form PLP-phenylhydrazone).
<p>
An oxygen-sensitive l-Ser dehydratase found recently in
<i>P. asaccharolyticus</i>. Inactivated upon air exposure; reactivated
with Fe<sup>2+</sup> under anaerobic conditions.  Composed of two
subunits, 25 and 30 kDa; which form heterodimers or hetero-octamers.
No PLP found:  stable in high concentrations of sodium
borohydride and phenylhydrazine under anaerobic conditions.
Has 3.8 mol Fe per 55 kDa protein, 5.6 mol acid-labile sulfide
per 55 kDa, which can form 4 Fe-4 S cluster per heterodimer.
Electron paramagnetic resonance studies indicate the cluster
is involved in substrate binding and that air inactivation causes
formation of a 3 Fe-4 S cluster.
<p>
Stereochemistry of the Fe-S l-Ser dehydratase reaction,
namely, side of enamine intermediate to which proton is
added was done using Thr as substrate
(k<sub>cat</sub>/K<sub>m</sub> for Thr is 500X lower than for Ser).
63% inversion/37% retention observed, compared to 96% for
PLP-dependent reaction using a Clostridial enzyme.  Reaction
of protonation occurs during release of substrate just after
dehydration, unlike with PLP.
<p>
The mechanism in aconitase action has been recently worked out (1988-9).
It catalyzes the reversible dehydration of citrate to cis-aconitate,
then hydration to (2R,3S)-isocitrate.  Inactivation of the
[4Fe-4S]<sup>2+</sup> cluster to [4Fe-4S]<sup>3+</sup> which loses
an Fe<sup>2+</sup> to form a [3Fe-4S]<sup>+</sup> occurs with
air exposure.  Activity can be regenerated with Fe<sup>2+</sup>
under reducing conditions.  Mï¿½ssbauer and EPR spectroscopy have
shown that the reversibly removable iron atom is coordinated to
a hydroxyl group, while the other 3 Fe atoms are bound to Cys residues.
Citrate as well as isocitrate are coordinated to the cluster via
their OH and the &alpha;-COO<sup>-</sup> groups (evidence by
ENDOR spectroscopy).  Elimination of the OH group is
facilitated (see diagram).
<p style="margin:1em auto;">
  <img src="./imagesGeneralScienceHTML/IronSulfurComplex.gif"
   alt="Iron sulfur complexes">
<p>
With the Ser dehydratase, the mechanism might be similar, but
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
DePamphilis ML
 </span>
 <span class="year">
(1993)
 </span>
 <span class="jname">
J. Biol. Chem.
 </span>
 <span class="jvol">
268,
 </span>
 <span class="pages">
1-4.
 </span>
<p class="reftitl">
Origins of DNA Replication in Metazoan Chromosomes
<p class="Para1">
<b>Minireview</b>.  DNA replication has been studied in several 
&ldquo;simple&rdquo; eukaryotic genomes such as the protozoa 
(<i>Tetrahymena</i>), yeast (<i>Saccharomyces</i>), slime mold 
(<i>Physarum</i>), and also in animal viruses, mitochondria, prokaryotic cells, 
plasmids, and bacteriophage.   All results show that specific proteins bind 
specific DNA sequences, and initiate unwinding at or near the binding site. 
Sequences of 50 to 1000 bp long encompass the origin of replication, and are 
<i>ori</i>.  This is synonymous with ARS, for &ldquo;autonomously replicating 
sequence,#148; which alone provides the ability to confer replication in cells 
or extracts with the required proteins.
<p>
In <i>Xenopus</i> there appears to be no consensus sequence for initiation, with 
an ability to initiate replication at seemingly random sites.  Replication 
bubbles are found at tandem repeats of histone genes in early Drosophila 
embryro.  This phenomenon may be unique to embryos in the cleavage stage 
however.  Mouse embyros fail to start replication in this way.  Towards the end 
of rapid proliferation, initiation appears to be more selective.
<p>
Mapping sites has been studied by several methods using the dihydrofolate 
reductase gene locus in Chinese hamster ovary (CHO) cells.  CHO C400 contain ca. 
1000 tandem copies of the DHFR gene.  Cells were synchronized and released into 
S phase with nascent DNA allowed to form in intact cells and cell extracts. 
Three initiation sites were identified, <i>ori</i>-&alpha;, -&beta;, and 
-&gamma;, and experiments were directed to the 12 kb fragment <i>ori</i>-&beta;. 
This was narrowed to 4.3 kb, then to 1.8 kb.  Another method used psoralen to 
UV-crosslink the two DNA strands, which prevents the advance of replication 
forks and block expansion of bubbles.  Hence the earliest labeled fragment found 
was only 500 bp.  The origin found cannot be dismissed as an artefact of 
syncrhonization nor of the special case of using amplified or unique DNA. 
Measuring the length of unique nascent DNA strands eliminates that concern, 
since if replication is bidirectional, the origin is at the center.
<p>
Most replication forks are bidirectional and start from a site 500-2000 bp long. 
This means that metazoan chromosomes are not unlike the lower forms when it 
comes to replication.  On the 3'&rarr;5' template, sythesis is continuous.  But 
the 5'&rarr;3' template requires short RNA-priming Okazaki fragments.  New 
strand synthesis and completion is followed by nucleosome assembly, with histone 
octamers ahead of the fork being distributed randomly to both arms as the fork 
passes.
<p>
Okazaki fragments have been obtained and purified, annealed to separate strands 
of restriction fragments representing the various segments of the 
<i>ori</i>-&beta; region.  80% of forks emanated from a 0.45 kb site downstream 
of the DHFR gene.  Origins of bidirectional replication have been identified in 
SV40 and polyomavirus replicating DNA using the same method, to a 2- or 22-bp 
region coincident with <i>ori</i>.
<p>
Emetine or cycloheximide are general inhibitors for protein synthesis and will 
preferentially inhibit Okazaki fragment synthesis in exponentially proliferating 
cells by an unknown mechanism.  Replication forks enriched in forward arms 
contain long nascent strands, on the retrograde arms At least 85% of forks in 
the DHFR ori-&beta; region came from an OBR within a 14 kb segment encompassing 
initiation sites identified in previous studies.  Forks also came from the 
ori-&gamma; and -&alpha;.  The same OBR can be found in CHO cells with 2 single 
copies of the origin and in CHO C400 cells with ~1000 tandem copies.
<p>
Two-dimensional gel electrophoresis of total DNA from S-phase cells can be used 
to analyze replication bubble and fork structures.  OBRs can be mapped to with 
&plusmn;300 bp of an ARS element.  Unfortunately under some conditions, 
replication appears to begin within a wide area at random sites, but in vivo, 
there appears to be more specific selection.
<p>
A large segment of the ori-&beta; identified as containing an OBR can be 
inserted and confer replicability to a plasmid without that ability. It is just 
that the specific sequence cannot be narrowed down yet.  There may be other 
factors that determine the creation of the replication bubble, and which forks 
are successful.  These factors are likely nucleoprotein, or different 
nucleoproteins having differential affinities for different sequences.  Metazoan 
DNA must first be organized into chromatin within a nuclear structure before DNA 
replication can commence.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Dieffenbach CW, Lowe TM, Dveksler GS
 </span>
 <span class="year">
(1993)
 </span>
 <span class="jname">
Genome Res.
 </span>
 <span class="jvol">
3,
 </span>
 <span class="pages">
S30-S37.
 </span>
<p class="reftitl">
General Concepts for PCR Primer Design
<p class="Para1">
<b>REVIEW</b>.  PCR involves certain commercially derived components: the
enzyme (usually recombinant), and ultrapure and buffered dNTPs.  But two 
components come from the researcher, the primer and template.  In designing 
primers there is a balance between specificity, reflected by frequence at which 
mispriming occurs, and efficiency of amplification.  Poorly specific primers often
show extra unrelated and undesirable amplicons.  How this balance is approached
depends on application:  PCR for medical diagnostics would design parameters and
reaction conditions are set for higher specificity and lower efficiency, since 
avoidance of false positives is a high priority.
<p>
Primer length usually affects specificity, along with annealing temperature, which
are related.  Oligos between 18 and 24 nt are very sequence specific.  When the 
primer gets too long, there will be fewer read primed templates.  This decreases
efficiency.  Generally primer design tries to get <i>T</i><sub>m</sub> values 
of &ge; 54&deg; and is a good balance.  Oligos &lt; 15 bases long are useful
as random short primers for simple genome mapping and subtraction library protocol
(see Williams, Lang, Pardee).  With respect to entropy, shorter primers anneal
faster.  Primers of 28-35 nt length are necessary if amplifying sequeneces where
degree of heterogeneity is expected.  When sequences of closely related molecules
(isoforms of protein or family of proteins; cloning of homologous gene from 
different species) and where sequences of viruses (like HIV-1) in which set of
primers with perfect complementarity to all templates is not expected is where
these longer primers are useful.  In these cases software compares all related
sequences and then finds region with least sequence variability for making primers.
In many cases the sequences of known functional domains in the translated sequence
should be exploited.  Untranslated sequences should be avoided.
<p>
The primer 3&prime; end placement is critical.  Where a conserved amino acid is 
known, the first 2 bases of the codon (or 3 bases if known to be encoded by 
amino acid with just one codon:  Met and Trp), then these should serve as 
3&prime; end. The 3&prime; end should be as perfectly base-paired as possible. 
Very little mismatch should exist in the last 5 or 6 bases, if at all possible. 
It is unwise to compensate for mismatching by reducing annealing temperature as 
results do not improve.
<p>
Primer lengths can increase a lot when elements are introduced to extend 
functionality.  Addition of T7 RNA polymerase binding sit, restriction sites,
GC clamp.  Adding unrelated sequences to 5&prime; end should not alter 
annealing of sequence-specific part of primer.  When a significant number of bases
are added to primer not matching template sequence, 4 to 5 cycles of amplification
can be done at <em>lower</em> annealing temp then the rest of the cycles at the
annealing temperature, calculated so that the 5&prime; end is now assumed to be
in the template.  Note that in establishing annealing temperature, for primers
shorter than 20 bases, the <i>T</i><sub>m</sub> = 4(G + C) + 2(A + T) can be used,
and a nearest neighbor calculation must be used for longer primers.
<p>
The 3&prime; end is the important end:  its matching is important to avoid
mispriming.  Another reason is that complemtarity of primer 3&prime; ends 
leads to primer dimers, and these are detrimental to amplification.  Particularly
with multiplex PCR where multiple primer pairs are present simultaneously, this
can be a problem.  Note that when hot start is used, primer-dimer product formation
is greatly reduced.
<p>
Oligos with 20 base long length and 50% G+C contentn have <i>T</i><sub>m</sub>
values from 56-62&deg;.  For a pair of primers, each should have well matched
GC content and <i>T</i><sub>m</sub> values.  When one of the primers has a 
significantly lower <i>T</i><sub>m</sub> there is a loss of specificity, while
the primer with higher <i>T</i><sub>m</sub> may misprime.  If annealing temperature
is too high, the lower <i>T</i><sub>m</sub> primer may not anneal at all.
<p>
Primer design software tyically makes requests for the length of the amplicon 
product, as it impacts amplification efficiency.  The length of amplicon in part
depends on the template.  In clinical diagnostics, PCR products from 150-1000 bp
size are chosen, but size depends on application.  Thus a clincal assay may choose
a product size of 120-300 bp as optimal.  Products are generally in range of 250 
to 750 bp range for quantitative RNA PCR; in this case there is balance between
efficiency of reverse transcription and PCR.  In particular, primer pairs should
stay inside the translated sequence region of mRNA.  In addition primer pairs 
should be on different exons; in this case RNA-specific PCR product will have 
different size whe there is contaminating DNA.  PCR is often used to clone a gene
or cDNA, in which case the size cannot be varied.
<p>
<b>Nested PCR</b>.  There are many situations in which problems of quantity and
quality of template are a problem.  In these cases, nested PCR is useful.  
The sample is first amplified for 20-30 cycles with an outer primer pair set,
then a small aliquot of reaction amplified for 15-25 cycles using inner primer set.
<!-- ============================================================= -->
<p class="tags">
HBsAg, mAb ELISA
<p class="ref">
 <span class="authors">
Toplikar E, Carlomagno A, Rojkin LF, Gariglio R, Lorenzo LE
 </span>
 <span class="year">
(1993)
 </span>
 <span class="jname">
J Clin Lab Anal
 </span>
 <span class="jvol">
7,
 </span>
 <span class="pages">
324-328.
 </span>
<p class="reftitl">
Development of an enzyme immunoassay for the detection of hepatitis B surface 
antigen employing monoclonal antibodies
<p class="Para1 abstract">
ABSTRACT. The use of monoclonal antibodies in the development of a third 
generation microtitration plate enzyme immunoassay for the detection of 
hepatitis B virus (HBV) is described. These antibodies were tested either alone 
or in combination with polyclonal antibodies raised in rabbit. Horseradish 
peroxidase was employed in the conjugates and 3,3',5,5'-tetramethylbenzidine was 
used as the enzyme substrate. Effect of type and concentration of antibodies 
used in the coating solution and in preparing conjugates as well as reaction 
time are discussed. The designed test employs a 100-microliters sample, with an 
overall incubation time of 3.5 hr. Serum or plasma (EDTA or citrate as 
anticoagulant) can be used. The sensitivity limit of the test was 0.4 ng/ml for 
subtype ad and 0.5 ng/ml for subtype ay. When used as screening test, 99.6% 
specificity was obtained; predictive value was 97.5% for a positive result and 
99.8% for a negative result. According to the performance of the test, it seems 
to be suitable for diagnosis in routine laboratory and screening in blood banks
<!-- ============================================================= -->
<p class="tags">
HBV, HCV, diagnostic markers
<p class="ref">
 <span class="authors">
Tr&eacute;po C, Zoulim F, Alonso C, Petit M-A, Pichoud C, Vitvitski L
 </span>
 <span class="year">
(1993)
 </span>
 <span class="jname">
Gut
 </span>
 <span class="pages">
S20-S25.
 </span>
<p class="reftitl">
Diagnostic Markers of Viral Hepatitis B and C
<p class="Para1">
Markers fall in two classes: diagnostic or prognostic and infection assessement.
HBsAg, anti-HBs and anti-HBc should be simultaneously assessed for diagnosis.
HBsAg specificity can be assessed by testing for anti-HBc or HBeAg/anti-HBe.
When anti-HBc exists without anti-HBs, PCR for HBV DNA should be assessed.
Potentially additional markers are pre-S1, pre-S2 and X.  Anti-polymerase
antibodies usually exist in earliest phases but can be an indicator of fulminant
hepatitis and correlate to ongoing HBV replication.  Since anti-pol Abs 
correlate with HBeAg and anti-HBe, those are suitable markers too.
<p>
HBxAg is likely involved in the pathogenesis of HBV since increased ALT
and HBxAg are correlated.  Anti-HBx is detectable earlier and found in both
acute and chronic infection.  Seroconversion to HBxAg occurs earlier than
HBeAg.
<p>
Pre-S Ag is found at greater levels when HBV DNA is positive than when it is 
negative.  Pre-S1 and Pre-S2 domains likely interacct with hepatocyte
receptors.  HBV may interact throught polymerized HSA via pre-S2 and on the
hepatocyte membrane via pre-S1.  Pre-S1 is 174 amino acids long but aa 21-47 
could interact with receptor binding site.  A mAb radioimmunoassay has been
developed.  there is no commercial system of anti-pol, HBxAg, anti-HBx, preS1,
and preS2.
<p>
Pre-core defective HBV variants positive for anti-HBe chronic hepatitis 
have been reported.  The mutation inhibits HBeAg expression.  The anti-HBe
positivity could be due to previous infection with wild type, or the response
is to degraded core proteins.
<!-- ============================================================= -->
<p class="tags">
DNA isolation, agarose gel
<p class="ref">
 <span class="authors">
Wang Z, Rossman TG
 </span>
 <span class="year">
(1994)
 </span>
 <span class="jname">
Nuc. Acids Res.
 </span>
 <span class="jvol">
22,
 </span>
 <span class="pages">
2862-2863.
 </span>
<p class="reftitl">
Isolation of DNA fragments from agarose gel by centrifugation
<p class="Para1">
Problems with DNA extraction from agarose gels include low recovery rates, 
insufficient removal of inhibitory substances, coprecipitation of agarose and
DNA which makes it difficult to resolubilize DNA.  DNA fragments in relatively
low percentage agarose 0.3-0.6% will migrate out of gel with centrifugal force
if gel is blocked with material during centrifugation.  In this case, the block
is with Sephadex G-10 particles.  The spin columns are one 2-ml and on 0.5-ml
conical tubes with caps removed, tiny bit of sterilized glass wool and 150 
&micro;l swollen G-10 particles.  Spin column is made by heating needle or pin
with Bunsen burner, lightly stabbing needle into the center of the bottom of
the 0.5 ml tube to make small hole, not bigger than 1 mm.  Glass wool is then 
stuffed into the hole using pipet tip to block it. 150 &micro;l swollen G-10
beads put on top of glass wool.  Now insert the 0.5 ml conical into 2 ml tube
and a spin column is formed.  To remove water from the G-10, spin tube at max
speed for 2 min.  Put the 0.5 ml conical in another clean 2 ml conical.  Place
the gel on top of the Sephadex:  if separation permits, try to use a mini gel
with 0.3-0.6% agarose; a [DNA] which is 1 &micro;g/&micro;l in the load, and
a short time (&lt; 2 h).  This helps recovery and quality of the DNA.  Cut out
with clean blade with long wave UV.  Avoid too much gel.  Within the tube, mash
the gel within the tube; the Millipore MF filter forceps works well for transfer
and mash.  Spin column for 10 min at max speed.  Transfer DNA solution to clean
1.5 ml tube, precipitate with 0.1 vol 3 M sodium acetate pH 5.2 and 0.7 vol 
ice-cold isopropanol.  Wash pellets twice with ice-cold 70% EtOH and then dissolve
in TE.  In general this method extracts 30-70% from the gel.  Recovery with this
method seems to be about 50%.  This seems to be efficient for all sizes.
<!-- ============================================================= -->
<p class="tags" id="orlando-1994">
receptor-associated protein (RAP)
<p class="ref">
 <span class="authors">
Orlando RA, Farquhar MG
 </span>
 <span class="year">
(1994)
 </span>
 <span class="jname">
Proc. Natl. Acad. Sci. USA
 </span>
 <span class="jvol">
91,
 </span>
 <span class="pages">
3161-3165
 </span>
<p class="reftitl">
Functional domains of the receptor-associated protein (RAP)
<p class="Para1">
Domains of the RAP polypeptide that bind to gp330, a member of the LDL receptor
family, were identified by construction fusion to GST.  RAP is 323 amino acids
long, and four overlapping regions were cloned:  1-113, 1-245, 1-260, and
85-323.  A direct solid phase binding assay was used to assess interaction.
Fusions were expressed in bacteria (<i>E coli 71/18</i>) in a pGEX2T vector
under IPTG induction.  Glutathione affinity columns were used to purify 
constructs and characterized on SDS-PAGE for size and purity, which was &gt; 95%
in all fusions.  RP-HPLC was also used to verify gel purity results.
<p>
Fusion proteins were immobilized on protein binding plates and incubated with
radioiodinated gp330.  The protein bound to both the full length RAP (1-323)
and to RAP 85-223, with similar molar amounts and affinities.  The RAP 1-245
had a lower affinity, possibly because of tertiary structure changes.  Thus the
binding site is between residues 85 and 245.  In a competition study, 
a fixed concentration of <sup>125</sup>I-gp330 is incubated with varying amounts
of full-length (non fusion) RAP and the fusion proteins immobilized to plates:
the bound gp330 was then determined.  Competition was clearly dose-dependent
and decreased with increasing [RAP 1-323].
<p>
New fusion constructs were created amplicons in the range 111-248, 111-184,
178-248, and 85-148.  Again the radiolabeled binding study was done: gp330
bound to both RAP 111-248 and 178-248.  Little or no binding ws seen with 111-184,
indicating that the site of interaction is now within the 70 amino acids between
positions 178 and 248.  Fusions were transferred to PVDF membrane and tested
for binding as well.  It was shown that RAP 1-113 and 85-148 constructs bound
to radiolabeled gp330, indicating that a second binding site is present at 
positions 85-148.
<p>
As RAP also was purified by heparin affinity chromatography, and also using
ligand blotting assay, the heparin binding site was identified.  Structural
homology analysis with other heparin binding motifs suggests that the binding
region is between positions 287 and 306.  Fusions were electrophoretically 
separated and heparin overlays performed to identify glycosaminoglycan binding
site.  Radiolabeled heparin was found to bind to RAP 85-323, RAP 1-323 (full
length), but not RAP 1-113 or RAP 1-260.  This shows that heparin binding is
in the 63 amino acids between positions 260 and 323.  Coincubating high 
concentration of hepain with radiolabeled heparin competed with this binding,
while chondroitin sulfate did not compete.
<p>
The structure of gp330 itself has a number of negative residues (Glu and Asp)
between positions 215-248, which also contains a putative N-glycosylation site.
It might also coordinate the divalent cation in Ca<sup>2+</sup>-dependent 
binding.  Within RAP on the N-terminal side of the gp330 binding site between
positions 85-148, are a series of Leu residues separated by 3- to 5-residue
spacing, constituting a leucine zipper:  these motifs are present in DNA-binding
proteins and promote homodimer formation.  It is possible that RAP forms a dimer
in the ER to promotve oligomerization of gp330-RAP heterodimers.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Arrondo JLR, Castresana J, Valpuesta JM, Go&ntilde;i FM
 </span>
 <span class="year">
(1994)
 </span>
 <span class="jname">
Biochemistry
 </span>
 <span class="jvol">
33,
 </span>
 <span class="pages">
11650-11655.
 </span>
<p class="reftitl">
Structure and Thermal Denaturation of Crystalline and Noncrystalline
Cytochrome Oxidase as Studied by Infrared Spectroscopy
<p class="Para1">
Cytochrome oxidase from beef heart mitochondria was prepared in native form or 
as two-dimensional crystals.  Protein was 90% crystalline according to EM images 
and consisted of dimeric forms; these were oriented inside-out with respect to 
inner membrane of mitochondria.  Specific enzyme activities were nearly 
identical for crystalline and noncrystalline preparations (400 vs. 420 mol cyt. 
<i>c</i><sub>ox</sub> min<sup>&minus;1</sup> [mol enzyme]<sup>&minus;1</sup>], 
resp.).  Samples were prepared in 10 mM phosphate buffer in H<sub>2</sub>O and 
D<sub>2</sub>O solvents (pH or pD 7.0).  Protein concentration was &asymp;40 
mg/ml (&asymp;0.2 mM).  FT-IR data corresponding to amide I band were decomposed 
using an improved method in which baselines were removed before curve fitting, 
and initial heights were set at 90% of those in original spectrum for side bands 
and for the most intense component, and at 70% for the other bands (see Abott et 
al., Appl. Spectrosc. 45: 1665, 1991;  Castrasana et al., BBRC 152: 69, 1988; 
Arrondo et al., FEBS Lett. 252:118, 1989).  Improved curve-fitting is done in 
three steps:
<ol style="list-style-type:lower-roman;">
<li>band shape of component bands is fixed at 0.1 Gaussian fraction
together with the band position, allowing widths and heights to approach
final values
<li>Gaussian fraction is left free
<li>band positions are left to change
</ol>
<p>
Decomposition is done using CURVEFIT running under SpectraCalc.
<p>
The amide I band of both crystalline and noncrystalline enzyme was decomposed. 
Number and initial position of component bands are first obtained by Fourier 
deconvolution and derivation (see 1988 BBRC paper).  In H<sub>2</sub>O solvent 
there are 9 component bands in 1700-1600 cm<sup>&minus;</sup> region, and 8 
components in the D<sub>2</sub> solvent.
<p>
Band assignment to conformational structures is difficult since there appear to 
be more bands than number of expected secondary structures.  Assignment is 
unambiguous for some, and others by comparison with data from other technique to 
obtain reasonable approximations.  1658 cm<sup>&minus;1</sup> corresponds to 
&alpha;-helix and unstructured peptide in H<sub>2</sub>O but only to 
&alpha;-helix in D<sub>2</sub>O.  1665 and 1690 cm<sup>&minus;1</sup> correspond 
to &beta;-turn and extended structures (&beta;-strands) are have signals in the 
1620-1640 cm<sup>&minus;1</sup> region.  1636 cm<sup>&minus;1</sup> is an 
intramolecular C=O vibration of &beta;-sheet; there is also a band at 1625 
cm<sup>&minus;1</sup>, described in homopolypeptides and found in denatured 
proteins in buffered D<sub>2</sub>O, but not common in native proteins.  Its 
presence has been described for concanavalin A, and is found in peptides with 
extended configuration having hydrogen bonding patterns which are not 
intramolecular &beta;-sheet but instead involving bonding of other molecular 
structures, such as monomer-monomer interactions.  The 1625 
cm<sup>&minus;1</sup> band is also found in TPI (triose phosphate isomerase) 
indicating hydrogen bonding to &alpha;-helix and in human LDL proteins having 
&ldquo;low frequency &beta;-sheets&rdquo; which are less accessible to solvent. 
The 1643 cm<sup>&minus;1</sup> band for integral membrane proteins gets 
contributions mainly from nonstructured conformations, including open loops. The 
table of bands and peak areas and widths therefore indicates a secondary 
structure of 40% &alpha;-helix, 20% extended (includes &beta;-sheet), 17% 
&beta;-turns, and 22% loops and unstructured conformation.
<p>
Spectra were also taken during thermal denaturation studies, since these can 
help identify the source of certain bands.  The range was 20 to 80&deg;, taken 
about every 2&deg;.  A thermal event occurred at &asymp;55-60&deg;, in which 
bands at 1682 and 1620 cm<sup>&minus;1</sup> either emerged or became prominent. 
These arise from protein aggregation after irreversible denaturation, occurring 
in both soluble and membrane proteins.  Enzyme activity is lost about 60&deg;. 
EM reveals loss of crystalline arrays, with extensive vesicle aggregation and 
collapse of the bilayer structure.
<p>
Bands which show &alpha;-helix structure show sharp shifts in band position and 
increases in bandwidth with thermal denaturation. With intermolecular 
&beta;-strands, the shift in position is not accompanied by increased bandwidth, 
but the band area increases on a percent basis.
<p>
Protein equilibrated in D<sub>2</sub>O does not always shows complete H/D 
exchange, which can be revealed by comparing spectra at 80&deg; in 
H<sub>2</sub>O and D<sub>2</sub>O.  Various components present at room 
temperature are still present on heat treatment with characteristic 
modifications.  The most prominent band seen for enzyme in H<sub>2</sub> and 
indicating &alpha;-helix there are changes in band width and percent area which 
are alike in both media, although there is a position shift from 1657.5 
cm<sup>&minus;1</sup> (20&deg;) to 1655 cm<sup>&minus;1</sup> (80&deg;).  This 
is smaller than the shift in D<sub>2</sub>O; hence the deuteration effect 
results in a larger shift.
<p>
There is little difference in the thermal denaturation behavior of 
noncrystalline and crystalline enzyme, a conformation of room temp spectral 
comparisons.  Thermal effects on &beta;-sheet and -edge (intermolecular 
hydrogen-bonded &beta;-strands) do differ between the crystalline and 
noncrystalline forms.  Crystalline enzyme shows a thermal event for the 
&beta;-edge at 6&deg; higher.  Moreover there is no change in band position 
corresponding to &beta;-sheet in the crystalline enzyme.  Both these findings 
suggest greater thermal stability in the crystalline enzyme.
<p>
<b>Discussion</b>.  The FT-IR spectra indicated 40% &alpha;-helix. Predictive 
algorithms indicated 21 transmembrane helices in the beef enzyme with multiple 
subunit assemblies.  If the average length is 22 aa/helix and the mass is 120 
Da/aa residue, ~34% of the amino acids are involved in &alpha;-helices.  A 
cryo-electron microscopic analysis of beef crystalline enzyme indicated 12-16 
&alpha;-helices/monomer, which is 32-42% helix.  Circular dichroism shows 37-44% 
&alpha;-helix, with a &beta;-sheet of 13-18% and turns of 12-14%.  These 
previous studies show good agreement with the FT-IR data, including the 20% 
&beta;-sheet and 17% &beta;-turn.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Reddy VY, Desorchers PE, Pizzo SV, Gonias SL, Sahakian JA, Levine RL, Weiss SJ
 </span>
 <span class="year">
(1994)
 </span>
 <span class="jname">
J. Biol. Chem.
 </span>
 <span class="jvol">
269,
 </span>
 <span class="pages">
4683-4691.
 </span>
<p class="reftitl">
Oxidative Dissociation of Human &alpha;<sub>2</sub>-Macroglobulin Tetramers
into Dysfunctional Dimers
<p class="Para1">
Purified &alpha;<sub>2</sub>-macroglobulin (&alpha;<sub>2</sub>M) treated with 25 &micro;M
HOCl caused a reaction in which 18.9 Eq oxidant/&alpha;<sub>2</sub>M subunit was consumed
in 15 min, with a complete loss of antiproteolytic activity (seen in
plot of &alpha;<sub>2</sub>M activity and consumption of oxidant Eq vs. time period of 120 min;
in addition a replot of &alpha;<sub>2</sub>M activity vs. oxidant Eq consumed/&alpha;<sub>2</sub>M subunit was
done).  Oxidant assay can be done using iodometry or monitoring 5-thio-2-nitrobenzoic
acid oxidation.  Chloramine (made from HOCl + ammonium chloride) reacted
more slowly in a biphasic way: in the first 15 min, 7.8 Eq reacts with each
&alpha;<sub>2</sub>M subunit to produce partially oxidized form &alpha;<sub>2</sub>M<sub>ox-1</sub>, which
appears to be indistinct from native form accoroding to 5% (pore limit) native
PAGE.  It also had full proteinase inhibiting activity (97.8%) as it could
trap porcine pancreatic elastase (PPE) in a cage inaccessible to &alpha;<sub>1</sub>-proteinase
inhibitor.  Native and &alpha;<sub>2</sub>M<sub>ox-1</sub> were mixed with <sup>3</sup>H-labeled
insoluble elastin with 2-fold molar excess of PPE;  PPE alone degrades 110
&micro;g elastin in 24 h and with native &alpha;<sub>2</sub>M present, it is virtually
inhibited (0.2 &micro;g).  The same was so for &alpha;<sub>2</sub>M<sub>ox-1</sub>: only
0.1 &micro;g was digested.  The association constant <i>k</i><sub>assoc</sub>
of &alpha;<sub>2</sub>M<sub>ox-1</sub> for PPE was
3.3&nbsp;&times;&nbsp;10<sup>6</sup>&nbsp;M<sup>&minus;1</sup>&nbsp;s<sup>&minus;1</sup>
by a competition assasy with &alpha;<sub>1</sub>-proteinase inhibitor (see
methods), and this values compares well with a previous determination of the
native &alpha;<sub>2</sub>M, 4.4&nbsp;&times;&nbsp;10<sup>6</sup>&nbsp;M<sup>&minus;1</sup>&nbsp;s<sup>&minus;1</sup>
from another report.
<p>
If chloramine action continues for 2 h, another ~8 Eq oxidant is consumed,
which is named &alpha;<sub>2</sub>M<sub>ox-2</sub>.  Native PAGE showed that unlike
&alpha;<sub>2</sub>M<sub>ox-1</sub>, which strongly resembled native &alpha;<sub>2</sub>M, &alpha;<sub>2</sub>M<sub>ox-2</sub>
was electrophoretically faster.  In a denaturing gel, all three forms moved
to the same point, indicating that oxidation destroys the tetramer to produce
dimers that remain together because of disulfide bonds.  On Superose 6 column
&alpha;<sub>2</sub>M eluted as a dimer.  With electron microscopy (&alpha;<sub>2</sub>M adsorbed to thin carbon
films),  negative staining shows that native and &alpha;<sub>2</sub>M<sub>ox-1</sub> have the
same appearance, which appears donut-shaped.  &alpha;<sub>2</sub>M<sub>ox-2</sub> however appears
to be in the shape of a twisted letter <q>S</q> and looks like dimers induced
by cadmium or mild acid treatment.  The oxidized dimers were unable to stop
inactivation of PPE by &alpha;<sub>1</sub>-proteinase inhibitor.  Other
proteinases were similar with PPE in their action by &alpha;<sub>2</sub>M (trypsin, human
neutrophil elastase). &alpha;<sub>2</sub>M<sub>ox-2</sub> was an entirely inactivated form
against the action of PPE (98.7% inacttivated).  When preformed &alpha;<sub>2</sub>M-PPE
complexes were treated with chloramine, proteinase inhibition was not reversed
and &alpha;<sub>2</sub>M tetramers did not dissociate into dimers although 18.1 Eq oxidant
were consumed, similar to what happens to the native &alpha;<sub>2</sub>M.  Also of interest
is that the chloramines of Tris and of &beta;-alanine (generated by HOCl
treatment of Tris and &beta;-Ala) also oxidized &alpha;<sub>2</sub>M, with ~8 and ~16 Eq
consumed per subunit.  None of these oxidizing changes occurred if the
the &alpha;<sub>2</sub>M in chlorinated oxidants were incubated in an excess of methionine.
<p>
&alpha;<sub>2</sub>M in all its forms was hydrolyzed at 155&deg; for 45 min (see <a
href="http://www.ncbi.nlm.nih.gov/pubmed/2880842">Roseman &amp; Levine,
<i>J. Biol. Chem.</i> <b>262</b>: 2101, 1987</a>) and then its amino acids
analyzed.  Native &alpha;<sub>2</sub>M has 1.1 methionine sulfoxides/subunit.  &alpha;<sub>2</sub>M<sub>ox-1</sub>
had 8.8 MetSO/subunit.  &alpha;<sub>2</sub>M and &alpha;<sub>2</sub>M<sub>ox-1</sub> were incubated with
[<sup>14</sup>C]methylamine, which reacts with the thiol esters in &alpha;<sub>2</sub>M, which
produces a titrable -SH group and Glx-952 alkylamide.  The fluorograph of a native
PAGE separation shows that native &alpha;<sub>2</sub>M and &alpha;<sub>2</sub>M<sub>ox-1</sub> migrate
together whereas &alpha;<sub>2</sub>M<sub>ox-2</sub> is electrophoretically faster.
5,5'-DTBNB analysis of thiol content:  1.00 thiol per subunit is found
in native &alpha;<sub>2</sub>M and 0.98 thiol/subunit in &alpha;<sub>2</sub>M<sub>ox-1</sub>.  A table shows that
there are no differences in oxidative modifications between native and ox-1 forms
of &alpha;<sub>2</sub>M, although 8 Eq were observed from iodometry.  Since the 8 Eq matches
the 8.8 MetSO/subunit and those were not detected by the amino acid analysis,
it is thus by oxidation of Met resides that NH<sub>2</sub>Cl works.  Not
all residues were oxidized however, as there are 25 Met residues per subunit.
For &alpha;<sub>2</sub>M<sub>ox-2</sub>, treated for a longer period with chloramine, 6
addiitional Met residues per subunit were oxidized;  thus T &rarr; D multimer
formation exposes 6 Met residues.  The thiol ester remains unaffected because
it was able to incorporate labeled methylamine and the 5,5'-DTBNB thiol measurement
shows 1 thiol/subunit.  &alpha;<sub>2</sub>M<sub>ox-1</sub> was treated with NH<sub>2</sub>Cl
for various periods of time (reaction termined with Met addiition), and
both antiproteolytic activity and Met content measured.  Plots of
MetSO residues/subunit vs percent &alpha;<sub>2</sub>M inactivated show a positively sloped line in the first
(0% value = 8 MetSO resid/subunit, 100% value = 15-16 MetSO resid/subunit).  MetSO is determined after acid hydrolysis
of CNBr-digested samples (see Levine, J. Biol. Chem. 258: 11823, 1983).
The linear plots show that activity is lost in a process linearly dependent on
modification of amino acid structure.
<p>
While oxidation of 6 Met residues led to inactivation, at least 8 Eq of chloramine
reactant were consumed.  2 Eq remain unaccounted.  Amino acid analysis did not
show modified residues, and bi-tyrosine was not seen in fluorometric analysis.
Alkaline hydrolysis gave variable results, but when a plot of Trp resid/subunit
vs. percent inactivation was done, a positively sloped line with <i>r</i> = 0.55
was sufficient to show that at 0% inactviation there is 11.2 Trp resid/subunit,
and at 100% inactivation, there are 10.0 Trp resid/subunit, indicating a modification
of two Trp residues per subunit and thus the 2 Eq.  Multicomponent analysis
of UV second derivative spectra of the differentially inactivated &alpha;<sub>2</sub>M samples
showed the interesting result.
<!-- ============================================================= -->
<p class="tags">
HBV strain analysis, genomics, phylogeny, proteomics
<p class="ref">
 <span class="authors">
Norder H, Courouce A-M, Magnius LO
 </span>
 <span class="year">
(1994)
 </span>
 <span class="jname">
Virology
 </span>
 <span class="jvol">
198,
 </span>
 <span class="pages">
489-503
 </span>
<p class="reftitl">
Complete genomes, phylogenetic relatedness, and structural proteins of six 
strains of the hepatitis B virus, four of which represent two new genotypes
<p class="Para1">
Genomes of xix HBV viral strains sequenced from overlapping PCR products. Four 
strains corresponded to two serological subtypes <i>ayw4</i> and 
<i>adw4q<sup>&minus;</sup></i>, which are S gene differences. 
<i>adrq<sup>&minus;</sup></i> (Pacific origin) are in Group C genome.
Phylogenetic dendrograms confirms grouping into six groups A to F
(21 human, 1 chimpanzee, 4 rodent genomes).  Sequence at nucleotide level
varied between 2 and 15%.  Group F genomes are most divergent (these are 
New World).  Group E has 17 unique amino acids and Group F 34 amino acids.  
At position 81 is a Ser&rarr;Ala change in immunodominant region of HBcAg,
and HBsAg has four extra Cys residues (positions 19, 183, 206, 220).
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Silver FH, Wang M-C, Pins GD
 </span>
 <span class="year">
(1995)
 </span>
 <span class="jname">
Biomaterials
 </span>
 <span class="jvol">
16,
 </span>
 <span class="pages">
891-903
 </span>
<p class="reftitl">
Preparation and Use of Fibrin Glue in Surgery
<p class="Para1">
<b>Review</b>.  Fibrinogen (Fbg) accounts for 0.2% vol of whole blood.
Three pairs of polypeptides are joined to form a symmetric dimeric
structure.  Central domain compsed of fibrinopeptides A and B (FbP-A and
FbP-B) cleaved by thrombin to form fibrin monomer.  Fibrin monomers
assemble noncovalently after release of FbP-A and -B, forming twisting
two-stranded fibrils.  Factor XIII is a fibrin stabilizing factor
which permits covalent cross-linking of fibrin (occurs optimally in
5-20 mM CaCl<sub>2</sub> with &Gamma;/2 = 0.15 M); FXIII also inhibits
activation of plasminogen to start clot lysis.  Fibrinolysis can also be
inhibitied by &epsilon;-aminocaproic acid, aprotinin, and
4-(aminomethyl)cyclohexanecarboxylic acid.
<p>
Fibrin glue (FG) can be made from pooled or single donor blood, be autogenous
or allogenic.  Clinically useful material must be deactivated for viruses
(solvent/detergent or dry heat), although the FDA does not approve use
of pooled blood products.  Fibrinogen is isolated by centrifugation in
combination with cryoprecipitation, ethanol, ammonium sulfate, or PEG
precipitation.  Concentrated solutions of Fbg are resolubilized and mixed
with thrombin to produce an FG or fibrin adhesive (adheres tissues together),
or as a fibrin sealant to close tissue defects during phases of surgery.
<p>
The best material for Fbg precipitation is citrated plasma (10% v/v sodium
citrate solution in water).  The whole blood is centrifuged at 600 <i>g</i>
for 10-20 min.  Cryoprecipitation is then used via a number of freeze-thaw
cycles.  This is useful for large-scale preparations.  Plasma is frozen
to &minus;20&deg; to &minus;80&deg;.  Freezing is at least for 1-24 h.
Thawing is then allowed at 4&deg; from between 12&nbsp;h to overnight.
The precipiate is obtained spinning from 1000-6500 <i>g</i> for 5-15&nbsp;min.
The pellet is then reconstituted in 0.5-1.0&nbsp;ml&nbsp;of supernatant.
<p>
To make the FG, a 40 mM CaCl<sub>2</sub> solution is prepared with 500-1000
U/ml thrombin and 1000 U/ml aprotinin.  A 1:1 or 1:2 Fbg:thrombin solution
is prepared and gelling time is 1-3 min.
<p>
A more detailed method is by Saltz et al.  Twice frozen-and-thawed plasma
is frozen at &lt; &minus;32&deg; and stored at &lt; &minus;18&deg; for 18 h.  The plasma
is thawed at 4&deg; in circulating water bath, then centrifuged at 0&deg;
at  4200 rpm for 12 min.  The concentrated Fbg is mixed with 40 mM calcium
chloride and 250 U/ml thrombin.
<p>
A more refined product is obtained by chemical methods.  1&nbsp;ml&nbsp;saturated
ammonium sulfate is added to 3-5&nbsp;ml&nbsp;plasma and after a period, this is
spun at 3000 rpm for 3-15&nbsp;min.  The precipitate is resolubilized in 1 ml
40 mM calcium chloride.  Solutions for 500-1000 U/ml thrombin in 40 mM
CaCl<sub>2</sub> are added 3:1 to the Fbg solution to make the glue.
<p>
Fbg can be precipitated using ethanol at 0&deg; added to about 10% (v/v).
The mixture is allowed to stand for 15&nbsp;min until centrifugation.  Polyethylene
glycol (PEG) with avg 1000 MW can be added to plasma to a final 10% (w/v).
The precipitate is spun at 8000 rpm for 10 min, and pellet resuspend in
minimal sodium phosphate buffer.  This yields about half of the fibrinogen.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Andersen GR, Koch T, S?rensen AH, Thirup S, Nyborg J, Dolmer K, Jacobsen L, 
Sottrup-Jensen L </span>
 <span class="year">
(1994)
 </span>
 <span class="jname">
Ann. N.Y. Acad. Sci.
 </span>
 <span class="jvol">
737,
 </span>
 <span class="pages">
444-446.
 </span>
<p class="reftitl">
Crystallization of Proteins of the &alpha;<sub>2</sub>-Macroglobulin Superfamily
<p class="Para1">
<b>REVIEW</b>.  Structural models for both native and transformed a2M have
been based on negative-stained EM.  CryoEM was recently reported for a
methylamine (MA)-treated a2M.  EM structure work on complement protein C3,
related to a2M was reported.  Our group has produced diffracting crystals
of human a2M and the MA-treated protein and a proteinase-complexed form, 
two different forms of human C3 (MA-C3 and MA-C3b), rat MA-&alpha;<sub>1</sub>M,
and egg white ovostatin.
<p>
Vapor diffuce in sitting or hanging drops under many conditions.  Solvent
content in crystals was 57-75%, above the normal range of 27-65%.
In many cases these crystals produced limited to no diffraction data,
but high intensity X-ray from synchrotron and low temperature methods 
improved the result.  Crystals are soaked in glycerol allowing flash freezing
in stream of cold nitrogen before X-ray exposure. The required crystallization
temperature had to be &lt;0&deg; C to obtain any diffraction data.
Crystallization conditions for MA-&alpha;<sub>2</sub>M produced hexagonal 
crystals of 0.8 mm &times; 0.8 mm &times; 0.5 mm with resolution down to 
10 &Aring; and merging R factor = 10.3%.  Phase information is being obtained
with heavy atom derivatives.
<!-- ============================================================= -->
<p class="tags">
single-stranded oligonucleotide purification, HPLC
<p class="ref">
 <span class="authors">
Warren WJ, Vella G
 </span>
 <span class="year">
(1994)
 </span>
 <span class="jname">
Meth. Molec. Biol.
 </span>
 <span class="jvol">
26,
 </span>
 <span class="pages">
233-264.
 </span>
<p class="reftitl">
Analysis and Purification of Synthetic Oligonucleotides by High-Performance
Liquid Chromatography
<p class="Para1">
Modifications to the typical deoxyribonucleotide base might include substitution
of oxygen atom for sulfur on the phosphate backbone (phoshorothioate) or substitution
of methyl or borane group instead of an oxygen to create methyl-phosphonate or 
boranophosphonate DNA.  Because coupling efficiency of automated synthesizers is
not 100% (about 98%), in addition to instance of depurination and strand cleavage
(<q>failure sequences</q>), this means purification is necessary to remove unwanted
products.  Downstream applications of synthesized oligos such as XRD and 
site-directed mutagenesis requires purification.  PAGE has been used for analysis
and purification, often when multiple samples require purification; limitations
include needing to radiolabel or use stain techniques that require post-cleanup
methods which leads to low recovery and contamination with gel matrix constituents.
Gel methods also do not allow for multimilligram to gram quantities needed in
therapeutics.
<p>
The general scheme of oligo synthesis and purification starts with cleavage for 
the oligo from solid phase support and removing protecting groups from the base 
and phosphates.  If the oligo is to be purified by electrophoresis on 
capillaries or slab gels, or analytical or preparative anion-exchange HPLC, or 
to be digested by enyzmes prior to RP-HPLC for base composition analysis, then 
the final 5&prime;-DMT should be removed.  In contrast, the 5&prime;-DMT should
be left intact if RP-HPLC is done to purify since it helps separate failure 
sequences that have been de-tritylated.  Two types of RP-HPLC can be done:
one is an solid phase cartridge such as OligoPak disposable, and which gives
&gt;85% purity up to 15 mg of oligo (about 1 &micro;mol oligo if a 20-mer)
For larger scale HPLC, milligram to gram amouts are purified to 95% purity.  
After purification the DMT group is removed and the oligo is extracted with 
ethyl acetate.  The general method is as follows:
<ol>
<li>The solid support (CPG) is transferred from synthesis column to screw-cap
tube.
<li>1-2 ml of fresh conc. ammonium hydroxide for &lt; 1.0 &micro;mol or 6 ml
for 15 &micro;mol scale is added to the tube.  It is best to keep ammonium hydroxide
refrigerated so that the ammonia gas does not evolve from liquid phase, and it should
be purchased in small quantities only.
<li>The tightly sealed tube is heated 55&deg; for 5 h or at room temp for 24 h.
The room temperature option is best if a high yield by RP-HPLC is done as detritylation
is minimized.  Note that if there is a significant volume loss during reaction
(&gt;20%), it may indicate that ammonium hydroxide was leaked out of the tube, and
the reaction may be incomplete.  Add more conc. NH<sub>4</sub>OH and continue the
deprotection.
<li>Transfer the liquid phase to new tube, add triethylamine in a volume to create
a 1% concentration (perhaps ~30 &micro;l).  This keeps the pH at a high level so that
detritylation does not occur.
<li>Cool sample on ice and dry at room temperature in centrifugal evaporator.
If necessary store at &minus;20&deg; until next step.
</ol>
<p>
The yield of oligo from synthesis can be determined spectrophotometrically.
To do this:
<ol>
<li>Dissolve crude oligomer in 1 ml of Milli-Q/distilled water.
<li>Prepare the following dilution, depending on scale of synthesis:
 <ul>
 <li>10 &micro;l to 1 ml water if 200 nmol or less
 <li>5 &micro;l to 1 ml if between 200-1000 nmol
 <li>1 &micro;l to 2 ml water if 1-15 &micro;mol
 </ul>
<li>Read 260 nm against water blank.
<li>Calculate the total AU<sub>260</sub> by multiplying the absorbance by the dilution
factor (100, 200, 2000) based on scale above.  Take the value and multiply it as
33 &micro;g / AU<sub>260</sub> to get amount in micrograms.
</ol>
<p>
If oligo has not been detritylated for the last base and is still on solid support
and will be purified or analyzed using electrophoresis or anion-exchange HPLC, then
detritylate as follows:
<ol>
<li>Dry all fractions with DMT-protected product in centrifugal evaporator.
<li>Resuspend in 100 &micro;l 80% acetic acid.  After 20 min at room temperature,
the bring the sample to dryness.
<li>Resuspend pellet in 500 &micro;l 1% triethylamine.  Again concentrate to dryness.
This neutralizes the acid, which might depurinate the oligo.
<li>If necessary the DMT group can be removed using liquid-liquid extraction:
 <ol>
 <li>Add 300 &micro;l water to dried tube and mix well
 <li>Add 300 &micro;l of ethyl acetate.  Vortex.
 <li>Remove the upper phase, which contains the DMT group, and discard.
 <li>Repeat ethyl acetate addition and extraction two more times.
 <li>Concentrate to dryness and store &minus;20&deg; until needed.
 </ol>
</ol>
<p>
<b>Anion-Exchange HPLC</b>.  Resolution using this method depend on the choice 
of column, gradient profile, and length of largest oligo.  A resolution of the 
<i>N</i> and <i>N</i> &minus; 1 oligonucleotides can be achieved when <i>N</i> 
&lt; 30 usually. With UV detector as little as 300 ng of nucleic acid can be 
observed, which is a sample having 0.2 mAU<sub>260</sub> per microliter being a 
50 &micro;l sample. Samples can be manually injected or loaded by autosampler. 
Any anion-exhange system is possible.  Silica particle and monolithic organic 
polymer systems can be used. Particle-based medium can take higher flow rates 
while polymer-based supports can withstand alkaline pH, and be cleaned with a 
combination of weak acid or base solutions to remove previous samples.  Both 
strong and weak anion exchangers have been used;  weak exchangers are used at 
&lt; pH 8.0 since they lose their charge at a higher pH.  A binary system is used 
with solvent A composed of 25 mM TrisHCl pH 8.0 containing 1 mM EDTA and 10% 
MeCN, and solvent B composed of A + 1.0 or 2.0 M NaCl is used.  A high 
resolution 4.6 mm &times; 10 cm column (e.g. Waters GenPak FAX) is good for 
rapid analysis.  This particular column is a 2.5 &micro;m nonporous particle 
functionalized with DEAE groups.  It is important to remove the DMT since its 
presence causes long retention times due to column matrix interactions.  Sample 
should be microcentrifuged then 0.22 &micro;m-filtered.  Sample volume for 
injection is such that 4 &micro;g (0.12 AU<sub>260</sub>).  In one program with 
a column temp of 30&deg;, flow = 0.75 ml/min, solvent B = A + 1.0 M NaCl, 
gradient from 10&rarr;60% B over 30 min, a 1 &micro;g injection separated a mix 
of pd(A) in lengths of 5, 10, and 12- to 18-mer: retention times were 13.5, 
21.5, 23.3, 23.9, 24.7, 25.3, 26.0, 26.3, and 26.7 min.  In a refinement with
a sample of pd(A) 25-30 and 40-60 at 0.001 AU<sub>260</sub>/&micro;l (100 ng),
the temperature is raised to 80&deg; and the gradient is from 10&rarr;70% B
over 30 min then held at 70% B for 10 min.  Better baseline resolution can be
obtained for the 40- to 60-mer series if the gradient gows from 10&rarr;50% B
rapidly (in 0.01 min) and the gradient from 50&rarr;70% is in 30 min.  Washing
of a polymer-based pH-tolerant column with 0.1 N phosphoric acid will minimize
sample carryover between injections.
<p>
With heteropolymers (typical ss oligos such as primers), a program for synthesis 
of a 20-mer using the same buffers and 45 &micro;g (7 nmol) load with the 
Tris/EDTA/MeCN buffer above (A) and B=A+1.0 M NaCl and a program of 15&rarr;35% 
B in 2 min, 35&rarr;55% B in 30 min, then holding 55% B for 10 min, the 20-mer 
elutes at about 34 min and several failure sequence peaks emerge before that 
between 7 and 33 min.  Both the presence of 10% MeCN and high temperature 
ensures that no secondary structure is formed.
<p>
<b>Chemically Modified Oligos</b>.  Oligos are often modified in ways that 
affect their chemistry.  In particular the phosphate backbone can be modified
by O-methylation or oxygen can be exchanged for sulfur.  Gel electrophoresis or
anion exchange HPLC can no longer by used for separating these modified oligos.
Sulfurization of phosphodiester bond actually generates a highly charged 
phosphorothioated oligo which will bind to an anion-exhcange column strongly
enough to require higher aalt concentration to elute.  In a program with the
Waters Gen-Pak FAX (4.6 &times; 100 mm) column using the Tris/EDTA/10% MeCN
with B = A + 2.0 M NaCl in a 10&rarr;100% B program (<i>T</i> = 30&deg;), 
a standard phosphodiester 21-mer oligo elutes at 15 min but a phosphorothioate
21-mer elutes at 40-42 min, not with the best resolution.
<p>
With capillary gel electrophoresis (CGE), oligos are analyzed (not prepared)
in much the same way as for slab gel separations (which can be analytical and
preparative).  Smaller failure sequences move more rapidly through the capillary
because of their smaller Stokes radii.  There is better heat dissipation in the
75 &micro;m diameter capillary and so higher field strengths are possible 
(250 V/cm, which for a 30-40 cm capillary is 7.5-10 kV).  The electrolyte is a
Tris/borate/urea buffer, and samples should be detritylated.  With CGE, direct
UV quantiation using the photometer can be done.  The disadvantage is that the
method is entirely analytical.
<p>
<b>Coupling Efficiency Calculation</b>.  The peak areas of all failure sequences
and the main oligomer are obtained.  The average coupling efficiency is found
according to the formula:
<p class="eqnserif">
average coupling efficiency = overall yield<sup>1/(<i>N</i>&minus;1)</sup>
<p>
The overall yield is the quotient of the main product peak area and the sum
of all peak areas of the failure sequences and the main product peak.  The value
<i>N</i> is the oligo length.  Suppose that the peak area values are 
2,208,160 for the main product peak and the sum of the peak areas of the failure
sequences are 216,877, then the overall yield is 0.91, and <i>N</i> = 20.
The avereage coupling efficiency is 0.91<sup>1/19</sup> = 0.995 or 99.5%.
<p>
<b>RP-HPLC</b>.  Oligo samples to be analyzed sould be centrifuged and/or filtered
with 0.22 &micro;m membrane.  For a Waters Delta-Pak C<sub>18</sub> 5 &micro;m 
column inject 0.12 AU<sub>260</sub> (~4 &micro;g).  The DMT group should be left
on the oligo for quantitation.  The A buffer is 0.1 M triethylammonium acetate
(TEAA) pH 6.5 in 5% MeCN, and the B solvent is 95% MeCN, with the program being
a gradient of 0&rarr;40% B over 25 min, holding at 40% B for 5 min, flow at 
1.0 ml/min at 30&deg;.  Between 4-15 min in the program, non-DMT containing 
failure sequences elute, while between 20-30 min the main 20-mer (with DMT on)
elutes along with trityl-on failure sequences.
<p>
With modified base oligos, such as DMT-containing phosphorothioate with a target
21-mer, and the same HPLC run program (gradient).  The types of oligos that 
elute have the same relative retention time as for normal phosphodiester oligos.
<p>
Rapid purification devices for oligos now exist.  Usually 1.0 gram of 1000 &Aring;
40 &micro;m styrene divinylbenzene matrix is wetted with 100% MeCN or MeOH (10 ml)
then with 15 ml 1.0 M TEAA pH 7.0.  Up to 50 AU<sub>260</sub> deprotected but
DMT-on olig in 15% ammonium hydroxide is applied to cartridge.  This might be
all of a 200 nmol scale or 20% of a 1.0 &micro;mol scale synthesis.  Failure
sequences and protecting groups are washed from column with 15 ml 3% ammonium
hydroxide.  The tritylated product is then eluted with 3 ml 40% MeCN.  Prior to
elution of product, the DMT can be removed with 10 ml of 2% TFA, and then the
oligo eluted with 20% MeCN instead.  The SDVB (styrene-divinylbenzene) solid support
is stable to pH 2-13, and so TFA and ammonium hydroxide can be used on-column.
The oligo can be concentrated to dryness and stored &minus;20&deg;.   Before and
after HPLC analysis shows effective cleanup; however, tritylated failure sequences
are still retained with the desired product.
<p>
Preparative RP-HPLC provides a variety of technological configurations.  Radial
Compression Technology uses less expensive polyethylene cartridge columns (instead
of stainless steel) which are radially (instead of axially) compressed. 
In general new methods should be developed at a smaller scale (2-5 ml columns)
since they consume less mobile phase and have shorter separation times.  The method
should first focus on removing failure sequences, then a loading study be done
to check capacity limits, for determining scale-up.  Flow rate scales with the
ratio of the square of column radius.  Mass load scales as the product of column
length ratio and the square of the column radius ratio.  Gradient duration
scales at the ratio of column length.  It should be noted that at maxium capacity
load of a column, the target peak may show significant tailing.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Moller HJ, Poulsen JH
 </span>
 <span class="year">
(1995)
 </span>
 <span class="jname">
Anal. Biochem.
 </span>
 <span class="jvol">
226,
 </span>
 <span class="pages">
371-374.
 </span>
<p class="reftitl">
Improved Method for Silver Staining of Glycoproteins in Thin Sodium
Dodecyl Sulfate Polyacrylamide Gels
<p class="Para1">
Glycoproteins weakly stain in silver stains probably because of steric 
inhibition of silver ion making its way to the protein. 8-25% gradient SDS-PAGE 
gels were prepared and run with several known and/or purified proteins.  Using a 
standard acidic method for silver nitrate impregnation, BSA and standard markers 
show up well whereas a proteoglycan (keratan sulfate-rich), chondroitin sulfate, 
mucin, and &alpha;<sub>1</sub>-acid glycoprotein are not visible.  When gels are 
pre-stained with Alcian Blue and then developed, one gel for 2 min and another 
for 6 min, the chondroitin sulfate and proteoglycans appear heavily stained 
along with weak staining of high MW mucins in the 2 min gel, with no staining of 
glycoproteins or proteins.  In the 6 min gel, which has heavy background 
staining, BSA and MW markers and glycoproteins appear with only weak staining 
from select glycoproteins.
<p>
Alcian Blue use is improved by an initial treatment with periodic acid/potassium 
metabisulfite (NaO-SO<sub>2</sub>-SO-ONa, 
Na<sub>2</sub>S<sub>2</sub>O<sub>5</sub>, CAS 7681-57-4). The procedure produces 
a clear gel with heavy staining of all glycoproteins.  The &alpha;-acid 
glycoprotein preparation shows a heavily stained 40 kDa band, weaker bands at 
34, 58, 68, and 80 kDa.  In the haptocorrin/intrinsic factor is a sharp 57 kDa 
band and more diffuse 85 kDa band, probably IF and haptocorrin, respectively. 
Sensitivity of this procedure was evaluated with a series of decreasing loads of 
mucin and &alpha;<sub>1</sub>-acid glycoprotein in amts from 0.3 to 1000 ng. 
Mucin showed the presence of 15 bands or more of which 10 could still be 
detected at 40 ng, whereas at 8 ng only a few bands were detected.
<p>
The procedure for <b>Periodic Acid Oxidation and Alcian Blue Prestaining</b> is 
performed as follows:
<ol>
<li>The following steps are performed at 30&deg; with reagents prewarmed:
 <ol>
 <li>Fix the gel in 10% trichloroacetic acid for 10 min
 <li>Wash gel twice for 2 min each in 5% acetic acid
 <li>Incubate gel in 1% periodic acid in 5% acetic acid for 20 min
 <li>Wash twice for 2 min each in 5% acetic acid
 <li>Wash twice for 2 min each in water
 <li>Incubate in 0.5% potassium metabisulfite for 12 min
 </ol>
<li>The following steps are performed at 50&deg; with reagents
properly prewarmed:
 <ol>
 <li>Wash the gel twice with 25% EtOH + 10% acetic acid for 2 min each wash.
 <li>Stain with 0.125% Alcian Blue in 25% EtOH/10% acetic acid for 15 min
 <li>Wash the gel three times with 25% EtOH/10% acetic acid for 1, 4, and 5 min 
 in that order.
 <li>Continue with the silver staining process, omitting any
 first fixation steps.
 </ol>
</ol>
<p>
The procedure for <b>Alcian Blue Prestaining</b> is performed
in a 50&deg; water bath with prewarmed reagents:
<ol>
<li>Wash three times with 25% EtOH + 10% acetic acid
for 5, 10, and 15 minutes, in that order.
<li>Stain with 0.125% Alcian Blue in 25% EtOH/10% acetic acid
	 for 15 min
<li>Wash three times with 25% EtOH/10% acetic acid for
	1, 4, and 5 minutes, in that order
</ol>
<p>This can then be directly silver stained.
<!-- ============================================================= -->
<p class="tags">
VEGF, yeast
<p class="ref">
 <span class="authors">
Mohanraj D, Olson T, Ramakrishnan S
 </span>
 <span class="year">
(1995)
 </span>
 <span class="jname">
Growth Factors
 </span>
 <span class="jvol">
12,
 </span>
 <span class="pages">
17-27.
 </span>
<p class="reftitl">
Expression of Biologically Active Human Vascular Endothelial Growth Factor in 
Yeast
<p class="Para1">
<i>Pichia pastoris</i> was used as a host for expression of recombinant protein.
The vector pHILS1 permits incorporation of a signal of the acid phosphatase gene 
for secretion although it is at low levels.  The VEGF-165 isoform was isolated 
from total RNA of the U937 cells and amplified using RT-PCR.  The primers
5&prime;-GGGCTCGAGCACCCATGGCAGAAGGA-3&prime; and 
5&prime;-GGGGGATCCCTGCCCGGCTCACCGCCTCGG-3&prime; are intended to produce an
amplicon that does not include the signal sequence and which includes an
<i>Xho</i>I site on the 5&prime; end and a <i>Bam</i>HI site on the 3&prime; end.
After PCR the 580 bp band is digested with <i>Xho</i>I and <i>Bam</i>HI
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
L&ouml;wer R, L&ouml;wer J, Kurth R
 </span>
 <span class="year">
(1996)
 </span>
 <span class="jname">
Proc. Natl. Acad. Sci. USA
 </span>
 <span class="jvol">
93,
 </span>
 <span class="pages">
5177-5184.
 </span>
<p class="reftitl">
The viruses in all of us:  Characteristics and biological significance of
human endogenous retrovirus sequences
<p class="Para1">
<b>REVIEW</b>.  HERV (human endogenous retro virus) sequences are integral
to human genomes.  At some point in human evolution exogenous retroviruses
inserted themselves into the germ line cells and replicated along with
host genes.  Endogenous retroviruses may be the &ldquo;endogenized&rdquo;
forms of exogenous retroviruses (mouse mammary tumor virus is an example),
or may have developed as retroelements.
<pre style="line-height:1em;">

PSEUDOGENE
            +------+
     >>>>---| cDNA |--AAAA-->>>>
            +------+

                |
                |
                |
                v

RETROGENE
              +------+
     >>>>-##--| cDNA |--AAAA-->>>>
              +------+

                |
                |
                |
                v

RETROPOSON
              +------+  +-----+
     >>>>-##--| ORF1 |--| pol |---AAAA-->>>>
              +------+  +-----+

                     |
                     |
                     |
                     |
RETROTRANSPOSON      v

         +-----+ $ +-----+  +-----+  * +-----+
     >>>>| LTR |---| gag |--| pol |----| LTR |>>>>
         +-----+   +-----+  +-----+    +-----+

                     |         ^
                     |         |
                     |         |
                     |         |
RETROVIRUS           v         |

         +-----+ $ +-----+  +-----+  +-----+ * +-----+
     >>>>| LTR |---| gag |--| pol |--| env |---| LTR |>>>>
         +-----+   +-----+  +-----+  +-----+   +-----+


    >>>> = direct repeat
    ##   = promoter
    AAAA = poly(A) tail
       $ = packaging signal
       * = polypurine tract
</pre>
<p>In this scheme, a <i>pseudogene</i> represents the rare occurence
of reverse transcription with reintegration of the cellular mRNA.
The pseudogene then acquires a promoter to be actively transcribed,
and becomes a <i>retrogene</i>.  The short interspersed elements
(SINEs) probably belong to this category, although they lack coding
capacities.  A <i>retroposon</i> is formed and resembles the
long interspersed elements (LINEs), with a G+C-rich promoter
and a gene coding for a partially characterized protein (ORF1) along
with a gene having RTase homology (<i>pol</i>).
<p>
<i>Retrotransposons</i> evolved to be found from protozoa to humans:
these elements have RTase genes linked to genes coding polyproteins
that self-aggregate and form core particles.  These are equivalent
to retroviral capsid proteins called <b>group-specific antigens</b>
(Gag).  The present of a packaging signal &Psi; means the retrotransposon
RNA allows the nucleic acid to be incorporated into particles.  These
include the presence of LTRs, which contain promoter sequences.
<p>
<i>Retroviruses</i> resemble retrotransposons in all respects except
the inclusion of an <i>env</i> region coding for viral membrane proteins.
While the <i>gag</i> core binds to the RNA and brings it to the membrane
surface, <i>env</i> proteins form the envelope and mediate binding to
next cell.  <i>env</i> adds the ability to spread from cells to individuals.
<p>
With Gag and Env proteins expressed, retrotransposition is suppressed
and budding and cellular export are driven.   Presence of Env also blocks
virus receptors to prevent re-infection by other genomes, and the protein
also interferes with re-integration.  While many copies of retroelements
exist in the genome, very limited copy numbers are seen with retroviruses,
which could be explained why proviral generation is geared more for export
than integrating to new sites.  The HERV-H provirus family has
<i>env</i>-related sequences and is present in only 50 copies, while
other proviruses without <i>env</i> show greater amplification.
<p>
Probes derived from animal retroviruses have been used to screen human
genomic libraries under low stringency, and multiple but defective
provirus families such as HERV-E, HERV-R, and HTDV/HERV-K have been
isolated and characterized.  Others were discovered during sequencing
of human gene loci during the genome project.  Families are classified
based on homology to retroviruses.  Class I families show sequence
similiarty to mammalian type C retroviruses.  The similarity of three
families in the <i>gag</i> and <i>env</i> genes (<i>pol</i> is usually
well conserved) is such that they form the ERI superfamily.  The murine
leukemia virus (MuLV) and baboon endogenous virus (BaEV) are the closest
infectious relatives.  Class II families have homology with
mammalian type B and D retrovirus strains.  The HDTV/HERV-K and
HERV-K family proviruses share homologies in <i>gag</i>, <i>pol</i>,
and <i>env</i>.  The taxonomy of HERV is not well organized.  Retroviruses
nomenclature goes by host and disease of the discoverer, and HERV is
being found in Old World monkeys for which a disease is not really apparent.
The tRNA specificity of the primer binding site is the basis of one
system of nomenclature (this is the one letter amino acid code attached
as a suffix to HERV).  All Class II elements have a lysine primer binding
site, but sequence variability is substantial, given there is no selection
pressure on defective genomes.
<p>
Electron microscopic studies of the basal membrane of syncytiotrophoblasts
showed retrovirus-like particles (RLVPs) budding.  Testicular tumor cell
lines from embyronic carcinomas and teratocarcinomas showed the presence
of RLVPs.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Schena M, Shalon D, Heller R, Chai A, Brown PO, Davis RW
 </span>
 <span class="year">
(1996)
 </span>
 <span class="jname">
Proc. Natl. Acad. Sci. USA
 </span>
 <span class="jvol">
93,
 </span>
 <span class="pages">
10614-10619.
 </span>
<p class="reftitl">
Parallel human genome analysis:  Microarray-based expression monitoring
of 1000 genes
<p class="Para1">
A cDNA library of human peripheral blood lymphocytes transformed with EBV
was prepared and inserts &gt; 600 bp cloned into &lambda;YES-R to make 10-100
million recombinants.  <i>E. coli</i> JM107/&lambda;KC was used to make
bacterial transformants.  At random colonies were selected and propagated
in 96 well plate, and DNA minilysates made by alkaline lysis using REAL
preps (Qiagen).  PAN133 (5'-CCTCTATACTTTAACGTCAAGG-3') and PAN32
(5'-TTGTGTGGAATTGTGAGCGG-3') were used as primers and were complementary
to &lambda;YES polylinker.  The Qiagen QIAquick columns were used to
purify PCR products in 96 well format.
<p>
PCR products were modified with amino acids and suspended to 0.5 mg/ml
in 3 &times; standard saline citrate (SSC).  These were arrayed from 96-well
microtiter plates on to silylated microscopic slides with high-speed
robotics.  1056 cDNAs representing 1046 human clones and 10 <i>Arabidopsis</i>
controls.  These were arrayed in 1.0 cm<sup>2</sup> areas.  Arrays were
printed and incubated 4&nbsp;h in a humid chamber (to allow rehydration),
then rinsed once in 0.2% SDS for 1 min, twice in water for 1 min, and
once for 5 min in sodium borohydride solution (1.0 g NaBH<sub>4</sub>
in 300&nbsp;ml&nbsp;PBS and 100&nbsp;ml&nbsp;absolute ethanol).  Arrays were submerged in
water 2 min at 95&deg;, transferred quickly to 0.2% SDS for 1 min, rinsed
twice in water, then air-dried and stored 25&deg; in the dark.
<p>
The heat shock response of Jurkat (human T) cells was examined using
microarrays.  Cells were heated to 43&deg; (along with 37&deg; controls),
lysed, and the mRNA isolated and reverse-transcribed in the presence
of fluorescein-dCTP (controls) and Cy5-dCTP (heat shock).  A second set
of reactions reversed this labeling (Cy5 for controls, fluorescein for
heat shock).  Fluorescent probe pairs were hybridized to a 1056-element
microarry; this is done to the 1.0-cm<sup>2</sup> microarray under a
14 &times; 14mm coverslip for 6-12&nbsp;h at 60&deg; in custom-built hyb chamber.
Arrays were washed at low stringency (1 &times; SSC/0.2 % SDS) at room temp.
and then 10 min at room temp high stringency (0.1 &times; SSC/0.2% SDS)
then scanned with confocal laser.
<p>
Hybridization was seen in &gt;95% of the array elements from human source,
but not of <i>Arabidopsis</i> controls.  These fluorescent intensities
spanned 3 orders of magnitude.  17 elements showed ratios of &ge; 2-fold
in comparsions of the expression of heat shock vs. controls:  11
were seen to be induced by heat shock, while 6 were the result of modest
repression by heat shock.
<p>
The 17 elements were sequenced on the proximal and distal ends.  Searches
of databases found 14 matches of the 17 clones, and in all cases
proximl and distal cDNA sequence mapped to the same gene.  Of the
1046 genes examined on microarray, the five most highly induced
in heat shock were heat shock protein 90&alpha; (hsp90&alpha;), dnaJ,
hsp90&beta;, polyubiquitin, and t-complex polypeptide-1 (tcp-1).
Three clones were not found in the database.  Clone B7 however showed
homology with an EST from <i>Caenorhabditis elegans</i>, and clones
B7-B9 show 2-fold induction, with basal low-level expression.
<p>
RNA dot-blotting was done to confirm expression.  The expression ratios
of the two procedures differed by &lt; 2-fold for genes identified in
heat shock, with wider differences in measuring absolute levels of
expression.  Microarray absolute expression correlated with with the RNA
blots within a factor of 5.
<p>
Jurkat cells were also treated with phorbol ester, harvested, lysed and
messages reverse-transcribed and used as probes for the microarray.
6 elements showed &ge; 2-fold signal increase with phorbol ester-treated
vs. controls.  5 of the 6 were matched against databases.  <i>PAC-1</i>
tyrosine phosphatase and nuclear factor-kappa B1 (<i>NF-&kappa;B1</i>)
were the most highly induced.  More modest activation was seen with
phosphoglycerate kinase and &beta;<sub>2</sub>-microglobulin.  Clone B19
was 2.1-fold increased, but did not match any entry, and had a low
basal expression level.  RNA blotting experiments confirmed the microarray
results.
<p>
Messages from human marrow, brain, prostate, and heart were prepared as
probes using Cy5-dCTP.  Jurkat mRNA was prepared as a control probe with
fluorescein-dCTP.  Each of the four Cy5-labeled probes were mixed with
an aliquot of fluorescein-labeled control, and the four mixtures used in
separate microarrays.  For the 15 heat shock and phorbol ester-regulated
genes there was detectable expression in the human tissues.  Level
of expression in Jurkat correlated closely with expression in four tissues.
&beta;-actin and cytochrome <i>c</i> oxidase were the most highly expressed
of the 15 genes in Jurkat cells, and were also highly expressed in the human
tissues.  Cytochrome <i>c</i> oxidase, hsp90&alpha;, and the novel B7
sequence were expressed more in the heart than any other tissue.
Transcript profiles are created:  these are 3D column charts with
gene on x-axis, tissue source on y-axis, and relative expression level on
z-axis.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Ory DS, Neugeboren BA, Mulligan RC
 </span>
 <span class="year">
(1996)
 </span>
 <span class="jname">
Proc. Natl. Acad. Sci. USA
 </span>
 <span class="jvol">
93,
 </span>
 <span class="pages">
11400-11406.
 </span>
<p class="reftitl">
A Stable Human-Derived Packaging Cell Line for Production of High Titer
Retrovirus/Vesicular Stomatitis Virus G Pseudotypes
<p class="Para1">
Human embryonic kidney 293 cells transformed with Adenovirus 5 were
maintained in DMEM + 10% inactivated FBS with Gln/P/S supplements.
Previous packaging cell lines were murine, but the human 293 cells
were used here for several reasons.  (1) Human lines do not contain
a large number of endogenous retroviral genomes nor transcribe viral-like
RNAs which can contribute to generation of replication-competent virus.
(2) Retroviral vectors produced in human cells are resistant to inactivation
by the immune system.  (3) 293 cells show high efficiency of transient
transfection.
<p>VSV-G can be toxic to cells if expressed at high level constitutively,
and a tetracycline-inducible system was used.  pMDtet.G contains a minimal
CMV immediate early promoter with 7 tet operators upstream, with intervening
sequence and poly(A) site from human &beta;globin.  pBC.tTA is a vector
with tet/VP16 transactivator, with full CMV immediate early promoter,
intervening sequence, and poly(A) signal from rat insulin II gene.
tet/VP16 binds to tet operator sequences and thus activates transcription
on the pMDtet.G vector.  When tetracycline levels are high, transcription is
suppressed; when low, it is activated.  pMD is an expression vector coding
MuMLV <i>gag-pol</i> under control of CMV IE and with intervening + poly(A)
site.  The vector has no retroviral sequences.
<p>
pMDtet.G and pCB.tTA were contransfected (in equimolar fashion)
simultaneously using a calcium phosphate precipitation method.  In addition
a plasmid encoding puromycin resistance was added.  Transfected cells
were exposed to 2 &micro;g/ml puromycin (Sigma).  1 &micro;g/ml tetracyline
was added to suppress virus expression.  72 drug-resistant clones were
obtained, and these checked for viral expression in absence of tetracycline.
Western analysis identified 12 with high levels of expression.  In the presence
of Tet, detection of virus protein was little to none.  Clone 293G
was chosen because its expression was twice that with transient transfection
with pMD.G.  Two bands from 293G are the glycosylated and unmodified
VSV-G forms, which gives one form when treated with N-glycosidase F;
the absence of complete glycosylation may indicate that expression overworks
the glycosylation system.
<p>
Morphological changes in 293 cells are seen with VSV-G expression, whether i
the 293G cells or in transiently transfected 293s.  Large multinucleated
syncytia are seen correlated to expression.  This may be related to a
putative fusagenic domain in the G protein between 123-137.
<p>
pMD.gagpol was linearized at the <i>Sca</i>I site and cotransfected with
a neomycin resistance plasmid into 293G cells using calcium phosphate.
69 clones resistant to G418 were isolated and culture supernatants screened
for RTase activity.  24 clones were positive for RTase, with activity
greater or equivalent to &Psi;CRE cells on initial screen.  Clone 293GPG
secretes more than ~25-fold more RTase than that by &Psi;CRE or &Psi;CRIP.
and ~10-fold more than Anjou 65 cells.  Absence of tetracycline indicated
induced expression of VSV-G by Western.
<p>
293GPG cells were contransfected with linearized (by <i>Ase</i>I)
MFG.SnlsLacZ retroviral vector, and a zeocin resistance plasmid.
16 resistant clones were isolated and cultured in tetracycline-free
medium.  After 96&nbsp;h supernatants were obtained and NIH 3T3 lymphocytes
infected and X-gal-stained.  Three of the 16 clones, GPGnlsLZ2, -LZ3,
and -LZ4 showed titers of 10<sup>7</sup> cfu/ml.  Several time points
were checked for optimal virus production, and the period between 48-96 h
was considered best.
<p>
293 cells also are efficient at producing high titer virus following
transient transfection.  Vector MFG.S produces stable transfectants
producing high titers, but transient transfection showed relatively
low titers of 10<sup>5</sup> cfu/ml.  An MFG.S derivative &Delta;U3nlsLacZ
was generated with the 5'-LTR was replaced with complete CMV IE promoter
and aligned so start was at the viral start site.  With lipofectAMINE
transfection of this construct, efficiency of 293GPG transfection was
40%.  Titers exceeded 10<sup>6</sup> cfu/ml after 24-72 h.
<p>
Large amounts of viral culture were produced from the GPGnlsLZ2 and -LZ3
clones.  Titers of 1.6 &times; 10<sup>6</sup> cfu/ml were concentrated
by ultracentrifugation &gt;1000-fold (to &gt;10<sup>9</sup> cfu/ml)
and recovery was 65-67%.  Serum inactivation studies were done on
&Psi;CRIPLZ producer clone amphotropic virus and the pseudotyped
virus from the GPGnlsLZ4 clone, and the active titer determined.
Amphotropic virus titer fell by ~250-fold, indicating substantial
serum inactivation (there was no decrease in titer in heat-inactivated
serum).  The VSV-G pseudotype virus titer lost only 5-fold in serum.
There are natural human antibodies against proteins carrying
Gal(&alpha;1&rarr;3)galactose terminal, unique to viral coat proteins.
If Gal(&alpha;1&rarr;3)galactose were included in the serum at 10 mg/ml,
the serum was unable to inactivate virus.

<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Louise-May s, Auffinger P, Westhoff E
 </span>
 <span class="year">
(1996)
 </span>
 <span class="jname">
Curr. Opin. Struct. Biol. 6:289-298
 </span>
 <span class="jvol">
6,
 </span>
 <span class="pages">
289-298.
 </span>
<p class="reftitl">
Calculations of Nucleic Acid Conformations
<p class="Para1">
Understanding nucleic acid conformations may be helpful in clinical
medicine.  A great deal is now known about conformations of atypical
polynucleotides such as RNA-DNA hybrids, helices of parallel strands,
triple helices, quadruplexes, telomere-like structures.  Computational
methods making use of energy minimizations, Monte Carlo and molecular
dynamics simulations, and algorithms of folding and supercoiling are
avaiable.  Improvements in computational power and more refined programs
make a more accurate prediction of conformation possible.
<p>
Geometries and calculated stabilities of normal, tautomeric and novel
hydrogen-bonded base pairs correlate well to the Slater-type orbital (STO)
approximations of core and valence orbitals of each atom with
linear combination of three Gaussian (3G) basis functions, a process
called STO-3G.  STO-3G strength is fairly accurate qualitative molecular
geometries, but there are problems with electron densities.  The
4-31G datasets also correlate with crystallographic structures and
thermodynamic data.  A restrained electrostatic potential (RESP) fitting
procedure is used in calibration of new molecular mechanics (MM) force
field for nucleic acids.  Conformations in which dispersion energies
predominate, such as base stacking, are obtained using Hartree-Fock (HF)
self-consistent field (SCF) methods in the first step optimization of
geometry, and then implementing methods that do electron correlation, such
as 2nd order Moller-Plesset perturbation theory (MP2).  Density functional
theory (DFT) approaches the result by used of calculated wave function.
Combined QM-MM calculations first make the Hamiltonian partitioned into
QM, MM, and QM-MM potentials.   Water-dimer ion-solvation interactions
using HF-MM potential are applications.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Ginzinger DG, Lewis MES, Ma Y, Jones BR, Liu G, Jones SD, Hayden MR
 </span>
 <span class="year">
(1996)
 </span>
 <span class="jname">
Proc. Natl. Acad. Sci. USA
 </span>
 <span class="jvol">
93,
 </span>
 <span class="pages">
11400-11406.
 </span>
<p class="reftitl">
A Mutation in the Lipoprotein Lipase Gene Is the Molecular Basis of
Chylomicronemia in a Colony of Domestic Cats
<p class="Para1">
A feline adipose tissue cDNA library was screened with a human lipoprotein lipase
(LPL) cDNA fragment obtained from a clone.  Positives clones were identified and
run on gels;  a clone with ~900 bp represented the largest fragment, but this
is ~500 bp shorter than the full length human cDNA.  A 5' RACE method was done
done to obtain more sequence, and the final product was 125 nt shorter than the
predicted gene.
<p>
The sequence of the promoter region of the mouse and human LPL genes is known and
quite homologous, and so a primer was developed to help find the 5' end of the
gene.  A primer N24 was produced for the antisense end in the middle of the
cat LPL cDNA.  The PCR reaction yielded a 550 bp product from adipose tissue
RNA that had been reverse transcribed, both from mouse and cat;  this product
size was predicted, and it hybridized to human LPL gene.  Thus it was possible
to have the full coding region and 125 bp of the 5' UTR.  There was amplification
of a transcript 5' to the transcription start site and it had sequence similarity
to mouse and human LPL gene and overlaps with cat LPL gene:  either an alternate
transcript was amplified (Northern analysis indicates a larger transcript), or
the amplification was of cat genomic DNA.
<p>
The cat cDNA has 1512 nt and has an open reading frame of 451 amino acids, with
a 34 nt 3' UTR and a 125 nt 5' UTR.  There are 3 amino acids inserted at the
start of the predicted mature peptide present also in the bovine LPL but not
found in the human cDNA.  There is 90% nucleotide identity between human and
cat cDNA and 94.5% predicted amino acid identity.  With mouse, rat, bovine,
and guinea pig there is ~85% identity with human, and so the cat has the
greatest homology.  Chicken (avian) LPL has 15-17 additional C-terminal residues,
and has a 73-77% identity with mammalian LPLs.
<p>
The normal and affected cat sequences were compared, and a single base change
(G&rarr;A) at position 1234 corresponding to amino acid position 412 and
resulting in a Gly&rarr;Arg conversion was found.  It was the only difference
observed.  This Gly is conserved across species and within members of the
lipase gene family;  particularly it is found in human and rat <em>hepatic</em>
lipase and in canine <em>pancreatic</em> lipase genes.
<p>
Human LpL transcript is seen as ~3.3 and 3.7 kb mRNA species arising from
alternate 3' terminal polyadenylation sites.  The cat shows multiple transcripts
with a 3.4 kb primary and ~8 and 1.6 kb transcripts.  Except for rat (with
3.6 kb transcript), there are multiple transcripts in other species.  Bovine
mRNA is 1.7 and 3.6 kb.  There are no differences in transcript size with
genotype (+/+, +/-, -/-).  Most abundant is found in cat heart with lower
levels in skeletal muscle, adipose, mammary gland, ovary, pancreas, brain,
kidney, lymph node, and lung.  No LpL mRNA is found in bone marrow, liver,
and frontal cortex.
<p>
-/- cats show similar phenotype to humans with LpL deficiency, including
a lactescent plasma due to chylomicronemia, peripheral xanthomata, lipemia
retinalis, and failure to thrive.  Interestingly -/- cats for the G412R mutation
lack both LpL mass and catalytic activity as shown with 5D2 antibody.
Like the mouse, the cat has most of total cholesterol as HDL form.  The cat
has distinct HDL<sub>2</sub> and HDL<sub>3</sub> subfractions like the human
not seen in mouse.  With a lowfat (12%) diet there are still high triglyceride
levels;  heterozygous cats show similar elevations.  Triglycerides are
associated with VLDL more than HDL (seen with FPLC analysis) comparing homozygotes
to wild type.
<!-- ================================================================ -->
<p class="tags">

<p class="ref">
  <span class="authors">
   Moradpour D, Schauer JI, Zurawski Jr VR, Wands JR, Boutin RH
  </span>
  <span class="year">
   (1996)
  </span>
  <span class="jname">
   Biochem. Biophys. Res. Comm.
  </span>
  <span class="jvol">
   221,
  </span>
  <span class="pages">
   82-88.
  </span>
<p class="reftitl">
  Efficient Gene Transfer into Mammalian Cells with Cholesteryl-Spermidine
<p class="Para1">
In vitro introduction of genetic material includes
<ul>
<li>calcium phosphate precipitation
<li>DEAE dextran
<li>cationic liposomes
<li>electroporation
</ul>
<p>
The compound described here is related to the cationic amphiphile 
3&beta;[N-(N&prime;,N&prime;-dimethylaminoethane)-carbamoyl]cholesterol 
(DC-chol), a derivative of cholesterol and lipospermines such as 
dioctadecyl-aminoglycylspermine (DOGS).
<p>
Spermine derivatives monoalkylated at N<sup>4</sup> secondary amine were 
prepared and tested for ability to transfect HuH-7 human hepatocellular 
carcinoma cells.  The rationale is that spermidine provides sufficient cationic 
interaction with the membrane and the alkyl substituent serves to provide 
lipophilicity for transmembrane introduction of compacted DNA.  Simple alkyls 
such as octyl, hexadecyl, and dodecyl groups were without effect.  A 
N<sup>4</sup>-(3&beta;-N-5-pentyl)carbamoyl)-cholesteryl did have excellent 
results though.  This compound is water soluble and is stable to many 
freeze-thaw cycles.
<p>
A dose-transfection curve with constant amount of pCMVluc+ was tested using a 
luciferase assay.  The figure shows that a DNA:cholesteryl-spermidine mole ratio 
of 1:6 to 1:10 is optimal (1:8 gave best results).  Altering that ratio reduced 
luciferase response.
<p>
DNA complexed with CS was analyzed on agarose gels and mobility retardation was 
shown to be complete, i.e., DNA negative charge had been completely neutralized 
and the complex was even positive.
<p>
A transfection time course (0, 30, 60, 120, 240, 360, 480 min) was done and it 
was seen that transfection was complete within 2.  2-4 h transfection times are 
suggested.
<p>
A mechanism of influenza virus infection involves the virus exploiting the drop 
in pH in the endosome, and this causes hemagluttinin to undergo a major 
conformational exposing 25 hydrophobic aa in the N-terminus of the HA2 subunit, 
these leading to membrane fusion (the HA2 fusogenic peptide).  Residues E11, 
E15, and D19 also become protonated, causing transition to &alpha;-helix that 
promotes membrane binding and destabilization.
<!-- ================================================================ -->
<p class="tags">
VEGF
<p class="ref" id="Thomas-VEGF">
 <span class="authors">
Thomas KA
 </span>
 <span class="year">
(1996)
 </span>
 <span class="jname">
J. Biol. Chem.
 </span>
 <span class="jvol">
271,
 </span>
 <span class="pages">
603-606.
 </span>
<p class="reftitl">
Vascular Endothelial Growth Factor, a Potent and Selective Angiogenic Agent
<p class="Para1">
<b>MiniREVIEW</b>.  Angiogenesis is necessary to provide vascular structures
to oxygenate tissues and bring them nutrients as well as to remove wastes
efficiently.  Several proteins were indentified and analyzed in the 1980s
with angiogenic properties, belonging to the fibroblast growth factor (FGF)
family.  They featured mitogenic activity for vascular endothelial cells 
but many other cell types as well.  In late 1980s, a growth factor was selected
for its ability to stimulate angiogenesis and vascular leakage only in 
endothelial cells.
<p>
VEGF shows 20% amino acid sequence identity with PDGF A and B chains with 
8 conserved Cys residues in the minimal PDGF receptor-binding domain.  
PDGF-BB homodimers have three intrachain disulfide bonds and two symmetric
intersubunit disulfide bonds.  VEGF is also a homodimer and its Cys positions
are conserved, suggesting similar 2&deg; and 3&deg; structure.  It has been
shown that placental growth factor (PlGF) has 53% amino acid sequence identify
with VEGF, including the 8 Cys residues in the receptor-binding domain.
Recently, heterodimers of all these growth factors (VEGF, PDGF-AB, PlGF)
have been identified.  The PlGF-VEGF subunit heterodimer is a potent mitogen
where teh PlGF homodimer is rather weak.
<p>
Human VEGF gene has 8 exons with the 121-amino acid form coded by exons
1-5 and 8.  Exon 7 is highly basic and is included in the 165-aa form,
which has a high heparin-binding ability and probably interacts extracellularly
with heparan proteoglycans.  With both exons 6 and 7, the full 189-aa
form is created:  exon 6 contains even more basic amino acids for binding
highly anionic extracellular polymers.  It is supposed that during tissue
damage or inflammatory processes, enzymes that degrade extracellular 
matrix polymers (heparinases, plasmin) causes mobilizes VEGF to stimulate
mitogenesis of endothelial cells.
<p>
VEGF has two homologous receptors, KDR (Flk-1 in mice) and Flt-1 which are 
expressed in vitro and in vivo during vascular development in embryogenesis. 
Each of these receptors are about 1300 amino acids long, have seven 
extracellular Ig-like domains, one short membrane-spanning sequence, and 
intracellular domain with Tyr-kinase activity.  KDR and Flt-1 have 45% amino 
acid identity, equivalent to the 5 Ig-like domains for PDFG&alpha; and &beta; 
receptors.  Flt-1 binds VEGF and PlGF with high affinity, while KDR bind VEGF 
but not PlGF homodimers.  VEGF binding to KDR causes DNA synthsis and 
chemotactic endothelial response.  However, PlGF tight binding to Flt-1 has no 
such response.  Flt-1 or KDR knockouts show lethality by 8-10 d development, but 
the KDR knockouts show no vascular endothelial cell proliferation, while Flt-1 
knockouts show many endothelial cells but poorly organized vessels, which 
indicates Flt-1 has a role in getting endothelial cells to organize with 
themselves or basement membranes.
<p>
Flt-1 splices transcripts to form full-length (membrane-located) and shorter
soluble forms (called sFlt-1); the truncation is on C-terminal side of the
6th Ig-like domain.  sFlt-1 has strong VEGF and PlGF binding activity, and
thus inhibits VEGF from stimulating mitogenesis but at concentrations which
are lower in stoichiometry to VEGF, indicating that inhibition is not by
simple binding.  Membrane-located receptors work like other Tyr kinases
by forming dimers on ligand binding:  it may be the case that VEGF-sFlt-1
complexes form a heterodimer with VEGF-Flt-1 (membrane-bound) complexes,
and thus inhibit signal transduction.  Since KDR receptors can be present,
S-Flt-1-VEGF complexes might hetero-dimerize with them to modulate activity.
The relative expression of Flt-1 and sFlt-1 thus properly modulates this
VEGF and PlGF activity.
<p>
Signal transduction then starts with auto- or transphosphorylation of Tyr 
residues in the kinase.  Insert sequence with several Tyr residues which get 
phosphorylated and generate docking sequences for complex formation with other 
downstream signalling proteins.  KDR has a 70 residue insert and Flt-1 has a 66 
residue insert.  Proteins with SH2 domains bind to the phosphorylated 
(activated) kinase.  These include PI3 kinase which creates 
3-phospho-polyphosphinositol forms.  It also phosphorylated phospholipase 
C&gamma; which hydrolyzes membrane-bound phosphatidylinositol-4,5-bisphosphate 
(PIP2) to inositol-1,4,5-trisphosphate (IP3) and 1,2-diacylglycerol (DAG):  IP3 
stimulates Ca<sup>2+</sup> release from ER, while DAG activates protein kinase 
C.  There is also phosphorylation of Ras GTPase-activating protein GAP and of 
NcK which has one SH2 and three SH3 docking domains can couple surface receptors 
to downstream effectors.
<p>
Angiogenesis is particularly important in menstruating females.  The ovarian
follicle undergoes neovascularization under VEGF control, and VEGF mRNA
expression is elevated in estrogen-responsive epithelial cells lining oviducts
and uterus.  It is also highly expressed in extraembryonic giant trophoblast
cells at sites of implanted fertilized eggs.  VEGF mRNA is detectable in adult
rats in lung alveolar cells, and proximal tubules and glomerular cells of
kidney.  Lower levels are in hepatocytes and brain.  Adrenal cortical and 
testicular Leydig cells show signiicant levels.  Many growth factors themselves
stimulate VEGF mRNA expression, and so any angiogenic effect they have is 
mediated through VEGF.  Phorbol esters induce VEGF protein expression more
than 5-fold in human keratinocytes.  Prostaglandin E<sub>2</sub> increases
VEGF mRNA and protein in preosteoblastic cell line in a manner promoted
by retinoic acid (which causes differentiation) and is inhibited by dexamethasone.
VEGF is aso promoted by cobalt chloride and inhibited by carbon monoxide.
The CoCl<sub>2</sub> effect may be because of replacement of iron in heme-containing
proteins particularly oxygen sensors.  As for CO inhibtion, it binds tightly
to iron and could cause the protein to adopt a configuration that indicates
there is plenty of oxygen.
<p>
In addition to a role in stimulating mitogenesis in vascular endothelial cells,
VEGF also induces chemotaxis and expression of collagenases and plasminogen
activators, which facilitate penetration of growing capillaries in tissues.
Within 5 min of injecting VEGF there is vascular leakage that resolves in 20-30 
min, but this occurs via intradermal and not intravascular administration:
with the latter no permeability is induced by VEGF, indicating other factors
are necessary.  Tumor cells produce VEGF, and VEGF expression may also be
driven by hypoxia occurring in poorly oxygenated cells inside solid tumor
masses.
<!-- ============================================================= -->
<p class="tags">
infectious disease, molecular diagnostics
<p class="ref">
 <span class="authors">
Reischl U
 </span>
 <span class="year">
(1996)
 </span>
 <span class="jname">
Front. Biosci.
 </span>
 <span class="jvol">
1,
 </span>
 <span class="pages">
e72-e77.
 </span>
<p class="reftitl">
Application of Molecular Biology-Based Methods to The Diagnosis of Infectious 
Diseases
<p class="Para1">
Laboratory diagnosis of infections can be approached from four different ways:
<ol>
<li>direct detection of pathogens, for example by microscopy or culture
<li>detection of protein components of those pathogens , for example by
immunochemistry with specific Abs (antigen capture ELISA)
<li>IgM-, IgG-, and IGA-specific detection of Abs against the pathogen,
with changes in the titer (the most sensitive method)
<li>detection of nucleic acids specific for the pathogen (e.g., by PCR)
</ol>
<p>
In partciular molecular biology-based approaches for HIV and tubeculosis
pathogen are among those disease agents that can be detected.
<p>
First generation ELISA tests for HIV were based on purified viral antigens
obtained from viral culture.  False positive tests were higher because of
the presence of cellular antigens packed into virions.  Second and third
generation assays for HIV tests are based on mainly recombinant antigens 
or synthetic peptides of selected immunodominant antigenic epitopes.  
Because cell-based proteins are absent, there is improvements in specificity
and sensitivity.    
<p>
[Explanation of PCR technique omitted.]
<p>
Tuberculosis kills 3 million persons globally each year according to WHO, which 
is the leading cause of death by any infectious disease.  As many as 1 in 3 
persons may be infected with <i>Mycobacterium tuberculosis</i>. Clinically the 
organism is identified by culture or by biochemical tests; but because the 
bacteria grow slowly, tests can take 1-2 months to complete.  The BACTEC&reg; 
radiometric assays can reduce detection to 10-14 days, but the most rapid is 
microscopy of acid fast-stained sputum samples.  The most rapid is by PCR which 
requires as little as 100-1000 bacilli to be done.  Resistance to multiple drugs 
(isoniazid, streptomycin, ethambutol, rifampin, PZA) means that eradication is 
far off.  This form of TB has mortality of 5 to 8 in 10 patients, with duration 
of 4-16 weeks between diagnosis and death of patient.  Detection of rifampin
resistance can be done using PCR:  sequences in the rpoB gene (the ss subunit of
RNA polymerase) are responsible for the resistance.
<!-- ============================================================= -->
<p class="tags">
HBV, recombinant vaccine response, surface proteins, western blotting
<p class="ref">
 <span class="authors">
De Castro L, Yoshida CF, Gaspar AM, Gomes SA
 </span>
 <span class="year">
(1996)
 </span>
 <span class="jname">
Acta Virol.
 </span>
 <span class="jvol">
40,
 </span>
 <span class="pages">
251-258.
 </span>
<p class="reftitl">
Western blot analysis of the reactivity between envelope proteins of hepatitis B 
viruses from Brazilian carriers and antibodies raised against recombinant 
hepatitis B vaccines
<p class="Para1 abstract">
ABSTRACT.  A Western blot assay was standardized to evaluate the antigenic 
reactivity of hepatitis B virus (HBV) strains circulating in Brazilian 
population with antibodies raised against recombinant hepatitis B (HB) vaccines. 
In this assay, HBV envelope proteins from infected human blood were detected by 
antibodies from rabbits immunized with either of two recombinant vaccines. These 
were Engerix B (Smith Kline Beecham, Belgium) containing exclusively S protein 
particles and TGP-943 (Takeda Chemical Industries, Japan) containing M protein 
particles. Forty-seven serum samples, presenting HB surface antigen. (HBsAg) 
reverse passive haemagglutination assay (RPHA) titers ranging from 1:32 to > or 
=1:4096 after HBV particles concentration, were tested. Twenty-seven samples 
were from acute hepatitis cases and 20 were from chronic cases (11 from 
cirrhotic patients and 9 from asymptomatic carriers). Four HBV serotypes, adw2, 
adw4, ayw2 and ayw3, were identified in these samples. Infectivity of these sera 
was evaluated by HBV DNA detection by polymerase chain reaction (PCR). HBV DNA 
was present in 62% of samples from acute cases and in all samples from chronic 
cases. Despite the differences between serotypes, genotypes, forms of infection, 
and infectivity of the samples, antibodies against both vaccines reacted with 
HBV envelope proteins from all but one sample. In one sample from cirrhotic 
patient, only a small protein of unexpected size reacted with TGP-943 antibodies
<!-- ============================================================= -->
<p class="tags">
HBsAg, EIA
<p class="ref">
 <span class="authors">
Kiremit&ccedil;i O, G&uuml;ven B, &Ouml;zsar S
 </span>
 <span class="year">
(1996)
 </span>
 <span class="jname">
Turk. J. Med. Sci.
 </span>
 <span class="jvol">
26,
 </span>
 <span class="pages">
445-449.
 </span>
<p class="reftitl">
Detection of Hepatitis Virus Surface Antigen (HBsAg) with Avidin/Biotin
Solid-Phase Enzymeimmunoassay (EIA)
<p class="Para1">
Samples from 82 HBsAg+ and 67 HBsAg- patients were analyzed with an avidin/biotin
solid-phase sandwich EIA with pAb from authors' laboratory and compared to
HBsAg-EIA kits from Wellcozyme UK and Sorin, Italy, which use mAb-based assays.
The in-house assay was developed by immunizing rabbits with subcutaneous
injections with 20 &micro;g commercial HBsAg vaccine (Gen-Hevac B, Pasteur)
with complete Freund's adjuvant, and at 3-week intervals with incomplete
Freund's adjuvant.  Bleeds from marginal ear view 10 d after each injection 
and antisera titers determined using Wellcozyme anti-HBs; one animal after
9 injections had a titer of 1:100,000 which was used, antiserum stored at 
&minus;20&deg;.  The immunoglobin fraction was enriched by adding sodium
sulfate to 18% w/v and after an hour adding 10-fold volumes of 18% sodium sulfate
in PBS (40 mM phosphate), then centrifugin 30 min at 4000 <i>g</i>; ppt was
wash twice with 18% sodium sulfate, then dissolved in minimal PBS, dialyzed in
cold with PBS three changes, protein assayed by Lowry and stock re-frozen.
Costar 3590 PS plates are coated with 100 &micro;l antiHBs rabbit serum, 
2 &micro;g/well, in 50 mM bicarbonate pH 9.6 overnight at 4&deg;, blocked
with 250 &micro;l of 1% BSA in PBS (Sigma) for 50 min at room temp.  Plates
washed twice with PBS, and allowed to dry for 1 h in vacuum oven, then sealed
and stored &minus;20&deg;.  Biotinylation was done with biotinamidocaproate
N-hydroxysuccinimide ester (BNHS) using 320 &micro;g rabbit Ig-enriched material
in 20 &micro;l PBS and 10 &micro; DMF for 4 h at room temp.  100 &micro;l
of 1 mg/ml glycine in PBS added for another 4 h.  1 ml of 1 mg/ml BSA in PBS
added and the prep dialyzed against PBS 4&deg; overnight.  This is stored frozen.
<p>
For the assays, positive and negative HBsAg controls (Sorin, Italy) are used.
All samples added without dilution and incubated 90 min 37&deg;, washed
five times with tris-buffered saline/detergent (10 mM TrisHCl pH 7.4, 
300 mM NaCl, 0.15% v/v Tween-20). 100 &micro;l biotin-conjugated anti-HBs
rabbit Ig at 1:200 is added in a solution of 5% calf serum, 5% horse serum,
0.5% rat serum, and 5% human serum negative for anti-HBcAg IgG and IgM, incubated
60 min at 37&deg;.  Plates were washed 5 times, then strepavidin-HRP (Sigma)
in 0.1% BSA in PBS added and plates incubated 30 min at room temp in dark.
After six washings, 150 &micro;l substrate (0.1% tetramethylbenzidine, 
0.004% hydrogen peroxide in 100 mM sodium acetate pH 5.5 in DMSO) is added
and developed 20 min in dark, stopped with 50 &micro;l of 4 N sulfuric acid.
Assays are read at 450 nm.
<p>
While all three assay systems found the 67 HBsAg-negative patients to be
negative, three of the samples were read negative by the in-house assay compared
to the other to commercial assays.
<p>
The cutoff value for a negative result is computed as 5 &times; the standard
deviation above the mean value for negative controls.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Wu SM, Boyer CM, Pizzo SV
 </span>
 <span class="year">
(1997)
 </span>
 <span class="jname">
J. Biol. Chem.
 </span>
 <span class="jvol">
272,
 </span>
 <span class="pages">
20627-20635.
 </span>
<p class="reftitl">
The Binding of Receptor-Recognized &alpha;<sub>2</sub>-Macroglobuline to the
Low Density Lipoprotein Receptor-Related Protein and the &alpha;<sub>2</sub>M
Signaling Receptor is Decoupled by Oxidation
<p class="Para1">
Alpha<sub>2</sub>-macroglobulin (&alpha;<sub>2</sub>M) must undergo a 
transformation by proteinases involving reaction of a thioester bond with free 
Lys residue on a proteinase and resulting in conformation change 
(&alpha;<sub>2</sub>M*).  Native gel electrophoresis, electron microscopy, 
sedimentation analysis, and circular dichroism also show evidence of a more 
compact molecule.  This compact molecule can bind the LRP, the LDLR-related 
protein.  LRP binds a number of ligands, and also by RAP, receptor-associated 
protein, which co-purifies with LRP.  Positively charged residues on 
&alpha;<sub>2</sub>M* interact with a 2nd complement-like repeat sequence on 
LRP, this sequence highly anionic.  Both studies with mAbs and recombinant 
protein show that the COOH terminus of &alpha;<sub>2</sub>M* reacts with LRP. 
&alpha;<sub>2</sub>M* (not &alpha;<sub>2</sub>M) can suppress superoxide anion 
generation, enhance PGE<sub>2</sub> and platelet activating factor release, and 
stimulate vascular smooth muscle cell proliferation. Two classes of 
&alpha;<sub>2</sub>M* binding proteins in peritoneal macrophages, both appearing 
to activate signaling cascades.  The low affinity class is 10-fold more abundant 
and is the LRP, but LRP is not involved in signaling.  Thus the signaling 
receptor identity is unknown.  Receptor binding by &alpha;<sub>2</sub>M is 
contained in a 20 kDa COOH-terminal fragment (RBF), and Lys-1374 is critical for 
this. <i>cis</i>-dichlorodiamine-platinum(II) (cis-DDP) modifies a region of 
&alpha;<sub>2</sub>M* upstream of the RBF and this causes decreased LRP binding, 
but has no effect on signaling effects.  Immunochemical studies also indicate 
that another &alpha;<sub>2</sub>M region binds LRP.  However RBF binds LRP and 
can induce &alpha;<sub>2</sub>M* signalling.
<p>
Near UV (220-400 nm) spectra were done on &alpha;<sub>2</sub>M, 
&alpha;<sub>2</sub>M*, and RBF, both on untreated and hypochlorite 
(NaOCl)-treated forms ( [NaOCl] can be assessed by measuring absorption at 292.5 
nm using 206 M<sup>&minus;1</sup> cm<sup>&minus;1</sup> extinction at pH 7.5). 
Peak differences were found at about 242 nm between &alpha;<sub>2</sub>M and 
&alpha;<sub>2</sub>M* in oxidized and nonoxidized forms.  Trinitrobenzene 
sulfonate titration of NH2-terminus and Lys shows that chloramine modification 
accounts for the increased 242 nm absorption.  Absorption at &lambda;<sub>300 
nm</sub> is due to dichloramine formation.  NaOCl had minimal effect on RBF. 
Native &alpha;<sub>2</sub>M and &alpha;<sub>2</sub>M* were susceptible to NaOCl 
oxication at all concentrations used (modifications as seen at 242 nm) plateaued 
at ~2 mM.  RBF resisted modification at even high oxidant concentrations.
<p>
&alpha;<sub>2</sub>M* was radioiodo-labeled and tested for its ability to bind 
to macrophages: after washing away unbound, the cells are lysed with SDS/NaOH 
and the lysate &gamma;-counted.  Untreated &alpha;<sub>2</sub>M* and oxidized 
&alpha;<sub>2</sub>M* were incubated together to examine competitive behavior. 
A decreased binding ability is observed when the oxidant concentration exceeds 
37.5 &micro;M.  At 75 &micro;M hypochlorite 90% of the binding was of the 
non-oxidized form.  Oxidation also affected RAP-sensitive binding of 
&alpha;<sub>2</sub>M*.  A slight increase in RAP-insensitive binding could be 
due to electrostatic interaction of oxidized &alpha;<sub>2</sub>M* and the cell 
surface.
<p>
Binding competition with native &alpha;<sub>2</sub>M was addressed also.  Native 
&alpha;<sub>2</sub>M binds to neither LRP or signaling receptor.  When it is 
oxidized, the &alpha;<sub>2</sub>M form has a better ability to compete for 
binding with &alpha;<sub>2</sub>M*, which means receptor recognition site is 
exposed.  When NaOCl &gt; 25 &micro;M, there was a concentration-dependent 
decrease in the ability to compete for &alpha;<sub>2</sub>M*.  In the RAP 
competition, oxidized native &alpha;<sub>2</sub>M loses its ability to compete, 
which means the oxidized form binds to LRP.
<p>
The previous experiments employed use of IODO-BEADS to achieve radio-iodination 
of the &alpha;<sub>2</sub>M proteins.  As an alternative the Bolton-Hunter 
method was used to label native &alpha;<sub>2</sub>M and &alpha;<sub>2</sub>M*, 
since the other method could potentially oxidize and alter chemistry of 
&alpha;<sub>2</sub>M itself.  The BH-&alpha;<sub>2</sub>M* form was oxidized and 
result shows that NaOCl &gt; 200 &micro;M results in complete loss of LRP 
binding. As little as 25 &micro;M NaOCl enhanced the binding of 
BH-&alpha;<sub>2</sub>M, results similar to the IODO-BEADS-iodinated form.  One 
difference is the need for higher oxidant concentrations to achieve similar 
results, although BH-&alpha;<sub>2</sub>M has a greater resistance to oxidation. 
Previous work shows that &alpha;<sub>2</sub>M oxidation results in fracturing 
along its dimeric axis.
<p>
RBF binds to LRP and the signaling receptor.  It was oxidized up to 125 &micro;M 
and there was no effect on its ability to compete with non-oxidized RBF to cell 
receptors.  Calcium signaling experiments were also done (data not shown) and 
there was no effect of oxidation of RBF on its ability to induce signaling.
<p>
Murine peritoneal macrophages were prepared in the presence of Fura-2/AM and 
adherent cells (on coverslips) mounted.  Then untreated and oxidized forms of 
&alpha;<sub>2</sub>M and &alpha;<sub>2</sub>M* were used to check calcium 
fluxes; 20 nM of the ligands were added with plots of calcium (nM) vs. time. 
NaOCl up to 200 &micro;M had no effect on the ability of &alpha;<sub>2</sub>M* 
to induce increases in intracellular [Ca<sup>2+</sup>], which rose from baseline 
to 100 and 150 nM, which peak levels at 400-500 nM within 10 s of exposure. 
When &alpha;<sub>2</sub>M* is boiled, it loses its signaling ability. Native 
&alpha;<sub>2</sub>M was treated with 50 and 125 &micro;M NaOCl, which are known 
to generate maximum exposure to LRP binding:  this oxidized form was unable to 
elicit signaling (the same as the native &alpha;<sub>2</sub>M, predictably).
<p>
Reducing SDS-PAGE was done for NaOCl-treated &alpha;<sub>2</sub>M that had been 
trypsinized (20 mol trypsin:1 mol substrate, 20 min @ 37&deg;).  If oxidation 
results in an unfolding to expose receptor binding, then it should be more 
susceptible to trypsin.  Gel results show that there are few larger MW bands 
with more extensively oxidized protein, indicating a greater susceptibility to 
trypsin. At concentrations &gt; 5 &micro;M the effect was observed. 
NaOCl-treated native &alpha;<sub>2</sub>M was then titrated for its thioester 
bond, rupture of which should expose the receptor recognition (similar to 
methylamine action).  Results did not show thioester bond breakage in the 
oxidized native form, which is consistent with other results.
<p>
Polyclonal antisera were produced against the RBF.  In previous work it was 
shown that the 20 kDa COOH terminus is exposed in the native structure. An ELISA 
with the antiserum showed the same effect.  With native &alpha;<sub>2</sub>M, 
the ELISA signal was weaker than for &alpha;<sub>2</sub>M* and RBF.  Oxidation 
of native &alpha;<sub>2</sub>M brought the signal to a higher level; at least 
12.5 &micro;M NaOCl is necessary to increase ELISA signal.
<p>
&alpha;<sub>2</sub>M* had two different <i>K</i><sub>d</sub> and 
<i>B</i><sub>max</sub> values so radioiodinated BH-&alpha;<sub>2</sub>M and 
BH-&alpha;<sub>2</sub>M*.  <sup>125</sup>I-BH-&alpha;<sub>2</sub>M* was oxidized 
with 200 &micro;M hypochlorite and then added at different concentrations (0.1, 
0.2, 0.3, 0.6, 5, and 10 nM) both in the presence and absence of 100-fold excess 
of RAP, incubated 16 h 4&deg;.  Cells were then washed twice, solubilized, and 
gamma counts taken.  Binding (as fmol ligand per millions cells) vs. 
concentration was plotted, as well as a Scatchard plot (bound/free vs. bound) 
with a straight line fitted.  Scatchard analysis shows a single class of high 
affinity binding sites (Kd ~83 pM) with <i>B</i><sub>max</sub> of 5.4 fmol per 
million cells. Systat analysis verified these numbers:  <i>K</i><sub>d</sub> = 
71 &plusmn; 12 pM, <i>B</i><sub>max</sub> = 6.2 &plusmn; 2.1 fmol/million cells. 
Binding of &alpha;<sub>2</sub>M* to lower affinity LRP is not seen as addition 
of 100-fold excess RAP did not alter the Kd or Bmax.  Oxidized native 
<sup>125</sup>I-BH-&alpha;<sub>2</sub>M was also studied, and Scatchard analysis 
shows a single class of lower affinity (Kd ~ 0.6 nM) binding sites and Bmax of 
55 fmol/million cells, 10-fold more receptors (Systat analysis pretty much 
showed the same results.  This binding is due to LRP since 100-fold excess RAP 
inhibited radiolabeling.  Thus &alpha;<sub>2</sub>M* binds the high affinity 
sites and causes signaling.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Colon LA, Reynolds KJ, Alicea-Maldonado, Fermier AM
 </span>
 <span class="year">
(1997)
 </span>
 <span class="jname">
Electrophoresis
 </span>
 <span class="jvol">
18,
 </span>
 <span class="pages">
2162-2174.
 </span>
<p class="reftitl">
Advances in Capillary Electrochromatography
<p class="Para1">
<b>REVIEW</b>.  Berraz (1943) discusses form of paper electrophoresis using
the term capillary electrochromatography (CEC) which had been used by Strain
to separate dyes on alumina.  Tiselius suggested electroosmosis to move solvent
through porous medium without hydrostatic pressure.  In mid-70s Pretorius et al
and in mid-80s Jorgenson and coworkers developed CEC:  Pretorius showed how
electroosmosis drives solvent through packed column and Jorgensen exploited 
EOF in fused silica capillaries to cause chromatographic (not electrophoretic)
separations.
<p>
Electrochromatography is any microseparation technique using electroosmosis
(for the most part)  rather than pressure-driven flow.  When through a capillary
column, it is CEC.  Separation is achieved by interaction through two phases
(partitions).  The flow is plug-like as opposed to parabolic in pressure-driven,
but the efficiencies of separation are those of HPLC.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Frishman D, Argos P
 </span>
 <span class="year">
(1997)
 </span>
 <span class="jname">
Proteins:  Structure, Function and Genetics
 </span>
 <span class="jvol">
27,
 </span>
 <span class="pages">
329-335.
 </span>
<p class="reftitl">
Seventy-Five Percent Accuracy in Protein Secondary Structure Prediction
<p class="Para1">
Accuracy of the prediction of secondary structure has been assisted by 
large-scaled sequencing of nucleotides.  Many programs have surpassed the 70% 
level of accuracy. Predictions are usually based on families of homologous 
sequences whose structure is actually known from structural analyses.  Multiple 
sequence alignments can improve accuray by 6-8% relative to using only a 
single sequence.  The computational complexity of multiple alignments is
expressed as <i>L<sup>n</sup></i>, where <i>L</i> is sequence length and
<i>n</i> is the number of sequences in the comparison.   Doubling the 
length increases complexity by 2<sup><i>n</i></sup> while doubling the number
of sequences increases complexity by <i>L</i><sup>2</sup>.  Shortcuts enabling
incorporation of biologically relevant info have been suggested to limit search space,
but hiearchical clustering is used with pairwise alignment starting with most
closely related pairs.  Overall alignment quality depends on pairwise similarity
scores of different sequences on evolutionary tree.  There is a preservation
of register and gaps at earlier stages and there is not a reconsideration of
producing a gapless result: ultimately there is a compromise between pairwise
and overall alignment quality.
<p>
It is critical to be careful about force alignments where no real homology exists,
since it can confuse alignment algorithms which incorporate multiple sequences
including wrong insertions to database.  This study proposes an alternative to 
use of additional information in set of related sequences.  A careful pairwise
alignment of the query sequence with all related sequences is done.  Then only
fragments with significant alignments are done.  Secondary structure propensities
of auxiliary-related sequences are combined with (projected on to) those of base
sequence and weighted according to degree of similarity.
<p>
The algorithm proposed here was trained/tested/compared using nonredundant
set of 125 globular protein tertiary structures of Rost and Sander, with
atomic coordinates from PDB.  The Heringa et al (set FA) was used for final 
training:  this contains 556 protein chains determined by X-ray analysis and
NMR with 30% pairwise sequence identity, no sequence &lt; 50 residues and 
crystallographic resolution of &gt; 2.5 &Aring;
<!-- ============================================================= -->
<p class="tags">
VEGF
<p class="ref" id="Ferrara-VEGF">
 <span class="authors">
Ferrara N, Davis-Smyth T
 </span>
 <span class="year">
(1997)
 </span>
 <span class="jname">
Endoc. Rev.
 </span>
 <span class="jvol">
18,
 </span>
 <span class="pages">
4-25.
 </span>
<p class="reftitl">
The Biology of Vascular Endothelial Growth Factor
<p class="Para1">
<b>REVIEW</b>.  Angiogenesis is required for embryogenesis and often the
repair of tissue in fully developed organisms.  It also plays a role in
pathogenesis of many diseases and disorders: proliferative retinopathies,
age-related macular degeneration (AMD), tumor growth, rheumatoid arthritis,
psoriasis, and more.  In the ocular diseases, angiogenesis is the primary 
cause of blindness, with leaking and clotting and fibrosis causing 
retinal detachment and irreversible damage to macula.  Regulators of 
angiogenesis show many growth and protein-based factors:  bFGF, aFGF,
TGF-&alpha; and -&beta;, hepatocyte growth factor, TNF-&alpha;, angiogenin,
IL-8, and more.  VEGF is a vital gene (its knockouts are lethal in embryogenesis)
and specific for creating endothelial formation of capillaries and vessels.
<p>
VEGP has an ED<sub>50</sub> of about 2-10 pM (~40 pg/ml) for mitogenic activity 
specifically acting on micro- and macrovascular endothelial cells found in 
arteries, veins, and lymphatics.  Endothelial cells in culture will invade
collagen gels when treated with VEGF in synergy with bFGF.  It induced sprouting
from rat aortic rings in collagen gel.  IGF-I and PDGF also induced sprouting
of endothelial cells by with fibroblastic proliferation.  Its effect on
in vivo and in vitro models of other tissues has also been demonstrated.
<p>
It also effects gene expression of serine protease urokinase-type and tissue-type
plasminogen activators (PA) and also PA inhibitor 1 (PAI-1).  Metalloproteinase
interstitital collagenase expression is increased in HUVECs but not skin fibroblasts.
These enzymes would permit endothelial cell migration.  Additionally VEGF
appears to stimulate hexose transport, likely to areas where proliferation will
take place at a rapid rate.
<p>
VEGF also promotes expression of VCAM-1 and ICAM-1 in endothelial cells, which
might lead to adhesion of activated NK cells (cells expressing CD18 and VLA-4);
consistent with a finding that IL-2 activated NK show adhesion to tumor
vasculature.  VEGF may promote chemotaxis of monocytes, and it may induce
colony formation in mature forms of granulocyte-macrophage progenitors already
stimulated by CSF.  VEGF receptors are present in endothelial and hematopoietic
cells which have a common origin in early embryogenesis.  VEGF might inhibit
maturation of antigen-presenting cells (dendritic cells) while having no effect
on the mature cells (does this allow tumors to avoid being taken out by
immune cells?).  VEGF-induced effects on vasodilatation and hypotension and
decreased cardiac output may be consequences of its primary action related to
decreased venous return because of endothelial cell-derived nitric oxide (NOx)
affecting hemodynamics by changes in vascular permeability.
<p>
The hVEGF gene is about 14 kb long, containing 8 exons and 7 introns, and located
on chromosome 6p21.3.  Four VEGF cDNAs have been identified and may code for
forms that are 121, 165, 189, and 206 amino acids long.  Alternative exon
splicing accounts for the different forms.  VEGF-165 does not contain exon 6,
and VEGF-121 is missing exons 6 and 7.  VEGF-189 has 24-residues added which
are very basic, and VEGF-206 has another 17 amino acids.  There is no intron
between the 24-aa of VEGF-189 and the 17-aa insertion in VEGF-206.  The 
5&prime; end of the 51 bp (17 aa) making up VEGF-206 begins with GT, consensus
for 5&prime;-splice doner for mRNA processing.  Transcriptional control may be
through Sp1, AP-1 and AP-2 with their elements in the promoter region.
<p>
Practically speaking VEGF-165 is the dominant form both in normal and transformed
cells, although VEGF-121 and -189 transcripts are observed.  VEGF-206 has only
been observed in fetal liver cDNA library.  The mouse gene is also an 8 exon
gene about 14 kb long and it generates three forms: VEGF-120, -164, and-188,
all one amino acid less.
<p>
Native VEGF is a quite basic homodimeric glycoprotein about 45 kDa, which
correspond to VEGF-165.  VEGF-121 is weakly acidic and does not bind heparin,
while the other three bind heparin with increasing ability as the insertions
are of highly basic amino acids.  VEGF-121 is freely soluble while VEGF-165
is secreted but likely bound to extracellular matrix.  VEGF-189 and -206
are likely completely bound to matrix and immobile.  heparinases and/or soluble
forms of heparin could make the larger forms soluble and bioactive.  Plasmin
might degrade the basic C-terminus of VEGF to produce a soluble 34 kDa fragment
(two 17 kDa of the homodimer?):  these are the first 110 N-terminal amino acids.
It is noted that VEGF-165 has 50 times more potency than VEGF-121 or -110
however.
<p>
Cell hypoxia in a major upregulator for VEGF expression in both normal and
transformed cell types.  Similarities may exist for its transcriptional control
for erythropoietin too.  Deletion and mutation analysis shows a 28 bp sequence
in 5&prime; promoter of rat and human gene controlled by hypoxia.  This 
corresponds to the binding site of hypoxia-inducible factor 1 (HIF-1) seen 
in Epo gene.  HIF-1 is a basic, heterodimeric helix-loop-helix binding 
protein:  sequences in the promoter/transcriptional region of VEGF controlling
a reporter and co-transfected with expression vectors for HIF-1 were able to
upregulate the reporter.
<p>
Human keratinocytes treated with serum, EGF, TGF-&beta;, or kerationcyte
growth factor show increased expression of VEGF mRNA.  Primary (nontransformed)
keratinocytes upreglate VEGF when treated with TGF-&alpha;.  All these results
may show that growth factor-induced angiogenesis is probably mediates not
diretly but throuugh VEGF. [More information about VEGF expression in 
transformed cells is given.]
<p>
<b>Receptors</b>.  There are two classes of high affinity receptors, one with 
<i>K</i><sub>d</sub> of 10 pM and the other 100 pM.  They have masses of 
180-220 kDa.  Mononuclear phagocytes have lower affinity receptors, which 
probably affect chemotaxis.  In situ autoradiography shows that binding
sites in fetal and adult are localized to vascular endothelum of large and
small vessels.  Flt-1 and Flk-1/KDR receptors are tyrosine kinases which
are high affinity.  Flk-1 (fetal liver kinase) is murine homologue to human
KDR, with 85% sequence identity.  Both have 7 Ig-like domains in extracellular
region and a Tyr kinase domain interrupted by kinase insert domain.  Flt-1
has highest affinity for rhVEGF<sub>165</sub> (Kd ~ 1--20 pM), while KDR
has lower affinity with Kd ~75-125 pM.  A soluble Flt-1 (sFlt-1) lacks a
7th Ig-like domain and binds VEGF but inhibits its action since it cannot
form an activated dimer with another VEGF-Flt-1 bound complex.  Flt-4
is another receptor specific not for VEGF but rather VEGF-C or VEGF-related
peptide (VRP).
<p>
These different receptors may use different signal transduction pathways.
In cells normally lacking VEGF receptors but transfected with plasmids
control expression of KDR or Flt-1, if KDR is used, chemotaxis is seen, but
if Flt-1, there is no response.  In fact Flt-1 may not act to induce
mitogenesis, which might be the role of KDR/Flk-1.  Placental growth factor
(PlGF) has high affinity to Flt-1 but not KDR cannot induce mitogenesis or
permeability enhancment, nor cannot it induce Tyr phosphorylation in endothelial
cells.  Rather, the binding to sFlt-1 may be a negative modulator in complexing
with KDR.  In another way, Flt-1 may cause signalling of different processes
distinct from mitogenesis/permeability.  Using yeast two-hybrid, Flt-1
and p85 subunit of PIP2 3-kinase, and interaction abolished by a Y1213F 
mutation, indicating a P-Tyr creates that interaction.  Receptor expression
regulation shows first they are expressed primarily in vascular endothelium.
Control of transcription is in a 1 kb 5&prime;-flanking fragment for Flt-1
and 4 kb fragment for KDR.  Hypoxia seems to induce receptors along with VEGF,
as well as forms of hypoxia (myocardial infarction).
<p>
The ligand-binding 7 Ig-like domains of receptors has been studied.  To compare 
structure and function, deletions and substitutions were done on the sequences. 
Deletion of 2nd Ig-like domain in Flt-1 abolished VEGF binding. Putting the 2nd 
domain of KDR into the Flt-1 (domain swapping) restores VEGF binding.  PlGF, 
which can displace VEGF from native receptor, is unable to displace VEGF in the 
substitution mutant.  In studies with a chimeric Flt-4 receptor, both VEGF and 
PlGF induced increased DNA synthesis.  Mutagenesis studies on VEGF have 
determined which regions/positions are critical to receptor binding: 
alanine-scanning analysis was done to identify positively charged surface in 
VEGF mediating binding, and a model of the PDGF-BB crystal structure was 
studied.  VEGF R82, K84, and H86 in a hairpin loop were found necessary for KDR 
binding, and D63, E64, and E67 were found necessary for binding Flt-1. R82A, 
K48A, and H86A all decreased KDR binding.  Triple mutants R82A|K84A|H86A VEGF 
and R82N|I83L|K84S VEGF showed minimal binding (latter creates a glycosylation 
site).  It was also shown that mutants affecting Flt-1 binding did not affect
activity of mutant VEGF as a mitogen, which it exerts through KDR.
<p>
<b>VEGF-Related Molecules</b>.   Three genes in mammals have been found that
are VEGF-related.  One is placental growth factor (PlGF), and the other two
are VEGF-B and VEGF-C (also called VEGF-related protein or VRP).  Two genome
sequences in parapoxvirus <i>orf</i> also have homology.  Multiple alignment
with VEGF-165 shows the identity.  PlGF has 53% identity with the PDGF-like
region of VEGF.  PlGF appears to bind to Flt-1 but not KDR, and it does not
induce mitogenesis.  PlGF will form a heterodimer with VEGF that is mitogenic
although with much less activity.
<p>
The receptor Flt-4 has a similar structure to Flt-1 and KDR but appears to 
bind to VEGF-C/VRP.  VEGF-C is a 399 residue long polypeptide with 32% identity
to VEGF: its C-terminal portion has 180 aa not found on VEGF but has Cys-rich
motifs.  VEGF-B is 188 aa long including its signal peptide, and is reported to
stimulate human and bovine endothelial cells.  Its distribution is in skeletal
and cardiac muscle and VEGF is coexpressed.  It may be involved in muscle 
angiogenesis.  As for the parapoxvirus genes, this virus affects goats and sheep
and occasionally humans and its lesions show microvascular proliferation in the
skin.
<p>
<b>Knockout studies</b>.  VEGF and its receptors are absolutely essential, being 
lethal in early embryogenesis.  Mouse embryos stop developing at day 8.5 to 9.5 
in utero with Flt-1 knockout.  Endothelial cells are detected in embryonic and 
extraembryonic tissue but there is no organization of vasculature. Flk-1 gene 
knockouts show neither vasculogenesis or blood islands, and hematopoietic 
precursors are disrupted.  Death also occurs in day 8.5 to 9.5.  VEGF knockouts
show similar effects, but with the finding that heterozygous knockouts
(VEGF +/-) died at day 11 to day 12.  Embryo growth was retarded and showed
developmental anomalies.  Significant evidence of vasculature defects were 
present everywhere.  Forebrain and neuroepithelium structures were disorganized
and showed signs of apoptosis.  As VEGF mRNA was detected in heterozygotes,
it was the case that abnormal development is related to gene dosage and not
maternal imprinting.
<p>
<b>Pathologies:  Tumor Angiogenesis</b>.  Tumors in numerous tissues have been
shown by in situ hybridization studies to express VEGF mRNA.  Only in one type
of breast cancer and bladder cancer was it not found (although it has been found
in other types).  Vascularity of tumor and VEGF mRNA were correlated, and the
mechanism is related to hypoxia.  Comparison of VEGF and PlGF co-expression 
revealed that VEGF correlated with malignancy and vascularity.  The VEGF mRNA
was in tumor but not endothelial cells, while Flt-1 and KDR mRNA upregulation
was in endothelial cells.  An exception is angiosarcoma where the cells are
derived from endothelial cells, in which VEGF and Flt-1 mRNA were co-expressed,
indicating autocrine rather than paracrine activation.
<p>
Cancer patients show elevated VEGF levels.  Studies on tumors where VEGF 
expression is high in tumor showed a greater likelihood of relapse and was
associated with a poor prognosis.  In tumor studies with nude mice, anti-VEGF
mAbs were used to study biological effect in adult organisms with respect to
tumorigenesis.  Several tumor types were injected along with antibodies.  
Growth of tumors was inhibited from 70-95%.  Combined use of anti-VEGF with
anticancer treatments like doxorubicin showed full regression of tumors such 
as MCF-7 breast carcinoma in nude mice.  Because injections of anti-VEGF did
not affect normal physiology, it is believed that tumors require high VEGF
levels in order to thrive and metastasize.
<p>
<b>Recombinant VEGF in Therapy</b>.  In some cases VEGF might be useful for 
ischemic limbs.  Intra-arterial or -muscular administration of rhVEGF-165
might be useful in these conditions where other interventional therapy does
not exist.  In a dog model of coronary insufficiency, the left circumflex
coronary artery was occluded and intraluminal injection of rhVEGF done 
distal to occlusion with enhancement of collateral blood flow demonstrated.
Studies with all sorts of coronary artery conditions using VEGF show promise.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Sander OI, Rensing C, Kuroda M, Mitra B, Rosen BP
 </span>
 <span class="year">
(1997)
 </span>
 <span class="jname">
J. Bacteriol.
 </span>
 <span class="jvol">
179,
 </span>
 <span class="pages">
3365-3367.
 </span>
<p class="reftitl">
Antimonite Is Accumulated by the Glycerol Facilitator GlpF in <i>Escherichia
coli</i>
<p class="Para1">

Resistance to Sb(III), As(III), and As(V) is encoded by the <i>ars</i>
resistance operon found on both chromosome and plamsid. The genes are active 
transporters that extrude the metalloids. In the bacteria phosphate 
transporters cause the uptake (in E coli, it is Pit and Pst). For this work E 
coli strain AW3110 was used as it lacks the <i>ars</i> operon in the chromosome.
 It was infected with &lambda;b221 rex::Tn<i>phoA</i>. This transposon creates 
a fusion protein with the downstream encoding the catalytic function of 
alkaline phosphatase but its signal peptide for export to periplasmic space is 
removed; expression of APase would be assumed to include a function that 
exports the fusion protein into the periplasmic space. Selection was with LB 
agar containing either potassium antimonyl tartrate or sodium arsenite at 1 mM 
+ 35 &micro;g/ml kanamycin. One mutant was obtained on antimonite-containing 
agar. The Sb-resistant strain OSBR1 was white when plated with 20 &micro;g/ml 
NCIP, indicating no alkaline phosphatase activity: this might be due to
intracellular localization of enzyme, out-of-frame fusion, or fusion with
reading frames in inverse orientation. P1 phage was then used for generalized
transduction: all transductants were Sb(III) resistant. A Southern
hybridization was done on BamHI-digested genomic DNA of OSBR1, using AW3110 as
a control, and a probe of 485 bp Tn<i>phoA</i>. The blot indicated that there
was only one Tn<i>phoA</i> insertion. There is no <i> Bam</i>HI site between
the fusion in Tn<i>phoA</i> and kanamycin phosphotransferase, and there is a
BamHI site following the 3&prime; end of kanamycin phosphotransferase,
chromosomal DNA of OSBR1 was digested with BamHI.  DNA proximal to the fusion
junction was cloned into pUC18 BamHI site, and transformed colonies screened
for Km<sup>r</sup>.  The resulting plasmid pOSBR1 has a 5.7 kb insert composed
of 800 bp of E coli chromosomal DNA and 4.9 kb from Tn<i>phoA</i>.  Sequence
analysis shows Tn<i>phoA</i> was inserted at 88 min in <i>glpF</i> gene.
A Qiagen kit was used to purify plasmid DNA and sequencing on Pharmacia
Cy5 autosequence kit and ALFexpress apparatus using dideoxy method; primers
were the end of the <i>phoA</i> gene and the BamHI site of the vector.
The <i>Bam</i>HI site was found inserted within <i>glpK</i> gene, and the
phoA gene was funed near codon 170 of glpF but in opposite orientation of
<i>glpF</i> reading frame, which is why colonies were white on indicator agar.
<p>
Strains of AW3110 and OSBR1 were streaked on M9 medium with either 0.2% glucose
or 0.5% glycerol as sole carbon source.  The parental strain grew on both media,
but the mutant only grew on Glc.  The failure of mutant to grow on glycerol is
most likely a polar effect related to <i>glpK</i> expression than transport defect.
An OD600 vs. dose response (range 0-1 &micro;M) on LB medium with potassium
antimonyl tartrate, antimony trichloride or sodium arsenite for 24 h 37&deg;
showed the AW3110 wild type is sensitive to Sb(III) but the mutant was
resistant.  The resistance was not related to what Sb(III) was complexed with
(tartrate or chloride).  Both mutant and wildtype were sensitive to As(III)
however, and so As(III) is likely not taken up by GlpF but by another route.
The MIC was determined for other metals (Cu, Cd, Zn, Ni, Cr) between AW3110
and OSBR1 and they were identical, so the GlpF is specific for Sb(III) transport.
<p>
Cells with normal GlpF can be plasmolyzed more rapidly by osmotic shock dilution
containing GlpF substrate xylitol than cells with defective GlpF.  The swelling
of cells is observed by decreased 600 nm light scattering.  Strain AW3110 has
a half time of swelling that is about 50 s, which is consistent with a reported
value of 0.7 min.  OSBR1 showed no optical decrease in the same period after
mixing with xylitol.  Testing for Sb(III) uptake was not possible since
Sb radioisotopes are not available, and antimony salts are not soluble at
concentrations required for optical measurement of GlpF activity.
<p>
At neutral pH in the environment, Sb(III) exists as the uncharged Sb(OH)<sub>3</sub>.
This is enters the cell via facilitated diffusion by GlpF, which sees it as a
polyol much like xylitol or glycerol.  Once in the cell, Sb(OH)<sub>3</sub>
complexes with H<sub>2</sub>O and loses a proton to form the anion
Sb(OH)<sub>4</sub><sup>&minus;</sup>.  This anion is driven out electrophoretically
by the ArsB membrane protein which also moves out other metalloids.  The
driving force is a transmembrane potential with a negative charge intracellularly
and positive charge extracellular:  the potential is created by the
respiratory chain driving protons out extracellularly (NADH + H<sup>+</sup>
(in cell) &rarr; NAD<sup>+</sup> + H<sup>+</sup> (out of cell)).
<!-- ============================================================= -->
<p class="tags">
Toxoplasma, ELISA
<p class="ref">
 <span class="authors">
Andrews CD, Dubey JP, Tenter AM, Webert DW
 </span>
 <span class="year">
(1997)
 </span>
 <span class="jname">
Vet. Parasitol.
 </span>
 <span class="jvol">
70,
 </span>
 <span class="pages">
1-11.
 </span>
<p class="reftitl">
<i>Toxoplasma gondii</i> recombinant antigens H4 and H11:  Use in ELISAs for 
detection of toxoplasmosis in swine
<p class="Para1">
Toxoplasmosis is world-wide zoonosis.  Infection is by ingestion of oocysts 
usually from cat feces or by eating meat of infected animals where the food 
preparation did not affect viability.  In particular pigs get infected and pork 
is a source of infection.  Porcine serological testing has unknown sensitivity 
and specificity.  In a recent test of several immunoassays validated by tissue 
bioassay (see Dubey et al 1995) the best was a modified agglutination test (MAT) 
using formalin-fixed tachyzoites, but the assay is expensive and not widely 
available.
<p>
Next best is an ELISA based on tachyzoite antigens where the tachyzoite is 
cultured in animal hosts or mammalian cells.  False positive and false negative 
reactions abound in with these antigens since there are cross reactions with 
epitopes common to other organism as well as <i>T gondii</i>, or there are 
reactions with proteins from the hosts used for in vitro or in vivo culture.  If 
recombinant antigens are used instead this can reduce the false positives at 
least.  In the early 1990s several groups attempted the use of recombinant 
antigens.
<p>
Native antigen extract was prepared from <i>T. gondii</i> tachyzoites obtained 
from Microbiological Research Corp (Utah).  Tachyzoites were sonicated, spun 
2000 <i>g</i> g for 15 min and supernatant stored at &minus;80&deg; with protein 
measure at 3.3 mg/ml.  A dilution of 1:400 was used for coating ELISA plates. 
Recombinants H4 and H11 were prepared as GST-fusion expressed in <i>E coli</i> 
hosts (Tenter and Johnson 1991).
<p>
ELISA was formatted in Falcon Pro-Bind MT plates in which Ag is coated in 100 
&micro;l pH 9.6 carbonate at its dilution and left overnight at 4&deg;. Storage 
of coated plates is in refrigerator in humidified chamber and used in 3 weeks. 
All reagents are delivered at 100 &micro;l per well and incubated 60 min at RT. 
Ag-coated plates are blocked with 1% BSA to stop nonspecific binding.  Washing 
between reagents is with PBS (pH 7.4) + 0.5% Tween 80 (PBS-T) with triplicate 
washes.  The ELISA begins with test serum suspected of holding anti-T gondii Ab. 
Then HRP-conjugate anti-swine Ab (IgG) is added (used 1:5000 in this method). 
Development is a 1-step ABTS peroxidase substrate for 10-15 min.  Reader is 
configured for 405 nm absorbance against a saline control blank. Positive and 
negative control sera are measured at 1:x where x is 20, 40, 80, 160, and 320. 
The coating antigen was also at different concentrations (10.0, 3.3, 1.1, and 
0.37 &micro;g/ml).  The control sera came from pigs that were experimentally 
infected or not infected, and they had been tested for <i>T gondii</i> Ab. 
Seven pigs were orally inoculated with 10,000 oocysts of the GT-1 strain of <i>T 
gondii</i> at 2 months of age (this is a strain causing virulence in mice). 
Blood was collected at inoculation and at 4, 5, 6, 7, 9, 11, 14, 19, 30, 36, and 
42 weeks thereafter (or until necropsy) into serum tubes, allowed to clot, and 
then centrifuged.  Serum was stored at &minus;20&deg; and repeatedly thawed and 
refrozen for multiple assays.  A MAT titer had been determined on these sera. 
One pig was negative in the bioassay but show high titer in the MAT and native 
antigen ELISA. Another pig died early and a substitute had to be brought in (no 
necropy and thus bioassay was done).  The study used the positive and negative 
controls to find optimum Ag dilution, and this dilution tested against high 
titer sera (instead of positive control), and pre-inoculation pool was used 
instead of negative sera.  Glutathione S-transferase was also titrated as a 
separate protein to see if it contributed to the background (it is part of the 
fusion).
<p>
<b>Result</b>.  Optimum dilutions for the recombinant antigens prepared was 
found to be 1:40.  There was wide variability in response among the test pigs: 
Pigs 1, 7, and 8 were the strongest responders and their sera at 35 and 40 days 
after infection was pooled as a positive control. Plots of OD<sub>405</sub> vs. 
[Ag] where Ag=H4 and Ag=H11 are presented in Fig 1.  At 1 &micro;g/ml H4 and 
about 4 &micro;g/ml H11 signal does not increase (1.2 OD for H4 and 1.2-1.4 for 
H11) using the 3-pig serum pool.  The pool of their preinoculation sera show an 
increase that goes up to 0.3-0.4 OD for 8-64 &micro;g/ml H4, and 0.6 OD for 
16-64 &micro;g/ml H11. Maximum signal difference for H4 was at 1 &micro;g/ml and 
for H11 was at 4 &micro;g/ml.  The optimized [Ag] was selected at 2.0 
&micro;g/ml for pooled H4-H11 antigen ELISA.  The pooled Ag ELISA gives a higher 
signal (OD) up to 1.6 OD, achieved by 2-4 &micro;g/ml protein.  Titration of GST 
in showed it that the positive and negative sera curves were fairly close, 
indicating reactivity had little to do with GST component.
<p>
The H4- and H11-ELISAs were used to measure titers over time (from 0 to 42 
weeks).  Plots of individual and pooled sera are shown in Fig 3 (OD405 vs weeks 
post incubation).  Pooled sera always gave a lower signal than some select 
individual sera of high responders.  Generally titers remain high for about 4 
months before declining.  In Fig 4 the OD vs time plot was done for the 
H4/H11-ELISA and shows again the pooled antigens produce stronger signals.
<p>
Native antigen extract was also used in the ELISA and OD signals increase 
sharply over time.  Results are comparable to MAT for chronic infection sera.
<p>
These results show that use of individual antigens even in a mix can be 
different than when using native antigen extract.  Thus some antigens may be 
expressed early in infection and not be present over the long course of an 
infection.   All but one animal were positive for infection in necropsy, so a 
correct mix  of antigens in early and late infection phases is necessary to 
avoid false negatives (low sensitivity).
<p>
In ELISA test for other animals, a mixed H4/H11-ELISA showed 99% specificity and 
95% sensitivity for cats.  For sheep an assay of 79% sensitivity and 80% 
specificity  was found.  In humans with acute toxoplasmosis, only 68% of sera 
reacted with one or both antigens, while only 14% with chronic infection reacted 
with either antigen. Thus H4 and H11 may only be acute stage proteins.  DNA 
fragment for antigen 11 is the 3' end of gene coding for dense granule protein 
GRA4, which reacts with mucosal Iga in serum, intestinal secretions, and in milk 
from mice and is probably involved in early stages of host-parasite reaction. H4 
antigen identity is unknown but in cat studies H4 is more reactive than H11, 
although it does react with sera from mice immunized with a <i>T gondii</i> 
isolate from sheep.  Hence a goal is to obtain a recombinant antigen mix that 
performs as well as native antigen extract.
<!-- ============================================================= -->
<p class="tags">
HIV, occupational exposure
<p class="ref">
 <span class="authors">
Pinto LA, Landay AL, Berzofsky JA, Kessler HA
 </span>
 <span class="year">
(1997)
 </span>
 <span class="jname">
Am. J. Med.
 </span>
 <span class="jvol">
102,
 </span>
 <span class="pages">
21-24.
 </span>
<p class="reftitl">
Immune Response to Human Immunodeficiency Virus (HIV) in Healthcare Workers 
Occupationally Exposed to HIV-Contaminated Blood
<p class="Para1">
Exposure to HIV does not always cause seroconversion or infection as assessed
by ELISA, western blot, or PCR evaluations.  It can induce HIV-specific
cell-mediated responses however.  The role of cellular response to HIV exposure
are involved in four areas of study:  
<ol>
<li>the acute phase of infection where HIV antigens invoke strong cellular
immune activity coinicident with drop in plasma viral load
<li>HIV-infected individuals who show no disease progression over the long
term and have stable CD4<sup>+</sup> T lymphocyte (CTL) counts
<li>infants who are sero- and PCR-positive but who sero-revert and are later
virus-negative
<li>individuals who show no evidence of infection but have been exposed to
HIV multiple times
</ol>
<p>
The last category includes high-risk groups such as sexually active gay men,
prostitutes, newborns from HIV-infected mothers, discordant sexual couples,
and occupationally exposed health care workers (HCWs).  HCWs who have 
percutaneous single exposure of finite magnitude and duration show seroconversion
rates of 3 in 1000.
<p>
21 of 28 (75%) of HCWs who had been exposed to HIV-contaminated blood were 
injected with HIV <i>env</i> protein and had an increased IL-2 production. 9 of 
38 (24%) of HCW exposed to HIV-seronegative blood also responded to <i>env</i> 
quite unexpectedly, and 3 of 35 (9%) of healthy blood bank donors responded also 
to <i>env</i>.  CTL responses were tested and positive in 7 of 20 (35%) of HCWs 
who had been exposed to HIV-positive blood, but there were no CTL responses in 
20 HCWs who had been exposed to uninfected blood or in the seven unexposed blood 
blank volunteers.  CTL response is more specific than IL-2 assessment.  IL-2 
responses (as measured by CTL proliferation based on radioisotope incorporation 
using precursor nucleotides) in response to HIV envelope peptides of 
CD4<sup>+</sup> and CD8<sup>+</sup>-purified T cells from one HIV-exposed but 
seronegative HCW shows that CD4<sup>+</sup> were the major IL-2 producers 
(controls were RPMI medium as negative and PHA-stimulated cells as positive: 
which also stimulates IL-2 in CD8<sup>+</sup> cells). CTL response were MHC 
class-I restricted and required in vitro presence of the HIV peptides.  The MHC 
restriction is proved by showing that antigen kills autologous Ag-stimulated 
Epstein-Barr virus transformed targets but not human leukocyte antigen class 
I-mismatched targets.  Pre-incubation of autologous targets with anti-class I 
mAb W6/32 completely blocked the CTL activity against peptide-pulsed EBV 
targets, further proof of HLA-restriction of these responses.  CD8<sup>+</sup> T 
cells also showed specific activity to T1 and PMN 18 peptides in a worker 
exposed to HIV-1 contaminated blood 425 days earlier.  That this is found 
indicates that HIV replication occurred to a limited extent as Ag presentation 
requires endogenously generated peptides in association with self class I 
molecules. Similar studies with macaques using SIV show these cellular 
responses, with long-lived memory cells, and that they protect against 
subsequent challenges with low inocula.  Further proof of CTL activity is from a 
vaccine trial in HIV-infected volunteers in which recombinant gpl60 in low doses 
induced T helper cell responses in absence of antibody production. Thus cellular 
immunity might protect occupational low dose exposure without leading to 
seroconversion.
<!-- ============================================================= -->
<p class="tags">
HBsAg, immunoaffinity purification
<p class="ref">
 <span class="authors">
Fern&aacute;ndez de Coss&iacute;o ME, D&iacute;az T, Galv&aacute;n A, 
Vald&eacute;z R, Gonz&aacute;lez E, Ayala M, D&iacute;az J, Bestagno M, Burrone 
O, Gavilondo J
 </span>
 <span class="year">
(1997)
 </span>
 <span class="jname">
J. Biotechnol.
 </span>
 <span class="jvol">
56,
 </span>
 <span class="pages">
69-80.
 </span>
<p class="reftitl"> 
Antigen recognition characteristics and comparative performance in 
immunoaffinity purification of two monoclonal antibodies specific for the 
hepatitis B virus surface antigen
<p class="Para1">
Industrial scale purification of rHBsAg described by Agraz et al (1994) is done 
with the mAb CB-Hep.1 produced by Fontirrochi et al (1993), and this is used
for the Heberbiovac HB (Havana) vaccine.  A celite extract of the recombinant
yeast is passed through immunoaffinity step to increase purity (10-25% in the
feedstock) to 70-80% with a 60% yield (Penton et al 1994 have characterized the
purified material).  Other hybridoma clones are now studied for their secreted
mAbs,  in particular a CB-Hep.0 clone with IgM isotype was studied.
<p>
A competition ELISA compares how the two mAbs work.  mAbs are conjuated with 
NHS-biotin and ELISA plates coated with rHBsAg.  Mixtures of the biotinylated 
mAbs (0.25 &micro;g/ml) at constant concentration and unconjugated mAbs at 
varying concentration (0.25 to 2.5 &micro;g/ml) are incubated 1 h/37&deg; and 
then the biotinylated mAb detected using Strepavidin-HRP.  Inhibition of the 
binding of conjugated mAb by the unconjugated was always &lt;20% with respect to 
control (each antibody against itself).  Residual inhibition might be explained 
by large size of IgM imposing a steric inaccessiblity to antigen by the IgG. To 
understand what epitope was involved, rHBsAg was reductively aklylated and 
detergent-treated (SDS), and this caused substantial loss of mAb binding.
When alkylation was not done, the CB-Hep.1 IgG binding was not lost but the
CB-Hep.0 IgM binding was.  In a western blot (where rHBsAg is boiled in 
2-mercaptoethanol reductant and treated with SDS) the IgM mAb loses its ability
to bind but not the IgG mAb, which identifies both the mono- and dimeric forms
of HBsAg.
<p>
The 226-aa HBsAg polypeptide has four transmembrane domains and two hydrophilic
loops, one oriented toward the virion core and the other outside the particle
(Stirk et al 1992).  All isoforms of the HBsAg have a common eptiope <i>a</i>
determinant, and two sets of subtype determinants, either <i>d</i> or <i>y</i> and
<i>r</i> or <i>w</i>, leading to four possible subtypes: <i>adw</i>, <i>ayw</i>,
<i>adr</i>, and <i>ayr</i>.  The <i>a</i> determinant is bordered between 
residues 101-159, which is Cys-rich and likely to involve multiple disulfide
bond interactions.  Reductive alkylation has been known to cause loss of HBsAg
immunogenicity.   Two important disulfide bonds are between C124-C137 and
C139-C147 (Dreesman et al 1982).  Qui et al have indicate that the C(K/R)TC
between 121-124 positions is significant for <i>a</i> determinant.
<p>
Using synthetic peptides that form an overlap of the <i>a</i> determinant region
<p style="white-space:pre;font-family:'Courier New',Courier,monospace;">
    <sup>118</sup>TGPCKTCTTPAQGNSMFPSCCCTKPT<sup>143</sup>           [D4-13]
      TGPCKTCTTPAQGNSMFPSCCC                  [D12-2]
      TGPCKTCTTPAQGNSMFPSC                    [D12-3]
         CKTCTTPAQGNSMFPSCCC                  [D12-4]
            CTTPAQGNSMFPSC                    [D12-6]
                      FPSCCCTKPTDGNCTCIPIP    [D4-18]
<p>
it was found that CB-Hep.0 mAb bound well to peptide D12-4 (aa 121-139) and 
CB-Hep.1 bound D12-6 (aa 121-137) with greater affinity.  CB-Hep.0 did not bind
as well if Cys138 and Cys139 were omitted.  The four residues CKTC were critical
for CB-Hep.1 binding.  Rooman et al (1991) have done an ab initio prediction of
3D structure to show that Cys121 and Cys124 likely interact and create a 
&beta;-turn type I conformation, and evidence is offered that these two Cys
residues form the disulfide.
<p>
For immunoaffinity purification, the CB-Hep.0 IgM requires 3 M potassium 
thiocyanate in the same way that the CB-Hep.1 IgG requires.  The IgM column
gave higher capacity and yield (85% vs 44% for the IgG mAb).  It is not strictly
because of valence number being higher for IgM.  Affinity constants for the 
CB-Hep.0 IgM was 1.9 &times; 10<sup>10</sup> M<sup>&minus;1</sup> and 
CB-Hep.1 was 4.6 &times; 10<sup>10</sup> M<sup>&minus;1</sup>.   As the constant
is more indicative of avidity than affinity, there may be other factors.
The fact that IgM and IgG recognize different features of the epitope may
account for differences.
<p>
A scFv fragment of CB-Hep.1 IgG mAb was prepared by this group and checked for
the ability to purify rHBsAg.   Total RNA was isolated from the CB-Hep.0 
hybridoma cell line using guanidinium thiocyanate-phenol-chloroform, and
heavy and light chain cDNAs synthesized;  amplification of the cDNA by PCR
was done using primers from Kabat et al 1991 using conserved sequences of
FR1 and CH1&mu; and Ck of mouse Igs.  The primers are:
<pre>
  Light &kappa; chain:
      5&prime;-GATATTGTGATGACCCAGTCTCCA-3&prime;   sense primer
      5&prime;-TGATACAGTTGGTGCAGC-3&prime;         antisense primer

  Heavy &mu; chain:
      5&prime;-AGGTG|CC|AAA|GCTGCAGG|CAGTCT|AGG-3&prime;   sense primer
      5&prime;-CATTTG?GAAGGACTGACTC-3&prime;         antisense primer

Base x|y indicates two-base degeneracy for the position 
with base x or y
</pre>
<p>
Amplicons were then treated with Klenow and T4 PNK to enable blunt-end 
cloning into pUC18/<i>Sma</i>I BAP vector.  The nucleotide sequences of the 
CB-Hep.0 variable regions were then determined.  The heavy chain variable
regions of CB-Hep.0 and CB-Hep.1 showed 81% identity.  These derived from the
germline VHJ606 and Jh2 genes (see Kasturi et al).  The VH gene of CB-Hep.0
has 12 amino acid changes that vary from VHJ606, with six changes occurring in
CDR1 and CDR2.  The VH gene in CB-Hep.1 has 18 amino acid changes, eight in CDR1
and CDR2.  Other changes are randomly distributed in framework regions.  As for the
light chain variable regions, they use different germline genes.  DNA and amino
acid homology is &lt; 70% 

<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Hofbauer R, Hofbauer L, Moser D, Salfinger H, Speiser W, Frass M, Kapiotis S
 </span>
 <span class="year">
(1998)
 </span>
 <span class="jname">
Life Sci.
 </span>
 <span class="jvol">
63,
 </span>
 <span class="pages">
1725-1737.
 </span>
<p class="reftitl">
Leukocyte Migration:  A New Triple Migration Chamber Assay Allows
Investigation of Various Cell Interactions Simultaneously
<p class="Para1">
The principle of this assay involves creating a three-chambered system
within a culture environment.  The upper chamber is composed of an
open ceiling with a floor containing a macroporous filter upon which
endothelial cells are grown to confluence.  The porosity of the filter
and its holder is such that cells can traverse through the pore.
The filter holder in this system then fits within another filter holder
system, which forms the middle chamber.  The floor of the chamber contains
a filter porous to soluble factors but not to cells (such that they can
migrate through the pore).  The holder-within-holder system then fits in
a well.  Enough spacing from ceiling to floor of each chamber is provided
by the system.  The lower chamber has at its ceiling the floor of the
outer filter holder, and as the floor the bottom of the well itself.
Cells are grown on the bottom of the well, and they serve to provide
a chemoattractant that diffuses upward through the floor of the middle
chamber and to the upper chamber, and the cells migrate to the middle
chamber.  The cells migrating the middle chamber are assayed, directly
or indirectly.
<p>
In the system devised here, smooth muscle cells are grown on the well bottom
to confluence.  At the same time, endothelial cells are grown to confluence
on a filter that will fit to the floor of the upper chamber.  On the day
of the assay, smooth muscle cells will be treated with TNF-&alpha;, which
will cause them to release a factor that stimulates migration of leukocytes.
These leukocytes are added to the upper chamber atop the endothelial cells.
As the chemoattractant is released from SMC at the bottom of the well,
leukocytes will attempt to transit across the endothelial cell layer, into
the middle chamber.  They however cannot pass from the middle chamber,
and so it is the cells making it to the middle chamber which are counted.
<p>
The lower filter holder has a 23 mm height and 12.5 mm diameter.
The filter inserted had a 17.5 mm height and 9 mm dia.  The height from the
bottom of the well to the floor of the middle chamber is 11 mm.
<p>
HUVEC were isolated from umbilical veins using collagenase type IA,
then cultured in M199 medium with 5% heat-inactivated FBS (56&deg;
for 30&nbsp;min) + 10 ng/ml ECGF (Technoclone, Austria), 1 &micro;g/ml
hydrocortisone (Sigma), antibiotics in 5% CO<sub>2</sub> at 37&deg;.
Substratum was fibronectin-coated 6-well plates.  HUVEC are then
transferred to migration chambers of the triple chamber.  A monolayer
forms after 3 days on filters, and viability is assessed using
calcein-acetomethylester fluorescence (Mol Probes).
<p>
Arterial smooth muscle cells from human umbilical artery (HUSMC)
can be obtained by cutting the arteries longitudinally and
removing endothelium with scalpel.  The arteries are cut into
1 mm<sup>3</sup> disks, and the pieces washed in M199 medium with
10% FCS and moved to 6-well plates, 4 disks per well.  Cells emerge
from the explant disks after 3-7 days.  The arterial disk remnants
are removed in the 3rd week of culture and cells adhering to plastic
grow to confluence over a week.  Subculturing is every 4-5 days
using trypsin/EDTA, and diluting 1:2 in M199 + 10% FBS.  Fresh
medium is given 24&nbsp;h later (after plating), then every 48&nbsp;h thereafter.
<p>
Peripheral blood mononuclear cells (PBMNCs) are obtained from the blood of
nonsmoking healthy person without clinical inflammation and aged 25-35 y.
Whole blood is layered over Ficoll for step density separation at
500 <i>g</i> for 30&nbsp;min.  Recovered PBMNCs are washed in PBS at
150 <i>g</i> to remove platelets and resuspended in M199 + 10% FBS.
This is layered over iso-osmotic 46% Percoll containing 5% FBS, and
then centrifuged 550 <i>g</i> for 30&nbsp;min.  Monocytes are recovered at the
interface, washed in PBS, resuspended in M199.
<p>
The <b>triple chamber migration</b> system is a filter-in-filter system.
Two filters divide a well in a 24-well page into three chambers.
HUSMC were cultured on fibronectin-precoated bottoms of the 24-well plate.
A filter holder (WKI Plastics, Berlin) holds an upper filter in the well.
The holder takes a polyethylene terephthalate 0.45 &micro;m microporous
membrane to collect transendothelial migrated PBMNC;  this is the lower
filter.  The upper filter is a polyethylene terephthalate 3.0 &micro;m
pore dia. microporous membrane (Falcon).  Endothelial cells are seeded
on a fibronectin-coated upper filter.  The filter is placed in the filter
holder of the lower filter.
<p>
Once the SMC and EMC are confluent, PBMNC are seeded to the top of the
upper chambers.  Time course studies indicate that 3 hours is optimal
for the assay.  SMCs are pre-treated with TNF-&alpha; (Immunotech) at
different doses (1, 10, 100 ng/ml) for 6 h.  The collecting filter
and upper filter (contains ECM) are then placed in well.  The PBMNC
are seeded in the upper chamber and incubated 5% CO<sub>2</sub> at
37&deg; for 3 h.  Replicate wells of 10 are tested.  The upper filter
is removed and set aside.  The transmigratory cells in the lower filter
are taken to a new well (in a new plate).  The fluorescent dye
calcein-AM is added to 20 &micro;M for 30&nbsp;min in the culture incubator.
A fluorimeter can now quantitate the cells moved.  As a control, migration
in SMC not stimulated by TNF-&alpha; was assessed (the 100% control).
<p>
One control was a 100 ng/ml TNF-&alpha; placed in the lower chamber
not containing SMC layer with the cell layers above added for 3 h.
An additional control was the preincubation with TNF-&alpha; over 6 h
of the PBMNC in the incubator.  These were washed twice in M199 medium
(neat), and then added to SMC/EMC layers as usual.
<p>
Different filters and holder systems were tried.  Transwell collagen
filters (Costar)  were okay, but not best.  Costar Clear polyester filters
of small (1.0 &micro;m) to large (8.0 &micro;m) pore size were very good,
as were Falcon polyethylene terephthalate (PET) inserts (from 0.45 to 8
&micro;m pore size).
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Webb DJ, Wen J, Karns LR, Kurilla MG, Gonias SL
 </span>
 <span class="year">
(1998)
 </span>
 <span class="jname">
J. Biol. Chem.
 </span>
 <span class="jvol">
273,
 </span>
 <span class="pages">
13339-13346.
 </span>
<p class="reftitl">
Localization of the Binding Site for Transforming Growth Factor-&beta; in Human 
&alpha;<sub>2</sub>-Macroglobulin to a 20-kDa Peptide That Also Contains the 
Bait Region
<p class="Para1">
Native &alpha;<sub>2</sub> macroglobulin (a2M), methylamine-treated a2M 
(a2M-MA), and BEST were analyzed using 5% SDS-PAGE under non-reducing and 
reducing conditions and transferred to PVDF.  Note some sample were 
iodoacetamide-treated.  Membranes were stained with Coomassie blue to check if 
bands were electrotransferred equivalently. The nonreduced sample showed a 360 
kDa band, which is the dimeric form with interchain disulfides, while reduced 
sample showed a 180 kDa band.  Use of methylamine does not alter mobility. The 
gel-separated native a2m and a2M-MA were transferred to PVDF and the blocked 
membranes incubated with <sup>125</sup>I-TGF-&beta;2:  TGF&beta;2 bound to both 
native a2M and a2M-MA, and this binding was unaffected by DTT/iodacetamide 
treatment. It also bound to BSA but only when DTT had been used, and did not 
occur after iodoacetamide was used;  thus this interaction may have involved the 
interaction of free sulfhydryls.  Additional experiments with 
<sup>125</sup>I-TGF&beta;1 and -NGF-&beta; showed that it binds native a2M, and 
also that BSA binds when DTT-treated, but also loses that binding abilty when 
iodoacetamide-treated.
<p>
The mouse homolog of human a2M is MUG.  In an earlier report, the native 
tetrameric mouse binds to TGF-&beta;1 and -&beta;2 as for human a2M, but not 
with significant affinity (<i>K</i><sub>D</sub> ~ 1.0 &micro;M).  In this 
experiment with <sup>125</sup>I-TGF&beta;2, small levels of MUG could bind, but this
was abolished after iodoacetamide treatment.  Thus there is no cryptic binding with
with MUG.
<p>
Five fusion proteins were run on SDS-PAGE and blotted to PVDF.  The cDNA of a2M
obtained from ATCC in a pAT153/<i>Pvu</i>II/8 vector.  A <i>Sac</i>I digest 
showed there was a cleavage site within the coding sequence, not expected but 
sequencing revealed that position 2431 is not C but T.  A fusion of different 
a2M coding regions and GST was done.  The first FP1 was  by digesting the 
pAT-a2m with <i>Bst</i>XI to take amino acis 122-415, filling in with T4 DNA 
polymerase to create blunt ends, then ligating to pGEX-3X <i>Sma</i>I site at 
end of GST C-terminal area.  FP2 was created by digesting 
pAT-&alpha;<sub>2</sub>M with <i>Eco</i>RI and <i>Nsi</i>I for residues 364-712, 
further digested with <i>Sac</i>I to generate region between 364-613.  This 
fragment is filled in and inserted into pGEX-2T at its <i>Sma</i>I site.  FP3 
and FP4 are obtained by digesting pAT-a2M with <i>Sac</i>I for fragments 614-797 
and 798-1082, respectively.  After filling in the ends with polymerase, they are 
inserted as for FP2.  FP5 is an <i>Xho</i>I-<i>Pst</i>I fragment from 1053-1302, 
is filled in, and inserted as for FP2-FP4.  Restriction analysis was done to 
determine correct orientation.  All final constructs were transfected to BL21 
<i>E coli</i> for inducible expression under IPTG.
<p>
Binding of labeled TGF-&beta;2 was now tested for the fusion products.  They
are run in SDS-PAGE and electroblotted to PDVF.  Band mobility in the gel
was at the expected position for the predicted protein product size.  Each
band reacted with anti-GST Ab indicating it was a fusion with GST.  Reaction of
other higher MW bands with anti-GST Ab most likely indicates protein aggregates
that are not separated by SDS.  Only the FP3 fusion was shown to bind to 
the <sup>I</sup>125-TGF-&beta;2; this ligand binding was not thiol based since
all thiols were alkylated.  Also TGF-&beta;2 did not bind to purified GST.
The binding of FP3 to TGF&beta;2 occurred whether the FP3 was affinity
purifed or in complex mixture.  Labeled TGF-&beta;2 was incubated with 
PVDF-immobilized FP3 in presence of excess unlabeled TGF-&beta; or solution
phase FP3 (1 &micro;M).  Solution phase FP3 experiment inhbiited binding of
labeled TGF-&beta;1 and -&beta;2 to bound FP3 by 94 and 92%, respectively.
0.2 &micro;M unlabeled TFG-&beta;1 inhibited labeled TGF-&beta;1 to immobilized
FP3 by 72%, and unlabeled TGF-&beta;2 at 0.2 &micro;M inhibited labeled TFG-&beta;2
binding to immobilized FP3 by 90%.
<p>
0.5 nM labeled TGF-&beta;2 was incubated with 0.5 &micro;M FP3 or FP4 in 
solution, no detergents.  Superose-12 was used to separate bound complex from 
free <sup>125</sup>I-TGF-&beta;2.  It should be noted that uncomplexed 
TGF-&beta; interact strongly with Superose, and its elution is considerably 
beyond included volume.  The OD<sub>280</sub> and radioactivity counts (as 
percentage of highest of all fractions) against time or fraction chromatogram 
shows FP3 assocaties strongly with TGF-&beta; in solution, while FP4 does not. 
42% of radioactvity of TGF-&beta;2 co-eluted with FP3, whereas only 6% was 
recovered with FP4.  Because gel filtration is a non-equilibrium method for 
determining protein-protein interaction, more complex could have been present 
initially than during column flow.  It should be noted that in solution, FP3 and 
FP4 associated as dimers with masses of 95 and 107 kDa respectively;  this 
appears to be true for other GST fusion proteins as well.
<p>
Microplate wells were coated with a2M-MA and 0.1 nM <sub>125</sub>I-TGF-&beta;1 
and -&beta;2 were incubated for 60 min:  3.4 fmol of TGF-&beta;1 and 2.6 fmol of 
TFG-&beta;2 were bound to immobilized a2M-MA.  Competition with unlabeled 
TFG-&beta; at 0.2 &micro;M (2000-fold higher) forms reduced labeled binding by 
75%.  Labeled TGF-&beta; was &lt; 5% when wells were not pre-coated with 
a2M.  Competition experiments were then done with FP3, FP4, and purified 
GST:  a2M-MA was coated to well and these were added in the presence of 
<sup>125</sup>I-TGF-&beta;1 and -&beta;2 at various concentrations of the 
FP3, FP4 and GST.  FP3 inhibited binding of labeled TGF-&beta;1 and -&beta;2
in concentration-dependent manner, with nearly complete inhibition when FP3
&gt; 100 nM.  Thus and FP3-TGF-&beta; complex cannot bind a2M-MA.  The
IC<sub>50</sub> is 33 nM for TGF-&beta;1 and 26 nM for TGF-&beta;2.  If 
binding of one FP3 to TGF-&beta; is occurring, then the IC<sub>50</sub> is
an accurate estimate of <i>K</i><sub>D</sub>, whereas if more than one FP3
binds the <i>K</i><sub>D</sub> will be lower than IC<sub>50</sub>.  FP4 and
GST had no effect on TGF-&beta;2 to bind to immobilized a2M-MA, and GST
had no effect to prevent TGF-&beta;1 from binding to immobilized a2M-MA.
The solution phase bindings experiments were repeated using FP3, native
a2M, monomeric a2M, a2M-MA and thrombospondin to compete with the TGF-&beta;
forms binding to immobilized a2M-MA.  IC<sub>50</sub> values were all similar
for FP3, native a2M, and a2M-MA with TGF-&beta;2, even though intact
a2M is a homotetramer with four FP3 domains.  IC<sub>50</sub> for FP3 was
lower than for a2M-MA and substantially lower than for native a2M.  Monomeric
a2M had slightly increased affinity for TGF-&beta;1 than native a2M, again
despite the latter have 4 mol of monomer.
<p>
TGF-&beta; will stimulate a decrease in the proliferation of fetal bovine heart 
endothelial (FBHE) cells in culture.  Cells were pulse-treated with TGF-&beta;1 
or TGF-&beta;2 at 10 pM for 60 min in presence or absence of FP3 or FP4.  Then 
media was supplemented with [<sup>3</sup>H]thymidine and its incorporation into 
DNA.  [<sup>3</sup>H]Thymidine was decreased by 69 and 57% with TGF-&beta;1
and -&beta;2, respectively.  FP4 presence in medium did not change that value.
But FP3 nearly completely inhibited the action of TGF-&beta;1 and TGF-&beta;2
to reduce incorporation of thymidine, to within 3 and 6% of controls in which
only thymidine was present with no TGF-&beta; added.
<p>
RAW 264.7 cells are capable of synthesizing TGF-&beta;.  These cells also have 
an inducible nitric-oxide synthase expression system, which is auto-induced or 
auto-activated by TGF-&beta;. Previously a2M-MA was shown to induce NOS and thus 
nitrite levels in these cells. The question is what the ability of FP3 is
to induce nitrite in the RAW 264.7 cell-conditioned medium. Nitrite was measured
with different levels of FP3 and a2M-MA incubated with cells.  NO synthesis
in response to a2M-MA and FP3 were equivalent.  FP4 and GST did not show any
effect.
<!-- ============================================================= -->
<p class="tags">
Pichia pastoris, protein glycosylation, alpha-1,2-mannosidase
<p class="ref">
 <span class="authors">
Martinet W, Maras M, Saelens X, Jou WM, Contreras R
 </span>
 <span class="year">
(1998)
 </span>
 <span class="jname">
Biotechnol. Lett.
 </span>
 <span class="jvol">
20,
 </span>
 <span class="pages">
1171-1177.
 </span>
<p class="reftitl">
Modification of the protein glycosylation pathway in the methylotrophic yeast 
<i>Pichia pastoris</i>
<p class="Para1">
Huffaker and Robbins (1982) described method to isolate <i>P pastoris</i> GS115
mutants having N-glycosylation defects.  These will incorporate less 
[<sup>3</sup>H]mannose than the wild-type and maintain long term storage.
<i>S cerevisiae</i> DRI9/T7 was radiolabeled in parallel as a control in
finding lethal selection.  The mutant strain was able to modify with very low
amounts of mannose without loss of cell viability.  It was generated by 
culturing GS115 with 3% ethyl methanesulfonate (EMS) in 0.1 phosphate pH 7 for
90 min at 30&deg;, then diluting 40-fold with 5% sodium thiosulfate, spinning,
and washing three times with sterile water.  Growth is overnight on YPD at 30&deg;,
pelleting cells and resuspending in 100 &micro;l YPD + 1% glucose and 10 mCi/ml
[<sup>3</sup>H]mannose, incubating 30 min at 30&deg;, spinning three times with
sterile water and storing in aliquots (with 25% glycerol in 0.5&times; YPD) at
&minus;80&deg; for 40 d.  A frozen aliquot was taken every 10 d and plated.
This mannose suicide selection method useful for <i>S cerevisiae</i> was 
not effective in finding glycosylation mutants in <i>Pichia</i>, indicating
they may have different glycosylation pathways.
<p>
Vanadate is normally toxic, but resistance can be gradually increased through
spotaneous mutations.  Medium was incorporated with 3-10 mM vanadate to 
find resistant strains.  These mutants show increased sensitivity to hygromycin
B aminoglycoside.  GS115 was plated on EMS with different vanadate concentrations.
Growth inhibition was clearly observed, although GS115 could grow in presence
of 20 mM vanadate.  It is believed (from studies with <i>H polymorpha</i>) that
the yeast detoxifies vanadate by showing increased cell vacuolization, cell
wall thickening and high amounts of polyphosphates:  the polyphosphates are
placed in the vacuole and the vanadate is sequested by the polyphosphates
placed there.
<p>
The enzyme neuraminidase (NA) is a finishing enzyme and is itself a glycoprotein.
It is hyperglycosylated as a high-mannose type N-glycan with 30-40 mannose
residues.  The N-glycans are then phosphorylated and cannot be trimmed with
&alpha;-mannosidase with or without alkaline phosphatase present; this may
indicate the phosphodiesters are present.  A <i>P pastoris</i> strain producing
NA was transformed with a &alpha;-1,2-mannosidase and this reveals N-glycans
with smaller than Man<sub>8</sub>GlcNAc<sub>2</sub>, usually not found in wild-type
yeast.  With methanol induction, &alpha;-1,3-mannosidase was present in the
culture medium at ~5 mg/L  (the membrane anchor domain may not be functional).
On westerns NA is a smear of bands from 60-85 kDa.  N-linked CHO analysis
shows small oligosaccharides (&lt;Man<sub>14</sub>GlcNAc<sub>2</sub>) were 
trimmed according to PAGE analysis of ANTS-labeld oligosaccharides.  Large
glycans (&gt; Man<sub>14</sub>GlcNAc<sub>2</sub>) showed partial digestion
with &alpha;-1,2-mannosidase co-expression.  Monosaccharide analysis (by
hydrolyzing with TFA to release constitutents) by HPAE-PAD show that the
N-glycan is made of Man and GlcNAc as expected but small amounts of Glc and
phosphorylated monosaccharides are present (Glc-P and Man-P).  
Man<sub>8</sub>GlcNAc<sub>2</sub> cores have two potential phosphorylation
sites, and Man-P and unprocessed Glc residues can inhibit the mannosidase.
<p>
An &alpha;-1,2-mannosidase gene which is a chimera of <i>S cerevisiae</i>
and <i>T. reesei</i> was made so that it targets glycoproteins late in the
ER (somewhat early in secretory protein pathway).  The catalytic domain from
<i>T reesei</i> was used while the membrane anchor from <i>S cerevisiae</i>
MNS1 (an ER-specific type II membrane protein, Camirand et al, 1991) was used.
The mannosidase was not secreted with NA as for the first mannosidase coexpression.
The intracellularly retained mannosidase caused a loss of small glycans
on NA.  The presence of the mannosidase creates trimmed features that would 
also create hyperglycosylation, which is also seen, thus stimulating more
mannosyltransferase activity;  the mannosidase does not lead complete digestion.
If the mannosidase is secreted in co-expression however, it could cause in vitro
digestion after which no more hyperglycosylation intracellularly is possible.
<p>
<i>Pichia</i> strains with heterologous &alpha;-1,2-mannosidase expression 
may help to produce human therapeutic proteins that do not have altered or
reduced immunogenicity.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Wu SM, Patel DD, Pizzo SV
 </span>
 <span class="year">
(1998)
 </span>
 <span class="jname">
J. Immunol.
 </span>
 <span class="jvol">
161,
 </span>
 <span class="pages">
4356-4365.
 </span>
<p class="reftitl">
Oxidized &alpha;<sub>2</sub>Macroglobulin (&alpha;<sub>2</sub>M) Differentially
Regulates Receptor Binding by Cytokines/Growth Factors:  Implications for Tissue
Injury and Repair Mechanisms in Inflammation
<p class="Para1">
&alpha;<sub>2</sub>M is present in high (micromolar) concentrations in plasma. 
Proteinases are trapped covalently by unique &beta;-cysteinyl-&gamma;-glutamyl 
thioester bond in which &alpha;<sub>2</sub>M conformation is altered to a more 
compacted structure which exposes &alpha;<sub>2</sub>M domains at the surface 
which can bind low-density lipoprotein receptor-related protein (LRP1) and an 
&alpha;<sub>2</sub>M signalling receptor.  The conformational change can also be 
induced by chemical treatments (small primary amine nucleophiles like 
methylamine).  Methyalmine-complexed &alpha;<sub>2</sub>M is designated 
&alpha;<sub>2</sub>M*. &alpha;<sub>2</sub>M can also bind cytokines of many 
types:  binding abolishes function of some of the cytokines but enhances the 
function of others.  Cytokine binding affinities show Kd constants in micromolar 
range for TNF-&alpha;, an early inflammatory mediator, and in the nanomolar 
range for late inflammatory mediators (bFGF, &beta;-NGF, PDGF, and TGF-&beta;) 
There is a difficulty in understanding how &alpha;<sub>2</sub>M can scavenge all 
of the latter growth factors but their ability to effect their function is not 
inhibited. Oxidatin of &alpha;<sub>2</sub>M abolishes its ability to inhibit 
proteinases (as it should, since their function can be involved in fighting 
infection).  Hypochlorite (not peroxide or hydroxyl radical) can abolish 
proteinase inhibition ability of &alpha;<sub>2</sub>M.  These reactions target 
Met and Trp residues, but also Lys (recently shown).
<p>
&alpha;<sub>2</sub>M and &alpha;<sub>2</sub>M* were treated with increasing 
concentrations of hypochlorite (NaOCl, 15 min @ 37&deg; in pH 7.4 PBS, at 0, 5, 
10, 50, and 100 &micro;M; [NaOCl] determined by spectrophotometer @ 292.5 nm) 
and oxidation was quenched with 0.2 mM L-Met addition.  5 &micro;g protein then 
loaded on to native pore-limit gel and run 3 h, stained with Coomassie BB. 
Tetramer of native &alpha;<sub>2</sub>M is present until 50 &micro;M, which then 
forms an electrophoretically faster form, the dimer. For &alpha;<sub>2</sub>M*, 
the tetramer is present at all concentrations, indicating that the disruption of 
the thioester makes &alpha;<sub>2</sub>M oxidation-resistant.  Coomassie 
staining intensity was reduced with the 100 &micro;M treatment for both 
&alpha;<sub>2</sub>M and &alpha;<sub>2</sub>M*, an effect previously reported. 
Western blotting and autoradiography demonstrated equivalent protein quantities 
in all lanes, so the reduced staining is because of an altered ability for 
Coomassie dye to bind to protein, not because protein has been decomposed.
<p>
TNF-&alpha;, IL-2, IL-6, bFGF, &beta;-NGF, PDGF-&beta;&beta;, TGF-&beta;1 and 
TGF-&beta;2 were obtained as purified commercial recombinants (R&amp;D Systems, 
Genzyme Diagnostics) and labeled with [<sup>125</sup>I]Bolton-Hunter or 
purchased labeled from NEN Life Sci Prod.  Labeled ligands are incubated 2 h in 
20 &micro;l of 250 &micro;g/ml &alpha;<sub>2</sub>M previously oxidized and run 
on native gel; gels are then stained and dried and exposed to X-ray film (gels 
run in triplicate).  Gels show that TNF-&alpha;, IL-2, IL-6, and bFGF have 
increased affinity for oxidized &alpha;<sub>2</sub>M.  Affinity is not a 
function of the multimeric state of &alpha;<sub>2</sub>M:  IL-2 and IL-6 have 
increased binding while it is still a tetramer.  Binding of &beta;-NGF, 
TGF-&beta;1, TGF-&beta;2 and to lesser extend PDGF-&beta;&beta; is reduced with 
increased oxidation of &alpha;<sub>2</sub>M.  It was more dramatic for 
TGF-&beta;2 than TGF-&beta;1, but TGF-&beta;2 is known to have a higher affinity 
for &alpha;<sub>2</sub>M.  Between-comparisons for cytokines were not possible 
since specific radioactivity was different for the labeled proteins.  As Met is 
added at high concentrations to quench oxidation, its presence (and that of Met 
sulfoxide) were assessed to see if they affect cytokine-&alpha;<sub>2</sub>M 
binding:  in untreated (unoxidized) &alpha;<sub>2</sub>M, labeled TGF&beta;1, 
PDGF-&beta;&beta; and TNF-&alpha; were incubatedd with presence or absence of 
Met or Met/NaOCl, and there was no difference in amount of cytokine bound.
<p>
Methylamine-treated &alpha;<sub>2</sub>M (&alpha;<sub>2</sub>M*) is supposed to 
represent &alpha;<sub>2</sub>M in the form when it is bound to proteinase, and 
may be the significant form of &alpha;<sub>2</sub>M present in inflammatory 
tissues.  The same oxidation-then-binding and gel analysis process was done as 
for unmodified oxidized &alpha;<sub>2</sub>M and the results show that all 
labeled ligands showed a decrease in affnity or otherwise no effect.
<p>
To determine alteration of binding constants, &alpha;<sub>2</sub>M and 
&alpha;<sub>2</sub>M* and its oxidized forms (what concentration?) were 
incubated 2 h @ 37&deg; with the radiolabeled forms of &beta;NGF, 
PDGF&beta;&beta;, TGF&beta;1 and TNF&alpha;, since these forms showed 
significant binding changes.  In each case the concentration of all 
&alpha;<sub>2</sub>M forms was varied from 0 to 4000 nM for TNF&alpha;, 0 to 
3000 for &beta;-NGF, and 0 to 2000 nM for the other two during cytokine binding. 
Gels were then run and radioactivity quantified by image analysis.  Nonlinear 
fitting was done on the points (mean of three replicates).  The Kd values are 
tabulated and it is seen that TNF&alpha; affnity increases by 5.3-fold with 
oxidation.  TGF-&beta;1 and &beta;-NGF decreased 3.1- and 5.9-fold respectively. 
With &alpha;<sub>2</sub>M* there is an 8.5- and 13.1-fold affinity 
<em>decrease</em> for PDGF-&beta;&beta; and TGF-&beta;1, respectively.  These 
findings agree with previous studies using cross-linking/electrophoresis assays 
in which it was reported that 10-20% dissociation of ligand can occur during the 
2 h gel electrophoresis when cross-linking is not done.  We tried using the 
chemical crosslinkers but saw a significant decrease in efficiency of xlinking 
to oxidized protein compared to unoxidized, probably because Lys residues are 
necessary for crosslinking with bis-(sulfosuccinimidyl)suberate, and these are 
affected by oxidation.
<p>
Several reports indicate that cytokines/growth factors show decreased binding to 
cell surface receptors in presence of &alpha;<sub>2</sub>M or 
&alpha;<sub>2</sub>M*, which competes for binding. If oxidation affects binding, 
how does this affect competition with binding to cells?  Radiolabeled TGF&beta; 
has a high affinity for both &alpha;<sub>2</sub>M and &alpha;<sub>2</sub>M* and 
is chosen as a model.  CCl64 mink lung epithelial cells were seeded to 24-well 
plates (500,000 cells/well) till confluent and 0.5 ng radiolabeled TGF&beta;1 
added in presence or absence of different concentrations of &alpha;<sub>2</sub>M 
or oxidized &alpha;<sub>2</sub>M and incubated 16 h at 4&deg;. 
Receptor-associated protein (RAP) was added to 6.65 &micro;M to inhibit all LRP 
ligands which determines amount of &alpha;<sub>2</sub>M/TGF&beta;1 complex that 
binds via LRP.  Unbound ligand is washed away with icecold HBSS and then cells 
solubilized with 0.5 M NaOH/0.1% SDS and cound in gamma counter. Total binding 
of [<sup>125</sup>I]TGF&beta;1 is calculated by that bound in absence of 
&alpha;<sub>2</sub>M.  Nonspecific binding is determined by using 1000-fold 
molar excess of unlabeled TGF&beta;1 and is 20-30% of the total binding value. 
Plot of bound labeled TGF to cell receptors (as percent of total binding) 
against &alpha;<sub>2</sub>M concentration show expected decreases as 
concentration of &alpha;<sub>2</sub>M increases.  More TGF was able to bind in 
the presence of &alpha;<sub>2</sub>M than 100 &micro;M NaOCl-treated 
&alpha;<sub>2</sub>M, indicating that the oxidized form had a reduced binding 
affinity to TGF&beta;1 With &alpha;<sub>2</sub>M* there was a greater binding 
affinity to TGF&beta;1 both for unoxidized and oxidized.  And again the oxidized 
form had a reduced binding affinity.  &alpha;<sub>2</sub>M had an 
IC<sub>50</sub> of 130 nM, reduced 4.1-fold with oxidized &alpha;<sub>2</sub>M 
with IC<sub>50</sub> = 530 nM.  For &alpha;<sub>2</sub>M* in both forms, the 
IC<sub>50</sub> ranged from 10 to 90 nM.
<p>
&alpha;<sub>2</sub>M does not bind LRP whereas oxidation does have affinity for 
LRP.  So why does TGF&beta; show up binding more to cells in the presence of 
oxidized &alpha;<sub>2</sub>M? Is it because of an absence of affinity for 
oxidized &alpha;<sub>2</sub>M, or that it binds to &alpha;<sub>2</sub>M and its 
own affinity for cells is enhanced?  CCL64 cells were incubated with oxidized 
&alpha;<sub>2</sub>M and radiolabeled TGF&beta;1 in presence or absence of 
50-fold molar excess of RAP, which inhibits LRP binding. Data show that RAP does 
not significantly alter cell surface binding in presence of oxidized 
&alpha;<sub>2</sub>M, and RAP alone has no effect on TGF&beta;1 binding.
<p>
To see if &alpha;<sub>2</sub>M forms (&alpha;<sub>2</sub>M* and oxidized 
&alpha;<sub>2</sub>M) affected cytokine or growth factor function, the following 
was done:  WEH1 13VAR cells were incubated in various concentration of 
TNF&alpha; in presence or absence of 1 mg/ml &alpha;<sub>2</sub>M or 
&alpha;<sub>2</sub>M* or in 1 mg/ml 100&micro;M NaOCl-treated 
&alpha;<sub>2</sub>M or &alpha;<sub>2</sub>M* and viability then measured using 
Celltiter 96.  For TGF&beta;, CCL64 cells were used and cell proliferation 
assessed using tritated thymidine assay.  For bFGF, fetal bovine heart 
endothelial cells were used.  In oxidized &alpha;<sub>2</sub>M, TNF-&alpha; 
bioactivity was inhibited by 66% compared to nonoxidized &alpha;<sub>2</sub>M, 
which had no effect.  In &alpha;<sub>2</sub>M*, there was no significant 
difference in inhibition between oxidized and nonoxidized protein, consistent 
with in vitro binding data.  TGF&beta; activity was significantly decreased in 
&alpha;<sub>2</sub>M (62%) and &alpha;<sub>2</sub>M* (82%).  The inhibition was 
abolished in presence of oxidized proteins. For bFGF, there is no difference in 
inhibtion between &alpha;<sub>2</sub>M and oxidized &alpha;<sub>2</sub>M 
although there is an apparent increase in oxidized &alpha;<sub>2</sub>M binding 
to bFGF.  There is a significant decrease in inhibition of bFGF bioactivity with 
oxidized compared to unoxidized protein.
<p>
If oxidation regulates &alpha;<sub>2</sub>M binding to cytokines and growth 
factors in vitro, does oxidized &alpha;<sub>2</sub>M present in patients with 
rheumatoid arthritis and other inflammatory disease have the same binding 
behavior?  &alpha;<sub>2</sub>M was purified from six patients with RA and five 
with osteoarthritis using anti-human &alpha;<sub>2</sub>M polyclonal 
immunoaffinity (purity verified by western).  &alpha;<sub>2</sub>M was also 
isolated from six healthy volunteers as controls.  Protein carbonyl content was 
determined using 2.4-dinitrophenylhydrazine. Four samples were assessed: 
&alpha;<sub>2</sub>M from healthy plasma, &alpha;<sub>2</sub>M from that source 
oxidized with 100 &micro;M hypochlorite, and the &alpha;<sub>2</sub>M samples 
from RASF and OASF. RASF &alpha;<sub>2</sub>M is 7-fold higher compared to 
healthy donors, and this level is comparable to &alpha;<sub>2</sub>M that has 
been hypochlorite-treated.  The four &alpha;<sub>2</sub>M samples were then 
incubated with radiolabeled TGF&beta; under standard conditions with gel 
electrophoresis and autoradiography and densitometry, and the 26% reduced 
binding of TGF to RASF &alpha;<sub>2</sub>M was significantly different from 
healthy &alpha;<sub>2</sub>M.  RASF &alpha;<sub>2</sub>M had a 55% decrease in 
TGF&beta; binding compared to &alpha;<sub>2</sub>M*.  Was the inhibition of 
binding already because RASF &alpha;<sub>2</sub>M is bound to endogenous 
TGF&beta;?  RASF &alpha;<sub>2</sub>M was acidified, which has been shown to 
release bound TGF&beta;, and TGF&beta; activity tested:  no activity was 
detected.
<!-- ============================================================= -->
<p class="tags">
serum protein electrophoresis, immunofixation electrophoresis, SPE, IFE
<p class="ref">
 <span class="authors">
Bossuyt X, Bogaerts A, Schiettekatte G, Blanckaert N
 </span>
 <span class="year">
(1998)
 </span>
 <span class="jname">
Clin. Chem.
 </span>
 <span class="jvol">
44,
 </span>
 <span class="pages">
944-949.
 </span>
<p class="reftitl">
Serum protein electrophoresis and immunofixation by a semiautomated
electrophoresis system
<p class="Para1">
SPE is used for detecting and identifying paraproteinemias.  Agarose gels and
cellulose acetate membranes are the medium of separation.  Linear densitometry
is used to quantify.  Immunoelectrophoresis or immunofixation is used to classify
the paraproteins.  IEP is quite lengthy and even less sensitive than IF.
The Helena REP is an automated system for protein electrophoresis.
Hydrasys-Hyrys and Sebia have produced semi-automated system using agarose
media.  The automation is in sample application, migration, incubation, staining,
destaining, and drying.  This article compares the Hydrasys-Hyrys and conventional
manual system.
<p>
The Hydrasys can run 30 samples every 20 min, and total processing was 40 min.
With the cellulose acetate system, only 16 samples (2 membranes &times; 8 samples
per membrane) could be run.  Resolution and band clarity were far better in the
Hydrasys Hydragel than in the Microzone cellulose acetate.  In one sample an
oligoclonal pattern could not be seen in the cellulose acetate but could be
seen with agarose.  Four of the samples were assessed using immunofixation
electrophoresis (IFE).  IFE is don by applying a sample at six different lanes
of an agarose gel, separating them by electrophoresis (20 W at 20&deg; for 9 min)
then applying either a fixative or antisera specific to IgG, IgA, IgM,
and &kappa; or &lambda; light chains for 5 min for immunoprecipitation. These
are then stained 4 min with violet blue, destained for 3, 2, and 6 min and
dried at 75&deg; for 8 min.  The manual IFE method (Beckman Paragon system)
clearly revealed the same paraprotein.  Reference intervals (median and
95% range for the system (see table).  All distributions are normal, and there
are no statistically significant differences related to sex.
<table class="td-center">
<col><col span="4" style="width:5em;">
<tr><th rowspan="2">Fraction  <th colspan="2">Median  <th colspan="2">95% Range
<tr>                                 <th>g/L  <th>%    <th>g/L       <th>%
<tr><td>albumin                      <td>48   <td>67.4 <td>41.9-53.5 <td>60.8-72.2
<tr><td>&alpha;<sub>1</sub>-globulin <td>1.5  <td>2.2  <td>1.0-2.1   <td>1.4-3.1
<tr><td>&alpha;<sub>2</sub>-globulin <td>6.7  <td>9.5  <td>5.1-8.1   <td>7.1-11.8
<tr><td>&beta;-globulin              <td>6.9  <td>9.6  <td>4.7-9.2   <td>6.9-12.5
<tr><td>&gamma;-globulin             <td>7.8  <td>11.2 <td>4.7-12.2  <td>7.1-15.9
</table>
<p>
A precision test using 20 aliquots of a single serum test were done as proposed
by NCCLS EP5-T2.  CV values were as follows: albumin 0.90%,
&alpha;<sub>1</sub>-globulin 4.61%, &alpha;<sub>2</sub>-globulin 1.95%,
&beta;-globulin 1.86%, &gamma;-globulin 2.4%.  Precision values both within
run and between runs was tested and values were much better than for cellulose
acetate.  These values are likely related to the automation of the system.
<p>
50 samples were evaluated on the Hydrasys-Hyrys and cellulose acetate to do a
method comparison and linear regression performed on each of the five fractions.
Correlation coeffcients were generally acceptable (most above 0.95).  Several
cases of paraproteins were investigated to compare the automated with IEP and
manual IFE.  No clinically significant differences were observed between manual
and automated.  In one case the Hydrasys system showed an IgG&lambda; monoclonal
component and an IgA component, while the manual showed only the IgA.
<p>
IEP detects free light chains more easily than IFE because IEP has a diffusion
step that guarantees Ab-Ag equivalence.  With IFE controlling the Ag concentration
is important to maximize sensitivity.  In some cases, the Ag might be overdiluted
and so a sample might need to be run undiluted.
<p>
Bence Jones proteins in the urine were assessed in 16 samples (10 &kappa; and
6 &lambda;)  An IgG monoclonal was seen in 4 of 16 cases.  The automated system
can use unconcentrated urine, whereas the manual requires urine concentration
first.
<!-- ============================================================= -->
<p class="tags">
genome sequencing, HBV, multiple genome comparison
<p class="ref">
 <span class="authors">
Takahashi K, Akahane Y, Hino K,Ohta Y, Mishiro S
 </span>
 <span class="year">
(1998)
 </span>
 <span class="jname">
Arch. Virol.
 </span>
 <span class="jvol">
143,
 </span>
 <span class="pages">
2313&ndash;2326.
 </span>
<p class="reftitl"> 
Hepatitis B virus genomic sequence in the circulation of hepatocellular 
carcinoma patients: Comparative analysis of 40 full-length isolates
<p class="Para1">
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Rossman MG, Tao Y
 </span>
 <span class="year">
(1999)
 </span>
 <span class="jname">
Structure
 </span>
 <span class="jvol">
7,
 </span>
 <span class="pages">
R43-R46.
 </span>
<p class="reftitl">
Courageous Science: Structural Studies of Bluetongue Virus Core
<p class="Para1">
D. Stuart and coworkers recently reported the 3-D core structure of
bluetongue virus.  The ~700 &Aring; diameter particle has a total MW of about
54 MDa and is the largest atomic structure to be determined crystallographically
to date.
<p>
Bluetongue virus is an orbivirus belonging to Reoviridae family of segmented
ds RNA icosahedral viruses with 2-3 layers of proteins forming capsid.
It causes disease in sheep and cattle and is endemic in the Middle East,
India, China, and western areas of the U.S.  The outer shell of orbiviruses
has proteins to recognize and interact with the target cell, and these
are stripped during cell entry, leaving a core with an outer layer
of 780 copies of VP7 protein (38 kDa) arranged in a 13 &times; 60 pattern
and an inner layer of 120 copies of VP3 protein (100 kDa) arranged in a
2 &times; 60 pattern.
<p>
VP7 has been structurally evaluated by itself.  It arranges into trimers
in a number of different crystal forms.  The monomer is 85 &Aring; long
and the length runs about parallel to the trimer axis.  The monomer
has two domains:  a helical domain formed by N- and C-terminal regions,
and &ldquo;jelly-roll&rdquo; domain formed by central one-third of polypeptide.
The latter domain forms the external surface of the core.  These jelly-roll
domains are a feature of other viral capsids.  EM reconstructions have
established the overall organization of proteins in the bluetongue virus.
<p>
A homologous model was proposed and then electron density averaging was
used to verify the model.  This was first done successfully with human
rhinovirus and poliovirus.  The ability to resolve unit cells with dimensions
so large (800 &Aring; or more) has been made possible by improvements in
the use of synchrotron radiation sources, better computational facilities
and software to coordinate millions of X-ray reflections.
<p>
The model indicated that a closed capsid must consist of many identical
protein subunits, and that being identical, each subunit must bind other
subunits in identical ways.  The environment must also then be identical
and only the only geometric arrangements allowing this are Platonic solids
including the tetrahedron, octahedron, and icosahedron.  The icosahedron
requires the greatest number of asymmetric objects (60) to form a complete
regular figure.
<p>
The inner subcore of bluetongue is composed of 120 subunits, a number at
variance with the Caspar and Klug triangulation rules for capsid face
construction.  The two molecules per icosahedral repeating unit adapt their
conformation to respective environments by structural changes especially
in the apical domans, close to the fivefold axes.  Nonetheless there are
exceptions to the Caspar-Klug rules where for instance a tail segment
with 12-fold symmetry is inserted into 5-fold symmetric vertex.
<p>
The notion of <b>conformational switching</b> is a concept in which
identical proteins bind to one another and in the process induce a
conformational change, and then when a 3rd subunit binds, it cannot bind
in the same way because it is presented with a different conformation.
The conformational changes induced might be different for this 3rd subunit.
Those changes induced in apical domains of the bluetongue VP3 can be
understood in terms of this switching.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Wu SM, Pizzo SV
 </span>
 <span class="year">
(1999)
 </span>
 <span class="jname">
Biochemistry
 </span>
 <span class="jvol">
38,
 </span>
 <span class="pages">
13983-13990.
 </span>
<p class="reftitl">
Mechanism of Hypochlorite-Mediated Inactivation of Proteinase Inhibition by
&alpha;<sub>2</sub>-Macroglobulin
<p class="Para1">
&alpha;<sub>2</sub>-Macroglobulin (&alpha;<sub>2</sub>M) is 720 kDa glycoprotein 
homotetramer present in high concentrations in plasma (2-4 mg/ml) which is 
really a dimer of dimers (the dimers are connected by interchain disulfides). 
It has a broad protease inhibition mechanism (serinyl, metallo, cysteinyl, 
aspartyl type) which traps proteases while allowing them to hydrolyze small mass 
substrates able to diffuse into the trap.  Proteases hydrolyze a <q>bait 
domain</q> in the protein which leads to rupture of a &beta;-
cysteinyl&mdash;&gamma;-glutamyl thioester bond with covalent bonding of 
&alpha;<sub>2</sub>M to the protease.  This leads to a more compact form of 
&alpha;<sub>2</sub>M that exposes an LRP1 binding site as well as cytokine-
binding sites. In disease and inflammation, while there are high levels of 
&alpha;<sub>2</sub>M in the extracellular fluid, there is still much proteolytic 
activity.  Oxidizing substances increase substantially in such milieus, reaching 
millimolar levels.  Several proteinase inhibitors in plasma other than 
&alpha;<sub>2</sub>M may be oxidized with loss of function; 
&alpha;<sub>2</sub>M was recently found to be oxidized in synovial fluid from 
knees obtained from rheumatoid arthritis patients. Oxidized &alpha;<sub>2</sub>M 
shows a better binding ability to <i>pro</i>inflammatory cytokines (TNF-&alpha;, 
IL-2, IL-6) and less able to bind tissue repair-promoting factors (PDGF, TGF-
&beta;, NGF).
<p>
<b>Results</b>.  Many amino acids exposed to the solvent surface are subject
to oxidation (Cys, Trp, Lys, Met, Tyr).  Oxidation of amine groups (N-terminal,
Lys) results in carbonyl group formation.  Oxidants were prepared at different
concentration from 1 &micro;M to 10 mM, with concentration determined by
iodometry (<a 
href="http://www.ncbi.nlm.nih.gov/sites/entrez?Db=pubmed&amp;Cmd=DetailsSearch&amp;Term=libbyU+005BAll+FieldsU+005D+AND+kaiserU+005BAll+FieldsU+005D+AND+1982U+005BAll+FieldsU+005D">Libby
et al. (1982) <i>J. Biol. Chem.</i> <b>257</b>: 5030</a>) and carbonyl groups
titrated using 2.4-dinitrophenylhydrazine (DNPH) with method of
Yan et al. (<i>Arch. Biochem. Biophys.</i> <b>327</b>: 300, 1996)
modified as so: 100 &micro;g &alpha;<sub>2</sub>M in 800 &micro;l PBS mixed with 200 &micro;l
10 mM DNPH in 2.0 M HCl for 1 h 25&deg;, then pptd with 150 &micro;l 70% TCA
 on ice for 10 min, microfuged 5 min, pellets washed in EtOH:EtOAc (1:1),
twice, then solubilized in 6 M guandidine HCl pH 7.4.  Absorbances are measured
at 276 and 370 nm, with nonspecific absorbance being tested in protein with no
DNPH present and used to subract from total. Hydroxyl radical (&bull;OH) and
hypchlorite (OCl<sup>&minus;</sup>) showed obvious concentration-dependent
oxidation (1 carbonyl per &alpha;<sub>2</sub>M subunit at 1 mM oxidant) whereas peroxide
(H<sub>2</sub>O<sub>2</sub>) had no effect at all.  Carbonyls were formed at
&lt;100 &micro;M,; EC<sub>50</sub> of hydroxyl radical and hypochlorite were
200 and 300 &micro;M, respectively.  Concentrations of this sort are easily
achieved physiologically in the inflammatory fluid.  Hypochlorite has been
<a href="http://www.ncbi.nlm.nih.gov/sites/entrez?Db=pubmed&amp;Cmd=DetailsSearch&amp;Term=ReddyU+005BAll+FieldsU+005D+AND+DesorchersU+005BAll+FieldsU+005D">shown</a>
to cause fracturing of &alpha;<sub>2</sub>M on the dimeric axis, disrupting structure.  Protein
that was treated with &ge; 100 &micro;M hypochlorite was run on 5-15% gradient
native PAGE, and the oxidized form migrated faster than the untreated.
This oxidized form also moved identically to &alpha;<sub>2</sub>M on SDS-PAGE; a conformationally
altered form (not oxidized) only migrates marginally faster than the wholly
native tetrameric form.  Hydroxyl radical, but at a significanly higher
concentration, generated &alpha;<sub>2</sub>M dimers.  Hydrogen peroxide had no effect.
With 1 mM hypochlorite, there is a noticeable reduction in Coomassie-stained
intensity.
<p>
The proteinase inhibition activity of oxidized &alpha;<sub>2</sub>M was assessed.  250 &micro;g/ml
(oxidized) &alpha;<sub>2</sub>M is added to 20 &micro;g/ml thermolysis (1:1) for 30 min 37&deg;.
25 &micro;l added in duplicate to wells of 96-well plate containing 100 &micro;l
10 mM Tris pH 8.0/2 mM CaCl<sub>2</sub>, with wells in just buffer or containing
2 mg/ml succinylated casein and left 1 h 37&deg;.  50 &micro; trinitrobenzene
sulfonate (TNBS) is added for 10 min and plates read at 450 nm.  Plot of
percent inhibitory activity (relative to untreated) vs. log oxidant concentration
showed that only hypochlorite and not hydroxyl radical had a concentration-dependent
effect on inhibiting &alpha;<sub>2</sub>M, which was different from the expected effect given
that hydroxyl radical and hypochlorite modified the structure by increasing
protein carbonyl.  The EC<sub>50</sub> of hypochlorite inactivation was
30 &micro;M, about one-tenth of the concentration necessary to produce carbonyl
formation.
<p>
Clearly protein carbonyl formation is not the means by which &alpha;<sub>2</sub>M activity
is inhibited.  It is understood that &alpha;<sub>2</sub>M has a bait region and that breaking
of the thioester bond leads to premature trapping and inhibition.  Hypochlorite
does not affect the bait region or thioester bond (based on other reports).
But since oxidation leads to an electrophoretically faster form, this structural
change from tetramer to dimer must affect the trapping mechanism.  The SDS-PAGE
in reducing conditions are the key:  hypochlorite and not peroxide or hydroxyl
radical can induce intersubunit cross-linking so that only dimers and not
monomers are observed.
<p>
&alpha;<sub>2</sub>M was treated with the chemical crosslinker bis(sulfosuccinimidyl)suberate
(BS<sup>3</sup>, Pierce) from 1 &micro;M to 10 mM in log-fold increases.
Native 5-15% PAGE (3 h, 150 V) shows an electrophoretically faster dimer (D)
form appearing at 100 &micro;M and very strong D bands at 1 and 10 mM BS<sub>3</sub>.
The tetrameric (T) form is present in all concentrations although a reduction
is seen in the Coomassie-stained band.  Plots of percent proteinase inhibitory
activity vs. BS3 concentration show that BS<sup>3</sup> only slightly reduces
(maximally 20%) &alpha;<sub>2</sub>M function at the highest concentration (10 mM,
statistically significant).  Zymaography (&alpha;<sub>2</sub>M treated with BS3 is mixed with
trypsin 2:1 proteinase:&alpha;<sub>2</sub>M and run on gradient PAGE) with gels incorporated
with gelatin shows clearing of the gel in the zones where the T and D forms
would exist, which indicates crosslinked &alpha;<sub>2</sub>M is still able to trap proteinases.
<p>
Since subunit crosslinking and tetramer-to-dimer fragmentation did not affect
reduce proteinase inhibition.  In earlier work, we found that hypochlorite increases
a proteins susceptibility to proteolytic degradation.  A 5:1 trypsin:oxidized &alpha;<sub>2</sub>M
preparation was incubated; this was done to make sure that trypsin itself was not
inactivated by &alpha;<sub>2</sub>M.  [<sup>125</sup>I]&alpha;<sub>2</sub>M was oxidized, trypsin-incubated, then run
5-15% native PAGE and exposed to phosphorimaging.  With no trypsin, D form &alpha;<sub>2</sub>M is
clearly present at high HOCl concentrations.  But with trypsin present, the D form
is completely undetected.  T form is always present but decreases with increases
on hypochlorite.  To determine the number of cleaved bonds, reaction with TNBS was
done and a plot of cleavage per &alpha;<sub>2</sub>M subunit vs. oxidant concentration was done.
Only with hypochlorite was a linear increase observed, and not with hydroxyl radical
or peroxide.  This result contrasts with the use of BS<sup>3</sup> crosslinker in which &alpha;<sub>2</sub>M
can still bind trypsin and not be degraded.
<p>
Hypochlorite-oxidized &alpha;<sub>2</sub>M was incubated with 2:1 trypsin:&alpha;<sub>2</sub>M for 20 min 37&deg;
and then run on SDS-PAGE which had been polymerized with 0.1% gelatin to do a zymogram.
The zymogram shows no clearing in the &alpha;<sub>2</sub>M-only lane, shows clearing at the bottom of
the trypsin-only lane, where the enzyme would be expected to migrate.  In the
&alpha;<sub>2</sub>M-trypsin mixture lanes with increasing HOCl concentrations, more bands of clearing
are observed as oxidation increases, indicating &alpha;<sub>2</sub>M's loss of proteinase inhibiting
activity.  Thus the combination of oxidation and exposure to trypsin accounts for a
loss of function.
<p>
If oxidation of &alpha;<sub>2</sub>M leads to its proteolysis, does this occur biologically?  &alpha;<sub>2</sub>M
was purified from synovial fluid of rheumatiod arthritis and osteoarthritis patients
and compared to &alpha;<sub>2</sub>M taken from plasma of healthy patients.  Western blots were prepared
from four different patients in each class.  The blots show a  band at 180 kDa (monomer)
in all 12 samples.  Two of the four RA patient samples did show lower MW bands
indicating &alpha;<sub>2</sub>M degradation.  Moreover protein carbonyl assay of the &alpha;<sub>2</sub>M present
shows a higher content of carbonyls present in &alpha;<sub>2</sub>M of rheumatoid arthritis patients
than in plasma of healthy controls.  Even the osteoarthritis patients showed elevated
carbonyl content, although not significantly.
<!-- ============================================================= -->
<p class="tags">
RNAi, dsRNA
<p class="ref">
 <span class="authors">
Sharp PA
 </span>
 <span class="year">
(1999)
 </span>
 <span class="jname">
Genes Dev.
 </span>
 <span class="jvol">
13,
 </span>
 <span class="pages">
139-141.
 </span>
<p class="reftitl">
RNAi and Double-Strand RNA
<p class="Para1">
<b>REVIEW</b>.  dsRNA has been shown to be signal for gene-specific silencing
of expression in many organisms.  Injection of dsRNA into <i>C. elegans</i> 
resulted in inhibition of gene expression in somatic cells of the injected
worm and its F<sub>1</sub> progeny.  Plants use RNAi to suppress gene 
expression (cosuppression).  Effects of dsRNA have been shown in parasite
microorganisms and in <i>Drosophila</i>.  RNAi may be important for development
as well as an antiviral mechanism.  Since RNAi does not appear to cause 
germ line modifications (F<sub>2</sub> phenotypes revert to normal in the
nematode), it is likely that effect is post-transcriptional.  RNAi activity
is seen when segments of mRNA in ds form are observed, but not with intron
sequences in ds RNA form.  In <i>C elegans</i> expression multigene operons
are transcribed as a long precursor and each single-gene mRNA produced by
trans-splicing and cleavage.  Genes <i>lin-15a</i> and <i>lin-15b</i> are
part of one operon and should be inactivated together to generate the 
multivulva phenotype:  injection of dsRNA of both genes creates the phenotype,
but of either gene alone the phenotype is not observed.  In an in situ
hybridization experiment
<!-- ============================================================= -->
<p class="tags">
VEGF, VEGF-R
<p class="ref" id="Neufeld-VEGF">
 <span class="authors">
Neufeld G, Cohen T, Gengrinovitch S, Poltorak Z
 </span>
 <span class="year">
(1999)
 </span>
 <span class="jname">
FASEB
 </span>
 <span class="jvol">
13,
 </span>
 <span class="pages">
9-22.
 </span>
<p class="reftitl">
Vascular endothelial growth factor (VEGF) and its receptors
<p class="Para1">
<b>REVIEW</b>.
Two different factors, one a heparin-binding angiogenic factor and another 
vascular permeability factor (VPF), have been identified as being coded by the
VEGF gene.  Specific receptors have been found being of two types:
VEGFR-1 (Flt-1) and VEGFR-2 (KDR/Flk-1) which are VEGF-specific tyrosine 
kinases receptors.  The receptor structure includes 7 Ig-like domains in the
extracellular region with a single membrane spanning.  VEGF activity can be
inhibited using specific neutralizing antibodies or by using dominant
negative VEGF receptors, iether of which arrest tumorigenesis.  VEGF
expression is regulated by hypoxia and also small molecule effectors
such as phorbol esters and any effectors activating Ap-1 and -2 and Sp1
transcription factors.  Other growth factors can induce expression of VEGF,
and the effects of angiogenesis attributed to them are probably mediated
by VEGF.
<table style="width:100%;">
<tr><th>Receptor/Sequester      <th>Receptor Features    
  <th style="width:6em;">Ligands
<tr><td>VEGFR-1 (Flt-1) 
  <td>Ig-like Loops 1-7<br>VEGF-binding: Loops 2 and 3
   <br>dimerization domain
   <br>split Tyr kinase domain
  <td>VEGF<sub>121</sub><br>    VEGF<sub>165</sub><br>
		VEGF-B<br>     PlGF-1<br>   PlGF-2
<tr><td>VEGFR-2 (KDR/Flk-1) 
  <td>Ig-like Loops 1-7<br>VEGF-binding: Loops 2 and 3
   <br>mitogenic, permeability-inducing <br>split Tyr kinase domain
  <td>VEGF<sub>121</sub><br>    VEGF<sub>145</sub>
   <br>VEGF<sub>165</sub> <br>     VEGF-C<br>   VEGF-D<br> HIV-1/Tat
<tr><td>VEGFR-3 (Flt-4) 
  <td>Ig-like Loops 1-7  <br>split Tyr kinase domain
  <td>VEGF-B   <br>VEGF-D
<tr><td>Heparan Sulfate<br>Proteoglycan
  <td>
  <td>VEGF<sub>145</sub> <br>VEGF<sub>165</sub>
   <br>VEGF<sub>189</sub>  <br>PlGF-2  <br>VEGF-B<sub>167</sub> <br>HIV-1/Tat
<tr><td>Neuropilin-1
  <td>MAM domain <br>FV/VIII domains <br>C1r/s domains
  <td>Sema-III <br>Sema-E <br>Sema-IV <br>VEGF<sub>165</sub> <br>PlGF-2
<tr><td>Neuropilin-2
  <td>MAM domain <br>FV/VIII domains <br>C1r/s domains
  <td>Sema-E <br>Sema-IV <br>VEGF<sub>165</sub>
</table>
<p>
Additional VEGF-related growth factors and their receptors have followed the 
discovery of VEGF.  VEGF-B through -D have recently been reported along with a 
third receptor (VEGFR-3, see table above for a list of VEGF family growth 
factors and receptors).
<p>
Targeted gene disruption (knockouts) have established the vital role of VEGF
in development and in maintenance.  A VEGF +/- heterozygote is lethal. VEGFR-1
and -2 mutagenesis studies shows that severe abnormalities in blood vessel formation
occur.  VEGFR-2 is required for endothelial cell differentiation and for their
precursors to move from posterior primitive streak to yolk sac.  VEGFR-1 mutations
do not stop differentiation but vessel formation was quite impaired. VEGFR-1
activation leads to cell migration but not really cell proliferation efficiently.
<p>
A good understanding of how hypoxia drives angiogenesis is seen with retinal
pathologies.  During retintal development, astrocytes adn neuronal precursors
spread and migrate from blood vessels.  As astrocytes migrate from vessels they
experience increasing hypoxia; and they are more sensitive than the neuronal 
cells and thus are sensors.  Astrocytes then express VEGF which diffuses to
begin angiogenesis:  a chemical gradient establishes direction of vessel
formation.  The interesting thing is that a certain threshold level of VEGF
is required to maintain endothelial cells as part of the vasculature otherwise
they undergo apoptosis.  Decreasing levels of VEGF lead to vascular disassembly
and reassembly in other areas.  Vascular stabilization without VEGF occurs
when pericytes beging to form part of the vessel structure and are independent 
of VEGF.
<p>
Premature infants suffer from a retinopathy related to the immediate 
hyperoxygenation they are subjected to because of poorly developed lungs.  This 
leads to retinal astrocytes diminishing VEGF with newly blood vessels atrophying 
or even undergoing apoptosis.  When the neonates come out of the incubators, 
retinal cells experience hypoxia followed by hypersecretion of VEGF from 
astrocytes, which leads to a pathological angiogenesis resulting in blindness. 
Similar retinopathies related to hypoxia and uncontrolled angiogenesis occur as 
well.  This model has also been demonstrated with transgenic mice which 
overproduce VEGF in the retina.  Quail embryo studies in which high 
concentrations of VEGF are present show unregulated and excessive fusion of 
vessels with abnormally large lumens, while inhibition or reduction of activity 
leads to impaired angiogenesis.  A truncated soluble VEGFR-1 which abolished 
angiogenic signal transduction was found to inhibit the development of the 
corpus luteum in a rat model of hormonally induced ovulation.
<p>
Tumor cells are essentially de-differentiated and, with the exception of 
teratomas, show no attempt at regular structure (epithelial or endothelial 
layers).  Nonetheless, solid tumors require vascularization to move nutrients in 
and wastes out.  VEGF induces permeability as well, which are fenestrations in
blood vessels as endothelial cell joining is disrupted in order to initiate an
extravascular fibrin gel, a matrix to support endothelial cell penetration 
(normally) and tumor cell with formation of a stroma inside the tumor.
In tumor cells, it appears that the <i>ras</i>/MAP kinase pathways are responsible
for elevated VEGF transcription.  Inactivation of p53 or the presence of hormonal
growth factors drive the hypoxia-independent control.
<p>
<b>HIV/AIDS</b>. HIV makes a protein called HIV-1/Tat which appears to bind and
activate VEGFR-2 (KDR/Flk-1).  Kaposi sarcoma is observed in AIDS, and this shows
considerable angiogenesis.  This is probably the mechanism through which it works.
<p>
Having found that inhibition of VEGF or its signal transduction pathways lead
to inhibition of the progression of tumors, several therapeutic agents based
on control VEGF or its receptor interactions have been underway.  These include
humanized neutralizing anti-VEGF mAbs, use of soluble VEGF receptors that bind
to VEGF and intercept interaction with membrane-bound receptors that are activating,
antisense VEGF mRNA expressing constructs, VEGF-toxin conjugates, antagonistic
VEGF mutants, and small molecule VEGFR function inhibitors.
<p>
Rather than inhibiting it, VEGF is being administered as therapies for ischemic
tissue diseases (particularly of the heart).
<p>
Many cytokines, growth factors, and gonadotropins stimulate angiogenesis, but
not directly:  they do it by upregulating VEGF expression in specific cell types
and thus it is indirect.  The same for downregulators of angiogenesis (VEGF expression).
Potentiators of VEGF include FGF-4, PDGF, TNF-&alpha;, TGF-&beta;, keratinocyte
growth factor (KGF), IGF-I, interleukin 1&beta; (IL-1&beta;), and IL-6.  IL-10
and IL-13 are VEGF release inhibitors.  As an example, during dermal wound healing,
KGF is produced and secreted which induces VEGF in kerationcytes.  Hydrogen 
peroxide produced in wound by neutrophils also induces keratinocyte VEGF production,
although H<sub>2</sub>O<sub>2</sub> does directly inactivate VEGF.  UV-B irradiation
also stimulates VEGF expression, as does NOx, which has always been known to 
cause vascular permeability and dilatation.  VEGF also appears to upregulate
NOx, making it a positive feedback loop.
<p>
<b>Regulation by Hypoxia</b>.  Both hypoxia and hypoglycemia stimulate VEGF
expression, because these can be related to a poor vasculature that brings in
nutrients.  Hypoxia does not affect VEGF-related factors (VEGF-B, VEGF-C, PlGF)
despite their angiogenic properties.  Hypoxia-inducible factor 1 (HIF-1) is
a transcription factor with a binding site in the VEGF promoter.  HIF-1 is normally
induced by hypoxia, but HIF-1 expression can also be upregualted by v-<i>src</i>
and thus indirectly stimulate VEGF.  Besides transcription, HIF-1 stabilizes
VEGF mRNA through proteins (HuR mRNA-binding protein) that bind to 3&prime;-UTR;
other proteins also likely bind but are not identified.  Th 5&prime;-UTR also
contains active internal ribosomal entry site (IRES) and alternate initiation
codons from which translation actually starts, rather than from cap-dependent 
processes.
<p>
Some tumors have mutations in the von Hippel-Lindau (vHL) protein, which is a
tumor suppressor like p53.  There is an assocation with increased angiogenesis
and vHL inactivation.  vHL inhibits production of hypoxia-regulated GLUT-1
glucose transporter as well as VEGF.  vHL controls transcriptional and posttranscriptional
aspects of VEGF.  vHL inhibits protein kinase C&zeta; and C&delta;; when vHL
is absent these kinases stabilized VEGF mRNA because of several proteins normally
induced by hypoxia in the 500 b region of 3&prime;-UTR.  vHL also will complex with
Sp1 TF and thus inhibit Sp1-mediated VEGF expression.  PDGF also acts to induce
VEGF through Sp1.  The suppressor p53 may also act to inhibit VEGF (contradictory
reports).
<p>
VEGF is expressed in five different isoforms:  121, 145, 165, 189, and 206, 
which all result from different mRNA species produced by alternative splicing. 
The larger forms usually get extra exons coding in the C-terminal regions that 
give VEGF the ability to bind heparin and heparan sulfate.  VEGF-121 does not 
get coded with exons 6 and 7 and cannot bind heparin or ECM.  Inclusion of exon 
7 produces VEGF-165 and provides heparin binding ability.  VEGF-145 includes 
only exon 6 (not 7) include both a heparin- and ECM-binding ability. VEGF-189 
and -206 include exons 6 and 7 and provide higher affinities to heparin and ECM 
(heparan sulfate) than the 145 and 165 forms.  Proteases (plasmin) cleave the 
VEGF-189 to produced VEGF-110 which is an active soluble fragment.  The 
VEGF-121, -145, and -165 forms have been shown to be mitogenic and angiogenic in 
vivo.  Most cell types produce several variants simultaneously.  The 121 and 165 
forms predominate, but 189 was widely present.  The 145 form appears to be 
limited to reproductive organs.
<p>
Heparin-binding forms of VEGF bind to cell surface of ECM-associated heparan 
sulfate proteoglycans and release bFGF, also bound to heparan sulfate and 
angiogenic in syngergy with VEGF.  The presence of heparin has been shown to 
improve binding of <sup>125</sup>I-VEGF<sub>165</sub> to VEGFR-2, although 
<i>heparin-like</i> molecules do not do so.  Oxidized VEGF<sub>165</sub> and 
VEGF<sub>121</sub> cannot bind VEGFR-2.  Glypican, a cell surface heparan 
sulfate proteoglycan are chaperone-like and restore ability of oxidized 
VEGF<sub>165</sub> to bind to VEGFR-2; this is not the case for oxidized 
VEGF<sub>121</sub> however.  Damaged VEGF<sub>121</sub> function cannot be 
restored like for other VEGF forms.
<p>
Both VEGFR-1 and -2 are cloned tyrosine kinase receptors.  VEGFR-3 is expressed 
in lymph vessels and binds VEGF-C and -D.  They all have seven Ig-like loop 
domains extracellularly, a single transmembrane sequence, and a split Tyr kinase 
domain intracellularly.  While predominantly expressed in endothelial cells,
they are also found in other cell types with one or both.  VEGFR-1 is found
in trophoblasts, monocytes, renal mesangial cells.  VEGFR-2 is in hematopoietic
stem cells, megakaryocytes, retinal progenitors.  VEGFR-2 has two forms in 
retinal cells formed by alternative splicing.  Tumorigenic cells express both
VEGFR-2 or VEGR-1; malignant melanoma cells are a prominent example.  VEGFR-1
and -2 have different signaling effects.  Both receptors are glycosylated, and
VEGFR-2 only does autophosphorylation if it is glycosylated.  Expression of
receptors is affected by hypoxia, though less so than for VEGF. Hypoxia enhances
VEGFR-1 but not VEGFR-2.  Upregulation of VEGFR-2 in hypoxia may be more
posttranscriptional (transcript stability) rather than induction.
<p>
<b>Receptor-ligand binding</b>.  VEGF has two separate domains: one binding to 
VEGFR-1 and the other to VEGFR-2. Ala-scanning mutagenesis shows that R82, K84, 
and H86 basic amino acids are critical for VEGFR-2, and D63, E64, and E67 acidic 
amino acids are necessary for VEGFR-1.  In the 3D structure, they are at 
opposite ends.  The VEGF dimer has its monomers linked in rough 
<q>head-to-tail</q> fashion by disulfide bridges and the main VEGFR-2 binding 
domains are at opposite ends, and so are the VEGFR-1 binding domains.  It has 
been shown that mutations of VEGFR-1 binding amino acids do not significantly 
affect VEGF dimer structure will still be viable for VEGFR-2 binding.  VEGF with 
deficiency in binding VEGFR-2 will not induce endothelial cell mitogenesis, 
while those with mutated VEGFR-1 binding could still induce endothelial cell 
mitogenesis. The VEGF homodimer is positioned to bind on one side to one 
receptor and the other side to a second one.  Thus two identical or different 
receptors (homo- or heterodimers) can be linked in which activation occurs.
VEGF binds to the 2nd and 3rd Ig-like loops and VEGFR-1 receptors can be bridged
in this way.  The 4th Ig-like loops appears to contain a receptor dimerization
domain.  It should be noted that c-kit receptor is a PDGF subfamily Tyr kinase
receptor with 5 Ig-like domains, possesses a dimerization domain in the 4th
Ig-like loop.
<p>
<b>Receptor-mediated responses</b>.  Studies with cells having either VEGFR-1 or 
-2 but not both show that VEGFR-2 induces mitogenesis and vascular permeability 
and cell motility/migration, but that VEGFR-1 stimulation induces cell 
motility/migration only.  In porcine aortic endothelial cells with 
recombinant VEGFR-2, VEGF action promoted phosphorylation of Shc and Nyc adapters,
Grb-2 binding, and MAP kinase activation.  MAP kinase activation was not observed
for VEGFR-1 binding, and there is some correlation between MAP kinase activation
and mitogenesis.  SHP-1 and -2 SH2 protein tyrosine phosphatases (PTPases) 
associated with VEGFR-2 after VEGF stimulation (thus they may modulate VEGF 
signaling).  For angiogenesis, activation/expression/mobilization of ECM 
proteases must follow, and thus be downstream processes related with VEGF.
<p>
<b>Neuropilin-1</b>.  Endothelial cells have receptors smaller than VEGFR-1 and
-2 which bind only to VEGF<sub>165</sub>; these receptors have also been found
on other cell types too, including prostate and breast cancer cells in very
large amounts.  VEGF<sub>165</sub> binds to these receptors through amino acids
on the exon 7 segment of the VEGF gene, which explains the isoform specificity.
To determine the receptor gene, its protein was purified on VEGF<sub>165</sub>
affinity chromatography using extracts of MDA-MB-231 breast cancer cells, and
the protein was shown to be neuropilin-1, which is also a receptor for several
semaphorins.  Semaphorins are factors acting as repellants of nerve growth cones.
A related VEGF<sub>165</sub> receptor was later identified by homology searches
and is neuropilin-2.  Neuropilin-1 can bind the heparin-binding form of PlGF, 
PlGF-2, but not PlGF-1.  Neuropilin-1 has a short intracellular domain
and probably do not act as independent receptors; cells with no VEGFR but having
neuropilin-1 show no apparent activation to VEGF<sub>165</sub>, but neuropilin-1
knockouts in transgenic mice are lethal and show poor cardiovascular development.
VEGF<sub>165</sub> binds better to VEGFR-2 in cells also expressing neuropilin-1,
so its role may be as a co-receptor: the migratory response was better with
neuropilin-1 than when it was absent.
<!-- ============================================================= -->
<p class="tags">
DNA purification, silica, guanidine thiocyanate
<p class="ref">
 <span class="authors">
Boom R, Sol CJA, Beld M, Weel J, Goudsmit J, Wertheim-Van Dillen P
 </span>
 <span class="year">
(1999)
 </span>
 <span class="jname">
J. Clin. Microbiol.
 </span>
 <span class="jvol">
37, 
 </span>
 <span class="pages">
615-619.
 </span>
<p class="reftitl">
Silica-Guanidiniumthiocyanate DNA Isolation Procedure Based on Selective Binding 
of Bovine Alpha-Casein to Silica Particles
<p class="Para1">
Our group has previously reported on the use of GuSCN in the binding complex between
DNA and silica and diatom particles.  In one purification scheme, the purified
DNA contained eluate materials inhibiting DNA-processing enzymes (e.g., restriction).
Buffer L6 is maded of 5.25 M GuSCN in 50 mM TrisHCl pH 6.4, 20 mM EDTA and 
1.3% Triton X-100.  Buffer L7A is made by adding alpha-casein to final 1 mg/ml
to L6.  Stored at room temp in dark it is stable for 2 months.  A wash buffer of
L2 with just GuSCN in Tris (no EDTA or Triton X-100) is used in the method as well.
To purify DNA in biofluids (CSF, serum, plasma, urine) or water, 20 &micro;l of
size-fractionated silica particles are mixed with L6 or L7A.  Elution is with 
100 &micro;l TE pH 8.
<p>
In testing the system, &lambda; phage DNA was added to eluates and activities
of restriction enzymes tested.  When pure water is extracted with L6, it is
clear there is inhibitor of <i>Eco</i>RI enzyme;  dilution of the eluate restores
activity.  Thus the inhibitor is in the purifying reagents.  If buffer L7A is
used, no inhibition of activity occurs.  Note that other enzymes were inhibited,
and other DNA substrates tested, so it is not specific to a particular DNA or
restriction enzyme.  When serum was used instead of pure water, there is no
inhibition, as for buffer L7A;  thus a factor in serum prevents this inhibtion.
If restriction enzyme digestion is affected, what about another post-purification
reaction such as PCR?  50 molecules of an internal control DNA were added to the
pure water and serum extracts in which the lysis buffer was L6 and L7A, then 
PCR was done, with the PCR result analyzed by hybridization and detection by
enzyme chemiluminescence (ECL).  L6 eluates of water inhibited the PCR significantly
while L7A eluates did not.  The serum extraction showed no inhibition.
<p>
What about other biofluids other than serum or pure water?  The L6 extractions of
serum and plasma did not involved inhibition, but those of urine and CSF were just
like water:  the L6 extractions inhibited restriction enyzme action but the
L7A extractions did not.  With PCR, there was a 2- to 3-fold increase in amplicon
amount when L7A was used instead of L6.  The use of L7A was about 93% of the level
of uninhibited controls (no eluates present).
<p>
What differences exist in the extractability of DNA between L6 and L7A?  Yields
for DNA of sizes from 100 bp to 23 kb were checked visually on gels. Results show
that the recovery was nearly 100%.
<p>
Extractions were done with L6 and L7A with either water or serum input, and eluates
were run on PAGE with silver staining.  The L7A extraction shows the presence of
32 kDa protein, which happens to be present as the major band of alpha-casein
preparation (also run in control).  To control the possibility that the alpha-casein
was bound to the tube walls instead of the silica particles, the reactions were
extensively washed with L2 buffer and each time transferred to new tubes after the
wash.  The recovery of alpha-casein shows that the protein bound to the particles,
and not the reaction tube walls.  With L6 lysis with serum however, no protein 
band was observed, which mean that protein binding is not a general property of 
silica particles.
<p>
How are these results explained?  The albumin in serum is known to scavenge for
inhibitors of PCR reactions and its presence could have prevented copurification
of the inhibitor; alternatively, alpha-casein-like proteins might be copurified
from serum and relieve inhbition.
<!-- ============================================================= -->
<p class="tags">
Yeast glycoproteins, N-glycans, O-glycans, NMR spectroscopy
<p class="ref">
 <span class="authors">
Gemmill TR, Trimble RB
 </span>
 <span class="year">
(1999)
 </span>
 <span class="jname">
Biochim. Biophys. Acta
 </span>
 <span class="jvol">
1426, 
 </span>
 <span class="pages">
227-237.
 </span>
<p class="reftitl">
Overview of <i>N</i>- and <i>O</i>-linked oligosaccharide structures found in 
various yeast species
<p class="Para1">
<b>REVIEW</b>.  Glycoprotein modification in yeast and mammals have many 
homologous components.  Secretion of glycoproteins in yeast and mammals is
quite similar.  Initial glycosylation occurs in the ER cotranslationally
and then the modified protein is further processed in the Golgi.  The yeast
cell wall is composed of glycoproteins and polyglucose polymers and the glycoproteins
are often retained in the periplasmic space.  Temperature-sensitive mutants
of <i>Saccharomyces cerevisiae</i> lose the ability to secrete proteins
(<i>sec</i><sup>&minus;</sup>), and these mutants have enabled dissection of 
secretory process.  However, differences in glycosylation between mammals and
fungi/yeast are such that they are antigenic or they have immunosuppressive
or lymphoproliferative effects.  Some sequester important hormones or molecules
(cytokines, structural proteins, etc) and cause adverse effects.  Termini of the
oligosaccharide are usually galactose or mannose, and these would be removed
from biological compartment by liver asialoglycoprotein receptor or macrophage
mannose receptor.
<p>
For N-glycosylated proteins, N-glycosylation begins with transfer of a 
triantennary (Glc-Glc-Glc-Man-Man-Man)([Man-Man]<sub>2</sub>Man)Man-GlcNAc-GlcNAc
from dolicholpyrophosphate to the N-X-S/T consensus sequence.  The three
Glc on one antenna are trimmed quickly by glucosidases I and II.  In
<i>S cerevisiae</i> (but not <i>S. pombe</i>), a Man is removed from another
antenna by ER Man<sub>9</sub>-&alpha;-mannosidase.  In buiiding the triantennary
oligosaccharide, five mannoses are attached to chitobiose-PP-dolichol lipid on
cytosolic face of ER:  the transferase uses GDP-Man.  The four remaining Man
and three Glc are added on luminal ER side using Dol-P-Man and Dol-P-Glc.
Further processing occurs in the Golgi in yeast.
<p>
All N-glycans of yeast have a high mannose core, with antennary extensions
different from mammals.  Single-celled eukaryotes, yeast no exception, do not
add sialic acids to the ends of glycan termini.  Instead anionic groups are
added to glycan via phosphorylation orby glucuronic acid or pyruvate.  A higher
eukaryotic parallel is the phosphorylation of high-mannose oligosaccharides
used to modify lysosomally targeted enzymes.
<p>
Generally only Man or Man+Gal make up yeast glycans.  Other species add unusual
monosaccharides such as Xyl.  Only <i>K lactis</i> is known to add GlcNAc although
other species of yeast are also suspected as well; note the UDP-GlcNAc antiporter
of <i>K lactis</i> has been cloned and characterized.  Yeast elongate the 
oligosaccharides into a <i>core</i> series (Hex<sub>&lt;15</sub>) and an 
<i>outer chain</i> family of structures (Hex<sub>&gt;15</sub>).  Elongation
is with Man only, left unmodified or phosphorylated, or including Gal with 
Man.  The elongation are &alpha;1&rarr;6 linkages.  Sometimes a single 
monosaccharide is added &alpha;1&rarr;2 as the start of another antenna.
Diesterified phosphate is added to elongated mannans, with phosphate linked by
C-1 of a mannose to another mannose on side chain of elongated oligosaccharide.
<p>
<b>O-glycosylation</b> differs entirely between yeast and mammals.  Mammalian
process starts in the Golgi with UDP-GalNAc donating GalNAc, while in yeast 
Man-P-Dol is the donator to Man in the ER.  Further addition of Gal, GlcNAc,
Fuc, and NeuNAc in the mammalian Golgi elongates the oligosaccharide.  In fungi
Man and Gal elongate the initial Man residue, all in the Golgi.  In particular
there is a Man-&alpha;1&rarr;2-Man-&alpha;1&rarr;2-Man-Ser/Thr motif in yeast
which might be antigenic in humans.  The Man<sub>1-3</sub>-Ser/Thr core might
be elongated with Gal or Man to form Hex<sub>&lt;8</sub> oligosaccharides.  
While O-glycans in mammals often have charged groups at the end of antenna,
yeast O-glycans show only rare reports of the same.
<p>
No generalizations can be made about interspecific differences in glycosylation.
In fact there are oligosaccharide structural heterogeneity exists on the same
protein in the individual organism.  Moreover, there are many reports using
harsh chemical methods to study molecular structure, which may have been altered
significantly.  Caution is advised if methods made use of these chemical methods.
<p>
<b>S. cerevisiae</b>.  If <i>S cerevisiae</i> does any trimming from the 
initial Glc<sub>3</sub>Man<sub>9</sub>GlcNAc<sub>2</sub>, it is to a core
Man<sub>8</sub>GlcNAc<sub>2</sub> which is then elongated.  If Man is added
&alpha;1&rarr;2 by Och1p to the &alpha;1&rarr;6-Man already on the core, then
the fate is not to remain at the core, but to elongate.  Whether the glycosylation
site remains core or becomes elongated may be due to steric hindrances of 
protein 3D structure inhibiting access of glycosyltransferases that perform
elongation.  Evidence of structural similarity suggests that outer-chain
specificity in elongation is preserved between species.  <i>S cerevisiae</i>
shows not Gal in oligos despite exisitence of UDP-Gal antiporter.  N-glycan
core elongation is with &alpha;-1&rarr;2, &alpha;-1&rarr;3, and &alpha;-1,6
Man.  The <i>Kre</i> and <i>Ktr</i> family of mannosyltransferases account for
&alpha;-1&rarr;2, and additions of &alpha;-1&rarr;3 to the &alpha;-1&rarr;2 is
done by Mnn1p in late Golgi.   This yeast also added phosphesters using enzymes
like Mnn4p and Mnn6p.  <i>O</i>-glycosylation is generally by addition of 
Man from 1 to 6 units and this featue is necessary for yeast survival.  The 
<i>PMT</i> family of mannosyltransferases start the initial Man addition to
Ser or Thr in the ER.  Elongation with further mannoses is by Mnn1p and the
<i>Kre</i> and <i>Ktr</i> transferases in the Golgi compartments.  Phosphates
are also attached to the O-glycans.
<p>
<b>Pichia pastoris</b>.  Core N-glycans with Man<sub>8-11</sub>GlcNAc<sub>2</sub>
and larger phosphomannans.  Some glycoproteins have smaller N-glycans attached
exclusively, and other proteins have large mannans attached exclusively.
Elongation starts with &alpha;-1&rarr;6 attachment of Man to a Man with 
&alpha;-1&rarr;3 link to the lower arm of the core Man<sub>8</sub>GlcNAc<sub>2</sub>.
Man elongation is shorter than with <i>S cerevisiae</i>.  Cores also get
phosphorylated.  Because <i>Pichia</i> is closely related to <i>Hansenula</i>, 
its biochemical properties might be similar this genus.
<p>

<i>S pombe</i>.  This species is gaining favor for eukaryotic gene expression. 
After removing the three Glc residues from the ER-glycosylated form, the 
Man<sub>9</sub>GlcNAc<sub>2</sub> is not trimmed to Man<sub>8</sub>GlcNAc
as with others.  To N-glycans, a &alpha;-1&rarr;2 and unusual &alpha;-1&rarr;3 
Gal is added to core.  Elogation is by &alpha;-1&rarr;6 Man is then decorated 
with &alpha;-1&rarr;2 Gal.  Mutations to the glycosylation machinery have been 
done, and Man-P-Dol synthase, galactosyltransferase, Gal epimerase, and UDP-Gal 
transporter haven been cloned.  Mutants of Gal epimerase and Gal transporter
show that they are not lethal.   Galactosyltransferases <i>gma12</i> and
<i>gth1</i> have been identified as well as five ORS with homology to <i>gth1</i>
(<i>gth2-gth6</i>).  The O-glycan formed by <i>S pombe</i> are Gal<sub>0-2</sub>Man<sub>1-3</sub>
variety.  Inner Man chains is as for <i>S cerevisiae</i>: Man<sub>2</sub>
by &alpha;-1&rarr;2, and Man<sub>3</sub> by same linkage.
<!-- ============================================================= -->
<p class="tags">
HBsAg, epitope mapping
<p class="ref">
 <span class="authors">
Paulij WP, de Wit PLM, S&uuml;nnen CMG, van Roosmalen MH, Petersen-van Ettekoven 
A, Cooreman MP,  Heijtink RA
 </span>
 <span class="year">
(1999)
 </span>
 <span class="jname">
J. Gen. Virol.
 </span>
 <span class="jvol">
80,
 </span>
 <span class="pages">
2121-2126.
 </span>
<p class="reftitl">
Localization of a unique hepatitis B virus epitope sheds new light on the 
structure of hepatitis B virus surface antigen
<p class="Para1">
Although all HBV vaccines are based on the S region of the surface antigen
possessing the <i>a</i> determinant which is the epitope that confers immunity,
Carmen et al (1990) have described an <q>escape mutant</q> in a child who
received passive-active post-exposure immunization.  A G587A point mutation
at codon 145 within the determinant results in an amino acid change.  To account
for all possible 3D structural features, large numbers of mAbs were screened 
to detect HBsAg variants and analyze the epitopes.  An epitope outside the 
<i>a</i> determinant was found outside the 3rd passage through the membrane.
<p>
mAb clones HBs.OT16, HBs.OT17, HBs.OT24, and HBs.OT40 were developed again 
native HBsAg obtained from sera of infected patients.  Clone 4-7B is another 
clone developed by a blood transfusion service.  12-mer peptides spanning the 
complete <i>adr</i> (Fujiyama et al 1983) sequence were synthesized, each 
shifted by one amino acid, and reactivity with mAbs determined using the Pepscan 
(Geysen et al 1986) procedure. After identifying the most reactive 12-mer, each 
amino acid in the 12-mer was substituted with Ala (unless the residue was Ala, 
in which it was replaced with Ser instead) to find the significance of each 
residue.  Other peptides were synthesized as necessary if they are natual variants
reported in the literature, to test their reactivity.
<p>
In addition recombinant HBsAg were synthesized for creating point mutations
using site-directed mutagenesis to find the escape mutant mechanism.  A 900 bp
containing PreS2-S sequence with subtype <i>ayw</i><sub>3</sub> was used.
Mutant constructs in plasmid pSP70 were digested with <i>Sma</i>I.  Gene fragments
were isolated from agarose gel (Gene Clean II kit, BIO 101) then blunt-end
ligated inbto pSG8puro at <i>Eco</i>RI site next to SV40 promoter, and orientation
verified by PCR using vector and insert primer.  Point mutations are 122 R/K,
134 Y/F, 145 G/R, 148 T/A, 160 K/R (see Peterson et al 1984, Ashton-Rickardt &amp;
Murray 1989, Waters et al 1992, Ohno et al 1993, Okamoto et al 1989).  Wild type
HBsAg is a positive control. COS-I cells are transformed for high expression level,
secretion, and glycosylation.  1.5 million cells are transiently transfected with
10 &micro;g plasmid DNA in 200 &micro;l PBS using electroporation (4 mm cuvette,
300 V and 125 &micro;F).  Cultured for 3 d in DMEM + 10% FBS + Gln and 50 &micro;g/ml
gentamycin.  Western blotting with mAb HBs.OT43 (specific for PreS2 domain).
Medium is harvested and any dilutions done in normal human serum.  For the ELISA,
wells are coating in 5 &micro;g/ml mAb 4-7B.  To achieve high concentrations of
the other OT mAb clones in low concentration, wells were coated with sheep anti-mouse
antibodies and then the murine anti-HBs mAbs added.
<p>
Rabbit polyclonal anti-HBs antibodies were made against the peptides that react
with 4-7B.  Peptides were to be 15 amino acids long and include the epitope
recognized by mAb 4-7B.  In addition several positions were varied for the 
epitope.  The sequence is LLVPFVQWFVGLSPT, which correspond to positions 175-189
of the S coding region.  By shortening the sequence to a 12-mer the minimal
reactive sequence was LLVPFVQWFVGL (positions 175-186).  The Ala substitution
studies show that F179, Q181, W182, G185 and L186 are essential.
<p>
Many positions in this 12-mer peptide show natural variation wiht amino acid 
substitutions.  The V177A substitution actually increases reactivity, and the
double sub V177A and V184A (both valines) slightly decreases reactivity from the
original.  The L176A single and double L176A/V184A substitutions have about the
same activity as the double Val subs.  The V184M sub is also somewhat reactive.
Other changes are damaging.  A P178Q or P178G change causes an 80% loss of
reactivity.  So does changing FV to SA at 183/184.  The F&rarr;S change alone
causes a greater loss of reactivity.  Norder et al reports that L175, F179,
V180, Q181, and L186 are all conserved in nature.  A V184D had a signficant
reduction of binding.
<p>
The P178Q and F183C substitutions are characteristic of strain 
<i>adw</i><sub>4</sub> The OTxx mAbs binding in the 120-160 region, which is the 
heart of the <i>a</i> determinant region.  A G145R or a T148A change 
significantly impacted mAb HBs.OT16 binding.  HBs.OT40 was affected more by 
G145R than T148A.  G145R increased binding of HBs.OT17.  The R or K at position 
122 distinguishes the <i>y</i> determinant.  Reactivity of 4-7B was unaffected 
by these changes, indicating that the substitutions did not probably alter 
epitope structure on another part of the protein chain.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Poole CF, Dias NC
 </span>
 <span class="year">
(2000)
 </span>
 <span class="jname">
J. Chromatogr. A
 </span>
 <span class="jvol">
892,
 </span>
 <span class="pages">
123-142.
 </span>
<p class="reftitl">
Practitioner's Guide To Method Development in Thin-Layer Chromatography
<p class="Para1">
<b>REVIEW</b>.  Zone capacity refers to the ability to hold an amount/mass of
a separated component at the completion of the method and which achieves
separation or baseline resolution.  Experimental observation shows that
zone capacity with respect to count of resolvable components is far less than
the theoretical.
<p>
Sample information is important.  Is it ionized and should consideration include
ion suppression techniques with buffered mobile phases where a p<i>K</i><sub>a</sub>
is known?  What about using ion-pairing reagents?  Will ion-exchange sorbents be
useful?  Derivatization of functional groups may be necessary with silica gel
layers, and in some cases a chemically bonded phase could be a better choice.
<p>
<b>Mode Selection</b>.  The following query path guides the chemist to the
proper mode:
<table class="nobkgd right">
<tr><td rowspan="3" class="rborder">Sample &lt; MW 1000
  <td colspan="2"  class="rborder"> &rarr; Organic Soluble?
  	  <td>LSC or BPC
  <tr><td rowspan="2"  class="rborder"> &rarr; Water Soluble?
  	  <td  class="rborder">&rarr; Non-ionic?
  	  	  <td>RPC or BPC
     <tr><td  class="rborder">&rarr; Ionic?
     	  <td>RPC or IPC
<tr><td rowspan="2"  class="rborder">&rarr; Sample &gt; MW 1000
   <td colspan="2" class="rborder">&rarr; Organic Soluble?
      <td>PC
   <tr><td colspan="2"  class="rborder">&rarr; Water Soluble?
   	<td>Cellulose
<tr><td colspan="4">
LSC = liquid-solid chromatography on inorganic oxide sorbent<br>
BPC = liquid-solid chromatography on chemically bonded sorbent<br>
RPC = reversed-phase:  water-containing mobile phase on a chemically
bonded stationary phase<br>
IPC = ion-pair chromatrography using reversed-phase conditions
PC = precipitation chromatography
</table>
<p>As for inorganic oxide and chemically bonded sorbents small-pore
(&lt; 10 nm avg pore diam) optimized for molecules having masses below
700; separation of polymers soluble in organic solvents are limited;
precipitation chromatography are separations based on solubility
differences in a solvent gradient generated by mobile phase demixing
is best option.  Water-soluble biopolymers are best separated on cellulose
layers.  Inorganic oxides (silica gel, alumina, kieselguhr, Florisil) are
are first choice for separating  low MW mass organics soluble in organics.
Selectivity may be affected by proportion of silanols in silica gel.
Alumina contains hydroxyl groups, aluminum cations, and oxide anions
which vary between preparations and should only be used when silica fails
or produces irregular zone profiles.
<p>
With silica compounds have very weak or very strong polarity may not
have good selectivity because of respective weak or strong retention.
Ionic or easily ionized compounds are separated using reversed-phase
with a buffered mobile phase for weak acids or bases or ion-pair reagents
for strong acids or bases.
<p>
As for ion-exchange sorbents, very few are available.  Aminopropylsiloxane-bonded
silica gel is a weak anion exchanger at low mobile phase pH.
<p>
As for chemically bonded phases, alkylsiloxane-bonded silica gel have
a high level of surface modifications and are incompatilbe with aqueous mobile
phases.  These are used for normal phase separations with compounds having
low polarity.  Alkylsiloxane-bonded silica gel with lower degree of surface
bonding and slightly larger particle size and modified binder are suitable for
reversed-phase.  Use RPC when resolving weak acids and bases after ion
suppression, strong acids and bases with ion-pairing, and in separation
of water-soluble polar organics.  Use RPC or NPC when resolving hydrocarbon-like
and polycyclic aromatics.
<p>
Other bonded phases include phenylsiloxane, but components resolving with
this resolve just as well with alkylsiloxane phases.  Cyanopropylsiloxane
is useful in both RPC and NPC;  with NPC it has properties similar to
low capacity silica gel, and in RPC it has properties similar to short-chain
alkylsiloxance with no selectivity for dipole-type interactions.
Aminopropylsiloxane is used mainly in NPC and IEX with limited retention
in RPC; it selectively retains compounds by hydrogen-bond interactions
in NPC and will function as weak anion exchanger in acidic mobile phase.
A spacer-bonded propanediol siloxane is used in NPC and RPC but more useful
for NPC because of low retention in RPC.  Compounds which hydrogen bond
or have dipole-type interactions retain well.
<p>
Strength of a solvent is its ability to cause migration in chromatography.
Strength is not a constant, but vary with stationary phase.  For example
water is strong in silica gel but weak with chemically bonded silica gel
layers.  Selectivitiy determines which solvents  have the strength for
a separation.
<p>
Four methods rank solvent selectivity.  Hildebrand solubility parameter
approach usually not used for TLC and is not considered.  Snyder's
solvent selectivity triangle is the most widely used but the least
satisfactory:  classification is based on interaction with three
prototypical solutes determined by gas-liquid distribution coefficients
corrected for differences in solvent molecular mass and dispersion interactions.
The sum of three polar distribution constants provide a measure of solvent
strength (P&prime;).  The selectivity coordinates are plotted on the surface
of a triangle (<i>x</i><sub>e</sub> + <i>x</i><sub>d</sub> +
<i>x</i><sub>n</sub> = 1) shows results grouping solvents into eight (8)
categories.  Solvents in the category have equivalent strengths characteristics.

<table>
<caption>
Solvent strength and selectivity parameters based on Snyderï¿½s selectivity triangle
(<i>S</i><sub>T</sub> is an empirical solvent strength parameter used in
reversed-phase chromatography)</caption>
<col><col style="text-align:center;">
<col span="5">
<tr><th rowspan="2">Solvent
	 <th rowspan="2">Selectivity group<th colspan="2">Solvent strength
 <th colspan="3">Solvent selectivity
<tr><th>(<i>P</i>&prime;)<th>(<i>S</i><sub>T</sub>)
	 <th><i>x</i><sub>e</sub> <th><i>x</i><sub>d</sub> <th><i>x</i><sub>n</sub>
<tr><td><i>n</i>-Butyl ether <td rowspan="4">I <td>2.1 <td> <td>0.44 <td>0.18 <td>0.38
<tr><td>Diisopropyl ether <td>2.4 <td> <td>0.48 <td>0.14 <td>0.38
<tr><td>Methyl <i>tert</i>-butyl ether <td>2.7 <td> <td> <td> <td>
<tr><td>Diethyl ether <td>2.8 <td> <td>0.53 <td>0.13 <td>0.34
<tr><td><i>n</i>-Butanol <td rowspan="5"> II <td>3.9 <td> <td>0.59 <td>0.19 <td>0.25
<tr><td>2-Propanol<td>3.9 <td> <td>0.55<td>0.19<td>0.27
<tr><td>1-Propanol<td>4.0 <td> <td>0.54<td>0.19<td>0.27
<tr><td>Ethanol<td>4.3<td>3.6<td>0.52<td>0.19<td>0.29
<tr><td>Methanol<td>5.1<td>3.0<td>0.48<td>0.22<td>0.31
<tr><td>Tetrahydrofuran<td rowspan="4">III<td>4.0<td>4.4<td>0.38<td>0.20<td>0.42
<tr><td>Pyridine<td>5.3<td><td>0.41<td>0.22<td>0.36
<tr><td>Methoxyethanol<td>5.5<td><td>0.38<td>0.24<td>0.38
<tr><td>Dimethylformamide<td>6.4<td><td>0.39<td>0.21<td>0.40
<tr><td>Acetic acid<td rowspan="2">IV<td>6.0<td><td>0.39<td>0.31<td>0.30
<tr><td>Formamide<td>9.6<td><td>0.38<td>0.33<td>0.30
<tr><td>Dichloromethane<td rowspan="2">V<td>4.3<td><td>0.27<td>0.33<td>0.40
<tr><td>1,1-Dichloroethane<td>3.5<td><td>0.30<td>0.21<td>0.49
<tr><td>Ethyl acetate<td rowspan="5">VI<td>4.4<td><td>0.34<td>0.23<td>0.43
<tr><td>Methyl ethyl ketone<td>4.7<td><td>0.35<td>0.22<td>0.43
<tr><td>Dioxane<td>4.8<td>3.5<td>0.36<td>0.24<td>0.40
<tr><td>Acetone<td>5.1<td>3.4<td>0.35<td>0.23<td>0.42
<tr><td>Acetonitrile<td>5.8<td>3.1<td>0.31<td>0.27<td>0.42
<tr><td>Toluene<td rowspan="3">VII<td>2.4<td><td>0.25<td>0.28<td>0.47
<tr><td>Benzene<td>2.7<td><td>0.23<td>0.32<td>0.45
<tr><td>Nitrobenzene<td>4.4<td><td>0.26<td>0.30<td>0.44
<tr><td>Chloroform<td rowspan="3">VIII<td>4.3<td><td>0.31<td>0.35<td>0.34
<tr><td>Dodecafluoroheptanol<td>8.8<td><td>0.33<td>0.40<td>0.27
<tr><td>Water<td>10.2<td>0 <td>0.37 <td>0.37 <td>0.25
</table>
<p>
A significant limitation of the solvent selectivity triangle approach is the
association of an individual molecular interaction with properties of a single
solute.  EtOH is used to determine solvent hydrogen bond basicity (<i>x</i><sub>e</sub>),
dioxane hydrogen bond acidity (<i>x</i><sub>d</sub>), and nitromethane dipole-dipole
interactions (<i>x</i><sub>n</sub>).  EtOH is a stronger hydrogen bond base
than it is an acid and considerably dipolar, and all three solutes are
are significant H-bond bases.  If solubility of EtOH is due to its H-bond
acidity, then the basicity value is inflated.  It is impossible to find a solvent
that is an H-bond base or acid that is not dipolar.
<p>
These problems are circumvented using the solvatochromic scale of solvent
selectivity based on studies by Kamlet-Taft and coworkers looking at spectroscopic
absorption band shifts connected with solvent effects.  These parameters are
are <i>&pi;</i><sub>1</sub>*, measure of solvent dipolarity and polarizability,
&alpha;<sub>1</sub> and &beta;<sub>1</sub> which are H-bond acidity and basicity,
respectively.  Hierarchical cluster analysis can be used to form dendrograms which
group the solvents based on selectivity.  Group membership is like Snyder's
groups with some exceptions.  Ethers are weakly dipolar and H-bond basic solvents
with no H-bond basicity.  Another group is headed by pyridine with stronger
dipolar/polarizable and H-bond basicity stronger than the ethers with no
H-bond basicity.  Nitrobenzene is loosely connected with the group and is
a weaker H-bond base.  Dichloromethane and chloroform are grouped together
but have differences; they have intermediate dipolarity and H-bond acidity but
not H-bond bases.  Ethyl acetate is part of a large group with intermediate
dipolarity/polarizability with no or weak H-bond acidity.  They are similar to
group containing pyridine but less intense.  MeCN is connected but behaves
independently.  Toluene and benzene have intermediate dipolarity and weak H-bond
basicity and no H-bond acidity.  Alcohols are strong H-bond acids and bases
with intermediate dipolarity/polarizability.  Methanol is apart from the main group
because of its greater capacity for intramolecular H-bonding.   Water and
2,2,2-trifluoroethanol are both strong H-bond acids with mininal H-bond basicity.
<p>
One drawback to the solvatochromic approach is that it gives weight to polar
interactions but not cohesion.  Transfer of solute from one solvent to another
occurs with cancellation of dispersion interactions, but the energy required
for cavity formation in the two solvents is not necessarily self-canceling
and when one of the solvents is water, cancellation of the cavity term is unlikely.
<p>
Abraham developed a strategy for gas chromatography which considers a contribution
from cavity formation in addition to polar interactions.  For gas-liquid
distribution:
<p class="eqnserif">
	log <i>K</i><sub>L</sub> = <i>c</i> + <i>l</i> log <i>L</i><sup>16</sup>
	+ <i>rR</i><sub>2</sub> + <i>s&pi;</i><sub>2</sub><sup>H</sup> +
	<i>a</i> &sum; <i>&alpha;</i><sub>2</sub><sup>H</sup> +
	<i>b</i> &sum; <i>&beta;</i><sub>2</sub><sup>H</sup>
<p>
where log <i>K</i><sub>L</sub> is the gas-liquid distribution constant,
log <i>L</i><sup>16</sup> is the distribution constant for the solute between
a gas and <i>n</i>-hexadecane at 298 K, <i>R</i><sub>2</sub> is the excess
molar refraction (in cm<sup>3</sup>/10), <i>&pi;</i><sub>2</sub><sup>H</sup>
is the ability of the solute to stabilize a neighboring dipole by virtue of
its capacity for orientation and induction interactions, and
&sum; <i>&alpha;</i><sub>2</sub><sup>H</sup> and
&sum; <i>&beta;</i><sub>2</sub><sup>H</sup> aer the solute's effective H-bond
acidity and H-bond basicity, respectively.  The system constants are defined
by the complementary interactions with the solute descriptors.  Constant <i>r</i>
refers to capacity of solvent for interaction with solute n- and &pi;-electrons,
constant <i>s</i> for capacity for dipole-type interactions, constant <i>a</i>
characterizes solvent's H-bond basicity (basic solvent interacts with acidic
solute), constant <i>b</i> the solvent's H-bond acidity, and the <i>l</i> together
with <i>c</i> term accounts for contributions from cavity formation and dispersion
interactions.  Multiple linear regression analysis is used to determine the
system constants for a group of varied solutes, sufficient in number to establihs
statistical and chemical validity of model.
<p>
A table shows the results for GLC analysis.  Benzene heads a group with low
cohesion, weak H-bond properties and is moderately dipolar/polarizable.
Chloroform and diiodomethane are singular solvents, with low cohesion
and intermediate dipolarity/polarizability but with significantly more H-bond
acidity than aromatic hydrocarbons.  Hydrocarbons and CCl<sub>4</sub> are low
cohesive solvents with little polar interaction capacity.  3-ethylphenol and
2,2,2-trifluoroethanol are loosely grouped together but behave independently.
They are strong H-bonding acids, moderately cohesive, and dipolar/polarizable.
Alcohols have a general blend of all polar interactions and are moderately cohesive.
MeCN and water are singular solvents, with water being the most cohesive and
strongest H-bonding acid, but also being a strong H-bond base and having significant
dipolar/polarizability.  MeCN is moderately cohesive, stongly dipolar,
and quite H-bond basic with weak H-bond acidity.
<p>
In selecting mobile phases for optimization, the initial use of the following
is suggested:  hexane, toluene, methyl <i>tert</i>-butyl ether, dichloromethane,
chloroform, ethyl acetate, acetone, pyridine, triethylamine, acetonitrile,
methanol, 2-propanol, 2,2,2-trifluoroethanol, acetic acid, and water.
Hexane is the strength-adjusting solvent for NPC when solutes are weak and
moderately polar.  For polar compounds the strongest solvent that fails to
result in migration of sample components is used.  This also allows solvents
that are immiscible with hexane to be introduced into mobile phase optimization
process.  For RPC water is always the strength-adjusting solvent and the
solvents useful for mobile phase optimization are restricted to ones miscilble
with water: MeOH, 2-proponal, 2,2,2-trifluoroethanol, acetone, pyridine (or
N,N-dimethylformamide) and MeCN.  Solvents which are strong H-bond bases with
weak dipolarity are not common.  Triethylamine is used as a ternary component
because of its low water solubility.  Acetone, MeCN, 2-propanol, methanol, and
2,2,2-trifluoroethanol provide a range of H-bond acidity.  MeCN (or dioxane),
acetone (or THF), and pyridine (or N,N-DMF) provide a range of H-bond basicity.
Only MeCN is a significant H-bond acid, but the rest are dipolar. N,N-DMF and
pyridine are stronger H-bond bases but otherwise similar to each other for polar
interaction capacity.  N,N-DMF has low volatility (but takes longer to evaporate)
while pyridine is malodorous and is stonger protonic base in aqueous systems.
These dictate whether pyridine or N,N-DMF is chosen.
<p>
In optimizing mobile phase, HPLC and TLC are very much alike in chemistries but
the major difference is that equiibrium is not obtained because of evaporation.
Use of multi-component systems may result in solvent gradients in the direction
of development as a result of demixing.  Stationary phase newly wetted will
show affinity for a component resulting in the solvent front having a different
composition compared to bulk mobile phase entering the layer.  With complete demixing
zones with sharp boundaries form, creating a chromatogram marked by sections with
different solvent composition, and thus selectivity.  These demixing effects are
less apparent when the chamber atmosphere is saturated.
<p>
Solvents of correct strength should move solute with an <i>R</i><sub>F</sub> between
0.2 and 0.8, and if of the correct selectivity will achieve resolved separation between
solute components.
<p>
<b>PRISMA</b>.  This model has been used to optimize multicomponent mobile phases
for TLC and HPLC.  Three solvents with highest selectivity are chosen initially
to provide the highest zone separation.  Between 3 and 5 solvents can be used
in the model.  Modifiers, e.g. acids and ion-pair reagents, can be added to
improve separation or stop tailing.  The model is a 3-dimensional geometric design
correlating the strength of the solvent with the selectivity of the mobile phase.
There are 3 parts:  the base or platform representing the modifier, the regular
part of the prism having congruent base and top surfaces, and an irregular truncated
top prism (frustum).  The length of the prism edges (<i>S</i><sub>A</sub>,
<i>S</i><sub>B</sub>, <i>S</i><sub>C</sub> correspond to solvent strengths
of neat solvents (A, B, C).  The edges of the prism would have unequal length
since the solvents have different strengths.  As such the very top surface of the
prism is not parallel or congrous with the base.  Cutting the top surface of the
lowest edge (weakest solvent) produces a regular prism.  The upper frustum is used
for mobile phase optimizatin of polar compounds in NPC, and the regular part is
used for separation of nonpolar compounds and moderately polar substances.
<p>
In polar compound optimization, points within the triangle correspond to the
volume fraction mobile phase componsition.  For binary solvents, only the side
of the triangle is used.  At a point (<i>P</i><sub>S</sub>) within the triangle
optimization begins at <i>P</i><sub>S</sub> = 333, in a triangle in which solvent
fractions are divided into 8 ways, such that the points of the triangle have
<i>P</i><sub>S</sub> = 811, 181, and 118.
<p>
For separation of nonpolar and moderately polar compounds, the initial solvent
composition should be at the center of theh triangular top face of the regular
prism (<i>P</i><sub>S</sub> = 333), diluted to bring all sample components
into the <i>R</i><sub>F</sub> range 0.2-0.8.  Three more runs are done with solvent
compositions near the apexes of the triangle.  When using saturated developing
chambers there is a linear correlation between <i>R</i><sub>F</sub> and solvent
stength at a constant <i>P</i><sub>S</sub> value [ln <i>R</i><sub>F</sub>
= <i>d</i>(<i>S</i><sub>T</sub> + <i>e</i>, where <i>S</i><sub>T</sub> is solvent
stengnth and <i>d</i> and <i>e</i> are regression constants].  At constant
solvent strength there is quadratic relationship between <i>R</i><sub>F</sub>
and selectivity points describing the retardation surface [<i>R</i><sub>F</sub>
= <i>a</i>(<i>P</i><sub>S</sub><i>P</i><sub>S</sub>)<sup>2</sup> +
<i>b</i>(<i>P</i><sub>S</sub>) + <i>c</i>, where <i>a</i>, <i>b</i>, and <i>c</i>
are regression constants.  Model should work for NPC on bonded layers, but has
not been applied to silica gel layers.  But in column chromatography it was
show that solute size differences and site-specific surface interaction are
not adequately accounted for and that these effects would result in poor
predictions for retention in TLC as well.  PRISMA is an entirely empirical
model distinct from HPLC-specific models that works better.  The solvation
parameter model below is suitable for RP-TLC:
<p class="eqnserif" >
	<i>R</i><sub>M</sub> = <i>c</i> + <i>mV</i><sub>X</sub> + <i>rR</i><sub>2</sub> +
	<i>s&pi;</i><sub>2</sub><sup>H</sup> + <i>a</i> &sum; &alpha;<sub>2</sub><sup>H</sup>
	+ <i>b</i> &sum; &beta;<sub>2</sub><sup>0</sup>
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Manzi A
 </span>
 <span class="year">
(2000),
 </span>
 <span class="jname">
Curr. Prot. Molec. Biol.
 </span>
 <span class="jvol">
&nbsp;
 </span>
 <span class="pages">
17.16.1-17.16.11
 </span>
<p class="reftitl">
Acid Hydrolysis for Release of Monosaccharides
<p class="Para1">
<b>METHOD</b>.  This section covers a range of methods from mild to strong.
Mild methods target the release of certain functional groups while strong methods
break the oligosaccharide to constitutent monomers.  Free monosaccharides should
be separated on pellicular resid using anion exchances  with pulsed amperometric
detection.  For GLC work, methanolysis and then trimethylsilylation of -OH
and -COOH groups should be done.  When no amino-sugars or uronic acids are
present (only neutral sugars), acid hydrolysis and then reduction and
peracetylation is done.  For HPLC, direct analysis after strong acid hydrolysis
can be done for neutral sugars, hexosamines, and uronic acids; for sialic acids
mild hydrolysis is done (or enzymatic release with sialidases of O-acyl
subtitutents are to be preserved).  HPLC work can be done with or with
derivatization.
<p><b>Acid Hydrolysis for Release of Hexose, Pentoses, Hexosamines, Uronic
Acids</b>.  Prepare the following materials:
<ul>
<li>an internal standard containing monosaccharide known not to be in sample
<li>standard mixture:  if GLC is to be done on the alditol acetates,
 prepare D-(+)-glucose (anhydrous, mixed anomers), D-(+)-galactose (crystalline),
  D-(+)-mannose (crystalline), and D-(+)-xylose (99%) in equimolar mixtures.
If HPAE-PAD is done, prepare glucose, mannose, and galactose above (no xylose)
  and add N-acetyl-D-galactosamine, N-acetyl-D-glucosamine, and D-glucuronolactone.
<li>99% MeOH anhydrous
<li>heating block or oven
<li>small glass culture tubes or 3.5 ml glass vials with PTFE-lined screw caps
<li>nitrogen stream o vacuum evap system
</ul>
<p>Perform the following steps:
<ol>
<li>Use a phenol-sulfuric acid assay to estimate total carbohydrates.
</ol>
<p>
<b>Fucose release by mild acid hydrolysis</b>.  If only fucose (and sialic
acids) are released then use this method.  If there is no particular need to
isolate fucose first, then do a total hydrolysis.  Prepare:
<ul>
<li>1 &micro;mol/ml L-fucose standard in water
<li>50 mM HCl from a constant-boiling grade HCl
<li>99% methanol (anhydrous)
<li>small glas culture tubes, or 3.5 mol glass vials with PTFE-lined screw caps
<li>heating block or oven set at 100&deg;
<li>nitrogen stream or vacuum evaporation system (centrifugal evap ok) or
    shaker-evaporator (Baxter Sci), or lyophilizer
<li>dialysis tubing with 500 Da cutoff
</ul>
<p>Perform:
<ol>
<li>estimate total carbohydrate composition with phenol-sulfuric acid assay
(see unit 17.9)
<li>dry down using vacuum evap or nitrogen stream three replicates of each
sample, which should contain &ge; 10 nmol carbohydrate, and plaed in the culture
tubes or glass vials.  Also dry down three replicates containing 10 &micro;l of
1 &micro;mol/ml L-fucose standard.  Use brand new vials or tubes; alternatively
used tubes that have been cleaned by heating &ge; 3 h at 50&deg; first in conc.
HNO<sub>3</sub> then in 6 M HCl can be used; vials are rinsed with type I water,
then EtOH, and dried in oven.  If &lt; 5 &micro;g total carbohydrate is
analyzed, glass vials should be sialinized by incubating 15 min at RT with 2%
(CH<sub>3</sub>)<sub>2</sub>Cl<sub>2</sub>Si in toluene; decant, then rinse in
MeOH and hot distilled water then let dry.
<li>Add 400 &micro;l of 50 mM HCl for up to 1 mg carbohydrate.  Flush all
containers with nitrogen and cap the tubes.
<li>Place containers in oven or heating block.  One of the replicates of
all standards and samples is heated 2 h.  Another replicate 4 h.  And the
3rd replicate for 8 h.  For heating block use, care must be taken that heat
is transferred to tubes.  Use pump oil or sand.  Most of the tube should be
immersed in the block or heating element or else distillation occurs in the
upper part of tube.  If oil is used, wipe away excess and wash vial with hexane
and let dry before opening.
<li>Dry down the sample with nitrogen stream or vacuum system.  Residues must
then be washed twice with 99% MeOH and re-dried each time to ensure acid
removal.  If the defucosylated glycoconjugate is to be recovered, reaction
mixture can be neutralized with 50 mM NaOH, then dialyzed at 4&deg; in
20 vol water in tubing with 500 MWCO.  The product can also be recovered using
Bio-Gel P2 or Sephadex G-15 gel filtration, using water mobile phase.
<li>Hydrolysates can be stored &minus;20&deg;  The results will show which
times of incubation gave the best results.
</ol>
<p>
<b>Sialic Acids Release</b>.  This releases quantitatively most types of
known sialic acids other than the 4-O-acetylated types. Prepare:
<ul>
<li>sodum formate, pH 5.5: weigh 68 mg soidum formate in water, bring to
1 L, adjust to pH 5.5 with 1 M formic acid
<li>0.1 N H<sub>2</sub>SO<sub>4</sub>
<li>10 M and 2 M acetic acid
<li>1% butylated hydroxytoluene (BHT) in EtOH
<li>Dowex AG 50W-X2, 200-400 mesh, H<sup>+</sup> form
<li>Dowex AG 3-X4A, 100-200, OH<sup>&minus;</sup> form
<li>10 mM and 1 M formic acid (ACS grade), ice cold
<li>dialysis tubign with 1000 and 12,000 Da cutoff
<li>heating block
<li>5 ml glass culture tubes with PTFE-lined screw caps
<li>0.5 &times; 10 cm glass chromatography colums or glass wool-plugged
Pasteur pipets
<li>20 ml glass test tubes
<li>lyophilizer, or shaker evaporator (Baxter)
</ul>
<p>Peform:
<ol>
<li>Homogenize sample and dialyze overnight at 4&deg; against 100 volumes
of sodium formate, pH 5.5 in tubing with 12,000 Da cutoff.  Lyophilize the
dialysate and store &minus;20&deg; if not to be analyzed immediately.  Dialysis
is not required if the sample does not have low MW contaminants.  If the crude
sample possibly contain O-acetylesterases, add to 1 mM diisopropyl
fluorophosphate (DFP, EXTREMELY TOXIC!!) for 15 min on ice.
<li>Dissolve sample in water.  De-O-acetylate the sample, release the sialic
acids with 0.1 N H<sub>2</sub>SO<sub>4</sub>, and determine total sialic acid
content using TBA assay. These are described elsewhere.
<li>From the first step, make three replicates of each sample, each about
10 nmol of sialic acids, to 5 ml glass culture tubes.
<li>Add 10 M acetic acid to final 2 M concentration, then add 1 vol of 1% BHT
per 100 vols of sample (BHT will slow down lipid peroxidation).
<li>Place the tubes in an 80&deg; oven or heating block, one replicate for 2 h,
another for 4 h, and another for 8 h.  Take care when using heating block (as
noted for the mild fucose hydrolysis above).
<li>Dry down the sample when cooled.  The de-O-acetylate then and determine
total free sialic acid by TBA.  Compare the numbers to see the release.
</ol>
<p><b>Releasing and purifying sialic acids</b>.  (Does not work for
4-O-acetylated sialic acids, see next method set).  Perform:
<ol>
<li>From the first step above, add 2 M acetic acid to the dried-down sample
from the steps above.  Add up to 400 &micro;l for 1 mg sample.  Incubate 80&deg;
for the best times discovered from the latter steps above.  Then cool the sample.
<li>Transfer reaction mix to dialysis tubing with 1000 MWCO.  Also wash tube
twice and transfer these contents to tubing.  Dialyze at 4&deg; in 10 vol water
and retain the dialysate (solution outside tubing).  Note that when equilibrium
is achieved, 90% of the free sialic acids are outside the tubing.  Dialysis can
be repeated to improve recovery, or just make the assumption.
<li>Prepare a 1 ml Dowex AG 50W-X2 resin in plugged pipets or in 0.5x10 cm column.
Use cold materials in a cold room to prepare the materials.  All eluents and
materials should be ice-cold and the steps done at 4&deg;.
<li>Load the dialysate to the column.  Wash with 6 ml cold water.  Collect
eluate in 15 ml glass test tube on ice.
<li>Check eluate pH:  if &gt; 3, to to next step.  If &lt; 3, add 10 mM sodium
formate pH 5.5 until pH &gt; 3.
<li>Set aside 1 ml of Dowex 3-X4A resin hydroxyl form for each column to
be prepared.  Mix resin with 3 vol of 1 M formic acid to convert to formate
form.  Leave 15 min.  Now add the resin to plugged Pasteur pipet or column.
Pass throught 10 mM sodium formate until the efflent reads pH 5.5.
<li>Load sample prepared from the step before the last one.  Wash column
immediately with 7 ml ice-cold 10 mM formic acid and discard effluent.
<li>Elute the sialic acids with 10 ml ice-cold 1 M formic acid.  Collect on ice
in 20 ml glass test tube.  Evaporate this to dryness using lyophilizer or
shaker-evaporator (water bath temp &lt; 37&deg;.  Store &minus;20&deg; until
analysis.
</ol>
<p><b>Release of 4-O-acetylated sialic acids</b>
<ol>
<li>Using an aliquot of sample, determine total sialic acid content using TBA
assay.
<li>Dialyze the sample, if not done, and adjust pH to 2.0 with conc. HCOOH.
Heat to 70&deg; for 1 h, cool, and spin.  Separate the supernatant and pellet,
but retain both.
<li>Dialyze supernatant 4&deg; overnight with 10 vol water using 1000 MWCO
tubing.  Repeat dialysis twice, recovering all solutions outside the tubing.
Dry these down with lyophilizer.
<li>Suspend the pellet in water, and adjust to pH 2 and heat as before.  Do it
twice (total three times).  Each time spin it down again and dialyze the
supernatants, collect the dialysates and lyophilize.
<li>Pool all the materials that were lyophilized and analyze by ion-exchange
chromatography.
</ol>
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Gonias SL, Carmichael A, Mettenburg JM,  Roadcap DW, Irvin WP, Webb DJ
 </span>
 <span class="year">
(2000),
 </span>
 <span class="jname">
J. Biol. Chem.
 </span>
 <span class="jvol">
275,
 </span>
 <span class="pages">
5826-5831
 </span>
<p class="reftitl">
Identical or Overlapping Sequences in the Primary Structure of Human 
&alpha;<sub>2</sub>-Macroglobulin Are Responsible for the Binding of Nerve 
Growth Factor-&beta;, Platelet-derived Growth Factor-&beta;, and Transforming 
Growth Factor-&beta;
<p class="Para1">
For this work, fusion proteins with GST containing segments of the 
&alpha;<sub>2</sub> maoroglobulin (a2M) sequence were created.  FP1 represents 
amino acids 99-392, FP2 aa 341-590, FP3 aa 591-774, FP4 aa 775-1059, FP5 aa 
1030-1279 (note this is a re-numbering to conform to the mature subunit.  FP6 
(aa 1242-1451) was constructed using PCR with a2M cDNA in pBluescript. 
TGF-&beta;2 binds to FP3 but not the other cloned sequences.  Previous studies 
have shown that TGF-&beta;1 does not bind to 18 kDa C-terminal peptide created 
by papain digestion.  In a first experiment <sup>125</sup>I-TGF-&beta;1 was 
confirmed to bind FP3 as before but not to any other including FP6.  It 
was also tested on PVDF-immobilized 18 kDa fragment and pure GST, and bound
to neither.  PDGF-BB binds intact a2M, but PDGF-AA does not:  labeled PDGF-BB
at 15 pM was incubated with PVDF-immobilized FP1 to FP6:  like TGF-&beta;1
it also binds to FP3, but no other.  At higher PDGF-BB concentrations (30 and
60 pM) results were same.  PDGF-BB also did not bind to 18 kDa fragment,
consistent with FP6 findings.
<p>
NGF-&beta;, NT-3, NT-4 and brain-derived growth factor are all neutrophins
which cross-compete for binding to a2M, indicating identical or overlapping
binding sites. <sup>125</sup>I-NGF-&beta; at 20 pM binds to FP3 but not to
other FPs or pure GST (40 pM also tested and found same).  Neither labeled
VEGF or TNF-&alpha; could bind to any of the a2M fusion proteins.  Thus
FP3 binding results with growth factors agree with earlier studies of these 
factors with intact a2M.  <sup>125</sup>I-TGF-&beta;1 at 200 pM was incubated
with FP3 and a2M-MA immobilized on wells of microplates:  6.9 fmol of TGF-&beta;1
binds to FP3 and 3.6 fmol of TGF-&beta;1 binds to a2M-MA, and with 200 nM
(1000-fold more) unlabeled TGF-&beta;1, binding inhibition is &gt; 85% 
(indicates specific binding).  There &lt; 0.1 fmol binding with FP6 is coated
to wells, or with wells just blocked with PBS-Tween 20.  PDGF-BB incubated
with a2M-MA and FP3 also inibited binding of labeled TGF-&beta;1.  At 800 nM
PDGF-BB (highest used), TGF-&beta;1 could not bind FP3 by &gt; 95%.  TGF-&beta;1
binding inhibition to FP3 was 300 nM PDGF-BB, which happens to be <i>K</i><sub>D</sub>
for PDGF-BB/FP3 interaction.  IC<sub>50</sub> of PDGF-BB for inhibiting 
TGF-&beta; to a2M-MA was 270 nm, close the 370 nM value reported for binding
PDGF-BB to a2M-MA in solution, indicating that a2M-MA immobilization does not
seriously affect interaction.
<p>
NGF also interfered with binding of TGF-&beta;1 to FP3 and a2M-MA coated to
microplate wells, by up to 85%.  IC<sub>50</sub> values were 180 and 150 nM
for TGF-&beta;1 binding to FP3 and a2M-MA respectively.  The <i>K</i><sub>D</sub>
of NGF-&beta; binding to a2M-MA in solution has been shown previously to be
110 nM.
<p>
Murine globulin (MUG) is monomeric form of &alpha;-macroglobulin that does not
bind TGF-&beta;.  Does PDFG-BB or NGF-&beta; bind MUG if TGF-&beta; binds a2M
but not MUG?  Radiolabeled PDGF-BB was tested with native a2M, a2M-MA, and MUG.
It bound to a2M and a2M-MA, but not MUG.  Results were identical with labeled
NGF-&beta;.
<p>
A growth factor binding domain of the PDGF-&beta; receptor (R&beta;Fc) was 
purified and coated to wells.  0.57 fmol of labeled PDGF-BB was bound to 
R&beta;/Fc with 87% specific binding.  The presence of FP3 and a2M-MA inhbited 
binding by 84 and 85%, respectively; control FP6 had no effect at all. In this 
configuration, FP3 and a2M-MA bind to labeled PDGF-BB so as to decrease 
concentration of free labeled ligand able to bind receptor. The ability of FP3 
and a2M-MA to inhibit binding of labeled PDGF-BB to its receptors was also 
tested with NIH 3T3 cells: 100 pM labeled PDGF-BB was incubated with cells at 
4&deg; and binding was found to be 28 fmol/mg cell protein.  The presence of
FP3 and a2M-MA reduce specific binding by 95% and FP6 did not inhibit binding
at all.

<!-- ============================================================= -->
<p class="tags">
cloning, TA cloning
<p class="ref">
 <span class="authors">
Zhou M-Y, Gomez-Sanchez CE
 </span>
 <span class="year">
(2000),
 </span>
 <span class="jname">
Curr. Issues Mol. Biol.
 </span>
 <span class="jvol">
2,
 </span>
 <span class="pages">
1-7.
 </span>
<p class="reftitl">
Universal TA Cloning
<p class="Para1">
<b>METHOD PROTOCOL</b>.  Cloning DNA fragments into plasmid vectors is
routine recombinant DNA technology.  The most commong cloning and subcloning 
uses DNA ligase to covalently link the fragment to linearized plasmid.
Ligase-free or ligase-independent cloning has been developed for PCR-amplified
fragments.  In this, long (10-12 bases) protruding 3&prime; ends on PCR-amplified
fragments are generated and annealed to complementary DNA sequences on 
the plasmid vector.  With the ligase-dependent method, cohesive ends are
generated using restriction enzymes on plamid vector and fragment to be inserted.
A less efficient form of ligation is blunt-end.  These ligase-dependent methods
require multiple purification and modification steps.
<p>
The <i>Taq</i> DNA polymerase (among others) adds to the 3&prime; end of blunt-end
molecules an additonal base that is not template-directed, an adenine (A) base
which shows terminal transferase-like activity.  For cloning then, a vector
which has been linearized with a 3&prime;-T end can assist in ligation and
cloning.  This method is referred to as TA cloning.  In fact, it should be
preferred to blunt-end cloning.
<!-- ============================================================= -->
<p class="tags">
oligonucleotide purification, SPE, RP-HPLC
<p class="ref">
 <span class="authors">
Gilar M, Bouvier ESP
 </span>
 <span class="year">
(2000),
 </span>
 <span class="jname">
J. Chromatog. A
 </span>
 <span class="jvol">
890,
 </span>
 <span class="pages">
167-177.
 </span>
<p class="reftitl">
Purification of crude DNA oligonucleotides by solid-phase extraction and 
reversed-phase high-performance liquid chromatography
<p class="Para1">
RP-HPLC of synthetic oligos is useful in that it separates using volatile
components (triethylamine).  Increasing oligo length has the effect of reducing
resolution, with good separations of 10- to 12-mers.  Slow mass transfer is
a problem with both RPLC and anion-exchange.  Separation selectivity is minimal
for oligos greather than 20-mers.  Nonporous sorbents can improve separation but
they have limited loading capacity.
<p>
The Waters Oasis HLB 96-well plate is often used to extract drug molecules from
serum or plasma and to remove proteins and other interferants.  The concern
for use in oligo purification is that oligos have a smaller diffusion coefficient
because they have much larger molecular masses.  A crude 0.2 &micro;mol product
representing 1 mg of a 25-mer is adsorbed which is below the loading capacity.
It is loaded in 0.1 M TEAA buffer pH 8 (note that it is slightly alkaline to
avoid loss of DMT).  Different flow rates were tried:  0.25 ml/min (gravity), 
0.5, 2, and 4 ml/min.  Significant sample breakthrough is seen at the higher
flow rate: nearly half of the phosphorothioate is lost comparing 0.25 against
4 ml/min rate.  This was more apparent when a 30-mer vs. a 15-mer was used.
(all were homopolymers).  Thus there is a kinetic aspect (mass transfer rate)
for adsorption.  To pass a 1 ml sample will take 4 min loading, but since recoveries
are better, the time investment is worth it.  An alternative to loss by sample
breakthrought is to increase sorbent surface accessibility by reducing particle
size of sorbent.  The standard particle is 30 &micro;m, but this was changed to
a 9 &micro;m particle.  In doing this, even 4 ml/min flow rates did not cause 
sample breakthrough.
<p>
A column was packed with Oasis HLB sorbent (2.1 &times; 20 mm with a 5 &micro;m
particle) and samples were loaded.  Different oligo types were loaded with
different sizes and washing, detritylation, and eolution conditions were optimized.
Post-loading washing conditions controlled selectivity:  the following are 
ratios of 0.1 M TEAA pH 8:MeCN mixes:  90:10 for phosphodiester &lt; 12-mer, 
86:14 for phosphodiester between 12- and 30-mer, 85:15 for phosphorothioate
&lt; 12-mer, and 80:20 for phosphorothioate from 12- to 30-mer.  1.0 ml is used
in the wash.  After the selected wash is done, then 1 ml of 2% TFA in water is
passed for detritylation, and then 1 ml of a 90:10 mix of 0.36 M TEAA pH 11.3:MeCN
to elute.
<p>
Experiments show that on-column detritylation leads to precipitation of the 
oligo within the matrix, with loss of recovery and degradation of column.
Use of 100% MeCN to elute is not recommended since it does not enhance DNA 
solubility.  Alkalinization of mobile phase leads to better recovery (elution), 
about 80% of the load, and adding 10% MeCN produced a 100% recovery.  Loading
conditions should be with TEAA at pH 8 rather than use of high pH conditions
with ammonium hydroxide present.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
G Cai, J Lian, SS Shapiro, DA Beacham
 </span>
 <span class="year">
(2000)
 </span>
 <span class="jname">
Meth. Cell Sci.
 </span>
 <span class="jvol">
22,
 </span>
 <span class="pages">
107-114.
 </span>
<p class="reftitl">
Evaluation of Endothelial Cell Migration with a Novel <i>In Vitro</i> Assay 
System
<p class="Para1">
HUVEC are isolated from umbilical veins and cultured.  The cells are pre-treated 
with 50 U/ml TNF&alpha; (sp. act of 1.6&nbsp;&times;&nbsp;10<sup>6</sup> U/nmol) 
in complete medium for 17 h before the migration assay.  The endothelial cells 
are dissociated by trypsin/EDTA for 30 s one day after reaching confluence, spun 
300 <i>g</i> for 5 min, washed 2 &times; in M199.  Cells are resuspended in 
complete medium (M199, 15% FBS, 100 &micro;g/ml sodium heparin, 100 &micro;g/ml ECGF) 
for the migration assay.  The ECGF is a crude prep from bovine hypothalamus (see 
text for description).  HUVEC migration is measured on von Willebrand's Factor 
or fibronectin bound to type I collagen (vWF/collagen or FN/collagen). Acidic 
bovine type I collagen (Vitrogen 100) in 12 mM HCl is neutralized with equal
volume of 12 mM NaOH, adjusted to 1.5 mg/ml by adding sterile PBS, on ice at 
4&deg;.  50 &micro;l is then spread using glass pipet spreader on 35 mm dish. 
Polymerization is complete in 60 min placed in 37&deg; incubator (no 
CO<sub>2</sub>).  A 25 &micro;l solution of 0.1 mg/ml vWF or FN in 0.1% BSA/PBS is 
pipetted evenly atop the collagen gel and allowed to perfuse for 60 min at 
37&deg;.  Excess is removed from the dish afterward. Dishes are allowed to 
air-dry in hood for 60 min.  Dishes are washed twice with sterile nanopure 
water, air-dried again for 30 min.
<p>
1 ml mineral oil is added to each well/dish.  Then a 10 &micro;l aliquot of 
endothelial cells in complete medium at 1.5 million/ml is pipetted through the 
oil on to the matrix surface.  The light mineral oil must be biologically inert, 
hydrophobic, consisting of saturated aliphatic and alicyclic hydrocarbons, with 
a density of 0.84 g/ml.  The cells under oil are incubated 37&deg; in 5% 
CO<sub>2</sub> for 1 hour.  Oil is then aspirated.  The HUVEC should form a 
circle radiating from the center of the well where they were spotted.  After 
washing twice in complete medium, cells are incubated in medium an additional 15 
min to allow all oil to float free.  Cells are washed twice again in medium. 
Then cells are incubated in a serum-free medium containing 20 ng/ml bFGF and 10 
ng/ml EGF. Migration is permitted for 24 h.
<p>
The optimal cell density was 1.5 million HUVEC/ml to distribute the cells
on the well.  Higher density resulted in multiple layers, and lower
densities left subconfluence.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
VG Kolupaeva, TV Pestov, CUT Hellen
 </span>
 <span class="year">
(2000)
 </span>
 <span class="jname">
J. Virol.
 </span>
 <span class="jvol">
74,
 </span>
 <span class="pages">
6242-6250.
 </span>
<p class="reftitl">
An Enyzmatic Footprinting Analysis of the Interaction of 40S Ribosomal
Subunits with the Internal Ribosomal Entry Site of Hepatitis C Virus
<p class="Para1">
HCV is a single-stranded (positive strand) RNA virus 9.6 kb with a
large orf and a 342 b long 5' non-translated region (5' NTR).
Interestingly, the coding region shows considerable sequency heterogeneity
while the 5'-NTR is strongly conserved.  Translation in HCV is initiated
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Johnson JE, Chiu W
 </span>
 <span class="year">
(2000)
 </span>
 <span class="jname">
Curr. Opin. Struct. Biol.
 </span>
 <span class="jvol">
10,
 </span>
 <span class="pages">
299-235.
 </span>
<p class="reftitl">
Structures of Virus and Virus-Like Particles
<p class="Para1">
Viruses are being studied by a combination of techniques, including
electron cryomicroscopy (cryo-EM), X-ray crystallography, and biochemical
techniques.
<p>
The most common folding motif in capsid subunits of newly solved structures
is the canonical virus &beta; sandwich.  New folds being demonstrated
either as indepedent subunits or separate domains fused to a &beta; sandwich.
The Norwalk virus is a ss RNA causing severe intestinal disorders, and
its subunit and particle organization is similar to a plant virus, the
tomato bushy stunt virus.  Norwalk's shell-forming domain is a &beta; sandwich,
and the protruding domain has the same fold as elongation factor Tu,
the first time a fold has been seen in a particle.
<p>
Hepatitis B is a
DNA virus with complex replication cycle involving an RNA intermediate
and reverse transcription.  The mature particle is enveloped, and the core
is assembled using a single subunit.  The core structure was established
to 7.5-9 &Aring; resolution by electron image reconstructure and at
3.5 &Aring; by X-ray crystallography.  Tracing the 149-residue subunit chain
could be done by cryo-EM, and the data agreed with crystal structure.
The fold in this subunit is made entirely of &alpha;-helix.
<p>
Bluetongue virus (BTV) is a ds RNA virus causing livestock disease.
The cores are icosahedral disassembly intermediates, formed by a <i>T</i>=13
outer shell and a 120-subunit inner shell.  The outer shell protein
structure was solved by examination of the recombinant protein and contains
an inner helical domain and a outer &beta; sandwich domain.  A 100 kDa
protein is the single subunit forming the inner shell packaging the ds RNA
and the replication enzyme set.  The inner shell subunit protein revealed
a mostly &alpha;-helical fold and has flexibility in tertiary structure
to create a packing arrangement inside the icosahedral shell.
<p>
Cricket paralysis virus (CRPV) and tobacco ringspot virus (TRSV)
are picorna-like
insect and plant viruses whose crystal structures show 8-stranded
&beta;-sandwich fold in each of the three domains in VP2, VP3, and VP1.
CRPV shows a novel domain swap in VP2 subunits forming part of the 2-fold
joint and change in gene order for small internal VP4 subunit.  TRSV
subnit has three &beta;-sandwich domains looking like beads on a string
and there is no proteolysis in polyprotein or particule maturation.
<p>
Viral genome packaging is the subject of many hypotheses.  These vary from
the completely controlled to the completely uncontrolled.  They include
models in which the shell is built around the core or the core is
inserted into a preformed shell.  The  data from studies generally favors
models involving docking proteins with ATPase activities.  Cryo-EM reveals
a transcriptional enzyme complex associated with RNA arranged in concentric
layers seen at all five-fold locations in low-resolution structures.
T7 ds DNA phase is packaged as an ordered coil, and this is seen for the
152 kb genome for HSV type 1 as well.
<p>
To determine the mix of appropriate proteins required for reconstitution
of virus or viral capsid, proteins are expressed in a baculovirus vector.
Separate genes are expressed in different baculovirus constructs which
are used to infect the same insect cell.  Particle formation is then
studied.  With herpesvirus, this approach was used to find 4 of 6
procapsid gene products that yielded an icosahedral particle of smaller size.
<p>
Several revelations have been made about particle assembly.  The BTV
inner shell is formed by ds RNA surrounded by a <i>T</i> = 1 120-subunit
icosahedral capsid.  The nuceloprotein is covered by a second protein layer
arranged with <i>T</i> = 13 icosahedral quasi-symmetry.  Between the layers
there is very little specific interaction:  only 13 points of contact, and
the chemistry of the outer surface of the inner layer is apparently designed
to be very nonspecific with the inner surface of the outer layer.  The
outer shell is probably only organized on the icosahedral 3-fold axis.
The rest of the shell is then assembled from the initial outer shell
assembly through specific interactions, and curvature dictated by the
<i>T</i> = 1 inner shell.
<p>
Scaffolding structures are usually temporary assembly components not found
in the mature form of the particle.  The &phi;X-174 ss DNA phage intially
forms a procapsid with scaffold and capsid proteins.  X-ray structures of
the procapsid show an <i>external</i> scaffold of 240 copies formed around
60 copies of the major capsid protein.  The helical scaffold protein
makes four different types of protein interactions with the capsid protein.
Phage P22 actually recycles its scaffolding proteins
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Glennon TM, Villa J, Warshel A
 </span>
 <span class="year">
(2000)
 </span>
 <span class="jname">
Biochemistry
 </span>
 <span class="jvol">
39,
 </span>
 <span class="pages">
9641-9651.
 </span>
<p class="reftitl">
How Does GAP Catalyze the GTPase Reaction of Ras?: A Computer Simulation
Study
<p class="Para1">
p21<sup>ras</sup>, which plays a key role in signal transduction pathways
controlling cell proliferation, is a GTP-binding protein which cycle between
an ON (GTP-bound) and OFF (GDP-bound) state.  Guanine-exchange factors
(GEFs) cause the exchange between GTP and GDP and thus control the ON and
OFF state.  GEFs become activated in order to cause this exchange.
The GTP-bound protein interacts with its effector to transmit the signal
downstream to the next component in the signaling cascade.  GTP-binding
proteins have an intrinsic GTPase activity which acts at different rates
to determine the timing of how long the signal lasts.  Mutations can
affect this timing often slowing the reaction rate and leading to
oncogenic Ras-proteins producing a continual growth signal.  30% of human
tumors are attributed to mutations in positions 12, 13, and 61.
<p>
Guanine exchange and GTPase activities are controlled by accessory proteins.
p21<sup>ras</sup> GTPase activity can be increased 10<sup>5</sup> times
in the presence of GTPase-activating proteins (GAPs).  How GAPs work is
not fully understood.  Two possibilities are that the GTPase transition
state is stabilized by interaction with GAP residues or the interaction of
GAP <q>pushes</q> Ras to the conformation stabilizing the transition state.
Because mutations of Arg789 of GAP reduce the rate constant of Ras-GAP
complex by about 2000-fold, the first option seems more likely: an
<q>arginine finger</q> apparently stabilizes the negatively charged transition
state.  The second option is the <b>isomerization</b> hypothesis where GAP
pre-organizes Ras allosterically into the catalytic configuration.
<p>
The Ras-GAP (GAP and p21<sup>ras</sup>) complex was studied in the presence
of an AlF<sub>3</sub> transition state analogue and its structure solved,
followed by structure of a small G protein and its GAP.  The structural
data permit computer simulation approaches.  This paper applies the
<b>Empirical Valence Bond (EVB)</b> method.
<p>
The simplified scheme of the ab initio potential surface of the GTPase
(GTP hydrolysis) reaction is shown in a energy hysteresis diagram
in which thin lines are drawn to show the progress through computed
transition state and minima energy levels; thick dashed lines represent
the <q>generic</q> pathway or smoothing of the hysteresis loop.
The concave-up curve starts at the bottom with H<sub>2</sub>O associating
with the terminal (gamma) PO<sub>4</sub><sup>2&minus;</sup> of GTP
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Liu G, Katherine JD, Excoffon KJDA, Wilson JE, McManus BM, Rogers QR, Miao L,
Kastelein JJP, Lewis MES, Hayden MR
 </span>
 <span class="year">
 (2000)
 </span>
 <span class="jname">
Hum. Gene Ther.
 </span>
 <span class="jvol">
11,
 </span>
 <span class="pages">
21-32.
 </span>
<p class="reftitl">
Phenotypic Correction of Feline Lipoprotein Lipase Deficiency by Adenoviral Gene Transfer.
<p class="Para1">
Two wild-type and six LPL-deficient male cats from a New Zealand colony
were fed Whiskas diet (protein: 57.5%, fat: 24.3%, ash: 18.2%).  Genotyping
was done according to Ginzinger et al (1996).
<p>
A vector of adenovirus 5 (Ad5) with E1/E3 removed by deletion and containing
human LPL cDNA under CMV promoter had been previously constructed (Liu et al
1997).  Plaques positive for an immunochemical and activity screen were
selected.  The clone with highest activity was selected after single round
of plaque purification.  Expression in 293 cells growing in 15 cm dishes was
done, along with a control alkaline phosphatase reporter also under CMV
control.  Recombinant construct was purified twice on CsCl ultracentrifugation.
Storage of virus was at -80&deg; after dialysis against HBSS with 10% glycerol
at 4&deg;.  Viral titering indicates 1:100 PFU/particle ratio.  Infected COS
cells show no cytopathic effect, which unlike 293 cells, do not complement
E1/E3 replication incompetence.
<p>
Cats were fasted overnight, put under isoflurane anesthesia, a catheter inserted
in cepahlic vein attached to i.v. saline administration system.  LPL-Ad-containing
virus at 8 billion PFU/kg was diluted with DMEM and infused over 10 min period.
100 &micro;l of same viral preparation was injected i.v. into mice also:
LPL and ALP transgenic expression in mice was confirmed after 2 d postinjection.
<p>
In a first experiment, cats were injected directly with active bovine LpL
i.v. at 9000 U/kg, producing 200 mU/ml which is the average activity in wildtype
cats, assuming plasma vol of 4.5% of body weight.  Plasma TGs drop precipitously
relative to the maximum level in LpL-deficient cats, whereas they do not drop
at all in wildtype cat control.  Injection of heparin in the control cat
caused a release of LpL activity with reduction of plasma TG by 80% in 10 min.
In deficient cats, the bovine LpL was lost in circulation with a half time
of less than 10 min, and less than 10% of the initial activity was seen after
1 h.  The nature of the loss of enzyme is unknown.
<p>
Adenovirus with LpL and AP (alkaline phosphatase) reporter control was injected
into mice in a separate experiment and there was an increase in LpL activity
and immunoreactive mass and specific activity, and a decrease in plasma TG.
Two wildtype cats were given 8 billion PFU/kg and after 2 d, the postheparin
plasma LpL activity increased more than 3-fold from baseline.  Activity was
&gt;2400 mU/ml after 7 d.
<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Stuyver L, De Gendt S, Van Geyt C, Zoulim F, Fried M, Schinazi RF, Rossau R
 </span>
 <span class="year">
(2000)
 </span>
 <span class="jname">
J. Gen. Virol.
 </span>
 <span class="jvol">
81,
 </span>
 <span class="pages">
67-74.
 </span>
<p class="reftitl">
A new genotype of hepatitis B virus: Complete genome and phylogenetic 
relatedness
<p class="Para1">

<!-- ============================================================= -->
<p class="tags">

<p class="ref">
 <span class="authors">
Seeger C, Mason W
 </span>
 <span class="year">
(2000)
 </span>
 <span class="jname">
Microbiol. and Molec. Biol. Rev.
 </span>
 <span class="jvol">
64,
 </span>
 <span class="pages">
51-58.
 </span>
<p class="reftitl">
Hepatitis B Virus Biology
<p class="Para1">
<b>REVIEW</b>.  HBV causes transient (acute) infection, of which 5 in 1000 cases
can result in fatal, fulminant hepatitis.  It can also result in chronic infections
of which 25% progress to untreatable liver cancer.  HBV-caused liver cancer
deaths exceed 1 million per year globally.  A vaccine has been available for 
20 years and programs to treat especially endemic population, such as China
with 120 million carriers (10% of population), are in progress.  Treatment
with IF-&alpha; is not effective in 50% of HBV carriers, and the effectiveness
of using alternative nucleoside analogs has not been fully studied.  Total 
carriers worldwide is thought to be 350 million, or 6 in 100 people.
<p>
The virus targets hepatocytes without causing cell death.  Virus is budded
from the cell to ensure that all hepatocytes.  Cells normally are long-lived
with half-life of 6-12 months and long cell cycle.  In absence of immune 
response, this is ideal condition for a virus that wants to infect everywhere.
The emergence of liver disease is actually believed to be because of the
immune response, and the immune system's attempt to kill all infected cells,
which effectively means the liver.
<p>
HBV is the prototype for the <i>Hepadnaviridae</i> virus family.  Similar 
viruses have been identified in woodchucks and ground squirrels, with 
about 70% sequence homology to HBV but not pathogenic to humans or primates.
HBV can infect other primates.  These mammalian viruses are classed under
the <i>Hepadnaviridae</i> into the genus <i>Orthohepadnavirus</i>.  While other
genera have similar genome organization, sequence homology is much lower.
These include <i>Avihepadnavirus</i> with hosts such as ducks, geese; used
as model for studying its biology of replication.  An important finding in the
avian virus was the development of immune tolerance if perinatal transmission
occurred during embryonic development.  Drug development studies were done 
studying woodchuck and duck virus, as well as a transgenic mouse model.
<p>
<b>The Liver</b>.  It has primary roles in metabolism, blood homeostatis,
detoxification of absorbed xenochemicals, and immunity.  70% of the cells 
forming the liver mass are hepatocytes.  The rest is bile duct epithelium
and Kupffer cells (macrophages).  Duct epithelial cells and hepatocytes are
closely related (believed to be derived from same progenitor), and the epithelium
may de-differentiate and form hepatocytes in injury.  The only target cell 
type of Hepadnaviridae may be hepatocytes; it is possible that duct epithelium
and subsets of cells in pancreas, kidneys, and lymphoid system can be infected
as well.
<p>
Functional units of liver are <b>lobules</b> (see figure below).  The understanding
is that blood enters the lobule by portal vein and hepatic artery and then
distributed by smaller vessels into sinusoidal spaces, which are channeled
by plates of hepatocytes.  In mammals the plates are one cell thick; in birds
they are two cells thick.  Blood flows through fenestrated epithelial cells
and by permanently located macrophages (Kupffer cells).  It eventually collects
in central veins and out the liver.  The average distance between portal vein
and central vein is about 20-30 hepatocytes.  Heme catabolism leads to bile
formation and secretion into canaliculi, which are small channels formed at
junctions of hepatocytes; its flow is opposite to flow of blood, passing throug
the canal of Hering into bile ducts.  These then flow to larger ducts, then
into gall bladder and intestine.
<div class="report-figure">
<img src="images/hbv/liver-lobule-anatomy.gif"
  alt="liver lobule microanatomy">
<p class="figure-caption">
<span class="figure-caption-header">Anatomy of the liver lobule</span>. The 
major cell types within the liver are illustrated. Blood enters at the portal 
tracts (encompassing the portal vein, hepatic artery, and bile ductules), 
distributes laterally through smaller veins to the sinusoidal spaces, and flows 
toward the central vein. The sinusoidal spaces are lined with endothelial cells 
and fixed macrophages (Kupffer cells). The endothelial cells are fenestrated, 
allowing free diffusion between the blood and hepatocytes. [Figure caption
text verbatim.]
</div>
<p>
Hepatocytes can proliferate in liver injury.  Normally their cell cycle in 
the differentiated state is considered quite long.  With 70% of the liver 
surgically removed, every hepatocyte can be activated to re-enter the cell cycle
at least once and restore the liver mass.  Replacement of hepatocytes may 
occur by progenitors thought to reside in regions of the portal tracts, possibly
the cells in the bile ducts and canals of Hering being identical or associated.
They may be induced to become oval cells, and then into hepatocytes.  
Understanding of dynamics of liver tissue regeneration biology is crucial
if the means by which HBV is cleared is by destruction of all infected cells
by the immune system followed by replacement from progenitor populations, or
if the virus is removed from infection cells themselves (de-infection).
<p>
<b>Infection</b>.  This process may not be so much receptor-dependent but instead
whether the virus can successfully control cell host biology and achieve its
own genome replication and reproduction.  It is thought that viruses can infect
lymphocytes since polynucleotide sequences have been found in these cells, but are
they the function of phagocytosis instead of infection;  others have found what
they believe are viral RNA transcribed from cccDNA template.  Nonetheless, 
lymphocytes isolated from infected woodchucks and stimulated with mitogen appear
to show replication.
<p>
<b>cccDNA Formation.</b>  The figures on life cycle and DNA replication below 
illustrate the first event in replication, which is conversion of rcDNA (relaxed 
circular DNA) into cccDNA (covalently closed circular DNA).  The rcDNA is the 
form inside the virion, is about 3 kb in size and partially double-stranded.
The cccDNA form is required for transcription of viral mRNAs, and is detected
within 24 h of infection.  The rcDNA&rarr;cccDNA mechanism is not understood, 
although a viral DNA polymerase may be involved since inhibitors prevent initiation
of infection.  The partial double-stranded genome is because of an incomplete
plus strand, and while the minus strand is complete, it is broken, and the 5&prime;
end of the break is covalently attached to a viral reverse transcriptase.  In
the cell, the RTase is detached, and the minus strand undergoes a ligation
repair.  Questions:  does the RTase itself catalyze its removal, and is the
bond breaking hydrolysis of DNA-protein, or does a cellular endonuclease cleave
the viral DNA close to 5&prime; end?  Do the events occur in cytoplasm or in
nucleus?  With respect to directing nuclear import, some studies reveal that
phosphorylated core proteins can bind to nuclear core complex.
<div class="report-figure">
<img src="images/hbv/hepadnavirus-life-cycle.gif"
  alt="Life cycle of the Hepadnavirus">
<p class="figure-caption">
<span class="figure-caption-header">Hepadnavirus life cycle</span>. 
The details of the replication cycle are discussed in the text. Briefly, during 
initiation of infection the viral rcDNA (or linear DNA) genome, with a protein 
(the viral reverse transcriptase) attached to the 5&prime; end of the minus 
strand and a short RNA attached to the 5&prime; end of the plus strand, is 
converted into cccDNA. During this process, both the protein and the RNA are 
removed. The cccDNA serves as the template for transcription of viral mRNAs (see 
Fig. 4). One of these, the pregenome, serves as the mRNA for the synthesis of 
core protein (nucleocapsid subunit) and the viral reverse transcriptase. The 
reverse transcriptase binds to the 5&prime; end of its own mRNA template, and 
the complex is then packaged into nucleocapsids, where viral DNA synthesis 
occurs (see Fig. 3). Once partially double-stranded DNA has been produced, 
nucleocapsids can undergo a maturation event that facilitates their acquisition 
of an outer envelope via budding into the ER. These nucleocapsids can also 
migrate to the nucleus to increase the copy number of cccDNA. Since cccDNA does 
not undergo semiconservative replication, all cccDNA is derived from viral DNA 
made in the cytoplasm via the reverse transcription pathway (217). Accumulation 
of viral envelope proteins prevents excessive cccDNA formation, which can be 
toxic to hepatocytes (200, 201). [Figure caption text verbatim.]
</div>

<p>
<b>Gene Expression</b>.  With cccDNA in the nucleus (where it is only found), 
its transcription produces pregenomic RNA (pgRNA).  The pgRNA will be a template 
for reverse transcription to virion DNA and also be processed into three 
subgenomic mRNAs (in avihepadnavirus, only two).   One mRNA encodes envelope 
proteins, while a second encodes the X protein, found only in mammalian viruses. 
Promoter and activity studied in cell culture systems and liver extracts has 
revealed four promoters and two enhancer elements, En1 and En2, in the genome. 
Each gene has one or more promoters, which are regulated by one or both of the 
enhancer elements, upstream of the promoter for the Core Ag.  Transcription
factors are those primarily hepatocyte-specific.  Results have been studied
in DHBV and WHV indicating importance of hepatocyte-specific transcription
factors.
<p>
Transcription proceeds past the polyadenylation signal for a full-size
terminally redundnant RNA.  This is regulated by a PET element (positive
effector of transcription) located between transcription initiation site
and poly(A) signal.  PET causes transcription through signal on 1st pass
but not 2nd pass.  If PET is deleted, transcription of smaller mRNAs encoding
envelope proteins in increased.  A signal NET (negative effector of transcription)
is needed to cause termination, and suppressed by PET.  If NET is deleted,
longer transcripts accumulate the cccDNA template is continuously transcribed.
<p>
Splicing has been observed in transcripts although there is no evidence that
splicing is required for gene expression.  Spliced forms of pregenome have been 
detected in DHBV-infected cells, but mutations to the splice site do not inhibit
virus release from transfected cell line, but release from primary hepatocyte
cultures is blocked.

<div class="report-figure">
<img src="images/hbv/hepadnavirus-DNA-replication.gif"
  alt="Replication of Hepadnavirus DNA">
<p class="figure-caption">
<span class="figure-caption-header">Hepadnavirus DNA Replication</span>. 
(A) Virus particles contain predominantly rcDNA with a complete minus strand and 
a partially synthesized plus strand. A small amount of linear DNA, formed as a 
result of aberrant plus-strand priming (in situ priming) is also found in the 
virus. (B) During initiation of infection, both forms of virion DNA are 
converted to a cccDNA; however, conversion of the linear form involves 
illegitimate recombination, which can lead to subsequent defects in the ability 
of the virus to replicate. cccDNA serves as template for the transcription of 
the pregenome. The reverse transcriptase binds to the epsilon stem-loop 
structure near the 5&prime; end of its own mRNA to facilitate packaging into 
nucleocapsids and initiation of reverse transcription by a protein-priming 
mechanism, utilizing a tyrosine located near the amino terminus of the reverse 
transcriptase itself. (C) Following the synthesis of 4 bases, the polymerase 
translocates to the 3&prime; end of the RNA template, where the 4 bases can 
anneal with complementary sequences. During the elongation of the minus strand 
to the 5&prime; end of the pregenome, all but the 5&prime; 17 to 18 bases of the 
pregenome, including the CAP and DR1, are degraded by the viral RNase H. (D) The 
remaining fragment then serves as the primer for plus-strand synthesis, usually 
following its translocation to DR2, with which it can hybridize because of the 
sequence identity between DR1 and DR2. A third translocation occurs when the 
plus strand reaches the 5&prime; end of the minus strand, to circularize the 
molecule and allow continued plus-strand elongation. This translocation may be 
facilitated by the short (~9-base) terminal redundancy on the minus strand. The 
plus strand is not completed prior to virion release; a repair reaction to 
produce a fully double stranded DNA occurs during initiation of a subsequent 
round of infection. (E) A fraction of the virions have linear genomes because 
priming of plus-strand DNA synthesis occurs in the absence of primer 
translocation. [Figure caption text verbatim.]
</div>
<p>
Tbe pregenome also serves as the mRNA for core protein and the polymerase.
The polymerase initiates from AUG start in distal part of core gene, but not
in same reading frame as the core.  The reading of the polymerase open reading
frame is less efficient than for the core open reading frame.  Given that
the core is formed from 240 subunits while only one, maybe two, polymerase
proteins become part of the core, it is thus assumed that 200 to 300 translations
of core must occur before allowance of translation of polymerase.  With the
synthesis of polymerase, it binds to the 5&prime; end of its own mRNA, and
this leads to reverse transcription and packaging, stopping any further 
translation of pgRNA.
<p>
<b>Proteins</b>.  The core and polymerase are required for DNA replication:
one leads the rcDNA to the nucleus, while the other eventually creates the
rcDNA for the virion.  Envelope proteins are required for the nucleocapsid
virion.  The hepatitis X protein (HBx) and e-antigen (HBeAg) have no known
function, although in vivo function cannot occur without X (transfected cells
can replicate virus though); HBeAg is  not required for in vivo infection.
Avihepadnavirus lacks an X gene.  HBeAg might be involved in depleting
Th1 helper cells: this suppresses a cytotoxic T lymphocyte (CTL) response.
<p>
The core polypeptide (HBcAg) forms a dimer through disulfide bonds, and 
this dimer then associates with for a total of 120 such dimers to form
and icosahedral subviral core particle with diameter = 36 nm and triangulation
number (<i>T</i>) = 4.  If cores have truncated C-termini, smaller shells are
formed from 90 dimer subunits and <i>T</i> = 3.  Cryoelectron microscopy
provided resolutin to 7.4 &Aring; and X-ray crystallography to 3.3 &Aring;.
Each core polypeptide has four &alpha;-helices and no &beta;-sheet.  Two
regions are involved in dimerization and the interfacing to assemble
dimers noncovalently.  Interactions with envelope proteins have been indicated
as well.
<p>
The polymerase consists of two major domains with intervening spacer region.
The amino terminal, called terminal protein [TP] domain, has a role in 
pgRNA packaging and priming minus strand.  The C-terminal has the active
site with reverse transcriptase function.  It has an RNase H activity.
The function of polymerase is dependent on host cell components, such as a 
molecular chaperones which are involved in protein priming and the polymerase
activity itself.
<p>
The envelope proteins are designated S, M, and L (small, medium, and large).
S is 226 amino acids long and refers to a name of domain (S domain). M and L
include the S domain translated sequence, but are longer at the amino 
terminal end, because the start codon is upstream of the S domain start.
Thus M start is S - 165 nt (in frame), with 55 extra residues; while L
is S - 489 nt (in frame) with 163 extra residues.  These add additional 
domains: the one for M is called preS2, and the domain added to L is 
preS1.  These are all glycosylated, type II transmembrane proteins which
form multimers.  Disulfide form from S domain Cys residues.  The mature virion
(Dane particle as seen in EM) is 42 nm diameter, has L and M components 
in roughly equal numbers, being about 30% of envelope protein content.
The S domain only protein, as well as larger envelope proteins, will form
filamentous and spherical particles which are secreted from infected cells
in over 100-fold excess over virions.  These antigens can rise to several 
hundred micrograms per milliliter in blood.  In acute (transient) infections
the particles and antibodies likely create immune complex syndromes.
<p>
The preS1 domain could be the substrate for the viral receptor. Epitopes
of preS1 on the surface antigen can induce virus-neutralizing Abs, and they
alter host rnage when recombination occurs.  The preS1 domain is a ligand
for core particles during envelope assembly, and preS1 epitopes on cytosolic
side of membrane must be displayed.  A fraction of L polypeptides must maintain
preS1 domain in interrio and remaining L polypeptides undergo conformational shift
to transfer receptor binding preS1 region across lipid bilayer to the exterior
of the virion.  Transfer appears to occur as viruses mature during passage through
secretory pathway.  M polypeptde function not understood, and preS2 domain is
apparently not essential to virus infection of hepatocyte cultures.  L protein
is modified both by glycosylation and N-terminal myristylation.  Myristylation
not required for virion assembly but rather is required for infectivity.
<p>
Avihepadnaviruses encode only two envelope proteins analogous to S and L, 
and it is not evidenty they are glycosylated.  Pre-S domain in phosphorylated
and N-myristylated.  Phosphorylation is unknown to play any role in envelope
assembly or infectivity.
<p>
The fourth gene X is unique to orthohepadnaviruses.  X is required for in vivo
virus replication but has no role in vitro in transfected cell cultures.  Anti-X
antibodies do form.  Cell culture studies indicate X may activate transcription
of host genes, including MHC genes and c-<i>myc</i>, in addition to activating
viral genes.  It is not a DNA-binding protein and not a typical transactivator.
It could possibly interact with an upstream messenger such cAMP-RE (CRE)-binding
protein and ATF-2, activating NF-&kappa;B, and contacting basal transcription 
factors.  Besides transactivation of multiple promoters, X is found to stimulate
signal transduction and binding in various degrees to p53, proteasome subunits,
and UV-damaged DNA binding protein.  It is at least known that X-defective virus
cannot initiate infection in vivo.
<div class="report-figure">
<img src="images/hbv/hbv-transcriptional-translational-map.gif"
  alt="Replication of Hepadnavirus DNA">
<p class="figure-caption">
<span class="figure-caption-header">Transcriptional and translational map of 
HBV</span>. The figure shows the physical map of the HBV genome. The inner 
circle depicts the rcDNA with the reverse transcriptase attached to the 5&prime; 
end of the complete minus-strand DNA (solid sphere) and a capped RNA oligomer 
attached to the 5&prime; end of the incomplete plus-strand DNA (solid half 
sphere). The positions of the direct repeats, DR1 and DR2, as well as the 
positions of the two enhancers, EN1 and EN2, are indicated. The outer circle 
depicts the three major viral RNAs, the core (C) or pgRNA, the preS (L) mRNA, 
and the S mRNA. The common 3&prime; ends of the three mRNAs are indicated by the 
letters A. Not shown in the figure is the putative X mRNA that spans the X 
coding region and terminates at the site indicated for the other three mRNAs. 
The four protein-coding regions are shown between the inner and outer circles. 
They include the precore (PC) and core genes, the polymerase gene, and the X 
gene. The envelope genes preS1 (L), preS2 (M), and surface (S) overlap with 
the polymerase open reading frame.
</div>
<p>
In the assembly of the virion, the polymerase (reverse transcriptase) is 
required for pgRNA packaging.  An RNA stem-loop at the 5&prime; end of pgRNA, 
called epsilon, forms and this triggers sequestration of viral RNA and 
polymerase into core particles.  Other factors in packaging include molecular 
chaperone complex hsp90, which stabilizes a transient conformation of polymerase 
facilitating binding to epsilon on pgRNA.  The ansamycin geldanamycin inhibits 
hsp90, and it also inhibits RNA packaging, and thus replication of the DBHV and 
HBV.
<p>
Epsilon also plays a role in DNA synthesis:  the polymerase serves as a protein 
primer.  This is provided by a tyrosine hydroxyl near the amino terminus of the 
polymerase, with phosphodiester linkage formation with dGMP.  Template for 
reaction is located in bulge region of epsilon which divides upper and lower 
stem regions. After 4 nucleotides are synthesized for the primer, priming stops. 
The nascent dNA is transferred to 3&prime; end of pgRNA, where base pair occurs 
with complemetary sequences (see the figure above).  Mechanism of translocation 
of the small strand to other end is not known.
<p>
Plus strand priming also involves translocation like minus strand.  The RNA 
oligomer derived from 5' end recombines with internal acceptor sit on 
minus-strand DNA.
<!-- ============================================================= -->
<p class="tags">
ELISA development
<p class="ref" id="elisa-devel">
 <span class="authors">
Horbeck P, Winston SE, Fuller SA
 </span>
 <span class="year">
(2000)
 </span>
 <span class="jname">
Curr. Prot. Molec. Biol.
 </span>
 <span class="jvol">
Unit 11.2
 </span>
 <span class="pages">
11.2.1-11.2.22
 </span>
<p class="reftitl">
Enzyme-Linked Immunosorbent Assays (ELISA)
<p class="Para1">
<b>LABORATORY MANUAL</b>.
This describes six different ELISA types:
<table style="font-size:83%;">
<tr><th>Protocol <th>Use  <th>Reagents   <th>Comments
<tr><td>indirect 
  <td><ul style="margin:0;"><li>antibody determination / screening
     <li>epitope mapping</ul>
  <td><ul style="margin:0;"><li>antigen, pure or semi-pure
      <li>test solution containing antibody
      <li>enzyme conjugate-Ig of immunized species</ul>
   <td><ul style="margin:0;">
    <li>use of pre-existing Igs not required
    <li>large amounts of Ag needed </ul>
<tr><td>direct competitive
  <td><ul style="margin:0;"><li>detection of soluble antigen
     <li>antigen screening</ul>
  <td><ul style="margin:0;"><li>antibody, pure or semi-pure
      <li>test solution containing antigen
      <li>enzyme conjugate-Ig specific for antigen</ul>
   <td><ul style="margin:0;">
    <li>two-step, rapid assay
    <li>able to measure antigenic cross-reactivity</ul>
<tr><td>classic capture or sandwich
  <td>same as for direct competitive
  <td><ul style="margin:0;"><li>capture antibody, pure or semi-pure
      <li>test solution containing antigen
      <li>enzyme conjugate-Ig specific for antigen</ul>
   <td><ul style="margin:0;">
    <li>most sensitive of antigen assays
    <li>needs relatively large amounts of capture antibody</ul>
<tr><td>double antibody sandwich
  <td><ul style="margin:0;"><li>antibody screening<li>epitope mapping</ul>
  <td><ul style="margin:0;"><li>capture antibody, specific for Ig of immunized species
      <li>test solution containing antigen
      <li>enzyme conjugate-Ig specific for antigen</ul>
   <td><ul style="margin:0;">
    <li>purified antigen not required
    <li>a 5-step assay</ul>
<tr><td>direct cellular
  <td>a screen and measure of antigen expression in cells
  <td><ul style="margin:0;"><li>cells expressing Ag of interest
      <li>enzyme conjugate-Ig specific for cellular antigen</ul>
   <td><ul style="margin:0;">
    <li>sensitive assay for bulk screening
    <li>insensitive to heterogeneicy of expressed in mixed population of cells</ul>
<tr><td>indirect cellular
  <td>a screen and measure of antibodies against cellular antigen
  <td><ul style="margin:0;"><li>cells used for immunizing
     <li>test solution containing antibody 
     <li>enzyme conjugate-Ig that binds Ig of immunized species</ul>
   <td>may not detect antibodies specific for cellular Ags expressed at low density
</table>
<p>
<b>Indirect ELISA</b>.  Useful for screening antisera or hybridoma supernatants
for specific antibodies.  ELISA plate wells are coated with antigen (1 mg
pure antigen can allow for coating from 80-800 microtiter plates).
<ul>
<li>Developing reagent: one of the following of
 <ul>
 <li>Protein A-alkaline phosphatase conjugate (Sigma P9650)
 <li>Protein G-alkaline phosphatase conjugate (Calbiochem 539304)
 <li>alkaline phosphatase conjugated to anti-F<sub>c</sub> of immunized species
 </ul>
<li>antigen solution
<li>PBS with 0.05% sodium azide (PBSN)
<li>borate-buffered saliene (BBS):  17 mM Na<sub>2</sub>B<sub>4</sub>O<sub>7</sub>
&bull; 10 H<sub>2</sub>O pH 8.5, 0.12 M sodium chloride
<li>blocking buffer:  BBS containing 0.05% Tween 20, 1 mM EDTA, 
0.25% BSA, 0.05% sodium azide, stored 4&deg;  (alternatives to BSA
include gelatin; 5% milk powder can be used but might nonspecifically interfere
with antibody binding).
<li>deionized or distilled water
<li>test antibody samples
<li>4-methylumbelliferyl phosphate (MUP) or p-nitrophenyl phosphate:
MUP solution is 0.2 mM 4-methylumbelliferyl phosphate (MUP, Sigam M8883),
50 mM sodium carbonate, 50 mM magnesium chloride, stored RT;
NPP solution is 3 mM <i>p</i>-nitrophenyl phosphate (NPP, Sigma 104-O),
50 mM sodium carbonate, 50 mM magnesium chloride, stored 4&deg;.
<li>Multichannel pipetter with tray and disposable pipet tips
<li>Immulon 2, Immulon 4, or equivalent plates (Dynatech 011-010-3450
and 011-010-3850)
<li>plastic squirt bottles
<li>microplate reader or spectrophotometer with 405 nm filter or 
 spectrofluorometer with 365 nm excitation/450 nm emission
</ul>
<ol>
<li>The optimal concentration of developing reagent (enzyme conjugate Ab)
is determined using criss-cross serial dilution protocol.  If all antibodies
binding antigen are to be detected, conjugates specific for Ig &kappa;
and &lambda; light chains should be used.  Protein A or G enzyme conjugates
might be used to screen monoclonal Abs.
<li>Criss-cross serial dilution should then be used to determined optimal
concentration for the coating reagent.  Final coating concentration is 
usually 0.2 to 10.0 &micro;g/ml, depending on purity.  Usually for a pure
antigen, &le; 2 &micro;g/ml is usual.  In generaly purity should be &gt; 3%.
Higher total protein concentrations will be necessary when the antigen is
only semipure.  Coating can be affected by pH, so varying this might help
to optimize too.
<li>With optimized coating concentration, dispense the proper volume
(50 &micro;l) into all wells of the microtiter plate.  Use tapping or
shaking to ensure the solution is properly distributed and there are no
bubbles.  Wrap in plastic wrap to see and incubate overnight at room
temperature or 2 h at 37&deg;.
<li>Rinse the plate three times with deionized or distilled water. Flick
the water out of the plate into the sink.
<li>Fill each well with blocking buffer as a stream from squirt bottle and
incube 30 min at room temperature.  While 0.05% Tween 20 may be sufficient to
block unblocked adsorbing sites, protein (usually 0.25% BSA) can be included
to bind. 
<li>Rinse three times with water filling of all wells, as used to rinse the
post-coating step.  Wrap in large paper tissue and flick face down onto 
several paper towels laying on the bench top
<li>Add antibody (50 &micro;l) samples which are diluted in blocking buffer
to each coated well, wrap in plastic wrap, and incubate for &ge; 2 h at room
temperature.  It should be noted that time to equilibrium binding is from 
5-10 h, although sufficient binding can occur in 1-2 h.  Note that multichannel
pipet tip re-use is possible by rinsing tips at least 5 times in blocking buffer,
and expelling fully on to paper towels.
<li>Rinse the plate as before in water, with full filling of wells.
<li>Fill wells with blocking buffer, vortex the plate, and incubate 10 min
at room temperature.
<li>Rinse three times with water, and after the final rinse, remove residual
liquid with tissue and towels as above.
<li>Add 50 &micro;l of developing reagent in blocking buffer using optimal
value determined.  Wrap in plastic and incubate &ge; 2 h at RT.
<li>Repeat the water, blocking buffer, and water washes as after the antibody 
sample incubation step.
<li>Add 75 &micro;l MUP or NPP substrate and leave 1 h at room temperature.
<li>For NPP visual inspection or reading in microplate reader can be done to
monitor color/fluorence changes.  Reaction can be stopped with 25 &micro;l
0.5 M NaOH.  For MUP, visual monitoring using long wavelength UV lamp or 
machinge monitoring using spectroflurometer can be done.  MUP is less subject
to spontaneous hydrolysis than NPP and 10-100 times faster.
</ol>
<p>
<b>Direct Competitive ELISA</b>.  Used to detect/screen or quantitate soluble
antigen and is useful when specific Ab and milligram amounts of purified or
semipurified Ag are available.  Plates are coated with Ag and the binding of
enzyme-conj Ab is inhibited by test solutions containing soluble Ag.  The signal
is inversely related to presence of Ag in the test solution: higher absorbance,
lower Ag concentration in test.  In addition an indirect assay format is possible
by substituting specific Ab for specific Ab-Enz conjugate.  This is followed by
incubation of a species- or isotype-specific conjugate as tertiary reactant.
<ul>
<li>specific antibody-alkaline phosphatase conjugate
<li>standard antigen solution
<li>test antigen solutions
<li>round- or cone-bottom well microtiter plates
</ul>
<ol>
<li>Use criss-cross serial dilution to determine Ab-APase conjugate and the
coating reagent optimal concentrations.  Prepare a 2&times; conjugate solution
by diluting conjugate in blocking buffer to twice its optimal concentration.
Final concentration is usually 25-500 ng/ml antibody.  3 ml will be needed for
each plate.
<li>Coat and block wells with each plate using proper volume (often 50 &micro;l)
using the steps in indirect protocol.
<li>Prepare six 4-fold serial dilutions (successive 1:3) of standard antigen in 
blocking buffer.  The range of concentrations should span the range of the 
ability of antigen to inhibit Ab binding to coating Ag.  An initial assay of six 
orders of magnitude from 1 &micro;M to 1 pM is typical. Note that the signal in 
inhibition of binding curves in competitive assays follows an <q>S</q> curve, 
and the region between 15 and 85% inhibition is the portion of the curve where 
small changes in concentration of inhibitor produce greatest changes in signal. 
Usually this is between 1 and 250 ng/ml inhibitor.
<li>Mix and incubate conjugate and inhibitor by added 75 &micro;l of 2&times;
conjugate (step 1) to each well of round- or cone-bottom plate followed by 75 &micro;l
of the inhibitor: either test antigen or standard antigen solution (step 3)
Mixing is achieved by pipetting up and down three times and incubating &ge;
30 min at RT
<li>Prepare uninhibited controls by mixing equal volumes of 2&times; conjugate
and blocking buffer.
<li>Transfer 50 &micro;l of conjugate + inhibitor (step 4)  or conjugate + buffer
(previous step) to antigen-coated plate (step 2) and incubate 2 h RT.
If doing replicates (duplicates), use adjacent columns.  Keep column 11
for uninhibited controls and column 12 for substrate without conjugate.
<li>Wash plate by filling three times with water, then fill with buffer
and vortex and let sit 10 min, then three times with water filling again.
<li>Develop with 75 &micro;l MUP or NPP and leave for &ge; 1 h at RT.
<li>To construct the standard Ag curve, plot absorbance/fluorescence against
log [Ag].  Use a curve fitter or interpolate by hand.
</ol>
<p>
<b>Sandwich ELISA</b>
<p>
<b>Double antibody sandwich ELISA</b>.


<b>Criss-Cross Serial Dilution</b>.  This is used to optimize all reagent
concentrations.  The coating reagent should be tested at four two-fold
serial dilutions (e.g. 10, 5, 2.5, 1.25 &micro;g/ml, or at a range that
is optinal for coating molecule and the type of ELISA plate.  The coating
protocol should be followed according to manufacturer.  After coating,
use a blocking solution.  The secondary reagent is basically the test
antibody or antigen within the buffer used for blocking.  With a starting
concentration of 200 ng/ml, four 4-fold dilutions are made with final
concentrations of 200, 50, 12.5, 3.125, and 0.78 ng/ml.  These are then
dispensed to the <em>first five</em> columns of all coated plates, dispensing
from lowest concentration (column 5) to highest concentration (column 1).
<!-- =============================================================
     =============================================================
     =============================================================
     ============================================================= -->
<hr>
<p>
This document has been divided into the three parts.  The other parts are:
<p><a href="GeneralScienceVol3%20(2001-present).html">General Science Volume 3</a>
  (from the year 2000 on)
<p><a href="GeneralScienceVol1%20(to%20and%20incl%201986).html">General Science Volume 1</a> (up to 1986)
</p>
</div> <!-- class=container -->
</body>
</html>
