<!DOCTYPE html>
<html>
<head>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8">
<title>Notes on the Literature in Proteomics (Vol 2: 2007-)</title>
<link href="../style/std.css" type="text/css" rel="stylesheet">
<link href="../style/biblio.css" type="text/css" rel="stylesheet">
<style type="text/css">
	table.td-top td {vertical-align:top;}
	.left-align {text-align:left;}
</style>
</head>
<body>
<p id="title">
Notes on the Literature in Proteomics
<p id="subtitle">
Vol 2:  2007-
<!-- ============================================================= -->
<hr style="margin-top:4em;">
<p>
This is the first in a series of documents containing notes (more than
abstracts but less than the full published reports) on the data and results
in papers relevant to research on methods and methodologies in proteomics,
or in which proteomics is used as a tool.  The (HTML) titles for these
documents are:

<ul>
<li>Notes on the Literature in Proteomics (Vol 1: 19xx-2006)
 <br>[this document:
 <span class="tt"><a href="ProteomicsLit(19xx-2006).html">ProteomicsLit(19xx-2006).html</a>]</span>
<li>Notes on the Literature in Proteomics (Vol 2: 2007-)
 <br>[this document:  <span class="tt">ProteomicsLit(2007-).html</span>]
</ul>


<!-- ================================================================ -->
<p class="tags">

<p class="ref">
 <span class="authors">
Chowdhury SM, Munske GR, Ronald RC, Bruce JE
 </span>
 <span class="year">
(2007)
 </span>
 <span class="jname">
J. Am. Soc. Mass Spectrom.
 </span>
 <span class="jvol">
18,
 </span>
 <span class="pages">
xxx
 </span>
<p class="reftitl">
Evaluation of Low Energy CID and ECD Fragmentation Behavior of Mono-Oxidized
Thio-Ether Bonds in Peptides
<p class="Para1">
Bottom-up (expression) proteomics involves protein isolation and comparison
of its tryptic peptide masses obtained by MS to a database set of sequences
for high confidence identifications.  In the process, proteins must be
reduced to break -S-S- bonds whose reformation is blocked with an
alkylating agent in a standard nucleophilic substitution, with other
reagents being Michael acceptors.  The product has a thioether bond
(-R-S-R&prime;-).  If iodoacetamide was used, a neutral fragment is lost
during low energy MS/MS with formation of dehydroalanine
(CH<sub>2</sub>=C(NHR)-COR&prime;) at the residue.  The part of the side
chain lost is a sulfenic acid (RSOH, where R is the rest of the structure
depending on blocking agent used).
<p>
For this study, two C-containing peptides (SEVAHRFKC [Pep1] and
CHWKQNDEQM [Pep2]) were synthesized on ABi 431-A, one with
N-terminal and other with C-terminal cysteines.  Pep1 was treated with
iodoacetamide then with 3% H<sub>2</sub>O<sub>2</sub> 15 min, desalted
with C18 ZipTip, vacuum dried and injected to ESI-FTICR MS
(Bruker Daltonics 7T APEX Q-FTICR, with CID spectra calibrated
using substance P, bradykinin, and Pep1) in 49% MeOH/2%
acetic acid solvent with a 20 &micro;l/h nanospray.  A [M+2H]<sup>2+</sup>
ion at <i>m/z</i> 575.2794 representing oxidized alkylated peptide
was passed as a parent to the collision cell with the trap voltage
altered to -1V, -6V, -9V, and -12V.  At -6V (low energy collision)
a peak at <i>m/z</i> 521.7770 shows at high intensity with
predicted neutral loss of RSOH (where R=58.0292).  Accurate mass calculation
shows this neutral fragment mass (RSOH, C<sub>2</sub>H<sub>5</sub>NO<sub>2</sub>S)
to be 107.0042 with the observed(measured) being 107.0048, an error of
5 ppm.  At -9V the ion also is at high intensity, thus at an energy low
enough that no other fragments but the neutral loss of RSOH occurring.
At -12V, the relative intensity was even greater for the ion.
<p>
Besides iodoacetamide, other blocking agents were studied. Maleimide
modifies thiols through Michael addition at a reactive alkenyl bond.
3-maleimidpropionic acid was used to modify Pep1.  A [M+2H]<sup>2+</sup>
ion at <i>m/z</i> 631.2848 is observed and collisional trap voltage altered.
Ions at <i>m/z</i> 521.7758 and 537.7613 appear, the latter its intensity
increasing with collision voltage, while the former had a low intensity
that did not change with increasing voltage.  Thus there is a loss
of R + H<sub>2</sub> from oxidized peptide, whose calculated mass is
187.0480 and whose measured mass is 187.0470.
<p>
4-vinylpyridine is another Cys-modifying agent.  It differs from being
maleimide-type in that there is asymmetry in the alkenyl bond.
Presence of an ion at <i>m/z</i> 599.2939 shows the [M+2H]<sup>2+</sup>
vinylpyridine-modified Pep1.  With increasing collisional trap voltage,
an ion at <i>m/z</i> 537.7590 increasing relative to base peak with
increasing voltage indicates neutral loss fragment R + water.  Also
increasing in relative intensity with collisional voltage increase is
an ion at <i>m/z</i> 1074.5105 (appearing strong in -9V spectrum,
1074.5125 in -12V spectrum), representing a +1 peptide with R + H<sub>2</sub> loss.
Also present to a small degree was the <i>m/z</i> 512.7590 ion, showing
a RSOH loss.
<p>
The N-terminal Cys peptide Pep2 was also used to see if it had the same
effect.  Peptide mass was checked for modified (and unoxidized) peptide.
In addition an electron capture dissociation (ECD) was used to generate
fragments (features a hot hollow cathode brought to 1.8-1.9 amperes and
located outside ICR cell, with side kick trapping voltage from +6V to -6V,
and electrons accelerated with 3 eV and 200 ms electron injection time).
Because Pep2 also has Met, it gets oxidized with the peroxide treatment.
The 3-maleimidopropionic-modified and oxidized peptide shows
[M+2H]<sup>2+</sup> ion at <i>m/z</i> 760.2837.  Increasing collisional
voltage produces ion at <i>m/z</i> 666.7595 representing 2+ ion with R +
H<sub>2</sub>O neutral loss.
<p>
The presence of Met as a sulfoxide is not different than an oxidized
thio-ether bond off the Cys side chain.  -15V spectra were obtained on
oxidized and untreated peptides and showed similar fragmentation patterns,
indicating that the Met sulfoxides do not fragment in the same way as
oxidized thio-ether modifications of Cys.  ECD spectra of oxidized
iodoacetamide-treated Pep1 generated three <q>c</q> ions, c6, c7, and
c8 (SEVAHR.F.K.C, where c6 is SEVAHR, etc.).  Also present is an [M+H]<sup>+</sup>
ion at <i>m/z</i> 1149.57 and 1091.57, the latter representing loss of
iodoacetamide from modified and oxidized peptide.  Thus R+H<sub>2</sub>
and RSOH losses are seen, but at energies that also cause peptide fragmentation.
Fragmentation patterns using ECD were similar for maleimide-treated oxidized
Pep2 and iodoacetamide-treated oxidized Pep1.  Generally with ECD
based spectra there is more fragmentation of the peptide backbone, and
in iodoacetamide-treated there is RSOH and R + H<sub>2</sub>O neutral losses,
but with maleimide treatment on R + H<sub>2</sub>O losses are seen.
<p>
The mechanism of gas phase &beta; elimination of oxidized thio-ethers
involves a coordination of an acidic (highly positively polarized) hydrogen
(&beta; to the sulfur atom and/or carbonyl group) to a non-bonding electron
pair on the oxygen atom forming the oxidized thio-ether.  Arrangements
of the bonds breaks the thio-ether bond to the R group, leaving
-CH<sub>2</sub>-S-OH sulfenic acid on the side and restoration of the
R group.  The sulfenic acid group is highly unstable, and leads to water
loss to produce -CH=S sulfide (relatively more stable).  This accounts
for the R + H<sub>2</sub>O loss.
<p>
In the other mechanism, the other thio-ether bond oriented to the side
chain amino acid is broken (rather than the one connecting the R group
modifier).  This is because the negative polar oxygen atom instead
bonds to the H atom on the &alpha;-carbon of the peptide backbone,
with a concerted breaking of the -C-CH<sub>2</sub>-S(=O)- thio-ether bond
to produce dehydroalalanine and the RSOH leaving group.  The modifying
group will dictate which mechanism dominates, if one dominates at all.
<!-- ================================================================ -->
<p class="tags">

<p class="ref">
 <span class="authors">
Alterovtiz G, Liu J, Afkhami E, Ramoni MF
 </span>
 <span class="year">
(2007)
 </span>
 <span class="jname">
Proteomics
 </span>
 <span class="jvol">
7,
 </span>
 <span class="pages">
2843-2855.
 </span>
<p class="reftitl">
Bayesian Methods for Proteomics
<p class="Para1">
<b>REVIEW.</b>
In tossing a coin, it is expected that for a proper coin, either of one of
two outcomes is equally likely.  But if a large count of heads begins to
appear, this additional information is used to determine the <b>predictive
power</b>.  Bayesian methods are a set of principles that incorporates
external information into the data analysis process.  This process starts
with an established probability distribution called the <b>prior distribution</b>.
Then one adds the information obtained by experience or polling or surveys
to create a <b>posterior distribution</b>.
<p>
<b>Independence</b> is a term used to describe events that occur indepedent&mdash;or
<b>unconditionally</b>&mdash;of other events.  In testing for a disease or
disorder, what is the probability of having the disease given a test result
indicating the disease, which is measure of the <b>sensitivity</b> of the test?
Suppose that the probability of a test being positive for a disease where
the disease is actually present is P(T+|D+)=0.95.  This means that the probability of
the test being negative when the disease is present is P(T-|D+)=0.05.  Suppose also the
probability of the test being negative when there is disease is P(T-|D-)=0.95,
and that it is negative when the disease is present is P(T+|D-)=0.05.
Additionally it is known that the disease is present in 1% of the public,
and so the probability of finding the disease is P(D+)=0.01.  Of interest to
the clinical scientist is the probability that the disease is present given the
test is positive:
<p class="eqnserif">
P(D+|T+) = {P(T+|D+)&times;P(D+)} / P(T+)
<p>
Note that P(T+) = {P(T+|D+)&times;P(D+)} + {P(T+|D-)&times;P(D-)}
<p>
The computed result is:
<p class="eqnserif">
(0.95 &times; 0.01) / (0.95 &times; 0.01 + 0.05 &times; 0.99) = 0.161.
<p>
This seems low, but because the disease is present in only 1% of the population,
this is the case.  Suppose a second test (T2) after a first test (T1) is done.
What is the probability of the disease presence given both a positive first and
second test result:
<p class="eqnserif">
[P(D+|T1 &cap; T2) = {P(T1+|D+)&times;P(T2+|D+)&times;P(D+)]/P(T1+ &cap; T2+)
<!-- ================================================================ -->
<p class="tags">

<p class="ref">
 <span class="authors">
Yanes O, Aviles, FX, Roepstorff P, Jorgensen TJD
 </span>
 <span class="year">
(2007)
 </span>
 <span class="jname">
J Am Soc Mass Spectrom
 </span>
 <span class="jvol">
18,
 </span>
 <span class="pages">
359-367.
 </span>
<p class="reftitl">
Exploring the <q>Intensity Fading</q> Phenomenon in the Study of Noncovalent
Interactions by MALDI-TOF Mass Spectrometry
<p class="Para1">
The noncovalent interactions studied here include bovine carboxypeptidase A
(CPA) with potato carboxypeptidase inhibitor (PCI) and trypsin with the bovine
pancreatic trypsin inhibitor (BPTI).  Leech carboxypeptidase inhibitor (LCI)
was also obtained.  LCI, PCI, CPA, and aprotinin were dissolved in 20 mM
ammonium acetate.  Trypsin was dissolved in MilliQ water.  A fraction of
proteins between 3-5 kDa using size-exclusion chromatography from <i>Hirudo
medicinalis</i> was lyophilized and dissolved in MilliQ water.  To measure
any inhibitory activity on CPA in this fraction or elsewhere, the CPA reaction
with substrate N-(4-methoxyphenylazoformyl)-Phe-OH at 350 nm was done.
<p>
Changes to relative intensity of PCI (<i>m/z</i> 4298) are studied  after
adding its interaction partner CPA with matrices SIN, DHAP, CHCA, ATT,
2,5-DHB, and HABA using dried droplet technique.  2,6-DHB, SIN and CHCA
were also prepared using the thin-layer sample prep method.  The thin-layer
technique dissolves matrix in acetone at 20 mg/ml (2,6-DHB at 100 mg/ml),
spots a 5 &micro;l aliquot of matrix on to target, then 0.5 &micro;l of
sample solution is deposited on top of the dried matrix layer.  After
drying, sample is washed with 2 &micro;l MilliQ water for a few seconds,
with the water removed.  In general, the dried droplet technique gave
better results.
<p>
When increasing amounts of CPA (100, 200, 400, 600 fmol) are added,
the PCI peak at <i>m/z</i> 4298 disappears in sinapic acid.  Other
peaks are clearly noticeable but do not <q>fade</q> as PCI does.
A CHCA/ammonium citrate matrix gave similar results, as did
ATT/ammonium acetate microcrystalline deposit matrix.
<p>
ATT is a less acidic matrix (Table 2 shows the pH values of the various
matrix solutions, reproduced here)
<table>
<caption>Table 2. pH values of the matrix solutions</caption>
<tr><th>MALDI matrices  <th>20% MeCN/water
  <th>20% MeCN / 20 mM ammonium citrate
  <th>20% MeCN/ 20 mM ammonium acetate
  <th>20% MeCN/ 20 mM ammonium dihydrogen phosphate
<tr><td>sinapic acid <td>2.5   <td>4     <td>4     <td>3.5
<tr><td>&alpha;-CHCA <td>2-2.5 <td>3-3.5 <td>3.5   <td>3.0
<tr><td>DHAP         <td>4.0   <td>5.0   <td>5.5   <td>4.5
<tr><td>ATT          <td>3.5   <td>4.5   <td>4.5-5 <td>4.0
<tr><td>2,5-DHB      <td>1.5   <td>3.0   <td>3.0   <td>2.5
<tr><td>HABA         <td>3.5-4 <td>5.0   <td>5-5.5 <td>4.5
</table>
<p>With ATT, two different crystal patterns developed.  At the rim of the
sample spot thin macrocrystalline long needles are prominent  with the
center covered with homogeneous microcrystalline layer (especially with
ammonium acetate).  When the laser is targeted on the macrocrystalline
needles, the PCI peak does not diminish (with or without ammonium salts),
but there is a diminution of signal in the microcrystalline layer.
<p>
High spot-to-spot signal variability is observed when the laser focus on
the long needle crystals.  In some positions, signal is lost quickly with
just a few laser shots.  Increase of laser fluence or number of laser shots
left the inhibitor unaffected: it has same relative signal as the uncomplexed
control.  With 2,5-DHB, PCI signal never diminished even with 2-fold molar
excess of CPA addition, indicating the matrix interferes with binding.
An important note:  There were no signals observed at <i>m/z</i> ~34,200, for
free CPA or at <i>m/z</i> ~38,500 predicted for the CPA-PCI complex when the
mass range was adjusted under these conditions.  Use of sinapic acid, CHCA,
and 2,6-DHB did not fade the PCI signal.
<p>
However, detectable signals were found with 7 pmol PCI + 10 pmol CPA using
the dried droplet method with various matrices.  CPA signal alone was observed
in the macrocrystalline ATT needles, about 4-fold above background.
CPA and CPA-PCI complex (5% as strong
as CPA alone) gave good signals with sinapic acid.  Signals were barely
detectable with HABA, with CPA-PCI about one-third to one-half of the CPA
signal.   The CPA-PCI complex actually had as stronger or even stronger
a signal than CPA alone, although both these were weakly above background.
With CHCA, both signals are strong, with the PCI-CPA signal about one-third
that of the CPA signal.  With 2,5-DHB, the CPA signal was barely above the
noise of background, although detectable.
<p>
Adjustments to laser fluence (energy per unit area) had no effect on fading.
There was no first-shot behavior in this intensity fading process:  relative
intensity is reduced to the same extent after 200 laser shots compared to
the first, with the exception of the ATT macrocrystalline areas.  Switching
from linear to reflectron configuration in positive ion mode does not
influence fading.
<p>
BPTI has no affinity for CPA.  In the results above, CPA-PCI interaction was
clearly affected by a particular matrix in a particular buffer.  PCI, CPA,
and BPTI were mixed in 20 pmol equal amounts and spotted.  The CPA-PCI complex
was detectable only with sinapic acid (in ammonium dihydrogen phosphate).
With ATT/ammonium acetate, a peak at <i>m/z</i> ~41,200 would calculate to be
a BPTI-CPA complex, very nonspecific.  There was no fading effect seen at all
with either the PCI-CPA in SIN or BPTI-CPA in ATT.  Strong PCI, and BPTI peaks
alone were seen with SIN, and with ATT, there was a weak PCI peak, very strong
BPTI peak, and even a [2BPTI]<sup>+</sup> peak.  Use of DHAP gave signals for
CPA, BPTI, and PCI alone.  With HABA, addtion of BPTI caused fading of the CPA
signal.  Use of time-delay extraction in linear and reflectron config (increasing
from 120 - 500 ns) had no discernable effect.
<p>
The trypsin-BPTI interaction was also tested using the 7 matrices and 2 spotting
techniques.  In a 1:1 interaction, intact complex is detected with SIN, DHAP,
CHCA, HABA (w/ and w/o ammonium salts), and high variabiliyt is seen with ATT
and 2,5-DHB.
<p>
Trypsin is not bound by LCI and these were mixed 1:1 and spotted.  LCI forms
a nonspecific complex with trypsin in all matrices, even more so with DHAP.
Important here though is the absence of signal fading, but this is true for
BPTI and for LCI.  When intensity fading MS is done in the sub-picomole range,
the results show a contrast:  BPTI will specifically fade when tryspin is
added, while LCI and PCI signals are unaffected.  But as with CPA-PCI, the
BPTI-trypsin complex is not observable in its mass range, unless in the picomole
range.
<p>
Why is the intensity fading effect not seen when picomolar amounts of
protein-peptide interactions used?  This may not be due effects of analytes on
higher amounts of proteins, but rather the sensitivity of microchannel plate-type
detectors.  With each laser burst, lower mass ions present strike the MCP first
and cause a saturation, reducing the count of higher molecular weight ions.
This has been confirmed using superconducting tunneling junction (cryodetectors).
<p>
<b>Conclusions</b>.
<ol>
<li>Detection of a true biomolecular interaction using MALDI MS cannot be
determined using a single solution alone in which the predicted mass signal
is found:  if the interaction is true, there should be a intensity fading
effect in the spectrum when one component is titrated against the other in
several solutions.  Multiple matrices may need to be tried.
<li>The best results obtained were from homogeneous microcrystalline layers
obtained using dried droplet technique.  No fading could be observed with
titration in which laser was focused on macrocrystalline areas of ATT and
2,5-DHB or with thinlayers of CHCA, 2,6-DHB, and SIN.
<li>Protein-protein interaction did not seem to be dependent upon matrix pH
for the proteins used in this study, although that may not always be the case.
</ol>
<!-- ================================================================ -->
<p class="tags">

<p class="ref">
 <span class="authors">
Sun W, Gao S, Wang L, Chen Y, Wu S, Wang X, Zheng D, Gao Y
 </span>
 <span class="year">
(2007)
 </span>
 <span class="jname">
Molec. Cell. Proteomics
 </span>
 <span class="jvol">
&nbsp;
 </span>
 <span class="pages">
MS T500022-MCP200
 </span>
<p class="reftitl">
Microwave-Assisted Protein Preparation and Enzymatic Digestion in Proteomics
<p class="Para1">
The LCQ Deca XP ion trap MS from Thermo-Finnigan and Voyager-DE Pro MALDI
ToF MS from ABi were used.  LC parts from Upchurch.  Columns are 5 &micro;m
300 &Aring; from Merck for C18, and for SCX from PolyLC.  BioRad's
MiniProtean II used for minigels.  Microwave is a Whirlpool VIP271 with
850 W max power.  All LC reagents are Merck, and trypsin, PMSF, and CHCA
are Sigma.
<p>
BSA was suspended 1 mg in 1 ml 25 mM ammonium bicarbonate pH 8.5, divided
into 200 &micro;l aliquots and five digestion protocols done: (1) reduce
in 10 mM DTT 57&deg; 1 h, alkylate room temp in 55 mM iodoacetamide
in dark for 30 min,
digest 1:50 w/w in trypsin 37&deg; overnight. (2) sample is placed in
0.6 ml conical tube, and reduce and alkylate as (1), and then microwaved
(see below). (3) Processed like sample (2), except that no alkylation step
is done.  (4) Processed like sample (2), except the DTT reduction is
done at 100&deg; and for only 5 min and the sample is not alkylated.
(5) This sample is like (2), but it is not reductively akylated at all.
The microwave procedure involves placing 1000 ml of water next to the
sample vials;  this acts an an energy sink.  The microwave is turned on
at max power for 1 min.  After irradiation, the vials are removed and
processed for MS analysis.  Trypsin is 0.5 &micro;M and ammonium bicarbonate
is 25 mM.  For the in-gel digestion protocol, a <q>standard</q> in which
10 mM DTT for 1 h, 55 mM iodoacetamide 30 min, 30 min dehydration in 100%
MeCN, 30 min rehydration in 0.5 &micro;M trypsin and 25 mM ammonium
bicarbonate, and then 16 h (overnight) is done.  Peptide extraction is
twice for 30 min in 5% formic acid with 50% MeCN (total time about 20 h).
The microwave extraction takes only 25 min:  the DTT/iodoacetamide
steps are omitted, 5 min dehydration in 100% MeCN, 5 min rehydration
in 0.5 &micro;M trypsin and ammonium bicarbonate, 5 min digestion using
microwave, and then twice for 5 min each the extraction reagents.
<p>
Results show that the MS spectrum of samples 1 and 2 were almost identical.
The only difference is that sample 2 is done in 1% of the time!  Samples
3 and 4 looked similar.  100 fmol from samples 1-4 were run on a single
LC column to MS, and the number of identified proteins were similar.
MS/MS sample identification for sample 5 was significantly reduced.
With a 1 minute digestion protocol, the need for alkylation is unnecessary
since disulfide bond reformation is unlikely within that period.
<!-- ================================================================ -->
<p class="tags">

<p class="ref">
 <span class="authors">
Luche S, Lelong C, Diemer H, Van Dorsselaer A, Rabilloud T
 </span>
 <span class="year">
(2007)
 </span>
 <span class="jname">
Proteomics
 </span>
 <span class="jvol">
7,
 </span>
 <span class="pages">
3224-3244.
 </span>
<p class="reftitl">
Ultrafast Coelectrophoretic Fluorescent Staining of Proteins with
Carbocyanines
<p class="Para1">
Protein staining in gels must balance considerations such as recovery of
any fixed proteins or alterations by staining that affect compatibility
with MS analysis.  Fluorescent dyes such as Sypro Ruby and Deep Purple
do not always show compatibility with MS as well.  Zinc and imidazole
staining have severe problems with linearity although sensitivity is
not an issue.  Nile Red staining is rapid and requires no fixation but
the stain is photolabile (presents difficulty in extended imaging and
excision) and there is a sensitivity limitation.  Sypro tangerine
needs only a single bath in saline but there is a cost and limited
sensitivity issue.
<p>
In fluorescent detection, a covalent labeling strategy can be used
as with the use of fluorescamine and MDPF;  there are improved techniques
in a recent report on multiplex visualization.  Sensitivity is a problem
since there are limited sites.
<p>
A second approach is the noncovalent association of the fluorophore which
alters its excitation/emission properties.  Ruthenium complexes and
epinococconone are examples.
<p>
Another approach is when the fluorophore is affected not by a direct association
but depends on the environment in which it is.  Nile Red and the SYPRO family
of dyes depend on this.  The SYPRO dyes are not really as sensitive as claimed
and cost substantially more.  Nile Red has more sensitivity but very limited
solubility and severe photolability.
<p>
<em>The ideal fluorescent probe should combine high absorptivity, an important
differential quantum yield between hydrophilic and lipophilic, and still be
water-soluble.  It should also show photostability.</em>
<p>
Carbocyanines show these characteristics, but staining of separated proteins
gave no promise.  The infusion of dye into a gel for co-electrophoresis has
been described for CBB and so it was tried here.  Use of water or
water-alcohol-acid to provide contrast after electrophoresis.  The acid
presence causes an intense background to develop in the low MW region because
of the presence of carrier ampholytes.  Lowering SDS concentration works a
previous report but results in severe tailing of the 2D spots.
<p>
The carbocyanine structure has a nucleus and can have long arm side chains.
The side chain can help to increase interaction with lipophilic environment
and thus effect a strong fluorescence, while there is a disadvantage in that
it can induce self-aggregation (and increase the background fluorescence
thereby) as well as reduce solubility.
<p>
200 &micro;g protein from J774 cell extracts were separated in a pH 4-8
gradient/Tris-taurine 2DE system and stained with different carbocyanine
dyes (oxacarbocyanine with diethyl, dipentyl, diheptyl, and dioctadecyl
lengths).  The diethyl side chains are too short and lack sensitivity,
while the dioctadecyl is insoluble and cannot reach optimal concentrations.
A 3 &micro;M concentration was found to be best:  lowering it decreased
sensitivity while increasing it did not enhance staining.
<!-- ================================================================ -->
<p class="tags">
FD
<p class="ref">
 <span class="authors">
Forbes MW, Jockusch RA, Young AB, Harrison AG
 </span>
 <span class="year">
(2007)
 </span>
 <span class="jname">
 J. Am. Soc. Mass Spectrom.
 </span>
 <span class="jvol">
18,
 </span>
 <span class="pages">
1959-1966.
 </span>
<p class="reftitl">
Fragmentation of protonated dipeptides containing arginine:  Effect of activation method
<p class="Para1">
Collisional activation in quadrupole ion traps is a very slow activation with
activation times approaches fractions of a second and low energy transfer to
reactant ions, and with similar characteristics to sustained off-resonance
irradiation (SORI) used in FT-ICR.  This provides for a great deal of
rearrangement to occur before fragmentation actually occurs.  Rearrangments
have low activation energies but also have unfavorable entropies of activation.
Fragmentation reactions however have higher activation energies and are more
favorable entropically.  Collisional activation of CID type in quadrupole cells
(in triple quad and hybrids) are of the beam-type.  Ubquitin gives more fragments
in collision cell CID than in QIT CID.  Farrugia and O'Hair reported that singly
protonated dipeptides Arg-Gly and Gly-Arg give identical spectra with QIT CID:
they proposed a mechanism in which there is time for rearrangments to form
a common intermediate which is a mixed anhydride,
NH<sub>2</sub>CH<sub>2</sub>C(=O)-O-C(=O)-CH(NH<sub>2</sub>)-(CH<sub>2</sub>)<sub>3</sub>NH-C(=NH<sub>2</sub><sup>+</sup>)NH<sub>2</sub>,
with intermediate salt bridge structures.  In MS<sup>3</sup> studies fragment
ions from two peptides gave identical mass spectra.  Gronert and coworkers have
reported QIT with alkali metal complexes of dipeptides can be rationalized
as formation of cationated mixed anhydride.  Other Gly-basic amino acid dipeptides
show the same reaction (Gly-Xxx, where Xxx = Lys or His), although there were
different relative abundances.  Fragmentation of [M+H]<sup>+</sup> forms of
Gly-Arg and Arg-Gly, in addition to fragmentation of their fragments, using
beam-type activation and cone-voltage CID or collisional activation in a QqTOF
instrument were studied.  Cone voltage CID  is where CID is between the atmospheric
pressure source and the quadrupole mass analyzer.
<p>
Peptides are dissolved in 50% MeCN + 0.5% formic acid (1:1 MeCN:1% HCOOH).
The spray is introduced at 30 &micro;l/min.  Nitrogen is used for nebulization
and desolvation.  For the hybrid QqTOF the solution is 50% MeOH + 0.05% HCOOH
and flowed at 80 &micro;l/min;  N<sub>2</sub> is used at nebulizing, desolvation,
and collision gas (in collsion cell).  MS<sup>2</sup> and MS<sup>3</sup> were
done with Bruker Esquire 3000<sup>+</sup> QIT, with 1 mM solution in 50% MeOH
at 10 &micro;l/min.  Precursors are isolate in 1 Th window and a resonant
excitation pulse (V<sub>ac</sub>) of 10 ms duration is used.  With MS<sup>2</sup>
precursors are positioned at <i>q<sub>z</sub></i> = 0.25 (low mass cutoff ~
<i>m/z</i> 60) and optimum frag efficiency with V<sub>ac</sub> = 0.75 V.  A higher
V<sub>ac</sub> = 2.25 V gives more energetic dissociation.  A larger trapping
potential <i>q<sub>z</sub></i> = 0.38 was used to obtain good trapping efficiences,
and increasing low mass cutoff to <i>m/z</i> 93.  The data will reflect three
different type of CID:  cone voltage CID, Q-ToF CID, and QIT-CID.
<p>
Data for Gly-Arg and Arg-Gly shows that CV-CID and QTOF-CID produce more fragments,
although in low intensity (abundance).  A comparison of the CV-CID spectra
for Gly-Arg and Arg-Gly shows very different fragment ion spectral intensities
for several species.  This is in contrast to QIT-CID spectra.  There are also
difference for the QTOF-CID fragment spectra too.  From the masses, elemental
compositions are determined mathematically.  In contrast, spectra from QIT-CID
for Arg-Gly and Gly-Arg under conditions of 0.75 V peak-to-peak excitation
voltage and <i>q<sub>z</sub></i> = 0.249 with 10 ms excitation time shows
identical spectra.  This confirms that rearrangements occur to a common intermediate
just before fragmenation ensues.  However, when the excitation voltage is
increased, requiring an increase in the low mass cutoff and the inability to
observe certain ions, there is fragmentation spectra with more ion species.
<p>

What about low excitation in beam-type collisions. MH<sup>+</sup> ions in a
QqTOF were set at 12 eV collision energy, near fragmentation threshold, and 
spectra normalized to <i>m/z</i> 215 [M + H - NH<sub>3</sub>]<sup>+</sup>. At 
low CE values, fragmenation products are [M + H - NH<sub>3</sub>]<sup>+</sup>, 
[M + H - 2 NH<sub>3</sub>]<sup>+</sup> (<i>m/z</i> 198), [M + H - 
guanidine]<sup>+</sup> (<i>m/z</i> 173) and a product at <i>m/z</i> 172. This 
products also occur with protonated Arg and Arg-containing peptides, but do not 
help in supporting an isomerization of MH<sup>+</sup>. For Gly-Arg, products 
are [M + H - H<sub>2</sub>O]<sup>+</sup> (<i>m/z</i> 214), [M + H - NH<sub>
3</sub> - H<sub>2</sub>]<sup>+</sup> (<i>m/z</i> 197), ArgH<sup>+</sup>(<i>
m/z</i> 175), and [M + H - glycinamide]<sup>+</sup> (<i>m/z</i> 158). For 
Arg-Gly, the dominant signal is <i>m/z</i> 157, the Arg b<sub>1</sub> ion. The 
a<sub>1</sub>-y<sub>1</sub> mechanism for amide bond cleavage without prior 
rearrangement explains the path to product. For Gly-Arg, the y<sub>1</sub> ion 
is formed, explaining the <i>m/z</i> 175 product, and for Arg-Gly, it si the <i>
m/z</i> 157 ion. These findings show that even for low energy beam-type 
collisions no rearrangement has occurred. The Gly-Arg product ions in beam-type 
can be observed in CID-QIT if the excitation energy is set high.
<p>
For the Arg-Gly spectrum, the low energy collision produces <i>m/z</i> 157,
a <i>b</i><sub>1</sub> ion which is Arg itself. &alpha;-aminoacylium ions
lose a neutral CO as they cannot form stable oxazolone structures.  If there
is nucleophile on side chain, cyclization using side chain might occur.  This
is true for <i>b</i><sub>1</sub> ions derived from Lys, His, Met, and Arg.
<p>
Gly-Arg fragments are numerous in product species, but the Arg-Gly shows
domination by a <i>m/z</i> [C<sub>4</sub>H<sub>8</sub>N]<sup>+</sup> species.
Common fragmentation route for the GlyArg and ArgGly is the product
protonated guanidine (<i>m/z</i> 60).  This also forms in fragmentation of
ArgH<sup>+</sup>.  In addition the charge can transfer to the complement
fragment instead of guanidine to produce <i>m/z</i> 173 ion.  Protonated
Arg also undergoes neutral losses of NH<sub>3</sub> to form a <i>m/z</i> 158
ion, loss of guanidine (<i>m/z</i> 116), and then to proline imimium wit
<i>m/z</i> 70.
<p>
With the <i>m/z</i> 215 [M + H - NH<sub>3</sub>]<sup>+</sup> in the QqTOF
fragmentation is isomer-dependent.  A scheme was produced that relies on
accurate mass of product ions and on what is known about neutral losses.
Six paths are described, with moiety lost shown and the product ion m/z:
<ol>
<li>-NH<sub>3</sub>, m/z 198 &rarr; -(H<sub>2</sub>O + CO) m/z 152
<li>-HN=C=NH, m/z 173
<li>-HN=C=O, m/z 172
<li>-(CH<sub>2</sub>=NH) + CO, m/z 158 &rarr; -(H<sub>2</sub>O + CO) m/z 112
<li>-C<sub>2</sub>H<sub>5</sub>NO (CH<sub>3</sub>C(=O)-NH<sub>2</sub>?), m/z 156
<li>-C<sub>5</sub>H<sub>9</sub>NO<sub>2</sub>, m/z 100
</ol>
<p>
The initial NH<sub>3</sub> to form the <i>m/z</i> 215 likely comes from the
guanidino group, as shown for ArgH<sup>+</sup> fragmentation. Elimination of a
neutral loss being HN=C=O (isocyanic acid?) to form the m/z 172 occurs in the
QIT-CID. For Gly-Arg, we see loss of CH<sub>2</sub>N<sub>2</sub> probably
carbodiimide. This dipeptide also loses methyleneimine (CH<sub>2</sub>=NH) and
CO for the m/z 158 ion. And Gly-Arg produces the m/z 100 ion, which might be
the fragment [H<sub>2</sub>N-CH<sub>2</sub>-C(=O)-NH=C=NH]<sup>+</sup> +
proline, formed from the m/z 215 which is where the Arg side chain has cyclized
with the &alpha;-amino-N of Arg. It should be noted that dipeptides of the type
Xxx-Arg do involve the neutral loss of a 115 Da size, which would correspond to
proline. This is not the case for m/z 215 ion derived from Arg-Xxx dipeptides.
Note that the m/z 156 ion comes directly from m/z 215 ion and also from m/z 173
ion, the latter de-guanidinated. Since they give rise to two distinct
structures and the neutral losses do not have the same moleculear formula, the
m/z 156 ion is probably two peak signals that cannot be resolved from one
another in the QqTOF.  With QIT-CID, m/z 173 fragmentation from the Gly-Arg
shows m/z 156 as dominant (ammonia loss) and also m/z 70, while QIT-CID for
m/z 173 from Arg-Gly showed also these two ions with same intensity but also
a minor m/z 172 ion, probably a loss of H from another isomeric intermediate.
Seven paths from m/z 173 ion (C<sub>7</sub>H<sub>13</sub>N<sub>2</sub>O<sub>3</sub>)
are also proposed:
<ol>
<li>-CH<sub>2</sub>=NH, m/z 144
<li>-NH<sub>3</sub>, m/z 156 &rarr; -(H<sub>2</sub>O + CO), m/z 110
<li>-(H<sub>2</sub>O + CO), m/z 127
<li>-(CH<sub>2</sub>=NH + CO), m/z 116 &rarr; ...
 <ol style="list-style-type:lower-roman;" start="5">
 <li>-CO, m/z 88 &rarr; -NH<sub>3</sub>, m/z 71
 <li>-(H<sub>2</sub>O + CO), m/z 70
 <li>-H<sub>2</sub>O, m/z 96
 </ol>
</ol>
<!-- ================================================================ -->
<p class="tags">

<p class="ref">
 <span class="authors">
Sun W, Gao S, Wang L, Chen Y, Wu S, Wang X, Zheng D, Gao Y
 </span>
 <span class="year">
(2007)
 </span>
 <span class="jname">
Arthritis &amp; Rheumatism
 </span>
 <span class="jvol">
56,
 </span>
 <span class="pages">
3588-3600.
 </span>
<p class="reftitl">
Salivary Proteomic and Genomic Biomarkers for Primary Sj&ouml;gren's Syndrome
<p class="Para1">
Sj&ouml;gren's syndrome (SS) is chronic autoimmune disorder described in 1933,
characterized by dry mouth (xerostomia) and dry eyes (keratoconjunctivitis
sicca).  The disease occurs in women primarily (9:1 over men) and affects up
to 4 million Americans (~1 in 100).
<!-- ================================================================ -->
<p class="tags">
FD
<p class="ref">
 <span class="authors">
Shi T, Siu KWM, Hopkinson AC
 </span>
 <span class="year">
(2007)
 </span>
 <span class="jname">
J. Phys. Chem. A
 </span>
 <span class="jvol">
111,
 </span>
 <span class="pages">
11562-11571.
 </span>
<p class="reftitl">
Generation of [La(peptide)]<sup>3+</sup> Complexes in the Gas Phase:
Determination of the Number of Binding Sites Provided by Dipeptide,
Tripeptide, and Tetrapeptide Ligands
<p class="Para1">
Gas-phase metal-ion complexes for neutral (non-deprotonated) peptides have
been reported only for monocations (alkali metal ions, copper, silver)
and dications (calcium, nickel, copper, zinc).  Triply charged metal ions
undergo charge reduction by proton abstraction from the peptide, producing
[M(peptide-H)]<sup>2+</sup>.  Bradykinin (RPPGFSPFR) was studied with
La<sup>3+</sup>, and all studies involved basic form of Arg or Cys, which
would be expected to interact with triply charged metal ion.  The question
is now the interaction is done (ionic? d-shell coordinate?).  In searching
for 3+ ions, La has the third lowest ionization energy (19.2 eV) of all
trivalents; its preferred coordinatin is 8.  La is often used to probe
alkaline earth metal-binding sites (esp. for Mg2+ and Ca2+) in proteins.
As La3+ with Ca2+ are <q>hard acid</q> cations (no valence shell electrons
for back-donation) they like to bind to <q>hard bases</q>: fluorine, oxygen
rather than the softer bases (nitrogen, phosphorus, sulfur).  Thus
these cations engage in mostly ionic rather than coordinate (as do
transition metals) interactions.
<p>
A prelimiary investigation could not produce
[La(Gly)<sub><i>n</i></sub>]<sup>3+</sup> for <i>n</i>=2-5.  When MeCN
was included, it become a strong binding solvent having complexes of
[La(Gly-Gly|Gly-Gly-Gly|Gly-Ala)(CH<sub>3</sub>CN)<sub><i>n</i></sub>]<sup>3+</sup>
for the three peptides shown.  CID did not dislodge the MeCN molecules,
but instead produced charge-reduced complexes:
[La(peptide-H)(CH<sub>3</sub>CN)<sub><i>m</i></sub>]<sup>2+</sup>,
where proton transfer goes to departing MeCN molecule.  High energy CID
saw singly charged ions of type [La(peptide-2H)]<sup>+</sup>.
<p>
In this work, La(III)nitrate is mixed with tetrapeptides LWMR (604 Da)
and MRFA (532 Da).  ESI shows protonated peptides
[peptide+<i>n</i>H]<sup><i>n</i>+</sup> for n = 1-2 and also
[La(NO<sub>3</sub>)(tetrapeptide)]<sup>2+</sup>.  Under <q>mild</q> conditions
(declustering potential DP = 20 eV), there is some [La(peptide)]<sup>3+</sup> but
in low abundance.  CID of [La(peptide)]<sup>3+</sup> at laboratory
collision energy (<i>E</i><sub>lab</sub>) of 15 eV and 8 mTorr water/nitrogen
collision gas produced water addition to complexes
[La(peptide)(H<sub>2</sub>O)<i>n</i>]<sup>3+</sup>.  The most abundant
complexes were [La(LWMR)(H<sub>2</sub>O)]<sup>3+</sup>
and [La(MRFA)(H<sub>2</sub>O)<i>2</i>]<sup>3+</sup>.  Previously the coordination
around La3+ was found to be 8 with water as an auxiliary ligand,
confirmed theoretically using DFT calculations.  Thus the interpretation
is that the peptide LWMR coordinates seven sites, while MRFA coordinates 6.
It must be noted that the C<sub>&alpha;</sub>-C<sub>&beta;</sub> bond on
Trp side chain was cleaved to give a m/z 130 ion and its complement
[La(LWMR-130)]<sup>2+</sup> at m/z 307.  The m/z 130 ion has been seen
with [Cu(dien)(GGW)]<sup>&bull;2+</sup> and [Trp+H]<sup>+</sup>.
DFT calculations have show metal ios bind preferentially to carbonyl O atoms
and terminal amino groups.  For [Ag(GGG)]<sup>+</sup>, the metal coordinates
to the 3 carbonyl O atoms and the terminal N atom.  For [Na(GG)]<sup>+</sup>
it coordinates to three atoms.  For [La(tetrapeptide)]<sup>3+</sup>,
the terminal -COO<sup>&minus;</sup> is ionized and there is a bidentate
fix from carbonyl O atom and hydroxl O anion.  In addition, the sulfur
participates in coordination at Met thioether function.  In addition the Trp
indole group can coordinate with metals as an electron-rich group.  Its
affinity is 5.7 kcal/mol higher than the phenyl ring provided by Phe.
<p>
Although [La(GGG)]<sup>3+</sup> could not be formed but the charge-reduced
[La(GGG-H)]<sup>2+</sup> could, substitution of one position with Arg
changed this.  [La(GGR)]3+ and [La(GRG)]3+ were found although in low
abundance.
<!-- ================================================================ -->
<p class="tags">

<p class="ref">
 <span class="authors">
Zhang Q, Tang N, Brock JWC, Mottaz HM, Ames JM, Baynes JW, Smith RD, Metz TO
 </span>
 <span class="year">
(2007)
 </span>
 <span class="jname">
J. Proteome Res.
 </span>
 <span class="jvol">
6,
 </span>
 <span class="pages">
2323-2330.
 </span>
<p class="reftitl">
Enrichment and Analysis of Nonenzymatically Glycated Peptides:
Boronate Affinity Chromatography Coupled with Electron-Transfer
Dissociation Mass Spectrometry
<p class="Para1">
A Glycogel II boronate affinity gel was used to isolate glycated poteins
and peptides because it has good gel penetration (high MWCO).  The ligand
is <i>m</i>-aminophenylboronic acid attached to agarose support.  Boronate
complexes with <i>cis</i>-diols in alkaline conditions forming reversible
5-member ring complex.  Elution can be done by lowering pH using 0.1 M
acetic acid or by using high sorbitol buffer which competes for boronate
binding (also has higher affinity).  RNase is small monomeric protein
with ten &epsilon;-amino ends from Lys and the single &alpha;-amino N-terminus.
Primary amino groups can be glycated (although natually RNase is not really
glycated). 1 &micro;mol (13.7 mg) RNase was dissolved in 1 ml 0.4 M glucose
in 0.2 M pH 7.4 phosphate buffer prepared in 0.2 &micro;m filtered water.
A drop of toluene was added to maintain sterile conditions to one half; to
one half 1 mM diethylenetriaminepentaacetic acid was added and it was placed
under nitrogen atmosphere, and kept at 37&deg; for 3-14 d, then stored at
&minus;20&deg;.  RNase incubated for full 14 days showed 87.1% glycation
for 3 days it was 60% glycated (assumed from binding fraction of boronate
column).  Only 0.6% of protein was bound that had not been treated with glucose.
&alpha;-1-antichymotrypsin and &alpha;-2-hs-glycoprotein are both glycoproteins
with six N-linked and two N-linked and three O-linked glycans, respectively,
typically used in periodate/hydrazide studies.  In the boronate column,
1.5% and 1.2%, respectively were bound.  The relatively low binding might
be due sialylations and the absence of cis-diols as a result of O-acetylation;
the negatively charged sialic acid residue could also not be bound to negatively
charged boronate.  This last experiment demonstrates that just because a
molecule has cis-diols, other factors can prevent its interaction with
the boronate column.
<p>
14-day-treated RNase bound to and eluted from the boronate column was
subjected to trypsin and the peptides re-applied to the Glycogel II column.
Only 14% of peptides were retained.  An Affi-gel 601 boronate column was
used (it differs in having a very low gel penetration, MWCO of 6000 Da), but
its ability to retain even lower MW peptides was less than the Glycogel II.
<p>
In another experiment, proteins bound and unbound to the Glycogel column
were digested with trypsin and examined on LC-MS/MS.  A different fraction
of this was pre-treated (desalted) on C18 SPE cartridges.  No glycated peptides
were observed in the unbound fraction, and all of them were seen in the
bound fraction.  An ETD MS/MS was used to obtain sequencing data (this
is Agilent 6340 Ion Trap with ETD and Agilent HLPL-Chip Cube MS interface,
well plate sampler, nanoflow pump, capillary pump.  Flouranthene anions from small
negative chemical ionization were used to produce electrons; MS was operated
in data-dependent mode with m/z 300-2200 with three CID and ETD scans done
alternately.  A table is presented showing the sequence of the eight identified
glycated peptides, their glycation site (two sites in case of one peptide).
Results are consistent with a previous study using different technique to
identify peptides and sites; some Lys residues were found that were glycated
not found in the other study; and this study did not find two sites glycated
whereas the other did (N-terminus and K37).  The absence of N-terminal glycation
could be due to limitation of current algorithm to find modifications of
N-terminus.  The inability to find a K37 glycation could be because of poor
retention on C18 SPE cartridges.  All glycations were found when peptides were
enriched on the boronate column.  Most were found when enrichment is done
on the protein level.
<p>
Commercially glycated human serum was enriched for glycated proteins
using boronate column.  About 28% of proteins are bound.  Eluted proteins are
are desalted and digested, and the peptides re-passed on the column, in which
5% of peptides are retained.  If glycated human serum is digested before any
enrichment, then the recovery of peptides is much reduced (1.2% of peptides).
Thus it is advisable to do a two-pass enrichment, first at protein then
at peptide level.
<p>
Different samples of serum glycated in vitro were analyzed on LC-MS/MS using
alternating ETD/CID MS/MS.   Processed spectra (Spectrum Mill software)
showed 88 unique peptides and 27 unique proteins.  Using ETD, 87.6% of peptides
were shown to be glycated in samples that had been two-step enriched.  76.4%
of peptides identified as glycated had only been enriched as peptides.
Only 12.3% of all peptides identified as glycated were enriched at protein level.
In samples where no enrichment was done, 6.4% of peptides were identified
as glycated.  Thus reduction of sample complexity promotes identificatin of
a greater number of PTM peptides.   This was true for either ETD-sequenced
or CID-sequenced MS data.  ETD was more effective in CID in sequencing the
PTM-peptides:  ETD produces more <i>c</i> and <i>z</i>-type ions, and these
spectra give a more unambiguous identifcation of glycosylated Lys side chain.
With CID, there is more neutal loss of water, and many <i>b</i> and <i>y</i>
ion types are missing.
<p>
Most glycated peptides mapped to the more abundant proteins in serum:
albumin, &alpha;-1-antitrypsin, apolipoproteins AI and AII, complement C3
precursor, serotranferrin precursor, not usual since the 22 most abundant proteins
make up ~99% of total protein mass of serum.  It is advisable to use serum
depletion methods of the major serum proteins in order to obtain low abundance
potential biomarkers.
<p>
A reversed protein sequence database was searched with these data sets, raw
results filtered using same criteria.  False discovery rates (FDR) for data
under ETD mode is lower (median FDR 1.9% for 7 data sets) than for CID mode
(median FDR 5.6% for 7 data sets).  The FDR is a direct reflection of
spectrum quality.  Unbound fraction analysis on LC-MS/MS revealed almost
no glycated peptide presence, particularly when two-step or peptide-level
enrichments were done.
<!-- ================================================================ -->
<p class="tags">
FD
<p class="ref">
 <span class="authors">
Unnithan AG, Myer MJ, Veale CJ, Danell AS
 </span>
 <span class="year">
(2007)
 </span>
 <span class="jname">
J. Am. Soc. Mass Spectrom.
 </span>
 <span class="jvol">
18,
 </span>
 <span class="pages">
2198-2203.
 </span>
<p class="reftitl">
MS/MS of Protonated Polyproline Peptides: The Influence of N-terminal Protonation
on Dissociation
<p class="Para1">
Polyproline forms two possible helical structures (not the traditional
&alpha;-helix) called ProI and ProII.  ProI is right-handed in solutin with
3.3 residues per turn and peptide bonds are <i>cis</i> in orientation.
ProII is left-handed with 3.0 residues per turn and all peptide bonds
<i>trans</i>.  Helix type depends on solvent.  Low protic solvents
(butanol, propanol) favor ProI, and ProII is usually in more protic solvents
(water).   If polyPro is put into gas phase from propanol, ProI helix features
persist, as shown for different lengths of the chain.  ESI however from water
did not show ProII helix type features persisting, suggesting helix is
not stabilized without solvation.  One feature of interest regardless of
solvent is H-bonding between protonated N-terminus and 2nd proline.  This
bonding cannot occur for internal Pro residues as they lack amide N hydrogens.
Molecular modeling with (Pro<sub>2</sub>+H)<sup>+</sup> shows the lowest
energy structure has hydrogens on the N-terminal N atom and H-bonding with
carbonyl O atoms on 1st and 2nd residue.  In peptides, normally <i>y</i> ion abundance
is significant because cleavage occurs at the N-terminal side of Pro;
proton affinity for proline and the strained structure of the b-type ion
have been proposed as reasons for selective formation of <i>y</i> ions.
<p>
Experiments were done using Bruker Esquire 3000plus Quad ion-trap MS with
positive ion mode ESI.  Helium is collision gas.  MS<sup>n</sup> is done with
CID, using 2.0 Da isolation window and q<sub>z</sub> ~ 0.25 for parent ion.
Activation time is 40 ms.  Amplitude energy optimized to minimize parent ion
intensity, maximize product ion, and minimize ion ejection (this value usually
0.50 V for this instrument).  Double resonance is done with modified Esquire LC
QIT.  Stored waveform inverse fourier transform (SWIFT) is used to eject product
suspected fo being reaction intermediate. Pro<sub><i>n</i></sub>-nh<sub>2</sub>
(<i>n</i> = 4-10) peptides and ProAlaPro<sub>8</sub>-nh<sub>2</sub> peptides are
synthesized with Fmoc on Rink amide MBHA resin (successful to get C-terminal
Pro).  Other peptides are from AnaSpec.  Solvent is 50% MeCN + 1% acetic acid.
<p>
The CID spectrum for [Pro<sub>8</sub>-nh<sub>2</sub> + H]<sup>+</sup>
is exclusively of <i>y</i> ions, although there are differences in abundances.
The y<sub>4</sub> and y<sub>6</sub> ions are major, resulting from loss of
even numbers of proline residues. The sodiated ion spectrum
[Pro<sub>8</sub>-nh<sub>2</sub> + Na]<sup>+</sup> is altogether different,
showing y ions of type [y<sub>n</sub>-H+Na]<sup>+</sup> where <i>n</i>=3-7.
Sodiated y<sub>4</sub> ion is not very intense, while protonated y<sub>4</sub>
is the most intense.  Also there is no y<sub>7</sub> ion in the protonated peptide, but
there is in the sodiated (cleavage between residues 1 and 2).  Sequential
loss from C-terminal is also not observed since the COOH is amidated (blocked);
with unmodified polyPor cationized with Na, rearrangements were observed
such that a peptide one shorter than the parent is seen.  A plot of
the relative intensities of y<sub>n-1</sub>, y<sub>n-2</sub>, y<sub>n-3</sub>,
and y<sub>n-4</sub> ions against the number of Pro residues <i>n</i>
(where <i>n</i> = 4-10) in a peptide reveals that the base peak for <i>n</i> = 4-7
is the yn-2 ion, and when <i>n</i> = 8-10, the base peak is the y<sub>n-4</sub>
ion.  y<sub>n-1</sub> and y<sub>n-3</sub> ion intensities increase as <i>n</i>
decreases, although not reaching base peak intensities.  The plot of protonated
polyPro and sodiated is similar:  a Gaussian shape centered at y<sub>n-2</sub> and
y<sub>n-3</sub> ion is seen.
<p>
The second position of Pro<sub>10</sub>-nh2 was substituted with Ala to see if
pairs of residues in protonated even-number loss required the residue be Pro.
The most intense ions are y<sub>8</sub>, y<sub>6</sub>, and y<sub>4</sub>,
with low intensity peaks at y<sub>3</sub>, y<sub>7</sub>, and b<sub>9</sub>.
The data indicate that for a peptide <i>n</i> residues long, y<sub>n-2</sub> ion forms
even if cleavage is Xxx-Pro where Xxx does not have to be Pro.
<p>
What role does the proton play in favoring dipeptide fragmentation?  Arg
was placed at the N-terminus as it is well known to sequester mobile protons.
ArgPro<sub>9</sub>-nh<sub>2</sub>.  Most intense product ion is a water loss.
There are y type ions as with other peptides, the most abundant of these being
y<sub>5</sub>.  Intensities of y<sub>8</sub>, y<sub>6</sub>, and y<sub>4</sub>
are not more significant than y<sub>7</sub>, y<sub>5</sub> and y<sub>3</sub> ions.
ArgPro<sub>4</sub>-nh<sub>2</sub> shows formation of y<sub>4</sub>,
y<sub>3</sub>, and y<sub>2</sub> in approx equal intensity; the base peak
is [M+H-H<sub>2</sub>O]<sup>+</sup>.  The proton can not be sequestered on
a basic residue for formation of y<sub>n-2</sub> and y<sub>n-4</sub> ions
to be major fragmentation pathway.  Without the strongly basic residue,
a proton cannot be held.  The most basic site would be the N-terminus.  This
is so for polyPro, since protons H-bond with 1st and 2nd residue carbonyl O atoms.
With collisional activation, the proton would move to other sites, usually
amide N atoms in preparation for fragmentation.  Acetyl-Pro<sub>10</sub>
CID spectra show an entire series of <i>y</i> ions, with y<sub>9</sub>,
y<sub>7</sub>, and y<sub>5</sub> being most abundant.  The intensities are
not distributed symmetrically; instead from y<sub>9</sub> to y<sub>4</sub>,
the intensities alternate between high and low.  MS<sup>3</sup> on y<sub>9</sub>
ion shows losses of even numbers of Pro residues, and y<sub>5</sub> and
y<sub>7</sub> formation exists through a y9 intermediate.  This y<sub>9</sub> ion
is analogous to [Pro<sub>9</sub>-nh<sub>2</sub> + H]<sup>+</sup> although
C-termini are different.  Identities of product ions in MS<sup>3</sup>
of [AcPro<sub>10</sub> + H]<sup>+</sup> &rarr; [y<sub>9</sub>]<sup>+</sup> &rarr;
products is same as MS/MS of [Pro<sub>9</sub>-nh<sub>2</sub> + H]<sup>+</sup>,
but relative intensities are different between y<sub>7</sub>/y<sub>5</sub>
(y<sub>n-2</sub>/y<sub>n-4</sub>). The y<sub>7</sub> ion is base peak in the
MS<sup>3</sup> spectrum while y<sub>5</sub> is base peak in MS/MS spectrum.
Thus y<sub>n-2</sub> and y<sub>n-4</sub> ions are formed even if dissociating
ion is an intermediate which came from parent with blocked N-terminus.
<p>
For polyPro<sub><i>n</i></sub> for <i>n</i> &gt; 6 we see y<sub>n-2</sub> and
y<sub>n-4</sub> ions.  A question remains whether y<sub>n-2</sub> is a precursor to
y<sub>n-4</sub>.  MS/MS spectra were acquired on
[Pro<sub>7</sub>-nh<sub>2</sub> + H]<sup>+</sup> with and without the condition
of double resonance. An intense y<sub>5</sub> and less intense y<sub>3</sub>
ions are produced normally.  Applying double resonance, the y<sub>5</sub>
is forcefully ejected to eliminate it being a precursor (i.e, immediately
when product is formed).  This eliminates the possibility that y<sub>3</sub> will
formed from it if it is formed from y<sub>7</sub> directly.  If y<sub>3</sub> abundance is
altered, that means that some or all of it is produced by y<sub>5</sub>.
<p>
The strongest suggestion from the data is that a mobile proton induces a
radical conformation change that causes proline fragmentation by twos.
Computation is now being done to understand how protonation at the N-terminus,
in line with the Pro helix dipole, could affect bond strength and alternating
amide nitrogens or the stability of neutrals formed by even numbers
of residues lost from the parent.
<!-- ================================================================ -->
<p class="tags">

<p class="ref">
 <span class="authors">
Hu S, Yen Y, Ann D, Wong DT
 </span>
 <span class="year">
(2007)
 </span>
 <span class="jname">
Drug Discovery Today
 </span>
 <span class="jvol">
12,
 </span>
 <span class="pages">
911-916.
 </span>
<p class="reftitl">
Implications of salivary proteomics in drug discovery and development:
a focus on cancer drug discovery
<p class="Para1">
Profiling salivary proteome can be used to assess health status.
<b>Biomarker</b> defined:  <q>measurable and quantifiable biological
parameters that can serve as indicators for health and physiology-related
assessments, such as pathogenic processes, environmental exposure, disease
diagnosis and prognosis, or pharmacologic responses to a therapeutic
intervention.</q>  Proteomics provides <q>protein signatures</q> or
<q>protein fingerprints</q>.
<p>Advantages include being non-invasive, low complexity sample during
processing.
<p>
<b>Biomarker potential</b>.  HER2 and CA15-3 found in higher amounts
in saliva of breast cancer patients relative to healthy and benign tumor
patients.  Patients with oral squamous cell carcinoma (OSCC) or head &amp; neck
squamous cell carcinoma show signatures of particular proteins: TNF-&alpha;,
IL-1, IL-6, IL-8, CD44, fibronectin, defensin-1, hyalouronidase, telomerase,
cytokeratin-19 fragment (CYFRA 21-1), tissue polypeptide antigen, CA125.
Subtractive proteomics shows OSCC-related markers:  Mac-2-binding protein,
catalase, calgranulin B, profilin, and CD59.
<p>
OSCC requires early detection.  5-year survival is more than 80% detected
at T1, but only 20-40% at laters (T3, T4) stages.  TNF-&alpha;, IL-1,
IL-6, and IL-8 are proangiogenic, proinflammatory cytokines and are
all significantly higher in saliva of patient with oral premalignant lesions.
These are all NF-&kappa;B-dependent proteins.
<p>
Biomarker study should not include these cytokines alone.  Perhaps a second
inflammatory disease group (patients with periodontal disease, for example)
and healthy controls.  Anti-p53 is an autoantibody that emerges in cancer
cases:  protein arrays might be used to detect these and other auto-antibodies.
<p>
<b>Pharmacoproteomics</b>.  This is proteomics related to the drug industry.
It is involved with target identification and validation, discovery of
efficacy and toxicity biomarkers, investigation into drug agonist mechanisms,
and chemo-resistance.  How does proteomics work?  Target validation:
how are cellular proteomes altered with use of matrix metalloproteinase
inhibitor drugs.  Quantitative proteomics provides such info.
<p>
<b>Toxicoproteomics</b>.  This is the <q>use of global protein expression
technologies to detect up- and down-regulation of genes associated with
drug toxcity risk; it allows for better understanding of the effects of
environmental and genetic factors, both in episodes of acute exposure to
toxicants and in the long-term development of disease.</q>  In one report
serum proteomic patterns by MS could point to organ toxicity related to
candidate drug use.
<!-- ================================================================ -->
<p class="tags">
FD
<p class="ref">
 <span class="authors">
Boutin M, Bich C, Afonso C, Fournier F, Tabet J-C
 </span>
 <span class="year">
(2007)
 </span>
 <span class="jname">
J. Mass Spectrom.
 </span>
 <span class="jvol">
42,
 </span>
 <span class="pages">
25-35.
 </span>
<p class="reftitl">
Negative-charge driven fragmentations for evidencing zwitterionic forms
from doubly charged coppered peptides
<p class="Para1">
Zwitterion forms are much less stable in gas phase than in aqueous solutions
(eg Gly is 22 kcal/mol more stable as uncharged on any moiety than as a zwitterion).
Stability increases in gas phase for zwitterion forms as proton affinity
increases: Arg has a zwitterion form about as stable as free acid.
<p>
Enkephalin analogs (YGGFL, YGGFL-nh<sub>2</sub>, YGGFK, YGGFQ, KYGGF, and QYGGF)
were synthesized and dissolved in water at 1 mg/ml.
CuCl<sub>2</sub> 6H<sub>2</sub>O was made to 500 pmol/&micro;l in water.
Samples are mixed in a final 50% MeOH so that 1:3 peptide/metal with 20
pmol/&micro;l.  A Bruker 3D ion-trap MS with ESI is used and infusion is
at 160 &micro;l per hour.  Capillary is -3500 V, end plate offset -500 V,
capillary exit +45 V, skimmer 1 = +15 V and skimmer 2 = +6 V.  These conditions
preserve the [M+Cu<sup>II</sup>]<sup>2+</sup> ions  Low mass cut off is 45 Th,
and scan range is 100-1000 Th.  The trap accumulation is limited to 10,000
to avoid space-charge effects.  Low energy sequential CID is done with resonant
excitation using amplitude of 0.70 V<sub>p-p</sub> and ion isolation window
of 2 Th.  A Bruker ApexQe hybrid Qq-FT-ICR with ApolloII ESI and shielded
7 T magnet was used for look for loss of NH<sub>4</sub><sup>+</sup>; operated
in broadband mode with external calibration.  CID using argon is done after
filtering with quadrupole.
<p>

ESI ion trap was used to select [M+Cu<sup>II</sup>]<sup>2+</sup> ions compared
to MH<sup>+</sup> ions.  The 2+ ion [YGGFL+Cu<sup>II</sup>]<sup>2+</sup> (m/z
309.1) shows in CID spectra peaks at m/z 300.6 and 600.2, which represent losses
of NH<sub>3</sub> and NH<sub>4</sub><sup>+</sup>, respectively. While neutral
loss is not unexpected, the loss of ammonium ion was confirmed by high
resolution (3 ppm) FT-ICR.  These losses indicate that positive charge is
transferred/distributed to the N-terminal amine.  Cu<sup>II</sup> ion must be
coordinated by the four amide groups forming the peptide backbone.  This proton
must come from one of several groups:  the -COOH at the C-terminus, the Tyr
phenol or protons on the amide groups of the backbone.  If the -COOH contributed
the proton, CO<sub>2</sub> neutral loss should be seen, and no remmant ion is
seen.  When the m/z 300.6 ion was run into MS<sup>3</sup>, HCOOH is lost,
confirming that the proton does not come from the carbooxyl group.  Rather the
proton must come from the phenol on Tyr.  A mechanism is presented that shows
how phenolate anion can delocalize the negative charge and show enough acidity
in the gas phase.  Other ions observed in <sub>m/z</sub> 300.6 MS<sup>3</sup>
analysis include <i>m/z</i> 147.0
[<i>b</i><sub>1</sub>-NH<sub>3</sub>]<sup>+</sup> and <i>m/z</i> 454.1. With
NH<sub>3</sub> loss, an ion-dipole complex then forms in which proton transfer
from the C-terminal COOH to the free NH<sub>3</sub> draws it off as
NH<sub>4</sub><sup>+</sup>; lifetime of the complex must be long enough to
permit proton transfer.  Precursor <i>m/z </i>309.1 gives a product of
<i>m/z</i> 256.1 which represents a neutral loss of
CH<sub>2</sub>=C<sub>6</sub>H<sub>4</sub>=O, namely Tyr side chain, observed in
decomposition of deprotonated Tyr-containing peptides (again, evidence that
phenol is deprotonated).  An ion-dipole complex forms to permit proton transfer
(probably from -COOH group), with product ions of m/z 107.1 and 511.2.  As
CO<sub>2</sub> is liberated from <i>m/z</i> 511.2 ion in sequential
MS<sup>3</sup>, this mechanism is supported.  An ion at m/z 301.6 results from a
15 Th loss from parent m/z 309.1, but this is not seen in CID spectra of
[YGGFL+<sup>65</sup>Cu<sup>II</sup>]<sup>2+</sup> or in the <sup>63</sup>Cu
isotope;  the 15 Th must come from interference.  The presence of copper in all
ions was verified:  [M+<sup>65</sup>Cu<sup>II</sup>]<sup>2+</sup> ion at
<i>m/z</i> 310.1 and all product ions above the precursor were shifted by 2 Th.
For signals between <i>m/z</i> 200-309 Th are shifted by 1 Th, corresponding to
copper-bearing doubly charged ions.  Below 200 Th, no m/z ratio shift is seen,
and thus ions are singly charged and not cationized.  The amide form of leu-enk
[(YGGFL-nh<sub>2</sub>)+Cu<sup>II</sup>]<sup>2+</sup> (<i>m/z</i> 308.6) shows a
large signal of <i>m/z</i> 255.6 O=C<sub>6</sub>H<sub>4</sub>=CH<sub>2</sub>
compared to protonated HO-C<sub>6</sub>H<sub>4</sub>-CH<sub>2</sub><sup>+</sup>
(<i>m/z</i> 107.1) and supports the implication of C-terminal proton in
formation of m/z 107.1 ion and thus the ion-dipole complex.  An alternative is
that the C-terminal amide O atom competes with N-terminal amine for protonation,
and a significant decrease in m/z 107.1 and 147.0 product ions because charged
losses are activated by -NH<sub>3</sub><sup>+</sup> site or produced after
NH<sub>3</sub> release from protonated amine.

<p>
YGGFK and YGGFQ peptides are isobaric pentapeptides
([M+Cu<sup>II</sup>]<sup>2+</sup>, <i>m/z</i> 316.6) and CID spectra show
similar product ions but with variations in abundance.  For YGGFK,
m/z 263.1 and 615.2 are more abundant, and <i>m/z</i> 147.1
[<i>b</i><sub>1</sub>-NH<sub>3</sub>] and <i>m/z</i> 263.6 are more abundant
in YGGFQ.  Ion m/z 263.1 enables distinction of the isobaric ions and
is the doubly charged product representing loss of
CH<sub>3</sub>C<sub>6</sub>H<sub>4</sub>O<sup>&bull;</sup>:  a proton transfer
driven by phenoxide negative charge in Tyr side chain and then the
one-electron reduction of Cu<sup>II</sup> occurs.  A zwitterion with negative
charge on phenol and positive charge on Lys occurs, making the N-terminal amine
free to coordinate Cu<sup>II</sup> with the proton transfer/reduction steps
proceed.  In contrast to YGGFK, most basic site on YGGKQ is N-terminal amine
and protonation prevents Cu(II) coordination.  The peptide YGGFL sees the
phenol side chain loss and generation of doubl charged m/z 263.6 and singly
charge m/z 107.1 and 526.2.  All three peptides also show -NH3 and -NH4+
losses.  A [b<sub>4</sub><sup>*</sup>-NH<sub>3</sub>] at <i>m/z</i> 469.2
ion is observed for YGGFK and YGGFQ, not for the Leu substitute.
<p>
Ion [KYGGF+CuII]2+ m/z 316.6 shows CID spectra with peaks at <i>m/z</i> 294.6
(CO2 loss) often seen with deprotonated peptides, and observations consistent
with negative charge on C-terminal -COO.  The proton becomes mobile migrates
to most basic site (Lys side chain amine), although it is proposed that the
proton forms a bridge between Lys -NH<sub>2</sub> and N-terminal -NH<sub>2</sub>.
With CO2 loss, the m/z 294.6 carbylide isomerizes with a 1,2 proton shift
from the benzylic C of Phe side chain, which can stabilize the negative charge.
Styrene (Phe side chain) is lost to produce ion m/z 242.6.  This loss does
not occur with QYGGF, indicating significance of Lys in process (Gln amide
has lower basicity).  IN QYGGF, the mobile proton in on the N-terminal amine,
and rather than the styrene loss, -NH<sub>3</sub> and -NH<sub>4</sub><sup>+</sup>
losses are promoted for this peptide.  Peaks seen for the isobaric petpides
of Gln- and Lys-YGGF are m/z 166.1 and 467.1, which are y<sub>1</sub> and
b<sub>4</sub><sup>*</sup> complementary ions.
<p>
When K and Q are placed at C-terminus (YGGFK, YGGFQ), negative charge
delocalization clearly alters.  Negative charge is absorbed by the Tyr side
chain and losses of NH<sub>3</sub> and
O=C<sub>6</sub>H<sub>4</sub>=CH<sub>2</sub> is favored, with
[b<sub>4</sub>-NH<sub>3</sub>] ion product.
<!-- ================================================================ -->
<p class="tags">

<p class="ref">
 <span class="authors">
Whitelegge JP, Zabrouskov V, Halgand F, Souda P, Bassilian S, Yan W, 
Wolinsky L, Loo JA, Wong DT, Faull KF
 </span>
 <span class="year">
(2007)
 </span>
 <span class="jname">
Int. J. Mass Spectrom.
 </span>
 <span class="jvol">
268,
 </span>
 <span class="pages">
190-197.
 </span>
<p class="reftitl">
Protein-Sequence Polymorphisms and Post-translational Modifications in Proteins 
from Human Saliva using Top-Down Fourier-transform Ion Cyclotron Resonance Mass 
Spectrometry
<p class="Para1">
<!-- ================================================================ -->
<p class="tags">

<p class="ref">
 <span class="authors">
Wu S, Skolnick J, Zhang Y
 </span>
 <span class="year">
(2007)
 </span>
 <span class="jname">
BMC Biol.
 </span>
 <span class="jvol">
5,
 </span>
 <span class="pages">
17.
 </span>
<p class="reftitl">
<i>Ab initio</i> modeling of small proteins by iterative TASSER simulations.
<p class="Para1">
<!-- ================================================================ -->
<p class="tags">

<p class="ref">
 <span class="authors">
Dettmer K, Aronov PA, Hammock BD
 </span>
 <span class="year">
(2007)
 </span>
 <span class="jname">
Mass Spectrom. Rev.
 </span>
 <span class="jvol">
26,
 </span>
 <span class="pages">
51-78.
 </span>
<p class="reftitl">
Mass Spectrometry-Based Metabolomics
<p class="Para1">
<b>REVIEW</b>.  The technology of genomics, proteomics, and transcriptomics have 
made significant strides but tools for comprehensive examination of the 
metabolome are still emerging (Bingo et al, 2004).  Metabolomics is closest to 
the phenotype. No single instrument can analyze all metabolites.  Bibliographic 
searches shows that metabolomics is fairly new, with 147 articles having 
keywords in 2003, and 203 in 2004.  Beecher (2003) estimates that there are 2000 
major metabolites in human, quantity and quality dependent on dietary state as 
much as other factors.  When secondary metabolites considered, the number is 
vastly larger.  Weckwerth (2003) figures that there are 200,000.
<p>
Two complementary approaches are used in metabolomic investigations. 
<b>Metabolic profiling</b> focuses on analysis of a group of metabolites either 
related to a specific metabolic pathway or a class of compounds.  Examples 
include quantitative analysis of fatty acids and their methyl esters by GC-FID 
or analysis of amino acids. In most cases profiling is hypothesis-driven: 
metabolites are selected for analysis and specific analytical methods developed 
for their determination.  Advances in technology have enabled the ability to 
quantitate metabolites simultaneously in a single analysis.  While a single 
biomarker determination can be as complex as profiling related key metabolites 
in a given biochemical pathway, the latter will provide a more complete, 
detailed description of metabolic perturbations than a single biomarker. One 
disadvantage of profiling is that it is not global or a true <q>omics</q> 
approach.  But when multiple profiling methods are used on the same condition or 
sample and data combined, the results will make metabolomics quite powerful. 
<b>Metabolic fingerprinting</b> is not about a rigorous systematic 
identification of metabolites as profiling is, but is involved with comparisons 
of patterns in response to changes in conditions or treatments, such as the 
occurrence of disease or exposure to toxin.  The workflow for fingerprinting is 
pretty much standard:  collect samples from the group of interest and another 
group representing the proper control, prepare samples by extraction (LLE, SPE, 
etc) and include internal standards, perform the instrumental analysis usually 
with derivatization (GC-MS, LC-MS, CE-MS), process the data typically by 
multivariate analysis (e.g. PCA) and perhaps produce score plots for groups and 
loading plots for markers, and identify markers using libraries, MS/MS, NMR, 
etc.  When the pattern analysis of metabolites is done on conditioned cell culture
medium, the method is called <b>metabolic footprinting</b>.  Fingerprinting
can be used as diagnostic tool; it can be used to establish a treatment baseline
and look for altered patterns.
<p>
Both fingerprinting and profiling can be used to discover new biomarkers.
<p>
Insofar as instruments, NMR, FT-IR, and MS coupled to separation techniques
using direct flow injection can be employed.  NMR is insufficiently sensitive
for low abundance metabolites although it can used for high-throughput 
fingerprinting.  MS-based metabolomics offers quantitative with high
selectivity and sensitivity with respect to identification.  Separation allows
resolution of metabolites (in a time dimension).  MS work requires sample 
prep which can result in metabolite loss.  In general parallel application of
several techniques is done (GC- and LC-MS) in order to get a comprehensive
assessment of the metabolome.  MS-based studies far outweight NMR-based work.
<p>
<b>Sampling &amp; Quenching</b>.  An analysis of statistical power should be
done to ensure that sufficient samples in the same analytical group and
to reduce biological variability for obtaining validated data.  For human
studies, confounding factors include diet, gender, age, genetic factors.
Fingerprinting requires large sample numbers.  Representative quality control
samples such as sample and analytical replicates and bland including method
blanks are to be analyzed.  To thwart continuing metabolic processes or
degradative techniques in sample containers, freezing by liquid nitrogen
or freeze clamping, acid treatment, or quenching in salt with aqueous MeOH
at low temperature should be considered.  Cold-buffered MeOH kept at 
&minus;40&deg; to &minus;50&deg; has been used to analyze microbial forms
(<i>E. coli, S. cerevisiae, A. niger</i>).  Note that addition of buffer
salts (like HEPES) might cause ion suppression in ESI-MS.  There might be
a great deal of data variability when times and methods vary with respect
to biofluid sampling and low temperature treatment.
<p>
<b>Sample Preparation</b>.  The preparative phase will extract the target
analytes of interest from a complex biological matrix in an attempt to 
eliminate or limit interference with analysis.  In the case of low abundance
metabolites, a preconcentration step will bring sample to sufficient to 
reach detection limits.  Sample prep methods include but not limited to
direct injection, liquid-liquid extraction, solid-phase extraction,
supercritical fluid extraction, accelerated solvent extraction, microwave-assited
extraction, protein precipitation, and membrane methods (dialysis, 
ultrafiltration, diafiltration), and ultracentrifugation.
<p>
Volatile organic compounds (VOCs) are often found in the urine, plasma,
and exhaled air metabolome.  Classes include alcohols, aldehydes, furans,
ketones, pyrroles, terpenes, and others.  In GC separations, extraction
solvents can often interfere/overlap with important analyte peaks, or
reducing their range of quantitation; analytes are cut off at the solvent
delay in GC-MS.  Headspace sampling with gastight syringe in combination with
cold trapping in temp-controlled cold injection system and GC-MS for analysis
of volatile metabolites in urine samples. 34 compounds were found in MS
libraries and reference compounds.  Another technique that is rapid and uses
no solvent is the headspace-solid phase microextraction (HS-SPME), which uses
a silica fiber coated with stationary phase for sampling.  The fiber is housed
in syringe-type assembly, and several different stationary phases  made
of non-polar to polar materials are available as fiber coating.  Analyte
distribution between sample and gas phase is affected by analyte-matrix
interactions.  Desorption of analytes from fiber achieved by thermal energy
of hot injector of GC or by solvent for coupling with HPLC.
<p>
SPE is a go-to method which provides a non-equilibrium exhaustive removal
from the mobile phase/matrix.  Compounds are retained by van der Waals
interactions, dipole-dipole interactions, H-bonding and electrostatic forces,
depending on analytes and sorbent.  Insofar as sorbents, silica, alkylated
silica (e.g. C18), carbon-based sorbents, ion-exchange media, polymeric 
materials, and restricted access materials (RAM) will remove macromolecules.
C18 reversed phase silica will not retain polar analytes and show a stability
at high pH, and analyte losses occur when the sorbent runs dry after 
equilibration step: these cautions should be noted.  As alternatives to silica-based
sorbents, funtionalized polymeric resins with polar functional groups on a 
nonpolar polymeric backbone give good retention of both polar and hydrophobic
analytes.  They have a wide pH range resistance and if run dry, will be less
prone to analyte loss.  Wang et al (2003) used C18 SPE to remove salts from 
serum after ultrafiltration on a 5000 Da cutoff filter.  The Oasis HLB system
is a functionalized copolymer of divinylbenzene and N-vinylpyrrolidone, and
was used to extract urine metabolities for LC-ESI-MS for screening.
<p>
Mixed phases are also being used for SPE sorbents.  Polymeric phases contain
strong and weak cation- and anion-exchange material to bind ionic metabolites
simultaneously.  Metabolites are recovered in organic solvents, which gets
neutrals and protonated acids, and then a second pH adjustment recovers basic
compounds.  
<p><b>Liquid-liquid extraction</b>.  It is a choice method in tissue 
extraction.  Tissues are frozen in liquid nitrogen, ground into fine powder
with mortar and pestle, mortar grinders, vibration mills, or by mixing with
beads of inert material, or by homogenizaiton of material in blenders with
a solvent.  High or ultra-speed supernatants are then shaken  with organic
solvents.  Polar metabolites are extract with 2-propanol, ethanol, methanol,
acidic MeOH, MeCN, water, or MeOH:water mixes.  Any solvent can be mixed.
More lipophilic metabolites are extracted with chloroform or ethyl acetate.
Water:MeOH:CHCl<sub>3</sub> or ethyl acetate extractions generate an upper
organic phase, which is rotary evaporated and resuspended for derivatization
or direct analysis.  In one experiment with plant tissue metabolites were
extracted with vibration mill with chloroform (200 &micro;l) followed by
addition of 800 &micro;l MeOH/water (75:25) to form a monophase.  
Weckwerth, Wenzel, and Fiehn (2004) developed a multiple solvent extraction
for sequential extraction of metabolites, proteins, and RNA from <i>Arabidopsis</i>
tissue.  Other extraction protocols have included high temperature extraction
with EtOH or MeOH, MeCN/water, 0.2% formic acid (50:50), acid or alkaline
treatment, cold MeOH, acidic MeOH-CH<sub>2</sub>Cl<sub>2</sub>-ethyl acetate.
In a systematic comparison of extraction reagents, one group tested at least
six or more systems and found that pure methanol (not hot) was the most
suitable for extracting polar intracellular yeast metabolites and perchloric
acid was the least suitable.
<p>
<b>Direct injection</b> for fingerprinting of urine on to LC-ESI-MS
is reported.  No metabolites are list, but high salt in urine causes
ion suppression and adduct formation, and nonvolatile residues deteriorate
performance.  Nanoelectrospray minimizes ion suppression and decreases
source contamination.  It is also useful to divert parts of the run,
such as the start of a run where salts are highly likely, away from the 
ESI interface to reduce likelihood of contamination.  When protein content is
high, metabolite separation can be poor and column backpressure might increase.
<p>
<b>Metabolic profiling in metabolic disorders</b>.  Inborn errors of metabolism
can be assessed by looking at urinary metabolome, especially using ESI-MS/MS.
Bloodspots dried on filter paper can be extracted with MeOH, with addition
of stable isotope-labeled internal standards, evaporation of solvent,
derivatization in acidifed butanol.  The butyl esters are analyzed using
FIA with ESI(+)-MS/MS with neutral loss of butyl formate (102 amu) for 
detecting neutral amino acids, neutral loss of 119 amu for ornithine and 
citrulline, and loss of <i>m/z</i> 161, for Arg.  Acylcarnitines are measured
using precursor scan of <i>m/z</i> 85.  Piraud et al (2003) developed a method
based on LC-ESI-MS/MS for analyzing 79 small molecules, mainly amino acids
related to metabolic disorders; underivatized amino acids and related analytes
were the objective for examination in a single run.  Interferences with
isobars and in-source fragmentation made the use of LC separation necessary.
<p>
<b>Lipidomics</b>.  Non-polar lipids have been routinely analyzed using
GC separation coupled to EI-MS.  Lipids can be methylated or silylated to
expand the range of analyzable molecules.  Of course this is limited to 
thermally stabile substances which can be volatilized at injection.  Triglycerides
are amenable to high temp GC work, but complex lipids (phospholipids, etc) 
require pre-sep of lipid classes, then hydrolysis, FFA derivatization, then
GC-EI-MS or GC-FID.  The method is very quantitative but labor intensive
and structural information is lost.
<p>
Besides hot split/splitless injection, analytical pyrolysis (Py) is used
in which nonvolatile sample is thermally degraded in inert gas prior to
GC injection.  Snyder et al (1990) used Py-GC-IT/MS to differentiate bacterial
species based on lipid profiles.  Basile et al (1998) used in situ thermally
assisted hydrolysis and methylation (THM) to differentiate bacteria.  Py
is coupled to GC-MS or for direct inlet to TQ/MS.  Quaternary N-alkylammonium
salts (e.g., tetramethylammonium hydroxide [TMAH]) form tetramethylammonium
carboxylates with organic acids and methylation takes place upon heating.
Ester-bound fatty acids are saponified and trans-methylated, but 
unsaturated fatty acids can undergo isomerization and recoveries are
incomplete.
<p>
Solid-state ion sources (such as FAB) and liquid-phase ion sources such as 
ESI or APcI allowed direct lipid analysis.  A lipidome analysis was developed
that is fully automated method making use of LC-ESI-MS quantitative for 
100 polar lipids (primarily phospholipids).  The shotgun lipidomics approach
uses crude tissue extracts obtained by Folch or Bligh-Dyer extraction methods.
Using both positive and negative mode ESI sample is directly injected.  In
ESI(+) mode acylcarnitines, phosphatidylcholine, lysophosphatidylcholine,
phosphatidylethanolamine, and sphingomyelins.  Free fatty acids, phosphatidic
acid, phosphatidylserine, phosphatidylinositol, phosphatidylglycerol, and 
phosphatidylethanolamine are ionized in ESI(-) mode.  ESI does not 
ionize triacylglycerols/neutral lipids; they can be detected as NH<sup>+</sup>,
Na<sup>+</sup>, and Li<sup>+</sup> adducts in ESI(+) mode.  In MS/MS instruments
phospholipids with their head groups show characteristic CID spectra allowing
identification.  For sphingolipid analysis (sphingolipidomics) neutral loss
and precursor-ion scans are used to identify lipids in samples as well as 
assess masses.  In ESI(+) precursor ion scans for <i>m/z</i> 184 
([C<sub>5</sub>H<sub>15</sub>NPO<sub>4</sub>]<sup>+</sup>) or neutral loss
scans for 59 amu (N(CH<sub>3</sub>)<sub>3</sub>) will identify 
[M+Na]<sup>+</sup> and [M+H]<sup>+</sup> ions corresponding to PtdChol and
lyso-PtdChol.  Precursor-ion scans for <i>m/z</I> 164 
([NaHO<sub>3</sub>POC<sub>2</sub>H<sub>4</sub>NH<sub>3</sub>]<sup>+</sup>)
can bue used to identify [M+Na]<sup>+</sup> and [M+H]<sup>+</sup> ions of
PtdEtOHNH<sub>3</sub><sup>+</sup>.  In ESI(-) neutral loss scans are done for
153 amu ([HO<sub>3</sub>POCH<sub>2</sub>C(OH)CH<sub>2</sub>] for identifying
precursors of Ptd-glycerol.  The peak <i>m/z</i> 184 is characteristic of 
sphingomyelin and ions <i>m/z</i> 264 and 266 are used for ceramide identification
in precursor scans.
<p>
Regio-chemistry of phospholipids is based on intensity of carboxylate anions
originating from <i>sn</i>-1 and <i>sn</i>-2 psoitions:  intensity of fragment
ion derived from <i>sn</i>-2 acyl group is greater than product ion from 
<i>sn</i>-1.  Branched saccharides of higher order sphingolipids (e.g.,
gangliosides) require MS/MS/MS experiments to look at structure.  Some class
of lipids can be analyzed in one go:  automated detection of over 800 phospholipid
species by direct injection ESI-MS/MS of cell extracts has been reported
using a lipid array (semi-quantitative data output) similar to DNA microarrays
(see Forrester et al, 2004).  Balazy (2004) recently reported on 
<q>eicosanomics</q>, the profiling of eicosanoids;  these derives from
arachidonic acid usiing enzymes like cyclooxygenase, lipoxygenases, 
cytochrome P450s, and oxidations not enzymatic.  LC or GC separation is
done to separate structural isomers, then MS-analyzed.  Several groups
have reported LC-ESI(-)-MS/MS studies (Margalit et al 1996, Newman et al
2002, Schmelzer et al 2005).
<p>
<b>Metabolic Fingerprinting</b>.  Fingerprints are described by their
data, which includes mass spectra data (<i>m/z</i> and intensities)
and by retention times when a chromatographic step is included.  These
data re then subjected to sample classification using multivariate data
analysis.  In many cases the metabolites are not identified (structures
not known).  Fingerprinting,  unlike profiling, refers to an <i>unbiased</i>
analysis of the entire detectable metabolome even if semi-quantitative
data are generated.
<p><b>Direct injection</b>.  This kind of fingerprinting involves no
prior separation by chromatography.  Samples are injected with ESI or
APcI, is high throughput, and samples analyzed in a few minutes.  
Short analysis time permites intersample reproducibility and accuracy
of the cluster analysis.  Smedsgaard et al (1996, 1997) examined fungal
extracts using ESI(+)-single quad MS.  Castrillo et al (2003) looked
at intracellular yeast extracts by ESI(+)-TQ MS.  Higher resolution TOF
and FTICR (rez = 6000-17,000 with 0.003-0.005 amu mass accuracy).
Q-ToF MS has been used for screening analysis of cell or plant extracts
(see Vaidyanathan et al 2002., Scholz et al 2004).  FT-ICR-MS has resolution
up to 10<sup>5</sup>-10<sup>6</sup> and mass accuracy down to 0.0001-0.001
amu, detection limits to femtomole to attomole range, and MS<sup>n</sup>
capabilities.  Aharoni et al (2002) infused directly crude ripening 
strawberry fruit extracts (MeOH:water 50:50 + 0.1% formic acid, or
100% MeCN) using ESI(+), ESI(-), and APCI(+) and APCI(-).  A total of 
5844 unique <sup>12</sup>C z=1 ions screened.  Hirai et al 2004 have used
FTICRMS for fingerprinting Arabidopsis to understand plant response to
nutritional stress.  Injections for ESI are done by continuous flow or 
by loop injection of 10-20 &micro;l sample at 5-50 &micro;l/min flow.
NanoESI has also been used and systems like Nanomate (Advion Biosciences)
are devised with liquid sampler and ESI chip of array of nanoelectrospray
nozzles (Scholz et al 2004).  Drawbacks:  Isobars may not be distinguished
for chemical isomers (need chromatography).  Also products might be
in-source fragments or adduct forms.  With ESI ion suppression is the 
big problem.  With many crude extracts, both signal suppression or 
enhancement can occur.
<p>
<b>LC-MS</b>.  Use of LC removes elements that cause ion suppression
resulting from coeluting compounds, isobaric interferences in case of
low resolving mass analyzers, and they usually separate isomers.
C18 reversed phase narrow bore columns with 3-5 &micro;m particles usually
used in metabolomics.  Use of monolithic silica-based C18 capillary used
in plant metabolomics (Tolstikov et al 2003).  Many samples require
better separation than can be achieved.  Smaller particle size 
(1.7 &micro;m) used in UPLC are now possible.  Sub-2 &micro;m particles
are very highly resolving and improve detection.  A 2.1 &times; 100 mm,
1.7 &micro;m C18 colum with gradient elution provides peak widths on 
order of 1.8 s (base width) in mouse urine, resolving 250 peaks in a
10 min period (Wilson et al 2005a, b).  Pressures of 10-15 kpsi are needed
to provide this separation.  RPLC provides separation of medium polar
and nonpolar substances, whereas polar elute with void.  Tolstikov and
Fiehn (2002) used HILIC for highly polar compounds.  HILIC is orthogonal
to RPLC and uses silica or derivatized silica, with groups being amino,
diol, amide, polysulfoethyl aspartamide, or polyhydroxyethyl aspartamide
with low-aqueous/high organic solvent systems.  Similar to normal phase
HILIC requires much water in mobile phase where polar analytes are partitioned
and retained.  In contrast to normal phase, HILIC uses water-miscible
polar organic solvents such as MeCN or MeOH.  Naidong (2003) has reviewed
HILIC in bioanalytical LC-MS/MS.  Triple quad or QToF enable MS/MS work
for structural breakdown of biomarkers.  Accurate mass measurement of 
precursor and product ions.  Ion traps allow MS<sup>n</sup> work.  MS/MS
can be used to probe metabolome for compound classes, characteristic ions
and neutral losses, for example CO<sub>2</sub>, 44 amu neutral loss in 
negative mode CID indicates R-COOH.
<p>
In doing LC-ESI-MS, both positive and negative ion mode should be done
for broad coverage.  Mass analysis can be done in a single run with positive
and negative mode detected simultaneously, but it depends on mass analyzer
used (Idborg-Bjorkman et al 2003).  This eliminates time of analysis
and any variability due to injection errors.  LC-MS fingerprinting has
been done for toxicity in urine (Idborg-Bjorkman et al 2003, Lafaye et
al 2003, Lenz et al 2004a,b; Williams et al 2005), genetic studies
(Plumb et al 2003a), plasma samples (Wang et al 2003), plant extracts
(Tolstikov &amp; Fiehn 2002, Tolstikov et al 2003).
<p>
<b>GC-MS</b>.  Before getting into the separation chemistry, it is 
necessry to note that the analytes usually require derivatization to make
them volatile.  Organic acids (R-COOH) and carbonyls (aldehydes and ketones)
are converted by oximation (see table)
<table style="font-size:83%;">
<tr><th>Derivatizing Agent  <th>Product  <th>Leaving Group in Reagent
<tr><td><i>N</i>-methyl-<i>N</i>-trimethylsilyltrifluoroacetamide (MSTFA)<br>
F<sub>3</sub>C&ndash;C(=O)&ndash;N(&ndash;CH<sub>3</sub>)&ndash;Si(CH<sub>3</sub>)<sub>3</sub>
 <td>trimethylsilylester<br> R&ndash;C(=O)&ndash;O&ndash;Si(CH<sub>3</sub>)<sub>3</sub>
 <td><i>N</i>-methyl-<i>N</i>-trifluoroacetamide<br>
F<sub>3</sub>C&ndash;C(=O)&ndash;N&ndash;CH<sub>3</sub>
<tr><td><i>N</i>-methyl-<i>N</i>-<i>tert</i>-butyldimethylsilyltrifluoroacetamide (MTBSTFA)<br>
F<sub>3</sub>C&ndash;C(=O)&ndash;N(&ndash;CH<sub>3</sub>)&ndash;Si(CH<sub>3</sub>)<sub>2</sub>&ndash;C(CH<sub>3</sub>)<sub>3</sub>
 <td><i>tert</i>-dimethylsilylester<br> 
 R&ndash;C(=O)&ndash;O&ndash;Si(CH<sub>3</sub>)<sub>2</sub>&ndash;C(CH<sub>3</sub>)<sub>3</sub>
 <td><i>N</i>-methyl-<i>N</i>-trifluoroacetamide<br>
F<sub>3</sub>C&ndash;C(=O)&ndash;N&ndash;CH<sub>3</sub>
<tr><td><i>N,O</i>-bis(trimethylsilyl)trifluoroacetamide (BSTFA)<br>
F<sub>3</sub>C&ndash;C[&ndash;O&ndash;Si(CH<sub>3</sub>)<sub>3</sub>]=N&ndash;Si(CH<sub>3</sub>)<sub>3</sub>
 <td>trimethylsilylester<br> 
 R&ndash;C(=O)&ndash;O&ndash;Si(CH<sub>3</sub>)<sub>3</sub>
 <td><i>N</i>-trimethyl-silyltrifluoroacetamide<br>
F<sub>3</sub>C&ndash;C(=O)&ndash;NH&ndash;Si(CH<sub>3</sub>)<sub>3</sub>
<p>trifluoroacetamide <br>
 F<sub>3</sub>C&ndash;C(=O)&ndash;NH<sub>2</sub>
<tr><td>O-methylhydroxylamine hydrochloride <br>
 CH<sub>3</sub>&ndash;O&ndash;NH<sub>2</sub>&bull;HCl
 <td>methoxime <br>
 R&ndash;C(&ndash;R&prime;)=N&ndash;O&ndash;CH<sub>3</sub>
 <td>HCl + H<sub>2</sub>O
</table>
<p>
These derivatizations give thermal stability as well.  Generally polar 
functional groups require derivatization.  Active hydrogens in -COOH,
-OH, -NH, and -SH are derivatized by alkylation, acylation or silylation.
An alkylsilyl group, usually trimethysilyl (TMS), is used to form ethers,
esters, sulfides, and amines.  There is better thermal stability with
silyl derivatives and a higher volatility and the MS spectra are more distinct
than with underivatized precursors.  BSTFA and MSTFA are predominant in
metabolomics.  BSTFA derivatization produces a volatile leaving group which
is vacuumed off before sample injection.  TMS acetamides from MSTFA and
very good for GC-MS.  1% trimethylchlorosilane is added as a catalyst.
Sample prep requires extensive drying prior to derivative use (water
easily interferes with reagent), and the derivative itself must not be
subjected to moisture or it degrades.  Side reactions include the 
conversion of Arg&rarr;Orn with BSTFA or MSTFA (see Halket et al 2005),
and Little (1999) has reported on artifacts complicating data interpretation.
<i>tert</i>-Butyldimethylsilylation (<i>t</i>BDMS) is used in fingerprinting.
EI-MS spectra using tBDMS derivatives have characteristic fragment
[M-57]<sup>+</sup> resulting from Si&ndash;t-butyl bond cleavage,
which has been helpful for identifying unknown metabolites (see
Fiehn et al 2000).  Birkemeyer et al (2003) have reported that tBDMS
derivatives are stable with 2% residul water in samples.  Carbonyl groups
are also derivatized into oximes with hydroxyl- and alkoyxamines.  This
stabilizes &alpha;-ketoacids and locks sugars into open rings.  The
derivatives are <i>syn</i> or <i>anti</i> isomers depending on orientation
of C=N bond.  These isomers are resolved on the GC producing 2 peaks.
Fiehn et al (2000) reports that alkoxyamines are superior to hydroxylamines
for identifcation purposes.
<p>
Broad coverage requires two-step derivatization, often methoximation followed
by silylation is used for GC-MS-based fingerprinting (three references
provided).  Gullberg et al (2004) used <q>design of experiment approach</q>:
plant tissues were extracted and derivatized for GC-ToF-MS.  In this
approach methoximation was varied as to reaction temp and time and followed
by silylation with MSTFA + 1% TMCS with silylation temp, reagent amount,
and cosolvent type (hexane, MeCN) studied.  They used 12 reference compounds
from different metabolite classes, and results showed that methoximation
with methoxyamine in pyridine for 17 h at room temp was optimal, and for
silylation MSTFA for 1 h at room temp was optimal overall.
<p>
Biofluid analysis typically uses lyophilization to remove the water, 
but matrix can interfere with subsequent steps.  Urea in urine samples
causes column and detector overload.  Fu et al (2000) added urease
to remove excessive urea prior to silylation and GC-MS.
<p>
Many GC-EI-MS work is done on quadrupoles (3 references) or ToF-based
analyzers.  ToF-MS use is more for fast analyte detection than high mass
resolution.  The LECO Instruments Pegasus ToF MS gives high data acquistion
rates at nominal mass accuracy.  However high res ToF MS is essential for
identification of unknowns.  By shortening GC column length and reducing diameters,
analytical times can be reduced from 40-60 min to &lt; 10 min.  Peak width are
narrow at 0.5-1 s and cannot be sampled by conventional scanning quadrupoles
(hence ToFs).
<p>
While capillary GC have high peak capacity, complex samples strain the resolution
of one column  In analyzing 1 ml human urine extract after silylation (MSTFA), 
deconvolution of chromatogram using AMDIS shows 1582 components, but many overlap.
To overcome this use of multidimensional GC (GC &times; GC) each peak from first
column (usu nonpolar) is loaded to second column for additional separation
to obtain a 3D chromatogram (two retention times, and peak intensity in 3rd
dimension).  MS data adds 4th dim.  A <b>modulater</b> is what connects to
GC columns to one another:  eluent from first column in taken off as a segment
and then focused in the modulator, usually by cooling in cryogenic modulators
with cryo-trapping using CO<sub>2</sub> or N<sub>2</sub>-cooled jets, focused,
and then rapidly heated to move into 2nd column as narrow band.  The use of a
focusing step reduces peak width of elution peak and decreases limit of detection.
Focusing alone without use of 2nd column increases peak capacity.  With a short
2nd column the 1st column fractions are separated within a few seconds.  Time
is important since trapping, release, and fraction separation must be finished
before next segment hits the modulator.  Resulting peaks quite narrow, widths
ranging from 80-600 ms (see Adahchour et al 2005).  In the MS dimension, because
peaks are narrow, a fast scan rate is required, and ToF used because its high
rez needed for identification.  New quadrupoles have scan speeds of 10,000 amu/sec
and are less expensive alternative for detection.
<p>
<b>CE-MS</b>.  Suitable for charged metabolite separation, very high analyte
resolution.  Not many reports in literature.  Soga et al (2003) developed
systems for anionic, cationic metabolites.  Also used nucleotides and CoA 
compounds, which provide comprehensive coverage.  Their MS was a single quad
with scan range of 30 amu, which maximizes signal for low abundance metabolites.
For covering total range for <i>m/z</i> 70-1027, 33 runs are required with 
16 h total analysis time for a single sample.  In a bacterial extract they found
1692 metabolites.  Sato et al looked at rice leaf metabolies using CE-MS and
CE-diode array detection (Sato et al 2004).  Capillary electrochromatography
(CEC) is a mix of LC and CE.  The capillary is packed with a stationary (often
monolithic) phase.  The mobile phase flow is caused by EOF, having plug-like
flow usually in CE (not parabolic), with highly resolved narrow peaks.
There is alos a pressure-assisted form of CEC, and CEC is interfaced to MS
(see Klampfl 2004).  Use of CEC in protein, peptide, amino acid and saccharide
analysis has been done.
<p>
<b>Flux Analysis</b>.   When measuring metabolite concentrations, what is 
measured is the steady state levels or snapshots of these concentrations.
These may not vary over time, even though the flux might change dramatically
(flux is the rate of reactant-to-product formation).  Thus steady state
concentration measurement does not show underlying biological processes and
describe phenotype (see Fig 1).  <b>Phenomics</b> is the <q>phenotypic analysis
of genomic information or entire mutant collections with goal of understanding
relationship between genes and higher levels of organization in the cell</q>
(Sauer 2004), and metabolism-wide flux measurements are a basis of this.
Flux analysis is done using isotope-labeled precursors, which were actually used
to determine biochemical pathways initially.  Accelerator MS (AMS) is used to 
quantify measurements in attomole range and monitor very low amounts of 
<sup>14</sup>C and other isotopes.  The in vivo flux of human folate 
(Lin et al 2004) and beta carotene (Dueker et al 2000, Lemke et al 2003) have
used AMS.  Buchholz et al (1999) have used HPLC-AMS to follow xenobiotics
such as atrazine.  AMS odes not elucidate structures of unknowns, but something
like chromatographic coeolution of labeled compounds with reference standard
can be used.  AMS is more suited to looking at flux in specific pathway
than for global flux analysis.  The setup is costly, and it is better to send
to a service or collaborate.
<p>
Use of <sup>13</sup>C or other stable isotopes can be detected with usual NMR
or MS, but their sensitivity is far less than AMS.  They provide structural 
information about labeled atom positions (positional isotopomers) and also
the number of labeled atoms (mass isotopomers).  MS provides better sensitivity
and higher throughput (Wittman 2002, Sauer 2004), and it detects all isotopic 
forms.  Special data analysis methods are required to interpret data.  Wittman 2002
has provided overview of flux analysis in MS.  GC-EI-MS is most often used
since fragmentation spectra provide isotopomer distribution information and
looks at metabolic fluxes.  The table below shows reports of metabolic flux
studies in various samples:
<table>
<caption>Metabolic flux studies</caption>
<tr><th>What Studied <th>Authors
<tr><td>microorganism <i>Phaffia rhodozyma</i> <td>Cannizzaro et al 2004
<tr><td><i>E. coli</i> <td>Fisher &amp; Sauer 2003
<tr><td>potato <td>Roessner-Tunali et al 2004
<tr><td>humans <td>Hellerstina et al 1997
</table>
<p>
 LC-ESI-MS/MS has been been used when nonvolatile ionics are studied or when
derivatization will or cannot be used (Mashego et al 2004, van Winden et al
2005).  MALDI-TOF MS has been used as well (Wittman &amp; Heinzle 2001) in
flux analysis.  MS-based fluxomics (Hellerstein 2004, Sauer 2004) is emerging
as a new field.
<p>
<b>Data Standards, Export, Analysis</b>.  Data handling and processing includes
noise reduction (background, smoothing), alignment of spectra and chromatograms.
Automation of feature identification (picking) as well as annotation of
MS and total-ion/extracted-ion metabolite chromatographic peaks.  One problem
is the numerous MS spectral data file formats.  ASCII (plain readable text)
is okay for low resolution or low volume MS data, but not high sampling rate
data in LC-MS applciations.  An XML standard has yet to be developed.  Pedrioloi
et al 2004 have specified the mzXML format, but this lacks descriptive info
for biological data.  
<a href="http://my.unidata.ucar.edu/content/software/netcdf/index.html">Network 
Common Data Form (NetCDF)</a> is binary format which includes storing MS data and most
instrument makers provide netCDF export options (Agilent's native format is
AIA and Finnigan's Excalibur is ANDI).  Libraries written in C, C++, Fortran,
Matlab, R, Java, and Python are available to access the netCDF format.
As for metabolomics analysis software there is COMSPARI (Katz et al 2004), 
XCMS (Siuzdak pers. comm. Scripps Res Inst.) or MZmine (Katajamaa et al 2006).
which can read netCDF.
<p>
Raw data usually stored as time-based MS spectra.  Continuous signal MS data
are processed to centroided forms to reduce spectral complexity before peak
identification.  Software handles both ion chromatograms and scanned (full spectra)
time points.  Data extraction starts with transforming MS spectra into vectors
of uniform length.  Binning of data is similar to that used in NMR metabolomics
(Holmes et al 2001, Viant 2003):  it is compatible to nominmal mass resolution
as no information is long and bin count is kept below 2000 with bin size = 1 amu
(rounding masses to integer values), with insertion of missing mass values 
and zero intensity assignment.  Binning of hi-rez MS data is disadvantageous because
small drifts in mass measurement can occur:  if accurate mass of metabolite is
close to a bin cutoff, a centroided peak from different data files may occur
in different bins by binning algorithm.  Noisy data and false positives without
sample randomization and where instrument drift is a big problem.  Continual
external calibration controls drift and enables instantaneous correction.  Drift
can be circumvented by increasing bin size can be increased to nominal resolution
which sacrifices high mass resolution.  Binning sizes too low produce huge
data sets although few bins have information;  signal splitting between bins is
also likely.
<p>
Metabolites usually have C, H, N, O, and some S, and P atoms.  H and N atoms
increase the decimal part while O, S, and P decrease it (their fractional masses
are on one side of the 0.500 line). O- and P-rich metabolites (like fructose
1,6-bisphosphate, C<sub>6</sub>H<sub>14</sub>O<sub>12</sub>P<sub>2</sub>)
should have a mass slightly less than nominal integer mass value.  Being small
molecules metabolite accurate masses will rarely exceed 0.4 amu in the decimal
portion for those majority of metabolites with masses below 500 amu.  Except for
heavy lipids, the optimal low resolution binning algorithm would be
[<i>m</i><sub>nom</sub> &minus; 0.1 amu, <i>m</i><sub>nom</sub> + 0.4 amu]
and [<i>m</i><sub>nom</sub> + 0.4 amu, <i>m</i><sub>nom</sub> + 0.9 amu].  In one
binning example using MarkerLynx (Micromass) to obtain metabolic fingerprints
of cell culture media in <i>m/z</i> 100-1200 range, 1008 metabolite accurate
masses were obtained to show a good histogram.  Scholz et al (2004) have dsecribed
approach to build a density function of masses weight by intensities by combining
all acquired spectra.  Masses at high intensity density are determined as maximum
of density function.  Mass intensity vectors transformed to uniform length means
entire dataset obtained by direct inject (single scan) analysis is represented
as 2D <i><b>m &times; f</b></i> matrix, where <i><b>m</b></i> is length of 
uniform mass vector and <i><b>f</b></i> is number of files.  When a separation 
step is included, multiple scans are added to the dimension, so it becomes
<b><i>m &times; s &times; f</i></b>, where <i><b>s</b></i> is number of scans
or retention time.
<p>
With this multidimensional data, classification schemes and statistical clustering
must ensue.  Principal components analysis (PCA) or parallel factor analysis
(PARAFAC) is suitable for 3D of data.  To simplify analysis, Plumb et al (2002)
have a system to sum mass spectra of chromatogram segments and convert to 
2D matrices.  These approaches are good for sample classificatin or discovery of
abundant metabolite peaks.  Detection of low abundance peaks in 3D data set may
be required for exact sample classification and for biomarker discovery (low
abundance type data).  All detected peaks in 3D data sets represented as
<i><b>n &times; f</b></i> matrix where <i>n</i> = #individual peaks detected in
all samples and <i><b>f</b></i> = count of files in datasets.  Peak detection
algorithms can be used to find metabolites at nominal resolution  (Windig et al 
1996, Koradi et al 1998, Hastings et al 2002)
<!-- ================================================================ -->
<p class="tags">
dimethylation; guanidination; dimethylation after guanidination; isotope 
labeling; liquid chromatography matrix-assisted laser desorption ionization; 
proteome quantification
<p class="ref">
 <span class="authors">
Ji C, Zhang N, Damaraju S, Damaraju VL, Carpenter P, Cass CE, Li L
 </span>
 <span class="year">
(2007)
 </span>
 <span class="jname">
Analytica Chimica Acta
 </span>
 <span class="jvol">
24,
 </span>
 <span class="pages">
219-226.
 </span>
<p class="reftitl">
A study of reproducibility of guanidination-dimethylation labeling and liquid 
chromatography matrix-assisted laser desorption ionization mass spectrometry for 
relative proteome quantification
<p class="Para1">
The Pierce Mem-PER kit was used to extract membrane proteins:  50 million cells
are washed with cold PBS, pelleted in 1.5 ml conical tube to which is added
150 &micro;l Reagent A to obtain homogeneous cell suspension by trituration.
Incubating 10 min with occasional vortexing, 450 &micro;l of Reagents B and C
in 1:2 proportion is added to lysed cells and again vortexed.  Tube is left on
ice fro 30 min with 5 min vortexing intervals, then spun max speed in microcentrifuge
for 3 min at 4&deg;.  The supernatant in a new tube is incubated 10 min at
37&deg;.  This is spun 2 min to get phase separation.  The bottom phase is 
retrieved.  A Bradford assay is done.  Then 500 &micro;g protein is digested
with trypsin (50 mM sodium bicarbonate pH 8.5, from 1 M stock) and enzyme added
10:1 (by mass?), with CaCl<sub>2</sub> added to 2 mM to prevent autolysis.
<p>
The cells in this case are CCRF-CEM human leukemia cells, representative of many
other cell lines.  Membrane protein enrichment is often done using a bouyant 
separation method, but the Pierce kit is quite effective at solubilization and
doing it in a shorter time.  It uses the Triton X series detergents that are
notorious for giving MS signals that are difficult;  in gel-based separations,
this is removed during second dimension separation.  Here, the TX-100 detergents
is removed in LC during SCX purification.
<p>
Three levels of variation can occur during these experiments.  Level 1 involves
technical variation with the membrane preparation.  A single culture of cells
is extracted and assayed, and the extract is divided into 4 tubes.  Level 2
variation takes the single culture of cells and divides them to 4 different
cultures and does 4 different extractions, each with its own protein assay.
Level 3 variation combines biological and technical variation:  three individual
batches (cultures of cells) are grown under identical conditions, and one batch
is used as a control and divided into two portions.  Thus for extractions are
done with four differetn protein assays.   There is no separation into distinct
samples at later steps (trypsin digestion, guanidination).  For each of the 
four tubes from each variational Level, tubes being a, b, c, d, tubes a and
b get light isotope dimethyl labeling, and c and d get heavy isotope dimethyl
labeling.  Tubes a and c, then b and d are mixed and analyzed using 1D
microbore LC-MALDI QqTOF MS.
<p>
All results show CV values of &lt; 15%, quite good for quantitative bioassays.
The only variation possible in Level 1 is at the stages of tryptic digestion,
isotope labeling, and the LC-MS analysis.  Level 2 variation contains Level 1
vation + variation associated with membrane fractionation.  Level 3 variation
includes the technical and some biological variation.  The relationship
<p class="eqnserif">
<i>s</i><sup>2</sup><sub>total</sub> = <i>s</i><sup>2</sup><sub>technical</sub>
+ <i>s</i><sup>2</sup><sub>biological</sub>
<p>
where <i>s</i><sub>total</sub> is tthe standard deviation calculated at level 3 
and <i>s</i><sub>technical</sub> is standard deviation calculated at level 2. 
The dataset at each level has duplicates, and so the pooled standard deviation 
from both levels 2 and 3 are used to calculate biological variation.  The value 
of <i>s</i><sup>2</sup><sub>biological</sub> = 0.0038 which is less than 
<i>s</i><sup>2</sup><sub>technical</sub> = 0.0174.  Thus the biological variation
is less than the technical variation.  In order to assess the biological variation,
it is necessary to reduce the technical variation.

<!-- ================================================================ -->
<p class="tags">

<p class="ref">
 <span class="authors">
Morris JS, Clark BN, Gutstein HB
 </span>
 <span class="year">
(2008)
 </span>
 <span class="jname">
Bioinformatics
 </span>
 <span class="jvol">
24,
 </span>
 <span class="pages">
529-536.
 </span>
<p class="reftitl">
<i>Pinnacle</i>:  A Fast, Automatic and Accurate Method for Detecting
and Quantifying Protein Spots in 2-Dimensional Gel Electrophoresis Data
<p class="Para1">
The Pinnacle method is described as follows:
<h1>1. Compute the average gel</h1>
<p>
The average gel is used for pinnacle detection, not a raw gel.  Construct
the average gel by averaging intensities pixel-by-pixel across all gels
in the experiment.  In other software a composite gel is made as a
representation of detected spots rather than pixel-wise averaging.
No background correction is done in computing the average gel.
<h1>2. Denoise the average gel using wavelet shrinkage</h1>
<p>Wavelet denoising is used to remove white noise from the image; it has been
routinely used in signal processing.  This smoothes out small irregularities
but retains larger signals that represent true spots.  Removal of the
irregularites reduces the count of computationally detected false positive
spots.
<p>
The <b>undecimated discrete wavelet transform</b> (UDWT) is used, as
implemented by the <a href="http://www.dsp.rice.edu/software/rwt.shtml">Rice
Wavelet Toolbox v 2.4</a>.  Denoising is done in three steps:
<ol>
<li>Compute coefficients for the average gel, given a choice of wavelet basis.
The coefficients represent frequency-location decomposition in both dimensions
of the image.  The dyadic wavelet transform (DDWT) is more efficient, but
UDWT gives translation-invariant results which means denoising is the same
even if the image is shifted or cropped in either dimension, the denoising
being more effective.  Results are rather dependent on wavelet basis:
by default, the Daubechies wavelet with four vanishing moments is used.
<li>Hard thresholding is applied to the wavelet coefficients:  this
sets all coefficients below a threshold &phi;=&delta;&sigma; to 0,
leaving coefficients &gt;&phi; unaffected.  The parameter &sigma;
is the robust estimator of the SD (see <a
href="http://dx.doi.org/10.1109/83.862633">Donoho and Johnstone</a>)
in using the median absolute deviation for highest frequency wavelet coefficients
divided by 0.6745, and where &delta; is a threshold parameter specified by
user and where this parameter controls more or less de-noising.  Values for
&delta; from 5-20 work well for MALDI-MS, and for 2D gels where background
white noise is not as strong, smaller values are better;  a default of
&delta;=2 is adequate.
<li>The denoised signal is reconstructed by applying an inverse UDWT to
	the threshold wavelet coefficients.  Thresholding works because white
	noise is equally distributed among all wavelet coefficients, while the
	signal is focused on a small number of coefficients.  Thresholding zeroes
	out the large number of wavelet coefficients of small magnitude which
	correspond mostly to noise, leaving the small count of coefficients of
	large magnitude representing signal.
</ol>

<h1>3. Detect <i>pinnacles</i> on the wavelet-denoised average gel</h1>
<p>
With de-noising completed, spot detection is done on wavelet-denoised average
gel using <b>pinnacles</b>.  A pixel location contains a pinnacle if it is
a local maximum in the horizontal and vertical dimension and with intensity
greater than a threshold which by default is the 75th percentile of the gel.
A list of pixel coordinates is produced marking the pinnacles in the average
gel that index <q>spots</q> of interest in the given gel set.

<h1>4. Combine any pinnacel within a specified proximity</h1>
<p>
If pinnacles are found within a given 2<i>k</i><sub>1</sub> + 1 &times;
2<i>k</i><sub>1</sub> + 1 square surrounding another pinnacle, pinnacles will
be combined with one that has the highest intensity, removing spurious double
peaks, and will accommodate imperfect alignment.  It is rare to see two
pinnacles which are &le; 5 units from each other with a particular scanner
resolution.  With a 1024 &times; 1024 image, a <i>k</i><sub>1</sub> = 2 is
used.

<h1>5. Quantify each spot for each gel by taking maximum intensity within
spec
ified neighborhood of the pinnacle in the average gel</h1>
<p>Each spot is now quantified for the gel by taking the maximum intensity
within the 2<i>k</i><sub>2</sub> + 1 &times; 2<i>k</i><sub>2</sub> + 1 square
formed by taking the corresponding pinnacle location in the average gel
and extending out &plusmn; <i>k</i><sub>2</sub> in the horizontal and verical
dimension in the individual gel.  Width <i>k</i><sub>2</sub> should be as
small as the proximity <i>k</i><sub>1</sub> in the previous step.  By default
<i>k</i><sub>2</sub> = <i>k</i><sub>1</sub>.  This permits finding the maximum
pinnacle intensity for a corresponding spot even when the individual gel
alignment is not perfect.  Alignment accuracy is thus determined by spots being
&plusmn; <i>k</i><sub>2</sub> within the horizontal or vertical dimension.
<h1>6. Apply background correction filters and normalize spot quantifications</h1>
<p>Background correction and normalization is done in the final step.  A global
minimum intensity representing the entire gel is sufficient when the background
appears relatively uniform;  if it appears to vary spatially, a windowed
minimum is then used.  Because pinnacle intensities rather than spot volumes
for quantifications, background is estimated only for pixel locations with
pinnacles and the calculation proceeds quickly.  Default window is &plusmn;
100 pixels in horizontal and vertical.  The window must be large enough to
extend beyond each spot region and avoid attenuation of quantified pinnacle
intensities.
<p>
Normalization is done by dividing each pinnacle intensity on a given gel
by mean pinnacle intensity for the given gel.  Wavelet de-noising can be
done to individual gels before quantification; generally it is not done
because it makes little difference in result.
<p>
With <i>N</i> individual gels and <i>p</i> spots, a matrix <i>N</i> &times;
<i>p</i> protein expression levels with no missing values remains.
During profiling or group comparison studies, the matrix is analyzed to find
which of the <i>p</i> spots is associated with factors of interest and worthy
of further study.
<p>
Pinnacles performance was compared with Progenesis and PDQuest both using
a dilution series (Nishihara and Champion, 2002) and group comparison study.
Reliability of spot quantifications was assessed by measuring linear
association (<i>R</i><sup>2</sup>).  Methods that yield more accurate
protein quantifications should result with a <i>R</i><sup>2</sup> closer
to 1.  Within different dilution groups the CV of spots in determined to
assess precision.
<p>
The Nishihara and Champion dilution series is a sample of E. coli loaded
on 2D gels over a 100-fold range:  0.5, 7.5, 10, 15, 30, 40, and 50 &micro;g.
Four replicate gels are run for each load.  20 spots were selected initially
but were found insufficient in distinguishing the abilities of the software.
Results for the 20 selected spots for the PG and PDQ software compared with
the Nishihara and Champion work.
<p>
SH-SY5Y neuroblastoma cells were grown to 60-70% confluence and harvested.
Cells were extracted with Sigma ProteoPrep total extraction kit, ultrasonicated
in 15 s bursts at 70% amplitude for 1 min.  The extract is centrifuged at
15000 <i>g</i> for 30 min at 15&deg;.  Samples are treated for 60 min at RT
with tributylphosphine added to 5 mM, then for iodoacetamide added to 15 mM
for 90 min in dark.  11 cm IPG (BioRad) strips in 100 &micro;l rehydration
buffer for 2 h RT.  Samples in 150 &micro;l are applied to strips and focused
up to 100 kVh.  This was done in triplicate strips at 6 different load
levels: 5, 10, 25, 50, 100, 150 &micro;g.  Strips are soaked in 2.5% SDS
in 50 mM Tris-acetate pH 7.0 for 20 min, then run on 8-16% gel.  Proteins
are illuminated with SYPRO Ruby after fixing in 10/7 for 30 min followed
by overnight staining.
<p>
6 male Sprague-Dawley rats were implanted with 75 mg slow-release morphine
pellets or placebo subcutaneously under isoflurane anesthesia.  Tolerance
is monitored using tail flick latency daily.  After 5 d, animals are sacrificed
and spinal cords harvested.  Proteins from the substantia gelatinosa region
(dissected by a transillumination method) are extracted and run on 2D.
<p>
All gel results are analyzed using Progenesis PG240 v 2006, PDQuest
ver 8.0 and the Pinnacle method.  Images are first aligned with TT900
(Nonlinear) and pinnacle detection and quantification is done on MATLAB
(v R2006a with XP-based PC.  Progenesis and PDQuest is done on unaligned
gels.
<p>
For Progenesis the Analysis Wizard is used, with grouping by protein load,
and same gel selected for reference gel for aligned and unaligned image sets.
Background subtraction is done and then combined warping and matching
is done for unaligned images, with property-based matching selected for
aligned images.  Normalization is not done:  it eliminates linearity of
quantifications with protein load used to evaluate reliability.  Minimum
spot volume set to 1, and split factor set to 9.  It produces a similar
number of spots to those reported by Nishihara and Champion.  No manual
editing is done and data are exported to Excel and spots present in 3 of
the 4 replicates for Nishihara &amp; Champion and in 3 of 3 replicates of the
SH-SY5Y dilution series
are determined.  Spot volumes of zero are used for spots present on other
gels with no match on the current gel.
<p>
For PDQuest gels are processed through the Spot Detection wizard.  Grouping
is done by protein load, with default background and match settings applied,
and the same master gel is selected for both aligned and unaligned images,
which is the same gel used as the top reference gel in Progenesis.
The 'give manual guidance' and 'test settings' featuers are used for the
advanced spot detection wizard.  The speckle filter and vertical and
horizontal streak filter in Advanced Controls was also used.  A similar
number of spots as reported by Nishihara and Champion were found.  Spot
volumes were not normalized and manual editing was not done.  Spots are
exported to Excel and present in 3 of the 4 Nishihara &amp; Champion and
3 of 3 replicates of SH-SY5Y dilution series were evaluated.
<p>
Total count of spots were summarized for each method and dilution series.
Unmatched spots were those present on only one gel and not matched to any
spot on any other gel.  Pinnacle had no unmatched spots since its concept
requires quantifications of every pinnacle on every gel.  Unmatched spots
are removed from the Progenesis and PDQuest evaltions in quantitative summaries.
Also removed are spots not present in at least 3 of 4 replicate gels for
at least one of the protein load groups in the N&amp;C study or in 3 of 3
replicates in the SH-SY5Y cell dilution series.
<p>
Matching percentage for Pinnacle applied to aligned gels is 100%  For PDquest
and Progenesis, 10% of the total number of spots is randomly selected that
meets the criteria, and then a count of spots matched by automatic algorithms
on all individual gels for which it was detected to the spot on reference gels.
The statistics here only deal with matching and not spot detection errors as
gels for which a given spot was not at all detected did not count as a mismatch
in the match percentage.  Incorrect spot splitting is also not considered as
mismatch; this is matching a spot in one gel to the same spot and an adjacent
one detected as one spot in another gel.
<p>
The mean spot quantification across replicates is computed for each protein load
group versus the true protein load and the coefficient of determination (<i>R</i><sup>2</sup>)
is calculated from simple linear regression.  If a correlation (<i>R</i>) is
negative, then <i>R</i><sup>2</sup> is set to zero.   This measure was done for
all detected spots rather than just a select set since it is a realistic assessment
of performance of each method across the entire gel.  The <i>R</i><sup>2</sup>
is summarized for all spots in a gel by the mean, a five-value percentile (5th
or Q05, 25th or Q25, median or Q50, 75th or Q75, and 95th or Q95). and by counting
the number of <q>reliable spots</q>, which are spots with <i>R</i><sup>2</sup>
&gt; 0.90, which is a correlation of at least 0.95 between group mean spot
quantification and protein load.  Thus reliable spots means spots that provide
a sense that are well quantified by the method.
<p>
Precision is computed at %CV for each spot detected in entire gel set across
gels within each protein load group.  In group mean comparisons, the two-sample
<i>t</i> test with variances assumed to be unequal are done for each detected
spot and the <i>p</i> values assessed.  Also summarized are the number of spots
with <i>q</i> values &lt; 0.10, which is a measure of the local false discovery
rate and estimates the probablility that a given spot is a false positive if
it is called significant.
<p>
Pinnacle detected 1403 spots in the N&amp;C panel, found and quantified for all gels
(none missing, by definition).  PDQuest detected 2692 spots of which 745
were unmatched (found only on one gel).  571 spots were found on more than one
gel but not included in analyses since they were NOT found on 3 of 4 gels for
at least one group, which is the N&amp;C exclusion criterion.  The percentage
of matches of the 1376 spots found on 3 of 4 gels in at leaast one group is thus
60%.  Progenesis found 1986 unique spots
<!-- ================================================================ -->
<p class="tags">

<p class="ref">
 <span class="authors">
Pretzer E, Wiktorowicz JE
 </span>
 <span class="year">
(2008)
 </span>
 <span class="jname">
Anal. Biochem.
 </span>
 <span class="jvol">
374,
 </span>
 <span class="pages">
250-262.
 </span>
<p class="reftitl">
Saturation Fluorescence Labeling of Proteins for Proteomic Analyses
<p class="Para1">
A criticism of 2D gel staining methods is that stains have small dynamic
range with low sensitivity and poor reproducibility, possibly because of
variation in reactivity of proteins with noncovalent dyes, precipitation
at pI, incomplete solubilization by SDS, variation in gel composition and
quality, among other factors.
<p>
Labeling techniques prior to protein separation and analysis are generally
either isotopic or fluorescent.  Cell cultures make use of metabolic labeling
via stabile isotope labeling of amino acids (SILAC) and the isotope tag
for relative and absolute quantification (iTRAQ).  In preparing the proteins
for MS, they must be digested to produce ions in the optimal mass range of the
instrument; the drawback is that low abundance proteins may be lost or
fall below detection limits of even a MS.  Relative quantification of proteins
is done at the analytical stage (MS) and not at the resolution/separation
stage (chromatography).  Use of the MS requires that even proteins not of
interest be included in the data analysis rather than proteins of interest
filtered through a previous separative step.
<p>
Derivatizing peptides/proteins with a sensitive (e.g. fluorescent) label
is purposeful because of quantitative accuracy and sensitivity.  DIGE and
saturation labeling are examples.  The approaches permit multiplexing and
internal standard use, detection of low abundance proteins.
<p>
BODIPY FL-Mal and BODIPY FL-Iaa are the maleimide and iodoacetamide forms
of the BODIPY flurochrome, respectively, which makes BODIPY thiol-reactive.
BODIPY is a special complex of fluoride-coordinated boron with a diaza-indacene
core providing the complex.  In a prior study, it was shown that thiourea
inhbited the alkylation of thiol groups with BODIPY TMR cadaverine Iaa.
Alternatively Rhodamine Red C2 maleimide and BODIPY FL-Mal were efficient
in the presence of thiourea.
<p>
A mixture of &alpha;-lactabulum (with 8 thiols) and enolase (1 thiol) was
treated with BODIPY FL-Iaa or FL-Mal in 8 M urea or in 7 M urea/2 M thiourea,
the usual conditions for preparing for IEF, at two pH values, 7.2 and 8.0.
Dye/thiol ratios were 10:1 (2.25 mM dye/225 &micro;M thiol).  2 mM TCEP is
present for reduction (0.88:1 reductant/dye ratio).  The B FL-Mal form labeled
in both urea and urea/thiourea and at both pH values.  Bright band fluroescence
is see along with a slight size shift in the &alpha;-lac band.  In contrast
the presence of thiourea severely inhibited the B FL-Iaa form, no labeling
at pH 7.2 and very little at pH 8.0.
<p>
Reducing agent levels must be optimized as well to prevent their interaction
with the thiol-reactive dyes.  A protein with no thiols and &alpha;-lac
<!-- ================================================================ -->
<p class="tags">

<p class="ref">
 <span class="authors">
Rubina AY, Kochinsky A, Makarov A, Zasedatelev AS
 </span>
 <span class="year">
(2008)
 </span>
 <span class="jname">
Proteomics
 </span>
 <span class="jvol">
8,
 </span>
 <span class="pages">
817-831.
 </span>
<p class="reftitl">
Why 3-D?  Gel-Based Microarrays in Proteomics
<p class="Para1">
<b>REVIEW.</b>
Mirzabekov developed a new generation of gel-drop biochips as one concept
in protein (micro)array technologies.  Factors involved in microarray
production differ for nucleic acids.  Choice of substrate, nature of target
(analyte), required density and sensitivity, methods of registration, cost,
and so on.  Many popular products employ chemically modified or coated
2-D surfaces with fluorescent labeling as part of analyte detection.
<p>
Immobilization media come in four types:  <ol><li>slides coated with polyacrylamide
gel (PAAG) or modified PAAG <li>agarose slides <li>NC-coated slides (e.g.,
FAST slides at whatman.com) <li>SuperProtein slides coated with hydrophobic
polymer (proprietary from Telechem International at arrayit.com).</ol>
<p>
Proteins are hydrophilic molecules for the most part and so a 3-D hydrophilic
environment is more suitable.  Guschin et al (1997) first reported use of
pre-formed polyacrylamide gel in a biochip. Arenkov et al advanced on this.
These were produced using complicated molds, later changed to use of pin
dispeners to produce hemispherical drops.  Acrylamide was replaced by methacrylamide
which made gel stronger, heat-resistant, and more porous.  Polymethacrylamide
gel elements can undergo 30-40 standard PCR cycles, and pore size can accomodate
the 150 kDa mass of the IgG antibody by passive diffusion.  Photo-induced
copolymerization immobilizes proteins of interest:  proteins are modified first
using an unsaturated moiety that participates in free radical polymerization
reactions.
<p>
Our group contributed the idea of incorporating the unsaturated moiety into
protein prior to polymerization with <i>N</i>-hydroxysuccinimide ester of
6-(<i>N</i>-methacryloyl)-aminohexanoic acid  or any other activated ester
of non-saturated carbonic acid.  In the 2nd approach, modification of protein
and  gel polymerization are done simultaneously:  proteins are modified on
amine or thiol groups with bifunctional gel-forming monomers in course of
polymerization.  Confocal microscopy confirms that immobilized molecules
are uniformly distributed in 3-D space.  Immobilization efficiency depends
on the nature of the protein and the extent of modification of methacrylate
groups, and varies from 40-75%.
<p>
3-D gel elements as hemispherical drops are arrayed on a hydrophobic surface
with drop size varying from pin diameter (50-500 &micro;m).  One example of
an array is shown with gel drops in 0.1 nl made through 100 &micro;m pin.
Typical elements will contain 100 fmol protein immobilized in 200 &micro;m
element.
<p>
Commercially product gel-coated slides show high variability in response.
With the technology described here, intra-array variation is not greater than
5%, while inter-array variation is about 10%.
<p>
Sodium periodate (NaIO<sub>4</sub>)-activated agarose is capable of nucleic acid
hybridization and protein immobilization, although used mostly for immobilizing
oligos for hybridization.  Proponents indicate they get better results than
with commercial Hydrogel-coated slides.  Sandwich immunoassay results showed
linear responses for a wide range of concentrations of immobilized Abs, and
intra- and interassay signal variation was under 2 and 9%, respectively.
They are easy and inexpensive to produce and compatible with many types of
arraying and recording equipment.  Drawbacks include fragility and incomplete
separation of probes.  Shelf life was not tested beyond 2 months, so that needs
to be determined.
<p>
Hydrogel-coated slides show the best overall performance in the commercial area,
with high LOD, S/N, and sensitivity.  However, intraslide variability is
20-40% and interslide variability is 60-100%.  Minimum spot size is 0.2 mm,
and so spot density limitations are a factor.  Probe penetration is by passive
diffision, and so incubation times are determined based on probe size; usually
distribution of probe in spot is uneven.  In another study comparing different
substrates, changes in factor inputs/variables had a huge impact on assay
performance, particularly with the nature of antibody-antigen pair.
<p>
Because of these problems, a model/benchmark test using prostate-specific
antigen (PSA) was chosen for a sandwich immunoassay using gel-based and surface
microchips.  Maximal density of anti-PSA antbodies on a 2-D surface is
about 30,000-40,000 molecules per &micro;m<sup>2</sup>. Maximum density
in a 3-D gel element is 5-50 million per &micro;m<sup>2</sup>.  Maximal
<i>working</i> density is in reality much lower because of kinetic considerations,
but remains an order of magnitude higher than for 2-D-based arrays.  Signals
of saturation are 8 times in sandwich-type assays and 13 times higher in direct
assays for 3D vs. 2D arrays.  Note that the fluorescence signal vs. probe
concentration [PSA] plot is a typical curve of the type, signal = [probe] / (K +
[probe]), where K is a suitable constant related also to the capacity or
concentration or mass of immobilizing antibody present.
<p>
<b>Chip purpose and assay design</b>.  The goals in producing some gel-based
chips is on identifying and quantifying certain proteins. <b>Forward phase</b>
assays make use of captured proteins (in most case, antibodies) immobilized to
chip element and multiples targets are screened.  In <b>reverse phase</b>
arrays, complex protein mixtures are immobilized and then screened by applying
a specific probe.
<p>
Functional assay designs typically immobilize the protein-binding molecule
(nucleic acids, oligosaccharides, small ligands) to the element and then
test interaction with proteins in solution.
<p>
All sorts of designs have been tested.  Immobilized antibodies testing antigens
in solution are typical, but immobilizing (protein) antigens and staining with
antibodies using direct and sandwich type methods have been done.  Ricin
and anti-ricin antibodies were tested by immobilizing the antibodies to gel
elements using chemically modified proteins with simultaneous copolymerization,
or by using biotinylated antibodies with gel-immobilized avidin.  An LOD
of 0.1 ng/ml was seen for both approaches.
<p>
Rubina et al found that HRP immobilized to gels retained 70% of its control
activity.  Barnase is a extracellular ribonucleoside-3' transferase with strong
preference for G-containing sequences, and the chip-immobilized protein shows
good enzymatic activity and substrate specificity:  a Ribo-G oligonucleotides is
completely hydrolyzed at room temp in less than 1 min, wherease a ribo-U
oligo took several hours.  Stability of long-term storage of the PSA biochip
shows signals 1 year later as good as they were at time of manufacture.
<p>
The assay is done by getting raw (fluorescence) signal data and substracting
background signal values from control gel spots.  Replicates should be
quadruplicates, and the median value of the signals is determined.
Statistical validity requires that the threshold be set 3-times above background.
For the reader in use, this is 0.3 attomol per gel element, and its dynamic
range is 4000:1.
<p>
Binding and/or dissociation can be registered in real time (continuous monitoring
of fluorescence?).  Only surface plasmon resonance allows real time determination
of binding/dissociation, allowing a kinetic analysis.  The drawback with SPR is
that it requires continuous flow of analyte on surface.
<p>
<b>Diffusion of proteins in gel drop elements</b>.  Computations of diffusion
coefficients indicate that proteins move in 5% PAAG with 2.7% cross-linking about
one-third the rate they would diffuse in water.  Improvement can be done by
using microelectrophoresis or forced constant flow (using peristaltic pump).
Experimental results show a 5-fold improvement using insulin and its antibody.
<p>
Tumor markers approved by the FDA for diagnostic and treatment monitoring include
PSA, alpha-fetoprotein, CA-125, CEA, CA 19-9, CA 15-3, NMP 22, and to a limited
extent chorionic gonadotropin.  Multiplex assays could reveal that combinations
of markers may be clinically informative.  Mor et al compared 169 proteins in
sera of healthy women and patients with ovarian cancer and identified four
which enabled a sensitivity of 95%, PPV of 95%, specificity of 95%, and NPV of 94%
when used in multiplex assay.  Both microarray and ELISA platforms showed a concordance
between the two.  For screening, a single gel biochip was developed to carry mAbs to
6 markers (later two were added).  One marker CA 125 at 200 kDa showed good penetration
under confocal.
<p>
PSA is a hopeful screening tool although in many cases there are false positives
and in rare cases where cancer occurs, PSA levels do not increase.  It circulates
in serum in a free form and bound to &alpha;<sub>1</sub> antichymotrypsin.  In
malignancy, total PSA is elevated with the fraction of free-to-total decreased.
PSA biochips composed of low percentage gel with low level of crosslinking were done.
mAbs are immobilized to be specific for both total and free forms.  Serum is added
with developing anti-PSA fluorophoric antibodies, the latter binding in sandwich
with solution PSA and unlabeled anti-PSA antibodies in gel elements.  There is also
a direct mode with immobilized PSA antigen.  Control elements must contain no
immobilized protein to guage medium background.
<p>
<b>Chip manufacture and quality assurance</b>.  Batches of 500 chips are inspected
by light and rejected if diameter of gel elements exceeds 5%.  Gel element size
between chips is evaluated and lots rejected if chip size CV exceeds 8%.  Gels are
treated with a blocking agent to minimize nonspecfic binding and covered with chambers
to allow a 50 &micro;l volume.  Reaction is 17 h (overnight).
<p>
Whatman PLC has produced its CombiChip Autoimmune that examines collagenosis and
vasculitis caused by several different auto-antibodies.  They are built on FAST
slides (NC-coated).  A slide has 16 pads (identical arrays), and each array has
14 different auto-antigens and internal calibrator with human IgG.  A kit has
4 slides, 4 incubation chambers, lyophilized labeled anti-human IgG, 6 ml blocking
buffer, 10 ml of 10X wash buffer.  It is received CE certificaition by EU for safety,
health, and environmental requirements without assessment of quality or clinical
relevance of device.
<p>
The PSA kit is approved for clinical applications by Federal Agency for Supervision
of Health Care and Social Development of the Russian Federation after a multicenter
clinical trial.  It is a one-patient one-chip principle and its costs is within the
cost of testing per sample using traditional ELISA.  It includes the ability to
calibrate the response without use of external reagents.
<p>
<b>Biotoxins</b>.  Toxins from many sources are of interest in testing for human
disease and injury.  Rubina et al (2005) developed a biochip to detect 6 protein
toxins (ricin, viscumin, staphylococcal enterotoxin B, tetanus toxin, diphtheria
toxin, and anthrax lethal factor) using immobilized antibodies.  The chip gel
element was sufficiently porous to allow for even distribution of input proteins.
Association constants in the chip could be estimated and were close to those in
solution:  thus even Cy3 modification does not alter structure required for binding
significantly.  Highest sensitivity was with sandwich approach using directly
labeled antibodies.  Dynamic range was 3 orders of magnitude, not possible with
2D arrays or ELISA.
<p>
<b>Microarray sensitivity</b>.  Immunoassays are essentially <i>ambient analyte</i>
or <i>mass-sensing</i> assays.  Ambient analyte types measure the analyte as a
proportion or concentration independent of its volume.  With mass-sensing, typical
being a well in an ELISA plate, all the target molecules are captured in the sample
and the result is expressed as an amount (mass) of the analyte.  The former approach
has the greater sensitivity, and the smaller the array element, the higher the
sensitivity.  Strategies to increase sensitivity include forced flow, but when the
spot/element size gets smaller, the increase in diffusion is quite small.  Better
strategies are to improve S/N which includes use of very high quality proteins
(purity) which are to be immobilized.  Another is to use better higher-resolution
imaging technology when measuring the signal.
<p>
<b>MALDI-MS analysis</b>.  Rubina produced an array of gel elements having a
400-1000 &micro;m diameter and 3 mm pitch on hydrophobic glass.  Bound proteins
and products of an enzymatic reaction are extracted from gel and mixed with matrix
before MALDI-MS.  In another case, the gel is infused with trypsin and digest
products analyzed; protein was identified using the PeptIdent program.
2D arrays could not produce enough material for analysis whereas 3D biochips could.
<p>
Biochips with complete combinatorial sets of oligonucleotides can be developed.
In standard microarray production, oligonucleotides are synthesized on the substrate
or pre-synthesized oligos are immobilized.  Each has their advantages and disadvantages.
<!-- ================================================================ -->
<p class="tags">

<p class="ref">
 <span class="authors">
Dowsey AW, Dunn MJ, Yang G-Z
 </span>
 <span class="year">
(2008)
 </span>
 <span class="jname">
Bioinformatics
 </span>
 <span class="jvol">
24,
 </span>
 <span class="pages">
950-957.
 </span>
<p class="reftitl">
Automated image alignment for 2D gel electrophoresis in a high-throughput
proteomics pipeline
<p class="Para1">
The Human Proteome Organization (HUPO) conducted Brain Proteomic Project pilot
studies comparing the MDLC-MS/MS with 2-DE based methods.  Only 57% of proteins
were detected by 2-DE and 29% by LC-based methods.  Both techniques are vital
and so it was concluded that a high throughput pipeline that automatically
aligns, models and differentially quantitates.  Dowsey et al (2003) reviewed
the use of bioinformatics for 2-DE.  In general pre-processing starts with
noise reductiona and background subtraction.  Each protein spot is then resolved
in two steps:  gel segmentation using watershed transform before each segment
is fitted with parametric spot model to separate co-migrating spots.  Then two
or more gels are compared for differential expression by point-pattern matching
their spot lists with metrics like spot volume.  Statistical testing is then
do with significant manual validation (Student's t test, ANOVA) to find outliers
representing up- and down-regulated proteins.  Some reports (Wheelock and Buckpitt)
show marked difference in performance.
<p>
If the only focus is on raw pixel values, features (spot shape, streaks, smears,
spot tails, and background) might be lost in spot detection phase.  The problem
is a quadruple, where source image <i>I</i><sub>s</sub> is brought into
alignment with reference image <i>I</i><sub>r</sub>, constrained by <i>M</i>,
the space of allowable transformations (mappings) on <i>I</i><sub>s</sub>, and
guided by optimization of a similarity function which is at its maximum when
the alignment is optimal:
<p class="eqnserif">
arg max<sub style="vertical-align:-1em;margin-left:-3em;"><i>m</i>&isin;<i>M</i></sub>
&nbsp;sim(<i>I</i><sub>s</sub> &#9675; <i>m</i><sub>&theta;</sub>,
<i>I</i><sub>r</sub>)
<p>The system Z3 (Smilansky, 2001) was the first hybrid 2D gel alignment
algorithm incorporating pixel info, comparing rectangular image regions each
having a few detected spots.  Following this was the direct algorithm
Multi-resolution Image Registration (MIR) which did not make use of spot
detection.  These register control points of a piecewise bilinear mapping
with the Broyden-Fletcher-Goldfarb-Shanno (BFGS) quasi-Newton optimizer and
closed form derivatives of the normalized cross-correlation (NCC) similarity
measure.  Convergence is robust since gels have smooth image gradients and
thus the NCC with respect to control points is smooth and continuous.
A multi-resolution approach is used to avoid convergence to local optima.
In comparing MIR and Z3, MIR scored better 29 of 30 times.  A leakage model
was used to extend the concept and compared favorably with PDQuest.
Multi-resolution with complex wavelets has been proposed because it has
 a realistic deformation model based on tensor-product B-splines, though
no compartive validation was done in either case.
<p>
The Robust Automated Image Normalization (RAIN) algorithm is intended to
answer certain questions, such as removing systematic geometric deformation
and expression bias in image domain for full automated aligntment.  The main
steps in RAIN are:
<ol>
<li>A smooth tensor-product B-spline surface is used to warp source gel
until all features are brought to correspond with those in reference gel.
Volume-invariance is used to ensure expression is not amplified under dilatation,
not attenuated by contraction.
<li>Simultaneously, additive and multiplicative bias field (modeled as
smooth B-spline surfaces) are fitted between images to account for systematic
regional changes in background and staining.  Thus alignment is guided by
underlying expression pattern but not influenced by expression bias and
background variation.
<li>The sum of squared residuals (SSR) is calculated to assess image
correspondence, which is pure pixel-based similarity measure.  SSR assumes
data with homogeneous variance, and so an approximate arcsinh transformation
is appled to raw pixel values before alignment.
<li>Gradient descent is used to find maximum of similarity measure with respect
to paramters of the warp and bias-fields.  A multi-resolution approach is
required to find a global optimum.  First coarse deformations are corrected
on subsamplied images, then progressively finer deformations are eliminated
to on progressively more and more detailed images.
<li>RAIN is implemented using GPGPU (general purpose computation on
graphics cards&mdash;<a href="http://www.gpgpu.org/">http://www.gpgpu.org/</a>
for performance.
</ol>
<p><b>Smooth volume-invariant B-spline warping</b>.  B-splines are shortest
possible polynomial splines with global continuity, chosen for efficiency
in optimizing geometrical transformations for image registration.  These are
smoothly connected piecewise polynomials where additional smoothness constraint
imposes continuity of the spline and its derivatives up to order <i>n</i>&minus;1,
effectively with only 1 degree of freedom per segment.   For numerical
computation purposes the following recursive definition is useful:
<p class="eqnserif">
<i>&beta;</i>(<i>t</i>) = &sum;<sub
style="vertical-align:-1.2em;margin-left:-1.3em;"><i>i</i>=1</sub>
<sup style="vertical-align:1.2em;margin-left:-1.3em;"><i>n</i>+1</sup>
<i>P<sub>i</sub>B<sub>i,k</sub></i>(<i>t</i>)
<p>
where <i>t</i><sub>min</sub>&nbsp;&le;&nbsp;<i>t</i>&nbsp;&lt;&nbsp;<i>t</i><sub>max</sub>, and
2&nbsp;&le;&nbsp;<i>k</i>&nbsp;&le;&nbsp;<i>n</i>&nbsp;+&nbsp;1 and where
position along the curve is given by <i>t</i> between start <i>t</i><sub>min</sub>
and end <i>t</i><sub>max</sub>, <i>P<sub>i</sub></i> are position vectors of the
<i>n</i>+1 vertices and <i>B<sub>i,k</sub></i> are normalized B-spline basis
functions.  B-splines are linearly separable and thus have a simple tensor-product
extension to 2D.  Standard warping techniques correct for optical distortion
and thus introduce protein overexpression after dilation and underexpression
during contraction.  Deformation in 2DE is a physical manifestation, and to preserve
volume-invariance the intensity of transformed pixels must be normalized, achieved
by consdering the set of first-order partial derivatives of the mapping with
respect to input coordinates.  The B-spline warp in RAIN is third-order to ensure
C2 continuity in intensity normalizaiton as well as provide smooth deformation
for spot modeling.  Thus accurate quantification can occur on warped gels.
<p><b>Bias-field correction of intensity inhomogeneity</b>.  Registration of
image depends on determinist mapping between intensity values of reference and
source images.  A global shift-invariant linear systematic bias can be accounted
fro by normalized cross-correlation, as used in MIR.  If systematic bias varies
spatially over each image without correction, image-based registration will be
erroneously guided towards its elimination by geometric means.  It is necessary to model
and correct for disparity orthogonally so geometric alignment can focus on true
underlying expression levels.  A novel shift-variant linear correction by
bias-field modelling is presented as a means of robustly modelling expression
bias and background variation in 2-DE.  Smoothly varying multiplicative and
additive bias-fields as single-valued third order B-spline surfaces are modelled
in this study.  Multiplicative bias represents staining nonlinearities and is
analogue of <q>normalization</q> in microarray studies.  Additive bias models
residual background variation and thus provides background correction between gels.
By ensuring that bias-fields are smoothly varying, their correction remains a
robust and well-posed problem.  The B-spline approach provides elegant framework
for direct incorporation of bias-field correction into the alignment process.
Intuitively bias-field correction should improve the result of image registration,
and vice versa.  The complete model for transformed source gel
<i>I</i><sub>t</sub> is:
<p class="eqnserif">
<i>I</i><sub>t</sub> = |det(<i>&beta;&prime;</i><sub>X</sub>,
<i>&beta;&prime;</i><sub>Y</sub>)| &bull;
(<i>I</i><sub>s</sub>(<i>&beta;</i><sub>X</sub>, <i>&beta;</i><sub>Y</sub>) +
<i>&beta;</i><sub>V</sub>) &bull; <i>e</i><sup><i>&beta;</i><sub>U</sub></sup>
<p>
where source image <i>I</i><sub>s</sub> is under determinant weighted
volume-invariant geometric transformation by a tensor-product uniform
B-spline surface (&beta;<sub>X</sub>, &beta;<sub>Y</sub>),
multiplicative bias-field correction &beta;<sub>U</sub>.  The total
set of parameters for optimization is C = {X,Y,U,V}.
<!-- ================================================================ -->
<p class="tags">

<p class="ref">
 <span class="authors">
Messana I, Inzitari R, Fanali C, Cabras T, Castagnola M
 </span>
 <span class="year">
(2008)
 </span>
 <span class="jname">
J. Sep. Sci.
 </span>
 <span class="jvol">
31,
 </span>
 <span class="pages">
1948-1963.
 </span>
<p class="reftitl">
Facts and artifacts in proteomics of body fluids. What proteomics of 
saliva is telling us?
<p class="Para1">
<b>REVIEW</b>.
Introduction about salivary proteomics and tools used to analyze proteins.
Study of salivary proteins as long as 50 years.  What is known are
severla classes.
<p>
<b>Proline-Rich Proteins</b>.  More than 60% of weight of all proteins
in saliva.  These compose an acidic, basic, and glycosylated basic class.
From two loci PRH-1 and -2 (chromosome 12p13), PRH-1 codes fro PIF-s,
Db-s and Pa isoforms, and PRH2 there is PRP-1 and -2 in the acidic groups.
The N-terminus is pyro-Glu and Ser7 and Ser22 are phosphorylated, with
minor amounts of unphosphoraylated, mono-, and triphosphorylated (Ser17)
forms seen.  From four of  these isoforms a 44-residue C-terminal fragment
can be removed to produce PRP-3 (from PRP-1), PRP-4 (from PRP-2), PIF-f
(from PIF-s) and Db-f (from Db-s).  Pa does not cleave, forming a dimer
with the Cys103 present in its structure. PRPs are involved in calcim
solubilization.
<p>
Basic and glycoprotein PRPs originate from four loci PRB-1 through -4,
whcih are near the aPRP cluster.  bPRPs may counter the toxicity of tannins,
while gPRPs may be involved in oral cavity lubrication.  aPRPs are present
in intact and truncated forms, but bPRPs seem to be present only as fragments
of the larger pro-protein.  Several bPRP peptide sequences have been
established:  IB-1, II-2,, P-D or IB-5, P-E or IB-9, P-F or IB-8c, P-H
or IB-4, IB-7, IB-8a or Con 1-variant.
<p>
<b>Histatins</b>.  This His-riach proteins shown to have anti-fungal
activity.  Histatins 1 and 3 are considered parent peptides.  These have
high sequence identity coding from genes <i>HIS1</i> and <i>HIS2</i>
from chromosome 4q13.  Histatin 3 undergoes extensive processing, cleaved
to make histatin 6, then histatin 5 and subsequently later fragments.
In histatin 3 is a RGYR&darr; convertase consensus not found in histatin 1.
Histatin 1 has Ser2 phosphorylation (not done on histatin 3 because Glu
is missing two residues on C-side).  Histatin 1 from submandibular glands
is poly-sulfated on four C-terminal Tyr residues; not done on histatin 3
because Tyr-27 equivalent is missing required for tyrosylprotein sulfotransferse
recognition.
<p>
<b>Statherin</b>.  A 43-aa long Tyr-rich peptide phosphorylated on Ser-2
and -3.  functions in oral calcium homeostasis and teeth mineralization.
Gene <i>STATH</i> found on chromosome 4q13.3 near histatins.  There may
be a relationship between Pro-rich protein P-B and statherin because they
are located closely on same chromosome, although a functional role is unknown
for P-B peptide.
<p>
<b>Cystatins</b>. The S-type cystatins are cysteine protease inhibitors,
likely to have roles in being anti-pathogen and controlling lysosomal
cathepsin activity.  Cystatin S is phophorylated on Ser-3 (cystatin S1)
and Ser-1 and -3 (cystatin S2).  All cystatin S genes are clustered on
chromosome 20p11.21 along with cystatin C and D genes (<i>CST1-5</i>)
Cystatin SA seems to be exclusively in the saliva while the S and SN
forms are detect in other body fluids and organs.
<p>
<b>&alpha;-Amylases</b>.  About 20% of the protein by weight and most
abundant protein.  By sequence not much different from pancreatic form.
Produce as many as 140 different spots on 2-D gels with all the different
glyco-isoforms.
<p>
Other proteins of note include mucins (MG1 and MG2), lysozyme, lactoferrin,
agglutini, peroxidase, carbonic anhydrase, IgA, IgG, albumin, S100 class
proteins, calcium-binding proteins, defensin, thymosin-&beta; and
various peptide hormones.  All peptides and proteins described above account
for 99% of the mass of the proteomes.  Whole saliva appears to contain more than
1400 proteins.  Many proteins detected in saliva may not have a functional role,
but could be diagnostic.
<p>

The problem with salivary proteome is the highly polymorphic nature of some 
proteins, such as PRPs.  PRPs have different common alleles in western 
populations (S, M, L, VL) are four alleles found at <i>PRB-1</i> and 
<i>PRB-3</i> loci; three alleles (PIF, Db, Pa) are at <i>PRH-1</i>; three (S, M, 
L) at <i>PRB2</i> and <i>PRP4</i> loci.  Alleles PRP-1 and PRP-2 have been 
characterized at the <i>PRH-2</i>.  A two-allele locus can generate at least
three genotypes (and three phenotypes if alleles are co-dominant).  A three-allele
locus can make six phenotypes (three homozygotes, three heterozygotes), and 
four-allele locus ten different phenotypes.  Thus the number of possible PRP
phenotypes are 3 &times; 6<sup>3</sup> &times; 10<sup>2</sup> = 64,800.  Also
alleles special to Asians at the <i>PRH1</i> locus include At, Au, Aw.  More
polymorphisms are created by insertions/deletions, tandem repeats, alternative
splicing, other posttranslational modification including heterogeneous glycosylation.
Histatin 3 has been found polymorphic as well; statherin has an SV2 variant
caused by alternative splicing.  Amylase has an A and B form



<!-- ================================================================ -->
<p class="tags">
FD
<p class="ref">
 <span class="authors">
Romanov V, Siu C-K, Verkerk UH, El Aribi H, Hopkinson AC, Siu KWM
 </span>
 <span class="year">
(2008)
 </span>
 <span class="jname">
J. Phys. Chem.
 </span>
 <span class="jvol">
112,
 </span>
 <span class="pages">
10912-10920.
 </span>
<p class="reftitl">
Binding Energies of the Silver Ion to Alcohols and
Amides: A Theoretical and Experimental Study
<p class="Para1">
Ag<sup>+</sup> has 46 electrons in ground state outer shell config with
4<i>s</i><sup>2</sup>4<i>p</i><sup>6</sup>4<i>d</i><sup>10</sup>.  It has
high affinity for metallothioneins, which function in metal transport and
detoxification.  It binds strongly to peptides in gas phase and can be used
in sequencing.  Use of density functional theory (DFT) show Ag+ will form
mono-, di- and tricoordinate complex with alpha-amino acids.
[more on this fascinating paper]
<!-- ================================================================ -->
<p class="tags">
FD
<p class="ref">
 <span class="authors">
Barlow CK, O'Hair RAJ
 </span>
 <span class="year">
(2008)
 </span>
 <span class="jname">
J. Mass Spectrom.
 </span>
 <span class="jvol">
43,
 </span>
 <span class="pages">
1301-1319.
 </span>
<p class="reftitl">
Gas-phase peptide fragmentation: how understanding the fundamentals provides
a springboard to developing new chemistry and novel proteomic tools
<p class="Para1">
<b>Special Feature: Tutorial</b>.  Current theories of protonated peptides
show that when there are <i>n</i> ionizing protons and <i>m</i> strongly basic
(sequestering) sites and <i>n</i> &gt; <i>m</i>, then <i>n</i> - <i>m</i>
ionizing protons are mobile.  When a proton moves from the thermodynamically
preferred amide O atom to an amide N atom, cleavage of amide bond results
when by neighboring group attack on amide C by N-terminal side amide O of the
adjacent (n - 1) residue.  There is a competition for cleavage at the different
residues in this backbone dissociation.  In cases where the proton is sequestered,
backbone cleavage only occurs when a charge-remote mechanism is available,
such as cleavage at carboxy side of Asp, or 1,2-elimination of methyl sulfoxide
from Met sulfoxide.  More recent work has been looking at deprotonated
(anionic) peptides such as electron capture dissociation (ECD), electron
transfer dissociation (ETD), or low energy CID radical cations formed from
metal complexes.
<p>
The first case study of of directing fragmentation of peptide ions by neighboring
group reactions.  For directin such reactions, competing reactions must be
controlled or prevented.  Protonated Phe can undergo loss of water and CO
by a mobile proton process in which the proton moves from the &alpha;-amino
group to the OH of the carboxyl with an immonium ion product; or it can lose the ammonia group which involves
formation of a phenonium ion (a 3-membered carbon ring with one of the carbons
on the phenyl).  DFT calculations at B3LYP/6-31+G(d,p) level show similar
energetics, and in fact both products are observed experimentally.  The
nucleophiliicity of the aromatic ring was altered by changing substituents at the
4 position (para to &alpha;-carbon of amino acid) with 13 different derivatives.
Strong electron-withdrawing groups (e.g., -NO<sub>2</sub>) favor the
water/CO/immonium pathway, while electron-releasing groups (e.g. -NH<sub>2</sub>)
favor the ammonia/phenonium pathway.  DFT calcuations of PES fro competing
reactions together with RRKM calculations confirm the experimental observations.
Since Phe residues cannot be conveniently modified in peptides, it is not possible
to direct fragmentation in a certain way.  Chemical modifications that fix a
charge at a positions sequester the (eliminate the mobile) proton; they do
promote fragmentation as a fixed electrophilic site.  N-acetylmethionine
will undergo water loss with formation of an oxazolone ring (in a peptide this
forms a b ion).  With the dimethylsulfonium side chain (creating a fixed
cation),  CID spectra instead show formation of dimethylsulfide when the
COOH carbonyl O atom attacks the sulfonium to form a hydroxy-oxenium 5-membered
ring.  Thus the attack of the carbonyl O atom occurs on either the N- or C-terminal
side.  Barriers to these pathways are calculated using DFT and are 14.3 and
16.2 kcal/mol for the C- and N-terminal path, respectively.  These barriers are
below those for amide bond cleavage so alkylation alters the competitive pathway.
The concept <q>selective extraction of labeled entities by charge derivitization
and tandem mass spectrometry</q> (SELECT) has been used to selectively identify
side chains in peptides.  When a peptide is treated with an alkylating agent
(RX) to form the methionyl sulfonium side chain, under low collision CID
there is a loss of RSCH<sub>3</sub>, which can be monitored using a neutral
loss scan on a triple quad MS.  100 pmol of BSA was reduced and
S-carboxyamidomethylated (with haloacetamide) then trypsinized.  Tryptic
peptides were then treated under acidic conditions with phenacylbromide
(C<sub>6</sub>H<sub>5</sub>COCH<sub>2</sub>Br).  The TIC mass chromatogram is
shown and there are several hundred multiply charged ions, corresponding to
100 peptides.  The mass spectrum at time 40 min shows.  The TIC chromatogram
is also shown for MS/MS scanning in neutral loss mode, and four peaks,
all containing Met, are shown at 20, 26, 42, and 49 min.  This chromatogram
is specified to monitor any peptides in CID MS/MS that show a loss of 83 Th
and 166 Th from doubly and singly charged Met precursor ions which lose
CH<sub>3</sub>SCH<sub>2</sub>COC<sub>6</sub>H<sub>5</sub>.
<p>
Biomolecular ions can be mixed with reagent ions to study ion/ion reactions
in the gas phase of the MS.  Among reactions studied are electron transfer
reactions between multiply charged DNA anions and reagent cations.  Syka
and coworkers showed that reagent anions can increase product ions significantly
where electron transfer occurs, particularly the <i>c</i> and <i>z</i> ions
arising from ECD (a process termed ETD).  McLuckey's group produced a model
based on Landau-Zener theory to show likelihood of electron vs. proton
transfer between reagent anions and [M+<i>n</i>H]<sup><i>n</i>+</sup>.
Thermodynamic and geometrci criteria affect choice of the right polyatomic
reagent anions for ETD; for example anions with electron affinity (EA) &ge; 3 eV
are probably unsuitable as ETD reagents;  Franck-Condon factors of the reagent
anion and corresponding neutral should be high.  Instruments should have the
ability to produce ETD-usable anions by chemical ionization (CI) while another
source produces protonated peptides.  Pulsed dual ESI setups allow this.
Carboxylate (RCO<sub>2</sub><sup>&minus;</sup>) are easily produced but are
unsuitable ETD since their EA values are high and they are more likely to
abstract protons.  However, in the gas phase under CID, they decarboxylate to
from carbanions (R<sup>&minus;</sup>) which have low EA values (&le; 2 eV).
Thus benzoate has an EA = 3.75 eV but its decarboxylate phenyl radical has
EA = 1.09 eV.  Coon's group has done extensive work with an LTQ-Orbitrap
to study this.
<p>
In the text above, the use of a fixed charge to direct the action of a neighboring
group was shown.  But if there is no neighboring group, fragmentation will be
charge-remote rather than charge-directed.  Phosphonium ion attached to the
N-terminus leads to formation of <i>a</i> and <i>b</i> type ions which result
from a charge remote fragmentation.  Another important charge remote fragmentation
is simple bond homolysis.  Although not usually observed in low energy CID
because it violates the <q>even-electron rule,</q> it has been show that
[M-H+ 2Li]<sup>+</sup> ions with a Phe residue undergo bond homolysis with
a benzyl radical (&bull;CH<sub>2</sub>C<sub>6</sub>H<sub>5</sub>), quite well
stabilized and the &alpha;-carbon radical cation on the peptide backbone,
also stabilized by electron release from amide N and electron pull by carbonyl
C, are formed.  Hu and Gross showed CID spectral evidence of triply deprotonated
Phe-containing peptides compes to divalent metal [Metal<sup>II</sup>(M-3H)]<sup>&minus;</sup>
showed benzyl radical loss with Ni<sup>2+</sup> and Cu<sup>2+</sup>.
Cys sulfhydryls have a rare post-translational modification of nitrosylation.
In MALDI the intact modified peptide is not seen since there is photo-induced
loss of NO (as radical) and reduction of thiol radical by matrix.  However in ESI
this is detected:  [M+<i>n</i>H-NO]<sup><i>n</i>+&bull;</sup> is formed from
S-nitrosylated peptides.  As bond homolysis is not common in low enegy CID,
chemical derivitization has been used to promote this type of cleavage.  One
grou prepared the diazo form on the N-terminal: the diazo group homolytically
cleaves to produce N<sub>2</sub> and two radicals, one of which fragments further to
produce an acetyl radical form of the N-terminus.  Two of the radical forms can
trigger peptide bond fragmentation by abstracting H atoms to produce a and z ions.
Another chemical derivatization uses a peroxycarbamate, in which the N-terminus
N atom becomes radicalized with homolytic cleavage, and the R side chain eliminates
in radical form as an imine without side chain is formed.  Side chains provide
even more possibilities for derivitization.  Lys side chains converted to
carbamates produce -NH&bull; radicals at the end of the side chain, which
has reactivity on the peptide backbone.  Nitrate esterification of serine
hydroxyls creates an alkoxy radical at the end of this side chain, which
rapidly eliminate HCHO (formaldehyde) to form the reactive alpha;-carbon
radical on the backbone.  If the derivatized peptides are acidified
in the gas phase however, a mobile proton can direct fragmentation rather
than the homolytic derivative; thus to stop this effect, the peptide is
instead cationized with Li for peroxycarbamates and Na for serine nitrate
esters.  For example, BzAlaSer(ONO<sub>2</sub>)OMe as [M+H]<sup>+</sup>
primarily forms a <i>m/z</i> 176 <i>b</i><sub>1</sub> ion consistent with
charge-directed amide bond cleavage.  When the sodiated ion [M+Na]<sup>+</sup>
is analyzed, CID spectrum is mostly a <i>m/z</i> 286 radical on the &alpha;-C,
with loss of NO<sub>2</sub> and HCHO.  An alternative to metal cationization
is to derivatize the N-terminus with a fixed charge group, such as a
pyridyl group.
<p>
Photo-activation of some derivatizes has been used to induce bond homolysis
as well.  N-dinitrophenyl (DNP) amino acid were treated with visible
(458-514.5 nm) and UV (333-385 nm) light and form odd-electron species.
UV photodissociation leads to a &alpha;-C and &beta;-C homolytic cleavage
of Tyr side chains:  CID spectra of protonated ions show y ion fragments.
Iodo-tyrosyl deriviatives will produce cleavage of the C-I bond and produce
radical in the modified residue, leading to backbone fragmentation with
<i>a</i> ion products.  In native proteins, iodination is done on surface
Tyr resiudes (not His can be iodinated in absence of Tyr); this makes it
possible to use it as a probe of tertiary structure.  A mechanism has been
proposed to show why the C<sup>&alpha;</sup>-C(O) bond is disrupted.
<p>
The nature of the radical ion can influence fragmentation.  Charge-
and radical-directed cleavage is possible when there is a mobile proton.
Na-cationized PhCOAlaNHC<sup>&bull;</sup>COOMe produces from the parent
<i>m/z</i> 286 a product of <i>m/z</i> 181 (loss of PhCO).  With the
protonated form, the parent <i>m/z</i> 286 dissociates to a
[<i>b</i><sub>2</sub>-H]<sup>+&bull;</sup> (<i>m/z</i> 232)
and <i>b</i><sub>1</sub> ion mostly.  Another spectrum is rich in
<i>y</i><sub>1</sub> (<i>m/z</i> 90) and shows also a
[<i>b</i><sub>1</sub>-H]<sup>+&bull;</sup> ion.  This results from
a hydrogen atom transfer that moves the radical to another &alpha;-C.
Histidine radical cation shows water loss, in a mechanism starting with
charge delocalized in imidazole ring and &alpha;-C is a spectator, and then
a charge-directed action similar to protonated His occurs.  Radical generated
on each of the three &alpha;-C of tripeptide [GGG]<sup>+&bull;</sup> show
that they do not isomerize by hydrogen atom transfer, but there is proton
shift as a mobile proton.
<p>
The table below summarize the present findings:
<table style="margin:1em auto;">
<tr><th>Even-electron ions (as exemplified by [M+H]<sup>+</sup>)
  <th>Odd-electron ions [M]<sup>+</sup>
<tr><td>Charge-directed and neighboring group processes often dominate.
  <td>Radical site directs cleavage.
<tr><td>Fragmentation can be preceded by proton migration (mobile proton).
 <td>Fragmentation can be preceded by H atom migration.
<tr><td>Cleavage of the peptide backbone provides sequence information.
 <td>Radicals residing at stable positions such as the C<sup>&alpha;</sup>
  position can be involved as intermediates or products.
<tr><td>Small molecule loss (e.g. H<sub>2</sub>O, NH<sub>3</sub>) from
  side chain can compete with sequence ion formation.
 <td>Side chains can be involved in fragmentation via:
  (1) loss of a neutral (e.g. CH<i>2</i>S) to generate a new
   radical cation (typically C<sup>&alpha;</sup>);
   (2) small radical loss to form an even-electron peptide ion
   (e.g. dehydroalanine residue)
</table>
<p>Work by Wee and coworkers with CID of the tripeptide library of GxR
shows what mechanisms control fragmentation.  Radical cations of the tripeptide
are generated, but charge sequestered by Arg side chain.  Fragmentation shows
y<sub>1</sub>, y<sub>2</sub>, [a<sub>3</sub>+H]<sup>+&bull;</sup> (decarboxylation)
ions, and ions resulting from partial or whole elimination of X side chaind.
Thus side-chain elimination, &beta;-elimination, amide bond cleavage, and
decarboxylation are all processes seen, and indicate H atom transfer can occur
before dissociation.  The &alpha;-C centers form captodatively stabilized
radicals.  When mobile protons are not available or not factors, hydrogen
atom transfer plays a role in radical rearrangment prior to fragmentation.
Nitroesters put at different residue positions for test molecules show
identical CID spectra, showing that movement of atom occurs prior to
fragmentation.  Other experiments are waiting to test the migration findings
(isotopic labels, etc).  In addition to derivatization, the structure of
radical ions might be probed by ion/molecule interactions:  currently work
on better probes is being done since the current set sluggishly reacts, and
McLuckey and others have shown that reactions with O<sub>2</sub> with <i>c</i>
and <i>z</i> ions produced by ETD lead to <i>z</i>-type radicals.
<p>
Although usually reduced prior to MS-based proteomics, there are some cases
in which -S-S- bonds should be preserved and studied in gas phase.  Low
energy CID however does not produce homolytic cleavage.  Small model systems
combining deuterium labeling and DFT calculations have shown that peptide
bond cleavage starts with a salt-bridge resulting in C-S bond cleavage.
<!-- ================================================================ -->
<p class="tags">

<p class="ref">
 <span class="authors">
Denny P, Hagen FK, Hardt M, Liao L, Yan W, Arellanno M, Bassilian S, Bedi GS, 
Boontheung P, Cociorva D, Delahunty CM, Denny T, Dunsmore J, Faull KF, Gilligan 
J, Gonzalez-Begne M, Halgand F, Hall SC, Han X, Henson B, Hewel J, Hu S, Jeffrey 
S, Jiang J, Loo JA, Ogorzalek Loo RR, Malamud D, Melvin JE, Miroshnychenko O, 
Navazesh M, Niles R, Park SK, Prakobphol A, Ramachandran P, Richert M, Robinson 
S, Sondej M, Souda P, Sullivan MA, Takashima J, Than S, Wang J, Whitelegge JP, 
Witkowska HE, Wolinsky L, Xie Y, Xu T, Yu W, Ytterberg J, Wong DT, Yates JR 3rd, 
Fisher SJ
 </span>
 <span class="year">
(2008)
 </span>
 <span class="jname">
J. Proteome Res.
 </span>
 <span class="jvol">
7,
 </span>
 <span class="pages">
1994-2006.
 </span>
<p class="reftitl">
The proteomes of human parotid and submandibular/sublingual gland salivas 
collected as the ductal secretions
<p class="Para1">

<!-- ================================================================ -->
<p class="tags">

<p class="ref">
 <span class="authors">
Yi J, Liu Z, Craft D, O'Mullan P, Ju G, Gelfand CA
 </span>
 <span class="year">
(2008)
 </span>
 <span class="jname">
J. Proteome Res.
 </span>
 <span class="jvol">
7,
 </span>
 <span class="pages">
5112-5118.
 </span>
<p class="reftitl">
Intrinsic Peptidase Activity Causes a Sequential Multi-Step Reaction
(SMSR) in Digestion of Human Plasma Peptides
<p class="Para1">
The stability of peptide biomarkers in collected blood sample is done by doing
a time course to monitor.  As controls for normalization, stable isotope-labeled
peptides were considered, but the spiked peptides are subject to preanalytical
degradation and so were the focus of this work.  Absolute quanitification (AQUA)
relies on spiking stable isotope-labeled peptides in the sample and then analyzing
by MS.  Fibrinopeptide A (FPA) is a blood peptide found at various levels in healthy
and disease states.  AQUA FPA was added to serum or plasma samples (in BD P100,
a research-use-only product) and in plasma tubes with EDTA, citrate, or heparin
collected from single venipuncture, or pooled from sera or plasma from 3 healthy
individuals.
<p>
Peptides were purchased from Sigma-Genosys and synthesized with one stable isotopic
amino acid residue, either [U-<sup>13</sup>C, <sup>15</sup>N]-L-Arg, or with
[U-<sup>13</sup>C], <sup>15</sup>N]-L-Phe, in which there are six and nine
<sup>13</sup>C on Arg and Phe, respectively, and four and one <sup>15</sup>N
on Arg and Phe, resp.  Thus each of the labeled amino acids has a mass of 10 Da
more than the natural form.  Peptide quality is assed by HPLC, MALDI-TOF MS,
and the vendor states a purity of 98%.
<p>
Labeled FPA has <i>m/z</i> 1546.69 and its abundance changed in time-dependent
way in several different collection tube types.  Peak intensities are normalized
to ion count, and in doing this, the time course shows decreasing intensities
in all collection tubes, eventually being undetectable.  FPA signal is lost
after 4 h in heparin, 8 h in citrate, 24 h in EDTA.  In the P100 tube it is still
detectable after 48 h, which contains protease inhibitors.  Peak intensity vs.
AQUA FPA concentration shows a linear plot with <i>r</i><sup>2</sup> = 0.95
in four replicates with concentration up to 100 fmol/&micro;l.  Similar results
are seent for C3f peptide, bradykinin, and GLP-1 (latter two, data not shown).
Plot of the ln(peak intensity) versus reaction/incubation time shows a linearity
(thus first-order).  Rate constants and half-lives of AQUA FPA are shown in tables.
The half-life of FPA in serum is too short for use in this type of analysis (15 min),
only in P100 and the plasma tubes in the stability long enough to be amenable to
analysis in this way.
<p>
Mass spectrum analysis shows that the instability of FPA is due to aminopeptidase
action, as the loss was from the N-terminus.
<!-- ================================================================ -->
<p class="tags">
FD
<p class="ref">
 <span class="authors">
Pu D, Vincent JB, Cassady CJ
 </span>
 <span class="year">
(2008)
 </span>
 <span class="jname">
J. Mass Spectrom.
 </span>
 <span class="jvol">
43,
 </span>
 <span class="pages">
773-781.
 </span>
<p class="reftitl">
The effects of chromium(III) coordination on the dissociation of acidic
peptides
<p class="Para1">
Cr(III) might be necessary for normal carbohydrate and lipid metabolism;
deficiency could result in type II diabetes and cardiovascular disease.
Transferrin and low-molecular weight chromium-binding substance (LWWCr,
also called chromodulin) bind to Cr(III).  Chromodulin from bovine liver
has 1438 Da mass (MALDI-TOF).  Amino acid analysis shows Gly, Cys, Asp, and
Glu, with acidic types more than half of sequence.  Sequencing not successful
(Edman, NMR, MS).  Rabbit and bovine liver forms bind 4 equiv.  Proton NMR,
XRA, EPR, and UV-Vis spectroscopies and variable temperature magnetic-susceptibility
shows Cr as 3+ cation, part of covalently bound anion-bridged multinuclear
assembly.  Cr atom contacts O atoms on -COOH groups of acid side chains, with
no coordination by N or S atoms.
<p>
MS instrument is Bruker BioApex 4.7e FT-ICR MS with 4.7 T superconducting
magnet, set up with ESI with 40&micro;l/h flow in positive ion mode of
50:50:1 MeOH:water:acetic acid.  Negative mode set up as 120 &micro;;/h
in 50:50:1 MeOH:water:ammonium hydroxide.  Parent ions are isolated
using correlated sweeps and activated by sustained off-resonance irradiation
(SORI), in which a 5-20 V<sub>pp</sub> pulse at ~700Hz off-resonance from
the parent ion at either higher or lower frequencey is done for 100-150 ms.
Conditions are chosen to produce maximum fragmentation of parent ions;
argon is pulsed to be at 10<sup>&minus;5</sup> torr for collision, with a
1-2 s delay to allow for collisions.  Energy is 100 eV for collision, calculated
by a published method.  Peptides are synthesized (Fmoc, in house):  AAADA,
AEAAAG, AGGAAAA, GGDDAA, AAGEEAA, DGDAAA, EAEAAA, DAADAA, EAAEAA, EAAAEA,
EAAAE, EEE, DDDD, and EEEEEEE.  These are dissolved in water to 2 mM.
CrCl<sub>3</sub> 6 H<sub>2</sub>O is dissolved in water to 0.5 M.  Molar ratios
of 1:4 to 1:12 of peptide:metal are prepared with peptide concentration still
2 mM.  For some peptides (AGGAAAA) the C-terminus is methylated by addition
of MeOH with addition of acetic anhydride to reduce water presence: 1 mg
petpide dissolved in mixture of 20 &micro;l HCl, 100 &micro;l acetic anydride,
500 &micro;l MeOH, left 10 min at room temp and used immediately.
<p>
Peptides with one acidic residue and the C-terminus (AAADA and AEAAAG) will
form a [M+Cr-2H]<sup>+</sup> complex.  The [M+Cr-2H]<sup>+</sup> complex
was subjected to SORI-CID and the product ions generated included
[<i>a</i><sub>5</sub>+Cr-3H]<sup>+</sup> for AAADA.  Other species include
[M+Cr-2H-CH<sub>3</sub>COOH]<sup>+</sup>,
[<i>a</i><sub>5</sub>+Cr-3H-H<sub>2</sub>O]<sup>+</sup>,
[<i>a</i><sub>5</sub>+Cr-3H-H<sub>2</sub>O-CO]<sup>+</sup>, and
[<i>a</i><sub>4</sub>+Cr-3H]<sup>+</sup>.  The loss of acetica acid in the
one ion is from an Asp side chain.  No chromium complexes are seen in negative
ion mode for peptides with one acidic side chain, since only two anionic
sites are provided, and the complex with Cr(III) would require at least a
four proton loss to produce a negatively charged ion.
<p>
With peptides containing at least two acidic side chains in different parts
of the sequence, there is a sequential elimination of water, indicating that
tight coordiation with COOH side chain is not always the case.  Other ions
produced from [M+Cr-2H]<sup>+</sup> include
[<i>b</i><sub>4</sub>+Cr-3H]<sup>+</sup>, [<i>b</i><sub>5</sub>+Cr-3H]<sup>+</sup>,
and [<i>y</i><sub>4</sub>+Cr-H]<sup>+</sup> from GGDDAA.  AAGEEEAA
produces [<i>b</i><sub>5</sub>+Cr-3H]<sup>+</sup> and
[<i>y</i><sub>4</sub>+Cr-H]<sup>+</sup>.  With <i>a</i> ions, Cr(III) displaces
3H<sup>+</sup> relative to <i>b</i> and <i>y</i> ions that form [M+H]<sup>+</sup>.
Thus [<i>y</i><sub>n</sub>+2H]<sup>+</sup> is the
protonated <sup>y</sup> series, meaning [<i>y</i><sub>4</sub>+Cr-H]<sup>+</sup>
is [<i>y</i><sub>4</sub>+2H+Cr-3H]<sup>+</sup>.  More fragments are produced
when there is a neutral residue in between the two acidic residues.
For fragmentation of [M+Cr-2H]<sup>+</sup> in SORI-CID of DGDAAA and EAEAAA
products [<i>a</i><sub>n</sub>+Cr-3H]<sup>+</sup> and
[<i>b</i><sub>n</sub>+Cr-3H]<sup>+</sup> for <i>n</i>=3, 4, and 5 along with
[<i>y</i><sub>n</sub>+Cr-H]<sup>+</sup>.  One or two molecules of water
as losses from [M+Cr-2H]<sup>+</sup> and backbone fragments of the <i>a</i>,
<i>b</i>, and <i>y</i> ions are also observed.  When two neutral residues
are in between, such as in DAADAA and EAAEAA, the same <i>a</i>, <i>b</i>,
and <i>y</i> ion series spectra are produced.  In addition a
[M+Cr-2H-CO]<sup>+</sup> ion is observed  from AAADA, DAADAA, and EAAAEA.
<p>
Peptides EEE and DDDD also produce a [M+Cr-2H]<sup>+</sup> ion and similar
<i>a</i> and <i>b</i> series metal-bound ions, with water loss ions also observed.
A few <i>y</i> series ions are seen.  With longer multiple acidic residues.
peptides bind more than one Cr(III).  EEEEEEE and EDGEECDCGE show
[M+2Cr-5H]<sup>+</sup> and [M+Cr-2H]<sup>+</sup> ions.  The
[M+2Cr-5H]<sup>+</sup> ion could not be produced in enough abundance for
fragmentation.  For EEEEEEE, the [M+Cr-2H]<sup>+</sup> parent produces
fewer fragments: [<i>b</i><sub>6</sub>+Cr-3H]<sup>+</sup> and
[<i>y</i><sub>n</sub>+Cr-H]<sup>+</sup> ions are produced;  water loss
ions from [<i>b</i><sub>4</sub>+Cr-3H]<sup>+</sup>,
[<i>b</i><sub>6</sub>+Cr-3H]<sup>+</sup>, [<i>y</i><sub>n</sub>+Cr-H]<sup>+</sup>
and the parent [M+Cr-2H]<sup>+</sup> are also observed.  Negative mode
analysis of EEE and DDDD show a [M+Cr-4H]<sup>&minus;</sup> ion in 5-10%
relative abundance to a [M-H]<sup>&minus;</sup> ion.  This same Cr-4H ion
was seen with DDDDDDD and EEEEEEE peptides.  Fragment ions were not possible to
observe.
<p>
The peptide AGGAAAA contains no ionizable side chain residues.  Mixed with
Cr(III), in positive mode there was a [M+Cr-2H]<sup>+</sup> ion observed,
with backbone fragments of [<i>a</i><sub>n</sub>+Cr-3H]<sup>+</sup>,
[<i>b</i><sub>n</sub>+Cr-3H]<sup>+</sup>, and [<i>y</i><sub>n</sub>+Cr-H]<sup>+</sup>
for <i>n</i>=3-7, in addition to water loss ions.  When this peptide was
esterfied with methyl group, there was still an observable [M+Cr-2H]<sup>+</sup>
ion although at an intensity of about 20% of that when the peptide has a
free COOH group.  This strongly indicates that Cr(III) can coordinate along
the peptide backbone.  No fragmentation studies were possible with [M+Cr-2H]<sup>+</sup>
because of the low intensity.
<p>
Does Cr(III) affect the nature of fragmentation of peptides?
Cr(III) was left out of the peptides to study the fragmentation of [M+H]<sup>+</sup> ions.
DAADAA and EAAAEA peptides had more a<sub>n</sub><sup>+</sup>,
b<sub>n</sub><sup>+</sup>, and [y<sub>n</sub>+2H]<sup>+</sup> ions in
SORI-CD spectra than seen with [M+Cr-2H]<sup>+</sup>, particularly in low mass
range.  Water loss fragmentation was particularly greater in Cr-free peptide.
No CO losses were see however.
<!-- ================================================================ -->
<p class="tags">

<p class="ref">
 <span class="authors">Omer Deutsch1
Fleissig Y, Zaks B,  Krief G, Aframian DJ, Palmon A
 </span>
 <span class="year">
(2008)
 </span>
 <span class="jname">
Electrophoresis
 </span>
 <span class="jvol">
29,
 </span>
 <span class="pages">
4150-4157.
 </span>
<p class="reftitl">
An approach to remove alpha amylase for proteomic analysis of low abundance 
biomarkers in human saliva
<p class="Para1">
<q>Proteomic characterization of human whole saliva for identification of 
disease-specific biomarkers is guaranteed to provide an easy-to-use and powerful 
diagnostic tool for defining the onset, progression and prognosis of human 
systemic diseases and, in particular, oral autoimmune and cancer diseases.</q>
Abundant proteins mask low abundance markers however.  Work by Weber, Foglietti
et al (1977), Buonocore et al (1975), Subbarmaiah &amp; Sharma (1985), and
Kobayashi et al (1997) shows cross-linked starch interacts with alpha-amylase
or immobilized amylase inhibitors.
<p>
Whole saliva was collected using protocol approved by ethics committee; unstimulated
from 4 healthy volunteers using spitting method, precalibrated tube.  Put on ice
and spun 14,000 <i>g</i> 20 min at 4&deg;.  Total protein using Bradford.  Saliva
was frozen and lyophilized; redissolved in 7 M urea, 2 M thiourea, 4% CHAPS.
then stored &minus;80&deg; until analysis.
<p>
To deplete amylase a 1 ml plastic syringe with 0.45 &micro;m filter at tip
(Whatman FP30/0.45) was used.  Syringe filled with Sigma S2630 potato starch
1 g total. 600 &micro;l of water hand-pressed (20 s) to moisturize substrate.
Then 2 &times; 500 &micro;l saliva passed through by hand pressing in 120 s time.
Amylase activity was determined using Bernfeld (1955) method using Sigma kit
A0521 with 96 mM 3,5-dinitrosalycilic acid (Sigma D-0550).  Elution of amylase
from starch affinity pack was done with 1 ml 10% SDS + 3% (v/v) mercaptoethanol
for 10 min, room temp; material was vortexed 10 s, then centrifuged.
<p>
8% mini SDS-PAGE gels were run (110 V, 70 min) in Laemmli buffer, stained with
0.25% CBB for 2 h, destained 20% EtOH/7% acetic acid 2 h, imaged with GS-800 
calibrated densitomter and Tina 2 software used for calculations.  <b>2D gels</b>:
45 &micro;g protein hydrated to 18 cm pH 3-10NL Immobiline DryStrip in 7 M urea,
2 M thiourea, 4% CHAPS, 1% DTT, 0.5% carrier ampholytes (BioLyte 40% 3/10)
for 16 h at 50 V / 20&deg;.  Voltage was increased to 10 kV at 20&deg; and 
another 8 h focus.  Strips were preprared for 2nd dimension to a 9.5-16.5%
gradient gels.  These were then stained in ammoniacal silver method.
Gel scanning was with the GS-800 and quantified using PDQuest 6.2.0.  To overcome
gel-to-gel variability, and then normalization done to semi-quantify spot intensities
and minimize staining variation.  For MS work, 2DE with 500 &micro;g sample
was done and fixed in 50% EtOH, 12% acetic acid 2 h then stained 16 h with 
CBB-G250 followed by 20% EtOH destaining.  Sterile excision of gels followed by
10 mM DTT 60&deg;/30 min and 10 mM iodoacetamide 30 min room temp in-situ 
incubation, then 1:100 enzyme-to-substrate trypsin overnight was done.  Tryptic
peptides resolved on 100 &micro;m &times; 200 mm using Everest RP medium (Grace
Vydc), program:  5-95% MeCN + 0.1% v/v formic acid at 400 nl/min over 45 min
in to Thermo Orbitrap ion-trap MS in pos ion mode with CID of five most dominant
ions in MS scan mode.  Sequest v 3.31 used to cluster and analyze MS data
with search on human Uniprot database.  Results filtered according to Xcorr
value (1.5 for +1 peptide, 2.2 for +2 peptides, and 3 for +3 peptides).  Identifications
required at least two or more peptides.
<p>
Alpha-amylase significantly interferes with the 60 kDa region, masking low
abundance proteins.  SDS-PAGE shows dramatically that the potato starch captures
nearly all the amylase.  The albumin band is seen as well as passes unbound.
Amylase is as much as 58% of the total salivary protein and when depleted
is 10% of it.  The 69 kDa albumin is 17% of whole salivary protein and increases
to 29% of depleted saliva.  Activity analysis shows that there is a 97% reduction
in amylase activity in depleted material;  the difference in activity reduction
and protein removal efficiency is attributable to other protein components 
present in the 60 kDa band area.  The bound fraction analyzed on SDS gels
shows a lane having a doublet band:  the lower is 89% of the densitometric mass,
while the upper band part is 11%.  29 peptides with 67% sequence coverage 
of alpha amylase came from lower thicker band, with only two peptides of
serum albumin with 2% coverage.  Upper band analysis produces 34 peptides
with 69% coverage of alpha amylase, and 33 peptides with 55% coverage of serum
albumin.  Clearly there was some depletion of albumin too, which is beneficial
in that it too masks low abundant proteins.  Hu et al (2006 <i>Proteomics</i>)
have suggested dissociating low MW products (e.g., cytokines) using 5-20%
MeCN addition in these analyses.
<p>
2D gel analysis was focused in on the amylase area.  Amylase has numerous 
isoforms with different mass mobilities.  Protein spots numbered 1-23 were
identified (spot numbers 10, 15, 20-23 were identifed by MS) and used in 
gel matching from other previous maps.  Spots 1-19 were identified as amylase.
Spot 24-40 were excised and identified by MS.  Spot 30 contained amylase with
reduced matched peptide numbers (usually indicates lower protein content
between two samples (see Bantscheff et al 2007).
<p>
By using depletion, 15 proteins not visualized before could be seen:
type II cytoskeletal 5 keratin (spots 24, 25); type I cytoskeletal 14 (spots 24-27);
angiotensinogen precursor (spot 24); Ig alpha-2 chain C region (spots 24, 25,
27-33); bacteriocidal/permeability-increasing protein-like 1 precursor (spots
24-27, 29, 35); Ig heavy chain V-III region VH26 (spots 25, 27, 29); 
antithrombin-III precursor (spot 25); type II cytoskeletal 6A keratin (spot 26);
Ig heavy chain V-III JON (spot 26); alpha-1-antitrypsin precursor (spots 26-29,
35, 36, 38); Ig heavy chain V-III region BRO (spot 28), serotransferrin precursor
(spots 30-33), Ig gamma-1 chain C region (spots 32, 35); hemopexin precursor
(spot 33); catalase (spots 35,36).  generally low abundance proteins not 
including mucins, PRPs, cystatins, statharins, and histatins.
<p>
Many spots yield more than one protein.  Medium range pH strip separations
and depletion of other higher abundance proteins in 60 kDa area (albumin, 
immunoglobulins) would increase spot resolution.  Spots 24-33 and 39-40 identify
as albumin with high sequence coverage;  Immunoglobulins are seen in spots 24-35.
Greenough et al (2004) have used multi-protein depletion systems
<!-- ================================================================ -->
<p class="tags">

<p class="ref">
 <span class="authors">Omer Deutsch1
Li S-J, Peng M, Li H, Liu B-S, Wang C, Wu J-R, Li Y-X, Zeng R
 </span>
 <span class="year">
(2008)
 </span>
 <span class="jname">
Nucl. Acid Res.
 </span>
 <span class="jvol">
37,
 </span>
 <span class="pages">
D907-D912.
 </span>
<p class="reftitl">
Sys-BodyFluid: a systematical database for human body fluid proteome research
<p class="Para1">
Proteomics is fast becoming a technique for identifying potential novel 
biomarkers for prognosis, diagnosis, therapeusis.  Body fluids (serum/plasma,
urine, CSF, saliva, bronchoalveolar lavage fluid, synovial fluid, nipple
aspirate, tear fluid, amniotic fluid, etc) are often studied.  The 
Max Planck Unified database of organellar, cellular, tissue, and body
fluid was recently published, storing data produced by the institution.
<p>
The Sys-BodyFluid database is a MySQL database composed of JavaServer
page interface.  Data is imported using Java into the database.  Protein
annotation data was downloaded from International Protein Index (IPI)
database, Gene Ontology, GOA database, and KEGG pathway.  The JFreeChart
open source Java library is used to plot the image of statistics data.
50 related PubMed publications published online were retrieved.  Primary 
sequences of proteins were obtained.  Each protein has corresponding ID
in the IPI, and sequences are BLASTed against the database with E-value
cutoff of 10<sup>&minus;8</sup> and BLAST-HSP coverage &gt; 0.9.  11 different
body fluids are represented in Sys-BodyFluid.  13 published papers produced
7748 proteins in the plasma/serum part.  These statistics are found at the
<a href="http://www.biosino.org/bodyfluid">web site</a> through the 
<b>DATABASE</b> link.  Data can be obtained by the <b>DOWNLOAD</b> link.
Utilities of Sys-BodyFluid includ searching by protein ID, name, and
sequence similarity (<b>SEARCH</b> link).  The <b>BROWSE</b> option allows 
comparison analysis between two or more body fluids.  Detailed annotation
is found for each protein.
<p>
Proteins identified in different body fluids were compared.  2928 proteins
are found in at least two body fluids, 1359 are represented in at least 
three.  Only 15 proteins are found in all body fluids (11).  For the 2928
Gene Ontology annotation was obtained and enrichment analysis using
BiNGO and Cytoscape, and each node in figure is a GO term.  From the analysis
some molecular functions include <q>protein binding</q> and <q>enzyme
regulator activity</q> being overrepresented in dataset, as well as the 
biological process like <q>transport</q> and <q>secretion.</q>
<!-- ================================================================ -->
<p class="tags">

<p class="ref">
 <span class="authors">Omer Deutsch1
Rousseau K, Kirkham S, Johnson L, Fitzpatrick B, Howard M, Adams EJ,
Rogers DF, Knight D, Clegg P, Thornton DJ
 </span>
 <span class="year">
(2008)
 </span>
 <span class="jname">
Biochem J.
 </span>
 <span class="jvol">
413,
 </span>
 <span class="pages">
545-552.
 </span>
<p class="reftitl">
Proteomic analysis of polymeric salivary mucins:  no evidence of MUC19
in human saliva
<p class="Para1">
Major macromolecular components of normal, unstimulated saliva are mucins.
Mucins come in two classes: (1) high MW polymeric, gel-forming MUC5B, 
(2) lower MW nonpolymeric MUC7.  Both these have large central domain
with tandemly repeated sequence high in Ser/Thr (the <q>tandem repeat</q>
or <q>mucin</q> domain) for O-glycosylation.  Both up- and downstream of 
MUC5B are sequences similar to von Willebrand factor D domains, important
in polymerization.  These are modified with highly anionic O-glycans;
function is hydration and lubrication of oral cavity, and they bind to 
and sequester microbes.  MUC6, MUC2, MUC5AC, and MUC5B are all gel-forming
mucins found in four consecutive genes at chromosome 11p15.5, expressed
in tissue- and cell-specific way.  MUC6 is expressed in mucous cells and
submucosal glands of stomahc.  MUC5B in mucous cells of salivary glands,
MUC5AC in goblet cells of airway and stomach, and MUC2 in goblet cells
of intestine.  There are parallels in the mouse too, although Muc5b is
not found in mouse saliva.  MUC19 was identified as a gel-forming mucin
although its complete amino acid nor mRNA sequence is known for humans.
In mouse, <i>Muc19</i> gene sequence is described and it is expressed in
sublingual and submandibular glands.  Porcine and bovine mRNA sequences
have been described with high similarity and found in submaxillary tissue.
<p>
6 healthy humans (20-35 y age) expectorated unstimulated saliva into 50 ml
tubes, which were iced.  Horse, cow, and pig saliva were obtained from drool
during feeding.  Informed consents were obtained.  Rats were anesthesitized
and pilocarpine administered to stimulate salivary secretion; drools was 
collected in petri dishes as they lay on their side.  Equal volumes of 8 M
guandine HCl add to all samples and up to 5 vol 4 M guaninde HCl added to 
solubilize mucins (gentle shaking at 4&deg; for 24 h).
<p>
Human and equine mucins were purified by 1.4 g/ml CsCl in 4 M guanidine HCl
in Beckman Ti45 at 40,000 rpm for 68 h, 15 &deg;.  Each tube was emptied
from top into 20 fractions  Aliquot is slot-blotted to NC paper and
stained using periodate-Schiff method.  PAS-rich fractions are pooled, 
then DTT/imidazole treated, dialyzed in water, and lyophilized.  Samples
are resuspened in 0.1 M ammonium bicarbonate, digested with trypsin. Peptides
are isolated on Sephacryl S-100 and run on RPLC-ESI-MS/MS with Water Q-ToF.
<p>
Published <i>Muc19</i> and <i>MUC5B</i> sequences were used in a BLAST search
of human, horse, bovine, pig, and rat genomces.  Genes were determined by
gene synteny and sequence similarity.  The <i>LRRK2</i> leucine-rich repeat
kinase 2 gene is found upstream  of <i>Muc19</i> in human (chromosome 12),
horse (no chromosome identified) and rat (chromosome 7);  the <i>CTNN1</i>
contactin 1 gene is found downstream of <i>Muc19</i>.  The cow genome also
has <i>LRRK2</i> upstream of <i>Muc19</i> (chromosome 5)
<!-- ================================================================ -->
<p class="tags">

<p class="ref">
 <span class="authors">
Zhang Y
 </span>
 <span class="year">
(2008)
 </span>
 <span class="jname">
BMC Bioinformatics
 </span>
 <span class="jvol">
9,
 </span>
 <span class="pages">
40-47.
 </span>
<p class="reftitl">
I-TASSER server for protein 3D structure prediction
<p class="Para1">
Quality estimation of 3D modeling programs has been the goal of "model
quality assessment programs" (MQAP) and the critical assessment of structure
programs (CASP).  I-TASSER is a hierarchical structure modeling approach
based on secondary structure-enhanced Profile-Profile threading Alignment
(PPA) and interative implementation of the Threading ASSEmbly Refinement
(TASSER) program.  Details of I-TASSER method are described by Wu et al
(<i>BMC Biol</i> 5: 17, 2007) and Zhang (<i>Proteins</i> 69: 108-117).
Target sequences are first threaded through a representative PDB structure
library (pairwise sequence identity cutoff = 70%) in order to search for
possible fold using four simple variants of PPA methods, with different 
combinations of the hidden Markov model and PSI-BLAST profiles and the 
Needleman-Wunsch and Smith-Waterman alignment algorithms.  Continuous 
framents are then excised from threading aligned regions which are used to
reassemble full length modles  while threading unaligned regions (mainly
loops) are built by <i>ab initio</i> modeling.  Conformational speace is 
searched by replica-exchange Monte Carlo simulations.  Structure trajectories
are clustered by SPICKER, and cluster centroids obtained by averageing coordinates
of clustered structures.
<!-- ================================================================ -->
<p class="tags">

<p class="ref">
 <span class="authors">
Sana TR, Waddell K, Fischer SM
 </span>
 <span class="year">
(2008)
 </span>
 <span class="jname">
J. Chromatog. B
 </span>
 <span class="jvol">
871
 </span>
 <span class="pages">
314-321
 </span>
<p class="reftitl">
A sample extraction and chromatographic strategy for increasing LC/MS detection 
coverage of the erythrocyte metabolome
<p class="Para1">

Most methods on metabolome preanalytical preservation involve altering physical 
states which typically do not affect chemical states.  Thus a rapid lowering of 
the temperature is logical since all metabolic activities, being chemical 
reactions, are temperature-dependent.  A system to lyse cells that includes the 
addition of -25 cold methanol (MeOH) is a sensible approach.  In addition 
altering the chemical state, namely the pH, of the metabolome to minimize at the 
very least heavily pH-dependent enzyme-catalyzed metabolic changes is a sensible 
approach.  MeOH better than MeCN in LC mobile phase as it has improved APCI 
efficiency for some analytes;  acetic vs. formic acid:  improved negative ion 
sensitivity for acidic compounds.  Analyte RT values std devs ranged from 
0.01-0.053 min in 1-18 min period (void out at 1 min).  As little as 35 peaks in 
ESI(-) for all pH extractions, as high as 585 peaks on ESI(+)/pH 6.  pH 2 RT SDs 
were large compared to other pH, and ESI(-) SD significantly greater than ESI(+) 
mode; difference in tenths of second (negligible).

Internal standards are used to track extraction performance.  Useful standards 
should co-extract with other analytes at different solvent pH, be low-cost, 
ionize by ESI and APCI in both positive and negative ion mode. 
2-(methylthio)benzothiazole [2-MTBTA] (weakly polar nonionic) and 
1-naphthylamine [1-NA] (basic) were added. 2-MTBTA is ionized in APCI but not 
ESI with significant pH dependence on recovery and salt:  minimally recovered 
with ammonium acetate buffer pH 6, maximum at pH 2.  pH 6 effect was probably 
because of salting out with greater solubility in CHCl3 at this pH; at pH 7 and 
9 the ionic strength is not so high so it is more likely to be soluble.  At 
strongly acidic pH 2, the aromatic N is probably protonated, and aqueous phase 
solubility more likely. 1-NA has good solubility in all extracts;  at pH 2 there 
is a 2-3 fold greater recovery.  It also ionizes in APCI.  As for an acidic 
standard 9-anthracene carboxylic acid is being tested and may be good for 
negative ion mode.  2370 nonredundant features were found across all pH 
extractions in all ionization modes.  Each extraction recovered about same 
number of metabolites (1173-1372); thus each extract recovers about half of all 
possible metabolites.  APCI(-) results are relatively poor and showed pH 
dependent effect: perhaps acetic acid as acidifier does not result in anionic 
generation.  As the importance of pH buffering has been stressed in a recent 
report on metabolite recovery, ammonium acetate at pH 6 was compared to pH 7 
unbuffered (but pH-adjusted) extraction and they shared 60% of metabolites when 
the overlap was checked.  If only a single pH extraction is done, only half the 
possible metabolites are detected.  By including a pH 2 and 9 extractions, 92% 
more metabolites are recovered.

In comparing APCI vs. ESI, it should be noted that metabolites are neutral or 
charged, polar and nonpolar.  No single source can be used.  ESI is normally 
preferred as it makes ions by charge exchange in solution, with charged ions 
desorbed to gas phase.  APCI produces charge exchange in gas phase from an 
aerosol.  The metabolite overlap is 7.3% between APCI and ESI, while ESI has 
2.5-fold more coverage, but APCI use increases metabolite features by a third.

Heme is hydrophobic (porphyrin structure) with four imidazole (rings) shaped to 
form larger ring.  TIC shows most abundant peak at 11.5 min in ESI(+) for pH 
9-extracted samples (not in other extracts); same true in APCI(+).  Between 
11.441 and 11.658 min, an isotope pattern consistent with presence of Fe atom 
and heme.  Mass errors for observed isotope ion were  0.8 ppm at 614, 1.3 ppm at 
616, 0.7 ppm at 617, and 0.1 ppm at 618 (proposed C34H32N4O4Fe from METLIN 
Personal database produces match based on mass error, isotope pattern, and 
understanding of biology).  EIC of most abundant isotope of heme shows 100-fold 
more heme in pH 9 extractions compared to pH 6 or 7, 10-fold more than in pH 2. 
This extraction likely due to pH extremes causing heme dissociation from Hb.  In 
looking at results for biliverdin (m/z 583.25511), which has no iron, EICs for 
biliverdin shows twin peaks between retention times 12.5-13 min (possibly 
diastereomers).  With Bvn, there is 10-fold increase in signal at pH 9 compared 
to others in ESI(+) mode, so Bvn is
<ol>
<li>Place cells (~0.5 ml) in typical 1.5-1.7 ml conical tube and in refrigerated 
microcentrifuge (4&deg;)
<li>Pellet intact cells for 2 min at 1000 <i>g</i> and transfer to ice-water 
bath immediately
<li>Remove supernatant without disturbing pellet
<li>Resuspend in 150 &micro;l ice-cold, pH-adjusted ultrapure water to lyse; and 
quicly place the tube in a dry ice/MeOH bath or a circulating bath at least 
&minus;25&deg; or below
<li>Put the tube in a 37&deg; water bath for no more than 30 s
<li>Add 600 &micro;l of &minus;20&deg; MeOH which includes internal metabolite 
standards 
<li>Vortex with complete mixing, then place tubes at room temperature
<li>Add 450 &micro;l chloroform (CHCl3):  vortex tubes briefly every 5 min over 
30 min period; keep tubes on cold bath
<li>Return tubes to room temp, and add 150 &micro;l ice-cold, pH-adjusted 
ultrapure water
<li>Spin tubes 1 min at 1000 <i>g</i> in refrigerated microcentrifuge (4&deg;)
<li>Place tubes in &minus;20&deg; freezer for 2-8 h to allow phase separation
<li>Transfer upper phases to separate tubes without disturbing cells debris at 
interphase
<li>Add 1 vol equivalent of acetonitrile (MeCN) to the tubes with the top 
aqueous phase and transfer to refrigerator (4&deg;) for 20 min.
<li>Spin at 10,000 <i>g</i> for 10 min at 4&deg; and transfer supernatant to new 
tube.
<li>Dry the tube in a vacuum centrifuge (or equivalent)
<li>Resuspend the dried material in 50-100 &micro;l of the starting LC mobile 
phase (it might be necessary to spin for a long time at high speed or filter 
sample prior to injection)
</ol> 
<!-- ================================================================ --> 
<p class="tags">

<p class="ref"> 
 <span class="authors"> 
P&eacute;r&egrave;s S, Molina L, Salvetat M, 
Granier C, Molina F 
 </span> 
 <span class="year"> 
(2008) 
 </span> 
 <span class="jname"> 
BMC Bioinformatics 
 </span> 
 <span class="jvol"> 
9, 
 </span> 
 <span class="pages"> 
460- 
 </span> 
 <p class="reftitl"> 
A new method for 2D gel spot alignment: application to the analysis of large 
sample sets in clinical proteomics 
 <p class="Para1"> 
Changes in the ability to alignment (match) spots result from variability in 
proteomes (biological variation) and those from inadequacy of the analytical 
technique.  Spot alignmnet is usually done by spot detection followed by 
spot-based image warping then alignment, or by pixel-based image warping then 
spot detection and then spot alignment.  The first method corrects distortion 
using user-defined landmark spots, which can be automated and make spatial 
correction implicit.  A fusion or synthetic gel is used to represent the whoe 
experiment.  If number of gels to be aligned is high, distortion needs to be 
modeled using low-order polynomial transformation.  Local geometric distortions 
are thus poorly corrected leading to increase of noise in spot alignment. With 
pixel-based warping, spatial correction is doen from the raw-image data making 
use of things learned from image processing studies.  This eliminates matching 
problems since spot detection is done in the next phase. While the method is 
convenient, there is a bias from the discontinous change in intensity among the 
set of aligned gels ending to affect spot intensity quantification. User needs 
to adjust or control spot alignment by hand, both time-consuming and a source of 
errors.  Commercial comparative analysis of 2D gels includes PDQuest, Melanie, 
Same Spots, Proteomweaver, Gellab identify spots of interest by image comparison 
(<q>gel matching</q>).  Some systems pair each gel of a matching set against a 
<q>reference</q> gel (Melanie, PDQuest, etc) and others using <em>recursive gel 
matching</em> (Same Spots, Proteomweaver, etc); each gel is used recursively as 
a reference one time during the matching process.  The spot alignment is still 
noisy.  A new algorithm for automatic spot alignment called Sili2DGel return the 
meaningful Spot Alignment Positions (SAP) for a given set of gels. 
<p> 
Sili2DGel is based on graph theory, with input data represented by graphs in 
which specific subgraphs are searched.  Results given in various formats (text 
file, synthetic gel, etc).  Ideally a protein will be represented as spots 
displayed at same coordinates on each gel.  If only a single reference gel is 
selected for match set, spot not found in reference will not be aligned to 
anything.  In Figure 2, Gel 1 is a reference with 4 spots: a1, b1, c1, and 
c1&prime;, where last two are tiny adjacent spots (resolved). Gel 2 has four 
spots:  a2, b2, c2, and d2; and Gel 3 also four spots: a3, b3, c3, and d3. Spots 
c2 and c3 are very large, with no two spot resolution.  The alignments for each 
spot on the reference gel <i>A</i> are: <i>A</i>(<i>a</i><sub>1</sub>) = 
{<i>a</i><sub>1</sub>; <i>a</i><sub>2</sub>; <i>a</i><sub>3</sub>}, 
<i>A</i>(<i>b</i><sub>1</sub>) = {<i>b</i><sub>1</sub>; <i>b</i><sub>3</sub>}, 
<i>A</i>(<i>c</i><sub>1</sub>) = {<i>c</i><sub>1</sub>; <i>c</i><sub>2</sub>; 
<i>c</i><sub>3</sub>}, and <i>A</i>(<i>c&prime;</i><sub>1</sub>) = 
{<i>c&prime;</i><sub>1</sub>; <i>c</i><sub>2</sub>; <i>c</i><sub>3</sub>}. Note 
that spot b2 does not align on Gel 2 and that spots d2 and d3 on gels 2 and 3 
have no spot on reference gel 1. Moreover two spots on reference gel 1 align 
with one single spot on gels 2 and 3.  Gel distortions will obviously affect 
matching, aligning spots not truly identical and misaliging spots that will not 
be matched but should.  Thus there are <i>noise</i> and <i>missing spots</i>. 
<p> 
The SAP does depend on pre-processing involving spot detection and 
identification of significant spots. Assume <i>S</i> is the set of all spots in 
<i>N</i> gels, so for any spot <i>i</i>, gel matching produces all spots 
<i>j</i> matching <i>i</i>.  The notation <i>i</i>&rarr;<i>j</i> is used when 
spot <i>i</i> matches <i>j</i>, and the alignment to <i>i</i> for all spots is 
symbolized as
<p class="eqnserif"> 
<i>A</i>(<i>i</i>) = {<i>j</i> &isin; <i>S</i> : <i>i</i>&rarr;<i>j</i>} &cup; 
{<i>i</i>} 
<p> 
Alignments represented by weighted undirected graph called <b>matching 
graph</b>.  In the graph, a node corresponds to spot and the edge is the match 
between spots.  Edge weight is proportional to number of matches between two 
spots, and should be &le; to number of gels.  If <i>G<sub>N</sub></i> is the 
matching graph of <i>N</i> gels and |<i>S</i>| is size of set <i>S</i>, then 
<p class="eqnserif"> 
<i>G<sub>N</sub></i> = (<i>V, E, w</i>) 
<p> 
where <i>V</i> = <i>S</i>, set of vertices of <i>G<sub>N</sub></i>, <i>E</i> = 
{(<i>i, j</i>): <i>i</i> &isin; <i>S</i>, <i>j</i> &isin; <i>A</i>(<i>i</i>), 
<i>i</i> &ne; <i>j</i>}, the set of edges of <i>G<sub>N</sub></i>, 
<i>w</i>(<i>i, j</i>) = |{<i>k</i> &isin; <i>S</i>: <i>i</i> &isin; 
<i>A</i>(<i>k</i>), <i>j</i> &isin; <i>A</i>(<i>k</i>)}|, &forall;(<i>i, j</i>) 
&isin; <i>E</i>, the weight of edge (<i>i, j</i>). Node labeling involves teh 
name of the gel and number of the spot, and edges are labeled with their weight. 
High density zones represent the SAP in the graph, (i.e., zones where a lot of 
nodes are pairwise adjacent). Associations which are highly incorrectly 
connected are made much of the time, and so in cleaning the graphy, edges are 
removed which ae wrong connections. SAP searching of <i>N</i> gels involves 
finding specific subgraphs of the matching graph <i>G<sub>N</sub></i>.  The best 
case occurs when all spots representing the same are pairwise-matched in 
<i>n</i> gels where <i>n</i> &le; <i>N</i>. These <i>n</i> spots are all 
connected together in the matching graph.  The subgraph is called a 
<b>clique</b> (a complete graph) with all its edges weighted by <i>n</i>.  Graph 
<i>G</i> = (<i>V, E</i>) is a clique if all vertices are pairwise adjacent, i.e. 
&forall;<i>i, j &isin; V</i> with <i>i &ne; j</i>, then (<i>i, j</i>) &isin; 
<i>E</i>.  Alignments are not always perfect: in a case where spots can match at 
least once but are not all pairwise adjacent and represented in the matching 
graph by a clique in which at least one edge is weighted by a value lower than 
<i>n</i>.  In the worst case some spots will be missing in alignment although 
they should belong to the alignment set. If two spots from different gels never 
match together during the whole recursive matching procedure, but match with 
many of the other spots, they are not adjacents in the matching graph and the 
subgraph is not a clique.  This subgraph contains a clique and some other nodes 
which are adjacents to several nodes of the clique and it is called a 
<b>pseudoclique</b>.   In all cases SAP are represented by dense clusters of 
nodes in the graph (i.e. nodes which are highly connected to each other) which 
are either cliques or pseudocliques. 
<p> 
<b>Algorithm of SAP Identification</b>.  Finding a maximal clique is a classical 
NP-hard problem and thus exact algorithms are guaranteed to return solution only 
in a time which increases exponentially with number of vertices in the graph. 
Exact solution smethods have limited performance on large datasets. The graph 
can be decomposed into reduced subgraphs and then SAP in corresponding reduced 
search spaces determined.  Search space is reduced by partitioning graph in all 
connected components (i.e., maximal connected subgraphs).  Edges with weight of 
1 are suppressed because it is assumed they are not sufficient to belong to a 
pseudoclique when searching the connected component; these cannot represent 
alignment of size 2 (which contains 2 spots).  Suppression of edges does not 
distort results because if spots are really in alignment, they would have 
connectivity with other spots of the alignment and would be restored later in 
process.  Pseudocliques are very dense clusters of the graph, and to select 
these very dense cluters, the <b>isthmuses</b>, which are edges separating a set 
of nodes of graph into two highly connected subgraphs, need to be eliminated. 
The <i>strength metric</i> (<i>sm</i>) determines the isthmuses by measuring how 
much of an edge is likely to separate a graph into two highly connected 
subgraphs: <p class="eqnserif" 
style="text-align:left;margin-left:5em;text-indent:-3em;"> <i>sm</i>(<i>u,v</i>) 
= [ |<i>W<sub>u</sub></i>| / ( |<i>W<sub>uv</sub></i>| + |<i>M<sub>u</sub></i>| 
+ |<i>M<sub>v</sub></i>| ) ] + <br> { [ 
<i>e</i>(<i>M<sub>u</sub>,W<sub>uv</sub></i>) + 
<i>e</i>(<i>M<sub>v</sub>,W<sub>uv</sub></i>) + 
<i>e</i>(<i>M<sub>u</sub>,M<sub>v</sub></i>) + <i>e</i>(<i>W<sub>uv</sub></i>) ] /<br> 
[ |<i>M<sub>u</sub></i>||<i>W<sub>uv</sub></i>| + 
|<i>M<sub>v</sub></i>||<i>W<sub>uv</sub></i>| + 
|<i>M<sub>u</sub></i>||<i>M<sub>v</sub></i>| + (|<i>W<sub>uv</sub></i>|/2) ] }
<p>
where <i>u, v</i> &isin; <i>V</i>, <i>M<sub>u</sub> = N<sub>u</sub></i> / 
<i>N<sub>v</sub></i> and <i>W<sub>uv</sub> = N<sub>u</sub></i> &cap; 
<i>N<sub>v</sub></i> with <i>N</i>(<i>u</i>) and <i>N</i>(<i>v</i>) denoting 
neighborhoods of <i>u</i> and <i>v</i>.  The function <i>e</i>(<i>A,B</i>) or 
<i>e</i>(<i>A</i>) denotes edge counts between two sets <i>A</i> and <i>B</i> or 
within set <i>A</i>. The first term of the sum in <i>sm</i> counts the number of 
triads (cycles of length 3) containing edge (<i>u,v</i>) and then later computes 
relative number of cycles of size 4 containing the edge.
<p>
Values of sm are between 0 and 2.  The lower value indicates edge is more likely
an isthmus whereas the higher value signifies it is likely center of cluster.
A null <i>sm</i> is an isthmus and a value of two is an edge in an isolated
clique.  A threshold <i>sm</i> value determines suppression to reduce graph.
If graph highly connected, the <i>sm</i> threshold should not be too low to
allow good reduction of graph.  Connected components are calculated and the SAP
can be researched into all reduced graphs.  The SAP search algorithm steps are
shown in the table below:
<table class="td-top">
<caption class="left-align"><i>SAP_search</i>(<i>sm, &gamma;</i>)</caption>
<tr><td>1. Graph construction <td>Remove edges having weight of 1
<tr><td>2. Graph reduction    <td><ol style="list-style-type:lower-alpha;">
  <li>Remove edges where strength metric is lower than <i>sm</i> 
  <li>Decompose graph into its connected components </ol>
<tr><td colspan="2">3. For each connected components, perform 
  <i>cluster_search</i>(<i>&gamma;</i>)
<tr><td colspan="2">4. Return the new (updated) graph
</table>
<p>
The clusters search algorithm is as follows:
<table class="td-top">
<caption class="left-align"><i>Clusters_search</i>(<i>&gamma;</i>)</caption>
<tr><td>1. Clique and pseudoclique search
 <td><ol style="list-style-type:lower-alpha;">
 <li>Find the maximal cliques and remove the cliques of size two if their 
nodes have more than 1 neighbor
 <li>For all cliques <i>C</i>, remove the nodes n for which 
max <i>w</i>(<i>n, i</i>)<sub><i>i&isin;C</i></sub> &ge; <i>&tau;</i>(<i>C</i>)
 <li>For all cliques C, add to C the ?-dense nodes n such that 
min <i>w</i>(<i>n, i</i>)<sub><i>i&isin;C</i></sub> = <i>&tau;</i>(<i>C</i>)
 <li>Remove from the list of cluster the included clusters</ol>
<tr><td>2. Select clusters according to their <i>s</i>-value
 <td><ol style="list-style-type:lower-alpha;">
 <li>Remove the <q>worst clusters</q>: 

For all clusters <i>C</i><sub>1</sub>, <i>C</i><sub>2</sub> such that 
|<i>V</i><sub><i>C</i><sub>1</sub></sub>| &ge; |<i>V</i><sub><i>C</i><sub>2</sub></sub>|, if 
|<i>V</i><sub><i>C</i><sub>1</sub></sub> &cap; <i>V</i><sub><i>C</i><sub>2</sub></sub>| &ge; &gamma; &times; 
|V<sub>C<sub>1</sub></sub>{ and s(C<sub>1</sub>) &ge; s(C<sub>2</sub>) and 
Gelnb(C<sub>1</sub>) &ge; Gelnb(C<sub>2</sub>), then remove C<sub>2</sub>

<li>For all clusters C<sub>1</sub>, C<sub>2</sub> such that 
|V<sub>C<sub>1</sub></sub> &cap; V<sub>C<sub>2</sub></sub>| &ge; &gamma; &times; 
min(|V<sub>C<sub>1</sub></sub>|, |V<sub>C<sub>2</sub></sub>|), add the nodes of 
min(s(C<sub>1</sub>), s(C<sub>2</sub>)) which have a maximal weight greater than 
&tau;(Cmin) in max(s(C<sub>1</sub>), s(C<sub>2</sub>)).
</ol>
</table>

<!-- ================================================================ -->
<p class="tags">

<p class="ref">
 <span class="authors">
Morris JS, Clark BN, Wei W, Gutstein HB
 </span>
 <span class="year">
(2009)
 </span>
 <span class="jname">
Univ. of Texas MD Anderson Cancer Center Dept. of Biostatistics Working Paper Series
 </span>
 <span class="jvol">
Paper 49
 </span>
 <span class="pages">
Available at: <a href="http://works.bepress.com/jeffrey_s_morris/38">http://works.bepress.com/jeffrey_s_morris/38</a>
 </span>
<p class="reftitl">
Comparison of Pinnacle and SameSpots for Spot Quantification and Differential
Expression in 2-Dimensional Gel Electrophoresis Studies
<p class="Para1">
Gel scans are digital images the processed to detect and quantify
intensities of protein spots.  Given <i>N</i> gels and <i>p</i>
protein spots detected across all gels, processing will produce
a matrix <b>Y</b> that is an <i>N &times; p</i> set of spot
intensities.
<p>
Detection, matching and quantification are the priorities of gel
analysis software.  The initial phase is detection of spots, then
finding spot boundaries and then calculating spot volumes, for
each individual gel.
<p>
Next is matching across gels.  This less to less than the
<i>N &times; p</i> spots intensities for missing spots.  A problem
with matching is related with spot boundaries since many spots
overlap.  Manual editing of gels can be very time-consuming
(from 1-4 h/gel, compromising objectivity and reproducibility of the analysis.
As number <i>N</i> of gels increase, studies are done with fewer samples and
this leads to reduced statistical power.
<p>
<i>Pinnacle</i> improves workflow because it improves spot detection and quantitation
compared to other commerical packages.  It requires pre-aligned gels, and from these
it first computes an average gel (arithmetic).  Others software produces a
composite gel distinct from an averaged gel.  Averaged gels allow borrowing
strength of all gels for spot detection.  Wavelet de-noising is applied, and
then spots detected on the average gel by pinnacle identification:  staining
intensities that are local maxima in two dimensions.  The maximum intensity in
the neighborhood of the pinnacle (<i>x, y</i>) coordinate is used for quantitation.
Because an intensity is used in quantitation, spot boundaries do not have to be
estimated in spot volume calculation, improving reliability and validity.
<p>
Progenesis <i>SameSpots</i> uses aligned gels and does spot detection across all
images, obtaining a master list and estimating boundaries.  Spot volumes are
determined in a <i>N &times; p</i> matrix <b>Y</b>.  No average gel is computed
for spot detection.  Spot boundary masks are overlaid.  The method results in no
missing data, and use of common spot boundaries for all gels should in principle
increase precision of spot volume estimates.  Karp and others have shown SameSpots
reduces variability and increases power to detect differential expression; in
theory it diminishes subjective editing.
<p>
Data sets used in this study
<!-- ================================================================ -->
<p class="tags">
FD
<p class="ref">
 <span class="authors">
Heaton AL, Moision RM, Armentrout PB
 </span>
 <span class="year">
(2008)
 </span>
 <span class="jname">
J. Phys. Chem. A
 </span>
 <span class="jvol">
112,
 </span>
 <span class="pages">
3319-3327.
 </span>
<p class="reftitl">
Experimental and Theoretical Studies of Sodium Cation Interactions 
with the Acidic Amino Acids and Their Amide Derivatives
<p class="Para1">
Asx and Glx amino acids are important in the coordinatin of Na<sup>+</sup>
and K<sup>+</sup> in the Na-K ATPase.  To study cation-protein interactions,
models using gas-phase threshold CID are used.  Bojesen et al have studied
the relative Na<sup>+</sup> binding affinity to each of the naturually
occurring 20 amino acids using Cook's kinetic method.  Kish et al have determined
relative sodium cation affinities of amino acids using Cook's kinetic method
based on dissociation characteristics of collisionally activated Na<sup>+</sup>-
heterodimer complexes.  In this study interaction of Na<sup>+</sup> to Asp,
Glu, Asn, and Gln are reported, with absolute bond dissociation energies
(BDEs) of the Na<sup>+</sup>(L) complexes.  Theoretical calculations are 
at the B3LYP/6-311+G-(d,p) level: provides structures, vibrational frequencies,
rotational constants to analyze the TCID data.  Experimental calculations
are performed for a number of possible Na<sup>+</sup>(L) geometries at
the MP2(full)/6-311+G(2d,2p), B3LYP/6-311+G-(2d,2p), and B3P86/6-311+G(2d,2p)
levles using geometries and zero-point energy corrections calculated at the
B3LYP/6-311+G(d,p) level of theory.
<p>
ESI conditions have been described previously.  Solvent is 50% MeOH with
100 &micro;M amino acid present with NaCl (all Sigma-Aldrich) in syringe pump
at 40 &micro;l/h using 35-gauge stainless steel needle biased at 2000 V.
Cone voltage is ~35 V.  Energies produced during ESI follow Maxwell-Boltzmann
distribution of rovibrational states at 300 K.  Ions are decelerated to 
defined kinetic energies and focused by rf octopole that traps ions radially.
Octopole passes through a static gas cell with Xe gas.  A quadrupole is used
for mass analysis after collision.  Detection is with high voltage dynode
scintillation ion detector; signal processed using standard pulse-counting
techniques.  Ion intensities are functions of collision energy, and converted
to absolute cross sections (see previous description).  Relative cross-section 
uncertainty is about &plusmn;5%, and absolute cross-section uncertainty is
about &plusmn;20%.  Ion kinetic energy distribution measured to be Gaussian
and typical FWHM is 0.1-0.2 eV(lab) for ESI source.  Uncertainties in absolute
energy scale are about &plusmn;0.05 eV (lab).  Ionic energies in laboratory
frame are converted to energies in the center-of-mass (CM) frame using
<i>E</i><sub>CM</sub> = <i>E</i><sub>lab</sub><i>m</i>/(<i>m</i> + <i>M</i>),
where <i>M</i> and <i>m</i> are masses of ionic and neutral reactants, 
respectively.
<p>
<b>Thermochemical Analysis</b>.  Threshold regions of CID reaction cross sections
are modeled using this equation:
<p class="eqnserif">
&sigma;(<i>E</i>) = (<i>n</i>&sigma;<sub>0</sub>/<i>E</i>)&sum;<i>g</i><sub>i</sub>&int;<sup><i>E</i></sup><sub style="margin-left:-0.75em;"><i>E</i><sub>0</sub>&minus;<i>E<sub>i</sub></i></sub>
(1 &minus; <i>e</i><sup>&minus;<i>k</i><sub>tot</sub>(<i>E</i>*)&tau;</sup>)(<i>E</i> &minus; &epsilon;)<sup>n&minus;1</sup>
d(&epsilon;)
<p>
where &sigma;<sub>0</sub> is an energy-independent scaling factor, <i>n</i>
is an adjustable parameter describing efficiency of collisional energy
transfer, <i>E</i> is relative kinetic energy of reactants, <i>E</i><sub>0</sub>
is threshold for dissociation of ground electronic and rovibrational state of
reactant ion at 0 K, &tau; is experimental time for dissociation (~0.5 ms =
5 &times; 10&minus;4 s in the extended dual octopole configuration as measured
by time-of-flight studies), &epsilon; is energy transferred from translation
during collision, and <i>E</i>* is internal energy of excited/energized molecule
(EM) after the collision (<i>E</i>* = &epsilon; + <i>E</i><sub>i</sub>).
The summation in the expression occurs over rovibrational states of reactant
ions, <i>i</i>, where <i>E</i><sub>i</sub> is excitation energy of each state
and <i>g</i><sub>i</sub> is fractional popolation of those states 
(&sum;<i>g</i><sub>i</sub> = 1).  Beyer-Swinehart algorithm used to evaluate
count and density of rovibrational states, and relative populations 
<i>g</i><sub>i</sub> calculated for Maxwell-Boltzmann distribution at 300 K.
The <i>k</i>(<i>E</i>*) term is unimolecular rate constant for dissociation
of energized molecule and defined by Rice-Ramsperger-Kassel-Marcus (RRKM)
theory as in this equation:
<p class="eqnserif">
<i>k</i>(<i>E</i>*) = d<i>N</i><sup>&dagger;</sup><sub>vr</sub>(<i>E</i>*
&minus; <i>E</i><sub>0</sub>) / <i>h&rho;</i><sub>vr</sub>(<i>E</i>*)
<p>
where <i>d</i> is reaction degeneracy, <i>N</i><sup>&dagger;</sup><sub>vr</sub>
(<i>E</i>* &minus; <i>E</i><sub>0</sub>) is sum of rovibrational states of
the transition state (TS) at an energy <i>E</i>* &minus; <i>E</i><sub>0</sub>,
and &rho;<sub>vr</sub>(<i>E</i>*) is the density of states of energized
molecule (EM) at available energy, <i>E</i>*.  These rate constants allow
kinetic shifts to be modeled (discussed below).
<p>
Several effect obscure interpretation, must be accounted during data analysis
to produce accurate thermodynamic information.
<ul>
<li><b>Energy broadening resulting from thermal motion of neutral collision
gas and kinetic energy distribution of reactant ion</b>.<br>
Explicitly convoluting the model over both kinetic energy distributions
is described in detail elsewhere, and will be used in accounting
<li><b>First equation only models cross sections representing products
formed from single collision event</b>.<br> To ensure single collisions,
data collected at three pressures of Xe gas (0.16, 0.08, and 0.04 mTorr),
and cross sections are evaluated for pressure effects, extrapolated to
zero pressure when necessary.
<li><b>Lifetime for dissociation</b>.<br> When reactant molecule size
increases, so do its vibrational modes of the ion, and time for energy
randomization into reaction coordinate after collision.  Hence some
excited-state molecules may not dissociate within the time scale of
experiment, leading to delayed onset for CID threshold, a kinetic
shift, becoming more noticeable as the molecular size increases.
Kinetic shifts are estimated by incorporating RRKM theory as shown
in first equation.  To evaluate rate constant in eq 1, sets of 
rovibrational frequencies for all EM and all TS are required.
As the Na<sup>+</sup>(L) metal-ligand interactions are mainly electrostatic,
most appropriate TS dissociation mode of intact ligand is the loose
association of cation and neutral ligand fragments.  Assumption is
validated for multidentate ligands showing crown ethers and for
tripeptide GGG. The TSs are treated as product-like: and so the TS
frequencies are for dissociation products.  Molecular parametes needed
for RRKM calculation taken from quantum mechanical calculations detailed
in next section.  Transitional frequencies in TSs are treated as rotors,
treatment corresponds to phase space limit (PSL).  Adiabatic 2D rotational
energy is treated using a statistical distribution with explicit summation
over all possible values of rotational quantum number.
</ul>
<p>
Model cross sections of eq 1 are convoluted with kinetic energy distribution
of reactants compared to data.  Nonlinear least squares used to optimize
&sigma;<sub>0</sub>, <i>n</i>, and <i>E</i><sub>0</sub>.  Derivation of 
final optimized BDE values at 0 K required two assumptions:  (i) there is
no activation barrier in excess of the reaction endothermicity for the loss
of ligans, which is generally true for ion-molecule reacctions and for 
heterolytic noncovalent bond dissociations here;  (ii) the measured 
threshold <i>E</i><sub>0</sub> values for dissociation are from ground-state
reactant to ground-state ion and neutral ligand products.  Given the long
time (~5 &times; 10<sup>&minus;4</sup> s) dissociating products should be
able to rearrange their low energy conformations after collisional 
excitation.
<p>
Simulated annealing methodology using the AMBER suite of programs and AMBER 
force field was used to generate starting structures for higher level 
optimizations.  Unique structures generated via simulated annealing were further 
optimized with NWChem at HF/3-21G level.  Energies computed with the low-level 
ab initio calculation show higher correlation with energies calculated at higher 
levels of theory than that with relative energies from the AMBER force field. 
Unique structures from HF/3-21G calculations within about 30 kJ/mol of the 
lowest-energy structure (~10 structures for each Na<sup>+</sup>(L) complex) were 
further optimized using Gaussian 03 at B3LYP/6-31G(d) level with <q>loose</q> 
optimization (maximum step size of 0.01 au and rms force of 0.0017 au) to 
facilitate rapid convergence.    All unique structures are optimized at 
B3LYP/6-311+G(d,p) level.  Rotational constants come from optimized structures,
and vibrational frequencies aer calculated at the level.  When used in internal
energy determinations or for RRKM calculations, the vib frequencies are scaled
by 0.99.  Single-point energies ae calculated at the MP2(full), B3LYP, and B3P86
levels using 6-311+G(2d,2p) basis set and the B3LYP/6-311+G(d,p) geometries.
Zero-point vibrational energy (ZPE) corrections were determined using scaled
vibrational frequencies.  Basis set superposition errors (BSSE) were estimated
using full counterpoise method.

<!-- ================================================================ --> 
<p class="tags">
FD
<p class="ref"> 
 <span class="authors"> 
Winter D, Lehman WD 
 </span>
 <span class="year"> 
(2009) 
 </span> 
 <span class="jname"> 
Proteomics 
 </span> 
 <span class="jvol"> 
9, 
 </span> 
 <span class="pages"> 
0000-0000 
 </span> 
<p class="reftitl"> Sequencing of the thirteen structurally isomeric quartets of 
N-terminal dipeptide motifs in peptides by collision-induced dissociation 
<p class="Para1"> The 2nd peptide bond from the N-terminus of multiply protonated 
peptides is usually cleaved by CID, thus making the b2 ion prominent.  From this 
is six-centered diketopiperazine and five-centered oxazolonium ring will form. 
The oxazolonium ring appears to form when the N-terminal residue carbonyl oxygen 
bonds electrophilically to the <i>n</i> + 1 carbonyl carbon, with breaking of 
the C-N bond between <i>n</i> + 1 and <i>n</i> + 2.   The product ion is the 
ringed dipeptide and <i>y</i><sub>max-2</sub>.  The b2 ion then fragments itself 
in secondary reactions:  neutral loss of NH<sub>3</sub> H<sub>2</sub>O, and CO. 
Mass range below m/z of b2 and above m/z of <i>y</i><sub>max-2</sub> show 
sequence-specific fragments.  Note that main chain fragmentation forms pairs: 
<i>a/x</i>, <i>b/y</i>, and <i>c/z</i> as the N- and C-terminal fragments 
cleaved along &alpha;C-carbonyl C, carbonyl C-amino N, and amino-N-&alpha;C 
bonds, respectively. 
<p> 

The <i>b</i><sub>2</sub>/<i>y</i><sub>max-2</sub> ion 
is usually observed in CID spectra.  Consider the MS/MS spectrum of SEPIASTK. 
The high and low mass range are mirrored to one another such that <i>m/z</i> 
<i>b</i><sub>2</sub> is locked in place to <i>m/z</i> of 
<i>y</i><sub>max-2</sub> Turning the high mass range upside down then shows the 
mirror pairing at the same <i>x</i>-axis position.  Other than the b2/y6 pair 
for the octapeptide, no other ion pair is seen.  Mainly neutral losses are seen. 
An immonium <b>S</b> (<i>m/z</i> 60) is seen in low abundance.  A weak 
<i>y</i><sub>max-1</sub> -18 ion is also seen as fragmentation between residue 1 
and 2.  The <i>b</i><sub>1</sub> ion is not observed:  it is probably the case 
that a <i>b</i><sub>1</sub> ion of serine rapidly decomposes to an 
<i>a</i><sub>1</sub> ion. <p> Isomers of <i>b</i><sub>2</sub> ions may be 
composed of two different amino acids. Each compositional doublet is basis for 
isomeric quartet.  GT and AS are examples of such a doublet:  A contains 
methylene group more than G, and T a methylene group less than S.  As b2 ions, 
GT, TG, AS, and SA form a quartet that is structurally isomeric and with 
identical brutto formula.  when composed of the standard 20 unmodified amino 
acids, 13 quartets of b2 ions can occur. <p> Peptides forming the quartet of 
N-terminal dipeptides NE, EN, QD, and DQ with <i>m/z</i> 244 were examined. 
MS/MS spectra were recorded for the octapeptides NE-X, EN-X, QD-X, and DQ-X 
where X = SLTLNR and the mass region below <i>m/z</i> 244 examined.  All four 
peptides show a <i>b</i><sub>2</sub> ion at <i>m/z</i> 244 as expected and also 
individual immonium ion abundances.  But the signature of the NE dipeptide, for 
example, is composed of four abundant ion at m/z 244 (<i>b</i><sub>2</sub>), 226 
(<i>b</i><sub>2</sub> &minus; 18), 216 (<i>a</i><sub>2</sub>) and 199 
(<i>a</i><sub>2</sub> &minus; 17). A lower abundance signal of Asn immonium at 
<i>m/z</i> 87 is also seen. For EN dipeptide the ion signature is m/z 244, 226 
and 102, the signature of Glu immonium.  The signature of QD are the ions 244, 
227 (<i>b</i><sub>2</sub> &minus; 17), and 226 (<i>b</i><sub>2</sub> &minus; 
18).  In the latter two, neutral loss of NH<sub>3</sub> and H<sub>2</sub>O are 
high and comparable in abundance.  The Q immonium at <i>m/z</i> 101 is also 
quite abundant.  The signature of DQ shows the <i>c</i><sub>1</sub> ion at 
<i>m/z</i> 133, a characteristic observed for all peptides with Q at position 2. 
Immonium ions of the N-terminal residue also provides the ability to 
differentiate between structural isomers of this sort.  Some low mass range 
fragments were studied as to whether they were generated as 2nd generation 
fragments from b2 ions rather than as molecular or other sequence ions:  pseudo 
MS<sup>3</sup> experiments are done on the b2 ion of the isomeric quartets (NE, 
EN, QD, DQ) by increasing the inlet cone voltage to isolate b2 ions and fragment 
them in the collision cell. Peaks corresponding to b2 &minus; 17 (lose of water) 
and the a2 &minus; 17 peak are observed. 

<p> 

With all b2 ions, the corresponding 
high mass ion should be <i>y</i><sub>max-2</sub>, and this should be seen in 
spectra and is.  Signature peaks for the isomeric quartet are x8 &minus; 18 for 
NE, x8 &minus;17 for EN.  There are z8 and y8 ions with satellites at &minus;17, 
&minus;35, and &minus;53.  For QD, a y8 with only minor fragmentation is seen: 
thus N-terminal D is more stable than N-terminal N or E. <p> The rest of this 
report shows fragmentation studied in the low and high mass regions.  In 
general, the proper pairings are observed:  for <i>a<sub>h</sub></i>, 
<i>b<sub>j</sub></i>, and <i>c<sub>k</sub></i>, the complementary high mass pair 
<i>z</i><sub>max-<i>h</i></sub>, <i>y</i><sub>max-<i>j</i></sub>, and 
<i>z</i><sub>max-<i>k</i></sub>, respectively, are observed. <ul> 
<li><i>c</i><sub>1</sub>/<i>z</i><sub>max-1</sub> are associated with Q or N at 
position 2.  For Q it probably derives from <i>b</i><sub>2</sub> ion secondary 
fragmentation.  Side chain amide N attacks carbony C of position 2, and 
<i>b</i><sub>2</sub> ion with 6-centered ring forms, differing from 5-centered 
oxazolonium ring, which facilitates <i>a</i><sub>2</sub> product ion.  Ions have 
X-Q sequence show little <i>a</i><sub>2</sub> ion formation. <li>Occurrence of 
<i>z</i><sub>max-1</sub> with Q or N at position 2 indicates direct cleavage of 
peptide backbone. <li><i>b</i><sub>1</sub>/<i>y</i><sub>max-1</sub> fragment 
pairs are seen with motifs having N-terminal H or W.  b1 ions are predicted to 
be stable for basic residues (R, K, W, H).  Instability of b1 for non-basic 
residues leads generally to <i>a</i><sub>1</sub> decomposition. <li>The 
<i>a</i><sub>1</sub>/<i>x</i><sub>max-1</sub> spectra are seen in 20% of 
spectra.  Immonium ions result directly from protonated molecular ions rather 
than secondary disintegration. </ul> 
<!-- ================================================================ --> 
<p class="tags">

<p class="ref"> 
 <span class="authors"> 
Liu J, Huang T-Y, McLuckey SA 
 </span> 
 <span class="year"> 
(2009) 
 </span> 
 <span class="jname"> 
Anal. Chem. 
 </span> 
 <span class="jvol"> 
81, 
 </span> 
 <span class="pages"> 
2159-2167. 
 </span> 
<p class="reftitl"> 
Simultaneous Transmission Mode Collision-Induced Dissociation and Ion/Ion 
Reactions for Top-Down Protein Identification/Characterization Using a 
Quadrupole/Time-of-Flight Tandem Mass Spectrometer
<p class="Para1"> 
Bottom-up protein id and characterization is done by digesting protein by 
chemical or enzymatic means before mass spectrometry while top-down approach 
uses interrogation of intact protein.  Bottom-up characterizations are mature 
but information about post-translational modifications can be lost. The top-down 
connects information about molecular weight and other parameters to the protein. 
<p> 
Whole protein dissociation can be done by ion-trap CID, infrared multiphoton 
dissociation (IRMPD), collisional activation by sustained off-resonance 
irradiation.  Other dissociations involve electron capture (ECD) and electron 
transfer (ETD). 
<p> 
A population of multiply charged protein cations was passed into the Q2 linear 
ion trap filled with mix of neutral inert gas and proton transfer reagent 
anions:  both ion/ion and ion/neutral reactions take place. Proteins of high 
energy under influence of LINAC potential pass into an anion population 
pre-trapped in Q2 LIT.  The scan function is as follows: inject anions (from ESI 
source) into Q2 LIT, inject cations and transmit to orthogonal ToF.  Ion/ion 
reactions lead to charge state reduction of protein precursor and their product 
ions.  Ion/neutral collsions will lead to ion activation or deactivation, ion 
scattering and/or ion focusing with relative contributions of each depending on 
pressure and kinetic energies of ions entering Q2.  Use of low CE gives no or 
minor CID contributions. Ion/ion interactions are favored at lower CE injection.
<p>
Factors affecting the interactions include the count of anions, controlled by 
injection time as well as low mass cutoff of the Q2 LIT, buffer gas pressure, 
and LINAC potentials.
<!-- ================================================================ --> 
<p class="tags">

<p class="ref">
 <span class="authors"> 
Bilusich D, Bowie JH 
 </span> 
 <span class="year"> 
(2009) 
 </span> 
 <span class="jname"> 
Mass Spec. Rev. 
 </span>
 <span class="jvol"> 
28, 
 </span>
 <span class="pages">
20-34.
 </span>
<p class="reftitl"> 
Fragmentations Of (M-H)<sup>&minus;</sup> Anions Of Underivatised Peptides. Part 
2:  Characteristic Cleavages Of Ser And Cys And Of Disulfides And Other 
Post-Translational Modifications, Together With Some Unusual Internal Processes
<p class="Para1">
<!-- ================================================================ -->
<p class="tags">
amino acids, peptides, radical cations, copper, mass spectrometry, 
collision-induced dissociation (CID), FD
<p class="ref">
 <span class="authors"> 
Hopkinson AC 
 </span>
 <span class="year"> 
(2009)
 </span>
 <span class="jname">
Mass Spec. Rev.
 </span>
 <span class="jvol"> 
28,
 </span>
 <span class="pages"> 
655-671. 
 </span> 
<p class="reftitl"> 
Radical Cations Of Amino Acids And Peptides:  Structures And Stabilities 
<p class="Para1">
<b>REVIEW</b>.  Peptides and proteins as ions are produced often by soft 
ionization using ESI and MALDI.  The protonated peptides can be subjected to 
collision-induced dissociation (CID), breaking up into <i>b</i>- and 
<i>y</i>-type ions, which can be used in structure identification.  The 
[M+<i>n</i>H<sup><i>n</i>+</sup> ions have closed-shell electronic structures 
whose fragmentation mechanisms is extensivley reported.  Radical cations 
(odd-electron ions) [M+<i>n</i>H]<sup>&bull;+(<i>n</i>&minus;1)+</sup> are 
formed by electron capture on multiprotonated peptides with subsequent 
fragmentations whose products are not closed shell ions, but which are 
<i>c<sub>n</sub></i> and <i>z<sub>m</sub></i> ions, where the 
N&mdash;C<sub>&alpha;</sub> bond is broken.  These radical cations are difficult 
to generate in gas phase.  Electron impact methods do not work on peptides which 
have a vapor pressure too low for reagent gas.  Laser ablation followed by UV 
photoionization provides a source, but is restricted to peptides with with 
aromatic amino acids acting as chromophores.  Hu and Loo found that 
[Cu(M)]<sup>&bull;2+</sup> where M = angiotensin can fragment with loss of 
CO<sub>2</sub> from C-terminus then lose alkyl radicals from side chains when 
C-terminal residue was Leu or Ile.  Turecek et al looked at fragmentation of 
complexes of type [Cu<sup>II</sup>(diimine)(M&minus;H)]<sup>&bull;+</sup> where 
diimine was either 2,2&prime;-bipyridine (bpy) or phenanthroline and M was one 
of 17 naturally occurring amino acids or a small peptide:  all complexes lose 
CO<sub>2</sub> initially when transitioning from 4-coordinate approxly square 
planar complex to structure where that is 3-coordinate with spin located on 
C<sub>&alpha;</sub> of the amino acid residue.  In fragmenting, aliphatic side 
chains have the least complicated:  Gly and Ala produce aminoalkyl radicals 
(H<sub>2</sub>NCHR<sup>&bull;</sup>); Val, Leu, and Ile lose radicals from the 
side chain (CH<sub>3</sub><sup>&bull;</sup> from Val, 
(H<sub>3</sub>C)<sub>2</sub>CH<sup>&bull;</sup> from Leu, 
C<sub>2</sub>H<sub>5</sub><sup>&bull;</sup>, CH<sub>3</sub><sup>&bull;</sup>, 
and H<sup>&bull;</sup> from Ile).  These fragmentation differences permit 
distinction and quantification of isobaric Ile and Leu.  Complexes of amino 
acids with N, O, and S atoms in side chains have unique fragmentations after the 
loss of CO<sub>2</sub>. Dipositive 
[Cu<sup>II</sup>(diimine)(M)]<sup>&bull;2+</sup> are seen when M is Arg or Lys 
and when it is is a variety of di- and tripeptides: CID results in charge 
separation where N-terminal is lost as an <i>a</i><sub>1</sub> ion along with CO 
neutral loss for dipeptides; for tripeptides as M, b2, a2, and a1 peptides are 
observed.  Copper is observed as part of the ion as a [Cu(bpy)(amino 
acid&minus;H)]<sup>&bull;+</sup> or 
[Cu<sup>II</sup>(bpy)(dipeptide&minus;H)]<sup>&bull;+</sup>.  Chu et al (2000, 2001)
have found that [Cu<sup>II</sup>(dien)(peptide)]<sup>&bull;2+</sup> will produce
a peptide radical cation in Leu-Enk YGGFL<sup>&bull;+</sup>.  The formation of the
radical cation was thought to require a low ionization energy amino acid (like
tyrosine) and a basic one (arginine/lysine).  They facilitate formation but are not
essential.
<!-- ================================================================ -->
<p class="tags">
mass spectrometry advances
<p class="ref">
 <span class="authors">
Morrow Jr, KJ
 </span>
 <span class="year">
(2009)
 </span>
 <span class="jname">
Genet. Eng. Biotechnol. News
 </span>
 <span class="jvol">
29,
 </span>
 <span class="pages">
online article 5 pages.
 </span>
<p class="reftitl">
New MS Tools Advance Biomolecular Studies:  Simpler, Cheaper, and Faster 
Instrumentation Facilitates Inclusion in Novel Applications
<p class="Para1">

<!-- ================================================================ -->
<p class="tags">
secondary structure, anti-parallel beta sheet, amyloid peptide
<p class="ref">
 <span class="authors">
Cerf E, Sarroukh R, Tamamizu-Kato S, Breydo L, Derclaye S, Dufrne Y,
Narayanaswami V, Goormaghtigh E, Ruysschaert J-M, Raussens V
 </span>
 <span class="year">
(2009)
 </span>
 <span class="jname">
Biochem. J.
 </span>
 <span class="jvol">
Pre-published manuscript
 </span>
 <span class="pages">
XX-XX
 </span>
<p class="reftitl">
Anti-Parallel &beta;-Sheet: A Signature Structure Of The Oligomeric Amyloid-Beta
Peptide
<p class="Para1">
Amyloid beta (A&beta;) peptide misfolding is thought to be a factor in
Alzheimer's Disease.  Misfolded fibrillar amyloid peptides/proteins aggregate
in extraneuronal spaces of the cerebrovascular regions, and neurofibrillary
tangles form inside neurons.  Amyloid plaques are made of A&beta; petpide
which are 38-43 residues long, released from proteolysis by &beta;- and
&gamma;-secretase of the amyloid precursor protein (APP).  The primary event
in pathogenesis is believed to be accumulation of A&beta;.  Tau protein tangle
formation possibly results from an imbalance between production and clearance
of A&beta;.  Plaques are composed principally of A&beta;(1-42) and A&beta;(1-40);
the former is quite amyloidogenic but less abundant overall, but is probably
responsible for initial deposition, probably promoting aggregation by its
two additional hydrophobic amino acids at the C-terminal end.  A&beta; exists
in several states, from monomer to larger soluble masses and then as insoluble
fibrils.  Oligomeric assemblies basically refer to assemblies that are
dimers, trimers, protofibrils, A&beta;-derived diffusible ligands (ADDLs),
and annular or pore-like oligomers.  A recent report shows that because there
are different aggregation pathways, oligomers can be classed as prefibrillar
and fibrillar.  Although it has been presumed that fibrillar plaque formation
is associated with neurodegeneration leading to memory loss and cognitive decline,
soluble A&beta; oligomers may be more neurotoxic and play a more direct role
in amyloid pathogenesis.  A&beta; oligomers could be intermediate to fibril
formation but are not necessarily required to form fibrils.  The ability of
conformation-dependent antibodies to show fibiils was found to be independent
of sequence.  Thus structure should be examined.  A&beta; oligomers are generally
&beta;-sheet dominant.  Parallel and anti-parallel formations have been shown
previously.
<p>
This study attempts to examine A&beta;(1-42) oligomers and fibrils.  Synthetic
peptides were prepared, the peptide dissolved in cold CF3CH(OH)CF<sub>3</sub>
at 2 mg/ml, left at 1 h RT, then aliquotted.  The HFIP was evaporated by nitrogen
and left under vacuum (SpeedVac), stored at -20&deg;.  A&beta;(1-42) was
prepared by dissolving in DMSO to 5 mM.  <b>For preparing oligomers</b>,
peptides was dissolved in F12 culture medium (no phenol red dye), or placed
in 20 mM TrisHCl pH 7.4 + 100 mM NaCl (Tris-buffered saline), all to a final
0.1 mM concentration and left at 4&deg; 24 h.  Fibrils are made by suspending
peptide in 10 mM HCl to final concentration of 0.1 mM and incubating 37&deg;
24 h, or in 0.5 mM HEPES pH 7.4 at 0.5 mM and incubating room temp under gentle
agitation for at least 30 d.
<p>
A&beta;(1-42) produces SDS-resistant oligomers and fibrils
<!-- ================================================================ -->
<p class="tags">
alpha helix, laser desorption ionization, mass spectrometry
<p class="ref">
 <span class="authors">
Shigeri Y, Fujii S, Obana R, Hagihara Y, Jitosho M, Kawasaki H, Arakawa R
 </span>
 <span class="year">
(2009)
 </span>
 <span class="jname">
J. Mass Spectrom.
 </span>
 <span class="pages">
<a href="http://dx.doi.org/10.1002/jms.1574">http://dx.doi.org/10.1002/jms.1574</a>
 </span>
<p class="reftitl">
Ionization Efficiency Of Alpha-Helical Peptides In Laser
Desorption/Ionization Mass Spectrometry
<p class="Para1">
<b>LETTER</b>.  A major problem in proteomic analysis is that &lt; 50-80% of
peptides from proteolysis are detected by mass spectrometry.  This leads to
lower confidence of protein identification and reduced sensitivity of PTMs.
Factors that affect ionization include molecular size, hydrophobicity,
ionizability of side chains, aromatic amino acid and secondary structure.
<p>
Neuropeptide Y (NPY) is 36 aa-long peptide that regulates food intake, vasoconstriction,
and neurotransmitter release; it may also have antimicrobial activity. NPY
tertiary structure possesses a polyPro helix (aa 1-8), a &beta;-turn (aa 9-13),
an amphipathic &alpha;-helix (aa 14-32).  Synthetic peptides and their analogs
reveal that the antimicrobial activity is in the helix.  Synthesized peptides
were made by Fmoc solid phase synthesis and HPLC-purified.  The identity and
purity of NPY and its analogs can be confirmed easily with MALDI-MS.
<p>
Wenschuh et al (<i>Rapid Comm Mass Spectrom</i> <b>12</b>: 115, 1998) showed
that &alpha;-helical peptides have lower ionization efficiency.  Their experiments
used two &alpha;-helical peptides and D-amino acid-substituted petpides as
internal standards.  The peptide secondary structure was studied by CD spectroscopy
under conditions used in MALDI-MS.  This study synthesized four sets of peptides
having &alpha;-helical structure  having different sequences.  Each set was
five peptides having different helical content but same amino acid composition
(achieved by positioning a Pro residue in different position).  MALDI-MS,
SALDI-MS (surface-assisted laser desorption/ionization), and DIOS (desorption/ionization
on porous silicon) were used to measure ionization efficiency.  Peptides containing
Tyr or Trp residues could be used to determine mass and purity using spectrophotometry
at 280 nm with extinction of 1280 and 5690 M<sup>-1</sup> cm<sup>-1</sup>,
respectively.  CD measurements were mad with JASCO J-820 spectropolarimeter
using 30 &micro;M peptide in a buffer of 1:3:4 (v/v/v) 22 mg/ml anhydrous citric
acid,  12 mg/ml ammonium citrate, and acetonitrile.  CD measurements were
accumulated 8 times and averaged using same samples; three temps (10, 25, and 37&deg;)
were investigated.  The equation of Forood et al:
<p class="eqnserif">
f<sub>H</sub> [<i>&theta;</i>]<sub>222</sub> / [&minus;40,000 &times; (1 &minus;
2.5/<i>n</i>)] &times; 100
<p>
was used to determine &alpha;-helical content:  f<sub>H</sub> is percent &alpha;-helix,
<i>n</i> is number of amino acid residues, and [<i>&theta;</i>]<sub>222</sub>
is ellipticity in deg cm<sup>2</sup> dmol<sup>-1</sup> at 222 nm.
<p>Matrix and peptides were prepared in the citrate buffer and spotted in
triplicate.  Each one of replicates had three spectra determined from one location
in a spot at different locations, producing nine spectra from a single spot.
The peak height of a deuterium-labeled peptide form was given as a 100% relative
intensity, present as an internal standard.  In general a 10 pmol/&micro;l
sample peptide and 10 pmol/&micro;l internal standard is mixed in equal volume with
10 mg/ml CHCA matrix, and 1 &micro;l spotted on the plate.  For SALDI-MS, 1
&micro;l of a 5 pmol/&micro;l sample peptide and 5 pmol/&micro;l internal peptide
are dried as a drop on stainless steel sample plate.  Then 1 &micro;l of Au
nanoparticle (20-30 nm) was added according to Watanabe et al (J. Mass Spectrom.
43: 1063, 2008).  For DIOS-MS, n-type silicon (100) wafers having resistivity
&rho; = 0.011-0.019 &Omega;-cm were anodically etched in 1:1 (v/v)
ethanol-46% hydrogen fluoride under a 100 W tungsten filament white light bulb
from 8.5 cm distance, with a 6 mA cm-2 current for 15 s, rinsed in EtOH and dried
in vacuum. One &micro;l of solution containing 5 pmol/&micro;l each peptide was
applied to the chips attached to stainless steel sample plate with conductive
double-sided tape  (see Kinumi et al).
<p>
The four representative &alpha;-helical peptides are NPY(13-36)
[PAEDLARYYSALRHYINLITRQRY-nh<sub>2</sub>, (M+H)<sup>+</sup> = 2981.6],
TD [TPAEDAAAAYKA-nh<sub>2</sub>, (M+H)<sup>+</sup> = 1176.6],
Vpr(13-33) [EPYNEWTLELLEELKSEAVRH-nh2, (M+H)<sup>+</sup> = 2583.3], and
Lyutide S3 [YMSEDELKAAEAAFKRHGPT-nh2, (M+H)<sup>+</sup> = 2249.1].  CD and/or
NMR determinations were made to verify helical content, either by this study
or others.  Since all peptides have Tyr, Trp, or Cys, spectrophotometry at 280 nm
could be used to quantify them.  For all peptides, the proline residue was
placed at different positions to break and affect helical content.
Tri-deuterated Ala or Leu were used to label all peptide sets with a stable
isotope to follow as internal standard.  In addition, the reversed sequence
of all peptides was synthesized.
<p>
CD measurements showed that helical content dropped off sharply as the
proline residue was moved from the N-terminus to the center;  when placed
at the C-terminus, rarely does the helical content restore compared to when
it is located N-terminus.  Reversed sequence peptides consistently showed
lower helical content, and helix formatin was temperature-dependent, with
better helix formation at lower temperatures and reduced formation at higher
temperature.
<p>
Ionization techniques were varied.  MALDI-MS is the standard, but SALDI-MS
typically uses nanomaterials such as gold nanoparticles with sample prep being
very simple, having low noise in background, high salt tolerance and fast data
collection.  DIOS-MS is matrix-free with electrochemically etched surface which
produces the ionization.  For SALDI-MS, citrate buffer as used here is ideal
for gold nanoparticles, and so it was used also in MALDI-MS.  All spectra showed
the ability to resolve NPY(13-36) and its deuterated internal standard (12 Da
higher).  Plots of relative intensity against percent helical content for the
three ionization modes were then constructed.  As helical content of NPY(13-36)
increases, ionization efficiency falls for MALDI, SALDI, and DIOS, most
remarkable for MALDI.  With the other peptides (TD, Vpr(13-33), and Lyutide),
there was a smaller <em>positive</em> rather than negative correlation
between efficiency and helical content.  In other words, as helical content
increased, so there were slight increases in relative intensity.
<p>Wenschuh et al synthesized KLAL and NPY peptides as &alpha;-helix peptides
and VT5 and VT6 as &beta;-sheet formers to examine ionization efficiency, in
addition to D-amino acid substitutes which break structure.  They concluded
based on the lower ionization efficiencies of stable &alpha;-helix and
&beta;-sheet formers that stable secondary structure affects ionization.
With exception of NPY, there was no strong indication that secondary structure
truly affects ionization.  If secondary structure is not truly a factor, then
what accounted for the correlation between helical content and ionization in
NPY?  For one thing, the isoelectric point of NPY is radically different from
the other peptides, Theoretical pI is 9.5 for NPY and ranges from 4.3 to 5.5
for others.  This explains perhaps the large


<!-- ================================================================ -->
<p class="tags">
electrochemistry, transition metal complexes
<p class="ref">
 <span class="authors">
Toma SH, Alexiou ADP, Toma HE, Arakia K, Eberlin MN
 </span>
 <span class="year">
(2009)
 </span>
 <span class="jname">
J. Mass Spectrom.
 </span>
 <span class="jvol">
44,
 </span>
 <span class="pages">
361-367.
 </span>
<p class="reftitl">
Can mass dissociation patterns of transition-metal complexes be predicted from 
electrochemical data?
<p class="Para1">
ESI permits analysis of many thermally instable and highly polar compounds by 
gentle ionization.  Cooks et al have shown that tandem MS can be used to assess
bond energies in gas phase from the CID patterns by measuring relative abundances
of CID dissociation patterns, and relative abundances of fragments ions result from
cleavage of specific bond in comparison to a refernce bond.  It is based on the
equation:
<p class="eqnserif">
<b>ln</b> (<i>k</i><sub>1</sub>/<i>k</i><sub>2</sub>) = 
(&Delta;<i>G</i><sub>[A-B]</sub> &minus; 
&Delta;<i>G</i><sub>[A-C]</sub>) / <i>RT</i>
<p>
where <i>k</i><sub>1</sub> and <i>k</i><sub>2</sub> are bond dissociation rates 
and &Delta;<i>G</i><sub>[A-B]</sub> and 
&Delta;<i>G</i><sub>[A-C]</sub> are bond free energies, respectively,
for cleavage of the test bond and the reference bond.  Thus in molecule BAC, 
there is a competition in fragmentation between formation of AB and AC fragments.
Thus <i>k</i><sub>1</sub> is rate constant for dissociation of B-AC and 
<i>k</i><sub>2</sub> rate constant for C-AB.  The equation can be re-described
as:
<p class="eqnserif">
<b>ln</b> (<i>s</i> <i>A</i><sub>(AC)</sub> / <i>A</i><sub>(AB)</sub>) = 
(&Delta;<i>G</i><sub>[B-AC]</sub> &minus; &Delta;<i>G</i><sub>[C-AB]</sub>) / 
<i>RT</i><sub>eff</sub>
<p>
where <i>s</i> is statistical factor proportional to number of B-AC and C-AB
in parent species (in the example, <i>s</i> = 1), and <i>A</i><sub>(AC)</sub>
and <i>A</i><sub>(AB)</sub> are integrated signals for AC and AB fragments
obtained by CID-MS, and <i>T</i><sub>eff</sub> is so-called effective temperature.
(see Ervin, Int J. Mass Spectrom 195/196, 271; 2000).
<!-- ================================================================ -->
<p class="tags">
amino acid analysis
<p class="ref">
 <span class="authors">
Kaspar H, Dettemer K, Gronwald W, Oefner PJ
 </span>
 <span class="year">
(2009)
 </span>
 <span class="jname">
Anal. Bioanal. Chem.
 </span>
 <span class="jvol">
393
 </span>
 <span class="pages">
445-452
 </span>
<p class="reftitl">
Advances in Amino Acid Analysis
<p class="Para1">
<b>REVIEW</b>.  Quantitative analysis of amino acids often required in several
circumstances.  This review covers the methods.
<p>
<b>Sample Preparation</b>.  To analyze free amino acids, protein removal is
necessary.  Can be done by precipitating with acids or organic solvent or
by ultrafiltration.  A common method is preciptating with sulfosalicylic acid.
Amino acids alone do not provide useful chemistry for separation as highly polar
substances:  thus reversed-phase and GC methods require derivatization.
Capillary electrophoresis can be used, but detection is not possible in the UV
unless a UV-labeling agent is used to enhance sensitivity.  Thus amino acis
are usually derivatized, on the primary amino group and less so the carboxyl
group.  Any labeling agent working on amimes should also label secondary amines
on Pro and HO-Pro.  The following derivatization agents are common:
<table>
<tr><th>Reagent <th>Detected Product of Derivatization
<tr><td>Ninhydrin  <td>Ruhemann's Purple
<tr><td><i>o</i>-Phthaladehyde (OPA)  <td>
<tr><td>Phenylisothiocyanate (PITC, PicoTag) <td>
<tr><td>6-Aminoquinolyl-N-hydroxysuccinimidyl carbamate (AQC, AccQ-Tag) <td>
<tr><td>2,5-Dioxopyrrolidin-1-yl-2-(4-methylpiperazin-1-yl)acetate (iTRAQ) <td>
<tr><td>N-methyl-N-trimethylsilyltrifluoroacetamide (MSTFA) <td>
<tr><td>Alkylchloroformate <td>
<tr><td>Fluorescein isothiocyanate (FITC) <td>
</table>
<p><b>LC methods with optical detection</b>.  Some form of high resolution
ion-exchange chromatography with post-column derivatization or reversed-phase
HPLC with pre-column derivatization is usual.  The gold standard is cation-exchange
with lithium buffer followed by post-column derivatization with ninhydrin and
UV detection.  With ninhydrin, primary amines generate Ruhemann's purple detected
in UV and at &lambda;<sub>max</sub> = 570 nm, and secondary amines at
&lambda;<sub>max</sub> = 440 nm.  Eluate needs to be monitored at 440 and 570 nm,
and separation is complete in 2 h.  OPA derivatization can be used post column
with WCX or precolumn with RP-HPLC; it requires a thiol (2-mercaptoEtOH) to
form fluorescent derivative detectable in UV at 340 nm or by flourescence with
excit = 340 and emiss=450 nm, and sometimes by amperometry.  Other precolumn
derivatizations includ PITC, dimethylamino-azobenzensulfonyl chloride (DABS-Cl),
9-fluorenylmethylchloroformate (FMOC-Cl), and 7-fluoro-4-nitrobenzo-2-oxa-1,3-diazole
(NBD-F).  Derivatization time varies from 1 min for OPA to 20 min for PITC.
Chromatographic run times range from 13 min for 23 compounds to 95 min for 38 cmpds.
PTH derivatives can be run on HPLC and be detected at 254 nm; basis for PicoTag
method produced by Waters.
<!-- ================================================================ -->
<p class="tags">
TICL, metabolomics
<p class="ref">
 <span class="authors">
Antonov AV, Dietmann S, Wong P, Mewes HW
 </span>
 <span class="year">
(2009)
 </span>
 <span class="jname">
FEBS J.
 </span>
 <span class="jvol">
276,
 </span>
 <span class="pages">
2084-2094.
 </span>
<p class="reftitl">
TICL  a web tool for network-based interpretation of compound lists inferred by 
high-throughput metabolomics
<p class="Para1">
Genomics and proteomics produces a global list of genes and proteins as a way
of examining complex molecular phenomena.  Bioinformatics and statistical 
methods are used in interpretation.  Expression differences arrive at a 
<i>p</i> value to determine probability that any observation could appear
at random.  The <b>Kyoto Encyclopedia of Genes and Genomes</b> or
<a href="http://www.biocarta.com/">BioCarta</a> have been used in analysis
of metabolic and signaling pathways.  Scoring functions that take into account
the distance between genes within metabolic pathway have been exploited;
the impact of a pair of genes is weighted with respect to distance between
genes in the metabolic pathway.  Draghici et al have proposed <q>impact
analysis</q> which goes beyond gene pairs and looks at topology of signaling
pathways; it propagates perturbations measured at gene levels through entire
pathway. Most tools for metabolomics deal with mass peak annotation problem.
MassTrix server recently developed give possiblity of uploading  high
precision mass spectrum, automatic annotation of of mass peaks, mapping 
identified substances to KEGG metabolic pathways.  KEGG atlas accepts list of 
compounds as input, and present graphical visualization in context of global
metabolic reaction network; it does not provide quantitative or statistical
analyses.  Assessing relationship quality requires knowledge of relative or
absolute quantitites.  TICL is a web tool that provides an analytical framework
for interpreting molecular mechanism  that unit a list of compounds.
<p>
Lu et al recently reported upon metabolic variation related to age-related
conditions and hypertension.  The spontaneous hypertensive rat (SHR) and
normotensive Wistar Kyoto control were investigated.  Blood plasma was
analyzed by GC-ToFMS.  After deconvolution, 187 peaks were quantitated,
78 identified.  Many identified compounds showed signficant variations among
all compounds and the subset identified.  Table 2 in Lu showed ~20 with 
significant decreases and increases from 10&rarr;18 weeks age in both SHR
and WKY rats.  16 compounds were mapped to global compound network inferred
from KEGG.  The KEGG figure (shown) would indicate no relationship among 
compounds and they have no specific canonical metabolic pathway, and visual
analysis can nto give a clear answer about how compounds are related.  When 
the list is sumbmitted to TICL, the quantiative values describe the relation
quality with confidence score in terms of <i>p</i> value.
<p>
Table 1 shows the <q>Enriched subnetworks</q> report with six models, the
maximum distance between compounds, count of input compounds in subnetwork,
and the <i>p</i> value.  Model 1 says that only two compounds from input
from list have a substrate-product relationship.  Model 2 allows for one 
missing compound, and so now four of the input list members are connected
in a network.  With model D<sub>5</sub>, four intermediate compounds can be
absent and 11 metabolites from list are integrated to network.  The <i>p</i>
values for each model are computed using Monte Carlo procedure.  Note the
D<sub>5</sub> model was most significant of all models (<i>p</i> &lt; 0.01).
<p>
TICL allows visualization.  A preformatted text file is downloaded and uses
the MEDUSA package.  Figure 2 shows model D<sub>5</sub>.  Lu reported
that 22 compounds were significantly different between SHR and WKY.  
14 compounds can be mapped to global compound network inferred from KEGG.
Again their locations on the KEGG map show no relation visually.  When 
subjected to TICL modeling.  Unlike the first set data, the second set
is not a statistically robust transformation network.  Model D<sub>6</sub>
allows up to five missing compounds between any two, but only 8 metabolites
from input can be integrated.  Unfortunately all models show no statistical
significance:  thus any relationship between compounds can be judged to have
occurred randomly.
<p>
TICL was applied to another case.  66 invasive ovarian carcinomas and 9
borderline tumors were analyzed using GC-ToF MS with automated mass
spectral deconvolution.  291 metabolites were identified, and 114 (39.1%)
were annotated known compounds.  51 metabolites were signficantly different
between borderline tumors and carcinomas (t test, <i>p</i> &lt; 0.01).
Denkert et al reported 26 significantly different metabolies, 21 known and
mapped to global metabolic network.  Again looking at model and the number
of compounds that become included in the network and the <i>p</i> value,
three models include 19 of 21 with <i>p</i> &lt; 0.01.  The first to do it
is model D<sub>4</sub>.
<p>
In another clinical cancer study, paired samples of normal and colorectal
cancer tissue were investigated by GC ToFMS with 206 metabolites detected,
and 82 were found significantly different.  When analyzed in TICL modeling,
almost all identified known metabolites are dependent.  Model D<sub>2</sub>
with one missing covers 37 in the list;  model D<sub>3</sub> with two
missing covers 49 inputs.  All models were significant (<i>p</i> &lt; 0.01).
In both cancer-related examples, TICL show how identified metabolites
represent a set of dependent compounds.  Although analyzed cases covered
different tissues, in both cases the discovered metabolic markers were not
independent; they define a related set of metabolic reactins which in turn
define a semi-noninterrupted network of metabolic transformations that covers
most of the identified compounds.
<p>
<b>Method</b>.  A list of compounds that are differentially present in
biological samples is developed as input.  The software creates a network
model by reconstructing more probable transformation/reaction routes;
this is a shortest-path-between-two-compounds problem, but is done for the
entire list.  The global metabolic metwork in the KEGG database is how
inference is done.  The contribution of TICL is to implement a robust statistical
framework to estimate the significance of the inferred model.  Monte Carlo
simulation infer the <i>p</i> values, being probablity that a random list
would have the same quality model.  KEGG Reaction Database is collection of
chemical structure transformation patterns for substrate-product (reaction)
pairs.  A global reaction network where reactions are nodes and compounds are
edges with connections of edges and reactions sharing same compounds.
A reaction has multiple reactant pairs, and the one appearing in KEGG metabolic
pathway is main pair.   Only compounds classified as main pairs are used in
building global reaction network.
<p>
The global compound network and the list of compounds are at the start of the 
procedure.  The distance between two compounds is the minimum number of 
consecutive steps required to go from one compound to another by working through 
existing paths on the global compound network.  Distance 1 means directly 
connected (substrate-product relationship), Distance 2 means connected by one 
intermediate, and so on.  The point is to miminize the distance between 
connected pairs of compounds.  Compunds are first mapped to global compounds 
network.  Then all compounds with distance 1 are connected by edges, and then 
connected subnetworks are sought. Subnetwork with maximal number of compounds is 
inferred network model D<sub>1</sub>.  In second step compounds with distance 2 
connected by edges, and subnetwork with maximal number of compunds is inferred 
then designated D<sub>2</sub>.  And the evaluation proceeds for other distances.
<p>
Assume an input compound list of size <i>N</i> using network inference procedure 
described above with network models D<sub>1</sub>, ..., D<sub><i>z</i></sub> 
allowing for 0, 1,...,<i>z</i> &minus; 1 intermediate compounds to be added to 
the model.  Denote S<sub>1</sub>, S<sub>2</sub>, ...,S<sub>z</sub> to be number 
of input compounds in inferred network models (these are the <q>size</q> of 
respective models D<sub>1</sub>,...,D<sub>z</sub>).  The S values will in effect 
be quality measures of the models.  It is necessary to estimate probability of 
inferring models of the same or larger size from randomly generated compounds 
lists of size <i>N</i>.   To estimate significance of inferred models, the 
values S<sub>1</sub>, S<sub>2</sub>, ... S<sub>z</sub> are compared with 
background distributions BD<sub>1</sub>,...,BD<sub>z</sub> which are computed 
uisng Monte Carlo simulation.  Background distributions BD<sub>1</sub>, ..., 
BD<sub>z</sub> are generated by repeating the simulation procedure for <i>k</i> 
times until an upper significance level is reached. A random gene list 
L<sub><i>j</i></sub> of size <i>N</i> (equal to size of the input list) is 
generated by sampling compounds from global compound network. Index <i>j</i> = 
1...<i>k</i> specifies each of the <i>k</i> random simulations. The network 
inference procedure described above is applied to the random list 
L<sub><i>j</i></sub> and the network models D<sub>1</sub>,...,D<sub>z</sub> are 
inferred. The size (number of input compounds) of inferred models 
D<sub>1</sub>...D<sub>z</sub> for random list L<sub><i>j</i></sub> as 
R<sub>1<i>j</i></sub>, ..., R<sub><i>zj</i></sub>.  After repeating the 
simulation procedure <i>k</i> times, the background distribution 
R<sub>1<i>j</i></sub> (<i>j</i> = 1 ... <i>k</i>) for models D<sub>1</sub>,
background distribution R<sub>2<i>j</i></sub> (<i>j</i> = 1 ... <i>k</i>) for models D<sub>2</sub>, and
backgound distribution R<sub><i>zj</i></sub> (<i>j</i> = 1 ... <i>k</i>) for
models D<sub><i>z</i></sub>.  The significance estimate for model D<sub>1</sub>
for input gene list compares S<sub>1</sub> to distribution R<sub>1<i>j</i></sub>.
<i>n</i> is number of values from distribution R<sub>1<i>j</i></sub> that
are &ge; S<sub>1</sub>.  Estimate of <i>P</i> from inferred network model
D<sub>1</sub> is <i>P</i> = (<i>n</i> + 1) / <i>k</i>.  This is likewise done
for all other models.  The <i>P</i> value is estimated as the share of random
simulations where size of inferred models for random compound lists of size
<i>N</i> are &ge; S<sub>1</sub>, S<sub>2</sub>, ..., S<sub><i>z</i></sub>
of inferred models for input compound list (size <i>N</i>).
<!-- ================================================================ -->
<p class="tags">
saliva, diabetes
<p class="ref">
 <span class="authors">
Cassiday L
 </span>
 <span class="year">
(2009)
 </span>
 <span class="jname">
J. Proteome Res.
 </span>
 <span class="jvol">
8,
 </span>
 <span class="pages">
3.
 </span>
<p class="reftitl">
Nothing to spit at:  the salivary proteome of type 2 diabetes
<p class="Para1">
Type 2 diabetes mellitus is reaching near-epidemic proportions, so reliable
noninvasive tests are more important than ever.  Typically blood from fasting
patients, sometimes with glucose tolerance testing, is required.  Roberts et
al. have recently compared the salivary proteome of type 2 patients with
healthy controls, which were published in J. Periodontol. Res.   Current
diagnostic tests in early detection include impaired fasting glucose (IFG),
impaired glucose tolerance (IGT), and hemoglobin A1C, all related to glucose
metabolism.  Whole saliva from 5 healthy and 5 type 2 patients were pooled
and run on 2DLC-ESI-MS/MS.  487 unique proteins were found 33% for first time
in human saliva.  Quantitation was done by spectral counting.  65 proteins
were differentially abundant in controls and diseased patients.  Many of these
proteins relate to metabolism and immunity.  Relative abundances of some of the 65
proteins checked by western and ELISA.  Protein levels were also checked for
those tested by IGT, or IGT+IFG.  Levels of alpha 1-antitrypsin and
alpha-2 macroglobulin correlate with disease progression.  A1AT in saliva
increases progressively as follows:  control &lt; IGT &lt; IGT + IFG &lt;
type 2 diabetes.
<!-- ================================================================ -->
<p class="tags">
support vector machines, secondary structure prediction
<p class="ref">
 <span class="authors">
Kountouris P, Hirst JD
 </span>
 <span class="year">
(2009)
 </span>
 <span class="jname">
BMC Bioinform.
 </span>
 <span class="jvol">
10,
 </span>
 <span class="pages">
437-450.
 </span>
<p class="reftitl">
Prediction of backbone dihedral angles and protein secondary structure 
using support vector machines
<p class="Para1">
Secondary structure prediction is crucial to a full structural characterization
of proteins.  The last 20 years has seen much more accurate prediction using
evolutionary information and machine learning algorithms.  In 1988 the use
of artificial neural networks (ANNs) was pioneered.  Feed-forward back-propagation
ANN, bidirectional recurrent ANN, cascade-correlation ANN, and cascaded ANN
with linear discriminant analysis have been tried.  PHD and PSIPRED use
multi-layer feed-forward ANNs with 77-78% predictive accuracy.  Analyses
also used hidden Markov models, multiple linear regression, and non-linear
dynamic systems.  Other systems used JPRED.  In 2001 support vector machine
method (SVM) was applied.  PMSVM enhances prediction with a dual-layer SVM
approach.  YASSPP improves SVM-based predictions with position-specifi and 
nonposition specific information.
<p>
Backbone dihedral angles &phi; and &psi; provide info about 3D structure,
ranging from &minus;180&deg; to +180&deg;.  Both sequence and NMR chemical
shift data can accurately determine dihedral angle, and angles can be used 
to give the 3D.  Rosetta server is the most successful for 3D prediction,
using prediction from HMMSTR of the 2&deg; structure and dihedral angles,
and an alphabet of 11 states describes them.  DeBrevern and coworkers have
used self-organizing maps and hidden Markov models to identify a structural
alphabet of 16 <q>protein blocks.</q>
<!-- ================================================================ -->
<p class="tags">

<p class="ref">
 <span class="authors">
Jansen JJ, Smit S, Hoefsloot HCJ, Smilde AK
 </span>
 <span class="year">
(2009)
 </span>
 <span class="jname">
Phytochem. Anal.
 </span>
 <span class="jvol">
21,
 </span>
 <span class="pages">
48-60.
 </span>
<p class="reftitl">
The Photographer and the Greenhouse: How to Analyze Plant Metabolomics Data
<p class="Para1">
Multivariate statistics textbooks have plenty of examples but may need more
on metabolomics.  Several reviews on use of statistics in metabolite data
are published, but they lack step-by-step descriptions
<p>
<b>Models</b>.  Metabolomic data sets are too huge to interpret without 
computer analysis.  The dictionary has two senses for it.  The 2nd sense
is that a model is a description or analogy used to help visualize something...
that cannot be directly observed.  The model for a plant is that it has
green parts above the soil line, and that the root is a white things used to
extract nutrients and water from soil.  However this would not describe algae.
Metabolites are substances varying in concentration.  Some of this might be 
caused by the perturbations of the epxeriment itself.
<p>
<b>Greenhouse</b>.  What relationship between nutritional regime quality and
plant development?  As an experiment, one might grow isogenic plants and restrict
water in half of them.  By growing in a greenhouse, all environmental factors
are kept constant.  <b>Nutrient regime</b> is the independent variable, c
chosen by researcher.  Suppose three properties are measured:  weight in milligrams,
leaf area (cm2), color (reading of UV/vis spectra).  These are dependent 
variables.  Plant positions are changed in greenhouse.  A floating ball with
three arrows all orthogonal to each other.  Plants are labeled with description
of their nutrient regime.  A photographer is employed to image the plants from
every angle, with all images being photographic perfection.  He is not a
plant biologist, so he cannot bring bias to the project.  The photograper
is a <q>mathematical model</q> because he transforms data to interpretable
models, being the photographs.  The photographer, not being an expert, is
sent back to the greenhouse to get more photos according to instructions they
give him, and this final stage is the <q>validation</q> of the models.
The <i>data preprocessing</i> part of the project where researchers assist
the photographer so that the view of plants is optimal.  As for the data, 
there are the dimensions of the greenhouse, but also plant color, weight
and leaf area.  Plant color is not a single number right now, but hundreds
of wavelengths.
<p>
<b>Alignment</b>.  Suppose on the color property that "green" is the only
one of value.  Suppose that the reflectance at 510 nm is of interest.
Some plants reflecting at 520 nm because of different phytochemistry will
have total intensity of reflectance underestimated.  To compensate,
520 nm reflectance is <i>aligned</i> to 510 nm, makign them comparable. Now
after alignment greeness = max height of aligned peak of spectrum.  Alignment
in NMR becomes necessary when peaks shift becase of slight changes in
sample enviroment during measurement (pH).  Forshed et al (2005) described
<i>bucketing</i> to solve NMR data shifts.  Retention time shifts in chromatography
use alignment.  Software alignment packages perform functions such as 
baseline correctoin, scaling, and peak picking.
<p>
<b>Normalization</b>. Color measurement now made of aligned spectra, and
greeness variable is extracted, but values cannot be compared unless response
magnitude queal for every measurement.  To enable this, a <i>normalization</i>
is done:  perhaps by attaching a lamp to the plant pot representing the 
standard light for making reflectance measurements, and the variation to the
standard light is an internal standard to correct greeness values.  Normalization
is often necessary with GC-MS and NMR data.  An alternative to internal standard
normalization is the taking an total area under every spectrum, and normalzing
to a constant sum to make machine response comparable.  Could have detrimental
effects on subsequent modeling when dynamic range between different metabolite
levels is large.  however an easy-to-implement method has been proposed 
(see Johansson and Wold, 1984).
<p>
<b>Centering</b>.  Color is now a single value.  Now the location of plants
in greenhouse.  For info-rich photography, the plants should be distributed
equally around entire greenhouse.  A <i>zero point</i> is chosen for measuring
weight, leaf area, and greenness (it is origin of axis).  The center ball
described above is placed here; all photos are to be centered on the ball.
Putting zero point on corner is intuitive, but is not desirable since the 
properties could cause positioning to create unoccupied areas of greenhouse.
If the ball is centered in the greenhouse, rather than representing the
<i>zero point</i>, it might instead represent the mean values of all the
three specified properties (a process called <q>mean centering</q>).
<p>
<b>Scaling</b>. When centering the plant weight variation was assumed to be
equal to leaf color.  Greenness might be expressed on scale of 0 to 1,
while plant weight varies from 1000-1500 mg.  Since the greenhouse would be
of skyscraper height with that numeric range, if photographer zooms out to
get weight, the color and leaf area would be obscured in photograph.
To compensate for the numeric variation, each variable can be divided by
its standard deviation, leading to a greenhouse with similar height, width,
and depth.  The relative position of the plants is now adjusted by scaling,
was not affected by centering.  Scaling must be done with caution however, 
noise may be amplified and confounding variables.
<p>
Multivariate analysis methods are applied for the purpose of looking at 
relationships between levels of different metabolites.  In <i>component 
models</i>, relationships between dependent variables are used to simplify
description of the data.  Components or latent variables are fitted to data
that are combinations of all measured metabolites.  No knowledge of relationship
among metabolites is necessary to fit components.  There might be a relationshp
between dependent variables:  greater weight may be related with greater leaf
area, and interest in greenness may emerge.
<p>
Methods can be <b>unsupervised</b> in that independent variables are not
used and only dependent variables used in modeling.  All variation in the 
plant is factored whether it comes from experimental effects or other sources.
<b>Supervised</b> methods in contrast focus on describing the subset of
variation related to the independent variable; that caused by the nutritional
regime.  Validation of variations found in treatment groups to ensure they
result from experimental manipulations.
<p>
The objective reflects how the photographer chooses position in the greenhouse:
in supervised methods he can see and use labels indicating the nutritional
regime, but unsupervised methods the labels are removed so he has no knowledge.
Only the photo angle differs between modely types:  position of plants in
greenhouse is same and focus is on center ball with arrows being equal.
<!-- ================================================================ -->
<p class="tags">

<p class="ref">
 <span class="authors">
Roessner U, Browne J
 </span>
 <span class="year">
(2009)
 </span>
 <span class="jname">
Biotechniques
 </span>
 <span class="jvol">
46,
 </span>
 <span class="pages">
363-365
 </span>
<p class="reftitl">
What is metabolomics all about?
<p class="Para1">
REVIEW.  Metabolites act as a spoken language, broadcasting signals
from genetic architecture and environment;  provide a functional readout
of physiological state of an organism.  Major tools are chromatography 
coupled to MS or NMR.  Another challenge is extracting information and 
interpreting data.  Four conceptual approaches:  target analysis, 
metabolite profiling, metabolomics, metabolic finger/foot-printing.
Targeted analysis essentially applied for many decades and involves 
identifying and quantifyin small sets of known metabolites usually using
a single analytical technique.  Profiling involves analyzing a larger set
of compoounds, both identified and unknowns:  plants, microbes, urine,
plasma have been studied.  
<!-- ================================================================ -->
<p class="tags">

<p class="ref">
 <span class="authors">
Yan W, Apweiler R, Balgley BM, Boontheung P, Bundy JL, Cargile BJ, Cole S, Fang 
X, Gonzalez-Begne M, Griffin TJ, Hagen F, Hu S, Wolinsky LE, Lee CS, Malamud D, 
Melvin JE, Menon R, Mueller M, Qiao R, Rhodus NL, Sevinsky JR, States D, 
Stephenson JL, Than S, Yates JR, Yu W, Xie H, Xie Y, Omenn GS, Loo JA,  Wong DT 
 </span>
 <span class="year">
(2009)
 </span>
 <span class="jname">
Proteomics Clin Appl.
 </span>
 <span class="jvol">
3,
 </span>
 <span class="pages">
116-134.
 </span>
<p class="reftitl">
Systematic comparison of the human saliva and plasma proteomes
<p class="Para1">
Saliva is important biofluid produced in glands and eventually mixed with
many other components in oral cavity.  Abnormalities in saliva are known.
Sj&ouml;gren's syndrome is an automimmune leading to salivary production decreases.
42 protein were found to be significantly elevated in Sj&ouml;gren's.  Oral 
cancers also show increased significant changes.  Five proteins were shown to be
elevated in oral cancers using LC-MS/MS.  Salivary mucins show changes in 
glycosylation in cystic fibrosis.  Levels of amylase and IgA are up in diabetics.
<p>
A problem with cataloging proteins are differences in protein levels (abundance)
and the existence of PTMs.  With proteomics technologies, reports have come
of protein IDs from hundreds to over 1000.  A <a href="http://www.hspp.ucla.edu/">project</a> to 
catalogo proteoms of parotid and submandibular and -lingual glans identified
1166 protein, 914 from parotid, and 917 in SM/MS, with 665 common ones.
<p>
Comparison of <b>saliva</b> and <b>blood plasma</b> proteomes is useful because:
<ul>
<li>overlap of proteins in both proteomes could be diagnostic alternative to
blood tests.
<li>changes in both proteomes can be correlated and diagnostic criteria 
established
<li>plasma proteome most extensively characterized at this time
</ul>
<p>The international HUPO Plasma Proteome Prject compiled 3020 distinct proteins
with two unique peptides per protein.  889 are high confidence identifications
based on statistical filters that adjust for protein lenght and multiple comparisons
testing.
<p>

To make comparison, the <a 
href="http://www.bioinformatics.med.umich.edu/hupo/ppp">HUPO plasma proteome 
dataset</a> was used (Adamski et al, Proteomics 5: 3226, 2005) with 3020 
proteins. Available also at <a href="http://www.ebi.ac.uk/pride">European 
Bioinformatics Institute</a> and incorporated into <a 
href="http://www.peptideatlas.org/">Peptide Atlas</a> at Institute for Systems 
Biology.  For the saliva, whole unstimulted saliva was collected from 4 healthy, 
1 ml spin in refrig microfuge for 30 min, quantified by BCA using BSA standard. 
The supernatants combined in equal volumes (200 &micro;l), brought to 0.1 M 
HEPES pH 8.0 with 5 mM TCEP and incubated 20 &micro;g trypsin overnight.  An RP
SepPak was used to concentrate and desalt peptdies and dried by vacuum centrifuge.
Preparative IEF was done using BD Biosciences free-flow electrophoresis
(see Xie et al, Mol. Cell. Proteomics 4: 1826, 2005).  This is followed by
using of Polysulfoethyl SCX in a guard column (PolyLC 5 &micro;m 300 &Aring;
in Javelin guard colum 1.0 mm &times; 10 mm) with Harvard Apparatus syringe
umpl with sub-&micro;liter per minute flow.  Fractions then redissolved in
200 &micro;l (10 mM potassium phosphtate pH 3.0 + 20% MeCN) and loaded 50 &micro;l/min.
Peptides eluted  stepwise increases in KCl at 50 &micro;l/min (steps are 20,
25, 50, 200 mM KCl).  Each fraction vacuum centrifuged, reconstituted in 30 &micro;l
HPLC loading solvent.  The LC-MS/MS is Microchrom Bioresources Paradigm MS4
coupled to ThermoFisher Sci LTQ linear IT-MS.  MS/MS spectra are queried
using SEQUEST against nonredundant human database (EBI's ipi.HUMAN v. 3.18fasta
with 62,000 records).  A reversed-sequence version was appended to end of foward
to look for false positive rate (see Peng et al J Proteome Res, 2: 43, 2003
for reasons).  Searches also assumed oxidized Met, and precusor peptide tolerance 
was &plusmn; 2.0 Da (no tryptic specificity!); fragment ion tolerance was &plusmn;
1.0 Da.  For all matched peptide sequences the Shimura algorithm 
<!-- ================================================================ -->
<p class="tags">

<p class="ref">
 <span class="authors">
Haddad I, Hiller K, Frimmersdorf E, Benkert B, Schomburg D, Jahn D
 </span>
 <span class="year">
(2009)
 </span>
 <span class="jname">
In Silico Biology
 </span>
 <span class="jvol">
9,
 </span>
 <span class="pages">
14-27.
 </span>
<p class="reftitl">
An emergent self-organizing map based analysis pipeline for comparative 
metabolome studies
<p class="Para1">
800 microbial and 100 eukaryotic genomes finished.  Metabolites levels being used
to assess.  Metabolites are evaluated under different conditions (treatments,
environments, genetic modification effects).  For each condition are a set of
values set up as a <i>mathematical vector</i>, called a <b>metabolic pattern</b>.
Each metabolite is a coordinate in high-dimensional space, and the vector is
the position of specific point in the coordinate system.  A dataset is arbitrary
number of metabolic patters expressed as scatterplot in high-dimensional metabolome
space.  The dataset is <i>multivariate</i>, as within each dimension there is
a high variability of value.
<p>
The variation in data comes from the following:
<ul>
<li>induced biological variation:  the variation of interest as it results from
the biological effect
<li>uninduced biological variation:  changes in metabolic concentrations under
identical experimental conditions
<li>statistical noise:  systematic and random errors in the experimental setup
</ul>
<p>
The strength or intensity of variation is shown in the order listed, the 
greatest being induced biological variation.  To analyze this data are principal 
components analysis (PCA), hierarchical cluster analysis (HCA) and unsupervised 
artificial neural network-based methods such as self-organizing maps (SOM).
<p>
<b>PCA</b> transforms input data to new coordinate system linearly.  The first
principal component is the dimension with the greatest variance, the second
component is dimension with 2nd greatest variance, and so on.  Only the first
cover most of the variation to describe data differences, while the remaining
components usually describe the noise.  PCA is used to reduce dimensionality of
multidimensional data with minimal info loss.  It also detects which metabolites
contrbite to the variance of the dataset.
<p>
<b>HCA</b>.  This method looks at the degree of similarity between data points:
their spatial distance in high dimensional space.  These are then successively
combined into clusters, and the clusters grouped into larger groups.  A dendrogram
is used to visualize the results.
<p>
<b>SOM</b>. Self-organizig maps or Kohonen Feature Maps are a class of artificial
neural networks that project high-dimensional data onto two-dimensional maps
which preserve topology&mdash;that is the input patterns which are geometrically
close to one another in high-dimensional space, and are mapped to close neurons
on the resulting feature map.  Map is arrangement of nodes on a grid, and each
node is metabolic pattern with arbritrary records for each metabolite.  In 
unsupervised training procedure, each patter of input data is connected to 
the specific node which shares highest similary with it.  Afterwards, the node
adjusts itself to the assigned pattern, and then adjusts patterns of its 
neighboring nodes gradually to its own; the influence attenuates with the 
distance to its neighbors.  The process of self-organization occurs over several
training cycles, resulting in a map, that can locate any metabolic pattern of
the input data with high reliability in the corresponding area on the map.
Implementations of SOMs providing a low number of nodes on the output map which
are adjusted to the number of expected clusters which are too few for the patterns
of input data and outliers cannot be detected using the strategy.  SOMs with
low number of nodes produce results similar to <i>k</i>-means clustering.
Real emergence in SOMs only expected to occur when number of nodes is a magnitude
larger than the number of input patterns:  these are <b>emergent self-organizing
maps</b> (ESOMs).  After training, most output nodes represent either a few or
none of the input data points, and preserve topology.  The U-matrix method allows
detection of a data cluster by exmaining overall structure of trained ESOM and
transforms this to dendrogram.
<p>
Cluster data from ESOM analysis or HCA should be evaluated by silhouette width
representation to identify incorrectly assigned data points in the structure.
This compares intra-cluster distances to inter-cluster distances.  Quality levels
are determined for each assigned data point and for all derived clusters.
Differences in clusters become detectable with hypothesis testing (Student's 
t, or 1-way ANOVA).  From this metabolites can be found as biomarkers that 
discriminate the condition or treatment.  These get reported in annotated databases
like KEGG, EcoCyc, MetaCyc, BRENDA.
<!-- ================================================================ -->
<p class="tags">

<p class="ref">
 <span class="authors">
Zhao L, Zhang Y, Wei J, Cao D, Liu K,  Qian X
 </span>
 <span class="year">
(2009)
 </span>
 <span class="jname">
Proteomics
 </span>
 <span class="jvol">
9,
 </span>
 <span class="pages">
4416-4420.
 </span>
<p class="reftitl">
A rapid isolation and identification method for blocked N-terminal peptides by 
isothiocyanate-coupled magnetic nanoparticles and MS
<p class="Para1">
<b>T.Brief</b>.   Separation of blocked and free N-terminal peptides/proteins 
has been done using many chromatographic methods.  SCX chromatography will 
prefractionate blocked N-termini.  Combinational fractional diagonal 
chromatography was used to separate free and acetylated when internal peptides 
are made hydrophobic with 2,4,6-trinitrobenzenesulfonic acid;  acetylated 
N-termini are common in archaea.  Isothiocyanate-coupled glass beads 
(glass&mdash;N=C=S) and isocyanate-coupled resins were developed to capture 
N-blocked peptides from protein digestions, then directly analyzed by MALDI MS. 
<p>
In this technique magnetic nanoparticles (MNPs) are modified with isothiocyanate 
(DITC) and tested with synthetic peptide mixtures.  This extends work with 
surface modifications on MNPs.  Ferroso-ferric oxide forms the core and is 
coated with DITC.  TEM (under JEOL 2000FX) shows how the nanoparticles were 
prepared with avg. diam of ~15 nm.  IR from 4000 to 400 cm<sup>&minus;1</sup> 
shows a strong peak at 1091 cm<sup>&minus;1</sup> which is 
&nu;<sub>-Si-O-Si-</sub> absorption, and signatures between 1650 and 1374 
cm<sup>&minus;1</sup> correspond to benzene ring stretching vibrations.  Band at 
801 cm<sup>&minus;1</sup> corresponds to 1,4-aryl group substituents.  Weak 
absorptions between 2130 and 2040 cm<sup>&minus;1</sup> are 
&nu;<sub>-N=C=S</sub> representing the surface groups.  The broad band peaking 
at 3430 cm<sup>&minus;1</sup> corresponds to the <b>&#333;</b><sub>-O-H</sub> 
from water present in an incompletely dried sample.
<p>
Peptides bind to DITC or ITC-coupled MNPs  and then selectively removed by 
magnetic separation.  N-blocked peptides are unreactive, and stay in solution. 
To test this, 40 pmol each of kemptide (LRRASLG) and its acetylated form were 
mixed and the mix incubated with DITC-coupled or intermediate stage 
(3-aminopropyl-coupled) MNPs.  Eluates were then analyzed by MALDI-ToF MS. The 
peak signal for kemptide disappeared from DITC-coupled MNP incubation, while 
acetylated kemptide was seen.  Both kemptide and acetylated form were present 
with 3-AP MNPs, indicating that ITC-MNPs specifically react with free amino 
peptides.  [Note:  The spectrum additionally showed [M+Na]<sup>+</sup> forms to 
both peptides also.]  Binding capacity was 2.67 nmol peptide per mg MNP (shown 
using RPLC with isocratic 50% MeCN+0.1% TFA solvent chromatography). The ITC 
couple was done to alternative particles (1 &micro;m commercial microbeads), and 
the MNPs had a 1.58-fold greater capacity.  It is likely that the smaller MNP 
size provides a larger surface area on a per gram basis than for microbeads.
<p>
10 pmol kemptide was labeled with iTRAQ 114 and 117 according to maker's protocol
and then analyzed on ABi MALDI ToF/ToF MS/MS, making them N-blocked.  After
incubating with MNPs, the recovery is checked.  The 117/114 ratio was found to 
be 88%:  this incomplete recovery may be due to nonspecifc adsorption of blocked
peptide to MNPs, and possibly to peptides on the surface of plastic container. 
Because Lys has an primary amino group, guanidination to form homoarginine
may also be necessary.  Failure to modify Lys is clear:  myoglobin was untreated
or guanidinated and the protein digested with trypsin and analyzed on MALDI-TOF,
and results show many missing signal peaks in the untreated compared to the
guanidinated.
<p>
Cytochrome <i>c</i> is known to be naturally <i>N</i>-acetylated.  After incubating
with ITC-coupled MNPs, trypsin-treated unbound material shows a peptide at
<i>m/z</i> 631 corresponding to Ac-GDVEK, the N-terminal cyt c peptide (K is
guanidinylated).  Mixed with a complex sample (acetyl-fibrinopeptide B
[Ac-EGVNDNEEGFFSAR], priopionyl-fibrinopeptide B [Prop-EGVNDNEEGFFSAR], 
alcohol dehydroganse digest, and BSA digest; 10 pmol of all peptides except
BSA, which is 15 pmol) which is guanidinated before treating to MNPs
<p>
<b>Preparation of MNPs</b>.  Fe(II) as FeCl<sub>2</sub> &bull; 4 H<sub>2</sub>O 
is mixed with Fe(III) as FeCl<sub>3</sub> &bull; 6 H<sub>2</sub>O in ammonium 
hydroxide (NH<sub>4</sub>OH) under nitrogen (N<sub>2</sub>) which leads to 
particle formation.  At 40&deg;, tetraethyl orthosilicate (TEOS, or 
tetraethyoxysilane) is added which produces silanols on the surface of the MNP. 
In the next step triemthoxypropylaminosilane 
[(CH<sub>3</sub>O)<sub>3</sub>&mdash;Si&mdash;(CH<sub>2</sub>)<sub>3</sub>NH<sub 
>2</sub>] is added at 60&deg; to produce on the MNP surface a 
triestersilylpropylamine. When mixed with 1,4-diisothiocyanylbenzene in 
dimethylformamide (DMF), this creates an MNP with a phenylisothiocyanate (PITC) 
attached to by the 4-phenyl substituent to the MNP core.  This is a particle 
form of the Edman reagent.
<!-- ================================================================ -->
<p class="tags">
cluster ion, FD
<p class="ref">
 <span class="authors">
Feketeova L, O'Hair RAJ
 </span>
 <span class="year">
(2009)
 </span>
 <span class="jname">
Rapid Comm. Mass Spectrom.
 </span>
 <span class="jvol">
23,
 </span>
 <span class="pages">
60-64.
 </span>
<p class="reftitl">
Comparison of collision- versus electron-induced dissociation of sodium chloride 
cluster cations
<p class="Para1">
A 1 mM NaCl in MeOH was used on a Finnigan LTQ-FT with ESI.  Flow was 
5 &micro;l/min with 3500-4500 V spray, 125&deg; capillary temperature,
10-30 arbitrary units nitrogen sheath pressure.  The system is a linear ion
trap (LTQ), ion-transfer optics and FT-ICR mass analyzer.  FT-ICR was used to
determine charge state. CID is done in LTQ with helium bath gas at 
5 mtorr.  A window of 1.5-8 Th was used in mass selection.  Normalized collision
energy is between 16-40%; activation (Q) 0.25-0.35, which assigns RF frequency
for ion fragmentation; 30 ms activation time for exciting via CID.  In 
electron-induced dissociation (EID), ions are transferred through a 2.25
&times; 10<sup>&minus;7</sup> torr into FT-ICR cell at pressure &lt; 1.5 &times; 10<sup>&minus;9</sup>
torr.  FT-ICR is supplied with low energy electrons from indirect heated
emitter cathode downstream of FT-ICR cell.  Ions are bombarded with electrons
at energies from 5 - 30 eV for times of 20-50 ms.
<p>
Previous studies with NaCl solutions show cluster ions of type 
(NaCl)<sub>m</sub>Na<sub>n</sub><sup>n+</sup> for n = 1-6.  When n = 1, then m = 
4, 13, and 22, so-called magic number clusters, where theoretical calculations 
predict high energies to remove one NaCl from these clusters, and indicating 
their stability.  For n = 2 doubly charged cluster, the values m = 11, 12, 17, 
20, 21, 26, 30, 24, 36, 44, 54, and 61 are magic numbers. Theoretical 
calculations predict that these arise by adding tri- or penta-atomic chains to 
two edges of a perfect neutral NaCl cube, or by removing chloride anion from a 
perfect singly charged [(NaCl)<sub>m</sub>Na]<sup>+</sup> cuboid.
Spectra of 1 mM NaCl in MeOH were collected and they followed a pattern of
(NaCl)<sub>m</sub>Na<sub>n</sub><sup>n+</sup>.  Relative intensities were a
function of the ESI source and the solution, but magic number intensities
were clearly the greatest (for n = 1, they are m = 4, 13, 22).  A series of
ions also present for n = 2 showed ions with m = 11, 12, and m &ge; 19.
High resolution FT-ICR MS revealed n = 3 clusters where m &ge; 49.
<p>
Fragmentation by CID and EID was done on selected n = 1 clusters:  m = 4 and 13
for magic number and n = 7 and 8 for nonmagic number.  When neutral species are
lost and they amount to more than one NaCl molecule, it is not possible to know
if they are lost as single molecules or as clusters (see the equations):
<p class="eqnserif">
[(NaCl)<sub>m</sub>Na]<sup>+</sup> &rarr; [(NaCl)<sub>m&minus;n</sub>Na]<sup>+</sup>
+ (NaCl)<sub>n</sub>  <br>
[(NaCl)<sub>m</sub>Na]<sup>+</sup> &rarr; [(NaCl)<sub>m&minus;n</sub>Na]<sup>+</sup>
+ n NaCl
<p>
Fragmentation spectra do not show differences between CID or EID in ion types,
only the relative intensities.  The dominant product from a m = 4 +1 ion
is m = 3, whereas m = 1 was the dominant product from EID.  For the m = 7
singly charged ion, the EID product is m = 4, while the CID product is m = 5.
With increasing cluster size of precursor, differences in product size are more
pronounced.  With precursor m = 8 and 13, there are fewer product ions in CID
than with EID:  EID produces all possible m &minus; n ions.  It is also clear
that the m = 13 / n = 1 signal is contaminated by the isobaric m = 26 / n = 2
signal.  In general CID produces a lower energy with vibrational excitation
while EID produces electronic excited states.
<p>
The n = 2 cations [(NaCl)<sub>m</sub>Na<sub>2</sub>]<sup>2+</sup> were examined
under CID and ion-electron reactions.  Electron capture dissociation (ECD)
has lower energies than EID.  Clusters with m = 11, 19, 20, 21, and 25 
precursors were studied.  For m = 21 with CID the fragmentation is mainly 
charge separation (shown in equation 2 below), while NaCl evaporation 
(shown in equation 1 below) is a minor channel.
<p style="margin:1em 0 0 1em;">
<b>NaCl evaporation</b>:
<p class="eqnserif" style="margin-top:0;">
[(NaCl)<sub>m</sub>Na<sub>2</sub>]<sup>2+</sup> &rarr; 
   [(NaCl)<sub>m&minus;n</sub>Na<sub>2</sub>]<sup>2+</sup>
   + (NaCl)<sub>n</sub>&nbsp;&nbsp;&nbsp;[1]
<p style="margin:1em 0 0 1em;">
<b>charge separation</b>:
<p class="eqnserif" style="margin-top:0;">
[(NaCl)<sub>m</sub>Na<sub>2</sub>]<sup>2+</sup> &rarr; 
   [(NaCl)<sub>m&minus;n</sub>Na]<sup>+</sup>
   + [(NaCl)<sub>n</sub>Na]<sup>+</sup>&nbsp;&nbsp;&nbsp;[2]
<p style="margin:1em 0 0 1em;">
<b>neutral radical loss</b>:
<p class="eqnserif" style="margin-top:0;">
[(NaCl)<sub>m</sub>Na<sub>2</sub>]<sup>2+</sup> + e<sup>&minus;</sup> &rarr; 
   [(NaCl)<sub>m&minus;n</sub>Na]<sup>+</sup>
   + [(NaCl)<sub>n</sub>Na]<sup>&bull;</sup>&nbsp;&nbsp;&nbsp;[3]
<p>
The main complementary pairs 
[(NaCl)Na]<sup>+</sup>/[(NaCl)<sub>20</sub>Na]<sup>+</sup>,
[(NaCl)<sub>4</sub>Na]<sup>+</sup>/[(NaCl)<sub>17</sub>Na]<sup>+</sup>,
and [(NaCl)<sub>8</sub>Na]<sup>+</sup>/[(NaCl)<sub>13</sub>Na]<sup>+</sup>.
With ECD there is losses of radicals Na and Na<sub>3</sub>Cl<sub>2</sub>
(see equation 3 above and m = 21, n = 0 and n = 2).  These fragments are
stable in the gas phase.  CID fragmentation of 
[(NaCl)<sub>11</sub>Na<sub>2</sub>]<sup>2+</sup> shows several charge
separation products with a base peak of [(NaCl)<sub>10</sub>Na]<sup>+</sup>.
ECD leads to neutral radical loss with n = 1-4 (equation 3 above).
Higher energy electron fragmentation causes all possible +1 fragments because
of charge separation (eqn 2).  The cluster m = 10 / n = 2 is also observed
resulting from a single NaCl loss, and is the smallest doubly charged cluster
observed.
<p>
For the m = 19 / n = 2 precursor, the m = 13 / n = 1 product dominates all
dissociation spectra (CID, ECD, EID).  This is a magic number cluster.
The channel for formation might be initial loss of Na<sub>3</sub>Cl<sub>2</sub>
followed by (NaCl)<sub>3</sub>, or loss via an intact Na<sub>7</sub>Cl<sub>6</sub>
via eqn 3.  The product [(NaCl)<sub>18</sub>Na<sub>2</sub>]<sup>2+</sup>
is also observed, being an NaCl loss (eqn 1).
<p>
CID of m = 20 / n = 2 gives a m = 8 / n = 1 base peak product.  With ECD
losses are via radicals (n = 0, 1, 2, 4), with products including (NaCl)Na<sup>+</sup>,
requiring a loss of (NaCl)<sub>19</sub> (unknown since the nature of losses
by neutrals are undetectable).  With EID the magic number m = 13 / n = 1
dominates.  Also slightly smaller 2+ products are formed via eqn 1 where m = 20
and n = 1, 2.
<p>
With m = 25 / n = 2 CID products are m = 22 / n = 1, a magic number, formed by 
the charge separation process (eqn 2).  n = 1 product ions with m &ge; 15 may 
come from NaCl unit losses of the magic cluster m = 22 product. The m = 22 / n = 
1 product ion dominates ECD and EID spectra as well.  ECD likely does it through 
radical ion losses (eqn 3, m = 25, n = 0 and 3) Na and 
Na<sub>4</sub>Cl<sub>3</sub>.  With EID fragments occur via charge separation
and NaCl evaporation pathways (n = 1, 2).

<!-- ================================================================ -->
<p class="tags">

<p class="ref">
 <span class="authors">
Roy A, Kucukural A, Zhang Y
 </span>
 <span class="year">
(2010)
 </span>
 <span class="jname">
Nat. Protcol.
 </span>
 <span class="jvol">
5,
 </span>
 <span class="pages">
725-738.
 </span>
<p class="reftitl">
I-TASSER: a unified platform for automated protein structure and function 
prediction
<p class="Para1">
Comparative modeling is one approach in protein structure prediction, where 
evolutionarily related homologous templtes are identified by sequence or 
sequence profile comparisons.  High resolution models are are generated by 
copying framework of the template structures.  Proteins from different 
evolutionary origins may have similar structure, and the threading methods are 
designed to make the query sequence directly on to the 3D structures of other 
solved protein with the goal of recognizing folds similar to the query even when 
there is no evolutionary relationship to the query and template protein.  In 
another approach <i>ab initio</i> modeling is done but with limited success on 
only small proteins, &lt; 120 amino acids.  Ab initio methods modeling uses 
evolutionary or knowledge-based information for getting spatial restraints and 
identifying local structural building blocks. I-TASSER (iterative threading 
assembly refinement) is an example of a composite approach and ranked best for 
automated protein structure prediction in the last two critical assessment of 
protein structure prediction (CASP) experiments.  Usefulness of a prediction
algorithm is measured on accuracy:  high resolution models with root mean
square deviation (RMSD) values in range of 1-2 &Aring; generated by CM
using close homologous templates, meet the highest structural requirements
and suitable for computational ligand-binding studies and virtual compound
screening.  Medium resolution models have 2-5 &Aring; RMSD, generated by
threading and CM using distantly homologous templates, and used for finding
spatial locations of functionally important residues (active sites, 
disease-related mutations).  Loop regions have functional importance but
high structual variability, although scaffold is conserved.  The lowest
resolution models, usually ab initio or based on weak hits from threading,
have uses include protein domain boundary identification, topology recognition,
and family/superfamily assignment.
<p>
Problems that arise are cases where absence of structural features or homology
nonetheless produce similar or identical function.  Many protein folds
and families are <q>functionally promiscuous,</q> with different folds performing
the same function.  Because of the success in online prediction, the steps
for submission of structure to I-TASSER server are offered here.
The method is done in four stages:
<ol>
<li>Stage 1: Threading.  Threading is a bioinformatics procedure for 
identifying template proteins from solved structure databases that have
a similar structure or similar structual motif as the query protein sequence.
The query sequence is matched against a non-redundant sequence database by
position-specific iterated BLAST (PSI-BLAST) to identify evolutionary relatives.
Sequence homologs are assembled in a multiple alignment and a sequence profile
created, and PSIPRED is along brought in for secondary structure prediction.
With both sequence profile and PSIPRED results, LOMETS is then used to thread
the query through the representative PDB structure library, a locally installed
meta-threading server combining seven threading programs: FUGUE, HHSEARCH,
MUSTER, PROSPECT, PPA, SP3, and SPARKS.  Within the threading programs, 
templates are ranked by sequence- and structure-based scores.  Top template
hits from each program are selected for further consideration.  Statistical
significance is used to judge the quality of template alignment (the Z score),
defined as energy score in std dev units relative to statistical mean of 
all alignments.
<li>Stage 2: Structural Assembly.  Continuous fragments in threading alignments
are excised from template structures, then used to assemble structural
conformations of sections that have aligned well with aligned regions, typically
loops and tails, which are built by ab initio modeling.  I-TASSER uses a
reduced model to represent the protein chain and improve conformational search
efficiency.  Each residue described by the &alpha;-C atom and side chain
center of mass.  Regions not aligned during threading process have lower
modeling accuracy results, and so structure modeling in these regiions 
confined to a lattice system of grid size 0.87 &Aring;, and this reduces
entropy for conformational search.  Such a grid size introduces considerable
uncertainty of conformational representations in CM (error range usu 1-2 &Aring;),
but there are no effects on ab initio modeling, whose error range is 4-6 &Aring;.
Thread-aligned regions have higher accuracy.  Modeling is off the lattice and
the template fragments are kept rigid during simulations to maintain high
resolution in these regions.  Fragment assembly is performed using modified
replica-exchange Monte Carlo simulation technique (Zhang et al Proteins 48:
192, 2002).  This implements several replica simulations in parallel at 
different temperatures, temps periodically exchanged between replicas, energy
barriers flattened by hyperbolic function to speed jumps of simulations between
different energy basins.  Overall simulation guided by composite knowledge-based
force field, including (1) general statistical terms derived from PDB
(C&alpha;/side-chain correlations, H-bonds, hydrophobicity); (2) spatial 
restraints from threading templates; (3) sequence-based contact predictions
from SVMSEQ (Zhang et al PNAS 103: 2604, 2006).  Conformations generatd in
low temp replicas during refinement simulation are clustered by SPICKER 
(Zhang &amp; Skolnick, J Comput Chem 25: 865, 2004) to identify low free
energy states.  Cluster centroids obtained by average 3D coordinates of all
clustered structural decoys.  Readers should read Wu et al (BMC Biol 5: 17, 2007)
and Zhang &amp; Skolnick (PNAS 101: 7594, 2004) for the on-and-off lattice
system, Zhang et al (Biophys J 85: 1145, 2003), Zhang (Proteins 69: 108, 2007), 
and Zhang (Proteins 77: 100, 2009) for force field development, and Zhang et al
(Proteins 48: 192, 2002) for the MC search engine.
<li>Stage 3: Model Selection and Refinement:  Fragment assembly simulation
is performed again starting from selected cluster centroids.  The inherent
I-TASSER potential remains unchanged in the 2nd run, and external constraints
are pooled from LOMETS threading alignments and PDB structures which are
structurally close to the cluster centroids, identified by TM-align.
The 2nd iteration removes steric clashes and refines the global topology of 
cluster centroids.  Decoys generated in 2nd round of simulations are clustered
again, and lowest energy structure selected as input for REMO, which generated
final structural models by building all-atom models from C&alpha; traces 
through optimization of H-bonding networks.
<li>Stage 4:  Structure-based functional annotation.  The function of the
query protein is inferred by structually matching predicted 3D models against
proteins of known structure and function in the PDB.  Three structure/function
libraries were constructed independently and updated every 2 weeks.  A library
of 5798 nonredundant entries with known EC numbers, 26045 nonredundant entries
with known Gene Ontology terms, and 19658 nonredundant entry of known
ligand-binding sites.  Global structural similariyt search is used to recognize
proteins with similar global fold, and local similarity search is 
complementary, identifying analogs with different fold but which have similar
function as there is conservation of active and binding sites.  A template
modeling score (TM-score), RMSD, sequence identity and coverage of structural
alignment composite scoring is used.  TM-score assesses topological similarity
of protein structure pairs, ranging from [0,1], with higher score indicating
better structural match.  A TM-score &lt; 0.17 statistically means a randomly
selected pair with gapless alignment taken from the PDB, whereas a score &gt; 0.5
means protein pairs of similar folds.
</ol>
<p>
The quality of the prediction will be used by biologists to determine how
they use the prediciton.  The <i>C</i>-score (confidence) is the quality
of the threading alignments and convergence of I-TASSER's structural 
assembly refinement simulations: 
<p class="eqn-serif"> 
<i>C</i>-score = ln (M/M<sub>tot</sub> 
&times; 1/&lang;RMSD&rang; &times; 1/7 
&sum;<sub><i>i</i> = 1 &rarr; 7</sub> 
<i>Z</i>(<i>i</i>)/<i>Z</i><sub>0</sub>(<i>i</i>) 
<p>
where M is number of structure decoys in cluster and M<sub>tot</sub> is number 
of decoys generated during I-TASSER simulations, &lang;RMSD&rang; is average 
RMSD of decoys to cluster centroid, <i>Z</i>(<i>i</i>) is <i>Z</i> score of best 
template generated by the <i>i</i>th threading in the seven LOMETS programs and 
<i>Z</i><sub>0</sub>(<i>i</i>) is a program-specified Z-score cut-off for 
distinguishin between good and bad templates.  A Pearson correlation shows that 
C-score and TM-score (absolute difference beteween model to native structure) is 
0.91.  If a C-score cutoff of &minus;1.5 is specified to select models of 
correct topology, both false-positive and -negative are below 0.1 (&gt;90% of 
predictions are correct).  With C-score and protein length combined, I-TASSER 
accuracy can be predicted with average error of 0.08 for TM-score and 2 &Aring; 
for RMSD.
<p>
I-TASSER forced field designed for modeling single domain proteins.  Procedure 
for multidomain proteins different but fully automated. (1) Domain boundaries 
are defined based on LOMETS threading programs: if a segment of query sequence 
of &gt; 80 residues has no alignmnent with template proteins in top two 
threading hits, target is treated as multiple domain protein and domain boundary 
defined at borders of aligned/unaligned sections. (2) Two types of assembly 
simulations are done:  (i) one simulation done for modeling whole-chain 
structure to provide guide for domain orientations (ii) another is carried tou 
for modeling single-domain structures individaully. For full-length modle, 
models of individual domains are docked using whole-chain I-TASSER model as 
template.  Docking simulation done using quick Metropolis Monte Carlo simulation 
where energy is defined as RMSD of individual domain models to whole chain 
ITASSER + reciprocal of number of interdomain steric clashes.
<p>
Where domain boundary information is available to user, the user should split 
the sequence into individual domains and submit each individually to server. 
This speeds up I-TASSER prediction and results in more reliable structure and 
function predictions.  Domain boundaries can be predicted using the <a 
href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cdd.shtml">NCBI CDD</a>.
<p>
Spatial informaiton including residue-residue contacts and distances can be
used as restraints to guide I-TASSER structure assembly simulations.  
New version of I-TASSER allows user to specify additional restraints
based on experimental evidence or biological insights.  These restraints
have higher accuracy than threading alignment derivations.  Using as few
as <i>N</i>/8 NOE restraints from NMR experiments (<i>N</i> = length of protein),
the simulation procedure can successfully fold 75% of proteins up to 200
residues, which cannot be fold without using spatial restraints because of
lack of appropriate templates.
<p>
Two methods can be used for inputting restraints
<ol>
<li>Restraint data from NMR or crosslinking experiments by uploading a
restraint file.  In column 1, specify restraint type as DIST or CONTACT.
For DIST restraints, column 2 and 4 are residue position (<i>i,j</i>) and
column 3 and 5 are atom names in residue.  Column 6 is distance in angstroms.
The software attempts to produce a simulation with the proper distance.
For CONTACT types, only the residue positions are specified in columns 2
and 3:  side-chain centers of mass of the two residues are brought into contact
during the simulation.
<li>PDB structures can be designated as a template instead of LOMETS threading
programs providing the template.  The solved PDB structure might not be
avialable to LOMETS or identified by LOMETS despite being avialable.  The
input is either a PDB-formatted structure file or a PDB ID.
</ol>
<p>
I-TASSER uses a template and needs to know 3D structure and alignment between 
query and template sequence.  If template structure is provided without 
alignment, I-TASSER generates the alignment with MUSTER, which incorporates many 
sources of information including secondary structure, sequence profile, solvent 
accessibility, and structure fragment profiles.  Alternatively users can provide 
aligments in two formats:  <a 
href="http://www.ncbi.nlm.nih.gov/blast/fasta.shtml">FASTA</a> and <a 
href="http://www.wwpdb.org/documentation/format32/sect9.html">3D</a>.
<p>
The setup requires:
<ol>
<li>Personal computer with Internet connection and web browser.
<li>Amino acid sequence of protein of interest at a minimum in FASTA format.
<li>A molecular visualization application such as RASMOL or PYMOL for
viewing 3D structur of modeled protein and predicted funcitonal sites.
</ol>
<p>
The procedure is as follows:
<ol>
<li>Find <a href="http://zhanglab.ccmb.med.umich.edu/I-TASSER">I-TASSER 
server</a> 
<li>The amino acid sequence of a single protein china is input to the 
form or it can be directly uploaded to the server (probably in a text file).
Sequences between 10-1500 are accepted.
<li>Include in the form an email address, the name of the protein
(optional).  Additional restraints (contact/distance) or specify solved
protein structure as template, prepare restraint file and upload, or provide
PDB ID or chain ID.
<li>There is an option to <a 
href="http://zhanglab.ccmb.med.umich.edu/I-TASSER/queue.php">track the modeling 
status</a>
</ol>
<!-- ================================================================ -->
<p class="tags">

<p class="ref">
 <span class="authors">
Pandit SB, Brylinski M, Zhou H, Gao M, Arakaki AK, Skolnick J
 </span>
 <span class="year">
(2010)
 </span>
 <span class="jname">
Bioinformatics
 </span>
 <span class="jvol">
26,
 </span>
 <span class="pages">
687-688.
 </span>
<p class="reftitl">
PSiFR: an integrated resource for prediction of protein structure and function
<p class="Para1">
TASSER was developed as an algorithm for protein tertiary structure prediction.
It uses a hiearchical approach in which there is template identification by
threading then tertiary structure assmebly from continuous template fragments.
TASSER refinces the threading templates to generate final models close to native
tertiary structure than the input templates.  METATASSER is an improvement that
incorporates information from consensus threading templates.  For rapid results
TASSER-Lite is a version that works when target and template identity is
&ge; 25%.  For function annotation, FINDSITE does ligand-binding prediction,
DBD-Hunter does DNA-binidng prediction, and EFICAz (Enzyme Function Inference
by A Combined Approach) is for enzyme function inference.  FINDSITE predicts
ligand-binding pockets based on binding site similarity among superimposed groups
of template structures identified from threading.  Threading acts as a filter
to establish that the set of identified template structures are evolutionarly
related.  Chemical properties of ligans likely to occupy binding site are given
by FINDSITE.  FINDSITE assigns Geno Ontology terms with a probability that 
corresponds to fraction of threading templates annotated with that molecular
function.
<p>
The <a href="http://psifr.cssb.biology.gatech.edu/">Protein Structure and 
Function Prediction Resource</a> (PSiFR) gives integrated tool for tertiary 
structure prediction and structure- and sequence-based function annotation.
<!-- ================================================================ -->
<p class="tags">

<p class="ref">
 <span class="authors">
Wishart DS
 </span>
 <span class="year">
(2010)
 </span>
 <span class="jname">
Meth. Molec. Biol.
 </span>
 <span class="jvol">
593,
 </span>
 <span class="pages">
283-313.
 </span>
<p class="reftitl">
Computational Approaches to Metabolomics
<p class="Para1">
Metabolomics (a/k/a metabonomics or metabolic profiling) is high-throughput
identification and quantification of small molecule metabolites in the metabolome.
The metabolome can be definded as complete collection of all small molecule
metabolites &lt; 1500 Da, including amino acids, sugars, organic acids, bases
vitamins, lipids, found a a specific cell, organ, or organism.
<p>
A great deal of metabolomic data is not yet in databases.  Much of the data
is from analytical instruments (MS, NMR, and separation by chromatogaphy).
Analytical software used in metabolite identification is little different from
software developed for and used in the other 'omics' fields.  Aside from
identification and quantifying metabolites, metabolomics is also about relating
the metabolite data to genes, proteins, pathways, physiology, and phenotypes.
<p>
Metabolic profiling has been in practice for thousands of years: in 5th century
BC, Hippocrates described examination of urine by color, taste, and smell as
a means of understanding health.  In mid-19th century constituents of biofluids
were identified and quantified and related to medical conditions.  As many as
5-15 chemical analyses can be done on blood and urine using analyzers and even
paper strips.  Metabolomics aims to characterize every metabolite however, both
in quantity and identity to get a <q>signature</q> or profile.  The temporal
state is significant, since metabolomes change in seconds to minutes while
proteomes change in minutes to hours, and genomes are essentially static.
With development of high rez and accurate MS instruments, high throughput NMR
spectrometers, improvements in CE, the next generation of chromatography (UPLC)
<!-- ================================================================ -->
<p class="tags">

<p class="ref">
 <span class="authors">
McAllister SR, Floudas CA
 </span>
 <span class="year">
(2010)
 </span>
 <span class="jname">
Comput. Optim. Appl.
 </span>
 <span class="jvol">
45,
 </span>
 <span class="pages">
377-413
 </span>
<p class="reftitl">
An improved hybrid global optimization method for protein tertiary structure
prediction.
<p class="Para1">
Leventhal's paradox states how a protein can find the native functional state if
the time scale for folding in on the order of milli- or even microseconds.
Anfinsen's hypothesis is that the native structure is found by determining the
global minimum free energy conformation.  Generally comparative modeling techniques
are used in which a homologous protein is found and the target protein aligned
to the template.  These comparative modeling techniques use at some point
sequence/sequence alignment algorithms (BLAST), but improvements are made using
profile sequence comparisons, hidden Markov models, and profile-profile alignment.
<b>Fold recognition</b> and <b>threading</b> methods are proposed to detect remotely
homologous proteins.  Instead of sequences, it is believed that folds are more
conserved and the number of folds is orders of magnitude less than the number of
protein sequences.  Threading methods are a class of fold recognition algorithms
and attempt to fit or thread the target protein onto the template protein.
Dynamic programming techniques, iterative approaches, and optimization-based
approaches have been used in threading methods.
<p>
Majority of methods are minimum energy conformation = native state approaches.
These are in two classes: (1) homology-based info done in searches and statistical
potentials and/or biased conformation searches are done; (2) homology-independent
information:  rather physics-based potentials and conformational search with
optimization algorithms.
<p>
<b>Fragment assembly</b> is a popular program using short subsequences of known
PDB structures to build the overall structure.  Optimization on the basic set
is then done with scoring function or statistical potentials in combination with
optimization algorithms.  Simulated annealing approaches assemble compact structures
from protein fragments, which are then optimized by the minimization of the scoring
function, based on conformational statistics of known proteins with Monte Carlo
minimization techniques.
<p>
Many prediction methods are hybrids of methods:  combining comparative modeling
with minimization of scoring function.  One combines multiple sequence comparisons,
threading, a united atom lattice model, optimization of scoring function, and clustering
for prediction of protein structure.  These have been applied iteratively to 
ab initio modeling of small proteins with promising results.  Other 3&deg; structure
prediction algorithms focus on minimization of knowledge-based scoring function,
using lattice-based approach or Monte Carlo techniques.
<p>
Homology-independent methods using the thermodynamicd minimum free energy 
structure but are computationally demanding, with times on the order of CPU-years
for all but small proteins.  They provide insight into mechanism, thermodynamics
and kinetics of protein folding.  Monte Carlo and molecular dynamics techniques
have been used for many folding studies.  A zipping-and-assembly mode using
replica exchange molecular dynamics for folding of small, single-domain proteins
(Dill and coworkers).  A hierarchical approach using a Metropolis Monte Carlo
algorithm has been applied to protein structure prediction (LINUS) by identifying
conformational biases from a discrete set of moves, and evaluating a simplified
physics-based force field.  Pande and co-workers have folded vilin to an RMSD
of 3 &Aring; using molecular dynamics implemented through the popular distributed
grid computing application (Folding@Home).  Scheraga and coworkers have the
simplified united-residue force field (UN-RES) in which initial calcualtions
produce a coarse structure model and then refinement using all-atom potential.
Each amino acid residue is reduced to two interaction sites; the reduction allows
stochastic conformational space annealing algorithm to efficiently identify low
energy regions of conformational space.  Use of UN-RES with replica exchange
Monte Carlo and minimization search technques also investigated.
<p>
ASTRO-FOLD is a first principles folding approach in which (1) &alpha;-helical
prediction is made by detailed free energy calculations; (2) mixed-integer linear
optimization formulation for &beta;-sheet prediction; (3) derivation of secondary
structure restraints and loop modeling; and (4) application of hybrid global
optimization algorithm to 3&deg; structure prediction.  ASTRO-FOLD goal: minimize
potential energy of a protein subject to constraints on dihedral angles and
distances.  Algorithm is combination of deterministic &alpha;BB global optimization
algorithm
<!-- ================================================================ -->
<p class="tags">

<p class="ref">
 <span class="authors">
Haigh BJ, Stewart KW, Whelan JRK, Barnett MPG, Smolenski GA, Wheeler TT
 </span>
 <span class="year">
(2010)
 </span>
 <span class="jname">
J. Clin. Periodontol.
 </span>
 <span class="jvol">
37,
 </span>
 <span class="pages">
241-247.
 </span>
<p class="reftitl">
Alterations in the salivary proteome associated with periodontitis
<p class="Para1">
<!-- ================================================================ -->
<p class="tags">

<p class="ref">
 <span class="authors">
Smejkal GB, Bauer DJ
 </span>
 <span class="year">
(2010)
 </span>
 <span class="jname">
Am. Biotechnol. Lab.
 </span>
 <span class="jvol">
28,
 </span>
 <span class="pages">
24-27.
 </span>
<p class="reftitl">
2-D in a Day: A Two-Dimensional Gel Electrophoresis Work Flow Completed
in Under Five Hours
<p class="Para1">
300 mg lyophilized <i>E coli</i> K12 was suspended in 10 ml of 7 M urea,
2 M thiourea, 4% CHAPS which was filtered over AG501-X8 mixed bed resin, 
filtered and stored in aliquots (all Hoefer reagents), and added were 
200 &micro;l 50 &times; of Sigma-Aldrich broad range inhibitor (AEBSF E-64,
bestatin, leupeptin, aprotinin, EDTA) and 250 &micro;l of 0.25 M 
tributylphosphine.  Sonicated 40 times in 1 s cycles with Branson
Sonicator 450 at 70% power with microtip probe.  10 mM acrylamide was added
and left 2 h to alkylate.  DTT was added to 50 mM to quench the alkylation.
(Note that acrylamide/TBP combination can be done simultaneously and is more
effective than DTT/iodoacetamide use.)  Acetone is added to 86% and intermittent
vortexing done for about 2 h.  Precipitant proteins are pelleted at 10,000 <i>g</i>
for 10 min.  Proteins are then resuspended in fresh deionized 
urea/thiourea/CHAPS, and then DTT added to 50 mM.  The Hoefer Protein 
Determination reagent is used (found to be 10 mg/ml).  Samples are diluted
to 650 and 1300 &micro;g/ml with solubilzation reagent.  IPG BlueStrips 
(Serva, Heidelberg Germany)   7 cm long and in ranges pH 4-7 and pH 3-10 NL
are rehydrated with 140 &micro;l with each protein concentration, protein
totals being 90 and 180 &micro;g.  Trays are sealed with zipper storage bags
containing damp paper towel (mineral oil works too).  For running IEF, gel
side is up (plastic adhered with oil) and nearly blotted dry wicks applied to
both ends.  IEF was done with rapid gradient rise from 0-6000 V or 0-12,000 V
in first hour, then constant 12,000 V.  Current was limited to 25 &micro;A 
and power 500 mW per strip.  When IPG strips were to be stained, they were 
fixed in 35:10% MeOH:MeCOOH for 60 min then stained in 0.1% Coomassie Brilliant
Blue G-250 in 34% MeOH, 17% AmSulfate, 3% phosphoric acid (for every 100 ml
stain, dissolve 0.1 g CBB G-250 in 34 ml MeOH, stir in 3.6 ml 85% phosphoric acid
for 2 h, in separate beaker dissolve 17 g AmSulfate in 50 ml water, then add
this slowly to dye/MeOH while stirring, adjust to 100 ml, stir for 2 h, filter).
Destaining was done in 10:7% MeOH:MeCOOH.
<p>
<b>2nd Dimension</b>.  11% SDS-PAGE gels were cast:  18.3 ml 30% acrylamide:bis
(29.2:0.8%), 12.5 ml 1.5 M TrisHCl pH 8.8, 18.9 ml water, 200 &micro;l of
10% ammonium persulfate (final 0.04%), and 60 &micro;l TEMED (final 0.08%).
25% isopropanol was used to flatten the top edge during polymerization.
Polymerization catalysts were set to allow polymerization in 15-20 min. 
Strips are equilibrated twice in Tris pH 8.8 + 3 M urea, 3% SDS, 50 mM DTT
0.005% bromophenol blue and 0.01% cresol purple.  No agarose overlay is used.
The staining of Witje et al was used for the 2nd dimension gel.
<p>
<b>Results</b>.  The <i>E coli</i> sample at 10 mg/ml protein had a conductivity 
of 159 &micro;S/cm, and the diluted to 0.65 mg/ml sample was 45 &micro;S/cm.
It was determined that 6000 to 8000 Vh was sufficient to complete the IEF.
At the lower protein loads, it was apparent that IEF could be completed in an
hour.  Heavier protein loads only required a bit longer.  A major reason for
slow IEF is the high initial conductivity of samples, which require desalting.
<!-- ================================================================ -->
<p class="tags">

<p class="ref">
 <span class="authors">
Abate S, Ahn YG, Kind T, Cataldi TRI, Fiehn O
 </span>
 <span class="year">
(2010)
 </span>
 <span class="jname">
Rapid Comm Mass Spectrom
 </span>
 <span class="jvol">
24,
 </span>
 <span class="pages">
1172-1180.
 </span>
<p class="reftitl">
Determination of elemental compositions by gas chromatography/time-of-flight 
mass spectrometry using chemical and electron ionization
<p class="Para1">
GC/MS has high sensitivity and specificity for many analyte classes, frequently 
used to profile primary metabolites.  EI mode usually used with compound 
identification by matching spectra to spectral database libraries, e.g. NIST08, 
which are compatible with many mass spectrometer types; quadrupoles however can 
be tuned to transmit higher <i>m/z</i> range which might result in different 
abundances compared to TOF spectra.  The two major challenges for improving 
GC/MS in profiling metabolites are (1) rapid annotation and identfication of 
unknowns and (2) integration of biological knowledge into interpeting data. When 
quadrupole GC/MS mass measuremtns of derivatized metabolites in complex mixtures 
are recalibrated and combined with isotope ratio constraints, unknown low MW 
substances can be identified despite residual mass errors of 0.01 <i>m/z</i> 
units (verified by mag sector MS).  EI spectra with their fragment spectra plus 
the retention index provide a way for identification.  When spectra are not in 
libraries, the actual parent mass must be determined.  Soft ionization is 
required using chemical ionization (CI), field ionization (FI) which is 
ionization in high electric fields close to emitter needles, and photoionization 
(PI) in atmospheric UV light or in UV under vacuum.  Total ion counts are low 
with CI and FI, but relative intensity is higher with FI and CI Reduction of the 
EI ionization energy from 70 to 30 eV or less might reduce fragmentation.  Since 
average C-C bond energies are 3.6 eV (348 kJ/mol), fragmentation can still 
occur.  At lower ionization energies the total ion current is several orders of 
magnitude lower and useless to practical GC/MS, particularly for low abundance. 
Amirav et al developed a <i>cold-EI</i>: supersonic molecular beam ionization is 
not commercially available in GC/MS.  Better resolution can be achieved for 
complex samples using mag sector instruments.  Software-based approaches have 
been successful for automated MW estimation of EI mass specctra with probability 
calcualtion to determine molecular ion (see Scott in Anal. Chim. Acta 285: 209, 
1994 and Scott in Chemomet. and Intelligent Laboratory Systems 27: 129, 1995), 
implemented in an NIST MS Search program (Stein JASMS 10: 770, 1999). The 
algorithms rely on molecular ions in the low base peak percent thresholds (1-5% 
bpi).  High quality spectra demand ions, including noise, co-eluting with 
analytes should be removed such as by using AMDIS mass spectral deconvolution 
algorithms; without noise-free spectra molecular ions from chromatographically 
unresolved mixtures is not possible.  After identification of molecular ions, 
accurate mass and isotope data can be used to calculate elemental compositions. 
Calculations are more successful with highly accurate data, more likely with 
high abundance ions.
<p>
In every case it is the goal to increase signal intensities (ion counts) at all 
peaks in order to provide good statistics for higher confidence elemental 
composition determinations.  Trimethylsilylated (TMS)-metabolites are found in 
the mass range 300-800 u.  The method shown here preferentially tunes TOF 
analysis for higher mass range in this range and discriminates against ions 
intensities in lower <i>m/z</i> range.  The instrument is tuned with 
perfluortributylamine (PFTBA) setting the value at four different voltages on
outer source, beam steering, and focuses 1 to 3 for maximizing absolute intensity
of <i>m/z</i> 264 and 502.  Beam steering can be altered to preferentially 
produce more ions in the higher or lower mass ranges.  The PFTBA spectrum in the
high mass tune file shows an rel. abundance of high mass ions that is higher than
the NIST PFTBA mass spectral database.  Besides rel abundance, absolute intensities
were also increased in the higher mass range
<!-- ================================================================ -->
<p class="tags">

<p class="ref">
 <span class="authors">
Ogawa Y, Miura Y, Harazono A, Kanai-Azuma M, Akimoto Y, Kawakami H, Yamaguchi T, 
Toda T, Endo T, Tsubuki M, Yanoshita R
 </span>
 <span class="year">
(2011)
 </span>
 <span class="jname">
Biol. Pharm. Bull.
 </span>
 <span class="jvol">
34,
 </span>
 <span class="pages">
13-23.
 </span>
<p class="reftitl">
Proteomic Analysis of Two Types of Exosomes in Human Whole Saliva
<p class="Para1">
Human saliva is a biofluidic combination of major and minor salivary glands and
gingival crevicular fluid (GCF).  Proteins found in saliva include alpha-amylase,
mucins, histatins, cystatins, and Pro-rich proteins. Exosomes are small membrane
vesicles of endocytic origin (30-100 nm) that are released and are found to be
present in many biofluids such as bronchoaveolar lavage and urine.  They are also
found in saliva.  Exosome function is not fully understood but they might function
in distant cell-to-cell communication (<i>trans</i> signaling) or in antigen 
presentation.  They contain mRNA and microRNA (miRNA).
<!-- ================================================================ -->
<p class="tags">
gene annotation, proteomics, post-translational modifications, Salmonella
<p class="ref">
 <span class="authors">
Ansong C, Nikola Toli, Purvine SO, Porwollik S, Jones M, Yoon H, Payne SH, 
Martin JL, Burnet MC, Monroe ME, Venepally P, Smith RD, Peterson SN, Heffron F, 
McClelland M, Adkins JN
 </span>
 <span class="year">
(2011)
 </span>
 <span class="jname">
Biol. Pharm. Bull.
 </span>
 <span class="jvol">
34,
 </span>
 <span class="pages">
13-23.
 </span>
<p class="reftitl">
Experimental annotation of post-translational features and translated coding 
regions in the pathogen <i>Salmonella</i> Typhimurium
<p class="Para1">
This study uses bottom-up proteomics, using chromatography-purified peptides 
analyzed on MS/MS spectra witha  <i>de novo</i> sequencing methodology using 
MS/MS data obtained at high resolution and mass accuracy to look at translated
sequences and their PTMs.  This analysis looked at <i>Salmonella enterica</i>
serovar Typhimurium 14028 and show 47 genes that were assigned the incorrect
translational start site.  12 genes not annotated and missed by gene prediction
programs were identified;  there may also be a role for one ORF in pathogenesis.
<p>
The 14028s genome was sequenced in this study.  The main chromosome is 4.87 Mb
and there is a 94 kb plasmid with 99% sequence homology to the LT2 strain 
virulence plasmid pSLT.  Genome analysis also included feature identification
<!-- ================================================================ -->
<p class="tags">
MS/MS fragmentation, oxazolone pathway, b-y ions, RRKM, arginine
<p class="ref">
 <span class="authors">
Chung Y-J, Yoon S-H, Kim M-S
 </span>
 <span class="year">
(2011)
 </span>
 <span class="jname">
Bull. Korean Chem. Soc.
 </span>
 <span class="jvol">
29,
 </span>
 <span class="pages">
2427-2433.
 </span>
<p class="reftitl">
Kinetics and Mechanism for the Formation of <i>b</i>- and <i>y</i>-type Ions from
Singly Protonated Peptides on a Microsecond Time Scale: The Arginine Mystery
<p class="Para1">
When internal energy of a protonated peptide ionized by ESI or MALDI is not large
charge-directed reactions are responsible for dissociation into <i>b</i> and
<i>y</i> ions.  The mobile proton model provides for two possibilities:
(i) one in which the proton moves along the backbone and (ii) one where it is
sequestered or localized.  With an Arg residue, protons lose mobility.  When
instead protonation is on the N-terminal amino group, the proton migrates
to amide nitrogens where the charge-directed reactions occur.  Evidence for
the model come from fragmentation efficiency data:  with <i>z</i>=+1 peptides
with an Arg a higher internal energy is necessary than for <i>z</i>=+2
peptides (one mobile H<sup>+</sup>).  Product ions of <i>b</i> and <i>y</i>
type have focused on <i>z</i>=+1 peptides of Gly and Ala.  The mechanism of
fragmentation is thought to focus on oxazolone pathway
[R-CH(-O-1)-NH-CH(-R)-(1-)C-C(=O)-R&prime;].  Paizs and Suhai (<i>Rapid Commun.
Mass Spectrom.</i> <b>16</b>: 375, 2002; <i>Mass Spectrom. Rev.</i> 24: 508,
2005) and Rodriquez et al (<i>JACS</i> <b>123</b>: 3006, 2001) have used
density functional theory to investigate the oxazolone pathway.  The global
energy minimum for a <i>z</i>=+2 peptide places the second proton on the N-terminal
-NH<sub>2</sub> in the Arg-peptide, and the reactive intermediate is when the
proton moves to amide nitrogen.  There is an intramolecular rotation to form
the oxazolone ring with cleavage to the ring derivative and the peptide, and
proton affinity determines whether the C-terminal fragment forms the <i>y</i>
ion or the N-terminal fragment forms the <i>b</i> ion.  It is not known however
if Arg-containing peptides function via the oxazolone pathway.
<p>
Using <i>z</i>=+1 peptides with an N-terminal Arg using time-resolved
ultraviolet photodissocation, rough estimations fro statistical
Rice-Ramsperger-Kassel-Marcus (RRKM) rate constants were made for dissociation
of a singly protonated peptide.
<p>
The relation between rate and energy is calculated using the RRKM expression:
<p class="eqnserif">
<i>k</i>(<i>E</i>) = &sigma;[<i>N</i><sup>&Dagger;</sup> (<i>E</i> &minus;
<i>E</i><sub>0</sub>)] &divide; [<i>h</i>&rho;(<i>E</i>)]
<p>
&rho;(<i>E</i>) is the vibrational state density of the reactant at internal
energy <i>E</i>, <i>E</i><sub>0</sub> is the critical energy of the reaction,
<i>N</i><sup>&Dagger;</sup> (<i>E</i> &minus; <i>E</i><sub>0</sub>) is the
vibrational state sum from 0 to <i>E</i> &minus; <i>E</i><sub>0</sub> at the
transition state, <i>h</i> is Planck's constant, and &sigma; is the reaction
path degeneracy, usually &sigma; = 1 for reactions involving large molecules.
Our group has given details of method and software used to calculate
<i>k</i>(<i>E</i> for dissocation of +1 peptides (Moon et al, JASMS 17: 1749,
2006; Moon et al JASMS 18: 1063, 2007; Sun et al J Phys Chem B 111: 2747, 2007).
Vibrational frequency set for [M+H]<sup>+</sup> peptide need to evaluate
<i>&rho;</i>(<i>E</i>) is generated by software when given the sequence.
Transition state frequency need for <i>N</i><sup>&Dagger;</sup> (<i>E</i> &minus;
<i>E</i><sub>0</sub>) is given when entropy of activation
&Delta;<i>S</i><sup>&Dagger;</sup> at 1000 K is designated.  The software
deletes the frequency of the vibration along the reaction coordinate (taken
as 1149 cm<sup>&minus;1</sup>) and adjusts six low frequency vibrations such
that the designated &Delta;<i>S</i><sup>&Dagger;</sup> results.
&Delta;<i>S</i><sup>&Dagger;</sup> = 0 eu (1 eu = 4.184 J K<sup>&minus;1</sup>
mol<sup>&minus;1</sup>) is assumed for <i>b</i>- and <i>y</i>-type ions.
Beyer-Swinehart algorithm is used to calculate state sum and density with
grain size of 1 cm<sup>&minus;1</sup>.  The internal energy distribution
<i>P</i>(<i>E</i>) for a [M+H]<sup>+</sup> peptide is
<p class="eqnserif">
<i>P</i>(<i>E</i>) &prop; <i>&rho;</i>(<i>E</i>) <i>e</i><sup>&minus;<i>E/kT</i>
<p>
<b>Results and Discussion</b>.  The DFT studies indicate multiple steps in the
oxazolone pathway to <i>b</i>- and <i>y</i>-types products:  isomerization,
tautomerization, and bond-breaking.   The scheme is as below:
<p class="eqnserif">
A &hArr; B &harr; C &rarr; D + E
<p>
where A is the global energy minimum, B is the structure with the mobile proton
having protonated an amide nitrogen, C is the oxazolone + peptide as a
proton-bound dimer, and D and E are the separation products.  Each step
with forward and backward reaction (or just forward) will have transition
state:  TS<sub>AB</sub>, TS<sub>BC</sub>, TS<sub>CD</sub>.  Potential energies
were calculated relative to the global energy minimum at DFT level by Paizs
and Suhai for channels yielding <i>b</i><sub>2</sub>-<i>y</i><sub>3</sub>,
<i>b</i><sub>3</sub>-<i>y</i><sub>2</sub>, and <i>b</i><sub>4</sub>-<i>y</i><sub>1</sub>
of a [M+H]<sup>+</sup> for the homopentamer AAAAA.  The reference did not
report the TS<sub>AB</sub> potential energy but it should be lower than all other
transition state PE values.  TS<sub>CD</sub> is likely to be equal to D + E
because separation of the proton-bound dimer is thought to occur without
significant reverse barrier.  The rate constant from C &rarr; D + E
(<i>k</i><sub>3</sub>) will generally be much larger than the reverse reaction
<i>k</i><sub>&minus;2</sub> in which the proton dimer unforms the oxazolone
(C &rarr; B).  The kinetic scheme is simplified to A &hArr; B &rarr; C, which
is a well-known case of unimolecular dissociation after isomerization.
B &rarr; C has a likely higher energy barrier than B &arrr; A or to a structure
with the extra proton at the amide oxygen.  Also &Delta;<i>S</i><sup>&Dagger;</sup>
for former reaction occurs by a 5-membered ring transition state and will be
smaller than for latter reaction that involve proton migration.  So
<i>k</i><sub>2</sub> &lt;&lt; <i>k</i><sub>&minus;1</sub>.  The overall rate
constant A &rarr; C will be
<i>k</i><sub>1</sub><i>k</i><sub>2</sub>/<i>k</i><sub>&minus;1</sub>
and a new RRKM expression becomes an approximation:
<p class="eqnserif">
<i>k</i>(<i>E</i>) = &sigma;[<i>N</i><sup>&Dagger;</sup> (<i>TS</i><sub>BC</sub>)]
&divide; [<i>h</i>&rho;(<i>A</i>)]
<p>
Participation of other isomerization/tautomerization steps with other TS values
at lower energies than TS<sub>BC</sub> would not affect rate constant in the
above kinetic scheme, but the rate equation will not hold if
<i>k</i><sub>3</sub> &lt;&lt; <i>k</i><sub>&minus;2</sub> does not hold.
If C &rarr; D + E is rate-determining, then <i>N</i><sup>&Dagger;</sup>(TS<sub>CD</sub>)
is a better calculation than <i>N</i><sup>&Dagger;</sup>(TS<sub>BC</sub>).
If the molecular internal energy increases, then TS<sub>BC</sub> is better
because it is a <q>tight</q> TC with zero of negative
&Delta;<i>S</i><sup>&Dagger;</sup> than TS<sub>CD</sub> with a <q>loose</q>
TS having positive &Delta;<i>S</i><sup>&Dagger;</sup>.  Note that the difference
is related to time scale on the reactions:  some are nanosecond-to-microsecond
while other are millisecond-to-second.
<p>
In work with PSD (post-source decay) and UV-PD
<!-- ============================================================= -->
<hr style="margin-top:4em;">
<p>
This is the first in a series of documents containing notes (more than
abstracts but less than the full published reports) on the data and results
in papers relevant to research on methods and methodologies in proteomics,
or in which proteomics is used as a tool.  The (HTML) titles for these
documents are:

<ul>
<li>Notes on the Literature in Proteomics (Vol 1: 19xx-2006)
 <br>[this document:
 <span class="tt"><a href="ProteomicsLit(19xx-2006).html">ProteomicsLit(19xx-2006).html</a>]</span>
<li>Notes on the Literature in Proteomics (Vol 2: 2007-)
 <br>[this document:  <span class="tt">ProteomicsLit(2007-).html</span>]
</ul>

</body>
</html>


